

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/3/243/393500","@type":"ScholarlyArticle","name":"Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA","datePublished":"2013-12-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2014-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst054","keywords":["whole-genome amplification","Illumina","SNPs","genome assembly","chimeric DNA"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/3/243/393500","author":[{"name":"Tsai, Isheng J.","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK    2  Faculty of Medicine, Division of Parasitology, Department of Infectious Disease ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Hunt, Martin","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK","@type":"Person"},{"name":"Holroyd, Nancy","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK","@type":"Person"},{"name":"Huckvale, Thomas","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK","@type":"Person"},{"name":"Berriman, Matthew","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK","@type":"Person"},{"name":"Kikuchi, Taisei","affiliation":"1  Parasite Genomics, Wellcome Trust Sanger Institute ,  Wellcome Trust Genome Campus, Cambridge CB10 1SA ,  UK    2  Faculty of Medicine, Division of Parasitology, Department of Infectious Disease ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"}],"description":"Abstract.  Advances in both high-throughput sequencing and whole-genome amplification (WGA) protocols have allowed genomes to be sequenced from femtograms of DN","pageStart":"243","pageEnd":"254","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA" />
    <meta property="og:description" content="Abstract.  Advances in both high-throughput sequencing and whole-genome amplification (WGA) protocols have allowed genomes to be sequenced from femtograms of DN" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/3/243/393500" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-12-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tsai, Isheng J." /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author_institution" content="Faculty of Medicine, Division of Parasitology, Department of Infectious Disease, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Hunt, Martin" /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author" content="Holroyd, Nancy" /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author" content="Huckvale, Thomas" /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author" content="Berriman, Matthew" /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author" content="Kikuchi, Taisei" /><meta name="citation_author_institution" content="Parasite Genomics, Wellcome Trust Sanger Institute, Wellcome Trust Genome Campus, Cambridge CB10 1SA, UK" /><meta name="citation_author_institution" content="Faculty of Medicine, Division of Parasitology, Department of Infectious Disease, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_title" content="Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA" /><meta name="citation_firstpage" content="243" /><meta name="citation_lastpage" content="254" /><meta name="citation_doi" content="10.1093/dnares/dst054" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2014/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Illumina HiSeq Sequencing Systems http://res.illumina.com/documents/systems/hiseq/datasheet_hiseq_systems.pdf " /><meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e24" /><meta name="citation_reference" content="Illumina 2009 Mate Pair Library V2 Sample Preparation Guide " /><meta name="citation_reference" content="citation_title=Generation of long insert pairs using a Cre-LoxP Inverse PCR approach; PloS One;  citation_year=2012;  citation_volume=7;  citation_pages=e29437" /><meta name="citation_reference" content="citation_title=Rapid, high-throughput library preparation for next-generation sequencing; Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=iii-iv; " /><meta name="citation_reference" content="citation_title=Whole-genome sequences of Chlamydia trachomatis directly from clinical samples without culture; Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=855-66; " /><meta name="citation_reference" content="citation_title=Molecular hyperdiversity defines populations of the nematode Caenorhabditis brenneri; Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=11056-60; " /><meta name="citation_reference" content="citation_title=Whole genome amplification—applications and advances; Curr. Opin. Biotechnol.;  citation_year=2002;  citation_volume=13;  citation_pages=65-7; " /><meta name="citation_reference" content="citation_title=Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer; Genomics;  citation_year=1992;  citation_volume=13;  citation_pages=718-25; " /><meta name="citation_reference" content="citation_title=Whole genome amplification from a single cell: implications for genetic analysis; Proc. Natl. Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=5847-51; " /><meta name="citation_reference" content="citation_title=Something from (almost) nothing: the impact of multiple displacement amplification on microbial ecology; ISME J.;  citation_year=2008;  citation_volume=2;  citation_pages=233-41; " /><meta name="citation_reference" content="citation_title=Primase-based whole genome amplification; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=e79" /><meta name="citation_reference" content="citation_title=Genomic sequencing of uncultured microorganisms from single cells; Nat. Rev. Microbiol.;  citation_year=2012;  citation_volume=10;  citation_pages=631-40; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing and microarray analysis of ex vivo Plasmodium vivax reveal selective pressure on putative drug resistance genes; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=20045-50; " /><meta name="citation_reference" content="citation_title=Genome-wide single-cell analysis of recombination activity and de novo mutation rates in human sperm; Cell;  citation_year=2012;  citation_volume=150;  citation_pages=402-12; " /><meta name="citation_reference" content="citation_title=Assessment of whole genome amplification-induced bias through high-throughput, massively parallel whole genome sequencing; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=216" /><meta name="citation_reference" content="citation_title=Mechanism of chimera formation during the Multiple Displacement Amplification reaction; BMC Biotechnol.;  citation_year=2007;  citation_volume=7;  citation_pages=19" /><meta name="citation_reference" content="citation_title=A procedure for highly specific, sensitive, and unbiased whole-genome amplification; Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=15499-504; " /><meta name="citation_reference" content="citation_title=An improved approach to mate-paired library preparation for Illumina sequencing; Methods Next Gener. Seq.;  citation_year=2013;  citation_volume=1;  citation_pages=10-20; " /><meta name="citation_reference" content="citation_title=RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=W622-7; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=568-76; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1861-9; " /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; Bioinformatics (Oxford, England);  citation_year=2010;  citation_volume=26;  citation_pages=841-2; " /><meta name="citation_reference" content="Team R.C. 2013 R: A Language and Environment for Statistical Computing " /><meta name="citation_reference" content="citation_title=SSAHA: a fast search method for large DNA databases; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1725-9; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of large genomes using compressed data structures; Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=549-56; " /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE; Bioinformatics (Oxford, England);  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=GAGE: a critical evaluation of genome assemblies and assembly algorithms; Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=557-67; " /><meta name="citation_reference" content="citation_title=Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=291-5; " /><meta name="citation_reference" content="citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Sequence-specific error profile of Illumina sequencers; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e90" /><meta name="citation_reference" content="citation_title=Genome sequence of the nematode C. elegans: a platform for investigating biology; Science;  citation_year=1998;  citation_volume=282;  citation_pages=2012-8; " /><meta name="citation_reference" content="citation_title=Summarizing and correcting the GC content bias in high-throughput sequencing; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e72" /><meta name="citation_reference" content="citation_title=Broad chromosomal domains of histone modification patterns in C. elegans; Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=227-36; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/3/243/1107672/dst054.pdf" /><meta name="description" content="Abstract.  Advances in both high-throughput sequencing and whole-genome amplification (WGA) protocols have allowed genomes to be sequenced from femtograms of DN" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/3/243/393500" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/3/243/393500" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/3/243/393500");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst054';
    </script>
<title>
	Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./393500" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BftSPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc2FpLCBJc2hlbmcgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQYXJhc2l0ZSBHZW5vbWljcywgV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgV2VsbGNvbWUgVHJ1c3QgR2Vub21lIENhbXB1cywgQ2FtYnJpZGdlIENCMTAgMVNBLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgTWVkaWNpbmUsIERpdmlzaW9uIG9mIFBhcmFzaXRvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2UsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1bnQsIE1hcnRpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBhcmFzaXRlIEdlbm9taWNzLCBXZWxsY29tZSBUcnVzdCBTYW5nZXIgSW5zdGl0dXRlLCBXZWxsY29tZSBUcnVzdCBHZW5vbWUgQ2FtcHVzLCBDYW1icmlkZ2UgQ0IxMCAxU0EsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9scm95ZCwgTmFuY3kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQYXJhc2l0ZSBHZW5vbWljcywgV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgV2VsbGNvbWUgVHJ1c3QgR2Vub21lIENhbXB1cywgQ2FtYnJpZGdlIENCMTAgMVNBLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1Y2t2YWxlLCBUaG9tYXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQYXJhc2l0ZSBHZW5vbWljcywgV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgV2VsbGNvbWUgVHJ1c3QgR2Vub21lIENhbXB1cywgQ2FtYnJpZGdlIENCMTAgMVNBLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJlcnJpbWFuLCBNYXR0aGV3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGFyYXNpdGUgR2Vub21pY3MsIFdlbGxjb21lIFRydXN0IFNhbmdlciBJbnN0aXR1dGUsIFdlbGxjb21lIFRydXN0IEdlbm9tZSBDYW1wdXMsIENhbWJyaWRnZSBDQjEwIDFTQSwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaWt1Y2hpLCBUYWlzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQYXJhc2l0ZSBHZW5vbWljcywgV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgV2VsbGNvbWUgVHJ1c3QgR2Vub21lIENhbXB1cywgQ2FtYnJpZGdlIENCMTAgMVNBLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgTWVkaWNpbmUsIERpdmlzaW9uIG9mIFBhcmFzaXRvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2UsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iU3VtbWFyaXppbmcgU3BlY2lmaWMgUHJvZmlsZXMgaW4gSWxsdW1pbmEgU2VxdWVuY2luZyBmcm9tIFdob2xlLUdlbm9tZSBBbXBsaWZpZWQgRE5BIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IklsbHVtaW5hIEhpU2VxIFNlcXVlbmNpbmcgU3lzdGVtcyBodHRwOi8vcmVzLmlsbHVtaW5hLmNvbS9kb2N1bWVudHMvc3lzdGVtcy9oaXNlcS9kYXRhc2hlZXRfaGlzZXFfc3lzdGVtcy5wZGYgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWxsdW1pbmEgbWF0ZS1wYWlyZWQgRE5BIHNlcXVlbmNpbmctbGlicmFyeSBwcmVwYXJhdGlvbiB1c2luZyBDcmUtTG94IHJlY29tYmluYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPWUyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IklsbHVtaW5hIDIwMDkgTWF0ZSBQYWlyIExpYnJhcnkgVjIgU2FtcGxlIFByZXBhcmF0aW9uIEd1aWRlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYXRpb24gb2YgbG9uZyBpbnNlcnQgcGFpcnMgdXNpbmcgYSBDcmUtTG94UCBJbnZlcnNlIFBDUiBhcHByb2FjaDsgUGxvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUyOTQzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkLCBoaWdoLXRocm91Z2hwdXQgbGlicmFyeSBwcmVwYXJhdGlvbiBmb3IgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9aWlpLWl2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2VzIG9mIENobGFteWRpYSB0cmFjaG9tYXRpcyBkaXJlY3RseSBmcm9tIGNsaW5pY2FsIHNhbXBsZXMgd2l0aG91dCBjdWx0dXJlOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTg1NS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGh5cGVyZGl2ZXJzaXR5IGRlZmluZXMgcG9wdWxhdGlvbnMgb2YgdGhlIG5lbWF0b2RlIENhZW5vcmhhYmRpdGlzIGJyZW5uZXJpOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz0xMTA1Ni02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFtcGxpZmljYXRpb27igJRhcHBsaWNhdGlvbnMgYW5kIGFkdmFuY2VzOyBDdXJyLiBPcGluLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTY1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZ2VuZXJhdGUgb2xpZ29udWNsZW90aWRlLXByaW1lZCBQQ1I6IGdlbmVyYWwgYW1wbGlmaWNhdGlvbiBvZiB0YXJnZXQgRE5BIGJ5IGEgc2luZ2xlIGRlZ2VuZXJhdGUgcHJpbWVyOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTcxOC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFtcGxpZmljYXRpb24gZnJvbSBhIHNpbmdsZSBjZWxsOiBpbXBsaWNhdGlvbnMgZm9yIGdlbmV0aWMgYW5hbHlzaXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9NTg0Ny01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29tZXRoaW5nIGZyb20gKGFsbW9zdCkgbm90aGluZzogdGhlIGltcGFjdCBvZiBtdWx0aXBsZSBkaXNwbGFjZW1lbnQgYW1wbGlmaWNhdGlvbiBvbiBtaWNyb2JpYWwgZWNvbG9neTsgSVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmltYXNlLWJhc2VkIHdob2xlIGdlbm9tZSBhbXBsaWZpY2F0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1lNzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHNlcXVlbmNpbmcgb2YgdW5jdWx0dXJlZCBtaWNyb29yZ2FuaXNtcyBmcm9tIHNpbmdsZSBjZWxsczsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02MzEtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nIGFuZCBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIGV4IHZpdm8gUGxhc21vZGl1bSB2aXZheCByZXZlYWwgc2VsZWN0aXZlIHByZXNzdXJlIG9uIHB1dGF0aXZlIGRydWcgcmVzaXN0YW5jZSBnZW5lczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MjAwNDUtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHNpbmdsZS1jZWxsIGFuYWx5c2lzIG9mIHJlY29tYmluYXRpb24gYWN0aXZpdHkgYW5kIGRlIG5vdm8gbXV0YXRpb24gcmF0ZXMgaW4gaHVtYW4gc3Blcm07IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTUwOyAgY2l0YXRpb25fcGFnZXM9NDAyLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIHdob2xlIGdlbm9tZSBhbXBsaWZpY2F0aW9uLWluZHVjZWQgYmlhcyB0aHJvdWdoIGhpZ2gtdGhyb3VnaHB1dCwgbWFzc2l2ZWx5IHBhcmFsbGVsIHdob2xlIGdlbm9tZSBzZXF1ZW5jaW5nOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBjaGltZXJhIGZvcm1hdGlvbiBkdXJpbmcgdGhlIE11bHRpcGxlIERpc3BsYWNlbWVudCBBbXBsaWZpY2F0aW9uIHJlYWN0aW9uOyBCTUMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9jZWR1cmUgZm9yIGhpZ2hseSBzcGVjaWZpYywgc2Vuc2l0aXZlLCBhbmQgdW5iaWFzZWQgd2hvbGUtZ2Vub21lIGFtcGxpZmljYXRpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTE1NDk5LTUwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW1wcm92ZWQgYXBwcm9hY2ggdG8gbWF0ZS1wYWlyZWQgbGlicmFyeSBwcmVwYXJhdGlvbiBmb3IgSWxsdW1pbmEgc2VxdWVuY2luZzsgTWV0aG9kcyBOZXh0IEdlbmVyLiBTZXEuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9iaU5BOiBhIHVzZXItZnJpZW5kbHksIGludGVncmF0ZWQgc29mdHdhcmUgc29sdXRpb24gZm9yIFJOQS1TZXEtYmFzZWQgdHJhbnNjcmlwdG9taWNzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1XNjIyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHZW5vbWUgQW5hbHlzaXMgVG9vbGtpdDogYSBNYXBSZWR1Y2UgZnJhbWV3b3JrIGZvciBhbmFseXppbmcgbmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIGRhdGE7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTI5Ny0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhclNjYW4gMjogc29tYXRpYyBtdXRhdGlvbiBhbmQgY29weSBudW1iZXIgYWx0ZXJhdGlvbiBkaXNjb3ZlcnkgaW4gY2FuY2VyIGJ5IGV4b21lIHNlcXVlbmNpbmc7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NTY4LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0cyBmaW5kZXI6IGEgcHJvZ3JhbSB0byBhbmFseXplIEROQSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXJ0ZWQgcmVwZWF0IHN0cnVjdHVyZSBvZiB0aGUgaHVtYW4gZ2Vub21lOiB0aGUgWC1jaHJvbW9zb21lIGNvbnRhaW5zIGEgcHJlcG9uZGVyYW5jZSBvZiBsYXJnZSwgaGlnaGx5IGhvbW9sb2dvdXMgaW52ZXJ0ZWQgcmVwZWF0cyB0aGF0IGNvbnRhaW4gdGVzdGVzIGdlbmVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE4NjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkVEVG9vbHM6IGEgZmxleGlibGUgc3VpdGUgb2YgdXRpbGl0aWVzIGZvciBjb21wYXJpbmcgZ2Vub21pYyBmZWF0dXJlczsgQmlvaW5mb3JtYXRpY3MgKE94Zm9yZCwgRW5nbGFuZCk7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz04NDEtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVGVhbSBSLkMuIDIwMTMgUjogQSBMYW5ndWFnZSBhbmQgRW52aXJvbm1lbnQgZm9yIFN0YXRpc3RpY2FsIENvbXB1dGluZyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TU0FIQTogYSBmYXN0IHNlYXJjaCBtZXRob2QgZm9yIGxhcmdlIEROQSBkYXRhYmFzZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTcyNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGUgbm92byBhc3NlbWJseSBvZiBsYXJnZSBnZW5vbWVzIHVzaW5nIGNvbXByZXNzZWQgZGF0YSBzdHJ1Y3R1cmVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NhZmZvbGRpbmcgcHJlLWFzc2VtYmxlZCBjb250aWdzIHVzaW5nIFNTUEFDRTsgQmlvaW5mb3JtYXRpY3MgKE94Zm9yZCwgRW5nbGFuZCk7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0FHRTogYSBjcml0aWNhbCBldmFsdWF0aW9uIG9mIGdlbm9tZSBhc3NlbWJsaWVzIGFuZCBhc3NlbWJseSBhbGdvcml0aG1zOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU1Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1wbGlmaWNhdGlvbi1mcmVlIElsbHVtaW5hIHNlcXVlbmNpbmctbGlicmFyeSBwcmVwYXJhdGlvbiBmYWNpbGl0YXRlcyBpbXByb3ZlZCBtYXBwaW5nIGFuZCBhc3NlbWJseSBvZiAoRytDKS1iaWFzZWQgZ2Vub21lczsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yOTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemluZyBJbGx1bWluYSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBsaWJyYXJ5IHByZXBhcmF0aW9uIGZvciBleHRyZW1lbHkgQVQtYmlhc2VkIGdlbm9tZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1zcGVjaWZpYyBlcnJvciBwcm9maWxlIG9mIElsbHVtaW5hIHNlcXVlbmNlcnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPWU5MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbmVtYXRvZGUgQy4gZWxlZ2FuczogYSBwbGF0Zm9ybSBmb3IgaW52ZXN0aWdhdGluZyBiaW9sb2d5OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI4MjsgIGNpdGF0aW9uX3BhZ2VzPTIwMTItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VtbWFyaXppbmcgYW5kIGNvcnJlY3RpbmcgdGhlIEdDIGNvbnRlbnQgYmlhcyBpbiBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTcyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJvYWQgY2hyb21vc29tYWwgZG9tYWlucyBvZiBoaXN0b25lIG1vZGlmaWNhdGlvbiBwYXR0ZXJucyBpbiBDLiBlbGVnYW5zOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzMvMjQzLzExMDc2NzIvZHN0MDU0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBZHZhbmNlcyBpbiBib3RoIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIGFuZCB3aG9sZS1nZW5vbWUgYW1wbGlmaWNhdGlvbiAoV0dBKSBwcm90b2NvbHMgaGF2ZSBhbGxvd2VkIGdlbm9tZXMgdG8gYmUgc2VxdWVuY2VkIGZyb20gZmVtdG9ncmFtcyBvZiBETiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzkzNTAwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZI8YKb+Yn1S2X2ILQpZT+GqoyRK/" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f243%2f393500" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f243%2f393500" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15369" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&amp;Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/3">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5805458">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805458">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805460">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805460">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805464">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805474">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805474">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805500">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805500">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805511">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805511">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805513">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805513">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805515">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805515">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805517">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805517">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5805519">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5805519">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/3/231/393171">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/3/255/393600">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Isheng J. Tsai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Medicine, Division of Parasitology, Department of Infectious Disease</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Isheng+J.+Tsai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsai I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsai I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Isheng J. Tsai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin Hunt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+Hunt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hunt M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hunt M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin Hunt</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nancy Holroyd</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nancy+Holroyd">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Holroyd N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Holroyd N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nancy Holroyd</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Thomas Huckvale</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Thomas+Huckvale">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huckvale T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huckvale T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Thomas Huckvale</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew Berriman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+Berriman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Berriman M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Berriman M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Matthew Berriman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taisei Kikuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Parasite Genomics, Wellcome Trust Sanger Institute</div>, <div class="addr-line">Wellcome Trust Genome Campus, Cambridge CB10 1SA</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Faculty of Medicine, Division of Parasitology, Department of Infectious Disease</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 985850990. Fax. +81 985843887. Email: <a href="mailto:tk6@sanger.ac.uk" target="_blank">tk6@sanger.ac.uk</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taisei+Kikuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taisei Kikuchi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">The authors wish it to be known that, in their opinion, the first two authors should be regarded as joint First Authors.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 243–254, <a href='https://doi.org/10.1093/dnares/dst054'>https://doi.org/10.1093/dnares/dst054</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 December 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 August 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 November 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="393500" href="/dnaresearch/article-pdf/21/3/243/1107672/dst054.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Isheng J. Tsai, Martin Hunt, Nancy Holroyd, Thomas Huckvale, Matthew Berriman, Taisei Kikuchi;  Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA, <em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 243–254, <a href='https://doi.org/10.1093/dnares/dst054'>https://doi.org/10.1093/dnares/dst054</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/393500?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/393500?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Summarizing%20Specific%20Profiles%20in%20Illumina%20Sequencing%20from%20Whole-Genome%20Amplified%20DNA&amp;publicationDate=2013-12-18&amp;volumeNum=21&amp;issueNum=3&amp;author=Tsai%2C%20Isheng%20J.%3B%20Hunt%2C%20Martin&amp;startPage=243&amp;endPage=254&amp;contentId=10.1093%2Fdnares%2Fdst054&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f243%2f393500" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5805458 id="5805458" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Advances in both high-throughput sequencing and whole-genome amplification (WGA) protocols have allowed genomes to be sequenced from femtograms of DNA, for example from individual cells or from precious clinical and archived samples. Using the highly curated <em>Caenorhabditis elegans</em> genome as a reference, we have sequenced and identified errors and biases associated with Illumina library construction, library insert size, different WGA methods and genome features such as GC bias and simple repeat content. Detailed analysis of the reads from amplified libraries revealed characteristics suggesting that majority of amplified fragment ends are identical but inverted versions of each other. Read coverage in amplified libraries is correlated with both tandem and inverted repeat content, while GC content only influences sequencing in long-insert libraries. Nevertheless, single nucleotide polymorphism (SNP) calls and assembly metrics from reads in amplified libraries show comparable results with unamplified libraries. To utilize the full potential of WGA to reveal the real biological interest, this article highlights the importance of recognizing additional sources of errors from amplified sequence reads and discusses the potential implications in downstream analyses.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome amplification&quot;">whole-genome amplification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Illumina">Illumina</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNPs">SNPs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome assembly&quot;">genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chimeric DNA&quot;">chimeric DNA</a></div>                            <h2 scrollto-destination=5805460 id="5805460" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The use of genomic data generated by so-called ‘next generation sequencing’ (NGS) has become commonplace in many fields of biological research, with sequencing-by-synthesis from Illumina currently the most popular. A standard Illumina paired-end (PE) library is made from DNA templates of about 500 bp in length, and a sequencing run can generate billions of paired reads of length 37–250 bp from either ends of these fragments.<sup><a reveal-id="DST054C1" data-open="DST054C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Reads from longer fragments of DNA can also be produced to aid the deconvolution of repetitive regions and for identifying large structural variations in genomes. A specialized mate-pair (MP) library, constructed by introducing a circularization step at the start of the library preparation, allows end sequencing from fragments of at least 2 kb.<sup><a reveal-id="DST054C2  DST054C4" data-open="DST054C2  DST054C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> This powerful technology can be applied to address a wide range of biological questions, such as variant calling and resolving haplotypes between individuals of a population or <em>de novo</em> assembly of complex genomes.</p><p>Advancement in library preparation also permits their creation from just a few nanograms of DNA.<sup><a reveal-id="DST054C5" data-open="DST054C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Nevertheless, obtaining even nanograms of starting material can be challenging in certain applications. One solution is to pool many samples to obtain sufficient DNA to construct one library. However, this approach is often not applicable to rare clinical or archived samples,<sup><a reveal-id="DST054C6" data-open="DST054C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and increases the complexity of downstream analysis. Within an assembly of pooled DNA samples, it can be particularly challenging to distinguish variants of a sequence that is repeated in the genome of one individual from allelic differences between multiple individuals. This problem increases with the levels of intraspecies variation, for instance, <em>C. brenneri</em> has 14.1% of polymorphic synonymous sites between individuals, comparable with hyperdiverse bacteria.<sup><a reveal-id="DST054C7" data-open="DST054C7" class="link link-ref link-reveal xref-bibr">7</a></sup> A potential solution is to use whole-genome amplification (WGA) techniques to reduce the amount of DNA required to make a sequencing library. Several WGA protocols have been proposed<sup><a reveal-id="DST054C8" data-open="DST054C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and can be divided into those based on polymerase change reaction (PCR) or those involving multiple strand displacement amplification (MDA). PCR-based techniques such as degenerate oligonucleotide primed PCR<sup><a reveal-id="DST054C9" data-open="DST054C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and primer extension PCR<sup><a reveal-id="DST054C10" data-open="DST054C10" class="link link-ref link-reveal xref-bibr">10</a></sup> can produce non-specific amplification artefacts and typically short-amplification products.<sup><a reveal-id="DST054C11" data-open="DST054C11" class="link link-ref link-reveal xref-bibr">11</a></sup> MDA uses the strand-displacing DNA polymerase from Phi29 bacteriophage and has several advantages over PCR-based protocols. For example, MDA can generate long-amplified fragments up to 100 kb, which are of feasible size to serve as templates from which MP libraries can be synthesized.<sup><a reveal-id="DST054C8" data-open="DST054C8" class="link link-ref link-reveal xref-bibr">8</a></sup> An alternative method of amplification called pWGA (primase-based WGA)<sup><a reveal-id="DST054C12" data-open="DST054C12" class="link link-ref link-reveal xref-bibr">12</a></sup> is based on <em>in vitro</em> reconstruction of the naturally existing cellular DNA replication machinery found in bacteriophage T7, and comparable performance with MDA in terms of efficiency and unbiased amplification has been reported.<sup><a reveal-id="DST054C12" data-open="DST054C12" class="link link-ref link-reveal xref-bibr">12</a></sup> WGA techniques have already been regularly used in genotyping bacteria,<sup><a reveal-id="DST054C13" data-open="DST054C13" class="link link-ref link-reveal xref-bibr">13</a></sup> other small organisms<sup><a reveal-id="DST054C14" data-open="DST054C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and human cells.<sup><a reveal-id="DST054C15" data-open="DST054C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Though still in its infancy, sequencing from single cells is now feasible using WGA and this opens up a new field of exploring heterogeneity within populations, or populations of cells in an organism or tissue, an ultimate goal for many geneticists and molecular biologists.</p><p>Despite the advances in WGA methods, preferential amplification of certain DNA fragments still exists,<sup><a reveal-id="DST054C16" data-open="DST054C16" class="link link-ref link-reveal xref-bibr">16</a></sup> resulting in uneven sequencing between different parts of a given genome and chimera formation.<sup><a reveal-id="DST054C17" data-open="DST054C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In this article we describe the effect of amplifying nanograms of genomic DNA, which is usually the available starting material of a <em>de novo</em> small eukaryote genome project, with three different protocols before constructing Illumina short- and long-insert libraries. Using the high quality genome sequence of <em>C. elegans</em> as a reference, we show that GC content and composition of inverted and tandem repeats play a major role in the variation of read coverage. We demonstrate that reads sequenced from amplified DNA can generate <em>de novo</em> assemblies that are of comparable quality with those from unamplified DNA. In addition, we have also examined capillary reads from libraries cloned from amplified high molecular weight DNA of the potato cyst nematode <em>Globodera pallida</em> genome project. As the research focus is turning to more unexplored organisms and single cells, WGA protocols will become the standard method of choice. Understanding the caveats and biases generated with these protocols is a paramount requirement for interpretation of results.</p>                            <h2 scrollto-destination=5805464 id="5805464" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5805465 id="5805465" class="section-title" data-legacy-id=s2a>2.1. Whole-genome amplification</h3>
<p>Genomic DNA was extracted from ∼10 000 <em>C. elegans</em> N2 nematodes using Genomic tip G20 (Qiagen) according to the manufacturer’s instructions. Ten nanograms of template DNA was used in each of three WGA reactions using: (i) GenomiPhi v2 (GE life science), hereafter termed Phi; (ii) Phi29 MDA plus trehalose, hereafter termed Tre; and (iii) Rapisome (BioHelix), hereafter termed Rap. The Phi WGA reactions were carried out in the 20 µl reaction mixture at 30°C for 90 min followed by heat denaturation at 95°C for 3 min according to the manufacturer's instructions. For Rap, DNA was mixed directly with the 25 µl reaction mixture and incubated at 37°C for 60 min according to the manufacturer's instructions. For Tre, Phi29 polymerase (Qiagen), 0.7 M trehalose and 16 h reaction time were used according to the method described in Pan <em>et al.</em><sup><a reveal-id="DST054C18" data-open="DST054C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Amplified products were purified using a QIAAmp DNA mini kit (Qiagen) and DNA concentrations were measured using Qubit (Life technologies).</p>                            <h3 scrollto-destination=5805467 id="5805467" class="section-title" data-legacy-id=s2b>2.2. Illumina library construction and sequencing</h3>
<p>One microgram of DNA was used to construct standard 450 bp libraries using a TruSeq DNA Sample Preparation Kit with the standard protocol (Illumina), after fragmentation on the Covaris, 3 kb mate-pair libraries were constructed following the protocol described in Park <em>et al.</em><sup><a reveal-id="DST054C19" data-open="DST054C19" class="link link-ref link-reveal xref-bibr">19</a></sup> with the following exceptions. Size selection was carried out with Agencourt AMPure XP beads (Beckman Coulter) in a buffer of 5% PEG and 0.95 M NaCl, aiming to remove most fragments of length &lt;1.5 kb. Nick translation was carried out for 11 min (libraries in Replicate 1) or 14 min (libraries in Replicate 2). Libraries were sequenced using the TruSeq SBS Kit v3-HS kit according to the manufacturer's recommended protocol (<a class="link link-uri" href="https://icom.illumina.com/">https://icom.illumina.com/</a>) in either Illumina MiSeq 150 cycles (for 3 kb MP libraries in Replicate 2) or the Illumina HiSeq 100 cycles (for all other libraries). <em>In situ</em>, the linearization, blocking and hybridization step was repeated to regenerate clusters, release the second strand for sequencing and to hybridize the R2 sequencing primer followed by another 100 or 150 cycles of sequencing to produce PE reads.</p>                            <h3 scrollto-destination=5805469 id="5805469" class="section-title" data-legacy-id=s2c>2.3. Data analysis</h3>
<p>Reads were trimmed based on base quality (an average phred score of 15 for every four bases) and the presence of adaptor sequences. For 450 bp fragment short-insert libraries Trimmomatic<sup><a reveal-id="DST054C20" data-open="DST054C20" class="link link-ref link-reveal xref-bibr">20</a></sup> was used with options: ‘<em>ILLUMINACLIP 2:40:15 LEADING:3 TRAILING:3 SLIDINGWINDOW:4:15 MINLEN:36</em>’. For 3 kb fragment long-insert libraries an in-house developed algorithm was used to trim biotin adaptor sequence which may present at either ends of the long-insert fragments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Three to 15% of reads were removed in short- and long-insert libraries, respectively, before undertaking the alignment stage (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>Illumina reads were aligned to the <em>C. elegans</em> reference genome (WS236 from Wormbase <a class="link link-uri" href="http://ftp://ftp.wormbase.org/">ftp://ftp.wormbase.org/</a>) using SMALT (<a class="link link-uri" href="http://www.sanger.ac.uk/resources/software/smalt/">http://www.sanger.ac.uk/resources/software/smalt/</a>). Repetitive mappings were allowed, and only alignments with at least 80% or 50% of the sequence read aligned to reference were considered in the short- and long-insert libraries, respectively. Additionally, a maximum of 600 bp and 100 000 bp insert size were considered for paired mappings in the short- and long-insert libraries, respectively. Duplicates were called using GATK.<sup><a reveal-id="DST054C21" data-open="DST054C21" class="link link-ref link-reveal xref-bibr">21</a></sup> SNPs were called using Varscan2<sup><a reveal-id="DST054C22" data-open="DST054C22" class="link link-ref link-reveal xref-bibr">22</a></sup> with options ‘−min-coverage 5—strand-filter 1’ using alignment files (in the form of bam format) as the input on each of the short-insert replicates. Tandem and inverted repeat content of the <em>C. elegans</em> genome was calculated using program trf<sup><a reveal-id="DST054C23" data-open="DST054C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and irf<sup><a reveal-id="DST054C24" data-open="DST054C24" class="link link-ref link-reveal xref-bibr">24</a></sup> with default parameters, respectively. GC content and repeat content analyses for 10 kb windows in the <em>C. elegans</em> genome was calculated using a combination of BEDTools<sup><a reveal-id="DST054C25" data-open="DST054C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and custom Perl and R<sup><a reveal-id="DST054C26" data-open="DST054C26" class="link link-ref link-reveal xref-bibr">26</a></sup> scripts. Capillary reads from the <em>G. pallida</em> genome project were first trimmed to increase bases having at least phred quality score of 40 and the vector contamination was removed. The resulting subset of reads was mapped against the <em>G. pallida</em> v1 assembly (<a class="link link-uri" href="http://ftp://ftp.sanger.ac.uk/pub/pathogens/Globodera/pallida/">ftp://ftp.sanger.ac.uk/pub/pathogens/Globodera/pallida/</a>) using SSAHA2.<sup><a reveal-id="DST054C27" data-open="DST054C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>Assemblies were constructed from short-insert libraries using SGA v.0.10.9<sup><a reveal-id="DST054C28" data-open="DST054C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with authors' recommended parameters (<a class="link link-uri" href="https://github.com/jts/sga/blob/master/src/examples/sga-celegans.sh">https://github.com/jts/sga/blob/master/src/examples/sga-celegans.sh</a>). Scaffolding was performed using SSPACE basic version 2.0<sup><a reveal-id="DST054C29" data-open="DST054C29" class="link link-ref link-reveal xref-bibr">29</a></sup> with default settings, with library sizes set to the median insert values as shown in Table <a reveal-id="DST054TB1" data-open="DST054TB1" class="link link-reveal link-table xref-fig">1</a> with a standard deviation of 0.5. These assemblies were then compared with the <em>C. elegans</em> genome (WS236) using GAGE.<sup><a reveal-id="DST054C30" data-open="DST054C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Because of the differences in number of reads between replicates, we only presented Replicate 1, which has more sequencing depth (Table <a reveal-id="DST054TB4" data-open="DST054TB4" class="link link-reveal link-table xref-fig">4</a>). A separate <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> summarizes assemblies where every library is normalized to the library with least number of reads. </p>                        <a id="5805473" scrollto-destination="5805473"></a>
<div content-id="DST054TB1" class="table-modal"><div class="table-wrap"><div id="DST054TB1" class="table-wrap-title" data-id="DST054TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Mapping statistics of sequenced reads from unamplified and amplified libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Platform </th><th>Library type </th><th>Total reads </th><th>Mapped (%) </th><th>Duplicates<sup>a</sup> (%) </th><th>Proper<sup>b</sup> (%) </th><th>Both mapped (%) </th><th>Median insert (bp) </th><th>Median coverage </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Short </td><td>41 688 676 </td><td>99.7 </td><td>0.8 </td><td>98.1 </td><td>99.5 </td><td>285 </td><td>38 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>16 333 732 </td><td>89.4 </td><td>0.6 </td><td>88 </td><td>89.3 </td><td>357 </td><td>13 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Short </td><td>48 501 916 </td><td>99.4 </td><td>0.7 </td><td>98 </td><td>98.8 </td><td>224 </td><td>46 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>20 656 296 </td><td>95.2 </td><td>0.8 </td><td>88.6 </td><td>93.7 </td><td>349 </td><td>18 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Short </td><td>44 481 270 </td><td>99.3 </td><td>1.1 </td><td>97.6 </td><td>98.7 </td><td>237 </td><td>40 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>25 188 788 </td><td>96.2 </td><td>0.7 </td><td>92 </td><td>95.3 </td><td>328 </td><td>22 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Short </td><td>26 277 398 </td><td>89.3 </td><td>2.8 </td><td>71.6 </td><td>80.1 </td><td>248 </td><td>15 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>22 278 134 </td><td>82.5 </td><td>35.7 </td><td>60 </td><td>73.7 </td><td>308 </td><td>3 </td></tr><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Long </td><td>60 856 860 </td><td>99.5 </td><td>9 </td><td>97.2 </td><td>99.1 </td><td>2631 </td><td>41 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 551 720 </td><td>86.4 </td><td>1 </td><td>79.1 </td><td>85.4 </td><td>2136 </td><td>2 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Long </td><td>61 735 210 </td><td>99.5 </td><td>3.5 </td><td>81.5 </td><td>99.2 </td><td>2576 </td><td>39 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 760 576 </td><td>99.3 </td><td>1.2 </td><td>80.2 </td><td>98.9 </td><td>2025 </td><td>2 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Long </td><td>55 842 586 </td><td>99.4 </td><td>3.4 </td><td>82.6 </td><td>99 </td><td>2285 </td><td>29 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 999 856 </td><td>99.4 </td><td>1 </td><td>73 </td><td>98.9 </td><td>2094 </td><td>2 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Long </td><td>58 443 656 </td><td>99.4 </td><td>7.6 </td><td>92.1 </td><td>99 </td><td>2591 </td><td>35 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>3 914 622 </td><td>99 </td><td>6.6 </td><td>91.8 </td><td>98.7 </td><td>2121 </td><td>2 </td></tr></tbody></table></div><div content-id="DST054TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Platform </th><th>Library type </th><th>Total reads </th><th>Mapped (%) </th><th>Duplicates<sup>a</sup> (%) </th><th>Proper<sup>b</sup> (%) </th><th>Both mapped (%) </th><th>Median insert (bp) </th><th>Median coverage </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Short </td><td>41 688 676 </td><td>99.7 </td><td>0.8 </td><td>98.1 </td><td>99.5 </td><td>285 </td><td>38 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>16 333 732 </td><td>89.4 </td><td>0.6 </td><td>88 </td><td>89.3 </td><td>357 </td><td>13 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Short </td><td>48 501 916 </td><td>99.4 </td><td>0.7 </td><td>98 </td><td>98.8 </td><td>224 </td><td>46 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>20 656 296 </td><td>95.2 </td><td>0.8 </td><td>88.6 </td><td>93.7 </td><td>349 </td><td>18 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Short </td><td>44 481 270 </td><td>99.3 </td><td>1.1 </td><td>97.6 </td><td>98.7 </td><td>237 </td><td>40 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>25 188 788 </td><td>96.2 </td><td>0.7 </td><td>92 </td><td>95.3 </td><td>328 </td><td>22 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Short </td><td>26 277 398 </td><td>89.3 </td><td>2.8 </td><td>71.6 </td><td>80.1 </td><td>248 </td><td>15 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>22 278 134 </td><td>82.5 </td><td>35.7 </td><td>60 </td><td>73.7 </td><td>308 </td><td>3 </td></tr><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Long </td><td>60 856 860 </td><td>99.5 </td><td>9 </td><td>97.2 </td><td>99.1 </td><td>2631 </td><td>41 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 551 720 </td><td>86.4 </td><td>1 </td><td>79.1 </td><td>85.4 </td><td>2136 </td><td>2 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Long </td><td>61 735 210 </td><td>99.5 </td><td>3.5 </td><td>81.5 </td><td>99.2 </td><td>2576 </td><td>39 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 760 576 </td><td>99.3 </td><td>1.2 </td><td>80.2 </td><td>98.9 </td><td>2025 </td><td>2 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Long </td><td>55 842 586 </td><td>99.4 </td><td>3.4 </td><td>82.6 </td><td>99 </td><td>2285 </td><td>29 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 999 856 </td><td>99.4 </td><td>1 </td><td>73 </td><td>98.9 </td><td>2094 </td><td>2 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Long </td><td>58 443 656 </td><td>99.4 </td><td>7.6 </td><td>92.1 </td><td>99 </td><td>2591 </td><td>35 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>3 914 622 </td><td>99 </td><td>6.6 </td><td>91.8 </td><td>98.7 </td><td>2121 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>All percentages are relative to total number of reads in each replicate.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>a</sup>Reads that are identical copies of other reads and have exact mapped coordinates on the genome.</p></div></div><div content-id="tbnfn3"><div class="fn"><p><sup>b</sup>Reads mapped in the correct orientation and at a distance corresponding to that predicted by the fragment library size.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805473" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST054TB1" class="table-wrap-title" data-id="DST054TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Mapping statistics of sequenced reads from unamplified and amplified libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Platform </th><th>Library type </th><th>Total reads </th><th>Mapped (%) </th><th>Duplicates<sup>a</sup> (%) </th><th>Proper<sup>b</sup> (%) </th><th>Both mapped (%) </th><th>Median insert (bp) </th><th>Median coverage </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Short </td><td>41 688 676 </td><td>99.7 </td><td>0.8 </td><td>98.1 </td><td>99.5 </td><td>285 </td><td>38 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>16 333 732 </td><td>89.4 </td><td>0.6 </td><td>88 </td><td>89.3 </td><td>357 </td><td>13 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Short </td><td>48 501 916 </td><td>99.4 </td><td>0.7 </td><td>98 </td><td>98.8 </td><td>224 </td><td>46 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>20 656 296 </td><td>95.2 </td><td>0.8 </td><td>88.6 </td><td>93.7 </td><td>349 </td><td>18 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Short </td><td>44 481 270 </td><td>99.3 </td><td>1.1 </td><td>97.6 </td><td>98.7 </td><td>237 </td><td>40 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>25 188 788 </td><td>96.2 </td><td>0.7 </td><td>92 </td><td>95.3 </td><td>328 </td><td>22 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Short </td><td>26 277 398 </td><td>89.3 </td><td>2.8 </td><td>71.6 </td><td>80.1 </td><td>248 </td><td>15 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>22 278 134 </td><td>82.5 </td><td>35.7 </td><td>60 </td><td>73.7 </td><td>308 </td><td>3 </td></tr><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Long </td><td>60 856 860 </td><td>99.5 </td><td>9 </td><td>97.2 </td><td>99.1 </td><td>2631 </td><td>41 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 551 720 </td><td>86.4 </td><td>1 </td><td>79.1 </td><td>85.4 </td><td>2136 </td><td>2 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Long </td><td>61 735 210 </td><td>99.5 </td><td>3.5 </td><td>81.5 </td><td>99.2 </td><td>2576 </td><td>39 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 760 576 </td><td>99.3 </td><td>1.2 </td><td>80.2 </td><td>98.9 </td><td>2025 </td><td>2 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Long </td><td>55 842 586 </td><td>99.4 </td><td>3.4 </td><td>82.6 </td><td>99 </td><td>2285 </td><td>29 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 999 856 </td><td>99.4 </td><td>1 </td><td>73 </td><td>98.9 </td><td>2094 </td><td>2 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Long </td><td>58 443 656 </td><td>99.4 </td><td>7.6 </td><td>92.1 </td><td>99 </td><td>2591 </td><td>35 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>3 914 622 </td><td>99 </td><td>6.6 </td><td>91.8 </td><td>98.7 </td><td>2121 </td><td>2 </td></tr></tbody></table></div><div content-id="DST054TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Platform </th><th>Library type </th><th>Total reads </th><th>Mapped (%) </th><th>Duplicates<sup>a</sup> (%) </th><th>Proper<sup>b</sup> (%) </th><th>Both mapped (%) </th><th>Median insert (bp) </th><th>Median coverage </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Short </td><td>41 688 676 </td><td>99.7 </td><td>0.8 </td><td>98.1 </td><td>99.5 </td><td>285 </td><td>38 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>16 333 732 </td><td>89.4 </td><td>0.6 </td><td>88 </td><td>89.3 </td><td>357 </td><td>13 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Short </td><td>48 501 916 </td><td>99.4 </td><td>0.7 </td><td>98 </td><td>98.8 </td><td>224 </td><td>46 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>20 656 296 </td><td>95.2 </td><td>0.8 </td><td>88.6 </td><td>93.7 </td><td>349 </td><td>18 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Short </td><td>44 481 270 </td><td>99.3 </td><td>1.1 </td><td>97.6 </td><td>98.7 </td><td>237 </td><td>40 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>25 188 788 </td><td>96.2 </td><td>0.7 </td><td>92 </td><td>95.3 </td><td>328 </td><td>22 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Short </td><td>26 277 398 </td><td>89.3 </td><td>2.8 </td><td>71.6 </td><td>80.1 </td><td>248 </td><td>15 </td></tr><tr><td> </td><td>2 </td><td>HiSeq </td><td>Short </td><td>22 278 134 </td><td>82.5 </td><td>35.7 </td><td>60 </td><td>73.7 </td><td>308 </td><td>3 </td></tr><tr><td>Unamplified </td><td>1 </td><td>HiSeq </td><td>Long </td><td>60 856 860 </td><td>99.5 </td><td>9 </td><td>97.2 </td><td>99.1 </td><td>2631 </td><td>41 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 551 720 </td><td>86.4 </td><td>1 </td><td>79.1 </td><td>85.4 </td><td>2136 </td><td>2 </td></tr><tr><td>Phi </td><td>1 </td><td>HiSeq </td><td>Long </td><td>61 735 210 </td><td>99.5 </td><td>3.5 </td><td>81.5 </td><td>99.2 </td><td>2576 </td><td>39 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 760 576 </td><td>99.3 </td><td>1.2 </td><td>80.2 </td><td>98.9 </td><td>2025 </td><td>2 </td></tr><tr><td>Tre </td><td>1 </td><td>HiSeq </td><td>Long </td><td>55 842 586 </td><td>99.4 </td><td>3.4 </td><td>82.6 </td><td>99 </td><td>2285 </td><td>29 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>2 999 856 </td><td>99.4 </td><td>1 </td><td>73 </td><td>98.9 </td><td>2094 </td><td>2 </td></tr><tr><td>Rap </td><td>1 </td><td>HiSeq </td><td>Long </td><td>58 443 656 </td><td>99.4 </td><td>7.6 </td><td>92.1 </td><td>99 </td><td>2591 </td><td>35 </td></tr><tr><td> </td><td>2 </td><td>MiSeq </td><td>Long </td><td>3 914 622 </td><td>99 </td><td>6.6 </td><td>91.8 </td><td>98.7 </td><td>2121 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p>All percentages are relative to total number of reads in each replicate.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>a</sup>Reads that are identical copies of other reads and have exact mapped coordinates on the genome.</p></div></div><div content-id="tbnfn3"><div class="fn"><p><sup>b</sup>Reads mapped in the correct orientation and at a distance corresponding to that predicted by the fragment library size.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805473" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5805474 id="5805474" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5805475 id="5805475" class="section-title" data-legacy-id=s3a>3.1. Genome amplification</h3>
<p><em>Caenorhabditis elegans</em> genomic DNA was amplified using three different protocols that utilized Phi29 polymerase alone (Phi); Phi29 polymerase supplemented with trehalose (Tre), which reportedly produces a more homogenous and unbiased amplification<sup><a reveal-id="DST054C18" data-open="DST054C18" class="link link-ref link-reveal xref-bibr">18</a></sup>; and Rapisome (Rap)—a commercially available version of pWGA, which does not utilize random primers and initial template denaturation that are possible factors in chimera formation in MDA reactions. After amplification, we observed DNA fragments of length&gt;10 kb resulting from all amplification methods. The average yield of amplified DNA from 10 ng template DNA was 46 ng/µl of reaction mixture (Phi), 31 ng/µl (Rap) and 95 ng/µl (Tre). These fragments were further fragmented and size selected to generate Illumina libraries (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=5805477 id="5805477" class="section-title" data-legacy-id=s3b>3.2. Illumina read quality</h3>
<p>A total of ∼495 million of both standard 450 bp PE (short-insert libraries) and 3 kb MP (long-insert libraries) Illumina reads were generated from amplified <em>C. elegans</em> genomic DNA fragments. Two technical replicates were generated from each amplification protocol and unamplified DNA. The availability of a high quality <em>C. elegans</em> reference genome allowed the quality of the libraries to be assessed by checking various mapping metrics such as number of reads mapped, orientation of mates in a pair. In all cases, at least 82% of the reads were aligned to the reference genome (Table <a reveal-id="DST054TB1" data-open="DST054TB1" class="link link-reveal link-table xref-fig">1</a>) and the insert size distribution, i.e. the distance between two mates of a read pair (including themselves) mapped to the genome in the correct orientation, was close to our expectation for selecting desired size fragments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p><p>Various errors can occur at different stages of Illumina library preparation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In both short- and long-insert library construction, duplicates can arise during PCR amplification, resulting in perfect copies of the original DNA templates being sequenced many times.<sup><a reveal-id="DST054C31" data-open="DST054C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Of the mapped reads in short-insert libraries, the number of PCR duplicates was ∼1%, except in Rap libraries, where Replicate 1 had 2.8% duplicates and Replicate 2 had 35.7% duplicates suggesting that the starting amplified products had lower molecular sizes with only a fraction of DNA desirable as DNA templates. We therefore considered Replicate 2 as a ‘failed’ library and excluded it from subsequent analysis. The number of PCR duplicates in long-insert libraries was consistently higher than in short-insert libraries, as observed by others.<sup><a reveal-id="DST054C4" data-open="DST054C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The number of PCR duplicates also appeared to be affected by individual runs; a consistently fewer duplicates were observed in the second replicate. Again, we found that PCR duplicates in Rap amplified long-insert libraries were high in both replicates.</p><p>Problematic reads can arise from several stages of long-insert library construction. First, fragments can be produced from circularized templates where nick translation has occurred in a segment that does not contain a biotinylated adaptor (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>) presumably the selection stages are sometimes ineffective. We found out that 0.5–4.8% of such reads present in long-insert libraries and they appear to be higher in Replicate 2 and not influenced by WGA protocols (Table <a reveal-id="DST054TB2" data-open="DST054TB2" class="link link-reveal link-table xref-fig">2</a>). Second, templates can still be circularized if size selection is not efficient (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Paired reads sequenced from these fragments should have short-insert sizes and be aligned in the correct (outward-facing in long-insert libraries) orientation. We found these reads to be present at much higher frequency in Replicate 2 which were sequenced as a batch (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). </p>                        <a id="5805481" scrollto-destination="5805481"></a>
<div content-id="DST054TB2" class="table-modal"><div class="table-wrap"><div id="DST054TB2" class="table-wrap-title" data-id="DST054TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Mapping statistics of improperly paired sequenced reads from unamplified and amplified libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Library type </th><th>Singletons (%) </th><th>Interchromosomal<sup>a</sup> (%) </th><th>Outies/innies<sup>b</sup> (%) </th><th>Wrong orientation<sup>c</sup> (%) </th><th>Incorrect insert size (%) </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>Short </td><td>0.2 </td><td>0.5 </td><td>0.5 </td><td>0.2 </td><td>0.2 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>0.2 </td><td>0.6 </td><td>0.3 </td><td>0.1 </td><td>0.3 </td></tr><tr><td>Phi </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.15 </td><td>0.1 </td><td>0.55 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1.5 </td><td>0.4 </td><td>0.3 </td><td>4.1 </td><td>0.3 </td></tr><tr><td>Tre </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.2 </td><td>0.1 </td><td>0.8 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1 </td><td>0.4 </td><td>0.3 </td><td>2.5 </td><td>0.1 </td></tr><tr><td>Rap </td><td>1 </td><td>Short </td><td>9.1 </td><td>4.2 </td><td>0.3 </td><td>3.7 </td><td>0.3 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>8.8 </td><td>0.5 </td><td>1.7 </td><td>3.1 </td><td>8.4 </td></tr><tr><td>Unamplified </td><td>1 </td><td>Long </td><td>0.4 </td><td>1 </td><td>0.5 </td><td>0.3 </td><td>0.1 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>1 </td><td>3.9 </td><td>1.4 </td><td>0.8 </td><td>0.2 </td></tr><tr><td>Phi </td><td>1 </td><td>Long </td><td>0.3 </td><td>2 </td><td>1.7 </td><td>13.3 </td><td>0.7 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>5.1 </td><td>3.1 </td><td>10.0 </td><td>0.5 </td></tr><tr><td>Tre </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.9 </td><td>1.0 </td><td>12.9 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>8.1 </td><td>4.8 </td><td>12.4 </td><td>0.6 </td></tr><tr><td>Rap </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.3 </td><td>0.5 </td><td>4.5 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.3 </td><td>3 </td><td>1.3 </td><td>2.2 </td><td>0.4 </td></tr></tbody></table></div><div content-id="DST054TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Library type </th><th>Singletons (%) </th><th>Interchromosomal<sup>a</sup> (%) </th><th>Outies/innies<sup>b</sup> (%) </th><th>Wrong orientation<sup>c</sup> (%) </th><th>Incorrect insert size (%) </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>Short </td><td>0.2 </td><td>0.5 </td><td>0.5 </td><td>0.2 </td><td>0.2 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>0.2 </td><td>0.6 </td><td>0.3 </td><td>0.1 </td><td>0.3 </td></tr><tr><td>Phi </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.15 </td><td>0.1 </td><td>0.55 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1.5 </td><td>0.4 </td><td>0.3 </td><td>4.1 </td><td>0.3 </td></tr><tr><td>Tre </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.2 </td><td>0.1 </td><td>0.8 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1 </td><td>0.4 </td><td>0.3 </td><td>2.5 </td><td>0.1 </td></tr><tr><td>Rap </td><td>1 </td><td>Short </td><td>9.1 </td><td>4.2 </td><td>0.3 </td><td>3.7 </td><td>0.3 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>8.8 </td><td>0.5 </td><td>1.7 </td><td>3.1 </td><td>8.4 </td></tr><tr><td>Unamplified </td><td>1 </td><td>Long </td><td>0.4 </td><td>1 </td><td>0.5 </td><td>0.3 </td><td>0.1 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>1 </td><td>3.9 </td><td>1.4 </td><td>0.8 </td><td>0.2 </td></tr><tr><td>Phi </td><td>1 </td><td>Long </td><td>0.3 </td><td>2 </td><td>1.7 </td><td>13.3 </td><td>0.7 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>5.1 </td><td>3.1 </td><td>10.0 </td><td>0.5 </td></tr><tr><td>Tre </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.9 </td><td>1.0 </td><td>12.9 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>8.1 </td><td>4.8 </td><td>12.4 </td><td>0.6 </td></tr><tr><td>Rap </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.3 </td><td>0.5 </td><td>4.5 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.3 </td><td>3 </td><td>1.3 </td><td>2.2 </td><td>0.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn4"><div class="fn"><p>All percentages are relative to total number of reads in each replicate shown in Table <a reveal-id="DST054TB1" data-open="DST054TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><div content-id="tbnfn5"><div class="fn"><p><sup>a</sup>Reads with mates mapped to different chromosomes.</p></div></div><div content-id="tbnfn6"><div class="fn"><p><sup>b</sup>Reads with mates mapped to the same chromosome that show incorrect orientation of facing either outwards (‘←→’; outies for short-insert libraries) or inwards (‘→←’ innies for long-insert libraries).</p></div></div><div content-id="tbnfn7"><div class="fn"><p><sup>c</sup>Reads with mates mapped to the same chromosome but shows the same orientation, i.e. ‘←←’ or ‘→→’. In the case of long-insert libraries, chimera formation is one of the causes of the formation of these reads.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805481" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST054TB2" class="table-wrap-title" data-id="DST054TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Mapping statistics of improperly paired sequenced reads from unamplified and amplified libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Library type </th><th>Singletons (%) </th><th>Interchromosomal<sup>a</sup> (%) </th><th>Outies/innies<sup>b</sup> (%) </th><th>Wrong orientation<sup>c</sup> (%) </th><th>Incorrect insert size (%) </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>Short </td><td>0.2 </td><td>0.5 </td><td>0.5 </td><td>0.2 </td><td>0.2 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>0.2 </td><td>0.6 </td><td>0.3 </td><td>0.1 </td><td>0.3 </td></tr><tr><td>Phi </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.15 </td><td>0.1 </td><td>0.55 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1.5 </td><td>0.4 </td><td>0.3 </td><td>4.1 </td><td>0.3 </td></tr><tr><td>Tre </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.2 </td><td>0.1 </td><td>0.8 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1 </td><td>0.4 </td><td>0.3 </td><td>2.5 </td><td>0.1 </td></tr><tr><td>Rap </td><td>1 </td><td>Short </td><td>9.1 </td><td>4.2 </td><td>0.3 </td><td>3.7 </td><td>0.3 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>8.8 </td><td>0.5 </td><td>1.7 </td><td>3.1 </td><td>8.4 </td></tr><tr><td>Unamplified </td><td>1 </td><td>Long </td><td>0.4 </td><td>1 </td><td>0.5 </td><td>0.3 </td><td>0.1 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>1 </td><td>3.9 </td><td>1.4 </td><td>0.8 </td><td>0.2 </td></tr><tr><td>Phi </td><td>1 </td><td>Long </td><td>0.3 </td><td>2 </td><td>1.7 </td><td>13.3 </td><td>0.7 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>5.1 </td><td>3.1 </td><td>10.0 </td><td>0.5 </td></tr><tr><td>Tre </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.9 </td><td>1.0 </td><td>12.9 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>8.1 </td><td>4.8 </td><td>12.4 </td><td>0.6 </td></tr><tr><td>Rap </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.3 </td><td>0.5 </td><td>4.5 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.3 </td><td>3 </td><td>1.3 </td><td>2.2 </td><td>0.4 </td></tr></tbody></table></div><div content-id="DST054TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Replicate </th><th>Library type </th><th>Singletons (%) </th><th>Interchromosomal<sup>a</sup> (%) </th><th>Outies/innies<sup>b</sup> (%) </th><th>Wrong orientation<sup>c</sup> (%) </th><th>Incorrect insert size (%) </th></tr></thead><tbody><tr><td>Unamplified </td><td>1 </td><td>Short </td><td>0.2 </td><td>0.5 </td><td>0.5 </td><td>0.2 </td><td>0.2 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>0.2 </td><td>0.6 </td><td>0.3 </td><td>0.1 </td><td>0.3 </td></tr><tr><td>Phi </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.15 </td><td>0.1 </td><td>0.55 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1.5 </td><td>0.4 </td><td>0.3 </td><td>4.1 </td><td>0.3 </td></tr><tr><td>Tre </td><td>1 </td><td>Short </td><td>0.6 </td><td>0.2 </td><td>0.1 </td><td>0.8 </td><td>0.0 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>1 </td><td>0.4 </td><td>0.3 </td><td>2.5 </td><td>0.1 </td></tr><tr><td>Rap </td><td>1 </td><td>Short </td><td>9.1 </td><td>4.2 </td><td>0.3 </td><td>3.7 </td><td>0.3 </td></tr><tr><td> </td><td>2 </td><td>Short </td><td>8.8 </td><td>0.5 </td><td>1.7 </td><td>3.1 </td><td>8.4 </td></tr><tr><td>Unamplified </td><td>1 </td><td>Long </td><td>0.4 </td><td>1 </td><td>0.5 </td><td>0.3 </td><td>0.1 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>1 </td><td>3.9 </td><td>1.4 </td><td>0.8 </td><td>0.2 </td></tr><tr><td>Phi </td><td>1 </td><td>Long </td><td>0.3 </td><td>2 </td><td>1.7 </td><td>13.3 </td><td>0.7 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>5.1 </td><td>3.1 </td><td>10.0 </td><td>0.5 </td></tr><tr><td>Tre </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.9 </td><td>1.0 </td><td>12.9 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.4 </td><td>8.1 </td><td>4.8 </td><td>12.4 </td><td>0.6 </td></tr><tr><td>Rap </td><td>1 </td><td>Long </td><td>0.4 </td><td>1.3 </td><td>0.5 </td><td>4.5 </td><td>0.6 </td></tr><tr><td> </td><td>2 </td><td>Long </td><td>0.3 </td><td>3 </td><td>1.3 </td><td>2.2 </td><td>0.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn4"><div class="fn"><p>All percentages are relative to total number of reads in each replicate shown in Table <a reveal-id="DST054TB1" data-open="DST054TB1" class="link link-reveal link-table xref-fig">1</a>.</p></div></div><div content-id="tbnfn5"><div class="fn"><p><sup>a</sup>Reads with mates mapped to different chromosomes.</p></div></div><div content-id="tbnfn6"><div class="fn"><p><sup>b</sup>Reads with mates mapped to the same chromosome that show incorrect orientation of facing either outwards (‘←→’; outies for short-insert libraries) or inwards (‘→←’ innies for long-insert libraries).</p></div></div><div content-id="tbnfn7"><div class="fn"><p><sup>c</sup>Reads with mates mapped to the same chromosome but shows the same orientation, i.e. ‘←←’ or ‘→→’. In the case of long-insert libraries, chimera formation is one of the causes of the formation of these reads.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805481" target="_blank">View Large</a></div></div><p>One of the main concerns in using amplification methods is their tendency to form chimeric DNA fragments, which seems to arise from a mechanism involving priming from displaced 3′ termini.<sup><a reveal-id="DST054C17" data-open="DST054C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Based on this mechanism, the majority of chimeric DNA fragments will be sequences where a segment, which is partially deleted, illustrated by Segments <em>a</em> and <em>b</em> in Fig. <a reveal-id="DST054F1" data-open="DST054F1" class="link link-reveal link-table xref-fig">1</a>A, joins to another Sequence <em>c</em> from the same chromosome which is inverted. Thus we could measure the tendency of amplified libraries forming chimeras by counting the number of read pairs that were mapped with both mates in the wrong orientation (i.e. forward–forward or reverse–reverse). All amplified libraries show an increased proportion of reads in the wrong orientation when compared with the unamplified counterparts (Table <a reveal-id="DST054TB2" data-open="DST054TB2" class="link link-reveal link-table xref-fig">2</a>). This pattern is more prevalent in long-insert reads, where long-insert libraries prepared using Phi and Tre show 10–12.9% of reads with the wrong orientation compared with 0.55–4.1% in short-insert libraries. The majority of these reads are evenly distributed across the chromosomes in all amplified samples (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>), suggesting that wrongly amplified fragments occur infrequently and randomly. As mentioned before, another characteristic of the chimeras is that part of the sequence will be deleted, and as a result the two segments where they map in the genome will appear to have been brought closer together as a consequence of the deletion (Fig. <a reveal-id="DST054F1" data-open="DST054F1" class="link link-reveal link-table xref-fig">1</a>B). Hence, when insert size is calculated based on the mapping positions, we expect to see a much broader insert size distribution than of non-chimeric reads. Indeed, this is what we observed in all WGA long-insert libraries (Replicate 1 of Phi shown in Fig. <a reveal-id="DST054F1" data-open="DST054F1" class="link link-reveal link-table xref-fig">1</a>C and rest in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>) with distances between mates mapped in wrong orientation sometimes even &gt;10 kb. By further looking at sequence reads that can be uniquely mapped into two different positions on the same chromosome, we found two cases that confirm the presence of chimeras in Phi and Tre amplified fragments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). </p>                        <a id="5805483" scrollto-destination="5805483"></a>
<div id="DST054F1" data-id="DST054F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05401.png?Expires=2147483647&Signature=4DJiwRYPwk9W3McX35oN40lJVfcSSZZdXtIlO3PdIjiNS~KMjjiW3gJ21jkpbpVKc96WeF83mj9gfOclAwzkXNHAMrbLQWD0jW5mdzUuVLBaOLM6YZ~DjXNBcdvbtwKT2q~YFSnwsCokn8klWEicj3gqxyPMI0dHTVCEeRQyXCpKt7Lulwb~~wmIbn8usukWZVRBLcV6ZwYp7NN3TsZAPGPz7MnHSbTTPuEDQnsdZjmWE7vQlJUrkYyzb7OVCx9d773iRegzs~VHEekYR8vMFXmWCKhr5WBhBE0XJTwQwH6aFgIkr~GX3oy5RQdJKHUT508crAgFafPw-8qt80uKxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A, B and D) Types of chimeric rearrangements. Each DNA sequence is represented by two or three adjacent segments. Arrows indicate directions of amplified fragments relative to the DNA sequence. (A) and (B) Segment a is copied, b is deleted and c is copied and reverse complemented. (D) The first part of the sequence is copied twice, with unknown sequence placed between the two copies. (C) Insert size distribution plot of wrong-orientation reads in Phi amplified libraries." path-from-xml="dst05401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805483/dst05401.tiff" path-from-xml="dst05401.tiff" target="_blank">View large</a><a section="5805483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05401.png?Expires=2147483647&Signature=ut4gx80~YzvQZPY1n34khlef3ViT7rRA4Z8Ykck~iSpDKoKsoZwrOyw--YdIRJMZMdOPxPerxbxUrOMlMZxpIcQRZJcCTH-LnRq9XTNLavtnQEF0DtC2CB-BTHErGu3-aWK6-9~l1ugbgW7RYswOSnMTGWHFg6qrwRC6QiVS2DpQWqKKKIhYcASvLEHi7j9OJs16nJ1AClWv0hC~O5ibziQwPpRDoed54u3xnWmc88kzKfDwdEuRZkXfItelOvSIODArItXo1i2OlTQm5t26l8ux6-5JnOJYGWKXtgmoRB7dE1YWLA1YARpApTAf5tJQuZktAz3mHHle8R36rPriVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805483&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A, B and D) Types of chimeric rearrangements. Each DNA sequence is represented by two or three adjacent segments. Arrows indicate directions of amplified fragments relative to the DNA sequence. (A) and (B) Segment a is copied, b is deleted and c is copied and reverse complemented. (D) The first part of the sequence is copied twice, with unknown sequence placed between the two copies. (C) Insert size distribution plot of wrong-orientation reads in Phi amplified libraries.</p></div></div><div content-id="DST054F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05401.png?Expires=2147483647&Signature=4DJiwRYPwk9W3McX35oN40lJVfcSSZZdXtIlO3PdIjiNS~KMjjiW3gJ21jkpbpVKc96WeF83mj9gfOclAwzkXNHAMrbLQWD0jW5mdzUuVLBaOLM6YZ~DjXNBcdvbtwKT2q~YFSnwsCokn8klWEicj3gqxyPMI0dHTVCEeRQyXCpKt7Lulwb~~wmIbn8usukWZVRBLcV6ZwYp7NN3TsZAPGPz7MnHSbTTPuEDQnsdZjmWE7vQlJUrkYyzb7OVCx9d773iRegzs~VHEekYR8vMFXmWCKhr5WBhBE0XJTwQwH6aFgIkr~GX3oy5RQdJKHUT508crAgFafPw-8qt80uKxw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A, B and D) Types of chimeric rearrangements. Each DNA sequence is represented by two or three adjacent segments. Arrows indicate directions of amplified fragments relative to the DNA sequence. (A) and (B) Segment a is copied, b is deleted and c is copied and reverse complemented. (D) The first part of the sequence is copied twice, with unknown sequence placed between the two copies. (C) Insert size distribution plot of wrong-orientation reads in Phi amplified libraries." path-from-xml="dst05401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805483/dst05401.tiff" path-from-xml="dst05401.tiff" target="_blank">View large</a><a section="5805483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05401.png?Expires=2147483647&Signature=ut4gx80~YzvQZPY1n34khlef3ViT7rRA4Z8Ykck~iSpDKoKsoZwrOyw--YdIRJMZMdOPxPerxbxUrOMlMZxpIcQRZJcCTH-LnRq9XTNLavtnQEF0DtC2CB-BTHErGu3-aWK6-9~l1ugbgW7RYswOSnMTGWHFg6qrwRC6QiVS2DpQWqKKKIhYcASvLEHi7j9OJs16nJ1AClWv0hC~O5ibziQwPpRDoed54u3xnWmc88kzKfDwdEuRZkXfItelOvSIODArItXo1i2OlTQm5t26l8ux6-5JnOJYGWKXtgmoRB7dE1YWLA1YARpApTAf5tJQuZktAz3mHHle8R36rPriVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805483&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A, B and D) Types of chimeric rearrangements. Each DNA sequence is represented by two or three adjacent segments. Arrows indicate directions of amplified fragments relative to the DNA sequence. (A) and (B) Segment a is copied, b is deleted and c is copied and reverse complemented. (D) The first part of the sequence is copied twice, with unknown sequence placed between the two copies. (C) Insert size distribution plot of wrong-orientation reads in Phi amplified libraries.</p></div></div><p>To our surprise, the insert size distribution of these chimeric reads all peaked close to 0 bp (Fig. <a reveal-id="DST054F1" data-open="DST054F1" class="link link-reveal link-table xref-fig">1</a>C, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>), suggesting that both mates in read pairs were overlapped or placed very close to each other. This observation cannot be solely explained by the priming mechanism where two different sequences on the chromosome were joined, and can only be explained if the same sequence is present on both ends of the sequenced fragment with one version inverted (Fig. <a reveal-id="DST054F1" data-open="DST054F1" class="link link-reveal link-table xref-fig">1</a>D). To investigate this phenomenon further, we looked at Sanger sequence data that were produced from potato cyst nematode (<em>Globodera pallida)</em> genomic DNA, amplified using Genomiphi (Phi) and cloned into plasmid or fosmid vectors<em>.</em> Various insert sizes, from 2 kb to 40 kb (fosmid) were sequenced from either end with read lengths of 200–600 bp. Indeed, we also found that the majority of mates of wrongly oriented reads overlapped with each other, concurring that the same region was sequenced twice with one version inverted. Interestingly, we found that the fraction of wrongly oriented reads was correlated with fragment size, and in extreme cases 85% of MPs derived from fosmids mapped in the wrong orientation and overlapped each other (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>).</p>                            <h3 scrollto-destination=5805485 id="5805485" class="section-title" data-legacy-id=s3c>3.3. Uniform read coverage across the genome</h3>
<p>One of the most important criteria for accurate variant calling and assemblies from Illumina reads is an even coverage of sequence data genome-wide. We first evaluated the variability in the depth of coverage of short-insert reads<sup><a reveal-id="DST054C32" data-open="DST054C32" class="link link-ref link-reveal xref-bibr">32</a></sup> by plotting the cumulative fraction of normalized depth of correctly paired read coverage that covers a given cumulative fraction of genome (Fig. <a reveal-id="DST054F2" data-open="DST054F2" class="link link-reveal link-table xref-fig">2</a>). Normalization of read coverage depth allows libraries of different coverage depths to be compared with each other. The theoretical line (Fig. <a reveal-id="DST054F2" data-open="DST054F2" class="link link-reveal link-table xref-fig">2</a>) indicates a perfectly uniform distribution of reads where 100% of the genome is covered by reads with a normalized and consistent depth of 1. Figure <a reveal-id="DST054F2" data-open="DST054F2" class="link link-reveal link-table xref-fig">2</a> shows that both replicates of the unamplified short-insert library have the closest fit to the theoretical line, suggesting the most uniform distribution of reads. The remaining samples show some level of deviation, suggesting non-uniform distribution across the genome. Distribution plots of the long-insert libraries show more deviation away from the theoretical distribution than short-insert libraries. This effect is more evident in the lower tail of the distribution, indicating a greater proportion of the genome has lower coverage. By inspecting regions of lower coverage across all libraries, the most evident patterns are regions enriched in G homopolymers tracts and GGC motifs<sup><a reveal-id="DST054C33" data-open="DST054C33" class="link link-ref link-reveal xref-bibr">33</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). </p>                        <a id="5805487" scrollto-destination="5805487"></a>
<div id="DST054F2" data-id="DST054F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05402.png?Expires=2147483647&Signature=KF1-tvTt7jOLml-z5baFyW4HMcQ1q1kENdmf0BON38etHLUNNs70UP2d70j6RdXt6obzAqNDzxc9gzHQBaYLz09P3VzPvG4pha4PLS87AasRqMW1yNaF29371WHB6KW3IzO-lVu-1yOuKJD2L5kSSUYGpL5BGD3E5YP0VL7p-a9pyWi36yw83ENr488LwUPmC71YRrPBdRNccG9woC-NAPfJu3D1igO~pXChDEHiQ9d55WwpDbcP2Tp2ODvl-Rr7n1hxO~2Glyrd3m3-Z1rt2~O6kNV3rD47A0WgBv1mF5~9jc5rMWH37hpZev0ZSe0cW9v6i2N9I9movtlCbsJA4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A plot of genome coverage against normalised average depth. Deviation from the theoretical curve (red) indicates less evenness in coverage depth distribution across the genome. Different protocols are plotted with different colours as listed in the legend, and dashed lines indicate read coverage from Replicate 1 of the long-insert libraries." path-from-xml="dst05402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805487/dst05402.tiff" path-from-xml="dst05402.tiff" target="_blank">View large</a><a section="5805487" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05402.png?Expires=2147483647&Signature=cSbhBMlrH78RRfD654VtyfDUn51fyb34HSZbC3AyiEgu9raIUFpq5Gm8mIsku7N0OdxWwM1bB06EMmxdPA6oIjKW0TbWWdoa2kcQ0d9gXBJW45LCpVChIbjGU0KdajmDFVhUDfEKFESlYipvA-Dso1erNK34Yd36hvZtsOt7~5o3E0E7eowYx~H83F~RbPBvBDHSHFjwvu8qqJVI3w~3xcFGV6RArGtvXvRCeJacVWblGoKc-PnzGFEi7rJTgpPfB5BkOdMqzmhfi0FkYwqj8irQjmTxwdCR5d4qgNXwbWho4VvAK4GsziAG-sL9Mi8M4ldtkonILEXXI7eEF~tCYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805487&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A plot of genome coverage against normalised average depth. Deviation from the theoretical curve (red) indicates less evenness in coverage depth distribution across the genome. Different protocols are plotted with different colours as listed in the legend, and dashed lines indicate read coverage from Replicate 1 of the long-insert libraries.</p></div></div><div content-id="DST054F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05402.png?Expires=2147483647&Signature=KF1-tvTt7jOLml-z5baFyW4HMcQ1q1kENdmf0BON38etHLUNNs70UP2d70j6RdXt6obzAqNDzxc9gzHQBaYLz09P3VzPvG4pha4PLS87AasRqMW1yNaF29371WHB6KW3IzO-lVu-1yOuKJD2L5kSSUYGpL5BGD3E5YP0VL7p-a9pyWi36yw83ENr488LwUPmC71YRrPBdRNccG9woC-NAPfJu3D1igO~pXChDEHiQ9d55WwpDbcP2Tp2ODvl-Rr7n1hxO~2Glyrd3m3-Z1rt2~O6kNV3rD47A0WgBv1mF5~9jc5rMWH37hpZev0ZSe0cW9v6i2N9I9movtlCbsJA4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A plot of genome coverage against normalised average depth. Deviation from the theoretical curve (red) indicates less evenness in coverage depth distribution across the genome. Different protocols are plotted with different colours as listed in the legend, and dashed lines indicate read coverage from Replicate 1 of the long-insert libraries." path-from-xml="dst05402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805487/dst05402.tiff" path-from-xml="dst05402.tiff" target="_blank">View large</a><a section="5805487" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05402.png?Expires=2147483647&Signature=cSbhBMlrH78RRfD654VtyfDUn51fyb34HSZbC3AyiEgu9raIUFpq5Gm8mIsku7N0OdxWwM1bB06EMmxdPA6oIjKW0TbWWdoa2kcQ0d9gXBJW45LCpVChIbjGU0KdajmDFVhUDfEKFESlYipvA-Dso1erNK34Yd36hvZtsOt7~5o3E0E7eowYx~H83F~RbPBvBDHSHFjwvu8qqJVI3w~3xcFGV6RArGtvXvRCeJacVWblGoKc-PnzGFEi7rJTgpPfB5BkOdMqzmhfi0FkYwqj8irQjmTxwdCR5d4qgNXwbWho4VvAK4GsziAG-sL9Mi8M4ldtkonILEXXI7eEF~tCYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805487&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A plot of genome coverage against normalised average depth. Deviation from the theoretical curve (red) indicates less evenness in coverage depth distribution across the genome. Different protocols are plotted with different colours as listed in the legend, and dashed lines indicate read coverage from Replicate 1 of the long-insert libraries.</p></div></div><p>Next, we grouped the Illumina short-insert read coverage at each base into 10 kb non-overlapping windows across the <em>C. elegans</em> genome. Using Chromosome I as an example, plotting the median coverage of bases in each window immediately revealed the difference in coverage between libraries (Fig. <a reveal-id="DST054F3" data-open="DST054F3" class="link link-reveal link-table xref-fig">3</a>). Some differences exist, but two technical replicates in each library show remarkably similar patterns with each other. Libraries with unamplified DNA show more uniform coverage across Chromosome I compared with amplified libraries. Interestingly, we observed that read coverage only seems to be more even towards the middle of Chromosome I in all WGA protocols, but is even throughout in Chromosome X (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). The unevenness in coverage of data from Illumina short-insert libraries at <em>C. elegans</em> autosome arms mirrors the distribution of inverted and tandem repetitive sequences, which also cluster more frequently at the autosome arms<sup><a reveal-id="DST054C34" data-open="DST054C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). To investigate this phenomenon, we re-annotated the tandem and inverted repeat contents of <em>C. elegans</em> genome and compared them with read coverage in 10 kb windows. Strikingly, we found that both tandem and inverted repeat content are significantly correlated with read coverage across all samples including unamplified libraries (<em>P</em> &lt; 0.001, Spearman's test). Read coverage in libraries treated with Phi and Tre decrease with increasing content of tandem and inverted repeats (Fig. <a reveal-id="DST054F4" data-open="DST054F4" class="link link-reveal link-table xref-fig">4</a>A and B). Libraries created with Rap also show an inverse relationship between read coverage and tandem repeat content, but positively correlated with inverted repeat content, which explains contrasting patterns of coverage on autosome arms against Phi and Tre. </p>                        <a id="5805489" scrollto-destination="5805489"></a>
<div id="DST054F3" data-id="DST054F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05403.png?Expires=2147483647&Signature=rDNq8uUJV-w-8nDgxAwGIRydbzB99B5sA8dw-ErylMGjWXol7nKAhs6-iJl2iKToX5k9OpgI7Juc8KzXmYXuko59Ka4Gk9IOgT03FpN9Z1VZqPaJMxZzIeV9zVAMQO0rXNu5WimYzOlmvZuLTBFrrcQrLResjiCqjuFNFzHVxPPxE9vBt484m~SOWFE9JqEDqtbJwOtNlj9dw0jqUbVGxLeUYVNNlTkLQojTTS-XMYstz~j996m1cN3xNBoAAztJaINQrTHhDvfqAgGbEoORMa9-B9jv2uTR0CTjMJDcYbWRMiG0-meZQYfGFHpmNC30lpr98V5ysE7qe4EwklGwlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Normalized coverage of 10 kb windows on Chr 1 of C. elegans. Red and blue colour depicts coverage of Replicates 1 and 2, respectively." path-from-xml="dst05403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805489/dst05403.tiff" path-from-xml="dst05403.tiff" target="_blank">View large</a><a section="5805489" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05403.png?Expires=2147483647&Signature=Qd4soHFm7MA8DyV6j35~7J53VuD7Y-xVYvFjlGCYgnoc9mM9HLi-8Z2OCyE-4~MgvZTsq9jp3tJJAIq7YyiRTY05Aq-c~HujWNTUqgGUbl6XFO0tO4vnldVaQ8D3w2-YM1ohTiQr4VtVBPZKAYm1QdSXuN1JNHVDo793z1DjZXBWm5LoH87fiQ3wWnm0IAH5EC8uW6uKXWWhBSSPW7IbUKzE-K7M77ylhkLZnjeMNCWkoj9PnCOTL6Wm4QCXx2vYfLOmuiNiCrOHzZ6EAnK6niWYNaIjIjDspgcPc4nLWUcsHQ9JfKGo4DrVy3W-hu9BH34FgfhgqkvuVkO7Yi6PNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805489&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Normalized coverage of 10 kb windows on Chr 1 of <em>C. elegans.</em> Red and blue colour depicts coverage of Replicates 1 and 2, respectively.</p></div></div><div content-id="DST054F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05403.png?Expires=2147483647&Signature=rDNq8uUJV-w-8nDgxAwGIRydbzB99B5sA8dw-ErylMGjWXol7nKAhs6-iJl2iKToX5k9OpgI7Juc8KzXmYXuko59Ka4Gk9IOgT03FpN9Z1VZqPaJMxZzIeV9zVAMQO0rXNu5WimYzOlmvZuLTBFrrcQrLResjiCqjuFNFzHVxPPxE9vBt484m~SOWFE9JqEDqtbJwOtNlj9dw0jqUbVGxLeUYVNNlTkLQojTTS-XMYstz~j996m1cN3xNBoAAztJaINQrTHhDvfqAgGbEoORMa9-B9jv2uTR0CTjMJDcYbWRMiG0-meZQYfGFHpmNC30lpr98V5ysE7qe4EwklGwlA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Normalized coverage of 10 kb windows on Chr 1 of C. elegans. Red and blue colour depicts coverage of Replicates 1 and 2, respectively." path-from-xml="dst05403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805489/dst05403.tiff" path-from-xml="dst05403.tiff" target="_blank">View large</a><a section="5805489" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05403.png?Expires=2147483647&Signature=Qd4soHFm7MA8DyV6j35~7J53VuD7Y-xVYvFjlGCYgnoc9mM9HLi-8Z2OCyE-4~MgvZTsq9jp3tJJAIq7YyiRTY05Aq-c~HujWNTUqgGUbl6XFO0tO4vnldVaQ8D3w2-YM1ohTiQr4VtVBPZKAYm1QdSXuN1JNHVDo793z1DjZXBWm5LoH87fiQ3wWnm0IAH5EC8uW6uKXWWhBSSPW7IbUKzE-K7M77ylhkLZnjeMNCWkoj9PnCOTL6Wm4QCXx2vYfLOmuiNiCrOHzZ6EAnK6niWYNaIjIjDspgcPc4nLWUcsHQ9JfKGo4DrVy3W-hu9BH34FgfhgqkvuVkO7Yi6PNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805489&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Normalized coverage of 10 kb windows on Chr 1 of <em>C. elegans.</em> Red and blue colour depicts coverage of Replicates 1 and 2, respectively.</p></div></div>                        <a id="5805490" scrollto-destination="5805490"></a>
<div id="DST054F4" data-id="DST054F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05404.png?Expires=2147483647&Signature=W1w~EhRgyPmlMaRRwaCpP6i-Z-xtBTRniVeda5kNCGgD9syo7P8nZrZ1Bfo-qlGNt6SorqEH-pjZhpLzNKlwfD2LGHa6wj0WG-8PYKTDoEWRDg30Hwv7kDm-scn6Iq1XO8QeNDK-jqnXFhv0guGytFRKDQb5~DGbMrIhmJc5l0Vhmpraquw-i-uqHmxjde8lO~7CnFDpg3Tb4~bhjtS~gPgMTF51US0TR2qYUqowuctUnAlNtUEoaPSONiO9Izh7-7yibJvEMRGwASywM7rSFfNaKbr1-HCHyVNGWmdEfTJlJi6IdGL~oJM1ubmbuyAzboo3VTiam~ZZGDLzUJuqTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatterplots showing relationships between (A) inverted and (B) tandem repeat content and normalized read coverage in 10 kb windows of C. elegans." path-from-xml="dst05404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805490/dst05404.tiff" path-from-xml="dst05404.tiff" target="_blank">View large</a><a section="5805490" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05404.png?Expires=2147483647&Signature=J8Xuyq2gG83zHmexjREyw7A6NqM-uuGAQc8bk8nfxHVxLdU-8qSCeZXD18Rqx~7VsOd-mso-CJ6fHbGPVxaBY29ooSHt6THbaJe7pK2k~4pu-tkIHE~mcS8E~b3ligwUp-JovEDHEUDpPdmaD-vKqpd-iR3JJDOstUW00mnavubp5B4OMIkoUf5AR7k2v8RskqWa51l11wSGMaoXYEePrHuXVMKKkGwBOPEmWtH3ZUK9zKTUpBNWX1PPTvm7Y9dbB1Nrmt4j9ZaM4ErFa64xSIhMaUspSeQ2aVsdW95mrz1S7btfss4FlFZrrJ1lW1Oy-dldg1lqOO9gNSUFULFBag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805490&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatterplots showing relationships between (A) inverted and (B) tandem repeat content and normalized read coverage in 10 kb windows of <em>C. elegans</em>.</p></div></div><div content-id="DST054F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05404.png?Expires=2147483647&Signature=W1w~EhRgyPmlMaRRwaCpP6i-Z-xtBTRniVeda5kNCGgD9syo7P8nZrZ1Bfo-qlGNt6SorqEH-pjZhpLzNKlwfD2LGHa6wj0WG-8PYKTDoEWRDg30Hwv7kDm-scn6Iq1XO8QeNDK-jqnXFhv0guGytFRKDQb5~DGbMrIhmJc5l0Vhmpraquw-i-uqHmxjde8lO~7CnFDpg3Tb4~bhjtS~gPgMTF51US0TR2qYUqowuctUnAlNtUEoaPSONiO9Izh7-7yibJvEMRGwASywM7rSFfNaKbr1-HCHyVNGWmdEfTJlJi6IdGL~oJM1ubmbuyAzboo3VTiam~ZZGDLzUJuqTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatterplots showing relationships between (A) inverted and (B) tandem repeat content and normalized read coverage in 10 kb windows of C. elegans." path-from-xml="dst05404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805490/dst05404.tiff" path-from-xml="dst05404.tiff" target="_blank">View large</a><a section="5805490" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05404.png?Expires=2147483647&Signature=J8Xuyq2gG83zHmexjREyw7A6NqM-uuGAQc8bk8nfxHVxLdU-8qSCeZXD18Rqx~7VsOd-mso-CJ6fHbGPVxaBY29ooSHt6THbaJe7pK2k~4pu-tkIHE~mcS8E~b3ligwUp-JovEDHEUDpPdmaD-vKqpd-iR3JJDOstUW00mnavubp5B4OMIkoUf5AR7k2v8RskqWa51l11wSGMaoXYEePrHuXVMKKkGwBOPEmWtH3ZUK9zKTUpBNWX1PPTvm7Y9dbB1Nrmt4j9ZaM4ErFa64xSIhMaUspSeQ2aVsdW95mrz1S7btfss4FlFZrrJ1lW1Oy-dldg1lqOO9gNSUFULFBag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805490&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatterplots showing relationships between (A) inverted and (B) tandem repeat content and normalized read coverage in 10 kb windows of <em>C. elegans</em>.</p></div></div><p>GC bias of Illumina reads has been widely reported in the literature.<sup><a reveal-id="DST054C32" data-open="DST054C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DST054C35" data-open="DST054C35" class="link link-ref link-reveal xref-bibr">35</a></sup> To analyse the effect of GC composition on uniformity of read coverage we calculated the distribution of GC of mapped reads to the reference, normalized by the average coverage across each set of replicate, and compared them against the theoretical distribution. Short-insert libraries prepared under all protocols showed a good fit to the theoretical distribution (Fig. <a reveal-id="DST054F5" data-open="DST054F5" class="link link-reveal link-table xref-fig">5</a>A); however, we see a strong positive bias towards higher GC in all long-insert libraries (Fig. <a reveal-id="DST054F5" data-open="DST054F5" class="link link-reveal link-table xref-fig">5</a>B). To characterize this observation further, we obtained the normalized coverage difference between short- and long-insert libraries in 10 kb windows and correlated with the GC content of the window. Compared with short-insert libraries, the majority of long-insert libraries show more amplification generated from higher extremes of GC regions and less amplification from lower extremes of GC regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). </p>                        <a id="5805492" scrollto-destination="5805492"></a>
<div id="DST054F5" data-id="DST054F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05405.png?Expires=2147483647&Signature=E5zaFnxM-KYuN5MHkYWMoPnu4H5W4inNK~AP61vd7fI9X-FQ7Kh~3jaP7dX6nG-rubikEn8dEBDo5qQ3vm6Z-quyMcgxdxQXuGN6GCh-5xSYGdLJYZOt6qsZU3wauuT3EbXhnvDHFHC~8FYB9Uk~o7JLrYtoh3R6B5fe5C0qYnPIBuBrb-uddhW5WggSFFxLqAUPzIbl6Bh5sPXuZrQFcjf1L8Byf-9ngdlvADl4P15FdSKFICEGD6RV7wb9rXzecsRBnljT8~zl~To7CwT4Y1kaOjXXvFHvRyopusU3Of4MPiXgBvivC2zSOBmr2dTzxXDE3PTTQIcUdiaqOonJOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GC content in sequenced reads of (A) short- and (B) long-insert libraries." path-from-xml="dst05405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805492/dst05405.tiff" path-from-xml="dst05405.tiff" target="_blank">View large</a><a section="5805492" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05405.png?Expires=2147483647&Signature=OExpE7MS-khBwwPSA9ldUeqoLBuhs59OrfBmjLO-rr4D9XJNAs36k60AJgz0apgYAsVLwTxhfJaAQIHiaQ-YBrIL8GbEGlZ3d0FgWPzgSpE1NRbcG~ufcwcL~Eh7wMx0lkot-ApO~suI36N2ml9GhU4eQaXDHnv8gaSQDgV8DzmyxeH-M4jhKmw4clWXv~7X9vtUyyRXOAEGBZF5D9wmyn3MRLUJXICvrSr7HgOJubXuRpwFP0RJxJDCChDsmLKKdCUmQxK0RQ-XMzxEjgcIzKkJKcXYKFRe5pa9NBBJ-zJeCzVSMipAUHcaSEYeU4~jDoQX~RbGV0UHaJA4x7NVsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805492&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GC content in sequenced reads of (A) short- and (B) long-insert libraries.</p></div></div><div content-id="DST054F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/m_dst05405.png?Expires=2147483647&Signature=E5zaFnxM-KYuN5MHkYWMoPnu4H5W4inNK~AP61vd7fI9X-FQ7Kh~3jaP7dX6nG-rubikEn8dEBDo5qQ3vm6Z-quyMcgxdxQXuGN6GCh-5xSYGdLJYZOt6qsZU3wauuT3EbXhnvDHFHC~8FYB9Uk~o7JLrYtoh3R6B5fe5C0qYnPIBuBrb-uddhW5WggSFFxLqAUPzIbl6Bh5sPXuZrQFcjf1L8Byf-9ngdlvADl4P15FdSKFICEGD6RV7wb9rXzecsRBnljT8~zl~To7CwT4Y1kaOjXXvFHvRyopusU3Of4MPiXgBvivC2zSOBmr2dTzxXDE3PTTQIcUdiaqOonJOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GC content in sequenced reads of (A) short- and (B) long-insert libraries." path-from-xml="dst05405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5805492/dst05405.tiff" path-from-xml="dst05405.tiff" target="_blank">View large</a><a section="5805492" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst05405.png?Expires=2147483647&Signature=OExpE7MS-khBwwPSA9ldUeqoLBuhs59OrfBmjLO-rr4D9XJNAs36k60AJgz0apgYAsVLwTxhfJaAQIHiaQ-YBrIL8GbEGlZ3d0FgWPzgSpE1NRbcG~ufcwcL~Eh7wMx0lkot-ApO~suI36N2ml9GhU4eQaXDHnv8gaSQDgV8DzmyxeH-M4jhKmw4clWXv~7X9vtUyyRXOAEGBZF5D9wmyn3MRLUJXICvrSr7HgOJubXuRpwFP0RJxJDCChDsmLKKdCUmQxK0RQ-XMzxEjgcIzKkJKcXYKFRe5pa9NBBJ-zJeCzVSMipAUHcaSEYeU4~jDoQX~RbGV0UHaJA4x7NVsQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5805492&ar=393500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GC content in sequenced reads of (A) short- and (B) long-insert libraries.</p></div></div>                            <h3 scrollto-destination=5805493 id="5805493" class="section-title" data-legacy-id=s3d>3.4. Application 1: variant callings in WGA libraries</h3>
<p>To investigate the effect on variant calling caused by the decreased uniformity in amplified libraries, SNPs were called using Varscan2<sup><a reveal-id="DST054C22" data-open="DST054C22" class="link link-ref link-reveal xref-bibr">22</a></sup> on each of the short-insert replicates. Since the same starting material was used for all samples, and considering the technical variations between different sequencing runs, we first inferred 643 homozygous and 2117 heterozygous SNPs that were called in both non-amplified replicates. Only one replicate of the Rap amplification was analysed, which therefore showed the least robust SNP calling accuracy. For samples amplified with either Phi or Tre, we found ∼80% homozygous SNP calls were also called in at least one replicate (Table <a reveal-id="DST054TB3" data-open="DST054TB3" class="link link-reveal link-table xref-fig">3</a>A). The majority of miss calls in libraries prepared using these two protocols from sites with a coverage depth ≤2, consistent with the expectation that some regions of the genome failed to amplify during the process. The effect of non-uniform read coverage of amplified libraries is further reflected in their heterozygous calls, where only 61–65% of SNPs from the non-amplified library were also called in at least one of the replicates from the two amplification protocols. An exhaustive investigation of the miss-called SNPs revealed that most of the missed heterozygotes were incorrectly called as homozygous (Table <a reveal-id="DST054TB3" data-open="DST054TB3" class="link link-reveal link-table xref-fig">3</a>B). </p>                        <a id="5805495" scrollto-destination="5805495"></a>
<div content-id="DST054TB3" class="table-modal"><div class="table-wrap"><div id="DST054TB3" class="table-wrap-title" data-id="DST054TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of variant calls</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Protocols </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td colspan="4">(A) </td></tr><tr><td colspan="4"> Homozygous SNPs (643) </td></tr><tr><td>  No/low coverage </td><td>105 </td><td>91 </td><td>192 </td></tr><tr><td>  Not called the same </td><td>28 </td><td>30 </td><td>28 </td></tr><tr><td>  Also called in one replicate </td><td>150 </td><td>154 </td><td>423 </td></tr><tr><td>  Called in both replicates </td><td>360 </td><td>368 </td><td>NA </td></tr><tr><td colspan="4"> Heterozygous SNPs (2117) </td></tr><tr><td>  No/low coverage </td><td>132 </td><td>85 </td><td>291 </td></tr><tr><td>  Not called the same </td><td>692 </td><td>650 </td><td>832 </td></tr><tr><td>  Also called in one replicate </td><td>705 </td><td>813 </td><td>994 </td></tr><tr><td>  Called in both replicates </td><td>588 </td><td>569 </td><td>NA </td></tr><tr><td>(B) </td><td> </td><td> </td><td> </td></tr><tr><td colspan="4"> Homozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>37 </td><td>36 </td><td>107 </td></tr><tr><td>  Called in one replicate </td><td>14 </td><td>28 </td><td>134 </td></tr><tr><td colspan="4"> Heterozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>105 </td><td>158 </td><td>528 </td></tr><tr><td>  Called in one replicate </td><td>44 </td><td>46 </td><td>1465 </td></tr></tbody></table></div><div content-id="DST054TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Protocols </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td colspan="4">(A) </td></tr><tr><td colspan="4"> Homozygous SNPs (643) </td></tr><tr><td>  No/low coverage </td><td>105 </td><td>91 </td><td>192 </td></tr><tr><td>  Not called the same </td><td>28 </td><td>30 </td><td>28 </td></tr><tr><td>  Also called in one replicate </td><td>150 </td><td>154 </td><td>423 </td></tr><tr><td>  Called in both replicates </td><td>360 </td><td>368 </td><td>NA </td></tr><tr><td colspan="4"> Heterozygous SNPs (2117) </td></tr><tr><td>  No/low coverage </td><td>132 </td><td>85 </td><td>291 </td></tr><tr><td>  Not called the same </td><td>692 </td><td>650 </td><td>832 </td></tr><tr><td>  Also called in one replicate </td><td>705 </td><td>813 </td><td>994 </td></tr><tr><td>  Called in both replicates </td><td>588 </td><td>569 </td><td>NA </td></tr><tr><td>(B) </td><td> </td><td> </td><td> </td></tr><tr><td colspan="4"> Homozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>37 </td><td>36 </td><td>107 </td></tr><tr><td>  Called in one replicate </td><td>14 </td><td>28 </td><td>134 </td></tr><tr><td colspan="4"> Heterozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>105 </td><td>158 </td><td>528 </td></tr><tr><td>  Called in one replicate </td><td>44 </td><td>46 </td><td>1465 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn8"><div class="fn"><p>(A) Fate of 643 homozygous and 2117 heterozygous SNP calls from both unamplified replicates; (B) fate of additional homozygous and heterozygous SNP calls from amplified replicates.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805495" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST054TB3" class="table-wrap-title" data-id="DST054TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of variant calls</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Protocols </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td colspan="4">(A) </td></tr><tr><td colspan="4"> Homozygous SNPs (643) </td></tr><tr><td>  No/low coverage </td><td>105 </td><td>91 </td><td>192 </td></tr><tr><td>  Not called the same </td><td>28 </td><td>30 </td><td>28 </td></tr><tr><td>  Also called in one replicate </td><td>150 </td><td>154 </td><td>423 </td></tr><tr><td>  Called in both replicates </td><td>360 </td><td>368 </td><td>NA </td></tr><tr><td colspan="4"> Heterozygous SNPs (2117) </td></tr><tr><td>  No/low coverage </td><td>132 </td><td>85 </td><td>291 </td></tr><tr><td>  Not called the same </td><td>692 </td><td>650 </td><td>832 </td></tr><tr><td>  Also called in one replicate </td><td>705 </td><td>813 </td><td>994 </td></tr><tr><td>  Called in both replicates </td><td>588 </td><td>569 </td><td>NA </td></tr><tr><td>(B) </td><td> </td><td> </td><td> </td></tr><tr><td colspan="4"> Homozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>37 </td><td>36 </td><td>107 </td></tr><tr><td>  Called in one replicate </td><td>14 </td><td>28 </td><td>134 </td></tr><tr><td colspan="4"> Heterozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>105 </td><td>158 </td><td>528 </td></tr><tr><td>  Called in one replicate </td><td>44 </td><td>46 </td><td>1465 </td></tr></tbody></table></div><div content-id="DST054TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Protocols </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td colspan="4">(A) </td></tr><tr><td colspan="4"> Homozygous SNPs (643) </td></tr><tr><td>  No/low coverage </td><td>105 </td><td>91 </td><td>192 </td></tr><tr><td>  Not called the same </td><td>28 </td><td>30 </td><td>28 </td></tr><tr><td>  Also called in one replicate </td><td>150 </td><td>154 </td><td>423 </td></tr><tr><td>  Called in both replicates </td><td>360 </td><td>368 </td><td>NA </td></tr><tr><td colspan="4"> Heterozygous SNPs (2117) </td></tr><tr><td>  No/low coverage </td><td>132 </td><td>85 </td><td>291 </td></tr><tr><td>  Not called the same </td><td>692 </td><td>650 </td><td>832 </td></tr><tr><td>  Also called in one replicate </td><td>705 </td><td>813 </td><td>994 </td></tr><tr><td>  Called in both replicates </td><td>588 </td><td>569 </td><td>NA </td></tr><tr><td>(B) </td><td> </td><td> </td><td> </td></tr><tr><td colspan="4"> Homozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>37 </td><td>36 </td><td>107 </td></tr><tr><td>  Called in one replicate </td><td>14 </td><td>28 </td><td>134 </td></tr><tr><td colspan="4"> Heterozygous SNPs </td></tr><tr><td>  Called differently in both unamplified replicates </td><td>105 </td><td>158 </td><td>528 </td></tr><tr><td>  Called in one replicate </td><td>44 </td><td>46 </td><td>1465 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn8"><div class="fn"><p>(A) Fate of 643 homozygous and 2117 heterozygous SNP calls from both unamplified replicates; (B) fate of additional homozygous and heterozygous SNP calls from amplified replicates.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805495" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5805496 id="5805496" class="section-title" data-legacy-id=s3e>3.5. Application 2: assembly generated by WGA prepared libraries</h3>
<p>In order to evaluate the effect that different WGA protocols may have in generating a genome assembly, we used the SGA assembler<sup><a reveal-id="DST054C28" data-open="DST054C28" class="link link-ref link-reveal xref-bibr">28</a></sup> to assemble reads from each of the libraries. These assemblies were then assessed using GAGE<sup><a reveal-id="DST054C30" data-open="DST054C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and summarized in Table <a reveal-id="DST054TB4" data-open="DST054TB4" class="link link-reveal link-table xref-fig">4</a><strong>.</strong> We only presented Replicate 1 for this analysis because of the much lower coverage in Replicate 2 observed in all libraries, with the smaller numbers of reads producing significantly worse assemblies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Having a genome reference allowed us to compute corrected N50 where contigs were broken at miss-assembled regions prior to the calculation of N50. The first observation is that the Rap amplified libraries produce the worst assembly, and as expected the most contiguous and accurate assembly is produced from unamplified library. The second observation is that the assemblies produced from Phi and Tre amplified libraries are almost identical to unamplified libraries, with only an additional 0–4.3% of the assembly missing compared with the reference and similar corrected N50 values. The variation in assembly quality between replicates was greater than between non-amplified and amplified libraries prepared by following either Phi or Tre protocols (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). </p>                        <a id="5805498" scrollto-destination="5805498"></a>
<div content-id="DST054TB4" class="table-modal"><div class="table-wrap"><div id="DST054TB4" class="table-wrap-title" data-id="DST054TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary statistics of assembly and scaffolding data from different libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Protocol </th><th colspan="4">Contig assembly<hr /> </th><th colspan="4">Scaffolding<hr /> </th></tr><tr><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td>Assembly size (bp) </td><td>94 641 187 </td><td>94 028 877 </td><td>94 541 978 </td><td>88 411 985 </td><td>96 571 590 </td><td>96 849 620 </td><td>96 382 191 </td><td>96 869 541 </td></tr><tr><td>contig number </td><td>13 386 </td><td>14 661 </td><td>21 721 </td><td>34 073 </td><td>9415 </td><td>8744 </td><td>9068 </td><td>9247 </td></tr><tr><td>contig average (kb) </td><td>7.1 </td><td>6.4 </td><td>4.4 </td><td>2.6 </td><td>10.3 </td><td>11.1 </td><td>10.6 </td><td>10.5 </td></tr><tr><td>largest contig (kb) </td><td>167.7 </td><td>147.8 </td><td>116.8 </td><td>41.8 </td><td>167.7 </td><td>187.5 </td><td>167.7 </td><td>167.7 </td></tr><tr><td>N50 (kb) </td><td>16.6 </td><td>15.7 </td><td>10.8 </td><td>4.2 </td><td>17.6 </td><td>24.1 </td><td>20.7 </td><td>18.4 </td></tr><tr><td>N50 (number) </td><td>1525 </td><td>1597 </td><td>2109 </td><td>5897 </td><td>1533 </td><td>1067 </td><td>1258 </td><td>1482 </td></tr><tr><td colspan="9">GAGE assessment </td></tr><tr><td>Corrected N50 (kb) </td><td>15.1 </td><td>14.1 </td><td>9.5 </td><td>3.5 </td><td>16.8 </td><td>22.7 </td><td>19.7 </td><td>17.7 </td></tr><tr><td>Corrected N50 (number) </td><td>1721 </td><td>1825 </td><td>2431 </td><td>7493 </td><td>1642 </td><td>1141 </td><td>1354 </td><td>1577 </td></tr><tr><td>Missing reference (%) </td><td>0.09 </td><td>0.09 </td><td>4.43 </td><td>0.14 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>Inversion </td><td>13 </td><td>21 </td><td>38 </td><td>50 </td><td>12 (−1) </td><td>17 (+4) </td><td>13 </td><td>15 (+2) </td></tr><tr><td>Relocation </td><td>7 </td><td>7 </td><td>11 </td><td>22 </td><td>17 (+10) </td><td>13 (+6) </td><td>19 (+12) </td><td>11 (+4) </td></tr><tr><td>Translocation </td><td>12 </td><td>16 </td><td>37 </td><td>30 </td><td>12 </td><td>12 </td><td>12 </td><td>11 (−1) </td></tr></tbody></table></div><div content-id="DST054TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Protocol </th><th colspan="4">Contig assembly<hr /> </th><th colspan="4">Scaffolding<hr /> </th></tr><tr><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td>Assembly size (bp) </td><td>94 641 187 </td><td>94 028 877 </td><td>94 541 978 </td><td>88 411 985 </td><td>96 571 590 </td><td>96 849 620 </td><td>96 382 191 </td><td>96 869 541 </td></tr><tr><td>contig number </td><td>13 386 </td><td>14 661 </td><td>21 721 </td><td>34 073 </td><td>9415 </td><td>8744 </td><td>9068 </td><td>9247 </td></tr><tr><td>contig average (kb) </td><td>7.1 </td><td>6.4 </td><td>4.4 </td><td>2.6 </td><td>10.3 </td><td>11.1 </td><td>10.6 </td><td>10.5 </td></tr><tr><td>largest contig (kb) </td><td>167.7 </td><td>147.8 </td><td>116.8 </td><td>41.8 </td><td>167.7 </td><td>187.5 </td><td>167.7 </td><td>167.7 </td></tr><tr><td>N50 (kb) </td><td>16.6 </td><td>15.7 </td><td>10.8 </td><td>4.2 </td><td>17.6 </td><td>24.1 </td><td>20.7 </td><td>18.4 </td></tr><tr><td>N50 (number) </td><td>1525 </td><td>1597 </td><td>2109 </td><td>5897 </td><td>1533 </td><td>1067 </td><td>1258 </td><td>1482 </td></tr><tr><td colspan="9">GAGE assessment </td></tr><tr><td>Corrected N50 (kb) </td><td>15.1 </td><td>14.1 </td><td>9.5 </td><td>3.5 </td><td>16.8 </td><td>22.7 </td><td>19.7 </td><td>17.7 </td></tr><tr><td>Corrected N50 (number) </td><td>1721 </td><td>1825 </td><td>2431 </td><td>7493 </td><td>1642 </td><td>1141 </td><td>1354 </td><td>1577 </td></tr><tr><td>Missing reference (%) </td><td>0.09 </td><td>0.09 </td><td>4.43 </td><td>0.14 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>Inversion </td><td>13 </td><td>21 </td><td>38 </td><td>50 </td><td>12 (−1) </td><td>17 (+4) </td><td>13 </td><td>15 (+2) </td></tr><tr><td>Relocation </td><td>7 </td><td>7 </td><td>11 </td><td>22 </td><td>17 (+10) </td><td>13 (+6) </td><td>19 (+12) </td><td>11 (+4) </td></tr><tr><td>Translocation </td><td>12 </td><td>16 </td><td>37 </td><td>30 </td><td>12 </td><td>12 </td><td>12 </td><td>11 (−1) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805498" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST054TB4" class="table-wrap-title" data-id="DST054TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Summary statistics of assembly and scaffolding data from different libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Protocol </th><th colspan="4">Contig assembly<hr /> </th><th colspan="4">Scaffolding<hr /> </th></tr><tr><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td>Assembly size (bp) </td><td>94 641 187 </td><td>94 028 877 </td><td>94 541 978 </td><td>88 411 985 </td><td>96 571 590 </td><td>96 849 620 </td><td>96 382 191 </td><td>96 869 541 </td></tr><tr><td>contig number </td><td>13 386 </td><td>14 661 </td><td>21 721 </td><td>34 073 </td><td>9415 </td><td>8744 </td><td>9068 </td><td>9247 </td></tr><tr><td>contig average (kb) </td><td>7.1 </td><td>6.4 </td><td>4.4 </td><td>2.6 </td><td>10.3 </td><td>11.1 </td><td>10.6 </td><td>10.5 </td></tr><tr><td>largest contig (kb) </td><td>167.7 </td><td>147.8 </td><td>116.8 </td><td>41.8 </td><td>167.7 </td><td>187.5 </td><td>167.7 </td><td>167.7 </td></tr><tr><td>N50 (kb) </td><td>16.6 </td><td>15.7 </td><td>10.8 </td><td>4.2 </td><td>17.6 </td><td>24.1 </td><td>20.7 </td><td>18.4 </td></tr><tr><td>N50 (number) </td><td>1525 </td><td>1597 </td><td>2109 </td><td>5897 </td><td>1533 </td><td>1067 </td><td>1258 </td><td>1482 </td></tr><tr><td colspan="9">GAGE assessment </td></tr><tr><td>Corrected N50 (kb) </td><td>15.1 </td><td>14.1 </td><td>9.5 </td><td>3.5 </td><td>16.8 </td><td>22.7 </td><td>19.7 </td><td>17.7 </td></tr><tr><td>Corrected N50 (number) </td><td>1721 </td><td>1825 </td><td>2431 </td><td>7493 </td><td>1642 </td><td>1141 </td><td>1354 </td><td>1577 </td></tr><tr><td>Missing reference (%) </td><td>0.09 </td><td>0.09 </td><td>4.43 </td><td>0.14 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>Inversion </td><td>13 </td><td>21 </td><td>38 </td><td>50 </td><td>12 (−1) </td><td>17 (+4) </td><td>13 </td><td>15 (+2) </td></tr><tr><td>Relocation </td><td>7 </td><td>7 </td><td>11 </td><td>22 </td><td>17 (+10) </td><td>13 (+6) </td><td>19 (+12) </td><td>11 (+4) </td></tr><tr><td>Translocation </td><td>12 </td><td>16 </td><td>37 </td><td>30 </td><td>12 </td><td>12 </td><td>12 </td><td>11 (−1) </td></tr></tbody></table></div><div content-id="DST054TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Protocol </th><th colspan="4">Contig assembly<hr /> </th><th colspan="4">Scaffolding<hr /> </th></tr><tr><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th><th>Unamplified </th><th>Phi </th><th>Tre </th><th>Rap </th></tr></thead><tbody><tr><td>Assembly size (bp) </td><td>94 641 187 </td><td>94 028 877 </td><td>94 541 978 </td><td>88 411 985 </td><td>96 571 590 </td><td>96 849 620 </td><td>96 382 191 </td><td>96 869 541 </td></tr><tr><td>contig number </td><td>13 386 </td><td>14 661 </td><td>21 721 </td><td>34 073 </td><td>9415 </td><td>8744 </td><td>9068 </td><td>9247 </td></tr><tr><td>contig average (kb) </td><td>7.1 </td><td>6.4 </td><td>4.4 </td><td>2.6 </td><td>10.3 </td><td>11.1 </td><td>10.6 </td><td>10.5 </td></tr><tr><td>largest contig (kb) </td><td>167.7 </td><td>147.8 </td><td>116.8 </td><td>41.8 </td><td>167.7 </td><td>187.5 </td><td>167.7 </td><td>167.7 </td></tr><tr><td>N50 (kb) </td><td>16.6 </td><td>15.7 </td><td>10.8 </td><td>4.2 </td><td>17.6 </td><td>24.1 </td><td>20.7 </td><td>18.4 </td></tr><tr><td>N50 (number) </td><td>1525 </td><td>1597 </td><td>2109 </td><td>5897 </td><td>1533 </td><td>1067 </td><td>1258 </td><td>1482 </td></tr><tr><td colspan="9">GAGE assessment </td></tr><tr><td>Corrected N50 (kb) </td><td>15.1 </td><td>14.1 </td><td>9.5 </td><td>3.5 </td><td>16.8 </td><td>22.7 </td><td>19.7 </td><td>17.7 </td></tr><tr><td>Corrected N50 (number) </td><td>1721 </td><td>1825 </td><td>2431 </td><td>7493 </td><td>1642 </td><td>1141 </td><td>1354 </td><td>1577 </td></tr><tr><td>Missing reference (%) </td><td>0.09 </td><td>0.09 </td><td>4.43 </td><td>0.14 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>Inversion </td><td>13 </td><td>21 </td><td>38 </td><td>50 </td><td>12 (−1) </td><td>17 (+4) </td><td>13 </td><td>15 (+2) </td></tr><tr><td>Relocation </td><td>7 </td><td>7 </td><td>11 </td><td>22 </td><td>17 (+10) </td><td>13 (+6) </td><td>19 (+12) </td><td>11 (+4) </td></tr><tr><td>Translocation </td><td>12 </td><td>16 </td><td>37 </td><td>30 </td><td>12 </td><td>12 </td><td>12 </td><td>11 (−1) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5805498" target="_blank">View Large</a></div></div><p>To assess the effect of the uneven sequencing coverage and the presence of erroneous orientation in the long-insert libraries, we ran the SSPACE<sup><a reveal-id="DST054C29" data-open="DST054C29" class="link link-ref link-reveal xref-bibr">29</a></sup> scaffolder on the best assembly (non-amplified Replicate 1) using reads from each long-insert library. We only used Replicate 1 for this analysis because of the much lower coverage in Replicate 2 observed in all samples. A total of 10 additional miss-assemblies were identified by GAGE after scaffolding with the unamplified long-insert library (Table <a reveal-id="DST054TB4" data-open="DST054TB4" class="link link-reveal link-table xref-fig">4</a>) and, comparably, the Phi amplified library created the same number of miss-assemblies but actually had the largest corrected N50, as well as containing the largest scaffold of all the assemblies. The assembly scaffolded with the Rap amplified library, which contains the fewest wrongly oriented reads in among all amplified libraries but has the most PCR duplicates, shows the fewest additional miss-assemblies and with both the N50 and longest scaffold length very similar to the unamplified sample.</p>                            <h2 scrollto-destination=5805500 id="5805500" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5805501 id="5805501" class="section-title" data-legacy-id=s4a>4.1. Bias due to WGA protocols and genome features</h3>
<p>There are two main types of biases present in Illumina read sequenced from amplified DNA: those that arise from the Illumina library construction and sequencing process, and those arising from the nature of DNA fragments generated from WGA. Reads of different insert size also exert influences on the extent of these biases. First, inverted repeats are known to generate chimeras in WGA protocols,<sup><a reveal-id="DST054C17" data-open="DST054C17" class="link link-ref link-reveal xref-bibr">17</a></sup> the extent of which we assessed by recording the number of wrongly oriented reads in the samples. In fact, the distribution of wrongly oriented reads reveals that previously proposed mechanism<sup><a reveal-id="DST054C17" data-open="DST054C17" class="link link-ref link-reveal xref-bibr">17</a></sup> only constitute a part of these, and instead the majority of these reads arise from fragments containing the same sequence on both ends with one end being an inverted version of another. The same trend was found in the <em>G. pallida</em> capillary reads, and a positive correlation was observed between numbers of chimeras and length of sequenced fragments. An explanation could be that the duplicated sequences are more likely to be present in larger fragments. Hence selection of such fragments in larger insert libraries after shearing will also result in preferentially selection for artefacts. In shorter insert Illumina libraries, the wrongly amplified sequences were broken down into smaller fragments and hence a decreased number of wrongly orientated reads was found. It would be interesting to sequence the whole wrongly amplified fragment to reveal the dominant nature of chimera formation in WGA.</p><p>Secondly, we obtained the quality statistics of read libraries such as read coverage aligned across the genome, proportions of wrongly oriented mapped reads and searched for potential biases due to GC content, repeat composition of the <em>C. elegans</em> genome and the different insert size libraries. Coverage biases in regions of genomes as a result of amplification have been previously observed in bacteria<sup><a reveal-id="DST054C16" data-open="DST054C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and in humans,<sup><a reveal-id="DST054C15" data-open="DST054C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and we found that the uneven coverage between chromosome arms and centres are correlated with tandem and inverted repeat content, both of which have been previously described to bias the coverage of WGA reads.<sup><a reveal-id="DST054C16" data-open="DST054C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST054C17" data-open="DST054C17" class="link link-ref link-reveal xref-bibr">17</a></sup> We show that in repeat regions Phi and Tre displayed lower read coverage, while Rap showed different biases depending on the repeat type, suggesting alternative mechanisms affecting the amplification process. It also may not be the repeat sequence <em>per se</em>, as repetitive regions in the genome are also associated with the global structural features such as histone modifications. For example, chromosomal arms of <em>C. elegans</em> show enrichment of H3K9 methylation,<sup><a reveal-id="DST054C36" data-open="DST054C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which are also zones of elevated meiotic recombination.<sup><a reveal-id="DST054C34" data-open="DST054C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p><p>Thirdly, we found the presence of GC bias in the reads from long-insert libraries including those without WGA, despite the <em>C. elegans</em> genome possessing an almost uniform GC content (36%) across all chromosomes.<sup><a reveal-id="DST054C34" data-open="DST054C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The current proposed mechanisms behind WGA do not explain a bias with GC content, and indeed we found no obvious effect of WGA libraries against GC content. Hence, the overall bias is mainly caused during the library construction process of the long-insert libraries and will be even more prevalent in genomes that possess more extreme GC content, such as <em>Plasmodium falciparum</em> (17% GC).</p><p>Finally, in this study we have analysed a total of three WGA protocols. We found that the addition of trehalose (Tre) does not seem to improve the coverage evenness against the libraries amplified with <em>Genomiphi</em> only (Phi), as all the metrics and biases are almost identical between two protocols. A possible explanation is that <em>Genomiphi</em> might have already contained reagents that work similarly to trehalose. On the other hand, libraries amplified using Rapisome (Rap) showed different patterns to <em>Genomiphi</em> based protocols. Constructing a good Illumina library from Rap amplified products seemed to be more difficult than Genomiphi based protocols, considering the lower number of mapped reads and higher number of PCR duplicates. However, Rap amplified libraries have shown lower number of wrongly orientated reads, suggesting an alternative mechanism that attributes to biases to different genome features. For instance, we show regions associated with inverted repeat actually increase read coverage in Rapisome libraries. It will be of interest to find out whether the lower percentage of wrongly orientated reads in Rapisome amplified libraries are also noticeable in longer insert libraries. If that was indeed the case, then Illumina libraries can be constructed from much larger amplified DNA fragments with still lower proportions of wrongly oriented reads.</p>                            <h3 scrollto-destination=5805506 id="5805506" class="section-title" data-legacy-id=s4b>4.2. Performance of WGA reads</h3>
<p>Sequence reads from amplified fragments can potentially give rise to false SNP calls, particularly in genomic regions that are difficult to amplify. We found that SNPs were under-called from reads generated from amplified DNA compared with unamplified samples. When reads were sequenced at good coverage, we found that ∼80% of homozygous SNP candidates from unamplified samples could also be identified correctly from amplified samples, while the majority of miss calls were due to insufficient read coverage. The limitation seems to be more prevalent in heterozygous SNPs, where only up to 65% of candidates could be accurately identified. Most of these SNPs were called as homozygous in one of the alleles, and this may be due to the fact that most SNP callers make heterozygous calls confidently if allele frequencies were close to 50%, for which non-uniform coverage results in departure from this assumption. Hence, although most of the homozygous SNPs can be accurately identified in WGA samples, it is essential to investigate the allele frequencies of variant sites in order to not miss calls that were otherwise heterozygous in the sample.</p><p>There have been advances in assembly algorithms that assemble genomes with fluctuating coverage in the samples, but we wished to emphasize the effect that reads sequenced from amplified fragments have on a typical assembly process, hence we applied commonly used methods to assemble the <em>C. elegans</em> genome using short-insert reads with the SGA assembler,<sup><a reveal-id="DST054C28" data-open="DST054C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and scaffolded using long-insert reads with the SSPACE scaffolder.<sup><a reveal-id="DST054C29" data-open="DST054C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Contig and scaffold statistics generated from amplified libraries were comparable with those from their unamplified counterparts. In fact, we found that variability between technical replicates influenced assembly metrics more than different WGA protocols. We also found that the wrongly oriented reads did not cause more miss-assemblies than expected. An explanation is that our long-insert libraries were sequenced at adequate coverage and problematic reads were present in minority. Occasions can arise where correct reads suggest joining of two scaffolds but problematic reads suggest otherwise, in which case the SSPACE scaffolder will scaffold correctly because the number of correct reads outweighs problematic reads. However, we expect that reads from larger insert WGA libraries would result in significantly more scaffolding errors because the majority of reads are problematic, with the error rate increasing with fragment length.</p>                            <h3 scrollto-destination=5805509 id="5805509" class="section-title" data-legacy-id=s4c>4.3. Conclusion</h3>
<p>WGA will remain an essential method in genomics for many years to come. The mechanisms of chimera formation and other sources of bias have been investigated and there are many new protocols that are currently being developed to minimize such effects. In the meantime, it is also important to evaluate the nature of biases in order to correctly interpret assembly results. Here, we have presented the effects of WGA on Illumina reads. We recognize the biases present from the reads, and show that the DNA amplified from all protocols investigated in this article is able to call the majority of SNPs and produce accurate assemblies comparable with those produced from unamplified DNA. Reads sequenced from longer fragments are still useful in scaffolding and problematic reads can be detected and filtered out as long as they remain a minority. We envisage also using WGA in other eukaryotic species (including nematodes) that possess similar GC content and base composition to <em>C. elegans</em>. Fundamentally, as biologists turn to sequencing more unexplored species or samples from which only very low amounts of starting material are available, our study provides a first initial assessment on Illumina sequencing from WGA DNA.</p>                            <h2 scrollto-destination=5805511 id="5805511" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>Illumina data are released to the European Nucleotide Archive (<a class="link link-uri" href="http://www.ebi.ac.uk/ena/">http://www.ebi.ac.uk/ena/</a>) under accession number ERP000964.</p>                            <h2 scrollto-destination=5805513 id="5805513" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5805515 id="5805515" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by JSPS KAKENHI (grants numbers 20353659 and 23248024) and the Wellcome Trust (grant no. WT 098051). I.J.T. was supported by JSPS Postdoctoral Fellowship Program for Foreign Researchers.</p>                            <h2 scrollto-destination=5805517 id="5805517" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Anna Protasio for her commenting on the manuscript.</p>                            <h2 scrollto-destination=5805519 id="5805519" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST054C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">Illumina</div> <div class="comment">HiSeq Sequencing Systems <a class="link link-uri" href="http://res.illumina.com/documents/systems/hiseq/datasheet_hiseq_systems.pdf">http://res.illumina.com/documents/systems/hiseq/datasheet_hiseq_systems.pdf</a></div></div></div></div></div><div content-id="DST054C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Nieuwerburgh</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Ledesma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Deforce</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gaasterland</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ordoukhanian</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Head</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> pg. <div class="fpage">e24</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkr1000" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkr1000</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.%20Van%20Nieuwerburgh&amp;author=R.C.%20Thompson&amp;author=J.%20Ledesma&amp;author=D.%20Deforce&amp;author=T.%20Gaasterland&amp;author=P.%20Ordoukhanian&amp;author=S.R.%20Head&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="collab">Illumina</div>, <div class="year">2009</div> <div class="comment">Mate Pair Library V2 Sample Preparation Guide</div></div></div></div></div><div content-id="DST054C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Nath</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Generation of long insert pairs using a Cre-LoxP Inverse PCR approach</div>, <div class="source">PloS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e29437</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0029437" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0029437</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20long%20insert%20pairs%20using%20a%20Cre-LoxP%20Inverse%20PCR%20approach&amp;author=Z.%20Peng&amp;author=Z.%20Zhao&amp;author=N.%20Nath&amp;publication_year=2012&amp;journal=PloS%20One&amp;volume=7&amp;pages=e29437" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grunenwald</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Baas</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Caruccio</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Syed</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Rapid, high-throughput library preparation for next-generation sequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2010</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">iii</div>-<div class="lpage">iv</div>)</div></div></div></div><div content-id="DST054C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seth-Smith</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Harris</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Skilton</div> <div class="given-names">R.J.</div></div>, et al. </div><div class="article-title">Whole-genome sequences of Chlamydia trachomatis directly from clinical samples without culture</div>, <div class="source">Genome Res.</div> , <div class="year">2013</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">855</div>-<div class="lpage">66</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequences%20of%20Chlamydia%20trachomatis%20directly%20from%20clinical%20samples%20without%20culture&amp;author=H.M.%20Seth-Smith&amp;author=S.R.%20Harris&amp;author=R.J.%20Skilton&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=855-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.150037.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.150037.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.150037.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23525359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">C.K.</div></div>,  <div class="name"><div class="surname">Thomas</div> <div class="given-names">C.G.</div></div>,  <div class="name"><div class="surname">Cutter</div> <div class="given-names">A.D.</div></div>. </div><div class="article-title">Molecular hyperdiversity defines populations of the nematode <em>Caenorhabditis brenneri</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2013</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">11056</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20hyperdiversity%20defines%20populations%20of%20the%20nematode%20Caenorhabditis%20brenneri&amp;author=A.%20Dey&amp;author=C.K.%20Chan&amp;author=C.G.%20Thomas&amp;author=A.D.%20Cutter&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=11056-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1303057110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1303057110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1303057110"> </span></div> </div></div></div></div></div><div content-id="DST054C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hawkins</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Detter</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Richardson</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">Whole genome amplification—applications and advances</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2002</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">65</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0958-1669(02)00286-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0958-1669(02)00286-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%E2%80%94applications%20and%20advances&amp;author=T.L.%20Hawkins&amp;author=J.C.%20Detter&amp;author=P.M.%20Richardson&amp;publication_year=2002&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=13&amp;pages=65-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0958-1669(02)00286-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0958-1669(02)00286-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0958-1669(02)00286-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11849960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Telenius</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Carter</div> <div class="given-names">N.P.</div></div>,  <div class="name"><div class="surname">Bebb</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Nordenskjold</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ponder</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Tunnacliffe</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer</div>, <div class="source">Genomics</div> , <div class="year">1992</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">718</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0888-7543(92)90147-K" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0888-7543(92)90147-K</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degenerate%20oligonucleotide-primed%20PCR%3A%20general%20amplification%20of%20target%20DNA%20by%20a%20single%20degenerate%20primer&amp;author=H.%20Telenius&amp;author=N.P.%20Carter&amp;author=C.E.%20Bebb&amp;author=M.%20Nordenskjold&amp;author=B.A.%20Ponder&amp;author=A.%20Tunnacliffe&amp;publication_year=1992&amp;journal=Genomics&amp;volume=13&amp;pages=718-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(92)90147-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(92)90147-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(92)90147-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1639399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Schmitt</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hubert</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Navidi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Arnheim</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Whole genome amplification from a single cell: implications for genetic analysis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">5847</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.89.13.5847" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.89.13.5847</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%20from%20a%20single%20cell%3A%20implications%20for%20genetic%20analysis&amp;author=L.%20Zhang&amp;author=X.%20Cui&amp;author=K.%20Schmitt&amp;author=R.%20Hubert&amp;author=W.%20Navidi&amp;author=N.%20Arnheim&amp;publication_year=1992&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=5847-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.13.5847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.13.5847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.13.5847"> </span></div> </div></div></div></div></div><div content-id="DST054C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Binga</div> <div class="given-names">E.K.</div></div>,  <div class="name"><div class="surname">Lasken</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Neufeld</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Something from (almost) nothing: the impact of multiple displacement amplification on microbial ecology</div>, <div class="source">ISME J.</div> , <div class="year">2008</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">233</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ismej.2008.10" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ismej.2008.10</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Something%20from%20(almost)%20nothing%3A%20the%20impact%20of%20multiple%20displacement%20amplification%20on%20microbial%20ecology&amp;author=E.K.%20Binga&amp;author=R.S.%20Lasken&amp;author=J.D.%20Neufeld&amp;publication_year=2008&amp;journal=ISME%20J.&amp;volume=2&amp;pages=233-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2008.10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2008.10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2008.10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18256705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">H.-J.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Primase-based whole genome amplification</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> pg. <div class="fpage">e79</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn377" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn377</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primase-based%20whole%20genome%20amplification&amp;author=Y.%20Li&amp;author=H.-J.%20Kim&amp;author=C.%20Zheng&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=e79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn377" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn377" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn377"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18559358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lasken</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Genomic sequencing of uncultured microorganisms from single cells</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">631</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrmicro2857" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrmicro2857</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20sequencing%20of%20uncultured%20microorganisms%20from%20single%20cells&amp;author=R.S.%20Lasken&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=10&amp;pages=631-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22890147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dharia</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Bright</div> <div class="given-names">a.T.</div></div>,  <div class="name"><div class="surname">Westenberger</div> <div class="given-names">S.J.</div></div>, et al. </div><div class="article-title">Whole-genome sequencing and microarray analysis of ex vivo Plasmodium vivax reveal selective pressure on putative drug resistance genes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">20045</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1003776107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.1003776107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20and%20microarray%20analysis%20of%20ex%20vivo%20Plasmodium%20vivax%20reveal%20selective%20pressure%20on%20putative%20drug%20resistance%20genes&amp;author=N.V.%20Dharia&amp;author=a.T.%20Bright&amp;author=S.J.%20Westenberger&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=20045-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1003776107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1003776107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1003776107"> </span></div> </div></div></div></div></div><div content-id="DST054C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">H.C.</div></div>,  <div class="name"><div class="surname">Behr</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Quake</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Genome-wide single-cell analysis of recombination activity and de novo mutation rates in human sperm</div>, <div class="source">Cell</div> , <div class="year">2012</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">402</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2012.06.030" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2012.06.030</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20single-cell%20analysis%20of%20recombination%20activity%20and%20de%20novo%20mutation%20rates%20in%20human%20sperm&amp;author=J.%20Wang&amp;author=H.C.%20Fan&amp;author=B.%20Behr&amp;author=S.R.%20Quake&amp;publication_year=2012&amp;journal=Cell&amp;volume=150&amp;pages=402-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.06.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.06.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.06.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22817899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pinard</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">de Winter</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sarkis</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">Assessment of whole genome amplification-induced bias through high-throughput, massively parallel whole genome sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">216</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-7-216" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-7-216</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20whole%20genome%20amplification-induced%20bias%20through%20high-throughput%2C%20massively%20parallel%20whole%20genome%20sequencing&amp;author=R.%20Pinard&amp;author=A.%20de%20Winter&amp;author=G.J.%20Sarkis&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lasken</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Stockwell</div> <div class="given-names">T.B.</div></div>. </div><div class="article-title">Mechanism of chimera formation during the Multiple Displacement Amplification reaction</div>, <div class="source">BMC Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> pg. <div class="fpage">19</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1472-6750-7-19" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1472-6750-7-19</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20chimera%20formation%20during%20the%20Multiple%20Displacement%20Amplification%20reaction&amp;author=R.S.%20Lasken&amp;author=T.B.%20Stockwell&amp;publication_year=2007&amp;journal=BMC%20Biotechnol.&amp;volume=7&amp;pages=19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1472-6750-7-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1472-6750-7-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1472-6750-7-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17430586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Urban</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Palejev</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">A procedure for highly specific, sensitive, and unbiased whole-genome amplification</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">15499</div>-<div class="lpage">504</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0808028105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0808028105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20procedure%20for%20highly%20specific%2C%20sensitive%2C%20and%20unbiased%20whole-genome%20amplification&amp;author=X.%20Pan&amp;author=A.E.%20Urban&amp;author=D.%20Palejev&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=15499-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0808028105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0808028105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0808028105"> </span></div> </div></div></div></div></div><div content-id="DST054C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Park</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Shirley</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Keane</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Swerdlow</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Quail</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">An improved approach to mate-paired library preparation for Illumina sequencing</div>, <div class="source">Methods Next Gener. Seq.</div> , <div class="year">2013</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">10</div>-<div class="lpage">20</div>)</div></div></div></div><div content-id="DST054C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bolger</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Nagel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Fernie</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Lunn</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Stitt</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div>. </div><div class="article-title">RobiNA: a user-friendly, integrated software solution for RNA-Seq-based transcriptomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">W622</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gks540" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gks540</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RobiNA%3A%20a%20user-friendly%2C%20integrated%20software%20solution%20for%20RNA-Seq-based%20transcriptomics&amp;author=M.%20Lohse&amp;author=A.M.%20Bolger&amp;author=A.%20Nagel&amp;author=A.R.%20Fernie&amp;author=J.E.%20Lunn&amp;author=M.%20Stitt&amp;author=B.%20Usadel&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=W622-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22684630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McKenna</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1297</div>-<div class="lpage">303</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.107524.110" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.107524.110</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koboldt</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Larson</div> <div class="given-names">D.E.</div></div>, et al. </div><div class="article-title">VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">568</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.129684.111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.129684.111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VarScan%202%3A%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&amp;author=D.C.%20Koboldt&amp;author=Q.%20Zhang&amp;author=D.E.%20Larson&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=568-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.129684.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.129684.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.129684.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22300766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">573</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/27.2.573" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/27.2.573</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Warburton</div> <div class="given-names">P.E.</div></div>,  <div class="name"><div class="surname">Giordano</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Inverted repeat structure of the human genome: the X-chromosome contains a preponderance of large, highly homologous inverted repeats that contain testes genes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1861</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2542904" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2542904</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inverted%20repeat%20structure%20of%20the%20human%20genome%3A%20the%20X-chromosome%20contains%20a%20preponderance%20of%20large%2C%20highly%20homologous%20inverted%20repeats%20that%20contain%20testes%20genes&amp;author=P.E.%20Warburton&amp;author=J.%20Giordano&amp;author=F.%20Cheung&amp;author=Y.%20Gelfand&amp;author=G.%20Benson&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1861-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2542904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2542904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2542904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinlan</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">I.M.</div></div>. </div><div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>, <div class="source">Bioinformatics (Oxford, England)</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">841</div>-<div class="lpage">2</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btq033" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btq033</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.R.%20Quinlan&amp;author=I.M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics%20(Oxford%2C%20England)&amp;volume=26&amp;pages=841-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Team</div> <div class="given-names">R.C.</div></div>. </div>, <div class="year">2013</div> <div class="comment">R: A Language and Environment for Statistical Computing</div></div></div></div></div><div content-id="DST054C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ning</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Mullikin</div> <div class="given-names">J.C.</div></div>. </div><div class="article-title">SSAHA: a fast search method for large DNA databases</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1725</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.194201" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.194201</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SSAHA%3A%20a%20fast%20search%20method%20for%20large%20DNA%20databases&amp;author=Z.%20Ning&amp;author=A.J.%20Cox&amp;author=J.C.%20Mullikin&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.194201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.194201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.194201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11591649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Simpson</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Efficient de novo assembly of large genomes using compressed data structures</div>, <div class="source">Genome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">549</div>-<div class="lpage">56</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.126953.111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.126953.111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20large%20genomes%20using%20compressed%20data%20structures&amp;author=J.T.%20Simpson&amp;author=R.%20Durbin&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=549-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.126953.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.126953.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.126953.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22156294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div>,  <div class="name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics (Oxford, England)</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">578</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btq683" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btq683</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics%20(Oxford%2C%20England)&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Phillippy</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Zimin</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">GAGE: a critical evaluation of genome assemblies and assembly algorithms</div>, <div class="source">Genome Res.</div> , <div class="year">2012</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">557</div>-<div class="lpage">67</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.131383.111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.131383.111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GAGE%3A%20a%20critical%20evaluation%20of%20genome%20assemblies%20and%20assembly%20algorithms&amp;author=S.L.%20Salzberg&amp;author=A.M.%20Phillippy&amp;author=A.%20Zimin&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=557-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.131383.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.131383.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.131383.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22147368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozarewa</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ning</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Quail</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Sanders</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Berriman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">291</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1311" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1311</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amplification-free%20Illumina%20sequencing-library%20preparation%20facilitates%20improved%20mapping%20and%20assembly%20of%20(G%2BC)-biased%20genomes&amp;author=I.%20Kozarewa&amp;author=Z.%20Ning&amp;author=M.A.%20Quail&amp;author=M.J.%20Sanders&amp;author=M.%20Berriman&amp;author=D.J.%20Turner&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=291-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19287394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oyola</div> <div class="given-names">S.O.</div></div>,  <div class="name"><div class="surname">Otto</div> <div class="given-names">T.D.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">1</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-13-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-13-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20Illumina%20next-generation%20sequencing%20library%20preparation%20for%20extremely%20AT-biased%20genomes&amp;author=S.O.%20Oyola&amp;author=T.D.%20Otto&amp;author=Y.%20Gu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Morimoto</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Sequence-specific error profile of Illumina sequencers</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> pg. <div class="fpage">e90</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkr344" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkr344</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-specific%20error%20profile%20of%20Illumina%20sequencers&amp;author=K.%20Nakamura&amp;author=T.%20Oshima&amp;author=T.%20Morimoto&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21576222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Press</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">York</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nw</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Genome sequence of the nematode <em>C. elegans</em>: a platform for investigating biology</div>, <div class="source">Science</div> , <div class="year">1998</div>, vol. <div class="volume">282</div> (pg. <div class="fpage">2012</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.282.5396.2012" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.282.5396.2012</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20nematode%20C.%20elegans%3A%20a%20platform%20for%20investigating%20biology&amp;author=H.%20Press&amp;author=N.%20York&amp;author=A.%20Nw&amp;publication_year=1998&amp;journal=Science&amp;volume=282&amp;pages=2012-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.282.5396.2012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.282.5396.2012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.282.5396.2012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9851916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Speed</div> <div class="given-names">T.P.</div></div>. </div><div class="article-title">Summarizing and correcting the GC content bias in high-throughput sequencing</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> pg. <div class="fpage">e72</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gks001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gks001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Summarizing%20and%20correcting%20the%20GC%20content%20bias%20in%20high-throughput%20sequencing&amp;author=Y.%20Benjamini&amp;author=T.P.%20Speed&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22323520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST054C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST054C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Rechtsteiner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Egelhofer</div> <div class="given-names">T.A.</div></div>, et al. </div><div class="article-title">Broad chromosomal domains of histone modification patterns in <em>C. elegans</em></div>, <div class="source">Genome Res.</div> , <div class="year">2011</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">227</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.115519.110" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.115519.110</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Broad%20chromosomal%20domains%20of%20histone%20modification%20patterns%20in%20C.%20elegans&amp;author=T.%20Liu&amp;author=A.%20Rechtsteiner&amp;author=T.A.%20Egelhofer&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=227-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.115519.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.115519.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.115519.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177964" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Prof. Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=393500&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst054/2/dst054-dst054supp.docx?Expires=2147483647&amp;Signature=Grm1i7TjTBHwEnBDCrs4Ay9Q2a1YT6QTzD2XXlIP3oQU1WU32nvfaOnbPQiUi9wwcU54QBUesYOOAw6fZ2NEkbNrgtRiq3NZLyeVgeNQT2r24yFcMYGodqdLviYcGWI7FKyy9PodvWe9DRjTGw8kgPGHym75qlC-FkgpaToD2Eaqjzr~k6SbWeVOh2T2NrMZiNoYorX6dTNTsuQ6WZU~onlNVselY1PcksLu6UrtJ2LLmj0BRzKJ2YI6ArtBssGMNblOumisUC64RRp6gO5-9MTPhcyoWSzwPggFJ99xluaTTzG9hF9YMWGyewYN1S6YEcw6Edw8oan-bG6MyBg98g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - docx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">443</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst054"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="393500"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f3%2f243%2f393500">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897900002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F3%2F243%2F393500" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/393500'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Summarizing Specific Profiles in Illumina Sequencing from Whole-Genome Amplified DNA" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst054");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f243%2f393500">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

