

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/6/487/481438","@type":"ScholarlyArticle","name":"Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale  De Novo  Analysis of Leaf Transcripts of  Aegilops tauschii , A Wild Wheat Progenitor","datePublished":"2012-11-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2012-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss028","keywords":["<italic>Aegilops tauschii</italic>","expression sequence tag","next generation sequencing","single nucleotide polymorphism","wheat"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/6/487/481438","author":[{"name":"Iehisa, Julio Cesar Masaru","affiliation":"1  Laboratory of Plant Genetics, Graduate School of Agricultural Science ,  Kobe University ,  Nada-ku, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Shimizu, Akifumi","affiliation":"2  Department of Biological Resources Management, School of Environmental Science ,  University of Shiga Prefecture ,  Shiga, Hikone 522-8533 ,  Japan","@type":"Person"},{"name":"Sato, Kazuhiro","affiliation":"3  Institute of Plant Science and Resources ,  Okayama University ,  Kurashiki 710-0046 ,  Japan","@type":"Person"},{"name":"Nasuda, Shuhei","affiliation":"4  Laboratory of Plant Genetics, Graduate School of Agriculture ,  Kyoto University ,  Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Takumi, Shigeo","affiliation":"1  Laboratory of Plant Genetics, Graduate School of Agricultural Science ,  Kobe University ,  Nada-ku, Kobe 657-8501 ,  Japan","@type":"Person"}],"description":"Abstract.  Construction of high-resolution genetic maps is important for genetic and genomic research, as well as for molecular breeding. Single nucleotide poly","pageStart":"487","pageEnd":"497","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale  De Novo  Analysis of Leaf Transcripts of  Aegilops tauschii , A Wild Wheat Progenitor","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale  De Novo  Analysis of Leaf Transcripts of  Aegilops tauschii , A Wild Wheat Progenitor" />
    <meta property="og:description" content="Abstract.  Construction of high-resolution genetic maps is important for genetic and genomic research, as well as for molecular breeding. Single nucleotide poly" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/6/487/481438" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-11-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iehisa, Julio Cesar Masaru" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agricultural Science, Kobe University, Nada-ku, Kobe 657-8501, Japan" /><meta name="citation_author" content="Shimizu, Akifumi" /><meta name="citation_author_institution" content="Department of Biological Resources Management, School of Environmental Science, University of Shiga Prefecture, Shiga, Hikone 522-8533, Japan" /><meta name="citation_author" content="Sato, Kazuhiro" /><meta name="citation_author_institution" content="Institute of Plant Science and Resources, Okayama University, Kurashiki 710-0046, Japan" /><meta name="citation_author" content="Nasuda, Shuhei" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agriculture, Kyoto University, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Takumi, Shigeo" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agricultural Science, Kobe University, Nada-ku, Kobe 657-8501, Japan" /><meta name="citation_title" content="Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale De Novo Analysis of Leaf Transcripts of Aegilops tauschii, A Wild Wheat Progenitor" /><meta name="citation_firstpage" content="487" /><meta name="citation_lastpage" content="497" /><meta name="citation_doi" content="10.1093/dnares/dss028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2012/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Discovery of the DD-analyser, one of the ancestors of Triticum vulgare; citation_text=[In Japanese]; Agric. Hortic.;  citation_year=1944;  citation_volume=19;  citation_pages=889-90; " /><meta name="citation_reference" content="citation_title=The artificial synthesis of Triticum spelta; Rec. Genet. Soc. Am.;  citation_year=1944;  citation_volume=13;  citation_pages=26-7; " /><meta name="citation_reference" content="citation_title=Allozyme variation in Aegilops squarrosa; Hereditas;  citation_year=1993;  citation_volume=119;  citation_pages=117-22; " /><meta name="citation_reference" content="citation_title=The structure of the Aegilops tauschii genepool and the evolution of hexaploid wheat; Theor. Appl. Genet.;  citation_year=1998;  citation_volume=97;  citation_pages=657-70; " /><meta name="citation_reference" content="citation_title=Aegilops tauschii: genetic variation in Iran; Genet. Resour. Crop Evol.;  citation_year=2006;  citation_volume=53;  citation_pages=579-86; " /><meta name="citation_reference" content="citation_title=Natural variation for fertile triploid F1 formation in allohexaploid wheat speciation; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=509-18; " /><meta name="citation_reference" content="citation_title=Flowering time diversification and dispersal in central Eurasian wild wheat Aegilops tauschii Coss.: genealogical and ecological framework; PLoS ONE;  citation_year=2008;  citation_volume=3;  citation_pages=e3138" /><meta name="citation_reference" content="citation_title=Genealogical analysis of subspecies divergence and spikelet-shape diversification in central Eurasian wild wheat Aegilops tauschii Coss.; Plant Syst. Evol.;  citation_year=2009;  citation_volume=279;  citation_pages=233-44; " /><meta name="citation_reference" content="citation_title=Natural variation of morphological traits in wild wheat progenitor Aegilops tauschii Coss.; Breed. Sci.;  citation_year=2009;  citation_volume=59;  citation_pages=579-88; " /><meta name="citation_reference" content="citation_title=Distinct genetic regulation of flowering time and grain-filling period based on empirical study of D-genome diversity in synthetic hexaploid wheat lines; Breed. Sci.;  citation_year=2011;  citation_volume=61;  citation_pages=130-41; " /><meta name="citation_reference" content="citation_title=Variation in abscisic acid responsiveness of Aegilops tauschii and hexaploid wheat synthetics due to the D-genome diversity; Genes Genet. Syst.;  citation_year=2012;  citation_volume=87;  citation_pages=9-18; " /><meta name="citation_reference" content="citation_title=Population structure of wild wheat D-genome progenitor Aegilops tauschii Coss.: implications for intraspecific lineage diversification and evolution of common wheat; Mol. Ecol.;  citation_year=2010;  citation_volume=19;  citation_pages=999-1013; " /><meta name="citation_reference" content="citation_title=Origin of Cultivated Wheat; citation_publisher=Lavoisier Publishing, Paris; The World Wheat Book: A History of Wheat Breeding;  citation_year=2001;  citation_pages=3-56; " /><meta name="citation_reference" content="citation_title=An intervarietal molecular marker map in Triticum aestivum L. Em. Thell. and comparison with a map from a wide cross; Theor. Appl. Genet.;  citation_year=1997;  citation_volume=94;  citation_pages=367-77; " /><meta name="citation_reference" content="citation_title=Analysis of gene-derived SNP marker polymorphism in US wheat (Triticum aestivum L.) cultivars; Mol. Breed.;  citation_year=2009;  citation_volume=23;  citation_pages=23-33; " /><meta name="citation_reference" content="citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.); Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=1086-99; " /><meta name="citation_reference" content="citation_title=A new-synthesized 6x-wheat; Hereditas;  citation_year=1949;  citation_volume=35;  citation_issue=Suppl;  citation_pages=307-19; " /><meta name="citation_reference" content="citation_title=Durum wheat as a candidate for the unknown female progenitor of bread wheat: an empirical study with a highly fertile F1 hybrid with Aegilops tauschii Coss.; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=1710-7; " /><meta name="citation_reference" content="citation_title=Novel germplasm resources for improving environmental stress tolerance of hexaploid wheat; Crop Sci.;  citation_year=2008;  citation_volume=48;  citation_pages=1255-65; " /><meta name="citation_reference" content="citation_title=Genetic and molecular dissection of naturally occurring variation; Curr. Opin. Plant Biol.;  citation_year=2001;  citation_volume=4;  citation_pages=130-5; " /><meta name="citation_reference" content="citation_title=A microsatellite map of wheat; Genetics;  citation_year=1998;  citation_volume=149;  citation_pages=2007-23; " /><meta name="citation_reference" content="citation_title=A high-density microsatellite consensus map for bread wheat (Triticum aestivum L.); Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=1105-14; " /><meta name="citation_reference" content="citation_title=Utilization of next-generation sequencing platforms in plant genomics and genetic variation discovery; Mol. Breed.;  citation_year=2010;  citation_volume=25;  citation_pages=553-70; " /><meta name="citation_reference" content="citation_title=Development and implementation of high-throughput SNP genotyping in barley; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=582" /><meta name="citation_reference" content="citation_title=Annotation-based genome-wide SNP discovery in the large and complex Aegilops tauschii genome using next-generation sequencing without a reference genome sequence; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=59" /><meta name="citation_reference" content="citation_title=Characterization of three vernalization insensitive3-like (VIL) homologs in wild wheat, Aegilops tauschii Coss.; Hereditas;  citation_year=2012;  citation_volume=149;  citation_pages=62-71; " /><meta name="citation_reference" content="citation_title=Sequencing and assembly of low copy and genic regions of isolated Triticum aestivum chromosome arm 7DS; Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=768-75; " /><meta name="citation_reference" content="citation_title=Unlocking the barley genome by chromosomal and comparative genomics; Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=1249-63; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=EMBOSS: The European Molecular Biology Open Software Suite; Trends Genet.;  citation_year=2000;  citation_volume=16;  citation_pages=276-7; " /><meta name="citation_reference" content="citation_title=Web-based primer design for single nucleotide polymorphism analysis; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=613-5; " /><meta name="citation_reference" content="citation_title=Application of real-time PCR-based SNP detection for mapping of Net2, a causal D-genome gene for hybrid necrosis in interspecific crosses between tetraploid wheat and Aegilops tauschii; Genes Genet. Syst.;  citation_year=2012;  citation_volume=87;  citation_pages=137-43; " /><meta name="citation_reference" content="citation_title=Trypanotolerance in west African cattle and the population genetic effects of selection. PhD Thesis; citation_publisher=University of Dublin, ;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations; Genomics;  citation_year=1987;  citation_volume=1;  citation_pages=178-81; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values; Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=Development of 5006 full-length cDNAs in barley: a tool for accessing cereal genomics resources; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=81-9; " /><meta name="citation_reference" content="citation_title=Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=20-8; " /><meta name="citation_reference" content="citation_title=CAP3: a DNA sequence assembly program; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=The maternal and early embryonic transcriptome of the milkweed bug Oncopeltus fasciatus; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=61" /><meta name="citation_reference" content="citation_title=Maize (Zea mays L.) genome diversity as revealed by RNA-sequencing; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e33071" /><meta name="citation_reference" content="citation_title=dCAPS, a simple technique for the genetic analysis of single nucleotide polymorphisms: experimental applications in Arabidopsis thaliana genetics; Plant J.;  citation_year=1998;  citation_volume=14;  citation_pages=387-92; " /><meta name="citation_reference" content="citation_title=A robust method for detecting single-nucleotide changes as polymorphic markers by PCR; Plant J.;  citation_year=1998;  citation_volume=14;  citation_pages=381-5; " /><meta name="citation_reference" content="citation_title=Utilization of the CAPS/dCAPS method to convert rice SNPs into PCR-based markers; Breed. Sci.;  citation_year=2005;  citation_volume=55;  citation_pages=93-8; " /><meta name="citation_reference" content="citation_title=A chromosome bin map of 16,000 expressed sequence tag loci and distribution of genes among the three genomes of polyploid wheat; Genetics;  citation_year=2004;  citation_volume=168;  citation_pages=701-12; " /><meta name="citation_reference" content="citation_title=GrainGenes 2.0. An improved resource for the small-grains community; Plant Physiol.;  citation_year=2005;  citation_volume=139;  citation_pages=643-51; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/6/487/1301990/dss028.pdf" /><meta name="description" content="Abstract.  Construction of high-resolution genetic maps is important for genetic and genomic research, as well as for molecular breeding. Single nucleotide poly" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/6/487/481438" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/6/487/481438" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/6/487/481438");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss028';
    </script>
<title>
	Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale De Novo Analysis of Leaf Transcripts of Aegilops tauschii, A Wild Wheat Progenitor | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./481438" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYRlPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJZWhpc2EsIEp1bGlvIENlc2FyIE1hc2FydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUGxhbnQgR2VuZXRpY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgS29iZSBVbml2ZXJzaXR5LCBOYWRhLWt1LCBLb2JlIDY1Ny04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaW1penUsIEFraWZ1bWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgUmVzb3VyY2VzIE1hbmFnZW1lbnQsIFNjaG9vbCBvZiBFbnZpcm9ubWVudGFsIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgU2hpZ2EgUHJlZmVjdHVyZSwgU2hpZ2EsIEhpa29uZSA1MjItODUzMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBLYXp1aGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBQbGFudCBTY2llbmNlIGFuZCBSZXNvdXJjZXMsIE9rYXlhbWEgVW5pdmVyc2l0eSwgS3VyYXNoaWtpIDcxMC0wMDQ2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5hc3VkYSwgU2h1aGVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQbGFudCBHZW5ldGljcywgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlLCBLeW90byBVbml2ZXJzaXR5LCBLeW90byA2MDYtODUwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWt1bWksIFNoaWdlbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUGxhbnQgR2VuZXRpY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgS29iZSBVbml2ZXJzaXR5LCBOYWRhLWt1LCBLb2JlIDY1Ny04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGlzY292ZXJ5IG9mIEhpZ2gtQ29uZmlkZW5jZSBTaW5nbGUgTnVjbGVvdGlkZSBQb2x5bW9ycGhpc21zIGZyb20gTGFyZ2UtU2NhbGUgRGUgTm92byBBbmFseXNpcyBvZiBMZWFmIFRyYW5zY3JpcHRzIG9mIEFlZ2lsb3BzIHRhdXNjaGlpLCBBIFdpbGQgV2hlYXQgUHJvZ2VuaXRvciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2YgdGhlIERELWFuYWx5c2VyLCBvbmUgb2YgdGhlIGFuY2VzdG9ycyBvZiBUcml0aWN1bSB2dWxnYXJlOyBjaXRhdGlvbl90ZXh0PVtJbiBKYXBhbmVzZV07IEFncmljLiBIb3J0aWMuOyAgY2l0YXRpb25feWVhcj0xOTQ0OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYXJ0aWZpY2lhbCBzeW50aGVzaXMgb2YgVHJpdGljdW0gc3BlbHRhOyBSZWMuIEdlbmV0LiBTb2MuIEFtLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbG96eW1lIHZhcmlhdGlvbiBpbiBBZWdpbG9wcyBzcXVhcnJvc2E7IEhlcmVkaXRhczsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz0xMTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHJ1Y3R1cmUgb2YgdGhlIEFlZ2lsb3BzIHRhdXNjaGlpIGdlbmVwb29sIGFuZCB0aGUgZXZvbHV0aW9uIG9mIGhleGFwbG9pZCB3aGVhdDsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTY1Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWVnaWxvcHMgdGF1c2NoaWk6IGdlbmV0aWMgdmFyaWF0aW9uIGluIElyYW47IEdlbmV0LiBSZXNvdXIuIENyb3AgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz01NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5hdHVyYWwgdmFyaWF0aW9uIGZvciBmZXJ0aWxlIHRyaXBsb2lkIEYxIGZvcm1hdGlvbiBpbiBhbGxvaGV4YXBsb2lkIHdoZWF0IHNwZWNpYXRpb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9NTA5LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG93ZXJpbmcgdGltZSBkaXZlcnNpZmljYXRpb24gYW5kIGRpc3BlcnNhbCBpbiBjZW50cmFsIEV1cmFzaWFuIHdpbGQgd2hlYXQgQWVnaWxvcHMgdGF1c2NoaWkgQ29zcy46IGdlbmVhbG9naWNhbCBhbmQgZWNvbG9naWNhbCBmcmFtZXdvcms7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzEzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVhbG9naWNhbCBhbmFseXNpcyBvZiBzdWJzcGVjaWVzIGRpdmVyZ2VuY2UgYW5kIHNwaWtlbGV0LXNoYXBlIGRpdmVyc2lmaWNhdGlvbiBpbiBjZW50cmFsIEV1cmFzaWFuIHdpbGQgd2hlYXQgQWVnaWxvcHMgdGF1c2NoaWkgQ29zcy47IFBsYW50IFN5c3QuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTIzMy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmF0dXJhbCB2YXJpYXRpb24gb2YgbW9ycGhvbG9naWNhbCB0cmFpdHMgaW4gd2lsZCB3aGVhdCBwcm9nZW5pdG9yIEFlZ2lsb3BzIHRhdXNjaGlpIENvc3MuOyBCcmVlZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTU3OS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzdGluY3QgZ2VuZXRpYyByZWd1bGF0aW9uIG9mIGZsb3dlcmluZyB0aW1lIGFuZCBncmFpbi1maWxsaW5nIHBlcmlvZCBiYXNlZCBvbiBlbXBpcmljYWwgc3R1ZHkgb2YgRC1nZW5vbWUgZGl2ZXJzaXR5IGluIHN5bnRoZXRpYyBoZXhhcGxvaWQgd2hlYXQgbGluZXM7IEJyZWVkLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MTMwLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gYWJzY2lzaWMgYWNpZCByZXNwb25zaXZlbmVzcyBvZiBBZWdpbG9wcyB0YXVzY2hpaSBhbmQgaGV4YXBsb2lkIHdoZWF0IHN5bnRoZXRpY3MgZHVlIHRvIHRoZSBELWdlbm9tZSBkaXZlcnNpdHk7IEdlbmVzIEdlbmV0LiBTeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTktMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gc3RydWN0dXJlIG9mIHdpbGQgd2hlYXQgRC1nZW5vbWUgcHJvZ2VuaXRvciBBZWdpbG9wcyB0YXVzY2hpaSBDb3NzLjogaW1wbGljYXRpb25zIGZvciBpbnRyYXNwZWNpZmljIGxpbmVhZ2UgZGl2ZXJzaWZpY2F0aW9uIGFuZCBldm9sdXRpb24gb2YgY29tbW9uIHdoZWF0OyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9OTk5LTEwMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbiBvZiBDdWx0aXZhdGVkIFdoZWF0OyBjaXRhdGlvbl9wdWJsaXNoZXI9TGF2b2lzaWVyIFB1Ymxpc2hpbmcsIFBhcmlzOyBUaGUgV29ybGQgV2hlYXQgQm9vazogQSBIaXN0b3J5IG9mIFdoZWF0IEJyZWVkaW5nOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fcGFnZXM9My01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gaW50ZXJ2YXJpZXRhbCBtb2xlY3VsYXIgbWFya2VyIG1hcCBpbiBUcml0aWN1bSBhZXN0aXZ1bSBMLiBFbS4gVGhlbGwuIGFuZCBjb21wYXJpc29uIHdpdGggYSBtYXAgZnJvbSBhIHdpZGUgY3Jvc3M7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0zNjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGdlbmUtZGVyaXZlZCBTTlAgbWFya2VyIHBvbHltb3JwaGlzbSBpbiBVUyB3aGVhdCAoVHJpdGljdW0gYWVzdGl2dW0gTC4pIGN1bHRpdmFyczsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdC1zcGVjaWZpYywgc2luZ2xlLW51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIGRpc2NvdmVyeSBhbmQgbGlua2FnZSBhbmFseXNpcyBpbiBoZXhhcGxvaWQgYnJlYWQgd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3LXN5bnRoZXNpemVkIDZ4LXdoZWF0OyBIZXJlZGl0YXM7ICBjaXRhdGlvbl95ZWFyPTE5NDk7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbDsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHVydW0gd2hlYXQgYXMgYSBjYW5kaWRhdGUgZm9yIHRoZSB1bmtub3duIGZlbWFsZSBwcm9nZW5pdG9yIG9mIGJyZWFkIHdoZWF0OiBhbiBlbXBpcmljYWwgc3R1ZHkgd2l0aCBhIGhpZ2hseSBmZXJ0aWxlIEYxIGh5YnJpZCB3aXRoIEFlZ2lsb3BzIHRhdXNjaGlpIENvc3MuOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE3MTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgZ2VybXBsYXNtIHJlc291cmNlcyBmb3IgaW1wcm92aW5nIGVudmlyb25tZW50YWwgc3RyZXNzIHRvbGVyYW5jZSBvZiBoZXhhcGxvaWQgd2hlYXQ7IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTEyNTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5kIG1vbGVjdWxhciBkaXNzZWN0aW9uIG9mIG5hdHVyYWxseSBvY2N1cnJpbmcgdmFyaWF0aW9uOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTMwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUgbWFwIG9mIHdoZWF0OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0yMDA3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtZGVuc2l0eSBtaWNyb3NhdGVsbGl0ZSBjb25zZW5zdXMgbWFwIGZvciBicmVhZCB3aGVhdCAoVHJpdGljdW0gYWVzdGl2dW0gTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTExMDUtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVV0aWxpemF0aW9uIG9mIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHBsYXRmb3JtcyBpbiBwbGFudCBnZW5vbWljcyBhbmQgZ2VuZXRpYyB2YXJpYXRpb24gZGlzY292ZXJ5OyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTU1My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGltcGxlbWVudGF0aW9uIG9mIGhpZ2gtdGhyb3VnaHB1dCBTTlAgZ2Vub3R5cGluZyBpbiBiYXJsZXk7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubm90YXRpb24tYmFzZWQgZ2Vub21lLXdpZGUgU05QIGRpc2NvdmVyeSBpbiB0aGUgbGFyZ2UgYW5kIGNvbXBsZXggQWVnaWxvcHMgdGF1c2NoaWkgZ2Vub21lIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHdpdGhvdXQgYSByZWZlcmVuY2UgZ2Vub21lIHNlcXVlbmNlOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhyZWUgdmVybmFsaXphdGlvbiBpbnNlbnNpdGl2ZTMtbGlrZSAoVklMKSBob21vbG9ncyBpbiB3aWxkIHdoZWF0LCBBZWdpbG9wcyB0YXVzY2hpaSBDb3NzLjsgSGVyZWRpdGFzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTYyLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGFuZCBhc3NlbWJseSBvZiBsb3cgY29weSBhbmQgZ2VuaWMgcmVnaW9ucyBvZiBpc29sYXRlZCBUcml0aWN1bSBhZXN0aXZ1bSBjaHJvbW9zb21lIGFybSA3RFM7IFBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz03NjgtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVubG9ja2luZyB0aGUgYmFybGV5IGdlbm9tZSBieSBjaHJvbW9zb21hbCBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3M7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMjQ5LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVNQk9TUzogVGhlIEV1cm9wZWFuIE1vbGVjdWxhciBCaW9sb2d5IE9wZW4gU29mdHdhcmUgU3VpdGU7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yNzYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2ViLWJhc2VkIHByaW1lciBkZXNpZ24gZm9yIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmFseXNpczsgVHJlbmRzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTYxMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcHBsaWNhdGlvbiBvZiByZWFsLXRpbWUgUENSLWJhc2VkIFNOUCBkZXRlY3Rpb24gZm9yIG1hcHBpbmcgb2YgTmV0MiwgYSBjYXVzYWwgRC1nZW5vbWUgZ2VuZSBmb3IgaHlicmlkIG5lY3Jvc2lzIGluIGludGVyc3BlY2lmaWMgY3Jvc3NlcyBiZXR3ZWVuIHRldHJhcGxvaWQgd2hlYXQgYW5kIEFlZ2lsb3BzIHRhdXNjaGlpOyBHZW5lcyBHZW5ldC4gU3lzdC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyeXBhbm90b2xlcmFuY2UgaW4gd2VzdCBBZnJpY2FuIGNhdHRsZSBhbmQgdGhlIHBvcHVsYXRpb24gZ2VuZXRpYyBlZmZlY3RzIG9mIHNlbGVjdGlvbi4gUGhEIFRoZXNpczsgY2l0YXRpb25fcHVibGlzaGVyPVVuaXZlcnNpdHkgb2YgRHVibGluLCA7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BUE1BS0VSOiBhbiBpbnRlcmFjdGl2ZSBjb21wdXRlciBwYWNrYWdlIGZvciBjb25zdHJ1Y3RpbmcgcHJpbWFyeSBnZW5ldGljIGxpbmthZ2UgbWFwcyBvZiBleHBlcmltZW50YWwgYW5kIG5hdHVyYWwgcG9wdWxhdGlvbnM7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xNzgtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlc3RpbWF0aW9uIG9mIG1hcCBkaXN0YW5jZSBmcm9tIHJlY29tYmluYXRpb24gdmFsdWVzOyBBbm4uIEV1Z2VuLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiA1MDA2IGZ1bGwtbGVuZ3RoIGNETkFzIGluIGJhcmxleTogYSB0b29sIGZvciBhY2Nlc3NpbmcgY2VyZWFsIGdlbm9taWNzIHJlc291cmNlczsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHNlcXVlbmNlIGFuYWx5c2lzIG9mIDI0LDc4MyBiYXJsZXkgZnVsbC1sZW5ndGggY0ROQXMgZGVyaXZlZCBmcm9tIDEyIGNsb25lIGxpYnJhcmllczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogYSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTg2OC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1hdGVybmFsIGFuZCBlYXJseSBlbWJyeW9uaWMgdHJhbnNjcmlwdG9tZSBvZiB0aGUgbWlsa3dlZWQgYnVnIE9uY29wZWx0dXMgZmFzY2lhdHVzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1haXplIChaZWEgbWF5cyBMLikgZ2Vub21lIGRpdmVyc2l0eSBhcyByZXZlYWxlZCBieSBSTkEtc2VxdWVuY2luZzsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMzA3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWRDQVBTLCBhIHNpbXBsZSB0ZWNobmlxdWUgZm9yIHRoZSBnZW5ldGljIGFuYWx5c2lzIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXM6IGV4cGVyaW1lbnRhbCBhcHBsaWNhdGlvbnMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgZ2VuZXRpY3M7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9Mzg3LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJvYnVzdCBtZXRob2QgZm9yIGRldGVjdGluZyBzaW5nbGUtbnVjbGVvdGlkZSBjaGFuZ2VzIGFzIHBvbHltb3JwaGljIG1hcmtlcnMgYnkgUENSOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM4MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VdGlsaXphdGlvbiBvZiB0aGUgQ0FQUy9kQ0FQUyBtZXRob2QgdG8gY29udmVydCByaWNlIFNOUHMgaW50byBQQ1ItYmFzZWQgbWFya2VyczsgQnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz05My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNocm9tb3NvbWUgYmluIG1hcCBvZiAxNiwwMDAgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZyBsb2NpIGFuZCBkaXN0cmlidXRpb24gb2YgZ2VuZXMgYW1vbmcgdGhlIHRocmVlIGdlbm9tZXMgb2YgcG9seXBsb2lkIHdoZWF0OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz03MDEtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYWluR2VuZXMgMi4wLiBBbiBpbXByb3ZlZCByZXNvdXJjZSBmb3IgdGhlIHNtYWxsLWdyYWlucyBjb21tdW5pdHk7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzOTsgIGNpdGF0aW9uX3BhZ2VzPTY0My01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE5LzYvNDg3LzEzMDE5OTAvZHNzMDI4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBDb25zdHJ1Y3Rpb24gb2YgaGlnaC1yZXNvbHV0aW9uIGdlbmV0aWMgbWFwcyBpcyBpbXBvcnRhbnQgZm9yIGdlbmV0aWMgYW5kIGdlbm9taWMgcmVzZWFyY2gsIGFzIHdlbGwgYXMgZm9yIG1vbGVjdWxhciBicmVlZGluZy4gU2luZ2xlIG51Y2xlb3RpZGUgcG9seSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDgxNDM4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFUv+S6H9zCUQs1/YB9LXm3lUHZT" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f487%2f481438" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f487%2f481438" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18878" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&amp;Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/6">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7303773">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303773">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303775">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303775">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303779">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303779">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303797">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303797">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303828">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303828">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303830">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303830">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303832">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303832">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7303834">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7303834">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/6/477/481278">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale <em>De Novo</em> Analysis of Leaf Transcripts of <em>Aegilops tauschii</em>, A Wild Wheat Progenitor
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Julio Cesar Masaru Iehisa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agricultural Science</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Nada-ku, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Julio+Cesar+Masaru+Iehisa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iehisa J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iehisa J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Julio Cesar Masaru Iehisa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akifumi Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Resources Management, School of Environmental Science</div>, <div class="insititution">University of Shiga Prefecture</div>, <div class="addr-line">Shiga, Hikone 522-8533</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akifumi+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akifumi Shimizu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhiro Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Institute of Plant Science and Resources</div>, <div class="insititution">Okayama University</div>, <div class="addr-line">Kurashiki 710-0046</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhiro+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuhiro Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuhei Nasuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuhei+Nasuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nasuda S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nasuda S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shuhei Nasuda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeo Takumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agricultural Science</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Nada-ku, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 78-803-5860. Fax. +81 78-803-5860. Email: <a href="mailto:takumi@kobe-u.ac.jp" target="_blank">takumi@kobe-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeo+Takumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takumi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takumi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeo Takumi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 487–497, <a href='https://doi.org/10.1093/dnares/dss028'>https://doi.org/10.1093/dnares/dss028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 November 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 April 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 October 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="481438" href="/dnaresearch/article-pdf/19/6/487/1301990/dss028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Julio Cesar Masaru Iehisa, Akifumi Shimizu, Kazuhiro Sato, Shuhei Nasuda, Shigeo Takumi;  Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale <em>De Novo</em> Analysis of Leaf Transcripts of <em>Aegilops tauschii</em>, A Wild Wheat Progenitor, <em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 487–497, <a href='https://doi.org/10.1093/dnares/dss028'>https://doi.org/10.1093/dnares/dss028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/481438?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/481438?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Discovery%20of%20High-Confidence%20Single%20Nucleotide%20Polymorphisms%20from%20Large-Scale%20%26lt%3Bem%26gt%3BDe%20Novo%26lt%3B%2Fem%26gt%3B%20Analysis%20of%20Leaf%20Transcripts%20of%20%26lt%3Bem%26gt%3BAegilops%20tauschii%26lt%3B%2Fem%26gt%3B%2C%20A%20Wild%20Wheat%20Progenitor&amp;publicationDate=2012-11-02&amp;volumeNum=19&amp;issueNum=6&amp;author=Iehisa%2C%20Julio%20Cesar%20Masaru%3B%20Shimizu%2C%20Akifumi&amp;startPage=487&amp;endPage=497&amp;contentId=10.1093%2Fdnares%2Fdss028&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f487%2f481438" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7303773 id="7303773" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Construction of high-resolution genetic maps is important for genetic and genomic research, as well as for molecular breeding. Single nucleotide polymorphisms (SNPs) are the predominant class of genetic variation and can be used as molecular markers. <em>Aegilops tauschii</em>, the D-genome donor of common wheat, is considered a valuable genetic resource for wheat improvement. Our previous study implied that <em>Ae. tauschii</em> accessions can be genealogically divided into two major lineages. In this study, the transcriptome of two <em>Ae. tauschii</em> accessions from each lineage, lineage 1 (L1) and 2 (L2), was sequenced, yielding 9435 SNPs and 739 insertion/deletion polymorphisms (indels) after <em>de novo</em> assembly of the reads. Based on 36 contig sequences, 31 SNPs and six indels were validated on 20 diverse <em>Ae. tauschii</em> accessions. Because almost all of the SNP markers were polymorphic between L1 and L2, and the D-genome donor of common wheat is presumed to belong to L2, these markers are available for D-genome typing in crosses between common wheat varieties and L1-derived synthetic wheat. Due to the conserved synteny between wheat and barley chromosomes, the high-density expressed sequence tag barley map and the hypothetical gene order in barley can be applied to develop markers on target chromosomal regions in wheat.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aegilops tauschii&quot;"><em>Aegilops tauschii</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;expression sequence tag&quot;">expression sequence tag</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism&quot;">single nucleotide polymorphism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="wheat">wheat</a></div>                            <h2 scrollto-destination=7303775 id="7303775" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Common wheat (<em>Triticum aestivum</em> L., genome constitution AABBDD) is an allohexaploid species that arose by natural hybridization between tetraploid wheat <em>Triticum turgidum</em> L. (AABB), including emmer and durum wheats, and a diploid wild wheat relative <em>Aegilops tauschii</em> Coss. (DD).<sup><a reveal-id="DSS028C1" data-open="DSS028C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS028C2" data-open="DSS028C2" class="link link-ref link-reveal xref-bibr">2</a></sup><em>Ae. tauschii</em> is widely distributed in Eurasia and shows abundant genetic variation.<sup><a reveal-id="DSS028C3  DSS028C11" data-open="DSS028C3  DSS028C11" class="link link-ref link-reveal xref-bibr">3–11</a></sup> Population structure analyses revealed two major phylogenetic lineages and an HG17 minor lineage [haplogroup lineage (HGL) 17] in <em>Ae. tauschii</em>.<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In turn, the major lineages 1 (L1) and 2 (L2) can be genealogically divided into six and three sublineages, respectively. It is supposed that the <em>Ae. tauschii</em> populations involved in the origin of common wheat are limited to a narrow distribution range, apparently restricted to L2, which has given rise to a founder effect in hexaploid wheat.<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS028C13" data-open="DSS028C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In fact, the D-genome of common wheat is less polymorphic than the A- and B-genomes.<sup><a reveal-id="DSS028C14  DSS028C16" data-open="DSS028C14  DSS028C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> Tetraploid wheat and <em>Ae. tauschii</em> can be crossed artificially to produce synthetic hexaploid wheat.<sup><a reveal-id="DSS028C17" data-open="DSS028C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS028C18" data-open="DSS028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> These synthetics can be used as intermediates to exploit the natural variation of <em>Ae. tauschii</em> in hexaploid wheat improvement.<sup><a reveal-id="DSS028C19" data-open="DSS028C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>High-resolution genetic map construction is important for genetic and genomic research, as well as for molecular breeding.<sup><a reveal-id="DSS028C20" data-open="DSS028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In wheat, genetic maps are commonly constructed using simple sequence repeats (SSR) markers,<sup><a reveal-id="DSS028C21" data-open="DSS028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS028C22" data-open="DSS028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> but the number of SSR markers reported in public databases such as the National BioResource Project (NBRP) KOMUGI website (<a class="link link-uri" href="http://www.shigen.nig.ac.jp/wheat/komugi/strains/aboutNbrpMarker.jsp">http://www.shigen.nig.ac.jp/wheat/komugi/strains/aboutNbrpMarker.jsp</a>) and GrainGene website (<a class="link link-uri" href="http://wheat.pw.usda.gov/GG2/maps.shtml">http://wheat.pw.usda.gov/GG2/maps.shtml</a>) is not sufficient for high-resolution maps. Although SSR are the most popular marker system, the constructed map resolution remains low in plant species without a known genome sequence. An alternative marker system, single nucleotide polymorphism (SNP), has received considerable attention because it is the predominant class of genetic variation.<sup><a reveal-id="DSS028C23" data-open="DSS028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> High-throughput SNP-typing systems can be developed for organisms with a reference genome or comprehensive expressed sequence tag (EST) database, e.g. barley.<sup><a reveal-id="DSS028C24" data-open="DSS028C24" class="link link-ref link-reveal xref-bibr">24</a></sup> But, in many plants having large and complex genomes and insufficient reference information, genome-wide SNP discovery is insufficient because of the presence of highly repetitive regions. In <em>Ae. tauschii</em>, an organism without a reference genome and composed of highly repetitive regions, annotation-based genome-wide SNP discovery has been developed to overcome these problems. In this strategy, Roche 454 shotgun reads with low genome coverage of one genotype were annotated, and then genomic and cDNA shotgun reads of another genotype were generated on SOLiD or Solexa platforms to identify putative SNPs. However, around 56% of the sequence length, characterized as a repetitive region, was excluded from the analysis.<sup><a reveal-id="DSS028C25" data-open="DSS028C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>In the present study, RNA-seq, a next generation sequencing technology for transcripts, was used as a cost-effective, simpler means of SNP discovery than annotation. SNPs, insertion/deletion polymorphisms (indels), and SSRs were discovered from <em>de novo</em> assembly data from ESTs of two <em>Ae. tauschii</em> accessions, PI476874 (L1) and IG47182 (L2), that were mapped on an SSR linkage map using an F<sub>2</sub> population.<sup><a reveal-id="DSS028C26" data-open="DSS028C26" class="link link-ref link-reveal xref-bibr">26</a></sup> We show that SNP markers can be developed in a chromosomal region of interest using the low-coverage genome sequence of wheat chromosome 7D<sup><a reveal-id="DSS028C27" data-open="DSS028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> or barley GenomeZipper results obtained from a synteny model conserved among barley, <em>Brachypodium</em>, rice, and sorghum.<sup><a reveal-id="DSS028C28" data-open="DSS028C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h2 scrollto-destination=7303779 id="7303779" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7303780 id="7303780" class="section-title" data-legacy-id=s2a>2.1. Plant material, RNA extraction, and next generation sequencing</h3>
<p>Twenty <em>Ae. tauschii</em> accessions from each sublineage were used (Table <a reveal-id="DSS028TB1" data-open="DSS028TB1" class="link link-reveal link-table xref-fig">1</a>). Two <em>Ae. tauschii</em> accessions, PI476874 from the L1-2 sublineage and IG47182 from the L2-2 sublineage, were used for cDNA sequencing. Total RNA was isolated from leaves of 21-day-old seedlings using an RNeasy Plant Mini Kit (QIAGEN, Hilden, Germany). mRNA was isolated from 45 µg of RNA using an mRNA Purification Kit (Takara-Bio, Ohtsu, Japan). A 200 ng aliquot of mRNA was used to fragment the mRNA and synthesize cDNA from it using a cDNA Synthesis System (Roche Diagnostics, Mannheim, Germany). Approximately 10<sup>8</sup> adapter-ligated cDNA molecules from the samples were used for library preparation using a GS FLX Titanium Rapid Library Preparation Kit (Roche Diagnostics). The library was sequenced with a GS FLX Titanium Sequencing Kit on a GS FLX System (Roche Diagnostics) according to the manufacturer's instructions. Files containing raw sequence data were deposited in the sequence read archive of the DNA Data Bank of Japan (accession number: DRA000536). </p>                        <a id="7303782" scrollto-destination="7303782"></a>
<div content-id="DSS028TB1" class="table-modal"><div class="table-wrap"><div id="DSS028TB1" class="table-wrap-title" data-id="DSS028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Lineage, sublineage, accession number, and origin of <em>Ae. tauschii</em> accessions used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Lineage–sublineage<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> </th><th>Accession number (country) </th></tr></thead><tbody><tr><td>L1–1 </td><td>IG48508 (Turkmenistan), KU-2627 (Afghanistan) </td></tr><tr><td>L1–2 </td><td><span class="underline">PI476874</span> (Afghanistan), IG126387 (Turkmenistan) </td></tr><tr><td>L1–3 </td><td>KU-2826 (Georgia), KU-2087 (Iran) </td></tr><tr><td>L1–4 </td><td>IG131606 (Kyrgyzstan), IG48559 (Tajikistan) </td></tr><tr><td>L1–5 </td><td>IG48747 (Armenia), KU-2144 (Iran) </td></tr><tr><td>L1–6 </td><td>AT47 (China), AT76 (China) </td></tr><tr><td>L2–1 </td><td>KU-2069 (Iran), KU-2811 (Armenia) </td></tr><tr><td>L2–2 </td><td><span class="underline">IG47182</span> (Azerbaijan), KU-2100 (Iran) </td></tr><tr><td>L2–3 </td><td>KU-2159 (Iran), KU-2093 (Iran) </td></tr><tr><td>HGL17 </td><td>AE454 (Georgia), AE929 (Georgia) </td></tr></tbody></table></div><div content-id="DSS028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Lineage–sublineage<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> </th><th>Accession number (country) </th></tr></thead><tbody><tr><td>L1–1 </td><td>IG48508 (Turkmenistan), KU-2627 (Afghanistan) </td></tr><tr><td>L1–2 </td><td><span class="underline">PI476874</span> (Afghanistan), IG126387 (Turkmenistan) </td></tr><tr><td>L1–3 </td><td>KU-2826 (Georgia), KU-2087 (Iran) </td></tr><tr><td>L1–4 </td><td>IG131606 (Kyrgyzstan), IG48559 (Tajikistan) </td></tr><tr><td>L1–5 </td><td>IG48747 (Armenia), KU-2144 (Iran) </td></tr><tr><td>L1–6 </td><td>AT47 (China), AT76 (China) </td></tr><tr><td>L2–1 </td><td>KU-2069 (Iran), KU-2811 (Armenia) </td></tr><tr><td>L2–2 </td><td><span class="underline">IG47182</span> (Azerbaijan), KU-2100 (Iran) </td></tr><tr><td>L2–3 </td><td>KU-2159 (Iran), KU-2093 (Iran) </td></tr><tr><td>HGL17 </td><td>AE454 (Georgia), AE929 (Georgia) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Underlining indicates the accessions used for RNA-seq.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303782" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS028TB1" class="table-wrap-title" data-id="DSS028TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Lineage, sublineage, accession number, and origin of <em>Ae. tauschii</em> accessions used in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Lineage–sublineage<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> </th><th>Accession number (country) </th></tr></thead><tbody><tr><td>L1–1 </td><td>IG48508 (Turkmenistan), KU-2627 (Afghanistan) </td></tr><tr><td>L1–2 </td><td><span class="underline">PI476874</span> (Afghanistan), IG126387 (Turkmenistan) </td></tr><tr><td>L1–3 </td><td>KU-2826 (Georgia), KU-2087 (Iran) </td></tr><tr><td>L1–4 </td><td>IG131606 (Kyrgyzstan), IG48559 (Tajikistan) </td></tr><tr><td>L1–5 </td><td>IG48747 (Armenia), KU-2144 (Iran) </td></tr><tr><td>L1–6 </td><td>AT47 (China), AT76 (China) </td></tr><tr><td>L2–1 </td><td>KU-2069 (Iran), KU-2811 (Armenia) </td></tr><tr><td>L2–2 </td><td><span class="underline">IG47182</span> (Azerbaijan), KU-2100 (Iran) </td></tr><tr><td>L2–3 </td><td>KU-2159 (Iran), KU-2093 (Iran) </td></tr><tr><td>HGL17 </td><td>AE454 (Georgia), AE929 (Georgia) </td></tr></tbody></table></div><div content-id="DSS028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Lineage–sublineage<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> </th><th>Accession number (country) </th></tr></thead><tbody><tr><td>L1–1 </td><td>IG48508 (Turkmenistan), KU-2627 (Afghanistan) </td></tr><tr><td>L1–2 </td><td><span class="underline">PI476874</span> (Afghanistan), IG126387 (Turkmenistan) </td></tr><tr><td>L1–3 </td><td>KU-2826 (Georgia), KU-2087 (Iran) </td></tr><tr><td>L1–4 </td><td>IG131606 (Kyrgyzstan), IG48559 (Tajikistan) </td></tr><tr><td>L1–5 </td><td>IG48747 (Armenia), KU-2144 (Iran) </td></tr><tr><td>L1–6 </td><td>AT47 (China), AT76 (China) </td></tr><tr><td>L2–1 </td><td>KU-2069 (Iran), KU-2811 (Armenia) </td></tr><tr><td>L2–2 </td><td><span class="underline">IG47182</span> (Azerbaijan), KU-2100 (Iran) </td></tr><tr><td>L2–3 </td><td>KU-2159 (Iran), KU-2093 (Iran) </td></tr><tr><td>HGL17 </td><td>AE454 (Georgia), AE929 (Georgia) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Underlining indicates the accessions used for RNA-seq.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303782" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7303783 id="7303783" class="section-title" data-legacy-id=s2b>2.2. Assembly, SNP and indel discovery, and SSR mining</h3>
<p>All reads from both accessions were pooled and <em>de novo</em> assembled with the GS <em>de novo</em> assembler algorithm (Newbler) version 2.6 (Roche Diagnostics) to generate reference contig sequences. During assembly, primer sequences and poly-A tails were trimmed from raw reads, and parameters of a minimum overlap length of 40 bp and minimum overlap identity of 90% were used. SNPs and indels were discovered by aligning all individual reads to the reference contig sequences using GS Reference Mapper version 2.6 (Roche Diagnostics). Only the accession-specific sequence variants (supported by at least two reads) were extracted as true polymorphisms from ‘All’ and ‘HC’ sequence differences produced by the GS Reference Mapper. SSR motifs within the reference contig sequences were identified by Sputnik software (<a class="link link-uri" href="http://espressosoftware.com/sputnik/index.html">http://espressosoftware.com/sputnik/index.html</a>), and indels were searched for within these motifs to extract polymorphic SSRs.</p>                            <h3 scrollto-destination=7303785 id="7303785" class="section-title" data-legacy-id=s2c>2.3. Gene annotation and analysis of synonymous and non-synonymous mutations</h3>
<p>The reference sequences were aligned with the National Center for Biotechnology Information (NCBI) non-redundant (nr) protein database and the <em>Brachypodium distachyon</em> (version 1.2, <a class="link link-uri" href="http://ftp://ftpmips.helmholtz-muenchen.de/plants/brachypodium/v1.2">ftp://ftpmips.helmholtz-muenchen.de/plants/brachypodium/v1.2</a>), rice (RAP-DB, <a class="link link-uri" href="http://rapdb.dna.affrc.go.jp/">http://rapdb.dna.affrc.go.jp/</a>), and wheat TriFLDB protein databases (<a class="link link-uri" href="http://trifldb.psc.riken.jp/index.pl">http://trifldb.psc.riken.jp/index.pl</a>) using BlastX with an <em>E</em>-value cutoff of 10<sup>−3</sup>. Gene ontology (GO) terms were assigned using Blast2GO<sup><a reveal-id="DSS028C29" data-open="DSS028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and a locally installed database based on BlastX hits against the NCBI nr database.</p><p>To study whether the discovered SNPs and indels affect amino acid sequences, the longest open reading frames (ORFs) of the isotigs that include all the 1793 contigs with high confidence (HC) polymorphism were extracted using EMBOSS.<sup><a reveal-id="DSS028C30" data-open="DSS028C30" class="link link-ref link-reveal xref-bibr">30</a></sup> The isotigs without any BlastX result, described above, were excluded from analysis. Only the ORFs supported by the BlastX search were selected.</p>                            <h3 scrollto-destination=7303788 id="7303788" class="section-title" data-legacy-id=s2d>2.4. SNP and SSR validation</h3>
<p>Validation of identified SNPs was performed through cleaved amplified polymorphic sequence (CAPS), derived CAPS (dCAPS), and high-resolution melting (HRM) methods. Gene-specific primer sequences, SNP locations, product length, and restriction enzymes are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. dCAPS primers were designed using the dCAPS Finder 2.0 program available on the website <a class="link link-uri" href="http://helix.wustl.edu/dcaps/dcaps.html">http://helix.wustl.edu/dcaps/dcaps.html</a>.<sup><a reveal-id="DSS028C31" data-open="DSS028C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The polymerase chain reaction (PCR) conditions for CAPS and dCAPS markers were 1 cycle of 94°C for 2 min and 40 cycles of 94°C for 30 s, 60°C (for CAPS) or 56°C (for dCAPS) for 30 s, and 68°C for 30 s. HRM analysis was performed using a LightCycler 480 Real-Time PCR System and LightCycler 480 HRM Master 2× reagents (Roche Diagnostics) according to our previous study.<sup><a reveal-id="DSS028C32" data-open="DSS028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> For HRM analysis, all samples were spiked with 10–50% of PI476874 DNA to facilitate discrimination of the homozygous genotype. All PCR products were checked on a 1.5% agarose gel to ensure the presence of a single band.</p><p>SSR markers, depending on the indel length (see section 3.4), were amplified following the same conditions for CAPS markers or were analysed by HRM as described above. The polymorphic information content (PIC) was calculated using Excel Microsatellite toolkit add-in software.<sup><a reveal-id="DSS028C33" data-open="DSS028C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p>                            <h3 scrollto-destination=7303791 id="7303791" class="section-title" data-legacy-id=s2e>2.5. Comparison of HC SNPs and previously reported SNP dataset</h3>
<p>For all of the 4337 HC SNPs, 100 bp sequences were extracted from 1748 contigs, positioning SNPs in the middle of the sequence. These sequences were BlastN searched against <em>Ae. tauschii</em> gene sequences with SNPs (<em>E</em>-value cutoff of 10 and gap extension penalty of 0) reported previously and published at <a class="link link-uri" href="http://avena.pw.usda.gov/wheatD/agsnp.shtml">http://avena.pw.usda.gov/wheatD/agsnp.shtml</a>.<sup><a reveal-id="DSS028C25" data-open="DSS028C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p>                            <h3 scrollto-destination=7303793 id="7303793" class="section-title" data-legacy-id=s2f>2.6. Linkage map construction</h3>
<p>A set of 104 F<sub>2</sub> individuals derived from a cross between PI476874 and IG47182 was used as the mapping population. A total of 19 cfd, 21 barc, 17 wmc, 20 gwm, 8 gdm, and 2 hbg SSR markers (<a class="link link-uri" href="http://wheat.pw.usda.gov/GG2/index.shtml">http://wheat.pw.usda.gov/GG2/index.shtml</a>, <a class="link link-uri" href="http://nics.naro.affrc.go.jp/team/dna_marker/">http://nics.naro.affrc.go.jp/team/dna_marker/</a>) were assigned to each chromosome as anchor markers. A genetic map was constructed using MAPMAKER/EXP version 3.0b.<sup><a reveal-id="DSS028C34" data-open="DSS028C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The genetic distances were calculated with the Kosambi function.<sup><a reveal-id="DSS028C35" data-open="DSS028C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h3 scrollto-destination=7303795 id="7303795" class="section-title" data-legacy-id=s2g>2.7. In silico mapping of contigs with HC SNPs on virtual barley chromosomes</h3>
<p>Shotgun barley genomic reads mapped on the virtual barley chromosomes were extracted from flow-sorted chromosome reads submitted by Mayer <em>et al</em>.<sup><a reveal-id="DSS028C28" data-open="DSS028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (accession number ERP000445). The tBlastX algorithm was used to align the 1793 contigs with extracted barley genomic reads and against a set of 5006 full-length cDNAs (accession numbers AK248134 to AK253139),<sup><a reveal-id="DSS028C36" data-open="DSS028C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and a set of 23623 full-length cDNAs (accession numbers AK353559 to AK377172),<sup><a reveal-id="DSS028C37" data-open="DSS028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> with an <em>E</em>-value cutoff of 10<sup>−1</sup>. The Blast search results obtained against <em>Brachypodium</em> and rice database were also used (see section 2.3).</p>                            <h2 scrollto-destination=7303797 id="7303797" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7303798 id="7303798" class="section-title" data-legacy-id=s3a>3.1. Sequencing and assembly of Ae. tauschii ESTs</h3>
<p>To see how many SNPs could be discovered in one RNA-seq experiment between two phylogenetically distinct <em>Ae. tauschii</em> accessions, PI476874 and IG47182 (respectively from L1 and L2) were selected. The sequencing of leaf cDNA libraries produced 669 383 and 700 124 reads corresponding, respectively, to 247 Mb and 254 Mb per accession after trimming. All reads from both accession-derived libraries were assembled using Newbler 2.6. Newbler assembled reads into a contig representing transcript regions, isotigs (sets of contigs representing a partial or entire transcript), and isogroups (sets of isotigs). During the assembly process, Newbler constructed multiple alignments of overlapping reads and divided them into consistent sequences, i.e. contigs. Alignments that cannot be divided are collected as isogroups, and branching structures between them are searched by traversal paths through connected branches, i.e. isotigs (isotigs and isogroups are Newbler-specific terms). Different isotigs within the same isogroup represent alternative splicing variants. Thus, an isogroup represents genes, isoforms, or gene families. In this study, sequences shorter than 100 bp were excluded from the following analyses. A total of 10 224 contigs were assembled into 9145 isotigs (including some contigs that were not combined into isotigs), with a total of 7753 isogroups obtained. The length of the majority of contigs and isotigs ranged between 500 and 1000 bp, with an average of 786 and 1021 bp, respectively (Table <a reveal-id="DSS028TB2" data-open="DSS028TB2" class="link link-reveal link-table xref-fig">2</a>). However, among isotigs, a sequence length as long as 11 328 bp was found (isotig02315), which showed sequence similarity to a <em>Brachypodium distachyon</em> E3 ubiquitin-protein ligase UPL1-like protein (XP_003575554.1, <em>E</em>-value = 0). </p>                        <a id="7303800" scrollto-destination="7303800"></a>
<div content-id="DSS028TB2" class="table-modal"><div class="table-wrap"><div id="DSS028TB2" class="table-wrap-title" data-id="DSS028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Contig and isotig sequence length distribution</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sequence length (bp) </th><th>Number of contig </th><th>Number of isotig </th></tr></thead><tbody><tr><td>100–500 </td><td>3300 </td><td>1162 </td></tr><tr><td>501–1000 </td><td>4350 </td><td>4612 </td></tr><tr><td>1001–1500 </td><td>1622 </td><td>1934 </td></tr><tr><td>1501–2000 </td><td>594 </td><td>828 </td></tr><tr><td>2001–2500 </td><td>199 </td><td>305 </td></tr><tr><td>&gt;2500 </td><td>159 </td><td>304 </td></tr><tr><td>Total </td><td>10 224 </td><td>9145 </td></tr><tr><td>Average length (bp) </td><td>786 </td><td>1021 </td></tr></tbody></table></div><div content-id="DSS028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sequence length (bp) </th><th>Number of contig </th><th>Number of isotig </th></tr></thead><tbody><tr><td>100–500 </td><td>3300 </td><td>1162 </td></tr><tr><td>501–1000 </td><td>4350 </td><td>4612 </td></tr><tr><td>1001–1500 </td><td>1622 </td><td>1934 </td></tr><tr><td>1501–2000 </td><td>594 </td><td>828 </td></tr><tr><td>2001–2500 </td><td>199 </td><td>305 </td></tr><tr><td>&gt;2500 </td><td>159 </td><td>304 </td></tr><tr><td>Total </td><td>10 224 </td><td>9145 </td></tr><tr><td>Average length (bp) </td><td>786 </td><td>1021 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303800" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS028TB2" class="table-wrap-title" data-id="DSS028TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Contig and isotig sequence length distribution</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sequence length (bp) </th><th>Number of contig </th><th>Number of isotig </th></tr></thead><tbody><tr><td>100–500 </td><td>3300 </td><td>1162 </td></tr><tr><td>501–1000 </td><td>4350 </td><td>4612 </td></tr><tr><td>1001–1500 </td><td>1622 </td><td>1934 </td></tr><tr><td>1501–2000 </td><td>594 </td><td>828 </td></tr><tr><td>2001–2500 </td><td>199 </td><td>305 </td></tr><tr><td>&gt;2500 </td><td>159 </td><td>304 </td></tr><tr><td>Total </td><td>10 224 </td><td>9145 </td></tr><tr><td>Average length (bp) </td><td>786 </td><td>1021 </td></tr></tbody></table></div><div content-id="DSS028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sequence length (bp) </th><th>Number of contig </th><th>Number of isotig </th></tr></thead><tbody><tr><td>100–500 </td><td>3300 </td><td>1162 </td></tr><tr><td>501–1000 </td><td>4350 </td><td>4612 </td></tr><tr><td>1001–1500 </td><td>1622 </td><td>1934 </td></tr><tr><td>1501–2000 </td><td>594 </td><td>828 </td></tr><tr><td>2001–2500 </td><td>199 </td><td>305 </td></tr><tr><td>&gt;2500 </td><td>159 </td><td>304 </td></tr><tr><td>Total </td><td>10 224 </td><td>9145 </td></tr><tr><td>Average length (bp) </td><td>786 </td><td>1021 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303800" target="_blank">View Large</a></div></div><p>For functional gene annotation, isotigs were reassembled to remove redundancy and to obtain longer transcripts using CAP3 with default parameter settings (90% identity and overlap of 40 bp)<sup><a reveal-id="DSS028C38" data-open="DSS028C38" class="link link-ref link-reveal xref-bibr">38</a></sup> according to Ewen-Campen <em>et al</em>.<sup><a reveal-id="DSS028C39" data-open="DSS028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Overall, 8895 assembled sequences were produced, and sequence similarity searches were conducted against the NCBI nr, rice, <em>Brachypodium</em>, and wheat protein databases using the BlastX algorithm. Of the query sequences, 90% had BlastX hits in the NCBI nr protein database. Similarly, 89 and 87% of the query sequences showed homology to annotated <em>Brachypodium</em> proteins and annotated rice proteins, respectively. On the other hand, 68% of the contig sequences had homology to wheat proteins, which might be due to the lower number of proteins deposited in the wheat database (8590 proteins) than the <em>Brachypodium</em> (31 029 proteins) and rice (40 353 proteins) databases.</p><p>To see whether the sequenced transcripts are functionally diverse or not, GO annotation was performed based on the BlastX hits against the NCBI nr database. In total, 7740 sequences were assigned to one or more GO annotations. Of the assigned GO terms, 13 260 were under the biological process domain, 10 280 under molecular function, and 9750 under cellular component (Fig. <a reveal-id="DSS028F1" data-open="DSS028F1" class="link link-reveal link-table xref-fig">1</a>). A large functional diversity of genes was found in the transcriptomic data. Because the total RNA for the transcriptome analysis was extracted from leaves, a large proportion of the assignments fell into the plastids category of the cellular component subontology. </p>                        <a id="7303803" scrollto-destination="7303803"></a>
<div id="DSS028F1" data-id="DSS028F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02801.jpeg?Expires=2147483647&Signature=jxgdoEQfCVdoSJouJyi0RAM7n4rTLNQhm9mXcubwFkiinu5Dh1dK3VOwq6Y~tb3TN27VBOwIZmq5IyzTEJNrBqC9B50FITv-pcoq6s1W3NhtSDirlFdE91~q9GuBYB4xxvvEjpFObCgVPF-CApFscg7skBVmI0qprZIXb1TP8thpYR687oNm~KLeqZX-u2WKjyevTtyBf8~oATe7yw3FfkVEMN176Rsw282Pyp4AM3CI3ej4r3~mi4K82gOmUkVZSr9U8DGWzkX2WAeZP38KMPpH8LtnMTHtTo4XOtKiPOn2D7MwvLdp~EjHiPWd5Ghmn9wODOFbyaTqbargxXKUtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO classification of assembled sequences. The results are summarized into three main subontologies: biological process, molecular function, and cellular component. Based on BlastX results against the NCBI nr protein database, 7740 sequences were assigned to one or more GO annotations." path-from-xml="dss02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303803/dss02801.jpeg" path-from-xml="dss02801.jpeg" target="_blank">View large</a><a section="7303803" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02801.jpeg?Expires=2147483647&Signature=2qeVfelYwt7Aozy6aaQRszMCSrnFPc9Fi3SVbdVrWDdkiHsROnBpp9QU7~igT-m~ydeuye2U4qbkIKjzFdguTxK5Lpc0K0NxgQMNjYSb7oARtQBSuTny5~Lt~e7mqtnhfbWp2WESclOSO3q2tfjpX8z5ogas3uixq9uf-xjSg~5EWiglLgZHwFv0XGTLxqbg2gjHfax~aIh3QWdXAOIFD7TH8l2TciWI017Ivi7MRrwhqDeu57bhmq9FcpkaiHAsGLaZDPAVXkWOCJgy7TujW5cgHzeO9Mqw~GtWtZR11PfO9-kIrkLD-RaVz8JfyV2Tw9aml1zsFTKux~CBfUlVSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303803&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO classification of assembled sequences. The results are summarized into three main subontologies: biological process, molecular function, and cellular component. Based on BlastX results against the NCBI nr protein database, 7740 sequences were assigned to one or more GO annotations.</p></div></div><div content-id="DSS028F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02801.jpeg?Expires=2147483647&Signature=jxgdoEQfCVdoSJouJyi0RAM7n4rTLNQhm9mXcubwFkiinu5Dh1dK3VOwq6Y~tb3TN27VBOwIZmq5IyzTEJNrBqC9B50FITv-pcoq6s1W3NhtSDirlFdE91~q9GuBYB4xxvvEjpFObCgVPF-CApFscg7skBVmI0qprZIXb1TP8thpYR687oNm~KLeqZX-u2WKjyevTtyBf8~oATe7yw3FfkVEMN176Rsw282Pyp4AM3CI3ej4r3~mi4K82gOmUkVZSr9U8DGWzkX2WAeZP38KMPpH8LtnMTHtTo4XOtKiPOn2D7MwvLdp~EjHiPWd5Ghmn9wODOFbyaTqbargxXKUtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO classification of assembled sequences. The results are summarized into three main subontologies: biological process, molecular function, and cellular component. Based on BlastX results against the NCBI nr protein database, 7740 sequences were assigned to one or more GO annotations." path-from-xml="dss02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303803/dss02801.jpeg" path-from-xml="dss02801.jpeg" target="_blank">View large</a><a section="7303803" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02801.jpeg?Expires=2147483647&Signature=2qeVfelYwt7Aozy6aaQRszMCSrnFPc9Fi3SVbdVrWDdkiHsROnBpp9QU7~igT-m~ydeuye2U4qbkIKjzFdguTxK5Lpc0K0NxgQMNjYSb7oARtQBSuTny5~Lt~e7mqtnhfbWp2WESclOSO3q2tfjpX8z5ogas3uixq9uf-xjSg~5EWiglLgZHwFv0XGTLxqbg2gjHfax~aIh3QWdXAOIFD7TH8l2TciWI017Ivi7MRrwhqDeu57bhmq9FcpkaiHAsGLaZDPAVXkWOCJgy7TujW5cgHzeO9Mqw~GtWtZR11PfO9-kIrkLD-RaVz8JfyV2Tw9aml1zsFTKux~CBfUlVSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303803&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO classification of assembled sequences. The results are summarized into three main subontologies: biological process, molecular function, and cellular component. Based on BlastX results against the NCBI nr protein database, 7740 sequences were assigned to one or more GO annotations.</p></div></div>                            <h3 scrollto-destination=7303804 id="7303804" class="section-title" data-legacy-id=s3b>3.2. SNP and indel discovery</h3>
<p>Polymorphic sites were discovered using GS Reference Mapper to align individual reads from both accessions against the reference contig sequences. Among the 10 351 total polymorphic sites, 9435 were SNPs and 739 were indels (Table <a reveal-id="DSS028TB3" data-open="DSS028TB3" class="link link-reveal link-table xref-fig">3</a>). Furthermore, according to the read depth (≥3 non-duplicate reads) and quality (QV ≥ 20), 4578 polymorphic sites were classified as HC polymorphisms, including 4337 SNPs (in 1748 contigs), 198 indels (in 153 contigs), and 44 variants with two or more nucleotide changes. The HC SNPs data are available online as <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental data</a> (Supplemental Data1.fna). HC SNP frequency was one SNP per 1854 bp, and when considering all the discovered SNPs the frequency, it was 852 bp per SNP (Table <a reveal-id="DSS028TB3" data-open="DSS028TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="7303806" scrollto-destination="7303806"></a>
<div content-id="DSS028TB3" class="table-modal"><div class="table-wrap"><div id="DSS028TB3" class="table-wrap-title" data-id="DSS028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Polymorphisms detected between two <em>Ae. tauschii</em> accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="2">Number of polymorphic sites<hr /> </th></tr><tr><th>Total polymorphic sites </th><th>HC polymorphic sites </th></tr></thead><tbody><tr><td>Total </td><td>10 351 (3578 contigs) </td><td>4578 (1793 contigs) </td></tr><tr><td>SNPs </td><td>9435 (3444 contigs) </td><td>4337 (1748 contigs) </td></tr><tr><td>Indel </td><td>739 (539 contigs) </td><td>198 (153 contigs) </td></tr><tr><td>Involving two or more nucleotides </td><td>112 (100 contigs) </td><td>44 (42 contigs) </td></tr><tr><td>Average bp per polymorphic site<sup>a</sup> </td><td>777 </td><td>1756 </td></tr><tr><td>Average bp per SNP<sup>a</sup> </td><td>852 </td><td>1854 </td></tr></tbody></table></div><div content-id="DSS028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="2">Number of polymorphic sites<hr /> </th></tr><tr><th>Total polymorphic sites </th><th>HC polymorphic sites </th></tr></thead><tbody><tr><td>Total </td><td>10 351 (3578 contigs) </td><td>4578 (1793 contigs) </td></tr><tr><td>SNPs </td><td>9435 (3444 contigs) </td><td>4337 (1748 contigs) </td></tr><tr><td>Indel </td><td>739 (539 contigs) </td><td>198 (153 contigs) </td></tr><tr><td>Involving two or more nucleotides </td><td>112 (100 contigs) </td><td>44 (42 contigs) </td></tr><tr><td>Average bp per polymorphic site<sup>a</sup> </td><td>777 </td><td>1756 </td></tr><tr><td>Average bp per SNP<sup>a</sup> </td><td>852 </td><td>1854 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Calculated by dividing the total number of (HC) polymorphic sites or SNPs by the total length of contigs (8 039 509 bp).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303806" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS028TB3" class="table-wrap-title" data-id="DSS028TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Polymorphisms detected between two <em>Ae. tauschii</em> accessions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="2">Number of polymorphic sites<hr /> </th></tr><tr><th>Total polymorphic sites </th><th>HC polymorphic sites </th></tr></thead><tbody><tr><td>Total </td><td>10 351 (3578 contigs) </td><td>4578 (1793 contigs) </td></tr><tr><td>SNPs </td><td>9435 (3444 contigs) </td><td>4337 (1748 contigs) </td></tr><tr><td>Indel </td><td>739 (539 contigs) </td><td>198 (153 contigs) </td></tr><tr><td>Involving two or more nucleotides </td><td>112 (100 contigs) </td><td>44 (42 contigs) </td></tr><tr><td>Average bp per polymorphic site<sup>a</sup> </td><td>777 </td><td>1756 </td></tr><tr><td>Average bp per SNP<sup>a</sup> </td><td>852 </td><td>1854 </td></tr></tbody></table></div><div content-id="DSS028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th colspan="2">Number of polymorphic sites<hr /> </th></tr><tr><th>Total polymorphic sites </th><th>HC polymorphic sites </th></tr></thead><tbody><tr><td>Total </td><td>10 351 (3578 contigs) </td><td>4578 (1793 contigs) </td></tr><tr><td>SNPs </td><td>9435 (3444 contigs) </td><td>4337 (1748 contigs) </td></tr><tr><td>Indel </td><td>739 (539 contigs) </td><td>198 (153 contigs) </td></tr><tr><td>Involving two or more nucleotides </td><td>112 (100 contigs) </td><td>44 (42 contigs) </td></tr><tr><td>Average bp per polymorphic site<sup>a</sup> </td><td>777 </td><td>1756 </td></tr><tr><td>Average bp per SNP<sup>a</sup> </td><td>852 </td><td>1854 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Calculated by dividing the total number of (HC) polymorphic sites or SNPs by the total length of contigs (8 039 509 bp).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303806" target="_blank">View Large</a></div></div><p>To study how many HC SNPs are also present in the polymorphism dataset reported by You <em>et al</em>.,<sup><a reveal-id="DSS028C25" data-open="DSS028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> a BlastN search was performed using as query 4337 100-bp sequences, in which each of the HC SNPs were positioned in the middle. Hits were obtained for all the 4337 sequences with an <em>E</em>-value cutoff of 10, of which 546 SNPs were common to our dataset (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). With an <em>E</em>-value cutoff of 10<sup>−5</sup>, 1943 hits were obtained, but the number of common SNPs between the two datasets did not differ greatly (538 SNPs). These results indicate that the majority of HC SNPs (around 3800) are only present in our dataset. Because the genome of <em>Ae. tauschii</em> is composed of large repetitive regions, only 44% of the total genomic read length has been available for SNP discovery.<sup><a reveal-id="DSS028C25" data-open="DSS028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Our results indicate that SNP discovery by RNA-seq is cost-effective in organisms with complex genomes, as reported in maize.<sup><a reveal-id="DSS028C40" data-open="DSS028C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>Although most contigs with HC polymorphisms contained one or two HC SNPs, 41 contigs had a large number of SNPs, ranging from 11 to 44 (Fig. <a reveal-id="DSS028F2" data-open="DSS028F2" class="link link-reveal link-table xref-fig">2</a>A). The number of SNPs per contig was not completely dependent on sequence length, indicated by the low coefficient of determination (<em>R<sup>2</sup></em> = 0.05) (Fig. <a reveal-id="DSS028F2" data-open="DSS028F2" class="link link-reveal link-table xref-fig">2</a>B). To study whether these nucleotide variations affect amino acid sequences, the longest ORFs from isotigs that include contigs with HC polymorphisms were extracted. The longest ORFs supported by BlastX search were found in 1538 out of the 2214 isotigs. Synonymous and non-synonymous mutations were analysed on contig sequences. In about 1100 out of the 1379 selected contigs, nucleotide variations were found mainly within the ORF, whereas the number of SNPs was similar between the ORF and untranslated region (UTR) in around 80 contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Although the number of SNPs was higher in ORFs than in UTRs, synonymous mutations predominated in most cases. Non-synonymous mutations outnumbered synonymous ones in 370 contigs. Nonsense mutations were observed in four contigs. However, we cannot discharge the possibility that the reads of paralogous genes are included in these highly polymorphic contigs. </p>                        <a id="7303809" scrollto-destination="7303809"></a>
<div id="DSS028F2" data-id="DSS028F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02802.jpeg?Expires=2147483647&Signature=cEae3KJnnst04cmss3PgCCacazNmL8gJAMvvmFp4B1X6rkeuW4BlqFuGTZeFYhj9apCnCkZkP0HHoaL06UHqrrrFcGgpYf1jnp-mshaVLS9cnxl0hRSpTWrF3v7yafKGIJ2V5hzYhjBlrKEQrm0zVRh6n-KOR6VD7Sdw7UGtkPEqXDYL0aG4t5hdPKI0ErDaURmX4QhHw3LFl0kmgbDfEFrMQYqnPQNwkI4c98FrDQb~JaK-4GpeEplXjN2Q3RpKXzySBHTB9p-XqcHUgtLGt~jC4drwBvFhMLuV55HZFZIVzf7cceLQ3py3DJafV3o0ZVE8pi2~dFvbAvZRaCeBkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of SNPs and indels in the contigs. (A) Frequency distribution of HC SNPs per contig (n = 1748 contigs). (B) Relationship between the number of SNPs per contig and the contig length; data are from 1748 contigs containing SNPs. Significant correlation is indicated by asterisks (***P &lt; 0.001). (C) Frequency distribution of HC indel length in base pairs per contig in a total of 153 contigs." path-from-xml="dss02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303809/dss02802.jpeg" path-from-xml="dss02802.jpeg" target="_blank">View large</a><a section="7303809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02802.jpeg?Expires=2147483647&Signature=Y2rG9i7BzWY9Y8rUPH9QN~ohl6HLueuqNNNE7E0Ecdv1HMupVypchYrcx0vV0gh60hRo9AqsXKNAoPo0v353ucvoTjoceFaaO2w89NTRrGNeIIsElQdo2fbqfLwyJGwuAvJwrZWR5BC9kf8dRllUofqS~8cwNo-6uyyRA82HFCnB4BINZ~pnGbSKRUNASB1aSDjCQJ23tNWvD0Y304xMT3VuLh4I5WSKMZgRMDMbc~OoUF16BevwCh~YGinQESBTVSKMLOS-6n9rqc1D6qthpH0aKJEBymezA5vAYzc-YWySOjmdDeUDIAEhgpDnE~z5yQNupJaFI-rN85Bixc9tZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303809&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of SNPs and indels in the contigs. (A) Frequency distribution of HC SNPs per contig (<em>n</em> = 1748 contigs). (B) Relationship between the number of SNPs per contig and the contig length; data are from 1748 contigs containing SNPs. Significant correlation is indicated by asterisks (***<em>P</em> &lt; 0.001). (C) Frequency distribution of HC indel length in base pairs per contig in a total of 153 contigs.</p></div></div><div content-id="DSS028F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02802.jpeg?Expires=2147483647&Signature=cEae3KJnnst04cmss3PgCCacazNmL8gJAMvvmFp4B1X6rkeuW4BlqFuGTZeFYhj9apCnCkZkP0HHoaL06UHqrrrFcGgpYf1jnp-mshaVLS9cnxl0hRSpTWrF3v7yafKGIJ2V5hzYhjBlrKEQrm0zVRh6n-KOR6VD7Sdw7UGtkPEqXDYL0aG4t5hdPKI0ErDaURmX4QhHw3LFl0kmgbDfEFrMQYqnPQNwkI4c98FrDQb~JaK-4GpeEplXjN2Q3RpKXzySBHTB9p-XqcHUgtLGt~jC4drwBvFhMLuV55HZFZIVzf7cceLQ3py3DJafV3o0ZVE8pi2~dFvbAvZRaCeBkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of SNPs and indels in the contigs. (A) Frequency distribution of HC SNPs per contig (n = 1748 contigs). (B) Relationship between the number of SNPs per contig and the contig length; data are from 1748 contigs containing SNPs. Significant correlation is indicated by asterisks (***P &lt; 0.001). (C) Frequency distribution of HC indel length in base pairs per contig in a total of 153 contigs." path-from-xml="dss02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303809/dss02802.jpeg" path-from-xml="dss02802.jpeg" target="_blank">View large</a><a section="7303809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02802.jpeg?Expires=2147483647&Signature=Y2rG9i7BzWY9Y8rUPH9QN~ohl6HLueuqNNNE7E0Ecdv1HMupVypchYrcx0vV0gh60hRo9AqsXKNAoPo0v353ucvoTjoceFaaO2w89NTRrGNeIIsElQdo2fbqfLwyJGwuAvJwrZWR5BC9kf8dRllUofqS~8cwNo-6uyyRA82HFCnB4BINZ~pnGbSKRUNASB1aSDjCQJ23tNWvD0Y304xMT3VuLh4I5WSKMZgRMDMbc~OoUF16BevwCh~YGinQESBTVSKMLOS-6n9rqc1D6qthpH0aKJEBymezA5vAYzc-YWySOjmdDeUDIAEhgpDnE~z5yQNupJaFI-rN85Bixc9tZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303809&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of SNPs and indels in the contigs. (A) Frequency distribution of HC SNPs per contig (<em>n</em> = 1748 contigs). (B) Relationship between the number of SNPs per contig and the contig length; data are from 1748 contigs containing SNPs. Significant correlation is indicated by asterisks (***<em>P</em> &lt; 0.001). (C) Frequency distribution of HC indel length in base pairs per contig in a total of 153 contigs.</p></div></div><p>The predominant length of indels was 3 bp, followed by 2, and 1 bp (Fig. <a reveal-id="DSS028F2" data-open="DSS028F2" class="link link-reveal link-table xref-fig">2</a>C). The longest indel mutation was 11 bp in contig04102 and contig05280. Indels were found in 90 out of the 1379 analysed contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>), of which 45 contigs included indel mutations within ORF, and frame-shift mutations occurred in 16 contigs. These results indicate that, despite the high number of polymorphic sites, the effect on protein sequence is not necessarily high.</p>                            <h3 scrollto-destination=7303811 id="7303811" class="section-title" data-legacy-id=s3c>3.3. SNP validation</h3>
<p>To validate the detected SNPs, 33 polymorphic contigs were randomly selected. Of the 33 contigs, 30 contained HC SNPs and the remaining 3 contained lower confidence (LC) SNPs. The validation was performed by genotyping 20 diverse <em>Ae. tauschii</em> accessions, chosen based on the intraspecific diversification patterns of <em>Ae. tauschii</em>.<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The selected accessions consisted of two accessions from each of the nine <em>Ae. tauschii</em> sublineages (six of L1 and three of L2) and HGL17, excluding any admixtures (Fig. <a reveal-id="DSS028F3" data-open="DSS028F3" class="link link-reveal link-table xref-fig">3</a>A). </p>                        <a id="7303813" scrollto-destination="7303813"></a>
<div id="DSS028F3" data-id="DSS028F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02803.jpeg?Expires=2147483647&Signature=pl8uE78gca41FXsLZPzWdWC8x50EdsclbIVqlsSfDPO75jm~JWJw9EEodUGq76zTCyiw88mOfTWRp-TbFeA~sl04247ZyxxK8QoQywjYSmz-ToaFjSPlQc2Z4PSge2yRmz0L1QOXYjEgBhzOR~4m661zL~fmNp8e4No~5P8qpgdZtCPJ7WV3E~ceSiMEDn4fQ9Qf9uGLISmQAx4XFnAWUB~D~niF3R7HGKh3n1b~CYDpGRFx4bS9ekAagWt5ayf7pYQ5GSVwkt9RLwnMhzg3n5-sET0x4-cnQtgjBe6SLNPv60IEDlwxr-AvHaILWiIuO27z9Fp67xgDqJKQxUuRVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of identified polymorphisms in the contigs. (A) Summary of genotyping results of the 31 polymorphic SNP markers in the 20 Ae. tauschii accessions. Genotypes of the two accessions used in transcriptome sequencing, PI476874 and IG47182, are shown in the first and last columns, respectively. White, black, and shaded regions indicate the PI476874, IG47182, and other alleles, respectively. (B) Frequency distribution of PIC values. PIC values of SNP markers (open bars) and SSR markers (black bars) among the 20 Ae. tauschii accessions. A total of 31 SNP and 6 SSR markers were used." path-from-xml="dss02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303813/dss02803.jpeg" path-from-xml="dss02803.jpeg" target="_blank">View large</a><a section="7303813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02803.jpeg?Expires=2147483647&Signature=x1pgrnk~lDjiGJtHrSs3aQVqN5P~x0Gb8CMlN~zmQXgx-5jdYr3Gj9i9Lf36E4qo28gAtGD2W-nKU0~G6zIw3N2G1O57ixq-sqGGOgTr3QlPz3b-pxOUTnpaJ6jtAEj3avls1YxGS3RKQ63WjbbkxkhJvIpAKBgOB3zG5I8RNqKfD~wqDnNRt-xak-GALku6J46WNorJV6xuja2Jbuz6Fi5Kzo08NjQIEteW7Mvj7glm9oh0In1BsaKIseuYCfH22rg8CZElDlEWoe9o~3ycWuR8wUlmf1Nv~0ivwdSEMHWc0b7cp~6A1M7xhYsv95h-i7oDSN18N-PnpxJZOpAQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303813&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of identified polymorphisms in the contigs. (A) Summary of genotyping results of the 31 polymorphic SNP markers in the 20 <em>Ae. tauschii</em> accessions. Genotypes of the two accessions used in transcriptome sequencing, PI476874 and IG47182, are shown in the first and last columns, respectively. White, black, and shaded regions indicate the PI476874, IG47182, and other alleles, respectively. (B) Frequency distribution of PIC values. PIC values of SNP markers (open bars) and SSR markers (black bars) among the 20 <em>Ae. tauschii</em> accessions. A total of 31 SNP and 6 SSR markers were used.</p></div></div><div content-id="DSS028F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02803.jpeg?Expires=2147483647&Signature=pl8uE78gca41FXsLZPzWdWC8x50EdsclbIVqlsSfDPO75jm~JWJw9EEodUGq76zTCyiw88mOfTWRp-TbFeA~sl04247ZyxxK8QoQywjYSmz-ToaFjSPlQc2Z4PSge2yRmz0L1QOXYjEgBhzOR~4m661zL~fmNp8e4No~5P8qpgdZtCPJ7WV3E~ceSiMEDn4fQ9Qf9uGLISmQAx4XFnAWUB~D~niF3R7HGKh3n1b~CYDpGRFx4bS9ekAagWt5ayf7pYQ5GSVwkt9RLwnMhzg3n5-sET0x4-cnQtgjBe6SLNPv60IEDlwxr-AvHaILWiIuO27z9Fp67xgDqJKQxUuRVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of identified polymorphisms in the contigs. (A) Summary of genotyping results of the 31 polymorphic SNP markers in the 20 Ae. tauschii accessions. Genotypes of the two accessions used in transcriptome sequencing, PI476874 and IG47182, are shown in the first and last columns, respectively. White, black, and shaded regions indicate the PI476874, IG47182, and other alleles, respectively. (B) Frequency distribution of PIC values. PIC values of SNP markers (open bars) and SSR markers (black bars) among the 20 Ae. tauschii accessions. A total of 31 SNP and 6 SSR markers were used." path-from-xml="dss02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303813/dss02803.jpeg" path-from-xml="dss02803.jpeg" target="_blank">View large</a><a section="7303813" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02803.jpeg?Expires=2147483647&Signature=x1pgrnk~lDjiGJtHrSs3aQVqN5P~x0Gb8CMlN~zmQXgx-5jdYr3Gj9i9Lf36E4qo28gAtGD2W-nKU0~G6zIw3N2G1O57ixq-sqGGOgTr3QlPz3b-pxOUTnpaJ6jtAEj3avls1YxGS3RKQ63WjbbkxkhJvIpAKBgOB3zG5I8RNqKfD~wqDnNRt-xak-GALku6J46WNorJV6xuja2Jbuz6Fi5Kzo08NjQIEteW7Mvj7glm9oh0In1BsaKIseuYCfH22rg8CZElDlEWoe9o~3ycWuR8wUlmf1Nv~0ivwdSEMHWc0b7cp~6A1M7xhYsv95h-i7oDSN18N-PnpxJZOpAQ0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303813&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of identified polymorphisms in the contigs. (A) Summary of genotyping results of the 31 polymorphic SNP markers in the 20 <em>Ae. tauschii</em> accessions. Genotypes of the two accessions used in transcriptome sequencing, PI476874 and IG47182, are shown in the first and last columns, respectively. White, black, and shaded regions indicate the PI476874, IG47182, and other alleles, respectively. (B) Frequency distribution of PIC values. PIC values of SNP markers (open bars) and SSR markers (black bars) among the 20 <em>Ae. tauschii</em> accessions. A total of 31 SNP and 6 SSR markers were used.</p></div></div><p>First, all 33 contigs were searched for restriction enzyme recognition sites, with at least 14 SNPs found and converted into CAPS markers. All 14 CAPS markers were polymorphic among the examined accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>A). An alternative to a CAPS marker is a dCAPS marker that consists of the introduction of one or more mismatches into the primer, creating a restriction site dependent on the presence or absence of the SNP.<sup><a reveal-id="DSS028C41" data-open="DSS028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Five of the SNPs that were not converted into CAPS markers were used to develop dCAPS markers by introducing one or two mismatches in the primer sequence. Two dCAPS markers could not be used for genotyping because unexpected bands were amplified by PCR, and a dCAPS marker for contig05220 was unavailable due to the generation of digestion-resistant PCR products in some accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>B). In dCAPS markers, a single mismatch at position 1, 2, or 3 from the 3′ end of the primer is preferred to avoid digestion-resistant products.<sup><a reveal-id="DSS028C42" data-open="DSS028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> However, in some cases, a mismatch at position 1 or 2 may also lead to undigested products, if the polymerase has proofreading exonuclease activity.<sup><a reveal-id="DSS028C43" data-open="DSS028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The digestion-resistant products were generated for unknown reasons because in the dCAPS analysis of the contig05220 marker, a single mismatch was introduced at position 1 and amplification was performed using a <em>Taq</em> DNA polymerase lacking proofreading activity. The remaining two dCAPS markers were successfully used in genotyping (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>C).</p><p>SNP genotyping based on HRM analysis has also been reported in hexaploid wheat and <em>Ae. tauschii</em>.<sup><a reveal-id="DSS028C32" data-open="DSS028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Primers for HRM analysis were designed for the remaining 17 contigs (including the three contigs that failed in the dCAPS analysis) to obtain PCR products of ∼100 bp. Only two of the 17 HRM primer sets failed in polymorphism detection, although both amplified genomic regions contained HC SNPs between PI476874 and IG47182. HRM analysis is useful as an alternative to SNP-typing because it is cost-effective when the restriction enzyme in CAPS/dCAPS markers is expensive. Another advantage of HRM is the possibility of detecting more than two alleles (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), which is not possible with CAPS/dCAPS. In addition, the relatively short length of the PCR products required for HRM analyses increases the probability that primer pairs are located within a single exon because more than half of the exons in plant genomes are less than 145–156 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Our results indicate that almost all of these SNPs can be converted to PCR-based molecular markers, at least in diploid wheat species.</p><p>To evaluate the allelic diversity within the <em>Ae. tauschii</em> population, PIC values were calculated for each validated SNP marker. PIC values ranged from 0.091 to 0.591 (average, 0.313) in the 20 <em>Ae. tauschii</em> accessions (Fig. <a reveal-id="DSS028F3" data-open="DSS028F3" class="link link-reveal link-table xref-fig">3</a>B). Among the 31 markers, three showed IG47182-specific SNP alleles, and 15 clearly distinguished the L1 accessions from L2 (Fig. <a reveal-id="DSS028F3" data-open="DSS028F3" class="link link-reveal link-table xref-fig">3</a>A). Because many of the validated SNPs were polymorphic between L1 and L2 accessions, the markers developed using information about these SNPs can be used in genotyping of mapping populations obtained from interlineage crosses (L1 × L2). Moreover, considering that the subspecies <em>strangulata</em> is a possible major D-genome donor and the evolutionary birthplace of hexaploid common wheat, the D-genome donor has been thought to belong to L2,<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> indicating that these markers might be useful for genotyping the progeny of common wheat cultivars crossed with L1-derived hexaploid wheat synthetics. Further studies are needed to confirm this hypothesis. The HGL17 accessions, considered one of the ancestral lineages of <em>Ae. tauschii</em>, are not included in either L1 or L2, but rather are located genealogically between these two lineages.<sup><a reveal-id="DSS028C12" data-open="DSS028C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Consistently, the genotype patterns of the two HGL17 accessions analysed (AE454 and AE929) were intermediates between L1 and L2 accessions.</p>                            <h3 scrollto-destination=7303817 id="7303817" class="section-title" data-legacy-id=s3d>3.4. EST-SSR mining and validation</h3>
<p>Among all 10 224 contigs, a total of 2778 SSR motifs were found in 2072 contigs and were composed mainly of trinucleotide motifs (Table <a reveal-id="DSS028TB4" data-open="DSS028TB4" class="link link-reveal link-table xref-fig">4</a>). Indel polymorphisms between PI476874 and IG47182 were searched as polymorphic SSRs in the 2778 motifs. In total, 54 polymorphic SSRs were found in 50 contigs, with 20 of them HC SSRs. Based on these data, 18 EST-SSR markers, including 14 HC and 4 LC polymorphic SSRs, were developed for validation in the 20 <em>Ae. tauschii</em> accessions. Seven EST-SSR markers generated multiple bands or amplified products with a length much longer than expected (probably due to the presence of introns) and were excluded from further analyses. Four out of the eight EST-SSR markers with more than a 4-bp difference were able to distinguish genotypes of samples in 13% non-denaturing polyacrylamide gels. HRM analysis was conducted for the other EST-SSR markers, including indels with less than a 5-bp difference, and this detected polymorphisms in only two of the seven EST-SSR markers. PIC values of these six markers ranged from 0.305 to 0.686, with an average of 0.424 (Fig. <a reveal-id="DSS028F3" data-open="DSS028F3" class="link link-reveal link-table xref-fig">3</a>B). The number of alleles in the EST-SSR markers ranged from two to six (average, 3.0), which was higher when compared with that in SNP markers with two to three alleles (average, 2.1). This difference might reflect the higher average PIC value of the EST-SSR markers. </p>                        <a id="7303819" scrollto-destination="7303819"></a>
<div content-id="DSS028TB4" class="table-modal"><div class="table-wrap"><div id="DSS028TB4" class="table-wrap-title" data-id="DSS028TB4"><div class="label -label">Table 4.</div><div class="caption"><p>SSR motifs found in 10 224 contigs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of motifs </th></tr></thead><tbody><tr><td>Total motifs </td><td>2778 (2072 contigs) </td></tr><tr><td>Dinucleotide motifs </td><td>201 </td></tr><tr><td>Trinucleotide motifs </td><td>1896 </td></tr><tr><td>Tetranucleotide motifs </td><td>380 </td></tr><tr><td>Pentanucleotide motifs </td><td>301 </td></tr><tr><td>Polymorphic SSRs </td><td>54 (50 contigs) </td></tr><tr><td>HC polymorphic SSRs </td><td>20 (19 contigs) </td></tr></tbody></table></div><div content-id="DSS028TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of motifs </th></tr></thead><tbody><tr><td>Total motifs </td><td>2778 (2072 contigs) </td></tr><tr><td>Dinucleotide motifs </td><td>201 </td></tr><tr><td>Trinucleotide motifs </td><td>1896 </td></tr><tr><td>Tetranucleotide motifs </td><td>380 </td></tr><tr><td>Pentanucleotide motifs </td><td>301 </td></tr><tr><td>Polymorphic SSRs </td><td>54 (50 contigs) </td></tr><tr><td>HC polymorphic SSRs </td><td>20 (19 contigs) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303819" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS028TB4" class="table-wrap-title" data-id="DSS028TB4"><div class="label -label">Table 4.</div><div class="caption"><p>SSR motifs found in 10 224 contigs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of motifs </th></tr></thead><tbody><tr><td>Total motifs </td><td>2778 (2072 contigs) </td></tr><tr><td>Dinucleotide motifs </td><td>201 </td></tr><tr><td>Trinucleotide motifs </td><td>1896 </td></tr><tr><td>Tetranucleotide motifs </td><td>380 </td></tr><tr><td>Pentanucleotide motifs </td><td>301 </td></tr><tr><td>Polymorphic SSRs </td><td>54 (50 contigs) </td></tr><tr><td>HC polymorphic SSRs </td><td>20 (19 contigs) </td></tr></tbody></table></div><div content-id="DSS028TB4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of motifs </th></tr></thead><tbody><tr><td>Total motifs </td><td>2778 (2072 contigs) </td></tr><tr><td>Dinucleotide motifs </td><td>201 </td></tr><tr><td>Trinucleotide motifs </td><td>1896 </td></tr><tr><td>Tetranucleotide motifs </td><td>380 </td></tr><tr><td>Pentanucleotide motifs </td><td>301 </td></tr><tr><td>Polymorphic SSRs </td><td>54 (50 contigs) </td></tr><tr><td>HC polymorphic SSRs </td><td>20 (19 contigs) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303819" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7303820 id="7303820" class="section-title" data-legacy-id=s3e>3.5. Genetic map construction</h3>
<p>We developed 37 polymorphic markers, including 31 SNPs and 6 EST-SSRs. To study the proportion of these markers that can be assigned to a genetic map, genotypes of 104 F<sub>2</sub> individuals obtained from a cross between PI476874 and IG47182 were determined. All these markers were successfully mapped onto a linkage map constructed using 87 publicly available SSR markers.<sup><a reveal-id="DSS028C26" data-open="DSS028C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS028C32" data-open="DSS028C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The total map length was 1282.1 cM, with an average interval of 9.5 cM between markers. In total, 12 markers were distorted on chromosome 1D, 24 markers on chromosome 2D, 6 markers on chromosome 4D, and 10 markers on chromosome 5D. Segregation ratios of the newly mapped markers are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>. The distribution of the 37 marker positions ranged from 1 on chromosome 1D to 12 on chromosome 5D (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Two markers, one SNP marker and an SSR marker, were derived from different sites of contig06827, and both mapped to the same location.</p><p>We performed a BlastN search of the 36 mapped contigs against the deletion-mapped ESTs of common wheat,<sup><a reveal-id="DSS028C44" data-open="DSS028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> obtaining 17 matches (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Of the 17 contigs, 14 (82%) were located on the same homoeologous group chromosomes as the deletion-mapped ESTs. Recently, a genome survey sequence of common wheat chromosome 7D was published.<sup><a reveal-id="DSS028C27" data-open="DSS028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> To compare the order of the six contigs that mapped to chromosome 7D of <em>Ae. tauschii</em>, a BlastN search was performed against the 7D Synthetic Build v2.0.<sup><a reveal-id="DSS028C27" data-open="DSS028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Four contigs were assigned to the short arm and two to the long arm of chromosome 7D, as expected based on the <em>Ae. tauschii</em> genetic map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The contig order in the survey sequence data was similar to that in the 7D linkage map, although the position of contig02996 differed slightly.</p><p>In barley, a hypothetical gene order has been proposed for each of the seven chromosomes based on conserved synteny among barley, <em>Brachypodium</em>, rice, and sorghum.<sup><a reveal-id="DSS028C28" data-open="DSS028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Due to the conserved synteny between wheat and barley,<sup><a reveal-id="DSS028C45" data-open="DSS028C45" class="link link-ref link-reveal xref-bibr">45</a></sup> the contigs with HC SNPs were mapped on the virtual barley chromosomes, aligning with the shotgun barley genomic sequences.<sup><a reveal-id="DSS028C28" data-open="DSS028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A total of 1450 contigs (<em>E</em>-value cutoff of 10<sup>−20</sup>) were mapped with a distribution of around 200 contigs per chromosome (Table <a reveal-id="DSS028TB5" data-open="DSS028TB5" class="link link-reveal link-table xref-fig">5</a>). The order of contigs on the genetic map of chromosomes 2D and 5D was also compared, respectively, with the virtual barley chromosomes 2H and 5H. These two chromosomes were selected because the number of mapped contigs was higher than that for the remaining chromosomes. Of the seven contigs that mapped to chromosome 2D, one failed to be assigned to any barley chromosomes, four were assigned to chromosome 2H, and two to other chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). To confirm this result, tBlastX search was performed against 5006 and 23 614 barley full-length cDNAs,<sup><a reveal-id="DSS028C36" data-open="DSS028C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS028C37" data-open="DSS028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and BlastX search was performed against <em>Brachypodium</em> and rice protein database. Based on these Blast hits, six out of the seven contigs were located on chromosome 2H (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>), indicating that some homologous genes on the chromosome other than 2H were hit in the former analysis. Contig06577 failed to be assigned to any barley chromosome. The contig order on chromosome 2D of <em>Ae. tauschii</em> corresponded to the gene order on chromosome 2H of barley (Fig. <a reveal-id="DSS028F4" data-open="DSS028F4" class="link link-reveal link-table xref-fig">4</a>A). Similarly, 12 out of 13 contigs of <em>Ae. tauschii</em> chromosome 5D were assigned to chromosome 5H of barley based on tBlastX hits against barley shotgun genomic reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). Contig05381 was assigned to chromosome 2H, and this result was confirmed by performing a Blast search against barley cDNA and <em>Brachypodium</em> and rice protein sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). The contigs on chromosome 5D of <em>Ae. tauschii</em>, except for contig05398, showed an order identical to that on barley chromosome 5H, whereas contig05398 was located in a slightly different region (Fig. <a reveal-id="DSS028F4" data-open="DSS028F4" class="link link-reveal link-table xref-fig">4</a>B). Previously, ∼14% of barley genes could not be assigned to virtual chromosomes by the GenomeZipper approach.<sup><a reveal-id="DSS028C28" data-open="DSS028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Therefore, contig06577 that failed to be assigned to the virtual barley chromosomes might correspond to the unassigned 14% of barley genes. Indeed, this contig showed sequence similarity with <em>Brachypodium</em> and rice genes Bradi1g19250 and Os07g0656700 that are probably located on the short arm of chromosome 2H, according to the synteny with barley (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). Furthermore, contig05381 (mapped on chromosome 5DL but assigned to barley 2HS) also showed sequence similarity with Bradi4g34390 and Os09g0502200 that probably map to chromosome 5HL of barley (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). The chromosomal locations of the remaining mapped contigs were similarly confirmed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). These results suggest that the contigs should be assigned to virtual barley chromosomes not only by alignment with the shotgun genomic reads, but also by alignment with <em>Brachypodium</em> and rice protein sequences. The chromosomal locations of unassigned contigs might be searched based on the synteny among barley, <em>Brachypodium</em>, and rice. Our results indicate that information on hypothetical gene order in barley may be useful for the development of molecular markers in target chromosomal regions of wheat. </p>                        <a id="7303824" scrollto-destination="7303824"></a>
<div content-id="DSS028TB5" class="table-modal"><div class="table-wrap"><div id="DSS028TB5" class="table-wrap-title" data-id="DSS028TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Number of contigs with HC SNPs mapped on the virtual barley chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome number </th><th colspan="3"><em>E</em>-value cutoff<hr /> </th></tr><tr><th>&lt;0.1 </th><th>&lt;10<sup>−5</sup> </th><th>&lt;10<sup>−20</sup> </th></tr></thead><tbody><tr><td>1H </td><td>229 </td><td>209 </td><td>197 </td></tr><tr><td>2H </td><td>247 </td><td>235 </td><td>219 </td></tr><tr><td>3H </td><td>234 </td><td>214 </td><td>205 </td></tr><tr><td>4H </td><td>203 </td><td>187 </td><td>182 </td></tr><tr><td>5H </td><td>275 </td><td>254 </td><td>241 </td></tr><tr><td>6H </td><td>211 </td><td>199 </td><td>192 </td></tr><tr><td>7H </td><td>259 </td><td>234 </td><td>214 </td></tr><tr><td>Total </td><td>1658 </td><td>1532 </td><td>1450 </td></tr></tbody></table></div><div content-id="DSS028TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome number </th><th colspan="3"><em>E</em>-value cutoff<hr /> </th></tr><tr><th>&lt;0.1 </th><th>&lt;10<sup>−5</sup> </th><th>&lt;10<sup>−20</sup> </th></tr></thead><tbody><tr><td>1H </td><td>229 </td><td>209 </td><td>197 </td></tr><tr><td>2H </td><td>247 </td><td>235 </td><td>219 </td></tr><tr><td>3H </td><td>234 </td><td>214 </td><td>205 </td></tr><tr><td>4H </td><td>203 </td><td>187 </td><td>182 </td></tr><tr><td>5H </td><td>275 </td><td>254 </td><td>241 </td></tr><tr><td>6H </td><td>211 </td><td>199 </td><td>192 </td></tr><tr><td>7H </td><td>259 </td><td>234 </td><td>214 </td></tr><tr><td>Total </td><td>1658 </td><td>1532 </td><td>1450 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303824" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS028TB5" class="table-wrap-title" data-id="DSS028TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Number of contigs with HC SNPs mapped on the virtual barley chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome number </th><th colspan="3"><em>E</em>-value cutoff<hr /> </th></tr><tr><th>&lt;0.1 </th><th>&lt;10<sup>−5</sup> </th><th>&lt;10<sup>−20</sup> </th></tr></thead><tbody><tr><td>1H </td><td>229 </td><td>209 </td><td>197 </td></tr><tr><td>2H </td><td>247 </td><td>235 </td><td>219 </td></tr><tr><td>3H </td><td>234 </td><td>214 </td><td>205 </td></tr><tr><td>4H </td><td>203 </td><td>187 </td><td>182 </td></tr><tr><td>5H </td><td>275 </td><td>254 </td><td>241 </td></tr><tr><td>6H </td><td>211 </td><td>199 </td><td>192 </td></tr><tr><td>7H </td><td>259 </td><td>234 </td><td>214 </td></tr><tr><td>Total </td><td>1658 </td><td>1532 </td><td>1450 </td></tr></tbody></table></div><div content-id="DSS028TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chromosome number </th><th colspan="3"><em>E</em>-value cutoff<hr /> </th></tr><tr><th>&lt;0.1 </th><th>&lt;10<sup>−5</sup> </th><th>&lt;10<sup>−20</sup> </th></tr></thead><tbody><tr><td>1H </td><td>229 </td><td>209 </td><td>197 </td></tr><tr><td>2H </td><td>247 </td><td>235 </td><td>219 </td></tr><tr><td>3H </td><td>234 </td><td>214 </td><td>205 </td></tr><tr><td>4H </td><td>203 </td><td>187 </td><td>182 </td></tr><tr><td>5H </td><td>275 </td><td>254 </td><td>241 </td></tr><tr><td>6H </td><td>211 </td><td>199 </td><td>192 </td></tr><tr><td>7H </td><td>259 </td><td>234 </td><td>214 </td></tr><tr><td>Total </td><td>1658 </td><td>1532 </td><td>1450 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7303824" target="_blank">View Large</a></div></div>                        <a id="7303825" scrollto-destination="7303825"></a>
<div id="DSS028F4" data-id="DSS028F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02804.jpeg?Expires=2147483647&Signature=g2EBB7j6c7ZFbK943sALmOmwio3WDiZMkVX7d92SEeDIjlpfWdG4l0ryJPnfxS~MQxcuCqb9rveiY8EeAenPvFb7O0BCD30TiYDH5YsuYIEipYN3y9JmCTa1OYIRBDD5evYmCzV1SBWl~ft4jF3Rj445920g7ffGySeaMs8H1HjZ8aN8u04uV8p5RV-Gnd5jPqvfAsjka6SDums5l6PkMuWGmd-yaHnSfSImyuepSRybFLMBEWGZPD76XJUNhdsM7sN-ks3bB4Ez5VKUKMQDEd5nXT70QjJnYk9aR-MDyQB2w-Rgu5Upj97e7czePRQN6ShBxDR~1sSBzupl5z7vBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny between chromosomes 2D and 5D of Ae. tauschii and 2H and 5H of barley. Based on tBlastX hits against barley full-length cDNA and Brachypodium protein database, the contig orders on chromosomes 2D (A) and 5D (B) were compared with those on the barley virtual chromosome in the genome zipper. The clone names and accession numbers of barley cDNAs are indicated in parentheses." path-from-xml="dss02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303825/dss02804.jpeg" path-from-xml="dss02804.jpeg" target="_blank">View large</a><a section="7303825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02804.jpeg?Expires=2147483647&Signature=LoKNUPpxm8vYgpvb1xVtVHeUGsxq0rf2f5VGuIkhS9IgbXwQuGqBYJbTxii2rr3VMfTR-XouhJrV1KRwLM5xHApkP7VCXmll6GcZTm1WeZ3WgwMd93s0BoJjx2EcDBJR43xcnH~Bpt~EiBAKXpIK3Oyn8~rTw8OouvA5dBYM0wP9vz1E5X4adUllI6YLngJAamrxlNgdSJY03CPBzjUg0Ro3ySBsrxTj-OT5UsNVvscx4HVmGAWo1gqK76YcLI2KiSEWrmvkVNz4uNk~YX9M~zWn4S0kazxIXDp3tQgfyehDz3vIYQc~gBfEbuSKN3HbJcycR5rOO5SkvrinUxL6Gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303825&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny between chromosomes 2D and 5D of <em>Ae. tauschii</em> and 2H and 5H of barley. Based on tBlastX hits against barley full-length cDNA and <em>Brachypodium</em> protein database, the contig orders on chromosomes 2D (A) and 5D (B) were compared with those on the barley virtual chromosome in the genome zipper. The clone names and accession numbers of barley cDNAs are indicated in parentheses.</p></div></div><div content-id="DSS028F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/m_dss02804.jpeg?Expires=2147483647&Signature=g2EBB7j6c7ZFbK943sALmOmwio3WDiZMkVX7d92SEeDIjlpfWdG4l0ryJPnfxS~MQxcuCqb9rveiY8EeAenPvFb7O0BCD30TiYDH5YsuYIEipYN3y9JmCTa1OYIRBDD5evYmCzV1SBWl~ft4jF3Rj445920g7ffGySeaMs8H1HjZ8aN8u04uV8p5RV-Gnd5jPqvfAsjka6SDums5l6PkMuWGmd-yaHnSfSImyuepSRybFLMBEWGZPD76XJUNhdsM7sN-ks3bB4Ez5VKUKMQDEd5nXT70QjJnYk9aR-MDyQB2w-Rgu5Upj97e7czePRQN6ShBxDR~1sSBzupl5z7vBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny between chromosomes 2D and 5D of Ae. tauschii and 2H and 5H of barley. Based on tBlastX hits against barley full-length cDNA and Brachypodium protein database, the contig orders on chromosomes 2D (A) and 5D (B) were compared with those on the barley virtual chromosome in the genome zipper. The clone names and accession numbers of barley cDNAs are indicated in parentheses." path-from-xml="dss02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7303825/dss02804.jpeg" path-from-xml="dss02804.jpeg" target="_blank">View large</a><a section="7303825" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss02804.jpeg?Expires=2147483647&Signature=LoKNUPpxm8vYgpvb1xVtVHeUGsxq0rf2f5VGuIkhS9IgbXwQuGqBYJbTxii2rr3VMfTR-XouhJrV1KRwLM5xHApkP7VCXmll6GcZTm1WeZ3WgwMd93s0BoJjx2EcDBJR43xcnH~Bpt~EiBAKXpIK3Oyn8~rTw8OouvA5dBYM0wP9vz1E5X4adUllI6YLngJAamrxlNgdSJY03CPBzjUg0Ro3ySBsrxTj-OT5UsNVvscx4HVmGAWo1gqK76YcLI2KiSEWrmvkVNz4uNk~YX9M~zWn4S0kazxIXDp3tQgfyehDz3vIYQc~gBfEbuSKN3HbJcycR5rOO5SkvrinUxL6Gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7303825&ar=481438&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny between chromosomes 2D and 5D of <em>Ae. tauschii</em> and 2H and 5H of barley. Based on tBlastX hits against barley full-length cDNA and <em>Brachypodium</em> protein database, the contig orders on chromosomes 2D (A) and 5D (B) were compared with those on the barley virtual chromosome in the genome zipper. The clone names and accession numbers of barley cDNAs are indicated in parentheses.</p></div></div>                            <h3 scrollto-destination=7303826 id="7303826" class="section-title" data-legacy-id=s3f>3.6. Conclusion</h3>
<p>Marker development in targeted chromosomal regions is important for map-based gene cloning and for molecular breeding in wheat and other crops. In this study, transcriptomes of two <em>Ae. tauschii</em> accessions belonging to different major lineages were compared, yielding 4578 HC-polymorphic sites from 1793 contigs. Thirty-one SNPs were validated in the 20 diverse <em>Ae. tauschii</em> accessions through conversion into CAPS, dCAPS, and HRM markers. Development of CAPS markers is dependent on the availability of appropriate restriction enzymes and their cost. In some dCAPS markers, generation of digestion-resistant PCR products interferes with precise genotyping. In this study, we showed that HRM analysis is available for SNP typing. The SNP markers developed in this study could be applied to construct an F<sub>2</sub> map between L1 and L2 accessions of <em>Ae. tauschii</em>, and probably between the D genomes of common wheat and L1-derived synthetic wheat. One next generation sequencing experiment on transcriptomes of parental accessions leading to <em>de novo</em>-generated SNP markers was able to identify the chromosomal location of more than 1700 genes in the wheat D genome. RNA-seq is a more cost-effective method for SNP discovery than the annotation-based one in organisms with large and complex genomes. In addition, due to the conserved synteny between wheat and barley chromosomes, the barley high-density EST map and linear gene order can be applied to develop markers on the target chromosomal regions of wheat. Accompanied by further progress on the wheat genome project coordinated by the International Wheat Genome Sequencing Consortium (<a class="link link-uri" href="http://www.wheatgenome.org/">http://www.wheatgenome.org/</a>), more information about genomic sequence data of each chromosome will become available for wheat marker development.</p>                            <h2 scrollto-destination=7303828 id="7303828" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss028/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=7303830 id="7303830" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by grants from the Ministry of Education, Culture, Sports, Science, and Technology (MEXT) of Japan (Grant-in-Aid for Scientific Research (B) No. 21380005 and Grant-in-Aid for Challenging Exploratory Research No. 23658010) to S.T., and by MEXT as part of a Joint Research Program implemented at the Institute of Plant Science and Resources, Okayama University in Japan.</p>                            <h2 scrollto-destination=7303832 id="7303832" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors would like to thank Ms Yuka Motoi at Okayama University for technical assistance.</p>                            <h2 scrollto-destination=7303834 id="7303834" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kihara</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Discovery of the DD-analyser, one of the ancestors of <em>Triticum vulgare</em></div>, <div class="source">Agric. Hortic.</div> , <div class="year">1944</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">889</div>-<div class="lpage">90</div>) <div class="comment">[In Japanese]</div></div></div></div></div><div content-id="DSS028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McFadden</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Sears</div> <div class="given-names">E.R.</div></div>. </div><div class="article-title">The artificial synthesis of <em>Triticum spelta</em></div>, <div class="source">Rec. Genet. Soc. Am.</div> , <div class="year">1944</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">26</div>-<div class="lpage">7</div>)</div></div></div></div><div content-id="DSS028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dudnikov</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Goncharov</div> <div class="given-names">N.P.</div></div>. </div><div class="article-title">Allozyme variation in <em>Aegilops squarrosa</em></div>, <div class="source">Hereditas</div> , <div class="year">1993</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">117</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allozyme%20variation%20in%20Aegilops%20squarrosa&amp;author=A.J.%20Dudnikov&amp;author=N.P.%20Goncharov&amp;publication_year=1993&amp;journal=Hereditas&amp;volume=119&amp;pages=117-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1601-5223.1993.00117.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1601-5223.1993.00117.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1601-5223.1993.00117.x"> </span></div> </div></div></div></div></div><div content-id="DSS028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dvorak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Luo</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Z.L.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">H.B.</div></div>. </div><div class="article-title">The structure of the <em>Aegilops tauschii</em> genepool and the evolution of hexaploid wheat</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">657</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20the%20Aegilops%20tauschii%20genepool%20and%20the%20evolution%20of%20hexaploid%20wheat&amp;author=J.%20Dvorak&amp;author=M.C.%20Luo&amp;author=Z.L.%20Yang&amp;author=H.B.%20Zhang&amp;publication_year=1998&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=97&amp;pages=657-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050942" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050942" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050942"> </span></div> </div></div></div></div></div><div content-id="DSS028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dudnikov</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>. </div><div class="article-title"><em>Aegilops tauschii</em>: genetic variation in Iran</div>, <div class="source">Genet. Resour. Crop Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">579</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aegilops%20tauschii%3A%20genetic%20variation%20in%20Iran&amp;author=A.J.%20Dudnikov&amp;author=T.%20Kawahara&amp;publication_year=2006&amp;journal=Genet.%20Resour.%20Crop%20Evol.&amp;volume=53&amp;pages=579-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-004-2681-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-004-2681-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-004-2681-3"> </span></div> </div></div></div></div></div><div content-id="DSS028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Natural variation for fertile triploid F<sub>1</sub> formation in allohexaploid wheat speciation</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">509</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20for%20fertile%20triploid%20F1%20formation%20in%20allohexaploid%20wheat%20speciation&amp;author=Y.%20Matsuoka&amp;author=S.%20Takumi&amp;author=T.%20Kawahara&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=509-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0584-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0584-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0584-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17639301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Flowering time diversification and dispersal in central Eurasian wild wheat <em>Aegilops tauschii</em> Coss.: genealogical and ecological framework</div>, <div class="source">PLoS ONE</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e3138</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0003138" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0003138</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flowering%20time%20diversification%20and%20dispersal%20in%20central%20Eurasian%20wild%20wheat%20Aegilops%20tauschii%20Coss.%3A%20genealogical%20and%20ecological%20framework&amp;author=Y.%20Matsuoka&amp;author=S.%20Takumi&amp;author=T.%20Kawahara&amp;publication_year=2008&amp;journal=PLoS%20ONE&amp;volume=3&amp;pages=e3138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003138" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003138" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003138"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18769547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishioka</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genealogical analysis of subspecies divergence and spikelet-shape diversification in central Eurasian wild wheat <em>Aegilops tauschii</em> Coss.</div>, <div class="source">Plant Syst. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">233</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genealogical%20analysis%20of%20subspecies%20divergence%20and%20spikelet-shape%20diversification%20in%20central%20Eurasian%20wild%20wheat%20Aegilops%20tauschii%20Coss.&amp;author=Y.%20Matsuoka&amp;author=E.%20Nishioka&amp;author=T.%20Kawahara&amp;author=S.%20Takumi&amp;publication_year=2009&amp;journal=Plant%20Syst.%20Evol.&amp;volume=279&amp;pages=233-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00606-009-0159-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00606-009-0159-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00606-009-0159-7"> </span></div> </div></div></div></div></div><div content-id="DSS028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nishioka</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Morihiro</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Natural variation of morphological traits in wild wheat progenitor <em>Aegilops tauschii</em> Coss.</div>, <div class="source">Breed. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">579</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20variation%20of%20morphological%20traits%20in%20wild%20wheat%20progenitor%20Aegilops%20tauschii%20Coss.&amp;author=S.%20Takumi&amp;author=E.%20Nishioka&amp;author=H.%20Morihiro&amp;author=T.%20Kawahara&amp;author=Y.%20Matsuoka&amp;publication_year=2009&amp;journal=Breed.%20Sci.&amp;volume=59&amp;pages=579-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.59.579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.59.579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.59.579"> </span></div> </div></div></div></div></div><div content-id="DSS028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kajimura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Murai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Distinct genetic regulation of flowering time and grain-filling period based on empirical study of D-genome diversity in synthetic hexaploid wheat lines</div>, <div class="source">Breed. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">130</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distinct%20genetic%20regulation%20of%20flowering%20time%20and%20grain-filling%20period%20based%20on%20empirical%20study%20of%20D-genome%20diversity%20in%20synthetic%20hexaploid%20wheat%20lines&amp;author=T.%20Kajimura&amp;author=K.%20Murai&amp;author=S.%20Takumi&amp;publication_year=2011&amp;journal=Breed.%20Sci.&amp;volume=61&amp;pages=130-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.61.130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.61.130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.61.130"> </span></div> </div></div></div></div></div><div content-id="DSS028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iehisa</div> <div class="given-names">J.C.M.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Variation in abscisic acid responsiveness of <em>Aegilops tauschii</em> and hexaploid wheat synthetics due to the D-genome diversity</div>, <div class="source">Genes Genet. Syst.</div> , <div class="year">2012</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">9</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20abscisic%20acid%20responsiveness%20of%20Aegilops%20tauschii%20and%20hexaploid%20wheat%20synthetics%20due%20to%20the%20D-genome%20diversity&amp;author=J.C.M.%20Iehisa&amp;author=S.%20Takumi&amp;publication_year=2012&amp;journal=Genes%20Genet.%20Syst.&amp;volume=87&amp;pages=9-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.87.9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.87.9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.87.9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22531790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yamasaki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Population structure of wild wheat D-genome progenitor <em>Aegilops tauschii</em> Coss.: implications for intraspecific lineage diversification and evolution of common wheat</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">999</div>-<div class="lpage">1013</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20structure%20of%20wild%20wheat%20D-genome%20progenitor%20Aegilops%20tauschii%20Coss.%3A%20implications%20for%20intraspecific%20lineage%20diversification%20and%20evolution%20of%20common%20wheat&amp;author=N.%20Mizuno&amp;author=M.%20Yamasaki&amp;author=Y.%20Matsuoka&amp;author=T.%20Kawahara&amp;author=S.%20Takumi&amp;publication_year=2010&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=999-1013" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.19.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.19.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.19.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20149088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feldman</div> <div class="given-names">M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Bonjean</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Angus</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">Origin of Cultivated Wheat</div>, <div class="source">The World Wheat Book: A History of Wheat Breeding</div> , <div class="year">2001</div><div class="publisher-loc">Paris</div><div class="publisher-name">Lavoisier Publishing</div>(pg. <div class="fpage">3</div>-<div class="lpage">56</div>)</div></div></div></div><div content-id="DSS028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cadalen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Boeuf</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bernard</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bernard</div> <div class="given-names">M.</div></div>. </div><div class="article-title">An intervarietal molecular marker map in <em>Triticum aestivum</em> L. Em. Thell. and comparison with a map from a wide cross</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">367</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20intervarietal%20molecular%20marker%20map%20in%20Triticum%20aestivum%20L.%20Em.%20Thell.%20and%20comparison%20with%20a%20map%20from%20a%20wide%20cross&amp;author=T.%20Cadalen&amp;author=C.%20Boeuf&amp;author=S.%20Bernard&amp;author=M.%20Bernard&amp;publication_year=1997&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=94&amp;pages=367-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s001220050425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs001220050425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs001220050425"> </span></div> </div></div></div></div></div><div content-id="DSS028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Akhunov</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Analysis of gene-derived SNP marker polymorphism in US wheat (<em>Triticum aestivum</em> L.) cultivars</div>, <div class="source">Mol. Breed.</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">23</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20gene-derived%20SNP%20marker%20polymorphism%20in%20US%20wheat%20(Triticum%20aestivum%20L.)%20cultivars&amp;author=S.%20Chao&amp;author=W.%20Zhang&amp;author=E.%20Akhunov&amp;publication_year=2009&amp;journal=Mol.%20Breed.&amp;volume=23&amp;pages=23-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-008-9210-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-008-9210-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-008-9210-6"> </span></div> </div></div></div></div></div><div content-id="DSS028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allen</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Barker</div> <div class="given-names">G.L.A.</div></div>,  <div class="name"><div class="surname">Berry</div> <div class="given-names">S.T.</div></div>, et al. </div><div class="article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1086</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%2C%20single-nucleotide%20polymorphism%20discovery%20and%20linkage%20analysis%20in%20hexaploid%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=A.M.%20Allen&amp;author=G.L.A.%20Barker&amp;author=S.T.%20Berry&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=1086-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21627760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kihara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lilienfeld</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A new-synthesized 6x-wheat</div>, <div class="source">Hereditas</div> , <div class="year">1949</div>, vol. <div class="volume">35</div> <div class="issue">Suppl</div>(pg. <div class="fpage">307</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new-synthesized%206x-wheat&amp;author=H.%20Kihara&amp;author=F.%20Lilienfeld&amp;publication_year=1949&amp;journal=Hereditas&amp;volume=35&amp;pages=307-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1601-5223.1949.tb03343.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1601-5223.1949.tb03343.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1601-5223.1949.tb03343.x"> </span></div> </div></div></div></div></div><div content-id="DSS028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nasuda</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Durum wheat as a candidate for the unknown female progenitor of bread wheat: an empirical study with a highly fertile F<sub>1</sub> hybrid with <em>Aegilops tauschii</em> Coss.</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">1710</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Durum%20wheat%20as%20a%20candidate%20for%20the%20unknown%20female%20progenitor%20of%20bread%20wheat%3A%20an%20empirical%20study%20with%20a%20highly%20fertile%20F1%20hybrid%20with%20Aegilops%20tauschii%20Coss.&amp;author=Y.%20Matsuoka&amp;author=S.%20Nasuda&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=1710-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1806-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1806-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1806-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trethowan</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Mujeeb-Kazi</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Novel germplasm resources for improving environmental stress tolerance of hexaploid wheat</div>, <div class="source">Crop Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">1255</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20germplasm%20resources%20for%20improving%20environmental%20stress%20tolerance%20of%20hexaploid%20wheat&amp;author=R.M.%20Trethowan&amp;author=A.%20Mujeeb-Kazi&amp;publication_year=2008&amp;journal=Crop%20Sci.&amp;volume=48&amp;pages=1255-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2007.08.0477" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2007.08.0477" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2007.08.0477"> </span></div> </div></div></div></div></div><div content-id="DSS028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genetic and molecular dissection of naturally occurring variation</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">130</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20molecular%20dissection%20of%20naturally%20occurring%20variation&amp;author=M.%20Yano&amp;publication_year=2001&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=4&amp;pages=130-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5266(00)00148-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5266(00)00148-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5266(00)00148-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11228435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Röder</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Korzun</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Wendehake</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A microsatellite map of wheat</div>, <div class="source">Genetics</div> , <div class="year">1998</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">2007</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20map%20of%20wheat&amp;author=M.%20R%C3%B6der&amp;author=V.%20Korzun&amp;author=K.%20Wendehake&amp;publication_year=1998&amp;journal=Genetics&amp;volume=149&amp;pages=2007-23" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9691054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Somers</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Isaac</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A high-density microsatellite consensus map for bread wheat (<em>Triticum aestivum</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">1105</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20microsatellite%20consensus%20map%20for%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=D.J.%20Somers&amp;author=P.%20Isaac&amp;author=K.%20Edwards&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=1105-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1740-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1740-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1740-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15490101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deschamps</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Utilization of next-generation sequencing platforms in plant genomics and genetic variation discovery</div>, <div class="source">Mol. Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">553</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Utilization%20of%20next-generation%20sequencing%20platforms%20in%20plant%20genomics%20and%20genetic%20variation%20discovery&amp;author=S.%20Deschamps&amp;author=M.A.%20Campbell&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=25&amp;pages=553-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9357-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9357-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9357-9"> </span></div> </div></div></div></div></div><div content-id="DSS028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Bhat</div> <div class="given-names">P.R.</div></div>,  <div class="name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Development and implementation of high-throughput SNP genotyping in barley</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">582</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20high-throughput%20SNP%20genotyping%20in%20barley&amp;author=T.J.%20Close&amp;author=P.R.%20Bhat&amp;author=S.%20Lonardi&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">You</div> <div class="given-names">F.M.</div></div>,  <div class="name"><div class="surname">Huo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Deal</div> <div class="given-names">K.R.</div></div>, et al. </div><div class="article-title">Annotation-based genome-wide SNP discovery in the large and complex <em>Aegilops tauschii</em> genome using next-generation sequencing without a reference genome sequence</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">59</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotation-based%20genome-wide%20SNP%20discovery%20in%20the%20large%20and%20complex%20Aegilops%20tauschii%20genome%20using%20next-generation%20sequencing%20without%20a%20reference%20genome%20sequence&amp;author=F.M.%20You&amp;author=N.%20Huo&amp;author=K.R.%20Deal&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21266061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hatano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Characterization of three <em>vernalization insensitive3-like</em> (<em>VIL</em>) homologs in wild wheat, <em>Aegilops tauschii</em> Coss.</div>, <div class="source">Hereditas</div> , <div class="year">2012</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">62</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20three%20vernalization%20insensitive3-like%20(VIL)%20homologs%20in%20wild%20wheat%2C%20Aegilops%20tauschii%20Coss.&amp;author=K.%20Koyama&amp;author=H.%20Hatano&amp;author=J.%20Nakamura&amp;author=S.%20Takumi&amp;publication_year=2012&amp;journal=Hereditas&amp;volume=149&amp;pages=62-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/more.2012.149.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmore.2012.149.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmore.2012.149.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22568701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berkman</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Skarshewski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lorenc</div> <div class="given-names">M.T.</div></div>, et al. </div><div class="article-title">Sequencing and assembly of low copy and genic regions of isolated <em>Triticum aestivum</em> chromosome arm 7DS</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2011</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">768</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20assembly%20of%20low%20copy%20and%20genic%20regions%20of%20isolated%20Triticum%20aestivum%20chromosome%20arm%207DS&amp;author=P.J.%20Berkman&amp;author=A.%20Skarshewski&amp;author=M.T.%20Lorenc&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=768-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21356002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mayer</div> <div class="given-names">K.F.X.</div></div>,  <div class="name"><div class="surname">Martis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hedley</div> <div class="given-names">P.E.</div></div>, et al. </div><div class="article-title">Unlocking the barley genome by chromosomal and comparative genomics</div>, <div class="source">Plant Cell</div> , <div class="year">2011</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1249</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unlocking%20the%20barley%20genome%20by%20chromosomal%20and%20comparative%20genomics&amp;author=K.F.X.%20Mayer&amp;author=M.%20Martis&amp;author=P.E.%20Hedley&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=1249-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.082537" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.082537" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.082537"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21467582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">García-Gómez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talón</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garc%C3%ADa-G%C3%B3mez&amp;author=J.%20Terol&amp;author=M.%20Tal%C3%B3n&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Longden</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Bleasby</div> <div class="given-names">A.</div></div>. </div><div class="article-title">EMBOSS: The European Molecular Biology Open Software Suite</div>, <div class="source">Trends Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">276</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=EMBOSS%3A%20The%20European%20Molecular%20Biology%20Open%20Software%20Suite&amp;author=P.%20Rice&amp;author=I.%20Longden&amp;author=A.%20Bleasby&amp;publication_year=2000&amp;journal=Trends%20Genet.&amp;volume=16&amp;pages=276-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)02024-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)02024-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)02024-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10827456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neff</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Turk</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kalishaman</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Web-based primer design for single nucleotide polymorphism analysis</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">613</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Web-based%20primer%20design%20for%20single%20nucleotide%20polymorphism%20analysis&amp;author=M.M.%20Neff&amp;author=E.%20Turk&amp;author=M.%20Kalishaman&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=613-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(02)02820-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(02)02820-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(02)02820-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446140" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuda</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Iehisa</div> <div class="given-names">J.C.M.</div></div>,  <div class="name"><div class="surname">Takumi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Application of real-time PCR-based SNP detection for mapping of <em>Net2</em>, a causal D-genome gene for hybrid necrosis in interspecific crosses between tetraploid wheat and <em>Aegilops tauschii</em></div>, <div class="source">Genes Genet. Syst.</div> , <div class="year">2012</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">137</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20real-time%20PCR-based%20SNP%20detection%20for%20mapping%20of%20Net2%2C%20a%20causal%20D-genome%20gene%20for%20hybrid%20necrosis%20in%20interspecific%20crosses%20between%20tetraploid%20wheat%20and%20Aegilops%20tauschii&amp;author=R.%20Matsuda&amp;author=J.C.M.%20Iehisa&amp;author=S.%20Takumi&amp;publication_year=2012&amp;journal=Genes%20Genet.%20Syst.&amp;volume=87&amp;pages=137-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.87.137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.87.137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.87.137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22820387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Park</div> <div class="given-names">S.D.E.</div></div>. </div><div class="article-title">Trypanotolerance in west African cattle and the population genetic effects of selection. PhD Thesis</div>, <div class="year">2001</div><div class="publisher-name">University of Dublin</div></div></div></div></div><div content-id="DSS028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Abrahamson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Barlow</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Daly</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations</div>, <div class="source">Genomics</div> , <div class="year">1987</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">178</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAPMAKER%3A%20an%20interactive%20computer%20package%20for%20constructing%20primary%20genetic%20linkage%20maps%20of%20experimental%20and%20natural%20populations&amp;author=E.S.%20Lander&amp;author=P.%20Green&amp;author=J.%20Abrahamson&amp;author=A.%20Barlow&amp;author=M.J.%20Daly&amp;publication_year=1987&amp;journal=Genomics&amp;volume=1&amp;pages=178-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(87)90010-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(87)90010-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(87)90010-3"> </span></div> </div></div></div></div></div><div content-id="DSS028C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kosambi</div> <div class="given-names">D.D.</div></div>. </div><div class="article-title">The estimation of map distance from recombination values</div>, <div class="source">Ann. Eugen.</div> , <div class="year">1944</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">172</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div> </div></div></div></div></div><div content-id="DSS028C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shin-I</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Development of 5006 full-length cDNAs in barley: a tool for accessing cereal genomics resources</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">81</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%205006%20full-length%20cDNAs%20in%20barley%3A%20a%20tool%20for%20accessing%20cereal%20genomics%20resources&amp;author=K.%20Sato&amp;author=T.%20Shin-I&amp;author=M.%20Seki&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=81-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19150987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsumoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Comprehensive sequence analysis of 24,783 barley full-length cDNAs derived from 12 clone libraries</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">20</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20sequence%20analysis%20of%2024%2C783%20barley%20full-length%20cDNAs%20derived%20from%2012%20clone%20libraries&amp;author=T.%20Matsumoto&amp;author=T.%20Tanaka&amp;author=H.%20Sakai&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=20-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.171579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.171579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.171579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21415278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Madan</div> <div class="given-names">A.</div></div>. </div><div class="article-title">CAP3: a DNA sequence assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">868</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20a%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewen-Campen</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Shaner</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Panfilio</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Roth</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Extavour</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">The maternal and early embryonic transcriptome of the milkweed bug <em>Oncopeltus fasciatus</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">61</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20maternal%20and%20early%20embryonic%20transcriptome%20of%20the%20milkweed%20bug%20Oncopeltus%20fasciatus&amp;author=B.%20Ewen-Campen&amp;author=N.%20Shaner&amp;author=K.A.%20Panfilio&amp;author=Y.%20Suzuki&amp;author=S.%20Roth&amp;author=C.G.%20Extavour&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21266083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansey</div> <div class="given-names">C.N.</div></div>,  <div class="name"><div class="surname">Vaillancourt</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sekhon</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">de Leon</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kaeppler</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>. </div><div class="article-title">Maize (<em>Zea mays</em> L.) genome diversity as revealed by RNA-sequencing</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e33071</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maize%20(Zea%20mays%20L.)%20genome%20diversity%20as%20revealed%20by%20RNA-sequencing&amp;author=C.N.%20Hansey&amp;author=B.%20Vaillancourt&amp;author=R.S.%20Sekhon&amp;author=N.%20de%20Leon&amp;author=S.M.%20Kaeppler&amp;author=C.R.%20Buell&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e33071" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0033071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0033071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0033071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22438891" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neff</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Neff</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Chory</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pepper</div> <div class="given-names">A.E.</div></div>. </div><div class="article-title">dCAPS, a simple technique for the genetic analysis of single nucleotide polymorphisms: experimental applications in <em>Arabidopsis thaliana</em> genetics</div>, <div class="source">Plant J.</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">387</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dCAPS%2C%20a%20simple%20technique%20for%20the%20genetic%20analysis%20of%20single%20nucleotide%20polymorphisms%3A%20experimental%20applications%20in%20Arabidopsis%20thaliana%20genetics&amp;author=M.M.%20Neff&amp;author=J.D.%20Neff&amp;author=J.%20Chory&amp;author=A.E.%20Pepper&amp;publication_year=1998&amp;journal=Plant%20J.&amp;volume=14&amp;pages=387-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1998.00124.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1998.00124.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1998.00124.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9628033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Michaels</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Amasino</div> <div class="given-names">R.M.</div></div>. </div><div class="article-title">A robust method for detecting single-nucleotide changes as polymorphic markers by PCR</div>, <div class="source">Plant J.</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">381</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%20method%20for%20detecting%20single-nucleotide%20changes%20as%20polymorphic%20markers%20by%20PCR&amp;author=S.D.%20Michaels&amp;author=R.M.%20Amasino&amp;publication_year=1998&amp;journal=Plant%20J.&amp;volume=14&amp;pages=381-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1998.00123.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1998.00123.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1998.00123.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9628032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Komori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nitta</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Utilization of the CAPS/dCAPS method to convert rice SNPs into PCR-based markers</div>, <div class="source">Breed. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">93</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Utilization%20of%20the%20CAPS%2FdCAPS%20method%20to%20convert%20rice%20SNPs%20into%20PCR-based%20markers&amp;author=T.%20Komori&amp;author=N.%20Nitta&amp;publication_year=2005&amp;journal=Breed.%20Sci.&amp;volume=55&amp;pages=93-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.55.93" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.55.93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.55.93"> </span></div> </div></div></div></div></div><div content-id="DSS028C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qi</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Echalier</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Chao</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A chromosome bin map of 16,000 expressed sequence tag loci and distribution of genes among the three genomes of polyploid wheat</div>, <div class="source">Genetics</div> , <div class="year">2004</div>, vol. <div class="volume">168</div> (pg. <div class="fpage">701</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosome%20bin%20map%20of%2016%2C000%20expressed%20sequence%20tag%20loci%20and%20distribution%20of%20genes%20among%20the%20three%20genomes%20of%20polyploid%20wheat&amp;author=L.L.%20Qi&amp;author=B.%20Echalier&amp;author=S.%20Chao&amp;publication_year=2004&amp;journal=Genetics&amp;volume=168&amp;pages=701-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.034868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.034868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.034868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15514046" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS028C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS028C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carollo</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Matthews</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Lazo</div> <div class="given-names">G.R.</div></div>, et al. </div><div class="article-title">GrainGenes 2.0. An improved resource for the small-grains community</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">139</div> (pg. <div class="fpage">643</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GrainGenes%202.0.%20An%20improved%20resource%20for%20the%20small-grains%20community&amp;author=V.%20Carollo&amp;author=D.E.%20Matthews&amp;author=G.R.%20Lazo&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=139&amp;pages=643-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.064485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.064485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.064485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16219925" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahiro Yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=481438&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss028-dss028supp_tables.xls?Expires=2147483647&amp;Signature=JcebmU0zCjbOH3Sazp~rUIJcsl~n-ztKfpf9w1V~PNdPjmAZMgZKt8nhmLleNYmhbBwEuCe7lkR~WeSz8mXTk4ZYStfYhncTFsq1rWQxCdcUNjqhJ50yiBMWO5jQXfaD2CzYesL19M7dmRbAXp30nFklT2vfaMEgPc1WU9JVjKfDfYkbVNJLzWOQdXkA5ZM9by8Drm6qagJ7uSs8dIn2FxQuGodeXpnmLLg9p1qu-r84cQybEH~onTGUcgfOfPrdxM0PGfSPPBFpX~8KtNItLTt3oGflnXyJ4ES35rPL4Pj7CxjYdz0N3Ogmq4eCai1N~FD5imK1KmNdtytMNbCePQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss028-dss028supp_tables.doc?Expires=2147483647&amp;Signature=2~nxQ1w3smgs1wAVeWj86WOnb1flUDiP6zGYDHcNxhdNmy7XbBAn~VlCeF2UBe6cD3hUXjS7Lb-2LFGkXly5v0WMVFrFU2TiU4AXoLrv4o5ylL6ZTdJY8k12PuDxNP3YGG3OtvmbkiNIplltCUeucul5XfKU33CLjHPRRFmp5KwTyrH-vCtAI1l61tCEK421-tBiOpoTi0MCjgTptoYIqqXcN8gx-k0ghCwu8KEO-flg50RZpZzMWNHj9wesaFEZCQobGU5JLj1VI-SKkcjcl8Sbhw8uj8jtmgNZ3qV0AGnfxhVLoJrFih1jgGMzbs9e8nNN~GAbkRvTRIMnssUgfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss028-dss028supp_fna.zip?Expires=2147483647&amp;Signature=E1T5pxmmY3YYCE6zcx5nH--HarcG8H4U-ib8t9d6U2Gwveq8dH9RFThxZ47-uv7N5CTrC2wNeEhh09Wtu9fMrdFNY-jCedvDFuChX6YkmStpKUZ6xYSO9goss0yu8YUVXSgKiOtPYTlxPDu9tyOZy2ZU6tLT6jVdtUUWfaye1U2qK6ZswuZ27ruNBPIb3L8bgpOY-hnE0mrrHWf2pD9p-g9~H0cCJxx26zeTSuWgxxldhVB9e6nsRR-a7p9eif81UK646538ozOh3Ni0aCbc2YjluQZcD20N-Jh6RYKGja39JeH~J7AvcBZhG6p-shMo~9wD2Af4e8CdXN8yAt3ZYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - zip file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss028/2/dss028-dss028supp_figs.pdf?Expires=2147483647&amp;Signature=f2784jZgfi-UKZGJf4pAYCadTxl-A0sIA0J29l3pqaBxOOwNMIjm9fWrBi3vXK82wbbW4VQ5X7ZNByqxmjLea72zpxKCN9xcGndWyHHkwAvV5dvaTrp~2zNiJnBtztAXczY4eBgY7eGaVRuX21R-kXtodh8ymg-Bpud6eYV~BsGIFFnuVMPPgooLBhC8O3Z~9-VuL9SGxmTwSUunErvsE9bezATEOKAIgemDLkYclW0zYu3-88WISbRmCDexRDOqL94B1H58l64qaW5A9kjwwCmaABToAi6lolyo1UNHILkkmvno0Mddt8H0DaiemUn02gZ60MViDsnixdvDuGfozA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">242</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">17</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="481438"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f6%2f487%2f481438">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000313135200005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (17)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F6%2F487%2F481438" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/481438'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Discovery of High-Confidence Single Nucleotide Polymorphisms from Large-Scale De Novo Analysis of Leaf Transcripts of Aegilops tauschii, A Wild Wheat Progenitor" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f487%2f481438">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

