

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/2/121/332636","@type":"ScholarlyArticle","name":"A draft genome of field pennycress ( Thlaspi arvense ) provides tools for the domestication of a new winter biofuel crop","datePublished":"2015-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2015-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu045","keywords":["field pennycress","<italic>Thlaspi arvense</italic>","<italic>de novo</italic> assembly","whole genome sequencing","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/2/121/332636","author":[{"name":"Dorn, Kevin M.","affiliation":"1  Department of Plant Biology ,  University of Minnesota ,  Saint Paul, MN 55108 ,  USA","@type":"Person"},{"name":"Fankhauser, Johnathon D.","affiliation":"1  Department of Plant Biology ,  University of Minnesota ,  Saint Paul, MN 55108 ,  USA","@type":"Person"},{"name":"Wyse, Donald L.","affiliation":"2  Department of Agronomy and Plant Genetics ,  University of Minnesota ,  Saint Paul, MN 55108 ,  USA","@type":"Person"},{"name":"Marks, M. David","affiliation":"1  Department of Plant Biology ,  University of Minnesota ,  Saint Paul, MN 55108 ,  USA","@type":"Person"}],"description":"Abstract.  Field pennycress (Thlaspi arvense L.) is being domesticated as a new winter cover crop and biofuel species for the Midwestern United States that can ","pageStart":"121","pageEnd":"131","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A draft genome of field pennycress ( Thlaspi arvense ) provides tools for the domestication of a new winter biofuel crop","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="draft genome of field pennycress ( Thlaspi arvense ) provides tools for the domestication of a new winter biofuel crop" />
    <meta property="og:description" content="Abstract.  Field pennycress (Thlaspi arvense L.) is being domesticated as a new winter cover crop and biofuel species for the Midwestern United States that can " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/2/121/332636" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-01-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Dorn, Kevin M." /><meta name="citation_author_institution" content="Department of Plant Biology, University of Minnesota, Saint Paul, MN 55108, USA" /><meta name="citation_author" content="Fankhauser, Johnathon D." /><meta name="citation_author_institution" content="Department of Plant Biology, University of Minnesota, Saint Paul, MN 55108, USA" /><meta name="citation_author" content="Wyse, Donald L." /><meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, MN 55108, USA" /><meta name="citation_author" content="Marks, M. David" /><meta name="citation_author_institution" content="Department of Plant Biology, University of Minnesota, Saint Paul, MN 55108, USA" /><meta name="citation_title" content="A draft genome of field pennycress (Thlaspi arvense) provides tools for the domestication of a new winter biofuel crop" /><meta name="citation_firstpage" content="121" /><meta name="citation_lastpage" content="131" /><meta name="citation_doi" content="10.1093/dnares/dsu045" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2015/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The reference genome of the halophytic plant Eutrema salsugineum; citation_author=Yang  R.; citation_author=Jarvis  D.E.; citation_author=Chen  H.;  citation_journal_title=Front. Plant Sci.;  citation_year=2013;  citation_volume=4;  citation_pages=46" /><meta name="citation_reference" content="citation_title=Genome sequencing of the high oil crop sesame provides insight into oil biosynthesis; citation_author=Wang  L.; citation_author=Yu  S.; citation_author=Tong  C.;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=R39" /><meta name="citation_reference" content="citation_title=The emerging biofuel crop Camelina sativa retains a highly undifferentiated hexaploid genome structure; citation_author=Kagale  S.; citation_author=Koh  C.S.; citation_author=Nixon  J.;  citation_journal_title=Nat. Commun.;  citation_year=2014;  citation_volume=5;  citation_pages=3706" /><meta name="citation_reference" content="citation_title=The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes; citation_author=Liu  S.; citation_author=Liu  Y.; citation_author=Yang  X.;  citation_journal_title=Nat. Commun.;  citation_year=2014;  citation_volume=5;  citation_pages=3930" /><meta name="citation_reference" content="citation_title=The Capsella rubella genome and the genomic consequences of rapid mating system evolution; citation_author=Slotte  T.; citation_author=Hazzouri  K.M.; citation_author=Agren  J.A.;  citation_journal_title=Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=831-5; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement; citation_author=Varshney  R.K.; citation_author=Song  C.; citation_author=Saxena  R.K.;  citation_journal_title=Nat. Biotechnol.;  citation_year=2013;  citation_volume=31;  citation_pages=240-6; " /><meta name="citation_reference" content="citation_title=Harvesting the promising fruits of genomics: applying genome sequencing technologies to crop breeding; citation_author=Varshney  R.K.; citation_author=Terauchi  R.; citation_author=McCouch  S.R.;  citation_journal_title=PLoS Biol.;  citation_year=2014;  citation_volume=12;  citation_pages=e1001883" /><meta name="citation_reference" content="citation_title=Next generation sequencing technologies for next generation plant breeding; citation_author=Ray  S.; citation_author=Satya  P.;  citation_journal_title=Front. Plant Sci.;  citation_year=2014;  citation_volume=5;  citation_pages=367" /><meta name="citation_reference" content="citation_title=New approaches to facilitate rapid domestication of a wild plant to an oilseed crop: example pennycress (Thlaspi arvense L.); citation_author=Sedbrook  J.C.; citation_author=Phippen  W.B.; citation_author=Marks  M.D.;  citation_journal_title=Plant Sci.;  citation_year=2014;  citation_volume=22,7C;  citation_pages=122-32; " /><meta name="citation_reference" content="citation_title=The biology of Canadian weeds 9. Thlaspi arvense L; citation_author=Best  K.F.A.; citation_author=Mcintyre  G.I.;  citation_journal_title=Canad. J. Plant Sci.;  citation_year=1975;  citation_volume=55;  citation_pages=279-92; " /><meta name="citation_reference" content="citation_title=The biology of Canadian weeds. 9. Thlaspi arvense L. (updated); citation_author=Warwick  S.I.; citation_author=Francis  A.; citation_author=Susko  D.J.;  citation_journal_title=Canad. J. Plant Sci.;  citation_year=2002;  citation_volume=82;  citation_pages=803-23; " /><meta name="citation_reference" content="citation_title=Evolution of genome size in Brassicaceae; citation_author=Johnston  J.S.; citation_author=Pepper  A.E.; citation_author=Hall  A.E.;  citation_journal_title=Ann. Bot.;  citation_year=2005;  citation_volume=95;  citation_pages=229-35; " /><meta name="citation_reference" content="citation_title=Molecular data reveal convergence in fruit characters used in the classification of Thlaspi s. l. (Brassicaceae); citation_author=Mummenhoff  K.; citation_author=Franzke  A.; citation_author=Koch  M.;  citation_journal_title=Bot. J. Linn. Soc.;  citation_year=1997;  citation_volume=125;  citation_pages=183-99; " /><meta name="citation_reference" content="citation_title=Root and shoot transcriptome analysis of two ecotypes of Noccaea caerulescens uncovers the role of NcNramp1 in Cd hyperaccumulation; citation_author=Milner  M.J.; citation_author=Mitani-Ueno  N.; citation_author=Yamaji  N.;  citation_journal_title=Plant J.;  citation_year=2014;  citation_volume=78;  citation_pages=398-410; " /><meta name="citation_reference" content="citation_title=The molecular physiology of heavy metal transport in the Zn/Cd hyperaccumulator Thlaspi caerulescens; citation_author=Pence  N.S.; citation_author=Larsen  P.B.; citation_author=Ebbs  S.D.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=4956-60; " /><meta name="citation_reference" content="citation_title=Taxonomy and systematics are key to biological information: Arabidopsis, Eutrema (Thellungiella), Noccaea and Schrenkiella (Brassicaceae) as examples; citation_author=Koch  M.A.; citation_author=German  D.A.;  citation_journal_title=Front. Plant Sci.;  citation_year=2013;  citation_volume=4;  citation_pages=267" /><meta name="citation_reference" content="citation_title=Cabbage family affairs: the evolutionary history of Brassicaceae; citation_author=Franzke  A.; citation_author=Lysak  M.A.; citation_author=Al-Shehbaz  I.A.; citation_author=Koch  M.A.; citation_author=Mummenhoff  K.;  citation_journal_title=Trends Plant Sci.;  citation_year=2011;  citation_volume=16;  citation_pages=108-16; " /><meta name="citation_reference" content="citation_title=Composition and physical properties of cress (Lepidium sativum L.) and field pennycress (Thlaspi arvense L.) oils; citation_author=Moser  B.R.; citation_author=Shah  S.N.; citation_author=Winkler-Moser  J.K.; citation_author=Vaughn  S.F.; citation_author=Evangelista  R.L.;  citation_journal_title=Ind. Crops Prod.;  citation_year=2009;  citation_volume=30;  citation_pages=199-205; " /><meta name="citation_reference" content="citation_title=De novo assembly of the pennycress (Thlaspi arvense) transcriptome provides tools for the development of a winter cover crop and biodiesel feedstock; citation_author=Dorn  K.M.; citation_author=Fankhauser  J.D.; citation_author=Wyse  D.L.; citation_author=Marks  M.D.;  citation_journal_title=Plant J.;  citation_year=2013;  citation_volume=75;  citation_pages=1028-38; " /><meta name="citation_reference" content="citation_title=A life cycle assessment of pennycress (Thlaspi aruense L.) -derived jet fuel and diesel; citation_author=Fan  J.Q.; citation_author=Shonnard  D.R.; citation_author=Kalnes  T.N.; citation_author=Johnsen  P.B.; citation_author=Rao  S.;  citation_journal_title=Biomass Bioenergy;  citation_year=2013;  citation_volume=55;  citation_pages=87-100; " /><meta name="citation_reference" content="citation_title=Production and evaluation of biodiesel from field pennycress (Thlaspi arvense L.) oil; citation_author=Moser  B.R.; citation_author=Knothe  G.; citation_author=Vaughn  S.F.; citation_author=Isbell  T.A.;  citation_journal_title=Energy Fuels;  citation_year=2009;  citation_volume=23;  citation_pages=4149-55; " /><meta name="citation_reference" content="citation_title=Soybean seed yield and quality as a response to field pennycress residue; citation_author=Phippen  W.B.; citation_author=Phippen  M.E.;  citation_journal_title=Crop Sci.;  citation_year=2012;  citation_volume=52;  citation_pages=2767-73; " /><meta name="citation_reference" content="citation_title=Agricultural sustainability and intensive production practices; citation_author=Tilman  D.; citation_author=Cassman  K.G.; citation_author=Matson  P.A.; citation_author=Naylor  R.; citation_author=Polasky  S.;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=418;  citation_pages=671-7; " /><meta name="citation_reference" content="citation_title=Market cost of renewable jet fuel adoption in the United States; citation_author=Winchester  N.; citation_author=McConnachie  D.; citation_author=Wollersheim  C.; citation_author=Waitz  I.;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Dormancy breaking and seed germination of the annual weeds Thlaspi arvense, Descurainia sophia and Malcolmia africana (Brassicaceae); citation_author=Karimmojeni  H.T.; citation_author=Taab  A.; citation_author=Rashidi  B.; citation_author=Bazrafshan  A.H.;  citation_journal_title=J. Plant Protect. Res.;  citation_year=2014;  citation_volume=54;  citation_pages=179-87; " /><meta name="citation_reference" content="citation_title=Breakage of seed dormancy of field pennycress (Thlaspi arvense) by growth-regulators, nitrate, and environmental-factors; citation_author=Saini  H.S.; citation_author=Bassi  P.K.; citation_author=Goudey  J.S.; citation_author=Spencer  M.S.;  citation_journal_title=Weed Sci.;  citation_year=1987;  citation_volume=35;  citation_pages=802-6; " /><meta name="citation_reference" content="citation_title=Extraction of pennycress (Thlaspi arvense L.) seed oil by full pressing; citation_author=Evangelista  R.L.; citation_author=Isbell  T.A.; citation_author=Cermak  S.C.;  citation_journal_title=Ind. Crops Prod.;  citation_year=2012;  citation_volume=37;  citation_pages=76-81; " /><meta name="citation_reference" content="citation_title=The development of Arabidopsis as a model plant; citation_author=Koornneef  M.; citation_author=Meinke  D.;  citation_journal_title=Plant J.;  citation_year=2010;  citation_volume=61;  citation_pages=909-21; " /><meta name="citation_reference" content="citation_title=Regulation of de novo fatty acid synthesis in maturing oilseeds of Arabidopsis; citation_author=Baud  S.; citation_author=Lepiniec  L.;  citation_journal_title=Plant Physiol. Biochem.;  citation_year=2009;  citation_volume=47;  citation_pages=448-55; " /><meta name="citation_reference" content="citation_title=Molecular networks regulating Arabidopsis seed maturation, after-ripening, dormancy and germination; citation_author=Holdsworth  M.J.; citation_author=Bentsink  L.; citation_author=Soppe  W.J.;  citation_journal_title=New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=33-54; " /><meta name="citation_reference" content="citation_title=Flowering time control and applications in plant breeding; citation_author=Jung  C.; citation_author=Muller  A.E.;  citation_journal_title=Trends Plant Sci.;  citation_year=2009;  citation_volume=14;  citation_pages=563-73; " /><meta name="citation_reference" content="citation_title=Targeting induced local lesions IN genomes (TILLING) for plant functional genomics; citation_author=McCallum  C.M.; citation_author=Comai  L.; citation_author=Greene  E.A.; citation_author=Henikoff  S.;  citation_journal_title=Plant Physiol.;  citation_year=2000;  citation_volume=123;  citation_pages=439-42; " /><meta name="citation_reference" content="citation_title=Deletion-based reverse genetics in Medicago truncatula; citation_author=Rogers  C.; citation_author=Wen  J.; citation_author=Chen  R.; citation_author=Oldroyd  G.;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=1077-86; " /><meta name="citation_reference" content="citation_title=Plant genome editing made easy: targeted mutagenesis in model and crop plants using the CRISPR/Cas system; citation_author=Belhaj  K.; citation_author=Chaparro-Garcia  A.; citation_author=Kamoun  S.; citation_author=Nekrasov  V.;  citation_journal_title=Plant Methods;  citation_year=2013;  citation_volume=9;  citation_pages=39" /><meta name="citation_reference" content="citation_title=Specialized binary vector for plant transformation: expression of the Arabidopsis thaliana AHAS gene in Nicotiana tabacum; citation_author=Olszewski  N.E.; citation_author=Martin  F.B.; citation_author=Ausubel  F.M.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=10765-82; " /><meta name="citation_reference" content="citation_title=A physical, genetic and functional sequence assembly of the barley genome; citation_author=International Barley Genome Sequencing Consortium; citation_author=Mayer  K.F.; citation_author=Waugh  R.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=711-6; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the rubber tree Hevea brasiliensis; citation_author=Rahman  A.Y.; citation_author=Usharraj  A.O.; citation_author=Misra  B.B.;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=75" /><meta name="citation_reference" content="citation_title=Scaffolding pre-assembled contigs using SSPACE; citation_author=Boetzer  M.; citation_author=Henkel  C.V.; citation_author=Jansen  H.J.; citation_author=Butler  D.; citation_author=Pirovano  W.;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=578-9; " /><meta name="citation_reference" content="citation_title=Toward almost closed genomes with GapFiller; citation_author=Boetzer  M.; citation_author=Pirovano  W.;  citation_journal_title=Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=R56" /><meta name="citation_reference" content="citation_title=MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes; citation_author=Cantarel  B.L.; citation_author=Korf  I.; citation_author=Robb  S.M.;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=188-96; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The Arabidopsis lyrata genome sequence and the basis of rapid genome size change; citation_author=Hu  T.T.; citation_author=Pattyn  P.; citation_author=Bakker  E.G.;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=476-81; " /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa; citation_author=Wang  X.; citation_author=Wang  H.; citation_author=Wang  J.;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=Identification of cucurbitacins and assembly of a draft genome for Aquilaria agallocha; citation_author=Chen  C.H.; citation_author=Kuo  T.C.; citation_author=Yang  M.H.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=578" /><meta name="citation_reference" content="citation_title=The draft genome of watermelon (Citrullus lanatus) and resequencing of 20 diverse accessions; citation_author=Guo  S.; citation_author=Zhang  J.; citation_author=Sun  H.;  citation_journal_title=Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=51-8; " /><meta name="citation_reference" content="Smit A. Hubley R. Green P.  1996–2010 , RepeatMasker Open-3.0. " /><meta name="citation_reference" content="citation_title=Visualizing next-generation sequencing data with JBrowse; citation_author=Westesson  O.; citation_author=Skinner  M.; citation_author=Holmes  I.;  citation_journal_title=Brief. Bioinform.;  citation_year=2013;  citation_volume=14;  citation_pages=172-7; " /><meta name="citation_reference" content="citation_title=Insights into salt tolerance from the genome of Thellungiella salsuginea; citation_author=Wu  H.J.; citation_author=Zhang  Z.; citation_author=Wang  J.Y.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=12219-24; " /><meta name="citation_reference" content="citation_title=A procedure for mapping arabidopsis mutations using codominant ecotype-specific pcr-based markers; citation_author=Konieczny  A.; citation_author=Ausubel  F.M.;  citation_journal_title=Plant J.;  citation_year=1993;  citation_volume=4;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Biology and biochemistry of glucosinolates; citation_author=Halkier  B.A.; citation_author=Gershenzon  J.;  citation_journal_title=Annu. Rev. Plant Biol.;  citation_year=2006;  citation_volume=57;  citation_pages=303-33; " /><meta name="citation_reference" content="citation_title=Glucosinolate metabolism and its control; citation_author=Grubb  C.D.; citation_author=Abel  S.;  citation_journal_title=Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=89-100; " /><meta name="citation_reference" content="citation_title=Evolution of specifier proteins in glucosinolate-containing plants; citation_author=Kuchernig  J.C.; citation_author=Burow  M.; citation_author=Wittstock  U.;  citation_journal_title=BMC Evol. Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=127" /><meta name="citation_reference" content="citation_title=A new type of enzymatic cleavage of mustard oil glucosides. Formation of allylthiocyanate in Thlaspi arvense L and benzylthiocynate in Lepidium ruderale L and Lepidium sativum L; citation_author=Gmelin  R.; citation_author=Virtanen  A.I.;  citation_journal_title=Acta Chem. Scand.;  citation_year=1959;  citation_volume=13;  citation_pages=1474-1475; " /><meta name="citation_reference" content="citation_title=A thiocyanate-forming protein generates multiple products upon allylglucosinolate breakdown in Thlaspi arvense; citation_author=Kuchernig  J.C.; citation_author=Backenkohler  A.; citation_author=Lubbecke  M.; citation_author=Burow  M.; citation_author=Wittstock  U.;  citation_journal_title=Phytochemistry;  citation_year=2011;  citation_volume=72;  citation_pages=1699-709; " /><meta name="citation_reference" content="citation_title=Effects of cold-pressing and seed cooking on functional properties of protein in pennycress (Thlaspi arvense L.) seed and press cakes; citation_author=Hojilla-Evangelista  M.P.; citation_author=Evangelista  R.L.; citation_author=Isbell  T.A.; citation_author=Selling  G.W.;  citation_journal_title=Ind. Crops Prod.;  citation_year=2013;  citation_volume=45;  citation_pages=223-9; " /><meta name="citation_reference" content="citation_title=Extraction of proteins from pennycress seeds and press cake; citation_author=Selling  G.W.; citation_author=Hojilla-Evangelista  M.P.; citation_author=Evangelista  R.L.; citation_author=Isbell  T.; citation_author=Price  N.; citation_author=Doll  K.M.;  citation_journal_title=Ind. Crops Prod.;  citation_year=2013;  citation_volume=41;  citation_pages=113-9; " /><meta name="citation_reference" content="citation_title=MODIFIED VACUOLE PHENOTYPE1 is an Arabidopsis myrosinase-associated protein involved in endomembrane protein trafficking; citation_author=Agee  A.E.; citation_author=Surpin  M.; citation_author=Sohn  E.J.;  citation_journal_title=Plant Physiol.;  citation_year=2010;  citation_volume=152;  citation_pages=120-32; " /><meta name="citation_reference" content="citation_title=The gene controlling the quantitative trait locus EPITHIOSPECIFIER MODIFIER1 alters glucosinolate hydrolysis and insect resistance in Arabidopsis; citation_author=Zhang  Z.; citation_author=Ober  J.A.; citation_author=Kliebenstein  D.J.;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1524-36; " /><meta name="citation_reference" content="citation_title=Whole genome and tandem duplicate retention facilitated glucosinolate pathway diversification in the mustard family; citation_author=Hofberger  J.A.; citation_author=Lyons  E.; citation_author=Edger  P.P.; citation_author=Chris Pires  J.; citation_author=Eric Schranz  M.;  citation_journal_title=Genome Biol. Evol.;  citation_year=2013;  citation_volume=5;  citation_pages=2155-73; " /><meta name="citation_reference" content="citation_title=NRT/PTR transporters are essential for translocation of glucosinolate defence compounds to seeds; citation_author=Nour-Eldin  H.H.; citation_author=Andersen  T.G.; citation_author=Burow  M.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=488;  citation_pages=531-4; " /><meta name="citation_reference" content="citation_title=Integration of biosynthesis and long-distance transport establish organ-specific glucosinolate profiles in vegetative Arabidopsis; citation_author=Andersen  T.G.; citation_author=Nour-Eldin  H.H.; citation_author=Fuller  V.L.; citation_author=Olsen  C.E.; citation_author=Burow  M.; citation_author=Halkier  B.A.;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3133-45; " /><meta name="citation_reference" content="citation_title=Cloning of DOG1, a quantitative trait locus controlling seed dormancy in Arabidopsis; citation_author=Bentsink  L.; citation_author=Jowett  J.; citation_author=Hanhart  C.J.; citation_author=Koornneef  M.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=17042-47; " /><meta name="citation_reference" content="citation_title=The time required for dormancy release in arabidopsis is determined by DELAY OF GERMINATION1 protein levels in freshly harvested seeds; citation_author=Nakabayashi  K.; citation_author=Bartsch  M.; citation_author=Xiang  Y.;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=2826-38; " /><meta name="citation_reference" content="citation_title=The genome sequence of the colonial chordate, Botryllus Schlosseri; citation_author=Voskoboynik  A.; citation_author=Neff  N.F.; citation_author=Sahoo  D.;  citation_journal_title=Elife;  citation_year=2013;  citation_volume=2;  citation_pages=e00569" /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing for plant breeding and genetics; citation_author=Poland  J.A.; citation_author=Rife  T.W.;  citation_journal_title=Plant Genome US;  citation_year=2012;  citation_volume=5;  citation_pages=92-102; " /><meta name="citation_reference" content="citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species; citation_author=Peterson  B.K.; citation_author=Weber  J.N.; citation_author=Kay  E.H.; citation_author=Fisher  H.S.; citation_author=Hoekstra  H.E.;  citation_journal_title=PloS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e37135" /><meta name="citation_reference" content="citation_title=Taxonomic and phylogenetic evaluation of the American ‘Thlaspi’ species: identity and relationship to the Eurasian genus Noccaea (Brassicaceae); citation_author=Koch  M.; citation_author=AL-Shehbaz  I.A.;  citation_journal_title=Syst. Bot.;  citation_year=2004;  citation_volume=29;  citation_pages=375-84; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/2/121/950917/dsu045.pdf" /><meta name="description" content="Abstract.  Field pennycress (Thlaspi arvense L.) is being domesticated as a new winter cover crop and biofuel species for the Midwestern United States that can " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/2/121/332636" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/2/121/332636" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/2/121/332636");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu045';
    </script>
<title>
	draft genome of field pennycress (Thlaspi arvense) provides tools for the domestication of a new winter biofuel crop | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./332636" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdPEATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRG9ybiwgS2V2aW4gTS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IEJpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgTWlubmVzb3RhLCBTYWludCBQYXVsLCBNTiA1NTEwOCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFua2hhdXNlciwgSm9obmF0aG9uIEQuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIE1pbm5lc290YSwgU2FpbnQgUGF1bCwgTU4gNTUxMDgsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ild5c2UsIERvbmFsZCBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyb25vbXkgYW5kIFBsYW50IEdlbmV0aWNzLCBVbml2ZXJzaXR5IG9mIE1pbm5lc290YSwgU2FpbnQgUGF1bCwgTU4gNTUxMDgsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hcmtzLCBNLiBEYXZpZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBNaW5uZXNvdGEsIFNhaW50IFBhdWwsIE1OIDU1MTA4LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgZHJhZnQgZ2Vub21lIG9mIGZpZWxkIHBlbm55Y3Jlc3MgKFRobGFzcGkgYXJ2ZW5zZSkgcHJvdmlkZXMgdG9vbHMgZm9yIHRoZSBkb21lc3RpY2F0aW9uIG9mIGEgbmV3IHdpbnRlciBiaW9mdWVsIGNyb3AiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTA0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlZmVyZW5jZSBnZW5vbWUgb2YgdGhlIGhhbG9waHl0aWMgcGxhbnQgRXV0cmVtYSBzYWxzdWdpbmV1bTsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIFIuOyBjaXRhdGlvbl9hdXRob3I9SmFydmlzICBELkUuOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIG9mIHRoZSBoaWdoIG9pbCBjcm9wIHNlc2FtZSBwcm92aWRlcyBpbnNpZ2h0IGludG8gb2lsIGJpb3N5bnRoZXNpczsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9WXUgIFMuOyBjaXRhdGlvbl9hdXRob3I9VG9uZyAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPVIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlbWVyZ2luZyBiaW9mdWVsIGNyb3AgQ2FtZWxpbmEgc2F0aXZhIHJldGFpbnMgYSBoaWdobHkgdW5kaWZmZXJlbnRpYXRlZCBoZXhhcGxvaWQgZ2Vub21lIHN0cnVjdHVyZTsgY2l0YXRpb25fYXV0aG9yPUthZ2FsZSAgUy47IGNpdGF0aW9uX2F1dGhvcj1Lb2ggIEMuUy47IGNpdGF0aW9uX2F1dGhvcj1OaXhvbiAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzcwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCcmFzc2ljYSBvbGVyYWNlYSBnZW5vbWUgcmV2ZWFscyB0aGUgYXN5bW1ldHJpY2FsIGV2b2x1dGlvbiBvZiBwb2x5cGxvaWQgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUxpdSAgUy47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIFkuOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzkzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDYXBzZWxsYSBydWJlbGxhIGdlbm9tZSBhbmQgdGhlIGdlbm9taWMgY29uc2VxdWVuY2VzIG9mIHJhcGlkIG1hdGluZyBzeXN0ZW0gZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9U2xvdHRlICBULjsgY2l0YXRpb25fYXV0aG9yPUhhenpvdXJpICBLLk0uOyBjaXRhdGlvbl9hdXRob3I9QWdyZW4gIEouQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9ODMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBjaGlja3BlYSAoQ2ljZXIgYXJpZXRpbnVtKSBwcm92aWRlcyBhIHJlc291cmNlIGZvciB0cmFpdCBpbXByb3ZlbWVudDsgY2l0YXRpb25fYXV0aG9yPVZhcnNobmV5ICBSLksuOyBjaXRhdGlvbl9hdXRob3I9U29uZyAgQy47IGNpdGF0aW9uX2F1dGhvcj1TYXhlbmEgIFIuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yNDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGFydmVzdGluZyB0aGUgcHJvbWlzaW5nIGZydWl0cyBvZiBnZW5vbWljczogYXBwbHlpbmcgZ2Vub21lIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzIHRvIGNyb3AgYnJlZWRpbmc7IGNpdGF0aW9uX2F1dGhvcj1WYXJzaG5leSAgUi5LLjsgY2l0YXRpb25fYXV0aG9yPVRlcmF1Y2hpICBSLjsgY2l0YXRpb25fYXV0aG9yPU1jQ291Y2ggIFMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMTg4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llcyBmb3IgbmV4dCBnZW5lcmF0aW9uIHBsYW50IGJyZWVkaW5nOyBjaXRhdGlvbl9hdXRob3I9UmF5ICBTLjsgY2l0YXRpb25fYXV0aG9yPVNhdHlhICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTM2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBhcHByb2FjaGVzIHRvIGZhY2lsaXRhdGUgcmFwaWQgZG9tZXN0aWNhdGlvbiBvZiBhIHdpbGQgcGxhbnQgdG8gYW4gb2lsc2VlZCBjcm9wOiBleGFtcGxlIHBlbm55Y3Jlc3MgKFRobGFzcGkgYXJ2ZW5zZSBMLik7IGNpdGF0aW9uX2F1dGhvcj1TZWRicm9vayAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPVBoaXBwZW4gIFcuQi47IGNpdGF0aW9uX2F1dGhvcj1NYXJrcyAgTS5ELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMiw3QzsgIGNpdGF0aW9uX3BhZ2VzPTEyMi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJpb2xvZ3kgb2YgQ2FuYWRpYW4gd2VlZHMgOS4gVGhsYXNwaSBhcnZlbnNlIEw7IGNpdGF0aW9uX2F1dGhvcj1CZXN0ICBLLkYuQS47IGNpdGF0aW9uX2F1dGhvcj1NY2ludHlyZSAgRy5JLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuYWQuIEouIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0yNzktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBiaW9sb2d5IG9mIENhbmFkaWFuIHdlZWRzLiA5LiBUaGxhc3BpIGFydmVuc2UgTC4gKHVwZGF0ZWQpOyBjaXRhdGlvbl9hdXRob3I9V2Fyd2ljayAgUy5JLjsgY2l0YXRpb25fYXV0aG9yPUZyYW5jaXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9U3Vza28gIEQuSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmFkLiBKLiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTgyOyAgY2l0YXRpb25fcGFnZXM9ODAzLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgZ2Vub21lIHNpemUgaW4gQnJhc3NpY2FjZWFlOyBjaXRhdGlvbl9hdXRob3I9Sm9obnN0b24gIEouUy47IGNpdGF0aW9uX2F1dGhvcj1QZXBwZXIgIEEuRS47IGNpdGF0aW9uX2F1dGhvcj1IYWxsICBBLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0yMjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBkYXRhIHJldmVhbCBjb252ZXJnZW5jZSBpbiBmcnVpdCBjaGFyYWN0ZXJzIHVzZWQgaW4gdGhlIGNsYXNzaWZpY2F0aW9uIG9mIFRobGFzcGkgcy4gbC4gKEJyYXNzaWNhY2VhZSk7IGNpdGF0aW9uX2F1dGhvcj1NdW1tZW5ob2ZmICBLLjsgY2l0YXRpb25fYXV0aG9yPUZyYW56a2UgIEEuOyBjaXRhdGlvbl9hdXRob3I9S29jaCAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJvdC4gSi4gTGlubi4gU29jLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjU7ICBjaXRhdGlvbl9wYWdlcz0xODMtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvb3QgYW5kIHNob290IHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgdHdvIGVjb3R5cGVzIG9mIE5vY2NhZWEgY2FlcnVsZXNjZW5zIHVuY292ZXJzIHRoZSByb2xlIG9mIE5jTnJhbXAxIGluIENkIGh5cGVyYWNjdW11bGF0aW9uOyBjaXRhdGlvbl9hdXRob3I9TWlsbmVyICBNLkouOyBjaXRhdGlvbl9hdXRob3I9TWl0YW5pLVVlbm8gIE4uOyBjaXRhdGlvbl9hdXRob3I9WWFtYWppICBOLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0zOTgtNDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIHBoeXNpb2xvZ3kgb2YgaGVhdnkgbWV0YWwgdHJhbnNwb3J0IGluIHRoZSBabi9DZCBoeXBlcmFjY3VtdWxhdG9yIFRobGFzcGkgY2FlcnVsZXNjZW5zOyBjaXRhdGlvbl9hdXRob3I9UGVuY2UgIE4uUy47IGNpdGF0aW9uX2F1dGhvcj1MYXJzZW4gIFAuQi47IGNpdGF0aW9uX2F1dGhvcj1FYmJzICBTLkQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NDk1Ni02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGF4b25vbXkgYW5kIHN5c3RlbWF0aWNzIGFyZSBrZXkgdG8gYmlvbG9naWNhbCBpbmZvcm1hdGlvbjogQXJhYmlkb3BzaXMsIEV1dHJlbWEgKFRoZWxsdW5naWVsbGEpLCBOb2NjYWVhIGFuZCBTY2hyZW5raWVsbGEgKEJyYXNzaWNhY2VhZSkgYXMgZXhhbXBsZXM7IGNpdGF0aW9uX2F1dGhvcj1Lb2NoICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9R2VybWFuICBELkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2FiYmFnZSBmYW1pbHkgYWZmYWlyczogdGhlIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIEJyYXNzaWNhY2VhZTsgY2l0YXRpb25fYXV0aG9yPUZyYW56a2UgIEEuOyBjaXRhdGlvbl9hdXRob3I9THlzYWsgIE0uQS47IGNpdGF0aW9uX2F1dGhvcj1BbC1TaGVoYmF6ICBJLkEuOyBjaXRhdGlvbl9hdXRob3I9S29jaCAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPU11bW1lbmhvZmYgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEwOC0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb24gYW5kIHBoeXNpY2FsIHByb3BlcnRpZXMgb2YgY3Jlc3MgKExlcGlkaXVtIHNhdGl2dW0gTC4pIGFuZCBmaWVsZCBwZW5ueWNyZXNzIChUaGxhc3BpIGFydmVuc2UgTC4pIG9pbHM7IGNpdGF0aW9uX2F1dGhvcj1Nb3NlciAgQi5SLjsgY2l0YXRpb25fYXV0aG9yPVNoYWggIFMuTi47IGNpdGF0aW9uX2F1dGhvcj1XaW5rbGVyLU1vc2VyICBKLksuOyBjaXRhdGlvbl9hdXRob3I9VmF1Z2huICBTLkYuOyBjaXRhdGlvbl9hdXRob3I9RXZhbmdlbGlzdGEgIFIuTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZC4gQ3JvcHMgUHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xOTktMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIHRoZSBwZW5ueWNyZXNzIChUaGxhc3BpIGFydmVuc2UpIHRyYW5zY3JpcHRvbWUgcHJvdmlkZXMgdG9vbHMgZm9yIHRoZSBkZXZlbG9wbWVudCBvZiBhIHdpbnRlciBjb3ZlciBjcm9wIGFuZCBiaW9kaWVzZWwgZmVlZHN0b2NrOyBjaXRhdGlvbl9hdXRob3I9RG9ybiAgSy5NLjsgY2l0YXRpb25fYXV0aG9yPUZhbmtoYXVzZXIgIEouRC47IGNpdGF0aW9uX2F1dGhvcj1XeXNlICBELkwuOyBjaXRhdGlvbl9hdXRob3I9TWFya3MgIE0uRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9MTAyOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsaWZlIGN5Y2xlIGFzc2Vzc21lbnQgb2YgcGVubnljcmVzcyAoVGhsYXNwaSBhcnVlbnNlIEwuKSAtZGVyaXZlZCBqZXQgZnVlbCBhbmQgZGllc2VsOyBjaXRhdGlvbl9hdXRob3I9RmFuICBKLlEuOyBjaXRhdGlvbl9hdXRob3I9U2hvbm5hcmQgIEQuUi47IGNpdGF0aW9uX2F1dGhvcj1LYWxuZXMgIFQuTi47IGNpdGF0aW9uX2F1dGhvcj1Kb2huc2VuICBQLkIuOyBjaXRhdGlvbl9hdXRob3I9UmFvICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbWFzcyBCaW9lbmVyZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz04Ny0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2R1Y3Rpb24gYW5kIGV2YWx1YXRpb24gb2YgYmlvZGllc2VsIGZyb20gZmllbGQgcGVubnljcmVzcyAoVGhsYXNwaSBhcnZlbnNlIEwuKSBvaWw7IGNpdGF0aW9uX2F1dGhvcj1Nb3NlciAgQi5SLjsgY2l0YXRpb25fYXV0aG9yPUtub3RoZSAgRy47IGNpdGF0aW9uX2F1dGhvcj1WYXVnaG4gIFMuRi47IGNpdGF0aW9uX2F1dGhvcj1Jc2JlbGwgIFQuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVuZXJneSBGdWVsczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTQxNDktNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNveWJlYW4gc2VlZCB5aWVsZCBhbmQgcXVhbGl0eSBhcyBhIHJlc3BvbnNlIHRvIGZpZWxkIHBlbm55Y3Jlc3MgcmVzaWR1ZTsgY2l0YXRpb25fYXV0aG9yPVBoaXBwZW4gIFcuQi47IGNpdGF0aW9uX2F1dGhvcj1QaGlwcGVuICBNLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0yNzY3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ3JpY3VsdHVyYWwgc3VzdGFpbmFiaWxpdHkgYW5kIGludGVuc2l2ZSBwcm9kdWN0aW9uIHByYWN0aWNlczsgY2l0YXRpb25fYXV0aG9yPVRpbG1hbiAgRC47IGNpdGF0aW9uX2F1dGhvcj1DYXNzbWFuICBLLkcuOyBjaXRhdGlvbl9hdXRob3I9TWF0c29uICBQLkEuOyBjaXRhdGlvbl9hdXRob3I9TmF5bG9yICBSLjsgY2l0YXRpb25fYXV0aG9yPVBvbGFza3kgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE4OyAgY2l0YXRpb25fcGFnZXM9NjcxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcmtldCBjb3N0IG9mIHJlbmV3YWJsZSBqZXQgZnVlbCBhZG9wdGlvbiBpbiB0aGUgVW5pdGVkIFN0YXRlczsgY2l0YXRpb25fYXV0aG9yPVdpbmNoZXN0ZXIgIE4uOyBjaXRhdGlvbl9hdXRob3I9TWNDb25uYWNoaWUgIEQuOyBjaXRhdGlvbl9hdXRob3I9V29sbGVyc2hlaW0gIEMuOyBjaXRhdGlvbl9hdXRob3I9V2FpdHogIEkuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb3JtYW5jeSBicmVha2luZyBhbmQgc2VlZCBnZXJtaW5hdGlvbiBvZiB0aGUgYW5udWFsIHdlZWRzIFRobGFzcGkgYXJ2ZW5zZSwgRGVzY3VyYWluaWEgc29waGlhIGFuZCBNYWxjb2xtaWEgYWZyaWNhbmEgKEJyYXNzaWNhY2VhZSk7IGNpdGF0aW9uX2F1dGhvcj1LYXJpbW1vamVuaSAgSC5ULjsgY2l0YXRpb25fYXV0aG9yPVRhYWIgIEEuOyBjaXRhdGlvbl9hdXRob3I9UmFzaGlkaSAgQi47IGNpdGF0aW9uX2F1dGhvcj1CYXpyYWZzaGFuICBBLkguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBQcm90ZWN0LiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MTc5LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmVha2FnZSBvZiBzZWVkIGRvcm1hbmN5IG9mIGZpZWxkIHBlbm55Y3Jlc3MgKFRobGFzcGkgYXJ2ZW5zZSkgYnkgZ3Jvd3RoLXJlZ3VsYXRvcnMsIG5pdHJhdGUsIGFuZCBlbnZpcm9ubWVudGFsLWZhY3RvcnM7IGNpdGF0aW9uX2F1dGhvcj1TYWluaSAgSC5TLjsgY2l0YXRpb25fYXV0aG9yPUJhc3NpICBQLksuOyBjaXRhdGlvbl9hdXRob3I9R291ZGV5ICBKLlMuOyBjaXRhdGlvbl9hdXRob3I9U3BlbmNlciAgTS5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9V2VlZCBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9ODAyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dHJhY3Rpb24gb2YgcGVubnljcmVzcyAoVGhsYXNwaSBhcnZlbnNlIEwuKSBzZWVkIG9pbCBieSBmdWxsIHByZXNzaW5nOyBjaXRhdGlvbl9hdXRob3I9RXZhbmdlbGlzdGEgIFIuTC47IGNpdGF0aW9uX2F1dGhvcj1Jc2JlbGwgIFQuQS47IGNpdGF0aW9uX2F1dGhvcj1DZXJtYWsgIFMuQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZC4gQ3JvcHMgUHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz03Ni04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGRldmVsb3BtZW50IG9mIEFyYWJpZG9wc2lzIGFzIGEgbW9kZWwgcGxhbnQ7IGNpdGF0aW9uX2F1dGhvcj1Lb29ybm5lZWYgIE0uOyBjaXRhdGlvbl9hdXRob3I9TWVpbmtlICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz05MDktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgZGUgbm92byBmYXR0eSBhY2lkIHN5bnRoZXNpcyBpbiBtYXR1cmluZyBvaWxzZWVkcyBvZiBBcmFiaWRvcHNpczsgY2l0YXRpb25fYXV0aG9yPUJhdWQgIFMuOyBjaXRhdGlvbl9hdXRob3I9TGVwaW5pZWMgIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTQ0OC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG5ldHdvcmtzIHJlZ3VsYXRpbmcgQXJhYmlkb3BzaXMgc2VlZCBtYXR1cmF0aW9uLCBhZnRlci1yaXBlbmluZywgZG9ybWFuY3kgYW5kIGdlcm1pbmF0aW9uOyBjaXRhdGlvbl9hdXRob3I9SG9sZHN3b3J0aCAgTS5KLjsgY2l0YXRpb25fYXV0aG9yPUJlbnRzaW5rICBMLjsgY2l0YXRpb25fYXV0aG9yPVNvcHBlICBXLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzk7ICBjaXRhdGlvbl9wYWdlcz0zMy01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmxvd2VyaW5nIHRpbWUgY29udHJvbCBhbmQgYXBwbGljYXRpb25zIGluIHBsYW50IGJyZWVkaW5nOyBjaXRhdGlvbl9hdXRob3I9SnVuZyAgQy47IGNpdGF0aW9uX2F1dGhvcj1NdWxsZXIgIEEuRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTYzLTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRpbmcgaW5kdWNlZCBsb2NhbCBsZXNpb25zIElOIGdlbm9tZXMgKFRJTExJTkcpIGZvciBwbGFudCBmdW5jdGlvbmFsIGdlbm9taWNzOyBjaXRhdGlvbl9hdXRob3I9TWNDYWxsdW0gIEMuTS47IGNpdGF0aW9uX2F1dGhvcj1Db21haSAgTC47IGNpdGF0aW9uX2F1dGhvcj1HcmVlbmUgIEUuQS47IGNpdGF0aW9uX2F1dGhvcj1IZW5pa29mZiAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyMzsgIGNpdGF0aW9uX3BhZ2VzPTQzOS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVsZXRpb24tYmFzZWQgcmV2ZXJzZSBnZW5ldGljcyBpbiBNZWRpY2FnbyB0cnVuY2F0dWxhOyBjaXRhdGlvbl9hdXRob3I9Um9nZXJzICBDLjsgY2l0YXRpb25fYXV0aG9yPVdlbiAgSi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBSLjsgY2l0YXRpb25fYXV0aG9yPU9sZHJveWQgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBnZW5vbWUgZWRpdGluZyBtYWRlIGVhc3k6IHRhcmdldGVkIG11dGFnZW5lc2lzIGluIG1vZGVsIGFuZCBjcm9wIHBsYW50cyB1c2luZyB0aGUgQ1JJU1BSL0NhcyBzeXN0ZW07IGNpdGF0aW9uX2F1dGhvcj1CZWxoYWogIEsuOyBjaXRhdGlvbl9hdXRob3I9Q2hhcGFycm8tR2FyY2lhICBBLjsgY2l0YXRpb25fYXV0aG9yPUthbW91biAgUy47IGNpdGF0aW9uX2F1dGhvcj1OZWtyYXNvdiAgVi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2lhbGl6ZWQgYmluYXJ5IHZlY3RvciBmb3IgcGxhbnQgdHJhbnNmb3JtYXRpb246IGV4cHJlc3Npb24gb2YgdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIEFIQVMgZ2VuZSBpbiBOaWNvdGlhbmEgdGFiYWN1bTsgY2l0YXRpb25fYXV0aG9yPU9sc3pld3NraSAgTi5FLjsgY2l0YXRpb25fYXV0aG9yPU1hcnRpbiAgRi5CLjsgY2l0YXRpb25fYXV0aG9yPUF1c3ViZWwgIEYuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEwNzY1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBoeXNpY2FsLCBnZW5ldGljIGFuZCBmdW5jdGlvbmFsIHNlcXVlbmNlIGFzc2VtYmx5IG9mIHRoZSBiYXJsZXkgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBCYXJsZXkgR2Vub21lIFNlcXVlbmNpbmcgQ29uc29ydGl1bTsgY2l0YXRpb25fYXV0aG9yPU1heWVyICBLLkYuOyBjaXRhdGlvbl9hdXRob3I9V2F1Z2ggIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDkxOyAgY2l0YXRpb25fcGFnZXM9NzExLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcnViYmVyIHRyZWUgSGV2ZWEgYnJhc2lsaWVuc2lzOyBjaXRhdGlvbl9hdXRob3I9UmFobWFuICBBLlkuOyBjaXRhdGlvbl9hdXRob3I9VXNoYXJyYWogIEEuTy47IGNpdGF0aW9uX2F1dGhvcj1NaXNyYSAgQi5CLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TY2FmZm9sZGluZyBwcmUtYXNzZW1ibGVkIGNvbnRpZ3MgdXNpbmcgU1NQQUNFOyBjaXRhdGlvbl9hdXRob3I9Qm9ldHplciAgTS47IGNpdGF0aW9uX2F1dGhvcj1IZW5rZWwgIEMuVi47IGNpdGF0aW9uX2F1dGhvcj1KYW5zZW4gIEguSi47IGNpdGF0aW9uX2F1dGhvcj1CdXRsZXIgIEQuOyBjaXRhdGlvbl9hdXRob3I9UGlyb3Zhbm8gIFcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgYWxtb3N0IGNsb3NlZCBnZW5vbWVzIHdpdGggR2FwRmlsbGVyOyBjaXRhdGlvbl9hdXRob3I9Qm9ldHplciAgTS47IGNpdGF0aW9uX2F1dGhvcj1QaXJvdmFubyAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVI1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BS0VSOiBhbiBlYXN5LXRvLXVzZSBhbm5vdGF0aW9uIHBpcGVsaW5lIGRlc2lnbmVkIGZvciBlbWVyZ2luZyBtb2RlbCBvcmdhbmlzbSBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9Q2FudGFyZWwgIEIuTC47IGNpdGF0aW9uX2F1dGhvcj1Lb3JmICBJLjsgY2l0YXRpb25fYXV0aG9yPVJvYmIgIFMuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTg4LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IGNpdGF0aW9uX2F1dGhvcj1BcmFiaWRvcHNpcyBHZW5vbWUgSW5pdGlhdGl2ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwODsgIGNpdGF0aW9uX3BhZ2VzPTc5Ni04MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBseXJhdGEgZ2Vub21lIHNlcXVlbmNlIGFuZCB0aGUgYmFzaXMgb2YgcmFwaWQgZ2Vub21lIHNpemUgY2hhbmdlOyBjaXRhdGlvbl9hdXRob3I9SHUgIFQuVC47IGNpdGF0aW9uX2F1dGhvcj1QYXR0eW4gIFAuOyBjaXRhdGlvbl9hdXRob3I9QmFra2VyICBFLkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTQ3Ni04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgbWVzb3BvbHlwbG9pZCBjcm9wIHNwZWNpZXMgQnJhc3NpY2EgcmFwYTsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFguOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgSC47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGN1Y3VyYml0YWNpbnMgYW5kIGFzc2VtYmx5IG9mIGEgZHJhZnQgZ2Vub21lIGZvciBBcXVpbGFyaWEgYWdhbGxvY2hhOyBjaXRhdGlvbl9hdXRob3I9Q2hlbiAgQy5ILjsgY2l0YXRpb25fYXV0aG9yPUt1byAgVC5DLjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIE0uSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU3OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2Ygd2F0ZXJtZWxvbiAoQ2l0cnVsbHVzIGxhbmF0dXMpIGFuZCByZXNlcXVlbmNpbmcgb2YgMjAgZGl2ZXJzZSBhY2Nlc3Npb25zOyBjaXRhdGlvbl9hdXRob3I9R3VvICBTLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBKLjsgY2l0YXRpb25fYXV0aG9yPVN1biAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iU21pdCBBLiBIdWJsZXkgUi4gR3JlZW4gUC4gIDE5OTbigJMyMDEwICwgUmVwZWF0TWFza2VyIE9wZW4tMy4wLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXN1YWxpemluZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhIHdpdGggSkJyb3dzZTsgY2l0YXRpb25fYXV0aG9yPVdlc3Rlc3NvbiAgTy47IGNpdGF0aW9uX2F1dGhvcj1Ta2lubmVyICBNLjsgY2l0YXRpb25fYXV0aG9yPUhvbG1lcyAgSS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyaWVmLiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTcyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gc2FsdCB0b2xlcmFuY2UgZnJvbSB0aGUgZ2Vub21lIG9mIFRoZWxsdW5naWVsbGEgc2Fsc3VnaW5lYTsgY2l0YXRpb25fYXV0aG9yPVd1ICBILkouOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgSi5ZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xMjIxOS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9jZWR1cmUgZm9yIG1hcHBpbmcgYXJhYmlkb3BzaXMgbXV0YXRpb25zIHVzaW5nIGNvZG9taW5hbnQgZWNvdHlwZS1zcGVjaWZpYyBwY3ItYmFzZWQgbWFya2VyczsgY2l0YXRpb25fYXV0aG9yPUtvbmllY3pueSAgQS47IGNpdGF0aW9uX2F1dGhvcj1BdXN1YmVsICBGLk0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9sb2d5IGFuZCBiaW9jaGVtaXN0cnkgb2YgZ2x1Y29zaW5vbGF0ZXM7IGNpdGF0aW9uX2F1dGhvcj1IYWxraWVyICBCLkEuOyBjaXRhdGlvbl9hdXRob3I9R2Vyc2hlbnpvbiAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0zMDMtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsdWNvc2lub2xhdGUgbWV0YWJvbGlzbSBhbmQgaXRzIGNvbnRyb2w7IGNpdGF0aW9uX2F1dGhvcj1HcnViYiAgQy5ELjsgY2l0YXRpb25fYXV0aG9yPUFiZWwgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTg5LTEwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHNwZWNpZmllciBwcm90ZWlucyBpbiBnbHVjb3Npbm9sYXRlLWNvbnRhaW5pbmcgcGxhbnRzOyBjaXRhdGlvbl9hdXRob3I9S3VjaGVybmlnICBKLkMuOyBjaXRhdGlvbl9hdXRob3I9QnVyb3cgIE0uOyBjaXRhdGlvbl9hdXRob3I9V2l0dHN0b2NrICBVLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgdHlwZSBvZiBlbnp5bWF0aWMgY2xlYXZhZ2Ugb2YgbXVzdGFyZCBvaWwgZ2x1Y29zaWRlcy4gRm9ybWF0aW9uIG9mIGFsbHlsdGhpb2N5YW5hdGUgaW4gVGhsYXNwaSBhcnZlbnNlIEwgYW5kIGJlbnp5bHRoaW9jeW5hdGUgaW4gTGVwaWRpdW0gcnVkZXJhbGUgTCBhbmQgTGVwaWRpdW0gc2F0aXZ1bSBMOyBjaXRhdGlvbl9hdXRob3I9R21lbGluICBSLjsgY2l0YXRpb25fYXV0aG9yPVZpcnRhbmVuICBBLkkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BY3RhIENoZW0uIFNjYW5kLjsgIGNpdGF0aW9uX3llYXI9MTk1OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE0NzQtMTQ3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0aGlvY3lhbmF0ZS1mb3JtaW5nIHByb3RlaW4gZ2VuZXJhdGVzIG11bHRpcGxlIHByb2R1Y3RzIHVwb24gYWxseWxnbHVjb3Npbm9sYXRlIGJyZWFrZG93biBpbiBUaGxhc3BpIGFydmVuc2U7IGNpdGF0aW9uX2F1dGhvcj1LdWNoZXJuaWcgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1CYWNrZW5rb2hsZXIgIEEuOyBjaXRhdGlvbl9hdXRob3I9THViYmVja2UgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnVyb3cgIE0uOyBjaXRhdGlvbl9hdXRob3I9V2l0dHN0b2NrICBVLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz0xNjk5LTcwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0cyBvZiBjb2xkLXByZXNzaW5nIGFuZCBzZWVkIGNvb2tpbmcgb24gZnVuY3Rpb25hbCBwcm9wZXJ0aWVzIG9mIHByb3RlaW4gaW4gcGVubnljcmVzcyAoVGhsYXNwaSBhcnZlbnNlIEwuKSBzZWVkIGFuZCBwcmVzcyBjYWtlczsgY2l0YXRpb25fYXV0aG9yPUhvamlsbGEtRXZhbmdlbGlzdGEgIE0uUC47IGNpdGF0aW9uX2F1dGhvcj1FdmFuZ2VsaXN0YSAgUi5MLjsgY2l0YXRpb25fYXV0aG9yPUlzYmVsbCAgVC5BLjsgY2l0YXRpb25fYXV0aG9yPVNlbGxpbmcgIEcuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZC4gQ3JvcHMgUHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0yMjMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0cmFjdGlvbiBvZiBwcm90ZWlucyBmcm9tIHBlbm55Y3Jlc3Mgc2VlZHMgYW5kIHByZXNzIGNha2U7IGNpdGF0aW9uX2F1dGhvcj1TZWxsaW5nICBHLlcuOyBjaXRhdGlvbl9hdXRob3I9SG9qaWxsYS1FdmFuZ2VsaXN0YSAgTS5QLjsgY2l0YXRpb25fYXV0aG9yPUV2YW5nZWxpc3RhICBSLkwuOyBjaXRhdGlvbl9hdXRob3I9SXNiZWxsICBULjsgY2l0YXRpb25fYXV0aG9yPVByaWNlICBOLjsgY2l0YXRpb25fYXV0aG9yPURvbGwgIEsuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZC4gQ3JvcHMgUHJvZC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0xMTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TU9ESUZJRUQgVkFDVU9MRSBQSEVOT1RZUEUxIGlzIGFuIEFyYWJpZG9wc2lzIG15cm9zaW5hc2UtYXNzb2NpYXRlZCBwcm90ZWluIGludm9sdmVkIGluIGVuZG9tZW1icmFuZSBwcm90ZWluIHRyYWZmaWNraW5nOyBjaXRhdGlvbl9hdXRob3I9QWdlZSAgQS5FLjsgY2l0YXRpb25fYXV0aG9yPVN1cnBpbiAgTS47IGNpdGF0aW9uX2F1dGhvcj1Tb2huICBFLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTI7ICBjaXRhdGlvbl9wYWdlcz0xMjAtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5lIGNvbnRyb2xsaW5nIHRoZSBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgRVBJVEhJT1NQRUNJRklFUiBNT0RJRklFUjEgYWx0ZXJzIGdsdWNvc2lub2xhdGUgaHlkcm9seXNpcyBhbmQgaW5zZWN0IHJlc2lzdGFuY2UgaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1PYmVyICBKLkEuOyBjaXRhdGlvbl9hdXRob3I9S2xpZWJlbnN0ZWluICBELkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTUyNC0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFuZCB0YW5kZW0gZHVwbGljYXRlIHJldGVudGlvbiBmYWNpbGl0YXRlZCBnbHVjb3Npbm9sYXRlIHBhdGh3YXkgZGl2ZXJzaWZpY2F0aW9uIGluIHRoZSBtdXN0YXJkIGZhbWlseTsgY2l0YXRpb25fYXV0aG9yPUhvZmJlcmdlciAgSi5BLjsgY2l0YXRpb25fYXV0aG9yPUx5b25zICBFLjsgY2l0YXRpb25fYXV0aG9yPUVkZ2VyICBQLlAuOyBjaXRhdGlvbl9hdXRob3I9Q2hyaXMgUGlyZXMgIEouOyBjaXRhdGlvbl9hdXRob3I9RXJpYyBTY2hyYW56ICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMTU1LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OUlQvUFRSIHRyYW5zcG9ydGVycyBhcmUgZXNzZW50aWFsIGZvciB0cmFuc2xvY2F0aW9uIG9mIGdsdWNvc2lub2xhdGUgZGVmZW5jZSBjb21wb3VuZHMgdG8gc2VlZHM7IGNpdGF0aW9uX2F1dGhvcj1Ob3VyLUVsZGluICBILkguOyBjaXRhdGlvbl9hdXRob3I9QW5kZXJzZW4gIFQuRy47IGNpdGF0aW9uX2F1dGhvcj1CdXJvdyAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODg7ICBjaXRhdGlvbl9wYWdlcz01MzEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpb24gb2YgYmlvc3ludGhlc2lzIGFuZCBsb25nLWRpc3RhbmNlIHRyYW5zcG9ydCBlc3RhYmxpc2ggb3JnYW4tc3BlY2lmaWMgZ2x1Y29zaW5vbGF0ZSBwcm9maWxlcyBpbiB2ZWdldGF0aXZlIEFyYWJpZG9wc2lzOyBjaXRhdGlvbl9hdXRob3I9QW5kZXJzZW4gIFQuRy47IGNpdGF0aW9uX2F1dGhvcj1Ob3VyLUVsZGluICBILkguOyBjaXRhdGlvbl9hdXRob3I9RnVsbGVyICBWLkwuOyBjaXRhdGlvbl9hdXRob3I9T2xzZW4gIEMuRS47IGNpdGF0aW9uX2F1dGhvcj1CdXJvdyAgTS47IGNpdGF0aW9uX2F1dGhvcj1IYWxraWVyICBCLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzEzMy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBvZiBET0cxLCBhIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBjb250cm9sbGluZyBzZWVkIGRvcm1hbmN5IGluIEFyYWJpZG9wc2lzOyBjaXRhdGlvbl9hdXRob3I9QmVudHNpbmsgIEwuOyBjaXRhdGlvbl9hdXRob3I9Sm93ZXR0ICBKLjsgY2l0YXRpb25fYXV0aG9yPUhhbmhhcnQgIEMuSi47IGNpdGF0aW9uX2F1dGhvcj1Lb29ybm5lZWYgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTE3MDQyLTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdGltZSByZXF1aXJlZCBmb3IgZG9ybWFuY3kgcmVsZWFzZSBpbiBhcmFiaWRvcHNpcyBpcyBkZXRlcm1pbmVkIGJ5IERFTEFZIE9GIEdFUk1JTkFUSU9OMSBwcm90ZWluIGxldmVscyBpbiBmcmVzaGx5IGhhcnZlc3RlZCBzZWVkczsgY2l0YXRpb25fYXV0aG9yPU5ha2FiYXlhc2hpICBLLjsgY2l0YXRpb25fYXV0aG9yPUJhcnRzY2ggIE0uOyBjaXRhdGlvbl9hdXRob3I9WGlhbmcgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjgyNi0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgY29sb25pYWwgY2hvcmRhdGUsIEJvdHJ5bGx1cyBTY2hsb3NzZXJpOyBjaXRhdGlvbl9hdXRob3I9Vm9za29ib3luaWsgIEEuOyBjaXRhdGlvbl9hdXRob3I9TmVmZiAgTi5GLjsgY2l0YXRpb25fYXV0aG9yPVNhaG9vICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWxpZmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUwMDU2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm90eXBpbmctYnktc2VxdWVuY2luZyBmb3IgcGxhbnQgYnJlZWRpbmcgYW5kIGdlbmV0aWNzOyBjaXRhdGlvbl9hdXRob3I9UG9sYW5kICBKLkEuOyBjaXRhdGlvbl9hdXRob3I9UmlmZSAgVC5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lIFVTOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05Mi0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvdWJsZSBkaWdlc3QgUkFEc2VxOiBhbiBpbmV4cGVuc2l2ZSBtZXRob2QgZm9yIGRlIG5vdm8gU05QIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyBpbiBtb2RlbCBhbmQgbm9uLW1vZGVsIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1QZXRlcnNvbiAgQi5LLjsgY2l0YXRpb25fYXV0aG9yPVdlYmVyICBKLk4uOyBjaXRhdGlvbl9hdXRob3I9S2F5ICBFLkguOyBjaXRhdGlvbl9hdXRob3I9RmlzaGVyICBILlMuOyBjaXRhdGlvbl9hdXRob3I9SG9la3N0cmEgIEguRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzcxMzUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXhvbm9taWMgYW5kIHBoeWxvZ2VuZXRpYyBldmFsdWF0aW9uIG9mIHRoZSBBbWVyaWNhbiDigJhUaGxhc3Bp4oCZIHNwZWNpZXM6IGlkZW50aXR5IGFuZCByZWxhdGlvbnNoaXAgdG8gdGhlIEV1cmFzaWFuIGdlbnVzIE5vY2NhZWEgKEJyYXNzaWNhY2VhZSk7IGNpdGF0aW9uX2F1dGhvcj1Lb2NoICBNLjsgY2l0YXRpb25fYXV0aG9yPUFMLVNoZWhiYXogIEkuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zNzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8yLzEyMS85NTA5MTcvZHN1MDQ1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBGaWVsZCBwZW5ueWNyZXNzIChUaGxhc3BpIGFydmVuc2UgTC4pIGlzIGJlaW5nIGRvbWVzdGljYXRlZCBhcyBhIG5ldyB3aW50ZXIgY292ZXIgY3JvcCBhbmQgYmlvZnVlbCBzcGVjaWVzIGZvciB0aGUgTWlkd2VzdGVybiBVbml0ZWQgU3RhdGVzIHRoYXQgY2FuICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMyNjM2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFy4kg5wvRHnl+qYwMjax3wn5qyG" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f121%2f332636" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f121%2f332636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12675" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/2/1/m_cover.gif?Expires=2147483647&amp;Signature=hysmEB~9lawj6xVWBe94BgZjn-52T1zCky0NQUfqUC0oo9butAuGx9Fz2~81AfaeIQBOgbncwhIoztWl9zrXZ9mEGvCQ2~LcG7Bwa9XqxluXvpqIEIlh39GN1RZQJp-sy-auaYL8YH84vMwT-GxzAQNPX9jbQecaDYg7WUclLlJZPuYKEDMdLYnPr9jImIeuzcEcFjA-Gxa7uDWn41TpDklWF8x96-aaWXRMdYSdDJNkvZ9w7VW0SpJ2tpqGK8F2~4c0ztdARneMy0YeiqNRhGHhvPbXt-fJTnFFeYw9w3qKaTnu6ljh7eZdi12aZINS6WG4axZPtcCwjT6-Li3LBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/2">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4588726">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588726">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588728">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588728">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588734">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588734">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588747">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588747">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588777">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588777">Conflict of interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588779">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588779">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588781">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588781">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588783">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588783">Data deposition</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588786">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588786">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4588788">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4588788">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/22/2/133/332691">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A draft genome of field pennycress (<em>Thlaspi arvense</em>) provides tools for the domestication of a new winter biofuel crop
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kevin M. Dorn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">University of Minnesota</div>, <div class="addr-line">Saint Paul, MN 55108</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kevin+M.+Dorn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dorn K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dorn K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kevin M. Dorn</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Johnathon D. Fankhauser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">University of Minnesota</div>, <div class="addr-line">Saint Paul, MN 55108</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Johnathon+D.+Fankhauser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fankhauser J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fankhauser J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Johnathon D. Fankhauser</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donald L. Wyse</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Agronomy and Plant Genetics</div>, <div class="insititution">University of Minnesota</div>, <div class="addr-line">Saint Paul, MN 55108</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Donald+L.+Wyse">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wyse D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wyse D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Donald L. Wyse</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. David Marks</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">University of Minnesota</div>, <div class="addr-line">Saint Paul, MN 55108</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 612-625-6737. Fax. +1 612-625-5754. E-mail: <a href="mailto:marks004@umn.edu" target="_blank">marks004@umn.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+David+Marks">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Marks M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Marks M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">M. David Marks</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Deceased on 17 August 2014</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 121–131, <a href='https://doi.org/10.1093/dnares/dsu045'>https://doi.org/10.1093/dnares/dsu045</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 January 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 October 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 December 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="332636" href="/dnaresearch/article-pdf/22/2/121/950917/dsu045.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kevin M. Dorn, Johnathon D. Fankhauser, Donald L. Wyse, M. David Marks;  A draft genome of field pennycress (<em>Thlaspi arvense</em>) provides tools for the domestication of a new winter biofuel crop, <em>DNA Research</em>, Volume 22, Issue 2, 1 April 2015, Pages 121–131, <a href='https://doi.org/10.1093/dnares/dsu045'>https://doi.org/10.1093/dnares/dsu045</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/332636?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/332636?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20draft%20genome%20of%20field%20pennycress%20(%26lt%3Bem%26gt%3BThlaspi%20arvense%26lt%3B%2Fem%26gt%3B)%20provides%20tools%20for%20the%20domestication%20of%20a%20new%20winter%20biofuel%20crop&amp;publicationDate=2015-01-27&amp;volumeNum=22&amp;issueNum=2&amp;author=Dorn%2C%20Kevin%20M.%3B%20Fankhauser%2C%20Johnathon%20D.&amp;startPage=121&amp;endPage=131&amp;contentId=10.1093%2Fdnares%2Fdsu045&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f121%2f332636" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4588726 id="4588726" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Field pennycress (<em>Thlaspi arvense</em> L.) is being domesticated as a new winter cover crop and biofuel species for the Midwestern United States that can be double-cropped between corn and soybeans. A genome sequence will enable the use of new technologies to make improvements in pennycress. To generate a draft genome, a hybrid sequencing approach was used to generate 47 Gb of DNA sequencing reads from both the Illumina and PacBio platforms. These reads were used to assemble 6,768 genomic scaffolds. The draft genome was annotated using the MAKER pipeline, which identified 27,390 predicted protein-coding genes, with almost all of these predicted peptides having significant sequence similarity to Arabidopsis proteins. A comprehensive analysis of pennycress gene homologues involved in glucosinolate biosynthesis, metabolism, and transport pathways revealed high sequence conservation compared with other Brassicaceae species, and helps validate the assembly of the pennycress gene space in this draft genome. Additional comparative genomic analyses indicate that the knowledge gained from years of basic Brassicaceae research will serve as a powerful tool for identifying gene targets whose manipulation can be predicted to result in improvements for pennycress.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;field pennycress&quot;">field pennycress</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Thlaspi arvense&quot;"><em>Thlaspi arvense</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo assembly&quot;"><em>de novo</em> assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome sequencing&quot;">whole genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=4588728 id="4588728" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Next-generation sequencing (NGS) has enabled the characterization and comparison of whole genomes for a growing list of plant species.<sup><a reveal-id="DSU045C1  DSU045C5" data-open="DSU045C1  DSU045C5" class="link link-ref link-reveal xref-bibr">1–5</a></sup> This same technology is being used to speed up and facilitate the breeding of crop plants.<sup><a reveal-id="DSU045C3" data-open="DSU045C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU045C6  DSU045C8" data-open="DSU045C6  DSU045C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> NGS also has the potential to enable new approaches to domesticate crops.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a></sup> One species targeted for domestication is <em>Thlaspi arvense</em> (field pennycress, pennycress herein). Pennycress is a member of the Brassicaceae in the tribe Thlaspideae native to Eastern Europe.<sup><a reveal-id="DSU045C10" data-open="DSU045C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Pennycress plants are diploid, propagate largely via self-fertilization, and have a 1C DNA content of 539 Mb.<sup><a reveal-id="DSU045C12" data-open="DSU045C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Naturalized populations are distributed worldwide, including North America, South America, and Australia.<sup><a reveal-id="DSU045C10" data-open="DSU045C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Previous molecular studies showed that the original <em>Thlaspi</em> genus was polyphylogenic, and several members of the <em>Thlaspi</em> genus were kept in a monophylogenic Thlaspideae tribe.<sup><a reveal-id="DSU045C13" data-open="DSU045C13" class="link link-ref link-reveal xref-bibr">13</a></sup><em>Thlaspi caerulescens</em>, which is a well-studied heavy metal accumulator,<sup><a reveal-id="DSU045C14" data-open="DSU045C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU045C15" data-open="DSU045C15" class="link link-ref link-reveal xref-bibr">15</a></sup> was moved to the genus <em>Noccaea</em> during this reorganization.<sup><a reveal-id="DSU045C13" data-open="DSU045C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSU045C16" data-open="DSU045C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Two members of the original <em>Thlaspi</em> genus, <em>Thlaspi ceratocarpum</em> and <em>Thlaspi alliaceum</em> (roadside pennycress), are in the Thlaspideae along with <em>Alliaria petiolata</em> (garlic mustard) and <em>Peltaria alliacea</em> (garlic cress) and several other species. The Brassicaceae are divided into three lineages, and the members of the Thlaspideae are in lineage II, which includes the <em>Brassica</em> genus, but not <em>Arabidopsis</em> (lineage I).<sup><a reveal-id="DSU045C17" data-open="DSU045C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Similar to <em>Brassica napus</em> and <em>Brassica rapa</em>, pennycress produces seeds with high oil content (30% by weight).<sup><a reveal-id="DSU045C18" data-open="DSU045C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Field pennycress is being developed as a new winter cover crop and biodiesel feedstock for the Midwestern United States that can be seeded into standing corn.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU045C19  DSU045C21" data-open="DSU045C19  DSU045C21" class="link link-ref link-reveal xref-bibr">19–21</a></sup> As a winter annual, pennycress germinates in the fall and overwinters as a vegetative rosette. It exhibits extreme winter hardiness, surviving the harsh winters that are common to the Canadian Plains and Midwestern United States.<sup><a reveal-id="DSU045C10" data-open="DSU045C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Pennycress flowers and matures early in the spring, producing up to 1,300 kg/hectare of seed that can be harvested in time for planting an additional summer annual crop of soybeans.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU045C22" data-open="DSU045C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The oils found in pennycress seed are suitable for biodiesel production,<sup><a reveal-id="DSU045C21" data-open="DSU045C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and the remaining seed meal is high in protein that could serve as an additional revenue source for farmers.<sup><a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> As a winter cover, pennycress would provide important ecosystem services. In much of the Midwest, the ground lies barren from late fall until early summer, which can increase nutrient runoff and soil erosion.<sup><a reveal-id="DSU045C23" data-open="DSU045C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The addition of a fall-planted pennycress cover crop should reduce nutrient leaching and erosion, which would help sustain current farming practices. In the United States alone, over 16 million hectares currently used in the corn to soybean rotation could be used for pennycress production without displacing current food production systems.<sup><a reveal-id="DSU045C20" data-open="DSU045C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSU045C24" data-open="DSU045C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>As a weedy species, there are many challenges that will need to be addressed in order to convert pennycress into a new crop species.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Pennycress already has many natural attributes such as the high seed yield and oil content described above. However, earlier maturing varieties are needed to ensure that pennycress can be harvested early, allowing for full-season soybean production. Seed dormancy also is an issue,<sup><a reveal-id="DSU045C25" data-open="DSU045C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU045C26" data-open="DSU045C26" class="link link-ref link-reveal xref-bibr">26</a></sup> as is common in many weed species; we have observed highly variable germination rates in preliminary field trials. Pennycress oil and protein meal are not currently suitable for animal or human consumption.<sup><a reveal-id="DSU045C27" data-open="DSU045C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Making pennycress seeds edible would add value to the crop. In addition, the fatty acids contained in the oil are adequate for conversion to biodiesel, but can be further optimized.<sup><a reveal-id="DSU045C18" data-open="DSU045C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSU045C21" data-open="DSU045C21" class="link link-ref link-reveal xref-bibr">21</a></sup> It is unclear whether traditional breeding programmes can address these concerns, as there may be insufficient natural variation in wild populations.</p><p><em>Arabidopsis thaliana</em> has been used as a key organism to address questions concerning plant development for the past 30 yrs.<sup><a reveal-id="DSU045C28" data-open="DSU045C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Arabidopsis research has resulted in an understanding of many plant developmental processes, such as the control of time to flower, of seed maturation, and of seed oil synthesis.<sup><a reveal-id="DSU045C29  DSU045C31" data-open="DSU045C29  DSU045C31" class="link link-ref link-reveal xref-bibr">29–31</a></sup> During the course of these studies, many mutations have been identified in Arabidopsis that confer what can be considered agronomically desirable phenotypes.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The function of pennycress genes can be predicted on the basis of their similarity to genes with known function in Arabidopsis.<sup><a reveal-id="DSU045C9" data-open="DSU045C9" class="link link-ref link-reveal xref-bibr">9</a></sup> With this information, it should be possible to identify mutations that confer desirable traits in pennycress using mutation-based breeding tools such as TILLING,<sup><a reveal-id="DSU045C32" data-open="DSU045C32" class="link link-ref link-reveal xref-bibr">32</a></sup> DeTILLING,<sup><a reveal-id="DSU045C33" data-open="DSU045C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and CRISPR-Cas.<sup><a reveal-id="DSU045C34" data-open="DSU045C34" class="link link-ref link-reveal xref-bibr">34</a></sup> To enable the use of these technologies and to take advantage of the information derived from research on Arabidopsis and other Brassica species, a genomic sequence is needed for pennycress.</p><p>To generate a draft genome for pennycress, a natural population of plants was isolated from a roadside near Coates, Minnesota called MN106. Little is known about natural pennycress populations at the genome sequence level. In this report, we have examined this population at the genome level using several different NGS strategies to <em>de novo</em> assemble a draft genome. This assembly captures the vast majority of pennycress gene space as predicted by comparison with other Brassica species. The draft genome detailing the pennycress gene space has allowed the function of pennycress genes to be predicted on the basis of their similarity to genes with known function in Arabidopsis. In addition, the analysis revealed that even in mixed natural populations, the selfing mode of reproduction maintains individual plants in a highly homozygous state.</p>                            <h2 scrollto-destination=4588734 id="4588734" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4588735 id="4588735" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p><em>Thlaspi arvense</em> line MN106 has been previously described and originates from Coates, MN.<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Seed from a MN106 bulk planting was germinated on moist Berger BM2 germination mix (Berger Inc., <a class="link link-uri" href="http://www.berger.ca">www.berger.ca</a>), stratified at 4°C for 7 days, and grown in climate-controlled growth chambers at the University of Minnesota (21°C, 16 h/8 h day/night cycles at 100 μmol/m<sup>2</sup>/s PAR). Individual plants were transplanted to 4-inch pots 2 weeks after germination. Six-week-old plants with established rosettes were vernalized at 4°C for 30 days in the dark. After vernalization, plants were returned to growth chambers. In all, nine plants were selected for DNA extraction. After tissue harvest, the same nine plants were maintained, and seeds were individually collected from each plant.</p>                            <h3 scrollto-destination=4588737 id="4588737" class="section-title" data-legacy-id=s2b>2.2. Genomic DNA isolation and DNA sequencing</h3>
<p>To enrich the total amount of nuclear DNA sequenced, nuclei were purified from young leaf tissue using a series of density gradient centrifugation steps using an adapted protocol.<sup><a reveal-id="DSU045C35" data-open="DSU045C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The full protocol for genomic DNA isolation is listed in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Methods</a>. Illumina sequencing (100 bp paired-end library run on Illumina HiSeq 2000, 250 bp paired-end library run on Illumina MiSeq, 50 bp long-jump mate pair libraries run on Illumina HiSeq 2000) was completed at the University of Minnesota Genomics Center. Pacific Biosciences (PacBio RSII) sequencing was completed at the Mayo Clinic Molecular Biology Core (Rochester, MN). All raw sequencing files have been submitted to NCBI Sequence Read Archive under accession number SRP033211. FASTQ files from all sequencing runs were imported and subjected to quality control using the Sequencing QC Report tool in CLC Genomics Workbench Version 6.5 (CLC Bio, <a class="link link-uri" href="http://www.clcbio.com">www.clcbio.com</a>). Details on DNA sequencing library preparation and quality control parameters are described in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Methods</a>.</p>                            <h3 scrollto-destination=4588739 id="4588739" class="section-title" data-legacy-id=s2c>2.3. Genome assembly, scaffolding, and annotation</h3>
<p>The pennycress draft genome was assembled and annotated using two desktop computers built specifically for this purpose. The components and specifications of these computers are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Genome sequencing reads were trimmed and <em>de novo</em> assembled in CLC Genomics Workbench Version 6.5 (CLC Bio, <a class="link link-uri" href="http://www.clcbio.com">www.clcbio.com</a>). The CLC assembler has previously been used in assembling complex plant genomes, such as the barley (<em>Hordeum vulgare</em>)<sup><a reveal-id="DSU045C36" data-open="DSU045C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and rubber tree (<em>Hevea brasiliensis</em>)<sup><a reveal-id="DSU045C37" data-open="DSU045C37" class="link link-ref link-reveal xref-bibr">37</a></sup> genome projects. Additionally, the CLC assembler has a low Random Access Memory (RAM) requirement, opposed to other popular <em>de Bruijn</em> graph assemblers that can require hundreds of gigabytes of RAM. Initial assembled scaffolds over 1000 bp from the CLC assembly were scaffolded using SSPACE,<sup><a reveal-id="DSU045C38" data-open="DSU045C38" class="link link-ref link-reveal xref-bibr">38</a></sup> and remaining gaps were filled using GAPFILLER.<sup><a reveal-id="DSU045C39" data-open="DSU045C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The genome assembly was annotated using the MAKER pipeline.<sup><a reveal-id="DSU045C40" data-open="DSU045C40" class="link link-ref link-reveal xref-bibr">40</a></sup> A full description of assembly and annotation parameters is available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Methods</a>.</p>                            <h3 scrollto-destination=4588741 id="4588741" class="section-title" data-legacy-id=s2d>2.4. Comparative genomics</h3>
<p>Comparative studies on the structural arrangement and synteny of the pennycress genome assembly were performed using SynMap (<a class="link link-uri" href="http://www.genomeevolution.org/CoGe/">www.genomeevolution.org/CoGe/</a>). To examine the synteny between the <em>Eutrema salsugineum</em> and <em>Thlaspi arvense</em> genomes, seven pseudochromosome sequences were constructed by concatenating the corresponding genomic scaffolds.<sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Thlaspi arvense</em> genomic scaffolds &gt;75 kb in length were compared with the seven <em>E. salsugineum</em> pseudochromosomes using a Syntenic Path Assembly in SynMap (DAGChainer – Relative Gene order, −D = 20, −A = 5, skip random/unknown chromosomes). BLASTp comparisons of the 27,390 predicted pennycress peptides were performed in CLC Genomics Workbench using the predicted peptide databases for <em>A. thaliana</em>,<sup><a reveal-id="DSU045C41" data-open="DSU045C41" class="link link-ref link-reveal xref-bibr">41</a></sup><em>Arabidopsis lyrata</em>,<sup><a reveal-id="DSU045C42" data-open="DSU045C42" class="link link-ref link-reveal xref-bibr">42</a></sup><em>B. rapa</em>,<sup><a reveal-id="DSU045C43" data-open="DSU045C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>Capsella rubella</em>,<sup><a reveal-id="DSU045C5" data-open="DSU045C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and <em>E. salsugineum</em><sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> using the following BLAST parameters: Expectation Value = 10, Word Size = 7, Filter Low Complexity = Yes, Protein Matrix = BLOSUM62, Gap Existence = 11, Gap Extension = 1<em>.</em> Peptide sequences for <em>A. thaliana</em> were obtained from <a class="link link-uri" href="http://ftp://ftp.arabidopsis.org/home/tair/Proteins/TAIR10_protein_lists/TAIR10_pep_20101214">ftp://ftp.arabidopsis.org/home/tair/Proteins/TAIR10_protein_lists/TAIR10_pep_20101214</a>. Sequences for the remaining species were obtained from Phytozome (<a class="link link-uri" href="http://www.phytozome.net">www.phytozome.net</a>). BLASTn analyses of the previously published pennycress transcriptome<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> against the genome assembly were performed using the following parameters: Match Cost = 2, Mismatch Cost = 3, Gap Existence = 2, Gap Extension = 2, Expectation Value = 10, Filter Low Complexity = Yes, Maximum Number of Hits = 15.</p>                            <h3 scrollto-destination=4588743 id="4588743" class="section-title" data-legacy-id=s2e>2.5. Read mapping and variant detection for cleaved amplified polymorphic sequence marker design</h3>
<p>Trimmed and filtered sequencing reads from the Illumina HiSeq 2000 100 bp paired-end experiment were re-mapped to the genome assembly to identify potential heterozygosity or variation in the MN106 accession sequenced. Reads were mapped to the assembly in CLC Genomics Workbench using the ‘Map Reads to Reference’ tool (masking mode = no masking, mismatch cost = 3, insertion cost = 3, deletion cost = 3, length fraction = 0.95, similarity fraction = 0.95, global alignment = no, auto detect paired distances = yes, non-specific match handling = map randomly) and variants called using the Probabilistic Variant Detection tool (ignore non-specific matches = yes, ignore broken pairs = yes, minimum coverage = 10, variant probability = 90, require both forward and reverse reads = no, maximum expected variants = 2, ignore quality scores = no). Sites declared as single-nucleotide polymorphisms (SNPs) were manually examined to identify cleaved amplified polymorphic sequence (CAPS) markers in which one variant was a member of a six base recognition site for a DNA restriction endonuclease, and the other variant resulted in the loss of the restriction site. Four sites were identified, and primers were designed to amplify these regions for CAPS analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). As a control, the primers were designed to flank both a CAPS site and a conserved restriction site that would be cleaved in both variants. DNA was isolated using the Mag-Bind EZ Plant DNA Kit (Omega BioTek, <a class="link link-uri" href="http://www.omegabiotek.com">www.omegabiotek.com</a>). Polymerase chain reactions were performed using Q5 High Fidelity DNA Polymerase (New England Biolabs, <a class="link link-uri" href="http://www.neb.com">www.neb.com</a>) with these DNA extracts and the corresponding CAPS primers to confirm the potential variants.</p>                            <h3 scrollto-destination=4588745 id="4588745" class="section-title" data-legacy-id=s2f>2.6. Comparative analysis of genes involved in glucosinolate metabolism and transport</h3>
<p><em>A. thaliana</em>, <em>B. rapa</em>, and <em>Brassica oleraceae</em> genes previously identified to be involved in glucosinolate biosynthesis, breakdown, and transport were derived from a previous study.<sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Alignments of predicted peptide sequences for each gene were performed in CLC Genomics Workbench using the Create Alignment tool (Gap open cost = 10, Gap extension cost = 1, End gap cost = Free, Alignment mode = Very accurate). Neighbour Joining trees were created in CLC Genomics Workbench (Protein distance measure = Jukes-Cantor, Perform bootstrap analysis = Yes, Replicates = 100). To examine expression levels of predicted gene models, Illumina RNAseq reads from the previously published transcriptome assembly<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> were trimmed and filtered (Illumina TruSeq Trim Adaptor 3, Ambiguous Trim = Yes, Ambiguous Limit = 2, Quality Trim = Yes, Quality Limit = 0.05, Also search on reversed sequence = Yes, Remove 5′ terminal nucleotides = Yes, Number of 5′ terminal nucleotides to remove = 10, Remove 3′ terminal nucleotides = no, Save broken pairs = Yes), and mapped to the annotated draft genome (Mapping Type = Map to gene regions only, Maximum number of hits for a read = 10, Strand Specific = Both, Count paired read as two = Yes, Expression Value = RPKM, Reference type = Genome annotated with genes and transcripts, Global alignment = no, Auto detect paired distances = Yes, Similarity fraction = 0.8, Length Fraction = 0.8, Mismatch cost = 2, Insertion cost = 3, Deletion cost = 3).</p>                            <h2 scrollto-destination=4588747 id="4588747" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=4588748 id="4588748" class="section-title" data-legacy-id=s3a>3.1. Genome sequencing and assembly</h3>
<p>Pennycress is a diploid species with a haploid number of seven chromosomes and a genome size of 539 Mb.<sup><a reveal-id="DSU045C12" data-open="DSU045C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In order to isolate a sufficient quantity of nuclear DNA for the various sequencing libraries used, nuclear DNA was isolated from nine plants derived from an MN106 isolate that had been maintained in the lab for several generations. This DNA was sequenced using both the Illumina and PacBio platforms, generating over 47 Gb of sequencing data representing over 87× coverage of the predicted genome size (Table <a reveal-id="DSU045TB1" data-open="DSU045TB1" class="link link-reveal link-table xref-fig">1</a>). Illumina HiSeq 2000 and MiSeq reads were <em>de novo</em> assembled using the <em>de Bruijn</em> graph-based CLC Genomics Workbench assembler. PacBio reads were not used to create the <em>de Bruijn</em> graph, but instead they were only used to resolve ambiguities during the graph building stage. </p>                        <a id="4588750" scrollto-destination="4588750"></a>
<div content-id="DSU045TB1" class="table-modal"><div class="table-wrap"><div id="DSU045TB1" class="table-wrap-title" data-id="DSU045TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genome sequencing, assembly, and annotation statistics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>Number of reads </th><th>No. of nucleotides after QC (Mb) </th></tr></thead><tbody><tr><td colspan="3"><em>Thlaspi arvense</em> genome sequencing </td></tr><tr><td> Illumina HiSeq 2000 (2 × 100 bp paired end) </td><td>352,394,426 </td><td>33,190.50 </td></tr><tr><td rowspan="2"> Illumina MiSeq (2 × 250 bp paired end) </td><td>6,291,688 (merged) </td><td>2,589.33 </td></tr><tr><td>8,548,686 (unmerged) </td><td>1,904.90 </td></tr><tr><td> Illumina HiSeq 2000 (2 × 50 bp Mate Pair)<br />2, 3, 5 kb inserts </td><td>209,815,249 </td><td>9,550.66 </td></tr><tr><td> PacBio SMRT cell (four cells) </td><td>110,751 </td><td>214.78 </td></tr><tr><td> </td><td>Scaffolds </td><td>Contigs in scaffolds </td></tr><tr><td colspan="3"><em>T. arvense</em> genome assembly </td></tr><tr><td> Number </td><td>6,768 </td><td>44,109 </td></tr><tr><td> Mean length (bp) </td><td>50,681 </td><td>7,375 </td></tr><tr><td> N50 (bp) </td><td>140,815 </td><td>21,096 </td></tr><tr><td> Total assembly length (bp) </td><td>343,012,389 </td><td>325,295,785 </td></tr><tr><td colspan="3"><em>T. arvense</em> genome annotation </td></tr><tr><td> Number of gene models </td><td>27,390 </td><td> </td></tr><tr><td>  Gene models highly similar to TAIR10 peptide (<em>e</em> &lt; 1 × 10<sup>−5</sup> and &gt;70% positive per cent) </td><td>85.94% </td><td> </td></tr><tr><td> GC content </td><td>37.99% </td><td> </td></tr><tr><td> Repetitive DNA sequence </td><td>24.38% </td><td> </td></tr><tr><td>  Retroelements (20.94%) </td><td>78,812 </td><td> </td></tr><tr><td>  DNA transposons (1.6%) </td><td>12,382 </td><td> </td></tr></tbody></table></div><div content-id="DSU045TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>Number of reads </th><th>No. of nucleotides after QC (Mb) </th></tr></thead><tbody><tr><td colspan="3"><em>Thlaspi arvense</em> genome sequencing </td></tr><tr><td> Illumina HiSeq 2000 (2 × 100 bp paired end) </td><td>352,394,426 </td><td>33,190.50 </td></tr><tr><td rowspan="2"> Illumina MiSeq (2 × 250 bp paired end) </td><td>6,291,688 (merged) </td><td>2,589.33 </td></tr><tr><td>8,548,686 (unmerged) </td><td>1,904.90 </td></tr><tr><td> Illumina HiSeq 2000 (2 × 50 bp Mate Pair)<br />2, 3, 5 kb inserts </td><td>209,815,249 </td><td>9,550.66 </td></tr><tr><td> PacBio SMRT cell (four cells) </td><td>110,751 </td><td>214.78 </td></tr><tr><td> </td><td>Scaffolds </td><td>Contigs in scaffolds </td></tr><tr><td colspan="3"><em>T. arvense</em> genome assembly </td></tr><tr><td> Number </td><td>6,768 </td><td>44,109 </td></tr><tr><td> Mean length (bp) </td><td>50,681 </td><td>7,375 </td></tr><tr><td> N50 (bp) </td><td>140,815 </td><td>21,096 </td></tr><tr><td> Total assembly length (bp) </td><td>343,012,389 </td><td>325,295,785 </td></tr><tr><td colspan="3"><em>T. arvense</em> genome annotation </td></tr><tr><td> Number of gene models </td><td>27,390 </td><td> </td></tr><tr><td>  Gene models highly similar to TAIR10 peptide (<em>e</em> &lt; 1 × 10<sup>−5</sup> and &gt;70% positive per cent) </td><td>85.94% </td><td> </td></tr><tr><td> GC content </td><td>37.99% </td><td> </td></tr><tr><td> Repetitive DNA sequence </td><td>24.38% </td><td> </td></tr><tr><td>  Retroelements (20.94%) </td><td>78,812 </td><td> </td></tr><tr><td>  DNA transposons (1.6%) </td><td>12,382 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>Summary statistics for DNA sequencing libraries, genome assembly and scaffolding, and genome annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588750" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU045TB1" class="table-wrap-title" data-id="DSU045TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genome sequencing, assembly, and annotation statistics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Library type </th><th>Number of reads </th><th>No. of nucleotides after QC (Mb) </th></tr></thead><tbody><tr><td colspan="3"><em>Thlaspi arvense</em> genome sequencing </td></tr><tr><td> Illumina HiSeq 2000 (2 × 100 bp paired end) </td><td>352,394,426 </td><td>33,190.50 </td></tr><tr><td rowspan="2"> Illumina MiSeq (2 × 250 bp paired end) </td><td>6,291,688 (merged) </td><td>2,589.33 </td></tr><tr><td>8,548,686 (unmerged) </td><td>1,904.90 </td></tr><tr><td> Illumina HiSeq 2000 (2 × 50 bp Mate Pair)<br />2, 3, 5 kb inserts </td><td>209,815,249 </td><td>9,550.66 </td></tr><tr><td> PacBio SMRT cell (four cells) </td><td>110,751 </td><td>214.78 </td></tr><tr><td> </td><td>Scaffolds </td><td>Contigs in scaffolds </td></tr><tr><td colspan="3"><em>T. arvense</em> genome assembly </td></tr><tr><td> Number </td><td>6,768 </td><td>44,109 </td></tr><tr><td> Mean length (bp) </td><td>50,681 </td><td>7,375 </td></tr><tr><td> N50 (bp) </td><td>140,815 </td><td>21,096 </td></tr><tr><td> Total assembly length (bp) </td><td>343,012,389 </td><td>325,295,785 </td></tr><tr><td colspan="3"><em>T. arvense</em> genome annotation </td></tr><tr><td> Number of gene models </td><td>27,390 </td><td> </td></tr><tr><td>  Gene models highly similar to TAIR10 peptide (<em>e</em> &lt; 1 × 10<sup>−5</sup> and &gt;70% positive per cent) </td><td>85.94% </td><td> </td></tr><tr><td> GC content </td><td>37.99% </td><td> </td></tr><tr><td> Repetitive DNA sequence </td><td>24.38% </td><td> </td></tr><tr><td>  Retroelements (20.94%) </td><td>78,812 </td><td> </td></tr><tr><td>  DNA transposons (1.6%) </td><td>12,382 </td><td> </td></tr></tbody></table></div><div content-id="DSU045TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Library type </th><th>Number of reads </th><th>No. of nucleotides after QC (Mb) </th></tr></thead><tbody><tr><td colspan="3"><em>Thlaspi arvense</em> genome sequencing </td></tr><tr><td> Illumina HiSeq 2000 (2 × 100 bp paired end) </td><td>352,394,426 </td><td>33,190.50 </td></tr><tr><td rowspan="2"> Illumina MiSeq (2 × 250 bp paired end) </td><td>6,291,688 (merged) </td><td>2,589.33 </td></tr><tr><td>8,548,686 (unmerged) </td><td>1,904.90 </td></tr><tr><td> Illumina HiSeq 2000 (2 × 50 bp Mate Pair)<br />2, 3, 5 kb inserts </td><td>209,815,249 </td><td>9,550.66 </td></tr><tr><td> PacBio SMRT cell (four cells) </td><td>110,751 </td><td>214.78 </td></tr><tr><td> </td><td>Scaffolds </td><td>Contigs in scaffolds </td></tr><tr><td colspan="3"><em>T. arvense</em> genome assembly </td></tr><tr><td> Number </td><td>6,768 </td><td>44,109 </td></tr><tr><td> Mean length (bp) </td><td>50,681 </td><td>7,375 </td></tr><tr><td> N50 (bp) </td><td>140,815 </td><td>21,096 </td></tr><tr><td> Total assembly length (bp) </td><td>343,012,389 </td><td>325,295,785 </td></tr><tr><td colspan="3"><em>T. arvense</em> genome annotation </td></tr><tr><td> Number of gene models </td><td>27,390 </td><td> </td></tr><tr><td>  Gene models highly similar to TAIR10 peptide (<em>e</em> &lt; 1 × 10<sup>−5</sup> and &gt;70% positive per cent) </td><td>85.94% </td><td> </td></tr><tr><td> GC content </td><td>37.99% </td><td> </td></tr><tr><td> Repetitive DNA sequence </td><td>24.38% </td><td> </td></tr><tr><td>  Retroelements (20.94%) </td><td>78,812 </td><td> </td></tr><tr><td>  DNA transposons (1.6%) </td><td>12,382 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>Summary statistics for DNA sequencing libraries, genome assembly and scaffolding, and genome annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588750" target="_blank">View Large</a></div></div><p>The initial CLC assembly resulted in 206,726 initial scaffolds ≥200 bp, encompassing 392,190,998 bp. Of this assembly, initial scaffolds over 1,000 bp (<em>n</em> = 50,064, 322,949,692 bp total length) were further joined using long-insert Illumina mate pair reads in SSPACE, and remaining gaps were filled using GAPFILLER. This analysis resulted in the formation of 6,768 final scaffolds that encompass over 343 Mb with an average scaffold length of 50,681 bp and N50 value of 140,815 bp (Table <a reveal-id="DSU045TB1" data-open="DSU045TB1" class="link link-reveal link-table xref-fig">1</a>). Over 60% of the assembled scaffolds are over 10,000 bp long, with 902 scaffolds over 100,000 bp and 9 scaffolds over 1,000,000 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The longest 3,000 scaffolds represented over 85% of the assembly length (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The 156,662 small scaffolds (&lt;1000 bp) from the initial CLC assembly that were excluded from the final scaffolding and gap-filling manipulations encompassed 69,241,306 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table 2</a>). These small initial scaffolds were excluded from further analyses as they likely represent sequences from repetitive regions of the genome, which are difficult to assemble and were unlikely to contribute to gene identification efforts. These small initial scaffolds represent a significant portion (12.8%) of the predicted genome size. With the addition of the initial small scaffolds, the total assembled length of the draft genome presented here is 412,253,695 bp, 76.5% of the predicted pennycress genome size of 539 Mb.</p><p>As the draft genome is incomplete (&lt;80% of the predicted genome size) and fragmented, developing a more complete and contiguous assembly will be important for both plant improvement efforts and answering basic questions about the genomics and evolution of pennycress. However, the reported contig N50 of 21 kb is well within the norm of other recently reported genome assemblies based on Illumina reads such as those for <em>Aquilaria agallocha</em>—14.6 kb,<sup><a reveal-id="DSU045C44" data-open="DSU045C44" class="link link-ref link-reveal xref-bibr">44</a></sup><em>Sesamum indium</em>—52.2 kb,<sup><a reveal-id="DSU045C2" data-open="DSU045C2" class="link link-ref link-reveal xref-bibr">2</a></sup><em>Citrullus lantus</em>—26.4 kb,<sup><a reveal-id="DSU045C45" data-open="DSU045C45" class="link link-ref link-reveal xref-bibr">45</a></sup> and <em>Cicer arietinum</em>—23.54 kb.<sup><a reveal-id="DSU045C6" data-open="DSU045C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The sequence of contigs directly reflects the sequenced gene space, which is especially important for a species like pennycress where this information enables one to identify genes of interest whose manipulation via either overexpression or knockdown can be predicted to confer agronomically desirable phenotypes.</p>                            <h3 scrollto-destination=4588753 id="4588753" class="section-title" data-legacy-id=s3b>3.2. Genome annotation with MAKER</h3>
<p>Genome annotation using the MAKER pipeline<sup><a reveal-id="DSU045C40" data-open="DSU045C40" class="link link-ref link-reveal xref-bibr">40</a></sup> annotated 27,390 predicted protein-coding genes with an average total length of 2,195 bp, average coding sequence length of 1,238 bp, and an average of 5.541 exons/gene (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S2</a>). More than 89% of the predicted gene models are supported by at least 5 RNA sequencing (RNAseq) reads from the previously published <em>de novo</em> transcriptome assembly, while 1452 gene models (5.3%) lack RNAseq read support (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S2</a>). Over 85% of the predicted peptides (23,538) have at least one highly significant BLASTp hit (<em>e</em> &lt; 1 × 10<sup>−5</sup> and &gt;70% positive percent) to an <em>A. thaliana</em> (TAIR10) predicted peptide (Table <a reveal-id="DSU045TB1" data-open="DSU045TB1" class="link link-reveal link-table xref-fig">1</a>). Another 1,876 predicted peptides (6.8%) show a significant hit (e &lt; 1 × 10<sup>−5</sup>), but at a lower positive percent value (&gt;60%). Only 173 predicted peptides lacking an <em>A.thaliana</em> BLASTp hit were found (Expectation value &gt;10) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S2</a>). BLASTn analyses of the 33,873 <em>de novo</em> assembled contigs from the pennycress transcriptome<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> against the draft genome indicate a high level of completeness of the predicted gene space. Over 88% of transcripts (30,053) had BLASTn hits (≥95% identity and <em>e</em> ≤ 1 × 10<sup>−5</sup>) in the genome, while 95.8% of assembled transcripts (32,458) had a significant hit (<em>e</em> ≤ 1 × 10<sup>−5</sup>) but at a lower percent identity threshold (&gt;75%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S3</a>). Only 167 transcriptome contigs lacked a BLASTn hit in the genome. (Expectation value &gt;10) A separate BLASTn analysis of these 167 transcripts against the genomes of <em>A. thaliana</em>,<sup><a reveal-id="DSU045C41" data-open="DSU045C41" class="link link-ref link-reveal xref-bibr">41</a></sup><em>A. lyrata</em>,<sup><a reveal-id="DSU045C42" data-open="DSU045C42" class="link link-ref link-reveal xref-bibr">42</a></sup><em>B. rapa</em>,<sup><a reveal-id="DSU045C43" data-open="DSU045C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>C. rubella</em>,<sup><a reveal-id="DSU045C5" data-open="DSU045C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and <em>E. salsugineum</em><sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> revealed that 40 transcripts lacked any match to these five species, while the remaining 127 consisted of low quality and short hits, likely indicative of misassembled sequences from the <em>de novo</em> assembled transcriptome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S4</a>). The repetitive DNA content of the final pennycress genome scaffolds was assessed by RepeatMasker.<sup><a reveal-id="DSU045C46" data-open="DSU045C46" class="link link-ref link-reveal xref-bibr">46</a></sup> It was found that repetitive elements constituted 24% of the draft genome. This analysis identified 78,812 retroelements encompassing &gt;71 Mb of the assembly, consisting mainly of Gypsy/DIRS1-type, long-terminal repeats (60.3 Mb). 12,382 DNA transposons representing &gt;5 Mb of the draft genome were also found, including hobo-Activator (0.8 Mb) and Tourist/Harbinger (0.9 Mb) type elements (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Both the raw sequencing reads and assembled sequences have been submitted to NCBI, which has been supplemented by a pennycress genome database containing a JBrowse genome browser,<sup><a reveal-id="DSU045C47" data-open="DSU045C47" class="link link-ref link-reveal xref-bibr">47</a></sup> BLAST database, and data repository available at pennycress.umn.edu.</p>                            <h3 scrollto-destination=4588755 id="4588755" class="section-title" data-legacy-id=s3c>3.3. Comparative genomics of the pennycress genome assembly</h3>
<p>Of the sequenced Brassicaceae genomes, pennycress is most closely related to <em>E. salsugineum</em>, which possesses a much smaller genome (241 Mb), but the same karyotype (<em>n</em> = 7).<sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU045C48" data-open="DSU045C48" class="link link-ref link-reveal xref-bibr">48</a></sup> To evaluate the relative completeness of the genome assembly, we used a syntenic path assembly comparison of the pennycress assembly to the <em>E. salsugineum</em> reference genome.<sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Pennycress genomic scaffolds &gt;75 kb long, representing 241 Mb (&gt;70%) of the assembly were compared with the 241 Mb reference genome of <em>E. salsugineum</em>. Large portions of the seven <em>E. salsugineum</em> pseudochromosomes possess a high degree of synteny with the pennycress assembly, indicative of the close evolutionary relationship between these two species, as well as a high level of completeness of conserved regions in the pennycress genome (Fig. <a reveal-id="DSU045F1" data-open="DSU045F1" class="link link-reveal link-table xref-fig">1</a>A). </p>                        <a id="4588757" scrollto-destination="4588757"></a>
<div id="DSU045F1" data-id="DSU045F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04501.png?Expires=2147483647&Signature=G3X63w0WmcUYImHy7xB3BwoVNTLr8VpFyZn08UVDpYKqwtNcSgiKPQjun4q6xHL9YOQU1njHOq14T9GypT4ogTlSyTVkL6XIfcfLdjir2At08GSLnvkngtRqxuzS0kdQbUwQBm5RibprfK6j1kDPjhjm0GHcNMTeGqKcXR6rcQgppZRH7e7Wipqz9gejhkqJdVYjHgZuG6V-5XHHGh~dpWp3xJ5hjj~pyqqyzpvweiBifvj9asOPVTjYamv0C7tuKVuudPjG-pDpxLus6mw5r20AqsLWRh1ms0yxcvHrxA99Pe7j1tOdfnBll4Dd4h~eTWVbiHdK5sz6-29-DPGxvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative genomics of pennycress and other Brassicaceae species. (A) Syntenic path assembly dot plots comparing pennycress scaffolds &gt;75 kilobases long to the seven Eutrema salsugineum pseudochromosomes fromYang et al.1 (B) BLASTp analysis of the 27,390 predicted pennycress peptides against predicted peptide sets from Capsella rubella,5Brassica rapa,43Arabidopsis thaliana (Bevan and Initiative, 2000), Arabidopsis lyrata,42 and Eutrema salsugineum. Highly similar is defined as pennycress predicted peptide having at least one BLASTp hit e &lt; 1 × 10−5 and positive sequence similarity &gt;70%. (C) BLASTp analysis of predicted pennycress peptides against a protein database containing the predicted peptides of the five Brassicaceae species listed. Predicted peptides with top hits (e ≤ 1 × 10−5 and &gt;70% hit length) to a predicted protein from the corresponding species are shown, with pennycress peptides with hits falling below this threshold shown in the lower right half of the pie chart." path-from-xml="dsu04501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588757/dsu04501.tiff" path-from-xml="dsu04501.tiff" target="_blank">View large</a><a section="4588757" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04501.png?Expires=2147483647&Signature=g4d-H7KOu2Pz~5ZmVU55SCWum--W-UVrIhxgGEdhf7mg520015XwiZCTV0ZYZlaG5HYy0yMe8qW7OdK0Muh0X3XSWQUFF1CAsBnvjSPaO41LevG9sRgA7-xyFCDQrSYJDnHRLuL-nj0CPYE1jit~nOaGA~DKrjVeYu1D9rQRxkVWodfoHnlxecfThOc0LZUaEB3FauxFBnVpgEA8zePIRAbd6-BjxgStroeMhlEE1P4niA2HyaczeJm8BIlILoYlyhM7JHI3EGgHqehNF-ABBjr7lhG0yWL3HSltxOCTn0sTW6zjVb8l93T2qKYqsJL0AvpSsxylOfXhaRt8TNEgtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588757&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative genomics of pennycress and other Brassicaceae species. (A) Syntenic path assembly dot plots comparing pennycress scaffolds &gt;75 kilobases long to the seven <em>Eutrema salsugineum</em> pseudochromosomes fromYang <em>et al.</em><sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> (B) BLASTp analysis of the 27,390 predicted pennycress peptides against predicted peptide sets from <em>Capsella rubella</em>,<sup><a reveal-id="DSU045C5" data-open="DSU045C5" class="link link-ref link-reveal xref-bibr">5</a></sup><em>Brassica rapa</em>,<sup><a reveal-id="DSU045C43" data-open="DSU045C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>Arabidopsis thaliana</em> (Bevan and Initiative, 2000), <em>Arabidopsis lyrata</em>,<sup><a reveal-id="DSU045C42" data-open="DSU045C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and <em>Eutrema salsugineum</em>. Highly similar is defined as pennycress predicted peptide having at least one BLASTp hit <em>e</em> &lt; 1 × 10<sup>−5</sup> and positive sequence similarity &gt;70%. (C) BLASTp analysis of predicted pennycress peptides against a protein database containing the predicted peptides of the five Brassicaceae species listed. Predicted peptides with top hits (<em>e</em> ≤ 1 × 10<sup>−5</sup> and &gt;70% hit length) to a predicted protein from the corresponding species are shown, with pennycress peptides with hits falling below this threshold shown in the lower right half of the pie chart.</p></div></div><div content-id="DSU045F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04501.png?Expires=2147483647&Signature=G3X63w0WmcUYImHy7xB3BwoVNTLr8VpFyZn08UVDpYKqwtNcSgiKPQjun4q6xHL9YOQU1njHOq14T9GypT4ogTlSyTVkL6XIfcfLdjir2At08GSLnvkngtRqxuzS0kdQbUwQBm5RibprfK6j1kDPjhjm0GHcNMTeGqKcXR6rcQgppZRH7e7Wipqz9gejhkqJdVYjHgZuG6V-5XHHGh~dpWp3xJ5hjj~pyqqyzpvweiBifvj9asOPVTjYamv0C7tuKVuudPjG-pDpxLus6mw5r20AqsLWRh1ms0yxcvHrxA99Pe7j1tOdfnBll4Dd4h~eTWVbiHdK5sz6-29-DPGxvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative genomics of pennycress and other Brassicaceae species. (A) Syntenic path assembly dot plots comparing pennycress scaffolds &gt;75 kilobases long to the seven Eutrema salsugineum pseudochromosomes fromYang et al.1 (B) BLASTp analysis of the 27,390 predicted pennycress peptides against predicted peptide sets from Capsella rubella,5Brassica rapa,43Arabidopsis thaliana (Bevan and Initiative, 2000), Arabidopsis lyrata,42 and Eutrema salsugineum. Highly similar is defined as pennycress predicted peptide having at least one BLASTp hit e &lt; 1 × 10−5 and positive sequence similarity &gt;70%. (C) BLASTp analysis of predicted pennycress peptides against a protein database containing the predicted peptides of the five Brassicaceae species listed. Predicted peptides with top hits (e ≤ 1 × 10−5 and &gt;70% hit length) to a predicted protein from the corresponding species are shown, with pennycress peptides with hits falling below this threshold shown in the lower right half of the pie chart." path-from-xml="dsu04501.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588757/dsu04501.tiff" path-from-xml="dsu04501.tiff" target="_blank">View large</a><a section="4588757" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04501.png?Expires=2147483647&Signature=g4d-H7KOu2Pz~5ZmVU55SCWum--W-UVrIhxgGEdhf7mg520015XwiZCTV0ZYZlaG5HYy0yMe8qW7OdK0Muh0X3XSWQUFF1CAsBnvjSPaO41LevG9sRgA7-xyFCDQrSYJDnHRLuL-nj0CPYE1jit~nOaGA~DKrjVeYu1D9rQRxkVWodfoHnlxecfThOc0LZUaEB3FauxFBnVpgEA8zePIRAbd6-BjxgStroeMhlEE1P4niA2HyaczeJm8BIlILoYlyhM7JHI3EGgHqehNF-ABBjr7lhG0yWL3HSltxOCTn0sTW6zjVb8l93T2qKYqsJL0AvpSsxylOfXhaRt8TNEgtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588757&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04501.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative genomics of pennycress and other Brassicaceae species. (A) Syntenic path assembly dot plots comparing pennycress scaffolds &gt;75 kilobases long to the seven <em>Eutrema salsugineum</em> pseudochromosomes fromYang <em>et al.</em><sup><a reveal-id="DSU045C1" data-open="DSU045C1" class="link link-ref link-reveal xref-bibr">1</a></sup> (B) BLASTp analysis of the 27,390 predicted pennycress peptides against predicted peptide sets from <em>Capsella rubella</em>,<sup><a reveal-id="DSU045C5" data-open="DSU045C5" class="link link-ref link-reveal xref-bibr">5</a></sup><em>Brassica rapa</em>,<sup><a reveal-id="DSU045C43" data-open="DSU045C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>Arabidopsis thaliana</em> (Bevan and Initiative, 2000), <em>Arabidopsis lyrata</em>,<sup><a reveal-id="DSU045C42" data-open="DSU045C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and <em>Eutrema salsugineum</em>. Highly similar is defined as pennycress predicted peptide having at least one BLASTp hit <em>e</em> &lt; 1 × 10<sup>−5</sup> and positive sequence similarity &gt;70%. (C) BLASTp analysis of predicted pennycress peptides against a protein database containing the predicted peptides of the five Brassicaceae species listed. Predicted peptides with top hits (<em>e</em> ≤ 1 × 10<sup>−5</sup> and &gt;70% hit length) to a predicted protein from the corresponding species are shown, with pennycress peptides with hits falling below this threshold shown in the lower right half of the pie chart.</p></div></div><p>To evaluate the quality of the genome assembly and predicted gene models, a combination of comparative analyses were used to compare the draft pennycress genome with the gene models of <em>A. thaliana</em>, <em>A. lyrata</em>, <em>B. rapa</em>, <em>C. rubella</em>, and <em>E. salsugineum</em>. BLASTp analyses of the 27,390 predicted pennycress peptides against five separate databases containing these five Brassicaceae species revealed that over 23,000 of the pennycress peptides had highly similar hits in all five species, with <em>E. salsugineum</em> possessing the highest proportion (&gt;89%) of highly similar predicted peptides (Fig. <a reveal-id="DSU045F1" data-open="DSU045F1" class="link link-reveal link-table xref-fig">1</a>B; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S5</a>). Similarly, in a BLASTp analysis of the pennycress predicted peptides against a single database containing all five Brassicaceae species, 15,414 predicted pennycress peptides had highly similar (<em>e</em> ≤ 1 × 10<sup>−5</sup> and ≥70% hit length) hits to <em>E. salsugineum</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S6</a>). An additional 8,965 predicted peptides had highly similar hits to <em>B. rapa</em>, <em>A. lyrata</em>, <em>A. thaliana</em>, and <em>C. rubella,</em> while 2,903 BLASTp hits fell outside of these parameters, and 108 of the predicted pennycress peptides lacked a BLASTp hit (Fig. <a reveal-id="DSU045F1" data-open="DSU045F1" class="link link-reveal link-table xref-fig">1</a>C).</p>                            <h3 scrollto-destination=4588759 id="4588759" class="section-title" data-legacy-id=s3d>3.4. Evaluation of zygosity in the sequenced population</h3>
<p>The draft genome was constructed using DNA isolated from nine plants that were several generations removed from a single population first identified in Coates, MN. A variant detection analysis was performed to assess the degree of homozygosity among the individual plants used for sequencing (see Materials and Methods for parameters used in this analysis). We detected 131,906 SNPs, which is ∼1 SNP per 2.6 kb. Given the high level of sequencing coverage and stringent quality control, it is unlikely that the predicted SNPs were solely due to sequencing errors. We developed three hypotheses to explain the SNPs. First, these SNPs represent inappropriately assembled duplicated regions of the pennycress genome. In this analysis, if the first hypothesis were true and the declared SNPs were artificial, then we would not expect any differences between the nine different plants. Second, the SNPs may represent evidence of heterozygosity throughout the genome that would occur if the plants were prone to a high degree of outcrossing. If the second hypothesis were true, then we would expect that the nine plants would segregate at ∼1 : 1 : 2 for the homozygosity of the presence or absence of a SNP or would be heterozygous with one chromatid containing the site and the other lacking the site, respectively. Third, distinct highly homozygous populations made up the original collection of plants used to isolate DNA that was sequenced. In this case, the genomes of any individual in the population would be expected to be largely homozygous at any particular locus.</p><p>CAPS<sup><a reveal-id="DSU045C49" data-open="DSU045C49" class="link link-ref link-reveal xref-bibr">49</a></sup> analysis using DNA isolated from progeny of the individual plants that were used to generate the draft genome was performed to distinguish between these three hypotheses. Primer sequences used to amplify regions used in the CAPS analysis shown in Fig. <a reveal-id="DSU045F2" data-open="DSU045F2" class="link link-reveal link-table xref-fig">2</a>A are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. Individuals were shown to either distinctly contain or lack the variant at four restriction enzyme sites, which eliminated the first hypothesis that these SNPs represented divergence in paralogous genes or misassembly of duplicated regions. Furthermore, none of the samples showed evidence of heterozygosity. Plants 3, 5, and 7 lacked the cut sites at the polymorphic regions, and plants 1, 2, 4, 6, 8, and 9 were homozygous for the cut sites (Fig. <a reveal-id="DSU045F2" data-open="DSU045F2" class="link link-reveal link-table xref-fig">2</a>B). This supports the third hypothesis that the original MN106 population contained at least two distinct, highly homogenous populations. The fact that three individuals lacked all the cut sites and six individuals contained all the cut sites is likely due to the fact that at every CAPS locus, one prominent variant was detected in the variant detection analysis. Loci with the prominent variant that contained the six base restriction site were chosen for the CAPS analysis. </p>                        <a id="4588762" scrollto-destination="4588762"></a>
<div id="DSU045F2" data-id="DSU045F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04502.png?Expires=2147483647&Signature=AelK90ZwSdlj0RQlEcXbWhQncX4AwLJwRcVPTf5D7P4o7cB965Ysu2G01c8tLLjV7jDs4iyVZtc7tuDOCuytsBbdIF4BaUT5dzM3wHU4wIRxPdCS73~lxModtNcPLVG0WnEu84NQhWRk6RM1EAZmp9mlC5Ddb7dpH9Kpk9V9JkBlnImTjLzV9xYXADe43p36u14ChYbeTcoBhh2fSSSVBL3JE5tRscotPTfEcg6d2Q0f5mEdug8~vqFAGmKffP~ppN8MS4mB88j3lFhuMaFj07ekL4yIF7nMta~xdilMudg6-bORlq0QB67YalxphAUXSNus~dbaD4lZM4e5xU0dbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAPS analysis of Thlaspi arvense line MN106 (A) Schematic of the four PCR fragments produced by the primer sets listed in Supplementary Table S4. The largest fragments used to distinguish between individuals containing the SNP (MN106 A genotype fragment—top, and MN106 B genotype fragment—bottom). (B) DNA was isolated from progeny of each of the nine plants used to produce the draft genome assembly, and analysed using four CAPS markers. PCR products for each plant are shown side-by-side undigested (uncut) and post-digestion (cut) with the corresponding restriction endonucleases. In all cases, samples 3, 5, and 7 share restriction digest patterns, corresponding to the MN106-B genotype. A negative control for the PCR reaction is shown in the last lane. (C) Morphology of developing T. arvense flowers. The top panel (1–3) shows the morphology of the unaltered flowers, while the bottom panel (4–6) shows the same series of flowers with sepals and petals either removed or rearranged to reveal the status of the stamens with regard to filament elongation and the shedding of pollen. (4) Neither filament elongation nor pollen shedding has commenced in (1). (5) Filaments have elongated, and pollen is being shed inside of the closed flower shown in (2). (5) Pollen densely covers the stigmatic surface by the time the flower is fully open in (6). All scale bars equal 1 mm." path-from-xml="dsu04502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588762/dsu04502.tiff" path-from-xml="dsu04502.tiff" target="_blank">View large</a><a section="4588762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04502.png?Expires=2147483647&Signature=3MeoZHB0pJV6Jb5Sz8OZ~b4xx5jSQKOvOn0SGwphvEP2zGBWobEDnPBO9nrH6F8-ilPI6YVOxdkHzXC1xdy21bO~Kym9cqhQgUIB7h8~yN-ScQ8CYuA3ctMpwCwzxmU-rxjmQl2uv2KxQeg8PrkRnPvOChAlCBG6Uwd6xYMGsoacxd384CZZpUZhCYN43crVoO6HzSX6XHYFwYEIFsLWJZTb-XYBrQuGMVC3VFMKcpcB7zN-z-rzSeNWrgLDR5WMcJ7Bgkf-uzi2uYhzDXk8EjdkEzf1m8Iiw6rlRx4wPNJxXw3EnNzxQs7WNVupEGBK4wVP7O-IdjfNEaOLbIndbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588762&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAPS analysis of <em>Thlaspi arve</em>nse line MN106 (A) Schematic of the four PCR fragments produced by the primer sets listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. The largest fragments used to distinguish between individuals containing the SNP (MN106 A genotype fragment—top, and MN106 B genotype fragment—bottom). (B) DNA was isolated from progeny of each of the nine plants used to produce the draft genome assembly, and analysed using four CAPS markers. PCR products for each plant are shown side-by-side undigested (uncut) and post-digestion (cut) with the corresponding restriction endonucleases. In all cases, samples 3, 5, and 7 share restriction digest patterns, corresponding to the MN106-B genotype. A negative control for the PCR reaction is shown in the last lane. (C) Morphology of developing <em>T. arvense</em> flowers. The top panel (1–3) shows the morphology of the unaltered flowers, while the bottom panel (4–6) shows the same series of flowers with sepals and petals either removed or rearranged to reveal the status of the stamens with regard to filament elongation and the shedding of pollen. (4) Neither filament elongation nor pollen shedding has commenced in (1). (5) Filaments have elongated, and pollen is being shed inside of the closed flower shown in (2). (5) Pollen densely covers the stigmatic surface by the time the flower is fully open in (6). All scale bars equal 1 mm.</p></div></div><div content-id="DSU045F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04502.png?Expires=2147483647&Signature=AelK90ZwSdlj0RQlEcXbWhQncX4AwLJwRcVPTf5D7P4o7cB965Ysu2G01c8tLLjV7jDs4iyVZtc7tuDOCuytsBbdIF4BaUT5dzM3wHU4wIRxPdCS73~lxModtNcPLVG0WnEu84NQhWRk6RM1EAZmp9mlC5Ddb7dpH9Kpk9V9JkBlnImTjLzV9xYXADe43p36u14ChYbeTcoBhh2fSSSVBL3JE5tRscotPTfEcg6d2Q0f5mEdug8~vqFAGmKffP~ppN8MS4mB88j3lFhuMaFj07ekL4yIF7nMta~xdilMudg6-bORlq0QB67YalxphAUXSNus~dbaD4lZM4e5xU0dbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAPS analysis of Thlaspi arvense line MN106 (A) Schematic of the four PCR fragments produced by the primer sets listed in Supplementary Table S4. The largest fragments used to distinguish between individuals containing the SNP (MN106 A genotype fragment—top, and MN106 B genotype fragment—bottom). (B) DNA was isolated from progeny of each of the nine plants used to produce the draft genome assembly, and analysed using four CAPS markers. PCR products for each plant are shown side-by-side undigested (uncut) and post-digestion (cut) with the corresponding restriction endonucleases. In all cases, samples 3, 5, and 7 share restriction digest patterns, corresponding to the MN106-B genotype. A negative control for the PCR reaction is shown in the last lane. (C) Morphology of developing T. arvense flowers. The top panel (1–3) shows the morphology of the unaltered flowers, while the bottom panel (4–6) shows the same series of flowers with sepals and petals either removed or rearranged to reveal the status of the stamens with regard to filament elongation and the shedding of pollen. (4) Neither filament elongation nor pollen shedding has commenced in (1). (5) Filaments have elongated, and pollen is being shed inside of the closed flower shown in (2). (5) Pollen densely covers the stigmatic surface by the time the flower is fully open in (6). All scale bars equal 1 mm." path-from-xml="dsu04502.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588762/dsu04502.tiff" path-from-xml="dsu04502.tiff" target="_blank">View large</a><a section="4588762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04502.png?Expires=2147483647&Signature=3MeoZHB0pJV6Jb5Sz8OZ~b4xx5jSQKOvOn0SGwphvEP2zGBWobEDnPBO9nrH6F8-ilPI6YVOxdkHzXC1xdy21bO~Kym9cqhQgUIB7h8~yN-ScQ8CYuA3ctMpwCwzxmU-rxjmQl2uv2KxQeg8PrkRnPvOChAlCBG6Uwd6xYMGsoacxd384CZZpUZhCYN43crVoO6HzSX6XHYFwYEIFsLWJZTb-XYBrQuGMVC3VFMKcpcB7zN-z-rzSeNWrgLDR5WMcJ7Bgkf-uzi2uYhzDXk8EjdkEzf1m8Iiw6rlRx4wPNJxXw3EnNzxQs7WNVupEGBK4wVP7O-IdjfNEaOLbIndbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588762&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04502.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAPS analysis of <em>Thlaspi arve</em>nse line MN106 (A) Schematic of the four PCR fragments produced by the primer sets listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>. The largest fragments used to distinguish between individuals containing the SNP (MN106 A genotype fragment—top, and MN106 B genotype fragment—bottom). (B) DNA was isolated from progeny of each of the nine plants used to produce the draft genome assembly, and analysed using four CAPS markers. PCR products for each plant are shown side-by-side undigested (uncut) and post-digestion (cut) with the corresponding restriction endonucleases. In all cases, samples 3, 5, and 7 share restriction digest patterns, corresponding to the MN106-B genotype. A negative control for the PCR reaction is shown in the last lane. (C) Morphology of developing <em>T. arvense</em> flowers. The top panel (1–3) shows the morphology of the unaltered flowers, while the bottom panel (4–6) shows the same series of flowers with sepals and petals either removed or rearranged to reveal the status of the stamens with regard to filament elongation and the shedding of pollen. (4) Neither filament elongation nor pollen shedding has commenced in (1). (5) Filaments have elongated, and pollen is being shed inside of the closed flower shown in (2). (5) Pollen densely covers the stigmatic surface by the time the flower is fully open in (6). All scale bars equal 1 mm.</p></div></div><p>Based on previous analyses, a high degree of homozygosity among wild pennycress individuals was not unexpected, as the cleistogamous development of pennycress flowers (pollination occurring inside a closed flower) minimizes outcrossing (Fig. <a reveal-id="DSU045F2" data-open="DSU045F2" class="link link-reveal link-table xref-fig">2</a>C).<sup><a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> These findings will aid pennycress breeding programmes, as progeny from crosses will be expected to readily self-pollinate with minimal need for isolation. In addition, the analysis provides an extra level of validation by showing that a subset of the identified SNPs are real and not created by sequencing or assembly errors. These SNPs have the potential to be used in breeding and geographical studies, and are available in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S7</a>. These results suggest that the original MN106 population consists of at least two distinct haplotypes on the basis of the CAPS analysis. This suggests that even in mixed populations, the selfing nature of pennycress reduces heterozygosity. This is an important finding, as it will facilitate the isolation and generation of highly inbred populations.</p>                            <h3 scrollto-destination=4588764 id="4588764" class="section-title" data-legacy-id=s3e>3.5. Analysis of genes involved in glucosinolate metabolism</h3>
<p>To explore the functionality of our assembly, we determined how well the predicted gene space accounted for genes involved in glucosinolate biosynthesis. Glucosinolates (GSLs) are a diverse class of secondary metabolites common throughout the Brassicaceae that are important for plant/pathogen and plant/herbivore interactions.<sup><a reveal-id="DSU045C50" data-open="DSU045C50" class="link link-ref link-reveal xref-bibr">50</a></sup> The underlying genetic mechanisms controlling the biosynthesis, transport, and breakdown of GSLs within the Brassicaceae have largely been dissected, with many of the genes responsible for this process having been characterized.<sup><a reveal-id="DSU045C51" data-open="DSU045C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Upon biosynthesis of the core glucosinolate structure (derived from one of several amino acids), GSLs generally lack bioactivity until plant tissue is damaged, leading to the hydrolysis of the GSL structure to one of several by-products. This breakdown process is mediated by enzymes called myrosinases, resulting in an unstable aglycone.<sup><a reveal-id="DSU045C51" data-open="DSU045C51" class="link link-ref link-reveal xref-bibr">51</a></sup> The diversification of the final glucosinolate breakdown by-product is mediated by several classes of specifier proteins.<sup><a reveal-id="DSU045C52" data-open="DSU045C52" class="link link-ref link-reveal xref-bibr">52</a></sup></p><p>Historically, pennycress has been characterized by its unique ‘garlic-like’ aroma that has been attributed to high levels of allylthiocyanate,<sup><a reveal-id="DSU045C53" data-open="DSU045C53" class="link link-ref link-reveal xref-bibr">53</a></sup> which is a major hydrolysis by-product of the major GSL in above-ground pennycress tissues: allylglucosinolate.<sup><a reveal-id="DSU045C54" data-open="DSU045C54" class="link link-ref link-reveal xref-bibr">54</a></sup> The high level of GSLs in pennycress gives rise to another common name of the species, stinkweed.<sup><a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The high level of GSLs and GSL by-products in pennycress is of significant agronomic and economic interest. Animal feed containing pennycress seed has traditionally been considered undesirable due to the high levels of GSLs in pennycress.<sup><a reveal-id="DSU045C11" data-open="DSU045C11" class="link link-ref link-reveal xref-bibr">11</a></sup> However, after seed is pressed for oil, the remaining seed meal remains high in protein<sup><a reveal-id="DSU045C55" data-open="DSU045C55" class="link link-ref link-reveal xref-bibr">55</a>,<a reveal-id="DSU045C56" data-open="DSU045C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and presents a potential new source of billions of kilograms per year of high protein meal. If varieties with low GSL levels in seed can be developed, there is a potential for using defatted pennycress meal as an animal feed supplement.</p><p>BLASTp analyses were used to identify putative orthologues to known glucosinolate genes in Arabidopsis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S2</a>). Genes involved in the GSL core biosynthesis and breakdown pathway were derived from <em>A. thaliana</em>, <em>B. rapa</em>, and <em>B. oleraceae</em>.<sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Putative orthologues were identified for GSL biosynthesis from methionine and tryptophan (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>A), along with several putative myrosinases (thioglucoside glucohydrolase—TGGs and atypical myrosinases—PEN2 and PEN3). Potential orthologues to several interesting specifier proteins were also identified (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>A—top). To obtain a semi-quantitative estimate of these predicted genes, RNAseq reads from the previously described transcriptome were used to obtain rough expression values for each gene model (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S1</a>). As these RNAseq reads represent a global library representing various tissues, this analysis provides an initial probe into the pennycress genes potentially responsible for the unique glucosinolate composition of pennycress. For example, two putative myrosinases (Ta16900 and Ta16899) represent the 120th and 129th most highly expressed gene models, respectively (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>A and B; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S1</a>). A modified vacuole phenotype1 (<em>MVP1</em>)-like pennycress gene was also identified (Ta16960—Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>A). MVP1 in Arabidopsis interacts with the myrosinase TGG2 to modulate myrosinase activity.<sup><a reveal-id="DSU045C57" data-open="DSU045C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p>                        <a id="4588768" scrollto-destination="4588768"></a>
<div id="DSU045F3" data-id="DSU045F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04503.png?Expires=2147483647&Signature=bbuy8BCrBM4WBsIbTkH41UnQ9UKysO~qiz1CjsI1dt81mDmlGFqBhOFVzDdWHls0yA0fG8kuaYgvqzOEQfV-zz8YJu6y47EewkJ4pn7KKAzXwySSMzhe6dHeTUm5cxn64EBFI5bfg1w2NA35QvOm1TvSj2VFVoI~Y6cnrBF6Xg8yFrblS7qthpIej4Nqwsq6wuwC2j9vdAcA0gBc99LmFnFO0P09SlPXj0AY9IMUf6sCnPBRpCQTb8iq2kxWDwxcQl12hs7LcXLPoJI9Jc0MtTHrmz81pZGf0HwBBfDtN20BiekJOECGccW1O1DL2YNOkMikzSM3BHgL5xgJwaCqxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of genes involved in glucosinolate metabolism and transport. (A) Overview of glucosinolate biosynthesis core structure (top) via methionine and tryptophan and breakdown (bottom) and corresponding orthologues in the pennycress genome pathway derived from Liu et al.4 Expression values (RPKM, in parentheses) are shown for each putative orthologues derived from the global RNAseq reads previously described.19 (B–G) Neighbour joining trees of TGG1/TGG2, MVP1, ESP, ESM1, GTR1, and GTR2-like predicted peptides (100 bootstrap replicates) from pennycress (identified in this study), Brassica rapa, and Brassica oleraceae.4" path-from-xml="dsu04503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588768/dsu04503.tiff" path-from-xml="dsu04503.tiff" target="_blank">View large</a><a section="4588768" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04503.png?Expires=2147483647&Signature=c~A1v7xkrBVtwu9eEA2ny1ttJuK8opzKQGQoRjQUZS6cLQf~Hjoh-aiJqAcmpEWNd7KBZQESaA8FA59iNjymo9DkRa-SB~wxCX-TqPr7kg4UVaDNwx0Iykj~QRLEUb5AhwaTwwN1I1Qa30O0H9EMFoLkLmddjC~9J9J7UPt5BLm~L96o4zKTRcs6Rjx1075c9zE~WzfPYYI-X~vvBZsWC8oWoSTJv4nWbxYQSxCWT3Lt9m-ZAry9Sas8yICqTn1cuJ7By7bHA4VO4mBNy-x9Dk8vR3rBNdXET5c2JNQ9K2~rnRAlzrHkiICYIbkwNQcnP64mF0pSXujJtbaIIxYvcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588768&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of genes involved in glucosinolate metabolism and transport. (A) Overview of glucosinolate biosynthesis core structure (top) via methionine and tryptophan and breakdown (bottom) and corresponding orthologues in the pennycress genome pathway derived from Liu <em>et al.</em><sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Expression values (RPKM, in parentheses) are shown for each putative orthologues derived from the global RNAseq reads previously described.<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (B–G) Neighbour joining trees of TGG1/TGG2, MVP1, ESP, ESM1, GTR1, and GTR2-like predicted peptides (100 bootstrap replicates) from pennycress (identified in this study), <em>Brassica rapa,</em> and <em>Brassica oleraceae</em>.<sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p></div></div><div content-id="DSU045F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/m_dsu04503.png?Expires=2147483647&Signature=bbuy8BCrBM4WBsIbTkH41UnQ9UKysO~qiz1CjsI1dt81mDmlGFqBhOFVzDdWHls0yA0fG8kuaYgvqzOEQfV-zz8YJu6y47EewkJ4pn7KKAzXwySSMzhe6dHeTUm5cxn64EBFI5bfg1w2NA35QvOm1TvSj2VFVoI~Y6cnrBF6Xg8yFrblS7qthpIej4Nqwsq6wuwC2j9vdAcA0gBc99LmFnFO0P09SlPXj0AY9IMUf6sCnPBRpCQTb8iq2kxWDwxcQl12hs7LcXLPoJI9Jc0MtTHrmz81pZGf0HwBBfDtN20BiekJOECGccW1O1DL2YNOkMikzSM3BHgL5xgJwaCqxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of genes involved in glucosinolate metabolism and transport. (A) Overview of glucosinolate biosynthesis core structure (top) via methionine and tryptophan and breakdown (bottom) and corresponding orthologues in the pennycress genome pathway derived from Liu et al.4 Expression values (RPKM, in parentheses) are shown for each putative orthologues derived from the global RNAseq reads previously described.19 (B–G) Neighbour joining trees of TGG1/TGG2, MVP1, ESP, ESM1, GTR1, and GTR2-like predicted peptides (100 bootstrap replicates) from pennycress (identified in this study), Brassica rapa, and Brassica oleraceae.4" path-from-xml="dsu04503.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4588768/dsu04503.tiff" path-from-xml="dsu04503.tiff" target="_blank">View large</a><a section="4588768" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu04503.png?Expires=2147483647&Signature=c~A1v7xkrBVtwu9eEA2ny1ttJuK8opzKQGQoRjQUZS6cLQf~Hjoh-aiJqAcmpEWNd7KBZQESaA8FA59iNjymo9DkRa-SB~wxCX-TqPr7kg4UVaDNwx0Iykj~QRLEUb5AhwaTwwN1I1Qa30O0H9EMFoLkLmddjC~9J9J7UPt5BLm~L96o4zKTRcs6Rjx1075c9zE~WzfPYYI-X~vvBZsWC8oWoSTJv4nWbxYQSxCWT3Lt9m-ZAry9Sas8yICqTn1cuJ7By7bHA4VO4mBNy-x9Dk8vR3rBNdXET5c2JNQ9K2~rnRAlzrHkiICYIbkwNQcnP64mF0pSXujJtbaIIxYvcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4588768&ar=332636&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04503.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of genes involved in glucosinolate metabolism and transport. (A) Overview of glucosinolate biosynthesis core structure (top) via methionine and tryptophan and breakdown (bottom) and corresponding orthologues in the pennycress genome pathway derived from Liu <em>et al.</em><sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Expression values (RPKM, in parentheses) are shown for each putative orthologues derived from the global RNAseq reads previously described.<sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (B–G) Neighbour joining trees of TGG1/TGG2, MVP1, ESP, ESM1, GTR1, and GTR2-like predicted peptides (100 bootstrap replicates) from pennycress (identified in this study), <em>Brassica rapa,</em> and <em>Brassica oleraceae</em>.<sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p></div></div><p>Potential orthologues to many important specifier proteins were also identified in this analysis and help explain the unique GSL activity in pennycress. For example, epithiospecifier modifier1 (ESM1) in Arabidopsis represses the formation of nitriles and favours isothiocyanate production.<sup><a reveal-id="DSU045C58" data-open="DSU045C58" class="link link-ref link-reveal xref-bibr">58</a></sup> An <em>ESM1</em>-like pennycress gene, Ta16274 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>A), was among the top 200 expressed genes models and could explain the high levels of isothiocyanate in pennycress.<sup><a reveal-id="DSU045C54" data-open="DSU045C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Additionally, another potential hydrolysis product of GSLs, epithionitriles, can only be formed from GSLs possessing terminal double bonds in their side chain (allylglucosinolate, for example). As allylglucosinolate represents over 90% of GSLs in above-ground tissue,<sup><a reveal-id="DSU045C54" data-open="DSU045C54" class="link link-ref link-reveal xref-bibr">54</a></sup> this <em>ESM1</em>-like pennycress gene could represent a key GSL specifier protein also responsible for epithionitrile production.</p><p>The evolution of the identified specifier proteins was also examined. It has been previously reported that GSL-related genes commonly exist in tandemly duplicated regions of the genome within the Brassicaceae.<sup><a reveal-id="DSU045C4" data-open="DSU045C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU045C59" data-open="DSU045C59" class="link link-ref link-reveal xref-bibr">59</a></sup> The predicted peptides of <em>A. thaliana</em> TGG1 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>B), MVP1 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>C), ESP (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>D), ESM1 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>E), GTR1 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>F), and GTR2 (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>G) and corresponding orthologues in pennycress, <em>B. rapa</em>, and <em>B. oleraceae</em> were compared to determine sequence similarity and retention of tandemly duplicated regions throughout the genome. We identified three putative pennycress myrosinases with high sequence similarity to the Arabidopsis myrosinases TGG1 (AT5G26000) (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>B), with an additional predicted peptide (Ta16899) being more similar to <em>B. oleraceae</em> TGG2-like peptides (Bol9025706 and Bol017328). Only a single MVP1-like pennycress predicted peptide was identified in the draft genome (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>C). Three orthologous predicted peptides for AtESP (AT1G54040) were identified (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>D). The previously described TaTFP,<sup><a reveal-id="DSU045C54" data-open="DSU045C54" class="link link-ref link-reveal xref-bibr">54</a></sup> which was cloned from a cDNA library, had a top BLASTp hit to Ta16961. However, the Ta16961 predicted peptide is nearly twice as long as the TaTFP predicted peptide (NCBI Accession JN244735) and appears to indicate either the TaTFP cDNA represented a unique splice variant or an erroneous splice site prediction from the annotation pipeline. An adjacent gene model, Ta16962, also had high similarity to TaTFP (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>D). Two tandem <em>ESM1</em>-like genes were also identified, similar to the tandemly duplicated orthologues in <em>B. rapa</em> (Bra027358 and Bra027359) and <em>B. oleraceae</em> (Bol005066 and Bol005067) (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>E).</p><p>Several orthologues to important GSL transport genes were also identified. The GSL transporters glucosinolate transporter (GTR) 1 and 2 have been shown to serve as major transporters responsible for the loading of GSLs into developing seeds in Arabidopsis, and loss-of-function mutations in these genes result in significantly reduced levels of GSLs in seeds.<sup><a reveal-id="DSU045C60" data-open="DSU045C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSU045C61" data-open="DSU045C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The identification of putative GTR1 and GTR2 orthologues (Fig. <a reveal-id="DSU045F3" data-open="DSU045F3" class="link link-reveal link-table xref-fig">3</a>F) represents important targets for improvement in our ongoing breeding programme.</p>                            <h3 scrollto-destination=4588772 id="4588772" class="section-title" data-legacy-id=s3f>3.6. Identification of predicted orthologues of Arabidopsis genes that confer desirable phenotypes when mutated</h3>
<p>There are several important traits that will need to be addressed during the domestication process to make pennycress a viable crop, such as reducing seed dormancy and increasing rates of flowering. In Arabidopsis, complete or partial loss of function mutations in a number of genes can confer agronomically desirable traits such as increased seed size, improved seed oil composition, earlier flowering and reduced seed dormancy, seed glucosinolate content, and seed pod shatter (Table <a reveal-id="DSU045TB2" data-open="DSU045TB2" class="link link-reveal link-table xref-fig">2</a>). Here we show that predicted orthologues of these genes are present in our draft assembly (Table <a reveal-id="DSU045TB2" data-open="DSU045TB2" class="link link-reveal link-table xref-fig">2</a>). While this is not an exhaustive list of potential gene targets, this represents a number of well-characterized genes responsible for controlling these phenotypes. Mutations in these genes in Arabidopsis can confer what would be considered agronomically desirable phenotypes. For example, pennycress GTR1- and GTR2-predicted orthologues are highly conserved at the predicted peptide level (88.5 and 94.8% positive match, respectively). Additionally, identifying natural or induced mutations in key regulatory genes like <em>DOG1</em><sup><a reveal-id="DSU045C62" data-open="DSU045C62" class="link link-ref link-reveal xref-bibr">62</a>,<a reveal-id="DSU045C63" data-open="DSU045C63" class="link link-ref link-reveal xref-bibr">63</a></sup> could lead to reduced levels of seed dormancy in pennycress. Similarly, developing winter annual lines that flower and mature rapidly in the spring should be possible through targeting genes involved in the vernalization and photoperiodic flowering time pathways, such as <em>FLC</em> and <em>FRI</em><sup><a reveal-id="DSU045C19" data-open="DSU045C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (Table <a reveal-id="DSU045TB2" data-open="DSU045TB2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Dataset S8</a>). The development of rapidly maturing lines will be particularly important in the upper Midwestern United States, as the shorter spring growing seasons at northern latitudes could limit the widespread adaptation of pennycress as a winter cover crop. By targeting specific important agronomic traits, and the underlying genetic mechanisms controlling these traits through the approaches mentioned above, we are poised to rapidly convert a wild weed species into a new highly valuable and sustainable winter oilseed crop. </p>                        <a id="4588774" scrollto-destination="4588774"></a>
<div content-id="DSU045TB2" class="table-modal"><div class="table-wrap"><div id="DSU045TB2" class="table-wrap-title" data-id="DSU045TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Putative orthologues controlling important agronomic traits in pennycress</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait of interest </th><th>Arabidopsis gene </th><th>AGI no. </th><th>Corresponding pennycress predicted peptide ID </th><th>Pennycress genome scaffold no. </th><th>Arabidopsis peptide length </th><th>Pennycress peptide length </th><th>% Identity </th><th>% Positive </th></tr></thead><tbody><tr><td rowspan="4">Seed dormancy </td><td><em>DOG1</em> </td><td>AT5G45830 </td><td>Ta16411 </td><td>141 </td><td>291 </td><td>284 </td><td>72.9 </td><td>80.9 </td></tr><tr><td><em>ABI3</em> </td><td>AT3G24650 </td><td>Ta24539 </td><td>630 </td><td>720 </td><td>728 </td><td>86.8 </td><td>90.6 </td></tr><tr><td><em>ABI4</em> </td><td>AT2G40220 </td><td>Ta07356 </td><td>85 </td><td>328 </td><td>395 </td><td>59.3 </td><td>66.3 </td></tr><tr><td><em>ABI5</em> </td><td>AT2G36270 </td><td>Ta25088 </td><td>828 </td><td>442 </td><td>434 </td><td>85.8 </td><td>89 </td></tr><tr><td rowspan="6">Seed size </td><td><em>WRI1</em> </td><td>AT3G54320 </td><td>Ta07949 </td><td>20 </td><td>438 </td><td>441 </td><td>80.2 </td><td>83.4 </td></tr><tr><td><em>DGAT1</em> </td><td>AT3G51520 </td><td>Ta05453 </td><td>12 </td><td>314 </td><td>566 </td><td>67.7 </td><td>70.1 </td></tr><tr><td><em>IKU2</em> </td><td>AT3G19700 </td><td>Ta01151 </td><td>3 </td><td>991 </td><td>1016 </td><td>79.7 </td><td>86.8 </td></tr><tr><td><em>KLU</em> </td><td>AT1G13710 </td><td>Ta14711 </td><td>213 </td><td>517 </td><td>488 </td><td>84.8 </td><td>88.2 </td></tr><tr><td><em>GL2</em> </td><td>AT1G79840 </td><td>Ta08884 </td><td>25 </td><td>776 </td><td>798 </td><td>82.6 </td><td>86 </td></tr><tr><td><em>MUM4</em> </td><td>AT1G53500 </td><td>Ta23332 </td><td>457 </td><td>667 </td><td>597 </td><td>83.6 </td><td>85.8 </td></tr><tr><td rowspan="3">Seed GSL content </td><td><em>HAG1</em> </td><td>AT3G54610 </td><td>Ta07928 </td><td>20 </td><td>568 </td><td>592 </td><td>85.8 </td><td>88.3 </td></tr><tr><td><em>GTR1</em> </td><td>AT3G47960 </td><td>Ta20037 </td><td>206 </td><td>636 </td><td>651 </td><td>81 </td><td>88.5 </td></tr><tr><td><em>GTR2</em> </td><td>AT5G62680 </td><td>Ta01982 </td><td>4 </td><td>616 </td><td>613 </td><td>90.3 </td><td>94.8 </td></tr><tr><td>Erucic acid content </td><td><em>FAE1</em> </td><td>AT4G34520 </td><td>Ta11742<sup>a</sup> </td><td>45 </td><td>506 </td><td>506 </td><td>86.4 </td><td>91.5 </td></tr><tr><td>Linoleic/Linolenic acid content </td><td><em>FAD2</em> </td><td>AT3G12120 </td><td>Ta12495 </td><td>59 </td><td>383 </td><td>404 </td><td>84.3 </td><td>88.2 </td></tr><tr><td rowspan="6">Seed pod shatter </td><td><em>SHP1</em> </td><td>AT3G58780 </td><td>Ta15094 </td><td>22 </td><td>273 </td><td>248 </td><td>86.8 </td><td>89.4 </td></tr><tr><td><em>SHP2</em> </td><td>AT2G42830 </td><td>Ta08438 </td><td>22 </td><td>248 </td><td>290 </td><td>76.6 </td><td>80.7 </td></tr><tr><td><em>IND</em> </td><td>AT4G00120 </td><td>Ta25465 </td><td>1003 </td><td>198 </td><td>172 </td><td>61.5 </td><td>69 </td></tr><tr><td><em>ALC</em> </td><td>AT5G67110 </td><td>Ta02444 </td><td>6 </td><td>210 </td><td>207 </td><td>72.8 </td><td>77.9 </td></tr><tr><td><em>RPL</em> </td><td>AT5G02030 </td><td>Ta15425 </td><td>92 </td><td>575 </td><td>639 </td><td>80.5 </td><td>83.3 </td></tr><tr><td><em>FUL</em> </td><td>AT5G60910 </td><td>Ta01807 </td><td>57 </td><td>242 </td><td>208 </td><td>63.3 </td><td>68.1 </td></tr><tr><td rowspan="2">Time to flower </td><td><em>FLC</em> </td><td>AT5G10140 </td><td>Ta00917 </td><td>1 </td><td>196 </td><td>203 </td><td>84.3 </td><td>89.7 </td></tr><tr><td><em>FRI</em> </td><td>AT4G00650 </td><td>Ta26225<sup>a</sup> </td><td>1344 </td><td>314 </td><td>367 </td><td>55 </td><td>65.8 </td></tr></tbody></table></div><div content-id="DSU045TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait of interest </th><th>Arabidopsis gene </th><th>AGI no. </th><th>Corresponding pennycress predicted peptide ID </th><th>Pennycress genome scaffold no. </th><th>Arabidopsis peptide length </th><th>Pennycress peptide length </th><th>% Identity </th><th>% Positive </th></tr></thead><tbody><tr><td rowspan="4">Seed dormancy </td><td><em>DOG1</em> </td><td>AT5G45830 </td><td>Ta16411 </td><td>141 </td><td>291 </td><td>284 </td><td>72.9 </td><td>80.9 </td></tr><tr><td><em>ABI3</em> </td><td>AT3G24650 </td><td>Ta24539 </td><td>630 </td><td>720 </td><td>728 </td><td>86.8 </td><td>90.6 </td></tr><tr><td><em>ABI4</em> </td><td>AT2G40220 </td><td>Ta07356 </td><td>85 </td><td>328 </td><td>395 </td><td>59.3 </td><td>66.3 </td></tr><tr><td><em>ABI5</em> </td><td>AT2G36270 </td><td>Ta25088 </td><td>828 </td><td>442 </td><td>434 </td><td>85.8 </td><td>89 </td></tr><tr><td rowspan="6">Seed size </td><td><em>WRI1</em> </td><td>AT3G54320 </td><td>Ta07949 </td><td>20 </td><td>438 </td><td>441 </td><td>80.2 </td><td>83.4 </td></tr><tr><td><em>DGAT1</em> </td><td>AT3G51520 </td><td>Ta05453 </td><td>12 </td><td>314 </td><td>566 </td><td>67.7 </td><td>70.1 </td></tr><tr><td><em>IKU2</em> </td><td>AT3G19700 </td><td>Ta01151 </td><td>3 </td><td>991 </td><td>1016 </td><td>79.7 </td><td>86.8 </td></tr><tr><td><em>KLU</em> </td><td>AT1G13710 </td><td>Ta14711 </td><td>213 </td><td>517 </td><td>488 </td><td>84.8 </td><td>88.2 </td></tr><tr><td><em>GL2</em> </td><td>AT1G79840 </td><td>Ta08884 </td><td>25 </td><td>776 </td><td>798 </td><td>82.6 </td><td>86 </td></tr><tr><td><em>MUM4</em> </td><td>AT1G53500 </td><td>Ta23332 </td><td>457 </td><td>667 </td><td>597 </td><td>83.6 </td><td>85.8 </td></tr><tr><td rowspan="3">Seed GSL content </td><td><em>HAG1</em> </td><td>AT3G54610 </td><td>Ta07928 </td><td>20 </td><td>568 </td><td>592 </td><td>85.8 </td><td>88.3 </td></tr><tr><td><em>GTR1</em> </td><td>AT3G47960 </td><td>Ta20037 </td><td>206 </td><td>636 </td><td>651 </td><td>81 </td><td>88.5 </td></tr><tr><td><em>GTR2</em> </td><td>AT5G62680 </td><td>Ta01982 </td><td>4 </td><td>616 </td><td>613 </td><td>90.3 </td><td>94.8 </td></tr><tr><td>Erucic acid content </td><td><em>FAE1</em> </td><td>AT4G34520 </td><td>Ta11742<sup>a</sup> </td><td>45 </td><td>506 </td><td>506 </td><td>86.4 </td><td>91.5 </td></tr><tr><td>Linoleic/Linolenic acid content </td><td><em>FAD2</em> </td><td>AT3G12120 </td><td>Ta12495 </td><td>59 </td><td>383 </td><td>404 </td><td>84.3 </td><td>88.2 </td></tr><tr><td rowspan="6">Seed pod shatter </td><td><em>SHP1</em> </td><td>AT3G58780 </td><td>Ta15094 </td><td>22 </td><td>273 </td><td>248 </td><td>86.8 </td><td>89.4 </td></tr><tr><td><em>SHP2</em> </td><td>AT2G42830 </td><td>Ta08438 </td><td>22 </td><td>248 </td><td>290 </td><td>76.6 </td><td>80.7 </td></tr><tr><td><em>IND</em> </td><td>AT4G00120 </td><td>Ta25465 </td><td>1003 </td><td>198 </td><td>172 </td><td>61.5 </td><td>69 </td></tr><tr><td><em>ALC</em> </td><td>AT5G67110 </td><td>Ta02444 </td><td>6 </td><td>210 </td><td>207 </td><td>72.8 </td><td>77.9 </td></tr><tr><td><em>RPL</em> </td><td>AT5G02030 </td><td>Ta15425 </td><td>92 </td><td>575 </td><td>639 </td><td>80.5 </td><td>83.3 </td></tr><tr><td><em>FUL</em> </td><td>AT5G60910 </td><td>Ta01807 </td><td>57 </td><td>242 </td><td>208 </td><td>63.3 </td><td>68.1 </td></tr><tr><td rowspan="2">Time to flower </td><td><em>FLC</em> </td><td>AT5G10140 </td><td>Ta00917 </td><td>1 </td><td>196 </td><td>203 </td><td>84.3 </td><td>89.7 </td></tr><tr><td><em>FRI</em> </td><td>AT4G00650 </td><td>Ta26225<sup>a</sup> </td><td>1344 </td><td>314 </td><td>367 </td><td>55 </td><td>65.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Identification of pennycress genes with high sequence similarity to <em>Arabidopsis</em> genes controlling key traits of interest, including genomic location and predicted peptide similarity.</p></div></div><div content-id="TN4"><div class="fn"><p><sup>a</sup>MAKER-derived gene models contained errors and were manually corrected to obtain predicted peptide sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588774" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU045TB2" class="table-wrap-title" data-id="DSU045TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Putative orthologues controlling important agronomic traits in pennycress</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait of interest </th><th>Arabidopsis gene </th><th>AGI no. </th><th>Corresponding pennycress predicted peptide ID </th><th>Pennycress genome scaffold no. </th><th>Arabidopsis peptide length </th><th>Pennycress peptide length </th><th>% Identity </th><th>% Positive </th></tr></thead><tbody><tr><td rowspan="4">Seed dormancy </td><td><em>DOG1</em> </td><td>AT5G45830 </td><td>Ta16411 </td><td>141 </td><td>291 </td><td>284 </td><td>72.9 </td><td>80.9 </td></tr><tr><td><em>ABI3</em> </td><td>AT3G24650 </td><td>Ta24539 </td><td>630 </td><td>720 </td><td>728 </td><td>86.8 </td><td>90.6 </td></tr><tr><td><em>ABI4</em> </td><td>AT2G40220 </td><td>Ta07356 </td><td>85 </td><td>328 </td><td>395 </td><td>59.3 </td><td>66.3 </td></tr><tr><td><em>ABI5</em> </td><td>AT2G36270 </td><td>Ta25088 </td><td>828 </td><td>442 </td><td>434 </td><td>85.8 </td><td>89 </td></tr><tr><td rowspan="6">Seed size </td><td><em>WRI1</em> </td><td>AT3G54320 </td><td>Ta07949 </td><td>20 </td><td>438 </td><td>441 </td><td>80.2 </td><td>83.4 </td></tr><tr><td><em>DGAT1</em> </td><td>AT3G51520 </td><td>Ta05453 </td><td>12 </td><td>314 </td><td>566 </td><td>67.7 </td><td>70.1 </td></tr><tr><td><em>IKU2</em> </td><td>AT3G19700 </td><td>Ta01151 </td><td>3 </td><td>991 </td><td>1016 </td><td>79.7 </td><td>86.8 </td></tr><tr><td><em>KLU</em> </td><td>AT1G13710 </td><td>Ta14711 </td><td>213 </td><td>517 </td><td>488 </td><td>84.8 </td><td>88.2 </td></tr><tr><td><em>GL2</em> </td><td>AT1G79840 </td><td>Ta08884 </td><td>25 </td><td>776 </td><td>798 </td><td>82.6 </td><td>86 </td></tr><tr><td><em>MUM4</em> </td><td>AT1G53500 </td><td>Ta23332 </td><td>457 </td><td>667 </td><td>597 </td><td>83.6 </td><td>85.8 </td></tr><tr><td rowspan="3">Seed GSL content </td><td><em>HAG1</em> </td><td>AT3G54610 </td><td>Ta07928 </td><td>20 </td><td>568 </td><td>592 </td><td>85.8 </td><td>88.3 </td></tr><tr><td><em>GTR1</em> </td><td>AT3G47960 </td><td>Ta20037 </td><td>206 </td><td>636 </td><td>651 </td><td>81 </td><td>88.5 </td></tr><tr><td><em>GTR2</em> </td><td>AT5G62680 </td><td>Ta01982 </td><td>4 </td><td>616 </td><td>613 </td><td>90.3 </td><td>94.8 </td></tr><tr><td>Erucic acid content </td><td><em>FAE1</em> </td><td>AT4G34520 </td><td>Ta11742<sup>a</sup> </td><td>45 </td><td>506 </td><td>506 </td><td>86.4 </td><td>91.5 </td></tr><tr><td>Linoleic/Linolenic acid content </td><td><em>FAD2</em> </td><td>AT3G12120 </td><td>Ta12495 </td><td>59 </td><td>383 </td><td>404 </td><td>84.3 </td><td>88.2 </td></tr><tr><td rowspan="6">Seed pod shatter </td><td><em>SHP1</em> </td><td>AT3G58780 </td><td>Ta15094 </td><td>22 </td><td>273 </td><td>248 </td><td>86.8 </td><td>89.4 </td></tr><tr><td><em>SHP2</em> </td><td>AT2G42830 </td><td>Ta08438 </td><td>22 </td><td>248 </td><td>290 </td><td>76.6 </td><td>80.7 </td></tr><tr><td><em>IND</em> </td><td>AT4G00120 </td><td>Ta25465 </td><td>1003 </td><td>198 </td><td>172 </td><td>61.5 </td><td>69 </td></tr><tr><td><em>ALC</em> </td><td>AT5G67110 </td><td>Ta02444 </td><td>6 </td><td>210 </td><td>207 </td><td>72.8 </td><td>77.9 </td></tr><tr><td><em>RPL</em> </td><td>AT5G02030 </td><td>Ta15425 </td><td>92 </td><td>575 </td><td>639 </td><td>80.5 </td><td>83.3 </td></tr><tr><td><em>FUL</em> </td><td>AT5G60910 </td><td>Ta01807 </td><td>57 </td><td>242 </td><td>208 </td><td>63.3 </td><td>68.1 </td></tr><tr><td rowspan="2">Time to flower </td><td><em>FLC</em> </td><td>AT5G10140 </td><td>Ta00917 </td><td>1 </td><td>196 </td><td>203 </td><td>84.3 </td><td>89.7 </td></tr><tr><td><em>FRI</em> </td><td>AT4G00650 </td><td>Ta26225<sup>a</sup> </td><td>1344 </td><td>314 </td><td>367 </td><td>55 </td><td>65.8 </td></tr></tbody></table></div><div content-id="DSU045TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait of interest </th><th>Arabidopsis gene </th><th>AGI no. </th><th>Corresponding pennycress predicted peptide ID </th><th>Pennycress genome scaffold no. </th><th>Arabidopsis peptide length </th><th>Pennycress peptide length </th><th>% Identity </th><th>% Positive </th></tr></thead><tbody><tr><td rowspan="4">Seed dormancy </td><td><em>DOG1</em> </td><td>AT5G45830 </td><td>Ta16411 </td><td>141 </td><td>291 </td><td>284 </td><td>72.9 </td><td>80.9 </td></tr><tr><td><em>ABI3</em> </td><td>AT3G24650 </td><td>Ta24539 </td><td>630 </td><td>720 </td><td>728 </td><td>86.8 </td><td>90.6 </td></tr><tr><td><em>ABI4</em> </td><td>AT2G40220 </td><td>Ta07356 </td><td>85 </td><td>328 </td><td>395 </td><td>59.3 </td><td>66.3 </td></tr><tr><td><em>ABI5</em> </td><td>AT2G36270 </td><td>Ta25088 </td><td>828 </td><td>442 </td><td>434 </td><td>85.8 </td><td>89 </td></tr><tr><td rowspan="6">Seed size </td><td><em>WRI1</em> </td><td>AT3G54320 </td><td>Ta07949 </td><td>20 </td><td>438 </td><td>441 </td><td>80.2 </td><td>83.4 </td></tr><tr><td><em>DGAT1</em> </td><td>AT3G51520 </td><td>Ta05453 </td><td>12 </td><td>314 </td><td>566 </td><td>67.7 </td><td>70.1 </td></tr><tr><td><em>IKU2</em> </td><td>AT3G19700 </td><td>Ta01151 </td><td>3 </td><td>991 </td><td>1016 </td><td>79.7 </td><td>86.8 </td></tr><tr><td><em>KLU</em> </td><td>AT1G13710 </td><td>Ta14711 </td><td>213 </td><td>517 </td><td>488 </td><td>84.8 </td><td>88.2 </td></tr><tr><td><em>GL2</em> </td><td>AT1G79840 </td><td>Ta08884 </td><td>25 </td><td>776 </td><td>798 </td><td>82.6 </td><td>86 </td></tr><tr><td><em>MUM4</em> </td><td>AT1G53500 </td><td>Ta23332 </td><td>457 </td><td>667 </td><td>597 </td><td>83.6 </td><td>85.8 </td></tr><tr><td rowspan="3">Seed GSL content </td><td><em>HAG1</em> </td><td>AT3G54610 </td><td>Ta07928 </td><td>20 </td><td>568 </td><td>592 </td><td>85.8 </td><td>88.3 </td></tr><tr><td><em>GTR1</em> </td><td>AT3G47960 </td><td>Ta20037 </td><td>206 </td><td>636 </td><td>651 </td><td>81 </td><td>88.5 </td></tr><tr><td><em>GTR2</em> </td><td>AT5G62680 </td><td>Ta01982 </td><td>4 </td><td>616 </td><td>613 </td><td>90.3 </td><td>94.8 </td></tr><tr><td>Erucic acid content </td><td><em>FAE1</em> </td><td>AT4G34520 </td><td>Ta11742<sup>a</sup> </td><td>45 </td><td>506 </td><td>506 </td><td>86.4 </td><td>91.5 </td></tr><tr><td>Linoleic/Linolenic acid content </td><td><em>FAD2</em> </td><td>AT3G12120 </td><td>Ta12495 </td><td>59 </td><td>383 </td><td>404 </td><td>84.3 </td><td>88.2 </td></tr><tr><td rowspan="6">Seed pod shatter </td><td><em>SHP1</em> </td><td>AT3G58780 </td><td>Ta15094 </td><td>22 </td><td>273 </td><td>248 </td><td>86.8 </td><td>89.4 </td></tr><tr><td><em>SHP2</em> </td><td>AT2G42830 </td><td>Ta08438 </td><td>22 </td><td>248 </td><td>290 </td><td>76.6 </td><td>80.7 </td></tr><tr><td><em>IND</em> </td><td>AT4G00120 </td><td>Ta25465 </td><td>1003 </td><td>198 </td><td>172 </td><td>61.5 </td><td>69 </td></tr><tr><td><em>ALC</em> </td><td>AT5G67110 </td><td>Ta02444 </td><td>6 </td><td>210 </td><td>207 </td><td>72.8 </td><td>77.9 </td></tr><tr><td><em>RPL</em> </td><td>AT5G02030 </td><td>Ta15425 </td><td>92 </td><td>575 </td><td>639 </td><td>80.5 </td><td>83.3 </td></tr><tr><td><em>FUL</em> </td><td>AT5G60910 </td><td>Ta01807 </td><td>57 </td><td>242 </td><td>208 </td><td>63.3 </td><td>68.1 </td></tr><tr><td rowspan="2">Time to flower </td><td><em>FLC</em> </td><td>AT5G10140 </td><td>Ta00917 </td><td>1 </td><td>196 </td><td>203 </td><td>84.3 </td><td>89.7 </td></tr><tr><td><em>FRI</em> </td><td>AT4G00650 </td><td>Ta26225<sup>a</sup> </td><td>1344 </td><td>314 </td><td>367 </td><td>55 </td><td>65.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Identification of pennycress genes with high sequence similarity to <em>Arabidopsis</em> genes controlling key traits of interest, including genomic location and predicted peptide similarity.</p></div></div><div content-id="TN4"><div class="fn"><p><sup>a</sup>MAKER-derived gene models contained errors and were manually corrected to obtain predicted peptide sequence.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588774" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=4588775 id="4588775" class="section-title" data-legacy-id=s3g>3.7. The future of genomics-based pennycress improvement</h3>
<p>While the annotated draft genome for pennycress presented in this report will provide new tools for the domestication of a new winter oilseed crop, there is still much work to be done to develop a broad base of genomic resources for pennycress. For example, the creation of a complete, anchored reference genome is needed. The increasing length and quality of DNA sequencing reads from third-generation (single molecule—PacBio or Oxford Nanopore) sequencing technologies and new library preparation techniques such as Illumina TruSeq Synthetic Long-Reads (LRseq, formerly Moleculo)<sup><a reveal-id="DSU045C64" data-open="DSU045C64" class="link link-ref link-reveal xref-bibr">64</a></sup> will allow for the <em>de novo</em> assembly of highly contiguous yet complex genomes. Combined with our ongoing development of a high-density genetic map using restriction site-associated DNA sequencing (RADseq),<sup><a reveal-id="DSU045C65" data-open="DSU045C65" class="link link-ref link-reveal xref-bibr">65</a>,<a reveal-id="DSU045C66" data-open="DSU045C66" class="link link-ref link-reveal xref-bibr">66</a></sup> the anchoring and ordering of a highly contiguous genome assembly to the genetic linkage map should be on the immediate horizon to produce a complete reference genome for pennycress. With a completed pennycress genome, the phylogenetic resolution of the Brassicaceae expanded lineage 2 can be improved, including providing tools for the phylogenetic resolution of <em>Thlaspi</em> species,<sup><a reveal-id="DSU045C67" data-open="DSU045C67" class="link link-ref link-reveal xref-bibr">67</a></sup> and answering questions on pennycress genome structure. The draft pennycress genome presented here will enable the development of a genomics-based breeding programme. In addition, the identification of the gene space will allow the use of techniques such as mutation breeding, TILLING of ethyl methanesulfonate-mutagenized populations, DeTILLING of fast neutron-mutagenized populations, and genomic selection. The use of precise genome editing techniques such as CRISPR/Cas9 and TALENs should also be on the horizon, as pennycress can be transformed using the <em>Agrobacterium</em>-mediated vacuum infiltration floral dip method (John Sedbrook, personal communication). Pennycress has the potential to be planted on over 16 million hectares in the United States alone and produce over 22 billion litres of oil suitable as a biodiesel feedstock.<sup><a reveal-id="DSU045C20" data-open="DSU045C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSU045C21" data-open="DSU045C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSU045C24" data-open="DSU045C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The successful domestication of pennycress has the potential to benefit farmers and the environment, provide a new source of biofuel, and reduce greenhouse gases associated with global climate change. The genomics-based domestication of pennycress represents an exciting example of the development of a new crop species.</p>                            <h2 scrollto-destination=4588777 id="4588777" class="section-title" data-legacy-id=s4>Conflict of interest statement</h2>
<p>Any opinions, findings, and conclusions or recommendations expressed in this material are those of the authors, and do not necessarily reflect the views of the US National Science Foundation.</p>                            <h2 scrollto-destination=4588779 id="4588779" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=4588781 id="4588781" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was funded by the UMN Institute on the Environment, UMN CBS, and UMN College of Food, Agriculture, and Natural Resource Sciences, the Minnesota Agricultural Experiment Station, a DOE/USDA Plant Feedstock Genomics for Bioenergy program grant to MDM (USDA 2014-67009-22305), and by US National Science Foundation Graduate Research Fellowships to K.M.D. (00006595) and J.D.F. Funding to pay the Open Access publication charges for this article was provided by a DOE/USDA Plant Feedstock Genomics for Bioenergy program grant to MDM (USDA 2014-67009-22305).</p>                            <h2 scrollto-destination=4588783 id="4588783" class="section-title" data-legacy-id=s7>Data deposition</h2>
<p>The Illumina and Pacific Biosciences sequencing reads for <em>Thlaspi arvense</em> line MN106 are available in the NCBI Sequence Read Archive under the accession number SRP033211. This Whole Genome Shotgun project has been deposited at DDBJ/EMBL/GenBank under the accession number AZNP00000000. The version described in this paper is version AZNP01000000. The genome assembly and related annotation files can also be downloaded from pennycress.umn.edu. </p>                        <a id="4588785" scrollto-destination="4588785"></a>
<div content-id="DSU045TB3" class="table-modal"><div class="table-wrap"><div id="DSU045TB3" class="table-wrap-title" data-id="DSU045TB3"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Accession </th><th>Run </th></tr></thead><tbody><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 100 bp paired end </td><td>SRX380646 </td><td>SRR1034657 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 250 bp paired end </td><td>SRX380649 </td><td>SRR1034659 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (2 kb insert) </td><td>SRX381531 </td><td>SRR1035703 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (3.5 kb insert) </td><td>SRX381541 </td><td>SRR1035705 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (7 kb insert) </td><td>SRX381551 </td><td>SRR1035715 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—PacBio RSII 10 kb insert reads </td><td>SRX380881 </td><td>SRR1035588 </td></tr></tbody></table></div><div content-id="DSU045TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Accession </th><th>Run </th></tr></thead><tbody><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 100 bp paired end </td><td>SRX380646 </td><td>SRR1034657 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 250 bp paired end </td><td>SRX380649 </td><td>SRR1034659 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (2 kb insert) </td><td>SRX381531 </td><td>SRR1035703 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (3.5 kb insert) </td><td>SRX381541 </td><td>SRR1035705 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (7 kb insert) </td><td>SRX381551 </td><td>SRR1035715 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—PacBio RSII 10 kb insert reads </td><td>SRX380881 </td><td>SRR1035588 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588785" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU045TB3" class="table-wrap-title" data-id="DSU045TB3"> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Accession </th><th>Run </th></tr></thead><tbody><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 100 bp paired end </td><td>SRX380646 </td><td>SRR1034657 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 250 bp paired end </td><td>SRX380649 </td><td>SRR1034659 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (2 kb insert) </td><td>SRX381531 </td><td>SRR1035703 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (3.5 kb insert) </td><td>SRX381541 </td><td>SRR1035705 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (7 kb insert) </td><td>SRX381551 </td><td>SRR1035715 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—PacBio RSII 10 kb insert reads </td><td>SRX380881 </td><td>SRR1035588 </td></tr></tbody></table></div><div content-id="DSU045TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Accession </th><th>Run </th></tr></thead><tbody><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 100 bp paired end </td><td>SRX380646 </td><td>SRR1034657 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 250 bp paired end </td><td>SRX380649 </td><td>SRR1034659 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (2 kb insert) </td><td>SRX381531 </td><td>SRR1035703 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (3.5 kb insert) </td><td>SRX381541 </td><td>SRR1035705 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—Illumina 50 bp mate pair (7 kb insert) </td><td>SRX381551 </td><td>SRR1035715 </td></tr><tr><td><em>Thlaspi arvense</em> line MN106—PacBio RSII 10 kb insert reads </td><td>SRX380881 </td><td>SRR1035588 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4588785" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4588786 id="4588786" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Aaron Becker and the University of Minnesota (UMN) Genomics Center provided assistance with experimental planning and generating sequencing data. Justin Kremer and Tom Kautz with the UMN College of Biological Sciences (CBS) Research and Learning Technologies group provided computer assistance and setting up the genome website. We thank Keo Corak for her help with CAPS analysis and Evan Johnson for providing editorial comments. We dedicate this paper to the memory of our colleague and friend, Johnathon D. Fankhauser.</p>                            <h2 scrollto-destination=4588788 id="4588788" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU045C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Jarvis</div> <div class="given-names">D.E.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">The reference genome of the halophytic plant <em>Eutrema salsugineum</em></div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">4</div>, <div class="fpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genome%20of%20the%20halophytic%20plant%20Eutrema%20salsugineum&amp;author=R.%20Yang&amp;author=D.E.%20Jarvis&amp;author=H.%20Chen&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci.&amp;volume=4&amp;pages=46" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23518688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Yu</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Tong</div> <div class="given-names">C.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Genome sequencing of the high oil crop sesame provides insight into oil biosynthesis</div>, <div class="source">Genome Biol.</div> , <div class="volume">15</div>, <div class="fpage">R39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20of%20the%20high%20oil%20crop%20sesame%20provides%20insight%20into%20oil%20biosynthesis&amp;author=L.%20Wang&amp;author=S.%20Yu&amp;author=C.%20Tong&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=R39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-2-r39" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-2-r39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-2-r39"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24576357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kagale</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Koh</div> <div class="given-names">C.S.</div></div> <div class="name"><div class="surname">Nixon</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">The emerging biofuel crop Camelina sativa retains a highly undifferentiated hexaploid genome structure</div>, <div class="source">Nat. Commun.</div> , <div class="volume">5</div>, <div class="fpage">3706</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20emerging%20biofuel%20crop%20Camelina%20sativa%20retains%20a%20highly%20undifferentiated%20hexaploid%20genome%20structure&amp;author=S.%20Kagale&amp;author=C.S.%20Koh&amp;author=J.%20Nixon&amp;publication_year=2014&amp;journal=Nat.%20Commun.&amp;volume=5&amp;pages=3706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms4706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms4706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms4706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24759634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes</div>, <div class="source">Nat. Commun.</div> , <div class="volume">5</div>, <div class="fpage">3930</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Brassica%20oleracea%20genome%20reveals%20the%20asymmetrical%20evolution%20of%20polyploid%20genomes&amp;author=S.%20Liu&amp;author=Y.%20Liu&amp;author=X.%20Yang&amp;publication_year=2014&amp;journal=Nat.%20Commun.&amp;volume=5&amp;pages=3930" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24852848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Slotte</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Hazzouri</div> <div class="given-names">K.M.</div></div> <div class="name"><div class="surname">Agren</div> <div class="given-names">J.A.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">The <em>Capsella rubella</em> genome and the genomic consequences of rapid mating system evolution</div>, <div class="source">Nat. Genet.</div> , <div class="volume">45</div>, <div class="fpage">831</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Capsella%20rubella%20genome%20and%20the%20genomic%20consequences%20of%20rapid%20mating%20system%20evolution&amp;author=T.%20Slotte&amp;author=K.M.%20Hazzouri&amp;author=J.A.%20Agren&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=831-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2669" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2669" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2669"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23749190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div> <div class="name"><div class="surname">Song</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Saxena</div> <div class="given-names">R.K.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Draft genome sequence of chickpea (Cicer arietinum) provides a resource for trait improvement</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">31</div>, <div class="fpage">240</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20chickpea%20(Cicer%20arietinum)%20provides%20a%20resource%20for%20trait%20improvement&amp;author=R.K.%20Varshney&amp;author=C.%20Song&amp;author=R.K.%20Saxena&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol.&amp;volume=31&amp;pages=240-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23354103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div> <div class="name"><div class="surname">Terauchi</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.R.</div></div></div> <div class="year">2014</div>, <div class="article-title">Harvesting the promising fruits of genomics: applying genome sequencing technologies to crop breeding</div>, <div class="source">PLoS Biol.</div> , <div class="volume">12</div>, <div class="fpage">e1001883</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Harvesting%20the%20promising%20fruits%20of%20genomics%3A%20applying%20genome%20sequencing%20technologies%20to%20crop%20breeding&amp;author=R.K.%20Varshney&amp;author=R.%20Terauchi&amp;author=S.R.%20McCouch&amp;publication_year=2014&amp;journal=PLoS%20Biol.&amp;volume=12&amp;pages=e1001883" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1001883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1001883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1001883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24914810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ray</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Satya</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, <div class="article-title">Next generation sequencing technologies for next generation plant breeding</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">5</div>, <div class="fpage">367</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20technologies%20for%20next%20generation%20plant%20breeding&amp;author=S.%20Ray&amp;author=P.%20Satya&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci.&amp;volume=5&amp;pages=367" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2014.00367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2014.00367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2014.00367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25126091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sedbrook</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Phippen</div> <div class="given-names">W.B.</div></div> <div class="name"><div class="surname">Marks</div> <div class="given-names">M.D.</div></div></div> <div class="year">2014</div>, <div class="article-title">New approaches to facilitate rapid domestication of a wild plant to an oilseed crop: example pennycress (Thlaspi arvense L.)</div>, <div class="source">Plant Sci.</div> , <div class="volume">22,7C</div>, <div class="fpage">122</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20approaches%20to%20facilitate%20rapid%20domestication%20of%20a%20wild%20plant%20to%20an%20oilseed%20crop%3A%20example%20pennycress%20(Thlaspi%20arvense%20L.)&amp;author=J.C.%20Sedbrook&amp;author=W.B.%20Phippen&amp;author=M.D.%20Marks&amp;publication_year=2014&amp;journal=Plant%20Sci.&amp;volume=22,7C&amp;pages=122-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2014.07.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2014.07.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2014.07.008"> </span></div> </div></div></div></div></div><div content-id="DSU045C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Best</div> <div class="given-names">K.F.A.</div></div> <div class="name"><div class="surname">Mcintyre</div> <div class="given-names">G.I.</div></div></div> <div class="year">1975</div>, <div class="article-title">The biology of Canadian weeds 9. <em>Thlaspi arvense</em> L</div>, <div class="source">Canad. J. Plant Sci.</div> , <div class="volume">55</div>, <div class="fpage">279</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biology%20of%20Canadian%20weeds%209.%20Thlaspi%20arvense%20L&amp;author=K.F.A.%20Best&amp;author=G.I.%20Mcintyre&amp;publication_year=1975&amp;journal=Canad.%20J.%20Plant%20Sci.&amp;volume=55&amp;pages=279-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4141/cjps75-039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4141%2Fcjps75-039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4141%2Fcjps75-039"> </span></div> </div></div></div></div></div><div content-id="DSU045C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Warwick</div> <div class="given-names">S.I.</div></div> <div class="name"><div class="surname">Francis</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Susko</div> <div class="given-names">D.J.</div></div></div> <div class="year">2002</div>, <div class="article-title">The biology of Canadian weeds. 9. <em>Thlaspi arvense</em> L. (updated)</div>, <div class="source">Canad. J. Plant Sci.</div> , <div class="volume">82</div>, <div class="fpage">803</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biology%20of%20Canadian%20weeds.%209.%20Thlaspi%20arvense%20L.%20(updated)&amp;author=S.I.%20Warwick&amp;author=A.%20Francis&amp;author=D.J.%20Susko&amp;publication_year=2002&amp;journal=Canad.%20J.%20Plant%20Sci.&amp;volume=82&amp;pages=803-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4141/P01-159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4141%2FP01-159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4141%2FP01-159"> </span></div> </div></div></div></div></div><div content-id="DSU045C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Johnston</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Pepper</div> <div class="given-names">A.E.</div></div> <div class="name"><div class="surname">Hall</div> <div class="given-names">A.E.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">Evolution of genome size in Brassicaceae</div>, <div class="source">Ann. Bot.</div> , <div class="volume">95</div>, <div class="fpage">229</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20genome%20size%20in%20Brassicaceae&amp;author=J.S.%20Johnston&amp;author=A.E.%20Pepper&amp;author=A.E.%20Hall&amp;publication_year=2005&amp;journal=Ann.%20Bot.&amp;volume=95&amp;pages=229-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mci016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmci016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmci016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15596470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Mummenhoff</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Franzke</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Koch</div> <div class="given-names">M.</div></div></div> <div class="year">1997</div>, <div class="article-title">Molecular data reveal convergence in fruit characters used in the classification of Thlaspi s. l. (Brassicaceae)</div>, <div class="source">Bot. J. Linn. Soc.</div> , <div class="volume">125</div>, <div class="fpage">183</div>–<div class="lpage">99</div>.</div></div></div></div><div content-id="DSU045C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Milner</div> <div class="given-names">M.J.</div></div> <div class="name"><div class="surname">Mitani-Ueno</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Yamaji</div> <div class="given-names">N.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Root and shoot transcriptome analysis of two ecotypes of Noccaea caerulescens uncovers the role of NcNramp1 in Cd hyperaccumulation</div>, <div class="source">Plant J.</div> , <div class="volume">78</div>, <div class="fpage">398</div>–<div class="lpage">410</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20and%20shoot%20transcriptome%20analysis%20of%20two%20ecotypes%20of%20Noccaea%20caerulescens%20uncovers%20the%20role%20of%20NcNramp1%20in%20Cd%20hyperaccumulation&amp;author=M.J.%20Milner&amp;author=N.%20Mitani-Ueno&amp;author=N.%20Yamaji&amp;publication_year=2014&amp;journal=Plant%20J.&amp;volume=78&amp;pages=398-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.12480" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12480" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.12480"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24547775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Pence</div> <div class="given-names">N.S.</div></div> <div class="name"><div class="surname">Larsen</div> <div class="given-names">P.B.</div></div> <div class="name"><div class="surname">Ebbs</div> <div class="given-names">S.D.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">The molecular physiology of heavy metal transport in the Zn/Cd hyperaccumulator Thlaspi caerulescens</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">97</div>, <div class="fpage">4956</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20physiology%20of%20heavy%20metal%20transport%20in%20the%20Zn%2FCd%20hyperaccumulator%20Thlaspi%20caerulescens&amp;author=N.S.%20Pence&amp;author=P.B.%20Larsen&amp;author=S.D.%20Ebbs&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=4956-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.9.4956" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.9.4956" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.9.4956"> </span></div> </div></div></div></div></div><div content-id="DSU045C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Koch</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">German</div> <div class="given-names">D.A.</div></div></div> <div class="year">2013</div>, <div class="article-title">Taxonomy and systematics are key to biological information: Arabidopsis, Eutrema (Thellungiella), Noccaea and Schrenkiella (Brassicaceae) as examples</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">4</div>, <div class="fpage">267</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taxonomy%20and%20systematics%20are%20key%20to%20biological%20information%3A%20Arabidopsis%2C%20Eutrema%20(Thellungiella)%2C%20Noccaea%20and%20Schrenkiella%20(Brassicaceae)%20as%20examples&amp;author=M.A.%20Koch&amp;author=D.A.%20German&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci.&amp;volume=4&amp;pages=267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2013.00267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2013.00267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2013.00267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23914192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Franzke</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Al-Shehbaz</div> <div class="given-names">I.A.</div></div> <div class="name"><div class="surname">Koch</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Mummenhoff</div> <div class="given-names">K.</div></div></div> <div class="year">2011</div>, <div class="article-title">Cabbage family affairs: the evolutionary history of Brassicaceae</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">16</div>, <div class="fpage">108</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cabbage%20family%20affairs%3A%20the%20evolutionary%20history%20of%20Brassicaceae&amp;author=A.%20Franzke&amp;author=M.A.%20Lysak&amp;author=I.A.%20Al-Shehbaz&amp;author=M.A.%20Koch&amp;author=K.%20Mummenhoff&amp;publication_year=2011&amp;journal=Trends%20Plant%20Sci.&amp;volume=16&amp;pages=108-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moser</div> <div class="given-names">B.R.</div></div> <div class="name"><div class="surname">Shah</div> <div class="given-names">S.N.</div></div> <div class="name"><div class="surname">Winkler-Moser</div> <div class="given-names">J.K.</div></div> <div class="name"><div class="surname">Vaughn</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Evangelista</div> <div class="given-names">R.L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Composition and physical properties of cress (Lepidium sativum L.) and field pennycress (Thlaspi arvense L.) oils</div>, <div class="source">Ind. Crops Prod.</div> , <div class="volume">30</div>, <div class="fpage">199</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composition%20and%20physical%20properties%20of%20cress%20(Lepidium%20sativum%20L.)%20and%20field%20pennycress%20(Thlaspi%20arvense%20L.)%20oils&amp;author=B.R.%20Moser&amp;author=S.N.%20Shah&amp;author=J.K.%20Winkler-Moser&amp;author=S.F.%20Vaughn&amp;author=R.L.%20Evangelista&amp;publication_year=2009&amp;journal=Ind.%20Crops%20Prod.&amp;volume=30&amp;pages=199-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.indcrop.2009.03.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.indcrop.2009.03.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.indcrop.2009.03.007"> </span></div> </div></div></div></div></div><div content-id="DSU045C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dorn</div> <div class="given-names">K.M.</div></div> <div class="name"><div class="surname">Fankhauser</div> <div class="given-names">J.D.</div></div> <div class="name"><div class="surname">Wyse</div> <div class="given-names">D.L.</div></div> <div class="name"><div class="surname">Marks</div> <div class="given-names">M.D.</div></div></div> <div class="year">2013</div>, <div class="article-title">De novo assembly of the pennycress (<em>Thlaspi arvense</em>) transcriptome provides tools for the development of a winter cover crop and biodiesel feedstock</div>, <div class="source">Plant J.</div> , <div class="volume">75</div>, <div class="fpage">1028</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20pennycress%20(Thlaspi%20arvense)%20transcriptome%20provides%20tools%20for%20the%20development%20of%20a%20winter%20cover%20crop%20and%20biodiesel%20feedstock&amp;author=K.M.%20Dorn&amp;author=J.D.%20Fankhauser&amp;author=D.L.%20Wyse&amp;author=M.D.%20Marks&amp;publication_year=2013&amp;journal=Plant%20J.&amp;volume=75&amp;pages=1028-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.12267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.12267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.12267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23786378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">J.Q.</div></div> <div class="name"><div class="surname">Shonnard</div> <div class="given-names">D.R.</div></div> <div class="name"><div class="surname">Kalnes</div> <div class="given-names">T.N.</div></div> <div class="name"><div class="surname">Johnsen</div> <div class="given-names">P.B.</div></div> <div class="name"><div class="surname">Rao</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, <div class="article-title">A life cycle assessment of pennycress (<em>Thlaspi aruense</em> L.) -derived jet fuel and diesel</div>, <div class="source">Biomass Bioenergy</div> , <div class="volume">55</div>, <div class="fpage">87</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20life%20cycle%20assessment%20of%20pennycress%20(Thlaspi%20aruense%20L.)%20-derived%20jet%20fuel%20and%20diesel&amp;author=J.Q.%20Fan&amp;author=D.R.%20Shonnard&amp;author=T.N.%20Kalnes&amp;author=P.B.%20Johnsen&amp;author=S.%20Rao&amp;publication_year=2013&amp;journal=Biomass%20Bioenergy&amp;volume=55&amp;pages=87-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biombioe.2012.12.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biombioe.2012.12.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biombioe.2012.12.040"> </span></div> </div></div></div></div></div><div content-id="DSU045C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moser</div> <div class="given-names">B.R.</div></div> <div class="name"><div class="surname">Knothe</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Vaughn</div> <div class="given-names">S.F.</div></div> <div class="name"><div class="surname">Isbell</div> <div class="given-names">T.A.</div></div></div> <div class="year">2009</div>, <div class="article-title">Production and evaluation of biodiesel from field pennycress (<em>Thlaspi arvense</em> L.) oil</div>, <div class="source">Energy Fuels</div> , <div class="volume">23</div>, <div class="fpage">4149</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20and%20evaluation%20of%20biodiesel%20from%20field%20pennycress%20(Thlaspi%20arvense%20L.)%20oil&amp;author=B.R.%20Moser&amp;author=G.%20Knothe&amp;author=S.F.%20Vaughn&amp;author=T.A.%20Isbell&amp;publication_year=2009&amp;journal=Energy%20Fuels&amp;volume=23&amp;pages=4149-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/ef900337g" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fef900337g" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fef900337g"> </span></div> </div></div></div></div></div><div content-id="DSU045C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Phippen</div> <div class="given-names">W.B.</div></div> <div class="name"><div class="surname">Phippen</div> <div class="given-names">M.E.</div></div></div> <div class="year">2012</div>, <div class="article-title">Soybean seed yield and quality as a response to field pennycress residue</div>, <div class="source">Crop Sci.</div> , <div class="volume">52</div>, <div class="fpage">2767</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soybean%20seed%20yield%20and%20quality%20as%20a%20response%20to%20field%20pennycress%20residue&amp;author=W.B.%20Phippen&amp;author=M.E.%20Phippen&amp;publication_year=2012&amp;journal=Crop%20Sci.&amp;volume=52&amp;pages=2767-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2012.03.0192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2012.03.0192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2012.03.0192"> </span></div> </div></div></div></div></div><div content-id="DSU045C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tilman</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Cassman</div> <div class="given-names">K.G.</div></div> <div class="name"><div class="surname">Matson</div> <div class="given-names">P.A.</div></div> <div class="name"><div class="surname">Naylor</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Polasky</div> <div class="given-names">S.</div></div></div> <div class="year">2002</div>, <div class="article-title">Agricultural sustainability and intensive production practices</div>, <div class="source">Nature</div> , <div class="volume">418</div>, <div class="fpage">671</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Agricultural%20sustainability%20and%20intensive%20production%20practices&amp;author=D.%20Tilman&amp;author=K.G.%20Cassman&amp;author=P.A.%20Matson&amp;author=R.%20Naylor&amp;author=S.%20Polasky&amp;publication_year=2002&amp;journal=Nature&amp;volume=418&amp;pages=671-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12167873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Winchester</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">McConnachie</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Wollersheim</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Waitz</div> <div class="given-names">I.</div></div></div> <div class="year">2013</div>, <div class="article-title">Market cost of renewable jet fuel adoption in the United States</div>, <div class="comment">MIT Joint Program on the Science and Policy of Global Change. <a class="link link-uri" href="http://globalchange.mit.edu/files/document/MITJPSPGC_Rpt238.pdf">http://globalchange.mit.edu/files/document/MITJPSPGC_Rpt238.pdf</a></div>.</div></div></div></div><div content-id="DSU045C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Karimmojeni</div> <div class="given-names">H.T.</div></div> <div class="name"><div class="surname">Taab</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Rashidi</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Bazrafshan</div> <div class="given-names">A.H.</div></div></div> <div class="year">2014</div>, <div class="article-title">Dormancy breaking and seed germination of the annual weeds <em>Thlaspi arvense</em>, <em>Descurainia sophia</em> and <em>Malcolmia africana</em> (Brassicaceae)</div>, <div class="source">J. Plant Protect. Res.</div> , <div class="volume">54</div>, <div class="fpage">179</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dormancy%20breaking%20and%20seed%20germination%20of%20the%20annual%20weeds%20Thlaspi%20arvense%2C%20Descurainia%20sophia%20and%20Malcolmia%20africana%20(Brassicaceae)&amp;author=H.T.%20Karimmojeni&amp;author=A.%20Taab&amp;author=B.%20Rashidi&amp;author=A.H.%20Bazrafshan&amp;publication_year=2014&amp;journal=J.%20Plant%20Protect.%20Res.&amp;volume=54&amp;pages=179-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2478/jppr-2014-0028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2478%2Fjppr-2014-0028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2478%2Fjppr-2014-0028"> </span></div> </div></div></div></div></div><div content-id="DSU045C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Saini</div> <div class="given-names">H.S.</div></div> <div class="name"><div class="surname">Bassi</div> <div class="given-names">P.K.</div></div> <div class="name"><div class="surname">Goudey</div> <div class="given-names">J.S.</div></div> <div class="name"><div class="surname">Spencer</div> <div class="given-names">M.S.</div></div></div> <div class="year">1987</div>, <div class="article-title">Breakage of seed dormancy of field pennycress (<em>Thlaspi arvense</em>) by growth-regulators, nitrate, and environmental-factors</div>, <div class="source">Weed Sci.</div> , <div class="volume">35</div>, <div class="fpage">802</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSU045C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Evangelista</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Isbell</div> <div class="given-names">T.A.</div></div> <div class="name"><div class="surname">Cermak</div> <div class="given-names">S.C.</div></div></div> <div class="year">2012</div>, <div class="article-title">Extraction of pennycress (Thlaspi arvense L.) seed oil by full pressing</div>, <div class="source">Ind. Crops Prod.</div> , <div class="volume">37</div>, <div class="fpage">76</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraction%20of%20pennycress%20(Thlaspi%20arvense%20L.)%20seed%20oil%20by%20full%20pressing&amp;author=R.L.%20Evangelista&amp;author=T.A.%20Isbell&amp;author=S.C.%20Cermak&amp;publication_year=2012&amp;journal=Ind.%20Crops%20Prod.&amp;volume=37&amp;pages=76-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.indcrop.2011.12.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.indcrop.2011.12.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.indcrop.2011.12.003"> </span></div> </div></div></div></div></div><div content-id="DSU045C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Koornneef</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Meinke</div> <div class="given-names">D.</div></div></div> <div class="year">2010</div>, <div class="article-title">The development of Arabidopsis as a model plant</div>, <div class="source">Plant J.</div> , <div class="volume">61</div>, <div class="fpage">909</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20development%20of%20Arabidopsis%20as%20a%20model%20plant&amp;author=M.%20Koornneef&amp;author=D.%20Meinke&amp;publication_year=2010&amp;journal=Plant%20J.&amp;volume=61&amp;pages=909-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2010.61.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2010.61.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2010.61.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20409266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Baud</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Lepiniec</div> <div class="given-names">L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Regulation of de novo fatty acid synthesis in maturing oilseeds of Arabidopsis</div>, <div class="source">Plant Physiol. Biochem.</div> , <div class="volume">47</div>, <div class="fpage">448</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20de%20novo%20fatty%20acid%20synthesis%20in%20maturing%20oilseeds%20of%20Arabidopsis&amp;author=S.%20Baud&amp;author=L.%20Lepiniec&amp;publication_year=2009&amp;journal=Plant%20Physiol.%20Biochem.&amp;volume=47&amp;pages=448-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plaphy.2008.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plaphy.2008.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plaphy.2008.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19136270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Holdsworth</div> <div class="given-names">M.J.</div></div> <div class="name"><div class="surname">Bentsink</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Soppe</div> <div class="given-names">W.J.</div></div></div> <div class="year">2008</div>, <div class="article-title">Molecular networks regulating Arabidopsis seed maturation, after-ripening, dormancy and germination</div>, <div class="source">New Phytol.</div> , <div class="volume">179</div>, <div class="fpage">33</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20networks%20regulating%20Arabidopsis%20seed%20maturation%2C%20after-ripening%2C%20dormancy%20and%20germination&amp;author=M.J.%20Holdsworth&amp;author=L.%20Bentsink&amp;author=W.J.%20Soppe&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=33-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18422904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Jung</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Muller</div> <div class="given-names">A.E.</div></div></div> <div class="year">2009</div>, <div class="article-title">Flowering time control and applications in plant breeding</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">14</div>, <div class="fpage">563</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flowering%20time%20control%20and%20applications%20in%20plant%20breeding&amp;author=C.%20Jung&amp;author=A.E.%20Muller&amp;publication_year=2009&amp;journal=Trends%20Plant%20Sci.&amp;volume=14&amp;pages=563-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2009.07.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2009.07.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2009.07.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19716745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">McCallum</div> <div class="given-names">C.M.</div></div> <div class="name"><div class="surname">Comai</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Greene</div> <div class="given-names">E.A.</div></div> <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div> <div class="year">2000</div>, <div class="article-title">Targeting induced local lesions IN genomes (TILLING) for plant functional genomics</div>, <div class="source">Plant Physiol.</div> , <div class="volume">123</div>, <div class="fpage">439</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeting%20induced%20local%20lesions%20IN%20genomes%20(TILLING)%20for%20plant%20functional%20genomics&amp;author=C.M.%20McCallum&amp;author=L.%20Comai&amp;author=E.A.%20Greene&amp;author=S.%20Henikoff&amp;publication_year=2000&amp;journal=Plant%20Physiol.&amp;volume=123&amp;pages=439-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.123.2.439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.123.2.439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.123.2.439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10859174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rogers</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Wen</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Oldroyd</div> <div class="given-names">G.</div></div></div> <div class="year">2009</div>, <div class="article-title">Deletion-based reverse genetics in Medicago truncatula</div>, <div class="source">Plant Physiol.</div> , <div class="volume">151</div>, <div class="fpage">1077</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion-based%20reverse%20genetics%20in%20Medicago%20truncatula&amp;author=C.%20Rogers&amp;author=J.%20Wen&amp;author=R.%20Chen&amp;author=G.%20Oldroyd&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=1077-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.142919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.142919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.142919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19759346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Belhaj</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Chaparro-Garcia</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Kamoun</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Nekrasov</div> <div class="given-names">V.</div></div></div> <div class="year">2013</div>, <div class="article-title">Plant genome editing made easy: targeted mutagenesis in model and crop plants using the CRISPR/Cas system</div>, <div class="source">Plant Methods</div> , <div class="volume">9</div>, <div class="fpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20genome%20editing%20made%20easy%3A%20targeted%20mutagenesis%20in%20model%20and%20crop%20plants%20using%20the%20CRISPR%2FCas%20system&amp;author=K.%20Belhaj&amp;author=A.%20Chaparro-Garcia&amp;author=S.%20Kamoun&amp;author=V.%20Nekrasov&amp;publication_year=2013&amp;journal=Plant%20Methods&amp;volume=9&amp;pages=39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1746-4811-9-39" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1746-4811-9-39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1746-4811-9-39"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24112467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Olszewski</div> <div class="given-names">N.E.</div></div> <div class="name"><div class="surname">Martin</div> <div class="given-names">F.B.</div></div> <div class="name"><div class="surname">Ausubel</div> <div class="given-names">F.M.</div></div></div> <div class="year">1988</div>, <div class="article-title">Specialized binary vector for plant transformation: expression of the Arabidopsis thaliana AHAS gene in Nicotiana tabacum</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">16</div>, <div class="fpage">10765</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specialized%20binary%20vector%20for%20plant%20transformation%3A%20expression%20of%20the%20Arabidopsis%20thaliana%20AHAS%20gene%20in%20Nicotiana%20tabacum&amp;author=N.E.%20Olszewski&amp;author=F.B.%20Martin&amp;author=F.M.%20Ausubel&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=10765-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.22.10765" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.22.10765" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.22.10765"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3060849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="collab">International Barley Genome Sequencing Consortium</div><div class="person-group"><div class="name"><div class="surname">Mayer</div> <div class="given-names">K.F.</div></div> <div class="name"><div class="surname">Waugh</div> <div class="given-names">R.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">A physical, genetic and functional sequence assembly of the barley genome</div>, <div class="source">Nature</div> , <div class="volume">491</div>, <div class="fpage">711</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20physical%2C%20genetic%20and%20functional%20sequence%20assembly%20of%20the%20barley%20genome&amp;author=K.F.%20Mayer&amp;author=R.%20Waugh&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=711-6" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rahman</div> <div class="given-names">A.Y.</div></div> <div class="name"><div class="surname">Usharraj</div> <div class="given-names">A.O.</div></div> <div class="name"><div class="surname">Misra</div> <div class="given-names">B.B.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Draft genome sequence of the rubber tree <em>Hevea brasiliensis</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20rubber%20tree%20Hevea%20brasiliensis&amp;author=A.Y.%20Rahman&amp;author=A.O.%20Usharraj&amp;author=B.B.%20Misra&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23375136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Henkel</div> <div class="given-names">C.V.</div></div> <div class="name"><div class="surname">Jansen</div> <div class="given-names">H.J.</div></div> <div class="name"><div class="surname">Butler</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, <div class="article-title">Scaffolding pre-assembled contigs using SSPACE</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">578</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scaffolding%20pre-assembled%20contigs%20using%20SSPACE&amp;author=M.%20Boetzer&amp;author=C.V.%20Henkel&amp;author=H.J.%20Jansen&amp;author=D.%20Butler&amp;author=W.%20Pirovano&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=578-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Boetzer</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Pirovano</div> <div class="given-names">W.</div></div></div> <div class="year">2012</div>, <div class="article-title">Toward almost closed genomes with GapFiller</div>, <div class="source">Genome Biol.</div> , <div class="volume">13</div>, <div class="fpage">R56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20almost%20closed%20genomes%20with%20GapFiller&amp;author=M.%20Boetzer&amp;author=W.%20Pirovano&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=R56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-6-r56" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-6-r56" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-6-r56"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22731987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Cantarel</div> <div class="given-names">B.L.</div></div> <div class="name"><div class="surname">Korf</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Robb</div> <div class="given-names">S.M.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">188</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAKER%3A%20an%20easy-to-use%20annotation%20pipeline%20designed%20for%20emerging%20model%20organism%20genomes&amp;author=B.L.%20Cantarel&amp;author=I.%20Korf&amp;author=S.M.%20Robb&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=188-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6743907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6743907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6743907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18025269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Arabidopsis Genome Initiative</div></div></div>. <div class="year">2000</div>, <div class="article-title">Analysis of the genome sequence of the flowering plant Arabidopsis thaliana</div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>–<div class="lpage">815</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genome%20sequence%20of%20the%20flowering%20plant%20Arabidopsis%20thaliana&amp;author=%20Arabidopsis%20Genome%20Initiative&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=796-815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">T.T.</div></div> <div class="name"><div class="surname">Pattyn</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Bakker</div> <div class="given-names">E.G.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">The Arabidopsis lyrata genome sequence and the basis of rapid genome size change</div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">476</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20lyrata%20genome%20sequence%20and%20the%20basis%20of%20rapid%20genome%20size%20change&amp;author=T.T.%20Hu&amp;author=P.%20Pattyn&amp;author=E.G.%20Bakker&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=476-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.807"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">The genome of the mesopolyploid crop species Brassica rapa</div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">1035</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">C.H.</div></div> <div class="name"><div class="surname">Kuo</div> <div class="given-names">T.C.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">M.H.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Identification of cucurbitacins and assembly of a draft genome for Aquilaria agallocha</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">578</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20cucurbitacins%20and%20assembly%20of%20a%20draft%20genome%20for%20Aquilaria%20agallocha&amp;author=C.H.%20Chen&amp;author=T.C.%20Kuo&amp;author=M.H.%20Yang&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=578" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-578"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25005802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">The draft genome of watermelon (<em>Citrullus lanatus</em>) and resequencing of 20 diverse accessions</div>, <div class="source">Nat. Genet.</div> , <div class="volume">45</div>, <div class="fpage">51</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20watermelon%20(Citrullus%20lanatus)%20and%20resequencing%20of%2020%20diverse%20accessions&amp;author=S.%20Guo&amp;author=J.%20Zhang&amp;author=H.%20Sun&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=51-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2470" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2470" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2470"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23179023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Smit</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hubley</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div></div> <div class="year">1996–2010</div>, <div class="comment">RepeatMasker Open-3.0</div>.</div></div></div></div><div content-id="DSU045C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Westesson</div> <div class="given-names">O.</div></div> <div class="name"><div class="surname">Skinner</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Holmes</div> <div class="given-names">I.</div></div></div> <div class="year">2013</div>, <div class="article-title">Visualizing next-generation sequencing data with JBrowse</div>, <div class="source">Brief. Bioinform.</div> , <div class="volume">14</div>, <div class="fpage">172</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualizing%20next-generation%20sequencing%20data%20with%20JBrowse&amp;author=O.%20Westesson&amp;author=M.%20Skinner&amp;author=I.%20Holmes&amp;publication_year=2013&amp;journal=Brief.%20Bioinform.&amp;volume=14&amp;pages=172-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbr078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbr078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbr078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22411711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">H.J.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">J.Y.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Insights into salt tolerance from the genome of <em>Thellungiella salsuginea</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">12219</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20salt%20tolerance%20from%20the%20genome%20of%20Thellungiella%20salsuginea&amp;author=H.J.%20Wu&amp;author=Z.%20Zhang&amp;author=J.Y.%20Wang&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=12219-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1209954109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1209954109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1209954109"> </span></div> </div></div></div></div></div><div content-id="DSU045C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Konieczny</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Ausubel</div> <div class="given-names">F.M.</div></div></div> <div class="year">1993</div>, <div class="article-title">A procedure for mapping arabidopsis mutations using codominant ecotype-specific pcr-based markers</div>, <div class="source">Plant J.</div> , <div class="volume">4</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20procedure%20for%20mapping%20arabidopsis%20mutations%20using%20codominant%20ecotype-specific%20pcr-based%20markers&amp;author=A.%20Konieczny&amp;author=F.M.%20Ausubel&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=4&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1993.04020403.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1993.04020403.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1993.04020403.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8106085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div> <div class="name"><div class="surname">Gershenzon</div> <div class="given-names">J.</div></div></div> <div class="year">2006</div>, <div class="article-title">Biology and biochemistry of glucosinolates</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="volume">57</div>, <div class="fpage">303</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20and%20biochemistry%20of%20glucosinolates&amp;author=B.A.%20Halkier&amp;author=J.%20Gershenzon&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=57&amp;pages=303-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.57.032905.105228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.57.032905.105228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.57.032905.105228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16669764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Grubb</div> <div class="given-names">C.D.</div></div> <div class="name"><div class="surname">Abel</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, <div class="article-title">Glucosinolate metabolism and its control</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">11</div>, <div class="fpage">89</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glucosinolate%20metabolism%20and%20its%20control&amp;author=C.D.%20Grubb&amp;author=S.%20Abel&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=89-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2005.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2005.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2005.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16406306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuchernig</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Wittstock</div> <div class="given-names">U.</div></div></div> <div class="year">2012</div>, <div class="article-title">Evolution of specifier proteins in glucosinolate-containing plants</div>, <div class="source">BMC Evol. Biol.</div> , <div class="volume">12</div>, <div class="fpage">127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20specifier%20proteins%20in%20glucosinolate-containing%20plants&amp;author=J.C.%20Kuchernig&amp;author=M.%20Burow&amp;author=U.%20Wittstock&amp;publication_year=2012&amp;journal=BMC%20Evol.%20Biol.&amp;volume=12&amp;pages=127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-12-127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-12-127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-12-127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22839361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gmelin</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Virtanen</div> <div class="given-names">A.I.</div></div></div> <div class="year">1959</div>, <div class="article-title">A new type of enzymatic cleavage of mustard oil glucosides. Formation of allylthiocyanate in <em>Thlaspi arvense</em> L and benzylthiocynate in <em>Lepidium ruderale</em> L and <em>Lepidium sativum</em> L</div>, <div class="source">Acta Chem. Scand.</div> , <div class="volume">13</div>, <div class="fpage">1474</div>–<div class="lpage">1475</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20type%20of%20enzymatic%20cleavage%20of%20mustard%20oil%20glucosides.%20Formation%20of%20allylthiocyanate%20in%20Thlaspi%20arvense%20L%20and%20benzylthiocynate%20in%20Lepidium%20ruderale%20L%20and%20Lepidium%20sativum%20L&amp;author=R.%20Gmelin&amp;author=A.I.%20Virtanen&amp;publication_year=1959&amp;journal=Acta%20Chem.%20Scand.&amp;volume=13&amp;pages=1474-1475" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3891/acta.chem.scand.13-1474" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3891%2Facta.chem.scand.13-1474" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3891%2Facta.chem.scand.13-1474"> </span></div> </div></div></div></div></div><div content-id="DSU045C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuchernig</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Backenkohler</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lubbecke</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Wittstock</div> <div class="given-names">U.</div></div></div> <div class="year">2011</div>, <div class="article-title">A thiocyanate-forming protein generates multiple products upon allylglucosinolate breakdown in <em>Thlaspi arvense</em></div>, <div class="source">Phytochemistry</div> , <div class="volume">72</div>, <div class="fpage">1699</div>–<div class="lpage">709</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20thiocyanate-forming%20protein%20generates%20multiple%20products%20upon%20allylglucosinolate%20breakdown%20in%20Thlaspi%20arvense&amp;author=J.C.%20Kuchernig&amp;author=A.%20Backenkohler&amp;author=M.%20Lubbecke&amp;author=M.%20Burow&amp;author=U.%20Wittstock&amp;publication_year=2011&amp;journal=Phytochemistry&amp;volume=72&amp;pages=1699-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.phytochem.2011.06.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.phytochem.2011.06.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.phytochem.2011.06.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21783213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hojilla-Evangelista</div> <div class="given-names">M.P.</div></div> <div class="name"><div class="surname">Evangelista</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Isbell</div> <div class="given-names">T.A.</div></div> <div class="name"><div class="surname">Selling</div> <div class="given-names">G.W.</div></div></div> <div class="year">2013</div>, <div class="article-title">Effects of cold-pressing and seed cooking on functional properties of protein in pennycress (<em>Thlaspi arvense</em> L.) seed and press cakes</div>, <div class="source">Ind. Crops Prod.</div> , <div class="volume">45</div>, <div class="fpage">223</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20cold-pressing%20and%20seed%20cooking%20on%20functional%20properties%20of%20protein%20in%20pennycress%20(Thlaspi%20arvense%20L.)%20seed%20and%20press%20cakes&amp;author=M.P.%20Hojilla-Evangelista&amp;author=R.L.%20Evangelista&amp;author=T.A.%20Isbell&amp;author=G.W.%20Selling&amp;publication_year=2013&amp;journal=Ind.%20Crops%20Prod.&amp;volume=45&amp;pages=223-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.indcrop.2012.12.026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.indcrop.2012.12.026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.indcrop.2012.12.026"> </span></div> </div></div></div></div></div><div content-id="DSU045C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Selling</div> <div class="given-names">G.W.</div></div> <div class="name"><div class="surname">Hojilla-Evangelista</div> <div class="given-names">M.P.</div></div> <div class="name"><div class="surname">Evangelista</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Isbell</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Price</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Doll</div> <div class="given-names">K.M.</div></div></div> <div class="year">2013</div>, <div class="article-title">Extraction of proteins from pennycress seeds and press cake</div>, <div class="source">Ind. Crops Prod.</div> , <div class="volume">41</div>, <div class="fpage">113</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraction%20of%20proteins%20from%20pennycress%20seeds%20and%20press%20cake&amp;author=G.W.%20Selling&amp;author=M.P.%20Hojilla-Evangelista&amp;author=R.L.%20Evangelista&amp;author=T.%20Isbell&amp;author=N.%20Price&amp;author=K.M.%20Doll&amp;publication_year=2013&amp;journal=Ind.%20Crops%20Prod.&amp;volume=41&amp;pages=113-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.indcrop.2012.04.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.indcrop.2012.04.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.indcrop.2012.04.009"> </span></div> </div></div></div></div></div><div content-id="DSU045C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Agee</div> <div class="given-names">A.E.</div></div> <div class="name"><div class="surname">Surpin</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Sohn</div> <div class="given-names">E.J.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">MODIFIED VACUOLE PHENOTYPE1 is an Arabidopsis myrosinase-associated protein involved in endomembrane protein trafficking</div>, <div class="source">Plant Physiol.</div> , <div class="volume">152</div>, <div class="fpage">120</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MODIFIED%20VACUOLE%20PHENOTYPE1%20is%20an%20Arabidopsis%20myrosinase-associated%20protein%20involved%20in%20endomembrane%20protein%20trafficking&amp;author=A.E.%20Agee&amp;author=M.%20Surpin&amp;author=E.J.%20Sohn&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=152&amp;pages=120-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.145078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.145078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.145078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19880612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Ober</div> <div class="given-names">J.A.</div></div> <div class="name"><div class="surname">Kliebenstein</div> <div class="given-names">D.J.</div></div></div> <div class="year">2006</div>, <div class="article-title">The gene controlling the quantitative trait locus EPITHIOSPECIFIER MODIFIER1 alters glucosinolate hydrolysis and insect resistance in Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">1524</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20gene%20controlling%20the%20quantitative%20trait%20locus%20EPITHIOSPECIFIER%20MODIFIER1%20alters%20glucosinolate%20hydrolysis%20and%20insect%20resistance%20in%20Arabidopsis&amp;author=Z.%20Zhang&amp;author=J.A.%20Ober&amp;author=D.J.%20Kliebenstein&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1524-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.039602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.039602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.039602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16679459" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hofberger</div> <div class="given-names">J.A.</div></div> <div class="name"><div class="surname">Lyons</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Edger</div> <div class="given-names">P.P.</div></div> <div class="name"><div class="surname">Chris Pires</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Eric Schranz</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, <div class="article-title">Whole genome and tandem duplicate retention facilitated glucosinolate pathway diversification in the mustard family</div>, <div class="source">Genome Biol. Evol.</div> , <div class="volume">5</div>, <div class="fpage">2155</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20and%20tandem%20duplicate%20retention%20facilitated%20glucosinolate%20pathway%20diversification%20in%20the%20mustard%20family&amp;author=J.A.%20Hofberger&amp;author=E.%20Lyons&amp;author=P.P.%20Edger&amp;author=J.%20Chris%20Pires&amp;author=M.%20Eric%20Schranz&amp;publication_year=2013&amp;journal=Genome%20Biol.%20Evol.&amp;volume=5&amp;pages=2155-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evt162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevt162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevt162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24171911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nour-Eldin</div> <div class="given-names">H.H.</div></div> <div class="name"><div class="surname">Andersen</div> <div class="given-names">T.G.</div></div> <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">NRT/PTR transporters are essential for translocation of glucosinolate defence compounds to seeds</div>, <div class="source">Nature</div> , <div class="volume">488</div>, <div class="fpage">531</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NRT%2FPTR%20transporters%20are%20essential%20for%20translocation%20of%20glucosinolate%20defence%20compounds%20to%20seeds&amp;author=H.H.%20Nour-Eldin&amp;author=T.G.%20Andersen&amp;author=M.%20Burow&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=531-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22864417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Andersen</div> <div class="given-names">T.G.</div></div> <div class="name"><div class="surname">Nour-Eldin</div> <div class="given-names">H.H.</div></div> <div class="name"><div class="surname">Fuller</div> <div class="given-names">V.L.</div></div> <div class="name"><div class="surname">Olsen</div> <div class="given-names">C.E.</div></div> <div class="name"><div class="surname">Burow</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Halkier</div> <div class="given-names">B.A.</div></div></div> <div class="year">2013</div>, <div class="article-title">Integration of biosynthesis and long-distance transport establish organ-specific glucosinolate profiles in vegetative Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3133</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20biosynthesis%20and%20long-distance%20transport%20establish%20organ-specific%20glucosinolate%20profiles%20in%20vegetative%20Arabidopsis&amp;author=T.G.%20Andersen&amp;author=H.H.%20Nour-Eldin&amp;author=V.L.%20Fuller&amp;author=C.E.%20Olsen&amp;author=M.%20Burow&amp;author=B.A.%20Halkier&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3133-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.110890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.110890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.110890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23995084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bentsink</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Jowett</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Hanhart</div> <div class="given-names">C.J.</div></div> <div class="name"><div class="surname">Koornneef</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, <div class="article-title">Cloning of DOG1, a quantitative trait locus controlling seed dormancy in Arabidopsis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">103</div>, <div class="fpage">17042</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20of%20DOG1%2C%20a%20quantitative%20trait%20locus%20controlling%20seed%20dormancy%20in%20Arabidopsis&amp;author=L.%20Bentsink&amp;author=J.%20Jowett&amp;author=C.J.%20Hanhart&amp;author=M.%20Koornneef&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=17042-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0607877103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0607877103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0607877103"> </span></div> </div></div></div></div></div><div content-id="DSU045C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Nakabayashi</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Bartsch</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Xiang</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">The time required for dormancy release in arabidopsis is determined by DELAY OF GERMINATION1 protein levels in freshly harvested seeds</div>, <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">2826</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20time%20required%20for%20dormancy%20release%20in%20arabidopsis%20is%20determined%20by%20DELAY%20OF%20GERMINATION1%20protein%20levels%20in%20freshly%20harvested%20seeds&amp;author=K.%20Nakabayashi&amp;author=M.%20Bartsch&amp;author=Y.%20Xiang&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=2826-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.100214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.100214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.100214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22829147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Voskoboynik</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Neff</div> <div class="given-names">N.F.</div></div> <div class="name"><div class="surname">Sahoo</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">The genome sequence of the colonial chordate, Botryllus Schlosseri</div>, <div class="source">Elife</div> , <div class="volume">2</div>, <div class="fpage">e00569</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20colonial%20chordate%2C%20Botryllus%20Schlosseri&amp;author=A.%20Voskoboynik&amp;author=N.F.%20Neff&amp;author=D.%20Sahoo&amp;publication_year=2013&amp;journal=Elife&amp;volume=2&amp;pages=e00569" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.00569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.00569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.00569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23840927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Poland</div> <div class="given-names">J.A.</div></div> <div class="name"><div class="surname">Rife</div> <div class="given-names">T.W.</div></div></div> <div class="year">2012</div>, <div class="article-title">Genotyping-by-sequencing for plant breeding and genetics</div>, <div class="source">Plant Genome US</div> , <div class="volume">5</div>, <div class="fpage">92</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20for%20plant%20breeding%20and%20genetics&amp;author=J.A.%20Poland&amp;author=T.W.%20Rife&amp;publication_year=2012&amp;journal=Plant%20Genome%20US&amp;volume=5&amp;pages=92-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2012.05.0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2012.05.0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2012.05.0005"> </span></div> </div></div></div></div></div><div content-id="DSU045C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Peterson</div> <div class="given-names">B.K.</div></div> <div class="name"><div class="surname">Weber</div> <div class="given-names">J.N.</div></div> <div class="name"><div class="surname">Kay</div> <div class="given-names">E.H.</div></div> <div class="name"><div class="surname">Fisher</div> <div class="given-names">H.S.</div></div> <div class="name"><div class="surname">Hoekstra</div> <div class="given-names">H.E.</div></div></div> <div class="year">2012</div>, <div class="article-title">Double digest RADseq: an inexpensive method for <em>de novo</em> SNP discovery and genotyping in model and non-model species</div>, <div class="source">PloS ONE</div> , <div class="volume">7</div>, <div class="fpage">e37135</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double%20digest%20RADseq%3A%20an%20inexpensive%20method%20for%20de%20novo%20SNP%20discovery%20and%20genotyping%20in%20model%20and%20non-model%20species&amp;author=B.K.%20Peterson&amp;author=J.N.%20Weber&amp;author=E.H.%20Kay&amp;author=H.S.%20Fisher&amp;author=H.E.%20Hoekstra&amp;publication_year=2012&amp;journal=PloS%20ONE&amp;volume=7&amp;pages=e37135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0037135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0037135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0037135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22675423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU045C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU045C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Koch</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">AL-Shehbaz</div> <div class="given-names">I.A.</div></div></div> <div class="year">2004</div>, <div class="article-title">Taxonomic and phylogenetic evaluation of the American ‘Thlaspi’ species: identity and relationship to the Eurasian genus Noccaea (Brassicaceae)</div>, <div class="source">Syst. Bot.</div> , <div class="volume">29</div>, <div class="fpage">375</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20and%20phylogenetic%20evaluation%20of%20the%20American%20%E2%80%98Thlaspi%E2%80%99%20species%3A%20identity%20and%20relationship%20to%20the%20Eurasian%20genus%20Noccaea%20(Brassicaceae)&amp;author=M.%20Koch&amp;author=I.A.%20AL-Shehbaz&amp;publication_year=2004&amp;journal=Syst.%20Bot.&amp;volume=29&amp;pages=375-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1600/036364404774195566" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1600%2F036364404774195566" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1600%2F036364404774195566"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=332636&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_table4.xls?Expires=2147483647&amp;Signature=ozPozldLyBnygknJfJ9XNGLteRagxdNRTCe54NnN5G066o8CnbrZ7UF0GV66-9cR5BEaCofWS9HJahttl70RYlgHOsgfGTIDt6Elt6mn0uBLcZ3V3AGHqM~9hi~nyuiLhYUx4eINsf674u8csjy9npj2qK45-FNf-2~IKKY-ouR1YzpDnvFK-xzbRFZR2DMRXJBbUmH0BjAlY9O1M-mEhwxw8DljFQKSgFOWx1Hsf7GtRA7lGqme5PbMjMZnWp~xeU8wjFppVD-BwcxGmnFJW4HbEitbjZmrB4SZaC40rIYDJdn-d9imYDUzazlS9yUiepMbmYwvuh0SNqpEnWeb8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_table3.xls?Expires=2147483647&amp;Signature=TXvBtYrr~CAnSpG~hXNOTWeo3m3sKFx2fx3f5mYz1nJ6-6A8ZMhfxu3ErSDpfnZ98CPahsMFjWpWv~YtU4bJJHm8f8VdVU10pwEypobBSuAbMxxJbty7WwwhbdC2ooIJET7UoJQjtuv3z7c-p-Okp7Lru567hS8-KOmt4DPXsKTPA~-5H6Ukp2gZrXw6rrMZ3BOOYIRT3TkwfinKP0Fmg~S5D-pSz6~8gHqHlrxPCu2AoC85edcb-j9mdeOpqI9KlABhvP~zAdR1bmEj7hITPMMm~IIcHCHl4a7PDuDBvBXAEGcad55ietSIMIBkoEUYT3ZKV81xf8JRlQrHzMIEjw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_table2.pdf?Expires=2147483647&amp;Signature=kmrumAY3Bk1LOnr1TxXTj1JZeIgjMYzVi9aE0UGsdK1wxgk8cJc-z6UWLaHH4TXKYo-yDcHcxkCICIaBN95NpoCntyirN3D75od8DVo4gaRh6Eo3uRL0mHuty8E5u7t1l9t0wnxdAxJ5IEWYYzjVoN5qSIaDLAxRNDnepf5VsOE~n8H8fa51xnfxNbW3ss4NKtjOQ7m5Q6cL5RSDJzPTvKQwiQ5nABCoNkNZqiHlIYZfNTaOQB6P~KT~EeBsge3iOD6sw5eV5q1UKGHHnrAg0DWrHYrwnp0ybBsoNu1Y47PnbJcS-AZSoI0WdXXWVM5s0aAdhneCJqdxUrdUP5AB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_table1.pdf?Expires=2147483647&amp;Signature=PXhRc9TGERltdpbhC~iTmAqMXyjuaae157VBBFpu398xQ~NqqnzqfoIqKUS-jj6v7MawtKld-It-qDBLGQAdZvnpMC3PZF6H0wdcgOQj1CxTyzHztujs4UdlQkOMlIevUpoByqLKgZjw4NRfuD6iqBOo5F7OthgcKnFplJhQQCkUF7CAIopMKAxBdNx~IOd9wi~ilxBI00m8vVLdbxKLENKs3n5VJlBjfFXe-6KTJRzvcSBlHE~6R-q9JbUJRVKNYivNnrj2ZOsj3CY7WmedTEofLZX8~MpyT3ZZq~cqAEZE3YtM4ndDKmc4-h1ai13CV6coBxfi7Lld3Q8gyf~-LA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_fig1.pdf?Expires=2147483647&amp;Signature=Rw1OmU4jkQnSTcCTJDxiZx1K8DKOsQjw50iVdTtsdpVWmPRwV6JrnQT6sF4nEi5E7zJBpGJ6dFgChT7QmZ~dOVrK4-8AAMSAtsfms9v15O7Zuo6U4LX-badenCjiEGoaebmz68FwllIQTZifE4TjXKoJT2Y-3VtAr5attdBrnuf1uPYh9xQKZ9ti8fzont7Ie52dVsjPcZUh24oLtnRwXkvyW6wCukHhkissJN~nihgc012qVBRj74PYUdU2PzZLxbyVfLFC0iyMBx4REkOj4gyTF6yp29HPneJlSKvYkBJEGcisZkvKF~~REzmH7ndpSFnJ~OS5zlkMSgTYKavFGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset8.txt?Expires=2147483647&amp;Signature=ebxluu7qG5woA~5ttCqq7Um4sGgOQy6dWvcNDs9IhJTUfdJ4~6nPB2HXZN43uedxS7JO~QyabBE~rpPaxuFv0YKzXP-FtHKhJV-YVMF3R8eCgMZq4Ih1hSssU7cPApxluCscv84GN74IyTTabWZHUzfvl6w86V0ZuU7S3QX9fu-I3C3uIrmDMEV31c8AV8d3dQo7fDlaZCNbUEET8OEWTCPm2Oxw2O0FDR5-FYxRloXDARRa3pFYPPdnpy1yrXMdfv90B0jHvB5KCNMt9UmcVzsVu3Lq29giTmChLYdc0P4F8GiPHVR45W7ofsUuUzqzqSaNwTGlRfQNzzmrlMxBhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - txt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset7.xlsx?Expires=2147483647&amp;Signature=iuI~McUKdLSkiIgehg3HpNMxJ28eQt~H-Bju~p8aYEKcJBwzrOppk2lcJhnaXeVLuYdw-qElSI2hw5Hw4M6wtSumdhzghxdwPvrRDahv~ckX2P-sUlCJARho8rRfCF0NY1LL6hgB~iiFaZcDdCTh66sJX2Uudqjxlw0C5rPEYSevYtFA~6uNDuhOLmD1OHGgMQmcvzHJ9MMlClgMt2EURcj0ddS8QaPawpOQM9In882Rvc1fLb4kLejy5VCEGHv8epwILL85pDd3Ac9l~soucSVV9F1TCHfKl7Hstgen-YYXiZUwuDISXSlkXV4OdNKCBml8nH0brPNdEEFMycaUrg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset6.xlsx?Expires=2147483647&amp;Signature=qknfA2UXuXiu8j9u~fnynrEEsd26hf0w5mzMCxeudXikFTbxr-UirZoQv9qDORFwZ1vwttm8PNrl4lVwSQQbpuP11J3jJeTwRHdBfQ-38ogReYeArrdUhyKktWvjgyPHErHCUl4tsN4XQhp0shWIAmsruuT9pFL~6m5HMCXKkFIQHuhzHnyVM2lRyFP4ximiBMKu8zaBRUVCNRq6mzpZT-QdEf1xcepdRMD4wT1JooN6FzQQzhR8po0UMKxiW08SQ3SYDRq9p5jsE8VASV4SF534~dN3vVI1YRDyD0rlorzTjfil30Go5SJRlKZcMiGwm1iZcXVmc~APYn9D7EQF8w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset5.xlsx?Expires=2147483647&amp;Signature=yd~erX4inN1VhYqyUX~DArPcSU13-3SZcWA1ink3V6CEuUJfkn18Uq8LrFEGYijOOBLxyze0--Rw9ujsddUcWHlm5b03oD8~9y6ZGUMFEUPfQ7ggkBxgoP2zpE4~8Wygiz9TGSSlEaUzFdnsQfdBOXWAmJmtu8kTLYOk2j-C00kBGUn5ldBYfD9kLBFRXsmjEypOPWwsGpoh8B6gzHZIImpXxLYjvS6t5fvd-2KentaQuGcVqN4hX0ebxMRgifDG7SrQK-fFCwQFABeL5XTJuA3y0812ySr0zKalqoq3CxwITut6CK3w-BRWE844CBM6xRVz9IjWjJhvKzv8gTpIOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset4.xlsx?Expires=2147483647&amp;Signature=Ma5YFf3RUkf2U3twtGTy167naorVotRnr9dQ~yVsfyOHICPEO4Av1-~SiFbV-KR1iZmRbmcrCXQPk5PKdiEGlP2oBF1UbFeoIGb6FXAQcrNP7iwA3bzN5K8cdkvs7Q0agmFyYBC7vMmTciCimldpuhfb~Yr7v1VZo~sbp61BjJyHtVWb1Uu8pMs36j6inP4vJWzKULU11HR20P6YO5Bw4CNVS7xXmlqs7iJVD34zHbNSO3q9xdR4yKwCoh-F1O941XLX9OM0i5tYu5CH~T-sQJIyC4txLzwgXqS-u~pc8YkcBvZDx7EguzF77xQJ8WTrImR-r3jAR2mwchrZIJWMmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset3.xls?Expires=2147483647&amp;Signature=FdAjtWLhphd3sUJi64M685gyreVQkSfdXZJpxEbXFtTOZEQueEGW5gKDKMZ2ZlYVWZoq9~4UhcSnJub6lTeqzFpjEBRwc0q0ao0-MeJoO0aMwptR3Iam1jMHucKWDbv2N3sUGW21zEHERNXKHIM4VwvYPwcQAFQY5pnlbXNJxhSAKY6lGheQsAJej8uE5UqQOhw-xCaO6an-l0SQBEr2GKPT8CgF7TJqnNdxcqJV3V7R~R74WSUKNyFD4zQeKJT5a~5XdaB1UEGPBC~RM7MUa6JOCcfJXeY1JZ09fN04b93o2g9tg8w~V5U0cbu2m4K0VOkLynrGcBdzYyR6T7jZVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset2.xlsx?Expires=2147483647&amp;Signature=zV~faUiZAz0anO-khNzd7lgXkuuEg9upzB-GdmhPMFdczufWu5fsJRs3C4C6MxYAgBRyaCMxpAytTXzIGQBvGHEgC2SYgSR3efiQec4HZd9SAGPtiWavLeI8CYyzHkSd~~jdMfnn4hZVENT9CxyF~eIXrAr1F0Ixru~mXjCnJ8ZSYHtNpjsVNXsnsGHagEFMAqqXMkeZ2mqPlibk1lCIh~fDeO9GqRf-3LCi8NOPKt8PP016Am3uVwRiCN8yHTKlm8s33QP9zwiEohcUY~UwgopV95jRwF8wejXc~GfjILnxDIaXwvLse920u~ClrO6Z5ywvKXJ2AGeaXJQw2SxiXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp_Dataset1.xlsx?Expires=2147483647&amp;Signature=m~b83qkQgpg4IVU7sZvC-LsgkLfUP0l6yyS43DHCIlOUWmbgJ7EvLEqIxOjJ6U7ppwdlXY1rohPlq5PoGbsW7~GirC4KEHVuVMmTGDufWtSgAKIKKMeB7Z6OtcM5r-KnUbBAF3Rpq56T9ob3hsE84BJHq1TIYl~U5ihMULl2URTdPhB0Bm6GO0LP1lIuRhGHS7blb8gPxPvCmOf9LGA0pfprAt14Hk55lXfq4mXIC3M-E1CTA6T1Mkw3~QU46gDCtBIi7Kre1e9nTEfitkI1KJt-HYdAwyOMkZnMB7djq8kpXcJYSUyFYWcS7UOMdTDd7lt3tjFG-zh6UpyMHxpd7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/2/10.1093/dnares/dsu045/2/dsu045-dsu045supp.doc?Expires=2147483647&amp;Signature=PwE~ok2xdclA3lAF-rst307PhwoYY8qSNFvCI5LdmRnPvPZYixA09vED~u6h8NLQLz9wqgoeH0v2ixoUPZmQv77MGY08HioQuiVT1b-u7ONQY8e8MNiBk0FVKMkC9wC8PSddn5JCdlnnal-FwEuSOkby8Hc7NgeoY8GgG2Wrx8ewLO7VDhUdie-yNeeadub0secukcFNHEny5uvQY6lXdvKdyk3WZdud4z6Fk5Hwy-K0SOhNziiA7bECrPY49KlREadSAQ8FICDoMIHfKbfRmAznRTZRz3mHF6KmLvbkvR5oxJXjQFdZFgze9KoWeWSAo~LjoOlsi8J70p0bbQavCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,652</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu045"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="332636"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000353644600001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f2%2f121%2f332636">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000353644600001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F2%2F121%2F332636" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/332636'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="draft genome of field pennycress (Thlaspi arvense) provides tools for the domestication of a new winter biofuel crop" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu045");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f2%2f121%2f332636">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

