

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/5/473/3769337","@type":"ScholarlyArticle","name":"Identification and allelic dissection uncover roles of lncRNAs in secondary growth of  Populus tomentosa ","datePublished":"2017-04-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2017-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx018","keywords":["association studies","epistatic interaction","long non-coding RNA","target mimics","Populus tomentosa"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/5/473/3769337","author":[{"name":"Zhou, Daling","affiliation":"1 Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   2 National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   3 Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China","@type":"Person"},{"name":"Du, Qingzhang","affiliation":"1 Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   2 National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   3 Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China","@type":"Person"},{"name":"Chen, Jinhui","affiliation":"1 Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   2 National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   3 Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China","@type":"Person"},{"name":"Wang, Qingshi","affiliation":"1 Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   2 National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   3 Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China","@type":"Person"},{"name":"Zhang, Deqiang","affiliation":"1 Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   2 National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China   3 Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China","@type":"Person"}],"description":"Abstract.  Long non-coding RNAs (lncRNAs) function in various biological processes. However, their roles in secondary growth of plants remain poorly understood.","pageStart":"473","pageEnd":"486","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Identification and allelic dissection uncover roles of lncRNAs in secondary growth of  Populus tomentosa ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Identification and allelic dissection uncover roles of lncRNAs in secondary growth of  Populus tomentosa " />
    <meta property="og:description" content="Abstract.  Long non-coding RNAs (lncRNAs) function in various biological processes. However, their roles in secondary growth of plants remain poorly understood." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/5/473/3769337" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-04-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zhou, Daling" /><meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Du, Qingzhang" /><meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Chen, Jinhui" /><meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Wang, Qingshi" /><meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_author" content="Zhang, Deqiang" /><meta name="citation_author_institution" content="Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China" /><meta name="citation_author_institution" content="Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China" /><meta name="citation_title" content="Identification and allelic dissection uncover roles of lncRNAs in secondary growth of Populus tomentosa" /><meta name="citation_firstpage" content="473" /><meta name="citation_lastpage" content="486" /><meta name="citation_doi" content="10.1093/dnares/dsx018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2017/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=On the classification of long non-coding RNAs,;  citation_journal_title=RNA Bio;  citation_year=2013;  citation_volume=10;  citation_pages=924-33; " /><meta name="citation_reference" content="citation_title=The evolution of lncRNA repertoires and expression patterns in tetrapods;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=505;  citation_pages=635-40; " /><meta name="citation_reference" content="citation_title=Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses;  citation_journal_title=Genes Dev;  citation_year=2011;  citation_volume=25;  citation_pages=1915-27; " /><meta name="citation_reference" content="citation_title=Evolution and functions of long noncoding RNAs;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=629-41; " /><meta name="citation_reference" content="citation_title=Cold-induced silencing by long antisense transcripts of an Arabidopsis polycomb target;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=462;  citation_pages=799-802; " /><meta name="citation_reference" content="citation_title=Vernalization-mediated epigenetic silencing by a long intronic noncoding RNA;  citation_journal_title=Science;  citation_year=2011;  citation_volume=331;  citation_pages=76-9; " /><meta name="citation_reference" content="citation_title=Target mimicry provides a new mechanism for regulation of microRNA activity;  citation_journal_title=Nat. Genet;  citation_year=2007;  citation_volume=39;  citation_pages=1033-7; " /><meta name="citation_reference" content="citation_title=Regulation of plant biomass production;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2010;  citation_volume=13;  citation_pages=298-303; " /><meta name="citation_reference" content="citation_title=Towards a systems approach for lignin biosynthesis in Populus trichocarpa: transcript abundance and specificity of the monolignol biosynthetic genes;  citation_journal_title=Plant Cell Physiol;  citation_year=2010;  citation_volume=51;  citation_pages=144-63; " /><meta name="citation_reference" content="citation_publisher=Springer, Berlin;  citation_title=Single Nucleotide Polymorphisms;  citation_year=2009;  citation_pages=3-22; " /><meta name="citation_reference" content="citation_title=Long noncoding RNAs-related diseases, cancers, and drugs;  citation_journal_title=Scientific World J;  citation_year=2013;  citation_volume=2013;  citation_pages=943539" /><meta name="citation_reference" content="citation_title=A long noncoding RNA regulates photoperiod-sensitive male sterility, an essential component of hybrid rice;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2012;  citation_volume=109;  citation_pages=2654-9; " /><meta name="citation_reference" content="citation_title=Potential etiologic and functional implications of genome-wide association loci for human diseases and traits;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2009;  citation_volume=106;  citation_pages=9362-7; " /><meta name="citation_reference" content="citation_title=Human disease-associated genetic variation impacts large intergenic non-coding RNA expression;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003201" /><meta name="citation_reference" content="citation_title=Changes in enzymic activities of nucleoside diphosphate sugar interconversions during differentiation of cambium to xylem in sycamore and poplar;  citation_journal_title=Bio. J;  citation_year=1977;  citation_volume=162;  citation_pages=267-79; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray);  citation_journal_title=Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-1604; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Prot;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=Genome-wide discovery and characterization of maize long non-coding RNAs;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=R40" /><meta name="citation_reference" content="citation_title=Prediction of novel long non-coding RNAs based on RNA-Seq data of mouse Klf1 knockout study;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=1" /><meta name="citation_reference" content="citation_title=HMMER web server: interactive sequence similarity searching;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W29-37; " /><meta name="citation_reference" content="citation_title=CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W345-9; " /><meta name="citation_reference" content="citation_title=Utilizing sequence intrinsic composition to classify protein-coding and long non-coding transcripts;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=e166" /><meta name="citation_reference" content="citation_title=Identification of novel transcripts in annotated genomes using RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2325-9; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1202-10; " /><meta name="citation_reference" content="citation_title=The TIGR rice genome annotation resource: improvements and new features;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D883-7; " /><meta name="citation_reference" content="citation_title=DREME: motif discovery in transcription factor ChIP-seq data;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1653-9; " /><meta name="citation_reference" content="citation_title=Assigning roles to DNA regulatory motifs using comparative genomics;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=860-6; " /><meta name="citation_reference" content="citation_title=Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=650-9; " /><meta name="citation_reference" content="citation_title=Genome-wide computational identification and manual annotation of human long noncoding RNA genes;  citation_journal_title=RNA;  citation_year=2010;  citation_volume=16;  citation_pages=1478-87; " /><meta name="citation_reference" content="citation_title=LncRNA proﬁle of glioblastoma reveals the potential role of lncRNAs in contributing to glioblastoma pathogenesis;  citation_journal_title=Int. J. Oncol;  citation_year=2012;  citation_volume=40;  citation_pages=2004" /><meta name="citation_reference" content="citation_title=RNAplex: a fast tool for RNA–RNA interaction search;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2657-63; " /><meta name="citation_reference" content="citation_title=Widespread long noncoding RNAs as endogenous target mimics for microRNAs in plants;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=161;  citation_pages=1875-84; " /><meta name="citation_reference" content="citation_title=A travel guide to Cytoscape plugins;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=1069-76; " /><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2− ΔΔCT method;  citation_journal_title=Methods;  citation_year=2001;  citation_volume=25;  citation_pages=402-8; " /><meta name="citation_reference" content="citation_title=Allelic variation in a cellulose synthase gene (PtoCesA4) associated with growth and wood properties in Populus tomentosa;  citation_journal_title=G3;  citation_year=2013;  citation_volume=3;  citation_pages=2069-84; " /><meta name="citation_reference" content="citation_title=Variation in growth, leaf, and wood property traits of Chinese white poplar (Populus tomentosa), a major industrial tree species in Northern China;  citation_journal_title=Can. J. For. Res;  citation_year=2014;  citation_volume=44;  citation_pages=326-339; " /><meta name="citation_reference" content="citation_title=Genetic architecture of growth traits in Populus revealed by integrated quantitative trait locus (QTL) analysis and association studies;  citation_journal_title=New Phytol;  citation_year=2016;  citation_volume=209;  citation_pages=1067-1082; " /><meta name="citation_reference" content="citation_title=SOAP: short oligonucleotide alignment program;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=713-4; " /><meta name="citation_reference" content="citation_title=The variant call format and VCFtools;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2156-8; " /><meta name="citation_reference" content="citation_publisher=Columbia University Press, ;  citation_title=Molecular Evolutionary Genetics;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination;  citation_journal_title=Theor. Popul. Biol;  citation_year=1975;  citation_volume=7;  citation_pages=256-276; " /><meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism;  citation_journal_title=Genetics;  citation_year=1989;  citation_volume=123;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=A unified mixed-model method for association mapping that accounts for multiple levels of relatedness;  citation_journal_title=Nat. Genet;  citation_year=2006;  citation_volume=38;  citation_pages=203-8; " /><meta name="citation_reference" content="citation_title=Genetic diversity and population structure of Chinese white poplar (Populus tomentosa) revealed by SSR markers;  citation_journal_title=J. Hered;  citation_year=2012;  citation_volume=103;  citation_pages=853-62; " /><meta name="citation_reference" content="citation_title=SPAGeDi: a versatile computer program to analyse spatial genetic structure at the individual or population levels;  citation_journal_title=Mol. Ecol. Notes;  citation_year=2002;  citation_volume=2;  citation_pages=618-20; " /><meta name="citation_reference" content="citation_title=Inferring weak population structure with the assistance of sample group information;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2009;  citation_volume=9;  citation_pages=1322-32; " /><meta name="citation_reference" content="citation_title=Statistical significance for genomewide studies;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=9440-5; " /><meta name="citation_reference" content="citation_title=Parallel and serial computing tools for testing single-locus and epistatic SNP effects of quantitative traits in genome-wide association studies;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=315" /><meta name="citation_reference" content="citation_title=Detection of SNP epistasis effects of quantitative traits using an extended Kempthorne model;  citation_journal_title=Physiol. Genomics;  citation_year=2007;  citation_volume=28;  citation_pages=46-52; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis uncovers regulation of long intergenic noncoding RNAs in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=4333-4345; " /><meta name="citation_reference" content="citation_title=Genome-wide screening and functional analysis identify a large number of long noncoding RNAs involved in the sexual reproduction of rice;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=512" /><meta name="citation_reference" content="citation_title=ChIPBase: a database for decoding the transcriptional regulation of long non-coding RNA and microRNA genes from ChIP-Seq data;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D177-87; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis reveals distinct patterns of epigenetic features in long non-coding RNA loci;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=10018-31; " /><meta name="citation_reference" content="citation_title=Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=458;  citation_pages=223-7; " /><meta name="citation_reference" content="citation_title=The H19 lincRNA is a developmental reservoir of miR-675 that suppresses growth and Igf1r;  citation_journal_title=Nature Cell Biol;  citation_year=2012;  citation_volume=14;  citation_pages=659-65; " /><meta name="citation_reference" content="citation_title=A transcriptional roadmap to wood formation;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2001;  citation_volume=98;  citation_pages=14732-37; " /><meta name="citation_reference" content="citation_title=Functions of microRNAs and related small RNAs in plants;  citation_journal_title=Nat. Genet;  citation_year=2006;  citation_volume=38;  citation_pages=S31-6; " /><meta name="citation_reference" content=" citation_title=Seminars in Cell and Developmental Biology;  citation_year=2014;  citation_volume=34;  citation_pages=9-14; " /><meta name="citation_reference" content="citation_title=Novel and mechanical stress–responsive microRNAs in Populus trichocarpa that are absent from Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=2186-203; " /><meta name="citation_reference" content="citation_title=Insights into the mechanism of plant development: interactions of miRNAs pathway with phytohormone response;  citation_journal_title=Biochem. Biophys. Res. Co;  citation_year=2009;  citation_volume=384;  citation_pages=1-5; " /><meta name="citation_reference" content="citation_title=Characterization of microRNAs expressed during secondary wall biosynthesis in Acacia mangium;  citation_journal_title=PloS One;  citation_year=2012;  citation_volume=7;  citation_pages=e49662" /><meta name="citation_reference" content="citation_title=Ptr-miR397a is a negative regulator of laccase genes affecting lignin content in Populus trichocarpa;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=10848-53; " /><meta name="citation_reference" content="citation_title=Coding-independent regulation of the tumor suppressor PTEN by competing endogenous mRNAs;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=344-57; " /><meta name="citation_reference" content="citation_title=Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in Populus trichocarpa;  citation_journal_title=J. Exp. Bot;  citation_year=2014;  citation_volume=65;  citation_pages=4975-83; " /><meta name="citation_reference" content="citation_title=Expression profile of small RNAs in Acacia mangium secondary xylem tissue with contrasting lignin content-potential regulatory sequences in monolignol biosynthetic pathway;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=S13" /><meta name="citation_reference" content="citation_title=Polymorphisms in cinnamoyl CoA reductase (CCR) are associated with variation in microfibril angle in Eucalyptus spp;  citation_journal_title=Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=1257-65; " /><meta name="citation_reference" content="citation_title=Pleiotropy in complex traits: challenges and strategies;  citation_journal_title=Nat. Rev. Genet;  citation_year=2013;  citation_volume=14;  citation_pages=483-95; " /><meta name="citation_reference" content="citation_title=Epistasis and quantitative traits: using model organisms to study gene-gene interactions;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=22-33; " /><meta name="citation_reference" content="citation_title=A global view of epistasis;  citation_journal_title=Nat. Genet;  citation_year=2005;  citation_volume=37;  citation_pages=13-4; " /><meta name="citation_reference" content="citation_title=Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems;  citation_journal_title=Nat. Rev. Genet;  citation_year=2008;  citation_volume=9;  citation_pages=855-67; " /><meta name="citation_reference" content="citation_title=Dynamic and widespread lncRNA expression in a sponge and the origin of animal complexity;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=2367-82; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/5/473/22892952/dsx018.pdf" /><meta name="description" content="Abstract.  Long non-coding RNAs (lncRNAs) function in various biological processes. However, their roles in secondary growth of plants remain poorly understood." />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/5/473/3769337" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/5/473/3769337" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/5/473/3769337");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx018';
    </script>
<title>
	Identification and allelic dissection uncover roles of lncRNAs in secondary growth of Populus tomentosa | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3769337" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be+wATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhvdSwgRGFsaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBBZHZhbmNlZCBJbm5vdmF0aW9uIENlbnRlciBmb3IgVHJlZSBCcmVlZGluZyBieSBNb2xlY3VsYXIgRGVzaWduLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkR1LCBRaW5nemhhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEFkdmFuY2VkIElubm92YXRpb24gQ2VudGVyIGZvciBUcmVlIEJyZWVkaW5nIGJ5IE1vbGVjdWxhciBEZXNpZ24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hlbiwgSmluaHVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmVpamluZyBBZHZhbmNlZCBJbm5vdmF0aW9uIENlbnRlciBmb3IgVHJlZSBCcmVlZGluZyBieSBNb2xlY3VsYXIgRGVzaWduLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBFbmdpbmVlcmluZyBMYWJvcmF0b3J5IGZvciBUcmVlIEJyZWVkaW5nLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuUi4gQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBHZW5ldGljcyBhbmQgQnJlZWRpbmcgaW4gRm9yZXN0IFRyZWVzIGFuZCBPcm5hbWVudGFsIFBsYW50cywgQ29sbGVnZSBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBUZWNobm9sb2d5LCBCZWlqaW5nIEZvcmVzdHJ5IFVuaXZlcnNpdHksIEJlaWppbmcgMTAwMDgzLCBQLiBSLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIFFpbmdzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEFkdmFuY2VkIElubm92YXRpb24gQ2VudGVyIGZvciBUcmVlIEJyZWVkaW5nIGJ5IE1vbGVjdWxhciBEZXNpZ24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIERlcWlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCZWlqaW5nIEFkdmFuY2VkIElubm92YXRpb24gQ2VudGVyIGZvciBUcmVlIEJyZWVkaW5nIGJ5IE1vbGVjdWxhciBEZXNpZ24sIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEVuZ2luZWVyaW5nIExhYm9yYXRvcnkgZm9yIFRyZWUgQnJlZWRpbmcsIENvbGxlZ2Ugb2YgQmlvbG9naWNhbCBTY2llbmNlcyBhbmQgVGVjaG5vbG9neSwgQmVpamluZyBGb3Jlc3RyeSBVbml2ZXJzaXR5LCBCZWlqaW5nIDEwMDA4MywgUC5SLiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEdlbmV0aWNzIGFuZCBCcmVlZGluZyBpbiBGb3Jlc3QgVHJlZXMgYW5kIE9ybmFtZW50YWwgUGxhbnRzLCBDb2xsZWdlIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIFRlY2hub2xvZ3ksIEJlaWppbmcgRm9yZXN0cnkgVW5pdmVyc2l0eSwgQmVpamluZyAxMDAwODMsIFAuIFIuIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJZGVudGlmaWNhdGlvbiBhbmQgYWxsZWxpYyBkaXNzZWN0aW9uIHVuY292ZXIgcm9sZXMgb2YgbG5jUk5BcyBpbiBzZWNvbmRhcnkgZ3Jvd3RoIG9mIFBvcHVsdXMgdG9tZW50b3NhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0ODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHRoZSBjbGFzc2lmaWNhdGlvbiBvZiBsb25nIG5vbi1jb2RpbmcgUk5Bcyw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQSBCaW87ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz05MjQtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb24gb2YgbG5jUk5BIHJlcGVydG9pcmVzIGFuZCBleHByZXNzaW9uIHBhdHRlcm5zIGluIHRldHJhcG9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUwNTsgIGNpdGF0aW9uX3BhZ2VzPTYzNS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5ub3RhdGlvbiBvZiBodW1hbiBsYXJnZSBpbnRlcmdlbmljIG5vbmNvZGluZyBSTkFzIHJldmVhbHMgZ2xvYmFsIHByb3BlcnRpZXMgYW5kIHNwZWNpZmljIHN1YmNsYXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE5MTUtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBhbmQgZnVuY3Rpb25zIG9mIGxvbmcgbm9uY29kaW5nIFJOQXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9NjI5LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2xkLWluZHVjZWQgc2lsZW5jaW5nIGJ5IGxvbmcgYW50aXNlbnNlIHRyYW5zY3JpcHRzIG9mIGFuIEFyYWJpZG9wc2lzIHBvbHljb21iIHRhcmdldDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ2MjsgIGNpdGF0aW9uX3BhZ2VzPTc5OS04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcm5hbGl6YXRpb24tbWVkaWF0ZWQgZXBpZ2VuZXRpYyBzaWxlbmNpbmcgYnkgYSBsb25nIGludHJvbmljIG5vbmNvZGluZyBSTkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzMxOyAgY2l0YXRpb25fcGFnZXM9NzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0IG1pbWljcnkgcHJvdmlkZXMgYSBuZXcgbWVjaGFuaXNtIGZvciByZWd1bGF0aW9uIG9mIG1pY3JvUk5BIGFjdGl2aXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTAzMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIHBsYW50IGJpb21hc3MgcHJvZHVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI5OC0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZHMgYSBzeXN0ZW1zIGFwcHJvYWNoIGZvciBsaWduaW4gYmlvc3ludGhlc2lzIGluIFBvcHVsdXMgdHJpY2hvY2FycGE6IHRyYW5zY3JpcHQgYWJ1bmRhbmNlIGFuZCBzcGVjaWZpY2l0eSBvZiB0aGUgbW9ub2xpZ25vbCBiaW9zeW50aGV0aWMgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTE0NC02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW47ICBjaXRhdGlvbl90aXRsZT1TaW5nbGUgTnVjbGVvdGlkZSBQb2x5bW9ycGhpc21zOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fcGFnZXM9My0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZyBub25jb2RpbmcgUk5Bcy1yZWxhdGVkIGRpc2Vhc2VzLCBjYW5jZXJzLCBhbmQgZHJ1Z3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVudGlmaWMgV29ybGQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDEzOyAgY2l0YXRpb25fcGFnZXM9OTQzNTM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsb25nIG5vbmNvZGluZyBSTkEgcmVndWxhdGVzIHBob3RvcGVyaW9kLXNlbnNpdGl2ZSBtYWxlIHN0ZXJpbGl0eSwgYW4gZXNzZW50aWFsIGNvbXBvbmVudCBvZiBoeWJyaWQgcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTI2NTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG90ZW50aWFsIGV0aW9sb2dpYyBhbmQgZnVuY3Rpb25hbCBpbXBsaWNhdGlvbnMgb2YgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gbG9jaSBmb3IgaHVtYW4gZGlzZWFzZXMgYW5kIHRyYWl0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTkzNjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gZGlzZWFzZS1hc3NvY2lhdGVkIGdlbmV0aWMgdmFyaWF0aW9uIGltcGFjdHMgbGFyZ2UgaW50ZXJnZW5pYyBub24tY29kaW5nIFJOQSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzIwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYW5nZXMgaW4gZW56eW1pYyBhY3Rpdml0aWVzIG9mIG51Y2xlb3NpZGUgZGlwaG9zcGhhdGUgc3VnYXIgaW50ZXJjb252ZXJzaW9ucyBkdXJpbmcgZGlmZmVyZW50aWF0aW9uIG9mIGNhbWJpdW0gdG8geHlsZW0gaW4gc3ljYW1vcmUgYW5kIHBvcGxhcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvLiBKOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTI2Ny03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBibGFjayBjb3R0b253b29kLCBQb3B1bHVzIHRyaWNob2NhcnBhIChUb3JyLiAmYW1wOyBHcmF5KTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTE2MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBnZW5lIGFuZCB0cmFuc2NyaXB0IGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgUk5BLXNlcSBleHBlcmltZW50cyB3aXRoIFRvcEhhdCBhbmQgQ3VmZmxpbmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTU2Mi03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGlzY292ZXJ5IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIG1haXplIGxvbmcgbm9uLWNvZGluZyBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPVI0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2Ygbm92ZWwgbG9uZyBub24tY29kaW5nIFJOQXMgYmFzZWQgb24gUk5BLVNlcSBkYXRhIG9mIG1vdXNlIEtsZjEga25vY2tvdXQgc3R1ZHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ITU1FUiB3ZWIgc2VydmVyOiBpbnRlcmFjdGl2ZSBzZXF1ZW5jZSBzaW1pbGFyaXR5IHNlYXJjaGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMjktMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNQQzogYXNzZXNzIHRoZSBwcm90ZWluLWNvZGluZyBwb3RlbnRpYWwgb2YgdHJhbnNjcmlwdHMgdXNpbmcgc2VxdWVuY2UgZmVhdHVyZXMgYW5kIHN1cHBvcnQgdmVjdG9yIG1hY2hpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9VzM0NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VdGlsaXppbmcgc2VxdWVuY2UgaW50cmluc2ljIGNvbXBvc2l0aW9uIHRvIGNsYXNzaWZ5IHByb3RlaW4tY29kaW5nIGFuZCBsb25nIG5vbi1jb2RpbmcgdHJhbnNjcmlwdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9ZTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG5vdmVsIHRyYW5zY3JpcHRzIGluIGFubm90YXRlZCBnZW5vbWVzIHVzaW5nIFJOQS1TZXE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjMyNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgSW5mb3JtYXRpb24gUmVzb3VyY2UgKFRBSVIpOiBpbXByb3ZlZCBnZW5lIGFubm90YXRpb24gYW5kIG5ldyB0b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMTIwMi0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFRJR1IgcmljZSBnZW5vbWUgYW5ub3RhdGlvbiByZXNvdXJjZTogaW1wcm92ZW1lbnRzIGFuZCBuZXcgZmVhdHVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDg4My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EUkVNRTogbW90aWYgZGlzY292ZXJ5IGluIHRyYW5zY3JpcHRpb24gZmFjdG9yIENoSVAtc2VxIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTY1My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NpZ25pbmcgcm9sZXMgdG8gRE5BIHJlZ3VsYXRvcnkgbW90aWZzIHVzaW5nIGNvbXBhcmF0aXZlIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTg2MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtaWRyYW5nZSB0cmFuc2NyaXB0aW9uIHByb2ZpbGVzIHJldmVhbCBleHByZXNzaW9uIGxldmVsIHJlbGF0aW9uc2hpcHMgaW4gaHVtYW4gdGlzc3VlIHNwZWNpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjUwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGNvbXB1dGF0aW9uYWwgaWRlbnRpZmljYXRpb24gYW5kIG1hbnVhbCBhbm5vdGF0aW9uIG9mIGh1bWFuIGxvbmcgbm9uY29kaW5nIFJOQSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTQ3OC04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG5jUk5BIHByb++sgWxlIG9mIGdsaW9ibGFzdG9tYSByZXZlYWxzIHRoZSBwb3RlbnRpYWwgcm9sZSBvZiBsbmNSTkFzIGluIGNvbnRyaWJ1dGluZyB0byBnbGlvYmxhc3RvbWEgcGF0aG9nZW5lc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MjAwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQXBsZXg6IGEgZmFzdCB0b29sIGZvciBSTkHigJNSTkEgaW50ZXJhY3Rpb24gc2VhcmNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI2NTctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdpZGVzcHJlYWQgbG9uZyBub25jb2RpbmcgUk5BcyBhcyBlbmRvZ2Vub3VzIHRhcmdldCBtaW1pY3MgZm9yIG1pY3JvUk5BcyBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTg3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0cmF2ZWwgZ3VpZGUgdG8gQ3l0b3NjYXBlIHBsdWdpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTA2OS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgcmVsYXRpdmUgZ2VuZSBleHByZXNzaW9uIGRhdGEgdXNpbmcgcmVhbC10aW1lIHF1YW50aXRhdGl2ZSBQQ1IgYW5kIHRoZSAy4oiSIM6UzpRDVCBtZXRob2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00MDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWxsZWxpYyB2YXJpYXRpb24gaW4gYSBjZWxsdWxvc2Ugc3ludGhhc2UgZ2VuZSAoUHRvQ2VzQTQpIGFzc29jaWF0ZWQgd2l0aCBncm93dGggYW5kIHdvb2QgcHJvcGVydGllcyBpbiBQb3B1bHVzIHRvbWVudG9zYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTIwNjktODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlhdGlvbiBpbiBncm93dGgsIGxlYWYsIGFuZCB3b29kIHByb3BlcnR5IHRyYWl0cyBvZiBDaGluZXNlIHdoaXRlIHBvcGxhciAoUG9wdWx1cyB0b21lbnRvc2EpLCBhIG1ham9yIGluZHVzdHJpYWwgdHJlZSBzcGVjaWVzIGluIE5vcnRoZXJuIENoaW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYW4uIEouIEZvci4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MzI2LTMzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhcmNoaXRlY3R1cmUgb2YgZ3Jvd3RoIHRyYWl0cyBpbiBQb3B1bHVzIHJldmVhbGVkIGJ5IGludGVncmF0ZWQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIChRVEwpIGFuYWx5c2lzIGFuZCBhc3NvY2lhdGlvbiBzdHVkaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIwOTsgIGNpdGF0aW9uX3BhZ2VzPTEwNjctMTA4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUDogc2hvcnQgb2xpZ29udWNsZW90aWRlIGFsaWdubWVudCBwcm9ncmFtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTcxMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdmFyaWFudCBjYWxsIGZvcm1hdCBhbmQgVkNGdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjE1Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sdW1iaWEgVW5pdmVyc2l0eSBQcmVzcywgOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T24gdGhlIG51bWJlciBvZiBzZWdyZWdhdGluZyBzaXRlcyBpbiBnZW5ldGljYWwgbW9kZWxzIHdpdGhvdXQgcmVjb21iaW5hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIFBvcHVsLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNTYtMjc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBtZXRob2QgZm9yIHRlc3RpbmcgdGhlIG5ldXRyYWwgbXV0YXRpb24gaHlwb3RoZXNpcyBieSBETkEgcG9seW1vcnBoaXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz01ODUtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdW5pZmllZCBtaXhlZC1tb2RlbCBtZXRob2QgZm9yIGFzc29jaWF0aW9uIG1hcHBpbmcgdGhhdCBhY2NvdW50cyBmb3IgbXVsdGlwbGUgbGV2ZWxzIG9mIHJlbGF0ZWRuZXNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MjAzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IGFuZCBwb3B1bGF0aW9uIHN0cnVjdHVyZSBvZiBDaGluZXNlIHdoaXRlIHBvcGxhciAoUG9wdWx1cyB0b21lbnRvc2EpIHJldmVhbGVkIGJ5IFNTUiBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIZXJlZDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz04NTMtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNQQUdlRGk6IGEgdmVyc2F0aWxlIGNvbXB1dGVyIHByb2dyYW0gdG8gYW5hbHlzZSBzcGF0aWFsIGdlbmV0aWMgc3RydWN0dXJlIGF0IHRoZSBpbmRpdmlkdWFsIG9yIHBvcHVsYXRpb24gbGV2ZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIE5vdGVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz02MTgtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVycmluZyB3ZWFrIHBvcHVsYXRpb24gc3RydWN0dXJlIHdpdGggdGhlIGFzc2lzdGFuY2Ugb2Ygc2FtcGxlIGdyb3VwIGluZm9ybWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTMyMi0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgc2lnbmlmaWNhbmNlIGZvciBnZW5vbWV3aWRlIHN0dWRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9OTQ0MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhbGxlbCBhbmQgc2VyaWFsIGNvbXB1dGluZyB0b29scyBmb3IgdGVzdGluZyBzaW5nbGUtbG9jdXMgYW5kIGVwaXN0YXRpYyBTTlAgZWZmZWN0cyBvZiBxdWFudGl0YXRpdmUgdHJhaXRzIGluIGdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIFNOUCBlcGlzdGFzaXMgZWZmZWN0cyBvZiBxdWFudGl0YXRpdmUgdHJhaXRzIHVzaW5nIGFuIGV4dGVuZGVkIEtlbXB0aG9ybmUgbW9kZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXNpb2wuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NDYtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIHVuY292ZXJzIHJlZ3VsYXRpb24gb2YgbG9uZyBpbnRlcmdlbmljIG5vbmNvZGluZyBSTkFzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NDMzMy00MzQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzY3JlZW5pbmcgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgaWRlbnRpZnkgYSBsYXJnZSBudW1iZXIgb2YgbG9uZyBub25jb2RpbmcgUk5BcyBpbnZvbHZlZCBpbiB0aGUgc2V4dWFsIHJlcHJvZHVjdGlvbiBvZiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTUxMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoSVBCYXNlOiBhIGRhdGFiYXNlIGZvciBkZWNvZGluZyB0aGUgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb24gb2YgbG9uZyBub24tY29kaW5nIFJOQSBhbmQgbWljcm9STkEgZ2VuZXMgZnJvbSBDaElQLVNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQxNzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIHJldmVhbHMgZGlzdGluY3QgcGF0dGVybnMgb2YgZXBpZ2VuZXRpYyBmZWF0dXJlcyBpbiBsb25nIG5vbi1jb2RpbmcgUk5BIGxvY2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTAwMTgtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tYXRpbiBzaWduYXR1cmUgcmV2ZWFscyBvdmVyIGEgdGhvdXNhbmQgaGlnaGx5IGNvbnNlcnZlZCBsYXJnZSBub24tY29kaW5nIFJOQXMgaW4gbWFtbWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ1ODsgIGNpdGF0aW9uX3BhZ2VzPTIyMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgSDE5IGxpbmNSTkEgaXMgYSBkZXZlbG9wbWVudGFsIHJlc2Vydm9pciBvZiBtaVItNjc1IHRoYXQgc3VwcHJlc3NlcyBncm93dGggYW5kIElnZjFyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmUgQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NjU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRyYW5zY3JpcHRpb25hbCByb2FkbWFwIHRvIHdvb2QgZm9ybWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xNDczMi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25zIG9mIG1pY3JvUk5BcyBhbmQgcmVsYXRlZCBzbWFsbCBSTkFzIGluIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPVMzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgY2l0YXRpb25fdGl0bGU9U2VtaW5hcnMgaW4gQ2VsbCBhbmQgRGV2ZWxvcG1lbnRhbCBCaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9OS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgYW5kIG1lY2hhbmljYWwgc3RyZXNz4oCTcmVzcG9uc2l2ZSBtaWNyb1JOQXMgaW4gUG9wdWx1cyB0cmljaG9jYXJwYSB0aGF0IGFyZSBhYnNlbnQgZnJvbSBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTIxODYtMjAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBpbnRvIHRoZSBtZWNoYW5pc20gb2YgcGxhbnQgZGV2ZWxvcG1lbnQ6IGludGVyYWN0aW9ucyBvZiBtaVJOQXMgcGF0aHdheSB3aXRoIHBoeXRvaG9ybW9uZSByZXNwb25zZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDbzsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zODQ7ICBjaXRhdGlvbl9wYWdlcz0xLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgbWljcm9STkFzIGV4cHJlc3NlZCBkdXJpbmcgc2Vjb25kYXJ5IHdhbGwgYmlvc3ludGhlc2lzIGluIEFjYWNpYSBtYW5naXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ5NjYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHRyLW1pUjM5N2EgaXMgYSBuZWdhdGl2ZSByZWd1bGF0b3Igb2YgbGFjY2FzZSBnZW5lcyBhZmZlY3RpbmcgbGlnbmluIGNvbnRlbnQgaW4gUG9wdWx1cyB0cmljaG9jYXJwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTA4NDgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZGluZy1pbmRlcGVuZGVudCByZWd1bGF0aW9uIG9mIHRoZSB0dW1vciBzdXBwcmVzc29yIFBURU4gYnkgY29tcGV0aW5nIGVuZG9nZW5vdXMgbVJOQXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9MzQ0LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBpZGVudGlmaWNhdGlvbiBhbmQgZnVuY3Rpb25hbCBwcmVkaWN0aW9uIG9mIG5vdmVsIGFuZCBkcm91Z2h0LXJlc3BvbnNpdmUgbGluY1JOQXMgaW4gUG9wdWx1cyB0cmljaG9jYXJwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz00OTc1LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIHByb2ZpbGUgb2Ygc21hbGwgUk5BcyBpbiBBY2FjaWEgbWFuZ2l1bSBzZWNvbmRhcnkgeHlsZW0gdGlzc3VlIHdpdGggY29udHJhc3RpbmcgbGlnbmluIGNvbnRlbnQtcG90ZW50aWFsIHJlZ3VsYXRvcnkgc2VxdWVuY2VzIGluIG1vbm9saWdub2wgYmlvc3ludGhldGljIHBhdGh3YXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPVMxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbHltb3JwaGlzbXMgaW4gY2lubmFtb3lsIENvQSByZWR1Y3Rhc2UgKENDUikgYXJlIGFzc29jaWF0ZWQgd2l0aCB2YXJpYXRpb24gaW4gbWljcm9maWJyaWwgYW5nbGUgaW4gRXVjYWx5cHR1cyBzcHA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTctNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsZWlvdHJvcHkgaW4gY29tcGxleCB0cmFpdHM6IGNoYWxsZW5nZXMgYW5kIHN0cmF0ZWdpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ4My05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpc3Rhc2lzIGFuZCBxdWFudGl0YXRpdmUgdHJhaXRzOiB1c2luZyBtb2RlbCBvcmdhbmlzbXMgdG8gc3R1ZHkgZ2VuZS1nZW5lIGludGVyYWN0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MjItMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2xvYmFsIHZpZXcgb2YgZXBpc3Rhc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTMtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpc3Rhc2lz4oCUdGhlIGVzc2VudGlhbCByb2xlIG9mIGdlbmUgaW50ZXJhY3Rpb25zIGluIHRoZSBzdHJ1Y3R1cmUgYW5kIGV2b2x1dGlvbiBvZiBnZW5ldGljIHN5c3RlbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODU1LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIGFuZCB3aWRlc3ByZWFkIGxuY1JOQSBleHByZXNzaW9uIGluIGEgc3BvbmdlIGFuZCB0aGUgb3JpZ2luIG9mIGFuaW1hbCBjb21wbGV4aXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0yMzY3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNS80NzMvMjI4OTI5NTIvZHN4MDE4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBMb25nIG5vbi1jb2RpbmcgUk5BcyAobG5jUk5BcykgZnVuY3Rpb24gaW4gdmFyaW91cyBiaW9sb2dpY2FsIHByb2Nlc3Nlcy4gSG93ZXZlciwgdGhlaXIgcm9sZXMgaW4gc2Vjb25kYXJ5IGdyb3d0aCBvZiBwbGFudHMgcmVtYWluIHBvb3JseSB1bmRlcnN0b29kLiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzc2OTMzNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRI2blFAvFiE3cr2Pf+v32pT+C8iw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f473%2f3769337" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f473%2f3769337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=99990" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&amp;Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/5">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106355642">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355642">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355644">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355644">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355650">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355650">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355674">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355674">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355700">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355700">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355713">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355713">Accession number</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355715">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355715">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355717">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355717">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355719">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355719">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355721">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355721">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/5/459/3778237">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/5/487/3778238">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Identification and allelic dissection uncover roles of lncRNAs in secondary growth of <em>Populus tomentosa</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daling Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">2</div>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">3</div>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daling+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daling Zhou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingzhang Du</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">2</div>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">3</div>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qingzhang+Du">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Du Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Du Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qingzhang Du</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jinhui Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">2</div>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">3</div>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jinhui+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jinhui Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qingshi Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">2</div>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">3</div>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qingshi+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qingshi Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Deqiang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Beijing Advanced Innovation Center for Tree Breeding by Molecular Design, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">2</div>National Engineering Laboratory for Tree Breeding, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P.R. China</div><div class="aff"><div class="label -label">3</div>Key Laboratory of Genetics and Breeding in Forest Trees and Ornamental Plants, College of Biological Sciences and Technology, Beijing Forestry University, Beijing 100083, P. R. China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx018-cor1" data-open="dsx018-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10 62336007. Fax. +86 10 62336164. Email: <a href="mailto:deqiangzhang@bjfu.edu.cn" target="_blank">deqiangzhang@bjfu.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Deqiang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Deqiang Zhang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx018-FM1"><div class="fn"><div class="label -label">*</div><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 473–486, <a href='https://doi.org/10.1093/dnares/dsx018'>https://doi.org/10.1093/dnares/dsx018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 April 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 November 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 April 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3769337" href="/dnaresearch/article-pdf/24/5/473/22892952/dsx018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Daling Zhou, Qingzhang Du, Jinhui Chen, Qingshi Wang, Deqiang Zhang;  Identification and allelic dissection uncover roles of lncRNAs in secondary growth of <em>Populus tomentosa</em>, <em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 473–486, <a href='https://doi.org/10.1093/dnares/dsx018'>https://doi.org/10.1093/dnares/dsx018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3769337?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Identification%20and%20allelic%20dissection%20uncover%20roles%20of%20lncRNAs%20in%20secondary%20growth%20of%20%26lt%3Bem%26gt%3BPopulus%20tomentosa%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-04-26&amp;volumeNum=24&amp;issueNum=5&amp;author=Zhou%2C%20Daling%3B%20Du%2C%20Qingzhang&amp;startPage=473&amp;endPage=486&amp;contentId=10.1093%2Fdnares%2Fdsx018&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f473%2f3769337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106355642 id="106355642" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Long non-coding RNAs (lncRNAs) function in various biological processes. However, their roles in secondary growth of plants remain poorly understood. Here, 15,691 lncRNAs were identified from vascular cambium, developing xylem, and mature xylem of <em>Populus tomentosa</em> with high and low biomass using RNA-seq, including 1,994 lncRNAs that were differentially expressed (DE) among the six libraries. 3,569 <em>cis</em>-regulated and 3,297 <em>trans</em>-regulated protein-coding genes were predicted as potential target genes (PTGs) of the DE lncRNAs to participate in biological regulation. Then, 476 and 28 lncRNAs were identified as putative targets and endogenous target mimics (eTMs) of <em>Populus</em> known microRNAs (miRNAs), respectively. Genome re-sequencing of 435 individuals from a natural population of <em>P. tomentosa</em> found 34,015 single nucleotide polymorphisms (SNPs) within 178 lncRNA loci and 522 PTGs. Single-SNP associations analysis detected 2,993 associations with 10 growth and wood-property traits under additive and dominance model. Epistasis analysis identified 17,656 epistatic SNP pairs, providing evidence for potential regulatory interactions between lncRNAs and their PTGs. Furthermore, a reconstructed epistatic network, representing interactions of 8 lncRNAs and 15 PTGs, might enrich regulation roles of genes in the phenylpropanoid pathway. These findings may enhance our understanding of non-coding genes in plants.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;association studies&quot;">association studies</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;epistatic interaction&quot;">epistatic interaction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;long non-coding RNA&quot;">long non-coding RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;target mimics&quot;">target mimics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Populus tomentosa&quot;">Populus tomentosa</a></div>                            <h2 scrollto-destination=106355644 id="106355644" class="section-title" >1. Introduction</h2>
<p>Long non-coding RNAs (lncRNAs) are non-coding transcripts longer than 200 nucleotides (nts). According to their genomic location, lncRNAs can be classified into intergenic, intronic, sense, and antisense types.<a reveal-id="dsx018-B1" data-open="dsx018-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> In contrast to protein-coding genes, most lncRNAs lack strong sequence conservation between species.<a reveal-id="dsx018-B2" data-open="dsx018-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> LncRNAs are usually expressed at low levels and often exhibit tissue-speciﬁc expression.<a reveal-id="dsx018-B3" data-open="dsx018-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Emerging evidence shows that lncRNAs play vital roles in numerous biological processes by regulating the expression of genes in close proximity (acting in <em>cis</em>) or at a distance (acting in <em>trans</em>) in the genome.<a reveal-id="dsx018-B4" data-open="dsx018-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a></p><p>With the advent of high-throughput sequencing, thousands of lncRNAs have been identified in several plant species. However, the detailed functional analysis of plant lncRNAs is still in its infancy. For example, vernalization in <em>Arabidopsis thaliana</em> is influenced by lncNAT ‘COOLAIR’ and intronic lncRNA ‘COLDAIR’.<a reveal-id="dsx018-B5" data-open="dsx018-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx018-B6" data-open="dsx018-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> LncRNAs also function as endogenous target mimics (eTMs) of microRNAs (miRNAs), providing a new mechanism for regulation of miRNA activity.<a reveal-id="dsx018-B7" data-open="dsx018-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>Trees, as model systems for research on plant’s secondary growth, promise to be one of the most abundant renewable sources for the production of wood biomass.<a reveal-id="dsx018-B8" data-open="dsx018-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Therefore, a better understanding of the regulation of secondary growth and wood biosynthesis is essential for improving wood quality and biomass. Previous studies of molecular mechanisms underlying plant’s secondary growth are mainly focused on functional studies of protein-coding genes, such as the 95 genes identified in the phenylpropanoid pathway.<a reveal-id="dsx018-B9" data-open="dsx018-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Nevertheless, few studies have systemically investigated the roles of lncRNAs in secondary growth of plants.</p><p>Single nucleotide polymorphisms (SNPs) represent the most frequent genetic variants among individuals and can affect gene expression, function, and phenotypes.<a reveal-id="dsx018-B10" data-open="dsx018-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Recent studies have linked SNPs in lncRNAs to the abnormal expression and dysregulation of the lncRNAs.<a reveal-id="dsx018-B11" data-open="dsx018-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> For example, a SNP in the lncRNA long-day-speciﬁc male-fertility–associated RNA affected the expression of the lncRNA, resulting in photoperiod-sensitive male sterility in rice.<a reveal-id="dsx018-B12" data-open="dsx018-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Genome-wide association studies in human have assigned 93% of the disease- or trait-predisposing SNPs to non-coding regions that could affect the expression of non-coding RNAs.<a reveal-id="dsx018-B13" data-open="dsx018-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx018-B14" data-open="dsx018-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Therefore, identification of phenotype-related SNPs within lncRNA loci and association analysis that link the genetic variations (SNPs) to phenotypic variations can provide new clues for their potential functions of lncRNAs.</p><p>Here, we systemically identified and characterized lncRNAs from the main vascular tissues (cambium, developing xylem, and mature xylem) of secondary growth in <em>Populus tomentosa</em> with high and low biomass. Prediction of functional motifs, potential target genes (PTGs), and miRNAs regulation for the lncRNAs illustrated their proposed functions in secondary growth. We found SNPs within the lncRNA loci and their PTGs in a natural population of <em>P. tomentosa</em>. Genetic association was then used to associate these SNPs with phenotype variations to further explore functions of lncRNAs. Overall, the findings provide insights into the roles of lncRNAs in secondary growth of plants.</p>                            <h2 scrollto-destination=106355650 id="106355650" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106355651 id="106355651" class="section-title" >2.1. Plant materials</h3>
<p>This study was carried out in a hybrid population of 722 F<sub>1</sub> individuals established by controlled crossing between two poplar parents, clone ‘Pt-3’ (<em>P. tomentosa</em>) as female and ‘Pt-TB14’ (<em>P. tomentosa</em>) as male. The progeny of this population were grown in 2009 in Guan Xian County, Shandong Province, China (36°23′N, 115°47′E) using a randomized complete block design with three replications. All individuals were scored for three growth traits, including diameter at breast height (DBH), tree height (H), and stem volume (V). Based on the measurement (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s1.png?Expires=2147483647&Signature=wb7kqOqnjka9TaPq-RPP1IKke2vLxJaN39OFFPGV26aR~gdMArUmL7a~czlkRk5RgBevdj023LqKANJrycvSy9VvtfdPpbhlJtMCuUzY16yICE~tjoAe~L0VpBTEXAifKGLv4sD1rMlgPoFzwFMTYdy8~j0BfmTVp03sIXFI~ZY-O8PRxoZyJ12XMUmB72rkyYlhTVZSOztnABkG0QSr4nRu90T~iJnhcAESXlBYYnq7i-x4lgQ9FdNqPpt6lpDxVacYaLIO2Ke-r4ccToWWMiD9MbFaDM3Niq8GL4d-Zar0uYKxoUuM2xyCRjwQVO~a~brgGaGwCF~kt30Fz3KF~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), three genotypes with highest biomass and three with lowest biomass (measured as wood volume) were selected to produce two pooled samples (high- and low-biomass pools) with three biological replicates. A <em>t</em>-test (SAS v8.2) showed a significant difference between the high and low biomass groups (<em>P</em> ≤ 0.01). For each pooled sample, the stem bark of each tree was gently removed in strips (100 mm wide and 350–400 mm long) and then the soft cambial cells were gently scraped off. After that, the developing xylem cells (newly formed xylem about 3 mm away from the exposed surface) below the cambium were immediately scraped away and the mature xylem cells were obtained by scraping deeper into the wood, using the technique described by Dalessandro and Northcote.<a reveal-id="dsx018-B15" data-open="dsx018-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> These tissues, including vascular cambium, developing xylem, and mature xylem from the high-biomass pools (HC, HD, and HM) and low-biomass pools (LC, LD, and LM), respectively, were immediately collected and frozen in liquid nitrogen and then stored at −80 °C for RNA extraction.</p>                            <h3 scrollto-destination=106355653 id="106355653" class="section-title" >2.2. RNA isolation, library construction and RNA-seq</h3>
<p>Total RNA from HC, HD, HM, LC, LD, and LM was extracted using the Qiagen RNAeasy kit (Qiagen China, Shanghai, China) according to the manufacturer’s instructions. Additional on-column DNase digestions were performed during the RNA purification using RNase-Free DNase Set (Qiagen). The six total RNA samples were monitored by NanoDrop ND-1000 and Agilent Bioanalyzer 2100 before RNA-seq and then used to construct strand-specific RNA-seq libraries separately according to the TruSeq RNA Sample Preparation Guide. After being quantified with the Qubit 2.0 Fluorometer and Agilent 2100 bioanalyzer, the six strand-specific libraries were sequenced on the Illumina HiSeq2500 platform with the paired-end programme. Library construction and Illumina sequencing were carried out by Shanghai Biotechnology Corporation (Shanghai, China). The sequencing data have been deposited in the NCBI Sequence Read Archive (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra/" target="_blank">http://www.ncbi.nlm.nih.gov/sra/</a>) under the accession number SRP073689.</p>                            <h3 scrollto-destination=106355655 id="106355655" class="section-title" >2.3. Prediction of <em>Populus</em> lncRNAs</h3>
<p>Pre-processing of raw sequencing data were performed using FASTX-Toolkit version 0.0.13 (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/index.html" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/index.html</a>) with default parameters by removing adapter sequences, low-quality reads (Q20 &lt; 20), sequences shorter than 20 nts. To rule out rRNAs, high-quality reads that passed the quality filters were aligned to <em>Populus</em> rRNA sequences using the Short Oligonucleotide Analysis Package (SOAP2; <a class="link link-uri" href="http://soap.genomics.org.cn/soapaligner.html" target="_blank">http://soap.genomics.org.cn/soapaligner.html</a>). <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s1.doc?Expires=2147483647&amp;Signature=e~TbIPr9vwUcsfRYa7izNq-gfa-sdKrSwrHTls3UvvsfSBwiwWx73FJmSHi1PIUZMwEDX2bHTBsxg0ooE0Irz8F8KR4VNNDx1H2ZckNzQqWI66QjW-DyyMYKeJxP5PxEOoYK6H~bc3EXhhRxgMSCf0AhxmKWO5MLPUfCnhE-IXh1OPonCPgoCdok0c0bOOTRpuhZLpUhwj~cyRmemyWigz-X9DgOy~34JBdQJuV2cJOJrOh-5SjjfPnCEksar9cuEsLJnah4Gh-T6lCo~y2eJp02puK8DA34Kh~mXbF43Z6gdXC4tt8JEJbnyFDko6~dIB1ep0eGjIqBO5PJl1WCgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> shows the summary of the sequencing data. Then clean reads were aligned to the <em>Populus trichocarpa</em> genome (version 3.0)<a reveal-id="dsx018-B16" data-open="dsx018-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> using Tophat,<a reveal-id="dsx018-B17" data-open="dsx018-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> with 3 base mismatches allowed. The reads with no more than three mismatches were used to construct transcripts of each sample separately using Cufflinks v2.1.1<sup>17</sup> based on the <em>P. trichocarpa</em> genome reference. To reduce transcriptional noise, only those assembled transcript isoforms that were detected in two or more experiments were retained for further analyses, as performed by Li <em>et al.</em><a reveal-id="dsx018-B18" data-open="dsx018-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Then, the Cuffcompare programme was used to merge the RefSeq, Ensembl, and UCSC <em>Populus</em> known genes into one set of gene annotations for comparison with the assembled transcripts. Expression levels (FPKM, fragments per kilobase of transcript per million fragments) of these assembled transcripts were calculated by Cufflinks v2.1.1.<a reveal-id="dsx018-B17" data-open="dsx018-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The prediction of lncRNAs from RNA-seq data was performed according to Sun <em>et al.</em><a reveal-id="dsx018-B19" data-open="dsx018-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and the pipeline was shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s2.png?Expires=2147483647&amp;Signature=b2rklauGA1wSTfsq6TUnvxnz9gnSJlKqd4pDovERscHqr8K6kVI91YXMSe-mlQM7RiPa69lVFjcB1zf8J0qyaoqayMRXkJGfI7i50C4fA6Vgh2Tw4v4pNJPs2vTE9gGyLu5MYmf8i-AcFkUNP3Jum4cNhOnj8BOyecsulNJRU0d4cJS-Duq~tQkDj2Kzu6D7hVHGo~cItFi-1ydqBmSMasBO~FSQW1e3i~bvolX3XKrnUiBa~CIiDoBETXpzUBA2iVqcdnEKxWCMUkHk5APFQnWao1khQsf2buzGCpsIsrYvJFvp7Tcgup0~dSh8dGs~t3WkP885ixSzzVdZ8VJEyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. We only retained transcripts longer than 200 nts with an open reading frame shorter than 100 amino acids and optimum expression threshold of FPKM &gt; 1.0 in at least one sample. After that, we used double filters to evaluate the protein-coding potential of the remaining transcripts. Transcripts were first aligned to the Pfam database using HMMER3.0<a reveal-id="dsx018-B20" data-open="dsx018-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> to eliminate transcripts with potential protein-coding ability (cutoff <em>E</em>-value ≤ 0.001). Next, we employed the Coding Potential Calculator (CPC) software<a reveal-id="dsx018-B21" data-open="dsx018-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and Coding–non-coding Index (CNCI) software<a reveal-id="dsx018-B22" data-open="dsx018-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> to evaluate the protein-coding potential of the remaining transcripts with default parameters. When using CPC, we used the protein-coding transcripts of <em>P. trichocarpa</em> as a reference. Only transcripts that did not pass the protein-coding-score test (CPC score &lt; 0, CNCI score &lt; 0) were classified as putative lncRNAs. LncRNAs were classified into four categories (‘i’, ‘o’, ‘u’, and ‘x’), consisting of lncRNAs without overlap with any genes (intergenic lncRNAs, ‘u’), antisense overlapping lncRNAs (antisense lncRNAs, ‘x’), intronic overlapping lncRNAs (intronic lncRNAs, ‘i’) and generic exonic overlapping lncRNAs (sense lncRNAs, ‘o’), according to their genomic locations relative to neighbouring genes as defined by cuffcompare programme in Cufflinks suite.<a reveal-id="dsx018-B23" data-open="dsx018-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> The difference in lncRNA expression was calculated as the fold change (FC) = FPKM of ‘sample1’/FPKM of ‘sample2’. LncRNA transcripts were considered to be differentially expressed (DE) if they meet the criteria of FC ≥ 2 or ≤ 0.5 with <em>P</em>-value &lt; 0.01.</p>                            <h3 scrollto-destination=106355657 id="106355657" class="section-title" >2.4. Genomic characterization and specific expression of <em>Populus</em> lncRNAs</h3>
<p>GC contents of the identified lncRNAs were calculated with EMBOSS explorer’s geecee tool (<a class="link link-uri" href="http://emboss.sourceforge.net/apps/cvs/emboss/apps/geecee.html" target="_blank">http://emboss.sourceforge.net/apps/cvs/emboss/apps/geecee.html</a>). To explore lncRNA conservation, all the lncRNA sequences identified here were aligned with BLASTN against the genome sequences of <em>A</em>. <em>thaliana</em><a reveal-id="dsx018-B24" data-open="dsx018-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and rice<a reveal-id="dsx018-B25" data-open="dsx018-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> with a cutoff <em>E</em>-value &lt; 1e-10. The genomes were downloaded from Phytozome (v9.1) (<a class="link link-uri" href="http://www.phytozome.net/" target="_blank">http://www.phytozome.net/</a>). The lncRNAs that had &gt;20% of their sequence matched to other genomes were defined as conserved lncRNAs. Motifs in lncRNAs were identified using DREME online software specially designed to find relatively short motifs with <em>E</em>-value &lt; 0.05 and the possible roles of each identified motif were predicted by using the GOMo (Gene Ontology for Motifs) annotation sever with <em>q</em>-value &lt; 0.05.<a reveal-id="dsx018-B26" data-open="dsx018-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx018-B27" data-open="dsx018-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The tissue specificity of lncRNA expression was evaluated according to the tissue-specific index, which ranges from 0 for housekeeping genes to 1 for tissue-restricted genes, as described by Yanai <em>et al.</em><a reveal-id="dsx018-B28" data-open="dsx018-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> The index was calculated as: tissue-specific index = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac xmlns=""><mrow><mrow><munderover><mo stretchy="false">∑</mo><mrow><mi>i</mi><mi> </mi><mo></mo><mo>=</mo><mi mathvariant="normal"> </mi><mo></mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></munderover><mrow><mo>(</mo><mn>1</mn><mo>-</mo><mfrac><mrow><msub><mrow><mi mathvariant="normal">e</mi><mi mathvariant="normal">x</mi><mi mathvariant="normal">p</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">e</mi><mi mathvariant="normal">x</mi><mi mathvariant="normal">p</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">x</mi></mrow></msub></mrow></mfrac><mo stretchy="true">)</mo></mrow></mrow></mrow><mrow><mi>n</mi><mo></mo><mi mathvariant="normal"> </mi><mo>-</mo><mo></mo><mi mathvariant="normal"> </mi><mn>1</mn></mrow></mfrac></math>⁠</span>, where n is the number of tissues; exp<sub><em>i</em></sub> is the expression value of each lncRNA in tissue, <em>i</em>; and exp<sub>max</sub> is the maximum expression value of each lncRNA among all tissues. Only the lncRNAs showing a tissue-specific index &gt; 0.9 were considered to be tissue-specific.</p>                            <h3 scrollto-destination=106355659 id="106355659" class="section-title" >2.5. Prediction of PTGs of <em>Populus</em> lncRNAs</h3>
<p>Two independent algorithms were then used to predict PTGs of the DE lncRNAs based on the regulatory effects of the lncRNA (<em>cis</em>- or <em>trans</em>-acting). For the first algorithm, genes transcribed within a 10 kb window upstream or downstream of each lncRNA were considered as potential <em>cis</em> target genes by using genome annotation and a genome browser, as described by Jia <em>et al.</em><a reveal-id="dsx018-B29" data-open="dsx018-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The second algorithm for the prediction of potential <em>trans</em> target genes is based on sequence complementarity and RNA duplex energy prediction to access the impact of lncRNA binding on mRNA molecules, as described previously in.<a reveal-id="dsx018-B30" data-open="dsx018-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Specifically, the algorithm first employed BLAST to select target sequences complementary to the lncRNAs with parameters of <em>e</em>-value &lt; 1e-5 and identity ≥ 95% and then used RNAplex to calculate the complementary energy between two sequences for further screening and selection of potential <em>trans</em> target genes.<a reveal-id="dsx018-B31" data-open="dsx018-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The predicted target genes were then annotated by PopGenie (<a class="link link-uri" href="http://www.popgenie.org/" target="_blank">http://www.popgenie.org/</a>) and gene ontology (GO) terms were analysed by agriGO (<a class="link link-uri" href="http://bioinfo.cau.edu.cn/agriGO/index.php" target="_blank">http://bioinfo.cau.edu.cn/agriGO/index.php</a>) to find significant GO categories [false discovery rate (FDR) ≤ 0.05]. In addition, we used the KEGG database (<a class="link link-uri" href="http://www.genome.ad.jp/kegg/" target="_blank">http://www.genome.ad.jp/kegg/</a>) and a hypergeometric statistic test to analyses their potential roles in the pathways.</p>                            <h3 scrollto-destination=106355661 id="106355661" class="section-title" >2.6. Prediction of lncRNAs as potential precursors, targets, and target mimics of miRNAs</h3>
<p>To explore whether some lncRNAs may act as precursors of miRNAs, the 401 <em>Populus</em> known miRNAs in miRBase (Release 20.0, <a class="link link-uri" href="http://www.mirbase.org/" target="_blank">http://www.mirbase.org/</a>) were aligned to the sequences of the novel lncRNAs and secondary structure prediction was then executed using the Vienna RNA package RNAfold web (<a class="link link-uri" href="http://rna.tbi.univie.ac.at/" target="_blank">http://rna.tbi.univie.ac.at/</a>). LncRNAs with classic stem-loop hairpins were regarded as putative precursors of miRNAs. Furthermore, to investigate whether some lncRNAs may be targeted by miRNAs, the target genes of miRNAs were predicted by psRNATarget (<a class="link link-uri" href="http://plantgrn.noble.org/psRNATarget/" target="_blank">http://plantgrn.noble.org/psRNATarget/</a>) with expectation ≤ 3 based on the near-perfect complementarity between miRNAs and target genes and target-site accessibility. The miRNA eTMs from <em>Populus</em> lncRNAs were predicted with local scripts by using the algorithm developed by Wu <em>et al.</em><a reveal-id="dsx018-B32" data-open="dsx018-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Quiet different from the authentic targets that are nearly perfectly complementary to miRNAs, eTMs can bind to a miRNA with a 3-nt bulge between the 5′ end 9th to 12th positions of the miRNA, thus serving as decoys for the miRNA to interfere with the binding of the miRNA to its authentic targets. Based on these prediction results, a potential interaction network of mRNAs, miRNAs, and lncRNAs was modelled with Cytoscape 3.2.<a reveal-id="dsx018-B33" data-open="dsx018-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a></p>                            <h3 scrollto-destination=106355663 id="106355663" class="section-title" >2.7. Real-time quantitative PCR</h3>
<p>Real-time quantitative PCR (qRT-PCR) was performed on a 7500 Fast RT-PCR System using the SYBR Premix Ex Taq as described in the manufacturer’s instructions. The cDNA template for the reactions was reverse-transcribed using total RNA extracted from cambium, developing xylem, and mature xylem of the high- and low-biomass samples, as described earlier. Primers were designed using Primer Express 3.0 software (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). The efficiency of the primers was calculated by performing RT-PCR on several dilutions of first-strand cDNAs. Efficiencies of the different primer sets were similar. The specificity of each primer set was checked by sequencing PCR products. All qRT-PCR amplifications were carried out in triplicate, with the standard reaction programme and the specificity of the amplified fragments was checked using the generated melting curve. The generated real-time data were analysed using the Opticon Monitor Analysis Software 3.1 tool and standardized to the levels of poplar ACTINII-like (Accession number: EF145577) using the 2<sup>−ΔCt</sup> or 2<sup>−ΔΔCt</sup> method.<a reveal-id="dsx018-B34" data-open="dsx018-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h3 scrollto-destination=106355665 id="106355665" class="section-title" >2.8. Association population</h3>
<p>The association population used for association studies consisted of 435 unrelated individuals of <em>P. tomentosa</em> (2n = 38), representing almost the entire geographic distribution of <em>P. tomentosa</em> as described in our previous study.<a reveal-id="dsx018-B35" data-open="dsx018-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> These trees were selected from the collection of 1,047 native individuals collected from the entire natural distribution of <em>P. tomentosa</em> (30–40°N, 105–125°E) in 1982 by using root cuttings, and were grown using a randomized complete block design with three replications in Guan Xian County, Shandong Province, China (36°23′N, 115°47′E).</p><p>The 435 individuals of the population were then scored for 10 quantitative traits, with at least three clones measured per genotype. The three growth traits (DBH, H, and V) and seven wood property traits have been measured and analysed by Du <em>et al.</em><a reveal-id="dsx018-B36" data-open="dsx018-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The wood property traits were microfibril angle (MFA), fibre length (FL), fibre width (FW), holocellulose content (HoC), hemicellulose content (HemC), α-cellulose content (αC), and lignin content (LiC). The phenotypic data used for association analysis are shown in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2.</p>                            <h3 scrollto-destination=106355668 id="106355668" class="section-title" >2.9. Re-sequencing of the association population and SNP calling</h3>
<p>The set of 435 individuals from the association population were re-sequenced using the Illumina GA2 instrument with an average of 15-fold coverage, and the libraries were constructed based on genomic DNA, following the manufacturer’s recommendations (Illumina). The sequence quality of paired-end short reads of 100 base pair (bp) was controlled by removing low-quality reads (≤50% of nucleotides with a quality score &lt; Q20).<a reveal-id="dsx018-B37" data-open="dsx018-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Then the paired-end short reads were aligned and mapped to the <em>Populus</em> reference genome using SOAPaligner/SOAP2 v2.20 with default options.<a reveal-id="dsx018-B38" data-open="dsx018-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> To get high-quality SNPs, only the reads that could be matched to a unique genomic location were used for SNP calling; reads that mapped to at least two genomic positions were excluded. After SNP calling, we used our previous SNP data from 10 candidate genes in 120 individuals identified by PCR-Sanger sequencing to validate our SNP calling results and found that the accuracy of SNP calling reached 97.5%.<a reveal-id="dsx018-B37" data-open="dsx018-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Then the SNPs among the association population were mined using the Variant Call Format tool (version 4.1).<a reveal-id="dsx018-B39" data-open="dsx018-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> BLASTN and BLASTX were employed with a cutoff <em>E</em>-value &lt; 1e-10 to obtain information in the location of the lncRNAs genes that represent DNA sequences encoding the lncRNAs and their PTGs, including promoter regions (2,000 bp upstream), or gene fragments, respectively. The genomic DNA sequences of the PTGs of the <em>P. tomentosa</em> lncRNAs have been deposited in GenBank under the Accession Number KX155245-KX155490 (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S3). We then used UltraEdit 3.2 (<a class="link link-uri" href="http://www.ultraedit.com/" target="_blank">http://www.ultraedit.com/</a>) to capture the genotypes of the SNPs from the SNP-calling pipeline. Low-quality SNP markers with missing data (&gt;10%) were excluded from further analysis.</p><p>SNP diversity and linkage disequilibrium (LD) analysis: On the basis of the SNP genotyping data, the average number of pair-wise differences per site between sequences, π,<a reveal-id="dsx018-B40" data-open="dsx018-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and the number of segregating sites, θ<sub>w</sub>,<a reveal-id="dsx018-B41" data-open="dsx018-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> were used for calculation of SNP diversity using TASSEL v5.0 (<a class="link link-uri" href="http://www.maizegenetics.net/" target="_blank">http://www.maizegenetics.net/</a>). Also, Tajima’s D<a reveal-id="dsx018-B42" data-open="dsx018-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> for neutrality tests was estimated using the genotypic data. LD tests were performed in the association population using common SNPs with missing data (&lt;25%) and minor allele frequencies (MAFs) &gt; 0.05 in the natural population. The squared correlation of allele frequencies value (<em>r</em><sup>2</sup>) was calculated in TASSEL v5.0. To assess the extent of LD, the decay of LD with physical distance (bp) between the common SNPs was estimated by nonlinear regression for the genes and lncRNAs within the same chromosome.</p>                            <h3 scrollto-destination=106355671 id="106355671" class="section-title" >2.10. Association analysis</h3>
<p>Single SNP-based associations: we used the mixed linear model (MLM) in TASSEL v5.0 to identify the SNP-trait associations in the association population. This model takes into account the effects of population structure and relatedness among individuals for marker-trait associations.<a reveal-id="dsx018-B43" data-open="dsx018-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Based on 20 genomic simple sequence repeat markers described previously in,<a reveal-id="dsx018-B44" data-open="dsx018-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> the pair-wise kinship coefficients (<em>K</em>) were evaluated by SPAGeDi 1.3,<a reveal-id="dsx018-B45" data-open="dsx018-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and the association population structure (<em>Q</em>) was obtained based on significant subpopulations (<em>k</em> = 3) using STRUCTURE 2.3.4.<a reveal-id="dsx018-B46" data-open="dsx018-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Specifically, the genotypic effects of associated SNPs could be effectively decomposed into additive and dominant effects under the model. Then, corrections for the testing of <em>P</em>-values for all the associations were performed using the FDR through QVALUE.<a reveal-id="dsx018-B47" data-open="dsx018-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> A <em>q</em>-value of 0.10 was considered as the significance threshold.</p><p>Epistasis analysis: EPISNP v2.0 (<a class="link link-uri" href="http://animalgene.umn.edu/episnpmpi/download.html" target="_blank">http://animalgene.umn.edu/episnpmpi/download.html</a>) provides an ideal genetic modelling method for multiple SNPs to investigate epistatic interactions and was developed for testing epistatic SNP effects on quantitative traits.<a reveal-id="dsx018-B48" data-open="dsx018-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> It incorporated tests of five epistasis effects for each pair of SNPs, including two-locus interaction, additive × additive (AA), additive × dominance (AD) or dominance × additive (DA), and dominance × dominance (DD), based on the extended Kempthorne model.<a reveal-id="dsx018-B49" data-open="dsx018-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The contribution rate of the pair-wise SNPs was calculated by the formula: <em>c</em> = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac xmlns=""><mrow><msub><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">S</mi></mrow><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">P</mi><mn>1</mn><mo></mo><mo>×</mo><mi mathvariant="normal">S</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">P</mi><mn>2</mn></mrow></msub></mrow><mrow><msub><mrow><mi mathvariant="normal">V</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi></mrow><mrow><mi mathvariant="normal">p</mi></mrow></msub></mrow></mfrac></math>⁠</span>, where SS<sub>SNP1 × SNP2</sub> is the variance of the significant (<em>P</em> &lt; 0.001) SNP1 × SNP2 interactive effect (AA, AD, DA, or DD), and the Var<sub>p</sub> is the phenotype variance. An <em>F</em>-test was used to test the significance of the two-locus interaction effect. The same FDR method was also used to correct for multiple tests.</p>                            <h2 scrollto-destination=106355674 id="106355674" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=106355675 id="106355675" class="section-title" >3.1. Genome-wide identification and characterization of lncRNAs in <em>P. tomentosa</em></h3>
<p>The systematic genome-wide identification of <em>P</em>. <em>tomentosa</em> lncRNAs was performed by high-throughput RNA-seq of six libraries of secondary vascular tissues (HC, HD, HM, LC, LD, and LM). In total, we identified 15,691 lncRNAs with the number ranging from 6,384 (HC) to 9,732 (LM) across the six sample pools (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). We then characterize the basic genomic features of these lncRNAs. When compared with protein-coding genes, the <em>Populus</em> lncRNAs are more evenly distributed with no obvious location preferences across chromosomes (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). According to their genomic locations, these lncRNAs were portioned into 15,402 sense lncRNAs, 160 intergenic lncRNAs, 77 antisense lncRNAs, and 52 intronic lncRNAs (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). The lncRNAs ranged from 201 to 7,862 nt in length with an median of 1,010 nt (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>), which were shorter than protein-coding genes of <em>P. tomentosa</em> (median length of 1,888 nt). The mean GC content was 41.3%, a little lower than that of protein-coding sequences (42.6%) (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S4). Conservation analysis showed that only 42.4 and 20.2% of <em>P. tomentosa</em> lncRNAs are conserved in <em>Arabidopsis</em> and rice, respectively (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>). Moreover, we identified that only 11 <em>Populus</em> lncRNAs could be processed to be 15 mature miRNAs (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1F;</a><span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s2.doc?Expires=2147483647&amp;Signature=bDPDShCJ1o-mq5qpuoPzn8N6WC2KQG4ci8VzA5FZTqzdEDLFAB3n-67KBQqCE2O1xnXG5PAVd-eY5soPNJfIrTgzVJzUkP2yU-hUVMcD15x8KDnF1ahD8QeQJb2Iw4I~ZIA8sxcgNCoWJvXTr~TgkmFAnzgiB580nDw-EwSuCYru635MZitViYmfxhyBFXiNta1Dq5nY4nIqN6gzTwrnMZDuJFElm4YQc86cRRlkWs9pYad-~9vPCA~FrKcz72j994viUtNtphFZiqfovSSiwLzENFbb6vr2~2VRQNS549T3CzG6kampMoCoiZDvQvpUgq~8KIarxPhDw-johJKHfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), implying that the majority of the lncRNAs identified here undergo processing by miRNA-independent pathways. </p>                        <a id="106355677" scrollto-destination="106355677"></a>
<div id="dsx018-F1" data-id="dsx018-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f1.png?Expires=2147483647&Signature=gi-9QLcoem4FRIZTzGjRnF~GSExbsjgZt7SlJpxwodJdu-W0WN7qKsG~RPFEUrel8ojRUyU3xLrkGFzWvAjZXnmOGTHfGECNUTFyCp-ifeqvgDUEgKvBJ~CQ3-UG4JRl-27d08xdaHOoGg964JwIXzWXub~n1TcO~MXzqYeJhk5Rxn4CKhP8iu3rRjoKMfw5QTCPEjk6EWTZz4ysLbIswL170M-akhazTNWGqwbaxBx68omhsERX1I6cDptPB2Yb8lph26MxlE8zT8y6TAnVLPQZcrl7YmTiCN7yX2s2EDaweGqjxDNQ0oPA3ny2M0JunmXSDkc4xr48~S4ykp1cRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of P. tomentosa lncRNAs. (A) Venn diagram showing the number of totally and uniquely identified lncRNAs in each sample. (B) Distribution of lncRNAs along each chromosome for each sample generated using Circos. (C) Distribution of four types of lncRNAs, including sense, intergenic, antisense, and intronic lncRNAs. (D) Length distribution of lncRNAs. (E) Sequence conservation of Populus lncRNAs in Arabidopsis and rice. (F) Predicted secondary structure of the lncRNA TCONS_00034791 (partial) and the precursor of ptc-miR403a. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355677/dsx018f1.png" path-from-xml="dsx018f1.png" target="_blank">View large</a><a section="106355677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f1.png?Expires=2147483647&Signature=W9eBkgmm0kLtetSCPIyB9izDtFFOBoFDBqB4ilwYLhylbH0OUdPCUzkY~~H6Icv3L~LXEP~AE2~NUSUDdeJ28yVAsWdpuxc6QKl7CiOOGyNkt22tfhtF1RS73KGQTOQCA7XVhYm8R2s6QwULckXuKvk0fZFpHS9k8laHXJZJMgKk6LDFIrZVqfybNvs6xe2HMtE0HECpqTDZfiZgdt3Ok313C~upYfvnjdM4cp~2Kapy5UmO-T6JfFAqOeROpkE2NkD17lOSaONmAAX~VE~8I-jruWfb4bJGD7Mh8pBh4WL82O6i63IL5RKC70uw00wsMWvHh3Jd8HoqQWA3balKaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355677&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics of <em>P. tomentosa</em> lncRNAs. (A) Venn diagram showing the number of totally and uniquely identified lncRNAs in each sample. (B) Distribution of lncRNAs along each chromosome for each sample generated using Circos. (C) Distribution of four types of lncRNAs, including sense, intergenic, antisense, and intronic lncRNAs. (D) Length distribution of lncRNAs. (E) Sequence conservation of <em>Populus</em> lncRNAs in <em>Arabidopsis</em> and rice. (F) Predicted secondary structure of the lncRNA <em>TCONS_00034791</em> (partial) and the precursor of ptc-miR403a. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div><div content-id="dsx018-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f1.png?Expires=2147483647&Signature=gi-9QLcoem4FRIZTzGjRnF~GSExbsjgZt7SlJpxwodJdu-W0WN7qKsG~RPFEUrel8ojRUyU3xLrkGFzWvAjZXnmOGTHfGECNUTFyCp-ifeqvgDUEgKvBJ~CQ3-UG4JRl-27d08xdaHOoGg964JwIXzWXub~n1TcO~MXzqYeJhk5Rxn4CKhP8iu3rRjoKMfw5QTCPEjk6EWTZz4ysLbIswL170M-akhazTNWGqwbaxBx68omhsERX1I6cDptPB2Yb8lph26MxlE8zT8y6TAnVLPQZcrl7YmTiCN7yX2s2EDaweGqjxDNQ0oPA3ny2M0JunmXSDkc4xr48~S4ykp1cRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristics of P. tomentosa lncRNAs. (A) Venn diagram showing the number of totally and uniquely identified lncRNAs in each sample. (B) Distribution of lncRNAs along each chromosome for each sample generated using Circos. (C) Distribution of four types of lncRNAs, including sense, intergenic, antisense, and intronic lncRNAs. (D) Length distribution of lncRNAs. (E) Sequence conservation of Populus lncRNAs in Arabidopsis and rice. (F) Predicted secondary structure of the lncRNA TCONS_00034791 (partial) and the precursor of ptc-miR403a. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355677/dsx018f1.png" path-from-xml="dsx018f1.png" target="_blank">View large</a><a section="106355677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f1.png?Expires=2147483647&Signature=W9eBkgmm0kLtetSCPIyB9izDtFFOBoFDBqB4ilwYLhylbH0OUdPCUzkY~~H6Icv3L~LXEP~AE2~NUSUDdeJ28yVAsWdpuxc6QKl7CiOOGyNkt22tfhtF1RS73KGQTOQCA7XVhYm8R2s6QwULckXuKvk0fZFpHS9k8laHXJZJMgKk6LDFIrZVqfybNvs6xe2HMtE0HECpqTDZfiZgdt3Ok313C~upYfvnjdM4cp~2Kapy5UmO-T6JfFAqOeROpkE2NkD17lOSaONmAAX~VE~8I-jruWfb4bJGD7Mh8pBh4WL82O6i63IL5RKC70uw00wsMWvHh3Jd8HoqQWA3balKaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355677&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristics of <em>P. tomentosa</em> lncRNAs. (A) Venn diagram showing the number of totally and uniquely identified lncRNAs in each sample. (B) Distribution of lncRNAs along each chromosome for each sample generated using Circos. (C) Distribution of four types of lncRNAs, including sense, intergenic, antisense, and intronic lncRNAs. (D) Length distribution of lncRNAs. (E) Sequence conservation of <em>Populus</em> lncRNAs in <em>Arabidopsis</em> and rice. (F) Predicted secondary structure of the lncRNA <em>TCONS_00034791</em> (partial) and the precursor of ptc-miR403a. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div>                            <h3 scrollto-destination=106355678 id="106355678" class="section-title" >3.2. Expression profiles of <em>Populus</em> lncRNAs in secondary vascular tissues</h3>
<p>We next explored the expression of the lncRNAs in secondary vascular tissues from the two biomass pools. First, for the maximal expression levels (exp<sub>max</sub>), we compared expression levels of each transcript among all sequenced samples and found that lncRNAs and mRNAs exhibit different density peaks and that the density peaks of mRNAs lag behind those of lncRNAs (<a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Indeed, expression categorization showed that the majority of lncRNAs displayed low expression levels, with only 2% highly expressed (exp<sub>max</sub> &gt; 50 FPKM) (<a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). The largest number of <em>Populus</em> lncRNAs showed expression peaks in LM (25.6%) followed by HM (24.1%) (<a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Then, using the tissue-specific expression index (see ‘Materials and methods’ section), we found 2,746 <em>Populus</em> lncRNAs showed tissue-specific expression (<a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>), and more than half of these (1,512 of 2,746) were preferentially expressed in stem xylem (907 in HM and 605 in LM; <a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2E</a>). Our qRT-PCR revealed nearly perfect concordance with the RNA-seq results for the expression patterns and verified the expression specificity of the tested lncRNAs (<a reveal-id="dsx018-F2" data-open="dsx018-F2" class="link link-reveal link-table xref-fig">Fig. 2F</a>). </p>                        <a id="106355680" scrollto-destination="106355680"></a>
<div id="dsx018-F2" data-id="dsx018-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f2.png?Expires=2147483647&Signature=RuRoes8DsD8IvsK6wmtvXuEEfv-1tihxKK4~URUdlYVR-ewEFKgpiPQLvq9tKZqwUk8hism-16YIxmp8XVEc24WTvP9tAjbgo1SnuX~2bOIJYZEyKpgPN17NybwWzjUpChqJAm13DKKvm2ZitezErEziXi05C--6PINGxzqD1YMCT2phYbXitawX2VNapFa5PvEioCQ1qCqAD4oPj7DqLoXIgQxkenJNpRmk0cNNX3YHTueZ3m-iWkzRianzhG7s~87ZaDvGNbxcIwL2~vp3vHoo-Co1a2NeI8X6oH9d7upLdnYFYhOMvCKw859LD2NizKbmiJnaHX1Ka0v9y2cEpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of Populus lncRNAs in different vascular tissues of high- and low-biomass pools. (A) Density plot of maximum expression levels of lncRNAs and mRNAs across the six samples (high and low biomass × vascular cambium, developing xylem, and mature xylem). The expression levels are normalized to log2FPKM. (B) The proportion of lncRNAs showing different expression levels across all samples. The expression levels (FPKM) of lncRNAs can be divided into three classes: Class I (expmax ≤ 10 FPKM), Class II (expmax &gt; 10 FPKM and ≤ 50 FPKM) Class III (expmax &gt; 50 FPKM). expmax,the maximum expression levels of lncRNAs across all samples. (C) Tissue distribution of expression peaks of lncRNAs and mRNAs. (D) The tissue-specific lncRNAs. The total numbers of lncRNAs in each group is given. (E) Heat maps of lncRNAs with tissue-specific expression (log2FPKM). Colour intensity represents the fractional density across the row of log-normalized FPKM. (F) Expression validation by qRT-PCR. Error bars represent standard deviations (n = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355680/dsx018f2.png" path-from-xml="dsx018f2.png" target="_blank">View large</a><a section="106355680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f2.png?Expires=2147483647&Signature=Jz4rpkDhiIaMt6P3iTNDwvbcZLUsHmCgoJi2x~uzS7iYHi4Vu8mi73JHP-t6Z2mNmsmC1TN5WxtahnBg1PKcyK2flXUQ0t-FTpHSMp7K3DJfExL55zN6jC4GjBm8IQtOvFJph~PEB5yOeRzsj8NOtKZXh3TC88vsodUXbPOUoiWFo2pRw0Kpe7B5i3~ZGT87dOvQZnx36i9huwWWLyPNcED9kbdFHq4QXFlK2B9fuHBuTHM6eZsZll6-uihsvetMfXu6JW3-eSzOr1f6zAFdsAmUprzSEcv-AiMAWUkX4SzQIcXajLZzh7IDV91RG9cio-4ReeR5Q8wCFMYlF4rjkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355680&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of <em>Populus</em> lncRNAs in different vascular tissues of high- and low-biomass pools. (A) Density plot of maximum expression levels of lncRNAs and mRNAs across the six samples (high and low biomass × vascular cambium, developing xylem, and mature xylem). The expression levels are normalized to log<sub>2</sub>FPKM. (B) The proportion of lncRNAs showing different expression levels across all samples. The expression levels (FPKM) of lncRNAs can be divided into three classes: Class I (exp<sub>max</sub> ≤ 10 FPKM), Class II (exp<sub>max</sub> &gt; 10 FPKM and ≤ 50 FPKM) Class III (exp<sub>max</sub> &gt; 50 FPKM). exp<sub>max</sub><em><sub>,</sub></em>the maximum expression levels of lncRNAs across all samples. (C) Tissue distribution of expression peaks of lncRNAs and mRNAs. (D) The tissue-specific lncRNAs. The total numbers of lncRNAs in each group is given. (E) Heat maps of lncRNAs with tissue-specific expression (log<sub>2</sub>FPKM). Colour intensity represents the fractional density across the row of log-normalized FPKM. (F) Expression validation by qRT-PCR. Error bars represent standard deviations (<em>n</em> = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div><div content-id="dsx018-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f2.png?Expires=2147483647&Signature=RuRoes8DsD8IvsK6wmtvXuEEfv-1tihxKK4~URUdlYVR-ewEFKgpiPQLvq9tKZqwUk8hism-16YIxmp8XVEc24WTvP9tAjbgo1SnuX~2bOIJYZEyKpgPN17NybwWzjUpChqJAm13DKKvm2ZitezErEziXi05C--6PINGxzqD1YMCT2phYbXitawX2VNapFa5PvEioCQ1qCqAD4oPj7DqLoXIgQxkenJNpRmk0cNNX3YHTueZ3m-iWkzRianzhG7s~87ZaDvGNbxcIwL2~vp3vHoo-Co1a2NeI8X6oH9d7upLdnYFYhOMvCKw859LD2NizKbmiJnaHX1Ka0v9y2cEpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of Populus lncRNAs in different vascular tissues of high- and low-biomass pools. (A) Density plot of maximum expression levels of lncRNAs and mRNAs across the six samples (high and low biomass × vascular cambium, developing xylem, and mature xylem). The expression levels are normalized to log2FPKM. (B) The proportion of lncRNAs showing different expression levels across all samples. The expression levels (FPKM) of lncRNAs can be divided into three classes: Class I (expmax ≤ 10 FPKM), Class II (expmax &gt; 10 FPKM and ≤ 50 FPKM) Class III (expmax &gt; 50 FPKM). expmax,the maximum expression levels of lncRNAs across all samples. (C) Tissue distribution of expression peaks of lncRNAs and mRNAs. (D) The tissue-specific lncRNAs. The total numbers of lncRNAs in each group is given. (E) Heat maps of lncRNAs with tissue-specific expression (log2FPKM). Colour intensity represents the fractional density across the row of log-normalized FPKM. (F) Expression validation by qRT-PCR. Error bars represent standard deviations (n = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355680/dsx018f2.png" path-from-xml="dsx018f2.png" target="_blank">View large</a><a section="106355680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f2.png?Expires=2147483647&Signature=Jz4rpkDhiIaMt6P3iTNDwvbcZLUsHmCgoJi2x~uzS7iYHi4Vu8mi73JHP-t6Z2mNmsmC1TN5WxtahnBg1PKcyK2flXUQ0t-FTpHSMp7K3DJfExL55zN6jC4GjBm8IQtOvFJph~PEB5yOeRzsj8NOtKZXh3TC88vsodUXbPOUoiWFo2pRw0Kpe7B5i3~ZGT87dOvQZnx36i9huwWWLyPNcED9kbdFHq4QXFlK2B9fuHBuTHM6eZsZll6-uihsvetMfXu6JW3-eSzOr1f6zAFdsAmUprzSEcv-AiMAWUkX4SzQIcXajLZzh7IDV91RG9cio-4ReeR5Q8wCFMYlF4rjkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355680&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of <em>Populus</em> lncRNAs in different vascular tissues of high- and low-biomass pools. (A) Density plot of maximum expression levels of lncRNAs and mRNAs across the six samples (high and low biomass × vascular cambium, developing xylem, and mature xylem). The expression levels are normalized to log<sub>2</sub>FPKM. (B) The proportion of lncRNAs showing different expression levels across all samples. The expression levels (FPKM) of lncRNAs can be divided into three classes: Class I (exp<sub>max</sub> ≤ 10 FPKM), Class II (exp<sub>max</sub> &gt; 10 FPKM and ≤ 50 FPKM) Class III (exp<sub>max</sub> &gt; 50 FPKM). exp<sub>max</sub><em><sub>,</sub></em>the maximum expression levels of lncRNAs across all samples. (C) Tissue distribution of expression peaks of lncRNAs and mRNAs. (D) The tissue-specific lncRNAs. The total numbers of lncRNAs in each group is given. (E) Heat maps of lncRNAs with tissue-specific expression (log<sub>2</sub>FPKM). Colour intensity represents the fractional density across the row of log-normalized FPKM. (F) Expression validation by qRT-PCR. Error bars represent standard deviations (<em>n</em> = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div>                            <h3 scrollto-destination=106355681 id="106355681" class="section-title" >3.3. Functional sequence motifs and PTGs of the DE lncRNAs</h3>
<p>Using pair-wise comparisons among the six samples, we identified 1,994 DE lncRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S5), representing differential expression within or between high- and low-biomass samples (<a reveal-id="dsx018-F3" data-open="dsx018-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). To explore whether these lncRNAs have conserved elements, 55 sequence motifs were identified in the DE lncRNAs with biological functions (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S6). Intriguingly, three of the motifs specifically participate in ‘vascular tissue pattern formation’, ‘plant cell wall’, ‘xylem development’, as well as response to plant hormone stimulus (<a reveal-id="dsx018-F3" data-open="dsx018-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Since lncRNAs play important roles in regulating gene expression, identification and analysis of their target genes may help us to explore their potential functions. Computational prediction identified a set of 5,352 PTGs corresponding to 8,931 lncRNA–target pairs, including 3,569 <em>cis-</em>regulated PTGs for 1,812 lncRNAs and 3,297 <em>trans-</em>regulated PTGs for 1,824 lncRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S7). </p>                        <a id="106355683" scrollto-destination="106355683"></a>
<div id="dsx018-F3" data-id="dsx018-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f3.png?Expires=2147483647&Signature=lW7Otgfc41mSBddBbXnmdsgsVF6ZiwMfg8z6DhygsCnKK2yPNBD6VwNKf~vEkXkOE3EBNaCSOoECXmHjBB23enKHhp9nz9pqAVqsH0hSABI7h75O0VKSHyFBqDvYZff3vm2jI9uC60iXVAj57BwFsUR-IYVRUZK9e2MU0EpCTIPtco2DANbJlxA14o6VBBXW5D0S0vFug9I0mkKq~fOpXIRUut0J16abSoF6uqGWpHaCUif5Dco0vkEbZsklm0s2Gy7U24NuF5pcRWwrIp-o2YKUc0Y6NBFCs~E6Qwi1rXfBqRbiWOSv-p4jm~h-8s197bNyO-GEnDBPhsg9BTMPDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The DE lncRNAs within or between high-biomass and low-biomass pools in the P. tomentosa population. (A) Numbers of the DE (FC ≥ 2 or ≤ 0.5, P &lt; 0.01) lncRNAs among the six samples. Category I (red double-headed arrow) represents the comparison between high- and low-biomass samples. Category II (blue double-headed arrow) and Category III (green double-headed arrow) represent the comparisons among tissue types within the high- and low-biomass samples, respectively. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) Sequence motifs of the DE lncRNAs (left panel) and their corresponding GO terms (right panel). The y-axis of the sequence logo represents information contents in bits. E-values for the motifs were calculated by comparison with shuffled sequences. (C) The expression levels of lncRNAs and their potential cis-/trans targets measured by qRT-PCR. Error bars indicate standard deviations (n = 3)." path-from-xml="dsx018f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355683/dsx018f3.png" path-from-xml="dsx018f3.png" target="_blank">View large</a><a section="106355683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f3.png?Expires=2147483647&Signature=3gNgsHkJJ80DxQY5jLPJxmsiyoKd97OGWDCVv6T3SMPDUqp1DPYGa8Tw~mDkfBoy1jK70o0fQnQCDct5Kc1fLLIQJ1tO4g4Ux1c74qbAjN1yc2aFZPi340waFFv6qWXMRLXRDd0IR~0yIt2Wn3JXzIeHx4ctFiLM6~DVfxcEu9WU5ZQ5mZrBvr4sL2cAq5K39HK~98tZ1XkOg5qWc2cvp0ivxX8lX4grvnqo~OSHP70Zv0d~HN293cPKFxPAu103uTCZuHraUyXNXVtnF60cuOVeaOebd6qCFp-nzkJ3ekmPONHNjhhYRTWvMQPxmP-n4Ly7RhhwJi~Cfteyu6cCLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355683&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The DE lncRNAs within or between high-biomass and low-biomass pools in the <em>P. tomentosa</em> population. (A) Numbers of the DE (FC ≥ 2 or ≤ 0.5, <em>P</em> &lt; 0.01) lncRNAs among the six samples. Category I (red double-headed arrow) represents the comparison between high- and low-biomass samples. Category II (blue double-headed arrow) and Category III (green double-headed arrow) represent the comparisons among tissue types within the high- and low-biomass samples, respectively. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) Sequence motifs of the DE lncRNAs (left panel) and their corresponding GO terms (right panel). The y-axis of the sequence logo represents information contents in bits. <em>E</em>-values for the motifs were calculated by comparison with shuffled sequences. (C) The expression levels of lncRNAs and their potential <em>cis-</em>/<em>trans</em> targets measured by qRT-PCR. Error bars indicate standard deviations (<em>n</em> = 3).</p></div></div><div content-id="dsx018-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f3.png?Expires=2147483647&Signature=lW7Otgfc41mSBddBbXnmdsgsVF6ZiwMfg8z6DhygsCnKK2yPNBD6VwNKf~vEkXkOE3EBNaCSOoECXmHjBB23enKHhp9nz9pqAVqsH0hSABI7h75O0VKSHyFBqDvYZff3vm2jI9uC60iXVAj57BwFsUR-IYVRUZK9e2MU0EpCTIPtco2DANbJlxA14o6VBBXW5D0S0vFug9I0mkKq~fOpXIRUut0J16abSoF6uqGWpHaCUif5Dco0vkEbZsklm0s2Gy7U24NuF5pcRWwrIp-o2YKUc0Y6NBFCs~E6Qwi1rXfBqRbiWOSv-p4jm~h-8s197bNyO-GEnDBPhsg9BTMPDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The DE lncRNAs within or between high-biomass and low-biomass pools in the P. tomentosa population. (A) Numbers of the DE (FC ≥ 2 or ≤ 0.5, P &lt; 0.01) lncRNAs among the six samples. Category I (red double-headed arrow) represents the comparison between high- and low-biomass samples. Category II (blue double-headed arrow) and Category III (green double-headed arrow) represent the comparisons among tissue types within the high- and low-biomass samples, respectively. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) Sequence motifs of the DE lncRNAs (left panel) and their corresponding GO terms (right panel). The y-axis of the sequence logo represents information contents in bits. E-values for the motifs were calculated by comparison with shuffled sequences. (C) The expression levels of lncRNAs and their potential cis-/trans targets measured by qRT-PCR. Error bars indicate standard deviations (n = 3)." path-from-xml="dsx018f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355683/dsx018f3.png" path-from-xml="dsx018f3.png" target="_blank">View large</a><a section="106355683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f3.png?Expires=2147483647&Signature=3gNgsHkJJ80DxQY5jLPJxmsiyoKd97OGWDCVv6T3SMPDUqp1DPYGa8Tw~mDkfBoy1jK70o0fQnQCDct5Kc1fLLIQJ1tO4g4Ux1c74qbAjN1yc2aFZPi340waFFv6qWXMRLXRDd0IR~0yIt2Wn3JXzIeHx4ctFiLM6~DVfxcEu9WU5ZQ5mZrBvr4sL2cAq5K39HK~98tZ1XkOg5qWc2cvp0ivxX8lX4grvnqo~OSHP70Zv0d~HN293cPKFxPAu103uTCZuHraUyXNXVtnF60cuOVeaOebd6qCFp-nzkJ3ekmPONHNjhhYRTWvMQPxmP-n4Ly7RhhwJi~Cfteyu6cCLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355683&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The DE lncRNAs within or between high-biomass and low-biomass pools in the <em>P. tomentosa</em> population. (A) Numbers of the DE (FC ≥ 2 or ≤ 0.5, <em>P</em> &lt; 0.01) lncRNAs among the six samples. Category I (red double-headed arrow) represents the comparison between high- and low-biomass samples. Category II (blue double-headed arrow) and Category III (green double-headed arrow) represent the comparisons among tissue types within the high- and low-biomass samples, respectively. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) Sequence motifs of the DE lncRNAs (left panel) and their corresponding GO terms (right panel). The y-axis of the sequence logo represents information contents in bits. <em>E</em>-values for the motifs were calculated by comparison with shuffled sequences. (C) The expression levels of lncRNAs and their potential <em>cis-</em>/<em>trans</em> targets measured by qRT-PCR. Error bars indicate standard deviations (<em>n</em> = 3).</p></div></div><p>We then analysed the relationship between expression of the lncRNAs and the PTGs for each of the nine comparisons. Among the 8,931 lncRNA–target pairs, 28% pairs showed the same trend in expression and 6% showed the opposite trend in at least one comparison (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials</a></span> S5 and S7). Also, for a specific gene pair, lncRNA and the corresponding PTG showed different trends in different comparisons. The non-coherent expression relationship between lncRNAs and their PTGs (<em>cis</em> and <em>trans</em> targets) were also confirmed by our qRT-PCR results (<a reveal-id="dsx018-F3" data-open="dsx018-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). For example, in the comparison of HM vs LM, three lncRNAs (<em>TCONS_00108374</em>, <em>TCONS_00011246</em>, and <em>TCONS_00185640</em>) showed the same expression trend as their potential <em>cis</em> and <em>trans</em> targets. In addition, <em>TCONS_00011246</em> and its <em>trans</em> target <em>LAC2</em> showed the opposite expression trend in the comparison of HC <em>vs</em> LC, while <em>TCONS_00108374</em> showed the same trend as its target <em>Ptr</em>-<em>IAA27</em>.</p><p>GO analysis of these PTGs found that they were representatively enriched in ‘cellulose biosynthetic process’, ‘response to hormone stimulus’, ‘transcriptional activity’, ‘cell wall modifications’, as well as ‘programmed cell death’ (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s3.png?Expires=2147483647&amp;Signature=DqVFWpPx53p8qtAhjsJXkVKsmPcuZh8~1MB2zt675ZYz3W1ak5KSBx96djdJiAYLUttsOEQlCtiWKznfGzfS6sPF~mikIb1wZRsvqTMuwrHzv0Zn--JLGFv9rcefjJkoVYmBFGoFv3U5amZF1fNHuUmG47Z72xDCnUIMALUjQuBAEu-GqHAuz9mGaZ6wfaOLxDdxiLrGCwjMZnYrAheguL~rZCXPt98HNvAMprjVisIwtMwj2mRxHif82ctQGunN4RWiIwwyKE7eeKZXwFbH3rzZYupruJERpmgFWSSFs7a7j63-VD3eERhy040eUlllVXmSwFG7D-PwkZOs6-HyVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Underlying these GO terms are many developmental regulators of secondary growth, including many transcription factors and genes encoding cell division and expansion proteins, cell wall-associated proteins, and stress-responsive proteins (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S8). Furthermore, we conducted KEGG pathway analysis and revealed that these PTGs were assigned into many pathways concerning secondary metabolite biosynthesis, such as ‘starch and sucrose metabolism’, ‘flavonoid biosynthesis’, and ‘phenylpropanoid biosynthesis’. Among these, 31 PTGs of 26 DE lncRNAs were detected in the phenylpropanoid pathway (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4A;</a><span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s3.doc?Expires=2147483647&amp;Signature=sSIdxdWqmP~1ooiKyMi20myhpvKEiF53M3TfAa8PhnQV2bG7ugBM0UC2x4cmjneC2bOfT1CyiaBpU23Qb0viM4Bp33SCgJIVwWShkKbCaoI1ACeXDK46ey4NQ1IyXCW2d6kaAMbinYBlB1EB2K23C3XY3JGLmoAAhxB8nB3luwpkToj2T7XjVNlbedfWYLkxDDmBMd0luCQroBuOOkmnby2hwnn68SyWLPjfV6HWktQF2~w3DWPxPBQ48CQPZs9Zp-s9DYSJOXxku1I3VHZAb7MbrnBaGoanICgQW7PFqAPQRGJjCBI5V-iAbFW7g9CV5sdI2quBYGtoSMM7qLd5Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). </p>                        <a id="106355686" scrollto-destination="106355686"></a>
<div id="dsx018-F4" data-id="dsx018-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f4.png?Expires=2147483647&amp;Signature=mFk~LdZl5alkpA8p9gk2c0lEUJ9is2RIPRothuZ1tJug7PM~My09XlS16pM-uf-a~TQKCido8qgQM49JoXZa5xnlFTrO3Rr8GaOrODrXmyLdUXkYAKI-1jpb3rOb9qfVu-~phQT4m6mP-QtcFPsyJEPt2VvY57xIsec0Jsr~hPRDcUxx8ihsj-U51j~9uvdixbkzfYmn6vmmMY2PZFikvqZ7pVDOePbS5jSWPM~K-lhjjZAMQ6VZMWGOoGiFGtz0if99sjg7XOavEc0GhcGGEeZAspzojEPehaPodOevge8bIeNOjPgJHw7GjCKFNlXyyidwdEyjRXYsd2KuVzxw4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The lncRNAs enriched in the phenylpropanoid pathway exhibited epistatic interactions with their PTGs. (A) Simplified phenylpropanoid pathway. Pathway information was obtained from http://www.genome.ad.jp/kegg/. Pathway genes from the lignin branch (right) and flavonoid branch (left) are shown in blue and red, respectively. Relative abundance of these transcripts and their corresponding lncRNAs from the six samples are shown by heatmap based on RNA-seq data. The expression values are normalized to log2FPKM. The genes encoding the following enzymes are included: phenylalanine ammonium lyase (PAL), 4-coumarate-CoA ligase (4CL), cinnamoyl-CoA reductase , cinnamyl alcohol dehydrogenase, hydroxycinnamoyl-CoA:quinate shikimate p-hydroxycinnamoyltransferase (HCT), chalcone synthase (CHS), chalcone isomerase, flavonone 3-hydroxylase, dihydroflavonol 4-reductase, and flavonol synthase. Supplementary Table S3 gives detailed information on these genes. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) The epistatic network of lncRNAs and their PTGs enriched in the phenylpropanoid pathway. This network contains 24 lncRNA–mRNA pairs (light blue lines) and 36 mRNA–mRNA pairs (white lines). Also, pairs of lncRNAs and their corresponding targets are connected by light blue, dashed lines. Eight lncRNAs and 15 PTGs are represented as red and blue circles, respectively. (C) Example of the epistatic network. A SNP (SNP498) within Ptr-4CL had epistatic interactions with other SNPs from genes (grey), including an upstream gene (Ptr-PAL) and two downstream genes (Ptr-CHS and Ptr-HCT), and the lncRNA, TCONS_00060904 (red). TCONS_00060904 is 5.23 kb downstream of its potential cis target Ptr-4CL (left panel). The right panel shows SNP507 in TCONS_00060904 and SNP498 within its potential target Ptr-4CL had distinct epistatic effects on HoC and LiC in different genotypic combinations." path-from-xml="dsx018f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355686/dsx018f4.png" path-from-xml="dsx018f4.png" target="_blank">View large</a><a section="106355686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f4.png?Expires=2147483647&amp;Signature=bQ76BlhFZDm-YpifXy41wgsA07MOyQkk-~zzG0G2qFTnpT0yf9ShNBeQ59BAbDV7Fz5GvygP4H9UYP86N9HFSH2H1UDj-NxsPK6OZScYAji5lXD7RcTEfZUio8Jx65P6FtycPhTB4xQdWdAcnFqF-~YELQuHQ8LxNXgoNv~y2aCWMXu3Vtdz51nqgDhCFXkH~-7eBWbiIx1HB-L~~GitDR1REAcrjyKKydSgiAl3S4q918Hp6Nc7WQbQRPFWif2AEUxXdwLmWwIUQhPLssMRjxfxnlZSjDSidg-vEhT4UQ5nLB6RQbwZa9zu-1WHVh5q4Cn03M02qWy2j8BTmTMbfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355686&amp;ar=3769337&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx018f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The lncRNAs enriched in the phenylpropanoid pathway exhibited epistatic interactions with their PTGs. (A) Simplified phenylpropanoid pathway. Pathway information was obtained from <a class="link link-uri" href="http://www.genome.ad.jp/kegg/" target="_blank">http://www.genome.ad.jp/kegg/</a>. Pathway genes from the lignin branch (right) and flavonoid branch (left) are shown in blue and red, respectively. Relative abundance of these transcripts and their corresponding lncRNAs from the six samples are shown by heatmap based on RNA-seq data. The expression values are normalized to log<sub>2</sub>FPKM. The genes encoding the following enzymes are included: phenylalanine ammonium lyase (PAL), 4-coumarate-CoA ligase (4CL), cinnamoyl-CoA reductase , cinnamyl alcohol dehydrogenase, hydroxycinnamoyl-CoA:quinate shikimate p-hydroxycinnamoyltransferase (HCT), chalcone synthase (CHS), chalcone isomerase, flavonone 3-hydroxylase, dihydroflavonol 4-reductase, and flavonol synthase. <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s3.doc?Expires=2147483647&amp;Signature=sSIdxdWqmP~1ooiKyMi20myhpvKEiF53M3TfAa8PhnQV2bG7ugBM0UC2x4cmjneC2bOfT1CyiaBpU23Qb0viM4Bp33SCgJIVwWShkKbCaoI1ACeXDK46ey4NQ1IyXCW2d6kaAMbinYBlB1EB2K23C3XY3JGLmoAAhxB8nB3luwpkToj2T7XjVNlbedfWYLkxDDmBMd0luCQroBuOOkmnby2hwnn68SyWLPjfV6HWktQF2~w3DWPxPBQ48CQPZs9Zp-s9DYSJOXxku1I3VHZAb7MbrnBaGoanICgQW7PFqAPQRGJjCBI5V-iAbFW7g9CV5sdI2quBYGtoSMM7qLd5Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> gives detailed information on these genes. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) The epistatic network of lncRNAs and their PTGs enriched in the phenylpropanoid pathway. This network contains 24 lncRNA–mRNA pairs (light blue lines) and 36 mRNA–mRNA pairs (white lines). Also, pairs of lncRNAs and their corresponding targets are connected by light blue, dashed lines. Eight lncRNAs and 15 PTGs are represented as red and blue circles, respectively. (C) Example of the epistatic network. A SNP (SNP498) within <em>Ptr-4CL</em> had epistatic interactions with other SNPs from genes (grey), including an upstream gene (<em>Ptr-PAL</em>) and two downstream genes (<em>Ptr-CHS</em> and <em>Ptr-HCT</em>), and the lncRNA, TCONS_00060904 (red). TCONS_00060904 is 5.23 kb downstream of its potential <em>cis</em> target <em>Ptr-4CL</em> (left panel). The right panel shows SNP507 in TCONS_00060904 and SNP498 within its potential target <em>Ptr-4CL</em> had distinct epistatic effects on HoC and LiC in different genotypic combinations.</p></div></div><div content-id="dsx018-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f4.png?Expires=2147483647&amp;Signature=mFk~LdZl5alkpA8p9gk2c0lEUJ9is2RIPRothuZ1tJug7PM~My09XlS16pM-uf-a~TQKCido8qgQM49JoXZa5xnlFTrO3Rr8GaOrODrXmyLdUXkYAKI-1jpb3rOb9qfVu-~phQT4m6mP-QtcFPsyJEPt2VvY57xIsec0Jsr~hPRDcUxx8ihsj-U51j~9uvdixbkzfYmn6vmmMY2PZFikvqZ7pVDOePbS5jSWPM~K-lhjjZAMQ6VZMWGOoGiFGtz0if99sjg7XOavEc0GhcGGEeZAspzojEPehaPodOevge8bIeNOjPgJHw7GjCKFNlXyyidwdEyjRXYsd2KuVzxw4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The lncRNAs enriched in the phenylpropanoid pathway exhibited epistatic interactions with their PTGs. (A) Simplified phenylpropanoid pathway. Pathway information was obtained from http://www.genome.ad.jp/kegg/. Pathway genes from the lignin branch (right) and flavonoid branch (left) are shown in blue and red, respectively. Relative abundance of these transcripts and their corresponding lncRNAs from the six samples are shown by heatmap based on RNA-seq data. The expression values are normalized to log2FPKM. The genes encoding the following enzymes are included: phenylalanine ammonium lyase (PAL), 4-coumarate-CoA ligase (4CL), cinnamoyl-CoA reductase , cinnamyl alcohol dehydrogenase, hydroxycinnamoyl-CoA:quinate shikimate p-hydroxycinnamoyltransferase (HCT), chalcone synthase (CHS), chalcone isomerase, flavonone 3-hydroxylase, dihydroflavonol 4-reductase, and flavonol synthase. Supplementary Table S3 gives detailed information on these genes. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) The epistatic network of lncRNAs and their PTGs enriched in the phenylpropanoid pathway. This network contains 24 lncRNA–mRNA pairs (light blue lines) and 36 mRNA–mRNA pairs (white lines). Also, pairs of lncRNAs and their corresponding targets are connected by light blue, dashed lines. Eight lncRNAs and 15 PTGs are represented as red and blue circles, respectively. (C) Example of the epistatic network. A SNP (SNP498) within Ptr-4CL had epistatic interactions with other SNPs from genes (grey), including an upstream gene (Ptr-PAL) and two downstream genes (Ptr-CHS and Ptr-HCT), and the lncRNA, TCONS_00060904 (red). TCONS_00060904 is 5.23 kb downstream of its potential cis target Ptr-4CL (left panel). The right panel shows SNP507 in TCONS_00060904 and SNP498 within its potential target Ptr-4CL had distinct epistatic effects on HoC and LiC in different genotypic combinations." path-from-xml="dsx018f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355686/dsx018f4.png" path-from-xml="dsx018f4.png" target="_blank">View large</a><a section="106355686" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f4.png?Expires=2147483647&amp;Signature=bQ76BlhFZDm-YpifXy41wgsA07MOyQkk-~zzG0G2qFTnpT0yf9ShNBeQ59BAbDV7Fz5GvygP4H9UYP86N9HFSH2H1UDj-NxsPK6OZScYAji5lXD7RcTEfZUio8Jx65P6FtycPhTB4xQdWdAcnFqF-~YELQuHQ8LxNXgoNv~y2aCWMXu3Vtdz51nqgDhCFXkH~-7eBWbiIx1HB-L~~GitDR1REAcrjyKKydSgiAl3S4q918Hp6Nc7WQbQRPFWif2AEUxXdwLmWwIUQhPLssMRjxfxnlZSjDSidg-vEhT4UQ5nLB6RQbwZa9zu-1WHVh5q4Cn03M02qWy2j8BTmTMbfA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355686&amp;ar=3769337&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx018f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The lncRNAs enriched in the phenylpropanoid pathway exhibited epistatic interactions with their PTGs. (A) Simplified phenylpropanoid pathway. Pathway information was obtained from <a class="link link-uri" href="http://www.genome.ad.jp/kegg/" target="_blank">http://www.genome.ad.jp/kegg/</a>. Pathway genes from the lignin branch (right) and flavonoid branch (left) are shown in blue and red, respectively. Relative abundance of these transcripts and their corresponding lncRNAs from the six samples are shown by heatmap based on RNA-seq data. The expression values are normalized to log<sub>2</sub>FPKM. The genes encoding the following enzymes are included: phenylalanine ammonium lyase (PAL), 4-coumarate-CoA ligase (4CL), cinnamoyl-CoA reductase , cinnamyl alcohol dehydrogenase, hydroxycinnamoyl-CoA:quinate shikimate p-hydroxycinnamoyltransferase (HCT), chalcone synthase (CHS), chalcone isomerase, flavonone 3-hydroxylase, dihydroflavonol 4-reductase, and flavonol synthase. <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s3.doc?Expires=2147483647&amp;Signature=sSIdxdWqmP~1ooiKyMi20myhpvKEiF53M3TfAa8PhnQV2bG7ugBM0UC2x4cmjneC2bOfT1CyiaBpU23Qb0viM4Bp33SCgJIVwWShkKbCaoI1ACeXDK46ey4NQ1IyXCW2d6kaAMbinYBlB1EB2K23C3XY3JGLmoAAhxB8nB3luwpkToj2T7XjVNlbedfWYLkxDDmBMd0luCQroBuOOkmnby2hwnn68SyWLPjfV6HWktQF2~w3DWPxPBQ48CQPZs9Zp-s9DYSJOXxku1I3VHZAb7MbrnBaGoanICgQW7PFqAPQRGJjCBI5V-iAbFW7g9CV5sdI2quBYGtoSMM7qLd5Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> gives detailed information on these genes. HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools. (B) The epistatic network of lncRNAs and their PTGs enriched in the phenylpropanoid pathway. This network contains 24 lncRNA–mRNA pairs (light blue lines) and 36 mRNA–mRNA pairs (white lines). Also, pairs of lncRNAs and their corresponding targets are connected by light blue, dashed lines. Eight lncRNAs and 15 PTGs are represented as red and blue circles, respectively. (C) Example of the epistatic network. A SNP (SNP498) within <em>Ptr-4CL</em> had epistatic interactions with other SNPs from genes (grey), including an upstream gene (<em>Ptr-PAL</em>) and two downstream genes (<em>Ptr-CHS</em> and <em>Ptr-HCT</em>), and the lncRNA, TCONS_00060904 (red). TCONS_00060904 is 5.23 kb downstream of its potential <em>cis</em> target <em>Ptr-4CL</em> (left panel). The right panel shows SNP507 in TCONS_00060904 and SNP498 within its potential target <em>Ptr-4CL</em> had distinct epistatic effects on HoC and LiC in different genotypic combinations.</p></div></div>                            <h3 scrollto-destination=106355687 id="106355687" class="section-title" >3.4. LncRNAs may interact with miRNAs as potential targets or target mimics of <em>Populus</em> miRNAs</h3>
<p>Studies have shown that lncRNAs may act as targets or eTMs of miRNAs. In this study, we identified 476 lncRNAs as potential targets of 213 miRNAs from 90 families (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S9) and 28 lncRNAs as eTMs of 14 miRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S10). Interestingly, in the case of seven miRNAs (ptc-miR169r, ptc-miR169y, ptc-miR171g-5p, ptc-miR408-5p, ptc-miR473a-5p, ptc-miR530a, and ptc-miR6459b), each of the miRNAs not only targeted lncRNAs but also was targeted by eTMs of lncRNAs. For example, ptc-miR169y was predicted to target <em>TCONS_00237001</em> (<a reveal-id="dsx018-F5" data-open="dsx018-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) and also have an eTM, <em>TCONS_00013311</em> (<a reveal-id="dsx018-F5" data-open="dsx018-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). Further experimental validation by qRT-PCR revealed a negative relationship between the expression of miRNAs and their targeting lncRNAs (<a reveal-id="dsx018-F5" data-open="dsx018-F5" class="link link-reveal link-table xref-fig">Fig. 5A and B</a>). Based on these results, we reconstructed an interaction network among miRNAs, lncRNAs, and their PTGs (mRNAs) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s4.png?Expires=2147483647&amp;Signature=K0De6IcOXLjhqUUWD4sWIFvK~JkKAa0HN6nVMp7vWUE9fIAcdCoU9hT4AeXOaLBeughr98m1D2josjVS86SO7ldnSkEVayZ~yb~1~2O22fAt3vQz4tIQPtzKlhV1XslOrWTgKk4azWPcJ-hlXy0m0oJAhj4V7wfh46KOYmqeYtPGic4iUZ1NAc8LhP5HLVnT4o8drGwxFwYzbK3sfWl3qdD~L13-Qw5n8761KPVNd28F8HHYO~wzd-IW0Th3IVW3v9XmCkPtl2Ln0ZG6GausNbOXLKjfN5bdcF1j7p82ujpyksvcrpzNXKhE407DulktJ6YIgAPIQK7mLGnVkgZuWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). However, all the results are predicted preliminarily based on bioinformatic analyses and need to be further validated. </p>                        <a id="106355689" scrollto-destination="106355689"></a>
<div id="dsx018-F5" data-id="dsx018-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f5.png?Expires=2147483647&Signature=S9go6p4t9UHksRovAC26OvYZCtBqGC0bYB~vMCeTHgcTFup6902kN1-D7i4h4njmMWYlo7avDcluPZwK3Eflmhq2m5DQ1KobbRiCIW0fZGWtPLbu0YgQr9tHDDkWlZ~DxS1ihw9i3A7y-6ck88O0x85dOPnVDqIw7oxnTrNUEx5DPzea86BEI~WyJrk1OeuefexFOiZ8umV1JFTpIkw-svP1wWSg61EabelGDkm76YV6zNsq1H-AeoYir-GVAsEr1o0ZG2COXyVd-7Sne4IqNcsozHHCSFYBrhemcBTmFCTUpPV~R4W4kqwif3kmnLJmEiSaApTe-Afki~2BN3Kfdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression analysis of Populus lncRNAs and miRNAs by qRT-PCR. (A) Expression of lncRNAs as potential targets of miRNAs. (B) Expression of lncRNAs as target mimics of miRNAs. Error bars indicate standard deviations (n = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355689/dsx018f5.png" path-from-xml="dsx018f5.png" target="_blank">View large</a><a section="106355689" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f5.png?Expires=2147483647&Signature=YwfMQMYYKAVsWGIAxE9NbC2mpCcAi9kIORpwQbSogBY852fEhWSO3Yut~Qut5hiYITk5OC1Bpblq3UKdNfIiDeqm4pWwPKocCJK4q395nE90S9rl0hjDDjqnxKGn7Cg1jJiTUdHSFhSWZEeD9vPHPOcsYr8o9RUzREeOpzFiCWjlGWFZ4woTRuLesdtNgobqwvHUrCQtBxCvaq~ePKn6iRECaUqLaSOaqI0piZ74BQaQBuIlyjyAvs-szRu~ayZ6mvmoq8V0z50ulmlFzYG1SZCmyWPWM5slkG5lkwCi5XpqPHqsX8hmQRRS3q3PpXtHGdmorgGqZou6RmiWT5QJuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355689&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression analysis of <em>Populus</em> lncRNAs and miRNAs by qRT-PCR. (A) Expression of lncRNAs as potential targets of miRNAs. (B) Expression of lncRNAs as target mimics of miRNAs. Error bars indicate standard deviations (<em>n</em> = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div><div content-id="dsx018-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f5.png?Expires=2147483647&Signature=S9go6p4t9UHksRovAC26OvYZCtBqGC0bYB~vMCeTHgcTFup6902kN1-D7i4h4njmMWYlo7avDcluPZwK3Eflmhq2m5DQ1KobbRiCIW0fZGWtPLbu0YgQr9tHDDkWlZ~DxS1ihw9i3A7y-6ck88O0x85dOPnVDqIw7oxnTrNUEx5DPzea86BEI~WyJrk1OeuefexFOiZ8umV1JFTpIkw-svP1wWSg61EabelGDkm76YV6zNsq1H-AeoYir-GVAsEr1o0ZG2COXyVd-7Sne4IqNcsozHHCSFYBrhemcBTmFCTUpPV~R4W4kqwif3kmnLJmEiSaApTe-Afki~2BN3Kfdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression analysis of Populus lncRNAs and miRNAs by qRT-PCR. (A) Expression of lncRNAs as potential targets of miRNAs. (B) Expression of lncRNAs as target mimics of miRNAs. Error bars indicate standard deviations (n = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools." path-from-xml="dsx018f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355689/dsx018f5.png" path-from-xml="dsx018f5.png" target="_blank">View large</a><a section="106355689" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f5.png?Expires=2147483647&Signature=YwfMQMYYKAVsWGIAxE9NbC2mpCcAi9kIORpwQbSogBY852fEhWSO3Yut~Qut5hiYITk5OC1Bpblq3UKdNfIiDeqm4pWwPKocCJK4q395nE90S9rl0hjDDjqnxKGn7Cg1jJiTUdHSFhSWZEeD9vPHPOcsYr8o9RUzREeOpzFiCWjlGWFZ4woTRuLesdtNgobqwvHUrCQtBxCvaq~ePKn6iRECaUqLaSOaqI0piZ74BQaQBuIlyjyAvs-szRu~ayZ6mvmoq8V0z50ulmlFzYG1SZCmyWPWM5slkG5lkwCi5XpqPHqsX8hmQRRS3q3PpXtHGdmorgGqZou6RmiWT5QJuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355689&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression analysis of <em>Populus</em> lncRNAs and miRNAs by qRT-PCR. (A) Expression of lncRNAs as potential targets of miRNAs. (B) Expression of lncRNAs as target mimics of miRNAs. Error bars indicate standard deviations (<em>n</em> = 3). HC, vascular cambium from high-biomass pools; LC, vascular cambium from low-biomass pools; HD, developing xylem from high-biomass pools; LD, developing xylem from low-biomass pools; HM, mature xylem from high-biomass pools; LM, mature xylem from low-biomass pools.</p></div></div>                            <h3 scrollto-destination=106355690 id="106355690" class="section-title" >3.5. Genotyping and LD tests for SNPs within lncRNA genes and PTGs in the <em>P. tomentosa</em> association population</h3>
<p>We selected the most abundant (exp<sub>max</sub> &gt; 50 FPKM) lncRNAs genes (see above) and their PTGs, which may play important roles in developmental processes. Among these, 22 PTGs of 16 lncRNAs were enriched in the phenylpropanoid pathway (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Based on the genome re-sequencing data of 435 <em>P. tomentosa</em> individuals from a natural population, we obtained 87,347 SNPs from high-quality genomic sequences of 178 lncRNA genes and 522 PTGs, after SNPs with missing data (&gt;10%) were excluded. The SNPs were not evenly distributed among genes, ranging from 1 to 556, with an average density of 32 SNPs per gene (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S11). SNPs in lncRNA genes had lower diversity (π<sub>T</sub> = 0.01804 and θ<sub>w</sub> = 0.0454) than SNPs in PTGs (π<sub>T</sub> = 0.03422 and θ<sub>w</sub> = 0.06733). After eliminating SNPs with missing data &gt; 25% or MAFs &lt; 0.05 in the natural population, we obtained 34,015 common SNPs, including 6,590 SNPs in 178 lncRNA genes and 32,428 SNPs in 522 PTGs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S12). To perform association analysis for these markers, we first conducted LD tests. The nonlinear regression showed a rapid decay of LD at the chromosome level within an average distance of 2,443 bp, where <em>r</em><sup>2</sup> values declined to 0.1 (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s5.png?Expires=2147483647&amp;Signature=AXD4FXRhrF-Q0qtF9uZQpadhFN63G6ym12aPf4SkXhCP6Nh9AXeZAVMKRZC85TJDHuVCFpxczaAQZra~m-MTZpHEgCFg3jpeTZaBuT0lgvD559qEPDV8vGn5WfKPRQrPkrm3e-duk7Mr~3ppymNwjidyTQ~QmF2BVxwBrNpSmiGtIhg7CKuYk~JcW4Q7CvUczI-vLmJk1ftEv9kS5nUbj1qoVDyXKGF0fjkoLlCI9NQveVQ4FNnZ0zRjpdBSs37qyDtEVkhVw3f2DDoPkYbIJC2REGdEi0meK0zveHSLSzwENB-sM1qrqsHGZinSDN1g02Du2zDWN20d392w2nOhxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                            <h3 scrollto-destination=106355692 id="106355692" class="section-title" >3.6. Single-SNP association testing for additive and dominant effects</h3>
<p>We used association analysis to examine the linkages between phenotypes and SNPs in the lncRNA genes and PTGs. We first conducted SNP-trait association tests between the 34,015 common SNPs and 10 tree growth and wood property traits by using a MLM in TASSEL v5.0. This detected 2,993 significant (<em>P</em> &lt; 0.001) associations, representing 1,163 unique SNPs in 83 lncRNA genes and 328 PTGs. The total numbers of significant associations varied across the 10 traits, and the phenotypic variance explained by individual SNPs ranged from 1.2 to 29.3% (<a reveal-id="dsx018-T1" data-open="dsx018-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Among the 1,163 trait-related markers, 329 SNPs from 19 lncRNA genes and 98 PTGs were simultaneously associated with two to three traits (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S13). Interestingly, similar association patterns were identified for SNPs within the same gene. For example, we found that 11 SNPs from <em>Ptr</em>-<em>PAL</em>, which functions in phenylpropanoid biosynthesis, were associated with four traits (DBH, HoC, MFA, and V). Two of the SNPs (SNP1108 and SNP1109) simultaneously associated with three common traits (DBH, MFA, and V). Most (∼77%) trait-associated lncRNAs shared at least one common trait with their corresponding PTGs (<a reveal-id="dsx018-F6" data-open="dsx018-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). <div content-id="dsx018-T1" class="table-modal"><div class="table-wrap"><div id="dsx018-T1" data-id="dsx018-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of significant (<em>P</em> &lt; 0.001) associations with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by TASSEL v5.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">Numbers of associations<hr /> </th><th rowspan="2"><em>R</em><sup>2</sup> (%)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr><tr><th>SNPs </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td>αC (%) </td><td>35 </td><td>6 </td><td>31 </td><td>14.15–26.86 </td></tr><tr><td>DBH (cm) </td><td>666 </td><td>48 </td><td>189 </td><td>14.15–29.06 </td></tr><tr><td>FW (µm) </td><td>61 </td><td>13 </td><td>53 </td><td>14.05–-29.02 </td></tr><tr><td>FL (mm) </td><td>10 </td><td>0 </td><td>11 </td><td>14.89–23.87 </td></tr><tr><td>HemC (%) </td><td>29 </td><td>1 </td><td>26 </td><td>14.03–24.90 </td></tr><tr><td>HoC (%) </td><td>78 </td><td>6 </td><td>49 </td><td>1.23–29.25 </td></tr><tr><td>LiC (%) </td><td>10 </td><td>1 </td><td>8 </td><td>14.04–22.01 </td></tr><tr><td>MFA (degree) </td><td>128 </td><td>23 </td><td>97 </td><td>17.13–27.26 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>473 </td><td>33 </td><td>155 </td><td>14.23–28.98 </td></tr><tr><td>H (m) </td><td>6 </td><td>3 </td><td>4 </td><td>14.01–23.84 </td></tr><tr><td>&lt;Total&gt; </td><td>1,163 </td><td>83 </td><td>328 </td><td>1.23–29.25 </td></tr></tbody></table></div><div content-id="dsx018-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">Numbers of associations<hr /> </th><th rowspan="2"><em>R</em><sup>2</sup> (%)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr><tr><th>SNPs </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td>αC (%) </td><td>35 </td><td>6 </td><td>31 </td><td>14.15–26.86 </td></tr><tr><td>DBH (cm) </td><td>666 </td><td>48 </td><td>189 </td><td>14.15–29.06 </td></tr><tr><td>FW (µm) </td><td>61 </td><td>13 </td><td>53 </td><td>14.05–-29.02 </td></tr><tr><td>FL (mm) </td><td>10 </td><td>0 </td><td>11 </td><td>14.89–23.87 </td></tr><tr><td>HemC (%) </td><td>29 </td><td>1 </td><td>26 </td><td>14.03–24.90 </td></tr><tr><td>HoC (%) </td><td>78 </td><td>6 </td><td>49 </td><td>1.23–29.25 </td></tr><tr><td>LiC (%) </td><td>10 </td><td>1 </td><td>8 </td><td>14.04–22.01 </td></tr><tr><td>MFA (degree) </td><td>128 </td><td>23 </td><td>97 </td><td>17.13–27.26 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>473 </td><td>33 </td><td>155 </td><td>14.23–28.98 </td></tr><tr><td>H (m) </td><td>6 </td><td>3 </td><td>4 </td><td>14.01–23.84 </td></tr><tr><td>&lt;Total&gt; </td><td>1,163 </td><td>83 </td><td>328 </td><td>1.23–29.25 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Percentage of the phenotypic variance explained.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx018-T1" data-id="dsx018-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of significant (<em>P</em> &lt; 0.001) associations with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by TASSEL v5.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">Numbers of associations<hr /> </th><th rowspan="2"><em>R</em><sup>2</sup> (%)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr><tr><th>SNPs </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td>αC (%) </td><td>35 </td><td>6 </td><td>31 </td><td>14.15–26.86 </td></tr><tr><td>DBH (cm) </td><td>666 </td><td>48 </td><td>189 </td><td>14.15–29.06 </td></tr><tr><td>FW (µm) </td><td>61 </td><td>13 </td><td>53 </td><td>14.05–-29.02 </td></tr><tr><td>FL (mm) </td><td>10 </td><td>0 </td><td>11 </td><td>14.89–23.87 </td></tr><tr><td>HemC (%) </td><td>29 </td><td>1 </td><td>26 </td><td>14.03–24.90 </td></tr><tr><td>HoC (%) </td><td>78 </td><td>6 </td><td>49 </td><td>1.23–29.25 </td></tr><tr><td>LiC (%) </td><td>10 </td><td>1 </td><td>8 </td><td>14.04–22.01 </td></tr><tr><td>MFA (degree) </td><td>128 </td><td>23 </td><td>97 </td><td>17.13–27.26 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>473 </td><td>33 </td><td>155 </td><td>14.23–28.98 </td></tr><tr><td>H (m) </td><td>6 </td><td>3 </td><td>4 </td><td>14.01–23.84 </td></tr><tr><td>&lt;Total&gt; </td><td>1,163 </td><td>83 </td><td>328 </td><td>1.23–29.25 </td></tr></tbody></table></div><div content-id="dsx018-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">Numbers of associations<hr /> </th><th rowspan="2"><em>R</em><sup>2</sup> (%)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr><tr><th>SNPs </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td>αC (%) </td><td>35 </td><td>6 </td><td>31 </td><td>14.15–26.86 </td></tr><tr><td>DBH (cm) </td><td>666 </td><td>48 </td><td>189 </td><td>14.15–29.06 </td></tr><tr><td>FW (µm) </td><td>61 </td><td>13 </td><td>53 </td><td>14.05–-29.02 </td></tr><tr><td>FL (mm) </td><td>10 </td><td>0 </td><td>11 </td><td>14.89–23.87 </td></tr><tr><td>HemC (%) </td><td>29 </td><td>1 </td><td>26 </td><td>14.03–24.90 </td></tr><tr><td>HoC (%) </td><td>78 </td><td>6 </td><td>49 </td><td>1.23–29.25 </td></tr><tr><td>LiC (%) </td><td>10 </td><td>1 </td><td>8 </td><td>14.04–22.01 </td></tr><tr><td>MFA (degree) </td><td>128 </td><td>23 </td><td>97 </td><td>17.13–27.26 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>473 </td><td>33 </td><td>155 </td><td>14.23–28.98 </td></tr><tr><td>H (m) </td><td>6 </td><td>3 </td><td>4 </td><td>14.01–23.84 </td></tr><tr><td>&lt;Total&gt; </td><td>1,163 </td><td>83 </td><td>328 </td><td>1.23–29.25 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Percentage of the phenotypic variance explained.</p></div></div></div></div></p>                        <a id="106355694" scrollto-destination="106355694"></a>
<div id="dsx018-F6" data-id="dsx018-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f6.png?Expires=2147483647&Signature=2iK3UKBzwHpwaNX6DoB0pVPoVqcIZz~dEGcMdMPKK4YuotXgZnwdD0IsL48DjvUF~NKB-SSN8pulmIRWc1kl7y4cLzJ9gqLRBX3X37IGdRBjmJLwKHJX8SB64oKbf~RJLfFoLsmx7O-3oD69~mxxZSMs05CBV~6A4CZwqAW6eVXCcC1ZvEwHQFzHGVICcKcBzoS61FS8K57ve65ZQ97vUuYrQr8FCE-gNyiEo6dCHbSIiTTFA02VdpT2dH2w6nNNbzd~0PAiL1rcqruJ1k-AMnNEtXpeNugOuXtFUF-Wn~xwmPzBkqV8mXozKd5nZmc9TJwh1-rxwQy~cki9v65rvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dissection of the complex genotype effects (additive/dominant) for the lncRNA (TCONS_00059927) and its potential cis-acting target genes. Five significant SNPs (SNP_22141, SNP_22266, SNP_22452, SNP_22534, and SNP_22868) within the lncRNA and its PTGs, Potri.003G095200 (upstream) and Potri.003G095400 (downstream) have additive/dominant effects on HoC and stem volume. The two target genes are 1.56 and 6.15 kb from the lncRNA, respectively." path-from-xml="dsx018f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355694/dsx018f6.png" path-from-xml="dsx018f6.png" target="_blank">View large</a><a section="106355694" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f6.png?Expires=2147483647&Signature=4wz6jsXt7Rs2noHXGg7E5HxCJbHqOdQZwIfd-zuWDt~NaqLpo7svdT3ounbk1rzm3cKPRQ4d9VB5EC5RVu-iUx0bfCOyxJqPd0k4yV5YFmCahvIo-xZoAWGOwNEplXp7gIlNzNv2vTVSvbADYpfWnXjpFlJu3RKlFPI-T5HpSp3rxStdfzPdbA1t9UENmi2FDXCOkicPTw3sJ9c5mdbh0pgfT69nDXvWQsvotgIW0jMUhs1bJGXZttMN27VorfPuj1cUrO4uoXfKzXsld9aXLpKWet86nJTQwuCYsD1ySd7zI3x1CYp9MOpPzLBHa7aEu~Kf4UJVxFvXX0zdlh8ryQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355694&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dissection of the complex genotype effects (additive/dominant) for the lncRNA (TCONS_00059927) and its potential <em>cis</em>-acting target genes. Five significant SNPs (SNP_22141, SNP_22266, SNP_22452, SNP_22534, and SNP_22868) within the lncRNA and its PTGs, Potri.003G095200 (upstream) and Potri.003G095400 (downstream) have additive/dominant effects on HoC and stem volume. The two target genes are 1.56 and 6.15 kb from the lncRNA, respectively.</p></div></div><div content-id="dsx018-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/m_dsx018f6.png?Expires=2147483647&Signature=2iK3UKBzwHpwaNX6DoB0pVPoVqcIZz~dEGcMdMPKK4YuotXgZnwdD0IsL48DjvUF~NKB-SSN8pulmIRWc1kl7y4cLzJ9gqLRBX3X37IGdRBjmJLwKHJX8SB64oKbf~RJLfFoLsmx7O-3oD69~mxxZSMs05CBV~6A4CZwqAW6eVXCcC1ZvEwHQFzHGVICcKcBzoS61FS8K57ve65ZQ97vUuYrQr8FCE-gNyiEo6dCHbSIiTTFA02VdpT2dH2w6nNNbzd~0PAiL1rcqruJ1k-AMnNEtXpeNugOuXtFUF-Wn~xwmPzBkqV8mXozKd5nZmc9TJwh1-rxwQy~cki9v65rvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dissection of the complex genotype effects (additive/dominant) for the lncRNA (TCONS_00059927) and its potential cis-acting target genes. Five significant SNPs (SNP_22141, SNP_22266, SNP_22452, SNP_22534, and SNP_22868) within the lncRNA and its PTGs, Potri.003G095200 (upstream) and Potri.003G095400 (downstream) have additive/dominant effects on HoC and stem volume. The two target genes are 1.56 and 6.15 kb from the lncRNA, respectively." path-from-xml="dsx018f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355694/dsx018f6.png" path-from-xml="dsx018f6.png" target="_blank">View large</a><a section="106355694" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018f6.png?Expires=2147483647&Signature=4wz6jsXt7Rs2noHXGg7E5HxCJbHqOdQZwIfd-zuWDt~NaqLpo7svdT3ounbk1rzm3cKPRQ4d9VB5EC5RVu-iUx0bfCOyxJqPd0k4yV5YFmCahvIo-xZoAWGOwNEplXp7gIlNzNv2vTVSvbADYpfWnXjpFlJu3RKlFPI-T5HpSp3rxStdfzPdbA1t9UENmi2FDXCOkicPTw3sJ9c5mdbh0pgfT69nDXvWQsvotgIW0jMUhs1bJGXZttMN27VorfPuj1cUrO4uoXfKzXsld9aXLpKWet86nJTQwuCYsD1ySd7zI3x1CYp9MOpPzLBHa7aEu~Kf4UJVxFvXX0zdlh8ryQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355694&ar=3769337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx018f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dissection of the complex genotype effects (additive/dominant) for the lncRNA (TCONS_00059927) and its potential <em>cis</em>-acting target genes. Five significant SNPs (SNP_22141, SNP_22266, SNP_22452, SNP_22534, and SNP_22868) within the lncRNA and its PTGs, Potri.003G095200 (upstream) and Potri.003G095400 (downstream) have additive/dominant effects on HoC and stem volume. The two target genes are 1.56 and 6.15 kb from the lncRNA, respectively.</p></div></div><p>For additive effects, we detected 455 significant associations representing 410 unique SNPs from 249 genes (38 lncRNAs and 211 PTGs) across all 10 traits (<a reveal-id="dsx018-T2" data-open="dsx018-T2" class="link link-reveal link-table xref-fig">Table 2</a>; <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S14). We found that 45 of the 410 SNPs were associated with two to three traits, indicating a remarkable pleiotropic effect of these markers. Correspondingly, for each trait, 1–102 genes exhibited additive effects on tree growth and wood properties. For dominant effects, we detected 266 significant associations representing 249 unique SNPs from 187 genes (33 lncRNA genes and 154 PTGs) (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S14). Of these associations, 151 showed a positive dominance effect and 115 had a negative value. For these 249 SNPs, each was associated with one to three traits with positive or negative dominant effects. Correspondingly, around 21% (40 of 187) of the genes associated with more than one trait. Generally, for a specific trait, most of the SNPs from the same gene exhibited concordant dominance. For example, one SNP (SNP_57423) within <em>TCONS_00141756</em> and five SNPs from the PTG <em>Potri.008G056000</em> were commonly linked to LiC with positive dominance. We also found a small portion of SNPs showing discordant dominance. For instance, two unique SNPs (SNP_4287 and SNP_4431) from the same gene (<em>Potri.001G030000</em>) had opposite dominant effects on MFA, suggesting the complex allelic combinations underlying stem growth and wood biosynthesis. Overall, we detected 600 associations with significant (<em>P</em> &lt; 0.001) additive or/and dominant effects for the 10 traits, representing 533 unique SNPs from 52 lncRNA genes and 254 PTGs (<a reveal-id="dsx018-T2" data-open="dsx018-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Of these associations, around 20% (121 of 600) have a combination of additive and dominant effects for a certain trait. <div content-id="dsx018-T2" class="table-modal"><div class="table-wrap"><div id="dsx018-T2" data-id="dsx018-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The additive and dominant effects of significant (<em>P</em> &lt; 0.001) SNPs associated with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by TASSEL v5.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="3">Trait </th><th colspan="3">Additive effect<hr /> </th><th colspan="3">Dominant effect<hr /> </th></tr><tr><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th></tr><tr><th>LncRNAs </th><th>Targets </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td rowspan="3">Tree growth </td><td>DBH (cm) </td><td>9 </td><td>62 </td><td>2.38–8.47 </td><td>5 </td><td>19 </td><td>−10.91–11.72 </td></tr><tr><td>H (m) </td><td>1 </td><td>0 </td><td>1.98 </td><td>1 </td><td>5 </td><td>−3.42 to − 2.03 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>25 </td><td>133 </td><td>0.16–0.95 </td><td>12 </td><td>67 </td><td>−1.06 to 1.55 </td></tr><tr><td rowspan="8">Wood properties </td><td>αC (%) </td><td>1 </td><td>25 </td><td>5.01–14.36 </td><td>2 </td><td>23 </td><td>–23.65 to -19.64 </td></tr><tr><td>FL (mm) </td><td>1 </td><td>10 </td><td>0.06–0.16 </td><td>1 </td><td>7 </td><td>−0.34 to 0.18 </td></tr><tr><td>FW (µm) </td><td>4 </td><td>15 </td><td>1.40–4.12 </td><td>2 </td><td>10 </td><td>1.60–5.30 </td></tr><tr><td>HemC (%) </td><td>2 </td><td>26 </td><td>6.81–20.71 </td><td>2 </td><td>14 </td><td>−17.24 to 15.52 </td></tr><tr><td>HoC (%) </td><td>8 </td><td>49 </td><td>5.62–21.00 </td><td>6 </td><td>27 </td><td>−32.75 to 20.77 </td></tr><tr><td>LiC (%) </td><td>1 </td><td>6 </td><td>1.67–5.52 </td><td>2 </td><td>3 </td><td>−4.23 to 3.85 </td></tr><tr><td>MFA (degree) </td><td>24 </td><td>105 </td><td>3.55–12.77 </td><td>14 </td><td>80 </td><td>−17.88 to 10.67 </td></tr><tr><td>&lt;Total&gt; </td><td>69 </td><td>390 </td><td>0.06–21.00 </td><td>44 </td><td>239 </td><td>−32.75 to 20.77 </td></tr></tbody></table></div><div content-id="dsx018-T2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="3">Trait </th><th colspan="3">Additive effect<hr /> </th><th colspan="3">Dominant effect<hr /> </th></tr><tr><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th></tr><tr><th>LncRNAs </th><th>Targets </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td rowspan="3">Tree growth </td><td>DBH (cm) </td><td>9 </td><td>62 </td><td>2.38–8.47 </td><td>5 </td><td>19 </td><td>−10.91–11.72 </td></tr><tr><td>H (m) </td><td>1 </td><td>0 </td><td>1.98 </td><td>1 </td><td>5 </td><td>−3.42 to − 2.03 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>25 </td><td>133 </td><td>0.16–0.95 </td><td>12 </td><td>67 </td><td>−1.06 to 1.55 </td></tr><tr><td rowspan="8">Wood properties </td><td>αC (%) </td><td>1 </td><td>25 </td><td>5.01–14.36 </td><td>2 </td><td>23 </td><td>–23.65 to -19.64 </td></tr><tr><td>FL (mm) </td><td>1 </td><td>10 </td><td>0.06–0.16 </td><td>1 </td><td>7 </td><td>−0.34 to 0.18 </td></tr><tr><td>FW (µm) </td><td>4 </td><td>15 </td><td>1.40–4.12 </td><td>2 </td><td>10 </td><td>1.60–5.30 </td></tr><tr><td>HemC (%) </td><td>2 </td><td>26 </td><td>6.81–20.71 </td><td>2 </td><td>14 </td><td>−17.24 to 15.52 </td></tr><tr><td>HoC (%) </td><td>8 </td><td>49 </td><td>5.62–21.00 </td><td>6 </td><td>27 </td><td>−32.75 to 20.77 </td></tr><tr><td>LiC (%) </td><td>1 </td><td>6 </td><td>1.67–5.52 </td><td>2 </td><td>3 </td><td>−4.23 to 3.85 </td></tr><tr><td>MFA (degree) </td><td>24 </td><td>105 </td><td>3.55–12.77 </td><td>14 </td><td>80 </td><td>−17.88 to 10.67 </td></tr><tr><td>&lt;Total&gt; </td><td>69 </td><td>390 </td><td>0.06–21.00 </td><td>44 </td><td>239 </td><td>−32.75 to 20.77 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx018-T2" data-id="dsx018-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>The additive and dominant effects of significant (<em>P</em> &lt; 0.001) SNPs associated with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by TASSEL v5.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="3">Trait </th><th colspan="3">Additive effect<hr /> </th><th colspan="3">Dominant effect<hr /> </th></tr><tr><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th></tr><tr><th>LncRNAs </th><th>Targets </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td rowspan="3">Tree growth </td><td>DBH (cm) </td><td>9 </td><td>62 </td><td>2.38–8.47 </td><td>5 </td><td>19 </td><td>−10.91–11.72 </td></tr><tr><td>H (m) </td><td>1 </td><td>0 </td><td>1.98 </td><td>1 </td><td>5 </td><td>−3.42 to − 2.03 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>25 </td><td>133 </td><td>0.16–0.95 </td><td>12 </td><td>67 </td><td>−1.06 to 1.55 </td></tr><tr><td rowspan="8">Wood properties </td><td>αC (%) </td><td>1 </td><td>25 </td><td>5.01–14.36 </td><td>2 </td><td>23 </td><td>–23.65 to -19.64 </td></tr><tr><td>FL (mm) </td><td>1 </td><td>10 </td><td>0.06–0.16 </td><td>1 </td><td>7 </td><td>−0.34 to 0.18 </td></tr><tr><td>FW (µm) </td><td>4 </td><td>15 </td><td>1.40–4.12 </td><td>2 </td><td>10 </td><td>1.60–5.30 </td></tr><tr><td>HemC (%) </td><td>2 </td><td>26 </td><td>6.81–20.71 </td><td>2 </td><td>14 </td><td>−17.24 to 15.52 </td></tr><tr><td>HoC (%) </td><td>8 </td><td>49 </td><td>5.62–21.00 </td><td>6 </td><td>27 </td><td>−32.75 to 20.77 </td></tr><tr><td>LiC (%) </td><td>1 </td><td>6 </td><td>1.67–5.52 </td><td>2 </td><td>3 </td><td>−4.23 to 3.85 </td></tr><tr><td>MFA (degree) </td><td>24 </td><td>105 </td><td>3.55–12.77 </td><td>14 </td><td>80 </td><td>−17.88 to 10.67 </td></tr><tr><td>&lt;Total&gt; </td><td>69 </td><td>390 </td><td>0.06–21.00 </td><td>44 </td><td>239 </td><td>−32.75 to 20.77 </td></tr></tbody></table></div><div content-id="dsx018-T2" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2" rowspan="3">Trait </th><th colspan="3">Additive effect<hr /> </th><th colspan="3">Dominant effect<hr /> </th></tr><tr><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th><th colspan="2">Numbers of SNPs<hr /> </th><th rowspan="2">Range of effect </th></tr><tr><th>LncRNAs </th><th>Targets </th><th>LncRNAs </th><th>Targets </th></tr></thead><tbody><tr><td rowspan="3">Tree growth </td><td>DBH (cm) </td><td>9 </td><td>62 </td><td>2.38–8.47 </td><td>5 </td><td>19 </td><td>−10.91–11.72 </td></tr><tr><td>H (m) </td><td>1 </td><td>0 </td><td>1.98 </td><td>1 </td><td>5 </td><td>−3.42 to − 2.03 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>25 </td><td>133 </td><td>0.16–0.95 </td><td>12 </td><td>67 </td><td>−1.06 to 1.55 </td></tr><tr><td rowspan="8">Wood properties </td><td>αC (%) </td><td>1 </td><td>25 </td><td>5.01–14.36 </td><td>2 </td><td>23 </td><td>–23.65 to -19.64 </td></tr><tr><td>FL (mm) </td><td>1 </td><td>10 </td><td>0.06–0.16 </td><td>1 </td><td>7 </td><td>−0.34 to 0.18 </td></tr><tr><td>FW (µm) </td><td>4 </td><td>15 </td><td>1.40–4.12 </td><td>2 </td><td>10 </td><td>1.60–5.30 </td></tr><tr><td>HemC (%) </td><td>2 </td><td>26 </td><td>6.81–20.71 </td><td>2 </td><td>14 </td><td>−17.24 to 15.52 </td></tr><tr><td>HoC (%) </td><td>8 </td><td>49 </td><td>5.62–21.00 </td><td>6 </td><td>27 </td><td>−32.75 to 20.77 </td></tr><tr><td>LiC (%) </td><td>1 </td><td>6 </td><td>1.67–5.52 </td><td>2 </td><td>3 </td><td>−4.23 to 3.85 </td></tr><tr><td>MFA (degree) </td><td>24 </td><td>105 </td><td>3.55–12.77 </td><td>14 </td><td>80 </td><td>−17.88 to 10.67 </td></tr><tr><td>&lt;Total&gt; </td><td>69 </td><td>390 </td><td>0.06–21.00 </td><td>44 </td><td>239 </td><td>−32.75 to 20.77 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=106355696 id="106355696" class="section-title" >3.7. Epistasis analysis of epistatic effects</h3>
<p>We next examined epistatic effects of the SNPs to further probe the complexity of this regulatory network. We used EPISNP v2.0 to analye the pair-wise interactive effects for 34,015 SNPs from lncRNA genes and PTGs across the 10 traits. This detected 17,656 significant (<em>P</em> &lt; 0.001) epistatic pairs for all 10 traits, representing 4,334 SNPs from 111 lncRNA genes and 448 PTGs (<a reveal-id="dsx018-T3" data-open="dsx018-T3" class="link link-reveal link-table xref-fig">Table 3</a>; <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S15). These 4,334 interactive SNPs, representing a coverage level of 8.6% of the tested markers, did not display detectable additive/dominant effects. These 17,656 pair-wise epistatic interactions could be further interpreted into 3,918 AA, 8,866 AD or DA, and 5,132 DD effects for the traits (<a reveal-id="dsx018-T3" data-open="dsx018-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The contribution rate of the significant epistatic SNP pairs ranged from 0.01% to 13.63% (<a reveal-id="dsx018-T3" data-open="dsx018-T3" class="link link-reveal link-table xref-fig">Table 3</a>), which was lower than that of single-locus associations, with only 76 epistatic pairs having a contribution rate &gt;1% (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S15). <div content-id="dsx018-T3" class="table-modal"><div class="table-wrap"><div id="dsx018-T3" data-id="dsx018-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>The epistatic effect of significant (<em>P</em> &lt; 0.001) SNPs associated with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by EPISNP v2.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">No. of epistatic SNP–SNP pairs<sup>a</sup><hr /> </th><th rowspan="2">No. of interacting SNPs </th><th rowspan="2">No. of epistatic gene–gene pairs </th><th colspan="2">No. of interactive genes<hr /> </th><th rowspan="2">Range of contribution rate (<em>c</em>, %) </th></tr><tr><th>AA </th><th>AD (DA) </th><th>DD </th><th>lncRNA </th><th>target </th></tr></thead><tbody><tr><td>DBH (cm) </td><td>404 </td><td>473 </td><td>194 </td><td>897 </td><td>790 </td><td>89 </td><td>249 </td><td>0.03–0.83 </td></tr><tr><td>H (m) </td><td>107 </td><td>240 </td><td>104 </td><td>548 </td><td>418 </td><td>71 </td><td>176 </td><td>0.03–1.00 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>1,230 </td><td>2,217 </td><td>1,380 </td><td>1,670 </td><td>3,221 </td><td>97 </td><td>359 </td><td>0.03–12.95 </td></tr><tr><td>αC (%) </td><td>448 </td><td>751 </td><td>455 </td><td>1,329 </td><td>1,340 </td><td>89 </td><td>302 </td><td>0.01–0.91 </td></tr><tr><td>FL (mm) </td><td>208 </td><td>505 </td><td>324 </td><td>860 </td><td>806 </td><td>72 </td><td>257 </td><td>0.01–13.63 </td></tr><tr><td>FW (µm) </td><td>242 </td><td>1,025 </td><td>657 </td><td>1,309 </td><td>1,374 </td><td>88 </td><td>307 </td><td>0.01–0.67 </td></tr><tr><td>HemC (%) </td><td>359 </td><td>998 </td><td>487 </td><td>1,375 </td><td>1,461 </td><td>91 </td><td>299 </td><td>0.02–3.78 </td></tr><tr><td>HoC (%) </td><td>601 </td><td>1,088 </td><td>656 </td><td>1,441 </td><td>1,644 </td><td>96 </td><td>308 </td><td>0.01–0.78 </td></tr><tr><td>LiC (%) </td><td>142 </td><td>332 </td><td>167 </td><td>694 </td><td>563 </td><td>77 </td><td>208 </td><td>0.01–1.06 </td></tr><tr><td>MFA (degree) </td><td>495 </td><td>1,784 </td><td>996 </td><td>1,599 </td><td>2,249 </td><td>95 </td><td>329 </td><td>0.03–2.28 </td></tr><tr><td>&lt;Total&gt; </td><td>3,918 </td><td>8,866 </td><td>5,132 </td><td>4,334 </td><td>8,898 </td><td>111 </td><td>448 </td><td>0.01–13.63 </td></tr></tbody></table></div><div content-id="dsx018-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">No. of epistatic SNP–SNP pairs<sup>a</sup><hr /> </th><th rowspan="2">No. of interacting SNPs </th><th rowspan="2">No. of epistatic gene–gene pairs </th><th colspan="2">No. of interactive genes<hr /> </th><th rowspan="2">Range of contribution rate (<em>c</em>, %) </th></tr><tr><th>AA </th><th>AD (DA) </th><th>DD </th><th>lncRNA </th><th>target </th></tr></thead><tbody><tr><td>DBH (cm) </td><td>404 </td><td>473 </td><td>194 </td><td>897 </td><td>790 </td><td>89 </td><td>249 </td><td>0.03–0.83 </td></tr><tr><td>H (m) </td><td>107 </td><td>240 </td><td>104 </td><td>548 </td><td>418 </td><td>71 </td><td>176 </td><td>0.03–1.00 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>1,230 </td><td>2,217 </td><td>1,380 </td><td>1,670 </td><td>3,221 </td><td>97 </td><td>359 </td><td>0.03–12.95 </td></tr><tr><td>αC (%) </td><td>448 </td><td>751 </td><td>455 </td><td>1,329 </td><td>1,340 </td><td>89 </td><td>302 </td><td>0.01–0.91 </td></tr><tr><td>FL (mm) </td><td>208 </td><td>505 </td><td>324 </td><td>860 </td><td>806 </td><td>72 </td><td>257 </td><td>0.01–13.63 </td></tr><tr><td>FW (µm) </td><td>242 </td><td>1,025 </td><td>657 </td><td>1,309 </td><td>1,374 </td><td>88 </td><td>307 </td><td>0.01–0.67 </td></tr><tr><td>HemC (%) </td><td>359 </td><td>998 </td><td>487 </td><td>1,375 </td><td>1,461 </td><td>91 </td><td>299 </td><td>0.02–3.78 </td></tr><tr><td>HoC (%) </td><td>601 </td><td>1,088 </td><td>656 </td><td>1,441 </td><td>1,644 </td><td>96 </td><td>308 </td><td>0.01–0.78 </td></tr><tr><td>LiC (%) </td><td>142 </td><td>332 </td><td>167 </td><td>694 </td><td>563 </td><td>77 </td><td>208 </td><td>0.01–1.06 </td></tr><tr><td>MFA (degree) </td><td>495 </td><td>1,784 </td><td>996 </td><td>1,599 </td><td>2,249 </td><td>95 </td><td>329 </td><td>0.03–2.28 </td></tr><tr><td>&lt;Total&gt; </td><td>3,918 </td><td>8,866 </td><td>5,132 </td><td>4,334 </td><td>8,898 </td><td>111 </td><td>448 </td><td>0.01–13.63 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>No., number.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Total numbers of significant (<em>P</em> &lt; 0.001) SNP–SNP pairs with different types of epistatic effects. The SNP-SNP epistatic effects can be interpreted into four types: AA, additive × additive effect; AD, additive × dominant effect; DA, dominant × additive effect; and DD, dominant × dominant effect.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx018-T3" data-id="dsx018-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>The epistatic effect of significant (<em>P</em> &lt; 0.001) SNPs associated with tree growth and wood properties in the association population of <em>P. tomentosa</em> detected by EPISNP v2.0</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">No. of epistatic SNP–SNP pairs<sup>a</sup><hr /> </th><th rowspan="2">No. of interacting SNPs </th><th rowspan="2">No. of epistatic gene–gene pairs </th><th colspan="2">No. of interactive genes<hr /> </th><th rowspan="2">Range of contribution rate (<em>c</em>, %) </th></tr><tr><th>AA </th><th>AD (DA) </th><th>DD </th><th>lncRNA </th><th>target </th></tr></thead><tbody><tr><td>DBH (cm) </td><td>404 </td><td>473 </td><td>194 </td><td>897 </td><td>790 </td><td>89 </td><td>249 </td><td>0.03–0.83 </td></tr><tr><td>H (m) </td><td>107 </td><td>240 </td><td>104 </td><td>548 </td><td>418 </td><td>71 </td><td>176 </td><td>0.03–1.00 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>1,230 </td><td>2,217 </td><td>1,380 </td><td>1,670 </td><td>3,221 </td><td>97 </td><td>359 </td><td>0.03–12.95 </td></tr><tr><td>αC (%) </td><td>448 </td><td>751 </td><td>455 </td><td>1,329 </td><td>1,340 </td><td>89 </td><td>302 </td><td>0.01–0.91 </td></tr><tr><td>FL (mm) </td><td>208 </td><td>505 </td><td>324 </td><td>860 </td><td>806 </td><td>72 </td><td>257 </td><td>0.01–13.63 </td></tr><tr><td>FW (µm) </td><td>242 </td><td>1,025 </td><td>657 </td><td>1,309 </td><td>1,374 </td><td>88 </td><td>307 </td><td>0.01–0.67 </td></tr><tr><td>HemC (%) </td><td>359 </td><td>998 </td><td>487 </td><td>1,375 </td><td>1,461 </td><td>91 </td><td>299 </td><td>0.02–3.78 </td></tr><tr><td>HoC (%) </td><td>601 </td><td>1,088 </td><td>656 </td><td>1,441 </td><td>1,644 </td><td>96 </td><td>308 </td><td>0.01–0.78 </td></tr><tr><td>LiC (%) </td><td>142 </td><td>332 </td><td>167 </td><td>694 </td><td>563 </td><td>77 </td><td>208 </td><td>0.01–1.06 </td></tr><tr><td>MFA (degree) </td><td>495 </td><td>1,784 </td><td>996 </td><td>1,599 </td><td>2,249 </td><td>95 </td><td>329 </td><td>0.03–2.28 </td></tr><tr><td>&lt;Total&gt; </td><td>3,918 </td><td>8,866 </td><td>5,132 </td><td>4,334 </td><td>8,898 </td><td>111 </td><td>448 </td><td>0.01–13.63 </td></tr></tbody></table></div><div content-id="dsx018-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Trait </th><th colspan="3">No. of epistatic SNP–SNP pairs<sup>a</sup><hr /> </th><th rowspan="2">No. of interacting SNPs </th><th rowspan="2">No. of epistatic gene–gene pairs </th><th colspan="2">No. of interactive genes<hr /> </th><th rowspan="2">Range of contribution rate (<em>c</em>, %) </th></tr><tr><th>AA </th><th>AD (DA) </th><th>DD </th><th>lncRNA </th><th>target </th></tr></thead><tbody><tr><td>DBH (cm) </td><td>404 </td><td>473 </td><td>194 </td><td>897 </td><td>790 </td><td>89 </td><td>249 </td><td>0.03–0.83 </td></tr><tr><td>H (m) </td><td>107 </td><td>240 </td><td>104 </td><td>548 </td><td>418 </td><td>71 </td><td>176 </td><td>0.03–1.00 </td></tr><tr><td>V (m<sup>3</sup>) </td><td>1,230 </td><td>2,217 </td><td>1,380 </td><td>1,670 </td><td>3,221 </td><td>97 </td><td>359 </td><td>0.03–12.95 </td></tr><tr><td>αC (%) </td><td>448 </td><td>751 </td><td>455 </td><td>1,329 </td><td>1,340 </td><td>89 </td><td>302 </td><td>0.01–0.91 </td></tr><tr><td>FL (mm) </td><td>208 </td><td>505 </td><td>324 </td><td>860 </td><td>806 </td><td>72 </td><td>257 </td><td>0.01–13.63 </td></tr><tr><td>FW (µm) </td><td>242 </td><td>1,025 </td><td>657 </td><td>1,309 </td><td>1,374 </td><td>88 </td><td>307 </td><td>0.01–0.67 </td></tr><tr><td>HemC (%) </td><td>359 </td><td>998 </td><td>487 </td><td>1,375 </td><td>1,461 </td><td>91 </td><td>299 </td><td>0.02–3.78 </td></tr><tr><td>HoC (%) </td><td>601 </td><td>1,088 </td><td>656 </td><td>1,441 </td><td>1,644 </td><td>96 </td><td>308 </td><td>0.01–0.78 </td></tr><tr><td>LiC (%) </td><td>142 </td><td>332 </td><td>167 </td><td>694 </td><td>563 </td><td>77 </td><td>208 </td><td>0.01–1.06 </td></tr><tr><td>MFA (degree) </td><td>495 </td><td>1,784 </td><td>996 </td><td>1,599 </td><td>2,249 </td><td>95 </td><td>329 </td><td>0.03–2.28 </td></tr><tr><td>&lt;Total&gt; </td><td>3,918 </td><td>8,866 </td><td>5,132 </td><td>4,334 </td><td>8,898 </td><td>111 </td><td>448 </td><td>0.01–13.63 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>No., number.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Total numbers of significant (<em>P</em> &lt; 0.001) SNP–SNP pairs with different types of epistatic effects. The SNP-SNP epistatic effects can be interpreted into four types: AA, additive × additive effect; AD, additive × dominant effect; DA, dominant × additive effect; and DD, dominant × dominant effect.</p></div></div></div></div></p><p>To improve our understanding of the genetic architecture that affects complex phenotypic variation, we drew the interconnected gene–gene networks for each trait, based on the distribution of the epistatic pairs of SNPs between chromosomes (13,412 pairs) and within chromosomes (3,983 pairs) (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s6.png?Expires=2147483647&Signature=lQgfSYozYPR4A5JFVV0O9l1yKth3QE1dJvkKmoZ2~bdl4hhAu8zV52YVzMXqdFmN3tNbUqgySvAZ4cjqHrKYXJE7drMAiL85QkP8QduBiT92lAHAV~1m~gJOd7sq3eIuaR-CtbNG78-0HVnGrD7S6V7r1Z9VoqviOiassNmQUoo19n5GgTrjNTG0AAOdyzT3-3qy3Ktx~TDVqXa8qr02plaKMCDnTs5A9CyvcxtTTByrJobPgk5MFj4G3HZTjvhZhH3cp4o3vVSMqoFkkMZ4mfb0cOtyRc2fBdr5DVwNmqfPjnBUFy2yD7foo7HHdLHB79bjkQsUYHKF0pKWwi7qcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In total, 8,898 gene–gene pairs among 111 lncRNA genes and 448 PTGs were detected for all 10 traits (<a reveal-id="dsx018-T3" data-open="dsx018-T3" class="link link-reveal link-table xref-fig">Table 3</a>). These epistatic gene pairs could be categorized into 1,963 lncRNA–lncRNA, 183 lncRNA–mRNA, and 6,653 mRNA–mRNA pairs (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s6.png?Expires=2147483647&Signature=lQgfSYozYPR4A5JFVV0O9l1yKth3QE1dJvkKmoZ2~bdl4hhAu8zV52YVzMXqdFmN3tNbUqgySvAZ4cjqHrKYXJE7drMAiL85QkP8QduBiT92lAHAV~1m~gJOd7sq3eIuaR-CtbNG78-0HVnGrD7S6V7r1Z9VoqviOiassNmQUoo19n5GgTrjNTG0AAOdyzT3-3qy3Ktx~TDVqXa8qr02plaKMCDnTs5A9CyvcxtTTByrJobPgk5MFj4G3HZTjvhZhH3cp4o3vVSMqoFkkMZ4mfb0cOtyRc2fBdr5DVwNmqfPjnBUFy2yD7foo7HHdLHB79bjkQsUYHKF0pKWwi7qcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Notably, among the 183 lncRNA–mRNA pairs, we found 166 (∼91%) lncRNA–target interactions involving 79 lncRNA genes and 146 corresponding PTGs for all 10 traits (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s6.png?Expires=2147483647&Signature=lQgfSYozYPR4A5JFVV0O9l1yKth3QE1dJvkKmoZ2~bdl4hhAu8zV52YVzMXqdFmN3tNbUqgySvAZ4cjqHrKYXJE7drMAiL85QkP8QduBiT92lAHAV~1m~gJOd7sq3eIuaR-CtbNG78-0HVnGrD7S6V7r1Z9VoqviOiassNmQUoo19n5GgTrjNTG0AAOdyzT3-3qy3Ktx~TDVqXa8qr02plaKMCDnTs5A9CyvcxtTTByrJobPgk5MFj4G3HZTjvhZhH3cp4o3vVSMqoFkkMZ4mfb0cOtyRc2fBdr5DVwNmqfPjnBUFy2yD7foo7HHdLHB79bjkQsUYHKF0pKWwi7qcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Thus this work may provide evidence for possible interactions between the lncRNAs and PTGs.</p><p>We then focused on the epistatic interactions among the 16 lncRNAs and 22 PTGs in the phenylpropanoid pathway (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). In total, we detected 60 gene–gene pairs among 8 lncRNA genes and 15 PTGs across all 10 traits, including 36 mRNA–mRNA and 24 lncRNA–mRNA pairs (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). For example, the SNP498 within <em>Ptr</em>-<em>4CL</em> has an epistatic effect with SNP1096 from its upstream gene <em>Ptr</em>-<em>PAL</em> on V (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). For the downstream genes <em>Ptr</em>-<em>HCT</em> and <em>Ptr</em>-<em>CHS</em> which were assigned to two different branches of the monolignol biosynthesis pathway, we also detected an epistatic interaction with <em>Ptr</em>-<em>4CL</em> for the MFA (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). These 60 pairs included 10 interacting pairs between lncRNA genes and the corresponding PTGs. For instance, different genotype combinations between SNP507 within <em>TCONS_00060904</em> and SNP498 from the PTGs <em>Ptr</em>-<em>4CL</em> showed considerable non-additive effects on LiC and HoC, although the two genes displayed no significant additive/dominant effect (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>).</p>                            <h2 scrollto-destination=106355700 id="106355700" class="section-title" >4. Discussion</h2>
<p>To date, systematic researches for lncRNAs have been conducted in only a few plants. In this study, we identified 15,691 <em>Populus</em> lncRNAs with a focus on secondary growth and wood biosynthesis (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Although there are some limitations for the definition of lncRNAs that merely rely on transcript size and coding potential, a relatively robust and reliable list of <em>Populus</em> lncRNAs is provided here for the relatively strict bioinformatics criteria used for the definition as lncRNAs. These lncRNAs shared many features of other species;<a reveal-id="dsx018-B18" data-open="dsx018-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsx018-B50" data-open="dsx018-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,<a reveal-id="dsx018-B51" data-open="dsx018-B51" class="link link-ref link-reveal xref-bibr">51</a></sup> they were shorter in length and less conserved in sequence compared with protein-coding transcripts, and expressed in a tissue-specific manner (<a reveal-id="dsx018-F1" data-open="dsx018-F1" class="link link-reveal link-table xref-fig">Figs 1 and 2</a>). These findings suggested that these <em>Populus</em> lncRNAs may exhibit a biological function, rather than merely being transcriptional noise. Thus, this work may provide new insight into the study of secondary growth of plants.</p>                            <h3 scrollto-destination=106355702 id="106355702" class="section-title" >4.1. Prediction of potential targets and functional motifs of <em>Populus</em> lncRNAs revealed that they may be involved in wood biosynthesis</h3>
<p>Functional characterization of lncRNAs remains at an early stage. The commonly used methods for lncRNA functional prediction are based on protein binding,<a reveal-id="dsx018-B52" data-open="dsx018-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> epigenetic modiﬁcation,<a reveal-id="dsx018-B53" data-open="dsx018-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> co-expression networks,<a reveal-id="dsx018-B54" data-open="dsx018-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> and miRNA regulation.<a reveal-id="dsx018-B55" data-open="dsx018-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Here, we employed a target prediction programme for <em>Populus</em> lncRNAs based on adjacent gene functions (<em>cis</em>-regulation) and sequence complementarity (<em>trans</em>-regulation). A total of 1,994 (12.6%) lncRNAs were DE between/within high-biomass and low-biomass pools (<a reveal-id="dsx018-F3" data-open="dsx018-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), comparable to protein-coding genes (8,573) under the same differential expression threshold. This comparison supports the idea that the lncRNAs have a biological purpose and are subject to complex transcriptional regulation during wood biosynthesis. Interestingly, lncRNAs in mature xylem exhibited more (2- to 3-fold) differential expression compared with vascular cambium and developing xylem from the high- and low-biomass pools, although approximately evenly split between genes that were differentially up- and down-regulated in each pair-wise comparison (<a reveal-id="dsx018-F3" data-open="dsx018-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). This may suggest that the regulation of lncRNAs is subject to transcriptional rearrangement during wood biosynthesis and more lncRNAs are involved in xylem development.</p><p>The importance of transcriptional regulation is illustrated by the coordinated expression of speciﬁc classes of genes in related biological processes occurring at speciﬁc stages of wood formation, which initiates in the vascular cambium and involves cell division, expansion, secondary wall formation and lignification in developing xylem, and finally, programmed cell death to form mature xylem.<a reveal-id="dsx018-B56" data-open="dsx018-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Supporting this, using a target prediction programme for the DE lncRNAs, we found many important PTGs encoding cell division and expansion proteins, cell wall-associated proteins, and stress-responsive proteins, as well as many transcription factors, including 31 PTGs in the phenylpropanoid pathway. GO analysis showed that the PTGs of lncRNAs highly expressed in mature xylem were enriched in many biological processes related to programmed cell death; however, the PTGs of lncRNAs highly expressed in vascular cambium and developing xylem were assigned to many GO terms, such as ‘carbohydrate metabolic process’, ‘polysaccharide biosynthetic process’, ‘cell morphogenesis’, and ‘response to hormone stimulus’ (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s3.png?Expires=2147483647&Signature=DqVFWpPx53p8qtAhjsJXkVKsmPcuZh8~1MB2zt675ZYz3W1ak5KSBx96djdJiAYLUttsOEQlCtiWKznfGzfS6sPF~mikIb1wZRsvqTMuwrHzv0Zn--JLGFv9rcefjJkoVYmBFGoFv3U5amZF1fNHuUmG47Z72xDCnUIMALUjQuBAEu-GqHAuz9mGaZ6wfaOLxDdxiLrGCwjMZnYrAheguL~rZCXPt98HNvAMprjVisIwtMwj2mRxHif82ctQGunN4RWiIwwyKE7eeKZXwFbH3rzZYupruJERpmgFWSSFs7a7j63-VD3eERhy040eUlllVXmSwFG7D-PwkZOs6-HyVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Thus, our observation of the activation of lncRNAs in mature xylem and relatively little difference between vascular cambium and developing xylem is expected. Additionally, we found that the PTGs of lncRNAs DE between these two pools were significantly enriched in many represented biological processes, such as ‘carbohydrate metabolic process’, ‘oxidation reduction’, ‘response to hormone stimulus’, and ‘regulation of protein stability’ (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s3.png?Expires=2147483647&Signature=DqVFWpPx53p8qtAhjsJXkVKsmPcuZh8~1MB2zt675ZYz3W1ak5KSBx96djdJiAYLUttsOEQlCtiWKznfGzfS6sPF~mikIb1wZRsvqTMuwrHzv0Zn--JLGFv9rcefjJkoVYmBFGoFv3U5amZF1fNHuUmG47Z72xDCnUIMALUjQuBAEu-GqHAuz9mGaZ6wfaOLxDdxiLrGCwjMZnYrAheguL~rZCXPt98HNvAMprjVisIwtMwj2mRxHif82ctQGunN4RWiIwwyKE7eeKZXwFbH3rzZYupruJERpmgFWSSFs7a7j63-VD3eERhy040eUlllVXmSwFG7D-PwkZOs6-HyVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), indicating that the coordinated regulation of lncRNAs and their potential targets in these biological processes may be responsible for the differentiation between these two biomass pools.</p><p>Noteworthily, all the target genes of lncRNAs predicted in our study are potential targets and need further experimental confirmation in future studies. Although the functions of lncRNAs remain largely unknown, previous studies have reported that lncRNAs may participate in various biological processes without the need to be mutually exclusive. Given the complex functions of lncRNAs and the limited strategies that merely rely on <em>cis</em>-/<em>trans</em>-regulation for the prediction of lncRNA targets in this study, more target genes likely remain to be explored and other approaches for prediction of target genes for lncRNAs need to be applied.</p>                            <h3 scrollto-destination=106355706 id="106355706" class="section-title" >4.2. <em>Populus</em> lncRNAs may play roles in miRNA-mediated regulatory networks in secondary wall formation</h3>
<p>MiRNAs function to destabilize mRNAs and repress the translation of protein-coding genes and play essential roles in plant development and physiology.<a reveal-id="dsx018-B57" data-open="dsx018-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Moreover, interesting cross-regulation between miRNAs and lncRNAs has recently become apparent,<a reveal-id="dsx018-B58" data-open="dsx018-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> implying that lncRNAs may interact with miRNAs apart from protein-coding genes. Thus, the intriguing mechanism of lncRNA–miRNA crosstalk may reveal a new layer of functions of lncRNAs. Indeed, in the present study, we identified 476 lncRNAs as potential targets of 213 miRNAs from 90 families (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S9). Among these miRNA families targeting lncRNAs, the stress-responsive miRNA, ptc-miR473a, which plays an important role in biosynthesis of cell wall metabolites and plant development,<a reveal-id="dsx018-B59" data-open="dsx018-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> exhibits near-perfectly complementarity to its target <em>TCONS_00108161</em>. Also, we found that <em>TCONS_00078539</em> was predicted to be the potential target of miR168, which could be involved in auxin signaling and secondary cell wall biosynthesis.<a reveal-id="dsx018-B60" data-open="dsx018-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx018-B61" data-open="dsx018-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> In addition, ptr-miR397, which could be a negative regulator of laccase genes affecting LiC,<a reveal-id="dsx018-B62" data-open="dsx018-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> was predicted to target three lncRNAs (<em>TCONS_00013182</em>, <em>TCONS_00015036</em>, and <em>TCONS_00028534</em>). These results indicated that <em>Populus</em> lncRNAs may play roles in secondary cell wall biosynthesis and wood formation as potential targets of many regulatory miRNAs.</p><p>Target mimicry is a recently identified mechanism of miRNA regulation initially studied in <em>Arabidopsis</em>,<a reveal-id="dsx018-B7" data-open="dsx018-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and subsequently in mammals;<a reveal-id="dsx018-B63" data-open="dsx018-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> lncRNAs can act as miRNA decoys or sponges to sequester miRNAs, thus favouring the expression of repressed target mRNAs. Indeed, many eTMs have been functionally confirmed in <em>Arabidopsis</em> and rice.<a reveal-id="dsx018-B32" data-open="dsx018-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> In our study, 28 lncRNAs were predicted to be eTMs of 14 miRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S10). Interestingly, consistent with the previous studies,<a reveal-id="dsx018-B51" data-open="dsx018-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a><sup>,</sup><a reveal-id="dsx018-B64" data-open="dsx018-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> we identified novel eTMs for four miRNAs (miR160, miR164, miR169, and miR482a.1). This may indicate that eTM-mediated regulation for these four miRNAs may be pervasive in plants. For example, miR164, which may affect lignin metabolism by regulating transcription factors that control secondary growth and wood composition,<a reveal-id="dsx018-B65" data-open="dsx018-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> was predicted to be absorbed by five eTMs. Additionally, in some cases, like miR408, which is involved in lignin deposition and polymerization,<a reveal-id="dsx018-B59" data-open="dsx018-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> might target seven lncRNAs and be absorbed by three eTMs. Therefore, based on these analyses, we infer that these lncRNAs potentially function via interacting with miRNAs in complicated networks involving interactions among lncRNAs, mRNAs, and miRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s4.png?Expires=2147483647&amp;Signature=K0De6IcOXLjhqUUWD4sWIFvK~JkKAa0HN6nVMp7vWUE9fIAcdCoU9hT4AeXOaLBeughr98m1D2josjVS86SO7ldnSkEVayZ~yb~1~2O22fAt3vQz4tIQPtzKlhV1XslOrWTgKk4azWPcJ-hlXy0m0oJAhj4V7wfh46KOYmqeYtPGic4iUZ1NAc8LhP5HLVnT4o8drGwxFwYzbK3sfWl3qdD~L13-Qw5n8761KPVNd28F8HHYO~wzd-IW0Th3IVW3v9XmCkPtl2Ln0ZG6GausNbOXLKjfN5bdcF1j7p82ujpyksvcrpzNXKhE407DulktJ6YIgAPIQK7mLGnVkgZuWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). However, all the findings are mainly based on our bioinformatics prediction, and the specific regulatory mechanism requires further investigation and validation, such as the cellular localization of miRNAs and lncRNAs predicted as miRNA targets and eTMs using <em>in situ</em> hybridization. We believe that the emergence of important roles for lncRNAs in secondary cell wall biosynthesis and tree growth and our investigation of lncRNA–miRNA interaction network may provide an important resource for further studies.</p>                            <h3 scrollto-destination=106355709 id="106355709" class="section-title" >4.3. Allelic variations of <em>Populus</em> lncRNAs and their PTGs indicated that they were associated with growth and wood properties</h3>
<p>Association mapping provides a valuable tool allowing us to identify the natural allelic variation responsible for a particular phenotype.<a reveal-id="dsx018-B66" data-open="dsx018-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> In this study, we performed association studies for lncRNAs and PTGs, integrating additive, dominant, and epistatic effects of allelic variations underlying wood biosynthesis and stem biomass. Thus, this study initiated the dissection of potential genetic regulatory mechanisms involving lncRNAs and their PTGs. In total, 1,163 unique SNPs from lncRNAs and their PTGs were significantly associated with growth and wood properties based on single-SNP associations (<a reveal-id="dsx018-T1" data-open="dsx018-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Also, ∼28% (329 of 1,163) SNPs were associated with at least two traits and each trait could be associated with 6 SNPs (H) to 666 SNPs (DBH) (<a reveal-id="dsx018-T1" data-open="dsx018-T1" class="link link-reveal link-table xref-fig">Table 1</a>), consistent with a previous study that genetic association studies often found that a locus associated with multiple, sometimes seemingly distinct traits. This observation also highlights the possibility that traits share common genetic pathways and underscores the relevance of pleiotropy.<a reveal-id="dsx018-B67" data-open="dsx018-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Intriguingly, ∼77% of the lncRNAs and their corresponding PTGs associated with at least one trait in common. For example, we found that SNPs within the lncRNA <em>TCONS_00059927</em> and its potential <em>cis</em> PTGs simultaneously associated with HoC and V under additive/dominant effects (<a reveal-id="dsx018-F6" data-open="dsx018-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>), indicating the potential genetic interaction between the lncRNA and its corresponding target genes in the pathways underlying this complex genetic network.</p><p>In quantitative genetics, epistasis, defined as the interaction between genotypes at two or more loci, is an important component of the genetic basis of complex traits and can also involve modification of the additive or dominant effects of the interacting loci. Therefore, phenotypic variations cannot be predicted simply by summing the effects of individual loci.<a reveal-id="dsx018-B68" data-open="dsx018-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> At the gene level, epistasis is a phenomenon whereby the effects of a given gene on a biological trait are masked or enhanced by one or more other genes.<a reveal-id="dsx018-B69" data-open="dsx018-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> Previous studies have documented that epistasis analysis has considerable potential to reveal interactions between genes and pathways.<a reveal-id="dsx018-B70" data-open="dsx018-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> In this study, we detected 17,656 significant SNP-SNP epistatic pairs, covering 8.6% of all investigated SNPs, associated with tree growth and wood properties (<a reveal-id="dsx018-T3" data-open="dsx018-T3" class="link link-reveal link-table xref-fig">Table 3</a>). Interestingly, we did not observe significant additive and/or dominant effects within these epistatic pairs. The result revealed pervasive epistatic effects among allelic mutations without significant main effects in the natural population of <em>Populus</em>, supporting the idea that epistasis plays a key role in genetic architecture and also providing essential information for a deeper understanding of the mechanisms of gene interactions.<a reveal-id="dsx018-B68" data-open="dsx018-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a></p><p>Examining a specific pathway, we found an epistatic interaction network among 8 lncRNAs and 15 PTGs involved in the phenylpropanoid pathway (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Among the 15 target genes, we detected 36 epistatic pairs across all 10 traits (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Consistent with the steps in the pathway, we found <em>Ptr</em>-<em>4CL</em> had epistatic interactions with its upstream gene <em>Ptr</em>-<em>PAL</em>, which encodes the first enzyme of the pathway and also the two downstream genes <em>Ptr</em>-<em>HCT</em> and <em>Ptr</em>-<em>CHS</em>, which encode enzymes assigned to the two different branches in the pathway (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). In addition, we identified epistatic interactions between genes where the biological evidence for how they interacted had not yet been found (<a reveal-id="dsx018-F4" data-open="dsx018-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). Intriguingly, our results are consistent with studies in vertebrates and sponges, where lncRNAs have been shown to be highly interconnected with multiple protein-coding genes based on expression profiles.<a reveal-id="dsx018-B2" data-open="dsx018-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsx018-B71" data-open="dsx018-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> In the epistatic network, 24 interacting pairs between 8 lncRNAs and 12 genes showed that the lncRNAs can have epistatic interactions with their corresponding target genes and also interact with other non-target genes predicted by our target prediction programme, suggesting that the lncRNAs may interact with their target genes by different mechanisms in the pathway. However, the precise mechanisms underlying such gene–gene interactions are largely unknown. Our findings therefore may provide a reference for follow-up studies to further explore this hypothesis. Collectively, our epistasis analysis of lncRNAs and their potential targets genes may enrich the interaction networks among genes in the pathway, since these genes may be part of a common regulatory network.</p>                            <h2 scrollto-destination=106355713 id="106355713" class="section-title" >Accession number</h2>
<p>SRA accession: SRP073689.</p>                            <h2 scrollto-destination=106355715 id="106355715" class="section-title" >Conflict of interest</h2>
<p>None declared. </p>                            <h2 scrollto-destination=106355717 id="106355717" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=106355719 id="106355719" class="section-title" >Funding</h2>
<p>This work was supported by the State ‘13.5’ Key Research Programme of China (No. 2016YFD0600102), and the Project of the National Natural Science Foundation of China (No. 31670333).</p>                            <h2 scrollto-destination=106355721 id="106355721" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx018-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ma</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Bajic</div> <div class="given-names">V. B.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">On the classification of long non-coding RNAs,</div><div class="source">RNA Bio</div> ., <div class="volume">10</div>, <div class="fpage">924</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20classification%20of%20long%20non-coding%20RNAs%2C&amp;author=L.%20Ma&amp;author=V.%20B.%20Bajic&amp;author=Z.%20Zhang&amp;publication_year=2013&amp;journal=RNA%20Bio&amp;volume=10&amp;pages=924-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/rna.24604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Frna.24604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Frna.24604"> </span></div> </div></div></div></div></div><div content-id="dsx018-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Necsulea</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Soumillon</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Warnefors</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">The evolution of lncRNA repertoires and expression patterns in tetrapods</div>. <div class="source">Nature</div> , <div class="volume">505</div>, <div class="fpage">635</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20lncRNA%20repertoires%20and%20expression%20patterns%20in%20tetrapods&amp;author=A.%20Necsulea&amp;author=M.%20Soumillon&amp;author=M.%20Warnefors&amp;publication_year=2014&amp;journal=Nature&amp;volume=505&amp;pages=635-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12943" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12943" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12943"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24463510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cabili</div>
               <div class="given-names">M.N.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses</div>. <div class="source">Genes Dev</div> ., <div class="volume">25</div>, <div class="fpage">1915</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20annotation%20of%20human%20large%20intergenic%20noncoding%20RNAs%20reveals%20global%20properties%20and%20specific%20subclasses&amp;author=M.N.%20Cabili&amp;author=C.%20Trapnell&amp;author=L.%20Goff&amp;publication_year=2011&amp;journal=Genes%20Dev&amp;volume=25&amp;pages=1915-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.17446611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.17446611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.17446611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ponting</div>
               <div class="given-names">C.P.</div>
            </div>, <div class="name string-name"><div class="surname">Oliver</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Reik</div> <div class="given-names">W.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Evolution and functions of long noncoding RNAs</div>. <div class="source">Cell</div> , <div class="volume">136</div>, <div class="fpage">629</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20functions%20of%20long%20noncoding%20RNAs&amp;author=C.P.%20Ponting&amp;author=P.L.%20Oliver&amp;author=W.%20Reik&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=629-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Swiezewski</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Magusin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">C.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Cold-induced silencing by long antisense transcripts of an Arabidopsis polycomb target</div>. <div class="source">Nature</div> , <div class="volume">462</div>, <div class="fpage">799</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cold-induced%20silencing%20by%20long%20antisense%20transcripts%20of%20an%20Arabidopsis%20polycomb%20target&amp;author=S.%20Swiezewski&amp;author=F.%20Liu&amp;author=A.%20Magusin&amp;author=C.%20Dean&amp;publication_year=2009&amp;journal=Nature&amp;volume=462&amp;pages=799-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08618" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08618"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heo</div>
               <div class="given-names">J.B.</div>
            </div>, <div class="name string-name"><div class="surname">Sung</div> <div class="given-names">S.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Vernalization-mediated epigenetic silencing by a long intronic noncoding RNA</div>. <div class="source">Science</div> , <div class="volume">331</div>, <div class="fpage">76</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vernalization-mediated%20epigenetic%20silencing%20by%20a%20long%20intronic%20noncoding%20RNA&amp;author=J.B.%20Heo&amp;author=S.%20Sung&amp;publication_year=2011&amp;journal=Science&amp;volume=331&amp;pages=76-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1197349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1197349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1197349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21127216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Franco-Zorrilla</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Valli</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Todesco</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Target mimicry provides a new mechanism for regulation of microRNA activity</div>. <div class="source">Nat. Genet</div> ., <div class="volume">39</div>, <div class="fpage">1033</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target%20mimicry%20provides%20a%20new%20mechanism%20for%20regulation%20of%20microRNA%20activity&amp;author=J.M.%20Franco-Zorrilla&amp;author=A.%20Valli&amp;author=M.%20Todesco&amp;publication_year=2007&amp;journal=Nat.%20Genet&amp;volume=39&amp;pages=1033-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17643101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Demura</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z.-H.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Regulation of plant biomass production</div>. <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">13</div>, <div class="fpage">298</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20plant%20biomass%20production&amp;author=T.%20Demura&amp;author=Z.-H.%20Ye&amp;publication_year=2010&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=13&amp;pages=298-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2010.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2010.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2010.03.002"> </span></div> </div></div></div></div></div><div content-id="dsx018-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shi</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Y.-H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Heber</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sederoff</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chiang</div> <div class="given-names">V.L.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Towards a systems approach for lignin biosynthesis in <em>Populus trichocarpa</em>: transcript abundance and specificity of the monolignol biosynthetic genes</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">51</div>, <div class="fpage">144</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20a%20systems%20approach%20for%20lignin%20biosynthesis%20in%20Populus%20trichocarpa%3A%20transcript%20abundance%20and%20specificity%20of%20the%20monolignol%20biosynthetic%20genes&amp;author=R.%20Shi&amp;author=Y.-H.%20Sun&amp;author=Q.%20Li&amp;author=S.%20Heber&amp;author=R.%20Sederoff&amp;author=V.L.%20Chiang&amp;publication_year=2010&amp;journal=Plant%20Cell%20Physiol&amp;volume=51&amp;pages=144-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcp175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcp175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcp175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19996151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shastry</div>
               <div class="given-names">B.S.</div>
            </div>
          </div>
          <div class="year">2009</div>, SNPs: impact on gene function and phenotype. In: <div class="source">Single Nucleotide Polymorphisms</div> , 
<div class="publisher-loc">Berlin</div>: 
<div class="publisher-name">Springer</div>, pp. <div class="fpage">3</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20Nucleotide%20Polymorphisms&amp;author=B.S.%20Shastry&amp;publication_year=2009&amp;book=Single%20Nucleotide%20Polymorphisms" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-1-60327-411-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-1-60327-411-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-1-60327-411-1"> </span></div> </div></div></div></div></div><div content-id="dsx018-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">J.-Y.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.-C.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">Y.-T.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Long noncoding RNAs-related diseases, cancers, and drugs</div>. <div class="source">Scientific World J</div> ., <div class="volume">2013</div>: <div class="fpage">943539</div>.</div></div></div></div><div content-id="dsx018-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ding</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Ouyang</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">A long noncoding RNA regulates photoperiod-sensitive male sterility, an essential component of hybrid rice</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">109</div>, <div class="fpage">2654</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20long%20noncoding%20RNA%20regulates%20photoperiod-sensitive%20male%20sterility%2C%20an%20essential%20component%20of%20hybrid%20rice&amp;author=J.%20Ding&amp;author=Q.%20Lu&amp;author=Y.%20Ouyang&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=109&amp;pages=2654-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1121374109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1121374109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1121374109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22308482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hindorff</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Sethupathy</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Junkins</div> <div class="given-names">H.A.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Potential etiologic and functional implications of genome-wide association loci for human diseases and traits</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">106</div>, <div class="fpage">9362</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Potential%20etiologic%20and%20functional%20implications%20of%20genome-wide%20association%20loci%20for%20human%20diseases%20and%20traits&amp;author=L.A.%20Hindorff&amp;author=P.%20Sethupathy&amp;author=H.A.%20Junkins&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=106&amp;pages=9362-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0903103106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0903103106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0903103106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19474294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Westra</div> <div class="given-names">H.-J.</div></div>, <div class="name string-name"><div class="surname">Karjalainen</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Human disease-associated genetic variation impacts large intergenic non-coding RNA expression</div>. <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003201</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20disease-associated%20genetic%20variation%20impacts%20large%20intergenic%20non-coding%20RNA%20expression&amp;author=V.%20Kumar&amp;author=H.-J.%20Westra&amp;author=J.%20Karjalainen&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23341781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dalessandro</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Northcote</div> <div class="given-names">D.</div></div></div>
          <div class="year">1977</div>, 
<div class="article-title">Changes in enzymic activities of nucleoside diphosphate sugar interconversions during differentiation of cambium to xylem in sycamore and poplar</div>. <div class="source">Bio. J</div> ., <div class="volume">162</div>, <div class="fpage">267</div>–<div class="lpage">79</div>.</div></div></div></div><div content-id="dsx018-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuskan</div>
               <div class="given-names">G.A.</div>
            </div>, <div class="name string-name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2006</div>, 
<div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>. <div class="source">Science</div> , <div class="volume">313</div>, <div class="fpage">1596</div>–<div class="lpage">1604</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-1604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>. <div class="source">Nat. Prot</div> ., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Prot&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div> </div></div></div></div></div><div content-id="dsx018-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Eichten</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide discovery and characterization of maize long non-coding RNAs</div>. <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">R40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20discovery%20and%20characterization%20of%20maize%20long%20non-coding%20RNAs&amp;author=L.%20Li&amp;author=S.R.%20Eichten&amp;author=R.%20Shimizu&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-2-r40" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-2-r40" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-2-r40"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24576388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sun</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Prediction of novel long non-coding RNAs based on RNA-Seq data of mouse Klf1 knockout study</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">13</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20novel%20long%20non-coding%20RNAs%20based%20on%20RNA-Seq%20data%20of%20mouse%20Klf1%20knockout%20study&amp;author=L.%20Sun&amp;author=Z.%20Zhang&amp;author=T.L.%20Bailey&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Finn</div>
               <div class="given-names">R.D.</div>
            </div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">HMMER web server: interactive sequence similarity searching</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W29</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HMMER%20web%20server%3A%20interactive%20sequence%20similarity%20searching&amp;author=R.D.%20Finn&amp;author=J.%20Clements&amp;author=S.R.%20Eddy&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W29-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kong</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">Z.-Q.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">CPC: assess the protein-coding potential of transcripts using sequence features and support vector machine</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W345</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CPC%3A%20assess%20the%20protein-coding%20potential%20of%20transcripts%20using%20sequence%20features%20and%20support%20vector%20machine&amp;author=L.%20Kong&amp;author=Y.%20Zhang&amp;author=Z.-Q.%20Ye&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W345-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm391" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm391" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm391"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17631615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sun</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bu</div> <div class="given-names">D.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Utilizing sequence intrinsic composition to classify protein-coding and long non-coding transcripts</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">e166</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Utilizing%20sequence%20intrinsic%20composition%20to%20classify%20protein-coding%20and%20long%20non-coding%20transcripts&amp;author=L.%20Sun&amp;author=H.%20Luo&amp;author=D.%20Bu&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23892401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Roberts</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Identification of novel transcripts in annotated genomes using RNA-Seq</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2325</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20novel%20transcripts%20in%20annotated%20genomes%20using%20RNA-Seq&amp;author=A.%20Roberts&amp;author=H.%20Pimentel&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2325-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21697122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lamesch</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Berardini</div> <div class="given-names">T.Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1202</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20improved%20gene%20annotation%20and%20new%20tools&amp;author=P.%20Lamesch&amp;author=T.Z.%20Berardini&amp;author=D.%20Li&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1202-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ouyang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Hamilton</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">The TIGR rice genome annotation resource: improvements and new features</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">D883</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TIGR%20rice%20genome%20annotation%20resource%3A%20improvements%20and%20new%20features&amp;author=S.%20Ouyang&amp;author=W.%20Zhu&amp;author=J.%20Hamilton&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D883-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17145706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bailey</div>
               <div class="given-names">T.L.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">DREME: motif discovery in transcription factor ChIP-seq data</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">1653</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DREME%3A%20motif%20discovery%20in%20transcription%20factor%20ChIP-seq%20data&amp;author=T.L.%20Bailey&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1653-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21543442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Buske</div>
               <div class="given-names">F.A.</div>
            </div>, <div class="name string-name"><div class="surname">Bodén</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bauer</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Assigning roles to DNA regulatory motifs using comparative genomics</div>. <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">860</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assigning%20roles%20to%20DNA%20regulatory%20motifs%20using%20comparative%20genomics&amp;author=F.A.%20Buske&amp;author=M.%20Bod%C3%A9n&amp;author=D.C.%20Bauer&amp;author=T.L.%20Bailey&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=860-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yanai</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Benjamin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shmoish</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2005</div>, 
<div class="article-title">Genome-wide midrange transcription profiles reveal expression level relationships in human tissue specification</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">650</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20midrange%20transcription%20profiles%20reveal%20expression%20level%20relationships%20in%20human%20tissue%20specification&amp;author=I.%20Yanai&amp;author=H.%20Benjamin&amp;author=M.%20Shmoish&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=650-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15388519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jia</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Osak</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bogu</div> <div class="given-names">G.K.</div></div>, <div class="name string-name"><div class="surname">Stanton</div> <div class="given-names">L.W.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Genome-wide computational identification and manual annotation of human long noncoding RNA genes</div>. <div class="source">RNA</div> , <div class="volume">16</div>, <div class="fpage">1478</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20computational%20identification%20and%20manual%20annotation%20of%20human%20long%20noncoding%20RNA%20genes&amp;author=H.%20Jia&amp;author=M.%20Osak&amp;author=G.K.%20Bogu&amp;author=L.W.%20Stanton&amp;author=R.%20Johnson&amp;author=L.%20Lipovich&amp;publication_year=2010&amp;journal=RNA&amp;volume=16&amp;pages=1478-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.1951310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1951310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.1951310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20587619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Han</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">LncRNA proﬁle of glioblastoma reveals the potential role of lncRNAs in contributing to glioblastoma pathogenesis</div>. <div class="source">Int. J. Oncol</div> ., <div class="volume">40</div>, <div class="fpage">2004</div>–12.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LncRNA%20pro%EF%AC%81le%20of%20glioblastoma%20reveals%20the%20potential%20role%20of%20lncRNAs%20in%20contributing%20to%20glioblastoma%20pathogenesis&amp;author=L.%20Han&amp;author=K.%20Zhang&amp;author=Z.%20Shi&amp;publication_year=2012&amp;journal=Int.%20J.%20Oncol&amp;volume=40&amp;pages=2004" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22446686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tafer</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Hofacker</div> <div class="given-names">I.L.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">RNAplex: a fast tool for RNA–RNA interaction search</div>. <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">2657</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAplex%3A%20a%20fast%20tool%20for%20RNA%E2%80%93RNA%20interaction%20search&amp;author=H.%20Tafer&amp;author=I.L.%20Hofacker&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2657-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn193" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn193" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18434344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">H.-J.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.-M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.-J.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Widespread long noncoding RNAs as endogenous target mimics for microRNAs in plants</div>. <div class="source">Plant Physiol</div> ., <div class="volume">161</div>, <div class="fpage">1875</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20long%20noncoding%20RNAs%20as%20endogenous%20target%20mimics%20for%20microRNAs%20in%20plants&amp;author=H.-J.%20Wu&amp;author=Z.-M.%20Wang&amp;author=M.%20Wang&amp;author=X.-J.%20Wang&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=161&amp;pages=1875-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.215962" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.215962" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.215962"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23429259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saito</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Smoot</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Ono</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">A travel guide to Cytoscape plugins</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">1069</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20travel%20guide%20to%20Cytoscape%20plugins&amp;author=R.%20Saito&amp;author=M.E.%20Smoot&amp;author=K.%20Ono&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=1069-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23132118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Livak</div>
               <div class="given-names">K.J.</div>
            </div>, <div class="name string-name"><div class="surname">Schmittgen</div> <div class="given-names">T.D.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2− ΔΔCT method</div>. <div class="source">Methods</div> , <div class="volume">25</div>, <div class="fpage">402</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202%E2%88%92%20%CE%94%CE%94CT%20method&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen&amp;publication_year=2001&amp;journal=Methods&amp;volume=25&amp;pages=402-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmeth.2001.1262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmeth.2001.1262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Du</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">W.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Allelic variation in a cellulose synthase gene (PtoCesA4) associated with growth and wood properties in Populus tomentosa</div>. <div class="source">G3</div> , <div class="volume">3</div>, <div class="fpage">2069</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%20in%20a%20cellulose%20synthase%20gene%20(PtoCesA4)%20associated%20with%20growth%20and%20wood%20properties%20in%20Populus%20tomentosa&amp;author=Q.%20Du&amp;author=B.%20Xu&amp;author=W.%20Pan&amp;publication_year=2013&amp;journal=G3&amp;volume=3&amp;pages=2069-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.007724" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.007724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.007724"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24048648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Du</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Gong</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Variation in growth, leaf, and wood property traits of Chinese white poplar (<em>Populus tomentosa</em>), a major industrial tree species in Northern China</div>. <div class="source">Can. J. For. Res</div> ., <div class="volume">44</div>, <div class="fpage">326</div>–<div class="lpage">339</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20growth%2C%20leaf%2C%20and%20wood%20property%20traits%20of%20Chinese%20white%20poplar%20(Populus%20tomentosa)%2C%20a%20major%20industrial%20tree%20species%20in%20Northern%20China&amp;author=Q.%20Du&amp;author=B.%20Xu&amp;author=C.%20Gong&amp;publication_year=2014&amp;journal=Can.%20J.%20For.%20Res&amp;volume=44&amp;pages=326-339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/cjfr-2013-0416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fcjfr-2013-0416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fcjfr-2013-0416"> </span></div> </div></div></div></div></div><div content-id="dsx018-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Du</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Gong</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Genetic architecture of growth traits in Populus revealed by integrated quantitative trait locus (QTL) analysis and association studies</div>. <div class="source">New Phytol</div> ., <div class="volume">209</div>, <div class="fpage">1067</div>–<div class="lpage">1082</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20architecture%20of%20growth%20traits%20in%20Populus%20revealed%20by%20integrated%20quantitative%20trait%20locus%20(QTL)%20analysis%20and%20association%20studies&amp;author=Q.%20Du&amp;author=C.%20Gong&amp;author=Q.%20Wang&amp;publication_year=2016&amp;journal=New%20Phytol&amp;volume=209&amp;pages=1067-1082" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.13695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.13695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.13695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26499329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kristiansen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">SOAP: short oligonucleotide alignment program</div>. <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">713</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP%3A%20short%20oligonucleotide%20alignment%20program&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=K.%20Kristiansen&amp;author=J.%20Wang&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=713-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18227114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Danecek</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Auton</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Abecasis</div> <div class="given-names">G.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">The variant call format and VCFtools</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2156</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20variant%20call%20format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2156-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr330" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr330" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr330"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21653522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1987</div>, <div class="source">Molecular Evolutionary Genetics</div> . 
<div class="publisher-name">Columbia University Press</div>.</div></div></div></div><div content-id="dsx018-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Watterson</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1975</div>, 
<div class="article-title">On the number of segregating sites in genetical models without recombination</div>. <div class="source">Theor. Popul. Biol</div> ., <div class="volume">7</div>, <div class="fpage">256</div>–<div class="lpage">276</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetical%20models%20without%20recombination&amp;author=G.%20Watterson&amp;publication_year=1975&amp;journal=Theor.%20Popul.%20Biol&amp;volume=7&amp;pages=256-276" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0040-5809(75)90020-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0040-5809(75)90020-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1145509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tajima</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">1989</div>, 
<div class="article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</div>. <div class="source">Genetics</div> , <div class="volume">123</div>, <div class="fpage">585</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F.%20Tajima&amp;publication_year=1989&amp;journal=Genetics&amp;volume=123&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Pressoir</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Briggs</div> <div class="given-names">W.H.</div></div></div>, et al.  <div class="year">2006</div>, 
<div class="article-title">A unified mixed-model method for association mapping that accounts for multiple levels of relatedness</div>. <div class="source">Nat. Genet</div> ., <div class="volume">38</div>, <div class="fpage">203</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unified%20mixed-model%20method%20for%20association%20mapping%20that%20accounts%20for%20multiple%20levels%20of%20relatedness&amp;author=J.%20Yu&amp;author=G.%20Pressoir&amp;author=W.H.%20Briggs&amp;publication_year=2006&amp;journal=Nat.%20Genet&amp;volume=38&amp;pages=203-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16380716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Du</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Genetic diversity and population structure of Chinese white poplar (<em>Populus tomentosa</em>) revealed by SSR markers</div>. <div class="source">J. Hered</div> ., <div class="volume">103</div>, <div class="fpage">853</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20population%20structure%20of%20Chinese%20white%20poplar%20(Populus%20tomentosa)%20revealed%20by%20SSR%20markers&amp;author=Q.%20Du&amp;author=B.%20Wang&amp;author=Z.%20Wei&amp;author=D.%20Zhang&amp;author=B.%20Li&amp;publication_year=2012&amp;journal=J.%20Hered&amp;volume=103&amp;pages=853-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/ess061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2Fess061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2Fess061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23008443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hardy</div>
               <div class="given-names">O.J.</div>
            </div>, <div class="name string-name"><div class="surname">Vekemans</div> <div class="given-names">X.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">SPAGeDi: a versatile computer program to analyse spatial genetic structure at the individual or population levels</div>. <div class="source">Mol. Ecol. Notes</div> , <div class="volume">2</div>, <div class="fpage">618</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPAGeDi%3A%20a%20versatile%20computer%20program%20to%20analyse%20spatial%20genetic%20structure%20at%20the%20individual%20or%20population%20levels&amp;author=O.J.%20Hardy&amp;author=X.%20Vekemans&amp;publication_year=2002&amp;journal=Mol.%20Ecol.%20Notes&amp;volume=2&amp;pages=618-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1471-8286.2002.00305.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1471-8286.2002.00305.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1471-8286.2002.00305.x"> </span></div> </div></div></div></div></div><div content-id="dsx018-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hubisz</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Falush</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Inferring weak population structure with the assistance of sample group information</div>. <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">9</div>, <div class="fpage">1322</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information&amp;author=M.J.%20Hubisz&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2009&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=9&amp;pages=1322-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2009.9.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2009.9.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2009.9.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21564903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Storey</div>
               <div class="given-names">J.D.</div>
            </div>, <div class="name string-name"><div class="surname">Tibshirani</div> <div class="given-names">R.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Statistical significance for genomewide studies</div>. <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">100</div>, <div class="fpage">9440</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20significance%20for%20genomewide%20studies&amp;author=J.D.%20Storey&amp;author=R.%20Tibshirani&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=9440-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1530509100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1530509100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1530509100"> </span></div> </div></div></div></div></div><div content-id="dsx018-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ma</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Runesha</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Dvorkin</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Garbe</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Da</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Parallel and serial computing tools for testing single-locus and epistatic SNP effects of quantitative traits in genome-wide association studies</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parallel%20and%20serial%20computing%20tools%20for%20testing%20single-locus%20and%20epistatic%20SNP%20effects%20of%20quantitative%20traits%20in%20genome-wide%20association%20studies&amp;author=L.%20Ma&amp;author=H.B.%20Runesha&amp;author=D.%20Dvorkin&amp;author=J.R.%20Garbe&amp;author=Y.%20Da&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18644146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">London</div> <div class="given-names">N.R.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Dvorkin</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Da</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Detection of SNP epistasis effects of quantitative traits using an extended Kempthorne model</div>. <div class="source">Physiol. Genomics</div> , <div class="volume">28</div>, <div class="fpage">46</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20SNP%20epistasis%20effects%20of%20quantitative%20traits%20using%20an%20extended%20Kempthorne%20model&amp;author=Y.%20Mao&amp;author=N.R.%20London&amp;author=L.%20Ma&amp;author=D.%20Dvorkin&amp;author=Y.%20Da&amp;publication_year=2007&amp;journal=Physiol.%20Genomics&amp;volume=28&amp;pages=46-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/physiolgenomics.00096.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fphysiolgenomics.00096.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fphysiolgenomics.00096.2006"> </span></div> </div></div></div></div></div><div content-id="dsx018-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Jung</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Genome-wide analysis uncovers regulation of long intergenic noncoding RNAs in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">4333</div>–<div class="lpage">4345</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20uncovers%20regulation%20of%20long%20intergenic%20noncoding%20RNAs%20in%20Arabidopsis&amp;author=J.%20Liu&amp;author=C.%20Jung&amp;author=J.%20Xu&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=4333-4345" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.112.102855" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.112.102855" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.112.102855"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.-C.</div>
            </div>, <div class="name string-name"><div class="surname">Liao</div> <div class="given-names">J.-Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Z.-Y.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide screening and functional analysis identify a large number of long noncoding RNAs involved in the sexual reproduction of rice</div>. <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20screening%20and%20functional%20analysis%20identify%20a%20large%20number%20of%20long%20noncoding%20RNAs%20involved%20in%20the%20sexual%20reproduction%20of%20rice&amp;author=Y.-C.%20Zhang&amp;author=J.-Y.%20Liao&amp;author=Z.-Y.%20Li&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=512" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-014-0512-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-014-0512-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-014-0512-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25517485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">J.-H.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.-H.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Qu</div> <div class="given-names">L.-H.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">ChIPBase: a database for decoding the transcriptional regulation of long non-coding RNA and microRNA genes from ChIP-Seq data</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D177</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ChIPBase%3A%20a%20database%20for%20decoding%20the%20transcriptional%20regulation%20of%20long%20non-coding%20RNA%20and%20microRNA%20genes%20from%20ChIP-Seq%20data&amp;author=J.-H.%20Yang&amp;author=J.-H.%20Li&amp;author=S.%20Jiang&amp;author=H.%20Zhou&amp;author=L.-H.%20Qu&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D177-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1060" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1060" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1060"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sati</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Ghosh</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jain</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Scaria</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Sengupta</div> <div class="given-names">S.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Genome-wide analysis reveals distinct patterns of epigenetic features in long non-coding RNA loci</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">10018</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20reveals%20distinct%20patterns%20of%20epigenetic%20features%20in%20long%20non-coding%20RNA%20loci&amp;author=S.%20Sati&amp;author=S.%20Ghosh&amp;author=V.%20Jain&amp;author=V.%20Scaria&amp;author=S.%20Sengupta&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=10018-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guttman</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Amit</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Garber</div> <div class="given-names">M.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals</div>. <div class="source">Nature</div> , <div class="volume">458</div>, <div class="fpage">223</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20signature%20reveals%20over%20a%20thousand%20highly%20conserved%20large%20non-coding%20RNAs%20in%20mammals&amp;author=M.%20Guttman&amp;author=I.%20Amit&amp;author=M.%20Garber&amp;publication_year=2009&amp;journal=Nature&amp;volume=458&amp;pages=223-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07672" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07672" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07672"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Keniry</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Oxley</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Monnier</div> <div class="given-names">P.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">The H19 lincRNA is a developmental reservoir of miR-675 that suppresses growth and Igf1r</div>. <div class="source">Nature Cell Biol</div> ., <div class="volume">14</div>, <div class="fpage">659</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20H19%20lincRNA%20is%20a%20developmental%20reservoir%20of%20miR-675%20that%20suppresses%20growth%20and%20Igf1r&amp;author=A.%20Keniry&amp;author=D.%20Oxley&amp;author=P.%20Monnier&amp;publication_year=2012&amp;journal=Nature%20Cell%20Biol&amp;volume=14&amp;pages=659-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb2521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb2521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb2521"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22684254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hertzberg</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Aspeborg</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schrader</div> <div class="given-names">J.</div></div></div>, et al.  <div class="year">2001</div>, 
<div class="article-title">A transcriptional roadmap to wood formation</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">98</div>, <div class="fpage">14732</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transcriptional%20roadmap%20to%20wood%20formation&amp;author=M.%20Hertzberg&amp;author=H.%20Aspeborg&amp;author=J.%20Schrader&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=98&amp;pages=14732-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.261293398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.261293398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.261293398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11724959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mallory</div>
               <div class="given-names">A.C.</div>
            </div>, <div class="name string-name"><div class="surname">Vaucheret</div> <div class="given-names">H.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Functions of microRNAs and related small RNAs in plants</div>. <div class="source">Nat. Genet</div> ., <div class="volume">38</div>, <div class="fpage">S31</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functions%20of%20microRNAs%20and%20related%20small%20RNAs%20in%20plants&amp;author=A.C.%20Mallory&amp;author=H.%20Vaucheret&amp;publication_year=2006&amp;journal=Nat.%20Genet&amp;volume=38&amp;pages=S31-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1791" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16736022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yoon</div>
               <div class="given-names">J.-H.</div>
            </div>, <div class="name string-name"><div class="surname">Abdelmohsen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Gorospe</div> <div class="given-names">M.</div></div></div>
          <div class="year">2014</div>, Functional interactions among microRNAs and long noncoding RNAs. In: <div class="source">Seminars in Cell and Developmental Biology</div> . <div class="source">Academic Press</div> , <div class="volume">34</div>, <div class="fpage">9</div>–<div class="lpage">14</div>.</div></div></div></div><div content-id="dsx018-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Y.-H.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Chiang</div> <div class="given-names">V.L.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Novel and mechanical stress–responsive microRNAs in <em>Populus trichocarpa</em> that are absent from Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">17</div>, <div class="fpage">2186</div>–<div class="lpage">203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20and%20mechanical%20stress%E2%80%93responsive%20microRNAs%20in%20Populus%20trichocarpa%20that%20are%20absent%20from%20Arabidopsis&amp;author=S.%20Lu&amp;author=Y.-H.%20Sun&amp;author=R.%20Shi&amp;author=C.%20Clark&amp;author=L.%20Li&amp;author=V.L.%20Chiang&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=2186-203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.033456" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.033456" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.033456"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15994906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Y.-Q.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Insights into the mechanism of plant development: interactions of miRNAs pathway with phytohormone response</div>. <div class="source">Biochem. Biophys. Res. Co</div> ., <div class="volume">384</div>, <div class="fpage">1</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20mechanism%20of%20plant%20development%3A%20interactions%20of%20miRNAs%20pathway%20with%20phytohormone%20response&amp;author=Q.%20Liu&amp;author=Y.-Q.%20Chen&amp;publication_year=2009&amp;journal=Biochem.%20Biophys.%20Res.%20Co&amp;volume=384&amp;pages=1-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2009.04.028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2009.04.028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2009.04.028"> </span></div> </div></div></div></div></div><div content-id="dsx018-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ong</div>
               <div class="given-names">S.S.</div>
            </div>, <div class="name string-name"><div class="surname">Wickneswari</div> <div class="given-names">R.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Characterization of microRNAs expressed during secondary wall biosynthesis in Acacia mangium</div>. <div class="source">PloS One</div> , <div class="volume">7</div>, <div class="fpage">e49662</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20microRNAs%20expressed%20during%20secondary%20wall%20biosynthesis%20in%20Acacia%20mangium&amp;author=S.S.%20Ong&amp;author=R.%20Wickneswari&amp;publication_year=2012&amp;journal=PloS%20One&amp;volume=7&amp;pages=e49662" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0049662" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0049662" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0049662"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23251324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">H.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Ptr-miR397a is a negative regulator of laccase genes affecting lignin content in <em>Populus trichocarpa</em></div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">110</div>, <div class="fpage">10848</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ptr-miR397a%20is%20a%20negative%20regulator%20of%20laccase%20genes%20affecting%20lignin%20content%20in%20Populus%20trichocarpa&amp;author=S.%20Lu&amp;author=Q.%20Li&amp;author=H.%20Wei&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=10848-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1308936110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1308936110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1308936110"> </span></div> </div></div></div></div></div><div content-id="dsx018-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tay</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Kats</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salmena</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">Coding-independent regulation of the tumor suppressor PTEN by competing endogenous mRNAs</div>. <div class="source">Cell</div> , <div class="volume">147</div>, <div class="fpage">344</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding-independent%20regulation%20of%20the%20tumor%20suppressor%20PTEN%20by%20competing%20endogenous%20mRNAs&amp;author=Y.%20Tay&amp;author=L.%20Kats&amp;author=L.%20Salmena&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=344-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.09.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.09.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.09.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22000013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shuai</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in <em>Populus trichocarpa</em></div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">65</div>, <div class="fpage">4975</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20and%20functional%20prediction%20of%20novel%20and%20drought-responsive%20lincRNAs%20in%20Populus%20trichocarpa&amp;author=P.%20Shuai&amp;author=D.%20Liang&amp;author=S.%20Tang&amp;publication_year=2014&amp;journal=J.%20Exp.%20Bot&amp;volume=65&amp;pages=4975-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/eru256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Feru256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Feru256"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24948679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ong</div>
               <div class="given-names">S.S.</div>
            </div>, <div class="name string-name"><div class="surname">Wickneswari</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Expression profile of small RNAs in <em>Acacia mangium</em> secondary xylem tissue with contrasting lignin content-potential regulatory sequences in monolignol biosynthetic pathway</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">S13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20profile%20of%20small%20RNAs%20in%20Acacia%20mangium%20secondary%20xylem%20tissue%20with%20contrasting%20lignin%20content-potential%20regulatory%20sequences%20in%20monolignol%20biosynthetic%20pathway&amp;author=S.S.%20Ong&amp;author=R.%20Wickneswari&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=S13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-S3-S13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-S3-S13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-S3-S13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22369296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thumma</div>
               <div class="given-names">B.R.</div>
            </div>, <div class="name string-name"><div class="surname">Nolan</div> <div class="given-names">M.F.</div></div>, <div class="name string-name"><div class="surname">Evans</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Moran</div> <div class="given-names">G.F.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Polymorphisms in cinnamoyl CoA reductase (CCR) are associated with variation in microfibril angle in <em>Eucalyptus</em> spp</div>. <div class="source">Genetics</div> , <div class="volume">171</div>, <div class="fpage">1257</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymorphisms%20in%20cinnamoyl%20CoA%20reductase%20(CCR)%20are%20associated%20with%20variation%20in%20microfibril%20angle%20in%20Eucalyptus%20spp&amp;author=B.R.%20Thumma&amp;author=M.F.%20Nolan&amp;author=R.%20Evans&amp;author=G.F.%20Moran&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=1257-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Solovieff</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Cotsapas</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Purcell</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Smoller</div> <div class="given-names">J.W.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Pleiotropy in complex traits: challenges and strategies</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">14</div>, <div class="fpage">483</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pleiotropy%20in%20complex%20traits%3A%20challenges%20and%20strategies&amp;author=N.%20Solovieff&amp;author=C.%20Cotsapas&amp;author=P.H.%20Lee&amp;author=S.M.%20Purcell&amp;author=J.W.%20Smoller&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet&amp;volume=14&amp;pages=483-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23752797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mackay</div>
               <div class="given-names">T.F.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">Epistasis and quantitative traits: using model organisms to study gene-gene interactions</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">22</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epistasis%20and%20quantitative%20traits%3A%20using%20model%20organisms%20to%20study%20gene-gene%20interactions&amp;author=T.F.%20Mackay&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=22-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24296533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Moore</div>
               <div class="given-names">J.H.</div>
            </div>
          </div>
          <div class="year">2005</div>, 
<div class="article-title">A global view of epistasis</div>. <div class="source">Nat. Genet</div> ., <div class="volume">37</div>, <div class="fpage">13</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20global%20view%20of%20epistasis&amp;author=J.H.%20Moore&amp;publication_year=2005&amp;journal=Nat.%20Genet&amp;volume=37&amp;pages=13-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0105-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0105-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0105-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15624016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Phillips</div>
               <div class="given-names">P.C.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Epistasis—the essential role of gene interactions in the structure and evolution of genetic systems</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">9</div>, <div class="fpage">855</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epistasis%E2%80%94the%20essential%20role%20of%20gene%20interactions%20in%20the%20structure%20and%20evolution%20of%20genetic%20systems&amp;author=P.C.%20Phillips&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet&amp;volume=9&amp;pages=855-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2452" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2452" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2452"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx018-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx018-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gaiti</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Fernandez-Valverde</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Nakanishi</div> <div class="given-names">N.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Dynamic and widespread lncRNA expression in a sponge and the origin of animal complexity</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">2367</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20and%20widespread%20lncRNA%20expression%20in%20a%20sponge%20and%20the%20origin%20of%20animal%20complexity&amp;author=F.%20Gaiti&amp;author=S.L.%20Fernandez-Valverde&amp;author=N.%20Nakanishi&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=2367-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msv117" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsv117" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsv117"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25976353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3769337&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s1.png?Expires=2147483647&amp;Signature=wb7kqOqnjka9TaPq-RPP1IKke2vLxJaN39OFFPGV26aR~gdMArUmL7a~czlkRk5RgBevdj023LqKANJrycvSy9VvtfdPpbhlJtMCuUzY16yICE~tjoAe~L0VpBTEXAifKGLv4sD1rMlgPoFzwFMTYdy8~j0BfmTVp03sIXFI~ZY-O8PRxoZyJ12XMUmB72rkyYlhTVZSOztnABkG0QSr4nRu90T~iJnhcAESXlBYYnq7i-x4lgQ9FdNqPpt6lpDxVacYaLIO2Ke-r4ccToWWMiD9MbFaDM3Niq8GL4d-Zar0uYKxoUuM2xyCRjwQVO~a~brgGaGwCF~kt30Fz3KF~A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s2.png?Expires=2147483647&amp;Signature=b2rklauGA1wSTfsq6TUnvxnz9gnSJlKqd4pDovERscHqr8K6kVI91YXMSe-mlQM7RiPa69lVFjcB1zf8J0qyaoqayMRXkJGfI7i50C4fA6Vgh2Tw4v4pNJPs2vTE9gGyLu5MYmf8i-AcFkUNP3Jum4cNhOnj8BOyecsulNJRU0d4cJS-Duq~tQkDj2Kzu6D7hVHGo~cItFi-1ydqBmSMasBO~FSQW1e3i~bvolX3XKrnUiBa~CIiDoBETXpzUBA2iVqcdnEKxWCMUkHk5APFQnWao1khQsf2buzGCpsIsrYvJFvp7Tcgup0~dSh8dGs~t3WkP885ixSzzVdZ8VJEyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s3.png?Expires=2147483647&amp;Signature=DqVFWpPx53p8qtAhjsJXkVKsmPcuZh8~1MB2zt675ZYz3W1ak5KSBx96djdJiAYLUttsOEQlCtiWKznfGzfS6sPF~mikIb1wZRsvqTMuwrHzv0Zn--JLGFv9rcefjJkoVYmBFGoFv3U5amZF1fNHuUmG47Z72xDCnUIMALUjQuBAEu-GqHAuz9mGaZ6wfaOLxDdxiLrGCwjMZnYrAheguL~rZCXPt98HNvAMprjVisIwtMwj2mRxHif82ctQGunN4RWiIwwyKE7eeKZXwFbH3rzZYupruJERpmgFWSSFs7a7j63-VD3eERhy040eUlllVXmSwFG7D-PwkZOs6-HyVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s4.png?Expires=2147483647&amp;Signature=K0De6IcOXLjhqUUWD4sWIFvK~JkKAa0HN6nVMp7vWUE9fIAcdCoU9hT4AeXOaLBeughr98m1D2josjVS86SO7ldnSkEVayZ~yb~1~2O22fAt3vQz4tIQPtzKlhV1XslOrWTgKk4azWPcJ-hlXy0m0oJAhj4V7wfh46KOYmqeYtPGic4iUZ1NAc8LhP5HLVnT4o8drGwxFwYzbK3sfWl3qdD~L13-Qw5n8761KPVNd28F8HHYO~wzd-IW0Th3IVW3v9XmCkPtl2Ln0ZG6GausNbOXLKjfN5bdcF1j7p82ujpyksvcrpzNXKhE407DulktJ6YIgAPIQK7mLGnVkgZuWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s5.png?Expires=2147483647&amp;Signature=AXD4FXRhrF-Q0qtF9uZQpadhFN63G6ym12aPf4SkXhCP6Nh9AXeZAVMKRZC85TJDHuVCFpxczaAQZra~m-MTZpHEgCFg3jpeTZaBuT0lgvD559qEPDV8vGn5WfKPRQrPkrm3e-duk7Mr~3ppymNwjidyTQ~QmF2BVxwBrNpSmiGtIhg7CKuYk~JcW4Q7CvUczI-vLmJk1ftEv9kS5nUbj1qoVDyXKGF0fjkoLlCI9NQveVQ4FNnZ0zRjpdBSs37qyDtEVkhVw3f2DDoPkYbIJC2REGdEi0meK0zveHSLSzwENB-sM1qrqsHGZinSDN1g02Du2zDWN20d392w2nOhxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_figure_s6.png?Expires=2147483647&amp;Signature=lQgfSYozYPR4A5JFVV0O9l1yKth3QE1dJvkKmoZ2~bdl4hhAu8zV52YVzMXqdFmN3tNbUqgySvAZ4cjqHrKYXJE7drMAiL85QkP8QduBiT92lAHAV~1m~gJOd7sq3eIuaR-CtbNG78-0HVnGrD7S6V7r1Z9VoqviOiassNmQUoo19n5GgTrjNTG0AAOdyzT3-3qy3Ktx~TDVqXa8qr02plaKMCDnTs5A9CyvcxtTTByrJobPgk5MFj4G3HZTjvhZhH3cp4o3vVSMqoFkkMZ4mfb0cOtyRc2fBdr5DVwNmqfPjnBUFy2yD7foo7HHdLHB79bjkQsUYHKF0pKWwi7qcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supplementary_data_(data_s1-s15).xls?Expires=2147483647&amp;Signature=E3Cwkc~MIEAIiEQNGxpsYYzhBpa2iy0vJaluHhR3VZ3dX1Y2WBFrx1WQP8473lpltXbmxQPDEVcQpisYdU4QzZ3xkROHWnpLC5YGcUYmYX9x6oDM7WVppYF4Mwscb27Wm6Sgvz6-vGvP~HJ9hTQ-UoJMdXE29BMCyl4Ha0DD65yw4QKOFPSRE6HCFeLn7opmRgV5~jGHrUzM60zJ5fvCnoqodxVIrMEs6exnLDFG2CFLyUtQrlXOQpSn1aaoYFvFZ1OzBN4XBwGp5Ca9zsmUHZ2KMc9L5rQKaetpP1q-QvRhdA1zUnimKFeK5~WAzS0GttQB6VLJ79OG054rz07vGg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s1.doc?Expires=2147483647&amp;Signature=e~TbIPr9vwUcsfRYa7izNq-gfa-sdKrSwrHTls3UvvsfSBwiwWx73FJmSHi1PIUZMwEDX2bHTBsxg0ooE0Irz8F8KR4VNNDx1H2ZckNzQqWI66QjW-DyyMYKeJxP5PxEOoYK6H~bc3EXhhRxgMSCf0AhxmKWO5MLPUfCnhE-IXh1OPonCPgoCdok0c0bOOTRpuhZLpUhwj~cyRmemyWigz-X9DgOy~34JBdQJuV2cJOJrOh-5SjjfPnCEksar9cuEsLJnah4Gh-T6lCo~y2eJp02puK8DA34Kh~mXbF43Z6gdXC4tt8JEJbnyFDko6~dIB1ep0eGjIqBO5PJl1WCgA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s2.doc?Expires=2147483647&amp;Signature=bDPDShCJ1o-mq5qpuoPzn8N6WC2KQG4ci8VzA5FZTqzdEDLFAB3n-67KBQqCE2O1xnXG5PAVd-eY5soPNJfIrTgzVJzUkP2yU-hUVMcD15x8KDnF1ahD8QeQJb2Iw4I~ZIA8sxcgNCoWJvXTr~TgkmFAnzgiB580nDw-EwSuCYru635MZitViYmfxhyBFXiNta1Dq5nY4nIqN6gzTwrnMZDuJFElm4YQc86cRRlkWs9pYad-~9vPCA~FrKcz72j994viUtNtphFZiqfovSSiwLzENFbb6vr2~2VRQNS549T3CzG6kampMoCoiZDvQvpUgq~8KIarxPhDw-johJKHfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx018/2/dsx018_supp_table_s3.doc?Expires=2147483647&amp;Signature=sSIdxdWqmP~1ooiKyMi20myhpvKEiF53M3TfAa8PhnQV2bG7ugBM0UC2x4cmjneC2bOfT1CyiaBpU23Qb0viM4Bp33SCgJIVwWShkKbCaoI1ACeXDK46ey4NQ1IyXCW2d6kaAMbinYBlB1EB2K23C3XY3JGLmoAAhxB8nB3luwpkToj2T7XjVNlbedfWYLkxDDmBMd0luCQroBuOOkmnby2hwnn68SyWLPjfV6HWktQF2~w3DWPxPBQ48CQPZs9Zp-s9DYSJOXxku1I3VHZAb7MbrnBaGoanICgQW7PFqAPQRGJjCBI5V-iAbFW7g9CV5sdI2quBYGtoSMM7qLd5Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">771</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3769337"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f5%2f473%2f3769337">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000413018400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F5%2F473%2F3769337" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3769337'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Identification and allelic dissection uncover roles of lncRNAs in secondary growth of Populus tomentosa" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f473%2f3769337">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

