

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/2/153/403337","@type":"ScholarlyArticle","name":"Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes  †  ","datePublished":"2013-11-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2014-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst047","keywords":["domestication","resequencing","soybean","variation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/2/153/403337","author":[{"name":"Chung, Won-Hyong","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Jeong, Namhee","affiliation":"2  Bio-Evaluation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Cheongwon, Chungbuk 363-883 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Jiwoong","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Woo Kyu","affiliation":"2  Bio-Evaluation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Cheongwon, Chungbuk 363-883 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Yun-Gyeong","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Sang-Heon","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Yoon, Woongchang","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Jin-Hyun","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    4  Department of Genetic Engineering ,  Dong-A University ,  Busan 604-714 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Ik-Young","affiliation":"5  National Instrumentation Center for Environmental Management ,  Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Hong-Kyu","affiliation":"4  Department of Genetic Engineering ,  Dong-A University ,  Busan 604-714 ,  Republic of Korea","@type":"Person"},{"name":"Moon, Jung-Kyung","affiliation":"6  National Institute of Crop Science, Rural Development Administration ,  Suwon 441-857 ,  Republic of Korea","@type":"Person"},{"name":"Kim, Namshin","affiliation":"1  Korean Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejeon 305-806 ,  Republic of Korea    3  Department of Bioinformatics ,  University of Science and Technology ,  Daejeon 305-806 ,  Republic of Korea","@type":"Person"},{"name":"Jeong, Soon-Chun","affiliation":"2  Bio-Evaluation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Cheongwon, Chungbuk 363-883 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Despite the importance of soybean as a major crop, genome-wide variation and evolution of cultivated soybeans are largely unknown. Here, we catalogue","pageStart":"153","pageEnd":"167","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes  †  ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes  †  " />
    <meta property="og:description" content="Abstract.  Despite the importance of soybean as a major crop, genome-wide variation and evolution of cultivated soybeans are largely unknown. Here, we catalogue" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/2/153/403337" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-11-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chung, Won-Hyong" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Jeong, Namhee" /><meta name="citation_author_institution" content="Bio-Evaluation Center, Korea Research Institute of Bioscience and Biotechnology, Cheongwon, Chungbuk 363-883, Republic of Korea" /><meta name="citation_author" content="Kim, Jiwoong" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Lee, Woo Kyu" /><meta name="citation_author_institution" content="Bio-Evaluation Center, Korea Research Institute of Bioscience and Biotechnology, Cheongwon, Chungbuk 363-883, Republic of Korea" /><meta name="citation_author" content="Lee, Yun-Gyeong" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Lee, Sang-Heon" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Yoon, Woongchang" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Kim, Jin-Hyun" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Genetic Engineering, Dong-A University, Busan 604-714, Republic of Korea" /><meta name="citation_author" content="Choi, Ik-Young" /><meta name="citation_author_institution" content="National Instrumentation Center for Environmental Management, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Choi, Hong-Kyu" /><meta name="citation_author_institution" content="Department of Genetic Engineering, Dong-A University, Busan 604-714, Republic of Korea" /><meta name="citation_author" content="Moon, Jung-Kyung" /><meta name="citation_author_institution" content="National Institute of Crop Science, Rural Development Administration, Suwon 441-857, Republic of Korea" /><meta name="citation_author" content="Kim, Namshin" /><meta name="citation_author_institution" content="Korean Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author_institution" content="Department of Bioinformatics, University of Science and Technology, Daejeon 305-806, Republic of Korea" /><meta name="citation_author" content="Jeong, Soon-Chun" /><meta name="citation_author_institution" content="Bio-Evaluation Center, Korea Research Institute of Bioscience and Biotechnology, Cheongwon, Chungbuk 363-883, Republic of Korea" /><meta name="citation_title" content="Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes" /><meta name="citation_firstpage" content="153" /><meta name="citation_lastpage" content="167" /><meta name="citation_doi" content="10.1093/dnares/dst047" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2014/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Archaeological soybean (Glycine max) in East Asia: does size matter?; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e26720" /><meta name="citation_reference" content="citation_title=Diversity of chloroplast DNA SSRs in wild and cultivated soybeans: evidence for multiple origins of cultivated soybean; Theor. Appl. Genet.;  citation_year=2002;  citation_volume=105;  citation_pages=645-53; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies for common diseases and complex traits; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=95-108; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies of 14 agronomic traits in rice landraces; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=961-7; " /><meta name="citation_reference" content="citation_title=Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection; Nat. Genet.;  citation_year=2010;  citation_volume=42;  citation_pages=1053-9; " /><meta name="citation_reference" content="citation_title=A map of rice genome variation reveals the origin of cultivated rice; Nature;  citation_year=2012;  citation_volume=490;  citation_pages=497-501; " /><meta name="citation_reference" content="citation_title=Comparative population genomics of maize domestication and improvement; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=808-11; " /><meta name="citation_reference" content="citation_title=Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=105-11; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=956-63; " /><meta name="citation_reference" content="citation_title=Multiple reference genomes and transcriptomes for Arabidopsis thaliana; Nature;  citation_year=2011;  citation_volume=477;  citation_pages=419-23; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing and intensive analysis of the undomesticated soybean (Glycine soja Sieb. and Zucc.) genome; Proc Natl Acad Sci USA;  citation_year=2010;  citation_volume=107;  citation_pages=22032-7; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=The sequence and de novo assembly of the giant panda genome; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=311-7; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_title=Dynamic genetic features of chromosomes revealed by comparison of soybean genetic and sequence-based physical maps; Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=1103-19; " /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler; Gigascience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=AUGUSTUS at EGASP: using EST, protein and genomic alignments for improved gene prediction in the human genome; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_issue=Suppl 1;  citation_pages=S11" /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=InterProScan—an integration platform for the signature-recognition methods in InterPro; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=847-8; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D211-22; " /><meta name="citation_reference" content="citation_title=Structure, expression, and mapping of two nodule-specific genes identified by mining public soybean EST databases; Gene;  citation_year=2006;  citation_volume=383;  citation_pages=71-80; " /><meta name="citation_reference" content="citation_title=Discovery and genotyping of genome structural polymorphism by sequencing on a population scale; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=269-76; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Inferring weak population structure with the assistance of sample group information; Mol. Ecol. Resour.;  citation_year=2009;  citation_volume=9;  citation_pages=1322-32; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies; Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1567-87; " /><meta name="citation_reference" content="citation_title=Haploview: analysis and visualization of LD and haplotype maps; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=263-5; " /><meta name="citation_reference" content="citation_title=On the number of segregating sites in genetical models without recombination; Theor. Popul. Biol.;  citation_year=1975;  citation_volume=7;  citation_pages=256-76; " /><meta name="citation_reference" content="citation_title=Evolutionary relationship of DNA sequences in finite populations; Genetics;  citation_year=1983;  citation_volume=105;  citation_pages=437-60; " /><meta name="citation_reference" content="citation_title=Estimation of levels of gene flow from DNA sequence data; Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=583-9; " /><meta name="citation_reference" content="citation_title=Evolutionary studies of the soybean: the frequency and distribution of alleles among collections of Glycine max and G. soja of various origin; Euphytica;  citation_year=1981;  citation_volume=30;  citation_pages=55-64; " /><meta name="citation_reference" content="citation_title=Speciation and cytogenetics; citation_publisher=American Society of Agronomy, Madison, Wisconsin; Soybeans: improvement, production, and uses;  citation_year=2004;  citation_pages=97-136; " /><meta name="citation_reference" content="citation_title=Glycine max (L.) Merr. subsp. soja (Sieb and Zucc.) Ohashi, comb. nov; J. Jpn. Bot.;  citation_year=1982;  citation_volume=57;  citation_pages=30" /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=Soybean germplasm pools in Asia revealed by nuclear SSRs; Theor. Appl. Genet.;  citation_year=2003;  citation_volume=106;  citation_pages=445-53; " /><meta name="citation_reference" content="citation_title=The Genomes of Oryza sativa: a history of duplications; PLoS Biol.;  citation_year=2005;  citation_volume=3;  citation_pages=e38" /><meta name="citation_reference" content="citation_title=Nuclear DNA content of some important plant species; Plant Mol. Biol. Rep.;  citation_year=1991;  citation_volume=9;  citation_pages=208-18; " /><meta name="citation_reference" content="citation_title=Statistical method for testing the neutral mutation hypothesis by DNA polymorphism; Genetics;  citation_year=1989;  citation_volume=123;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=Overexpressing AtPAP15 enhances phosphorus efficiency in soybean; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=233-40; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of purple acid phosphatase phytases from wheat, barley, maize, and rice; Plant Physiol.;  citation_year=2011;  citation_volume=156;  citation_pages=1087-100; " /><meta name="citation_reference" content="citation_title=Improving emergence of modified phosphorus composition soybeans: genotypes, germplasm, environments, and selection; Crop Sci.;  citation_year=2011;  citation_volume=51;  citation_pages=1946-55; " /><meta name="citation_reference" content="citation_title=QTL mapping of domestication-related traits in soybean (Glycine max); Ann. Bot.;  citation_year=2007;  citation_volume=100;  citation_pages=1027-38; " /><meta name="citation_reference" content="citation_title=Highly variable patterns of linkage disequilibrium in multiple soybean populations; Genetics;  citation_year=2007;  citation_volume=175;  citation_pages=1937-44; " /><meta name="citation_reference" content="citation_title=Fast model-based estimation of ancestry in unrelated individuals; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1655-64; " /><meta name="citation_reference" content="citation_title=The effects of artificial selection on the maize genome; Science;  citation_year=2005;  citation_volume=308;  citation_pages=1310-4; " /><meta name="citation_reference" content="citation_title=Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds; Science;  citation_year=2009;  citation_volume=324;  citation_pages=528-32; " /><meta name="citation_reference" content="citation_title=The genomic signature of dog domestication reveals adaptation to a starch-rich diet; Nature;  citation_year=2013;  citation_volume=495;  citation_pages=360-4; " /><meta name="citation_reference" content="citation_title=Corn and humans: recombination and linkage disequilibrium in two genomes of similar size; Trends Genet.;  citation_year=2004;  citation_volume=20;  citation_pages=103-11; " /><meta name="citation_reference" content="citation_title=A first-generation haplotype map of maize; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1115-7; " /><meta name="citation_reference" content="citation_title=Recombination and linkage disequilibrium in Arabidopsis thaliana; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=1151-5; " /><meta name="citation_reference" content="citation_title=The molecular genetics of crop domestication; Cell;  citation_year=2006;  citation_volume=127;  citation_pages=1309-21; " /><meta name="citation_reference" content="citation_title=The origin of the naked grains of maize; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=714-9; " /><meta name="citation_reference" content="citation_title=Genetic control of rice plant architecture under domestication; Nat. Genet.;  citation_year=2008;  citation_volume=40;  citation_pages=1365-9; " /><meta name="citation_reference" content="citation_title=Using membrane transporters to improve crops for sustainable food production; Nature;  citation_year=2013;  citation_volume=497;  citation_pages=60-6; " /><meta name="citation_reference" content="citation_title=The war of the whorls: genetic interactions controlling flower development; Nature;  citation_year=1991;  citation_volume=353;  citation_pages=31-7; " /><meta name="citation_reference" content="citation_title=A distinct mechanism regulating a pollen-specific guanine nucleotide exchange factor for the small GTPase Rop in Arabidopsis thaliana; Proc Natl Acad Sci USA;  citation_year=2007;  citation_volume=104;  citation_pages=18830-5; " /><meta name="citation_reference" content="citation_title=Transcript profile analyses of maize silks reveal effective activation of genes involved in microtubule-based movement, ubiquitin-dependent protein degradation, and transport in the pollination process; PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e53545" /><meta name="citation_reference" content="citation_title=TCP genes: a family snapshot ten years later; Trends Plant Sci.;  citation_year=2010;  citation_volume=15;  citation_pages=31-9; " /><meta name="citation_reference" content="citation_title=Molecular footprints of domestication and improvement in soybean revealed by whole genome re-sequencing; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=579" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/2/153/1126808/dst047.pdf" /><meta name="description" content="Abstract.  Despite the importance of soybean as a major crop, genome-wide variation and evolution of cultivated soybeans are largely unknown. Here, we catalogue" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/2/153/403337" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/2/153/403337" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/2/153/403337");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst047';
    </script>
<title>
	Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes† | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./403337" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdeOATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2h1bmcsIFdvbi1IeW9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhbiBCaW9pbmZvcm1hdGlvbiBDZW50ZXIsIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBEYWVqZW9uIDMwNS04MDYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmVvbmcsIE5hbWhlZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpby1FdmFsdWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIENoZW9uZ3dvbiwgQ2h1bmdidWsgMzYzLTg4MywgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEppd29vbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLb3JlYW4gQmlvaW5mb3JtYXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgRGFlamVvbiAzMDUtODA2LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlZSwgV29vIEt5dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpby1FdmFsdWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIENoZW9uZ3dvbiwgQ2h1bmdidWsgMzYzLTg4MywgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIFl1bi1HeWVvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLb3JlYW4gQmlvaW5mb3JtYXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgRGFlamVvbiAzMDUtODA2LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIFVuaXZlcnNpdHkgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgRGFlamVvbiAzMDUtODA2LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxlZSwgU2FuZy1IZW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZb29uLCBXb29uZ2NoYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEppbi1IeXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEdlbmV0aWMgRW5naW5lZXJpbmcsIERvbmctQSBVbml2ZXJzaXR5LCBCdXNhbiA2MDQtNzE0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob2ksIElrLVlvdW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdHJ1bWVudGF0aW9uIENlbnRlciBmb3IgRW52aXJvbm1lbnRhbCBNYW5hZ2VtZW50LCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob2ksIEhvbmctS3l1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHZW5ldGljIEVuZ2luZWVyaW5nLCBEb25nLUEgVW5pdmVyc2l0eSwgQnVzYW4gNjA0LTcxNCwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb29uLCBKdW5nLUt5dW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIENyb3AgU2NpZW5jZSwgUnVyYWwgRGV2ZWxvcG1lbnQgQWRtaW5pc3RyYXRpb24sIFN1d29uIDQ0MS04NTcsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBOYW1zaGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS29yZWFuIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBVbml2ZXJzaXR5IG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIERhZWplb24gMzA1LTgwNiwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKZW9uZywgU29vbi1DaHVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvLUV2YWx1YXRpb24gQ2VudGVyLCBLb3JlYSBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgQ2hlb25nd29uLCBDaHVuZ2J1ayAzNjMtODgzLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUG9wdWxhdGlvbiBTdHJ1Y3R1cmUgYW5kIERvbWVzdGljYXRpb24gUmV2ZWFsZWQgYnkgSGlnaC1EZXB0aCBSZXNlcXVlbmNpbmcgb2YgS29yZWFuIEN1bHRpdmF0ZWQgYW5kIFdpbGQgU295YmVhbiBHZW5vbWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyY2hhZW9sb2dpY2FsIHNveWJlYW4gKEdseWNpbmUgbWF4KSBpbiBFYXN0IEFzaWE6IGRvZXMgc2l6ZSBtYXR0ZXI/OyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI2NzIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IG9mIGNobG9yb3BsYXN0IEROQSBTU1JzIGluIHdpbGQgYW5kIGN1bHRpdmF0ZWQgc295YmVhbnM6IGV2aWRlbmNlIGZvciBtdWx0aXBsZSBvcmlnaW5zIG9mIGN1bHRpdmF0ZWQgc295YmVhbjsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz02NDUtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXMgZm9yIGNvbW1vbiBkaXNlYXNlcyBhbmQgY29tcGxleCB0cmFpdHM7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTk1LTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZGllcyBvZiAxNCBhZ3Jvbm9taWMgdHJhaXRzIGluIHJpY2UgbGFuZHJhY2VzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTk2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXNlcXVlbmNpbmcgb2YgMzEgd2lsZCBhbmQgY3VsdGl2YXRlZCBzb3liZWFuIGdlbm9tZXMgaWRlbnRpZmllcyBwYXR0ZXJucyBvZiBnZW5ldGljIGRpdmVyc2l0eSBhbmQgc2VsZWN0aW9uOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtYXAgb2YgcmljZSBnZW5vbWUgdmFyaWF0aW9uIHJldmVhbHMgdGhlIG9yaWdpbiBvZiBjdWx0aXZhdGVkIHJpY2U7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTA7ICBjaXRhdGlvbl9wYWdlcz00OTctNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBwb3B1bGF0aW9uIGdlbm9taWNzIG9mIG1haXplIGRvbWVzdGljYXRpb24gYW5kIGltcHJvdmVtZW50OyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTgwOC0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZXF1ZW5jaW5nIDUwIGFjY2Vzc2lvbnMgb2YgY3VsdGl2YXRlZCBhbmQgd2lsZCByaWNlIHlpZWxkcyBtYXJrZXJzIGZvciBpZGVudGlmeWluZyBhZ3Jvbm9taWNhbGx5IGltcG9ydGFudCBnZW5lczsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2YgbXVsdGlwbGUgQXJhYmlkb3BzaXMgdGhhbGlhbmEgcG9wdWxhdGlvbnM7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9OTU2LTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSByZWZlcmVuY2UgZ2Vub21lcyBhbmQgdHJhbnNjcmlwdG9tZXMgZm9yIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc3OyAgY2l0YXRpb25fcGFnZXM9NDE5LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2luZyBhbmQgaW50ZW5zaXZlIGFuYWx5c2lzIG9mIHRoZSB1bmRvbWVzdGljYXRlZCBzb3liZWFuIChHbHljaW5lIHNvamEgU2llYi4gYW5kIFp1Y2MuKSBnZW5vbWU7IFByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MjIwMzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmYXN0LCBsb2NrLWZyZWUgYXBwcm9hY2ggZm9yIGVmZmljaWVudCBwYXJhbGxlbCBjb3VudGluZyBvZiBvY2N1cnJlbmNlcyBvZiBrLW1lcnM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NzY0LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYW5kIGRlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIGdpYW50IHBhbmRhIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTMxMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9tZSBBbmFseXNpcyBUb29sa2l0OiBhIE1hcFJlZHVjZSBmcmFtZXdvcmsgZm9yIGFuYWx5emluZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmcmFtZXdvcmsgZm9yIHZhcmlhdGlvbiBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIGRhdGE7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NDkxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgZ2VuZXRpYyBmZWF0dXJlcyBvZiBjaHJvbW9zb21lcyByZXZlYWxlZCBieSBjb21wYXJpc29uIG9mIHNveWJlYW4gZ2VuZXRpYyBhbmQgc2VxdWVuY2UtYmFzZWQgcGh5c2ljYWwgbWFwczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjY7ICBjaXRhdGlvbl9wYWdlcz0xMTAzLTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQZGVub3ZvMjogYW4gZW1waXJpY2FsbHkgaW1wcm92ZWQgbWVtb3J5LWVmZmljaWVudCBzaG9ydC1yZWFkIGRlIG5vdm8gYXNzZW1ibGVyOyBHaWdhc2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BVUdVU1RVUyBhdCBFR0FTUDogdXNpbmcgRVNULCBwcm90ZWluIGFuZCBnZW5vbWljIGFsaWdubWVudHMgZm9yIGltcHJvdmVkIGdlbmUgcHJlZGljdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsIDE7ICBjaXRhdGlvbl9wYWdlcz1TMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW7igJRhbiBpbnRlZ3JhdGlvbiBwbGF0Zm9ybSBmb3IgdGhlIHNpZ25hdHVyZS1yZWNvZ25pdGlvbiBtZXRob2RzIGluIEludGVyUHJvOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTg0Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1EMjExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUsIGV4cHJlc3Npb24sIGFuZCBtYXBwaW5nIG9mIHR3byBub2R1bGUtc3BlY2lmaWMgZ2VuZXMgaWRlbnRpZmllZCBieSBtaW5pbmcgcHVibGljIHNveWJlYW4gRVNUIGRhdGFiYXNlczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODM7ICBjaXRhdGlvbl9wYWdlcz03MS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIG9mIGdlbm9tZSBzdHJ1Y3R1cmFsIHBvbHltb3JwaGlzbSBieSBzZXF1ZW5jaW5nIG9uIGEgcG9wdWxhdGlvbiBzY2FsZTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0yNjktNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcnJpbmcgd2VhayBwb3B1bGF0aW9uIHN0cnVjdHVyZSB3aXRoIHRoZSBhc3Npc3RhbmNlIG9mIHNhbXBsZSBncm91cCBpbmZvcm1hdGlvbjsgTW9sLiBFY29sLiBSZXNvdXIuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzIyLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOiBsaW5rZWQgbG9jaSBhbmQgY29ycmVsYXRlZCBhbGxlbGUgZnJlcXVlbmNpZXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhhcGxvdmlldzogYW5hbHlzaXMgYW5kIHZpc3VhbGl6YXRpb24gb2YgTEQgYW5kIGhhcGxvdHlwZSBtYXBzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI2My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgbnVtYmVyIG9mIHNlZ3JlZ2F0aW5nIHNpdGVzIGluIGdlbmV0aWNhbCBtb2RlbHMgd2l0aG91dCByZWNvbWJpbmF0aW9uOyBUaGVvci4gUG9wdWwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTc1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNTYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSByZWxhdGlvbnNoaXAgb2YgRE5BIHNlcXVlbmNlcyBpbiBmaW5pdGUgcG9wdWxhdGlvbnM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTQzNy02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBsZXZlbHMgb2YgZ2VuZSBmbG93IGZyb20gRE5BIHNlcXVlbmNlIGRhdGE7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTU4My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgc3R1ZGllcyBvZiB0aGUgc295YmVhbjogdGhlIGZyZXF1ZW5jeSBhbmQgZGlzdHJpYnV0aW9uIG9mIGFsbGVsZXMgYW1vbmcgY29sbGVjdGlvbnMgb2YgR2x5Y2luZSBtYXggYW5kIEcuIHNvamEgb2YgdmFyaW91cyBvcmlnaW47IEV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWF0aW9uIGFuZCBjeXRvZ2VuZXRpY3M7IGNpdGF0aW9uX3B1Ymxpc2hlcj1BbWVyaWNhbiBTb2NpZXR5IG9mIEFncm9ub215LCBNYWRpc29uLCBXaXNjb25zaW47IFNveWJlYW5zOiBpbXByb3ZlbWVudCwgcHJvZHVjdGlvbiwgYW5kIHVzZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdseWNpbmUgbWF4IChMLikgTWVyci4gc3Vic3AuIHNvamEgKFNpZWIgYW5kIFp1Y2MuKSBPaGFzaGksIGNvbWIuIG5vdjsgSi4gSnBuLiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHBhbGFlb3BvbHlwbG9pZCBzb3liZWFuOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDYzOyAgY2l0YXRpb25fcGFnZXM9MTc4LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb3liZWFuIGdlcm1wbGFzbSBwb29scyBpbiBBc2lhIHJldmVhbGVkIGJ5IG51Y2xlYXIgU1NSczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz00NDUtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHZW5vbWVzIG9mIE9yeXphIHNhdGl2YTogYSBoaXN0b3J5IG9mIGR1cGxpY2F0aW9uczsgUExvUyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBETkEgY29udGVudCBvZiBzb21lIGltcG9ydGFudCBwbGFudCBzcGVjaWVzOyBQbGFudCBNb2wuIEJpb2wuIFJlcC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTIwOC0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgbWV0aG9kIGZvciB0ZXN0aW5nIHRoZSBuZXV0cmFsIG11dGF0aW9uIGh5cG90aGVzaXMgYnkgRE5BIHBvbHltb3JwaGlzbTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9NTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVyZXhwcmVzc2luZyBBdFBBUDE1IGVuaGFuY2VzIHBob3NwaG9ydXMgZWZmaWNpZW5jeSBpbiBzb3liZWFuOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0yMzMtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgcHVycGxlIGFjaWQgcGhvc3BoYXRhc2UgcGh5dGFzZXMgZnJvbSB3aGVhdCwgYmFybGV5LCBtYWl6ZSwgYW5kIHJpY2U7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTEwODctMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgZW1lcmdlbmNlIG9mIG1vZGlmaWVkIHBob3NwaG9ydXMgY29tcG9zaXRpb24gc295YmVhbnM6IGdlbm90eXBlcywgZ2VybXBsYXNtLCBlbnZpcm9ubWVudHMsIGFuZCBzZWxlY3Rpb247IENyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTE5NDYtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVFUTCBtYXBwaW5nIG9mIGRvbWVzdGljYXRpb24tcmVsYXRlZCB0cmFpdHMgaW4gc295YmVhbiAoR2x5Y2luZSBtYXgpOyBBbm4uIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTAyNy0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaGx5IHZhcmlhYmxlIHBhdHRlcm5zIG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gbXVsdGlwbGUgc295YmVhbiBwb3B1bGF0aW9uczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc1OyAgY2l0YXRpb25fcGFnZXM9MTkzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBtb2RlbC1iYXNlZCBlc3RpbWF0aW9uIG9mIGFuY2VzdHJ5IGluIHVucmVsYXRlZCBpbmRpdmlkdWFsczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZWZmZWN0cyBvZiBhcnRpZmljaWFsIHNlbGVjdGlvbiBvbiB0aGUgbWFpemUgZ2Vub21lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwODsgIGNpdGF0aW9uX3BhZ2VzPTEzMTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgc3VydmV5IG9mIFNOUCB2YXJpYXRpb24gdW5jb3ZlcnMgdGhlIGdlbmV0aWMgc3RydWN0dXJlIG9mIGNhdHRsZSBicmVlZHM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9NTI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21pYyBzaWduYXR1cmUgb2YgZG9nIGRvbWVzdGljYXRpb24gcmV2ZWFscyBhZGFwdGF0aW9uIHRvIGEgc3RhcmNoLXJpY2ggZGlldDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NTsgIGNpdGF0aW9uX3BhZ2VzPTM2MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JuIGFuZCBodW1hbnM6IHJlY29tYmluYXRpb24gYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gdHdvIGdlbm9tZXMgb2Ygc2ltaWxhciBzaXplOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZpcnN0LWdlbmVyYXRpb24gaGFwbG90eXBlIG1hcCBvZiBtYWl6ZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjY7ICBjaXRhdGlvbl9wYWdlcz0xMTE1LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29tYmluYXRpb24gYW5kIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTE1MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9sZWN1bGFyIGdlbmV0aWNzIG9mIGNyb3AgZG9tZXN0aWNhdGlvbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjc7ICBjaXRhdGlvbl9wYWdlcz0xMzA5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2luIG9mIHRoZSBuYWtlZCBncmFpbnMgb2YgbWFpemU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz03MTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjb250cm9sIG9mIHJpY2UgcGxhbnQgYXJjaGl0ZWN0dXJlIHVuZGVyIGRvbWVzdGljYXRpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTM2NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBtZW1icmFuZSB0cmFuc3BvcnRlcnMgdG8gaW1wcm92ZSBjcm9wcyBmb3Igc3VzdGFpbmFibGUgZm9vZCBwcm9kdWN0aW9uOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk3OyAgY2l0YXRpb25fcGFnZXM9NjAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHdhciBvZiB0aGUgd2hvcmxzOiBnZW5ldGljIGludGVyYWN0aW9ucyBjb250cm9sbGluZyBmbG93ZXIgZGV2ZWxvcG1lbnQ7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTM7ICBjaXRhdGlvbl9wYWdlcz0zMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRpc3RpbmN0IG1lY2hhbmlzbSByZWd1bGF0aW5nIGEgcG9sbGVuLXNwZWNpZmljIGd1YW5pbmUgbnVjbGVvdGlkZSBleGNoYW5nZSBmYWN0b3IgZm9yIHRoZSBzbWFsbCBHVFBhc2UgUm9wIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBQcm9jIE5hdGwgQWNhZCBTY2kgVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTE4ODMwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgcHJvZmlsZSBhbmFseXNlcyBvZiBtYWl6ZSBzaWxrcyByZXZlYWwgZWZmZWN0aXZlIGFjdGl2YXRpb24gb2YgZ2VuZXMgaW52b2x2ZWQgaW4gbWljcm90dWJ1bGUtYmFzZWQgbW92ZW1lbnQsIHViaXF1aXRpbi1kZXBlbmRlbnQgcHJvdGVpbiBkZWdyYWRhdGlvbiwgYW5kIHRyYW5zcG9ydCBpbiB0aGUgcG9sbGluYXRpb24gcHJvY2VzczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU1MzU0NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRDUCBnZW5lczogYSBmYW1pbHkgc25hcHNob3QgdGVuIHllYXJzIGxhdGVyOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBmb290cHJpbnRzIG9mIGRvbWVzdGljYXRpb24gYW5kIGltcHJvdmVtZW50IGluIHNveWJlYW4gcmV2ZWFsZWQgYnkgd2hvbGUgZ2Vub21lIHJlLXNlcXVlbmNpbmc7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTU3OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8yLzE1My8xMTI2ODA4L2RzdDA0Ny5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRGVzcGl0ZSB0aGUgaW1wb3J0YW5jZSBvZiBzb3liZWFuIGFzIGEgbWFqb3IgY3JvcCwgZ2Vub21lLXdpZGUgdmFyaWF0aW9uIGFuZCBldm9sdXRpb24gb2YgY3VsdGl2YXRlZCBzb3liZWFucyBhcmUgbGFyZ2VseSB1bmtub3duLiBIZXJlLCB3ZSBjYXRhbG9ndWUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQwMzMzNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRFWKyABijkqjC64lxb7g2n6eNmmQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f153%2f403337" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f153%2f403337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15712" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/2/1/m_cover.gif?Expires=2147483647&amp;Signature=C2ZH5dkUQEctPPta7W831-4mGKGGgEq~Aqbfql5zal9lq-CH7ZJcQtnaLb-2~TFW5o1XuWK-If8uoA0C0x3E8Xqh7vx6q7XeJXntpLTv0xMKSiGPs8b5JT8WwzNVzF-MVRjRIurpCdYeMF6bhbaWGNZLajJ34JAWHyXvGj72zxR2I~0JkQXfyp1Fd2IS9rmTmm3EZ-Vtcf0zL2z0kWasBqYnVob3EwZTj2l448hvYoYYZgYfsgU2j1vdpw3GUsXrCjCI5GsZ1ht-EZrYHZzJ3zkCMOPIc~J0lFrmNiix6fEEbm~y5NNGWoTY3GIiDMTmJpXRxBN715elzocAQQfDUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/2">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5954607">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954607">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954611">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954611">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954615">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954678">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954678">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954710">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954710">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954712">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954712">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954714">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954714">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954716">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954716">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954718">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954718">Acknowledgments</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5954720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5954720">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/2/141/402760">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/2/169/404005">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes<sup><a reveal-id="AN1" data-open="AN1" class="link link-ref link-reveal xref-fn">†</a></sup>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Won-Hyong Chung</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Won-Hyong+Chung">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chung W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chung W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Won-Hyong Chung</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Namhee Jeong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bio-Evaluation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Cheongwon, Chungbuk 363-883</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Namhee+Jeong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeong N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeong N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Namhee Jeong</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiwoong Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiwoong+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jiwoong Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Woo Kyu Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bio-Evaluation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Cheongwon, Chungbuk 363-883</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Woo+Kyu+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Woo Kyu Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun-Gyeong Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun-Gyeong+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yun-Gyeong Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sang-Heon Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sang-Heon+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sang-Heon Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Woongchang Yoon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Woongchang+Yoon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoon W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoon W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Woongchang Yoon</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jin-Hyun Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Genetic Engineering</div>, <div class="insititution">Dong-A University</div>, <div class="addr-line">Busan 604-714</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jin-Hyun+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jin-Hyun Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ik-Young Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">National Instrumentation Center for Environmental Management</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ik-Young+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ik-Young Choi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hong-Kyu Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Genetic Engineering</div>, <div class="insititution">Dong-A University</div>, <div class="addr-line">Busan 604-714</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hong-Kyu+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hong-Kyu Choi</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jung-Kyung Moon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">National Institute of Crop Science, Rural Development Administration</div>, <div class="addr-line">Suwon 441-857</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jung-Kyung+Moon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moon J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moon J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jung-Kyung Moon</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Namshin Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Korean Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioinformatics</div>, <div class="insititution">University of Science and Technology</div>, <div class="addr-line">Daejeon 305-806</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 43-240-6540. Fax. +82 43-240-6549. Email: <a href="mailto:scjeong@kribb.re.kr" target="_blank">scjeong@kribb.re.kr</a> (S.-C.J.); Tel. +82 42-879-8540. Fax. +82 42-879-8519. Email: <a href="mailto:deepreds@kribb.re.kr" target="_blank">deepreds@kribb.re.kr</a> (N.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Namshin+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Namshin Kim</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soon-Chun Jeong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bio-Evaluation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Cheongwon, Chungbuk 363-883</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 43-240-6540. Fax. +82 43-240-6549. Email: <a href="mailto:scjeong@kribb.re.kr" target="_blank">scjeong@kribb.re.kr</a> (S.-C.J.); Tel. +82 42-879-8540. Fax. +82 42-879-8519. Email: <a href="mailto:deepreds@kribb.re.kr" target="_blank">deepreds@kribb.re.kr</a> (N.K.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soon-Chun+Jeong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeong S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeong S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Soon-Chun Jeong</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">The short-read sequence data are deposited into the European Nucleotide Archive under accession number ERP002622. The whole-genome SNP and indel data set have been deposited into NCBI dbSNP, with submission number records from ss7843006444 to ss792422648. The Sanger sequencing data for SNP and indel validation have been deposited with the GenBank data library under accession nos. KF220802–KF221057.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">These authors contributed equally to this article.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 153–167, <a href='https://doi.org/10.1093/dnares/dst047'>https://doi.org/10.1093/dnares/dst047</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 November 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">20 July 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">16 October 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="403337" href="/dnaresearch/article-pdf/21/2/153/1126808/dst047.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Won-Hyong Chung, Namhee Jeong, Jiwoong Kim, Woo Kyu Lee, Yun-Gyeong Lee, Sang-Heon Lee, Woongchang Yoon, Jin-Hyun Kim, Ik-Young Choi, Hong-Kyu Choi, Jung-Kyung Moon, Namshin Kim, Soon-Chun Jeong;  Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes<sup></sup>, <em>DNA Research</em>, Volume 21, Issue 2, 1 April 2014, Pages 153–167, <a href='https://doi.org/10.1093/dnares/dst047'>https://doi.org/10.1093/dnares/dst047</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/403337?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/403337?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Population%20Structure%20and%20Domestication%20Revealed%20by%20High-Depth%20Resequencing%20of%20Korean%20Cultivated%20and%20Wild%20Soybean%20Genomes%26lt%3Bsup%26gt%3B%26lt%3Ba%20reveal-id%3D%26quot%3BAN1%26quot%3B%20data-open%3D%26quot%3BAN1%26quot%3B%20class%3D%26quot%3Blink%20link-ref%20link-reveal%20xref-fn%26quot%3B%26gt%3B%E2%80%A0%26lt%3B%2Fa%26gt%3B%26lt%3B%2Fsup%26gt%3B&amp;publicationDate=2013-11-21&amp;volumeNum=21&amp;issueNum=2&amp;author=Chung%2C%20Won-Hyong%3B%20Jeong%2C%20Namhee&amp;startPage=153&amp;endPage=167&amp;contentId=10.1093%2Fdnares%2Fdst047&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f153%2f403337" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5954607 id="5954607" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Despite the importance of soybean as a major crop, genome-wide variation and evolution of cultivated soybeans are largely unknown. Here, we catalogued genome variation in an annual soybean population by high-depth resequencing of 10 cultivated and 6 wild accessions and obtained 3.87 million high-quality single-nucleotide polymorphisms (SNPs) after excluding the sites with missing data in any accession. Nuclear genome phylogeny supported a single origin for the cultivated soybeans. We identified 10-fold longer linkage disequilibrium (LD) in the wild soybean relative to wild maize and rice. Despite the small population size, the long LD and large SNP data allowed us to identify 206 candidate domestication regions with significantly lower diversity in the cultivated, but not in the wild, soybeans. Some of the genes in these candidate regions were associated with soybean homologues of canonical domestication genes. However, several examples, which are likely specific to soybean or eudicot crop plants, were also observed. Consequently, the variation data identified in this study should be valuable for breeding and for identifying agronomically important genes in soybeans. However, the long LD of wild soybeans may hinder pinpointing causal gene(s) in the candidate regions.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="domestication">domestication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="resequencing">resequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="soybean">soybean</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="variation">variation</a></div>                            <h2 scrollto-destination=5954611 id="5954611" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Soybean is an important worldwide plant source of dietary protein and oil, and its capability of nitrogen fixation during symbiosis with rhizobia plays an important role in establishing sustainable agriculture systems. Cultivated soybean (<em>Glycine max</em>) is thought to have been domesticated from wild soybean (<em>Glycine soja</em>) with distribution in East Asia, including Korea, Japan, and most parts of China, as early as 7000–9000 years ago,<sup><a reveal-id="DST047C1" data-open="DST047C1" class="link link-ref link-reveal xref-bibr">1</a></sup> although the exact domestication location(s) are under debate.<sup><a reveal-id="DST047C1" data-open="DST047C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST047C2" data-open="DST047C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Soybean has undergone radical phenotypic changes in the seed size, colour, shattering, seed dormancy, flowering time, and plant architecture during the process of domestication. Localization of the major causative genes, responsible for these traits, would facilitate improvements in soybean using marker-assisted breeding. Genome-wide association studies (GWAS) of the diverse soybean germplasm may offer the potential to rapidly resolve complex traits to gene-level resolution, as in maize and <em>Arabidopsis</em>. However, GWAS take years to construct segregating populations and require a high density of genome-wide markers. Association mapping is also prone to miss causative gene alleles, thereby requiring additional sequence-level mapping.<sup><a reveal-id="DST047C3" data-open="DST047C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The availability of a comprehensive genome variation catalogue for both cultivated and wild soybeans would greatly facilitate the identification of functional variations in elite varieties by comparing the genomic variations in an elite variety with data from the controls. Resequencing analyses of major crop plants, including soybean, rice, and maize, using a low-depth sequencing coverage have been reported recently.<sup><a reveal-id="DST047C4  DST047C7" data-open="DST047C4  DST047C7" class="link link-ref link-reveal xref-bibr">4–7</a></sup> However, the strategy of pooling many accessions for variation calling might miss rare variants, as indicated in a dense resequencing study of 50 diverse rice accessions<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and deep <em>Arabidopsis</em> sequencing studies.<sup><a reveal-id="DST047C9" data-open="DST047C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST047C10" data-open="DST047C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Here, we provide dense variation data based on an analysis of high-depth resequencing data of a diverse group of 10 cultivated and 6 wild soybean genomes sequenced to &gt;14× mean depth. Thus, our results will likely be useful for marker-assisted breeding and sequence-level gene mapping of soybean.</p>                            <h2 scrollto-destination=5954615 id="5954615" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5954617 id="5954617" class="section-title" data-legacy-id=s2a>2.1. Sampling</h3>
<p>All samples were grown in greenhouses and in the field for morphological confirmation.</p>                            <h3 scrollto-destination=5954622 id="5954622" class="section-title" data-legacy-id=s2b>2.2. Sample preparation and sequencing</h3>
<p>Seeds of each of soybean accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>) were germinated in a dark chamber at 25°C in a pot. After primary leaves of the germinated seedlings opened, all etiolated shoots of the seedlings except their cotyledons were collected to extract genomic DNA. Sequencing libraries were constructed according to the manufacturer's instructions (Illumina, San Diego, CA, USA). By applying the Illumina HiSeq2000 platform, we generated 305 Gb of 101 bp paired-end short reads from 15 samples with insert sizes of 237–336 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The short-read sequence data are deposited into the European Nucleotide Archive under accession number ERP002622. We imported 39.8 Gb of the 76 bp paired-end reads for IT182932, which was reported previously by Kim <em>et al.</em><sup><a reveal-id="DST047C11" data-open="DST047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Over 94% of the reads (86.3% for IT182932) were mapped onto the whole Williams 82 reference genome. A total of 86.3–90% of reads were mapped onto the chromosomes (Gm01–Gm20) except reads of IT182932 (76.1%). After removing duplicated reads, we used 78.9–88.5% (71.9% for IT182932) of the sequenced reads for downstream analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                            <h3 scrollto-destination=5954627 id="5954627" class="section-title" data-legacy-id=s2c>2.3. Genome size estimation</h3>
<p>The estimation of genome size for each accession was computed based on counting of occurrences of <em>K</em>-mers (substring of length <em>K</em>). We estimated the genome size using JELLYFISH 1.1.5<sup><a reveal-id="DST047C12" data-open="DST047C12" class="link link-ref link-reveal xref-bibr">12</a></sup> with <em>K</em>-mer as 17 bp JELLYFISH counted 17-mer frequencies per depth. As the peak of 17-mer frequency (<em>M</em>) in reads is correlated with the real sequencing depth (<em>N</em>), read length (<em>L</em>), and <em>K</em>-mer length (<em>K</em>), their relationship can be expressed in a formula used in the Panda genome project: <em>M</em> = <em>N</em> × (<em>L</em> – <em>K</em> + 1)/<em>L</em>.<sup><a reveal-id="DST047C13" data-open="DST047C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Then, the estimated genome size was obtained by dividing the total sequence length by the real sequencing depth.</p>                            <h3 scrollto-destination=5954631 id="5954631" class="section-title" data-legacy-id=s2d>2.4. Reads mapping</h3>
<p>Paired-end reads were mapped onto the Williams 82 reference genome (Glyma1) using Burrows Wheeler Aligner (version 0.5.9) with default options.<sup><a reveal-id="DST047C14" data-open="DST047C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The reference genome sequence was downloaded from the JGI genome portal (<a class="link link-uri" href="http://ftp.jgi-psf.org/pub/compgen/phytozome/V9.0/Gmax/assembly/Gmax_189.fa.gz">ftp.jgi-psf.org/pub/compgen/phytozome/V9.0/Gmax/assembly/Gmax_189.fa.gz</a>, accessed on 10 November 2013). <em>Glycine max</em> chloroplast genome (NC_007942) was separately downloaded from the NCBI ftp site (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/Glycine_max/CHR_Pltd/gma_ref_V1.0_chrPltd.fa.gz">ftp://ftp.ncbi.nlm.nih.gov/genomes/Glycine_max/CHR_Pltd/gma_ref_V1.0_chrPltd.fa.gz</a>, accessed on 10 November 2013) and included in the reference genome. After initial mapping of the raw reads onto the reference genome, we screened the reads that were mapped as unplaced scaffolds and the unmapped reads. Aligned reads considered to be PCR duplicates were removed using the MarkDuplicates in the Picard software package 1.48 (<a class="link link-uri" href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a>, accessed on 10 November 2013). Mate information was re-synchronized using the Picard FixMateInformation tool. Alignments around the small indels were re-aligned with IndelRealigner, and base-pair quality scores (QUAL) were recalibrated with CountCovariates and TableRecalibration in the Genome Analysis Toolkit (GATK; version 1.0.5974).<sup><a reveal-id="DST047C15" data-open="DST047C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The realigned, recalibrated SAM (Sequence Alignment/Map) files produced by these processing steps were used for single-nucleotide polymorphism (SNP)/indel detection and for all alignment-related statistics, such as allele counts.</p>                            <h3 scrollto-destination=5954636 id="5954636" class="section-title" data-legacy-id=s2e>2.5. SNP and indel detection</h3>
<p>Individual genotyping was performed to identify SNPs and indels for each of the 16 samples. Then, multi-sample SNP genotyping was performed to compare the genotypes on the variant positions, where all samples were able to be genotyped. UnifiedGenotyper in GATK<sup><a reveal-id="DST047C16" data-open="DST047C16" class="link link-ref link-reveal xref-bibr">16</a></sup> was used to generate the initial SNP and small indel calls. The SNP/indels were called with standard call confidence (-stand_call_conf) set to 30.0 and standard emit confidence (-stand_emit_conf) set to 10.0. An indel model (-glm INDEL) was enabled in indel calling. To identify high-quality variants, the initially called SNP/indels were evaluated with a Gaussian mixture model that was built with known soybean variants (dbSNP; version 133), and outliers were discarded. To reduce the SNP/indel false discovery rate, raw variant calls were filtered using VariantFiltration in GATK for the following annotations: QUAL of ≤50.0, depth of coverage (DP) ≤5, call quality divided by depth (QD) ≤5, mapping quality (MQ) ≤30, strand bias (SB) greater than or equal to −1.0, MQ zero reads (MQ0) ≥4, and MQ0 divided by depth (MQ0/DP) ≥0.1. We designed random primer pairs across the genome and performed Sanger sequencing to validate genotypes called using our Illumina data. Agreement between locations of SNPs/indels mapped in the Hwangkeum × IT182932 population,<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and SNPs/indels called in this study was also examined for validation.</p>                            <h3 scrollto-destination=5954641 id="5954641" class="section-title" data-legacy-id=s2f>2.6. Identification of non-reference genes</h3>
<p>We assembled the unmapped reads from each sample into contigs using SOAPdenovo.<sup><a reveal-id="DST047C18" data-open="DST047C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Default parameters were used and only contigs, not scaffolds, were constructed. We first assembled the unmapped reads separately in each accession, and contigs shorter than 2 kb were excluded to identify novel sequences. We then used the self-alignment approach to exclude the redundant sequences. In total, we identified 10 035 contigs with a total length of 31.5 Mb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). In a previous analysis of IT182832 short reads using different methods, Kim <em>et al</em>.<sup><a reveal-id="DST047C11" data-open="DST047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reported a total length of 8.3 Mb much longer than 0.83 Mb obtained in this study. However, a total length of contigs longer than 300 bp from IT182932 in the current analysis was 9.1 Mb, indicating that this difference likely reflects different methods and cut-off criteria. We conducted <em>de novo</em> gene annotation with AUGUSTUS<sup><a reveal-id="DST047C19" data-open="DST047C19" class="link link-ref link-reveal xref-bibr">19</a></sup> for the 10 035 contigs. After annotation, we excluded the redundant genes that were assembled in different accessions. Only one copy of the genes with &gt;90% identity and &gt;90% coverage by BLAT was retained. In total, we annotated 1363 possible genes <em>de novo</em> containing ≥100 amino acids. Then, we used BLASTP<sup><a reveal-id="DST047C20" data-open="DST047C20" class="link link-ref link-reveal xref-bibr">20</a></sup> to compare the candidate novel genes against the NCBI non-redundant (nr) database (&gt;60% identity and 60% coverage). Then, we further performed functional annotation using InterProScan<sup><a reveal-id="DST047C21" data-open="DST047C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and HMMer3.<sup><a reveal-id="DST047C22" data-open="DST047C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>We attempted to validate some of these gain genes using a PCR amplification method. We chose 51 gain genes (transposon genes excluded) from the cultivars Williams 82K (<em>n</em> = 4) and Hwangkeum (8), as well as the wild soybean IT162825 (39). PCRs were performed using the cultivated soybeans Williams 82K and Hwangkeum, as well as the wild soybeans IT182932 and IT162825 as templates. In addition to singlet PCR using a pair of primers designed from each of the chosen genes, we performed multiplex PCR by adding a pair of primers designed from the soybean actin gene to be certain of the absence of the expected PCR products. The actin gene primers were: forward 5′-TGGACTCTGGTGAT GGTGTC-3′ and reverse 5′-CTCCAATCCAAACACTGTA-3′.<sup><a reveal-id="DST047C23" data-open="DST047C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p>                            <h3 scrollto-destination=5954647 id="5954647" class="section-title" data-legacy-id=s2g>2.7. Identification of gene loss events</h3>
<p>To identify gene loss events, we first discovered medium-size deletion (ranging from 200 to 300 000 bp) sites in the Williams 82 reference genome using Genome STRiP 1.0.3.<sup><a reveal-id="DST047C24" data-open="DST047C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Genome STRiP identifies structural variations in populations by using information from read pairs with unexpected alignments as well as analysis of read depth. We ran the Genome STRiP with default parameters for 15 accessions except IT182932. Calls ‘not detected’ in more than three accessions were excluded. This reduced false positives due to low depth of resequencing data at some regions or due to errors in the reference genome sequence. As a result, we identified 10 928 medium-size deletions in various accessions and then examined if these deletions were associated with gene loss events based on gene annotations from the soybean reference v1.1 gene set, which was downloaded from the SoyBase website (<a class="link link-uri" href="http://www.soybase.org">http://www.soybase.org</a>, accessed on 10 November 2013). Validation of the gene loss events was performed by designing pairs of primers that amplify 200–300 bp DNA fragments flanking the deletion sites. We randomly chose 55 deletions of various sizes and used PCR to experimentally validate the deletions.</p>                            <h3 scrollto-destination=5954651 id="5954651" class="section-title" data-legacy-id=s2h>2.8. Gene ontology term enrichment analysis</h3>
<p>Gene ontology (GO) term enrichment analysis was performed on the non-reference (or gained) gene set, the gene set associated with gene loss events, and the identified candidate domestication or improvement gene set. We first loaded protein sequences of the gene sets into Pathway Studio ver. 9 (Elsevier). The enrichment analyses were performed using the ResNet Plant database (version 4.0) provided by this software. The identified orthologues using the BLAST best reciprocal hit method were used for the enrichment analysis. The algorithm returned a probability value (<em>P</em>) for the statistical significance test of gene enrichment between the input gene set and GO group. We considered a GO term to be enriched if the <em>P</em>-value was &lt;0.001.</p>                            <h3 scrollto-destination=5954656 id="5954656" class="section-title" data-legacy-id=s2i>2.9. Construction of phylogeny</h3>
<p>SNPs were used to calculate the genetic distances between different accessions. The neighbour-joining method<sup><a reveal-id="DST047C25" data-open="DST047C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was applied to construct the phylogenetic tree based on the <em>p</em>-distance method and the bootstrap confidence analysis with 1000 replicates. The trees were drawn to scale, with branch lengths in the same units as those of the evolutionary distances, which are in the units of the number of base differences per site. The constructed phylogenetic tree was plotted using MEGA5.<sup><a reveal-id="DST047C26" data-open="DST047C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=5954662 id="5954662" class="section-title" data-legacy-id=s2j>2.10. Population structure inference</h3>
<p>We used the STRUCTURE version 2.3.2,<sup><a reveal-id="DST047C27" data-open="DST047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which is based on the maximum-likelihood method, to investigate the population structure across different values of <em>K</em> (number of putative ancestral clusters of allelic similarity). We used an admixture model with correlated allele frequency<sup><a reveal-id="DST047C28" data-open="DST047C28" class="link link-ref link-reveal xref-bibr">28</a></sup> to assign individuals into <em>K</em> clusters. A 10,000 burn-in period of Chain Monte Carlo searches followed by 20,000 replicate runs were performed at each <em>K</em> from 2 to 7. We conducted STRUCTURE analysis for 17 accessions consisting of the reference genome (Williams 82), 10 cultivated, and 6 wild soybeans as well as an expanded set consisting of these 17 and 31 accessions from Lam <em>et al.</em><sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p>                            <h3 scrollto-destination=5954671 id="5954671" class="section-title" data-legacy-id=s2k>2.11. Calculation of linkage disequilibrium</h3>
<p>The correlation coefficient (<em>r</em><sup>2</sup>) of alleles was calculated to measure linkage disequilibrium (LD) levels in both the wild and cultivated soybeans using the Haploview software.<sup><a reveal-id="DST047C29" data-open="DST047C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The parameters were set as follows: maximum intermarker distance for LD comparisons (maxdistance) to 1000, output to pairwise LD text table format (dprime), minimum minor allele frequency (minMAF) to 0.1, and minimum Hardy-Weinberg <em>P</em>-value (hwcutoff) to 0.001. The average <em>r</em><sup>2</sup> values were calculated for pairwise marker distances, and the values were smoothed using the LOWESS function of R for both cultivated and wild soybean populations.</p>                            <h3 scrollto-destination=5954676 id="5954676" class="section-title" data-legacy-id=s2l>2.12. Estimation of population parameters and detection of putative artificially selected genes</h3>
<p>Population statistics <em>π</em>, <em>θ</em><sub>w</sub>, and <em>F</em><sub>ST</sub> were calculated using in-house custom scripts, as described previously.<sup><a reveal-id="DST047C30  DST047C32" data-open="DST047C30  DST047C32" class="link link-ref link-reveal xref-bibr">30–32</a></sup> The analysis using the reduction of diversity (ROD = 1 − <em>π</em><sub>cul</sub>/<em>π</em><sub>wild</sub>) introduced by Xu <em>et al</em>.<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> was adapted to our samples. ROD values were calculated based on the ratio of diversity in the cultivated soybeans to the diversity in the wild soybeans (<em>π</em><sub>cul</sub>/<em>π</em><sub>wild</sub>) in 20% sliding windows along the entire genome. We performed this analysis using windows of 10, 50, 100, 150, 200, and 500 kb. The windows with &gt;0.98 ROD values were picked out as candidate selective sweep regions, and genes in these regions were identified as putative genes under selection. Windows in multiple pericentromeric regions in our initial analysis showed high ROD values, as observed in a maize resequencing study.<sup><a reveal-id="DST047C7" data-open="DST047C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Because pericentromeric regions harbour few genes and correlations between genetic map and reference genome sequence in these regions are generally low,<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> we masked pericentromeric regions from further analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In addition, we masked windows in the bottom 1% of SNP frequency.</p>                            <h2 scrollto-destination=5954678 id="5954678" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5954679 id="5954679" class="section-title" data-legacy-id=s3a>3.1. Sequencing and mapping</h3>
<p>Wild and cultivated (domesticated) soybeans are classified into distinct species: cultivated soybean is <em>G. max</em> and its wild relatives are <em>G. soja</em>.<sup><a reveal-id="DST047C33" data-open="DST047C33" class="link link-ref link-reveal xref-bibr">33</a></sup> However, a cultivated plant and its wild relative can normally cross and produce a fertile F<sub>1</sub> hybrid generation despite distinctive phenotypes (Fig. <a reveal-id="DST047F1" data-open="DST047F1" class="link link-reveal link-table xref-fig">1</a>a). Furthermore, <em>G. soja</em> is the only wild member of the subgenus <em>Soja</em> distributed in East Asia.<sup><a reveal-id="DST047C34" data-open="DST047C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Thus, this crop and its wild ancestor have been proposed to be more appropriately classified as <em>G. max</em> subsp. <em>max</em> and <em>G. max</em> subsp. <em>soja</em>, respectively,<sup><a reveal-id="DST047C35" data-open="DST047C35" class="link link-ref link-reveal xref-bibr">35</a></sup> although most of the current literature and this study still use <em>G. soja</em> as the scientific name of the annual wild soybean. With this consideration, we use the term ‘cultivated’ and ‘wild’ soybeans to refer to <em>G. max</em> and <em>G. soja</em>, respectively, throughout this study to avoid the ambiguity of the current nomenclature. </p>                        <a id="5954681" scrollto-destination="5954681"></a>
<div id="DST047F1" data-id="DST047F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04701.png?Expires=2147483647&Signature=A6Jt~MnbnTu0rUXJCt3uUH1C8eQWosbWFOWtvnzU4MNYeWFvhEXNF0BUN-q8BX9laF1AdIdD9ZISPpkgrIycA07jYwaCte9KQBoGzrw6LKNrj0ONj3byQuEZgXO-RRUCuG2VDuBCrHWFBxNfJiB06PW~whaIcSL~hWr43My1j0nn80dqMps5BKxlXCpphaR7RVBjkrt5LgIbUfels9Z5sHk2CKTkT-pxlAa-NxurIPwkOUOCXtyjnSGZp0lMuDFWgb3Kd-now85FlHn072ZMSieFW4g3KrAdqQabUKs6Pn1HjTyDhS1ElAJpqT3d3SyMlrQ6jdJfaeIgOpI1mMSeUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Soybean population structure. (a) Changing morphology of domesticated soybean (left) and its wild relative (right). (b) Neighbour-joining phylogenetic tree of soybean nuclear genomes based on the high-quality SNPs, with the evolutionary distances measured by the p-distance. All branches except one denoted were supported by 100% bootstrap values from 1000 bootstrap replications. Taxa in the neighbour-joining tree (right) are represented by different colours: wild (red) and cultivated (blue) soybeans. Cultivated soybeans were tentatively grouped into C1, C2, and C3. (c) Bayesian clustering of samples using the STRUCTURE program. Each accession is represented by a vertical bar and each colour represents one population. An asterisk indicates a narrow pink segment, which is visible when enlarged. The mean value of ln-likelihood when K changed from 2 to 7 was −41525152, −37378506, −37280892, −32931343, −34405078, and −37518839, respectively." path-from-xml="dst04701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954681/dst04701.tiff" path-from-xml="dst04701.tiff" target="_blank">View large</a><a section="5954681" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04701.png?Expires=2147483647&Signature=meeDblOloA~xKQoR1qBDYRSNZtaBsU80O-zhLepypLFK917ei7fNWJ6sznm1Ab1NXsqhumektEU4KmsbxV3lO8rFaJDYNDYcaZS6sPwDwnuABYOFZ6sdTyySz7dkTGkwNXRc6qbPxYyxpaP8Cltqe4nV4dawqoEVxVyN4L3i17YCbzUy2YZz3yEY~TDgfB~RuUG1TTMtg~TrQB6yrItYcChVH5eOCXGFlUEqV4YE69i~pXeX0kQutfiGEuAeY~EbDzU5E2PqQq2TkYDQ8rK2hcsHgCr4w-A6g7vHBxXEnB~104D8B9VJyV-SYzquR4upSsRFYimfQaWlWAiRjX7Jnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954681&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Soybean population structure. (a) Changing morphology of domesticated soybean (left) and its wild relative (right). (b) Neighbour-joining phylogenetic tree of soybean nuclear genomes based on the high-quality SNPs, with the evolutionary distances measured by the <em>p</em>-distance. All branches except one denoted were supported by 100% bootstrap values from 1000 bootstrap replications. Taxa in the neighbour-joining tree (right) are represented by different colours: wild (red) and cultivated (blue) soybeans. Cultivated soybeans were tentatively grouped into C1, C2, and C3. (c) Bayesian clustering of samples using the STRUCTURE program. Each accession is represented by a vertical bar and each colour represents one population. An asterisk indicates a narrow pink segment, which is visible when enlarged. The mean value of ln-likelihood when <em>K</em> changed from 2 to 7 was −41525152, −37378506, −37280892, −32931343, −34405078, and −37518839, respectively.</p></div></div><div content-id="DST047F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04701.png?Expires=2147483647&Signature=A6Jt~MnbnTu0rUXJCt3uUH1C8eQWosbWFOWtvnzU4MNYeWFvhEXNF0BUN-q8BX9laF1AdIdD9ZISPpkgrIycA07jYwaCte9KQBoGzrw6LKNrj0ONj3byQuEZgXO-RRUCuG2VDuBCrHWFBxNfJiB06PW~whaIcSL~hWr43My1j0nn80dqMps5BKxlXCpphaR7RVBjkrt5LgIbUfels9Z5sHk2CKTkT-pxlAa-NxurIPwkOUOCXtyjnSGZp0lMuDFWgb3Kd-now85FlHn072ZMSieFW4g3KrAdqQabUKs6Pn1HjTyDhS1ElAJpqT3d3SyMlrQ6jdJfaeIgOpI1mMSeUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Soybean population structure. (a) Changing morphology of domesticated soybean (left) and its wild relative (right). (b) Neighbour-joining phylogenetic tree of soybean nuclear genomes based on the high-quality SNPs, with the evolutionary distances measured by the p-distance. All branches except one denoted were supported by 100% bootstrap values from 1000 bootstrap replications. Taxa in the neighbour-joining tree (right) are represented by different colours: wild (red) and cultivated (blue) soybeans. Cultivated soybeans were tentatively grouped into C1, C2, and C3. (c) Bayesian clustering of samples using the STRUCTURE program. Each accession is represented by a vertical bar and each colour represents one population. An asterisk indicates a narrow pink segment, which is visible when enlarged. The mean value of ln-likelihood when K changed from 2 to 7 was −41525152, −37378506, −37280892, −32931343, −34405078, and −37518839, respectively." path-from-xml="dst04701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954681/dst04701.tiff" path-from-xml="dst04701.tiff" target="_blank">View large</a><a section="5954681" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04701.png?Expires=2147483647&Signature=meeDblOloA~xKQoR1qBDYRSNZtaBsU80O-zhLepypLFK917ei7fNWJ6sznm1Ab1NXsqhumektEU4KmsbxV3lO8rFaJDYNDYcaZS6sPwDwnuABYOFZ6sdTyySz7dkTGkwNXRc6qbPxYyxpaP8Cltqe4nV4dawqoEVxVyN4L3i17YCbzUy2YZz3yEY~TDgfB~RuUG1TTMtg~TrQB6yrItYcChVH5eOCXGFlUEqV4YE69i~pXeX0kQutfiGEuAeY~EbDzU5E2PqQq2TkYDQ8rK2hcsHgCr4w-A6g7vHBxXEnB~104D8B9VJyV-SYzquR4upSsRFYimfQaWlWAiRjX7Jnw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954681&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Soybean population structure. (a) Changing morphology of domesticated soybean (left) and its wild relative (right). (b) Neighbour-joining phylogenetic tree of soybean nuclear genomes based on the high-quality SNPs, with the evolutionary distances measured by the <em>p</em>-distance. All branches except one denoted were supported by 100% bootstrap values from 1000 bootstrap replications. Taxa in the neighbour-joining tree (right) are represented by different colours: wild (red) and cultivated (blue) soybeans. Cultivated soybeans were tentatively grouped into C1, C2, and C3. (c) Bayesian clustering of samples using the STRUCTURE program. Each accession is represented by a vertical bar and each colour represents one population. An asterisk indicates a narrow pink segment, which is visible when enlarged. The mean value of ln-likelihood when <em>K</em> changed from 2 to 7 was −41525152, −37378506, −37280892, −32931343, −34405078, and −37518839, respectively.</p></div></div><p>We selected 10 cultivated soybean accessions that represent green, sprout, sauce-tofu, and cooking-with-rice soybeans as four groups classified in terms of food use in Korea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Among these cultivars, four are considered to be landraces collected in Korea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>) and six are improved varieties. To strictly control the quality of our sequencing and SNP calling, we also included Williams 82K,<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> a variant of the Williams 82 strain, which was used to generate the reference soybean genome sequence.<sup><a reveal-id="DST047C36" data-open="DST047C36" class="link link-ref link-reveal xref-bibr">36</a></sup> All improved soybeans, except Williams 82 which was bred by backcross breeding in the USA, were bred using a pedigree breeding method in Korea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Parental lines of improved varieties originated from China, Japan, Korea, Taiwan, and the USA. Although some of the parental lines share common ancestor(s) in their pedigrees, a unique parent in each of the five improved varieties should contribute ∼50% genetic variation from a unique landrace through this breeding method. Six wild accession samples were collected according to the geographic distribution of the wild soybean in Korea (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1 and Fig. S1</a>). As Korea is located in the centre of the ancient soybean cultivation and wild soybean distribution areas and has a long history of soybean use as food, we hypothesized that intensive sequencing of representative Korean soybean genomes might reveal the genetic diversity of soybean.<sup><a reveal-id="DST047C37" data-open="DST047C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p><p>We sequenced these 15 chosen accessions to &gt;17× coverage (raw data) using an Illumina HiSeq2000 instrument except the wild soybean IT182932 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Genome resequencing analysis of IT182932 was reported by Kim <em>et al.</em><sup><a reveal-id="DST047C11" data-open="DST047C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Raw reads of 39.8 Gb of the IT182932 genome produced by Illumina-GA were re-analysed in this study. Variation calling results of Williams 82K, Hwangkeum, and IT182932 were partly reported in our previous study, which compared soybean genetic and sequence-based physical maps.<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> However, population-level analysis using dense variation data of these 10 cultivated and 6 wild accessions is first reported in this study. After removing duplicate reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), the final mean depth was &gt;14× for all accessions and &gt;97% of the reference genome was covered by more than one read and &gt;94% by more than five reads for all accessions, but IT182932 (96.1 and 85.4%, respectively) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Homozygous variation callings required a minimum of five reads. The genome coverages were higher and much less variable than those of a similar study for rice accessions that ranged from 79 to 94%,<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> indicating that, relative to rice, soybeans contain smaller portions of individual-specific or diverged sequences. The high variability in mapping rates among rice accessions is likely due to significant genome size differences among rice subspecies.<sup><a reveal-id="DST047C38" data-open="DST047C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The similarity of genome sizes between cultivated and wild soybean accessions was further tested by estimating the genome sizes of soybean accessions using a distribution of 17-mer frequency in the usable sequencing reads to determine the sequencing depth.<sup><a reveal-id="DST047C13" data-open="DST047C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We obtained estimated genome sizes (±SEM) of 1107 ± 11 and 1106 ± 22 Mb from 10 cultivated and 6 wild soybeans, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The two estimated values were not significantly different (<em>t</em>-test, <em>P</em>-value = 0.990; <em>F</em>-test, <em>P</em>-value = 0.211). Thus, relative to <em>Oryza sativa</em> (rice) that is divided into several subspecies, the cultivated and wild soybeans that are widely divided into two different species, <em>G. max</em> and <em>G. soja</em>, should be regarded as a single species. The mean value of 1106 Mb from 16 soybeans was also similar to that of 1115 Mb estimated by flow cytometry,<sup><a reveal-id="DST047C39" data-open="DST047C39" class="link link-ref link-reveal xref-bibr">39</a></sup> suggesting that the Williams 82 sequence assembly of 955 Mb<sup><a reveal-id="DST047C36" data-open="DST047C36" class="link link-ref link-reveal xref-bibr">36</a></sup> may miss ∼150 Mb.</p>                            <h3 scrollto-destination=5954685 id="5954685" class="section-title" data-legacy-id=s3b>3.2. Identification of gene gain and loss events across the soybean genome</h3>
<p>High-depth resequencing data allowed us to identify gene gain and loss events across the soybean genome. We assembled unmapped reads for each accession into contigs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>) and then used <em>de novo</em> gene prediction to annotate 1326 putative genes in the contigs, of which 345 had homologues in the NCBI nr database and 343 of these had homologues in plants (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7, S8</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data Set 1</a>). Interestingly, of the eight genes that we annotated in two sequences identified as the missing sequences of the Williams 82 genome assembly in our previous report,<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> seven had &gt;85% similarity in these 343 genes and four were identical in these 343 genes, indicating that many of the unmapped genes represent genes from missing sequences in the Williams 82 genome assembly. To test whether some of these unmapped genes are found only in a certain accession, we chose 51 such ‘novel’ genes for PCR validation. When two cultivated and two wild accessions were tested, the proper PCR product size for all 51 was amplified in one or more of the four tested accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Of the 51, 20, including four from Williams 82K, were amplified in all four accessions and 31 were absent in one or more of the tested four, indicating that although some of our <em>de novo</em> assembled genes reflected incompleteness of the current Williams 82 genome assembly, &gt;60% were gain genes whose presence/absence in the soybean population were variable (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Interestingly, the presence/absence patterns of 12 non-reference genome genes agreed to those predicted from our <em>de novo</em> assembly. The 343 gain genes consist of 133 characterized gene homologues, 108 uncharacterized gene homologues, and 102 transposon-related genes, indicating that some of these novel genes may have arisen from the movement of transposable elements, which commonly create incomplete pseudogenes in plants. However, in the GO analysis using the Pathway Studio (Elsevier), we found no GO term enriched by ≥2 genes in the 133 genes.</p><p>In addition to the novel non-reference genome genes, we also identified genes absent in some accessions relative to the reference genome. We used the information of discordant paired-end reads at the deletion sites to collect evidence supporting these gene loss events. We identified 10 928 medium-size deletions between 229 and 308 146 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data Set 2</a>). Validation of the deletions by PCR suggested a false-positive rate of 26.6% in the dataset, which is higher than that (1 of 9) obtained with analysis of rice genome<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3 and Table S9</a>). Our close examination of the false-positive deletions showed that many of them correspond to the sites containing ambiguous 100–1000 N bps in the reference assembly as observed in our previous study.<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> However, because some of the N-stretch-containing sites were shown to be positives in our validation, the predicted N-stretch-containing deletions were retained in our current data set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data Set 2</a>). Of the deletions, 1737 were associated with gene loss events. The GO analysis of the 1737 lost genes yielded many enriched GO terms. Of the 30 significantly enriched GO terms in the biological process category, 12 were related to seed development, germination, and flower/leaf development reminiscent of the domestication-related traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). Domestication loci tend to show altered allele frequency.<sup><a reveal-id="DST047C40" data-open="DST047C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Most of the gene loss events in the 12 enriched GO terms did not show differential frequency between the cultivated and wild soybeans. However, 10 events showed distinct occurrence in the cultivated or wild soybeans (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>), thereby suggesting that they may be candidate domestication genes. Functional studies of the 10 lost genes have been mostly performed in <em>Arabidopsis</em>, which is not a crop species, and only one gene, <em>PAP15</em> (<em>Glyma11g36510</em>), has functionally been studied in several crop species including soybean.<sup><a reveal-id="DST047C41" data-open="DST047C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST047C42" data-open="DST047C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Overexpression of <em>Arabidopsis PAP15</em> in soybean increased phytase activity in the soybean tissue.<sup><a reveal-id="DST047C41" data-open="DST047C41" class="link link-ref link-reveal xref-bibr">41</a></sup> As numerous studies, including a recent report from Maupin and Rainey,<sup><a reveal-id="DST047C43" data-open="DST047C43" class="link link-ref link-reveal xref-bibr">43</a></sup> have shown a significant relationship between phytate content and germination percentage of soybean seeds, which is a domestication-related trait,<sup><a reveal-id="DST047C44" data-open="DST047C44" class="link link-ref link-reveal xref-bibr">44</a></sup> our result indicated that <em>PAP15</em> is a strong domestication candidate gene.</p>                            <h3 scrollto-destination=5954688 id="5954688" class="section-title" data-legacy-id=s3c>3.3. Variation across the soybean genome</h3>
<p>We identified ∼9 million candidate SNPs in all 16 accessions by mapping reads for each accession to the reference genome sequence (NCBI dbSNP [ss7843006444–ss792422648]; viewable via our Soya Genome Browser (<a class="link link-uri" href="http://soya.rna.kr">http://soya.rna.kr</a>, accessed on 10 November 2013)). To obtain high-quality SNPs for population analyses, we excluded SNPs with missing data in any of the 16 accessions, as these would make subsequent inferences unreliable, yielding a final total of 3 871 469 high-quality SNPs (Table <a reveal-id="DST047TB1" data-open="DST047TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). This represents the largest high-quality SNP dataset obtained in soybeans. In a low-depth resequencing analysis of 31 soybean accessions, Lam <em>et al.</em><sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> reported 966 612 high-quality SNPs. Although most of the SNPs were located in intergenic regions, the number of genic SNPs, 788 809 (20.4%), are almost comparable with the total number of SNPs reported by Lam <em>et al.</em><sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> A total of 1 109 412 non-singleton SNPs, which occurred in more than two accessions as homozygous SNPs, were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data Set 3</a>). The non-singleton (common) SNPs can serve as a resource to identify tag SNPs that will be needed to capture most haplotype variation for GWAS and designing useful breeding programmes in soybean.<sup><a reveal-id="DST047C45" data-open="DST047C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Thus, our data may be useful to identify important soybean genes by serving as molecular markers for designing soybean SNP arrays and for breeding. Among the 3.8 million high-quality SNPs, 1 687 232 were found in cultivars. A large proportion (1 106 593; 65.6%) of these SNPs was also found in wild soybean accessions, indicating that most genetic variation in cultivated soybean is derived from variation in wild soybean. Of the remaining 580 639 (34.4%) cultivar-specific SNPs, some might have been false positives due to the relatively small wild soybean sample size. We identified 3 290 830 high-quality SNPs in the six wild soybeans, of which 66.4% were wild-specific. Of the SNPs in the wild soybeans, 49.9% were a single wild accession-specific, while 36.2% of the SNPs in the cultivated soybean were a single cultivated accession-specific (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Collectively, our results suggested that wild soybean has a much more diverse gene pool than cultivars and thus, may contain useful genetic resources for improving soybean. </p>                        <a id="5954690" scrollto-destination="5954690"></a>
<div content-id="DST047TB1" class="table-modal"><div class="table-wrap"><div id="DST047TB1" class="table-wrap-title" data-id="DST047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of SNPs and indels variations for cultivated and wild soybeans obtained from individual or multi-sample genotyping</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Group </th><th rowspan="2">Sample size </th><th colspan="3">SNP<hr /> </th><th colspan="3">Indel<hr /> </th></tr><tr><th>Total </th><th>Genic </th><th>CDS<sup>a</sup> </th><th>Total </th><th>Genic </th><th>CDS </th></tr></thead><tbody><tr><td colspan="8">Individual genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>4 182 059 </td><td>618 493 </td><td>139 107 </td><td>799 470 </td><td>131 032 </td><td>7269 </td></tr><tr><td> Wild type </td><td>6 </td><td>7 626 486 </td><td>2 276 394 </td><td>252 245 </td><td>1 447 750 </td><td>246 653 </td><td>13 003 </td></tr><tr><td> Total </td><td>16 </td><td>9 028 250 </td><td>1 138 197 </td><td>296 648 </td><td>1 769 260 </td><td>294 390 </td><td>15 764 </td></tr><tr><td colspan="8">Multi-sample genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>1 687 232 </td><td>352 771 </td><td>78 172 </td><td>225 609 </td><td>57 543 </td><td>3972 </td></tr><tr><td> Wild type </td><td>6 </td><td>3 290 830 </td><td>675 710 </td><td>147 121 </td><td>430 564 </td><td>108 560 </td><td>6824 </td></tr><tr><td> Total </td><td>16 </td><td>3 871 469 </td><td>788 809 </td><td>173 293 </td><td>499 865 </td><td>125 289 </td><td>8222 </td></tr></tbody></table></div><div content-id="DST047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Group </th><th rowspan="2">Sample size </th><th colspan="3">SNP<hr /> </th><th colspan="3">Indel<hr /> </th></tr><tr><th>Total </th><th>Genic </th><th>CDS<sup>a</sup> </th><th>Total </th><th>Genic </th><th>CDS </th></tr></thead><tbody><tr><td colspan="8">Individual genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>4 182 059 </td><td>618 493 </td><td>139 107 </td><td>799 470 </td><td>131 032 </td><td>7269 </td></tr><tr><td> Wild type </td><td>6 </td><td>7 626 486 </td><td>2 276 394 </td><td>252 245 </td><td>1 447 750 </td><td>246 653 </td><td>13 003 </td></tr><tr><td> Total </td><td>16 </td><td>9 028 250 </td><td>1 138 197 </td><td>296 648 </td><td>1 769 260 </td><td>294 390 </td><td>15 764 </td></tr><tr><td colspan="8">Multi-sample genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>1 687 232 </td><td>352 771 </td><td>78 172 </td><td>225 609 </td><td>57 543 </td><td>3972 </td></tr><tr><td> Wild type </td><td>6 </td><td>3 290 830 </td><td>675 710 </td><td>147 121 </td><td>430 564 </td><td>108 560 </td><td>6824 </td></tr><tr><td> Total </td><td>16 </td><td>3 871 469 </td><td>788 809 </td><td>173 293 </td><td>499 865 </td><td>125 289 </td><td>8222 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Variations in coding sequences (CDS) of representative genes. As alternative transcripts were not included, the numbers presented should be regarded as approximate.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5954690" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST047TB1" class="table-wrap-title" data-id="DST047TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of SNPs and indels variations for cultivated and wild soybeans obtained from individual or multi-sample genotyping</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Group </th><th rowspan="2">Sample size </th><th colspan="3">SNP<hr /> </th><th colspan="3">Indel<hr /> </th></tr><tr><th>Total </th><th>Genic </th><th>CDS<sup>a</sup> </th><th>Total </th><th>Genic </th><th>CDS </th></tr></thead><tbody><tr><td colspan="8">Individual genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>4 182 059 </td><td>618 493 </td><td>139 107 </td><td>799 470 </td><td>131 032 </td><td>7269 </td></tr><tr><td> Wild type </td><td>6 </td><td>7 626 486 </td><td>2 276 394 </td><td>252 245 </td><td>1 447 750 </td><td>246 653 </td><td>13 003 </td></tr><tr><td> Total </td><td>16 </td><td>9 028 250 </td><td>1 138 197 </td><td>296 648 </td><td>1 769 260 </td><td>294 390 </td><td>15 764 </td></tr><tr><td colspan="8">Multi-sample genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>1 687 232 </td><td>352 771 </td><td>78 172 </td><td>225 609 </td><td>57 543 </td><td>3972 </td></tr><tr><td> Wild type </td><td>6 </td><td>3 290 830 </td><td>675 710 </td><td>147 121 </td><td>430 564 </td><td>108 560 </td><td>6824 </td></tr><tr><td> Total </td><td>16 </td><td>3 871 469 </td><td>788 809 </td><td>173 293 </td><td>499 865 </td><td>125 289 </td><td>8222 </td></tr></tbody></table></div><div content-id="DST047TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Group </th><th rowspan="2">Sample size </th><th colspan="3">SNP<hr /> </th><th colspan="3">Indel<hr /> </th></tr><tr><th>Total </th><th>Genic </th><th>CDS<sup>a</sup> </th><th>Total </th><th>Genic </th><th>CDS </th></tr></thead><tbody><tr><td colspan="8">Individual genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>4 182 059 </td><td>618 493 </td><td>139 107 </td><td>799 470 </td><td>131 032 </td><td>7269 </td></tr><tr><td> Wild type </td><td>6 </td><td>7 626 486 </td><td>2 276 394 </td><td>252 245 </td><td>1 447 750 </td><td>246 653 </td><td>13 003 </td></tr><tr><td> Total </td><td>16 </td><td>9 028 250 </td><td>1 138 197 </td><td>296 648 </td><td>1 769 260 </td><td>294 390 </td><td>15 764 </td></tr><tr><td colspan="8">Multi-sample genotyping </td></tr><tr><td> Cultivated </td><td>10 </td><td>1 687 232 </td><td>352 771 </td><td>78 172 </td><td>225 609 </td><td>57 543 </td><td>3972 </td></tr><tr><td> Wild type </td><td>6 </td><td>3 290 830 </td><td>675 710 </td><td>147 121 </td><td>430 564 </td><td>108 560 </td><td>6824 </td></tr><tr><td> Total </td><td>16 </td><td>3 871 469 </td><td>788 809 </td><td>173 293 </td><td>499 865 </td><td>125 289 </td><td>8222 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Variations in coding sequences (CDS) of representative genes. As alternative transcripts were not included, the numbers presented should be regarded as approximate.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5954690" target="_blank">View Large</a></div></div><p>In addition to SNPs, we also detected 1 769 260 candidate insertions (1–35 bp) and deletions (1–56 bp) (indels) by mapping reads with gaps allowed (Table <a reveal-id="DST047TB1" data-open="DST047TB1" class="link link-reveal link-table xref-fig">1</a>) (NCBI dbSNP [ss7843006444–ss792422648]; Soya Genome Browser (<a class="link link-uri" href="http://soya.rna.kr">http://soya.rna.kr</a>, accessed on 10 November 2013)). After excluding indels with missing data in any of the 16 accessions, 499 865 high-quality indels were retained. We observed slightly more deletions than insertions. Single-base-pair indels were the most frequent, and numbers of longer indels decreased abruptly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Similar to trends observed for SNPs, the wild soybeans contained more indels than the cultivated soybeans (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a> and Table <a reveal-id="DST047TB1" data-open="DST047TB1" class="link link-reveal link-table xref-fig">1</a>), and rare variants comprised a large proportion of the total indels with ∼73.2% of indels found in less than four accessions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Among the ∼0.5 million high-quality indels, we identified 86.1% (430 564) of them in the wild soybeans, of which 79.8% were wild-specific. Most of the indels were located in intergenic regions, and 1.6% (8222) of the indels were located in coding sequences.</p><p>To validate the variation calling results, we designed primer pairs from genomic regions of nine randomly selected genes as well as of two candidate domestication genes described below and performed Sanger sequencing. We sequenced 24.5 kb of soybean genomic DNA from each of the 16 accessions (GenBank: KF220802–KF221057). Of the 112 high-quality SNPs and 31 high-quality indels, except a Glyma20g32540 genomic region that showed unusually the high number of heterozygous variations, 109 (97% accuracy) and 28 (90% accuracy), respectively, were validated. Locations of SNPs/indels mapped in the Hwangkeum × IT182932 population<sup><a reveal-id="DST047C17" data-open="DST047C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and SNPs/indels called in this study agreed to each other. Our close examination of the <em>Glyma20g32540</em> genomic sequence in the Williams 82 reference genome assembly suggested that the high heterozygous variation callings were caused by that reads from its homeologous genomic region containing <em>Glyma10g35017</em>, part of which was ambiguous with one 100-N bp, were mapped onto the Glyma20g32540 region. This notion was further confirmed by determining the ambiguous sequence by sequencing PCR products encompassing the 100-N-bp region. These results indicated that, due to the missing or ambiguous parts of the reference genome sequence, some degree of heterozygous SNP callings in these regions are inevitable. These validations also served to confirm the high quality of our SNP and indel dataset.</p>                            <h3 scrollto-destination=5954693 id="5954693" class="section-title" data-legacy-id=s3d>3.4. Population structure of cultivated and wild soybeans</h3>
<p>To examine the genetic population structure and relationships among the cultivated and wild soybeans, we constructed a neighbour-joining tree<sup><a reveal-id="DST047C25" data-open="DST047C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DST047C26" data-open="DST047C26" class="link link-ref link-reveal xref-bibr">26</a></sup> based on the 3 329 657 high-quality SNPs after excluding SNPs with heterozygous genotypes in some of the 16 accessions as well as SNPs identified in the unanchored 17 Mb scaffolds. The cultivated and wild soybeans formed two clearly separated subclades (Fig. <a reveal-id="DST047F1" data-open="DST047F1" class="link link-reveal link-table xref-fig">1</a>b). Seoritae was most closely related to the wild soybeans in our phylogenetic tree. Pairwise distances between each pair of the wild soybeans were greater than those between any pairs of the cultivated soybeans, indicating that geographic distances of the collection sites in Korea are large enough to prevent intermixing of those wild soybeans analysed in this study, and that diversity between any pair of the collected wild soybeans is greater than those among all cultivated soybeans. When we compared pairwise distances between IT182848 and any of the wild and cultivated soybeans (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>), the mean distance value between the IT182848 and wild soybeans is not significantly different from that between the IT182848 and cultivated soybeans (<em>t</em>-test, <em>P</em> = 0.324). Taken together, our phylogenetic tree analysis suggested that the cultivated soybean population is a subclade of the wild soybean population rather than an independent species or subspecies.</p><p>To further investigate the population structure, we used the STRUCTURE program,<sup><a reveal-id="DST047C27" data-open="DST047C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which estimates individual ancestry and admixture proportions assuming that <em>K</em> populations exist based on a maximum-likelihood method.<sup><a reveal-id="DST047C46" data-open="DST047C46" class="link link-ref link-reveal xref-bibr">46</a></sup> We analysed the data by increasing <em>K</em> (the number of populations) from 2 to 7 (Fig. <a reveal-id="DST047F1" data-open="DST047F1" class="link link-reveal link-table xref-fig">1</a>c). We found a division between the cultivated and wild soybeans for <em>K</em> = 2. For <em>K</em>'s ranging from 3 to 7, the variety of Seoritae consistently appeared to be slightly different from the remaining cultivated soybeans. Although numbers of the groups for <em>K</em> &gt; 5 are smaller than that of <em>K</em> likely because of small population size, the number of groups within the wild soybean group increased with an increasing number of <em>K</em> up to three groups at <em>K</em> = 7. These results were consistent with the data from our phylogenetic analysis in that the cultivated soybeans are largely divided into three subgroups within a subclade (which was supported by results with an additional 31 soybeans described below), and each of the wild accessions contains more accession-specific SNPs than any of the cultivated accessions.</p><p>The evolutionary history of soybean and the range of diversity of our resequenced cultivated soybeans were further revolved by incorporating the SNP data (downloaded from the BGI soybean resequencing ftp site; <a class="link link-uri" href="http://ftp://public.genomics.org.cn/BGI/soybean_resequencing/">ftp://public.genomics.org.cn/BGI/soybean_resequencing/</a>, accessed on 10 November 2013) from 17 wild and 14 cultivated soybean genomes resequenced at a low depth (∼5×) reported by Lam <em>et al.</em><sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> into our current SNP data set. We then extracted SNPs covered by all 31 accessions without the missing data or heterozygous genotypes. By intersecting these SNPs with the set of 3.3 million high-quality SNPs identified in our 16 accessions, we obtained 208 684 SNPs that could be used to analyse all 23 wild and 24 cultivated soybean accessions. The results again showed that cultivated soybeans represent a subclade of a large soybean clade, indicating that wild soybean is the direct progenitor of cultivated soybean (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). Excluding the four admixture accessions,<sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> which were also supported by our population structure analysis, the cultivated soybeans formed three well-defined subgroups, each of which contained at least one accession of the 10 accessions examined in this study. This indicated that the diversity level of the cultivated soybeans obtained in this study likely serves as a leverage for elucidation of the comprehensive diversity of the global cultivated soybeans. However, the six wild strains resequenced in this study were clustered into a single small clade, indicating that the diversity level of wild soybean in the East Asia is much greater than that obtained in this study.</p>                            <h3 scrollto-destination=5954697 id="5954697" class="section-title" data-legacy-id=s3e>3.5. Identification of genome regions most affected by selection during soybean evolution</h3>
<p>Phenotypic traits associated with domestication syndrome that have undergone repeated artificial selection usually have a reduction in nucleotide diversity and altered allele frequency in the domestication loci,<sup><a reveal-id="DST047C40" data-open="DST047C40" class="link link-ref link-reveal xref-bibr">40</a></sup> as parameters that have successfully been used to identify putative artificially selected genes in crops and domesticated animals including maize (e.g.<sup><a reveal-id="DST047C7" data-open="DST047C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST047C47" data-open="DST047C47" class="link link-ref link-reveal xref-bibr">47</a></sup>), rice,<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> cattle,<sup><a reveal-id="DST047C48" data-open="DST047C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and dogs.<sup><a reveal-id="DST047C49" data-open="DST047C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Our large SNP dataset (&gt;3.8 M) from both the wild and cultivated soybeans provides an opportunity to identify the selected genes by comparing the polymorphism levels between cultivated and wild accessions.</p><p>The processes of domestication, which is repeated artificial selection, can increase LD through the genome or in genomic segments flanking domestication loci.<sup><a reveal-id="DST047C50" data-open="DST047C50" class="link link-ref link-reveal xref-bibr">50</a></sup> LD decayed to half of its maximum value at ∼120 kb for wild soybeans and at ∼340 kb for cultivated soybeans (Fig. <a reveal-id="DST047F2" data-open="DST047F2" class="link link-reveal link-table xref-fig">2</a>a). These LD distances are quite similar to those (wild, ∼75 kb and cultivated, ∼150 kb) estimated by low-depth resequencing data of 17 wild and 14 cultivated soybeans.<sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The LD distance of cultivated soybean is comparable with those of subpopulations of rice subspecies <em>japonica</em>, which ranged from 180 to 300 kb.<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> However, LD distance of the wild soybeans is extremely longer than those of the wild plants analysed at the genome level to date, which are usually within &lt;10 kb (e.g.<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST047C51" data-open="DST047C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DST047C52" data-open="DST047C52" class="link link-ref link-reveal xref-bibr">52</a></sup>). This long distance is likely due to the selfing nature of wild soybeans unlike wild maize and rice species.<sup><a reveal-id="DST047C45" data-open="DST047C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Thus, our results suggest that although two to three times longer LD distance for the cultivated than that for wild soybean accessions indicated the existence of some degree of domestication or improvement bottleneck in the cultivated soybeans, the relatively long LD of the wild soybeans may allow us to easily identify approximate domestication or improvement regions using whole-genome resequencing data. However, it would be more difficult to pinpoint causal domestication gene(s) in the identified regions in soybean relative to rice or maize. </p>                        <a id="5954700" scrollto-destination="5954700"></a>
<div id="DST047F2" data-id="DST047F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04702.png?Expires=2147483647&Signature=Bg93nU1Aj95By1FPuEdyMIcvGDu50DVwcJP2ydfgdbFtEGj-JUsFLRecf8drZb7uQKyQ3V~fO~zwML4btDYgFIFjrTB~Zoz4iixkL6Y7-PRo~LFAGsxkKIKEjxFqSzUkAgi~qsch8Y4Q2K288dZoTedXstjdtCIRWiIF5gcy2TTSeYlQdAukzM1NcBZvwizIvdIf0~c7FO~aYcO-LAR58XYllEm9~8CFLnYECdK-V3vKLbkiO8fTEqBAuULFfCzP999C7F7XtcGEPg4zrq~wtQL89W5eBuQiXT5tMMl8m5AQtkRNdxhCayTGkWcJAhkrv95Y4bw-wvyX4K6TZyBVcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide analysis of nucleotide diversity and selection. (a) LOWESS curves of LD decay patterns determined by squared correlations of allele frequencies (r2) against distance between polymorphic sites in cultivated (red) and wild (blue) soybeans. (b and c) Distributions of ROD values (b) and Z-transformed fixation index (FST) values (c) for cultivated relatives to wild soybeans in 100-kb windows across the genome. ROD = 0.98 corresponds with –log10(1 − ROD) = 1.70. The chromosome number is indicated along the x-axis." path-from-xml="dst04702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954700/dst04702.tiff" path-from-xml="dst04702.tiff" target="_blank">View large</a><a section="5954700" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04702.png?Expires=2147483647&Signature=sFCNNoGk5I5TTvsmvmLysJpHpILjL6AltDNwddqB2nK8C8DvawgxC66JjDYFKKDfCbHfTd0iJk0r7VdcxgD3O~xCckVIOemwMXs3nernWQeFeH6koHa1gHqWwd4wYAX0-Lb-LrcTEPEe1zKwtTq2t4LezlEUQDCQFp~YM1j~kG7y4Esu04uzMocg6-VI7mix8gaOAK8iqbsnzwG2iPYRguLYxYtFrNYVspFRl-CIiBOptYvw-ZCDoqDC3y816gRSu8gGgrxnNhmHDPHFqC~1c0iwO6YRNy6lV9yohFW~qhso-Hlgx-UoHHGwEclnA6ujsAqbr9mnvNEgWUACFgCNbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954700&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide analysis of nucleotide diversity and selection. (a) LOWESS curves of LD decay patterns determined by squared correlations of allele frequencies (<em>r</em><sup>2</sup>) against distance between polymorphic sites in cultivated (red) and wild (blue) soybeans. (b and c) Distributions of ROD values (b) and <em>Z</em>-transformed fixation index (<em>F</em><sub>ST</sub>) values (c) for cultivated relatives to wild soybeans in 100-kb windows across the genome. ROD = 0.98 corresponds with <em>–</em>log<sub>10</sub>(1 − ROD) = 1.70. The chromosome number is indicated along the <em>x-</em>axis.</p></div></div><div content-id="DST047F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04702.png?Expires=2147483647&Signature=Bg93nU1Aj95By1FPuEdyMIcvGDu50DVwcJP2ydfgdbFtEGj-JUsFLRecf8drZb7uQKyQ3V~fO~zwML4btDYgFIFjrTB~Zoz4iixkL6Y7-PRo~LFAGsxkKIKEjxFqSzUkAgi~qsch8Y4Q2K288dZoTedXstjdtCIRWiIF5gcy2TTSeYlQdAukzM1NcBZvwizIvdIf0~c7FO~aYcO-LAR58XYllEm9~8CFLnYECdK-V3vKLbkiO8fTEqBAuULFfCzP999C7F7XtcGEPg4zrq~wtQL89W5eBuQiXT5tMMl8m5AQtkRNdxhCayTGkWcJAhkrv95Y4bw-wvyX4K6TZyBVcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide analysis of nucleotide diversity and selection. (a) LOWESS curves of LD decay patterns determined by squared correlations of allele frequencies (r2) against distance between polymorphic sites in cultivated (red) and wild (blue) soybeans. (b and c) Distributions of ROD values (b) and Z-transformed fixation index (FST) values (c) for cultivated relatives to wild soybeans in 100-kb windows across the genome. ROD = 0.98 corresponds with –log10(1 − ROD) = 1.70. The chromosome number is indicated along the x-axis." path-from-xml="dst04702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954700/dst04702.tiff" path-from-xml="dst04702.tiff" target="_blank">View large</a><a section="5954700" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04702.png?Expires=2147483647&Signature=sFCNNoGk5I5TTvsmvmLysJpHpILjL6AltDNwddqB2nK8C8DvawgxC66JjDYFKKDfCbHfTd0iJk0r7VdcxgD3O~xCckVIOemwMXs3nernWQeFeH6koHa1gHqWwd4wYAX0-Lb-LrcTEPEe1zKwtTq2t4LezlEUQDCQFp~YM1j~kG7y4Esu04uzMocg6-VI7mix8gaOAK8iqbsnzwG2iPYRguLYxYtFrNYVspFRl-CIiBOptYvw-ZCDoqDC3y816gRSu8gGgrxnNhmHDPHFqC~1c0iwO6YRNy6lV9yohFW~qhso-Hlgx-UoHHGwEclnA6ujsAqbr9mnvNEgWUACFgCNbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954700&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide analysis of nucleotide diversity and selection. (a) LOWESS curves of LD decay patterns determined by squared correlations of allele frequencies (<em>r</em><sup>2</sup>) against distance between polymorphic sites in cultivated (red) and wild (blue) soybeans. (b and c) Distributions of ROD values (b) and <em>Z</em>-transformed fixation index (<em>F</em><sub>ST</sub>) values (c) for cultivated relatives to wild soybeans in 100-kb windows across the genome. ROD = 0.98 corresponds with <em>–</em>log<sub>10</sub>(1 − ROD) = 1.70. The chromosome number is indicated along the <em>x-</em>axis.</p></div></div><p>We sought to identify regions with significantly lower levels of polymorphisms in the cultivated soybeans relative to the wild soybeans to detect selective sweeps driven by artificial selection.<sup><a reveal-id="DST047C53" data-open="DST047C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Thus, we used an ROD (ROD = 1 − <em>π</em><sub>cul</sub>/<em>π</em><sub>wild</sub>) method<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> to identify regions of the genome most affected by selection during soybean evolution. ROD values were calculated based on the ratio of diversity in the cultivated soybeans to diversity in the wild soybeans (<em>π</em><sub>cul</sub>/<em>π</em><sub>wild</sub>) in 20% sliding windows along the entire genome except pericentromeric regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). We performed this analysis using windows of 10, 50, 100, 150, 200, and 500 kb with &gt;0.98 as a cut-off ROD value (Fig. <a reveal-id="DST047F2" data-open="DST047F2" class="link link-reveal link-table xref-fig">2</a>b and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). The estimated genome diversity levels using <em>π</em> and <em>θ</em><sub>w</sub> parameters<sup><a reveal-id="DST047C31" data-open="DST047C31" class="link link-ref link-reveal xref-bibr">31</a></sup> were &gt;2-fold lower in the cultivated soybeans (<em>π</em> = 0.46 × 10<sup>−3</sup> and <em>θ</em><sub>w</sub> = 0.46 × 10<sup>−3</sup>) compared with those in the wild soybeans (<em>π</em> = 1.08 × 10<sup>−3</sup> and <em>θ</em><sub>w</sub> = 1.15 × 10<sup>−3</sup>). The windows having &gt;0.98 ROD values that showed extremely low diversity in cultivars might have experienced cultivar-specific selective sweeps. Adjacent windows with &gt;0.98 ROD values were grouped into candidate domestication regions (CDRs), which likely represent the effect of a single selective sweep, as previously suggested.<sup><a reveal-id="DST047C7" data-open="DST047C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The detected CDRs in each of the 10, 50, 100, 150, 200, and 500-kb windows covered ∼15.5, 9.8, 7.2, 5.7, 4.4, and 1.6%, respectively, of the analysed soybean genome. Thus, the 100-kb windows showed genome coverage similar to that (∼7.6%) of CDRs obtained in a recent maize study.<sup><a reveal-id="DST047C7" data-open="DST047C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Portions of the detected chromosomal regions with significant ROD values in different windows overlapped with each other. The regions detected by larger windows were a subset of those detected by smaller windows except for slight differences in window margins (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). We further calculated the divergence index <em>F</em><sub>ST</sub> using the same sliding window approach. We found that &gt;80 and &gt;50% of the windows with &gt;0.98 ROD values were found in 25 and 10% upper tails, respectively, of the <em>F</em><sub>ST</sub> distribution in the larger than 100-kb windows (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S14 and Fig. S8</a>). Relative to the extreme 90% distribution of <em>japonica</em> ROD regions falling in the 5% right tail of the genome-wide <em>F</em><sub>ST</sub> distribution,<sup><a reveal-id="DST047C8" data-open="DST047C8" class="link link-ref link-reveal xref-bibr">8</a></sup> broader <em>F</em><sub>ST</sub> distribution of the soybean ROD regions is likely due to much longer LD distances (120 kb) of the wild soybeans relative to those of the wild rice and maize (&lt;10 kb), as shown in our LD estimation described above. Collectively, our results, including the LD estimation, genome coverage of CDRs, and comparison between distributions of <em>F</em><sub>ST</sub> and ROD values, suggested that, in this study, the 100-kb window would maximize the precision for detecting the candidate regions under selection.</p><p>We focused analyses on the 206 CDRs identified using the 100-kb windows, which contained 3068 genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S15</a>). CDRs contained an average of 14.9 genes and had a mean size of 189 kb. The longest fragment extended to 880 kb, and the median was 140 kb. We considered all genes in these CDRs to be candidates of artificially selected genes. It is likely that many of these genes were not themselves subjected to selection, but rather hitchhiked along with the actual gene targeted for artificial selection. More analyses, including transgenic experiments, are needed to identify the actual selected genes.</p><p>No canonical soybean domestication gene has been cloned yet. The genes annotated in the detected CDRs should prove useful, both for dissecting known quantitative trait loci (QTL) and identifying novel candidate domestication genes. When we searched for homologues of the domestication genes characterized in other crop plants such as rice, maize, and tomato, homologues of several canonical domestication genes including <em>tga1</em>, <em>qSH1</em>, <em>fw2.2</em>, <em>PROG1</em>, <em>DAG1</em>, <em>DAG2</em>, <em>Tunicate1</em>, and <em>OsMADS56</em> were successfully identified in our putative artificial selection gene set (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>). Although the majority of those canonical domestication genes have been functionally characterized in monocot crops distantly related to the soybean, most of CDRs containing their soybean homologues are, to our surprise, associated with QTL hotspots for domestication-related traits, such as plant height, leaflet shape, seed weight, pod dehiscence, and pod number. For example, <em>Glyma17g08840</em> homologous to <em>tga1</em> and <em>Glyma17g08861</em> homologous to <em>OsMADS56</em> are located in the same CDR on Gm17 and are associated with QTL that regulate multiple domestication-related traits, including leaflet length, plant weight, seed weight, and yield (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a> and Fig. <a reveal-id="DST047F3" data-open="DST047F3" class="link link-reveal link-table xref-fig">3</a>a). At the same CDR, we observed a putative transcription factor Glyma17g08761 containing TCP (TEOSINTE BRANCNED 1, cycloidea and PCF) domain characteristic of the maize gene <em>tb1</em> that regulates the plant and inflorescence structure, although its TCP domain showed less degree of amino acid identity (∼50%) than that of several other soybean TCP-containing proteins (∼70%) to that of the tb1 protein (Fig. <a reveal-id="DST047F3" data-open="DST047F3" class="link link-reveal link-table xref-fig">3</a>b). Thus, consistent with the multiple QTL reports by several studies, the CDR on Gm17 appears to be a hotspot for the domestication genes. However, its duplicated chromosomal segment from palaeopolyploidization,<sup><a reveal-id="DST047C36" data-open="DST047C36" class="link link-ref link-reveal xref-bibr">36</a></sup> which is corresponding to the north end of Gm05, was not detected as a CDR. </p>                        <a id="5954704" scrollto-destination="5954704"></a>
<div id="DST047F3" data-id="DST047F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04703.png?Expires=2147483647&Signature=jJRuEIGQJ4knbzS~jSlecdt1IgDxzR0vX3QQXagNElE7xjVBIfeu4q-MeHXnWSrGnNRmZGB6SDIdQvrngJSxCJo3D2mQirVsvfkSpOLXSmbn5x-HdmCQOn1R8pUrCrOMT1DWS~80~rYBg0xMh7yOq8HimsgNzrGE2189IVOaLTV0AVfQJhTQ~uPR0hcWlyGcIhHFuHDdfMOuEyqVJJaOZyofYQEAV20Jrmf-~oYcxdvFUH2E0~OQpbngD5W9Hy9zTHfKT-p42Ko4qRkRvbTgR83srUTxXvsFNu-8oW~XkfcvkZ8fahU-RhWhifUpQzWXvVamNRT2fGlswucWXaMHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Features of candidate domestication genes homologous to cloned canonical domestication genes. (a) ROD (red), and average fixation index, FST (blue), plotted for 100-kb windows across a 10-Mb region (upper panel) or for 10-kb windows across a 1-Mb region (lower panel) of chromosome 17, which harbours a cluster of candidate domestication genes homologous to cloned canonical domestication genes. Strong candidate domestication genes in the region are shown below (a). Grey boxes indicate a 180-kb chromosomal region having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, –log10(1 − ROD) values of ≥4 are shown as corresponding with –log10(1 − ROD) = 4. (b) Neighbour-joining phylogenetic tree showing relationship among soybean TCP family proteins, which appeared in CDRs, and functionally characterized representative members of other species, which were described by Martín-Trillo and Cubas.60 The percentage of bootstrap samples is shown to indicate the reliability for branching. Only the TCP domain was used for the analysis. See Supplementary Fig. S9 for the phylogenetic relationship among all predicted TCP proteins of soybean and representative members of other species. (c) Structure of three candidate domestication proteins showing conserved domains (coloured boxes) and positions of amino acid substitutions by nsSNPs fixed in wild soybeans. Glyma12g29991 is homologous to qSH1 and Glyma05g03660 and Glyma17g14191 are homologous to OsMADS56. POX is a functionally unknown domain named ‘associated with HOX’; homeobox is BEL1-type homeobox; MADS is SRF-type MADS box; K is K-box region." path-from-xml="dst04703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954704/dst04703.tiff" path-from-xml="dst04703.tiff" target="_blank">View large</a><a section="5954704" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04703.png?Expires=2147483647&Signature=4KwtX51GguDqE3JOViZnFpcGVNfC1-nu52O6gANIDSC1D7ee7Q508DJGkxNbrgblpXM4NndH84XQX7VVkGilfKCCewo7oFZyLnKNRk5HEA6Gzg~~xaLzma3d~1G4Eh7JL5URGFZSQcQz8J4SSHzmZQYoU0knl5LD841Yr6IZXf0Xkert6VbquCUB7~IfHandoXsAehTmUmUadBiccT-MagC-Bf4UDonhFrQsIfGxRH~NaeAsG2oAelccmTAhI~k1eon7~Ml7nMmMuNgzY8IkGCWG4-vo78~F82D~V50uYXKGrdfv8hTnKuaYtwQORguyjfo0VMj8gjRZ2EoAn-5y9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954704&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Features of candidate domestication genes homologous to cloned canonical domestication genes. (a) ROD (red), and average fixation index, <em>F</em><sub>ST</sub> (blue), plotted for 100-kb windows across a 10-Mb region (upper panel) or for 10-kb windows across a 1-Mb region (lower panel) of chromosome 17, which harbours a cluster of candidate domestication genes homologous to cloned canonical domestication genes. Strong candidate domestication genes in the region are shown below (a). Grey boxes indicate a 180-kb chromosomal region having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, <em>–</em>log<sub>10</sub>(1 − ROD) values of ≥4 are shown as corresponding with <em>–</em>log<sub>10</sub>(1 − ROD) = 4. (b) Neighbour-joining phylogenetic tree showing relationship among soybean TCP family proteins, which appeared in CDRs, and functionally characterized representative members of other species, which were described by Martín-Trillo and Cubas.<sup><a reveal-id="DST047C60" data-open="DST047C60" class="link link-ref link-reveal xref-bibr">60</a></sup> The percentage of bootstrap samples is shown to indicate the reliability for branching. Only the TCP domain was used for the analysis. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a> for the phylogenetic relationship among all predicted TCP proteins of soybean and representative members of other species. (c) Structure of three candidate domestication proteins showing conserved domains (coloured boxes) and positions of amino acid substitutions by nsSNPs fixed in wild soybeans. Glyma12g29991 is homologous to qSH1 and Glyma05g03660 and Glyma17g14191 are homologous to OsMADS56. POX is a functionally unknown domain named ‘associated with HOX’; homeobox is BEL1-type homeobox; MADS is SRF-type MADS box; <em>K</em> is <em>K</em>-box region.</p></div></div><div content-id="DST047F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04703.png?Expires=2147483647&Signature=jJRuEIGQJ4knbzS~jSlecdt1IgDxzR0vX3QQXagNElE7xjVBIfeu4q-MeHXnWSrGnNRmZGB6SDIdQvrngJSxCJo3D2mQirVsvfkSpOLXSmbn5x-HdmCQOn1R8pUrCrOMT1DWS~80~rYBg0xMh7yOq8HimsgNzrGE2189IVOaLTV0AVfQJhTQ~uPR0hcWlyGcIhHFuHDdfMOuEyqVJJaOZyofYQEAV20Jrmf-~oYcxdvFUH2E0~OQpbngD5W9Hy9zTHfKT-p42Ko4qRkRvbTgR83srUTxXvsFNu-8oW~XkfcvkZ8fahU-RhWhifUpQzWXvVamNRT2fGlswucWXaMHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Features of candidate domestication genes homologous to cloned canonical domestication genes. (a) ROD (red), and average fixation index, FST (blue), plotted for 100-kb windows across a 10-Mb region (upper panel) or for 10-kb windows across a 1-Mb region (lower panel) of chromosome 17, which harbours a cluster of candidate domestication genes homologous to cloned canonical domestication genes. Strong candidate domestication genes in the region are shown below (a). Grey boxes indicate a 180-kb chromosomal region having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, –log10(1 − ROD) values of ≥4 are shown as corresponding with –log10(1 − ROD) = 4. (b) Neighbour-joining phylogenetic tree showing relationship among soybean TCP family proteins, which appeared in CDRs, and functionally characterized representative members of other species, which were described by Martín-Trillo and Cubas.60 The percentage of bootstrap samples is shown to indicate the reliability for branching. Only the TCP domain was used for the analysis. See Supplementary Fig. S9 for the phylogenetic relationship among all predicted TCP proteins of soybean and representative members of other species. (c) Structure of three candidate domestication proteins showing conserved domains (coloured boxes) and positions of amino acid substitutions by nsSNPs fixed in wild soybeans. Glyma12g29991 is homologous to qSH1 and Glyma05g03660 and Glyma17g14191 are homologous to OsMADS56. POX is a functionally unknown domain named ‘associated with HOX’; homeobox is BEL1-type homeobox; MADS is SRF-type MADS box; K is K-box region." path-from-xml="dst04703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954704/dst04703.tiff" path-from-xml="dst04703.tiff" target="_blank">View large</a><a section="5954704" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04703.png?Expires=2147483647&Signature=4KwtX51GguDqE3JOViZnFpcGVNfC1-nu52O6gANIDSC1D7ee7Q508DJGkxNbrgblpXM4NndH84XQX7VVkGilfKCCewo7oFZyLnKNRk5HEA6Gzg~~xaLzma3d~1G4Eh7JL5URGFZSQcQz8J4SSHzmZQYoU0knl5LD841Yr6IZXf0Xkert6VbquCUB7~IfHandoXsAehTmUmUadBiccT-MagC-Bf4UDonhFrQsIfGxRH~NaeAsG2oAelccmTAhI~k1eon7~Ml7nMmMuNgzY8IkGCWG4-vo78~F82D~V50uYXKGrdfv8hTnKuaYtwQORguyjfo0VMj8gjRZ2EoAn-5y9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954704&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Features of candidate domestication genes homologous to cloned canonical domestication genes. (a) ROD (red), and average fixation index, <em>F</em><sub>ST</sub> (blue), plotted for 100-kb windows across a 10-Mb region (upper panel) or for 10-kb windows across a 1-Mb region (lower panel) of chromosome 17, which harbours a cluster of candidate domestication genes homologous to cloned canonical domestication genes. Strong candidate domestication genes in the region are shown below (a). Grey boxes indicate a 180-kb chromosomal region having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, <em>–</em>log<sub>10</sub>(1 − ROD) values of ≥4 are shown as corresponding with <em>–</em>log<sub>10</sub>(1 − ROD) = 4. (b) Neighbour-joining phylogenetic tree showing relationship among soybean TCP family proteins, which appeared in CDRs, and functionally characterized representative members of other species, which were described by Martín-Trillo and Cubas.<sup><a reveal-id="DST047C60" data-open="DST047C60" class="link link-ref link-reveal xref-bibr">60</a></sup> The percentage of bootstrap samples is shown to indicate the reliability for branching. Only the TCP domain was used for the analysis. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a> for the phylogenetic relationship among all predicted TCP proteins of soybean and representative members of other species. (c) Structure of three candidate domestication proteins showing conserved domains (coloured boxes) and positions of amino acid substitutions by nsSNPs fixed in wild soybeans. Glyma12g29991 is homologous to qSH1 and Glyma05g03660 and Glyma17g14191 are homologous to OsMADS56. POX is a functionally unknown domain named ‘associated with HOX’; homeobox is BEL1-type homeobox; MADS is SRF-type MADS box; <em>K</em> is <em>K</em>-box region.</p></div></div><p>Many of causal mutations in the cloned domestication genes are amino acid changes, although regulatory changes almost equally contribute to causal mutations.<sup><a reveal-id="DST047C53" data-open="DST047C53" class="link link-ref link-reveal xref-bibr">53</a></sup> To further characterize the genomic impact of domestication, we examined whether variations in coding sequences of the nine soybean genes homologous to canonical domestication genes described above changed the amino acid sequences of the expressed proteins (Table <a reveal-id="DST047TB2" data-open="DST047TB2" class="link link-reveal link-table xref-fig">2</a>). Of the nine genes, three including Glyma16g02550.3, Glyma08g07260.3, and Glyma17g08861.1 did not contain any non-synonymous SNP (nsSNP). The remaining six genes contained various numbers of nsSNPs with different minor allele frequencies, most of which are &lt;0.5. Interestingly, each of Glyma12g29991.3 (homologous to qSH1) and Glyma05g03660.8 (homologous to OsMADS56) contained a single fixed nsSNP between the resequenced cultivated and wild soybean populations. Both differences encode non-conservative amino acid substitutions and may affect protein function (Fig. <a reveal-id="DST047F3" data-open="DST047F3" class="link link-reveal link-table xref-fig">3</a>c). In case of Glyma17g14191.1 (homologous to OsMADS56), we observed two conservative amino acid substitutions and one 4 amino acid insertion at a short amino acid span between positions 178 and 183 fixed between the cultivated and wild soybeans. In all the three genes, the fixed amino acid substitutions lie outside of the conserved domains (Fig. <a reveal-id="DST047F3" data-open="DST047F3" class="link link-reveal link-table xref-fig">3</a>c), consistent with the previous cloning studies of canonical domestication genes including <em>tga1</em><sup><a reveal-id="DST047C54" data-open="DST047C54" class="link link-ref link-reveal xref-bibr">54</a></sup> and <em>PROG1</em>.<sup><a reveal-id="DST047C55" data-open="DST047C55" class="link link-ref link-reveal xref-bibr">55</a></sup></p>                        <a id="5954706" scrollto-destination="5954706"></a>
<div content-id="DST047TB2" class="table-modal"><div class="table-wrap"><div id="DST047TB2" class="table-wrap-title" data-id="DST047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Coding sequence diversity and amino acid differences in 10 cultivated and 6 wild resequenced soybean genomes at domestication candidate genes homologous to canonical domestication genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Domestication gene </th><th rowspan="3">Soybean candidate gene </th><th colspan="2" rowspan="2">Length (bp)<hr /> </th><th rowspan="2">ROD </th><th rowspan="2"><em>F</em><sub>ST</sub> </th><th colspan="4">Genetic diversity in genic region<hr /> </th><th colspan="4">Genetic diversity in CDS<hr /> </th><th colspan="2" rowspan="2">Amino acid difference<sup>a</sup> from reference<hr /> </th></tr><tr><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th></tr><tr><th>Genic </th><th>CDS </th><th>100-kb </th><th>100-kb </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Wild (MAF) </th><th>Cul (MAF) </th></tr></thead><tbody><tr><td><em>tga1</em> </td><td><em>Glyma17g08840.2</em> </td><td>5477 </td><td>1635 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.42 </td><td>0.37 </td><td>0.26 </td><td>0 </td><td>0.37 </td><td>0 </td><td>0.27 </td><td>Q491H (0.5) </td><td>nd </td></tr><tr><td><em>qSH1</em> </td><td><em>Glyma12g29991.3</em> </td><td>7859 </td><td>2385 </td><td>0.994 </td><td>0.427 </td><td>0.31 </td><td>1.88 </td><td>1.12 </td><td>0.54 </td><td>0 </td><td>1.12 </td><td>0 </td><td>1.47 </td><td>N34D (0.17); P42S (0.17); H240L (0.17); C658T (0.17); N674H (1) </td><td>nd </td></tr><tr><td><em>fw2.2</em> </td><td><em>Glyma15g01990.3</em> </td><td>2332 </td><td>507 </td><td>0.995 </td><td>0.667 </td><td>0 </td><td>8.75 </td><td>1.05 </td><td>0 </td><td>0 </td><td>1.05 </td><td>0 </td><td>0.86 </td><td>T648A (0.33) </td><td>nd </td></tr><tr><td><em>PROG1</em> </td><td><em>Glyma17g18110.1</em> </td><td>1601 </td><td>633 </td><td>0.994 </td><td>0.493 </td><td>0.22 </td><td>3.16 </td><td>1.05 </td><td>0.22 </td><td>0.56 </td><td>1.05 </td><td>0.56 </td><td>1.38 </td><td>V21_D22insSW (0.17); N76K (0.17) </td><td>H16D (0.2) </td></tr><tr><td><em>DAG1</em>; <em>DAG2</em> </td><td><em>Glyma16g02550.3</em> </td><td>2343 </td><td>831 </td><td>0.987 </td><td>0.727 </td><td>0 </td><td>0.71 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td><em>Tunicate1</em> </td><td><em>Glyma08g07260.3</em> </td><td>9994 </td><td>615 </td><td>0.997 </td><td>0.470 </td><td>0.19 </td><td>2.87 </td><td>0 </td><td>0.25 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td rowspan="3"><em>OsMADS56</em> </td><td><em>Glyma05g03660.8</em> </td><td>7654 </td><td>684 </td><td>0.994 </td><td>0.542 </td><td>0.26 </td><td>5.38 </td><td>1.27 </td><td>0.23 </td><td>0.78 </td><td>1.27 </td><td>0.52 </td><td>1.28 </td><td>I124V (0.17), T218A (1) </td><td>nd </td></tr><tr><td><em>Glyma17g14191.1</em> </td><td>8420 </td><td>663 </td><td>0.991 </td><td>0.546 </td><td>0.002 </td><td>4.18 </td><td>1.41 </td><td>0.004 </td><td>0 </td><td>1.41 </td><td>0 </td><td>1.32 </td><td>G177F (1); T180S (1); N182_V183insDAEL (1) </td><td>nd </td></tr><tr><td><em>Glyma17g08861.1</em> </td><td>5779 </td><td>1143 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.62 </td><td>2.9 </td><td>0.31 </td><td>0 </td><td>0.29 </td><td>0 </td><td>0.38 </td><td>nd </td><td>nd </td></tr></tbody></table></div><div content-id="DST047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Domestication gene </th><th rowspan="3">Soybean candidate gene </th><th colspan="2" rowspan="2">Length (bp)<hr /> </th><th rowspan="2">ROD </th><th rowspan="2"><em>F</em><sub>ST</sub> </th><th colspan="4">Genetic diversity in genic region<hr /> </th><th colspan="4">Genetic diversity in CDS<hr /> </th><th colspan="2" rowspan="2">Amino acid difference<sup>a</sup> from reference<hr /> </th></tr><tr><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th></tr><tr><th>Genic </th><th>CDS </th><th>100-kb </th><th>100-kb </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Wild (MAF) </th><th>Cul (MAF) </th></tr></thead><tbody><tr><td><em>tga1</em> </td><td><em>Glyma17g08840.2</em> </td><td>5477 </td><td>1635 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.42 </td><td>0.37 </td><td>0.26 </td><td>0 </td><td>0.37 </td><td>0 </td><td>0.27 </td><td>Q491H (0.5) </td><td>nd </td></tr><tr><td><em>qSH1</em> </td><td><em>Glyma12g29991.3</em> </td><td>7859 </td><td>2385 </td><td>0.994 </td><td>0.427 </td><td>0.31 </td><td>1.88 </td><td>1.12 </td><td>0.54 </td><td>0 </td><td>1.12 </td><td>0 </td><td>1.47 </td><td>N34D (0.17); P42S (0.17); H240L (0.17); C658T (0.17); N674H (1) </td><td>nd </td></tr><tr><td><em>fw2.2</em> </td><td><em>Glyma15g01990.3</em> </td><td>2332 </td><td>507 </td><td>0.995 </td><td>0.667 </td><td>0 </td><td>8.75 </td><td>1.05 </td><td>0 </td><td>0 </td><td>1.05 </td><td>0 </td><td>0.86 </td><td>T648A (0.33) </td><td>nd </td></tr><tr><td><em>PROG1</em> </td><td><em>Glyma17g18110.1</em> </td><td>1601 </td><td>633 </td><td>0.994 </td><td>0.493 </td><td>0.22 </td><td>3.16 </td><td>1.05 </td><td>0.22 </td><td>0.56 </td><td>1.05 </td><td>0.56 </td><td>1.38 </td><td>V21_D22insSW (0.17); N76K (0.17) </td><td>H16D (0.2) </td></tr><tr><td><em>DAG1</em>; <em>DAG2</em> </td><td><em>Glyma16g02550.3</em> </td><td>2343 </td><td>831 </td><td>0.987 </td><td>0.727 </td><td>0 </td><td>0.71 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td><em>Tunicate1</em> </td><td><em>Glyma08g07260.3</em> </td><td>9994 </td><td>615 </td><td>0.997 </td><td>0.470 </td><td>0.19 </td><td>2.87 </td><td>0 </td><td>0.25 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td rowspan="3"><em>OsMADS56</em> </td><td><em>Glyma05g03660.8</em> </td><td>7654 </td><td>684 </td><td>0.994 </td><td>0.542 </td><td>0.26 </td><td>5.38 </td><td>1.27 </td><td>0.23 </td><td>0.78 </td><td>1.27 </td><td>0.52 </td><td>1.28 </td><td>I124V (0.17), T218A (1) </td><td>nd </td></tr><tr><td><em>Glyma17g14191.1</em> </td><td>8420 </td><td>663 </td><td>0.991 </td><td>0.546 </td><td>0.002 </td><td>4.18 </td><td>1.41 </td><td>0.004 </td><td>0 </td><td>1.41 </td><td>0 </td><td>1.32 </td><td>G177F (1); T180S (1); N182_V183insDAEL (1) </td><td>nd </td></tr><tr><td><em>Glyma17g08861.1</em> </td><td>5779 </td><td>1143 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.62 </td><td>2.9 </td><td>0.31 </td><td>0 </td><td>0.29 </td><td>0 </td><td>0.38 </td><td>nd </td><td>nd </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Cul, cultivated soybean; Wild, wild soybean; CDS, coding sequence; MAF, minor allele frequency; nd, not detected.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The format for an amino acid difference is X#Y, where X is the amino acid of the Williams 82 reference genome, # is the position of the substitution, and Y is the new amino acid; and X#_Y#insAB, where X and Y are the amino acids of the Williams 82 reference genome, #s are the positions of the insertion, and insAB indicates A and B amino acids were inserted.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5954706" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST047TB2" class="table-wrap-title" data-id="DST047TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Coding sequence diversity and amino acid differences in 10 cultivated and 6 wild resequenced soybean genomes at domestication candidate genes homologous to canonical domestication genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Domestication gene </th><th rowspan="3">Soybean candidate gene </th><th colspan="2" rowspan="2">Length (bp)<hr /> </th><th rowspan="2">ROD </th><th rowspan="2"><em>F</em><sub>ST</sub> </th><th colspan="4">Genetic diversity in genic region<hr /> </th><th colspan="4">Genetic diversity in CDS<hr /> </th><th colspan="2" rowspan="2">Amino acid difference<sup>a</sup> from reference<hr /> </th></tr><tr><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th></tr><tr><th>Genic </th><th>CDS </th><th>100-kb </th><th>100-kb </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Wild (MAF) </th><th>Cul (MAF) </th></tr></thead><tbody><tr><td><em>tga1</em> </td><td><em>Glyma17g08840.2</em> </td><td>5477 </td><td>1635 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.42 </td><td>0.37 </td><td>0.26 </td><td>0 </td><td>0.37 </td><td>0 </td><td>0.27 </td><td>Q491H (0.5) </td><td>nd </td></tr><tr><td><em>qSH1</em> </td><td><em>Glyma12g29991.3</em> </td><td>7859 </td><td>2385 </td><td>0.994 </td><td>0.427 </td><td>0.31 </td><td>1.88 </td><td>1.12 </td><td>0.54 </td><td>0 </td><td>1.12 </td><td>0 </td><td>1.47 </td><td>N34D (0.17); P42S (0.17); H240L (0.17); C658T (0.17); N674H (1) </td><td>nd </td></tr><tr><td><em>fw2.2</em> </td><td><em>Glyma15g01990.3</em> </td><td>2332 </td><td>507 </td><td>0.995 </td><td>0.667 </td><td>0 </td><td>8.75 </td><td>1.05 </td><td>0 </td><td>0 </td><td>1.05 </td><td>0 </td><td>0.86 </td><td>T648A (0.33) </td><td>nd </td></tr><tr><td><em>PROG1</em> </td><td><em>Glyma17g18110.1</em> </td><td>1601 </td><td>633 </td><td>0.994 </td><td>0.493 </td><td>0.22 </td><td>3.16 </td><td>1.05 </td><td>0.22 </td><td>0.56 </td><td>1.05 </td><td>0.56 </td><td>1.38 </td><td>V21_D22insSW (0.17); N76K (0.17) </td><td>H16D (0.2) </td></tr><tr><td><em>DAG1</em>; <em>DAG2</em> </td><td><em>Glyma16g02550.3</em> </td><td>2343 </td><td>831 </td><td>0.987 </td><td>0.727 </td><td>0 </td><td>0.71 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td><em>Tunicate1</em> </td><td><em>Glyma08g07260.3</em> </td><td>9994 </td><td>615 </td><td>0.997 </td><td>0.470 </td><td>0.19 </td><td>2.87 </td><td>0 </td><td>0.25 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td rowspan="3"><em>OsMADS56</em> </td><td><em>Glyma05g03660.8</em> </td><td>7654 </td><td>684 </td><td>0.994 </td><td>0.542 </td><td>0.26 </td><td>5.38 </td><td>1.27 </td><td>0.23 </td><td>0.78 </td><td>1.27 </td><td>0.52 </td><td>1.28 </td><td>I124V (0.17), T218A (1) </td><td>nd </td></tr><tr><td><em>Glyma17g14191.1</em> </td><td>8420 </td><td>663 </td><td>0.991 </td><td>0.546 </td><td>0.002 </td><td>4.18 </td><td>1.41 </td><td>0.004 </td><td>0 </td><td>1.41 </td><td>0 </td><td>1.32 </td><td>G177F (1); T180S (1); N182_V183insDAEL (1) </td><td>nd </td></tr><tr><td><em>Glyma17g08861.1</em> </td><td>5779 </td><td>1143 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.62 </td><td>2.9 </td><td>0.31 </td><td>0 </td><td>0.29 </td><td>0 </td><td>0.38 </td><td>nd </td><td>nd </td></tr></tbody></table></div><div content-id="DST047TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Domestication gene </th><th rowspan="3">Soybean candidate gene </th><th colspan="2" rowspan="2">Length (bp)<hr /> </th><th rowspan="2">ROD </th><th rowspan="2"><em>F</em><sub>ST</sub> </th><th colspan="4">Genetic diversity in genic region<hr /> </th><th colspan="4">Genetic diversity in CDS<hr /> </th><th colspan="2" rowspan="2">Amino acid difference<sup>a</sup> from reference<hr /> </th></tr><tr><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>π</em> × 10<sup>−3</sup><hr /> </th><th colspan="2"><em>θ<sub>w</sub></em> × 10<sup>−3</sup><hr /> </th></tr><tr><th>Genic </th><th>CDS </th><th>100-kb </th><th>100-kb </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Cul </th><th>Wild </th><th>Wild (MAF) </th><th>Cul (MAF) </th></tr></thead><tbody><tr><td><em>tga1</em> </td><td><em>Glyma17g08840.2</em> </td><td>5477 </td><td>1635 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.42 </td><td>0.37 </td><td>0.26 </td><td>0 </td><td>0.37 </td><td>0 </td><td>0.27 </td><td>Q491H (0.5) </td><td>nd </td></tr><tr><td><em>qSH1</em> </td><td><em>Glyma12g29991.3</em> </td><td>7859 </td><td>2385 </td><td>0.994 </td><td>0.427 </td><td>0.31 </td><td>1.88 </td><td>1.12 </td><td>0.54 </td><td>0 </td><td>1.12 </td><td>0 </td><td>1.47 </td><td>N34D (0.17); P42S (0.17); H240L (0.17); C658T (0.17); N674H (1) </td><td>nd </td></tr><tr><td><em>fw2.2</em> </td><td><em>Glyma15g01990.3</em> </td><td>2332 </td><td>507 </td><td>0.995 </td><td>0.667 </td><td>0 </td><td>8.75 </td><td>1.05 </td><td>0 </td><td>0 </td><td>1.05 </td><td>0 </td><td>0.86 </td><td>T648A (0.33) </td><td>nd </td></tr><tr><td><em>PROG1</em> </td><td><em>Glyma17g18110.1</em> </td><td>1601 </td><td>633 </td><td>0.994 </td><td>0.493 </td><td>0.22 </td><td>3.16 </td><td>1.05 </td><td>0.22 </td><td>0.56 </td><td>1.05 </td><td>0.56 </td><td>1.38 </td><td>V21_D22insSW (0.17); N76K (0.17) </td><td>H16D (0.2) </td></tr><tr><td><em>DAG1</em>; <em>DAG2</em> </td><td><em>Glyma16g02550.3</em> </td><td>2343 </td><td>831 </td><td>0.987 </td><td>0.727 </td><td>0 </td><td>0.71 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td><em>Tunicate1</em> </td><td><em>Glyma08g07260.3</em> </td><td>9994 </td><td>615 </td><td>0.997 </td><td>0.470 </td><td>0.19 </td><td>2.87 </td><td>0 </td><td>0.25 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>nd </td><td>nd </td></tr><tr><td rowspan="3"><em>OsMADS56</em> </td><td><em>Glyma05g03660.8</em> </td><td>7654 </td><td>684 </td><td>0.994 </td><td>0.542 </td><td>0.26 </td><td>5.38 </td><td>1.27 </td><td>0.23 </td><td>0.78 </td><td>1.27 </td><td>0.52 </td><td>1.28 </td><td>I124V (0.17), T218A (1) </td><td>nd </td></tr><tr><td><em>Glyma17g14191.1</em> </td><td>8420 </td><td>663 </td><td>0.991 </td><td>0.546 </td><td>0.002 </td><td>4.18 </td><td>1.41 </td><td>0.004 </td><td>0 </td><td>1.41 </td><td>0 </td><td>1.32 </td><td>G177F (1); T180S (1); N182_V183insDAEL (1) </td><td>nd </td></tr><tr><td><em>Glyma17g08861.1</em> </td><td>5779 </td><td>1143 </td><td>0.982 </td><td>0.452 </td><td>0.2 </td><td>2.62 </td><td>2.9 </td><td>0.31 </td><td>0 </td><td>0.29 </td><td>0 </td><td>0.38 </td><td>nd </td><td>nd </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Cul, cultivated soybean; Wild, wild soybean; CDS, coding sequence; MAF, minor allele frequency; nd, not detected.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>The format for an amino acid difference is X#Y, where X is the amino acid of the Williams 82 reference genome, # is the position of the substitution, and Y is the new amino acid; and X#_Y#insAB, where X and Y are the amino acids of the Williams 82 reference genome, #s are the positions of the insertion, and insAB indicates A and B amino acids were inserted.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5954706" target="_blank">View Large</a></div></div><p>We searched for significantly enriched GO terms among genes in CDRs and identified several GO terms reminiscent of domestication-related traits or molecular features of cloned domestication genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S17</a>). The candidate genes were enriched in the GO terms related to seed development, morphology, growth, and transcriptional regulation. Interestingly, the terms ‘embryo development ending in seed dormancy’ and ‘response to abscisic acid stimulus’ in the biological process category, which are related to seed germination, were also observed to be enriched terms (<em>P</em> &lt; 0.001) in the GO analysis of the lost genes described above. In the molecular function and cellular component categories, several terms such as ‘DNA binding’ and ‘nucleus’, which are related to transcription factors, were enriched (<em>P</em> &lt; 0.001). This is somewhat consistent with the fact that the majority of domestication genes obtained by map-based cloning have been transcription factor genes.<sup><a reveal-id="DST047C53" data-open="DST047C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Several GO terms related to transport and stress responses, which play important roles in improving yield and stress resistance,<sup><a reveal-id="DST047C56" data-open="DST047C56" class="link link-ref link-reveal xref-bibr">56</a></sup> were also identified.</p><p>Genes regulating plant morphology, to which most of the domestication-related traits belong, typically show pleiotropic effects as exemplified by the ABC model.<sup><a reveal-id="DST047C57" data-open="DST047C57" class="link link-ref link-reveal xref-bibr">57</a></sup> These genes may appear in more than two enriched GO terms relevant to domestication-related traits in the biological process category. By searching for candidate domestication genes enriched in multiple GO terms, we found three such genes as <em>ROPGEF12</em> (homologous to <em>Glyma07g02250</em>, <em>Glyma13g43380</em>, and <em>Glyma15g01930</em>), <em>TCP4</em> (<em>Glyma12g14200</em>), and <em>TCP5</em> (<em>Glyma17g08761</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). <em>ROPGEF12</em> is involved in polar growth of pollen tubes in <em>Arabidopsis</em><sup><a reveal-id="DST047C58" data-open="DST047C58" class="link link-ref link-reveal xref-bibr">58</a></sup> and is expressed differentially during the pollination process in the maize.<sup><a reveal-id="DST047C59" data-open="DST047C59" class="link link-ref link-reveal xref-bibr">59</a></sup> Among the three <em>ROPGEF12</em> homologues, <em>Glyma07g02250</em> was particularly interesting because its chromosomal location was associated with multiple QTL for plant morphology reported by several independent studies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). The TCP5 homologue, <em>Glyma17g08761</em>, was observed in the CDR-containing homologues of <em>tga1</em> and <em>OsMADS56</em> on Gm17 described above. The soybean genome encodes 57 predicted TCPs. By examining the genes in CDRs, we found two additional TCPs, <em>Glyma05g03610</em> and <em>Glyma17g14160</em>, whose gene symbols have not been assigned. Our phylogenetic analysis showed that <em>Glyma12g14200</em> and <em>Glyma17g08761</em> belong to the CIN clade of Class II TCPs, and <em>Glyma05g03610</em> and <em>Glyma17g14160</em> belong to Class I TCPs<sup><a reveal-id="DST047C60" data-open="DST047C60" class="link link-ref link-reveal xref-bibr">60</a></sup> (Fig. <a reveal-id="DST047F3" data-open="DST047F3" class="link link-reveal link-table xref-fig">3</a>b and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>). These results suggested that our CDRs do not contain a TCP gene belonging to the CYC/TB1 clade containing the canonical domestication gene <em>tb1</em>. Nevertheless, as most of the functionally studied TCPs are involved in cell proliferation and branching, these four soybean genes are strong candidate domestication genes. This notion is further supported by that these genes are associated with multiple QTL for plant morphology (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). Interestingly, the three <em>ROPGEF12</em> homologues are located on the three duplicated chromosomal segments from palaeopolyploidization<sup><a reveal-id="DST047C36" data-open="DST047C36" class="link link-ref link-reveal xref-bibr">36</a></sup> (Fig. <a reveal-id="DST047F4" data-open="DST047F4" class="link link-reveal link-table xref-fig">4</a>). Two TCP genes, <em>Glyma05g03610</em> and <em>Glyma17g14160</em>, whose TCP domains are 100% identical but whose overall identity is 79%, are also located on the duplicated chromosomal segments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S10</a>). These observations likely reflect the important role that co-selection between these ancient duplicated segments has had in soybean domestication. </p>                        <a id="5954709" scrollto-destination="5954709"></a>
<div id="DST047F4" data-id="DST047F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04704.png?Expires=2147483647&Signature=aVBqoPYFVqqCx-A0tKBdIoHMSDYpKwG8bgI-NU1y-OMah5sQtG~UNN0hAPZWfNS~vV3swt-WFQiaJf-m9~iIcY7BgmyImDfwdFyWBAVhRtlO3RCceZHYtzHORM10TqxquUX0kcooZOsWaTa5NwqvfjlEV0STYMN75NdoNDeT-iN47izX0tLxyIzI7AzC47lF8019ptBu3XrpgjEp8Hx1V4eptUKnv4SjhlnhjwiKHgRJ7enxe7HooPXmx0mAWRN5NGp1Rkv80vdjouZK1ijQO1y9Utne9TEukEyo-npygso37NKk9VaeU1KjqlQBEl0rQBxIGDYrl76j9ORQpqggdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Selection of three ROPGEF12 homeologs. (a) ROD (red), and average fixation index, FST (blue), plotted for 100-kb windows (upper panel) or 10-kb windows (lower panel) across 10-Mb or 1-Mb regions, respectively, of three duplicated chromosomal segments from palaeopolyploidization harbouring a ROPGEF12 homologue. Grey boxes indicate 140-kb (Gm07), 100-kb (Gm13), and 360-kb (Gm15) chromosomal regions having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, –log10(1 − ROD) values of ≥4 are shown as corresponding with –log10(1 − ROD) = 4. (b) Homeologous (duplicated) relationship between genes on the three duplicated chromosomal segments. Predicted genes are indicated by coloured block arrows except black arrows for ROPGEF12 homologues. Grey boxes between genes show homeologs." path-from-xml="dst04704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954709/dst04704.tiff" path-from-xml="dst04704.tiff" target="_blank">View large</a><a section="5954709" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04704.png?Expires=2147483647&Signature=oE5vMZSm2JsBHbIa7fEVOGTNdFwSHYLgdXVqPWwu~QEQwhty1gLP2LQXHOuqPUI0jxBSjn8j4N8IGfW6o2pWq3lNT0si99JfUPRmPJ3oQgmcRq8wwGWlU2Kxm3~BiDnTgy-odDn4YZfSTuvARgxMIfSIOhU1jQ3DGd0Lj6QD3UyoyGPEBzS68diIiNTrJBP2y6n2NPGZYoIZcNYPzzhEFfS8NbHRtFXZeNsbguQ9hbrBxoe5aJOya~RmHZ80AxADJn5Bu17RXPbAZJZQzioBeJf8BrtWgrkANY6byOy8fmYlnn2mxR7rc206hIMu4GihPd-kh8i6-n9rjEx~sgZG5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954709&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Selection of three ROPGEF12 homeologs. (a) ROD (red), and average fixation index, <em>F</em><sub>ST</sub> (blue), plotted for 100-kb windows (upper panel) or 10-kb windows (lower panel) across 10-Mb or 1-Mb regions, respectively, of three duplicated chromosomal segments from palaeopolyploidization harbouring a ROPGEF12 homologue. Grey boxes indicate 140-kb (Gm07), 100-kb (Gm13), and 360-kb (Gm15) chromosomal regions having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, <em>–</em>log<sub>10</sub>(1 − ROD) values of ≥4 are shown as corresponding with <em>–</em>log<sub>10</sub>(1 − ROD) = 4. (b) Homeologous (duplicated) relationship between genes on the three duplicated chromosomal segments. Predicted genes are indicated by coloured block arrows except black arrows for ROPGEF12 homologues. Grey boxes between genes show homeologs.</p></div></div><div content-id="DST047F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/m_dst04704.png?Expires=2147483647&Signature=aVBqoPYFVqqCx-A0tKBdIoHMSDYpKwG8bgI-NU1y-OMah5sQtG~UNN0hAPZWfNS~vV3swt-WFQiaJf-m9~iIcY7BgmyImDfwdFyWBAVhRtlO3RCceZHYtzHORM10TqxquUX0kcooZOsWaTa5NwqvfjlEV0STYMN75NdoNDeT-iN47izX0tLxyIzI7AzC47lF8019ptBu3XrpgjEp8Hx1V4eptUKnv4SjhlnhjwiKHgRJ7enxe7HooPXmx0mAWRN5NGp1Rkv80vdjouZK1ijQO1y9Utne9TEukEyo-npygso37NKk9VaeU1KjqlQBEl0rQBxIGDYrl76j9ORQpqggdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Selection of three ROPGEF12 homeologs. (a) ROD (red), and average fixation index, FST (blue), plotted for 100-kb windows (upper panel) or 10-kb windows (lower panel) across 10-Mb or 1-Mb regions, respectively, of three duplicated chromosomal segments from palaeopolyploidization harbouring a ROPGEF12 homologue. Grey boxes indicate 140-kb (Gm07), 100-kb (Gm13), and 360-kb (Gm15) chromosomal regions having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, –log10(1 − ROD) values of ≥4 are shown as corresponding with –log10(1 − ROD) = 4. (b) Homeologous (duplicated) relationship between genes on the three duplicated chromosomal segments. Predicted genes are indicated by coloured block arrows except black arrows for ROPGEF12 homologues. Grey boxes between genes show homeologs." path-from-xml="dst04704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5954709/dst04704.tiff" path-from-xml="dst04704.tiff" target="_blank">View large</a><a section="5954709" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst04704.png?Expires=2147483647&Signature=oE5vMZSm2JsBHbIa7fEVOGTNdFwSHYLgdXVqPWwu~QEQwhty1gLP2LQXHOuqPUI0jxBSjn8j4N8IGfW6o2pWq3lNT0si99JfUPRmPJ3oQgmcRq8wwGWlU2Kxm3~BiDnTgy-odDn4YZfSTuvARgxMIfSIOhU1jQ3DGd0Lj6QD3UyoyGPEBzS68diIiNTrJBP2y6n2NPGZYoIZcNYPzzhEFfS8NbHRtFXZeNsbguQ9hbrBxoe5aJOya~RmHZ80AxADJn5Bu17RXPbAZJZQzioBeJf8BrtWgrkANY6byOy8fmYlnn2mxR7rc206hIMu4GihPd-kh8i6-n9rjEx~sgZG5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5954709&ar=403337&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst04704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Selection of three ROPGEF12 homeologs. (a) ROD (red), and average fixation index, <em>F</em><sub>ST</sub> (blue), plotted for 100-kb windows (upper panel) or 10-kb windows (lower panel) across 10-Mb or 1-Mb regions, respectively, of three duplicated chromosomal segments from palaeopolyploidization harbouring a ROPGEF12 homologue. Grey boxes indicate 140-kb (Gm07), 100-kb (Gm13), and 360-kb (Gm15) chromosomal regions having &gt;0.98 ROD values in 100-kb windows and its corresponding region in 10-kb windows. For simplicity, <em>–</em>log<sub>10</sub>(1 − ROD) values of ≥4 are shown as corresponding with <em>–</em>log<sub>10</sub>(1 − ROD) = 4. (b) Homeologous (duplicated) relationship between genes on the three duplicated chromosomal segments. Predicted genes are indicated by coloured block arrows except black arrows for ROPGEF12 homologues. Grey boxes between genes show homeologs.</p></div></div>                            <h2 scrollto-destination=5954710 id="5954710" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p>In this study, we have provided a large genome variation dataset for the wild and cultivated soybeans. Millions of variations in the representative wild and cultivated soybean strains provided an opportunity to finely resolve the domestication or improvement history of cultivated soybean. The population structure and phylogenetic analyses not only support the hypothesis that cultivated soybeans (<em>G. max</em>) are a subclade of their progenitor wild soybeans (<em>G. soja</em>), but also disprove the hypothesis of multiple soybean domestication events in East Asia. We identified thousands of candidate genes that may have been artificially selected during the soybean domestication or improvement. The SNPs will be useful as dense markers of genome variation for marker-assisted mapping of important soybean traits as well as for pinpointing agronomically important genes in soybeans. The candidate genes selected during domestication may be agronomically important, and our results generally support that a translational genomics study would be productive for identifying the soybean domestication genes. Taken together with the data from Lam <em>et al</em>.<sup><a reveal-id="DST047C5" data-open="DST047C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and from Li <em>et al</em>.,<sup><a reveal-id="DST047C61" data-open="DST047C61" class="link link-ref link-reveal xref-bibr">61</a></sup> which was published after the submission of the present paper with 25 soybean genomes sequenced to a low depth with a mean of 3.38×, the data generated in this study provide a valuable resource for improving soybean as well as elucidating the origin and evolution of soybean.</p>                            <h2 scrollto-destination=5954712 id="5954712" class="section-title" data-legacy-id=s5>Authors' contributions</h2>
<p>W.H.C., N.J., N.K., and S.C.J. wrote the article. J.K.M., N.K., and S.C.J. designed the research. W.H.C., J.K., Y.G.L., S.H.L., W.Y., J.H.K., H.K.C., N.K., and S.C.J. carried out analysis of the genome sequences. N.J., W.K.L., I.Y.C., and S.C.J. performed genome sequencing and plant experiments.</p>                            <h2 scrollto-destination=5954714 id="5954714" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5954716 id="5954716" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This study was supported principally by a grant from the Next-Generation BioGreen 21 Program (PJ008124), by the Rural Development Administration, and partly by supported by the National Research Foundation of Korea grant funded by the Korea government (no. 20110030770) and by the Korea Research Institute of Bioscience and Biotechnology Research Initiative Program.</p>                            <h2 scrollto-destination=5954718 id="5954718" class="backacknowledgements-title" >Acknowledgments</h2>
<p>We thank Dr Jeff Doyle for discussion and Dr Jungho Park for artwork assistance.</p>                            <h2 scrollto-destination=5954720 id="5954720" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST047C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Crawford</div> <div class="given-names">G.W.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>. </div><div class="article-title">Archaeological soybean (<em>Glycine max</em>) in East Asia: does size matter?</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e26720</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaeological%20soybean%20(Glycine%20max)%20in%20East%20Asia%3A%20does%20size%20matter%3F&amp;author=G.A.%20Lee&amp;author=G.W.%20Crawford&amp;author=L.%20Liu&amp;author=Y.%20Sasaki&amp;author=X.%20Chen&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e26720" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0026720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0026720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0026720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22073186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shimamoto</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Diversity of chloroplast DNA SSRs in wild and cultivated soybeans: evidence for multiple origins of cultivated soybean</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">645</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20chloroplast%20DNA%20SSRs%20in%20wild%20and%20cultivated%20soybeans%3A%20evidence%20for%20multiple%20origins%20of%20cultivated%20soybean&amp;author=H.%20Xu&amp;author=J.%20Abe&amp;author=Y.%20Gai&amp;author=Y.%20Shimamoto&amp;publication_year=2002&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=105&amp;pages=645-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-002-0972-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-002-0972-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-002-0972-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12582476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirschhorn</div> <div class="given-names">J.N.</div></div>,  <div class="name"><div class="surname">Daly</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Genome-wide association studies for common diseases and complex traits</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">95</div>-<div class="lpage">108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20for%20common%20diseases%20and%20complex%20traits&amp;author=J.N.%20Hirschhorn&amp;author=M.J.%20Daly&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=95-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1521"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15716906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Sang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide association studies of 14 agronomic traits in rice landraces</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">961</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%2014%20agronomic%20traits%20in%20rice%20landraces&amp;author=X.%20Huang&amp;author=X.%20Wei&amp;author=T.%20Sang&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=961-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20972439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lam</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Resequencing of 31 wild and cultivated soybean genomes identifies patterns of genetic diversity and selection</div>, <div class="source">Nat. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">1053</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%20of%2031%20wild%20and%20cultivated%20soybean%20genomes%20identifies%20patterns%20of%20genetic%20diversity%20and%20selection&amp;author=H.M.%20Lam&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;publication_year=2010&amp;journal=Nat.%20Genet.&amp;volume=42&amp;pages=1053-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.715" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.715" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.715"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Kurata</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">A map of rice genome variation reveals the origin of cultivated rice</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">490</div> (pg. <div class="fpage">497</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20map%20of%20rice%20genome%20variation%20reveals%20the%20origin%20of%20cultivated%20rice&amp;author=X.%20Huang&amp;author=N.%20Kurata&amp;author=X.%20Wei&amp;publication_year=2012&amp;journal=Nature&amp;volume=490&amp;pages=497-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11532" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11532" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11532"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23034647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hufford</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">van Heerwaarden</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Comparative population genomics of maize domestication and improvement</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">808</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20population%20genomics%20of%20maize%20domestication%20and%20improvement&amp;author=M.B.%20Hufford&amp;author=X.%20Xu&amp;author=J.%20van%20Heerwaarden&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=808-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2309"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Resequencing 50 accessions of cultivated and wild rice yields markers for identifying agronomically important genes</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">105</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resequencing%2050%20accessions%20of%20cultivated%20and%20wild%20rice%20yields%20markers%20for%20identifying%20agronomically%20important%20genes&amp;author=X.%20Xu&amp;author=X.%20Liu&amp;author=S.%20Ge&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2050"> </span></div> </div></div></div></div></div><div content-id="DST047C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">956</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20multiple%20Arabidopsis%20thaliana%20populations&amp;author=J.%20Cao&amp;author=K.%20Schneeberger&amp;author=S.%20Ossowski&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=956-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Stegle</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Behr</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Multiple reference genomes and transcriptomes for <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">477</div> (pg. <div class="fpage">419</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20reference%20genomes%20and%20transcriptomes%20for%20Arabidopsis%20thaliana&amp;author=X.%20Gan&amp;author=O.%20Stegle&amp;author=J.%20Behr&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=419-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Van</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Whole-genome sequencing and intensive analysis of the undomesticated soybean (<em>Glycine soja</em> Sieb. and Zucc.) genome</div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">22032</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20and%20intensive%20analysis%20of%20the%20undomesticated%20soybean%20(Glycine%20soja%20Sieb.%20and%20Zucc.)%20genome&amp;author=M.Y.%20Kim&amp;author=S.%20Lee&amp;author=K.%20Van&amp;publication_year=2010&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=107&amp;pages=22032-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1009526107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1009526107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1009526107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21131573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div>. </div><div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">764</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">The sequence and de novo assembly of the giant panda genome</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">311</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20and%20de%20novo%20assembly%20of%20the%20giant%20panda%20genome&amp;author=R.%20Li&amp;author=W.%20Fan&amp;author=G.%20Tian&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=311-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1754</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McKenna</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1297</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DePristo</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">491</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">W.K.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Dynamic genetic features of chromosomes revealed by comparison of soybean genetic and sequence-based physical maps</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">1103</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20genetic%20features%20of%20chromosomes%20revealed%20by%20comparison%20of%20soybean%20genetic%20and%20sequence-based%20physical%20maps&amp;author=W.K.%20Lee&amp;author=N.%20Kim&amp;author=J.%20Kim&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=1103-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-2039-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-2039-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-2039-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23306355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</div>, <div class="source">Gigascience</div> , <div class="year">2012</div>, vol. <div class="volume">1</div> pg. <div class="fpage">18</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigascience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tzvetkova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>. </div><div class="article-title">AUGUSTUS at EGASP: using EST, protein and genomic alignments for improved gene prediction in the human genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> <div class="issue">Suppl 1</div>pg. <div class="fpage">S11</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%20at%20EGASP%3A%20using%20EST%2C%20protein%20and%20genomic%20alignments%20for%20improved%20gene%20prediction%20in%20the%20human%20genome&amp;author=M.%20Stanke&amp;author=A.%20Tzvetkova&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=S11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-s1-s11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-s1-s11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-s1-s11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16925833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">InterProScan—an integration platform for the signature-recognition methods in InterPro</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">847</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%E2%80%94an%20integration%20platform%20for%20the%20signature-recognition%20methods%20in%20InterPro&amp;author=E.M.%20Zdobnov&amp;author=R.%20Apweiler&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=847-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.9.847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.9.847"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11590104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tate</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2010</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">D211</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=J.%20Mistry&amp;author=J.%20Tate&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D211-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19920124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jeong</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">J.Y.</div></div>, et al. </div><div class="article-title">Structure, expression, and mapping of two nodule-specific genes identified by mining public soybean EST databases</div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">383</div> (pg. <div class="fpage">71</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2C%20expression%2C%20and%20mapping%20of%20two%20nodule-specific%20genes%20identified%20by%20mining%20public%20soybean%20EST%20databases&amp;author=S.C.%20Jeong&amp;author=K.%20Yang&amp;author=J.Y.%20Park&amp;publication_year=2006&amp;journal=Gene&amp;volume=383&amp;pages=71-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.07.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.07.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.07.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Handsaker</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Korn</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Nemesh</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">McCarroll</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Discovery and genotyping of genome structural polymorphism by sequencing on a population scale</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">269</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20genotyping%20of%20genome%20structural%20polymorphism%20by%20sequencing%20on%20a%20population%20scale&amp;author=R.E.%20Handsaker&amp;author=J.M.%20Korn&amp;author=J.%20Nemesh&amp;author=S.A.%20McCarroll&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21317889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hubisz</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Falush</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Inferring weak population structure with the assistance of sample group information</div>, <div class="source">Mol. Ecol. Resour.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1322</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20weak%20population%20structure%20with%20the%20assistance%20of%20sample%20group%20information&amp;author=M.J.%20Hubisz&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2009&amp;journal=Mol.%20Ecol.%20Resour.&amp;volume=9&amp;pages=1322-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2009.9.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2009.9.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2009.9.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21564903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Falush</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">1567</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1567-87" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barrett</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Fry</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Maller</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Daly</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Haploview: analysis and visualization of LD and haplotype maps</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">263</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Haploview%3A%20analysis%20and%20visualization%20of%20LD%20and%20haplotype%20maps&amp;author=J.C.%20Barrett&amp;author=B.%20Fry&amp;author=J.%20Maller&amp;author=M.J.%20Daly&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=263-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth457"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15297300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watterson</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">On the number of segregating sites in genetical models without recombination</div>, <div class="source">Theor. Popul. Biol.</div> , <div class="year">1975</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">256</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20number%20of%20segregating%20sites%20in%20genetical%20models%20without%20recombination&amp;author=G.A.%20Watterson&amp;publication_year=1975&amp;journal=Theor.%20Popul.%20Biol.&amp;volume=7&amp;pages=256-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0040-5809(75)90020-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0040-5809(75)90020-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0040-5809(75)90020-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1145509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Evolutionary relationship of DNA sequences in finite populations</div>, <div class="source">Genetics</div> , <div class="year">1983</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">437</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20relationship%20of%20DNA%20sequences%20in%20finite%20populations&amp;author=F.%20Tajima&amp;publication_year=1983&amp;journal=Genetics&amp;volume=105&amp;pages=437-60" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6628982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hudson</div> <div class="given-names">R.R.</div></div>,  <div class="name"><div class="surname">Slatkin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maddison</div> <div class="given-names">W.P.</div></div>. </div><div class="article-title">Estimation of levels of gene flow from DNA sequence data</div>, <div class="source">Genetics</div> , <div class="year">1992</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">583</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20levels%20of%20gene%20flow%20from%20DNA%20sequence%20data&amp;author=R.R.%20Hudson&amp;author=M.%20Slatkin&amp;author=W.P.%20Maddison&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=583-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1427045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Broich</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Palmer</div> <div class="given-names">R.G.</div></div>. </div><div class="article-title">Evolutionary studies of the soybean: the frequency and distribution of alleles among collections of <em>Glycine max</em> and <em>G. soja</em> of various origin</div>, <div class="source">Euphytica</div> , <div class="year">1981</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">55</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20studies%20of%20the%20soybean%3A%20the%20frequency%20and%20distribution%20of%20alleles%20among%20collections%20of%20Glycine%20max%20and%20G.%20soja%20of%20various%20origin&amp;author=S.L.%20Broich&amp;author=R.G.%20Palmer&amp;publication_year=1981&amp;journal=Euphytica&amp;volume=30&amp;pages=55-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00033659" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00033659" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00033659"> </span></div> </div></div></div></div></div><div content-id="DST047C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hymowitz</div> <div class="given-names">T.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Boerma</div> <div class="given-names">H.R.</div></div>,  <div class="name"><div class="surname">Specht</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Speciation and cytogenetics</div>, <div class="source">Soybeans: improvement, production, and uses</div> , <div class="year">2004</div><div class="publisher-loc">Madison, Wisconsin</div><div class="publisher-name">American Society of Agronomy</div>(pg. <div class="fpage">97</div>-<div class="lpage">136</div>)</div></div></div></div><div content-id="DST047C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Glycine max (L.) Merr. subsp. soja (Sieb and Zucc.) Ohashi, comb. nov</div>, <div class="source">J. Jpn. Bot.</div> , <div class="year">1982</div>, vol. <div class="volume">57</div> pg. <div class="fpage">30</div> </div></div></div></div><div content-id="DST047C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kanazawa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shimamoto</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Soybean germplasm pools in Asia revealed by nuclear SSRs</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">445</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soybean%20germplasm%20pools%20in%20Asia%20revealed%20by%20nuclear%20SSRs&amp;author=J.%20Abe&amp;author=D.H.%20Xu&amp;author=Y.%20Suzuki&amp;author=A.%20Kanazawa&amp;author=Y.%20Shimamoto&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=106&amp;pages=445-53" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12589544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">The Genomes of <em>Oryza sativa</em>: a history of duplications</div>, <div class="source">PLoS Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e38</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genomes%20of%20Oryza%20sativa%3A%20a%20history%20of%20duplications&amp;author=J.%20Yu&amp;author=J.%20Wang&amp;author=W.%20Lin&amp;publication_year=2005&amp;journal=PLoS%20Biol.&amp;volume=3&amp;pages=e38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15685292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arumuganathan</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Earle</div> <div class="given-names">E.D.</div></div>. </div><div class="article-title">Nuclear DNA content of some important plant species</div>, <div class="source">Plant Mol. Biol. Rep.</div> , <div class="year">1991</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">208</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20of%20some%20important%20plant%20species&amp;author=K.%20Arumuganathan&amp;author=E.D.%20Earle&amp;publication_year=1991&amp;journal=Plant%20Mol.%20Biol.%20Rep.&amp;volume=9&amp;pages=208-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02672069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02672069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02672069"> </span></div> </div></div></div></div></div><div content-id="DST047C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tajima</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Statistical method for testing the neutral mutation hypothesis by DNA polymorphism</div>, <div class="source">Genetics</div> , <div class="year">1989</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">585</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20method%20for%20testing%20the%20neutral%20mutation%20hypothesis%20by%20DNA%20polymorphism&amp;author=F.%20Tajima&amp;publication_year=1989&amp;journal=Genetics&amp;volume=123&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2513255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liao</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Overexpressing AtPAP15 enhances phosphorus efficiency in soybean</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">233</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overexpressing%20AtPAP15%20enhances%20phosphorus%20efficiency%20in%20soybean&amp;author=X.%20Wang&amp;author=Y.%20Wang&amp;author=J.%20Tian&amp;author=B.L.%20Lim&amp;author=X.%20Yan&amp;author=H.%20Liao&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=233-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.138891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.138891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.138891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19587103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dionisio</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">C.K.</div></div>,  <div class="name"><div class="surname">Holm</div> <div class="given-names">P.B.</div></div>, et al. </div><div class="article-title">Cloning and characterization of purple acid phosphatase phytases from wheat, barley, maize, and rice</div>, <div class="source">Plant Physiol.</div> , <div class="year">2011</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1087</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20purple%20acid%20phosphatase%20phytases%20from%20wheat%2C%20barley%2C%20maize%2C%20and%20rice&amp;author=G.%20Dionisio&amp;author=C.K.%20Madsen&amp;author=P.B.%20Holm&amp;publication_year=2011&amp;journal=Plant%20Physiol.&amp;volume=156&amp;pages=1087-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.164756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.164756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.164756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21220762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maupin</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Rainey</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Improving emergence of modified phosphorus composition soybeans: genotypes, germplasm, environments, and selection</div>, <div class="source">Crop Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">1946</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20emergence%20of%20modified%20phosphorus%20composition%20soybeans%3A%20genotypes%2C%20germplasm%2C%20environments%2C%20and%20selection&amp;author=L.M.%20Maupin&amp;author=K.M.%20Rainey&amp;publication_year=2011&amp;journal=Crop%20Sci.&amp;volume=51&amp;pages=1946-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2010.10.0585" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2010.10.0585" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2010.10.0585"> </span></div> </div></div></div></div></div><div content-id="DST047C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">J.</div></div>. </div><div class="article-title">QTL mapping of domestication-related traits in soybean (<em>Glycine max</em>)</div>, <div class="source">Ann. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">1027</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20domestication-related%20traits%20in%20soybean%20(Glycine%20max)&amp;author=B.%20Liu&amp;author=T.%20Fujita&amp;author=Z.H.%20Yan&amp;author=S.%20Sakamoto&amp;author=D.%20Xu&amp;author=J.%20Abe&amp;publication_year=2007&amp;journal=Ann.%20Bot.&amp;volume=100&amp;pages=1027-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcm149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcm149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcm149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17684023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hyten</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">I.Y.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">Highly variable patterns of linkage disequilibrium in multiple soybean populations</div>, <div class="source">Genetics</div> , <div class="year">2007</div>, vol. <div class="volume">175</div> (pg. <div class="fpage">1937</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20variable%20patterns%20of%20linkage%20disequilibrium%20in%20multiple%20soybean%20populations&amp;author=D.L.%20Hyten&amp;author=I.Y.%20Choi&amp;author=Q.%20Song&amp;publication_year=2007&amp;journal=Genetics&amp;volume=175&amp;pages=1937-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.069740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.069740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.069740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17287533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alexander</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Novembre</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lange</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Fast model-based estimation of ancestry in unrelated individuals</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1655</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20model-based%20estimation%20of%20ancestry%20in%20unrelated%20individuals&amp;author=D.H.%20Alexander&amp;author=J.%20Novembre&amp;author=K.%20Lange&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1655-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.094052.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.094052.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.094052.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19648217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wright</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Bi</div> <div class="given-names">I.V.</div></div>,  <div class="name"><div class="surname">Schroeder</div> <div class="given-names">S.G.</div></div>, et al. </div><div class="article-title">The effects of artificial selection on the maize genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">308</div> (pg. <div class="fpage">1310</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effects%20of%20artificial%20selection%20on%20the%20maize%20genome&amp;author=S.I.%20Wright&amp;author=I.V.%20Bi&amp;author=S.G.%20Schroeder&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=1310-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15919994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibbs</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Van Tassell</div> <div class="given-names">C.P.</div></div>, et al. </div><div class="article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">528</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20SNP%20variation%20uncovers%20the%20genetic%20structure%20of%20cattle%20breeds&amp;author=R.A.%20Gibbs&amp;author=J.F.%20Taylor&amp;author=C.P.%20Van%20Tassell&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=528-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1167936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1167936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1167936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Axelsson</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ratnakumar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Arendt</div> <div class="given-names">M.L.</div></div>, et al. </div><div class="article-title">The genomic signature of dog domestication reveals adaptation to a starch-rich diet</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">495</div> (pg. <div class="fpage">360</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20signature%20of%20dog%20domestication%20reveals%20adaptation%20to%20a%20starch-rich%20diet&amp;author=E.%20Axelsson&amp;author=A.%20Ratnakumar&amp;author=M.L.%20Arendt&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=360-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11837" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11837" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11837"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23354050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rafalski</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Morgante</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Corn and humans: recombination and linkage disequilibrium in two genomes of similar size</div>, <div class="source">Trends Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">103</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Corn%20and%20humans%3A%20recombination%20and%20linkage%20disequilibrium%20in%20two%20genomes%20of%20similar%20size&amp;author=A.%20Rafalski&amp;author=M.%20Morgante&amp;publication_year=2004&amp;journal=Trends%20Genet.&amp;volume=20&amp;pages=103-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2003.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2003.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2003.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14746992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gore</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Chia</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Elshire</div> <div class="given-names">R.J.</div></div>, et al. </div><div class="article-title">A first-generation haplotype map of maize</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1115</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first-generation%20haplotype%20map%20of%20maize&amp;author=M.A.%20Gore&amp;author=J.M.%20Chia&amp;author=R.J.%20Elshire&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1115-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1177837" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1177837" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1177837"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Plagnol</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">T.T.</div></div>, et al. </div><div class="article-title">Recombination and linkage disequilibrium in <em>Arabidopsis thaliana</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">1151</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20and%20linkage%20disequilibrium%20in%20Arabidopsis%20thaliana&amp;author=S.%20Kim&amp;author=V.%20Plagnol&amp;author=T.T.%20Hu&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=1151-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17676040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doebley</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">B.D.</div></div>. </div><div class="article-title">The molecular genetics of crop domestication</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">1309</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20genetics%20of%20crop%20domestication&amp;author=J.F.%20Doebley&amp;author=B.S.%20Gaut&amp;author=B.D.%20Smith&amp;publication_year=2006&amp;journal=Cell&amp;volume=127&amp;pages=1309-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17190597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nussbaum-Wagler</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">The origin of the naked grains of maize</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">714</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20the%20naked%20grains%20of%20maize&amp;author=H.%20Wang&amp;author=T.%20Nussbaum-Wagler&amp;author=B.%20Li&amp;publication_year=2005&amp;journal=Nature&amp;volume=436&amp;pages=714-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03863"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16079849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">J.P.</div></div>, et al. </div><div class="article-title">Genetic control of rice plant architecture under domestication</div>, <div class="source">Nat. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">1365</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20rice%20plant%20architecture%20under%20domestication&amp;author=J.%20Jin&amp;author=W.%20Huang&amp;author=J.P.%20Gao&amp;publication_year=2008&amp;journal=Nat.%20Genet.&amp;volume=40&amp;pages=1365-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18820696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schroeder</div> <div class="given-names">J.I.</div></div>,  <div class="name"><div class="surname">Delhaize</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Frommer</div> <div class="given-names">W.B.</div></div>, et al. </div><div class="article-title">Using membrane transporters to improve crops for sustainable food production</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">497</div> (pg. <div class="fpage">60</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20membrane%20transporters%20to%20improve%20crops%20for%20sustainable%20food%20production&amp;author=J.I.%20Schroeder&amp;author=E.%20Delhaize&amp;author=W.B.%20Frommer&amp;publication_year=2013&amp;journal=Nature&amp;volume=497&amp;pages=60-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23636397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coen</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Meyerowitz</div> <div class="given-names">E.M.</div></div>. </div><div class="article-title">The war of the whorls: genetic interactions controlling flower development</div>, <div class="source">Nature</div> , <div class="year">1991</div>, vol. <div class="volume">353</div> (pg. <div class="fpage">31</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20war%20of%20the%20whorls%3A%20genetic%20interactions%20controlling%20flower%20development&amp;author=E.S.%20Coen&amp;author=E.M.%20Meyerowitz&amp;publication_year=1991&amp;journal=Nature&amp;volume=353&amp;pages=31-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/353031a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F353031a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F353031a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1715520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">McCormick</div> <div class="given-names">S.</div></div>. </div><div class="article-title">A distinct mechanism regulating a pollen-specific guanine nucleotide exchange factor for the small GTPase Rop in <em>Arabidopsis thaliana</em></div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">18830</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20distinct%20mechanism%20regulating%20a%20pollen-specific%20guanine%20nucleotide%20exchange%20factor%20for%20the%20small%20GTPase%20Rop%20in%20Arabidopsis%20thaliana&amp;author=Y.%20Zhang&amp;author=S.%20McCormick&amp;publication_year=2007&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=104&amp;pages=18830-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0705874104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0705874104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0705874104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18000057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xu</div> <div class="given-names">X.H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.S.</div></div>. </div><div class="article-title">Transcript profile analyses of maize silks reveal effective activation of genes involved in microtubule-based movement, ubiquitin-dependent protein degradation, and transport in the pollination process</div>, <div class="source">PLoS One</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e53545</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20profile%20analyses%20of%20maize%20silks%20reveal%20effective%20activation%20of%20genes%20involved%20in%20microtubule-based%20movement%2C%20ubiquitin-dependent%20protein%20degradation%2C%20and%20transport%20in%20the%20pollination%20process&amp;author=X.H.%20Xu&amp;author=F.%20Wang&amp;author=H.%20Chen&amp;author=W.%20Sun&amp;author=X.S.%20Zhang&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e53545" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0053545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0053545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0053545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23301084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martín-Trillo</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cubas</div> <div class="given-names">P.</div></div>. </div><div class="article-title">TCP genes: a family snapshot ten years later</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2010</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">31</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TCP%20genes%3A%20a%20family%20snapshot%20ten%20years%20later&amp;author=M.%20Mart%C3%ADn-Trillo&amp;author=P.%20Cubas&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci.&amp;volume=15&amp;pages=31-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2009.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2009.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2009.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19963426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST047C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST047C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Y.H.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">J.X.</div></div>, et al. </div><div class="article-title">Molecular footprints of domestication and improvement in soybean revealed by whole genome re-sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">579</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20footprints%20of%20domestication%20and%20improvement%20in%20soybean%20revealed%20by%20whole%20genome%20re-sequencing&amp;author=Y.H.%20Li&amp;author=S.C.%20Zhao&amp;author=J.X.%20Ma&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=579" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-579" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-579" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-579"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23984715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=403337&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst047-dst047supp_Data_Set3.zip?Expires=2147483647&amp;Signature=dbMh1YsUZwUP2JZHHEOKcGiSN-lZOiZ37sc7Sw4u9mmIGy9XHDA94l6v3ZaJxiaiKlg0oCMNGJe81TM6BqIffyEeSXcsjJjrtZAHOOAJFkrFFPNy3A--IXUNJvCs4Xq5h4ve11bgHm0x5ID66g5LmiGo8TGMKENOblrX2Xv6ZVX0LCJVC1wOfJubIwbOyz4Gj4kqogyflW93RsOq~3ojqrwcYCAtCpYAsFQb-aJSyBPA5kdYEKjaADGxGPWlqEzZ5ur2wLVivMooAVnzSIEUNYy7e2bJA-MNiyXKl3Qr~GSQ~oxFUUt-Ci1u6IkgS~9eR463XEP~gv5eIOR1qzoqBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - zip file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst047-dst047supp_Data_Set2.zip?Expires=2147483647&amp;Signature=ia2MmJYYfFJLv-c3LD1YOxyLpA-pqoRBdZvv3Y-yxRYRx3PlBBDR7Hyn1lrk2jAxK9EcNXivv3KoFTuRMULgDUkN5QnIIVF5xaUQAilNncDpS6PyqSR~nbXNA~GYDD85wwYHNUnz0~59odVaDCHAHMKyRBzb0pqqmOVT7ISgMvV6XC9FwK11lFrrtlVm2~xH~PrX3Z-Ogju93dUhn0f~Ck2owNfLaVBlXyY7a6at9EfxG1LEtL6FHa5zaleDn09Akkk7duGejhXEEr2m5XyWwGQbjBCRpLrfIPSSigW~1Xd~61ub6n5SD7f41soJRTMB2Z8j2BtI7hi2Bma2Xuindg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - zip file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst047-dst047supp_Data_Set1.zip?Expires=2147483647&amp;Signature=Efm24Cc3ZYBrgHR8-pSAL7ePdttGL97uxJkF7sLuZxFvVdvFyubljs1SVT0s7uCmCgdstCmkbIJv31BY8hF3jjSwWo4fZHMIJhGNxGRlwhu0LOCOV6YILcITSGYoTIk3unVPxgHOZwjjOiyDG7x9uv9bD~W2qAyWJPClVoXIdBfP3pPOCRIAU86jePvf8mafHPA84K4elowKNV-68MyTvS466u5fRAiHLeJEzJIw2coNUz-U2FmEoz1zYYqgaLyVwoJaWT01XDgzbhZ6Mlj0qUIUCN75Y8VcjyRrdlphKprBX8Mo1FwSeZjk0OltAA~Obn8xbN6F-aWKif0Uk6UjQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - zip file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst047-dst047supp2.xlsx?Expires=2147483647&amp;Signature=t79yn6nHHC9yrffKX88Bs9m9alybhnlp9jvXflQqu3NUWssjjK4s~sVvr2iMapY3GapNy9i1D2VJVTtt4mwW0XbobdfJkqADcIro3zbULyBptVNM7JvJKPrJ1kBJgivSLQD5ZTYtxjm2Ja5M8QwYSj87vPojakvSgLIogtsyxPEpKs2A2zupWQAU9s63-n8dFXYupLOlPSzn54k5TcrPsgTg0CihW6QTyfDd8CQWB5gGAAb9jxq0vY7nzhRHu15bGoaAfHmZjq1pGUlZGXe5sqGJPVIU3lblbptjxggc3w2lB0tCcC-a15tNn23dGVqDzGQ8WEk1-EtRy06k2sInvg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/2/10.1093/dnares/dst047/2/dst047-dst047supp1.pdf?Expires=2147483647&amp;Signature=HHyex5AFzlMr78-MWpC5ArwQkG5gT28AZYGsJ26RnsRW1C9efDSwCh6c7VjkycDP1qdQUJs6IG1hIy0o3aCULTMFNGx22pacpikW8uslYH5S8WO0oEl-44ez6mNJ58S6egjkkpqfMrMYAw5rMo5MyW4MlX6EfP66IbUrjx331YMc6ku83oKzG5mnKVkNv5--yHLxRM7DoP6TDZhDjrgUDGXv7KFEuxEekXKzQjNF4mYI4Eq~YyJCC~wma8yrH555x54TR~aDfFObjGmo4XbocN-ERSXum52W2pw~xTf5MZHGERnhOFhc05huuSYAaT-axZXe4WADN1~PVx2ouQwPew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">582</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">45</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst047"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="403337"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897500005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f2%2f153%2f403337">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897500005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (45)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F2%2F153%2F403337" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/403337'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Population Structure and Domestication Revealed by High-Depth Resequencing of Korean Cultivated and Wild Soybean Genomes†" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst047");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f2%2f153%2f403337">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

