

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/2/73/376500","@type":"ScholarlyArticle","name":"Enhanced Recombinant Protein Productivity by Genome Reduction in  Bacillus subtilis ","datePublished":"2008-03-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2008-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn002","keywords":["<italic>Bacillus subtilis</italic>","genome size reduction","recombinant protein productivity"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/2/73/376500","author":[{"name":"Morimoto, Takuya","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan    2  Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Kadoya, Ryosuke","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan    2  Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Endo, Keiji","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Tohata, Masatoshi","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Sawada, Kazuhisa","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Liu, Shengao","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Ozawa, Tadahiro","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Kodama, Takeko","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan    3  Department of Bioscience and Textile Technology, Interdisciplinary Graduate School of Science and Technology ,  Shinshu University ,  3-15-1 Tokida, Ueda, Nagano 386-8567 ,  Japan","@type":"Person"},{"name":"Kakeshita, Hiroshi","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan    4  Graduate School of Life and Environmental Sciences ,  Tsukuba University ,  1-1-1 Ten-noudai, Tsukuba, Ibaraki 305-8572 ,  Japan","@type":"Person"},{"name":"Kageyama, Yasushi","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Manabe, Kenji","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Kanaya, Shigehiko","affiliation":"2  Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"},{"name":"Ara, Katsutoshi","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Ozaki, Katsuya","affiliation":"1  Biological Science Laboratories ,  Kao Corporation ,  2606 Akabane, Ichikai, Haga, Tochigi 321-3497 ,  Japan","@type":"Person"},{"name":"Ogasawara, Naotake","affiliation":"2  Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0101 ,  Japan","@type":"Person"}],"description":"Abstract.  The emerging field of synthetic genomics is expected to facilitate the generation of microorganisms with the potential to achieve a sustainable socie","pageStart":"73","pageEnd":"81","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Enhanced Recombinant Protein Productivity by Genome Reduction in  Bacillus subtilis ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Enhanced Recombinant Protein Productivity by Genome Reduction in  Bacillus subtilis " />
    <meta property="og:description" content="Abstract.  The emerging field of synthetic genomics is expected to facilitate the generation of microorganisms with the potential to achieve a sustainable socie" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/2/73/376500" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-03-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Morimoto, Takuya" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Kadoya, Ryosuke" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Endo, Keiji" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Tohata, Masatoshi" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Sawada, Kazuhisa" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Liu, Shengao" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Ozawa, Tadahiro" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Kodama, Takeko" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author_institution" content="Department of Bioscience and Textile Technology, Interdisciplinary Graduate School of Science and Technology, Shinshu University, 3-15-1 Tokida, Ueda, Nagano 386-8567, Japan" /><meta name="citation_author" content="Kakeshita, Hiroshi" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author_institution" content="Graduate School of Life and Environmental Sciences, Tsukuba University, 1-1-1 Ten-noudai, Tsukuba, Ibaraki 305-8572, Japan" /><meta name="citation_author" content="Kageyama, Yasushi" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Manabe, Kenji" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Kanaya, Shigehiko" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_author" content="Ara, Katsutoshi" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Ozaki, Katsuya" /><meta name="citation_author_institution" content="Biological Science Laboratories, Kao Corporation, 2606 Akabane, Ichikai, Haga, Tochigi 321-3497, Japan" /><meta name="citation_author" content="Ogasawara, Naotake" /><meta name="citation_author_institution" content="Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan" /><meta name="citation_title" content="Enhanced Recombinant Protein Productivity by Genome Reduction in Bacillus subtilis" /><meta name="citation_firstpage" content="73" /><meta name="citation_lastpage" content="81" /><meta name="citation_doi" content="10.1093/dnares/dsn002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2008/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Synthetic biology: designs for life; Nature;  citation_year=2007;  citation_volume=448;  citation_pages=32-33; " /><meta name="citation_reference" content="citation_title=Designing biological systems; Genes Develop.;  citation_year=2007;  citation_volume=21;  citation_pages=242-254; " /><meta name="citation_reference" content="citation_title=Synthetic biology projects in vitro; Genome Res;  citation_year=2007;  citation_volume=17;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Emergent properties of reduced-genome Escherichia coli; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1044-1046; " /><meta name="citation_reference" content="citation_title=Escherichia coli minimum genome factory; Biotechnol. Appl. Biochem.;  citation_year=2007;  citation_volume=46;  citation_pages=157-167; " /><meta name="citation_reference" content="citation_title=Bacillus minimum genome factory: effective utilization of microbial genome information; Biotechnol. Appl. Biochem.;  citation_year=2007;  citation_volume=46;  citation_pages=169-178; " /><meta name="citation_reference" content="citation_title=Genome engineering reveals large dispensable regions in Bacillus subtilis; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=2076-2090; " /><meta name="citation_reference" content="citation_title=Bacillus subtilis as cell factory for pharmaceutical proteins: a biotechnological approach to optimize the host organism; Biochim. Biophys. Acta;  citation_year=2004;  citation_volume=1694;  citation_pages=299-310; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of the gram-positive bacterium Bacillus subtilis; Nature;  citation_year=1997;  citation_volume=390;  citation_pages=249-256; " /><meta name="citation_reference" content="citation_title=Essential Bacillus subtilis genes; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=4678-4683; " /><meta name="citation_reference" content="citation_title=Requirements for transformation in Bacillus Subtilis; J. Bacteriol.;  citation_year=1961;  citation_volume=81;  citation_pages=741-746; " /><meta name="citation_reference" content="citation_title=Growth, maintenance and general techniques, In; citation_publisher=John Wiley &amp; Sons, Chichester, New York, Brisbane, Toronto, Singapore; Molecular Biological Methods for Bacillus;  citation_year=1990;  citation_pages=549" /><meta name="citation_reference" content="citation_title=Deduced amino acid sequence and possible catalytic residues of a thermostable, alkaline cellulase from an Alkaliphilic bacillus strain; Biosci. Biotech. Biochem.;  citation_year=2000;  citation_volume=64;  citation_pages=2281-2289; " /><meta name="citation_reference" content="citation_title=Purification and properties of an alkaline protease from alkalophilic Bacillus sp. KSM-K16; Appl. Microbiol. Biotechnol.;  citation_year=1995;  citation_volume=43;  citation_pages=473-481; " /><meta name="citation_reference" content="citation_title=Distribution of stable DnaA binding sites on the Bacillus subtilis genome detected using a modified ChIP-chip method; DNA Res;  citation_year=2007;  citation_volume=14;  citation_pages=155-168; " /><meta name="citation_reference" content="citation_title=Regulation of a promoter that is utilized by minor forms of RNA polymerase holoenzyme in Bacillus subtilis; J. Mol. Biol.;  citation_year=1986;  citation_volume=191;  citation_pages=615-624; " /><meta name="citation_reference" content="citation_title=Microarray data normalization and transformation; Nature Genet;  citation_year=2002;  citation_volume=32;  citation_pages=496-501; " /><meta name="citation_reference" content="citation_title=Elucidation of gene-to-gene and metabolite-to-gene networks in Arabidopsis by integration of metabolomics and transcriptomics; J. Biol. Chem.;  citation_year=2005;  citation_volume=280;  citation_pages=25590-25595; " /><meta name="citation_reference" content="citation_title=Construction of a novel tetracycline resistance gene cassette useful as a marker on the Bacillus subtilis chromosome; Biosci. Biotech. Biochem.;  citation_year=1992;  citation_volume=56;  citation_pages=685-686; " /><meta name="citation_reference" content="citation_title=A new mutation delivery system for genome-scale approaches in Bacillus subtilis; Mol. Microbiol.;  citation_year=2002;  citation_volume=46;  citation_pages=25-36; " /><meta name="citation_reference" content="citation_title=Analysis and preparation of sugars, In:; citation_publisher=Academic Press, New York, San Francisco, London; Methods in Carbohydrate Chemistry;  citation_year=1962;  citation_pages=388-389; " /><meta name="citation_reference" content="citation_title=Sporulation of Bacillus subtilis; Curr. Opin. Microbiol.;  citation_year=2004;  citation_volume=7;  citation_pages=579-586; " /><meta name="citation_reference" content="citation_title=DNA microarray analysis of Bacillus subtilis DegU, ComA and PhoP regulons: an approach to comprehensive analysis of B. subtilis two-component regulatory systems; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=3804-3813; " /><meta name="citation_reference" content="citation_title=The pleiotropic response regulator DegU functions as a priming protein in competence development in Bacillus subtilis; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=9246-9251; " /><meta name="citation_reference" content="citation_title=Effects of the sinR and degU32 (Hy) mutations on the regulation of the aprE gene in Bacillus subtilis; Mol. Gen. Genet.;  citation_year=1997;  citation_volume=253;  citation_pages=562-567; " /><meta name="citation_reference" content="citation_title=Environmental regulation of Bacillus subtilis sigma(D)-dependent gene expression; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=3055-3062; " /><meta name="citation_reference" content="citation_title=Systematic analysis of SigD-regulated genes in Bacillus subtilis by DNA microarray and northern blotting analyses; Gene;  citation_year=2004;  citation_volume=329;  citation_pages=125-136; " /><meta name="citation_reference" content="citation_title=Phosphate starvation-inducible proteins of Bacillus subtilis: proteomics and transcriptional analysis; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=4478-4490; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/2/73/1070084/dsn002.pdf" /><meta name="description" content="Abstract.  The emerging field of synthetic genomics is expected to facilitate the generation of microorganisms with the potential to achieve a sustainable socie" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/2/73/376500" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/2/73/376500" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/2/73/376500");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn002';
    </script>
<title>
	Enhanced Recombinant Protein Productivity by Genome Reduction in Bacillus subtilis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./376500" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbxaPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpbW90bywgVGFrdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9naWNhbCBTY2llbmNlIExhYm9yYXRvcmllcywgS2FvIENvcnBvcmF0aW9uLCAyNjA2IEFrYWJhbmUsIEljaGlrYWksIEhhZ2EsIFRvY2hpZ2kgMzIxLTM0OTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2Fkb3lhLCBSeW9zdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9naWNhbCBTY2llbmNlIExhYm9yYXRvcmllcywgS2FvIENvcnBvcmF0aW9uLCAyNjA2IEFrYWJhbmUsIEljaGlrYWksIEhhZ2EsIFRvY2hpZ2kgMzIxLTM0OTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRW5kbywgS2VpamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFNjaWVuY2UgTGFib3JhdG9yaWVzLCBLYW8gQ29ycG9yYXRpb24sIDI2MDYgQWthYmFuZSwgSWNoaWthaSwgSGFnYSwgVG9jaGlnaSAzMjEtMzQ5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb2hhdGEsIE1hc2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2xvZ2ljYWwgU2NpZW5jZSBMYWJvcmF0b3JpZXMsIEthbyBDb3Jwb3JhdGlvbiwgMjYwNiBBa2FiYW5lLCBJY2hpa2FpLCBIYWdhLCBUb2NoaWdpIDMyMS0zNDk3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhd2FkYSwgS2F6dWhpc2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFNjaWVuY2UgTGFib3JhdG9yaWVzLCBLYW8gQ29ycG9yYXRpb24sIDI2MDYgQWthYmFuZSwgSWNoaWthaSwgSGFnYSwgVG9jaGlnaSAzMjEtMzQ5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaXUsIFNoZW5nYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFNjaWVuY2UgTGFib3JhdG9yaWVzLCBLYW8gQ29ycG9yYXRpb24sIDI2MDYgQWthYmFuZSwgSWNoaWthaSwgSGFnYSwgVG9jaGlnaSAzMjEtMzQ5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPemF3YSwgVGFkYWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFNjaWVuY2UgTGFib3JhdG9yaWVzLCBLYW8gQ29ycG9yYXRpb24sIDI2MDYgQWthYmFuZSwgSWNoaWthaSwgSGFnYSwgVG9jaGlnaSAzMjEtMzQ5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2RhbWEsIFRha2VrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2xvZ2ljYWwgU2NpZW5jZSBMYWJvcmF0b3JpZXMsIEthbyBDb3Jwb3JhdGlvbiwgMjYwNiBBa2FiYW5lLCBJY2hpa2FpLCBIYWdhLCBUb2NoaWdpIDMyMS0zNDk3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZSBhbmQgVGV4dGlsZSBUZWNobm9sb2d5LCBJbnRlcmRpc2NpcGxpbmFyeSBHcmFkdWF0ZSBTY2hvb2wgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgU2hpbnNodSBVbml2ZXJzaXR5LCAzLTE1LTEgVG9raWRhLCBVZWRhLCBOYWdhbm8gMzg2LTg1NjcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FrZXNoaXRhLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9naWNhbCBTY2llbmNlIExhYm9yYXRvcmllcywgS2FvIENvcnBvcmF0aW9uLCAyNjA2IEFrYWJhbmUsIEljaGlrYWksIEhhZ2EsIFRvY2hpZ2kgMzIxLTM0OTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgYW5kIEVudmlyb25tZW50YWwgU2NpZW5jZXMsIFRzdWt1YmEgVW5pdmVyc2l0eSwgMS0xLTEgVGVuLW5vdWRhaSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODU3MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYWdleWFtYSwgWWFzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2xvZ2ljYWwgU2NpZW5jZSBMYWJvcmF0b3JpZXMsIEthbyBDb3Jwb3JhdGlvbiwgMjYwNiBBa2FiYW5lLCBJY2hpa2FpLCBIYWdhLCBUb2NoaWdpIDMyMS0zNDk3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hbmFiZSwgS2VuamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9sb2dpY2FsIFNjaWVuY2UgTGFib3JhdG9yaWVzLCBLYW8gQ29ycG9yYXRpb24sIDI2MDYgQWthYmFuZSwgSWNoaWthaSwgSGFnYSwgVG9jaGlnaSAzMjEtMzQ5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5heWEsIFNoaWdlaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyYSwgS2F0c3V0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2xvZ2ljYWwgU2NpZW5jZSBMYWJvcmF0b3JpZXMsIEthbyBDb3Jwb3JhdGlvbiwgMjYwNiBBa2FiYW5lLCBJY2hpa2FpLCBIYWdhLCBUb2NoaWdpIDMyMS0zNDk3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik96YWtpLCBLYXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvbG9naWNhbCBTY2llbmNlIExhYm9yYXRvcmllcywgS2FvIENvcnBvcmF0aW9uLCAyNjA2IEFrYWJhbmUsIEljaGlrYWksIEhhZ2EsIFRvY2hpZ2kgMzIxLTM0OTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2dhc2F3YXJhLCBOYW90YWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFbmhhbmNlZCBSZWNvbWJpbmFudCBQcm90ZWluIFByb2R1Y3Rpdml0eSBieSBHZW5vbWUgUmVkdWN0aW9uIGluIEJhY2lsbHVzIHN1YnRpbGlzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW50aGV0aWMgYmlvbG9neTogZGVzaWducyBmb3IgbGlmZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0ODsgIGNpdGF0aW9uX3BhZ2VzPTMyLTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXNpZ25pbmcgYmlvbG9naWNhbCBzeXN0ZW1zOyBHZW5lcyBEZXZlbG9wLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI0Mi0yNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXRpYyBiaW9sb2d5IHByb2plY3RzIGluIHZpdHJvOyBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbWVyZ2VudCBwcm9wZXJ0aWVzIG9mIHJlZHVjZWQtZ2Vub21lIEVzY2hlcmljaGlhIGNvbGk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTA0NC0xMDQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIG1pbmltdW0gZ2Vub21lIGZhY3Rvcnk7IEJpb3RlY2hub2wuIEFwcGwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTU3LTE2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjaWxsdXMgbWluaW11bSBnZW5vbWUgZmFjdG9yeTogZWZmZWN0aXZlIHV0aWxpemF0aW9uIG9mIG1pY3JvYmlhbCBnZW5vbWUgaW5mb3JtYXRpb247IEJpb3RlY2hub2wuIEFwcGwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTY5LTE3ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGVuZ2luZWVyaW5nIHJldmVhbHMgbGFyZ2UgZGlzcGVuc2FibGUgcmVnaW9ucyBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTIwNzYtMjA5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjaWxsdXMgc3VidGlsaXMgYXMgY2VsbCBmYWN0b3J5IGZvciBwaGFybWFjZXV0aWNhbCBwcm90ZWluczogYSBiaW90ZWNobm9sb2dpY2FsIGFwcHJvYWNoIHRvIG9wdGltaXplIHRoZSBob3N0IG9yZ2FuaXNtOyBCaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2OTQ7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBncmFtLXBvc2l0aXZlIGJhY3Rlcml1bSBCYWNpbGx1cyBzdWJ0aWxpczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM5MDsgIGNpdGF0aW9uX3BhZ2VzPTI0OS0yNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzc2VudGlhbCBCYWNpbGx1cyBzdWJ0aWxpcyBnZW5lczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9NDY3OC00NjgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXF1aXJlbWVudHMgZm9yIHRyYW5zZm9ybWF0aW9uIGluIEJhY2lsbHVzIFN1YnRpbGlzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTYxOyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9NzQxLTc0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3Jvd3RoLCBtYWludGVuYW5jZSBhbmQgZ2VuZXJhbCB0ZWNobmlxdWVzLCBJbjsgY2l0YXRpb25fcHVibGlzaGVyPUpvaG4gV2lsZXkgJmFtcDsgU29ucywgQ2hpY2hlc3RlciwgTmV3IFlvcmssIEJyaXNiYW5lLCBUb3JvbnRvLCBTaW5nYXBvcmU7IE1vbGVjdWxhciBCaW9sb2dpY2FsIE1ldGhvZHMgZm9yIEJhY2lsbHVzOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fcGFnZXM9NTQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVkdWNlZCBhbWlubyBhY2lkIHNlcXVlbmNlIGFuZCBwb3NzaWJsZSBjYXRhbHl0aWMgcmVzaWR1ZXMgb2YgYSB0aGVybW9zdGFibGUsIGFsa2FsaW5lIGNlbGx1bGFzZSBmcm9tIGFuIEFsa2FsaXBoaWxpYyBiYWNpbGx1cyBzdHJhaW47IEJpb3NjaS4gQmlvdGVjaC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz0yMjgxLTIyODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1cmlmaWNhdGlvbiBhbmQgcHJvcGVydGllcyBvZiBhbiBhbGthbGluZSBwcm90ZWFzZSBmcm9tIGFsa2Fsb3BoaWxpYyBCYWNpbGx1cyBzcC4gS1NNLUsxNjsgQXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTQ3My00ODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RyaWJ1dGlvbiBvZiBzdGFibGUgRG5hQSBiaW5kaW5nIHNpdGVzIG9uIHRoZSBCYWNpbGx1cyBzdWJ0aWxpcyBnZW5vbWUgZGV0ZWN0ZWQgdXNpbmcgYSBtb2RpZmllZCBDaElQLWNoaXAgbWV0aG9kOyBETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTU1LTE2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBhIHByb21vdGVyIHRoYXQgaXMgdXRpbGl6ZWQgYnkgbWlub3IgZm9ybXMgb2YgUk5BIHBvbHltZXJhc2UgaG9sb2VuenltZSBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTE7ICBjaXRhdGlvbl9wYWdlcz02MTUtNjI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2FycmF5IGRhdGEgbm9ybWFsaXphdGlvbiBhbmQgdHJhbnNmb3JtYXRpb247IE5hdHVyZSBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ni01MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsdWNpZGF0aW9uIG9mIGdlbmUtdG8tZ2VuZSBhbmQgbWV0YWJvbGl0ZS10by1nZW5lIG5ldHdvcmtzIGluIEFyYWJpZG9wc2lzIGJ5IGludGVncmF0aW9uIG9mIG1ldGFib2xvbWljcyBhbmQgdHJhbnNjcmlwdG9taWNzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yODA7ICBjaXRhdGlvbl9wYWdlcz0yNTU5MC0yNTU5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgbm92ZWwgdGV0cmFjeWNsaW5lIHJlc2lzdGFuY2UgZ2VuZSBjYXNzZXR0ZSB1c2VmdWwgYXMgYSBtYXJrZXIgb24gdGhlIEJhY2lsbHVzIHN1YnRpbGlzIGNocm9tb3NvbWU7IEJpb3NjaS4gQmlvdGVjaC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz02ODUtNjg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBtdXRhdGlvbiBkZWxpdmVyeSBzeXN0ZW0gZm9yIGdlbm9tZS1zY2FsZSBhcHByb2FjaGVzIGluIEJhY2lsbHVzIHN1YnRpbGlzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0yNS0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgYW5kIHByZXBhcmF0aW9uIG9mIHN1Z2FycywgSW46OyBjaXRhdGlvbl9wdWJsaXNoZXI9QWNhZGVtaWMgUHJlc3MsIE5ldyBZb3JrLCBTYW4gRnJhbmNpc2NvLCBMb25kb247IE1ldGhvZHMgaW4gQ2FyYm9oeWRyYXRlIENoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk2MjsgIGNpdGF0aW9uX3BhZ2VzPTM4OC0zODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwb3J1bGF0aW9uIG9mIEJhY2lsbHVzIHN1YnRpbGlzOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01NzktNTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWljcm9hcnJheSBhbmFseXNpcyBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBEZWdVLCBDb21BIGFuZCBQaG9QIHJlZ3Vsb25zOiBhbiBhcHByb2FjaCB0byBjb21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIEIuIHN1YnRpbGlzIHR3by1jb21wb25lbnQgcmVndWxhdG9yeSBzeXN0ZW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0zODA0LTM4MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwbGVpb3Ryb3BpYyByZXNwb25zZSByZWd1bGF0b3IgRGVnVSBmdW5jdGlvbnMgYXMgYSBwcmltaW5nIHByb3RlaW4gaW4gY29tcGV0ZW5jZSBkZXZlbG9wbWVudCBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz05MjQ2LTkyNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdHMgb2YgdGhlIHNpblIgYW5kIGRlZ1UzMiAoSHkpIG11dGF0aW9ucyBvbiB0aGUgcmVndWxhdGlvbiBvZiB0aGUgYXByRSBnZW5lIGluIEJhY2lsbHVzIHN1YnRpbGlzOyBNb2wuIEdlbi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1MzsgIGNpdGF0aW9uX3BhZ2VzPTU2Mi01Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudmlyb25tZW50YWwgcmVndWxhdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyBzaWdtYShEKS1kZXBlbmRlbnQgZ2VuZSBleHByZXNzaW9uOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTMwNTUtMzA2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBhbmFseXNpcyBvZiBTaWdELXJlZ3VsYXRlZCBnZW5lcyBpbiBCYWNpbGx1cyBzdWJ0aWxpcyBieSBETkEgbWljcm9hcnJheSBhbmQgbm9ydGhlcm4gYmxvdHRpbmcgYW5hbHlzZXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI5OyAgY2l0YXRpb25fcGFnZXM9MTI1LTEzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhvc3BoYXRlIHN0YXJ2YXRpb24taW5kdWNpYmxlIHByb3RlaW5zIG9mIEJhY2lsbHVzIHN1YnRpbGlzOiBwcm90ZW9taWNzIGFuZCB0cmFuc2NyaXB0aW9uYWwgYW5hbHlzaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9NDQ3OC00NDkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvMi83My8xMDcwMDg0L2RzbjAwMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGVtZXJnaW5nIGZpZWxkIG9mIHN5bnRoZXRpYyBnZW5vbWljcyBpcyBleHBlY3RlZCB0byBmYWNpbGl0YXRlIHRoZSBnZW5lcmF0aW9uIG9mIG1pY3Jvb3JnYW5pc21zIHdpdGggdGhlIHBvdGVudGlhbCB0byBhY2hpZXZlIGEgc3VzdGFpbmFibGUgc29jaWUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM3NjUwMCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQfLNRnk5B3le02JOy4TZTtEqzikg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f73%2f376500" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f73%2f376500" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14648" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&amp;Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/2">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5484381">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484381">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484383">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484383">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484388">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484388">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484393">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484393">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484410">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484410">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484412">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484412">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484414">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484414">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484416">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484416">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484418">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484418">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/2/63/376035">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/2/83/375768">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Enhanced Recombinant Protein Productivity by Genome Reduction in <em>Bacillus subtilis</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Morimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Morimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morimoto T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morimoto T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takuya Morimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryosuke Kadoya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryosuke+Kadoya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kadoya R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kadoya R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ryosuke Kadoya</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keiji Endo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keiji+Endo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Endo K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Endo K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Keiji Endo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masatoshi Tohata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masatoshi+Tohata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tohata M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tohata M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masatoshi Tohata</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuhisa Sawada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuhisa+Sawada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sawada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sawada K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kazuhisa Sawada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shengao Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shengao+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shengao Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadahiro Ozawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadahiro+Ozawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ozawa T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ozawa T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tadahiro Ozawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeko Kodama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioscience and Textile Technology, Interdisciplinary Graduate School of Science and Technology</div>, <div class="insititution">Shinshu University</div>, <div class="addr-line">3-15-1 Tokida, Ueda, Nagano 386-8567</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeko+Kodama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kodama T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kodama T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takeko Kodama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Kakeshita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Life and Environmental Sciences</div>, <div class="insititution">Tsukuba University</div>, <div class="addr-line">1-1-1 Ten-noudai, Tsukuba, Ibaraki 305-8572</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Kakeshita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kakeshita H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kakeshita H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroshi Kakeshita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasushi Kageyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasushi+Kageyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kageyama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kageyama Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasushi Kageyama</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenji Manabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenji+Manabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Manabe K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Manabe K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kenji Manabe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiko Kanaya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiko+Kanaya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanaya S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanaya S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shigehiko Kanaya</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsutoshi Ara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsutoshi+Ara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ara K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ara K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Katsutoshi Ara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuya Ozaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Biological Science Laboratories</div>, <div class="insititution">Kao Corporation</div>, <div class="addr-line">2606 Akabane, Ichikai, Haga, Tochigi 321-3497</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuya+Ozaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ozaki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ozaki K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Katsuya Ozaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naotake Ogasawara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0101</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. + 81 743-72-5430. Fax. <div class="fax">+ 81 743-72-5439</div>. E-mail: <a href="mailto:nogasawa@bs.naist.jp" target="_blank">nogasawa@bs.naist.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naotake+Ogasawara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogasawara N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogasawara N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Naotake Ogasawara</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 73–81, <a href='https://doi.org/10.1093/dnares/dsn002'>https://doi.org/10.1093/dnares/dsn002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 March 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 December 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 January 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="376500" href="/dnaresearch/article-pdf/15/2/73/1070084/dsn002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Takuya Morimoto, Ryosuke Kadoya, Keiji Endo, Masatoshi Tohata, Kazuhisa Sawada, Shengao Liu, Tadahiro Ozawa, Takeko Kodama, Hiroshi Kakeshita, Yasushi Kageyama, Kenji Manabe, Shigehiko Kanaya, Katsutoshi Ara, Katsuya Ozaki, Naotake Ogasawara;  Enhanced Recombinant Protein Productivity by Genome Reduction in <em>Bacillus subtilis</em>, <em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 73–81, <a href='https://doi.org/10.1093/dnares/dsn002'>https://doi.org/10.1093/dnares/dsn002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/376500?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/376500?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Enhanced%20Recombinant%20Protein%20Productivity%20by%20Genome%20Reduction%20in%20%26lt%3Bem%26gt%3BBacillus%20subtilis%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2008-03-11&amp;volumeNum=15&amp;issueNum=2&amp;author=Morimoto%2C%20Takuya%3B%20Kadoya%2C%20Ryosuke&amp;startPage=73&amp;endPage=81&amp;contentId=10.1093%2Fdnares%2Fdsn002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f73%2f376500" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5484381 id="5484381" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The emerging field of synthetic genomics is expected to facilitate the generation of microorganisms with the potential to achieve a sustainable society. One approach towards this goal is the reduction of microbial genomes by rationally designed deletions to create simplified cells with predictable behavior that act as a platform to build in various genetic systems for specific purposes. We report a novel <em>Bacillus subtilis</em> strain, MBG874, depleted of 874 kb (20%) of the genomic sequence. When compared with wild-type cells, the regulatory network of gene expression of the mutant strain is reorganized after entry into the transition state due to the synergistic effect of multiple deletions, and productivity of extracellular cellulase and protease from transformed plasmids harboring the corresponding genes is remarkably enhanced. To our knowledge, this is the first report demonstrating that genome reduction actually contributes to the creation of bacterial cells with a practical application in industry. Further systematic analysis of changes in the transcriptional regulatory network of MGB874 cells in relation to protein productivity should facilitate the generation of improved <em>B. subtilis</em> cells as hosts of industrial protein production.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Bacillus subtilis&quot;"><em>Bacillus subtilis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome size reduction&quot;">genome size reduction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;recombinant protein productivity&quot;">recombinant protein productivity</a></div>                            <h2 scrollto-destination=5484383 id="5484383" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The emerging field of synthetic genomics is expected to facilitate the generation of microorganisms with the potential to achieve a sustainable society.<sup><a reveal-id="DSN002C1  DSN002C3" data-open="DSN002C1  DSN002C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> One approach towards this goal is the reduction of microbial genomes by rationally designed deletions to create simplified cells with predictable behavior that act as a platform to build in various genetic systems for specific purposes. It is expected that metabolic waste in these cells will be decreased, as fewer dispensable proteins are synthesized. Challenges to reduce the genome size by the stepwise introduction of large-scale deletions have been undertaken for <em>Escherichia coli</em><sup><a reveal-id="DSN002C4" data-open="DSN002C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSN002C5" data-open="DSN002C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and <em>Bacillus subtilis</em>.<sup><a reveal-id="DSN002C6" data-open="DSN002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSN002C7" data-open="DSN002C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>In a previous study, Posfai et al.<sup><a reveal-id="DSN002C4" data-open="DSN002C4" class="link link-ref link-reveal xref-bibr">4</a></sup> reported an <em>E. coli</em> strain, MDS42, in which ∼15% of the genome sequence (0.71 Mb) was removed by planned and sequential deletions to eliminate all mobile DNA and cryptic virulence genes. The resultant strain displayed normal cell growth and protein expression comparable with that of the parental strain, MG1655. Interestingly, genome reduction gave rise to unexpected beneficial properties. MDS42 cells exhibit high electroporation efficiency and allow the accurate propagation of plasmids that are unstable in other strains. Another <em>E. coli</em> strain, MGF-01, with a 22% genome size reduction (1.03 Mb) was created in Japan using a similar strategy.<sup><a reveal-id="DSN002C5" data-open="DSN002C5" class="link link-ref link-reveal xref-bibr">5</a></sup> MGF-01 cells also displayed an unexpected phenotype in that growth was as rapid as the parental W3110 strain in M9 minimal medium during the exponential phase, but continued after the wild-type strain had progressed to the stationary phase. In addition, MGF-01 secreted twice the amount of threonine as the wild-type strain.</p><p><em>B. subtilis</em>, one of the most extensively studied model microorganisms, displays superior ability to produce various secretary enzymes. This ability has been widely applied to produce various useful enzymes in the industrial field.<sup><a reveal-id="DSN002C8" data-open="DSN002C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The 4.2 Mb <em>B. subtilis</em> genome contains 10 horizontally acquired prophage (SPβ and PBSX) and prophage-like (pro1-7 and skin) sequences.<sup><a reveal-id="DSN002C9" data-open="DSN002C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In addition, 2.8% of the genome encompasses two large operons that produce secondary metabolites (<em>pks</em> and <em>pps</em>). Systematic inactivation of the protein-coding regions revealed that only 271 genes are indispensable for growth in rich medium (LB) at 37°C, as growth is impossible in the absence of these genes.<sup><a reveal-id="DSN002C10" data-open="DSN002C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Westers et al.<sup><a reveal-id="DSN002C7" data-open="DSN002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> reported a <em>B. subtilis</em> Δ6 mutant strain with a 7.7% reduction of the genome (0.53 Mb) produced by deleting two prophage (SPβ, PBSX), three prophage-like sequences (pro1, pro6, skin), and the <em>pks</em> operon. However, phenotypic characterization of the Δ6 cells disclosed no unique properties, including secretion of AmyQ protein, relative to wild-type 168 cells.<sup><a reveal-id="DSN002C7" data-open="DSN002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Our group generated a strain, MGB469, in which all prophage and prophage-like sequences, except pro7, as well as <em>pks</em> and <em>pps</em> operons, were deleted.<sup><a reveal-id="DSN002C6" data-open="DSN002C6" class="link link-ref link-reveal xref-bibr">6</a></sup> In this case, cell growth was normal, but no beneficial properties were apparent, including exogenous protein production from the plasmid harboring the corresponding genes. Moreover, we constructed the MG1M strain containing an additional six deletions in the MGB469 genome, resulting in a 0.99 Mb reduction in genome size.<sup><a reveal-id="DSN002C6" data-open="DSN002C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, the MG1M strain displayed unstable phenotypes with regard to growth rate, cell morphology, and recombinant protein productivity after successive culture, making it inappropriate for further studies.</p><p>Here, we report the generation of another multiple-deletion series of mutants, from MGB469 as the starting strain through to MGB874 with a total 0.87 Mb genome deletion. When compared with wild-type cells, the regulatory network of gene expression of the mutant strain is reorganized after entry into the transition state due to the synergistic effect of multiple deletions, and productivity of extracellular cellulase and protease from transformed plasmids harboring the corresponding genes is remarkably enhanced. Our results demonstrate the effectiveness of synthetic genomics in creating novel and useful bacteria for industrial use.</p>                            <h2 scrollto-destination=5484388 id="5484388" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=5484389 id="5484389" class="section-title" data-legacy-id=s2a>2.1. Culture media and measurement of enzyme activity</h3>
<p>Spizizen's minimal medium (SMM)<sup><a reveal-id="DSN002C11" data-open="DSN002C11" class="link link-ref link-reveal xref-bibr">11</a></sup> comprising 0.2% (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>, 1.4% K<sub>2</sub>HPO<sub>4</sub>, 0.6% KH<sub>2</sub>PO<sub>4</sub>, 0.1% Na<sub>3</sub>C<sub>6</sub>H<sub>5</sub>O<sub>7</sub>·2H<sub>2</sub>O, and 0.02% MgSO<sub>4</sub>·7H<sub>2</sub>O was supplemented with 0.5% glucose and trace elements.<sup><a reveal-id="DSN002C12" data-open="DSN002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> LB contained 1% tryptone peptone (Difco), 0.5% yeast extract (Difco), and 0.5% NaCl. The 2xL-Mal medium contained 2% tryptone peptone (Difco), 1% yeast extract (Difco), 1% NaCl, 7.5% maltose hydrate, and 7.5 μg/mL MnSO<sub>4</sub>. Protease and cellulase activities in culture medium were determined as described previously.<sup><a reveal-id="DSN002C13" data-open="DSN002C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSN002C14" data-open="DSN002C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                            <h3 scrollto-destination=5484391 id="5484391" class="section-title" data-legacy-id=s2b>2.2 High-resolution transcriptome analysis</h3>
<p>The custom Affymetrix tiling chip used contains 55 430 25-mer probes for the coding strand of protein coding regions at 25–30 bp intervals and 72 218 probes for both strands of intergenic regions at 2–3 bp intervals.<sup><a reveal-id="DSN002C15" data-open="DSN002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Cultures (200 mL) of wild-type 168 and MGB874 cells (OD<sub>600</sub> = 0.6) in 2xL-Mal at 30°C were inoculated into 15 L of 2xL-Mal medium within a 30 L jar fermenter. Total RNA was extracted from <em>B. subtilis</em> cells (10 OD<sub>600</sub>), as described previously.<sup><a reveal-id="DSN002C16" data-open="DSN002C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Synthesis of cDNA, terminal labeling, and oligonucleotide chip hybridization were performed following the Affymetrix instruction manual. Briefly, cDNA was synthesized from 10 μg total RNA using random primers and reverse transcriptase (Superscript III, Invitrogen), followed by purification using QiaQuick columns (Qiagen) and digestion with DNase I (GE Healthcare Bioscience). Next, cDNA fragments were terminally labeled with biotin-ddUTP using a ENZO BioArray Terminal Labeling Kit (Enzo Life Sciences). Hybridization with the oligonucleotide chip was performed for 16 h at 42°C, followed by washing, staining, and scanning using the GeneChip Instrument System, according to the manufacturer's instructions (Affymetrix). Transcriptional signals were analyzed and visualized along the genome coordinate with the <em>In Silico</em> Molecular Cloning program, Array Edition (<em>In Silico</em> Biology). To compensate for the differences in hybridization efficiency of each 25-mer probe on the chip, we divided the hybridization intensities of cDNA synthesized from total RNA by those of total genome DNA. The signal intensities of each experiment were adjusted to confer a signal average of 500 and normalized by MA plot analysis for comparison of MGB874 and wild-type 168 data.<sup><a reveal-id="DSN002C17" data-open="DSN002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSN002C18" data-open="DSN002C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The average signal intensities of probes in each coding sequence were calculated after removal of the lowest and highest intensities.</p>                            <h2 scrollto-destination=5484393 id="5484393" class="section-title" data-legacy-id=s3>3. Results and Discussion</h2>
                            <h3 scrollto-destination=5484394 id="5484394" class="section-title" data-legacy-id=s3a>3.1. Multiple-deletion series mutants design</h3>
<p>To construct the multiple-deletion series mutants, we initially identified contiguous genome sequences &gt;10 kb that did not code for RNA or essential proteins. In addition, we excluded all known and possible genes involved in primary metabolism to maintain growth in minimum medium, as well as those related to DNA metabolism to avoid genome instability. In total, 74 regions, including prophage, prophage-like, and secondary metabolite-producing sequences, were selected and individually replaced with the tetracycline-resistant gene (<em>tet</em>) by selection on LB plates. As a result, we obtained deletion mutants for 63 regions, totaling up to ∼2 Mb (Fig. <a reveal-id="DSN002F1" data-open="DSN002F1" class="link link-reveal link-table xref-fig">1</a>A and Supplementary Table S1). However, a number of the deletion mutants did not grow in SMM, and others showed impaired growth, even in liquid LB medium (Supplementary Table S1).</p>                        <a id="5484396" scrollto-destination="5484396"></a>
<div id="DSN002F1" data-id="DSN002F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00201.jpeg?Expires=2147483647&Signature=gsMliY8t8SQ-gJgay4wiYFvKL40RaZ1WwRuWcUbF-P1wYAqZvFjnf2V0Plzwe6mM1BSV~-0cL7SheEfZcQITdOrq4RClWmbxsPbjwVK-4o0u2x5S65ahzawG8YLF8imMfcXOqq0Tg6M0z5IH~tLIVG0n12ODmdjDVexIo7l8Q0CdJnd6ENotGSBOf3MV~kiV7nBg5XzzOgNxmMoSoJ6rs~x0P7U9YuyzZ~GaO3AwRoYVoZ0JxiavrtThnZ9uJB8BElKS8pZiRXhGcI1iC3~ZJSuCtra0RK4AEK5U1aB5VAdR~0vVTupQfyPnERE6DN43fbUmt8YOkg8~VMdszegEtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Design and basal phenotypic analysis of MGB874. (A) Outer concentric ring: genome coordinate (bases) of the B. subtilis 168 genome. Ring 2 (green): positions of deleted sequences in MGB874, including prophages and prophage-like regions (SPβ, PBSX, skin, pro1-7) and polyketide and plipastatin synthesis operons (pks, pps). Ring 3 (dark blue): regions of single deletion. Rings 4 and 5 (light blue): protein coding regions in clockwise (Ring 4) and counterclockwise (Ring 5) orientations. Ring 6 (red): rRNA and tRNA genes. (B) Growth profiles of MGB874 (red squares) and wild-type 168 (blue diamonds) cells in LB and SMM medium. The doubling time of growth is specified. (C) Cell morphology, chromosome distribution, and mean values of cell lengths of wild-type 168 and MGB874 cells. MGB874 and 168 cells were cultured at 37°C in LB or SMM medium, and images were obtained during the exponential growth phase after staining with 4,6-diamindino-2-phenylindole (DAPI). The average cell length is indicated (∼200 cells analyzed)." path-from-xml="dsn00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484396/dsn00201.jpeg" path-from-xml="dsn00201.jpeg" target="_blank">View large</a><a section="5484396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00201.jpeg?Expires=2147483647&Signature=qkI7BZzoxMIvAoUan02cpmV0VaTz2BXcqpVEu-g00snNUssh28Xn8CcG~hpk09XHPIA3kTYfI3Y0gXHIT6vVFST-DvSxuUDajlWih-J7eEk-LRITeBKur31jh7XcDgO5OHUgaTUUyiCSvo-~3OLGRqSTFlbgiFMNDg4vmb07MLaMujpHo7t3YI1E~ALXyvWvRqavsEAEji9gFMsMEB7pCXHlJH1pyhGqR3wE4KYHESyVJKYH~BVXlbognhKXg3ywEahsxFqEXsCeBcn7K6tsUhmTTLCEY-yDNO9KAQzU5g9A6fD4BwLA57E~9DLX4Sz9Z2GymwUZa2uGyGIJIe-pcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484396&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Design and basal phenotypic analysis of MGB874. (<strong>A</strong>) Outer concentric ring: genome coordinate (bases) of the <em>B. subtilis</em> 168 genome. Ring 2 (green): positions of deleted sequences in MGB874, including prophages and prophage-like regions (SPβ, PBSX, skin, pro1-7) and polyketide and plipastatin synthesis operons (<em>pks</em>, <em>pps</em>). Ring 3 (dark blue): regions of single deletion. Rings 4 and 5 (light blue): protein coding regions in clockwise (Ring 4) and counterclockwise (Ring 5) orientations. Ring 6 (red): rRNA and tRNA genes. (<strong>B</strong>) Growth profiles of MGB874 (red squares) and wild-type 168 (blue diamonds) cells in LB and SMM medium. The doubling time of growth is specified. (<strong>C</strong>) Cell morphology, chromosome distribution, and mean values of cell lengths of wild-type 168 and MGB874 cells. MGB874 and 168 cells were cultured at 37°C in LB or SMM medium, and images were obtained during the exponential growth phase after staining with 4,6-diamindino-2-phenylindole (DAPI). The average cell length is indicated (∼200 cells analyzed).</p></div></div><div content-id="DSN002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00201.jpeg?Expires=2147483647&Signature=gsMliY8t8SQ-gJgay4wiYFvKL40RaZ1WwRuWcUbF-P1wYAqZvFjnf2V0Plzwe6mM1BSV~-0cL7SheEfZcQITdOrq4RClWmbxsPbjwVK-4o0u2x5S65ahzawG8YLF8imMfcXOqq0Tg6M0z5IH~tLIVG0n12ODmdjDVexIo7l8Q0CdJnd6ENotGSBOf3MV~kiV7nBg5XzzOgNxmMoSoJ6rs~x0P7U9YuyzZ~GaO3AwRoYVoZ0JxiavrtThnZ9uJB8BElKS8pZiRXhGcI1iC3~ZJSuCtra0RK4AEK5U1aB5VAdR~0vVTupQfyPnERE6DN43fbUmt8YOkg8~VMdszegEtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Design and basal phenotypic analysis of MGB874. (A) Outer concentric ring: genome coordinate (bases) of the B. subtilis 168 genome. Ring 2 (green): positions of deleted sequences in MGB874, including prophages and prophage-like regions (SPβ, PBSX, skin, pro1-7) and polyketide and plipastatin synthesis operons (pks, pps). Ring 3 (dark blue): regions of single deletion. Rings 4 and 5 (light blue): protein coding regions in clockwise (Ring 4) and counterclockwise (Ring 5) orientations. Ring 6 (red): rRNA and tRNA genes. (B) Growth profiles of MGB874 (red squares) and wild-type 168 (blue diamonds) cells in LB and SMM medium. The doubling time of growth is specified. (C) Cell morphology, chromosome distribution, and mean values of cell lengths of wild-type 168 and MGB874 cells. MGB874 and 168 cells were cultured at 37°C in LB or SMM medium, and images were obtained during the exponential growth phase after staining with 4,6-diamindino-2-phenylindole (DAPI). The average cell length is indicated (∼200 cells analyzed)." path-from-xml="dsn00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484396/dsn00201.jpeg" path-from-xml="dsn00201.jpeg" target="_blank">View large</a><a section="5484396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00201.jpeg?Expires=2147483647&Signature=qkI7BZzoxMIvAoUan02cpmV0VaTz2BXcqpVEu-g00snNUssh28Xn8CcG~hpk09XHPIA3kTYfI3Y0gXHIT6vVFST-DvSxuUDajlWih-J7eEk-LRITeBKur31jh7XcDgO5OHUgaTUUyiCSvo-~3OLGRqSTFlbgiFMNDg4vmb07MLaMujpHo7t3YI1E~ALXyvWvRqavsEAEji9gFMsMEB7pCXHlJH1pyhGqR3wE4KYHESyVJKYH~BVXlbognhKXg3ywEahsxFqEXsCeBcn7K6tsUhmTTLCEY-yDNO9KAQzU5g9A6fD4BwLA57E~9DLX4Sz9Z2GymwUZa2uGyGIJIe-pcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484396&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Design and basal phenotypic analysis of MGB874. (<strong>A</strong>) Outer concentric ring: genome coordinate (bases) of the <em>B. subtilis</em> 168 genome. Ring 2 (green): positions of deleted sequences in MGB874, including prophages and prophage-like regions (SPβ, PBSX, skin, pro1-7) and polyketide and plipastatin synthesis operons (<em>pks</em>, <em>pps</em>). Ring 3 (dark blue): regions of single deletion. Rings 4 and 5 (light blue): protein coding regions in clockwise (Ring 4) and counterclockwise (Ring 5) orientations. Ring 6 (red): rRNA and tRNA genes. (<strong>B</strong>) Growth profiles of MGB874 (red squares) and wild-type 168 (blue diamonds) cells in LB and SMM medium. The doubling time of growth is specified. (<strong>C</strong>) Cell morphology, chromosome distribution, and mean values of cell lengths of wild-type 168 and MGB874 cells. MGB874 and 168 cells were cultured at 37°C in LB or SMM medium, and images were obtained during the exponential growth phase after staining with 4,6-diamindino-2-phenylindole (DAPI). The average cell length is indicated (∼200 cells analyzed).</p></div></div>                            <h3 scrollto-destination=5484397 id="5484397" class="section-title" data-legacy-id=s3b>3.2. Step-by-step B. subtilis genome reduction</h3>
<p>Next, we attempted to construct a reduced genome <em>B. subtilis</em> strain by step-by-step deletions in regions where single deletions did not affect cell growth, applying the <em>upp</em> (encoding uracil-phosphoribosyl-transferase) cassette and 5-fluorouracil (5-FU ) selection<sup><a reveal-id="DSN002C20" data-open="DSN002C20" class="link link-ref link-reveal xref-bibr">20</a></sup> to remove the drug-resistant markers used to introduce primary deletions (Fig. <a reveal-id="DSN002F2" data-open="DSN002F2" class="link link-reveal link-table xref-fig">2</a>). Accordingly, sequential deletions in 11 non-essential gene clusters were introduced in the MGB469 strain to generate MGB874 depleted of 874 kb (20.7%) of the sequence spanning 865 genes from the original <em>B. subtilis</em> 168 genome (Fig. <a reveal-id="DSN002F1" data-open="DSN002F1" class="link link-reveal link-table xref-fig">1</a>A and Supplementary Table S2). Although the growth rate of MGB874 cells was reduced (30% in LB and 50% in SMM) compared with the wild-type 168 strain (Fig. <a reveal-id="DSN002F1" data-open="DSN002F1" class="link link-reveal link-table xref-fig">1</a>B), cell morphology and chromosome distribution were normal (Fig. <a reveal-id="DSN002F1" data-open="DSN002F1" class="link link-reveal link-table xref-fig">1</a>C). The reduced growth rate indicates that certain non-annotated and deleted genes contribute to the metabolic capacity of <em>B. subtilis</em> cells under normal growth conditions. It is also possible that this phenotype is caused by unexpected synergetic effects of the deletions of annotated genes. Moreover, MGB874 cells did not form spores (data not shown), owing to the deletion of genes essential for spore formation, including <em>spoIVCB</em> and <em>spoIIIC</em> that encode the N- and C- terminal regions of sporulation-specific sigma factor-K, respectively.</p>                        <a id="5484399" scrollto-destination="5484399"></a>
<div id="DSN002F2" data-id="DSN002F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00202.jpeg?Expires=2147483647&Signature=DUaDM6KYbJvWCWH8REAF2FZsI6Xv-zHGK15J5GZrZ8NtH0sjT-N9ismfOso53u3eFZjjaXQ9WSm6vjNj4NqzVzBzoz-yBpAmk5GCazLJNli2VZ8HA0yRwiW9n9b1F7UFUk9TAwWdrUF9ZOZzV-LNvgERO8oWFc1f4hA70-efkzyIJq9guwpB~Y-rcb66IxrMz29pS4N66oFc5Tjl0chN9aj7Ak5BZeLwqh8xNJftmnVoHrY0bdpO02JB8Nye0zszQnXCNCVxHo-ga-q-RbhbwBFYCoHnlhrD21fpVqiM4ScdOv23Ot0hsiExNhwgF-hoYW8feoWz4Uc-ZgdlDiEOHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequential introduction of large-scale deletions into the B. subtilis genome. A derivative of B. subtilis 168, 168 Δupp, in which the upp gene encoding uracil-phosphoribosyl-transferase is inactivated by replacement with the erythromycin-resistance gene (erm), was used as the starting strain for generation of the deletion mutant series. The entire length of the tetracycline-resistant gene (tet) cassette with its 5′-regulatory region was amplified from the pBEST307 plasmid.19 At least 500 bp of sequences flanking both sides of the region to be deleted were amplified by PCR and joined upstream and downstream of the tet cassette by ligation using overlapping sequences in primers. The B. subtilis 168 Δupp strain and its derivatives were transformed with the resultant fragment to obtain a strain in which the target sequence was replaced with the tet gene (Step 1). Next, to obtain markerless mutants, fragments upstream and downstream of the target sequence were amplified, ligated, and cloned into the pBRcat/upp plasmid harboring the upp and chloramphenicol resistance (cat) gene. The resultant plasmid was integrated into the genome of the primary transformant with selection for tetracycline and chloramphenicol resistance (Step 2). The resultant strain became 5-FU sensitive as a result of introduction of the functional upp gene, and mutants without the plasmid sequence were selected on LB plates containing 10 μM 5FU (Step 3). The crosses indicate the recombination site." path-from-xml="dsn00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484399/dsn00202.jpeg" path-from-xml="dsn00202.jpeg" target="_blank">View large</a><a section="5484399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00202.jpeg?Expires=2147483647&Signature=oyXNy-H54hVGJ7o1J1M6ivOoc2jQniIg5ANHjwGHLOXczbRcH707B6lNNAKE-iCZcTpGZsWng3qi8tX3PNeZMnEPqdtB-Be2fsWQVTbCk4TkL3fWSwfMV6yNLQXD9ND5g7U03wVh2KpiE9p0oyi6xJic3ry4nL0I7ottOTUNg2~fMr8bJGf4j8vIPUY5VgTlrs6BifrflG5BL0UdlKsiRnaCfadC9~N5Cv19QiOdbWOvlxnaLJ2Csu8DGDh-SpGHJZg7EST47NsWkztTGeISR1LzoVbNB2b8nst9~SI1ACxKm1v-2-AlWJ~X2fjK~lDs02U7hcatIQzf4NZsG57ryg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484399&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequential introduction of large-scale deletions into the <em>B. subtilis</em> genome. A derivative of <em>B. subtilis</em> 168, 168 Δ<em>upp</em>, in which the <em>upp</em> gene encoding uracil-phosphoribosyl-transferase is inactivated by replacement with the erythromycin-resistance gene (<em>erm</em>), was used as the starting strain for generation of the deletion mutant series. The entire length of the tetracycline-resistant gene (<em>tet</em>) cassette with its 5′-regulatory region was amplified from the pBEST307 plasmid.<sup>19</sup> At least 500 bp of sequences flanking both sides of the region to be deleted were amplified by PCR and joined upstream and downstream of the <em>tet</em> cassette by ligation using overlapping sequences in primers. The <em>B. subtilis</em> 168 Δ<em>upp</em> strain and its derivatives were transformed with the resultant fragment to obtain a strain in which the target sequence was replaced with the <em>tet</em> gene (Step 1). Next, to obtain markerless mutants, fragments upstream and downstream of the target sequence were amplified, ligated, and cloned into the pBRcat/upp plasmid harboring the <em>upp</em> and chloramphenicol resistance (<em>cat</em>) gene. The resultant plasmid was integrated into the genome of the primary transformant with selection for tetracycline and chloramphenicol resistance (Step 2). The resultant strain became 5-FU sensitive as a result of introduction of the functional <em>upp</em> gene, and mutants without the plasmid sequence were selected on LB plates containing 10 μM 5FU (Step 3). The crosses indicate the recombination site.</p></div></div><div content-id="DSN002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00202.jpeg?Expires=2147483647&Signature=DUaDM6KYbJvWCWH8REAF2FZsI6Xv-zHGK15J5GZrZ8NtH0sjT-N9ismfOso53u3eFZjjaXQ9WSm6vjNj4NqzVzBzoz-yBpAmk5GCazLJNli2VZ8HA0yRwiW9n9b1F7UFUk9TAwWdrUF9ZOZzV-LNvgERO8oWFc1f4hA70-efkzyIJq9guwpB~Y-rcb66IxrMz29pS4N66oFc5Tjl0chN9aj7Ak5BZeLwqh8xNJftmnVoHrY0bdpO02JB8Nye0zszQnXCNCVxHo-ga-q-RbhbwBFYCoHnlhrD21fpVqiM4ScdOv23Ot0hsiExNhwgF-hoYW8feoWz4Uc-ZgdlDiEOHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequential introduction of large-scale deletions into the B. subtilis genome. A derivative of B. subtilis 168, 168 Δupp, in which the upp gene encoding uracil-phosphoribosyl-transferase is inactivated by replacement with the erythromycin-resistance gene (erm), was used as the starting strain for generation of the deletion mutant series. The entire length of the tetracycline-resistant gene (tet) cassette with its 5′-regulatory region was amplified from the pBEST307 plasmid.19 At least 500 bp of sequences flanking both sides of the region to be deleted were amplified by PCR and joined upstream and downstream of the tet cassette by ligation using overlapping sequences in primers. The B. subtilis 168 Δupp strain and its derivatives were transformed with the resultant fragment to obtain a strain in which the target sequence was replaced with the tet gene (Step 1). Next, to obtain markerless mutants, fragments upstream and downstream of the target sequence were amplified, ligated, and cloned into the pBRcat/upp plasmid harboring the upp and chloramphenicol resistance (cat) gene. The resultant plasmid was integrated into the genome of the primary transformant with selection for tetracycline and chloramphenicol resistance (Step 2). The resultant strain became 5-FU sensitive as a result of introduction of the functional upp gene, and mutants without the plasmid sequence were selected on LB plates containing 10 μM 5FU (Step 3). The crosses indicate the recombination site." path-from-xml="dsn00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484399/dsn00202.jpeg" path-from-xml="dsn00202.jpeg" target="_blank">View large</a><a section="5484399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00202.jpeg?Expires=2147483647&Signature=oyXNy-H54hVGJ7o1J1M6ivOoc2jQniIg5ANHjwGHLOXczbRcH707B6lNNAKE-iCZcTpGZsWng3qi8tX3PNeZMnEPqdtB-Be2fsWQVTbCk4TkL3fWSwfMV6yNLQXD9ND5g7U03wVh2KpiE9p0oyi6xJic3ry4nL0I7ottOTUNg2~fMr8bJGf4j8vIPUY5VgTlrs6BifrflG5BL0UdlKsiRnaCfadC9~N5Cv19QiOdbWOvlxnaLJ2Csu8DGDh-SpGHJZg7EST47NsWkztTGeISR1LzoVbNB2b8nst9~SI1ACxKm1v-2-AlWJ~X2fjK~lDs02U7hcatIQzf4NZsG57ryg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484399&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequential introduction of large-scale deletions into the <em>B. subtilis</em> genome. A derivative of <em>B. subtilis</em> 168, 168 Δ<em>upp</em>, in which the <em>upp</em> gene encoding uracil-phosphoribosyl-transferase is inactivated by replacement with the erythromycin-resistance gene (<em>erm</em>), was used as the starting strain for generation of the deletion mutant series. The entire length of the tetracycline-resistant gene (<em>tet</em>) cassette with its 5′-regulatory region was amplified from the pBEST307 plasmid.<sup>19</sup> At least 500 bp of sequences flanking both sides of the region to be deleted were amplified by PCR and joined upstream and downstream of the <em>tet</em> cassette by ligation using overlapping sequences in primers. The <em>B. subtilis</em> 168 Δ<em>upp</em> strain and its derivatives were transformed with the resultant fragment to obtain a strain in which the target sequence was replaced with the <em>tet</em> gene (Step 1). Next, to obtain markerless mutants, fragments upstream and downstream of the target sequence were amplified, ligated, and cloned into the pBRcat/upp plasmid harboring the <em>upp</em> and chloramphenicol resistance (<em>cat</em>) gene. The resultant plasmid was integrated into the genome of the primary transformant with selection for tetracycline and chloramphenicol resistance (Step 2). The resultant strain became 5-FU sensitive as a result of introduction of the functional <em>upp</em> gene, and mutants without the plasmid sequence were selected on LB plates containing 10 μM 5FU (Step 3). The crosses indicate the recombination site.</p></div></div>                            <h3 scrollto-destination=5484400 id="5484400" class="section-title" data-legacy-id=s3c>3.3. Exogenous protein productivity in the B. subtilis genome reduction mutants</h3>
<p>To assess the productivity of MGB874 cells in terms of exogenous protein secretion, we examined the production of thermostable alkaline cellulase, Egl237,<sup><a reveal-id="DSN002C13" data-open="DSN002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and alkaline protease (M-protease)<sup><a reveal-id="DSN002C14" data-open="DSN002C14" class="link link-ref link-reveal xref-bibr">14</a></sup> from a multicopy plasmid, pHY300PLK (∼50 copies per cell) harboring the respective genes under the control of a constitutive SigA-dependent promoter of Egl237.<sup><a reveal-id="DSN002C13" data-open="DSN002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> We introduced the pHYS237<sup>6</sup> (for cellulase production) or pHP237-K16<sup>6</sup> (for protease production) plasmids into the multiple-deletion series strains generated (MGB469 to MGB874) and measured the protease and cellulase activities after 75 h culture in 2xL-Mal, a model medium for industrial protein production (Fig. <a reveal-id="DSN002F3" data-open="DSN002F3" class="link link-reveal link-table xref-fig">3</a>A). Unexpectedly, the production of both enzymes increased in proportion to genome deletion lengths, with maximum levels estimated in the MGB874 strain. The activities of cellulase and protease in the culture medium of MGB874 cells were about 1.7- and 2.5-fold higher than those from wild-type cells, respectively. Increase in cell mass was similar for MGB874 and wild-type cells (Fig. <a reveal-id="DSN002F3" data-open="DSN002F3" class="link link-reveal link-table xref-fig">3</a>B), and cellulase production was arrested in wild-type 168 cells from 28 h upwards (Fig. <a reveal-id="DSN002F3" data-open="DSN002F3" class="link link-reveal link-table xref-fig">3</a>C). In contrast, the cellulase level in MGB874 cells continued to increase throughout the culture period to about twice that obtained from wild-type 168. The production period for protease was similarly elongated in the mutant strain (data not shown). Furthermore, maltose consumption in the culture medium was enhanced in MGB874 cells (Fig. <a reveal-id="DSN002F3" data-open="DSN002F3" class="link link-reveal link-table xref-fig">3</a>D), indicating that the efficiency of carbon source utilization also is improved as a result of genome reduction.</p>                        <a id="5484402" scrollto-destination="5484402"></a>
<div id="DSN002F3" data-id="DSN002F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00203.jpeg?Expires=2147483647&Signature=5Lu45f34Q9CqXrDxBNwcFYs6rP2zJEYuDDg~UfX3~xh~a6ysJuNXfr2-NPWSibsCkadclg-2-kbEO-6a9~7SVmCnQq23GCeD9zDUh9vTXzaxqdaJS7p8JIqrQYVvHSV9P-rfnx9b5eOf0yTHUXLS2XDK2Q5yPLo6BAYfx4y3fASJqHL70l-4NNJrEuV6sDjfL73tAYfq0VNU2VqHVqX9-XsrR07MxXz72a4SLZ-CvE25AhuQL8CYnibM6aWbEv4~2K9Fn3Y6QXz1NuTy3CIrDeuQkMJ9SpFG8a7hYXtnTn9ucNtl4RWaESCNJ4JAucivmBTHTq-5Clisu6wAs6WK-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Productivity of extracellular enzymes by the multiple-deletion series strains. (A) Relative activities of cellulase Egl237 (black bars) and M-protease (white bars) in growth medium of the multiple-deletion series, compared with those of the wild-type 168 strain after 75 h cultivation in 2xL-Mal medium, are indicated with error bars (average of three experiments). (B) The growth profiles of wild-type 168 (open diamonds) and MGB874 (closed squares) are presented. Arrows with a–f indicate the times of cell collection for transcriptome analysis. (C) Cellulase production as a function of cell growth. Extracellular cellulase activities of wild-type 168 (white bars) and MGB874 (black bars) cultures (0.4 μL) are shown. (D) Consumption of sugars in the growth medium. Wild-type 168 (open diamonds) and MGB874 (closed squares) cultures were collected at the indicated times. The total amount of sugar was determined according to a previous report.21" path-from-xml="dsn00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484402/dsn00203.jpeg" path-from-xml="dsn00203.jpeg" target="_blank">View large</a><a section="5484402" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00203.jpeg?Expires=2147483647&Signature=G2dShWW98sTOFLNvl70MdW0fjpU6rUB0dejNP2U6Z51nMkxwpAWnPkX-qvvE2~Ws82i~cHuF3JRbVWC~e6Ropmk7rCcQwgP8MFepdejs4kaQu-jjMldXKis9RoGrKfUIKy~Hj1Yz~lP24j9QT2wBasSNkytsUo19QZTr0R4ZzxDui5oZW7Xsmgd-QgG1E6veTp1lytxkYp7mNa2ewbAE8Vd-1YoOf-q8Knqf1RuzCMGQtNVNna-8qSdCh4H3D2Woz8SL8GrNQowIlyt2572GH0i3KemLJPB7EUP6pcyNmqP8nlfhXo5eounG6AIf92dXEp90KGeu23zZ6RjhloKuaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484402&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Productivity of extracellular enzymes by the multiple-deletion series strains. (<strong>A</strong>) Relative activities of cellulase Egl237 (black bars) and M-protease (white bars) in growth medium of the multiple-deletion series, compared with those of the wild-type 168 strain after 75 h cultivation in 2xL-Mal medium, are indicated with error bars (average of three experiments). (<strong>B</strong>) The growth profiles of wild-type 168 (open diamonds) and MGB874 (closed squares) are presented. Arrows with a–f indicate the times of cell collection for transcriptome analysis. (<strong>C</strong>) Cellulase production as a function of cell growth. Extracellular cellulase activities of wild-type 168 (white bars) and MGB874 (black bars) cultures (0.4 μL) are shown. (<strong>D</strong>) Consumption of sugars in the growth medium. Wild-type 168 (open diamonds) and MGB874 (closed squares) cultures were collected at the indicated times. The total amount of sugar was determined according to a previous report.<sup>21</sup></p></div></div><div content-id="DSN002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00203.jpeg?Expires=2147483647&Signature=5Lu45f34Q9CqXrDxBNwcFYs6rP2zJEYuDDg~UfX3~xh~a6ysJuNXfr2-NPWSibsCkadclg-2-kbEO-6a9~7SVmCnQq23GCeD9zDUh9vTXzaxqdaJS7p8JIqrQYVvHSV9P-rfnx9b5eOf0yTHUXLS2XDK2Q5yPLo6BAYfx4y3fASJqHL70l-4NNJrEuV6sDjfL73tAYfq0VNU2VqHVqX9-XsrR07MxXz72a4SLZ-CvE25AhuQL8CYnibM6aWbEv4~2K9Fn3Y6QXz1NuTy3CIrDeuQkMJ9SpFG8a7hYXtnTn9ucNtl4RWaESCNJ4JAucivmBTHTq-5Clisu6wAs6WK-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Productivity of extracellular enzymes by the multiple-deletion series strains. (A) Relative activities of cellulase Egl237 (black bars) and M-protease (white bars) in growth medium of the multiple-deletion series, compared with those of the wild-type 168 strain after 75 h cultivation in 2xL-Mal medium, are indicated with error bars (average of three experiments). (B) The growth profiles of wild-type 168 (open diamonds) and MGB874 (closed squares) are presented. Arrows with a–f indicate the times of cell collection for transcriptome analysis. (C) Cellulase production as a function of cell growth. Extracellular cellulase activities of wild-type 168 (white bars) and MGB874 (black bars) cultures (0.4 μL) are shown. (D) Consumption of sugars in the growth medium. Wild-type 168 (open diamonds) and MGB874 (closed squares) cultures were collected at the indicated times. The total amount of sugar was determined according to a previous report.21" path-from-xml="dsn00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484402/dsn00203.jpeg" path-from-xml="dsn00203.jpeg" target="_blank">View large</a><a section="5484402" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00203.jpeg?Expires=2147483647&Signature=G2dShWW98sTOFLNvl70MdW0fjpU6rUB0dejNP2U6Z51nMkxwpAWnPkX-qvvE2~Ws82i~cHuF3JRbVWC~e6Ropmk7rCcQwgP8MFepdejs4kaQu-jjMldXKis9RoGrKfUIKy~Hj1Yz~lP24j9QT2wBasSNkytsUo19QZTr0R4ZzxDui5oZW7Xsmgd-QgG1E6veTp1lytxkYp7mNa2ewbAE8Vd-1YoOf-q8Knqf1RuzCMGQtNVNna-8qSdCh4H3D2Woz8SL8GrNQowIlyt2572GH0i3KemLJPB7EUP6pcyNmqP8nlfhXo5eounG6AIf92dXEp90KGeu23zZ6RjhloKuaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484402&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Productivity of extracellular enzymes by the multiple-deletion series strains. (<strong>A</strong>) Relative activities of cellulase Egl237 (black bars) and M-protease (white bars) in growth medium of the multiple-deletion series, compared with those of the wild-type 168 strain after 75 h cultivation in 2xL-Mal medium, are indicated with error bars (average of three experiments). (<strong>B</strong>) The growth profiles of wild-type 168 (open diamonds) and MGB874 (closed squares) are presented. Arrows with a–f indicate the times of cell collection for transcriptome analysis. (<strong>C</strong>) Cellulase production as a function of cell growth. Extracellular cellulase activities of wild-type 168 (white bars) and MGB874 (black bars) cultures (0.4 μL) are shown. (<strong>D</strong>) Consumption of sugars in the growth medium. Wild-type 168 (open diamonds) and MGB874 (closed squares) cultures were collected at the indicated times. The total amount of sugar was determined according to a previous report.<sup>21</sup></p></div></div>                            <h3 scrollto-destination=5484403 id="5484403" class="section-title" data-legacy-id=s3d>3.4. Reprogramming of the transcriptional regulatory network in MGB874 cells</h3>
<p>To assess the molecular events underlying these unexpected phenomena, we compared the transcriptome profiles of MGB874 and wild-type 168 cells during growth in 2xL-Mal medium using a custom Affymetrix tiling chip, as described previously,<sup><a reveal-id="DSN002C15" data-open="DSN002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and expression levels of each genes in MGB874 and 168 cells were determined as described in Materials and Methods (Supplementary Table S3). Transcriptome analysis revealed that the expression of a limited number of genes was disrupted in MGB874 cells at the early exponential growth phase. The correlation coefficient of the expression levels of each gene between the two strains was 0.94 (Fig. <a reveal-id="DSN002F4" data-open="DSN002F4" class="link link-reveal link-table xref-fig">4</a>A), but decreased as a function of culture time to 0.71 after 60 h (Fig. <a reveal-id="DSN002F4" data-open="DSN002F4" class="link link-reveal link-table xref-fig">4</a>B–F).</p>                        <a id="5484405" scrollto-destination="5484405"></a>
<div id="DSN002F4" data-id="DSN002F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00204.jpeg?Expires=2147483647&Signature=yGf5GgqhbValEa7bVnBSheiHPMvmVlUEYElFJlWGpywAMcrOVNlgb7OIYWmRmEOs1m9hnWU0cKR8EtBXiAjoe1lMaVzZHPF1MgNZ7rw7rqmn~leIzoY8XPIZLzZb0Pt9tO8P8mLFoqosGqZIweWfOmBxcbR5p2aShat7Yg1YUxMWvlpxDLjnRdbM0qFsX2RNpLZYebkWWNLJd1aM7yfSVlopcm1AHaAQGG3yBM5CiMpybcF2drBnEm2gNQiTvbRY3RiBUf-F54d6V9QTezmrOeolnSuQ4qjHUyQ4uAw-G4ZSvv9WNjox-ahPLtyYkX5sKHsOj8PGGj8FdvMfeo5oCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of gene expression in wild-type 168 and MGB874 cells. Total RNA was extracted from both strains grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) hours (Fig. 3B) and used for tiling chip analysis. The average signal intensities of probes in each coding sequence were calculated after removal of the lowest and highest intensities. Scatter plots of expression in wild-type 168 and MGB874 cells are presented. Genes deleted in MGB874 were excluded from the analysis. The correlation coefficient is indicated." path-from-xml="dsn00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484405/dsn00204.jpeg" path-from-xml="dsn00204.jpeg" target="_blank">View large</a><a section="5484405" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00204.jpeg?Expires=2147483647&Signature=15AMFt-N~TzcV3N7id25v8DsdZ3bpctIHk15Mbn4fJM7TwRe28AMiHDQtCuKBDabGi3YEK5O180Wna95e4nl9xC456LDbmzGNa5JKpipR3xIZlkDvtVp86BMMMh7wwSakF6xpXNpaCmUCJb7vKfJ7wRdt7oJj5U0MGendlZYurXE4ghRVQQXJt14TgInQnSeqiVWHbcDzJX5p7vaKep27qKgLrXHM1UPO~91dAOuQpDGFICZG~aOsydqxgtedM40wj658ICg8N0o6oKTocTvuECMccRRgVVKR5SWzSnysAHWw4fHmQrfWDjvmKT8FrwO7kWyBjHIuH1rEybRRvdKSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484405&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of gene expression in wild-type 168 and MGB874 cells. Total RNA was extracted from both strains grown in 2xL-Mal medium for 1 (<strong>A</strong>), 7.5 (<strong>B</strong>), 13 (<strong>C</strong>), 26 (<strong>D</strong>), 40 (<strong>E</strong>), and 60 (<strong>F</strong>) hours (Fig. 3B) and used for tiling chip analysis. The average signal intensities of probes in each coding sequence were calculated after removal of the lowest and highest intensities. Scatter plots of expression in wild-type 168 and MGB874 cells are presented. Genes deleted in MGB874 were excluded from the analysis. The correlation coefficient is indicated.</p></div></div><div content-id="DSN002F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00204.jpeg?Expires=2147483647&Signature=yGf5GgqhbValEa7bVnBSheiHPMvmVlUEYElFJlWGpywAMcrOVNlgb7OIYWmRmEOs1m9hnWU0cKR8EtBXiAjoe1lMaVzZHPF1MgNZ7rw7rqmn~leIzoY8XPIZLzZb0Pt9tO8P8mLFoqosGqZIweWfOmBxcbR5p2aShat7Yg1YUxMWvlpxDLjnRdbM0qFsX2RNpLZYebkWWNLJd1aM7yfSVlopcm1AHaAQGG3yBM5CiMpybcF2drBnEm2gNQiTvbRY3RiBUf-F54d6V9QTezmrOeolnSuQ4qjHUyQ4uAw-G4ZSvv9WNjox-ahPLtyYkX5sKHsOj8PGGj8FdvMfeo5oCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of gene expression in wild-type 168 and MGB874 cells. Total RNA was extracted from both strains grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) hours (Fig. 3B) and used for tiling chip analysis. The average signal intensities of probes in each coding sequence were calculated after removal of the lowest and highest intensities. Scatter plots of expression in wild-type 168 and MGB874 cells are presented. Genes deleted in MGB874 were excluded from the analysis. The correlation coefficient is indicated." path-from-xml="dsn00204.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484405/dsn00204.jpeg" path-from-xml="dsn00204.jpeg" target="_blank">View large</a><a section="5484405" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00204.jpeg?Expires=2147483647&Signature=15AMFt-N~TzcV3N7id25v8DsdZ3bpctIHk15Mbn4fJM7TwRe28AMiHDQtCuKBDabGi3YEK5O180Wna95e4nl9xC456LDbmzGNa5JKpipR3xIZlkDvtVp86BMMMh7wwSakF6xpXNpaCmUCJb7vKfJ7wRdt7oJj5U0MGendlZYurXE4ghRVQQXJt14TgInQnSeqiVWHbcDzJX5p7vaKep27qKgLrXHM1UPO~91dAOuQpDGFICZG~aOsydqxgtedM40wj658ICg8N0o6oKTocTvuECMccRRgVVKR5SWzSnysAHWw4fHmQrfWDjvmKT8FrwO7kWyBjHIuH1rEybRRvdKSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484405&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00204.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of gene expression in wild-type 168 and MGB874 cells. Total RNA was extracted from both strains grown in 2xL-Mal medium for 1 (<strong>A</strong>), 7.5 (<strong>B</strong>), 13 (<strong>C</strong>), 26 (<strong>D</strong>), 40 (<strong>E</strong>), and 60 (<strong>F</strong>) hours (Fig. 3B) and used for tiling chip analysis. The average signal intensities of probes in each coding sequence were calculated after removal of the lowest and highest intensities. Scatter plots of expression in wild-type 168 and MGB874 cells are presented. Genes deleted in MGB874 were excluded from the analysis. The correlation coefficient is indicated.</p></div></div><p>When nutrients in the culture medium were exhausted, <em>B. subtilis</em> cells entered the transition state, accompanied by various phenomena, such as competence for transformation, secretion of degradative enzymes, and induction of motility through a complex network of global regulators, including ComPA and DegSU two-component systems, the AbrB transition state regulator, and the alternative sigma factor, SigD. Finally, phosphorylation of a two-component regulator, Spo0A, triggers a regulatory program for spore formation involving the activation of an alternative sigma factor, Spo0H, and sporulation-specific sigma factors, SigF and SigG, in prespores and SigE and SigK in mother cells.<sup><a reveal-id="DSN002C22" data-open="DSN002C22" class="link link-ref link-reveal xref-bibr">22</a></sup> A closer examination of the changes in transcriptome profiles associated with entry into the stationary phase, and activation of the sporulation program in MGB874 and wild-type cells disclosed remarkable differences in these processes between the two strains. The ComA regulator that induces competence was activated earlier in MGB874 cells, observed as the expression of the <em>srf</em> operon and <em>pel</em> gene that are directly stimulated by ComA.<sup><a reveal-id="DSN002C23" data-open="DSN002C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Moreover, competence genes that are indirectly activated by ComA via the master regulator, ComK, were induced earlier in MGB874 cells, compared with the wild-type strain (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>A). The <em>degU</em> regulator participating in competence activation in the non-phosphorylated form<sup><a reveal-id="DSN002C24" data-open="DSN002C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and protein production in the highly phosphorylated form<sup><a reveal-id="DSN002C25" data-open="DSN002C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was induced earlier and maintained at high levels throughout the culture period in MGB874 cells (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>B). This altered feature may also contribute to the earlier competence development in MGB874 cells. However, degradative protein production, possibly dependent on DegU, was similar in both MGB874 and wild-type cells, except for major intracellular serine protease (IspA) and bacillopeptidase F (Bpr), which were strongly induced in MGB874 cells at the late growth phase (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>B). In contrast, inhibition of the expression of the transition state regulator, AbrB, which represses transition-state genes in the exponential growth phase and is suppressed by phospho-Spo0A,<sup><a reveal-id="DSN002C22" data-open="DSN002C22" class="link link-ref link-reveal xref-bibr">22</a></sup> was delayed in MGB874 cells (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>C). Consistent with this observation, MGB874 cells displayed disrupted expression of SigH, which is repressed by AbrB and activated by phospho-Spo0A. In addition, activation of sporulation-specific sigma factors was delayed in MGB874 cells, as evident from the delay in the expression patterns of genes under their control (except those directed by SigK, which is deleted in MGB874) (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>D). These findings suggest that the transition state is extended in MGB874 cells, compared with the wild-type strain. Furthermore, expression levels of the <em>ctaABCDEFG</em> operon encoding cytochrome <em>c</em> oxidase and the <em>atpIBEFHAGDC</em> operon encoding F<sub>0</sub>F<sub>1</sub>ATP synthase in MGB874 cells were maintained throughout the culture period (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>E). Interestingly, <em>appDFABC</em> and <em>dppBCDE</em> encoding ABC transporters for oligopeptide and dipeptide, respectively, <em>malP</em> representing maltose-specific PTS enzyme, and <em>bioWFADB</em> coding for biotin biosynthesis enzymes were active throughout the culture period (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>E). Thus, it appears that metabolic activities are maintained in the extended transition state of MGB874 cells. Global regulators for the transition state influence each other in terms of expression, forming a complex regulatory network. Moreover, activities of ComA, DegU, and Spo0F (phosphorylates Spo0A) are regulated by Rap family proteins to control the precise activation timing.<sup><a reveal-id="DSN002C22" data-open="DSN002C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Six <em>rap</em> genes among the 11 encoded in the <em>B. subtilis</em> genome were deleted in the MGB874 strain, which may attribute to the disruption of transition state timing and gene expression during sporulation. A significant increase of protein productivity occurred in the deletion process from MGB625 to MGB723 (Fig. <a reveal-id="DSN002F3" data-open="DSN002F3" class="link link-reveal link-table xref-fig">3</a>A). The <em>rapG</em> gene and its regulator <em>phrG</em> were deleted in this process, suggesting that the deletion of RapG had a major impact on protein productivity among deletions of six rap genes.</p>                        <a id="5484407" scrollto-destination="5484407"></a>
<div id="DSN002F5" data-id="DSN002F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00205.jpeg?Expires=2147483647&Signature=aZvTB00dUTbN8~1M5cFXbSWqJ3RHYZ8a-COhakkBWlKrXsJ0jMlhlJqyG-pBUgF1saiaBNXbmgkoOoGzZvpVMg55NTADvF1Dgk2iN3Vt1nqqGal4iwYTCr0wGDgF5INU3fClnqSJ1WMTHK95W~kxZkYa82sr1aoBETGNXzCRrpX~4iCull31csAuRnsxbI78qxl6wyorgyOePg6L5a-1TvTiEby4K6MCpqEtAvNJGbg9LS1RjK96en~4H6ZIGJwCriAFQcJZH6RxmJTZxJ5fAFHlcdTOYxarOIkKSou5Pj3de-zuYflgRibgGTIhjGchVA9hicd-8LI2BCTJgM5n4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of genes with significantly disrupted expression in MGB874 cells, compared with wild-type cells. The average signal intensities of probes in each coding sequence in wild-type 168 (open diamonds) and MGB874 (closed squares) cells grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) h (Fig. 3B) are indicated. Regulators of expression are specified in parentheses. (A) ComA and ComK-dependent genes. (B) DegU-dependent genes. (C) Transition state regulators. (D) Sporulation genes. (E) Genes with markedly elongated expression in MGB874 cells. (F) Genes with markedly suppressed expression in MGB874 cells. (G) Genes that are specifically induced in MGB874 cells. (H) Phosphate metabolism-related genes." path-from-xml="dsn00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484407/dsn00205.jpeg" path-from-xml="dsn00205.jpeg" target="_blank">View large</a><a section="5484407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00205.jpeg?Expires=2147483647&Signature=v50vVGf~JJ7oUUFVob8bB1ctUtANCe3bzaQEmxKS~WthP30o2T5aLRUYR1NRhklYJjozY6V4qeaXoMTOAYfIovWNDhLznAAfqGEuynuJVZhyfIajRk1OU9Kz6epIr9tqGGnyck7AVGAGV2lxFQkBOxyEircSHyhMFn6a1hZcPErR-cX8SUtbSMHlXIoHm7QSOizzpZOyIf1tIVBqu~ZuTrETUTVFcPZp8msayMNKH7nUbsBkWhUkdwXbGOYevV24wERqmGkEBPOjQJhH8Fe7wZv15~xrCX8unc2~97ZRdm8Gs26wiKTdfgV1AwjJTjgxqX--MSlIserqg9D9Htq~gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484407&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of genes with significantly disrupted expression in MGB874 cells, compared with wild-type cells. The average signal intensities of probes in each coding sequence in wild-type 168 (open diamonds) and MGB874 (closed squares) cells grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) h (Fig. 3B) are indicated. Regulators of expression are specified in parentheses. (<strong>A</strong>) ComA and ComK-dependent genes. (<strong>B</strong>) DegU-dependent genes. (<strong>C</strong>) Transition state regulators. (<strong>D</strong>) Sporulation genes. (<strong>E</strong>) Genes with markedly elongated expression in MGB874 cells. (<strong>F</strong>) Genes with markedly suppressed expression in MGB874 cells. (<strong>G</strong>) Genes that are specifically induced in MGB874 cells. (<strong>H</strong>) Phosphate metabolism-related genes.</p></div></div><div content-id="DSN002F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/m_dsn00205.jpeg?Expires=2147483647&Signature=aZvTB00dUTbN8~1M5cFXbSWqJ3RHYZ8a-COhakkBWlKrXsJ0jMlhlJqyG-pBUgF1saiaBNXbmgkoOoGzZvpVMg55NTADvF1Dgk2iN3Vt1nqqGal4iwYTCr0wGDgF5INU3fClnqSJ1WMTHK95W~kxZkYa82sr1aoBETGNXzCRrpX~4iCull31csAuRnsxbI78qxl6wyorgyOePg6L5a-1TvTiEby4K6MCpqEtAvNJGbg9LS1RjK96en~4H6ZIGJwCriAFQcJZH6RxmJTZxJ5fAFHlcdTOYxarOIkKSou5Pj3de-zuYflgRibgGTIhjGchVA9hicd-8LI2BCTJgM5n4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of genes with significantly disrupted expression in MGB874 cells, compared with wild-type cells. The average signal intensities of probes in each coding sequence in wild-type 168 (open diamonds) and MGB874 (closed squares) cells grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) h (Fig. 3B) are indicated. Regulators of expression are specified in parentheses. (A) ComA and ComK-dependent genes. (B) DegU-dependent genes. (C) Transition state regulators. (D) Sporulation genes. (E) Genes with markedly elongated expression in MGB874 cells. (F) Genes with markedly suppressed expression in MGB874 cells. (G) Genes that are specifically induced in MGB874 cells. (H) Phosphate metabolism-related genes." path-from-xml="dsn00205.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484407/dsn00205.jpeg" path-from-xml="dsn00205.jpeg" target="_blank">View large</a><a section="5484407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn00205.jpeg?Expires=2147483647&Signature=v50vVGf~JJ7oUUFVob8bB1ctUtANCe3bzaQEmxKS~WthP30o2T5aLRUYR1NRhklYJjozY6V4qeaXoMTOAYfIovWNDhLznAAfqGEuynuJVZhyfIajRk1OU9Kz6epIr9tqGGnyck7AVGAGV2lxFQkBOxyEircSHyhMFn6a1hZcPErR-cX8SUtbSMHlXIoHm7QSOizzpZOyIf1tIVBqu~ZuTrETUTVFcPZp8msayMNKH7nUbsBkWhUkdwXbGOYevV24wERqmGkEBPOjQJhH8Fe7wZv15~xrCX8unc2~97ZRdm8Gs26wiKTdfgV1AwjJTjgxqX--MSlIserqg9D9Htq~gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484407&ar=376500&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00205.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of genes with significantly disrupted expression in MGB874 cells, compared with wild-type cells. The average signal intensities of probes in each coding sequence in wild-type 168 (open diamonds) and MGB874 (closed squares) cells grown in 2xL-Mal medium for 1 (A), 7.5 (B), 13 (C), 26 (D), 40 (E), and 60 (F) h (Fig. 3B) are indicated. Regulators of expression are specified in parentheses. (<strong>A</strong>) ComA and ComK-dependent genes. (<strong>B</strong>) DegU-dependent genes. (<strong>C</strong>) Transition state regulators. (<strong>D</strong>) Sporulation genes. (<strong>E</strong>) Genes with markedly elongated expression in MGB874 cells. (<strong>F</strong>) Genes with markedly suppressed expression in MGB874 cells. (<strong>G</strong>) Genes that are specifically induced in MGB874 cells. (<strong>H</strong>) Phosphate metabolism-related genes.</p></div></div><p>Maintenance of metabolic activities in the extended transition state of MGB874 cells would be a basis for increased enzyme productivity. In addition, the remarkable changes in gene expression observed might be related to higher protein productivity of MGB874 cells (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>F). <em>B. subtilis</em> SigD, activated in the transition state, is responsible for the transcription of genes for flagellar assembly, motility, chemotaxis, and autolysis.<sup><a reveal-id="DSN002C26" data-open="DSN002C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSN002C27" data-open="DSN002C27" class="link link-ref link-reveal xref-bibr">27</a></sup> However, significantly low transcription of SigD and genes under its control was observed in MGB874 cells. The <em>argGH</em> and <em>argCJBD</em> operons for arginine biosynthesis were induced during the transition state in wild-type cells, but not in the mutant MGB874 cells. In contrast, a number of genes were strongly induced in MGB874 cells, including <em>yxkJ</em> encoding L-malate and citrate transporter, <em>levDEFG</em> encoding fructose-specific PTS enzyme, and <em>oxdC</em> encoding cytosolic oxalate decarboxylase (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>G). We additionally observed the earlier and stronger induction of genes related to phosphate metabolism under the control of the PhoPR two-component system<sup><a reveal-id="DSN002C28" data-open="DSN002C28" class="link link-ref link-reveal xref-bibr">28</a></sup> in MGB874 cells, including <em>glpQ</em> encoding glycerophosphoryl diester phosphodiesterase, <em>phoD</em> (phosphodiesterase), <em>phoA</em> (alkaline phosphatase A), <em>pstSCAB</em> (phosphate transporter), and <em>tuaABCDEFG</em> (teichuronic acid biosynthesis) (Fig. <a reveal-id="DSN002F5" data-open="DSN002F5" class="link link-reveal link-table xref-fig">5</a>H).</p><p>These results indicate that considerable reprogramming of the transcriptional regulatory network occurs in MGB874 cells, probably due to the synergistic effect of multiple deletions, although the molecular basis for these changes and their relationship to increased protein productivity await further investigation.</p>                            <h2 scrollto-destination=5484410 id="5484410" class="section-title" data-legacy-id=s4>4. Conclusion</h2>
<p>To our knowledge, this is the first report demonstrating that genome reduction actually contributes to the creation of bacterial cells with a practical application in industry. It is not clear at the moment that phenomena we observed are due to a global synergistic effect of large-scale genome reduction or mainly due to the deletion of several regulators. Further systematic analysis of changes in the transcriptional regulatory network of MGB874 cells in relation to protein productivity should facilitate the generation of improved <em>B. subtilis</em> cells as hosts of industrial protein production. We are in the process of introducing further rational deletions on the basis of transcriptome data, gene function information, and comparative genomics, with a view to generating simple, predictable cells comprising genes with defined functions as a new platform of development of bacterial strains for industrial use.</p>                            <h2 scrollto-destination=5484412 id="5484412" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work is part of the subproject ‘Development of a Technology for the Creation of a Host Cell’ included within the industrial technology project ‘Development of Generic Technology for Production Process Starting Productive Function’ of the Ministry of Economy, Trade and Industry, entrusted by the New Energy and Industrial Technology Development Organization (NEDO), Japan.</p>                            <h2 scrollto-destination=5484414 id="5484414" class="section-title" data-legacy-id=s6>Supplementary Data</h2>
<p>Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5484416 id="5484416" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Shu Ishikawa and Taku Oshima for help in transcriptome analysis and to Junichi Sekiguchi, Kouji Nakamura, and Fujio Kawamura for valuable discussions. This work is part of the subproject ‘Development of a Technology for the Creation of a Host Cell’ included within the industrial technology project ‘Development of Generic Technology for Production Process Starting Productive Function’ of the Ministry of Economy, Trade and Industry, entrusted by the New Energy and Industrial Technology Development Organization (NEDO), Japan.</p>                            <h2 scrollto-destination=5484418 id="5484418" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ball</div> <div class="given-names">P</div></div>. </div><div class="article-title">Synthetic biology: designs for life</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">448</div> (pg. <div class="fpage">32</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthetic%20biology%3A%20designs%20for%20life&amp;author=P%20Ball&amp;publication_year=2007&amp;journal=Nature&amp;volume=448&amp;pages=32-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/448032a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F448032a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F448032a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17611530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Drubin</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Way</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Silver</div> <div class="given-names">P. A</div></div>. </div><div class="article-title">Designing biological systems</div>, <div class="source">Genes Develop.</div> , <div class="year">2007</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">242</div>-<div class="lpage">254</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Designing%20biological%20systems&amp;author=D.%20A.%20Drubin&amp;author=J.%20C.%20Way&amp;author=P.%20A%20Silver&amp;publication_year=2007&amp;journal=Genes%20Develop.&amp;volume=21&amp;pages=242-254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1507207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1507207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1507207"> </span></div> </div></div></div></div></div><div content-id="DSN002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Forster</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">G. M</div></div>. </div><div class="article-title">Synthetic biology projects <em>in vitro</em></div>, <div class="source">Genome Res</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthetic%20biology%20projects%20in%20vitro&amp;author=A.%20C.%20Forster&amp;author=G.%20M%20Church&amp;publication_year=2007&amp;journal=Genome%20Res&amp;volume=17&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5776007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5776007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5776007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17151344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Posfai</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Plunkett</div> <div class="given-names">G.</div>III</div>,  <div class="name"><div class="surname">Feher</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Frisch</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Keil</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Umenhoffer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kolisnychenko</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Stahl</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">S. S.</div></div>,  <div class="name"><div class="surname">de Arruda</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Emergent properties of reduced-genome <em>Escherichia coli</em></div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1044</div>-<div class="lpage">1046</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergent%20properties%20of%20reduced-genome%20Escherichia%20coli&amp;author=G.%20Posfai&amp;author=G.%20Plunkett&amp;author=T.%20Feher&amp;author=D.%20Frisch&amp;author=G.%20M.%20Keil&amp;author=K.%20Umenhoffer&amp;author=V.%20Kolisnychenko&amp;author=B.%20Stahl&amp;author=S.%20S.%20Sharma&amp;author=M.%20de%20Arruda&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1044-1046" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1126439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1126439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1126439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16645050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fujio</div> <div class="given-names">T</div></div>. </div><div class="article-title"><em>Escherichia coli</em> minimum genome factory</div>, <div class="source">Biotechnol. Appl. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">157</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20minimum%20genome%20factory&amp;author=H.%20Mizoguchi&amp;author=H.%20Mori&amp;author=T%20Fujio&amp;publication_year=2007&amp;journal=Biotechnol.%20Appl.%20Biochem.&amp;volume=46&amp;pages=157-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BA20060107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBA20060107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBA20060107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17300222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ara</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ozaki</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamane</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sekiguchi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N</div></div>. </div><div class="article-title"><em>Bacillus</em> minimum genome factory: effective utilization of microbial genome information</div>, <div class="source">Biotechnol. Appl. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">169</div>-<div class="lpage">178</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacillus%20minimum%20genome%20factory%3A%20effective%20utilization%20of%20microbial%20genome%20information&amp;author=K.%20Ara&amp;author=K.%20Ozaki&amp;author=K.%20Nakamura&amp;author=K.%20Yamane&amp;author=J.%20Sekiguchi&amp;author=N%20Ogasawara&amp;publication_year=2007&amp;journal=Biotechnol.%20Appl.%20Biochem.&amp;volume=46&amp;pages=169-178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BA20060111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBA20060111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBA20060111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17115975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Westers</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Dorenbos</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">van Dijl</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Kabel</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Flanagan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Jude</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Seror</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Beekman</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Darmon</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Genome engineering reveals large dispensable regions in <em>Bacillus subtilis</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2076</div>-<div class="lpage">2090</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20engineering%20reveals%20large%20dispensable%20regions%20in%20Bacillus%20subtilis&amp;author=H.%20Westers&amp;author=R.%20Dorenbos&amp;author=J.%20M.%20van%20Dijl&amp;author=J.%20Kabel&amp;author=T.%20Flanagan&amp;author=K.%20M.%20Devine&amp;author=F.%20Jude&amp;author=S.%20J.%20Seror&amp;author=A.%20C.%20Beekman&amp;author=E.%20Darmon&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=2076-2090" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Westers</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Westers</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Quax</div> <div class="given-names">W. J</div></div>. </div><div class="article-title"><em>Bacillus subtilis</em> as cell factory for pharmaceutical proteins: a biotechnological approach to optimize the host organism</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">2004</div>, vol. <div class="volume">1694</div> (pg. <div class="fpage">299</div>-<div class="lpage">310</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacillus%20subtilis%20as%20cell%20factory%20for%20pharmaceutical%20proteins%3A%20a%20biotechnological%20approach%20to%20optimize%20the%20host%20organism&amp;author=L.%20Westers&amp;author=H.%20Westers&amp;author=W.%20J%20Quax&amp;publication_year=2004&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1694&amp;pages=299-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbamcr.2004.02.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbamcr.2004.02.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbamcr.2004.02.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15546673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunst</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Albertini</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Alloni</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Azevedo</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Bertero</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Bessieres</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bolotin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borchert</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The complete genome sequence of the gram-positive bacterium <em>Bacillus subtilis</em></div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">390</div> (pg. <div class="fpage">249</div>-<div class="lpage">256</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20the%20gram-positive%20bacterium%20Bacillus%20subtilis&amp;author=F.%20Kunst&amp;author=N.%20Ogasawara&amp;author=I.%20Moszer&amp;author=A.%20M.%20Albertini&amp;author=G.%20Alloni&amp;author=V.%20Azevedo&amp;author=M.%20G.%20Bertero&amp;author=P.%20Bessieres&amp;author=A.%20Bolotin&amp;author=S.%20Borchert&amp;publication_year=1997&amp;journal=Nature&amp;volume=390&amp;pages=249-256" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/36786" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F36786" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F36786"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9384377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ehrlich</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Albertini</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Amati</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Andersen</div> <div class="given-names">K. K.</div></div>,  <div class="name"><div class="surname">Arnaud</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ashikaga</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Aymerich</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bessieres</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Essential <em>Bacillus subtilis</em> genes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">4678</div>-<div class="lpage">4683</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essential%20Bacillus%20subtilis%20genes&amp;author=K.%20Kobayashi&amp;author=S.%20D.%20Ehrlich&amp;author=A.%20Albertini&amp;author=G.%20Amati&amp;author=K.%20K.%20Andersen&amp;author=M.%20Arnaud&amp;author=K.%20Asai&amp;author=S.%20Ashikaga&amp;author=S.%20Aymerich&amp;author=P.%20Bessieres&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=4678-4683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0730515100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0730515100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0730515100"> </span></div> </div></div></div></div></div><div content-id="DSN002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anagnostopoulos</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Spizizen</div> <div class="given-names">J</div></div>. </div><div class="article-title">Requirements for transformation in <em>Bacillus Subtilis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1961</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">741</div>-<div class="lpage">746</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Requirements%20for%20transformation%20in%20Bacillus%20Subtilis&amp;author=C.%20Anagnostopoulos&amp;author=J%20Spizizen&amp;publication_year=1961&amp;journal=J.%20Bacteriol.&amp;volume=81&amp;pages=741-746" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16561900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harwood</div> <div class="given-names">C. R.</div></div>,  <div class="name"><div class="surname">Archibald</div> <div class="given-names">A. R</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Harwood</div> <div class="given-names">C. R.</div></div>,  <div class="name"><div class="surname">Cutting</div> <div class="given-names">S. M</div></div>. </div><div class="article-title">Growth, maintenance and general techniques, In</div>, <div class="source">Molecular Biological Methods for Bacillus</div> , <div class="year">1990</div><div class="publisher-loc">Chichester, New York, Brisbane, Toronto, Singapore</div><div class="publisher-name">John Wiley &amp; Sons</div>pg. <div class="fpage">549</div> </div></div></div></div><div content-id="DSN002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hakamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hatada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshimatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">S</div></div>. </div><div class="article-title">Deduced amino acid sequence and possible catalytic residues of a thermostable, alkaline cellulase from an Alkaliphilic <em>bacillus</em> strain</div>, <div class="source">Biosci. Biotech. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">2281</div>-<div class="lpage">2289</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deduced%20amino%20acid%20sequence%20and%20possible%20catalytic%20residues%20of%20a%20thermostable%2C%20alkaline%20cellulase%20from%20an%20Alkaliphilic%20bacillus%20strain&amp;author=Y.%20Hakamada&amp;author=Y.%20Hatada&amp;author=K.%20Koike&amp;author=T.%20Yoshimatsu&amp;author=S.%20Kawai&amp;author=T.%20Kobayashi&amp;author=S%20Ito&amp;publication_year=2000&amp;journal=Biosci.%20Biotech.%20Biochem.&amp;volume=64&amp;pages=2281-2289" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.64.2281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.64.2281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.64.2281"> </span></div> </div></div></div></div></div><div content-id="DSN002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kobayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hakamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hitomi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yoshimatsu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ito</div> <div class="given-names">S</div></div>. </div><div class="article-title">Purification and properties of an alkaline protease from alkalophilic <em>Bacillus</em> sp. KSM-K16</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">1995</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">473</div>-<div class="lpage">481</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20properties%20of%20an%20alkaline%20protease%20from%20alkalophilic%20Bacillus%20sp.%20KSM-K16&amp;author=T.%20Kobayashi&amp;author=Y.%20Hakamada&amp;author=S.%20Adachi&amp;author=J.%20Hitomi&amp;author=T.%20Yoshimatsu&amp;author=K.%20Koike&amp;author=S.%20Kawai&amp;author=S%20Ito&amp;publication_year=1995&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=43&amp;pages=473-481" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00218452" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00218452" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00218452"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7632397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ogura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yoshimura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okumura</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cho</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N</div></div>. </div><div class="article-title">Distribution of stable DnaA binding sites on the <em>Bacillus subtilis</em> genome detected using a modified ChIP-chip method</div>, <div class="source">DNA Res</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">155</div>-<div class="lpage">168</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20stable%20DnaA%20binding%20sites%20on%20the%20Bacillus%20subtilis%20genome%20detected%20using%20a%20modified%20ChIP-chip%20method&amp;author=S.%20Ishikawa&amp;author=Y.%20Ogura&amp;author=M.%20Yoshimura&amp;author=H.%20Okumura&amp;author=E.%20Cho&amp;author=Y.%20Kawai&amp;author=K.%20Kurokawa&amp;author=T.%20Oshima&amp;author=N%20Ogasawara&amp;publication_year=2007&amp;journal=DNA%20Res&amp;volume=14&amp;pages=155-168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Igo</div> <div class="given-names">M. M.</div></div>,  <div class="name"><div class="surname">Losick</div> <div class="given-names">R</div></div>. </div><div class="article-title">Regulation of a promoter that is utilized by minor forms of RNA polymerase holoenzyme in <em>Bacillus subtilis</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1986</div>, vol. <div class="volume">191</div> (pg. <div class="fpage">615</div>-<div class="lpage">624</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20a%20promoter%20that%20is%20utilized%20by%20minor%20forms%20of%20RNA%20polymerase%20holoenzyme%20in%20Bacillus%20subtilis&amp;author=M.%20M.%20Igo&amp;author=R%20Losick&amp;publication_year=1986&amp;journal=J.%20Mol.%20Biol.&amp;volume=191&amp;pages=615-624" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(86)90449-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(86)90449-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(86)90449-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3100810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quackenbush</div> <div class="given-names">J</div></div>. </div><div class="article-title">Microarray data normalization and transformation</div>, <div class="source">Nature Genet</div> , <div class="year">2002</div>, vol. <div class="volume">32</div> <div class="supplement">Suppl</div>(pg. <div class="fpage">496</div>-<div class="lpage">501</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20data%20normalization%20and%20transformation&amp;author=J%20Quackenbush&amp;publication_year=2002&amp;journal=Nature%20Genet&amp;volume=32&amp;pages=496-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12454644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirai</div> <div class="given-names">M. Y.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fujikawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goodenowe</div> <div class="given-names">D. B.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kitayama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Elucidation of gene-to-gene and metabolite-to-gene networks in <em>Arabidopsis</em> by integration of metabolomics and transcriptomics</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2005</div>, vol. <div class="volume">280</div> (pg. <div class="fpage">25590</div>-<div class="lpage">25595</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elucidation%20of%20gene-to-gene%20and%20metabolite-to-gene%20networks%20in%20Arabidopsis%20by%20integration%20of%20metabolomics%20and%20transcriptomics&amp;author=M.%20Y.%20Hirai&amp;author=M.%20Klein&amp;author=Y.%20Fujikawa&amp;author=M.%20Yano&amp;author=D.%20B.%20Goodenowe&amp;author=Y.%20Yamazaki&amp;author=S.%20Kanaya&amp;author=Y.%20Nakamura&amp;author=M.%20Kitayama&amp;author=H.%20Suzuki&amp;publication_year=2005&amp;journal=J.%20Biol.%20Chem.&amp;volume=280&amp;pages=25590-25595" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M502332200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M502332200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M502332200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15866872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itaya</div> <div class="given-names">M</div></div>. </div><div class="article-title">Construction of a novel tetracycline resistance gene cassette useful as a marker on the <em>Bacillus subtilis</em> chromosome</div>, <div class="source">Biosci. Biotech. Biochem.</div> , <div class="year">1992</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">685</div>-<div class="lpage">686</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20novel%20tetracycline%20resistance%20gene%20cassette%20useful%20as%20a%20marker%20on%20the%20Bacillus%20subtilis%20chromosome&amp;author=M%20Itaya&amp;publication_year=1992&amp;journal=Biosci.%20Biotech.%20Biochem.&amp;volume=56&amp;pages=685-686" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.56.685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.56.685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.56.685"> </span></div> </div></div></div></div></div><div content-id="DSN002C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fabret</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ehrlich</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Noirot</div> <div class="given-names">P. A</div></div>. </div><div class="article-title">A new mutation delivery system for genome-scale approaches in <em>Bacillus subtilis</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">25</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20mutation%20delivery%20system%20for%20genome-scale%20approaches%20in%20Bacillus%20subtilis&amp;author=C.%20Fabret&amp;author=S.%20D.%20Ehrlich&amp;author=P.%20A%20Noirot&amp;publication_year=2002&amp;journal=Mol.%20Microbiol.&amp;volume=46&amp;pages=25-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2002.03140.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2002.03140.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2002.03140.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12366828" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hodge</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Hofreiter</div> <div class="given-names">B. T</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Whistler</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Wolfrom</div> <div class="given-names">M. L</div></div>. </div><div class="article-title">Analysis and preparation of sugars, In:</div>, <div class="source">Methods in Carbohydrate Chemistry</div> , <div class="year">1962</div><div class="publisher-loc">New York, San Francisco, London</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">388</div>-<div class="lpage">389</div>)</div></div></div></div><div content-id="DSN002C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Piggot</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Hilbert</div> <div class="given-names">D. W</div></div>. </div><div class="article-title">Sporulation of <em>Bacillus subtilis</em></div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">579</div>-<div class="lpage">586</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sporulation%20of%20Bacillus%20subtilis&amp;author=P.%20J.%20Piggot&amp;author=D.%20W%20Hilbert&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=7&amp;pages=579-586" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2004.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2004.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2004.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15556029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T</div></div>. </div><div class="article-title">DNA microarray analysis of <em>Bacillus subtilis</em> DegU, ComA and PhoP regulons: an approach to comprehensive analysis of <em>B. subtilis</em> two-component regulatory systems</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">3804</div>-<div class="lpage">3813</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20microarray%20analysis%20of%20Bacillus%20subtilis%20DegU%2C%20ComA%20and%20PhoP%20regulons%3A%20an%20approach%20to%20comprehensive%20analysis%20of%20B.%20subtilis%20two-component%20regulatory%20systems&amp;author=M.%20Ogura&amp;author=H.%20Yamaguchi&amp;author=K.%20Yoshida&amp;author=Y.%20Fujita&amp;author=T%20Tanaka&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=3804-3813" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.18.3804" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.18.3804" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.18.3804"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11557812" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamoen</div> <div class="given-names">L. W.</div></div>,  <div class="name"><div class="surname">Van Werkhoven</div> <div class="given-names">A. F.</div></div>,  <div class="name"><div class="surname">Venema</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Dubnau</div> <div class="given-names">D</div></div>. </div><div class="article-title">The pleiotropic response regulator DegU functions as a priming protein in competence development in <em>Bacillus subtilis</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">9246</div>-<div class="lpage">9251</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20pleiotropic%20response%20regulator%20DegU%20functions%20as%20a%20priming%20protein%20in%20competence%20development%20in%20Bacillus%20subtilis&amp;author=L.%20W.%20Hamoen&amp;author=A.%20F.%20Van%20Werkhoven&amp;author=G.%20Venema&amp;author=D%20Dubnau&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=9246-9251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.160010597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.160010597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.160010597"> </span></div> </div></div></div></div></div><div content-id="DSN002C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olmos</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">de Anda</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ferrari</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bolivar</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Valle</div> <div class="given-names">F</div></div>. </div><div class="article-title">Effects of the <em>sinR</em> and <em>degU32</em> (Hy) mutations on the regulation of the <em>aprE</em> gene in <em>Bacillus subtilis</em></div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">253</div> (pg. <div class="fpage">562</div>-<div class="lpage">567</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effects%20of%20the%20sinR%20and%20degU32%20(Hy)%20mutations%20on%20the%20regulation%20of%20the%20aprE%20gene%20in%20Bacillus%20subtilis&amp;author=J.%20Olmos&amp;author=R.%20de%20Anda&amp;author=E.%20Ferrari&amp;author=F.%20Bolivar&amp;author=F%20Valle&amp;publication_year=1997&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=253&amp;pages=562-567" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004380050358" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004380050358" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004380050358"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9065689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mirel</div> <div class="given-names">D. B.</div></div>,  <div class="name"><div class="surname">Estacio</div> <div class="given-names">W. F.</div></div>,  <div class="name"><div class="surname">Mathieu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Olmsted</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ramirez</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Marquez-Magana</div> <div class="given-names">L. M</div></div>. </div><div class="article-title">Environmental regulation of <em>Bacillus subtilis</em> sigma(D)-dependent gene expression</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">3055</div>-<div class="lpage">3062</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20regulation%20of%20Bacillus%20subtilis%20sigma(D)-dependent%20gene%20expression&amp;author=D.%20B.%20Mirel&amp;author=W.%20F.%20Estacio&amp;author=M.%20Mathieu&amp;author=E.%20Olmsted&amp;author=J.%20Ramirez&amp;author=L.%20M%20Marquez-Magana&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=3055-3062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.11.3055-3062.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.11.3055-3062.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.11.3055-3062.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10809682" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Serizawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sekiguchi</div> <div class="given-names">J</div></div>. </div><div class="article-title">Systematic analysis of SigD-regulated genes in <em>Bacillus subtilis</em> by DNA microarray and northern blotting analyses</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">329</div> (pg. <div class="fpage">125</div>-<div class="lpage">136</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20analysis%20of%20SigD-regulated%20genes%20in%20Bacillus%20subtilis%20by%20DNA%20microarray%20and%20northern%20blotting%20analyses&amp;author=M.%20Serizawa&amp;author=H.%20Yamamoto&amp;author=H.%20Yamaguchi&amp;author=Y.%20Fujita&amp;author=K.%20Kobayashi&amp;author=N.%20Ogasawara&amp;author=J%20Sekiguchi&amp;publication_year=2004&amp;journal=Gene&amp;volume=329&amp;pages=125-136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2003.12.024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2003.12.024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2003.12.024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15033535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN002C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN002C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Antelmann</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Scharf</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hecker</div> <div class="given-names">M</div></div>. </div><div class="article-title">Phosphate starvation-inducible proteins of <em>Bacillus subtilis</em>: proteomics and transcriptional analysis</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">4478</div>-<div class="lpage">4490</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phosphate%20starvation-inducible%20proteins%20of%20Bacillus%20subtilis%3A%20proteomics%20and%20transcriptional%20analysis&amp;author=H.%20Antelmann&amp;author=C.%20Scharf&amp;author=M%20Hecker&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=4478-4490" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.16.4478-4490.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.16.4478-4490.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.16.4478-4490.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10913081" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=376500&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsn002/2/dsn002-dsn002supp.pdf?Expires=2147483647&amp;Signature=O-9-iP90WfTw9b3~~ciBDUWbt-f5G2mzRlSPSISa3K~aT0JNPZV5mT1TGLVGf59~HBD3knVoQqg2cBUP4Ula4GOTOzIuGZOld~R3Vix8MLXTN58JJV8KC7zX43SHm3aL1kyAkaDv-IOn0REGoBJy4wX9~Ylk0S2qUrSss7V8E4bZvj-cqNth9sjXIk4ibkOj5L3kDlFJveFsF8nYNgQsS-7-gwL1ZmtVenpqAtfVQL2P9qZc35oUBpQCZAS07esSwTtQHmbUJ6kNA-VVhhiBYIlTI4BArBTE4BBuIHsKwKGYtFDi7x2b~egoS1mSYT48OeMLyicXmYKLcHH0MrWFpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">556</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">91</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="376500"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f2%2f73%2f376500">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000256489900002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (91)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F2%2F73%2F376500" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/376500'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Enhanced Recombinant Protein Productivity by Genome Reduction in Bacillus subtilis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f73%2f376500">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

