

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/69/441516","@type":"ScholarlyArticle","name":"MetaVelvet-SL: an extension of the Velvet assembler to a  de novo  metagenomic assembler utilizing supervised learning","datePublished":"2014-11-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu041","keywords":["metagenomic","<italic>de novo</italic> assembler","short read","supervised learning","microbial community"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/69/441516","author":[{"name":"Afiahayati, ","affiliation":"Department of Biosciences and Informatics ,  Keio University ,  3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522 ,  Japan","@type":"Person"},{"name":"Sato, Kengo","affiliation":"Department of Biosciences and Informatics ,  Keio University ,  3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522 ,  Japan","@type":"Person"},{"name":"Sakakibara, Yasubumi","affiliation":"Department of Biosciences and Informatics ,  Keio University ,  3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522 ,  Japan","@type":"Person"}],"description":"Abstract.  The assembly of multiple genomes from mixed sequence reads is a bottleneck in metagenomic analysis. A single-genome assembly program (assembler) is n","pageStart":"69","pageEnd":"77","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"MetaVelvet-SL: an extension of the Velvet assembler to a  de novo  metagenomic assembler utilizing supervised learning","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="MetaVelvet-SL: an extension of the Velvet assembler to a  de novo  metagenomic assembler utilizing supervised learning" />
    <meta property="og:description" content="Abstract.  The assembly of multiple genomes from mixed sequence reads is a bottleneck in metagenomic analysis. A single-genome assembly program (assembler) is n" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/69/441516" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-11-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Afiahayati" /><meta name="citation_author_institution" content="Department of Biosciences and Informatics, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522, Japan" /><meta name="citation_author" content="Sato, Kengo" /><meta name="citation_author_institution" content="Department of Biosciences and Informatics, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522, Japan" /><meta name="citation_author" content="Sakakibara, Yasubumi" /><meta name="citation_author_institution" content="Department of Biosciences and Informatics, Keio University, 3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522, Japan" /><meta name="citation_title" content="MetaVelvet-SL: an extension of the Velvet assembler to a de novo metagenomic assembler utilizing supervised learning" /><meta name="citation_firstpage" content="69" /><meta name="citation_lastpage" content="77" /><meta name="citation_doi" content="10.1093/dnares/dsu041" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Next generation sequencing and bioinformatic bottlenecks: the current state of metagenomic data analysis; Curr. Opin. Biotechnol.;  citation_year=2012;  citation_volume=23;  citation_pages=9-15; " /><meta name="citation_reference" content="citation_title=Metavelvet: an extension of velvet assembler to de novo metagenome assembly from short sequence reads; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e155" /><meta name="citation_reference" content="citation_title=Bioinformatics for whole-genome shotgun sequencing of microbial communities; PLoS Comput. Biol.;  citation_year=2005;  citation_volume=1;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=A de novo metagenomic assembly program for shotgun dna reads; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1455-62; " /><meta name="citation_reference" content="citation_title=Genovo: de novo assembly for metagenomes; J. Comput. Biol.;  citation_year=2011;  citation_volume=18;  citation_pages=429-43; " /><meta name="citation_reference" content="citation_title=Sequence assembly demystified; Nat. Rev. Genet.;  citation_year=2013;  citation_volume=14;  citation_pages=157-67; " /><meta name="citation_reference" content="citation_title=Meta-IDBA: a de novo assembler for metagenomic data; Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=i94-101; " /><meta name="citation_reference" content="citation_title=An extended genovo metagenomic assembler by incorporating paired-end information; PeerJ;  citation_year=2013;  citation_volume=1;  citation_pages=e196" /><meta name="citation_reference" content="citation_title=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1420-8; " /><meta name="citation_reference" content="citation_title=Ray Meta: scalable de novo metagenome assembly and profiling; Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=r22" /><meta name="citation_reference" content="citation_title=Omega: an overlap-graph de novo assembler for metagenomics; Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2717-2722; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Pebble and Rock band: heuristic resolution of repeats and scaffolding in the velvet short-read de novo assembler; PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e8407" /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler; GigaScience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=A whole-genome assembly of Drosophila; Science;  citation_year=2009;  citation_volume=287;  citation_pages=2196-204; " /><meta name="citation_reference" content="citation_title=Metagenomic microbial community profiling using unique clade-specific marker genes; Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=811-4; " /><meta name="citation_reference" content="citation_title=A human gut microbial gene catalogue established by metagenomic sequencing; Nature;  citation_year=2010;  citation_volume=464;  citation_pages=59-65; " /><meta name="citation_reference" content="citation_title=Structure, function and diversity of the healthy human microbiome; citation_author=Consortium THMP; Nature;  citation_year=2012;  citation_volume=486;  citation_pages=207-14; " /><meta name="citation_reference" content="citation_title=LIBSVM: a library for support vector machines; ACM Trans. Intell. Syst. Technol.;  citation_year=2011;  citation_volume=2;  citation_pages=1-27; " /><meta name="citation_reference" content="citation_title=Species abundance distributions and richness estimations in fungal metagenomics lessons learned from community ecology; Mol. Ecol.;  citation_year=2011;  citation_volume=20;  citation_pages=275-85; " /><meta name="citation_reference" content="citation_title=The balanced accuracy and its posterior distribution; citation_text=IEEE, Washington, DC, USA;  citation_year=2010;  citation_pages=3121-4; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/69/1210427/dsu041.pdf" /><meta name="description" content="Abstract.  The assembly of multiple genomes from mixed sequence reads is a bottleneck in metagenomic analysis. A single-genome assembly program (assembler) is n" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/69/441516" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/69/441516" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/69/441516");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu041';
    </script>
<title>
	MetaVelvet-SL: an extension of the Velvet assembler to a de novo metagenomic assembler utilizing supervised learning | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./441516" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BegwPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBZmlhaGF5YXRpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlcyBhbmQgSW5mb3JtYXRpY3MsIEtlaW8gVW5pdmVyc2l0eSwgMy0xNC0xIEhpeW9zaGksIEtvaG9rdS1rdSwgWW9rb2hhbWEgMjIzLTg1MjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgS2VuZ28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2VzIGFuZCBJbmZvcm1hdGljcywgS2VpbyBVbml2ZXJzaXR5LCAzLTE0LTEgSGl5b3NoaSwgS29ob2t1LWt1LCBZb2tvaGFtYSAyMjMtODUyMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYWtha2liYXJhLCBZYXN1YnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZXMgYW5kIEluZm9ybWF0aWNzLCBLZWlvIFVuaXZlcnNpdHksIDMtMTQtMSBIaXlvc2hpLCBLb2hva3Uta3UsIFlva29oYW1hIDIyMy04NTIyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTWV0YVZlbHZldC1TTDogYW4gZXh0ZW5zaW9uIG9mIHRoZSBWZWx2ZXQgYXNzZW1ibGVyIHRvIGEgZGUgbm92byBtZXRhZ2Vub21pYyBhc3NlbWJsZXIgdXRpbGl6aW5nIHN1cGVydmlzZWQgbGVhcm5pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBiaW9pbmZvcm1hdGljIGJvdHRsZW5lY2tzOiB0aGUgY3VycmVudCBzdGF0ZSBvZiBtZXRhZ2Vub21pYyBkYXRhIGFuYWx5c2lzOyBDdXJyLiBPcGluLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTktMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGF2ZWx2ZXQ6IGFuIGV4dGVuc2lvbiBvZiB2ZWx2ZXQgYXNzZW1ibGVyIHRvIGRlIG5vdm8gbWV0YWdlbm9tZSBhc3NlbWJseSBmcm9tIHNob3J0IHNlcXVlbmNlIHJlYWRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1lMTU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvaW5mb3JtYXRpY3MgZm9yIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmcgb2YgbWljcm9iaWFsIGNvbW11bml0aWVzOyBQTG9TIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPWUyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGUgbm92byBtZXRhZ2Vub21pYyBhc3NlbWJseSBwcm9ncmFtIGZvciBzaG90Z3VuIGRuYSByZWFkczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xNDU1LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdm86IGRlIG5vdm8gYXNzZW1ibHkgZm9yIG1ldGFnZW5vbWVzOyBKLiBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDI5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhc3NlbWJseSBkZW15c3RpZmllZDsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YS1JREJBOiBhIGRlIG5vdm8gYXNzZW1ibGVyIGZvciBtZXRhZ2Vub21pYyBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPWk5NC0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV4dGVuZGVkIGdlbm92byBtZXRhZ2Vub21pYyBhc3NlbWJsZXIgYnkgaW5jb3Jwb3JhdGluZyBwYWlyZWQtZW5kIGluZm9ybWF0aW9uOyBQZWVySjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9ZTE5NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlEQkEtVUQ6IGEgZGUgbm92byBhc3NlbWJsZXIgZm9yIHNpbmdsZS1jZWxsIGFuZCBtZXRhZ2Vub21pYyBzZXF1ZW5jaW5nIGRhdGEgd2l0aCBoaWdobHkgdW5ldmVuIGRlcHRoOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE0MjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF5IE1ldGE6IHNjYWxhYmxlIGRlIG5vdm8gbWV0YWdlbm9tZSBhc3NlbWJseSBhbmQgcHJvZmlsaW5nOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz1yMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbWVnYTogYW4gb3ZlcmxhcC1ncmFwaCBkZSBub3ZvIGFzc2VtYmxlciBmb3IgbWV0YWdlbm9taWNzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MTctMjcyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVsdmV0OiBhbGdvcml0aG1zIGZvciBkZSBub3ZvIHNob3J0IHJlYWQgYXNzZW1ibHkgdXNpbmcgZGUgYnJ1aWpuIGdyYXBoczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGViYmxlIGFuZCBSb2NrIGJhbmQ6IGhldXJpc3RpYyByZXNvbHV0aW9uIG9mIHJlcGVhdHMgYW5kIHNjYWZmb2xkaW5nIGluIHRoZSB2ZWx2ZXQgc2hvcnQtcmVhZCBkZSBub3ZvIGFzc2VtYmxlcjsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU4NDA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUGRlbm92bzI6IGFuIGVtcGlyaWNhbGx5IGltcHJvdmVkIG1lbW9yeS1lZmZpY2llbnQgc2hvcnQtcmVhZCBkZSBub3ZvIGFzc2VtYmxlcjsgR2lnYVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB3aG9sZS1nZW5vbWUgYXNzZW1ibHkgb2YgRHJvc29waGlsYTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yODc7ICBjaXRhdGlvbl9wYWdlcz0yMTk2LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWMgbWljcm9iaWFsIGNvbW11bml0eSBwcm9maWxpbmcgdXNpbmcgdW5pcXVlIGNsYWRlLXNwZWNpZmljIG1hcmtlciBnZW5lczsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz04MTEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBodW1hbiBndXQgbWljcm9iaWFsIGdlbmUgY2F0YWxvZ3VlIGVzdGFibGlzaGVkIGJ5IG1ldGFnZW5vbWljIHNlcXVlbmNpbmc7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz01OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLCBmdW5jdGlvbiBhbmQgZGl2ZXJzaXR5IG9mIHRoZSBoZWFsdGh5IGh1bWFuIG1pY3JvYmlvbWU7IGNpdGF0aW9uX2F1dGhvcj1Db25zb3J0aXVtIFRITVA7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODY7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxJQlNWTTogYSBsaWJyYXJ5IGZvciBzdXBwb3J0IHZlY3RvciBtYWNoaW5lczsgQUNNIFRyYW5zLiBJbnRlbGwuIFN5c3QuIFRlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWVzIGFidW5kYW5jZSBkaXN0cmlidXRpb25zIGFuZCByaWNobmVzcyBlc3RpbWF0aW9ucyBpbiBmdW5nYWwgbWV0YWdlbm9taWNzIGxlc3NvbnMgbGVhcm5lZCBmcm9tIGNvbW11bml0eSBlY29sb2d5OyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9Mjc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFsYW5jZWQgYWNjdXJhY3kgYW5kIGl0cyBwb3N0ZXJpb3IgZGlzdHJpYnV0aW9uOyBjaXRhdGlvbl90ZXh0PUlFRUUsIFdhc2hpbmd0b24sIERDLCBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl9wYWdlcz0zMTIxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8xLzY5LzEyMTA0MjcvZHN1MDQxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgYXNzZW1ibHkgb2YgbXVsdGlwbGUgZ2Vub21lcyBmcm9tIG1peGVkIHNlcXVlbmNlIHJlYWRzIGlzIGEgYm90dGxlbmVjayBpbiBtZXRhZ2Vub21pYyBhbmFseXNpcy4gQSBzaW5nbGUtZ2Vub21lIGFzc2VtYmx5IHByb2dyYW0gKGFzc2VtYmxlcikgaXMgbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQxNTE2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJYRE17HdGAmNDGyfBc3MYTa6Wec" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f69%2f441516" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f69%2f441516" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6594366">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594366">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594368">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594368">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594379">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594379">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594392">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594392">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594430">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594430">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594432">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594432">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594434">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594434">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6594436">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6594436">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/53/441064">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/79/440782">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MetaVelvet-SL: an extension of the Velvet assembler to a <em>de novo</em> metagenomic assembler utilizing supervised learning
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Afiahayati</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biosciences and Informatics</div>, <div class="insititution">Keio University</div>, <div class="addr-line">3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Afiahayati">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Afiahayati ">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Afiahayati &quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Afiahayati</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kengo Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biosciences and Informatics</div>, <div class="insititution">Keio University</div>, <div class="addr-line">3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kengo+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kengo Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasubumi Sakakibara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biosciences and Informatics</div>, <div class="insititution">Keio University</div>, <div class="addr-line">3-14-1 Hiyoshi, Kohoku-ku, Yokohama 223-8522</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 45-566-1791. Fax. +81 45-566-1791. E-mail: <a href="mailto:yasu@bio.keio.ac.jp" target="_blank">yasu@bio.keio.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasubumi+Sakakibara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakakibara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakakibara Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasubumi Sakakibara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 69–77, <a href='https://doi.org/10.1093/dnares/dsu041'>https://doi.org/10.1093/dnares/dsu041</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 August 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="441516" href="/dnaresearch/article-pdf/22/1/69/1210427/dsu041.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Afiahayati, Kengo Sato, Yasubumi Sakakibara;  MetaVelvet-SL: an extension of the Velvet assembler to a <em>de novo</em> metagenomic assembler utilizing supervised learning, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 69–77, <a href='https://doi.org/10.1093/dnares/dsu041'>https://doi.org/10.1093/dnares/dsu041</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/441516?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/441516?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=MetaVelvet-SL%3A%20an%20extension%20of%20the%20Velvet%20assembler%20to%20a%20%26lt%3Bem%26gt%3Bde%20novo%26lt%3B%2Fem%26gt%3B%20metagenomic%20assembler%20utilizing%20supervised%20learning&amp;publicationDate=2014-11-27&amp;volumeNum=22&amp;issueNum=1&amp;author=Sato%2C%20Kengo&amp;startPage=69&amp;endPage=77&amp;contentId=10.1093%2Fdnares%2Fdsu041&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f69%2f441516" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6594366 id="6594366" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The assembly of multiple genomes from mixed sequence reads is a bottleneck in metagenomic analysis. A single-genome assembly program (assembler) is not capable of resolving metagenome sequences, so assemblers designed specifically for metagenomics have been developed. MetaVelvet is an extension of the single-genome assembler Velvet. It has been proved to generate assemblies with higher N50 scores and higher quality than single-genome assemblers such as Velvet and SOAPdenovo when applied to metagenomic sequence reads and is frequently used in this research community. One important open problem for MetaVelvet is its low accuracy and sensitivity in detecting chimeric nodes in the assembly (de Bruijn) graph, which prevents the generation of longer contigs and scaffolds. We have tackled this problem of classifying chimeric nodes using supervised machine learning to significantly improve the performance of MetaVelvet and developed a new tool, called MetaVelvet-SL. A Support Vector Machine is used for learning the classification model based on 94 features extracted from candidate nodes. In extensive experiments, MetaVelvet-SL outperformed the original MetaVelvet and other state-of-the-art metagenomic assemblers, IDBA-UD, Ray Meta and Omega, to reconstruct accurate longer assemblies with higher N50 scores for both simulated data sets and real data sets of human gut microbial sequences.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenomic">metagenomic</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo assembler&quot;"><em>de novo</em> assembler</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;short read&quot;">short read</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;supervised learning&quot;">supervised learning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microbial community&quot;">microbial community</a></div>                            <h2 scrollto-destination=6594368 id="6594368" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Metagenomic research studies genetic material recovered directly from environmental samples. Next-generation sequencing (NGS) technologies have enabled an explosion in sequencing with increased throughput and decreased cost,<sup><a reveal-id="DSU041C1" data-open="DSU041C1" class="link link-ref link-reveal xref-bibr">1</a></sup> which provides opportunities to generate sequence reads from metagenomes effectively covering highly diverse microbial populations, even for genomes with low coverage. An important step in metagenomic analysis is the assembly of multiple genomes from mixed sequence reads of the multiple species that exist in the sample.<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This can present problems, because, in a microbial community, the number of genomes and the coverage of each genome are initially unknown and the coverage distribution is inhomogeneous and potentially skewed.<sup><a reveal-id="DSU041C1  DSU041C8" data-open="DSU041C1  DSU041C8" class="link link-ref link-reveal xref-bibr">1–8</a></sup> Another major difficulty is the short length of sequence reads from next-generation sequencers.<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Currently, there are several <em>de novo</em> assemblers that attempt to analyse metagenomic data. MAP,<sup><a reveal-id="DSU041C4" data-open="DSU041C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Genovo<sup><a reveal-id="DSU041C5" data-open="DSU041C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and Xgenovo<sup><a reveal-id="DSU041C8" data-open="DSU041C8" class="link link-ref link-reveal xref-bibr">8</a></sup> are used for rather long sequence reads, while MetaVelvet,<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Meta-IDBA,<sup><a reveal-id="DSU041C7" data-open="DSU041C7" class="link link-ref link-reveal xref-bibr">7</a></sup> IDBA-UD,<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Ray Meta<sup><a reveal-id="DSU041C10" data-open="DSU041C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and Omega<sup><a reveal-id="DSU041C11" data-open="DSU041C11" class="link link-ref link-reveal xref-bibr">11</a></sup> are used for short sequence reads. MAP was designed for the sequence reads produced by Sanger (700–1,000 bp) and 454 sequencing technology (200–500 bp). It uses an improved Overlap-Layout-Consensus (OLC) strategy integrating mate pair information.<sup><a reveal-id="DSU041C4" data-open="DSU041C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Genovo was designed for 454 sequencing data: it is a metagenomic assembler employing a generative probabilistic model.<sup><a reveal-id="DSU041C5" data-open="DSU041C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Xgenovo is an extension of Genovo incorporating paired-end information.<sup><a reveal-id="DSU041C8" data-open="DSU041C8" class="link link-ref link-reveal xref-bibr">8</a></sup> MetaVelvet, Meta-IDBA and IDBA-UD use the de Bruijn graph approach. These assemblers are specifically designed for the huge numbers of short reads generated by Illumina-type next-generation sequencers that enable deep sequencing of the inhomogeneous and divergent species in a microbial community. IDBA-UD is an extension of Meta-IDBA dealing with the uneven sequencing depths of different regions of genomes from different species.<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Both MetaVelvet<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and IDBA-UD<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> have been shown to produce longer high-quality assemblies than single-genome assemblers, such as Velvet<sup><a reveal-id="DSU041C12" data-open="DSU041C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU041C13" data-open="DSU041C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and SOAPdenovo2.<sup><a reveal-id="DSU041C14" data-open="DSU041C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Ray Meta is an extension of the Ray assembler for <em>de novo</em> metagenome assembly, which is scalable because it is highly distributed computing.<sup><a reveal-id="DSU041C10" data-open="DSU041C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Omega is a metagenomic assembler using overlap graph approach. Omega was most recently proposed for rather longer Illumina sequencing data of microbial communities.<sup><a reveal-id="DSU041C11" data-open="DSU041C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>MetaVelvet<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> is an extension of a single-genome assembly program (assembler), named Velvet.<sup><a reveal-id="DSU041C12" data-open="DSU041C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU041C13" data-open="DSU041C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The fundamental concept used in MetaVelvet is that a de Bruijn graph constructed from mixed sequence reads of multiple species is considered to be equivalent to the union of multiple de Bruijn sub-graphs, each of which is constructed from the sequence reads of individual species. The strategy of MetaVelvet is, first, to decompose a de Bruijn graph constructed from mixed short reads into individual sub-graphs and, second, to assemble scaffolds from each decomposed de Bruijn sub-graph to build an isolated genome.</p><p>For the graph disconnection task, MetaVelvet identifies nodes shared between two sub-graphs (named chimeric nodes) and disconnects two sub-graphs by splitting the shared nodes, as illustrated in Fig. <a reveal-id="DSU041F1" data-open="DSU041F1" class="link link-reveal link-table xref-fig">1</a>. Chimeric nodes are shared between the genomes of two closely related species and could represent orthologous sequences, conserved sequences (e.g. rRNA sequences) or horizontal transfer sequences. </p>                        <a id="6594373" scrollto-destination="6594373"></a>
<div id="DSU041F1" data-id="DSU041F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04101.png?Expires=2147483647&Signature=IFH23VbaDKrxI5B9SqYnZFnbiQEsOmzmDFoJwOEw-R4aCM-ORRyO5oX2UXyJ51Ye30FhNjSa~xOX6MxkmiOyT9dnxtHE8ZQbRtJkhzewptIyNhPqCvtUMubMe-KcOkMdMzY7S5Y4BWDYbOMUZy5YD6Q5mH~Kus9TT7v6gl3Z-nR1VIzF2s95qJQgcnwuAALO7wffYZLGkn7QmWOiIFqUGzHP44~c79YVOgcsScT9aTNZLIJ3eoI2Y94pwbU8gsqUs~2J59SnZPeojIxRh0Us7dW7clIUn4BtbW8PBdzb4-xC6Ek~YPEpKLDKP30bMk6TUUCw-2HOj5MqehNhoLRxMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chimeric nodes need to be split to obtain independent sub-graphs in a metagenomic assembly." path-from-xml="dsu04101.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594373/dsu04101.tif" path-from-xml="dsu04101.tif" target="_blank">View large</a><a section="6594373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04101.png?Expires=2147483647&Signature=XIsU7qdxM685v65LObWGXmW7qTYrZjaqKxficbKJ177U7mdn7tqjtrtqqu2kTkHe6joI4IqYwe7J4Y5AeUITnk2dSdmOaJl5vmIBYDWF2PlgM~k1aUM2EvLQsqo4N2PRKLPeU10ndcurour0OKWvd8r9A6UNis7QyMZfzT-nfB-CFgJIEIlqBxCL9z5h0ZhVWy~8BKP0LO5URWMJYSCYQ5El3Q45WktpigHyIRgu5cjw2Z8SxX57VGF47wmtR75jj4az67Aoyj-Tm0sQ3hzp86B0o73y4XFdKxG26C7IGCJuME01qIogsMON3X-2WKP7r0fBUSxW9zlUh-GNfwB2NA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594373&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04101.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chimeric nodes need to be split to obtain independent sub-graphs in a metagenomic assembly.</p></div></div><div content-id="DSU041F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04101.png?Expires=2147483647&Signature=IFH23VbaDKrxI5B9SqYnZFnbiQEsOmzmDFoJwOEw-R4aCM-ORRyO5oX2UXyJ51Ye30FhNjSa~xOX6MxkmiOyT9dnxtHE8ZQbRtJkhzewptIyNhPqCvtUMubMe-KcOkMdMzY7S5Y4BWDYbOMUZy5YD6Q5mH~Kus9TT7v6gl3Z-nR1VIzF2s95qJQgcnwuAALO7wffYZLGkn7QmWOiIFqUGzHP44~c79YVOgcsScT9aTNZLIJ3eoI2Y94pwbU8gsqUs~2J59SnZPeojIxRh0Us7dW7clIUn4BtbW8PBdzb4-xC6Ek~YPEpKLDKP30bMk6TUUCw-2HOj5MqehNhoLRxMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chimeric nodes need to be split to obtain independent sub-graphs in a metagenomic assembly." path-from-xml="dsu04101.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594373/dsu04101.tif" path-from-xml="dsu04101.tif" target="_blank">View large</a><a section="6594373" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04101.png?Expires=2147483647&Signature=XIsU7qdxM685v65LObWGXmW7qTYrZjaqKxficbKJ177U7mdn7tqjtrtqqu2kTkHe6joI4IqYwe7J4Y5AeUITnk2dSdmOaJl5vmIBYDWF2PlgM~k1aUM2EvLQsqo4N2PRKLPeU10ndcurour0OKWvd8r9A6UNis7QyMZfzT-nfB-CFgJIEIlqBxCL9z5h0ZhVWy~8BKP0LO5URWMJYSCYQ5El3Q45WktpigHyIRgu5cjw2Z8SxX57VGF47wmtR75jj4az67Aoyj-Tm0sQ3hzp86B0o73y4XFdKxG26C7IGCJuME01qIogsMON3X-2WKP7r0fBUSxW9zlUh-GNfwB2NA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594373&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04101.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chimeric nodes need to be split to obtain independent sub-graphs in a metagenomic assembly.</p></div></div><p>To identify chimeric nodes, MetaVelvet uses simple heuristics based on coverage difference and paired-end information, which results in low accuracy and low sensitivity. Our primary goal in this study is to improve chimeric node detection and generate longer accurate scaffolds. Such scaffolds can help to extract more information from the reads, leading to the discovery of more genes and better functional annotation.<sup><a reveal-id="DSU041C15" data-open="DSU041C15" class="link link-ref link-reveal xref-bibr">15</a></sup> To do this, we have developed an assembler called MetaVelvet-SL, which classifies every node in a de Bruijn graph constructed from mixed short reads of multiple species into the following four types by employing supervised machine learning.
The first new procedure in MetaVelvet-SL is to develop the model to classify a node at a crossing point between two paths as chimeric or not. In the process of learning the model, 94 features are extracted for each chimeric node candidate, which is a node at a crossing point that has two incoming edges and two outgoing edges. A Support Vector Machine (SVM) is used for learning the classification model. If chimeric nodes can be identified correctly, it means that the de Bruijn graph can be disconnected appropriately by splitting the chimeric nodes. The second new procedure in MetaVelvet-SL is that the expected coverage to extract the unique nodes is calculated for each sub-graph. Based on the assumption that each sub-graph represents a single species, the expected coverage per sub-graph can precisely determine the unique nodes of each species, even those with low coverage. This expected coverage calculation per sub-graph replaces the original MetaVelvet strategy of detecting multiple peaks on the histograms of <em>k</em>-mer frequencies and defining each peak as expected coverage.</p><ol class="order"><li><p>Chimeric node: A node which is shared between the genomes of two closely related species. This node should be split. Chimeric nodes are illustrated in Fig. <a reveal-id="DSU041F1" data-open="DSU041F1" class="link link-reveal link-table xref-fig">1</a>.</p></li><li><p>Repeat node: A repeat node represents a sequence that occurs several times in the genome. Note that, in multiple genome assembly, nodes at a crossing point between two incoming and two outgoing edges are not necessarily repeats. Such nodes are sometimes chimeric nodes.</p></li><li><p>Unique node: A unique node is one which is neither chimeric nor a repeat. The challenge is to determine the unique nodes of each species correctly.</p></li><li><p>Low-coverage node: In metagenomic assembly, all nodes, even those with low coverage, must be examined to account for species with small populations, but low-coverage nodes must be distinguished from nodes generated by sequencing errors.</p></li></ol><p>MetaVelvet-SL consists of two main modules: first, the supervised learning module to develop a model for the classification of chimeric nodes and, second, the assembly module. (The technical details are described in the Materials and methods section.) MetaVelvet-SL also provides a couple of tools to allow users to generate their classification model using prior knowledge about the taxonomic profile of the target microbial community. The taxonomic profile can be inferred from sequence reads by using taxonomic profiling methods, such as MetaPhlAn.<sup><a reveal-id="DSU041C16" data-open="DSU041C16" class="link link-ref link-reveal xref-bibr">16</a></sup> We have developed a pipeline connecting MetaPhlAn and MetaVelvet-SL. The pipeline automatically generates a classification model only from metagenomic sequence read data. This customized classification model could be well suited to the assembly of the target metagenomes. MetaVelvet-SL also provides a library of pre-trained classification models for several typical environments, such as soil, deep sea, mud, human blood, intestine and mouth.</p><p>The source code of MetaVelvet-SL, the pipeline connecting MetaPhlAn and MetaVelvet-SL, and the library of classification models for several typical environments are freely available under the GNU General Public License at <a class="link link-uri" href="http://metavelvet.dna.bio.keio.ac.jp">http://metavelvet.dna.bio.keio.ac.jp</a>.</p><p>We conducted computational experiments to evaluate the performance of Metavelvet-SL. The assembly performance of Metavelvet-SL was compared with those of MetaVelvet, the state-of-the-art metagenomic assemblers IDBA-UD, Ray Meta and Omega, and a standard single-genome assembler for massive short sequencing reads, SOAPdenovo2. For simulated data sets, first, we conducted experiments using the pipeline connecting MetaPhlAn and MetaVelvet-SL. The taxonomic profile for the training data set was inferred by MetaPhlAn. Second, to measure the performance of MetaVelvet-SL for different degrees of similarity between the training data set and the assembly data set, we conducted experiments using three levels of training data sets from the highest to the lowest similarity (genus, family and order) to the assembly data sets. For all assembly data sets, MetaVelvet-SL with any training data set generated the highest accurate N50 scores and longest maximum length of accurate scaffolds among the assemblers. (The N50 score is a standard statistical measure that evaluates assembly quality. Scaffolds with higher N50 scores are especially beneficial for the identification of protein-coding genes.<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup>) On real data sets of human gut microbial short read data, sequenced as part of the MetaHIT project<sup><a reveal-id="DSU041C17" data-open="DSU041C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the Human Microbiome Project Consortium,<sup><a reveal-id="DSU041C18" data-open="DSU041C18" class="link link-ref link-reveal xref-bibr">18</a></sup> MetaVelvet-SL using models constructed by supervised learning from the taxonomy profile inferred by MetaPhlAn generated longer scaffolds.</p>                            <h2 scrollto-destination=6594379 id="6594379" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
<p>A de Bruijn graph is a data structure that compactly represents overlaps between short reads. Several <em>de novo</em> methods based on de Bruijn graphs have been proposed to assemble short reads generated from next-generation sequencers for single genomes and metagenomes.<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSU041C7" data-open="DSU041C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSU041C12" data-open="DSU041C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU041C13" data-open="DSU041C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In a de Bruijn graph, a <em>k</em>-mer (word of length <em>k</em>) is assigned to a node, so the size of a de Bruijn graph is independent of the size of the input of reads. The assembly (reconstruction) of the target genome from the de Bruijn graph can be reduced to finding an Eulerian path that is computable in polynomial time.</p><p>First, we briefly review the Velvet and MetaVelvet assemblers upon which our method is based. Then we describe MetaVelvet-SL, our extension of Velvet to metagenomic assembly, that utilizes supervised learning.</p>                            <h3 scrollto-destination=6594382 id="6594382" class="section-title" data-legacy-id=s2a>2.1. Brief outline of Velvet and MetaVelvet</h3>
<p>Velvet is slightly different from other de Bruijn-graph-based assemblers in that each node is attached to a twin node that represents a series of <em>k</em>-mers and their reverse complements for reads from both strands. For each input read, Velvet defines an ordered set of overlapping <em>k</em>-mers. The ordered set is cut whenever an overlap with another read begins or ends. For each uninterrupted ordered subset of the original <em>k</em>-mers, a node is created. Velvet has three functions to manipulate the de Bruijn graph: node merging for simplification, removing tips and removing bubbles for error reduction. Velvet has two functions, Pebble and Rock Band, for constructing the scaffold and for repeat resolution using paired-end and long-read information. In these functions, Velvet distinguishes the unique nodes from the repeat nodes based on the node coverage. A repeat node represents a sequence that occurs several times in the genome and can be described as a node at a crossing point between two paths with multiple incoming and outgoing edges. In multiple genome assembly, such nodes are not necessarily repeats since they can sometimes be shared between the genomes of two closely related species and represent orthologous sequences, conserved sequences (such as rRNA sequences) or horizontal transfer sequences.</p><p>MetaVelvet is an extension of Velvet for the assembly of metagenomes. The fundamental concept used in MetaVelvet is that a de Bruijn graph constructed from mixed sequence reads of multiple species is considered to be equivalent to the union of multiple de Bruijn sub-graphs, each of which is constructed from sequence reads of individual species. MetaVelvet has two functions. First, MetaVelvet decomposes a de Bruijn graph constructed from mixed short reads into individual sub-graphs. To do this, MetaVelvet calculates the histogram of <em>k</em>-mer frequencies and detects multiple peaks on the histogram, each peak of which would correspond to one species in a microbial community. Then, MetaVelvet classifies every node into one of the peaks to form sub-graphs composed of nodes belonging to the same peak. MetaVelvet identifies shared (chimeric) nodes between two sub-graphs and disconnects the sub-graphs by splitting the shared nodes, as illustrated in Fig. <a reveal-id="DSU041F1" data-open="DSU041F1" class="link link-reveal link-table xref-fig">1</a>. To distinguish chimeric nodes from repeat nodes, MetaVelvet uses coverage difference and paired-end information. The second function of MetaVelvet builds scaffolds using Velvet's Pebble and Rock Band functions by treating each decomposed de Bruijn sub-graph as an isolated species genome.</p>                            <h3 scrollto-destination=6594385 id="6594385" class="section-title" data-legacy-id=s2b>2.2. Extension to MetaVelvet-SL</h3>
<p>MetaVelvet-SL consists of three major procedures, as illustrated in Fig. <a reveal-id="DSU041F2" data-open="DSU041F2" class="link link-reveal link-table xref-fig">2</a>. </p><ol class="order"><li><p>Construction of a de Bruijn graph.</p><p>MetaVelvet-SL constructs a de Bruijn graph from mixed sequence reads of multiple species genomes using Velvet functions.</p></li><li><p>Learning and classification of chimeric nodes.</p><p>This procedure, first, extracts the chimeric node candidates from the main de Bruijn graph. A chimeric node candidate is defined as a node that has two incoming edges and two outgoing edges. MetaVelvet-SL utilizes LIBSVM<sup><a reveal-id="DSU041C19" data-open="DSU041C19" class="link link-ref link-reveal xref-bibr">19</a></sup> to develop a model for classification of chimeric nodes. We used the RBF kernel that is recommended by LIBSVM. Grid search was used to find optimal parameters for the RBF kernel. For the classification, 94 features are extracted for each chimeric node candidate. These features are (Fig. <a reveal-id="DSU041F3" data-open="DSU041F3" class="link link-reveal link-table xref-fig">3</a>): <ul class="alpha-lower"><li><p>The dinucleotide frequencies in the chimeric node candidate, the two incoming nodes and the two outgoing nodes. Each node has 16 dinucleotide frequencies (AA, AT, AC, AG, TA, … , GG). (16 features × 5 = 80 features).</p></li><li><p>The number of paired-end reads supporting the connection between the incoming node with higher coverage and the outgoing node with higher coverage (1 feature).</p></li><li><p>The number of paired-end reads supporting the connection between the incoming node with higher coverage and the outgoing node with lower coverage (1 feature).</p></li><li><p>The number of paired-end reads supporting the connection between the incoming node with lower coverage and the outgoing node with higher coverage (1 feature).</p></li><li><p>The number of paired-end reads supporting the connection between the incoming node with lower coverage and the outgoing node with lower coverage (1 feature).</p></li><li><p>The ratio between the coverage of each incoming node and the coverage of the chimeric node candidate (2 features).</p></li><li><p>The ratio between the coverage of each outgoing node and the coverage of the chimeric node candidate (2 features).</p></li><li><p>The coverage of the chimeric node candidate (1 feature).</p></li><li><p>The lengths of contigs attached to the chimeric node candidate, the two incoming nodes and the two outgoing nodes (5 features).</p></li></ul></p><p>There are three defined classes. Classes 1 and 2 are the positive classes, which contain chimeric nodes, while Class 3 is the negative class, containing non-chimeric nodes. Class 1 contains chimeric nodes in which the incoming node of higher coverage and the outgoing node of higher coverage come from a same species, and the incoming node of lower coverage and the outgoing node of lower coverage come from another species. Class 2 contains chimeric nodes in which the incoming node of higher coverage and the outgoing node of lower coverage come from a same species, and the incoming node of lower coverage and the outgoing node of higher coverage come from another species. Classes 1 and 2 are illustrated in Fig. <a reveal-id="DSU041F3" data-open="DSU041F3" class="link link-reveal link-table xref-fig">3</a>.</p><p>One additional task in MetaVelvet-SL is the preparation of the training sample that is required for learning the classification model. MetaVelvet-SL uses prior knowledge about the taxonomic profile (composition) of the target microbial community to generate the training sample. This taxonomic profile can be inferred from sequence reads by using taxonomic profiling methods, such as MetaPhlAn.<sup><a reveal-id="DSU041C16" data-open="DSU041C16" class="link link-ref link-reveal xref-bibr">16</a></sup> MetaVelvet-SL has the following functions to generate the training sample. First, by using the taxonomic profile, a set of reference genomes that belong to species that are the same as or are closely related to those in the taxonomy profile are collected from the public genome database. Second, the collected reference genome sequences are used to generate simulated sequence reads, and a de Bruijn graph is constructed from the simulated read data. Third, by aligning each node in the de Bruijn graph to the reference genome sequences, it can be determined to which species genome each node belongs. Fourth, each node at a crossing point between two paths in the de Bruijn graph is labelled as Class 1, 2 or 3, generating the training samples.</p></li><li><p>Final assembly tasks:</p><p>Metavelvet-SL has five major steps for this procedure. <ul class="alpha-lower"><li><p>Load the main de Bruijn graph which has been constructed.</p></li><li><p>Extract the chimeric node candidates and classify them based on the model that has been learned. After obtaining the classifications of chimeric node candidates, split the chimeric nodes that are classified as being Class 1 or 2.</p></li><li><p>Decompose the de Bruijn graph into connected sub-graphs.</p></li><li><p>Identify unique nodes. The expected coverage of each sub-graph is calculated to determine the unique nodes based on the formula used to identify a unique node in Velvet.<sup><a reveal-id="DSU041C12" data-open="DSU041C12" class="link link-ref link-reveal xref-bibr">12</a></sup> This formula is given in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>.</p></li><li><p>Perform the scaffolding procedure (Pebble and Rock Band procedures). The scaffolding procedure is iterated for each set of unique nodes from the lowest expected coverage to the highest expected coverage.</p></li></ul></p></li></ol>                        <a id="6594388" scrollto-destination="6594388"></a>
<div id="DSU041F2" data-id="DSU041F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04102.png?Expires=2147483647&Signature=w-NxgED-u-OtByNIClDeP0RSEXxxa86JJhlqOp7w1mD9P1xJ5ztry5XoG~iyc25DtVITAQBCrza1Kb0pq~Sz2ENQ6wm3apyXebq2jbA8eAR3IANBYRVoX6578vpS~NHHOI7o1enbsHzu8a4R3R7MOWa8OkdBizjnHhd9m9Zd16fvZTgl83giyvzo4s5-V-Zbn~996uocK0HSL194ZWyhZAIP1g1yIGsrXaib5bMEdkG~9F8uJLQap36ifCzOAvC7iKZjzEku5m62aj1hbNf2OqlDWZJcySz7A5VBa~sS6rWFKEMIkYmO-KfBxbLHbn0Gig0LtOHhnIHX4PayN1FCVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MetaVelvet-SL system consists of three major procedures: (i) construction of a de Bruijn graph; (ii) classification of chimeric nodes and (iii) final assembly tasks." path-from-xml="dsu04102.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594388/dsu04102.tif" path-from-xml="dsu04102.tif" target="_blank">View large</a><a section="6594388" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04102.png?Expires=2147483647&Signature=DoQ1ybtpcaQZs986juqqhKJACGXfjUCr~h0IbaI06-JuHIUl6-N~YSXTlULbjwQse4DjYWnA7Wh3ZXN8lGZjlCU~s57N6obd4m733rkotunhpugqGOrnBUcjDEOBo8-~md5BmdV~sd4Vp5f9hNMxuR70OTcMfAWX2JRnkVTyADd727q-OytVHF8VwogqAPIA4~j6zcmKmAMsS~~A9OS12tfXshVpmuEe9O4dPE7vHZh5ohNeuNiJsZbVfBQP9IP2zcBSg5I7Ib0J4BX7XSyA87baf3C13uxf4f96GQqEEsnzc2EvibjcPnHKObILcuJjBQJyfW5993z6~90W16J9rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594388&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04102.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MetaVelvet-SL system consists of three major procedures: (i) construction of a de Bruijn graph; (ii) classification of chimeric nodes and (iii) final assembly tasks.</p></div></div><div content-id="DSU041F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04102.png?Expires=2147483647&Signature=w-NxgED-u-OtByNIClDeP0RSEXxxa86JJhlqOp7w1mD9P1xJ5ztry5XoG~iyc25DtVITAQBCrza1Kb0pq~Sz2ENQ6wm3apyXebq2jbA8eAR3IANBYRVoX6578vpS~NHHOI7o1enbsHzu8a4R3R7MOWa8OkdBizjnHhd9m9Zd16fvZTgl83giyvzo4s5-V-Zbn~996uocK0HSL194ZWyhZAIP1g1yIGsrXaib5bMEdkG~9F8uJLQap36ifCzOAvC7iKZjzEku5m62aj1hbNf2OqlDWZJcySz7A5VBa~sS6rWFKEMIkYmO-KfBxbLHbn0Gig0LtOHhnIHX4PayN1FCVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MetaVelvet-SL system consists of three major procedures: (i) construction of a de Bruijn graph; (ii) classification of chimeric nodes and (iii) final assembly tasks." path-from-xml="dsu04102.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594388/dsu04102.tif" path-from-xml="dsu04102.tif" target="_blank">View large</a><a section="6594388" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04102.png?Expires=2147483647&Signature=DoQ1ybtpcaQZs986juqqhKJACGXfjUCr~h0IbaI06-JuHIUl6-N~YSXTlULbjwQse4DjYWnA7Wh3ZXN8lGZjlCU~s57N6obd4m733rkotunhpugqGOrnBUcjDEOBo8-~md5BmdV~sd4Vp5f9hNMxuR70OTcMfAWX2JRnkVTyADd727q-OytVHF8VwogqAPIA4~j6zcmKmAMsS~~A9OS12tfXshVpmuEe9O4dPE7vHZh5ohNeuNiJsZbVfBQP9IP2zcBSg5I7Ib0J4BX7XSyA87baf3C13uxf4f96GQqEEsnzc2EvibjcPnHKObILcuJjBQJyfW5993z6~90W16J9rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594388&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04102.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MetaVelvet-SL system consists of three major procedures: (i) construction of a de Bruijn graph; (ii) classification of chimeric nodes and (iii) final assembly tasks.</p></div></div>                        <a id="6594389" scrollto-destination="6594389"></a>
<div id="DSU041F3" data-id="DSU041F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04103.png?Expires=2147483647&Signature=gSeQvatL-qNJ-DJW7tPJoHqfx4C1cM2QrCUk70E6daENypHkQsaKImf-top0aAUciN9tdOtVlgRv3ZORbh9JeYLD3pqhNWg2NcZrmyzeUoZzwMgMIUphpW4k5jIeTZWHo1adYUs9YPb4y1j3gYOYbfSrhr~BnndySrWJFL~P~Uy48UpszoOlncm4vgNZI-8nzXRyaf6A0zUcllhcWYyYgI-2Ryz6vhuyouDrDvwTLRbWfNCZDLsZy-3XFSv8FNqfu5jPxhlc6ry-PXr5Ikp3A4Knq7zDnQGcwvp1CuS3sJVOeH8x7ga-tEYxp7pWcDXSpDxN2tOuoGkt4hly9ohp1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chimeric nodes fall into two classes. Nodes of the same colour represent the same species. The number in each node represents the coverage value of the node. A contig sequence is also attached to each node." path-from-xml="dsu04103.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594389/dsu04103.tif" path-from-xml="dsu04103.tif" target="_blank">View large</a><a section="6594389" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04103.png?Expires=2147483647&Signature=CliqYjriFUjGdi3ayfXcR6bAysaJ~1UuvESW6ygfzvSCVa8Ei~RT7oJsbKCyNhX5~I1JIpL21j1zS3V0x03X4SG1eSGC5aAqtgsyrCXSrAuj7CUpBKSB7oDJzUD1H0sPZaE0auErNpN7hyFC1No~twJiKk3SzPyyRMPTMOI37x4gYjsZiKw7IGLG-M1oYyvQvXANkDQy5Y21HBn32JtI6UAKApxjHUWzmir59Ja-wolC8JQeG~V26vTAsy9OIZwJfcm12OcOBjx4y4RG~UeGfuVPIahOm1rPBFlD~mS-1TTGMmLJKOorJuGq8u01r1hPNUD4Z76NQf0RnTEuVMu4sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594389&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04103.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chimeric nodes fall into two classes. Nodes of the same colour represent the same species. The number in each node represents the coverage value of the node. A contig sequence is also attached to each node.</p></div></div><div content-id="DSU041F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04103.png?Expires=2147483647&Signature=gSeQvatL-qNJ-DJW7tPJoHqfx4C1cM2QrCUk70E6daENypHkQsaKImf-top0aAUciN9tdOtVlgRv3ZORbh9JeYLD3pqhNWg2NcZrmyzeUoZzwMgMIUphpW4k5jIeTZWHo1adYUs9YPb4y1j3gYOYbfSrhr~BnndySrWJFL~P~Uy48UpszoOlncm4vgNZI-8nzXRyaf6A0zUcllhcWYyYgI-2Ryz6vhuyouDrDvwTLRbWfNCZDLsZy-3XFSv8FNqfu5jPxhlc6ry-PXr5Ikp3A4Knq7zDnQGcwvp1CuS3sJVOeH8x7ga-tEYxp7pWcDXSpDxN2tOuoGkt4hly9ohp1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chimeric nodes fall into two classes. Nodes of the same colour represent the same species. The number in each node represents the coverage value of the node. A contig sequence is also attached to each node." path-from-xml="dsu04103.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594389/dsu04103.tif" path-from-xml="dsu04103.tif" target="_blank">View large</a><a section="6594389" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04103.png?Expires=2147483647&Signature=CliqYjriFUjGdi3ayfXcR6bAysaJ~1UuvESW6ygfzvSCVa8Ei~RT7oJsbKCyNhX5~I1JIpL21j1zS3V0x03X4SG1eSGC5aAqtgsyrCXSrAuj7CUpBKSB7oDJzUD1H0sPZaE0auErNpN7hyFC1No~twJiKk3SzPyyRMPTMOI37x4gYjsZiKw7IGLG-M1oYyvQvXANkDQy5Y21HBn32JtI6UAKApxjHUWzmir59Ja-wolC8JQeG~V26vTAsy9OIZwJfcm12OcOBjx4y4RG~UeGfuVPIahOm1rPBFlD~mS-1TTGMmLJKOorJuGq8u01r1hPNUD4Z76NQf0RnTEuVMu4sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594389&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04103.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chimeric nodes fall into two classes. Nodes of the same colour represent the same species. The number in each node represents the coverage value of the node. A contig sequence is also attached to each node.</p></div></div><p>In the implementation, MetaVelvet-SL consists of two main modules: (i) the supervised learning module to develop a model for the classification of chimeric nodes and (ii) the assembly module. We provide both modules for MetaVelvet-SL's users. Users can infer the taxonomic profile from sequence reads using several well-known accurate taxonomic profiling methods, such as MetaPhlAn.<sup><a reveal-id="DSU041C16" data-open="DSU041C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Alternatively, users can generate a classification model by using prior knowledge about the taxonomy profile of the target microbial community. In either case, the resulting customized model could be well suited to the assembly of the target metagenomes. MetaVelvet-SL also provides a library of pre-trained classification models for several typical environments, such as soil, deep sea, mud, human blood, intestine and mouth.</p><p>Like other assemblers, the input for the assembly module is a set of reads from metagenomes. A pipeline connecting MetaPhlAn and MetaVelvet-SL has been developed which allows users to automatically generate a classification model and then assemble their metagenomic short read data. The source code of MetaVelvet-SL, the pipeline connecting MetaPhlAn and MetaVelvet-SL, and the library of learning models for several typical environments are freely available at <a class="link link-uri" href="http://metavelvet.dna.bio.keio.ac.jp">http://metavelvet.dna.bio.keio.ac.jp</a>.</p>                            <h2 scrollto-destination=6594392 id="6594392" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>The results of MetaVelvet-SL were compared with those from the original MetaVelvet (version 1.2.02),<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> the last version of other state-of-the-art metagenomic assemblers such as IDBA-UD,<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Ray Meta (version 2.3.1)<sup><a reveal-id="DSU041C10" data-open="DSU041C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and Omega (version 1.0.2),<sup><a reveal-id="DSU041C11" data-open="DSU041C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and the single-genome assembler for massive short sequencing reads SOAPdenovo2.<sup><a reveal-id="DSU041C14" data-open="DSU041C14" class="link link-ref link-reveal xref-bibr">14</a></sup> We conducted extensive experiments to evaluate the performance on simulated data sets and on real metagenomic data sets of human gut microbial short read data.</p>                            <h3 scrollto-destination=6594394 id="6594394" class="section-title" data-legacy-id=s3a>3.1. Simulated data set</h3>
<p>We generated simulated metagenomic sequence reads using the most frequently used simulator—the DWGSIM component in the DNAA package (available at <a class="link link-uri" href="http://sourceforge.net/projects/dnaa">http://sourceforge.net/projects/dnaa</a>). We generated short reads with a length of 80 bp and used the default Illumina sequencing noise, whose error rate is 1%. The average and standard deviation of the insert size for paired-end reads were set at 500 and 50 bp, respectively.</p><p>To measure the performance for various taxonomic levels of diversity, we generated four types of assembly data sets from distant to closer taxonomic levels (order, family, genus and species). We selected 20 genomes for each data set and generated short read data sets from the 20 genomes. Since the log-normal distribution has been generally used to model microbial abundance distributions,<sup><a reveal-id="DSU041C20" data-open="DSU041C20" class="link link-ref link-reveal xref-bibr">20</a></sup> we used the log-normal distribution for species abundance. MetaVelvet-SL requires a training data set for learning the classification model. First, we conducted experiments using the pipeline connecting MetaPhlAn and MetaVelvet-SL. The taxonomic profile for the training data set was inferred by MetaPhlAn. Second, to measure the performance of MetaVelvet-SL for different degrees of similarity between the training data set and the assembly data set, we conducted experiments using three levels of training data sets that contain different reference genomes from the assembly data sets. The three levels of training data sets consist of similarities to the assembly data set, from the highest to the lowest (genus, family and order). The genus-level training data set contains different species but in the same genus from the assembly data set. The family-level training data set contains different genus but in the same family from the assembly data set. The order-level training data set contains different families but in the same order from the assembly data set. The list of selected genomes, the coverage of each genome, the number of reads generated and the length of each reference genome for each training data set and each assembly data set are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1–S20</a>.</p><p>We compared the performance of MetaVelvet-SL with those of MetaVelvet,<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> other state-of-the-art superior metagenomic assemblers, IDBA-UD<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> with the default parameters for metagenomic assembly, Ray Meta<sup><a reveal-id="DSU041C10" data-open="DSU041C10" class="link link-ref link-reveal xref-bibr">10</a></sup> with the <em>k</em>-mer size suggested in the Ray Meta paper and Omega<sup><a reveal-id="DSU041C11" data-open="DSU041C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with the overlap length suggested in the Omega's instruction, and a single-genome assembler, SOAPdenovo2<sup><a reveal-id="DSU041C14" data-open="DSU041C14" class="link link-ref link-reveal xref-bibr">14</a></sup> with the same <em>k</em>-mer size as MetaVelvet and MetaVelvet-SL.</p><p>We evaluated the assembly performance with Nm50 and three other measurements: the total length, the maximum length and the number of accurate scaffolds (sub-scaffolds not containing any chimeric region). We defined Nm50, the corrected N50 length for metagenomic assembly results. We cut every scaffold at chimeric mis-assembled points into sub-scaffolds so that the sub-scaffolds no longer contain any chimeric region. The usual N50 is defined to indicate the scaffold length such that 50% of the total length of scaffolds lies in scaffolds of this size or larger. Nm50 is N50 length of the sub-scaffolds not containing any chimeric region. Chimeric regions in a scaffold were determined by two steps. First, the best-fit alignments between a scaffold and the set of input reference genomes are calculated using BLAST so that the predicted reference genome for the scaffold can be obtained. Second, if any region in the scaffold is aligned to another reference genome different from the predicted reference genome for the scaffold, the region is determined as a chimeric region. (The technical details are described in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data.</a>)</p><p>The statistics of assembly results are shown in Table <a reveal-id="DSU041TB1" data-open="DSU041TB1" class="link link-reveal link-table xref-fig">1</a>. The taxonomic profile for the training data set used in MetaVelvet-SL was inferred by MetaPhlAn. For all assembly data sets, MetaVelvet-SL generated higher Nm50 and longer maximum length of scaffolds than MetaVelvet, IDBA-UD, Ray Meta, Omega and SOAPdenovo2. The total length of scaffolds is similar among the assemblers. The results showed that MetaVelvet-SL generated longer accurate scaffolds. </p>                        <a id="6594400" scrollto-destination="6594400"></a>
<div content-id="DSU041TB1" class="table-modal"><div class="table-wrap"><div id="DSU041TB1" class="table-wrap-title" data-id="DSU041TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of assembly results for simulated data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">Order </td></tr><tr><td> Nm50 (bp) </td><td><strong>695,261</strong> </td><td>222,972 </td><td>243,336 </td><td>10,529 </td><td>154,899 </td><td>29,668 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,546,677</strong> </td><td>1,312,990 </td><td>1,700,200 </td><td>141,628 </td><td>927,783 </td><td>405,783 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>70,711,008 </td><td>70,743,471 </td><td>72,886,145 </td><td>72,067,809 </td><td>71,331,763 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>2,889 </td><td>1,395 </td><td>37,391 </td><td>1,184 </td><td>17,075 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>12,606 </td><td>283,378 </td><td>29,980 </td><td>401,873 </td><td>59,275 </td></tr><tr><td colspan="7">Family </td></tr><tr><td> Nm50 (bp) </td><td><strong>375,942</strong> </td><td>227,243 </td><td>251,915 </td><td>6,751 </td><td>167,523 </td><td>42,500 </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,875,576</strong> </td><td>1,570,565 </td><td>1,247,435 </td><td>167,532 </td><td>1,181,121 </td><td>521,402 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>76,369,071 </td><td>81,721,590 </td><td>86,524,823 </td><td>84,549,043 </td><td>84,756,027 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>5,456 </td><td>1,884 </td><td>58,952 </td><td>1,655 </td><td>18,704 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>14,855 </td><td>379,757 </td><td>17,155 </td><td>544,821 </td><td>27,233 </td></tr><tr><td colspan="7">Genus </td></tr><tr><td> Nm50 (bp) </td><td><strong>226,033</strong> </td><td>100,132 </td><td>121,196 </td><td>4,642 </td><td>91,637 </td><td>16,533 </td></tr><tr><td> Maximum length (bp) </td><td><strong>2,259,591</strong> </td><td>2,099,603 </td><td>1,246,124 </td><td>85,991 </td><td>1,212,747 </td><td>212,138 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>84,636,187 </td><td>79,218,358 </td><td>81,965,701 </td><td>83,171,453 </td><td>73,537,116 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>14,802 </td><td>10,362 </td><td>97,463 </td><td>6,822 </td><td>24,244 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>19,514 </td><td>306,073 </td><td>35,648 </td><td>1,259,371 </td><td>97,573 </td></tr><tr><td colspan="7">Species </td></tr><tr><td> Nm50 (bp) </td><td><strong>174,495</strong> </td><td>91,159 </td><td>74,670 </td><td>4,469 </td><td>80,592 </td><td>13,053 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,808,921</strong> </td><td>1,878,401 </td><td>2,107,202 </td><td>103,314 </td><td>702,714 </td><td>193,065 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>82,381,332 </td><td>65,980,631 </td><td>85,892,445 </td><td>74,075,828 </td><td>67,422,938 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>29,472 </td><td>18,864 </td><td>132,284 </td><td>17,077 </td><td>24,096 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>17,610 </td><td>353,353 </td><td>20,082 </td><td>352,521 </td><td>208,417 </td></tr></tbody></table></div><div content-id="DSU041TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">Order </td></tr><tr><td> Nm50 (bp) </td><td><strong>695,261</strong> </td><td>222,972 </td><td>243,336 </td><td>10,529 </td><td>154,899 </td><td>29,668 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,546,677</strong> </td><td>1,312,990 </td><td>1,700,200 </td><td>141,628 </td><td>927,783 </td><td>405,783 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>70,711,008 </td><td>70,743,471 </td><td>72,886,145 </td><td>72,067,809 </td><td>71,331,763 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>2,889 </td><td>1,395 </td><td>37,391 </td><td>1,184 </td><td>17,075 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>12,606 </td><td>283,378 </td><td>29,980 </td><td>401,873 </td><td>59,275 </td></tr><tr><td colspan="7">Family </td></tr><tr><td> Nm50 (bp) </td><td><strong>375,942</strong> </td><td>227,243 </td><td>251,915 </td><td>6,751 </td><td>167,523 </td><td>42,500 </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,875,576</strong> </td><td>1,570,565 </td><td>1,247,435 </td><td>167,532 </td><td>1,181,121 </td><td>521,402 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>76,369,071 </td><td>81,721,590 </td><td>86,524,823 </td><td>84,549,043 </td><td>84,756,027 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>5,456 </td><td>1,884 </td><td>58,952 </td><td>1,655 </td><td>18,704 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>14,855 </td><td>379,757 </td><td>17,155 </td><td>544,821 </td><td>27,233 </td></tr><tr><td colspan="7">Genus </td></tr><tr><td> Nm50 (bp) </td><td><strong>226,033</strong> </td><td>100,132 </td><td>121,196 </td><td>4,642 </td><td>91,637 </td><td>16,533 </td></tr><tr><td> Maximum length (bp) </td><td><strong>2,259,591</strong> </td><td>2,099,603 </td><td>1,246,124 </td><td>85,991 </td><td>1,212,747 </td><td>212,138 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>84,636,187 </td><td>79,218,358 </td><td>81,965,701 </td><td>83,171,453 </td><td>73,537,116 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>14,802 </td><td>10,362 </td><td>97,463 </td><td>6,822 </td><td>24,244 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>19,514 </td><td>306,073 </td><td>35,648 </td><td>1,259,371 </td><td>97,573 </td></tr><tr><td colspan="7">Species </td></tr><tr><td> Nm50 (bp) </td><td><strong>174,495</strong> </td><td>91,159 </td><td>74,670 </td><td>4,469 </td><td>80,592 </td><td>13,053 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,808,921</strong> </td><td>1,878,401 </td><td>2,107,202 </td><td>103,314 </td><td>702,714 </td><td>193,065 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>82,381,332 </td><td>65,980,631 </td><td>85,892,445 </td><td>74,075,828 </td><td>67,422,938 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>29,472 </td><td>18,864 </td><td>132,284 </td><td>17,077 </td><td>24,096 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>17,610 </td><td>353,353 </td><td>20,082 </td><td>352,521 </td><td>208,417 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>All computations were executed using Intel(R) Xeon(R) E5540 processors (2.53 GHz), with 96-GB physical memory, except for a few cases. Top performances are shown in bold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594400" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB1" class="table-wrap-title" data-id="DSU041TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of assembly results for simulated data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">Order </td></tr><tr><td> Nm50 (bp) </td><td><strong>695,261</strong> </td><td>222,972 </td><td>243,336 </td><td>10,529 </td><td>154,899 </td><td>29,668 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,546,677</strong> </td><td>1,312,990 </td><td>1,700,200 </td><td>141,628 </td><td>927,783 </td><td>405,783 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>70,711,008 </td><td>70,743,471 </td><td>72,886,145 </td><td>72,067,809 </td><td>71,331,763 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>2,889 </td><td>1,395 </td><td>37,391 </td><td>1,184 </td><td>17,075 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>12,606 </td><td>283,378 </td><td>29,980 </td><td>401,873 </td><td>59,275 </td></tr><tr><td colspan="7">Family </td></tr><tr><td> Nm50 (bp) </td><td><strong>375,942</strong> </td><td>227,243 </td><td>251,915 </td><td>6,751 </td><td>167,523 </td><td>42,500 </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,875,576</strong> </td><td>1,570,565 </td><td>1,247,435 </td><td>167,532 </td><td>1,181,121 </td><td>521,402 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>76,369,071 </td><td>81,721,590 </td><td>86,524,823 </td><td>84,549,043 </td><td>84,756,027 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>5,456 </td><td>1,884 </td><td>58,952 </td><td>1,655 </td><td>18,704 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>14,855 </td><td>379,757 </td><td>17,155 </td><td>544,821 </td><td>27,233 </td></tr><tr><td colspan="7">Genus </td></tr><tr><td> Nm50 (bp) </td><td><strong>226,033</strong> </td><td>100,132 </td><td>121,196 </td><td>4,642 </td><td>91,637 </td><td>16,533 </td></tr><tr><td> Maximum length (bp) </td><td><strong>2,259,591</strong> </td><td>2,099,603 </td><td>1,246,124 </td><td>85,991 </td><td>1,212,747 </td><td>212,138 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>84,636,187 </td><td>79,218,358 </td><td>81,965,701 </td><td>83,171,453 </td><td>73,537,116 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>14,802 </td><td>10,362 </td><td>97,463 </td><td>6,822 </td><td>24,244 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>19,514 </td><td>306,073 </td><td>35,648 </td><td>1,259,371 </td><td>97,573 </td></tr><tr><td colspan="7">Species </td></tr><tr><td> Nm50 (bp) </td><td><strong>174,495</strong> </td><td>91,159 </td><td>74,670 </td><td>4,469 </td><td>80,592 </td><td>13,053 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,808,921</strong> </td><td>1,878,401 </td><td>2,107,202 </td><td>103,314 </td><td>702,714 </td><td>193,065 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>82,381,332 </td><td>65,980,631 </td><td>85,892,445 </td><td>74,075,828 </td><td>67,422,938 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>29,472 </td><td>18,864 </td><td>132,284 </td><td>17,077 </td><td>24,096 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>17,610 </td><td>353,353 </td><td>20,082 </td><td>352,521 </td><td>208,417 </td></tr></tbody></table></div><div content-id="DSU041TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">Order </td></tr><tr><td> Nm50 (bp) </td><td><strong>695,261</strong> </td><td>222,972 </td><td>243,336 </td><td>10,529 </td><td>154,899 </td><td>29,668 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,546,677</strong> </td><td>1,312,990 </td><td>1,700,200 </td><td>141,628 </td><td>927,783 </td><td>405,783 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>70,711,008 </td><td>70,743,471 </td><td>72,886,145 </td><td>72,067,809 </td><td>71,331,763 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>2,889 </td><td>1,395 </td><td>37,391 </td><td>1,184 </td><td>17,075 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>12,606 </td><td>283,378 </td><td>29,980 </td><td>401,873 </td><td>59,275 </td></tr><tr><td colspan="7">Family </td></tr><tr><td> Nm50 (bp) </td><td><strong>375,942</strong> </td><td>227,243 </td><td>251,915 </td><td>6,751 </td><td>167,523 </td><td>42,500 </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,875,576</strong> </td><td>1,570,565 </td><td>1,247,435 </td><td>167,532 </td><td>1,181,121 </td><td>521,402 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>76,369,071 </td><td>81,721,590 </td><td>86,524,823 </td><td>84,549,043 </td><td>84,756,027 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>5,456 </td><td>1,884 </td><td>58,952 </td><td>1,655 </td><td>18,704 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>14,855 </td><td>379,757 </td><td>17,155 </td><td>544,821 </td><td>27,233 </td></tr><tr><td colspan="7">Genus </td></tr><tr><td> Nm50 (bp) </td><td><strong>226,033</strong> </td><td>100,132 </td><td>121,196 </td><td>4,642 </td><td>91,637 </td><td>16,533 </td></tr><tr><td> Maximum length (bp) </td><td><strong>2,259,591</strong> </td><td>2,099,603 </td><td>1,246,124 </td><td>85,991 </td><td>1,212,747 </td><td>212,138 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>84,636,187 </td><td>79,218,358 </td><td>81,965,701 </td><td>83,171,453 </td><td>73,537,116 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>14,802 </td><td>10,362 </td><td>97,463 </td><td>6,822 </td><td>24,244 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>19,514 </td><td>306,073 </td><td>35,648 </td><td>1,259,371 </td><td>97,573 </td></tr><tr><td colspan="7">Species </td></tr><tr><td> Nm50 (bp) </td><td><strong>174,495</strong> </td><td>91,159 </td><td>74,670 </td><td>4,469 </td><td>80,592 </td><td>13,053 </td></tr><tr><td> Maximum length (bp) </td><td><strong>3,808,921</strong> </td><td>1,878,401 </td><td>2,107,202 </td><td>103,314 </td><td>702,714 </td><td>193,065 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>82,381,332 </td><td>65,980,631 </td><td>85,892,445 </td><td>74,075,828 </td><td>67,422,938 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>29,472 </td><td>18,864 </td><td>132,284 </td><td>17,077 </td><td>24,096 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>17,610 </td><td>353,353 </td><td>20,082 </td><td>352,521 </td><td>208,417 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>All computations were executed using Intel(R) Xeon(R) E5540 processors (2.53 GHz), with 96-GB physical memory, except for a few cases. Top performances are shown in bold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594400" target="_blank">View Large</a></div></div><p>For all assembly data sets, Ray Meta required the largest computation times, followed by IDBA-UD. The computation times of MetaVelvet-SL increased at low taxonomic levels compared with MetaVelvet. This is mainly because MetaVelvet-SL requires the computation time for learning the classification model and classifying chimeric node candidates the number of which is larger in low taxonomic levels. The genomes become more similar and share more <em>k</em>-mers in low taxonomic levels. Table <a reveal-id="DSU041TB2" data-open="DSU041TB2" class="link link-reveal link-table xref-fig">2</a> represents the number of chimeric node candidates in de Bruijn graph constructed from each assembly data set. The species data set has the highest number of chimeric node candidates which is &gt;10 times of the number of chimeric node candidates in the order data set. </p>                        <a id="6594402" scrollto-destination="6594402"></a>
<div content-id="DSU041TB2" class="table-modal"><div class="table-wrap"><div id="DSU041TB2" class="table-wrap-title" data-id="DSU041TB2"><div class="label -label">Table 2.</div><div class="caption"><p>The number of chimeric node candidates in de Bruijn graph constructed from each assembly data set</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Positive<hr /> </th><th>Negative </th><th rowspan="2">Total no. of chimeric node candidates </th></tr><tr><th>Class 1 </th><th>Class 2 </th><th>Class 3 </th></tr></thead><tbody><tr><td>Order </td><td>82 </td><td>0 </td><td>1,515 </td><td>1,597 </td></tr><tr><td>Family </td><td>146 </td><td>1 </td><td>2,456 </td><td>2,603 </td></tr><tr><td>Genus </td><td>2,918 </td><td>731 </td><td>8,505 </td><td>12,154 </td></tr><tr><td>Species </td><td>3,074 </td><td>246 </td><td>14,589 </td><td>17,909 </td></tr></tbody></table></div><div content-id="DSU041TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Positive<hr /> </th><th>Negative </th><th rowspan="2">Total no. of chimeric node candidates </th></tr><tr><th>Class 1 </th><th>Class 2 </th><th>Class 3 </th></tr></thead><tbody><tr><td>Order </td><td>82 </td><td>0 </td><td>1,515 </td><td>1,597 </td></tr><tr><td>Family </td><td>146 </td><td>1 </td><td>2,456 </td><td>2,603 </td></tr><tr><td>Genus </td><td>2,918 </td><td>731 </td><td>8,505 </td><td>12,154 </td></tr><tr><td>Species </td><td>3,074 </td><td>246 </td><td>14,589 </td><td>17,909 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594402" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB2" class="table-wrap-title" data-id="DSU041TB2"><div class="label -label">Table 2.</div><div class="caption"><p>The number of chimeric node candidates in de Bruijn graph constructed from each assembly data set</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Positive<hr /> </th><th>Negative </th><th rowspan="2">Total no. of chimeric node candidates </th></tr><tr><th>Class 1 </th><th>Class 2 </th><th>Class 3 </th></tr></thead><tbody><tr><td>Order </td><td>82 </td><td>0 </td><td>1,515 </td><td>1,597 </td></tr><tr><td>Family </td><td>146 </td><td>1 </td><td>2,456 </td><td>2,603 </td></tr><tr><td>Genus </td><td>2,918 </td><td>731 </td><td>8,505 </td><td>12,154 </td></tr><tr><td>Species </td><td>3,074 </td><td>246 </td><td>14,589 </td><td>17,909 </td></tr></tbody></table></div><div content-id="DSU041TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Positive<hr /> </th><th>Negative </th><th rowspan="2">Total no. of chimeric node candidates </th></tr><tr><th>Class 1 </th><th>Class 2 </th><th>Class 3 </th></tr></thead><tbody><tr><td>Order </td><td>82 </td><td>0 </td><td>1,515 </td><td>1,597 </td></tr><tr><td>Family </td><td>146 </td><td>1 </td><td>2,456 </td><td>2,603 </td></tr><tr><td>Genus </td><td>2,918 </td><td>731 </td><td>8,505 </td><td>12,154 </td></tr><tr><td>Species </td><td>3,074 </td><td>246 </td><td>14,589 </td><td>17,909 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594402" target="_blank">View Large</a></div></div><p>The statistics of assembly of MetaVelvet-SL using different training datasets are shown in Table <a reveal-id="DSU041TB3" data-open="DSU041TB3" class="link link-reveal link-table xref-fig">3</a>. The models used in MetaVelvet-SL were generated from the taxonomic profiles predicted by MetaPhlAn and three similarity levels of training data sets (genus, family and order) for each assembly data set. There was no significant difference among the assembly results using different training data sets. This result showed that MetaVelvet-SL was robust for the dissimilarity between the training data set and the assembly data set. </p>                        <a id="6594404" scrollto-destination="6594404"></a>
<div content-id="DSU041TB3" class="table-modal"><div class="table-wrap"><div id="DSU041TB3" class="table-wrap-title" data-id="DSU041TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of assembly results of MetaVelvet-SL using different training data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">MetaVelvet-SL<hr /> </th></tr><tr><th>(+MetaPhlAn) </th><th>Genus-level training data set </th><th>Family-level training data set </th><th>Order-level training data set </th></tr></thead><tbody><tr><td colspan="5">Order </td></tr><tr><td> Nm50 (bp) </td><td>695,261 </td><td>672,952 </td><td>686,074 </td><td>695,557 </td></tr><tr><td> Maximum length (bp) </td><td>3,546,677 </td><td>3,415,875 </td><td>3,547,025 </td><td>3,818,061 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>69,881,185 </td><td>69,288,924 </td><td>69,387,743 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>4,379 </td><td>4,747 </td><td>4,829 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>26,773 </td><td>26,612 </td><td>26,660 </td></tr><tr><td colspan="5">Family </td></tr><tr><td> Nm50 (bp) </td><td>375,942 </td><td>377,604 </td><td>384,795 </td><td>384,795 </td></tr><tr><td> Maximum length (bp) </td><td>1,875,576 </td><td>2,326,125 </td><td>2,326,197 </td><td>1,927,551 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>83,888,560 </td><td>83,877,454 </td><td>83,877,321 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>8,777 </td><td>8,687 </td><td>8,679 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>35,477 </td><td>35,577 </td><td>35,469 </td></tr><tr><td colspan="5">Genus </td></tr><tr><td> Nm50 (bp) </td><td>226,033 </td><td>233,924 </td><td>266,018 </td><td>257,292 </td></tr><tr><td> Maximum length (bp) </td><td>2,259,591 </td><td>2,888,749 </td><td>2,974,950 </td><td>2,843,963 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>82,525,756 </td><td>83,385,233 </td><td>83,634,261 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>11,450 </td><td>9,376 </td><td>8,512 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>187,963 </td><td>188,285 </td><td>188,325 </td></tr><tr><td colspan="5">Species </td></tr><tr><td> Nm50 (bp) </td><td>174,495 </td><td>166,528 </td><td>158,509 </td><td>167,722 </td></tr><tr><td> Maximum length (bp) </td><td>3,808,921 </td><td>3,292,179 </td><td>3,292,250 </td><td>3,292,179 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>81,141,481 </td><td>81,447,218 </td><td>81,414,639 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>19,114 </td><td>21,073 </td><td>22,735 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>195,512 </td><td>195,534 </td><td>195,460 </td></tr></tbody></table></div><div content-id="DSU041TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">MetaVelvet-SL<hr /> </th></tr><tr><th>(+MetaPhlAn) </th><th>Genus-level training data set </th><th>Family-level training data set </th><th>Order-level training data set </th></tr></thead><tbody><tr><td colspan="5">Order </td></tr><tr><td> Nm50 (bp) </td><td>695,261 </td><td>672,952 </td><td>686,074 </td><td>695,557 </td></tr><tr><td> Maximum length (bp) </td><td>3,546,677 </td><td>3,415,875 </td><td>3,547,025 </td><td>3,818,061 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>69,881,185 </td><td>69,288,924 </td><td>69,387,743 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>4,379 </td><td>4,747 </td><td>4,829 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>26,773 </td><td>26,612 </td><td>26,660 </td></tr><tr><td colspan="5">Family </td></tr><tr><td> Nm50 (bp) </td><td>375,942 </td><td>377,604 </td><td>384,795 </td><td>384,795 </td></tr><tr><td> Maximum length (bp) </td><td>1,875,576 </td><td>2,326,125 </td><td>2,326,197 </td><td>1,927,551 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>83,888,560 </td><td>83,877,454 </td><td>83,877,321 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>8,777 </td><td>8,687 </td><td>8,679 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>35,477 </td><td>35,577 </td><td>35,469 </td></tr><tr><td colspan="5">Genus </td></tr><tr><td> Nm50 (bp) </td><td>226,033 </td><td>233,924 </td><td>266,018 </td><td>257,292 </td></tr><tr><td> Maximum length (bp) </td><td>2,259,591 </td><td>2,888,749 </td><td>2,974,950 </td><td>2,843,963 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>82,525,756 </td><td>83,385,233 </td><td>83,634,261 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>11,450 </td><td>9,376 </td><td>8,512 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>187,963 </td><td>188,285 </td><td>188,325 </td></tr><tr><td colspan="5">Species </td></tr><tr><td> Nm50 (bp) </td><td>174,495 </td><td>166,528 </td><td>158,509 </td><td>167,722 </td></tr><tr><td> Maximum length (bp) </td><td>3,808,921 </td><td>3,292,179 </td><td>3,292,250 </td><td>3,292,179 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>81,141,481 </td><td>81,447,218 </td><td>81,414,639 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>19,114 </td><td>21,073 </td><td>22,735 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>195,512 </td><td>195,534 </td><td>195,460 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594404" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB3" class="table-wrap-title" data-id="DSU041TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of assembly results of MetaVelvet-SL using different training data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">MetaVelvet-SL<hr /> </th></tr><tr><th>(+MetaPhlAn) </th><th>Genus-level training data set </th><th>Family-level training data set </th><th>Order-level training data set </th></tr></thead><tbody><tr><td colspan="5">Order </td></tr><tr><td> Nm50 (bp) </td><td>695,261 </td><td>672,952 </td><td>686,074 </td><td>695,557 </td></tr><tr><td> Maximum length (bp) </td><td>3,546,677 </td><td>3,415,875 </td><td>3,547,025 </td><td>3,818,061 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>69,881,185 </td><td>69,288,924 </td><td>69,387,743 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>4,379 </td><td>4,747 </td><td>4,829 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>26,773 </td><td>26,612 </td><td>26,660 </td></tr><tr><td colspan="5">Family </td></tr><tr><td> Nm50 (bp) </td><td>375,942 </td><td>377,604 </td><td>384,795 </td><td>384,795 </td></tr><tr><td> Maximum length (bp) </td><td>1,875,576 </td><td>2,326,125 </td><td>2,326,197 </td><td>1,927,551 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>83,888,560 </td><td>83,877,454 </td><td>83,877,321 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>8,777 </td><td>8,687 </td><td>8,679 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>35,477 </td><td>35,577 </td><td>35,469 </td></tr><tr><td colspan="5">Genus </td></tr><tr><td> Nm50 (bp) </td><td>226,033 </td><td>233,924 </td><td>266,018 </td><td>257,292 </td></tr><tr><td> Maximum length (bp) </td><td>2,259,591 </td><td>2,888,749 </td><td>2,974,950 </td><td>2,843,963 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>82,525,756 </td><td>83,385,233 </td><td>83,634,261 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>11,450 </td><td>9,376 </td><td>8,512 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>187,963 </td><td>188,285 </td><td>188,325 </td></tr><tr><td colspan="5">Species </td></tr><tr><td> Nm50 (bp) </td><td>174,495 </td><td>166,528 </td><td>158,509 </td><td>167,722 </td></tr><tr><td> Maximum length (bp) </td><td>3,808,921 </td><td>3,292,179 </td><td>3,292,250 </td><td>3,292,179 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>81,141,481 </td><td>81,447,218 </td><td>81,414,639 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>19,114 </td><td>21,073 </td><td>22,735 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>195,512 </td><td>195,534 </td><td>195,460 </td></tr></tbody></table></div><div content-id="DSU041TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">MetaVelvet-SL<hr /> </th></tr><tr><th>(+MetaPhlAn) </th><th>Genus-level training data set </th><th>Family-level training data set </th><th>Order-level training data set </th></tr></thead><tbody><tr><td colspan="5">Order </td></tr><tr><td> Nm50 (bp) </td><td>695,261 </td><td>672,952 </td><td>686,074 </td><td>695,557 </td></tr><tr><td> Maximum length (bp) </td><td>3,546,677 </td><td>3,415,875 </td><td>3,547,025 </td><td>3,818,061 </td></tr><tr><td> Total scaffold length (bp) </td><td>69,383,955 </td><td>69,881,185 </td><td>69,288,924 </td><td>69,387,743 </td></tr><tr><td> Number of scaffolds </td><td>4,755 </td><td>4,379 </td><td>4,747 </td><td>4,829 </td></tr><tr><td> Required CPU time (s) </td><td>26,735 </td><td>26,773 </td><td>26,612 </td><td>26,660 </td></tr><tr><td colspan="5">Family </td></tr><tr><td> Nm50 (bp) </td><td>375,942 </td><td>377,604 </td><td>384,795 </td><td>384,795 </td></tr><tr><td> Maximum length (bp) </td><td>1,875,576 </td><td>2,326,125 </td><td>2,326,197 </td><td>1,927,551 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,890,270 </td><td>83,888,560 </td><td>83,877,454 </td><td>83,877,321 </td></tr><tr><td> Number of scaffolds </td><td>8,809 </td><td>8,777 </td><td>8,687 </td><td>8,679 </td></tr><tr><td> Required CPU time (s) </td><td>35,685 </td><td>35,477 </td><td>35,577 </td><td>35,469 </td></tr><tr><td colspan="5">Genus </td></tr><tr><td> Nm50 (bp) </td><td>226,033 </td><td>233,924 </td><td>266,018 </td><td>257,292 </td></tr><tr><td> Maximum length (bp) </td><td>2,259,591 </td><td>2,888,749 </td><td>2,974,950 </td><td>2,843,963 </td></tr><tr><td> Total scaffold length (bp) </td><td>83,281,358 </td><td>82,525,756 </td><td>83,385,233 </td><td>83,634,261 </td></tr><tr><td> Number of scaffolds </td><td>10,555 </td><td>11,450 </td><td>9,376 </td><td>8,512 </td></tr><tr><td> Required CPU time (s) </td><td>188,170 </td><td>187,963 </td><td>188,285 </td><td>188,325 </td></tr><tr><td colspan="5">Species </td></tr><tr><td> Nm50 (bp) </td><td>174,495 </td><td>166,528 </td><td>158,509 </td><td>167,722 </td></tr><tr><td> Maximum length (bp) </td><td>3,808,921 </td><td>3,292,179 </td><td>3,292,250 </td><td>3,292,179 </td></tr><tr><td> Total scaffold length (bp) </td><td>81,524,460 </td><td>81,141,481 </td><td>81,447,218 </td><td>81,414,639 </td></tr><tr><td> Number of scaffolds </td><td>22,440 </td><td>19,114 </td><td>21,073 </td><td>22,735 </td></tr><tr><td> Required CPU time (s) </td><td>195,454 </td><td>195,512 </td><td>195,534 </td><td>195,460 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594404" target="_blank">View Large</a></div></div><p>Since one of our primary goals in this work is to improve the sensitivity and accuracy for detecting chimeric nodes by supervised learning, we compared the classification capability of chimeric nodes by MetaVelvet-SL and MetaVelvet. The results of the classification are shown in Table <a reveal-id="DSU041TB4" data-open="DSU041TB4" class="link link-reveal link-table xref-fig">4</a>. We evaluated the sensitivity and the accuracy. Sensitivity is the true positive rate, the percentage of true identified chimeric nodes (positive classes consists of Class 1 and Class 2), while accuracy is the percentage of true results, both true positive (true identified chimeric nodes) and true negative (true identified non-chimeric nodes, consisting of Class 3). As shown in Table <a reveal-id="DSU041TB2" data-open="DSU041TB2" class="link link-reveal link-table xref-fig">2</a>, all of the assembly data sets have imbalanced classes of chimeric node candidates. Therefore, to avoid inflated performance estimates on imbalanced data sets, we calculated the balanced accuracy too. Balanced accuracy is the average between sensitivity and specificity.<sup><a reveal-id="DSU041C21" data-open="DSU041C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Specificity is the true negative rate, the percentage of true identified non-chimeric nodes. As shown in Table <a reveal-id="DSU041TB4" data-open="DSU041TB4" class="link link-reveal link-table xref-fig">4</a>, the balanced accuracy of MetaVelvet-SL, even for the lowest similarity level of training data set (order-level training data set), was higher than MetaVelvet. </p>                        <a id="6594406" scrollto-destination="6594406"></a>
<div content-id="DSU041TB4" class="table-modal"><div class="table-wrap"><div id="DSU041TB4" class="table-wrap-title" data-id="DSU041TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Classification results for chimeric nodes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="12">MetaVelvet-SL<hr /> </th><th colspan="3">MetaVelvet<hr /> </th></tr><tr><th colspan="3">(Training: Genus)<hr /> </th><th colspan="3">(Training: Family)<hr /> </th><th colspan="3">(Training: Order)<hr /> </th><th colspan="3">(+MetaPhlAn)<hr /> </th><th colspan="3"> </th></tr><tr><th> </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th></tr></thead><tbody><tr><td>Order </td><td>84.15 </td><td>94.49 </td><td>89.60 </td><td>54.88 </td><td>96.12 </td><td>76.61 </td><td>57.32 </td><td>95.49 </td><td>77.44 </td><td>42.68 </td><td>96.62 </td><td>71.11 </td><td>42.68 </td><td>92.42 </td><td>67.74 </td></tr><tr><td>Family </td><td>80.27 </td><td>97.54 </td><td>89.42 </td><td>63.27 </td><td>97.08 </td><td>81.80 </td><td>55.10 </td><td>96.85 </td><td>77.23 </td><td>65.99 </td><td>97.35 </td><td>82.61 </td><td>44.90 </td><td>93.32 </td><td>69.21 </td></tr><tr><td>Genus </td><td>62.76 </td><td>76.92 </td><td>72.88 </td><td>50.07 </td><td>65.03 </td><td>60.76 </td><td>54.29 </td><td>67.88 </td><td>64.00 </td><td>58.98 </td><td>81.90 </td><td>75.35 </td><td>33.13 </td><td>62.37 </td><td>46.91 </td></tr><tr><td>Species </td><td>52.80 </td><td>54.84 </td><td>54.05 </td><td>36.83 </td><td>70.60 </td><td>57.32 </td><td>33.10 </td><td>82.19 </td><td>63.23 </td><td>40.24 </td><td>83.15 </td><td>66.58 </td><td>15.21 </td><td>71.61 </td><td>48.09 </td></tr></tbody></table></div><div content-id="DSU041TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="12">MetaVelvet-SL<hr /> </th><th colspan="3">MetaVelvet<hr /> </th></tr><tr><th colspan="3">(Training: Genus)<hr /> </th><th colspan="3">(Training: Family)<hr /> </th><th colspan="3">(Training: Order)<hr /> </th><th colspan="3">(+MetaPhlAn)<hr /> </th><th colspan="3"> </th></tr><tr><th> </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th></tr></thead><tbody><tr><td>Order </td><td>84.15 </td><td>94.49 </td><td>89.60 </td><td>54.88 </td><td>96.12 </td><td>76.61 </td><td>57.32 </td><td>95.49 </td><td>77.44 </td><td>42.68 </td><td>96.62 </td><td>71.11 </td><td>42.68 </td><td>92.42 </td><td>67.74 </td></tr><tr><td>Family </td><td>80.27 </td><td>97.54 </td><td>89.42 </td><td>63.27 </td><td>97.08 </td><td>81.80 </td><td>55.10 </td><td>96.85 </td><td>77.23 </td><td>65.99 </td><td>97.35 </td><td>82.61 </td><td>44.90 </td><td>93.32 </td><td>69.21 </td></tr><tr><td>Genus </td><td>62.76 </td><td>76.92 </td><td>72.88 </td><td>50.07 </td><td>65.03 </td><td>60.76 </td><td>54.29 </td><td>67.88 </td><td>64.00 </td><td>58.98 </td><td>81.90 </td><td>75.35 </td><td>33.13 </td><td>62.37 </td><td>46.91 </td></tr><tr><td>Species </td><td>52.80 </td><td>54.84 </td><td>54.05 </td><td>36.83 </td><td>70.60 </td><td>57.32 </td><td>33.10 </td><td>82.19 </td><td>63.23 </td><td>40.24 </td><td>83.15 </td><td>66.58 </td><td>15.21 </td><td>71.61 </td><td>48.09 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>Sen (%) means the percentage of sensitivity; Acc (%) means the percentage of accuracy and BA (%) means the percentage of balanced accuracy.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594406" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB4" class="table-wrap-title" data-id="DSU041TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Classification results for chimeric nodes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="12">MetaVelvet-SL<hr /> </th><th colspan="3">MetaVelvet<hr /> </th></tr><tr><th colspan="3">(Training: Genus)<hr /> </th><th colspan="3">(Training: Family)<hr /> </th><th colspan="3">(Training: Order)<hr /> </th><th colspan="3">(+MetaPhlAn)<hr /> </th><th colspan="3"> </th></tr><tr><th> </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th></tr></thead><tbody><tr><td>Order </td><td>84.15 </td><td>94.49 </td><td>89.60 </td><td>54.88 </td><td>96.12 </td><td>76.61 </td><td>57.32 </td><td>95.49 </td><td>77.44 </td><td>42.68 </td><td>96.62 </td><td>71.11 </td><td>42.68 </td><td>92.42 </td><td>67.74 </td></tr><tr><td>Family </td><td>80.27 </td><td>97.54 </td><td>89.42 </td><td>63.27 </td><td>97.08 </td><td>81.80 </td><td>55.10 </td><td>96.85 </td><td>77.23 </td><td>65.99 </td><td>97.35 </td><td>82.61 </td><td>44.90 </td><td>93.32 </td><td>69.21 </td></tr><tr><td>Genus </td><td>62.76 </td><td>76.92 </td><td>72.88 </td><td>50.07 </td><td>65.03 </td><td>60.76 </td><td>54.29 </td><td>67.88 </td><td>64.00 </td><td>58.98 </td><td>81.90 </td><td>75.35 </td><td>33.13 </td><td>62.37 </td><td>46.91 </td></tr><tr><td>Species </td><td>52.80 </td><td>54.84 </td><td>54.05 </td><td>36.83 </td><td>70.60 </td><td>57.32 </td><td>33.10 </td><td>82.19 </td><td>63.23 </td><td>40.24 </td><td>83.15 </td><td>66.58 </td><td>15.21 </td><td>71.61 </td><td>48.09 </td></tr></tbody></table></div><div content-id="DSU041TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="12">MetaVelvet-SL<hr /> </th><th colspan="3">MetaVelvet<hr /> </th></tr><tr><th colspan="3">(Training: Genus)<hr /> </th><th colspan="3">(Training: Family)<hr /> </th><th colspan="3">(Training: Order)<hr /> </th><th colspan="3">(+MetaPhlAn)<hr /> </th><th colspan="3"> </th></tr><tr><th> </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th><th>Sen </th><th>Acc </th><th>BA </th></tr></thead><tbody><tr><td>Order </td><td>84.15 </td><td>94.49 </td><td>89.60 </td><td>54.88 </td><td>96.12 </td><td>76.61 </td><td>57.32 </td><td>95.49 </td><td>77.44 </td><td>42.68 </td><td>96.62 </td><td>71.11 </td><td>42.68 </td><td>92.42 </td><td>67.74 </td></tr><tr><td>Family </td><td>80.27 </td><td>97.54 </td><td>89.42 </td><td>63.27 </td><td>97.08 </td><td>81.80 </td><td>55.10 </td><td>96.85 </td><td>77.23 </td><td>65.99 </td><td>97.35 </td><td>82.61 </td><td>44.90 </td><td>93.32 </td><td>69.21 </td></tr><tr><td>Genus </td><td>62.76 </td><td>76.92 </td><td>72.88 </td><td>50.07 </td><td>65.03 </td><td>60.76 </td><td>54.29 </td><td>67.88 </td><td>64.00 </td><td>58.98 </td><td>81.90 </td><td>75.35 </td><td>33.13 </td><td>62.37 </td><td>46.91 </td></tr><tr><td>Species </td><td>52.80 </td><td>54.84 </td><td>54.05 </td><td>36.83 </td><td>70.60 </td><td>57.32 </td><td>33.10 </td><td>82.19 </td><td>63.23 </td><td>40.24 </td><td>83.15 </td><td>66.58 </td><td>15.21 </td><td>71.61 </td><td>48.09 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>Sen (%) means the percentage of sensitivity; Acc (%) means the percentage of accuracy and BA (%) means the percentage of balanced accuracy.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594406" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6594407 id="6594407" class="section-title" data-legacy-id=s3b>3.2. Real data set</h3>
<p>To evaluate the performance of MetaVelvet-SL on real metagenomic data, we used human gut microbial data sets. We assembled five human gut microbial data sets: MH0006 (ERS006497), MH0012 (ERS006494) and MH0047 (ERS006592) from the MetaHIT Consortium<sup><a reveal-id="DSU041C17" data-open="DSU041C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and SRS017227 and SRS018661 from the Human Microbiome Project Consortium.<sup><a reveal-id="DSU041C18" data-open="DSU041C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Two of the data sets (MH0006 and MH0012) were the deepest and second deepest data sets while another data set, MH0047, is a low-coverage data set.</p><p>As for the simulated data sets, the real data sets were assembled by MetaVelvet-SL, MetaVelvet,<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> IDBA-UD<sup><a reveal-id="DSU041C9" data-open="DSU041C9" class="link link-ref link-reveal xref-bibr">9</a></sup> with the default parameters for metagenomic assembly, Ray Meta<sup><a reveal-id="DSU041C10" data-open="DSU041C10" class="link link-ref link-reveal xref-bibr">10</a></sup> with the <em>k</em>-mer size suggested in the Ray Meta paper, Omega<sup><a reveal-id="DSU041C11" data-open="DSU041C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with the overlap length suggested in the Omega's instruction and SOAPdenovo2.<sup><a reveal-id="DSU041C14" data-open="DSU041C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The statistics of assembly performances are summarized in Table <a reveal-id="DSU041TB5" data-open="DSU041TB5" class="link link-reveal link-table xref-fig">5</a>. The classification model for MetaVelvet-SL was obtained by the pipeline using MetaPhlAn to infer the taxonomic profile and then generating the training data set. </p>                        <a id="6594414" scrollto-destination="6594414"></a>
<div content-id="DSU041TB5" class="table-modal"><div class="table-wrap"><div id="DSU041TB5" class="table-wrap-title" data-id="DSU041TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Assembly results for the real human gut microbial data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+ MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">MH0006 (ERS006497) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,073,577</strong> </td><td>82,400 </td><td>424,786 </td><td>248,752 </td><td>245,285 </td><td>293,858 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>366,474,614</strong> </td><td>228,356,028 </td><td>293,629,444 </td><td>314,842,356 </td><td>211,199,449 </td><td>134,644,249 </td></tr><tr><td> Number of scaffold </td><td>927,151 </td><td>387,193 </td><td>197,401 </td><td>521,577 </td><td>609,062 </td><td>150,907 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>9,148,384</strong> </td><td>909,250 </td><td>6,002,739 </td><td>3,042,215 </td><td>2,260,838 </td><td>2,527,198 </td></tr><tr><td colspan="7">MH0012 (ERS006494) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,320,619</strong> </td><td>119,936 </td><td>594,225 </td><td>792,429 </td><td>512,973 </td><td>1,144,479 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>357,949,718</strong> </td><td>255,566,175 </td><td>290,340,811 </td><td>325,057,612 </td><td>272,663,103 </td><td>170,102,775 </td></tr><tr><td> Number of scaffold </td><td>718,438 </td><td>327,103 </td><td>198,771 </td><td>482,983 </td><td>635,814 </td><td>125,383 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>22,126,171</strong> </td><td>2,129,027 </td><td>10,344,620 </td><td>8,856,698 </td><td>6,977,480 </td><td>10,229,304 </td></tr><tr><td colspan="7">MH0047 (ERS006592) </td></tr><tr><td> Maximum length (bp) </td><td><strong>188,905</strong> </td><td>69,475 </td><td>185,593 </td><td>44,319 </td><td>137,473 </td><td>52,084 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>101,916,143</strong> </td><td>75,290,864 </td><td>75,032,143 </td><td>88,092,865 </td><td>50,174,724 </td><td>29,134,928 </td></tr><tr><td> Number of scaffold </td><td>374,148 </td><td>210,477 </td><td>89,786 </td><td>263,713 </td><td>141,466 </td><td>31,961 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>906,906</strong> </td><td>237,568 </td><td>802,594 </td><td>201,366 </td><td>544,742 </td><td>208,223 </td></tr><tr><td colspan="7">SRS017227 </td></tr><tr><td> Maximum length (bp) </td><td><strong>478,428</strong> </td><td>108,476 </td><td>372,927 </td><td>227,256 </td><td>199,208 </td><td>217,259 </td></tr><tr><td> Total scaffold length (bp) </td><td>370,496,571 </td><td>250,969,598 </td><td>349,934,212 </td><td><strong>395,257,497</strong> </td><td>273,595,801 </td><td>206,705,202 </td></tr><tr><td> Number of scaffold </td><td>602,463 </td><td>485,307 </td><td>282,097 </td><td>802,952 </td><td>536,708 </td><td>217,259 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>4,709,551</strong> </td><td>1,064,102 </td><td>4,010,530 </td><td>2,227,194 </td><td>2,501,039 </td><td>1,617,896 </td></tr><tr><td colspan="7">SRS018661 </td></tr><tr><td> Maximum length (bp) </td><td><strong>699,395</strong> </td><td>111,404 </td><td>511,735 </td><td>426,297 </td><td>274,042 </td><td>180,946 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>114,676,867</strong> </td><td>71,339,406 </td><td>109,507,232 </td><td>107,557,997 </td><td>75,351,327 </td><td>47,619,933 </td></tr><tr><td> Number of scaffold </td><td>284,036 </td><td>195,950 </td><td>109,860 </td><td>274,896 </td><td>212,267 </td><td>34,244 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>1,945,258</strong> </td><td>253,138 </td><td>1,296,606 </td><td>848,798 </td><td>1,004,371 </td><td>611,636 </td></tr></tbody></table></div><div content-id="DSU041TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+ MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">MH0006 (ERS006497) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,073,577</strong> </td><td>82,400 </td><td>424,786 </td><td>248,752 </td><td>245,285 </td><td>293,858 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>366,474,614</strong> </td><td>228,356,028 </td><td>293,629,444 </td><td>314,842,356 </td><td>211,199,449 </td><td>134,644,249 </td></tr><tr><td> Number of scaffold </td><td>927,151 </td><td>387,193 </td><td>197,401 </td><td>521,577 </td><td>609,062 </td><td>150,907 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>9,148,384</strong> </td><td>909,250 </td><td>6,002,739 </td><td>3,042,215 </td><td>2,260,838 </td><td>2,527,198 </td></tr><tr><td colspan="7">MH0012 (ERS006494) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,320,619</strong> </td><td>119,936 </td><td>594,225 </td><td>792,429 </td><td>512,973 </td><td>1,144,479 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>357,949,718</strong> </td><td>255,566,175 </td><td>290,340,811 </td><td>325,057,612 </td><td>272,663,103 </td><td>170,102,775 </td></tr><tr><td> Number of scaffold </td><td>718,438 </td><td>327,103 </td><td>198,771 </td><td>482,983 </td><td>635,814 </td><td>125,383 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>22,126,171</strong> </td><td>2,129,027 </td><td>10,344,620 </td><td>8,856,698 </td><td>6,977,480 </td><td>10,229,304 </td></tr><tr><td colspan="7">MH0047 (ERS006592) </td></tr><tr><td> Maximum length (bp) </td><td><strong>188,905</strong> </td><td>69,475 </td><td>185,593 </td><td>44,319 </td><td>137,473 </td><td>52,084 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>101,916,143</strong> </td><td>75,290,864 </td><td>75,032,143 </td><td>88,092,865 </td><td>50,174,724 </td><td>29,134,928 </td></tr><tr><td> Number of scaffold </td><td>374,148 </td><td>210,477 </td><td>89,786 </td><td>263,713 </td><td>141,466 </td><td>31,961 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>906,906</strong> </td><td>237,568 </td><td>802,594 </td><td>201,366 </td><td>544,742 </td><td>208,223 </td></tr><tr><td colspan="7">SRS017227 </td></tr><tr><td> Maximum length (bp) </td><td><strong>478,428</strong> </td><td>108,476 </td><td>372,927 </td><td>227,256 </td><td>199,208 </td><td>217,259 </td></tr><tr><td> Total scaffold length (bp) </td><td>370,496,571 </td><td>250,969,598 </td><td>349,934,212 </td><td><strong>395,257,497</strong> </td><td>273,595,801 </td><td>206,705,202 </td></tr><tr><td> Number of scaffold </td><td>602,463 </td><td>485,307 </td><td>282,097 </td><td>802,952 </td><td>536,708 </td><td>217,259 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>4,709,551</strong> </td><td>1,064,102 </td><td>4,010,530 </td><td>2,227,194 </td><td>2,501,039 </td><td>1,617,896 </td></tr><tr><td colspan="7">SRS018661 </td></tr><tr><td> Maximum length (bp) </td><td><strong>699,395</strong> </td><td>111,404 </td><td>511,735 </td><td>426,297 </td><td>274,042 </td><td>180,946 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>114,676,867</strong> </td><td>71,339,406 </td><td>109,507,232 </td><td>107,557,997 </td><td>75,351,327 </td><td>47,619,933 </td></tr><tr><td> Number of scaffold </td><td>284,036 </td><td>195,950 </td><td>109,860 </td><td>274,896 </td><td>212,267 </td><td>34,244 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>1,945,258</strong> </td><td>253,138 </td><td>1,296,606 </td><td>848,798 </td><td>1,004,371 </td><td>611,636 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Top performances are shown in bold. MetaVelvet-SL, MetaVelvet and SOAPdenovo2 set the <em>k</em>-mer size at 37 for the MH0006 and MH0047 data sets, 43 for the MH0012 data set and 51 for SRS017227 and SRS018661.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594414" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB5" class="table-wrap-title" data-id="DSU041TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Assembly results for the real human gut microbial data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+ MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">MH0006 (ERS006497) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,073,577</strong> </td><td>82,400 </td><td>424,786 </td><td>248,752 </td><td>245,285 </td><td>293,858 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>366,474,614</strong> </td><td>228,356,028 </td><td>293,629,444 </td><td>314,842,356 </td><td>211,199,449 </td><td>134,644,249 </td></tr><tr><td> Number of scaffold </td><td>927,151 </td><td>387,193 </td><td>197,401 </td><td>521,577 </td><td>609,062 </td><td>150,907 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>9,148,384</strong> </td><td>909,250 </td><td>6,002,739 </td><td>3,042,215 </td><td>2,260,838 </td><td>2,527,198 </td></tr><tr><td colspan="7">MH0012 (ERS006494) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,320,619</strong> </td><td>119,936 </td><td>594,225 </td><td>792,429 </td><td>512,973 </td><td>1,144,479 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>357,949,718</strong> </td><td>255,566,175 </td><td>290,340,811 </td><td>325,057,612 </td><td>272,663,103 </td><td>170,102,775 </td></tr><tr><td> Number of scaffold </td><td>718,438 </td><td>327,103 </td><td>198,771 </td><td>482,983 </td><td>635,814 </td><td>125,383 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>22,126,171</strong> </td><td>2,129,027 </td><td>10,344,620 </td><td>8,856,698 </td><td>6,977,480 </td><td>10,229,304 </td></tr><tr><td colspan="7">MH0047 (ERS006592) </td></tr><tr><td> Maximum length (bp) </td><td><strong>188,905</strong> </td><td>69,475 </td><td>185,593 </td><td>44,319 </td><td>137,473 </td><td>52,084 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>101,916,143</strong> </td><td>75,290,864 </td><td>75,032,143 </td><td>88,092,865 </td><td>50,174,724 </td><td>29,134,928 </td></tr><tr><td> Number of scaffold </td><td>374,148 </td><td>210,477 </td><td>89,786 </td><td>263,713 </td><td>141,466 </td><td>31,961 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>906,906</strong> </td><td>237,568 </td><td>802,594 </td><td>201,366 </td><td>544,742 </td><td>208,223 </td></tr><tr><td colspan="7">SRS017227 </td></tr><tr><td> Maximum length (bp) </td><td><strong>478,428</strong> </td><td>108,476 </td><td>372,927 </td><td>227,256 </td><td>199,208 </td><td>217,259 </td></tr><tr><td> Total scaffold length (bp) </td><td>370,496,571 </td><td>250,969,598 </td><td>349,934,212 </td><td><strong>395,257,497</strong> </td><td>273,595,801 </td><td>206,705,202 </td></tr><tr><td> Number of scaffold </td><td>602,463 </td><td>485,307 </td><td>282,097 </td><td>802,952 </td><td>536,708 </td><td>217,259 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>4,709,551</strong> </td><td>1,064,102 </td><td>4,010,530 </td><td>2,227,194 </td><td>2,501,039 </td><td>1,617,896 </td></tr><tr><td colspan="7">SRS018661 </td></tr><tr><td> Maximum length (bp) </td><td><strong>699,395</strong> </td><td>111,404 </td><td>511,735 </td><td>426,297 </td><td>274,042 </td><td>180,946 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>114,676,867</strong> </td><td>71,339,406 </td><td>109,507,232 </td><td>107,557,997 </td><td>75,351,327 </td><td>47,619,933 </td></tr><tr><td> Number of scaffold </td><td>284,036 </td><td>195,950 </td><td>109,860 </td><td>274,896 </td><td>212,267 </td><td>34,244 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>1,945,258</strong> </td><td>253,138 </td><td>1,296,606 </td><td>848,798 </td><td>1,004,371 </td><td>611,636 </td></tr></tbody></table></div><div content-id="DSU041TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>MetaVelvet-SL (+ MetaPhlAn) </th><th>MetaVelvet </th><th>IDBA-UD </th><th>SOAPdenovo2 </th><th>Ray Meta </th><th>Omega </th></tr></thead><tbody><tr><td colspan="7">MH0006 (ERS006497) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,073,577</strong> </td><td>82,400 </td><td>424,786 </td><td>248,752 </td><td>245,285 </td><td>293,858 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>366,474,614</strong> </td><td>228,356,028 </td><td>293,629,444 </td><td>314,842,356 </td><td>211,199,449 </td><td>134,644,249 </td></tr><tr><td> Number of scaffold </td><td>927,151 </td><td>387,193 </td><td>197,401 </td><td>521,577 </td><td>609,062 </td><td>150,907 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>9,148,384</strong> </td><td>909,250 </td><td>6,002,739 </td><td>3,042,215 </td><td>2,260,838 </td><td>2,527,198 </td></tr><tr><td colspan="7">MH0012 (ERS006494) </td></tr><tr><td> Maximum length (bp) </td><td><strong>1,320,619</strong> </td><td>119,936 </td><td>594,225 </td><td>792,429 </td><td>512,973 </td><td>1,144,479 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>357,949,718</strong> </td><td>255,566,175 </td><td>290,340,811 </td><td>325,057,612 </td><td>272,663,103 </td><td>170,102,775 </td></tr><tr><td> Number of scaffold </td><td>718,438 </td><td>327,103 </td><td>198,771 </td><td>482,983 </td><td>635,814 </td><td>125,383 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>22,126,171</strong> </td><td>2,129,027 </td><td>10,344,620 </td><td>8,856,698 </td><td>6,977,480 </td><td>10,229,304 </td></tr><tr><td colspan="7">MH0047 (ERS006592) </td></tr><tr><td> Maximum length (bp) </td><td><strong>188,905</strong> </td><td>69,475 </td><td>185,593 </td><td>44,319 </td><td>137,473 </td><td>52,084 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>101,916,143</strong> </td><td>75,290,864 </td><td>75,032,143 </td><td>88,092,865 </td><td>50,174,724 </td><td>29,134,928 </td></tr><tr><td> Number of scaffold </td><td>374,148 </td><td>210,477 </td><td>89,786 </td><td>263,713 </td><td>141,466 </td><td>31,961 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>906,906</strong> </td><td>237,568 </td><td>802,594 </td><td>201,366 </td><td>544,742 </td><td>208,223 </td></tr><tr><td colspan="7">SRS017227 </td></tr><tr><td> Maximum length (bp) </td><td><strong>478,428</strong> </td><td>108,476 </td><td>372,927 </td><td>227,256 </td><td>199,208 </td><td>217,259 </td></tr><tr><td> Total scaffold length (bp) </td><td>370,496,571 </td><td>250,969,598 </td><td>349,934,212 </td><td><strong>395,257,497</strong> </td><td>273,595,801 </td><td>206,705,202 </td></tr><tr><td> Number of scaffold </td><td>602,463 </td><td>485,307 </td><td>282,097 </td><td>802,952 </td><td>536,708 </td><td>217,259 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>4,709,551</strong> </td><td>1,064,102 </td><td>4,010,530 </td><td>2,227,194 </td><td>2,501,039 </td><td>1,617,896 </td></tr><tr><td colspan="7">SRS018661 </td></tr><tr><td> Maximum length (bp) </td><td><strong>699,395</strong> </td><td>111,404 </td><td>511,735 </td><td>426,297 </td><td>274,042 </td><td>180,946 </td></tr><tr><td> Total scaffold length (bp) </td><td><strong>114,676,867</strong> </td><td>71,339,406 </td><td>109,507,232 </td><td>107,557,997 </td><td>75,351,327 </td><td>47,619,933 </td></tr><tr><td> Number of scaffold </td><td>284,036 </td><td>195,950 </td><td>109,860 </td><td>274,896 </td><td>212,267 </td><td>34,244 </td></tr><tr><td> AUC of N-len(x) </td><td><strong>1,945,258</strong> </td><td>253,138 </td><td>1,296,606 </td><td>848,798 </td><td>1,004,371 </td><td>611,636 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>Top performances are shown in bold. MetaVelvet-SL, MetaVelvet and SOAPdenovo2 set the <em>k</em>-mer size at 37 for the MH0006 and MH0047 data sets, 43 for the MH0012 data set and 51 for SRS017227 and SRS018661.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594414" target="_blank">View Large</a></div></div><div class="block-child-p">When the total scaffold lengths of two assemblies are quite different in the human gut microbial data sets, the naive use of N50 score is inadequate, because the longer total length decreases the N50 score. The generalized score N-len(<em>x</em>) is more appropriate for comparing scaffold integrity than the raw N50 score.<sup><a reveal-id="DSU041C2" data-open="DSU041C2" class="link link-ref link-reveal xref-bibr">2</a></sup> N-len(<em>x</em>) is defined by <a id="DSU041M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU041M1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">N-len(</mtext><mi xmlns="">x</mi><mrow xmlns=""><mo stretchy="false">)</mo><mo>=</mo><mrow><mo stretchy="false">|</mo></mrow></mrow><mrow xmlns=""><msub><mi>S</mi><mi>i</mi></msub></mrow><mrow xmlns=""><mo stretchy="false">|</mo></mrow><mspace width="thinmathspace" xmlns=""></mspace><mrow xmlns=""><mi mathvariant="normal">such</mi><mspace width="thinmathspace"></mspace><mi mathvariant="normal">that</mi><mspace width="thinmathspace"></mspace></mrow><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mspace width="thinmathspace"></mspace><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>i</mi></munderover><mrow xmlns=""><mrow><mo stretchy="false">|</mo></mrow><mrow><msub><mi>S</mi><mi>j</mi></msub></mrow><mrow><mo stretchy="false">|</mo></mrow><mo>≥</mo><mi>x</mi></mrow><mspace width="thinmathspace" xmlns=""></mspace><mtext xmlns="">and</mtext><mspace width="thinmathspace" xmlns=""></mspace><munderover xmlns=""><mo movablelimits="false">∑</mo><mrow><mspace width="thinmathspace"></mspace><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow></munderover><mrow xmlns=""><mrow><mo stretchy="false">|</mo></mrow><mrow><msub><mi>S</mi><mi>j</mi></msub></mrow><mrow><mo stretchy="false">|</mo></mrow><mo>&lt;</mo><mi>x</mi></mrow><mo xmlns="">,</mo></math></div></div><div class="label">(1)</div></div> where <em>S<sub>1</sub>, S<sub>2</sub>, … , S<sub>n</sub></em> denote the list of scaffolds in descending order of length as output by an assembler. The N50 score corresponds to the N-len(<em>x</em>) score for <em>x</em> = <em>L</em>/2 (<em>x</em> is 50% of <em>L</em>), where <em>L</em> denotes the total scaffold length. The N-len(<em>x</em>) plots for the MH0006 data sets produced by MetaVelvet-SL, MetaVelvet, IDBA-UD, SOAPdenovo2, Ray Meta and Omega are shown in Fig. <a reveal-id="DSU041F4" data-open="DSU041F4" class="link link-reveal link-table xref-fig">4</a>. MetaVelvet-SL significantly increased the scaffold integrity. For example, when <em>x</em> = 5,000,000, the N-len(<em>x</em>) score of MetaVelvet-SL was 306,496, the N-len(<em>x</em>) score of MetaVelvet was 24,554, the N-len(<em>x</em>) score of IDBA-UD was 178,659, the N-len(<em>x</em>) score of SOAPdenovo2 was 90,861, the N-len(<em>x</em>) score of Ray Meta was 101,726 and the N-len(<em>x</em>) score of Omega was 117,010. (The N-len(<em>x</em>) plots for the MH0012, MH0047, SRS017227, and SRS018661 data sets are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S1–S4</a>.) As in the MetaVelvet paper, we calculated the area under the curve (AUC) of N-len(<em>x</em>) for 0 <em>&lt; x ≤ L</em> in units of 1,000,000 bp; that is, the cumulative sum of N-len(<em>x</em>) scores (0 <em>&lt; x ≤ L</em>), where <em>L</em> denotes the total scaffold length. </div>                        <a id="6594418" scrollto-destination="6594418"></a>
<div id="DSU041F4" data-id="DSU041F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04104.png?Expires=2147483647&Signature=PL4tO17hM6UwfuESv5tfWjX-VI7YLLMvw3fJEHNLYxJkGEzjq3nO-Rzks5xXQQJ84RdvJcUHlebOHRU1YdeLezivJY0H4JC2sEE1Hp0oZTp3xp5kGMQSUQaT2NEbzXgQi4mcH5bgKWtnAQUkAm1hz1FWRaCNCeFIq0YUU8ghl07J8Fk76f7a0qzpgTB0TRxYtvqr7Grm5bi7ioECEpTUytgVpEMXdinF-iGjU2Q0Ja38Sy1clPbFBr0~DawnUS31DJjs0J9LW3jG6bpMIfSG6iYTQLYaQqYkdvWtoU8qNZE~nOQIbXH6o3tYH10BCFmt2735SOK8UDqH8SBaXvfeGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The N-len(x) plots for the MH0006 data set of human gut microbial data." path-from-xml="dsu04104.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594418/dsu04104.tif" path-from-xml="dsu04104.tif" target="_blank">View large</a><a section="6594418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04104.png?Expires=2147483647&Signature=JOzHXKfS74l9cR83v7p60oIXsBg1zeI0Y6WPMvbL~v5fMKqbNrkxXq~R9U-KcE-sr46BSBg5E4e-qdzEWD9now1rOu0RGXXWGhqwREVbcS1zELJojxkk953674k2uJaXzqlmEnZBOC8roNkLnLHxmR9UKhZ5~Q~gnjdgvzKuiIqzUu7ulRwnOJTmhCXbbRnIQx5rE173fXlCfLkGbSPcwSO5eKIsln-8VlGM-CS6BeJWa8DzfbF56Rujm8N5ZoOWfVwLu90AChFqeIWX4F3iW7RUjSxCfb0SSbcci5E-FvqVuU9qn9vKF1z4k-6kE4BHiRY8JN7zg5LyVzN6ZGXy9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594418&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04104.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The N-len(x) plots for the MH0006 data set of human gut microbial data.</p></div></div><div content-id="DSU041F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/m_dsu04104.png?Expires=2147483647&Signature=PL4tO17hM6UwfuESv5tfWjX-VI7YLLMvw3fJEHNLYxJkGEzjq3nO-Rzks5xXQQJ84RdvJcUHlebOHRU1YdeLezivJY0H4JC2sEE1Hp0oZTp3xp5kGMQSUQaT2NEbzXgQi4mcH5bgKWtnAQUkAm1hz1FWRaCNCeFIq0YUU8ghl07J8Fk76f7a0qzpgTB0TRxYtvqr7Grm5bi7ioECEpTUytgVpEMXdinF-iGjU2Q0Ja38Sy1clPbFBr0~DawnUS31DJjs0J9LW3jG6bpMIfSG6iYTQLYaQqYkdvWtoU8qNZE~nOQIbXH6o3tYH10BCFmt2735SOK8UDqH8SBaXvfeGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The N-len(x) plots for the MH0006 data set of human gut microbial data." path-from-xml="dsu04104.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6594418/dsu04104.tif" path-from-xml="dsu04104.tif" target="_blank">View large</a><a section="6594418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu04104.png?Expires=2147483647&Signature=JOzHXKfS74l9cR83v7p60oIXsBg1zeI0Y6WPMvbL~v5fMKqbNrkxXq~R9U-KcE-sr46BSBg5E4e-qdzEWD9now1rOu0RGXXWGhqwREVbcS1zELJojxkk953674k2uJaXzqlmEnZBOC8roNkLnLHxmR9UKhZ5~Q~gnjdgvzKuiIqzUu7ulRwnOJTmhCXbbRnIQx5rE173fXlCfLkGbSPcwSO5eKIsln-8VlGM-CS6BeJWa8DzfbF56Rujm8N5ZoOWfVwLu90AChFqeIWX4F3iW7RUjSxCfb0SSbcci5E-FvqVuU9qn9vKF1z4k-6kE4BHiRY8JN7zg5LyVzN6ZGXy9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6594418&ar=441516&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04104.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The N-len(x) plots for the MH0006 data set of human gut microbial data.</p></div></div><p>MetaVelvet-SL generated much longer accurate scaffolds than MetaVelvet, IDBA-UD, Ray Meta, Omega and SOAPdenovo2, showing that MetaVelvet-SL improved scaffold integrity. MetaVelvet-SL outperformed MetaVelvet, IDBA-UD, Ray Meta, Omega and SOAPdenovo2 for all data sets in terms of all three of the performance indicators (total length of scaffolds, maximum length of scaffolds and AUC), except for the SRS017227 data sets, SOAPdenovo2 generated slightly longer total length of scaffolds. The identification of chimeric nodes by MetaVelvet-SL using classification models generated from the taxonomic profile inferred by MetaPhlAn is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S21</a>.</p><p>Comparisons between the taxonomic profile predicted by MetaPhlAn and the taxonomic profile based on assembly results of MetaVelvet-SL using BLAST were accomplished. The NCBI genomic reference sequences were used, which provide stable references, as the database for BLAST searching. The database contains 41,913 organisms as of September 2014 (Release 67). The numbers of species predicted by MetaPhlAn and predicted from the assembly of MetaVelvet-SL are shown in Table <a reveal-id="DSU041TB6" data-open="DSU041TB6" class="link link-reveal link-table xref-fig">6</a>. The taxonomic profile based on assembly results covered &gt;90% of the taxonomic profile predicted by MetaPhlAn. This result indicated that the assembly capacity of MetaVelvet-SL was high enough to capture the target diverse microbial community. As shown in Table <a reveal-id="DSU041TB6" data-open="DSU041TB6" class="link link-reveal link-table xref-fig">6</a>, much larger number of species was predicted from assembly results by BLAST than predicted by MetaPhlAn. This is mainly because MetaPhlAn used 2,887 genomes available from the Integrated Microbial Genomes (IMG) system, which were much fewer than the number of organisms in the NCBI database used in BLAST searching. The taxonomic profile predicted by MetaPhlAn for each real data set is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S22–S26</a>, whereas the taxonomy profile predicted from the assembly of MetaVelvet-SL using BLAST is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S27–S31</a>. </p>                        <a id="6594425" scrollto-destination="6594425"></a>
<div content-id="DSU041TB6" class="table-modal"><div class="table-wrap"><div id="DSU041TB6" class="table-wrap-title" data-id="DSU041TB6"><div class="label -label">Table 6.</div><div class="caption"><p>The number of species in the taxonomic profile predicted by MetaPhlAn and the taxonomic profile based on assembly results of MetaVelvet-SL using BLAST</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of species predicted by both </th><th>Number of species predicted only by MetaPhlAn </th><th>Number of species predicted only by assembly </th></tr></thead><tbody><tr><td>MH0006 (ERS006497) </td><td>99 </td><td>5 </td><td>2,932 </td></tr><tr><td>MH0012 (ERS006494) </td><td>124 </td><td>9 </td><td>2,872 </td></tr><tr><td>MH0047 (ERS006592) </td><td>65 </td><td>2 </td><td>2,137 </td></tr><tr><td>SRS017227 </td><td>83 </td><td>3 </td><td>2,992 </td></tr><tr><td>SRS018661 </td><td>81 </td><td>8 </td><td>1,529 </td></tr></tbody></table></div><div content-id="DSU041TB6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of species predicted by both </th><th>Number of species predicted only by MetaPhlAn </th><th>Number of species predicted only by assembly </th></tr></thead><tbody><tr><td>MH0006 (ERS006497) </td><td>99 </td><td>5 </td><td>2,932 </td></tr><tr><td>MH0012 (ERS006494) </td><td>124 </td><td>9 </td><td>2,872 </td></tr><tr><td>MH0047 (ERS006592) </td><td>65 </td><td>2 </td><td>2,137 </td></tr><tr><td>SRS017227 </td><td>83 </td><td>3 </td><td>2,992 </td></tr><tr><td>SRS018661 </td><td>81 </td><td>8 </td><td>1,529 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN4"><div class="fn"><p>The first column represents the number of species predicted by MetaPhlAn and predicted from assembly results by BLAST (intersection). The second column represents the number of species only predicted by MetaPhlAn and not predicted from assembly results by BLAST. The third column represents the number of species only predicted from assembly results by BLAST and not predicted by MetaPhlAn.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594425" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU041TB6" class="table-wrap-title" data-id="DSU041TB6"><div class="label -label">Table 6.</div><div class="caption"><p>The number of species in the taxonomic profile predicted by MetaPhlAn and the taxonomic profile based on assembly results of MetaVelvet-SL using BLAST</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of species predicted by both </th><th>Number of species predicted only by MetaPhlAn </th><th>Number of species predicted only by assembly </th></tr></thead><tbody><tr><td>MH0006 (ERS006497) </td><td>99 </td><td>5 </td><td>2,932 </td></tr><tr><td>MH0012 (ERS006494) </td><td>124 </td><td>9 </td><td>2,872 </td></tr><tr><td>MH0047 (ERS006592) </td><td>65 </td><td>2 </td><td>2,137 </td></tr><tr><td>SRS017227 </td><td>83 </td><td>3 </td><td>2,992 </td></tr><tr><td>SRS018661 </td><td>81 </td><td>8 </td><td>1,529 </td></tr></tbody></table></div><div content-id="DSU041TB6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of species predicted by both </th><th>Number of species predicted only by MetaPhlAn </th><th>Number of species predicted only by assembly </th></tr></thead><tbody><tr><td>MH0006 (ERS006497) </td><td>99 </td><td>5 </td><td>2,932 </td></tr><tr><td>MH0012 (ERS006494) </td><td>124 </td><td>9 </td><td>2,872 </td></tr><tr><td>MH0047 (ERS006592) </td><td>65 </td><td>2 </td><td>2,137 </td></tr><tr><td>SRS017227 </td><td>83 </td><td>3 </td><td>2,992 </td></tr><tr><td>SRS018661 </td><td>81 </td><td>8 </td><td>1,529 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN4"><div class="fn"><p>The first column represents the number of species predicted by MetaPhlAn and predicted from assembly results by BLAST (intersection). The second column represents the number of species only predicted by MetaPhlAn and not predicted from assembly results by BLAST. The third column represents the number of species only predicted from assembly results by BLAST and not predicted by MetaPhlAn.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6594425" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6594426 id="6594426" class="section-title" data-legacy-id=s3c>3.3. Conclusion</h3>
<p>Extensive experiments on simulated and real metagenomic data sets showed that MetaVelvet-SL outperformed other metagenomic assemblers MetaVelvet, IDBA-UD, Ray Meta and Omega as well as a single-genome assembler, SOAPdenovo2.</p><p>The main strategy in MetaVelvet-SL is to develop a model to classify a candidate node at a crossing point between two incoming and two outgoing edges as a chimeric node or not. We also developed a procedure to identify unique nodes more precisely based on the expected coverage for each sub-graph and considered very low-coverage nodes by determining an appropriate threshold to remove error nodes. Since MetaVelvet-SL needs to learn a model for the classification of chimeric nodes, we have provided a pipeline connecting MetaPhlAn and MetaVelvet-SL, which can generate a classification model and assemble automatically. MetaVelvet-SL also provides a library of pre-trained classification models for several typical environments such as soil, deep sea, mud, human blood, intestine and mouth.</p><p>MetaVelvet-SL defines a chimeric node as a node that has two incoming edges and two outgoing edges. In a de Bruijn graph, in single-genome assembly, a node with multiple incoming and outgoing edges represents a repeat node. In multiple genome assembly, such node is not necessarily a repeat since it is sometimes shared between the genomes of two closely related species and represents orthologous sequences, conserved sequences (such as rRNA sequences) or horizontal transfer sequences. In a de Bruijn graph, nodes having multiple incoming and outgoing edges can be divided into (i) those with two incoming edges and two outgoing edges and (ii) those with higher order connectivity (i.e. more than two incoming edges and more than two outgoing edges). The number of nodes having higher order connectivity is much fewer than the number of nodes having two incoming edges and two outgoing edges. We counted the number of nodes having multiple incoming and outgoing edges in de Bruijn graphs for both simulated data sets and real data sets of human gut microbial short read data. On average, the number of nodes having higher order connectivity is only 1.79% of the number of nodes having two incoming edges and two outgoing edges. The number of nodes for each data set is provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S32</a>. Although MetaVelvet-SL defines a candidate for a chimeric node as a node that has two incoming edges and two outgoing edges, MetaVelvet-SL outperformed the other metagenomic assemblers MetaVelvet, IDBA-UD, Ray Meta and Omega, and also a single genome assembler, SOAPdenovo2. We continue to consider the impact of higher order connectivity.</p>                            <h2 scrollto-destination=6594430 id="6594430" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6594432 id="6594432" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a Grant-in-Aid for Scientific Research in Innovative Areas No. 221S0002 and Scientific Research (A) No. 23241066 from the Ministry of Education, Culture, Sports, Science and Technology of Japan. Funding to pay the Open Access publication charges for this article was provided by Grant-in-Aid for Scientific Research in Innovative Areas No. 221S0002 from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=6594434 id="6594434" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Dr Nicola Segata for his permission to use MetaPhlAn as a part of the pipeline connecting MetaVelvet-SL and MetaPhlAn.</p>                            <h2 scrollto-destination=6594436 id="6594436" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU041C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scholz</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Lo</div> <div class="given-names">C.C.</div></div>,  <div class="name"><div class="surname">Chain</div> <div class="given-names">P.S.</div></div>. </div><div class="article-title">Next generation sequencing and bioinformatic bottlenecks: the current state of metagenomic data analysis</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">9</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20and%20bioinformatic%20bottlenecks%3A%20the%20current%20state%20of%20metagenomic%20data%20analysis&amp;author=M.B.%20Scholz&amp;author=C.C.%20Lo&amp;author=P.S.%20Chain&amp;publication_year=2012&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=23&amp;pages=9-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2011.11.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2011.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2011.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22154470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Namiki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hachiya</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sakakibara</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Metavelvet: an extension of velvet assembler to de novo metagenome assembly from short sequence reads</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> pg. <div class="fpage">e155</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metavelvet%3A%20an%20extension%20of%20velvet%20assembler%20to%20de%20novo%20metagenome%20assembly%20from%20short%20sequence%20reads&amp;author=T.%20Namiki&amp;author=T.%20Hachiya&amp;author=H.%20Tanaka&amp;author=Y.%20Sakakibara&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks678" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks678" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks678"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22821567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Bioinformatics for whole-genome shotgun sequencing of microbial communities</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">1</div> pg. <div class="fpage">e24</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20for%20whole-genome%20shotgun%20sequencing%20of%20microbial%20communities&amp;author=K.%20Chen&amp;author=L.%20Pachter&amp;publication_year=2005&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=1&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0010024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0010024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0010024"> </span></div> </div></div></div></div></div><div content-id="DSU041C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lai</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ding</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A de novo metagenomic assembly program for shotgun dna reads</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1455</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20de%20novo%20metagenomic%20assembly%20program%20for%20shotgun%20dna%20reads&amp;author=B.%20Lai&amp;author=R.%20Ding&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1455-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laserson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jojic</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Koller</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Genovo: de novo assembly for metagenomes</div>, <div class="source">J. Comput. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">429</div>-<div class="lpage">43</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genovo%3A%20de%20novo%20assembly%20for%20metagenomes&amp;author=J.%20Laserson&amp;author=V.%20Jojic&amp;author=D.%20Koller&amp;publication_year=2011&amp;journal=J.%20Comput.%20Biol.&amp;volume=18&amp;pages=429-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2010.0244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2010.0244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2010.0244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21385045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagarajan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Sequence assembly demystified</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">157</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20assembly%20demystified&amp;author=N.%20Nagarajan&amp;author=M.%20Pop&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=14&amp;pages=157-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23358380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Leung</div> <div class="given-names">H.C.M.</div></div>,  <div class="name"><div class="surname">Yiu</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">Meta-IDBA: a de novo assembler for metagenomic data</div>, <div class="source">Bioinformatics</div> , <div class="year">2011</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">i94</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meta-IDBA%3A%20a%20de%20novo%20assembler%20for%20metagenomic%20data&amp;author=Y.%20Peng&amp;author=H.C.M.%20Leung&amp;author=S.M.%20Yiu&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=i94-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21685107" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Afiahayati</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sakakibara</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">An extended genovo metagenomic assembler by incorporating paired-end information</div>, <div class="source">PeerJ</div> , <div class="year">2013</div>, vol. <div class="volume">1</div> pg. <div class="fpage">e196</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20extended%20genovo%20metagenomic%20assembler%20by%20incorporating%20paired-end%20information&amp;author=A.%20Afiahayati&amp;author=K.%20Sato&amp;author=Y.%20Sakakibara&amp;publication_year=2013&amp;journal=PeerJ&amp;volume=1&amp;pages=e196" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7717/peerj.196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7717%2Fpeerj.196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7717%2Fpeerj.196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24281688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Leung</div> <div class="given-names">H.C.M.</div></div>,  <div class="name"><div class="surname">Yiu</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1420</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IDBA-UD%3A%20a%20de%20novo%20assembler%20for%20single-cell%20and%20metagenomic%20sequencing%20data%20with%20highly%20uneven%20depth&amp;author=Y.%20Peng&amp;author=H.C.M.%20Leung&amp;author=S.M.%20Yiu&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1420-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boisvert</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Raymond</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Godzaridis</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Ray Meta: scalable de novo metagenome assembly and profiling</div>, <div class="source">Genome Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">r22</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ray%20Meta%3A%20scalable%20de%20novo%20metagenome%20assembly%20and%20profiling&amp;author=S.%20Boisvert&amp;author=F.%20Raymond&amp;author=E.%20Godzaridis&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=r22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-12-r122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-12-r122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-12-r122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22452995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haider</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ahn</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Bushnell</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Omega: an overlap-graph de novo assembler for metagenomics</div>, <div class="source">Bioinformatics</div> , <div class="year">2014</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2717</div>-<div class="lpage">2722</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btu395" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btu395</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Omega%3A%20an%20overlap-graph%20de%20novo%20assembler%20for%20metagenomics&amp;author=B.%20Haider&amp;author=T.%20Ahn&amp;author=B.%20Bushnell&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2717-2722" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu395" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu395" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu395"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24947750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">McEwen</div> <div class="given-names">G.K.</div></div>,  <div class="name"><div class="surname">Margulies</div> <div class="given-names">E.H.</div></div>, et al. </div><div class="article-title">Pebble and Rock band: heuristic resolution of repeats and scaffolding in the velvet short-read de novo assembler</div>, <div class="source">PLoS ONE</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e8407</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pebble%20and%20Rock%20band%3A%20heuristic%20resolution%20of%20repeats%20and%20scaffolding%20in%20the%20velvet%20short-read%20de%20novo%20assembler&amp;author=D.R.%20Zerbino&amp;author=G.K.%20McEwen&amp;author=E.H.%20Margulies&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e8407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0008407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0008407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0008407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20027311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</div>, <div class="source">GigaScience</div> , <div class="year">2012</div>, vol. <div class="volume">1</div> pg. <div class="fpage">18</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=GigaScience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>, et al. </div><div class="article-title">A whole-genome assembly of Drosophila</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">2196</div>-<div class="lpage">204</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20whole-genome%20assembly%20of%20Drosophila&amp;author=E.W.%20Myers&amp;author=G.G.%20Sutton&amp;author=A.L.%20Delcher&amp;publication_year=2009&amp;journal=Science&amp;volume=287&amp;pages=2196-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5461.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5461.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5461.2196"> </span></div> </div></div></div></div></div><div content-id="DSU041C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Segata</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Waldron</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Ballarini</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Metagenomic microbial community profiling using unique clade-specific marker genes</div>, <div class="source">Nat. Methods</div> , <div class="year">2012</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">811</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20microbial%20community%20profiling%20using%20unique%20clade-specific%20marker%20genes&amp;author=N.%20Segata&amp;author=L.%20Waldron&amp;author=A.%20Ballarini&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=811-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2066" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2066" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2066"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22688413" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">A human gut microbial gene catalogue established by metagenomic sequencing</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">464</div> (pg. <div class="fpage">59</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;author=J.%20Qin&amp;author=R.%20Li&amp;author=J.%20Raes&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=59-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="collab">Consortium THMP</div><div class="article-title">Structure, function and diversity of the healthy human microbiome</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">486</div> (pg. <div class="fpage">207</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">C.C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">LIBSVM: a library for support vector machines</div>, <div class="source">ACM Trans. Intell. Syst. Technol.</div> , <div class="year">2011</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LIBSVM%3A%20a%20library%20for%20support%20vector%20machines&amp;author=C.C.%20Chang&amp;author=C.J.%20Lin&amp;publication_year=2011&amp;journal=ACM%20Trans.%20Intell.%20Syst.%20Technol.&amp;volume=2&amp;pages=1-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1145/1961189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1145%2F1961189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1145%2F1961189"> </span></div> </div></div></div></div></div><div content-id="DSU041C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Unterseher</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jumpponen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Opik</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Species abundance distributions and richness estimations in fungal metagenomics lessons learned from community ecology</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">275</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20abundance%20distributions%20and%20richness%20estimations%20in%20fungal%20metagenomics%20lessons%20learned%20from%20community%20ecology&amp;author=M.%20Unterseher&amp;author=A.%20Jumpponen&amp;author=M.%20Opik&amp;publication_year=2011&amp;journal=Mol.%20Ecol.&amp;volume=20&amp;pages=275-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.20.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.20.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.20.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21155911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU041C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU041C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brodersen</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Ong</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Stephan</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Buhmann</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">The balanced accuracy and its posterior distribution</div>, <div class="year">2010</div><div class="conf-name">Proceedings of the 20th International Conference on Pattern Recognition</div>(pg. <div class="fpage">3121</div>-<div class="lpage">4</div>) <div class="comment">IEEE, Washington, DC, USA</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=441516&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu041/2/dsu041-dsu041supp.pdf?Expires=2147483647&amp;Signature=injjb77U3Ge6zwP7FCJSy~IkZP0btSWgAapF35rwqOQNT8MjX65MyhYSyL-jqlhY2jeFoXwWaZIItx2DmQIecOA3RHNK399u5akqbiN0cHmOM1rNLGpkzj95gy2s~q8ExGmn1gZGqW0pOsy6o~pFrQtL0FV8HeO8HzP5CoYLCYmvx55SENjw75lgiFuYueNn2UyZJeu9OQH1cw8tbKqBubzI4mU7o9UcYJh~noMB3kCYO-A11coHm6NOkjOnNB3uU9BWdlHpdCijTCPQHseECZqL8Zip20sf2sh46K32p-0IxMVu1zP8cwcv2RKK0~0uj6MQ82hWktKJArFDIdnHPg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">847</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">28</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu041"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="441516"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f69%2f441516">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (28)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F69%2F441516" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/441516'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="MetaVelvet-SL: an extension of the Velvet assembler to a de novo metagenomic assembler utilizing supervised learning" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu041");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f69%2f441516">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

