

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/13/2754540","@type":"ScholarlyArticle","name":"Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging","datePublished":"2014-10-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu034","keywords":["DNA methylation","target enrichment","massively parallel sequencing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/13/2754540","author":[{"name":"Miura, Fumihito","affiliation":"1  Department of Biophysics and Biochemistry ,  Graduate School of Science, University of Tokyo ,  Bunkyo-ku, Tokyo 113-0033 ,  Japan    2  Core Research for Evolutional Science and Technology (CREST) ,  Japan Science and Technology Agency (JST) ,  Higashi-ku, Fukuoka 812-8582 ,  Japan    3  Department of Biochemistry ,  Kyushu University Graduate School of Medical Sciences ,  Higashi-ku, Fukuoka 812-8582 ,  Japan","@type":"Person"},{"name":"Ito, Takashi","affiliation":"1  Department of Biophysics and Biochemistry ,  Graduate School of Science, University of Tokyo ,  Bunkyo-ku, Tokyo 113-0033 ,  Japan    2  Core Research for Evolutional Science and Technology (CREST) ,  Japan Science and Technology Agency (JST) ,  Higashi-ku, Fukuoka 812-8582 ,  Japan    3  Department of Biochemistry ,  Kyushu University Graduate School of Medical Sciences ,  Higashi-ku, Fukuoka 812-8582 ,  Japan","@type":"Person"}],"description":"The current gold standard method for methylome analysis is whole-genome bisulfite sequencing (WGBS), but its cost is substantial, especially for the purpose of ","pageStart":"13","pageEnd":"18","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging" />
    <meta property="og:description" content="The current gold standard method for methylome analysis is whole-genome bisulfite sequencing (WGBS), but its cost is substantial, especially for the purpose of " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/13/2754540" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-10-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Miura, Fumihito" /><meta name="citation_author_institution" content="Department of Biophysics and Biochemistry, Graduate School of Science, University of Tokyo, Bunkyo-ku, Tokyo 113-0033, Japan" /><meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Agency (JST), Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author_institution" content="Department of Biochemistry, Kyushu University Graduate School of Medical Sciences, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Ito, Takashi" /><meta name="citation_author_institution" content="Department of Biophysics and Biochemistry, Graduate School of Science, University of Tokyo, Bunkyo-ku, Tokyo 113-0033, Japan" /><meta name="citation_author_institution" content="Core Research for Evolutional Science and Technology (CREST), Japan Science and Technology Agency (JST), Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author_institution" content="Department of Biochemistry, Kyushu University Graduate School of Medical Sciences, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_title" content="Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging" /><meta name="citation_firstpage" content="13" /><meta name="citation_lastpage" content="18" /><meta name="citation_doi" content="10.1093/dnares/dsu034" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=452;  citation_pages=215-9; " /><meta name="citation_reference" content="citation_title=Highly integrated single-base resolution maps of the epigenome in Arabidopsis;  citation_journal_title=Cell;  citation_year=2008;  citation_volume=133;  citation_pages=523-36; " /><meta name="citation_reference" content="citation_title=Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5868-77; " /><meta name="citation_reference" content="citation_title=Preparation of reduced representation bisulfite sequencing libraries for genome-scale DNA methylation profiling;  citation_journal_title=Nat. Protoc.;  citation_year=2011;  citation_volume=6;  citation_pages=468-81; " /><meta name="citation_reference" content="citation_title=Targeted bisulfite sequencing by solution hybrid selection and massively parallel sequencing;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e127" /><meta name="citation_reference" content="citation_title=High resolution profiling of human exon methylation by liquid hybridization capture-based bisulfite sequencing;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=597" /><meta name="citation_reference" content="citation_title=Analyzing the cancer methylome through targeted bisulfite sequencing;  citation_journal_title=Cancer Lett.;  citation_year=2013;  citation_volume=340;  citation_pages=171-8; " /><meta name="citation_reference" content="citation_title=Amplification-free whole-genome bisulfite sequencing by post-bisulfite adaptor tagging;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e136" /><meta name="citation_reference" content="citation_title=Ultra-low-input, tagmentation-based whole-genome bisulfite sequencing;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=1139-43; " /><meta name="citation_reference" content="citation_title=Contribution of intragenic DNA methylation in mouse gametic DNA methylomes to establish oocyte-specific heritable marks;  citation_journal_title=PLoS Genet.;  citation_year=2012;  citation_volume=8;  citation_pages=e1002440" /><meta name="citation_reference" content="citation_title=Mouse oocyte methylomes at base resolution reveal genome-wide accumulation of non-CpG methylation and role of DNA methyltransferases;  citation_journal_title=PLoS Genet.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003439" /><meta name="citation_reference" content="citation_title=High resolution DNA methylome analysis of primordial germ cells identifies gender-specific reprogramming in mice;  citation_journal_title=Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=616-27; " /><meta name="citation_reference" content="citation_title=Single-cell genome-wide bisulfite sequencing for assessing epigenetic heterogeneity;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=817-20; " /><meta name="citation_reference" content="citation_title=PBAT: post-bisulfite adaptor tagging for highly sensitive whole-genome bisulfite sequencing;  citation_journal_title=Methods Mol. Biol.;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution;  citation_journal_title=Science;  citation_year=2012;  citation_volume=336;  citation_pages=934-7; " /><meta name="citation_reference" content="citation_title=Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=149;  citation_pages=1368-80; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/13/17462770/dsu034.pdf" /><meta name="description" content="The current gold standard method for methylome analysis is whole-genome bisulfite sequencing (WGBS), but its cost is substantial, especially for the purpose of " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/13/2754540" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/13/2754540" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/13/2754540");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu034';
    </script>
<title>
	Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754540" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcExPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXVyYSwgRnVtaWhpdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3BoeXNpY3MgYW5kIEJpb2NoZW1pc3RyeSwgR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgVG9reW8sIEJ1bmt5by1rdSwgVG9reW8gMTEzLTAwMzMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29yZSBSZXNlYXJjaCBmb3IgRXZvbHV0aW9uYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoQ1JFU1QpLCBKYXBhbiBTY2llbmNlIGFuZCBUZWNobm9sb2d5IEFnZW5jeSAoSlNUKSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgS3l1c2h1IFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgU2NpZW5jZXMsIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9waHlzaWNzIGFuZCBCaW9jaGVtaXN0cnksIEdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCBVbml2ZXJzaXR5IG9mIFRva3lvLCBCdW5reW8ta3UsIFRva3lvIDExMy0wMDMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvcmUgUmVzZWFyY2ggZm9yIEV2b2x1dGlvbmFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKENSRVNUKSwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3kgKEpTVCksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9jaGVtaXN0cnksIEt5dXNodSBVbml2ZXJzaXR5IEdyYWR1YXRlIFNjaG9vbCBvZiBNZWRpY2FsIFNjaWVuY2VzLCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGlnaGx5IHNlbnNpdGl2ZSB0YXJnZXRlZCBtZXRoeWxvbWUgc2VxdWVuY2luZyBieSBwb3N0LWJpc3VsZml0ZSBhZGFwdG9yIHRhZ2dpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNob3RndW4gYmlzdWxwaGl0ZSBzZXF1ZW5jaW5nIG9mIHRoZSBBcmFiaWRvcHNpcyBnZW5vbWUgcmV2ZWFscyBETkEgbWV0aHlsYXRpb24gcGF0dGVybmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MjsgIGNpdGF0aW9uX3BhZ2VzPTIxNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgaW50ZWdyYXRlZCBzaW5nbGUtYmFzZSByZXNvbHV0aW9uIG1hcHMgb2YgdGhlIGVwaWdlbm9tZSBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzM7ICBjaXRhdGlvbl9wYWdlcz01MjMtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgcmVwcmVzZW50YXRpb24gYmlzdWxmaXRlIHNlcXVlbmNpbmcgZm9yIGNvbXBhcmF0aXZlIGhpZ2gtcmVzb2x1dGlvbiBETkEgbWV0aHlsYXRpb24gYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NjgtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZXBhcmF0aW9uIG9mIHJlZHVjZWQgcmVwcmVzZW50YXRpb24gYmlzdWxmaXRlIHNlcXVlbmNpbmcgbGlicmFyaWVzIGZvciBnZW5vbWUtc2NhbGUgRE5BIG1ldGh5bGF0aW9uIHByb2ZpbGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00NjgtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIGJpc3VsZml0ZSBzZXF1ZW5jaW5nIGJ5IHNvbHV0aW9uIGh5YnJpZCBzZWxlY3Rpb24gYW5kIG1hc3NpdmVseSBwYXJhbGxlbCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1lMTI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCByZXNvbHV0aW9uIHByb2ZpbGluZyBvZiBodW1hbiBleG9uIG1ldGh5bGF0aW9uIGJ5IGxpcXVpZCBoeWJyaWRpemF0aW9uIGNhcHR1cmUtYmFzZWQgYmlzdWxmaXRlIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTU5NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5emluZyB0aGUgY2FuY2VyIG1ldGh5bG9tZSB0aHJvdWdoIHRhcmdldGVkIGJpc3VsZml0ZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DYW5jZXIgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzQwOyAgY2l0YXRpb25fcGFnZXM9MTcxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtcGxpZmljYXRpb24tZnJlZSB3aG9sZS1nZW5vbWUgYmlzdWxmaXRlIHNlcXVlbmNpbmcgYnkgcG9zdC1iaXN1bGZpdGUgYWRhcHRvciB0YWdnaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1lMTM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmEtbG93LWlucHV0LCB0YWdtZW50YXRpb24tYmFzZWQgd2hvbGUtZ2Vub21lIGJpc3VsZml0ZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTExMzktNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyaWJ1dGlvbiBvZiBpbnRyYWdlbmljIEROQSBtZXRoeWxhdGlvbiBpbiBtb3VzZSBnYW1ldGljIEROQSBtZXRoeWxvbWVzIHRvIGVzdGFibGlzaCBvb2N5dGUtc3BlY2lmaWMgaGVyaXRhYmxlIG1hcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI0NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3VzZSBvb2N5dGUgbWV0aHlsb21lcyBhdCBiYXNlIHJlc29sdXRpb24gcmV2ZWFsIGdlbm9tZS13aWRlIGFjY3VtdWxhdGlvbiBvZiBub24tQ3BHIG1ldGh5bGF0aW9uIGFuZCByb2xlIG9mIEROQSBtZXRoeWx0cmFuc2ZlcmFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzQzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggcmVzb2x1dGlvbiBETkEgbWV0aHlsb21lIGFuYWx5c2lzIG9mIHByaW1vcmRpYWwgZ2VybSBjZWxscyBpZGVudGlmaWVzIGdlbmRlci1zcGVjaWZpYyByZXByb2dyYW1taW5nIGluIG1pY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NjE2LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtY2VsbCBnZW5vbWUtd2lkZSBiaXN1bGZpdGUgc2VxdWVuY2luZyBmb3IgYXNzZXNzaW5nIGVwaWdlbmV0aWMgaGV0ZXJvZ2VuZWl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODE3LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQkFUOiBwb3N0LWJpc3VsZml0ZSBhZGFwdG9yIHRhZ2dpbmcgZm9yIGhpZ2hseSBzZW5zaXRpdmUgd2hvbGUtZ2Vub21lIGJpc3VsZml0ZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBzZXF1ZW5jaW5nIG9mIDUtbWV0aHlsY3l0b3NpbmUgYW5kIDUtaHlkcm94eW1ldGh5bGN5dG9zaW5lIGF0IHNpbmdsZS1iYXNlIHJlc29sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM2OyAgY2l0YXRpb25fcGFnZXM9OTM0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UtcmVzb2x1dGlvbiBhbmFseXNpcyBvZiA1LWh5ZHJveHltZXRoeWxjeXRvc2luZSBpbiB0aGUgbWFtbWFsaWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xMzY4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjIvMS8xMy8xNzQ2Mjc3MC9kc3UwMzQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJUaGUgY3VycmVudCBnb2xkIHN0YW5kYXJkIG1ldGhvZCBmb3IgbWV0aHlsb21lIGFuYWx5c2lzIGlzIHdob2xlLWdlbm9tZSBiaXN1bGZpdGUgc2VxdWVuY2luZyAoV0dCUyksIGJ1dCBpdHMgY29zdCBpcyBzdWJzdGFudGlhbCwgZXNwZWNpYWxseSBmb3IgdGhlIHB1cnBvc2Ugb2YgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNzU0NTQwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZHh2MqQfDmoQqIgcFj9DsyI4zGt/" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f13%2f2754540" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f13%2f2754540" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196371">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196371">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196376">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196376">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196388">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196388">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196397">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196397">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196402">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196402">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196404">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196404">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196406">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196406">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196408">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196408">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/1/2754537">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/19/439863">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumihito Miura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biophysics and Biochemistry</div>, <div class="insititution">Graduate School of Science, University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-0033</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Core Research for Evolutional Science and Technology (CREST)</div>, <div class="insititution">Japan Science and Technology Agency (JST)</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">Kyushu University Graduate School of Medical Sciences</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumihito+Miura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miura F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miura F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumihito Miura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Ito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biophysics and Biochemistry</div>, <div class="insititution">Graduate School of Science, University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-0033</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Core Research for Evolutional Science and Technology (CREST)</div>, <div class="insititution">Japan Science and Technology Agency (JST)</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">Kyushu University Graduate School of Medical Sciences</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 92-642-6095. Fax. +81 92-642-2103. Email: <a href="mailto:tito@med.kyushu-u.ac.jp" target="_blank">tito@med.kyushu-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Ito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ito T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ito T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Ito</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 1318, <a href='https://doi.org/10.1093/dnares/dsu034'>https://doi.org/10.1093/dnares/dsu034</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 October 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 August 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 September 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754540" href="/dnaresearch/article-pdf/22/1/13/17462770/dsu034.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Fumihito Miura, Takashi Ito;  Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 1318, <a href='https://doi.org/10.1093/dnares/dsu034'>https://doi.org/10.1093/dnares/dsu034</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754540?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Highly%20sensitive%20targeted%20methylome%20sequencing%20by%20post-bisulfite%20adaptor%20tagging&amp;publicationDate=2014-10-16&amp;volumeNum=22&amp;issueNum=1&amp;author=Miura%2C%20Fumihito%3B%20Ito%2C%20Takashi&amp;startPage=13&amp;endPage=18&amp;contentId=10.1093%2Fdnares%2Fdsu034&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f13%2f2754540" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>The current gold standard method for methylome analysis is whole-genome bisulfite sequencing (WGBS), but its cost is substantial, especially for the purpose of multi-sample comparison of large methylomes. Shotgun bisulfite sequencing of target-enriched DNA, or targeted methylome sequencing (TMS), can be a flexible, cost-effective alternative to WGBS. However, the current TMS protocol requires a considerable amount of input DNA and hence is hardly applicable to samples of limited quantity. Here we report a method to overcome this limitation by using post-bisulfite adaptor tagging (PBAT), in which adaptor tagging is conducted after bisulfite treatment to circumvent bisulfite-induced loss of intact sequencing templates, thereby enabling TMS of a 100-fold smaller amount of input DNA with far fewer cycles of polymerase chain reaction than in the current protocol. We thus expect that the PBAT-mediated TMS will serve as an invaluable method in epigenomics.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;target enrichment&quot;">target enrichment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;massively parallel sequencing&quot;">massively parallel sequencing</a></div>                            <h2 scrollto-destination=87196371 id="87196371" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Methylation occurring at the position 5 of cytosine residues in DNA is an epigenetic modification critically involved in the regulation of eukaryotic genomes. Accordingly, the genome-wide distribution of 5-methylcytosine residues, or the methylome, has been attracting intense attention from a wide audience in a variety of research disciplines. The recent advent of next-generation sequencing has revolutionized the way of interrogating the methylome: it has realized whole-genome bisulfite sequencing (WGBS) or genome-wide methylation analysis at single-base resolution.<sup><a reveal-id="DSU034C1" data-open="DSU034C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU034C2" data-open="DSU034C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The power of WGBS has been well demonstrated by many findings that would never have been achieved with other technologies. Although WGBS represents the gold standard for methylome analysis and is rapidly becoming the method of choice, its cost has remained substantial, thereby preventing it from being widely used for multi-sample comparison of large methylomes including those of mammals.</p><p>The most popular alternative to WGBS is reduced-representation bisulfite sequencing (RRBS), which efficiently enriches CpG-rich regions through restriction enzyme digestion to reduce the cost of sequencing, while maintaining deep coverage of a subset of CpG sites.<sup><a reveal-id="DSU034C3" data-open="DSU034C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Notably, RRBS can be applied to a minute amount of input DNA.<sup><a reveal-id="DSU034C4" data-open="DSU034C4" class="link link-ref link-reveal xref-bibr">4</a></sup> However, it cannot be used to examine particular regions of interest unless they are adequately flanked by the restriction enzyme sites. In this context, the shotgun bisulfite sequencing of subgenomic regions enriched using solution hybridization capture technology, referred to hereafter as targeted methylome sequencing (TMS), is ideal, because it can in principle target any unique genomic region.<sup><a reveal-id="DSU034C5" data-open="DSU034C5" class="link link-ref link-reveal xref-bibr">5</a><a reveal-id="DSU034C7" data-open="DSU034C7" class="link link-ref link-reveal xref-bibr">7</a></sup> However, all of the TMS protocols reported thus far require not only a considerable amount of input DNA (i.e. 3 g or more), but also a large number of polymerase chain reaction (PCR) cycles (i.e. 1020 cycles), making it difficult to apply TMS to samples of limited quantity.</p><p>We recently developed a highly efficient protocol for WGBS library construction termed post-bisulfite adaptor tagging (PBAT).<sup><a reveal-id="DSU034C8" data-open="DSU034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Although it is well known that bisulfite treatment destroys DNA, all of the conventional ligation-based WGBS protocols as well as the tagmentation-based one<sup><a reveal-id="DSU034C9" data-open="DSU034C9" class="link link-ref link-reveal xref-bibr">9</a></sup> treat adaptor-tagged library DNAs with bisulfite, inevitably leading to a considerable loss of intact sequencing template molecules. To circumvent this bisulfite-induced loss, we proposed the PBAT strategy in which adaptor tagging is performed after bisulfite treatment. This simple trick allowed us to prepare a PCR-free WGBS library from as little as 125 pg of input DNA.<sup><a reveal-id="DSU034C8" data-open="DSU034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> We routinely achieve a PCR-free, 30-fold coverage of mammalian methylomes from 30 ng of input DNA. Indeed, PBAT has been applied to mouse WGBS from 400 to 1,000 germinal vesicle-stage oocytes<sup><a reveal-id="DSU034C10" data-open="DSU034C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSU034C11" data-open="DSU034C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and a few thousand primordial germ cells,<sup><a reveal-id="DSU034C12" data-open="DSU034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> notably without any global PCR amplification. More recently, it has even been applied to single-cell genome-wide bisulfite sequencing with the aid of PCR.<sup><a reveal-id="DSU034C13" data-open="DSU034C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><div class="block-child-p">As all of the current TMS protocols include bisulfite treatment of adaptor-tagged library DNA, we reasoned that PBAT can improve the efficiency of TMS to develop a low-input protocol (Fig.<a reveal-id="DSU034F1" data-open="DSU034F1" class="link link-reveal link-table xref-fig">1</a>). We have indeed succeeded in the development of a highly efficient TMS method applicable to samples of limited quantity. <div data-id="DSU034F1" class="fig fig-section"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03401.jpeg?Expires=2147483647&Signature=S1i4ELFMdqBIY~KvBn4UaPugI08Dlq0a731T~3Jxq9o7q1kM9Dc2HBEOLEK2tNkKk6JGGFLNUXADOQPWFJblwlbywxrNcPYQCzzEmhMW31ueZXsw1Yr2tlCWSCSFBPxgHMPySWDkV4xk91SWEcwf3vAOpHS5WJUtzCyRYhZaoOnWCXr4nbhUzNScwFk-r3uN3NdQ4-pdbQTrdUZXWl2-~Y4Y8ALsU2sHcIv8Zg4h5~1nKbaSSOeDlNxdBjKXJOjeBZ-J7fw16e4wHK08gZgkhxUiTwcJexOouekVd1MwXjVixnqIr2ssTO8EiegO7ep4jWc-71YECkn1KeKdgpzJ~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two strategies for TMS. (A) Conventional procedures comprise adaptor tagging of fragmented genomic DNAs (Steps 1 and 2), target enrichment by hybridization (Steps 3 and 4) and bisulfite treatment of enriched library DNAs (Step 5) followed by PCR amplification (Step 6). The bisulfite treatment (Step 5) induces DNA breaks, inevitably leading to severe loss of intact sequencing template molecules. (B) PBAT-mediated procedure comprises target enrichment by hybridization (Steps 13), bisulfite treatment (Step 4) and adaptor tagging (Step 5), thereby circumventing the bisulfite-induced loss of intact sequencing template molecules." path-from-xml="dsu03401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196375/dsu03401.jpeg" path-from-xml="dsu03401.jpeg" target="_blank">View large</a><a section="87196375" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03401.jpeg?Expires=2147483647&Signature=UkDOvzu3YiOvVO-tS8WCETIZ50c5oSeis42ShBa1C0oy84vDhSgkSyChrAReL45EN3oPGkXn3ZJwdiHvf5VUET2adSwjFfzuW3l2ftWG4tTNB7-JuCKYNHbPhInoodayLU43p~mYI~53aEyeSMIs9APoboGJmUdkd~pNNQM2xaUWkI2PtbsW1jpXsNLMHzlV1ub37ITkXoTR-MGvdLytKrNaP9qg~vykMdsU44ZZQCdfhrW0SWo6T38ZWLt8ZmMCy6m8N8WU3L3FJcLZYIzF~hOFpRQKOR9~DacZISm9JUBCWJNThObZeUeAlNhodD4gks8spr~BT2aqR4ygHlXfUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196375&ar=2754540&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two strategies for TMS. (A) Conventional procedures comprise adaptor tagging of fragmented genomic DNAs (Steps 1 and 2), target enrichment by hybridization (Steps 3 and 4) and bisulfite treatment of enriched library DNAs (Step 5) followed by PCR amplification (Step 6). The bisulfite treatment (Step 5) induces DNA breaks, inevitably leading to severe loss of intact sequencing template molecules. (B) PBAT-mediated procedure comprises target enrichment by hybridization (Steps 13), bisulfite treatment (Step 4) and adaptor tagging (Step 5), thereby circumventing the bisulfite-induced loss of intact sequencing template molecules.</p></div></div><div content-id="DSU034F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03401.jpeg?Expires=2147483647&Signature=S1i4ELFMdqBIY~KvBn4UaPugI08Dlq0a731T~3Jxq9o7q1kM9Dc2HBEOLEK2tNkKk6JGGFLNUXADOQPWFJblwlbywxrNcPYQCzzEmhMW31ueZXsw1Yr2tlCWSCSFBPxgHMPySWDkV4xk91SWEcwf3vAOpHS5WJUtzCyRYhZaoOnWCXr4nbhUzNScwFk-r3uN3NdQ4-pdbQTrdUZXWl2-~Y4Y8ALsU2sHcIv8Zg4h5~1nKbaSSOeDlNxdBjKXJOjeBZ-J7fw16e4wHK08gZgkhxUiTwcJexOouekVd1MwXjVixnqIr2ssTO8EiegO7ep4jWc-71YECkn1KeKdgpzJ~A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two strategies for TMS. (A) Conventional procedures comprise adaptor tagging of fragmented genomic DNAs (Steps 1 and 2), target enrichment by hybridization (Steps 3 and 4) and bisulfite treatment of enriched library DNAs (Step 5) followed by PCR amplification (Step 6). The bisulfite treatment (Step 5) induces DNA breaks, inevitably leading to severe loss of intact sequencing template molecules. (B) PBAT-mediated procedure comprises target enrichment by hybridization (Steps 13), bisulfite treatment (Step 4) and adaptor tagging (Step 5), thereby circumventing the bisulfite-induced loss of intact sequencing template molecules." path-from-xml="dsu03401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196375/dsu03401.jpeg" path-from-xml="dsu03401.jpeg" target="_blank">View large</a><a section="87196375" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03401.jpeg?Expires=2147483647&Signature=UkDOvzu3YiOvVO-tS8WCETIZ50c5oSeis42ShBa1C0oy84vDhSgkSyChrAReL45EN3oPGkXn3ZJwdiHvf5VUET2adSwjFfzuW3l2ftWG4tTNB7-JuCKYNHbPhInoodayLU43p~mYI~53aEyeSMIs9APoboGJmUdkd~pNNQM2xaUWkI2PtbsW1jpXsNLMHzlV1ub37ITkXoTR-MGvdLytKrNaP9qg~vykMdsU44ZZQCdfhrW0SWo6T38ZWLt8ZmMCy6m8N8WU3L3FJcLZYIzF~hOFpRQKOR9~DacZISm9JUBCWJNThObZeUeAlNhodD4gks8spr~BT2aqR4ygHlXfUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196375&ar=2754540&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two strategies for TMS. (A) Conventional procedures comprise adaptor tagging of fragmented genomic DNAs (Steps 1 and 2), target enrichment by hybridization (Steps 3 and 4) and bisulfite treatment of enriched library DNAs (Step 5) followed by PCR amplification (Step 6). The bisulfite treatment (Step 5) induces DNA breaks, inevitably leading to severe loss of intact sequencing template molecules. (B) PBAT-mediated procedure comprises target enrichment by hybridization (Steps 13), bisulfite treatment (Step 4) and adaptor tagging (Step 5), thereby circumventing the bisulfite-induced loss of intact sequencing template molecules.</p></div></div></div>                            <h2 scrollto-destination=87196376 id="87196376" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196377 id="87196377" class="section-title" data-legacy-id=s2a>2.1. Preparation of DNA</h3>
<p>Both human and mouse genomic DNAs used in the model experiments were purchased from Promega. Genomic DNA from IMR90 primary human lung fibroblasts was a generous gift from Yae Kanai. The indicated amount of genomic DNA was dissolved in 130 l of 10 mM TrisHCl (pH 8.0) and sheared with Covaris S220 to the indicated size. We used AMPure XP to purify the fragmented DNA as follows. First, the shared DNA (130 l) was mixed with 1.8 volume (234 l) of the AMPure XP reagent and stood for 15 min at room temperature. Next, the beads were collected using a magnet stand, and the supernatant was removed. The pelleted beads were then rinsed with 70% ethanol and dried by standing at 37C for 5 min. Finally, DNA was eluted from the beads to 20 l of RNase-free water. The eluted DNA solution was dried in a vacuum concentrator and dissolved in 7 l of RNase-free water.</p>                            <h3 scrollto-destination=87196379 id="87196379" class="section-title" data-legacy-id=s2b>2.2. Target enrichment</h3>
<p>Enrichment of targets with liquid-phase hybridization capture was performed using the reagents in SureSelect Human or Mouse Methyl-Seq kit (Agilent). Genomic DNA (7 l) fragmented and purified as above was supplemented with 3 l of formamide (Wako, Biochemistry grade) and overlaid with 80 l of mineral oil (Sigma). The DNA was completely denatured by incubating the tube at 99C for 10 min, cooled down to 65C and kept at 65C for at least 5 min before adding the following reagents. Hybridization buffer was prepared by mixing 7.5, 0.3, 3.0 and 3.0 l of Hyb#1, #2, #3 and #4, respectively. Capture probe mix was prepared by mixing 5.0, 0.5 and 1.0 l of capture probe solution, RNase Inhibitor and RNase-free water, respectively. The hybridization buffer and the capture probe mix were individually overlaid with 80 l of mineral oil and incubated at 65C for 10 min. These two solutions were then combined and mixed thoroughly by pipetting. The combined solution was transferred to the tube containing the denatured input DNA kept at 65C as above and mixed thoroughly with the DNA solution by pipetting. The tube was incubated at 65C for at least 24 h to allow hybridization between the probes and the targets.</p><p>Fifty microlitres of well-suspended solution of DynaBeads MyOne Streptavidin T1 (Life Technologies) was taken into a 1.5-ml tube, and the beads were washed 2 times with 200 l of Binding Buffer. To the pelleted beads, the hybridization reaction supplemented with 200 l of Binding Buffer was added and mixed well. After incubation with rocking at room temperature for 30 min, the beads were collected using a magnetic stand and washed with 500 l of Wash Buffer 1. The beads were then subjected to three rounds of washing, each composed of re-suspension in pre-warmed Buffer 2 followed by incubation at 65C for 10 min. Following thorough removal of washing solutions from the tube, enriched DNA was eluted by incubating the beads in 20 l of Elution solution at room temperature for 20 min. The eluate was immediately used for bisulfite treatment.</p>                            <h3 scrollto-destination=87196382 id="87196382" class="section-title" data-legacy-id=s2c>2.3. Bisulfite treatment</h3>
<p>EZ DNA Methylation-Gold kit (Zymo Research) was used for the bisulfite treatment of target-enriched DNA, according to manufacturer's instruction. The enriched DNA solution (20 l) was directly mixed with 130 l of CT conversion reagent freshly prepared before use. The mixture was incubated at 64C for 2.5 h. Note that the incubation step at 98C for 10 min described in the instruction was omitted, because the target-enriched DNA was already denatured. Following the purification and desulfonation steps, bisulfite-treated DNA was eluted with 20 l of M-Elution buffer.</p>                            <h3 scrollto-destination=87196384 id="87196384" class="section-title" data-legacy-id=s2d>2.4. PBAT library construction and Illumina sequencing</h3>
<p>We used the bisulfite-treated DNA for library preparation according to the PBAT protocol<sup><a reveal-id="DSU034C14" data-open="DSU034C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (also available from <a class="link link-uri" href="http://crest-ihec.jp/english/epigenome/index.html">http://crest-ihec.jp/english/epigenome/index.html</a>), except for the primers used in the first- and second-strand synthesis. The primer used for the first-strand synthesis was 5-ACA CTC TTT CCC TAC ACG ACG CTC TTC CGA TCT WWW WNN NN-3 (W = A or T). The indexed primers used for the second-strand synthesis was 5-CAA GCA GAA GAC GGC ATA CGA GAT XXX XXX GTA AAA CGA CGG CCA GCA GGA AAC AGC TAT GAC WWW WNN NN-3, in which XXX XXX stands for the index sequence of each primer. We sequenced the constructed TMS libraries using Illumina HiSeq2500 as described previously.<sup><a reveal-id="DSU034C14" data-open="DSU034C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p>                            <h3 scrollto-destination=87196386 id="87196386" class="section-title" data-legacy-id=s2e>2.5. Data analysis</h3>
<p>The obtained reads were mapped to human and mouse DNA using the hg19 and mm9 assemblies, respectively, analysed and visualized as described previously.<sup><a reveal-id="DSU034C8" data-open="DSU034C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p>                            <h2 scrollto-destination=87196388 id="87196388" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87196389 id="87196389" class="section-title" data-legacy-id=s3a>3.1. PBAT to target-enriched DNA</h3>
<p>To test whether PBAT is applicable to TMS library construction, we applied it to target-enriched DNA prepared using the RNA probes provided in the Agilent SureSelect Mouse Methyl-Seq kit, which are designed to cover mouse genomic regions spanning 109 Mb in total. As PBAT employs random primer extension, it is ideal that the primer hybridizes to the 3-flanking region of each target region in the input DNA and initiates the synthesis of a complementary DNA strand to fully span the target region. We therefore assumed the size of the input DNA to be critical and first examined its effects on the yield of the library. Starting from 3 g of input DNA, as recommended by the manufacturer, we prepared DNA fragments with an average length of 180 bp, 400 bp or 2 kb (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>A). From these DNA preparations, we enriched the targets by solution hybridization capture and successfully generated PCR-free single-read PBAT libraries, each of which was sufficient for one or more lanes in the HiSeq2500 system (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>B). Note that the original Methyl-Seq protocol requires a total of 14 cycles of global PCR amplification when starting from the same amount of input DNA (i.e. 3 g). As expected, the raw yield of library increased with the size of input DNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>B). However, the normalized yield by the size of input DNA indicated that the 400-bp DNA fragments were most efficient. In addition, the library generated from the 2-kb DNA fragments contained a substantial number of off-target reads that were mapped near to but not onto the targets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>C). We therefore decided to use input DNA fragmented to a size of 400 bp for further development.</p>                            <h3 scrollto-destination=87196391 id="87196391" class="section-title" data-legacy-id=s3b>3.2. Novel primers for improved yield of indexed TMS libraries</h3>
<p>It is highly likely that the users of TMS intend to sequence two or more indexed libraries in a single lane for efficient data collection. Thus, we next attempted to improve the efficiency of the PBAT paired-end/indexed read protocol, because it is 2-fold less sensitive than the single-read protocol.<sup><a reveal-id="DSU034C14" data-open="DSU034C14" class="link link-ref link-reveal xref-bibr">14</a></sup> While the original PBAT protocol uses a random tetramer sequence (N<sub>4</sub>) attached to the 3-end of primers containing Illumina adaptor sequences,<sup><a reveal-id="DSU034C8" data-open="DSU034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> we found that novel primers containing a semi-random tetramer composed solely of A or T bases immediately upstream of the random tetramer (W<sub>4</sub>N<sub>4</sub>; W = A or T) improved the yield of TMS library by 4-fold, with a marginal effect on the GC bias in target coverage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). We also found that the addition of formamide to the hybridization capture reaction at a final concentration of 10% improved the reproducibility of the target enrichment step (data not shown). These modifications were critical to perform highly sensitive and robust construction of TMS libraries.</p>                            <h3 scrollto-destination=87196393 id="87196393" class="section-title" data-legacy-id=s3c>3.3. Characterization of low-input PBAT-mediated TMS libraries</h3>
<div class="block-child-p">Using the protocol optimized as above, we prepared indexed TMS libraries from 3,000 to 10 ng of human and mouse DNA using the human and mouse RNA probe sets obtained from Agilent, respectively (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>). The yield of library DNA was nearly linearly correlated with the amount of input DNA (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&amp;Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>A). The smaller the amount of input DNA, the more evident the adaptor dimers were (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&amp;Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>B). We performed the minimum cycles of PCR amplification to obtain sufficient DNA for a single HiSeq2500 lane (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>). We then used a half lane to sequence each of the human libraries and mapped the obtained reads to the reference human genome sequence. (Note that we combined the library DNA with the same amount of the PhiX control to compensate the extreme base bias of bisulfite-converted sequences.) The mapping rate was inversely correlated with the amount of input DNA within the range of 3,000 to 30 ng, but it showed a prominent decline when the DNA input was reduced to 10 ng (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>). In addition, all of the libraries except the one generated from 10 ng of DNA showed similar statistics regarding the coverage of the targets (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>A). The methylation levels of CpG sites covered by 20 or more reads were highly consistent among these five libraries (<em>R</em><sup>2</sup> &gt; 0.95, when the window size and the step were 500 and 250 bp, respectively) (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>B). We inspected the methylation status of imprinted genes and found that the expected 50% methylation level was faithfully recapitulated, even in the library generated from 30 ng of DNA, but not in the library obtained from 10 ng of DNA (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>C). These results suggest that the PBAT-mediated TMS method is reliably applicable to as little as 30 ng of input DNA. <div content-id="DSU034TB1" class="table-modal"><div class="table-wrap"><div id="DSU034TB1" data-id="DSU034TB1" class="table-wrap-title"><div class="label -label">Table1.</div><div class="caption"><p>Summary of library construction</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Target species (Target size)</th><th rowspan="2">Method for library construction</th><th rowspan="2">DNA source</th><th rowspan="2">Amount of input DNA (ng)</th><th rowspan="2">Library yield before PCR amplification<sup>a</sup> (amol)</th><th rowspan="2">Equivalent number of lanes<sup>b</sup></th><th rowspan="2">Number of PCR cycles to have DNA enough for a single lane</th><th rowspan="2">Number of reads (% uniquely mapped reads)</th><th colspan="3">Average depth of mapped reads obtained from a half lane of HiSeq2500 rapid mode<hr /></th></tr><tr><th>On target</th><th>Off target (near, 400 bp)</th><th>Off target (far, &gt;400 bp)</th></tr></thead><tbody><tr><td rowspan="8">Human (84 Mb)</td><td>Methyl-Seq</td><td rowspan="7">Promega human genomic DNA</td><td>3,000</td><td>1,340.0</td><td>1.00</td><td>11</td><td>110.3 M (91.1%)</td><td>98.2</td><td>9.6</td><td>0.2</td></tr><tr><td rowspan="7">PBAT</td><td>3,000</td><td>5,649.2</td><td>4.35</td><td>0</td><td>73.1 M (78.9%)</td><td>43.4</td><td>9.3</td><td>0.1</td></tr><tr><td>1,000</td><td>2,607.3</td><td>2.01</td><td>0</td><td>73.0 M (77.0%)</td><td>42.7</td><td>8.9</td><td>0.1</td></tr><tr><td>300</td><td>477.3</td><td>0.37</td><td>2</td><td>68.6 M (76.2%)</td><td>39.6</td><td>8.2</td><td>0.1</td></tr><tr><td>100</td><td>259.6</td><td>0.20</td><td>3</td><td>79.4 M (74.1%)</td><td>44.2</td><td>9.2</td><td>0.2</td></tr><tr><td>30</td><td>140.5</td><td>0.11</td><td>4</td><td>77.3 M (63.5%)</td><td>37.1</td><td>7.9</td><td>0.2</td></tr><tr><td>10</td><td>70.8</td><td>0.05</td><td>5</td><td>67.0 M (38.4%)</td><td>18.2</td><td>4.1</td><td>0.2</td></tr><tr><td>IMR90</td><td>300</td><td>220.0</td><td>0.17</td><td>4</td><td>70.8 M (85.6%)</td><td>46.3</td><td>9.4</td><td>0.1</td></tr><tr><td rowspan="6">Mouse (109 Mb)</td><td rowspan="6">PBAT</td><td rowspan="6">Promega mouse genomic DNA</td><td>3,000</td><td>3,645.3</td><td>2.80</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>1,000</td><td>2,353.3</td><td>1.81</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>300</td><td>679.5</td><td>0.52</td><td>2</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>100</td><td>285.2</td><td>0.22</td><td>3</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>30</td><td>151.9</td><td>0.12</td><td>4</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>10</td><td>71.2</td><td>0.05</td><td>5</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr></tbody></table></div><div content-id="DSU034TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Target species (Target size)</th><th rowspan="2">Method for library construction</th><th rowspan="2">DNA source</th><th rowspan="2">Amount of input DNA (ng)</th><th rowspan="2">Library yield before PCR amplification<sup>a</sup> (amol)</th><th rowspan="2">Equivalent number of lanes<sup>b</sup></th><th rowspan="2">Number of PCR cycles to have DNA enough for a single lane</th><th rowspan="2">Number of reads (% uniquely mapped reads)</th><th colspan="3">Average depth of mapped reads obtained from a half lane of HiSeq2500 rapid mode<hr /></th></tr><tr><th>On target</th><th>Off target (near, 400 bp)</th><th>Off target (far, &gt;400 bp)</th></tr></thead><tbody><tr><td rowspan="8">Human (84 Mb)</td><td>Methyl-Seq</td><td rowspan="7">Promega human genomic DNA</td><td>3,000</td><td>1,340.0</td><td>1.00</td><td>11</td><td>110.3 M (91.1%)</td><td>98.2</td><td>9.6</td><td>0.2</td></tr><tr><td rowspan="7">PBAT</td><td>3,000</td><td>5,649.2</td><td>4.35</td><td>0</td><td>73.1 M (78.9%)</td><td>43.4</td><td>9.3</td><td>0.1</td></tr><tr><td>1,000</td><td>2,607.3</td><td>2.01</td><td>0</td><td>73.0 M (77.0%)</td><td>42.7</td><td>8.9</td><td>0.1</td></tr><tr><td>300</td><td>477.3</td><td>0.37</td><td>2</td><td>68.6 M (76.2%)</td><td>39.6</td><td>8.2</td><td>0.1</td></tr><tr><td>100</td><td>259.6</td><td>0.20</td><td>3</td><td>79.4 M (74.1%)</td><td>44.2</td><td>9.2</td><td>0.2</td></tr><tr><td>30</td><td>140.5</td><td>0.11</td><td>4</td><td>77.3 M (63.5%)</td><td>37.1</td><td>7.9</td><td>0.2</td></tr><tr><td>10</td><td>70.8</td><td>0.05</td><td>5</td><td>67.0 M (38.4%)</td><td>18.2</td><td>4.1</td><td>0.2</td></tr><tr><td>IMR90</td><td>300</td><td>220.0</td><td>0.17</td><td>4</td><td>70.8 M (85.6%)</td><td>46.3</td><td>9.4</td><td>0.1</td></tr><tr><td rowspan="6">Mouse (109 Mb)</td><td rowspan="6">PBAT</td><td rowspan="6">Promega mouse genomic DNA</td><td>3,000</td><td>3,645.3</td><td>2.80</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>1,000</td><td>2,353.3</td><td>1.81</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>300</td><td>679.5</td><td>0.52</td><td>2</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>100</td><td>285.2</td><td>0.22</td><td>3</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>30</td><td>151.9</td><td>0.12</td><td>4</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>10</td><td>71.2</td><td>0.05</td><td>5</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>M: million.</p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">a</div><p>Yield of the library amplified by 11 cycles of PCR was shown for Methyl-Seq.</p></div></div><div content-id="TN3"><div class="fn"><div class="label -label">b</div><p>Based on an assumption that a single lane requires 1,300 amol of library DNA.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU034TB1" data-id="DSU034TB1" class="table-wrap-title"><div class="label -label">Table1.</div><div class="caption"><p>Summary of library construction</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Target species (Target size)</th><th rowspan="2">Method for library construction</th><th rowspan="2">DNA source</th><th rowspan="2">Amount of input DNA (ng)</th><th rowspan="2">Library yield before PCR amplification<sup>a</sup> (amol)</th><th rowspan="2">Equivalent number of lanes<sup>b</sup></th><th rowspan="2">Number of PCR cycles to have DNA enough for a single lane</th><th rowspan="2">Number of reads (% uniquely mapped reads)</th><th colspan="3">Average depth of mapped reads obtained from a half lane of HiSeq2500 rapid mode<hr /></th></tr><tr><th>On target</th><th>Off target (near, 400 bp)</th><th>Off target (far, &gt;400 bp)</th></tr></thead><tbody><tr><td rowspan="8">Human (84 Mb)</td><td>Methyl-Seq</td><td rowspan="7">Promega human genomic DNA</td><td>3,000</td><td>1,340.0</td><td>1.00</td><td>11</td><td>110.3 M (91.1%)</td><td>98.2</td><td>9.6</td><td>0.2</td></tr><tr><td rowspan="7">PBAT</td><td>3,000</td><td>5,649.2</td><td>4.35</td><td>0</td><td>73.1 M (78.9%)</td><td>43.4</td><td>9.3</td><td>0.1</td></tr><tr><td>1,000</td><td>2,607.3</td><td>2.01</td><td>0</td><td>73.0 M (77.0%)</td><td>42.7</td><td>8.9</td><td>0.1</td></tr><tr><td>300</td><td>477.3</td><td>0.37</td><td>2</td><td>68.6 M (76.2%)</td><td>39.6</td><td>8.2</td><td>0.1</td></tr><tr><td>100</td><td>259.6</td><td>0.20</td><td>3</td><td>79.4 M (74.1%)</td><td>44.2</td><td>9.2</td><td>0.2</td></tr><tr><td>30</td><td>140.5</td><td>0.11</td><td>4</td><td>77.3 M (63.5%)</td><td>37.1</td><td>7.9</td><td>0.2</td></tr><tr><td>10</td><td>70.8</td><td>0.05</td><td>5</td><td>67.0 M (38.4%)</td><td>18.2</td><td>4.1</td><td>0.2</td></tr><tr><td>IMR90</td><td>300</td><td>220.0</td><td>0.17</td><td>4</td><td>70.8 M (85.6%)</td><td>46.3</td><td>9.4</td><td>0.1</td></tr><tr><td rowspan="6">Mouse (109 Mb)</td><td rowspan="6">PBAT</td><td rowspan="6">Promega mouse genomic DNA</td><td>3,000</td><td>3,645.3</td><td>2.80</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>1,000</td><td>2,353.3</td><td>1.81</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>300</td><td>679.5</td><td>0.52</td><td>2</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>100</td><td>285.2</td><td>0.22</td><td>3</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>30</td><td>151.9</td><td>0.12</td><td>4</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>10</td><td>71.2</td><td>0.05</td><td>5</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr></tbody></table></div><div content-id="DSU034TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Target species (Target size)</th><th rowspan="2">Method for library construction</th><th rowspan="2">DNA source</th><th rowspan="2">Amount of input DNA (ng)</th><th rowspan="2">Library yield before PCR amplification<sup>a</sup> (amol)</th><th rowspan="2">Equivalent number of lanes<sup>b</sup></th><th rowspan="2">Number of PCR cycles to have DNA enough for a single lane</th><th rowspan="2">Number of reads (% uniquely mapped reads)</th><th colspan="3">Average depth of mapped reads obtained from a half lane of HiSeq2500 rapid mode<hr /></th></tr><tr><th>On target</th><th>Off target (near, 400 bp)</th><th>Off target (far, &gt;400 bp)</th></tr></thead><tbody><tr><td rowspan="8">Human (84 Mb)</td><td>Methyl-Seq</td><td rowspan="7">Promega human genomic DNA</td><td>3,000</td><td>1,340.0</td><td>1.00</td><td>11</td><td>110.3 M (91.1%)</td><td>98.2</td><td>9.6</td><td>0.2</td></tr><tr><td rowspan="7">PBAT</td><td>3,000</td><td>5,649.2</td><td>4.35</td><td>0</td><td>73.1 M (78.9%)</td><td>43.4</td><td>9.3</td><td>0.1</td></tr><tr><td>1,000</td><td>2,607.3</td><td>2.01</td><td>0</td><td>73.0 M (77.0%)</td><td>42.7</td><td>8.9</td><td>0.1</td></tr><tr><td>300</td><td>477.3</td><td>0.37</td><td>2</td><td>68.6 M (76.2%)</td><td>39.6</td><td>8.2</td><td>0.1</td></tr><tr><td>100</td><td>259.6</td><td>0.20</td><td>3</td><td>79.4 M (74.1%)</td><td>44.2</td><td>9.2</td><td>0.2</td></tr><tr><td>30</td><td>140.5</td><td>0.11</td><td>4</td><td>77.3 M (63.5%)</td><td>37.1</td><td>7.9</td><td>0.2</td></tr><tr><td>10</td><td>70.8</td><td>0.05</td><td>5</td><td>67.0 M (38.4%)</td><td>18.2</td><td>4.1</td><td>0.2</td></tr><tr><td>IMR90</td><td>300</td><td>220.0</td><td>0.17</td><td>4</td><td>70.8 M (85.6%)</td><td>46.3</td><td>9.4</td><td>0.1</td></tr><tr><td rowspan="6">Mouse (109 Mb)</td><td rowspan="6">PBAT</td><td rowspan="6">Promega mouse genomic DNA</td><td>3,000</td><td>3,645.3</td><td>2.80</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>1,000</td><td>2,353.3</td><td>1.81</td><td>0</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>300</td><td>679.5</td><td>0.52</td><td>2</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>100</td><td>285.2</td><td>0.22</td><td>3</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>30</td><td>151.9</td><td>0.12</td><td>4</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr><tr><td>10</td><td>71.2</td><td>0.05</td><td>5</td><td>nd</td><td>nd</td><td>nd</td><td>nd</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>M: million.</p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">a</div><p>Yield of the library amplified by 11 cycles of PCR was shown for Methyl-Seq.</p></div></div><div content-id="TN3"><div class="fn"><div class="label -label">b</div><p>Based on an assumption that a single lane requires 1,300 amol of library DNA.</p></div></div></div></div><div data-id="DSU034F2" class="fig fig-section"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03402.jpeg?Expires=2147483647&amp;Signature=UW5isaCEKABaREaFfWabC34Ro5DgzvFtFsCYX4SwDyCzobWtw0r8FTtkjzbWVRsg~bx5OYYc8HqsgiYtsV01~KwuipcjlD2cssVZe2e0NBtbxLHZTl9aXYSXRJkvpdqJlZRIdcJQc5Yk1UsxDO-xdagrUUp3mSj6oInWaqth6R~wjLgr7vaA7T33Bi5Sxu2mVWlgqwDmKwCo1zWu-0LCDr5EwVENPlDBV2MXBA4VNfKfUeEw-x7MjDqqLiNM4LV8Ial9rlcKMjToQeX0IqFy-CFDJ416NGon5TmnaiICqc41c6xQvdEH5xrvdHJ5QLnSV80fVF-wUhfUmHv~Ocds2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance of PBAT-mediated TMS. (A) Target coverage. The fraction of the targets covered by differing minimal depth of reads was shown for the six PBAT and one Methyl-Seq libraries generated from the indicated amount of input DNA. Note that the average read depth of the Methyl-Seq library was twice or more higher than those of the PBAT libraries (Table1). (B) Consistency among TMS data. Methylation levels were compared among the six TMS libraries generated from 3,000 to 10 ng of human genomic DNA using the PBAT-mediated procedure as well as the one generated from 3,000 ng of input DNA using the original Methyl-Seq protocol (DRA002274-002280). The numbers and the images in the boxes above and below the diagonal indicated the coefficients of determination (R2) and the scatter plot of methylation levels, respectively, between all the possible combinations among the seven data sets. The moving averages of methylation levels (window size, 500 bp; step size, 250 bp) were calculated based on CpGs covered by 20 or more reads. (C) A snapshot of TMS data. Data around the imprinted control region (ICR) for PEG3 were compared among the seven libraries generated with either PBAT or Methyl-Seq using the indicated amount of input DNA. Red bars and grey shadows indicated the methylation levels of individual CpG sites and the depth of reads, respectively. Note that most reads were mapped to the bottom strand, as the RNA probes used in the experiment were designed from the top strand. As expected, a region around the PEG3 promoter showed 50% methylation level due to the imprinted monoallelic methylation. The green dashed box denoted the ICR of PEG3 (chr.19: 57,351,728 to 57,352,173 in hg19 human reference genome sequence)." path-from-xml="dsu03402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196394/dsu03402.jpeg" path-from-xml="dsu03402.jpeg" target="_blank">View large</a><a section="87196394" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03402.jpeg?Expires=2147483647&amp;Signature=dIiqlGjKkefP~diO7dHc9l66ovtpShTK7GYQO7FXDQa2iaYhZV3bO2QuORUFtVLK4pRnYkPNLCxMbS4ntaiOeL-cOSWRhw1HJjN5IdxgaJOpdKkShY7wq5X9zc-1rSCasAS-bIn9pN0yRVm1v3lOQWR5L1x-gVObXJTXOL1NDD8nbmJ7~hfQTGeXBmFP3DhGHi3yBASPoc~oPlN6AKO5Kq5qYR4btwGuF6n3QNk357RFAPQQjRwiVrtpZCpDhKQDkKV425A2T3Cq4EhdXhFtHpTlfeJrG4l7q6qCdbuhrJRPsMmKHczb~ToXfX85ahhpL2F0JL0z5337b5VyQuQ1Ug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196394&amp;ar=2754540&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of PBAT-mediated TMS. (A) Target coverage. The fraction of the targets covered by differing minimal depth of reads was shown for the six PBAT and one Methyl-Seq libraries generated from the indicated amount of input DNA. Note that the average read depth of the Methyl-Seq library was twice or more higher than those of the PBAT libraries (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>). (B) Consistency among TMS data. Methylation levels were compared among the six TMS libraries generated from 3,000 to 10 ng of human genomic DNA using the PBAT-mediated procedure as well as the one generated from 3,000 ng of input DNA using the original Methyl-Seq protocol (DRA002274-002280). The numbers and the images in the boxes above and below the diagonal indicated the coefficients of determination (<em>R</em><sup>2</sup>) and the scatter plot of methylation levels, respectively, between all the possible combinations among the seven data sets. The moving averages of methylation levels (window size, 500 bp; step size, 250 bp) were calculated based on CpGs covered by 20 or more reads. (C) A snapshot of TMS data. Data around the imprinted control region (ICR) for <em>PEG3</em> were compared among the seven libraries generated with either PBAT or Methyl-Seq using the indicated amount of input DNA. Red bars and grey shadows indicated the methylation levels of individual CpG sites and the depth of reads, respectively. Note that most reads were mapped to the bottom strand, as the RNA probes used in the experiment were designed from the top strand. As expected, a region around the <em>PEG3</em> promoter showed 50% methylation level due to the imprinted monoallelic methylation. The green dashed box denoted the ICR of <em>PEG3</em> (chr.19: 57,351,728 to 57,352,173 in hg19 human reference genome sequence).</p></div></div><div content-id="DSU034F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03402.jpeg?Expires=2147483647&amp;Signature=UW5isaCEKABaREaFfWabC34Ro5DgzvFtFsCYX4SwDyCzobWtw0r8FTtkjzbWVRsg~bx5OYYc8HqsgiYtsV01~KwuipcjlD2cssVZe2e0NBtbxLHZTl9aXYSXRJkvpdqJlZRIdcJQc5Yk1UsxDO-xdagrUUp3mSj6oInWaqth6R~wjLgr7vaA7T33Bi5Sxu2mVWlgqwDmKwCo1zWu-0LCDr5EwVENPlDBV2MXBA4VNfKfUeEw-x7MjDqqLiNM4LV8Ial9rlcKMjToQeX0IqFy-CFDJ416NGon5TmnaiICqc41c6xQvdEH5xrvdHJ5QLnSV80fVF-wUhfUmHv~Ocds2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance of PBAT-mediated TMS. (A) Target coverage. The fraction of the targets covered by differing minimal depth of reads was shown for the six PBAT and one Methyl-Seq libraries generated from the indicated amount of input DNA. Note that the average read depth of the Methyl-Seq library was twice or more higher than those of the PBAT libraries (Table1). (B) Consistency among TMS data. Methylation levels were compared among the six TMS libraries generated from 3,000 to 10 ng of human genomic DNA using the PBAT-mediated procedure as well as the one generated from 3,000 ng of input DNA using the original Methyl-Seq protocol (DRA002274-002280). The numbers and the images in the boxes above and below the diagonal indicated the coefficients of determination (R2) and the scatter plot of methylation levels, respectively, between all the possible combinations among the seven data sets. The moving averages of methylation levels (window size, 500 bp; step size, 250 bp) were calculated based on CpGs covered by 20 or more reads. (C) A snapshot of TMS data. Data around the imprinted control region (ICR) for PEG3 were compared among the seven libraries generated with either PBAT or Methyl-Seq using the indicated amount of input DNA. Red bars and grey shadows indicated the methylation levels of individual CpG sites and the depth of reads, respectively. Note that most reads were mapped to the bottom strand, as the RNA probes used in the experiment were designed from the top strand. As expected, a region around the PEG3 promoter showed 50% methylation level due to the imprinted monoallelic methylation. The green dashed box denoted the ICR of PEG3 (chr.19: 57,351,728 to 57,352,173 in hg19 human reference genome sequence)." path-from-xml="dsu03402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196394/dsu03402.jpeg" path-from-xml="dsu03402.jpeg" target="_blank">View large</a><a section="87196394" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03402.jpeg?Expires=2147483647&amp;Signature=dIiqlGjKkefP~diO7dHc9l66ovtpShTK7GYQO7FXDQa2iaYhZV3bO2QuORUFtVLK4pRnYkPNLCxMbS4ntaiOeL-cOSWRhw1HJjN5IdxgaJOpdKkShY7wq5X9zc-1rSCasAS-bIn9pN0yRVm1v3lOQWR5L1x-gVObXJTXOL1NDD8nbmJ7~hfQTGeXBmFP3DhGHi3yBASPoc~oPlN6AKO5Kq5qYR4btwGuF6n3QNk357RFAPQQjRwiVrtpZCpDhKQDkKV425A2T3Cq4EhdXhFtHpTlfeJrG4l7q6qCdbuhrJRPsMmKHczb~ToXfX85ahhpL2F0JL0z5337b5VyQuQ1Ug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196394&amp;ar=2754540&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance of PBAT-mediated TMS. (A) Target coverage. The fraction of the targets covered by differing minimal depth of reads was shown for the six PBAT and one Methyl-Seq libraries generated from the indicated amount of input DNA. Note that the average read depth of the Methyl-Seq library was twice or more higher than those of the PBAT libraries (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>). (B) Consistency among TMS data. Methylation levels were compared among the six TMS libraries generated from 3,000 to 10 ng of human genomic DNA using the PBAT-mediated procedure as well as the one generated from 3,000 ng of input DNA using the original Methyl-Seq protocol (DRA002274-002280). The numbers and the images in the boxes above and below the diagonal indicated the coefficients of determination (<em>R</em><sup>2</sup>) and the scatter plot of methylation levels, respectively, between all the possible combinations among the seven data sets. The moving averages of methylation levels (window size, 500 bp; step size, 250 bp) were calculated based on CpGs covered by 20 or more reads. (C) A snapshot of TMS data. Data around the imprinted control region (ICR) for <em>PEG3</em> were compared among the seven libraries generated with either PBAT or Methyl-Seq using the indicated amount of input DNA. Red bars and grey shadows indicated the methylation levels of individual CpG sites and the depth of reads, respectively. Note that most reads were mapped to the bottom strand, as the RNA probes used in the experiment were designed from the top strand. As expected, a region around the <em>PEG3</em> promoter showed 50% methylation level due to the imprinted monoallelic methylation. The green dashed box denoted the ICR of <em>PEG3</em> (chr.19: 57,351,728 to 57,352,173 in hg19 human reference genome sequence).</p></div></div></div><p>We also performed Methyl-Seq from the same human genomic DNA according to the protocol provided by the manufacturer, except that fewer PCR cycles were used (i.e. 11 cycles instead of 14 cycles), and compared the results with those of PBAT-mediated TMS. While the two data sets showed largely consistent methylation levels (<em>R</em><sup>2</sup> = 0.88) (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>B), they were distinct in terms of the GC content of the covered targets: Methyl-Seq and PBAT-mediated TMS preferentially covered AT-rich and GC-rich targets, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). This was presumably because the extensive global PCR step in Methyl-Seq failed to amplify GC-rich regions and because the random priming from the PBAT adaptor primers was rather compromised in AT-rich regions. We also note that the Methyl-Seq data recapitulated the methylation status of imprinted genes less precisely than the PBAT data (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>C).</p><div class="block-child-p">Having confirmed the high efficiency and consistency of PBAT-mediated TMS, we next compared it with WGBS. We performed TMS using 300 ng of genomic DNA from human IMR90 cells and compared the resulting data with PBAT-mediated WGBS data from the same cell line. The methylation levels of CpG sites covered by 20 or more reads showed an excellent correlation between the two data sets in either the window-based or the nucleotide-based analysis (Fig.<a reveal-id="DSU034F3" data-open="DSU034F3" class="link link-reveal link-table xref-fig">3</a>). These results indicated that TMS can be a reliable alternative to WGBS. <div data-id="DSU034F3" class="fig fig-section"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03403.jpeg?Expires=2147483647&Signature=vcLfuWdv3e9HPKQlwLiqZxmLi89nkGwx3momlTQMlRtLJFdWgIpnFsi6tOyqsSovlWYxZwG7Yqwgx596tg8d6H31odtjcJMJX4OG1HsZYT8F~9NgOkPbmLhk5rMmdY5K3KOaIpqItrwhoG4S~6IcInwWwYs4lGsKLCvssNKJ7PUEKIEUo6L3ZHibdw56u26r30PKEjj3-LyO3ZDRafz6~FAVlN-BNW304A9m6~bi3tJzfGhQ8BvXGmO5kY0vlIHe9HHJw~u96yalUOYpcQdpNRi7CujRJUEwbk9LgqB0y~1tpM7ZC0Pw4HTt1FlLJV4uJPYdR-GvBgrnwRK8--~KmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consistency between TMS and WGBS data. Methylation levels were compared between the TMS data (DRA002281) (Table1) and a publicly available WGBS data (DRA002248) on human IMR90 cells using the CpG sites covered by 20 or more reads. Methylation levels were plotted for moving windows (window size, 500 bp; stepping size, 250 bp) (A) and for individual CpG sites (B)." path-from-xml="dsu03403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196396/dsu03403.jpeg" path-from-xml="dsu03403.jpeg" target="_blank">View large</a><a section="87196396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03403.jpeg?Expires=2147483647&Signature=Utfhffln51ffKR3jWXnp0sWwGF28Nci2lQ6jHiwaWnt49n6tCZZQLCG9-R~1lTo2GvE~ePO55bT9diyN0QbWl5KcF5Wp2MXsLQ0s2C2d0eMwxfsQHmCPO9kph6mChZYGIMjw8Dw3VNR3LSgIKlXDPZYtWZjq9TziT2Ce0PYZjd-w0bEz~aYIjJztExbWgQ4aR5U8yjpOc705hG4E5ko5kAfmUpFM-4BYaD9b49LSKCwcSMD82dDq8wjYJiMQ9HUn5NSr2f11dtQLMTAUqGp95gdy9d5BcfImfzkyXHg9wj4U9YEYe4GgrMZ2EPWVtJ~WFRg1npfuEevpgQZ9FmdxkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196396&ar=2754540&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consistency between TMS and WGBS data. Methylation levels were compared between the TMS data (DRA002281) (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>) and a publicly available WGBS data (DRA002248) on human IMR90 cells using the CpG sites covered by 20 or more reads. Methylation levels were plotted for moving windows (window size, 500 bp; stepping size, 250 bp) (A) and for individual CpG sites (B).</p></div></div><div content-id="DSU034F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/m_dsu03403.jpeg?Expires=2147483647&Signature=vcLfuWdv3e9HPKQlwLiqZxmLi89nkGwx3momlTQMlRtLJFdWgIpnFsi6tOyqsSovlWYxZwG7Yqwgx596tg8d6H31odtjcJMJX4OG1HsZYT8F~9NgOkPbmLhk5rMmdY5K3KOaIpqItrwhoG4S~6IcInwWwYs4lGsKLCvssNKJ7PUEKIEUo6L3ZHibdw56u26r30PKEjj3-LyO3ZDRafz6~FAVlN-BNW304A9m6~bi3tJzfGhQ8BvXGmO5kY0vlIHe9HHJw~u96yalUOYpcQdpNRi7CujRJUEwbk9LgqB0y~1tpM7ZC0Pw4HTt1FlLJV4uJPYdR-GvBgrnwRK8--~KmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consistency between TMS and WGBS data. Methylation levels were compared between the TMS data (DRA002281) (Table1) and a publicly available WGBS data (DRA002248) on human IMR90 cells using the CpG sites covered by 20 or more reads. Methylation levels were plotted for moving windows (window size, 500 bp; stepping size, 250 bp) (A) and for individual CpG sites (B)." path-from-xml="dsu03403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196396/dsu03403.jpeg" path-from-xml="dsu03403.jpeg" target="_blank">View large</a><a section="87196396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu03403.jpeg?Expires=2147483647&Signature=Utfhffln51ffKR3jWXnp0sWwGF28Nci2lQ6jHiwaWnt49n6tCZZQLCG9-R~1lTo2GvE~ePO55bT9diyN0QbWl5KcF5Wp2MXsLQ0s2C2d0eMwxfsQHmCPO9kph6mChZYGIMjw8Dw3VNR3LSgIKlXDPZYtWZjq9TziT2Ce0PYZjd-w0bEz~aYIjJztExbWgQ4aR5U8yjpOc705hG4E5ko5kAfmUpFM-4BYaD9b49LSKCwcSMD82dDq8wjYJiMQ9HUn5NSr2f11dtQLMTAUqGp95gdy9d5BcfImfzkyXHg9wj4U9YEYe4GgrMZ2EPWVtJ~WFRg1npfuEevpgQZ9FmdxkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196396&ar=2754540&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consistency between TMS and WGBS data. Methylation levels were compared between the TMS data (DRA002281) (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>) and a publicly available WGBS data (DRA002248) on human IMR90 cells using the CpG sites covered by 20 or more reads. Methylation levels were plotted for moving windows (window size, 500 bp; stepping size, 250 bp) (A) and for individual CpG sites (B).</p></div></div></div>                            <h2 scrollto-destination=87196397 id="87196397" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>WGBS has become the gold standard method in methylomics for its unsurpassed resolution and coverage. However, it is too expensive to be used for analysing multiple samples. Efforts have thus been paid to develop cost-effective alternatives to WGBS, including various TMS methods based on padlock probes, array capture and solution hybridization capture.<sup><a reveal-id="DSU034C7" data-open="DSU034C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Among these approaches, the last one is highly flexible, but current protocols require a considerable amount of input DNA and are not applicable to samples of limited quantity.<sup><a reveal-id="DSU034C5" data-open="DSU034C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU034C6" data-open="DSU034C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Furthermore, they include extensive global PCR amplification, which may increase the risk of inaccurate estimate of methylation levels, especially when the quantity of input DNA is limited. Therefore, a novel TMS protocol is desirable that requires a much smaller amount of input DNA with far fewer cycles of global PCR amplification than current ones. As the inefficiency of current protocols is likely attributable to bisulfite-induced degradation of target-enriched library DNA (Fig.<a reveal-id="DSU034F1" data-open="DSU034F1" class="link link-reveal link-table xref-fig">1</a>A), we reasoned that the PBAT strategy can circumvent the adverse effect of bisulfite treatment to improve the efficiency of TMS (Fig.<a reveal-id="DSU034F1" data-open="DSU034F1" class="link link-reveal link-table xref-fig">1</a>B).</p><p>We tested the possibility using the biotinylated RNA probes in the Agilent Methyl-Seq kit designed to cover CpG islands with their shores and shelves, enhancers, promoters, differentially methylated regions and other regulatory elements. Our results demonstrated that PBAT enables TMS from a 100-fold smaller amount of input DNA (i.e. 30 versus 3,000 ng) with considerably fewer cycles of PCR amplification (i.e. 4 versus 1114 cycles) than the original protocol provided by the manufacturer (Table<a reveal-id="DSU034TB1" data-open="DSU034TB1" class="link link-reveal link-table xref-fig">1</a>; Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>). It also enables PCR-free TMS, provided that 1 g of input DNA is available. Furthermore, the coverage statistics and the methylation levels of imprinted genes indicated that the PBAT-mediated protocol has superior coverage and accuracy than the original one (Fig.<a reveal-id="DSU034F2" data-open="DSU034F2" class="link link-reveal link-table xref-fig">2</a>). We also confirmed high consistency between PBAT-mediated TMS and WGBS (Fig.<a reveal-id="DSU034F3" data-open="DSU034F3" class="link link-reveal link-table xref-fig">3</a>), proving that the former can serve as a reliable surrogate for the latter. Although further improvements are necessary to generate high-quality TMS libraries from &lt;30 ng of input DNA and to achieve more even coverage of the targets regardless of their GC contents, the PBAT-mediated protocol significantly outperforms the original Methyl-Seq protocol. It will be also useful for base-resolution analysis of 5-hydroxymethylcytosine when applied to genomic DNA treated with adequate oxidants or enzymes.<sup><a reveal-id="DSU034C15" data-open="DSU034C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSU034C16" data-open="DSU034C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>We also evaluated the cost performance of PBAT-mediated TMS in comparison with those of PBAT-mediated WGBS and RRBS. The TMS is 10 times less expensive than the WGBS, when the former and the latter use the high output mode of HiSeq to achieve 40-fold coverage of the target regions and the whole genome, respectively. Note that, the deeper the coverage is, the more cost effective the TMS is. While the target regions in the TMS cover only 2.8% of the human genome, they include approximately one-seventh of the total CpG sites. On the other hand, the TMS costs 45 times more but covers only 1.5 times more CpG sites than RRBS does. Nevertheless, the TMS likely remains highly competitive to RRBS in many instances, since the former covers a much broader range of genomic elements than the latter.</p><p>Taken together, PBAT significantly enhances the utility of TMS to enable various novel applications, especially those analysing a large number of precious samples. The PBAT-mediated TMS will thus serve as an invaluable tool for epigenomics.</p>                            <h2 scrollto-destination=87196402 id="87196402" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>The TMS data sets from this study have been submitted to the DDBJ Sequence Read Archive under the accession numbers DRA002274-002281.</p>                            <h2 scrollto-destination=87196404 id="87196404" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by Research Program of Innovative Cell Biology by Innovative Technology (Cell Innovation) (to T.I.), Platform for Drug Discovery, Informatics and Structural Life Science (to T.I.) and Grant-in-Aid for Scientific Research on Innovative Areas (25129702) (to F.M.) from the Ministry of Education, Culture, Sports, Science and Technology (MEXT) of Japan. Funding to pay the Open Access publication charges for this article was provided by MEXT.</p>                            <h2 scrollto-destination=87196406 id="87196406" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are indebted to Kaho Minoura (Agilent Technologies) for helpful supports in target enrichment using the SureSelect technology. We are grateful to Kenjiro Shirane, Hidehiro Toh, Kenji Ichiyanagi and Hiroyuki Sasaki (Kyushu University) for help in Illumina sequencing and IMR90 WGBS data. We also thank Yae Kanai (National Cancer Research Center) for IMR90 DNA.</p>                            <h2 scrollto-destination=87196408 id="87196408" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU034C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cokus</div>
               <div class="given-names">S.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Feng</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">X.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Shotgun bisulphite sequencing of the <em>Arabidopsis</em> genome reveals DNA methylation patterning</div>
          <div class="source">Nature</div>
          <div class="year">2008</div>
          <div class="volume">452</div>
          <div class="fpage">215</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shotgun%20bisulphite%20sequencing%20of%20the%20Arabidopsis%20genome%20reveals%20DNA%20methylation%20patterning&amp;author=S.J.%20Cokus&amp;author=S.%20Feng&amp;author=X.%20Zhang&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=215-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278030" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lister</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">O'Malley</div>
               <div class="given-names">R.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tonti-Filippini</div>
               <div class="given-names">J.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Highly integrated single-base resolution maps of the epigenome in <em>Arabidopsis</em></div>
          <div class="source">Cell</div>
          <div class="year">2008</div>
          <div class="volume">133</div>
          <div class="fpage">523</div>
          <div class="lpage">36</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20integrated%20single-base%20resolution%20maps%20of%20the%20epigenome%20in%20Arabidopsis&amp;author=R.%20Lister&amp;author=R.C.%20O'Malley&amp;author=J.%20Tonti-Filippini&amp;publication_year=2008&amp;journal=Cell&amp;volume=133&amp;pages=523-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.03.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.03.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.03.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18423832" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Meissner</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gnirke</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bell</div>
               <div class="given-names">G.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ramsahoye</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lander</div>
               <div class="given-names">E.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jaenisch</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis</div>
          <div class="source">Nucleic Acids Res.</div>
          <div class="year">2005</div>
          <div class="volume">33</div>
          <div class="fpage">5868</div>
          <div class="lpage">77</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20representation%20bisulfite%20sequencing%20for%20comparative%20high-resolution%20DNA%20methylation%20analysis&amp;author=A.%20Meissner&amp;author=A.%20Gnirke&amp;author=G.W.%20Bell&amp;author=B.%20Ramsahoye&amp;author=E.S.%20Lander&amp;author=R.%20Jaenisch&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki901" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki901" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki901"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16224102" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">Z.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bock</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Boyle</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gnirke</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Meissner</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Preparation of reduced representation bisulfite sequencing libraries for genome-scale DNA methylation profiling</div>
          <div class="source">Nat. Protoc.</div>
          <div class="year">2011</div>
          <div class="volume">6</div>
          <div class="fpage">468</div>
          <div class="lpage">81</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20reduced%20representation%20bisulfite%20sequencing%20libraries%20for%20genome-scale%20DNA%20methylation%20profiling&amp;author=H.%20Gu&amp;author=Z.D.%20Smith&amp;author=C.%20Bock&amp;author=P.%20Boyle&amp;author=A.%20Gnirke&amp;author=A.%20Meissner&amp;publication_year=2011&amp;journal=Nat.%20Protoc.&amp;volume=6&amp;pages=468-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2010.190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2010.190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2010.190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21412275" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">E.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pei</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Srivastava</div>
               <div class="given-names">G.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Targeted bisulfite sequencing by solution hybrid selection and massively parallel sequencing</div>
          <div class="source">Nucleic Acids Res.</div>
          <div class="year">2011</div>
          <div class="volume">39</div>
          <div class="fpage">e127</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20bisulfite%20sequencing%20by%20solution%20hybrid%20selection%20and%20massively%20parallel%20sequencing&amp;author=E.J.%20Lee&amp;author=L.%20Pei&amp;author=G.%20Srivastava&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21785137" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ji</div>
               <div class="given-names">G.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">High resolution profiling of human exon methylation by liquid hybridization capture-based bisulfite sequencing</div>
          <div class="source">BMC Genomics</div>
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">597</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20resolution%20profiling%20of%20human%20exon%20methylation%20by%20liquid%20hybridization%20capture-based%20bisulfite%20sequencing&amp;author=J.%20Wang&amp;author=H.%20Jiang&amp;author=G.%20Ji&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=597" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-597"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22151801" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">E.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Luo</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wilson</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shi</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="article-title">Analyzing the cancer methylome through targeted bisulfite sequencing</div>
          <div class="source">Cancer Lett.</div>
          <div class="year">2013</div>
          <div class="volume">340</div>
          <div class="fpage">171</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20the%20cancer%20methylome%20through%20targeted%20bisulfite%20sequencing&amp;author=E.J.%20Lee&amp;author=J.%20Luo&amp;author=J.M.%20Wilson&amp;author=H.%20Shi&amp;publication_year=2013&amp;journal=Cancer%20Lett.&amp;volume=340&amp;pages=171-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.canlet.2012.10.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.canlet.2012.10.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.canlet.2012.10.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23200671" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miura</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Enomoto</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dairiki</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ito</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Amplification-free whole-genome bisulfite sequencing by post-bisulfite adaptor tagging</div>
          <div class="source">Nucleic Acids Res.</div>
          <div class="year">2012</div>
          <div class="volume">40</div>
          <div class="fpage">e136</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amplification-free%20whole-genome%20bisulfite%20sequencing%20by%20post-bisulfite%20adaptor%20tagging&amp;author=F.%20Miura&amp;author=Y.%20Enomoto&amp;author=R.%20Dairiki&amp;author=T.%20Ito&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks454" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks454" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks454"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22649061" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Adey</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shendure</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Ultra-low-input, tagmentation-based whole-genome bisulfite sequencing</div>
          <div class="source">Genome Res.</div>
          <div class="year">2012</div>
          <div class="volume">22</div>
          <div class="fpage">1139</div>
          <div class="lpage">43</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultra-low-input%2C%20tagmentation-based%20whole-genome%20bisulfite%20sequencing&amp;author=A.%20Adey&amp;author=J.%20Shendure&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=1139-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.136242.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.136242.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.136242.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22466172" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sakurai</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Imai</div>
               <div class="given-names">M.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Contribution of intragenic DNA methylation in mouse gametic DNA methylomes to establish oocyte-specific heritable marks</div>
          <div class="source">PLoS Genet.</div>
          <div class="year">2012</div>
          <div class="volume">8</div>
          <div class="fpage">e1002440</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contribution%20of%20intragenic%20DNA%20methylation%20in%20mouse%20gametic%20DNA%20methylomes%20to%20establish%20oocyte-specific%20heritable%20marks&amp;author=H.%20Kobayashi&amp;author=T.%20Sakurai&amp;author=M.%20Imai&amp;publication_year=2012&amp;journal=PLoS%20Genet.&amp;volume=8&amp;pages=e1002440" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002440" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002440" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002440"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22242016" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shirane</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Toh</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">H.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Mouse oocyte methylomes at base resolution reveal genome-wide accumulation of non-CpG methylation and role of DNA methyltransferases</div>
          <div class="source">PLoS Genet.</div>
          <div class="year">2013</div>
          <div class="volume">9</div>
          <div class="fpage">e1003439</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mouse%20oocyte%20methylomes%20at%20base%20resolution%20reveal%20genome-wide%20accumulation%20of%20non-CpG%20methylation%20and%20role%20of%20DNA%20methyltransferases&amp;author=K.%20Shirane&amp;author=H.%20Toh&amp;author=H.%20Kobayashi&amp;publication_year=2013&amp;journal=PLoS%20Genet.&amp;volume=9&amp;pages=e1003439" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23637617" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sakurai</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miura</div>
               <div class="given-names">F.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">High resolution DNA methylome analysis of primordial germ cells identifies gender-specific reprogramming in mice</div>
          <div class="source">Genome Res.</div>
          <div class="year">2013</div>
          <div class="volume">23</div>
          <div class="fpage">616</div>
          <div class="lpage">27</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20resolution%20DNA%20methylome%20analysis%20of%20primordial%20germ%20cells%20identifies%20gender-specific%20reprogramming%20in%20mice&amp;author=H.%20Kobayashi&amp;author=T.%20Sakurai&amp;author=F.%20Miura&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=616-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.148023.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.148023.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.148023.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23410886" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smallwood</div>
               <div class="given-names">S.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">H.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Angermueller</div>
               <div class="given-names">C.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Single-cell genome-wide bisulfite sequencing for assessing epigenetic heterogeneity</div>
          <div class="source">Nat. Methods</div>
          <div class="year">2014</div>
          <div class="volume">11</div>
          <div class="fpage">817</div>
          <div class="lpage">20</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20genome-wide%20bisulfite%20sequencing%20for%20assessing%20epigenetic%20heterogeneity&amp;author=S.A.%20Smallwood&amp;author=H.J.%20Lee&amp;author=C.%20Angermueller&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=817-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25042786" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miura</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ito</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">PBAT: post-bisulfite adaptor tagging for highly sensitive whole-genome bisulfite sequencing</div>
          <div class="source">Methods Mol. Biol.</div>
          <div class="year">2014</div>
          <div class="comment">in press</div>
        </div></div></div></div><div content-id="DSU034C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Booth</div>
               <div class="given-names">M.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Branco</div>
               <div class="given-names">M.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ficz</div>
               <div class="given-names">G.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Quantitative sequencing of 5-methylcytosine and 5-hydroxymethylcytosine at single-base resolution</div>
          <div class="source">Science</div>
          <div class="year">2012</div>
          <div class="volume">336</div>
          <div class="fpage">934</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20sequencing%20of%205-methylcytosine%20and%205-hydroxymethylcytosine%20at%20single-base%20resolution&amp;author=M.J.%20Booth&amp;author=M.R.%20Branco&amp;author=G.%20Ficz&amp;publication_year=2012&amp;journal=Science&amp;volume=336&amp;pages=934-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1220671" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1220671" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1220671"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22539555" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU034C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU034C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hon</div>
               <div class="given-names">G.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Szulwach</div>
               <div class="given-names">K.E.</div>
            </div>
            et al. 
          </div>
          <div class="article-title">Base-resolution analysis of 5-hydroxymethylcytosine in the mammalian genome</div>
          <div class="source">Cell</div>
          <div class="year">2012</div>
          <div class="volume">149</div>
          <div class="fpage">1368</div>
          <div class="lpage">80</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-resolution%20analysis%20of%205-hydroxymethylcytosine%20in%20the%20mammalian%20genome&amp;author=M.%20Yu&amp;author=G.C.%20Hon&amp;author=K.E.%20Szulwach&amp;publication_year=2012&amp;journal=Cell&amp;volume=149&amp;pages=1368-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.04.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.04.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.04.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22608086" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754540&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093_dnares_dsu034/2/dsu034_Supplementary_Data.zip?Expires=2147483647&amp;Signature=VzKaW64MyeWPrB7nQoMweOCDadrXG~TXf8W1pGhSBdDQ5JYDg5cjvuzhvoy-aX6C4997LjHGXzdobSVCXDNz-KDckJb-1gqASgZCCQH8uLVP8v4VrelaAoxnh41MdM04x7A-5cnLNEGj657VbD57nGq4HWPRdOk6qL7CA8-M3aS4vmt8ulKQz-qfD8DC~TqpfyPyJ5TYRhbL5GQQbEcvbIBXkE0aUtNMtckNtgY8~BW90-p8Rqm8auktbrA61hRd9TKS3jatQrWNqIcYO~ySz5N2-~7svabjhBCndA6U~M1-y0qblugM8Z3v5tGcxdsqmPJ-AbHYWjyuzGESukTYqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,114</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">16</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu034"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754540"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f13%2f2754540">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (16)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F13%2F2754540" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754540'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Highly sensitive targeted methylome sequencing by post-bisulfite adaptor tagging" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu034");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f13%2f2754540">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

