

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/6/343/484317","@type":"ScholarlyArticle","name":"Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes","datePublished":"2010-09-23","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2010-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq022","keywords":["gene pairs","grass genomes","conservation","rearrangement","coexpression"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/6/343/484317","author":[{"name":"Krom, Nicholas","affiliation":"Department of Biological Sciences ,  Michigan Technological University ,  Houghton, MI 49931 ,  USA","@type":"Person"},{"name":"Ramakrishna, Wusirika","affiliation":"Department of Biological Sciences ,  Michigan Technological University ,  Houghton, MI 49931 ,  USA","@type":"Person"}],"description":"Abstract.  Gene order and content differ among homologous regions of closely related genomes. Similarities in the expression profiles of physically adjacent gen","pageStart":"343","pageEnd":"352","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes" />
    <meta property="og:description" content="Abstract.  Gene order and content differ among homologous regions of closely related genomes. Similarities in the expression profiles of physically adjacent gen" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/6/343/484317" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-09-23" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Krom, Nicholas" /><meta name="citation_author_institution" content="Department of Biological Sciences, Michigan Technological University, Houghton, MI 49931, USA" /><meta name="citation_author" content="Ramakrishna, Wusirika" /><meta name="citation_author_institution" content="Department of Biological Sciences, Michigan Technological University, Houghton, MI 49931, USA" /><meta name="citation_title" content="Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes" /><meta name="citation_firstpage" content="343" /><meta name="citation_lastpage" content="352" /><meta name="citation_doi" content="10.1093/dnares/dsq022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2010/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Comparative analysis of divergent and convergent gene pairs and their expression patterns in rice, Arabidopsis, and Populus; Plant Physiol.;  citation_year=2008;  citation_volume=147;  citation_pages=1763-73; " /><meta name="citation_reference" content="citation_title=Date of the monocot-dicot divergence estimated from chloroplast DNA sequence data; Proc. Natl Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=6201-5; " /><meta name="citation_reference" content="citation_title=Dating the monocot–dicot divergence and the origin of core eudicots using whole chloroplast genomes; J. Mol. Evol.;  citation_year=2004;  citation_volume=58;  citation_pages=424-41; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; " /><meta name="citation_reference" content="citation_title=Poaceae genomes: going from unattainable to becoming a model clade for comparative plant genomics; Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=111-6; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project (IRGSP); Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses; Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; " /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics; Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of the model grass Brachypodium distachyon; citation_author=International Brachypodium Initiative (IBI); Nature;  citation_year=2010;  citation_volume=463;  citation_pages=763-8; " /><meta name="citation_reference" content="citation_title=Numerous small rearrangements of gene content, order and orientation differentiate grass genomes; Plant Mol. Biol.;  citation_year=2002;  citation_volume=48;  citation_pages=821-7; " /><meta name="citation_reference" content="citation_title=A complex history of rearrangement in an orthologous region of the maize, sorghum and rice genomes; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=12265-70; " /><meta name="citation_reference" content="citation_title=The evolutionary dynamics of eukaryotic gene order; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=299-310; " /><meta name="citation_reference" content="citation_title=Complex genomic rearrangements lead to novel primate gene function; Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=343-51; " /><meta name="citation_reference" content="citation_title=Arabidopsis MPSS. An online resource for quantitative expression analysis; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=801-13; " /><meta name="citation_reference" content="citation_title=RiceArrayNet: a database for correlating gene expression from transcriptome profiling, and its application to the analysis of coexpressed genes in rice; Plant Physiol.;  citation_year=2009;  citation_volume=151;  citation_pages=16-33; " /><meta name="citation_reference" content="citation_title=Gene loss and movement in the maize genome; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1924-31; " /><meta name="citation_reference" content="citation_title=Rice Annotation Project; citation_author=The Rice Annotation Project Database (RAP-DB): 2008 update; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_issue=Database issue;  citation_pages=D1028-33; " /><meta name="citation_reference" content="citation_title=Consistent over-estimation of gene number in complex plant genomes; Curr. Opin. Plant Biol.;  citation_year=2004;  citation_volume=7;  citation_pages=732-6; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative analysis of putative bidirectional promoters from rice, Arabidopsis, and Populus,; Gene;  citation_year=2009;  citation_volume=429;  citation_pages=65-73; " /><meta name="citation_reference" content="citation_title=Conservation of divergent transcription in fungi; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=207-11; " /><meta name="citation_reference" content="citation_title=Clusters of co-expressed genes in mammalian genomes are conserved by natural selection; Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=767-75; " /><meta name="citation_reference" content="citation_title=Evolutionary origin and maintenance of coexpressed gene clusters in mammals; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1715-23; " /><meta name="citation_reference" content="citation_title=Identification and characterization of lineage-specific genes within the Poaceae; Plant Physiol.;  citation_year=2007;  citation_volume=145;  citation_pages=1311-22; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/6/343/1309279/dsq022.pdf" /><meta name="description" content="Abstract.  Gene order and content differ among homologous regions of closely related genomes. Similarities in the expression profiles of physically adjacent gen" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/6/343/484317" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/6/343/484317" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/6/343/484317");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq022';
    </script>
<title>
	Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484317" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bf40PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLcm9tLCBOaWNob2xhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTWljaGlnYW4gVGVjaG5vbG9naWNhbCBVbml2ZXJzaXR5LCBIb3VnaHRvbiwgTUkgNDk5MzEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJhbWFrcmlzaG5hLCBXdXNpcmlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgTWljaGlnYW4gVGVjaG5vbG9naWNhbCBVbml2ZXJzaXR5LCBIb3VnaHRvbiwgTUkgNDk5MzEsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29uc2VydmF0aW9uLCBSZWFycmFuZ2VtZW50LCBhbmQgRGVsZXRpb24gb2YgR2VuZSBQYWlycyBEdXJpbmcgdGhlIEV2b2x1dGlvbiBvZiBGb3VyIEdyYXNzIEdlbm9tZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM1MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgZGl2ZXJnZW50IGFuZCBjb252ZXJnZW50IGdlbmUgcGFpcnMgYW5kIHRoZWlyIGV4cHJlc3Npb24gcGF0dGVybnMgaW4gcmljZSwgQXJhYmlkb3BzaXMsIGFuZCBQb3B1bHVzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xNzYzLTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EYXRlIG9mIHRoZSBtb25vY290LWRpY290IGRpdmVyZ2VuY2UgZXN0aW1hdGVkIGZyb20gY2hsb3JvcGxhc3QgRE5BIHNlcXVlbmNlIGRhdGE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz02MjAxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGluZyB0aGUgbW9ub2NvdOKAk2RpY290IGRpdmVyZ2VuY2UgYW5kIHRoZSBvcmlnaW4gb2YgY29yZSBldWRpY290cyB1c2luZyB3aG9sZSBjaGxvcm9wbGFzdCBnZW5vbWVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9NDI0LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIGJsYWNrIGNvdHRvbndvb2QsIFBvcHVsdXMgdHJpY2hvY2FycGEgKFRvcnIuICZhbXA7IEdyYXkpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtNjA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2FjZWFlIGdlbm9tZXM6IGdvaW5nIGZyb20gdW5hdHRhaW5hYmxlIHRvIGJlY29taW5nIGEgbW9kZWwgY2xhZGUgZm9yIGNvbXBhcmF0aXZlIHBsYW50IGdlbm9taWNzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xMTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1hcC1iYXNlZCBzZXF1ZW5jZSBvZiB0aGUgcmljZSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIFJpY2UgR2Vub21lIFNlcXVlbmNpbmcgUHJvamVjdCAoSVJHU1ApOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM2OyAgY2l0YXRpb25fcGFnZXM9NzkzLTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNvcmdodW0gYmljb2xvciBnZW5vbWUgYW5kIHRoZSBkaXZlcnNpZmljYXRpb24gb2YgZ3Jhc3NlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ1NzsgIGNpdGF0aW9uX3BhZ2VzPTU1MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQjczIG1haXplIGdlbm9tZTogY29tcGxleGl0eSwgZGl2ZXJzaXR5LCBhbmQgZHluYW1pY3M7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9MTExMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyBhbmQgYW5hbHlzaXMgb2YgdGhlIG1vZGVsIGdyYXNzIEJyYWNoeXBvZGl1bSBkaXN0YWNoeW9uOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBCcmFjaHlwb2RpdW0gSW5pdGlhdGl2ZSAoSUJJKTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTc2My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdW1lcm91cyBzbWFsbCByZWFycmFuZ2VtZW50cyBvZiBnZW5lIGNvbnRlbnQsIG9yZGVyIGFuZCBvcmllbnRhdGlvbiBkaWZmZXJlbnRpYXRlIGdyYXNzIGdlbm9tZXM7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz04MjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wbGV4IGhpc3Rvcnkgb2YgcmVhcnJhbmdlbWVudCBpbiBhbiBvcnRob2xvZ291cyByZWdpb24gb2YgdGhlIG1haXplLCBzb3JnaHVtIGFuZCByaWNlIGdlbm9tZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTAwOyAgY2l0YXRpb25fcGFnZXM9MTIyNjUtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgZXVrYXJ5b3RpYyBnZW5lIG9yZGVyOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV4IGdlbm9taWMgcmVhcnJhbmdlbWVudHMgbGVhZCB0byBub3ZlbCBwcmltYXRlIGdlbmUgZnVuY3Rpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MzQzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBNUFNTLiBBbiBvbmxpbmUgcmVzb3VyY2UgZm9yIHF1YW50aXRhdGl2ZSBleHByZXNzaW9uIGFuYWx5c2lzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzU7ICBjaXRhdGlvbl9wYWdlcz04MDEtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpY2VBcnJheU5ldDogYSBkYXRhYmFzZSBmb3IgY29ycmVsYXRpbmcgZ2VuZSBleHByZXNzaW9uIGZyb20gdHJhbnNjcmlwdG9tZSBwcm9maWxpbmcsIGFuZCBpdHMgYXBwbGljYXRpb24gdG8gdGhlIGFuYWx5c2lzIG9mIGNvZXhwcmVzc2VkIGdlbmVzIGluIHJpY2U7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTE2LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGxvc3MgYW5kIG1vdmVtZW50IGluIHRoZSBtYWl6ZSBnZW5vbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTkyNC0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZSBBbm5vdGF0aW9uIFByb2plY3Q7IGNpdGF0aW9uX2F1dGhvcj1UaGUgUmljZSBBbm5vdGF0aW9uIFByb2plY3QgRGF0YWJhc2UgKFJBUC1EQik6IDIwMDggdXBkYXRlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9pc3N1ZT1EYXRhYmFzZSBpc3N1ZTsgIGNpdGF0aW9uX3BhZ2VzPUQxMDI4LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zaXN0ZW50IG92ZXItZXN0aW1hdGlvbiBvZiBnZW5lIG51bWJlciBpbiBjb21wbGV4IHBsYW50IGdlbm9tZXM7IEN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz03MzItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgcHV0YXRpdmUgYmlkaXJlY3Rpb25hbCBwcm9tb3RlcnMgZnJvbSByaWNlLCBBcmFiaWRvcHNpcywgYW5kIFBvcHVsdXMsOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQyOTsgIGNpdGF0aW9uX3BhZ2VzPTY1LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2YgZGl2ZXJnZW50IHRyYW5zY3JpcHRpb24gaW4gZnVuZ2k7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0ZXJzIG9mIGNvLWV4cHJlc3NlZCBnZW5lcyBpbiBtYW1tYWxpYW4gZ2Vub21lcyBhcmUgY29uc2VydmVkIGJ5IG5hdHVyYWwgc2VsZWN0aW9uOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NzY3LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgb3JpZ2luIGFuZCBtYWludGVuYW5jZSBvZiBjb2V4cHJlc3NlZCBnZW5lIGNsdXN0ZXJzIGluIG1hbW1hbHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNzE1LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBsaW5lYWdlLXNwZWNpZmljIGdlbmVzIHdpdGhpbiB0aGUgUG9hY2VhZTsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQ1OyAgY2l0YXRpb25fcGFnZXM9MTMxMS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE3LzYvMzQzLzEzMDkyNzkvZHNxMDIyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBHZW5lIG9yZGVyIGFuZCBjb250ZW50IGRpZmZlciBhbW9uZyBob21vbG9nb3VzIHJlZ2lvbnMgb2YgY2xvc2VseSByZWxhdGVkIGdlbm9tZXMuIFNpbWlsYXJpdGllcyBpbiB0aGUgZXhwcmVzc2lvbiBwcm9maWxlcyBvZiBwaHlzaWNhbGx5IGFkamFjZW50IGdlbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDg0MzE3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCdp75+3cZXm56Lx4wWHbmhpW4up" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f343%2f484317" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f343%2f484317" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19064" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&amp;Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/6">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7356621">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356621">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356625">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356625">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356636">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356636">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356653">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356653">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356690">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356690">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356692">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356692">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7356695">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7356695">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/6/325/485103">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/6/353/484440">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicholas Krom</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Michigan Technological University</div>, <div class="addr-line">Houghton, MI 49931</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicholas+Krom">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Krom N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Krom N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicholas Krom</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wusirika Ramakrishna</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences</div>, <div class="insititution">Michigan Technological University</div>, <div class="addr-line">Houghton, MI 49931</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 906-487-3068. Fax. <div class="fax">+1 906-487-3167</div>. E-mail: <a href="mailto:wusirika@mtu.edu" target="_blank">wusirika@mtu.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wusirika+Ramakrishna">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramakrishna W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ramakrishna W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wusirika Ramakrishna</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 343–352, <a href='https://doi.org/10.1093/dnares/dsq022'>https://doi.org/10.1093/dnares/dsq022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">23 September 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 March 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 September 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484317" href="/dnaresearch/article-pdf/17/6/343/1309279/dsq022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Nicholas Krom, Wusirika Ramakrishna;  Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes, <em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 343–352, <a href='https://doi.org/10.1093/dnares/dsq022'>https://doi.org/10.1093/dnares/dsq022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484317?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484317?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Conservation%2C%20Rearrangement%2C%20and%20Deletion%20of%20Gene%20Pairs%20During%20the%20Evolution%20of%20Four%20Grass%20Genomes&amp;publicationDate=2010-09-23&amp;volumeNum=17&amp;issueNum=6&amp;author=Krom%2C%20Nicholas%3B%20Ramakrishna%2C%20Wusirika&amp;startPage=343&amp;endPage=352&amp;contentId=10.1093%2Fdnares%2Fdsq022&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f343%2f484317" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7356621 id="7356621" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Gene order and content differ among homologous regions of closely related genomes. Similarities in the expression profiles of physically adjacent genes suggest that the proper functioning of these genes depends on maintaining a specific position relative to each other. To better understand the results of the interaction of these two genomic forces, convergent, divergent, and tandem gene pairs in rice and sorghum, as well as their homologs in rice, sorghum, maize, and <em>Brachypodium</em> were analyzed. The status of each pair in all four species: whether it was conserved, inverted, rearranged, or missing homologs was determined. We observed that divergent gene pairs had lower rates of conservation than convergent or tandem pairs, but higher rates of rearranged pairs and missing homologs in maize than in any other species. We also discovered species-specific gene pairs in rice and sorghum. In rice, gene pairs with strongly correlated expression levels were conserved significantly more often than those with little or no correlation. We assigned three types of gene pair to one of 14 possible evolutionary history categories to uncover their evolutionary dynamics during the evolution of grass genomes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene pairs&quot;">gene pairs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;grass genomes&quot;">grass genomes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="conservation">conservation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rearrangement">rearrangement</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="coexpression">coexpression</a></div>                            <h2 scrollto-destination=7356625 id="7356625" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>One of the primary areas of investigation in comparative genomics is the identification and characterization of homologous regions in closely related genomes. The subjects of these investigations range in scale from multi-megabase syntenic regions covering most of a chromosome to small loci containing just a few genes. Studying the syntenic regions can uncover large-scale events in the evolutionary history of a genome, such as segmental duplications or polyploidization; however, these regions in different species can differ significantly. Such variation results from a large number of genomic alterations occurring over time while maintaining sufficient collinearity to define regions of synteny. On the other hand, comparative analysis of small loci can produce detailed evolutionary histories of groups of neighboring genes and provide examples of the types of changes possible in a genome. However, it is difficult to expand these studies to a genome-wide scale due to the number of genes involved and the problem of generalizing these types of changes to allow their quantification.</p><p>In this study, we conduct an intermediate form of comparative analysis. By examining pairs of adjacent genes, we can detect changes at the level of single genes and still observe relationships between genes. Owing to the simplicity and small scale of our subjects, all changes can be assigned to a manageable number of classes, thereby producing results that are easily interpreted in genome-wide studies of this type. Our previous investigation compared gene pairs<sup><a reveal-id="DSQ022C1" data-open="DSQ022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> in three plant species (rice, <em>Arabidopsis</em>, and <em>Populus</em>) which diverged 130–200 million years ago (mya).<sup><a reveal-id="DSQ022C2  DSQ022C4" data-open="DSQ022C2  DSQ022C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> This study, on the other hand, compares four members of the Poaceae family (rice, sorghum, maize, and <em>Brachypodium</em>) whose last common ancestor dates to 50–70 mya.<sup><a reveal-id="DSQ022C2" data-open="DSQ022C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSQ022C5" data-open="DSQ022C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Genomes of the four grass species used in this study have been sequenced.<sup><a reveal-id="DSQ022C6  DSQ022C9" data-open="DSQ022C6  DSQ022C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> The shorter evolutionary distances separating these species simplifies the interpretation of any observed genomic rearrangements, due to the reduced probability of multiple independent events affecting the same region. However, many small rearrangements have been identified in earlier comparative studies of Poaceae genomes,<sup><a reveal-id="DSQ022C10" data-open="DSQ022C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSQ022C11" data-open="DSQ022C11" class="link link-ref link-reveal xref-bibr">11</a></sup> providing sufficient variation among genomes to identify any trends regarding selection for or against disruption of ancestral gene pairs. It has been hypothesized that gene order is not entirely random, but rather is connected to gene function and regulation,<sup><a reveal-id="DSQ022C12" data-open="DSQ022C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and that genomic rearrangements can alter the function of genes or even lead to the creation of new gene families. Thus, gene order possibly contributes to phenotypic differences between species, even when individual genes are conserved.<sup><a reveal-id="DSQ022C13" data-open="DSQ022C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>We previously reported<sup><a reveal-id="DSQ022C1" data-open="DSQ022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> that their strand-wise arrangement has a significant influence on many characteristics of gene pairs. For this study, we classified all pairs of adjacent genes as convergent (→ ←), divergent (← →), or tandem (→ → or ← ←), we identified homologous genes in other species and determined the status of each pair (conserved, inverted, moved, or missing homologs). We also estimated the effect of correlated expression on these types of gene-pair rearrangements. To gain an understanding of the evolutionary timing of the rearrangements we observed, a putative evolutionary history was created for each gene pair, based on its status in each of the four species. Overall, this study provides an overview of the frequencies and types of genomic rearrangements within a subset of the Poaceae, as well as many other properties of the genomes being studied.</p>                            <h2 scrollto-destination=7356636 id="7356636" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7356637 id="7356637" class="section-title" data-legacy-id=s2a>2.1. Identification of gene pairs</h3>
<p>Genome sequence and annotation data were downloaded for rice (<em>Oryza sativa</em> subsp. <em>japonica</em>; <a class="link link-uri" href="http://rice.plantbiology.msu.edu">http://rice.plantbiology.msu.edu</a>, MSU rice pseudomolecules release 6), sorghum (<em>Sorghum bicolor</em>; <a class="link link-uri" href="http://www.phytozome.net/sorghum">http://www.phytozome.net/sorghum</a>, sequence assembly v1.0, gene set v1.4), maize (<em>Zea mays</em>; <a class="link link-uri" href="http://www.maizesequence.org">http://www.maizesequence.org</a>, release 3a.50), and <em>Brachypodium</em> (<em>Brachypodium distachyon</em>; <a class="link link-uri" href="http://www.brachypodium.org">http://www.brachypodium.org</a>, 8X coverage release). A second set of rice sequence and annotation data was obtained from the Rice Annotation Project (RAP; <a class="link link-uri" href="http://rapdb.dna.affrc.go.jp/">http://rapdb.dna.affrc.go.jp/</a>, sequence build 5) for comparison with the MSU rice gene set and was processed and analyzed using the same methods as the MSU rice and sorghum gene sets. Annotated genes in the rice and sorghum genomes were sorted by chromosome and position and then, based on which strand the gene is transcribed from, all pairs of adjacent genes were classified as either convergent (→ ←), divergent (← →), or tandem (→ → or ← ←) pairs. Pairs containing hypothetical- or transposon-related genes, as determined by annotation and RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">www.repeatmasker.org</a>; 50% or greater transposon content of unspliced sequence), were excluded from all analyses.</p>                            <h3 scrollto-destination=7356641 id="7356641" class="section-title" data-legacy-id=s2b>2.2. Comparative sequence analysis</h3>
<p>The coding region sequences of all rice and sorghum gene pairs were aligned with the genome assemblies of the other three species using BLASTN. For each gene, individual hits (presumably corresponding to single exons) with <em>e</em>-values of 0.00001 or less were grouped with other nearby hits on the same strand and contig to produce a putative homolog. The locations of each pair's homologs were then used to determine the pair's status in that species. Pairs were considered ‘conserved' if both genes had homologs in the original strand-wise arrangement (convergent, divergent, or tandem) within 50 kb of each other and had no other genes inserted between them. Fifty kilobases were chosen to allow for insertions of repetitive DNA such as nested retrotransposons. ‘Inverted' pairs also possessed homologs within the cutoff distance, but had different strand-wise arrangements than the original pair. Pairs were considered ‘rearranged' if homologs for both genes were found but were &gt;50 kb apart, separated by other genes, or located on different contigs.  Those pairs in which one or both genes were missing homologs in a given species were also identified. In addition to 50 kb, genes residing within 20 and 75 kb were used to perform the above analysis.</p>                            <h3 scrollto-destination=7356643 id="7356643" class="section-title" data-legacy-id=s2c>2.3. Expression analysis</h3>
<p>Two types of quantitative expression data were collected for all rice genes (MSU data): microarray and massively parallel signature sequencing (MPSS). First, MPSS<sup><a reveal-id="DSQ022C14" data-open="DSQ022C14" class="link link-ref link-reveal xref-bibr">14</a></sup> data were downloaded from the Rice MPSS Database (<a class="link link-uri" href="http://mpss.udel.edu/rice/">http://mpss.udel.edu/rice/</a>). Only 17-bp signatures of classes 1, 2, 5, and 7 that mapped to a single gene were used, and abundance values &lt;5 were ignored as background interference. When multiple signatures had significant abundance values in the same library, their average abundance was used. Correlated expression between genes in convergent, divergent, and tandem pairs was examined by calculating the Pearson correlation coefficient using each gene's average abundance values across 72 libraries.</p><p>Microarray data were downloaded from the Yale rice project (<a class="link link-uri" href="http://bioinformatics.med.yale.edu/rc/overview.jspx">http://bioinformatics.med.yale.edu/rc/overview.jspx</a>) for a total of 446 hybridizations. Correlated expression was again tested with the Pearson correlation coefficient, this time pairing data points for each gene from the same hybridization and channel, where data were available for both genes for that hybridization and channel.</p><p>Correlation coefficients were calculated separately for each expression data type. Thus, if expression data were present for both genes in both sets, a gene pair would have both an MPSS-derived and microarray-derived correlation coefficient. For those pairs with both coefficients, the pair was considered strongly correlated if at least one was greater than the cut-off value.</p><p>The fraction of strongly correlated gene pairs falling into each of the four categories (conserved, inverted, rearranged, or missing homologs) was then compared with the fractions of uncorrelated or weakly correlated pairs in those same categories. The statistical significance of their differences was evaluated using the normal approximation of the binomial test, with a significance level of <em>P</em> &lt; 0.05 (<em>Z</em> &gt; 1.6449). Two definitions of ‘strong' correlation were used: the first one using a correlation coefficient cut-off of 0.5 used in other studies,<sup><a reveal-id="DSQ022C1" data-open="DSQ022C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ022C15" data-open="DSQ022C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and the second one based on a <em>P</em> &lt; 0.05 significance level for a normal distribution with a mean of 0.187 and a standard deviation of 0.247, the sample statistics from the set of all calculated correlation coefficients, giving a minimum ‘strong' correlation of 0.594.</p>                            <h3 scrollto-destination=7356649 id="7356649" class="section-title" data-legacy-id=s2d>2.4. Evolutionary analysis of gene pairs</h3>
<p>The evolutionary history of each gene pair was constructed by comparing the status of the pair in each of the four species in this study. The likelihood of a given scenario was based on the number of gene rearrangements, deletions, and conservation using the fewest possible changes to the homologous regions of the last common ancestor to arrive at the present state. Gene pairs were then assigned to one of 14 groups based on their putative histories that could produce the observed results of the comparative analysis. Rice MSU data were used for this analysis.</p>                            <h2 scrollto-destination=7356653 id="7356653" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7356654 id="7356654" class="section-title" data-legacy-id=s3a>3.1. Conservation and rearrangements</h3>
<p>Convergent, divergent, and tandem gene pairs in the rice and sorghum genomes were identified as described in methods. Four thousand and eight hundred convergent pairs, 3711 divergent pairs, and 9428 tandem pairs from the MSU rice gene set and 5059 convergent, 4913 divergent, and 11847 tandem pairs from the sorghum gene set were identified.</p><p>The primary goal of this analysis was to determine how frequently the exact arrangement of a pair of adjacent genes is conserved in the genomes of other grass species and what changes have taken place when the pair is not conserved. Out of the four grass species selected for this study, rice and sorghum were chosen as starting points for comparisons because their sequence and annotation data sets were considerably better than those of maize and <em>Brachypodium</em>. Our comparative sequence analysis placed each rice or sorghum gene pair into one of four categories based on the presence or the absence of homologous genes and their locations in the genome. A pair of adjacent genes in rice/sorghum was considered to be conserved in other genomes when both the individual sequences of genes and the strand-wise arrangement of the pair were conserved. If a pair's homologs were found to be still adjacent but with a different strand-wise arrangement, then the pair was designated ‘inverted'. Homologs falling on different contigs, separated by other genes, or &gt;50 kb apart were considered ‘rearranged'. Finally, one or both genes in the original pair may be lacking homologs because they were deleted in that species’ lineage or they arose in the ancestors of rice or sorghum after diverging from their last common ancestor. Together, these categories include all the major events of genomic evolution at this scale, and the relative frequencies of these events provide insight into the importance of proximity and strand-wise arrangement to proper gene function and regulation. To estimate the statistical significance of the variation observed among pair types, we compared the fraction of pairs of each type that fell into each category with the corresponding fraction of all pairs in that category.</p><p>Conservation rates for both rice and sorghum gene pairs (Figs <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSQ022F2" data-open="DSQ022F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>) followed quite similar patterns. In every comparison, divergent gene pairs were conserved least often, with conservation frequencies significantly below the all pair average. Similarly, convergent pairs were conserved significantly more often than the all pair average. Tandem pair conservation rates were generally close to the all pair average. The conservation frequency differed a great deal among pair types in maize, where divergent pairs are conserved at rates close to half that of convergent or tandem pairs. Further, in <em>Brachypodium</em>, rice gene pairs are more frequently conserved than sorghum pairs. Although <em>Brachypodium</em> appears to be closer to rice than sorghum based on the divergence times of 40–53 Myr for <em>Brachypodium</em>–rice and 45–60 Myr for <em>Brachypodium</em>–sorghum based on synonymous substitution rates (Ks) of orthologous gene pairs,<sup><a reveal-id="DSQ022C9" data-open="DSQ022C9" class="link link-ref link-reveal xref-bibr">9</a></sup> these evolutionary relationships are not well resolved. Likewise, maize is more closely related to sorghum than to rice<sup><a reveal-id="DSQ022C11" data-open="DSQ022C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSQ022C16" data-open="DSQ022C16" class="link link-ref link-reveal xref-bibr">16</a></sup>; thus, sorghum pairs have higher rates of conservation in maize than do rice pairs. </p>                        <a id="7356658" scrollto-destination="7356658"></a>
<div id="DSQ022F1" data-id="DSQ022F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02201.jpeg?Expires=2147483647&Signature=2oWISCtzVeF6Hk2xJEShyGhsN8mmQ-7DShTusabiGrpOTj9GihLeL514Wh94TGq88~voE5RrZds-IY7EWlKQOlinB9Ad6vQIhJYV5KCPKkRmMOhqs~Rw3fIm97odATFUe8z3ZEHc4bTqlJb5icCiJWOYHRMbmxv7V7jQwSWlFtQmrp4vvrtlbXUD8DiGEaU8f8RxNRm~pV2bsZkkI3Po~hUG-NbZHezQf61Ku2RGs-4WRkRM7TiiAIccbUYoiVtadWk5tCY-AAcqU7R21pf2bl1ljf7JrLp4BKzQYrsl6GY~693TG7KdHnfvLWEzPKkhP2Az6Y~ywD4kwrR-OvjAAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation and rearrangement of rice gene pairs (MSU data) in sorghum, maize, and Brachypodium.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (P &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using Z-test." path-from-xml="dsq02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356658/dsq02201.jpeg" path-from-xml="dsq02201.jpeg" target="_blank">View large</a><a section="7356658" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02201.jpeg?Expires=2147483647&Signature=M6q-ZxvYLCQ-85~1FyJoLyc1bM8CkJXOzUYOv9kU4v9UiuYmBzmrRKJFZIH-sZBTlRMEUQnqMv92R4GpsHSz1nIMqoZQvTW~eTDm4fv~Tp9PqPeb1S-Jna3JJJ~BTAvWr4IQT~1M2Amo8IuJBfvJ8h6~trwSGuJX3z6T~cLRzTw3kiWt4fos3IBR0lrE5qlemJ0K12vVYTKVU9uOnFFXeb1kwPHL~uPxTbWMHVpdRQ5KvymXv1f5lpNyey~YpYfHneW4GIo4-vh4e597htY0GndfmERqHuUxEw-pc53xtJH0WduaBYc8P~5HkhvQxA17Gu2MYZyGk56Lk69f8VbE7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356658&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conservation and rearrangement of rice gene pairs (MSU data) in sorghum, maize, and <em>Brachypodium</em>.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (<em>P</em> &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using <em>Z</em>-test.</p></div></div><div content-id="DSQ022F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02201.jpeg?Expires=2147483647&Signature=2oWISCtzVeF6Hk2xJEShyGhsN8mmQ-7DShTusabiGrpOTj9GihLeL514Wh94TGq88~voE5RrZds-IY7EWlKQOlinB9Ad6vQIhJYV5KCPKkRmMOhqs~Rw3fIm97odATFUe8z3ZEHc4bTqlJb5icCiJWOYHRMbmxv7V7jQwSWlFtQmrp4vvrtlbXUD8DiGEaU8f8RxNRm~pV2bsZkkI3Po~hUG-NbZHezQf61Ku2RGs-4WRkRM7TiiAIccbUYoiVtadWk5tCY-AAcqU7R21pf2bl1ljf7JrLp4BKzQYrsl6GY~693TG7KdHnfvLWEzPKkhP2Az6Y~ywD4kwrR-OvjAAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation and rearrangement of rice gene pairs (MSU data) in sorghum, maize, and Brachypodium.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (P &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using Z-test." path-from-xml="dsq02201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356658/dsq02201.jpeg" path-from-xml="dsq02201.jpeg" target="_blank">View large</a><a section="7356658" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02201.jpeg?Expires=2147483647&Signature=M6q-ZxvYLCQ-85~1FyJoLyc1bM8CkJXOzUYOv9kU4v9UiuYmBzmrRKJFZIH-sZBTlRMEUQnqMv92R4GpsHSz1nIMqoZQvTW~eTDm4fv~Tp9PqPeb1S-Jna3JJJ~BTAvWr4IQT~1M2Amo8IuJBfvJ8h6~trwSGuJX3z6T~cLRzTw3kiWt4fos3IBR0lrE5qlemJ0K12vVYTKVU9uOnFFXeb1kwPHL~uPxTbWMHVpdRQ5KvymXv1f5lpNyey~YpYfHneW4GIo4-vh4e597htY0GndfmERqHuUxEw-pc53xtJH0WduaBYc8P~5HkhvQxA17Gu2MYZyGk56Lk69f8VbE7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356658&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conservation and rearrangement of rice gene pairs (MSU data) in sorghum, maize, and <em>Brachypodium</em>.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (<em>P</em> &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using <em>Z</em>-test.</p></div></div>                        <a id="7356661" scrollto-destination="7356661"></a>
<div id="DSQ022F2" data-id="DSQ022F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02202.jpeg?Expires=2147483647&Signature=sZpocc7paXkpjno~wAk-vWnvhrv7AujPmDdXtF5MIA7zKML-NhtcladgUt-SNNvUw3BCWtc1KGkjkDxZhRR93IgRXpqOIITXXGVTazI-y7UQy6056PSCLb8DAH-141d~psKpPe-UWzsrbZYEERbekgMvgECpvKDAowD2gtCFFyAGbIlqRrenWH-mMbd7LxsxColHsn7x1wlKd3AHM9UcqnjJNn68kAW3oPJRHhEjqA4QmH0eWA86qByfZrbCoNePdUTC5YBKO0EhCLGL0tk88JY0am7iKsg9eYn0GVMp8bigD4qJw~E7dP0B2mhudTU~mjSDrcLuXgmTVVY14tzAtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation and rearrangement of sorghum gene pairs in rice (MSU data), maize, and Brachypodium.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (P &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using Z-test." path-from-xml="dsq02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356661/dsq02202.jpeg" path-from-xml="dsq02202.jpeg" target="_blank">View large</a><a section="7356661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02202.jpeg?Expires=2147483647&Signature=aqU85VRv2v~fzurXUfAOygaS1uyEzLjuZlBf0qnxRgj7v7uTIlH~XMOsBssks9ZOQVvfuhgUKZ42jrnxdzUB7V1cEiFD9f0liKEnIi-I-6WaHh4aATi~FDLPSf~wxBqkWd1K~NEZDB5JWYNMTz90eS-ArZxY7SL48mFq4S-NAmNiNIruYqiUxRZpJku-LeM~zG0G41TmwrOWf1NlyCFSXgXAS9i-Mw7JYUqFpW3nGaQCXbkjVmgTWmuV4K0OI1aAfe46u69-taRE8umO8D0~~76Axbyjngk9rrJog5e5N5WBAdta2E-phMj-7v1hxhnAg300EPW5WgL4DamInTj4LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356661&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conservation and rearrangement of sorghum gene pairs in rice (MSU data), maize, and <em>Brachypodium</em>.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (<em>P</em> &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using <em>Z</em>-test.</p></div></div><div content-id="DSQ022F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02202.jpeg?Expires=2147483647&Signature=sZpocc7paXkpjno~wAk-vWnvhrv7AujPmDdXtF5MIA7zKML-NhtcladgUt-SNNvUw3BCWtc1KGkjkDxZhRR93IgRXpqOIITXXGVTazI-y7UQy6056PSCLb8DAH-141d~psKpPe-UWzsrbZYEERbekgMvgECpvKDAowD2gtCFFyAGbIlqRrenWH-mMbd7LxsxColHsn7x1wlKd3AHM9UcqnjJNn68kAW3oPJRHhEjqA4QmH0eWA86qByfZrbCoNePdUTC5YBKO0EhCLGL0tk88JY0am7iKsg9eYn0GVMp8bigD4qJw~E7dP0B2mhudTU~mjSDrcLuXgmTVVY14tzAtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conservation and rearrangement of sorghum gene pairs in rice (MSU data), maize, and Brachypodium.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (P &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using Z-test." path-from-xml="dsq02202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356661/dsq02202.jpeg" path-from-xml="dsq02202.jpeg" target="_blank">View large</a><a section="7356661" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02202.jpeg?Expires=2147483647&Signature=aqU85VRv2v~fzurXUfAOygaS1uyEzLjuZlBf0qnxRgj7v7uTIlH~XMOsBssks9ZOQVvfuhgUKZ42jrnxdzUB7V1cEiFD9f0liKEnIi-I-6WaHh4aATi~FDLPSf~wxBqkWd1K~NEZDB5JWYNMTz90eS-ArZxY7SL48mFq4S-NAmNiNIruYqiUxRZpJku-LeM~zG0G41TmwrOWf1NlyCFSXgXAS9i-Mw7JYUqFpW3nGaQCXbkjVmgTWmuV4K0OI1aAfe46u69-taRE8umO8D0~~76Axbyjngk9rrJog5e5N5WBAdta2E-phMj-7v1hxhnAg300EPW5WgL4DamInTj4LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356661&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conservation and rearrangement of sorghum gene pairs in rice (MSU data), maize, and <em>Brachypodium</em>.  A ‘+' or ‘−' at the base of a column indicates that pairs in that class are significantly (<em>P</em> &lt; 0.01) overrepresented or underrepresented, respectively, compared with the general population of gene pairs using <em>Z</em>-test.</p></div></div><p>Inversion of one or both genes was quite a rare phenomenon, ranging from 0.5% of sorghum tandem pairs inverted in rice (Fig. <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a>A) to 2% of sorghum divergent pairs in maize (Fig. <a reveal-id="DSQ022F2" data-open="DSQ022F2" class="link link-reveal link-table xref-fig">2</a>B). We observed some variation among different pair types and comparison species, but the variation was not statistically significant.</p><p>Among rearranged pairs, rice and sorghum gene pairs differ more than conserved and inverted pairs. Overall, sorghum pairs are more likely to be rearranged than rice pairs, with the rearrangement rates of sorghum pairs being ∼10–20% higher than those of rice pairs (Figs <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSQ022F2" data-open="DSQ022F2" class="link link-reveal link-table xref-fig">2</a>). For both rice and sorghum pairs, rearrangement is most common in maize (Figs <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a>B and <a reveal-id="DSQ022F2" data-open="DSQ022F2" class="link link-reveal link-table xref-fig">2</a>B) by a substantial margin and is least common in <em>Brachypodium</em> (Figs <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a>C and <a reveal-id="DSQ022F2" data-open="DSQ022F2" class="link link-reveal link-table xref-fig">2</a>C). Divergent pairs are consistently the most commonly rearranged type, whereas convergent and tandem pairs are generally rearranged with similar frequencies in all comparison species.</p><p>Considerable differences between rice and sorghum were also noted among gene pairs lacking homologs for one or both genes. Rice pairs missing homologs in sorghum are roughly one-half more common than sorghum pairs missing homologs in rice (15.7% versus 9.9%). In maize, rice genes are missing homologs more than twice as often as sorghum genes (22.7% versus 10%). The fractions of rice and sorghum pairs without homologs in <em>Brachypodium</em> are almost equal. Pair type does not seem to affect the presence or the absence of homologs because none of the pair types in any comparison species deviated significantly from the all pair average.</p><p>Conservation and rearrangement rates differ between maize and the other comparison species most likely as a result of three primary sources. First, the genomic sequence of maize used in our study is in the form of individual BAC sequences, most of which are shorter than ∼250 kb, rather than in the form of assembled sequences of near-chromosome length. Smaller sequences are, of course, less likely to contain complete gene pairs, especially if their intergenic regions have accumulated other genes or transposons over time. Second, transposons make up a much larger fraction of the maize genome than that of rice, sorghum, or <em>Brachypodium</em> genomes. In addition to physically disrupting the region into which they insert themselves, transposons can also increase the likelihood of recombinations, deletions, and other alterations in any area they inhabit. Third, the ancestors of maize quite likely suffered large-scale gene loss.<sup><a reveal-id="DSQ022C16" data-open="DSQ022C16" class="link link-ref link-reveal xref-bibr">16</a></sup> If the first gene in a pair is deleted from one copy and the second gene was deleted in the other copy of the pair, both genes would still exist in the genome, but would no longer be paired. This type of occurrence would explain why more rice and sorghum gene pairs had more physically distant homologs in maize than in any other comparison species. All these factors would reduce the frequency of gene-pair conservation and correspondingly increase rearranged pairs, as we observed.</p><p>In addition to 50 kb, genes residing within 20 and 75 kb were used to perform the above analysis. Although the number of gene pairs varied, the results showed same trends that were observed with 50 kb limit (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>).</p><p>Comparative analysis was performed using a second set of rice genome annotation data, from the RAP (<a class="link link-uri" href="http://rapdb.dna.affrc.go.jp)">http://rapdb.dna.affrc.go.jp)</a>,<sup><a reveal-id="DSQ022C17" data-open="DSQ022C17" class="link link-ref link-reveal xref-bibr">17</a></sup> to evaluate the effect of different methods of genome annotation on our results. We found that the analysis of the MSU rice gene set as well as the RAP gene set exhibited similar major trends regarding differences among the three pair types and comparison species. Although many of the same conclusions about conservation and rearrangement of gene pairs could be inferred from both the MSU and RAP results, significant differences exist between the two data sets (Table <a reveal-id="DSQ022TB1" data-open="DSQ022TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). In sorghum, gene pairs identified using the RAP annotation data were conserved significantly less frequently than the MSU rice gene pairs, whereas the fractions of rearranged pairs were significantly higher among all pair types in sorghum and <em>Brachypodium</em>, and among tandem gene pairs in maize. Last, pairs missing homologs were less common overall when using the RAP annotation data, with the largest differences being found in maize and <em>Brachypodium</em>. However, comparing Fig. <a reveal-id="DSQ022F1" data-open="DSQ022F1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, and Table <a reveal-id="DSQ022TB1" data-open="DSQ022TB1" class="link link-reveal link-table xref-fig">1</a>, nine statistically significant comparisons of conserved and rearranged gene pairs were common when MSU and RAP gene sets were used as starting points. The remaining two statistically significant comparisons (rice convergent pairs conserved and rearranged in <em>Brachypodium</em>) using the MSU gene set were close to being significant using the RAP gene set. These variations are most likely due to differences in RAP and MSU annotations. The MSU gene set contains considerably more genes than the RAP set (57 840 versus 34 780), a distinction that remains even after excluding hypothetical- and transposon-related genes (29 686 versus 22 308). Therefore, it is likely that either the MSU set contains a large number of incorrectly predicted genes or the RAP set is missing a similar number of real genes. The lower rates of missing homologs among the RAP set suggests that at least some of the genes found only in the MSU set are either false positives or unique to rice. These unique genes could also be low copy number transposable elements which are transcribed but not annotated as transposable elements.<sup><a reveal-id="DSQ022C18" data-open="DSQ022C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                        <a id="7356670" scrollto-destination="7356670"></a>
<div content-id="DSQ022TB1" class="table-modal"><div class="table-wrap"><div id="DSQ022TB1" class="table-wrap-title" data-id="DSQ022TB1"><div class="label -label">Table 1.</div><div class="caption"><p>RAP rice gene-pair conservation and rearrangement</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Pairs </th><th>Conserved </th><th><em>Z</em>-value </th><th>Inverted </th><th><em>Z</em>-value </th><th>Rearranged </th><th><em>Z</em>-value </th><th>Missing homologs </th><th><em>Z</em>-value </th></tr></thead><tbody><tr><td colspan="10">Rice versus sorghum </td></tr><tr><td> Convergent </td><td>3641 </td><td>1169 (32.1%) </td><td><strong>4.91</strong> </td><td>112 (3.1%) </td><td>0.78 </td><td>1875 (51.5%) </td><td>−<strong>6.50</strong> </td><td>485 (13.3%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>648 (19.2%) </td><td>−<strong>4.01</strong> </td><td>119 (3.5%) </td><td>1.02 </td><td>2160 (64.0%) </td><td><strong>4.82</strong> </td><td>449 (13.3%) </td><td>−0.31 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1887 (24.9%) </td><td>−0.51 </td><td>39 (0.5%) </td><td>−1.12 </td><td>4580 (60.4%) </td><td>1.95 </td><td>1076 (14.2%) </td><td>0.37 </td></tr><tr><td> Total </td><td>14 599 </td><td>3704 (25.4%) </td><td> </td><td>270 (1.8%) </td><td> </td><td>8615 (59.0%) </td><td> </td><td>2010 (13.8%) </td><td> </td></tr><tr><td colspan="10">Rice versus maize </td></tr><tr><td> Convergent </td><td>3641 </td><td>987 (27.1%) </td><td><strong>3.96</strong> </td><td>103 (2.8%) </td><td>0.32 </td><td>1880 (51.6%) </td><td>−<strong>4.92</strong> </td><td>671 (18.4%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>472 (14.0%) </td><td>−<strong>4.71</strong> </td><td>146 (4.3%) </td><td>1.20 </td><td>2137 (63.3%) </td><td><strong>5.75</strong> </td><td>621 (18.4%) </td><td>−0.33 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1678 (22.1%) </td><td>0.62 </td><td>92 (1.2%) </td><td>−0.95 </td><td>4347 (57.3%) </td><td>0.04 </td><td>1465 (19.3%) </td><td>0.41 </td></tr><tr><td> Total </td><td>14 599 </td><td>3137 (21.5%) </td><td> </td><td>341 (2.3%) </td><td> </td><td>8364 (57.3%) </td><td> </td><td>2757 (18.9%) </td><td> </td></tr><tr><td colspan="10">Rice versus <em>Brachypodium</em> </td></tr><tr><td> Convergent </td><td>3641 </td><td>1429 (39.2%) </td><td>1.82 </td><td>117 (3.2%) </td><td>0.50 </td><td>1569 (43.1%) </td><td>−2.25 </td><td>526 (14.4%) </td><td>−0.23 </td></tr><tr><td> Divergent </td><td>3376 </td><td>1128 (33.4%) </td><td>−<strong>2.48</strong> </td><td>154 (4.6%) </td><td>1.29 </td><td>1587 (47.0%) </td><td>0.88 </td><td>507 (15.0%) </td><td>0.14 </td></tr><tr><td> Tandem </td><td>7582 </td><td>2830 (37.3%) </td><td>0.47 </td><td>81 (1.1%) </td><td>−1.17 </td><td>3541 (46.7%) </td><td>0.96 </td><td>1130 (14.9%) </td><td>0.10 </td></tr><tr><td> Total </td><td>14 599 </td><td>5387 (36.9%) </td><td> </td><td>352 (2.4%) </td><td> </td><td>6697 (45.9%) </td><td> </td><td>2163 (14.8%) </td><td> </td></tr></tbody></table></div><div content-id="DSQ022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Pairs </th><th>Conserved </th><th><em>Z</em>-value </th><th>Inverted </th><th><em>Z</em>-value </th><th>Rearranged </th><th><em>Z</em>-value </th><th>Missing homologs </th><th><em>Z</em>-value </th></tr></thead><tbody><tr><td colspan="10">Rice versus sorghum </td></tr><tr><td> Convergent </td><td>3641 </td><td>1169 (32.1%) </td><td><strong>4.91</strong> </td><td>112 (3.1%) </td><td>0.78 </td><td>1875 (51.5%) </td><td>−<strong>6.50</strong> </td><td>485 (13.3%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>648 (19.2%) </td><td>−<strong>4.01</strong> </td><td>119 (3.5%) </td><td>1.02 </td><td>2160 (64.0%) </td><td><strong>4.82</strong> </td><td>449 (13.3%) </td><td>−0.31 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1887 (24.9%) </td><td>−0.51 </td><td>39 (0.5%) </td><td>−1.12 </td><td>4580 (60.4%) </td><td>1.95 </td><td>1076 (14.2%) </td><td>0.37 </td></tr><tr><td> Total </td><td>14 599 </td><td>3704 (25.4%) </td><td> </td><td>270 (1.8%) </td><td> </td><td>8615 (59.0%) </td><td> </td><td>2010 (13.8%) </td><td> </td></tr><tr><td colspan="10">Rice versus maize </td></tr><tr><td> Convergent </td><td>3641 </td><td>987 (27.1%) </td><td><strong>3.96</strong> </td><td>103 (2.8%) </td><td>0.32 </td><td>1880 (51.6%) </td><td>−<strong>4.92</strong> </td><td>671 (18.4%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>472 (14.0%) </td><td>−<strong>4.71</strong> </td><td>146 (4.3%) </td><td>1.20 </td><td>2137 (63.3%) </td><td><strong>5.75</strong> </td><td>621 (18.4%) </td><td>−0.33 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1678 (22.1%) </td><td>0.62 </td><td>92 (1.2%) </td><td>−0.95 </td><td>4347 (57.3%) </td><td>0.04 </td><td>1465 (19.3%) </td><td>0.41 </td></tr><tr><td> Total </td><td>14 599 </td><td>3137 (21.5%) </td><td> </td><td>341 (2.3%) </td><td> </td><td>8364 (57.3%) </td><td> </td><td>2757 (18.9%) </td><td> </td></tr><tr><td colspan="10">Rice versus <em>Brachypodium</em> </td></tr><tr><td> Convergent </td><td>3641 </td><td>1429 (39.2%) </td><td>1.82 </td><td>117 (3.2%) </td><td>0.50 </td><td>1569 (43.1%) </td><td>−2.25 </td><td>526 (14.4%) </td><td>−0.23 </td></tr><tr><td> Divergent </td><td>3376 </td><td>1128 (33.4%) </td><td>−<strong>2.48</strong> </td><td>154 (4.6%) </td><td>1.29 </td><td>1587 (47.0%) </td><td>0.88 </td><td>507 (15.0%) </td><td>0.14 </td></tr><tr><td> Tandem </td><td>7582 </td><td>2830 (37.3%) </td><td>0.47 </td><td>81 (1.1%) </td><td>−1.17 </td><td>3541 (46.7%) </td><td>0.96 </td><td>1130 (14.9%) </td><td>0.10 </td></tr><tr><td> Total </td><td>14 599 </td><td>5387 (36.9%) </td><td> </td><td>352 (2.4%) </td><td> </td><td>6697 (45.9%) </td><td> </td><td>2163 (14.8%) </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Test used is the binomial test (normal approximation) with cutoff of <em>P</em> &lt; 0.01 (<em>Z</em> &lt; 2.3267) shown in bold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356670" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ022TB1" class="table-wrap-title" data-id="DSQ022TB1"><div class="label -label">Table 1.</div><div class="caption"><p>RAP rice gene-pair conservation and rearrangement</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Pairs </th><th>Conserved </th><th><em>Z</em>-value </th><th>Inverted </th><th><em>Z</em>-value </th><th>Rearranged </th><th><em>Z</em>-value </th><th>Missing homologs </th><th><em>Z</em>-value </th></tr></thead><tbody><tr><td colspan="10">Rice versus sorghum </td></tr><tr><td> Convergent </td><td>3641 </td><td>1169 (32.1%) </td><td><strong>4.91</strong> </td><td>112 (3.1%) </td><td>0.78 </td><td>1875 (51.5%) </td><td>−<strong>6.50</strong> </td><td>485 (13.3%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>648 (19.2%) </td><td>−<strong>4.01</strong> </td><td>119 (3.5%) </td><td>1.02 </td><td>2160 (64.0%) </td><td><strong>4.82</strong> </td><td>449 (13.3%) </td><td>−0.31 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1887 (24.9%) </td><td>−0.51 </td><td>39 (0.5%) </td><td>−1.12 </td><td>4580 (60.4%) </td><td>1.95 </td><td>1076 (14.2%) </td><td>0.37 </td></tr><tr><td> Total </td><td>14 599 </td><td>3704 (25.4%) </td><td> </td><td>270 (1.8%) </td><td> </td><td>8615 (59.0%) </td><td> </td><td>2010 (13.8%) </td><td> </td></tr><tr><td colspan="10">Rice versus maize </td></tr><tr><td> Convergent </td><td>3641 </td><td>987 (27.1%) </td><td><strong>3.96</strong> </td><td>103 (2.8%) </td><td>0.32 </td><td>1880 (51.6%) </td><td>−<strong>4.92</strong> </td><td>671 (18.4%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>472 (14.0%) </td><td>−<strong>4.71</strong> </td><td>146 (4.3%) </td><td>1.20 </td><td>2137 (63.3%) </td><td><strong>5.75</strong> </td><td>621 (18.4%) </td><td>−0.33 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1678 (22.1%) </td><td>0.62 </td><td>92 (1.2%) </td><td>−0.95 </td><td>4347 (57.3%) </td><td>0.04 </td><td>1465 (19.3%) </td><td>0.41 </td></tr><tr><td> Total </td><td>14 599 </td><td>3137 (21.5%) </td><td> </td><td>341 (2.3%) </td><td> </td><td>8364 (57.3%) </td><td> </td><td>2757 (18.9%) </td><td> </td></tr><tr><td colspan="10">Rice versus <em>Brachypodium</em> </td></tr><tr><td> Convergent </td><td>3641 </td><td>1429 (39.2%) </td><td>1.82 </td><td>117 (3.2%) </td><td>0.50 </td><td>1569 (43.1%) </td><td>−2.25 </td><td>526 (14.4%) </td><td>−0.23 </td></tr><tr><td> Divergent </td><td>3376 </td><td>1128 (33.4%) </td><td>−<strong>2.48</strong> </td><td>154 (4.6%) </td><td>1.29 </td><td>1587 (47.0%) </td><td>0.88 </td><td>507 (15.0%) </td><td>0.14 </td></tr><tr><td> Tandem </td><td>7582 </td><td>2830 (37.3%) </td><td>0.47 </td><td>81 (1.1%) </td><td>−1.17 </td><td>3541 (46.7%) </td><td>0.96 </td><td>1130 (14.9%) </td><td>0.10 </td></tr><tr><td> Total </td><td>14 599 </td><td>5387 (36.9%) </td><td> </td><td>352 (2.4%) </td><td> </td><td>6697 (45.9%) </td><td> </td><td>2163 (14.8%) </td><td> </td></tr></tbody></table></div><div content-id="DSQ022TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Pairs </th><th>Conserved </th><th><em>Z</em>-value </th><th>Inverted </th><th><em>Z</em>-value </th><th>Rearranged </th><th><em>Z</em>-value </th><th>Missing homologs </th><th><em>Z</em>-value </th></tr></thead><tbody><tr><td colspan="10">Rice versus sorghum </td></tr><tr><td> Convergent </td><td>3641 </td><td>1169 (32.1%) </td><td><strong>4.91</strong> </td><td>112 (3.1%) </td><td>0.78 </td><td>1875 (51.5%) </td><td>−<strong>6.50</strong> </td><td>485 (13.3%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>648 (19.2%) </td><td>−<strong>4.01</strong> </td><td>119 (3.5%) </td><td>1.02 </td><td>2160 (64.0%) </td><td><strong>4.82</strong> </td><td>449 (13.3%) </td><td>−0.31 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1887 (24.9%) </td><td>−0.51 </td><td>39 (0.5%) </td><td>−1.12 </td><td>4580 (60.4%) </td><td>1.95 </td><td>1076 (14.2%) </td><td>0.37 </td></tr><tr><td> Total </td><td>14 599 </td><td>3704 (25.4%) </td><td> </td><td>270 (1.8%) </td><td> </td><td>8615 (59.0%) </td><td> </td><td>2010 (13.8%) </td><td> </td></tr><tr><td colspan="10">Rice versus maize </td></tr><tr><td> Convergent </td><td>3641 </td><td>987 (27.1%) </td><td><strong>3.96</strong> </td><td>103 (2.8%) </td><td>0.32 </td><td>1880 (51.6%) </td><td>−<strong>4.92</strong> </td><td>671 (18.4%) </td><td>−0.31 </td></tr><tr><td> Divergent </td><td>3376 </td><td>472 (14.0%) </td><td>−<strong>4.71</strong> </td><td>146 (4.3%) </td><td>1.20 </td><td>2137 (63.3%) </td><td><strong>5.75</strong> </td><td>621 (18.4%) </td><td>−0.33 </td></tr><tr><td> Tandem </td><td>7582 </td><td>1678 (22.1%) </td><td>0.62 </td><td>92 (1.2%) </td><td>−0.95 </td><td>4347 (57.3%) </td><td>0.04 </td><td>1465 (19.3%) </td><td>0.41 </td></tr><tr><td> Total </td><td>14 599 </td><td>3137 (21.5%) </td><td> </td><td>341 (2.3%) </td><td> </td><td>8364 (57.3%) </td><td> </td><td>2757 (18.9%) </td><td> </td></tr><tr><td colspan="10">Rice versus <em>Brachypodium</em> </td></tr><tr><td> Convergent </td><td>3641 </td><td>1429 (39.2%) </td><td>1.82 </td><td>117 (3.2%) </td><td>0.50 </td><td>1569 (43.1%) </td><td>−2.25 </td><td>526 (14.4%) </td><td>−0.23 </td></tr><tr><td> Divergent </td><td>3376 </td><td>1128 (33.4%) </td><td>−<strong>2.48</strong> </td><td>154 (4.6%) </td><td>1.29 </td><td>1587 (47.0%) </td><td>0.88 </td><td>507 (15.0%) </td><td>0.14 </td></tr><tr><td> Tandem </td><td>7582 </td><td>2830 (37.3%) </td><td>0.47 </td><td>81 (1.1%) </td><td>−1.17 </td><td>3541 (46.7%) </td><td>0.96 </td><td>1130 (14.9%) </td><td>0.10 </td></tr><tr><td> Total </td><td>14 599 </td><td>5387 (36.9%) </td><td> </td><td>352 (2.4%) </td><td> </td><td>6697 (45.9%) </td><td> </td><td>2163 (14.8%) </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Test used is the binomial test (normal approximation) with cutoff of <em>P</em> &lt; 0.01 (<em>Z</em> &lt; 2.3267) shown in bold.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356670" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7356671 id="7356671" class="section-title" data-legacy-id=s3b>3.2. Effects of correlated expression on rearrangements</h3>
<p>The Pearson correlation coefficient of all rice gene pairs was calculated using microarray and MPSS expression data. Those pairs with coefficients of 0.5 or greater were considered significantly correlated as described earlier.<sup><a reveal-id="DSQ022C1" data-open="DSQ022C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The full set of rice gene pairs (MSU data set) was divided into correlated and uncorrelated sets, and difference in the frequencies of each type of rearrangement within these sets was tested for significance using the binomial test. The purpose of this test was to determine whether gene pairs with correlated expression levels were subject to any of the various types of rearrangements at a significantly different rate than uncorrelated pairs.</p><p>Correlated divergent and tandem gene pairs were more common among conserved pairs in all three species (Table <a reveal-id="DSQ022TB2" data-open="DSQ022TB2" class="link link-reveal link-table xref-fig">2</a>), although the increase in conservation was statistically significant only for tandem pairs in sorghum and for divergent and tandem pairs in <em>Brachypodium</em>. The difference in the conservation rates between correlated and uncorrelated pairs was highest for tandem pairs, followed closely by divergent pairs, whereas the effect of correlation on convergent pairs was considerably weaker. Conservation of coexpressed tandem gene pairs could be due to tandemly duplicated genes which are likely to have similar expression patterns. Conservation of coexpressed divergent gene pairs could be due to bidirectional promoters regulating these gene pairs.<sup><a reveal-id="DSQ022C19" data-open="DSQ022C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                        <a id="7356674" scrollto-destination="7356674"></a>
<div content-id="DSQ022TB2" class="table-modal"><div class="table-wrap"><div id="DSQ022TB2" class="table-wrap-title" data-id="DSQ022TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Conservation and rearrangement of correlated rice gene pairs with <em>r</em> &gt; 0.5</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Total </th><th colspan="3">Conserved<hr /> </th><th colspan="3">Inverted<hr /> </th><th colspan="3">Rearranged<hr /> </th><th colspan="3">Missing homologs<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th>Number </th><th>Percentage </th><th><em>Z</em>-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th></tr></thead><tbody><tr><td colspan="15">Rice versus sorghum </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>153 </td><td>46.5 </td><td rowspan="2">1.61 </td><td>7 </td><td>2.1 </td><td rowspan="2">1.91 </td><td>135 </td><td>41.0 </td><td rowspan="2">−0.42 </td><td>34 </td><td>10.3 </td><td rowspan="2">−<strong>2.21</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1883 </td><td>42.1 </td><td>47 </td><td>1.1 </td><td>1886 </td><td>42.2 </td><td>655 </td><td>14.6 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>97 </td><td>32.8 </td><td rowspan="2">1.28 </td><td>4 </td><td>1.4 </td><td rowspan="2">0.80 </td><td>153 </td><td>51.7 </td><td rowspan="2">−0.53 </td><td>42 </td><td>14.2 </td><td rowspan="2">−1.07 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1003 </td><td>29.4 </td><td>31 </td><td>0.9 </td><td>1818 </td><td>53.2 </td><td>563 </td><td>16.5 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>276 </td><td>42.4 </td><td rowspan="2"><strong>2.98</strong> </td><td>4 </td><td>0.6 </td><td rowspan="2">0.23 </td><td>263 </td><td>40.4 </td><td rowspan="2">−<strong>3.21</strong> </td><td>108 </td><td>16.6 </td><td rowspan="2">0.40 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3227 </td><td>36.8 </td><td>48 </td><td>0.5 </td><td>4096 </td><td>46.7 </td><td>1406 </td><td>16.0 </td></tr><tr><td colspan="15">Rice versus maize </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>101 </td><td>30.7 </td><td rowspan="2">0.94 </td><td>5 </td><td>1.5 </td><td rowspan="2">0.56 </td><td>171 </td><td>52.0 </td><td rowspan="2">0.92 </td><td>52 </td><td>15.8 </td><td rowspan="2">−<strong>2.32</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1268 </td><td>28.4 </td><td>53 </td><td>1.2 </td><td>2210 </td><td>49.4 </td><td>940 </td><td>21.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>42 </td><td>14.2 </td><td rowspan="2">1.34 </td><td>4 </td><td>1.4 </td><td rowspan="2">−0.85 </td><td>181 </td><td>61.1 </td><td rowspan="2">−0.85 </td><td>69 </td><td>23.3 </td><td rowspan="2">0.23 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>399 </td><td>11.7 </td><td>70 </td><td>2.0 </td><td>2169 </td><td>63.5 </td><td>777 </td><td>22.8 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>166 </td><td>25.5 </td><td rowspan="2">1.90 </td><td>6 </td><td>0.9 </td><td rowspan="2">−0.52 </td><td>316 </td><td>48.5 </td><td rowspan="2">−<strong>2.22</strong> </td><td>163 </td><td>25.0 </td><td rowspan="2">0.87 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>1966 </td><td>22.4 </td><td>100 </td><td>1.1 </td><td>4641 </td><td>52.9 </td><td>2070 </td><td>23.6 </td></tr><tr><td colspan="15">Rice versus Brachypodium </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>141 </td><td>42.9 </td><td rowspan="2">0.35 </td><td>4 </td><td>1.2 </td><td rowspan="2">−0.48 </td><td>124 </td><td>37.7 </td><td rowspan="2">0.06 </td><td>60 </td><td>18.2 </td><td rowspan="2">−0.37 </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1873 </td><td>41.9 </td><td>69 </td><td>1.5 </td><td>1678 </td><td>37.5 </td><td>851 </td><td>19.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>112 </td><td>37.8 </td><td rowspan="2"><strong>1.98</strong> </td><td>5 </td><td>1.7 </td><td rowspan="2">0.19 </td><td>118 </td><td>39.9 </td><td rowspan="2">−1.41 </td><td>61 </td><td>20.6 </td><td rowspan="2">−0.61 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1108 </td><td>32.4 </td><td>53 </td><td>1.6 </td><td>1500 </td><td>43.9 </td><td>754 </td><td>22.1 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>266 </td><td>40.9 </td><td rowspan="2"><strong>2.23</strong> </td><td>8 </td><td>1.2 </td><td rowspan="2">0.45 </td><td>247 </td><td>37.9 </td><td rowspan="2">−<strong>2.28</strong> </td><td>130 </td><td>20.0 </td><td rowspan="2">0.01 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3217 </td><td>36.7 </td><td>92 </td><td>1.0 </td><td>3717 </td><td>42.3 </td><td>1751 </td><td>19.9 </td></tr></tbody></table></div><div content-id="DSQ022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Total </th><th colspan="3">Conserved<hr /> </th><th colspan="3">Inverted<hr /> </th><th colspan="3">Rearranged<hr /> </th><th colspan="3">Missing homologs<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th>Number </th><th>Percentage </th><th><em>Z</em>-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th></tr></thead><tbody><tr><td colspan="15">Rice versus sorghum </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>153 </td><td>46.5 </td><td rowspan="2">1.61 </td><td>7 </td><td>2.1 </td><td rowspan="2">1.91 </td><td>135 </td><td>41.0 </td><td rowspan="2">−0.42 </td><td>34 </td><td>10.3 </td><td rowspan="2">−<strong>2.21</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1883 </td><td>42.1 </td><td>47 </td><td>1.1 </td><td>1886 </td><td>42.2 </td><td>655 </td><td>14.6 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>97 </td><td>32.8 </td><td rowspan="2">1.28 </td><td>4 </td><td>1.4 </td><td rowspan="2">0.80 </td><td>153 </td><td>51.7 </td><td rowspan="2">−0.53 </td><td>42 </td><td>14.2 </td><td rowspan="2">−1.07 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1003 </td><td>29.4 </td><td>31 </td><td>0.9 </td><td>1818 </td><td>53.2 </td><td>563 </td><td>16.5 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>276 </td><td>42.4 </td><td rowspan="2"><strong>2.98</strong> </td><td>4 </td><td>0.6 </td><td rowspan="2">0.23 </td><td>263 </td><td>40.4 </td><td rowspan="2">−<strong>3.21</strong> </td><td>108 </td><td>16.6 </td><td rowspan="2">0.40 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3227 </td><td>36.8 </td><td>48 </td><td>0.5 </td><td>4096 </td><td>46.7 </td><td>1406 </td><td>16.0 </td></tr><tr><td colspan="15">Rice versus maize </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>101 </td><td>30.7 </td><td rowspan="2">0.94 </td><td>5 </td><td>1.5 </td><td rowspan="2">0.56 </td><td>171 </td><td>52.0 </td><td rowspan="2">0.92 </td><td>52 </td><td>15.8 </td><td rowspan="2">−<strong>2.32</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1268 </td><td>28.4 </td><td>53 </td><td>1.2 </td><td>2210 </td><td>49.4 </td><td>940 </td><td>21.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>42 </td><td>14.2 </td><td rowspan="2">1.34 </td><td>4 </td><td>1.4 </td><td rowspan="2">−0.85 </td><td>181 </td><td>61.1 </td><td rowspan="2">−0.85 </td><td>69 </td><td>23.3 </td><td rowspan="2">0.23 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>399 </td><td>11.7 </td><td>70 </td><td>2.0 </td><td>2169 </td><td>63.5 </td><td>777 </td><td>22.8 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>166 </td><td>25.5 </td><td rowspan="2">1.90 </td><td>6 </td><td>0.9 </td><td rowspan="2">−0.52 </td><td>316 </td><td>48.5 </td><td rowspan="2">−<strong>2.22</strong> </td><td>163 </td><td>25.0 </td><td rowspan="2">0.87 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>1966 </td><td>22.4 </td><td>100 </td><td>1.1 </td><td>4641 </td><td>52.9 </td><td>2070 </td><td>23.6 </td></tr><tr><td colspan="15">Rice versus Brachypodium </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>141 </td><td>42.9 </td><td rowspan="2">0.35 </td><td>4 </td><td>1.2 </td><td rowspan="2">−0.48 </td><td>124 </td><td>37.7 </td><td rowspan="2">0.06 </td><td>60 </td><td>18.2 </td><td rowspan="2">−0.37 </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1873 </td><td>41.9 </td><td>69 </td><td>1.5 </td><td>1678 </td><td>37.5 </td><td>851 </td><td>19.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>112 </td><td>37.8 </td><td rowspan="2"><strong>1.98</strong> </td><td>5 </td><td>1.7 </td><td rowspan="2">0.19 </td><td>118 </td><td>39.9 </td><td rowspan="2">−1.41 </td><td>61 </td><td>20.6 </td><td rowspan="2">−0.61 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1108 </td><td>32.4 </td><td>53 </td><td>1.6 </td><td>1500 </td><td>43.9 </td><td>754 </td><td>22.1 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>266 </td><td>40.9 </td><td rowspan="2"><strong>2.23</strong> </td><td>8 </td><td>1.2 </td><td rowspan="2">0.45 </td><td>247 </td><td>37.9 </td><td rowspan="2">−<strong>2.28</strong> </td><td>130 </td><td>20.0 </td><td rowspan="2">0.01 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3217 </td><td>36.7 </td><td>92 </td><td>1.0 </td><td>3717 </td><td>42.3 </td><td>1751 </td><td>19.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Values in the ‘<em>Z</em>' columns are test statistics of the binomial test. Values in bold denote significant differences (<em>P</em> &lt; 0.05) in the frequency of strongly correlated pairs in each category compared with the frequency of uncorrelated pairs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356674" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ022TB2" class="table-wrap-title" data-id="DSQ022TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Conservation and rearrangement of correlated rice gene pairs with <em>r</em> &gt; 0.5</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Total </th><th colspan="3">Conserved<hr /> </th><th colspan="3">Inverted<hr /> </th><th colspan="3">Rearranged<hr /> </th><th colspan="3">Missing homologs<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th>Number </th><th>Percentage </th><th><em>Z</em>-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th></tr></thead><tbody><tr><td colspan="15">Rice versus sorghum </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>153 </td><td>46.5 </td><td rowspan="2">1.61 </td><td>7 </td><td>2.1 </td><td rowspan="2">1.91 </td><td>135 </td><td>41.0 </td><td rowspan="2">−0.42 </td><td>34 </td><td>10.3 </td><td rowspan="2">−<strong>2.21</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1883 </td><td>42.1 </td><td>47 </td><td>1.1 </td><td>1886 </td><td>42.2 </td><td>655 </td><td>14.6 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>97 </td><td>32.8 </td><td rowspan="2">1.28 </td><td>4 </td><td>1.4 </td><td rowspan="2">0.80 </td><td>153 </td><td>51.7 </td><td rowspan="2">−0.53 </td><td>42 </td><td>14.2 </td><td rowspan="2">−1.07 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1003 </td><td>29.4 </td><td>31 </td><td>0.9 </td><td>1818 </td><td>53.2 </td><td>563 </td><td>16.5 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>276 </td><td>42.4 </td><td rowspan="2"><strong>2.98</strong> </td><td>4 </td><td>0.6 </td><td rowspan="2">0.23 </td><td>263 </td><td>40.4 </td><td rowspan="2">−<strong>3.21</strong> </td><td>108 </td><td>16.6 </td><td rowspan="2">0.40 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3227 </td><td>36.8 </td><td>48 </td><td>0.5 </td><td>4096 </td><td>46.7 </td><td>1406 </td><td>16.0 </td></tr><tr><td colspan="15">Rice versus maize </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>101 </td><td>30.7 </td><td rowspan="2">0.94 </td><td>5 </td><td>1.5 </td><td rowspan="2">0.56 </td><td>171 </td><td>52.0 </td><td rowspan="2">0.92 </td><td>52 </td><td>15.8 </td><td rowspan="2">−<strong>2.32</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1268 </td><td>28.4 </td><td>53 </td><td>1.2 </td><td>2210 </td><td>49.4 </td><td>940 </td><td>21.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>42 </td><td>14.2 </td><td rowspan="2">1.34 </td><td>4 </td><td>1.4 </td><td rowspan="2">−0.85 </td><td>181 </td><td>61.1 </td><td rowspan="2">−0.85 </td><td>69 </td><td>23.3 </td><td rowspan="2">0.23 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>399 </td><td>11.7 </td><td>70 </td><td>2.0 </td><td>2169 </td><td>63.5 </td><td>777 </td><td>22.8 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>166 </td><td>25.5 </td><td rowspan="2">1.90 </td><td>6 </td><td>0.9 </td><td rowspan="2">−0.52 </td><td>316 </td><td>48.5 </td><td rowspan="2">−<strong>2.22</strong> </td><td>163 </td><td>25.0 </td><td rowspan="2">0.87 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>1966 </td><td>22.4 </td><td>100 </td><td>1.1 </td><td>4641 </td><td>52.9 </td><td>2070 </td><td>23.6 </td></tr><tr><td colspan="15">Rice versus Brachypodium </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>141 </td><td>42.9 </td><td rowspan="2">0.35 </td><td>4 </td><td>1.2 </td><td rowspan="2">−0.48 </td><td>124 </td><td>37.7 </td><td rowspan="2">0.06 </td><td>60 </td><td>18.2 </td><td rowspan="2">−0.37 </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1873 </td><td>41.9 </td><td>69 </td><td>1.5 </td><td>1678 </td><td>37.5 </td><td>851 </td><td>19.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>112 </td><td>37.8 </td><td rowspan="2"><strong>1.98</strong> </td><td>5 </td><td>1.7 </td><td rowspan="2">0.19 </td><td>118 </td><td>39.9 </td><td rowspan="2">−1.41 </td><td>61 </td><td>20.6 </td><td rowspan="2">−0.61 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1108 </td><td>32.4 </td><td>53 </td><td>1.6 </td><td>1500 </td><td>43.9 </td><td>754 </td><td>22.1 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>266 </td><td>40.9 </td><td rowspan="2"><strong>2.23</strong> </td><td>8 </td><td>1.2 </td><td rowspan="2">0.45 </td><td>247 </td><td>37.9 </td><td rowspan="2">−<strong>2.28</strong> </td><td>130 </td><td>20.0 </td><td rowspan="2">0.01 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3217 </td><td>36.7 </td><td>92 </td><td>1.0 </td><td>3717 </td><td>42.3 </td><td>1751 </td><td>19.9 </td></tr></tbody></table></div><div content-id="DSQ022TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Total </th><th colspan="3">Conserved<hr /> </th><th colspan="3">Inverted<hr /> </th><th colspan="3">Rearranged<hr /> </th><th colspan="3">Missing homologs<hr /> </th></tr><tr><th> </th><th> </th><th> </th><th>Number </th><th>Percentage </th><th><em>Z</em>-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th><th>Number </th><th>Percentage </th><th>Z-value </th></tr></thead><tbody><tr><td colspan="15">Rice versus sorghum </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>153 </td><td>46.5 </td><td rowspan="2">1.61 </td><td>7 </td><td>2.1 </td><td rowspan="2">1.91 </td><td>135 </td><td>41.0 </td><td rowspan="2">−0.42 </td><td>34 </td><td>10.3 </td><td rowspan="2">−<strong>2.21</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1883 </td><td>42.1 </td><td>47 </td><td>1.1 </td><td>1886 </td><td>42.2 </td><td>655 </td><td>14.6 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>97 </td><td>32.8 </td><td rowspan="2">1.28 </td><td>4 </td><td>1.4 </td><td rowspan="2">0.80 </td><td>153 </td><td>51.7 </td><td rowspan="2">−0.53 </td><td>42 </td><td>14.2 </td><td rowspan="2">−1.07 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1003 </td><td>29.4 </td><td>31 </td><td>0.9 </td><td>1818 </td><td>53.2 </td><td>563 </td><td>16.5 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>276 </td><td>42.4 </td><td rowspan="2"><strong>2.98</strong> </td><td>4 </td><td>0.6 </td><td rowspan="2">0.23 </td><td>263 </td><td>40.4 </td><td rowspan="2">−<strong>3.21</strong> </td><td>108 </td><td>16.6 </td><td rowspan="2">0.40 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3227 </td><td>36.8 </td><td>48 </td><td>0.5 </td><td>4096 </td><td>46.7 </td><td>1406 </td><td>16.0 </td></tr><tr><td colspan="15">Rice versus maize </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>101 </td><td>30.7 </td><td rowspan="2">0.94 </td><td>5 </td><td>1.5 </td><td rowspan="2">0.56 </td><td>171 </td><td>52.0 </td><td rowspan="2">0.92 </td><td>52 </td><td>15.8 </td><td rowspan="2">−<strong>2.32</strong> </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1268 </td><td>28.4 </td><td>53 </td><td>1.2 </td><td>2210 </td><td>49.4 </td><td>940 </td><td>21.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>42 </td><td>14.2 </td><td rowspan="2">1.34 </td><td>4 </td><td>1.4 </td><td rowspan="2">−0.85 </td><td>181 </td><td>61.1 </td><td rowspan="2">−0.85 </td><td>69 </td><td>23.3 </td><td rowspan="2">0.23 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>399 </td><td>11.7 </td><td>70 </td><td>2.0 </td><td>2169 </td><td>63.5 </td><td>777 </td><td>22.8 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>166 </td><td>25.5 </td><td rowspan="2">1.90 </td><td>6 </td><td>0.9 </td><td rowspan="2">−0.52 </td><td>316 </td><td>48.5 </td><td rowspan="2">−<strong>2.22</strong> </td><td>163 </td><td>25.0 </td><td rowspan="2">0.87 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>1966 </td><td>22.4 </td><td>100 </td><td>1.1 </td><td>4641 </td><td>52.9 </td><td>2070 </td><td>23.6 </td></tr><tr><td colspan="15">Rice versus Brachypodium </td></tr><tr><td rowspan="2"> Convergent </td><td>Correlated </td><td>329 </td><td>141 </td><td>42.9 </td><td rowspan="2">0.35 </td><td>4 </td><td>1.2 </td><td rowspan="2">−0.48 </td><td>124 </td><td>37.7 </td><td rowspan="2">0.06 </td><td>60 </td><td>18.2 </td><td rowspan="2">−0.37 </td></tr><tr><td>Uncorrelated </td><td>4471 </td><td>1873 </td><td>41.9 </td><td>69 </td><td>1.5 </td><td>1678 </td><td>37.5 </td><td>851 </td><td>19.0 </td></tr><tr><td rowspan="2"> Divergent </td><td>Correlated </td><td>296 </td><td>112 </td><td>37.8 </td><td rowspan="2"><strong>1.98</strong> </td><td>5 </td><td>1.7 </td><td rowspan="2">0.19 </td><td>118 </td><td>39.9 </td><td rowspan="2">−1.41 </td><td>61 </td><td>20.6 </td><td rowspan="2">−0.61 </td></tr><tr><td>Uncorrelated </td><td>3415 </td><td>1108 </td><td>32.4 </td><td>53 </td><td>1.6 </td><td>1500 </td><td>43.9 </td><td>754 </td><td>22.1 </td></tr><tr><td rowspan="2"> Tandem </td><td>Correlated </td><td>651 </td><td>266 </td><td>40.9 </td><td rowspan="2"><strong>2.23</strong> </td><td>8 </td><td>1.2 </td><td rowspan="2">0.45 </td><td>247 </td><td>37.9 </td><td rowspan="2">−<strong>2.28</strong> </td><td>130 </td><td>20.0 </td><td rowspan="2">0.01 </td></tr><tr><td>Uncorrelated </td><td>8777 </td><td>3217 </td><td>36.7 </td><td>92 </td><td>1.0 </td><td>3717 </td><td>42.3 </td><td>1751 </td><td>19.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Values in the ‘<em>Z</em>' columns are test statistics of the binomial test. Values in bold denote significant differences (<em>P</em> &lt; 0.05) in the frequency of strongly correlated pairs in each category compared with the frequency of uncorrelated pairs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356674" target="_blank">View Large</a></div></div><p>The small sample size used in the examination of the effect of correlated expression on gene-pair inversion reduced the effectiveness of the binomial test. In contrast, correlated expression in rice gene pairs appears to select against the disruption of a gene pair's physical arrangement in the other three grass genomes, with tandem pairs being significantly underrepresented in all three comparison species among pairs whose homologs are physically distant (Table <a reveal-id="DSQ022TB2" data-open="DSQ022TB2" class="link link-reveal link-table xref-fig">2</a>). Divergent pairs showing correlated expression are also underrepresented in all three species, although the difference is not statistically significant. Correlated convergent pairs showed no significant difference compared with non-correlated pairs.</p><p>Rice-correlated convergent pairs were strongly underrepresented among pairs lacking homologs in sorghum and maize.</p><p>A second analysis was performed, this time using a statistically significant (see ‘Materials and methods' section for details) cut-off value for ‘strong' correlation of <em>R</em> &gt; 0.594 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The greatest difference between this analysis and the first one using a cut-off of <em>R</em> &gt; 0.5 is the number of strongly correlated pairs. The reduced sample size substantially affects the ability of the binomial test in determining significant variation between correlated and non-correlated pairs. As a result, statistical significance was observed only in convergent gene pairs missing homologs in maize, which were significantly underrepresented among strongly correlated pairs. Otherwise, most trends noted in the first analysis were also observed in the second analysis, with modest increases in conservation and inversion frequency and decreases in rearrangement frequency. The fractions of pairs missing homologs, either unchanged or reduced with the <em>R</em> &gt; 0.5 definition of strong correlation, were more often increased with the alternative definition. These differences most likely result from the smaller sample size and its effect on the statistical test rather than any real biological differences between the set of gene pairs with <em>R</em> &gt; 0.5 and those with <em>R</em> &gt; 0.594.</p><p>Rice gene pairs displaying strongly correlated expression levels were more likely to be conserved in sorghum, maize, and <em>Brachypodium</em>. These results lend further support to the hypothesis that the strand-wise arrangement of pairs of adjacent genes may be essential to the regulatory schemes of some strongly correlated gene pairs, such that rearrangements disturbing the pair would be selected against. Correlated expression levels have also been found to increase the likelihood of conservation among fungi.<sup><a reveal-id="DSQ022C20" data-open="DSQ022C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The largest increases in the frequency of conservation as a result of correlated expression was observed among divergent and tandem pairs, a pattern that has been observed before in a comparison of human, mouse, and chicken gene clusters.<sup><a reveal-id="DSQ022C21" data-open="DSQ022C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSQ022C22" data-open="DSQ022C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Therefore, correlated expression levels increasing the likelihood of conservation appears to be a universal phenomenon in eukaryotes including plant genomes.</p>                            <h3 scrollto-destination=7356681 id="7356681" class="section-title" data-legacy-id=s3c>3.3. Evolutionary history of gene pairs</h3>
<p>The estimated evolutionary history of each rice or sorghum gene pair was arrived at by examining the status of each gene pair in its three comparison species. For this analysis, a pair could be in one of three states in each species: conserved, rearranged (physically distant homologs, any inversion, and insertions), or deleted (one or both homologs nonexistent). On the basis of the possible combinations of these states, 14 categories of evolutionary history were devised. The putative evolutionary tree for the four species consisted of two branches, one with rice and <em>Brachypodium</em>, the other with sorghum and maize. Similarities within branches, as well as differences between them, served as the basis for many of the 14 categories.</p><p>The first category consisted of those pairs whose exact arrangement was shared in all four species (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a>, four species, and Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>A). Results varied little between the rice- and sorghum-based analyses. Convergent pairs were the most common in this class, with over 18% of pairs falling into this category, followed by tandem pairs (∼10%) and divergent pairs (∼6%). </p>                        <a id="7356684" scrollto-destination="7356684"></a>
<div content-id="DSQ022TB3" class="table-modal"><div class="table-wrap"><div id="DSQ022TB3" class="table-wrap-title" data-id="DSQ022TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Evolutionary history of rice and sorghum gene pairs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Rice<hr /> </th><th colspan="3">Sorghum<hr /> </th></tr><tr><th> </th><th>Convergent </th><th>Divergent </th><th>Tandem </th><th>Convergent </th><th>Divergent </th><th>Tandem </th></tr></thead><tbody><tr><td>Pair with conserved orientation in four species </td><td>874 (18.2%) </td><td>207 (5.6%) </td><td>934 (9.9%) </td><td>921 (18.2%) </td><td>258 (5.3%) </td><td>1222 (10.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in three species </td></tr><tr><td> One rearrangement </td><td>878 (18.3%) </td><td>591 (15.9%) </td><td>1827 (19.4%) </td><td>932 (18.4%) </td><td>800 (16.3%) </td><td>2245 (18.9%) </td></tr><tr><td> One deletion </td><td>64 (1.3%) </td><td>33 (0.9%) </td><td>151 (1.6%) </td><td>56 (1.1%) </td><td>28 (0.6%) </td><td>158 (1.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in two species </td></tr><tr><td colspan="7"> Branch-specific pairs </td></tr><tr><td>  Others rearranged </td><td>396 (8.3%) </td><td>434 (11.7%) </td><td>945 (10.0%) </td><td>408 (8.1%) </td><td>215 (4.4%) </td><td>840 (7.1%) </td></tr><tr><td>  Others deleted </td><td>53 (1.1%) </td><td>30 (0.8%) </td><td>129 (1.4%) </td><td>20 (0.4%) </td><td>14 (0.3%) </td><td>101 (0.9%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>32 (0.7%) </td><td>25 (0.7%) </td><td>78 (0.8%) </td><td>45 (0.9%) </td><td>24 (0.5%) </td><td>110 (0.9%) </td></tr><tr><td colspan="7"> Cross-branch conserved pairs </td></tr><tr><td>  Others rearranged </td><td>340 (7.1%) </td><td>346 (9.3%) </td><td>974 (10.3%) </td><td>438 (8.7%) </td><td>718 (14.6%) </td><td>1481 (12.5%) </td></tr><tr><td>  Others deleted </td><td>24 (0.5%) </td><td>12 (0.3%) </td><td>71 (0.8%) </td><td>10 (0.2%) </td><td>4 (0.1%) </td><td>51 (0.4%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>68 (1.4%) </td><td>45 (1.2%) </td><td>163 (1.7%) </td><td>55 (1.1%) </td><td>57 (1.2%) </td><td>225 (1.9%) </td></tr><tr><td colspan="7">Pair with unique orientation in one species </td></tr><tr><td> Species-specific gene(s) </td><td>505 (10.5%) </td><td>475 (12.8%) </td><td>1130 (12.0%) </td><td>197 (3.9%) </td><td>218 (4.4%) </td><td>529 (4.5%) </td></tr><tr><td> Common genes, species-specific pair </td><td>1037 (21.6%) </td><td>1047 (28.2%) </td><td>2038 (21.6%) </td><td>1293 (25.6%) </td><td>1730 (35.2%) </td><td>3189 (26.9%) </td></tr><tr><td> Branch-specific genes, species-specific pair </td><td>73 (1.5%) </td><td>51 (1.4%) </td><td>170 (1.8%) </td><td>158 (3.1%) </td><td>219 (4.5%) </td><td>457 (3.9%) </td></tr><tr><td> Two rearrangements, one deletion, mixed </td><td>299 (6.2%) </td><td>278 (7.5%) </td><td>530 (5.6%) </td><td>395 (7.8%) </td><td>460 (9.4%) </td><td>913 (7.7%) </td></tr><tr><td> One rearrangement, two deletions, mixed </td><td>157 (3.3%) </td><td>137 (3.7%) </td><td>288 (3.1%) </td><td>131 (2.6%) </td><td>168 (3.4%) </td><td>326 (2.8%) </td></tr></tbody></table></div><div content-id="DSQ022TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Rice<hr /> </th><th colspan="3">Sorghum<hr /> </th></tr><tr><th> </th><th>Convergent </th><th>Divergent </th><th>Tandem </th><th>Convergent </th><th>Divergent </th><th>Tandem </th></tr></thead><tbody><tr><td>Pair with conserved orientation in four species </td><td>874 (18.2%) </td><td>207 (5.6%) </td><td>934 (9.9%) </td><td>921 (18.2%) </td><td>258 (5.3%) </td><td>1222 (10.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in three species </td></tr><tr><td> One rearrangement </td><td>878 (18.3%) </td><td>591 (15.9%) </td><td>1827 (19.4%) </td><td>932 (18.4%) </td><td>800 (16.3%) </td><td>2245 (18.9%) </td></tr><tr><td> One deletion </td><td>64 (1.3%) </td><td>33 (0.9%) </td><td>151 (1.6%) </td><td>56 (1.1%) </td><td>28 (0.6%) </td><td>158 (1.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in two species </td></tr><tr><td colspan="7"> Branch-specific pairs </td></tr><tr><td>  Others rearranged </td><td>396 (8.3%) </td><td>434 (11.7%) </td><td>945 (10.0%) </td><td>408 (8.1%) </td><td>215 (4.4%) </td><td>840 (7.1%) </td></tr><tr><td>  Others deleted </td><td>53 (1.1%) </td><td>30 (0.8%) </td><td>129 (1.4%) </td><td>20 (0.4%) </td><td>14 (0.3%) </td><td>101 (0.9%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>32 (0.7%) </td><td>25 (0.7%) </td><td>78 (0.8%) </td><td>45 (0.9%) </td><td>24 (0.5%) </td><td>110 (0.9%) </td></tr><tr><td colspan="7"> Cross-branch conserved pairs </td></tr><tr><td>  Others rearranged </td><td>340 (7.1%) </td><td>346 (9.3%) </td><td>974 (10.3%) </td><td>438 (8.7%) </td><td>718 (14.6%) </td><td>1481 (12.5%) </td></tr><tr><td>  Others deleted </td><td>24 (0.5%) </td><td>12 (0.3%) </td><td>71 (0.8%) </td><td>10 (0.2%) </td><td>4 (0.1%) </td><td>51 (0.4%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>68 (1.4%) </td><td>45 (1.2%) </td><td>163 (1.7%) </td><td>55 (1.1%) </td><td>57 (1.2%) </td><td>225 (1.9%) </td></tr><tr><td colspan="7">Pair with unique orientation in one species </td></tr><tr><td> Species-specific gene(s) </td><td>505 (10.5%) </td><td>475 (12.8%) </td><td>1130 (12.0%) </td><td>197 (3.9%) </td><td>218 (4.4%) </td><td>529 (4.5%) </td></tr><tr><td> Common genes, species-specific pair </td><td>1037 (21.6%) </td><td>1047 (28.2%) </td><td>2038 (21.6%) </td><td>1293 (25.6%) </td><td>1730 (35.2%) </td><td>3189 (26.9%) </td></tr><tr><td> Branch-specific genes, species-specific pair </td><td>73 (1.5%) </td><td>51 (1.4%) </td><td>170 (1.8%) </td><td>158 (3.1%) </td><td>219 (4.5%) </td><td>457 (3.9%) </td></tr><tr><td> Two rearrangements, one deletion, mixed </td><td>299 (6.2%) </td><td>278 (7.5%) </td><td>530 (5.6%) </td><td>395 (7.8%) </td><td>460 (9.4%) </td><td>913 (7.7%) </td></tr><tr><td> One rearrangement, two deletions, mixed </td><td>157 (3.3%) </td><td>137 (3.7%) </td><td>288 (3.1%) </td><td>131 (2.6%) </td><td>168 (3.4%) </td><td>326 (2.8%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356684" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ022TB3" class="table-wrap-title" data-id="DSQ022TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Evolutionary history of rice and sorghum gene pairs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Rice<hr /> </th><th colspan="3">Sorghum<hr /> </th></tr><tr><th> </th><th>Convergent </th><th>Divergent </th><th>Tandem </th><th>Convergent </th><th>Divergent </th><th>Tandem </th></tr></thead><tbody><tr><td>Pair with conserved orientation in four species </td><td>874 (18.2%) </td><td>207 (5.6%) </td><td>934 (9.9%) </td><td>921 (18.2%) </td><td>258 (5.3%) </td><td>1222 (10.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in three species </td></tr><tr><td> One rearrangement </td><td>878 (18.3%) </td><td>591 (15.9%) </td><td>1827 (19.4%) </td><td>932 (18.4%) </td><td>800 (16.3%) </td><td>2245 (18.9%) </td></tr><tr><td> One deletion </td><td>64 (1.3%) </td><td>33 (0.9%) </td><td>151 (1.6%) </td><td>56 (1.1%) </td><td>28 (0.6%) </td><td>158 (1.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in two species </td></tr><tr><td colspan="7"> Branch-specific pairs </td></tr><tr><td>  Others rearranged </td><td>396 (8.3%) </td><td>434 (11.7%) </td><td>945 (10.0%) </td><td>408 (8.1%) </td><td>215 (4.4%) </td><td>840 (7.1%) </td></tr><tr><td>  Others deleted </td><td>53 (1.1%) </td><td>30 (0.8%) </td><td>129 (1.4%) </td><td>20 (0.4%) </td><td>14 (0.3%) </td><td>101 (0.9%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>32 (0.7%) </td><td>25 (0.7%) </td><td>78 (0.8%) </td><td>45 (0.9%) </td><td>24 (0.5%) </td><td>110 (0.9%) </td></tr><tr><td colspan="7"> Cross-branch conserved pairs </td></tr><tr><td>  Others rearranged </td><td>340 (7.1%) </td><td>346 (9.3%) </td><td>974 (10.3%) </td><td>438 (8.7%) </td><td>718 (14.6%) </td><td>1481 (12.5%) </td></tr><tr><td>  Others deleted </td><td>24 (0.5%) </td><td>12 (0.3%) </td><td>71 (0.8%) </td><td>10 (0.2%) </td><td>4 (0.1%) </td><td>51 (0.4%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>68 (1.4%) </td><td>45 (1.2%) </td><td>163 (1.7%) </td><td>55 (1.1%) </td><td>57 (1.2%) </td><td>225 (1.9%) </td></tr><tr><td colspan="7">Pair with unique orientation in one species </td></tr><tr><td> Species-specific gene(s) </td><td>505 (10.5%) </td><td>475 (12.8%) </td><td>1130 (12.0%) </td><td>197 (3.9%) </td><td>218 (4.4%) </td><td>529 (4.5%) </td></tr><tr><td> Common genes, species-specific pair </td><td>1037 (21.6%) </td><td>1047 (28.2%) </td><td>2038 (21.6%) </td><td>1293 (25.6%) </td><td>1730 (35.2%) </td><td>3189 (26.9%) </td></tr><tr><td> Branch-specific genes, species-specific pair </td><td>73 (1.5%) </td><td>51 (1.4%) </td><td>170 (1.8%) </td><td>158 (3.1%) </td><td>219 (4.5%) </td><td>457 (3.9%) </td></tr><tr><td> Two rearrangements, one deletion, mixed </td><td>299 (6.2%) </td><td>278 (7.5%) </td><td>530 (5.6%) </td><td>395 (7.8%) </td><td>460 (9.4%) </td><td>913 (7.7%) </td></tr><tr><td> One rearrangement, two deletions, mixed </td><td>157 (3.3%) </td><td>137 (3.7%) </td><td>288 (3.1%) </td><td>131 (2.6%) </td><td>168 (3.4%) </td><td>326 (2.8%) </td></tr></tbody></table></div><div content-id="DSQ022TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Rice<hr /> </th><th colspan="3">Sorghum<hr /> </th></tr><tr><th> </th><th>Convergent </th><th>Divergent </th><th>Tandem </th><th>Convergent </th><th>Divergent </th><th>Tandem </th></tr></thead><tbody><tr><td>Pair with conserved orientation in four species </td><td>874 (18.2%) </td><td>207 (5.6%) </td><td>934 (9.9%) </td><td>921 (18.2%) </td><td>258 (5.3%) </td><td>1222 (10.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in three species </td></tr><tr><td> One rearrangement </td><td>878 (18.3%) </td><td>591 (15.9%) </td><td>1827 (19.4%) </td><td>932 (18.4%) </td><td>800 (16.3%) </td><td>2245 (18.9%) </td></tr><tr><td> One deletion </td><td>64 (1.3%) </td><td>33 (0.9%) </td><td>151 (1.6%) </td><td>56 (1.1%) </td><td>28 (0.6%) </td><td>158 (1.3%) </td></tr><tr><td colspan="7">Pair with conserved orientation in two species </td></tr><tr><td colspan="7"> Branch-specific pairs </td></tr><tr><td>  Others rearranged </td><td>396 (8.3%) </td><td>434 (11.7%) </td><td>945 (10.0%) </td><td>408 (8.1%) </td><td>215 (4.4%) </td><td>840 (7.1%) </td></tr><tr><td>  Others deleted </td><td>53 (1.1%) </td><td>30 (0.8%) </td><td>129 (1.4%) </td><td>20 (0.4%) </td><td>14 (0.3%) </td><td>101 (0.9%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>32 (0.7%) </td><td>25 (0.7%) </td><td>78 (0.8%) </td><td>45 (0.9%) </td><td>24 (0.5%) </td><td>110 (0.9%) </td></tr><tr><td colspan="7"> Cross-branch conserved pairs </td></tr><tr><td>  Others rearranged </td><td>340 (7.1%) </td><td>346 (9.3%) </td><td>974 (10.3%) </td><td>438 (8.7%) </td><td>718 (14.6%) </td><td>1481 (12.5%) </td></tr><tr><td>  Others deleted </td><td>24 (0.5%) </td><td>12 (0.3%) </td><td>71 (0.8%) </td><td>10 (0.2%) </td><td>4 (0.1%) </td><td>51 (0.4%) </td></tr><tr><td>  One deletion, one rearrangement </td><td>68 (1.4%) </td><td>45 (1.2%) </td><td>163 (1.7%) </td><td>55 (1.1%) </td><td>57 (1.2%) </td><td>225 (1.9%) </td></tr><tr><td colspan="7">Pair with unique orientation in one species </td></tr><tr><td> Species-specific gene(s) </td><td>505 (10.5%) </td><td>475 (12.8%) </td><td>1130 (12.0%) </td><td>197 (3.9%) </td><td>218 (4.4%) </td><td>529 (4.5%) </td></tr><tr><td> Common genes, species-specific pair </td><td>1037 (21.6%) </td><td>1047 (28.2%) </td><td>2038 (21.6%) </td><td>1293 (25.6%) </td><td>1730 (35.2%) </td><td>3189 (26.9%) </td></tr><tr><td> Branch-specific genes, species-specific pair </td><td>73 (1.5%) </td><td>51 (1.4%) </td><td>170 (1.8%) </td><td>158 (3.1%) </td><td>219 (4.5%) </td><td>457 (3.9%) </td></tr><tr><td> Two rearrangements, one deletion, mixed </td><td>299 (6.2%) </td><td>278 (7.5%) </td><td>530 (5.6%) </td><td>395 (7.8%) </td><td>460 (9.4%) </td><td>913 (7.7%) </td></tr><tr><td> One rearrangement, two deletions, mixed </td><td>157 (3.3%) </td><td>137 (3.7%) </td><td>288 (3.1%) </td><td>131 (2.6%) </td><td>168 (3.4%) </td><td>326 (2.8%) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7356684" target="_blank">View Large</a></div></div>                        <a id="7356685" scrollto-destination="7356685"></a>
<div id="DSQ022F3" data-id="DSQ022F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02203.jpeg?Expires=2147483647&Signature=A72yvzbNpR3CAYpM9YvQ11ulBq7to5gHfd32tp0rRU2L6gejdNcJPdivnY9szwBXuV4~IPFi9sUqo~JVvlfr9X1mBVZ~igBQuKrMj~ahRiZ0BSx1V4qZ9s7NTzQDrAKCqKqocjZ0Ihy5-gEdGcq-NiJunjwV6wB45~nCeyzn4bqfEC-1b0iV3Km48LXP7c9qFwFpW~~rz6EhbFmu43AZglVefY0t4rQUbXQXfk1FgvgWeB246JRzkgqBDVpHFQWFrz2S9NjlwtsENwhbCLPFiNC8SEh~dS9VXgTzIc4Cbs3YGOUnm82yz4VvwHwVy644nrYganA8w4twk7tAK5nBpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Categories of gene-pair evolution. Each image is a representative of the many specific scenarios that may be found in that category. The bottom branch of each tree represents the species in which the pair was first identified (i.e. either rice or sorghum), and the two genes in question are shown in a divergent pair in these examples. Rearrangements are represented by the inversion of one gene, inversion of both genes, insertions within the pair, or translocation to other regions or chromosomes. Likewise, deletions may involve one gene, as shown, or both genes in the pair. In some of the scenarios where the pair is conserved in two species (D, E, G, and H), the rearranged or deleted states are just as likely to be the ancestral state as the divergent pair shown. In scenario L, it is also possible that both genes existed in the common ancestor and a deletion took place in the top branch rather than new gene(s) being created." path-from-xml="dsq02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356685/dsq02203.jpeg" path-from-xml="dsq02203.jpeg" target="_blank">View large</a><a section="7356685" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02203.jpeg?Expires=2147483647&Signature=PqhvM1SCoPP4~ILwP7b4ZoVa0TM0-rFKHlhWUlFXMhQJyLL4x8OZ6~lnsHkItgpKzthJIsLTl7xclBid3Myl1Kkp~CFJaAovXukoaM8Ppp0CzUCDXe8RtKX3HyyTIDQeRvXaC9-83wTD-Dy8NgqLEYj85XNWI25VqUsukUput7EN4kvPyvqqEXTzasW1vmQ9HhtB7O6I3g8PtdkaX6hTwl4cI5tJC0hMwAY071FHEFwC8g2x-UDxx85RvT-Ll0-O3Jy3J56kP~ReHfuhzBqFSK2FZRjb9gp2zEQGBF3VSKoZaP2cPlZmKgYiBwxhlsIPH159xKEU1XvYFX22OWnWpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356685&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Categories of gene-pair evolution. Each image is a representative of the many specific scenarios that may be found in that category. The bottom branch of each tree represents the species in which the pair was first identified (i.e. either rice or sorghum), and the two genes in question are shown in a divergent pair in these examples. Rearrangements are represented by the inversion of one gene, inversion of both genes, insertions within the pair, or translocation to other regions or chromosomes. Likewise, deletions may involve one gene, as shown, or both genes in the pair. In some of the scenarios where the pair is conserved in two species (D, E, G, and H), the rearranged or deleted states are just as likely to be the ancestral state as the divergent pair shown. In scenario L, it is also possible that both genes existed in the common ancestor and a deletion took place in the top branch rather than new gene(s) being created.</p></div></div><div content-id="DSQ022F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/m_dsq02203.jpeg?Expires=2147483647&Signature=A72yvzbNpR3CAYpM9YvQ11ulBq7to5gHfd32tp0rRU2L6gejdNcJPdivnY9szwBXuV4~IPFi9sUqo~JVvlfr9X1mBVZ~igBQuKrMj~ahRiZ0BSx1V4qZ9s7NTzQDrAKCqKqocjZ0Ihy5-gEdGcq-NiJunjwV6wB45~nCeyzn4bqfEC-1b0iV3Km48LXP7c9qFwFpW~~rz6EhbFmu43AZglVefY0t4rQUbXQXfk1FgvgWeB246JRzkgqBDVpHFQWFrz2S9NjlwtsENwhbCLPFiNC8SEh~dS9VXgTzIc4Cbs3YGOUnm82yz4VvwHwVy644nrYganA8w4twk7tAK5nBpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Categories of gene-pair evolution. Each image is a representative of the many specific scenarios that may be found in that category. The bottom branch of each tree represents the species in which the pair was first identified (i.e. either rice or sorghum), and the two genes in question are shown in a divergent pair in these examples. Rearrangements are represented by the inversion of one gene, inversion of both genes, insertions within the pair, or translocation to other regions or chromosomes. Likewise, deletions may involve one gene, as shown, or both genes in the pair. In some of the scenarios where the pair is conserved in two species (D, E, G, and H), the rearranged or deleted states are just as likely to be the ancestral state as the divergent pair shown. In scenario L, it is also possible that both genes existed in the common ancestor and a deletion took place in the top branch rather than new gene(s) being created." path-from-xml="dsq02203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7356685/dsq02203.jpeg" path-from-xml="dsq02203.jpeg" target="_blank">View large</a><a section="7356685" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq02203.jpeg?Expires=2147483647&Signature=PqhvM1SCoPP4~ILwP7b4ZoVa0TM0-rFKHlhWUlFXMhQJyLL4x8OZ6~lnsHkItgpKzthJIsLTl7xclBid3Myl1Kkp~CFJaAovXukoaM8Ppp0CzUCDXe8RtKX3HyyTIDQeRvXaC9-83wTD-Dy8NgqLEYj85XNWI25VqUsukUput7EN4kvPyvqqEXTzasW1vmQ9HhtB7O6I3g8PtdkaX6hTwl4cI5tJC0hMwAY071FHEFwC8g2x-UDxx85RvT-Ll0-O3Jy3J56kP~ReHfuhzBqFSK2FZRjb9gp2zEQGBF3VSKoZaP2cPlZmKgYiBwxhlsIPH159xKEU1XvYFX22OWnWpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7356685&ar=484317&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Categories of gene-pair evolution. Each image is a representative of the many specific scenarios that may be found in that category. The bottom branch of each tree represents the species in which the pair was first identified (i.e. either rice or sorghum), and the two genes in question are shown in a divergent pair in these examples. Rearrangements are represented by the inversion of one gene, inversion of both genes, insertions within the pair, or translocation to other regions or chromosomes. Likewise, deletions may involve one gene, as shown, or both genes in the pair. In some of the scenarios where the pair is conserved in two species (D, E, G, and H), the rearranged or deleted states are just as likely to be the ancestral state as the divergent pair shown. In scenario L, it is also possible that both genes existed in the common ancestor and a deletion took place in the top branch rather than new gene(s) being created.</p></div></div><p>Pairs conserved in two of their three comparison species most likely underwent a single species-specific rearrangement or deletion (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a> three species, and Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>B and C). The former event was by far the most common, comprising 16–19% of all pairs, compared with the ∼1% or less of pairs with one or both homologs deleted in a single species. Rice and sorghum results differed by less than one percentage point across all pair types.</p><p>The six categories comprise pairs conserved in only one other species were divided into two groups (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a>, two species), those in which the pair was conserved within one branch of the evolutionary tree (i.e. a rice pair conserved in <em>Brachypodium</em>), referred to here as a ‘branch-specific' pair (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>D–F), and those in which the pair was conserved in one species in each branch, a state referred to as ‘cross-branch conservation' (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>G–I). Among both branch-specific and cross-branch conserved pairs, it was far more common (7–12% of pairs in rice and 4–15% in sorghum) for the pair to be rearranged in the other two species than for it to be deleted in one (0.5–1.9%) or both species (0.1–1.4%). In rice, branch-specific pairs were slightly more common than cross-branch conserved pairs; in sorghum, the opposite was true. There were only two sets of genes in which rice and sorghum differed substantially. The first was branch-specific divergent pairs with the pair being rearranged in the other two species, including 11.7% of rice divergent pairs but only 4.4% of sorghum pairs (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a>, two species). The opposite situation was observed among cross-branch conserved divergent pairs, again with two rearrangements. These pairs made up 9.3% of rice divergent pairs, compared with 14.6% of sorghum pairs.</p><p>The last five categories consist of pairs with unique orientation in only one species (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a>, one species). Pairs whose genes exist in all four species but whose pair-wise arrangement is found in a single species (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>K) were the most common category in both species (25–35% in sorghum and 21–28% in rice). Divergent pairs fell into this category substantially more often than convergent or tandem pairs in both rice and sorghum. In rice, the second most common category (10–12% of all pairs) is those pairs containing one or more genes unique to that species (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>J); in sorghum, this category is approximately one-third that of rice. These observations are most likely due to differences in the gene annotation methods used by the two genome projects and/or a larger number of genes unique to rice than to sorghum reflecting their biology. This conclusion is supported by RAP data which identified 56, 28, and 172 species-specific genes that are part of convergent, divergent, and tandem gene pairs, respectively, in rice but not the other three grass genomes, respectively, compared with 505, 475, and 1130 species-specific rice genes in MSU data that are part of convergent, divergent, and tandem gene pairs, respectively. A few other studies have also identified several genes and gene families specific to rice and sorghum.<sup><a reveal-id="DSQ022C7" data-open="DSQ022C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSQ022C23" data-open="DSQ022C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The third category (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>L) for all pair types were more than 2-fold in sorghum compared with rice (Table <a reveal-id="DSQ022TB3" data-open="DSQ022TB3" class="link link-reveal link-table xref-fig">3</a>, one species). The distribution of pairs among the remaining two categories (Fig. <a reveal-id="DSQ022F3" data-open="DSQ022F3" class="link link-reveal link-table xref-fig">3</a>M and N) showed little variation both between rice and sorghum and between pair types.</p><p>Overall, our study provides valuable insights into conservation and rearrangement of gene pairs during the evolution of the grasses serving as basis for future investigations on functional interactions between adjacent genes.</p>                            <h2 scrollto-destination=7356690 id="7356690" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7356692 id="7356692" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the National Research Initiative of the USDA Cooperative State Research, Education and Extension Service (2007-35301-18036).</p>                            <h2 scrollto-destination=7356695 id="7356695" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ022C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krom</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Comparative analysis of divergent and convergent gene pairs and their expression patterns in rice, <em>Arabidopsis</em>, and <em>Populus</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">147</div> (pg. <div class="fpage">1763</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20divergent%20and%20convergent%20gene%20pairs%20and%20their%20expression%20patterns%20in%20rice%2C%20Arabidopsis%2C%20and%20Populus&amp;author=N.%20Krom&amp;author=W.%20Ramakrishna&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=147&amp;pages=1763-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.122416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.122416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.122416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18515639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Y.W.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Date of the monocot-dicot divergence estimated from chloroplast DNA sequence data</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1989</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">6201</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.86.16.6201" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.86.16.6201</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Date%20of%20the%20monocot-dicot%20divergence%20estimated%20from%20chloroplast%20DNA%20sequence%20data&amp;author=K.H.%20Wolfe&amp;author=M.%20Gouy&amp;author=Y.W.%20Yang&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1989&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=6201-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.16.6201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.16.6201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.16.6201"> </span></div> </div></div></div></div></div><div content-id="DSQ022C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaw</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">C.C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H.L.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Dating the monocot–dicot divergence and the origin of core eudicots using whole chloroplast genomes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">424</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00239-003-2564-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00239-003-2564-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dating%20the%20monocot%E2%80%93dicot%20divergence%20and%20the%20origin%20of%20core%20eudicots%20using%20whole%20chloroplast%20genomes&amp;author=S.M.%20Chaw&amp;author=C.C.%20Chang&amp;author=H.L.%20Chen&amp;author=W.H.%20Li&amp;publication_year=2004&amp;journal=J.%20Mol.%20Evol.&amp;volume=58&amp;pages=424-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-003-2564-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-003-2564-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-003-2564-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15114421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">604</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1128691" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1128691</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buell</div> <div class="given-names">C.R.</div></div>. </div><div class="article-title">Poaceae genomes: going from unattainable to becoming a model clade for comparative plant genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">111</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.108.128926" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.108.128926</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poaceae%20genomes%3A%20going%20from%20unattainable%20to%20becoming%20a%20model%20clade%20for%20comparative%20plant%20genomics&amp;author=C.R.%20Buell&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=111-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.128926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.128926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.128926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19005087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="collab">International Rice Genome Sequencing Project (IRGSP)</div><div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature03895" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature03895</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">457</div> (pg. <div class="fpage">551</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature07723" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature07723</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.H.%20Paterson&amp;author=J.E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div>, et al. </div><div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">326</div> (pg. <div class="fpage">1112</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="collab">International Brachypodium Initiative (IBI)</div><div class="article-title">Genome sequencing and analysis of the model grass <em>Brachypodium distachyon</em></div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">763</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08747" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08747</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20148030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Numerous small rearrangements of gene content, order and orientation differentiate grass genomes</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">821</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1014841515249" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1014841515249</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Numerous%20small%20rearrangements%20of%20gene%20content%2C%20order%20and%20orientation%20differentiate%20grass%20genomes&amp;author=J.L.%20Bennetzen&amp;author=W.%20Ramakrishna&amp;publication_year=2002&amp;journal=Plant%20Mol.%20Biol.&amp;volume=48&amp;pages=821-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1014841515249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1014841515249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1014841515249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11999852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ilic</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sanmiguel</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">A complex history of rearrangement in an orthologous region of the maize, sorghum and rice genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">12265</div>-<div class="lpage">70</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.1434476100" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.1434476100</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complex%20history%20of%20rearrangement%20in%20an%20orthologous%20region%20of%20the%20maize%2C%20sorghum%20and%20rice%20genomes&amp;author=K.%20Ilic&amp;author=P.J.%20Sanmiguel&amp;author=J.L.%20Bennetzen&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=12265-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1434476100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1434476100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1434476100"> </span></div> </div></div></div></div></div><div content-id="DSQ022C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Pal</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lercher</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">The evolutionary dynamics of eukaryotic gene order</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">299</div>-<div class="lpage">310</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1319" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1319</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20dynamics%20of%20eukaryotic%20gene%20order&amp;author=L.D.%20Hurst&amp;author=C.%20Pal&amp;author=M.J.%20Lercher&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=299-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15131653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ciccarelli</div> <div class="given-names">F.D.</div></div>,  <div class="name"><div class="surname">Von Mering</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Suyuma</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Harrington</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Izaurralde</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Complex genomic rearrangements lead to novel primate gene function</div>, <div class="source">Genome Res.</div> , <div class="year">2005</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">343</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.3266405" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.3266405</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20genomic%20rearrangements%20lead%20to%20novel%20primate%20gene%20function&amp;author=F.D.%20Ciccarelli&amp;author=C.%20Von%20Mering&amp;author=M.%20Suyuma&amp;author=E.D.%20Harrington&amp;author=E.%20Izaurralde&amp;author=P.%20Bork&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=343-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3266405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3266405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3266405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15710750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meyers</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Vu</div> <div class="given-names">T.H.</div></div>, et al. </div><div class="article-title"><em>Arabidopsis</em> MPSS. An online resource for quantitative expression analysis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">801</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.039495" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.039495</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20MPSS.%20An%20online%20resource%20for%20quantitative%20expression%20analysis&amp;author=B.C.%20Meyers&amp;author=D.K.%20Lee&amp;author=T.H.%20Vu&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=801-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.039495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.039495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.039495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173564" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">T-H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">Y-K.</div></div>,  <div class="name"><div class="surname">Pham</div> <div class="given-names">T.T.M.</div></div>, et al. </div><div class="article-title">RiceArrayNet: a database for correlating gene expression from transcriptome profiling, and its application to the analysis of coexpressed genes in rice</div>, <div class="source">Plant Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">16</div>-<div class="lpage">33</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.109.139030" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.109.139030</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RiceArrayNet%3A%20a%20database%20for%20correlating%20gene%20expression%20from%20transcriptome%20profiling%2C%20and%20its%20application%20to%20the%20analysis%20of%20coexpressed%20genes%20in%20rice&amp;author=T-H.%20Lee&amp;author=Y-K.%20Kim&amp;author=T.T.M.%20Pham&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=151&amp;pages=16-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.139030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.139030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.139030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19605550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Swigonova</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">Gene loss and movement in the maize genome</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1924</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2701104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2701104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20loss%20and%20movement%20in%20the%20maize%20genome&amp;author=J.%20Lai&amp;author=J.%20Ma&amp;author=Z.%20Swigonova&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1924-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2701104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2701104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2701104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="collab">The Rice Annotation Project Database (RAP-DB): 2008 update</div><div class="article-title">Rice Annotation Project</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> <div class="issue">Database issue</div>(pg. <div class="fpage">D1028</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18089549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Coleman</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Consistent over-estimation of gene number in complex plant genomes</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">732</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.pbi.2004.09.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.pbi.2004.09.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consistent%20over-estimation%20of%20gene%20number%20in%20complex%20plant%20genomes&amp;author=J.L.%20Bennetzen&amp;author=C.%20Coleman&amp;author=R.%20Liu&amp;author=J.%20Ma&amp;author=W.%20Ramakrishna&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=7&amp;pages=732-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2004.09.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2004.09.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2004.09.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15491923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dhadi</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Krom</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ramakrishna</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Genome-wide comparative analysis of putative bidirectional promoters from rice, Arabidopsis, and Populus,</div>, <div class="source">Gene</div> , <div class="year">2009</div>, vol. <div class="volume">429</div> (pg. <div class="fpage">65</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20analysis%20of%20putative%20bidirectional%20promoters%20from%20rice%2C%20Arabidopsis%2C%20and%20Populus%2C&amp;author=S.R.%20Dhadi&amp;author=N.%20Krom&amp;author=W.%20Ramakrishna&amp;publication_year=2009&amp;journal=Gene&amp;volume=429&amp;pages=65-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.09.034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.09.034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.09.034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18973799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kensche</div> <div class="given-names">P.R.</div></div>,  <div class="name"><div class="surname">Oti</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dutilh</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">Conservation of divergent transcription in fungi</div>, <div class="source">Trends Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">207</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tig.2008.02.003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tig.2008.02.003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20divergent%20transcription%20in%20fungi&amp;author=P.R.%20Kensche&amp;author=M.%20Oti&amp;author=B.E.%20Dutilh&amp;author=M.A.%20Huynen&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=207-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2008.02.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2008.02.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2008.02.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Huminiecki</div> <div class="given-names">L.B.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>. </div><div class="article-title">Clusters of co-expressed genes in mammalian genomes are conserved by natural selection</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">767</div>-<div class="lpage">75</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msi062" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msi062</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clusters%20of%20co-expressed%20genes%20in%20mammalian%20genomes%20are%20conserved%20by%20natural%20selection&amp;author=G.A.%20Singer&amp;author=A.T.%20Lloyd&amp;author=L.B.%20Huminiecki&amp;author=K.H.%20Wolfe&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=767-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15574806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Semon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Evolutionary origin and maintenance of coexpressed gene clusters in mammals</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1715</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msl034" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msl034</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20origin%20and%20maintenance%20of%20coexpressed%20gene%20clusters%20in%20mammals&amp;author=M.%20Semon&amp;author=L.%20Duret&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1715-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16757654" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ022C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ022C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campbell</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Identification and characterization of lineage-specific genes within the Poaceae</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">1311</div>-<div class="lpage">22</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.107.104513" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.107.104513</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20lineage-specific%20genes%20within%20the%20Poaceae&amp;author=M.A.%20Campbell&amp;author=W.%20Zhu&amp;author=N.%20Jiang&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=145&amp;pages=1311-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.104513" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.104513" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.104513"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17951464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484317&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq022/2/dsq022-dsq022supp_tables.pdf?Expires=2147483647&amp;Signature=WOQXQIMr45tcrWYPsTo4sd2dwNlqMKe84SORU5iIQ9EeiBxC5b8Q2NXUxWVn2ZgmhRi-QZ4zXia1h87ydzoBGd7te2Lzx-MslAr9CAPtaNxv6OgpTMSHkmq0FglAMPBZ6t66RNCyjAByc8hvwTiG-wsFH8qL7hXMBtjG0cAOLS2NeB9LUHCD93XWYXEb6BZkRTGVc2C-gDfbdMRSZ4leVBufcz9Tqhk8CRdD7oYAR-igsOR6rjVCiDvqK9gPSTn3xr0zZntXsgISnBxnXcWBe8VkTGxYMxpyuTUytHMPJCOXm3Qo28MFn5U8exIyJZIGVu-Fco9wkt~ypKFaUjeW3Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">92</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484317"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f6%2f343%2f484317">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000285800200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F6%2F343%2F484317" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484317'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Conservation, Rearrangement, and Deletion of Gene Pairs During the Evolution of Four Grass Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f343%2f484317">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

