

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/425/2462458","@type":"ScholarlyArticle","name":"Improving a  Synechocystis -based photoautotrophic chassis through systematic genome mapping and validation of neutral sites","datePublished":"2015-10-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv024","keywords":["neutral sites","photoautotrophic chassis","<italic>Synechocystis</italic>","synthetic biology"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/425/2462458","author":[{"name":"Pinto, Filipe","affiliation":"1  i3S—Instituto de Investigação e Inovação em Saúde ,  Universidade do Porto ,  Porto ,  Portugal    2  IBMC—Instituto de Biologia Molecular e Celular ,  Universidade do Porto ,  Porto 4150-180 ,  Portugal    3  Faculdade de Ciências, Departamento de Biologia ,  Universidade do Porto ,  Porto 4150-171 ,  Portugal","@type":"Person"},{"name":"Pacheco, Catarina C.","affiliation":"1  i3S—Instituto de Investigação e Inovação em Saúde ,  Universidade do Porto ,  Porto ,  Portugal    2  IBMC—Instituto de Biologia Molecular e Celular ,  Universidade do Porto ,  Porto 4150-180 ,  Portugal","@type":"Person"},{"name":"Oliveira, Paulo","affiliation":"1  i3S—Instituto de Investigação e Inovação em Saúde ,  Universidade do Porto ,  Porto ,  Portugal    2  IBMC—Instituto de Biologia Molecular e Celular ,  Universidade do Porto ,  Porto 4150-180 ,  Portugal","@type":"Person"},{"name":"Montagud, Arnau","affiliation":"4  Instituto Universitario de Matemática Pura y Aplicada ,  Universitat Politècnica de València ,  Valencia 46022 ,  Spain","@type":"Person"},{"name":"Landels, Andrew","affiliation":"5  ChELSI Institute, Department of Chemical and Biological Engineering ,  University of Sheffield ,  Sheffield S10 2TN ,  UK","@type":"Person"},{"name":"Couto, Narciso","affiliation":"5  ChELSI Institute, Department of Chemical and Biological Engineering ,  University of Sheffield ,  Sheffield S10 2TN ,  UK","@type":"Person"},{"name":"Wright, Phillip C.","affiliation":"5  ChELSI Institute, Department of Chemical and Biological Engineering ,  University of Sheffield ,  Sheffield S10 2TN ,  UK","@type":"Person"},{"name":"Urchueguía, Javier F.","affiliation":"4  Instituto Universitario de Matemática Pura y Aplicada ,  Universitat Politècnica de València ,  Valencia 46022 ,  Spain","@type":"Person"},{"name":"Tamagnini, Paula","affiliation":"1  i3S—Instituto de Investigação e Inovação em Saúde ,  Universidade do Porto ,  Porto ,  Portugal    2  IBMC—Instituto de Biologia Molecular e Celular ,  Universidade do Porto ,  Porto 4150-180 ,  Portugal    3  Faculdade de Ciências, Departamento de Biologia ,  Universidade do Porto ,  Porto 4150-171 ,  Portugal","@type":"Person"}],"description":"Abstract.  The use of microorganisms as cell factories frequently requires extensive molecular manipulation. Therefore, the identification of genomic neutral si","pageStart":"425","pageEnd":"437","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Improving a  Synechocystis -based photoautotrophic chassis through systematic genome mapping and validation of neutral sites","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Improving a  Synechocystis -based photoautotrophic chassis through systematic genome mapping and validation of neutral sites" />
    <meta property="og:description" content="Abstract.  The use of microorganisms as cell factories frequently requires extensive molecular manipulation. Therefore, the identification of genomic neutral si" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/425/2462458" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-10-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Pinto, Filipe" /><meta name="citation_author_institution" content="i3S—Instituto de Investigação e Inovação em Saúde, Universidade do Porto, Porto, Portugal" /><meta name="citation_author_institution" content="IBMC—Instituto de Biologia Molecular e Celular, Universidade do Porto, Porto 4150-180, Portugal" /><meta name="citation_author_institution" content="Faculdade de Ciências, Departamento de Biologia, Universidade do Porto, Porto 4150-171, Portugal" /><meta name="citation_author" content="Pacheco, Catarina C." /><meta name="citation_author_institution" content="i3S—Instituto de Investigação e Inovação em Saúde, Universidade do Porto, Porto, Portugal" /><meta name="citation_author_institution" content="IBMC—Instituto de Biologia Molecular e Celular, Universidade do Porto, Porto 4150-180, Portugal" /><meta name="citation_author" content="Oliveira, Paulo" /><meta name="citation_author_institution" content="i3S—Instituto de Investigação e Inovação em Saúde, Universidade do Porto, Porto, Portugal" /><meta name="citation_author_institution" content="IBMC—Instituto de Biologia Molecular e Celular, Universidade do Porto, Porto 4150-180, Portugal" /><meta name="citation_author" content="Montagud, Arnau" /><meta name="citation_author_institution" content="Instituto Universitario de Matemática Pura y Aplicada, Universitat Politècnica de València, Valencia 46022, Spain" /><meta name="citation_author" content="Landels, Andrew" /><meta name="citation_author_institution" content="ChELSI Institute, Department of Chemical and Biological Engineering, University of Sheffield, Sheffield S10 2TN, UK" /><meta name="citation_author" content="Couto, Narciso" /><meta name="citation_author_institution" content="ChELSI Institute, Department of Chemical and Biological Engineering, University of Sheffield, Sheffield S10 2TN, UK" /><meta name="citation_author" content="Wright, Phillip C." /><meta name="citation_author_institution" content="ChELSI Institute, Department of Chemical and Biological Engineering, University of Sheffield, Sheffield S10 2TN, UK" /><meta name="citation_author" content="Urchueguía, Javier F." /><meta name="citation_author_institution" content="Instituto Universitario de Matemática Pura y Aplicada, Universitat Politècnica de València, Valencia 46022, Spain" /><meta name="citation_author" content="Tamagnini, Paula" /><meta name="citation_author_institution" content="i3S—Instituto de Investigação e Inovação em Saúde, Universidade do Porto, Porto, Portugal" /><meta name="citation_author_institution" content="IBMC—Instituto de Biologia Molecular e Celular, Universidade do Porto, Porto 4150-180, Portugal" /><meta name="citation_author_institution" content="Faculdade de Ciências, Departamento de Biologia, Universidade do Porto, Porto 4150-171, Portugal" /><meta name="citation_title" content="Improving a Synechocystis-based photoautotrophic chassis through systematic genome mapping and validation of neutral sites" /><meta name="citation_firstpage" content="425" /><meta name="citation_lastpage" content="437" /><meta name="citation_doi" content="10.1093/dnares/dsv024" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Benjamin Cummings, San Francisco;  citation_title=Brock biology of microorganisms;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Bacillus subtilis: model Gram-positive synthetic biology chassis; citation_publisher=Academic Press, Amsterdam, The Netherlands;  citation_title=Methods in microbiology;  citation_year=2013;  citation_pages=87-117; " /><meta name="citation_reference" content="citation_title=Application of synthetic biology for production of chemicals in yeast Saccharomyces cerevisiae;  citation_journal_title=FEMS Yeast Res;  citation_year=2014;  citation_volume=15;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Synthetic biology of cyanobacteria: unique challenges and opportunities;  citation_journal_title=Front. Microbiol.;  citation_year=2013;  citation_volume=4;  citation_pages=246" /><meta name="citation_reference" content="citation_title=Introduction to the cyanobacteria; citation_publisher=Kluwer Academic Publishers, Dordrecht, The Netherlands;  citation_title=The ecology of cyanobacteria—their diversity in time and space;  citation_year=2000;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Reconstruction and analysis of genome-scale metabolic model of a photosynthetic bacterium;  citation_journal_title=BMC Syst. Biol.;  citation_year=2010;  citation_volume=4;  citation_pages=156" /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC 6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions;  citation_journal_title=DNA Res.;  citation_year=1996;  citation_volume=3;  citation_pages=109-36; " /><meta name="citation_reference" content="citation_title=Genetic tools for cyanobacteria;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2002;  citation_volume=58;  citation_pages=123-37; " /><meta name="citation_reference" content="citation_title=Synthetic biology in cyanobacteria: engineering and analyzing novel functions; citation_publisher=Academic Press, New York;  citation_title=Methods enzymol;  citation_year=2011;  citation_pages=539-79; " /><meta name="citation_reference" content="citation_title=Broad-host-range vector system for synthetic biology and biotechnology in cyanobacteria;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=e136" /><meta name="citation_reference" content="citation_title=Construction of specific mutations in photosystem II photosynthetic reaction center by genetic engineering methods in Synechocystis 6803; citation_publisher=Academic Press, New York;  citation_title=Methods enzymol;  citation_year=1988;  citation_pages=766-78; " /><meta name="citation_reference" content="citation_title=Role of disulfide linkage and putative intermolecular binding residues in the stability and binding of the extrinsic manganese-stabilizing protein to the photosystem II reaction center;  citation_journal_title=Biochemistry US;  citation_year=1994;  citation_volume=33;  citation_pages=13712-18; " /><meta name="citation_reference" content="citation_title=A heme oxygenase isoform is essential for aerobic growth in the cyanobacterium Synechocystis sp. PCC 6803: modes of differential operation of two isoforms/enzymes to adapt to low oxygen environments in cyanobacteria;  citation_journal_title=Plant Cell Physiol.;  citation_year=2011;  citation_volume=52;  citation_pages=1744-56; " /><meta name="citation_reference" content="citation_title=Circadian expression of the dnaK gene in the cyanobacterium Synechocystis sp. strain PCC 6803;  citation_journal_title=J. Bacteriol.;  citation_year=1995;  citation_volume=177;  citation_pages=5606-11; " /><meta name="citation_reference" content="citation_title=Fine-tuning of photoautotrophic protein production by combining promoters and neutral sites in the cyanobacterium Synechocystis sp. strain PCC 6803;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2015;  citation_volume=81;  citation_pages=6857-63; " /><meta name="citation_reference" content="citation_title=Specialized techniques for site-directed mutagenesis in cyanobacteria; citation_publisher=Humana Press, Totowa;  citation_title=Circadian rhythms;  citation_year=2007;  citation_pages=155-71; " /><meta name="citation_reference" content="citation_title=Microevolution in cyanobacteria: re-sequencing a motile substrain of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=435-48; " /><meta name="citation_reference" content="citation_title=Identification of substrain-specific mutations by massively parallel whole-genome resequencing of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=67-79; " /><meta name="citation_reference" content="citation_title=Purification and properties of unicellular blue-green algae (order Chroococcales);  citation_journal_title=Bacteriol. Rev.;  citation_year=1971;  citation_volume=35;  citation_pages=171-205; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, New York;  citation_title=Molecular cloning: a laboratory manual;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Hydrogenases in Nostoc sp. strain PCC 73102, a strain lacking a bidirectional enzyme;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=1997;  citation_volume=63;  citation_pages=1801-7; " /><meta name="citation_reference" content="citation_title=Genes involved in the maturation of hydrogenase(s) in the nonheterocystous cyanobacterium Lyngbya majuscula CCAP 1446/4;  citation_journal_title=Int. J. Hydrogen Energ.;  citation_year=2006;  citation_volume=31;  citation_pages=1469-77; " /><meta name="citation_reference" content="citation_title=Transcription profiles of hydrogenases related genes in the cyanobacterium Lyngbya majuscula CCAP 1446/4;  citation_journal_title=BMC Microbiol.;  citation_year=2009;  citation_volume=9;  citation_pages=67" /><meta name="citation_reference" content="citation_title=Small mobilizable multi-purpose cloning vectors derived from the Escherichia coli plasmids pK18 and pK19: selection of defined deletions in the chromosome of Corynebacterium glutamicum;  citation_journal_title=Gene;  citation_year=1994;  citation_volume=145;  citation_pages=69-73; " /><meta name="citation_reference" content="citation_title=Deletion mutagenesis of the 5′ psbA2 region in Synechocystis 6803: identification of a putative cis element involved in photoregulation;  citation_journal_title=Mol. Cell. Biol. Res. Commun.;  citation_year=2000;  citation_volume=3;  citation_pages=292-8; " /><meta name="citation_reference" content="citation_title=The 5′-upstream cis-acting sequences of a cyanobacterial psbA gene: analysis of their roles in basal, light-dependent and circadian transcription;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2002;  citation_volume=267;  citation_pages=684-94; " /><meta name="citation_reference" content="citation_title=Selection of suitable reference genes for RT-qPCR analyses in cyanobacteria;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e34983" /><meta name="citation_reference" content="citation_title=The MIQE Guidelines: minimum information for publication of quantitative real-time PCR experiments;  citation_journal_title=Clin. Chem.;  citation_year=2009;  citation_volume=55;  citation_pages=611-22; " /><meta name="citation_reference" content="citation_title=Assay of proteins in the presence of interfering materials;  citation_journal_title=Anal. Biochem.;  citation_year=1976;  citation_volume=70;  citation_pages=241-50; " /><meta name="citation_reference" content="citation_title=An easy-to-use Decoy Database Builder software tool, implementing different decoy strategies for false discovery rate calculation in automated MS/MS protein identifications;  citation_journal_title=Proteomics;  citation_year=2008;  citation_volume=8;  citation_pages=1129-37; " /><meta name="citation_reference" content="citation_title=Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry;  citation_journal_title=Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=207-14; " /><meta name="citation_reference" content="citation_title=iTRAQ underestimation in simple and complex mixtures: “the good, the bad and the ugly”;  citation_journal_title=J. Proteome Res.;  citation_year=2009;  citation_volume=8;  citation_pages=5347-55; " /><meta name="citation_reference" content="citation_title=A quantitative proteomic analysis of biofilm adaptation by the periodontal pathogen Tannerella forsythia;  citation_journal_title=Proteomics;  citation_year=2010;  citation_volume=10;  citation_pages=3130-41; " /><meta name="citation_reference" content="citation_title=Genetic engineering in marine cyanobacteria;  citation_journal_title=J. Appl. Phycol.;  citation_year=1995;  citation_volume=7;  citation_pages=77-84; " /><meta name="citation_reference" content="citation_title=Transformation of cyanobacteria; citation_publisher=Springer, New York;  citation_year=2007;  citation_volume=616;  citation_pages=12-22; " /><meta name="citation_reference" content="citation_title=Stability of a promiscuous plasmid in different hosts: no guarantee for a long-term relationship;  citation_journal_title=Microbiology;  citation_year=2007;  citation_volume=153;  citation_pages=452-63; " /><meta name="citation_reference" content="citation_title=Replication and conjugative mobilization of broad host-range IncQ plasmids;  citation_journal_title=Plasmid;  citation_year=2009;  citation_volume=62;  citation_pages=57-70; " /><meta name="citation_reference" content="citation_title=Up-regulation of plasmid-encoded genes during stationary phase in Synechocystis sp. PCC 6803, a cyanobacterium;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2012;  citation_volume=78;  citation_pages=5448-51; " /><meta name="citation_reference" content="citation_title=Ploidy in cyanobacteria;  citation_journal_title=FEMS Microbiol. Lett.;  citation_year=2011;  citation_volume=323;  citation_pages=124-31; " /><meta name="citation_reference" content="citation_title=Identification of secreted proteins of the cyanobacterium Synechocystis sp. strain PCC 6803;  citation_journal_title=FEMS Microbiol. Lett.;  citation_year=2000;  citation_volume=193;  citation_pages=213-6; " /><meta name="citation_reference" content="citation_title=CDD: specific functional annotation with the Conserved Domain Database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D205-10; " /><meta name="citation_reference" content="citation_title=An experimentally anchored map of transcriptional start sites in the model cyanobacterium Synechocystis sp. PCC6803;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=2124-9; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the primary transcriptome of Synechocystis sp. PCC 6803;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=527-39; " /><meta name="citation_reference" content="citation_title=Mutation of ndh genes leads to inhibition of CO2 uptake rather than HCO3− uptake in Synechocystis sp. strain PCC 6803;  citation_journal_title=J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=2591-6; " /><meta name="citation_reference" content="citation_title=Stress responses of Synechocystis sp. strain PCC 6803 mutants impaired in genes encoding putative alternative sigma factors;  citation_journal_title=Microbiology;  citation_year=2000;  citation_volume=146;  citation_pages=2877-89; " /><meta name="citation_reference" content="citation_title=Non-coding RNAs and their epigenetic regulatory mechanisms;  citation_journal_title=Biol. Cell;  citation_year=2010;  citation_volume=102;  citation_pages=645-55; " /><meta name="citation_reference" content="citation_publisher=Caister Academic Press, Norfolk, UK;  citation_title=Non-coding RNAs and the epigenetic regulation of gene expression: drivers of natural selection;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression;  citation_journal_title=RNA;  citation_year=2010;  citation_volume=16;  citation_pages=991-1006; " /><meta name="citation_reference" content="citation_title=An AbrB-like transcriptional regulator, Sll0822, is essential for the activation of nitrogen-regulated genes in Synechocystis sp. PCC 6803;  citation_journal_title=Plant Physiol.;  citation_year=2008;  citation_volume=148;  citation_pages=660-70; " /><meta name="citation_reference" content="citation_title=An AbrB-like protein regulates the expression of the bidirectional hydrogenase in Synechocystis sp. strain PCC 6803;  citation_journal_title=J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=1011-9; " /><meta name="citation_reference" content="citation_title=Experimental and modeling analysis of Synechocystis sp. PCC 6803 growth;  citation_journal_title=J. Mol. Microbiol. Biotechnol.;  citation_year=2012;  citation_volume=22;  citation_pages=71-82; " /><meta name="citation_reference" content="citation_title=DNA supercoiling regulates the stress-inducible expression of genes in the cyanobacterium Synechocystis;  citation_journal_title=Mol. Biosyst.;  citation_year=2009;  citation_volume=5;  citation_pages=1904-12; " /><meta name="citation_reference" content="citation_title=Proteome analysis of salt stress response in the cyanobacterium Synechocystis sp. strain PCC 6803;  citation_journal_title=Proteomics;  citation_year=2006;  citation_volume=6;  citation_pages=2733-45; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/425/17463062/dsv024.pdf" /><meta name="description" content="Abstract.  The use of microorganisms as cell factories frequently requires extensive molecular manipulation. Therefore, the identification of genomic neutral si" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/425/2462458" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/425/2462458" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/425/2462458");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv024';
    </script>
<title>
	Improving a Synechocystis-based photoautotrophic chassis through systematic genome mapping and validation of neutral sites | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462458" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bc6OATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGludG8sIEZpbGlwZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9ImkzU+KAlEluc3RpdHV0byBkZSBJbnZlc3RpZ2HDp8OjbyBlIElub3Zhw6fDo28gZW0gU2HDumRlLCBVbml2ZXJzaWRhZGUgZG8gUG9ydG8sIFBvcnRvLCBQb3J0dWdhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklCTUPigJRJbnN0aXR1dG8gZGUgQmlvbG9naWEgTW9sZWN1bGFyIGUgQ2VsdWxhciwgVW5pdmVyc2lkYWRlIGRvIFBvcnRvLCBQb3J0byA0MTUwLTE4MCwgUG9ydHVnYWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bGRhZGUgZGUgQ2nDqm5jaWFzLCBEZXBhcnRhbWVudG8gZGUgQmlvbG9naWEsIFVuaXZlcnNpZGFkZSBkbyBQb3J0bywgUG9ydG8gNDE1MC0xNzEsIFBvcnR1Z2FsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFjaGVjbywgQ2F0YXJpbmEgQy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJpM1PigJRJbnN0aXR1dG8gZGUgSW52ZXN0aWdhw6fDo28gZSBJbm92YcOnw6NvIGVtIFNhw7pkZSwgVW5pdmVyc2lkYWRlIGRvIFBvcnRvLCBQb3J0bywgUG9ydHVnYWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJQk1D4oCUSW5zdGl0dXRvIGRlIEJpb2xvZ2lhIE1vbGVjdWxhciBlIENlbHVsYXIsIFVuaXZlcnNpZGFkZSBkbyBQb3J0bywgUG9ydG8gNDE1MC0xODAsIFBvcnR1Z2FsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2xpdmVpcmEsIFBhdWxvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iaTNT4oCUSW5zdGl0dXRvIGRlIEludmVzdGlnYcOnw6NvIGUgSW5vdmHDp8OjbyBlbSBTYcO6ZGUsIFVuaXZlcnNpZGFkZSBkbyBQb3J0bywgUG9ydG8sIFBvcnR1Z2FsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSUJNQ+KAlEluc3RpdHV0byBkZSBCaW9sb2dpYSBNb2xlY3VsYXIgZSBDZWx1bGFyLCBVbml2ZXJzaWRhZGUgZG8gUG9ydG8sIFBvcnRvIDQxNTAtMTgwLCBQb3J0dWdhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vbnRhZ3VkLCBBcm5hdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0byBVbml2ZXJzaXRhcmlvIGRlIE1hdGVtw6F0aWNhIFB1cmEgeSBBcGxpY2FkYSwgVW5pdmVyc2l0YXQgUG9saXTDqGNuaWNhIGRlIFZhbMOobmNpYSwgVmFsZW5jaWEgNDYwMjIsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGFuZGVscywgQW5kcmV3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2hFTFNJIEluc3RpdHV0ZSwgRGVwYXJ0bWVudCBvZiBDaGVtaWNhbCBhbmQgQmlvbG9naWNhbCBFbmdpbmVlcmluZywgVW5pdmVyc2l0eSBvZiBTaGVmZmllbGQsIFNoZWZmaWVsZCBTMTAgMlROLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNvdXRvLCBOYXJjaXNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2hFTFNJIEluc3RpdHV0ZSwgRGVwYXJ0bWVudCBvZiBDaGVtaWNhbCBhbmQgQmlvbG9naWNhbCBFbmdpbmVlcmluZywgVW5pdmVyc2l0eSBvZiBTaGVmZmllbGQsIFNoZWZmaWVsZCBTMTAgMlROLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldyaWdodCwgUGhpbGxpcCBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNoRUxTSSBJbnN0aXR1dGUsIERlcGFydG1lbnQgb2YgQ2hlbWljYWwgYW5kIEJpb2xvZ2ljYWwgRW5naW5lZXJpbmcsIFVuaXZlcnNpdHkgb2YgU2hlZmZpZWxkLCBTaGVmZmllbGQgUzEwIDJUTiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVcmNodWVndcOtYSwgSmF2aWVyIEYuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRvIFVuaXZlcnNpdGFyaW8gZGUgTWF0ZW3DoXRpY2EgUHVyYSB5IEFwbGljYWRhLCBVbml2ZXJzaXRhdCBQb2xpdMOoY25pY2EgZGUgVmFsw6huY2lhLCBWYWxlbmNpYSA0NjAyMiwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYW1hZ25pbmksIFBhdWxhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iaTNT4oCUSW5zdGl0dXRvIGRlIEludmVzdGlnYcOnw6NvIGUgSW5vdmHDp8OjbyBlbSBTYcO6ZGUsIFVuaXZlcnNpZGFkZSBkbyBQb3J0bywgUG9ydG8sIFBvcnR1Z2FsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSUJNQ+KAlEluc3RpdHV0byBkZSBCaW9sb2dpYSBNb2xlY3VsYXIgZSBDZWx1bGFyLCBVbml2ZXJzaWRhZGUgZG8gUG9ydG8sIFBvcnRvIDQxNTAtMTgwLCBQb3J0dWdhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsZGFkZSBkZSBDacOqbmNpYXMsIERlcGFydGFtZW50byBkZSBCaW9sb2dpYSwgVW5pdmVyc2lkYWRlIGRvIFBvcnRvLCBQb3J0byA0MTUwLTE3MSwgUG9ydHVnYWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkltcHJvdmluZyBhIFN5bmVjaG9jeXN0aXMtYmFzZWQgcGhvdG9hdXRvdHJvcGhpYyBjaGFzc2lzIHRocm91Z2ggc3lzdGVtYXRpYyBnZW5vbWUgbWFwcGluZyBhbmQgdmFsaWRhdGlvbiBvZiBuZXV0cmFsIHNpdGVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3YwMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNS8xMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1CZW5qYW1pbiBDdW1taW5ncywgU2FuIEZyYW5jaXNjbzsgIGNpdGF0aW9uX3RpdGxlPUJyb2NrIGJpb2xvZ3kgb2YgbWljcm9vcmdhbmlzbXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY2lsbHVzIHN1YnRpbGlzOiBtb2RlbCBHcmFtLXBvc2l0aXZlIHN5bnRoZXRpYyBiaW9sb2d5IGNoYXNzaXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgQW1zdGVyZGFtLCBUaGUgTmV0aGVybGFuZHM7ICBjaXRhdGlvbl90aXRsZT1NZXRob2RzIGluIG1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3BhZ2VzPTg3LTExNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwbGljYXRpb24gb2Ygc3ludGhldGljIGJpb2xvZ3kgZm9yIHByb2R1Y3Rpb24gb2YgY2hlbWljYWxzIGluIHllYXN0IFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBZZWFzdCBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW50aGV0aWMgYmlvbG9neSBvZiBjeWFub2JhY3RlcmlhOiB1bmlxdWUgY2hhbGxlbmdlcyBhbmQgb3Bwb3J0dW5pdGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvZHVjdGlvbiB0byB0aGUgY3lhbm9iYWN0ZXJpYTsgY2l0YXRpb25fcHVibGlzaGVyPUtsdXdlciBBY2FkZW1pYyBQdWJsaXNoZXJzLCBEb3JkcmVjaHQsIFRoZSBOZXRoZXJsYW5kczsgIGNpdGF0aW9uX3RpdGxlPVRoZSBlY29sb2d5IG9mIGN5YW5vYmFjdGVyaWHigJR0aGVpciBkaXZlcnNpdHkgaW4gdGltZSBhbmQgc3BhY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl9wYWdlcz0xLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbnN0cnVjdGlvbiBhbmQgYW5hbHlzaXMgb2YgZ2Vub21lLXNjYWxlIG1ldGFib2xpYyBtb2RlbCBvZiBhIHBob3Rvc3ludGhldGljIGJhY3Rlcml1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIHRoZSB1bmljZWxsdWxhciBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDMuIElJLiBTZXF1ZW5jZSBkZXRlcm1pbmF0aW9uIG9mIHRoZSBlbnRpcmUgZ2Vub21lIGFuZCBhc3NpZ25tZW50IG9mIHBvdGVudGlhbCBwcm90ZWluLWNvZGluZyByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTA5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHRvb2xzIGZvciBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MTIzLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW50aGV0aWMgYmlvbG9neSBpbiBjeWFub2JhY3RlcmlhOiBlbmdpbmVlcmluZyBhbmQgYW5hbHl6aW5nIG5vdmVsIGZ1bmN0aW9uczsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgZW56eW1vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3BhZ2VzPTUzOS03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJvYWQtaG9zdC1yYW5nZSB2ZWN0b3Igc3lzdGVtIGZvciBzeW50aGV0aWMgYmlvbG9neSBhbmQgYmlvdGVjaG5vbG9neSBpbiBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1lMTM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIHNwZWNpZmljIG11dGF0aW9ucyBpbiBwaG90b3N5c3RlbSBJSSBwaG90b3N5bnRoZXRpYyByZWFjdGlvbiBjZW50ZXIgYnkgZ2VuZXRpYyBlbmdpbmVlcmluZyBtZXRob2RzIGluIFN5bmVjaG9jeXN0aXMgNjgwMzsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgZW56eW1vbDsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3BhZ2VzPTc2Ni03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBkaXN1bGZpZGUgbGlua2FnZSBhbmQgcHV0YXRpdmUgaW50ZXJtb2xlY3VsYXIgYmluZGluZyByZXNpZHVlcyBpbiB0aGUgc3RhYmlsaXR5IGFuZCBiaW5kaW5nIG9mIHRoZSBleHRyaW5zaWMgbWFuZ2FuZXNlLXN0YWJpbGl6aW5nIHByb3RlaW4gdG8gdGhlIHBob3Rvc3lzdGVtIElJIHJlYWN0aW9uIGNlbnRlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbWlzdHJ5IFVTOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTM3MTItMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGVtZSBveHlnZW5hc2UgaXNvZm9ybSBpcyBlc3NlbnRpYWwgZm9yIGFlcm9iaWMgZ3Jvd3RoIGluIHRoZSBjeWFub2JhY3Rlcml1bSBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMzogbW9kZXMgb2YgZGlmZmVyZW50aWFsIG9wZXJhdGlvbiBvZiB0d28gaXNvZm9ybXMvZW56eW1lcyB0byBhZGFwdCB0byBsb3cgb3h5Z2VuIGVudmlyb25tZW50cyBpbiBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MTc0NC01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lyY2FkaWFuIGV4cHJlc3Npb24gb2YgdGhlIGRuYUsgZ2VuZSBpbiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gc3RyYWluIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTE3NzsgIGNpdGF0aW9uX3BhZ2VzPTU2MDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUtdHVuaW5nIG9mIHBob3RvYXV0b3Ryb3BoaWMgcHJvdGVpbiBwcm9kdWN0aW9uIGJ5IGNvbWJpbmluZyBwcm9tb3RlcnMgYW5kIG5ldXRyYWwgc2l0ZXMgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0MgNjgwMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTY4NTctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpYWxpemVkIHRlY2huaXF1ZXMgZm9yIHNpdGUtZGlyZWN0ZWQgbXV0YWdlbmVzaXMgaW4gY3lhbm9iYWN0ZXJpYTsgY2l0YXRpb25fcHVibGlzaGVyPUh1bWFuYSBQcmVzcywgVG90b3dhOyAgY2l0YXRpb25fdGl0bGU9Q2lyY2FkaWFuIHJoeXRobXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvZXZvbHV0aW9uIGluIGN5YW5vYmFjdGVyaWE6IHJlLXNlcXVlbmNpbmcgYSBtb3RpbGUgc3Vic3RyYWluIG9mIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQzNS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2Ygc3Vic3RyYWluLXNwZWNpZmljIG11dGF0aW9ucyBieSBtYXNzaXZlbHkgcGFyYWxsZWwgd2hvbGUtZ2Vub21lIHJlc2VxdWVuY2luZyBvZiBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz02Ny03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHVyaWZpY2F0aW9uIGFuZCBwcm9wZXJ0aWVzIG9mIHVuaWNlbGx1bGFyIGJsdWUtZ3JlZW4gYWxnYWUgKG9yZGVyIENocm9vY29jY2FsZXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CYWN0ZXJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5NzE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xNzEtMjA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgUHJlc3MsIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmc6IGEgbGFib3JhdG9yeSBtYW51YWw7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5ZHJvZ2VuYXNlcyBpbiBOb3N0b2Mgc3AuIHN0cmFpbiBQQ0MgNzMxMDIsIGEgc3RyYWluIGxhY2tpbmcgYSBiaWRpcmVjdGlvbmFsIGVuenltZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTE4MDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgaW52b2x2ZWQgaW4gdGhlIG1hdHVyYXRpb24gb2YgaHlkcm9nZW5hc2UocykgaW4gdGhlIG5vbmhldGVyb2N5c3RvdXMgY3lhbm9iYWN0ZXJpdW0gTHluZ2J5YSBtYWp1c2N1bGEgQ0NBUCAxNDQ2LzQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gSHlkcm9nZW4gRW5lcmcuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTQ2OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBwcm9maWxlcyBvZiBoeWRyb2dlbmFzZXMgcmVsYXRlZCBnZW5lcyBpbiB0aGUgY3lhbm9iYWN0ZXJpdW0gTHluZ2J5YSBtYWp1c2N1bGEgQ0NBUCAxNDQ2LzQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNtYWxsIG1vYmlsaXphYmxlIG11bHRpLXB1cnBvc2UgY2xvbmluZyB2ZWN0b3JzIGRlcml2ZWQgZnJvbSB0aGUgRXNjaGVyaWNoaWEgY29saSBwbGFzbWlkcyBwSzE4IGFuZCBwSzE5OiBzZWxlY3Rpb24gb2YgZGVmaW5lZCBkZWxldGlvbnMgaW4gdGhlIGNocm9tb3NvbWUgb2YgQ29yeW5lYmFjdGVyaXVtIGdsdXRhbWljdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ1OyAgY2l0YXRpb25fcGFnZXM9NjktNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlbGV0aW9uIG11dGFnZW5lc2lzIG9mIHRoZSA14oCyIHBzYkEyIHJlZ2lvbiBpbiBTeW5lY2hvY3lzdGlzIDY4MDM6IGlkZW50aWZpY2F0aW9uIG9mIGEgcHV0YXRpdmUgY2lzIGVsZW1lbnQgaW52b2x2ZWQgaW4gcGhvdG9yZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwuIEJpb2wuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjkyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSA14oCyLXVwc3RyZWFtIGNpcy1hY3Rpbmcgc2VxdWVuY2VzIG9mIGEgY3lhbm9iYWN0ZXJpYWwgcHNiQSBnZW5lOiBhbmFseXNpcyBvZiB0aGVpciByb2xlcyBpbiBiYXNhbCwgbGlnaHQtZGVwZW5kZW50IGFuZCBjaXJjYWRpYW4gdHJhbnNjcmlwdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjY3OyAgY2l0YXRpb25fcGFnZXM9Njg0LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWxlY3Rpb24gb2Ygc3VpdGFibGUgcmVmZXJlbmNlIGdlbmVzIGZvciBSVC1xUENSIGFuYWx5c2VzIGluIGN5YW5vYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzQ5ODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTUlRRSBHdWlkZWxpbmVzOiBtaW5pbXVtIGluZm9ybWF0aW9uIGZvciBwdWJsaWNhdGlvbiBvZiBxdWFudGl0YXRpdmUgcmVhbC10aW1lIFBDUiBleHBlcmltZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2xpbi4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz02MTEtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2F5IG9mIHByb3RlaW5zIGluIHRoZSBwcmVzZW5jZSBvZiBpbnRlcmZlcmluZyBtYXRlcmlhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFuYWwuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9MjQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlYXN5LXRvLXVzZSBEZWNveSBEYXRhYmFzZSBCdWlsZGVyIHNvZnR3YXJlIHRvb2wsIGltcGxlbWVudGluZyBkaWZmZXJlbnQgZGVjb3kgc3RyYXRlZ2llcyBmb3IgZmFsc2UgZGlzY292ZXJ5IHJhdGUgY2FsY3VsYXRpb24gaW4gYXV0b21hdGVkIE1TL01TIHByb3RlaW4gaWRlbnRpZmljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMTI5LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXQtZGVjb3kgc2VhcmNoIHN0cmF0ZWd5IGZvciBpbmNyZWFzZWQgY29uZmlkZW5jZSBpbiBsYXJnZS1zY2FsZSBwcm90ZWluIGlkZW50aWZpY2F0aW9ucyBieSBtYXNzIHNwZWN0cm9tZXRyeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yMDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWlUUkFRIHVuZGVyZXN0aW1hdGlvbiBpbiBzaW1wbGUgYW5kIGNvbXBsZXggbWl4dHVyZXM6IOKAnHRoZSBnb29kLCB0aGUgYmFkIGFuZCB0aGUgdWdseeKAnTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUHJvdGVvbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NTM0Ny01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBxdWFudGl0YXRpdmUgcHJvdGVvbWljIGFuYWx5c2lzIG9mIGJpb2ZpbG0gYWRhcHRhdGlvbiBieSB0aGUgcGVyaW9kb250YWwgcGF0aG9nZW4gVGFubmVyZWxsYSBmb3JzeXRoaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3Rlb21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0zMTMwLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGVuZ2luZWVyaW5nIGluIG1hcmluZSBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBcHBsLiBQaHljb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz03Ny04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmb3JtYXRpb24gb2YgY3lhbm9iYWN0ZXJpYTsgY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBOZXcgWW9yazsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02MTY7ICBjaXRhdGlvbl9wYWdlcz0xMi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhYmlsaXR5IG9mIGEgcHJvbWlzY3VvdXMgcGxhc21pZCBpbiBkaWZmZXJlbnQgaG9zdHM6IG5vIGd1YXJhbnRlZSBmb3IgYSBsb25nLXRlcm0gcmVsYXRpb25zaGlwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9NDUyLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBsaWNhdGlvbiBhbmQgY29uanVnYXRpdmUgbW9iaWxpemF0aW9uIG9mIGJyb2FkIGhvc3QtcmFuZ2UgSW5jUSBwbGFzbWlkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhc21pZDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTU3LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VcC1yZWd1bGF0aW9uIG9mIHBsYXNtaWQtZW5jb2RlZCBnZW5lcyBkdXJpbmcgc3RhdGlvbmFyeSBwaGFzZSBpbiBTeW5lY2hvY3lzdGlzIHNwLiBQQ0MgNjgwMywgYSBjeWFub2JhY3Rlcml1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTU0NDgtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsb2lkeSBpbiBjeWFub2JhY3RlcmlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzIzOyAgY2l0YXRpb25fcGFnZXM9MTI0LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzZWNyZXRlZCBwcm90ZWlucyBvZiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBzcC4gc3RyYWluIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTkzOyAgY2l0YXRpb25fcGFnZXM9MjEzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNERDogc3BlY2lmaWMgZnVuY3Rpb25hbCBhbm5vdGF0aW9uIHdpdGggdGhlIENvbnNlcnZlZCBEb21haW4gRGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPUQyMDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV4cGVyaW1lbnRhbGx5IGFuY2hvcmVkIG1hcCBvZiB0cmFuc2NyaXB0aW9uYWwgc3RhcnQgc2l0ZXMgaW4gdGhlIG1vZGVsIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIFBDQzY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MjEyNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiB0aGUgcHJpbWFyeSB0cmFuc2NyaXB0b21lIG9mIFN5bmVjaG9jeXN0aXMgc3AuIFBDQyA2ODAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTUyNy0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gb2YgbmRoIGdlbmVzIGxlYWRzIHRvIGluaGliaXRpb24gb2YgQ08yIHVwdGFrZSByYXRoZXIgdGhhbiBIQ08z4oiSIHVwdGFrZSBpbiBTeW5lY2hvY3lzdGlzIHNwLiBzdHJhaW4gUENDIDY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTgyOyAgY2l0YXRpb25fcGFnZXM9MjU5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJlc3MgcmVzcG9uc2VzIG9mIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0MgNjgwMyBtdXRhbnRzIGltcGFpcmVkIGluIGdlbmVzIGVuY29kaW5nIHB1dGF0aXZlIGFsdGVybmF0aXZlIHNpZ21hIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDY7ICBjaXRhdGlvbl9wYWdlcz0yODc3LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob24tY29kaW5nIFJOQXMgYW5kIHRoZWlyIGVwaWdlbmV0aWMgcmVndWxhdG9yeSBtZWNoYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTY0NS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNhaXN0ZXIgQWNhZGVtaWMgUHJlc3MsIE5vcmZvbGssIFVLOyAgY2l0YXRpb25fdGl0bGU9Tm9uLWNvZGluZyBSTkFzIGFuZCB0aGUgZXBpZ2VuZXRpYyByZWd1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbjogZHJpdmVycyBvZiBuYXR1cmFsIHNlbGVjdGlvbjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBjb21wYXJpc29uIG9mIG1pY3JvYXJyYXkgcHJvZmlsaW5nLCByZWFsLXRpbWUgUENSLCBhbmQgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzIGZvciBtZWFzdXJpbmcgZGlmZmVyZW50aWFsIG1pY3JvUk5BIGV4cHJlc3Npb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk5MS0xMDA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBBYnJCLWxpa2UgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvciwgU2xsMDgyMiwgaXMgZXNzZW50aWFsIGZvciB0aGUgYWN0aXZhdGlvbiBvZiBuaXRyb2dlbi1yZWd1bGF0ZWQgZ2VuZXMgaW4gU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0ODsgIGNpdGF0aW9uX3BhZ2VzPTY2MC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gQWJyQi1saWtlIHByb3RlaW4gcmVndWxhdGVzIHRoZSBleHByZXNzaW9uIG9mIHRoZSBiaWRpcmVjdGlvbmFsIGh5ZHJvZ2VuYXNlIGluIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0MgNjgwMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOTA7ICBjaXRhdGlvbl9wYWdlcz0xMDExLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhbmQgbW9kZWxpbmcgYW5hbHlzaXMgb2YgU3luZWNob2N5c3RpcyBzcC4gUENDIDY4MDMgZ3Jvd3RoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz03MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHN1cGVyY29pbGluZyByZWd1bGF0ZXMgdGhlIHN0cmVzcy1pbmR1Y2libGUgZXhwcmVzc2lvbiBvZiBnZW5lcyBpbiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9zeXN0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTkwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVvbWUgYW5hbHlzaXMgb2Ygc2FsdCBzdHJlc3MgcmVzcG9uc2UgaW4gdGhlIGN5YW5vYmFjdGVyaXVtIFN5bmVjaG9jeXN0aXMgc3AuIHN0cmFpbiBQQ0MgNjgwMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjczMy00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzYvNDI1LzE3NDYzMDYyL2RzdjAyNC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHVzZSBvZiBtaWNyb29yZ2FuaXNtcyBhcyBjZWxsIGZhY3RvcmllcyBmcmVxdWVudGx5IHJlcXVpcmVzIGV4dGVuc2l2ZSBtb2xlY3VsYXIgbWFuaXB1bGF0aW9uLiBUaGVyZWZvcmUsIHRoZSBpZGVudGlmaWNhdGlvbiBvZiBnZW5vbWljIG5ldXRyYWwgc2kiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI0NjI0NTgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkOcP9RrRansMZt8nI34h+oPvRWKA=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f425%2f2462458" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f425%2f2462458" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198743">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198743">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198747">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198747">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198753">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198753">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198811">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198811">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198828">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198828">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198835">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198835">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198837">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198837">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198839">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198839">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/413/2462440">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/439/2462479">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Improving a <em>Synechocystis</em>-based photoautotrophic chassis through systematic genome mapping and validation of neutral sites
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Filipe Pinto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">i3S—Instituto de Investigação e Inovação em Saúde</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">IBMC—Instituto de Biologia Molecular e Celular</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-180</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculdade de Ciências, Departamento de Biologia</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-171</div>, <div class="country">Portugal</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Filipe+Pinto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pinto F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pinto F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Filipe Pinto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Catarina C. Pacheco</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">i3S—Instituto de Investigação e Inovação em Saúde</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">IBMC—Instituto de Biologia Molecular e Celular</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-180</div>, <div class="country">Portugal</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Catarina+C.+Pacheco">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pacheco C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pacheco C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Catarina C. Pacheco</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paulo Oliveira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">i3S—Instituto de Investigação e Inovação em Saúde</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">IBMC—Instituto de Biologia Molecular e Celular</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-180</div>, <div class="country">Portugal</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paulo+Oliveira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oliveira P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oliveira P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Paulo Oliveira</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arnau Montagud</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Instituto Universitario de Matemática Pura y Aplicada</div>, <div class="insititution">Universitat Politècnica de València</div>, <div class="addr-line">Valencia 46022</div>, <div class="country">Spain</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>Present Address: INSERM U900, Institut Curie, Paris, 75248, France.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arnau+Montagud">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Montagud A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Montagud A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arnau Montagud</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew Landels</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">ChELSI Institute, Department of Chemical and Biological Engineering</div>, <div class="insititution">University of Sheffield</div>, <div class="addr-line">Sheffield S10 2TN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+Landels">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Landels A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Landels A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew Landels</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Narciso Couto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">ChELSI Institute, Department of Chemical and Biological Engineering</div>, <div class="insititution">University of Sheffield</div>, <div class="addr-line">Sheffield S10 2TN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Narciso+Couto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Couto N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Couto N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Narciso Couto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Phillip C. Wright</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">ChELSI Institute, Department of Chemical and Biological Engineering</div>, <div class="insititution">University of Sheffield</div>, <div class="addr-line">Sheffield S10 2TN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Phillip+C.+Wright">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wright P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wright P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Phillip C. Wright</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Javier F. Urchuegu&#237;a</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Instituto Universitario de Matemática Pura y Aplicada</div>, <div class="insititution">Universitat Politècnica de València</div>, <div class="addr-line">Valencia 46022</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Javier+F.+Urchuegu%c3%ada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Urchuegu&#237;a J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Urchuegu&#237;a J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Javier F. Urchueguía</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paula Tamagnini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">i3S—Instituto de Investigação e Inovação em Saúde</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">IBMC—Instituto de Biologia Molecular e Celular</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-180</div>, <div class="country">Portugal</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Faculdade de Ciências, Departamento de Biologia</div>, <div class="insititution">Universidade do Porto</div>, <div class="addr-line">Porto 4150-171</div>, <div class="country">Portugal</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +351 226-074-900. Fax. +351 226-099-157. E-mail: <a href="mailto:pmtamagn@ibmc.up.pt" target="_blank">pmtamagn@ibmc.up.pt</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paula+Tamagnini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tamagnini P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tamagnini P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Paula Tamagnini</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present Address: INSERM U900, Institut Curie, Paris, 75248, France.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 425–437, <a href='https://doi.org/10.1093/dnares/dsv024'>https://doi.org/10.1093/dnares/dsv024</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 August 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">15 September 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462458" href="/dnaresearch/article-pdf/22/6/425/17463062/dsv024.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Filipe Pinto, Catarina C. Pacheco, Paulo Oliveira, Arnau Montagud, Andrew Landels, Narciso Couto, Phillip C. Wright, Javier F. Urchueguía, Paula Tamagnini;  Improving a <em>Synechocystis</em>-based photoautotrophic chassis through systematic genome mapping and validation of neutral sites, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 425–437, <a href='https://doi.org/10.1093/dnares/dsv024'>https://doi.org/10.1093/dnares/dsv024</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462458?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Improving%20a%20%26lt%3Bem%26gt%3BSynechocystis%26lt%3B%2Fem%26gt%3B-based%20photoautotrophic%20chassis%20through%20systematic%20genome%20mapping%20and%20validation%20of%20neutral%20sites&amp;publicationDate=2015-10-21&amp;volumeNum=22&amp;issueNum=6&amp;author=Pinto%2C%20Filipe%3B%20Pacheco%2C%20Catarina%20C.&amp;startPage=425&amp;endPage=437&amp;contentId=10.1093%2Fdnares%2Fdsv024&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f425%2f2462458" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198743 id="87198743" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The use of microorganisms as cell factories frequently requires extensive molecular manipulation. Therefore, the identification of genomic neutral sites for the stable integration of ectopic DNA is required to ensure a successful outcome. Here we describe the genome mapping and validation of five neutral sites in the chromosome of <em>Synechocystis</em> sp. PCC 6803, foreseeing the use of this cyanobacterium as a photoautotrophic chassis. To evaluate the neutrality of these loci, insertion/deletion mutants were produced, and to assess their functionality, a synthetic green fluorescent reporter module was introduced. The constructed integrative vectors include a BioBrick-compatible multiple cloning site insulated by transcription terminators, constituting robust cloning interfaces for synthetic biology approaches. Moreover, <em>Synechocystis</em> mutants (chassis) ready to receive purpose-built synthetic modules/circuits are also available. This work presents a systematic approach to map and validate chromosomal neutral sites in cyanobacteria, and that can be extended to other organisms.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;neutral sites&quot;">neutral sites</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;photoautotrophic chassis&quot;">photoautotrophic chassis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Synechocystis"><em>Synechocystis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;synthetic biology&quot;">synthetic biology</a></div>                            <h2 scrollto-destination=87198747 id="87198747" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microorganisms have been widely used in biotechnological applications to produce added-value products.<sup><a reveal-id="DSV024C1" data-open="DSV024C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The Gram-negative bacterium <em>Escherichia coli</em> is the most commonly used, but others such as the Gram-positive bacterium <em>Bacillus subtilis</em> and the eukaryotic yeast <em>Saccharomyces cerevisiae</em> have also been successfully exploited.<sup><a reveal-id="DSV024C2" data-open="DSV024C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV024C3" data-open="DSV024C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Heterotrophic organisms require supplementation of the growth medium with external carbon sources increasing production costs; therefore, autotrophic organisms emerge as a valid alternative.<sup><a reveal-id="DSV024C4" data-open="DSV024C4" class="link link-ref link-reveal xref-bibr">4</a></sup> In this context, cyanobacteria are promising ‘low-cost’ cell factories since they can use CO<sub>2</sub> as carbon source, water as reducing power, light as energy source and some strains are even able to fix atmospheric N<sub>2</sub> (nitrogen source). These organisms are found in almost any ecological niche on Earth,<sup><a reveal-id="DSV024C5" data-open="DSV024C5" class="link link-ref link-reveal xref-bibr">5</a></sup> which reflects their high degree of metabolic plasticity. Among cyanobacteria, the unicellular non-N<sub>2</sub>-fixing <em>Synechocystis</em> sp. PCC 6803 (henceforth referred to as <em>Synechocystis</em>) is the most studied strain, and the large amount of genomic, proteomic and physiological information generated over the years allowed the construction of genome-scale models, enabling the prediction of the system behavior.<sup><a reveal-id="DSV024C6" data-open="DSV024C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Moreover, it was the first photosynthetic organism to have its genome sequenced,<sup><a reveal-id="DSV024C7" data-open="DSV024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> comprising a 3.6 Mb chromosome and 7 plasmids (<a class="link link-uri" href="http://genome.microbedb.jp/cyanobase/Synechocystis">http://genome.microbedb.jp/cyanobase/Synechocystis</a>), it is naturally transformable and a number of molecular tools are available for its genetic manipulation.<sup><a reveal-id="DSV024C4" data-open="DSV024C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV024C8" data-open="DSV024C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSV024C9" data-open="DSV024C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Altogether, these characteristics allow for the exploitation of <em>Synechocystis</em> as a photoautotrophic biotechnological platform. In contrast to <em>E. coli</em>, for which several replicative plasmids from different compatibility groups are available, these tools are limited for <em>Synechocystis</em>.<sup><a reveal-id="DSV024C8  DSV024C10" data-open="DSV024C8  DSV024C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> Instead, integrative vectors are usually used to implement new functionalities, integrating ectopic DNA into the genome through homologous recombination.<sup><a reveal-id="DSV024C11" data-open="DSV024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> To date, a few integration sites in <em>Synechocystis</em>' genome have been used,<sup><a reveal-id="DSV024C11  DSV024C15" data-open="DSV024C11  DSV024C15" class="link link-ref link-reveal xref-bibr">11–15</a></sup> although most of them have not been extensively characterized as <em>neutral sites</em>—loci that can be disrupted without affecting cellular viability or cause any distinguishable phenotype.<sup><a reveal-id="DSV024C16" data-open="DSV024C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Therefore, a comprehensive and systematic approach to identify and characterize genomic neutral sites would greatly improve the ability to introduce new functionalities into <em>Synechocystis</em>.</p><p>In this work, a set of neutral sites in the chromosome of <em>Synechocystis</em> has been identified and characterized. It presents a systematic approach to map and validate neutral sites in the genome of an organism, foreseeing its use as a photoautotrophic chassis for industrial applications. Moreover, the new integrative vectors have been designed to be compatible with the BioBrick™ RFC[10] standard (<a class="link link-uri" href="http://www.biobricks.org/">http://www.biobricks.org/</a>), which will make easier to use <em>Synechocystis</em> in synthetic biology approaches. The availability of several integration loci, spread throughout the cyanobacterium chromosome, will allow the implementation of complex synthetic circuits into this platform. Furthermore, the strategy reported here can be easily extended to other organisms/chassis.</p>                            <h2 scrollto-destination=87198753 id="87198753" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87198755 id="87198755" class="section-title" data-legacy-id=s2a>2.1. Organisms and maintenance culture conditions</h3>
<p>Wild type and mutants of the cyanobacterium <em>Synechocystis</em> sp. PCC 6803 substrain Kazusa<sup><a reveal-id="DSV024C17" data-open="DSV024C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV024C18" data-open="DSV024C18" class="link link-ref link-reveal xref-bibr">18</a></sup> were maintained in BG11 medium<sup><a reveal-id="DSV024C19" data-open="DSV024C19" class="link link-ref link-reveal xref-bibr">19</a></sup> at 25°C, under a 16 h light/8 h dark regimen. Light intensity was 20 µmol photons m<sup>−2</sup> s<sup>−1</sup> in all experiments. For solid medium, BG11 was supplemented with 1.5% (wt/vol) noble agar (Difco), 0.3% (wt/vol) sodium thiosulfate and 10 mM TES-KOH buffer, pH 8.2. For the selection and maintenance of mutants, BG11 medium was supplemented with kanamycin (Km, 10–600 µg ml<sup>−1</sup>). All the characterization experiments were performed in the absence of selective pressure. For cloning purposes, <em>E. coli</em> strain DH5α (Stratagene) was used. <em>Escherichia coli</em> cells were grown at 37°C on LB medium,<sup><a reveal-id="DSV024C20" data-open="DSV024C20" class="link link-ref link-reveal xref-bibr">20</a></sup> supplemented with ampicillin (100 µg ml<sup>−1</sup>) or kanamycin (50 µg ml<sup>−1</sup>).</p>                            <h3 scrollto-destination=87198760 id="87198760" class="section-title" data-legacy-id=s2b>2.2. Bioinformatics</h3>
<p>The list of the predicted/annotated ORFs of <em>Synechocystis</em> was retrieved from CyanoBase (Distribution of Sequence and Annotated Data Files, <a class="link link-uri" href="http://ftp://ftp.kazusa.or.jp/pub/CyanoBase/Synechocystis">ftp://ftp.kazusa.or.jp/pub/CyanoBase/Synechocystis</a>, accessed: 08 May 2009). From the 3,264 ORFs listed, only those putatively encoding unknown or hypothetical proteins were selected. From these, the final selection was based on the following criteria: (i) length of the putatively encoded proteins ≤301 amino acids, (ii) encoded proteins with no transmembrane domains predicted by the TMHMM Server v. 2.0 (<a class="link link-uri" href="http://www.cbs.dtu.dk/services/TMHMM/">http://www.cbs.dtu.dk/services/TMHMM/</a>), (iii) no interaction with other proteins as assessed by the two-hybrid system (CyanoBase data) and (iv) no relevant similarities found at the protein sequence level, when comparing with other sequences using BLASTP (<a class="link link-uri" href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">http://blast.ncbi.nlm.nih.gov/Blast.cgi</a>). The genomic context of the ORFs was also taken into consideration, and sites were disregarded when in the vicinity of genes with assigned putative functions (see Fig. <a reveal-id="DSV024F1" data-open="DSV024F1" class="link link-reveal link-table xref-fig">1</a>). To assess transcription of the selected ORFs, primers were designed automatically from the genomic sequence<sup><a reveal-id="DSV024C7" data-open="DSV024C7" class="link link-ref link-reveal xref-bibr">7</a></sup> using the Beacon Designer 6 software (PREMIER Biosoft International). For other purposes, primers were designed manually and analysed using the Integrated DNA Technologies web resource OligoAnalyzer v3.1 (<a class="link link-uri" href="http://eu.idtdna.com/analyzer/Applications/OligoAnalyzer/">http://eu.idtdna.com/analyzer/Applications/OligoAnalyzer/</a>). </p>                        <a id="87198764" scrollto-destination="87198764"></a>
<div id="DSV024F1" data-id="DSV024F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02401.png?Expires=2147483647&Signature=P9q6S8DxvrpWl5CJsncmR8wKZZkF6NIYjnQJCuGLeM2OLrc1CnFuWBc4fv4i9S-a99OtmcCCoClKSQO~HBNDt~uo6efrMyLh7miyxg-U9JkrYx6Xfc3qXrp0cietKRkNeFQMGQ4OaAV0ZWDyU7NnKKOhzbSDjJlsOX45~wk-W~qqp69lmD2CZGmcylFLxMENrVcfEGJp~vkUKFVgLLWErtBGTfosCLObA2BfTdYBNH-Wn1PqNpcEdwIPXDAd4~-p6iAnQ5iHc5ynFZwmpeDyPkhCxnINeQBHjzA~Krqlw7xNLD1seguDxUKZ399wg8bDL6GFRNypf2YJqRfHacgbVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal location and genomic context of the Synechocystis' putative neutral sites. (A) Green, sites chosen for mutants construction, Red, sites that fulfil the neutral sites initial selection criteria, but that were subsequently disregarded after additional analyses. (B) Genomic context of each putative neutral site (5,000 bp). Blue, ORFs corresponding to the neutral sites, Red, ORFs encoding proteins with assigned functions; tp, putative transposase; ‘, partial ORFs." path-from-xml="dsv02401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198764/dsv02401.tiff" path-from-xml="dsv02401.tiff" target="_blank">View large</a><a section="87198764" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02401.png?Expires=2147483647&Signature=iaSxxTV0z1p1UtQYpEPa2Ppr0auhZ8Dl-emmt24w2P7RHcr3heSBCfJtnVuXC48O1fYRCX-xaKam4IAWV9T58d~GBUEMsTRdo9Wbx2xeKFN9IQcCLUenka8YRt2AHd~2sJbujAsI4OmAlA~7ruzUaHF~LGY9MDlp82cX17u-YCPe~GpExdBddKcwJzgo9WuKWVeMUkh--1OVoiqXOH9xQRYiiIqEv8Z23STyWfHjtB65ZgXmLXrh9kBOqtb3rROG7~1TQmnxJx34N3zpnaXHFCVp1DTEs2tgVpI4l2lpDBe0YYF0l-ouivmNPvv~4eubg3qT7egx-9-EkeF-fRzMew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198764&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal location and genomic context of the <em>Synechocystis</em>' putative neutral sites. (A) Green, sites chosen for mutants construction, Red, sites that fulfil the neutral sites initial selection criteria, but that were subsequently disregarded after additional analyses. (B) Genomic context of each putative neutral site (5,000 bp). Blue, ORFs corresponding to the neutral sites, Red, ORFs encoding proteins with assigned functions; tp, putative transposase; ‘, partial ORFs.</p></div></div><div content-id="DSV024F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02401.png?Expires=2147483647&Signature=P9q6S8DxvrpWl5CJsncmR8wKZZkF6NIYjnQJCuGLeM2OLrc1CnFuWBc4fv4i9S-a99OtmcCCoClKSQO~HBNDt~uo6efrMyLh7miyxg-U9JkrYx6Xfc3qXrp0cietKRkNeFQMGQ4OaAV0ZWDyU7NnKKOhzbSDjJlsOX45~wk-W~qqp69lmD2CZGmcylFLxMENrVcfEGJp~vkUKFVgLLWErtBGTfosCLObA2BfTdYBNH-Wn1PqNpcEdwIPXDAd4~-p6iAnQ5iHc5ynFZwmpeDyPkhCxnINeQBHjzA~Krqlw7xNLD1seguDxUKZ399wg8bDL6GFRNypf2YJqRfHacgbVA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal location and genomic context of the Synechocystis' putative neutral sites. (A) Green, sites chosen for mutants construction, Red, sites that fulfil the neutral sites initial selection criteria, but that were subsequently disregarded after additional analyses. (B) Genomic context of each putative neutral site (5,000 bp). Blue, ORFs corresponding to the neutral sites, Red, ORFs encoding proteins with assigned functions; tp, putative transposase; ‘, partial ORFs." path-from-xml="dsv02401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198764/dsv02401.tiff" path-from-xml="dsv02401.tiff" target="_blank">View large</a><a section="87198764" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02401.png?Expires=2147483647&Signature=iaSxxTV0z1p1UtQYpEPa2Ppr0auhZ8Dl-emmt24w2P7RHcr3heSBCfJtnVuXC48O1fYRCX-xaKam4IAWV9T58d~GBUEMsTRdo9Wbx2xeKFN9IQcCLUenka8YRt2AHd~2sJbujAsI4OmAlA~7ruzUaHF~LGY9MDlp82cX17u-YCPe~GpExdBddKcwJzgo9WuKWVeMUkh--1OVoiqXOH9xQRYiiIqEv8Z23STyWfHjtB65ZgXmLXrh9kBOqtb3rROG7~1TQmnxJx34N3zpnaXHFCVp1DTEs2tgVpI4l2lpDBe0YYF0l-ouivmNPvv~4eubg3qT7egx-9-EkeF-fRzMew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198764&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal location and genomic context of the <em>Synechocystis</em>' putative neutral sites. (A) Green, sites chosen for mutants construction, Red, sites that fulfil the neutral sites initial selection criteria, but that were subsequently disregarded after additional analyses. (B) Genomic context of each putative neutral site (5,000 bp). Blue, ORFs corresponding to the neutral sites, Red, ORFs encoding proteins with assigned functions; tp, putative transposase; ‘, partial ORFs.</p></div></div>                            <h3 scrollto-destination=87198765 id="87198765" class="section-title" data-legacy-id=s2c>2.3. DNA and RNA extraction</h3>
<p>Cyanobacterial genomic DNA (gDNA) extraction was carried out according to the procedure described previously.<sup><a reveal-id="DSV024C21" data-open="DSV024C21" class="link link-ref link-reveal xref-bibr">21</a></sup> For RNA extraction, <em>Synechocystis</em> cells were collected by centrifugation (10 min at 3,850 <em>g</em>) and frozen at −80°C. RNA was extracted using the TRIzol<sup>®</sup> reagent (Invitrogen) according to the method described previously.<sup><a reveal-id="DSV024C22" data-open="DSV024C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=87198770 id="87198770" class="section-title" data-legacy-id=s2d>2.4. RT-PCR for target ORFs transcription analysis</h3>
<p>To assess target ORFs transcription, RNA was extracted from <em>Synechocystis</em> cells cultivated to an OD<sub>730</sub> of 0.8–0.9, under continuous light at 30°C. Further transcriptional studies were performed with RNA extracted from samples collected in three different growth phases (OD<sub>730</sub> of ∼0.4, 2 and 9, see Fig. <a reveal-id="DSV024F2" data-open="DSV024F2" class="link link-reveal link-table xref-fig">2</a>), from three independent cultures grown under the same conditions. RT-PCRs were carried out as described elsewhere.<sup><a reveal-id="DSV024C23" data-open="DSV024C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Briefly, cDNA was synthesized in duplicate from 1 µg of total RNA using the iScript™ Select cDNA Synthesis Kit (Bio-Rad) and the random primers supplied, following the manufacturer's instructions. The two RT reactions from each sample were pooled together and each PCR reaction contained 1.5 µl of cDNA, 0.1 µM of each primer (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), 200 µM dNTPs mix, 1× PCR reaction buffer and 0.025 U of illustra™ <em>Taq</em> DNA polymerase (GE Healthcare). The PCR profile was as follows: 2 min at 94°C followed by 40 cycles of 30 s at 94°C, 30 s at 50°C and 30 s at 72°C, concluding with a 7-min extension step at 72°C. RNA contamination by gDNA prior to cDNA synthesis was assessed by PCR using primers specific for the <em>Synechocystis</em> 16S rDNA, and the RT reaction efficiency was assessed using the same primers pair. All the RT-PCRs included a negative control (omission of template) and a positive control (<em>Synechocystis</em> gDNA). </p>                        <a id="87198773" scrollto-destination="87198773"></a>
<div id="DSV024F2" data-id="DSV024F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02402.png?Expires=2147483647&Signature=CiJQD-I4lagSOb7Zc1E3pLUB0njnPcGHnoxold3dcoQDpLOjoBtS1g4JQomIMt1~84e-WISmM1DM4f5M-Z1TuoWu9Taa8jXKvfsY1wjLQ46AD9HBc~v~ONpBJ-nGtc-vxiW4-lEN~lv~YGvFHfWlrk2NGukP3SJt-q9MNbXLRCilrd2c9qNi2peZ86e4am-a~A8mhuBZQRIOmZNvZAgwMKjlO1V5o0UknjDnEmYuahZMW07TdVF8wjRHXSEvWJsxEV66j3Jpsnu-dyWFDWIhsxLeQ~6L1Fxz6r~H-etnd4DjLqCp6RLw5REepSdXJknqtXSUUCHu6otxB0hWxPw4hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synechocystis growth curves, sample collection and transcription analysis of eight selected loci (N5, N6, N7, N8, N10, N14, N15 and N16). (A) Growth curves of three independent cultures of Synechocystis with sample collection points for RNA extraction indicated by arrows (1—OD730 ≈ 0.4; 2—OD730 ≈ 1.9; 3—OD730 ≈ 9.0). (B) RT-PCR transcription analysis using total RNA extracted from cells collected at the time points indicated in A. These results are representative of three biological replicates and technical duplicates. −, negative control (absence of template); +, positive control (genomic DNA); and M, GeneRuler DNA ladder (Thermo Fisher Scientific Inc.)." path-from-xml="dsv02402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198773/dsv02402.tiff" path-from-xml="dsv02402.tiff" target="_blank">View large</a><a section="87198773" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02402.png?Expires=2147483647&Signature=rPiFxNhXOtohz4U8FyhHwhXO1N6VRD4XBF1Mf-nAO2Y7MIZ7CB4qjiYYqa4Jtqfc8I-4pVSiMkxTS1vdpgBp967lnFa7rSw8m5PjgLynfignt~g9iPLj6QqMIjFsbYNpEodsp7VwILR3Nf-f8cbhFhTMyfN-VwEoXg9dfbmYZS9FfHrE6Gmjb8Tj1XHdzj9eKULw3P9G5cMFCbX3ooBtYgGEgUD~J2f~Qx-tpb22bPJu6aeC8T2IV5Os5sgKG6pBQV9674rH5g47kCV0j8oKsR6Fh~HxDSVzGFBQGLmoUV2TecGXrtL6aOXjnh7LNbNS8xQPP-~pQQ82-vdq~j1Odw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198773&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Synechocystis</em> growth curves, sample collection and transcription analysis of eight selected loci (N5, N6, N7, N8, N10, N14, N15 and N16). (A) Growth curves of three independent cultures of <em>Synechocystis</em> with sample collection points for RNA extraction indicated by arrows (1—OD<sub>730</sub> ≈ 0.4; 2—OD<sub>730</sub> ≈ 1.9; 3—OD<sub>730</sub> ≈ 9.0). (B) RT-PCR transcription analysis using total RNA extracted from cells collected at the time points indicated in A. These results are representative of three biological replicates and technical duplicates. −, negative control (absence of template); +, positive control (genomic DNA); and M, GeneRuler DNA ladder (Thermo Fisher Scientific Inc.).</p></div></div><div content-id="DSV024F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02402.png?Expires=2147483647&Signature=CiJQD-I4lagSOb7Zc1E3pLUB0njnPcGHnoxold3dcoQDpLOjoBtS1g4JQomIMt1~84e-WISmM1DM4f5M-Z1TuoWu9Taa8jXKvfsY1wjLQ46AD9HBc~v~ONpBJ-nGtc-vxiW4-lEN~lv~YGvFHfWlrk2NGukP3SJt-q9MNbXLRCilrd2c9qNi2peZ86e4am-a~A8mhuBZQRIOmZNvZAgwMKjlO1V5o0UknjDnEmYuahZMW07TdVF8wjRHXSEvWJsxEV66j3Jpsnu-dyWFDWIhsxLeQ~6L1Fxz6r~H-etnd4DjLqCp6RLw5REepSdXJknqtXSUUCHu6otxB0hWxPw4hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synechocystis growth curves, sample collection and transcription analysis of eight selected loci (N5, N6, N7, N8, N10, N14, N15 and N16). (A) Growth curves of three independent cultures of Synechocystis with sample collection points for RNA extraction indicated by arrows (1—OD730 ≈ 0.4; 2—OD730 ≈ 1.9; 3—OD730 ≈ 9.0). (B) RT-PCR transcription analysis using total RNA extracted from cells collected at the time points indicated in A. These results are representative of three biological replicates and technical duplicates. −, negative control (absence of template); +, positive control (genomic DNA); and M, GeneRuler DNA ladder (Thermo Fisher Scientific Inc.)." path-from-xml="dsv02402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198773/dsv02402.tiff" path-from-xml="dsv02402.tiff" target="_blank">View large</a><a section="87198773" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02402.png?Expires=2147483647&Signature=rPiFxNhXOtohz4U8FyhHwhXO1N6VRD4XBF1Mf-nAO2Y7MIZ7CB4qjiYYqa4Jtqfc8I-4pVSiMkxTS1vdpgBp967lnFa7rSw8m5PjgLynfignt~g9iPLj6QqMIjFsbYNpEodsp7VwILR3Nf-f8cbhFhTMyfN-VwEoXg9dfbmYZS9FfHrE6Gmjb8Tj1XHdzj9eKULw3P9G5cMFCbX3ooBtYgGEgUD~J2f~Qx-tpb22bPJu6aeC8T2IV5Os5sgKG6pBQV9674rH5g47kCV0j8oKsR6Fh~HxDSVzGFBQGLmoUV2TecGXrtL6aOXjnh7LNbNS8xQPP-~pQQ82-vdq~j1Odw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198773&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Synechocystis</em> growth curves, sample collection and transcription analysis of eight selected loci (N5, N6, N7, N8, N10, N14, N15 and N16). (A) Growth curves of three independent cultures of <em>Synechocystis</em> with sample collection points for RNA extraction indicated by arrows (1—OD<sub>730</sub> ≈ 0.4; 2—OD<sub>730</sub> ≈ 1.9; 3—OD<sub>730</sub> ≈ 9.0). (B) RT-PCR transcription analysis using total RNA extracted from cells collected at the time points indicated in A. These results are representative of three biological replicates and technical duplicates. −, negative control (absence of template); +, positive control (genomic DNA); and M, GeneRuler DNA ladder (Thermo Fisher Scientific Inc.).</p></div></div>                            <h3 scrollto-destination=87198775 id="87198775" class="section-title" data-legacy-id=s2e>2.5. Homologous regions amplification and fusion by PCR</h3>
<p>PCRs to amplify the regions flanking the target loci (see below) contained 7 ng of gDNA, 0.2 µM of each primer, 250 µM dNTPs mix, 1× <em>Pfu</em> reaction buffer with MgSO<sub>4</sub> and 0.025 U of <em>Pfu</em> DNA polymerase (Thermo Fisher Scientific Inc.). The PCR profile was as follows: 1 min at 94°C followed by 35 cycles of 30 s at 94°C, 45 s at 48°C and 90 s at 72°C, concluding with a 7-min extension step at 72°C. Agarose gel electrophoresis was performed by standard protocols using 1× TAE buffer;<sup><a reveal-id="DSV024C20" data-open="DSV024C20" class="link link-ref link-reveal xref-bibr">20</a></sup> DNA fragments were purified using the GFX™ PCR DNA and Gel Band Purification Kit (GE Healthcare), according to the manufacturer's instructions. ‘Overlap-PCRs’ (see below) contained 80 ng of each purified DNA fragment, 0.125 µM of each primer, 250 µM dNTPs mix, 1× PCR reaction buffer and 0.025 U of illustra™ <em>Taq</em> DNA polymerase (GE Healthcare). The PCR profile was: 5 min at 95°C followed by 10 cycles of 30 s at 95°C, 45 s at 48°C and 90 s at 72°C, and 30 cycles of 30 s at 95°C, 45 s at 55°C and 90 s at 72°C, concluding with a 7-min extension step at 72°C. The oligonucleotide primers used to amplify the homologous regions are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=87198779 id="87198779" class="section-title" data-legacy-id=s2f>2.6. Plasmid construction</h3>
<p>The plasmids used to produce the <em>Synechocystis</em> mutants were constructed following a rational design (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). For this purpose, the chromosomal regions up and downstream of the target loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) were amplified by PCR using the <em>Pfu</em> polymerase and primers containing tags with restriction sites for cloning purposes (N<em>n</em>.5O/N<em>n</em>.5I and N<em>n</em>.3O/N<em>n</em>.3I, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The corresponding up and downstream fragments of each locus were fused by ‘overlap-PCR’, using the <em>Taq</em> polymerase (see above). The resulting products were cloned into the pGEM<sup>®</sup>-T easy vector (Promega), according to the manufacturer's instructions, originating the pSN<em>n</em>* plasmids series (<em>n</em> for ‘neutral site number’). The pGEM<sup>®</sup>-T easy vector restriction sites incompatible with the BioBrick RFC[10] standard were eliminated by digesting the vector with <em>Xho</em>I and <em>Sal</em>I and recircularizing the vector (originating the pSN<em>n</em>** series). The BioBrick RFC[10] standard incompatible restriction sites within the amplified chromosomal regions of pSN5** and pSN10** were eliminated using the QuikChange<sup>®</sup> Multi Site-Directed Mutagenesis Kit (Stratagene), according to the manufacturer's instructions, using the primers N5_SDM and N10_SDM, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), originating the pSN5*** and the pSN10*** plasmids. A BioBrick-compatible cloning interface was synthesized (Epoch Life Science Inc.), containing the <em>Mun</em>I, <em>Xba</em>I, <em>Not</em>I, <em>Spe</em>I and <em>Pst</em>I restriction sites flanked by two double transcription terminators (based on the registry parts BBa_B1006, BBa_B0062, BBa_B0053 and BBa_B0020). This interface was excised from the delivery vector (pBSK-FP300) using <em>Xma</em>I and <em>Age</em>I and subsequently cloned into the <em>Xma</em>I site of the pSN<em>n</em>** or pSN<em>n</em>*** plasmids, originating the pSN<em>n</em> plasmid series (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Afterwards, two selection cassettes, one containing the <em>nptII</em> gene (encoding a neomycin phosphotransferase that confers resistance to neomycin and kanamycin) and the <em>sacB</em> gene (encoding a levansucrase that confers sucrose sensitivity), and another containing only the <em>nptII</em> gene, were amplified by PCR from the plasmid pK18mobsacB,<sup><a reveal-id="DSV024C24" data-open="DSV024C24" class="link link-ref link-reveal xref-bibr">24</a></sup> using the primer pairs Km.KmScFwd/KmScRev and Km.KmScFwd/KmRev, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These selection cassettes were cloned into the pGEM-T easy vector, excised with <em>Xma</em>I and cloned into the <em>Xma</em>I site of the pSN<em>n</em> plasmid series. The final plasmid series containing only the Km-resistance cassette was named pSN<em>n</em>K, and the plasmid series containing the cassette with the two selective markers was named pSN<em>n</em>KS. Additionally, based on previous works,<sup><a reveal-id="DSV024C25" data-open="DSV024C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSV024C26" data-open="DSV024C26" class="link link-ref link-reveal xref-bibr">26</a></sup> the minimal promoter sequence required for <em>Synechocystis psbA2</em> basal transcription (−38 to +14 bp (base-pairs); +1 corresponding to the transcription start site) was synthesized (<em>P<sub>psbA2</sub></em><sub>*</sub>), meeting the BioBrick RFC[10] standard specifications (DNA 2.0 Inc.). This promoter region was cloned into the BioBrick vector pSB1A2 containing the part BBa_E0240 (composed by the RBS part BBa_B0032, the GFP-coding gene part BBa_E0040 and the double transcription terminator part BBa_B0015), according to the BioBrick standard cloning procedure. The BioBrick composite part BBa_E0240 (promoterless GFP generator) and the part with the synthetic constitutive promoter <em>P<sub>psbA2</sub></em><sub>*</sub> were digested with <em>Xba</em>I and <em>Spe</em>I and cloned into the <em>Xba</em>I site of both pSN<em>n</em> and pSN<em>n</em>K series, originating the pSN<em>n</em>K.gfp and pSN<em>n</em>K.Cgfp plasmid series (C for ‘constitutive promoter’). These plasmids were used to assess proper cloning interface transcriptional insulation (promoterless <em>gfp</em>) and functionality of the neutral sites (<em>gfp</em> under the control of the constitutive promoter). The vectors were named as follows: <em>P</em> for ‘plasmid’, S for ‘<em>Synechocystis</em>’, N for ‘neutral site’, <em>n</em> for the ‘number of the neutral site’ (5, 8, 10, 15 or 16), K for ‘kanamycin resistance cassette’, KS for the ‘double selection cassette (Km<sup>R</sup>/Suc<sup>S</sup>)’, C for ‘constitutive <em>P<sub>psbA2</sub></em><sub>*</sub> promoter’ and gfp for the ‘GFP generator BioBrick part BBa_E0240’. Plasmids DNA was isolated from overnight grown <em>E. coli</em> cultures, using the GenElute™ Plasmid Miniprep Kit (Sigma-Aldrich), according to the manufacturer's instructions. All the BioBrick parts can be found at the Registry of Standard Biological Parts website (<a class="link link-uri" href="http://parts.igem.org">http://parts.igem.org</a>), and the maps and sequences (FASTA format) of the final plasmids are available as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S2–S26</a></span>.</p>                            <h3 scrollto-destination=87198782 id="87198782" class="section-title" data-legacy-id=s2g>2.7. Generation of <em>Synechocystis</em> mutants</h3>
<p><em>Synechocystis</em> was transformed based on the procedure described previously.<sup><a reveal-id="DSV024C11" data-open="DSV024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Briefly, <em>Synechocystis</em> was grown in standard conditions to an OD<sub>730</sub> ≈ 0.5. Cells were harvested by centrifugation (10 min at 3,850 <em>g</em>) and resuspended in BG11 to a final OD<sub>730</sub> ≈ 2.5. The purified plasmids were incubated for 5 h with 500 µl of those cells (final plasmid DNA concentration of 20 µg ml<sup>−1</sup>), in light and at 25°C. Cells were then spread onto Immobilon™-NC membranes (0.45 µm pore size, 82 mm, Millipore) resting on solid BG11 plates, grown in standard growth conditions and transferred to selective plates after 24 h (Km, 10 µg ml<sup>−1</sup>). Transformants were observed after 1–2 weeks. For complete segregation, Km-resistant colonies were grown at increasing Km concentrations (25 and 50 µg ml<sup>−1</sup>) and finally transferred into liquid medium. The mutant full segregation was confirmed by PCR, using primers within and external to each site (N<em>n</em>.FI/N<em>n</em>.RI or N<em>n</em>.FO/KmRev, respectively; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), and by Southern blot with probes labeled with digoxigenin using the DIG DNA labeling kit (Roche Molecular Biochemicals) and following the manufacturer's instructions. The probes were obtained by labeling the PCR-amplified 3′ homologous regions for mutants in sites N5, N8, N10 and N16, and the 5′ homologous region for mutants in site N15 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). gDNA digestions were performed with <em>Mun</em>I, with the exception of N8 mutants for which the gDNA was digested with <em>Spe</em>I. When necessary antibiotic concentration was increased up to 600 µg ml<sup>−1</sup> to fully segregate the mutants.</p>                            <h3 scrollto-destination=87198788 id="87198788" class="section-title" data-legacy-id=s2h>2.8. Growth experiments</h3>
<p>For the growth experiments, 50 ml of BG11 medium was inoculated with <em>Synechocystis</em> wild type or SN<em>n</em>K mutants to OD<sub>730</sub> ≈ 0.3. The cultures were grown to OD<sub>730</sub> ≈ 1.0, under the experimental light regimen (continuous light or 12 h light/12 h dark cycles) and subsequently diluted to a final OD<sub>730</sub> ≈ 0.1 in fresh BG11. Ten millilitres of each diluted culture was transferred to sterile 25-ml borosilicate glass Erlenmeyer flasks, closed with cellulose stopper, and when necessary, glucose was added to a final concentration of 5 mM. The flasks were incubated at constant temperature (30°C) and the indicated light regimens. All experiments were performed in triplicate, and the OD<sub>730</sub> of the cultures was recorded daily for 10 consecutive days.</p>                            <h3 scrollto-destination=87198793 id="87198793" class="section-title" data-legacy-id=s2i>2.9. Statistical analysis of cyanobacteria-specific growth</h3>
<div class="block-child-p">The quantitative analysis of <em>Synechocystis</em> wild type and neutral sites mutant growth in various conditions was based on the optical density (OD) measurements (see above). For each bin, <em>i</em>, a specific growth parameter (<em>µ</em>), representing the exponent of exponential growth within the given time interval was calculated as:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>μ</mi><mi>i</mi></msub></mrow><mo xmlns="">=</mo><mstyle xmlns=""><mrow><mfrac><mrow><mi>log</mi><mo>⁡</mo><mo stretchy="false">[</mo><mrow><msub><mi>x</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mrow><mo>/</mo></mrow><mrow><msub><mi>x</mi><mi>i</mi></msub></mrow><mo stretchy="false">]</mo></mrow><mrow><mrow><msub><mi>t</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>−</mo><mrow><msub><mi>t</mi><mi>i</mi></msub></mrow></mrow></mfrac></mrow></mstyle></math></div></div></div> where <em>t<sub>i</sub></em> is time, in units of days running from Day 0 to <em>t<sub>i</sub></em><sub>(max)</sub> and <em>x<sub>i</sub></em> is the corresponding measured OD. As test duration was 10 days in all strains and stationary growth was reached at similar moments, the specific growth rate of all the exponential phase of each replica of each organism was considered to be a reliable parameter to assess growth patterns. ANOVA statistical analysis of the growth data was performed to compare the wild type with each mutant organism, using the Wolfram's Mathematica<sup>©</sup> software. This ANOVA was three-way where a model with all the effects and interactions up to the third order was tested. Values of up to second-order interactions can be studied in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>; third-order interactions were in accordance to these results.</div>                            <h3 scrollto-destination=87198797 id="87198797" class="section-title" data-legacy-id=s2j>2.10. RT-PCR and RT-qPCR for <em>gfp</em> transcription analysis</h3>
<p>For total RNA extraction, 400 ml of BG11 medium was inoculated to an OD<sub>730</sub> ≈ 0.2 with pre-cultures of <em>Synechocystis</em> wild type and mutants harboring the <em>gfp</em> gene (SN<em>n</em>K.gfp and SN<em>n</em>K.Cgfp mutants' series). Three independent biological replicates of each strain were grown to an OD<sub>730</sub> ≈ 1, at 28°C in glass gas washing bottles with aeration, under continuous light. Cells from 100 ml of culture were collected by centrifugation (10 min at 3,850 <em>g</em>); pellets were treated with RNAprotect Bacteria Reagent (Qiagen) and stored at −80°C. Total RNA was extracted as described previously<sup><a reveal-id="DSV024C27" data-open="DSV024C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and stored at −80°C until further analysis. RNA quality and contamination verifications, cDNA synthesis, RT-PCR and RT-qPCR experiments were performed as described<sup><a reveal-id="DSV024C27" data-open="DSV024C27" class="link link-ref link-reveal xref-bibr">27</a></sup> using primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The genes 16S, <em>petB</em> and <em>rnpB</em> were amplified using the primers described elsewhere<sup><a reveal-id="DSV024C27" data-open="DSV024C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and used as reference genes for data normalization. All RT-qPCR parameters in these experiments were in agreement with the MIQE (minimum information for publication of quantitative real-time PCR experiments) guidelines.<sup><a reveal-id="DSV024C28" data-open="DSV024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> RT-qPCR data from three biological replicates and three technical replicates were analysed using qbase<sup>+</sup> v2.6 (Biogazelle), and the statistical analysis was performed by means of a one-way ANOVA, using GraphPad Prism v6 (GraphPad software Inc.).</p>                            <h3 scrollto-destination=87198801 id="87198801" class="section-title" data-legacy-id=s2k>2.11. Western blot</h3>
<p>For the western blot, 20 ml of the same cultures used for <em>gfp</em> transcription analyses (by RT-PCR and RT-qPCR) was collected by centrifugation (10 min at 3,850 <em>g</em>) and stored at −80°C until further use. Cells were resuspended in lysis buffer (10 mM HEPES pH 8.0, 10 mM EDTA pH 8.0, 0.5% (vol/vol) Triton X-100, 2 mM DTT) supplemented with a cocktail of protease inhibitors (cOmplete Mini, EDTA free, Roche) and disrupted by sonication on ice with a Branson Sonifier 250 using 5 cycles of 15 s (50% dutty cycle, Output 3) intercalated with 1 min off duty. Cell debris was separated by two rounds of 10 min centrifugation at 10,000 <em>g</em> and 4°C. Protein extracts were stored at −20°C until further analysis. Protein quantification was performed using the BCA Protein Assay Kit (Thermo Fisher Scientific Inc.), and 15 µg of each extract was separated by electrophoresis on 12% (wt/vol) SDS-polyacrylamide gels. Protein extracts were visualized with colloidal Coomassie blue staining (Sigma) or transferred to a Hybond™-C extra nitrocellulose membrane (GE Healthcare). After the transfer, membranes were blocked for 3 h with 5% (wt/vol) milk powder in TTBS [Tris-buffered saline (TBS) with 0.05% (vol/vol) Tween 20] and then incubated overnight at 4°C with GFP-specific monoclonal mouse antibody (Roche) in fresh blocking buffer. After washing with TTBS, blots were incubated for 1 h with goat-anti-mouse IgG (Invitrogen) linked to horseradish peroxidase. Membranes were washed with TTBS before immunodetection, which was performed by chemiluminescence using ECL Western Blotting Analysis System detection reagents.</p>                            <h3 scrollto-destination=87198803 id="87198803" class="section-title" data-legacy-id=s2l>2.12. Confocal microscopy</h3>
<p>Ten millilitres of BG11 medium was inoculated with <em>Synechocystis</em> wild type or mutants harbouring the <em>gfp</em> gene (SN<em>n</em>K.gfp and SN<em>n</em>K.Cgfp mutants' series) to OD<sub>730</sub> ≈ 0.15. The cultures were grown to OD<sub>730</sub> ≈ 1.0, under continuous light at 30°C and subsequently diluted to a final OD<sub>730</sub> ≈ 0.1 in fresh BG11. Ten millilitres of each diluted culture was incubated under the same light and temperature conditions as above and grown until OD<sub>730</sub> ≈ 1.0. Two hundred microlitres of each culture was collected by centrifugation, resuspended in 20 µl of fresh BG11 and 40 µl of 1% (wt/vol) low-melting point agarose dissolved in BG11 medium was added. Ten microlitres of each mixture was immediately loaded on a glass slide and covered with a coverslip. The GFP emission (collected between 500 and 540 nm) was observed when the cells were exposed to a laser beam at 488 nm, using a Leica TCS SP5 II confocal microscope (Leica Microsystems). Cyanobacterial autofluorescence was collected between 640 and 680 nm after excitation at 633 nm. Images were analysed with the LAS AF Lite v3.3 software (Leica Microsystems). GFP fluorescence measurements were also performed for these samples as described below.</p>                            <h3 scrollto-destination=87198805 id="87198805" class="section-title" data-legacy-id=s2m>2.13. GFP fluorescence measurements</h3>
<p>Fifty millilitres of BG11 medium was inoculated with <em>Synechocystis</em> wild type or mutants harbouring the <em>gfp</em> gene (SN<em>n</em>K.gfp and SN<em>n</em>K.Cgfp mutants' series) to OD<sub>730</sub> ≈ 0.1. The cultures were grown to OD<sub>730</sub> ≈ 2.5, under continuous light at 30°C, and subsequently diluted to a final OD<sub>730</sub> ≈ 0.2 in fresh BG11. Ten millilitres of each diluted culture was transferred to three sterile 25-ml borosilicate glass flasks and closed with cellulose stoppers. The flasks were incubated under the same light and temperature conditions, and samples for total fluorescence measurements were collected after 2, 4, 7, 9 and 11 days after inoculation. For fluorescence measurements, three aliquots of 200 µl of each biological replicate were distributed in a Nunc™ MicroWell™ 96-Well Optical-Bottom Plates (Thermo Fisher Scientific Inc.), and GFP fluorescence and OD<sub>790</sub> were detected using the Synergy 2 Multi-Mode Microplate Reader and the Gen5™ software (BioTek Instruments, Inc.). For fluorescence detection, an excitation filter of 485/20 nm and an emission filter of 528/20 nm were used (sensibility set for 110 nm). OD<sub>790</sub> was measured using an absorbance filter of 790 nm. Wells with BG11 were included for background fluorescence measurement, and plate readings were carried out in duplicate. Absolute fluorescence values were normalized to the background values of the BG11 medium, and the wild type was used as fluorescence baseline. Mean GFP fluorescence per cell was normalized to the OD<sub>790</sub> and is expressed in arbitrary units (A.U.).</p>                            <h3 scrollto-destination=87198807 id="87198807" class="section-title" data-legacy-id=s2n>2.14. Isobaric tags for relative and absolute quantification</h3>
<p>For the proteomics analysis, 50 ml cultures of <em>Synechocystis</em> wild type (WT), the mutants constitutively expressing <em>gfp</em> (SN5K.Cgfp, SN8K.Cgfp, SN10K.Cgfp, SN15K.Cgfp and SN16K.Cgfp) and a mutant carrying a promoterless <em>gfp</em> gene at site N15 (SN15K.gfp) were grown in quadruplicate at 30°C under continuous light, until they reached an OD<sub>730</sub> ≈ 0.7. The culture volumes used for sample collection were normalized to the culture with lowest OD<sub>730</sub>, to guarantee biomass uniformity among samples. The cultures were then centrifuged for 10 min at 3,850 <em>g</em>, and cell pellets were preserved at −80°C prior to protein extraction. The four replicates of each strain were paired and combined together, resulting in 14 samples for analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The samples were run across two isobaric tags for relative and absolute quantifications (iTRAQs) 8-plex with the pooled WT samples being a common control in both experiments creating a point of reference between them.</p><p>For protein extraction, cells were washed with 150 mM sodium chloride and spun down for 10 min at 4,000 <em>g</em>, and the supernatant was discarded. Pellets were resuspended in 200 µl of lysis buffer [200 mM triethylammonium bicarbonate (TEAB), 5 mM dithiothreitol (DTT), plant protease inhibitor cocktail and 0.1% (vol/vol) nonidet P40 from Sigma] and transferred to a 2 ml LoBind microcentrifuge tubes (Eppendorf) containing ∼200 mg of 200 µm Zirconium beads. Samples were then pulsed on a bead-beater for 10 cycles of 60 s with a 2-min cooling step on ice between cycles. Samples were centrifuged at 13,000 <em>g</em> for 10 min at 4°C, and supernatants were transferred to new microfuge tubes. Pellets were further resuspended in 200 µl of lysis buffer, pulsed for a further 5 cycles on the bead-beater and centrifuged as before, and supernatants were combined to the previous ones. The combined supernatants were further clarified by centrifugation at 13,000 <em>g</em> for 10 min at 4°C to ensure that any remaining cell debris was removed from the extracts. The clarified extracts (∼400 µl) were incubated with 2 µl of benzonase nuclease (Novagene) for 2 min on ice. Protein content in each extract was precipitated with the addition of four volumes of cold acetone and was incubated overnight at −20°C. The precipitated proteins were then spun down, and the supernatants were discarded. The protein pellets were then resuspended in 200 mM TEAB, and protein concentration was estimated using the modified Lowry spectrophotometric method as described.<sup><a reveal-id="DSV024C29" data-open="DSV024C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The concentration of the protein extracts and the efficiency of the lysis method were evaluated by sodium dodecyl sulfate polyacrylamide gel.</p><p>From each sample, 100 µg of the protein extract was collected for enzymatic digestion, and the final volume was adjusted to 30 µl with 200 mM TEAB, pH 8.5. Proteins were reduced with 10 mM TCEP [tris(2-carboxyethyl)phosphine] for 1 h at 60°C. Cysteines were blocked using 20 mM methyl methanethiosulfonate (MMTS) for 10 min at room temperature. The reduced and blocked proteins were digested overnight at 37°C, using trypsin 1:25 (wt/wt) (Promega) in 200 mM TEAB, pH 8.5. iTRAQ, 8-plex reagents (ABSciex) were resuspended in 100 µl isopropanol and added to the appropriate digested proteins. The peptides and labels were incubated at room temperature for 2 h. After incubation, unreacted iTRAQ reagents were quenched by adding 0.1% (vol/vol) hydroxylamine and incubated for 15 min at room temperature. All labeled peptide fractions were combined and split into two separate tubes. These peptide samples were dried using a vacuum concentrator and stored at −20°C prior to off-line fractionation using a HyperCarb porous graphitic column (particle size: 3 µm, length: 50 mm, diameter: 2.1 mm, pore size: 5 µm, Thermo Scientific) on a UHPLC Ultimate 3000 RS (Dionex). Dried peptides were resuspended in 100 µl of Buffer A [0.1% (vol/vol) trifluoroacetic acid (TFA), 3% (vol/vol) HPLC-grade acetonitrile (ACN) in HPLC-grade water] and were eluted using a linear gradient of Buffer B [0.1% (vol/vol) TFA, 97% (vol/vol) HPLC-grade ACN in HPLC-grade water] ranging from 10 to 60% over 2 h. Peptide elution was monitored using a wavelength of 214 nm and with Chromeleon software, version 6.50 (Dionex). Fractions were collected at 2-min intervals across the gradient and dried using a vacuum concentrator. The dried fractions were stored at −20°C prior to mass spectrometry analysis. The second (reverse-phase) dimension of the chromatographic separation was performed on an Ultimate 3000 capillary HPLC system (Dionex) on-line connected to a quadrupole time-of-flight (Q-ToF) QSTAR XL mass spectrometer (ABSciex). Each HyperCarb fraction was resuspended in loading buffer [0.1% (vol/vol) TFA, 3% (vol/vol) HPLC-grade ACN in HPLC-grade water], and two injections were carried out. Chromatographic separation was performed on a 75 µm × 15 cm C18 analytical column (particle size: 3 µm, pore size: 100 Å, LC Packings, CA, USA) preceded by a C18 trap column (Dionex/LC Packings). A 90-min gradient elution was performed using Buffer A [0.1% (vol/vol) formic acid (FA) and 3% (vol/vol) HPLC-grade ACN in HPLC-grade water] and Buffer B [0.1% (vol/vol) FA and 97% (vol/vol) HPLC-grade ACN in HPLC-grade water], during which Buffer B increased from 3 to 35% at a flow rate of 300 nl min<sup>−1</sup>. In the mass spectrometer, MS survey scans were acquired at 1 Hz with a <em>m/z</em> (mass-to-charge) range of 350−1,600, and two dynamically selected ions with a +2 or +3 charge state were isolated and fragmented by collision-induced dissociation (CID). The collision energy was optimized to maximize the intensities of the reporter ions without losing sequence information. Each MS/MS (tandem mass spectrum) was accumulated for 3 s. Data were acquired and processed using Analyst QS version 2.0 (ABSciex). Protein identification and quantification were carried out using Phenyx Software version 2.6 (Genebio). MS/MS spectra were searched against the UniProt flat-text database (accessed April 2015), containing 3,576 protein sequences of <em>Synechocystis</em>, with the protein sequences of GFP and NPTII added to it. Mass tolerance was set to 0.3 Da and MS/MS tolerance was set to 0.1 Da. The peptide modifications were the same for each search (fixed modifications: methylthio for cysteines, iTRAQ 8-plex for lysine and peptide N-terminus and variable modifications: oxidation of methionine). The false discovery rate (FDR) was calculated by simultaneously using the true and the reverse databases generated with Phenyx,<sup><a reveal-id="DSV024C30" data-open="DSV024C30" class="link link-ref link-reveal xref-bibr">30</a></sup> as previously described.<sup><a reveal-id="DSV024C31" data-open="DSV024C31" class="link link-ref link-reveal xref-bibr">31</a></sup> An FDR of 1% with at least two unique peptides was imposed on the data, and peptide spectral matches (PSMs) falling above this value were excluded. The peptide lists generated by Phenyx were analysed as described elsewhere.<sup><a reveal-id="DSV024C32" data-open="DSV024C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Each iTRAQ was analysed separately to this point and then merged to generate background proteomic effects across the samples. The peptide quantifications were merged as follows: the lists were checked to find a list of all proteins successfully identified with at least two unique peptides and at least three PSMs in both iTRAQs. The iTRAQ quantification values of these proteins were normalized separately using median correction, to account for internal label variation, before a fixed ‘<em>α</em>’ value (0.5 with this data) was added to all quantifications, to remove the effects of missing labels and noise in the data on the ratio-driven quantification used with iTRAQ. The quantifications were ratio-transformed relative to the mean of the WT controls in each sample and then scaled to each other so that the mean of the WT controls in each sample was equal. A cluster analysis in the form of both a dendrogram (Ward linkage) and principal components analysis (PCA) was then performed to look for background differences between the samples. Significant differences were identified using our in-house protein pipeline, which utilizes a series of paired <em>t</em>-tests with Bonferroni correction, as described elsewhere.<sup><a reveal-id="DSV024C33" data-open="DSV024C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The free online tool KEGG color-mapper (<a class="link link-uri" href="http://www.genome.jp/kegg/tool/map_pathway2.html">http://www.genome.jp/kegg/tool/map_pathway2.html</a>, accessed: 27 April 2015) was used to highlight metabolic pathways that were systematically significantly up- or down-regulated.</p>                            <h2 scrollto-destination=87198811 id="87198811" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87198812 id="87198812" class="section-title" data-legacy-id=s3a>3.1. Selection of putative neutral sites</h3>
<p>To identify putative neutral sites in the chromosome of <em>Synechocystis</em>, the list of annotated open reading frames (ORFs) with the respective encoded proteins was retrieved from CyanoBase (<a class="link link-uri" href="http://genome.kazusa.or.jp/cyanobase">http://genome.kazusa.or.jp/cyanobase</a>). From the 3,264 ORFs listed, 1,532 of the putatively encoded proteins were annotated as being unknown or hypothetical, and a subsequent bioinformatics analysis was performed to identify those matching the following criteria: (i) length of the putatively encoded proteins ≤301 amino acids, (ii) encoded proteins with no predicted transmembrane domains, (iii) no reported interaction with other proteins and (iv) no relevant similarities found at the protein sequence level. This analysis led to the selection of 16 candidate neutral sites—termed N1–N16 (Fig. <a reveal-id="DSV024F1" data-open="DSV024F1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Afterwards, a qualitative analysis of the genomic context of these ORFs was performed (Fig. <a reveal-id="DSV024F1" data-open="DSV024F1" class="link link-reveal link-table xref-fig">1</a>B) and revealed that the N2 site superimposes the 5′ region of an ORF (<em>slr0369</em>) putatively encoding a cation or drug efflux system protein, the N4 site overlaps the 5′ region of an ORF (<em>slr0415</em>) putatively encoding a Na<sup>+</sup>/H<sup>+</sup> antiporter and the N9 site corresponds to the ORF located immediately upstream and in the same direction as an ORF (<em>sll0182</em>) putatively encoding an ABC-type transporter. Therefore, these three sites were disregarded, and transcription analyses were performed for the other 13 loci.</p>                            <h3 scrollto-destination=87198814 id="87198814" class="section-title" data-legacy-id=s3b>3.2. Transcription analysis and final selection</h3>
<p>To assess whether there was no transcription or whether the transcript levels of the identified loci were negligible, RT-PCRs were initially performed using RNA extracted from <em>Synechocystis</em> cells collected at the exponential growth phase (OD<sub>730</sub> ≈ 0.8). The results revealed high transcript levels for N3, N11, N12 and N13 sites, low transcript levels for N6, N8, N14 and N15 sites, low primer efficiency for N1 site and no detectable amplification for the remaining sites (N5, N7, N10 and N16). Regarding N1 site, the transcription analysis was unreliable, and the locus was too small to design other primers so this site disregarded (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S27</a></span>). Further transcriptional studies were performed collecting cells at different phases (early exponential, exponential and stationary), to evaluate a possible differential transcription of sites N5, N6, N7, N8, N10, N14, N15 and N16 (Fig. <a reveal-id="DSV024F2" data-open="DSV024F2" class="link link-reveal link-table xref-fig">2</a>). The results showed that there was residual transcription for N6 and N14 sites throughout the various growth phases, and therefore, these two sites were disregarded. Concerning the other sites, it was possible to detect residual transcription for N8 and N15 sites, but only at one point of the growth curve—samples collected at the early exponential. Thus, six sites (N5, N7, N8, N10, N15 and N16) were selected to generate mutants to validate their ‘neutrality’. Subsequent cloning difficulties in constructing N7-related plasmids, probably due to secondary structures formation, resulted in this site also being disregarded. The final five target sites (N5, N8, N10, N15 and N16) are evenly distributed throughout the chromosome, with intervals in the order of 60−90° (72° corresponds to 1/5 of the chromosome, see Fig. <a reveal-id="DSV024F1" data-open="DSV024F1" class="link link-reveal link-table xref-fig">1</a>A).</p>                            <h3 scrollto-destination=87198816 id="87198816" class="section-title" data-legacy-id=s3c>3.3. Generation of mutants</h3>
<p>The integrative plasmids used to transform <em>Synechocystis</em> and characterize the neutral sites (Table <a reveal-id="DSV024TB1" data-open="DSV024TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>) were constructed as described in Material and Methods (for details, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). <em>Synechocystis</em> mutants were obtained by natural transformation with the respective plasmid (Table <a reveal-id="DSV024TB1" data-open="DSV024TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>), and full segregation was confirmed by PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S28</a></span>) and Southern blot (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S29</a></span>). All the mutants produced in this study are fully segregated, with all chromosomal copies mutated and the insertion of ectopic genes was also confirmed (resistance cassettes and <em>gfp</em> modules). The mutants produced using the pSN<em>n</em>KS plasmid series (SN<em>n</em>KS mutants, containing the double selection cassette) exhibited the expected phenotypes of kanamycin resistance (Km<sup>R</sup>) and sucrose sensitivity (Suc<sup>S</sup>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S30</a></span>). </p>                        <a id="87198818" scrollto-destination="87198818"></a>
<div content-id="DSV024TB1" class="table-modal"><div class="table-wrap"><div id="DSV024TB1" class="table-wrap-title" data-id="DSV024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of organisms and plasmids used/generated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism name<sup>a</sup> </th><th>Description </th><th>Source </th></tr></thead><tbody><tr><td><em>Escherichia coli</em> DH5α </td><td>Transformation/cloning strain </td><td>Invitrogen </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>Wild-type strain </td><td>Pasteur Culture Collection </td></tr><tr><td>SN<em>n</em>K </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>KS </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance/sucrose sensitivity cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>K.gfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the promoterless GFP encoding sequence </td><td>This work </td></tr><tr><td>SN<em>n</em>K.Cgfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the GFP encoding sequence under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>) </td><td>This work </td></tr><tr><td>Plasmid name<sup>a</sup> </td><td>Description </td><td>Source </td></tr><tr><td>pGEM<sup>®</sup>-T easy </td><td>T/A cloning vector </td><td>Promega </td></tr><tr><td>pK18mobsacB </td><td>Small mobilizable multi-purpose cloning vector derived from the <em>E. coli</em> plasmids pK18, conferring resistance to kanamycin (<em>nptII</em>), and sensitivity to sucrose (<em>sacB</em>) </td><td>National BioResource Project (NIG, Japan): <em>E. coli</em> </td></tr><tr><td>pSB1A2-E0240 </td><td>BioBrick™ vector containing the part BBa_E0240 (GFP generator) </td><td>Registry of Standard Biological Parts, MIT </td></tr><tr><td>pJ201-<em>P<sub>psbA2</sub></em><sub>*</sub> </td><td>Synthesized minimal <em>psbA2</em> promoter region, meeting the BioBrick RFC[10]standard specifications, cloned into the delivery vector pJ201 </td><td>DNA 2.0 Inc., this work </td></tr><tr><td>pBSK-FP300 </td><td>Synthetic interface containing a BioBrick-compatible multiple cloning site (<em>Mun</em>I, <em>Not</em>I, <em>Xba</em>I, <em>Spe</em>I, <em>Not</em>I, <em>Pst</em>I) insulated with the transcriptional terminators BBa_B1006 and BBa_B0062 (upstream), and BBa_B0053 and BBa_B020 (downstream) </td><td>Epoch Life Science Inc., this work </td></tr><tr><td>pSN<em>n</em> </td><td>pGEM-T easy-based plasmid series containing the BioBrick-compatible interface flanked by the two regions for double homologous recombination on neutral site <em>n</em> </td><td>This work </td></tr><tr><td>pSN<em>n</em>K </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>KS </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance/sucrose sensitivity cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.gfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 cloned in the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.Cgfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>), cloned in the BioBrick-compatible interface </td><td>This work </td></tr></tbody></table></div><div content-id="DSV024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism name<sup>a</sup> </th><th>Description </th><th>Source </th></tr></thead><tbody><tr><td><em>Escherichia coli</em> DH5α </td><td>Transformation/cloning strain </td><td>Invitrogen </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>Wild-type strain </td><td>Pasteur Culture Collection </td></tr><tr><td>SN<em>n</em>K </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>KS </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance/sucrose sensitivity cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>K.gfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the promoterless GFP encoding sequence </td><td>This work </td></tr><tr><td>SN<em>n</em>K.Cgfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the GFP encoding sequence under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>) </td><td>This work </td></tr><tr><td>Plasmid name<sup>a</sup> </td><td>Description </td><td>Source </td></tr><tr><td>pGEM<sup>®</sup>-T easy </td><td>T/A cloning vector </td><td>Promega </td></tr><tr><td>pK18mobsacB </td><td>Small mobilizable multi-purpose cloning vector derived from the <em>E. coli</em> plasmids pK18, conferring resistance to kanamycin (<em>nptII</em>), and sensitivity to sucrose (<em>sacB</em>) </td><td>National BioResource Project (NIG, Japan): <em>E. coli</em> </td></tr><tr><td>pSB1A2-E0240 </td><td>BioBrick™ vector containing the part BBa_E0240 (GFP generator) </td><td>Registry of Standard Biological Parts, MIT </td></tr><tr><td>pJ201-<em>P<sub>psbA2</sub></em><sub>*</sub> </td><td>Synthesized minimal <em>psbA2</em> promoter region, meeting the BioBrick RFC[10]standard specifications, cloned into the delivery vector pJ201 </td><td>DNA 2.0 Inc., this work </td></tr><tr><td>pBSK-FP300 </td><td>Synthetic interface containing a BioBrick-compatible multiple cloning site (<em>Mun</em>I, <em>Not</em>I, <em>Xba</em>I, <em>Spe</em>I, <em>Not</em>I, <em>Pst</em>I) insulated with the transcriptional terminators BBa_B1006 and BBa_B0062 (upstream), and BBa_B0053 and BBa_B020 (downstream) </td><td>Epoch Life Science Inc., this work </td></tr><tr><td>pSN<em>n</em> </td><td>pGEM-T easy-based plasmid series containing the BioBrick-compatible interface flanked by the two regions for double homologous recombination on neutral site <em>n</em> </td><td>This work </td></tr><tr><td>pSN<em>n</em>K </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>KS </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance/sucrose sensitivity cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.gfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 cloned in the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.Cgfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>), cloned in the BioBrick-compatible interface </td><td>This work </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup><em>n</em> = 5, 8, 10, 15, 16.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198818" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV024TB1" class="table-wrap-title" data-id="DSV024TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of organisms and plasmids used/generated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism name<sup>a</sup> </th><th>Description </th><th>Source </th></tr></thead><tbody><tr><td><em>Escherichia coli</em> DH5α </td><td>Transformation/cloning strain </td><td>Invitrogen </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>Wild-type strain </td><td>Pasteur Culture Collection </td></tr><tr><td>SN<em>n</em>K </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>KS </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance/sucrose sensitivity cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>K.gfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the promoterless GFP encoding sequence </td><td>This work </td></tr><tr><td>SN<em>n</em>K.Cgfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the GFP encoding sequence under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>) </td><td>This work </td></tr><tr><td>Plasmid name<sup>a</sup> </td><td>Description </td><td>Source </td></tr><tr><td>pGEM<sup>®</sup>-T easy </td><td>T/A cloning vector </td><td>Promega </td></tr><tr><td>pK18mobsacB </td><td>Small mobilizable multi-purpose cloning vector derived from the <em>E. coli</em> plasmids pK18, conferring resistance to kanamycin (<em>nptII</em>), and sensitivity to sucrose (<em>sacB</em>) </td><td>National BioResource Project (NIG, Japan): <em>E. coli</em> </td></tr><tr><td>pSB1A2-E0240 </td><td>BioBrick™ vector containing the part BBa_E0240 (GFP generator) </td><td>Registry of Standard Biological Parts, MIT </td></tr><tr><td>pJ201-<em>P<sub>psbA2</sub></em><sub>*</sub> </td><td>Synthesized minimal <em>psbA2</em> promoter region, meeting the BioBrick RFC[10]standard specifications, cloned into the delivery vector pJ201 </td><td>DNA 2.0 Inc., this work </td></tr><tr><td>pBSK-FP300 </td><td>Synthetic interface containing a BioBrick-compatible multiple cloning site (<em>Mun</em>I, <em>Not</em>I, <em>Xba</em>I, <em>Spe</em>I, <em>Not</em>I, <em>Pst</em>I) insulated with the transcriptional terminators BBa_B1006 and BBa_B0062 (upstream), and BBa_B0053 and BBa_B020 (downstream) </td><td>Epoch Life Science Inc., this work </td></tr><tr><td>pSN<em>n</em> </td><td>pGEM-T easy-based plasmid series containing the BioBrick-compatible interface flanked by the two regions for double homologous recombination on neutral site <em>n</em> </td><td>This work </td></tr><tr><td>pSN<em>n</em>K </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>KS </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance/sucrose sensitivity cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.gfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 cloned in the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.Cgfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>), cloned in the BioBrick-compatible interface </td><td>This work </td></tr></tbody></table></div><div content-id="DSV024TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism name<sup>a</sup> </th><th>Description </th><th>Source </th></tr></thead><tbody><tr><td><em>Escherichia coli</em> DH5α </td><td>Transformation/cloning strain </td><td>Invitrogen </td></tr><tr><td><em>Synechocystis</em> sp. PCC 6803 </td><td>Wild-type strain </td><td>Pasteur Culture Collection </td></tr><tr><td>SN<em>n</em>K </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>KS </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance/sucrose sensitivity cassette </td><td>This work </td></tr><tr><td>SN<em>n</em>K.gfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the promoterless GFP encoding sequence </td><td>This work </td></tr><tr><td>SN<em>n</em>K.Cgfp </td><td><em>Synechocystis</em> mutants with replacement of the neutral site N<em>n</em> by a kanamycin resistance cassette and the GFP encoding sequence under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>) </td><td>This work </td></tr><tr><td>Plasmid name<sup>a</sup> </td><td>Description </td><td>Source </td></tr><tr><td>pGEM<sup>®</sup>-T easy </td><td>T/A cloning vector </td><td>Promega </td></tr><tr><td>pK18mobsacB </td><td>Small mobilizable multi-purpose cloning vector derived from the <em>E. coli</em> plasmids pK18, conferring resistance to kanamycin (<em>nptII</em>), and sensitivity to sucrose (<em>sacB</em>) </td><td>National BioResource Project (NIG, Japan): <em>E. coli</em> </td></tr><tr><td>pSB1A2-E0240 </td><td>BioBrick™ vector containing the part BBa_E0240 (GFP generator) </td><td>Registry of Standard Biological Parts, MIT </td></tr><tr><td>pJ201-<em>P<sub>psbA2</sub></em><sub>*</sub> </td><td>Synthesized minimal <em>psbA2</em> promoter region, meeting the BioBrick RFC[10]standard specifications, cloned into the delivery vector pJ201 </td><td>DNA 2.0 Inc., this work </td></tr><tr><td>pBSK-FP300 </td><td>Synthetic interface containing a BioBrick-compatible multiple cloning site (<em>Mun</em>I, <em>Not</em>I, <em>Xba</em>I, <em>Spe</em>I, <em>Not</em>I, <em>Pst</em>I) insulated with the transcriptional terminators BBa_B1006 and BBa_B0062 (upstream), and BBa_B0053 and BBa_B020 (downstream) </td><td>Epoch Life Science Inc., this work </td></tr><tr><td>pSN<em>n</em> </td><td>pGEM-T easy-based plasmid series containing the BioBrick-compatible interface flanked by the two regions for double homologous recombination on neutral site <em>n</em> </td><td>This work </td></tr><tr><td>pSN<em>n</em>K </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>KS </td><td>pSN<em>n</em> plasmid series with the kanamycin resistance/sucrose sensitivity cassette cloned upstream of the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.gfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 cloned in the BioBrick-compatible interface </td><td>This work </td></tr><tr><td>pSN<em>n</em>K.Cgfp </td><td>pSN<em>n</em>K plasmid series with the GFP generator BioBrick BBa_E0240 under the control of a constitutive promoter (<em>P<sub>psbA2</sub></em><sub>*</sub>), cloned in the BioBrick-compatible interface </td><td>This work </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup><em>n</em> = 5, 8, 10, 15, 16.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198818" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87198819 id="87198819" class="section-title" data-legacy-id=s3d>3.4. Statistical analysis of cyanobacteria-specific growth</h3>
<p>To assess the neutrality of the sites, <em>Synechocystis</em> mutants were produced exchanging each putative neutral site by a kanamycin resistance cassette, using the pSN<em>n</em>K plasmid series (Table <a reveal-id="DSV024TB1" data-open="DSV024TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). The successful generation of fully segregated mutants in the five sites showed that the mutations did not affect cell viability under standard growth conditions. The growth of the wild type and the five mutants was evaluated in standard growth conditions for <em>Synechocystis</em>: continuous light (20 µmol photons m<sup>−2</sup> s<sup>−1</sup>; CL) or 12 h light (20 µmol photons m<sup>−2</sup> s<sup>−1</sup>)/12 h dark cycles (LD), in autotrophy (no glucose; G0) or mixotrophy (5 mM glucose; G5), and at a constant temperature of 30°C. The OD of the cultures (OD<sub>730</sub>) was monitored daily for 10 consecutive days, and the resulting growth curves for each organism and cultivation conditions were obtained (Fig. <a reveal-id="DSV024F3" data-open="DSV024F3" class="link link-reveal link-table xref-fig">3</a>). To assess the effect of the mutations on growth, a statistical analysis of variance (ANOVA) of the growth data was used to compare the wild type and the mutants, using the specific growth rate—<em>µ</em> (see Material and Methods for details). The use of this parameter was considered reliable, since the test duration was the same in all conditions, and growth patterns were similar within the different mutants and replicas. The results of the three-way ANOVA analysis performed on the growth results, including <em>P</em>-values of up to second-order interactions can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. Third-order interactions were in accordance to these results. Comparing the effect on growth of each mutation to the wild type, mutants SN5K and SN8K showed highly significant <em>P</em>-values, while for SN10K the <em>P</em>-value is ∼0.01, and for SN15K and SN16K, the <em>P</em>-values are not statistically significant (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). In detail, mutant SN5K grows slower than the wild type (particularly in the presence of glucose), while mutant SN8K grows faster under continuous light (with or without glucose). Taking these results into consideration, the chromosomal loci neutrality may be qualitatively ordered as WT ≈ N15 ≈ N16 &gt; N10 &gt; N8 &gt;&gt; N5, in terms of growth. For the remaining variables considered in the ANOVA, irradiance and glucose were the major parameters affecting growth (with highly significant <em>P</em>-values). </p>                        <a id="87198821" scrollto-destination="87198821"></a>
<div id="DSV024F3" data-id="DSV024F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02403.png?Expires=2147483647&Signature=pZUAqnnbPZ5HnLuHEoB5XXjy799BcP3r2uvqieQkMNDEk5NSCEKo53oJMqjTMFkwufFImesVT7lodgQQy85Vo1O0S20OpaRfhxRIpMNawHjmfnM9qRJ6GUqC4JrtwBk~SlV0tzymLDQNzhKXmyyJrQgw5cDICX1q3jYuDFQ9xq0IG9eOngbsAD6WrJcrAbS9IJFEjBrWpKFpAXFWoKodosb2g-FERfKcTBCQDZgc6J1BpuzzPJn3fiq3IuXD5cX3aoimybf~DoCYVXfM0tRdCjeOM77Ds1PcFVnK4VOOx4S~4cl2oks3KPg0EfUZIYaiXT1UPwqnWeK4xfq4VRYZgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Growth curves of Synechocystis wild type and mutants in the five neutral sites (SN5K, SN8K, SN10K, SN15K, SN16K) cultivated under different conditions. Ten millilitres of culture was grown in 25-ml flasks at 30°C, and OD730 values were recorded daily for 10 consecutive days in continuous light (CL) or 12 h light (20µmolphotons/m2/s)/12 h dark cycles (LD), in autotrophy (G0) or mixotrophy (5 mM glucose, G5). These results are representative of three biological replicates and technical duplicates, and the differences among them are not significant; error bars show ±S.D." path-from-xml="dsv02403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198821/dsv02403.tiff" path-from-xml="dsv02403.tiff" target="_blank">View large</a><a section="87198821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02403.png?Expires=2147483647&Signature=TqDMR~~9VqTRMU5BeM6BF36uMSQ1A2PofL6odjuIVk6xjUIHFUNbous5eVQ-8kxdldO3o-U67QLbdatNpCH5y3-CleYvEh15MgDUQnl-N1r0eHIV6rPUZ5Smd7ZxoMy12NuCBsyd8oIiLllSgQrGIu1OWQduY3ePmteVrGneUy5ICT2DRRbdXbCtpJ7u32e0yj~6tK1gDYKTNNMV6zB9XjSBEtKjqnKhg8CNBBH~1ceyXcNsoKq-ER9c1nWchfn2Fz4q5Kzs6pOJ3ZUJbRJX4pRFC6Dd~wzz7SQ6iQChqPhLBy0KudruNabhsZ964g2UACi2ZIqfekvHf1v3bg70fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198821&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Growth curves of <em>Synechocystis</em> wild type and mutants in the five neutral sites (SN5K, SN8K, SN10K, SN15K, SN16K) cultivated under different conditions. Ten millilitres of culture was grown in 25-ml flasks at 30°C, and OD<sub>730</sub> values were recorded daily for 10 consecutive days in continuous light (CL) or 12 h light (20µmolphotons/m<sup>2</sup>/s)/12 h dark cycles (LD), in autotrophy (G0) or mixotrophy (5 mM glucose, G5). These results are representative of three biological replicates and technical duplicates, and the differences among them are not significant; error bars show ±S.D.</p></div></div><div content-id="DSV024F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02403.png?Expires=2147483647&Signature=pZUAqnnbPZ5HnLuHEoB5XXjy799BcP3r2uvqieQkMNDEk5NSCEKo53oJMqjTMFkwufFImesVT7lodgQQy85Vo1O0S20OpaRfhxRIpMNawHjmfnM9qRJ6GUqC4JrtwBk~SlV0tzymLDQNzhKXmyyJrQgw5cDICX1q3jYuDFQ9xq0IG9eOngbsAD6WrJcrAbS9IJFEjBrWpKFpAXFWoKodosb2g-FERfKcTBCQDZgc6J1BpuzzPJn3fiq3IuXD5cX3aoimybf~DoCYVXfM0tRdCjeOM77Ds1PcFVnK4VOOx4S~4cl2oks3KPg0EfUZIYaiXT1UPwqnWeK4xfq4VRYZgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Growth curves of Synechocystis wild type and mutants in the five neutral sites (SN5K, SN8K, SN10K, SN15K, SN16K) cultivated under different conditions. Ten millilitres of culture was grown in 25-ml flasks at 30°C, and OD730 values were recorded daily for 10 consecutive days in continuous light (CL) or 12 h light (20µmolphotons/m2/s)/12 h dark cycles (LD), in autotrophy (G0) or mixotrophy (5 mM glucose, G5). These results are representative of three biological replicates and technical duplicates, and the differences among them are not significant; error bars show ±S.D." path-from-xml="dsv02403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198821/dsv02403.tiff" path-from-xml="dsv02403.tiff" target="_blank">View large</a><a section="87198821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02403.png?Expires=2147483647&Signature=TqDMR~~9VqTRMU5BeM6BF36uMSQ1A2PofL6odjuIVk6xjUIHFUNbous5eVQ-8kxdldO3o-U67QLbdatNpCH5y3-CleYvEh15MgDUQnl-N1r0eHIV6rPUZ5Smd7ZxoMy12NuCBsyd8oIiLllSgQrGIu1OWQduY3ePmteVrGneUy5ICT2DRRbdXbCtpJ7u32e0yj~6tK1gDYKTNNMV6zB9XjSBEtKjqnKhg8CNBBH~1ceyXcNsoKq-ER9c1nWchfn2Fz4q5Kzs6pOJ3ZUJbRJX4pRFC6Dd~wzz7SQ6iQChqPhLBy0KudruNabhsZ964g2UACi2ZIqfekvHf1v3bg70fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198821&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Growth curves of <em>Synechocystis</em> wild type and mutants in the five neutral sites (SN5K, SN8K, SN10K, SN15K, SN16K) cultivated under different conditions. Ten millilitres of culture was grown in 25-ml flasks at 30°C, and OD<sub>730</sub> values were recorded daily for 10 consecutive days in continuous light (CL) or 12 h light (20µmolphotons/m<sup>2</sup>/s)/12 h dark cycles (LD), in autotrophy (G0) or mixotrophy (5 mM glucose, G5). These results are representative of three biological replicates and technical duplicates, and the differences among them are not significant; error bars show ±S.D.</p></div></div>                            <h3 scrollto-destination=87198822 id="87198822" class="section-title" data-legacy-id=s3e>3.5. Synthetic interface robustness and neutral sites functionality</h3>
<p>To use the constructed vectors as tools for the integration of synthetic modules, a BioBrick-compatible cloning interface was designed and synthesized. A DNA sequence containing the <em>Mun</em>I, <em>Not</em>I, <em>Xba</em>I, <em>Spe</em>I and <em>Pst</em>I restriction sites, compatible with the RFC[10] assembly standard, was flanked by two double transcription terminators (BioBricks BBa_B1006 and BBa_B0062 upstream, and BBa_B0053 and BBa_B0020 downstream), to prevent any potential promoter activity from backbone DNA sequences. The interface was synthesized and cloned between the two homologous recombination sequences of each vector. To evaluate the proper transcriptional insulation of the interface, a promoterless green fluorescent protein generator (GFP; BioBrick composite part BBa_E0240) was cloned in the pSN<em>n</em>K plasmid series, and <em>Synechocystis</em> mutants were produced (mutants SN<em>n</em>K.gfp, Table <a reveal-id="DSV024TB1" data-open="DSV024TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Additionally, to assess the functionality of the neutral sites, <em>Synechocystis</em> mutants carrying the GFP generator under the control of a constitutive synthetic promoter (minimal <em>psbA2</em> promoter, <em>P<sub>psbA2</sub></em><sub>*</sub>) were constructed (mutants SN<em>n</em>K.Cgfp, Table <a reveal-id="DSV024TB1" data-open="DSV024TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). <em>gfp</em> transcription was assessed by RT-PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S31A</a></span>), and GFP expression was analysed by western blot (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S31B</a></span>), confocal microscopy (Fig. <a reveal-id="DSV024F4" data-open="DSV024F4" class="link link-reveal link-table xref-fig">4</a>A) and fluorimetry (Fig. <a reveal-id="DSV024F4" data-open="DSV024F4" class="link link-reveal link-table xref-fig">4</a>B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S32</a></span>). In all cases, <em>gfp</em> transcripts and GFP levels could only be detected in mutants where the promoter <em>P<sub>psbA2</sub></em><sub>*</sub> is present (SN<em>n</em>K.Cgfp); in these mutants, GFP fluorescence is relatively constant over time (11 days). Moreover, to determine whether the same <em>P<sub>psbA2*</sub></em> promoter-<em>gfp</em> construction is equivalent in terms of transcription strength at the different sites (chromosomal three-dimensional structure can affect promoter strength), RT-qPCR was performed using RNA extracted from the SN<em>n</em>K.Cgfp mutants (wild type was used as negative control). All RT-qPCR parameters in these experiments were in agreement with the MIQE guidelines.<sup><a reveal-id="DSV024C28" data-open="DSV024C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The normalized fold expression of <em>gfp</em> in the mutants was calculated, and there are no statistically significant differences between the five sites, as calculated by one-way ANOVA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S33</a></span>). </p>                        <a id="87198824" scrollto-destination="87198824"></a>
<div id="DSV024F4" data-id="DSV024F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02404.png?Expires=2147483647&Signature=fAjZlmZLFl35lSzKdnfbzlP3azIZyi4lv0KjrxWZDx1bPYb9A5aYGlCaq1oH4Aiay3US8Q1K9NILqVAGQvWMFn8SDf-7ts7V8EF7LRcIl0m0bDy5ePLbf-Cmvoz1urP6nAM1tu5gs8~1drCmCwq3ynmNy1V-VoH5ANZYj6l4MNBpCGDwimprSBxEOk5MKl-u4yPHUVjreU0424JKRI72U9HswXsvdM3zhV8~0DXCAXwGpyWjbzCBrZasljucFikERhP9CcedqD6YxEt7bKNnBFyfTfh60AIISO6IXoiOLnRqoPK8zX6BFkkCnWEPV5k~OmWSTVp~7exaXE2FozoF5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of GFP expression in Synechocystis wild type (WT) and insertion mutants containing a promoterless GFP generator (SNnK.gfp) or the GFP generator under the control of the constitutive minimal Ppsba2* synthetic promoter (SNnK.Cgfp). (A) Confocal micrographs of Synechocystis: autofluorescence is depicted in the left column, GFP signal in the middle column and the merged signals in the right column. Scale bars, 2.5 µm. (B) Normalized fluorescence of the cultures analysed in A. Measurements were performed in triplicate, using 200 µl of each culture, and fluorescence was normalized to OD790. Normalized fluorescence from the wild type was used as baseline. Bars indicate mean ± S.D." path-from-xml="dsv02404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198824/dsv02404.tiff" path-from-xml="dsv02404.tiff" target="_blank">View large</a><a section="87198824" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02404.png?Expires=2147483647&Signature=fTolMNTT25hRPc4XULkkIY9WEWrZXMEVaF~ek9yq6afLkoXFayzWn6h-rzodmY7AZXMajbcwmDSc~e-Vbrr6pQ0ep3R0InyzPKTO1jDfphhK050NbZ9wYArYUJeSYB7b1dg05bIpbpAz-80Bwgg9ZLxSZz49jU6lXm-1FeiHX9SMaz4911JNMbTqRNDkZm6gQOyIVYa-4EOfjF9LOVRIh94V2ziJLjqe-rCEdwvYNKgvNVNbcf4SycCx0GxU-XSbPkCLRVAdBJBN0Mfy2-~YhcAXPogWqbGxIJ0HiqvB-jcNoqe3WKCpdVJ9XfnVAw5UTuEqa27Yhp-kJnKrTAYY1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198824&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of GFP expression in <em>Synechocystis</em> wild type (WT) and insertion mutants containing a promoterless GFP generator (SN<em>n</em>K.gfp) or the GFP generator under the control of the constitutive minimal <em>P<sub>psba2</sub></em><sub>*</sub> synthetic promoter (SN<em>n</em>K.Cgfp). (A) Confocal micrographs of <em>Synechocystis</em>: autofluorescence is depicted in the left column, GFP signal in the middle column and the merged signals in the right column. Scale bars, 2.5 µm. (B) Normalized fluorescence of the cultures analysed in A. Measurements were performed in triplicate, using 200 µl of each culture, and fluorescence was normalized to OD<sub>790</sub>. Normalized fluorescence from the wild type was used as baseline. Bars indicate mean ± S.D.</p></div></div><div content-id="DSV024F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02404.png?Expires=2147483647&Signature=fAjZlmZLFl35lSzKdnfbzlP3azIZyi4lv0KjrxWZDx1bPYb9A5aYGlCaq1oH4Aiay3US8Q1K9NILqVAGQvWMFn8SDf-7ts7V8EF7LRcIl0m0bDy5ePLbf-Cmvoz1urP6nAM1tu5gs8~1drCmCwq3ynmNy1V-VoH5ANZYj6l4MNBpCGDwimprSBxEOk5MKl-u4yPHUVjreU0424JKRI72U9HswXsvdM3zhV8~0DXCAXwGpyWjbzCBrZasljucFikERhP9CcedqD6YxEt7bKNnBFyfTfh60AIISO6IXoiOLnRqoPK8zX6BFkkCnWEPV5k~OmWSTVp~7exaXE2FozoF5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of GFP expression in Synechocystis wild type (WT) and insertion mutants containing a promoterless GFP generator (SNnK.gfp) or the GFP generator under the control of the constitutive minimal Ppsba2* synthetic promoter (SNnK.Cgfp). (A) Confocal micrographs of Synechocystis: autofluorescence is depicted in the left column, GFP signal in the middle column and the merged signals in the right column. Scale bars, 2.5 µm. (B) Normalized fluorescence of the cultures analysed in A. Measurements were performed in triplicate, using 200 µl of each culture, and fluorescence was normalized to OD790. Normalized fluorescence from the wild type was used as baseline. Bars indicate mean ± S.D." path-from-xml="dsv02404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198824/dsv02404.tiff" path-from-xml="dsv02404.tiff" target="_blank">View large</a><a section="87198824" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02404.png?Expires=2147483647&Signature=fTolMNTT25hRPc4XULkkIY9WEWrZXMEVaF~ek9yq6afLkoXFayzWn6h-rzodmY7AZXMajbcwmDSc~e-Vbrr6pQ0ep3R0InyzPKTO1jDfphhK050NbZ9wYArYUJeSYB7b1dg05bIpbpAz-80Bwgg9ZLxSZz49jU6lXm-1FeiHX9SMaz4911JNMbTqRNDkZm6gQOyIVYa-4EOfjF9LOVRIh94V2ziJLjqe-rCEdwvYNKgvNVNbcf4SycCx0GxU-XSbPkCLRVAdBJBN0Mfy2-~YhcAXPogWqbGxIJ0HiqvB-jcNoqe3WKCpdVJ9XfnVAw5UTuEqa27Yhp-kJnKrTAYY1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198824&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of GFP expression in <em>Synechocystis</em> wild type (WT) and insertion mutants containing a promoterless GFP generator (SN<em>n</em>K.gfp) or the GFP generator under the control of the constitutive minimal <em>P<sub>psba2</sub></em><sub>*</sub> synthetic promoter (SN<em>n</em>K.Cgfp). (A) Confocal micrographs of <em>Synechocystis</em>: autofluorescence is depicted in the left column, GFP signal in the middle column and the merged signals in the right column. Scale bars, 2.5 µm. (B) Normalized fluorescence of the cultures analysed in A. Measurements were performed in triplicate, using 200 µl of each culture, and fluorescence was normalized to OD<sub>790</sub>. Normalized fluorescence from the wild type was used as baseline. Bars indicate mean ± S.D.</p></div></div>                            <h3 scrollto-destination=87198825 id="87198825" class="section-title" data-legacy-id=s3f>3.6. Proteomes of the wild type versus neutral site mutants</h3>
<p>To evaluate changes on the proteome of <em>Synechocystis</em>, associated with the deletion of each neutral site/expression of synthetic modules, an iTRAQ analysis was performed. Across the two iTRAQ experiments performed (see Material and Methods for details), 536 of the identified proteins with two or more unique peptides at an FDR of 1% were common to both studies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S8–S10</a></span>). During data analysis, the relative quantifications of the two iTRAQ experiments were scaled, so that the mean of the control samples (wild type) in both iTRAQs was equal. The hierarchical clustering of the iTRAQ labels following combination clearly shows a clustering of the wild-type technical (a and b) and biological (1 and 2) replicates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S34</a></span>). Close to the wild type, there is a replicate of the mutant containing the promoterless <em>gfp</em> module (SN15K.gfp), while all the mutants constitutively expressing GFP constitute a single cluster (with the biological replicates clustering closely), with the exception of two samples (bSN5K.Cgfp2 and bSN10K.Cgfp2) that do not cluster with any of the others (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S34</a></span>). During data analysis, it became apparent that for these two samples, there was a significantly higher concentration of membrane proteins compared with the other biological replicates from the same set. This was likely due to technical variability during protein extraction and as a result, both the bSN5K.Cgfp2 and bSN10K.Cgfp2 outliers were removed from further analysis. A PCA plot showing the first two components (87.79%, 8.77%) of protein variation (Fig. <a reveal-id="DSV024F5" data-open="DSV024F5" class="link link-reveal link-table xref-fig">5</a>), highlighted that as expected both the heterologous proteins GFP and NPTII (NPTII: aminoglycoside 3′-phosphotransferase, conferring resistance to neomycin/kanamycin) clustered away from the rest of the proteome. Only one native protein, P74485, clustered away with the NPTII protein. Overall, in all cases the proteome was found to be unchanged in any systematic way compared with the wild type, although a reduction of high-abundance phycobiliproteins in the mutants expressing GFP was observed (the effect was not seen in the promoterless <em>gfp</em> module mutant). This reduction was possibly an artefact of the relative quantification methods utilized in the iTRAQ workflow and would require absolute protein quantification against total protein weight per cell to verify. </p>                        <a id="87198827" scrollto-destination="87198827"></a>
<div id="DSV024F5" data-id="DSV024F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02405.png?Expires=2147483647&Signature=c~Ya76bj1emv9qfwgDuFd1lLlHwRk9vBNWZ9kVpFPSJOt8ORtjGO9bSIYR3Fwc-GpeXzQFKBZOWoHZEiW-j49mWtCxEfJuPO1UACC00hZ1aIH-HAOenuYzEeJ9ewuLM26RkYx-D-ViogyYi7qYVMS1N5wYnlf7o65wPqaew6sEDFJ4~9bUP3TcDymUECOi6zIsBgQTYFF9HcUIyrLlb~p5yAtM2wYlTkn3YD5yFrth~Kl~MiR096dJOFL1ou5CmmUa4XBZNzSIO4UWIBa60LHbvnBsU-SexNAIcMmUuv2cPQDSJRDavx8JOKev9HtnpQUxWMGpmRDGhRc3ONzjvuHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCA of the relative abundances of proteins identified in the iTRAQ experiments for Synechocystis wild type and neutral site mutants containing a gfp module (with or without the constitutive promoter). The first component (x-axis) is responsible for 87.79% of the variation in the samples, and the second component (y-axis) is responsible for 8.79%. The heterologous proteins GFP (P42212) and NPTII (P00552, aminoglycoside 3′-phosphotransferase conferring resistance to neomycin/kanamycin), and a native small stress-related protein (P74485) are clearly separated from the bulk of the proteome (central protein cluster). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198827/dsv02405.tiff" path-from-xml="dsv02405.tiff" target="_blank">View large</a><a section="87198827" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02405.png?Expires=2147483647&Signature=gWadRT~3bSPU6faHLGqtKKpbxLvTzaYioidZOPHB4Hb~gF2Xote5LXciTcifDyc9aMLfldkqkEKkEsC2NHDgXvLC21uXaVkoUqfUORbFLk09jWD4JxjjbYvexlK0rA0Pn8vhL2UYuvE9G4dZewjwabiky9chwSiB7FTFPyi4DQK3UnI0S~VuDBv3~2Dt7ZHc07Q3lfzKuoEyJ41woxglAq8ju9pcX4PedYTulzs4WABUnLH3DnrdfQmctEh7gTZSZ156Ct~VoxeOw2EkFdVM7GyyIxI6rTscyqmR5EhX~XA5UQ6WuIhCskl3UtyjVfcOouGeRa-5ZEhov1GEkj5T0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198827&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCA of the relative abundances of proteins identified in the iTRAQ experiments for <em>Synechocystis</em> wild type and neutral site mutants containing a <em>gfp</em> module (with or without the constitutive promoter). The first component (<em>x</em>-axis) is responsible for 87.79% of the variation in the samples, and the second component (<em>y</em>-axis) is responsible for 8.79%. The heterologous proteins GFP (P42212) and NPTII (P00552, aminoglycoside 3′-phosphotransferase conferring resistance to neomycin/kanamycin), and a native small stress-related protein (P74485) are clearly separated from the bulk of the proteome (central protein cluster). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV024F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/m_dsv02405.png?Expires=2147483647&Signature=c~Ya76bj1emv9qfwgDuFd1lLlHwRk9vBNWZ9kVpFPSJOt8ORtjGO9bSIYR3Fwc-GpeXzQFKBZOWoHZEiW-j49mWtCxEfJuPO1UACC00hZ1aIH-HAOenuYzEeJ9ewuLM26RkYx-D-ViogyYi7qYVMS1N5wYnlf7o65wPqaew6sEDFJ4~9bUP3TcDymUECOi6zIsBgQTYFF9HcUIyrLlb~p5yAtM2wYlTkn3YD5yFrth~Kl~MiR096dJOFL1ou5CmmUa4XBZNzSIO4UWIBa60LHbvnBsU-SexNAIcMmUuv2cPQDSJRDavx8JOKev9HtnpQUxWMGpmRDGhRc3ONzjvuHQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCA of the relative abundances of proteins identified in the iTRAQ experiments for Synechocystis wild type and neutral site mutants containing a gfp module (with or without the constitutive promoter). The first component (x-axis) is responsible for 87.79% of the variation in the samples, and the second component (y-axis) is responsible for 8.79%. The heterologous proteins GFP (P42212) and NPTII (P00552, aminoglycoside 3′-phosphotransferase conferring resistance to neomycin/kanamycin), and a native small stress-related protein (P74485) are clearly separated from the bulk of the proteome (central protein cluster). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv02405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198827/dsv02405.tiff" path-from-xml="dsv02405.tiff" target="_blank">View large</a><a section="87198827" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv02405.png?Expires=2147483647&Signature=gWadRT~3bSPU6faHLGqtKKpbxLvTzaYioidZOPHB4Hb~gF2Xote5LXciTcifDyc9aMLfldkqkEKkEsC2NHDgXvLC21uXaVkoUqfUORbFLk09jWD4JxjjbYvexlK0rA0Pn8vhL2UYuvE9G4dZewjwabiky9chwSiB7FTFPyi4DQK3UnI0S~VuDBv3~2Dt7ZHc07Q3lfzKuoEyJ41woxglAq8ju9pcX4PedYTulzs4WABUnLH3DnrdfQmctEh7gTZSZ156Ct~VoxeOw2EkFdVM7GyyIxI6rTscyqmR5EhX~XA5UQ6WuIhCskl3UtyjVfcOouGeRa-5ZEhov1GEkj5T0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198827&ar=2462458&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCA of the relative abundances of proteins identified in the iTRAQ experiments for <em>Synechocystis</em> wild type and neutral site mutants containing a <em>gfp</em> module (with or without the constitutive promoter). The first component (<em>x</em>-axis) is responsible for 87.79% of the variation in the samples, and the second component (<em>y</em>-axis) is responsible for 8.79%. The heterologous proteins GFP (P42212) and NPTII (P00552, aminoglycoside 3′-phosphotransferase conferring resistance to neomycin/kanamycin), and a native small stress-related protein (P74485) are clearly separated from the bulk of the proteome (central protein cluster). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h2 scrollto-destination=87198828 id="87198828" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p><em>Synechocystis</em> is a well-studied cyanobacterium with molecular tools developed for its genetic manipulation. Replicative plasmids have been used to introduce foreign DNA into this organism via conjugation or electroporation.<sup><a reveal-id="DSV024C34" data-open="DSV024C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSV024C35" data-open="DSV024C35" class="link link-ref link-reveal xref-bibr">35</a></sup> These plasmids are usually based on the broad-host-range <em>E. coli</em> IncQ plasmid RSF1010 and are not specific for cyanobacteria.<sup><a reveal-id="DSV024C9" data-open="DSV024C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSV024C10" data-open="DSV024C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Although presenting some advantages, such as being relatively easy to construct and quick to introduce in the cell, broad-host-range replicative plasmids can exhibit some instability.<sup><a reveal-id="DSV024C36" data-open="DSV024C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSV024C37" data-open="DSV024C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In addition, (i) the use of large native DNA fragments may lead to the recombination of the replicative vectors with the chromosome, (ii) removal of selective pressure may lead to loss of the vector and (iii) DNA structure of the vectors may have a strong influence on transcription. Altogether these aspects may compromise the system reliability and behaviour. In addition, it has been shown that <em>Synechocystis</em>' endogenous plasmids copy number per chromosome vary with nutritional conditions and growth phase.<sup><a reveal-id="DSV024C38" data-open="DSV024C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Taking into consideration that the number of chromosomes also varies throughout growth,<sup><a reveal-id="DSV024C39" data-open="DSV024C39" class="link link-ref link-reveal xref-bibr">39</a></sup> plasmids copy number calculations may become even more complex. Therefore, stable integration of ectopic DNA into <em>Synechocystis</em>' chromosome is a viable alternative for the implementation of synthetic modules into this photoautotrophic chassis. For this purpose, the availability of fully characterized chromosomal neutral sites is crucial.</p><p>To date, only a few chromosomal loci have been used to introduce foreign DNA into <em>Synechocystis</em>, but they have not been extensively characterized. One of these loci disrupts the gene <em>slr0168</em><sup><a reveal-id="DSV024C11" data-open="DSV024C11" class="link link-ref link-reveal xref-bibr">11</a></sup> that encodes a protein that was shown to be secreted.<sup><a reveal-id="DSV024C40" data-open="DSV024C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Although its function is unknown, Slr0168 possesses a DUF4114 protein domain, found towards the C-terminal of various bacterial proteins and suggested to carry enzymatic activity.<sup><a reveal-id="DSV024C41" data-open="DSV024C41" class="link link-ref link-reveal xref-bibr">41</a></sup> An additional locus was described as a ‘nonessential, nontranscribed’<sup><a reveal-id="DSV024C12" data-open="DSV024C12" class="link link-ref link-reveal xref-bibr">12</a></sup> region; however, RNA-seq works<sup><a reveal-id="DSV024C42" data-open="DSV024C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV024C43" data-open="DSV024C43" class="link link-ref link-reveal xref-bibr">43</a></sup> showed that the genes flanking that insertion locus are transcribed. Another locus described disrupts the <em>ssl0410</em> gene,<sup><a reveal-id="DSV024C14" data-open="DSV024C14" class="link link-ref link-reveal xref-bibr">14</a></sup> which is located downstream the gene encoding a subunit of the type I NAD(P)H dehydrogenase complex (<em>ndhB</em>) that is important for CO<sub>2</sub> uptake in <em>Synechocystis</em>.<sup><a reveal-id="DSV024C44" data-open="DSV024C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The intergenic region between <em>slr2030</em> and <em>slr2031</em>,<sup><a reveal-id="DSV024C13" data-open="DSV024C13" class="link link-ref link-reveal xref-bibr">13</a></sup> which encodes the regulatory protein RsbU, has also been used for chromosomal integrations. Nonetheless, <em>slr2031</em> is constitutively transcribed, and the respective mutant showed a defect in growth recovery after nitrogen- and sulfur-starvation-induced stationary phase.<sup><a reveal-id="DSV024C45" data-open="DSV024C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Furthermore, <em>slr2030</em> and <em>slr2031</em> are transcribed as an operon.<sup><a reveal-id="DSV024C42" data-open="DSV024C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV024C43" data-open="DSV024C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSV024C45" data-open="DSV024C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Consequently, insertions in that intergenic region may disrupt the transcriptional unit and lead to the same phenotype described for <em>slr2031</em> mutants. More recently, three other neutral sites have been identified,<sup><a reveal-id="DSV024C15" data-open="DSV024C15" class="link link-ref link-reveal xref-bibr">15</a></sup> based on a RNA-sequencing study,<sup><a reveal-id="DSV024C42" data-open="DSV024C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and characterized under continuous light, at 30°C. In that study, it was shown that the insertion of a reporter gene in the different sites resulted, however, in different fluorescence intensity. Since the integration sites described previously do not match at least one of the selection criteria established in this study, they were not considered here.</p><p>Within this work, several putative neutral sites were identified following an unbiased systematic approach and further characterized for the stable integration of synthetic circuits into <em>Synechocystis</em>' chromosome. Although non-coding DNA (ncDNA) regions could be used for this purpose, they may possess <em>cis</em>-regulatory control sequences, thus affecting the transcription profiles of several genes.<sup><a reveal-id="DSV024C46" data-open="DSV024C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSV024C47" data-open="DSV024C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Therefore, in this work, only ORFs putatively encoding unknown or hypothetical proteins were considered. After an initial genomic context analysis of the selected ORFs, RT-PCR transcription studies were performed at different phases of <em>Synechocystis</em> growth curve, and the transcription levels appeared to be negligible for the five sites characterized in this work. These results are partially supported by RNA-sequencing studies,<sup><a reveal-id="DSV024C42" data-open="DSV024C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV024C43" data-open="DSV024C43" class="link link-ref link-reveal xref-bibr">43</a></sup> reporting that the number of direct readings for sites N5, N8 and N16 was close to the cut-off threshold under the tested conditions, while for sites N10 and N15, transcripts were detected mainly in the exponential phase (transcriptional unit TU698),<sup><a reveal-id="DSV024C43" data-open="DSV024C43" class="link link-ref link-reveal xref-bibr">43</a></sup> or induced by dark (transcriptional unit TU1565),<sup><a reveal-id="DSV024C42" data-open="DSV024C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSV024C43" data-open="DSV024C43" class="link link-ref link-reveal xref-bibr">43</a></sup> respectively. This technique allows the high-throughput identification of transcriptional units and may be useful for the identification of additional neutral sites, namely genomic regions that are not transcribed under all the conditions tested. In all cases, complementary validation methods should be used (e.g. RT-qPCR)<sup><a reveal-id="DSV024C48" data-open="DSV024C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and cell viability and fitness assessed. The successful generation of fully segregated mutants, with each selected neutral site replaced by a kanamycin resistance cassette, showed that they do not encode proteins essential to the viability of the cyanobacterium under the tested growth conditions. Several studies reported that genes crucial to the viability of <em>Synechocystis</em> cannot be successfully and completely inactivated, resulting in heteroploid mutant cells, containing both chromosomes harbouring the wild-type gene and the knockout locus.<sup><a reveal-id="DSV024C49" data-open="DSV024C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSV024C50" data-open="DSV024C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Moreover, the neutrality of each site was assessed by analysing the growth of mutants SN5K, SN8K, SN10K, SN15K and SN16K cultivated under different conditions. The results showed that the N15 and N16 sites mutants exhibit growth patterns similar to those of the wild type, while the N5, N8 and N10 site mutants present statistically significant differences. The N5 mutant clearly exhibits a growth decrease, in particular when grown under light/dark cycles in the presence of glucose. Therefore, this should be taken into account when choosing this site for the integration of ectopic DNA. Interestingly, the mutant on site N8 exhibited growth rates slightly higher than the parental strain under continuous light, a feature that may be advantageous for biotechnological applications. As for the remaining variables considered in the ANOVA, and similarly to the results obtained by Lopo and co-workers,<sup><a reveal-id="DSV024C51" data-open="DSV024C51" class="link link-ref link-reveal xref-bibr">51</a></sup> irradiance and the presence of glucose were the major parameters affecting growth (highly significant <em>P</em>-values).</p><p>All the integrative plasmids constructed in this work possess a BioBrick-compatible synthetic interface for cloning purposes. The robustness of this synthetic interface was assessed using the promoterless GFP generator (BioBrick BBa_E0240), which showed no leakiness from the chromosomal backbone for each site, since no/negligible levels of <em>gfp</em> transcripts were detected by RT-PCR, and no GFP expression was observed by western blot, confocal microscopy or whole cell fluorescence measurements. In contrast, using the same techniques, <em>gfp</em> transcripts and GFP signal could be detected in all the mutants generated bearing the GFP generator under the control of a constitutive synthetic promoter (based on the photosensitive promoter of the <em>psbA2</em> gene of <em>Synechocystis</em>). This shows that these five sites can effectively be used for the functional integration of synthetic modules. Moreover, the relative quantification of transcripts by RT-qPCR showed that <em>gfp</em> transcription is not affected by the position of the neutral site in the chromosome, and consequently, the five sites may be considered equivalent in this respect.</p><p>The PCA derived from the iTRAQ experiment showed that the overall proteomes of the neutral site mutants do not change in any systematic way (with the exception of a reduction in the high-abundance phycobiliproteins) compared with the wild type. In addition, the heterologous proteins GFP and NPTII clustered away from the rest of the proteome, an expected result since these proteins are absent in the wild type. Only one native protein, P74485, clustered away from the bulk proteome. This small protein is largely uncharacterized, but has been reported as being related to stress conditions.<sup><a reveal-id="DSV024C52" data-open="DSV024C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSV024C53" data-open="DSV024C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>Altogether, these results show that the selected chromosomal sites are functional and suitable to receive ectopic DNA. In addition, the vectors constructed in this work followed a rational design that allows the stable integration of synthetic modules into <em>Synechocystis</em> foreseeing, e.g. the production of specialty chemicals. Moreover, the possibility to generate markerless mutants by transforming the SN<em>n</em>KS mutants with the pSN<em>n</em> plamids containing a module of interest enables the use of these vectors to sequentially introduce more than one synthetic module into the chromosome. The existence of more than one integration site, spread throughout the chromosome of <em>Synechocystis</em>, allows the implementation of complex synthetic circuits. The vectors design also considered the possibility to use different resistance cassettes making the implementation faster. This work contributes to the field of synthetic biology by allowing the use of <em>Synechocystis</em> as a photoautotrophic chassis, providing not only standard BioBrick-compatible vectors to integrate synthetic modules into fully characterized chromosomal neutral sites, but also chassis prone to receive a synthetic module without selective markers. Furthermore, our work constitutes a proof of concept for mapping and validating genomic neutral sites that can be applied to other biological systems.</p>                            <h2 scrollto-destination=87198835 id="87198835" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the European Commission through the Seventh Framework Programme, FP7-ENERGY-2012-1-2STAGE-308518 (CyanoFactory), from EU FP6-NEST-2005-Path-SYN project BioModularH2 (contract no. 043340) and from National Funds through Fundação para a Ciência e a Tecnologia (FCT) (grants SFRH/BD/36378/2007 to F.P., SFRH/BPD/64095/2009 to C.C.P., SFRH/BPD/74894/2010 to P.O.). We also acknowledge the Engineering and Physical Sciences Research Council (EPSRC) for funding (EP/E036252/1) and The University of Sheffield for Scholarship funding. Funding to pay the Open Access publication charges for this article was provided by the European Commission through the Seventh Framework Programme, FP7-ENERGY-2012-1-2STAGE-308518 (CyanoFactory).</p>                            <h2 scrollto-destination=87198837 id="87198837" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to C. Harley for reading the manuscript, M. Lopo for the preliminary growth experiments, and the ‘National BioResource Project (NIG, Japan): E.coli’ for providing the plasmid pK18mobsacB.</p>                            <h2 scrollto-destination=87198839 id="87198839" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV024C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Madigan</div>
               <div class="given-names">M.T.</div>
            </div>, <div class="name string-name"><div class="surname">Martinko</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Stahl</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">D.P.</div></div></div>
          <div class="year">2010</div>, <div class="source">Brock biology of microorganisms</div> . <div class="publisher-name">Benjamin Cummings</div>: <div class="publisher-loc">San Francisco</div>.</div></div></div></div><div content-id="DSV024C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Harwood</div>
               <div class="given-names">C.R.</div>
            </div>, <div class="name string-name"><div class="surname">Pohl</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wipat</div> <div class="given-names">A.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Bacillus subtilis: model Gram-positive synthetic biology chassis</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Colin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Anil</div> <div class="given-names">W.</div></div></div> (eds), <div class="source">Methods in microbiology</div> . <div class="publisher-name">Academic Press</div>: <div class="publisher-loc">Amsterdam, The Netherlands</div>, pp. <div class="fpage">87</div>–<div class="lpage">117</div>.</div></div></div></div><div content-id="DSV024C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Borodina</div> <div class="given-names">I.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Application of synthetic biology for production of chemicals in yeast <em>Saccharomyces cerevisiae</em></div>, <div class="source">FEMS Yeast Res</div> ., <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.</div></div></div></div><div content-id="DSV024C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Berla</div>
               <div class="given-names">B.M.</div>
            </div>, <div class="name string-name"><div class="surname">Saha</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Immethun</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Maranas</div> <div class="given-names">C.D.</div></div>, <div class="name string-name"><div class="surname">Moon</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Pakrasi</div> <div class="given-names">H.B.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Synthetic biology of cyanobacteria: unique challenges and opportunities</div>, <div class="source">Front. Microbiol.</div> , <div class="volume">4</div>, <div class="fpage">246</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthetic%20biology%20of%20cyanobacteria%3A%20unique%20challenges%20and%20opportunities&amp;author=B.M.%20Berla&amp;author=R.%20Saha&amp;author=C.M.%20Immethun&amp;author=C.D.%20Maranas&amp;author=T.S.%20Moon&amp;author=H.B.%20Pakrasi&amp;publication_year=2013&amp;journal=Front.%20Microbiol.&amp;volume=4&amp;pages=246" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2013.00246" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2013.00246" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2013.00246"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24009604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Whitton</div>
               <div class="given-names">B.A.</div>
            </div>, <div class="name string-name"><div class="surname">Potts</div> <div class="given-names">M.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Introduction to the cyanobacteria</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Whitton</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Potts</div> <div class="given-names">M.</div></div></div> (eds), <div class="source">The ecology of cyanobacteria—their diversity in time and space</div> . <div class="publisher-name">Kluwer Academic Publishers</div>: <div class="publisher-loc">Dordrecht, The Netherlands</div>, pp. <div class="fpage">1</div>–<div class="lpage">11</div>.</div></div></div></div><div content-id="DSV024C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Montagud</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Navarro</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Fernandez de Cordoba</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Urchueguia</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Patil</div> <div class="given-names">K.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Reconstruction and analysis of genome-scale metabolic model of a photosynthetic bacterium</div>, <div class="source">BMC Syst. Biol.</div> , <div class="volume">4</div>, <div class="fpage">156</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20and%20analysis%20of%20genome-scale%20metabolic%20model%20of%20a%20photosynthetic%20bacterium&amp;author=A.%20Montagud&amp;author=E.%20Navarro&amp;author=P.%20Fernandez%20de%20Cordoba&amp;author=J.%20Urchueguia&amp;author=K.%20Patil&amp;publication_year=2010&amp;journal=BMC%20Syst.%20Biol.&amp;volume=4&amp;pages=156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1752-0509-4-156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1752-0509-4-156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1752-0509-4-156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21083885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kotani</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">1996</div>, <div class="article-title">Sequence analysis of the genome of the unicellular cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions</div>, <div class="source">DNA Res.</div> , <div class="volume">3</div>, <div class="fpage">109</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20the%20unicellular%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803.%20II.%20Sequence%20determination%20of%20the%20entire%20genome%20and%20assignment%20of%20potential%20protein-coding%20regions&amp;author=T.%20Kaneko&amp;author=S.%20Sato&amp;author=H.%20Kotani&amp;publication_year=1996&amp;journal=DNA%20Res.&amp;volume=3&amp;pages=109-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/3.3.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F3.3.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F3.3.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Koksharova</div>
               <div class="given-names">O.A.</div>
            </div>, <div class="name string-name"><div class="surname">Wolk</div> <div class="given-names">C.P.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Genetic tools for cyanobacteria</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="volume">58</div>, <div class="fpage">123</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20tools%20for%20cyanobacteria&amp;author=O.A.%20Koksharova&amp;author=C.P.%20Wolk&amp;publication_year=2002&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=58&amp;pages=123-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-001-0864-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-001-0864-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-001-0864-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11876404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Heidorn</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Camsund</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">H.-H.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Synthetic biology in cyanobacteria: engineering and analyzing novel functions</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Chris</div> <div class="given-names">V.</div></div></div> (ed.), <div class="source">Methods enzymol</div> . <div class="publisher-name">Academic Press</div>: <div class="publisher-loc">New York</div>, pp. <div class="fpage">539</div>–<div class="lpage">79</div>.</div></div></div></div><div content-id="DSV024C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Taton</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Unglaub</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">N.E.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Broad-host-range vector system for synthetic biology and biotechnology in cyanobacteria</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">e136</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Broad-host-range%20vector%20system%20for%20synthetic%20biology%20and%20biotechnology%20in%20cyanobacteria&amp;author=A.%20Taton&amp;author=F.%20Unglaub&amp;author=N.E.%20Wright&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=e136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25074377" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Williams</div>
               <div class="given-names">J.G.K.</div>
            </div>
          </div>
          <div class="year">1988</div>, <div class="article-title">Construction of specific mutations in photosystem II photosynthetic reaction center by genetic engineering methods in Synechocystis 6803</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Packer</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Glazer</div> <div class="given-names">A.N.</div></div></div> (eds), <div class="source">Methods enzymol</div> . <div class="publisher-name">Academic Press</div>: <div class="publisher-loc">New York</div>, pp. <div class="fpage">766</div>–<div class="lpage">78</div>.</div></div></div></div><div content-id="DSV024C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Burnap</div>
               <div class="given-names">R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sherman</div> <div class="given-names">L.A.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">Role of disulfide linkage and putative intermolecular binding residues in the stability and binding of the extrinsic manganese-stabilizing protein to the photosystem II reaction center</div>, <div class="source">Biochemistry US</div> , <div class="volume">33</div>, <div class="fpage">13712</div>–<div class="lpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20disulfide%20linkage%20and%20putative%20intermolecular%20binding%20residues%20in%20the%20stability%20and%20binding%20of%20the%20extrinsic%20manganese-stabilizing%20protein%20to%20the%20photosystem%20II%20reaction%20center&amp;author=R.L.%20Burnap&amp;author=M.%20Qian&amp;author=J.R.%20Shen&amp;author=Y.%20Inoue&amp;author=L.A.%20Sherman&amp;publication_year=1994&amp;journal=Biochemistry%20US&amp;volume=33&amp;pages=13712-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00250a023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00250a023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00250a023"> </span></div> </div></div></div></div></div><div content-id="DSV024C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aoki</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fujita</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">A heme oxygenase isoform is essential for aerobic growth in the cyanobacterium <em>Synechocystis</em> sp. PCC 6803: modes of differential operation of two isoforms/enzymes to adapt to low oxygen environments in cyanobacteria</div>, <div class="source">Plant Cell Physiol.</div> , <div class="volume">52</div>, <div class="fpage">1744</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20heme%20oxygenase%20isoform%20is%20essential%20for%20aerobic%20growth%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20PCC%206803%3A%20modes%20of%20differential%20operation%20of%20two%20isoforms%2Fenzymes%20to%20adapt%20to%20low%20oxygen%20environments%20in%20cyanobacteria&amp;author=R.%20Aoki&amp;author=T.%20Goto&amp;author=Y.%20Fujita&amp;publication_year=2011&amp;journal=Plant%20Cell%20Physiol.&amp;volume=52&amp;pages=1744-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcr108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcr108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcr108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21828104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Aoki</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Kondo</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ishiura</div> <div class="given-names">M.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Circadian expression of the <em>dnaK</em> gene in the cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">177</div>, <div class="fpage">5606</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circadian%20expression%20of%20the%20dnaK%20gene%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=S.%20Aoki&amp;author=T.%20Kondo&amp;author=M.%20Ishiura&amp;publication_year=1995&amp;journal=J.%20Bacteriol.&amp;volume=177&amp;pages=5606-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.177.19.5606-5611.1995" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.177.19.5606-5611.1995" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.177.19.5606-5611.1995"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7559349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ng</div>
               <div class="given-names">A.H.</div>
            </div>, <div class="name string-name"><div class="surname">Berla</div> <div class="given-names">B.M.</div></div>, <div class="name string-name"><div class="surname">Pakrasi</div> <div class="given-names">H.B.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Fine-tuning of photoautotrophic protein production by combining promoters and neutral sites in the cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">81</div>, <div class="fpage">6857</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-tuning%20of%20photoautotrophic%20protein%20production%20by%20combining%20promoters%20and%20neutral%20sites%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=A.H.%20Ng&amp;author=B.M.%20Berla&amp;author=H.B.%20Pakrasi&amp;publication_year=2015&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=81&amp;pages=6857-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01349-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01349-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01349-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26209663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clerico</div>
               <div class="given-names">E.M.</div>
            </div>, <div class="name string-name"><div class="surname">Ditty</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Golden</div> <div class="given-names">S.S.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Specialized techniques for site-directed mutagenesis in cyanobacteria</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Rosato</div> <div class="given-names">E.</div></div></div> (ed.), <div class="source">Circadian rhythms</div> . <div class="publisher-name">Humana Press</div>: <div class="publisher-loc">Totowa</div>, pp. <div class="fpage">155</div>–<div class="lpage">71</div>.</div></div></div></div><div content-id="DSV024C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trautmann</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Voß</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wilde</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Al-Babili</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Microevolution in cyanobacteria: re-sequencing a motile substrain of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">435</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microevolution%20in%20cyanobacteria%3A%20re-sequencing%20a%20motile%20substrain%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=D.%20Trautmann&amp;author=B.%20Vo%C3%9F&amp;author=A.%20Wilde&amp;author=S.%20Al-Babili&amp;author=W.R.%20Hess&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=435-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23069868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kanesaki</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Shiwa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tajima</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Identification of substrain-specific mutations by massively parallel whole-genome resequencing of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res.</div> , <div class="volume">19</div>, <div class="fpage">67</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20substrain-specific%20mutations%20by%20massively%20parallel%20whole-genome%20resequencing%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=Y.%20Kanesaki&amp;author=Y.%20Shiwa&amp;author=N.%20Tajima&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=67-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22193367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stanier</div>
               <div class="given-names">R.Y.</div>
            </div>, <div class="name string-name"><div class="surname">Kunisawa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mandel</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cohenbaz</div> <div class="given-names">G.</div></div></div>
          <div class="year">1971</div>, <div class="article-title">Purification and properties of unicellular blue-green algae (order <em>Chroococcales</em>)</div>, <div class="source">Bacteriol. Rev.</div> , <div class="volume">35</div>, <div class="fpage">171</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20properties%20of%20unicellular%20blue-green%20algae%20(order%20Chroococcales)&amp;author=R.Y.%20Stanier&amp;author=R.%20Kunisawa&amp;author=M.%20Mandel&amp;author=G.%20Cohenbaz&amp;publication_year=1971&amp;journal=Bacteriol.%20Rev.&amp;volume=35&amp;pages=171-205" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4998365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sambrook</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Russell</div> <div class="given-names">D.W.</div></div></div>
          <div class="year">2001</div>, <div class="source">Molecular cloning: a laboratory manual</div> . <div class="publisher-name">Cold Spring Harbor Laboratory Press</div>: <div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="DSV024C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamagnini</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Troshina</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Oxelfelt</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Salema</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lindblad</div> <div class="given-names">P.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Hydrogenases in <em>Nostoc</em> sp. strain PCC 73102, a strain lacking a bidirectional enzyme</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">63</div>, <div class="fpage">1801</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hydrogenases%20in%20Nostoc%20sp.%20strain%20PCC%2073102%2C%20a%20strain%20lacking%20a%20bidirectional%20enzyme&amp;author=P.%20Tamagnini&amp;author=O.%20Troshina&amp;author=F.%20Oxelfelt&amp;author=R.%20Salema&amp;author=P.%20Lindblad&amp;publication_year=1997&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=63&amp;pages=1801-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16535596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Leitão</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Pereira</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bondoso</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Genes involved in the maturation of hydrogenase(s) in the nonheterocystous cyanobacterium <em>Lyngbya majuscula</em> CCAP 1446/4</div>, <div class="source">Int. J. Hydrogen Energ.</div> , <div class="volume">31</div>, <div class="fpage">1469</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20involved%20in%20the%20maturation%20of%20hydrogenase(s)%20in%20the%20nonheterocystous%20cyanobacterium%20Lyngbya%20majuscula%20CCAP%201446%2F4&amp;author=E.%20Leit%C3%A3o&amp;author=S.%20Pereira&amp;author=J.%20Bondoso&amp;publication_year=2006&amp;journal=Int.%20J.%20Hydrogen%20Energ.&amp;volume=31&amp;pages=1469-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijhydene.2006.06.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijhydene.2006.06.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijhydene.2006.06.012"> </span></div> </div></div></div></div></div><div content-id="DSV024C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ferreira</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Moradas-Ferreira</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mendes</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tamagnini</div> <div class="given-names">P.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Transcription profiles of hydrogenases related genes in the cyanobacterium <em>Lyngbya majuscula</em> CCAP 1446/4</div>, <div class="source">BMC Microbiol.</div> , <div class="volume">9</div>, <div class="fpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20profiles%20of%20hydrogenases%20related%20genes%20in%20the%20cyanobacterium%20Lyngbya%20majuscula%20CCAP%201446%2F4&amp;author=D.%20Ferreira&amp;author=F.%20Pinto&amp;author=P.%20Moradas-Ferreira&amp;author=M.%20Mendes&amp;author=P.%20Tamagnini&amp;publication_year=2009&amp;journal=BMC%20Microbiol.&amp;volume=9&amp;pages=67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-9-67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-9-67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-9-67"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19351394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schäfer</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Tauch</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jäger</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kalinowski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Thierbach</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Pühler</div> <div class="given-names">A.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">Small mobilizable multi-purpose cloning vectors derived from the <em>Escherichia coli</em> plasmids pK18 and pK19: selection of defined deletions in the chromosome of <em>Corynebacterium glutamicum</em></div>, <div class="source">Gene</div> , <div class="volume">145</div>, <div class="fpage">69</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20mobilizable%20multi-purpose%20cloning%20vectors%20derived%20from%20the%20Escherichia%20coli%20plasmids%20pK18%20and%20pK19%3A%20selection%20of%20defined%20deletions%20in%20the%20chromosome%20of%20Corynebacterium%20glutamicum&amp;author=A.%20Sch%C3%A4fer&amp;author=A.%20Tauch&amp;author=W.%20J%C3%A4ger&amp;author=J.%20Kalinowski&amp;author=G.%20Thierbach&amp;author=A.%20P%C3%BChler&amp;publication_year=1994&amp;journal=Gene&amp;volume=145&amp;pages=69-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(94)90324-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(94)90324-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(94)90324-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8045426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eriksson</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Salih</div> <div class="given-names">G.F.</div></div>, <div class="name string-name"><div class="surname">Ghebramedhin</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jansson</div> <div class="given-names">C.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Deletion mutagenesis of the 5′ <em>psbA2</em> region in <em>Synechocystis</em> 6803: identification of a putative cis element involved in photoregulation</div>, <div class="source">Mol. Cell. Biol. Res. Commun.</div> , <div class="volume">3</div>, <div class="fpage">292</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion%20mutagenesis%20of%20the%205%E2%80%B2%20psbA2%20region%20in%20Synechocystis%206803%3A%20identification%20of%20a%20putative%20cis%20element%20involved%20in%20photoregulation&amp;author=J.%20Eriksson&amp;author=G.F.%20Salih&amp;author=H.%20Ghebramedhin&amp;author=C.%20Jansson&amp;publication_year=2000&amp;journal=Mol.%20Cell.%20Biol.%20Res.%20Commun.&amp;volume=3&amp;pages=292-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mcbr.2000.0227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmcbr.2000.0227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmcbr.2000.0227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10964753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shibato</div>
               <div class="given-names">J.S.</div>
            </div>, <div class="name string-name"><div class="surname">Agrawal</div> <div class="given-names">G.A.</div></div>, <div class="name string-name"><div class="surname">Kato</div> <div class="given-names">H.K.</div></div>, <div class="name string-name"><div class="surname">Asayama</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Shirai</div> <div class="given-names">M.S.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">The 5′-upstream <em>cis</em>-acting sequences of a cyanobacterial <em>psbA</em> gene: analysis of their roles in basal, light-dependent and circadian transcription</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">267</div>, <div class="fpage">684</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%205%E2%80%B2-upstream%20cis-acting%20sequences%20of%20a%20cyanobacterial%20psbA%20gene%3A%20analysis%20of%20their%20roles%20in%20basal%2C%20light-dependent%20and%20circadian%20transcription&amp;author=J.S.%20Shibato&amp;author=G.A.%20Agrawal&amp;author=H.K.%20Kato&amp;author=M.A.%20Asayama&amp;author=M.S.%20Shirai&amp;publication_year=2002&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=267&amp;pages=684-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-002-0704-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-002-0704-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-002-0704-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12172808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pinto</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Pacheco</div> <div class="given-names">C.C.</div></div>, <div class="name string-name"><div class="surname">Ferreira</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Moradas-Ferreira</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Tamagnini</div> <div class="given-names">P.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Selection of suitable reference genes for RT-qPCR analyses in cyanobacteria</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e34983</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20of%20suitable%20reference%20genes%20for%20RT-qPCR%20analyses%20in%20cyanobacteria&amp;author=F.%20Pinto&amp;author=C.C.%20Pacheco&amp;author=D.%20Ferreira&amp;author=P.%20Moradas-Ferreira&amp;author=P.%20Tamagnini&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e34983" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0034983" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0034983" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0034983"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22496882" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bustin</div>
               <div class="given-names">S.A.</div>
            </div>, <div class="name string-name"><div class="surname">Benes</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Garson</div> <div class="given-names">J.A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The MIQE Guidelines: minimum information for publication of quantitative real-time PCR experiments</div>, <div class="source">Clin. Chem.</div> , <div class="volume">55</div>, <div class="fpage">611</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MIQE%20Guidelines%3A%20minimum%20information%20for%20publication%20of%20quantitative%20real-time%20PCR%20experiments&amp;author=S.A.%20Bustin&amp;author=V.%20Benes&amp;author=J.A.%20Garson&amp;publication_year=2009&amp;journal=Clin.%20Chem.&amp;volume=55&amp;pages=611-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1373/clinchem.2008.112797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1373%2Fclinchem.2008.112797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1373%2Fclinchem.2008.112797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19246619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bensadoun</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Weinstein</div> <div class="given-names">D.</div></div></div>
          <div class="year">1976</div>, <div class="article-title">Assay of proteins in the presence of interfering materials</div>, <div class="source">Anal. Biochem.</div> , <div class="volume">70</div>, <div class="fpage">241</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assay%20of%20proteins%20in%20the%20presence%20of%20interfering%20materials&amp;author=A.%20Bensadoun&amp;author=D.%20Weinstein&amp;publication_year=1976&amp;journal=Anal.%20Biochem.&amp;volume=70&amp;pages=241-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0003-2697(76)80064-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0003-2697(76)80064-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0003-2697(76)80064-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1259145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Reidegeld</div>
               <div class="given-names">K.A.</div>
            </div>, <div class="name string-name"><div class="surname">Eisenacher</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kohl</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">An easy-to-use Decoy Database Builder software tool, implementing different decoy strategies for false discovery rate calculation in automated MS/MS protein identifications</div>, <div class="source">Proteomics</div> , <div class="volume">8</div>, <div class="fpage">1129</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20easy-to-use%20Decoy%20Database%20Builder%20software%20tool%2C%20implementing%20different%20decoy%20strategies%20for%20false%20discovery%20rate%20calculation%20in%20automated%20MS%2FMS%20protein%20identifications&amp;author=K.A.%20Reidegeld&amp;author=M.%20Eisenacher&amp;author=M.%20Kohl&amp;publication_year=2008&amp;journal=Proteomics&amp;volume=8&amp;pages=1129-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18338823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Elias</div>
               <div class="given-names">J.E.</div>
            </div>, <div class="name string-name"><div class="surname">Gygi</div> <div class="given-names">S.P.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry</div>, <div class="source">Nat. Methods</div> , <div class="volume">4</div>, <div class="fpage">207</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target-decoy%20search%20strategy%20for%20increased%20confidence%20in%20large-scale%20protein%20identifications%20by%20mass%20spectrometry&amp;author=J.E.%20Elias&amp;author=S.P.%20Gygi&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=207-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17327847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ow</div>
               <div class="given-names">S.Y.</div>
            </div>, <div class="name string-name"><div class="surname">Salim</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Noirel</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Evans</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Rehman</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">P.C.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">iTRAQ underestimation in simple and complex mixtures: “the good, the bad and the ugly”</div>, <div class="source">J. Proteome Res.</div> , <div class="volume">8</div>, <div class="fpage">5347</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=iTRAQ%20underestimation%20in%20simple%20and%20complex%20mixtures%3A%20%E2%80%9Cthe%20good%2C%20the%20bad%20and%20the%20ugly%E2%80%9D&amp;author=S.Y.%20Ow&amp;author=M.%20Salim&amp;author=J.%20Noirel&amp;author=C.%20Evans&amp;author=I.%20Rehman&amp;author=P.C.%20Wright&amp;publication_year=2009&amp;journal=J.%20Proteome%20Res.&amp;volume=8&amp;pages=5347-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/pr900634c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fpr900634c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fpr900634c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19754192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pham</div>
               <div class="given-names">T.K.</div>
            </div>, <div class="name string-name"><div class="surname">Roy</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Noirel</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Douglas</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Stafford</div> <div class="given-names">G.P.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">A quantitative proteomic analysis of biofilm adaptation by the periodontal pathogen Tannerella forsythia</div>, <div class="source">Proteomics</div> , <div class="volume">10</div>, <div class="fpage">3130</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20quantitative%20proteomic%20analysis%20of%20biofilm%20adaptation%20by%20the%20periodontal%20pathogen%20Tannerella%20forsythia&amp;author=T.K.%20Pham&amp;author=S.%20Roy&amp;author=J.%20Noirel&amp;author=I.%20Douglas&amp;author=P.C.%20Wright&amp;author=G.P.%20Stafford&amp;publication_year=2010&amp;journal=Proteomics&amp;volume=10&amp;pages=3130-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pmic.200900448" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpmic.200900448" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpmic.200900448"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20806225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Matsunaga</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Takeyama</div> <div class="given-names">H.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Genetic engineering in marine cyanobacteria</div>, <div class="source">J. Appl. Phycol.</div> , <div class="volume">7</div>, <div class="fpage">77</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20engineering%20in%20marine%20cyanobacteria&amp;author=T.%20Matsunaga&amp;author=H.%20Takeyama&amp;publication_year=1995&amp;journal=J.%20Appl.%20Phycol.&amp;volume=7&amp;pages=77-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00003555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00003555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00003555"> </span></div> </div></div></div></div></div><div content-id="DSV024C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vioque</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2007</div>, <div class="article-title">Transformation of cyanobacteria</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">León</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Galván</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fernández</div> <div class="given-names">E.</div></div></div> (eds), Volume <div class="volume">616</div>, <div class="comment">Transgenic Microalgae as Green Cell Factories</div>, <div class="comment">Adv. Exp. Med. Biol.</div>, <div class="publisher-name">Springer</div>: <div class="publisher-loc">New York</div>, pp. <div class="fpage">12</div>–<div class="lpage">22</div>.</div></div></div></div><div content-id="DSV024C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">De Gelder</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Ponciano</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Joyce</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Top</div> <div class="given-names">E.M.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Stability of a promiscuous plasmid in different hosts: no guarantee for a long-term relationship</div>, <div class="source">Microbiology</div> , <div class="volume">153</div>, <div class="fpage">452</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stability%20of%20a%20promiscuous%20plasmid%20in%20different%20hosts%3A%20no%20guarantee%20for%20a%20long-term%20relationship&amp;author=L.%20De%20Gelder&amp;author=J.M.%20Ponciano&amp;author=P.%20Joyce&amp;author=E.M.%20Top&amp;publication_year=2007&amp;journal=Microbiology&amp;volume=153&amp;pages=452-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2006/001784-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2006%2F001784-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2006%2F001784-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17259616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Meyer</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">Replication and conjugative mobilization of broad host-range IncQ plasmids</div>, <div class="source">Plasmid</div> , <div class="volume">62</div>, <div class="fpage">57</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replication%20and%20conjugative%20mobilization%20of%20broad%20host-range%20IncQ%20plasmids&amp;author=R.%20Meyer&amp;publication_year=2009&amp;journal=Plasmid&amp;volume=62&amp;pages=57-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plasmid.2009.05.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plasmid.2009.05.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plasmid.2009.05.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19465049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Berla</div>
               <div class="given-names">B.M.</div>
            </div>, <div class="name string-name"><div class="surname">Pakrasi</div> <div class="given-names">H.B.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Up-regulation of plasmid-encoded genes during stationary phase in <em>Synechocystis</em> sp. PCC 6803, a cyanobacterium</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">78</div>, <div class="fpage">5448</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Up-regulation%20of%20plasmid-encoded%20genes%20during%20stationary%20phase%20in%20Synechocystis%20sp.%20PCC%206803%2C%20a%20cyanobacterium&amp;author=B.M.%20Berla&amp;author=H.B.%20Pakrasi&amp;publication_year=2012&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=78&amp;pages=5448-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01174-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01174-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01174-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22636001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Griese</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Lange</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Soppa</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Ploidy in cyanobacteria</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="volume">323</div>, <div class="fpage">124</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ploidy%20in%20cyanobacteria&amp;author=M.%20Griese&amp;author=C.%20Lange&amp;author=J.%20Soppa&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=323&amp;pages=124-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2011.323.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2011.323.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2011.323.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22092711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sergeyenko</div>
               <div class="given-names">T.V.</div>
            </div>, <div class="name string-name"><div class="surname">Los</div> <div class="given-names">D.A.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Identification of secreted proteins of the cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="volume">193</div>, <div class="fpage">213</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20secreted%20proteins%20of%20the%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=T.V.%20Sergeyenko&amp;author=D.A.%20Los&amp;publication_year=2000&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=193&amp;pages=213-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2000.193.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2000.193.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2000.193.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11111026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marchler-Bauer</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Chitsaz</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">CDD: specific functional annotation with the Conserved Domain Database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">37</div>, <div class="fpage">D205</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CDD%3A%20specific%20functional%20annotation%20with%20the%20Conserved%20Domain%20Database&amp;author=A.%20Marchler-Bauer&amp;author=J.B.%20Anderson&amp;author=F.%20Chitsaz&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D205-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18984618" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mitschke</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Georg</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">An experimentally anchored map of transcriptional start sites in the model cyanobacterium <em>Synechocystis</em> sp. PCC6803</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">2124</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20experimentally%20anchored%20map%20of%20transcriptional%20start%20sites%20in%20the%20model%20cyanobacterium%20Synechocystis%20sp.%20PCC6803&amp;author=J.%20Mitschke&amp;author=J.%20Georg&amp;author=I.%20Scholz&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=2124-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1015154108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1015154108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1015154108"> </span></div> </div></div></div></div></div><div content-id="DSV024C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kopf</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Klähn</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Matthiessen</div> <div class="given-names">J.K.F.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Voß</div> <div class="given-names">B.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Comparative analysis of the primary transcriptome of <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">527</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20primary%20transcriptome%20of%20Synechocystis%20sp.%20PCC%206803&amp;author=M.%20Kopf&amp;author=S.%20Kl%C3%A4hn&amp;author=I.%20Scholz&amp;author=J.K.F.%20Matthiessen&amp;author=W.R.%20Hess&amp;author=B.%20Vo%C3%9F&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=527-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24935866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ohkawa</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Price</div> <div class="given-names">G.D.</div></div>, <div class="name string-name"><div class="surname">Badger</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Mutation of <em>ndh</em> genes leads to inhibition of CO<sub>2</sub> uptake rather than HCO<sub>3</sub><sup>−</sup> uptake in <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">182</div>, <div class="fpage">2591</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20of%20ndh%20genes%20leads%20to%20inhibition%20of%20CO2%20uptake%20rather%20than%20HCO3%E2%88%92%20uptake%20in%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=H.%20Ohkawa&amp;author=G.D.%20Price&amp;author=M.R.%20Badger&amp;author=T.%20Ogawa&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=2591-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.9.2591-2596.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.9.2591-2596.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.9.2591-2596.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10762263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huckauf</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Nomura</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Forchhammer</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hagemann</div> <div class="given-names">M.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Stress responses of <em>Synechocystis</em> sp. strain PCC 6803 mutants impaired in genes encoding putative alternative sigma factors</div>, <div class="source">Microbiology</div> , <div class="volume">146</div>, <div class="fpage">2877</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress%20responses%20of%20Synechocystis%20sp.%20strain%20PCC%206803%20mutants%20impaired%20in%20genes%20encoding%20putative%20alternative%20sigma%20factors&amp;author=J.%20Huckauf&amp;author=C.%20Nomura&amp;author=K.%20Forchhammer&amp;author=M.%20Hagemann&amp;publication_year=2000&amp;journal=Microbiology&amp;volume=146&amp;pages=2877-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-146-11-2877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-146-11-2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-146-11-2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11065366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Lai</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Non-coding RNAs and their epigenetic regulatory mechanisms</div>, <div class="source">Biol. Cell</div> , <div class="volume">102</div>, <div class="fpage">645</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-coding%20RNAs%20and%20their%20epigenetic%20regulatory%20mechanisms&amp;author=H.%20Zhou&amp;author=H.%20Hu&amp;author=M.%20Lai&amp;publication_year=2010&amp;journal=Biol.%20Cell&amp;volume=102&amp;pages=645-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BC20100029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBC20100029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBC20100029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21077844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morris</div>
               <div class="given-names">K.V.</div>
            </div>
          </div>
          <div class="year">2012</div>, <div class="source">Non-coding RNAs and the epigenetic regulation of gene expression: drivers of natural selection</div> . <div class="publisher-name">Caister Academic Press</div>: <div class="publisher-loc">Norfolk, UK</div>.</div></div></div></div><div content-id="DSV024C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Git</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Dvinge</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Salmon-Divon</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Systematic comparison of microarray profiling, real-time PCR, and next-generation sequencing technologies for measuring differential microRNA expression</div>, <div class="source">RNA</div> , <div class="volume">16</div>, <div class="fpage">991</div>–<div class="lpage">1006</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20comparison%20of%20microarray%20profiling%2C%20real-time%20PCR%2C%20and%20next-generation%20sequencing%20technologies%20for%20measuring%20differential%20microRNA%20expression&amp;author=A.%20Git&amp;author=H.%20Dvinge&amp;author=M.%20Salmon-Divon&amp;publication_year=2010&amp;journal=RNA&amp;volume=16&amp;pages=991-1006" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.1947110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1947110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.1947110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20360395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ishii</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Hihara</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">An AbrB-like transcriptional regulator, Sll0822, is essential for the activation of nitrogen-regulated genes in <em>Synechocystis</em> sp. PCC 6803</div>, <div class="source">Plant Physiol.</div> , <div class="volume">148</div>, <div class="fpage">660</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20AbrB-like%20transcriptional%20regulator%2C%20Sll0822%2C%20is%20essential%20for%20the%20activation%20of%20nitrogen-regulated%20genes%20in%20Synechocystis%20sp.%20PCC%206803&amp;author=A.%20Ishii&amp;author=Y.%20Hihara&amp;publication_year=2008&amp;journal=Plant%20Physiol.&amp;volume=148&amp;pages=660-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.123505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.123505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.123505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18667724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Oliveira</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Lindblad</div> <div class="given-names">P.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">An AbrB-like protein regulates the expression of the bidirectional hydrogenase in <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">190</div>, <div class="fpage">1011</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20AbrB-like%20protein%20regulates%20the%20expression%20of%20the%20bidirectional%20hydrogenase%20in%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=P.%20Oliveira&amp;author=P.%20Lindblad&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=1011-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01605-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01605-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01605-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18039761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lopo</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Montagud</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Navarro</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Experimental and modeling analysis of <em>Synechocystis</em> sp. PCC 6803 growth</div>, <div class="source">J. Mol. Microbiol. Biotechnol.</div> , <div class="volume">22</div>, <div class="fpage">71</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20and%20modeling%20analysis%20of%20Synechocystis%20sp.%20PCC%206803%20growth&amp;author=M.%20Lopo&amp;author=A.%20Montagud&amp;author=E.%20Navarro&amp;publication_year=2012&amp;journal=J.%20Mol.%20Microbiol.%20Biotechnol.&amp;volume=22&amp;pages=71-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000336850" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000336850" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000336850"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22508451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Prakash</div>
               <div class="given-names">J.S.S.</div>
            </div>, <div class="name string-name"><div class="surname">Sinetova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zorina</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">DNA supercoiling regulates the stress-inducible expression of genes in the cyanobacterium <em>Synechocystis</em></div>, <div class="source">Mol. Biosyst.</div> , <div class="volume">5</div>, <div class="fpage">1904</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20supercoiling%20regulates%20the%20stress-inducible%20expression%20of%20genes%20in%20the%20cyanobacterium%20Synechocystis&amp;author=J.S.S.%20Prakash&amp;author=M.%20Sinetova&amp;author=A.%20Zorina&amp;publication_year=2009&amp;journal=Mol.%20Biosyst.&amp;volume=5&amp;pages=1904-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/b903022k" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fb903022k" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fb903022k"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19763333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV024C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV024C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fulda</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Mikkat</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Proteome analysis of salt stress response in the cyanobacterium <em>Synechocystis</em> sp. strain PCC 6803</div>, <div class="source">Proteomics</div> , <div class="volume">6</div>, <div class="fpage">2733</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20analysis%20of%20salt%20stress%20response%20in%20the%20cyanobacterium%20Synechocystis%20sp.%20strain%20PCC%206803&amp;author=S.%20Fulda&amp;author=S.%20Mikkat&amp;author=F.%20Huang&amp;publication_year=2006&amp;journal=Proteomics&amp;volume=6&amp;pages=2733-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16572470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462458&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv024/1/dsv024_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lfEIsMhkO8yiQ5Qm8bsI~gpfHMlNCya1ct22-8lNxR2YgxPbwqT1zp8QQzAzNl1qZuSJw1CaQEFdbgIRfTynZDzkh2jsQfVaknibuAHzeUsDMj~oQh57cwbdpM7cOLqJzKyO6NIRcg6WkDX9~kRf0Fqnu84lIsO3Ib-OhQPdqdqcCawXaTXivK6llUz9HckKEnXMMfI3r~0B6wCfHjdUrAUKbBCIeSb6d3vbs1ZRP42iFtUNKfYy76-iuiCRFPbF5Um03KBHR2rcLZ~-c~5IZ9m0UfRTMszQHGgAT~Y5ukg54wHBLwWHVrCGuAXnILcudKl~sb0mY8GRwXiEpA0jDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">544</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">10</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv024"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462458"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f425%2f2462458">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (10)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F425%2F2462458" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462458'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Improving a Synechocystis-based photoautotrophic chassis through systematic genome mapping and validation of neutral sites" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv024");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f425%2f2462458">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

