

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/2/63/376035","@type":"ScholarlyArticle","name":"Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs","datePublished":"2008-02-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2008-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm036","keywords":["alternative splicing","transcription initiation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/2/63/376035","author":[{"name":"Chern, Tzu-Ming","affiliation":"Division of Bioinformatics ,  Biozentrum ,  University of Basel ,  Klingelbergstrasse 50-70, Basel CH-4056 ,  Switzerland","@type":"Person"},{"name":"Paul, Nicodeme","affiliation":"Division of Bioinformatics ,  Biozentrum ,  University of Basel ,  Klingelbergstrasse 50-70, Basel CH-4056 ,  Switzerland","@type":"Person"},{"name":"van Nimwegen, Erik","affiliation":"Division of Bioinformatics ,  Biozentrum ,  University of Basel ,  Klingelbergstrasse 50-70, Basel CH-4056 ,  Switzerland","@type":"Person"},{"name":"Zavolan, Mihaela","affiliation":"Division of Bioinformatics ,  Biozentrum ,  University of Basel ,  Klingelbergstrasse 50-70, Basel CH-4056 ,  Switzerland","@type":"Person"}],"description":"Abstract.  High-throughput sequencing studies revealed that the majority of human and mouse multi-exon genes have multiple splice forms. High-density oligonucle","pageStart":"63","pageEnd":"72","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs" />
    <meta property="og:description" content="Abstract.  High-throughput sequencing studies revealed that the majority of human and mouse multi-exon genes have multiple splice forms. High-density oligonucle" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/2/63/376035" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-02-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chern, Tzu-Ming" /><meta name="citation_author_institution" content="Division of Bioinformatics, Biozentrum, University of Basel, Klingelbergstrasse 50-70, Basel CH-4056, Switzerland" /><meta name="citation_author" content="Paul, Nicodeme" /><meta name="citation_author_institution" content="Division of Bioinformatics, Biozentrum, University of Basel, Klingelbergstrasse 50-70, Basel CH-4056, Switzerland" /><meta name="citation_author" content="van Nimwegen, Erik" /><meta name="citation_author_institution" content="Division of Bioinformatics, Biozentrum, University of Basel, Klingelbergstrasse 50-70, Basel CH-4056, Switzerland" /><meta name="citation_author" content="Zavolan, Mihaela" /><meta name="citation_author_institution" content="Division of Bioinformatics, Biozentrum, University of Basel, Klingelbergstrasse 50-70, Basel CH-4056, Switzerland" /><meta name="citation_title" content="Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs" /><meta name="citation_firstpage" content="63" /><meta name="citation_lastpage" content="72" /><meta name="citation_doi" content="10.1093/dnares/dsm036" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2008/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Impact of alternative initiation, splicing, and termination on the diversity of the mRNA transcripts encoded by the mouse transcriptome; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1290-1300; " /><meta name="citation_reference" content="citation_title=Transcriptome and genome conservation of alternative splicing events in humans and mice; Pac. Symp. Biocomput.;  citation_year=2004;  citation_pages=66-77; " /><meta name="citation_reference" content="citation_title=Exon recognition in vertebrate splicing; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=2411-2414; " /><meta name="citation_reference" content="citation_title=An alternative-exon database and its statistical analysis; DNA Cell Biol.;  citation_year=2000;  citation_volume=19;  citation_pages=739-756; " /><meta name="citation_reference" content="citation_title=Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5053-5062; " /><meta name="citation_reference" content="citation_title=Intronic sequences flanking alternatively spliced exons are conserved between human and mouse; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=1631-1637; " /><meta name="citation_reference" content="citation_title=Sequence conservation, relative isoform frequencies, and nonsense-mediated decay in evolutionarily conserved alternative splicing; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=12813-12818; " /><meta name="citation_reference" content="citation_title=Differentiated evolutionary rates in alternative exons and the implications for splicing regulation; BMC Evol. Biol.;  citation_year=2006;  citation_volume=6;  citation_pages=50" /><meta name="citation_reference" content="citation_title=Promoter usage and alternative splicing; Curr. Opin. Cell. Biol.;  citation_year=2005;  citation_volume=17;  citation_pages=262-268; " /><meta name="citation_reference" content="citation_title=Chromatin, transcript elongation and alternative splicing; Nat. Struct. Mol. Biol.;  citation_year=2006;  citation_volume=13;  citation_pages=5-7; " /><meta name="citation_reference" content="citation_title=Transcriptional activators differ in their abilities to control alternative splicing; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=43110-43114; " /><meta name="citation_reference" content="citation_title=Alternative splicing of the fibronectin EIIIB exon depends on specific TGCATG repeats; Mol. Cell. Biol.;  citation_year=1998;  citation_volume=18;  citation_pages=3900-3906; " /><meta name="citation_reference" content="citation_title=Direct coupling of transcription and mRNA processing through the thermogenic coactivator PGC − 1; Mol. Cell;  citation_year=2000;  citation_volume=6;  citation_pages=307-316; " /><meta name="citation_reference" content="citation_title=Promoter architecture modulates CFTR exon 9 skipping; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=1511-1517; " /><meta name="citation_reference" content="citation_title=Coordinate regulation of transcription and splicing by steroid receptor coregulators; Science;  citation_year=2002;  citation_volume=298;  citation_pages=416-419; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the mammalian genome; Science;  citation_year=2005;  citation_volume=309;  citation_pages=1559-1563; " /><meta name="citation_reference" content="citation_title=Integrative annotation of 21,037 human genes validated by full-length cDNA clones; PLoS Biol.;  citation_year=2004;  citation_volume=2;  citation_pages=856-875; " /><meta name="citation_reference" content="citation_title=A simple physical model predicts small exon length variations; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=e45" /><meta name="citation_reference" content="citation_title=SPA: a probabilistic algorithm for spliced alignment; PLoS Genet.;  citation_year=2006;  citation_volume=2;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=Splice variation in mouse full-length cDNAs identified by mapping to the mouse genome; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1377-1385; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of mammalian promoter architecture and evolution; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=626-635; " /><meta name="citation_reference" content="citation_title=CAGE: cap analysis of gene expression; Nat. Methods.;  citation_year=2006;  citation_volume=3;  citation_pages=211-222; " /><meta name="citation_reference" content="citation_title=Alternative splicing in the human, mouse and rat genomes is associated with an increased frequency of exon creation and/or loss; Nat. Genet.;  citation_year=2003;  citation_volume=34;  citation_pages=177-180; " /><meta name="citation_reference" content="citation_title=RNA splice junctions of different classes of eukaryotes: sequence statistics and functional implications in gene expression; Nucletic. Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=7155-7174; " /><meta name="citation_reference" content="citation_title=Promoter-proximal pausing of RNA polymerase II defines a general rate-limiting step after transcription initiation; Genes Dev.;  citation_year=1995;  citation_volume=9;  citation_pages=559-572; " /><meta name="citation_reference" content="citation_title=The human SWI/SNF subunit Brm is a regulator of alternative splicing; Nat. Struct. Mol. Biol.;  citation_year=2006;  citation_volume=13;  citation_pages=22-29; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/2/63/1068923/dsm036.pdf" /><meta name="description" content="Abstract.  High-throughput sequencing studies revealed that the majority of human and mouse multi-exon genes have multiple splice forms. High-density oligonucle" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/2/63/376035" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/2/63/376035" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/2/63/376035");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm036';
    </script>
<title>
	Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./376035" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYo9PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVybiwgVHp1LU1pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9pbmZvcm1hdGljcywgQmlvemVudHJ1bSwgVW5pdmVyc2l0eSBvZiBCYXNlbCwgS2xpbmdlbGJlcmdzdHJhc3NlIDUwLTcwLCBCYXNlbCBDSC00MDU2LCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhdWwsIE5pY29kZW1lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgQmlvaW5mb3JtYXRpY3MsIEJpb3plbnRydW0sIFVuaXZlcnNpdHkgb2YgQmFzZWwsIEtsaW5nZWxiZXJnc3RyYXNzZSA1MC03MCwgQmFzZWwgQ0gtNDA1NiwgU3dpdHplcmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJ2YW4gTmltd2VnZW4sIEVyaWsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9pbmZvcm1hdGljcywgQmlvemVudHJ1bSwgVW5pdmVyc2l0eSBvZiBCYXNlbCwgS2xpbmdlbGJlcmdzdHJhc3NlIDUwLTcwLCBCYXNlbCBDSC00MDU2LCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilphdm9sYW4sIE1paGFlbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9pbmZvcm1hdGljcywgQmlvemVudHJ1bSwgVW5pdmVyc2l0eSBvZiBCYXNlbCwgS2xpbmdlbGJlcmdzdHJhc3NlIDUwLTcwLCBCYXNlbCBDSC00MDU2LCBTd2l0emVybGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcHV0YXRpb25hbCBBbmFseXNpcyBvZiBGdWxsLWxlbmd0aCBjRE5BcyBSZXZlYWxzIEZyZXF1ZW50IENvdXBsaW5nIEJldHdlZW4gVHJhbnNjcmlwdGlvbmFsIGFuZCBTcGxpY2luZyBQcm9ncmFtcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wYWN0IG9mIGFsdGVybmF0aXZlIGluaXRpYXRpb24sIHNwbGljaW5nLCBhbmQgdGVybWluYXRpb24gb24gdGhlIGRpdmVyc2l0eSBvZiB0aGUgbVJOQSB0cmFuc2NyaXB0cyBlbmNvZGVkIGJ5IHRoZSBtb3VzZSB0cmFuc2NyaXB0b21lOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEyOTAtMTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmQgZ2Vub21lIGNvbnNlcnZhdGlvbiBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBldmVudHMgaW4gaHVtYW5zIGFuZCBtaWNlOyBQYWMuIFN5bXAuIEJpb2NvbXB1dC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl9wYWdlcz02Ni03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhvbiByZWNvZ25pdGlvbiBpbiB2ZXJ0ZWJyYXRlIHNwbGljaW5nOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0yNDExLTI0MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFsdGVybmF0aXZlLWV4b24gZGF0YWJhc2UgYW5kIGl0cyBzdGF0aXN0aWNhbCBhbmFseXNpczsgRE5BIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03MzktNzU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXN0cmlidXRpb24gb2YgU1IgcHJvdGVpbiBleG9uaWMgc3BsaWNpbmcgZW5oYW5jZXIgbW90aWZzIGluIGh1bWFuIHByb3RlaW4tY29kaW5nIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz01MDUzLTUwNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbmljIHNlcXVlbmNlcyBmbGFua2luZyBhbHRlcm5hdGl2ZWx5IHNwbGljZWQgZXhvbnMgYXJlIGNvbnNlcnZlZCBiZXR3ZWVuIGh1bWFuIGFuZCBtb3VzZTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0xNjMxLTE2Mzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGNvbnNlcnZhdGlvbiwgcmVsYXRpdmUgaXNvZm9ybSBmcmVxdWVuY2llcywgYW5kIG5vbnNlbnNlLW1lZGlhdGVkIGRlY2F5IGluIGV2b2x1dGlvbmFyaWx5IGNvbnNlcnZlZCBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMjgxMy0xMjgxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWF0ZWQgZXZvbHV0aW9uYXJ5IHJhdGVzIGluIGFsdGVybmF0aXZlIGV4b25zIGFuZCB0aGUgaW1wbGljYXRpb25zIGZvciBzcGxpY2luZyByZWd1bGF0aW9uOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvbW90ZXIgdXNhZ2UgYW5kIGFsdGVybmF0aXZlIHNwbGljaW5nOyBDdXJyLiBPcGluLiBDZWxsLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI2Mi0yNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tYXRpbiwgdHJhbnNjcmlwdCBlbG9uZ2F0aW9uIGFuZCBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgTmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCBhY3RpdmF0b3JzIGRpZmZlciBpbiB0aGVpciBhYmlsaXRpZXMgdG8gY29udHJvbCBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9NDMxMTAtNDMxMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIG9mIHRoZSBmaWJyb25lY3RpbiBFSUlJQiBleG9uIGRlcGVuZHMgb24gc3BlY2lmaWMgVEdDQVRHIHJlcGVhdHM7IE1vbC4gQ2VsbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zOTAwLTM5MDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpcmVjdCBjb3VwbGluZyBvZiB0cmFuc2NyaXB0aW9uIGFuZCBtUk5BIHByb2Nlc3NpbmcgdGhyb3VnaCB0aGUgdGhlcm1vZ2VuaWMgY29hY3RpdmF0b3IgUEdDIOKIkiAxOyBNb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0zMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb21vdGVyIGFyY2hpdGVjdHVyZSBtb2R1bGF0ZXMgQ0ZUUiBleG9uIDkgc2tpcHBpbmc7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI3ODsgIGNpdGF0aW9uX3BhZ2VzPTE1MTEtMTUxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29vcmRpbmF0ZSByZWd1bGF0aW9uIG9mIHRyYW5zY3JpcHRpb24gYW5kIHNwbGljaW5nIGJ5IHN0ZXJvaWQgcmVjZXB0b3IgY29yZWd1bGF0b3JzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5ODsgIGNpdGF0aW9uX3BhZ2VzPTQxNi00MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc2NyaXB0aW9uYWwgbGFuZHNjYXBlIG9mIHRoZSBtYW1tYWxpYW4gZ2Vub21lOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwOTsgIGNpdGF0aW9uX3BhZ2VzPTE1NTktMTU2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpdmUgYW5ub3RhdGlvbiBvZiAyMSwwMzcgaHVtYW4gZ2VuZXMgdmFsaWRhdGVkIGJ5IGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmVzOyBQTG9TIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz04NTYtODc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBwaHlzaWNhbCBtb2RlbCBwcmVkaWN0cyBzbWFsbCBleG9uIGxlbmd0aCB2YXJpYXRpb25zOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1BBOiBhIHByb2JhYmlsaXN0aWMgYWxnb3JpdGhtIGZvciBzcGxpY2VkIGFsaWdubWVudDsgUExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWUyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljZSB2YXJpYXRpb24gaW4gbW91c2UgZnVsbC1sZW5ndGggY0ROQXMgaWRlbnRpZmllZCBieSBtYXBwaW5nIHRvIHRoZSBtb3VzZSBnZW5vbWU7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTM3Ny0xMzg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBtYW1tYWxpYW4gcHJvbW90ZXIgYXJjaGl0ZWN0dXJlIGFuZCBldm9sdXRpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NjI2LTYzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FHRTogY2FwIGFuYWx5c2lzIG9mIGdlbmUgZXhwcmVzc2lvbjsgTmF0LiBNZXRob2RzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjExLTIyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gdGhlIGh1bWFuLCBtb3VzZSBhbmQgcmF0IGdlbm9tZXMgaXMgYXNzb2NpYXRlZCB3aXRoIGFuIGluY3JlYXNlZCBmcmVxdWVuY3kgb2YgZXhvbiBjcmVhdGlvbiBhbmQvb3IgbG9zczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xNzctMTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgc3BsaWNlIGp1bmN0aW9ucyBvZiBkaWZmZXJlbnQgY2xhc3NlcyBvZiBldWthcnlvdGVzOiBzZXF1ZW5jZSBzdGF0aXN0aWNzIGFuZCBmdW5jdGlvbmFsIGltcGxpY2F0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb247IE51Y2xldGljLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NzE1NS03MTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9tb3Rlci1wcm94aW1hbCBwYXVzaW5nIG9mIFJOQSBwb2x5bWVyYXNlIElJIGRlZmluZXMgYSBnZW5lcmFsIHJhdGUtbGltaXRpbmcgc3RlcCBhZnRlciB0cmFuc2NyaXB0aW9uIGluaXRpYXRpb247IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTU1OS01NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBTV0kvU05GIHN1YnVuaXQgQnJtIGlzIGEgcmVndWxhdG9yIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nOyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIyLTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvMi82My8xMDY4OTIzL2RzbTAzNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgc3R1ZGllcyByZXZlYWxlZCB0aGF0IHRoZSBtYWpvcml0eSBvZiBodW1hbiBhbmQgbW91c2UgbXVsdGktZXhvbiBnZW5lcyBoYXZlIG11bHRpcGxlIHNwbGljZSBmb3Jtcy4gSGlnaC1kZW5zaXR5IG9saWdvbnVjbGUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM3NjAzNSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSj/Ns8x8LnuJs1ngjnH93mxYATjQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f63%2f376035" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f63%2f376035" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14648" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/2/1/m_cover.gif?Expires=2147483647&amp;Signature=jBzuI8dgooxO0nphGYu-tLH3TVo76XEeD81zZx-MgIsc2SQI98l6oXzUAOw3kAD0bJpcrcMkkhnYStSW5y2~q3JOlV2FttMiyv5m3clGsX95PYDI2~DoNsvkLoRmEdsJYZZj35inOV7PJt-QglYpSkDoNbvcuIEdixUgHJzSXLBAk-KJUxG32VmFxjy39-b9cjvb0fhX0gj9pzU7vQN1FN-kQBI9WHEa9~EK5xrswQznL63VawmvSxjXeLsbCouDExdjhZi4WDyg0uzsPm-FOpjC88OTsUIlrXJa8kWoXjZuZoDssNSp7qioKcDPPKPSqSy1pVgX8ijDxe8vBRvHzw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/2">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5474871">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474871">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474873">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474873">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474876">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474876">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474908">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474908">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474934">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474934">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474936">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474936">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5474938">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5474938">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/15/2/73/376500">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tzu-Ming Chern</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Division of Bioinformatics</div>, <div class="addr-line">Biozentrum</div>, <div class="insititution">University of Basel</div>, <div class="addr-line">Klingelbergstrasse 50-70, Basel CH-4056</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tzu-Ming+Chern">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chern T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chern T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tzu-Ming Chern</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicodeme Paul</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Division of Bioinformatics</div>, <div class="addr-line">Biozentrum</div>, <div class="insititution">University of Basel</div>, <div class="addr-line">Klingelbergstrasse 50-70, Basel CH-4056</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicodeme+Paul">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paul N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paul N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicodeme Paul</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erik van Nimwegen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Division of Bioinformatics</div>, <div class="addr-line">Biozentrum</div>, <div class="insititution">University of Basel</div>, <div class="addr-line">Klingelbergstrasse 50-70, Basel CH-4056</div>, <div class="country">Switzerland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erik+van+Nimwegen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=van Nimwegen E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;van Nimwegen E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Erik van Nimwegen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mihaela Zavolan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Division of Bioinformatics</div>, <div class="addr-line">Biozentrum</div>, <div class="insititution">University of Basel</div>, <div class="addr-line">Klingelbergstrasse 50-70, Basel CH-4056</div>, <div class="country">Switzerland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +41 61-267-1576. Fax. <div class="fax">+41 61-267-1584</div>. E-mail: <a href="mailto:mihaela.zavolan@unibas.ch" target="_blank">mihaela.zavolan@unibas.ch</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mihaela+Zavolan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zavolan M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zavolan M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mihaela Zavolan</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 63–72, <a href='https://doi.org/10.1093/dnares/dsm036'>https://doi.org/10.1093/dnares/dsm036</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 February 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 August 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">25 December 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="376035" href="/dnaresearch/article-pdf/15/2/63/1068923/dsm036.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Tzu-Ming Chern, Nicodeme Paul, Erik van Nimwegen, Mihaela Zavolan;  Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs, <em>DNA Research</em>, Volume 15, Issue 2, 1 April 2008, Pages 63–72, <a href='https://doi.org/10.1093/dnares/dsm036'>https://doi.org/10.1093/dnares/dsm036</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/376035?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/376035?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Computational%20Analysis%20of%20Full-length%20cDNAs%20Reveals%20Frequent%20Coupling%20Between%20Transcriptional%20and%20Splicing%20Programs&amp;publicationDate=2008-02-14&amp;volumeNum=15&amp;issueNum=2&amp;author=Chern%2C%20Tzu-Ming%3B%20Paul%2C%20Nicodeme&amp;startPage=63&amp;endPage=72&amp;contentId=10.1093%2Fdnares%2Fdsm036&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f63%2f376035" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5474871 id="5474871" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>High-throughput sequencing studies revealed that the majority of human and mouse multi-exon genes have multiple splice forms. High-density oligonucleotide array-based measurements have further established that many exons are expressed in a tissue-specific manner. The mechanisms underlying the tissue-dependent expression of most alternative exons remain, however, to be understood. In this study, we focus on one possible mechanism, namely the coupling of (tissue specific) transcription regulation with alternative splicing. We analyzed the FANTOM3 and H-Invitational datasets of full-length mouse and human cDNAs, respectively, and found that in transcription units with multiple start sites, the inclusion of at least 15% and possibly up to 30% of the ‘cassette’ exons correlates with the use of specific transcription start sites (TSS). The vast majority of TSS-associated exons are conserved between human and mouse, yet the conservation is weaker when compared with TSS-independent exons. Additionally, the currently available data only support a weak correlation between the probabilities of TSS association of orthologous exons. Our analysis thus suggests frequent coupling of transcriptional and splicing programs, and provides a large dataset of exons on which the molecular basis of this coupling can be further studied.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription initiation&quot;">transcription initiation</a></div>                            <h2 scrollto-destination=5474873 id="5474873" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The most common form of splice variation is the inclusion of an exon in some, but not all, of the transcripts of a gene.<sup><a reveal-id="DSM036C1" data-open="DSM036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM036C2" data-open="DSM036C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Numerous studies have been dedicated to specific instances of such exons, which are known by various names such as ‘cassette’, ‘alternative’, ‘skipped’, and ‘cryptic’ exons. The regulatory signals leading to the inclusion or exclusion of a cassette exon also form a vast topic of research. Computational studies are converging toward the view that cassette exons are generally less recognizable to the splicing machinery than constitutive exons due to their shorter length,<sup><a reveal-id="DSM036C3" data-open="DSM036C3" class="link link-ref link-reveal xref-bibr">3</a></sup> lower strength of splice sites,<sup><a reveal-id="DSM036C4" data-open="DSM036C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and poor representation of general splice enhancers,<sup><a reveal-id="DSM036C1" data-open="DSM036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM036C5" data-open="DSM036C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The tissue-specific inclusion of these exons appears to be dependent upon specific regulatory elements, at least some of which are located in the strongly conserved intronic regions that flank the cassette exons.<sup><a reveal-id="DSM036C2" data-open="DSM036C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSM036C6  DSM036C8" data-open="DSM036C6  DSM036C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup></p><p>One attractive hypothesis concerning the mechanism of tissue-specific inclusion of cassette exons involves the direct coupling between tissue-dependent transcription and splicing. It has been shown, for instance, that the promoter from which transcription is initiated can affect the inclusion of downstream exons through the recruitment of transcription factors and co-activators that modulate the elongation rate of RNA polymerase II<sup><a reveal-id="DSM036C9" data-open="DSM036C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSM036C10" data-open="DSM036C10" class="link link-ref link-reveal xref-bibr">10</a></sup> (kinetic model). In turn, a low polymerase elongation rate can promote the inclusion of some exons that are skipped when the elongation rate is high, as has been shown for the fibronectin EDI exon.<sup><a reveal-id="DSM036C11" data-open="DSM036C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Alternatively, protein factors that are recruited at the stage of transcription initiation may interact with splicing factors (recruitment model). For instance, the inclusion of the fibronectin EDII exon, normally promoted by the SRp40 protein,<sup><a reveal-id="DSM036C12" data-open="DSM036C12" class="link link-ref link-reveal xref-bibr">12</a></sup> is inhibited when transcription is initiated at a promoter containing a binding site for the PPARγ transcription factor. This is because the PPARγ transcription factor recruits an SRp40 inhibitor, the PGC-1 co-activator.<sup><a reveal-id="DSM036C13" data-open="DSM036C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The role of the promoter architecture on internal splicing has also been demonstrated for the cystic fibrosis transmembrane regulator<sup><a reveal-id="DSM036C14" data-open="DSM036C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and for the steroid-sensitive genes.<sup><a reveal-id="DSM036C15" data-open="DSM036C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Such studies have been limited, however, to very few genes, prompting us to evaluate the extent to which transcriptional and splicing programs appear to be coordinated at the level of the whole transcriptome.</p>                            <h2 scrollto-destination=5474876 id="5474876" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5474877 id="5474877" class="section-title" data-legacy-id=s2a>2.1. Transcript mapping and splice analysis</h3>
<p>The sequences used in the study consist of the 102 797 Fantom3 mouse cDNAs,<sup><a reveal-id="DSM036C16" data-open="DSM036C16" class="link link-ref link-reveal xref-bibr">16</a></sup> 52 070 mouse mRNAs from Genbank, and 167 992 human cDNAs from version 3.0 (<a class="link link-uri" href="http://www.h-invitational.jp/">http://www.h-invitational.jp/</a>) of the H-Invitational project.<sup><a reveal-id="DSM036C17" data-open="DSM036C17" class="link link-ref link-reveal xref-bibr">17</a></sup> We have mapped the mouse cDNAs to the mm7 assembly of the mouse genome and the human cDNAs to the hg18 assembly of the human genome, both available from the University of California at Santa Cruz.</p><p>For the identification of splice variants, we used the automated splicing analysis pipeline that we have previously developed.<sup><a reveal-id="DSM036C1" data-open="DSM036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM036C18" data-open="DSM036C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Briefly, we first mapped all cDNAs to their respective genome using our spliced alignment algorithm (SPA).<sup><a reveal-id="DSM036C19" data-open="DSM036C19" class="link link-ref link-reveal xref-bibr">19</a></sup> To avoid biases from transcripts that are badly mapped due to a high rate of sequencing errors or erroneous assembly, we select only those transcripts that have at least 75% of their nucleotides mapped to the genome, with at least 95% identity or less than ten mismatches in each exon. This procedure yielded 132 681 mouse and 110 978 human mapped transcripts, which we clustered such that the mapping of each transcript in a cluster shares at least one exonic nucleotide with at least one other transcript in the cluster.<sup><a reveal-id="DSM036C1" data-open="DSM036C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM036C18" data-open="DSM036C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM036C20" data-open="DSM036C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We obtained 42 407 mouse and 22 116 human clusters (transcription units) that we analyzed for splice variation. We were interested only in cassette exons with no other form of splice variation. We identified these as internal exons that were completely contained in an intron implied by the mapping of another transcript in the cluster, having the same splice boundaries in all transcripts in which they were included. Our final mouse dataset consisted of 29 416 transcripts and 4 964 internal cassette exons, and the human dataset of 79 030 transcripts and 11 664 internal cassette exons.</p>                            <h3 scrollto-destination=5474880 id="5474880" class="section-title" data-legacy-id=s2b>2.2. Quantifying the evidence for coupling between the choice of transcription start sites and the inclusion/exclusion of internal exons</h3>
<p>For each transcription unit, we first identified (1) the set of internal cassette exons and (2) the set of transcription start sites (TSSs). The cassette exon annotation was determined as outlined above. To identify different TSSs used within a transcription unit, we had to define precisely what we mean by a unique TSS. The analysis of mammalian TSSs by Carninci et al.<sup><a reveal-id="DSM036C21" data-open="DSM036C21" class="link link-ref link-reveal xref-bibr">21</a></sup> has shown that most TSSs show some amount of variability. Especially at TSSs located in CpG islands, one finds transcripts starting from many different nearby sites covering tens and sometimes hundreds of nucleotides. We, therefore, needed to group transcripts whose apparent start sites were ‘near’ each other and then identify different TSSs with the different clusters of apparent start sites. We decided to take a conservative approach to this clustering of apparent start sites in a transcription unit by considering all transcripts that started within the same exon to derive from the same TSS. That is, in our analysis different TSSs correspond to <em>different initial exons</em> in the transcripts.</p><p>In addition, we tested the validity of our results, on a separate dataset in which we use only transcripts whose initial exons were confirmed by CAGE tag data.<sup><a reveal-id="DSM036C22" data-open="DSM036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In the latter case, the initial exon was considered confirmed as a TSS if one or more CAGE tags were found within 100 bp of the start of the exon in the genome. Since the results did not change, and the requirement of CAGE validation of TSSs reduced the size of our data-set significantly, we did not use the CAGE validated TSSs further.</p><p>For each internal cassette exon, we collected all transcripts that could have included the exon as an internal exon, i.e. those transcripts that contained exons both upstream and downstream of the genomic location of the exon in question, and determined the TSS that was used for each of these transcripts. We thus obtained a list of TSSs that were used in the set of transcripts in which the cassette exon could have been included. For further analyses, we kept only cassette exons for which multiple TSSs were identified. We then counted, for each TSS in the list, how many transcripts starting from this TSS included the exon, and how many transcripts excluded the exon. For each internal exon, we thus obtained counts of the number of times each TSS was used in a transcript whose locus covered the exon, and the number of times the exon was included and excluded with each of the TSSs.</p><div class="block-child-p">To identify exons whose inclusion depends on which TSS was used, we used a Bayesian model selection procedure that compared the probabilities of the observed counts under a TSS-independent model and a TSS-dependent model. Considering a particular cassette exon, let <em>t</em><sub><em>i</em></sub> denote the total number of times TSS <em>i</em> was used, <em>n</em><sub><em>i</em></sub> the number of times the exon was included when TSS <em>i</em> was used, <em>t</em> the total number of transcripts, and <em>n</em> the total number of times that the exon was included. For the independent model, we assumed that the <em>n</em> inclusions are distributed at random among the <em>t</em> transcripts. Under this model, the probability of the observed counts {<em>n</em><sub><em>i</em></sub>}, given the counts {<em>t</em><sub><em>i</em></sub>}, and <em>n</em> is <a id="DSM036M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm03607.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03607.gif?Expires=2147483647&amp;Signature=kI6zHwqwiqhHxZ6Zys0KquTkfEz3cL6XpwEXBNE8o3Bmjj9lFvmzqvGatToOihfutHfQz-5GGRKqU5jMXRFwOoMIoKC3Oe5Ap8JSqLYNejVlnvZ2pS69q1u~mHOLocr-2MJeciI0AoplqXKIpORZ1FdBCbjnu2auS6LPqxtY6PJ3TfxI2V7wTQudKwnSz8dX9oDgIG8MTS~lnMAOV~TSjpFt1CeSIahwsUsWZZd5axlemJBxgkSpl-hepCm15v9kbtyyzlW9ePlKel2RJCR6Tf7nOig-DMBBWNRiIoPUD-khHITj6X-ywyFs8tgTv5h0YoOwl20f9UFPbI9bP49h4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03607.gif" /></a></div></div><div class="label">1</div></div> For the dependent model, we assumed that the rates of inclusion and exclusion for the different TSSs are set by some unknown mechanism. Given our general ignorance about the mechanism or mechanisms determining these rates, there is no reason to assume that any set of counts {<em>n</em><sub><em>i</em></sub>} is more or less likely than any other set of counts. We, therefore, assumed that all possible counts {<em>n</em><sub><em>i</em></sub>} that are consistent with the totals {<em>t</em><sub><em>i</em></sub>} and the total number of inclusions <em>n</em> are all equally likely, meaning that <a id="DSM036M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm03608.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03608.gif?Expires=2147483647&amp;Signature=oJ6z3qfhQyzoXb~tAHaF7YGrYVhn2AAhr1nPZed1YUdoVRqtKd7DRcjCK75Lh7MSX9E1LnUTY0-65zEb7YaxSk0oVi-uw66X~O4tCw4p8kMWg5FU9EOS4PWc8WONAnnXh-qD4fBvwJ0dVSnAjjKZrJJ-YxkTQ8DwUpBwEgd2RrhijyJTLOyOgRVmOcLE5jnS0aipU-DB5yJuFnZlx7SqAD98I49xdkbGP8hhOaBVtSb0k4EIqvKc5LJdyetvEH7rGL4t7dnEmIq7wdB21-1Z5XHNkPi1BXL8T6q1MYBncZp9-Lzy7cBLFqOlt9wfv~7yLxapdRnISpIxu-oLULzrdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03608.gif" /></a></div></div><div class="label">2</div></div> where <em>C</em>(<em>n</em>, {<em>t</em><sub><em>i</em></sub>}) is the total number of different sets of counts {<em>n</em><sub><em>i</em></sub>} that are possible, given the totals <em>n</em> and {<em>t</em><sub><em>i</em></sub>}. The total count numbers <em>C</em>(<em>n</em>, {<em>t</em><sub><em>i</em></sub>}) can be determined recursively. Let <em>C</em><sub><em>i</em></sub>(<em>r</em>) be the number of different inclusion counts <em>n</em><sub>1</sub> through <em>n</em><sub><em>i</em></sub> that can be assigned to TSSs 1 through <em>i</em>, such that <em>r</em> of the <em>n</em> total inclusions remain. We have the following recursion relation for <em>C</em><sub><em>i</em></sub>(<em>r</em>): <a id="DSM036M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm03609.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03609.gif?Expires=2147483647&amp;Signature=nGx07gU3p3qUXMTkZi-0YDe0SmLitmq6af7nQWpUivYEia23PhIQ19~raxxN0fS90b5HR9TUYW0ANEjlsn1-~sou1DzbwTB1HGfROqfmhoR8GYRQDQE-ecwD0lL5YbnO0gSrsZZ9cGYCnpRvvTYKPzG8lpy9ZXE3K1r0dtNPPDXDnW1YUPNePbhr4QvUkT0WBWP8qzgYqdxByIIrLlu19PdvNyF2gEUsSHH0pVhcWLK6QTMzN1kDQWxdCbtOVTEqH2Xqzd8TxubH3IZSi1PB~Vz-37jcoOZZjWJvdiR5YxxTOB1L2ge52okCe-MuIxDZKWy0elpSzBECy3HCSiDL7A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03609.gif" /></a></div></div><div class="label">3</div></div> We initialize the recursion by setting <a id="DSM036M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsm03610.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03610.gif?Expires=2147483647&amp;Signature=wthJ5U~qtNpfhQ9QGdw59A9alYlv9AlafZxVby-y5Cl1vnsqZMVcpfG-RsG~e07J0AYjyekpleqzGZeodkuIo6uc591X-JF7alY~44taNAzM25mFzUpyNblDjdNsVGwXYr69Qv6H7e31pwUKk~AinhJkkcPXcUHLbI1m6nze2QBpoTWOEp-oy~jk~Mokq7LzVnIQe9ckJIV7wqN5QTQDpdYy80WDmw4HK1cwESKYyH2~9p~Hcs4kSdo5OZ5nN1xhWYrd0xuZhqgv~Q4tB1e1BUGjWCcHFuv0JI3WnnGm4bO7bCob-gbFPLI97BlXdYPtcLSl9Fyjo-H0EtsUF5Y2hA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03610.gif" /></a></div></div><div class="label">4</div></div> that is, before we assign counts to any TSS there have to be precisely <em>n</em> inclusions left. Once we arrive at the last (<em>p</em>th) TSS, our count <em>C</em>(<em>n</em>, {<em>t</em><sub><em>i</em></sub>}) is given by <em>C</em><sub><em>p</em></sub>(0), i.e. there should be no inclusions left.</div><div class="block-child-p">To estimate the total fraction <em>f</em> of cassette internal exons whose inclusion is dependent on TSSs, we calculated the probability <em>P</em>(<em>D</em>|<em>f</em>) of the data of all cassette exons assuming that a fraction <em>f</em> was dependent. Let <em>P</em><sub>indep</sub>(<em>k</em>) and <em>P</em><sub>dep</sub>(<em>k</em>) denote the probabilities of the counts for the <em>k</em>th cassette exon given the independent and dependent model, respectively. For each exon <em>k</em>, these quantities are computed according to Equations (<a href="#DSM036M1" class="sectionLink xref-sec">1</a>) and (<a href="#DSM036M2" class="sectionLink xref-sec">2</a>). We then have <a id="DSM036M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M5"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03611.gif?Expires=2147483647&Signature=bHXsYoxmgDK49dFZaCGTfhcuYMD44QcvmKXutO~X94XUAplMqKBUh4D4R-Y0skIsZEzU3xLP-B2zGFHfPC8T9ZN8QLYgHzwmmUAEBQU-Y4EIOV6MoE4Scdjb3spgpeYFSdEKfqWc5znxlHfJy6imWW8jmItqMlCVvpHVb1qjkJAddVj6R35iISPN1e1SjvTBbNYBnTOHFCxq-DDcYfUJTdXk1wWyAv6gh0P3msqrdh5cnd0qLLdJkWO9mAfPYCgp3axu2glAntdfo-JNTsTJLIN-twFvbXv8-bIZ~P5jcldltbKrf98cvbBVL~YT5-oA1lCBqKTsZkTv4QB13B4Wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm03611.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03611.gif?Expires=2147483647&Signature=Qb5vKjYwdGM1I~PfQd~uEyylDkRr2fDocv1ZazF0LpRpoLTKgtgJd-1hPH6UYak2S4vGWsqQoExqQtIAMJiioLGAfXHSwzBabzwPj88966fQWdLpTU0zTUX4BQvieqwaMCNHzr~wWlcU4r9DowElhSv~qmRu7O2AsKOzToWHfGUq9GSxj8AwupxOJvLrgooxJoJfIP8rGEGUGcYv4axURYHQI~saFVru90M1mLpPy8ULcvd61dXhi9tLAbC-MSB6oqcgRdYC~rSdW~RDScGcPRskMIRdVUl~H0cKwluoa6tNL3XmV4rWjrMcf0q6yAt8y9dF3znJMEtPatjPjD6aDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03611.gif" /></a></div></div><div class="label">5</div></div></div><div class="block-child-p">Using a uniform prior over <em>f</em> the posterior probability <em>P</em>(<em>f</em>|<em>D</em>) for the fraction of dependent exons given the data simply becomes <a id="DSM036M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M6"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03612.gif?Expires=2147483647&Signature=kG0MQlOB0TVBb56mM4lMXUos5jZOqYjWoMqccfKRF6o9AFUZ~KUiPX3OU~J7k2nNJT0hYVXJyHxeMVOOW8-ww5jpwCpogZdfAVEBsPcmD6ly3Lp2oht9eAhpuqqqK7egZ3NIr0Bam-4UxNNVZEH1lq4ObwbV7V5l~nY6osaqqpBukwn~XxAcouTrgTGn5NLTzka8QP8xZ2wlGgZqth1rTBRRFlHXPX7yDZg6RsN~Ql6vKQysiEicgIIFKONw~Uep024HL7iIp8UfOCqz2BmsQ3TOhtBBN6UhXv0Nv1oifUHmVzO42mPeXHreGmsUCvRHkWWwJboa9cC1p6nboZ7flQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm03612.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03612.gif?Expires=2147483647&Signature=IPx9JfzcKoNYtzW3DhpFXA3AO~U~8Uw2Ok~FJeaPHGLDkJ6IV7UscueCna1IrimavVqwuj1QoCHWNPBe-mTqha7OHMNCfaITLRH~JfkEObokq5oIx3wZ3DtQiJXY54tQjhQF1yP~lxmsItdIdEBC2RN5j5grcS0Bkks13DfDXSiHmQx2ARBDfzESNUbw7eI3j-krvJLDysRogph7J2Aj0kv~cWW4Qcl8FPzfXcoylHgRokptAeYNJzbrVxvSNqNo01WK2oZiTo9DqxuT3vK5bTV3lmrmA3ibzMqLYTwJzXmACDFhfhyYpuB77RF2UqPvW~W2alMmz~6Gv1H6mpd3sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03612.gif" /></a></div></div><div class="label">6</div></div></div><p>This distribution is shown as the solid line in Fig. <a reveal-id="DSM036F2" data-open="DSM036F2" class="link link-reveal link-table xref-fig">2</a>. We calculated the expectation value 〈<em>f</em>〉 = ∫ <em>f</em><em>P</em>(<em>f</em> | <em>D</em>) d<em>f</em> and using 〈<em>f</em>〉 as a prior probability of the independent model we computed, for each individual exon, the posterior probability that the inclusion of the exon is TSS associated.</p><p>We also generated a randomized dataset <em>D</em><sub>rand</sub> by, for each exon, randomly distributing the <em>n</em> inclusions of the exon among the different TSSs, in such a way that the total number of transcripts <em>t</em><sub><em>i</em></sub> for each TSS <em>i</em> stays the same. That is, the data <em>D</em><sub>rand</sub> were generated in accordance with the independent model, keeping the total inclusion counts <em>n</em>, and total TSS counts {<em>t</em><sub><em>i</em></sub>} of the real data. The distribution <em>P</em>(<em>f</em>|<em>D</em><sub>rand</sub>) is shown as the dotted line in Fig. <a reveal-id="DSM036F2" data-open="DSM036F2" class="link link-reveal link-table xref-fig">2</a>.</p>                            <h3 scrollto-destination=5474889 id="5474889" class="section-title" data-legacy-id=s2c>2.3. Extraction of constitutive exons</h3>
<p>We used our database to identify internal exons that were included in more than ten transcripts and did not have any splice variation. We obtained 5136 and 6377 such exons for mouse and human, respectively, and we used these as internal constitutive exons.</p>                            <h3 scrollto-destination=5474891 id="5474891" class="section-title" data-legacy-id=s2d>2.4. Computation of the exon conservation statistics</h3>
<p>We used the whole-genome alignments provided by the University of California, at Santa Cruz (<a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/mm7/vsHg18/axtNet/">http://hgdownload.cse.ucsc.edu/goldenPath/mm7/vsHg18/axtNet/</a> and <a class="link link-uri" href="http://hgdownload.cse.ucsc.edu/goldenPath/hg18/vsmm7/axtNet/">http://hgdownload.cse.ucsc.edu/goldenPath/hg18/vsmm7/axtNet/</a>) to extract alignments of mouse exons in our dataset with the corresponding orthologous regions from the human genome, and of human exons with the corresponding orthologous regions from the mouse genome. We found orthologous human regions for 438 of the 496 TSS-associated, 469 of the 496 TSS-independent, and 5122 of the 5136 constitutive internal mouse exons. Similarly, we found orthologous mouse regions for 1003 of the 1166 TSS-associated, 1078 of the 1166 TSS-independent, and 6344 of the 6377 constitutive internal human exons. On the basis of the extracted alignments, we determined the fraction of all mouse exon nucleotides that are perfectly conserved in human, and the fraction of human exon nucleotides that are conserved in mouse.</p><p>To compute the correlation between posterior probabilities of TSS association in human and mouse, we used the following procedure. We started with cassette exons from transcription units with multiple TSSs from human. On the basis of the whole-enome alignments, we found the orthologous mouse exons as described above, and then we intersected the coordinates of these orthologs with the coordinates of mouse cassette exons that were part of transcription units with multiple TSSs. This procedure gave us the list of orthologous cassette exons that were part of transcription units with multiple TSSs in both human and mouse. We checked that we obtain the same list of exons if we start from the mouse cassette exons and compute their human orthologs. We did not set a threshold of minimal conservation between orthologous exons, as the dataset is already relatively small, and our previous results were not sensitive to the precise threshold of conservation that we used.</p>                            <h3 scrollto-destination=5474897 id="5474897" class="section-title" data-legacy-id=s2e>2.5. Computation of the average distance to an inclusion-promoting TSS and to a skipping-promoting TSS</h3>
<p>We identified all transcripts in which a particular cassette exon was included, and for each of them, we determined the distance in the genome between the TSS and the start of the cassette exon. We then averaged these distances to obtain the average distance of the exon to an inclusion-promoting TSS. Similarly, we identified all transcripts in which a particular cassette exon was skipped, and we computed the average distance of the exon to a skipping-promoting TSS.</p>                            <h3 scrollto-destination=5474906 id="5474906" class="section-title" data-legacy-id=s2f>2.6. Analysis of 5′ EST data</h3>
<p>To address the issue of biases in the coverage of gene structures that may have been introduced by the selection of clones for full-length cDNA sequencing, we have performed the same analysis using only 5′ EST sequences. On the basis of the October 2007 UniGene database, we extracted 3 738 929 human 5′ EST sequences. We removed from this set those ESTs with low sequence quality (over 3% ambiguous nucleotides), we then trimmed the polyA tails and discarded those ESTs whose length was &lt;95 after polyA tail removal (using the trimpoly program of the SeqClean package from <a class="link link-uri" href="http://compbio.dfci.harvard.edu/tgi/software/">http://compbio.dfci.harvard.edu/tgi/software/</a>). We thus obtained 3 700 028 ESTs, which we mapped to the hg18 assembly of the human genome using our SPA program.<sup><a reveal-id="DSM036C19" data-open="DSM036C19" class="link link-ref link-reveal xref-bibr">19</a></sup> After mapping, we retained for further analysis only ESTs, which contained at least two exons, were mapped with overall 95% identity, and whose every exon was mapped with at least 98% identity to the human genome. This selection left 1 276 669 good quality ESTs, which we clustered based on exon overlap and annotated for splice variation as described in Section 2.1.</p>                            <h2 scrollto-destination=5474908 id="5474908" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5474909 id="5474909" class="section-title" data-legacy-id=s3a>3.1. The inclusion/skipping of internal exons is correlated with the usage of specific TSSs</h3>
<div class="block-child-p">We used a Bayesian approach to estimate the fraction <em>f</em> of internal cassette exons whose inclusion depends on the choice of TSS. We thus considered two models: the first assumes that the probability of exon inclusion is independent of the TSS used to transcribe the pre-mRNA, i.e. the probability of exon inclusion is the same for all TSSs, and the second assumes that for each TSS there is an independent probability of exon inclusion, which can be different between different TSSs. For a given <em>f</em>, we can write the probability of the data as <a id="DSM036M7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSM036M7"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03613.gif?Expires=2147483647&Signature=WF5372tN9x0~UB-6jYzdf0OVvGhve-FsRGO-YIlyzPwMPblBvX6HW1EjMdoXTr5lItL8HNRaP17ue9kp5xO8AqdBcIQ3KaVZGBsZe6rzrUbEg~tyf5~OnzcH6l5Eu4HmVjWhsEt1gYWFixQLH-HoVqBRtXTIAClZ42UcjiJtamnolVMsjwxe6Qho2WrsDvgMT7vBoCW1eadNaTEUsYt-Fn9S1qNS4ZknciuCpKAUVEiOV3z59sl6VawTFfbeqwfVZpx1a4P3KvsEGHHx6cnMEXrGYaAcsqHyEUQ2W8gGx7uCbdYNns-0FZpvjDqS7j7m6ymepLl6~Js1cVBK-uTv5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsm03613.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03613.gif?Expires=2147483647&Signature=NbVZ8h6iJwtj8QrMSPlaKjhhioQwdP1y8UTdq7FSSnDIEHYjtqbKQ0KkLdN1QeqVLOKmkDoxEdOGC7rbaVqJ3XZWKhlW1sX-sYtDrBmEn3bLsDf686yrvi8MqhPdOG0BpLpxENl8SLszSB3lNIJqRlZrv5R0Ta9O1U33Ux6jNH6un4OtGPHlzKVtPjDv5xvMm98eRIWlE3tM64PkuY~Dh4fqOl-fX8WrR~qcPX5ZyrhDXXkC-nO7AKTZGaFx4AqaF~XNhzsunFZfRiGt47Xdk3ykhYSsYP9rAioCDcFAP8MCHi2RdDWIHdO5dB3UX7ozVV1Daz7kw78JCz9hSHwxhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsm03613.gif" /></a></div></div><div class="label">7</div></div> with <em>P</em><sub>dep</sub>(<em>k</em>) and <em>P</em><sub>indep</sub>(<em>k</em>) being the probabilities of the data for cassette exon <em>k</em> under the dependent and independent models, respectively. In order to obtain these probabilities we collected, for each internal exon, the set of transcripts in the dataset that could have contained the exon, i.e. those transcripts that contain exons both upstream and downstream of the genomic location of the cassette exon in question. We divided this set of transcripts into groups that use the same TSSs, and counted the number of transcripts in which the exon was included, and the number of transcripts in which the exon was excluded in each TSS group. A specific example of this computation is shown in Fig. <a reveal-id="DSM036F1" data-open="DSM036F1" class="link link-reveal link-table xref-fig">1</a>.</div>                        <a id="5474911" scrollto-destination="5474911"></a>
<div id="DSM036F1" data-id="DSM036F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03601.jpeg?Expires=2147483647&Signature=XGrw~rn9digviLSF4V3zW~XGru7thNqPml8O3SgSgFXzPCpJo0~H~14FwNlUYzF6~wX07Zr12mCNWV6kC~ZxbFXbTp~Z4NdyOrNZ5bm79hwHGIbd~b~i6lJSovAzgmiy0RhzmygXr3ZimsurwcEFJfs69r7LLnm5RF0PjjrMl2EbvPxIPREajc7aRzEKYJzXQ7aV~zphluu8m8leV2JtGbSrI7YTn1IOJwR4X0EDTQREGCG76SGJDN3DU8N9PY9nyKcpOLy3LvajIJG2hf5appvgL3RxIwXyTg3IgbLOyDauaM7tsKxL2X293oe8xT8e2FxtHcInVpevqe8TdkxUGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs. Exons are indicated by (green) boxes and introns by red lines. Cassette exons are shown with a black frame. The internal cassette exon that has a high probability of TSS association is indicated by the thin black rectangle. The number of times a specific TSS-splicing pattern was observed in the data is indicated by the multiplicity on the right. For simplicity, we show only those transcripts whose genomic locus contains the cassette exon, and we truncate the first three transcripts past the TSS-associated cassette exon. Table 1 summarizes the data in the figure, indicating how many times the exon was included and how many times skipped when a particular TSS was used." path-from-xml="dsm03601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474911/dsm03601.jpeg" path-from-xml="dsm03601.jpeg" target="_blank">View large</a><a section="5474911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03601.jpeg?Expires=2147483647&Signature=aI0371qbgHk5i3Ydd6xC1QyHD0y6eOcWq274CiDL5G~sxIANcWcU5nn6gQUV-IX130dHKbPYrhbUTSm6~0jzc~0VHnnG6ncPeh9WsTL4VbfDETV7zmHMFcWLOJDzIa~vWHG2poAMsTovqQuROykKIMkDurj71-5mF7zMd7E1gvgJv4FKGRhMBiXV9n4NVxykrcboT1H8Kkkfy3XvW-CVWYD3q~pPDpdwV9dn39Lb65~rUG2jaAU56nRqYPs6VSb7wS2t8ZJ67GWte7zlc4RZOi3rYBW7u-2-SjUjiBlkAvbHd9uinM8stBAoArlepA2Wl0uZyNu82c8~zO7FiAj5Rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474911&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs. Exons are indicated by (green) boxes and introns by red lines. Cassette exons are shown with a black frame. The internal cassette exon that has a high probability of TSS association is indicated by the thin black rectangle. The number of times a specific TSS-splicing pattern was observed in the data is indicated by the multiplicity on the right. For simplicity, we show only those transcripts whose genomic locus contains the cassette exon, and we truncate the first three transcripts past the TSS-associated cassette exon. Table <a reveal-id="DSM036TB1" data-open="DSM036TB1" class="link link-reveal link-table xref-fig">1</a> summarizes the data in the figure, indicating how many times the exon was included and how many times skipped when a particular TSS was used.</p></div></div><div content-id="DSM036F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03601.jpeg?Expires=2147483647&Signature=XGrw~rn9digviLSF4V3zW~XGru7thNqPml8O3SgSgFXzPCpJo0~H~14FwNlUYzF6~wX07Zr12mCNWV6kC~ZxbFXbTp~Z4NdyOrNZ5bm79hwHGIbd~b~i6lJSovAzgmiy0RhzmygXr3ZimsurwcEFJfs69r7LLnm5RF0PjjrMl2EbvPxIPREajc7aRzEKYJzXQ7aV~zphluu8m8leV2JtGbSrI7YTn1IOJwR4X0EDTQREGCG76SGJDN3DU8N9PY9nyKcpOLy3LvajIJG2hf5appvgL3RxIwXyTg3IgbLOyDauaM7tsKxL2X293oe8xT8e2FxtHcInVpevqe8TdkxUGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs. Exons are indicated by (green) boxes and introns by red lines. Cassette exons are shown with a black frame. The internal cassette exon that has a high probability of TSS association is indicated by the thin black rectangle. The number of times a specific TSS-splicing pattern was observed in the data is indicated by the multiplicity on the right. For simplicity, we show only those transcripts whose genomic locus contains the cassette exon, and we truncate the first three transcripts past the TSS-associated cassette exon. Table 1 summarizes the data in the figure, indicating how many times the exon was included and how many times skipped when a particular TSS was used." path-from-xml="dsm03601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474911/dsm03601.jpeg" path-from-xml="dsm03601.jpeg" target="_blank">View large</a><a section="5474911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03601.jpeg?Expires=2147483647&Signature=aI0371qbgHk5i3Ydd6xC1QyHD0y6eOcWq274CiDL5G~sxIANcWcU5nn6gQUV-IX130dHKbPYrhbUTSm6~0jzc~0VHnnG6ncPeh9WsTL4VbfDETV7zmHMFcWLOJDzIa~vWHG2poAMsTovqQuROykKIMkDurj71-5mF7zMd7E1gvgJv4FKGRhMBiXV9n4NVxykrcboT1H8Kkkfy3XvW-CVWYD3q~pPDpdwV9dn39Lb65~rUG2jaAU56nRqYPs6VSb7wS2t8ZJ67GWte7zlc4RZOi3rYBW7u-2-SjUjiBlkAvbHd9uinM8stBAoArlepA2Wl0uZyNu82c8~zO7FiAj5Rg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474911&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs. Exons are indicated by (green) boxes and introns by red lines. Cassette exons are shown with a black frame. The internal cassette exon that has a high probability of TSS association is indicated by the thin black rectangle. The number of times a specific TSS-splicing pattern was observed in the data is indicated by the multiplicity on the right. For simplicity, we show only those transcripts whose genomic locus contains the cassette exon, and we truncate the first three transcripts past the TSS-associated cassette exon. Table <a reveal-id="DSM036TB1" data-open="DSM036TB1" class="link link-reveal link-table xref-fig">1</a> summarizes the data in the figure, indicating how many times the exon was included and how many times skipped when a particular TSS was used.</p></div></div><p>As described in Section 2.2, we can use Equation (<a href="#DSM036M7" class="sectionLink xref-sec">7</a>) and Bayes' theorem to calculate the posterior probability <em>P</em>(<em>f</em>|<em>D</em>) that a fraction <em>f</em> of all cassette exons is dependent on the TSS. The distributions <em>P</em>(<em>f</em>|<em>D</em>) obtained for both the human and mouse data are shown as solid lines in Fig. <a reveal-id="DSM036F2" data-open="DSM036F2" class="link link-reveal link-table xref-fig">2</a>. They suggest that the inclusion of about 24% (99% posterior probability interval 17.4–30.6%) of all cassette exons in our mouse dataset and 30% (99% posterior probability interval 26.2–34.9%) of all cassette exons in our human dataset is dependent on the TSSs of the corresponding transcripts. To additionally test the statistical significance of this result, we created randomized datasets <em>D</em><sub>rand</sub> by permuting, for each cassette exon, the inclusions and exclusions among the TSSs in such a way that the total number of times each TSS was used, and the total number of times the exon was included remained unchanged. The posterior distributions <em>P</em>(<em>f</em>|<em>D</em><sub>rand</sub>) obtained by applying the Bayesian procedure to the randomized data are shown as dashed lines in Fig. <a reveal-id="DSM036F2" data-open="DSM036F2" class="link link-reveal link-table xref-fig">2</a>. These distributions show that the Bayesian procedure correctly infers that the inclusion of &lt;5% (and likely none) of the cassette exons in <em>D</em><sub>rand</sub> depends on TSS. Moreover, Fig. <a reveal-id="DSM036F2" data-open="DSM036F2" class="link link-reveal link-table xref-fig">2</a> shows the striking difference between the real and randomized data, which is due to the enrichment of cassette exons with high posterior probability of TSS dependency in the real data.</p>                        <a id="5474913" scrollto-destination="5474913"></a>
<div id="DSM036F2" data-id="DSM036F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03602.jpeg?Expires=2147483647&Signature=pCEPUj88Ai61Ibt-XtyM5mbI4Xpxxy3081Y2ZDPDBMyjNHnrH-ejWJtadjdLf-P~VELn-IIVxBZpAjEbK0h3187BTx080g6WekYzZsXQCzfXiLKyHNYp~wdJhXSBlZcJNWUY2WdvUDW5V~GqfNbJoyj2NrQqQWQ9HEcwh6WE1VfRJaSJBA8X6sP0ZjJ0egYqeWAMH9TKT-yU6YA-0QdvnvGFXGLNz5b4tkvnwqAXDev4bGRlNCkX2uLhutMovR0QPOtgf6mfr6WVIDZF1K0OZgxoYxPztimhtB5aEMKdH~7lWnsCq2eQbUvO8hn~xLGiEEtvgqjaj5bDqfF-HNh2PQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The posterior probability P(f|D) that the inclusion of cassette exons is dependent on the TSS in a fraction f of all cassette exons (solid line). The dashed line shows the same distribution P(f|Drand) for a randomized dataset containing the same marginal counts of inclusion and TSS usage for each exon. The upper panel shows the human data, and the lower panel the mouse data." path-from-xml="dsm03602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474913/dsm03602.jpeg" path-from-xml="dsm03602.jpeg" target="_blank">View large</a><a section="5474913" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03602.jpeg?Expires=2147483647&Signature=F0vchnelOUd~P~LZxqAQ9coR8NnxMxtkOnozZ6kF9VgDVGYnfX9MnDUhSXj~qKcMse6h2PQ~dKJQiKsBafAUrWya-0mjBuDe4vngNZ~xLY61ovDk4WcxxgLu8xY6zgqBM0pef9RrJzR9AfzMkuaga3ysUF0mMuLMVfL9EkRL~DSBb13WTBLlez4h6Zoh3aJ2DfCwQgdland3Vkl6x0suvrOva7ZlSUVlqWgpVfaLWTUFzN9LQm8d8lh7SAvwxiCeSv2ZgBLB6agEydi9ynCBy5HAbFSsZ5sgBX8GU57R5niMoZvWCzL6LbtCTaE5DdzG1SmTqp~MYyHotJ7GHy6tAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474913&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The posterior probability <em>P</em>(<em>f</em>|<em>D</em>) that the inclusion of cassette exons is dependent on the TSS in a fraction <em>f</em> of all cassette exons (solid line). The dashed line shows the same distribution <em>P</em>(<em>f</em>|<em>D</em><sub>rand</sub>) for a randomized dataset containing the same marginal counts of inclusion and TSS usage for each exon. The upper panel shows the human data, and the lower panel the mouse data.</p></div></div><div content-id="DSM036F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03602.jpeg?Expires=2147483647&Signature=pCEPUj88Ai61Ibt-XtyM5mbI4Xpxxy3081Y2ZDPDBMyjNHnrH-ejWJtadjdLf-P~VELn-IIVxBZpAjEbK0h3187BTx080g6WekYzZsXQCzfXiLKyHNYp~wdJhXSBlZcJNWUY2WdvUDW5V~GqfNbJoyj2NrQqQWQ9HEcwh6WE1VfRJaSJBA8X6sP0ZjJ0egYqeWAMH9TKT-yU6YA-0QdvnvGFXGLNz5b4tkvnwqAXDev4bGRlNCkX2uLhutMovR0QPOtgf6mfr6WVIDZF1K0OZgxoYxPztimhtB5aEMKdH~7lWnsCq2eQbUvO8hn~xLGiEEtvgqjaj5bDqfF-HNh2PQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The posterior probability P(f|D) that the inclusion of cassette exons is dependent on the TSS in a fraction f of all cassette exons (solid line). The dashed line shows the same distribution P(f|Drand) for a randomized dataset containing the same marginal counts of inclusion and TSS usage for each exon. The upper panel shows the human data, and the lower panel the mouse data." path-from-xml="dsm03602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474913/dsm03602.jpeg" path-from-xml="dsm03602.jpeg" target="_blank">View large</a><a section="5474913" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03602.jpeg?Expires=2147483647&Signature=F0vchnelOUd~P~LZxqAQ9coR8NnxMxtkOnozZ6kF9VgDVGYnfX9MnDUhSXj~qKcMse6h2PQ~dKJQiKsBafAUrWya-0mjBuDe4vngNZ~xLY61ovDk4WcxxgLu8xY6zgqBM0pef9RrJzR9AfzMkuaga3ysUF0mMuLMVfL9EkRL~DSBb13WTBLlez4h6Zoh3aJ2DfCwQgdland3Vkl6x0suvrOva7ZlSUVlqWgpVfaLWTUFzN9LQm8d8lh7SAvwxiCeSv2ZgBLB6agEydi9ynCBy5HAbFSsZ5sgBX8GU57R5niMoZvWCzL6LbtCTaE5DdzG1SmTqp~MYyHotJ7GHy6tAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474913&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The posterior probability <em>P</em>(<em>f</em>|<em>D</em>) that the inclusion of cassette exons is dependent on the TSS in a fraction <em>f</em> of all cassette exons (solid line). The dashed line shows the same distribution <em>P</em>(<em>f</em>|<em>D</em><sub>rand</sub>) for a randomized dataset containing the same marginal counts of inclusion and TSS usage for each exon. The upper panel shows the human data, and the lower panel the mouse data.</p></div></div><p>Using the estimated fraction 〈f〉 (the mean of the posterior distribution <em>P</em>(<em>f</em>|<em>D</em>)) as a prior that the inclusion of a cassette exon depends on TSS, we computed the posterior probability that the inclusion of each exon in our dataset is TSS dependent (see Section 2.2). These data are given in the Supplementary table, and can also be further explored using the server that we established at <a class="link link-uri" href="http://www.spaed.unibas.ch/Promoter_data/TSS_cassette_exons_spaed_human.html">http://www.spaed.unibas.ch/Promoter_data/TSS_cassette_exons_spaed_human.html</a> and <a class="link link-uri" href="http://www.spaed.unibas.ch/Promoter_data/TSS_cassette_exons_spaed_mouse.html">http://www.spaed.unibas.ch/Promoter_data/TSS_cassette_exons_spaed_mouse.html</a>.</p><p>Fig. <a reveal-id="DSM036F1" data-open="DSM036F1" class="link link-reveal link-table xref-fig">1</a> shows the cassette exon with the highest posterior probability of TSS dependence in our mouse data. The exon belongs to the gene aspartate beta hydrolase has a posterior probability of TSS dependency of 0.89, and is indicated in the figure by the thin black rectangle. For clarity, we showed only the transcripts whose genomic loci contain the cassette exon, and we also truncated some of the transcripts after the exon in question. There are three different TSSs upstream of the exon, and a total of 23 transcripts that could have included this exon. Of the 11 transcripts originating in the two upstream TSSs, only one includes the cassette exon. In contrast, nine of the 12 transcripts originating from the third TSS include the cassette exon.</p><p>One may wonder to what extent our results are affected by imperfect efficiency of full length cDNA capture. That is, if a significant fraction of the cDNAs is not full length, the apparent TSSs for these transcripts would be incorrect, and one may wonder how they would influence our results. We have addressed this question by performing the same analysis using only transcripts whose start site was confirmed by CAGE tag data,<sup><a reveal-id="DSM036C22" data-open="DSM036C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and obtained essentially the same results. However, since requiring additional confirmation of TSSs substantially reduces the sizes of our datasets, we did not use this dataset further.</p><p>It is important to note that we do not need to find the precise locations of the TSSs which may in fact be much less precise than initially thought,<sup><a reveal-id="DSM036C21" data-open="DSM036C21" class="link link-ref link-reveal xref-bibr">21</a></sup> but we only need to separate our transcripts into sets that arose from the same transcription initiation regions, controlled by specific sets of regulatory signals. We decided to simply assume that transcripts with the same initial exon arose from the same TSS and that transcripts with different initial exons arose from different TSSs. Two types of errors may occur in this classification. First, transcripts that arose from two different, but nearby TSSs may be assigned to a single common TSS. Second, if a transcript is severely truncated due to cloning or sequencing errors, it may appear to start in an exon which is downstream from its real initial exon. Since the first type of error reduces our ability to distinguish different TSSs, and the second error per definition must be uncorrelated with splicing, the effect of both types of errors will be to <em>reduce</em> the correlations between TSS usage and splicing. Therefore, the clear correlations that we observe in spite of these potential errors should be considered to provide a lower bound on the correlations that do exist.</p><p>Another concern may be that the gene structures inferred from full-length cDNA are not representative, because the full-length cDNA sequencing projects generally included a prioritization step, that may have caused an apparent enrichment in rare splice variants. To address this issue, we have constructed a database of splice variants using solely human 5′-end ESTs, which we obtained based on the UniGene annotation. We analyzed these data using the same model as we used for full-length cDNAs. The 99% probability interval computed using this dataset was 0.74–0.78, compared with 0–0.009 obtained using the corresponding randomized dataset. This indicates that TSS-associated splice events are in fact even more frequent than initially estimated from full-length cDNA data. The likely reason why the estimate of the fraction of TSS-associated exons is larger when using 5′ EST data compared with full-length cDNA data is illustrated in Fig. <a reveal-id="DSM036F3" data-open="DSM036F3" class="link link-reveal link-table xref-fig">3</a>. The exon indicated by the box belongs to the cAMP-dependent protein kinase catalytic beta subunit (PRKACB), is always skipped when the two upstream promoters are used (136 ESTs), and is generally included with the most downstream promoter (46 of 56 ESTs). These counts are very unlikely under a model in which the promoter usage and exon inclusion are uncoupled. Generally, many exons that in the cDNA data did not have sufficient coverage to allow us to detect their TSS association do have sufficient coverage in the 5′ EST data to allow this inference to be made, and consequently, the fraction of exons inferred to be TSS associated is larger.</p>                        <a id="5474919" scrollto-destination="5474919"></a>
<div id="DSM036F3" data-id="DSM036F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03603.jpeg?Expires=2147483647&Signature=hbWwG73HzXWIFpH2g9zcskEYxaQ0VICzfgfMAKDMSZ1xE2V4cDCQoDAACxaMfV5EmGLJUd1DaESqhAyzPVI8uZ3XyPvtHiPEqgzx8ioPfGanIdNu7xrIRNIYoD2y3N-LbMK5p9YMolPi6vXEamAif6tAV8SsQNbSTbAtlSxw8W~z5DhWpnNiDBm4Dzk7CKYRQ2CCpxODDqtrZoWCgvAx8IUOI82hsVkCEe7S3ThvXgm~B-cilQu0QFIT1wALBiVF0KxL0K0Yqy5Qk8iFQ0V0hwrzCC9NBG7Wh4bXa0S0qP4zfw14YMADmcFr8UegXe9gVzJCX3K4fFjtPw9j1iwBIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs as inferred from 5′ EST data. The conventions used in this representation are the same as for Fig. 1. Exons shown in green have invariant splice boundaries, those in yellow have variable splice donor sites, and those in cyan intron inclusion. The inclusion of the exon indicated by the black box occurs only when the most downstream TSSs are used." path-from-xml="dsm03603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474919/dsm03603.jpeg" path-from-xml="dsm03603.jpeg" target="_blank">View large</a><a section="5474919" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03603.jpeg?Expires=2147483647&Signature=VxVn5KDfN1mtEw5O4xTPkLSydVsiiP9vl08~K0LCl7uNYHLlUk2ciqPe5xuSMkydzF-HnJ7nBXesS9j-B6AnHO9TVQV-7JoQLhX92xYC~TI152b0qY9tM5CLgPiwXHW6gRmWQbOzoDn0-TamvAtF3LSR80kgyk4TFUMKA5k~aSKuT2W4zwoB2yBtv1sOTyuFNoZtwtoBOedBFPohi3-h3F6LLzEjNYtERFq-VWbCjl6zEA5KyYPrUcL9rpVYI8xspRHb6z3iRG7fDI6onVCCza2c21cn9lZMaDSRwUmyxVWMGn9xeCWNJubrrpNqTEJkCSB-LD3QaouOGnTVFLr2Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474919&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs as inferred from 5′ EST data. The conventions used in this representation are the same as for Fig. 1. Exons shown in green have invariant splice boundaries, those in yellow have variable splice donor sites, and those in cyan intron inclusion. The inclusion of the exon indicated by the black box occurs only when the most downstream TSSs are used.</p></div></div><div content-id="DSM036F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03603.jpeg?Expires=2147483647&Signature=hbWwG73HzXWIFpH2g9zcskEYxaQ0VICzfgfMAKDMSZ1xE2V4cDCQoDAACxaMfV5EmGLJUd1DaESqhAyzPVI8uZ3XyPvtHiPEqgzx8ioPfGanIdNu7xrIRNIYoD2y3N-LbMK5p9YMolPi6vXEamAif6tAV8SsQNbSTbAtlSxw8W~z5DhWpnNiDBm4Dzk7CKYRQ2CCpxODDqtrZoWCgvAx8IUOI82hsVkCEe7S3ThvXgm~B-cilQu0QFIT1wALBiVF0KxL0K0Yqy5Qk8iFQ0V0hwrzCC9NBG7Wh4bXa0S0qP4zfw14YMADmcFr8UegXe9gVzJCX3K4fFjtPw9j1iwBIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs as inferred from 5′ EST data. The conventions used in this representation are the same as for Fig. 1. Exons shown in green have invariant splice boundaries, those in yellow have variable splice donor sites, and those in cyan intron inclusion. The inclusion of the exon indicated by the black box occurs only when the most downstream TSSs are used." path-from-xml="dsm03603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474919/dsm03603.jpeg" path-from-xml="dsm03603.jpeg" target="_blank">View large</a><a section="5474919" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03603.jpeg?Expires=2147483647&Signature=VxVn5KDfN1mtEw5O4xTPkLSydVsiiP9vl08~K0LCl7uNYHLlUk2ciqPe5xuSMkydzF-HnJ7nBXesS9j-B6AnHO9TVQV-7JoQLhX92xYC~TI152b0qY9tM5CLgPiwXHW6gRmWQbOzoDn0-TamvAtF3LSR80kgyk4TFUMKA5k~aSKuT2W4zwoB2yBtv1sOTyuFNoZtwtoBOedBFPohi3-h3F6LLzEjNYtERFq-VWbCjl6zEA5KyYPrUcL9rpVYI8xspRHb6z3iRG7fDI6onVCCza2c21cn9lZMaDSRwUmyxVWMGn9xeCWNJubrrpNqTEJkCSB-LD3QaouOGnTVFLr2Fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474919&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Example of an internal cassette exon whose inclusion is strongly correlated with specific TSSs as inferred from 5′ EST data. The conventions used in this representation are the same as for Fig. 1. Exons shown in green have invariant splice boundaries, those in yellow have variable splice donor sites, and those in cyan intron inclusion. The inclusion of the exon indicated by the black box occurs only when the most downstream TSSs are used.</p></div></div>                            <h3 scrollto-destination=5474920 id="5474920" class="section-title" data-legacy-id=s3b>3.2. Evolutionary conservation of TSS association</h3>
<p>If the correlation that we inferred between transcriptional and splicing events is functionally relevant, one would expect that the TSS dependence tends to be conserved between orthologous cassette exons. This could, for instance, manifest itself in a correlation of the posterior probabilities of TSS dependence of orthologous cassette exons. To check this, we started with the human and mouse cassette exons that are part of transcription units with multiple start sites, and we used the UCSC human-to-mouse and mouse-to-human whole-genome alignments to identify orthologous exons (see Section 2.4). This procedure yielded 668 pairs of orthologs. We then computed the correlation coefficient between the posterior probabilities of orthologous exons. As shown in Fig. <a reveal-id="DSM036F4" data-open="DSM036F4" class="link link-reveal link-table xref-fig">4</a> we obtained a weak, but significant (<em>P</em> = 0.002) correlation between the probabilities of TSS association of orthologous exons, providing some evidence that TSS dependence of exon inclusion is evolutionarily conserved.</p>                        <a id="5474922" scrollto-destination="5474922"></a>
<div id="DSM036F4" data-id="DSM036F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03604.jpeg?Expires=2147483647&Signature=24sjyyNQ-d4uO8MDyjPptPo9hNJBuNtaJjF8Ah-nUJCVFau2LJXY0zEnwwwiAjCM3svriCyz45FUCJsKGjOP3qpSMRPyTOW-U~IUH8F71KGT~EtrXg4WFKTGIcAjmn8PCmFCHDr59vg~rhG4zr4x3LsL4Qbqk2sly5vjIMHJDc~xfsQvpOKS2UVrTZV4t2UD~mraz6XLTACSyL7OVlqs51lbGIrzwuHdoXHs~WE9~7off00lOf9Lfv3S1HGvLAValOo8KhIZSKafs010jNRfv5tk-Af0-vNAvz36stsp3zO6TwX-R0vx2ng9yMJzeznCX4hcTfTCJRrI5inay-SIxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the posterior probabilities of TSS association for orthologous human–mouse cassette exons. Each dot represents one exon, with the x-coordinate being the posterior probability of TSS association of the human exon and the y-coordinate being the the posterior probability of TSS association of the orthologous mouse exon. The correlation coefficient is r = 0.12, P-value = 0.002." path-from-xml="dsm03604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474922/dsm03604.jpeg" path-from-xml="dsm03604.jpeg" target="_blank">View large</a><a section="5474922" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03604.jpeg?Expires=2147483647&Signature=Luz-5pbSvsZMAdHOvykTaZ6EvcgvtKDJCmtmiIi0PCI1rGFWALrtoOSLBsoMth1EggiJP6vTqQrTWXCKjJxGm8UogmG3vTmGqYgDFG~Z9lokJXz8IiA8p~LXLb~~AgCowgyJ06pXVHyVqBRLKmLk3Te9k3gVe0Sjnqyz7jHWVmyGAT0fIghONw16riAzerLqo3AbUHxs3sU3zI2NAjSQiX6E3CxTrEecpFawCgHkS3ukZan7ydelyorgk8e1ay4xvWKoGIv7Cb5~N8F1jUXIpFj0AKcOt8ivbQss8vCx~0qEsrfEjR8OkWlesdrfHYwZQVCFdhG4XYfs1hzWDCE2bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474922&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the posterior probabilities of TSS association for orthologous human–mouse cassette exons. Each dot represents one exon, with the <em>x</em>-coordinate being the posterior probability of TSS association of the human exon and the <em>y</em>-coordinate being the the posterior probability of TSS association of the orthologous mouse exon. The correlation coefficient is <em>r</em> = 0.12, <em>P</em>-value = 0.002.</p></div></div><div content-id="DSM036F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03604.jpeg?Expires=2147483647&Signature=24sjyyNQ-d4uO8MDyjPptPo9hNJBuNtaJjF8Ah-nUJCVFau2LJXY0zEnwwwiAjCM3svriCyz45FUCJsKGjOP3qpSMRPyTOW-U~IUH8F71KGT~EtrXg4WFKTGIcAjmn8PCmFCHDr59vg~rhG4zr4x3LsL4Qbqk2sly5vjIMHJDc~xfsQvpOKS2UVrTZV4t2UD~mraz6XLTACSyL7OVlqs51lbGIrzwuHdoXHs~WE9~7off00lOf9Lfv3S1HGvLAValOo8KhIZSKafs010jNRfv5tk-Af0-vNAvz36stsp3zO6TwX-R0vx2ng9yMJzeznCX4hcTfTCJRrI5inay-SIxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the posterior probabilities of TSS association for orthologous human–mouse cassette exons. Each dot represents one exon, with the x-coordinate being the posterior probability of TSS association of the human exon and the y-coordinate being the the posterior probability of TSS association of the orthologous mouse exon. The correlation coefficient is r = 0.12, P-value = 0.002." path-from-xml="dsm03604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474922/dsm03604.jpeg" path-from-xml="dsm03604.jpeg" target="_blank">View large</a><a section="5474922" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03604.jpeg?Expires=2147483647&Signature=Luz-5pbSvsZMAdHOvykTaZ6EvcgvtKDJCmtmiIi0PCI1rGFWALrtoOSLBsoMth1EggiJP6vTqQrTWXCKjJxGm8UogmG3vTmGqYgDFG~Z9lokJXz8IiA8p~LXLb~~AgCowgyJ06pXVHyVqBRLKmLk3Te9k3gVe0Sjnqyz7jHWVmyGAT0fIghONw16riAzerLqo3AbUHxs3sU3zI2NAjSQiX6E3CxTrEecpFawCgHkS3ukZan7ydelyorgk8e1ay4xvWKoGIv7Cb5~N8F1jUXIpFj0AKcOt8ivbQss8vCx~0qEsrfEjR8OkWlesdrfHYwZQVCFdhG4XYfs1hzWDCE2bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474922&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the posterior probabilities of TSS association for orthologous human–mouse cassette exons. Each dot represents one exon, with the <em>x</em>-coordinate being the posterior probability of TSS association of the human exon and the <em>y</em>-coordinate being the the posterior probability of TSS association of the orthologous mouse exon. The correlation coefficient is <em>r</em> = 0.12, <em>P</em>-value = 0.002.</p></div></div><p>One of the best examples of an evolutionarily conserved relationship is shown in Fig. <a reveal-id="DSM036F5" data-open="DSM036F5" class="link link-reveal link-table xref-fig">5</a>. The exon with a high probability of TSS association (<em>P</em> = 1 in human and <em>P</em> = 0.73 in mouse) is indicated by an arrow. It is included in the skeletal form of tropomyosin, which uses the most upstream TSS, and is excluded in other forms of tropomyosin, which also tend to use downstream TSSs.</p>                        <a id="5474924" scrollto-destination="5474924"></a>
<div id="DSM036F5" data-id="DSM036F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03605.jpeg?Expires=2147483647&Signature=o5dcgPNdLOoQfyqC9yLA6-ufJuZAPWXqH1lMc9fQyJ0VaETD0n-b6q75c6oQI9pmOWbAXUk-jpd03iMBOgltZ9dQVc39LS8LnoVZIb8Pk11lA1j4abC0QvXKFDMK-TiigbPAW7vqsQKiIp-kvTeekAya1t9vItjt1utOmze5PbGs--4ALdefY1LeEI0GqSFZ-7NLfLDRe4MyzHN97VBqri8jlN28-E6dcIu0t7s6kkJX7zWrxpL0tYmAPuvQfJcr6jiKaKEUNXadTGoxKM2pKbK5UvyfVVqmM2YEM5gD-QkpZ1IfF8d6rHA6pIm50sQD2hcNLJ4xQG5J~1vHTdJQXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conserved TSS association for a tropomyosin exon. The intron/exon structure of tropomyosin transcripts initiated from various TSSs is shown, with exons represented as black boxes and introns as red lines connecting the exons. The number of times each transcript form was observed in our dataset is indicated by the ‘multiplicity’ column on the right-hand side of the transcripts. The orthologous cassette exons are indicated by arrows, and their posterior probabilities of TSS association are indicated." path-from-xml="dsm03605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474924/dsm03605.jpeg" path-from-xml="dsm03605.jpeg" target="_blank">View large</a><a section="5474924" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03605.jpeg?Expires=2147483647&Signature=aJWOl6AnFPoAYjNdH~kmuDQ19VwbgGBT94OCqyRNP~BfOmtCOk5Ztrh4Q~7GSzk42QalSryp8qfH4ZZmr15No~eUtgmPCVx2xpmVSudo0kBhpnXCi5ND~ssYAjcfw2Am4W6hvbOb0w-YU0aGlXmj9AjizH6ng24b9xfXourDi2STIcIAXlUeH1kpexk6eQjYf6ge-IlZVx~4h96y36n6-VXIiV7UI8WqdceGRwVLcQ8kmVioTqT5w939ofqkHeFXkxOpMyDN8oMotUVXx7iBZM0-SFKdqesbPXeiKmRpPJB~zBSiA131Yf4ecTpYif6RSeXlmWje5-wb1JqaoqDQKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474924&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conserved TSS association for a tropomyosin exon. The intron/exon structure of tropomyosin transcripts initiated from various TSSs is shown, with exons represented as black boxes and introns as red lines connecting the exons. The number of times each transcript form was observed in our dataset is indicated by the ‘multiplicity’ column on the right-hand side of the transcripts. The orthologous cassette exons are indicated by arrows, and their posterior probabilities of TSS association are indicated.</p></div></div><div content-id="DSM036F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03605.jpeg?Expires=2147483647&Signature=o5dcgPNdLOoQfyqC9yLA6-ufJuZAPWXqH1lMc9fQyJ0VaETD0n-b6q75c6oQI9pmOWbAXUk-jpd03iMBOgltZ9dQVc39LS8LnoVZIb8Pk11lA1j4abC0QvXKFDMK-TiigbPAW7vqsQKiIp-kvTeekAya1t9vItjt1utOmze5PbGs--4ALdefY1LeEI0GqSFZ-7NLfLDRe4MyzHN97VBqri8jlN28-E6dcIu0t7s6kkJX7zWrxpL0tYmAPuvQfJcr6jiKaKEUNXadTGoxKM2pKbK5UvyfVVqmM2YEM5gD-QkpZ1IfF8d6rHA6pIm50sQD2hcNLJ4xQG5J~1vHTdJQXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Conserved TSS association for a tropomyosin exon. The intron/exon structure of tropomyosin transcripts initiated from various TSSs is shown, with exons represented as black boxes and introns as red lines connecting the exons. The number of times each transcript form was observed in our dataset is indicated by the ‘multiplicity’ column on the right-hand side of the transcripts. The orthologous cassette exons are indicated by arrows, and their posterior probabilities of TSS association are indicated." path-from-xml="dsm03605.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474924/dsm03605.jpeg" path-from-xml="dsm03605.jpeg" target="_blank">View large</a><a section="5474924" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03605.jpeg?Expires=2147483647&Signature=aJWOl6AnFPoAYjNdH~kmuDQ19VwbgGBT94OCqyRNP~BfOmtCOk5Ztrh4Q~7GSzk42QalSryp8qfH4ZZmr15No~eUtgmPCVx2xpmVSudo0kBhpnXCi5ND~ssYAjcfw2Am4W6hvbOb0w-YU0aGlXmj9AjizH6ng24b9xfXourDi2STIcIAXlUeH1kpexk6eQjYf6ge-IlZVx~4h96y36n6-VXIiV7UI8WqdceGRwVLcQ8kmVioTqT5w939ofqkHeFXkxOpMyDN8oMotUVXx7iBZM0-SFKdqesbPXeiKmRpPJB~zBSiA131Yf4ecTpYif6RSeXlmWje5-wb1JqaoqDQKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474924&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03605.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Conserved TSS association for a tropomyosin exon. The intron/exon structure of tropomyosin transcripts initiated from various TSSs is shown, with exons represented as black boxes and introns as red lines connecting the exons. The number of times each transcript form was observed in our dataset is indicated by the ‘multiplicity’ column on the right-hand side of the transcripts. The orthologous cassette exons are indicated by arrows, and their posterior probabilities of TSS association are indicated.</p></div></div><p>The level of evolutionary conservation of cassette exons has previously been related to the rate of inclusion of the exons in mature mRNAs: the so-called ‘major form’ exons, which are predominantly included, are as conserved as constitutive exons, whereas ‘minor form’ exons, which are predominantly skipped, appear to be of a more recent evolutionary origin,<sup><a reveal-id="DSM036C23" data-open="DSM036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> rarely having orthologs between human and rodents. To understand where TSS-associated exons fit in this evolutionary scenario, we analyzed the degree of human–mouse conservation of the following categories of exons: (<a reveal-id="DSM036C1" data-open="DSM036C1" class="link link-ref link-reveal xref-bibr">1</a>) TSS-associated exons—those with the top 10% values of posterior probability of TSS association, (<a reveal-id="DSM036C2" data-open="DSM036C2" class="link link-ref link-reveal xref-bibr">2</a>) TSS-independent exons—those with the bottom 10% values of posterior probability of TSS association, and (<a reveal-id="DSM036C3" data-open="DSM036C3" class="link link-ref link-reveal xref-bibr">3</a>) constitutive exons—exons included with no variation in more than ten transcripts. For each of these exons, we extracted the mouse—human and human–mouse alignments from the whole genome alignments provided by the UCSC (see Section 2.4). We then computed the fraction of exons that have orthologs in the other species and the fraction of nucleotides in each exon that are conserved. As shown in Table <a reveal-id="DSM036TB1" data-open="DSM036TB1" class="link link-reveal link-table xref-fig">1</a>, we found that the large majority of TSS-associated exons are conserved between mouse and human (438/496 = 88.3% of mouse and 1003/1166 = 86.02% of human TSS-associated exons). Particularly, TSS-associated exons are much more strongly conserved than ‘minor form’ exons, only 27–31% of which having been reported to be conserved between human and rodents,<sup><a reveal-id="DSM036C23" data-open="DSM036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> However, TSS-associated exons are significantly less conserved than TSS-independent exons (469/496 = 94.6% in mouse and 1077/1166 = 92.4% in human, P-value of the χ<sup>2</sup> test = 6.7 × 10<sup>−4</sup> for mouse and 1.12 × 10<sup>−6</sup> for human). These results are not sensitive to the precise threshold beyond which we consider an exon ‘conserved’. Among those TSS-associated exons that do have orthologs, the proportion of conserved nucleotides is lower compared with TSS-independent exons (P-value: 2.9 × 10<sup>−12</sup> for human and 2.2 × 10<sup>−3</sup> for mouse), as well as compared with constitutive exons (P-value of the Wilcoxon test &lt;2.2 × 10<sup>−16</sup> for human and 4.7 × 10<sup>−2</sup> for mouse). Consistent with previous results relating the degree of evolutionary conservation to the inclusion rate of the exons,<sup><a reveal-id="DSM036C23" data-open="DSM036C23" class="link link-ref link-reveal xref-bibr">23</a></sup> we found that the overall inclusion rate of TSS-associated exons is lower than the inclusion rate of TSS-independent exons (Table <a reveal-id="DSM036TB1" data-open="DSM036TB1" class="link link-reveal link-table xref-fig">1</a>). Thus, the results suggest that some of the TSS-associated exons are of relatively recent evolutionary origin, and it will be interesting to establish whether the TSSs that promote their inclusion have also undergone recent evolutionary changes. On the other hand, the relatively weak correlation between the probabilities of TSS association of orthologous exons is likely to be in part due to the fact that the human and mouse sequencing projects did not cover sufficiently similar sets of tissues. This would be reflected in different relative usage of the alternative TSSs and exons between human and mouse and these, in turn, would be reflected in disparate probabilities of TSS association of orthologous exons in the two species.</p>                        <a id="5474926" scrollto-destination="5474926"></a>
<div content-id="DSM036TB1" class="table-modal"><div class="table-wrap"><div id="DSM036TB1" class="table-wrap-title" data-id="DSM036TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of TSS associated, TSS independent, and constitutive exons</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Data set </th><th>Number exons </th><th>With orthologs </th><th>Median proportion conserved nucleotides </th><th>Inclusion rate </th><th>Proportion symmetrical </th></tr></thead><tbody><tr><td>Human exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>1166 </td><td>1003 </td><td>0.84 </td><td>0.75 </td><td>0.41 </td></tr><tr><td>TSS independent </td><td>1166 </td><td>1077 </td><td>0.87 </td><td>0.86 </td><td>0.45 </td></tr><tr><td>Constitutive </td><td>6377 </td><td>6343 </td><td>0.88 </td><td>1 </td><td>0.38 </td></tr><tr><td>Mouse exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>496 </td><td>438 </td><td>0.86 </td><td>0.75 </td><td>0.42 </td></tr><tr><td>TSS independent </td><td>496 </td><td>469 </td><td>0.88 </td><td>0.88 </td><td>0.48 </td></tr><tr><td>Constitutive </td><td>5136 </td><td>5117 </td><td>0.87 </td><td>1 </td><td>0.4 </td></tr></tbody></table></div><div content-id="DSM036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Data set </th><th>Number exons </th><th>With orthologs </th><th>Median proportion conserved nucleotides </th><th>Inclusion rate </th><th>Proportion symmetrical </th></tr></thead><tbody><tr><td>Human exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>1166 </td><td>1003 </td><td>0.84 </td><td>0.75 </td><td>0.41 </td></tr><tr><td>TSS independent </td><td>1166 </td><td>1077 </td><td>0.87 </td><td>0.86 </td><td>0.45 </td></tr><tr><td>Constitutive </td><td>6377 </td><td>6343 </td><td>0.88 </td><td>1 </td><td>0.38 </td></tr><tr><td>Mouse exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>496 </td><td>438 </td><td>0.86 </td><td>0.75 </td><td>0.42 </td></tr><tr><td>TSS independent </td><td>496 </td><td>469 </td><td>0.88 </td><td>0.88 </td><td>0.48 </td></tr><tr><td>Constitutive </td><td>5136 </td><td>5117 </td><td>0.87 </td><td>1 </td><td>0.4 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474926" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM036TB1" class="table-wrap-title" data-id="DSM036TB1"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of TSS associated, TSS independent, and constitutive exons</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Data set </th><th>Number exons </th><th>With orthologs </th><th>Median proportion conserved nucleotides </th><th>Inclusion rate </th><th>Proportion symmetrical </th></tr></thead><tbody><tr><td>Human exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>1166 </td><td>1003 </td><td>0.84 </td><td>0.75 </td><td>0.41 </td></tr><tr><td>TSS independent </td><td>1166 </td><td>1077 </td><td>0.87 </td><td>0.86 </td><td>0.45 </td></tr><tr><td>Constitutive </td><td>6377 </td><td>6343 </td><td>0.88 </td><td>1 </td><td>0.38 </td></tr><tr><td>Mouse exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>496 </td><td>438 </td><td>0.86 </td><td>0.75 </td><td>0.42 </td></tr><tr><td>TSS independent </td><td>496 </td><td>469 </td><td>0.88 </td><td>0.88 </td><td>0.48 </td></tr><tr><td>Constitutive </td><td>5136 </td><td>5117 </td><td>0.87 </td><td>1 </td><td>0.4 </td></tr></tbody></table></div><div content-id="DSM036TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Data set </th><th>Number exons </th><th>With orthologs </th><th>Median proportion conserved nucleotides </th><th>Inclusion rate </th><th>Proportion symmetrical </th></tr></thead><tbody><tr><td>Human exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>1166 </td><td>1003 </td><td>0.84 </td><td>0.75 </td><td>0.41 </td></tr><tr><td>TSS independent </td><td>1166 </td><td>1077 </td><td>0.87 </td><td>0.86 </td><td>0.45 </td></tr><tr><td>Constitutive </td><td>6377 </td><td>6343 </td><td>0.88 </td><td>1 </td><td>0.38 </td></tr><tr><td>Mouse exons </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>TSS associated </td><td>496 </td><td>438 </td><td>0.86 </td><td>0.75 </td><td>0.42 </td></tr><tr><td>TSS independent </td><td>496 </td><td>469 </td><td>0.88 </td><td>0.88 </td><td>0.48 </td></tr><tr><td>Constitutive </td><td>5136 </td><td>5117 </td><td>0.87 </td><td>1 </td><td>0.4 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5474926" target="_blank">View Large</a></div></div><p>Finally, we did not find a consistent trend in the proportion of ‘symmetrical’ exons, i.e. the proportion of exons whose length is a multiple of three, among our different categories of exons. We did not specifically select the exons for being part of coding regions, but rather we only considered internal exons in our analysis. As shown in Table <a reveal-id="DSM036TB1" data-open="DSM036TB1" class="link link-reveal link-table xref-fig">1</a>, the proportion of symmetrical exons is significantly higher than expected by chance, i.e. 1/3, for all exon types (TSS-associated, TSS-independent, constitutive). In human, TSS-associated exons have a significantly higher tendency for symmetry compared with constitutive internal exons (<em>P</em>-value of the χ<sup>2</sup> test = 0.02), and lower, but not significantly, compared with TSS-independent exons (<em>P</em>-value of the χ<sup>2</sup> test = 0.07). The tendencies are similar in mouse, but the differences are not statistically significant (<em>P</em>-value of the χ<sup>2</sup> test = 0.36 in the comparison with constitutive exons and 0.097 in the comparison with TSS-independent exons).</p>                            <h3 scrollto-destination=5474928 id="5474928" class="section-title" data-legacy-id=s3c>3.3 Insights into the mechanism of coupling between transcriptional and splicing events</h3>
<p>We can envision two mechanisms that could give rise to the correlation that we observe between transcriptional and splicing events. One is that tissue-specific transcriptional and splicing events are induced by independent, but tissue-specific transcription and splicing factors. The second mechanism involves a direct influence of tissue-specific transcription factors on internal splicing. We reasoned that if an exon is always included when one TSS is used and always skipped when another TSS is used, yet the two TSSs are both used in the same tissue, then the TSS association may be due to direct coupling of transcription with exon selection. To identify such cases, we used the library annotation of the transcripts in our datasets. We found that for 21% of the human and 14% of the mouse TSS-associated exons the inclusion- and skipping-promoting TSS have been both used in the same tissue.</p><p>One model that has been proposed for the coupling between transcriptional and splicing events is known as the ‘kinetic model’,<sup><a reveal-id="DSM036C10" data-open="DSM036C10" class="link link-ref link-reveal xref-bibr">10</a></sup> which postulates that cassette exons with weaker splice signals compared with constitutive exons tend to be skipped when the transcript is produced by a fast-elongating polymerase. In contrast, when the polymerase elongation rate is low, the spliceosome has sufficient time to assemble on these cassette exons, which are then spliced into the mature mRNA. We used a web server implementing the Shapiro and Senapathy model<sup><a reveal-id="DSM036C24" data-open="DSM036C24" class="link link-ref link-reveal xref-bibr">24</a></sup> (<a class="link link-uri" href="http://ast.bioinfo.tau.ac.il/SpliceSiteFrame.htm">http://ast.bioinfo.tau.ac.il/SpliceSiteFrame.htm</a>) to evaluate the strength of the splice sites of the cassette exons as well as of the exons flanking them. We found that the strength of the splice sites is comparable between the cassette exons and the exons flanking them (data not shown), suggesting that one of the pre-requisites of the kinetic model (weaker splice sites) is not met by our sets of TSS-associated exons.</p><p>Given that the elongation rate is not homogeneous along a gene, and that promoter-proximal pausing of RNA polymerase II is common,<sup><a reveal-id="DSM036C25" data-open="DSM036C25" class="link link-ref link-reveal xref-bibr">25</a></sup> we reasoned that the elongation rate of the polymerase may be generally lower at the start of the transcripts, allowing better recognition of cassette exons that are close to the TSS. Therefore, we asked whether the TSS-associated exons tend to be included when the TSS closest to them is used, as this may allow sufficient time for spliceosome assembly. We found this not to be the case: for all TSS-associated exons, we computed the average distance <em>D</em><sub><em>i</em></sub> between the exon and the TSSs of transcripts that included the exon, and the average distance <em>D</em><sub><em>s</em></sub> between the exon and the TSSs of transcripts that excluded the exon. We then constructed a histogram of the difference <em>D</em><sub><em>i</em></sub> − <em>D</em><sub><em>s</em></sub> across cassette exons (Fig. <a reveal-id="DSM036F6" data-open="DSM036F6" class="link link-reveal link-table xref-fig">6</a>). To improve visibility of the histogram, we split it into a part where <em>D</em><sub><em>i</em></sub> − <em>D</em><sub><em>s</em></sub> &gt; 0 and a part where <em>D</em><sub><em>i</em></sub> − <em>D</em><sub><em>s</em></sub> &lt; 0, and plotted the absolute distance |<em>D</em><sub><em>i</em></sub> − <em>D</em><sub><em>s</em></sub>| on a logarithmic scale. We found that approximately the same proportion of TSS-associated exons are preferentially included with the most upstream TSSs as are included with the most downstream TSSs (Fig. <a reveal-id="DSM036F6" data-open="DSM036F6" class="link link-reveal link-table xref-fig">6</a>). This suggests that proximity to the TSS, which may be indicative of lower polymerase elongation rate, cannot explain the inclusion pattern of TSS-associated cassette exons.</p>                        <a id="5474932" scrollto-destination="5474932"></a>
<div id="DSM036F6" data-id="DSM036F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03606.jpeg?Expires=2147483647&Signature=pQRb~6SESTcgSlGWmNXw1fF9WuHnxL-DUluIgT8SKAcN7rTG3bOMe2rivIMGOPb7JIOvInVE~UYoeAtmIoS-pVpJLSAzbSZCvSb-mtYc6m38GzIYWBbC-URWuAZXBeSsnu9cQQU6hoQVgqHW5m40Ri2afAj11CNDTfPrkuEv4upK5bHd6nS~BiABFOtiOb0aNORh-zEzPpa3GU0h7jCVKHtJdRxAz-cfK7a0AGEdI0oQRIKwhkdAJ6nnzzB2mKjxchzJgPpUA5pzpkgsM91ifliw83I178LpMAz1un52qaKMtLtGKQd3fM4c1G3pgTAV33FAvfYBCKo3Qywa~zlW3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of the signed difference between the logarithm (base 10) of the average distance to TSS when the exon is included and the average distance to TSS when the exon is skipped. Human data are shown in black and mouse data are shown in red." path-from-xml="dsm03606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474932/dsm03606.jpeg" path-from-xml="dsm03606.jpeg" target="_blank">View large</a><a section="5474932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03606.jpeg?Expires=2147483647&Signature=FnY5bOm7ivvGluhAxasNHmWi-Lq9Tr35PcqmijkyLbr~iUY8EmIsM7Zm7i5dZy737Id~6f4~5YlJ2b~pN03TJk07TTOghRysMY7uSMEc3u53zUxfrFDJF2gI0fmToyT5-9bOSnDP9Xl77ymIOuSuKQ2aK~Cs~ipcfJNYmSImPViCSiyrFMYeOa8~dhBgp9KrUetJRNdD54bJDk8PdEKWC7kBz6josButNhZurKc3ybZb38eJ4ewgz4F6DF1H2buDzGfphyPloIl-nLqOZtWGkF9p-DQyqPeBszRA1qWKomx8zWdNMMz4SBiaDl16c-dbQDaAn4b4eM8CNE9uUf05DQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474932&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of the signed difference between the logarithm (base 10) of the average distance to TSS when the exon is included and the average distance to TSS when the exon is skipped. Human data are shown in black and mouse data are shown in red.</p></div></div><div content-id="DSM036F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/m_dsm03606.jpeg?Expires=2147483647&Signature=pQRb~6SESTcgSlGWmNXw1fF9WuHnxL-DUluIgT8SKAcN7rTG3bOMe2rivIMGOPb7JIOvInVE~UYoeAtmIoS-pVpJLSAzbSZCvSb-mtYc6m38GzIYWBbC-URWuAZXBeSsnu9cQQU6hoQVgqHW5m40Ri2afAj11CNDTfPrkuEv4upK5bHd6nS~BiABFOtiOb0aNORh-zEzPpa3GU0h7jCVKHtJdRxAz-cfK7a0AGEdI0oQRIKwhkdAJ6nnzzB2mKjxchzJgPpUA5pzpkgsM91ifliw83I178LpMAz1un52qaKMtLtGKQd3fM4c1G3pgTAV33FAvfYBCKo3Qywa~zlW3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of the signed difference between the logarithm (base 10) of the average distance to TSS when the exon is included and the average distance to TSS when the exon is skipped. Human data are shown in black and mouse data are shown in red." path-from-xml="dsm03606.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5474932/dsm03606.jpeg" path-from-xml="dsm03606.jpeg" target="_blank">View large</a><a section="5474932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm03606.jpeg?Expires=2147483647&Signature=FnY5bOm7ivvGluhAxasNHmWi-Lq9Tr35PcqmijkyLbr~iUY8EmIsM7Zm7i5dZy737Id~6f4~5YlJ2b~pN03TJk07TTOghRysMY7uSMEc3u53zUxfrFDJF2gI0fmToyT5-9bOSnDP9Xl77ymIOuSuKQ2aK~Cs~ipcfJNYmSImPViCSiyrFMYeOa8~dhBgp9KrUetJRNdD54bJDk8PdEKWC7kBz6josButNhZurKc3ybZb38eJ4ewgz4F6DF1H2buDzGfphyPloIl-nLqOZtWGkF9p-DQyqPeBszRA1qWKomx8zWdNMMz4SBiaDl16c-dbQDaAn4b4eM8CNE9uUf05DQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5474932&ar=376035&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm03606.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of the signed difference between the logarithm (base 10) of the average distance to TSS when the exon is included and the average distance to TSS when the exon is skipped. Human data are shown in black and mouse data are shown in red.</p></div></div><p>To conclude, we estimated that exons whose inclusion in the mature mRNA is correlated with specific TSSs are rather common, i.e. they represent at least 15% of all internal cassette exons. The correlation may be due to direct coupling between transcription and splicing or indirect coupling, due, for instance, to a tissue-specific signaling pathway that activates both the transcription factors responsible for determining the TSS as well as the splicing factors responsible for the inclusion or exclusion of the cassette exon. Nonetheless, for at least 14–21% of the TSS-associated exons, TSSs that are associated with exclusion and TSSs that are associated with inclusion occur <em>both</em> in the same tissue, suggesting a more direct connection between the TSS and exon inclusion for at least these exons. The details of the molecular mechanism underlying this dependency remain to be uncovered, and may involve a gene-specific component, as suggested by our observation that some exons are predominantly included when the proximal TSSs are used, while other exons are predominantly skipped. One way of implementing a gene-(and tissue-) specific component could be through dynamic changes in the local chromatin structure that induce in turn local variations in the polymerase elongation rate.<sup><a reveal-id="DSM036C26" data-open="DSM036C26" class="link link-ref link-reveal xref-bibr">26</a></sup> This is a topic for future study and the cassette exons for which we estimated a high probability of TSS association provide good starting points for experimental investigations. </p>                            <h2 scrollto-destination=5474934 id="5474934" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p>Supplementary data are available online at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5474936 id="5474936" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work has been supported in part by the European Network for Alternative Splicing (EURASNET). TM Chern is also supported by a fellowship from the South African National Research Foundation. We gratefully acknowledge the VitalIT team of the Swiss Institute of Bioinformatics, in particular Ioannis Xenarios, for computational support.</p>                            <h2 scrollto-destination=5474938 id="5474938" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM036C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zavolan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kondo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Schonbach</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hume</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hayashizaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Impact of alternative initiation, splicing, and termination on the diversity of the mRNA transcripts encoded by the mouse transcriptome</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1290</div>-<div class="lpage">1300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20alternative%20initiation%2C%20splicing%2C%20and%20termination%20on%20the%20diversity%20of%20the%20mRNA%20transcripts%20encoded%20by%20the%20mouse%20transcriptome&amp;author=M.%20Zavolan&amp;author=S.%20Kondo&amp;author=C.%20Schonbach&amp;author=J.%20Adachi&amp;author=D.%20Hume&amp;author=Y.%20Hayashizaki&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1290-1300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1017303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1017303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1017303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12819126" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugnet</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kent</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ares</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Transcriptome and genome conservation of alternative splicing events in humans and mice</div>, <div class="source">Pac. Symp. Biocomput.</div> , <div class="year">2004</div>(pg. <div class="fpage">66</div>-<div class="lpage">77</div>)</div></div></div></div><div content-id="DSM036C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Berget</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Exon recognition in vertebrate splicing</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">2411</div>-<div class="lpage">2414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exon%20recognition%20in%20vertebrate%20splicing&amp;author=S.%20Berget&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=2411-2414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.6.2411" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.6.2411" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.6.2411"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7852296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stamm</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Stoilov</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Stoss</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div>. </div><div class="article-title">An alternative-exon database and its statistical analysis</div>, <div class="source">DNA Cell Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">739</div>-<div class="lpage">756</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20alternative-exon%20database%20and%20its%20statistical%20analysis&amp;author=S.%20Stamm&amp;author=J.%20Zhu&amp;author=K.%20Nakai&amp;author=P.%20Stoilov&amp;author=O.%20Stoss&amp;author=M.%20Zhang&amp;publication_year=2000&amp;journal=DNA%20Cell%20Biol.&amp;volume=19&amp;pages=739-756" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/104454900750058107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F104454900750058107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F104454900750058107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11177572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krainer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Distribution of SR protein exonic splicing enhancer motifs in human protein-coding genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">5053</div>-<div class="lpage">5062</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20SR%20protein%20exonic%20splicing%20enhancer%20motifs%20in%20human%20protein-coding%20genes&amp;author=J.%20Wang&amp;author=P.%20Smith&amp;author=A.%20Krainer&amp;author=M.%20Zhang&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5053-5062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki810"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16147989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Ast</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Intronic sequences flanking alternatively spliced exons are conserved between human and mouse</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">1631</div>-<div class="lpage">1637</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intronic%20sequences%20flanking%20alternatively%20spliced%20exons%20are%20conserved%20between%20human%20and%20mouse&amp;author=R.%20Sorek&amp;author=G.%20Ast&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=1631-1637" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1208803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1208803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1208803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12840041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baek</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Sequence conservation, relative isoform frequencies, and nonsense-mediated decay in evolutionarily conserved alternative splicing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">12813</div>-<div class="lpage">12818</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20conservation%2C%20relative%20isoform%20frequencies%2C%20and%20nonsense-mediated%20decay%20in%20evolutionarily%20conserved%20alternative%20splicing&amp;author=D.%20Baek&amp;author=P.%20Green&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=12813-12818" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506139102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506139102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506139102"> </span></div> </div></div></div></div></div><div content-id="DSM036C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plass</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Eyras</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Differentiated evolutionary rates in alternative exons and the implications for splicing regulation</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> pg. <div class="fpage">50</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differentiated%20evolutionary%20rates%20in%20alternative%20exons%20and%20the%20implications%20for%20splicing%20regulation&amp;author=M.%20Plass&amp;author=E.%20Eyras&amp;publication_year=2006&amp;journal=BMC%20Evol.%20Biol.&amp;volume=6&amp;pages=50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-6-50" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-6-50" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-6-50"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16792801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kornblihtt</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Promoter usage and alternative splicing</div>, <div class="source">Curr. Opin. Cell. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">262</div>-<div class="lpage">268</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20usage%20and%20alternative%20splicing&amp;author=A.%20Kornblihtt&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Cell.%20Biol.&amp;volume=17&amp;pages=262-268" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2005.04.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2005.04.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2005.04.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15901495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kornblihtt</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Chromatin, transcript elongation and alternative splicing</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">5</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%2C%20transcript%20elongation%20and%20alternative%20splicing&amp;author=A.%20Kornblihtt&amp;publication_year=2006&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=13&amp;pages=5-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb0106-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb0106-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb0106-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16395314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nogues</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kadener</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Cramer</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bentley</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kornblihtt</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Transcriptional activators differ in their abilities to control alternative splicing</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">43110</div>-<div class="lpage">43114</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20activators%20differ%20in%20their%20abilities%20to%20control%20alternative%20splicing&amp;author=G.%20Nogues&amp;author=S.%20Kadener&amp;author=P.%20Cramer&amp;author=D.%20Bentley&amp;author=A.%20Kornblihtt&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=43110-43114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M208418200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M208418200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M208418200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12221105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lim</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Alternative splicing of the fibronectin EIIIB exon depends on specific TGCATG repeats</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">3900</div>-<div class="lpage">3906</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20the%20fibronectin%20EIIIB%20exon%20depends%20on%20specific%20TGCATG%20repeats&amp;author=L.%20Lim&amp;author=P.%20Sharp&amp;publication_year=1998&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=18&amp;pages=3900-3906" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.18.7.3900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.18.7.3900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.18.7.3900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9632774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Monsalve</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Adelmant</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Puigserver</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Spiegelman</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Direct coupling of transcription and mRNA processing through the thermogenic coactivator PGC − 1</div>, <div class="source">Mol. Cell</div> , <div class="year">2000</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">307</div>-<div class="lpage">316</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20coupling%20of%20transcription%20and%20mRNA%20processing%20through%20the%20thermogenic%20coactivator%20PGC%20%E2%88%92%201&amp;author=M.%20Monsalve&amp;author=Z.%20Wu&amp;author=G.%20Adelmant&amp;author=P.%20Puigserver&amp;author=M.%20Fan&amp;author=B.%20Spiegelman&amp;publication_year=2000&amp;journal=Mol.%20Cell&amp;volume=6&amp;pages=307-316" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1097-2765(00)00031-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1097-2765(00)00031-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1097-2765(00)00031-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10983978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pagani</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Stuani</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zuccato</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kornblihtt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Baralle</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Promoter architecture modulates CFTR exon 9 skipping</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">1511</div>-<div class="lpage">1517</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter%20architecture%20modulates%20CFTR%20exon%209%20skipping&amp;author=F.%20Pagani&amp;author=C.%20Stuani&amp;author=E.%20Zuccato&amp;author=A.%20Kornblihtt&amp;author=F.%20Baralle&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=1511-1517" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M209676200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M209676200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M209676200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12421814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Auboeuf</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Honig</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Berget</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">O'Malley</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Coordinate regulation of transcription and splicing by steroid receptor coregulators</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">298</div> (pg. <div class="fpage">416</div>-<div class="lpage">419</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coordinate%20regulation%20of%20transcription%20and%20splicing%20by%20steroid%20receptor%20coregulators&amp;author=D.%20Auboeuf&amp;author=A.%20Honig&amp;author=S.%20Berget&amp;author=B.%20O'Malley&amp;publication_year=2002&amp;journal=Science&amp;volume=298&amp;pages=416-419" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1073734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1073734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1073734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12376702" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kasukawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gough</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Frith</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Maeda</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">The transcriptional landscape of the mammalian genome</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">1559</div>-<div class="lpage">1563</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20mammalian%20genome&amp;author=P.%20Carninci&amp;author=T.%20Kasukawa&amp;author=S.%20Katayama&amp;author=J.%20Gough&amp;author=M.%20Frith&amp;author=N.%20Maeda&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=1559-1563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1112014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1112014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1112014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16141072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Imanishi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">O'Donovan</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Fukuchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Koyanagi</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Integrative annotation of 21,037 human genes validated by full-length cDNA clones</div>, <div class="source">PLoS Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">856</div>-<div class="lpage">875</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20annotation%20of%2021%2C037%20human%20genes%20validated%20by%20full-length%20cDNA%20clones&amp;author=T.%20Imanishi&amp;author=T.%20Itoh&amp;author=Y.%20Suzuki&amp;author=C.%20O'Donovan&amp;author=S.%20Fukuchi&amp;author=K.%20Koyanagi&amp;publication_year=2004&amp;journal=PLoS%20Biol.&amp;volume=2&amp;pages=856-875" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020162"> </span></div> </div></div></div></div></div><div content-id="DSM036C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chern</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">van Nimwegen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kai</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hayashizaki</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A simple physical model predicts small exon length variations</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e45</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20physical%20model%20predicts%20small%20exon%20length%20variations&amp;author=T.%20Chern&amp;author=E.%20van%20Nimwegen&amp;author=C.%20Kai&amp;author=J.%20Kawai&amp;author=P.%20Carninci&amp;author=Y.%20Hayashizaki&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=e45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Nimwegen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sheridan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zavolan</div> <div class="given-names">M</div></div>. </div><div class="article-title">SPA: a probabilistic algorithm for spliced alignment</div>, <div class="source">PLoS Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e24</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPA%3A%20a%20probabilistic%20algorithm%20for%20spliced%20alignment&amp;author=E.%20van%20Nimwegen&amp;author=N.%20Paul&amp;author=R.%20Sheridan&amp;author=M%20Zavolan&amp;publication_year=2006&amp;journal=PLoS%20Genet.&amp;volume=2&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16683023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zavolan</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">van Nimwegen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Gaasterland</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Splice variation in mouse full-length cDNAs identified by mapping to the mouse genome</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1377</div>-<div class="lpage">1385</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splice%20variation%20in%20mouse%20full-length%20cDNAs%20identified%20by%20mapping%20to%20the%20mouse%20genome&amp;author=M.%20Zavolan&amp;author=E.%20van%20Nimwegen&amp;author=T.%20Gaasterland&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1377-1385" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.191702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.191702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.191702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12213775" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sandelin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lenhard</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Katayama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shimokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ponjavic</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of mammalian promoter architecture and evolution</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">626</div>-<div class="lpage">635</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20mammalian%20promoter%20architecture%20and%20evolution&amp;author=P.%20Carninci&amp;author=A.%20Sandelin&amp;author=B.%20Lenhard&amp;author=S.%20Katayama&amp;author=K.%20Shimokawa&amp;author=J.%20Ponjavic&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=626-635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1789" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1789" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1789"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16645617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kodzius</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kojima</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishiyori</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fukuda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tagami</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">CAGE: cap analysis of gene expression</div>, <div class="source">Nat. Methods.</div> , <div class="year">2006</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">211</div>-<div class="lpage">222</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAGE%3A%20cap%20analysis%20of%20gene%20expression&amp;author=R.%20Kodzius&amp;author=M.%20Kojima&amp;author=H.%20Nishiyori&amp;author=M.%20Nakamura&amp;author=S.%20Fukuda&amp;author=M.%20Tagami&amp;publication_year=2006&amp;journal=Nat.%20Methods.&amp;volume=3&amp;pages=211-222" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth0306-211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth0306-211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth0306-211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16489339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Modrek</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Alternative splicing in the human, mouse and rat genomes is associated with an increased frequency of exon creation and/or loss</div>, <div class="source">Nat. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">177</div>-<div class="lpage">180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20in%20the%20human%2C%20mouse%20and%20rat%20genomes%20is%20associated%20with%20an%20increased%20frequency%20of%20exon%20creation%20and%2For%20loss&amp;author=B.%20Modrek&amp;author=C.%20Lee&amp;publication_year=2003&amp;journal=Nat.%20Genet.&amp;volume=34&amp;pages=177-180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1159" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1159" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1159"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12730695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shapiro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Senapathy</div> <div class="given-names">P.</div></div>. </div><div class="article-title">RNA splice junctions of different classes of eukaryotes: sequence statistics and functional implications in gene expression</div>, <div class="source">Nucletic. Acids Res.</div> , <div class="year">1987</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">7155</div>-<div class="lpage">7174</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20splice%20junctions%20of%20different%20classes%20of%20eukaryotes%3A%20sequence%20statistics%20and%20functional%20implications%20in%20gene%20expression&amp;author=M.%20Shapiro&amp;author=P.%20Senapathy&amp;publication_year=1987&amp;journal=Nucletic.%20Acids%20Res.&amp;volume=15&amp;pages=7155-7174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.17.7155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.17.7155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.17.7155"> </span></div> </div></div></div></div></div><div content-id="DSM036C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krumm</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Groudine</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Promoter-proximal pausing of RNA polymerase II defines a general rate-limiting step after transcription initiation</div>, <div class="source">Genes Dev.</div> , <div class="year">1995</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">559</div>-<div class="lpage">572</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Promoter-proximal%20pausing%20of%20RNA%20polymerase%20II%20defines%20a%20general%20rate-limiting%20step%20after%20transcription%20initiation&amp;author=A.%20Krumm&amp;author=L.%20Hickey&amp;author=M.%20Groudine&amp;publication_year=1995&amp;journal=Genes%20Dev.&amp;volume=9&amp;pages=559-572" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.9.5.559" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.9.5.559" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.9.5.559"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7698646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM036C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM036C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Batsche</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Yavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Muchardt</div> <div class="given-names">C.</div></div>. </div><div class="article-title">The human SWI/SNF subunit Brm is a regulator of alternative splicing</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">22</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20SWI%2FSNF%20subunit%20Brm%20is%20a%20regulator%20of%20alternative%20splicing&amp;author=E.%20Batsche&amp;author=M.%20Yavin&amp;author=C.%20Muchardt&amp;publication_year=2006&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=13&amp;pages=22-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb1030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb1030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb1030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16341228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=376035&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/2/10.1093/dnares/dsm036/2/dsm036-dsm036supp.pdf?Expires=2147483647&amp;Signature=vRlhATioSHj6J-MISDgUWDySFIyJF6QWEFp5ox4IHqruI3EFpzqMxPnYixrV4qpQAyO4c6Vx~feLjj4JfPQdUV-1eZS-obs0q6z84nZ4V~MHjG9pE88FxItCjH0a4A9Yp74QBVtY1~xvt8av~MZVJVxUBOXdLUPkFtqlTcjQ-sHXOU6PYlmQ5IqS7t5wkGQ0bj2SzlSOLdHQ8~UW9AffVD5vsRQGUCJ3FYU~MW6YA8XJ-GRhGqndeWsKWRknFiJaRoBdDxTv5vjoxfQef0ox6V6Y07NQ5-Pwe0rSBLwbKpa5jT-AQbZdO6tG9Rf09Le3NqidEtK8A1SzIyZVL3ZCMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">110</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm036"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="376035"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000256489900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f2%2f63%2f376035">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000256489900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F2%2F63%2F376035" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/376035'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Computational Analysis of Full-length cDNAs Reveals Frequent Coupling Between Transcriptional and Splicing Programs" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm036");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f2%2f63%2f376035">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

