

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1726,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/6/393/484773","@type":"ScholarlyArticle","name":"Genome Sequence of  Kitasatospora setae  NBRC 14216 T : An Evolutionary Snapshot of the Family  Streptomycetaceae ","datePublished":"2010-11-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2010-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq026","keywords":["<italic>Kitasatospora setae</italic>","complete genome","<italic>Streptomyces</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/6/393/484773","author":[{"name":"Ichikawa, Natsuko","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Oguchi, Akio","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Ikeda, Haruo","affiliation":"2  Kitasato Institute for Life Sciences ,  Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 252-0373 ,  Japan","@type":"Person"},{"name":"Ishikawa, Jun","affiliation":"3  Department of Bioactive Molecules ,  National Institute of Infectious Diseases ,  1-23-1 Toyama, Shinjuku-ku, Tokyo 162-8640 ,  Japan","@type":"Person"},{"name":"Kitani, Shigeru","affiliation":"4  International Center for Biotechnology ,  Osaka University ,  2-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Watanabe, Yumi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Nakamura, Sanae","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Katano, Yoko","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Kishi, Emi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Sasagawa, Machi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Ankai, Akiho","affiliation":"5  Tohoku Regional Office, National Institute of Technology and Evaluation ,  4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833 ,  Japan","@type":"Person"},{"name":"Fukui, Shigehiro","affiliation":"5  Tohoku Regional Office, National Institute of Technology and Evaluation ,  4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833 ,  Japan","@type":"Person"},{"name":"Hashimoto, Yoshimi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Kamata, Sachi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Otoguro, Misa","affiliation":"6  NITE Biological Resource Center (NBRC), Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-5-8 Kazusakamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tanikawa, Satoshi","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"},{"name":"Nihira, Takuya","affiliation":"4  International Center for Biotechnology ,  Osaka University ,  2-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Horinouchi, Sueharu","affiliation":"7  Department of Biotechnology, Graduate School of Agriculture and Life Sciences ,  The University of Tokyo ,  1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Ohnishi, Yasuo","affiliation":"7  Department of Biotechnology, Graduate School of Agriculture and Life Sciences ,  The University of Tokyo ,  1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Hayakawa, Masayuki","affiliation":"8  Division of Applied Biological Sciences, Interdisciplinary Graduate School of Medicine and Engineering ,  University of Yamanashi ,  Takeda-4, Kofu, Yamanashi 400-8511 ,  Japan","@type":"Person"},{"name":"Kuzuyama, Tomohisa","affiliation":"9  Biotechnology Research Center ,  The University of Tokyo ,  1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Arisawa, Akira","affiliation":"10  Bioresource Laboratories ,  Mercian Corp. ,  1808 Nakaizumi, Iwata, Shizuoka 438-0078 ,  Japan","@type":"Person"},{"name":"Nomoto, Fumiki","affiliation":"11  Nagase &amp; Co. Ltd ,  Research and Development Center ,  2-2-3 Murotani, Nishi-ku, Kobe, Hyogo 651-2241 ,  Japan","@type":"Person"},{"name":"Miura, Hiromi","affiliation":"12  Kitasato Institute for Life Sciences ,  Kitasato University ,  5-9-1 Shirokane, Minato-ku, Tokyo 108-8641 ,  Japan","@type":"Person"},{"name":"Takahashi, Yoko","affiliation":"12  Kitasato Institute for Life Sciences ,  Kitasato University ,  5-9-1 Shirokane, Minato-ku, Tokyo 108-8641 ,  Japan","@type":"Person"},{"name":"Fujita, Nobuyuki","affiliation":"1  NITE Bioresource Information Center, Department of Biotechnology ,  National Institute of Technology and Evaluation ,  2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066 ,  Japan","@type":"Person"}],"description":"Abstract.  Kitasatospora setae NBRC 14216T (=KM-6054T) is known to produce setamycin (bafilomycin B1) possessing antitrichomonal activity. The genus Kitasatospo","pageStart":"393","pageEnd":"406","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome Sequence of  Kitasatospora setae  NBRC 14216 T : An Evolutionary Snapshot of the Family  Streptomycetaceae ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome Sequence of  Kitasatospora setae  NBRC 14216 T : An Evolutionary Snapshot of the Family  Streptomycetaceae " />
    <meta property="og:description" content="Abstract.  Kitasatospora setae NBRC 14216T (=KM-6054T) is known to produce setamycin (bafilomycin B1) possessing antitrichomonal activity. The genus Kitasatospo" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/6/393/484773" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-11-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Ichikawa, Natsuko" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Oguchi, Akio" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Ikeda, Haruo" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 252-0373, Japan" /><meta name="citation_author" content="Ishikawa, Jun" /><meta name="citation_author_institution" content="Department of Bioactive Molecules, National Institute of Infectious Diseases, 1-23-1 Toyama, Shinjuku-ku, Tokyo 162-8640, Japan" /><meta name="citation_author" content="Kitani, Shigeru" /><meta name="citation_author_institution" content="International Center for Biotechnology, Osaka University, 2-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Watanabe, Yumi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Nakamura, Sanae" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Katano, Yoko" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Kishi, Emi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Sasagawa, Machi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Ankai, Akiho" /><meta name="citation_author_institution" content="Tohoku Regional Office, National Institute of Technology and Evaluation, 4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833, Japan" /><meta name="citation_author" content="Fukui, Shigehiro" /><meta name="citation_author_institution" content="Tohoku Regional Office, National Institute of Technology and Evaluation, 4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833, Japan" /><meta name="citation_author" content="Hashimoto, Yoshimi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Kamata, Sachi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Otoguro, Misa" /><meta name="citation_author_institution" content="NITE Biological Resource Center (NBRC), Department of Biotechnology, National Institute of Technology and Evaluation, 2-5-8 Kazusakamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tanikawa, Satoshi" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Nihira, Takuya" /><meta name="citation_author_institution" content="International Center for Biotechnology, Osaka University, 2-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Horinouchi, Sueharu" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Ohnishi, Yasuo" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Hayakawa, Masayuki" /><meta name="citation_author_institution" content="Division of Applied Biological Sciences, Interdisciplinary Graduate School of Medicine and Engineering, University of Yamanashi, Takeda-4, Kofu, Yamanashi 400-8511, Japan" /><meta name="citation_author" content="Kuzuyama, Tomohisa" /><meta name="citation_author_institution" content="Biotechnology Research Center, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Arisawa, Akira" /><meta name="citation_author_institution" content="Bioresource Laboratories, Mercian Corp., 1808 Nakaizumi, Iwata, Shizuoka 438-0078, Japan" /><meta name="citation_author" content="Nomoto, Fumiki" /><meta name="citation_author_institution" content="Nagase &amp; Co. Ltd, Research and Development Center, 2-2-3 Murotani, Nishi-ku, Kobe, Hyogo 651-2241, Japan" /><meta name="citation_author" content="Miura, Hiromi" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, 5-9-1 Shirokane, Minato-ku, Tokyo 108-8641, Japan" /><meta name="citation_author" content="Takahashi, Yoko" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, 5-9-1 Shirokane, Minato-ku, Tokyo 108-8641, Japan" /><meta name="citation_author" content="Fujita, Nobuyuki" /><meta name="citation_author_institution" content="NITE Bioresource Information Center, Department of Biotechnology, National Institute of Technology and Evaluation, 2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066, Japan" /><meta name="citation_title" content="Genome Sequence of Kitasatospora setae NBRC 14216T: An Evolutionary Snapshot of the Family Streptomycetaceae" /><meta name="citation_firstpage" content="393" /><meta name="citation_lastpage" content="406" /><meta name="citation_doi" content="10.1093/dnares/dsq026" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2010/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The evolution of development in Streptomyces analysed by genome comparisons; FEMS Microbiol. Rev.;  citation_year=2006;  citation_volume=30;  citation_pages=651-72; " /><meta name="citation_reference" content="citation_title=Mining and polishing of the treasure trove in the bacterial genus Streptomyces; Biosci. Biotechnol. Biochem.;  citation_year=2007;  citation_volume=71;  citation_pages=283-99; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the model actinomycete Streptomyces coelicolor A3(2); Nature;  citation_year=2002;  citation_volume=417;  citation_pages=141-7; " /><meta name="citation_reference" content="citation_title=Complete genome sequence and comparative analysis of the industrial microorganism Streptomyces avermitilis; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=526-31; " /><meta name="citation_reference" content="citation_title=Genome sequence of the streptomycin-producing microorganism Streptomyces griseus IFO 13350; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=4050-60; " /><meta name="citation_reference" content="citation_title=Tyropeptins A and B, new proteasome inhibitors produced by Kitasatospora sp. MK993-dF2. II. Structure determination and synthesis; J. Antibiot. (Tokyo);  citation_year=2001;  citation_volume=54;  citation_pages=1004-12; " /><meta name="citation_reference" content="citation_title=Talosins A and B: new isoflavonol glycosides with potent antifungal activity from Kitasatospora kifunensis MJM341. I. Taxonomy, fermentation, isolation, and biological activities; J. Antibiot. (Tokyo);  citation_year=2006;  citation_volume=59;  citation_pages=633-9; " /><meta name="citation_reference" content="citation_title=Kitasatospora, a new genus of the order Actinomycetales; J. Antibiot. (Tokyo);  citation_year=1982;  citation_volume=35;  citation_pages=1013-9; " /><meta name="citation_reference" content="citation_title=Taxonomic status of Kitasatospora, and proposed unification with Streptomyces on the basis of phenotypic and 16S rRNA analysis and emendation of Streptomyces Waksman and Henrici 1943, 339AL; Int. J. Syst. Bacteriol.;  citation_year=1992;  citation_volume=42;  citation_pages=156-60; " /><meta name="citation_reference" content="citation_title=A taxonomic review of the genera Kitasatospora and Streptoverticillium by analysis of ribosomal protein AT-L30; Int. J. Syst. Bacteriol.;  citation_year=1994;  citation_volume=44;  citation_pages=285-92; " /><meta name="citation_reference" content="citation_title=A proposal to revive the genus Kitasatospora (Omura, Takahashi, Iwai, and Tanaka 1982); Int. J. Syst. Bacteriol.;  citation_year=1997;  citation_volume=47;  citation_pages=1048-54; " /><meta name="citation_reference" content="citation_title=Design and evaluation of an oligonucleotide-microarray for the detection of different species of the genus Kitasatospora; J. Microbiol. Methods;  citation_year=2006;  citation_volume=65;  citation_pages=226-36; " /><meta name="citation_reference" content="citation_title=Electrical DNA-chip-based identification of different species of the genus Kitasatospora; Appl. Microbiol. Biotechnol.;  citation_year=2008;  citation_volume=77;  citation_pages=1181-8; " /><meta name="citation_reference" content="citation_title=Relationship between cell morphology and the types of diaminopimelic acid in Kitasatospora setalba; J. Gen. Appl. Microbiol.;  citation_year=1983;  citation_volume=29;  citation_pages=459-65; " /><meta name="citation_reference" content="citation_title=Some characteristics of aerial and submerged spores of Kitasatospora setalba; J. Gen. Appl. Microbiol.;  citation_year=1984;  citation_volume=30;  citation_pages=223-9; " /><meta name="citation_reference" content="citation_title=Type of diaminopimelic acid different in aerial and vegetative mycelia of setamycin-producing actinomycete KM-6054; J. Antibiot. (Tokyo);  citation_year=1981;  citation_volume=34;  citation_pages=1633-4; " /><meta name="citation_reference" content="citation_title=Taxonomic study and morphological differentiation of an actinomycete genus, Kitasatospora; Zentralbl. Bakteriol.;  citation_year=1999;  citation_volume=289;  citation_pages=265-84; " /><meta name="citation_reference" content="citation_title=Sequence analysis of three plasmids harboured in Rhodococcus erythropolis strain PR4; Environ. Microbiol.;  citation_year=2006;  citation_volume=8;  citation_pages=334-46; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the soil actinomycete Kocuria rhizophila; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=4139-46; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. II. Error probabilities; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=186-94; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=175-85; " /><meta name="citation_reference" content="citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=673-9; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=FramePlot: a new implementation of the frame analysis for predicting protein-coding regions in bacterial DNA with a high G + C content; FEMS Microbiol. Lett.;  citation_year=1999;  citation_volume=174;  citation_pages=251-3; " /><meta name="citation_reference" content="citation_title=The universal protein resource (UniProt); citation_author=UniProt, Consortium; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D190-5; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D224-8; " /><meta name="citation_reference" content="citation_title=HAMAP: a database of completely sequenced microbial proteome sets and manually curated microbial protein families in UniProtKB/Swiss-Prot; Nucleic Acids Res.;  citation_year=2008;  citation_volume=37;  citation_pages=D471-8; " /><meta name="citation_reference" content="citation_title=KEGG for linking genomes to life and the environment; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D480-4; " /><meta name="citation_reference" content="citation_title=TransportDB: a comprehensive database resource for cytoplasmic membrane transport systems and outer membrane channels; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D274-9; " /><meta name="citation_reference" content="citation_title=MEROPS: the peptidase database; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D320-5; " /><meta name="citation_reference" content="citation_title=Rfam: annotating non-coding RNAs in complete genomes; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=D121-4; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=Origin of replication in circular prokaryotic chromosomes; Environ. Microbiol.;  citation_year=2006;  citation_volume=8;  citation_pages=353-61; " /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison; BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_reference" content="citation_title=MBGD: a platform for microbial comparative genomics based on the automated construction of orthologous groups; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D343-6; " /><meta name="citation_reference" content="citation_title=Structural elements of the Streptomyces oriC region and their interactions with the DnaA protein; Microbiology;  citation_year=1998;  citation_volume=144;  citation_issue=Pt 5;  citation_pages=1281-90; " /><meta name="citation_reference" content="citation_title=Recruitment of terminal protein to the ends of Streptomyces linear plasmids and chromosomes by a novel telomere-binding protein essential for linear DNA replication; Genes Dev.;  citation_year=2003;  citation_volume=17;  citation_pages=774-85; " /><meta name="citation_reference" content="citation_title=The telomeres of Streptomyces chromosomes contain conserved palindromic sequences with potential to form complex secondary structures; Mol. Microbiol.;  citation_year=1998;  citation_volume=28;  citation_pages=905-16; " /><meta name="citation_reference" content="citation_title=Toward automatic reconstruction of a highly resolved tree of life; Science;  citation_year=2006;  citation_volume=311;  citation_pages=1283-7; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Catenulispora acidiphila type strain (ID 139908T); Stand. Genomic Sci.;  citation_year=2009;  citation_volume=1;  citation_pages=119-25; " /><meta name="citation_reference" content="citation_title=Genomics of Actinobacteria: tracing the evolutionary history of an ancient phylum; Microbiol. Mol. Biol. Rev.;  citation_year=2007;  citation_volume=71;  citation_pages=495-548; " /><meta name="citation_reference" content="citation_title=Morphogenetic surfactants and their role in the formation of aerial hyphae in Streptomyces coelicolor; Mol. Microbiol.;  citation_year=2006;  citation_volume=59;  citation_pages=731-42; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the erythromycin-producing bacterium Saccharopolyspora erythraea NRRL23338; Nat. Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=447-53; " /><meta name="citation_reference" content="citation_title=Evolutionary flux of potentially bldA-dependent Streptomyces genes containing the rare leucine codon TTA; Antonie Van Leeuwenhoek;  citation_year=2008;  citation_volume=94;  citation_pages=111-26; " /><meta name="citation_reference" content="citation_title=SapB and the chaplins: connections between morphogenetic proteins in Streptomyces coelicolor; Mol. Microbiol.;  citation_year=2007;  citation_volume=64;  citation_pages=602-13; " /><meta name="citation_reference" content="citation_title=A novel class of secreted hydrophobic proteins is involved in aerial hyphae formation in Streptomyces coelicolor by forming amyloid-like fibrils; Genes Dev.;  citation_year=2003;  citation_volume=17;  citation_pages=1714-26; " /><meta name="citation_reference" content="citation_title=Cloning and functional analysis by gene disruption of a gene encoding a γ-butyrolactone autoregulator receptor from Kitasatospora setae; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=3423-30; " /><meta name="citation_reference" content="citation_title=Biosynthesis of gamma-butyrolactone autoregulators that switch on secondary metabolism and morphological development in Streptomyces; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=2378-83; " /><meta name="citation_reference" content="citation_title=Molecular cloning, characterization, and chromosomal localization of dapF, the Escherichia coli gene for diaminopimelate epimerase; J. Bacteriol.;  citation_year=1987;  citation_volume=169;  citation_pages=1454-59; " /><meta name="citation_reference" content="citation_title=Specificity of the uridine-diphosphate-N-acetylmuramyl-L-alanyl-D-glutamate: meso-2,6-diaminopimelate synthetase from Escherichia coli; Eur. J. Biochem.;  citation_year=1985;  citation_volume=153;  citation_pages=81-7; " /><meta name="citation_reference" content="citation_title=Incorporation of LL-diaminopimelic acid into peptidoglycan of Escherichia coli mutants lacking diaminopimelate epimerase encoded by dapF; J. Bacteriol.;  citation_year=1988;  citation_volume=170;  citation_pages=2031-9; " /><meta name="citation_reference" content="citation_title=Cell division during growth and development of Streptomyces; Recent Res. Devel. Bacteriol.;  citation_year=2003;  citation_volume=1;  citation_pages=71-90; " /><meta name="citation_reference" content="citation_title=Visualization of penicillin-binding proteins during sporulation of Streptomyces griseus; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=2125-32; " /><meta name="citation_reference" content="citation_title=SsgA-like proteins determine the fate of peptidoglycan during sporulation of Streptomyces coelicolor; Mol. Microbiol.;  citation_year=2005;  citation_volume=58;  citation_pages=929-44; " /><meta name="citation_reference" content="citation_title=Isolation of new actinomycete strains for the screening of new bioactive compounds; J. Gen. Appl. Microbiol.;  citation_year=2003;  citation_volume=49;  citation_pages=141-54; " /><meta name="citation_reference" content="citation_title=Setamycin, a new antibiotic; J. Antibiot. (Tokyo);  citation_year=1981;  citation_volume=34;  citation_pages=1253-56; " /><meta name="citation_reference" content="citation_title=Genomic basis for natural product biosynthetic diversity in the actinomycetes; Nat. Prod. Rep.;  citation_year=2009;  citation_volume=26;  citation_pages=1362-84; " /><meta name="citation_reference" content="citation_title=Identification and functional analysis of genes controlling biosynthesis of 2-methylisoborneol; Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=7422-7; " /><meta name="citation_reference" content="citation_title=Geosmin biosynthesis in Streptomyces avermitilis. Molecular cloning, expression, and mechanistic study of the germacradienol/geosmin synthase; J. Antibiot. (Tokyo);  citation_year=2006;  citation_volume=59;  citation_pages=471-9; " /><meta name="citation_reference" content="citation_title=Biosynthesis of the earthy odorant geosmin by a bifunctional Streptomyces coelicolor enzyme; Nat. Chem. Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=711-5; " /><meta name="citation_reference" content="citation_title=Molecular analysis of the kirromycin biosynthetic gene cluster revealed beta-alanine as precursor of the pyridone moiety; Chem. Biol.;  citation_year=2008;  citation_volume=15;  citation_pages=175-88; " /><meta name="citation_reference" content="citation_title=Identification and characterization of genes required for biosynthesis and transport of the siderophore vibrioferrin in Vibrio parahaemolyticus; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=6938-49; " /><meta name="citation_reference" content="citation_title=Genetic organization of the region encoding regulation, biosynthesis, and transport of rhizobactin 1021, a siderophore produced by Sinorhizobium meliloti; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=2576-85; " /><meta name="citation_reference" content="citation_title=Nocardamin production by Streptomyces avermitilis; Actinomycetologica;  citation_year=2009;  citation_volume=23;  citation_pages=34-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/6/393/1310605/dsq026.pdf" /><meta name="description" content="Abstract.  Kitasatospora setae NBRC 14216T (=KM-6054T) is known to produce setamycin (bafilomycin B1) possessing antitrichomonal activity. The genus Kitasatospo" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/6/393/484773" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/6/393/484773" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/6/393/484773");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq026';
    </script>
<title>
	Genome Sequence of Kitasatospora setae NBRC 14216T: An Evolutionary Snapshot of the Family Streptomycetaceae | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484773" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ24ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSWNoaWthd2EsIE5hdHN1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOSVRFIEJpb3Jlc291cmNlIEluZm9ybWF0aW9uIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZhbHVhdGlvbiwgMi00OS0xMCBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ndWNoaSwgQWtpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvcmVzb3VyY2UgSW5mb3JtYXRpb24gQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSWtlZGEsIEhhcnVvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCAxLTE1LTEgS2l0YXNhdG8sIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDI1Mi0wMzczLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklzaGlrYXdhLCBKdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2FjdGl2ZSBNb2xlY3VsZXMsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCAxLTIzLTEgVG95YW1hLCBTaGluanVrdS1rdSwgVG9reW8gMTYyLTg2NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2l0YW5pLCBTaGlnZXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDZW50ZXIgZm9yIEJpb3RlY2hub2xvZ3ksIE9zYWthIFVuaXZlcnNpdHksIDItMSBZYW1hZGFva2EsIFN1aXRhLCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYXRhbmFiZSwgWXVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvcmVzb3VyY2UgSW5mb3JtYXRpb24gQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW11cmEsIFNhbmFlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTklURSBCaW9yZXNvdXJjZSBJbmZvcm1hdGlvbiBDZW50ZXIsIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2YWx1YXRpb24sIDItNDktMTAgTmlzaGloYXJhLCBTaGlidXlhLWt1LCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXRhbm8sIFlva28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOSVRFIEJpb3Jlc291cmNlIEluZm9ybWF0aW9uIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZhbHVhdGlvbiwgMi00OS0xMCBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iktpc2hpLCBFbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOSVRFIEJpb3Jlc291cmNlIEluZm9ybWF0aW9uIENlbnRlciwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZhbHVhdGlvbiwgMi00OS0xMCBOaXNoaWhhcmEsIFNoaWJ1eWEta3UsIFRva3lvIDE1MS0wMDY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhc2FnYXdhLCBNYWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvcmVzb3VyY2UgSW5mb3JtYXRpb24gQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQW5rYWksIEFraWhvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVG9ob2t1IFJlZ2lvbmFsIE9mZmljZSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2YWx1YXRpb24sIDQtNS0xOCBIaWdhc2hpc2VuZGFpLCBNaXlhZ2luby1rdSwgU2VuZGFpLCBNaXlhZ2kgOTgzLTA4MzMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVrdWksIFNoaWdlaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRvaG9rdSBSZWdpb25hbCBPZmZpY2UsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCA0LTUtMTggSGlnYXNoaXNlbmRhaSwgTWl5YWdpbm8ta3UsIFNlbmRhaSwgTWl5YWdpIDk4My0wODMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhhc2hpbW90bywgWW9zaGltaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvcmVzb3VyY2UgSW5mb3JtYXRpb24gQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FtYXRhLCBTYWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvcmVzb3VyY2UgSW5mb3JtYXRpb24gQ2VudGVyLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YS1rdSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT3RvZ3VybywgTWlzYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5JVEUgQmlvbG9naWNhbCBSZXNvdXJjZSBDZW50ZXIgKE5CUkMpLCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5IGFuZCBFdmFsdWF0aW9uLCAyLTUtOCBLYXp1c2FrYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmlrYXdhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTklURSBCaW9yZXNvdXJjZSBJbmZvcm1hdGlvbiBDZW50ZXIsIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2YWx1YXRpb24sIDItNDktMTAgTmlzaGloYXJhLCBTaGlidXlhLWt1LCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWhpcmEsIFRha3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVybmF0aW9uYWwgQ2VudGVyIGZvciBCaW90ZWNobm9sb2d5LCBPc2FrYSBVbml2ZXJzaXR5LCAyLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9yaW5vdWNoaSwgU3VlaGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgMS0xLTEgWWF5b2ksIEJ1bmt5by1rdSwgVG9reW8gMTEzLTg2NTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2huaXNoaSwgWWFzdW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDEtMS0xIFlheW9pLCBCdW5reW8ta3UsIFRva3lvIDExMy04NjU3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhheWFrYXdhLCBNYXNheXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEFwcGxpZWQgQmlvbG9naWNhbCBTY2llbmNlcywgSW50ZXJkaXNjaXBsaW5hcnkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljaW5lIGFuZCBFbmdpbmVlcmluZywgVW5pdmVyc2l0eSBvZiBZYW1hbmFzaGksIFRha2VkYS00LCBLb2Z1LCBZYW1hbmFzaGkgNDAwLTg1MTEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3V6dXlhbWEsIFRvbW9oaXNhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvdGVjaG5vbG9neSBSZXNlYXJjaCBDZW50ZXIsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCAxLTEtMSBZYXlvaSwgQnVua3lvLWt1LCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBcmlzYXdhLCBBa2lyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb3Jlc291cmNlIExhYm9yYXRvcmllcywgTWVyY2lhbiBDb3JwLiwgMTgwOCBOYWthaXp1bWksIEl3YXRhLCBTaGl6dW9rYSA0MzgtMDA3OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb21vdG8sIEZ1bWlraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hZ2FzZSAmYW1wOyBDby4gTHRkLCBSZXNlYXJjaCBhbmQgRGV2ZWxvcG1lbnQgQ2VudGVyLCAyLTItMyBNdXJvdGFuaSwgTmlzaGkta3UsIEtvYmUsIEh5b2dvIDY1MS0yMjQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pdXJhLCBIaXJvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLaXRhc2F0byBJbnN0aXR1dGUgZm9yIExpZmUgU2NpZW5jZXMsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDUtOS0xIFNoaXJva2FuZSwgTWluYXRvLWt1LCBUb2t5byAxMDgtODY0MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaGFzaGksIFlva28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLaXRhc2F0byBJbnN0aXR1dGUgZm9yIExpZmUgU2NpZW5jZXMsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDUtOS0xIFNoaXJva2FuZSwgTWluYXRvLWt1LCBUb2t5byAxMDgtODY0MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWppdGEsIE5vYnV5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTklURSBCaW9yZXNvdXJjZSBJbmZvcm1hdGlvbiBDZW50ZXIsIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2YWx1YXRpb24sIDItNDktMTAgTmlzaGloYXJhLCBTaGlidXlhLWt1LCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZSBTZXF1ZW5jZSBvZiBLaXRhc2F0b3Nwb3JhIHNldGFlIE5CUkMgMTQyMTZUOiBBbiBFdm9sdXRpb25hcnkgU25hcHNob3Qgb2YgdGhlIEZhbWlseSBTdHJlcHRvbXljZXRhY2VhZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTAvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGRldmVsb3BtZW50IGluIFN0cmVwdG9teWNlcyBhbmFseXNlZCBieSBnZW5vbWUgY29tcGFyaXNvbnM7IEZFTVMgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9NjUxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaW5pbmcgYW5kIHBvbGlzaGluZyBvZiB0aGUgdHJlYXN1cmUgdHJvdmUgaW4gdGhlIGJhY3RlcmlhbCBnZW51cyBTdHJlcHRvbXljZXM7IEJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0yODMtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbW9kZWwgYWN0aW5vbXljZXRlIFN0cmVwdG9teWNlcyBjb2VsaWNvbG9yIEEzKDIpOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE3OyAgY2l0YXRpb25fcGFnZXM9MTQxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBhbmQgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIGluZHVzdHJpYWwgbWljcm9vcmdhbmlzbSBTdHJlcHRvbXljZXMgYXZlcm1pdGlsaXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01MjYtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgc3RyZXB0b215Y2luLXByb2R1Y2luZyBtaWNyb29yZ2FuaXNtIFN0cmVwdG9teWNlcyBncmlzZXVzIElGTyAxMzM1MDsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOTA7ICBjaXRhdGlvbl9wYWdlcz00MDUwLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UeXJvcGVwdGlucyBBIGFuZCBCLCBuZXcgcHJvdGVhc29tZSBpbmhpYml0b3JzIHByb2R1Y2VkIGJ5IEtpdGFzYXRvc3BvcmEgc3AuIE1LOTkzLWRGMi4gSUkuIFN0cnVjdHVyZSBkZXRlcm1pbmF0aW9uIGFuZCBzeW50aGVzaXM7IEouIEFudGliaW90LiAoVG9reW8pOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MTAwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFsb3NpbnMgQSBhbmQgQjogbmV3IGlzb2ZsYXZvbm9sIGdseWNvc2lkZXMgd2l0aCBwb3RlbnQgYW50aWZ1bmdhbCBhY3Rpdml0eSBmcm9tIEtpdGFzYXRvc3BvcmEga2lmdW5lbnNpcyBNSk0zNDEuIEkuIFRheG9ub215LCBmZXJtZW50YXRpb24sIGlzb2xhdGlvbiwgYW5kIGJpb2xvZ2ljYWwgYWN0aXZpdGllczsgSi4gQW50aWJpb3QuIChUb2t5byk7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz02MzMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2l0YXNhdG9zcG9yYSwgYSBuZXcgZ2VudXMgb2YgdGhlIG9yZGVyIEFjdGlub215Y2V0YWxlczsgSi4gQW50aWJpb3QuIChUb2t5byk7ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0xMDEzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRheG9ub21pYyBzdGF0dXMgb2YgS2l0YXNhdG9zcG9yYSwgYW5kIHByb3Bvc2VkIHVuaWZpY2F0aW9uIHdpdGggU3RyZXB0b215Y2VzIG9uIHRoZSBiYXNpcyBvZiBwaGVub3R5cGljIGFuZCAxNlMgclJOQSBhbmFseXNpcyBhbmQgZW1lbmRhdGlvbiBvZiBTdHJlcHRvbXljZXMgV2Frc21hbiBhbmQgSGVucmljaSAxOTQzLCAzMzlBTDsgSW50LiBKLiBTeXN0LiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9MTU2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRheG9ub21pYyByZXZpZXcgb2YgdGhlIGdlbmVyYSBLaXRhc2F0b3Nwb3JhIGFuZCBTdHJlcHRvdmVydGljaWxsaXVtIGJ5IGFuYWx5c2lzIG9mIHJpYm9zb21hbCBwcm90ZWluIEFULUwzMDsgSW50LiBKLiBTeXN0LiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9Mjg1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb3Bvc2FsIHRvIHJldml2ZSB0aGUgZ2VudXMgS2l0YXNhdG9zcG9yYSAoT211cmEsIFRha2FoYXNoaSwgSXdhaSwgYW5kIFRhbmFrYSAxOTgyKTsgSW50LiBKLiBTeXN0LiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9MTA0OC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVzaWduIGFuZCBldmFsdWF0aW9uIG9mIGFuIG9saWdvbnVjbGVvdGlkZS1taWNyb2FycmF5IGZvciB0aGUgZGV0ZWN0aW9uIG9mIGRpZmZlcmVudCBzcGVjaWVzIG9mIHRoZSBnZW51cyBLaXRhc2F0b3Nwb3JhOyBKLiBNaWNyb2Jpb2wuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz0yMjYtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVsZWN0cmljYWwgRE5BLWNoaXAtYmFzZWQgaWRlbnRpZmljYXRpb24gb2YgZGlmZmVyZW50IHNwZWNpZXMgb2YgdGhlIGdlbnVzIEtpdGFzYXRvc3BvcmE7IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz0xMTgxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlbGF0aW9uc2hpcCBiZXR3ZWVuIGNlbGwgbW9ycGhvbG9neSBhbmQgdGhlIHR5cGVzIG9mIGRpYW1pbm9waW1lbGljIGFjaWQgaW4gS2l0YXNhdG9zcG9yYSBzZXRhbGJhOyBKLiBHZW4uIEFwcGwuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz00NTktNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbWUgY2hhcmFjdGVyaXN0aWNzIG9mIGFlcmlhbCBhbmQgc3VibWVyZ2VkIHNwb3JlcyBvZiBLaXRhc2F0b3Nwb3JhIHNldGFsYmE7IEouIEdlbi4gQXBwbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIyMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UeXBlIG9mIGRpYW1pbm9waW1lbGljIGFjaWQgZGlmZmVyZW50IGluIGFlcmlhbCBhbmQgdmVnZXRhdGl2ZSBteWNlbGlhIG9mIHNldGFteWNpbi1wcm9kdWNpbmcgYWN0aW5vbXljZXRlIEtNLTYwNTQ7IEouIEFudGliaW90LiAoVG9reW8pOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTYzMy00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXhvbm9taWMgc3R1ZHkgYW5kIG1vcnBob2xvZ2ljYWwgZGlmZmVyZW50aWF0aW9uIG9mIGFuIGFjdGlub215Y2V0ZSBnZW51cywgS2l0YXNhdG9zcG9yYTsgWmVudHJhbGJsLiBCYWt0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4OTsgIGNpdGF0aW9uX3BhZ2VzPTI2NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UgYW5hbHlzaXMgb2YgdGhyZWUgcGxhc21pZHMgaGFyYm91cmVkIGluIFJob2RvY29jY3VzIGVyeXRocm9wb2xpcyBzdHJhaW4gUFI0OyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMzQtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgc29pbCBhY3Rpbm9teWNldGUgS29jdXJpYSByaGl6b3BoaWxhOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTQxMzktNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UtY2FsbGluZyBvZiBhdXRvbWF0ZWQgc2VxdWVuY2VyIHRyYWNlcyB1c2luZyBwaHJlZC4gSUkuIEVycm9yIHByb2JhYmlsaXRpZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xODYtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UtY2FsbGluZyBvZiBhdXRvbWF0ZWQgc2VxdWVuY2VyIHRyYWNlcyB1c2luZyBwaHJlZC4gSS4gQWNjdXJhY3kgYXNzZXNzbWVudDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE3NS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgYmFjdGVyaWFsIGdlbmVzIGFuZCBlbmRvc3ltYmlvbnQgRE5BIHdpdGggR2xpbW1lcjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02NzMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GcmFtZVBsb3Q6IGEgbmV3IGltcGxlbWVudGF0aW9uIG9mIHRoZSBmcmFtZSBhbmFseXNpcyBmb3IgcHJlZGljdGluZyBwcm90ZWluLWNvZGluZyByZWdpb25zIGluIGJhY3RlcmlhbCBETkEgd2l0aCBhIGhpZ2ggRyArIEMgY29udGVudDsgRkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE3NDsgIGNpdGF0aW9uX3BhZ2VzPTI1MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdW5pdmVyc2FsIHByb3RlaW4gcmVzb3VyY2UgKFVuaVByb3QpOyBjaXRhdGlvbl9hdXRob3I9VW5pUHJvdCwgQ29uc29ydGl1bTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDE5MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZGV2ZWxvcG1lbnRzIGluIHRoZSBJbnRlclBybyBkYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDIyNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IQU1BUDogYSBkYXRhYmFzZSBvZiBjb21wbGV0ZWx5IHNlcXVlbmNlZCBtaWNyb2JpYWwgcHJvdGVvbWUgc2V0cyBhbmQgbWFudWFsbHkgY3VyYXRlZCBtaWNyb2JpYWwgcHJvdGVpbiBmYW1pbGllcyBpbiBVbmlQcm90S0IvU3dpc3MtUHJvdDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDQ3MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHIGZvciBsaW5raW5nIGdlbm9tZXMgdG8gbGlmZSBhbmQgdGhlIGVudmlyb25tZW50OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENDgwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9ydERCOiBhIGNvbXByZWhlbnNpdmUgZGF0YWJhc2UgcmVzb3VyY2UgZm9yIGN5dG9wbGFzbWljIG1lbWJyYW5lIHRyYW5zcG9ydCBzeXN0ZW1zIGFuZCBvdXRlciBtZW1icmFuZSBjaGFubmVsczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDI3NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRVJPUFM6IHRoZSBwZXB0aWRhc2UgZGF0YWJhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPUQzMjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmZhbTogYW5ub3RhdGluZyBub24tY29kaW5nIFJOQXMgaW4gY29tcGxldGUgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9RDEyMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVJBR09STiwgYSBwcm9ncmFtIHRvIGRldGVjdCB0Uk5BIGdlbmVzIGFuZCB0bVJOQSBnZW5lcyBpbiBudWNsZW90aWRlIHNlcXVlbmNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIHJlcGxpY2F0aW9uIGluIGNpcmN1bGFyIHByb2thcnlvdGljIGNocm9tb3NvbWVzOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zNTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZU1hdGNoZXI6IGEgZ3JhcGhpY2FsIHVzZXIgaW50ZXJmYWNlIGZvciBETkEgc2VxdWVuY2UgY29tcGFyaXNvbjsgQk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQkdEOiBhIHBsYXRmb3JtIGZvciBtaWNyb2JpYWwgY29tcGFyYXRpdmUgZ2Vub21pY3MgYmFzZWQgb24gdGhlIGF1dG9tYXRlZCBjb25zdHJ1Y3Rpb24gb2Ygb3J0aG9sb2dvdXMgZ3JvdXBzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1EMzQzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgZWxlbWVudHMgb2YgdGhlIFN0cmVwdG9teWNlcyBvcmlDIHJlZ2lvbiBhbmQgdGhlaXIgaW50ZXJhY3Rpb25zIHdpdGggdGhlIERuYUEgcHJvdGVpbjsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE0NDsgIGNpdGF0aW9uX2lzc3VlPVB0IDU7ICBjaXRhdGlvbl9wYWdlcz0xMjgxLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNydWl0bWVudCBvZiB0ZXJtaW5hbCBwcm90ZWluIHRvIHRoZSBlbmRzIG9mIFN0cmVwdG9teWNlcyBsaW5lYXIgcGxhc21pZHMgYW5kIGNocm9tb3NvbWVzIGJ5IGEgbm92ZWwgdGVsb21lcmUtYmluZGluZyBwcm90ZWluIGVzc2VudGlhbCBmb3IgbGluZWFyIEROQSByZXBsaWNhdGlvbjsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTc3NC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRlbG9tZXJlcyBvZiBTdHJlcHRvbXljZXMgY2hyb21vc29tZXMgY29udGFpbiBjb25zZXJ2ZWQgcGFsaW5kcm9taWMgc2VxdWVuY2VzIHdpdGggcG90ZW50aWFsIHRvIGZvcm0gY29tcGxleCBzZWNvbmRhcnkgc3RydWN0dXJlczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9OTA1LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgYXV0b21hdGljIHJlY29uc3RydWN0aW9uIG9mIGEgaGlnaGx5IHJlc29sdmVkIHRyZWUgb2YgbGlmZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTE7ICBjaXRhdGlvbl9wYWdlcz0xMjgzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBDYXRlbnVsaXNwb3JhIGFjaWRpcGhpbGEgdHlwZSBzdHJhaW4gKElEIDEzOTkwOFQpOyBTdGFuZC4gR2Vub21pYyBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWNzIG9mIEFjdGlub2JhY3RlcmlhOiB0cmFjaW5nIHRoZSBldm9sdXRpb25hcnkgaGlzdG9yeSBvZiBhbiBhbmNpZW50IHBoeWx1bTsgTWljcm9iaW9sLiBNb2wuIEJpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3JwaG9nZW5ldGljIHN1cmZhY3RhbnRzIGFuZCB0aGVpciByb2xlIGluIHRoZSBmb3JtYXRpb24gb2YgYWVyaWFsIGh5cGhhZSBpbiBTdHJlcHRvbXljZXMgY29lbGljb2xvcjsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NzMxLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGVyeXRocm9teWNpbi1wcm9kdWNpbmcgYmFjdGVyaXVtIFNhY2NoYXJvcG9seXNwb3JhIGVyeXRocmFlYSBOUlJMMjMzMzg7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00NDctNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBmbHV4IG9mIHBvdGVudGlhbGx5IGJsZEEtZGVwZW5kZW50IFN0cmVwdG9teWNlcyBnZW5lcyBjb250YWluaW5nIHRoZSByYXJlIGxldWNpbmUgY29kb24gVFRBOyBBbnRvbmllIFZhbiBMZWV1d2VuaG9lazsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTExMS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2FwQiBhbmQgdGhlIGNoYXBsaW5zOiBjb25uZWN0aW9ucyBiZXR3ZWVuIG1vcnBob2dlbmV0aWMgcHJvdGVpbnMgaW4gU3RyZXB0b215Y2VzIGNvZWxpY29sb3I7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTYwMi0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBjbGFzcyBvZiBzZWNyZXRlZCBoeWRyb3Bob2JpYyBwcm90ZWlucyBpcyBpbnZvbHZlZCBpbiBhZXJpYWwgaHlwaGFlIGZvcm1hdGlvbiBpbiBTdHJlcHRvbXljZXMgY29lbGljb2xvciBieSBmb3JtaW5nIGFteWxvaWQtbGlrZSBmaWJyaWxzOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTcxNC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgZnVuY3Rpb25hbCBhbmFseXNpcyBieSBnZW5lIGRpc3J1cHRpb24gb2YgYSBnZW5lIGVuY29kaW5nIGEgzrMtYnV0eXJvbGFjdG9uZSBhdXRvcmVndWxhdG9yIHJlY2VwdG9yIGZyb20gS2l0YXNhdG9zcG9yYSBzZXRhZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xODY7ICBjaXRhdGlvbl9wYWdlcz0zNDIzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9zeW50aGVzaXMgb2YgZ2FtbWEtYnV0eXJvbGFjdG9uZSBhdXRvcmVndWxhdG9ycyB0aGF0IHN3aXRjaCBvbiBzZWNvbmRhcnkgbWV0YWJvbGlzbSBhbmQgbW9ycGhvbG9naWNhbCBkZXZlbG9wbWVudCBpbiBTdHJlcHRvbXljZXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MjM3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcsIGNoYXJhY3Rlcml6YXRpb24sIGFuZCBjaHJvbW9zb21hbCBsb2NhbGl6YXRpb24gb2YgZGFwRiwgdGhlIEVzY2hlcmljaGlhIGNvbGkgZ2VuZSBmb3IgZGlhbWlub3BpbWVsYXRlIGVwaW1lcmFzZTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjk7ICBjaXRhdGlvbl9wYWdlcz0xNDU0LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGVjaWZpY2l0eSBvZiB0aGUgdXJpZGluZS1kaXBob3NwaGF0ZS1OLWFjZXR5bG11cmFteWwtTC1hbGFueWwtRC1nbHV0YW1hdGU6IG1lc28tMiw2LWRpYW1pbm9waW1lbGF0ZSBzeW50aGV0YXNlIGZyb20gRXNjaGVyaWNoaWEgY29saTsgRXVyLiBKLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTM7ICBjaXRhdGlvbl9wYWdlcz04MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNvcnBvcmF0aW9uIG9mIExMLWRpYW1pbm9waW1lbGljIGFjaWQgaW50byBwZXB0aWRvZ2x5Y2FuIG9mIEVzY2hlcmljaGlhIGNvbGkgbXV0YW50cyBsYWNraW5nIGRpYW1pbm9waW1lbGF0ZSBlcGltZXJhc2UgZW5jb2RlZCBieSBkYXBGOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTIwMzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbCBkaXZpc2lvbiBkdXJpbmcgZ3Jvd3RoIGFuZCBkZXZlbG9wbWVudCBvZiBTdHJlcHRvbXljZXM7IFJlY2VudCBSZXMuIERldmVsLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz03MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlzdWFsaXphdGlvbiBvZiBwZW5pY2lsbGluLWJpbmRpbmcgcHJvdGVpbnMgZHVyaW5nIHNwb3J1bGF0aW9uIG9mIFN0cmVwdG9teWNlcyBncmlzZXVzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTE4MDsgIGNpdGF0aW9uX3BhZ2VzPTIxMjUtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNzZ0EtbGlrZSBwcm90ZWlucyBkZXRlcm1pbmUgdGhlIGZhdGUgb2YgcGVwdGlkb2dseWNhbiBkdXJpbmcgc3BvcnVsYXRpb24gb2YgU3RyZXB0b215Y2VzIGNvZWxpY29sb3I7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTkyOS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIG5ldyBhY3Rpbm9teWNldGUgc3RyYWlucyBmb3IgdGhlIHNjcmVlbmluZyBvZiBuZXcgYmlvYWN0aXZlIGNvbXBvdW5kczsgSi4gR2VuLiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MTQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXRhbXljaW4sIGEgbmV3IGFudGliaW90aWM7IEouIEFudGliaW90LiAoVG9reW8pOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTI1My01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBiYXNpcyBmb3IgbmF0dXJhbCBwcm9kdWN0IGJpb3N5bnRoZXRpYyBkaXZlcnNpdHkgaW4gdGhlIGFjdGlub215Y2V0ZXM7IE5hdC4gUHJvZC4gUmVwLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEzNjItODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIGdlbmVzIGNvbnRyb2xsaW5nIGJpb3N5bnRoZXNpcyBvZiAyLW1ldGh5bGlzb2Jvcm5lb2w7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9NzQyMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW9zbWluIGJpb3N5bnRoZXNpcyBpbiBTdHJlcHRvbXljZXMgYXZlcm1pdGlsaXMuIE1vbGVjdWxhciBjbG9uaW5nLCBleHByZXNzaW9uLCBhbmQgbWVjaGFuaXN0aWMgc3R1ZHkgb2YgdGhlIGdlcm1hY3JhZGllbm9sL2dlb3NtaW4gc3ludGhhc2U7IEouIEFudGliaW90LiAoVG9reW8pOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NDcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb3N5bnRoZXNpcyBvZiB0aGUgZWFydGh5IG9kb3JhbnQgZ2Vvc21pbiBieSBhIGJpZnVuY3Rpb25hbCBTdHJlcHRvbXljZXMgY29lbGljb2xvciBlbnp5bWU7IE5hdC4gQ2hlbS4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTcxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYW5hbHlzaXMgb2YgdGhlIGtpcnJvbXljaW4gYmlvc3ludGhldGljIGdlbmUgY2x1c3RlciByZXZlYWxlZCBiZXRhLWFsYW5pbmUgYXMgcHJlY3Vyc29yIG9mIHRoZSBweXJpZG9uZSBtb2lldHk7IENoZW0uIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTc1LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBnZW5lcyByZXF1aXJlZCBmb3IgYmlvc3ludGhlc2lzIGFuZCB0cmFuc3BvcnQgb2YgdGhlIHNpZGVyb3Bob3JlIHZpYnJpb2ZlcnJpbiBpbiBWaWJyaW8gcGFyYWhhZW1vbHl0aWN1czsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz02OTM4LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIG9yZ2FuaXphdGlvbiBvZiB0aGUgcmVnaW9uIGVuY29kaW5nIHJlZ3VsYXRpb24sIGJpb3N5bnRoZXNpcywgYW5kIHRyYW5zcG9ydCBvZiByaGl6b2JhY3RpbiAxMDIxLCBhIHNpZGVyb3Bob3JlIHByb2R1Y2VkIGJ5IFNpbm9yaGl6b2JpdW0gbWVsaWxvdGk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTgzOyAgY2l0YXRpb25fcGFnZXM9MjU3Ni04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9jYXJkYW1pbiBwcm9kdWN0aW9uIGJ5IFN0cmVwdG9teWNlcyBhdmVybWl0aWxpczsgQWN0aW5vbXljZXRvbG9naWNhOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MzQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE3LzYvMzkzLzEzMTA2MDUvZHNxMDI2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBLaXRhc2F0b3Nwb3JhIHNldGFlIE5CUkMgMTQyMTZUICg9S00tNjA1NFQpIGlzIGtub3duIHRvIHByb2R1Y2Ugc2V0YW15Y2luIChiYWZpbG9teWNpbiBCMSkgcG9zc2Vzc2luZyBhbnRpdHJpY2hvbW9uYWwgYWN0aXZpdHkuIFRoZSBnZW51cyBLaXRhc2F0b3NwbyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDg0NzczJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZBT+p580K7eQf2GNMApCEEtVhY0H" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f393%2f484773" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f393%2f484773" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19064" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&amp;Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/6">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7367063">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367063">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367068">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367068">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367082">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367082">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367109">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367109">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367192">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367192">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367196">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367196">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7367199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7367199">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/6/381/484555">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome Sequence of <em>Kitasatospora setae</em> NBRC 14216<sup>T</sup>: An Evolutionary Snapshot of the Family <em>Streptomycetaceae</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Natsuko Ichikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Natsuko+Ichikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ichikawa N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ichikawa N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Natsuko Ichikawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akio Oguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akio+Oguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oguchi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oguchi A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akio Oguchi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruo Ikeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Kitasato Institute for Life Sciences</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 252-0373</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruo+Ikeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikeda H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikeda H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Haruo Ikeda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jun Ishikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Bioactive Molecules</div>, <div class="insititution">National Institute of Infectious Diseases</div>, <div class="addr-line">1-23-1 Toyama, Shinjuku-ku, Tokyo 162-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jun+Ishikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishikawa J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishikawa J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jun Ishikawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeru Kitani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">International Center for Biotechnology</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">2-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeru+Kitani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kitani S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kitani S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigeru Kitani</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yumi Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yumi+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yumi Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sanae Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sanae+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sanae Nakamura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoko Katano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoko+Katano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katano Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katano Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoko Katano</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Emi Kishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Emi+Kishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishi E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishi E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Emi Kishi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Machi Sasagawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Machi+Sasagawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasagawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasagawa M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Machi Sasagawa</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiho Ankai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Tohoku Regional Office, National Institute of Technology and Evaluation</div>, <div class="addr-line">4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiho+Ankai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ankai A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ankai A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiho Ankai</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigehiro Fukui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Tohoku Regional Office, National Institute of Technology and Evaluation</div>, <div class="addr-line">4-5-18 Higashisendai, Miyagino-ku, Sendai, Miyagi 983-0833</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigehiro+Fukui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukui S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukui S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shigehiro Fukui</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshimi Hashimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshimi+Hashimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hashimoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hashimoto Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshimi Hashimoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachi Kamata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachi+Kamata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kamata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kamata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachi Kamata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Misa Otoguro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">NITE Biological Resource Center (NBRC), Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-5-8 Kazusakamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Misa+Otoguro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Otoguro M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Otoguro M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Misa Otoguro</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tanikawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tanikawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanikawa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanikawa S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tanikawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takuya Nihira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">International Center for Biotechnology</div>, <div class="insititution">Osaka University</div>, <div class="addr-line">2-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takuya+Nihira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nihira T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nihira T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takuya Nihira</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sueharu Horinouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Biotechnology, Graduate School of Agriculture and Life Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sueharu+Horinouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Horinouchi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Horinouchi S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sueharu Horinouchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuo Ohnishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Biotechnology, Graduate School of Agriculture and Life Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuo+Ohnishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohnishi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohnishi Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yasuo Ohnishi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayuki Hayakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Division of Applied Biological Sciences, Interdisciplinary Graduate School of Medicine and Engineering</div>, <div class="insititution">University of Yamanashi</div>, <div class="addr-line">Takeda-4, Kofu, Yamanashi 400-8511</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayuki+Hayakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayakawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayakawa M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masayuki Hayakawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomohisa Kuzuyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="addr-line">Biotechnology Research Center</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomohisa+Kuzuyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuzuyama T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuzuyama T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomohisa Kuzuyama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akira Arisawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="insititution">Bioresource Laboratories</div>, <div class="insititution">Mercian Corp.</div>, <div class="addr-line">1808 Nakaizumi, Iwata, Shizuoka 438-0078</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akira+Arisawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arisawa A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arisawa A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akira Arisawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumiki Nomoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">11</div><div class="insititution">Nagase &amp; Co. Ltd</div>, <div class="insititution">Research and Development Center</div>, <div class="addr-line">2-2-3 Murotani, Nishi-ku, Kobe, Hyogo 651-2241</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumiki+Nomoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nomoto F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nomoto F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fumiki Nomoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromi Miura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">12</div><div class="addr-line">Kitasato Institute for Life Sciences</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">5-9-1 Shirokane, Minato-ku, Tokyo 108-8641</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromi+Miura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miura H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miura H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiromi Miura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoko Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">12</div><div class="addr-line">Kitasato Institute for Life Sciences</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">5-9-1 Shirokane, Minato-ku, Tokyo 108-8641</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoko+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoko Takahashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuyuki Fujita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">NITE Bioresource Information Center, Department of Biotechnology</div>, <div class="insititution">National Institute of Technology and Evaluation</div>, <div class="addr-line">2-49-10 Nishihara, Shibuya-ku, Tokyo 151-0066</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-3481-1933. Fax. <div class="fax">+81 3-3481-8424</div>. E-mail: <a href="mailto:fujita-nobuyuki@nite.go.jp" target="_blank">fujita-nobuyuki@nite.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuyuki+Fujita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujita N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujita N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuyuki Fujita</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 393–406, <a href='https://doi.org/10.1093/dnares/dsq026'>https://doi.org/10.1093/dnares/dsq026</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 November 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 July 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 September 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484773" href="/dnaresearch/article-pdf/17/6/393/1310605/dsq026.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Natsuko Ichikawa, Akio Oguchi, Haruo Ikeda, Jun Ishikawa, Shigeru Kitani, Yumi Watanabe, Sanae Nakamura, Yoko Katano, Emi Kishi, Machi Sasagawa, Akiho Ankai, Shigehiro Fukui, Yoshimi Hashimoto, Sachi Kamata, Misa Otoguro, Satoshi Tanikawa, Takuya Nihira, Sueharu Horinouchi, Yasuo Ohnishi, Masayuki Hayakawa, Tomohisa Kuzuyama, Akira Arisawa, Fumiki Nomoto, Hiromi Miura, Yoko Takahashi, Nobuyuki Fujita;  Genome Sequence of <em>Kitasatospora setae</em> NBRC 14216<sup>T</sup>: An Evolutionary Snapshot of the Family <em>Streptomycetaceae</em>, <em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 393–406, <a href='https://doi.org/10.1093/dnares/dsq026'>https://doi.org/10.1093/dnares/dsq026</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484773?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484773?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20Sequence%20of%20%26lt%3Bem%26gt%3BKitasatospora%20setae%26lt%3B%2Fem%26gt%3B%20NBRC%2014216%26lt%3Bsup%26gt%3BT%26lt%3B%2Fsup%26gt%3B%3A%20An%20Evolutionary%20Snapshot%20of%20the%20Family%20%26lt%3Bem%26gt%3BStreptomycetaceae%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2010-11-08&amp;volumeNum=17&amp;issueNum=6&amp;author=Ichikawa%2C%20Natsuko%3B%20Oguchi%2C%20Akio&amp;startPage=393&amp;endPage=406&amp;contentId=10.1093%2Fdnares%2Fdsq026&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f393%2f484773" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7367063 id="7367063" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Kitasatospora setae</em> NBRC 14216<sup>T</sup> (=KM-6054<sup>T</sup>) is known to produce setamycin (bafilomycin B1) possessing antitrichomonal activity. The genus <em>Kitasatospora</em> is morphologically similar to the genus <em>Streptomyces</em>, although they are distinguishable from each other on the basis of cell wall composition and the 16S rDNA sequence. We have determined the complete genome sequence of <em>K. setae</em> NBRC 14216<sup>T</sup> as the first <em>Streptomycetaceae</em> genome other than <em>Streptomyces</em>. The genome is a single linear chromosome of 8 783 278 bp with terminal inverted repeats of 127 148 bp, predicted to encode 7569 protein-coding genes, 9 rRNA operons, 1 tmRNA and 74 tRNA genes. Although these features resemble those of <em>Streptomyces</em>, genome-wide comparison of orthologous genes between <em>K. setae</em> and <em>Streptomyces</em> revealed smaller extent of synteny. Multilocus phylogenetic analysis based on amino acid sequences unequivocally placed <em>K. setae</em> outside the <em>Streptomyces</em> genus. Although many of the genes related to morphological differentiation identified in <em>Streptomyces</em> were highly conserved in <em>K. setae</em>, there were some differences such as the apparent absence of the AmfS (SapB) class of surfactant protein and differences in the copy number and variation of paralogous components involved in cell wall synthesis.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Kitasatospora setae&quot;"><em>Kitasatospora setae</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;complete genome&quot;">complete genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Streptomyces"><em>Streptomyces</em></a></div>                            <h2 scrollto-destination=7367068 id="7367068" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Among many filamentous bacteria belonging to the phylum <em>Actinobacteria</em>, <em>Streptomyces</em> species have been extensively studied because of the ability to produce various bioactive secondary metabolites and the complex process of morphological differentiation. The life cycle of <em>Streptomyces</em> is initiated by the germination of spores that develop into vegetative mycelia. In response to environmental signals, aerial mycelia emerge from the colony surface and differentiate into chains of spores. Many pioneering works about the regulation of secondary metabolism and differentiation of <em>Streptomyces</em> species were done using model organisms such as <em>S. coelicolor</em> A3(2) and <em>S. griseus</em> IFO 13350.<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ026C2" data-open="DSQ026C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The complete genome sequences of three <em>Streptomyces</em> species, <em>S. coelicolor</em> A3(2),<sup><a reveal-id="DSQ026C3" data-open="DSQ026C3" class="link link-ref link-reveal xref-bibr">3</a></sup><em>S. avermitilis</em> MA-4680<sup>T<a reveal-id="DSQ026C4" data-open="DSQ026C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and <em>S. griseus</em> IFO 13350,<sup><a reveal-id="DSQ026C5" data-open="DSQ026C5" class="link link-ref link-reveal xref-bibr">5</a></sup> have been reported and further accelerated the studies. <em>Kitasatospora setae</em> is a soil-habiting mycelial bacterium belonging to the same family <em>Streptomycetaceae</em>. All 23 validly published species belonging to the genus <em>Kitasatospora</em> exhibit a similar life style and morphology with <em>Streptomyces</em> species. <em>Kitasatospora</em> may also be comparable with <em>Streptomyces</em> in its capacity to produce bioactive secondary metabolites. The type strain of <em>K. setae</em>, NBRC 14216<sup>T</sup>, is known to produce setamycin (bafilomycin B1) and bafilomycin A1, specific inhibitors of vacuolar ATPase and commonly used as biochemical reagents for investigation of molecular transport in eukaryotic cells. This genus also includes several other strains reported as producers of bioactive compounds including a proteasome inhibitor and an anti-fungal agent.<sup><a reveal-id="DSQ026C6" data-open="DSQ026C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSQ026C7" data-open="DSQ026C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>After the first proposal of the genus <em>Kitasatospora</em> (originally <em>Kitasatosporia</em>) by Omura <em>et al</em>. in 1982,<sup><a reveal-id="DSQ026C8" data-open="DSQ026C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the taxonomic position of <em>Kitasatospora</em> had been under a debate. It was once reclassified as a synonym of <em>Streptomyces</em> based on morphology and partial 16S rDNA analysis reported by Wellington <em>et al</em>.<sup><a reveal-id="DSQ026C9" data-open="DSQ026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Ochi and Hiranuma.<sup><a reveal-id="DSQ026C10" data-open="DSQ026C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Afterward, Zhang <em>et al</em>.<sup><a reveal-id="DSQ026C11" data-open="DSQ026C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reported that <em>Streptomyces</em> and <em>Kitasatospora</em> form distinct phyletic groups in the detailed inspection of the 16S rDNA and the 16S–23S rDNA internal spacer region and proposed the revival of the genus <em>Kitasatospora</em>. Such a history made <em>Kitasatospora</em> a suitable model for the development of methods distinguishing closely related species and genera based on the DNA sequence.<sup><a reveal-id="DSQ026C12" data-open="DSQ026C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSQ026C13" data-open="DSQ026C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Besides the molecular phylogenetic approach, there are clear phenotypic criteria to distinguish <em>Kitasatospora</em> from <em>Streptomyces</em>, of which most notable is the chemical composition of cell wall peptidoglycan.<sup><a reveal-id="DSQ026C14" data-open="DSQ026C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSQ026C15" data-open="DSQ026C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Cell wall peptidoglycan of streptomycetes contains the <span class="small-caps">ll</span> isomer of diaminopimelic acid (DAP), whereas many other sporoactinomycetes contain the <em>meso</em> isomer. The peptidoglycan of <em>Kitasatospora</em>, in contrast, contains both <span class="small-caps">ll</span>- and <em>meso</em>-DAP.<sup><a reveal-id="DSQ026C16" data-open="DSQ026C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In <em>K. setae</em> NBRC 14216<sup>T</sup>, aerial spores on solid culture and submerged spores in liquid culture both contain exclusively <span class="small-caps">ll</span>-DAP, whereas mycelia in both cultures contain mainly <em>meso</em>-DAP.<sup><a reveal-id="DSQ026C14" data-open="DSQ026C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSQ026C15" data-open="DSQ026C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSQ026C17" data-open="DSQ026C17" class="link link-ref link-reveal xref-bibr">17</a></sup> This suggests that the composition of cell wall peptidoglycan would be regulated in <em>Kitasatospora</em> depending on differentiation stages. Many of the genes involved in the formation of aerial mycelium (<em>bld</em> genes and others) and sporulation (<em>whi</em> genes and others) have been identified by genetic analysis of <em>Streptomyces</em> species,<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ026C2" data-open="DSQ026C2" class="link link-ref link-reveal xref-bibr">2</a></sup> but the genetic and molecular basis for the differential incorporation of DAP isomers in <em>Kitasatospora</em> raises new questions. Considering the unique taxonomic position of <em>Kitasatospora</em>, as well as the common and distinct features of <em>Kitasatospora</em> and <em>Streptomyces</em>, <em>Kitasatospora</em> would be a key microorganism for further understanding of the evolution of not only actinobacteria within the family <em>Streptomycetaceae</em> but also other mycelial actinobacteria.</p><p>We determined the complete genome sequence of <em>K. setae</em> NBRC 14216<sup>T</sup> (=KM-6054<sup>T</sup>) and compared it with <em>Streptomyces</em> genomes. Although the overall topology and gene organization of the <em>K. setae</em> NBRC 14216<sup>T</sup> genome showed close resemblance to <em>Streptomyces</em> genomes, there are discriminative distances between them. We established a robust phylogenetic position of the genus <em>Kitasatospora</em> by multilocus phylogenetic analysis and confirmed the previous results based on 16S rDNA sequences. We also describe the conservation and variation of differentiation-related genes predicted from the annotation of <em>K. setae</em> NBRC 14216<sup>T</sup> genome, as well as the possible coding capacity of the genome for bioactive secondary metabolites.</p>                            <h2 scrollto-destination=7367082 id="7367082" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7367087 id="7367087" class="section-title" data-legacy-id=s2a>2.1. Genome sequencing, assembly and validation</h3>
<p>DNA shotgun libraries with average insert sizes of 1.6 and 6.5 kb were constructed in pUC118 (TaKaRa), whereas a fosmid library with average insert size of 37 kb was constructed in pCC1FOS (EPICENTRE) as described previously.<sup><a reveal-id="DSQ026C18" data-open="DSQ026C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSQ026C19" data-open="DSQ026C19" class="link link-ref link-reveal xref-bibr">19</a></sup> A total of 50 400 clones (34 560, 10 752 and 5088 clones from libraries with 1.6, 6.5 and 37 kb inserts, respectively) were subjected to sequencing from both ends of the inserts on either ABI 3730xl DNA Analyzer (Applied Biosystems) or Base Station DNA Fragment Analyzer BST-0100 (MJ Research, Inc.). Sequence reads were trimmed at a threshold quality value of 20 by Phred and assembled by Phrap and CONSED assembly tools.<sup><a reveal-id="DSQ026C20" data-open="DSQ026C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSQ026C21" data-open="DSQ026C21" class="link link-ref link-reveal xref-bibr">21</a></sup> For alignment and validation of contigs, Optical Mapping (OpGen) was used. Gaps between contigs were closed by sequencing PCR products, which bridge two neighboring contigs. Finally, each base of <em>K. setae</em> NBRC 14216<sup>T</sup> genome was ensured to be sequenced from multiple clones and from both directions with Phrap quality score ≥70 or from one direction with Phrap quality score ≥40. Chromosomal terminus was determined after attaching adenine and thymine homopolymers to the naked 3′ ends of the chromosome as described previously.<sup><a reveal-id="DSQ026C5" data-open="DSQ026C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p>                            <h3 scrollto-destination=7367093 id="7367093" class="section-title" data-legacy-id=s2b>2.2. Data analysis and annotation</h3>
<p>The prediction of open reading frames (ORFs) was performed using Glimmer3.<sup><a reveal-id="DSQ026C22" data-open="DSQ026C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The initial set of ORFs was manually selected from the prediction result in combination with BLASTP<sup><a reveal-id="DSQ026C23" data-open="DSQ026C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and FramePlot<sup><a reveal-id="DSQ026C24" data-open="DSQ026C24" class="link link-ref link-reveal xref-bibr">24</a></sup> results. Each ORF was annotated manually using in-house genome annotation system OCSS (unpublished). Similarity search results against Uniprot,<sup><a reveal-id="DSQ026C25" data-open="DSQ026C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Interpro<sup><a reveal-id="DSQ026C26" data-open="DSQ026C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and HAMAP<sup><a reveal-id="DSQ026C27" data-open="DSQ026C27" class="link link-ref link-reveal xref-bibr">27</a></sup> databases were used for functional prediction. The KEGG<sup><a reveal-id="DSQ026C28" data-open="DSQ026C28" class="link link-ref link-reveal xref-bibr">28</a></sup> database was used for the reconstruction of metabolic pathways. If necessary, annotation was confirmed by molecular phylogenetic analysis using ClustalW, NJplot or GARLI (<a class="link link-uri" href="http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html">http://www.bio.utexas.edu/faculty/antisense/garli/Garli.html</a>). Putative transporters and peptidases were independently evaluated using TransportDB<sup><a reveal-id="DSQ026C29" data-open="DSQ026C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and MEROPS<sup><a reveal-id="DSQ026C30" data-open="DSQ026C30" class="link link-ref link-reveal xref-bibr">30</a></sup> databases, respectively. Non-coding genes were predicted using the Rfam,<sup><a reveal-id="DSQ026C31" data-open="DSQ026C31" class="link link-ref link-reveal xref-bibr">31</a></sup> tRNAscan-SE<sup><a reveal-id="DSQ026C32" data-open="DSQ026C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and ARAGORN<sup><a reveal-id="DSQ026C33" data-open="DSQ026C33" class="link link-ref link-reveal xref-bibr">33</a></sup> programs. Putative <em>oriC</em> region was located using originx<sup><a reveal-id="DSQ026C34" data-open="DSQ026C34" class="link link-ref link-reveal xref-bibr">34</a></sup> program. Putative ORFs related to mobile genetic elements were predicted and their boundaries were inferred with the assistance of GenomeMatcher<sup><a reveal-id="DSQ026C35" data-open="DSQ026C35" class="link link-ref link-reveal xref-bibr">35</a></sup> software. For accurate assignment of orthologs from actinobacterial genomes, comparative data compiled in the MBGD<sup><a reveal-id="DSQ026C36" data-open="DSQ026C36" class="link link-ref link-reveal xref-bibr">36</a></sup> database were used with further molecular phylogenetic evaluation, if necessary.</p>                            <h3 scrollto-destination=7367100 id="7367100" class="section-title" data-legacy-id=s2c>2.3. Data and strain submission</h3>
<p>The nucleotide sequence of the <em>K. setae</em> NBRC 14216<sup>T</sup> genome has been deposited in the DDBJ/EMBL/GenBank databases under accession number AP010968. The annotated genome sequence is also available at the genome database DOGAN (<a class="link link-uri" href="http://www.bio.nite.go.jp/dogan/Top">http://www.bio.nite.go.jp/dogan/Top</a>). The microbial strain and genomic DNA clones used for the sequencing are available through the NBRC (NITE Biological Resource Center, Chiba, Japan, <a class="link link-uri" href="http://www.nbrc.nite.go.jp/e/index.html">http://www.nbrc.nite.go.jp/e/index.html</a>).</p>                            <h2 scrollto-destination=7367109 id="7367109" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7367115 id="7367115" class="section-title" data-legacy-id=s3a>3.1. General features of the K. setae NBRC 14216<sup>T</sup> genome</h3>
<p>The <em>K. setae</em> genome was composed of a single linear chromosome of 8 783 278 bp with 127 148 bp of terminal inverted repeats (TIRs). These characteristics in the genome topology were similar to those of <em>Streptomyces</em>,<sup><a reveal-id="DSQ026C3" data-open="DSQ026C3" class="link link-ref link-reveal xref-bibr">3</a>–<a reveal-id="DSQ026C5" data-open="DSQ026C5" class="link link-ref link-reveal xref-bibr">5</a></sup> although the microorganism does not harbor a linear or circular plasmid. The general features of the <em>K. setae</em> genome are summarized in Table <a reveal-id="DSQ026TB1" data-open="DSQ026TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSQ026F1" data-open="DSQ026F1" class="link link-reveal link-table xref-fig">1</a>. The chromosome was predicted to encode 74 tRNA genes, 9 copies of ribosomal RNA operon and 7569 protein-coding genes. Among the predicted protein-coding genes, 53.5% (4049 ORFs) were assigned putative functions. The average G + C content of the chromosome was 74.2%, which is among the highest in actinobacteria, being nearly equal to that of <em>Kineococcus radiotolerans</em> (<a class="link link-uri" href="http://genome.jgi-psf.org/finished_microbes/kinra/">http://genome.jgi-psf.org/finished_microbes/kinra/</a>). The putative replication origin containing 20 DnaA box-like sequences was located at the center of the chromosome. This putative origin was flanked by <em>dnaA</em> and <em>dnaN</em> as in the cases of almost all bacteria, including <em>Streptomyces</em> species.<sup><a reveal-id="DSQ026C37" data-open="DSQ026C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                        <a id="7367122" scrollto-destination="7367122"></a>
<div content-id="DSQ026TB1" class="table-modal"><div class="table-wrap"><div id="DSQ026TB1" class="table-wrap-title" data-id="DSQ026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of <em>K. setae</em> NBRC 14216<sup>T</sup> genome and <em>Streptomyces</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>TIR (bp) </th><th>G + C Content (%) </th><th>CDS (no.) </th><th>rRNA operons (no.) </th><th>tRNA genes (no.) </th><th>Average CDS length (bp) </th><th>Coding density (%) </th><th>Reciprocal BLAST best-hit pair (no.)<sup>a</sup> </th></tr></thead><tbody><tr><td><em>K. setae</em> NBRC 14216<sup>T</sup> </td><td>8 783 278 </td><td>127 148 </td><td>74.2 </td><td>7569 </td><td>9 </td><td>74 </td><td>1012 </td><td>87.0 </td><td>— </td></tr><tr><td><em>S. coelicolor</em> A3(2) </td><td>8 667 507 </td><td>21 653 </td><td>72.1 </td><td>7825 </td><td>6 </td><td>63 </td><td>991 </td><td>88.9 </td><td>3550 </td></tr><tr><td><em>S. avermitilis</em> MA-4680<sup>T</sup><sup>b</sup> </td><td>9 025 608 </td><td>49 </td><td>70.7 </td><td>7582 </td><td>6 </td><td>68 </td><td>1027 </td><td>86.3 </td><td>3498 </td></tr><tr><td><em>S. griseus</em> IFO 13350 </td><td>8 545 929 </td><td>132 910 </td><td>72.2 </td><td>7138 </td><td>6 </td><td>66 </td><td>1055 </td><td>88.1 </td><td>3513 </td></tr><tr><td><em>S. scabies</em> 87.22 </td><td>10 148 695 </td><td>18 488 </td><td>71.5 </td><td>8746 </td><td>6 </td><td>75 </td><td>1005 </td><td>86.2 </td><td>3534 </td></tr></tbody></table></div><div content-id="DSQ026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>TIR (bp) </th><th>G + C Content (%) </th><th>CDS (no.) </th><th>rRNA operons (no.) </th><th>tRNA genes (no.) </th><th>Average CDS length (bp) </th><th>Coding density (%) </th><th>Reciprocal BLAST best-hit pair (no.)<sup>a</sup> </th></tr></thead><tbody><tr><td><em>K. setae</em> NBRC 14216<sup>T</sup> </td><td>8 783 278 </td><td>127 148 </td><td>74.2 </td><td>7569 </td><td>9 </td><td>74 </td><td>1012 </td><td>87.0 </td><td>— </td></tr><tr><td><em>S. coelicolor</em> A3(2) </td><td>8 667 507 </td><td>21 653 </td><td>72.1 </td><td>7825 </td><td>6 </td><td>63 </td><td>991 </td><td>88.9 </td><td>3550 </td></tr><tr><td><em>S. avermitilis</em> MA-4680<sup>T</sup><sup>b</sup> </td><td>9 025 608 </td><td>49 </td><td>70.7 </td><td>7582 </td><td>6 </td><td>68 </td><td>1027 </td><td>86.3 </td><td>3498 </td></tr><tr><td><em>S. griseus</em> IFO 13350 </td><td>8 545 929 </td><td>132 910 </td><td>72.2 </td><td>7138 </td><td>6 </td><td>66 </td><td>1055 </td><td>88.1 </td><td>3513 </td></tr><tr><td><em>S. scabies</em> 87.22 </td><td>10 148 695 </td><td>18 488 </td><td>71.5 </td><td>8746 </td><td>6 </td><td>75 </td><td>1005 </td><td>86.2 </td><td>3534 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Numbers of best-hit pair between <em>K. setae</em> NBRC 14216<sup>T</sup> and each <em>Streptomyces</em> are calculated using BLASTP program with a threshold <em>E</em>-value of 1e<sup>−20</sup>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>On the basis of the latest annotation data of <em>S. avermitilis</em> MA-4680<sup>T</sup> maintained by H. Ikeda (<a class="link link-uri" href="http://avermitilis.ls.kitasato-u.ac.jp">http://avermitilis.ls.kitasato-u.ac.jp</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367122" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ026TB1" class="table-wrap-title" data-id="DSQ026TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General features of <em>K. setae</em> NBRC 14216<sup>T</sup> genome and <em>Streptomyces</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>TIR (bp) </th><th>G + C Content (%) </th><th>CDS (no.) </th><th>rRNA operons (no.) </th><th>tRNA genes (no.) </th><th>Average CDS length (bp) </th><th>Coding density (%) </th><th>Reciprocal BLAST best-hit pair (no.)<sup>a</sup> </th></tr></thead><tbody><tr><td><em>K. setae</em> NBRC 14216<sup>T</sup> </td><td>8 783 278 </td><td>127 148 </td><td>74.2 </td><td>7569 </td><td>9 </td><td>74 </td><td>1012 </td><td>87.0 </td><td>— </td></tr><tr><td><em>S. coelicolor</em> A3(2) </td><td>8 667 507 </td><td>21 653 </td><td>72.1 </td><td>7825 </td><td>6 </td><td>63 </td><td>991 </td><td>88.9 </td><td>3550 </td></tr><tr><td><em>S. avermitilis</em> MA-4680<sup>T</sup><sup>b</sup> </td><td>9 025 608 </td><td>49 </td><td>70.7 </td><td>7582 </td><td>6 </td><td>68 </td><td>1027 </td><td>86.3 </td><td>3498 </td></tr><tr><td><em>S. griseus</em> IFO 13350 </td><td>8 545 929 </td><td>132 910 </td><td>72.2 </td><td>7138 </td><td>6 </td><td>66 </td><td>1055 </td><td>88.1 </td><td>3513 </td></tr><tr><td><em>S. scabies</em> 87.22 </td><td>10 148 695 </td><td>18 488 </td><td>71.5 </td><td>8746 </td><td>6 </td><td>75 </td><td>1005 </td><td>86.2 </td><td>3534 </td></tr></tbody></table></div><div content-id="DSQ026TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>TIR (bp) </th><th>G + C Content (%) </th><th>CDS (no.) </th><th>rRNA operons (no.) </th><th>tRNA genes (no.) </th><th>Average CDS length (bp) </th><th>Coding density (%) </th><th>Reciprocal BLAST best-hit pair (no.)<sup>a</sup> </th></tr></thead><tbody><tr><td><em>K. setae</em> NBRC 14216<sup>T</sup> </td><td>8 783 278 </td><td>127 148 </td><td>74.2 </td><td>7569 </td><td>9 </td><td>74 </td><td>1012 </td><td>87.0 </td><td>— </td></tr><tr><td><em>S. coelicolor</em> A3(2) </td><td>8 667 507 </td><td>21 653 </td><td>72.1 </td><td>7825 </td><td>6 </td><td>63 </td><td>991 </td><td>88.9 </td><td>3550 </td></tr><tr><td><em>S. avermitilis</em> MA-4680<sup>T</sup><sup>b</sup> </td><td>9 025 608 </td><td>49 </td><td>70.7 </td><td>7582 </td><td>6 </td><td>68 </td><td>1027 </td><td>86.3 </td><td>3498 </td></tr><tr><td><em>S. griseus</em> IFO 13350 </td><td>8 545 929 </td><td>132 910 </td><td>72.2 </td><td>7138 </td><td>6 </td><td>66 </td><td>1055 </td><td>88.1 </td><td>3513 </td></tr><tr><td><em>S. scabies</em> 87.22 </td><td>10 148 695 </td><td>18 488 </td><td>71.5 </td><td>8746 </td><td>6 </td><td>75 </td><td>1005 </td><td>86.2 </td><td>3534 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Numbers of best-hit pair between <em>K. setae</em> NBRC 14216<sup>T</sup> and each <em>Streptomyces</em> are calculated using BLASTP program with a threshold <em>E</em>-value of 1e<sup>−20</sup>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>On the basis of the latest annotation data of <em>S. avermitilis</em> MA-4680<sup>T</sup> maintained by H. Ikeda (<a class="link link-uri" href="http://avermitilis.ls.kitasato-u.ac.jp">http://avermitilis.ls.kitasato-u.ac.jp</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367122" target="_blank">View Large</a></div></div>                        <a id="7367125" scrollto-destination="7367125"></a>
<div id="DSQ026F1" data-id="DSQ026F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02601.jpeg?Expires=2147483647&Signature=3AifHSGhWnYbPlDr0iQ4-muDpWBMKoXQZhIy~gfSu8hTIK4Ge9GV9s7zvWuOnB5D7-uAarSIFOs8t3kQte-fXOFcMQpRnNJV3k4NjRd336Wb1-c4oKK90vbaAEmPayq1kO64xaOwki4ATqYWdcX3DXi~rPmDdqgFlSvW2n5ek6lg-c1oPjb1S3wP5ypt1FTaZ-rVMkb04Vd3anumKFWHIzgTg807HwkCE1Rxtr3HW6-5qFWeyiYKlgRlLDmC3qRR-aUsh3HVl1CE-na4zQGHiNFzS7Gbi~47GZQAUl-DOi765H2tnOHIH44Uucg0Q1V9NRQKHCTRl6xjMbYamKV8fA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the K. setae NBRC 14216T chromosome. (i) G + C content. (ii) GC skew. (iii) ORFs encoded in forward (upper) and reverse (lower) strand. Each ORF is colored on the basis of the predicted function. (iv) RNA encoding genes. rRNA operons are colored in red and tRNAs are colored in blue. (v) Putative insertion sequences. (vi) Secondary metabolism gene clusters. (vii) Red bars indicate ORFs conserved commonly in all four Streptomyces genomes." path-from-xml="dsq02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367125/dsq02601.jpeg" path-from-xml="dsq02601.jpeg" target="_blank">View large</a><a section="7367125" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02601.jpeg?Expires=2147483647&Signature=H8RpbVegKPZNKR06okUjqzMZvbvZ1TcXRwV00E-O0Ijw59p~hDj278gXzztmvKNKa4KNb1crto2~bKWVBK7W7xBpzwR3n4xxb6RBE~MaEtUHI6BPzLQTTyWHLkLLu1adCnTgwsX26HeWSVzndbisH~MlMMrVYGjcqYkxCMnRijXsmoJvRiyTQy5KSretmWmvEf3tyZwJn4-Rs5JLRNP9Z9qOCo4W8uWTXNw9Zck3B7wmQniRfD7SU~-2g4591Crl7VOD5HZgnDCZzjtm00-eT6TX4E8R-IaGVKhdul2UsFaJ5X~KfQ1B4gWN3zKkTjbudbieRN0i~nkfpFIOYo9vDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367125&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the <em>K. setae</em> NBRC 14216<sup>T</sup> chromosome. (i) G + C content. (ii) GC skew. (iii) ORFs encoded in forward (upper) and reverse (lower) strand. Each ORF is colored on the basis of the predicted function. (iv) RNA encoding genes. rRNA operons are colored in red and tRNAs are colored in blue. (v) Putative insertion sequences. (vi) Secondary metabolism gene clusters. (vii) Red bars indicate ORFs conserved commonly in all four <em>Streptomyces</em> genomes.</p></div></div><div content-id="DSQ026F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02601.jpeg?Expires=2147483647&Signature=3AifHSGhWnYbPlDr0iQ4-muDpWBMKoXQZhIy~gfSu8hTIK4Ge9GV9s7zvWuOnB5D7-uAarSIFOs8t3kQte-fXOFcMQpRnNJV3k4NjRd336Wb1-c4oKK90vbaAEmPayq1kO64xaOwki4ATqYWdcX3DXi~rPmDdqgFlSvW2n5ek6lg-c1oPjb1S3wP5ypt1FTaZ-rVMkb04Vd3anumKFWHIzgTg807HwkCE1Rxtr3HW6-5qFWeyiYKlgRlLDmC3qRR-aUsh3HVl1CE-na4zQGHiNFzS7Gbi~47GZQAUl-DOi765H2tnOHIH44Uucg0Q1V9NRQKHCTRl6xjMbYamKV8fA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of the K. setae NBRC 14216T chromosome. (i) G + C content. (ii) GC skew. (iii) ORFs encoded in forward (upper) and reverse (lower) strand. Each ORF is colored on the basis of the predicted function. (iv) RNA encoding genes. rRNA operons are colored in red and tRNAs are colored in blue. (v) Putative insertion sequences. (vi) Secondary metabolism gene clusters. (vii) Red bars indicate ORFs conserved commonly in all four Streptomyces genomes." path-from-xml="dsq02601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367125/dsq02601.jpeg" path-from-xml="dsq02601.jpeg" target="_blank">View large</a><a section="7367125" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02601.jpeg?Expires=2147483647&Signature=H8RpbVegKPZNKR06okUjqzMZvbvZ1TcXRwV00E-O0Ijw59p~hDj278gXzztmvKNKa4KNb1crto2~bKWVBK7W7xBpzwR3n4xxb6RBE~MaEtUHI6BPzLQTTyWHLkLLu1adCnTgwsX26HeWSVzndbisH~MlMMrVYGjcqYkxCMnRijXsmoJvRiyTQy5KSretmWmvEf3tyZwJn4-Rs5JLRNP9Z9qOCo4W8uWTXNw9Zck3B7wmQniRfD7SU~-2g4591Crl7VOD5HZgnDCZzjtm00-eT6TX4E8R-IaGVKhdul2UsFaJ5X~KfQ1B4gWN3zKkTjbudbieRN0i~nkfpFIOYo9vDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367125&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of the <em>K. setae</em> NBRC 14216<sup>T</sup> chromosome. (i) G + C content. (ii) GC skew. (iii) ORFs encoded in forward (upper) and reverse (lower) strand. Each ORF is colored on the basis of the predicted function. (iv) RNA encoding genes. rRNA operons are colored in red and tRNAs are colored in blue. (v) Putative insertion sequences. (vi) Secondary metabolism gene clusters. (vii) Red bars indicate ORFs conserved commonly in all four <em>Streptomyces</em> genomes.</p></div></div><p>The finished sequence consisted of a big contig representing the major part of the chromosome connected to the same small contig at both termini. As neither sequence variation nor assembly inconsistency was found in the terminal contig, we concluded that the chromosome has inverted identical sequences at both extremities. Terminal sequences of linear chromosomes and plasmids of <em>Streptomyces</em> and <em>Rhodococcus</em> can be classified into at least six groups. The terminal sequence of the <em>K. setae</em> chromosome was distinct from any of these groups; while the first 13 bp sequence exactly matched with that of major <em>Streptomyces</em> groups I and II (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), the subsequent region containing palindrome structures and loops, which are known in <em>Streptomyces</em> to be required for binding of telomere-associated protein (Tap), was not conserved.<sup><a reveal-id="DSQ026C38" data-open="DSQ026C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSQ026C39" data-open="DSQ026C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Tap and terminal protein (Tpg) encoding genes (KSE_73020 and KSE_73030) were detected in the <em>K. setae</em> chromosome with lower similarity (42–45% amino acid identity) to those of <em>Streptomyces. Kitasatospora setae</em> thus seems to have similar mechanisms for chromosome maintenance; it possesses a linear chromosome with TIRs, replicates bidirectionally from a centrally located <em>oriC</em> region and maintains terminal sequences by Tap and Tpg. However, the binding specificity of Tap might be different even from those recognizing group I and II replicons of <em>Streptomyces</em>.</p><p>No critical difference was found between <em>K. setae</em> and <em>Streptomyces</em> species in the predicted primary metabolism such as carbohydrate metabolism, amino acid metabolism, nucleic acid metabolism and respiration. The numbers of other ubiquitous components, such as membrane transporters, peptidases, transcriptional regulators and sigma factors, were also almost equivalent to those of <em>Streptomyces</em>.</p>                            <h3 scrollto-destination=7367136 id="7367136" class="section-title" data-legacy-id=s3b>3.2. Taxonomic reevaluation and comparative analysis of K. setae NBRC 14216<sup>T</sup></h3>
<p>From morphological similarity and rDNA relatedness to <em>Streptomyces</em> species, genus <em>Kitasatospora</em> was once regarded as a synonym of <em>Streptomyces</em>.<sup><a reveal-id="DSQ026C9" data-open="DSQ026C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Although phylogenetic analysis based on the 16S rDNA sequence usually separates <em>Streptomyces</em> and <em>Kitasatospora</em> species into distinct sister groups, the results sometimes depend on the choice of the outgroup<sup><a reveal-id="DSQ026C11" data-open="DSQ026C11" class="link link-ref link-reveal xref-bibr">11</a></sup> or the region used for the alignment, depicting the difficulties in determining correct taxonomic relationships only from the nucleotide sequences of rDNA. In order to obtain a more robust measure of the taxonomic position of <em>Kitasatospora</em> by taking advantage of its genomic information, we performed a multilocus phylogenetic analysis using 31 conserved amino acid sequences. Ciccarelli <em>et al</em>.<sup><a reveal-id="DSQ026C40" data-open="DSQ026C40" class="link link-ref link-reveal xref-bibr">40</a></sup> reported the construction of a tree of life across all three domains using 191 species including 14 actinobacteria. We adopted the same method by utilizing 58 actinobacterial genomes with the <em>Escherichia coli</em> genome as an outgroup (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S1 and S2</a>). Amino acid sequences were aligned for each of the 31 conserved protein genes, and then all 31 alignments were concatenated and ambiguous portions were deleted before performing phylogenetic reconstruction. Figure <a reveal-id="DSQ026F2" data-open="DSQ026F2" class="link link-reveal link-table xref-fig">2</a> shows a phylogenetic tree obtained by the neighbor-joining method (a phylogenetic tree obtained by the maximum-likelihood method is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Relationships between major taxonomic groups were broadly consistent with the previous results obtained by 16S rDNA sequences. Within the phylogenetic tree, 82% of all predicted branches were supported by bootstrap proportions greater than 90% (i.e. 900 of 1000). Four <em>Streptomyces</em> species were grouped in the same branch, with <em>S. griseus</em> IFO 13350 branching out at the deepest position. <em>Kitasatospora setae</em> NBRC 14216<sup>T</sup> was placed within the same clan as <em>Streptomyces</em> creating the outermost branch. All these results were supported by bootstrap proportions of 100%, reinforcing the idea that the genera <em>Kitasatospora</em> and <em>Streptomyces</em> were generated from a common progenitor and have diverged into distinct sister groups. The closest to this group was <em>Catenulispora acidiphila</em> DSM 44928,<sup><a reveal-id="DSQ026C41" data-open="DSQ026C41" class="link link-ref link-reveal xref-bibr">41</a></sup> a mycelial actinobacterium with a circular chromosome. </p>                        <a id="7367144" scrollto-destination="7367144"></a>
<div id="DSQ026F2" data-id="DSQ026F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02602.jpeg?Expires=2147483647&Signature=RndsYo9~CwYs7-ylxNksTUuMCphgZu~VtlhVcWAlQUM78gnfN595hK3v9gXyla0n-UvmeQBBJdZkmvwQTWDtY0aHofkJ0LHGjGCVIy-TLEkcg9TF1q4gMsphvmLpB0HzxZF0NltHJxuKprOyd5df17JTb99XChtJJWvjQ-5KZWnkVW0tGFaJjPgmL29tv2bWTlOAeo9AeMH~LzGPtpexwZ8a806DekfxQaNVKBG0jOlUsr8~k~0hRzHggY68zXkGoBO1QjHuFL5nb1gAoi6N5eWRHnDIim8JkWCgcMf~yfg11w4U79BYLBC6ezRM1j3NjCb65owPYAOikxOTm5PCyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree based on amino acid sequences of 31 protein-coding genes analyzed by the neighbor-joining method. Branches with less than 90% bootstrap support are represented in dashed lines. Lists of organisms and genes used for the analysis are shown in SupplementaryTables S1 and S2, respectively. Names of the organisms mentioned in the text are shown in bold type." path-from-xml="dsq02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367144/dsq02602.jpeg" path-from-xml="dsq02602.jpeg" target="_blank">View large</a><a section="7367144" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02602.jpeg?Expires=2147483647&Signature=z2klvYn3w8~4qYCPbh0KsexBCvu2g-esSrVeToHhofkgLv5Bq0pZR4ioJWl1Lpcl-fpJri6JDo3G4Knz1c6We75~6Qsa~AM-DYdcPkQgrms6WtANn9sJhiSKYOF3W6HxxCNaQVB0z0vLW~hjg9rd0SaAY9Y3ZNEbW9fLC9Z2bBozefiJgu6pTcDwGRqIiN4fFszLdEzPAA2OtMzsFgM~u1LqFgA3A1EDNTErQ57owX3lsWHjJMzi2Q8pryUT4S0YErCivoiSm3JTr4-mkoxgy~7t-GZkQ4PghfpfUd84za6WHwvqlVDWpJeZvlElKwvw8zJabXdnwRBOWARGWqUMEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367144&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree based on amino acid sequences of 31 protein-coding genes analyzed by the neighbor-joining method. Branches with less than 90% bootstrap support are represented in dashed lines. Lists of organisms and genes used for the analysis are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsq026/DC1" target="_blank">Tables S1 and S2</a>, respectively. Names of the organisms mentioned in the text are shown in bold type.</p></div></div><div content-id="DSQ026F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02602.jpeg?Expires=2147483647&Signature=RndsYo9~CwYs7-ylxNksTUuMCphgZu~VtlhVcWAlQUM78gnfN595hK3v9gXyla0n-UvmeQBBJdZkmvwQTWDtY0aHofkJ0LHGjGCVIy-TLEkcg9TF1q4gMsphvmLpB0HzxZF0NltHJxuKprOyd5df17JTb99XChtJJWvjQ-5KZWnkVW0tGFaJjPgmL29tv2bWTlOAeo9AeMH~LzGPtpexwZ8a806DekfxQaNVKBG0jOlUsr8~k~0hRzHggY68zXkGoBO1QjHuFL5nb1gAoi6N5eWRHnDIim8JkWCgcMf~yfg11w4U79BYLBC6ezRM1j3NjCb65owPYAOikxOTm5PCyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree based on amino acid sequences of 31 protein-coding genes analyzed by the neighbor-joining method. Branches with less than 90% bootstrap support are represented in dashed lines. Lists of organisms and genes used for the analysis are shown in SupplementaryTables S1 and S2, respectively. Names of the organisms mentioned in the text are shown in bold type." path-from-xml="dsq02602.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367144/dsq02602.jpeg" path-from-xml="dsq02602.jpeg" target="_blank">View large</a><a section="7367144" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02602.jpeg?Expires=2147483647&Signature=z2klvYn3w8~4qYCPbh0KsexBCvu2g-esSrVeToHhofkgLv5Bq0pZR4ioJWl1Lpcl-fpJri6JDo3G4Knz1c6We75~6Qsa~AM-DYdcPkQgrms6WtANn9sJhiSKYOF3W6HxxCNaQVB0z0vLW~hjg9rd0SaAY9Y3ZNEbW9fLC9Z2bBozefiJgu6pTcDwGRqIiN4fFszLdEzPAA2OtMzsFgM~u1LqFgA3A1EDNTErQ57owX3lsWHjJMzi2Q8pryUT4S0YErCivoiSm3JTr4-mkoxgy~7t-GZkQ4PghfpfUd84za6WHwvqlVDWpJeZvlElKwvw8zJabXdnwRBOWARGWqUMEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367144&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02602.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree based on amino acid sequences of 31 protein-coding genes analyzed by the neighbor-joining method. Branches with less than 90% bootstrap support are represented in dashed lines. Lists of organisms and genes used for the analysis are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/cgi/content/full/dsq026/DC1" target="_blank">Tables S1 and S2</a>, respectively. Names of the organisms mentioned in the text are shown in bold type.</p></div></div><p>For further analysis of the relationship between <em>K. setae</em> and <em>Streptomyces</em>, we compared all annotated proteins of <em>K. setae</em> and four <em>Streptomyces</em> species. More than half of ORFs predicted in <em>K. setae</em> had orthologs (reciprocal best-hit pairs) in each of four <em>Streptomyces</em> species using a BLASTP threshold of <em>E</em> &lt; 10<sup>−20</sup> (Table <a reveal-id="DSQ026TB1" data-open="DSQ026TB1" class="link link-reveal link-table xref-fig">1</a>). About 34% of <em>K. setae</em> ORFs had orthologs in all four <em>Streptomyces</em> genomes. The average amino acid identity between orthologous pairs from <em>K. setae</em> and <em>Streptomyces</em> species was around 60%. Despite such high similarities observed between orthologs, genome-wide comparison using ortholog plots demonstrated smaller extent of synteny between <em>K. setae</em> and <em>Streptomyces</em> genome (Fig. <a reveal-id="DSQ026F3" data-open="DSQ026F3" class="link link-reveal link-table xref-fig">3</a>A–C). Many short synteny blocks were observed along either of the diagonal lines, suggesting that frequent inversions around the replication origin have had occurred. This is in contrast to the higher extent of colinearities with only 2–4 inversions observed in comparative analysis among <em>Streptomyces</em> genomes.<sup><a reveal-id="DSQ026C3" data-open="DSQ026C3" class="link link-ref link-reveal xref-bibr">3</a>–<a reveal-id="DSQ026C5" data-open="DSQ026C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Conserved core region of the <em>K. setae</em> genome predicted by threading major synteny blocks was ∼5 Mb in length ranging from KSE_13770 to KSE_57600, which is ∼1 Mb smaller than the core regions deduced from comparison among <em>Streptomyces</em> genomes.<sup><a reveal-id="DSQ026C5" data-open="DSQ026C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Long-range synteny between <em>K. setae</em> and <em>C. acidiphila</em> genomes was much less obvious compared with that between <em>K. setae</em> and <em>Streptomyces</em> genomes (Fig. <a reveal-id="DSQ026F3" data-open="DSQ026F3" class="link link-reveal link-table xref-fig">3</a>D), although the number of orthologous gene pairs between <em>K. setae</em> and <em>C. acidiphila</em> (3185) was only 10% smaller than that between <em>K. setae</em> and <em>Streptomyces</em> (3498–3550, Table <a reveal-id="DSQ026TB1" data-open="DSQ026TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="7367154" scrollto-destination="7367154"></a>
<div id="DSQ026F3" data-id="DSQ026F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02603.jpeg?Expires=2147483647&Signature=IKYx-2q1w8oh0aDrXTWDI-0c--fsy46jsgTc1vq8fEoBYt9Tr8N9vv0DEN7nztzLp8HCpgw0pRPQ9EelccsZ3KsKSyCc9O1U7KuAXLx~XO7Vb6-jJ6wLQgiNU1XjgygkYyCzEkW~fQcJntRJg6KsAFMAaFyBa7dcdHPqDM6AQJAAcR~MoGTtrmLHX1I4uh85-E9JDk6oecdygz2wLqVlBNmAm0TxYBbXNtt7ePYE0UZPhVw8~7hPtqg9wegpchhkdG5BI0GWLRqJjgqLHTjybyRFgD-88gMQTc-0e0kIFNGzVAJ1Ysa6la1BDrio1Hg94txqWPKFBEC5Rm~ehkEyog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny between the genomes of K. setae NBRC 14216T and S. coelicolor A3(2) (A), K. setae NBRC 14216T and S. avermitilis MA-4680T (B), K. setae NBRC 14216T and S. griseus IFO 13350 (C) and K. setae NBRC 14216T and C. acidiphila DSM 44928 (D). Reciprocal BLAST best-hit pairs with a threshold value of E &lt; 10−20 were plotted. The direction of each chromosome was adjusted so that the dnaA gene faces the same direction. Green bar in each panel represents the conserved core region on the K. setae chromosome." path-from-xml="dsq02603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367154/dsq02603.jpeg" path-from-xml="dsq02603.jpeg" target="_blank">View large</a><a section="7367154" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02603.jpeg?Expires=2147483647&Signature=b~yaYXiYDa3JFPXdtMmugJgmwXJamXH~KUZ1jjhCFkI9LVg9hW2rAznEsK2~Wgryo818ZWhABAEgii5w8GiQUyZa5kZ2PLs3kDUcpTVz4zN5J0H~FuL1N-MgmAIlGKv9ZMY4vit~srZVUh~KqLICd~ecs9R0R1OPBNdS-cLBainJi0XT9C6APWuxkvzj~3T2JsIsxDlAospBsP6j8o54rubxdv~I-u4422sstuDlXAG~X6Bf8gtbag44lEfkIYjDzq1hH~koglsgkyQFyJiZzfpWlYJc6Vf8w-2H-5kC~08WUEjWiSpf6qlSGy3s5wvl7nvw1aSSe4Vowbt1twUOlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367154&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny between the genomes of <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. coelicolor</em> A3(2) (A), <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. avermitilis</em> MA-4680<sup>T</sup> (B), <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. griseus</em> IFO 13350 (C) and <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>C. acidiphila</em> DSM 44928 (D). Reciprocal BLAST best-hit pairs with a threshold value of <em>E</em> &lt; 10<sup>−20</sup> were plotted. The direction of each chromosome was adjusted so that the <em>dnaA</em> gene faces the same direction. Green bar in each panel represents the conserved core region on the <em>K. setae</em> chromosome.</p></div></div><div content-id="DSQ026F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02603.jpeg?Expires=2147483647&Signature=IKYx-2q1w8oh0aDrXTWDI-0c--fsy46jsgTc1vq8fEoBYt9Tr8N9vv0DEN7nztzLp8HCpgw0pRPQ9EelccsZ3KsKSyCc9O1U7KuAXLx~XO7Vb6-jJ6wLQgiNU1XjgygkYyCzEkW~fQcJntRJg6KsAFMAaFyBa7dcdHPqDM6AQJAAcR~MoGTtrmLHX1I4uh85-E9JDk6oecdygz2wLqVlBNmAm0TxYBbXNtt7ePYE0UZPhVw8~7hPtqg9wegpchhkdG5BI0GWLRqJjgqLHTjybyRFgD-88gMQTc-0e0kIFNGzVAJ1Ysa6la1BDrio1Hg94txqWPKFBEC5Rm~ehkEyog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Synteny between the genomes of K. setae NBRC 14216T and S. coelicolor A3(2) (A), K. setae NBRC 14216T and S. avermitilis MA-4680T (B), K. setae NBRC 14216T and S. griseus IFO 13350 (C) and K. setae NBRC 14216T and C. acidiphila DSM 44928 (D). Reciprocal BLAST best-hit pairs with a threshold value of E &lt; 10−20 were plotted. The direction of each chromosome was adjusted so that the dnaA gene faces the same direction. Green bar in each panel represents the conserved core region on the K. setae chromosome." path-from-xml="dsq02603.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367154/dsq02603.jpeg" path-from-xml="dsq02603.jpeg" target="_blank">View large</a><a section="7367154" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02603.jpeg?Expires=2147483647&Signature=b~yaYXiYDa3JFPXdtMmugJgmwXJamXH~KUZ1jjhCFkI9LVg9hW2rAznEsK2~Wgryo818ZWhABAEgii5w8GiQUyZa5kZ2PLs3kDUcpTVz4zN5J0H~FuL1N-MgmAIlGKv9ZMY4vit~srZVUh~KqLICd~ecs9R0R1OPBNdS-cLBainJi0XT9C6APWuxkvzj~3T2JsIsxDlAospBsP6j8o54rubxdv~I-u4422sstuDlXAG~X6Bf8gtbag44lEfkIYjDzq1hH~koglsgkyQFyJiZzfpWlYJc6Vf8w-2H-5kC~08WUEjWiSpf6qlSGy3s5wvl7nvw1aSSe4Vowbt1twUOlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367154&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02603.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Synteny between the genomes of <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. coelicolor</em> A3(2) (A), <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. avermitilis</em> MA-4680<sup>T</sup> (B), <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>S. griseus</em> IFO 13350 (C) and <em>K. setae</em> NBRC 14216<sup>T</sup> and <em>C. acidiphila</em> DSM 44928 (D). Reciprocal BLAST best-hit pairs with a threshold value of <em>E</em> &lt; 10<sup>−20</sup> were plotted. The direction of each chromosome was adjusted so that the <em>dnaA</em> gene faces the same direction. Green bar in each panel represents the conserved core region on the <em>K. setae</em> chromosome.</p></div></div>                            <h3 scrollto-destination=7367160 id="7367160" class="section-title" data-legacy-id=s3c>3.3. Conservation and variation of genes related to developmental regulation</h3>
<p><em>Streptomyces</em> is well characterized as a model organism by its complex life cycle; it grows as a thread-like mycelium and forms aerial mycelium in the air under nutrient-limited conditions.<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ026C2" data-open="DSQ026C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSQ026C42" data-open="DSQ026C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSQ026C43" data-open="DSQ026C43" class="link link-ref link-reveal xref-bibr">43</a></sup> A series of genes whose mutations cause defects in aerial growth (‘<em>bld</em>’ genes) were characterized in <em>S. coelicolor</em> A3(2). A complex interaction cascade among <em>bld</em> gene products results in the secretion of surfactant proteins, which assist <em>Streptomyces</em> in extending aerial mycelium upward and in differentiating into spores. A number of genes in ‘<em>whi</em>’ loci were also characterized mainly in <em>S. coelicolor</em> A3(2) as causing sporulation deficiency. Most of these classical differentiation genes were found to have orthologs in <em>K. setae</em>. The number of conserved <em>bld</em> and <em>whi</em> genes and their similarity (amino acid identity) to <em>S. coelicolor</em> A3(2) counterparts were higher than those in other mycelial actinobacteria, such as <em>C. acidiphila</em> DSM 44928, <em>Saccharopolyspora erythraea</em> NRRL 23338<sup><a reveal-id="DSQ026C44" data-open="DSQ026C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and <em>Salinispora arenicola</em> CNS 205 (Table <a reveal-id="DSQ026TB2" data-open="DSQ026TB2" class="link link-reveal link-table xref-fig">2</a>), consistent with the closer morphological similarity of <em>K. setae</em> to <em>Streptomyces</em>. Notably, <em>bld</em> gene pairs orthologous between <em>K. setae</em> and <em>Streptomyces</em> species represented 80–97% amino acid identities, which are much higher than the average (60%) of all orthologous pairs and comparable with those between <em>Streptomyces</em> species. Higher than the average conservation of <em>bld</em> genes was also observed between <em>C. acidiphila</em> and <em>Streptomyces</em> (58–95% amino acid identities). The <em>bldA</em> tRNA gene was also highly conserved in <em>K. setae</em>, with its cognate codon UUA being the rarest in <em>K. setae</em>; used only in 69 predicted ORFs, most of which were either of unknown function or with predicted regulatory functions. In four completely sequenced <em>Streptomyces</em> genomes, only three genes were found to have a conserved UUA codon at the same position in each ortholog.<sup><a reveal-id="DSQ026C45" data-open="DSQ026C45" class="link link-ref link-reveal xref-bibr">45</a></sup> One of these, the <em>bldH</em> gene, shared a UUA codon at an equivalent position also in <em>K. setae</em>. The high conservation of these regulatory components may further imply that each component in the regulatory cascade undergoes a higher than the usual number of interactions with other <em>bld</em> gene products or with other cellular components. </p>                        <a id="7367166" scrollto-destination="7367166"></a>
<div content-id="DSQ026TB2" class="table-modal"><div class="table-wrap"><div id="DSQ026TB2" class="table-wrap-title" data-id="DSQ026TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Conservation of aerial mycelium and spore formation-related genes in <em>K. setae</em> NBRC 14216<sup>T</sup> and other mycelial actinobacteria</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Kitasatospora setae</em> NBRC 14216<sup>Ta</sup> </th><th><em>Streptomyces coelicolor</em> A3(2) </th><th><em>Streptomyces avermitilis</em> MA-4680<sup>T</sup> </th><th><em>Streptomyces griseus</em> IFO 13350 </th><th><em>Streptomyces scabies</em> 87.22 </th><th><em>Catenulispora acidiphila</em> DSM 44928 </th><th><em>Saccharopolyspora erythraea</em> NRRL 23338 </th><th><em>Salinispora arenicola</em> CNS 205 </th></tr></thead><tbody><tr><td colspan="9">Aerial mycelium </td></tr><tr><td> <em>bldA</em> </td><td>KSE_t0069 </td><td>SCOt24 </td><td>SAV_t57 </td><td>SGR_tRNA42 </td><td>SCAB_t50 </td><td>Caci_R0007 </td><td>SACE_8016 </td><td>Sare_R0014 </td></tr><tr><td> <em>bldB</em> </td><td>(KSE_16220)<sup>b</sup> (64) </td><td>SCO5723 (SCO7246) </td><td>SAV_2529 (SAV_1241) </td><td>SGR_1796 </td><td>SCAB_25271 (SCAB_84941) </td><td>ND </td><td>SACE_6156 </td><td>Sare_3856 </td></tr><tr><td> <em>bldC</em> </td><td>KSE_41620 (97) </td><td>SCO4091 </td><td>SAV_4130 </td><td>SGR_3882 </td><td>SCAB_47901 </td><td>Caci_0301 </td><td>SACE_6926 </td><td>Sare_0170 </td></tr><tr><td> <em>bldD</em> </td><td>KSE_13950 (92) </td><td>SCO1489 </td><td>SAV_6861 </td><td>SGR_6045 </td><td>SCAB_75171 </td><td>Caci_2399 </td><td>SACE_2077 </td><td>Sare_1844 </td></tr><tr><td> <em>bldG</em> (=<em>rsbV</em>) </td><td>KSE_35840 (96) </td><td>SCO3549 </td><td>SAV_4614 </td><td>SGR_3307 </td><td>SCAB_40861 </td><td>Caci_8450 </td><td>SACE_4194 </td><td>Sare_4413 </td></tr><tr><td> <em>bldH</em> (=<em>adpA</em>) </td><td>KSE_26930 (80) </td><td>SCO2792 </td><td>SAV_5261 </td><td>SGR_4742 </td><td>SCAB_57831 </td><td>Caci_5972 </td><td>SACE_4523 </td><td>ND </td></tr><tr><td> <em>bldKA</em>-<em>bldKE</em> </td><td>KSE_48250–KSE_48290 (41–78) </td><td>SCO5112–SCO5116 </td><td>SAV_3152–SAV_3156; SAV_3172–SAV_3176 </td><td>SGR_2414–SGR_2418 </td><td>SCAB_31501–SCAB_31541 </td><td>Caci_7147–Caci_7151 </td><td>ND </td><td>ND </td></tr><tr><td> <em>bldM</em> (=<em>whiK</em>) </td><td>KSE_31060 (97) </td><td>SCO4768 </td><td>SAV_4998 </td><td>SGR_2759 </td><td>SCAB_36231 </td><td>Caci_1011 </td><td>SACE_6712 </td><td>Sare_4229 </td></tr><tr><td> <em>bldN</em> (=<em>adsA</em>) </td><td>KSE_33690 (92) </td><td>SCO3323 </td><td>SAV_4735 </td><td>SGR_4151 </td><td>SCAB_39121 </td><td>Caci_8330 </td><td>SACE_6951 </td><td>Sare_0420 </td></tr><tr><td> <em>amfC</em> </td><td>KSE_43740 (54) </td><td>SCO4184 </td><td>SAV_4026 </td><td>SGR_3974 </td><td>SCAB_49711 </td><td>Caci_8239 </td><td>SACE_7115 </td><td>ND </td></tr><tr><td colspan="9">Lantibiotic-like surfactant </td></tr><tr><td> <em>amfR</em> (=<em>ramR</em>) </td><td>ND </td><td>SCO6685 </td><td>SAV_7499 </td><td>SGR_2393 </td><td>SCAB_8642<sup>c</sup> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>amfS</em> (=<em>ramS</em>) </td><td>ND </td><td>SCO6682 </td><td>SAV_7502 </td><td>SGR_2396 </td><td>SCAB_8621 </td><td>Caci_4240 </td><td>SACE_4231 </td><td>ND </td></tr><tr><td> <em>amfB</em> (=<em>ramA</em>) </td><td>ND </td><td>SCO6683 </td><td>SAV_7501 </td><td>SGR_2395 </td><td>SCAB_8631 </td><td>Caci_4242 </td><td>SACE_4232 </td><td>ND </td></tr><tr><td> <em>amfA</em> (=<em>ramB</em>) </td><td>ND </td><td>SCO6684 </td><td>SAV_7500 </td><td>SGR_2394 </td><td>SCAB_8641 </td><td>Caci_4243 </td><td>SACE_4233 </td><td>ND </td></tr><tr><td> <em>amfT</em> (=<em>ramC</em>) </td><td>KSE_63000 (44) </td><td>SCO6681 </td><td>SAV_7503 </td><td>SGR_2397 </td><td>SCAB_8611 </td><td>Caci_4239 </td><td>SACE_4230 </td><td>ND </td></tr><tr><td colspan="9">Sporulation </td></tr><tr><td> <em>whiA</em> </td><td>KSE_55390 (89) </td><td>SCO1950 </td><td>SAV_6294 </td><td>SGR_5572 </td><td>SCAB_69681 </td><td>Caci_5609 </td><td>SACE_2141 </td><td>Sare_3326 </td></tr><tr><td> <em>whiB</em> </td><td>KSE_29410 (83) </td><td>SCO3034 </td><td>SAV_5042 </td><td>SGR_4503 </td><td>SCAB_55081 </td><td>Caci_7609 </td><td>SACE_6464 </td><td>ND </td></tr><tr><td> <em>whiD</em> (=<em>wblB</em>) </td><td>KSE_31070 (90) </td><td>SCO4767 </td><td>SAV_4997 </td><td>SGR_2760 </td><td>SCAB_36241 </td><td>Caci_1009 </td><td>SACE_5583 </td><td>ND </td></tr><tr><td> <em>whiE</em> </td><td>KSE_72410–KSE_72480 (46–67) </td><td>SCO5314–SCO5321 </td><td>SAV_2837–SAV_2844 </td><td>ND </td><td>SCAB_43281–SCAB43341 </td><td>Caci1083–Caci1090 </td><td>ND </td><td>Sare_2682–Sare_2690 </td></tr><tr><td> <em>whiG</em> </td><td>KSE_52540 (79) </td><td>SCO5621 </td><td>SAV_2630 </td><td>SGR_1866 </td><td>SCAB_26051 </td><td>Caci_1526 </td><td>SACE_6040 </td><td>ND </td></tr><tr><td> <em>whiH</em> </td><td>KSE_54050 (74) </td><td>SCO5819 </td><td>SAV_2445 </td><td>SGR_1702 </td><td>SCAB_24461 </td><td>Caci_1760 </td><td>ND </td><td>ND </td></tr><tr><td> <em>whiI</em> </td><td>KSE_55090 (77) </td><td>SCO6029 </td><td>SAV_2230 </td><td>SGR_1475 </td><td>SCAB_20881 </td><td>Caci_6781 </td><td>SACE_1833 </td><td>ND </td></tr><tr><td> <em>sigF</em> </td><td>KSE_37310 (69) </td><td>SCO4035 </td><td>SAV_4185 </td><td>SGR_3551 </td><td>SCAB_47291 </td><td>Caci_7708 </td><td>ND </td><td>ND </td></tr><tr><td> <em>sigN</em> </td><td>KSE_37320 (80) </td><td>SCO4034 </td><td>SAV_4186 </td><td>SGR_3552 </td><td>SCAB_47281 </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>crgA</em> (=<em>whiP</em>) </td><td>KSE_39340 (50) </td><td>SCO3854 </td><td>SAV_4331 </td><td>SGR_3718 </td><td>SCAB_45631 </td><td>Caci_0043 </td><td>SACE_0037 </td><td>Sare_0058 </td></tr></tbody></table></div><div content-id="DSQ026TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Kitasatospora setae</em> NBRC 14216<sup>Ta</sup> </th><th><em>Streptomyces coelicolor</em> A3(2) </th><th><em>Streptomyces avermitilis</em> MA-4680<sup>T</sup> </th><th><em>Streptomyces griseus</em> IFO 13350 </th><th><em>Streptomyces scabies</em> 87.22 </th><th><em>Catenulispora acidiphila</em> DSM 44928 </th><th><em>Saccharopolyspora erythraea</em> NRRL 23338 </th><th><em>Salinispora arenicola</em> CNS 205 </th></tr></thead><tbody><tr><td colspan="9">Aerial mycelium </td></tr><tr><td> <em>bldA</em> </td><td>KSE_t0069 </td><td>SCOt24 </td><td>SAV_t57 </td><td>SGR_tRNA42 </td><td>SCAB_t50 </td><td>Caci_R0007 </td><td>SACE_8016 </td><td>Sare_R0014 </td></tr><tr><td> <em>bldB</em> </td><td>(KSE_16220)<sup>b</sup> (64) </td><td>SCO5723 (SCO7246) </td><td>SAV_2529 (SAV_1241) </td><td>SGR_1796 </td><td>SCAB_25271 (SCAB_84941) </td><td>ND </td><td>SACE_6156 </td><td>Sare_3856 </td></tr><tr><td> <em>bldC</em> </td><td>KSE_41620 (97) </td><td>SCO4091 </td><td>SAV_4130 </td><td>SGR_3882 </td><td>SCAB_47901 </td><td>Caci_0301 </td><td>SACE_6926 </td><td>Sare_0170 </td></tr><tr><td> <em>bldD</em> </td><td>KSE_13950 (92) </td><td>SCO1489 </td><td>SAV_6861 </td><td>SGR_6045 </td><td>SCAB_75171 </td><td>Caci_2399 </td><td>SACE_2077 </td><td>Sare_1844 </td></tr><tr><td> <em>bldG</em> (=<em>rsbV</em>) </td><td>KSE_35840 (96) </td><td>SCO3549 </td><td>SAV_4614 </td><td>SGR_3307 </td><td>SCAB_40861 </td><td>Caci_8450 </td><td>SACE_4194 </td><td>Sare_4413 </td></tr><tr><td> <em>bldH</em> (=<em>adpA</em>) </td><td>KSE_26930 (80) </td><td>SCO2792 </td><td>SAV_5261 </td><td>SGR_4742 </td><td>SCAB_57831 </td><td>Caci_5972 </td><td>SACE_4523 </td><td>ND </td></tr><tr><td> <em>bldKA</em>-<em>bldKE</em> </td><td>KSE_48250–KSE_48290 (41–78) </td><td>SCO5112–SCO5116 </td><td>SAV_3152–SAV_3156; SAV_3172–SAV_3176 </td><td>SGR_2414–SGR_2418 </td><td>SCAB_31501–SCAB_31541 </td><td>Caci_7147–Caci_7151 </td><td>ND </td><td>ND </td></tr><tr><td> <em>bldM</em> (=<em>whiK</em>) </td><td>KSE_31060 (97) </td><td>SCO4768 </td><td>SAV_4998 </td><td>SGR_2759 </td><td>SCAB_36231 </td><td>Caci_1011 </td><td>SACE_6712 </td><td>Sare_4229 </td></tr><tr><td> <em>bldN</em> (=<em>adsA</em>) </td><td>KSE_33690 (92) </td><td>SCO3323 </td><td>SAV_4735 </td><td>SGR_4151 </td><td>SCAB_39121 </td><td>Caci_8330 </td><td>SACE_6951 </td><td>Sare_0420 </td></tr><tr><td> <em>amfC</em> </td><td>KSE_43740 (54) </td><td>SCO4184 </td><td>SAV_4026 </td><td>SGR_3974 </td><td>SCAB_49711 </td><td>Caci_8239 </td><td>SACE_7115 </td><td>ND </td></tr><tr><td colspan="9">Lantibiotic-like surfactant </td></tr><tr><td> <em>amfR</em> (=<em>ramR</em>) </td><td>ND </td><td>SCO6685 </td><td>SAV_7499 </td><td>SGR_2393 </td><td>SCAB_8642<sup>c</sup> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>amfS</em> (=<em>ramS</em>) </td><td>ND </td><td>SCO6682 </td><td>SAV_7502 </td><td>SGR_2396 </td><td>SCAB_8621 </td><td>Caci_4240 </td><td>SACE_4231 </td><td>ND </td></tr><tr><td> <em>amfB</em> (=<em>ramA</em>) </td><td>ND </td><td>SCO6683 </td><td>SAV_7501 </td><td>SGR_2395 </td><td>SCAB_8631 </td><td>Caci_4242 </td><td>SACE_4232 </td><td>ND </td></tr><tr><td> <em>amfA</em> (=<em>ramB</em>) </td><td>ND </td><td>SCO6684 </td><td>SAV_7500 </td><td>SGR_2394 </td><td>SCAB_8641 </td><td>Caci_4243 </td><td>SACE_4233 </td><td>ND </td></tr><tr><td> <em>amfT</em> (=<em>ramC</em>) </td><td>KSE_63000 (44) </td><td>SCO6681 </td><td>SAV_7503 </td><td>SGR_2397 </td><td>SCAB_8611 </td><td>Caci_4239 </td><td>SACE_4230 </td><td>ND </td></tr><tr><td colspan="9">Sporulation </td></tr><tr><td> <em>whiA</em> </td><td>KSE_55390 (89) </td><td>SCO1950 </td><td>SAV_6294 </td><td>SGR_5572 </td><td>SCAB_69681 </td><td>Caci_5609 </td><td>SACE_2141 </td><td>Sare_3326 </td></tr><tr><td> <em>whiB</em> </td><td>KSE_29410 (83) </td><td>SCO3034 </td><td>SAV_5042 </td><td>SGR_4503 </td><td>SCAB_55081 </td><td>Caci_7609 </td><td>SACE_6464 </td><td>ND </td></tr><tr><td> <em>whiD</em> (=<em>wblB</em>) </td><td>KSE_31070 (90) </td><td>SCO4767 </td><td>SAV_4997 </td><td>SGR_2760 </td><td>SCAB_36241 </td><td>Caci_1009 </td><td>SACE_5583 </td><td>ND </td></tr><tr><td> <em>whiE</em> </td><td>KSE_72410–KSE_72480 (46–67) </td><td>SCO5314–SCO5321 </td><td>SAV_2837–SAV_2844 </td><td>ND </td><td>SCAB_43281–SCAB43341 </td><td>Caci1083–Caci1090 </td><td>ND </td><td>Sare_2682–Sare_2690 </td></tr><tr><td> <em>whiG</em> </td><td>KSE_52540 (79) </td><td>SCO5621 </td><td>SAV_2630 </td><td>SGR_1866 </td><td>SCAB_26051 </td><td>Caci_1526 </td><td>SACE_6040 </td><td>ND </td></tr><tr><td> <em>whiH</em> </td><td>KSE_54050 (74) </td><td>SCO5819 </td><td>SAV_2445 </td><td>SGR_1702 </td><td>SCAB_24461 </td><td>Caci_1760 </td><td>ND </td><td>ND </td></tr><tr><td> <em>whiI</em> </td><td>KSE_55090 (77) </td><td>SCO6029 </td><td>SAV_2230 </td><td>SGR_1475 </td><td>SCAB_20881 </td><td>Caci_6781 </td><td>SACE_1833 </td><td>ND </td></tr><tr><td> <em>sigF</em> </td><td>KSE_37310 (69) </td><td>SCO4035 </td><td>SAV_4185 </td><td>SGR_3551 </td><td>SCAB_47291 </td><td>Caci_7708 </td><td>ND </td><td>ND </td></tr><tr><td> <em>sigN</em> </td><td>KSE_37320 (80) </td><td>SCO4034 </td><td>SAV_4186 </td><td>SGR_3552 </td><td>SCAB_47281 </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>crgA</em> (=<em>whiP</em>) </td><td>KSE_39340 (50) </td><td>SCO3854 </td><td>SAV_4331 </td><td>SGR_3718 </td><td>SCAB_45631 </td><td>Caci_0043 </td><td>SACE_0037 </td><td>Sare_0058 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The amino acid identity between <em>K. setae</em> NBRC 14216<sup>T</sup> ORF and <em>S. coelicolor</em> A3(2) ortholog was shown in parentheses. The range of amino acid identity was shown where multiple genes were assigned.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>bldB</em> gene paralog found in <em>S. coelicolor</em> A3(2) and <em>S. avermitilis</em> MA-4680<sup>T</sup> are also included. KSE_16220 corresponds to an ortholog of the <em>bldB</em> paralog.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>This ORF tentatively named SCAB_8642 (corresponding to position 975 194–975 802 on the complementary strand) has not yet been incorporated in the submitter annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367166" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ026TB2" class="table-wrap-title" data-id="DSQ026TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Conservation of aerial mycelium and spore formation-related genes in <em>K. setae</em> NBRC 14216<sup>T</sup> and other mycelial actinobacteria</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Kitasatospora setae</em> NBRC 14216<sup>Ta</sup> </th><th><em>Streptomyces coelicolor</em> A3(2) </th><th><em>Streptomyces avermitilis</em> MA-4680<sup>T</sup> </th><th><em>Streptomyces griseus</em> IFO 13350 </th><th><em>Streptomyces scabies</em> 87.22 </th><th><em>Catenulispora acidiphila</em> DSM 44928 </th><th><em>Saccharopolyspora erythraea</em> NRRL 23338 </th><th><em>Salinispora arenicola</em> CNS 205 </th></tr></thead><tbody><tr><td colspan="9">Aerial mycelium </td></tr><tr><td> <em>bldA</em> </td><td>KSE_t0069 </td><td>SCOt24 </td><td>SAV_t57 </td><td>SGR_tRNA42 </td><td>SCAB_t50 </td><td>Caci_R0007 </td><td>SACE_8016 </td><td>Sare_R0014 </td></tr><tr><td> <em>bldB</em> </td><td>(KSE_16220)<sup>b</sup> (64) </td><td>SCO5723 (SCO7246) </td><td>SAV_2529 (SAV_1241) </td><td>SGR_1796 </td><td>SCAB_25271 (SCAB_84941) </td><td>ND </td><td>SACE_6156 </td><td>Sare_3856 </td></tr><tr><td> <em>bldC</em> </td><td>KSE_41620 (97) </td><td>SCO4091 </td><td>SAV_4130 </td><td>SGR_3882 </td><td>SCAB_47901 </td><td>Caci_0301 </td><td>SACE_6926 </td><td>Sare_0170 </td></tr><tr><td> <em>bldD</em> </td><td>KSE_13950 (92) </td><td>SCO1489 </td><td>SAV_6861 </td><td>SGR_6045 </td><td>SCAB_75171 </td><td>Caci_2399 </td><td>SACE_2077 </td><td>Sare_1844 </td></tr><tr><td> <em>bldG</em> (=<em>rsbV</em>) </td><td>KSE_35840 (96) </td><td>SCO3549 </td><td>SAV_4614 </td><td>SGR_3307 </td><td>SCAB_40861 </td><td>Caci_8450 </td><td>SACE_4194 </td><td>Sare_4413 </td></tr><tr><td> <em>bldH</em> (=<em>adpA</em>) </td><td>KSE_26930 (80) </td><td>SCO2792 </td><td>SAV_5261 </td><td>SGR_4742 </td><td>SCAB_57831 </td><td>Caci_5972 </td><td>SACE_4523 </td><td>ND </td></tr><tr><td> <em>bldKA</em>-<em>bldKE</em> </td><td>KSE_48250–KSE_48290 (41–78) </td><td>SCO5112–SCO5116 </td><td>SAV_3152–SAV_3156; SAV_3172–SAV_3176 </td><td>SGR_2414–SGR_2418 </td><td>SCAB_31501–SCAB_31541 </td><td>Caci_7147–Caci_7151 </td><td>ND </td><td>ND </td></tr><tr><td> <em>bldM</em> (=<em>whiK</em>) </td><td>KSE_31060 (97) </td><td>SCO4768 </td><td>SAV_4998 </td><td>SGR_2759 </td><td>SCAB_36231 </td><td>Caci_1011 </td><td>SACE_6712 </td><td>Sare_4229 </td></tr><tr><td> <em>bldN</em> (=<em>adsA</em>) </td><td>KSE_33690 (92) </td><td>SCO3323 </td><td>SAV_4735 </td><td>SGR_4151 </td><td>SCAB_39121 </td><td>Caci_8330 </td><td>SACE_6951 </td><td>Sare_0420 </td></tr><tr><td> <em>amfC</em> </td><td>KSE_43740 (54) </td><td>SCO4184 </td><td>SAV_4026 </td><td>SGR_3974 </td><td>SCAB_49711 </td><td>Caci_8239 </td><td>SACE_7115 </td><td>ND </td></tr><tr><td colspan="9">Lantibiotic-like surfactant </td></tr><tr><td> <em>amfR</em> (=<em>ramR</em>) </td><td>ND </td><td>SCO6685 </td><td>SAV_7499 </td><td>SGR_2393 </td><td>SCAB_8642<sup>c</sup> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>amfS</em> (=<em>ramS</em>) </td><td>ND </td><td>SCO6682 </td><td>SAV_7502 </td><td>SGR_2396 </td><td>SCAB_8621 </td><td>Caci_4240 </td><td>SACE_4231 </td><td>ND </td></tr><tr><td> <em>amfB</em> (=<em>ramA</em>) </td><td>ND </td><td>SCO6683 </td><td>SAV_7501 </td><td>SGR_2395 </td><td>SCAB_8631 </td><td>Caci_4242 </td><td>SACE_4232 </td><td>ND </td></tr><tr><td> <em>amfA</em> (=<em>ramB</em>) </td><td>ND </td><td>SCO6684 </td><td>SAV_7500 </td><td>SGR_2394 </td><td>SCAB_8641 </td><td>Caci_4243 </td><td>SACE_4233 </td><td>ND </td></tr><tr><td> <em>amfT</em> (=<em>ramC</em>) </td><td>KSE_63000 (44) </td><td>SCO6681 </td><td>SAV_7503 </td><td>SGR_2397 </td><td>SCAB_8611 </td><td>Caci_4239 </td><td>SACE_4230 </td><td>ND </td></tr><tr><td colspan="9">Sporulation </td></tr><tr><td> <em>whiA</em> </td><td>KSE_55390 (89) </td><td>SCO1950 </td><td>SAV_6294 </td><td>SGR_5572 </td><td>SCAB_69681 </td><td>Caci_5609 </td><td>SACE_2141 </td><td>Sare_3326 </td></tr><tr><td> <em>whiB</em> </td><td>KSE_29410 (83) </td><td>SCO3034 </td><td>SAV_5042 </td><td>SGR_4503 </td><td>SCAB_55081 </td><td>Caci_7609 </td><td>SACE_6464 </td><td>ND </td></tr><tr><td> <em>whiD</em> (=<em>wblB</em>) </td><td>KSE_31070 (90) </td><td>SCO4767 </td><td>SAV_4997 </td><td>SGR_2760 </td><td>SCAB_36241 </td><td>Caci_1009 </td><td>SACE_5583 </td><td>ND </td></tr><tr><td> <em>whiE</em> </td><td>KSE_72410–KSE_72480 (46–67) </td><td>SCO5314–SCO5321 </td><td>SAV_2837–SAV_2844 </td><td>ND </td><td>SCAB_43281–SCAB43341 </td><td>Caci1083–Caci1090 </td><td>ND </td><td>Sare_2682–Sare_2690 </td></tr><tr><td> <em>whiG</em> </td><td>KSE_52540 (79) </td><td>SCO5621 </td><td>SAV_2630 </td><td>SGR_1866 </td><td>SCAB_26051 </td><td>Caci_1526 </td><td>SACE_6040 </td><td>ND </td></tr><tr><td> <em>whiH</em> </td><td>KSE_54050 (74) </td><td>SCO5819 </td><td>SAV_2445 </td><td>SGR_1702 </td><td>SCAB_24461 </td><td>Caci_1760 </td><td>ND </td><td>ND </td></tr><tr><td> <em>whiI</em> </td><td>KSE_55090 (77) </td><td>SCO6029 </td><td>SAV_2230 </td><td>SGR_1475 </td><td>SCAB_20881 </td><td>Caci_6781 </td><td>SACE_1833 </td><td>ND </td></tr><tr><td> <em>sigF</em> </td><td>KSE_37310 (69) </td><td>SCO4035 </td><td>SAV_4185 </td><td>SGR_3551 </td><td>SCAB_47291 </td><td>Caci_7708 </td><td>ND </td><td>ND </td></tr><tr><td> <em>sigN</em> </td><td>KSE_37320 (80) </td><td>SCO4034 </td><td>SAV_4186 </td><td>SGR_3552 </td><td>SCAB_47281 </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>crgA</em> (=<em>whiP</em>) </td><td>KSE_39340 (50) </td><td>SCO3854 </td><td>SAV_4331 </td><td>SGR_3718 </td><td>SCAB_45631 </td><td>Caci_0043 </td><td>SACE_0037 </td><td>Sare_0058 </td></tr></tbody></table></div><div content-id="DSQ026TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Kitasatospora setae</em> NBRC 14216<sup>Ta</sup> </th><th><em>Streptomyces coelicolor</em> A3(2) </th><th><em>Streptomyces avermitilis</em> MA-4680<sup>T</sup> </th><th><em>Streptomyces griseus</em> IFO 13350 </th><th><em>Streptomyces scabies</em> 87.22 </th><th><em>Catenulispora acidiphila</em> DSM 44928 </th><th><em>Saccharopolyspora erythraea</em> NRRL 23338 </th><th><em>Salinispora arenicola</em> CNS 205 </th></tr></thead><tbody><tr><td colspan="9">Aerial mycelium </td></tr><tr><td> <em>bldA</em> </td><td>KSE_t0069 </td><td>SCOt24 </td><td>SAV_t57 </td><td>SGR_tRNA42 </td><td>SCAB_t50 </td><td>Caci_R0007 </td><td>SACE_8016 </td><td>Sare_R0014 </td></tr><tr><td> <em>bldB</em> </td><td>(KSE_16220)<sup>b</sup> (64) </td><td>SCO5723 (SCO7246) </td><td>SAV_2529 (SAV_1241) </td><td>SGR_1796 </td><td>SCAB_25271 (SCAB_84941) </td><td>ND </td><td>SACE_6156 </td><td>Sare_3856 </td></tr><tr><td> <em>bldC</em> </td><td>KSE_41620 (97) </td><td>SCO4091 </td><td>SAV_4130 </td><td>SGR_3882 </td><td>SCAB_47901 </td><td>Caci_0301 </td><td>SACE_6926 </td><td>Sare_0170 </td></tr><tr><td> <em>bldD</em> </td><td>KSE_13950 (92) </td><td>SCO1489 </td><td>SAV_6861 </td><td>SGR_6045 </td><td>SCAB_75171 </td><td>Caci_2399 </td><td>SACE_2077 </td><td>Sare_1844 </td></tr><tr><td> <em>bldG</em> (=<em>rsbV</em>) </td><td>KSE_35840 (96) </td><td>SCO3549 </td><td>SAV_4614 </td><td>SGR_3307 </td><td>SCAB_40861 </td><td>Caci_8450 </td><td>SACE_4194 </td><td>Sare_4413 </td></tr><tr><td> <em>bldH</em> (=<em>adpA</em>) </td><td>KSE_26930 (80) </td><td>SCO2792 </td><td>SAV_5261 </td><td>SGR_4742 </td><td>SCAB_57831 </td><td>Caci_5972 </td><td>SACE_4523 </td><td>ND </td></tr><tr><td> <em>bldKA</em>-<em>bldKE</em> </td><td>KSE_48250–KSE_48290 (41–78) </td><td>SCO5112–SCO5116 </td><td>SAV_3152–SAV_3156; SAV_3172–SAV_3176 </td><td>SGR_2414–SGR_2418 </td><td>SCAB_31501–SCAB_31541 </td><td>Caci_7147–Caci_7151 </td><td>ND </td><td>ND </td></tr><tr><td> <em>bldM</em> (=<em>whiK</em>) </td><td>KSE_31060 (97) </td><td>SCO4768 </td><td>SAV_4998 </td><td>SGR_2759 </td><td>SCAB_36231 </td><td>Caci_1011 </td><td>SACE_6712 </td><td>Sare_4229 </td></tr><tr><td> <em>bldN</em> (=<em>adsA</em>) </td><td>KSE_33690 (92) </td><td>SCO3323 </td><td>SAV_4735 </td><td>SGR_4151 </td><td>SCAB_39121 </td><td>Caci_8330 </td><td>SACE_6951 </td><td>Sare_0420 </td></tr><tr><td> <em>amfC</em> </td><td>KSE_43740 (54) </td><td>SCO4184 </td><td>SAV_4026 </td><td>SGR_3974 </td><td>SCAB_49711 </td><td>Caci_8239 </td><td>SACE_7115 </td><td>ND </td></tr><tr><td colspan="9">Lantibiotic-like surfactant </td></tr><tr><td> <em>amfR</em> (=<em>ramR</em>) </td><td>ND </td><td>SCO6685 </td><td>SAV_7499 </td><td>SGR_2393 </td><td>SCAB_8642<sup>c</sup> </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>amfS</em> (=<em>ramS</em>) </td><td>ND </td><td>SCO6682 </td><td>SAV_7502 </td><td>SGR_2396 </td><td>SCAB_8621 </td><td>Caci_4240 </td><td>SACE_4231 </td><td>ND </td></tr><tr><td> <em>amfB</em> (=<em>ramA</em>) </td><td>ND </td><td>SCO6683 </td><td>SAV_7501 </td><td>SGR_2395 </td><td>SCAB_8631 </td><td>Caci_4242 </td><td>SACE_4232 </td><td>ND </td></tr><tr><td> <em>amfA</em> (=<em>ramB</em>) </td><td>ND </td><td>SCO6684 </td><td>SAV_7500 </td><td>SGR_2394 </td><td>SCAB_8641 </td><td>Caci_4243 </td><td>SACE_4233 </td><td>ND </td></tr><tr><td> <em>amfT</em> (=<em>ramC</em>) </td><td>KSE_63000 (44) </td><td>SCO6681 </td><td>SAV_7503 </td><td>SGR_2397 </td><td>SCAB_8611 </td><td>Caci_4239 </td><td>SACE_4230 </td><td>ND </td></tr><tr><td colspan="9">Sporulation </td></tr><tr><td> <em>whiA</em> </td><td>KSE_55390 (89) </td><td>SCO1950 </td><td>SAV_6294 </td><td>SGR_5572 </td><td>SCAB_69681 </td><td>Caci_5609 </td><td>SACE_2141 </td><td>Sare_3326 </td></tr><tr><td> <em>whiB</em> </td><td>KSE_29410 (83) </td><td>SCO3034 </td><td>SAV_5042 </td><td>SGR_4503 </td><td>SCAB_55081 </td><td>Caci_7609 </td><td>SACE_6464 </td><td>ND </td></tr><tr><td> <em>whiD</em> (=<em>wblB</em>) </td><td>KSE_31070 (90) </td><td>SCO4767 </td><td>SAV_4997 </td><td>SGR_2760 </td><td>SCAB_36241 </td><td>Caci_1009 </td><td>SACE_5583 </td><td>ND </td></tr><tr><td> <em>whiE</em> </td><td>KSE_72410–KSE_72480 (46–67) </td><td>SCO5314–SCO5321 </td><td>SAV_2837–SAV_2844 </td><td>ND </td><td>SCAB_43281–SCAB43341 </td><td>Caci1083–Caci1090 </td><td>ND </td><td>Sare_2682–Sare_2690 </td></tr><tr><td> <em>whiG</em> </td><td>KSE_52540 (79) </td><td>SCO5621 </td><td>SAV_2630 </td><td>SGR_1866 </td><td>SCAB_26051 </td><td>Caci_1526 </td><td>SACE_6040 </td><td>ND </td></tr><tr><td> <em>whiH</em> </td><td>KSE_54050 (74) </td><td>SCO5819 </td><td>SAV_2445 </td><td>SGR_1702 </td><td>SCAB_24461 </td><td>Caci_1760 </td><td>ND </td><td>ND </td></tr><tr><td> <em>whiI</em> </td><td>KSE_55090 (77) </td><td>SCO6029 </td><td>SAV_2230 </td><td>SGR_1475 </td><td>SCAB_20881 </td><td>Caci_6781 </td><td>SACE_1833 </td><td>ND </td></tr><tr><td> <em>sigF</em> </td><td>KSE_37310 (69) </td><td>SCO4035 </td><td>SAV_4185 </td><td>SGR_3551 </td><td>SCAB_47291 </td><td>Caci_7708 </td><td>ND </td><td>ND </td></tr><tr><td> <em>sigN</em> </td><td>KSE_37320 (80) </td><td>SCO4034 </td><td>SAV_4186 </td><td>SGR_3552 </td><td>SCAB_47281 </td><td>ND </td><td>ND </td><td>ND </td></tr><tr><td> <em>crgA</em> (=<em>whiP</em>) </td><td>KSE_39340 (50) </td><td>SCO3854 </td><td>SAV_4331 </td><td>SGR_3718 </td><td>SCAB_45631 </td><td>Caci_0043 </td><td>SACE_0037 </td><td>Sare_0058 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The amino acid identity between <em>K. setae</em> NBRC 14216<sup>T</sup> ORF and <em>S. coelicolor</em> A3(2) ortholog was shown in parentheses. The range of amino acid identity was shown where multiple genes were assigned.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>bldB</em> gene paralog found in <em>S. coelicolor</em> A3(2) and <em>S. avermitilis</em> MA-4680<sup>T</sup> are also included. KSE_16220 corresponds to an ortholog of the <em>bldB</em> paralog.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>This ORF tentatively named SCAB_8642 (corresponding to position 975 194–975 802 on the complementary strand) has not yet been incorporated in the submitter annotation.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367166" target="_blank">View Large</a></div></div><p>On the contrary, we could not identify most of the <em>amf</em> genes in <em>K. setae</em> (Table <a reveal-id="DSQ026TB2" data-open="DSQ026TB2" class="link link-reveal link-table xref-fig">2</a>). The <em>amf</em> genes are highly conserved among <em>Streptomyces</em> species and are necessary for the synthesis of AmfS [SapB in <em>S. coelicolor</em> A3(2)] surfactant protein, which is known in <em>Streptomyces</em> to be secreted before the initiation of aerial growth. The presence of <em>amfAB</em> homologs in <em>C. acidiphila</em><sup><a reveal-id="DSQ026C41" data-open="DSQ026C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and <em>S. erythraea</em>,<sup><a reveal-id="DSQ026C44" data-open="DSQ026C44" class="link link-ref link-reveal xref-bibr">44</a></sup> mycelial actinobacteria more distantly related to <em>Streptomyces</em>, may suggest that the pre-existing AmfS system was eliminated in the <em>Kitasatospora</em> lineage, although other possibilities such as the horizontal acquisition of these genes in each lineage cannot be excluded. In addition to the lack of the <em>amf</em> gene cluster, differences from <em>Streptomyces</em> were also suggested in the variation of paralogous components (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>) such as SsgA-like proteins involved in peptidoglycan synthesis in sporogenic cell division, WhiB family transcriptional regulators and chaplins, another class of surfactant protein shown in <em>S. coelicolor</em> to play a part in aerial mycelium formation.<sup><a reveal-id="DSQ026C46" data-open="DSQ026C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSQ026C47" data-open="DSQ026C47" class="link link-ref link-reveal xref-bibr">47</a></sup></p><p>A number of <em>whi</em> genes control the process of sporulation septation and spore maturation in <em>Streptomyces</em>. Of the classical <em>whi</em> genes, <em>whiA</em>, <em>whiB</em> and <em>whiD</em> are commonly found in actinobacteria, and their exact functions in simpler (non-sporulating) actinobacteria need to be elucidated.<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a></sup> All other <em>whi</em> genes, which had been considered specific to <em>Streptomyces</em> species,<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ026C42" data-open="DSQ026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> were also conserved in <em>K. setae</em> with remarkably high amino acid identities (69–90%). Despite such close similarities in genetic background, preliminary experimental results suggest that <em>K. setae</em> produces seemingly less mature spores compared with <em>Streptomyces</em>; (i) although <em>K. setae</em> genome encodes full set of <em>whiE</em> gene cluster (encoding biosynthetic enzymes for polyketide spore pigment), no expression of <em>whiE</em> genes nor the production of spore pigment was observed in any culture conditions for sporulation, (ii) both aerial and submerged spores produced by <em>K. setae</em> are much more sensitive to freeze-thaw cycles than those of <em>Streptomyces</em> (S. Kitani and H. Ikeda, unpublished observations). Further studies are needed to elucidate mechanisms underlying such differences.</p><p>The initiation of secondary metabolite synthesis is known to be linked with morphological differentiation in <em>S. griseus</em> via the gamma-butyrolactone autoregulator cascade. In this regard, it would be noteworthy that <em>K. setae</em> possesses three homologs of the autoregulator receptor: KsbA (KSE_58650), KsbB (KSE_01050t and KSE_75690t; identical genes encoded in the TIR) and KsbC (KSE_44580). However, KsbA in <em>K. setae</em> was experimentally confirmed to be involved only in secondary metabolism,<sup><a reveal-id="DSQ026C48" data-open="DSQ026C48" class="link link-ref link-reveal xref-bibr">48</a></sup> but not in morphological differentiation, whereas the involvement of other two remains to be clarified. In addition, the AfsA family protein, which contains two A-factor biosynthesis repeat motifs, has been reported to be a crucial enzyme to synthesize the gamma-butyrolactone autoregulator in <em>Streptomyces</em>.<sup><a reveal-id="DSQ026C49" data-open="DSQ026C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Interestingly, three AfsA family proteins were encoded on the genome of <em>K. setae</em>: KsbS2 (KSE_01060t and KSE_75680t; identical and present in the TIR, similar to the case of KsbB), KsbS3 (KSE_22970) and KsbS4 (KSE_44600). These findings suggested that there might be a more complicated signaling network for secondary metabolism and/or morphological differentiation in <em>K. setae</em>.</p>                            <h3 scrollto-destination=7367174 id="7367174" class="section-title" data-legacy-id=s3d>3.4. Genes involved in peptidoglycan biosynthesis</h3>
<p>One of the most important features of the genus <em>Kitasatospora</em> from the chemotaxonomic viewpoint is that the cell wall peptidoglycan contains both <span class="small-caps">ll</span>-DAP and <em>meso</em>-DAP.<sup><a reveal-id="DSQ026C14" data-open="DSQ026C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSQ026C15" data-open="DSQ026C15" class="link link-ref link-reveal xref-bibr">15</a></sup> DAP analysis of <em>Kitasatospora</em> strains showed that spores contain only <span class="small-caps">ll</span>-DAP, whereas mycelia contain mostly <em>meso</em>-DAP.<sup><a reveal-id="DSQ026C16" data-open="DSQ026C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSQ026C17" data-open="DSQ026C17" class="link link-ref link-reveal xref-bibr">17</a></sup> This observation suggests that <em>Kitasatospora</em> incorporates different DAP isomers into the cell wall depending on differentiation stage. We can speculate two steps which might be responsible for the differential incorporation of DAP isomers. (i) DAP biosynthesis and isomerization: <span class="small-caps">ll</span>-DAP is isomerized to <em>meso</em>-DAP by <em>dapF</em> gene product in the course of lysine biosynthesis.<sup><a reveal-id="DSQ026C50" data-open="DSQ026C50" class="link link-ref link-reveal xref-bibr">50</a></sup> (ii) Incorporation of DAP into peptidoglycan: the addition of DAP to UDP-<em>N</em>-acetylmuramyl-<span class="small-caps">l</span>-alanyl-<span class="small-caps">d</span>-glutamate precursor is catalyzed by an enzyme encoded by <em>murE</em>.<sup><a reveal-id="DSQ026C51" data-open="DSQ026C51" class="link link-ref link-reveal xref-bibr">51</a></sup></p><p>As in the case of <em>Streptomyces</em> species whose peptidoglycan is composed of <span class="small-caps">ll</span>-DAP, <em>K. setae</em> had only one <em>murE</em> gene (KSE_21280), which is located in the center of a long conserved <em>dcw</em> (division cell wall) cluster. Amino acid residues known to be responsible for substrate recognition in the enzymes from <em>meso</em>-DAP containing bacteria are also conserved in the enzymes from <span class="small-caps">ll</span>-DAP containing bacteria including <em>Streptomyces</em>. The MurE protein purified from <em>E. coli</em> can incorporate <span class="small-caps">ll</span>-DAP in addition to <em>meso</em>-DAP, the natural substrate in this bacterium.<sup><a reveal-id="DSQ026C52" data-open="DSQ026C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Thus, the MurE protein may not be responsible for the differentiation-dependent alterations in peptidoglycan synthesis, unless the substrate specificity is regulated by unknown factor(s). On the other hand, <em>K. setae</em> had three <em>dapF</em> genes (KSE_32600, KSE_32630 and KSE_53750) whose products share highly conserved active site residues. Phylogenetic analysis based on amino acid sequences showed that two of the DapF proteins (encoded by KSE_32600 and KSE_53750) were closely related to those from <span class="small-caps">ll</span>-DAP containing actinobacteria such as <em>Streptomyces</em> species and <em>Nocardioides</em> sp. BAA-499/JS614 (Fig. <a reveal-id="DSQ026F4" data-open="DSQ026F4" class="link link-reveal link-table xref-fig">4</a>A). In contrast, the third DapF protein (KSE_32630) was positioned apart from the actinobacterial groups, but in the vicinity of proteins from <em>meso</em>-DAP containing <em>Listeria</em>, <em>Lactobacillus</em> and <em>Clostridium</em>. In the <em>dapF</em> mutant strain of <em>E. coli</em>, an increase in <span class="small-caps">ll</span>-DAP containing peptidoglycan was observed,<sup><a reveal-id="DSQ026C52" data-open="DSQ026C52" class="link link-ref link-reveal xref-bibr">52</a></sup> suggesting that the relative size of <span class="small-caps">ll</span>- and <em>meso</em>-DAP pools would affect the composition of peptidoglycan. Differential regulation of the multiple <em>dapF</em> genes, or their protein products, in the course of morphological differentiation might play an important role in the change in peptidoglycan composition. </p>                        <a id="7367178" scrollto-destination="7367178"></a>
<div id="DSQ026F4" data-id="DSQ026F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02604.jpeg?Expires=2147483647&Signature=D26xJ5taa2Hqgen~GWWrPGMDZsvCScI4COfhGfcJYCegn9Sy-zcnElalJ12w8ILRaTEbMCBP3XXlDjGg82bZr3dZJCrFSwHrh56O3nzFAigpuXYVIBpNrIWDlwK0257qzEPmYsjnDjjS9VV4SoFTql3IECqmUpZTj49Ucx268NScO0Nktr67ZJQB3S6JvFpkRl-o7bo~nlK2a9oEM3b1VQkVHD6KOn4nEtkXXPMdf~02areyb1toku~YdrLdNvN10Oz-Nr2ewc8yY1GqkltiVFcMLegGVmiN9~fk~AGXc-lKGrXhk6gagEG2EdC8AaBbjQtY42MyRFmO-Y63oUM6cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid phylogenetic trees of DapF (A) and HMW-PBPs (B). Branches with less than 70% bootstrap support are represented in dotted line. The distribution of peptidoglycan amino acid component is also presented in (A). Streptomyces coelicolor A3(2) PBPs and their orthologs were grouped and expression patterns were assigned according to Noens et al.55 in (B); those known to be expressed in aerial mycelium and vegetative mycelium are shown by AM and VM, respectively. Amino acid sequences were derived from the HAMAP database (MF_00197)." path-from-xml="dsq02604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367178/dsq02604.jpeg" path-from-xml="dsq02604.jpeg" target="_blank">View large</a><a section="7367178" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02604.jpeg?Expires=2147483647&Signature=GEb22PcZcoMc7kqXJcVa6id1aSBeeZZHiT4rIPyEn6JqIuoC37809qcnwq~snN-4K7eCW~aw78iZmqq1~eQMvv4GbitK70Uc5luzy8jzOkxIrv~EFUrAgX91PLtoqnWvJzZlX-BjvjJq1hwN0AVcFEyzrWwFEX6sfJf1CNMad0Wlu7QfC1qhbjBcKwWs-3yE2NL6Of9EUvQwROiPjCjSfCPdPV-qKwMbhCc6MeruKPqNs73zFqevozUOQbVbuBTazWoUMZDj13VQS6cgstMnXeXTJEUpLRYJbfG3pBN1fdSd3SvmHQbPWuVV3gqhfgQ~5Mb51yLSXBbksDWDoNIiEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367178&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid phylogenetic trees of DapF (A) and HMW-PBPs (B). Branches with less than 70% bootstrap support are represented in dotted line. The distribution of peptidoglycan amino acid component is also presented in (A). <em>Streptomyces coelicolor</em> A3(2) PBPs and their orthologs were grouped and expression patterns were assigned according to Noens <em>et al</em>.<sup><a reveal-id="DSQ026C55" data-open="DSQ026C55" class="link link-ref link-reveal xref-bibr">55</a></sup> in (B); those known to be expressed in aerial mycelium and vegetative mycelium are shown by AM and VM, respectively. Amino acid sequences were derived from the HAMAP database (MF_00197).</p></div></div><div content-id="DSQ026F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/m_dsq02604.jpeg?Expires=2147483647&Signature=D26xJ5taa2Hqgen~GWWrPGMDZsvCScI4COfhGfcJYCegn9Sy-zcnElalJ12w8ILRaTEbMCBP3XXlDjGg82bZr3dZJCrFSwHrh56O3nzFAigpuXYVIBpNrIWDlwK0257qzEPmYsjnDjjS9VV4SoFTql3IECqmUpZTj49Ucx268NScO0Nktr67ZJQB3S6JvFpkRl-o7bo~nlK2a9oEM3b1VQkVHD6KOn4nEtkXXPMdf~02areyb1toku~YdrLdNvN10Oz-Nr2ewc8yY1GqkltiVFcMLegGVmiN9~fk~AGXc-lKGrXhk6gagEG2EdC8AaBbjQtY42MyRFmO-Y63oUM6cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amino acid phylogenetic trees of DapF (A) and HMW-PBPs (B). Branches with less than 70% bootstrap support are represented in dotted line. The distribution of peptidoglycan amino acid component is also presented in (A). Streptomyces coelicolor A3(2) PBPs and their orthologs were grouped and expression patterns were assigned according to Noens et al.55 in (B); those known to be expressed in aerial mycelium and vegetative mycelium are shown by AM and VM, respectively. Amino acid sequences were derived from the HAMAP database (MF_00197)." path-from-xml="dsq02604.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7367178/dsq02604.jpeg" path-from-xml="dsq02604.jpeg" target="_blank">View large</a><a section="7367178" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq02604.jpeg?Expires=2147483647&Signature=GEb22PcZcoMc7kqXJcVa6id1aSBeeZZHiT4rIPyEn6JqIuoC37809qcnwq~snN-4K7eCW~aw78iZmqq1~eQMvv4GbitK70Uc5luzy8jzOkxIrv~EFUrAgX91PLtoqnWvJzZlX-BjvjJq1hwN0AVcFEyzrWwFEX6sfJf1CNMad0Wlu7QfC1qhbjBcKwWs-3yE2NL6Of9EUvQwROiPjCjSfCPdPV-qKwMbhCc6MeruKPqNs73zFqevozUOQbVbuBTazWoUMZDj13VQS6cgstMnXeXTJEUpLRYJbfG3pBN1fdSd3SvmHQbPWuVV3gqhfgQ~5Mb51yLSXBbksDWDoNIiEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7367178&ar=484773&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02604.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amino acid phylogenetic trees of DapF (A) and HMW-PBPs (B). Branches with less than 70% bootstrap support are represented in dotted line. The distribution of peptidoglycan amino acid component is also presented in (A). <em>Streptomyces coelicolor</em> A3(2) PBPs and their orthologs were grouped and expression patterns were assigned according to Noens <em>et al</em>.<sup><a reveal-id="DSQ026C55" data-open="DSQ026C55" class="link link-ref link-reveal xref-bibr">55</a></sup> in (B); those known to be expressed in aerial mycelium and vegetative mycelium are shown by AM and VM, respectively. Amino acid sequences were derived from the HAMAP database (MF_00197).</p></div></div><p><em>Streptomyces</em>, a representative of mycelial bacteria, is known to have two distinct modes of cell division. During vegetative growth, cell division occurs occasionally resulting in widely and irregularly spaced cross-walls. In sporulating mycelia, in contrast, sporulating septa are deposited in a coordinated manner resulting in regularly spaced spores.<sup><a reveal-id="DSQ026C53" data-open="DSQ026C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The latter suggests a regulated synthesis of peptidoglycan at septa and spore walls. High-molecular weight PBPs (HMW-PBPs) catalyze the cross-linking of peptidoglycan peptides. <em>Streptomyces</em> contain a number of HMW-PBP genes of which some were shown to be expressed at specific stages of differentiation.<sup><a reveal-id="DSQ026C54" data-open="DSQ026C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSQ026C55" data-open="DSQ026C55" class="link link-ref link-reveal xref-bibr">55</a></sup><em>Kitasatospora setae</em> seems to have nine HMW-PBP genes, one of which (KSE_46190) being highly homologous to <em>S. coelicolor</em> A3(2) genes (SCO3771 and SCO3156) known to be expressed during aerial growth and sporulation. On the other hand, the gene (SCO5039) specifically expressed in vegetative mycelium in <em>S. coelicolor</em> seemed not conserved in <em>K. setae</em> (Fig. <a reveal-id="DSQ026F4" data-open="DSQ026F4" class="link link-reveal link-table xref-fig">4</a>B). Such differences in stage-specific PBPs could also be reflected in differentiation specific incorporation of peptidoglycan units in <em>K. setae</em>. In this context, it is also interesting to note that SsgA-like family proteins (SALPs), likely involved in the regulation of peptidoglycan synthesis in sporogenic cell division, are much diversified in <em>K. setae</em> compared with <em>Streptomyces. Kitasatospora setae</em> encodes at least 12 SALPs, about twice as many as those found in <em>Streptomyces</em> species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                            <h3 scrollto-destination=7367180 id="7367180" class="section-title" data-legacy-id=s3e>3.5. Gene clusters for secondary metabolite biosyntheses</h3>
<p>Bacteria belonging to the genus <em>Kitasatospora</em> have been explored as potential new sources of various bioactive metabolites.<sup><a reveal-id="DSQ026C56" data-open="DSQ026C56" class="link link-ref link-reveal xref-bibr">56</a></sup><em>Kitasatospora setae</em> NBRC 14216<sup>T</sup> is known to produce setamycin (bafilomycin B1), which bears antitrichomonal activity.<sup><a reveal-id="DSQ026C57" data-open="DSQ026C57" class="link link-ref link-reveal xref-bibr">57</a></sup> A total of 24 genes or gene clusters in the <em>K. setae</em> genome were predicted to be involved in the biosynthesis of secondary metabolites (Table <a reveal-id="DSQ026TB3" data-open="DSQ026TB3" class="link link-reveal link-table xref-fig">3</a>). Of the 24 clusters, more than 60% (16 clusters) were located in subtelomeric regions; 11 were in the right subtelomeric region, whereas 5 were in the left subtelomeric region. The number of predicted gene clusters for secondary metabolism was slightly lower than that predicted in <em>Streptomyces</em> species (36 in <em>S. griseus</em> IFO 13350, 37 in <em>S. avermitilis</em> MA-4680<sup>T</sup>, 30 in <em>S. coelicolor</em> A3(2) and at least 20 in <em>S. scabies</em> 87.22), but apparently higher than that in other prokaryotes,<sup><a reveal-id="DSQ026C58" data-open="DSQ026C58" class="link link-ref link-reveal xref-bibr">58</a></sup> underscoring the importance of the genus <em>Kitasatospora</em> as the source of bioactive compounds. </p>                        <a id="7367184" scrollto-destination="7367184"></a>
<div content-id="DSQ026TB3" class="table-modal"><div class="table-wrap"><div id="DSQ026TB3" class="table-wrap-title" data-id="DSQ026TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Biosynthetic genes of secondary metabolites in <em>K. setae</em> NBRC 14216<sup>T</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Orfid </th><th>Note </th></tr></thead><tbody><tr><td colspan="2">Terpene </td></tr><tr><td> KSE_00200t (=KSE_76540t)<sup>a</sup> </td><td>Sesquiterpene </td></tr><tr><td> KSE_12950 </td><td>Sesquiterpene </td></tr><tr><td> KSE_17590-KSE_17630 </td><td>Squalene/hopanoid </td></tr><tr><td> KSE_46080 </td><td>Germacradienol/geosmin </td></tr><tr><td> KSE_70210-KSE_70220 </td><td>2-Methylisoborneol </td></tr><tr><td colspan="2">NRPS and PKS </td></tr><tr><td> KSE_18000 </td><td>Type III PKS </td></tr><tr><td> KSE_22630-KSE_22810 </td><td>Discrete NRPS </td></tr><tr><td> KSE_27200-KSE_27290 </td><td>Type I PKS </td></tr><tr><td> KSE_33340-KSE_33360 </td><td>Discrete NRPS </td></tr><tr><td> KSE_58150-KSE_58200 </td><td>NRPS </td></tr><tr><td> KSE_61120 </td><td>NRPS (incomplete) </td></tr><tr><td> KSE_65510-KSE_65560 </td><td>NRPS and type I PKS </td></tr><tr><td> KSE_65960-KSE_66030 </td><td>NRPS and type II PKS </td></tr><tr><td> KSE_70570-KSE_70620 </td><td>Type I PKS and NRPS (factumycin) </td></tr><tr><td> KSE_72410-KSE_72480 </td><td>Type II PKS (spore pigment) </td></tr><tr><td> KSE_73410-KSE_73580 </td><td>Type I PKS (bafilomycins) </td></tr><tr><td> KSE_75420-KSE_75430 </td><td>NRPS </td></tr><tr><td colspan="2">Other </td></tr><tr><td> KSE_04750-KSE_04770 </td><td>Lantibiotic </td></tr><tr><td> KSE_09030-KSE_09170 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_12660-KSE_12700 </td><td>Siderophore </td></tr><tr><td> KSE_27300-KSE_27440 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_45610-KSE_45680 </td><td>Lantibiotic </td></tr><tr><td> KSE_58810-KSE_58830 </td><td>Lantibiotic </td></tr><tr><td> KSE_53800-KSE_53830 </td><td>Siderophore </td></tr></tbody></table></div><div content-id="DSQ026TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Orfid </th><th>Note </th></tr></thead><tbody><tr><td colspan="2">Terpene </td></tr><tr><td> KSE_00200t (=KSE_76540t)<sup>a</sup> </td><td>Sesquiterpene </td></tr><tr><td> KSE_12950 </td><td>Sesquiterpene </td></tr><tr><td> KSE_17590-KSE_17630 </td><td>Squalene/hopanoid </td></tr><tr><td> KSE_46080 </td><td>Germacradienol/geosmin </td></tr><tr><td> KSE_70210-KSE_70220 </td><td>2-Methylisoborneol </td></tr><tr><td colspan="2">NRPS and PKS </td></tr><tr><td> KSE_18000 </td><td>Type III PKS </td></tr><tr><td> KSE_22630-KSE_22810 </td><td>Discrete NRPS </td></tr><tr><td> KSE_27200-KSE_27290 </td><td>Type I PKS </td></tr><tr><td> KSE_33340-KSE_33360 </td><td>Discrete NRPS </td></tr><tr><td> KSE_58150-KSE_58200 </td><td>NRPS </td></tr><tr><td> KSE_61120 </td><td>NRPS (incomplete) </td></tr><tr><td> KSE_65510-KSE_65560 </td><td>NRPS and type I PKS </td></tr><tr><td> KSE_65960-KSE_66030 </td><td>NRPS and type II PKS </td></tr><tr><td> KSE_70570-KSE_70620 </td><td>Type I PKS and NRPS (factumycin) </td></tr><tr><td> KSE_72410-KSE_72480 </td><td>Type II PKS (spore pigment) </td></tr><tr><td> KSE_73410-KSE_73580 </td><td>Type I PKS (bafilomycins) </td></tr><tr><td> KSE_75420-KSE_75430 </td><td>NRPS </td></tr><tr><td colspan="2">Other </td></tr><tr><td> KSE_04750-KSE_04770 </td><td>Lantibiotic </td></tr><tr><td> KSE_09030-KSE_09170 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_12660-KSE_12700 </td><td>Siderophore </td></tr><tr><td> KSE_27300-KSE_27440 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_45610-KSE_45680 </td><td>Lantibiotic </td></tr><tr><td> KSE_58810-KSE_58830 </td><td>Lantibiotic </td></tr><tr><td> KSE_53800-KSE_53830 </td><td>Siderophore </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>KSE_00200t was embedded in TIR and identical to KSE_76540t.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367184" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ026TB3" class="table-wrap-title" data-id="DSQ026TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Biosynthetic genes of secondary metabolites in <em>K. setae</em> NBRC 14216<sup>T</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Orfid </th><th>Note </th></tr></thead><tbody><tr><td colspan="2">Terpene </td></tr><tr><td> KSE_00200t (=KSE_76540t)<sup>a</sup> </td><td>Sesquiterpene </td></tr><tr><td> KSE_12950 </td><td>Sesquiterpene </td></tr><tr><td> KSE_17590-KSE_17630 </td><td>Squalene/hopanoid </td></tr><tr><td> KSE_46080 </td><td>Germacradienol/geosmin </td></tr><tr><td> KSE_70210-KSE_70220 </td><td>2-Methylisoborneol </td></tr><tr><td colspan="2">NRPS and PKS </td></tr><tr><td> KSE_18000 </td><td>Type III PKS </td></tr><tr><td> KSE_22630-KSE_22810 </td><td>Discrete NRPS </td></tr><tr><td> KSE_27200-KSE_27290 </td><td>Type I PKS </td></tr><tr><td> KSE_33340-KSE_33360 </td><td>Discrete NRPS </td></tr><tr><td> KSE_58150-KSE_58200 </td><td>NRPS </td></tr><tr><td> KSE_61120 </td><td>NRPS (incomplete) </td></tr><tr><td> KSE_65510-KSE_65560 </td><td>NRPS and type I PKS </td></tr><tr><td> KSE_65960-KSE_66030 </td><td>NRPS and type II PKS </td></tr><tr><td> KSE_70570-KSE_70620 </td><td>Type I PKS and NRPS (factumycin) </td></tr><tr><td> KSE_72410-KSE_72480 </td><td>Type II PKS (spore pigment) </td></tr><tr><td> KSE_73410-KSE_73580 </td><td>Type I PKS (bafilomycins) </td></tr><tr><td> KSE_75420-KSE_75430 </td><td>NRPS </td></tr><tr><td colspan="2">Other </td></tr><tr><td> KSE_04750-KSE_04770 </td><td>Lantibiotic </td></tr><tr><td> KSE_09030-KSE_09170 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_12660-KSE_12700 </td><td>Siderophore </td></tr><tr><td> KSE_27300-KSE_27440 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_45610-KSE_45680 </td><td>Lantibiotic </td></tr><tr><td> KSE_58810-KSE_58830 </td><td>Lantibiotic </td></tr><tr><td> KSE_53800-KSE_53830 </td><td>Siderophore </td></tr></tbody></table></div><div content-id="DSQ026TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Orfid </th><th>Note </th></tr></thead><tbody><tr><td colspan="2">Terpene </td></tr><tr><td> KSE_00200t (=KSE_76540t)<sup>a</sup> </td><td>Sesquiterpene </td></tr><tr><td> KSE_12950 </td><td>Sesquiterpene </td></tr><tr><td> KSE_17590-KSE_17630 </td><td>Squalene/hopanoid </td></tr><tr><td> KSE_46080 </td><td>Germacradienol/geosmin </td></tr><tr><td> KSE_70210-KSE_70220 </td><td>2-Methylisoborneol </td></tr><tr><td colspan="2">NRPS and PKS </td></tr><tr><td> KSE_18000 </td><td>Type III PKS </td></tr><tr><td> KSE_22630-KSE_22810 </td><td>Discrete NRPS </td></tr><tr><td> KSE_27200-KSE_27290 </td><td>Type I PKS </td></tr><tr><td> KSE_33340-KSE_33360 </td><td>Discrete NRPS </td></tr><tr><td> KSE_58150-KSE_58200 </td><td>NRPS </td></tr><tr><td> KSE_61120 </td><td>NRPS (incomplete) </td></tr><tr><td> KSE_65510-KSE_65560 </td><td>NRPS and type I PKS </td></tr><tr><td> KSE_65960-KSE_66030 </td><td>NRPS and type II PKS </td></tr><tr><td> KSE_70570-KSE_70620 </td><td>Type I PKS and NRPS (factumycin) </td></tr><tr><td> KSE_72410-KSE_72480 </td><td>Type II PKS (spore pigment) </td></tr><tr><td> KSE_73410-KSE_73580 </td><td>Type I PKS (bafilomycins) </td></tr><tr><td> KSE_75420-KSE_75430 </td><td>NRPS </td></tr><tr><td colspan="2">Other </td></tr><tr><td> KSE_04750-KSE_04770 </td><td>Lantibiotic </td></tr><tr><td> KSE_09030-KSE_09170 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_12660-KSE_12700 </td><td>Siderophore </td></tr><tr><td> KSE_27300-KSE_27440 </td><td>Similar to valanimycin biosynthetic genes </td></tr><tr><td> KSE_45610-KSE_45680 </td><td>Lantibiotic </td></tr><tr><td> KSE_58810-KSE_58830 </td><td>Lantibiotic </td></tr><tr><td> KSE_53800-KSE_53830 </td><td>Siderophore </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>KSE_00200t was embedded in TIR and identical to KSE_76540t.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7367184" target="_blank">View Large</a></div></div><p>Of the 24 clusters, 5 were estimated for terpene biosynthesis, 12 for polyketides or non-ribosomal peptides, 2 for siderophores and 5 for lantibiotics and others. Five ORFs containing the terpene synthase domain (IPR005630) were classified by the phylogenetic analysis described by Komatsu <em>et al</em>.,<sup><a reveal-id="DSQ026C59" data-open="DSQ026C59" class="link link-ref link-reveal xref-bibr">59</a></sup> indicating that KSE_46080 was in the group of germacradienol/geosmin synthase<sup><a reveal-id="DSQ026C60" data-open="DSQ026C60" class="link link-ref link-reveal xref-bibr">60</a>,<a reveal-id="DSQ026C61" data-open="DSQ026C61" class="link link-ref link-reveal xref-bibr">61</a></sup> and KSE_70210 in the group of 2-methylisoborneol synthase. Consistently, geosmin and 2-methylisoborneol were identified from the culture of <em>K. setae</em>. Of the 12 clusters for polyketides or non-ribosomal peptides, setamycin (bafilomycin B1) cluster was estimated and experimentally proved to be KSE_73410–KSE_73580 (H. Ikeda, unpublished results). An 84 kb region (KSE_70410–KSE_70650) containing the cluster KSE_70570–KSE_70620 showed striking resemblance in features of gene organization and in deduced amino acid sequences of each ORF to the kirromycin biosynthetic gene cluster, the first characterized combined <em>cis</em>–<em>trans</em>-PKS cluster in <em>Streptomyces collinus</em> Tu 365.<sup><a reveal-id="DSQ026C62" data-open="DSQ026C62" class="link link-ref link-reveal xref-bibr">62</a></sup> This region was experimentally confirmed to be responsible for the biosynthesis of factumycin, an antibiotic structurally related to kirromycin (H. Ikeda, unpublished results). Regarding the two siderophore clusters, the one (KSE_12660–KSE_12700) showed good similarity in gene arrangement and deduced amino sequence of each ORF to a cluster in <em>S. avermitilis</em> (SAV_7320–SAV_7323, lacking a homolog corresponding to KSE_12680), both of which were similar to vibrioferrin cluster identified in <em>Vibrio parahaemolyticus</em>.<sup><a reveal-id="DSQ026C63" data-open="DSQ026C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Another one (KSE_53800–KSE_53830) was similar in gene arrangement to the rhizobactin cluster in <em>Sinorhizobium meliloti</em>,<sup><a reveal-id="DSQ026C64" data-open="DSQ026C64" class="link link-ref link-reveal xref-bibr">64</a></sup> but differed from any of the known siderophore biosynthesis clusters of <em>Streptomyces</em>. Interestingly, <em>K. setae</em> lacks a gene cluster for nocardamin (desferrioxamine) biosynthesis that was commonly found in genome-sequenced <em>Streptomyces</em> species.<sup><a reveal-id="DSQ026C58" data-open="DSQ026C58" class="link link-ref link-reveal xref-bibr">58</a>,<a reveal-id="DSQ026C65" data-open="DSQ026C65" class="link link-ref link-reveal xref-bibr">65</a></sup></p>                            <h3 scrollto-destination=7367188 id="7367188" class="section-title" data-legacy-id=s3f>3.6. Conclusion</h3>
<p>Streptomycetes are thought to have emerged ∼440 million years ago. At present, <em>Streptomyces</em> species are some of the most highly differentiated microorganisms with a complex life cycle. The cell wall peptidoglycan composed of <span class="small-caps">ll</span>-DAP is another feature to be considered to distinguish <em>Streptomyces</em> from other sporoactinomycetes. Here, we analyzed the first genome sequence of a <em>Streptomycetaceae</em> bacterium other than <em>Streptomyces</em> species. Phylogenetic analysis based on amino acid sequences, together with a genome-wide comparison of the predicted genes such as those related to morphological differentiation and cell wall biosynthesis, suggests that the genera <em>Streptomyces</em> and <em>Kitasatospora</em> were diverged directly from their last common ancestor. The chromosomal linearity and the presence of the TIR sequence would also be the features inherited from the common ancestor. Many differentiation-related genes highly conserved in <em>K. setae</em> and <em>Streptomyces</em> species might have been acquired sequentially in the evolution, as previously suggested from the comparison of <em>Streptomyces</em> with other groups of mycelial actinobacteria, <em>Thermobifida fusca</em> and <em>Frankia</em> species.<sup><a reveal-id="DSQ026C1" data-open="DSQ026C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSQ026C42" data-open="DSQ026C42" class="link link-ref link-reveal xref-bibr">42</a></sup> If additional genomic sequences determined more recently such as <em>C. acidiphila</em>, <em>S. erythraea</em>, <em>Salinispora</em> species and <em>K. setae</em> were taken into consideration, however, the evolutionary pathway of differentiation-related genes seemed not necessarily be straightforward, and gene losses and horizontal acquisitions, in particular lineage, might also have to be considered (Table <a reveal-id="DSQ026TB2" data-open="DSQ026TB2" class="link link-reveal link-table xref-fig">2</a>). Phylogenetic data presented in Fig. <a reveal-id="DSQ026F2" data-open="DSQ026F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a> also suggest that mycelial actinobacteria are not monophyletic; some organisms such as <em>S. erythraea</em> and <em>T. fusca</em> belong to sub-branches that also contain bacteria without most of the developmental genes. Extraordinary high similarity observed in <em>bld</em> regulatory genes, which are dispersed in the genome, might also reflect recent horizontal transfer. Although the highly regulated differentiation system of <em>Streptomyces</em> has attracted much attention, reasons for the unique abundance of <span class="small-caps">ll</span>-DAP in peptidoglycan may have been overlooked. We provided the possible background of the relative abundance of <em>meso</em>- and <span class="small-caps">ll</span>-DAP in <em>K. setae.</em> Further analysis and comparison of two genera, including genome sequencing of another <em>Kitasatospora</em> species, will provide deeper understanding of the evolution of actinobacteria.</p>                            <h2 scrollto-destination=7367192 id="7367192" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7367196 id="7367196" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank S. Omata, S. Gomi, S. Fujinami, A. Kawakoshi, K. Koga and C. Koga for the assistance in annotation and A. Hosoyama, T. Harada, R. Igarashi and H. Horikawa for draft sequencing. We also thank T. Tamura, K. Mori and K. Suzuki for the help and fruitful discussions about phylogenetic analysis and H. Komaki for valuable discussions about secondary metabolism-related genes.</p>                            <h2 scrollto-destination=7367199 id="7367199" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ026C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chater</div> <div class="given-names">K.F.</div></div>,  <div class="name"><div class="surname">Chandra</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The evolution of development in <em>Streptomyces</em> analysed by genome comparisons</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2006</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">651</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20development%20in%20Streptomyces%20analysed%20by%20genome%20comparisons&amp;author=K.F.%20Chater&amp;author=G.%20Chandra&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=30&amp;pages=651-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2006.00033.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2006.00033.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2006.00033.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16911038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Mining and polishing of the treasure trove in the bacterial genus <em>Streptomyces</em></div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2007</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">283</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0168-583X(01)00748-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0168-583X(01)00748-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20and%20polishing%20of%20the%20treasure%20trove%20in%20the%20bacterial%20genus%20Streptomyces&amp;author=S.%20Horinouchi&amp;publication_year=2007&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=71&amp;pages=283-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.60627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.60627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.60627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bentley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Chater</div> <div class="given-names">K.F.</div></div>,  <div class="name"><div class="surname">Cerdeno-Tarraga</div> <div class="given-names">A.M.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the model actinomycete <em>Streptomyces coelicolor</em> A3(2)</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">141</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20model%20actinomycete%20Streptomyces%20coelicolor%20A3(2)&amp;author=S.D.%20Bentley&amp;author=K.F.%20Chater&amp;author=A.M.%20Cerdeno-Tarraga&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=141-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/417141a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F417141a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F417141a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12000953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikeda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hanamoto</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Complete genome sequence and comparative analysis of the industrial microorganism <em>Streptomyces avermitilis</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">526</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20and%20comparative%20analysis%20of%20the%20industrial%20microorganism%20Streptomyces%20avermitilis&amp;author=H.%20Ikeda&amp;author=J.%20Ishikawa&amp;author=A.%20Hanamoto&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=526-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt820" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt820" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt820"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12692562" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genome sequence of the streptomycin-producing microorganism <em>Streptomyces griseus</em> IFO 13350</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">4050</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2003.12.023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2003.12.023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20streptomycin-producing%20microorganism%20Streptomyces%20griseus%20IFO%2013350&amp;author=Y.%20Ohnishi&amp;author=J.%20Ishikawa&amp;author=H.%20Hara&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=4050-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00204-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00204-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00204-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Momose</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Sekizawa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hirosawa</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Tyropeptins A and B, new proteasome inhibitors produced by <em>Kitasatospora</em> sp. MK993-dF2. II. Structure determination and synthesis</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">2001</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">1004</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S1350-4487(96)00085-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S1350-4487(96)00085-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tyropeptins%20A%20and%20B%2C%20new%20proteasome%20inhibitors%20produced%20by%20Kitasatospora%20sp.%20MK993-dF2.%20II.%20Structure%20determination%20and%20synthesis&amp;author=I.%20Momose&amp;author=R.%20Sekizawa&amp;author=S.%20Hirosawa&amp;publication_year=2001&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=54&amp;pages=1004-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7164/antibiotics.54.1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7164%2Fantibiotics.54.1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7164%2Fantibiotics.54.1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11858653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoon</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Suh</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Talosins A and B: new isoflavonol glycosides with potent antifungal activity from <em>Kitasatospora kifunensis</em> MJM341. I. Taxonomy, fermentation, isolation, and biological activities</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">2006</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">633</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/1350-4487(94)00084-E" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/1350-4487(94)00084-E</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Talosins%20A%20and%20B%3A%20new%20isoflavonol%20glycosides%20with%20potent%20antifungal%20activity%20from%20Kitasatospora%20kifunensis%20MJM341.%20I.%20Taxonomy%2C%20fermentation%2C%20isolation%2C%20and%20biological%20activities&amp;author=T.M.%20Yoon&amp;author=J.W.%20Kim&amp;author=J.G.%20Kim&amp;author=W.G.%20Kim&amp;author=J.W.%20Suh&amp;publication_year=2006&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=59&amp;pages=633-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ja.2006.84" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fja.2006.84" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fja.2006.84"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17191678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">H.</div></div>. </div><div class="article-title"><em>Kitasatospora</em>, a new genus of the order Actinomycetales</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">1982</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">1013</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Kitasatospora%2C%20a%20new%20genus%20of%20the%20order%20Actinomycetales&amp;author=S.%20Omura&amp;author=Y.%20Takahashi&amp;author=Y.%20Iwai&amp;author=H.%20Tanaka&amp;publication_year=1982&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=35&amp;pages=1013-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7164/antibiotics.35.1013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7164%2Fantibiotics.35.1013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7164%2Fantibiotics.35.1013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7142001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wellington</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Stackebrandt</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Sanders</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Wolstrup</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jorgensen</div> <div class="given-names">N.O.</div></div>. </div><div class="article-title">Taxonomic status of <em>Kitasatospora</em>, and proposed unification with <em>Streptomyces</em> on the basis of phenotypic and 16S rRNA analysis and emendation of <em>Streptomyces</em> Waksman and Henrici 1943, 339AL</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1992</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">156</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1097/00004032-197108000-00016" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1097/00004032-197108000-00016</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20status%20of%20Kitasatospora%2C%20and%20proposed%20unification%20with%20Streptomyces%20on%20the%20basis%20of%20phenotypic%20and%2016S%20rRNA%20analysis%20and%20emendation%20of%20Streptomyces%20Waksman%20and%20Henrici%201943%2C%20339AL&amp;author=E.M.%20Wellington&amp;author=E.%20Stackebrandt&amp;author=D.%20Sanders&amp;author=J.%20Wolstrup&amp;author=N.O.%20Jorgensen&amp;publication_year=1992&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=42&amp;pages=156-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-42-1-156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-42-1-156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-42-1-156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1371059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ochi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hiranuma</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A taxonomic review of the genera <em>Kitasatospora</em> and <em>Streptoverticillium</em> by analysis of ribosomal protein AT-L30</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">285</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20taxonomic%20review%20of%20the%20genera%20Kitasatospora%20and%20Streptoverticillium%20by%20analysis%20of%20ribosomal%20protein%20AT-L30&amp;author=K.%20Ochi&amp;author=H.%20Hiranuma&amp;publication_year=1994&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=44&amp;pages=285-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-44-2-285" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-44-2-285" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-44-2-285"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8186092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A proposal to revive the genus <em>Kitasatospora</em> (Omura, Takahashi, Iwai, and Tanaka 1982)</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1997</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1048</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20proposal%20to%20revive%20the%20genus%20Kitasatospora%20(Omura%2C%20Takahashi%2C%20Iwai%2C%20and%20Tanaka%201982)&amp;author=Z.%20Zhang&amp;author=Y.%20Wang&amp;author=J.%20Ruan&amp;publication_year=1997&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=47&amp;pages=1048-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-47-4-1048" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-47-4-1048" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-47-4-1048"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9336904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gunther</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Groth</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Grabley</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Munder</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Design and evaluation of an oligonucleotide-microarray for the detection of different species of the genus <em>Kitasatospora</em></div>, <div class="source">J. Microbiol. Methods</div> , <div class="year">2006</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">226</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2005.02.013" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2005.02.013</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20and%20evaluation%20of%20an%20oligonucleotide-microarray%20for%20the%20detection%20of%20different%20species%20of%20the%20genus%20Kitasatospora&amp;author=S.%20Gunther&amp;author=I.%20Groth&amp;author=S.%20Grabley&amp;author=T.%20Munder&amp;publication_year=2006&amp;journal=J.%20Microbiol.%20Methods&amp;volume=65&amp;pages=226-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mimet.2005.07.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2005.07.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mimet.2005.07.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16099060" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moller</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schuler</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gunther</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Carlsohn</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Munder</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fritzsche</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Electrical DNA-chip-based identification of different species of the genus <em>Kitasatospora</em></div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">1181</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Electrical%20DNA-chip-based%20identification%20of%20different%20species%20of%20the%20genus%20Kitasatospora&amp;author=R.%20Moller&amp;author=T.%20Schuler&amp;author=S.%20Gunther&amp;author=M.R.%20Carlsohn&amp;author=T.%20Munder&amp;author=W.%20Fritzsche&amp;publication_year=2008&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=77&amp;pages=1181-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-007-1241-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-007-1241-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-007-1241-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17994235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Relationship between cell morphology and the types of diaminopimelic acid in <em>Kitasatospora setalba</em></div>, <div class="source">J. Gen. Appl. Microbiol.</div> , <div class="year">1983</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">459</div>-<div class="lpage">65</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2008.04.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2008.04.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relationship%20between%20cell%20morphology%20and%20the%20types%20of%20diaminopimelic%20acid%20in%20Kitasatospora%20setalba&amp;author=Y.%20Takahashi&amp;author=Y.%20Iwai&amp;author=S.%20Omura&amp;publication_year=1983&amp;journal=J.%20Gen.%20Appl.%20Microbiol.&amp;volume=29&amp;pages=459-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2323/jgam.29.459" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2323%2Fjgam.29.459" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2323%2Fjgam.29.459"> </span></div> </div></div></div></div></div><div content-id="DSQ026C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kuwana</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Some characteristics of aerial and submerged spores of <em>Kitasatospora setalba</em></div>, <div class="source">J. Gen. Appl. Microbiol.</div> , <div class="year">1984</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">223</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2009.10.089" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2009.10.089</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Some%20characteristics%20of%20aerial%20and%20submerged%20spores%20of%20Kitasatospora%20setalba&amp;author=Y.%20Takahashi&amp;author=T.%20Kuwana&amp;author=Y.%20Iwai&amp;author=S.%20Omura&amp;publication_year=1984&amp;journal=J.%20Gen.%20Appl.%20Microbiol.&amp;volume=30&amp;pages=223-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2323/jgam.30.223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2323%2Fjgam.30.223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2323%2Fjgam.30.223"> </span></div> </div></div></div></div></div><div content-id="DSQ026C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kojima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Otoguro</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Oiwa</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Type of diaminopimelic acid different in aerial and vegetative mycelia of setamycin-producing actinomycete KM-6054</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">1981</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1633</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Type%20of%20diaminopimelic%20acid%20different%20in%20aerial%20and%20vegetative%20mycelia%20of%20setamycin-producing%20actinomycete%20KM-6054&amp;author=S.%20Omura&amp;author=Y.%20Iwai&amp;author=Y.%20Takahashi&amp;author=K.%20Kojima&amp;author=K.%20Otoguro&amp;author=R.%20Oiwa&amp;publication_year=1981&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=34&amp;pages=1633-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7164/antibiotics.34.1633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7164%2Fantibiotics.34.1633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7164%2Fantibiotics.34.1633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7333980" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Seino</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Taxonomic study and morphological differentiation of an actinomycete genus, <em>Kitasatospora</em></div>, <div class="source">Zentralbl. Bakteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">265</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2009.10.075" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2009.10.075</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taxonomic%20study%20and%20morphological%20differentiation%20of%20an%20actinomycete%20genus%2C%20Kitasatospora&amp;author=Y.%20Takahashi&amp;author=A.%20Seino&amp;author=Y.%20Iwai&amp;author=S.%20Omura&amp;publication_year=1999&amp;journal=Zentralbl.%20Bakteriol.&amp;volume=289&amp;pages=265-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0934-8840(99)80065-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0934-8840(99)80065-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0934-8840(99)80065-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10467659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sekine</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tanikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Omata</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Sequence analysis of three plasmids harboured in <em>Rhodococcus erythropolis</em> strain PR4</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">334</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20three%20plasmids%20harboured%20in%20Rhodococcus%20erythropolis%20strain%20PR4&amp;author=M.%20Sekine&amp;author=S.%20Tanikawa&amp;author=S.%20Omata&amp;publication_year=2006&amp;journal=Environ.%20Microbiol.&amp;volume=8&amp;pages=334-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2006.8.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2006.8.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2006.8.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takarada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sekine</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kosugi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the soil actinomycete <em>Kocuria rhizophila</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">4139</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.radmeas.2005.01.009" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.radmeas.2005.01.009</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20soil%20actinomycete%20Kocuria%20rhizophila&amp;author=H.%20Takarada&amp;author=M.%20Sekine&amp;author=H.%20Kosugi&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=4139-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01853-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01853-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01853-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18408034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. II. Error probabilities</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">186</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20II.%20Error%20probabilities&amp;author=B.%20Ewing&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=186-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wendl</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. I. Accuracy assessment</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">175</div>-<div class="lpage">85</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.asr.2009.08.027" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.asr.2009.08.027</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20I.%20Accuracy%20assessment&amp;author=B.%20Ewing&amp;author=L.%20Hillier&amp;author=M.C.%20Wendl&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=175-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Bratke</div> <div class="given-names">K.A.</div></div>,  <div class="name"><div class="surname">Powers</div> <div class="given-names">E.C.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">673</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/rpd/ncp265" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/rpd/ncp265</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20bacterial%20genes%20and%20endosymbiont%20DNA%20with%20Glimmer&amp;author=A.L.%20Delcher&amp;author=K.A.%20Bratke&amp;author=E.C.%20Powers&amp;author=S.L.%20Salzberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=673-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hotta</div> <div class="given-names">K.</div></div>. </div><div class="article-title">FramePlot: a new implementation of the frame analysis for predicting protein-coding regions in bacterial DNA with a high G + C content</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">1999</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">251</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FramePlot%3A%20a%20new%20implementation%20of%20the%20frame%20analysis%20for%20predicting%20protein-coding%20regions%20in%20bacterial%20DNA%20with%20a%20high%20G%20%2B%20C%20content&amp;author=J.%20Ishikawa&amp;author=K.%20Hotta&amp;publication_year=1999&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=174&amp;pages=251-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.1999.174.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.1999.174.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.1999.174.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10339816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="collab">UniProt, Consortium</div><div class="article-title">The universal protein resource (UniProt)</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D190</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18045787" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>, et al. </div><div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D224</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17202162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Auchincloss</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Coudert</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">HAMAP: a database of completely sequenced microbial proteome sets and manually curated microbial protein families in UniProtKB/Swiss-Prot</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D471</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HAMAP%3A%20a%20database%20of%20completely%20sequenced%20microbial%20proteome%20sets%20and%20manually%20curated%20microbial%20protein%20families%20in%20UniProtKB%2FSwiss-Prot&amp;author=T.%20Lima&amp;author=A.H.%20Auchincloss&amp;author=E.%20Coudert&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D471-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18849571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Araki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">KEGG for linking genomes to life and the environment</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D480</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%20for%20linking%20genomes%20to%20life%20and%20the%20environment&amp;author=M.%20Kanehisa&amp;author=M.%20Araki&amp;author=S.%20Goto&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D480-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm882" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm882" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm882"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18077471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ren</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Paulsen</div> <div class="given-names">I.T.</div></div>. </div><div class="article-title">TransportDB: a comprehensive database resource for cytoplasmic membrane transport systems and outer membrane channels</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D274</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TransportDB%3A%20a%20comprehensive%20database%20resource%20for%20cytoplasmic%20membrane%20transport%20systems%20and%20outer%20membrane%20channels&amp;author=Q.%20Ren&amp;author=K.%20Chen&amp;author=I.T.%20Paulsen&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D274-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl925"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17135193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rawlings</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Morton</div> <div class="given-names">F.R.</div></div>,  <div class="name"><div class="surname">Kok</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Kong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Barrett</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">MEROPS: the peptidase database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D320</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEROPS%3A%20the%20peptidase%20database&amp;author=N.D.%20Rawlings&amp;author=F.R.%20Morton&amp;author=C.Y.%20Kok&amp;author=J.%20Kong&amp;author=A.J.%20Barrett&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D320-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm954" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm954" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm954"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17991683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Moxon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khanna</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Rfam: annotating non-coding RNAs in complete genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">D121</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rfam%3A%20annotating%20non-coding%20RNAs%20in%20complete%20genomes&amp;author=S.%20Griffiths-Jones&amp;author=S.%20Moxon&amp;author=M.%20Marshall&amp;author=A.%20Khanna&amp;author=S.R.%20Eddy&amp;author=A.%20Bateman&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=D121-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki081"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15608160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laslett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Canback</div> <div class="given-names">B.</div></div>. </div><div class="article-title">ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">11</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARAGORN%2C%20a%20program%20to%20detect%20tRNA%20genes%20and%20tmRNA%20genes%20in%20nucleotide%20sequences&amp;author=D.%20Laslett&amp;author=B.%20Canback&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Worning</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Origin of replication in circular prokaryotic chromosomes</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">353</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20replication%20in%20circular%20prokaryotic%20chromosomes&amp;author=P.%20Worning&amp;author=L.J.%20Jensen&amp;author=P.F.%20Hallin&amp;author=H.H.%20Staerfeldt&amp;author=D.W.%20Ussery&amp;publication_year=2006&amp;journal=Environ.%20Microbiol.&amp;volume=8&amp;pages=353-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2006.8.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2006.8.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2006.8.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">376</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>. </div><div class="article-title">MBGD: a platform for microbial comparative genomics based on the automated construction of orthologous groups</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D343</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MBGD%3A%20a%20platform%20for%20microbial%20comparative%20genomics%20based%20on%20the%20automated%20construction%20of%20orthologous%20groups&amp;author=I.%20Uchiyama&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D343-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17135196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jakimowicz</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Majka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Messer</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Structural elements of the <em>Streptomyces oriC</em> region and their interactions with the DnaA protein</div>, <div class="source">Microbiology</div> , <div class="year">1998</div>, vol. <div class="volume">144</div> <div class="issue">Pt 5</div>(pg. <div class="fpage">1281</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20elements%20of%20the%20Streptomyces%20oriC%20region%20and%20their%20interactions%20with%20the%20DnaA%20protein&amp;author=D.%20Jakimowicz&amp;author=J.%20Majka&amp;author=W.%20Messer&amp;publication_year=1998&amp;journal=Microbiology&amp;volume=144&amp;pages=1281-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-144-5-1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-144-5-1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-144-5-1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9611803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Cohen</div> <div class="given-names">S.N.</div></div>. </div><div class="article-title">Recruitment of terminal protein to the ends of <em>Streptomyces</em> linear plasmids and chromosomes by a novel telomere-binding protein essential for linear DNA replication</div>, <div class="source">Genes Dev.</div> , <div class="year">2003</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">774</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recruitment%20of%20terminal%20protein%20to%20the%20ends%20of%20Streptomyces%20linear%20plasmids%20and%20chromosomes%20by%20a%20novel%20telomere-binding%20protein%20essential%20for%20linear%20DNA%20replication&amp;author=K.%20Bao&amp;author=S.N.%20Cohen&amp;publication_year=2003&amp;journal=Genes%20Dev.&amp;volume=17&amp;pages=774-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1060303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1060303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1060303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12651895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">C.W.</div></div>. </div><div class="article-title">The telomeres of <em>Streptomyces</em> chromosomes contain conserved palindromic sequences with potential to form complex secondary structures</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1998</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">905</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20telomeres%20of%20Streptomyces%20chromosomes%20contain%20conserved%20palindromic%20sequences%20with%20potential%20to%20form%20complex%20secondary%20structures&amp;author=C.H.%20Huang&amp;author=Y.S.%20Lin&amp;author=Y.L.%20Yang&amp;author=S.W.%20Huang&amp;author=C.W.%20Chen&amp;publication_year=1998&amp;journal=Mol.%20Microbiol.&amp;volume=28&amp;pages=905-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1998.00856.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1998.00856.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1998.00856.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9663678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ciccarelli</div> <div class="given-names">F.D.</div></div>,  <div class="name"><div class="surname">Doerks</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">von Mering</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Creevey</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Toward automatic reconstruction of a highly resolved tree of life</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">311</div> (pg. <div class="fpage">1283</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20automatic%20reconstruction%20of%20a%20highly%20resolved%20tree%20of%20life&amp;author=F.D.%20Ciccarelli&amp;author=T.%20Doerks&amp;author=C.%20von%20Mering&amp;author=C.J.%20Creevey&amp;author=B.%20Snel&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Science&amp;volume=311&amp;pages=1283-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1123061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1123061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1123061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16513982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Copeland</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lapidus</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rio</div> <div class="given-names">T.G.D.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Catenulispora acidiphila</em> type strain (ID 139908T)</div>, <div class="source">Stand. Genomic Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">119</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Catenulispora%20acidiphila%20type%20strain%20(ID%20139908T)&amp;author=A.%20Copeland&amp;author=A.%20Lapidus&amp;author=T.G.D.%20Rio&amp;publication_year=2009&amp;journal=Stand.%20Genomic%20Sci.&amp;volume=1&amp;pages=119-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4056/sigs.17259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4056%2Fsigs.17259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4056%2Fsigs.17259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ventura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Canchaya</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Tauch</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genomics of <em>Actinobacteria</em>: tracing the evolutionary history of an ancient phylum</div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">2007</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">495</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Actinobacteria%3A%20tracing%20the%20evolutionary%20history%20of%20an%20ancient%20phylum&amp;author=M.%20Ventura&amp;author=C.%20Canchaya&amp;author=A.%20Tauch&amp;publication_year=2007&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=71&amp;pages=495-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00005-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00005-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00005-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17804669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Willey</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Willems</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kodani</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nodwell</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Morphogenetic surfactants and their role in the formation of aerial hyphae in <em>Streptomyces coelicolor</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">731</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Morphogenetic%20surfactants%20and%20their%20role%20in%20the%20formation%20of%20aerial%20hyphae%20in%20Streptomyces%20coelicolor&amp;author=J.M.%20Willey&amp;author=A.%20Willems&amp;author=S.%20Kodani&amp;author=J.R.%20Nodwell&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=59&amp;pages=731-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2006.59.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2006.59.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2006.59.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16420347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliynyk</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Samborskyy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lester</div> <div class="given-names">J.B.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the erythromycin-producing bacterium <em>Saccharopolyspora erythraea</em> NRRL23338</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">447</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20erythromycin-producing%20bacterium%20Saccharopolyspora%20erythraea%20NRRL23338&amp;author=M.%20Oliynyk&amp;author=M.%20Samborskyy&amp;author=J.B.%20Lester&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=447-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17369815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chandra</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Chater</div> <div class="given-names">K.F.</div></div>. </div><div class="article-title">Evolutionary flux of potentially <em>bldA</em>-dependent <em>Streptomyces</em> genes containing the rare leucine codon TTA</div>, <div class="source">Antonie Van Leeuwenhoek</div> , <div class="year">2008</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">111</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20flux%20of%20potentially%20bldA-dependent%20Streptomyces%20genes%20containing%20the%20rare%20leucine%20codon%20TTA&amp;author=G.%20Chandra&amp;author=K.F.%20Chater&amp;publication_year=2008&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=94&amp;pages=111-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10482-008-9231-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10482-008-9231-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10482-008-9231-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18320344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Capstick</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Willey</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Buttner</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Elliot</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">SapB and the chaplins: connections between morphogenetic proteins in <em>Streptomyces coelicolor</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">602</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SapB%20and%20the%20chaplins%3A%20connections%20between%20morphogenetic%20proteins%20in%20Streptomyces%20coelicolor&amp;author=D.S.%20Capstick&amp;author=J.M.%20Willey&amp;author=M.J.%20Buttner&amp;author=M.A.%20Elliot&amp;publication_year=2007&amp;journal=Mol.%20Microbiol.&amp;volume=64&amp;pages=602-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.64.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.64.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.64.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17462011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Claessen</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Rink</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">de Jong</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">A novel class of secreted hydrophobic proteins is involved in aerial hyphae formation in <em>Streptomyces coelicolor</em> by forming amyloid-like fibrils</div>, <div class="source">Genes Dev.</div> , <div class="year">2003</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1714</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20class%20of%20secreted%20hydrophobic%20proteins%20is%20involved%20in%20aerial%20hyphae%20formation%20in%20Streptomyces%20coelicolor%20by%20forming%20amyloid-like%20fibrils&amp;author=D.%20Claessen&amp;author=R.%20Rink&amp;author=W.%20de%20Jong&amp;publication_year=2003&amp;journal=Genes%20Dev.&amp;volume=17&amp;pages=1714-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.264303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.264303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.264303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12832396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">S.U.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.K.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">Y.I.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nihira</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Cloning and functional analysis by gene disruption of a gene encoding a γ-butyrolactone autoregulator receptor from <em>Kitasatospora setae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">3423</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20functional%20analysis%20by%20gene%20disruption%20of%20a%20gene%20encoding%20a%20%CE%B3-butyrolactone%20autoregulator%20receptor%20from%20Kitasatospora%20setae&amp;author=S.U.%20Choi&amp;author=C.K.%20Lee&amp;author=Y.I.%20Hwang&amp;author=H.%20Kinoshita&amp;author=T.%20Nihira&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=3423-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.11.3423-3430.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.11.3423-3430.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.11.3423-3430.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15150228" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Funa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Biosynthesis of gamma-butyrolactone autoregulators that switch on secondary metabolism and morphological development in <em>Streptomyces</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">2378</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20gamma-butyrolactone%20autoregulators%20that%20switch%20on%20secondary%20metabolism%20and%20morphological%20development%20in%20Streptomyces&amp;author=J.Y.%20Kato&amp;author=N.%20Funa&amp;author=H.%20Watanabe&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=2378-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0607472104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0607472104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0607472104"> </span></div> </div></div></div></div></div><div content-id="DSQ026C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Mengin-Lecreulx</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Stragier</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Molecular cloning, characterization, and chromosomal localization of dapF, the <em>Escherichia coli</em> gene for diaminopimelate epimerase</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1987</div>, vol. <div class="volume">169</div> (pg. <div class="fpage">1454</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%2C%20characterization%2C%20and%20chromosomal%20localization%20of%20dapF%2C%20the%20Escherichia%20coli%20gene%20for%20diaminopimelate%20epimerase&amp;author=C.%20Richaud&amp;author=W.%20Higgins&amp;author=D.%20Mengin-Lecreulx&amp;author=P.%20Stragier&amp;publication_year=1987&amp;journal=J.%20Bacteriol.&amp;volume=169&amp;pages=1454-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3031013" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abo-Ghalia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Michaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Blanot</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">van Heijenoort</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Specificity of the uridine-diphosphate-N-acetylmuramyl-L-alanyl-D-glutamate: meso-2,6-diaminopimelate synthetase from <em>Escherichia coli</em></div>, <div class="source">Eur. J. Biochem.</div> , <div class="year">1985</div>, vol. <div class="volume">153</div> (pg. <div class="fpage">81</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specificity%20of%20the%20uridine-diphosphate-N-acetylmuramyl-L-alanyl-D-glutamate%3A%20meso-2%2C6-diaminopimelate%20synthetase%20from%20Escherichia%20coli&amp;author=M.%20Abo-Ghalia&amp;author=C.%20Michaud&amp;author=D.%20Blanot&amp;author=J.%20van%20Heijenoort&amp;publication_year=1985&amp;journal=Eur.%20J.%20Biochem.&amp;volume=153&amp;pages=81-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.1985.153.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.1985.153.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.1985.153.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3905407" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mengin-Lecreulx</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Michaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Richaud</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Blanot</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">van Heijenoort</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Incorporation of LL-diaminopimelic acid into peptidoglycan of <em>Escherichia coli</em> mutants lacking diaminopimelate epimerase encoded by dapF</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1988</div>, vol. <div class="volume">170</div> (pg. <div class="fpage">2031</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incorporation%20of%20LL-diaminopimelic%20acid%20into%20peptidoglycan%20of%20Escherichia%20coli%20mutants%20lacking%20diaminopimelate%20epimerase%20encoded%20by%20dapF&amp;author=D.%20Mengin-Lecreulx&amp;author=C.%20Michaud&amp;author=C.%20Richaud&amp;author=D.%20Blanot&amp;author=J.%20van%20Heijenoort&amp;publication_year=1988&amp;journal=J.%20Bacteriol.&amp;volume=170&amp;pages=2031-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3283102" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flardh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">van Wezel</div> <div class="given-names">G.P.</div></div>. </div><div class="article-title">Cell division during growth and development of <em>Streptomyces</em></div>, <div class="source">Recent Res. Devel. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">71</div>-<div class="lpage">90</div>)</div></div></div></div><div content-id="DSQ026C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kendrick</div> <div class="given-names">K.E.</div></div>. </div><div class="article-title">Visualization of penicillin-binding proteins during sporulation of <em>Streptomyces griseus</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">2125</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualization%20of%20penicillin-binding%20proteins%20during%20sporulation%20of%20Streptomyces%20griseus&amp;author=J.%20Hao&amp;author=K.E.%20Kendrick&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=2125-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9555895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noens</div> <div class="given-names">E.E.</div></div>,  <div class="name"><div class="surname">Mersinias</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Traag</div> <div class="given-names">B.A.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">C.P.</div></div>,  <div class="name"><div class="surname">Koerten</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">van Wezel</div> <div class="given-names">G.P.</div></div>. </div><div class="article-title">SsgA-like proteins determine the fate of peptidoglycan during sporulation of <em>Streptomyces coelicolor</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">929</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SsgA-like%20proteins%20determine%20the%20fate%20of%20peptidoglycan%20during%20sporulation%20of%20Streptomyces%20coelicolor&amp;author=E.E.%20Noens&amp;author=V.%20Mersinias&amp;author=B.A.%20Traag&amp;author=C.P.%20Smith&amp;author=H.K.%20Koerten&amp;author=G.P.%20van%20Wezel&amp;publication_year=2005&amp;journal=Mol.%20Microbiol.&amp;volume=58&amp;pages=929-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2005.04883.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2005.04883.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2005.04883.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16262781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Isolation of new actinomycete strains for the screening of new bioactive compounds</div>, <div class="source">J. Gen. Appl. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">141</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20new%20actinomycete%20strains%20for%20the%20screening%20of%20new%20bioactive%20compounds&amp;author=Y.%20Takahashi&amp;author=S.%20Omura&amp;publication_year=2003&amp;journal=J.%20Gen.%20Appl.%20Microbiol.&amp;volume=49&amp;pages=141-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2323/jgam.49.141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2323%2Fjgam.49.141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2323%2Fjgam.49.141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Otoguro</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishikiori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Oiwa</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Iwai</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Setamycin, a new antibiotic</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">1981</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1253</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Setamycin%2C%20a%20new%20antibiotic&amp;author=S.%20Omura&amp;author=K.%20Otoguro&amp;author=T.%20Nishikiori&amp;author=R.%20Oiwa&amp;author=Y.%20Iwai&amp;publication_year=1981&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=34&amp;pages=1253-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7164/antibiotics.34.1253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7164%2Fantibiotics.34.1253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7164%2Fantibiotics.34.1253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7309621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nett</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Moore</div> <div class="given-names">B.S.</div></div>. </div><div class="article-title">Genomic basis for natural product biosynthetic diversity in the actinomycetes</div>, <div class="source">Nat. Prod. Rep.</div> , <div class="year">2009</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1362</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20basis%20for%20natural%20product%20biosynthetic%20diversity%20in%20the%20actinomycetes&amp;author=M.%20Nett&amp;author=H.%20Ikeda&amp;author=B.S.%20Moore&amp;publication_year=2009&amp;journal=Nat.%20Prod.%20Rep.&amp;volume=26&amp;pages=1362-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/b817069j" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fb817069j" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fb817069j"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19844637" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Komatsu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Oikawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Identification and functional analysis of genes controlling biosynthesis of 2-methylisoborneol</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">7422</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20functional%20analysis%20of%20genes%20controlling%20biosynthesis%20of%202-methylisoborneol&amp;author=M.%20Komatsu&amp;author=M.%20Tsuda&amp;author=S.%20Omura&amp;author=H.%20Oikawa&amp;author=H.%20Ikeda&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=7422-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0802312105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0802312105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0802312105"> </span></div> </div></div></div></div></div><div content-id="DSQ026C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cane</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Geosmin biosynthesis in <em>Streptomyces avermitilis</em>. Molecular cloning, expression, and mechanistic study of the germacradienol/geosmin synthase</div>, <div class="source">J. Antibiot. (Tokyo)</div> , <div class="year">2006</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">471</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Geosmin%20biosynthesis%20in%20Streptomyces%20avermitilis.%20Molecular%20cloning%2C%20expression%2C%20and%20mechanistic%20study%20of%20the%20germacradienol%2Fgeosmin%20synthase&amp;author=D.E.%20Cane&amp;author=X.%20He&amp;author=S.%20Kobayashi&amp;author=S.%20Omura&amp;author=H.%20Ikeda&amp;publication_year=2006&amp;journal=J.%20Antibiot.%20(Tokyo)&amp;volume=59&amp;pages=471-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ja.2006.66" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fja.2006.66" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fja.2006.66"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17080683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Cane</div> <div class="given-names">D.E.</div></div>. </div><div class="article-title">Biosynthesis of the earthy odorant geosmin by a bifunctional <em>Streptomyces coelicolor</em> enzyme</div>, <div class="source">Nat. Chem. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">711</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20the%20earthy%20odorant%20geosmin%20by%20a%20bifunctional%20Streptomyces%20coelicolor%20enzyme&amp;author=J.%20Jiang&amp;author=X.%20He&amp;author=D.E.%20Cane&amp;publication_year=2007&amp;journal=Nat.%20Chem.%20Biol.&amp;volume=3&amp;pages=711-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nchembio.2007.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnchembio.2007.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnchembio.2007.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17873868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Laiple</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Pross</div> <div class="given-names">E.K.</div></div>, et al. </div><div class="article-title">Molecular analysis of the kirromycin biosynthetic gene cluster revealed beta-alanine as precursor of the pyridone moiety</div>, <div class="source">Chem. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">175</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20the%20kirromycin%20biosynthetic%20gene%20cluster%20revealed%20beta-alanine%20as%20precursor%20of%20the%20pyridone%20moiety&amp;author=T.%20Weber&amp;author=K.J.%20Laiple&amp;author=E.K.%20Pross&amp;publication_year=2008&amp;journal=Chem.%20Biol.&amp;volume=15&amp;pages=175-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chembiol.2007.12.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chembiol.2007.12.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chembiol.2007.12.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18291322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanabe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Funahashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Miyoshi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shinoda</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Identification and characterization of genes required for biosynthesis and transport of the siderophore vibrioferrin in <em>Vibrio parahaemolyticus</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">6938</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20genes%20required%20for%20biosynthesis%20and%20transport%20of%20the%20siderophore%20vibrioferrin%20in%20Vibrio%20parahaemolyticus&amp;author=T.%20Tanabe&amp;author=T.%20Funahashi&amp;author=H.%20Nakao&amp;author=S.%20Miyoshi&amp;author=S.%20Shinoda&amp;author=S.%20Yamamoto&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=6938-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.23.6938-6949.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.23.6938-6949.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.23.6938-6949.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14617658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lynch</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">O'Brien</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Welch</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genetic organization of the region encoding regulation, biosynthesis, and transport of rhizobactin 1021, a siderophore produced by <em>Sinorhizobium meliloti</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">2576</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20organization%20of%20the%20region%20encoding%20regulation%2C%20biosynthesis%2C%20and%20transport%20of%20rhizobactin%201021%2C%20a%20siderophore%20produced%20by%20Sinorhizobium%20meliloti&amp;author=D.%20Lynch&amp;author=J.%20O'Brien&amp;author=T.%20Welch&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=2576-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.8.2576-2585.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.8.2576-2585.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.8.2576-2585.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11274118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ026C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ026C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ueki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Takamatsu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Nocardamin production by <em>Streptomyces avermitilis</em></div>, <div class="source">Actinomycetologica</div> , <div class="year">2009</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">34</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nocardamin%20production%20by%20Streptomyces%20avermitilis&amp;author=M.%20Ueki&amp;author=R.%20Suzuki&amp;author=S.%20Takamatsu&amp;publication_year=2009&amp;journal=Actinomycetologica&amp;volume=23&amp;pages=34-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3209/saj.SAJ230203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3209%2Fsaj.SAJ230203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3209%2Fsaj.SAJ230203"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484773&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq026/2/dsq026-dsq026supp.pdf?Expires=2147483647&amp;Signature=S8-ZX7l5MohImSDsmOXvzN~JsvPBzfqdEMyKVJpcn8xOxpGvi-qL-h9cN5upBbZYV5C20ES0CdIYDLkMKASvYWsCCal5kBEApNgwVyHytxDzhNVhO-GT1fDWoBevr5AsZ0ImzJzPgHCXgySoIUw9JBry83tmeTR8F6QttcJDcrElZDDYProcAVGX~9xvwnOIHLybxIQf~7oNiBKp3TuLnbVW8--qHvYSZeEeevruUDIJoge-udrITmmHTMDJSFVdQUT5uQZ5tfuS3VlFhfIW1-k8Pdv7BseUZxdCnmnx~vUC~eeAupjB9SArWBZd5Z7KwXdFrcs4LtG0ZUdruKUDEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">373</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">39</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq026"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484773"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f6%2f393%2f484773">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000285800200006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (39)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F6%2F393%2F484773" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484773'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome Sequence of Kitasatospora setae NBRC 14216T: An Evolutionary Snapshot of the Family Streptomycetaceae" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq026");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f393%2f484773">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

