

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/14/4/169/466883","@type":"ScholarlyArticle","name":"Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes","datePublished":"2007-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/14/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2007-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsm018","keywords":["metagenomics","human gut microbiota","gene family","conjugative transposon"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/14/4/169/466883","author":[{"name":"Kurokawa, Ken","affiliation":"1  Laboratory of Comparative Genomics, Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama-cho, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"2  Life Science Research Group, Research Center for Advanced Science and Technology ,  Mitsubishi Research Institute ,  3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141 ,  Japan","@type":"Person"},{"name":"Kuwahara, Tomomi","affiliation":"3  Department of Molecular Bacteriology, Institute of Health Biosciences ,  The University of Tokushima Graduate School ,  3-18-5 Kuramoto-cho, Tokushima 770-8503 ,  Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"4  Kitasato Institute for Life Science ,  Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan","@type":"Person"},{"name":"Toh, Hidehiro","affiliation":"4  Kitasato Institute for Life Science ,  Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan    5  Center for Basic Research ,  Kitasato Institute ,  5-9-1 Shirokane, Minato-ku, Tokyo 108-8641 ,  Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Takami, Hideto","affiliation":"7  Extremophiles Research Program, Extremobiosphere Research Center ,  Japan Agency for Marine-Earth Science and Technology ,  2-15 Natsushima, Yokosuka, Kanagawa 237-0061 ,  Japan","@type":"Person"},{"name":"Morita, Hidetoshi","affiliation":"8  School of Veterinary Medicine ,  Azabu University ,  1-17-71 Fuchinobe, Sagamihara, Kanagawa 229-8501 ,  Japan","@type":"Person"},{"name":"Sharma, Vineet K.","affiliation":"6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Srivastava, Tulika P.","affiliation":"6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Taylor, Todd D.","affiliation":"6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Noguchi, Hideki","affiliation":"9  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Mori, Hiroshi","affiliation":"1  Laboratory of Comparative Genomics, Graduate School of Information Science ,  Nara Institute of Science and Technology ,  8916-5 Takayama-cho, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"10  Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Ehrlich, Dusko S.","affiliation":"11  Institut National de la Recherche Agronomique ,  Domaine de Vilvet, 8352 Jouy en Josas ,  France","@type":"Person"},{"name":"Itoh, Kikuji","affiliation":"12  Department of Veterinary Medicine, Graduate School of Agricultural and Life Sciences ,  The University of Tokyo ,  1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Takagi, Toshihisa","affiliation":"9  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Sakaki, Yoshiyuki","affiliation":"6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"10  Frontier Science Research Center ,  University of Miyazaki ,  5200 Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"4  Kitasato Institute for Life Science ,  Kitasato University ,  1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555 ,  Japan    6  RIKEN Genomic Sciences Center ,  1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045 ,  Japan    9  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"}],"description":"Abstract.  Numerous microbes inhabit the human intestine, many of which are uncharacterized or uncultivable. They form a complex microbial community that deeply","pageStart":"169","pageEnd":"181","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes" />
    <meta property="og:description" content="Abstract.  Numerous microbes inhabit the human intestine, many of which are uncharacterized or uncultivable. They form a complex microbial community that deeply" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/14/4/169/466883" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2007-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Life Science Research Group, Research Center for Advanced Science and Technology, Mitsubishi Research Institute, 3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141, Japan" /><meta name="citation_author" content="Kuwahara, Tomomi" /><meta name="citation_author_institution" content="Department of Molecular Bacteriology, Institute of Health Biosciences, The University of Tokushima Graduate School, 3-18-5 Kuramoto-cho, Tokushima 770-8503, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Kitasato Institute for Life Science, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author" content="Toh, Hidehiro" /><meta name="citation_author_institution" content="Kitasato Institute for Life Science, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="Center for Basic Research, Kitasato Institute, 5-9-1 Shirokane, Minato-ku, Tokyo 108-8641, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Takami, Hideto" /><meta name="citation_author_institution" content="Extremophiles Research Program, Extremobiosphere Research Center, Japan Agency for Marine-Earth Science and Technology, 2-15 Natsushima, Yokosuka, Kanagawa 237-0061, Japan" /><meta name="citation_author" content="Morita, Hidetoshi" /><meta name="citation_author_institution" content="School of Veterinary Medicine, Azabu University, 1-17-71 Fuchinobe, Sagamihara, Kanagawa 229-8501, Japan" /><meta name="citation_author" content="Sharma, Vineet K." /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Srivastava, Tulika P." /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Taylor, Todd D." /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Noguchi, Hideki" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Mori, Hiroshi" /><meta name="citation_author_institution" content="Laboratory of Comparative Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5 Takayama-cho, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Ehrlich, Dusko S." /><meta name="citation_author_institution" content="Institut National de la Recherche Agronomique, Domaine de Vilvet, 8352 Jouy en Josas, France" /><meta name="citation_author" content="Itoh, Kikuji" /><meta name="citation_author_institution" content="Department of Veterinary Medicine, Graduate School of Agricultural and Life Sciences, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Takagi, Toshihisa" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Sakaki, Yoshiyuki" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Frontier Science Research Center, University of Miyazaki, 5200 Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Kitasato Institute for Life Science, Kitasato University, 1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_title" content="Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes" /><meta name="citation_firstpage" content="169" /><meta name="citation_lastpage" content="181" /><meta name="citation_doi" content="10.1093/dnares/dsm018" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="14" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2007/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Host-bacterial mutualism in the human intestine; Science;  citation_year=2005;  citation_volume=307;  citation_pages=1915-1920; " /><meta name="citation_reference" content="citation_title=Ecological and evolutionary forces shaping microbial diversity in the human intestine; Cell;  citation_year=2006;  citation_volume=124;  citation_pages=837-848; " /><meta name="citation_reference" content="citation_title=Microbial biota of the human intestine: a tribute to some pioneering scientists; Curr. Issues. Intest. Microbiol;  citation_year=2001;  citation_volume=2;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Role of human microflora in health and disease; Eur J Clin Microbiol. Infect. Dis.;  citation_year=1992;  citation_volume=11;  citation_pages=1012-1015; " /><meta name="citation_reference" content="citation_title=Diversity of the human intestinal microbial flora; Science;  citation_year=2005;  citation_volume=308;  citation_pages=1635-1638; " /><meta name="citation_reference" content="citation_title=Intestinal microflora in early infancy: composition and development; Acta Paediatr.;  citation_year=2003;  citation_volume=91;  citation_pages=48-55; " /><meta name="citation_reference" content="citation_title=Dietary modulation of the human colonic microbiota: introducing the concept of prebiotics; J. Nutr.;  citation_year=1995;  citation_volume=125;  citation_pages=1401-1412; " /><meta name="citation_reference" content="citation_title=Gut flora in health and disease; Lancet;  citation_year=2003;  citation_volume=361;  citation_pages=512-519; " /><meta name="citation_reference" content="citation_title=Commensal host-bacterial relationships in the gut; Science;  citation_year=2001;  citation_volume=292;  citation_pages=1115-1118; " /><meta name="citation_reference" content="citation_title=The gut microbiota as an environmental factor that regulates fat storage; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=15718-15723; " /><meta name="citation_reference" content="citation_title=Recognition of commensal microflora by toll-like receptors is required for intestinal homeostasis; Cell;  citation_year=2004;  citation_volume=118;  citation_pages=229-241; " /><meta name="citation_reference" content="citation_title=A humanized gnotobiotic mouse model of host-archaeal-bacterial mutualism; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=10011-10016; " /><meta name="citation_reference" content="citation_title=Microbial ecology: human gut microbes associated with obesity; Nature;  citation_year=2006;  citation_volume=444;  citation_pages=1022-1023; " /><meta name="citation_reference" content="citation_title=Immunity, inflammation, and allergy in the gut; Science;  citation_year=2005;  citation_volume=307;  citation_pages=1920-1925; " /><meta name="citation_reference" content="citation_title=Reduced diversity of faecal microbiota in Crohn's disease revealed by a metagenomic approach; Gut;  citation_year=2006;  citation_volume=55;  citation_pages=205-211; " /><meta name="citation_reference" content="citation_title=Diet, anaerobic bacterial metabolism, and colon cancer: a review of the literature; J. Clin. Gastroenterol;  citation_year=2005;  citation_volume=39;  citation_pages=98-109; " /><meta name="citation_reference" content="citation_title=Review article: the role of bacteria in onset and perpetuation of inflammatory bowel disease; Aliment. Pharmacol. Ther.;  citation_year=2006;  citation_volume=24;  citation_pages=11-18; " /><meta name="citation_reference" content="citation_title=An obesity-associated gut microbiome with increased capacity for energy harvest; Nature;  citation_year=2006;  citation_volume=444;  citation_pages=1027-1031; " /><meta name="citation_reference" content="citation_title=Normal indigenous flora, In; citation_publisher=Academic Press, NY; Human Intestinal Microflora in Health and Disease;  citation_year=1983;  citation_pages=3-31; " /><meta name="citation_reference" content="citation_publisher=DNA sequencing of environmental samples, Metagenomics; Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=805-814; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis of the human distal gut microbiome; Science;  citation_year=2006;  citation_volume=312;  citation_pages=1355-1359; " /><meta name="citation_reference" content="citation_title=Comparison of fecal microflora of elderly persons in rural and urban areas of Japan; Appl. Environ. Microbiol.;  citation_year=1989;  citation_volume=55;  citation_pages=1100-1105; " /><meta name="citation_reference" content="citation_title=Molecular monitoring of succession of bacterial communities in human neonates; Appl. Environ. Microbiol.;  citation_year=2002;  citation_volume=68;  citation_pages=219-226; " /><meta name="citation_reference" content="citation_title=Age and disease related changes in intestinal bacterial populations assessed by cell culture, 16S RNA abundance, and community cellular fatty acid profiles; Gut;  citation_year=2001;  citation_volume=48;  citation_pages=198-205; " /><meta name="citation_reference" content="citation_title=Investigations into the influence of host genetics on the predominant eubacteria in the faecal microflora of children; J. Med. Microbiol.;  citation_year=2005;  citation_volume=54;  citation_pages=1239-1242; " /><meta name="citation_reference" content="citation_title=Environmental genome shotgun sequencing of the Sargasso Sea; Science;  citation_year=2004;  citation_volume=304;  citation_pages=66-74; " /><meta name="citation_reference" content="citation_title=A novel method for making nested deletions and its application for sequencing of a 300 kb region of human APP locus; Nuc. Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=1802-1808; " /><meta name="citation_reference" content="citation_title=PCAP: a whole-genome assembly program; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2164-2170; " /><meta name="citation_reference" content="citation_title=MetaGene: prokaryotic gene finding from environmental genome shotgun sequences; Nucl. Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=5623-5630; " /><meta name="citation_reference" content="citation_title=Characterization of transposon Tn1549, conferring VanB-type resistance in Enterococcus spp; Microbiology;  citation_year=2000;  citation_volume=146;  citation_issue=Pt 6;  citation_pages=1481-1489; " /><meta name="citation_reference" content="citation_title=The multidrug-resistant human pathogen Clostridium difficile has a highly mobile, mosaic genome; Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=779-786; " /><meta name="citation_reference" content="citation_title=Role of mobile DNA in the evolution of vancomycin-resistant Enterococcus faecalis; Science;  citation_year=2003;  citation_volume=299;  citation_pages=2071-2074; " /><meta name="citation_reference" content="citation_title=Molecular genetic anatomy of inter- and intraserotype variation in the human bacterial pathogen group A Streptococcus; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=7059-7064; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucl. Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Community genomics among stratified microbial assemblages in the ocean's interior; Science;  citation_year=2006;  citation_volume=311;  citation_pages=496-503; " /><meta name="citation_reference" content="citation_title=Measuring genome conservation across taxa: divided strains and united kingdoms; Nucl. Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=616-621; " /><meta name="citation_reference" content="citation_title=SHOT: a web server for the construction of genome phylogenies; Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=158-162; " /><meta name="citation_reference" content="citation_title=Open source clustering software; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1453-1454; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics of microbial communities; Science;  citation_year=2005;  citation_volume=308;  citation_pages=554-557; " /><meta name="citation_reference" content="citation_title=An efficient algorithm for large-scale detection of protein families; Nucl. Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=1575-1584; " /><meta name="citation_reference" content="citation_title=Pfam: clans, web tools and services; Nucl. Acids Res.;  citation_year=2006;  citation_volume=34;  citation_issue=Database Issue;  citation_pages=D247-D251; " /><meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers; Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1994;  citation_volume=2;  citation_pages=28-36; " /><meta name="citation_reference" content="citation_title=The co-evolution of host cationic antimicrobial peptides and microbial resistance; Nat. Rev. Microbiol.;  citation_year=2006;  citation_volume=4;  citation_pages=529-536; " /><meta name="citation_reference" content="citation_title=Food-derived mutagens and carcinogens; Cancer Res.;  citation_year=1992;  citation_volume=52;  citation_pages=2092s-2098s; " /><meta name="citation_reference" content="citation_title=A model of host-microbial interactions in an open mammalian ecosystem; Science;  citation_year=1996;  citation_volume=273;  citation_pages=1380-1383; " /><meta name="citation_reference" content="citation_title=Characterization of a Tn5382-like transposon containing the vanB2 gene cluster in a Clostridium strain isolated from human faeces; J. Antimicrob. Chemother.;  citation_year=2005;  citation_volume=55;  citation_pages=466-474; " /><meta name="citation_reference" content="citation_title=The Butyrivibrio fibrisolvens tet(W) gene is carried on the novel conjugative transposon TnB1230, which contains duplicated nitroreductase coding sequences; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=3656-3659; " /><meta name="citation_reference" content="citation_title=Transfer of vancomycin resistance transposon Tn1549 from Clostridium symbiosum to Enterococcus spp. in the gut of gnotobiotic mice; Antimicrob. Agents. Chemother.;  citation_year=2006;  citation_volume=50;  citation_pages=1054-1062; " /><meta name="citation_reference" content="citation_title=Evidence for a chromosome-borne resistance transposon (Tn916) in Streptococcus faecalis that is capable of “conjugal” transfer in the absence of a conjugative plasmid; J. Bacteriol;  citation_year=1981;  citation_volume=145;  citation_pages=494-502; " /><meta name="citation_reference" content="citation_title=Biodiversity and ecosystem productivity in a fluctuating environment: the insurance hypothesis; Proc. Natl. Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=1463-1468; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/14/4/169/17462688/dsm018.pdf" /><meta name="description" content="Abstract.  Numerous microbes inhabit the human intestine, many of which are uncharacterized or uncultivable. They form a complex microbial community that deeply" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/14/4/169/466883" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/14/4/169/466883" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/14/4/169/466883");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsm018';
    </script>
<title>
	Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./466883" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfKJATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgQ29tcGFyYXRpdmUgR2Vub21pY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBJbmZvcm1hdGlvbiBTY2llbmNlLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUgVGFrYXlhbWEtY2hvLCBJa29tYSwgTmFyYSA2MzAtMDE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG9oLCBUYWtlaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxpZmUgU2NpZW5jZSBSZXNlYXJjaCBHcm91cCwgUmVzZWFyY2ggQ2VudGVyIGZvciBBZHZhbmNlZCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBNaXRzdWJpc2hpIFJlc2VhcmNoIEluc3RpdHV0ZSwgMy02IE90ZW1hY2hpIDItY2hvbWUgQ2hpeW9kYS1rdSwgVG9reW8gMTAwLTgxNDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3V3YWhhcmEsIFRvbW9taSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJhY3RlcmlvbG9neSwgSW5zdGl0dXRlIG9mIEhlYWx0aCBCaW9zY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9rdXNoaW1hIEdyYWR1YXRlIFNjaG9vbCwgMy0xOC01IEt1cmFtb3RvLWNobywgVG9rdXNoaW1hIDc3MC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9zaGltYSwgS2Vuc2hpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLaXRhc2F0byBJbnN0aXR1dGUgZm9yIExpZmUgU2NpZW5jZSwgS2l0YXNhdG8gVW5pdmVyc2l0eSwgMS0xNS0xIEtpdGFzYXRvLCBTYWdhbWloYXJhLCBLYW5hZ2F3YSAyMjgtODU1NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb2gsIEhpZGVoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2UsIEtpdGFzYXRvIFVuaXZlcnNpdHksIDEtMTUtMSBLaXRhc2F0bywgU2FnYW1paGFyYSwgS2FuYWdhd2EgMjI4LTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBCYXNpYyBSZXNlYXJjaCwgS2l0YXNhdG8gSW5zdGl0dXRlLCA1LTktMSBTaGlyb2thbmUsIE1pbmF0by1rdSwgVG9reW8gMTA4LTg2NDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG95b2RhLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRha2FtaSwgSGlkZXRvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRXh0cmVtb3BoaWxlcyBSZXNlYXJjaCBQcm9ncmFtLCBFeHRyZW1vYmlvc3BoZXJlIFJlc2VhcmNoIENlbnRlciwgSmFwYW4gQWdlbmN5IGZvciBNYXJpbmUtRWFydGggU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgMi0xNSBOYXRzdXNoaW1hLCBZb2tvc3VrYSwgS2FuYWdhd2EgMjM3LTAwNjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yaXRhLCBIaWRldG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgVmV0ZXJpbmFyeSBNZWRpY2luZSwgQXphYnUgVW5pdmVyc2l0eSwgMS0xNy03MSBGdWNoaW5vYmUsIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOS04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoYXJtYSwgVmluZWV0IEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUklLRU4gR2Vub21pYyBTY2llbmNlcyBDZW50ZXIsIDEtNy0yMiBTdWVoaXJvLWNobywgVHN1cnVtaS1rdSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNyaXZhc3RhdmEsIFR1bGlrYSBQLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJJS0VOIEdlbm9taWMgU2NpZW5jZXMgQ2VudGVyLCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYXlsb3IsIFRvZGQgRC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBHZW5vbWljIFNjaWVuY2VzIENlbnRlciwgMS03LTIyIFN1ZWhpcm8tY2hvLCBUc3VydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTm9ndWNoaSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2hpd2Fub2hhLCBLYXNoaXdhLCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JpLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBDb21wYXJhdGl2ZSBHZW5vbWljcywgR3JhZHVhdGUgU2Nob29sIG9mIEluZm9ybWF0aW9uIFNjaWVuY2UsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYS1jaG8sIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ndXJhLCBZb3NoaXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnJvbnRpZXIgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDUyMDAgS2l5b3Rha2UsIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkVocmxpY2gsIER1c2tvIFMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXQgTmF0aW9uYWwgZGUgbGEgUmVjaGVyY2hlIEFncm9ub21pcXVlLCBEb21haW5lIGRlIFZpbHZldCwgODM1MiBKb3V5IGVuIEpvc2FzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG9oLCBLaWt1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFZldGVyaW5hcnkgTWVkaWNpbmUsIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgYW5kIExpZmUgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCAxLTEtMSBZYXlvaSwgQnVua3lvLWt1LCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthZ2ksIFRvc2hpaGlzYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCA1LTEtNSBLYXNoaXdhbm9oYSwgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FrYWtpLCBZb3NoaXl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBHZW5vbWljIFNjaWVuY2VzIENlbnRlciwgMS03LTIyIFN1ZWhpcm8tY2hvLCBUc3VydW1pLWt1LCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF5YXNoaSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCA1MjAwIEtpeW90YWtlLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXR0b3JpLCBNYXNhaGlyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktpdGFzYXRvIEluc3RpdHV0ZSBmb3IgTGlmZSBTY2llbmNlLCBLaXRhc2F0byBVbml2ZXJzaXR5LCAxLTE1LTEgS2l0YXNhdG8sIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJJS0VOIEdlbm9taWMgU2NpZW5jZXMgQ2VudGVyLCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29tcGFyYXRpdmUgTWV0YWdlbm9taWNzIFJldmVhbGVkIENvbW1vbmx5IEVucmljaGVkIEdlbmUgU2V0cyBpbiBIdW1hbiBHdXQgTWljcm9iaW9tZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE4MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbTAxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA3LzAxLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9zdC1iYWN0ZXJpYWwgbXV0dWFsaXNtIGluIHRoZSBodW1hbiBpbnRlc3RpbmU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA3OyAgY2l0YXRpb25fcGFnZXM9MTkxNS0xOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FY29sb2dpY2FsIGFuZCBldm9sdXRpb25hcnkgZm9yY2VzIHNoYXBpbmcgbWljcm9iaWFsIGRpdmVyc2l0eSBpbiB0aGUgaHVtYW4gaW50ZXN0aW5lOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTgzNy04NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBiaW90YSBvZiB0aGUgaHVtYW4gaW50ZXN0aW5lOiBhIHRyaWJ1dGUgdG8gc29tZSBwaW9uZWVyaW5nIHNjaWVudGlzdHM7IEN1cnIuIElzc3Vlcy4gSW50ZXN0LiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgaHVtYW4gbWljcm9mbG9yYSBpbiBoZWFsdGggYW5kIGRpc2Vhc2U7IEV1ciBKIENsaW4gTWljcm9iaW9sLiBJbmZlY3QuIERpcy47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDEyLTEwMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBvZiB0aGUgaHVtYW4gaW50ZXN0aW5hbCBtaWNyb2JpYWwgZmxvcmE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA4OyAgY2l0YXRpb25fcGFnZXM9MTYzNS0xNjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlc3RpbmFsIG1pY3JvZmxvcmEgaW4gZWFybHkgaW5mYW5jeTogY29tcG9zaXRpb24gYW5kIGRldmVsb3BtZW50OyBBY3RhIFBhZWRpYXRyLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTQ4LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWV0YXJ5IG1vZHVsYXRpb24gb2YgdGhlIGh1bWFuIGNvbG9uaWMgbWljcm9iaW90YTogaW50cm9kdWNpbmcgdGhlIGNvbmNlcHQgb2YgcHJlYmlvdGljczsgSi4gTnV0ci47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTI1OyAgY2l0YXRpb25fcGFnZXM9MTQwMS0xNDEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HdXQgZmxvcmEgaW4gaGVhbHRoIGFuZCBkaXNlYXNlOyBMYW5jZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzYxOyAgY2l0YXRpb25fcGFnZXM9NTEyLTUxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbWVuc2FsIGhvc3QtYmFjdGVyaWFsIHJlbGF0aW9uc2hpcHMgaW4gdGhlIGd1dDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTI7ICBjaXRhdGlvbl9wYWdlcz0xMTE1LTExMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBndXQgbWljcm9iaW90YSBhcyBhbiBlbnZpcm9ubWVudGFsIGZhY3RvciB0aGF0IHJlZ3VsYXRlcyBmYXQgc3RvcmFnZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTU3MTgtMTU3MjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29nbml0aW9uIG9mIGNvbW1lbnNhbCBtaWNyb2Zsb3JhIGJ5IHRvbGwtbGlrZSByZWNlcHRvcnMgaXMgcmVxdWlyZWQgZm9yIGludGVzdGluYWwgaG9tZW9zdGFzaXM7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MjI5LTI0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBodW1hbml6ZWQgZ25vdG9iaW90aWMgbW91c2UgbW9kZWwgb2YgaG9zdC1hcmNoYWVhbC1iYWN0ZXJpYWwgbXV0dWFsaXNtOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0xMDAxMS0xMDAxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGVjb2xvZ3k6IGh1bWFuIGd1dCBtaWNyb2JlcyBhc3NvY2lhdGVkIHdpdGggb2Jlc2l0eTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0NDsgIGNpdGF0aW9uX3BhZ2VzPTEwMjItMTAyMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1tdW5pdHksIGluZmxhbW1hdGlvbiwgYW5kIGFsbGVyZ3kgaW4gdGhlIGd1dDsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDc7ICBjaXRhdGlvbl9wYWdlcz0xOTIwLTE5MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZHVjZWQgZGl2ZXJzaXR5IG9mIGZhZWNhbCBtaWNyb2Jpb3RhIGluIENyb2huJ3MgZGlzZWFzZSByZXZlYWxlZCBieSBhIG1ldGFnZW5vbWljIGFwcHJvYWNoOyBHdXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0yMDUtMjExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWV0LCBhbmFlcm9iaWMgYmFjdGVyaWFsIG1ldGFib2xpc20sIGFuZCBjb2xvbiBjYW5jZXI6IGEgcmV2aWV3IG9mIHRoZSBsaXRlcmF0dXJlOyBKLiBDbGluLiBHYXN0cm9lbnRlcm9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9OTgtMTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZpZXcgYXJ0aWNsZTogdGhlIHJvbGUgb2YgYmFjdGVyaWEgaW4gb25zZXQgYW5kIHBlcnBldHVhdGlvbiBvZiBpbmZsYW1tYXRvcnkgYm93ZWwgZGlzZWFzZTsgQWxpbWVudC4gUGhhcm1hY29sLiBUaGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTExLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBvYmVzaXR5LWFzc29jaWF0ZWQgZ3V0IG1pY3JvYmlvbWUgd2l0aCBpbmNyZWFzZWQgY2FwYWNpdHkgZm9yIGVuZXJneSBoYXJ2ZXN0OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDQ0OyAgY2l0YXRpb25fcGFnZXM9MTAyNy0xMDMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3JtYWwgaW5kaWdlbm91cyBmbG9yYSwgSW47IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgTlk7IEh1bWFuIEludGVzdGluYWwgTWljcm9mbG9yYSBpbiBIZWFsdGggYW5kIERpc2Vhc2U7ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl9wYWdlcz0zLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RE5BIHNlcXVlbmNpbmcgb2YgZW52aXJvbm1lbnRhbCBzYW1wbGVzLCBNZXRhZ2Vub21pY3M7IE5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTgwNS04MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBkaXN0YWwgZ3V0IG1pY3JvYmlvbWU7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzEyOyAgY2l0YXRpb25fcGFnZXM9MTM1NS0xMzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29uIG9mIGZlY2FsIG1pY3JvZmxvcmEgb2YgZWxkZXJseSBwZXJzb25zIGluIHJ1cmFsIGFuZCB1cmJhbiBhcmVhcyBvZiBKYXBhbjsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTExMDAtMTEwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIG1vbml0b3Jpbmcgb2Ygc3VjY2Vzc2lvbiBvZiBiYWN0ZXJpYWwgY29tbXVuaXRpZXMgaW4gaHVtYW4gbmVvbmF0ZXM7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0yMTktMjI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ2UgYW5kIGRpc2Vhc2UgcmVsYXRlZCBjaGFuZ2VzIGluIGludGVzdGluYWwgYmFjdGVyaWFsIHBvcHVsYXRpb25zIGFzc2Vzc2VkIGJ5IGNlbGwgY3VsdHVyZSwgMTZTIFJOQSBhYnVuZGFuY2UsIGFuZCBjb21tdW5pdHkgY2VsbHVsYXIgZmF0dHkgYWNpZCBwcm9maWxlczsgR3V0OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MTk4LTIwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXN0aWdhdGlvbnMgaW50byB0aGUgaW5mbHVlbmNlIG9mIGhvc3QgZ2VuZXRpY3Mgb24gdGhlIHByZWRvbWluYW50IGV1YmFjdGVyaWEgaW4gdGhlIGZhZWNhbCBtaWNyb2Zsb3JhIG9mIGNoaWxkcmVuOyBKLiBNZWQuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjM5LTEyNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVudmlyb25tZW50YWwgZ2Vub21lIHNob3RndW4gc2VxdWVuY2luZyBvZiB0aGUgU2FyZ2Fzc28gU2VhOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwNDsgIGNpdGF0aW9uX3BhZ2VzPTY2LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIG1ldGhvZCBmb3IgbWFraW5nIG5lc3RlZCBkZWxldGlvbnMgYW5kIGl0cyBhcHBsaWNhdGlvbiBmb3Igc2VxdWVuY2luZyBvZiBhIDMwMCBrYiByZWdpb24gb2YgaHVtYW4gQVBQIGxvY3VzOyBOdWMuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xODAyLTE4MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBDQVA6IGEgd2hvbGUtZ2Vub21lIGFzc2VtYmx5IHByb2dyYW07IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjE2NC0yMTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhR2VuZTogcHJva2FyeW90aWMgZ2VuZSBmaW5kaW5nIGZyb20gZW52aXJvbm1lbnRhbCBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jZXM7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz01NjIzLTU2MzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdHJhbnNwb3NvbiBUbjE1NDksIGNvbmZlcnJpbmcgVmFuQi10eXBlIHJlc2lzdGFuY2UgaW4gRW50ZXJvY29jY3VzIHNwcDsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE0NjsgIGNpdGF0aW9uX2lzc3VlPVB0IDY7ICBjaXRhdGlvbl9wYWdlcz0xNDgxLTE0ODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtdWx0aWRydWctcmVzaXN0YW50IGh1bWFuIHBhdGhvZ2VuIENsb3N0cmlkaXVtIGRpZmZpY2lsZSBoYXMgYSBoaWdobHkgbW9iaWxlLCBtb3NhaWMgZ2Vub21lOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTc3OS03ODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgbW9iaWxlIEROQSBpbiB0aGUgZXZvbHV0aW9uIG9mIHZhbmNvbXljaW4tcmVzaXN0YW50IEVudGVyb2NvY2N1cyBmYWVjYWxpczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTk7ICBjaXRhdGlvbl9wYWdlcz0yMDcxLTIwNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBnZW5ldGljIGFuYXRvbXkgb2YgaW50ZXItIGFuZCBpbnRyYXNlcm90eXBlIHZhcmlhdGlvbiBpbiB0aGUgaHVtYW4gYmFjdGVyaWFsIHBhdGhvZ2VuIGdyb3VwIEEgU3RyZXB0b2NvY2N1czsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NzA1OS03MDY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktMzQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbXVuaXR5IGdlbm9taWNzIGFtb25nIHN0cmF0aWZpZWQgbWljcm9iaWFsIGFzc2VtYmxhZ2VzIGluIHRoZSBvY2VhbidzIGludGVyaW9yOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ni01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lYXN1cmluZyBnZW5vbWUgY29uc2VydmF0aW9uIGFjcm9zcyB0YXhhOiBkaXZpZGVkIHN0cmFpbnMgYW5kIHVuaXRlZCBraW5nZG9tczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTYxNi02MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNIT1Q6IGEgd2ViIHNlcnZlciBmb3IgdGhlIGNvbnN0cnVjdGlvbiBvZiBnZW5vbWUgcGh5bG9nZW5pZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNTgtMTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcGVuIHNvdXJjZSBjbHVzdGVyaW5nIHNvZnR3YXJlOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE0NTMtMTQ1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWV0YWdlbm9taWNzIG9mIG1pY3JvYmlhbCBjb21tdW5pdGllczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDg7ICBjaXRhdGlvbl9wYWdlcz01NTQtNTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBlZmZpY2llbnQgYWxnb3JpdGhtIGZvciBsYXJnZS1zY2FsZSBkZXRlY3Rpb24gb2YgcHJvdGVpbiBmYW1pbGllczsgTnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTU3NS0xNTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZmFtOiBjbGFucywgd2ViIHRvb2xzIGFuZCBzZXJ2aWNlczsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX2lzc3VlPURhdGFiYXNlIElzc3VlOyAgY2l0YXRpb25fcGFnZXM9RDI0Ny1EMjUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaXR0aW5nIGEgbWl4dHVyZSBtb2RlbCBieSBleHBlY3RhdGlvbiBtYXhpbWl6YXRpb24gdG8gZGlzY292ZXIgbW90aWZzIGluIGJpb3BvbHltZXJzOyBQcm9jLiBJbnQuIENvbmYuIEludGVsbC4gU3lzdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MjgtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjby1ldm9sdXRpb24gb2YgaG9zdCBjYXRpb25pYyBhbnRpbWljcm9iaWFsIHBlcHRpZGVzIGFuZCBtaWNyb2JpYWwgcmVzaXN0YW5jZTsgTmF0LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTUyOS01MzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvb2QtZGVyaXZlZCBtdXRhZ2VucyBhbmQgY2FyY2lub2dlbnM7IENhbmNlciBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9MjA5MnMtMjA5OHM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9kZWwgb2YgaG9zdC1taWNyb2JpYWwgaW50ZXJhY3Rpb25zIGluIGFuIG9wZW4gbWFtbWFsaWFuIGVjb3N5c3RlbTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzM7ICBjaXRhdGlvbl9wYWdlcz0xMzgwLTEzODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBUbjUzODItbGlrZSB0cmFuc3Bvc29uIGNvbnRhaW5pbmcgdGhlIHZhbkIyIGdlbmUgY2x1c3RlciBpbiBhIENsb3N0cmlkaXVtIHN0cmFpbiBpc29sYXRlZCBmcm9tIGh1bWFuIGZhZWNlczsgSi4gQW50aW1pY3JvYi4gQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NTsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ni00NzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCdXR5cml2aWJyaW8gZmlicmlzb2x2ZW5zIHRldChXKSBnZW5lIGlzIGNhcnJpZWQgb24gdGhlIG5vdmVsIGNvbmp1Z2F0aXZlIHRyYW5zcG9zb24gVG5CMTIzMCwgd2hpY2ggY29udGFpbnMgZHVwbGljYXRlZCBuaXRyb3JlZHVjdGFzZSBjb2Rpbmcgc2VxdWVuY2VzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTM2NTYtMzY1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXIgb2YgdmFuY29teWNpbiByZXNpc3RhbmNlIHRyYW5zcG9zb24gVG4xNTQ5IGZyb20gQ2xvc3RyaWRpdW0gc3ltYmlvc3VtIHRvIEVudGVyb2NvY2N1cyBzcHAuIGluIHRoZSBndXQgb2YgZ25vdG9iaW90aWMgbWljZTsgQW50aW1pY3JvYi4gQWdlbnRzLiBDaGVtb3RoZXIuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MTA1NC0xMDYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgYSBjaHJvbW9zb21lLWJvcm5lIHJlc2lzdGFuY2UgdHJhbnNwb3NvbiAoVG45MTYpIGluIFN0cmVwdG9jb2NjdXMgZmFlY2FsaXMgdGhhdCBpcyBjYXBhYmxlIG9mIOKAnGNvbmp1Z2Fs4oCdIHRyYW5zZmVyIGluIHRoZSBhYnNlbmNlIG9mIGEgY29uanVnYXRpdmUgcGxhc21pZDsgSi4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE0NTsgIGNpdGF0aW9uX3BhZ2VzPTQ5NC01MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2RpdmVyc2l0eSBhbmQgZWNvc3lzdGVtIHByb2R1Y3Rpdml0eSBpbiBhIGZsdWN0dWF0aW5nIGVudmlyb25tZW50OiB0aGUgaW5zdXJhbmNlIGh5cG90aGVzaXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9MTQ2My0xNDY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTQvNC8xNjkvMTc0NjI2ODgvZHNtMDE4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBOdW1lcm91cyBtaWNyb2JlcyBpbmhhYml0IHRoZSBodW1hbiBpbnRlc3RpbmUsIG1hbnkgb2Ygd2hpY2ggYXJlIHVuY2hhcmFjdGVyaXplZCBvciB1bmN1bHRpdmFibGUuIFRoZXkgZm9ybSBhIGNvbXBsZXggbWljcm9iaWFsIGNvbW11bml0eSB0aGF0IGRlZXBseSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY2ODgzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZBK+Ns+S7YFyFEoT/YUlj2kBzDXi" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f169%2f466883" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f169%2f466883" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18275" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/14/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/14/4/1/m_cover.gif?Expires=2147483647&amp;Signature=VQq3AbM7~E~J8C7iOLHK~5VCdppwq9PRDTjb1bvRYHWOf31q6JSg2JeA2AND8jpTXS3uun0O3ycR1MjA5ULlGRJxJt0EJ9aXJvqUCaIp78MGSA6jQLuQYnndUTfRjR~dpkt6HXe8sJrEGDP9qsl2v6etZByNmWx8TbR3gMCwqIG8C12K4IHVUKooevWug0tJTan8mnIzI5fnc-fdGf2FLa5ttkajY2s3QiSPd3Nrp-ovZdKQq-ABe8o36FDrtDjtIG-pp9aJEak~jJ4vn5ZtDzSxF~SJtEoVtqHLbFn6CdRP146b2bNAWdmcguu-GvCIEn4Nk26GuGWfv2r4RlAJOg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/14/4">
                        <div class="volume trailing-comma">Volume 14</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2007        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195783">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195783">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195785">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195785">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195789">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195789">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195812">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195812">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195855">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195855">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195857">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195857">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195859">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195859">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/14/4/155/466553">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/14/4/183/466388">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Comparative Genomics, Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama-cho, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Life Science Research Group, Research Center for Advanced Science and Technology</div>, <div class="insititution">Mitsubishi Research Institute</div>, <div class="addr-line">3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomomi Kuwahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Molecular Bacteriology, Institute of Health Biosciences</div>, <div class="insititution">The University of Tokushima Graduate School</div>, <div class="addr-line">3-18-5 Kuramoto-cho, Tokushima 770-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomomi+Kuwahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuwahara T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuwahara T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomomi Kuwahara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Kitasato Institute for Life Science</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hidehiro Toh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Kitasato Institute for Life Science</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Center for Basic Research</div>, <div class="insititution">Kitasato Institute</div>, <div class="addr-line">5-9-1 Shirokane, Minato-ku, Tokyo 108-8641</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hidehiro+Toh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toh H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toh H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hidehiro Toh</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideto Takami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Extremophiles Research Program, Extremobiosphere Research Center</div>, <div class="insititution">Japan Agency for Marine-Earth Science and Technology</div>, <div class="addr-line">2-15 Natsushima, Yokosuka, Kanagawa 237-0061</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideto+Takami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takami H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideto Takami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hidetoshi Morita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">School of Veterinary Medicine</div>, <div class="insititution">Azabu University</div>, <div class="addr-line">1-17-71 Fuchinobe, Sagamihara, Kanagawa 229-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hidetoshi+Morita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morita H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morita H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hidetoshi Morita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vineet K. Sharma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vineet+K.+Sharma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sharma V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sharma V&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Vineet K. Sharma</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tulika P. Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tulika+P.+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srivastava T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tulika P. Srivastava</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Todd D. Taylor</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Todd+D.+Taylor">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Taylor T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Taylor T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Todd D. Taylor</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Noguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Noguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Noguchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Noguchi H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hideki Noguchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Comparative Genomics, Graduate School of Information Science</div>, <div class="insititution">Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama-cho, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiroshi Mori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="addr-line">Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dusko S. Ehrlich</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">11</div><div class="insititution">Institut National de la Recherche Agronomique</div>, <div class="addr-line">Domaine de Vilvet, 8352 Jouy en Josas</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dusko+S.+Ehrlich">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ehrlich D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ehrlich D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Dusko S. Ehrlich</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kikuji Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">12</div><div class="addr-line">Department of Veterinary Medicine, Graduate School of Agricultural and Life Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kikuji+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kikuji Itoh</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshihisa Takagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshihisa+Takagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takagi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takagi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Toshihisa Takagi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Sakaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Sakaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sakaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sakaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshiyuki Sakaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="addr-line">Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">5200 Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Masahira Hattori. Tel. +81 7136-4070. Fax. <div class="fax">+81 4-7136-4084</div>. E-mail: <a href="mailto:hattori@k.u-tokyo.ac.jp" target="_blank">hattori@k.u-tokyo.ac.jp</a>; or Tetsuya Hayashi. Tel. +81 985-85-0871. Fax. <div class="fax">+81 985-85-6475</div>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Kitasato Institute for Life Science</div>, <div class="insititution">Kitasato University</div>, <div class="addr-line">1-15-1 Kitasato, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="insititution">RIKEN Genomic Sciences Center</div>, <div class="addr-line">1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama, Kanagawa 230-0045</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">9</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Masahira Hattori. Tel. +81 7136-4070. Fax. <div class="fax">+81 4-7136-4084</div>. E-mail: <a href="mailto:hattori@k.u-tokyo.ac.jp" target="_blank">hattori@k.u-tokyo.ac.jp</a>; or Tetsuya Hayashi. Tel. +81 985-85-0871. Fax. <div class="fax">+81 985-85-6475</div>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 169–181, <a href='https://doi.org/10.1093/dnares/dsm018'>https://doi.org/10.1093/dnares/dsm018</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2007</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 June 2007</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 July 2007</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="466883" href="/dnaresearch/article-pdf/14/4/169/17462688/dsm018.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ken Kurokawa, Takehiko Itoh, Tomomi Kuwahara, Kenshiro Oshima, Hidehiro Toh, Atsushi Toyoda, Hideto Takami, Hidetoshi Morita, Vineet K. Sharma, Tulika P. Srivastava, Todd D. Taylor, Hideki Noguchi, Hiroshi Mori, Yoshitoshi Ogura, Dusko S. Ehrlich, Kikuji Itoh, Toshihisa Takagi, Yoshiyuki Sakaki, Tetsuya Hayashi, Masahira Hattori;  Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes, <em>DNA Research</em>, Volume 14, Issue 4, 1 January 2007, Pages 169–181, <a href='https://doi.org/10.1093/dnares/dsm018'>https://doi.org/10.1093/dnares/dsm018</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/466883?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/466883?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparative%20Metagenomics%20Revealed%20Commonly%20Enriched%20Gene%20Sets%20in%20Human%20Gut%20Microbiomes&amp;publicationDate=2007-10-03&amp;volumeNum=14&amp;issueNum=4&amp;author=Kurokawa%2C%20Ken%3B%20Itoh%2C%20Takehiko&amp;startPage=169&amp;endPage=181&amp;contentId=10.1093%2Fdnares%2Fdsm018&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f169%2f466883" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195783 id="87195783" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Numerous microbes inhabit the human intestine, many of which are uncharacterized or uncultivable. They form a complex microbial community that deeply affects human physiology. To identify the genomic features common to all human gut microbiomes as well as those variable among them, we performed a large-scale comparative metagenomic analysis of fecal samples from 13 healthy individuals of various ages, including unweaned infants. We found that, while the gut microbiota from unweaned infants were simple and showed a high inter-individual variation in taxonomic and gene composition, those from adults and weaned children were more complex but showed a high functional uniformity regardless of age or sex. In searching for the genes over-represented in gut microbiomes, we identified 237 gene families commonly enriched in adult-type and 136 families in infant-type microbiomes, with a small overlap. An analysis of their predicted functions revealed various strategies employed by each type of microbiota to adapt to its intestinal environment, suggesting that these gene sets encode the core functions of adult and infant-type gut microbiota. By analysing the orphan genes, 647 new gene families were identified to be exclusively present in human intestinal microbiomes. In addition, we discovered a conjugative transposon family explosively amplified in human gut microbiomes, which strongly suggests that the intestine is a ‘hot spot’ for horizontal gene transfer between microbes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenomics">metagenomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;human gut microbiota&quot;">human gut microbiota</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene family&quot;">gene family</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;conjugative transposon&quot;">conjugative transposon</a></div>                            <h2 scrollto-destination=87195785 id="87195785" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>All surfaces of the human body are inhabited by complex microbial communities (microbiota).<sup><a reveal-id="DSM018C1  DSM018C4" data-open="DSM018C1  DSM018C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> In adults, the combined microbial populations exceed 100 trillion cells, about 10 times the total number of cells composing the human body. Most reside in the intestinal tract, and in particular in the distal colon where cell densities are as high as 10<sup>12</sup> g<sup>−1</sup> of human feces. The members of the human intestinal microbiota are classified into more than 50 genera and hundreds of species representing nine bacterial and one archaeal division.<sup><a reveal-id="DSM018C1" data-open="DSM018C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM018C5  DSM018C7" data-open="DSM018C5  DSM018C7" class="link link-ref link-reveal xref-bibr">5–7</a></sup> The total number of genes encoded by their collective genomes (microbiome) is estimated to exceed that of the human genome by at least one order of magnitude.<sup><a reveal-id="DSM018C1" data-open="DSM018C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Our gut microbiota possesses many metabolic capabilities which are lacking in the human host and, thus, can be viewed as indispensable for human life.<sup><a reveal-id="DSM018C3" data-open="DSM018C3" class="link link-ref link-reveal xref-bibr">3</a></sup> It contributes to host nutrition by enhancing the efficacy of energy harvest from ingested food and by synthesizing essential vitamins.<sup><a reveal-id="DSM018C1" data-open="DSM018C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM018C8" data-open="DSM018C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSM018C9" data-open="DSM018C9" class="link link-ref link-reveal xref-bibr">9</a></sup> It also affects a broad range of physiological properties of the human host, controlling, for instance, intestinal epithelial cell proliferation and differentiation, energy balance, pH, the development of the immune system, and protection against pathogens.<sup><a reveal-id="DSM018C1" data-open="DSM018C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSM018C10  DSM018C12" data-open="DSM018C10  DSM018C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> Imbalance of the intestinal microbiota can predispose individuals to a variety of disease states ranging from inflammatory bowel diseases to allergy and obesity.<sup><a reveal-id="DSM018C13  DSM018C18" data-open="DSM018C13  DSM018C18" class="link link-ref link-reveal xref-bibr">13–18</a></sup></p><p>The composition, dynamics, and functions of the human intestinal microbiota have been studied mostly using culture-based approaches and analyses of 16S ribosomal RNA sequences.<sup><a reveal-id="DSM018C3" data-open="DSM018C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSM018C5" data-open="DSM018C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSM018C13" data-open="DSM018C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSM018C19" data-open="DSM018C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Notwithstanding these efforts, our understanding of this microbial community is still very limited, particularly with regard to the overall gene content, because of its high complexity and our inability to cultivate most of the microbial species residing in the gut. For instance, although it has been established that the microbiota of adults and unweaned infants differ in composition,<sup><a reveal-id="DSM018C6" data-open="DSM018C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSM018C7" data-open="DSM018C7" class="link link-ref link-reveal xref-bibr">7</a></sup> it is largely unknown how such compositional differences affect the overall gene contents and functional properties.</p><p>To explore the genomic features of complex microbial communities including uncultivable microbes, a culture-independent metagenomic approach is practical.<sup><a reveal-id="DSM018C20" data-open="DSM018C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In a pioneering study, it has been recently applied to fecal samples from two adult Americans.<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> However, the data obtained from only two adults are insufficient for understanding the structure and functional capabilities of gut microbial communities, as they may be largely affected by various intrinsic and environmental factors, such as age, diet, and host genotype.<sup><a reveal-id="DSM018C7" data-open="DSM018C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSM018C16" data-open="DSM018C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSM018C22  DSM018C25" data-open="DSM018C22  DSM018C25" class="link link-ref link-reveal xref-bibr">22–25</a></sup> Here, we extended the study to 13 healthy Japanese individuals, including adults, weaned children, and unweaned infants. The analysis not only indicated the presence of gene sets commonly enriched in human gut microbiomes, but also revealed intriguing differences of microbiomes between adults and unweaned infants and between American and Japanese individuals.</p>                            <h2 scrollto-destination=87195789 id="87195789" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195790 id="87195790" class="section-title" data-legacy-id=s2a>2.1. Subjects</h3>
<p>All the subjects were healthy Japanese individuals. The ages, genders, and familial relationships of all subjects are summarized in Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>. All subjects or their parents were informed of the purpose and protocol of this study. None of the subjects were given dietary restrictions except for antibiotics, probiotics, fermented foods (fermented beans, yogurt, etc.), and well-known functional foods for at least 4 weeks prior to sampling. None had a history of gastrointestinal disorder at the time of sampling, and none had unusual eating behaviors.</p>                        <a id="87195792" scrollto-destination="87195792"></a>
<div content-id="DSM018TB1" class="table-modal"><div class="table-wrap"><div id="DSM018TB1" class="table-wrap-title" data-id="DSM018TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Summary of the samples, sequencing, assembly, and gene annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Sample status </th><th rowspan="2">Sample name </th><th rowspan="2">Sex </th><th rowspan="2">Age </th><th rowspan="2">Total reads </th><th rowspan="2">Total read length<sup>a</sup> (bp) </th><th colspan="2">contigs </th><th colspan="2">singlets </th><th rowspan="2">Total lengh of assembled sequence (Mb) </th><th rowspan="2">Total genes<sup>b</sup> (<em>A</em>) </th><th rowspan="2">COG-assigned genes (<em>B</em>) </th><th rowspan="2">Identified COGs<sup>c</sup> (<em>C</em>) </th><th rowspan="2">Average COG size<sup>d</sup> (<em>B</em>)/(<em>C</em>) </th><th rowspan="2">In-house NR DB hit genes<sup>e</sup> (<em>D</em>) </th><th rowspan="2">Orphan genes<sup>f</sup> (<em>A</em>) − (<em>D</em>) </th></tr><tr><th>average length (bp) </th><th>contigs   </th><th>average length (bp) </th><th>reads   </th></tr></thead><tbody><tr><td>Individual </td><td>In-A </td><td>Male </td><td>45 years </td><td>81687 </td><td>52509363 </td><td>2809.0 </td><td>5410 </td><td>902.0 </td><td>16330 </td><td>29.93 </td><td>38778 </td><td>18058 </td><td>2355 </td><td>7.67 </td><td>30210 </td><td>8568 </td></tr><tr><td>Individual </td><td>In-B </td><td>Male </td><td>6 months </td><td>80617 </td><td>62792581 </td><td>3701.6 </td><td>1721 </td><td>1003.1 </td><td>8481 </td><td>14.88 </td><td>20063 </td><td>9393 </td><td>1617 </td><td>5.81 </td><td>15127 </td><td>4936 </td></tr><tr><td>Individual </td><td>In-D </td><td>Male </td><td>35 years </td><td>84237 </td><td>55137918 </td><td>2159.1 </td><td>7613 </td><td>911.8 </td><td>36312 </td><td>49.55 </td><td>67740 </td><td>29998 </td><td>2559 </td><td>11.72 </td><td>49079 </td><td>18661 </td></tr><tr><td>Individual </td><td>In-E </td><td>Male </td><td>3 months </td><td>80852 </td><td>56781600 </td><td>2301.1 </td><td>4819 </td><td>1008.7 </td><td>16838 </td><td>28.07 </td><td>37652 </td><td>18688 </td><td>2107 </td><td>8.87 </td><td>28513 </td><td>9139 </td></tr><tr><td>Individual </td><td>In-M </td><td>Female </td><td>4 months </td><td>89340 </td><td>57808421 </td><td>3375.1 </td><td>4794 </td><td>655.7 </td><td>15541 </td><td>26.37 </td><td>34330 </td><td>18187 </td><td>2857 </td><td>6.37 </td><td>27050 </td><td>7280 </td></tr><tr><td>Individual </td><td>In-R </td><td>Female </td><td>24 years </td><td>85787 </td><td>55404826 </td><td>1920.6 </td><td>8935 </td><td>811.3 </td><td>36524 </td><td>46.79 </td><td>63356 </td><td>28612 </td><td>2655 </td><td>10.78 </td><td>46104 </td><td>17252 </td></tr><tr><td rowspan="3">Family I </td><td>F1-S </td><td>Male </td><td>30 years </td><td>78452 </td><td>53568019 </td><td>2153.6 </td><td>7545 </td><td>806.5 </td><td>28038 </td><td>38.86 </td><td>54151 </td><td>25173 </td><td>2531 </td><td>9.95 </td><td>40771 </td><td>13380 </td></tr><tr><td>F1-T </td><td>Female </td><td>28 years </td><td>81348 </td><td>55365235 </td><td>1980.4 </td><td>7389 </td><td>791.5 </td><td>37458 </td><td>44.28 </td><td>65156 </td><td>31230 </td><td>2921 </td><td>10.69 </td><td>47955 </td><td>17201 </td></tr><tr><td>F1-U </td><td>Female </td><td>7 months </td><td>82525 </td><td>53864663 </td><td>2779.4 </td><td>4854 </td><td>850.4 </td><td>14430 </td><td>25.76 </td><td>35260 </td><td>20365 </td><td>2519 </td><td>8.08 </td><td>28711 </td><td>6549 </td></tr><tr><td rowspan="4">Family II </td><td>F2-V </td><td>Male </td><td>37 years </td><td>80772 </td><td>55926002 </td><td>2008.7 </td><td>7919 </td><td>809.2 </td><td>38442 </td><td>47.02 </td><td>66461 </td><td>33535 </td><td>2873 </td><td>11.67 </td><td>49955 </td><td>16506 </td></tr><tr><td>F2-W </td><td>Female </td><td>36 years </td><td>79163 </td><td>54885684 </td><td>2289.8 </td><td>6778 </td><td>833.0 </td><td>30550 </td><td>40.97 </td><td>57213 </td><td>27680 </td><td>2609 </td><td>10.61 </td><td>43625 </td><td>13588 </td></tr><tr><td>F2-X </td><td>Male </td><td>3 years </td><td>80858 </td><td>56587120 </td><td>2296.0 </td><td>5032 </td><td>832.1 </td><td>34252 </td><td>40.05 </td><td>57446 </td><td>26599 </td><td>2669 </td><td>9.97 </td><td>42452 </td><td>14994 </td></tr><tr><td>F2-Y </td><td>Female </td><td>1.5 years </td><td>79754 </td><td>56276047 </td><td>2044.2 </td><td>9159 </td><td>849.9 </td><td>32461 </td><td>46.31 </td><td>64942 </td><td>30870 </td><td>2664 </td><td>11.59 </td><td>50349 </td><td>14593 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td>1065392 </td><td>726907479 </td><td>2299.8 </td><td>81968 </td><td>839.9 </td><td>345657 </td><td>478.84 </td><td>662548 </td><td>318388 </td><td>3268 </td><td>97.43 </td><td>499901 </td><td>162647 </td></tr><tr><td rowspan="2">American </td><td>Sub. 7 </td><td>Female </td><td>28 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15.94 </td><td>22329 </td><td>12223 </td><td>2160 </td><td>5.66 </td><td>18443 </td><td>3886 </td></tr><tr><td>Sub. 8 </td><td>Male </td><td>37 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>20.49 </td><td>27579 </td><td>14962 </td><td>2249 </td><td>6.65 </td><td>23518 </td><td>4061 </td></tr><tr><td>Soil </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>144.90 </td><td>212128 </td><td>84060 </td><td>4423 </td><td>19.01 </td><td>125302 </td><td>86826 </td></tr><tr><td rowspan="3">Whale fall </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29.78 </td><td>46478 </td><td>23222 </td><td>3140 </td><td>7.40 </td><td>32847 </td><td>13631 </td></tr><tr><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>30.86 </td><td>47274 </td><td>21826 </td><td>2922 </td><td>7.47 </td><td>32038 </td><td>15236 </td></tr><tr><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>28.87 </td><td>42870 </td><td>21076 </td><td>2894 </td><td>7.28 </td><td>31410 </td><td>11460 </td></tr><tr><td>Sargasso </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>816.55 </td><td>1406274 </td><td>746941 </td><td>5184 </td><td>144.09 </td><td>1038259 </td><td>368015 </td></tr></tbody></table></div><div content-id="DSM018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Sample status </th><th rowspan="2">Sample name </th><th rowspan="2">Sex </th><th rowspan="2">Age </th><th rowspan="2">Total reads </th><th rowspan="2">Total read length<sup>a</sup> (bp) </th><th colspan="2">contigs </th><th colspan="2">singlets </th><th rowspan="2">Total lengh of assembled sequence (Mb) </th><th rowspan="2">Total genes<sup>b</sup> (<em>A</em>) </th><th rowspan="2">COG-assigned genes (<em>B</em>) </th><th rowspan="2">Identified COGs<sup>c</sup> (<em>C</em>) </th><th rowspan="2">Average COG size<sup>d</sup> (<em>B</em>)/(<em>C</em>) </th><th rowspan="2">In-house NR DB hit genes<sup>e</sup> (<em>D</em>) </th><th rowspan="2">Orphan genes<sup>f</sup> (<em>A</em>) − (<em>D</em>) </th></tr><tr><th>average length (bp) </th><th>contigs   </th><th>average length (bp) </th><th>reads   </th></tr></thead><tbody><tr><td>Individual </td><td>In-A </td><td>Male </td><td>45 years </td><td>81687 </td><td>52509363 </td><td>2809.0 </td><td>5410 </td><td>902.0 </td><td>16330 </td><td>29.93 </td><td>38778 </td><td>18058 </td><td>2355 </td><td>7.67 </td><td>30210 </td><td>8568 </td></tr><tr><td>Individual </td><td>In-B </td><td>Male </td><td>6 months </td><td>80617 </td><td>62792581 </td><td>3701.6 </td><td>1721 </td><td>1003.1 </td><td>8481 </td><td>14.88 </td><td>20063 </td><td>9393 </td><td>1617 </td><td>5.81 </td><td>15127 </td><td>4936 </td></tr><tr><td>Individual </td><td>In-D </td><td>Male </td><td>35 years </td><td>84237 </td><td>55137918 </td><td>2159.1 </td><td>7613 </td><td>911.8 </td><td>36312 </td><td>49.55 </td><td>67740 </td><td>29998 </td><td>2559 </td><td>11.72 </td><td>49079 </td><td>18661 </td></tr><tr><td>Individual </td><td>In-E </td><td>Male </td><td>3 months </td><td>80852 </td><td>56781600 </td><td>2301.1 </td><td>4819 </td><td>1008.7 </td><td>16838 </td><td>28.07 </td><td>37652 </td><td>18688 </td><td>2107 </td><td>8.87 </td><td>28513 </td><td>9139 </td></tr><tr><td>Individual </td><td>In-M </td><td>Female </td><td>4 months </td><td>89340 </td><td>57808421 </td><td>3375.1 </td><td>4794 </td><td>655.7 </td><td>15541 </td><td>26.37 </td><td>34330 </td><td>18187 </td><td>2857 </td><td>6.37 </td><td>27050 </td><td>7280 </td></tr><tr><td>Individual </td><td>In-R </td><td>Female </td><td>24 years </td><td>85787 </td><td>55404826 </td><td>1920.6 </td><td>8935 </td><td>811.3 </td><td>36524 </td><td>46.79 </td><td>63356 </td><td>28612 </td><td>2655 </td><td>10.78 </td><td>46104 </td><td>17252 </td></tr><tr><td rowspan="3">Family I </td><td>F1-S </td><td>Male </td><td>30 years </td><td>78452 </td><td>53568019 </td><td>2153.6 </td><td>7545 </td><td>806.5 </td><td>28038 </td><td>38.86 </td><td>54151 </td><td>25173 </td><td>2531 </td><td>9.95 </td><td>40771 </td><td>13380 </td></tr><tr><td>F1-T </td><td>Female </td><td>28 years </td><td>81348 </td><td>55365235 </td><td>1980.4 </td><td>7389 </td><td>791.5 </td><td>37458 </td><td>44.28 </td><td>65156 </td><td>31230 </td><td>2921 </td><td>10.69 </td><td>47955 </td><td>17201 </td></tr><tr><td>F1-U </td><td>Female </td><td>7 months </td><td>82525 </td><td>53864663 </td><td>2779.4 </td><td>4854 </td><td>850.4 </td><td>14430 </td><td>25.76 </td><td>35260 </td><td>20365 </td><td>2519 </td><td>8.08 </td><td>28711 </td><td>6549 </td></tr><tr><td rowspan="4">Family II </td><td>F2-V </td><td>Male </td><td>37 years </td><td>80772 </td><td>55926002 </td><td>2008.7 </td><td>7919 </td><td>809.2 </td><td>38442 </td><td>47.02 </td><td>66461 </td><td>33535 </td><td>2873 </td><td>11.67 </td><td>49955 </td><td>16506 </td></tr><tr><td>F2-W </td><td>Female </td><td>36 years </td><td>79163 </td><td>54885684 </td><td>2289.8 </td><td>6778 </td><td>833.0 </td><td>30550 </td><td>40.97 </td><td>57213 </td><td>27680 </td><td>2609 </td><td>10.61 </td><td>43625 </td><td>13588 </td></tr><tr><td>F2-X </td><td>Male </td><td>3 years </td><td>80858 </td><td>56587120 </td><td>2296.0 </td><td>5032 </td><td>832.1 </td><td>34252 </td><td>40.05 </td><td>57446 </td><td>26599 </td><td>2669 </td><td>9.97 </td><td>42452 </td><td>14994 </td></tr><tr><td>F2-Y </td><td>Female </td><td>1.5 years </td><td>79754 </td><td>56276047 </td><td>2044.2 </td><td>9159 </td><td>849.9 </td><td>32461 </td><td>46.31 </td><td>64942 </td><td>30870 </td><td>2664 </td><td>11.59 </td><td>50349 </td><td>14593 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td>1065392 </td><td>726907479 </td><td>2299.8 </td><td>81968 </td><td>839.9 </td><td>345657 </td><td>478.84 </td><td>662548 </td><td>318388 </td><td>3268 </td><td>97.43 </td><td>499901 </td><td>162647 </td></tr><tr><td rowspan="2">American </td><td>Sub. 7 </td><td>Female </td><td>28 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15.94 </td><td>22329 </td><td>12223 </td><td>2160 </td><td>5.66 </td><td>18443 </td><td>3886 </td></tr><tr><td>Sub. 8 </td><td>Male </td><td>37 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>20.49 </td><td>27579 </td><td>14962 </td><td>2249 </td><td>6.65 </td><td>23518 </td><td>4061 </td></tr><tr><td>Soil </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>144.90 </td><td>212128 </td><td>84060 </td><td>4423 </td><td>19.01 </td><td>125302 </td><td>86826 </td></tr><tr><td rowspan="3">Whale fall </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29.78 </td><td>46478 </td><td>23222 </td><td>3140 </td><td>7.40 </td><td>32847 </td><td>13631 </td></tr><tr><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>30.86 </td><td>47274 </td><td>21826 </td><td>2922 </td><td>7.47 </td><td>32038 </td><td>15236 </td></tr><tr><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>28.87 </td><td>42870 </td><td>21076 </td><td>2894 </td><td>7.28 </td><td>31410 </td><td>11460 </td></tr><tr><td>Sargasso </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>816.55 </td><td>1406274 </td><td>746941 </td><td>5184 </td><td>144.09 </td><td>1038259 </td><td>368015 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The total read length with Phred scores of &gt;15.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The number of genes predicted from the non-redundant sequences of each sample by the MetaGene program.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The number of COGs to which at least one gene was assigned in each microbime.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The average numbers of predicted genes belonging to each COG [(<em>B</em>)/(<em>C</em>)].</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>The number of predicted genes showing similarity to genes in the ‘in-house extended NR database’.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The number of genes showing no significant similarity to genes in the ‘in-house extended NR database’ (e-value ≥ 1.0e − 5 in the BLASTP analysis).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195792" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSM018TB1" class="table-wrap-title" data-id="DSM018TB1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Summary of the samples, sequencing, assembly, and gene annotation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Sample status </th><th rowspan="2">Sample name </th><th rowspan="2">Sex </th><th rowspan="2">Age </th><th rowspan="2">Total reads </th><th rowspan="2">Total read length<sup>a</sup> (bp) </th><th colspan="2">contigs </th><th colspan="2">singlets </th><th rowspan="2">Total lengh of assembled sequence (Mb) </th><th rowspan="2">Total genes<sup>b</sup> (<em>A</em>) </th><th rowspan="2">COG-assigned genes (<em>B</em>) </th><th rowspan="2">Identified COGs<sup>c</sup> (<em>C</em>) </th><th rowspan="2">Average COG size<sup>d</sup> (<em>B</em>)/(<em>C</em>) </th><th rowspan="2">In-house NR DB hit genes<sup>e</sup> (<em>D</em>) </th><th rowspan="2">Orphan genes<sup>f</sup> (<em>A</em>) − (<em>D</em>) </th></tr><tr><th>average length (bp) </th><th>contigs   </th><th>average length (bp) </th><th>reads   </th></tr></thead><tbody><tr><td>Individual </td><td>In-A </td><td>Male </td><td>45 years </td><td>81687 </td><td>52509363 </td><td>2809.0 </td><td>5410 </td><td>902.0 </td><td>16330 </td><td>29.93 </td><td>38778 </td><td>18058 </td><td>2355 </td><td>7.67 </td><td>30210 </td><td>8568 </td></tr><tr><td>Individual </td><td>In-B </td><td>Male </td><td>6 months </td><td>80617 </td><td>62792581 </td><td>3701.6 </td><td>1721 </td><td>1003.1 </td><td>8481 </td><td>14.88 </td><td>20063 </td><td>9393 </td><td>1617 </td><td>5.81 </td><td>15127 </td><td>4936 </td></tr><tr><td>Individual </td><td>In-D </td><td>Male </td><td>35 years </td><td>84237 </td><td>55137918 </td><td>2159.1 </td><td>7613 </td><td>911.8 </td><td>36312 </td><td>49.55 </td><td>67740 </td><td>29998 </td><td>2559 </td><td>11.72 </td><td>49079 </td><td>18661 </td></tr><tr><td>Individual </td><td>In-E </td><td>Male </td><td>3 months </td><td>80852 </td><td>56781600 </td><td>2301.1 </td><td>4819 </td><td>1008.7 </td><td>16838 </td><td>28.07 </td><td>37652 </td><td>18688 </td><td>2107 </td><td>8.87 </td><td>28513 </td><td>9139 </td></tr><tr><td>Individual </td><td>In-M </td><td>Female </td><td>4 months </td><td>89340 </td><td>57808421 </td><td>3375.1 </td><td>4794 </td><td>655.7 </td><td>15541 </td><td>26.37 </td><td>34330 </td><td>18187 </td><td>2857 </td><td>6.37 </td><td>27050 </td><td>7280 </td></tr><tr><td>Individual </td><td>In-R </td><td>Female </td><td>24 years </td><td>85787 </td><td>55404826 </td><td>1920.6 </td><td>8935 </td><td>811.3 </td><td>36524 </td><td>46.79 </td><td>63356 </td><td>28612 </td><td>2655 </td><td>10.78 </td><td>46104 </td><td>17252 </td></tr><tr><td rowspan="3">Family I </td><td>F1-S </td><td>Male </td><td>30 years </td><td>78452 </td><td>53568019 </td><td>2153.6 </td><td>7545 </td><td>806.5 </td><td>28038 </td><td>38.86 </td><td>54151 </td><td>25173 </td><td>2531 </td><td>9.95 </td><td>40771 </td><td>13380 </td></tr><tr><td>F1-T </td><td>Female </td><td>28 years </td><td>81348 </td><td>55365235 </td><td>1980.4 </td><td>7389 </td><td>791.5 </td><td>37458 </td><td>44.28 </td><td>65156 </td><td>31230 </td><td>2921 </td><td>10.69 </td><td>47955 </td><td>17201 </td></tr><tr><td>F1-U </td><td>Female </td><td>7 months </td><td>82525 </td><td>53864663 </td><td>2779.4 </td><td>4854 </td><td>850.4 </td><td>14430 </td><td>25.76 </td><td>35260 </td><td>20365 </td><td>2519 </td><td>8.08 </td><td>28711 </td><td>6549 </td></tr><tr><td rowspan="4">Family II </td><td>F2-V </td><td>Male </td><td>37 years </td><td>80772 </td><td>55926002 </td><td>2008.7 </td><td>7919 </td><td>809.2 </td><td>38442 </td><td>47.02 </td><td>66461 </td><td>33535 </td><td>2873 </td><td>11.67 </td><td>49955 </td><td>16506 </td></tr><tr><td>F2-W </td><td>Female </td><td>36 years </td><td>79163 </td><td>54885684 </td><td>2289.8 </td><td>6778 </td><td>833.0 </td><td>30550 </td><td>40.97 </td><td>57213 </td><td>27680 </td><td>2609 </td><td>10.61 </td><td>43625 </td><td>13588 </td></tr><tr><td>F2-X </td><td>Male </td><td>3 years </td><td>80858 </td><td>56587120 </td><td>2296.0 </td><td>5032 </td><td>832.1 </td><td>34252 </td><td>40.05 </td><td>57446 </td><td>26599 </td><td>2669 </td><td>9.97 </td><td>42452 </td><td>14994 </td></tr><tr><td>F2-Y </td><td>Female </td><td>1.5 years </td><td>79754 </td><td>56276047 </td><td>2044.2 </td><td>9159 </td><td>849.9 </td><td>32461 </td><td>46.31 </td><td>64942 </td><td>30870 </td><td>2664 </td><td>11.59 </td><td>50349 </td><td>14593 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td>1065392 </td><td>726907479 </td><td>2299.8 </td><td>81968 </td><td>839.9 </td><td>345657 </td><td>478.84 </td><td>662548 </td><td>318388 </td><td>3268 </td><td>97.43 </td><td>499901 </td><td>162647 </td></tr><tr><td rowspan="2">American </td><td>Sub. 7 </td><td>Female </td><td>28 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15.94 </td><td>22329 </td><td>12223 </td><td>2160 </td><td>5.66 </td><td>18443 </td><td>3886 </td></tr><tr><td>Sub. 8 </td><td>Male </td><td>37 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>20.49 </td><td>27579 </td><td>14962 </td><td>2249 </td><td>6.65 </td><td>23518 </td><td>4061 </td></tr><tr><td>Soil </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>144.90 </td><td>212128 </td><td>84060 </td><td>4423 </td><td>19.01 </td><td>125302 </td><td>86826 </td></tr><tr><td rowspan="3">Whale fall </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29.78 </td><td>46478 </td><td>23222 </td><td>3140 </td><td>7.40 </td><td>32847 </td><td>13631 </td></tr><tr><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>30.86 </td><td>47274 </td><td>21826 </td><td>2922 </td><td>7.47 </td><td>32038 </td><td>15236 </td></tr><tr><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>28.87 </td><td>42870 </td><td>21076 </td><td>2894 </td><td>7.28 </td><td>31410 </td><td>11460 </td></tr><tr><td>Sargasso </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>816.55 </td><td>1406274 </td><td>746941 </td><td>5184 </td><td>144.09 </td><td>1038259 </td><td>368015 </td></tr></tbody></table></div><div content-id="DSM018TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Sample status </th><th rowspan="2">Sample name </th><th rowspan="2">Sex </th><th rowspan="2">Age </th><th rowspan="2">Total reads </th><th rowspan="2">Total read length<sup>a</sup> (bp) </th><th colspan="2">contigs </th><th colspan="2">singlets </th><th rowspan="2">Total lengh of assembled sequence (Mb) </th><th rowspan="2">Total genes<sup>b</sup> (<em>A</em>) </th><th rowspan="2">COG-assigned genes (<em>B</em>) </th><th rowspan="2">Identified COGs<sup>c</sup> (<em>C</em>) </th><th rowspan="2">Average COG size<sup>d</sup> (<em>B</em>)/(<em>C</em>) </th><th rowspan="2">In-house NR DB hit genes<sup>e</sup> (<em>D</em>) </th><th rowspan="2">Orphan genes<sup>f</sup> (<em>A</em>) − (<em>D</em>) </th></tr><tr><th>average length (bp) </th><th>contigs   </th><th>average length (bp) </th><th>reads   </th></tr></thead><tbody><tr><td>Individual </td><td>In-A </td><td>Male </td><td>45 years </td><td>81687 </td><td>52509363 </td><td>2809.0 </td><td>5410 </td><td>902.0 </td><td>16330 </td><td>29.93 </td><td>38778 </td><td>18058 </td><td>2355 </td><td>7.67 </td><td>30210 </td><td>8568 </td></tr><tr><td>Individual </td><td>In-B </td><td>Male </td><td>6 months </td><td>80617 </td><td>62792581 </td><td>3701.6 </td><td>1721 </td><td>1003.1 </td><td>8481 </td><td>14.88 </td><td>20063 </td><td>9393 </td><td>1617 </td><td>5.81 </td><td>15127 </td><td>4936 </td></tr><tr><td>Individual </td><td>In-D </td><td>Male </td><td>35 years </td><td>84237 </td><td>55137918 </td><td>2159.1 </td><td>7613 </td><td>911.8 </td><td>36312 </td><td>49.55 </td><td>67740 </td><td>29998 </td><td>2559 </td><td>11.72 </td><td>49079 </td><td>18661 </td></tr><tr><td>Individual </td><td>In-E </td><td>Male </td><td>3 months </td><td>80852 </td><td>56781600 </td><td>2301.1 </td><td>4819 </td><td>1008.7 </td><td>16838 </td><td>28.07 </td><td>37652 </td><td>18688 </td><td>2107 </td><td>8.87 </td><td>28513 </td><td>9139 </td></tr><tr><td>Individual </td><td>In-M </td><td>Female </td><td>4 months </td><td>89340 </td><td>57808421 </td><td>3375.1 </td><td>4794 </td><td>655.7 </td><td>15541 </td><td>26.37 </td><td>34330 </td><td>18187 </td><td>2857 </td><td>6.37 </td><td>27050 </td><td>7280 </td></tr><tr><td>Individual </td><td>In-R </td><td>Female </td><td>24 years </td><td>85787 </td><td>55404826 </td><td>1920.6 </td><td>8935 </td><td>811.3 </td><td>36524 </td><td>46.79 </td><td>63356 </td><td>28612 </td><td>2655 </td><td>10.78 </td><td>46104 </td><td>17252 </td></tr><tr><td rowspan="3">Family I </td><td>F1-S </td><td>Male </td><td>30 years </td><td>78452 </td><td>53568019 </td><td>2153.6 </td><td>7545 </td><td>806.5 </td><td>28038 </td><td>38.86 </td><td>54151 </td><td>25173 </td><td>2531 </td><td>9.95 </td><td>40771 </td><td>13380 </td></tr><tr><td>F1-T </td><td>Female </td><td>28 years </td><td>81348 </td><td>55365235 </td><td>1980.4 </td><td>7389 </td><td>791.5 </td><td>37458 </td><td>44.28 </td><td>65156 </td><td>31230 </td><td>2921 </td><td>10.69 </td><td>47955 </td><td>17201 </td></tr><tr><td>F1-U </td><td>Female </td><td>7 months </td><td>82525 </td><td>53864663 </td><td>2779.4 </td><td>4854 </td><td>850.4 </td><td>14430 </td><td>25.76 </td><td>35260 </td><td>20365 </td><td>2519 </td><td>8.08 </td><td>28711 </td><td>6549 </td></tr><tr><td rowspan="4">Family II </td><td>F2-V </td><td>Male </td><td>37 years </td><td>80772 </td><td>55926002 </td><td>2008.7 </td><td>7919 </td><td>809.2 </td><td>38442 </td><td>47.02 </td><td>66461 </td><td>33535 </td><td>2873 </td><td>11.67 </td><td>49955 </td><td>16506 </td></tr><tr><td>F2-W </td><td>Female </td><td>36 years </td><td>79163 </td><td>54885684 </td><td>2289.8 </td><td>6778 </td><td>833.0 </td><td>30550 </td><td>40.97 </td><td>57213 </td><td>27680 </td><td>2609 </td><td>10.61 </td><td>43625 </td><td>13588 </td></tr><tr><td>F2-X </td><td>Male </td><td>3 years </td><td>80858 </td><td>56587120 </td><td>2296.0 </td><td>5032 </td><td>832.1 </td><td>34252 </td><td>40.05 </td><td>57446 </td><td>26599 </td><td>2669 </td><td>9.97 </td><td>42452 </td><td>14994 </td></tr><tr><td>F2-Y </td><td>Female </td><td>1.5 years </td><td>79754 </td><td>56276047 </td><td>2044.2 </td><td>9159 </td><td>849.9 </td><td>32461 </td><td>46.31 </td><td>64942 </td><td>30870 </td><td>2664 </td><td>11.59 </td><td>50349 </td><td>14593 </td></tr><tr><td>Total </td><td> </td><td> </td><td> </td><td>1065392 </td><td>726907479 </td><td>2299.8 </td><td>81968 </td><td>839.9 </td><td>345657 </td><td>478.84 </td><td>662548 </td><td>318388 </td><td>3268 </td><td>97.43 </td><td>499901 </td><td>162647 </td></tr><tr><td rowspan="2">American </td><td>Sub. 7 </td><td>Female </td><td>28 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>15.94 </td><td>22329 </td><td>12223 </td><td>2160 </td><td>5.66 </td><td>18443 </td><td>3886 </td></tr><tr><td>Sub. 8 </td><td>Male </td><td>37 years </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>20.49 </td><td>27579 </td><td>14962 </td><td>2249 </td><td>6.65 </td><td>23518 </td><td>4061 </td></tr><tr><td>Soil </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>144.90 </td><td>212128 </td><td>84060 </td><td>4423 </td><td>19.01 </td><td>125302 </td><td>86826 </td></tr><tr><td rowspan="3">Whale fall </td><td>1 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>29.78 </td><td>46478 </td><td>23222 </td><td>3140 </td><td>7.40 </td><td>32847 </td><td>13631 </td></tr><tr><td>2 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>30.86 </td><td>47274 </td><td>21826 </td><td>2922 </td><td>7.47 </td><td>32038 </td><td>15236 </td></tr><tr><td>3 </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>28.87 </td><td>42870 </td><td>21076 </td><td>2894 </td><td>7.28 </td><td>31410 </td><td>11460 </td></tr><tr><td>Sargasso </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td>816.55 </td><td>1406274 </td><td>746941 </td><td>5184 </td><td>144.09 </td><td>1038259 </td><td>368015 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The total read length with Phred scores of &gt;15.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The number of genes predicted from the non-redundant sequences of each sample by the MetaGene program.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The number of COGs to which at least one gene was assigned in each microbime.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The average numbers of predicted genes belonging to each COG [(<em>B</em>)/(<em>C</em>)].</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>The number of predicted genes showing similarity to genes in the ‘in-house extended NR database’.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The number of genes showing no significant similarity to genes in the ‘in-house extended NR database’ (e-value ≥ 1.0e − 5 in the BLASTP analysis).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195792" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195793 id="87195793" class="section-title" data-legacy-id=s2b>2.2. Isolation of bacterial DNA from fecal samples</h3>
<p>The collected fecal samples were immediately placed under anaerobic conditions using the Anaero-Pack system (Mitsubishi Gas Chemical Co.), and transported at 4°C to the laboratory within 24 h. Three grams of wet fecal sample were suspended vigorously in phosphate-buffered saline (8 mM Na<sub>2</sub>HPO<sub>4</sub>, 137 mM NaCl, 2.7 mM KCl, and 1.5 mM KH<sub>2</sub>PO<sub>4</sub>). The suspension was filtered through a 100 µm mesh nylon filter (Falcon). The bacterial cells in the filtrates were collected by centrifugation at 5000 × <em>g</em> for 10 min at 4°C, suspended in 10 ml of Tris–EDTA containing 10 mM Tris-HCl and 1 mM EDTA (pH 8), and then used for DNA isolation. The lysis of bacterial cells was carried out under almost the same conditions as those described in the literature.<sup><a reveal-id="DSM018C26" data-open="DSM018C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In brief, 1.5 mg of lysozyme was added to 10 ml of cell suspension. After incubation at 37°C for 1 h with gentle mixing, 2 mg of proteinase K were added and the mixture was incubated at 55°C for 5 min. Subsequently, 1.2 ml of 10% (wt/vol) sodium dodecyl sulfate was added to the cell suspension, which was further incubated at 55°C for 1 h with gentle mixing. High-molecular-weight DNA was isolated and purified by phenol/chloroform extraction, RNase A treatment, ethanol, and finally polyethyleneglycol precipitation.</p><p>To assess the efficiency of cell lysis, the total number of microbial cells was determined before and after the lytic treatment. A chamber with a depth of 0.02 mm (Erma, Tokyo, Japan) was used for the bacterial cell counting under a phase-contrast microscope (Leica; Leitz DMR model). The cell counting was repeated three times for each sample. The average efficiency of the cell lysis was apparently around 70% or more.</p>                            <h3 scrollto-destination=87195796 id="87195796" class="section-title" data-legacy-id=s2c>2.3. DNA sequencing, assembly, and gene prediction</h3>
<p>Shotgun libraries were constructed from randomly sheared bacterial DNA (2–3 kb) (HydroShear, GeneMachines) and the pUC18 vector. Template DNA for the sequencing was prepared by polymerase chain reaction (PCR) amplification of the insert DNA using a TaKaRa ExTaq kit (Takara Bio) and GeneAmp PCR System 9700 (ABI) as described previously.<sup><a reveal-id="DSM018C27" data-open="DSM018C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Sequencing was carried out for both ends of the clones using the BigDye v3.1 chemistry on ABI3730 sequencers (ABI) or the ET chemistry on MegaBACE4500 sequencers (GE Healthcare). The shotgun reads from each sample were individually assembled to generate non-redundant metasequences using the PCAP software<sup><a reveal-id="DSM018C28" data-open="DSM018C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with default parameters.</p><p>The MetaGene program,<sup><a reveal-id="DSM018C29" data-open="DSM018C29" class="link link-ref link-reveal xref-bibr">29</a></sup> which is based on a Hidden Markov Model (HMM) algorithm, was employed to predict potential protein-coding regions (open reading frames, ORFs ≥20 amino acids) from the metasequences of each sample. Prior to gene finding, we masked low-quality sequences (Phred score &lt;15) using ‘Xs’.</p>                            <h3 scrollto-destination=87195799 id="87195799" class="section-title" data-legacy-id=s2d>2.4. Database construction</h3>
<p>The ‘in-house extended NR database’ included the data set from the GenBank non-redundant amino-acid database (version 26, May 2007) plus a dataset obtained by MetaGene prediction from 44 unpublished microbial genome sequences (Supplementary Table S1). These unpublished sequences were obtained from the public database and the websites of the Genome Sequencing Center of Washington University, St Louis (<a class="link link-uri" href="http://genome.wustl.edu/sub_genome_group_index.cgi?GROUP">http://genome.wustl.edu/sub_genome_group_index.cgi?GROUP</a> = 3) and the Human Metagenome Consortium Japan (HMGJ; <a class="link link-uri" href="http://www.metagenome.jp/">http://www.metagenome.jp/</a>). The ‘reference dataset for COG (Cluster of Orthologous Groups of proteins) assignment’ (Supplementary Table S2) contained 343 microbial genome sequences where COG assignment has been made for all the genes by the NCBI. The ‘in-house reference database’ (referred to as ‘Ref-DB’) was constructed by selecting 243 microbial genomes (Supplementary Table S3) from the reference dataset for COG assignment. To avoid the effect of multiply sequenced species, we selected one representative strain from each species. To identify the genomic features specific to human gut microbiomes, known gut microbes were also excluded from Ref-DB.</p><p>To search for the genes related to the Tn<em>1549</em>-like conjugative transposons (CTns), we also constructed a modified Ref-DB by adding the genes on Tn<em>1549</em> from <em>Enterococcus faecalis</em> strain BM4382 (AAF72340-68)<sup><a reveal-id="DSM018C30" data-open="DSM018C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and on CTn2, CTn4, and CTn5 of <em>Clostridium difficile</em> strain 630.<sup><a reveal-id="DSM018C31" data-open="DSM018C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Since Ref-DB originally includes those on the Tn<em>1549</em>-like CTns from <em>E. faecalis</em> strain V583 <sup><a reveal-id="DSM018C32" data-open="DSM018C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and <em>Streptococcus pyogenes</em> strain MGAS10750,<sup><a reveal-id="DSM018C33" data-open="DSM018C33" class="link link-ref link-reveal xref-bibr">33</a></sup> the modified Ref-DB contains six known Tn<em>1549</em>-like CTns.</p>                            <h3 scrollto-destination=87195802 id="87195802" class="section-title" data-legacy-id=s2e>2.5. Clustering analysis of pairwise microbiome comparison</h3>
<p>BLASTP<sup><a reveal-id="DSM018C34" data-open="DSM018C34" class="link link-ref link-reveal xref-bibr">34</a></sup> analyses of all protein sequences from one microbiome against all those from every other were used to estimate the genomic similarities existing in all possible microbiome comparisons.<sup><a reveal-id="DSM018C18" data-open="DSM018C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSM018C35" data-open="DSM018C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Bit-scores of the best hits from every single sequence from one microbiome against another were summed up to yield a cumulative pairwise bit-score value. The cumulative bit-score values from all possible pairwise comparisons were then used to construct a distance matrix.<sup><a reveal-id="DSM018C18" data-open="DSM018C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Since the cumulative bit-score values from microbiome A to microbiome B [denoted as Σ(A,B)] is not equal to that from microbiome B to microbiome A [Σ(B,A)] in the non-reciprocal BLASTP analysis, we used the minimum of the two values, min[Σ(A,B), Σ(B,A)], as a measure of the similarity between microbiomes A and B. To normalize the differences derived from the size differences among the microbiomes, we calculated the measure of genome conservation distance, D2.<sup><a reveal-id="DSM018C36" data-open="DSM018C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSM018C37" data-open="DSM018C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The obtained distance matrix was then used for clustering by the multi-dimensional scaling (MDS) method.</p>                            <h3 scrollto-destination=87195804 id="87195804" class="section-title" data-legacy-id=s2f>2.6. Taxonomic assignment</h3>
<p>Taxonomic assignment of protein-coding genes was performed according to the best-hit pairs in the BLASTP analysis against the in-house extended NR database where the taxonomic information for all genes is available. For the first screening, all BLASTP results were filtered by e-value (≤1.0e − 8) and hit length coverage (≥50% of a query sequence). Then, to tentatively assign the taxonomic origins of the genes, we adopted a 90% BLASTP identity threshold. The genes with best-hit pairs under the threshold were assigned as ‘No hits’.</p>                            <h3 scrollto-destination=87195806 id="87195806" class="section-title" data-legacy-id=s2g>2.7. COG assignment and evaluation of enrichment</h3>
<p>COG assignment of predicted gene products was made on the basis of the BLASTP analysis against the reference dataset for COG assignment. After filtering the BLASTP results by e-value (≤1.0e − 8) and aligned length coverage (≥50% of a query sequence), the COG assignment of each gene product was made according to the COG information of its best-hit pair in the reference dataset for COG assignment. If the best-hit pair was not assigned to any COG, the gene product was considered to be ‘uncharacterized’. On the basis of COG assignments, the size of each COG (the number of gene products belonging to each COG) was then counted up for every microbiome. Since most of the genes located at the contig ends and in singletons are predicted as ‘partial genes’ by MetaGene, their hit counts were corrected by the length ratio of each ‘partial gene’ to the reference to minimize multiple counts of fragmented genes. When a predicted gene product had a best-hit pair with a reference protein that has been assigned to multiple COGs, this hit was divided by the number of assigned COGs and the value was dispensed evenly to each COG. The size of each COG was normalized by the total number of genes predicted in each microbiome (‘normalized %’). The average size of each COG in previously sequenced microbial genomes in Ref-DB was also calculated and normalized by the average total gene number (‘DB %’). Finally, the magnitude of enrichment (enrichment value) of each COG was calculated for every microbiome by dividing the ‘normalized %’ by the ‘DB%’. COGs with an average enrichment value of &gt; 2.0 were defined as enriched COGs in each microbiome because, in all the gut microbiomes examined, the enrichment values for 126 COGs belonging to the essential gene sets defined in <em>Escherichia coli</em> and <em>Bacillus subtilis</em> mostly varied between 0.3 and 1.9, with a mean value of 0.92. A clustering analysis of microbiomes based on the COG enrichment values was performed using only the 868 COGs where at least one member was observed in every microbiome to avoid the effects of unidentified COGs. The matrix was clustered independently in the microbiomes and in the COGs using the pairwise complete-linkage hierarchical clustering of the uncentered correlation (cluster-1.31<sup>38</sup>).</p>                            <h3 scrollto-destination=87195808 id="87195808" class="section-title" data-legacy-id=s2h>2.8. Orphan genes in human gut microbiomes</h3>
<p>Orphan genes whose products showed no significant similarity to known proteins were surveyed from the 662,548 genes predicted in the 13 samples by the BLASTP analysis with the e-value threshold of 1.0e − 5 against the ‘in-house extended NR database’. We also obtained orphan genes from the metagenomic sequences of other environmental microbiomes<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSM018C26" data-open="DSM018C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSM018C39" data-open="DSM018C39" class="link link-ref link-reveal xref-bibr">39</a></sup> under the same condition. We then performed an all-to-all BLASTP analysis among the orphan gene products and selected the pairs having BLAST scores greater than 60. The gene products selected were subsequently clustered using the TribeMCL program<sup><a reveal-id="DSM018C40" data-open="DSM018C40" class="link link-ref link-reveal xref-bibr">40</a></sup> with default parameters. Of the clusters comprising only the gene products derived from human gut microbiomes, those containing ≥30 members were further subjected to motif search/extraction with the HMMER program against the Pfam motif database<sup><a reveal-id="DSM018C41" data-open="DSM018C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and to identification of the conserved amino-acid sequences by using the MEME program.<sup><a reveal-id="DSM018C42" data-open="DSM018C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=87195810 id="87195810" class="section-title" data-legacy-id=s2i>2.9. Accession numbers</h3>
<p>All the assembled sequence data have been deposited in DDBJ/EMBL/GenBank under accession numbers BAAU01000001-BAAU01028900 (subject F1-S), BAAV01000001-BAAV01036326 (F1-T), BAAW01000001-BAAW01016539 (F1-U), BAAX01000001-BAAX01036455 (F2-V), BAAY01000001-BAAY01030198 (F2-W), BAAZ01000001-BAAZ01031237 (F2-X), BABA01000001-BABA01035177 (F2-Y), BABB01000001-BABB01020226 (In-A), BABC01000001-BABC01009958 (In-B), BABD01000001-BABD01037296 (In-D), BABE01000001-BABE01020532 (In-E), BABF01000001-BABF01016164 (In-M), and BABG01000001-BABG01034797 (In-R).</p>                            <h2 scrollto-destination=87195812 id="87195812" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87195813 id="87195813" class="section-title" data-legacy-id=s3a>3.1. Samples, sequencing, and gene prediction</h3>
<p>The subjects analysed in this study included seven adults (aged 24–45 years), two children (3 and 1.5 years), and four unweaned infants (3–7 months). Seven of the subjects belonged to two unrelated families consisting of three and four members, respectively (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>We isolated the microbial DNA from each fecal sample, constructed shotgun libraries (see ‘Materials and methods’ for details), and produced a total of 1,057,481 shotgun reads (about 80,000 reads for each) representing sequences of about 727 Mb at a Phred score of &gt;15 (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>). Relatively large numbers of total shotgun reads (67.6% on average) were assembled into contigs for each sample (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>), contrasted with the soil sample in which less than 1% of the total reads (nearly 150, 000 reads) were assembled.<sup><a reveal-id="DSM018C39" data-open="DSM018C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Although 52–80% (in most cases less than 60%) of the total reads were assembled in the adults and children, 79–89% were assembled in the unweaned infants. Therefore, the lengths of the non-redundant metasequences significantly differed between adult/child and infant microbiomes: 38.9–49.6 Mb for all adults and children except for one adult (29.9 Mb in In-A) and 14.9–28.1 Mb for all infants (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>). The total length of the contigs and singletons obtained from the 13 samples was 478.8 Mb.</p><p>From each non-redundant metasequence, we identified 20,063–67,740 potential protein-coding genes (≥20 amino acids) by using the MetaGene program (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>). We might have overestimated the number of genes in our metasequences by false prediction and/or double counting of the fragmented ORFs that were derived from the same gene. The MetaGene program, however, predicted 1,406,000 ORFs in the Sargasso Sea metagenomic data,<sup><a reveal-id="DSM018C26" data-open="DSM018C26" class="link link-ref link-reveal xref-bibr">26</a></sup> which is 8.7% more than that (1,284,108 ORFs) identified from the same dataset by evidence-based gene finding.</p>                            <h3 scrollto-destination=87195817 id="87195817" class="section-title" data-legacy-id=s3b>3.2. Composition of human gut microbiota</h3>
<p>To compare the overall sequence similarities among the microbiomes from fecal and other-environmental samples,<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSM018C26" data-open="DSM018C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSM018C39" data-open="DSM018C39" class="link link-ref link-reveal xref-bibr">39</a></sup> we performed a reciprocal BLASTP analysis of the whole gene set for each microbiome, followed by MDS clustering against the D2 normalized distance matrix (see ‘Materials and methods’). The data indicated that all gut microbiomes from the adults and weaned children form a distinct group (Fig. <a reveal-id="DSM018F1" data-open="DSM018F1" class="link link-reveal link-table xref-fig">1</a>). In contrast, those from the unweaned infants were highly divergent from each other and from the microbiomes of the adults and children, as well as from those of other environments.</p>                        <a id="87195819" scrollto-destination="87195819"></a>
<div id="DSM018F1" data-id="DSM018F1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01801.jpeg?Expires=2147483647&Signature=wLOOrvJi5xjfQM3kgCOLXeXA1PccpPf37JAHQYIjtqLjO2bQWdOXBJ6FitKCtbxRVyOzHRLvxEmu0AWJ9vN0SGrzlHDJwxGD6h7LBwzSbVPhw~7RfHF6cHi2KSQNmuHPGbScTJ~5mUU2s6u9J0i8ugd1L18wpD1bXCodJxaZkjhl0UrWpJRBZQzrngCj91aCyrIDLy3f-s6~BtZsOZGtuicW9VmWgu7A34gcmaXVqpY94cz~puCKyL7LIQABCrVHq3Su8cv--2S747T7~HWV8Ol5QPmie88acWVJJb5l7nFiFgXJQMMcGvCZSwRaRvtkx3gWVjtJzwu01~ejm~m2Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering analysis of microbiomes based on cumulative bitscore comparisons. MDS was applied to the distance matrix calculated from reciprocal pairwise BLASTP analysis among all predicted gene products. The dots represent fecal samples from adults and children (blue), unweaned infants (red), Americans (green), and samples from other natural environments (orange). Whale falls 1–3, Sargasso and soil indicate the metadata for microbial communities from the deep sea,39 surface seawater,26 and the surface soil of a farm,39 respectively." path-from-xml="dsm01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195819/dsm01801.jpeg" path-from-xml="dsm01801.jpeg" target="_blank">View large</a><a section="87195819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01801.jpeg?Expires=2147483647&Signature=0BC0uhk8tSyL4UfSqgPbCPsM8-s4Atkzyu3z0SkD13uJ2QArJBToz1w6lBab6V~jpwyasKDu4gIDVg2qeCPIQNKZU3sokyDuFgC80rto3XRKw9GM5VtiTtYdbcGvOMPTVs1NtxY63TfzlBsF6AGYWvKqbxZZXW6YdFjPr1wBqMOX~Uo3IZJhFVGA7jjtt0xLHeQh~XrvtuEjRKiDVVI-shsuHPgd2AHTOQGFrDLU5x0h3N4sh1RCWb-~2fPpaMG8V72TtowX7BlwxkvLIorf6ckrEVamfR~W4tiQXjMBPs3RiGIAW~rOxZfnihKY-~M9IoMbQ27gIGfLj~Kqk36vrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195819&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering analysis of microbiomes based on cumulative bitscore comparisons. MDS was applied to the distance matrix calculated from reciprocal pairwise BLASTP analysis among all predicted gene products. The dots represent fecal samples from adults and children (blue), unweaned infants (red), Americans (green), and samples from other natural environments (orange). Whale falls 1–3, Sargasso and soil indicate the metadata for microbial communities from the deep sea,<sup>39</sup> surface seawater,<sup>26</sup> and the surface soil of a farm,<sup>39</sup> respectively.</p></div></div><div content-id="DSM018F1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01801.jpeg?Expires=2147483647&Signature=wLOOrvJi5xjfQM3kgCOLXeXA1PccpPf37JAHQYIjtqLjO2bQWdOXBJ6FitKCtbxRVyOzHRLvxEmu0AWJ9vN0SGrzlHDJwxGD6h7LBwzSbVPhw~7RfHF6cHi2KSQNmuHPGbScTJ~5mUU2s6u9J0i8ugd1L18wpD1bXCodJxaZkjhl0UrWpJRBZQzrngCj91aCyrIDLy3f-s6~BtZsOZGtuicW9VmWgu7A34gcmaXVqpY94cz~puCKyL7LIQABCrVHq3Su8cv--2S747T7~HWV8Ol5QPmie88acWVJJb5l7nFiFgXJQMMcGvCZSwRaRvtkx3gWVjtJzwu01~ejm~m2Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Clustering analysis of microbiomes based on cumulative bitscore comparisons. MDS was applied to the distance matrix calculated from reciprocal pairwise BLASTP analysis among all predicted gene products. The dots represent fecal samples from adults and children (blue), unweaned infants (red), Americans (green), and samples from other natural environments (orange). Whale falls 1–3, Sargasso and soil indicate the metadata for microbial communities from the deep sea,39 surface seawater,26 and the surface soil of a farm,39 respectively." path-from-xml="dsm01801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195819/dsm01801.jpeg" path-from-xml="dsm01801.jpeg" target="_blank">View large</a><a section="87195819" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01801.jpeg?Expires=2147483647&Signature=0BC0uhk8tSyL4UfSqgPbCPsM8-s4Atkzyu3z0SkD13uJ2QArJBToz1w6lBab6V~jpwyasKDu4gIDVg2qeCPIQNKZU3sokyDuFgC80rto3XRKw9GM5VtiTtYdbcGvOMPTVs1NtxY63TfzlBsF6AGYWvKqbxZZXW6YdFjPr1wBqMOX~Uo3IZJhFVGA7jjtt0xLHeQh~XrvtuEjRKiDVVI-shsuHPgd2AHTOQGFrDLU5x0h3N4sh1RCWb-~2fPpaMG8V72TtowX7BlwxkvLIorf6ckrEVamfR~W4tiQXjMBPs3RiGIAW~rOxZfnihKY-~M9IoMbQ27gIGfLj~Kqk36vrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195819&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Clustering analysis of microbiomes based on cumulative bitscore comparisons. MDS was applied to the distance matrix calculated from reciprocal pairwise BLASTP analysis among all predicted gene products. The dots represent fecal samples from adults and children (blue), unweaned infants (red), Americans (green), and samples from other natural environments (orange). Whale falls 1–3, Sargasso and soil indicate the metadata for microbial communities from the deep sea,<sup>39</sup> surface seawater,<sup>26</sup> and the surface soil of a farm,<sup>39</sup> respectively.</p></div></div><p>To determine the microbial composition at the genus level, we next conducted a BLASTP analysis of all the predicted genes against the genes in our ‘in-house extended NR database’ (see ‘Materials and methods’). With a threshold of 90% BLASTP identity, 17–43% of the predicted genes could be assigned to particular genera (35–65 genera, 121 in total) in the adults and children (Fig. <a reveal-id="DSM018F2" data-open="DSM018F2" class="link link-reveal link-table xref-fig">2</a>). A significantly higher proportion of genes (35–55%) was assignable (31–61 genera, 84 in total) in the unweaned infants, but, overall, the data indicated that the majority of gut microbes are as yet uncharacterized. We detected a total of 142 genera from the 13 samples in this analysis.</p>                        <a id="87195821" scrollto-destination="87195821"></a>
<div id="DSM018F2" data-id="DSM018F2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01802.jpeg?Expires=2147483647&Signature=LVbyHSPQrkde0UgI9gFlnOEo30wx7mcxmi8ugAKUByH0cOhSsD-my4GWUDx1NRhIRj1Oy8PCz3KvOBZyWOsWNdXfvyFRZrRub4RV1JH6JlGL6ZxhvQrBYauEZkNfgjU3mw-ws1bWTPZmxYl39dP2kbbmbXOq6MDEv8Ffe~Md3Aoiqj3bfQIfYt~2o3jBg23Rzt69k1~uZ8ljBfzysfTRBXnnqDlN~j0oul0JCKm4nYf-dfi4K~Dbsx7yuvEF0byWLJDuXifOnyvEQ~IJYBlw7rHOAHLZG7tvYCKhDK-7GeobGN-pDYvdPAWc9WZO4fAD47U3o~ch5FpFAM77UnaPlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Compositional view of human intestinal microbiomes. A compositional view of microbiomes based on the taxonomic assignment of protein-coding genes is shown. The stacked bars represent the compositions of each sample estimated from the results of BLASTP analysis with a 90% threshold identity. ‘Others’ includes the genera whose proportions were less than 1% in any of the samples. The data for the fecal samples from two American adults (‘Sub. 7’ and ‘Sub. 8’) are also shown." path-from-xml="dsm01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195821/dsm01802.jpeg" path-from-xml="dsm01802.jpeg" target="_blank">View large</a><a section="87195821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01802.jpeg?Expires=2147483647&Signature=IAsVWpE1eQI2d0x1CiunuT-fJsdWj1y2iysTaS-WCztjOsy0CFW18~Z1~OjEWGekKnwsT9i8xGGJEAqMpEaBb093H4sJW6mulJawgrF9qrNcDt0LaVtpwiobnLwikY4Q2mqAIb1q59wdC7S5qN7yeoH5cBEoyIdx4q1FeReQS23NUIiulRXYrETeL~eBmQ9YNlv9oroHujn3I-2xh2Rc9ophT~VavipiplDB-ZcPn2iSm1JzAYw3iA4VnbPJ53gvfNHRkhYHwpbjp57OBZnAi7WJZMeJ4ngKbxiFUS1hdqXF43dmcFQa0RjkLq0EyRpb2-cQO~SNUqItM1FC8x7DPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195821&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Compositional view of human intestinal microbiomes. A compositional view of microbiomes based on the taxonomic assignment of protein-coding genes is shown. The stacked bars represent the compositions of each sample estimated from the results of BLASTP analysis with a 90% threshold identity. ‘Others’ includes the genera whose proportions were less than 1% in any of the samples. The data for the fecal samples from two American adults (‘Sub. 7’ and ‘Sub. 8’) are also shown.</p></div></div><div content-id="DSM018F2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01802.jpeg?Expires=2147483647&Signature=LVbyHSPQrkde0UgI9gFlnOEo30wx7mcxmi8ugAKUByH0cOhSsD-my4GWUDx1NRhIRj1Oy8PCz3KvOBZyWOsWNdXfvyFRZrRub4RV1JH6JlGL6ZxhvQrBYauEZkNfgjU3mw-ws1bWTPZmxYl39dP2kbbmbXOq6MDEv8Ffe~Md3Aoiqj3bfQIfYt~2o3jBg23Rzt69k1~uZ8ljBfzysfTRBXnnqDlN~j0oul0JCKm4nYf-dfi4K~Dbsx7yuvEF0byWLJDuXifOnyvEQ~IJYBlw7rHOAHLZG7tvYCKhDK-7GeobGN-pDYvdPAWc9WZO4fAD47U3o~ch5FpFAM77UnaPlw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Compositional view of human intestinal microbiomes. A compositional view of microbiomes based on the taxonomic assignment of protein-coding genes is shown. The stacked bars represent the compositions of each sample estimated from the results of BLASTP analysis with a 90% threshold identity. ‘Others’ includes the genera whose proportions were less than 1% in any of the samples. The data for the fecal samples from two American adults (‘Sub. 7’ and ‘Sub. 8’) are also shown." path-from-xml="dsm01802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195821/dsm01802.jpeg" path-from-xml="dsm01802.jpeg" target="_blank">View large</a><a section="87195821" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01802.jpeg?Expires=2147483647&Signature=IAsVWpE1eQI2d0x1CiunuT-fJsdWj1y2iysTaS-WCztjOsy0CFW18~Z1~OjEWGekKnwsT9i8xGGJEAqMpEaBb093H4sJW6mulJawgrF9qrNcDt0LaVtpwiobnLwikY4Q2mqAIb1q59wdC7S5qN7yeoH5cBEoyIdx4q1FeReQS23NUIiulRXYrETeL~eBmQ9YNlv9oroHujn3I-2xh2Rc9ophT~VavipiplDB-ZcPn2iSm1JzAYw3iA4VnbPJ53gvfNHRkhYHwpbjp57OBZnAi7WJZMeJ4ngKbxiFUS1hdqXF43dmcFQa0RjkLq0EyRpb2-cQO~SNUqItM1FC8x7DPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195821&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Compositional view of human intestinal microbiomes. A compositional view of microbiomes based on the taxonomic assignment of protein-coding genes is shown. The stacked bars represent the compositions of each sample estimated from the results of BLASTP analysis with a 90% threshold identity. ‘Others’ includes the genera whose proportions were less than 1% in any of the samples. The data for the fecal samples from two American adults (‘Sub. 7’ and ‘Sub. 8’) are also shown.</p></div></div><p>Despite the low proportion of assigned genes, their taxonomic distribution indicated a clear compositional change after weaning. In the adults and weaned children, the major constituents were always <em>Bacteroides</em>, followed by several genera belonging to the division Firmicutes, such as <em>Eubacterium</em>, <em>Ruminococcus</em>, and <em>Clostridium</em>, and the genus <em>Bifidobacterium</em>. In the infants, <em>Bifidobacterium</em> and/or a few genera from the family <em>Enterobacteriaceae</em>, such as <em>Escherichia</em>, <em>Raoultella</em>, and <em>Klebsiella</em>, were the major constituents. A significant level of inter-individual variation was observed also among the adults and children, but there was a much higher variation among the unweaned infants.</p><p>To further evaluate the microbial composition at the species level, we examined the intra-genus diversity of <em>Bacteroides</em> and <em>Bifidobacterium</em>, the most dominant genera in the adults and children and in the infants, respectively. Taking advantage of the number of available genome sequences belonging to these two genera (11 species from <em>Bacteroides</em>, five from <em>Bifidobacterium</em>), we performed a mapping analysis of the shotgun reads to these genomes using the BLASTN program<sup><a reveal-id="DSM018C34" data-open="DSM018C34" class="link link-ref link-reveal xref-bibr">34</a></sup> with a threshold of ≥95% identity and ≥150 bp aligned length. The analysis demonstrated that, for each infant, more than 80% of the shotgun reads were mapped to <em>Bifidobacterium</em> derived from a single species or genome (Supplementary Table S4). In contrast, the shotgun reads from the adults and children were mapped to multiple <em>Bacteroides</em> species except for In-A, suggesting a compositional complexity of this genus in human gut microbiota.</p><p>Together with the results of the shotgun sequence assembly (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>), the data from these compositional analyses showed a clear structural difference between the microbiota of the unweaned infants and those of the adults and weaned children. Infant microbiota were dominated by a few microbial species or strains, exhibiting rather simple structures, but showed a remarkable inter-individual variation. In contrast, most microbiota of the adults and children were much more complex in species composition, but exhibited high levels of overall sequence similarity between the samples. In In-A, the shotgun read sequences apparently derived from a few species of <em>Bacteroides</em> and <em>Eubacterium</em> were notably dominated (data not shown), accounting for the significantly shorter non-redundant sequence of In-A (see Subsection 3.1 and Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>It should also be noted that the samples from Japanese and American<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> adults differed significantly in composition, particularly in terms of <em>Bacteroides</em> and archaeal species (Fig. <a reveal-id="DSM018F2" data-open="DSM018F2" class="link link-reveal link-table xref-fig">2</a>). The gut microbiomes from two American samples contained very few sequences and genes assigned to <em>Bacteroides</em> species and a significant number of sequences and genes assigned to an archaeal species, whereas the gut microbiomes from the Japanese samples contained a high ratio of sequences and genes assigned to <em>Bacteroides</em> species and almost no archaeal sequences and genes. Further studies should establish the reasons for these intriguing differences, which could be due to various factors including the genetic background and dietary style of the hosts, but also to differences in the experimental conditions between the two studies.</p>                            <h3 scrollto-destination=87195826 id="87195826" class="section-title" data-legacy-id=s3c>3.3. Functional assignment of predicted genes</h3>
<p>Functional assignment of predicted genes (662,548 in total) was made on the basis of BLASTP analysis against the ‘reference dataset for COG assignment’ (see ‘Materials and methods’). By this analysis, about 48% of the predicted genes were assigned to a total of 3,268 COGs (Table <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a>). The number of COGs identified in the infant microbiomes showed remarkable inter-individual variation (1617–2857 COGs), in contrast to those of the adults and children (2355–2921 COGs). Also, the number of orthologous genes belonging to each COG in the infants was on average about two-third of that observed in the adults and children. These results indicate that the gene repertoires in the gut microbiomes are more variable and functionally less redundant in infants than in adults and children.</p><p>To explore the functional characteristics of human intestinal microbiota, we looked for significantly over- or under-represented COGs in gut microbiomes when compared with Ref-DB (see ‘Materials and methods’). As shown in Fig. <a reveal-id="DSM018F3" data-open="DSM018F3" class="link link-reveal link-table xref-fig">3</a>, human gut microbiomes showed patterns distinct from those of other environments such as sea and soil. The over-representation of COGs classified into the ‘Carbohydrate transport and metabolism’ category and the under-representation of those for ‘Lipid transport and metabolism’ were observed in all the human gut microbiomes examined in this study. However, a clear difference was observed between the adults/children and the unweaned infants. The gut microbiomes from the adults and children exhibited a uniform pattern, and the over-representation of COGs for ‘Defense mechanisms’ and under-representation of ‘Cell motility’, ‘Secondary metabolites biosynthesis, transport and catabolism’ and ‘Post-translational modification, protein turnover, chaperones’ were remarkable (Fig. <a reveal-id="DSM018F3" data-open="DSM018F3" class="link link-reveal link-table xref-fig">3</a>B). In contrast, the infant microbiomes showed variable patterns (Fig. <a reveal-id="DSM018F3" data-open="DSM018F3" class="link link-reveal link-table xref-fig">3</a>C). The enrichment values of all the COGs in each microbiome are shown in Supplementary Table S5.</p>                        <a id="87195829" scrollto-destination="87195829"></a>
<div id="DSM018F3" data-id="DSM018F3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01803.jpeg?Expires=2147483647&Signature=j8yryPV7Y9A5LkMUHfHdyVK~7IMphmAViWgpUKYiLk6TXM8SRm2yHelFH-IyYY50b~edaXW46DvKSprSRtmQS~4MYlOHYqNwdPg~--f8Nq86SxoEykZl7sRoaozLv1BVUdeRRIkBSter7Lkqi3AeeV0pDpRXv8WpKpqIoYgklb1Y8rNXTTcM-4wYtaFAjFAfHTlozk-kPAiciAuG5yhfi-XWReLliUcaGWLutcXjbpwLOebN4VygiYEHDCLPo~o6RdM5UMvPXa5A8iowAP36hu3hGhOf~vDUhnac2I2LIigmpvG4arb1CgcOZ~ce3EMsSUd-PKznXs-qLg~FRHIe9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of the COG assignment of predicted genes. (A) Comparison of the distribution patterns of COG-assigned genes between each type of microbiome and Ref-DB (for Ref-DB, see ‘Materials and methods’). Fecal samples from Japanese adults and children (nine samples), Japanese infants (four) and American adults (two), and three samples from whale fall were each averaged. (B) Distribution of COG-assigned genes in the nine microbiomes from adults and children. (C) Distribution of COG-assigned genes in the four microbiomes from unweaned infants; C: Energy production and conversion; D: Cell cycle control, mitosis, and meiosis; E: Amino acid transport and metabolism; F: Nucleotide transport and metabolism; G: Carbohydrate transport and metabolism; H: Coenzyme transport and metabolism; I: Lipid transport and metabolism; J: Translation; K: Transcription; L: Replication, recombination and repair; M: Cell wall/membrane biogenesis; N: Cell motility; O: Post-translational modification, protein turnover, chaperones; P: Inorganic ion transport and metabolism; Q: Secondary metabolites biosynthesis, transport and catabolism; R: General function prediction only; S: Function unknown; T: Signal transduction mechanisms; U: Intracellular trafficking and secretion; V: Defense mechanisms. The genes not assignable to any COGs are not shown in this figure." path-from-xml="dsm01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195829/dsm01803.jpeg" path-from-xml="dsm01803.jpeg" target="_blank">View large</a><a section="87195829" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01803.jpeg?Expires=2147483647&Signature=e~vsO8LJMl1kzwX2sGXCmZqXvRWbCDKBRDdTRwl2Uc0-FnhaQ3mJisZ9oetP2qPXL7PL79C6HHwXLQ9LwVQO93pGDUePmKkLewgrsS36nEOQBxwLg8bHlUycUMr-dAO-jOIgRFneQArgcIsHRYBvGf0UZMTfiW8mNcDlLjbEj0z4aEoZRRgyX-8q5nxngT8XB4oFRF4pEAhfSIKOctC7SX8H5Wg9wiYfJDHzsk~7ZhcaNjsT6P69vq0tRTkfIu-NSWevExap94XJyeZsSWfRp9tyHerb3HcvZekXvhDr4WKUDphuaOSkIh08lQH42AIW8YOUFdaHgWjRV4En0jHuDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195829&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of the COG assignment of predicted genes. (<strong>A</strong>) Comparison of the distribution patterns of COG-assigned genes between each type of microbiome and Ref-DB (for Ref-DB, see ‘Materials and methods’). Fecal samples from Japanese adults and children (nine samples), Japanese infants (four) and American adults (two), and three samples from whale fall were each averaged. (<strong>B</strong>) Distribution of COG-assigned genes in the nine microbiomes from adults and children. (<strong>C</strong>) Distribution of COG-assigned genes in the four microbiomes from unweaned infants; C: Energy production and conversion; D: Cell cycle control, mitosis, and meiosis; E: Amino acid transport and metabolism; F: Nucleotide transport and metabolism; G: Carbohydrate transport and metabolism; H: Coenzyme transport and metabolism; I: Lipid transport and metabolism; J: Translation; K: Transcription; L: Replication, recombination and repair; M: Cell wall/membrane biogenesis; N: Cell motility; O: Post-translational modification, protein turnover, chaperones; P: Inorganic ion transport and metabolism; Q: Secondary metabolites biosynthesis, transport and catabolism; R: General function prediction only; S: Function unknown; T: Signal transduction mechanisms; U: Intracellular trafficking and secretion; V: Defense mechanisms. The genes not assignable to any COGs are not shown in this figure.</p></div></div><div content-id="DSM018F3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01803.jpeg?Expires=2147483647&Signature=j8yryPV7Y9A5LkMUHfHdyVK~7IMphmAViWgpUKYiLk6TXM8SRm2yHelFH-IyYY50b~edaXW46DvKSprSRtmQS~4MYlOHYqNwdPg~--f8Nq86SxoEykZl7sRoaozLv1BVUdeRRIkBSter7Lkqi3AeeV0pDpRXv8WpKpqIoYgklb1Y8rNXTTcM-4wYtaFAjFAfHTlozk-kPAiciAuG5yhfi-XWReLliUcaGWLutcXjbpwLOebN4VygiYEHDCLPo~o6RdM5UMvPXa5A8iowAP36hu3hGhOf~vDUhnac2I2LIigmpvG4arb1CgcOZ~ce3EMsSUd-PKznXs-qLg~FRHIe9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of the COG assignment of predicted genes. (A) Comparison of the distribution patterns of COG-assigned genes between each type of microbiome and Ref-DB (for Ref-DB, see ‘Materials and methods’). Fecal samples from Japanese adults and children (nine samples), Japanese infants (four) and American adults (two), and three samples from whale fall were each averaged. (B) Distribution of COG-assigned genes in the nine microbiomes from adults and children. (C) Distribution of COG-assigned genes in the four microbiomes from unweaned infants; C: Energy production and conversion; D: Cell cycle control, mitosis, and meiosis; E: Amino acid transport and metabolism; F: Nucleotide transport and metabolism; G: Carbohydrate transport and metabolism; H: Coenzyme transport and metabolism; I: Lipid transport and metabolism; J: Translation; K: Transcription; L: Replication, recombination and repair; M: Cell wall/membrane biogenesis; N: Cell motility; O: Post-translational modification, protein turnover, chaperones; P: Inorganic ion transport and metabolism; Q: Secondary metabolites biosynthesis, transport and catabolism; R: General function prediction only; S: Function unknown; T: Signal transduction mechanisms; U: Intracellular trafficking and secretion; V: Defense mechanisms. The genes not assignable to any COGs are not shown in this figure." path-from-xml="dsm01803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195829/dsm01803.jpeg" path-from-xml="dsm01803.jpeg" target="_blank">View large</a><a section="87195829" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01803.jpeg?Expires=2147483647&Signature=e~vsO8LJMl1kzwX2sGXCmZqXvRWbCDKBRDdTRwl2Uc0-FnhaQ3mJisZ9oetP2qPXL7PL79C6HHwXLQ9LwVQO93pGDUePmKkLewgrsS36nEOQBxwLg8bHlUycUMr-dAO-jOIgRFneQArgcIsHRYBvGf0UZMTfiW8mNcDlLjbEj0z4aEoZRRgyX-8q5nxngT8XB4oFRF4pEAhfSIKOctC7SX8H5Wg9wiYfJDHzsk~7ZhcaNjsT6P69vq0tRTkfIu-NSWevExap94XJyeZsSWfRp9tyHerb3HcvZekXvhDr4WKUDphuaOSkIh08lQH42AIW8YOUFdaHgWjRV4En0jHuDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195829&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of the COG assignment of predicted genes. (<strong>A</strong>) Comparison of the distribution patterns of COG-assigned genes between each type of microbiome and Ref-DB (for Ref-DB, see ‘Materials and methods’). Fecal samples from Japanese adults and children (nine samples), Japanese infants (four) and American adults (two), and three samples from whale fall were each averaged. (<strong>B</strong>) Distribution of COG-assigned genes in the nine microbiomes from adults and children. (<strong>C</strong>) Distribution of COG-assigned genes in the four microbiomes from unweaned infants; C: Energy production and conversion; D: Cell cycle control, mitosis, and meiosis; E: Amino acid transport and metabolism; F: Nucleotide transport and metabolism; G: Carbohydrate transport and metabolism; H: Coenzyme transport and metabolism; I: Lipid transport and metabolism; J: Translation; K: Transcription; L: Replication, recombination and repair; M: Cell wall/membrane biogenesis; N: Cell motility; O: Post-translational modification, protein turnover, chaperones; P: Inorganic ion transport and metabolism; Q: Secondary metabolites biosynthesis, transport and catabolism; R: General function prediction only; S: Function unknown; T: Signal transduction mechanisms; U: Intracellular trafficking and secretion; V: Defense mechanisms. The genes not assignable to any COGs are not shown in this figure.</p></div></div><p>Profiling analysis based on the COG enrichment values calculated for each microbiome further demonstrated that, while the gut microbiomes of the adults and children showed similar profiles, those of the infants had distinct and more variable profiles (Fig. <a reveal-id="DSM018F4" data-open="DSM018F4" class="link link-reveal link-table xref-fig">4</a>; also see Supplementary Fig. S1). It may be noteworthy that neither this analysis nor the overall sequence similarity analysis shown in Fig. <a reveal-id="DSM018F1" data-open="DSM018F1" class="link link-reveal link-table xref-fig">1</a> provided any conclusive evidence for the resemblance of the genomic features of gut microbiomes among family members and within the sexes.</p>                        <a id="87195831" scrollto-destination="87195831"></a>
<div id="DSM018F4" data-id="DSM018F4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01804.jpeg?Expires=2147483647&Signature=SSxFo7i7cOxz8hGa6YcjMVIRL~F5e12XpiQcDDiAdVbxIPiJtZCw05eQfGOGwoVQUNolkMR32SeBxbaTt4kbhWb5mfXVvWvGe675jwUsMwjGCXDi0xnWA5c7ZckYNbN1Zg5VOVIHdthffftjTdnkmFEgB4ChFN6sCNjFndVPjmb35DHDP9l-6X3TrUARSuwhlOKv3QNIkM5-qtdqKwyCyFcDBAwvAClrF3S4EAAROet6gOWvxoNYVSqOJipQ4pcj19BHSFoQN5iKOujUyQay9AD5TY4Gun7HIgMQsuuWIv-3RLleXv-kaqTwyAUr6HmPT5drM71lWBylLgxLpv5aaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between human intestinal microbiomes and other-environmental microbiomes based on their functional profiles. The result of a clustering analysis of microbiomes based on the enrichment values of each COG calculated for each microbiome is shown. The matrix was clustered independently in the samples and COGs using the pairwise complete-linkage hierarchical clustering of the uncentered correlation (cluster-1.31). Also, see Supplementary Fig. S1." path-from-xml="dsm01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195831/dsm01804.jpeg" path-from-xml="dsm01804.jpeg" target="_blank">View large</a><a section="87195831" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01804.jpeg?Expires=2147483647&Signature=PmChYxFiWHvj1QvHTfnAF~TAh7zZMI4ogcw07XCMgP4cVYGlCg0MUoq~z41UWmRZrYGQCX983Fs9iQuVInyiR5hLYqctQDhqwj1qY73zjlioxjoEVp-mi~fnOfXIqUQ7SEOHABipB8reujlhyqKWy-tgGTueGKHJ9BXd0tehj4cOwngqPazkRLLeSdsBeer1nU8IbTQKX~p0CozEHaM03YaPOrjXcTAxRSvYQ9aPkDDFRGBMtJLa5kFm~534KM3UlNmXxTywyUgTqq6-K1I4KOQJxAe8pQT9RbBX~OYYmd55N3z~4XBIK8Qw1MDK8Bi648g6q3LwT9VEFywjRdLvAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195831&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between human intestinal microbiomes and other-environmental microbiomes based on their functional profiles. The result of a clustering analysis of microbiomes based on the enrichment values of each COG calculated for each microbiome is shown. The matrix was clustered independently in the samples and COGs using the pairwise complete-linkage hierarchical clustering of the uncentered correlation (cluster-1.31). Also, see Supplementary Fig. S1.</p></div></div><div content-id="DSM018F4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01804.jpeg?Expires=2147483647&Signature=SSxFo7i7cOxz8hGa6YcjMVIRL~F5e12XpiQcDDiAdVbxIPiJtZCw05eQfGOGwoVQUNolkMR32SeBxbaTt4kbhWb5mfXVvWvGe675jwUsMwjGCXDi0xnWA5c7ZckYNbN1Zg5VOVIHdthffftjTdnkmFEgB4ChFN6sCNjFndVPjmb35DHDP9l-6X3TrUARSuwhlOKv3QNIkM5-qtdqKwyCyFcDBAwvAClrF3S4EAAROet6gOWvxoNYVSqOJipQ4pcj19BHSFoQN5iKOujUyQay9AD5TY4Gun7HIgMQsuuWIv-3RLleXv-kaqTwyAUr6HmPT5drM71lWBylLgxLpv5aaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between human intestinal microbiomes and other-environmental microbiomes based on their functional profiles. The result of a clustering analysis of microbiomes based on the enrichment values of each COG calculated for each microbiome is shown. The matrix was clustered independently in the samples and COGs using the pairwise complete-linkage hierarchical clustering of the uncentered correlation (cluster-1.31). Also, see Supplementary Fig. S1." path-from-xml="dsm01804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195831/dsm01804.jpeg" path-from-xml="dsm01804.jpeg" target="_blank">View large</a><a section="87195831" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01804.jpeg?Expires=2147483647&Signature=PmChYxFiWHvj1QvHTfnAF~TAh7zZMI4ogcw07XCMgP4cVYGlCg0MUoq~z41UWmRZrYGQCX983Fs9iQuVInyiR5hLYqctQDhqwj1qY73zjlioxjoEVp-mi~fnOfXIqUQ7SEOHABipB8reujlhyqKWy-tgGTueGKHJ9BXd0tehj4cOwngqPazkRLLeSdsBeer1nU8IbTQKX~p0CozEHaM03YaPOrjXcTAxRSvYQ9aPkDDFRGBMtJLa5kFm~534KM3UlNmXxTywyUgTqq6-K1I4KOQJxAe8pQT9RbBX~OYYmd55N3z~4XBIK8Qw1MDK8Bi648g6q3LwT9VEFywjRdLvAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195831&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between human intestinal microbiomes and other-environmental microbiomes based on their functional profiles. The result of a clustering analysis of microbiomes based on the enrichment values of each COG calculated for each microbiome is shown. The matrix was clustered independently in the samples and COGs using the pairwise complete-linkage hierarchical clustering of the uncentered correlation (cluster-1.31). Also, see Supplementary Fig. S1.</p></div></div>                            <h3 scrollto-destination=87195832 id="87195832" class="section-title" data-legacy-id=s3d>3.4. A gene set commonly enriched in adult-type gut microbiomes</h3>
<p>COGs that are commonly enriched in the microbiomes of all adults and children were searched, and 237 COGs met the following criteria: (i) the average enrichment value exceeds 2.0 and (ii) the enrichment value in all subjects exceeds 1.0 (Fig. <a reveal-id="DSM018F5" data-open="DSM018F5" class="link link-reveal link-table xref-fig">5</a> and Supplementary Table S6). When the samples from two adult Americans were included in the analysis, 79% (188) of the 237 COGs still met the above criteria, even though the American data contained, unusually, only a few sequences derived from <em>Bacteroides</em> species (Fig. <a reveal-id="DSM018F2" data-open="DSM018F2" class="link link-reveal link-table xref-fig">2</a>). In contrast, only 5–10% of these COGs exhibited an enrichment value of &gt;2.0 in the microbiomes of other environments. Therefore, these COGs are specifically enriched in adult-type gut microbiomes, and thus may encode important functions for the gut microbiota itself as well as for its host.</p>                        <a id="87195834" scrollto-destination="87195834"></a>
<div id="DSM018F5" data-id="DSM018F5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01805.jpeg?Expires=2147483647&Signature=XbgvOrhiyFpSq5qCTO11j8~~Km7NCJlM0gYBdvPJzMxeLL8uXcmxvYH9nid8w5m0BgTQqE4FkkoYi-iZKY99aE~AzerDao5uRlOT6URusAyf1JsRe7HfII7dqX1MwHDkdRgEIs7BoQ3txU63uRaWGs5r0oOJrgcEEiSqC6xw5BeJy1~8HZfie6ClRBec~2jge6Xpa~oHGS9l991X~n1HqA4tVs2~YoxhjUKwsn303hOofyAj1NOww3bsm~u916C0BGM7bPgqTKKLk2YRcV0kKX18aYdbbnyLgMGvNIaNmSJrb6tBT1v7-~rVZuTtH4p2QHw1P2xTo1N-2-YXIQ11LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional distribution of commonly enriched COGs. The functional distribution of commonly enriched COGs in adult/child microbiomes (‘A-gutCEGs’), in infant microbiomes (‘I-gutCEGs’), or in both types of microbiomes is shown: see Fig. 3 for the COG categorization." path-from-xml="dsm01805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195834/dsm01805.jpeg" path-from-xml="dsm01805.jpeg" target="_blank">View large</a><a section="87195834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01805.jpeg?Expires=2147483647&Signature=ChGJZvK29pNfMfOLE5-ECwfxOEgriYbCQGwrncw6BgCU-ZxN1DoKjwXPpB6rx9bSmx9LC85CjNHfVeIhjcBOKGGDdhTE9L8Mqip0E1abijqtJz4ZnjpxFzb~Z-IZRk-uUrpJuQY9bGIoWb1wnBCmd2nsMo~UyEsk~1ZCj~cX0sCyShwskTE9v38m0M81kHFy2oyrJD3K2DY9kv1pWVl7LkNn~Aniwt6vbtNTHmTV863C3CYNUx~loG7-tOVHp0DdFOGxLyeNrnk2i8KlyT1YzLK0ksnSuuwUekJScpF0QPldREfztfjwI3jpzcomcvXBxDDMq78Z6TYSHZDIjTEigw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195834&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional distribution of commonly enriched COGs. The functional distribution of commonly enriched COGs in adult/child microbiomes (‘A-gutCEGs’), in infant microbiomes (‘I-gutCEGs’), or in both types of microbiomes is shown: see Fig. 3 for the COG categorization.</p></div></div><div content-id="DSM018F5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01805.jpeg?Expires=2147483647&Signature=XbgvOrhiyFpSq5qCTO11j8~~Km7NCJlM0gYBdvPJzMxeLL8uXcmxvYH9nid8w5m0BgTQqE4FkkoYi-iZKY99aE~AzerDao5uRlOT6URusAyf1JsRe7HfII7dqX1MwHDkdRgEIs7BoQ3txU63uRaWGs5r0oOJrgcEEiSqC6xw5BeJy1~8HZfie6ClRBec~2jge6Xpa~oHGS9l991X~n1HqA4tVs2~YoxhjUKwsn303hOofyAj1NOww3bsm~u916C0BGM7bPgqTKKLk2YRcV0kKX18aYdbbnyLgMGvNIaNmSJrb6tBT1v7-~rVZuTtH4p2QHw1P2xTo1N-2-YXIQ11LA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional distribution of commonly enriched COGs. The functional distribution of commonly enriched COGs in adult/child microbiomes (‘A-gutCEGs’), in infant microbiomes (‘I-gutCEGs’), or in both types of microbiomes is shown: see Fig. 3 for the COG categorization." path-from-xml="dsm01805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195834/dsm01805.jpeg" path-from-xml="dsm01805.jpeg" target="_blank">View large</a><a section="87195834" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01805.jpeg?Expires=2147483647&Signature=ChGJZvK29pNfMfOLE5-ECwfxOEgriYbCQGwrncw6BgCU-ZxN1DoKjwXPpB6rx9bSmx9LC85CjNHfVeIhjcBOKGGDdhTE9L8Mqip0E1abijqtJz4ZnjpxFzb~Z-IZRk-uUrpJuQY9bGIoWb1wnBCmd2nsMo~UyEsk~1ZCj~cX0sCyShwskTE9v38m0M81kHFy2oyrJD3K2DY9kv1pWVl7LkNn~Aniwt6vbtNTHmTV863C3CYNUx~loG7-tOVHp0DdFOGxLyeNrnk2i8KlyT1YzLK0ksnSuuwUekJScpF0QPldREfztfjwI3jpzcomcvXBxDDMq78Z6TYSHZDIjTEigw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195834&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional distribution of commonly enriched COGs. The functional distribution of commonly enriched COGs in adult/child microbiomes (‘A-gutCEGs’), in infant microbiomes (‘I-gutCEGs’), or in both types of microbiomes is shown: see Fig. 3 for the COG categorization.</p></div></div><p>Pyruvate-formate lyase (COG1882), which catalyzes the non-oxidative conversion of pyruvate to formate and acetyl-coenzyme A, was enriched. Unexpectedly, however, genes for the formate hydrogenlyase system that decomposes formate to CO<sub>2</sub> and H<sub>2</sub> were rather under-represented. In this regard, of interest is the enrichment of formyltetrahydrofolate synthetase (COG2759), methenyl tetrahydrofolate cyclohydrolase (COG3404), and methionine synthase (COG1410), all of which are enzymes involved in the regulation of one carbon pool by folate. Their enrichment may suggest that one carbon unit of formate can be utilized effectively by the gut microbiota in the folate-mediated cycle of one carbon pool. In contrast to the enrichment of enzymes for anaerobic pyruvate metabolism, the pyruvate dehydrogenase complex (COG2069) was profoundly depleted. All components of the oxidative tricarboxylic acid (TCA) cycle and the membrane respiratory chain (with the exception of NADH:ubiquinone oxidoreductase) were also significantly under-represented in all subjects, but phosphoenolpyruvate carboxykinase (COG1866) and pyruvate carboxyltransferase (COG5016), which generate oxaloacetate, an entry substrate to the TCA cycle in the reductive pathway, were enriched. Together with the striking depletion of most gene families whose products scavenge oxygen radicals, these findings reflect well the fact that the adult gut ecosystem is a kingdom of strict anaerobes.</p><p>The enrichment of carbohydrate metabolism genes was also striking: 24% (53 COGs) of the commonly enriched COGs had this function. At least 14 families of glycosyl hydrolases for plant-derived dietary polysaccharides and host tissue-derived proteoglycans or glycoconjugates were enriched in the adults. In addition, many enzymes involved in the metabolism of mono- or disaccharides released by these glycosyl hydrolases, such as <span class="small-caps">l</span>-fucose isomerase (COG2407), <span class="small-caps">l</span>-arabinose isomerase (COG2160), and galactokinase (COG0153), were also over-represented. Several peptidase families (COG1362, COG2195, COG3340, and COG3579) were also enriched, but most genes for fatty-acid metabolism were selectively reduced in number. These findings support the notion that the colonic microbiota utilizes otherwise indigestible polysaccharides and peptides as major resources for energy production and biosynthesis of cellular components.<sup><a reveal-id="DSM018C7" data-open="DSM018C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSM018C8" data-open="DSM018C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The enrichment of phosphoenolpyruvate carboxykinase (COG1866), glycogen synthase (COG0297) and ADP-glucose pyrophosphorylase (COG0448) suggests that energy storage is also an important activity of adult-type gut microbiota. This activity may be required for the gut microbiota to cope with intermittent nutrient supply in the adult gut. The enrichment of antimicrobial peptide transporters (COG0577 and COG1132) and a multidrug efflux pump (COG0534) is also of interest. Host intestinal cells produce various cationic antimicrobial peptides (CAMPs), such as beta-defensins.<sup><a reveal-id="DSM018C43" data-open="DSM018C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Many microorganisms also produce CAMPs to compete with other microbes sharing the same niche. The enrichment of antimicrobial peptide transporters and the multidrug efflux pump may play a primary role in the stable colonization of gut microbes in the adult intestine by conferring resistance to CAMPs.</p><p>The enrichment of several enzymes for DNA repair is also noteworthy (Supplementary Table S6). These enzymes may be needed to repair microbial DNA damage caused by genotoxic substances, such as nitrosamines and heterocyclic amines contained in ingested foods and secondary bile acids and nitroso compounds synthesized in the intestine via gut microbiota-involving processes.<sup><a reveal-id="DSM018C44" data-open="DSM018C44" class="link link-ref link-reveal xref-bibr">44</a></sup> It is conceivable that not only the host cells but also intestinal microbes are constantly exposed to such genotoxic compounds.</p><p>Another distinguishing feature of the adult-type microbiota is the striking depletion of genes for the biosynthesis of flagella and chemotaxis (Supplementary Table S5). This implies that motility and chemotaxis are not required for the intestinal microbes to persist in the gut, where the contents are constantly stirred by peristalsis. Rather, flagellated microbes may be easily eliminated by the host immune system because flagella are highly immunogenic. Abnegation of motility may be another adaptation mechanism of gut microbes to the intestinal environment.</p>                            <h3 scrollto-destination=87195839 id="87195839" class="section-title" data-legacy-id=s3e>3.5. A gene set commonly enriched in infant-type gut microbiomes</h3>
<p>Despite the high inter-individual variation, 136 COGs were found to be commonly enriched in the infant microbiomes (Fig. <a reveal-id="DSM018F5" data-open="DSM018F5" class="link link-reveal link-table xref-fig">5</a> and Supplementary Table S6). Of these, 58 were also over-represented in the adult/child microbiomes.</p><p>Genes for anaerobic energy production were also enriched in infants, but genes for the pyruvate dehydrogenase complex and all components of the oxidative TCA cycle were present in the infants at a frequency similar to that in Ref-DB. These findings may reflect the compositional feature of the infant gut microbiota, which contains considerable numbers of facultative anaerobes (Fig. <a reveal-id="DSM018F2" data-open="DSM018F2" class="link link-reveal link-table xref-fig">2</a>).</p><p>In infant microbiomes, about 35% (<a reveal-id="DSM018C47" data-open="DSM018C47" class="link link-ref link-reveal xref-bibr">47</a>) of the 136 enriched COGs were for ‘Carbohydrate transport and metabolism,’ including 12 families of glycosyl hydrolases, nine of which were enriched also in the adult gut microbiomes. Unexpectedly, they included several enzymes that degrade non-digestible polysaccharides of plant origin, such as pullulanase and related glycosidases (COG1523), arabinogalactan endo-1,4-beta-galactosidase (COG3867), and endopolygalacturonase (COG5434). These enzymes may act to degrade oligosaccharides in breast milk or host-derived proteoglycans like mucin to maintain the functional homeostasis of gut epithelia.<sup><a reveal-id="DSM018C45" data-open="DSM018C45" class="link link-ref link-reveal xref-bibr">45</a></sup> It is also possible that the gut microbiota is ready to utilize plant-derived polysaccharides to some extent before weaning.</p><p>The over-representation of various transport systems was also characteristic to infants, with 22% (<a reveal-id="DSM018C29" data-open="DSM018C29" class="link link-ref link-reveal xref-bibr">29</a>) of the 136 enriched COGs being transporters. In particular, the enrichment of phosphotransferase systems that mediate active sugar transport was remarkable. This prokaryote-specific transport system may play a central role in the uptake of lactose and other easily digestible simple sugars rich in breast milk. The over-representation of other transporters may also be advantageous to the microbes in the infant intestine because breast milk contains many other essential nutrients such as amino acids, long-chain fatty acids, nucleotides, vitamins, and minerals in a readily available form. The difference in diet between adults and unweaned infants appears to affect other functional properties as well. For instance, the genes for defense mechanisms and DNA repair that were over-represented in the adults were not so in the infants.</p>                            <h3 scrollto-destination=87195844 id="87195844" class="section-title" data-legacy-id=s3f>3.6. A CTn family amplified in the intestine</h3>
<p>Although this remains largely unproven, the distal colon has been regarded an ecologically suitable site for horizontal gene transfer (HGT) between microorganisms due to its high microbial cell density.<sup><a reveal-id="DSM018C2" data-open="DSM018C2" class="link link-ref link-reveal xref-bibr">2</a></sup> We identified many gene families related to transposases and bacteriophages in the metagenomic data, but their over-representation was noted only in certain individuals (Supplementary Table S5). An exception to this was a set of genes homologous to those on Tn<em>1549</em>-like CTns, which was notably enriched in most of the gut microbiomes analysed here (Fig. <a reveal-id="DSM018F6" data-open="DSM018F6" class="link link-reveal link-table xref-fig">6</a>). Tn<em>1549</em> was originally identified in an <em>E. faecalis</em> strain,<sup><a reveal-id="DSM018C30" data-open="DSM018C30" class="link link-ref link-reveal xref-bibr">30</a></sup> thereafter, its relatives have been identified in another <em>E. faecalis</em> strain, <em>Butyrivibrio</em>, <em>Clostridium</em>, and <em>Streptococcus</em>.<sup><a reveal-id="DSM018C31" data-open="DSM018C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSM018C32" data-open="DSM018C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSM018C46" data-open="DSM018C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSM018C47" data-open="DSM018C47" class="link link-ref link-reveal xref-bibr">47</a></sup> It has also been recorded that Tn<em>1549</em> is transferable between <em>C. symbiosum</em> and <em>Enterococcus</em> spp.<sup><a reveal-id="DSM018C48" data-open="DSM018C48" class="link link-ref link-reveal xref-bibr">48</a></sup> The homologues found in the metagenomic data accounted for 0.8% of all the predicted genes (5,325 genes in total) and were also enriched in the two fecal samples from American individuals<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a></sup> (Fig. <a reveal-id="DSM018F6" data-open="DSM018F6" class="link link-reveal link-table xref-fig">6</a> and Supplementary Table S7). They were highly divergent in sequence from the corresponding genes on six known Tn<em>1549</em>-like CTns (Fig. <a reveal-id="DSM018F6" data-open="DSM018F6" class="link link-reveal link-table xref-fig">6</a>C), but frequently appeared as gene clusters on contigs. We identified 89 contigs that contained four or more genes related to Tn<em>1549</em>-like CTns. These genes appeared there in the same or a similar gene organization as seen in Tn<em>1549</em>-like CTns (Supplementary Table S7), suggesting that they were derived from divergent members of a Tn<em>1549</em>-like CTn family, which we refer to as ‘CTnRINT’ (CTn rich in intestine). By analysing these contigs, we found that CTnRINT members contain a variety of genes, such as those for ABC-type multidrug transport systems, in the regions corresponding to that for the vancomycin-resistance genes on Tn<em>1549</em> (data not shown). As shown in Fig. <a reveal-id="DSM018F6" data-open="DSM018F6" class="link link-reveal link-table xref-fig">6</a>B, other known Tn<em>1549</em>-like CTns also contain various accessory genes in this region. These findings strongly suggest that the CTnRINT family is largely involved in the process of HGT in the human intestine. It seems reasonable that conjugal elements, which mediate genetic exchanges and transmittance through cell–cell contact, are key players in HGT in the colon.</p>                        <a id="87195846" scrollto-destination="87195846"></a>
<div id="DSM018F6" data-id="DSM018F6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01806.jpeg?Expires=2147483647&Signature=xOEh0RY50W2NevFhHBFdJBl77UUPgdr4uCXjL9uqQuTmLyiGTpSkiGfqXdHrHTJBtHJhysm6S4wh4VDMBrP6xmhpSdAKjyALsDk90omEZSOv~S2BONp3xCTqRLQN~fENMQWB7RjyT6pLwJa8dDd9tVs9BTAQq36eKnLgxdwyZQkU9RK8rOkJGsSpRXXE2YaGbUJigCuGgIynLj0NoFmOAgTJKEPAvE32g2oDKmP8-AeePX6kQcoAb7i35aeDPBxGCvujsNo~-hPkVYQlsWqlCBdPAVCkIUH65lHdDRrzWRPEzNc2S0tKeADjEhd9M5q41icUoGsNtZHnvQVEorrTVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The Tn1549-like CTn family, ‘CTnRINT’, explosively amplified in human gut microbiomes. (A) Numbers of genes homologous to those on six known Tn1549-like CTns. Genes derived from each fecal sample are shown in different colors. Plum: In-A; brown: F2-V; light yellow: F2-W; light cyan: In-D; dark red: F1-S; salmon: F1-T; royal blue: In-R; pale turquoise: F2-X; midnight blue: F2-Y; magenta: F1-U; yellow: In-B; cyan: In-M; and maroon: In-E. (B) Gene organizations of six known Tn1549-like CTns. Tn1549 from E. faecalis strain BM4382, an unnamed CTn from E. faecalis V583, CTn4, CTn2 and CTn5 from C. difficile, and an unnamed CTn from S. pyogenes MGAS10750 are shown. Pentagons and circles represent ORFs. ORFs in the conserved regions are depicted in green, and orthologues are connected with pink vertical lines. The three ORFs depicted in blue are tentatively positioned in this figure to more clearly show their orthologous relationships, and their actual positions are indicated by dotted arrow lines. Orange pentagons represent the accessory genes of the six CTns. Those from Tn1549 and the CTn of E. faecalis V583 include vancomycin-resistance genes.30,32 (C) Sequence diversity of the Tn1549-like CTn family (‘CTnRINT’). All the CTnRINT-related gene products were searched for their best-hit homologues among those on the six known CTnRINT family members. The six columns represent Tn1549, the CTn from E. faecalis V583, CTn4, CTn2 and CTn5 from C. difficile, and the CTn from S. pyogenes MGAS10750, respectively (from left to right). The BLASTP identity (%) for a CTnRINT-related gene product was plotted in the CTn column where its best-hit homologue was identified." path-from-xml="dsm01806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195846/dsm01806.jpeg" path-from-xml="dsm01806.jpeg" target="_blank">View large</a><a section="87195846" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01806.jpeg?Expires=2147483647&Signature=IF1arwRNHqJ2JljCGk0LyVfRRwSpT~gFSF~KhcwgS-bR2caCjEF5AIinvpkj8OXkOkygSPOuvICXPcwL-Yl0cTDezF4d8yoUOwJ6BINX4faCdG5efKfWHKawNnXfkKdR40juIuxNgUJzeqfvEQyg8mOi~4-3vlqVpF9KpNg3iRFhJlxlhYFmh5eWk3y5GOE35EFemZuOjIyCyw1IEx1LaHQ745RJ99m6r~N-BHTQF-8Pot6oBGq4UP0zCrUE8iBzPv8Kvt~5jqVU10BjaW6IZaDdRu8nbVsCKIOlXhbCXJNtoT2CEsfzCAZKiYOduqdTnl5Ql9vF5w7moKyEgxOm1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195846&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The Tn<em>1549</em>-like CTn family, ‘CTnRINT’, explosively amplified in human gut microbiomes. (<strong>A</strong>) Numbers of genes homologous to those on six known Tn<em>1549</em>-like CTns. Genes derived from each fecal sample are shown in different colors. Plum: In-A; brown: F2-V; light yellow: F2-W; light cyan: In-D; dark red: F1-S; salmon: F1-T; royal blue: In-R; pale turquoise: F2-X; midnight blue: F2-Y; magenta: F1-U; yellow: In-B; cyan: In-M; and maroon: In-E. (<strong>B</strong>) Gene organizations of six known Tn<em>1549</em>-like CTns. Tn<em>1549</em> from <em>E. faecalis</em> strain BM4382, an unnamed CTn from <em>E. faecalis</em> V583, CTn4, CTn2 and CTn5 from <em>C. difficile</em>, and an unnamed CTn from <em>S. pyogenes</em> MGAS10750 are shown. Pentagons and circles represent ORFs. ORFs in the conserved regions are depicted in green, and orthologues are connected with pink vertical lines. The three ORFs depicted in blue are tentatively positioned in this figure to more clearly show their orthologous relationships, and their actual positions are indicated by dotted arrow lines. Orange pentagons represent the accessory genes of the six CTns. Those from Tn<em>1549</em> and the CTn of <em>E. faecalis</em> V583 include vancomycin-resistance genes.<sup>30,32</sup> (<strong>C</strong>) Sequence diversity of the Tn<em>1549</em>-like CTn family (‘CTnRINT’). All the CTnRINT-related gene products were searched for their best-hit homologues among those on the six known CTnRINT family members. The six columns represent Tn<em>1549</em>, the CTn from <em>E. faecalis</em> V583, CTn4, CTn2 and CTn5 from <em>C. difficile</em>, and the CTn from <em>S. pyogenes</em> MGAS10750, respectively (from left to right). The BLASTP identity (%) for a CTnRINT-related gene product was plotted in the CTn column where its best-hit homologue was identified.</p></div></div><div content-id="DSM018F6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/m_dsm01806.jpeg?Expires=2147483647&Signature=xOEh0RY50W2NevFhHBFdJBl77UUPgdr4uCXjL9uqQuTmLyiGTpSkiGfqXdHrHTJBtHJhysm6S4wh4VDMBrP6xmhpSdAKjyALsDk90omEZSOv~S2BONp3xCTqRLQN~fENMQWB7RjyT6pLwJa8dDd9tVs9BTAQq36eKnLgxdwyZQkU9RK8rOkJGsSpRXXE2YaGbUJigCuGgIynLj0NoFmOAgTJKEPAvE32g2oDKmP8-AeePX6kQcoAb7i35aeDPBxGCvujsNo~-hPkVYQlsWqlCBdPAVCkIUH65lHdDRrzWRPEzNc2S0tKeADjEhd9M5q41icUoGsNtZHnvQVEorrTVw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The Tn1549-like CTn family, ‘CTnRINT’, explosively amplified in human gut microbiomes. (A) Numbers of genes homologous to those on six known Tn1549-like CTns. Genes derived from each fecal sample are shown in different colors. Plum: In-A; brown: F2-V; light yellow: F2-W; light cyan: In-D; dark red: F1-S; salmon: F1-T; royal blue: In-R; pale turquoise: F2-X; midnight blue: F2-Y; magenta: F1-U; yellow: In-B; cyan: In-M; and maroon: In-E. (B) Gene organizations of six known Tn1549-like CTns. Tn1549 from E. faecalis strain BM4382, an unnamed CTn from E. faecalis V583, CTn4, CTn2 and CTn5 from C. difficile, and an unnamed CTn from S. pyogenes MGAS10750 are shown. Pentagons and circles represent ORFs. ORFs in the conserved regions are depicted in green, and orthologues are connected with pink vertical lines. The three ORFs depicted in blue are tentatively positioned in this figure to more clearly show their orthologous relationships, and their actual positions are indicated by dotted arrow lines. Orange pentagons represent the accessory genes of the six CTns. Those from Tn1549 and the CTn of E. faecalis V583 include vancomycin-resistance genes.30,32 (C) Sequence diversity of the Tn1549-like CTn family (‘CTnRINT’). All the CTnRINT-related gene products were searched for their best-hit homologues among those on the six known CTnRINT family members. The six columns represent Tn1549, the CTn from E. faecalis V583, CTn4, CTn2 and CTn5 from C. difficile, and the CTn from S. pyogenes MGAS10750, respectively (from left to right). The BLASTP identity (%) for a CTnRINT-related gene product was plotted in the CTn column where its best-hit homologue was identified." path-from-xml="dsm01806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195846/dsm01806.jpeg" path-from-xml="dsm01806.jpeg" target="_blank">View large</a><a section="87195846" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm01806.jpeg?Expires=2147483647&Signature=IF1arwRNHqJ2JljCGk0LyVfRRwSpT~gFSF~KhcwgS-bR2caCjEF5AIinvpkj8OXkOkygSPOuvICXPcwL-Yl0cTDezF4d8yoUOwJ6BINX4faCdG5efKfWHKawNnXfkKdR40juIuxNgUJzeqfvEQyg8mOi~4-3vlqVpF9KpNg3iRFhJlxlhYFmh5eWk3y5GOE35EFemZuOjIyCyw1IEx1LaHQ745RJ99m6r~N-BHTQF-8Pot6oBGq4UP0zCrUE8iBzPv8Kvt~5jqVU10BjaW6IZaDdRu8nbVsCKIOlXhbCXJNtoT2CEsfzCAZKiYOduqdTnl5Ql9vF5w7moKyEgxOm1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195846&ar=466883&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsm01806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The Tn<em>1549</em>-like CTn family, ‘CTnRINT’, explosively amplified in human gut microbiomes. (<strong>A</strong>) Numbers of genes homologous to those on six known Tn<em>1549</em>-like CTns. Genes derived from each fecal sample are shown in different colors. Plum: In-A; brown: F2-V; light yellow: F2-W; light cyan: In-D; dark red: F1-S; salmon: F1-T; royal blue: In-R; pale turquoise: F2-X; midnight blue: F2-Y; magenta: F1-U; yellow: In-B; cyan: In-M; and maroon: In-E. (<strong>B</strong>) Gene organizations of six known Tn<em>1549</em>-like CTns. Tn<em>1549</em> from <em>E. faecalis</em> strain BM4382, an unnamed CTn from <em>E. faecalis</em> V583, CTn4, CTn2 and CTn5 from <em>C. difficile</em>, and an unnamed CTn from <em>S. pyogenes</em> MGAS10750 are shown. Pentagons and circles represent ORFs. ORFs in the conserved regions are depicted in green, and orthologues are connected with pink vertical lines. The three ORFs depicted in blue are tentatively positioned in this figure to more clearly show their orthologous relationships, and their actual positions are indicated by dotted arrow lines. Orange pentagons represent the accessory genes of the six CTns. Those from Tn<em>1549</em> and the CTn of <em>E. faecalis</em> V583 include vancomycin-resistance genes.<sup>30,32</sup> (<strong>C</strong>) Sequence diversity of the Tn<em>1549</em>-like CTn family (‘CTnRINT’). All the CTnRINT-related gene products were searched for their best-hit homologues among those on the six known CTnRINT family members. The six columns represent Tn<em>1549</em>, the CTn from <em>E. faecalis</em> V583, CTn4, CTn2 and CTn5 from <em>C. difficile</em>, and the CTn from <em>S. pyogenes</em> MGAS10750, respectively (from left to right). The BLASTP identity (%) for a CTnRINT-related gene product was plotted in the CTn column where its best-hit homologue was identified.</p></div></div><p>In addition to the CTnRINT family, we found that integrases/site-specific recombinases belonging to COG4974 were remarkably expanded in the microbiomes of the adults and children. Most of them were apparently derived from several types of integrative mobile genetic elements such as Tn<em>916</em>-like CTns<sup><a reveal-id="DSM018C49" data-open="DSM018C49" class="link link-ref link-reveal xref-bibr">49</a></sup>, suggesting that other types of integrative elements are also richly present in human gut microbiomes.</p>                            <h3 scrollto-destination=87195848 id="87195848" class="section-title" data-legacy-id=s3g>3.7. Orphan gene families in human gut microbiomes</h3>
<p>Of the 662,548 genes predicted in the 13 samples, 162,647 were orphan genes (25% of the total genes). Similarly, 503,115 orphan genes were obtained from other-environmental microbiomes.<sup><a reveal-id="DSM018C21" data-open="DSM018C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSM018C26" data-open="DSM018C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSM018C39" data-open="DSM018C39" class="link link-ref link-reveal xref-bibr">39</a></sup> An all-to-all BLASTP analysis of these 665,762 orphan gene products followed by a clustering analysis (see ‘Materials and methods’) yielded 160,543 clusters and 461,435 singletons. Of the 160,543 clusters, 647 comprised five or more gene products derived only from human fecal samples (Supplementary Table S8). The largest two clusters, ID37 and ID39 containing 48 and 47 members, respectively, were present in the microbiomes of all the Japanese adults and children, and also in those of the American adults. For eight clusters that comprise ≥30 gene members, we performed motif search/extraction analyses by using the HMMER program against the Pfam motif database.<sup><a reveal-id="DSM018C41" data-open="DSM018C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Only two gene products in cluster ID44 showed a significant similarity to a Pfam motif (PF07508; recombinase); other gene products in cluster ID44 showed no significant similarity in the Pfam database. We could identify the conserved amino-acid sequences (38–50 amino acids) for each cluster by using the MEME program<sup><a reveal-id="DSM018C42" data-open="DSM018C42" class="link link-ref link-reveal xref-bibr">42</a></sup> (Supplementary Fig. S2). These conserved sequences may represent new motifs specific to human gut microbiomes.</p>                            <h3 scrollto-destination=87195850 id="87195850" class="section-title" data-legacy-id=s3h>3.8. Remarks and future perspectives</h3>
<p>The present study is the first large-scale comparative metagenomic analysis of human gut microbiomes. The data provided several new lines of insight into the genomic features of gut microbiota. First, our data clearly demonstrated a difference in overall composition and gene repertoire between adult- and infant-type gut microbiomes. The simple and less redundant features of the infant-type gut microbiota are probably linked to its high inter-individual variability (Figs. <a reveal-id="DSM018F1" data-open="DSM018F1" class="link link-reveal link-table xref-fig">1</a>, 2, and 3C). We suggest that the infant-type can be viewed as unstable, yet dynamic and adaptable. Conversely, the functional uniformity observed in the adult-type microbiota (Figs. <a reveal-id="DSM018F1" data-open="DSM018F1" class="link link-reveal link-table xref-fig">1</a>, 3B, and 4) may be attributable to its more complex nature (Fig. <a reveal-id="DSM018F2" data-open="DSM018F2" class="link link-reveal link-table xref-fig">2</a> and Tables <a reveal-id="DSM018TB1" data-open="DSM018TB1" class="link link-reveal link-table xref-fig">1</a> and S2), which in turn suggests that the insurance hypothesis<sup><a reveal-id="DSM018C50" data-open="DSM018C50" class="link link-ref link-reveal xref-bibr">50</a></sup> for the benefit of biodiversity may be relevant to the gut.</p><p>Secondly, a comparison of the gene contents between gut microbiota and previously sequenced microbes revealed 237 COGs commonly enriched in adult-type microbiomes and 136 COGs in infant-type microbiomes (Fig. <a reveal-id="DSM018F5" data-open="DSM018F5" class="link link-reveal link-table xref-fig">5</a> and Supplementary Table S6). The characterization of these genes revealed distinct nutrient acquisition strategies in each type of microbiota, possibly to accommodate very different diets of their hosts. The analysis also revealed several possible strategies through which intestinal microbes adapt to the intestinal environment and establish symbiotic relationships with their host. Thus, these genes, which we refer to as ‘Adult- or Infant-gut Commonly Enriched Genes’ (A-gutCEGs or I-gutCEGs), appear to encode some of the core functions of each type of microbiota. It is noteworthy that the two gene sets contain as many as 104 gene families of unknown functions (Fig. <a reveal-id="DSM018F5" data-open="DSM018F5" class="link link-reveal link-table xref-fig">5</a>). The <em>in vitro</em> and <em>in vivo</em> functions of these uncharacterized genes as well as those of the 647 ‘new’ gene families (Supplementary Table S8) would be important topics of future studies.</p><p>Thirdly, a survey of the enriched genes revealed an abundance of mobile genetic elements in the human intestinal gene pool, emphasizing that the human gut microbiota is a “hot spot” for HGT between microbes. Of particular importance is the abundance of conjugal elements including CTnRINT. Considering their high transfer efficiency, the broad range of hosts, and the frequent carriage of drug-resistance genes, it would be prudent to reassess the heavy use of antibiotics in modern medicine.</p><p>Finally, the metagenomic datasets presented here will be of great use for understanding the roles of gut microbiota in the etiology of human diseases and also for scientifically evaluating the efficacy of probiotics, prebiotics and other ‘functional foods’ that are widely used for modulating the intestinal microbiota in an effort to improve our health<sup><a reveal-id="DSM018C7" data-open="DSM018C7" class="link link-ref link-reveal xref-bibr">7</a></sup>.</p>                            <h2 scrollto-destination=87195855 id="87195855" class="section-title" data-legacy-id=s4>Funding</h2>
<p>This work was supported by Grants-in-Aid for Scientific Research on Priority Areas ‘Comprehensive Genomics’ (M.H.) and ‘Applied Genomics’ (T.H. and T.K.) from the Ministry of Education, Science, and Technology of Japan, and a Grant-in-Aid from the Institute for Bioinformatics Research and Development, the Japan Science and Technology Agency (BIRD-JST) (K.K.).</p>                            <h2 scrollto-destination=87195857 id="87195857" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Jun Ishikawa, Pawan K. Dhar, and Takeaki Taniguchi for their help and support with the data analysis, Noriko Itoh, Hiromi Inaba, Asako Tamura, Keiko Furuya, Kanako Motomura, Yasue Yamashita, Chie Yoshino, and Yuri Hayakawa for their technical assistances, Yumiko Hayashi for her editorial assistance, and those responsible for the microbial genome sequencing project at the Washington University Genome Sequencing Center for providing open access to unpublished genome sequence data.</p>                            <h2 scrollto-destination=87195859 id="87195859" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSM018C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Backhed</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ley</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Sonnenburg</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I.</div></div>. </div><div class="article-title">Host-bacterial mutualism in the human intestine</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1915</div>-<div class="lpage">1920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Host-bacterial%20mutualism%20in%20the%20human%20intestine&amp;author=F.%20Backhed&amp;author=R.%20E.%20Ley&amp;author=J.%20L.%20Sonnenburg&amp;author=D.%20A.%20Peterson&amp;author=J.%20I.%20Gordon&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1915-1920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1104816" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1104816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1104816"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15790844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I.</div></div>. </div><div class="article-title">Ecological and evolutionary forces shaping microbial diversity in the human intestine</div>, <div class="source">Cell</div> , <div class="year">2006</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">837</div>-<div class="lpage">848</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecological%20and%20evolutionary%20forces%20shaping%20microbial%20diversity%20in%20the%20human%20intestine&amp;author=R.%20E.%20Ley&amp;author=D.%20A.%20Peterson&amp;author=J.%20I.%20Gordon&amp;publication_year=2006&amp;journal=Cell&amp;volume=124&amp;pages=837-848" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.02.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.02.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.02.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16497592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Savage</div> <div class="given-names">D. C</div></div>. </div><div class="article-title">Microbial biota of the human intestine: a tribute to some pioneering scientists</div>, <div class="source">Curr. Issues. Intest. Microbiol</div> , <div class="year">2001</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20biota%20of%20the%20human%20intestine%3A%20a%20tribute%20to%20some%20pioneering%20scientists&amp;author=D.%20C%20Savage&amp;publication_year=2001&amp;journal=Curr.%20Issues.%20Intest.%20Microbiol&amp;volume=2&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11709852" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tancrede</div> <div class="given-names">C</div></div>. </div><div class="article-title">Role of human microflora in health and disease</div>, <div class="source">Eur J Clin Microbiol. Infect. Dis.</div> , <div class="year">1992</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1012</div>-<div class="lpage">1015</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20human%20microflora%20in%20health%20and%20disease&amp;author=C%20Tancrede&amp;publication_year=1992&amp;journal=Eur%20J%20Clin%20Microbiol.%20Infect.%20Dis.&amp;volume=11&amp;pages=1012-1015" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01967791" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01967791" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01967791"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1295753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eckburg</div> <div class="given-names">P. B.</div></div>,  <div class="name"><div class="surname">Bik</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Bernstein</div> <div class="given-names">C. N.</div></div>, et al. </div><div class="article-title">Diversity of the human intestinal microbial flora</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">308</div> (pg. <div class="fpage">1635</div>-<div class="lpage">1638</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20the%20human%20intestinal%20microbial%20flora&amp;author=P.%20B.%20Eckburg&amp;author=E.%20M.%20Bik&amp;author=C.%20N.%20Bernstein&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=1635-1638" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1110591" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1110591" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1110591"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15831718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fanaro</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chierici</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Guerrini</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Vigi</div> <div class="given-names">V.</div></div>. </div><div class="article-title">Intestinal microflora in early infancy: composition and development</div>, <div class="source">Acta Paediatr.</div> , <div class="year">2003</div>, vol. <div class="volume">91</div> <div class="supplement">(Suppl.)</div>(pg. <div class="fpage">48</div>-<div class="lpage">55</div>)</div></div></div></div><div content-id="DSM018C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gibson</div> <div class="given-names">G. R.</div></div>,  <div class="name"><div class="surname">Roberfroid</div> <div class="given-names">M. B</div></div>. </div><div class="article-title">Dietary modulation of the human colonic microbiota: introducing the concept of prebiotics</div>, <div class="source">J. Nutr.</div> , <div class="year">1995</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">1401</div>-<div class="lpage">1412</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dietary%20modulation%20of%20the%20human%20colonic%20microbiota%3A%20introducing%20the%20concept%20of%20prebiotics&amp;author=G.%20R.%20Gibson&amp;author=M.%20B%20Roberfroid&amp;publication_year=1995&amp;journal=J.%20Nutr.&amp;volume=125&amp;pages=1401-1412" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7782892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guarner</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Malagelada</div> <div class="given-names">J. R</div></div>. </div><div class="article-title">Gut flora in health and disease</div>, <div class="source">Lancet</div> , <div class="year">2003</div>, vol. <div class="volume">361</div> (pg. <div class="fpage">512</div>-<div class="lpage">519</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gut%20flora%20in%20health%20and%20disease&amp;author=F.%20Guarner&amp;author=J.%20R%20Malagelada&amp;publication_year=2003&amp;journal=Lancet&amp;volume=361&amp;pages=512-519" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0140-6736(03)12489-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0140-6736(03)12489-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0140-6736(03)12489-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12583961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hooper</div> <div class="given-names">L. V.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I</div></div>. </div><div class="article-title">Commensal host-bacterial relationships in the gut</div>, <div class="source">Science</div> , <div class="year">2001</div>, vol. <div class="volume">292</div> (pg. <div class="fpage">1115</div>-<div class="lpage">1118</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Commensal%20host-bacterial%20relationships%20in%20the%20gut&amp;author=L.%20V.%20Hooper&amp;author=J.%20I%20Gordon&amp;publication_year=2001&amp;journal=Science&amp;volume=292&amp;pages=1115-1118" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1058709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1058709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1058709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11352068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Backhed</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ding</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">The gut microbiota as an environmental factor that regulates fat storage</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">15718</div>-<div class="lpage">15723</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20gut%20microbiota%20as%20an%20environmental%20factor%20that%20regulates%20fat%20storage&amp;author=F.%20Backhed&amp;author=H.%20Ding&amp;author=T.%20Wang&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=15718-15723" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0407076101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0407076101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0407076101"> </span></div> </div></div></div></div></div><div content-id="DSM018C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rakoff-Nahoum</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paglino</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Eslami-Varzaneh</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Edberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Medzhitov</div> <div class="given-names">R</div></div>. </div><div class="article-title">Recognition of commensal microflora by toll-like receptors is required for intestinal homeostasis</div>, <div class="source">Cell</div> , <div class="year">2004</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">229</div>-<div class="lpage">241</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20commensal%20microflora%20by%20toll-like%20receptors%20is%20required%20for%20intestinal%20homeostasis&amp;author=S.%20Rakoff-Nahoum&amp;author=J.%20Paglino&amp;author=F.%20Eslami-Varzaneh&amp;author=S.%20Edberg&amp;author=R%20Medzhitov&amp;publication_year=2004&amp;journal=Cell&amp;volume=118&amp;pages=229-241" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2004.07.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2004.07.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2004.07.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15260992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Samuel</div> <div class="given-names">B. S.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I</div></div>. </div><div class="article-title">A humanized gnotobiotic mouse model of host-archaeal-bacterial mutualism</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">10011</div>-<div class="lpage">10016</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20humanized%20gnotobiotic%20mouse%20model%20of%20host-archaeal-bacterial%20mutualism&amp;author=B.%20S.%20Samuel&amp;author=J.%20I%20Gordon&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=10011-10016" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0602187103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0602187103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0602187103"> </span></div> </div></div></div></div></div><div content-id="DSM018C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ley</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I</div></div>. </div><div class="article-title">Microbial ecology: human gut microbes associated with obesity</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">444</div> (pg. <div class="fpage">1022</div>-<div class="lpage">1023</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20ecology%3A%20human%20gut%20microbes%20associated%20with%20obesity&amp;author=R.%20E.%20Ley&amp;author=P.%20J.%20Turnbaugh&amp;author=S.%20Klein&amp;author=J.%20I%20Gordon&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=1022-1023" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/4441022a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F4441022a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F4441022a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Macdonald</div> <div class="given-names">T. T.</div></div>,  <div class="name"><div class="surname">Monteleone</div> <div class="given-names">G</div></div>. </div><div class="article-title">Immunity, inflammation, and allergy in the gut</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1920</div>-<div class="lpage">1925</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Immunity%2C%20inflammation%2C%20and%20allergy%20in%20the%20gut&amp;author=T.%20T.%20Macdonald&amp;author=G%20Monteleone&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1920-1925" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1106442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1106442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1106442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15790845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Manichanh</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rigottier-Gois</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bonnaud</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Reduced diversity of faecal microbiota in Crohn's disease revealed by a metagenomic approach</div>, <div class="source">Gut</div> , <div class="year">2006</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">205</div>-<div class="lpage">211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20diversity%20of%20faecal%20microbiota%20in%20Crohn's%20disease%20revealed%20by%20a%20metagenomic%20approach&amp;author=C.%20Manichanh&amp;author=L.%20Rigottier-Gois&amp;author=E.%20Bonnaud&amp;publication_year=2006&amp;journal=Gut&amp;volume=55&amp;pages=205-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/gut.2005.073817" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fgut.2005.073817" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fgut.2005.073817"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16188921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McGarr</div> <div class="given-names">S. E.</div></div>,  <div class="name"><div class="surname">Ridlon</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Hylemon</div> <div class="given-names">P. B</div></div>. </div><div class="article-title">Diet, anaerobic bacterial metabolism, and colon cancer: a review of the literature</div>, <div class="source">J. Clin. Gastroenterol</div> , <div class="year">2005</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">98</div>-<div class="lpage">109</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diet%2C%20anaerobic%20bacterial%20metabolism%2C%20and%20colon%20cancer%3A%20a%20review%20of%20the%20literature&amp;author=S.%20E.%20McGarr&amp;author=J.%20M.%20Ridlon&amp;author=P.%20B%20Hylemon&amp;publication_year=2005&amp;journal=J.%20Clin.%20Gastroenterol&amp;volume=39&amp;pages=98-109" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15681903" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seksik</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sokol</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lepage</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Review article: the role of bacteria in onset and perpetuation of inflammatory bowel disease</div>, <div class="source">Aliment. Pharmacol. Ther.</div> , <div class="year">2006</div>, vol. <div class="volume">24</div> <div class="supplement">Suppl 3</div>(pg. <div class="fpage">11</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Review%20article%3A%20the%20role%20of%20bacteria%20in%20onset%20and%20perpetuation%20of%20inflammatory%20bowel%20disease&amp;author=P.%20Seksik&amp;author=H.%20Sokol&amp;author=P.%20Lepage&amp;publication_year=2006&amp;journal=Aliment.%20Pharmacol.%20Ther.&amp;volume=24&amp;pages=11-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/apt.2006.24.issue-s3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fapt.2006.24.issue-s3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fapt.2006.24.issue-s3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16961738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turnbaugh</div> <div class="given-names">P. J.</div></div>,  <div class="name"><div class="surname">Ley</div> <div class="given-names">R. E.</div></div>,  <div class="name"><div class="surname">Mahowald</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Magrini</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Mardis</div> <div class="given-names">E. R.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I</div></div>. </div><div class="article-title">An obesity-associated gut microbiome with increased capacity for energy harvest</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">444</div> (pg. <div class="fpage">1027</div>-<div class="lpage">1031</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20obesity-associated%20gut%20microbiome%20with%20increased%20capacity%20for%20energy%20harvest&amp;author=P.%20J.%20Turnbaugh&amp;author=R.%20E.%20Ley&amp;author=M.%20A.%20Mahowald&amp;author=V.%20Magrini&amp;author=E.%20R.%20Mardis&amp;author=J.%20I%20Gordon&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=1027-1031" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17183312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finegold</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Sutter</div> <div class="given-names">V. L.</div></div>,  <div class="name"><div class="surname">Mathisen</div> <div class="given-names">G. E</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Hentges</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Normal indigenous flora, In</div>, <div class="source">Human Intestinal Microflora in Health and Disease</div> , <div class="year">1983</div><div class="publisher-loc">NY</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">3</div>-<div class="lpage">31</div>)</div></div></div></div><div content-id="DSM018C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tringe</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Rubin</div> <div class="given-names">E. M</div></div>. </div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> <div class="publisher-loc">Metagenomics</div><div class="publisher-name">DNA sequencing of environmental samples</div>(pg. <div class="fpage">805</div>-<div class="lpage">814</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16304596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gill</div> <div class="given-names">S. R.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Deboy</div> <div class="given-names">R. T.</div></div>, et al. </div><div class="article-title">Metagenomic analysis of the human distal gut microbiome</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">1355</div>-<div class="lpage">1359</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20of%20the%20human%20distal%20gut%20microbiome&amp;author=S.%20R.%20Gill&amp;author=M.%20Pop&amp;author=R.%20T.%20Deboy&amp;publication_year=2006&amp;journal=Science&amp;volume=312&amp;pages=1355-1359" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1124234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1124234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1124234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16741115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benno</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Endo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mizutani</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Namba</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Komori</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mitsuoka</div> <div class="given-names">T</div></div>. </div><div class="article-title">Comparison of fecal microflora of elderly persons in rural and urban areas of Japan</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1989</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">1100</div>-<div class="lpage">1105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20fecal%20microflora%20of%20elderly%20persons%20in%20rural%20and%20urban%20areas%20of%20Japan&amp;author=Y.%20Benno&amp;author=K.%20Endo&amp;author=T.%20Mizutani&amp;author=Y.%20Namba&amp;author=T.%20Komori&amp;author=T%20Mitsuoka&amp;publication_year=1989&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=55&amp;pages=1100-1105" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2547333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Favier</div> <div class="given-names">C. F.</div></div>,  <div class="name"><div class="surname">Vaughan</div> <div class="given-names">E. E.</div></div>,  <div class="name"><div class="surname">De Vos</div> <div class="given-names">W. M.</div></div>,  <div class="name"><div class="surname">Akkermans</div> <div class="given-names">A. D</div></div>. </div><div class="article-title">Molecular monitoring of succession of bacterial communities in human neonates</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2002</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">219</div>-<div class="lpage">226</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20monitoring%20of%20succession%20of%20bacterial%20communities%20in%20human%20neonates&amp;author=C.%20F.%20Favier&amp;author=E.%20E.%20Vaughan&amp;author=W.%20M.%20De%20Vos&amp;author=A.%20D%20Akkermans&amp;publication_year=2002&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=68&amp;pages=219-226" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.68.1.219-226.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.68.1.219-226.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.68.1.219-226.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11772630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hopkins</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Macfarlane</div> <div class="given-names">G. T</div></div>. </div><div class="article-title">Age and disease related changes in intestinal bacterial populations assessed by cell culture, 16S RNA abundance, and community cellular fatty acid profiles</div>, <div class="source">Gut</div> , <div class="year">2001</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">198</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Age%20and%20disease%20related%20changes%20in%20intestinal%20bacterial%20populations%20assessed%20by%20cell%20culture%2C%2016S%20RNA%20abundance%2C%20and%20community%20cellular%20fatty%20acid%20profiles&amp;author=M.%20J.%20Hopkins&amp;author=R.%20Sharp&amp;author=G.%20T%20Macfarlane&amp;publication_year=2001&amp;journal=Gut&amp;volume=48&amp;pages=198-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/gut.48.2.198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fgut.48.2.198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fgut.48.2.198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11156640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stewart</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Chadwick</div> <div class="given-names">V. S.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">A</div></div>. </div><div class="article-title">Investigations into the influence of host genetics on the predominant eubacteria in the faecal microflora of children</div>, <div class="source">J. Med. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">1239</div>-<div class="lpage">1242</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigations%20into%20the%20influence%20of%20host%20genetics%20on%20the%20predominant%20eubacteria%20in%20the%20faecal%20microflora%20of%20children&amp;author=J.%20A.%20Stewart&amp;author=V.%20S.%20Chadwick&amp;author=A%20Murray&amp;publication_year=2005&amp;journal=J.%20Med.%20Microbiol.&amp;volume=54&amp;pages=1239-1242" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/jmm.0.46189-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fjmm.0.46189-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fjmm.0.46189-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16278440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Venter</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Remington</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J. F.</div></div>, et al. </div><div class="article-title">Environmental genome shotgun sequencing of the Sargasso Sea</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">66</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20genome%20shotgun%20sequencing%20of%20the%20Sargasso%20Sea&amp;author=J.%20C.%20Venter&amp;author=K.%20Remington&amp;author=J.%20F.%20Heidelberg&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=66-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15001713" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsukahara</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Furuhata</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">A novel method for making nested deletions and its application for sequencing of a 300 kb region of human APP locus</div>, <div class="source">Nuc. Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1802</div>-<div class="lpage">1808</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20method%20for%20making%20nested%20deletions%20and%20its%20application%20for%20sequencing%20of%20a%20300%20kb%20region%20of%20human%20APP%20locus&amp;author=M.%20Hattori&amp;author=F.%20Tsukahara&amp;author=Y.%20Furuhata&amp;publication_year=1997&amp;journal=Nuc.%20Acids%20Res.&amp;volume=25&amp;pages=1802-1808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.9.1802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.9.1802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.9.1802"> </span></div> </div></div></div></div></div><div content-id="DSM018C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aluru</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">S. P.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L</div></div>. </div><div class="article-title">PCAP: a whole-genome assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2164</div>-<div class="lpage">2170</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCAP%3A%20a%20whole-genome%20assembly%20program&amp;author=X.%20Huang&amp;author=J.%20Wang&amp;author=S.%20Aluru&amp;author=S.%20P.%20Yang&amp;author=L%20Hillier&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2164-2170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1390403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1390403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1390403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">T</div></div>. </div><div class="article-title">MetaGene: prokaryotic gene finding from environmental genome shotgun sequences</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5623</div>-<div class="lpage">5630</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetaGene%3A%20prokaryotic%20gene%20finding%20from%20environmental%20genome%20shotgun%20sequences&amp;author=H.%20Noguchi&amp;author=J.%20Park&amp;author=T%20Takagi&amp;publication_year=2006&amp;journal=Nucl.%20Acids%20Res.&amp;volume=34&amp;pages=5623-5630" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl723"> </span></div> </div></div></div></div></div><div content-id="DSM018C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garnier</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Taourit</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Glaser</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Courvalin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Galimand</div> <div class="given-names">M</div></div>. </div><div class="article-title">Characterization of transposon Tn<em>1549</em>, conferring VanB-type resistance in <em>Enterococcus</em> spp</div>, <div class="source">Microbiology</div> , <div class="year">2000</div>, vol. <div class="volume">146</div> <div class="issue">Pt 6</div>(pg. <div class="fpage">1481</div>-<div class="lpage">1489</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20transposon%20Tn1549%2C%20conferring%20VanB-type%20resistance%20in%20Enterococcus%20spp&amp;author=F.%20Garnier&amp;author=S.%20Taourit&amp;author=P.%20Glaser&amp;author=P.%20Courvalin&amp;author=M%20Galimand&amp;publication_year=2000&amp;journal=Microbiology&amp;volume=146&amp;pages=1481-1489" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-146-6-1481" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-146-6-1481" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-146-6-1481"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10846226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sebaihia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wren</div> <div class="given-names">B. W.</div></div>,  <div class="name"><div class="surname">Mullany</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">The multidrug-resistant human pathogen <em>Clostridium difficile</em> has a highly mobile, mosaic genome</div>, <div class="source">Nat. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">779</div>-<div class="lpage">786</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20multidrug-resistant%20human%20pathogen%20Clostridium%20difficile%20has%20a%20highly%20mobile%2C%20mosaic%20genome&amp;author=M.%20Sebaihia&amp;author=B.%20W.%20Wren&amp;author=P.%20Mullany&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=779-786" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1830" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1830" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1830"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16804543" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paulsen</div> <div class="given-names">I. T.</div></div>,  <div class="name"><div class="surname">Banerjei</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">G. S.</div></div>, et al. </div><div class="article-title">Role of mobile DNA in the evolution of vancomycin-resistant <em>Enterococcus faecalis</em></div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">299</div> (pg. <div class="fpage">2071</div>-<div class="lpage">2074</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20mobile%20DNA%20in%20the%20evolution%20of%20vancomycin-resistant%20Enterococcus%20faecalis&amp;author=I.%20T.%20Paulsen&amp;author=L.%20Banerjei&amp;author=G.%20S.%20Myers&amp;publication_year=2003&amp;journal=Science&amp;volume=299&amp;pages=2071-2074" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1080613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1080613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1080613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beres</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Richter</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Nagiec</div> <div class="given-names">M. J.</div></div>, et al. </div><div class="article-title">Molecular genetic anatomy of inter- and intraserotype variation in the human bacterial pathogen group A Streptococcus</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">7059</div>-<div class="lpage">7064</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20genetic%20anatomy%20of%20inter-%20and%20intraserotype%20variation%20in%20the%20human%20bacterial%20pathogen%20group%20A%20Streptococcus&amp;author=S.%20B.%20Beres&amp;author=E.%20W.%20Richter&amp;author=M.%20J.%20Nagiec&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=7059-7064" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0510279103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0510279103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0510279103"> </span></div> </div></div></div></div></div><div content-id="DSM018C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div> </div></div></div></div></div><div content-id="DSM018C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DeLong</div> <div class="given-names">E. F.</div></div>,  <div class="name"><div class="surname">Preston</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Mincer</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Community genomics among stratified microbial assemblages in the ocean's interior</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">311</div> (pg. <div class="fpage">496</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community%20genomics%20among%20stratified%20microbial%20assemblages%20in%20the%20ocean's%20interior&amp;author=E.%20F.%20DeLong&amp;author=C.%20M.%20Preston&amp;author=T.%20Mincer&amp;publication_year=2006&amp;journal=Science&amp;volume=311&amp;pages=496-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1120250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1120250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1120250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16439655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Ahren</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Goldovsky</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Janssen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A</div></div>. </div><div class="article-title">Measuring genome conservation across taxa: divided strains and united kingdoms</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">616</div>-<div class="lpage">621</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measuring%20genome%20conservation%20across%20taxa%3A%20divided%20strains%20and%20united%20kingdoms&amp;author=V.%20Kunin&amp;author=D.%20Ahren&amp;author=L.%20Goldovsky&amp;author=P.%20Janssen&amp;author=C.%20A%20Ouzounis&amp;publication_year=2005&amp;journal=Nucl.%20Acids%20Res.&amp;volume=33&amp;pages=616-621" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki181" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki181" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki181"> </span></div> </div></div></div></div></div><div content-id="DSM018C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korbel</div> <div class="given-names">J. O.</div></div>,  <div class="name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P</div></div>. </div><div class="article-title">SHOT: a web server for the construction of genome phylogenies</div>, <div class="source">Trends Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">158</div>-<div class="lpage">162</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SHOT%3A%20a%20web%20server%20for%20the%20construction%20of%20genome%20phylogenies&amp;author=J.%20O.%20Korbel&amp;author=B.%20Snel&amp;author=M.%20A.%20Huynen&amp;author=P%20Bork&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=158-162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(01)02597-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(01)02597-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(01)02597-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11858840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Hoon</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Imoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nolan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Miyano</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Open source clustering software</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1453</div>-<div class="lpage">1454</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Open%20source%20clustering%20software&amp;author=M.%20J.%20de%20Hoon&amp;author=S.%20Imoto&amp;author=J.%20Nolan&amp;author=S.%20Miyano&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=1453-1454" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14871861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tringe</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Mering von</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Comparative metagenomics of microbial communities</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">308</div> (pg. <div class="fpage">554</div>-<div class="lpage">557</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20of%20microbial%20communities&amp;author=S.%20G.%20Tringe&amp;author=C.%20Mering%20von&amp;author=A.%20Kobayashi&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=554-557" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15845853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Enright</div> <div class="given-names">A. J.</div></div>,  <div class="name"><div class="surname">Van Dongen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ouzounis</div> <div class="given-names">C. A</div></div>. </div><div class="article-title">An efficient algorithm for large-scale detection of protein families</div>, <div class="source">Nucl. Acids Res</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1575</div>-<div class="lpage">1584</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20algorithm%20for%20large-scale%20detection%20of%20protein%20families&amp;author=A.%20J.%20Enright&amp;author=S.%20Van%20Dongen&amp;author=C.%20A%20Ouzounis&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res&amp;volume=30&amp;pages=1575-1584" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.7.1575" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.7.1575" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.7.1575"> </span></div> </div></div></div></div></div><div content-id="DSM018C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Mistry</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schuster-Bockler</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Pfam: clans, web tools and services</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> <div class="issue">Database Issue</div>(pg. <div class="fpage">D247</div>-<div class="lpage">D251</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20clans%2C%20web%20tools%20and%20services&amp;author=R.%20D.%20Finn&amp;author=J.%20Mistry&amp;author=B.%20Schuster-Bockler&amp;author=S.%20Griffiths-Jones&amp;publication_year=2006&amp;journal=Nucl.%20Acids%20Res.&amp;volume=34&amp;pages=D247-D251" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj149"> </span></div> </div></div></div></div></div><div content-id="DSM018C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Elkan</div> <div class="given-names">C</div></div>. </div><div class="article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers</div>, <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">28</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fitting%20a%20mixture%20model%20by%20expectation%20maximization%20to%20discover%20motifs%20in%20biopolymers&amp;author=T.%20L.%20Bailey&amp;author=C%20Elkan&amp;publication_year=1994&amp;journal=Proc.%20Int.%20Conf.%20Intell.%20Syst.%20Mol.%20Biol.&amp;volume=2&amp;pages=28-36" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peschel</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sahl</div> <div class="given-names">H. G</div></div>. </div><div class="article-title">The co-evolution of host cationic antimicrobial peptides and microbial resistance</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">529</div>-<div class="lpage">536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20co-evolution%20of%20host%20cationic%20antimicrobial%20peptides%20and%20microbial%20resistance&amp;author=A.%20Peschel&amp;author=H.%20G%20Sahl&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=4&amp;pages=529-536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16778838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wakabayashi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nagao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Esumi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sugimura</div> <div class="given-names">T</div></div>. </div><div class="article-title">Food-derived mutagens and carcinogens</div>, <div class="source">Cancer Res.</div> , <div class="year">1992</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">2092s</div>-<div class="lpage">2098s</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Food-derived%20mutagens%20and%20carcinogens&amp;author=K.%20Wakabayashi&amp;author=M.%20Nagao&amp;author=H.%20Esumi&amp;author=T%20Sugimura&amp;publication_year=1992&amp;journal=Cancer%20Res.&amp;volume=52&amp;pages=2092s-2098s" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1544146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bry</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Falk</div> <div class="given-names">P. G.</div></div>,  <div class="name"><div class="surname">Midtvedt</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">J. I</div></div>. </div><div class="article-title">A model of host-microbial interactions in an open mammalian ecosystem</div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">273</div> (pg. <div class="fpage">1380</div>-<div class="lpage">1383</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20model%20of%20host-microbial%20interactions%20in%20an%20open%20mammalian%20ecosystem&amp;author=L.%20Bry&amp;author=P.%20G.%20Falk&amp;author=T.%20Midtvedt&amp;author=J.%20I%20Gordon&amp;publication_year=1996&amp;journal=Science&amp;volume=273&amp;pages=1380-1383" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.273.5280.1380" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.273.5280.1380" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.273.5280.1380"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8703071" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Domingo</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Huletsky</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bernal</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Characterization of a Tn<em>5382</em>-like transposon containing the <em>vanB2</em> gene cluster in a <em>Clostridium</em> strain isolated from human faeces</div>, <div class="source">J. Antimicrob. Chemother.</div> , <div class="year">2005</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">466</div>-<div class="lpage">474</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20Tn5382-like%20transposon%20containing%20the%20vanB2%20gene%20cluster%20in%20a%20Clostridium%20strain%20isolated%20from%20human%20faeces&amp;author=M.%20C.%20Domingo&amp;author=A.%20Huletsky&amp;author=A.%20Bernal&amp;publication_year=2005&amp;journal=J.%20Antimicrob.%20Chemother.&amp;volume=55&amp;pages=466-474" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jac/dki029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjac%2Fdki029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjac%2Fdki029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15731199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Melville</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Brunel</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Flint</div> <div class="given-names">H. J.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">K. P</div></div>. </div><div class="article-title">The <em>Butyrivibrio fibrisolvens tet</em>(W) gene is carried on the novel conjugative transposon Tn<em>B1230</em>, which contains duplicated nitroreductase coding sequences</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">3656</div>-<div class="lpage">3659</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Butyrivibrio%20fibrisolvens%20tet(W)%20gene%20is%20carried%20on%20the%20novel%20conjugative%20transposon%20TnB1230%2C%20which%20contains%20duplicated%20nitroreductase%20coding%20sequences&amp;author=C.%20M.%20Melville&amp;author=R.%20Brunel&amp;author=H.%20J.%20Flint&amp;author=K.%20P%20Scott&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=3656-3659" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.11.3656-3659.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.11.3656-3659.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.11.3656-3659.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15150255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Launay</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ballard</div> <div class="given-names">S. A.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">P. D.</div></div>,  <div class="name"><div class="surname">Grayson</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">T</div></div>. </div><div class="article-title">Transfer of vancomycin resistance transposon Tn<em>1549</em> from <em>Clostridium symbiosum</em> to <em>Enterococcus</em> spp. in the gut of gnotobiotic mice</div>, <div class="source">Antimicrob. Agents. Chemother.</div> , <div class="year">2006</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">1054</div>-<div class="lpage">1062</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20vancomycin%20resistance%20transposon%20Tn1549%20from%20Clostridium%20symbiosum%20to%20Enterococcus%20spp.%20in%20the%20gut%20of%20gnotobiotic%20mice&amp;author=A.%20Launay&amp;author=S.%20A.%20Ballard&amp;author=P.%20D.%20Johnson&amp;author=M.%20L.%20Grayson&amp;author=T%20Lambert&amp;publication_year=2006&amp;journal=Antimicrob.%20Agents.%20Chemother.&amp;volume=50&amp;pages=1054-1062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.50.3.1054-1062.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.50.3.1054-1062.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.50.3.1054-1062.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16495268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Franke</div> <div class="given-names">A. E.</div></div>,  <div class="name"><div class="surname">Clewell</div> <div class="given-names">D. B</div></div>. </div><div class="article-title">Evidence for a chromosome-borne resistance transposon (Tn<em>916</em>) in <em>Streptococcus faecalis</em> that is capable of “conjugal” transfer in the absence of a conjugative plasmid</div>, <div class="source">J. Bacteriol</div> , <div class="year">1981</div>, vol. <div class="volume">145</div> (pg. <div class="fpage">494</div>-<div class="lpage">502</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20a%20chromosome-borne%20resistance%20transposon%20(Tn916)%20in%20Streptococcus%20faecalis%20that%20is%20capable%20of%20%E2%80%9Cconjugal%E2%80%9D%20transfer%20in%20the%20absence%20of%20a%20conjugative%20plasmid&amp;author=A.%20E.%20Franke&amp;author=D.%20B%20Clewell&amp;publication_year=1981&amp;journal=J.%20Bacteriol&amp;volume=145&amp;pages=494-502" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6257641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSM018C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSM018C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yachi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Loreau</div> <div class="given-names">M</div></div>. </div><div class="article-title">Biodiversity and ecosystem productivity in a fluctuating environment: the insurance hypothesis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">1463</div>-<div class="lpage">1468</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biodiversity%20and%20ecosystem%20productivity%20in%20a%20fluctuating%20environment%3A%20the%20insurance%20hypothesis&amp;author=S.%20Yachi&amp;author=M%20Loreau&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=1463-1468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.4.1463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.4.1463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.4.1463"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2007. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=466883&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/14/4/10.1093_dnares_dsm018/1/dsm018_Supplementary_Data.zip?Expires=2147483647&amp;Signature=HJsZcmwNXRvVemCXdbVQNX8HB7-dJIdMyjMPD8JuoDwBn7c~WBAgFhcQr2WxMlMI34lEaTSeO2Dj1dEUU5LB01w-uh~HkFi1W8pMMDuj-s~pIn6xkv7zGWxcxi-2h8vTLAixleojCKJaXsMMqVRFrHsQQSpyd5eyCyyJq4fpq6ZzWFCD0LVmcsEn4ijDdsyGt3z5AndslZ0t4rBWIiUGkviS~k-ITcOu9cjJGXNS7Bm~c6Y8n7HQqgJdaTLkMBjz8HXeTTP-wh7vwG-zpnRDTQ~QdP1y~wv3mJXmhRnsCHeISSktjfgdms8EpN8cq~1kemAzP9iUvXJn21AzzcyLkw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,522</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">515</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsm018"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="466883"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000250863200003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f14%2f4%2f169%2f466883">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000250863200003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (515)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F14%2F4%2F169%2F466883" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/466883'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparative Metagenomics Revealed Commonly Enriched Gene Sets in Human Gut Microbiomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsm018");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f14%2f4%2f169%2f466883">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

