

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1690,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/4/227/375737","@type":"ScholarlyArticle","name":"Genome Structure of the Legume,  Lotus japonicus ","datePublished":"2008-05-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2008-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn008","keywords":["<italic>Lotus japonicus</italic>","genome structure","Fabaceae","comparative analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/4/227/375737","author":[{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakamura, Yasukazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kaneko, Takakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Asamizu, Erika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Tomohiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakao, Mitsuteru","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ono, Akiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kawashima, Kumiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fujishiro, Tsunakazu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Katoh, Midori","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakazaki, Naomi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shimizu, Yoshimi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shinpo, Sayaka","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Takahashi, Chika","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Wada, Tsuyuko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ohmido, Nobuko","affiliation":"2  Graduate School of Human Development and Environment ,  Kobe University, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Hayashi, Makoto","affiliation":"3  Department of Biotechnology ,  Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka ,  Japan","@type":"Person"},{"name":"Fukui, Kiichi","affiliation":"3  Department of Biotechnology ,  Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka ,  Japan","@type":"Person"},{"name":"Baba, Tomoya","affiliation":"4  Institute of Advanced Biosciences, Keio University ,  Tsuruoka, Yamagata 997-0017 ,  Japan","@type":"Person"},{"name":"Nakamichi, Tomoko","affiliation":"5  Graduate School of Biological Sciences, Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Mori, Hirotada","affiliation":"5  Graduate School of Biological Sciences, Nara Institute of Science and Technology ,  8916-5 Takayama, Ikoma, Nara 630-0192 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  The legume Lotus japonicus has been widely used as a model system to investigate the genetic background of legume-specific phenomena such as symbioti","pageStart":"227","pageEnd":"239","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome Structure of the Legume,  Lotus japonicus ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome Structure of the Legume,  Lotus japonicus " />
    <meta property="og:description" content="Abstract.  The legume Lotus japonicus has been widely used as a model system to investigate the genetic background of legume-specific phenomena such as symbioti" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/4/227/375737" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-05-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakamura, Yasukazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kaneko, Takakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Asamizu, Erika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Tomohiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakao, Mitsuteru" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ono, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawashima, Kumiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fujishiro, Tsunakazu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Katoh, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakazaki, Naomi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shimizu, Yoshimi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shinpo, Sayaka" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Takahashi, Chika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Wada, Tsuyuko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohmido, Nobuko" /><meta name="citation_author_institution" content="Graduate School of Human Development and Environment, Kobe University, Kobe 657-8501, Japan" /><meta name="citation_author" content="Hayashi, Makoto" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka, Japan" /><meta name="citation_author" content="Fukui, Kiichi" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka, Japan" /><meta name="citation_author" content="Baba, Tomoya" /><meta name="citation_author_institution" content="Institute of Advanced Biosciences, Keio University, Tsuruoka, Yamagata 997-0017, Japan" /><meta name="citation_author" content="Nakamichi, Tomoko" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Mori, Hirotada" /><meta name="citation_author_institution" content="Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0192, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Genome Structure of the Legume, Lotus japonicus" /><meta name="citation_firstpage" content="227" /><meta name="citation_lastpage" content="239" /><meta name="citation_doi" content="10.1093/dnares/dsn008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2008/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Characteristics of the Lotus japonicus gene repertoire deduced from large-scale expressed sequence tag (EST) analysis; Plant Mol. Biol.;  citation_year=2004;  citation_volume=54;  citation_pages=405-414; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map of the model legume Lotus japonicus using an intraspecific F2 population; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=301-310; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the model legume Lotus japonicus and strategies for fast mapping of new loci; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=1673-1683; " /><meta name="citation_reference" content="citation_title=Genetics of symbiosis in Lotus japonicus: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci; Mol. Plant Microbe Interact.;  citation_year=2006;  citation_volume=19;  citation_pages=80-91; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=311-318; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. II. Sequence features and mapping of sixty-five TAC clones which cover the 6.5 Mb regions of the genome; DNA Res.;  citation_year=2002;  citation_volume=9;  citation_pages=63-70; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. III. Sequence features and mapping of sixty-two TAC clones which cover the 6.7 Mb regions of the genome; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=27-33; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. IV. Sequence features and mapping of seventy-three TAC clones which cover the 7.5 Mb regions of the genome; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=115-122; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. V. Sequence features and mapping of sixty-four TAC clones which cover the 6.4 Mb regions of the genome; DNA Res.;  citation_year=2003;  citation_volume=10;  citation_pages=277-285; " /><meta name="citation_reference" content="citation_title=‘Miyakojima’ MG-20: an early-flowering accession suitable for indoor handling, J. Plant Res.; Lotus japonicus;  citation_year=2000;  citation_volume=113;  citation_pages=507-509; " /><meta name="citation_reference" content="citation_title=Whole-genome sequence assembly for mammalian genomes: Arachne 2; Genome Res.;  citation_year=2003;  citation_volume=13,;  citation_pages=91-96; " /><meta name="citation_reference" content="citation_title=Comprehensive structural analysis of the genome of red clover (Trifolium pratense L.); DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=301-364; " /><meta name="citation_reference" content="citation_title=Development of a quantitative pachytene chromosome map in Oryza sativa by imaging methods; Genes Genet. Syst.;  citation_year=2003;  citation_volume=78;  citation_pages=155-161; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucl. Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1269-1276; " /><meta name="citation_reference" content="citation_title=The transposable element landscape of the model legume Lotus japonicus; Genetics;  citation_year=2006;  citation_volume=174;  citation_pages=2215-2228; " /><meta name="citation_reference" content="citation_title=GenBank; Nucl. Acids Res.;  citation_year=2007;  citation_volume=32;  citation_pages=21-25; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucl. Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-964; " /><meta name="citation_reference" content="citation_title=A computational screen for methylation guide snoRNAs in yeast; Science;  citation_year=1999;  citation_volume=283;  citation_pages=1168-1171; " /><meta name="citation_reference" content="citation_title=Prediction of signal recognition particle RNA genes; Nucl. Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=3368-3377; " /><meta name="citation_reference" content="citation_title=The microRNA registry; citation_text=(Database issue); Nucl. Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D109-D111; " /><meta name="citation_reference" content="citation_title=Computational identification of plant microRNAs and their targets, including a stress-induced miRNA; Mol. Cell;  citation_year=2004;  citation_volume=14;  citation_pages=787-799; " /><meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding; Nucl. Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=1107-1115; " /><meta name="citation_reference" content="citation_title=Prediction of complete gene structures in human genomic DNA; J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=78-94; " /><meta name="citation_reference" content="citation_title=Locating protein-coding regions in human DNA sequences by a multiple sensor-neural network approach; Proc. Natl. Acad. Sci., USA;  citation_year=1991;  citation_volume=88;  citation_pages=11261-11265; " /><meta name="citation_reference" content="citation_title=Splice site prediction in Arabidopsis thaliana DNA by combining local and global sequence information; Nucl. Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=3439-3452; " /><meta name="citation_reference" content="citation_title=Prediction of locally optimal splice sites in plant pre-mRNA with applications to gene identification in Arabidopsis thaliana genomic DNA; Nucl. Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=4748-4757; " /><meta name="citation_reference" content="citation_title=The universal protein resource (UniProt); citation_author=UniProt Consortium; Nucl. Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=193-197; " /><meta name="citation_reference" content="citation_title=A tool for analyzing and annotating genomic sequences; Genomics;  citation_year=1997;  citation_volume=46;  citation_pages=37-45; " /><meta name="citation_reference" content="citation_title=The TIGR gene indices: clustering and assembling EST and known genes and integration with eukaryotic genomes; Nucl. Acids Res.;  citation_year=2005;  citation_volume=33;  citation_issue=Database issue;  citation_pages=D71-D74; " /><meta name="citation_reference" content="citation_title=A general method applicable to the search for similarities in the amino acid sequence of two proteins; J. Mol. Biol.;  citation_year=1970;  citation_volume=48;  citation_pages=443-453; " /><meta name="citation_reference" content="citation_title=An overview of sequence comparison; citation_publisher=Addison Wesley, Boston; Time Warps, String Edits and Macromolecules: the Theory and Practice of Sequence Comparison;  citation_year=1983;  citation_pages=1-44; " /><meta name="citation_reference" content="citation_title=Plant cis-acting regulatory DNA elements (PLACE) database; Nucl. Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=297-300; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis Genome Initiative; Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project; Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray); Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-1604; " /><meta name="citation_reference" content="citation_title=The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla; citation_author=The French–Italian Public Consortium for Grapevine Genome Characterization; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=463-467; " /><meta name="citation_reference" content="citation_title=New developments in the InterPro database; Nucl. Acids Res.;  citation_year=2007;  citation_volume=35;  citation_issue=Database issue;  citation_pages=D224-D228; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=The Gene Ontology Consortium; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-29; " /><meta name="citation_reference" content="citation_title=PSORT: a program for detecting sorting signals in proteins and predicting their subcellular localization; Trends Biochem. Sci.;  citation_year=1999;  citation_volume=24;  citation_pages=34-36; " /><meta name="citation_reference" content="citation_title=Predicting subcellular localization of proteins based on their N-terminal amino acid sequence; J. Mol. Biol.;  citation_year=2000;  citation_volume=300;  citation_pages=1005-1016; " /><meta name="citation_reference" content="citation_title=SOSUI: classification and secondary structure prediction system for membrane proteins; Bioinformatics;  citation_year=1998;  citation_volume=14;  citation_pages=378-379; " /><meta name="citation_reference" content="citation_title=Genome and chromosome dimensions of Lotus japonicus; J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_pages=435-442; " /><meta name="citation_reference" content="citation_title=Chromosomal map of the model legume Lotus japonicus; Genetics;  citation_year=2002;  citation_volume=161;  citation_pages=1661-1672; " /><meta name="citation_reference" content="citation_title=A SINE family widely distributed in the plant kingdom and its evolutionary history; Plant Mol. Biol.;  citation_year=2006;  citation_volume=61;  citation_pages=505-514; " /><meta name="citation_reference" content="citation_title=Empirical analysis of transcriptional activity in the Arabidopsis genome; Science;  citation_year=2003;  citation_volume=302;  citation_pages=842-846; " /><meta name="citation_reference" content="citation_title=Widespread occurrence of antisense transcription in the human genome; Nat. Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=379-386; " /><meta name="citation_reference" content="citation_title=Comparison of the transcript profiles from the root and the nodulating root of the model legume Lotus japonicus by serial analysis of gene expression; Mol. Plant Microbe Interact.;  citation_year=2005;  citation_volume=18;  citation_pages=487-498; " /><meta name="citation_reference" content="citation_title=Antiquity of microRNAs and their targets in land plants; Plant Cell;  citation_year=2005;  citation_volume=17;  citation_pages=1658-1673; " /><meta name="citation_reference" content="citation_title=The ARF family of transcription factors and their role in plant hormone-responsive transcription; Cell Mol. Life Sci.;  citation_year=1998;  citation_volume=54;  citation_pages=619-627; " /><meta name="citation_reference" content="citation_title=A novel family of calmodulin-binding transcription activators in multicellular organisms; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=21851-21861; " /><meta name="citation_reference" content="citation_title=Molecular characterisation of the Arabidopsis SBP-box genes; Gene;  citation_year=1999;  citation_volume=237;  citation_pages=91-104; " /><meta name="citation_reference" content="citation_title=Two-component signal transduction pathways in Arabidopsis; Plant Physiol.;  citation_year=2002;  citation_volume=129;  citation_pages=500-515; " /><meta name="citation_reference" content="citation_title=A cytokinin perception mutant colonized by Rhizobium in the absence of nodule organogenesis; Science;  citation_year=2007;  citation_volume=315;  citation_pages=101-104; " /><meta name="citation_reference" content="citation_title=A gain-of-function mutation in a cytokinin receptor triggers spontaneous root nodule organogenesis; Science;  citation_year=2007;  citation_volume=315;  citation_pages=104-107; " /><meta name="citation_reference" content="citation_title=Transgenic Lotus japonicus with an ethylene receptor gene Cm-ERS1/H70A enhances formation of infection threads and nodule primordia; Plant Cell Physiol.;  citation_year=2004;  citation_volume=45;  citation_pages=427-435; " /><meta name="citation_reference" content="citation_title=Pseudo-response regulators (PRRs) or true oscillator components (TOCs); Plant Cell Physiol.;  citation_year=2005;  citation_volume=46;  citation_pages=677-685; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the receptor-like kinase family in Arabidopsis and rice; Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=1220-1234; " /><meta name="citation_reference" content="citation_title=A plant receptor-like kinase required for both bacterial and fungal symbiosis; Nature;  citation_year=2002;  citation_volume=417;  citation_pages=959-962; " /><meta name="citation_reference" content="citation_title=Shoot control of root development and nodulation is mediated by a receptor-like kinase; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=422-426; " /><meta name="citation_reference" content="citation_title=HAR1 mediates systemic regulation of symbiotic organ development; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=426-429; " /><meta name="citation_reference" content="citation_title=Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=585-592; " /><meta name="citation_reference" content="citation_title=Molecular evolution of lysin motif-type receptor-like kinases in plants; Plant Physiol.;  citation_year=2007;  citation_volume=144;  citation_pages=623-636; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of ATP-binding cassette (ABC) proteins in a model legume plant, Lotus japonicus: comparison with Arabidopsis ABC protein family; DNA Res.;  citation_year=2006;  citation_volume=13;  citation_pages=205-228; " /><meta name="citation_reference" content="citation_title=Water-selective and multifunctional aquaporins from Lotus japonicus nodules, (Abstract); Planta;  citation_year=2000;  citation_volume=210;  citation_pages=741-748; " /><meta name="citation_reference" content="citation_title=Proteome analysis. Novel proteins identified at the peribacteroid membrane from Lotus japonicus root nodules; Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_pages=1080-1090; " /><meta name="citation_reference" content="citation_title=Comparative genomics of rice and Arabidopsis. Analysis of 727 cytochrome P450 genes and pseudogenes from a monocot and a dicot; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=756-772; " /><meta name="citation_reference" content="citation_title=Genome-wide analyses of the structural gene families involved in the legume-specific 5-deoxyisoflavonoid biosynthesis of Lotus japonicus; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=25-36; " /><meta name="citation_reference" content="citation_title=Legume genome evolution viewed through the Medicago truncatula and Lotus japonicus genomes; Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=14959-14964; " /><meta name="citation_reference" content="citation_title=Exploitation of colinear relationships between the genomes of Lotus japonicus, Pisum sativum and Arabidopsis thaliana, for positional cloning of a legume symbiosis gene; Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=442-449; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/4/227/1067935/dsn008.pdf" /><meta name="description" content="Abstract.  The legume Lotus japonicus has been widely used as a model system to investigate the genetic background of legume-specific phenomena such as symbioti" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/4/227/375737" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/4/227/375737" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/4/227/375737");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn008';
    </script>
<title>
	Genome Structure of the Legume, Lotus japonicus | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./375737" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbK5ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgU2h1c2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FtdXJhLCBZYXN1a2F6dSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5la28sIFRha2FrYXp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFzYW1penUsIEVyaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthdG8sIFRvbW9oaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FvLCBNaXRzdXRlcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FzYW1vdG8sIFNoaWdlbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2F0YW5hYmUsIEFraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ubywgQWtpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YXNoaW1hLCBLdW1pa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVqaXNoaXJvLCBUc3VuYWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0b2gsIE1pZG9yaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hhcmEsIE1pdHN1eW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2lzaGlkYSwgWW9zaGllIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbmFtaSwgQ2hpaGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthemFraSwgTmFvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dSwgWW9zaGltaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlucG8sIFNheWFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaGFzaGksIENoaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhZGEsIFRzdXl1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYWRhLCBNYW5hYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2htaWRvLCBOb2J1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgSHVtYW4gRGV2ZWxvcG1lbnQgYW5kIEVudmlyb25tZW50LCBLb2JlIFVuaXZlcnNpdHksIEtvYmUgNjU3LTg1MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF5YXNoaSwgTWFrb3RvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRW5naW5lZXJpbmcsIE9zYWthIFVuaXZlcnNpdHkgMi0xIFlhbWFkYW9rYSwgU3VpdGEgNTY1LTA4NzEsIE9zYWthLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1a3VpLCBLaWljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBFbmdpbmVlcmluZywgT3Nha2EgVW5pdmVyc2l0eSAyLTEgWWFtYWRhb2thLCBTdWl0YSA1NjUtMDg3MSwgT3Nha2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFiYSwgVG9tb3lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEFkdmFuY2VkIEJpb3NjaWVuY2VzLCBLZWlvIFVuaXZlcnNpdHksIFRzdXJ1b2thLCBZYW1hZ2F0YSA5OTctMDAxNywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthbWljaGksIFRvbW9rbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzLCBOYXJhIEluc3RpdHV0ZSBvZiBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCA4OTE2LTUgVGFrYXlhbWEsIElrb21hLCBOYXJhIDYzMC0wMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmksIEhpcm90YWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIE5hcmEgSW5zdGl0dXRlIG9mIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIDg5MTYtNSBUYWtheWFtYSwgSWtvbWEsIE5hcmEgNjMwLTAxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lIFN0cnVjdHVyZSBvZiB0aGUgTGVndW1lLCBMb3R1cyBqYXBvbmljdXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIzOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbjAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXN0aWNzIG9mIHRoZSBMb3R1cyBqYXBvbmljdXMgZ2VuZSByZXBlcnRvaXJlIGRlZHVjZWQgZnJvbSBsYXJnZS1zY2FsZSBleHByZXNzZWQgc2VxdWVuY2UgdGFnIChFU1QpIGFuYWx5c2lzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NDA1LTQxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcCBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyB1c2luZyBhbiBpbnRyYXNwZWNpZmljIEYyIHBvcHVsYXRpb247IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMDEtMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMgYW5kIHN0cmF0ZWdpZXMgZm9yIGZhc3QgbWFwcGluZyBvZiBuZXcgbG9jaTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25fcGFnZXM9MTY3My0xNjgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBzeW1iaW9zaXMgaW4gTG90dXMgamFwb25pY3VzOiByZWNvbWJpbmFudCBpbmJyZWQgbGluZXMsIGNvbXBhcmF0aXZlIGdlbmV0aWMgbWFwcywgYW5kIG1hcCBwb3NpdGlvbiBvZiAzNSBzeW1iaW90aWMgbG9jaTsgTW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTgwLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIGEgTG90dXMgamFwb25pY3VzIGdlbm9tZS4gSS4gU2VxdWVuY2UgZmVhdHVyZXMgYW5kIG1hcHBpbmcgb2YgZmlmdHktc2l4IFRBQyBjbG9uZXMgd2hpY2ggY292ZXIgdGhlIDUuNCBNYiByZWdpb25zIG9mIHRoZSBnZW5vbWU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMzE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIGEgTG90dXMgamFwb25pY3VzIGdlbm9tZS4gSUkuIFNlcXVlbmNlIGZlYXR1cmVzIGFuZCBtYXBwaW5nIG9mIHNpeHR5LWZpdmUgVEFDIGNsb25lcyB3aGljaCBjb3ZlciB0aGUgNi41IE1iIHJlZ2lvbnMgb2YgdGhlIGdlbm9tZTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTYzLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIGEgTG90dXMgamFwb25pY3VzIGdlbm9tZS4gSUlJLiBTZXF1ZW5jZSBmZWF0dXJlcyBhbmQgbWFwcGluZyBvZiBzaXh0eS10d28gVEFDIGNsb25lcyB3aGljaCBjb3ZlciB0aGUgNi43IE1iIHJlZ2lvbnMgb2YgdGhlIGdlbm9tZTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmFseXNpcyBvZiBhIExvdHVzIGphcG9uaWN1cyBnZW5vbWUuIElWLiBTZXF1ZW5jZSBmZWF0dXJlcyBhbmQgbWFwcGluZyBvZiBzZXZlbnR5LXRocmVlIFRBQyBjbG9uZXMgd2hpY2ggY292ZXIgdGhlIDcuNSBNYiByZWdpb25zIG9mIHRoZSBnZW5vbWU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTE1LTEyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmFseXNpcyBvZiBhIExvdHVzIGphcG9uaWN1cyBnZW5vbWUuIFYuIFNlcXVlbmNlIGZlYXR1cmVzIGFuZCBtYXBwaW5nIG9mIHNpeHR5LWZvdXIgVEFDIGNsb25lcyB3aGljaCBjb3ZlciB0aGUgNi40IE1iIHJlZ2lvbnMgb2YgdGhlIGdlbm9tZTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNzctMjg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT3igJhNaXlha29qaW1h4oCZIE1HLTIwOiBhbiBlYXJseS1mbG93ZXJpbmcgYWNjZXNzaW9uIHN1aXRhYmxlIGZvciBpbmRvb3IgaGFuZGxpbmcsIEouIFBsYW50IFJlcy47IExvdHVzIGphcG9uaWN1czsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz01MDctNTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2UgYXNzZW1ibHkgZm9yIG1hbW1hbGlhbiBnZW5vbWVzOiBBcmFjaG5lIDI7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzLDsgIGNpdGF0aW9uX3BhZ2VzPTkxLTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIHN0cnVjdHVyYWwgYW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBvZiByZWQgY2xvdmVyIChUcmlmb2xpdW0gcHJhdGVuc2UgTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMwMS0zNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgcXVhbnRpdGF0aXZlIHBhY2h5dGVuZSBjaHJvbW9zb21lIG1hcCBpbiBPcnl6YSBzYXRpdmEgYnkgaW1hZ2luZyBtZXRob2RzOyBHZW5lcyBHZW5ldC4gU3lzdC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0xNTUtMTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBkZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBzZXF1ZW5jZSBmYW1pbGllcyBpbiBzZXF1ZW5jZWQgZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjY5LTEyNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc3Bvc2FibGUgZWxlbWVudCBsYW5kc2NhcGUgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3NDsgIGNpdGF0aW9uX3BhZ2VzPTIyMTUtMjIyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuQmFuazsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTIxLTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtOTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXB1dGF0aW9uYWwgc2NyZWVuIGZvciBtZXRoeWxhdGlvbiBndWlkZSBzbm9STkFzIGluIHllYXN0OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI4MzsgIGNpdGF0aW9uX3BhZ2VzPTExNjgtMTE3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiBzaWduYWwgcmVjb2duaXRpb24gcGFydGljbGUgUk5BIGdlbmVzOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzM2OC0zMzc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWljcm9STkEgcmVnaXN0cnk7IGNpdGF0aW9uX3RleHQ9KERhdGFiYXNlIGlzc3VlKTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPUQxMDktRDExMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBpZGVudGlmaWNhdGlvbiBvZiBwbGFudCBtaWNyb1JOQXMgYW5kIHRoZWlyIHRhcmdldHMsIGluY2x1ZGluZyBhIHN0cmVzcy1pbmR1Y2VkIG1pUk5BOyBNb2wuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03ODctNzk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lTWFyay5obW06IG5ldyBzb2x1dGlvbnMgZm9yIGdlbmUgZmluZGluZzsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTExMDctMTExNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiBjb21wbGV0ZSBnZW5lIHN0cnVjdHVyZXMgaW4gaHVtYW4gZ2Vub21pYyBETkE7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjY4OyAgY2l0YXRpb25fcGFnZXM9NzgtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2F0aW5nIHByb3RlaW4tY29kaW5nIHJlZ2lvbnMgaW4gaHVtYW4gRE5BIHNlcXVlbmNlcyBieSBhIG11bHRpcGxlIHNlbnNvci1uZXVyYWwgbmV0d29yayBhcHByb2FjaDsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiwgVVNBOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MTEyNjEtMTEyNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljZSBzaXRlIHByZWRpY3Rpb24gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgRE5BIGJ5IGNvbWJpbmluZyBsb2NhbCBhbmQgZ2xvYmFsIHNlcXVlbmNlIGluZm9ybWF0aW9uOyBOdWNsLiBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MzQzOS0zNDUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIGxvY2FsbHkgb3B0aW1hbCBzcGxpY2Ugc2l0ZXMgaW4gcGxhbnQgcHJlLW1STkEgd2l0aCBhcHBsaWNhdGlvbnMgdG8gZ2VuZSBpZGVudGlmaWNhdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBnZW5vbWljIEROQTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQ3NDgtNDc1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVuaXZlcnNhbCBwcm90ZWluIHJlc291cmNlIChVbmlQcm90KTsgY2l0YXRpb25fYXV0aG9yPVVuaVByb3QgQ29uc29ydGl1bTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTE5My0xOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdG9vbCBmb3IgYW5hbHl6aW5nIGFuZCBhbm5vdGF0aW5nIGdlbm9taWMgc2VxdWVuY2VzOyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgVElHUiBnZW5lIGluZGljZXM6IGNsdXN0ZXJpbmcgYW5kIGFzc2VtYmxpbmcgRVNUIGFuZCBrbm93biBnZW5lcyBhbmQgaW50ZWdyYXRpb24gd2l0aCBldWthcnlvdGljIGdlbm9tZXM7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9pc3N1ZT1EYXRhYmFzZSBpc3N1ZTsgIGNpdGF0aW9uX3BhZ2VzPUQ3MS1ENzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtZXRob2QgYXBwbGljYWJsZSB0byB0aGUgc2VhcmNoIGZvciBzaW1pbGFyaXRpZXMgaW4gdGhlIGFtaW5vIGFjaWQgc2VxdWVuY2Ugb2YgdHdvIHByb3RlaW5zOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTcwOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9NDQzLTQ1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gb3ZlcnZpZXcgb2Ygc2VxdWVuY2UgY29tcGFyaXNvbjsgY2l0YXRpb25fcHVibGlzaGVyPUFkZGlzb24gV2VzbGV5LCBCb3N0b247IFRpbWUgV2FycHMsIFN0cmluZyBFZGl0cyBhbmQgTWFjcm9tb2xlY3VsZXM6IHRoZSBUaGVvcnkgYW5kIFByYWN0aWNlIG9mIFNlcXVlbmNlIENvbXBhcmlzb247ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICBjaXRhdGlvbl9wYWdlcz0xLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBjaXMtYWN0aW5nIHJlZ3VsYXRvcnkgRE5BIGVsZW1lbnRzIChQTEFDRSkgZGF0YWJhc2U7IE51Y2wuIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yOTctMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IGNpdGF0aW9uX2F1dGhvcj1BcmFiaWRvcHNpcyBHZW5vbWUgSW5pdGlhdGl2ZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwODsgIGNpdGF0aW9uX3BhZ2VzPTc5Ni04MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYXAtYmFzZWQgc2VxdWVuY2Ugb2YgdGhlIHJpY2UgZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9SW50ZXJuYXRpb25hbCBSaWNlIEdlbm9tZSBTZXF1ZW5jaW5nIFByb2plY3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzY7ICBjaXRhdGlvbl9wYWdlcz03OTMtODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIGJsYWNrIGNvdHRvbndvb2QsIFBvcHVsdXMgdHJpY2hvY2FycGEgKFRvcnIuICZhbXA7IEdyYXkpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTMxMzsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtMTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdyYXBldmluZSBnZW5vbWUgc2VxdWVuY2Ugc3VnZ2VzdHMgYW5jZXN0cmFsIGhleGFwbG9pZGl6YXRpb24gaW4gbWFqb3IgYW5naW9zcGVybSBwaHlsYTsgY2l0YXRpb25fYXV0aG9yPVRoZSBGcmVuY2jigJNJdGFsaWFuIFB1YmxpYyBDb25zb3J0aXVtIGZvciBHcmFwZXZpbmUgR2Vub21lIENoYXJhY3Rlcml6YXRpb247IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDk7ICBjaXRhdGlvbl9wYWdlcz00NjMtNDY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZGV2ZWxvcG1lbnRzIGluIHRoZSBJbnRlclBybyBkYXRhYmFzZTsgTnVjbC4gQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX2lzc3VlPURhdGFiYXNlIGlzc3VlOyAgY2l0YXRpb25fcGFnZXM9RDIyNC1EMjI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neTsgY2l0YXRpb25fYXV0aG9yPVRoZSBHZW5lIE9udG9sb2d5IENvbnNvcnRpdW07IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjUtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBTT1JUOiBhIHByb2dyYW0gZm9yIGRldGVjdGluZyBzb3J0aW5nIHNpZ25hbHMgaW4gcHJvdGVpbnMgYW5kIHByZWRpY3RpbmcgdGhlaXIgc3ViY2VsbHVsYXIgbG9jYWxpemF0aW9uOyBUcmVuZHMgQmlvY2hlbS4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM0LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHN1YmNlbGx1bGFyIGxvY2FsaXphdGlvbiBvZiBwcm90ZWlucyBiYXNlZCBvbiB0aGVpciBOLXRlcm1pbmFsIGFtaW5vIGFjaWQgc2VxdWVuY2U7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MzAwOyAgY2l0YXRpb25fcGFnZXM9MTAwNS0xMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT1NVSTogY2xhc3NpZmljYXRpb24gYW5kIHNlY29uZGFyeSBzdHJ1Y3R1cmUgcHJlZGljdGlvbiBzeXN0ZW0gZm9yIG1lbWJyYW5lIHByb3RlaW5zOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM3OC0zNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBhbmQgY2hyb21vc29tZSBkaW1lbnNpb25zIG9mIExvdHVzIGphcG9uaWN1czsgSi4gUGxhbnQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz00MzUtNDQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXM7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTE2NjEtMTY3MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBTSU5FIGZhbWlseSB3aWRlbHkgZGlzdHJpYnV0ZWQgaW4gdGhlIHBsYW50IGtpbmdkb20gYW5kIGl0cyBldm9sdXRpb25hcnkgaGlzdG9yeTsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02MTsgIGNpdGF0aW9uX3BhZ2VzPTUwNS01MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVtcGlyaWNhbCBhbmFseXNpcyBvZiB0cmFuc2NyaXB0aW9uYWwgYWN0aXZpdHkgaW4gdGhlIEFyYWJpZG9wc2lzIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDI7ICBjaXRhdGlvbl9wYWdlcz04NDItODQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIG9jY3VycmVuY2Ugb2YgYW50aXNlbnNlIHRyYW5zY3JpcHRpb24gaW4gdGhlIGh1bWFuIGdlbm9tZTsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM3OS0zODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdGhlIHRyYW5zY3JpcHQgcHJvZmlsZXMgZnJvbSB0aGUgcm9vdCBhbmQgdGhlIG5vZHVsYXRpbmcgcm9vdCBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1cyBieSBzZXJpYWwgYW5hbHlzaXMgb2YgZ2VuZSBleHByZXNzaW9uOyBNb2wuIFBsYW50IE1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NDg3LTQ5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aXF1aXR5IG9mIG1pY3JvUk5BcyBhbmQgdGhlaXIgdGFyZ2V0cyBpbiBsYW5kIHBsYW50czsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE2NTgtMTY3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFSRiBmYW1pbHkgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGFuZCB0aGVpciByb2xlIGluIHBsYW50IGhvcm1vbmUtcmVzcG9uc2l2ZSB0cmFuc2NyaXB0aW9uOyBDZWxsIE1vbC4gTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NjE5LTYyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBmYW1pbHkgb2YgY2FsbW9kdWxpbi1iaW5kaW5nIHRyYW5zY3JpcHRpb24gYWN0aXZhdG9ycyBpbiBtdWx0aWNlbGx1bGFyIG9yZ2FuaXNtczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MjE4NTEtMjE4NjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpc2F0aW9uIG9mIHRoZSBBcmFiaWRvcHNpcyBTQlAtYm94IGdlbmVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzNzsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvLWNvbXBvbmVudCBzaWduYWwgdHJhbnNkdWN0aW9uIHBhdGh3YXlzIGluIEFyYWJpZG9wc2lzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz01MDAtNTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGN5dG9raW5pbiBwZXJjZXB0aW9uIG11dGFudCBjb2xvbml6ZWQgYnkgUmhpem9iaXVtIGluIHRoZSBhYnNlbmNlIG9mIG5vZHVsZSBvcmdhbm9nZW5lc2lzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMS0xMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Fpbi1vZi1mdW5jdGlvbiBtdXRhdGlvbiBpbiBhIGN5dG9raW5pbiByZWNlcHRvciB0cmlnZ2VycyBzcG9udGFuZW91cyByb290IG5vZHVsZSBvcmdhbm9nZW5lc2lzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTEwNC0xMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZ2VuaWMgTG90dXMgamFwb25pY3VzIHdpdGggYW4gZXRoeWxlbmUgcmVjZXB0b3IgZ2VuZSBDbS1FUlMxL0g3MEEgZW5oYW5jZXMgZm9ybWF0aW9uIG9mIGluZmVjdGlvbiB0aHJlYWRzIGFuZCBub2R1bGUgcHJpbW9yZGlhOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NDI3LTQzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHNldWRvLXJlc3BvbnNlIHJlZ3VsYXRvcnMgKFBSUnMpIG9yIHRydWUgb3NjaWxsYXRvciBjb21wb25lbnRzIChUT0NzKTsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTY3Ny02ODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSByZWNlcHRvci1saWtlIGtpbmFzZSBmYW1pbHkgaW4gQXJhYmlkb3BzaXMgYW5kIHJpY2U7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0xMjIwLTEyMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGxhbnQgcmVjZXB0b3ItbGlrZSBraW5hc2UgcmVxdWlyZWQgZm9yIGJvdGggYmFjdGVyaWFsIGFuZCBmdW5nYWwgc3ltYmlvc2lzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE3OyAgY2l0YXRpb25fcGFnZXM9OTU5LTk2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hvb3QgY29udHJvbCBvZiByb290IGRldmVsb3BtZW50IGFuZCBub2R1bGF0aW9uIGlzIG1lZGlhdGVkIGJ5IGEgcmVjZXB0b3ItbGlrZSBraW5hc2U7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjA7ICBjaXRhdGlvbl9wYWdlcz00MjItNDI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IQVIxIG1lZGlhdGVzIHN5c3RlbWljIHJlZ3VsYXRpb24gb2Ygc3ltYmlvdGljIG9yZ2FuIGRldmVsb3BtZW50OyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDIwOyAgY2l0YXRpb25fcGFnZXM9NDI2LTQyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgcmVjb2duaXRpb24gb2Ygc3ltYmlvdGljIGJhY3RlcmlhIHJlcXVpcmVzIHR3byBMeXNNIHJlY2VwdG9yLWxpa2Uga2luYXNlczsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyNTsgIGNpdGF0aW9uX3BhZ2VzPTU4NS01OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldm9sdXRpb24gb2YgbHlzaW4gbW90aWYtdHlwZSByZWNlcHRvci1saWtlIGtpbmFzZXMgaW4gcGxhbnRzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz02MjMtNjM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBBVFAtYmluZGluZyBjYXNzZXR0ZSAoQUJDKSBwcm90ZWlucyBpbiBhIG1vZGVsIGxlZ3VtZSBwbGFudCwgTG90dXMgamFwb25pY3VzOiBjb21wYXJpc29uIHdpdGggQXJhYmlkb3BzaXMgQUJDIHByb3RlaW4gZmFtaWx5OyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0yMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdhdGVyLXNlbGVjdGl2ZSBhbmQgbXVsdGlmdW5jdGlvbmFsIGFxdWFwb3JpbnMgZnJvbSBMb3R1cyBqYXBvbmljdXMgbm9kdWxlcywgKEFic3RyYWN0KTsgUGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTIxMDsgIGNpdGF0aW9uX3BhZ2VzPTc0MS03NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3Rlb21lIGFuYWx5c2lzLiBOb3ZlbCBwcm90ZWlucyBpZGVudGlmaWVkIGF0IHRoZSBwZXJpYmFjdGVyb2lkIG1lbWJyYW5lIGZyb20gTG90dXMgamFwb25pY3VzIHJvb3Qgbm9kdWxlczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTMxOyAgY2l0YXRpb25fcGFnZXM9MTA4MC0xMDkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBvZiByaWNlIGFuZCBBcmFiaWRvcHNpcy4gQW5hbHlzaXMgb2YgNzI3IGN5dG9jaHJvbWUgUDQ1MCBnZW5lcyBhbmQgcHNldWRvZ2VuZXMgZnJvbSBhIG1vbm9jb3QgYW5kIGEgZGljb3Q7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTc1Ni03NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2VzIG9mIHRoZSBzdHJ1Y3R1cmFsIGdlbmUgZmFtaWxpZXMgaW52b2x2ZWQgaW4gdGhlIGxlZ3VtZS1zcGVjaWZpYyA1LWRlb3h5aXNvZmxhdm9ub2lkIGJpb3N5bnRoZXNpcyBvZiBMb3R1cyBqYXBvbmljdXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxlZ3VtZSBnZW5vbWUgZXZvbHV0aW9uIHZpZXdlZCB0aHJvdWdoIHRoZSBNZWRpY2FnbyB0cnVuY2F0dWxhIGFuZCBMb3R1cyBqYXBvbmljdXMgZ2Vub21lczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MTQ5NTktMTQ5NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvaXRhdGlvbiBvZiBjb2xpbmVhciByZWxhdGlvbnNoaXBzIGJldHdlZW4gdGhlIGdlbm9tZXMgb2YgTG90dXMgamFwb25pY3VzLCBQaXN1bSBzYXRpdnVtIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYSwgZm9yIHBvc2l0aW9uYWwgY2xvbmluZyBvZiBhIGxlZ3VtZSBzeW1iaW9zaXMgZ2VuZTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz00NDItNDQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNC8yMjcvMTA2NzkzNS9kc24wMDgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBsZWd1bWUgTG90dXMgamFwb25pY3VzIGhhcyBiZWVuIHdpZGVseSB1c2VkIGFzIGEgbW9kZWwgc3lzdGVtIHRvIGludmVzdGlnYXRlIHRoZSBnZW5ldGljIGJhY2tncm91bmQgb2YgbGVndW1lLXNwZWNpZmljIHBoZW5vbWVuYSBzdWNoIGFzIHN5bWJpb3RpIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzU3MzcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkt77WD4mjZH6kD+mFeaA92xHamVg=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f227%2f375737" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f227%2f375737" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14644" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&amp;Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/4">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5468708">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468708">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468710">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468710">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468713">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468713">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468740">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468740">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468790">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468790">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468792">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468792">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468794">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468794">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5468796">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5468796">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/4/215/376506">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/4/241/376722">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome Structure of the Legume, <em>Lotus japonicus</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasukazu Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasukazu+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yasukazu Nakamura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takakazu Kaneko</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takakazu+Kaneko">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kaneko T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kaneko T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takakazu Kaneko</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Erika Asamizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Erika+Asamizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asamizu E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Asamizu E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Erika Asamizu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomohiko Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomohiko+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tomohiko Kato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuteru Nakao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuteru+Nakao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakao M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakao M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mitsuteru Nakao</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Ono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Ono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ono A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Ono</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kumiko Kawashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kumiko+Kawashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawashima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawashima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kumiko Kawashima</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsunakazu Fujishiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsunakazu+Fujishiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujishiro T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujishiro T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsunakazu Fujishiro</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Katoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Katoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katoh M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katoh M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Midori Katoh</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minami C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naomi Nakazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naomi+Nakazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakazaki N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Naomi Nakazaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshimi Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshimi+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshimi Shimizu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sayaka Shinpo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sayaka+Shinpo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shinpo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shinpo S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sayaka Shinpo</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chika Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chika+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chika Takahashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyuko Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyuko+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsuyuko Wada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuko Ohmido</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Graduate School of Human Development and Environment</div>, <div class="insititution">Kobe University, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuko+Ohmido">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohmido N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohmido N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuko Ohmido</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Makoto Hayashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiichi Fukui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University 2-1 Yamadaoka, Suita 565-0871, Osaka</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kiichi+Fukui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukui K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukui K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kiichi Fukui</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoya Baba</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Institute of Advanced Biosciences, Keio University</div>, <div class="addr-line">Tsuruoka, Yamagata 997-0017</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoya+Baba">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Baba T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Baba T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomoya Baba</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoko Nakamichi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Graduate School of Biological Sciences, Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoko+Nakamichi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamichi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamichi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomoko Nakamichi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirotada Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Graduate School of Biological Sciences, Nara Institute of Science and Technology</div>, <div class="addr-line">8916-5 Takayama, Ikoma, Nara 630-0192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirotada+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hirotada Mori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3933. Fax. <div class="fax">+81 438-52-3934</div>. E-mail: <a href="mailto:tabata@kazusa.or.jp" target="_blank">tabata@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 227–239, <a href='https://doi.org/10.1093/dnares/dsn008'>https://doi.org/10.1093/dnares/dsn008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 May 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">01 April 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 April 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="375737" href="/dnaresearch/article-pdf/15/4/227/1067935/dsn008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shusei Sato, Yasukazu Nakamura, Takakazu Kaneko, Erika Asamizu, Tomohiko Kato, Mitsuteru Nakao, Shigemi Sasamoto, Akiko Watanabe, Akiko Ono, Kumiko Kawashima, Tsunakazu Fujishiro, Midori Katoh, Mitsuyo Kohara, Yoshie Kishida, Chiharu Minami, Shinobu Nakayama, Naomi Nakazaki, Yoshimi Shimizu, Sayaka Shinpo, Chika Takahashi, Tsuyuko Wada, Manabu Yamada, Nobuko Ohmido, Makoto Hayashi, Kiichi Fukui, Tomoya Baba, Tomoko Nakamichi, Hirotada Mori, Satoshi Tabata;  Genome Structure of the Legume, <em>Lotus japonicus</em>, <em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 227–239, <a href='https://doi.org/10.1093/dnares/dsn008'>https://doi.org/10.1093/dnares/dsn008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/375737?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/375737?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20Structure%20of%20the%20Legume%2C%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2008-05-28&amp;volumeNum=15&amp;issueNum=4&amp;author=Sato%2C%20Shusei%3B%20Nakamura%2C%20Yasukazu&amp;startPage=227&amp;endPage=239&amp;contentId=10.1093%2Fdnares%2Fdsn008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f227%2f375737" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5468708 id="5468708" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The legume <em>Lotus japonicus</em> has been widely used as a model system to investigate the genetic background of legume-specific phenomena such as symbiotic nitrogen fixation. Here, we report structural features of the <em>L. japonicus</em> genome. The 315.1-Mb sequences determined in this and previous studies correspond to 67% of the genome (472 Mb), and are likely to cover 91.3% of the gene space. Linkage mapping anchored 130-Mb sequences onto the six linkage groups. A total of 10 951 complete and 19 848 partial structures of protein-encoding genes were assigned to the genome. Comparative analysis of these genes revealed the expansion of several functional domains and gene families that are characteristic of <em>L. japonicus</em>. Synteny analysis detected traces of whole-genome duplication and the presence of synteny blocks with other plant genomes to various degrees. This study provides the first opportunity to look into the complex and unique genetic system of legumes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lotus japonicus&quot;"><em>Lotus japonicus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome structure&quot;">genome structure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Fabaceae">Fabaceae</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative analysis&quot;">comparative analysis</a></div>                            <h2 scrollto-destination=5468710 id="5468710" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Fabaceae is the third largest family of flowering plants, comprising 650 genera and 18 000 species with a variety of characteristics; many of which have long been targets of breeding because of their agronomic and industrial importance. Among them, a few species have been chosen as ‘model legumes’ for use in genetic and physiological studies. <em>Lotus japonicus</em> is a typical model legume with the characteristics of a short life cycle (2–3 months), self-fertility, and a relatively simple genome architecture of diploidy (<em>n</em> = 6), i.e. small in size, 472 Mb. Mutants in various biological phenomena specific to legumes such as symbiotic nitrogen fixation, and those common to flowering plants such as flower morphogenesis, have been characterized and the genes responsible have been isolated and further studied. The availability of the <em>Agrobacterium</em>-mediated DNA transformation system and genomic resources including a large number of expressed sequences tag (EST)/cDNA clones,<sup><a reveal-id="DSN008C1" data-open="DSN008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> high-density genetic linkage maps,<sup><a reveal-id="DSN008C2  DSN008C4" data-open="DSN008C2  DSN008C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> and partial genome sequences<sup><a reveal-id="DSN008C5  DSN008C9" data-open="DSN008C5  DSN008C9" class="link link-ref link-reveal xref-bibr">5–9</a></sup> has played an essential role in this process.</p><p>The whole-genome sequences of two plant species, <em>Arabidopsis thaliana</em> (Cruciferae) and <em>Oryza sativa</em> (Poaceae), have drastically accelerated research into their genetic systems by providing investigators with both gene sequences and positional information. However, the process of high-accuracy sequencing is expensive and labor intensive since physical maps are created and a large number of individual BAC clones are used as templates. In contrast, draft sequencing according to the whole-genome shotgun approach in combination with BAC end sequencing generates information about the general genome structure at a fairly reasonable cost, but the resulting data are rather fragmentary and need an additional effort to be connected with physical/genetic maps. In this study, we aimed to analyze the genome structure of <em>L. japonicus</em> to reveal gene and genome features that are characteristic of legume plants. For this purpose, we adopted various established technologies including genomic library construction, DNA sequencing, fluorescent <em>in situ</em> hybridization (FISH), genetic mapping and bioinformatics in such a way that the advantages of each technology were combined in a cost-effective manner. Here, we report the first whole-genome structure of the legume, <em>L. japonicus</em>, its characteristic genome features, and a variety of information and material resources that were developed during this study.</p>                            <h2 scrollto-destination=5468713 id="5468713" class="section-title" data-legacy-id=s2>2. Material and methods</h2>
                            <h3 scrollto-destination=5468714 id="5468714" class="section-title" data-legacy-id=s2a>2.1. Plant and DNA materials</h3>
<p><em>Lotus japonicus</em> accession Miyakojima MG-20 was provided by Masayoshi Kawaguchi, University of Tokyo.<sup><a reveal-id="DSN008C10" data-open="DSN008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Transformation-competent artificial chromosome (TAC) genomic libraries were constructed according to standard methods as described previously.<sup><a reveal-id="DSN008C5" data-open="DSN008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> BAC genomic libraries were constructed using the genomic DNA of <em>L. japonicus</em> accession MG-20 partially digested with either <em>Sau</em>3AI or <em>Eco</em>RI and pBeloBAC as a cloning vector. The average insert size of these libraries was 104 kb for the <em>Sau</em>3AI library and 101 and 88 kb for two independent preparations of <em>Eco</em>RI libraries. Both libraries covered the haploid genome 8.4 times in total.</p><p>The seeds and genomic libraries can be obtained from LegumeBase supported by the National BioResource Project (<a class="link link-uri" href="http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp">http://www.legumebase.agr.miyazaki-u.ac.jp/index.jsp</a>).</p>                            <h3 scrollto-destination=5468717 id="5468717" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing and assembly</h3>
<p>Two types of sequencing approaches were combined to sequence the <em>L. japonicus</em> genome: clone-by-clone sequencing and shotgun sequencing of selected regions of the genome.</p><p>TAC/BAC clones were selected from the genomic libraries as seed points using the sequence information from ESTs and cDNA markers from <em>L. japonicus</em> and other legumes. The nucleotide sequence of each clone was determined according to the shotgun strategy with three to five times redundancy. A total of 1909 TAC/BAC clones, those newly sequenced in this study and those that had been sequenced previously,<sup><a reveal-id="DSN008C5  DSN008C9" data-open="DSN008C5  DSN008C9" class="link link-ref link-reveal xref-bibr">5–9</a></sup> were assembled into 954 scaffolds using the Paracel Genome Assembler (PGA; version 2.6.2, Paracel Co., 2002), followed by manual TAC/BAC end-pair scaffolding, resulting in high-quality genomic sequence (HGS) contigs.</p><p>In parallel, shotgun sequencing of a selected TAC mixture (STM) enriched in gene spaces and a whole genomic DNA from which highly repetitive and organelle genomic sequences were subtracted (selected genomic regions, SGRs) was carried out. The TAC clones, neither end sequence of which hit repetitive or organelle genomic sequences in the <em>L. japonicus</em> genome, were selected from the libraries, pooled, and subjected to shotgun sequencing. For the SGRs, a genomic library with an average insert size of 2.5 kb was generated using pBluescript SK− as the cloning vector. For subtraction, polymerase chain reaction (PCR)-amplified fragments of LjTR1 were biotinylated using Biotin-High Prime (Roche, Basel, Switzerland) and used as a driver in subtractive hybridization with the WGS library. The WGS library was single-stranded prior to hybridization by combined action of gene II and exonuclease III. Hybrids were removed using Dynabeads M-280 Streptavidin (Invitrogen, Carlsbad, CA, USA) and the remaining single-stranded WGS library was double-stranded using Klenow fragments (Takara Bio, Japan) and transformed into host <em>E. coli</em> ElectroTen-Blue (Agilent Technologies, Santa Clara, CA, USA).</p><p>A total of 808 816 reads from STM generated from 4603 TAC inserts and 847 513 SGR reads were assembled into a set of 109 986 contigs, 147 805 446 bp in length (selected genome assembly, SGA) by the Arachne assembler, version 2.01.<sup><a reveal-id="DSN008C11" data-open="DSN008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The SGA sequences were then subjected to assemble with the HGS, and finally, a total of 110 940 supercontigs with a total coverage of 315 073 275 tentative genomic sequence (TGS) bases were obtained.</p>                            <h3 scrollto-destination=5468722 id="5468722" class="section-title" data-legacy-id=s2c>2.3. Linkage mapping</h3>
<p>Two types of PCR-based DNA marker, SSLP and dCAPS, were generated using the sequence information in order to locate sequence contigs on the genetic linkage map, as described previously.<sup><a reveal-id="DSN008C5" data-open="DSN008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The analysis of segregation data for SSR and dCAPS markers and linkage map integration were carried out using the F2 mapping population of accessions Miyakojima MG-20 and Gifu B-129, which were previously used for construction of the fine genetic linkage map.<sup><a reveal-id="DSN008C2" data-open="DSN008C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p>                            <h3 scrollto-destination=5468724 id="5468724" class="section-title" data-legacy-id=s2d>2.4. Fluorescent in situ hybridization</h3>
<p>The 26S rDNA, 5S rDNA, and TAC clones were used as probes for FISH analysis. The 26S and the 5S rDNA probes were produced by PCR using primer pairs that were designed based on the rRNA and 5S RNA gene sequences in the <em>L. japonicus</em> genome.</p><p>The FISH analysis using rRNA genes and TAC clones was performed on well-prepared chromosome spreads according to the method described previously.<sup><a reveal-id="DSN008C12" data-open="DSN008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The preparations were observed under a fluorescence microscope (OLYMPUS BX50) equipped with a sensitive cooled CCD camera (PXL1400), and the pro-metaphase chromosome spreads with clear patterns were photographed using blue or green light excitation and emission filters. Captured images were digitally stored in a computer and analyzed using CHIAS3 imaging software.<sup><a reveal-id="DSN008C13" data-open="DSN008C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                            <h3 scrollto-destination=5468727 id="5468727" class="section-title" data-legacy-id=s2e>2.5. Repetitive sequences</h3>
<p>Repetitive elements in the TGS were identified by comparing all of the contig units, HGS and those produced from the STM and SGRs, each using BLASTN<sup><a reveal-id="DSN008C14" data-open="DSN008C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSN008C15" data-open="DSN008C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and processing the outputs using the RECON program.<sup><a reveal-id="DSN008C16" data-open="DSN008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A total of 214 consensus sequences of repetitive elements that appeared at least 20 times were identified. The consensus sequences of these elements were subjected to a similarity search against known repeat elements in the RepBase (<a class="link link-uri" href="http://www.girinst.org/">http://www.girinst.org/</a>). For the consensus sequences with features of Class I or II transposable elements (TEs), full-length candidate sequences were identified by comparing 10 kb upstream and downstream of the corresponding genomic regions to find long terminal repeats or terminal inverted repeats. For unclassified consensus sequences, the longest representative sequences were selected by comparing the corresponding genome sequences using the CLUSTALW multiple alignment program. Full-length elements of TE and representative sequences of unclassified repeats were collected into a repeat sequence library along with the previously reported TE sequences<sup><a reveal-id="DSN008C17" data-open="DSN008C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and used as references for RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">www.repeatmasker.org</a>) analysis to delineate the occurrence of these elements in the total TGS and TAC/BAC end sequences.</p>                            <h3 scrollto-destination=5468729 id="5468729" class="section-title" data-legacy-id=s2f>2.6. Assignment of RNA-encoding genes</h3>
<p>In order to identify the potential RNA-encoding genes, a structural RNA sequence library was extracted from GenBank,<sup><a reveal-id="DSN008C18" data-open="DSN008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and searched for homologous sequences in the <em>L. japonicus</em> genome with the use of the BLASTN function in BLAST. Transfer RNA genes were predicted using tRNAscan-SE, version 1.4,<sup><a reveal-id="DSN008C19" data-open="DSN008C19" class="link link-ref link-reveal xref-bibr">19</a></sup> in a eukaryotic mode with default parameters. Genes for small nucleolar RNAs (snoRNAs) and 7S large nucleolar RNAs (LRNAs) were predicted using SnoScan<sup><a reveal-id="DSN008C20" data-open="DSN008C20" class="link link-ref link-reveal xref-bibr">20</a></sup> with a yeast model and srpSCAN,<sup><a reveal-id="DSN008C21" data-open="DSN008C21" class="link link-ref link-reveal xref-bibr">21</a></sup> respectively. A total of 93 genes for small nuclear RNAs including those for 21 U1, 19 U2, 7 U4, 23 U5, 21 U6, and 2 U12 were identified based on their similarity to known genes.</p><p>Candidates for miRNA precursors were identified by a search using the <em>Arabidopsis</em> mature miRNA sequences, which were retrieved from the miRNA Registry database (<a class="link link-uri" href="http://www.sanger.ac.uk/Software/Rfam/mirna/">http://www.sanger.ac.uk/Software/Rfam/mirna/</a>).<sup><a reveal-id="DSN008C22" data-open="DSN008C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The <em>L. japonicus</em> genome and EST sequences were searched for sequence patterns that are characteristic of miRNA using the FUZZNUC program that is available at EMBOSS (<a class="link link-uri" href="http://emboss.sourceforge.net/">http://emboss.sourceforge.net/</a>) with the permission of a maximum of two nucleotide mismatches. A stem-loop structure was searched within a distance of 500 nucleotides from the coding region of the mature miRNA based on the finding that the longest <em>Arabidopsis thaliana</em> precursor sequences identified to date are 411 nucleotides long.<sup><a reveal-id="DSN008C23" data-open="DSN008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Target genes of the miRNA candidates were searched in the coding sequences (CDS) and 3′ untranslated regions (UTRs) of the presumptive genes assigned in the TGS.</p>                            <h3 scrollto-destination=5468732 id="5468732" class="section-title" data-legacy-id=s2g>2.7. Assignment of protein-encoding genes</h3>
<p>Tentative genomic sequence was subjected to gene prediction and modeling by the Kazusa Annotation PipelinE for <em>Lotus japonicus</em> (KAPSEL).<sup><a reveal-id="DSN008C5" data-open="DSN008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The KAPSEL employs <em>ab initio</em> gene-finding software and similarity searches in order to generate the elements for gene model production. The <em>ab initio</em> gene-finding software used in the pipeline includes GeneMark.hmm,<sup><a reveal-id="DSN008C24" data-open="DSN008C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Genscan<sup><a reveal-id="DSN008C25" data-open="DSN008C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and Grail<sup><a reveal-id="DSN008C26" data-open="DSN008C26" class="link link-ref link-reveal xref-bibr">26</a></sup> using the <em>A. thaliana</em>-trained matrix. Splice-site candidates were deduced by NetGene2<sup><a reveal-id="DSN008C29" data-open="DSN008C29" class="link link-ref link-reveal xref-bibr">27</a></sup> and SplicePredictor.<sup><a reveal-id="DSN008C28" data-open="DSN008C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The similarity searches to detect potential protein-coding exons were performed using the BLASTX function of BLAST against the UniProtKB database.<sup><a reveal-id="DSN008C29" data-open="DSN008C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The assigned exon candidates were extracted from the original sequence library, then mapped on the TGS more precisely using the dps and nap programs in the program suite of the analysis and annotation tool (AAT) package.<sup><a reveal-id="DSN008C30" data-open="DSN008C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Similarity searches of transcript sequences were performed by aligning the TGS against the Gene Indices<sup><a reveal-id="DSN008C31" data-open="DSN008C31" class="link link-ref link-reveal xref-bibr">31</a></sup> for legume species including <em>L. japonicus, M. truncatula and Glycine max.</em> The assigned transcript sequences were mapped on the TGS using the dds and gap2 programs in AAT to confirm working models of protein-encoding genes. As a result of the automated annotation process, a total of 19 848 partial and 10 951 complete models were assigned as protein-encoding genes in the TGS, except for those related to TEs. The 76.4-Mb sequences in the HGS were edited and annotated manually to ensure high-quality gene prediction.</p><p>The genes thus assigned were denoted by IDs with the clone (LjT**** for TACs and LjB**** for BACs) or contig (CM****) names followed by sequential numbers from one end to another. Of these, manually annotated genes on the HGS were followed by “.nc”, and others were followed by “.nd”. The genes assigned on the SGA sequences were denoted by IDs with the assemble consensus names (LjSGA_****) followed by sequential numbers from one end to another in the insert.</p><p>A global alignment of the genome sequences and ESTs was performed using the NEEDLE program<sup><a reveal-id="DSN008C32" data-open="DSN008C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSN008C33" data-open="DSN008C33" class="link link-ref link-reveal xref-bibr">33</a></sup> that is provided at the EMBOSS site (<a class="link link-uri" href="http://emboss.sourceforge.net/">http://emboss.sourceforge.net/</a>). To identify a possible TATA box-like motif for recognition by RNA polymerase II, a search against the plant cis-acting regulatory DNA elements (PLACE) database<sup><a reveal-id="DSN008C34" data-open="DSN008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (<a class="link link-uri" href="http://www.dna.affrc.go.jp/PLACE/">http://www.dna.affrc.go.jp/PLACE/</a>) was carried out.</p>                            <h3 scrollto-destination=5468736 id="5468736" class="section-title" data-legacy-id=s2h>2.8. Similarity and domain searches</h3>
<p>Functional annotation for deduced <em>L. japonicus</em> proteins was performed by a similarity search against the genes of known function and a domain analysis. The similarity search was performed using the gapped BLASTP function of BLAST against the UniProtKB database and protein-encoding genes deduced in <em>A. thaliana</em>,<sup><a reveal-id="DSN008C35" data-open="DSN008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> rice,<sup><a reveal-id="DSN008C36" data-open="DSN008C36" class="link link-ref link-reveal xref-bibr">36</a></sup><em>Populus trichocarpa</em>,<sup><a reveal-id="DSN008C37" data-open="DSN008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and grapevine.<sup><a reveal-id="DSN008C38" data-open="DSN008C38" class="link link-ref link-reveal xref-bibr">38</a></sup> For the analysis of gene families and functional domains, the predicted proteome was searched against InterPro.<sup><a reveal-id="DSN008C39" data-open="DSN008C39" class="link link-ref link-reveal xref-bibr">39</a></sup> InterPro annotations were assigned to each functional motif and domain, and then the annotation information was translated into GO functional descriptions.<sup><a reveal-id="DSN008C40" data-open="DSN008C40" class="link link-ref link-reveal xref-bibr">40</a></sup> GO descriptions were grouped into the GOslim categories of molecular function and biological process. Subcellular localization of targeting signals and transmembrane helices of deduced protein-coding genes were predicted using the following programs: PSORT,<sup><a reveal-id="DSN008C41" data-open="DSN008C41" class="link link-ref link-reveal xref-bibr">41</a></sup> TargetP,<sup><a reveal-id="DSN008C42" data-open="DSN008C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and SOSUI.<sup><a reveal-id="DSN008C43" data-open="DSN008C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p>                            <h3 scrollto-destination=5468738 id="5468738" class="section-title" data-legacy-id=s2i>2.9. Synteny analysis</h3>
<p>Synteny was detected by identifying arrays of predicted protein-encoding genes between target genomic regions. Translated amino acid sequences of the products of genes assigned on the mapped TGS were compared with those in the reference genomes, and a BLASTP <em>E</em>-value of &lt;1e−20 was considered to be significant. Synteny blocks were surveyed on the basis of physically linked sequence units such as contigs and singlet clones. A synteny block was defined as the region where three or more conserved homologs were located within a 100-kb DNA stretch in the two genomes.</p>                            <h2 scrollto-destination=5468740 id="5468740" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5468741 id="5468741" class="section-title" data-legacy-id=s3a>3.1. Genome sequencing</h3>
<p>Preliminary random and TAC end sequencing revealed the presence of various types of repetitive sequences in the <em>L. japonicus</em> genome. The FISH analysis showed that some of these sequences were distributed along the entire genome with occasional condensation to different extents, strongly suggesting that the gene spaces are intermingled with repeated sequences. In order to efficiently obtain sequence information for the gene space, we combined two independent approaches: clone-by-clone sequencing from seed points of the genome and shotgun sequencing of both STM enriched in gene spaces and a whole-genomic DNA from which highly repetitive and organelle genome sequences were subtracted (SGRs).</p><p>A total of 1314 TAC and BAC clones were selected based on the sequences of ESTs, cDNA and gene information from <em>L. japonicus</em> and other legumes, and an additional 584 TAC and BAC clones were selected by overlaps. Their nucleotide sequences were determined according to the shotgun method with three to five times redundancy (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Further assembly of the sequences of the 1898 clones produced 954 supercontigs with a total length of 167 267 829 bp. Among these sequences, 76 366 532 bp (46%) covered by 823 clones were Phase 3 sequences (HGS). In parallel, 4603 TAC clones, neither ends of which hit the repetitive sequences, were pooled and subjected to shotgun sequencing. Assembly of 808 816 STM random sequences and 847 513 SGR sequences, the sum of which was estimated to give 2.4 times the genome coverage, generated an additional 109 986 contigs that were 147 805 446 bp long in total (SGA). The efficiency of the STM and SGR approaches was indicated by the ratio of centrometic and heterochromatic repeat sequences in SGA sequences, which was about one-fifth of that in TAC/BAC end sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The total length of the determined sequences (TGS) amounted to 315 073 275 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Although the TGS corresponded to 67% of the reported <em>L. japonicus</em> genome (472 Mb),<sup><a reveal-id="DSN008C44" data-open="DSN008C44" class="link link-ref link-reveal xref-bibr">44</a></sup> it can be estimated that the TGS covers 91.3% of the gene space because 11 404 out of 12 485 collections of tentative consensus (TC) sequences of the <em>L. japonicus</em> Gene Index provided by the Gene Index Project (<a class="link link-uri" href="http://compbio.dfci.harvard.edu/tgi/plant.html">http://compbio.dfci.harvard.edu/tgi/plant.html</a>) were located on the TGS. TGS was used as standard information for further analysis of gene structure and function in <em>L. japonicus</em> in this study.</p>                            <h3 scrollto-destination=5468744 id="5468744" class="section-title" data-legacy-id=s3b>3.2. Construction of the sequence-tagged genetic linkage map</h3>
<p>In order to anchor the obtained genomic sequences to the genetic linkage map, DNA markers were developed for the sequenced TAC and BAC clones, and genetic mapping was carried out. As a result, a total of 788 microsatellite and 80 derived cleaved amplified polymorphic sequences (dCAPS) markers were generated and mapped onto the six linkage groups, which resulted in anchoring of the HGS and the connecting TAC/BAC and SGR sequences onto the linkage map. In total, 594 supercontigs containing 130 251 279 bp, corresponding to 41% of the TGS, were anchored onto the genetic linkage map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>Authenticity of the above genetic linkage map was examined by FISH. TAC clones genetically mapped at distal and proximal positions to the linkage groups were used as probes for hybridization. As shown in Fig. <a reveal-id="DSN008F1" data-open="DSN008F1" class="link link-reveal link-table xref-fig">1</a>, all of the clones examined were successfully located at the expected positions on the corresponding chromosomes. For chromosome 5, four clones were positioned inversely in the expected order on the genetic linkage map; therefore, we revised the marker order of the genetic map. Genetic and physical distances did not agree very well, as reported in <em>A. thaliana</em> and rice.<sup><a reveal-id="DSN008C35" data-open="DSN008C35" class="link link-ref link-reveal xref-bibr">35</a>,<a reveal-id="DSN008C36" data-open="DSN008C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                        <a id="5468747" scrollto-destination="5468747"></a>
<div id="DSN008F1" data-id="DSN008F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00801.jpeg?Expires=2147483647&Signature=Ora680cTqz7WGk9S76VSlYnZQ6Pf0xgfy1cFpBBcmuQ2zI5snKY7wALaTk0qvdwAIKOK~fWILWkolvorPkhlDZ88v8YJENavoIwcPgCUwk-JJUmmx7qeyeQUsi1kXa6iQH5KYD~EQasAOU33B1nS~mW8Bs5kZ4M88l2SxqCFodjiUvUg-K654TpNIj4plo2IKTOTOIH5061TCFxhJTOPF4ibxDr428UrW0850ahYQ5JNa8IfJYlKC8fFnP7dA6TYpeRgCEy3xgnOMfLwmSIeCeLemhpnv0Wvr30vLQQtmoFOoXtVnTt1ZKERfaS8FIOJgIy1DsxoZDgLvREADCClRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH detection and the integration map of L. japonicus. (A and C) Mitotic prometaphase chromosomes and meiotic pachytene chromosomes were stained with DAPI. (B) TAC clone, LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on the long arm of chromosome 5. 45S rDNA (red) was detected on the short arms of chromosomes 2, 5, and 6. (D) LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on chromosome 5 of the pachytene chromosome. The error bar represents 5 μm. (E) Integration among three maps; mitotic prometaphase chromosome map, meiotic pachytene chromosome map, linkage map of L. japonicus. Red circles show TAC clones, yellow and orange show ribosomal RNA genes, green shows tandem repeat LjTR1, blue shows retrotransposon LjRE2 representing the centromere. The length ratio among six chromosomes was adjusted to the ratio of pachytene chromosomes." path-from-xml="dsn00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468747/dsn00801.jpeg" path-from-xml="dsn00801.jpeg" target="_blank">View large</a><a section="5468747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00801.jpeg?Expires=2147483647&Signature=TYzfC4Uk3MOtR5KThIEpGG0Jl-qvc6pbcHZAvBbk8JpwEUuIaKqaeKcONLUNYMbEhrYTfi8VVNL763VIPES5unxT~4WsRvkrt6Tty1onsH1FVMcn3i7myDcyfXaZcQ-UReg5dQvIDg9gs1mxTomOSbRGxZHirpMbmSiK-tpezATitR4S7TgI5dVnKe3ypuH1Z6bP4~7Twf1XQZc~HBCQVG8F442yfm-4uY9ANDFWJE4UKDWxv~AD-KhuShydStciWzNz2vahD99iLiPp0eGn8EttGMYvO6CuNNSY-PzbXV7zS1QOzO100nNyEtjUiLD56L6Bbt0Q3yUVpKhfoI1KKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468747&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FISH detection and the integration map of <em>L. japonicus</em>. (<strong>A</strong> and <strong>C</strong>) Mitotic prometaphase chromosomes and meiotic pachytene chromosomes were stained with DAPI. (<strong>B</strong>) TAC clone, LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on the long arm of chromosome 5. 45S rDNA (red) was detected on the short arms of chromosomes 2, 5, and 6. (<strong>D</strong>) LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on chromosome 5 of the pachytene chromosome. The error bar represents 5 μm. (<strong>E</strong>) Integration among three maps; mitotic prometaphase chromosome map, meiotic pachytene chromosome map, linkage map of <em>L. japonicus</em>. Red circles show TAC clones, yellow and orange show ribosomal RNA genes, green shows tandem repeat LjTR1, blue shows retrotransposon LjRE2 representing the centromere. The length ratio among six chromosomes was adjusted to the ratio of pachytene chromosomes.</p></div></div><div content-id="DSN008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00801.jpeg?Expires=2147483647&Signature=Ora680cTqz7WGk9S76VSlYnZQ6Pf0xgfy1cFpBBcmuQ2zI5snKY7wALaTk0qvdwAIKOK~fWILWkolvorPkhlDZ88v8YJENavoIwcPgCUwk-JJUmmx7qeyeQUsi1kXa6iQH5KYD~EQasAOU33B1nS~mW8Bs5kZ4M88l2SxqCFodjiUvUg-K654TpNIj4plo2IKTOTOIH5061TCFxhJTOPF4ibxDr428UrW0850ahYQ5JNa8IfJYlKC8fFnP7dA6TYpeRgCEy3xgnOMfLwmSIeCeLemhpnv0Wvr30vLQQtmoFOoXtVnTt1ZKERfaS8FIOJgIy1DsxoZDgLvREADCClRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="FISH detection and the integration map of L. japonicus. (A and C) Mitotic prometaphase chromosomes and meiotic pachytene chromosomes were stained with DAPI. (B) TAC clone, LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on the long arm of chromosome 5. 45S rDNA (red) was detected on the short arms of chromosomes 2, 5, and 6. (D) LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on chromosome 5 of the pachytene chromosome. The error bar represents 5 μm. (E) Integration among three maps; mitotic prometaphase chromosome map, meiotic pachytene chromosome map, linkage map of L. japonicus. Red circles show TAC clones, yellow and orange show ribosomal RNA genes, green shows tandem repeat LjTR1, blue shows retrotransposon LjRE2 representing the centromere. The length ratio among six chromosomes was adjusted to the ratio of pachytene chromosomes." path-from-xml="dsn00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468747/dsn00801.jpeg" path-from-xml="dsn00801.jpeg" target="_blank">View large</a><a section="5468747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00801.jpeg?Expires=2147483647&Signature=TYzfC4Uk3MOtR5KThIEpGG0Jl-qvc6pbcHZAvBbk8JpwEUuIaKqaeKcONLUNYMbEhrYTfi8VVNL763VIPES5unxT~4WsRvkrt6Tty1onsH1FVMcn3i7myDcyfXaZcQ-UReg5dQvIDg9gs1mxTomOSbRGxZHirpMbmSiK-tpezATitR4S7TgI5dVnKe3ypuH1Z6bP4~7Twf1XQZc~HBCQVG8F442yfm-4uY9ANDFWJE4UKDWxv~AD-KhuShydStciWzNz2vahD99iLiPp0eGn8EttGMYvO6CuNNSY-PzbXV7zS1QOzO100nNyEtjUiLD56L6Bbt0Q3yUVpKhfoI1KKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468747&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>FISH detection and the integration map of <em>L. japonicus</em>. (<strong>A</strong> and <strong>C</strong>) Mitotic prometaphase chromosomes and meiotic pachytene chromosomes were stained with DAPI. (<strong>B</strong>) TAC clone, LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on the long arm of chromosome 5. 45S rDNA (red) was detected on the short arms of chromosomes 2, 5, and 6. (<strong>D</strong>) LjT30P03 (corresponding marker: TM0148) (green, arrowhead) was detected on chromosome 5 of the pachytene chromosome. The error bar represents 5 μm. (<strong>E</strong>) Integration among three maps; mitotic prometaphase chromosome map, meiotic pachytene chromosome map, linkage map of <em>L. japonicus</em>. Red circles show TAC clones, yellow and orange show ribosomal RNA genes, green shows tandem repeat LjTR1, blue shows retrotransposon LjRE2 representing the centromere. The length ratio among six chromosomes was adjusted to the ratio of pachytene chromosomes.</p></div></div><p>The positions of centromeres were deduced by cytological features in both prometaphase and pachytene chromosomes and were further confirmed by FISH analysis using the pericentromere-specific retroelement LjRE2 (described later) as a probe (Fig. <a reveal-id="DSN008F1" data-open="DSN008F1" class="link link-reveal link-table xref-fig">1</a>). TAC clones genetically mapped at distal positions of each linkage group were located on the telomeric regions of all the chromosomes by FISH with the exception of the bottom of chromosome 4 and the top of chromosome 6 (Fig. <a reveal-id="DSN008F1" data-open="DSN008F1" class="link link-reveal link-table xref-fig">1</a>). However, none of these clones and the extending sequences contained the consensus telomeric repeat, CCCTAAA.<sup><a reveal-id="DSN008C45" data-open="DSN008C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Small condensed structures were observed at most of the telomeric regions of the chromosomes.<sup><a reveal-id="DSN008C44" data-open="DSN008C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The presence of such heterochromatic regions may have prevented the extension of the analyzed sequences to the telomeric repeat. Indeed, a short tandem repeat sequence, LjTR4, specific to the subtelomeric region was found on the contig (CM0105) at the bottom of chromosome 1, where no telomeric condentation was observed.<sup><a reveal-id="DSN008C44" data-open="DSN008C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                            <h3 scrollto-destination=5468749 id="5468749" class="section-title" data-legacy-id=s3c>3.3. Repetitive sequences</h3>
<p>A total of 33 730 di-, tri-, and tetra-nucleotide simple sequence repeats (SSRs) that were equal to or longer than 15 bp were identified in the TGS (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Provided that the size of the TGS is 315 Mb, the frequency of occurrence of the above SSRs was estimated to be one SSR in every 9.3 kb. Di-, tri-, and tetra-nucleotide SSRs accounted for 48.6, 44.4, and 7.0% of the identified SSRs, respectively. The SSR patterns that appeared frequently were (AT)n, (AAG)n, and (AAAT)n, each representing 63% of di-, 28% of tri-, and 46% of tetra-nucleotide repeat units. The tri-nucleotide SSRs, particularly (GGT)n and (GGA)n, were preferentially found in exons. (AG)n was enriched in 5′ and 3′ UTRs, and (AC)n frequently occurred in 5′ and 3′ UTRs and introns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>A search using the repeat sequence finding program RECON<sup><a reveal-id="DSN008C14" data-open="DSN008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> against the TGS unraveled the occurrence of a variety of repeat elements including Class I and Class II TE subfamilies and those that are difficult to classify into known subfamilies (Table <a reveal-id="DSN008TB1" data-open="DSN008TB1" class="link link-reveal link-table xref-fig">1</a>). Over 35 000 copies of miniature-inverted repeat TEs (MITEs) were identified, which were classified into more than 60 types. A substantial portion of these MITEs was found in introns and UTRs. A short insert size Class I retroelement, a member of the short-interspersed nucleotide elements, was also preferentially observed in introns and the 3′ UTRs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>).<sup><a reveal-id="DSN008C46" data-open="DSN008C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                        <a id="5468752" scrollto-destination="5468752"></a>
<div content-id="DSN008TB1" class="table-modal"><div class="table-wrap"><div id="DSN008TB1" class="table-wrap-title" data-id="DSN008TB1"><div class="label -label">Table 1</div><div class="caption"><p>Repetitive sequences in the <em>L. japonicus</em> TGS</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Copy number </th><th>Coverage (kb) </th><th>Fraction of genome (%) </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td>SINEs </td><td>252 </td><td>31.4 </td><td>0.01 </td></tr><tr><td>LINEs </td><td>13 225 </td><td>387.1 </td><td>1.19 </td></tr><tr><td>LTR: Ty1/copia </td><td>25 421 </td><td>23 225.2 </td><td>7.16 </td></tr><tr><td>LTR: Ty3/gypsy </td><td>36 445 </td><td>28 606.3 </td><td>8.81 </td></tr><tr><td>Other LTR </td><td>11 194 </td><td>6660.4 </td><td>2.05 </td></tr><tr><td>Total class I </td><td>86 537 </td><td>62 394.4 </td><td>19.23 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td>Coding class II </td><td>11 786 </td><td>3160.0 </td><td>0.97 </td></tr><tr><td>MITE </td><td>35 393 </td><td>7568.4 </td><td>2.33 </td></tr><tr><td>Total class II </td><td>47 179 </td><td>10 728.4 </td><td>3.31 </td></tr><tr><td>Short tandem repeats </td><td>5147 </td><td>4259.5 </td><td>1.31 </td></tr><tr><td>Unclassified </td><td>109 361 </td><td>33 851.3 </td><td>10.43 </td></tr></tbody></table></div><div content-id="DSN008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Copy number </th><th>Coverage (kb) </th><th>Fraction of genome (%) </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td>SINEs </td><td>252 </td><td>31.4 </td><td>0.01 </td></tr><tr><td>LINEs </td><td>13 225 </td><td>387.1 </td><td>1.19 </td></tr><tr><td>LTR: Ty1/copia </td><td>25 421 </td><td>23 225.2 </td><td>7.16 </td></tr><tr><td>LTR: Ty3/gypsy </td><td>36 445 </td><td>28 606.3 </td><td>8.81 </td></tr><tr><td>Other LTR </td><td>11 194 </td><td>6660.4 </td><td>2.05 </td></tr><tr><td>Total class I </td><td>86 537 </td><td>62 394.4 </td><td>19.23 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td>Coding class II </td><td>11 786 </td><td>3160.0 </td><td>0.97 </td></tr><tr><td>MITE </td><td>35 393 </td><td>7568.4 </td><td>2.33 </td></tr><tr><td>Total class II </td><td>47 179 </td><td>10 728.4 </td><td>3.31 </td></tr><tr><td>Short tandem repeats </td><td>5147 </td><td>4259.5 </td><td>1.31 </td></tr><tr><td>Unclassified </td><td>109 361 </td><td>33 851.3 </td><td>10.43 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5468752" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN008TB1" class="table-wrap-title" data-id="DSN008TB1"><div class="label -label">Table 1</div><div class="caption"><p>Repetitive sequences in the <em>L. japonicus</em> TGS</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Copy number </th><th>Coverage (kb) </th><th>Fraction of genome (%) </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td>SINEs </td><td>252 </td><td>31.4 </td><td>0.01 </td></tr><tr><td>LINEs </td><td>13 225 </td><td>387.1 </td><td>1.19 </td></tr><tr><td>LTR: Ty1/copia </td><td>25 421 </td><td>23 225.2 </td><td>7.16 </td></tr><tr><td>LTR: Ty3/gypsy </td><td>36 445 </td><td>28 606.3 </td><td>8.81 </td></tr><tr><td>Other LTR </td><td>11 194 </td><td>6660.4 </td><td>2.05 </td></tr><tr><td>Total class I </td><td>86 537 </td><td>62 394.4 </td><td>19.23 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td>Coding class II </td><td>11 786 </td><td>3160.0 </td><td>0.97 </td></tr><tr><td>MITE </td><td>35 393 </td><td>7568.4 </td><td>2.33 </td></tr><tr><td>Total class II </td><td>47 179 </td><td>10 728.4 </td><td>3.31 </td></tr><tr><td>Short tandem repeats </td><td>5147 </td><td>4259.5 </td><td>1.31 </td></tr><tr><td>Unclassified </td><td>109 361 </td><td>33 851.3 </td><td>10.43 </td></tr></tbody></table></div><div content-id="DSN008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Copy number </th><th>Coverage (kb) </th><th>Fraction of genome (%) </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td>SINEs </td><td>252 </td><td>31.4 </td><td>0.01 </td></tr><tr><td>LINEs </td><td>13 225 </td><td>387.1 </td><td>1.19 </td></tr><tr><td>LTR: Ty1/copia </td><td>25 421 </td><td>23 225.2 </td><td>7.16 </td></tr><tr><td>LTR: Ty3/gypsy </td><td>36 445 </td><td>28 606.3 </td><td>8.81 </td></tr><tr><td>Other LTR </td><td>11 194 </td><td>6660.4 </td><td>2.05 </td></tr><tr><td>Total class I </td><td>86 537 </td><td>62 394.4 </td><td>19.23 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td>Coding class II </td><td>11 786 </td><td>3160.0 </td><td>0.97 </td></tr><tr><td>MITE </td><td>35 393 </td><td>7568.4 </td><td>2.33 </td></tr><tr><td>Total class II </td><td>47 179 </td><td>10 728.4 </td><td>3.31 </td></tr><tr><td>Short tandem repeats </td><td>5147 </td><td>4259.5 </td><td>1.31 </td></tr><tr><td>Unclassified </td><td>109 361 </td><td>33 851.3 </td><td>10.43 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5468752" target="_blank">View Large</a></div></div><p>As TGS were enriched in sequences from the gene-rich euchromatic regions, we used less biased TAC end sequences to survey the highly repetitive sequences in the entire genome of <em>L. japonicus</em>. By clustering 37 000 TAC end sequences, 34 types of highly repetitive sequences, each of which appeared in more than 50 end sequences, could be identified. These included five short tandem repeats with a repeat unit ranging from 172 to 333 bp (LjTR1 to 5), two retroelements (LjRE1 and 2) and nine unclassified repeats (LjUR1 to 9), one of which is the centromere-associated sequence Ljcen1 (GenBank accession no. AF390569) reported previously by Pedrosa et al.<sup><a reveal-id="DSN008C45" data-open="DSN008C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The FISH analysis showed that LjRE2 was present exclusively in the pericentromeric heterochromatic regions, while LjRE1 was distributed along the entire chromosomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>), and that LjTR1 was localized in highly condensed heterochromatic regions of the genome known as the chromosome knobs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=5468754 id="5468754" class="section-title" data-legacy-id=s3d>3.4. Genes encoding non-coding RNAs</h3>
<p>A combination of prediction and similarity searches against the TGS identified 638 transfer RNA genes corresponding to 49 species of anticodons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>), which are sufficient for coding all of the amino acids in the universal codon table. In addition, two complete units of 18S–5.8S–26S ribosomal RNA genes and two or more copies of 5S ribosomal RNA genes with an interval of approximately 195 nucleotides were found in the TGS, strongly suggesting successful enrichment of non-repetitive regions of the genome. As to other structural RNA genes, 207 genes for snoRNAs, seven high-confidence predictions for 7S LRNAs, and 93 genes for small nuclear RNAs were deduced.</p><p>Genome-wide transcriptome analyses have identified a large number of non-coding RNA species being transcribed in the genome.<sup><a reveal-id="DSN008C47" data-open="DSN008C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSN008C48" data-open="DSN008C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Using the previously accumulated information from <em>L. japoncius</em> transcripts by EST<sup><a reveal-id="DSN008C1" data-open="DSN008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and SAGE<sup><a reveal-id="DSN008C49" data-open="DSN008C49" class="link link-ref link-reveal xref-bibr">49</a></sup> analyses, we searched for transcripts derived from non-coding (intergenic) genomic regions. As a result, 1300 ESTs without similarity to known and predicted protein sequences were obtained, and 460 SAGE tag species solely identified in the SAGE analysis were identified as possible transcripts from intergenic regions in the genome. Since ESTs are partial sequences and SAGE tags originate from the 3′-most anchoring enzyme site of the transcripts, these ‘intergenic’ candidates may include those from UTRs of genes. Gene prediction of the genome sequence is done based on the presence of coding sequences of appropriate length. It is, therefore, possible that transcripts derived from unpredicted regions on the genome may encode a protein with a short open reading frame or that functions at the RNA level.</p><p>In order to identify the miRNA precursor candidate in the <em>L. japoncius</em> genome, we performed a search using the <em>A. thaliana</em> mature miRNA sequences deposited in the miRNA registry database in the Sanger Institute (<a class="link link-uri" href="http://microrna.sanger.ac.uk/sequences/">http://microrna.sanger.ac.uk/sequences/</a>). As a result, we found 53 precursor sequence candidates encoding 26 miRNA species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Among the miRNAs detected only in <em>A. thaliana</em> (eudicots), but not in other species like mosses, ferns, gymnosperms and monocots,<sup><a reveal-id="DSN008C50" data-open="DSN008C50" class="link link-ref link-reveal xref-bibr">50</a></sup> miRNAs 393, 395, 398, and 399 were found in the <em>L. japonicus</em> genome. In order to examine whether other leguminous plants possess these miRNAs, the TC sequences of <em>G. max</em> and <em>Medicago truncatula</em> were subjected to a search. As a result, miRNAs 395, 398, and 399 were found in the TC of at least one of the two legumes whereas miRNA 393 was detected in the <em>M. truncatula</em> BAC sequence (AC147434). We therefore speculate that these four miRNAs developed concurrently with the appearance of eudicots. We also focused on the copy number of miRNA genes (precursors). There are six genes encoding miR156 in <em>A. thaliana</em>, whereas only one was found in the <em>L. japonicus</em> genome. On the contrary, multiple genes in <em>L. japonicus</em> such as those encoding miRNAs 171, 399, 164, 166, 169, 172, and 319 were also found in multiple in <em>A. thaliana</em> (data not shown). An increase in the copy number of conserved miRNA species was not observed, indicating the possibility that conserved types of miRNAs had not been recruited in legume-specific gene regulatory mechanisms. We suppose that a significant part of gene regulation by miRNA in leguminous species is done by those that evolved specifically in the legume lineage. To identify species-specific miRNAs, a computational prediction was performed and 1312 miRNA candidates were found in the <em>L. japonicus</em> genome sequence, which can be grouped into 605 families (L. Schauser, personal communication).</p><p>We attempted the identification of antisense transcription in the <em>L. japonicus</em> genome by comparing coding sequences with overlapping ESTs on both sense and antisense DNA strands. To discriminate between ESTs of antisense transcripts and wrongly oriented clones, only 3′ ESTs with a poly(T) signal and 5′ ESTs without a poly(T) signal were selected for this analysis. As a result, we found 322 such presumptive protein-coding genes. We suspect that some of them were derived from unselected antisense transcription; however, we obtained several pieces of evidence that indicate selective antisense transcription. We checked for the sequences of the exon–intron splice junctions and TATA box-like motif proximal to the antisense transcripts, and as a result, (an) intron(s) could be predicted in antisense transcripts of 41 genes, where the donor and acceptor sequence of the exon–intron splice junction was GT-AG in 49 cases and GC-AG in one case. We found TATA box-like motifs proximal to the antisense transcripts in ∼60% of the 322 protein-coding genes.</p>                            <h3 scrollto-destination=5468759 id="5468759" class="section-title" data-legacy-id=s3e>3.5. Characteristic features of protein-encoding genes</h3>
<p>Tentative genomic sequence was subjected to the automatic assignment of protein-encoding genes, and 10 951 complete and 19 848 partial gene structures, except those for transposons/retrotranspons, could be identified. Of the 30 799 presumptive protein-encoding genes, 16 096 (52%) carried ESTs with sequence identity of over 95% for a stretch 50 nucleotides long. Structural features of the protein-encoding genes in <em>L. japonicus</em> were investigated in detail using 5248 genes manually annotated in the 76.4 Mb from HGS. As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>, the basic structure of protein-encoding genes in <em>L. japonicus</em> is similar to that of <em>A. thaliana</em><sup><a reveal-id="DSN008C35" data-open="DSN008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> except for the average lengths of genes and introns: 2917 versus 1918 bp and 395 versus 157 bp in <em>L. japonicus</em> and <em>A. thaliana</em>, respectively. The average gene density in <em>L. japonicus</em> was estimated to be one gene in every 10.2 kb (30 799 genes in 315 073 kb), which is longer than that in <em>A. thaliana</em>, one gene in every 4.5 kb. The total gene number in the <em>L. japonicus</em> genome could be deduced to be 34 245 by calculations based on the following assumptions: (i) the average density of the protein-encoding genes in the euchromatic regions is one gene in every 10.2 kb, (ii) the TGS is 315 073 kb long, and (iii) gene-space coverage of the TGS is estimated to be 90.2%.</p><p>A similarity search of translated amino acid sequences of the presumptive 30 799 protein-encoding genes was performed against the UniProt knowledgebase as a protein sequence library.<sup><a reveal-id="DSN008C29" data-open="DSN008C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The result indicated that 19 405 (63%) genes had significant (&lt;1e−20 cutoff of BLAST <em>E</em>-value) sequence similarity to genes of known function, 6790 (22%) were similar to hypothetical genes and the remaining 4604 (15%) did not show any similarity to the registered gene sequences. Eighty-five percent (24 212 genes out of 28 581), 51% (34 098 genes out of 66 710), 74% (31 456 genes out of 42 373), and 73% (22 328 genes out of 30 434) of the protein-encoding genes deduced from genomic sequences of <em>A. thaliana</em>,<sup><a reveal-id="DSN008C35" data-open="DSN008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> rice,<sup><a reveal-id="DSN008C36" data-open="DSN008C36" class="link link-ref link-reveal xref-bibr">36</a></sup><em>P. trichocarpa</em>,<sup><a reveal-id="DSN008C37" data-open="DSN008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and grapevine,<sup><a reveal-id="DSN008C38" data-open="DSN008C38" class="link link-ref link-reveal xref-bibr">38</a></sup> respectively, showed significant sequence similarity (&lt;1e−20) to those in <em>L. japonicus</em>. The presumptive protein-encoding genes were classified into functional categories according to GO,<sup><a reveal-id="DSN008C40" data-open="DSN008C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and the result was summarized in plant GOslim functional categories based on the InterPro domain, as shown in Fig. <a reveal-id="DSN008F2" data-open="DSN008F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8.</a></p>                        <a id="5468762" scrollto-destination="5468762"></a>
<div id="DSN008F2" data-id="DSN008F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00802.jpeg?Expires=2147483647&Signature=rxR7yc6qfAvTjK69e5n-QHbD1JN~0~0Bg8IN~ST-E2n6pCYPPK2mLHXWueh47Z3ZLRdEXnOxEA3JYlazVPWlZTpi7rloTqzPzIYb3NsQ5splL5jeG6DD6RypTYMDUpJXjZxGc0L-kL1bh19~ZekNX8AcsIPnDNSdUtF8fYTfcawBxq5QD9-eSdOyVXPsgnq0dV09HaBMkBIOeGrYKSPRGAIwneOwAr2S1X~ZK5KyNy~fKYhMgJesWlHOXQ1xy-9UcuZAYqP8ocePQETwo7AeMawLRXbfAjJJCqw4oRsls9ItU1cov4EXzDthhXVQ5VC-XmBR6hgJWmkEEQeO6pheag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plant GOslim category classifications. The results of functional categorization are shown for 10 representative GOslim classes of ‘Molecular function’ (A) and ‘Biological process’ (B). GO was automatically annotated on the L. japonicus protein-encoding genes by searching InterPro domains." path-from-xml="dsn00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468762/dsn00802.jpeg" path-from-xml="dsn00802.jpeg" target="_blank">View large</a><a section="5468762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00802.jpeg?Expires=2147483647&Signature=uqLrrsd9KtV9Lm2hLh6c5JIuJ9K9tCTeX2yrXKZuG2wt9wIm~EidQyKV47HLCmkFU~vSWJVM0eme59NaC9rfp8a87mV5XgfhG3tCshsqxcQ51ag9YdkdLrpKh95BQ9EtPhGmWqP5LBqFQpVPLtH7aHyf2wu5wPDbzsIQqcjWp4xj0sMjnaYdEp6fMCf1UERDSgapi0PZ5B3we1prneMTY4vrh51dRUt0W4Ui1eRTEu6gUwksf6MlE6GqdbOW5WMxV1v4gJwdQEyVvI710iJg2al8S7gSzLuEC3Z4KLAB9xafAflxeM0zqBQgKdWKF02~qiJ9g8EenGcP-iOoRYbAUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468762&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plant GOslim category classifications. The results of functional categorization are shown for 10 representative GOslim classes of ‘Molecular function’ (<strong>A</strong>) and ‘Biological process’ (<strong>B</strong>). GO was automatically annotated on the <em>L. japonicus</em> protein-encoding genes by searching InterPro domains.</p></div></div><div content-id="DSN008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00802.jpeg?Expires=2147483647&Signature=rxR7yc6qfAvTjK69e5n-QHbD1JN~0~0Bg8IN~ST-E2n6pCYPPK2mLHXWueh47Z3ZLRdEXnOxEA3JYlazVPWlZTpi7rloTqzPzIYb3NsQ5splL5jeG6DD6RypTYMDUpJXjZxGc0L-kL1bh19~ZekNX8AcsIPnDNSdUtF8fYTfcawBxq5QD9-eSdOyVXPsgnq0dV09HaBMkBIOeGrYKSPRGAIwneOwAr2S1X~ZK5KyNy~fKYhMgJesWlHOXQ1xy-9UcuZAYqP8ocePQETwo7AeMawLRXbfAjJJCqw4oRsls9ItU1cov4EXzDthhXVQ5VC-XmBR6hgJWmkEEQeO6pheag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plant GOslim category classifications. The results of functional categorization are shown for 10 representative GOslim classes of ‘Molecular function’ (A) and ‘Biological process’ (B). GO was automatically annotated on the L. japonicus protein-encoding genes by searching InterPro domains." path-from-xml="dsn00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468762/dsn00802.jpeg" path-from-xml="dsn00802.jpeg" target="_blank">View large</a><a section="5468762" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00802.jpeg?Expires=2147483647&Signature=uqLrrsd9KtV9Lm2hLh6c5JIuJ9K9tCTeX2yrXKZuG2wt9wIm~EidQyKV47HLCmkFU~vSWJVM0eme59NaC9rfp8a87mV5XgfhG3tCshsqxcQ51ag9YdkdLrpKh95BQ9EtPhGmWqP5LBqFQpVPLtH7aHyf2wu5wPDbzsIQqcjWp4xj0sMjnaYdEp6fMCf1UERDSgapi0PZ5B3we1prneMTY4vrh51dRUt0W4Ui1eRTEu6gUwksf6MlE6GqdbOW5WMxV1v4gJwdQEyVvI710iJg2al8S7gSzLuEC3Z4KLAB9xafAflxeM0zqBQgKdWKF02~qiJ9g8EenGcP-iOoRYbAUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468762&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plant GOslim category classifications. The results of functional categorization are shown for 10 representative GOslim classes of ‘Molecular function’ (<strong>A</strong>) and ‘Biological process’ (<strong>B</strong>). GO was automatically annotated on the <em>L. japonicus</em> protein-encoding genes by searching InterPro domains.</p></div></div>                            <h4 scrollto-destination=5468763 id="5468763" class="section-title" data-legacy-id=s3e1>3.5.1. Transcription factors</h4>
<p>Genes encoding transcription factors (TFs) were identified in the <em>L. japonicus</em> genome by assessing the TrEMBL and InterPro search results. The classification of TFs was performed according to the TF categories in the Arabidopsis Gene Regulatory Information Server at the Ohio State University (<a class="link link-uri" href="http://arabidopsis.med.ohio-state.edu/AtTFDB/">http://arabidopsis.med.ohio-state.edu/AtTFDB/</a>). A total of 1481 genes were classified into 31 TF categories (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). EST matching provided evidence of transcription to 604 (41%) predicted TF genes (data not shown). TF genes classified into the C3H family were most frequently observed in the <em>L. japonicus</em> genome (183 genes), followed by 156 MYB and 142 AP2-EREBP family genes. While the distribution of TFs in <em>L. japonicus</em> and <em>A. thaliana</em> was generally similar (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>), we found 75 ARF family genes in the <em>L. japonicus</em> genome, which is more than three times the number in <em>A. thaliana</em> (24 genes). ARF binds to auxin response elements in the promoter region of auxin-regulated genes.<sup><a reveal-id="DSN008C51" data-open="DSN008C51" class="link link-ref link-reveal xref-bibr">51</a></sup> We found a relatively large number of unique sequences showing similarity to ARF genes in the Gene Index Project, 87 (31 singletons and 56 TCs) and 33 (13 singletons and 20 TCs), of <em>G. Max</em> and <em>M. truncatula</em>, respectively. An increased gene number (more than double) in the <em>L. japonicus</em> genome compared to <em>A. thaliana</em> was also observed for CAMTA and SBP family genes. CAMTA is a conserved TF among multicellular eukaryotes that is suggested to be involved in calcium signaling by direct binding of calmodulin.<sup><a reveal-id="DSN008C52" data-open="DSN008C52" class="link link-ref link-reveal xref-bibr">52</a></sup> SBP is a plant specific TF that is characterized by a conserved 76 amino acid protein domain.<sup><a reveal-id="DSN008C53" data-open="DSN008C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The observed characteristics of the <em>L. japonicus</em> genome as well as those of <em>G. max</em> and <em>M. truncatula</em> suggest the possibility that some TF families have diverged to adapt to biological processes characteristic of leguminous plants such as nodulation and symbiotic nitrogen fixation with rhizobia.</p>                            <h4 scrollto-destination=5468765 id="5468765" class="section-title" data-legacy-id=s3e2>3.5.2. Two-component systems</h4>
<p>A two-component system is a mechanism that is conserved in a wide variety of intracellular signal transduction pathways in response to certain external or internal stimuli in prokaryotes; plants have come to adopt this system as one of their multiple signal transduction mechanisms. This system is typically composed of histidine kinases (HKs), response regulators (RRs), and less typically some intermediators, all of which participate in the phosphor-transfer reaction. It has been shown that the two-component elements are involved in signaling of the plant hormones, ethylene, and cytokinin in <em>A. thaliana</em>,<sup><a reveal-id="DSN008C54" data-open="DSN008C54" class="link link-ref link-reveal xref-bibr">54</a></sup> and that these hormones play crucial roles in controlling nodule formation in legumes.<sup><a reveal-id="DSN008C55  DSN008C57" data-open="DSN008C55  DSN008C57" class="link link-ref link-reveal xref-bibr">55–57</a></sup></p><p>In the <em>L. japoncius</em> genome, 23 genes for HKs, including five and nine genes for ethylene and cytokinin receptors, respectively, were identified by similarity and domain searches. <em>Lotus</em> HK 1 (LHK1) is a member of the cytokinin receptor family and plays a crucial role in initiating cortical cell divisions in response to rhizobial signaling;<sup><a reveal-id="DSN008C55" data-open="DSN008C55" class="link link-ref link-reveal xref-bibr">55</a>,<a reveal-id="DSN008C56" data-open="DSN008C56" class="link link-ref link-reveal xref-bibr">56</a></sup> this gene was located on chr 4 (chr4.CM0042.1660.nc), with a highly conserved paralog on chr 2 (chr2.LjB11M03.80.nc). A total of 20 genes for RRs, 11 of which have domains characteristic of DNA-binding transcriptional factors in their C-terminal portions, were identified in the <em>L. japonicus</em> genome. However, two of them (chr4.CM0042.1440.nc and chr4.CM0229.220.nc) had a replacement of the phospho-accepting Asp residue with Gly or Ala residues, suggesting that these genes might have lost the ability to participate in the His–Asp phosphor-relay pathway. It has been reported that a group of pseudo-response regulators (PRRs), in which the phospho-accepting Asp residue is replaced by Glu residue, are known to be distinct from the typical RR in that they are not involved in any phosphor-relay signaling. Five such genes including TOC1 in <em>A. thaliana</em> are known to be involved in the circadian rhythm.<sup><a reveal-id="DSN008C58" data-open="DSN008C58" class="link link-ref link-reveal xref-bibr">58</a></sup> In <em>L. japonicus</em>, seven genes with a pseudo-receiver residue (Glu) were identified for PRR. One of these genes, chr4.CM0087.600.nc, was most similar to TOC1 (APRR1), while the other two genes, chr3.CM0208.230.nc and chr3.TM0616.60.nd, are likely to be candidates for clock components because they share a CCT motif in the C-terminal portion with the APRR components.</p>                            <h4 scrollto-destination=5468768 id="5468768" class="section-title" data-legacy-id=s3e3>3.5.3. Receptor-like kinases</h4>
<p>Receptor-like kinases (RLKs) play important roles in sensing diverse signals in plant growth, development, and defense responses. A typical RLK consists of a signal sequence, diverse extracellular domains, a transmembrane region, and an intracellular eukaryotic protein kinase domain. RLKs in the <em>A. thaliana</em> genome form the largest class of transmembrane sensors and are encoded by more than 600 genes.<sup><a reveal-id="DSN008C59" data-open="DSN008C59" class="link link-ref link-reveal xref-bibr">59</a></sup> The presumptive products of 1267 genes in the <em>L. japonicus</em> genome contained the protein kinase domain (InterProID, IPR000719), whereas those of 362 genes exhibited a domain composition that was similar to known RLKs. It has been reported that RLKs with leucine-rich repeats (LRRs: IPR001611) such as SYMRK (chr2.CM0177.340.nc)<sup><a reveal-id="DSN008C60" data-open="DSN008C60" class="link link-ref link-reveal xref-bibr">60</a></sup> and HAR1 (chr3.CM0216.560.nc)<sup><a reveal-id="DSN008C61" data-open="DSN008C61" class="link link-ref link-reveal xref-bibr">61</a>,<a reveal-id="DSN008C62" data-open="DSN008C62" class="link link-ref link-reveal xref-bibr">62</a></sup> were essential in the process of symbiotic nitrogen fixation in <em>L. japonicu</em>s. The classification based on InterPro indicated that the presumptive products of 124 genes contained the LRR (IPR001611) in their extracellular region.</p><p>The cascade of legume plant responses leading to nodule formation is triggered by recognition of lipo-chitooligosaccharide Nod factors that are produced by rhizobia by a family of receptor kinases containing the LysM domain (IPR002482) in their extracellular region.<sup><a reveal-id="DSN008C63" data-open="DSN008C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Presumptive products of 12 RLK genes in <em>L. japonicus</em> contained the exracellular LysM domain including Nfr1 (chr2.CM0545.460.nc) and Nfr5 (chr2.CM0323.400.nd), which have been proposed to be Nod-factor receptors, while only five such genes were found in <em>A. thaliana</em>. Therefore, it can be speculated that some of the remaining nine LysM–RLKs may participate in signal transduction during the process of nodule formation and pathogen recognition, as reported in <em>A. thaliana</em> and rice.<sup><a reveal-id="DSN008C64" data-open="DSN008C64" class="link link-ref link-reveal xref-bibr">64</a></sup></p>                            <h4 scrollto-destination=5468771 id="5468771" class="section-title" data-legacy-id=s3e4>3.5.4. Transporters</h4>
<p>Transporters play indispensable roles in various cellular processes including the delivery of nutrients, elimination of waste products, and control of hormone efflux. As symbiotic nitrogen fixation requires the exchange of nutrients between host legume plant cells and intracellular rhizobia, a variety of transporters should be involved. A total of 1310 genes encoding possible transporters were deduced in the <em>L. japonicus</em> genome based on their sequence similarity to 995 <em>A. thaliana</em> genes registered in the TransportDB (<a class="link link-uri" href="http://www.membranetransport.org/">http://www.membranetransport.org/</a>). These genes were subsequently classified into 49 gene families. Some of these families were composed of a large number of paralogs (97–205 members), including the ATP-binding cassette (ABC) superfamily,<sup><a reveal-id="DSN008C65" data-open="DSN008C65" class="link link-ref link-reveal xref-bibr">65</a></sup> the P-type ATPase (P-ATPase) superfamily, the drug/metabolite transporter superfamily, and the major facilitator superfamily. They account for a significant portion of the large number of transporter genes in <em>L. japonicus</em>, as is the case in <em>A. thaliana</em>.</p><p>One of the transporter families that are characteristic to legumes encodes major intrinsic membrane proteins (MIPs), which are known to control water and solute homeostasis and constitute 10–15% of the total membrane proteins in the symbiosome. Eleven of the 37 genes for MIPs assigned in the <em>L. japonicus</em> genome were paralogous members of the LIMP2 gene (chr4.CM0046.520.nc), a multifunctional aquaglyceroporin, which exhibits specific expression in mature nitrogen fixing nodules.<sup><a reveal-id="DSN008C66" data-open="DSN008C66" class="link link-ref link-reveal xref-bibr">66</a></sup> The LIMP2 gene and seven of the paralogs were clustered on chr 4 and chr 6 in tandem arrays. Proteome analysis in the nodule of <em>L. japonicus</em> has identified several types of peribacteroid membrane proteins. LIMP1 is one such protein known to encode a water-specific aquaporin.<sup><a reveal-id="DSN008C67" data-open="DSN008C67" class="link link-ref link-reveal xref-bibr">67</a></sup> A gene for LIMP1 (chr1.CM0012.450.nc) as well as two paralogs (chr1.LjT36P09.70.nd and LjSGA_047026.1) were assigned in the <em>L. japonicus</em> genome.</p>                            <h4 scrollto-destination=5468774 id="5468774" class="section-title" data-legacy-id=s3e5>3.5.5. Cytochrome P450 gene family</h4>
<p>One of the characteristic features of legume plants is the production of a large variety of secondary metabolites including legume-specific 5-deoxyisoflavonoid. Among the genes encoding enzymes involved in the biosynthesis of secondary metabolites, those for Cytochrome P450 (Cyt P450) play crucial roles. The genes for Cyt P450 constitute a relatively large gene family, which are further classified into sub-families on the basis of nomenclature files on the Cyt P450 Homepage (<a class="link link-uri" href="http://drnelson.utmem.edu/cytochromeP450.html">http://drnelson.utmem.edu/cytochromeP450.html</a>). A total of 313 putative genes for Cyt P450 were assigned to the <em>L. japonicus</em> genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>), whereas 272 and 455 genes for Cyt P450 have been reported in <em>A. thaliana</em> and rice, respectively.<sup><a reveal-id="DSN008C68" data-open="DSN008C68" class="link link-ref link-reveal xref-bibr">68</a></sup> Many of the Cyt P450 genes in <em>L. japonicus</em> formed tandem arrays, as in <em>A. thaliana</em> and rice, where those in a single array belong to the same gene family. The composition of some of the sub-families of Cyt P450 genes in <em>L. japonicus</em> was remarkably different from that of <em>A. thaliana</em>. Significant amplification of the members was observed in the sub-families CYP74A, CYP75B, CYC76C, CYP82C, CYP83B, CYP93C, and CYP716A, when compared with <em>A. thaliana</em>. Of these, the CYP75B and CYP93C sub-families have been reported to be involved in flavonoid synthesis;<sup><a reveal-id="DSN008C68" data-open="DSN008C68" class="link link-ref link-reveal xref-bibr">68</a>,<a reveal-id="DSN008C69" data-open="DSN008C69" class="link link-ref link-reveal xref-bibr">69</a></sup> thus, expansion of these sub-families may be involved in the complexity of flavonoids in <em>L. japonicus</em>.</p>                            <h4 scrollto-destination=5468776 id="5468776" class="section-title" data-legacy-id=s3e6>3.5.6. Disease resistance genes</h4>
<p>Most of the disease resistance genes (R genes) encode intracellular proteins with a centrally located nucleotide binding site (NBS) and a C-terminal LRR region. This NBS–LRR family of R genes can be classified into two subgroups, toll and interleukin-1 receptor (TIR) and non-TIR, based on whether or not the putative gene product contains a region similar to the TIR domain at their amino termini. We identified 174 TIR NBS–LRR-type and 55 non-TIR NBS–LRR-type R genes and gene segments in the TGS of <em>L. japonicus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). The ratio of TIR versus non-TIR is similar to that found in <em>A. thaliana</em> (85 TIR NBS–LRR versus 36 non-TIR NBS–LRR), but differs from that in <em>P. trichocarpa</em> (64 TIR NBS–LRR versus 233 non-TIR NBS–LRR).<sup><a reveal-id="DSN008C37" data-open="DSN008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Fifty-five of the 120 (83 TIR NBS–LRR and 37 non-TIR NBS–LRR) presumptive R genes assigned in the contiguous sequences of single or connected TAC/BAC clones were found as singletons, while the remaining 65 formed gene clusters, each consists of two to seven members of the same subfamily. Eighty-seven presumptive R genes genetically located on the linkage map were distributed unevenly among chromosomes: 17 (14 TIR NBS–LRR and 3 non-TIR NBS–LRR) on chr 1, 33 (18 and <a reveal-id="DSN008C15" data-open="DSN008C15" class="link link-ref link-reveal xref-bibr">15</a>) on chr 2, 20 (15 and <a reveal-id="DSN008C5" data-open="DSN008C5" class="link link-ref link-reveal xref-bibr">5</a>) on chr 3, 7 (5 and <a reveal-id="DSN008C2" data-open="DSN008C2" class="link link-ref link-reveal xref-bibr">2</a>) on chr 4, 5 (0 and <a reveal-id="DSN008C5" data-open="DSN008C5" class="link link-ref link-reveal xref-bibr">5</a>) on chr 5, and 5 (2 and <a reveal-id="DSN008C3" data-open="DSN008C3" class="link link-ref link-reveal xref-bibr">3</a>) on chr 6 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>).</p>                            <h4 scrollto-destination=5468778 id="5468778" class="section-title" data-legacy-id=s3e7>3.5.7. Genes conserved in leguminous plants</h4>
<p>In order to identify genes conserved in legumes but not in other plant species, predicted gene sequences of <em>L. japonicus</em> were subjected to comparison with predicted genes on the genomes of <em>A. thaliana</em>, rice, <em>P. trichocarpa</em> and grapevine as well as protein sequences in the TrEMBL protein database. Predicted gene sequences from the <em>M. truncatula</em> genome and <em>G. max</em> gene index were used as references for legume protein coding genes. Translated amino acid sequences of the predicted genes in <em>L. japonicus</em> were subjected to BLAST searches, and an <em>E</em>-value of &lt;1e−20 was considered to be significant. As a result, we found 1190 genes (4%) that were conserved in leguminous species, but not in the currently available protein sequences of other plant species. InterPro annotations of these genes were surveyed to investigate the conserved motifs identified in these legume-specific genes, and 31 InterPro motifs were found to be conserved in five or more genes. Of these motifs, Cyclin-like F-box (IPR001810), LRR 2 (IPR013101), TIR (IPR000157) and trypsin-alpha amylase inhibitor (IPR003612) were found in 61, 26, 13 and 11 genes, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental Table S12</a>). Furthermore, a total of 2824 predicted genes in the <em>L. japonicus</em> genome were without a counterpart in the databases. Among them, complete gene structures were predicted for 599 genes; the most common InterPro motif was the RNA-binding region RNP-1 (IPR000504) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>). A similarity search against the <em>L. japonicus</em> ESTs indicated that at least 718 of these genes (25.4%) are transcribed. The entire lists of the legume- and <em>L. japonicus</em>-specific genes are provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S14 and S15.</a></p>                            <h3 scrollto-destination=5468780 id="5468780" class="section-title" data-legacy-id=s3f>3.6. Gene amplification and genome rearrangement</h3>
<p>In order to detect gene amplification at the genome-wide level, reciprocal comparisons among the sequences of the presumptive protein-encoding genes in the <em>L. japonicus</em> genome were carried out using the BLASTP program. As a result, 56% of the genes (17 380 out of 30 799) matched one or more genes (with the cut-off &lt;1e−20): 14% had one matched gene, 7% had two genes, and the remaining 35% had three or more matches. It should be noted that these results may be underestimated because genes that are truncated at the ends of a large number of contigs and those assigned to sequences of low accuracy in the TGS are likely to be overlooked under the above conditions. Indeed, when the genes assigned to the 1909 TAC/BAC clones whose sequences had a stretch of ∼100 kb of high accuracy were subjected to search, 65% of the genes (10 151 out of 15 512) occurred as multiple copies; this is the same rate as that observed in <em>A. thaliana</em>. Further searches using the 76.4 Mb HGS, where the relative positions of most of the genes was reliable, showed that 772 genes formed 310 groups of tandem arrays of two or more family genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>). This corresponds to 7% of the genes assigned in the HGS (11 380), which is lower than those observed in <em>A. thaliana</em> (17%), rice (14%), and <em>P. trichocarpa</em> (11%). One possible explanation could be that frequent insertion of retroelements in the <em>L. japonicus</em> genome have disrupted tandem array structures, which is consistent with our observation that 9% of the genes assigned in HGS had family members in a range of 100 kb .</p><p>During the course of the comparative analysis of the <em>L. japonicus</em> genome with those of other plants, we occasionally found that two distinct regions of the <em>L. japonicus</em> genome showed synteny with a single locus of other genomes. When these regions were plotted on the chromosomes, traces of large scale segmental duplications, possibly because of whole-genome duplication (WGD), became apparent (Fig. <a reveal-id="DSN008F3" data-open="DSN008F3" class="link link-reveal link-table xref-fig">3</a>). These regions corresponded to 39% (231 out of 594) of the mapped TGS. Duplication of large blocks was found between chr 2/4, chr 4/6, chr 2/6, chr 1/3, chr 1/5, and chr 3/3. Only 13% (880 out of 6819) of the genes assigned in these regions were conserved in both of the duplicated regions, which is significantly lower than that between the syntenic regions of <em>L. japonicus</em> and <em>M. truncatula</em> where about 60% of the predicted genes were conserved. This observation supports the estimation that WGD occurred before the separation of <em>L. japonicus</em> and <em>M. truncatula</em> described in a previous report.<sup><a reveal-id="DSN008C70" data-open="DSN008C70" class="link link-ref link-reveal xref-bibr">70</a></sup></p>                        <a id="5468783" scrollto-destination="5468783"></a>
<div id="DSN008F3" data-id="DSN008F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00803.jpeg?Expires=2147483647&Signature=k5rCqqa59LtOZMddUftaGNKt2RHJBTbpxbuOv9Zi61p0Ps7OpPWAcLG~I7m1jAjdcz7d9eCNV6OBE3FLWuh9hQB-KYPjYWykInG-ttLIUyy9JURJYUZK46lAfDhFFr567lFDBvU3r0Vf~3qu-RMm2rhPLzhP61FSykiwvaJkIJzHkhi0zjs-5jJUmlLKzu76DdCXdUQ6xWeWs0Hn10pscJ4wAdT3gbjD6h2oWSnhGDTjKDKa~ut8hcchS2XSdpBB-7PquUb~cQ2ahnh~GPezVTPctKVjrbTqKnieUc~6OT~hE96Nun9Ri6u66gBlYDr1N8F2FTcrJqg7DN3It1naaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Segmentally duplicated regions in the L. japonicus genome. Individual chromosomes are depicted as horizontal bars. Colored boxes with the same color code connected by the blue line indicate each pair of duplicated segments. Duplicated segments in reversed orientation are connected with dotted lines. The scale is in megabases." path-from-xml="dsn00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468783/dsn00803.jpeg" path-from-xml="dsn00803.jpeg" target="_blank">View large</a><a section="5468783" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00803.jpeg?Expires=2147483647&Signature=C-ZUZeU26Qgi2IoiDPYvoQetwSCaXqfqgkn0fTuP6-aJ2h9a-7p1RCT0Hkikdfo-qrctOnpMXuEVLNAnIqghElUsqj92bK8WvSin6qM6QXY-i0yVM3HTabVV~KFOPbPnTwwPvUpbaStxB2rDYbZJ01O16w6aXcaE6VT9-UhY43Z80jy4KrOmG37iKvlB2jIBvX-zB931yATxfVwD8ps00D0JU73rKLInsKLQLsPmpRQ2m3mbaBgBCC53VeMfxRVf6iCKs-rZXSCNFXS46Qphw9E9Uh60peV5s2MnWf6rqB6aM71V3tp6WixJLLu7QOsevPU0PRs6FxZCBa1PGFDYPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468783&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Segmentally duplicated regions in the <em>L. japonicus</em> genome. Individual chromosomes are depicted as horizontal bars. Colored boxes with the same color code connected by the blue line indicate each pair of duplicated segments. Duplicated segments in reversed orientation are connected with dotted lines. The scale is in megabases.</p></div></div><div content-id="DSN008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/m_dsn00803.jpeg?Expires=2147483647&Signature=k5rCqqa59LtOZMddUftaGNKt2RHJBTbpxbuOv9Zi61p0Ps7OpPWAcLG~I7m1jAjdcz7d9eCNV6OBE3FLWuh9hQB-KYPjYWykInG-ttLIUyy9JURJYUZK46lAfDhFFr567lFDBvU3r0Vf~3qu-RMm2rhPLzhP61FSykiwvaJkIJzHkhi0zjs-5jJUmlLKzu76DdCXdUQ6xWeWs0Hn10pscJ4wAdT3gbjD6h2oWSnhGDTjKDKa~ut8hcchS2XSdpBB-7PquUb~cQ2ahnh~GPezVTPctKVjrbTqKnieUc~6OT~hE96Nun9Ri6u66gBlYDr1N8F2FTcrJqg7DN3It1naaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Segmentally duplicated regions in the L. japonicus genome. Individual chromosomes are depicted as horizontal bars. Colored boxes with the same color code connected by the blue line indicate each pair of duplicated segments. Duplicated segments in reversed orientation are connected with dotted lines. The scale is in megabases." path-from-xml="dsn00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5468783/dsn00803.jpeg" path-from-xml="dsn00803.jpeg" target="_blank">View large</a><a section="5468783" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn00803.jpeg?Expires=2147483647&Signature=C-ZUZeU26Qgi2IoiDPYvoQetwSCaXqfqgkn0fTuP6-aJ2h9a-7p1RCT0Hkikdfo-qrctOnpMXuEVLNAnIqghElUsqj92bK8WvSin6qM6QXY-i0yVM3HTabVV~KFOPbPnTwwPvUpbaStxB2rDYbZJ01O16w6aXcaE6VT9-UhY43Z80jy4KrOmG37iKvlB2jIBvX-zB931yATxfVwD8ps00D0JU73rKLInsKLQLsPmpRQ2m3mbaBgBCC53VeMfxRVf6iCKs-rZXSCNFXS46Qphw9E9Uh60peV5s2MnWf6rqB6aM71V3tp6WixJLLu7QOsevPU0PRs6FxZCBa1PGFDYPA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5468783&ar=375737&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Segmentally duplicated regions in the <em>L. japonicus</em> genome. Individual chromosomes are depicted as horizontal bars. Colored boxes with the same color code connected by the blue line indicate each pair of duplicated segments. Duplicated segments in reversed orientation are connected with dotted lines. The scale is in megabases.</p></div></div>                            <h3 scrollto-destination=5468784 id="5468784" class="section-title" data-legacy-id=s3g>3.7. Comparative analysis with genomes of model plants and crop legume</h3>
<p>Microsynteny against the <em>M. truncatula</em> genome was detected for 70% (418 out of 594) of the mapped TGS in <em>L. japonicus</em>, although traces of local duplications, inversions, deletions, and insertions were also observed. By combining the syntenic relations assigned at the clone and contig level in the TGS with their positional information on the genetic linkage map, the microsynteny could be expanded to macrosynteny at the level of whole chromosomes, which confirmed and strengthened previous observations.<sup><a reveal-id="DSN008C70" data-open="DSN008C70" class="link link-ref link-reveal xref-bibr">70</a></sup> A syntenic relationship has also been detected between the genomes of <em>L. japonicus</em> and <em>A. thaliana</em> to a lesser degree. Conservation of relative positions of three or more genes has been observed in 69% (408 out of 594) of the mapped TGS of <em>L. japonicus.</em> This limited degree of syntenic relationship was proved to be useful for map-based cloning in <em>L. japonicus</em>.<sup><a reveal-id="DSN008C71" data-open="DSN008C71" class="link link-ref link-reveal xref-bibr">71</a></sup> A similar level of synteny was detected against <em>P. trichocarpa</em> where 69% (409 out of 594) of the mapped TGS of <em>L. japonicus</em> exhibited syntenic relations. The microsyntenic relationships between the above plant species may provide useful information for predicting gene organization in the ancestral genome of dicot plants.</p>                            <h3 scrollto-destination=5468786 id="5468786" class="section-title" data-legacy-id=s3h>3.8. Conclusions</h3>
<p>To date, two types of sequencing strategy have been adopted in order to fully sequence plant genomes: clone-by-clone sequencing based on physical maps and whole-genome shotgun sequencing. Both strategies have their advantages and disadvantages in terms of labor, cost, speed, and data accuracy. In this study, we modified the two strategies into seed-based clone-by-clone and selected genome shotgun sequencing, combined them, and anchored the obtained sequences onto maps by linkage analysis and FISH. This strategy takes advantage of the strengths of the conventional methods, and provides information that meets the majority of requirements for a wide variety of uses in fundamental and applied biology. It can therefore be applied with appropriate modifications to the genomes of diverse plants species, especially large crop genomes that will be interesting to sequence in the future.</p><p>The nucleotide sequences and their positional information generated in this study revealed characteristic features of the legume genomes: the expansion of functional domains and gene families for TFs, elements of the two-component system, transporters, P450 proteins, proteins involved in disease resistance, and the presence of unique genes such as those for receptor kinases essential for symbiotic nitrogen fixation and those specific to legume plants. Synteny analysis among the sequenced plant genomes detected traces of WGD and subsequent genome evolution in <em>L. japonicus</em>, and also the presence of synteny blocks with genomes of model and crop plants to various degrees. Conservation of colinearity of the genes and synteny blocks on the chromosomes among plant species indicates that the genome information for <em>L. japonicus</em> could serve as a hub for transferring knowledge toward the improvement of crop legumes. The detailed and comprehensive analyses of the genome information reported in this study may lead to the identification of a variety of legume- and plant-specific genetic systems.</p><p>The genome sequence information is available through the international databases (DDBJ/Genbank/EMBL) under accession numbers DF093176–DF093536 (for genome clone contigs), AP009625–AP010255 (for genome clone sequences) and BABK01000001–BABK01109986 (for SGA sequences). The online database that provides nucleotide sequences of the pseudo-molecules with annotations describing the predicted genes as well as information on the DNA markers and the genetic linkage maps is available at <a class="link link-uri" href="http://kazusa.or.jp/lotus/">http://kazusa.or.jp/lotus/</a>.</p>                            <h2 scrollto-destination=5468790 id="5468790" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a></p>                            <h2 scrollto-destination=5468792 id="5468792" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation. Funding for open access charge: The Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=5468794 id="5468794" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Drs. T. Aoki and L. Schauser for valuable discussions on the genes involved in secondly metabolism and the genes encoding miRNAs, respectively. We also acknowledge Drs. D. Holligan and S. R. Wessler for providing the sequence information on transposable elements.</p>                            <h2 scrollto-destination=5468796 id="5468796" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Characteristics of the <em>Lotus japonicus</em> gene repertoire deduced from large-scale expressed sequence tag (EST) analysis</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">405</div>-<div class="lpage">414</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characteristics%20of%20the%20Lotus%20japonicus%20gene%20repertoire%20deduced%20from%20large-scale%20expressed%20sequence%20tag%20(EST)%20analysis&amp;author=E.%20Asamizu&amp;author=Y.%20Nakamura&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol.&amp;volume=54&amp;pages=405-414" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLAN.0000036372.46942.b8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLAN.0000036372.46942.b8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLAN.0000036372.46942.b8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15284495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miyahara</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Construction of a genetic linkage map of the model legume <em>Lotus japonicus</em> using an intraspecific F2 population</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">301</div>-<div class="lpage">310</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20using%20an%20intraspecific%20F2%20population&amp;author=M.%20Hayashi&amp;author=A.%20Miyahara&amp;author=S.%20Sato&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=301-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Krusell</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A genetic linkage map of the model legume <em>Lotus japonicus</em> and strategies for fast mapping of new loci</div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">1673</div>-<div class="lpage">1683</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20and%20strategies%20for%20fast%20mapping%20of%20new%20loci&amp;author=N.%20Sandal&amp;author=L.%20Krusell&amp;author=S.%20Radutoiu&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1673-1683" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">T. R.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genetics of symbiosis in <em>Lotus japonicus</em>: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2006</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">80</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20symbiosis%20in%20Lotus%20japonicus%3A%20recombinant%20inbred%20lines%2C%20comparative%20genetic%20maps%2C%20and%20map%20position%20of%2035%20symbiotic%20loci&amp;author=N.%20Sandal&amp;author=T.%20R.%20Petersen&amp;author=J.%20Murray&amp;publication_year=2006&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=19&amp;pages=80-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16404956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">311</div>-<div class="lpage">318</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20I.%20Sequence%20features%20and%20mapping%20of%20fifty-six%20TAC%20clones%20which%20cover%20the%205.4%20Mb%20regions%20of%20the%20genome&amp;author=S.%20Sato&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=311-318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. II. Sequence features and mapping of sixty-five TAC clones which cover the 6.5 Mb regions of the genome</div>, <div class="source">DNA Res.</div> , <div class="year">2002</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">63</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20II.%20Sequence%20features%20and%20mapping%20of%20sixty-five%20TAC%20clones%20which%20cover%20the%206.5%20Mb%20regions%20of%20the%20genome&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;author=E.%20Asamizu&amp;publication_year=2002&amp;journal=DNA%20Res.&amp;volume=9&amp;pages=63-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/9.2.63" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F9.2.63" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F9.2.63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12056416" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Structural analysis of a Lotus japonicus genome. III. Sequence features and mapping of sixty-two TAC clones which cover the 6.7 Mb regions of the genome</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">27</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20III.%20Sequence%20features%20and%20mapping%20of%20sixty-two%20TAC%20clones%20which%20cover%20the%206.7%20Mb%20regions%20of%20the%20genome&amp;author=T.%20Kaneko&amp;author=E.%20Asamizu&amp;author=T.%20Kato&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=27-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12693552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. IV. Sequence features and mapping of seventy-three TAC clones which cover the 7.5 Mb regions of the genome</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">115</div>-<div class="lpage">122</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20IV.%20Sequence%20features%20and%20mapping%20of%20seventy-three%20TAC%20clones%20which%20cover%20the%207.5%20Mb%20regions%20of%20the%20genome&amp;author=E.%20Asamizu&amp;author=T.%20Kato&amp;author=S.%20Sato&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=115-122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.3.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.3.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.3.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12886953" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. V. Sequence features and mapping of sixty-four TAC clones which cover the 6.4 Mb regions of the genome</div>, <div class="source">DNA Res.</div> , <div class="year">2003</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">277</div>-<div class="lpage">285</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20V.%20Sequence%20features%20and%20mapping%20of%20sixty-four%20TAC%20clones%20which%20cover%20the%206.4%20Mb%20regions%20of%20the%20genome&amp;author=T.%20Kato&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;publication_year=2003&amp;journal=DNA%20Res.&amp;volume=10&amp;pages=277-285" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/10.6.277" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F10.6.277" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F10.6.277"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15029958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">‘Miyakojima’ MG-20: an early-flowering accession suitable for indoor handling, <em>J. Plant Res.</em></div>, <div class="source">Lotus japonicus</div> , <div class="year">2000</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">507</div>-<div class="lpage">509</div>)</div></div></div></div><div content-id="DSN008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jaffe</div> <div class="given-names">D. B.</div></div>,  <div class="name"><div class="surname">Butler</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gnerre</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Whole-genome sequence assembly for mammalian genomes: Arachne 2</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13,</div> (pg. <div class="fpage">91</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequence%20assembly%20for%20mammalian%20genomes%3A%20Arachne%202&amp;author=D.%20B.%20Jaffe&amp;author=J.%20Butler&amp;author=S.%20Gnerre&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13,&amp;pages=91-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.828403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.828403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.828403"> </span></div> </div></div></div></div></div><div content-id="DSN008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">Comprehensive structural analysis of the genome of red clover (<em>Trifolium pratense</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">301</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20structural%20analysis%20of%20the%20genome%20of%20red%20clover%20(Trifolium%20pratense%20L.)&amp;author=S.%20Sato&amp;author=S.%20Isobe&amp;author=E.%20Asamizu&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=301-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohmido</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Fukui</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Development of a quantitative pachytene chromosome map in <em>Oryza sativa</em> by imaging methods</div>, <div class="source">Genes Genet. Syst.</div> , <div class="year">2003</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">155</div>-<div class="lpage">161</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20quantitative%20pachytene%20chromosome%20map%20in%20Oryza%20sativa%20by%20imaging%20methods&amp;author=S.%20Kato&amp;author=N.%20Ohmido&amp;author=K.%20Fukui&amp;publication_year=2003&amp;journal=Genes%20Genet.%20Syst.&amp;volume=78&amp;pages=155-161" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.78.155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.78.155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.78.155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12773815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div> </div></div></div></div></div><div content-id="DSN008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Automated de novo identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1269</div>-<div class="lpage">1276</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.%20R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1269-1276" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holligan</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Pritham</div> <div class="given-names">E. J.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">The transposable element landscape of the model legume <em>Lotus japonicus</em></div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">2215</div>-<div class="lpage">2228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transposable%20element%20landscape%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=D.%20Holligan&amp;author=X.%20Zhang&amp;author=N.%20Jiang&amp;author=E.%20J.%20Pritham&amp;author=S.%20R.%20Wessler&amp;publication_year=2006&amp;journal=Genetics&amp;volume=174&amp;pages=2215-2228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.062752" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.062752" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.062752"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Ostell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wheeler</div> <div class="given-names">D. L.</div></div>. </div><div class="article-title">GenBank</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">21</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBank&amp;author=D.%20A.%20Benson&amp;author=I.%20Karsch-Mizrachi&amp;author=D.%20J.%20Lipman&amp;author=J.%20Ostell&amp;author=D.%20L.%20Wheeler&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res.&amp;volume=32&amp;pages=21-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl986"> </span></div> </div></div></div></div></div><div content-id="DSN008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res.&amp;volume=25&amp;pages=955-964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div> </div></div></div></div></div><div content-id="DSN008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">A computational screen for methylation guide snoRNAs in yeast</div>, <div class="source">Science</div> , <div class="year">1999</div>, vol. <div class="volume">283</div> (pg. <div class="fpage">1168</div>-<div class="lpage">1171</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computational%20screen%20for%20methylation%20guide%20snoRNAs%20in%20yeast&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1999&amp;journal=Science&amp;volume=283&amp;pages=1168-1171" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.283.5405.1168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.283.5405.1168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.283.5405.1168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10024243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Regalia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rosenblad</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Samuelsson</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Prediction of signal recognition particle RNA genes</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">3368</div>-<div class="lpage">3377</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20signal%20recognition%20particle%20RNA%20genes&amp;author=M.%20Regalia&amp;author=M.%20A.%20Rosenblad&amp;author=T.%20Samuelsson&amp;publication_year=2002&amp;journal=Nucl.%20Acids%20Res.&amp;volume=30&amp;pages=3368-3377" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf468"> </span></div> </div></div></div></div></div><div content-id="DSN008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffiths-Jones</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The microRNA registry</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D109</div>-<div class="lpage">D111</div>) <div class="comment">(Database issue)</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20microRNA%20registry&amp;author=S.%20Griffiths-Jones&amp;publication_year=2004&amp;journal=Nucl.%20Acids%20Res.&amp;volume=32&amp;pages=D109-D111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh023"> </span></div> </div></div></div></div></div><div content-id="DSN008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones-Rhoades</div> <div class="given-names">M. W.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div>. </div><div class="article-title">Computational identification of plant microRNAs and their targets, including a stress-induced miRNA</div>, <div class="source">Mol. Cell</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">787</div>-<div class="lpage">799</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20identification%20of%20plant%20microRNAs%20and%20their%20targets%2C%20including%20a%20stress-induced%20miRNA&amp;author=M.%20W.%20Jones-Rhoades&amp;author=D.%20P.%20Bartel&amp;publication_year=2004&amp;journal=Mol.%20Cell&amp;volume=14&amp;pages=787-799" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2004.05.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2004.05.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2004.05.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15200956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukashin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark.hmm: new solutions for gene finding</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1107</div>-<div class="lpage">1115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark.hmm%3A%20new%20solutions%20for%20gene%20finding&amp;author=A.%20Lukashin&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Nucl.%20Acids%20Res.&amp;volume=26&amp;pages=1107-1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.4.1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.4.1107"> </span></div> </div></div></div></div></div><div content-id="DSN008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burge</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Prediction of complete gene structures in human genomic DNA</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">78</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20complete%20gene%20structures%20in%20human%20genomic%20DNA&amp;author=C.%20Burge&amp;author=S.%20Karlin&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=78-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9149143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uberbacher</div> <div class="given-names">E. C.</div></div>,  <div class="name"><div class="surname">Mural</div> <div class="given-names">R. J.</div></div>. </div><div class="article-title">Locating protein-coding regions in human DNA sequences by a multiple sensor-neural network approach</div>, <div class="source">Proc. Natl. Acad. Sci., USA</div> , <div class="year">1991</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">11261</div>-<div class="lpage">11265</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20protein-coding%20regions%20in%20human%20DNA%20sequences%20by%20a%20multiple%20sensor-neural%20network%20approach&amp;author=E.%20C.%20Uberbacher&amp;author=R.%20J.%20Mural&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%2C%20USA&amp;volume=88&amp;pages=11261-11265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.24.11261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.24.11261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.24.11261"> </span></div> </div></div></div></div></div><div content-id="DSN008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hebsgaard</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Korning</div> <div class="given-names">P. G.</div></div>,  <div class="name"><div class="surname">Tolstrup</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Engelbrecht</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rouzé</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Splice site prediction in <em>Arabidopsis thaliana</em> DNA by combining local and global sequence information</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1996</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">3439</div>-<div class="lpage">3452</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splice%20site%20prediction%20in%20Arabidopsis%20thaliana%20DNA%20by%20combining%20local%20and%20global%20sequence%20information&amp;author=S.%20M.%20Hebsgaard&amp;author=P.%20G.%20Korning&amp;author=N.%20Tolstrup&amp;author=J.%20Engelbrecht&amp;author=P.%20Rouz%C3%A9&amp;author=S.%20Brunak&amp;publication_year=1996&amp;journal=Nucl.%20Acids%20Res.&amp;volume=24&amp;pages=3439-3452" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.17.3439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.17.3439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.17.3439"> </span></div> </div></div></div></div></div><div content-id="DSN008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Kleffe</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Prediction of locally optimal splice sites in plant pre-mRNA with applications to gene identification in <em>Arabidopsis thaliana</em> genomic DNA</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">4748</div>-<div class="lpage">4757</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20locally%20optimal%20splice%20sites%20in%20plant%20pre-mRNA%20with%20applications%20to%20gene%20identification%20in%20Arabidopsis%20thaliana%20genomic%20DNA&amp;author=V.%20Brendel&amp;author=J.%20Kleffe&amp;publication_year=1998&amp;journal=Nucl.%20Acids%20Res.&amp;volume=26&amp;pages=4748-4757" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.20.4748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.20.4748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.20.4748"> </span></div> </div></div></div></div></div><div content-id="DSN008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="collab">UniProt Consortium</div><div class="article-title">The universal protein resource (UniProt)</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">193</div>-<div class="lpage">197</div>)</div></div></div></div><div content-id="DSN008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Adams</div> <div class="given-names">M. D.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kerlavage</div> <div class="given-names">A. R.</div></div>. </div><div class="article-title">A tool for analyzing and annotating genomic sequences</div>, <div class="source">Genomics</div> , <div class="year">1997</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">37</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tool%20for%20analyzing%20and%20annotating%20genomic%20sequences&amp;author=X.%20Huang&amp;author=M.%20D.%20Adams&amp;author=H.%20Zhou&amp;author=A.%20R.%20Kerlavage&amp;publication_year=1997&amp;journal=Genomics&amp;volume=46&amp;pages=37-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1997.4984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1997.4984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1997.4984"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9403056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsai</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sunkara</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The TIGR gene indices: clustering and assembling EST and known genes and integration with eukaryotic genomes</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> <div class="issue">Database issue</div>(pg. <div class="fpage">D71</div>-<div class="lpage">D74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TIGR%20gene%20indices%3A%20clustering%20and%20assembling%20EST%20and%20known%20genes%20and%20integration%20with%20eukaryotic%20genomes&amp;author=Y.%20Lee&amp;author=J.%20Tsai&amp;author=S.%20Sunkara&amp;publication_year=2005&amp;journal=Nucl.%20Acids%20Res.&amp;volume=33&amp;pages=D71-D74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki064"> </span></div> </div></div></div></div></div><div content-id="DSN008C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Needleman</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Wunsch</div> <div class="given-names">C. D.</div></div>. </div><div class="article-title">A general method applicable to the search for similarities in the amino acid sequence of two proteins</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1970</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">443</div>-<div class="lpage">453</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20method%20applicable%20to%20the%20search%20for%20similarities%20in%20the%20amino%20acid%20sequence%20of%20two%20proteins&amp;author=S.%20B.%20Needleman&amp;author=C.%20D.%20Wunsch&amp;publication_year=1970&amp;journal=J.%20Mol.%20Biol.&amp;volume=48&amp;pages=443-453" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(70)90057-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(70)90057-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(70)90057-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5420325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kruskal</div> <div class="given-names">J. B.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Sankoff</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kruskal</div> <div class="given-names">J. B.</div></div>. </div><div class="article-title">An overview of sequence comparison</div>, <div class="source">Time Warps, String Edits and Macromolecules: the Theory and Practice of Sequence Comparison</div> , <div class="year">1983</div><div class="publisher-loc">Boston</div><div class="publisher-name">Addison Wesley</div>(pg. <div class="fpage">1</div>-<div class="lpage">44</div>)</div></div></div></div><div content-id="DSN008C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ugawa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwamoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Korenaga</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Plant cis-acting regulatory DNA elements (PLACE) database</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">297</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20cis-acting%20regulatory%20DNA%20elements%20(PLACE)%20database&amp;author=K.%20Higo&amp;author=Y.%20Ugawa&amp;author=M.%20Iwamoto&amp;author=T.%20Korenaga&amp;publication_year=1999&amp;journal=Nucl.%20Acids%20Res.&amp;volume=27&amp;pages=297-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.297"> </span></div> </div></div></div></div></div><div content-id="DSN008C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="collab">Arabidopsis Genome Initiative</div><div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">408</div> (pg. <div class="fpage">796</div>-<div class="lpage">815</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="collab">International Rice Genome Sequencing Project</div><div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">436</div> (pg. <div class="fpage">793</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuskan</div> <div class="given-names">G. A.</div></div>,  <div class="name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">The genome of black cottonwood, <em>Populus trichocarpa</em> (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="year">2006</div>, vol. <div class="volume">313</div> (pg. <div class="fpage">1596</div>-<div class="lpage">1604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.%20A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-1604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="collab">The French–Italian Public Consortium for Grapevine Genome Characterization</div><div class="article-title">The grapevine genome sequence suggests ancestral hexaploidization in major angiosperm phyla</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">463</div>-<div class="lpage">467</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17721507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mulder</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T. K.</div></div>, et al. </div><div class="article-title">New developments in the InterPro database</div>, <div class="source">Nucl. Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> <div class="issue">Database issue</div>(pg. <div class="fpage">D224</div>-<div class="lpage">D228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20developments%20in%20the%20InterPro%20database&amp;author=N.%20J.%20Mulder&amp;author=R.%20Apweiler&amp;author=T.%20K.%20Attwood&amp;publication_year=2007&amp;journal=Nucl.%20Acids%20Res.&amp;volume=35&amp;pages=D224-D228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl841"> </span></div> </div></div></div></div></div><div content-id="DSN008C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="collab">The Gene Ontology Consortium</div><div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Horton</div> <div class="given-names">P.</div></div>. </div><div class="article-title">PSORT: a program for detecting sorting signals in proteins and predicting their subcellular localization</div>, <div class="source">Trends Biochem. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">34</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PSORT%3A%20a%20program%20for%20detecting%20sorting%20signals%20in%20proteins%20and%20predicting%20their%20subcellular%20localization&amp;author=K.%20Nakai&amp;author=P.%20Horton&amp;publication_year=1999&amp;journal=Trends%20Biochem.%20Sci.&amp;volume=24&amp;pages=34-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(98)01336-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(98)01336-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(98)01336-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10087920" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Emanuelsson</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Predicting subcellular localization of proteins based on their N-terminal amino acid sequence</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">300</div> (pg. <div class="fpage">1005</div>-<div class="lpage">1016</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20subcellular%20localization%20of%20proteins%20based%20on%20their%20N-terminal%20amino%20acid%20sequence&amp;author=O.%20Emanuelsson&amp;author=H.%20Nielsen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;publication_year=2000&amp;journal=J.%20Mol.%20Biol.&amp;volume=300&amp;pages=1005-1016" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.3903" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.3903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.3903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10891285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirokawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Boon-Chieng</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mitaku</div> <div class="given-names">S.</div></div>. </div><div class="article-title">SOSUI: classification and secondary structure prediction system for membrane proteins</div>, <div class="source">Bioinformatics</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">378</div>-<div class="lpage">379</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOSUI%3A%20classification%20and%20secondary%20structure%20prediction%20system%20for%20membrane%20proteins&amp;author=T.%20Hirokawa&amp;author=S.%20Boon-Chieng&amp;author=S.%20Mitaku&amp;publication_year=1998&amp;journal=Bioinformatics&amp;volume=14&amp;pages=378-379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/14.4.378" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F14.4.378" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F14.4.378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9632836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miyamoto</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Genome and chromosome dimensions of <em>Lotus japonicus</em></div>, <div class="source">J. Plant Res.</div> , <div class="year">2000</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">435</div>-<div class="lpage">442</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20and%20chromosome%20dimensions%20of%20Lotus%20japonicus&amp;author=M.%20Ito&amp;author=J.%20Miyamoto&amp;author=Y.%20Mori&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=435-442" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013952" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013952" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013952"> </span></div> </div></div></div></div></div><div content-id="DSN008C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pedrosa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stougaard</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schweizer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bachmair</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Chromosomal map of the model legume <em>Lotus japonicus</em></div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">161</div> (pg. <div class="fpage">1661</div>-<div class="lpage">1672</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20map%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=A.%20Pedrosa&amp;author=N.%20Sandal&amp;author=J.%20Stougaard&amp;author=D.%20Schweizer&amp;author=A.%20Bachmair&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1661-1672" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fawcett</div> <div class="given-names">J. A.</div></div>,  <div class="name"><div class="surname">Kawahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yasui</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">A SINE family widely distributed in the plant kingdom and its evolutionary history</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">505</div>-<div class="lpage">514</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20SINE%20family%20widely%20distributed%20in%20the%20plant%20kingdom%20and%20its%20evolutionary%20history&amp;author=J.%20A.%20Fawcett&amp;author=T.%20Kawahara&amp;author=H.%20Watanabe&amp;author=Y.%20Yasui&amp;publication_year=2006&amp;journal=Plant%20Mol.%20Biol.&amp;volume=61&amp;pages=505-514" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-006-0026-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-006-0026-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-006-0026-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16830182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dale</div> <div class="given-names">J. M.</div></div>, et al. </div><div class="article-title">Empirical analysis of transcriptional activity in the <em>Arabidopsis</em> genome</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">842</div>-<div class="lpage">846</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Empirical%20analysis%20of%20transcriptional%20activity%20in%20the%20Arabidopsis%20genome&amp;author=K.%20Yamada&amp;author=J.%20Lim&amp;author=J.%20M.%20Dale&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=842-846" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1088305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1088305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1088305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14593172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yelin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Dahary</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sorek</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Widespread occurrence of antisense transcription in the human genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2003</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">379</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20antisense%20transcription%20in%20the%20human%20genome&amp;author=R.%20Yelin&amp;author=D.%20Dahary&amp;author=R.%20Sorek&amp;publication_year=2003&amp;journal=Nat.%20Biotechnol.&amp;volume=21&amp;pages=379-386" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12640466" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Comparison of the transcript profiles from the root and the nodulating root of the model legume <em>Lotus japonicus</em> by serial analysis of gene expression</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="year">2005</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">487</div>-<div class="lpage">498</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20the%20transcript%20profiles%20from%20the%20root%20and%20the%20nodulating%20root%20of%20the%20model%20legume%20Lotus%20japonicus%20by%20serial%20analysis%20of%20gene%20expression&amp;author=E.%20Asamizu&amp;author=Y.%20Nakamura&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;publication_year=2005&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=18&amp;pages=487-498" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-18-0487" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-18-0487" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-18-0487"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15915647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Axtell</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Bartel</div> <div class="given-names">D. P.</div></div>. </div><div class="article-title">Antiquity of microRNAs and their targets in land plants</div>, <div class="source">Plant Cell</div> , <div class="year">2005</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1658</div>-<div class="lpage">1673</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antiquity%20of%20microRNAs%20and%20their%20targets%20in%20land%20plants&amp;author=M.%20J.%20Axtell&amp;author=D.%20P.%20Bartel&amp;publication_year=2005&amp;journal=Plant%20Cell&amp;volume=17&amp;pages=1658-1673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.032185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.032185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.032185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15849273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guilfoyle</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">Ulmasov</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hagen</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The ARF family of transcription factors and their role in plant hormone-responsive transcription</div>, <div class="source">Cell Mol. Life Sci.</div> , <div class="year">1998</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">619</div>-<div class="lpage">627</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ARF%20family%20of%20transcription%20factors%20and%20their%20role%20in%20plant%20hormone-responsive%20transcription&amp;author=T.%20J.%20Guilfoyle&amp;author=T.%20Ulmasov&amp;author=G.%20Hagen&amp;publication_year=1998&amp;journal=Cell%20Mol.%20Life%20Sci.&amp;volume=54&amp;pages=619-627" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s000180050190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs000180050190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs000180050190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9711229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bouche</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Scharlat</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Snedden</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bouchez</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Fromm</div> <div class="given-names">H.</div></div>. </div><div class="article-title">A novel family of calmodulin-binding transcription activators in multicellular organisms</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">21851</div>-<div class="lpage">21861</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20family%20of%20calmodulin-binding%20transcription%20activators%20in%20multicellular%20organisms&amp;author=N.%20Bouche&amp;author=A.%20Scharlat&amp;author=W.%20Snedden&amp;author=D.%20Bouchez&amp;author=H.%20Fromm&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=21851-21861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M200268200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M200268200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M200268200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11925432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cardon</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Höhmann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Klein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nettesheim</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Saedler</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Huijser</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Molecular characterisation of the <em>Arabidopsis</em> SBP-box genes</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">237</div> (pg. <div class="fpage">91</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterisation%20of%20the%20Arabidopsis%20SBP-box%20genes&amp;author=G.%20Cardon&amp;author=S.%20H%C3%B6hmann&amp;author=J.%20Klein&amp;author=K.%20Nettesheim&amp;author=H.%20Saedler&amp;author=P.%20Huijser&amp;publication_year=1999&amp;journal=Gene&amp;volume=237&amp;pages=91-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00308-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00308-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00308-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10524240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hwang</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">H. C.</div></div>,  <div class="name"><div class="surname">Sheen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Two-component signal transduction pathways in <em>Arabidopsis</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2002</div>, vol. <div class="volume">129</div> (pg. <div class="fpage">500</div>-<div class="lpage">515</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two-component%20signal%20transduction%20pathways%20in%20Arabidopsis&amp;author=I.%20Hwang&amp;author=H.%20C.%20Chen&amp;author=J.%20Sheen&amp;publication_year=2002&amp;journal=Plant%20Physiol.&amp;volume=129&amp;pages=500-515" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.005504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.005504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.005504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12068096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Karas</div> <div class="given-names">B. J.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Amyot</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Szczyglowski</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A cytokinin perception mutant colonized by Rhizobium in the absence of nodule organogenesis</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">101</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20cytokinin%20perception%20mutant%20colonized%20by%20Rhizobium%20in%20the%20absence%20of%20nodule%20organogenesis&amp;author=J.%20D.%20Murray&amp;author=B.%20J.%20Karas&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;author=L.%20Amyot&amp;author=K.%20Szczyglowski&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=101-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1132514" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1132514" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1132514"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17110535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tirichine</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>, et al. </div><div class="article-title">A gain-of-function mutation in a cytokinin receptor triggers spontaneous root nodule organogenesis</div>, <div class="source">Science</div> , <div class="year">2007</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">104</div>-<div class="lpage">107</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20gain-of-function%20mutation%20in%20a%20cytokinin%20receptor%20triggers%20spontaneous%20root%20nodule%20organogenesis&amp;author=L.%20Tirichine&amp;author=N.%20Sandal&amp;author=L.%20H.%20Madsen&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=104-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1132397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1132397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1132397"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17110537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nukui</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ezura</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Transgenic <em>Lotus japonicus</em> with an ethylene receptor gene Cm-ERS1/H70A enhances formation of infection threads and nodule primordia</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">427</div>-<div class="lpage">435</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transgenic%20Lotus%20japonicus%20with%20an%20ethylene%20receptor%20gene%20Cm-ERS1%2FH70A%20enhances%20formation%20of%20infection%20threads%20and%20nodule%20primordia&amp;author=N.%20Nukui&amp;author=H.%20Ezura&amp;author=K.%20Minamisawa&amp;publication_year=2004&amp;journal=Plant%20Cell%20Physiol.&amp;volume=45&amp;pages=427-435" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pch046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpch046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpch046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15111717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mizuno</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakamichi</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Pseudo-response regulators (PRRs) or true oscillator components (TOCs)</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">677</div>-<div class="lpage">685</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudo-response%20regulators%20(PRRs)%20or%20true%20oscillator%20components%20(TOCs)&amp;author=T.%20Mizuno&amp;author=N.%20Nakamichi&amp;publication_year=2005&amp;journal=Plant%20Cell%20Physiol.&amp;volume=46&amp;pages=677-685" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pci087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpci087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpci087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15767264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shiu</div> <div class="given-names">S. H.</div></div>,  <div class="name"><div class="surname">Karlowski</div> <div class="given-names">W. M.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tzeng</div> <div class="given-names">Y. H.</div></div>,  <div class="name"><div class="surname">Mayer</div> <div class="given-names">K. F.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W. H.</div></div>. </div><div class="article-title">Comparative analysis of the receptor-like kinase family in <em>Arabidopsis</em> and rice</div>, <div class="source">Plant Cell</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1220</div>-<div class="lpage">1234</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20receptor-like%20kinase%20family%20in%20Arabidopsis%20and%20rice&amp;author=S.%20H.%20Shiu&amp;author=W.%20M.%20Karlowski&amp;author=R.%20Pan&amp;author=Y.%20H.%20Tzeng&amp;author=K.%20F.%20Mayer&amp;author=W.%20H.%20Li&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=1220-1234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.020834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.020834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.020834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15105442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stracke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kistner</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A plant receptor-like kinase required for both bacterial and fungal symbiosis</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">959</div>-<div class="lpage">962</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plant%20receptor-like%20kinase%20required%20for%20both%20bacterial%20and%20fungal%20symbiosis&amp;author=S.%20Stracke&amp;author=C.%20Kistner&amp;author=S.%20Yoshida&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=959-962" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12087405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krusell</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Shoot control of root development and nodulation is mediated by a receptor-like kinase</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">422</div>-<div class="lpage">426</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shoot%20control%20of%20root%20development%20and%20nodulation%20is%20mediated%20by%20a%20receptor-like%20kinase&amp;author=L.%20Krusell&amp;author=L.%20H.%20Madsen&amp;author=S.%20Sato&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=422-426" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12442170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishimura</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">G. J.</div></div>, et al. </div><div class="article-title">HAR1 mediates systemic regulation of symbiotic organ development</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">426</div>-<div class="lpage">429</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HAR1%20mediates%20systemic%20regulation%20of%20symbiotic%20organ%20development&amp;author=R.%20Nishimura&amp;author=M.%20Hayashi&amp;author=G.%20J.%20Wu&amp;publication_year=2002&amp;journal=Nature&amp;volume=420&amp;pages=426-429" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12442172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">E. B.</div></div>, et al. </div><div class="article-title">Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">585</div>-<div class="lpage">592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20recognition%20of%20symbiotic%20bacteria%20requires%20two%20LysM%20receptor-like%20kinases&amp;author=S.%20Radutoiu&amp;author=L.%20H.%20Madsen&amp;author=E.%20B.%20Madsen&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=585-592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X. C.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Findley</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Molecular evolution of lysin motif-type receptor-like kinases in plants</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">623</div>-<div class="lpage">636</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20lysin%20motif-type%20receptor-like%20kinases%20in%20plants&amp;author=X.%20C.%20Zhang&amp;author=X.%20Wu&amp;author=S.%20Findley&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=144&amp;pages=623-636" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.107.097097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.107.097097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.107.097097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17449649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugiyama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Shitan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yazaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genome-wide analysis of ATP-binding cassette (ABC) proteins in a model legume plant, <em>Lotus japonicus</em>: comparison with <em>Arabidopsis</em> ABC protein family</div>, <div class="source">DNA Res.</div> , <div class="year">2006</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">205</div>-<div class="lpage">228</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20ATP-binding%20cassette%20(ABC)%20proteins%20in%20a%20model%20legume%20plant%2C%20Lotus%20japonicus%3A%20comparison%20with%20Arabidopsis%20ABC%20protein%20family&amp;author=A.%20Sugiyama&amp;author=N.%20Shitan&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=S.%20Tabata&amp;author=K.%20Yazaki&amp;publication_year=2006&amp;journal=DNA%20Res.&amp;volume=13&amp;pages=205-228" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsl013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsl013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsl013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17164256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guenther</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">D. M.</div></div>. </div><div class="article-title">Water-selective and multifunctional aquaporins from <em>Lotus japonicus</em> nodules, (Abstract)</div>, <div class="source">Planta</div> , <div class="year">2000</div>, vol. <div class="volume">210</div> (pg. <div class="fpage">741</div>-<div class="lpage">748</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Water-selective%20and%20multifunctional%20aquaporins%20from%20Lotus%20japonicus%20nodules%2C%20(Abstract)&amp;author=J.%20F.%20Guenther&amp;author=D.%20M.%20Roberts&amp;publication_year=2000&amp;journal=Planta&amp;volume=210&amp;pages=741-748" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004250050675" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004250050675" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004250050675"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10805445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wienkoop</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Saalbach</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Proteome analysis. Novel proteins identified at the peribacteroid membrane from <em>Lotus japonicus</em> root nodules</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">131</div> (pg. <div class="fpage">1080</div>-<div class="lpage">1090</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20analysis.%20Novel%20proteins%20identified%20at%20the%20peribacteroid%20membrane%20from%20Lotus%20japonicus%20root%20nodules&amp;author=S.%20Wienkoop&amp;author=G.%20Saalbach&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=1080-1090" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.015362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.015362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.015362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Schuler</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Paquette</div> <div class="given-names">S. M.</div></div>,  <div class="name"><div class="surname">Werck-Reichhart</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bak</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Comparative genomics of rice and <em>Arabidopsis</em>. Analysis of 727 cytochrome P450 genes and pseudogenes from a monocot and a dicot</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">756</div>-<div class="lpage">772</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20rice%20and%20Arabidopsis.%20Analysis%20of%20727%20cytochrome%20P450%20genes%20and%20pseudogenes%20from%20a%20monocot%20and%20a%20dicot&amp;author=D.%20R.%20Nelson&amp;author=M.%20A.%20Schuler&amp;author=S.%20M.%20Paquette&amp;author=D.%20Werck-Reichhart&amp;author=S.%20Bak&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=756-772" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.039826" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.039826" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.039826"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15208422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimada</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Akashi</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide analyses of the structural gene families involved in the legume-specific 5-deoxyisoflavonoid biosynthesis of <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">25</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analyses%20of%20the%20structural%20gene%20families%20involved%20in%20the%20legume-specific%205-deoxyisoflavonoid%20biosynthesis%20of%20Lotus%20japonicus&amp;author=N.%20Shimada&amp;author=S.%20Sato&amp;author=T.%20Akashi&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=25-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN008C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannon</div> <div class="given-names">S. B.</div></div>,  <div class="name"><div class="surname">Sterck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Legume genome evolution viewed through the <em>Medicago truncatula</em> and <em>Lotus japonicus</em> genomes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">14959</div>-<div class="lpage">14964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legume%20genome%20evolution%20viewed%20through%20the%20Medicago%20truncatula%20and%20Lotus%20japonicus%20genomes&amp;author=S.%20B.%20Cannon&amp;author=L.%20Sterck&amp;author=S.%20Rombauts&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=14959-14964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0603228103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0603228103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0603228103"> </span></div> </div></div></div></div></div><div content-id="DSN008C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN008C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stracke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Exploitation of colinear relationships between the genomes of <em>Lotus japonicus</em>, <em>Pisum sativum</em> and <em>Arabidopsis thaliana</em>, for positional cloning of a legume symbiosis gene</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">108</div> (pg. <div class="fpage">442</div>-<div class="lpage">449</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploitation%20of%20colinear%20relationships%20between%20the%20genomes%20of%20Lotus%20japonicus%2C%20Pisum%20sativum%20and%20Arabidopsis%20thaliana%2C%20for%20positional%20cloning%20of%20a%20legume%20symbiosis%20gene&amp;author=S.%20Stracke&amp;author=S.%20Sato&amp;author=N.%20Sandal&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=442-449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1438-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1438-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1438-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14557858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=375737&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn008/2/dsn008-dsn008supp.pdf?Expires=2147483647&amp;Signature=vJHsf9SUGoqAtsMpVUvJaPudWWCb8rctNbnxFycYQSR7vUtnmAAwucCP3VwODJAGjqoSKWNgs4PjP1V-AL55o7h-RIclkjQINLMtqF6Ptiq0Jou79PspXrtzFfaANfpMf3QsvzFqpdqpOU~XFROz7Ewhvbg-RA9oK4EsrRgRlnfqQfLI2Yf6hBA5LqrWL0K9DX4Om0buSss0L26TTnGegwxdz91SMCmEJV3iSq6sFA9tYMKllJDLIW7~gqKhJzBFLmd~QG~36qhqUF3uL-uqNKpe885U16sJlCU3iWgtfU6SAQDGvGpK7WhXLuIm5ySweVy83OjHGFl6x23-01aN1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,148</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">378</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="375737"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f4%2f227%2f375737">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260564600005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (378)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F4%2F227%2F375737" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/375737'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome Structure of the Legume, Lotus japonicus" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f227%2f375737">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

