

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":2,"applicationTime":1899,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/4/135/348357","@type":"ScholarlyArticle","name":"Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl007","keywords":["G+C content variation","human–mouse orthologs","nucleotide substitutions in synonymous codons"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/4/135/348357","author":[{"name":"Takahashi, Naoki","affiliation":"Department of Clinical Laboratory Science, Graduate Course of Medical Science and Technology  Division of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan","@type":"Person"},{"name":"Nakashima, Hiroshi","affiliation":"Department of Clinical Laboratory Science, Graduate Course of Medical Science and Technology  Division of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan","@type":"Person"}],"description":"Abstract.  We conducted a genome-wide analysis of variations in guanine plus cytosine (G+C) content at the third codon position at silent substitution sites of ","pageStart":"135","pageEnd":"140","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences" />
    <meta property="og:description" content="Abstract.  We conducted a genome-wide analysis of variations in guanine plus cytosine (G+C) content at the third codon position at silent substitution sites of " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/4/135/348357" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takahashi, Naoki" /><meta name="citation_author_institution" content="Department of Clinical Laboratory Science, Graduate Course of Medical Science and TechnologyDivision of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan" /><meta name="citation_author" content="Nakashima, Hiroshi" /><meta name="citation_author_institution" content="Department of Clinical Laboratory Science, Graduate Course of Medical Science and TechnologyDivision of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan" /><meta name="citation_title" content="Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences" /><meta name="citation_firstpage" content="135" /><meta name="citation_lastpage" content="140" /><meta name="citation_doi" content="10.1093/dnares/dsl007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; citation_author=International Human Genome Sequencing Consortium.; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome; citation_author=Mouse Genome Sequencing consortium.; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=520-562; " /><meta name="citation_reference" content="citation_title=Genome sequence of the Brown Norway Rat yields insights into mammalian evolution; citation_author=Rat Genome Sequencing Project Consortium.; Nature;  citation_year=2004;  citation_volume=428;  citation_pages=493-521; " /><meta name="citation_reference" content="citation_title=Initial sequence of the chimpanzee genome and comparison with the human genome; citation_author=The Chimpanzee Sequencing and Analysis Consortium.; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=69-87; " /><meta name="citation_reference" content="citation_title=Genes on human chromosome 19 show extreme divergence from the mouse orthologs and a high GC content; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1751-1756; " /><meta name="citation_reference" content="citation_title=Covariation in frequencies of substitution, deletion, transposition, and recombination during eutherian evolution; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=13-26; " /><meta name="citation_reference" content="citation_title=Patterns of insertions and their covariation with substitutions in the rat, mouse, and human genomes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=517-527; " /><meta name="citation_reference" content="citation_title=Characterization of evolutionary rates and constraints in three mammalian genomes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=539-548; " /><meta name="citation_reference" content="citation_title=Patterns of nucleotide substitution, insertion and deletion in the human genome inferred from pseudogenes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=5338-5348; " /><meta name="citation_reference" content="citation_title=Neutral substitutions occur at a faster rate in exons than in noncoding DNA in primate genomes; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=838-844; " /><meta name="citation_reference" content="citation_title=Inferring nonneutral evolution from human-chimp-mouse orthologous gene trios; Science;  citation_year=2003;  citation_volume=302;  citation_pages=1960-1963; " /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution; Proc. Natl Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=166-169; " /><meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=383-397; " /><meta name="citation_reference" content="citation_title=The mosaic genome of warm-blooded vertebrates; Science;  citation_year=1985;  citation_volume=228;  citation_pages=953-958; " /><meta name="citation_reference" content="citation_title=Lengths of chromosomal segments conserved since divergence of man and mouse; Proc. Natl Acad. Sci. USA;  citation_year=1984;  citation_volume=81;  citation_pages=814-818; " /><meta name="citation_reference" content="citation_title=High codon-usage changes in mammalian genes; Mol. Biol. Evol.;  citation_year=1988;  citation_volume=5;  citation_pages=192-194; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool; J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-410; " /><meta name="citation_reference" content="citation_title=Workings of the genetic code; Trend Biochem. Sci.;  citation_year=1980;  citation_volume=5;  citation_pages=327-331; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=A “humanized” green fluorescent protein cDNA adapted for high-level expression in mammalian cells; J. Virol.;  citation_year=1996;  citation_volume=70;  citation_pages=4646-4654; " /><meta name="citation_reference" content="citation_title=Evident diversity of codon usage patterns of human genes with respect to chromosome banding patterns and chromosome numbers; relation between nucleotide sequence data and cytogenetic data; Nucleic Acids Res.;  citation_year=1991;  citation_volume=19;  citation_pages=4333-4339; " /><meta name="citation_reference" content="citation_title=Reconstructing the genomic architecture of ancestral mammals: lessons from human, mouse, and rat genomes; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=507-516; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/4/135/998359/dsl007.pdf" /><meta name="description" content="Abstract.  We conducted a genome-wide analysis of variations in guanine plus cytosine (G+C) content at the third codon position at silent substitution sites of " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/4/135/348357" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/4/135/348357" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/4/135/348357");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl007';
    </script>
<title>
	Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./348357" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcY1PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWthaGFzaGksIE5hb2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDbGluaWNhbCBMYWJvcmF0b3J5IFNjaWVuY2UsIEdyYWR1YXRlIENvdXJzZSBvZiBNZWRpY2FsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3lEaXZpc2lvbiBvZiBIZWFsdGggU2NpZW5jZSwgS2FuYXphd2EgVW5pdmVyc2l0eSwgNS0xMS04MCBLb2RhdHN1bm8sIEthbmF6YXdhIDkyMC0wOTQyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2FzaGltYSwgSGlyb3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ2xpbmljYWwgTGFib3JhdG9yeSBTY2llbmNlLCBHcmFkdWF0ZSBDb3Vyc2Ugb2YgTWVkaWNhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5RGl2aXNpb24gb2YgSGVhbHRoIFNjaWVuY2UsIEthbmF6YXdhIFVuaXZlcnNpdHksIDUtMTEtODAgS29kYXRzdW5vLCBLYW5hemF3YSA5MjAtMDk0MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik5lZ2F0aXZlIENvcnJlbGF0aW9uIG9mIEcrQyBDb250ZW50IGF0IFNpbGVudCBTdWJzdGl0dXRpb24gU2l0ZXMgQmV0d2VlbiBPcnRob2xvZ291cyBIdW1hbiBhbmQgTW91c2UgUHJvdGVpbi1Db2RpbmcgU2VxdWVuY2VzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2wwMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNi8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluaXRpYWwgc2VxdWVuY2luZyBhbmQgYW5hbHlzaXMgb2YgdGhlIGh1bWFuIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUludGVybmF0aW9uYWwgSHVtYW4gR2Vub21lIFNlcXVlbmNpbmcgQ29uc29ydGl1bS47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBtb3VzZSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1Nb3VzZSBHZW5vbWUgU2VxdWVuY2luZyBjb25zb3J0aXVtLjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQyMDsgIGNpdGF0aW9uX3BhZ2VzPTUyMC01NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgQnJvd24gTm9yd2F5IFJhdCB5aWVsZHMgaW5zaWdodHMgaW50byBtYW1tYWxpYW4gZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9UmF0IEdlbm9tZSBTZXF1ZW5jaW5nIFByb2plY3QgQ29uc29ydGl1bS47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00Mjg7ICBjaXRhdGlvbl9wYWdlcz00OTMtNTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNlIG9mIHRoZSBjaGltcGFuemVlIGdlbm9tZSBhbmQgY29tcGFyaXNvbiB3aXRoIHRoZSBodW1hbiBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1UaGUgQ2hpbXBhbnplZSBTZXF1ZW5jaW5nIGFuZCBBbmFseXNpcyBDb25zb3J0aXVtLjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNzsgIGNpdGF0aW9uX3BhZ2VzPTY5LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcyBvbiBodW1hbiBjaHJvbW9zb21lIDE5IHNob3cgZXh0cmVtZSBkaXZlcmdlbmNlIGZyb20gdGhlIG1vdXNlIG9ydGhvbG9ncyBhbmQgYSBoaWdoIEdDIGNvbnRlbnQ7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTE3NTEtMTc1NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q292YXJpYXRpb24gaW4gZnJlcXVlbmNpZXMgb2Ygc3Vic3RpdHV0aW9uLCBkZWxldGlvbiwgdHJhbnNwb3NpdGlvbiwgYW5kIHJlY29tYmluYXRpb24gZHVyaW5nIGV1dGhlcmlhbiBldm9sdXRpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTMtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIGluc2VydGlvbnMgYW5kIHRoZWlyIGNvdmFyaWF0aW9uIHdpdGggc3Vic3RpdHV0aW9ucyBpbiB0aGUgcmF0LCBtb3VzZSwgYW5kIGh1bWFuIGdlbm9tZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTE3LTUyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBldm9sdXRpb25hcnkgcmF0ZXMgYW5kIGNvbnN0cmFpbnRzIGluIHRocmVlIG1hbW1hbGlhbiBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTUzOS01NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIG9mIG51Y2xlb3RpZGUgc3Vic3RpdHV0aW9uLCBpbnNlcnRpb24gYW5kIGRlbGV0aW9uIGluIHRoZSBodW1hbiBnZW5vbWUgaW5mZXJyZWQgZnJvbSBwc2V1ZG9nZW5lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NTMzOC01MzQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXV0cmFsIHN1YnN0aXR1dGlvbnMgb2NjdXIgYXQgYSBmYXN0ZXIgcmF0ZSBpbiBleG9ucyB0aGFuIGluIG5vbmNvZGluZyBETkEgaW4gcHJpbWF0ZSBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTgzOC04NDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVycmluZyBub25uZXV0cmFsIGV2b2x1dGlvbiBmcm9tIGh1bWFuLWNoaW1wLW1vdXNlIG9ydGhvbG9nb3VzIGdlbmUgdHJpb3M7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzAyOyAgY2l0YXRpb25fcGFnZXM9MTk2MC0xOTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3VhbmluZSBhbmQgY3l0b3NpbmUgY29udGVudCBvZiBnZW5vbWljIEROQSBhbmQgYmFjdGVyaWFsIGV2b2x1dGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni0xNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtZWxpb3JhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczogcmF0ZXMgb2YgY2hhbmdlIGFuZCBleGNoYW5nZTsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTM4My0zOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb3NhaWMgZ2Vub21lIG9mIHdhcm0tYmxvb2RlZCB2ZXJ0ZWJyYXRlczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjg7ICBjaXRhdGlvbl9wYWdlcz05NTMtOTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZW5ndGhzIG9mIGNocm9tb3NvbWFsIHNlZ21lbnRzIGNvbnNlcnZlZCBzaW5jZSBkaXZlcmdlbmNlIG9mIG1hbiBhbmQgbW91c2U7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz04MTQtODE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoIGNvZG9uLXVzYWdlIGNoYW5nZXMgaW4gbWFtbWFsaWFuIGdlbmVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xOTItMTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjE1OyAgY2l0YXRpb25fcGFnZXM9NDAzLTQxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ya2luZ3Mgb2YgdGhlIGdlbmV0aWMgY29kZTsgVHJlbmQgQmlvY2hlbS4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzI3LTMzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGljZWxsdWxhciBvcmdhbmlzbXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIOKAnGh1bWFuaXplZOKAnSBncmVlbiBmbHVvcmVzY2VudCBwcm90ZWluIGNETkEgYWRhcHRlZCBmb3IgaGlnaC1sZXZlbCBleHByZXNzaW9uIGluIG1hbW1hbGlhbiBjZWxsczsgSi4gVmlyb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9NDY0Ni00NjU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW50IGRpdmVyc2l0eSBvZiBjb2RvbiB1c2FnZSBwYXR0ZXJucyBvZiBodW1hbiBnZW5lcyB3aXRoIHJlc3BlY3QgdG8gY2hyb21vc29tZSBiYW5kaW5nIHBhdHRlcm5zIGFuZCBjaHJvbW9zb21lIG51bWJlcnM7IHJlbGF0aW9uIGJldHdlZW4gbnVjbGVvdGlkZSBzZXF1ZW5jZSBkYXRhIGFuZCBjeXRvZ2VuZXRpYyBkYXRhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz00MzMzLTQzMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29uc3RydWN0aW5nIHRoZSBnZW5vbWljIGFyY2hpdGVjdHVyZSBvZiBhbmNlc3RyYWwgbWFtbWFsczogbGVzc29ucyBmcm9tIGh1bWFuLCBtb3VzZSwgYW5kIHJhdCBnZW5vbWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTUwNy01MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xMy80LzEzNS85OTgzNTkvZHNsMDA3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBXZSBjb25kdWN0ZWQgYSBnZW5vbWUtd2lkZSBhbmFseXNpcyBvZiB2YXJpYXRpb25zIGluIGd1YW5pbmUgcGx1cyBjeXRvc2luZSAoRytDKSBjb250ZW50IGF0IHRoZSB0aGlyZCBjb2RvbiBwb3NpdGlvbiBhdCBzaWxlbnQgc3Vic3RpdHV0aW9uIHNpdGVzIG9mICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzQ4MzU3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFAxDw2CWp8DXNU00/INa6tGtu4f" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f135%2f348357" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f135%2f348357" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13440" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/4/0/m_dnaresearch13_4.cover.gif?Expires=2147483647&amp;Signature=zpj2n5NzsMk30zA4qehj6yWrvGBU~IiLu4DURkTxdl~55txhlJ5z0a2IpdyGYQcF4vlVunu-~Wh08QUqWKvj48dNgA7n8vLabfgEOY0CoaiBxls7NyW6tCeXW6Sfd2X4lV4uoHFAXj3NLXFLZshkdiFCqF5MJYIi0qOwNjsIUdAwRNhaR9xqEQSv1J65s5eb8WSbVSZ7jKourjIePJi-0rEJRs9B7Kjh~xeLnZz~SmpYlxNUZP4MEUv8qvfsHNtVbSMWxc-iOv9niMhvvwUPs612TggXmBVQynK3LwlztmX3LfsmbR-PpnPA2Dhenm3q1yRIy9w5VQzSlMaqz8FNQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/4">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4944810">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944810">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944819">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944819">1. Correlation of G+C Content at the Third Codon Position at Silent Substitution Sites</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944829">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944829">2. Classification of Orthologous Sequences According to G+C Content at the Third Codon Position at Silent Substitution Sites</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944835">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944835">3. Frequencies of Substitution</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944843">4. The Implications of Substitutions at Silent Sites</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4944849">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4944849">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/13/4/141/348695">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Clinical Laboratory Science, Graduate Course of Medical Science and Technology</div><div class="addr-line">Division of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naoki Takahashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Nakashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="insititution">Department of Clinical Laboratory Science, Graduate Course of Medical Science and Technology</div><div class="addr-line">Division of Health Science, Kanazawa University, 5-11-80 Kodatsuno, Kanazawa 920-0942, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-76-265-2582, Fax. +81-76-234-4360, E-mail: <a href="mailto:naka@kenroku.kanazawa-u.ac.jp" target="_blank">naka@kenroku.kanazawa-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Nakashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakashima H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakashima H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Nakashima</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 135–140, <a href='https://doi.org/10.1093/dnares/dsl007'>https://doi.org/10.1093/dnares/dsl007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">31 January 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">22 August 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="348357" href="/dnaresearch/article-pdf/13/4/135/998359/dsl007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Naoki Takahashi, Hiroshi Nakashima;  Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences, <em>DNA Research</em>, Volume 13, Issue 4, 1 January 2006, Pages 135–140, <a href='https://doi.org/10.1093/dnares/dsl007'>https://doi.org/10.1093/dnares/dsl007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/348357?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/348357?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Negative%20Correlation%20of%20G%2BC%20Content%20at%20Silent%20Substitution%20Sites%20Between%20Orthologous%20Human%20and%20Mouse%20Protein-Coding%20Sequences&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=4&amp;author=Takahashi%2C%20Naoki%3B%20Nakashima%2C%20Hiroshi&amp;startPage=135&amp;endPage=140&amp;contentId=10.1093%2Fdnares%2Fdsl007&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f135%2f348357" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4944810 id="4944810" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We conducted a genome-wide analysis of variations in guanine plus cytosine (G+C) content at the third codon position at silent substitution sites of orthologous human and mouse protein-coding nucleotide sequences. Alignments of 3776 human protein-coding DNA sequences with mouse orthologs having &gt;50 synonymous codons were analyzed, and nucleotide substitutions were counted by comparing sequences in the alignments extracted from gap-free regions. The G+C content at silent sites in these pairs of genes showed a strong negative correlation (<em>r</em> = −0.93). Some gene pairs showed significant differences in G+C content at the third codon position at silent substitution sites. For example, human thymine-DNA glycosylase was A+T-rich at the silent substitution sites, while the orthologous mouse sequence was G+C-rich at the corresponding sites. In contrast, human matrix metalloproteinase 23B was G+C-rich at silent substitution sites, while the mouse ortholog was A+T-rich. We discuss possible implications of this significant negative correlation of G+C content at silent sites.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;G+C content variation&quot;">G+C content variation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;human–mouse orthologs&quot;">human–mouse orthologs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nucleotide substitutions in synonymous codons&quot;">nucleotide substitutions in synonymous codons</a></div><p>The availability of complete mammalian genome sequences<a reveal-id="b1  b4" data-open="b1  b4" class="link link-ref link-reveal xref-bibr">1–4</a> provides an opportunity to characterize nucleotide substitution patterns among mammalian genomes by comparative sequence analysis.<a reveal-id="b5  b11" data-open="b5  b11" class="link link-ref link-reveal xref-bibr">5–11</a> The G+C content in bacterial genomes varies among species from 25% to 75%, but it is relatively homogeneous for genes within a given bacterial genome.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> However, the G+C content of genes in a given mammalian genome varies considerably, because mammalian genomes are a mosaic of long (over hundreds of kilobases) DNA segments known as isochores.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> Some G+C-poor isochores have G+C contents as low as 35%, while G+C-rich isochores have G+C contents as high as 60%. It is known that the genes within a given isochore are fairly homogeneous in G+C content. It has been reported that some homologous mammalian genes that occupy different chromosomal positions differ considerably in their base composition and codon usage.<a reveal-id="b14  b16" data-open="b14  b16" class="link link-ref link-reveal xref-bibr">14–16</a> Human α- and β-globin genes are an example of this position-dependent variation. The α-globin gene cluster occupies a G+C-rich region, while the β-globin gene resides in a G+C-poor region.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p><p>In comparing alignments of orthologous human and mouse sequences, we noted that silent substitution sites at the third codon position were biased toward G+C-rich or A+T-rich nucleotides. For example, human thymine-DNA glycosylase was A+T-rich at silent substitution sites, while the orthologous mouse sequence was G+C-rich at the corresponding sites. However, human matrix metalloproteinase 23B was G+C-rich at silent substitution sites, whereas the mouse ortholog was A+T-rich at those sites. Since complete human and mouse genome sequences are now available, we conducted a comparative genome-wide analysis of G+C content variation at silent substitution sites in orthologous human and mouse sequences.</p>                            <h2 scrollto-destination=4944819 id="4944819" class="section-title" >1. Correlation of G+C Content at the Third Codon Position at Silent Substitution Sites</h2>
<p>The G+C content at the third codon position in synonymous codons, i.e. silent substitutions of the same amino acid, was determined for both human and mouse sequences. For simplicity, only synonymous codons that had an identical nucleotide at the first codon position were considered. The G+C contents at the third codon position at silent substitution sites in the 3776 pairs of human and mouse genes are plotted in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, showing a high negative correlation (<em>r</em> = −0.93). The plot indicates distinct variations in G+C content at mutual silent sites in many human and mouse orthologs.</p>                        <a id="4944823" scrollto-destination="4944823"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/m_dsl007f1.jpeg?Expires=2147483647&Signature=GbN1HpSgzuEt1sfYOa27agsNFsoNbZc4QMrddzPHBcbj-H4vHq4EC9HJVtZrTX5WI9gR8l9m-aFcAhoNjq2STeSAMYep5Q0iMgxms5LyQvqP4Z1O7F8Kr2RN9mL8~ELTzi~J4~jUwzuJjUgZ433JDyDT6-jjJjdweUITmaxUmWhe3JCuHZkzWckH-7r0e5LfLEW6iOEUt0gqkXsYqszjoptEDZk5WtcQ-fevefHWNUyAEiit0XEK8gfik5oS7~1l1tkuioT0G97J7XK2MWO4lCwVJTlL7NIe3x~5XEUHkIW-rPOJy2-KiRz271kYhrcEHFUeIsgfEAH4a2cKk3IYsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="G+C contents at silent sites in 3776 human protein-coding sequences versus G+C contents at silent sites in 3776 corresponding mouse sequences. The human and mouse cDNA sequences were obtained from Reference Sequence Release 11 from the U.S. National Center for Biotechnology Information (). The protein-encoding nucleotide sequences were selected according to the feature table for the data. The amino acid sequences of 28 893 human cDNAs and 25 298 mouse cDNAs were obtained by translation. Orthologs were identified by the two-directional best hit approach using BLASTP.17 Pairs of a given sequence were selected if they showed &gt;30% amino acid identity over three-fourths of the total length. To avoid bias, proteins showing &gt;30% sequence identity with other proteins in the same species were excluded. Gap-free alignment regions longer than 100 amino acid residues and the corresponding DNA sequences were analyzed. Based on this criterion, 3776 pairwise alignments of human and mouse sequences that had &gt;50 synonymous codons were chosen for analysis in this study." path-from-xml="dsl007f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944823/dsl007f1.jpeg" path-from-xml="dsl007f1.jpeg" target="_blank">View large</a><a section="4944823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/dsl007f1.jpeg?Expires=2147483647&Signature=fHoNuwJihyCE4QdQoIOtWaSEj57tp3xmW-NbjjSuw8K5~aK29CYFoanYyKbsPUGWqThNSkPMYlJlUBc8Ro6pUO2zrLbczXYQjskfk~MxXRcMGzdLLu~ttJX8NqssH~UvPdcZAwuvqrNhboRAeHl54087Ijtzwkp8DMYnNrhD4OuRnoke6pyzyUPI3ptlGVu~y8GuOSi-AXIEtA90BYE49fCUi6jYc8SDi-VmR~VGtHI9o4W92DV0tvEgyfNce-BnSJQyENJHxAXy4CeBYRXC8FnA1h2sSG-4unzA7S0C07WjtQIUB5bDJB1D07tv8EUINhg~jEFP8hnirt9JHlKwjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944823&ar=348357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl007f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>G+C contents at silent sites in 3776 human protein-coding sequences versus G+C contents at silent sites in 3776 corresponding mouse sequences. The human and mouse cDNA sequences were obtained from Reference Sequence Release 11 from the U.S. National Center for Biotechnology Information (<a class="link link-uri" href="ftp://ftp.ncbi.nih.gov/refseq/" target="_blank"></a>). The protein-encoding nucleotide sequences were selected according to the feature table for the data. The amino acid sequences of 28 893 human cDNAs and 25 298 mouse cDNAs were obtained by translation. Orthologs were identified by the two-directional best hit approach using BLASTP.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> Pairs of a given sequence were selected if they showed &gt;30% amino acid identity over three-fourths of the total length. To avoid bias, proteins showing &gt;30% sequence identity with other proteins in the same species were excluded. Gap-free alignment regions longer than 100 amino acid residues and the corresponding DNA sequences were analyzed. Based on this criterion, 3776 pairwise alignments of human and mouse sequences that had &gt;50 synonymous codons were chosen for analysis in this study.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/m_dsl007f1.jpeg?Expires=2147483647&Signature=GbN1HpSgzuEt1sfYOa27agsNFsoNbZc4QMrddzPHBcbj-H4vHq4EC9HJVtZrTX5WI9gR8l9m-aFcAhoNjq2STeSAMYep5Q0iMgxms5LyQvqP4Z1O7F8Kr2RN9mL8~ELTzi~J4~jUwzuJjUgZ433JDyDT6-jjJjdweUITmaxUmWhe3JCuHZkzWckH-7r0e5LfLEW6iOEUt0gqkXsYqszjoptEDZk5WtcQ-fevefHWNUyAEiit0XEK8gfik5oS7~1l1tkuioT0G97J7XK2MWO4lCwVJTlL7NIe3x~5XEUHkIW-rPOJy2-KiRz271kYhrcEHFUeIsgfEAH4a2cKk3IYsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="G+C contents at silent sites in 3776 human protein-coding sequences versus G+C contents at silent sites in 3776 corresponding mouse sequences. The human and mouse cDNA sequences were obtained from Reference Sequence Release 11 from the U.S. National Center for Biotechnology Information (). The protein-encoding nucleotide sequences were selected according to the feature table for the data. The amino acid sequences of 28 893 human cDNAs and 25 298 mouse cDNAs were obtained by translation. Orthologs were identified by the two-directional best hit approach using BLASTP.17 Pairs of a given sequence were selected if they showed &gt;30% amino acid identity over three-fourths of the total length. To avoid bias, proteins showing &gt;30% sequence identity with other proteins in the same species were excluded. Gap-free alignment regions longer than 100 amino acid residues and the corresponding DNA sequences were analyzed. Based on this criterion, 3776 pairwise alignments of human and mouse sequences that had &gt;50 synonymous codons were chosen for analysis in this study." path-from-xml="dsl007f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944823/dsl007f1.jpeg" path-from-xml="dsl007f1.jpeg" target="_blank">View large</a><a section="4944823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/dsl007f1.jpeg?Expires=2147483647&Signature=fHoNuwJihyCE4QdQoIOtWaSEj57tp3xmW-NbjjSuw8K5~aK29CYFoanYyKbsPUGWqThNSkPMYlJlUBc8Ro6pUO2zrLbczXYQjskfk~MxXRcMGzdLLu~ttJX8NqssH~UvPdcZAwuvqrNhboRAeHl54087Ijtzwkp8DMYnNrhD4OuRnoke6pyzyUPI3ptlGVu~y8GuOSi-AXIEtA90BYE49fCUi6jYc8SDi-VmR~VGtHI9o4W92DV0tvEgyfNce-BnSJQyENJHxAXy4CeBYRXC8FnA1h2sSG-4unzA7S0C07WjtQIUB5bDJB1D07tv8EUINhg~jEFP8hnirt9JHlKwjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944823&ar=348357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl007f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>G+C contents at silent sites in 3776 human protein-coding sequences versus G+C contents at silent sites in 3776 corresponding mouse sequences. The human and mouse cDNA sequences were obtained from Reference Sequence Release 11 from the U.S. National Center for Biotechnology Information (<a class="link link-uri" href="ftp://ftp.ncbi.nih.gov/refseq/" target="_blank"></a>). The protein-encoding nucleotide sequences were selected according to the feature table for the data. The amino acid sequences of 28 893 human cDNAs and 25 298 mouse cDNAs were obtained by translation. Orthologs were identified by the two-directional best hit approach using BLASTP.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> Pairs of a given sequence were selected if they showed &gt;30% amino acid identity over three-fourths of the total length. To avoid bias, proteins showing &gt;30% sequence identity with other proteins in the same species were excluded. Gap-free alignment regions longer than 100 amino acid residues and the corresponding DNA sequences were analyzed. Based on this criterion, 3776 pairwise alignments of human and mouse sequences that had &gt;50 synonymous codons were chosen for analysis in this study.</p></div></div><p>In 2-fold degenerate codons, the equivalent third position nucleotides are either two purines (A/G) or two pyrimidines (C/T). Therefore, their silent substitutions always result in G+C content variation. However, not every silent substitution in 4-fold degenerate codons yields G+C content variation. To further examine G+C content correlation at silent substitution sites, the G+C contents in 4-fold degenerate codons were analyzed. The G+C contents at silent substitution sites were determined for the eight sets of 4-fold degenerate codons (Ala: GCN, Arg: CGN, Gly: GGN, Leu: CUN, Pro: CCN, Ser: UCN, Thr: ACN and Val: GUN) in human and mouse sequences. The plot of the G+C contents in 4-fold degenerate sites between 2084 human and mouse orthologous sequences having &gt;50 4-fold degenerate codons had a high negative correlation coefficient (<em>r</em> = −0.82; data not shown).</p>                            <h2 scrollto-destination=4944829 id="4944829" class="section-title" >2. Classification of Orthologous Sequences According to G+C Content at the Third Codon Position at Silent Substitution Sites</h2>
<p>Human and mouse orthologous sequence pairs were divided into three groups according to the G+C content at the third codon position in synonymous substitution sites. In group (a), the human gene had a much lower G+C content than that in the mouse gene, in group (b) the human gene had a much higher G+C content than that in the mouse gene, and in group (c) the human gene had a G+C content similar to that in the mouse gene. The number of genes in the three groups varied according to the criterion of the classification. Using a cut-off of 30% lower G+C content in the human gene than in the mouse, 25.4% (960/3776) of the orthologous sequence pairs were classified in group (a). Based on 30% higher G+C content in the human gene than in the mouse gene, group (b) contained 17.2% (648/3776) of the orthologous sequence pairs. Group (c) contained the remaining 57.4% (2168/3776) of sequence pairs, which showed deviations between −30 and +30% in G+C content when human and mouse orthologous sequences were compared. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> lists the proteins encoded by 10 well-characterized genes in each of the three groups. Most of the gene pairs within groups (a), (b) and (c) had different chromosomal locations. However, some genes within the same group had the same chromosomal locations. For example, in group (a), human potassium channel tetramerization domain containing protein 3 and human ribosomal protein S6 kinase are both located on human chromosome 1q41, and both of the corresponding mouse genes are on mouse chromosome 1H6. In group (b), human agrin, human calcium binding protein Cab45 precursor, and human matrix metalloproteinase 23B are all located on human chromosome 1p36.33, and the corresponding mouse genes are on mouse chromosome 4E2. These findings suggest that the G+C content at some silent substitution sites might be determined by their chromosomal locations. This finding is consistent with the report by Bernardi et al.<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p>                        <a id="4944833" scrollto-destination="4944833"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Variations in G+C content at the third codon position in synonymous substitution sites (G+C III) for protein-coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Entire alignment </th><th colspan="3">Synonymous codons </th></tr><tr><th> </th><th colspan="6"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th>No. gene product </th><th>Length </th><th>Identity </th><th colspan="2">G+C (%) </th><th colspan="2">Chromosomal location </th><th colspan="3">G+C III (%) </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Nucleotides </th><th>(%) </th><th>Human </th><th>Mouse </th><th>Human </th><th>Mouse </th><th>No. </th><th>Human </th><th>Mouse </th></tr></thead><tbody><tr><td colspan="10">(a) G+C III content: human ≪ mouse </td></tr><tr><td>    1. Cytidine monophosphate N-acetylneuraminic acid synthetase </td><td>1203 </td><td char=".">88.0 </td><td char=".">43.1 </td><td char=".">49.9 </td><td>12p12.1 </td><td>6G2 </td><td>106 </td><td char=".">19.8 </td><td char=".">84.0 </td></tr><tr><td>    2. DNA helicase HEL308 </td><td>849 </td><td char=".">80.8 </td><td char=".">40.6 </td><td char=".">50.1 </td><td>4q21.23 </td><td>5E4 </td><td>83 </td><td char=".">18.1 </td><td char=".">80.7 </td></tr><tr><td>    3. DNA primase large subunit, 58 kDa </td><td>1515 </td><td char=".">84.4 </td><td char=".">41.4 </td><td char=".">48.6 </td><td>6p12 </td><td>1B </td><td>136 </td><td char=".">19.9 </td><td char=".">80.2 </td></tr><tr><td>    4. Dolichyl phosphate glucosyltransferase </td><td>900 </td><td char=".">87.2 </td><td char=".">41.2 </td><td char=".">48.0 </td><td>13q13.3 </td><td>3C </td><td>71 </td><td char=".">15.5 </td><td char=".">81.7 </td></tr><tr><td>    5. Geranylgeranyltransferase type I, beta subunit </td><td>1131 </td><td char=".">87.1 </td><td char=".">44.4 </td><td char=".">51.6 </td><td>5q22.3 </td><td>18C </td><td>119 </td><td char=".">18.5 </td><td char=".">79.0 </td></tr><tr><td>    6. Phosphatidylinositol glycan, class B mannose transferase </td><td>1482 </td><td char=".">80.2 </td><td char=".">38.7 </td><td char=".">50.0 </td><td>15q21 </td><td>9D </td><td>126 </td><td char=".">12.7 </td><td char=".">86.5 </td></tr><tr><td>    7. Phosphoinositide-3-kinase, regulatory subunit 4, p150 </td><td>3477 </td><td char=".">85.3 </td><td char=".">42.5 </td><td char=".">51.8 </td><td>3q21.3 </td><td>9F1 </td><td>389 </td><td char=".">15.2 </td><td char=".">82.5 </td></tr><tr><td>    8. Potassium channel tetramerization domain containing protein </td><td>1815 </td><td char=".">83.3 </td><td char=".">43.4 </td><td char=".">54.8 </td><td>1q41 </td><td>1H6 </td><td>225 </td><td char=".">12.4 </td><td char=".">83.1 </td></tr><tr><td>    9. Ribosomal protein S6 kinase, 52 kDa, polypeptide 1 </td><td>990 </td><td char=".">80.6 </td><td char=".">44.0 </td><td char=".">56.1 </td><td>1q41 </td><td>1H6 </td><td>88 </td><td char=".">14.8 </td><td char=".">85.2 </td></tr><tr><td>    10. Thymine-DNA glycosylase </td><td>915 </td><td char=".">85.8 </td><td char=".">40.2 </td><td char=".">48.5 </td><td>12q24.1 </td><td>10C1 </td><td>102 </td><td char=".">19.6 </td><td char=".">85.3 </td></tr><tr><td colspan="10">(b) G+C III content: human ≫ mouse </td></tr><tr><td>    1. Agrin (neuronal aggregation factor) </td><td>2448 </td><td char=".">80.2 </td><td char=".">69.0 </td><td char=".">58.7 </td><td>1p36.33 </td><td>4E2 </td><td>270 </td><td char=".">81.9 </td><td char=".">23.0 </td></tr><tr><td>    2. Calcium binding protein Cab45 precursor </td><td>1074 </td><td char=".">82.8 </td><td char=".">60.6 </td><td char=".">51.5 </td><td>1p36.33 </td><td>4E2 </td><td>103 </td><td char=".">85.4 </td><td char=".">24.3 </td></tr><tr><td>    3. Galactose-3-O-sulfotransferase 2 </td><td>657 </td><td char=".">74.1 </td><td char=".">69.0 </td><td char=".">54.8 </td><td>2q37.3 </td><td>1D </td><td>54 </td><td char=".">96.3 </td><td char=".">31.5 </td></tr><tr><td>    4. Alpha-L-iduronidase precursor </td><td>1827 </td><td char=".">80.2 </td><td char=".">67.9 </td><td char=".">56.7 </td><td>4p16.3 </td><td>5E5 </td><td>164 </td><td char=".">87.2 </td><td char=".">21.3 </td></tr><tr><td>    5. Matrix metalloproteinase 23B </td><td>996 </td><td char=".">81.6 </td><td char=".">68.7 </td><td char=".">56.1 </td><td>1p36.33 </td><td>4E2 </td><td>104 </td><td char=".">93.3 </td><td char=".">13.5 </td></tr><tr><td>    6. Methyl-CpG binding domain protein 3 </td><td>801 </td><td char=".">89.1 </td><td char=".">65.2 </td><td char=".">58.8 </td><td>19p13.3 </td><td>10C1 </td><td>73 </td><td char=".">86.3 </td><td char=".">20.6 </td></tr><tr><td>    7. N-methyl D-aspartate receptor 1 isoform NR1-3 </td><td>2814 </td><td char=".">90.7 </td><td char=".">60.8 </td><td char=".">55.8 </td><td>9q34.3 </td><td>2A3 </td><td>235 </td><td char=".">81.3 </td><td char=".">24.7 </td></tr><tr><td>    8. Plakophilin 3 </td><td>2391 </td><td char=".">86.4 </td><td char=".">67.3 </td><td char=".">60.5 </td><td>11p15 </td><td>7F5 </td><td>216 </td><td char=".">83.8 </td><td char=".">21.3 </td></tr><tr><td>    9. Prostaglandin E receptor 1 subtype EP1 </td><td>642 </td><td char=".">80.7 </td><td char=".">76.6 </td><td char=".">65.6 </td><td>19p13.1 </td><td>8C2 </td><td>75 </td><td char=".">89.3 </td><td char=".">29.3 </td></tr><tr><td>    10. Renin binding protein </td><td>1200 </td><td char=".">85.2 </td><td char=".">61.5 </td><td char=".">53.8 </td><td>Xq28 </td><td>XA7.3 </td><td>92 </td><td char=".">82.6 </td><td char=".">22.8 </td></tr><tr><td colspan="10">(c) G+C III content: human ≈ mouse </td></tr><tr><td>    1. Apolipoprotein B </td><td>5394 </td><td char=".">82.3 </td><td char=".">46.8 </td><td char=".">46.9 </td><td>2p24 </td><td>12A1 </td><td>365 </td><td char=".">49.0 </td><td char=".">47.7 </td></tr><tr><td>    2. Dihydrolipoamide S-succinyltransferase </td><td>1203 </td><td char=".">88.2 </td><td char=".">51.3 </td><td char=".">51.6 </td><td>14q24.3 </td><td>12D3 </td><td>88 </td><td char=".">47.7 </td><td char=".">47.7 </td></tr><tr><td>    3. 5′-3′ exoribonuclease 2 </td><td>2736 </td><td char=".">91.9 </td><td char=".">43.9 </td><td char=".">44.2 </td><td>20p11.2 </td><td>2H1 </td><td>146 </td><td char=".">45.9 </td><td char=".">45.2 </td></tr><tr><td>    4. Glutathione synthetase </td><td>1422 </td><td char=".">88.3 </td><td char=".">56.4 </td><td char=".">55.3 </td><td>20q11.2 </td><td>2H1 </td><td>87 </td><td char=".">50.6 </td><td char=".">50.6 </td></tr><tr><td>    5. 5-hydroxytryptamine (seretonin) receptor 3B </td><td>1263 </td><td char=".">80.1 </td><td char=".">50.5 </td><td char=".">50.9 </td><td>11q23.1 </td><td>9B </td><td>92 </td><td char=".">50.0 </td><td char=".">48.9 </td></tr><tr><td>    6. Ornithine carbamoyltransferase </td><td>1062 </td><td char=".">88.9 </td><td char=".">44.1 </td><td char=".">44.9 </td><td>Xp21.1 </td><td>XA1 </td><td>70 </td><td char=".">48.6 </td><td char=".">48.6 </td></tr><tr><td>    7. Polymerase (DNA directed), alpha </td><td>3270 </td><td char=".">90.8 </td><td char=".">41.7 </td><td char=".">40.5 </td><td>Xp22.1 </td><td>XC </td><td>154 </td><td char=".">47.4 </td><td char=".">48.1 </td></tr><tr><td>    8. Transaldolase 1 </td><td>1011 </td><td char=".">88.1 </td><td char=".">54.9 </td><td char=".">55.0 </td><td>11p15.5 </td><td>7F3 </td><td>81 </td><td char=".">54.3 </td><td char=".">55.6 </td></tr><tr><td>    9. Triosephosphate isomerase 1 </td><td>747 </td><td char=".">88.6 </td><td char=".">56.6 </td><td char=".">55.7 </td><td>12p13 </td><td>6F2 </td><td>68 </td><td char=".">52.9 </td><td char=".">52.9 </td></tr><tr><td>    10. Ubiquitin protein ligase E3C </td><td>3249 </td><td char=".">87.3 </td><td char=".">45.9 </td><td char=".">46.1 </td><td>7q36.3 </td><td>5B1 </td><td>265 </td><td char=".">50.2 </td><td char=".">49.8 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Entire alignment </th><th colspan="3">Synonymous codons </th></tr><tr><th> </th><th colspan="6"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th>No. gene product </th><th>Length </th><th>Identity </th><th colspan="2">G+C (%) </th><th colspan="2">Chromosomal location </th><th colspan="3">G+C III (%) </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Nucleotides </th><th>(%) </th><th>Human </th><th>Mouse </th><th>Human </th><th>Mouse </th><th>No. </th><th>Human </th><th>Mouse </th></tr></thead><tbody><tr><td colspan="10">(a) G+C III content: human ≪ mouse </td></tr><tr><td>    1. Cytidine monophosphate N-acetylneuraminic acid synthetase </td><td>1203 </td><td char=".">88.0 </td><td char=".">43.1 </td><td char=".">49.9 </td><td>12p12.1 </td><td>6G2 </td><td>106 </td><td char=".">19.8 </td><td char=".">84.0 </td></tr><tr><td>    2. DNA helicase HEL308 </td><td>849 </td><td char=".">80.8 </td><td char=".">40.6 </td><td char=".">50.1 </td><td>4q21.23 </td><td>5E4 </td><td>83 </td><td char=".">18.1 </td><td char=".">80.7 </td></tr><tr><td>    3. DNA primase large subunit, 58 kDa </td><td>1515 </td><td char=".">84.4 </td><td char=".">41.4 </td><td char=".">48.6 </td><td>6p12 </td><td>1B </td><td>136 </td><td char=".">19.9 </td><td char=".">80.2 </td></tr><tr><td>    4. Dolichyl phosphate glucosyltransferase </td><td>900 </td><td char=".">87.2 </td><td char=".">41.2 </td><td char=".">48.0 </td><td>13q13.3 </td><td>3C </td><td>71 </td><td char=".">15.5 </td><td char=".">81.7 </td></tr><tr><td>    5. Geranylgeranyltransferase type I, beta subunit </td><td>1131 </td><td char=".">87.1 </td><td char=".">44.4 </td><td char=".">51.6 </td><td>5q22.3 </td><td>18C </td><td>119 </td><td char=".">18.5 </td><td char=".">79.0 </td></tr><tr><td>    6. Phosphatidylinositol glycan, class B mannose transferase </td><td>1482 </td><td char=".">80.2 </td><td char=".">38.7 </td><td char=".">50.0 </td><td>15q21 </td><td>9D </td><td>126 </td><td char=".">12.7 </td><td char=".">86.5 </td></tr><tr><td>    7. Phosphoinositide-3-kinase, regulatory subunit 4, p150 </td><td>3477 </td><td char=".">85.3 </td><td char=".">42.5 </td><td char=".">51.8 </td><td>3q21.3 </td><td>9F1 </td><td>389 </td><td char=".">15.2 </td><td char=".">82.5 </td></tr><tr><td>    8. Potassium channel tetramerization domain containing protein </td><td>1815 </td><td char=".">83.3 </td><td char=".">43.4 </td><td char=".">54.8 </td><td>1q41 </td><td>1H6 </td><td>225 </td><td char=".">12.4 </td><td char=".">83.1 </td></tr><tr><td>    9. Ribosomal protein S6 kinase, 52 kDa, polypeptide 1 </td><td>990 </td><td char=".">80.6 </td><td char=".">44.0 </td><td char=".">56.1 </td><td>1q41 </td><td>1H6 </td><td>88 </td><td char=".">14.8 </td><td char=".">85.2 </td></tr><tr><td>    10. Thymine-DNA glycosylase </td><td>915 </td><td char=".">85.8 </td><td char=".">40.2 </td><td char=".">48.5 </td><td>12q24.1 </td><td>10C1 </td><td>102 </td><td char=".">19.6 </td><td char=".">85.3 </td></tr><tr><td colspan="10">(b) G+C III content: human ≫ mouse </td></tr><tr><td>    1. Agrin (neuronal aggregation factor) </td><td>2448 </td><td char=".">80.2 </td><td char=".">69.0 </td><td char=".">58.7 </td><td>1p36.33 </td><td>4E2 </td><td>270 </td><td char=".">81.9 </td><td char=".">23.0 </td></tr><tr><td>    2. Calcium binding protein Cab45 precursor </td><td>1074 </td><td char=".">82.8 </td><td char=".">60.6 </td><td char=".">51.5 </td><td>1p36.33 </td><td>4E2 </td><td>103 </td><td char=".">85.4 </td><td char=".">24.3 </td></tr><tr><td>    3. Galactose-3-O-sulfotransferase 2 </td><td>657 </td><td char=".">74.1 </td><td char=".">69.0 </td><td char=".">54.8 </td><td>2q37.3 </td><td>1D </td><td>54 </td><td char=".">96.3 </td><td char=".">31.5 </td></tr><tr><td>    4. Alpha-L-iduronidase precursor </td><td>1827 </td><td char=".">80.2 </td><td char=".">67.9 </td><td char=".">56.7 </td><td>4p16.3 </td><td>5E5 </td><td>164 </td><td char=".">87.2 </td><td char=".">21.3 </td></tr><tr><td>    5. Matrix metalloproteinase 23B </td><td>996 </td><td char=".">81.6 </td><td char=".">68.7 </td><td char=".">56.1 </td><td>1p36.33 </td><td>4E2 </td><td>104 </td><td char=".">93.3 </td><td char=".">13.5 </td></tr><tr><td>    6. Methyl-CpG binding domain protein 3 </td><td>801 </td><td char=".">89.1 </td><td char=".">65.2 </td><td char=".">58.8 </td><td>19p13.3 </td><td>10C1 </td><td>73 </td><td char=".">86.3 </td><td char=".">20.6 </td></tr><tr><td>    7. N-methyl D-aspartate receptor 1 isoform NR1-3 </td><td>2814 </td><td char=".">90.7 </td><td char=".">60.8 </td><td char=".">55.8 </td><td>9q34.3 </td><td>2A3 </td><td>235 </td><td char=".">81.3 </td><td char=".">24.7 </td></tr><tr><td>    8. Plakophilin 3 </td><td>2391 </td><td char=".">86.4 </td><td char=".">67.3 </td><td char=".">60.5 </td><td>11p15 </td><td>7F5 </td><td>216 </td><td char=".">83.8 </td><td char=".">21.3 </td></tr><tr><td>    9. Prostaglandin E receptor 1 subtype EP1 </td><td>642 </td><td char=".">80.7 </td><td char=".">76.6 </td><td char=".">65.6 </td><td>19p13.1 </td><td>8C2 </td><td>75 </td><td char=".">89.3 </td><td char=".">29.3 </td></tr><tr><td>    10. Renin binding protein </td><td>1200 </td><td char=".">85.2 </td><td char=".">61.5 </td><td char=".">53.8 </td><td>Xq28 </td><td>XA7.3 </td><td>92 </td><td char=".">82.6 </td><td char=".">22.8 </td></tr><tr><td colspan="10">(c) G+C III content: human ≈ mouse </td></tr><tr><td>    1. Apolipoprotein B </td><td>5394 </td><td char=".">82.3 </td><td char=".">46.8 </td><td char=".">46.9 </td><td>2p24 </td><td>12A1 </td><td>365 </td><td char=".">49.0 </td><td char=".">47.7 </td></tr><tr><td>    2. Dihydrolipoamide S-succinyltransferase </td><td>1203 </td><td char=".">88.2 </td><td char=".">51.3 </td><td char=".">51.6 </td><td>14q24.3 </td><td>12D3 </td><td>88 </td><td char=".">47.7 </td><td char=".">47.7 </td></tr><tr><td>    3. 5′-3′ exoribonuclease 2 </td><td>2736 </td><td char=".">91.9 </td><td char=".">43.9 </td><td char=".">44.2 </td><td>20p11.2 </td><td>2H1 </td><td>146 </td><td char=".">45.9 </td><td char=".">45.2 </td></tr><tr><td>    4. Glutathione synthetase </td><td>1422 </td><td char=".">88.3 </td><td char=".">56.4 </td><td char=".">55.3 </td><td>20q11.2 </td><td>2H1 </td><td>87 </td><td char=".">50.6 </td><td char=".">50.6 </td></tr><tr><td>    5. 5-hydroxytryptamine (seretonin) receptor 3B </td><td>1263 </td><td char=".">80.1 </td><td char=".">50.5 </td><td char=".">50.9 </td><td>11q23.1 </td><td>9B </td><td>92 </td><td char=".">50.0 </td><td char=".">48.9 </td></tr><tr><td>    6. Ornithine carbamoyltransferase </td><td>1062 </td><td char=".">88.9 </td><td char=".">44.1 </td><td char=".">44.9 </td><td>Xp21.1 </td><td>XA1 </td><td>70 </td><td char=".">48.6 </td><td char=".">48.6 </td></tr><tr><td>    7. Polymerase (DNA directed), alpha </td><td>3270 </td><td char=".">90.8 </td><td char=".">41.7 </td><td char=".">40.5 </td><td>Xp22.1 </td><td>XC </td><td>154 </td><td char=".">47.4 </td><td char=".">48.1 </td></tr><tr><td>    8. Transaldolase 1 </td><td>1011 </td><td char=".">88.1 </td><td char=".">54.9 </td><td char=".">55.0 </td><td>11p15.5 </td><td>7F3 </td><td>81 </td><td char=".">54.3 </td><td char=".">55.6 </td></tr><tr><td>    9. Triosephosphate isomerase 1 </td><td>747 </td><td char=".">88.6 </td><td char=".">56.6 </td><td char=".">55.7 </td><td>12p13 </td><td>6F2 </td><td>68 </td><td char=".">52.9 </td><td char=".">52.9 </td></tr><tr><td>    10. Ubiquitin protein ligase E3C </td><td>3249 </td><td char=".">87.3 </td><td char=".">45.9 </td><td char=".">46.1 </td><td>7q36.3 </td><td>5B1 </td><td>265 </td><td char=".">50.2 </td><td char=".">49.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The length, the percentage of nucleotide sequence identity, the G+C content of the entire alignment, and chromosomal locations for human and mouse are shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944833" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Variations in G+C content at the third codon position in synonymous substitution sites (G+C III) for protein-coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Entire alignment </th><th colspan="3">Synonymous codons </th></tr><tr><th> </th><th colspan="6"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th>No. gene product </th><th>Length </th><th>Identity </th><th colspan="2">G+C (%) </th><th colspan="2">Chromosomal location </th><th colspan="3">G+C III (%) </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Nucleotides </th><th>(%) </th><th>Human </th><th>Mouse </th><th>Human </th><th>Mouse </th><th>No. </th><th>Human </th><th>Mouse </th></tr></thead><tbody><tr><td colspan="10">(a) G+C III content: human ≪ mouse </td></tr><tr><td>    1. Cytidine monophosphate N-acetylneuraminic acid synthetase </td><td>1203 </td><td char=".">88.0 </td><td char=".">43.1 </td><td char=".">49.9 </td><td>12p12.1 </td><td>6G2 </td><td>106 </td><td char=".">19.8 </td><td char=".">84.0 </td></tr><tr><td>    2. DNA helicase HEL308 </td><td>849 </td><td char=".">80.8 </td><td char=".">40.6 </td><td char=".">50.1 </td><td>4q21.23 </td><td>5E4 </td><td>83 </td><td char=".">18.1 </td><td char=".">80.7 </td></tr><tr><td>    3. DNA primase large subunit, 58 kDa </td><td>1515 </td><td char=".">84.4 </td><td char=".">41.4 </td><td char=".">48.6 </td><td>6p12 </td><td>1B </td><td>136 </td><td char=".">19.9 </td><td char=".">80.2 </td></tr><tr><td>    4. Dolichyl phosphate glucosyltransferase </td><td>900 </td><td char=".">87.2 </td><td char=".">41.2 </td><td char=".">48.0 </td><td>13q13.3 </td><td>3C </td><td>71 </td><td char=".">15.5 </td><td char=".">81.7 </td></tr><tr><td>    5. Geranylgeranyltransferase type I, beta subunit </td><td>1131 </td><td char=".">87.1 </td><td char=".">44.4 </td><td char=".">51.6 </td><td>5q22.3 </td><td>18C </td><td>119 </td><td char=".">18.5 </td><td char=".">79.0 </td></tr><tr><td>    6. Phosphatidylinositol glycan, class B mannose transferase </td><td>1482 </td><td char=".">80.2 </td><td char=".">38.7 </td><td char=".">50.0 </td><td>15q21 </td><td>9D </td><td>126 </td><td char=".">12.7 </td><td char=".">86.5 </td></tr><tr><td>    7. Phosphoinositide-3-kinase, regulatory subunit 4, p150 </td><td>3477 </td><td char=".">85.3 </td><td char=".">42.5 </td><td char=".">51.8 </td><td>3q21.3 </td><td>9F1 </td><td>389 </td><td char=".">15.2 </td><td char=".">82.5 </td></tr><tr><td>    8. Potassium channel tetramerization domain containing protein </td><td>1815 </td><td char=".">83.3 </td><td char=".">43.4 </td><td char=".">54.8 </td><td>1q41 </td><td>1H6 </td><td>225 </td><td char=".">12.4 </td><td char=".">83.1 </td></tr><tr><td>    9. Ribosomal protein S6 kinase, 52 kDa, polypeptide 1 </td><td>990 </td><td char=".">80.6 </td><td char=".">44.0 </td><td char=".">56.1 </td><td>1q41 </td><td>1H6 </td><td>88 </td><td char=".">14.8 </td><td char=".">85.2 </td></tr><tr><td>    10. Thymine-DNA glycosylase </td><td>915 </td><td char=".">85.8 </td><td char=".">40.2 </td><td char=".">48.5 </td><td>12q24.1 </td><td>10C1 </td><td>102 </td><td char=".">19.6 </td><td char=".">85.3 </td></tr><tr><td colspan="10">(b) G+C III content: human ≫ mouse </td></tr><tr><td>    1. Agrin (neuronal aggregation factor) </td><td>2448 </td><td char=".">80.2 </td><td char=".">69.0 </td><td char=".">58.7 </td><td>1p36.33 </td><td>4E2 </td><td>270 </td><td char=".">81.9 </td><td char=".">23.0 </td></tr><tr><td>    2. Calcium binding protein Cab45 precursor </td><td>1074 </td><td char=".">82.8 </td><td char=".">60.6 </td><td char=".">51.5 </td><td>1p36.33 </td><td>4E2 </td><td>103 </td><td char=".">85.4 </td><td char=".">24.3 </td></tr><tr><td>    3. Galactose-3-O-sulfotransferase 2 </td><td>657 </td><td char=".">74.1 </td><td char=".">69.0 </td><td char=".">54.8 </td><td>2q37.3 </td><td>1D </td><td>54 </td><td char=".">96.3 </td><td char=".">31.5 </td></tr><tr><td>    4. Alpha-L-iduronidase precursor </td><td>1827 </td><td char=".">80.2 </td><td char=".">67.9 </td><td char=".">56.7 </td><td>4p16.3 </td><td>5E5 </td><td>164 </td><td char=".">87.2 </td><td char=".">21.3 </td></tr><tr><td>    5. Matrix metalloproteinase 23B </td><td>996 </td><td char=".">81.6 </td><td char=".">68.7 </td><td char=".">56.1 </td><td>1p36.33 </td><td>4E2 </td><td>104 </td><td char=".">93.3 </td><td char=".">13.5 </td></tr><tr><td>    6. Methyl-CpG binding domain protein 3 </td><td>801 </td><td char=".">89.1 </td><td char=".">65.2 </td><td char=".">58.8 </td><td>19p13.3 </td><td>10C1 </td><td>73 </td><td char=".">86.3 </td><td char=".">20.6 </td></tr><tr><td>    7. N-methyl D-aspartate receptor 1 isoform NR1-3 </td><td>2814 </td><td char=".">90.7 </td><td char=".">60.8 </td><td char=".">55.8 </td><td>9q34.3 </td><td>2A3 </td><td>235 </td><td char=".">81.3 </td><td char=".">24.7 </td></tr><tr><td>    8. Plakophilin 3 </td><td>2391 </td><td char=".">86.4 </td><td char=".">67.3 </td><td char=".">60.5 </td><td>11p15 </td><td>7F5 </td><td>216 </td><td char=".">83.8 </td><td char=".">21.3 </td></tr><tr><td>    9. Prostaglandin E receptor 1 subtype EP1 </td><td>642 </td><td char=".">80.7 </td><td char=".">76.6 </td><td char=".">65.6 </td><td>19p13.1 </td><td>8C2 </td><td>75 </td><td char=".">89.3 </td><td char=".">29.3 </td></tr><tr><td>    10. Renin binding protein </td><td>1200 </td><td char=".">85.2 </td><td char=".">61.5 </td><td char=".">53.8 </td><td>Xq28 </td><td>XA7.3 </td><td>92 </td><td char=".">82.6 </td><td char=".">22.8 </td></tr><tr><td colspan="10">(c) G+C III content: human ≈ mouse </td></tr><tr><td>    1. Apolipoprotein B </td><td>5394 </td><td char=".">82.3 </td><td char=".">46.8 </td><td char=".">46.9 </td><td>2p24 </td><td>12A1 </td><td>365 </td><td char=".">49.0 </td><td char=".">47.7 </td></tr><tr><td>    2. Dihydrolipoamide S-succinyltransferase </td><td>1203 </td><td char=".">88.2 </td><td char=".">51.3 </td><td char=".">51.6 </td><td>14q24.3 </td><td>12D3 </td><td>88 </td><td char=".">47.7 </td><td char=".">47.7 </td></tr><tr><td>    3. 5′-3′ exoribonuclease 2 </td><td>2736 </td><td char=".">91.9 </td><td char=".">43.9 </td><td char=".">44.2 </td><td>20p11.2 </td><td>2H1 </td><td>146 </td><td char=".">45.9 </td><td char=".">45.2 </td></tr><tr><td>    4. Glutathione synthetase </td><td>1422 </td><td char=".">88.3 </td><td char=".">56.4 </td><td char=".">55.3 </td><td>20q11.2 </td><td>2H1 </td><td>87 </td><td char=".">50.6 </td><td char=".">50.6 </td></tr><tr><td>    5. 5-hydroxytryptamine (seretonin) receptor 3B </td><td>1263 </td><td char=".">80.1 </td><td char=".">50.5 </td><td char=".">50.9 </td><td>11q23.1 </td><td>9B </td><td>92 </td><td char=".">50.0 </td><td char=".">48.9 </td></tr><tr><td>    6. Ornithine carbamoyltransferase </td><td>1062 </td><td char=".">88.9 </td><td char=".">44.1 </td><td char=".">44.9 </td><td>Xp21.1 </td><td>XA1 </td><td>70 </td><td char=".">48.6 </td><td char=".">48.6 </td></tr><tr><td>    7. Polymerase (DNA directed), alpha </td><td>3270 </td><td char=".">90.8 </td><td char=".">41.7 </td><td char=".">40.5 </td><td>Xp22.1 </td><td>XC </td><td>154 </td><td char=".">47.4 </td><td char=".">48.1 </td></tr><tr><td>    8. Transaldolase 1 </td><td>1011 </td><td char=".">88.1 </td><td char=".">54.9 </td><td char=".">55.0 </td><td>11p15.5 </td><td>7F3 </td><td>81 </td><td char=".">54.3 </td><td char=".">55.6 </td></tr><tr><td>    9. Triosephosphate isomerase 1 </td><td>747 </td><td char=".">88.6 </td><td char=".">56.6 </td><td char=".">55.7 </td><td>12p13 </td><td>6F2 </td><td>68 </td><td char=".">52.9 </td><td char=".">52.9 </td></tr><tr><td>    10. Ubiquitin protein ligase E3C </td><td>3249 </td><td char=".">87.3 </td><td char=".">45.9 </td><td char=".">46.1 </td><td>7q36.3 </td><td>5B1 </td><td>265 </td><td char=".">50.2 </td><td char=".">49.8 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="6">Entire alignment </th><th colspan="3">Synonymous codons </th></tr><tr><th> </th><th colspan="6"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th>No. gene product </th><th>Length </th><th>Identity </th><th colspan="2">G+C (%) </th><th colspan="2">Chromosomal location </th><th colspan="3">G+C III (%) </th></tr><tr><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>Nucleotides </th><th>(%) </th><th>Human </th><th>Mouse </th><th>Human </th><th>Mouse </th><th>No. </th><th>Human </th><th>Mouse </th></tr></thead><tbody><tr><td colspan="10">(a) G+C III content: human ≪ mouse </td></tr><tr><td>    1. Cytidine monophosphate N-acetylneuraminic acid synthetase </td><td>1203 </td><td char=".">88.0 </td><td char=".">43.1 </td><td char=".">49.9 </td><td>12p12.1 </td><td>6G2 </td><td>106 </td><td char=".">19.8 </td><td char=".">84.0 </td></tr><tr><td>    2. DNA helicase HEL308 </td><td>849 </td><td char=".">80.8 </td><td char=".">40.6 </td><td char=".">50.1 </td><td>4q21.23 </td><td>5E4 </td><td>83 </td><td char=".">18.1 </td><td char=".">80.7 </td></tr><tr><td>    3. DNA primase large subunit, 58 kDa </td><td>1515 </td><td char=".">84.4 </td><td char=".">41.4 </td><td char=".">48.6 </td><td>6p12 </td><td>1B </td><td>136 </td><td char=".">19.9 </td><td char=".">80.2 </td></tr><tr><td>    4. Dolichyl phosphate glucosyltransferase </td><td>900 </td><td char=".">87.2 </td><td char=".">41.2 </td><td char=".">48.0 </td><td>13q13.3 </td><td>3C </td><td>71 </td><td char=".">15.5 </td><td char=".">81.7 </td></tr><tr><td>    5. Geranylgeranyltransferase type I, beta subunit </td><td>1131 </td><td char=".">87.1 </td><td char=".">44.4 </td><td char=".">51.6 </td><td>5q22.3 </td><td>18C </td><td>119 </td><td char=".">18.5 </td><td char=".">79.0 </td></tr><tr><td>    6. Phosphatidylinositol glycan, class B mannose transferase </td><td>1482 </td><td char=".">80.2 </td><td char=".">38.7 </td><td char=".">50.0 </td><td>15q21 </td><td>9D </td><td>126 </td><td char=".">12.7 </td><td char=".">86.5 </td></tr><tr><td>    7. Phosphoinositide-3-kinase, regulatory subunit 4, p150 </td><td>3477 </td><td char=".">85.3 </td><td char=".">42.5 </td><td char=".">51.8 </td><td>3q21.3 </td><td>9F1 </td><td>389 </td><td char=".">15.2 </td><td char=".">82.5 </td></tr><tr><td>    8. Potassium channel tetramerization domain containing protein </td><td>1815 </td><td char=".">83.3 </td><td char=".">43.4 </td><td char=".">54.8 </td><td>1q41 </td><td>1H6 </td><td>225 </td><td char=".">12.4 </td><td char=".">83.1 </td></tr><tr><td>    9. Ribosomal protein S6 kinase, 52 kDa, polypeptide 1 </td><td>990 </td><td char=".">80.6 </td><td char=".">44.0 </td><td char=".">56.1 </td><td>1q41 </td><td>1H6 </td><td>88 </td><td char=".">14.8 </td><td char=".">85.2 </td></tr><tr><td>    10. Thymine-DNA glycosylase </td><td>915 </td><td char=".">85.8 </td><td char=".">40.2 </td><td char=".">48.5 </td><td>12q24.1 </td><td>10C1 </td><td>102 </td><td char=".">19.6 </td><td char=".">85.3 </td></tr><tr><td colspan="10">(b) G+C III content: human ≫ mouse </td></tr><tr><td>    1. Agrin (neuronal aggregation factor) </td><td>2448 </td><td char=".">80.2 </td><td char=".">69.0 </td><td char=".">58.7 </td><td>1p36.33 </td><td>4E2 </td><td>270 </td><td char=".">81.9 </td><td char=".">23.0 </td></tr><tr><td>    2. Calcium binding protein Cab45 precursor </td><td>1074 </td><td char=".">82.8 </td><td char=".">60.6 </td><td char=".">51.5 </td><td>1p36.33 </td><td>4E2 </td><td>103 </td><td char=".">85.4 </td><td char=".">24.3 </td></tr><tr><td>    3. Galactose-3-O-sulfotransferase 2 </td><td>657 </td><td char=".">74.1 </td><td char=".">69.0 </td><td char=".">54.8 </td><td>2q37.3 </td><td>1D </td><td>54 </td><td char=".">96.3 </td><td char=".">31.5 </td></tr><tr><td>    4. Alpha-L-iduronidase precursor </td><td>1827 </td><td char=".">80.2 </td><td char=".">67.9 </td><td char=".">56.7 </td><td>4p16.3 </td><td>5E5 </td><td>164 </td><td char=".">87.2 </td><td char=".">21.3 </td></tr><tr><td>    5. Matrix metalloproteinase 23B </td><td>996 </td><td char=".">81.6 </td><td char=".">68.7 </td><td char=".">56.1 </td><td>1p36.33 </td><td>4E2 </td><td>104 </td><td char=".">93.3 </td><td char=".">13.5 </td></tr><tr><td>    6. Methyl-CpG binding domain protein 3 </td><td>801 </td><td char=".">89.1 </td><td char=".">65.2 </td><td char=".">58.8 </td><td>19p13.3 </td><td>10C1 </td><td>73 </td><td char=".">86.3 </td><td char=".">20.6 </td></tr><tr><td>    7. N-methyl D-aspartate receptor 1 isoform NR1-3 </td><td>2814 </td><td char=".">90.7 </td><td char=".">60.8 </td><td char=".">55.8 </td><td>9q34.3 </td><td>2A3 </td><td>235 </td><td char=".">81.3 </td><td char=".">24.7 </td></tr><tr><td>    8. Plakophilin 3 </td><td>2391 </td><td char=".">86.4 </td><td char=".">67.3 </td><td char=".">60.5 </td><td>11p15 </td><td>7F5 </td><td>216 </td><td char=".">83.8 </td><td char=".">21.3 </td></tr><tr><td>    9. Prostaglandin E receptor 1 subtype EP1 </td><td>642 </td><td char=".">80.7 </td><td char=".">76.6 </td><td char=".">65.6 </td><td>19p13.1 </td><td>8C2 </td><td>75 </td><td char=".">89.3 </td><td char=".">29.3 </td></tr><tr><td>    10. Renin binding protein </td><td>1200 </td><td char=".">85.2 </td><td char=".">61.5 </td><td char=".">53.8 </td><td>Xq28 </td><td>XA7.3 </td><td>92 </td><td char=".">82.6 </td><td char=".">22.8 </td></tr><tr><td colspan="10">(c) G+C III content: human ≈ mouse </td></tr><tr><td>    1. Apolipoprotein B </td><td>5394 </td><td char=".">82.3 </td><td char=".">46.8 </td><td char=".">46.9 </td><td>2p24 </td><td>12A1 </td><td>365 </td><td char=".">49.0 </td><td char=".">47.7 </td></tr><tr><td>    2. Dihydrolipoamide S-succinyltransferase </td><td>1203 </td><td char=".">88.2 </td><td char=".">51.3 </td><td char=".">51.6 </td><td>14q24.3 </td><td>12D3 </td><td>88 </td><td char=".">47.7 </td><td char=".">47.7 </td></tr><tr><td>    3. 5′-3′ exoribonuclease 2 </td><td>2736 </td><td char=".">91.9 </td><td char=".">43.9 </td><td char=".">44.2 </td><td>20p11.2 </td><td>2H1 </td><td>146 </td><td char=".">45.9 </td><td char=".">45.2 </td></tr><tr><td>    4. Glutathione synthetase </td><td>1422 </td><td char=".">88.3 </td><td char=".">56.4 </td><td char=".">55.3 </td><td>20q11.2 </td><td>2H1 </td><td>87 </td><td char=".">50.6 </td><td char=".">50.6 </td></tr><tr><td>    5. 5-hydroxytryptamine (seretonin) receptor 3B </td><td>1263 </td><td char=".">80.1 </td><td char=".">50.5 </td><td char=".">50.9 </td><td>11q23.1 </td><td>9B </td><td>92 </td><td char=".">50.0 </td><td char=".">48.9 </td></tr><tr><td>    6. Ornithine carbamoyltransferase </td><td>1062 </td><td char=".">88.9 </td><td char=".">44.1 </td><td char=".">44.9 </td><td>Xp21.1 </td><td>XA1 </td><td>70 </td><td char=".">48.6 </td><td char=".">48.6 </td></tr><tr><td>    7. Polymerase (DNA directed), alpha </td><td>3270 </td><td char=".">90.8 </td><td char=".">41.7 </td><td char=".">40.5 </td><td>Xp22.1 </td><td>XC </td><td>154 </td><td char=".">47.4 </td><td char=".">48.1 </td></tr><tr><td>    8. Transaldolase 1 </td><td>1011 </td><td char=".">88.1 </td><td char=".">54.9 </td><td char=".">55.0 </td><td>11p15.5 </td><td>7F3 </td><td>81 </td><td char=".">54.3 </td><td char=".">55.6 </td></tr><tr><td>    9. Triosephosphate isomerase 1 </td><td>747 </td><td char=".">88.6 </td><td char=".">56.6 </td><td char=".">55.7 </td><td>12p13 </td><td>6F2 </td><td>68 </td><td char=".">52.9 </td><td char=".">52.9 </td></tr><tr><td>    10. Ubiquitin protein ligase E3C </td><td>3249 </td><td char=".">87.3 </td><td char=".">45.9 </td><td char=".">46.1 </td><td>7q36.3 </td><td>5B1 </td><td>265 </td><td char=".">50.2 </td><td char=".">49.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The length, the percentage of nucleotide sequence identity, the G+C content of the entire alignment, and chromosomal locations for human and mouse are shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944833" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4944835 id="4944835" class="section-title" >3. Frequencies of Substitution</h2>
<p>The parts of the alignments for thymine-DNA glycosylase (human gene: NM_003211.3 and mouse gene: NM_011561.1) and for matrix metalloproteinase 23B (human gene: NM_006983.1 and mouse gene: NM_011985.1) are shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2a and b</a>, respectively. In <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>, nucleotides A and T at silent substitution sites are red, nucleotides G and C at silent substitution sites are blue, and other nucleotides are shown in yellow. Amino acids are shown along the DNA sequences.</p>                        <a id="4944839" scrollto-destination="4944839"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/m_dsl007f2.jpeg?Expires=2147483647&Signature=VOJCZe0yoicsJOoI3aIT51Ld13UJaJwH8ZyXlpd~opzW5e0zP4PXQKjsX6e0cHO9SOndT3N2cqia6hdATlCxhDgPEc-7QOl61ZYNdvEz3UCpZ8o1l2xgBgnJBja1qQ7x~Za48Y71biPdR0R65Ju-vaXRchZ~C1ls41XJIflR8D7Sjp2nOpaTPOgMebaS2dwRVX9uKDrJ9ql7zM8-sKhTdlatYR2IqQ0Y8W0jS7brh2AvQdfjq057yRjL9XifqqBqG5VQulgoTSY6ySFFxaX2iP8PIqYy~73WtHNk46jXpvsTysDkEFEGs9xQmWt3tn8avYU-aTCFJ0H9IxBqhmdTMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Panels (a) and (b) show the alignment between sequences of human and mouse thymine-DNA glycosylase, and alignment of human and mouse sequences of matrix metalloproteinase 23B." path-from-xml="dsl007f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944839/dsl007f2.jpeg" path-from-xml="dsl007f2.jpeg" target="_blank">View large</a><a section="4944839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/dsl007f2.jpeg?Expires=2147483647&Signature=OqRns-rONzFUJWQR3XyRdQCbVulhkI1xyMqATEBEdhmb9121O51X~9BqK2qZWiKDx5cSF89vrMt6y5Wks-xH-awDckpoUaqqBAAajloRQjcS31qHZUTO2BKu-G-frtrcjr5-My1ExvqP2t6u9CDVasNrem5R4oB1Eqx8HhqoD7qezRvs1befNjjU5dK6igjd6-OUwg0nx9cqCC4BwILyBDA6OpW9NuT7H8K5ZVlGQO2bF8eVNr73hrncZqU12ceQ7E5dBDJ2EzAS5KJic6h4t-eVVScHrH36J1HlWaTj4owqpOMW53Xa4-Kqp9zRW8rxCaS01rscql0c-L4ZB48hgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944839&ar=348357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl007f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Panels (a) and (b) show the alignment between sequences of human and mouse thymine-DNA glycosylase, and alignment of human and mouse sequences of matrix metalloproteinase 23B.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/m_dsl007f2.jpeg?Expires=2147483647&Signature=VOJCZe0yoicsJOoI3aIT51Ld13UJaJwH8ZyXlpd~opzW5e0zP4PXQKjsX6e0cHO9SOndT3N2cqia6hdATlCxhDgPEc-7QOl61ZYNdvEz3UCpZ8o1l2xgBgnJBja1qQ7x~Za48Y71biPdR0R65Ju-vaXRchZ~C1ls41XJIflR8D7Sjp2nOpaTPOgMebaS2dwRVX9uKDrJ9ql7zM8-sKhTdlatYR2IqQ0Y8W0jS7brh2AvQdfjq057yRjL9XifqqBqG5VQulgoTSY6ySFFxaX2iP8PIqYy~73WtHNk46jXpvsTysDkEFEGs9xQmWt3tn8avYU-aTCFJ0H9IxBqhmdTMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Panels (a) and (b) show the alignment between sequences of human and mouse thymine-DNA glycosylase, and alignment of human and mouse sequences of matrix metalloproteinase 23B." path-from-xml="dsl007f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4944839/dsl007f2.jpeg" path-from-xml="dsl007f2.jpeg" target="_blank">View large</a><a section="4944839" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/4/10.1093/dnares/dsl007/2/dsl007f2.jpeg?Expires=2147483647&Signature=OqRns-rONzFUJWQR3XyRdQCbVulhkI1xyMqATEBEdhmb9121O51X~9BqK2qZWiKDx5cSF89vrMt6y5Wks-xH-awDckpoUaqqBAAajloRQjcS31qHZUTO2BKu-G-frtrcjr5-My1ExvqP2t6u9CDVasNrem5R4oB1Eqx8HhqoD7qezRvs1befNjjU5dK6igjd6-OUwg0nx9cqCC4BwILyBDA6OpW9NuT7H8K5ZVlGQO2bF8eVNr73hrncZqU12ceQ7E5dBDJ2EzAS5KJic6h4t-eVVScHrH36J1HlWaTj4owqpOMW53Xa4-Kqp9zRW8rxCaS01rscql0c-L4ZB48hgg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4944839&ar=348357&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl007f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Panels (a) and (b) show the alignment between sequences of human and mouse thymine-DNA glycosylase, and alignment of human and mouse sequences of matrix metalloproteinase 23B.</p></div></div><p>Nucleotide substitutions were observed in 14% (738 506/5 401 758) of all of the total nucleotides contained within the 3776 pairs of orthologous human and mouse genes. Substitution frequencies in codons were 18% at the first position, 12% at the second and 70% at the third. Silent nucleotide substitutions with an identical nucleotide at the first codon position accounted for 58% (425 945/738 506) of the total substitutions. The substitution frequency of transitions (purine–purine and pyrimidine–pyrimidine substitutions) was 66.1%, and that of transversions (purine–pyrimidine and pyrimidine–purine substitutions) was 33.9%. Transitions accounted for 72.1%, and transversions for 27.9%, of the total silent nucleotide substitutions. Transitions were more frequent than transversions at silent substitutions because transitions at the third codon position are essentially silent.</p><div class="block-child-p">When a silent substitution was observed at an alignment site, a silent nucleotide substitution was assumed to occur once in either branch at a synonymous codon site since the divergence of human and mouse lineage. Silent substitutions at synonymous codon sites between human and mouse sequences were estimated. Nucleotide substitutions were considered from the human sequence. The frequencies of the four nucleotides A, T, C and G at the third codon position at silent substitution sites in human sequence were expressed as <em>f</em>(A), <em>f</em>(T), <em>f</em>(C) and <em>f</em>(G). Let α and β be the transition and the transversion substitution rate per year per site. <em>T</em> indicates the divergence time between human and mouse. Then, the nucleotide substitution frequencies at silent sites from human to mouse were calculated as shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>. Substitution frequency at G or C nucleotide in human silent sites is 2<em>T</em>(α+2β)·(<em>f</em>(G)+<em>f</em>(C)), and that in mouse silent sites is 2<em>T</em>(α+β)·(<em>f</em>(A)+<em>f</em>(T))+2<em>T</em>β·(<em>f</em>(G)+<em>f</em>(C)), which is equivalent to 2<em>T</em>(α+β)-2<em>T</em>α·(<em>f</em>(G)+<em>f</em>(C)). The above nucleotide substitution frequencies were expressed as <em>X</em> and <em>Y</em>, respectively. <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>X</mi><mo>=</mo><mn>2</mn><mi>T</mi><mo>(</mo><mi>α</mi><mo>+</mo><mn>2</mn><mi>β</mi><mo>)</mo><mo>·</mo><mo>(</mo><mi>f</mi><mo>(</mo><mtext>G</mtext><mo>)</mo><mo>+</mo><mi>f</mi><mo>(</mo><mtext>C</mtext><mo>)</mo><mo>)</mo></mrow></math></div></div><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Y</mi><mo>=</mo><mn>2</mn><mi>T</mi><mo>(</mo><mi>α</mi><mo>+</mo><mi>β</mi><mo>)</mo><mo>-</mo><mn>2</mn><mi>T</mi><mi>α</mi><mo>(</mo><mi>f</mi><mo>(</mo><mtext>G</mtext><mo>)</mo><mo>+</mo><mi>f</mi><mo>(</mo><mtext>C</mtext><mo>)</mo><mo>)</mo></mrow></math></div></div> The relationship between <em>X</em> and <em>Y</em> is <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>Y</mi><mo>=</mo><mn>2</mn><mi>T</mi><mo>(</mo><mi>α</mi><mo>+</mo><mi>β</mi><mo>)</mo><mo>-</mo><mo>{</mo><mi>α</mi><mo>/</mo><mtext> </mtext><mo>(</mo><mi>α</mi><mo>+</mo><mi>β</mi><mo>)</mo><mo>}</mo><mo>·</mo><mi>X</mi></mrow></math></div></div> This equation indicates that <em>Y</em> increases when <em>X</em> decreases and <em>Y</em> decreases when <em>X</em> increases. This result indicated a negative correlation in the variation of G+C content at silent sites between the two DNA sequences.</div>                        <a id="4944842" scrollto-destination="4944842"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Rates of nucleotide substitution</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">Mouse nucleotide </th></tr><tr><th> </th><th colspan="4"><hr /> </th></tr><tr><th>Human nucleotide </th><th>A </th><th>T </th><th>C </th><th>G </th></tr></thead><tbody><tr><td>A </td><td> </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> αf(A) </td></tr><tr><td>T </td><td>2<em>T</em> βf(T) </td><td> </td><td>2<em>T</em> αf(T) </td><td>2<em>T</em> βf(T) </td></tr><tr><td>C </td><td>2<em>T</em> βf(C) </td><td>2<em>T</em> αf(C) </td><td> </td><td>2<em>T</em> βf(C) </td></tr><tr><td>G </td><td>2<em>T</em> αf(G) </td><td>2<em>T</em> βf(G) </td><td>2<em>T</em> βf(G) </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">Mouse nucleotide </th></tr><tr><th> </th><th colspan="4"><hr /> </th></tr><tr><th>Human nucleotide </th><th>A </th><th>T </th><th>C </th><th>G </th></tr></thead><tbody><tr><td>A </td><td> </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> αf(A) </td></tr><tr><td>T </td><td>2<em>T</em> βf(T) </td><td> </td><td>2<em>T</em> αf(T) </td><td>2<em>T</em> βf(T) </td></tr><tr><td>C </td><td>2<em>T</em> βf(C) </td><td>2<em>T</em> αf(C) </td><td> </td><td>2<em>T</em> βf(C) </td></tr><tr><td>G </td><td>2<em>T</em> αf(G) </td><td>2<em>T</em> βf(G) </td><td>2<em>T</em> βf(G) </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944842" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Rates of nucleotide substitution</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">Mouse nucleotide </th></tr><tr><th> </th><th colspan="4"><hr /> </th></tr><tr><th>Human nucleotide </th><th>A </th><th>T </th><th>C </th><th>G </th></tr></thead><tbody><tr><td>A </td><td> </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> αf(A) </td></tr><tr><td>T </td><td>2<em>T</em> βf(T) </td><td> </td><td>2<em>T</em> αf(T) </td><td>2<em>T</em> βf(T) </td></tr><tr><td>C </td><td>2<em>T</em> βf(C) </td><td>2<em>T</em> αf(C) </td><td> </td><td>2<em>T</em> βf(C) </td></tr><tr><td>G </td><td>2<em>T</em> αf(G) </td><td>2<em>T</em> βf(G) </td><td>2<em>T</em> βf(G) </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="4">Mouse nucleotide </th></tr><tr><th> </th><th colspan="4"><hr /> </th></tr><tr><th>Human nucleotide </th><th>A </th><th>T </th><th>C </th><th>G </th></tr></thead><tbody><tr><td>A </td><td> </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> βf(A) </td><td>2<em>T</em> αf(A) </td></tr><tr><td>T </td><td>2<em>T</em> βf(T) </td><td> </td><td>2<em>T</em> αf(T) </td><td>2<em>T</em> βf(T) </td></tr><tr><td>C </td><td>2<em>T</em> βf(C) </td><td>2<em>T</em> αf(C) </td><td> </td><td>2<em>T</em> βf(C) </td></tr><tr><td>G </td><td>2<em>T</em> αf(G) </td><td>2<em>T</em> βf(G) </td><td>2<em>T</em> βf(G) </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4944842" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=4944843 id="4944843" class="section-title" >4. The Implications of Substitutions at Silent Sites</h2>
<p>Because substitutions at silent sites in codons do not change amino acids, no effect on proteins would be expected, and these substitutions are commonly thought of as being evolutionarily neutral. However, substitutions at silent sites do alter codon usage. Grantham reported that synonymous codons are used differently by different organisms,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> and Ikemura found a strong positive correlation between codon usage and tRNA content in unicellular organisms.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The codon-choice patterns of genes are often very different among multicellular eukaryotes, and codon usage in mammals is known to have dramatic effects on the translation rate.<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> Our findings on the differential codon usage between human and mouse genes suggest the possibility of different expression patterns.</p><p>Evidence indicates that genes with a high G+C content at the third codon position are usually surrounded by long G+C-rich genomic sequences, while those with a low G+C content at the third position are usually surrounded by long A+T-rich sequences.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Human–mouse genome sequence comparisons demonstrated a large number of rearrangements of conserved syntenic segments.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> Since human and mouse genomes exhibit large variations in G+C content (e.g. isochores),<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> the rearrangements might produce a large deviation in G+C content between human and mouse genes by changing the surrounding sequences. The gene pairs classified into groups (a) and (b), which showed a large variation in G+C content at silent substitution sites, are considered to be products of the rearrangements of syntenic segments. The genes located in an identical syntenic segment exhibited similar G+C content variation. Gene rearrangements could be the cause of large variations in G+C content at silent substitution sites, and might lead to a significant negative correlation.</p><p>Nucleotide substitutions between human and mouse sequences have accumulated during evolution ever since their divergence from a common ancestor. It is generally assumed that the substitutions occurred independently in the two species, and there would seem to be no connection between the silent nucleotide substitutions in humans and mice. The results of this study raise the question of whether correlated substitutions at silent sites might have some possible evolutionary function. Further study is needed to address this issue.</p><p>We are grateful to one anonymous referee and an editor for constructive suggestions regarding negative correlation.</p>                            <h2 scrollto-destination=4944849 id="4944849" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium.</div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="collab">Mouse Genome Sequencing consortium.</div><div class="article-title">Initial sequencing and comparative analysis of the mouse genome</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">520</div>-<div class="lpage">562</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="collab">Rat Genome Sequencing Project Consortium.</div><div class="article-title">Genome sequence of the Brown Norway Rat yields insights into mammalian evolution</div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">428</div> (pg. <div class="fpage">493</div>-<div class="lpage">521</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02426" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02426" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02426"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15057822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="collab">The Chimpanzee Sequencing and Analysis Consortium.</div><div class="article-title">Initial sequence of the chimpanzee genome and comparison with the human genome</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">69</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04072"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16136131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Castresana</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genes on human chromosome 19 show extreme divergence from the mouse orthologs and a high GC content</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1751</div>-<div class="lpage">1756</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20on%20human%20chromosome%2019%20show%20extreme%20divergence%20from%20the%20mouse%20orthologs%20and%20a%20high%20GC%20content&amp;author=J.%20Castresana&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1751-1756" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.8.1751" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.8.1751" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.8.1751"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11937628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hardison</div> <div class="given-names">R. C.</div></div>,  <div class="name"><div class="surname">Roskin</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Covariation in frequencies of substitution, deletion, transposition, and recombination during eutherian evolution</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">13</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Covariation%20in%20frequencies%20of%20substitution%2C%20deletion%2C%20transposition%2C%20and%20recombination%20during%20eutherian%20evolution&amp;author=R.%20C.%20Hardison&amp;author=K.%20M.%20Roskin&amp;author=S.%20Yang&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=13-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.844103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.844103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.844103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12529302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Smit</div> <div class="given-names">A. F.</div></div>,  <div class="name"><div class="surname">Schwartz</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Patterns of insertions and their covariation with substitutions in the rat, mouse, and human genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">517</div>-<div class="lpage">527</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20insertions%20and%20their%20covariation%20with%20substitutions%20in%20the%20rat%2C%20mouse%2C%20and%20human%20genomes&amp;author=S.%20Yang&amp;author=A.%20F.%20Smit&amp;author=S.%20Schwartz&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=517-527" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1984404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1984404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1984404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cooper</div> <div class="given-names">G. M.</div></div>,  <div class="name"><div class="surname">Brudno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stone</div> <div class="given-names">E. A.</div></div>,  <div class="name"><div class="surname">Dubchak</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Batzoglou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sidow</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Characterization of evolutionary rates and constraints in three mammalian genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">539</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20evolutionary%20rates%20and%20constraints%20in%20three%20mammalian%20genomes&amp;author=G.%20M.%20Cooper&amp;author=M.%20Brudno&amp;author=E.%20A.%20Stone&amp;author=I.%20Dubchak&amp;author=S.%20Batzoglou&amp;author=A.%20Sidow&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=539-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2034704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2034704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2034704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Patterns of nucleotide substitution, insertion and deletion in the human genome inferred from pseudogenes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">5338</div>-<div class="lpage">5348</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20nucleotide%20substitution%2C%20insertion%20and%20deletion%20in%20the%20human%20genome%20inferred%20from%20pseudogenes&amp;author=Z.%20Zhang&amp;author=M.%20Gerstein&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=5338-5348" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12954770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Subramanian</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Neutral substitutions occur at a faster rate in exons than in noncoding DNA in primate genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">838</div>-<div class="lpage">844</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neutral%20substitutions%20occur%20at%20a%20faster%20rate%20in%20exons%20than%20in%20noncoding%20DNA%20in%20primate%20genomes&amp;author=S.%20Subramanian&amp;author=S.%20Kumar&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=838-844" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1152803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1152803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1152803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12727904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">A. G.</div></div>,  <div class="name"><div class="surname">Glanowski</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Inferring nonneutral evolution from human-chimp-mouse orthologous gene trios</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">1960</div>-<div class="lpage">1963</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20nonneutral%20evolution%20from%20human-chimp-mouse%20orthologous%20gene%20trios&amp;author=A.%20G.%20Clark&amp;author=S.%20Glanowski&amp;author=R.%20Nielsen&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=1960-1963" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1088821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1088821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1088821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14671302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muto</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Osawa</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">166</div>-<div class="lpage">169</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution&amp;author=A.%20Muto&amp;author=S.%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=166-169" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Amelioration of bacterial genomes: rates of change and exchange</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">383</div>-<div class="lpage">397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amelioration%20of%20bacterial%20genomes%3A%20rates%20of%20change%20and%20exchange&amp;author=J.%20G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=383-397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9089078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bernardi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Olofsson</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Filipski</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The mosaic genome of warm-blooded vertebrates</div>, <div class="source">Science</div> , <div class="year">1985</div>, vol. <div class="volume">228</div> (pg. <div class="fpage">953</div>-<div class="lpage">958</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mosaic%20genome%20of%20warm-blooded%20vertebrates&amp;author=G.%20Bernardi&amp;author=B.%20Olofsson&amp;author=J.%20Filipski&amp;publication_year=1985&amp;journal=Science&amp;volume=228&amp;pages=953-958" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.4001930" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.4001930" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.4001930"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4001930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nadeau</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">B. A.</div></div>. </div><div class="article-title">Lengths of chromosomal segments conserved since divergence of man and mouse</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1984</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">814</div>-<div class="lpage">818</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lengths%20of%20chromosomal%20segments%20conserved%20since%20divergence%20of%20man%20and%20mouse&amp;author=J.%20H.%20Nadeau&amp;author=B.%20A.%20Taylor&amp;publication_year=1984&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=81&amp;pages=814-818" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.81.3.814" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.81.3.814" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.81.3.814"> </span></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mouchiroud</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>. </div><div class="article-title">High codon-usage changes in mammalian genes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1988</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">192</div>-<div class="lpage">194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20codon-usage%20changes%20in%20mammalian%20genes&amp;author=D.%20Mouchiroud&amp;author=C.%20Gautier&amp;publication_year=1988&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=5&amp;pages=192-194" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3367784" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1990</div>, vol. <div class="volume">215</div> (pg. <div class="fpage">403</div>-<div class="lpage">410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Workings of the genetic code</div>, <div class="source">Trend Biochem. Sci.</div> , <div class="year">1980</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">327</div>-<div class="lpage">331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Workings%20of%20the%20genetic%20code&amp;author=R.%20Grantham&amp;publication_year=1980&amp;journal=Trend%20Biochem.%20Sci.&amp;volume=5&amp;pages=327-331" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0968-0004(80)90143-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0968-0004(80)90143-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0968-0004(80)90143-7"> </span></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zolotukhin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Potter</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hauswirth</div> <div class="given-names">W. W.</div></div>,  <div class="name"><div class="surname">Guy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Muzyczka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">A “humanized” green fluorescent protein cDNA adapted for high-level expression in mammalian cells</div>, <div class="source">J. Virol.</div> , <div class="year">1996</div>, vol. <div class="volume">70</div> (pg. <div class="fpage">4646</div>-<div class="lpage">4654</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20%E2%80%9Chumanized%E2%80%9D%20green%20fluorescent%20protein%20cDNA%20adapted%20for%20high-level%20expression%20in%20mammalian%20cells&amp;author=S.%20Zolotukhin&amp;author=M.%20Potter&amp;author=W.%20W.%20Hauswirth&amp;author=J.%20Guy&amp;author=N.%20Muzyczka&amp;publication_year=1996&amp;journal=J.%20Virol.&amp;volume=70&amp;pages=4646-4654" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676491" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wada</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Evident diversity of codon usage patterns of human genes with respect to chromosome banding patterns and chromosome numbers; relation between nucleotide sequence data and cytogenetic data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1991</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">4333</div>-<div class="lpage">4339</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evident%20diversity%20of%20codon%20usage%20patterns%20of%20human%20genes%20with%20respect%20to%20chromosome%20banding%20patterns%20and%20chromosome%20numbers%3B%20relation%20between%20nucleotide%20sequence%20data%20and%20cytogenetic%20data&amp;author=T.%20Ikemura&amp;author=K.%20Wada&amp;publication_year=1991&amp;journal=Nucleic%20Acids%20Res.&amp;volume=19&amp;pages=4333-4339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/19.16.4333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F19.16.4333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F19.16.4333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1886761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bourque</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pevzner</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Tesler</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Reconstructing the genomic architecture of ancestral mammals: lessons from human, mouse, and rat genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">507</div>-<div class="lpage">516</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20the%20genomic%20architecture%20of%20ancestral%20mammals%3A%20lessons%20from%20human%2C%20mouse%2C%20and%20rat%20genomes&amp;author=G.%20Bourque&amp;author=P.%20A.%20Pevzner&amp;author=G.%20Tesler&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=507-516" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1975204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1975204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1975204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15059991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Short Communication">Short Communication</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=348357&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">72</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="348357"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242120100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f4%2f135%2f348357">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242120100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F4%2F135%2F348357" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/348357'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Negative Correlation of G+C Content at Silent Substitution Sites Between Orthologous Human and Mouse Protein-Coding Sequences" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f4%2f135%2f348357">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

