

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/241/1745420","@type":"ScholarlyArticle","name":"A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads","datePublished":"2016-04-20","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw011","keywords":["polymorphic transposon","high-throughput sequencing","rice blast fungus","maize"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/241/1745420","author":[{"name":"Kang, Houxiang","affiliation":"1  State Key Laboratory for Biology of Plant Diseases and Insect Pest ,  Institute of Plant Protection, Chinese Academy of Agricultural Sciences ,  Beijing 100193 ,  China","@type":"Person"},{"name":"Zhu, Dan","affiliation":"1  State Key Laboratory for Biology of Plant Diseases and Insect Pest ,  Institute of Plant Protection, Chinese Academy of Agricultural Sciences ,  Beijing 100193 ,  China    2  Department of Agronomy ,  Hunan Agricultural University ,  Changsha, Hunan 410128 ,  China","@type":"Person"},{"name":"Lin, Runmao","affiliation":"3  Department of Plant Pathology ,  Institute of Vegetables and flowers, Chinese Academy of Agriculture Science ,  Beijing 100081 ,  China","@type":"Person"},{"name":"Opiyo, Stephen Obol","affiliation":"4  Molecular and Cellular Imaging Center – Columbus ,  Ohio Agricultural Research and Development Center ,  Columbus, OH 43210 ,  USA","@type":"Person"},{"name":"Jiang, Ning","affiliation":"5  Department of Horticulture ,  Michigan State University ,  1066 Bogue Street, East Lansing, MI 48823 ,  USA","@type":"Person"},{"name":"Shiu, Shin-Han","affiliation":"6  Department of Plant Biology ,  Michigan State University ,  East Lansing, MI 48823 ,  USA","@type":"Person"},{"name":"Wang, Guo-Liang","affiliation":"1  State Key Laboratory for Biology of Plant Diseases and Insect Pest ,  Institute of Plant Protection, Chinese Academy of Agricultural Sciences ,  Beijing 100193 ,  China    7  Department of Plant Pathology ,  Ohio State University ,  Columbus, OH 43210 ,  USA","@type":"Person"}],"description":"Abstract.  Identification of polymorphic transposable elements (TEs) is important because TE polymorphism creates genetic diversity and influences the function ","pageStart":"241","pageEnd":"251","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads" />
    <meta property="og:description" content="Abstract.  Identification of polymorphic transposable elements (TEs) is important because TE polymorphism creates genetic diversity and influences the function " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/241/1745420" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-20" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kang, Houxiang" /><meta name="citation_author_institution" content="State Key Laboratory for Biology of Plant Diseases and Insect Pest, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author" content="Zhu, Dan" /><meta name="citation_author_institution" content="State Key Laboratory for Biology of Plant Diseases and Insect Pest, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Agronomy, Hunan Agricultural University, Changsha, Hunan 410128, China" /><meta name="citation_author" content="Lin, Runmao" /><meta name="citation_author_institution" content="Department of Plant Pathology, Institute of Vegetables and flowers, Chinese Academy of Agriculture Science, Beijing 100081, China" /><meta name="citation_author" content="Opiyo, Stephen Obol" /><meta name="citation_author_institution" content="Molecular and Cellular Imaging Center – Columbus, Ohio Agricultural Research and Development Center, Columbus, OH 43210, USA" /><meta name="citation_author" content="Jiang, Ning" /><meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, 1066 Bogue Street, East Lansing, MI 48823, USA" /><meta name="citation_author" content="Shiu, Shin-Han" /><meta name="citation_author_institution" content="Department of Plant Biology, Michigan State University, East Lansing, MI 48823, USA" /><meta name="citation_author" content="Wang, Guo-Liang" /><meta name="citation_author_institution" content="State Key Laboratory for Biology of Plant Diseases and Insect Pest, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing 100193, China" /><meta name="citation_author_institution" content="Department of Plant Pathology, Ohio State University, Columbus, OH 43210, USA" /><meta name="citation_title" content="A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads" /><meta name="citation_firstpage" content="241" /><meta name="citation_lastpage" content="251" /><meta name="citation_doi" content="10.1093/dnares/dsw011" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Selfish genes, the phenotype paradigm and genome evolution;  citation_journal_title=Nature;  citation_year=1980;  citation_volume=284;  citation_pages=601-3; " /><meta name="citation_reference" content="citation_title=Selfish DNA: the ultimate parasite;  citation_journal_title=Nature;  citation_year=1980;  citation_volume=284;  citation_pages=604-7; " /><meta name="citation_reference" content="citation_title=Transposable elements: an abundant and natural source of regulatory sequences for host genes;  citation_journal_title=Annu. Rev. Genet.;  citation_year=2012;  citation_volume=46;  citation_pages=21-42; " /><meta name="citation_reference" content="citation_title=Identification of a functional transposon insertion in the maize domestication gene tb1;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1160-3; " /><meta name="citation_reference" content="citation_title=Transposable elements, epigenetics, and genome evolution;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=758-67; " /><meta name="citation_reference" content="citation_title=Transposable element insertions have strongly affected human evolution;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=19945-48; " /><meta name="citation_reference" content="citation_title=How important are transposons for plant evolution?;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2013;  citation_volume=14;  citation_pages=49-61; " /><meta name="citation_reference" content="citation_title=Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=133" /><meta name="citation_reference" content="citation_title=Doubling genome size without polyploidization: dynamics of retrotransposition-driven genomic expansions in Oryza australiensis, a wild relative of rice;  citation_journal_title=Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1262-9; " /><meta name="citation_reference" content="citation_title=Do plants have a one-way ticket to genomic obesity?;  citation_journal_title=Plant Cell;  citation_year=1997;  citation_volume=9;  citation_pages=1509-14; " /><meta name="citation_reference" content="citation_title=Transposable elements and host genome evolution;  citation_journal_title=Trends Ecol. Evol.;  citation_year=2000;  citation_volume=15;  citation_pages=95-9; " /><meta name="citation_reference" content="citation_title=Using RepeatMasker to identify repetitive elements in genomic sequences;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2009;  citation_pages=Unit 4.10" /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1269-76; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=PILER: identification and classification of genomic repeats;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=i152-8; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796" /><meta name="citation_reference" content="citation_title=The sequence of the human genome;  citation_journal_title=Science;  citation_year=2001;  citation_volume=291;  citation_pages=1304-51; " /><meta name="citation_reference" content="citation_title=Genome sequence of the Brown Norway rat yields insights into mammalian evolution;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=428;  citation_pages=493-521; " /><meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. japonica);  citation_journal_title=Science;  citation_year=2002;  citation_volume=296;  citation_pages=92-100; " /><meta name="citation_reference" content="citation_title=A draft sequence of the rice genome (Oryza sativa L. ssp. indica);  citation_journal_title=Science;  citation_year=2002;  citation_volume=296;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs;  citation_journal_title=Heredity;  citation_year=2010;  citation_volume=104;  citation_pages=520-33; " /><meta name="citation_reference" content="citation_title=TEMP: a computational method for analyzing transposable element polymorphism in populations;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=6826-38; " /><meta name="citation_reference" content="citation_title=The use of RelocaTE and unassembled short reads to produce high-resolution snapshots of transposable element generated diversity in rice;  citation_journal_title=G3 (Bethesda);  citation_year=2013;  citation_volume=3;  citation_pages=949-57; " /><meta name="citation_reference" content="citation_title=T-lex: a program for fast and accurate assessment of transposable element presence using next-generation sequencing data;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=e36" /><meta name="citation_reference" content="citation_title=A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications;  citation_journal_title=Biotechniques;  citation_year=1993;  citation_volume=14;  citation_pages=748-50; " /><meta name="citation_reference" content="citation_title=Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2865-71; " /><meta name="citation_reference" content="citation_title=Multiple sequence alignment using ClustalW and ClustalX;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2002;  citation_pages=Unit 2.3" /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Experimental evolution reveals genome-wide spectrum and dynamics of mutations in the rice blast fungus, Magnaporthe oryzae;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e65416" /><meta name="citation_reference" content="citation_title=The genome sequence of the rice blast fungus Magnaporthe grisea;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=434;  citation_pages=980-6; " /><meta name="citation_reference" content="citation_title=The B73 maize genome: complexity, diversity, and dynamics;  citation_journal_title=Science;  citation_year=2009;  citation_volume=326;  citation_pages=1112-5; " /><meta name="citation_reference" content="citation_title=Pot2, an inverted repeat transposon from the rice blast fungus Magnaporthe grisea;  citation_journal_title=Mol. Gen. Genet.;  citation_year=1994;  citation_volume=245;  citation_pages=339-48; " /><meta name="citation_reference" content="citation_title=Gain of virulence caused by insertion of a Pot3 transposon in a Magnaporthe grisea avirulence gene;  citation_journal_title=Mol. Plant Microbe Interact.;  citation_year=2001;  citation_volume=14;  citation_pages=671-4; " /><meta name="citation_reference" content="citation_title=Occan, a novel transposon in the Fot1 family, is ubiquitously found in several Magnaporthe grisea isolates;  citation_journal_title=Curr. Genet.;  citation_year=2003;  citation_volume=42;  citation_pages=322-31; " /><meta name="citation_reference" content="citation_title=The paleontology of intergene retrotransposons of maize;  citation_journal_title=Nat. Genet.;  citation_year=1998;  citation_volume=20;  citation_pages=43-5; " /><meta name="citation_reference" content="citation_title=Zeon-1, a member of a new maize retrotransposon family;  citation_journal_title=Mol. Gen. Genet.;  citation_year=1995;  citation_volume=248;  citation_pages=471-80; " /><meta name="citation_reference" content="citation_title=Molecular analysis of ds controlling element mutations at the adh1 locus of maize;  citation_journal_title=Science;  citation_year=1984;  citation_volume=223;  citation_pages=1265-8; " /><meta name="citation_reference" content="citation_title=Molecular analysis of the Ubiquitous (Uq) transposable element system of Zea mays;  citation_journal_title=Mol. Gen. Genet.;  citation_year=1991;  citation_volume=230;  citation_pages=201-8; " /><meta name="citation_reference" content="citation_title=Recent, extensive, and preferential insertion of members of the miniature inverted-repeat transposable element family Heartbreaker into genic regions of maize;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=1160-5; " /><meta name="citation_reference" content="citation_title=P instability factor: an active maize transposon system associated with the amplification of Tourist-like MITEs and a new superfamily of transposases;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=12572-77; " /><meta name="citation_reference" content="citation_title=Insertion-deletion polymorphisms in 3 ‘ regions of maize genes occur frequently and can be used as highly informative genetic markers;  citation_journal_title=Plant Mol. Biol.;  citation_year=2002;  citation_volume=48;  citation_pages=539-47; " /><meta name="citation_reference" content="citation_title=Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=14992-97; " /><meta name="citation_reference" content="citation_title=Active transposition in genomes;  citation_journal_title=Annu. Rev. Genet.;  citation_year=2012;  citation_volume=46;  citation_pages=651" /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=The Top 10 fungal pathogens in molecular plant pathology;  citation_journal_title=Mol. Plant Pathol.;  citation_year=2012;  citation_volume=13;  citation_pages=414-30; " /><meta name="citation_reference" content="citation_title=Novel insights into rice innate immunity against bacterial and fungal pathogens;  citation_journal_title=Annu. Rev. Phytopathol.;  citation_year=2014;  citation_volume=52;  citation_pages=213-41; " /><meta name="citation_reference" content="citation_title=The Magnaporthe oryzae avirulence gene AvrPiz-t encodes a predicted secreted protein that triggers the immunity in rice mediated by the blast resistance gene Piz-t;  citation_journal_title=Mol. Plant Microbe Interact.;  citation_year=2009;  citation_volume=22;  citation_pages=411-20; " /><meta name="citation_reference" content="citation_title=Transposition of plasmid-borne Tn10 elements does not exhibit simple length-dependence;  citation_journal_title=Genetics;  citation_year=1985;  citation_volume=111;  citation_pages=705-13; " /><meta name="citation_reference" content="citation_title=A transposase-independent mechanism gives rise to precise excision of IS256 from insertion sites in Staphylococcus epidermidis;  citation_journal_title=J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=1488-90; " /><meta name="citation_reference" content="citation_title=Transposition of a rice Mutator-like element in the yeast Saccharomyces cerevisiae;  citation_journal_title=Plant Cell;  citation_year=2015;  citation_volume=27;  citation_pages=132-48; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/241/17463115/dsw011.pdf" /><meta name="description" content="Abstract.  Identification of polymorphic transposable elements (TEs) is important because TE polymorphism creates genetic diversity and influences the function " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/241/1745420" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/241/1745420" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/241/1745420");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw011';
    </script>
<title>
	novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745420" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeZ9PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYW5nLCBIb3V4aWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IGZvciBCaW9sb2d5IG9mIFBsYW50IERpc2Vhc2VzIGFuZCBJbnNlY3QgUGVzdCwgSW5zdGl0dXRlIG9mIFBsYW50IFByb3RlY3Rpb24sIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIEJlaWppbmcgMTAwMTkzLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpodSwgRGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgZm9yIEJpb2xvZ3kgb2YgUGxhbnQgRGlzZWFzZXMgYW5kIEluc2VjdCBQZXN0LCBJbnN0aXR1dGUgb2YgUGxhbnQgUHJvdGVjdGlvbiwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQmVpamluZyAxMDAxOTMsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBZ3Jvbm9teSwgSHVuYW4gQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIENoYW5nc2hhLCBIdW5hbiA0MTAxMjgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGluLCBSdW5tYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFBhdGhvbG9neSwgSW5zdGl0dXRlIG9mIFZlZ2V0YWJsZXMgYW5kIGZsb3dlcnMsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyZSBTY2llbmNlLCBCZWlqaW5nIDEwMDA4MSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPcGl5bywgU3RlcGhlbiBPYm9sIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTW9sZWN1bGFyIGFuZCBDZWxsdWxhciBJbWFnaW5nIENlbnRlciDigJMgQ29sdW1idXMsIE9oaW8gQWdyaWN1bHR1cmFsIFJlc2VhcmNoIGFuZCBEZXZlbG9wbWVudCBDZW50ZXIsIENvbHVtYnVzLCBPSCA0MzIxMCwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmlhbmcsIE5pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSwgTWljaGlnYW4gU3RhdGUgVW5pdmVyc2l0eSwgMTA2NiBCb2d1ZSBTdHJlZXQsIEVhc3QgTGFuc2luZywgTUkgNDg4MjMsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaXUsIFNoaW4tSGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW9sb2d5LCBNaWNoaWdhbiBTdGF0ZSBVbml2ZXJzaXR5LCBFYXN0IExhbnNpbmcsIE1JIDQ4ODIzLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBHdW8tTGlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBmb3IgQmlvbG9neSBvZiBQbGFudCBEaXNlYXNlcyBhbmQgSW5zZWN0IFBlc3QsIEluc3RpdHV0ZSBvZiBQbGFudCBQcm90ZWN0aW9uLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDE5MywgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFBhdGhvbG9neSwgT2hpbyBTdGF0ZSBVbml2ZXJzaXR5LCBDb2x1bWJ1cywgT0ggNDMyMTAsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQSBub3ZlbCBtZXRob2QgZm9yIGlkZW50aWZ5aW5nIHBvbHltb3JwaGljIHRyYW5zcG9zYWJsZSBlbGVtZW50cyB2aWEgc2Nhbm5pbmcgb2YgaGlnaC10aHJvdWdocHV0IHNob3J0IHJlYWRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGZpc2ggZ2VuZXMsIHRoZSBwaGVub3R5cGUgcGFyYWRpZ20gYW5kIGdlbm9tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT0yODQ7ICBjaXRhdGlvbl9wYWdlcz02MDEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZmlzaCBETkE6IHRoZSB1bHRpbWF0ZSBwYXJhc2l0ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTI4NDsgIGNpdGF0aW9uX3BhZ2VzPTYwNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc3Bvc2FibGUgZWxlbWVudHM6IGFuIGFidW5kYW50IGFuZCBuYXR1cmFsIHNvdXJjZSBvZiByZWd1bGF0b3J5IHNlcXVlbmNlcyBmb3IgaG9zdCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0yMS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBmdW5jdGlvbmFsIHRyYW5zcG9zb24gaW5zZXJ0aW9uIGluIHRoZSBtYWl6ZSBkb21lc3RpY2F0aW9uIGdlbmUgdGIxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTExNjAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NhYmxlIGVsZW1lbnRzLCBlcGlnZW5ldGljcywgYW5kIGdlbm9tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9NzU4LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc3Bvc2FibGUgZWxlbWVudCBpbnNlcnRpb25zIGhhdmUgc3Ryb25nbHkgYWZmZWN0ZWQgaHVtYW4gZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xOTk0NS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IGltcG9ydGFudCBhcmUgdHJhbnNwb3NvbnMgZm9yIHBsYW50IGV2b2x1dGlvbj87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00OS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx1IGVsZW1lbnRzIGNvbnRhaW4gbWFueSBiaW5kaW5nIHNpdGVzIGZvciB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgYW5kIG1heSBwbGF5IGEgcm9sZSBpbiByZWd1bGF0aW9uIG9mIGRldmVsb3BtZW50YWwgcHJvY2Vzc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvdWJsaW5nIGdlbm9tZSBzaXplIHdpdGhvdXQgcG9seXBsb2lkaXphdGlvbjogZHluYW1pY3Mgb2YgcmV0cm90cmFuc3Bvc2l0aW9uLWRyaXZlbiBnZW5vbWljIGV4cGFuc2lvbnMgaW4gT3J5emEgYXVzdHJhbGllbnNpcywgYSB3aWxkIHJlbGF0aXZlIG9mIHJpY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTI2Mi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbyBwbGFudHMgaGF2ZSBhIG9uZS13YXkgdGlja2V0IHRvIGdlbm9taWMgb2Jlc2l0eT87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE1MDktMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYWJsZSBlbGVtZW50cyBhbmQgaG9zdCBnZW5vbWUgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgRWNvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz05NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBSZXBlYXRNYXNrZXIgdG8gaWRlbnRpZnkgcmVwZXRpdGl2ZSBlbGVtZW50cyBpbiBnZW5vbWljIHNlcXVlbmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gUHJvdG9jLiBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3BhZ2VzPVVuaXQgNC4xMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBkZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBzZXF1ZW5jZSBmYW1pbGllcyBpbiBzZXF1ZW5jZWQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjY5LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBmYW1pbGllcyBpbiBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPWkzNTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UElMRVI6IGlkZW50aWZpY2F0aW9uIGFuZCBjbGFzc2lmaWNhdGlvbiBvZiBnZW5vbWljIHJlcGVhdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9aTE1Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBmbG93ZXJpbmcgcGxhbnQgQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2Ugb2YgdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTE7ICBjaXRhdGlvbl9wYWdlcz0xMzA0LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIEJyb3duIE5vcndheSByYXQgeWllbGRzIGluc2lnaHRzIGludG8gbWFtbWFsaWFuIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTQyODsgIGNpdGF0aW9uX3BhZ2VzPTQ5My01MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZHJhZnQgc2VxdWVuY2Ugb2YgdGhlIHJpY2UgZ2Vub21lIChPcnl6YSBzYXRpdmEgTC4gc3NwLiBqYXBvbmljYSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjk2OyAgY2l0YXRpb25fcGFnZXM9OTItMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRyYWZ0IHNlcXVlbmNlIG9mIHRoZSByaWNlIGdlbm9tZSAoT3J5emEgc2F0aXZhIEwuIHNzcC4gaW5kaWNhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz03OS05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgcmVwZWF0cyBhbmQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHNlcXVlbmNlZCBnZW5vbWVzOiBob3cgdG8gZmluZCB5b3VyIHdheSB0aHJvdWdoIHRoZSBkZW5zZSBmb3Jlc3Qgb2YgcHJvZ3JhbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTUyMC0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VEVNUDogYSBjb21wdXRhdGlvbmFsIG1ldGhvZCBmb3IgYW5hbHl6aW5nIHRyYW5zcG9zYWJsZSBlbGVtZW50IHBvbHltb3JwaGlzbSBpbiBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9NjgyNi0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVzZSBvZiBSZWxvY2FURSBhbmQgdW5hc3NlbWJsZWQgc2hvcnQgcmVhZHMgdG8gcHJvZHVjZSBoaWdoLXJlc29sdXRpb24gc25hcHNob3RzIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50IGdlbmVyYXRlZCBkaXZlcnNpdHkgaW4gcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgKEJldGhlc2RhKTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9OTQ5LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ULWxleDogYSBwcm9ncmFtIGZvciBmYXN0IGFuZCBhY2N1cmF0ZSBhc3Nlc3NtZW50IG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50IHByZXNlbmNlIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPWUzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmFwaWQgQ1RBQiBETkEgaXNvbGF0aW9uIHRlY2huaXF1ZSB1c2VmdWwgZm9yIFJBUEQgZmluZ2VycHJpbnRpbmcgYW5kIG90aGVyIFBDUiBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03NDgtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpbmRlbDogYSBwYXR0ZXJuIGdyb3d0aCBhcHByb2FjaCB0byBkZXRlY3QgYnJlYWsgcG9pbnRzIG9mIGxhcmdlIGRlbGV0aW9ucyBhbmQgbWVkaXVtIHNpemVkIGluc2VydGlvbnMgZnJvbSBwYWlyZWQtZW5kIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI4NjUtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB1c2luZyBDbHVzdGFsVyBhbmQgQ2x1c3RhbFg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFByb3RvYy4gQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl9wYWdlcz1Vbml0IDIuMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1VU0NMRTogbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHdpdGggaGlnaCBhY2N1cmFjeSBhbmQgaGlnaCB0aHJvdWdocHV0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xNzkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cyB3aGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBlcmltZW50YWwgZXZvbHV0aW9uIHJldmVhbHMgZ2Vub21lLXdpZGUgc3BlY3RydW0gYW5kIGR5bmFtaWNzIG9mIG11dGF0aW9ucyBpbiB0aGUgcmljZSBibGFzdCBmdW5ndXMsIE1hZ25hcG9ydGhlIG9yeXphZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2NTQxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHJpY2UgYmxhc3QgZnVuZ3VzIE1hZ25hcG9ydGhlIGdyaXNlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNDsgIGNpdGF0aW9uX3BhZ2VzPTk4MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQjczIG1haXplIGdlbm9tZTogY29tcGxleGl0eSwgZGl2ZXJzaXR5LCBhbmQgZHluYW1pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI2OyAgY2l0YXRpb25fcGFnZXM9MTExMi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3QyLCBhbiBpbnZlcnRlZCByZXBlYXQgdHJhbnNwb3NvbiBmcm9tIHRoZSByaWNlIGJsYXN0IGZ1bmd1cyBNYWduYXBvcnRoZSBncmlzZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ1OyAgY2l0YXRpb25fcGFnZXM9MzM5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYWluIG9mIHZpcnVsZW5jZSBjYXVzZWQgYnkgaW5zZXJ0aW9uIG9mIGEgUG90MyB0cmFuc3Bvc29uIGluIGEgTWFnbmFwb3J0aGUgZ3Jpc2VhIGF2aXJ1bGVuY2UgZ2VuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTY3MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PY2NhbiwgYSBub3ZlbCB0cmFuc3Bvc29uIGluIHRoZSBGb3QxIGZhbWlseSwgaXMgdWJpcXVpdG91c2x5IGZvdW5kIGluIHNldmVyYWwgTWFnbmFwb3J0aGUgZ3Jpc2VhIGlzb2xhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0zMjItMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwYWxlb250b2xvZ3kgb2YgaW50ZXJnZW5lIHJldHJvdHJhbnNwb3NvbnMgb2YgbWFpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WmVvbi0xLCBhIG1lbWJlciBvZiBhIG5ldyBtYWl6ZSByZXRyb3RyYW5zcG9zb24gZmFtaWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI0ODsgIGNpdGF0aW9uX3BhZ2VzPTQ3MS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuYWx5c2lzIG9mIGRzIGNvbnRyb2xsaW5nIGVsZW1lbnQgbXV0YXRpb25zIGF0IHRoZSBhZGgxIGxvY3VzIG9mIG1haXplOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTIyMzsgIGNpdGF0aW9uX3BhZ2VzPTEyNjUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuYWx5c2lzIG9mIHRoZSBVYmlxdWl0b3VzIChVcSkgdHJhbnNwb3NhYmxlIGVsZW1lbnQgc3lzdGVtIG9mIFplYSBtYXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEdlbi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTIzMDsgIGNpdGF0aW9uX3BhZ2VzPTIwMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNlbnQsIGV4dGVuc2l2ZSwgYW5kIHByZWZlcmVudGlhbCBpbnNlcnRpb24gb2YgbWVtYmVycyBvZiB0aGUgbWluaWF0dXJlIGludmVydGVkLXJlcGVhdCB0cmFuc3Bvc2FibGUgZWxlbWVudCBmYW1pbHkgSGVhcnRicmVha2VyIGludG8gZ2VuaWMgcmVnaW9ucyBvZiBtYWl6ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xMTYwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVAgaW5zdGFiaWxpdHkgZmFjdG9yOiBhbiBhY3RpdmUgbWFpemUgdHJhbnNwb3NvbiBzeXN0ZW0gYXNzb2NpYXRlZCB3aXRoIHRoZSBhbXBsaWZpY2F0aW9uIG9mIFRvdXJpc3QtbGlrZSBNSVRFcyBhbmQgYSBuZXcgc3VwZXJmYW1pbHkgb2YgdHJhbnNwb3Nhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTEyNTcyLTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNlcnRpb24tZGVsZXRpb24gcG9seW1vcnBoaXNtcyBpbiAzIOKAmCByZWdpb25zIG9mIG1haXplIGdlbmVzIG9jY3VyIGZyZXF1ZW50bHkgYW5kIGNhbiBiZSB1c2VkIGFzIGhpZ2hseSBpbmZvcm1hdGl2ZSBnZW5ldGljIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz01MzktNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcy1yZWd1bGF0b3J5IGNvZGUgb2Ygc3RyZXNzLXJlc3BvbnNpdmUgdHJhbnNjcmlwdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTQ5OTItOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjdGl2ZSB0cmFuc3Bvc2l0aW9uIGluIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9NjUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBnYXBwZWQtcmVhZCBhbGlnbm1lbnQgd2l0aCBCb3d0aWUgMjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zNTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgVG9wIDEwIGZ1bmdhbCBwYXRob2dlbnMgaW4gbW9sZWN1bGFyIHBsYW50IHBhdGhvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NDE0LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBpbnNpZ2h0cyBpbnRvIHJpY2UgaW5uYXRlIGltbXVuaXR5IGFnYWluc3QgYmFjdGVyaWFsIGFuZCBmdW5nYWwgcGF0aG9nZW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIFBoeXRvcGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTIxMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1hZ25hcG9ydGhlIG9yeXphZSBhdmlydWxlbmNlIGdlbmUgQXZyUGl6LXQgZW5jb2RlcyBhIHByZWRpY3RlZCBzZWNyZXRlZCBwcm90ZWluIHRoYXQgdHJpZ2dlcnMgdGhlIGltbXVuaXR5IGluIHJpY2UgbWVkaWF0ZWQgYnkgdGhlIGJsYXN0IHJlc2lzdGFuY2UgZ2VuZSBQaXotdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBNaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQxMS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NpdGlvbiBvZiBwbGFzbWlkLWJvcm5lIFRuMTAgZWxlbWVudHMgZG9lcyBub3QgZXhoaWJpdCBzaW1wbGUgbGVuZ3RoLWRlcGVuZGVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTcwNS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0cmFuc3Bvc2FzZS1pbmRlcGVuZGVudCBtZWNoYW5pc20gZ2l2ZXMgcmlzZSB0byBwcmVjaXNlIGV4Y2lzaW9uIG9mIElTMjU2IGZyb20gaW5zZXJ0aW9uIHNpdGVzIGluIFN0YXBoeWxvY29jY3VzIGVwaWRlcm1pZGlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTE0ODgtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zaXRpb24gb2YgYSByaWNlIE11dGF0b3ItbGlrZSBlbGVtZW50IGluIHRoZSB5ZWFzdCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMzItNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy8zLzI0MS8xNzQ2MzExNS9kc3cwMTEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIElkZW50aWZpY2F0aW9uIG9mIHBvbHltb3JwaGljIHRyYW5zcG9zYWJsZSBlbGVtZW50cyAoVEVzKSBpcyBpbXBvcnRhbnQgYmVjYXVzZSBURSBwb2x5bW9ycGhpc20gY3JlYXRlcyBnZW5ldGljIGRpdmVyc2l0eSBhbmQgaW5mbHVlbmNlcyB0aGUgZnVuY3Rpb24gIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0xNzQ1NDIwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZMTVuiFj8MRYYZSOgHrYaEeSWrRV" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f241%2f1745420" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f241%2f1745420" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199296">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199296">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199298">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199298">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199303">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199303">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199313">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199313">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199340">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199340">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199347">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199347">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199349">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199349">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199351">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199351">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199353">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199353">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/225/1745389">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/253/1745562">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Houxiang Kang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory for Biology of Plant Diseases and Insect Pest</div>, <div class="insititution">Institute of Plant Protection, Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100193</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86-10-62817045. E-mail: <a href="mailto:kanghouxiangcaas@163.com" target="_blank">kanghouxiangcaas@163.com</a> (H.K.); Tel. +1-614-292-8231. E-mail: <a href="mailto:wang.620@osu.edu" target="_blank">wang.620@osu.edu</a> (G.-L.W.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Houxiang+Kang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kang H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kang H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Houxiang Kang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dan Zhu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory for Biology of Plant Diseases and Insect Pest</div>, <div class="insititution">Institute of Plant Protection, Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100193</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Agronomy</div>, <div class="insititution">Hunan Agricultural University</div>, <div class="addr-line">Changsha, Hunan 410128</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dan+Zhu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhu D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhu D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dan Zhu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Runmao Lin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Plant Pathology</div>, <div class="insititution">Institute of Vegetables and flowers, Chinese Academy of Agriculture Science</div>, <div class="addr-line">Beijing 100081</div>, <div class="insititution">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Runmao+Lin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lin R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lin R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Runmao Lin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephen Obol Opiyo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Molecular and Cellular Imaging Center – Columbus</div>, <div class="insititution">Ohio Agricultural Research and Development Center</div>, <div class="addr-line">Columbus, OH 43210</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephen+Obol+Opiyo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Opiyo S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Opiyo S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stephen Obol Opiyo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ning Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Horticulture</div>, <div class="insititution">Michigan State University</div>, <div class="addr-line">1066 Bogue Street, East Lansing, MI 48823</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ning+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ning Jiang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shin-Han Shiu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Plant Biology</div>, <div class="insititution">Michigan State University</div>, <div class="addr-line">East Lansing, MI 48823</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shin-Han+Shiu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shiu S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shiu S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shin-Han Shiu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guo-Liang Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">State Key Laboratory for Biology of Plant Diseases and Insect Pest</div>, <div class="insititution">Institute of Plant Protection, Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100193</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Plant Pathology</div>, <div class="insititution">Ohio State University</div>, <div class="addr-line">Columbus, OH 43210</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86-10-62817045. E-mail: <a href="mailto:kanghouxiangcaas@163.com" target="_blank">kanghouxiangcaas@163.com</a> (H.K.); Tel. +1-614-292-8231. E-mail: <a href="mailto:wang.620@osu.edu" target="_blank">wang.620@osu.edu</a> (G.-L.W.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guo-Liang+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guo-Liang Wang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 241–251, <a href='https://doi.org/10.1093/dnares/dsw011'>https://doi.org/10.1093/dnares/dsw011</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">20 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745420" href="/dnaresearch/article-pdf/23/3/241/17463115/dsw011.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Houxiang Kang, Dan Zhu, Runmao Lin, Stephen Obol Opiyo, Ning Jiang, Shin-Han Shiu, Guo-Liang Wang;  A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 241–251, <a href='https://doi.org/10.1093/dnares/dsw011'>https://doi.org/10.1093/dnares/dsw011</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745420?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20novel%20method%20for%20identifying%20polymorphic%20transposable%20elements%20via%20scanning%20of%20high-throughput%20short%20reads&amp;publicationDate=2016-04-20&amp;volumeNum=23&amp;issueNum=3&amp;author=Kang%2C%20Houxiang%3B%20Zhu%2C%20Dan&amp;startPage=241&amp;endPage=251&amp;contentId=10.1093%2Fdnares%2Fdsw011&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f241%2f1745420" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199296 id="87199296" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Identification of polymorphic transposable elements (TEs) is important because TE polymorphism creates genetic diversity and influences the function of genes in the host genome. However, <em>de novo</em> scanning of polymorphic TEs remains a challenge. Here, we report a novel computational method, called PTEMD (polymorphic TEs and their movement detection), for <em>de novo</em> discovery of genome-wide polymorphic TEs. PTEMD searches highly identical sequences using reads supported breakpoint evidences. Using PTEMD, we identified 14 polymorphic TE families (905 sequences) in rice blast fungus <em>Magnaporthe oryzae</em>, and 68 (10,618 sequences) in maize. We validated one polymorphic TE family experimentally, MoTE-1; all MoTE-1 family members are located in different genomic loci in the three tested isolates. We found that 57.1% (8 of 14) of the PTEMD-detected polymorphic TE families in <em>M. oryzae</em> are active. Furthermore, our data indicate that there are more polymorphic DNA transposons in maize than their counterparts of retrotransposons despite the fact that retrotransposons occupy largest fraction of genomic mass. We demonstrated that PTEMD is an effective tool for identifying polymorphic TEs in <em>M. oryzae</em> and maize genomes. PTEMD and the genome-wide polymorphic TEs in <em>M. oryzae</em> and maize are publically available at <a class="link link-uri" href="http://www.kanglab.cn/blast/PTEMD_V1.02.htm">http://www.kanglab.cn/blast/PTEMD_V1.02.htm</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;polymorphic transposon&quot;">polymorphic transposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-throughput sequencing&quot;">high-throughput sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rice blast fungus&quot;">rice blast fungus</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="maize">maize</a></div>                            <h2 scrollto-destination=87199298 id="87199298" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Transposable elements (TEs) are repetitive DNA sequences capable of moving in genomes. On the basis of their transposition mechanism, TEs can be divided into two major classes: RNA-based retrotransposons (Class I) and DNA transposons (Class II). Since the discovery of TEs &gt;50 years ago, TEs have been regarded as genetic parasites.<sup><a reveal-id="DSW011C1  DSW011C3" data-open="DSW011C1  DSW011C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Recent studies have, however, indicated that TEs and their movements affect genome size, genome stability, gene function, gene evolution, and epigenetic regulation.<sup><a reveal-id="DSW011C4  DSW011C8" data-open="DSW011C4  DSW011C8" class="link link-ref link-reveal xref-bibr">4–8</a></sup> Also, TEs are important factor for genome size variation in plants.<sup><a reveal-id="DSW011C9" data-open="DSW011C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In grasses, the diploid genome sizes vary ∼30-fold, and most of the variation is due to the amplification of LTR retrotransposons in the intergenic regions of genomes.<sup><a reveal-id="DSW011C10" data-open="DSW011C10" class="link link-ref link-reveal xref-bibr">10</a></sup> It has become increasingly clear that TEs might be a major genomic source of genetic diversity that enables host genomes to respond to environmental changes.<sup><a reveal-id="DSW011C11" data-open="DSW011C11" class="link link-ref link-reveal xref-bibr">11</a></sup></p><p>Despite the presence of thousands or millions of TEs in a genome, most of them are silenced or are no longer mobile. In other words, few of them are still actively transposing that create TE polymorphism in closely related individuals in a species. Nevertheless, it is the polymorphic or active TEs that play the most important role in inserting polymorphism and allelic diversity, which are essential for population dynamics.</p><p>Given the importance of polymorphic TEs, however, to the best of our knowledge, no program has been designed for <em>de novo</em> scanning polymorphic TEs in a genome-wide fashion. So far, programs such as RepeatMasker,<sup><a reveal-id="DSW011C12" data-open="DSW011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> RECON,<sup><a reveal-id="DSW011C13" data-open="DSW011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Repeatscout,<sup><a reveal-id="DSW011C14" data-open="DSW011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and Piler<sup><a reveal-id="DSW011C15" data-open="DSW011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> have been used to identify repeat sequences (including TEs). RepeatMasker, which detects repeat sequences by using both genome sequences and repeat sequence libraries,<sup><a reveal-id="DSW011C12" data-open="DSW011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> has been used in many genome projects.<sup><a reveal-id="DSW011C16  DSW011C20" data-open="DSW011C16  DSW011C20" class="link link-ref link-reveal xref-bibr">16–20</a></sup> Piler is a signature-based repeat searching tool, the program searches a query sequence for particular structures or motifs that are characteristic of a known repeat sequence.<sup><a reveal-id="DSW011C15" data-open="DSW011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> An important limitation of RepeatMasker and Piler is that they rely on the structure and characteristics of known elements and therefore cannot detect novel elements. In contrast, Repeatscout<sup><a reveal-id="DSW011C14" data-open="DSW011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> uses <em>k-</em>mer and spaced seed approaches<sup><a reveal-id="DSW011C21" data-open="DSW011C21" class="link link-ref link-reveal xref-bibr">21</a></sup> that can identify repeat families <em>de novo</em> based on genome sequences without the use of repeat libraries. In addition to identifying repeat sequences, some programs, like TEMP,<sup><a reveal-id="DSW011C22" data-open="DSW011C22" class="link link-ref link-reveal xref-bibr">22</a></sup> RelocaTE,<sup><a reveal-id="DSW011C23" data-open="DSW011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and T-lex,<sup><a reveal-id="DSW011C24" data-open="DSW011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> can detect TE insertion polymorphisms (TIPs) based on high-throughput pair-end short reads or next-generation sequencing (NGS) reads from a single sample or multiple samples from individuals of a population. However, all of the above-mentioned tools do not identify TEs <em>de novo</em>. Therefore, there is a need to develop a program for <em>de novo</em> identification of polymorphic TEs genome-widely.</p><p>In this study, we developed a novel method named polymorphic TEs and their movement detection (PTEMD) that identifies polymorphic TEs <em>de novo</em> and uncovers genome-wide TIPs using reference genome and high-throughput short reads. Our method relies on reads situated at ‘breakpoints’, i.e. reads that span inserted TEs and their flanking sequences. The new method was written in a Linux-based program. Using the PTEMD program with representative fungal and plant genomes, we have detected multiple polymorphic TE families and TIPs.</p>                            <h2 scrollto-destination=87199303 id="87199303" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199304 id="87199304" class="section-title" data-legacy-id=s2a>2.1. Genome sequence data</h3>
<p>The <em>Magnaporthe oryzae</em> reference genome sequence (version 8) was downloaded from the Broad Institute (<a class="link link-uri" href="http://www.broadinstitute.org">http://www.broadinstitute.org</a>). To test the PTEMD program, we re-sequenced the genomes of <em>M. oryzae</em> strains HM-1 and HM-2, which were isolated from rice lesions by the single-spore method (data available from the PTEMD program homepage <a class="link link-uri" href="http://www.kanglab.cn/blast/PTEMD_V1.02.htm">http://www.kanglab.cn/blast/PTEMD_V1.02.htm</a>). The strains were cultured on oatmeal medium, and the hyphae were collected. DNA was extracted from hyphae using the CTAB method,<sup><a reveal-id="DSW011C25" data-open="DSW011C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and the purified DNA was used for constructing Illumina sequencing libraries with about 500-bp insert size. The libraries were sequenced with the Illumina Hiseq 2000 platform to generate 101-bp pair-ends reads with about ×50 coverage of the <em>M. oryzae</em> genome. The re-sequence data sets of <em>M. oryzae</em> generations 0, 10, and 20 were downloaded from a <em>M. oryzae</em> re-sequencing project (GenBank accession number: one isolate of generation 0: SRX220856; three isolates of generation 10: SRX220857, SRX220858, and SRX220859; three isolates of generation 20: SRX220860, SRX220861, and SRX220862). The maize B73 reference genome sequence was downloaded from the Plant Genome Database (<a class="link link-uri" href="http://www.plantgdb.org/">http://www.plantgdb.org/</a>). Mo17 re-sequencing data set was downloaded from GenBank SRA data sets (GenBank accession number SRX245309).</p>                            <h3 scrollto-destination=87199306 id="87199306" class="section-title" data-legacy-id=s2b>2.2. Comparison of PTEMD with different programs</h3>
<p>PTEMD-A focus is to identify polymorphic TEs genome-wide, and it is the first program that focuses on <em>de novo</em> identifying polymorphic TEs. The programs RepeatMasker,<sup><a reveal-id="DSW011C12" data-open="DSW011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Repeatscout,<sup><a reveal-id="DSW011C14" data-open="DSW011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Piler,<sup><a reveal-id="DSW011C15" data-open="DSW011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and RECON<sup><a reveal-id="DSW011C13" data-open="DSW011C13" class="link link-ref link-reveal xref-bibr">13</a></sup> were compared with PTEMD-A for <em>de novo</em> scanning of repeat sequences in <em>M. oryzae</em> genome. The library used in this study is the Repbase library (version 19.06; <a class="link link-uri" href="http://www.girinst.org/repbase/">http://www.girinst.org/repbase/</a>). The programs used in this study are RepeatMasker (version 4.05; <a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>), Repeatscout version 1.0.5<sup><a reveal-id="DSW011C14" data-open="DSW011C14" class="link link-ref link-reveal xref-bibr">14</a></sup>, Piler (<a class="link link-uri" href="http://drive5.com/piler">http://drive5.com/piler</a>), and RECON version 1.08 (<a class="link link-uri" href="http://www.repeatmasker.org/RECON-1.08.tar.gz">http://www.repeatmasker.org/RECON-1.08.tar.gz</a>). The default or general parameters were used for the programs.</p><p>PTEMD-B focuses on identifying the TIPs, and its performance was compared with the well designed and widely used TIP scanning programs, such as RelocaTE version 1.05,<sup><a reveal-id="DSW011C23" data-open="DSW011C23" class="link link-ref link-reveal xref-bibr">23</a></sup> T-lex (version 2),<sup><a reveal-id="DSW011C24" data-open="DSW011C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and TEMP (version 1.01).<sup><a reveal-id="DSW011C22" data-open="DSW011C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h3 scrollto-destination=87199309 id="87199309" class="section-title" data-legacy-id=s2c>2.3. PCR validation of TIPs</h3>
<p>In order to validate the predicted TIPs, 30 positions in HM-1 and 30 in HM-2 were randomly selected from the PTEMD-detected MoTE-1 insertion regions. We designed 60 specific primer pairs for the 60 selected insertion regions. PCR (Tm: 55°C, 36 cycles) was carried out to determine the TE polymorphisms between reference genome and HM-1/2. When reference genome is subjected to PCR, all of the primer pairs amplify 100- to 300-bp products. The products were resolved on 1% agarose gel through electrophoresis to distinguish the size difference due to TE insertion between HM-1/2 and the reference genome.</p>                            <h3 scrollto-destination=87199311 id="87199311" class="section-title" data-legacy-id=s2d>2.4. Runtime and memory requirement of PTEMD</h3>
<p>The present version of PTEMD was programmed in Perl and was tested in a Linux system. We tested PTEMD using a small genome (<em>M. oryzae</em>), and a large and repeat-rich genome (maize; <em>Zea mays</em> L<em>.</em>). In the <em>M. oryzae</em> genome, the 40-Mb genome sequence and 2-Gb clean bases of high-throughput reads data (×50 coverage of the genome) were used; ∼1.5 h of CPU time was needed for detecting all of the polymorphic TE families, and 10 min of CPU time was needed for <em>de novo</em> detection of the new insertion sites for a single 2-kb TE sequence. When the maize genome B73 was tested (genome size ≈ 2.1 Gb), 70-Gb clean bases high-throughput reads data (accession number: SRX245309, about ×35 coverage of the genome) were used, ∼23 h of CPU time was needed for detecting all of the polymorphic TE families.</p>                            <h2 scrollto-destination=87199313 id="87199313" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87199314 id="87199314" class="section-title" data-legacy-id=s3a>3.1. PTEMD detection of polymorphic TEs and TIPs</h3>
<p>Active TEs or recently active TEs often create insertion/deletion (indel) polymorphisms in the population. PTEMD utilizes this feature to collect TEs or other movable sequences that are involved in the formation of polymorphisms. PTEMD consists of PTEMD-A (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>A) and PTEMD-B (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>B) pipelines. The objective of PTEMD-A is to identify the polymorphic TE families. First, high-throughput short reads will be mapped to the reference genome (Fig. <a reveal-id="DSW011F1" data-open="DSW011F1" class="link link-reveal link-table xref-fig">1</a>A, left and middle part, partially assembled genomes are also been supported). For a certain locus, if there is a TE insertion in the reference genome but absent from the genome under re-sequencing, a ‘gap’ will form where reads are mapped to the flanking sequences but not the sequence in between (detail parameters refer to Pindel program).<sup><a reveal-id="DSW011C26" data-open="DSW011C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Those sequences in the gap represent putative mobile unit and will be retrieved from the reference genome, thereafter, PTEMD uses a pair-wise sequences alignment strategy to construct a distance matrix (Fig. <a reveal-id="DSW011F1" data-open="DSW011F1" class="link link-reveal link-table xref-fig">1</a>A, right part) for all retrieved sequences. We used a time-saving method in this pair-wise sequence-alignment step. Only sequences of similar size were used for the pair-wise alignment (not exceeding 10% length), and the sequence identity was recorded as a vector in each distance-matrix calculating cycle. The sequences with low sequence identity (&lt;75%) were not used for the alignment in the next step of the traversal cycle. This reduced the running time by 53–73% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>A–C) in the tested data sets. Since intact polymorphic transposons are usually highly similar, only those sequences which have at least three homologous sequences sharing ≥97% identity over 95% length will be retained and classified into different TE families, using this criterion, our result showed that only a small part of PTEMD-detected sequence families are simple repeats or truncated TEs (6.7% in <em>M. oryzae</em> and 16.0% in maize), which may be due to indel/homologous recombination/sequence rearrangement. In other words, most of the PTEMD-detected sequence families in <em>M. oryzae</em> and maize are <em>bona fide</em> intact transposons. </p>                        <a id="87199316" scrollto-destination="87199316"></a>
<div id="DSW011F1" data-id="DSW011F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01101.png?Expires=2147483647&Signature=f6gOnNcchEmcdxwjaGHWGhZ6X43SazZ0yNAoM8wpdACI7y44wGx0UQEvTHOsLZw4Nh48KBJDTS077aWqvhnZ5r~zt4hjllVx7GM9xT2ygqu4XsdBGLUVvdu6RAFhRnhNhb7ZOu-lSXcHT6paQzRVl3zYqrFEUTBKni~YZUH3HOOOUBmH1S-7Z8LXPwpzIKMuh1M2q814BqnrJ8fMLPlsQGThZ6wz9yJOjQvpPruJ6p4MqhWvOVvLPZtuLNRZ91A7GwOp2SUu48R~h2cRyfUqfvo-JtBf8RA~x05aRrTF3S0KerdL1oRobRBHsUkpTVribra1AKrrYaR9yGdGtuFz2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The core algorithm of PTEMD. (A) PTEMD-A, de novo active TE scanning pipeline. The high-throughput short reads are first mapped to the reference genome and the genomic regions correspond to mapping gaps (the dotted line indicated the gaps) are identified and classified into different repeat families with neighbour-join clustering. (B) PTEMD-B, de novo TIPs detecting pipeline. The middle panel represents the high-throughput reads which partially mapped to the TE sequence, the un-matched part of the reads were re-located to the reference genome sequences. There are two major mapping patterns: (i) the interval distance on the reference genome between left and right mapped reads is near or equal to 0 (top panel of B), representing a TIP in the re-sequenced individual; (ii) the interval distance between left and right mapped reads is equal or almost equal to the length of the TE (bottom panel of B), representing a same TE distribution between the reference and the re-sequenced individual. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199316/dsw01101.tiff" path-from-xml="dsw01101.tiff" target="_blank">View large</a><a section="87199316" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01101.png?Expires=2147483647&Signature=UIMY2rV~pzo9A8mEtNfsGTd1tOW03V~7TbX9xfk5f6Ts64yUmGTUc5OixCallcM6VwALVSodScNqOH~GX7v8oXpyy9wOJ79rSgkLLqmBmr1RztIBtZXSJolJaO4bjXaOvrS~0l-PUT7~jeRmmUZRH-oluz6SF5It6M7ABX3~2L9mkehhPIZhLCghby8pT-VVdZjwbiP1wxgiCxtaFURHd4w7oV6M5Xk3TLxpw86VKWIYdWBeqPM5B0d6AXqMx2sPiwKH~hShabENrDr6j7phTdjrtJn2qh5cw3XSI0Zs~EB8fB3WOy6WDdVzGqA~OvCkch5xki-4Fi-cou7bQwyJDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199316&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The core algorithm of PTEMD. (A) PTEMD-A, <em>de novo</em> active TE scanning pipeline. The high-throughput short reads are first mapped to the reference genome and the genomic regions correspond to mapping gaps (the dotted line indicated the gaps) are identified and classified into different repeat families with neighbour-join clustering. (B) PTEMD-B, <em>de novo</em> TIPs detecting pipeline. The middle panel represents the high-throughput reads which partially mapped to the TE sequence, the un-matched part of the reads were re-located to the reference genome sequences. There are two major mapping patterns: (i) the interval distance on the reference genome between left and right mapped reads is near or equal to 0 (top panel of B), representing a TIP in the re-sequenced individual; (ii) the interval distance between left and right mapped reads is equal or almost equal to the length of the TE (bottom panel of B), representing a same TE distribution between the reference and the re-sequenced individual. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01101.png?Expires=2147483647&Signature=f6gOnNcchEmcdxwjaGHWGhZ6X43SazZ0yNAoM8wpdACI7y44wGx0UQEvTHOsLZw4Nh48KBJDTS077aWqvhnZ5r~zt4hjllVx7GM9xT2ygqu4XsdBGLUVvdu6RAFhRnhNhb7ZOu-lSXcHT6paQzRVl3zYqrFEUTBKni~YZUH3HOOOUBmH1S-7Z8LXPwpzIKMuh1M2q814BqnrJ8fMLPlsQGThZ6wz9yJOjQvpPruJ6p4MqhWvOVvLPZtuLNRZ91A7GwOp2SUu48R~h2cRyfUqfvo-JtBf8RA~x05aRrTF3S0KerdL1oRobRBHsUkpTVribra1AKrrYaR9yGdGtuFz2g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The core algorithm of PTEMD. (A) PTEMD-A, de novo active TE scanning pipeline. The high-throughput short reads are first mapped to the reference genome and the genomic regions correspond to mapping gaps (the dotted line indicated the gaps) are identified and classified into different repeat families with neighbour-join clustering. (B) PTEMD-B, de novo TIPs detecting pipeline. The middle panel represents the high-throughput reads which partially mapped to the TE sequence, the un-matched part of the reads were re-located to the reference genome sequences. There are two major mapping patterns: (i) the interval distance on the reference genome between left and right mapped reads is near or equal to 0 (top panel of B), representing a TIP in the re-sequenced individual; (ii) the interval distance between left and right mapped reads is equal or almost equal to the length of the TE (bottom panel of B), representing a same TE distribution between the reference and the re-sequenced individual. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01101.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199316/dsw01101.tiff" path-from-xml="dsw01101.tiff" target="_blank">View large</a><a section="87199316" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01101.png?Expires=2147483647&Signature=UIMY2rV~pzo9A8mEtNfsGTd1tOW03V~7TbX9xfk5f6Ts64yUmGTUc5OixCallcM6VwALVSodScNqOH~GX7v8oXpyy9wOJ79rSgkLLqmBmr1RztIBtZXSJolJaO4bjXaOvrS~0l-PUT7~jeRmmUZRH-oluz6SF5It6M7ABX3~2L9mkehhPIZhLCghby8pT-VVdZjwbiP1wxgiCxtaFURHd4w7oV6M5Xk3TLxpw86VKWIYdWBeqPM5B0d6AXqMx2sPiwKH~hShabENrDr6j7phTdjrtJn2qh5cw3XSI0Zs~EB8fB3WOy6WDdVzGqA~OvCkch5xki-4Fi-cou7bQwyJDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199316&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01101.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The core algorithm of PTEMD. (A) PTEMD-A, <em>de novo</em> active TE scanning pipeline. The high-throughput short reads are first mapped to the reference genome and the genomic regions correspond to mapping gaps (the dotted line indicated the gaps) are identified and classified into different repeat families with neighbour-join clustering. (B) PTEMD-B, <em>de novo</em> TIPs detecting pipeline. The middle panel represents the high-throughput reads which partially mapped to the TE sequence, the un-matched part of the reads were re-located to the reference genome sequences. There are two major mapping patterns: (i) the interval distance on the reference genome between left and right mapped reads is near or equal to 0 (top panel of B), representing a TIP in the re-sequenced individual; (ii) the interval distance between left and right mapped reads is equal or almost equal to the length of the TE (bottom panel of B), representing a same TE distribution between the reference and the re-sequenced individual. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To determine the exact boundary of the classified TE sequences, we first align each classified sequence cluster to a clustalw<sup><a reveal-id="DSW011C27" data-open="DSW011C27" class="link link-ref link-reveal xref-bibr">27</a></sup> format file using MUSCLE<sup><a reveal-id="DSW011C28" data-open="DSW011C28" class="link link-ref link-reveal xref-bibr">28</a></sup>; then we use a stringent criteria (at both left and right sides of the sequence cluster, the terminal base has at least 50% consistent within the sequence cluster) to get the longest consensus sequence of the cluster (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>D). We then annotated the sequences by aligning them with NT/NR, Repbase, and maize TE database (<a class="link link-uri" href="http://maizetedb.org/~maize/">http://maizetedb.org/~maize/</a>), and manually classified them based on their structure, terminal sequences, and target site duplications (TSDs). Those classified sequences are the PTEMD-A-identified polymorphic TE families.</p><p>The objective of PTEMD-B is to identify the TIPs. PTEMD-B aligns the high-throughput reads to the TE sequence (Fig. <a reveal-id="DSW011F1" data-open="DSW011F1" class="link link-reveal link-table xref-fig">1</a>B, middle, detail parameter refer to BWA program MEM algorithm),<sup><a reveal-id="DSW011C29" data-open="DSW011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> the reads which partially mapped to the terminus of the TE sequence are selected from the re-sequencing data set, the un-mapped part of those sequences (Fig. <a reveal-id="DSW011F1" data-open="DSW011F1" class="link link-reveal link-table xref-fig">1</a>B, highlighted with blue and green) are isolated and re-mapped to the reference genome sequences. There are two major mapping patterns; TE insertion only in the re-sequenced genome, and TE in the same location compared with the reference genome (Fig. <a reveal-id="DSW011F1" data-open="DSW011F1" class="link link-reveal link-table xref-fig">1</a>B). For the detection of TIPs, it requires both sides of the flanking sequences mapped to the same location in the genome that lacks the insertion. As a result, the detected TIPs include only polymorphism created through indels including transpositions. The TIPs do not include sequence rearrangements caused by recombination, inversion, and translocations. In addition, due to the short length of the reads, it is impossible for the program to distinguish whether the TIP is caused by a full-length TE or a TE with internal deletions. In summary, by using the two-step approach, PTEMD is able to quickly scan polymorphic TE families and their insertion landscapes from high-throughput short reads. The output format of PTEMD-B is shown in Fig. <a reveal-id="DSW011F2" data-open="DSW011F2" class="link link-reveal link-table xref-fig">2</a>, the file contains the detail insertion site/s, TSD, and all of the reads situated at ‘breakpoints’. </p>                        <a id="87199319" scrollto-destination="87199319"></a>
<div id="DSW011F2" data-id="DSW011F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01102.png?Expires=2147483647&Signature=CgE1mWceyPEUEbOI0Vk0LY3XtPSQI~80kg9BTXLd3QbhfPs8Ha5jdtL3Q8tCPuhkvCk2iknXSjQEJeR2ttKHGbJqVzZCWbWel-CWh9CIDYhWauLSR12Y5wwlGwCVxc9Zwt0fsRF24sdgY0Zw0NSEKDjB7jY3suCeEvXAex3eMJysIDHdzQJ1I1EF5bfFMX0WaiiOE7d24Ncsf~4fNh1S1Wl5qBXkCmK9nHV5EFpRzL9PCKfG8xX~TO-MKDPFtkbcMTMDGIHVxXJ3yoirWr20Fqqg8CFTG44LvEopYzyUkcUHcoGor3Y3ogcx6SvAN8-tVpI1IgmvdefaZ8aUPhmxVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Output format of PTEMD-B. (A) TE insertion without TSD. (B) TE insertion with a TSD ‘GAAA’. All of the reads information at the breakpoint region will be included in the PTEMD-B output file." path-from-xml="dsw01102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199319/dsw01102.tiff" path-from-xml="dsw01102.tiff" target="_blank">View large</a><a section="87199319" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01102.png?Expires=2147483647&Signature=3zGCaxIh0kF0mSYDpSpoHVD-3CEz5Ew7-JTWrcTOAZ-3JHg5UgE6d1BoXXtde7sMIY~91oN4PzsWFc9V4PVDaM1ie-nkUkLzWoqSIFAfEc5KLuVerVsTAbQtS6ehQ5ExmGsxOCXnYCJqYgA7VlkVbKozjbNryP0dpLigBcdCSt1RukvTNBZKU8Y-3k4po9dRO8piD9nwx~n5A1dmFp3xRyR0-C4u95ZxR5NB6a-nxySkukxjmD5Rur1IqbCvxEwJXLHfu-eoOkJ1pdOx97IanS9GYph0QYvQioy4nwzwKKgGBnEp-IMfncUPGxNru2bzL~kDLDuBx3N0DYgF1diSYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199319&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Output format of PTEMD-B. (A) TE insertion without TSD. (B) TE insertion with a TSD ‘GAAA’. All of the reads information at the breakpoint region will be included in the PTEMD-B output file.</p></div></div><div content-id="DSW011F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01102.png?Expires=2147483647&Signature=CgE1mWceyPEUEbOI0Vk0LY3XtPSQI~80kg9BTXLd3QbhfPs8Ha5jdtL3Q8tCPuhkvCk2iknXSjQEJeR2ttKHGbJqVzZCWbWel-CWh9CIDYhWauLSR12Y5wwlGwCVxc9Zwt0fsRF24sdgY0Zw0NSEKDjB7jY3suCeEvXAex3eMJysIDHdzQJ1I1EF5bfFMX0WaiiOE7d24Ncsf~4fNh1S1Wl5qBXkCmK9nHV5EFpRzL9PCKfG8xX~TO-MKDPFtkbcMTMDGIHVxXJ3yoirWr20Fqqg8CFTG44LvEopYzyUkcUHcoGor3Y3ogcx6SvAN8-tVpI1IgmvdefaZ8aUPhmxVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Output format of PTEMD-B. (A) TE insertion without TSD. (B) TE insertion with a TSD ‘GAAA’. All of the reads information at the breakpoint region will be included in the PTEMD-B output file." path-from-xml="dsw01102.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199319/dsw01102.tiff" path-from-xml="dsw01102.tiff" target="_blank">View large</a><a section="87199319" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01102.png?Expires=2147483647&Signature=3zGCaxIh0kF0mSYDpSpoHVD-3CEz5Ew7-JTWrcTOAZ-3JHg5UgE6d1BoXXtde7sMIY~91oN4PzsWFc9V4PVDaM1ie-nkUkLzWoqSIFAfEc5KLuVerVsTAbQtS6ehQ5ExmGsxOCXnYCJqYgA7VlkVbKozjbNryP0dpLigBcdCSt1RukvTNBZKU8Y-3k4po9dRO8piD9nwx~n5A1dmFp3xRyR0-C4u95ZxR5NB6a-nxySkukxjmD5Rur1IqbCvxEwJXLHfu-eoOkJ1pdOx97IanS9GYph0QYvQioy4nwzwKKgGBnEp-IMfncUPGxNru2bzL~kDLDuBx3N0DYgF1diSYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199319&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01102.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Output format of PTEMD-B. (A) TE insertion without TSD. (B) TE insertion with a TSD ‘GAAA’. All of the reads information at the breakpoint region will be included in the PTEMD-B output file.</p></div></div><p>To evaluate the impact of sequencing coverage on PTEMD performance, HM-1 and HM-2 genome sequence data were randomly split into data sets with ×1, ×5, ×10, ×20, ×30, ×40, and ×50 coverage. These seven data sets were then analysed by the PTEMD, and the results were used to evaluate the performance of PTEMD on the depth of re-sequencing data sets. A polymorphic TE family was defined as the presence of at least three TIPs in the reference genome compared with the re-sequenced data sets, with ≥97% sequence identity over 95% length. As expected, the number of TE families detected was positively correlated with sequencing depth (<em>r</em>= 0.98 ± 0.01, <em>P</em> = 0.00015 ± 0.00005, Fig. <a reveal-id="DSW011F3" data-open="DSW011F3" class="link link-reveal link-table xref-fig">3</a>A). Although the number of polymorphic TE families continued to increase as the coverage approached ×50, the slope decreased, and at ×50 coverage, 15 TE families were identified in the <em>M. oryzae</em> genome. Among the 15 TE families, 14 represent intact TEs and 1 is a TE fragment (see next section and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> for details). The total copy number of those 14 intact TE families in rice blast reference genome is 905. </p>                        <a id="87199321" scrollto-destination="87199321"></a>
<div id="DSW011F3" data-id="DSW011F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01103.png?Expires=2147483647&Signature=otSNaRACm5-mJmD5LI6NofhED-XSshNaXq4FBrnNViCJmYTSxZgr~ux5HMSffnjryHUNqG5YrAVB7COWGrQinos9ViXJ2W7fDrdRYzCUwaktgLFP541Gh1EWG2sWytVWYx8rCsjXXtHzpD2XlspnshYHN2aFuZXUIN105alDoqSXIxCQ3fXfrORhckKHFX6kd-EqDTW4~GsYEFZpj6qgsfPZhvLvvyO77IJxSaO3K9wWfxMC4n14-YMP9luurp~x6xV7EUWtE7VQMo0~pwNjzLCmOP0bAqeJn84aeV0TaTK92tAApZfXxo0yh-zzCo~xVI3rcR0bHEUk39h8inctKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The PTEMD performance on polymorphic TE scanning and TIPs detection under different depths of re-sequencing data sets in M. oryzae. Y-axis represents the number of the TE families detected with the vertical line indicates standard deviation (A) and represents the number of detected TIPs with the vertical line indicates standard deviation (B–E). (A) PTEMD-A-detected TE families (HM-1 and HM-2 are two strains). X-axis: reads coverage of M. oryzae genome. (B) PTEMD-B-detected TIPs in HM-1 strain for MoTE-1–3. (C) PTEMD-B-detected TIPs in HM-1 strain for MoTE-4 to 6. (D) PTEMD-B-detected TIPs in HM-2 strain for MoTE-1–3. (E) PTEMD-B-detected TIPs in HM-2 strain for MoTE-4–6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199321/dsw01103.tiff" path-from-xml="dsw01103.tiff" target="_blank">View large</a><a section="87199321" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01103.png?Expires=2147483647&Signature=MHjbh4h0qI49f1bY-ACm6MHDHjnBphsMLG9SxdTsnkBVXjD3nW-b0zTrUOdKAel0nlmnTyhp9jJTV755c0O0KaHrprYSnA~yMeGkaV6g5w9i298hcdg8cRl1fG5nDQGphilAfnwY5iDenZH6NyQG9jcZlBqFMTK1RsgzJV6bbTx3jJvQZDjjw31Zo78GLr31wpdabvp3PJDJuM7V9ERvXTyTHUYjk7vQmcppymyeh~ui~hi9-dB5bWMdAtRTUkuOb9tDdNB4JMYzj4N3GVh-pvs86XD0M5zITumKfO2M0CtWQTLiFvv9GX9IGS~-mU6zDGO~Vznj1IySeFmq3F5hgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199321&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The PTEMD performance on polymorphic TE scanning and TIPs detection under different depths of re-sequencing data sets in <em>M. oryzae. Y</em>-axis represents the number of the TE families detected with the vertical line indicates standard deviation (A) and represents the number of detected TIPs with the vertical line indicates standard deviation (B–E). (A) PTEMD-A-detected TE families (HM-1 and HM-2 are two strains). <em>X</em>-axis: reads coverage of <em>M. oryzae</em> genome. (B) PTEMD-B-detected TIPs in HM-1 strain for MoTE-1–3. (C) PTEMD-B-detected TIPs in HM-1 strain for MoTE-4 to 6. (D) PTEMD-B-detected TIPs in HM-2 strain for MoTE-1–3. (E) PTEMD-B-detected TIPs in HM-2 strain for MoTE-4–6. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01103.png?Expires=2147483647&Signature=otSNaRACm5-mJmD5LI6NofhED-XSshNaXq4FBrnNViCJmYTSxZgr~ux5HMSffnjryHUNqG5YrAVB7COWGrQinos9ViXJ2W7fDrdRYzCUwaktgLFP541Gh1EWG2sWytVWYx8rCsjXXtHzpD2XlspnshYHN2aFuZXUIN105alDoqSXIxCQ3fXfrORhckKHFX6kd-EqDTW4~GsYEFZpj6qgsfPZhvLvvyO77IJxSaO3K9wWfxMC4n14-YMP9luurp~x6xV7EUWtE7VQMo0~pwNjzLCmOP0bAqeJn84aeV0TaTK92tAApZfXxo0yh-zzCo~xVI3rcR0bHEUk39h8inctKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The PTEMD performance on polymorphic TE scanning and TIPs detection under different depths of re-sequencing data sets in M. oryzae. Y-axis represents the number of the TE families detected with the vertical line indicates standard deviation (A) and represents the number of detected TIPs with the vertical line indicates standard deviation (B–E). (A) PTEMD-A-detected TE families (HM-1 and HM-2 are two strains). X-axis: reads coverage of M. oryzae genome. (B) PTEMD-B-detected TIPs in HM-1 strain for MoTE-1–3. (C) PTEMD-B-detected TIPs in HM-1 strain for MoTE-4 to 6. (D) PTEMD-B-detected TIPs in HM-2 strain for MoTE-1–3. (E) PTEMD-B-detected TIPs in HM-2 strain for MoTE-4–6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01103.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199321/dsw01103.tiff" path-from-xml="dsw01103.tiff" target="_blank">View large</a><a section="87199321" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01103.png?Expires=2147483647&Signature=MHjbh4h0qI49f1bY-ACm6MHDHjnBphsMLG9SxdTsnkBVXjD3nW-b0zTrUOdKAel0nlmnTyhp9jJTV755c0O0KaHrprYSnA~yMeGkaV6g5w9i298hcdg8cRl1fG5nDQGphilAfnwY5iDenZH6NyQG9jcZlBqFMTK1RsgzJV6bbTx3jJvQZDjjw31Zo78GLr31wpdabvp3PJDJuM7V9ERvXTyTHUYjk7vQmcppymyeh~ui~hi9-dB5bWMdAtRTUkuOb9tDdNB4JMYzj4N3GVh-pvs86XD0M5zITumKfO2M0CtWQTLiFvv9GX9IGS~-mU6zDGO~Vznj1IySeFmq3F5hgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199321&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01103.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The PTEMD performance on polymorphic TE scanning and TIPs detection under different depths of re-sequencing data sets in <em>M. oryzae. Y</em>-axis represents the number of the TE families detected with the vertical line indicates standard deviation (A) and represents the number of detected TIPs with the vertical line indicates standard deviation (B–E). (A) PTEMD-A-detected TE families (HM-1 and HM-2 are two strains). <em>X</em>-axis: reads coverage of <em>M. oryzae</em> genome. (B) PTEMD-B-detected TIPs in HM-1 strain for MoTE-1–3. (C) PTEMD-B-detected TIPs in HM-1 strain for MoTE-4 to 6. (D) PTEMD-B-detected TIPs in HM-2 strain for MoTE-1–3. (E) PTEMD-B-detected TIPs in HM-2 strain for MoTE-4–6. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The average sequence identity between the genomes of HM-1 and HM-2 and that of reference genome was 99.97 ± 0.01%. It is estimated that some of the PETMD-A detected polymorphic TEs may be still active or recently active. To test this, we analysed a genetically purified, single-isolated data sets.<sup><a reveal-id="DSW011C30" data-open="DSW011C30" class="link link-ref link-reveal xref-bibr">30</a></sup> Using PTEMD, we detected direct breakpoint reads evidences for MoTE-1, -2, -3, -4, -5, -6, -7, and -11 within 10–20 generations, indicated that more than half of the PTEMD-detected polymorphic TE families may still be active.</p><p>To determine the false-positive rate of PTEMD-B, we used the 14 intact TE families, and identified TIPs and evaluated the false-positive of the TIPs. A TIP event was defined as a TE present in HM-1/HM-2 and absent in the rice blast reference genome.<sup><a reveal-id="DSW011C31" data-open="DSW011C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Using the same data sets as in Fig. <a reveal-id="DSW011F3" data-open="DSW011F3" class="link link-reveal link-table xref-fig">3</a>A, we found that the number of TIPs detected was positively correlated with sequencing coverage (MoTE-1: <em>r</em>= 0.94 ± 0.05, <em>P</em> = 0.0014 ± 0.0018; MoTE-2: <em>r</em>= 0.95 ± 0.01, <em>P</em> = 0.0003 ± 0.0002; MoTE-3: <em>r</em>= 0.95 ± 0.007, <em>P</em> = 0.00045 ± 0.0002; MoTE-4: <em>r</em>= 0.73 ± 0.03, <em>P</em> = 0.04 ± 0.01; MoTE-5: <em>r</em>= 0.75 ± 0.03, <em>P</em> = 0.04 ± 0.007; MoTE-6: <em>r</em>= 0.90 ± 0.06, <em>P</em> = 0.004 ± 0.004). For example, the number of detected TIPs with MoTE-1 increased monotonically when the coverage increased but became saturated at about ×30 to ×50 in both HM-1 (Fig. <a reveal-id="DSW011F3" data-open="DSW011F3" class="link link-reveal link-table xref-fig">3</a>B) and HM-2 (Fig. <a reveal-id="DSW011F3" data-open="DSW011F3" class="link link-reveal link-table xref-fig">3</a>D). When the coverage increases beyond ×30, the number of PTEMD-detected TIPs still increases but at a declining rate. The number of detected TIPs for other TE families demonstrates the same tendency as MoTE-1 (Fig. <a reveal-id="DSW011F3" data-open="DSW011F3" class="link link-reveal link-table xref-fig">3</a>B–E).</p><p>To test whether an identified TIPs was authentic, we determined the false-positive rate of the predicted TE insertions by amplifying the expected fragments in the HM-1 or HM-2 genome with PCR. For experimental validation, we selected the MoTE-1 family because the insertion polymorphic rates of MoTE-1 for all three isolates (HM-1, HM-2, and the reference genome) were 100% (Fig. <a reveal-id="DSW011F4" data-open="DSW011F4" class="link link-reveal link-table xref-fig">4</a>A, all of the MoTE-1 sequences are located in different regions). Among the TIPs involving the MoTE-1 family and detected by PTEMD-B, 60 TIPs from HM-1 and HM-2 were randomly selected for validation (Fig. <a reveal-id="DSW011F4" data-open="DSW011F4" class="link link-reveal link-table xref-fig">4</a>A, the blue and red triangles represent the selected TIPs). If the predicted TIP was authentic, the difference in PCR product size between a fungal isolate with the TE insertion and a isolate without the TE insertion would be the length of MoTE-1, which is ∼1.8 kb. Using <em>M. oryzae</em> reference genome sequences, we designed 60 primer pairs flanking the predicted insertion sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The results showed that 100% of MoTE-1 TIPs in HM-1 and 93% of those in HM-2 represented true insertion polymorphism (Fig. <a reveal-id="DSW011F4" data-open="DSW011F4" class="link link-reveal link-table xref-fig">4</a>B and C). To assess why two of the predicted TIPs in HM-2 (Fig. <a reveal-id="DSW011F4" data-open="DSW011F4" class="link link-reveal link-table xref-fig">4</a>C, black arrows regions, 2-P6 and 2-P13) were not detected by PCR, we examined the sequence coverage for these two predicted TIPs and found that both had fewer supporting reads (2 and 3 breakpoint reads supported) than the validated insertions (≥4 breakpoint reads supported, <em>P</em>&lt; 0.01). Our finding indicated that the false-positive rate of the PTEMD-B is quite low at ∼3.3% for the MoTE-1 family. </p>                        <a id="87199325" scrollto-destination="87199325"></a>
<div id="DSW011F4" data-id="DSW011F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01104.png?Expires=2147483647&Signature=R0txfq~mOfnoMHnncKGeKf~KpAiepLgJ1rVzgRmNOQjq4F2jKm0pM6ic0rgC7zt2O~skJEtrEKedply5qJb3-AM8NrO12obY6zzkWE84JdRhdNDWspH1OphU5-wpKCAFyHWBQWnHYPz-U5-P4EMNJA6kHAT4j2d~pHbOpH7v1UzaEFUct0jGBlBYXcmJrh8rWP5mVCE18eEieYpgOL6xfgdD~xijTDPUsLel63AfTDbS9yk7fcBbR1BFHOS~8OwIqEQg0VqXtACyPRbZQbOl2rCJpGv76V-2bdo1z6nYJfb0RU~6Je6yVJfjmquhsl-RS65xkpIFHGsgtqQ7Yhmm7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TIPs of MoTE-1 among M. orzyae strains of reference genome, HM-1 and HM-2 and their validation. (A) Distributions of the MoTE-1 TIPs in the M. oryzae reference genome, HM-1 and HM-2 genomes. Black lines: M. oryzae chromosomes (X represents sequence that are not located in the seven chromosomes). Black arrows: location of the MoTE-1 TIPs in the reference genome. Gray and light gray arrows: location of MoTE-1 TIPs in HM-1 and HM-2 strains, respectively. All of the black, blue, and red arrows were located in different positions, indicated the polymorphism distribution of MoTE-1 in different strains. (B and C) PCR validating of the randomly selected 60 PTEMD-B-detected MoTE-1 TIPs [blue and red triangles in (A) represent the selected TIPs]; 30 pairs of primers complementary to flanking sequence of TIPs (B, from 1-P1 to 1-P30) for HM-1 vs. reference genome and 30 primer pairs (C, from 2-P1 to 2-P30) for HM-2 vs. reference genome were used in the testing. The length of MoTE-1 is 1.86 kb, and the length of the 60 PCR productions in reference genome varies from 100 to 300 bp (B and C, bottom panels; top panels of B and C represent the PCR production size in HM-1 and HM-2 isolates). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199325/dsw01104.tiff" path-from-xml="dsw01104.tiff" target="_blank">View large</a><a section="87199325" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01104.png?Expires=2147483647&Signature=469J16VD6R~Tg9Xxtq59CC16Nv51TpvzQSh7aukFI4De8mynas~yby8QXkdbjq~7ow9YIbtuEc6Z3fBW~3UsAvqeILR8~oLn9ma1XXu~0lkxfhc-tq5qSaZirOq7ZLN44BBsDMIxINsM~1DRLwMyD2GC52o7KvzF~8suoAn01NYjfdtU8BtiklJqHp3O6mxy8sL38nsb7Yb-TeBPjmbU0jTYnZE4ObrnOQM-U~hn4QTUhkwS-lUQJq8ogQEVniImOv7b8VvXO1-83wYUUi6GpsfiRiw~YO~Rq6ojtR4D-FPuKIVZR12jBggzW8qpCuCXoOHQCX7CckNURega6trc7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199325&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TIPs of MoTE-1 among <em>M. orzyae</em> strains of reference genome, HM-1 and HM-2 and their validation. (A) Distributions of the MoTE-1 TIPs in the <em>M. oryzae</em> reference genome, HM-1 and HM-2 genomes. Black lines: <em>M. oryzae</em> chromosomes (X represents sequence that are not located in the seven chromosomes). Black arrows: location of the MoTE-1 TIPs in the reference genome. Gray and light gray arrows: location of MoTE-1 TIPs in HM-1 and HM-2 strains, respectively. All of the black, blue, and red arrows were located in different positions, indicated the polymorphism distribution of MoTE-1 in different strains. (B and C) PCR validating of the randomly selected 60 PTEMD-B-detected MoTE-1 TIPs [blue and red triangles in (A) represent the selected TIPs]; 30 pairs of primers complementary to flanking sequence of TIPs (B, from 1-P1 to 1-P30) for HM-1 vs. reference genome and 30 primer pairs (C, from 2-P1 to 2-P30) for HM-2 vs. reference genome were used in the testing. The length of MoTE-1 is 1.86 kb, and the length of the 60 PCR productions in reference genome varies from 100 to 300 bp (B and C, bottom panels; top panels of B and C represent the PCR production size in HM-1 and HM-2 isolates). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01104.png?Expires=2147483647&Signature=R0txfq~mOfnoMHnncKGeKf~KpAiepLgJ1rVzgRmNOQjq4F2jKm0pM6ic0rgC7zt2O~skJEtrEKedply5qJb3-AM8NrO12obY6zzkWE84JdRhdNDWspH1OphU5-wpKCAFyHWBQWnHYPz-U5-P4EMNJA6kHAT4j2d~pHbOpH7v1UzaEFUct0jGBlBYXcmJrh8rWP5mVCE18eEieYpgOL6xfgdD~xijTDPUsLel63AfTDbS9yk7fcBbR1BFHOS~8OwIqEQg0VqXtACyPRbZQbOl2rCJpGv76V-2bdo1z6nYJfb0RU~6Je6yVJfjmquhsl-RS65xkpIFHGsgtqQ7Yhmm7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="TIPs of MoTE-1 among M. orzyae strains of reference genome, HM-1 and HM-2 and their validation. (A) Distributions of the MoTE-1 TIPs in the M. oryzae reference genome, HM-1 and HM-2 genomes. Black lines: M. oryzae chromosomes (X represents sequence that are not located in the seven chromosomes). Black arrows: location of the MoTE-1 TIPs in the reference genome. Gray and light gray arrows: location of MoTE-1 TIPs in HM-1 and HM-2 strains, respectively. All of the black, blue, and red arrows were located in different positions, indicated the polymorphism distribution of MoTE-1 in different strains. (B and C) PCR validating of the randomly selected 60 PTEMD-B-detected MoTE-1 TIPs [blue and red triangles in (A) represent the selected TIPs]; 30 pairs of primers complementary to flanking sequence of TIPs (B, from 1-P1 to 1-P30) for HM-1 vs. reference genome and 30 primer pairs (C, from 2-P1 to 2-P30) for HM-2 vs. reference genome were used in the testing. The length of MoTE-1 is 1.86 kb, and the length of the 60 PCR productions in reference genome varies from 100 to 300 bp (B and C, bottom panels; top panels of B and C represent the PCR production size in HM-1 and HM-2 isolates). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01104.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199325/dsw01104.tiff" path-from-xml="dsw01104.tiff" target="_blank">View large</a><a section="87199325" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01104.png?Expires=2147483647&Signature=469J16VD6R~Tg9Xxtq59CC16Nv51TpvzQSh7aukFI4De8mynas~yby8QXkdbjq~7ow9YIbtuEc6Z3fBW~3UsAvqeILR8~oLn9ma1XXu~0lkxfhc-tq5qSaZirOq7ZLN44BBsDMIxINsM~1DRLwMyD2GC52o7KvzF~8suoAn01NYjfdtU8BtiklJqHp3O6mxy8sL38nsb7Yb-TeBPjmbU0jTYnZE4ObrnOQM-U~hn4QTUhkwS-lUQJq8ogQEVniImOv7b8VvXO1-83wYUUi6GpsfiRiw~YO~Rq6ojtR4D-FPuKIVZR12jBggzW8qpCuCXoOHQCX7CckNURega6trc7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199325&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01104.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>TIPs of MoTE-1 among <em>M. orzyae</em> strains of reference genome, HM-1 and HM-2 and their validation. (A) Distributions of the MoTE-1 TIPs in the <em>M. oryzae</em> reference genome, HM-1 and HM-2 genomes. Black lines: <em>M. oryzae</em> chromosomes (X represents sequence that are not located in the seven chromosomes). Black arrows: location of the MoTE-1 TIPs in the reference genome. Gray and light gray arrows: location of MoTE-1 TIPs in HM-1 and HM-2 strains, respectively. All of the black, blue, and red arrows were located in different positions, indicated the polymorphism distribution of MoTE-1 in different strains. (B and C) PCR validating of the randomly selected 60 PTEMD-B-detected MoTE-1 TIPs [blue and red triangles in (A) represent the selected TIPs]; 30 pairs of primers complementary to flanking sequence of TIPs (B, from 1-P1 to 1-P30) for HM-1 vs. reference genome and 30 primer pairs (C, from 2-P1 to 2-P30) for HM-2 vs. reference genome were used in the testing. The length of MoTE-1 is 1.86 kb, and the length of the 60 PCR productions in reference genome varies from 100 to 300 bp (B and C, bottom panels; top panels of B and C represent the PCR production size in HM-1 and HM-2 isolates). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199326 id="87199326" class="section-title" data-legacy-id=s3b>3.2. Comparison of PTEMD with other relative programs</h3>
<p>To assess the proportion of polymorphic TE families in genome, we <em>de novo</em> detected all of the repeat sequences families in <em>M. oryzae</em> genome using RepeatMasker, Repeatscout, Piler, and RECON; 36, 79, 42, and 44 repeat families were identified, respectively (Fig. <a reveal-id="DSW011F5" data-open="DSW011F5" class="link link-reveal link-table xref-fig">5</a>A, Venn diagram of the four program detected repeat sequence families), about half of the families are redundant, meanwhile, total non-redundant repeat families are 138 (Fig. <a reveal-id="DSW011F5" data-open="DSW011F5" class="link link-reveal link-table xref-fig">5</a>B, the blue and blue-red overlapped region). Using PTEMD-A, we detected 14 polymorphic TE families. Among the 14 TE families, 12 families have highly similar (97% identity over 95% length) homologue within the 138 non-redundant repeat families, the other two families are only detected by PTEMD-A (Fig. <a reveal-id="DSW011F5" data-open="DSW011F5" class="link link-reveal link-table xref-fig">5</a>B). In other words, only ∼10% of the repeat families in <em>M. oryzae</em> genome are polymorphic TE families. The two polymorphic TE families that are only detected by PTEMD-A have low copy numbers (ranging from three to seven) in <em>M. oryzae</em> genome, and this explained why they are missed by other repeat scanning programs. </p>                        <a id="87199328" scrollto-destination="87199328"></a>
<div id="DSW011F5" data-id="DSW011F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01105.png?Expires=2147483647&Signature=a4~6GoFMcSXaZbdFQ4t59Z2OAgSM05GzmKu9ES3ipTT676z-hbk1JSyG5tfL7WHka7S8imvlw4HmMAvT51wK1pwvKDrJhQDk9B9DoSI9QUh9eThhmcx5c88M7R76tvqy7~mjXqYNn8az5TjfR~N8T5bJ97VLOPZYmVPjG6qjBhsshLaYPmAxp3DCNYhZnUVcMc1mHG-3poVFJ20brMMWlvB5qAOFoF3WKn6c~iXsg0wHtDWOJhYVdW41WcgYGgwEw24TzRuvPDO8VYxn0qcB~Wm~XLNJUAzhMWJl-rLOru6jWbuYJ9PqNw-Yrd~N4-ieG53qtSw~Vm8-gjziKCKUcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the comparing of PTEMD-detected TEs and TIPs with related programs. (A) Repeat sequence families independently detected by four programs of Repeatscout, Piler, RepeatMasker, and RECON; a total number of 138 non-redundant repeat families have been identified in M. oryzae genome. (B) Comparing the 14 PTEMD-A-detected candidate active TEs with the 138 non-redundant repeat families, 12 families are overlapped. (C) Comparing the TIPs detected by PTEMD-B, TEMP, RelocaTE, and T-lex. The total non-redundant TIPs detected by all of the four tools is 301, and 217 of them are co-detected by all of the four tools. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199328/dsw01105.tiff" path-from-xml="dsw01105.tiff" target="_blank">View large</a><a section="87199328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01105.png?Expires=2147483647&Signature=nKIcqU0VVBW7v6zSVd~3wfhvdYXSBdcryCkwPpOp6lFpjlnZcQ5LL-E2GSb2ntqNUSNNAUNdG40XofgKvk13HFX0pr7ZuMYiLSRhk7uLtwSzMVJbLRQaEPkOzH0FkeE1G2wj8HGAc7ehiqR2iU3JBZCI7VE6Ebb1l~mSf1zoO4n1SNxqMvq5xacSy9DiVYW0yygHPrh3vjyjCkj3fEB4BwBDz~YgK~ZOvQ1XNGFB8-08FP8-iQgsPczZKq1tunDTHAfN3znBX4UbG3TFA2D8ZPC64kkA7-O4rLfkKRHHWW-INpDRjQwdnAXm5iUPs1ZgPPM1~fwY-4Jh6e~FPAjgWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199328&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the comparing of PTEMD-detected TEs and TIPs with related programs. (A) Repeat sequence families independently detected by four programs of Repeatscout, Piler, RepeatMasker, and RECON; a total number of 138 non-redundant repeat families have been identified in <em>M. oryzae</em> genome. (B) Comparing the 14 PTEMD-A-detected candidate active TEs with the 138 non-redundant repeat families, 12 families are overlapped. (C) Comparing the TIPs detected by PTEMD-B, TEMP, RelocaTE, and T-lex. The total non-redundant TIPs detected by all of the four tools is 301, and 217 of them are co-detected by all of the four tools. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01105.png?Expires=2147483647&Signature=a4~6GoFMcSXaZbdFQ4t59Z2OAgSM05GzmKu9ES3ipTT676z-hbk1JSyG5tfL7WHka7S8imvlw4HmMAvT51wK1pwvKDrJhQDk9B9DoSI9QUh9eThhmcx5c88M7R76tvqy7~mjXqYNn8az5TjfR~N8T5bJ97VLOPZYmVPjG6qjBhsshLaYPmAxp3DCNYhZnUVcMc1mHG-3poVFJ20brMMWlvB5qAOFoF3WKn6c~iXsg0wHtDWOJhYVdW41WcgYGgwEw24TzRuvPDO8VYxn0qcB~Wm~XLNJUAzhMWJl-rLOru6jWbuYJ9PqNw-Yrd~N4-ieG53qtSw~Vm8-gjziKCKUcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagram showing the comparing of PTEMD-detected TEs and TIPs with related programs. (A) Repeat sequence families independently detected by four programs of Repeatscout, Piler, RepeatMasker, and RECON; a total number of 138 non-redundant repeat families have been identified in M. oryzae genome. (B) Comparing the 14 PTEMD-A-detected candidate active TEs with the 138 non-redundant repeat families, 12 families are overlapped. (C) Comparing the TIPs detected by PTEMD-B, TEMP, RelocaTE, and T-lex. The total non-redundant TIPs detected by all of the four tools is 301, and 217 of them are co-detected by all of the four tools. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01105.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199328/dsw01105.tiff" path-from-xml="dsw01105.tiff" target="_blank">View large</a><a section="87199328" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01105.png?Expires=2147483647&Signature=nKIcqU0VVBW7v6zSVd~3wfhvdYXSBdcryCkwPpOp6lFpjlnZcQ5LL-E2GSb2ntqNUSNNAUNdG40XofgKvk13HFX0pr7ZuMYiLSRhk7uLtwSzMVJbLRQaEPkOzH0FkeE1G2wj8HGAc7ehiqR2iU3JBZCI7VE6Ebb1l~mSf1zoO4n1SNxqMvq5xacSy9DiVYW0yygHPrh3vjyjCkj3fEB4BwBDz~YgK~ZOvQ1XNGFB8-08FP8-iQgsPczZKq1tunDTHAfN3znBX4UbG3TFA2D8ZPC64kkA7-O4rLfkKRHHWW-INpDRjQwdnAXm5iUPs1ZgPPM1~fwY-4Jh6e~FPAjgWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199328&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01105.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagram showing the comparing of PTEMD-detected TEs and TIPs with related programs. (A) Repeat sequence families independently detected by four programs of Repeatscout, Piler, RepeatMasker, and RECON; a total number of 138 non-redundant repeat families have been identified in <em>M. oryzae</em> genome. (B) Comparing the 14 PTEMD-A-detected candidate active TEs with the 138 non-redundant repeat families, 12 families are overlapped. (C) Comparing the TIPs detected by PTEMD-B, TEMP, RelocaTE, and T-lex. The total non-redundant TIPs detected by all of the four tools is 301, and 217 of them are co-detected by all of the four tools. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Previous result demonstrated that PTEMD-B have the best performance when the re-sequencing data are over ×30 coverage of the reference genome. Using ×50 re-sequencing data sets (HM-1 strain) and the reference genome, we detected the TIPs for the 14 intact TE families using PTEMD-B, TEMP, RelocaTE, and T-lex, and 248, 266, 255, and 268 TIPs have been detected, respectively. Most of them (217 TIPs) are co-detected by all of the four tools (Fig. <a reveal-id="DSW011F5" data-open="DSW011F5" class="link link-reveal link-table xref-fig">5</a>C, Venn diagram of the four programs detected TIPs), the non-redundant TIPs are 301, and only 9% (27 of 301) of the detected non-redundant TIPs are tool specific; these indicated that all of the four methods have the high-efficiency performance under high coverage of re-sequencing data set in <em>M. oryzae</em> genome. As a result, PTEMD performs better for repeat identification and is comparable for scanning of TIPs compared with other programs. However, since PTEMD is the only program that assumes both functions, it is the ‘one step’ tool for identifying polymorphic TEs and their insertions without the need of any other information except genomic sequences.</p>                            <h3 scrollto-destination=87199330 id="87199330" class="section-title" data-legacy-id=s3c>3.3. Polymorphic TE families in the <em>M. oryzae</em> and maize genomes</h3>
<p>Having demonstrated that PTEMD can detect polymorphic TEs that are likely to have been recently active, we next tested PTEMD with representative genomes. To assess the usability of PTEMD on genomes with widely different sizes and repeat contents, the genomes of <em>M. oryzae</em> (with a 40-Mb genome and 9.7% repetitive sequences)<sup><a reveal-id="DSW011C31" data-open="DSW011C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and maize (with a 2.1-Gb genome and over 85% repetitive sequences)<sup><a reveal-id="DSW011C32" data-open="DSW011C32" class="link link-ref link-reveal xref-bibr">32</a></sup> were used.</p><p>As mentioned earlier, we re-sequenced two field strains of <em>M. oryzae</em> (HM-1 and HM-2) at ×50 depth and compared the sequences to that of the reference genome. The average sequence identity between the genomes of HM-1 and HM-2 and that of reference genome was 99.97 ± 0.01%. Using PTEMD-A, we identified 15 TE families (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>, 14 intact TE families and 1 partial TE family). To determine whether those 15 TE families were previously reported, we compared them with three databases: (i) Repbase, (ii) GenBank nucleotide sequences database (NT), and (iii) GenBank protein sequences database (NR). The results indicated that 14 families have matches in the Repbase/NT/NR databases (threshold: 1e-10) (Fig. <a reveal-id="DSW011F6" data-open="DSW011F6" class="link link-reveal link-table xref-fig">6</a>). MoTE-2, -3, and -8 are previously reported transposons Pot2,<sup><a reveal-id="DSW011C33" data-open="DSW011C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Pot3,<sup><a reveal-id="DSW011C34" data-open="DSW011C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and OCCAN,<sup><a reveal-id="DSW011C35" data-open="DSW011C35" class="link link-ref link-reveal xref-bibr">35</a></sup> respectively; another 11 (MoTE-1, -4, -5, -6, -7, -9, -10, -11, -12, -13, and -14) are annotated in the <em>M. oryzae</em> genome sequencing project;<sup><a reveal-id="DSW011C31" data-open="DSW011C31" class="link link-ref link-reveal xref-bibr">31</a></sup> 4 (MoTE-1, -2, -3, and -8) are DNA transposons; 2 (MoTE-6 and MoTE-14) are related to LINEs; and 2 (MoTE-9 and MoTE-13) are LTR elements. We further verified/classified all 15 elements based on their structure, TSD, and similarity to known elements at protein level (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). All the four DNA elements are <em>Pogo</em>-like elements that belong to <em>Tc1</em>/<em>Mariner</em>-like superfamily. Three of the elements are <em>Gypsy</em>-like LTR elements. MoTE-9 represents an intact LTR element, whereas MoTE-13 represents a solo LTR. MoTE-15, on the other hand, only represents part of the internal region of a LTR retrotransposon, suggesting the detection of MoTE-15 is not due to TIP, but due to structural variation like indels among individual members of an element family. In other words, the fragment represented by MoTE-15 is not present in all elements, thus creates apparent indel polymorphism and is detected in the mapping process. All the remaining elements are LINEs or their deletion derivatives. Except MoTE-14, all the other seven families are related to MoTE-6. Interestingly, the seven elements all end with simple repeats (ACT)<em><sub>n</sub></em> (<em>n</em> = 5–9), which is different from the poly (A) tails of most known LINEs. This result indicates that PTEMD works well with small genomes like that of <em>M. oryzae</em>, and all the sequences recovered are bona fide transposon sequences albeit not all of them represent intact elements. </p>                        <a id="87199333" scrollto-destination="87199333"></a>
<div id="DSW011F6" data-id="DSW011F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01106.png?Expires=2147483647&Signature=bMpwQMsO8z5zHRv5j0cIsK-YwOrcGQ92kf7kHY0MHJwx8qLJgE2zoiyI~4JtauQCGJGrLcshuPMoKTOTh9fwXZSi4An3nlaBzYkJXALWpplyOsu67nvNWPINqUxSLy2pCmEPqfpZI3xsu2EU-e9iz4FwXBKUxdc~d5fKW3L3v5i2Cf56HSLTdXcd0txcpyhSLgA0Mqnj3oA6z8csCz82weYOFZApjGJQBy9lnmnuAtWuABnItB5GlbOsgVI6Lefj8zmAB4x57mT7v6~8H5JPY-kFqtAbYBQxupv8g-yR79Xp-mdiwJDbcSFjYti8-KxX8stL0ZVQFk63vS3M1M5dww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation and TIPs of 15 repeat families detected by PTEMD in M. oryzae. The annotation of each TE family is based on the most similar repeats in RepBase and NCBI NT and NR databases. The third column indicates the Top-paralogous E-value of the sequence alignments between the TEs and known repeats for each family. Middle panel: PTEMD-detected TIPs in HM-1 and HM-2 strains compared with the number of TEs in the reference genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199333/dsw01106.tiff" path-from-xml="dsw01106.tiff" target="_blank">View large</a><a section="87199333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01106.png?Expires=2147483647&Signature=lLdTEZY-yIbr71P~EsE53wwCtd6KXPNxJ5uUZQ3cNkSKYaCsJgVCj~IuNLCpOpiIOFtOT3ij1jlgrAxguS5tamr8XfxuHdu7ezDyzWvK8ZfwLoIaQuyOZA4CemcdT1k2ENyzP7Di3dPx382VAYxn-E6l0G0s5YZVy3JFrFVE3IZ6omuMWP9t-EvIQyHZZIZfbCCWgZ9ZHCwqh7PFi4ajTC8ANNY~A3xJheLg2AukL2Fv0o20hwU6wZoOh24Qe3MB4YJYUUI7AZy3SA5aw2yBHwB7hdAe4~ef9fJxxHHdCx1gHDatbC9YWDiS0woKtT3cgLE8zxKsB6wQfXYB6z7gYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199333&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation and TIPs of 15 repeat families detected by PTEMD in <em>M. oryzae</em>. The annotation of each TE family is based on the most similar repeats in RepBase and NCBI NT and NR databases. The third column indicates the Top-paralogous <em>E</em>-value of the sequence alignments between the TEs and known repeats for each family. Middle panel: PTEMD-detected TIPs in HM-1 and HM-2 strains compared with the number of TEs in the reference genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01106.png?Expires=2147483647&Signature=bMpwQMsO8z5zHRv5j0cIsK-YwOrcGQ92kf7kHY0MHJwx8qLJgE2zoiyI~4JtauQCGJGrLcshuPMoKTOTh9fwXZSi4An3nlaBzYkJXALWpplyOsu67nvNWPINqUxSLy2pCmEPqfpZI3xsu2EU-e9iz4FwXBKUxdc~d5fKW3L3v5i2Cf56HSLTdXcd0txcpyhSLgA0Mqnj3oA6z8csCz82weYOFZApjGJQBy9lnmnuAtWuABnItB5GlbOsgVI6Lefj8zmAB4x57mT7v6~8H5JPY-kFqtAbYBQxupv8g-yR79Xp-mdiwJDbcSFjYti8-KxX8stL0ZVQFk63vS3M1M5dww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation and TIPs of 15 repeat families detected by PTEMD in M. oryzae. The annotation of each TE family is based on the most similar repeats in RepBase and NCBI NT and NR databases. The third column indicates the Top-paralogous E-value of the sequence alignments between the TEs and known repeats for each family. Middle panel: PTEMD-detected TIPs in HM-1 and HM-2 strains compared with the number of TEs in the reference genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01106.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199333/dsw01106.tiff" path-from-xml="dsw01106.tiff" target="_blank">View large</a><a section="87199333" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01106.png?Expires=2147483647&Signature=lLdTEZY-yIbr71P~EsE53wwCtd6KXPNxJ5uUZQ3cNkSKYaCsJgVCj~IuNLCpOpiIOFtOT3ij1jlgrAxguS5tamr8XfxuHdu7ezDyzWvK8ZfwLoIaQuyOZA4CemcdT1k2ENyzP7Di3dPx382VAYxn-E6l0G0s5YZVy3JFrFVE3IZ6omuMWP9t-EvIQyHZZIZfbCCWgZ9ZHCwqh7PFi4ajTC8ANNY~A3xJheLg2AukL2Fv0o20hwU6wZoOh24Qe3MB4YJYUUI7AZy3SA5aw2yBHwB7hdAe4~ef9fJxxHHdCx1gHDatbC9YWDiS0woKtT3cgLE8zxKsB6wQfXYB6z7gYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199333&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01106.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation and TIPs of 15 repeat families detected by PTEMD in <em>M. oryzae</em>. The annotation of each TE family is based on the most similar repeats in RepBase and NCBI NT and NR databases. The third column indicates the Top-paralogous <em>E</em>-value of the sequence alignments between the TEs and known repeats for each family. Middle panel: PTEMD-detected TIPs in HM-1 and HM-2 strains compared with the number of TEs in the reference genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To further test PTEMD with a repeat-rich genome, we used maize as a model system. Over 85% of the maize genome, which is large in size (2.1 Gb), consists of repetitive sequences.<sup><a reveal-id="DSW011C32" data-open="DSW011C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In comparison with the Mo17 re-sequencing data set (GenBank accession number SRX245309) with B73 reference genome sequences, we identified 81 polymorphic repetitive sequence families (named ZmTE-1 to 81; sequences and their annotations are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The copy number in the reference genome is shown in Fig. <a reveal-id="DSW011F7" data-open="DSW011F7" class="link link-reveal link-table xref-fig">7</a>. To determine the identity of those 81 sequences families, we compared them with four databases (Repbase, NT, NR, and the maize-specific TE database: <a class="link link-uri" href="http://maizetedb.org/~maize/">http://maizetedb.org/~maize/</a>). For those without matches in the database, we manually classify them based on their structure, terminal sequences, and TSDs. Overall, 75 (92.6%) out of the 81 families represent TEs (Fig. <a reveal-id="DSW011F7" data-open="DSW011F7" class="link link-reveal link-table xref-fig">7</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) with 68 intact TEs and 7 partial TE sequences. The total copy number of those 68 TE families in B73 is 10,618. The five non-TE families are centromeric repeats, knob sequences, telomeric repeats, and local repeats. In addition, one family represents a chimeric structure from both DNA transposon and retrotransposon. Among the 75 TE families, 19 (25.3%) families are from retrotransposons and 56 (74.7%) are from DNA elements. The 19 retrotransposon families include one SINE, eight intact LTR retrotransposons, four solo LTRs, and six truncated or fragmented LTR elements (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Two of the solo LTRs are similar to the high copy number LTR element <em>Huck-2</em>.<sup><a reveal-id="DSW011C36" data-open="DSW011C36" class="link link-ref link-reveal xref-bibr">36</a></sup> One of the intact LTRs corresponds to <em>Zeon-1</em>, a moderate copy number element.<sup><a reveal-id="DSW011C37" data-open="DSW011C37" class="link link-ref link-reveal xref-bibr">37</a></sup> If we consider the 6 truncated or fragmented elements likely derived from structural changes such as deletions, only 13 retrotransposons are polymorphic between Mo17 and B73 genomes. Among the 56 DNA elements, only one represents fragmented element, the remainder stand for intact elements, which are likely from recent transposition. This suggests that there are 4-fold polymorphic DNA transposons vs. retrotransposons. The 55 intact DNA elements are all small (&lt;1 kb) non-autonomous DNA transposons including 51 miniature inverted TEs (MITEs). Notably, 41 (80%) of them are <em>Tourist</em> MITEs, which belong to <em>PIF/Harbinger</em> superfamily of DNA transposons. Several previously characterized DNA transposons are identified in this study, including <em>Ds1</em>, <em>Heartbreaker</em>, <em>Heart healer</em>, <em>mPIF</em>, <em>Ruq-st</em> , and the MITE that was found in maize <em>Br2-3</em> allele.<sup><a reveal-id="DSW011C38" data-open="DSW011C38" class="link link-ref link-reveal xref-bibr">38</a>–<a reveal-id="DSW011C42" data-open="DSW011C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Finally, it is worth of mention that for either species, the output of PTEMD-A does not contain any composite elements such as nested insertions, which is likely attributed to the stringent criteria that we used to define a TE family (≥97% identity over 95% length). </p>                        <a id="87199335" scrollto-destination="87199335"></a>
<div id="DSW011F7" data-id="DSW011F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01107.png?Expires=2147483647&Signature=WSCwRdq2yVTBH4WH3IfM10zDICyyhhA4BkzvPR-MgIYKRoXYlvk0PiZ2lmX~X87c7UII1pLGdcF04SYmi2E6XodtGrA~Znno60-wnhhc993H0imXkAhxls~YuBSH5M0Op~uwypNLHtkYlUfAueNqp8vLVJLoLUmieDnj-jr1z6de1yttPq21eq8~luQLc5qckncTeT4x-xPsk7wF2QZn4xUqfgaIjaRHC4ngMfjLNaSZJKNbXc0dNZj2hJP45Su~njaWNUf59nQsR02T9B1y2LxvlRaa-lqYB1Ifbigp96m-by6CY8vwi1jlLbCrc031E9GusT8dxAELGCNRJgDGRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation and TIPs of 81 sequences families detected by PTEMD in maize. The annotation of each TE/repeats family is based on the most similar repeats in RepBase, NCBI NT and NR databases, and maize TE database. Other features are shown similarly as in Fig. 6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01107.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199335/dsw01107.tiff" path-from-xml="dsw01107.tiff" target="_blank">View large</a><a section="87199335" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01107.png?Expires=2147483647&Signature=vwuhbHjjl-2XqVv-yfZwsCRoTVoBynJNwUhxkQNaXaRiUzWbfHKCToiviYgYfbkbQSVRyqaJDmtPV-T~ceiWXxY6QG0jvVIwWYMYq5GvTJyJ7eRTBggabFxFFuq14Rhkslphv4rJM4D~I0pL9bMvbIueITSR-bnQu9qDw4LKTheKTch~KkaMLjEETM9J2y3hCT7F8P8ojmy83NWpz8QjDbt7ufQxHKCStzldSiavxrLOiWmWmeqEj~pwHQcmtGheGwVsOJTP2MlOVJctFEUQ8n~T~AZnwZ01evCVj1f5JPlAfdxSDnhx1nv3OAFvXWLicCcExSTIxuXhnu-SAGV29A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199335&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01107.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation and TIPs of 81 sequences families detected by PTEMD in maize. The annotation of each TE/repeats family is based on the most similar repeats in RepBase, NCBI NT and NR databases, and maize TE database. Other features are shown similarly as in Fig. <a reveal-id="DSW011F6" data-open="DSW011F6" class="link link-reveal link-table xref-fig">6</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01107.png?Expires=2147483647&Signature=WSCwRdq2yVTBH4WH3IfM10zDICyyhhA4BkzvPR-MgIYKRoXYlvk0PiZ2lmX~X87c7UII1pLGdcF04SYmi2E6XodtGrA~Znno60-wnhhc993H0imXkAhxls~YuBSH5M0Op~uwypNLHtkYlUfAueNqp8vLVJLoLUmieDnj-jr1z6de1yttPq21eq8~luQLc5qckncTeT4x-xPsk7wF2QZn4xUqfgaIjaRHC4ngMfjLNaSZJKNbXc0dNZj2hJP45Su~njaWNUf59nQsR02T9B1y2LxvlRaa-lqYB1Ifbigp96m-by6CY8vwi1jlLbCrc031E9GusT8dxAELGCNRJgDGRg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Annotation and TIPs of 81 sequences families detected by PTEMD in maize. The annotation of each TE/repeats family is based on the most similar repeats in RepBase, NCBI NT and NR databases, and maize TE database. Other features are shown similarly as in Fig. 6. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01107.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199335/dsw01107.tiff" path-from-xml="dsw01107.tiff" target="_blank">View large</a><a section="87199335" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01107.png?Expires=2147483647&Signature=vwuhbHjjl-2XqVv-yfZwsCRoTVoBynJNwUhxkQNaXaRiUzWbfHKCToiviYgYfbkbQSVRyqaJDmtPV-T~ceiWXxY6QG0jvVIwWYMYq5GvTJyJ7eRTBggabFxFFuq14Rhkslphv4rJM4D~I0pL9bMvbIueITSR-bnQu9qDw4LKTheKTch~KkaMLjEETM9J2y3hCT7F8P8ojmy83NWpz8QjDbt7ufQxHKCStzldSiavxrLOiWmWmeqEj~pwHQcmtGheGwVsOJTP2MlOVJctFEUQ8n~T~AZnwZ01evCVj1f5JPlAfdxSDnhx1nv3OAFvXWLicCcExSTIxuXhnu-SAGV29A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199335&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01107.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Annotation and TIPs of 81 sequences families detected by PTEMD in maize. The annotation of each TE/repeats family is based on the most similar repeats in RepBase, NCBI NT and NR databases, and maize TE database. Other features are shown similarly as in Fig. <a reveal-id="DSW011F6" data-open="DSW011F6" class="link link-reveal link-table xref-fig">6</a>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199336 id="87199336" class="section-title" data-legacy-id=s3d>3.4. TIPs in <em>M. oryzae</em> and maize</h3>
<p>Compared with other TE sequences, polymorphic TEs create more TIPs in different individuals. To further assess the potential activity of TEs in <em>M. oryzae</em> and maize, we calculated the TIPs created by intact elements. By using PTEMD-B to compare the re-sequenced data sets of <em>M. oryzae</em> (HM-1 strain) and maize (Mo17) with their reference genomes, we detected 248 TIPs in <em>M. oryzae</em> (Fig. <a reveal-id="DSW011F8" data-open="DSW011F8" class="link link-reveal link-table xref-fig">8</a>A, detail in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) and 1455 TIPs in maize (Fig. <a reveal-id="DSW011F8" data-open="DSW011F8" class="link link-reveal link-table xref-fig">8</a>B, detail in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The average TIPs per 10 Mb chromosome regions in <em>M. oryzae</em> and maize are 62.0 and 6.9 (Fig. <a reveal-id="DSW011F8" data-open="DSW011F8" class="link link-reveal link-table xref-fig">8</a>C), respectively. We calculated the number of TIPs distributed in the genome (1 Mb windows was used), TIPs in both <em>M. oryzae</em> and maize were nonuniformly distributed (detail shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S3 and S4</a></span>). In <em>M. oryzae</em>, among the detected 248 TIPs, 91.1% (226 of 248) are caused by MoTE-1, -2, -3, and -4, albeit the copy number of those four TE families in reference genome contain only 32.2% (291 of 905) of total detected polymorphic TE sequences. And MoTE-1, -2, -3 are DNA transposons and MoTE-4 is a retrotransposon. In maize, among the detected 1455 TIPs, there were a total 539 TIPs derived from retrotransposons and 916 TIPs from DNA transposons. This was translated to 44 TIPs per retrotransposon family and 16 TIPs for DNA transposons. However, a close examination indicated that the majority (351) of TIPs from retrotransposons were attributed to ZmTE-4 and ZmTE-8 (Fig. <a reveal-id="DSW011F7" data-open="DSW011F7" class="link link-reveal link-table xref-fig">7</a>); both are related to Huck-2 elements. If the two families were excluded, each retrotransposon family generated 20 TIPs, which was largely comparable to that of DNA transposons. As a result, the contribution to polymorphism of individual class I and class II elements was similar with the exception of Huck-2, which generated nearly 10 times more polymorphic insertions than other elements. Despite the presence of more polymorphic DNA transposon families as well as more TIPs from DNA transposons, the average size of polymorphic DNA transposons is only 311 bp, whereas that for retrotransposon is 3.8 kb, which is a dozen-fold that of DNA transposons. This explains why DNA elements contribute to much less genome size than retrotransposons in maize. </p>                        <a id="87199338" scrollto-destination="87199338"></a>
<div id="DSW011F8" data-id="DSW011F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01108.png?Expires=2147483647&Signature=Lxk788VAyhBqTOJO5BwbCi4Phb5CVL8y0SJ0qPhZl6hxS9hBI1ttheTIJuGCOcr-KVq-N~hMBkM1gn8iLWWEy4-OwKTM3p67ppBCHwRF7PybdOPuHV~e-9s1aXjktNvSVbGMBnIeGbDzf7UDjUKbPriqbzs-u~cYhHbyb7xViLKnA6liXRMLa56YdIHxj-GvstslI49IHw8D7oAkfM-uQu2k6LvWGuwxIfLhtZi1XrYpRj~hqeto2PJCSYhxZ0VkEs2SQss15S0Ws1xUMpROoMZneE5I1ApMG2rtAEvp4QWLAC-xzdz0~89Cm8pXddf03TRWaTzkYjAKw6cUru-RaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The classification of the TIPs in M. oryzae and maize genomes. (A and B) Classification of the PTEMD-B-detected TIPs in M. oryzae (A) and maize (B), respectively. Different parts represent the classification of the TIPs. (C) Average TIPs per 10 Mb chromosome regions in M. oryzae and maize. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01108.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199338/dsw01108.tiff" path-from-xml="dsw01108.tiff" target="_blank">View large</a><a section="87199338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01108.png?Expires=2147483647&Signature=eFSyhYJLXATG4xYJXy7PAYNrihTtxtRlZYNNwFD0kbBYZRZeQlh2h0vshj9KyXVHrmXv6nh98w-lUNLOD0-yQs4LQFMswPDyRJjB29Vs3FEqOWpBCnARjF-Za5Ee24BgYzuTYS9Weqo9YtA6LYfvCtKvl~I2bW7agwVCgkfb2-HchHvVDguftUIKPG-SZ5UX7ztzxeSjc675uCXowUVKnJKqZOSG5x7zUk-cFP5aGvyPvXzE8TGBieN2TJgrMaEF3Lq26VV43jqTDZlLl6RWWXBkc5mASiL3LQf~SQ2Y4asWCs9spmVa4R9KAT~HCfIwnzUx21jyU0muNRfEcz~8Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199338&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01108.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The classification of the TIPs in <em>M. oryzae</em> and maize genomes. (A and B) Classification of the PTEMD-B-detected TIPs in <em>M. oryzae</em> (A) and maize (B), respectively. Different parts represent the classification of the TIPs. (C) Average TIPs per 10 Mb chromosome regions in <em>M. oryzae</em> and maize. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW011F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/m_dsw01108.png?Expires=2147483647&Signature=Lxk788VAyhBqTOJO5BwbCi4Phb5CVL8y0SJ0qPhZl6hxS9hBI1ttheTIJuGCOcr-KVq-N~hMBkM1gn8iLWWEy4-OwKTM3p67ppBCHwRF7PybdOPuHV~e-9s1aXjktNvSVbGMBnIeGbDzf7UDjUKbPriqbzs-u~cYhHbyb7xViLKnA6liXRMLa56YdIHxj-GvstslI49IHw8D7oAkfM-uQu2k6LvWGuwxIfLhtZi1XrYpRj~hqeto2PJCSYhxZ0VkEs2SQss15S0Ws1xUMpROoMZneE5I1ApMG2rtAEvp4QWLAC-xzdz0~89Cm8pXddf03TRWaTzkYjAKw6cUru-RaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The classification of the TIPs in M. oryzae and maize genomes. (A and B) Classification of the PTEMD-B-detected TIPs in M. oryzae (A) and maize (B), respectively. Different parts represent the classification of the TIPs. (C) Average TIPs per 10 Mb chromosome regions in M. oryzae and maize. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01108.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199338/dsw01108.tiff" path-from-xml="dsw01108.tiff" target="_blank">View large</a><a section="87199338" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw01108.png?Expires=2147483647&Signature=eFSyhYJLXATG4xYJXy7PAYNrihTtxtRlZYNNwFD0kbBYZRZeQlh2h0vshj9KyXVHrmXv6nh98w-lUNLOD0-yQs4LQFMswPDyRJjB29Vs3FEqOWpBCnARjF-Za5Ee24BgYzuTYS9Weqo9YtA6LYfvCtKvl~I2bW7agwVCgkfb2-HchHvVDguftUIKPG-SZ5UX7ztzxeSjc675uCXowUVKnJKqZOSG5x7zUk-cFP5aGvyPvXzE8TGBieN2TJgrMaEF3Lq26VV43jqTDZlLl6RWWXBkc5mASiL3LQf~SQ2Y4asWCs9spmVa4R9KAT~HCfIwnzUx21jyU0muNRfEcz~8Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199338&ar=1745420&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01108.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The classification of the TIPs in <em>M. oryzae</em> and maize genomes. (A and B) Classification of the PTEMD-B-detected TIPs in <em>M. oryzae</em> (A) and maize (B), respectively. Different parts represent the classification of the TIPs. (C) Average TIPs per 10 Mb chromosome regions in <em>M. oryzae</em> and maize. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To further identify the potential effects of TE insertions on gene functions, we first grouped the TIPs into four major classes: (i) TE insertions in the gene coding regions; (ii) TE insertions in the core-promoter regions (the 500 bp upstream sequence of the gene transcription start site)<sup><a reveal-id="DSW011C43" data-open="DSW011C43" class="link link-ref link-reveal xref-bibr">43</a></sup>; (iii) TE insertions in the intron regions; and (iv) TE insertions in intergenic regions. Then, we downloaded the GFF files for the three genomes and developed Perl scripts (available from the author of this paper) to extract and classify the TIPs. Because TE insertions in coding regions directly interrupt the genes, we expected that few of the TIPs would be located in coding regions. Among the 248 TIPs detected in the <em>M. oryzae</em> genome, however, 172, 34, 8, and 34 were inserted in intergenic regions, core-promoter regions, intron regions, and coding regions, respectively, and the percentages were 69.4, 13.7, 3.2, and 13.7%, respectively (Fig. <a reveal-id="DSW011F8" data-open="DSW011F8" class="link link-reveal link-table xref-fig">8</a>A and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). This indicated that 30.6% of TIPs in <em>M. oryzae</em> genome were located in the genic regions. Among the 1455 TIPs detected in the maize genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>), 72.0, 8.3, 16.5, and 3.2% were inserted in the intergenic regions, core-promoter regions, intron regions, and coding regions, respectively (Fig. <a reveal-id="DSW011F8" data-open="DSW011F8" class="link link-reveal link-table xref-fig">8</a>B).</p>                            <h2 scrollto-destination=87199340 id="87199340" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Two methods can be used to determine whether a DNA fragment is a potential active TE.<sup><a reveal-id="DSW011C44" data-open="DSW011C44" class="link link-ref link-reveal xref-bibr">44</a></sup> In the first method, researchers detect the fragment's activity by monitoring <em>de novo</em> insertions in the next generation. In the second method, researchers identify TIPs among individuals. We developed PTEMD to detect both polymorphic TEs and genome-wide TIPs using reference genomes and high-throughput reads data sets. Unlike previously developed library-based or structure-based methods, PTEMD identifies polymorphic TEs (PTEMD-A) and their TIPs (PTEMD-B) by searching all of the evidences of sequence movement at the TIP sites. PTEMD-A does not depend on any TE sequence or libraries. The program detects all of the mobile sequences by comparing the re-sequencing data with the reference genome and then classifies the sequences into clusters, which are filtered and classified into TE families. Our result indicated that, although non-TE sequences are included in the PTEMD-A output file, however, the proportion of non-TE families is low (6.7% in <em>M. oryzae</em> and 16.0% in maize), indicating that most of the PTEMD-A-detected sequences are intact polymorphic TE families. Moreover, we analysed the genetically purified, single-isolated re-sequencing data sets, and we found the direct evidences of current mobility for more than half of the PTEMD-A-detected TE families in <em>M. oryzae</em>. This indicates that many of the PTEMD-A-detected polymorphic TE families could be currently active.</p><p>PTEMD-B maps the high-throughput short reads to the TE sequences and maps all of the partially mapped reads to the reference genome sequences. The position information is then analysed, and all of the TIPs including breakpoint evidence are presented in the PTEMD output file. As shown for the genome of <em>M. oryzae</em> and maize, only 6.7% (1 of 15) in <em>M. oryzae</em> and 16.0% (13 of 81) in maize of the PTEMD-detected mobile sequence families are not intact TEs; the polymorphism of those sequence families may be due to indel/homologous recombination/sequence rearrangement, in other words, most of the PTEMD-detected mobile sequence families are good candidates for active TEs.</p><p>We integrated the cores of the Pindel,<sup><a reveal-id="DSW011C26" data-open="DSW011C26" class="link link-ref link-reveal xref-bibr">26</a></sup> BWA<sup><a reveal-id="DSW011C29" data-open="DSW011C29" class="link link-ref link-reveal xref-bibr">29</a></sup>/Bowtie2,<sup><a reveal-id="DSW011C45" data-open="DSW011C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Muscle,<sup><a reveal-id="DSW011C28" data-open="DSW011C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and BLAST<sup><a reveal-id="DSW011C46" data-open="DSW011C46" class="link link-ref link-reveal xref-bibr">46</a></sup> programs into the PTEMD program as the sequence-alignment engines. BWA is a widely used high-throughput reads alignment program that can quickly map high-throughput short reads to the reference genome sequences.<sup><a reveal-id="DSW011C29" data-open="DSW011C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The BWT-MEM algorithm is used in PTEMD to map the high-throughput reads to both the genome and TE sequences. Muscle<sup><a reveal-id="DSW011C28" data-open="DSW011C28" class="link link-ref link-reveal xref-bibr">28</a></sup> is used in PTEMD for clustering the sequence families. BLAST<sup><a reveal-id="DSW011C46" data-open="DSW011C46" class="link link-ref link-reveal xref-bibr">46</a></sup> is used to align the TE sequence to the reference genomes. The PTEMD output file contains all of the detected TE families and their detailed distribution positions including all of the supporting breakpoint evidence. The PTEMD output file is therefore useful for checking the raw sequence data of the identified TE families or new TE insertions.</p><p>Previous programs have used three main methods for repeats scanning. The first method is a library-based approach, such as that used in RepeatMasker.<sup><a reveal-id="DSW011C12" data-open="DSW011C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The main drawback of programs that use these kinds of homology-dependent searches is that they can only detect sequences that are already known, i.e. they cannot detect completely novel elements. The second method is signature-based repeat searching, such as Piler.<sup><a reveal-id="DSW011C15" data-open="DSW011C15" class="link link-ref link-reveal xref-bibr">15</a></sup> This method can search a query sequence for existing structures or motifs that are characteristic of a known repeat sequence, it can be used to find new repeats but not new classes of repeats. The third method is represented by <em>k-</em>mer and spaced seed approaches, such as those used in Repeatscout.<sup><a reveal-id="DSW011C14" data-open="DSW011C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This approach can <em>de novo</em> identify the repeat families based on genome sequences and has been widely used for scanning new TE families. Unlike these three methods, PTEMD is a homology-independent method that can scan active TE candidates (also including other polymorphic repeats) and their genome-wide insertion sites using high-throughput pair-end short reads and reference genomes. PTEMD therefore overcomes some important drawbacks of the previously published methods and will become an important tool for <em>de novo</em> polymorphic TE scanning.</p><p><em>M. oryzae</em> is a model pathomycete that can quickly overcome the host resistance.<sup><a reveal-id="DSW011C47" data-open="DSW011C47" class="link link-ref link-reveal xref-bibr">47</a></sup> More than 80 resistance genes have been identified in rice but most of these resistance genes remain effective for only 2–3 years.<sup><a reveal-id="DSW011C48" data-open="DSW011C48" class="link link-ref link-reveal xref-bibr">48</a></sup> A previous study provided evidence that a TE insertion into the promoter region of the avirulence gene <em>Piz-t</em> caused an avirulent strain to become to virulent<sup><a reveal-id="DSW011C49" data-open="DSW011C49" class="link link-ref link-reveal xref-bibr">49</a></sup>; however, the genome-wide polymorphic TE families and their distribution landscapes among different <em>M. oryzae</em> strains were still largely unknown. Using PTEMD, we identified 15 polymorphic TE families in <em>M. oryzae</em> (14 intact TEs and 1 partial TE). In reference genome, the 14 intact TE families contain 905 copies, and the total size of those TE families is ∼1.5 Mb (3.8% of the <em>M. oryzae</em> genome). The top four TE families (MoTE-1, -2, -3, and -4) are widely distributed in strains HM-1 and HM-2. Further analysis of the dynamics and evolution of these top four TE families in field populations may help elucidate how these TEs affect <em>M. oryzae</em> virulence.</p><p>To evaluate the performance of PTEMD with large and repeat-rich genomes, we identified 68 TE families in the maize genome are polymorphic and likely active in the recent past. The total copy number of the TE sequences is 10,618 in the maize genome. Those TE sequences cover 10.1 Mb of maize genome (0.48% of genome). Our results provide novel insights about the transposon biology. First of all, TE polymorphism varies greatly between the two tested species. The highest TE polymorphism is detected in <em>M. oryzae</em>: its genome size is only 2% that of maize but harbours 14 families of polymorphic TE families. Second, despite that most of the maize genome consists of LTR retrotransposons, DNA elements contribute more to polymorphism among different varieties compared with LTR retrotransposons. LTR elements make up the largest component of the genome due to their large size, not high activity. Third, not a single autonomous DNA transposon has been detected to be recently mobile in maize, and all the detected DNA transposons are very small (&lt;1 kb) non-autonomous elements. Obviously, some of the autonomous elements in maize are still functioning because they are providing transposition machinery for non-autonomous elements. It is likely their large sizes make them less competent for transposition, consistent with the notion that DNA element size is critical for the transposition activity.<sup><a reveal-id="DSW011C50  DSW011C52" data-open="DSW011C50  DSW011C52" class="link link-ref link-reveal xref-bibr">50–52</a></sup> Finally, the polymorphic TEs discovered using PTEMD can be used as potential novel tagging tools for breeding or studying gene function. As more and more genome sequences are available in the future, PTEMD will be an important tool in identifying polymorphic TEs in different organisms.</p>                            <h2 scrollto-destination=87199347 id="87199347" class="section-title" data-legacy-id=s5>Availability</h2>
<p>PTEMD program, HM-1 and HM-2 strains' re-sequencing data sets (sequences in this study), and the genome-wide polymorphic TEs in <em>M. oryzae</em> and maize are freely and publically available at <a class="link link-uri" href="http://www.kanglab.cn/blast/PTEMD_V1.02.htm">http://www.kanglab.cn/blast/PTEMD_V1.02.htm</a>.</p>                            <h2 scrollto-destination=87199349 id="87199349" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported in part by the National Natural Science Foundation of China to H.K. (grant numbers 31571964 and 31201476), the 973 Project (grant number 2012CB114005) of the Ministry of Science and Technology China to G.-L.W., and the US National Science Foundation to N.J. and S.-H.S. (grant number IOS-1126998) and to S.-H.S. (grant number MCB-1119778). Funding to pay the Open Access publication charges for this article was provided by the Institute of Plant Protection, Chinese Academy of Agricultural Sciences.</p>                            <h2 scrollto-destination=87199351 id="87199351" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Shuangyong Yan for reading, discussing, and editing the paper.</p>                            <h2 scrollto-destination=87199353 id="87199353" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW011C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Doolittle</div>
               <div class="given-names">W.F.</div>
            </div>, <div class="name string-name"><div class="surname">Sapienza</div> <div class="given-names">C.</div></div></div>
          <div class="year">1980</div>, <div class="article-title">Selfish genes, the phenotype paradigm and genome evolution</div>, <div class="source">Nature</div> , <div class="volume">284</div>, <div class="fpage">601</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20genes%2C%20the%20phenotype%20paradigm%20and%20genome%20evolution&amp;author=W.F.%20Doolittle&amp;author=C.%20Sapienza&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=601-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284601a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284601a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284601a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6245369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Orgel</div>
               <div class="given-names">L.E.</div>
            </div>, <div class="name string-name"><div class="surname">Crick</div> <div class="given-names">F.H.</div></div></div>
          <div class="year">1980</div>, <div class="article-title">Selfish DNA: the ultimate parasite</div>, <div class="source">Nature</div> , <div class="volume">284</div>, <div class="fpage">604</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selfish%20DNA%3A%20the%20ultimate%20parasite&amp;author=L.E.%20Orgel&amp;author=F.H.%20Crick&amp;publication_year=1980&amp;journal=Nature&amp;volume=284&amp;pages=604-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/284604a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F284604a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F284604a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7366731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rebollo</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Romanish</div> <div class="given-names">M.T.</div></div>, <div class="name string-name"><div class="surname">Mager</div> <div class="given-names">D.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Transposable elements: an abundant and natural source of regulatory sequences for host genes</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">46</div>, <div class="fpage">21</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%3A%20an%20abundant%20and%20natural%20source%20of%20regulatory%20sequences%20for%20host%20genes&amp;author=R.%20Rebollo&amp;author=M.T.%20Romanish&amp;author=D.L.%20Mager&amp;publication_year=2012&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=46&amp;pages=21-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110711-155621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110711-155621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110711-155621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22905872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Studer</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Ross-Ibarra</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Doebley</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Identification of a functional transposon insertion in the maize domestication gene tb1</div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">1160</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20functional%20transposon%20insertion%20in%20the%20maize%20domestication%20gene%20tb1&amp;author=A.%20Studer&amp;author=Q.%20Zhao&amp;author=J.%20Ross-Ibarra&amp;author=J.%20Doebley&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1160-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.942" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.942" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.942"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21946354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fedoroff</div>
               <div class="given-names">N.V.</div>
            </div>
          </div>
          <div class="year">2012</div>, <div class="article-title">Transposable elements, epigenetics, and genome evolution</div>, <div class="source">Science</div> , <div class="volume">338</div>, <div class="fpage">758</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%2C%20epigenetics%2C%20and%20genome%20evolution&amp;author=N.V.%20Fedoroff&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=758-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.338.6108.758" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.338.6108.758" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.338.6108.758"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23145453" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Britten</div>
               <div class="given-names">R.J.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">Transposable element insertions have strongly affected human evolution</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">19945</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20element%20insertions%20have%20strongly%20affected%20human%20evolution&amp;author=R.J.%20Britten&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=19945-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1014330107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1014330107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1014330107"> </span></div> </div></div></div></div></div><div content-id="DSW011C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lisch</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2013</div>, <div class="article-title">How important are transposons for plant evolution?</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">14</div>, <div class="fpage">49</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20important%20are%20transposons%20for%20plant%20evolution%3F&amp;author=D.%20Lisch&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=14&amp;pages=49-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3374" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3374"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23247435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Polak</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Domany</div> <div class="given-names">E.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Alu elements contain many binding sites for transcription factors and may play a role in regulation of developmental processes</div>, <div class="source">BMC Genomics</div> , <div class="volume">7</div>, <div class="fpage">133</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alu%20elements%20contain%20many%20binding%20sites%20for%20transcription%20factors%20and%20may%20play%20a%20role%20in%20regulation%20of%20developmental%20processes&amp;author=P.%20Polak&amp;author=E.%20Domany&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=133" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16740159" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piegu</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Guyot</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Picault</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Doubling genome size without polyploidization: dynamics of retrotransposition-driven genomic expansions in <em>Oryza australiensis</em>, a wild relative of rice</div>, <div class="source">Genome Res.</div> , <div class="volume">16</div>, <div class="fpage">1262</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Doubling%20genome%20size%20without%20polyploidization%3A%20dynamics%20of%20retrotransposition-driven%20genomic%20expansions%20in%20Oryza%20australiensis%2C%20a%20wild%20relative%20of%20rice&amp;author=B.%20Piegu&amp;author=R.%20Guyot&amp;author=N.%20Picault&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1262-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5290206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5290206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5290206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16963705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bennetzen</div>
               <div class="given-names">J.L.</div>
            </div>, <div class="name string-name"><div class="surname">Kellogg</div> <div class="given-names">E.A.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Do plants have a one-way ticket to genomic obesity?</div>, <div class="source">Plant Cell</div> , <div class="volume">9</div>, <div class="fpage">1509</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20plants%20have%20a%20one-way%20ticket%20to%20genomic%20obesity%3F&amp;author=J.L.%20Bennetzen&amp;author=E.A.%20Kellogg&amp;publication_year=1997&amp;journal=Plant%20Cell&amp;volume=9&amp;pages=1509-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.9.9.1509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.9.9.1509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.9.9.1509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12237393" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kidwell</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Lisch</div> <div class="given-names">D.R.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Transposable elements and host genome evolution</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="volume">15</div>, <div class="fpage">95</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20and%20host%20genome%20evolution&amp;author=M.G.%20Kidwell&amp;author=D.R.%20Lisch&amp;publication_year=2000&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=15&amp;pages=95-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(99)01817-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(99)01817-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(99)01817-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10675923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tarailo-Graovac</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">N.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Using RepeatMasker to identify repetitive elements in genomic sequences</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <div class="comment">Chapter 4</div>, <div class="fpage">Unit 4.10</div>.</div></div></div></div><div content-id="DSW011C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bao</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Automated de novo identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">1269</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">De novo identification of repeat families in large genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">PILER: identification and classification of genomic repeats</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">i152</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PILER%3A%20identification%20and%20classification%20of%20genomic%20repeats&amp;author=R.C.%20Edgar&amp;author=E.W.%20Myers&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i152-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Initiative</div>
               <div class="given-names">A.G.</div>
            </div>
          </div>
          <div class="year">2000</div>, <div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genome%20sequence%20of%20the%20flowering%20plant%20Arabidopsis%20thaliana&amp;author=A.G.%20Initiative&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=796" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Venter</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Adams</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">The sequence of the human genome</div>, <div class="source">Science</div> , <div class="volume">291</div>, <div class="fpage">1304</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20of%20the%20human%20genome&amp;author=J.C.%20Venter&amp;author=M.D.%20Adams&amp;author=E.W.%20Myers&amp;publication_year=2001&amp;journal=Science&amp;volume=291&amp;pages=1304-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1058040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1058040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1058040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11181995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gibbs</div>
               <div class="given-names">R.A.</div>
            </div>, <div class="name string-name"><div class="surname">Weinstock</div> <div class="given-names">G.M.</div></div>, <div class="name string-name"><div class="surname">Metzker</div> <div class="given-names">M.L.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Genome sequence of the Brown Norway rat yields insights into mammalian evolution</div>, <div class="source">Nature</div> , <div class="volume">428</div>, <div class="fpage">493</div>–<div class="lpage">521</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20Brown%20Norway%20rat%20yields%20insights%20into%20mammalian%20evolution&amp;author=R.A.%20Gibbs&amp;author=G.M.%20Weinstock&amp;author=M.L.%20Metzker&amp;publication_year=2004&amp;journal=Nature&amp;volume=428&amp;pages=493-521" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02426" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02426" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02426"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15057822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goff</div>
               <div class="given-names">S.A.</div>
            </div>, <div class="name string-name"><div class="surname">Ricke</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lan</div> <div class="given-names">T.-H.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">A draft sequence of the rice genome (<em>Oryza sativa</em> L. ssp. <em>japonica</em>)</div>, <div class="source">Science</div> , <div class="volume">296</div>, <div class="fpage">92</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20rice%20genome%20(Oryza%20sativa%20L.%20ssp.%20japonica)&amp;author=S.A.%20Goff&amp;author=D.%20Ricke&amp;author=T.-H.%20Lan&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=92-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11935018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">A draft sequence of the rice genome (<em>Oryza sativa</em> L. ssp. <em>indica</em>)</div>, <div class="source">Science</div> , <div class="volume">296</div>, <div class="fpage">79</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20draft%20sequence%20of%20the%20rice%20genome%20(Oryza%20sativa%20L.%20ssp.%20indica)&amp;author=J.%20Yu&amp;author=S.%20Hu&amp;author=J.%20Wang&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=79-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1068037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1068037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1068037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11935017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lerat</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">Identifying repeats and transposable elements in sequenced genomes: how to find your way through the dense forest of programs</div>, <div class="source">Heredity</div> , <div class="volume">104</div>, <div class="fpage">520</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20repeats%20and%20transposable%20elements%20in%20sequenced%20genomes%3A%20how%20to%20find%20your%20way%20through%20the%20dense%20forest%20of%20programs&amp;author=E.%20Lerat&amp;publication_year=2010&amp;journal=Heredity&amp;volume=104&amp;pages=520-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2009.165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2009.165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2009.165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19935826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhuang</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Theurkauf</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Weng</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">TEMP: a computational method for analyzing transposable element polymorphism in populations</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">6826</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TEMP%3A%20a%20computational%20method%20for%20analyzing%20transposable%20element%20polymorphism%20in%20populations&amp;author=J.%20Zhuang&amp;author=J.%20Wang&amp;author=W.%20Theurkauf&amp;author=Z.%20Weng&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=6826-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24753423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Robb</div>
               <div class="given-names">S.M.</div>
            </div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Valencia</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">The use of RelocaTE and unassembled short reads to produce high-resolution snapshots of transposable element generated diversity in rice</div>, <div class="source">G3 (Bethesda)</div> , <div class="volume">3</div>, <div class="fpage">949</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20use%20of%20RelocaTE%20and%20unassembled%20short%20reads%20to%20produce%20high-resolution%20snapshots%20of%20transposable%20element%20generated%20diversity%20in%20rice&amp;author=S.M.%20Robb&amp;author=L.%20Lu&amp;author=E.%20Valencia&amp;publication_year=2013&amp;journal=G3%20(Bethesda)&amp;volume=3&amp;pages=949-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.005348" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.005348" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.005348"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23576519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fiston-Lavier</div>
               <div class="given-names">A.S.</div>
            </div>, <div class="name string-name"><div class="surname">Carrigan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Gonzalez</div> <div class="given-names">J.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">T-lex: a program for fast and accurate assessment of transposable element presence using next-generation sequencing data</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">e36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=T-lex%3A%20a%20program%20for%20fast%20and%20accurate%20assessment%20of%20transposable%20element%20presence%20using%20next-generation%20sequencing%20data&amp;author=A.S.%20Fiston-Lavier&amp;author=M.%20Carrigan&amp;author=D.A.%20Petrov&amp;author=J.%20Gonzalez&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=e36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21177644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stewart</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Via</div> <div class="given-names">L.E.</div></div></div>
          <div class="year">1993</div>, <div class="article-title">A rapid CTAB DNA isolation technique useful for RAPD fingerprinting and other PCR applications</div>, <div class="source">Biotechniques</div> , <div class="volume">14</div>, <div class="fpage">748</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20CTAB%20DNA%20isolation%20technique%20useful%20for%20RAPD%20fingerprinting%20and%20other%20PCR%20applications&amp;author=C.%20Stewart&amp;author=L.E.%20Via&amp;publication_year=1993&amp;journal=Biotechniques&amp;volume=14&amp;pages=748-50" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8512694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ye</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ning</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2865</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pindel%3A%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads&amp;author=K.%20Ye&amp;author=M.H.%20Schulz&amp;author=Q.%20Long&amp;author=R.%20Apweiler&amp;author=Z.%20Ning&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2865-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19561018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Thompson</div>
               <div class="given-names">J.D.</div>
            </div>, <div class="name string-name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Multiple sequence alignment using ClustalW and ClustalX</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <div class="comment">Chapter 2</div>, <div class="fpage">Unit 2.3</div>.</div></div></div></div><div content-id="DSW011C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="year">2004</div>, <div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">32</div>, <div class="fpage">1792</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%20wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jeon</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">G.W.</div></div>, <div class="name string-name"><div class="surname">Dean</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">Y.H.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Experimental evolution reveals genome-wide spectrum and dynamics of mutations in the rice blast fungus, <em>Magnaporthe oryzae</em></div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e65416</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20evolution%20reveals%20genome-wide%20spectrum%20and%20dynamics%20of%20mutations%20in%20the%20rice%20blast%20fungus%2C%20Magnaporthe%20oryzae&amp;author=J.%20Jeon&amp;author=J.%20Choi&amp;author=G.W.%20Lee&amp;author=R.A.%20Dean&amp;author=Y.H.%20Lee&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e65416" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0065416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0065416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0065416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23741492" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dean</div>
               <div class="given-names">R.A.</div>
            </div>, <div class="name string-name"><div class="surname">Talbot</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Ebbole</div> <div class="given-names">D.J.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">The genome sequence of the rice blast fungus <em>Magnaporthe grisea</em></div>, <div class="source">Nature</div> , <div class="volume">434</div>, <div class="fpage">980</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20rice%20blast%20fungus%20Magnaporthe%20grisea&amp;author=R.A.%20Dean&amp;author=N.J.%20Talbot&amp;author=D.J.%20Ebbole&amp;publication_year=2005&amp;journal=Nature&amp;volume=434&amp;pages=980-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15846337" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schnable</div>
               <div class="given-names">P.S.</div>
            </div>, <div class="name string-name"><div class="surname">Ware</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fulton</div> <div class="given-names">R.S.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The B73 maize genome: complexity, diversity, and dynamics</div>, <div class="source">Science</div> , <div class="volume">326</div>, <div class="fpage">1112</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20B73%20maize%20genome%3A%20complexity%2C%20diversity%2C%20and%20dynamics&amp;author=P.S.%20Schnable&amp;author=D.%20Ware&amp;author=R.S.%20Fulton&amp;publication_year=2009&amp;journal=Science&amp;volume=326&amp;pages=1112-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1178534" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1178534" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1178534"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kachroo</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Leong</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Chattoo</div> <div class="given-names">B.B.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">Pot2, an inverted repeat transposon from the rice blast fungus <em>Magnaporthe grisea</em></div>, <div class="source">Mol. Gen. Genet.</div> , <div class="volume">245</div>, <div class="fpage">339</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pot2%2C%20an%20inverted%20repeat%20transposon%20from%20the%20rice%20blast%20fungus%20Magnaporthe%20grisea&amp;author=P.%20Kachroo&amp;author=S.A.%20Leong&amp;author=B.B.%20Chattoo&amp;publication_year=1994&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=245&amp;pages=339-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00290114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00290114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00290114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7816044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kang</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lebrun</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Farrall</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Valent</div> <div class="given-names">B.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Gain of virulence caused by insertion of a Pot3 transposon in a <em>Magnaporthe grisea</em> avirulence gene</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="volume">14</div>, <div class="fpage">671</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gain%20of%20virulence%20caused%20by%20insertion%20of%20a%20Pot3%20transposon%20in%20a%20Magnaporthe%20grisea%20avirulence%20gene&amp;author=S.%20Kang&amp;author=M.H.%20Lebrun&amp;author=L.%20Farrall&amp;author=B.%20Valent&amp;publication_year=2001&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=14&amp;pages=671-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2001.14.5.671" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2001.14.5.671" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2001.14.5.671"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11332731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kito</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fukiya</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sone</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tomita</div> <div class="given-names">F.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Occan, a novel transposon in the Fot1 family, is ubiquitously found in several <em>Magnaporthe grisea</em> isolates</div>, <div class="source">Curr. Genet.</div> , <div class="volume">42</div>, <div class="fpage">322</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Occan%2C%20a%20novel%20transposon%20in%20the%20Fot1%20family%2C%20is%20ubiquitously%20found%20in%20several%20Magnaporthe%20grisea%20isolates&amp;author=H.%20Kito&amp;author=Y.%20Takahashi&amp;author=J.%20Sato&amp;author=S.%20Fukiya&amp;author=T.%20Sone&amp;author=F.%20Tomita&amp;publication_year=2003&amp;journal=Curr.%20Genet.&amp;volume=42&amp;pages=322-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00294-002-0365-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00294-002-0365-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00294-002-0365-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12612805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">SanMiguel</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B.S.</div></div>, <div class="name string-name"><div class="surname">Tikhonov</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nakajima</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Bennetzen</div> <div class="given-names">J.L.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">The paleontology of intergene retrotransposons of maize</div>, <div class="source">Nat. Genet.</div> , <div class="volume">20</div>, <div class="fpage">43</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20paleontology%20of%20intergene%20retrotransposons%20of%20maize&amp;author=P.%20SanMiguel&amp;author=B.S.%20Gaut&amp;author=A.%20Tikhonov&amp;author=Y.%20Nakajima&amp;author=J.L.%20Bennetzen&amp;publication_year=1998&amp;journal=Nat.%20Genet.&amp;volume=20&amp;pages=43-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/1695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F1695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F1695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9731528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hu</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Das</div> <div class="given-names">O.P.</div></div>, <div class="name string-name"><div class="surname">Messing</div> <div class="given-names">J.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Zeon-1, a member of a new maize retrotransposon family</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="volume">248</div>, <div class="fpage">471</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zeon-1%2C%20a%20member%20of%20a%20new%20maize%20retrotransposon%20family&amp;author=W.%20Hu&amp;author=O.P.%20Das&amp;author=J.%20Messing&amp;publication_year=1995&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=248&amp;pages=471-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02191647" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02191647" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02191647"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7565611" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sutton</div>
               <div class="given-names">W.D.</div>
            </div>, <div class="name string-name"><div class="surname">Gerlach</div> <div class="given-names">W.L.</div></div>, <div class="name string-name"><div class="surname">Peacock</div> <div class="given-names">W.J.</div></div>, <div class="name string-name"><div class="surname">Schwartz</div> <div class="given-names">D.</div></div></div>
          <div class="year">1984</div>, <div class="article-title">Molecular analysis of ds controlling element mutations at the adh1 locus of maize</div>, <div class="source">Science</div> , <div class="volume">223</div>, <div class="fpage">1265</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20ds%20controlling%20element%20mutations%20at%20the%20adh1%20locus%20of%20maize&amp;author=W.D.%20Sutton&amp;author=W.L.%20Gerlach&amp;author=W.J.%20Peacock&amp;author=D.%20Schwartz&amp;publication_year=1984&amp;journal=Science&amp;volume=223&amp;pages=1265-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.223.4642.1265" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.223.4642.1265" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.223.4642.1265"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17759347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pisabarro</div>
               <div class="given-names">A.G.</div>
            </div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">W.F.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Saedler</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gierl</div> <div class="given-names">A.</div></div></div>
          <div class="year">1991</div>, <div class="article-title">Molecular analysis of the Ubiquitous (Uq) transposable element system of <em>Zea mays</em></div>, <div class="source">Mol. Gen. Genet.</div> , <div class="volume">230</div>, <div class="fpage">201</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20analysis%20of%20the%20Ubiquitous%20(Uq)%20transposable%20element%20system%20of%20Zea%20mays&amp;author=A.G.%20Pisabarro&amp;author=W.F.%20Martin&amp;author=P.A.%20Peterson&amp;author=H.%20Saedler&amp;author=A.%20Gierl&amp;publication_year=1991&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=230&amp;pages=201-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00290669" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00290669" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00290669"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1660565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Arbuckle</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Recent, extensive, and preferential insertion of members of the miniature inverted-repeat transposable element family Heartbreaker into genic regions of maize</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">97</div>, <div class="fpage">1160</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recent%2C%20extensive%2C%20and%20preferential%20insertion%20of%20members%20of%20the%20miniature%20inverted-repeat%20transposable%20element%20family%20Heartbreaker%20into%20genic%20regions%20of%20maize&amp;author=Q.%20Zhang&amp;author=J.%20Arbuckle&amp;author=S.R.%20Wessler&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=1160-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.3.1160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.3.1160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.3.1160"> </span></div> </div></div></div></div></div><div content-id="DSW011C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">X.Y.</div>
            </div>, <div class="name string-name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Eggleston</div> <div class="given-names">W.B.</div></div>, <div class="name string-name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">P instability factor: an active maize transposon system associated with the amplification of Tourist-like MITEs and a new superfamily of transposases</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">98</div>, <div class="fpage">12572</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=P%20instability%20factor%3A%20an%20active%20maize%20transposon%20system%20associated%20with%20the%20amplification%20of%20Tourist-like%20MITEs%20and%20a%20new%20superfamily%20of%20transposases&amp;author=X.Y.%20Zhang&amp;author=C.%20Feschotte&amp;author=Q.%20Zhang&amp;author=N.%20Jiang&amp;author=W.B.%20Eggleston&amp;author=S.R.%20Wessler&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=12572-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.211442198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.211442198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.211442198"> </span></div> </div></div></div></div></div><div content-id="DSW011C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bhattramakki</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Dolan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hanafey</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Insertion-deletion polymorphisms in 3 ‘ regions of maize genes occur frequently and can be used as highly informative genetic markers</div>, <div class="source">Plant Mol. Biol.</div> , <div class="volume">48</div>, <div class="fpage">539</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion-deletion%20polymorphisms%20in%203%20%E2%80%98%20regions%20of%20maize%20genes%20occur%20frequently%20and%20can%20be%20used%20as%20highly%20informative%20genetic%20markers&amp;author=D.%20Bhattramakki&amp;author=M.%20Dolan&amp;author=M.%20Hanafey&amp;publication_year=2002&amp;journal=Plant%20Mol.%20Biol.&amp;volume=48&amp;pages=539-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1014841612043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1014841612043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1014841612043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12004893" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zou</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Mackaluso</div> <div class="given-names">J.D.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Cis-regulatory code of stress-responsive transcription in Arabidopsis thaliana</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">14992</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cis-regulatory%20code%20of%20stress-responsive%20transcription%20in%20Arabidopsis%20thaliana&amp;author=C.%20Zou&amp;author=K.%20Sun&amp;author=J.D.%20Mackaluso&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=14992-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1103202108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1103202108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1103202108"> </span></div> </div></div></div></div></div><div content-id="DSW011C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">C.R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Burns</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Boeke</div> <div class="given-names">J.D.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Active transposition in genomes</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">46</div>, <div class="fpage">651</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20transposition%20in%20genomes&amp;author=C.R.L.%20Huang&amp;author=K.H.%20Burns&amp;author=J.D.%20Boeke&amp;publication_year=2012&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=46&amp;pages=651" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-110711-155616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-110711-155616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-110711-155616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23145912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dean</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Van Kan</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Pretorius</div> <div class="given-names">Z.A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">The Top 10 fungal pathogens in molecular plant pathology</div>, <div class="source">Mol. Plant Pathol.</div> , <div class="volume">13</div>, <div class="fpage">414</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Top%2010%20fungal%20pathogens%20in%20molecular%20plant%20pathology&amp;author=R.%20Dean&amp;author=J.A.%20Van%20Kan&amp;author=Z.A.%20Pretorius&amp;publication_year=2012&amp;journal=Mol.%20Plant%20Pathol.&amp;volume=13&amp;pages=414-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1364-3703.2011.00783.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1364-3703.2011.00783.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1364-3703.2011.00783.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22471698" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Triplett</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Leach</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.-L.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Novel insights into rice innate immunity against bacterial and fungal pathogens</div>, <div class="source">Annu. Rev. Phytopathol.</div> , <div class="volume">52</div>, <div class="fpage">213</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20insights%20into%20rice%20innate%20immunity%20against%20bacterial%20and%20fungal%20pathogens&amp;author=W.%20Liu&amp;author=J.%20Liu&amp;author=L.%20Triplett&amp;author=J.E.%20Leach&amp;author=G.-L.%20Wang&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Phytopathol.&amp;volume=52&amp;pages=213-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-phyto-102313-045926" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-phyto-102313-045926" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-phyto-102313-045926"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24906128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The Magnaporthe oryzae avirulence gene AvrPiz-t encodes a predicted secreted protein that triggers the immunity in rice mediated by the blast resistance gene Piz-t</div>, <div class="source">Mol. Plant Microbe Interact.</div> , <div class="volume">22</div>, <div class="fpage">411</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Magnaporthe%20oryzae%20avirulence%20gene%20AvrPiz-t%20encodes%20a%20predicted%20secreted%20protein%20that%20triggers%20the%20immunity%20in%20rice%20mediated%20by%20the%20blast%20resistance%20gene%20Piz-t&amp;author=W.%20Li&amp;author=B.%20Wang&amp;author=J.%20Wu&amp;publication_year=2009&amp;journal=Mol.%20Plant%20Microbe%20Interact.&amp;volume=22&amp;pages=411-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-22-4-0411" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-22-4-0411" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-22-4-0411"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19271956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Way</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Kleckner</div> <div class="given-names">N.</div></div></div>
          <div class="year">1985</div>, <div class="article-title">Transposition of plasmid-borne Tn10 elements does not exhibit simple length-dependence</div>, <div class="source">Genetics</div> , <div class="volume">111</div>, <div class="fpage">705</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposition%20of%20plasmid-borne%20Tn10%20elements%20does%20not%20exhibit%20simple%20length-dependence&amp;author=J.C.%20Way&amp;author=N.%20Kleckner&amp;publication_year=1985&amp;journal=Genetics&amp;volume=111&amp;pages=705-13" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2998922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hennig</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Ziebuhr</div> <div class="given-names">W.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">A transposase-independent mechanism gives rise to precise excision of IS256 from insertion sites in <em>Staphylococcus epidermidis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="volume">190</div>, <div class="fpage">1488</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20transposase-independent%20mechanism%20gives%20rise%20to%20precise%20excision%20of%20IS256%20from%20insertion%20sites%20in%20Staphylococcus%20epidermidis&amp;author=S.%20Hennig&amp;author=W.%20Ziebuhr&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=1488-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01290-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01290-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01290-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18065530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW011C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW011C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Ferguson</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Transposition of a rice Mutator-like element in the yeast <em>Saccharomyces cerevisiae</em></div>, <div class="source">Plant Cell</div> , <div class="volume">27</div>, <div class="fpage">132</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposition%20of%20a%20rice%20Mutator-like%20element%20in%20the%20yeast%20Saccharomyces%20cerevisiae&amp;author=D.%20Zhao&amp;author=A.%20Ferguson&amp;author=N.%20Jiang&amp;publication_year=2015&amp;journal=Plant%20Cell&amp;volume=27&amp;pages=132-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.128488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.128488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.128488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25587002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745420&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw011/1/dsw011_Supplementary_Data.zip?Expires=2147483647&amp;Signature=4Bj0u5MCtq7hcsfnsFLuC9dTW8GYqDgfCKTWsNvSLuoxd7PPIltJKSWtccq8DgCPP6pM2Cn~Zohm77qIajAIugbtoGiqM8YYM2iwz6xJHXAKAoj2wQkozrvYmgCpfcfugae2ER0pLJjeAgjXmF7VGhsCDeatqyE-VjMz85iEV1FuTUoxyWOf5xfYPGt1jbg~RJ6gXmaHmLMP6bGImRUqB1PAVXSG53KKF75w8PXwA3lHdUu40y5n0FJx-5Agd670Mory3UqqCH84JBkWGxAik-aHk57Ygg8GwTdh8RjjhRUQJLPXxA-ZUBV9ijY~Fd-aTa6d1fT3qiQsfj-JpolDdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">474</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw011"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745420"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f241%2f1745420">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F241%2F1745420" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745420'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw011");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f241%2f1745420">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

