

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/6/661/2754550","@type":"ScholarlyArticle","name":"Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template","datePublished":"2014-09-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2014-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu028","keywords":["whole-genome amplification","AT-rich","malaria","tetramethylammonium chloride"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/6/661/2754550","author":[{"name":"Oyola, Samuel O.","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Manske, Magnus","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Campino, Susana","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Claessens, Antoine","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Hamilton, William L.","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Kekre, Mihir","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Drury, Eleanor","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Mead, Daniel","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Gu, Yong","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Miles, Alistair","affiliation":"2  MRC Centre for Genomics and Global Health ,  University of Oxford ,  Oxford OX3 7BN ,  UK    3  Wellcome Trust Centre for Human Genetics ,  University of Oxford ,  Oxford OX3 7BN ,  UK","@type":"Person"},{"name":"MacInnis, Bronwyn","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK    2  MRC Centre for Genomics and Global Health ,  University of Oxford ,  Oxford OX3 7BN ,  UK","@type":"Person"},{"name":"Newbold, Chris","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK    4  Weatherall Institute of Molecular Medicine ,  University of Oxford ,  Oxford OX3 9DS ,  UK","@type":"Person"},{"name":"Berriman, Matthew","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK","@type":"Person"},{"name":"Kwiatkowski, Dominic P.","affiliation":"1  Wellcome Trust Sanger Institute ,  Hinxton ,  UK    2  MRC Centre for Genomics and Global Health ,  University of Oxford ,  Oxford OX3 7BN ,  UK    3  Wellcome Trust Centre for Human Genetics ,  University of Oxford ,  Oxford OX3 7BN ,  UK","@type":"Person"}],"description":"Pathogen genome sequencing directly from clinical samples is quickly gaining importance in genetic and medical research studies. However, low DNA yield from blo","pageStart":"661","pageEnd":"671","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template" />
    <meta property="og:description" content="Pathogen genome sequencing directly from clinical samples is quickly gaining importance in genetic and medical research studies. However, low DNA yield from blo" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/6/661/2754550" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-09-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Oyola, Samuel O." /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Manske, Magnus" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Campino, Susana" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Claessens, Antoine" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Hamilton, William L." /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Kekre, Mihir" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Drury, Eleanor" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Mead, Daniel" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Gu, Yong" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Miles, Alistair" /><meta name="citation_author_institution" content="MRC Centre for Genomics and Global Health, University of Oxford, Oxford OX3 7BN, UK" /><meta name="citation_author_institution" content="Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK" /><meta name="citation_author" content="MacInnis, Bronwyn" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author_institution" content="MRC Centre for Genomics and Global Health, University of Oxford, Oxford OX3 7BN, UK" /><meta name="citation_author" content="Newbold, Chris" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, University of Oxford, Oxford OX3 9DS, UK" /><meta name="citation_author" content="Berriman, Matthew" /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author" content="Kwiatkowski, Dominic P." /><meta name="citation_author_institution" content="Wellcome Trust Sanger Institute, Hinxton, UK" /><meta name="citation_author_institution" content="MRC Centre for Genomics and Global Health, University of Oxford, Oxford OX3 7BN, UK" /><meta name="citation_author_institution" content="Wellcome Trust Centre for Human Genetics, University of Oxford, Oxford OX3 7BN, UK" /><meta name="citation_title" content="Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template" /><meta name="citation_firstpage" content="661" /><meta name="citation_lastpage" content="671" /><meta name="citation_doi" content="10.1093/dnares/dsu028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2014/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Analysis of Plasmodium falciparum diversity in natural infections by deep sequencing;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=487;  citation_pages=375-9; " /><meta name="citation_reference" content="citation_title=Multiple populations of artemisinin-resistant Plasmodium falciparum in Cambodia;  citation_journal_title=Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=648-55; " /><meta name="citation_reference" content="citation_title=Whole genome amplification: abundant supplies of DNA from precious samples or clinical specimens;  citation_journal_title=Trends Biotechnol.;  citation_year=2003;  citation_volume=21;  citation_pages=531-5; " /><meta name="citation_reference" content="citation_title=Optimisation of DNA and RNA extraction from archival formalin-fixed tissue;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=e12" /><meta name="citation_reference" content="citation_title=Sampling and storage of blood and the detection of malaria parasites by polymerase chain reaction;  citation_journal_title=Trans. R. Soc. Trop. Med. Hyg.;  citation_year=1999;  citation_volume=93;  citation_pages=50-3; " /><meta name="citation_reference" content="citation_title=Guidelines for collecting and maintaining archives for genetic monitoring;  citation_journal_title=Conservation Genet. Resour.;  citation_year=2012;  citation_volume=4;  citation_pages=527-36; " /><meta name="citation_reference" content="citation_title=DNA extraction from archival formalin-fixed, paraffin-embedded tissue sections based on the antigen retrieval principle: heating under the influence of pH;  citation_journal_title=J. Histochem. Cytochem.;  citation_year=2002;  citation_volume=50;  citation_pages=1005-11; " /><meta name="citation_reference" content="citation_title=Short report: rapid DNA extraction from archive blood spots on filter paper for genotyping of Plasmodium falciparum;  citation_journal_title=Am. J. Trop. Med. Hyg.;  citation_year=2005;  citation_volume=72;  citation_pages=249-51; " /><meta name="citation_reference" content="citation_title=Long-term storage limits PCR-based analyses of malaria parasites in archival dried blood spots;  citation_journal_title=Malar. J.;  citation_year=2012;  citation_volume=11;  citation_pages=339" /><meta name="citation_reference" content="citation_title=Determination of genetic variation within Plasmodium falciparum by using enzymatically amplified DNA from filter paper disks impregnated with whole blood;  citation_journal_title=J. Clin. Microbiol.;  citation_year=1991;  citation_volume=29;  citation_pages=1171-4; " /><meta name="citation_reference" content="citation_title=Whole-genome multiple displacement amplification from single cells;  citation_journal_title=Nat. Protoc.;  citation_year=2006;  citation_volume=1;  citation_pages=1965-70; " /><meta name="citation_reference" content="citation_title=Comprehensive human genome amplification using multiple displacement amplification;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=5261-6; " /><meta name="citation_reference" content="citation_title=Rapid amplification of plasmid and phage DNA using Phi 29 DNA polymerase and multiply-primed rolling circle amplification;  citation_journal_title=Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1095-9; " /><meta name="citation_reference" content="citation_title=Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer;  citation_journal_title=Genomics;  citation_year=1992;  citation_volume=13;  citation_pages=718-25; " /><meta name="citation_reference" content="citation_title=Whole genome amplification from a single cell: implications for genetic analysis;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=5847-51; " /><meta name="citation_reference" content="citation_title=Genomic DNA amplification by the multiple displacement amplification (MDA) method;  citation_journal_title=Biochem. Soc. Trans.;  citation_year=2009;  citation_volume=37;  citation_pages=450-3; " /><meta name="citation_reference" content="citation_title=Active DNA unwinding dynamics during processive DNA replication;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=8115-20; " /><meta name="citation_reference" content="citation_title=Ramification amplification: a novel isothermal DNA amplification method;  citation_journal_title=Mol. Diagn.;  citation_year=2001;  citation_volume=6;  citation_pages=141-50; " /><meta name="citation_reference" content="citation_title=Assessment of whole genome amplification-induced bias through high-throughput, massively parallel whole genome sequencing;  citation_journal_title=BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=216" /><meta name="citation_reference" content="citation_title=Impact of whole genome amplification on analysis of copy number variants;  citation_journal_title=Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=e80" /><meta name="citation_reference" content="citation_title=Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes;  citation_journal_title=Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=291-5; " /><meta name="citation_reference" content="citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Erythrocyte binding protein PfRH5 polymorphisms determine species-specific pathways of Plasmodium falciparum invasion;  citation_journal_title=Cell Host Microbe;  citation_year=2008;  citation_volume=4;  citation_pages=40-51; " /><meta name="citation_reference" content="citation_title=Characterization of whole genome amplified (WGA) DNA for use in genotyping assay development;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=217" /><meta name="citation_reference" content="citation_title=Optimization and evaluation of single-cell whole-genome multiple displacement amplification;  citation_journal_title=Hum. Mutat.;  citation_year=2006;  citation_volume=27;  citation_pages=496-503; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=Mechanism of chimera formation during the multiple displacement amplification reaction;  citation_journal_title=BMC Biotechnol.;  citation_year=2007;  citation_volume=7;  citation_pages=19" /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=Genome-wide detection of single-nucleotide and copy-number variations of a single human cell;  citation_journal_title=Science;  citation_year=2012;  citation_volume=338;  citation_pages=1622-6; " /><meta name="citation_reference" content="citation_title=Whole genome amplificationâ€”the solution for a common problem in forensic casework?;  citation_journal_title=Int. Congr. Ser.;  citation_year=2004;  citation_volume=12,61;  citation_pages=24-6; " /><meta name="citation_reference" content="citation_title=Single-cell genomics for dissection of complex malaria infections;  citation_journal_title=Genome Res.;  citation_year=2014;  citation_volume=24;  citation_pages=1028-38; " /><meta name="citation_reference" content="citation_title=Efficient depletion of host DNA contamination in malaria clinical sequencing;  citation_journal_title=J. Clin. Microbiol.;  citation_year=2013;  citation_volume=51;  citation_pages=745-51; " /><meta name="citation_reference" content="citation_title=A method for selectively enriching microbial DNA from contaminating vertebrate host DNA;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e76096" /><meta name="citation_reference" content="citation_title=Using CF11 cellulose columns to inexpensively and effectively remove human DNA from Plasmodium falciparum-infected whole blood samples;  citation_journal_title=Malar. J.;  citation_year=2012;  citation_volume=11;  citation_pages=41" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/6/661/17462958/dsu028.pdf" /><meta name="description" content="Pathogen genome sequencing directly from clinical samples is quickly gaining importance in genetic and medical research studies. However, low DNA yield from blo" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/6/661/2754550" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/6/661/2754550" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/6/661/2754550");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu028';
    </script>
<title>
	Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754550" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaFkPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPeW9sYSwgU2FtdWVsIE8uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYW5za2UsIE1hZ251cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldlbGxjb21lIFRydXN0IFNhbmdlciBJbnN0aXR1dGUsIEhpbnh0b24sIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FtcGlubywgU3VzYW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDbGFlc3NlbnMsIEFudG9pbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJXZWxsY29tZSBUcnVzdCBTYW5nZXIgSW5zdGl0dXRlLCBIaW54dG9uLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhbWlsdG9uLCBXaWxsaWFtIEwuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLZWtyZSwgTWloaXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJXZWxsY29tZSBUcnVzdCBTYW5nZXIgSW5zdGl0dXRlLCBIaW54dG9uLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRydXJ5LCBFbGVhbm9yIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNZWFkLCBEYW5pZWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJXZWxsY29tZSBUcnVzdCBTYW5nZXIgSW5zdGl0dXRlLCBIaW54dG9uLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1LCBZb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaWxlcywgQWxpc3RhaXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNUkMgQ2VudHJlIGZvciBHZW5vbWljcyBhbmQgR2xvYmFsIEhlYWx0aCwgVW5pdmVyc2l0eSBvZiBPeGZvcmQsIE94Zm9yZCBPWDMgN0JOLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldlbGxjb21lIFRydXN0IENlbnRyZSBmb3IgSHVtYW4gR2VuZXRpY3MsIFVuaXZlcnNpdHkgb2YgT3hmb3JkLCBPeGZvcmQgT1gzIDdCTiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWNJbm5pcywgQnJvbnd5biIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldlbGxjb21lIFRydXN0IFNhbmdlciBJbnN0aXR1dGUsIEhpbnh0b24sIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTVJDIENlbnRyZSBmb3IgR2Vub21pY3MgYW5kIEdsb2JhbCBIZWFsdGgsIFVuaXZlcnNpdHkgb2YgT3hmb3JkLCBPeGZvcmQgT1gzIDdCTiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOZXdib2xkLCBDaHJpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldlbGxjb21lIFRydXN0IFNhbmdlciBJbnN0aXR1dGUsIEhpbnh0b24sIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VhdGhlcmFsbCBJbnN0aXR1dGUgb2YgTW9sZWN1bGFyIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE94Zm9yZCwgT3hmb3JkIE9YMyA5RFMsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmVycmltYW4sIE1hdHRoZXciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJXZWxsY29tZSBUcnVzdCBTYW5nZXIgSW5zdGl0dXRlLCBIaW54dG9uLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt3aWF0a293c2tpLCBEb21pbmljIFAuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgSGlueHRvbiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNUkMgQ2VudHJlIGZvciBHZW5vbWljcyBhbmQgR2xvYmFsIEhlYWx0aCwgVW5pdmVyc2l0eSBvZiBPeGZvcmQsIE94Zm9yZCBPWDMgN0JOLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IldlbGxjb21lIFRydXN0IENlbnRyZSBmb3IgSHVtYW4gR2VuZXRpY3MsIFVuaXZlcnNpdHkgb2YgT3hmb3JkLCBPeGZvcmQgT1gzIDdCTiwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik9wdGltaXplZCBXaG9sZS1HZW5vbWUgQW1wbGlmaWNhdGlvbiBTdHJhdGVneSBmb3IgRXh0cmVtZWx5IEFULUJpYXNlZCBUZW1wbGF0ZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjY2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gZGl2ZXJzaXR5IGluIG5hdHVyYWwgaW5mZWN0aW9ucyBieSBkZWVwIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODc7ICBjaXRhdGlvbl9wYWdlcz0zNzUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcG9wdWxhdGlvbnMgb2YgYXJ0ZW1pc2luaW4tcmVzaXN0YW50IFBsYXNtb2RpdW0gZmFsY2lwYXJ1bSBpbiBDYW1ib2RpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz02NDgtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlIGdlbm9tZSBhbXBsaWZpY2F0aW9uOiBhYnVuZGFudCBzdXBwbGllcyBvZiBETkEgZnJvbSBwcmVjaW91cyBzYW1wbGVzIG9yIGNsaW5pY2FsIHNwZWNpbWVuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltaXNhdGlvbiBvZiBETkEgYW5kIFJOQSBleHRyYWN0aW9uIGZyb20gYXJjaGl2YWwgZm9ybWFsaW4tZml4ZWQgdGlzc3VlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz1lMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYW1wbGluZyBhbmQgc3RvcmFnZSBvZiBibG9vZCBhbmQgdGhlIGRldGVjdGlvbiBvZiBtYWxhcmlhIHBhcmFzaXRlcyBieSBwb2x5bWVyYXNlIGNoYWluIHJlYWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmFucy4gUi4gU29jLiBUcm9wLiBNZWQuIEh5Zy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz01MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HdWlkZWxpbmVzIGZvciBjb2xsZWN0aW5nIGFuZCBtYWludGFpbmluZyBhcmNoaXZlcyBmb3IgZ2VuZXRpYyBtb25pdG9yaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db25zZXJ2YXRpb24gR2VuZXQuIFJlc291ci47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTUyNy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIGV4dHJhY3Rpb24gZnJvbSBhcmNoaXZhbCBmb3JtYWxpbi1maXhlZCwgcGFyYWZmaW4tZW1iZWRkZWQgdGlzc3VlIHNlY3Rpb25zIGJhc2VkIG9uIHRoZSBhbnRpZ2VuIHJldHJpZXZhbCBwcmluY2lwbGU6IGhlYXRpbmcgdW5kZXIgdGhlIGluZmx1ZW5jZSBvZiBwSDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSGlzdG9jaGVtLiBDeXRvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0xMDA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG9ydCByZXBvcnQ6IHJhcGlkIEROQSBleHRyYWN0aW9uIGZyb20gYXJjaGl2ZSBibG9vZCBzcG90cyBvbiBmaWx0ZXIgcGFwZXIgZm9yIGdlbm90eXBpbmcgb2YgUGxhc21vZGl1bSBmYWxjaXBhcnVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BbS4gSi4gVHJvcC4gTWVkLiBIeWcuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9MjQ5LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nLXRlcm0gc3RvcmFnZSBsaW1pdHMgUENSLWJhc2VkIGFuYWx5c2VzIG9mIG1hbGFyaWEgcGFyYXNpdGVzIGluIGFyY2hpdmFsIGRyaWVkIGJsb29kIHNwb3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NYWxhci4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmF0aW9uIG9mIGdlbmV0aWMgdmFyaWF0aW9uIHdpdGhpbiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gYnkgdXNpbmcgZW56eW1hdGljYWxseSBhbXBsaWZpZWQgRE5BIGZyb20gZmlsdGVyIHBhcGVyIGRpc2tzIGltcHJlZ25hdGVkIHdpdGggd2hvbGUgYmxvb2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENsaW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xMTcxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBtdWx0aXBsZSBkaXNwbGFjZW1lbnQgYW1wbGlmaWNhdGlvbiBmcm9tIHNpbmdsZSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xOTY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGh1bWFuIGdlbm9tZSBhbXBsaWZpY2F0aW9uIHVzaW5nIG11bHRpcGxlIGRpc3BsYWNlbWVudCBhbXBsaWZpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTUyNjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgYW1wbGlmaWNhdGlvbiBvZiBwbGFzbWlkIGFuZCBwaGFnZSBETkEgdXNpbmcgUGhpIDI5IEROQSBwb2x5bWVyYXNlIGFuZCBtdWx0aXBseS1wcmltZWQgcm9sbGluZyBjaXJjbGUgYW1wbGlmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMDk1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZ2VuZXJhdGUgb2xpZ29udWNsZW90aWRlLXByaW1lZCBQQ1I6IGdlbmVyYWwgYW1wbGlmaWNhdGlvbiBvZiB0YXJnZXQgRE5BIGJ5IGEgc2luZ2xlIGRlZ2VuZXJhdGUgcHJpbWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTcxOC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIGFtcGxpZmljYXRpb24gZnJvbSBhIHNpbmdsZSBjZWxsOiBpbXBsaWNhdGlvbnMgZm9yIGdlbmV0aWMgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9NTg0Ny01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBETkEgYW1wbGlmaWNhdGlvbiBieSB0aGUgbXVsdGlwbGUgZGlzcGxhY2VtZW50IGFtcGxpZmljYXRpb24gKE1EQSkgbWV0aG9kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBTb2MuIFRyYW5zLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTQ1MC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3RpdmUgRE5BIHVud2luZGluZyBkeW5hbWljcyBkdXJpbmcgcHJvY2Vzc2l2ZSBETkEgcmVwbGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTgxMTUtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhbWlmaWNhdGlvbiBhbXBsaWZpY2F0aW9uOiBhIG5vdmVsIGlzb3RoZXJtYWwgRE5BIGFtcGxpZmljYXRpb24gbWV0aG9kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIERpYWduLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIHdob2xlIGdlbm9tZSBhbXBsaWZpY2F0aW9uLWluZHVjZWQgYmlhcyB0aHJvdWdoIGhpZ2gtdGhyb3VnaHB1dCwgbWFzc2l2ZWx5IHBhcmFsbGVsIHdob2xlIGdlbm9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcGFjdCBvZiB3aG9sZSBnZW5vbWUgYW1wbGlmaWNhdGlvbiBvbiBhbmFseXNpcyBvZiBjb3B5IG51bWJlciB2YXJpYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9ZTgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1wbGlmaWNhdGlvbi1mcmVlIElsbHVtaW5hIHNlcXVlbmNpbmctbGlicmFyeSBwcmVwYXJhdGlvbiBmYWNpbGl0YXRlcyBpbXByb3ZlZCBtYXBwaW5nIGFuZCBhc3NlbWJseSBvZiAoRytDKS1iaWFzZWQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yOTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemluZyBJbGx1bWluYSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBsaWJyYXJ5IHByZXBhcmF0aW9uIGZvciBleHRyZW1lbHkgQVQtYmlhc2VkIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fcnl0aHJvY3l0ZSBiaW5kaW5nIHByb3RlaW4gUGZSSDUgcG9seW1vcnBoaXNtcyBkZXRlcm1pbmUgc3BlY2llcy1zcGVjaWZpYyBwYXRod2F5cyBvZiBQbGFzbW9kaXVtIGZhbGNpcGFydW0gaW52YXNpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgSG9zdCBNaWNyb2JlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MC01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiB3aG9sZSBnZW5vbWUgYW1wbGlmaWVkIChXR0EpIEROQSBmb3IgdXNlIGluIGdlbm90eXBpbmcgYXNzYXkgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltaXphdGlvbiBhbmQgZXZhbHVhdGlvbiBvZiBzaW5nbGUtY2VsbCB3aG9sZS1nZW5vbWUgbXVsdGlwbGUgZGlzcGxhY2VtZW50IGFtcGxpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTXV0YXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDk2LTUwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFuZGVtIHJlcGVhdHMgZmluZGVyOiBhIHByb2dyYW0gdG8gYW5hbHl6ZSBETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBvZiBjaGltZXJhIGZvcm1hdGlvbiBkdXJpbmcgdGhlIG11bHRpcGxlIGRpc3BsYWNlbWVudCBhbXBsaWZpY2F0aW9uIHJlYWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9tZSBBbmFseXNpcyBUb29sa2l0OiBhIE1hcFJlZHVjZSBmcmFtZXdvcmsgZm9yIGFuYWx5emluZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZGV0ZWN0aW9uIG9mIHNpbmdsZS1udWNsZW90aWRlIGFuZCBjb3B5LW51bWJlciB2YXJpYXRpb25zIG9mIGEgc2luZ2xlIGh1bWFuIGNlbGw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9MTYyMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZSBnZW5vbWUgYW1wbGlmaWNhdGlvbuKAlHRoZSBzb2x1dGlvbiBmb3IgYSBjb21tb24gcHJvYmxlbSBpbiBmb3JlbnNpYyBjYXNld29yaz87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gQ29uZ3IuIFNlci47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTIsNjE7ICBjaXRhdGlvbl9wYWdlcz0yNC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUtY2VsbCBnZW5vbWljcyBmb3IgZGlzc2VjdGlvbiBvZiBjb21wbGV4IG1hbGFyaWEgaW5mZWN0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGVwbGV0aW9uIG9mIGhvc3QgRE5BIGNvbnRhbWluYXRpb24gaW4gbWFsYXJpYSBjbGluaWNhbCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTUxOyAgY2l0YXRpb25fcGFnZXM9NzQ1LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1ldGhvZCBmb3Igc2VsZWN0aXZlbHkgZW5yaWNoaW5nIG1pY3JvYmlhbCBETkEgZnJvbSBjb250YW1pbmF0aW5nIHZlcnRlYnJhdGUgaG9zdCBETkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzYwOTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBDRjExIGNlbGx1bG9zZSBjb2x1bW5zIHRvIGluZXhwZW5zaXZlbHkgYW5kIGVmZmVjdGl2ZWx5IHJlbW92ZSBodW1hbiBETkEgZnJvbSBQbGFzbW9kaXVtIGZhbGNpcGFydW0taW5mZWN0ZWQgd2hvbGUgYmxvb2Qgc2FtcGxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFsYXIuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNi82NjEvMTc0NjI5NTgvZHN1MDI4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iUGF0aG9nZW4gZ2Vub21lIHNlcXVlbmNpbmcgZGlyZWN0bHkgZnJvbSBjbGluaWNhbCBzYW1wbGVzIGlzIHF1aWNrbHkgZ2FpbmluZyBpbXBvcnRhbmNlIGluIGdlbmV0aWMgYW5kIG1lZGljYWwgcmVzZWFyY2ggc3R1ZGllcy4gSG93ZXZlciwgbG93IEROQSB5aWVsZCBmcm9tIGJsbyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjc1NDU1MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRque/yFZpHnyumu62+jPY39TIGCw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f661%2f2754550" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f661%2f2754550" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=22047" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&amp;Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/6">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197843">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197843">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197848">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197848">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197857">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197857">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197875">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197875">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197882">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197882">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197884">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197884">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197886">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197889">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/6/649/2754548">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/6/673/2754551">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Samuel O. Oyola</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +44 1223-494-994. Fax. +44 1223-494-919. Email: <a href="mailto:so1@sanger.ac.uk" target="_blank">so1@sanger.ac.uk</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Samuel+O.+Oyola">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oyola S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oyola S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Samuel O. Oyola</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Magnus Manske</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Magnus+Manske">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Manske M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Manske M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Magnus Manske</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Susana Campino</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Susana+Campino">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Campino S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Campino S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Susana Campino</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antoine Claessens</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antoine+Claessens">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Claessens A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Claessens A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Antoine Claessens</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">William L. Hamilton</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=William+L.+Hamilton">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hamilton W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hamilton W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">William L. Hamilton</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mihir Kekre</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mihir+Kekre">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kekre M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kekre M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mihir Kekre</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eleanor Drury</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eleanor+Drury">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Drury E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Drury E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eleanor Drury</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniel Mead</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daniel+Mead">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mead D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mead D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Daniel Mead</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Gu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Gu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gu Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yong Gu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alistair Miles</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">MRC Centre for Genomics and Global Health</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 7BN</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Wellcome Trust Centre for Human Genetics</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 7BN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alistair+Miles">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miles A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miles A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Alistair Miles</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bronwyn MacInnis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">MRC Centre for Genomics and Global Health</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 7BN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bronwyn+MacInnis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=MacInnis B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;MacInnis B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Bronwyn MacInnis</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chris Newbold</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Weatherall Institute of Molecular Medicine</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 9DS</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chris+Newbold">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Newbold C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Newbold C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chris Newbold</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Matthew Berriman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Matthew+Berriman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Berriman M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Berriman M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Matthew Berriman</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dominic P. Kwiatkowski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Wellcome Trust Sanger Institute</div>, <div class="addr-line">Hinxton</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">MRC Centre for Genomics and Global Health</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 7BN</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Wellcome Trust Centre for Human Genetics</div>, <div class="insititution">University of Oxford</div>, <div class="addr-line">Oxford OX3 7BN</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dominic+P.+Kwiatkowski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kwiatkowski D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kwiatkowski D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Dominic P. Kwiatkowski</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 661â€“671, <a href='https://doi.org/10.1093/dnares/dsu028'>https://doi.org/10.1093/dnares/dsu028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 September 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 June 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">22 August 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754550" href="/dnaresearch/article-pdf/21/6/661/17462958/dsu028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Samuel O. Oyola, Magnus Manske, Susana Campino, Antoine Claessens, William L. Hamilton, Mihir Kekre, Eleanor Drury, Daniel Mead, Yong Gu, Alistair Miles, Bronwyn MacInnis, Chris Newbold, Matthew Berriman, Dominic P. Kwiatkowski;  Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template, <em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 661â€“671, <a href='https://doi.org/10.1093/dnares/dsu028'>https://doi.org/10.1093/dnares/dsu028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754550?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Optimized%20Whole-Genome%20Amplification%20Strategy%20for%20Extremely%20AT-Biased%20Template&amp;publicationDate=2014-09-19&amp;volumeNum=21&amp;issueNum=6&amp;author=Oyola%2C%20Samuel%20O.%3B%20Manske%2C%20Magnus&amp;startPage=661&amp;endPage=671&amp;contentId=10.1093%2Fdnares%2Fdsu028&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f661%2f2754550" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Pathogen genome sequencing directly from clinical samples is quickly gaining importance in genetic and medical research studies. However, low DNA yield from blood-borne pathogens is often a limiting factor. The problem worsens in extremely base-biased genomes such as the AT-rich <em>Plasmodium falciparum</em>. We present a strategy for whole-genome amplification (WGA) of low-yield samples from <em>P. falciparum</em> prior to short-read sequencing. We have developed WGA conditions that incorporate tetramethylammonium chloride for improved amplification and coverage of AT-rich regions of the genome. We show that this method reduces amplification bias and chimera formation. Our data show that this method is suitable for as low as 10 pg input DNA, and offers the possibility of sequencing the parasite genome from small blood samples.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome amplification&quot;">whole-genome amplification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="AT-rich">AT-rich</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="malaria">malaria</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tetramethylammonium chloride&quot;">tetramethylammonium chloride</a></div>                            <h2 scrollto-destination=87197843 id="87197843" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Timely detection of emerging genetic variants and other evolutionary features associated with important clinical phenotypes such as increased virulence and drug resistance are central to malaria control strategies. Genome sequencing of parasite populations has been identified as an effective tool for detecting genetic changes.<sup><a reveal-id="DSU028C1" data-open="DSU028C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU028C2" data-open="DSU028C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Despite the current success in the sequencing technology, there remain significant challenges in achieving global genetic surveillance of parasite populations in the field. Most genome-scale analyses, such as whole-genome sequencing, require large amounts of clean genetic material that is often difficult to obtain,<sup><a reveal-id="DSU028C3" data-open="DSU028C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and therefore a serious impediment to genetic analysis on many clinical samples. A large number of valuable clinical specimens are collected in the form of small samples that yield low quantity and quality of genetic material.<sup><a reveal-id="DSU028C4" data-open="DSU028C4" class="link link-ref link-reveal xref-bibr">4</a>â€“<a reveal-id="DSU028C7" data-open="DSU028C7" class="link link-ref link-reveal xref-bibr">7</a></sup> A common method for collecting clinical samples in the field is through heel/finger-pricks.<sup><a reveal-id="DSU028C5" data-open="DSU028C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSU028C7" data-open="DSU028C7" class="link link-ref link-reveal xref-bibr">7</a>â€“<a reveal-id="DSU028C10" data-open="DSU028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> However, the quantity and quality of parasite genetic material that can be extracted from these small blood samples usually fall below the threshold required by genome sequencing platforms.</p><p>To alleviate the problem of low DNA quantities, whole-genome amplification (WGA) is now routinely applied in many applications,<sup><a reveal-id="DSU028C3" data-open="DSU028C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU028C11" data-open="DSU028C11" class="link link-ref link-reveal xref-bibr">11</a></sup> but has yet to be optimized for use in genomes of extreme base composition such as <em>Plasmodium falciparum</em>. Two major forms of WGA have been described: multiple displacement amplification (MDA)<sup><a reveal-id="DSU028C12" data-open="DSU028C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU028C13" data-open="DSU028C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and PCR-based amplification methods.<sup><a reveal-id="DSU028C14" data-open="DSU028C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU028C15" data-open="DSU028C15" class="link link-ref link-reveal xref-bibr">15</a></sup> MDA has been the method of choice for a wider range of genome amplification studies, because it produces longer DNA products with extensive genome coverage.<sup><a reveal-id="DSU028C16" data-open="DSU028C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>MDA is based on Ï†29 polymerase, which, in the presence of random hexamers annealed to denatured DNA, uses an MDA mechanism to synthesize high-molecular-weight DNA from very minute amounts of input material under isothermal conditions.<sup><a reveal-id="DSU028C17" data-open="DSU028C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU028C18" data-open="DSU028C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The best results, however, have been obtained from genomes with relatively balanced base composition.<sup><a reveal-id="DSU028C11" data-open="DSU028C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU028C19" data-open="DSU028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU028C20" data-open="DSU028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Amplification of genomes with imbalanced base composition, such as the AT-rich <em>P. falciparum</em>, has remained a challenge.<sup><a reveal-id="DSU028C21" data-open="DSU028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSU028C22" data-open="DSU028C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>In this study, we sought to identify and optimize a WGA system suitable for an AT-base-biased genome of <em>P. falciparum</em>. Using standard conditions as outlined for each system, we tested the efficiency of non-MDA- and MDA-based methods. Initial findings showed that MDA-based systems produced a more uniform genome coverage than non-MDA methods (data not shown). We have optimized an identified MDA system to produce an improved genome coverage and a reduced base-bias with more accurate genome representation. We show that our optimized WGA conditions are suitable for as low as 10 picograms (pg) <em>P. falciparum</em> input DNA, producing high-sequence concordance with unamplified genomic DNA. This development promises a significant tool to aid implementation of the global genetic surveillance of parasite populations through small blood sample sequencing.</p>                            <h2 scrollto-destination=87197848 id="87197848" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87197849 id="87197849" class="section-title" data-legacy-id=s2a>2.1. DNA samples</h3>
<p><em>Plasmodium falciparum</em> 3D7 genomic DNA was a gift from Prof. Chris Newbold (University of Oxford). The clinical isolates were obtained from the Malaria Genetics Group's Sequencing Sample Repository at the Wellcome Trust Sanger Institute. Other genomic DNA was extracted from 17 progeny clones of <em>P. falciparum</em> strains derived from genetic cross between 7G8xGB4<sup><a reveal-id="DSU028C23" data-open="DSU028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and a 3D7 strain (3D7_glasgow).</p>                            <h3 scrollto-destination=87197851 id="87197851" class="section-title" data-legacy-id=s2b>2.2. Whole-genome amplification</h3>
<p>All non-MDA WGA were performed following individual kit manufacturer's instructions. MDA-based WGA was performed using either REPLI-g Mini kit (Qiagen) or Genomiphi kit (GE Healthcare). For Genomiphi, the kit manufacturer's instructions were followed without modification. For the REPLI-g Mini kit, manufacturer's instructions were followed during preliminary tests. The following modifications were performed in developing optimized conditions for the REPLI-g Mini kit: nuclease-free water and all tubes were UV-treated before use. WGA reactions were performed in 0.2 ml PCR tubes. Buffer D1 stock solution (Qiagen) was reconstituted by adding 500 Âµl of nuclease-free water, and a working solution was prepared by mixing the stock solution and nuclease-free water in the ratio of 1 : 3.5, respectively. Unmodified Buffer N1 was reconstituted by mixing Stop solution (Qiagen) and nuclease-free water in the ratio of 1 : 5.7. Modified buffer N1 was prepared by including tetramethylammonium chloride (TMAC) at a concentration of 300 mM. To denature DNA templates, 5 Âµl of the DNA solution was mixed with 5 Âµl of buffer D1 (working solution prepared as described above). The mixture was vortexed and centrifuged briefly before incubating at room temperature for 3 min. Denatured DNA was neutralized by adding 10 Âµl of either unmodified or modified buffer N1. Neutralized DNA was mixed by vortexing and centrifuged briefly. To amplify the DNA template, denatured and neutralized sample was mixed with 29 Âµl of REPLI-g Mini Reaction Buffer and 1 Âµl of REPLI-g Mini DNA polymerase to obtain a final reaction volume of 50 Âµl. The reaction mixture was incubated at 30Â°C for 16 h using an MJ Research PTC-225 thermal cycling system (GMI, Inc., USA) with the heating lid set to track at +5Â°C. Amplified DNA was cleaned using Agencourt Ampure XP beads (Beckman Coulter) using sample to beads ratio of 1 : 1 and eluted with 50 Âµl of EB (Qiagen).</p>                            <h3 scrollto-destination=87197853 id="87197853" class="section-title" data-legacy-id=s2c>2.3. Illumina library preparation and sequencing</h3>
<p>All sequencing libraries were prepared as PCR-free. Whole-genome amplified or unamplified genomic DNA (1.5 Î¼g in 75 Âµl of TE buffer) was sheared using a Covaris S2 (Covaris, Inc., Woburn, MA, USA) to obtain a fragment-size distribution of âˆ¼300 to âˆ¼600 bp. The sheared DNA fragments were end-repaired and A-tailed using the NEBNext DNA sample preparation kit (NEB), following an Illumina sample preparation protocol. Pre-annealed paired-end Illumina PCR-free adapters were ligated to the A-tailed fragments in a 50-Âµl reaction containing 10 Âµl of DNA sample, 1Ã— Quick T4 DNA ligase buffer, 10 Âµl of PCR-free PE-adapter mixture, 5 Âµl of Quick T4 DNA ligase (NEB) and incubated at 20Â°C for 30 min. The ligation reaction was cleaned twice using Agencourt Ampure XP beads (Beckman Coulter). Cleaned DNA was eluted with 20 Âµl of buffer EB. Aliquots were analysed using an Agilent 2100 Bioanalyzer (Agilent Technologies) to determine the size distribution and to check for adapter contamination. Samples were sequenced using either Illumina Hiseq 2500 or Miseq technologies (San Diego, CA, USA) with 75 bp read length and the paired-end read options. Corresponding WGA and non-WGA samples were run in the same lanes, using different multiplex tags. This strategy reduces potential confounding artefacts relating to sequencing chemistry.</p>                            <h3 scrollto-destination=87197855 id="87197855" class="section-title" data-legacy-id=s2d>2.4. Read mapping and genotype concordance analysis</h3>
<p>Reads were mapped against the <em>P. falciparum</em> reference sequence (<a class="link link-uri" href="http://plasmodb.org/common/downloads/release-10.0/Pfalciparum3D7/fasta/data/">http://plasmodb.org/common/downloads/release-10.0/Pfalciparum3D7/fasta/data/</a>), using BWA (V0.6.2). We analysed the genotype calls from 17 samples derived from the progeny clones of genetic crosses of 7G8xGB4 laboratory strains,<sup><a reveal-id="DSU028C23" data-open="DSU028C23" class="link link-ref link-reveal xref-bibr">23</a></sup> including one parent reference strain 3D7 as a control. For genotype concordance analysis, we generated <em>de novo</em> variation calls using samtools (V0.1.1.19) mpileup and bfctools (V0.1.17), calling on both WGA and non-WGA samples. For <em>in silico</em> genotyping, we used a list of 20 737 high-quality single-nucleotide polymorphism (SNP) positions and alleles generated from the sequence of genetic crosses (unpublished data). We performed <em>in silico</em> genotyping of both the WGA and non-WGA samples using samtools mpileup and counting alleles present in at least five reads.</p>                            <h2 scrollto-destination=87197857 id="87197857" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197858 id="87197858" class="section-title" data-legacy-id=s3a>3.1. WGA using standard methods</h3>
<div class="block-child-p">We performed tests on various WGA systems to identify those suitable for AT-rich <em>P. falciparum</em> genome. We compared non-MDAâ€”Endcore-Rapid (NuGen), MALBAC (Yikon Genomics), Rapisome (Biohelix) and MDAâ€”Repli_g (Qiagen) and Genomiphi (GE Healthcare) systems. Our preliminary data (not shown) indicated that non-MDA methods were less tolerant to the AT-biased genome compared with their MDA counterparts. With non-MDA amplification, we observed excess bias towards regions with high GC content and poor coverage of high AT regions of the genome resulting into high levels of allele dropout. Further optimization focused on the MDA systems. Repli_g and Genomiphi are the most commonly used methods for MDA. Although both kits have shown similar results in many systems,<sup><a reveal-id="DSU028C24" data-open="DSU028C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU028C25" data-open="DSU028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> the majority of studies have been with genomes of relatively balanced base composition. Here, we have assessed the performance of Repli_g and Genomiphi by amplifying an AT-rich <em>P. falciparum</em> genome. Amplification of pure <em>P. falciparum</em> 3D7 genomic DNA by both kits produced very similar results as assessed by genome coverage analysis metrics. However, amplification of host-contaminated clinical samples produced dissimilar results in terms of base composition and overall genome coverage (Fig.Â <a reveal-id="DSU028F1" data-open="DSU028F1" class="link link-reveal link-table xref-fig">1</a>). Whereas amplification products generated by both Repli_g and Genomiphi did not maintain the exact original host/parasite DNA proportions, Genomiphi products were more biased towards the host DNA (Fig.Â <a reveal-id="DSU028F1" data-open="DSU028F1" class="link link-reveal link-table xref-fig">1</a>), an observation suggesting bias towards templates of neutral base composition originating from the host genome. Based on the level of bias, we chose to optimize Repli_g for amplification of the <em>P. falciparum</em> genome. <div data-id="DSU028F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02801.jpeg?Expires=2147483647&Signature=guucx4~ntJvRpeoLHDysiBfduSg60Ex3-UTGbjou6Nao-Wa7F7t6JP~OpHqICMQ3hD-RvZX2HNNP1cboZn7tLMUTn~To4dryMqwCQkhNKy3l1F22Y9Q2kjQIxwhPr9RDO2uJfNJ-b3Duc6-Tw-edueobKjZziy-xBJeyLhjuxAyjXXZSRri46-ejcb9pmiB02Xug2BJNO9Fp2T1FabxFnJ13yQIpO-vzpkYaxT603GDkDkl1vV20LWWNvNGds7lVP6v2M~OmWswXBnzpWpD6VPsrfEYQjm9cDghV7h4W-rAbWZcffG3gebBEospJ9rML4yvqkatWh5mYFjYr6n2jeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lorenz curves of genome coverage analysis. Cumulative fraction of sequence reads against cumulative fraction of genome covered is shown for each sample. The diagonal line represents an ideal perfect uniform coverage. The further the sample curves deviate from the diagonal line the more bias in genome coverage they are. Repli_g (solid lines) and Genomiphi (dashed lines) sample pairs were normalized to equal number of reads. Repli_g and Genomiphi amplification of P. falciparum 3D7 genomic DNA (3D7) resulted in identical coverage uniformity as shown in overlapping solid red (Repli_g_3D7) and dashed red (Genomiphi_3D7) curves, respectively. Amplification of clinical samples with varied proportions of human DNA contaminations produced different uniformity of coverage with Genomiphi (dashed lines) showing slightly more bias than their corresponding Repli_g (solid lines) amplified samples. 3D7, pure P. falciparum genomic DNA; PK76, clinical sample with 53% host DNA contamination amplified with either Genomiphi (Genomiphi_PK76) or Repli_g (Regpli_g_PK76); PD21, a clinical sample with 7% host contamination amplified with either Genomiphi (Genomiphi_PD21) or Repli_g (Regpli_g_PD21); PK67, a clinical sample with 19% host contamination amplified with either Genomiphi (Genomiphi_PK67) or Repli_g (Regpli_g_PK67)." path-from-xml="dsu02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197859/dsu02801.jpeg" path-from-xml="dsu02801.jpeg" target="_blank">View large</a><a section="87197859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02801.jpeg?Expires=2147483647&Signature=df915O-BXtQn-WbEnSuvZcvvwYruBEmZuU7SkjKcLdQRkyyqm2xd66Se9wCLkk3iP1ewm41-gEM5x9a97IUH3zDxYyTdjSheACKMzNmEemOFXAcZoAG3RbFYJJ1rXmuBRgN~QkWT6LCLNzga8EU6AY3fd2J9IplRBxM5oqbP0IEP0NfkNgTrxuFVC19n7E3KkEOKcQ92i1SQAndM1L01qUbU6LJJ5i4R0mJTX3Iplh1Rb~ZV5ZQgTI6rZ6nljFe4wUiY9tWK5QGSAyTtsUj7yw7p8ez1tUOWjrqOuq3eGlQrP4YfHFUVBkt8UjSA~7QKEqXHK1V668~vhtN0DF8P6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197859&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lorenz curves of genome coverage analysis. Cumulative fraction of sequence reads against cumulative fraction of genome covered is shown for each sample. The diagonal line represents an ideal perfect uniform coverage. The further the sample curves deviate from the diagonal line the more bias in genome coverage they are. Repli_g (solid lines) and Genomiphi (dashed lines) sample pairs were normalized to equal number of reads. Repli_g and Genomiphi amplification of <em>P. falciparum</em> 3D7 genomic DNA (3D7) resulted in identical coverage uniformity as shown in overlapping solid red (Repli_g_3D7) and dashed red (Genomiphi_3D7) curves, respectively. Amplification of clinical samples with varied proportions of human DNA contaminations produced different uniformity of coverage with Genomiphi (dashed lines) showing slightly more bias than their corresponding Repli_g (solid lines) amplified samples. 3D7, pure <em>P. falciparum</em> genomic DNA; PK76, clinical sample with 53% host DNA contamination amplified with either Genomiphi (Genomiphi_PK76) or Repli_g (Regpli_g_PK76); PD21, a clinical sample with 7% host contamination amplified with either Genomiphi (Genomiphi_PD21) or Repli_g (Regpli_g_PD21); PK67, a clinical sample with 19% host contamination amplified with either Genomiphi (Genomiphi_PK67) or Repli_g (Regpli_g_PK67).</p></div></div><div content-id="DSU028F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02801.jpeg?Expires=2147483647&Signature=guucx4~ntJvRpeoLHDysiBfduSg60Ex3-UTGbjou6Nao-Wa7F7t6JP~OpHqICMQ3hD-RvZX2HNNP1cboZn7tLMUTn~To4dryMqwCQkhNKy3l1F22Y9Q2kjQIxwhPr9RDO2uJfNJ-b3Duc6-Tw-edueobKjZziy-xBJeyLhjuxAyjXXZSRri46-ejcb9pmiB02Xug2BJNO9Fp2T1FabxFnJ13yQIpO-vzpkYaxT603GDkDkl1vV20LWWNvNGds7lVP6v2M~OmWswXBnzpWpD6VPsrfEYQjm9cDghV7h4W-rAbWZcffG3gebBEospJ9rML4yvqkatWh5mYFjYr6n2jeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lorenz curves of genome coverage analysis. Cumulative fraction of sequence reads against cumulative fraction of genome covered is shown for each sample. The diagonal line represents an ideal perfect uniform coverage. The further the sample curves deviate from the diagonal line the more bias in genome coverage they are. Repli_g (solid lines) and Genomiphi (dashed lines) sample pairs were normalized to equal number of reads. Repli_g and Genomiphi amplification of P. falciparum 3D7 genomic DNA (3D7) resulted in identical coverage uniformity as shown in overlapping solid red (Repli_g_3D7) and dashed red (Genomiphi_3D7) curves, respectively. Amplification of clinical samples with varied proportions of human DNA contaminations produced different uniformity of coverage with Genomiphi (dashed lines) showing slightly more bias than their corresponding Repli_g (solid lines) amplified samples. 3D7, pure P. falciparum genomic DNA; PK76, clinical sample with 53% host DNA contamination amplified with either Genomiphi (Genomiphi_PK76) or Repli_g (Regpli_g_PK76); PD21, a clinical sample with 7% host contamination amplified with either Genomiphi (Genomiphi_PD21) or Repli_g (Regpli_g_PD21); PK67, a clinical sample with 19% host contamination amplified with either Genomiphi (Genomiphi_PK67) or Repli_g (Regpli_g_PK67)." path-from-xml="dsu02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197859/dsu02801.jpeg" path-from-xml="dsu02801.jpeg" target="_blank">View large</a><a section="87197859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02801.jpeg?Expires=2147483647&Signature=df915O-BXtQn-WbEnSuvZcvvwYruBEmZuU7SkjKcLdQRkyyqm2xd66Se9wCLkk3iP1ewm41-gEM5x9a97IUH3zDxYyTdjSheACKMzNmEemOFXAcZoAG3RbFYJJ1rXmuBRgN~QkWT6LCLNzga8EU6AY3fd2J9IplRBxM5oqbP0IEP0NfkNgTrxuFVC19n7E3KkEOKcQ92i1SQAndM1L01qUbU6LJJ5i4R0mJTX3Iplh1Rb~ZV5ZQgTI6rZ6nljFe4wUiY9tWK5QGSAyTtsUj7yw7p8ez1tUOWjrqOuq3eGlQrP4YfHFUVBkt8UjSA~7QKEqXHK1V668~vhtN0DF8P6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197859&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Lorenz curves of genome coverage analysis. Cumulative fraction of sequence reads against cumulative fraction of genome covered is shown for each sample. The diagonal line represents an ideal perfect uniform coverage. The further the sample curves deviate from the diagonal line the more bias in genome coverage they are. Repli_g (solid lines) and Genomiphi (dashed lines) sample pairs were normalized to equal number of reads. Repli_g and Genomiphi amplification of <em>P. falciparum</em> 3D7 genomic DNA (3D7) resulted in identical coverage uniformity as shown in overlapping solid red (Repli_g_3D7) and dashed red (Genomiphi_3D7) curves, respectively. Amplification of clinical samples with varied proportions of human DNA contaminations produced different uniformity of coverage with Genomiphi (dashed lines) showing slightly more bias than their corresponding Repli_g (solid lines) amplified samples. 3D7, pure <em>P. falciparum</em> genomic DNA; PK76, clinical sample with 53% host DNA contamination amplified with either Genomiphi (Genomiphi_PK76) or Repli_g (Regpli_g_PK76); PD21, a clinical sample with 7% host contamination amplified with either Genomiphi (Genomiphi_PD21) or Repli_g (Regpli_g_PD21); PK67, a clinical sample with 19% host contamination amplified with either Genomiphi (Genomiphi_PK67) or Repli_g (Regpli_g_PK67).</p></div></div></div>                            <h3 scrollto-destination=87197860 id="87197860" class="section-title" data-legacy-id=s3b>3.2. Optimized WGA conditions reduce amplification bias and improve coverage on low complexity regions</h3>
<div class="block-child-p">Non-coding regions of <em>P. falciparum</em> DNA contain âˆ¼90% A+T base composition. Amplification of AT-rich genomes is a challenge to almost all commercially available polymerases. We have previously shown that addition of TMAC improves coverage of low GC regions of the genome during PCR,<sup><a reveal-id="DSU028C22" data-open="DSU028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> but the same has not been tested with Ï†29, the MDA polymerase. To investigate the effect of TMAC on MDA, we amplified <em>P. falciparum</em> 3D7 genomic DNA of varied input amounts (0.1â€“2 ng) both in the presence and absence of TMAC. We compared the quantity and quality of both products and observed that, like many commercial PCR polymerases, Ï†29 is inhibited by TMAC at certain levels of concentration. We have determined 60 mM as a concentration that is non-inhibitory to the polymerase and optimal for WGA of an AT-biased genome. Although the quantity of the amplification product was higher in the absence (standard procedure) than in the presence of 60 mM TMAC (data not shown), the quality of MDA product, in terms of coverage and base composition, was improved in the optimized procedure where TMAC was added (FigsÂ <a reveal-id="DSU028F2" data-open="DSU028F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSU028F3" data-open="DSU028F3" class="link link-reveal link-table xref-fig">3</a>). As shown in Fig.Â <a reveal-id="DSU028F2" data-open="DSU028F2" class="link link-reveal link-table xref-fig">2</a>, amplification using a standard protocol (Std) resulted in excessive bias in regions of low complexity. Inspection of the over-amplified regions reveals sequences of low complexity and numerous repeat patterns. We used a tandem repeat finder programme<sup><a reveal-id="DSU028C26" data-open="DSU028C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and revealed numerous sequences in tandem repeat conformation that may have affected amplification bias. The top three tandem repeats are provided in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Unlike the standard WGA protocol, our optimized amplification procedure abolished excessive amplification bias and produced a more uniform coverage. The mechanism by which this bias is corrected is not clear, but it is conceivable that TMAC stabilizes and stiffens the DNA backbone, thereby minimizing <em>cis</em>-priming by the looping of displaced DNA strands during amplification. We also show that base composition bias increased with a decrease in the amount of input DNA during amplification with standard protocols. This bias was not observed in samples amplified with the optimized (Opt) conditions (Fig.Â <a reveal-id="DSU028F3" data-open="DSU028F3" class="link link-reveal link-table xref-fig">3</a>). <div data-id="DSU028F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02802.jpeg?Expires=2147483647&amp;Signature=t9nQaGyhySACQmt3S1iTow2gQNazlBHKlQFdVCNiUSHiNNBRqpEnb0Ef9zop74EVHD4j0aw3lNjsEiM2FF14ZPPKVEi8dhCGfC0jOGOmFzUgmrQ7csBm~fMUPcg0DmKEP03CRaxiLkXJ-62Oik2oqn1-vrU1TtZ6yFYThPnrCTFFdtlHKtZ~TVKEQB9pD-QJb-kPs2ZzDuwGktYFZb4VcT8R1ZytE1lV0BRvLqda48V5O6iEuPpX5BffK8rQLJlLF8ww9rIbrM0o~b6Mq1nUVD2NsZtr6rCt3thQ7LTnwHkbwj4x-wl~AIPp-cvzncUuy~8XNtcfbVP57Hza29uDpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LookSeq analysis of coverage uniformity. Plasmodium falciparum 3D7 genomic DNA amplified with Repli_g following standard or optimized procedures. gDNA (top panel) shows bulk genomic DNA sequenced without amplification. All samples amplified using the standard (Std WGA) procedure show regions of amplification bias (marked by *1 and *2, see Supplementary Data), whereas their corresponding counterparts amplified following the optimized procedure (Optimized WGA) show less bias and relatively uniform coverage. Different amounts of input DNA were used for each sample set (Sample 1, 0.1 ng; Sample 2, 0.5 ng; Sample 3, 1 ng)." path-from-xml="dsu02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197861/dsu02802.jpeg" path-from-xml="dsu02802.jpeg" target="_blank">View large</a><a section="87197861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02802.jpeg?Expires=2147483647&amp;Signature=nZB9hpS96sCi3~oNGxCLLe4ATtEiwgjPTqrC06V4~LfmnMuij2kcMQT8KCpa8lDgzVMsUiXlRCPeFlVNOTtcTrZupuP2-2ixH21bWi0DqgdYnPoV-7lb3Ad7dlzo4jigqZIlGwscU2xopiMBpphq3S~RMrz31gLEv4L70-XemV5FCRPOv7AYJunK8JMrkv6-Z~~zYRU-tc3KhOWTGtrkUdrlE5fSw2wEjDUJld0CUsOY7E6ieu5kze1LTfYHxAHe9ijxh-9nf2FBAqpuIuK5y0o-fvR6r~V1XJbop7uAPhkVYJ8IpiQGq2r6-kNFHALxUGWvrmbEVjTeA0Vs5Sw-XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197861&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LookSeq analysis of coverage uniformity. <em>Plasmodium falciparum</em> 3D7 genomic DNA amplified with Repli_g following standard or optimized procedures. gDNA (top panel) shows bulk genomic DNA sequenced without amplification. All samples amplified using the standard (Std WGA) procedure show regions of amplification bias (marked by <strong>*</strong>1 and <strong>*</strong>2, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), whereas their corresponding counterparts amplified following the optimized procedure (Optimized WGA) show less bias and relatively uniform coverage. Different amounts of input DNA were used for each sample set (Sample 1, 0.1 ng; Sample 2, 0.5 ng; Sample 3, 1 ng).</p></div></div><div content-id="DSU028F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02802.jpeg?Expires=2147483647&amp;Signature=t9nQaGyhySACQmt3S1iTow2gQNazlBHKlQFdVCNiUSHiNNBRqpEnb0Ef9zop74EVHD4j0aw3lNjsEiM2FF14ZPPKVEi8dhCGfC0jOGOmFzUgmrQ7csBm~fMUPcg0DmKEP03CRaxiLkXJ-62Oik2oqn1-vrU1TtZ6yFYThPnrCTFFdtlHKtZ~TVKEQB9pD-QJb-kPs2ZzDuwGktYFZb4VcT8R1ZytE1lV0BRvLqda48V5O6iEuPpX5BffK8rQLJlLF8ww9rIbrM0o~b6Mq1nUVD2NsZtr6rCt3thQ7LTnwHkbwj4x-wl~AIPp-cvzncUuy~8XNtcfbVP57Hza29uDpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LookSeq analysis of coverage uniformity. Plasmodium falciparum 3D7 genomic DNA amplified with Repli_g following standard or optimized procedures. gDNA (top panel) shows bulk genomic DNA sequenced without amplification. All samples amplified using the standard (Std WGA) procedure show regions of amplification bias (marked by *1 and *2, see Supplementary Data), whereas their corresponding counterparts amplified following the optimized procedure (Optimized WGA) show less bias and relatively uniform coverage. Different amounts of input DNA were used for each sample set (Sample 1, 0.1 ng; Sample 2, 0.5 ng; Sample 3, 1 ng)." path-from-xml="dsu02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197861/dsu02802.jpeg" path-from-xml="dsu02802.jpeg" target="_blank">View large</a><a section="87197861" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02802.jpeg?Expires=2147483647&amp;Signature=nZB9hpS96sCi3~oNGxCLLe4ATtEiwgjPTqrC06V4~LfmnMuij2kcMQT8KCpa8lDgzVMsUiXlRCPeFlVNOTtcTrZupuP2-2ixH21bWi0DqgdYnPoV-7lb3Ad7dlzo4jigqZIlGwscU2xopiMBpphq3S~RMrz31gLEv4L70-XemV5FCRPOv7AYJunK8JMrkv6-Z~~zYRU-tc3KhOWTGtrkUdrlE5fSw2wEjDUJld0CUsOY7E6ieu5kze1LTfYHxAHe9ijxh-9nf2FBAqpuIuK5y0o-fvR6r~V1XJbop7uAPhkVYJ8IpiQGq2r6-kNFHALxUGWvrmbEVjTeA0Vs5Sw-XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197861&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LookSeq analysis of coverage uniformity. <em>Plasmodium falciparum</em> 3D7 genomic DNA amplified with Repli_g following standard or optimized procedures. gDNA (top panel) shows bulk genomic DNA sequenced without amplification. All samples amplified using the standard (Std WGA) procedure show regions of amplification bias (marked by <strong>*</strong>1 and <strong>*</strong>2, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), whereas their corresponding counterparts amplified following the optimized procedure (Optimized WGA) show less bias and relatively uniform coverage. Different amounts of input DNA were used for each sample set (Sample 1, 0.1 ng; Sample 2, 0.5 ng; Sample 3, 1 ng).</p></div></div><div data-id="DSU028F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02803.jpeg?Expires=2147483647&amp;Signature=YJFzKVnv~MQWqykDMeQnf7C0vJPIblK4FfnxTM7dY4QGZ34aOuIaYiW8S~PjK2bNeVPCRrB8s7jJl7oJSCL8y4Sc7Y6Xi42IrpxWQKgUqS0MubmI6lqSOmIv7pbLozIRlRz4RULxfaRtSWXjVgWMMECxUQWmQNPfxhynj1QQbYAtuQkjKqYn7~WkAeLUqyNU71MORe1FLgqhx~Doj9DnQQyfc3am79FSYITCN0dK0~nokcadXCK9nUUeXkJJiRk0OYhiwQSWcVyrT2jL6OMl1enbws96VYhlkoVVUoGEmciAEVwst1QAGpzE0Nmc7fdxR6I~~nVgaHO9ofCnFypzzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content analysis. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 0.1 to 2 ng as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free and reads obtained were analysed for G+C content profile. A non-WGA sample (gDNA) shows a GC content of âˆ¼19.4%, a profile that was closely matched by products amplified using the optimized procedure (Opt). Samples amplified following the standard procedure (Std) showed biased GC content shown as a shift towards the neutral base composition." path-from-xml="dsu02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197861/dsu02803.jpeg" path-from-xml="dsu02803.jpeg" target="_blank">View large</a><a section="87197861" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02803.jpeg?Expires=2147483647&amp;Signature=Ro3--snlaksDehzI~YH-strQBxsxhLPKs1EMHK-speqoYZzaGXNd88rIJNh8nm56VumJR8eioujaHP~VxXvUMyzvboaWIIX7BnT-zMNW~-KVNqX5EJFOVKzCMbmmAzNc-E63i9LXPDh5G-~0FFRZ3I1V3BkjbwVSXYz00GaUCIZx3q1SXXUQao39PgY9fFpgHuQuZk1H5zpJgQuZZV~jdjrNIaVlJ6nq6WrYlxKODqvI~a9SIxhSTf3mCvbVBeRv0cvIoHXDO8x9AtbFQli4b7upklVyZlowXEgGlhgDSgaD0IW2xaXGKKO2NIAAWo2VfyGNWFEoaIqtgAdyRv-zvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197861&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content analysis. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 0.1 to 2 ng as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free and reads obtained were analysed for G+C content profile. A non-WGA sample (gDNA) shows a GC content of âˆ¼19.4%, a profile that was closely matched by products amplified using the optimized procedure (Opt). Samples amplified following the standard procedure (Std) showed biased GC content shown as a shift towards the neutral base composition.</p></div></div><div content-id="DSU028F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02803.jpeg?Expires=2147483647&amp;Signature=YJFzKVnv~MQWqykDMeQnf7C0vJPIblK4FfnxTM7dY4QGZ34aOuIaYiW8S~PjK2bNeVPCRrB8s7jJl7oJSCL8y4Sc7Y6Xi42IrpxWQKgUqS0MubmI6lqSOmIv7pbLozIRlRz4RULxfaRtSWXjVgWMMECxUQWmQNPfxhynj1QQbYAtuQkjKqYn7~WkAeLUqyNU71MORe1FLgqhx~Doj9DnQQyfc3am79FSYITCN0dK0~nokcadXCK9nUUeXkJJiRk0OYhiwQSWcVyrT2jL6OMl1enbws96VYhlkoVVUoGEmciAEVwst1QAGpzE0Nmc7fdxR6I~~nVgaHO9ofCnFypzzA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content analysis. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 0.1 to 2 ng as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free and reads obtained were analysed for G+C content profile. A non-WGA sample (gDNA) shows a GC content of âˆ¼19.4%, a profile that was closely matched by products amplified using the optimized procedure (Opt). Samples amplified following the standard procedure (Std) showed biased GC content shown as a shift towards the neutral base composition." path-from-xml="dsu02803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197861/dsu02803.jpeg" path-from-xml="dsu02803.jpeg" target="_blank">View large</a><a section="87197861" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02803.jpeg?Expires=2147483647&amp;Signature=Ro3--snlaksDehzI~YH-strQBxsxhLPKs1EMHK-speqoYZzaGXNd88rIJNh8nm56VumJR8eioujaHP~VxXvUMyzvboaWIIX7BnT-zMNW~-KVNqX5EJFOVKzCMbmmAzNc-E63i9LXPDh5G-~0FFRZ3I1V3BkjbwVSXYz00GaUCIZx3q1SXXUQao39PgY9fFpgHuQuZk1H5zpJgQuZZV~jdjrNIaVlJ6nq6WrYlxKODqvI~a9SIxhSTf3mCvbVBeRv0cvIoHXDO8x9AtbFQli4b7upklVyZlowXEgGlhgDSgaD0IW2xaXGKKO2NIAAWo2VfyGNWFEoaIqtgAdyRv-zvA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197861&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content analysis. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 0.1 to 2 ng as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free and reads obtained were analysed for G+C content profile. A non-WGA sample (gDNA) shows a GC content of âˆ¼19.4%, a profile that was closely matched by products amplified using the optimized procedure (Opt). Samples amplified following the standard procedure (Std) showed biased GC content shown as a shift towards the neutral base composition.</p></div></div></div>                            <h3 scrollto-destination=87197862 id="87197862" class="section-title" data-legacy-id=s3c>3.3. Optimized conditions reduce chimera formation and maintain template base composition during MDA by Ï†29</h3>
<div class="block-child-p">Formation of chimeric reads is a major problem with MDA technology.<sup><a reveal-id="DSU028C27" data-open="DSU028C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Chimeras cause serious mapping and assembly problems, and therefore reduce the quality and quantity of the WGA product. We analysed the formation of chimeras by comparing WGA products following standard and optimized procedures. The number of chimeric reads increased as the amount of input DNA was reduced (Fig.Â <a reveal-id="DSU028F4" data-open="DSU028F4" class="link link-reveal link-table xref-fig">4</a>). However, optimizing WGA condition by including TMAC additive decreased the formation of chimeras and improved the quality of reads in <em>P. falciparum</em> WGA. <div data-id="DSU028F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02804.jpeg?Expires=2147483647&Signature=Edu8VRVFlnt-7PsIaIRPL2ZifekM2WVhrxYru3hpLbpPptuPCrm-tIbGgKb8i-LGfVPbd1lZfuI7C5w5PM4FueXVhpL6AcaiWTx9yONX8TIwWCiiWlTuxWTasCM92uOtjAc~GzQUFIFW5U-ATQXlw5BKju8l0JCXcTlX~GX2A1F~mzuke751fVyAwwYBd0yhObsdFQ5quiKUtyi3aQAT~-UHXTkTwEeUYvUfxQ~A0tLJIDfXPQNNatnmv9ziRJGSJuZwkW7V4rsxcW-uM0qM7Wez06HKUB1IO5jnI39Awr-iCYPR6g7rd4cOsvLG~9y~xHvyq7nfszZPGo7UAOohiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar graph of chimeric read analysis. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 2 ng to 100 fg as shown in sample labels) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free, and reads obtained were normalized and analysed for the presence of chimeric reads. A non-WGA sample (control) showed the least number of chimers. Using the standard procedure, the number of chimers increased with a decrease in the amount of input DNA. Samples amplified using the optimized procedure showed a decrease in chimer formation that remained low and steady from 2 ng to 100 pg of input DNA." path-from-xml="dsu02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197863/dsu02804.jpeg" path-from-xml="dsu02804.jpeg" target="_blank">View large</a><a section="87197863" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02804.jpeg?Expires=2147483647&Signature=OX6yxDCINz0gYB1MoVY8jUXIU~aEdvQ5HC5afASWVz2bQEwmQdrquuYHi0qfGSMrVvMCM1PX2SfB02AXLNUQv4cBBhSgtFff2s1Q0RedgoVc1QVDhb7CIFLefIv4djL90yxHj34taPhrroqwL~XpTCyAIO22-ujy9AOeX9IV1s5FUDsfVnJ02aFU7iY0kEmWo0pdnYQevFEdEOhQjb5Zro6tbcxF0f1QbnY0d9QvrGm2OHdcd5S9Rqjxs1E~c94-XyoHvKuaCZNegt5H4KnDt9aXhnxrpRWcBhr3gGKqZYjNXEMEFmy9cDs4GiroRFwncsLwwxhhHabluU6unzzTuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197863&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar graph of chimeric read analysis. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 2 ng to 100 fg as shown in sample labels) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free, and reads obtained were normalized and analysed for the presence of chimeric reads. A non-WGA sample (control) showed the least number of chimers. Using the standard procedure, the number of chimers increased with a decrease in the amount of input DNA. Samples amplified using the optimized procedure showed a decrease in chimer formation that remained low and steady from 2 ng to 100 pg of input DNA.</p></div></div><div content-id="DSU028F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02804.jpeg?Expires=2147483647&Signature=Edu8VRVFlnt-7PsIaIRPL2ZifekM2WVhrxYru3hpLbpPptuPCrm-tIbGgKb8i-LGfVPbd1lZfuI7C5w5PM4FueXVhpL6AcaiWTx9yONX8TIwWCiiWlTuxWTasCM92uOtjAc~GzQUFIFW5U-ATQXlw5BKju8l0JCXcTlX~GX2A1F~mzuke751fVyAwwYBd0yhObsdFQ5quiKUtyi3aQAT~-UHXTkTwEeUYvUfxQ~A0tLJIDfXPQNNatnmv9ziRJGSJuZwkW7V4rsxcW-uM0qM7Wez06HKUB1IO5jnI39Awr-iCYPR6g7rd4cOsvLG~9y~xHvyq7nfszZPGo7UAOohiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bar graph of chimeric read analysis. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 2 ng to 100 fg as shown in sample labels) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free, and reads obtained were normalized and analysed for the presence of chimeric reads. A non-WGA sample (control) showed the least number of chimers. Using the standard procedure, the number of chimers increased with a decrease in the amount of input DNA. Samples amplified using the optimized procedure showed a decrease in chimer formation that remained low and steady from 2 ng to 100 pg of input DNA." path-from-xml="dsu02804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197863/dsu02804.jpeg" path-from-xml="dsu02804.jpeg" target="_blank">View large</a><a section="87197863" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02804.jpeg?Expires=2147483647&Signature=OX6yxDCINz0gYB1MoVY8jUXIU~aEdvQ5HC5afASWVz2bQEwmQdrquuYHi0qfGSMrVvMCM1PX2SfB02AXLNUQv4cBBhSgtFff2s1Q0RedgoVc1QVDhb7CIFLefIv4djL90yxHj34taPhrroqwL~XpTCyAIO22-ujy9AOeX9IV1s5FUDsfVnJ02aFU7iY0kEmWo0pdnYQevFEdEOhQjb5Zro6tbcxF0f1QbnY0d9QvrGm2OHdcd5S9Rqjxs1E~c94-XyoHvKuaCZNegt5H4KnDt9aXhnxrpRWcBhr3gGKqZYjNXEMEFmy9cDs4GiroRFwncsLwwxhhHabluU6unzzTuA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197863&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bar graph of chimeric read analysis. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 2 ng to 100 fg as shown in sample labels) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Amplification products were sequenced as PCR-free, and reads obtained were normalized and analysed for the presence of chimeric reads. A non-WGA sample (control) showed the least number of chimers. Using the standard procedure, the number of chimers increased with a decrease in the amount of input DNA. Samples amplified using the optimized procedure showed a decrease in chimer formation that remained low and steady from 2 ng to 100 pg of input DNA.</p></div></div></div>                            <h3 scrollto-destination=87197864 id="87197864" class="section-title" data-legacy-id=s3d>3.4. WGA from 10 pg of input P. falciparum genomic DNA</h3>
<div class="block-child-p">Most studies with Ï†29 MDA have used an input DNA of â‰¥10 ng for WGA.<sup><a reveal-id="DSU028C19" data-open="DSU028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU028C20" data-open="DSU028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In many cases, this amount may be difficult to obtain from valuable clinical specimens. We set to find out the minimal amount of parasite DNA that can be successfully amplified by the optimized conditions to obtain uniform genome coverage. We performed WGA on <em>P. falciparum</em> genomic DNA with an input amount ranging from 2 ng down to 100 femtograms (fg). WGA products were multiplexed and sequenced using Illumina MiSeq or HiSeq 2500 machines. Sequence reads generated were analysed to determine the minimum threshold required to produce optimal coverage suitable for various whole-genome studies including genotyping by SNP analysis. To assess the quality of the sequence data generated from each input amount, reads were mapped to the reference genome using BWA. CallableLoci program of the genome analyser tool kit (GATK)<sup><a reveal-id="DSU028C28" data-open="DSU028C28" class="link link-ref link-reveal xref-bibr">28</a></sup> was used to inspect and count the proportion of the genome with high-quality base coverage (callable bases), positions of the genome with zero coverage (uncovered bases) and the size of coverage gaps. Using these metrics, we show that the number of callable loci (high-quality bases) remained relatively high for samples with input DNA ranging from 2 ng to 10 pg. However, the quality of sequence data dropped sharply for samples with input DNA &lt;10 pg (Fig.Â <a reveal-id="DSU028F5" data-open="DSU028F5" class="link link-reveal link-table xref-fig">5</a>). A similar trend was observed with the proportion of gap sizes (length of uncovered bases, Fig.Â <a reveal-id="DSU028F5" data-open="DSU028F5" class="link link-reveal link-table xref-fig">5</a> bottom panel) and chimeric reads (Fig.Â <a reveal-id="DSU028F4" data-open="DSU028F4" class="link link-reveal link-table xref-fig">4</a>), where a sharp increase in these values was observed for samples with input DNA &lt;10 pg. From these observations, we concluded that 10 pg is the minimal amount of <em>P. falciparum</em> input DNA that produces quality genome coverage under the optimized MDA conditions described. This amount of DNA is equivalent to only âˆ¼380 parasite genomes. <div data-id="DSU028F5" class="fig fig-section"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02805.jpeg?Expires=2147483647&Signature=0THV8Tnd-ZkK0t1~~51OfSdbcZ6kumSN94~v~8SzST1y1HzrKLW92nvEBXYx6~6jXAYo97cymicf62LQgVC1-wFp~rwkGKJ394UI~hVmBV9LBo9CPZOMDf1NHuoUWxaasb4vMSrfJ8SpXxaQKu2iNPrC5JnniUNf7S2k8CDsTKrsW0AU1c6O24g9wjP3gBpj-Rf5Nph~GskzlkphEA1xxxJoFpRs0dsEfxuau0vLAvdgh5n6fB6hfvlMz31fQ6Laj48CUWG4rW~1FtA8fRSJ6Sp83goHrbJlAaquayeeQ9xmjx8fa5QU-cx1gizIMNTVI9CSBq76-cjop5JvvTXBZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determining the lowest threshold of input DNA mass for WGA. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 2 ng to 100 fg, as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Samples were multiplexed and sequenced using a fast turn-around Illumina Miseq machine. Sequence reads mapped to the reference were normalized and analysed for coverage and base quality using the â€˜CallableLociâ€™ program of GATK. A non-WGA sample was used as an unamplified control. Input quantities ranging from 2 ng down to 10 pg produced reads with high-quality â€˜callableâ€™ bases covering over 60% of the genome. Input DNA below the10-pg threshold produced poor base quality with only &lt;30% of genome covered with â€˜callableâ€™ bases. Input DNA &lt;10 pg showed a sharp increase in positions with zero coverage (bases uncovered) and an increase in gap sizes in genome coverage." path-from-xml="dsu02805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197865/dsu02805.jpeg" path-from-xml="dsu02805.jpeg" target="_blank">View large</a><a section="87197865" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02805.jpeg?Expires=2147483647&Signature=111rLYsmUF6PNLVFLJ6MCK82NulSKC0qKIo7q~UpaNS80G31XcmlUHEuHwt88WQ30Ox8dwtRJ45FjeRub39r5gCK1oxCX0b-D~LaAwZggQ6H0QpnJnF6Vbe5GhDC9BitmfPwGl88YKv7R5Abq4D3ZsAbMMFHPAsPBEuIJP33obT39cLL9NNSXf5chLkgOiPI~JVHeKdE9VCBYp77JSJj1qySwr4cOKS~TJWtspv5TKrKHvCk~kWv9ycpwmzCtMMyepbjqa2PL7-MgdRwOCr3aSnmJO~V3QLceic6w~9PVmEDg3czhZjSy2jUWd29MxjPH7fbtj-3sv~UfFtc90MvDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197865&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determining the lowest threshold of input DNA mass for WGA. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 2 ng to 100 fg, as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Samples were multiplexed and sequenced using a fast turn-around Illumina Miseq machine. Sequence reads mapped to the reference were normalized and analysed for coverage and base quality using the â€˜CallableLociâ€™ program of GATK. A non-WGA sample was used as an unamplified control. Input quantities ranging from 2 ng down to 10 pg produced reads with high-quality â€˜callableâ€™ bases covering over 60% of the genome. Input DNA below the10-pg threshold produced poor base quality with only &lt;30% of genome covered with â€˜callableâ€™ bases. Input DNA &lt;10 pg showed a sharp increase in positions with zero coverage (bases uncovered) and an increase in gap sizes in genome coverage.</p></div></div><div content-id="DSU028F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02805.jpeg?Expires=2147483647&Signature=0THV8Tnd-ZkK0t1~~51OfSdbcZ6kumSN94~v~8SzST1y1HzrKLW92nvEBXYx6~6jXAYo97cymicf62LQgVC1-wFp~rwkGKJ394UI~hVmBV9LBo9CPZOMDf1NHuoUWxaasb4vMSrfJ8SpXxaQKu2iNPrC5JnniUNf7S2k8CDsTKrsW0AU1c6O24g9wjP3gBpj-Rf5Nph~GskzlkphEA1xxxJoFpRs0dsEfxuau0vLAvdgh5n6fB6hfvlMz31fQ6Laj48CUWG4rW~1FtA8fRSJ6Sp83goHrbJlAaquayeeQ9xmjx8fa5QU-cx1gizIMNTVI9CSBq76-cjop5JvvTXBZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determining the lowest threshold of input DNA mass for WGA. Different amounts of P. falciparum 3D7 genomic DNA (ranging from 2 ng to 100 fg, as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Samples were multiplexed and sequenced using a fast turn-around Illumina Miseq machine. Sequence reads mapped to the reference were normalized and analysed for coverage and base quality using the â€˜CallableLociâ€™ program of GATK. A non-WGA sample was used as an unamplified control. Input quantities ranging from 2 ng down to 10 pg produced reads with high-quality â€˜callableâ€™ bases covering over 60% of the genome. Input DNA below the10-pg threshold produced poor base quality with only &lt;30% of genome covered with â€˜callableâ€™ bases. Input DNA &lt;10 pg showed a sharp increase in positions with zero coverage (bases uncovered) and an increase in gap sizes in genome coverage." path-from-xml="dsu02805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197865/dsu02805.jpeg" path-from-xml="dsu02805.jpeg" target="_blank">View large</a><a section="87197865" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02805.jpeg?Expires=2147483647&Signature=111rLYsmUF6PNLVFLJ6MCK82NulSKC0qKIo7q~UpaNS80G31XcmlUHEuHwt88WQ30Ox8dwtRJ45FjeRub39r5gCK1oxCX0b-D~LaAwZggQ6H0QpnJnF6Vbe5GhDC9BitmfPwGl88YKv7R5Abq4D3ZsAbMMFHPAsPBEuIJP33obT39cLL9NNSXf5chLkgOiPI~JVHeKdE9VCBYp77JSJj1qySwr4cOKS~TJWtspv5TKrKHvCk~kWv9ycpwmzCtMMyepbjqa2PL7-MgdRwOCr3aSnmJO~V3QLceic6w~9PVmEDg3czhZjSy2jUWd29MxjPH7fbtj-3sv~UfFtc90MvDg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197865&ar=2754550&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu02805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determining the lowest threshold of input DNA mass for WGA. Different amounts of <em>P. falciparum</em> 3D7 genomic DNA (ranging from 2 ng to 100 fg, as shown in sample names) were used as an input template for amplification by Repli_g following the standard or optimized procedure. Samples were multiplexed and sequenced using a fast turn-around Illumina Miseq machine. Sequence reads mapped to the reference were normalized and analysed for coverage and base quality using the â€˜CallableLociâ€™ program of GATK. A non-WGA sample was used as an unamplified control. Input quantities ranging from 2 ng down to 10 pg produced reads with high-quality â€˜callableâ€™ bases covering over 60% of the genome. Input DNA below the10-pg threshold produced poor base quality with only &lt;30% of genome covered with â€˜callableâ€™ bases. Input DNA &lt;10 pg showed a sharp increase in positions with zero coverage (bases uncovered) and an increase in gap sizes in genome coverage.</p></div></div></div>                            <h3 scrollto-destination=87197866 id="87197866" class="section-title" data-legacy-id=s3e>3.5. Detailed analysis of WGA products from a 10-pg input template DNA</h3>
<p>Standard WGA methods produce products that show sequence representation bias, allelic dropout and amplification artefacts. These problems tend to increase as the amount of input template decreases. To adequately evaluate the quality of the WGA products produced from an input of 10 pgâ€”1,000-fold smaller than the standard input amountsâ€”we performed analysis on genomic DNA extracted from <em>P. falciparum</em> strains derived from the progeny clones of genetic cross between 7G8xGB4 laboratory strains, as well as the 3D7 Strain. Genomic DNA extracted from 17 progeny clones was sequenced both as WGA and as bulk genomic DNA (non-WGA). We performed detailed analysis by comparing sequence data generated from WGA and their matching unamplified genomic DNA (non-WGA). WGA and non-WGA samples yielded a median of 3.5 and 3.4 billion base sequences, respectively, with 94.8 and 95.3% of the read mapping to the reference sequence (TableÂ <a reveal-id="DSU028TB1" data-open="DSU028TB1" class="link link-reveal link-table xref-fig">1</a>). WGA samples showed coverage between 180Ã— and 500Ã—, whereas non-WGA samples showed coverage between 90Ã— and 250Ã—. A median of 1.5 and 1.2% of genome bases was not covered in WGA and non-WGA samples, respectively (TableÂ <a reveal-id="DSU028TB1" data-open="DSU028TB1" class="link link-reveal link-table xref-fig">1</a>). <div content-id="DSU028TB1" class="table-modal"><div class="table-wrap"><div id="DSU028TB1" data-id="DSU028TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Sequence coverage and mapping analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="2">Total reads (million)<hr />Â </th><th colspan="3">Total base (Gb)<hr />Â </th><th colspan="2">Mapped reads (%)<hr />Â </th><th colspan="2">Uncovered bases (%)<hr />Â </th></tr><tr><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGA/non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>30.8Â </td><td>43.5Â </td><td>2.3Â </td><td>3.3Â </td><td>0.7Â </td><td>98.8Â </td><td>98.6Â </td><td>0.2Â </td><td>0.0Â </td></tr><tr><td>DEV_18_05_11Â </td><td>74.5Â </td><td>50.7Â </td><td>5.6Â </td><td>3.8Â </td><td>1.5Â </td><td>94.6Â </td><td>95.0Â </td><td>1.1Â </td><td>1.2Â </td></tr><tr><td>D2_18_05_11Â </td><td>41.7Â </td><td>41.4Â </td><td>3.1Â </td><td>3.1Â </td><td>1.0Â </td><td>94.2Â </td><td>95.2Â </td><td>1.2Â </td><td>1.0Â </td></tr><tr><td>WE2Â </td><td>69.7Â </td><td>35.2Â </td><td>5.2Â </td><td>2.6Â </td><td>2.0Â </td><td>95.1Â </td><td>95.1Â </td><td>1.2Â </td><td>1.3Â </td></tr><tr><td>GB4_NIHÂ </td><td>45.2Â </td><td>37.9Â </td><td>3.4Â </td><td>2.8Â </td><td>1.2Â </td><td>94.2Â </td><td>94.5Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JC3Â </td><td>45.1Â </td><td>47.6Â </td><td>3.4Â </td><td>3.6Â </td><td>0.9Â </td><td>94.3Â </td><td>95.1Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JF6Â </td><td>49.1Â </td><td>57.3Â </td><td>3.7Â </td><td>4.3Â </td><td>0.9Â </td><td>94.3Â </td><td>95.5Â </td><td>1.4Â </td><td>1.1Â </td></tr><tr><td>QF5Â </td><td>46.5Â </td><td>66.6Â </td><td>3.5Â </td><td>5.0Â </td><td>0.7Â </td><td>95.2Â </td><td>95.9Â </td><td>1.4Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>59.1Â </td><td>36.9Â </td><td>4.4Â </td><td>2.8Â </td><td>1.6Â </td><td>95.5Â </td><td>90.6Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>NF10Â </td><td>48.4Â </td><td>35.0Â </td><td>3.6Â </td><td>2.6Â </td><td>1.4Â </td><td>95.5Â </td><td>95.0Â </td><td>1.5Â </td><td>1.4Â </td></tr><tr><td>XF12_18_05_11Â </td><td>39.2Â </td><td>58.6Â </td><td>2.9Â </td><td>4.4Â </td><td>0.7Â </td><td>94.7Â </td><td>95.6Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>AL2_13_05_11Â </td><td>38.3Â </td><td>40.1Â </td><td>2.9Â </td><td>3.0Â </td><td>1.0Â </td><td>94.8Â </td><td>95.4Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>XD8Â </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>93.6Â </td><td>95.5Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>7G8_NIHÂ </td><td>61.1Â </td><td>40.5Â </td><td>4.6Â </td><td>3.0Â </td><td>1.5Â </td><td>94.2Â </td><td>94.9Â </td><td>1.6Â </td><td>1.5Â </td></tr><tr><td>JONÂ </td><td>51.3Â </td><td>71.8Â </td><td>3.8Â </td><td>5.4Â </td><td>0.7Â </td><td>94.0Â </td><td>95.4Â </td><td>1.6Â </td><td>1.1Â </td></tr><tr><td>TF1Â </td><td>57.7Â </td><td>65.2Â </td><td>4.3Â </td><td>4.9Â </td><td>0.9Â </td><td>95.5Â </td><td>95.7Â </td><td>1.8Â </td><td>1.4Â </td></tr><tr><td>JC9Â </td><td>34.0Â </td><td>61.3Â </td><td>2.5Â </td><td>4.6Â </td><td>0.6Â </td><td>95.0Â </td><td>95.6Â </td><td>2.5Â </td><td>1.2Â </td></tr><tr><td>MedianÂ </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>94.7Â </td><td>95.4Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>Mean Â± SDÂ </td><td>49.3 Â± 12Â </td><td>49.1 Â± 12Â </td><td>3.7 Â± 0.9Â </td><td>3.7 Â± 0.9Â </td><td>1.0 Â± 0.4Â </td><td>94.9 Â± 1.2Â </td><td>95.2 Â± 1.5Â </td><td>1.4 Â± 0.4Â </td><td>1.1 Â± 0.3Â </td></tr></tbody></table></div><div content-id="DSU028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="2">Total reads (million)<hr />Â </th><th colspan="3">Total base (Gb)<hr />Â </th><th colspan="2">Mapped reads (%)<hr />Â </th><th colspan="2">Uncovered bases (%)<hr />Â </th></tr><tr><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGA/non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>30.8Â </td><td>43.5Â </td><td>2.3Â </td><td>3.3Â </td><td>0.7Â </td><td>98.8Â </td><td>98.6Â </td><td>0.2Â </td><td>0.0Â </td></tr><tr><td>DEV_18_05_11Â </td><td>74.5Â </td><td>50.7Â </td><td>5.6Â </td><td>3.8Â </td><td>1.5Â </td><td>94.6Â </td><td>95.0Â </td><td>1.1Â </td><td>1.2Â </td></tr><tr><td>D2_18_05_11Â </td><td>41.7Â </td><td>41.4Â </td><td>3.1Â </td><td>3.1Â </td><td>1.0Â </td><td>94.2Â </td><td>95.2Â </td><td>1.2Â </td><td>1.0Â </td></tr><tr><td>WE2Â </td><td>69.7Â </td><td>35.2Â </td><td>5.2Â </td><td>2.6Â </td><td>2.0Â </td><td>95.1Â </td><td>95.1Â </td><td>1.2Â </td><td>1.3Â </td></tr><tr><td>GB4_NIHÂ </td><td>45.2Â </td><td>37.9Â </td><td>3.4Â </td><td>2.8Â </td><td>1.2Â </td><td>94.2Â </td><td>94.5Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JC3Â </td><td>45.1Â </td><td>47.6Â </td><td>3.4Â </td><td>3.6Â </td><td>0.9Â </td><td>94.3Â </td><td>95.1Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JF6Â </td><td>49.1Â </td><td>57.3Â </td><td>3.7Â </td><td>4.3Â </td><td>0.9Â </td><td>94.3Â </td><td>95.5Â </td><td>1.4Â </td><td>1.1Â </td></tr><tr><td>QF5Â </td><td>46.5Â </td><td>66.6Â </td><td>3.5Â </td><td>5.0Â </td><td>0.7Â </td><td>95.2Â </td><td>95.9Â </td><td>1.4Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>59.1Â </td><td>36.9Â </td><td>4.4Â </td><td>2.8Â </td><td>1.6Â </td><td>95.5Â </td><td>90.6Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>NF10Â </td><td>48.4Â </td><td>35.0Â </td><td>3.6Â </td><td>2.6Â </td><td>1.4Â </td><td>95.5Â </td><td>95.0Â </td><td>1.5Â </td><td>1.4Â </td></tr><tr><td>XF12_18_05_11Â </td><td>39.2Â </td><td>58.6Â </td><td>2.9Â </td><td>4.4Â </td><td>0.7Â </td><td>94.7Â </td><td>95.6Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>AL2_13_05_11Â </td><td>38.3Â </td><td>40.1Â </td><td>2.9Â </td><td>3.0Â </td><td>1.0Â </td><td>94.8Â </td><td>95.4Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>XD8Â </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>93.6Â </td><td>95.5Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>7G8_NIHÂ </td><td>61.1Â </td><td>40.5Â </td><td>4.6Â </td><td>3.0Â </td><td>1.5Â </td><td>94.2Â </td><td>94.9Â </td><td>1.6Â </td><td>1.5Â </td></tr><tr><td>JONÂ </td><td>51.3Â </td><td>71.8Â </td><td>3.8Â </td><td>5.4Â </td><td>0.7Â </td><td>94.0Â </td><td>95.4Â </td><td>1.6Â </td><td>1.1Â </td></tr><tr><td>TF1Â </td><td>57.7Â </td><td>65.2Â </td><td>4.3Â </td><td>4.9Â </td><td>0.9Â </td><td>95.5Â </td><td>95.7Â </td><td>1.8Â </td><td>1.4Â </td></tr><tr><td>JC9Â </td><td>34.0Â </td><td>61.3Â </td><td>2.5Â </td><td>4.6Â </td><td>0.6Â </td><td>95.0Â </td><td>95.6Â </td><td>2.5Â </td><td>1.2Â </td></tr><tr><td>MedianÂ </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>94.7Â </td><td>95.4Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>Mean Â± SDÂ </td><td>49.3 Â± 12Â </td><td>49.1 Â± 12Â </td><td>3.7 Â± 0.9Â </td><td>3.7 Â± 0.9Â </td><td>1.0 Â± 0.4Â </td><td>94.9 Â± 1.2Â </td><td>95.2 Â± 1.5Â </td><td>1.4 Â± 0.4Â </td><td>1.1 Â± 0.3Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>Sequence reads obtained from WGA and non-WGA samples were analysed for coverage and mapping statistics. Nearly identical average number of sequence reads, mapping and coverage statistics were obtained for both WGA and non-WGA datasets. However, coverage distribution was less uniform in WGA compared with non-WGA datasets.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU028TB1" data-id="DSU028TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Sequence coverage and mapping analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="2">Total reads (million)<hr />Â </th><th colspan="3">Total base (Gb)<hr />Â </th><th colspan="2">Mapped reads (%)<hr />Â </th><th colspan="2">Uncovered bases (%)<hr />Â </th></tr><tr><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGA/non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>30.8Â </td><td>43.5Â </td><td>2.3Â </td><td>3.3Â </td><td>0.7Â </td><td>98.8Â </td><td>98.6Â </td><td>0.2Â </td><td>0.0Â </td></tr><tr><td>DEV_18_05_11Â </td><td>74.5Â </td><td>50.7Â </td><td>5.6Â </td><td>3.8Â </td><td>1.5Â </td><td>94.6Â </td><td>95.0Â </td><td>1.1Â </td><td>1.2Â </td></tr><tr><td>D2_18_05_11Â </td><td>41.7Â </td><td>41.4Â </td><td>3.1Â </td><td>3.1Â </td><td>1.0Â </td><td>94.2Â </td><td>95.2Â </td><td>1.2Â </td><td>1.0Â </td></tr><tr><td>WE2Â </td><td>69.7Â </td><td>35.2Â </td><td>5.2Â </td><td>2.6Â </td><td>2.0Â </td><td>95.1Â </td><td>95.1Â </td><td>1.2Â </td><td>1.3Â </td></tr><tr><td>GB4_NIHÂ </td><td>45.2Â </td><td>37.9Â </td><td>3.4Â </td><td>2.8Â </td><td>1.2Â </td><td>94.2Â </td><td>94.5Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JC3Â </td><td>45.1Â </td><td>47.6Â </td><td>3.4Â </td><td>3.6Â </td><td>0.9Â </td><td>94.3Â </td><td>95.1Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JF6Â </td><td>49.1Â </td><td>57.3Â </td><td>3.7Â </td><td>4.3Â </td><td>0.9Â </td><td>94.3Â </td><td>95.5Â </td><td>1.4Â </td><td>1.1Â </td></tr><tr><td>QF5Â </td><td>46.5Â </td><td>66.6Â </td><td>3.5Â </td><td>5.0Â </td><td>0.7Â </td><td>95.2Â </td><td>95.9Â </td><td>1.4Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>59.1Â </td><td>36.9Â </td><td>4.4Â </td><td>2.8Â </td><td>1.6Â </td><td>95.5Â </td><td>90.6Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>NF10Â </td><td>48.4Â </td><td>35.0Â </td><td>3.6Â </td><td>2.6Â </td><td>1.4Â </td><td>95.5Â </td><td>95.0Â </td><td>1.5Â </td><td>1.4Â </td></tr><tr><td>XF12_18_05_11Â </td><td>39.2Â </td><td>58.6Â </td><td>2.9Â </td><td>4.4Â </td><td>0.7Â </td><td>94.7Â </td><td>95.6Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>AL2_13_05_11Â </td><td>38.3Â </td><td>40.1Â </td><td>2.9Â </td><td>3.0Â </td><td>1.0Â </td><td>94.8Â </td><td>95.4Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>XD8Â </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>93.6Â </td><td>95.5Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>7G8_NIHÂ </td><td>61.1Â </td><td>40.5Â </td><td>4.6Â </td><td>3.0Â </td><td>1.5Â </td><td>94.2Â </td><td>94.9Â </td><td>1.6Â </td><td>1.5Â </td></tr><tr><td>JONÂ </td><td>51.3Â </td><td>71.8Â </td><td>3.8Â </td><td>5.4Â </td><td>0.7Â </td><td>94.0Â </td><td>95.4Â </td><td>1.6Â </td><td>1.1Â </td></tr><tr><td>TF1Â </td><td>57.7Â </td><td>65.2Â </td><td>4.3Â </td><td>4.9Â </td><td>0.9Â </td><td>95.5Â </td><td>95.7Â </td><td>1.8Â </td><td>1.4Â </td></tr><tr><td>JC9Â </td><td>34.0Â </td><td>61.3Â </td><td>2.5Â </td><td>4.6Â </td><td>0.6Â </td><td>95.0Â </td><td>95.6Â </td><td>2.5Â </td><td>1.2Â </td></tr><tr><td>MedianÂ </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>94.7Â </td><td>95.4Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>Mean Â± SDÂ </td><td>49.3 Â± 12Â </td><td>49.1 Â± 12Â </td><td>3.7 Â± 0.9Â </td><td>3.7 Â± 0.9Â </td><td>1.0 Â± 0.4Â </td><td>94.9 Â± 1.2Â </td><td>95.2 Â± 1.5Â </td><td>1.4 Â± 0.4Â </td><td>1.1 Â± 0.3Â </td></tr></tbody></table></div><div content-id="DSU028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="2">Total reads (million)<hr />Â </th><th colspan="3">Total base (Gb)<hr />Â </th><th colspan="2">Mapped reads (%)<hr />Â </th><th colspan="2">Uncovered bases (%)<hr />Â </th></tr><tr><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGA/non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th><th>WGAÂ </th><th>Non-WGAÂ </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>30.8Â </td><td>43.5Â </td><td>2.3Â </td><td>3.3Â </td><td>0.7Â </td><td>98.8Â </td><td>98.6Â </td><td>0.2Â </td><td>0.0Â </td></tr><tr><td>DEV_18_05_11Â </td><td>74.5Â </td><td>50.7Â </td><td>5.6Â </td><td>3.8Â </td><td>1.5Â </td><td>94.6Â </td><td>95.0Â </td><td>1.1Â </td><td>1.2Â </td></tr><tr><td>D2_18_05_11Â </td><td>41.7Â </td><td>41.4Â </td><td>3.1Â </td><td>3.1Â </td><td>1.0Â </td><td>94.2Â </td><td>95.2Â </td><td>1.2Â </td><td>1.0Â </td></tr><tr><td>WE2Â </td><td>69.7Â </td><td>35.2Â </td><td>5.2Â </td><td>2.6Â </td><td>2.0Â </td><td>95.1Â </td><td>95.1Â </td><td>1.2Â </td><td>1.3Â </td></tr><tr><td>GB4_NIHÂ </td><td>45.2Â </td><td>37.9Â </td><td>3.4Â </td><td>2.8Â </td><td>1.2Â </td><td>94.2Â </td><td>94.5Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JC3Â </td><td>45.1Â </td><td>47.6Â </td><td>3.4Â </td><td>3.6Â </td><td>0.9Â </td><td>94.3Â </td><td>95.1Â </td><td>1.3Â </td><td>1.1Â </td></tr><tr><td>JF6Â </td><td>49.1Â </td><td>57.3Â </td><td>3.7Â </td><td>4.3Â </td><td>0.9Â </td><td>94.3Â </td><td>95.5Â </td><td>1.4Â </td><td>1.1Â </td></tr><tr><td>QF5Â </td><td>46.5Â </td><td>66.6Â </td><td>3.5Â </td><td>5.0Â </td><td>0.7Â </td><td>95.2Â </td><td>95.9Â </td><td>1.4Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>59.1Â </td><td>36.9Â </td><td>4.4Â </td><td>2.8Â </td><td>1.6Â </td><td>95.5Â </td><td>90.6Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>NF10Â </td><td>48.4Â </td><td>35.0Â </td><td>3.6Â </td><td>2.6Â </td><td>1.4Â </td><td>95.5Â </td><td>95.0Â </td><td>1.5Â </td><td>1.4Â </td></tr><tr><td>XF12_18_05_11Â </td><td>39.2Â </td><td>58.6Â </td><td>2.9Â </td><td>4.4Â </td><td>0.7Â </td><td>94.7Â </td><td>95.6Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>AL2_13_05_11Â </td><td>38.3Â </td><td>40.1Â </td><td>2.9Â </td><td>3.0Â </td><td>1.0Â </td><td>94.8Â </td><td>95.4Â </td><td>1.5Â </td><td>1.3Â </td></tr><tr><td>XD8Â </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>93.6Â </td><td>95.5Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>7G8_NIHÂ </td><td>61.1Â </td><td>40.5Â </td><td>4.6Â </td><td>3.0Â </td><td>1.5Â </td><td>94.2Â </td><td>94.9Â </td><td>1.6Â </td><td>1.5Â </td></tr><tr><td>JONÂ </td><td>51.3Â </td><td>71.8Â </td><td>3.8Â </td><td>5.4Â </td><td>0.7Â </td><td>94.0Â </td><td>95.4Â </td><td>1.6Â </td><td>1.1Â </td></tr><tr><td>TF1Â </td><td>57.7Â </td><td>65.2Â </td><td>4.3Â </td><td>4.9Â </td><td>0.9Â </td><td>95.5Â </td><td>95.7Â </td><td>1.8Â </td><td>1.4Â </td></tr><tr><td>JC9Â </td><td>34.0Â </td><td>61.3Â </td><td>2.5Â </td><td>4.6Â </td><td>0.6Â </td><td>95.0Â </td><td>95.6Â </td><td>2.5Â </td><td>1.2Â </td></tr><tr><td>MedianÂ </td><td>46.9Â </td><td>45.0Â </td><td>3.5Â </td><td>3.4Â </td><td>1.0Â </td><td>94.7Â </td><td>95.4Â </td><td>1.5Â </td><td>1.2Â </td></tr><tr><td>Mean Â± SDÂ </td><td>49.3 Â± 12Â </td><td>49.1 Â± 12Â </td><td>3.7 Â± 0.9Â </td><td>3.7 Â± 0.9Â </td><td>1.0 Â± 0.4Â </td><td>94.9 Â± 1.2Â </td><td>95.2 Â± 1.5Â </td><td>1.4 Â± 0.4Â </td><td>1.1 Â± 0.3Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>Sequence reads obtained from WGA and non-WGA samples were analysed for coverage and mapping statistics. Nearly identical average number of sequence reads, mapping and coverage statistics were obtained for both WGA and non-WGA datasets. However, coverage distribution was less uniform in WGA compared with non-WGA datasets.</p></div></div></div></div></p>                            <h3 scrollto-destination=87197868 id="87197868" class="section-title" data-legacy-id=s3f>3.6. Genotype concordance analysis</h3>
<p>To evaluate the fidelity of sequence representation by WGA, we analysed the genotype calls from 17 cross samples and the parent reference strain. We performed <em>de novo</em> and <em>in silico</em> genotype calls on sequence generated from both the WGA and non-WGA samples. Genotype concordance was determined by comparing SNP and InDel calls from matched pairs of both non-WGA and WGA samples.</p>                            <h4 scrollto-destination=87197870 id="87197870" class="section-title" data-legacy-id=s3f1>3.6.1. Concordance of de novo SNPs and InDel calls</h4>
<p>For each pair of WGA and non-WGA sample, we generated <em>de novo</em> variation calls simultaneously. A median of 20,338 biallelic <em>de novo</em> SNPs with a quality score of â‰¥250 were called for each sample. For the 3D7 reference strain (3D7_Glasgow), only 185 <em>de novo</em> SNPs were called (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Both of these numbers are well within the expected range for this method.<sup><a reveal-id="DSU028C1" data-open="DSU028C1" class="link link-ref link-reveal xref-bibr">1</a></sup> As given in TableÂ <a reveal-id="DSU028TB2" data-open="DSU028TB2" class="link link-reveal link-table xref-fig">2</a>, call pairs were grouped into â€˜Perfect Concordanceâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™. Identical calls in both non-WGA and WGA samples had a median of 97.3% and a mean of 96.7% Â± 1.2 that are in perfect concordance, and new alleles present only in the WGA samples (considered as WGA-introduced alleles) had a median of 1.2% and a mean of 1.8 Â± 1.1%. Most of the WGA-introduced alleles were due to a mixed call in WGA, where the non-WGA counterpart only showed the reference allele. Discordant single-allele calls were extremely rare with a median of zero SNPs and a total of 10 occurrences across all the samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Additionally, <em>de novo</em> SNP calls in 5,075,789 well-covered coding positions (TableÂ <a reveal-id="DSU028TB2" data-open="DSU028TB2" class="link link-reveal link-table xref-fig">2</a>) produced perfect concordance with a median of 98.5% and a mean of 98.0 Â± 1.4%, while the occurrence of WGA-introduced alleles dropped to a median of 0.7% and a mean of 0.8 Â± 0.4%. In this subset, no discordant calls (alt/ref or ref/alt) were observed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Calls that were grouped as â€˜Undeterminedâ€™ represent alleles that were missing or wrong alleles (present in the non-WGA samples but are neither the reference nor the alternative allele). <div content-id="DSU028TB2" class="table-modal"><div class="table-wrap"><div id="DSU028TB2" data-id="DSU028TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>WGA and non-WGA SNP concordance analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> SNP calls in all regions<hr />Â </th><th colspan="3"><em>De novo</em> SNP calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>JONÂ </td><td>93.8Â </td><td>4.4Â </td><td>1.8Â </td><td>97.2Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>JF6Â </td><td>94.4Â </td><td>4.3Â </td><td>1.3Â </td><td>94.2Â </td><td>5.0Â </td><td>0.8Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.0Â </td><td>2.6Â </td><td>2.4Â </td><td>96.9Â </td><td>2.2Â </td><td>0.9Â </td></tr><tr><td>QF5Â </td><td>95.7Â </td><td>2.5Â </td><td>1.8Â </td><td>99.0Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>QF5Â </td><td>96.3Â </td><td>2.6Â </td><td>1.1Â </td><td>98.5Â </td><td>0.8Â </td><td>0.7Â </td></tr><tr><td>JC9Â </td><td>96.5Â </td><td>1.6Â </td><td>1.9Â </td><td>98.2Â </td><td>1.3Â </td><td>0.5Â </td></tr><tr><td>XF12_18_05_11Â </td><td>97.3Â </td><td>1.4Â </td><td>1.4Â </td><td>98.4Â </td><td>0.3Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.3Â </td><td>0.8Â </td><td>2.0Â </td><td>95.6Â </td><td>2.4Â </td><td>2.0Â </td></tr><tr><td>NF10Â </td><td>97.3Â </td><td>1.5Â </td><td>1.3Â </td><td>98.7Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>DEV_18_05_11Â </td><td>97.3Â </td><td>1.1Â </td><td>1.6Â </td><td>98.9Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>JC3Â </td><td>97.4Â </td><td>1.1Â </td><td>1.6Â </td><td>98.6Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.4Â </td><td>1.1Â </td><td>1.5Â </td><td>98.5Â </td><td>0.5Â </td><td>0.9Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.4Â </td><td>1.2Â </td><td>1.4Â </td><td>99.0Â </td><td>0.6Â </td><td>0.5Â </td></tr><tr><td>WE2Â </td><td>97.5Â </td><td>1.0Â </td><td>1.6Â </td><td>97.9Â </td><td>1.6Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.5Â </td><td>1.1Â </td><td>1.4Â </td><td>97.1Â </td><td>2.4Â </td><td>0.5Â </td></tr><tr><td>TF1Â </td><td>97.6Â </td><td>1.0Â </td><td>1.3Â </td><td>98.8Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>3D7_GlasgowÂ </td><td>97.8Â </td><td>1.1Â </td><td>1.1Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>MedianÂ </td><td>97.3Â </td><td>1.2Â </td><td>1.5Â </td><td>98.5Â </td><td>0.6Â </td><td>0.7Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.7 Â± 1.2Â </td><td>1.8 Â± 1.1Â </td><td>1.6 Â± 0.3Â </td><td>98.0 Â± 1.4Â </td><td>1.3 Â± 1.2Â </td><td>0.8 Â± 0.4Â </td></tr></tbody></table></div><div content-id="DSU028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> SNP calls in all regions<hr />Â </th><th colspan="3"><em>De novo</em> SNP calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>JONÂ </td><td>93.8Â </td><td>4.4Â </td><td>1.8Â </td><td>97.2Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>JF6Â </td><td>94.4Â </td><td>4.3Â </td><td>1.3Â </td><td>94.2Â </td><td>5.0Â </td><td>0.8Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.0Â </td><td>2.6Â </td><td>2.4Â </td><td>96.9Â </td><td>2.2Â </td><td>0.9Â </td></tr><tr><td>QF5Â </td><td>95.7Â </td><td>2.5Â </td><td>1.8Â </td><td>99.0Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>QF5Â </td><td>96.3Â </td><td>2.6Â </td><td>1.1Â </td><td>98.5Â </td><td>0.8Â </td><td>0.7Â </td></tr><tr><td>JC9Â </td><td>96.5Â </td><td>1.6Â </td><td>1.9Â </td><td>98.2Â </td><td>1.3Â </td><td>0.5Â </td></tr><tr><td>XF12_18_05_11Â </td><td>97.3Â </td><td>1.4Â </td><td>1.4Â </td><td>98.4Â </td><td>0.3Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.3Â </td><td>0.8Â </td><td>2.0Â </td><td>95.6Â </td><td>2.4Â </td><td>2.0Â </td></tr><tr><td>NF10Â </td><td>97.3Â </td><td>1.5Â </td><td>1.3Â </td><td>98.7Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>DEV_18_05_11Â </td><td>97.3Â </td><td>1.1Â </td><td>1.6Â </td><td>98.9Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>JC3Â </td><td>97.4Â </td><td>1.1Â </td><td>1.6Â </td><td>98.6Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.4Â </td><td>1.1Â </td><td>1.5Â </td><td>98.5Â </td><td>0.5Â </td><td>0.9Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.4Â </td><td>1.2Â </td><td>1.4Â </td><td>99.0Â </td><td>0.6Â </td><td>0.5Â </td></tr><tr><td>WE2Â </td><td>97.5Â </td><td>1.0Â </td><td>1.6Â </td><td>97.9Â </td><td>1.6Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.5Â </td><td>1.1Â </td><td>1.4Â </td><td>97.1Â </td><td>2.4Â </td><td>0.5Â </td></tr><tr><td>TF1Â </td><td>97.6Â </td><td>1.0Â </td><td>1.3Â </td><td>98.8Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>3D7_GlasgowÂ </td><td>97.8Â </td><td>1.1Â </td><td>1.1Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>MedianÂ </td><td>97.3Â </td><td>1.2Â </td><td>1.5Â </td><td>98.5Â </td><td>0.6Â </td><td>0.7Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.7 Â± 1.2Â </td><td>1.8 Â± 1.1Â </td><td>1.6 Â± 0.3Â </td><td>98.0 Â± 1.4Â </td><td>1.3 Â± 1.2Â </td><td>0.8 Â± 0.4Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>WGA and non-WGA data were analysed for <em>de novo</em> SNP calling concordance. Matching sequence data from cross samples were paired and the SNP calling was performed simultaneously. Concordance rates were generated by comparing WGA and related non-WGA calls. Columns on the left show comparison of all calls covering the entire genome, whereas columns on the right show a subset of all calls covering only the high-quality coding regions of the genome. Values in the perfect concordance columns represent the proportion of calls that were identical in both WGA and non-WGA samples. WGA new allele columns show the proportion of SNPs that were present only in the WGA and not in their matching non-WGA alleles. Undetermined column shows the proportion of SNPs that were called in wrong or missing alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU028TB2" data-id="DSU028TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>WGA and non-WGA SNP concordance analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> SNP calls in all regions<hr />Â </th><th colspan="3"><em>De novo</em> SNP calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>JONÂ </td><td>93.8Â </td><td>4.4Â </td><td>1.8Â </td><td>97.2Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>JF6Â </td><td>94.4Â </td><td>4.3Â </td><td>1.3Â </td><td>94.2Â </td><td>5.0Â </td><td>0.8Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.0Â </td><td>2.6Â </td><td>2.4Â </td><td>96.9Â </td><td>2.2Â </td><td>0.9Â </td></tr><tr><td>QF5Â </td><td>95.7Â </td><td>2.5Â </td><td>1.8Â </td><td>99.0Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>QF5Â </td><td>96.3Â </td><td>2.6Â </td><td>1.1Â </td><td>98.5Â </td><td>0.8Â </td><td>0.7Â </td></tr><tr><td>JC9Â </td><td>96.5Â </td><td>1.6Â </td><td>1.9Â </td><td>98.2Â </td><td>1.3Â </td><td>0.5Â </td></tr><tr><td>XF12_18_05_11Â </td><td>97.3Â </td><td>1.4Â </td><td>1.4Â </td><td>98.4Â </td><td>0.3Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.3Â </td><td>0.8Â </td><td>2.0Â </td><td>95.6Â </td><td>2.4Â </td><td>2.0Â </td></tr><tr><td>NF10Â </td><td>97.3Â </td><td>1.5Â </td><td>1.3Â </td><td>98.7Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>DEV_18_05_11Â </td><td>97.3Â </td><td>1.1Â </td><td>1.6Â </td><td>98.9Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>JC3Â </td><td>97.4Â </td><td>1.1Â </td><td>1.6Â </td><td>98.6Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.4Â </td><td>1.1Â </td><td>1.5Â </td><td>98.5Â </td><td>0.5Â </td><td>0.9Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.4Â </td><td>1.2Â </td><td>1.4Â </td><td>99.0Â </td><td>0.6Â </td><td>0.5Â </td></tr><tr><td>WE2Â </td><td>97.5Â </td><td>1.0Â </td><td>1.6Â </td><td>97.9Â </td><td>1.6Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.5Â </td><td>1.1Â </td><td>1.4Â </td><td>97.1Â </td><td>2.4Â </td><td>0.5Â </td></tr><tr><td>TF1Â </td><td>97.6Â </td><td>1.0Â </td><td>1.3Â </td><td>98.8Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>3D7_GlasgowÂ </td><td>97.8Â </td><td>1.1Â </td><td>1.1Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>MedianÂ </td><td>97.3Â </td><td>1.2Â </td><td>1.5Â </td><td>98.5Â </td><td>0.6Â </td><td>0.7Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.7 Â± 1.2Â </td><td>1.8 Â± 1.1Â </td><td>1.6 Â± 0.3Â </td><td>98.0 Â± 1.4Â </td><td>1.3 Â± 1.2Â </td><td>0.8 Â± 0.4Â </td></tr></tbody></table></div><div content-id="DSU028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> SNP calls in all regions<hr />Â </th><th colspan="3"><em>De novo</em> SNP calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th><th>Perfect concordance (%)Â </th><th>WGA new allele (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>JONÂ </td><td>93.8Â </td><td>4.4Â </td><td>1.8Â </td><td>97.2Â </td><td>1.6Â </td><td>1.2Â </td></tr><tr><td>JF6Â </td><td>94.4Â </td><td>4.3Â </td><td>1.3Â </td><td>94.2Â </td><td>5.0Â </td><td>0.8Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.0Â </td><td>2.6Â </td><td>2.4Â </td><td>96.9Â </td><td>2.2Â </td><td>0.9Â </td></tr><tr><td>QF5Â </td><td>95.7Â </td><td>2.5Â </td><td>1.8Â </td><td>99.0Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>QF5Â </td><td>96.3Â </td><td>2.6Â </td><td>1.1Â </td><td>98.5Â </td><td>0.8Â </td><td>0.7Â </td></tr><tr><td>JC9Â </td><td>96.5Â </td><td>1.6Â </td><td>1.9Â </td><td>98.2Â </td><td>1.3Â </td><td>0.5Â </td></tr><tr><td>XF12_18_05_11Â </td><td>97.3Â </td><td>1.4Â </td><td>1.4Â </td><td>98.4Â </td><td>0.3Â </td><td>1.2Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.3Â </td><td>0.8Â </td><td>2.0Â </td><td>95.6Â </td><td>2.4Â </td><td>2.0Â </td></tr><tr><td>NF10Â </td><td>97.3Â </td><td>1.5Â </td><td>1.3Â </td><td>98.7Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>DEV_18_05_11Â </td><td>97.3Â </td><td>1.1Â </td><td>1.6Â </td><td>98.9Â </td><td>0.5Â </td><td>0.5Â </td></tr><tr><td>JC3Â </td><td>97.4Â </td><td>1.1Â </td><td>1.6Â </td><td>98.6Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.4Â </td><td>1.1Â </td><td>1.5Â </td><td>98.5Â </td><td>0.5Â </td><td>0.9Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.4Â </td><td>1.2Â </td><td>1.4Â </td><td>99.0Â </td><td>0.6Â </td><td>0.5Â </td></tr><tr><td>WE2Â </td><td>97.5Â </td><td>1.0Â </td><td>1.6Â </td><td>97.9Â </td><td>1.6Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.5Â </td><td>1.1Â </td><td>1.4Â </td><td>97.1Â </td><td>2.4Â </td><td>0.5Â </td></tr><tr><td>TF1Â </td><td>97.6Â </td><td>1.0Â </td><td>1.3Â </td><td>98.8Â </td><td>0.5Â </td><td>0.7Â </td></tr><tr><td>3D7_GlasgowÂ </td><td>97.8Â </td><td>1.1Â </td><td>1.1Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>MedianÂ </td><td>97.3Â </td><td>1.2Â </td><td>1.5Â </td><td>98.5Â </td><td>0.6Â </td><td>0.7Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.7 Â± 1.2Â </td><td>1.8 Â± 1.1Â </td><td>1.6 Â± 0.3Â </td><td>98.0 Â± 1.4Â </td><td>1.3 Â± 1.2Â </td><td>0.8 Â± 0.4Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p>WGA and non-WGA data were analysed for <em>de novo</em> SNP calling concordance. Matching sequence data from cross samples were paired and the SNP calling was performed simultaneously. Concordance rates were generated by comparing WGA and related non-WGA calls. Columns on the left show comparison of all calls covering the entire genome, whereas columns on the right show a subset of all calls covering only the high-quality coding regions of the genome. Values in the perfect concordance columns represent the proportion of calls that were identical in both WGA and non-WGA samples. WGA new allele columns show the proportion of SNPs that were present only in the WGA and not in their matching non-WGA alleles. Undetermined column shows the proportion of SNPs that were called in wrong or missing alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div></div></p><p>As with SNPs, we also performed biallelic <em>de novo</em> InDel calls with a quality score of â‰¥250 for both WGA and non-WGA samples. As given in TableÂ <a reveal-id="DSU028TB3" data-open="DSU028TB3" class="link link-reveal link-table xref-fig">3</a>, InDel calls from the whole genome with perfect concordance between WGA and non-WGA samples had a median of 97.6% and a mean of 96.8 Â± 1.7%, and WGA-introduced InDels had a median of 1.5% and a mean of 2.4 Â± 1.8%. Similarly, <em>de novo</em> InDel calls in the coding regions only produced near-perfect concordance with a median of 98.7% and a mean of 98.2 Â± 1.4%. WGA-introduced InDels in these regions dropped to a median of 0.4% and a mean of 1.4 Â± 1.3%. <div content-id="DSU028TB3" class="table-modal"><div class="table-wrap"><div id="DSU028TB3" data-id="DSU028TB3" class="table-wrap-title"><div class="label -label">TableÂ 3.</div><div class="caption"><p>WGA and non-WGA InDel concordance analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> InDel calls in whole genome<hr />Â </th><th colspan="3"><em>De novo</em> InDel calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>UndeterminedÂ </th><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>95.2Â </td><td>4.8Â </td><td>0.0Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>JC3Â </td><td>97.6Â </td><td>1.5Â </td><td>0.9Â </td><td>98.8Â </td><td>0.8Â </td><td>0.4Â </td></tr><tr><td>QF5Â </td><td>97.2Â </td><td>2.1Â </td><td>0.6Â </td><td>96.1Â </td><td>3.6Â </td><td>0.3Â </td></tr><tr><td>JF6Â </td><td>91.5Â </td><td>7.8Â </td><td>0.7Â </td><td>96.5Â </td><td>3.4Â </td><td>0.2Â </td></tr><tr><td>TF1Â </td><td>98.2Â </td><td>1.2Â </td><td>0.5Â </td><td>99.4Â </td><td>0.5Â </td><td>0.1Â </td></tr><tr><td>DEV_18_05_11Â </td><td>98.1Â </td><td>1.0Â </td><td>0.8Â </td><td>99.2Â </td><td>0.3Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.8Â </td><td>98.7Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.9Â </td><td>98.6Â </td><td>0.7Â </td><td>0.7Â </td></tr><tr><td>WE2Â </td><td>98.0Â </td><td>1.1Â </td><td>0.9Â </td><td>99.0Â </td><td>0.7Â </td><td>0.3Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.8Â </td><td>2.5Â </td><td>1.7Â </td><td>97.1Â </td><td>1.6Â </td><td>1.3Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.8Â </td><td>1.0Â </td><td>1.2Â </td><td>99.1Â </td><td>0.3Â </td><td>0.6Â </td></tr><tr><td>NF10Â </td><td>97.4Â </td><td>1.8Â </td><td>0.8Â </td><td>98.3Â </td><td>1.1Â </td><td>0.6Â </td></tr><tr><td>XF12_18_05_11Â </td><td>98.0Â </td><td>1.4Â </td><td>0.6Â </td><td>99.0Â </td><td>0.6Â </td><td>0.4Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.7Â </td><td>1.5Â </td><td>0.8Â </td><td>99.3Â </td><td>0.3Â </td><td>0.3Â </td></tr><tr><td>XD8Â </td><td>95.5Â </td><td>3.9Â </td><td>0.7Â </td><td>95.7Â </td><td>3.6Â </td><td>0.7Â </td></tr><tr><td>JONÂ </td><td>95.2Â </td><td>4.2Â </td><td>0.6Â </td><td>96.1Â </td><td>3.7Â </td><td>0.3Â </td></tr><tr><td>JC9Â </td><td>96.4Â </td><td>2.9Â </td><td>0.7Â </td><td>98.4Â </td><td>1.5Â </td><td>0.1Â </td></tr><tr><td>MedianÂ </td><td>97.6Â </td><td>1.5Â </td><td>0.8Â </td><td>98.7Â </td><td>0.7Â </td><td>0.4Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.8 Â± 1.7Â </td><td>2.4 Â± 1.8Â </td><td>0.8 Â± 0.3Â </td><td>98.2 Â± 1.4Â </td><td>1.4 Â± 1.3Â </td><td>0.4 Â± 0.3Â </td></tr></tbody></table></div><div content-id="DSU028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> InDel calls in whole genome<hr />Â </th><th colspan="3"><em>De novo</em> InDel calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>UndeterminedÂ </th><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>95.2Â </td><td>4.8Â </td><td>0.0Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>JC3Â </td><td>97.6Â </td><td>1.5Â </td><td>0.9Â </td><td>98.8Â </td><td>0.8Â </td><td>0.4Â </td></tr><tr><td>QF5Â </td><td>97.2Â </td><td>2.1Â </td><td>0.6Â </td><td>96.1Â </td><td>3.6Â </td><td>0.3Â </td></tr><tr><td>JF6Â </td><td>91.5Â </td><td>7.8Â </td><td>0.7Â </td><td>96.5Â </td><td>3.4Â </td><td>0.2Â </td></tr><tr><td>TF1Â </td><td>98.2Â </td><td>1.2Â </td><td>0.5Â </td><td>99.4Â </td><td>0.5Â </td><td>0.1Â </td></tr><tr><td>DEV_18_05_11Â </td><td>98.1Â </td><td>1.0Â </td><td>0.8Â </td><td>99.2Â </td><td>0.3Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.8Â </td><td>98.7Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.9Â </td><td>98.6Â </td><td>0.7Â </td><td>0.7Â </td></tr><tr><td>WE2Â </td><td>98.0Â </td><td>1.1Â </td><td>0.9Â </td><td>99.0Â </td><td>0.7Â </td><td>0.3Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.8Â </td><td>2.5Â </td><td>1.7Â </td><td>97.1Â </td><td>1.6Â </td><td>1.3Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.8Â </td><td>1.0Â </td><td>1.2Â </td><td>99.1Â </td><td>0.3Â </td><td>0.6Â </td></tr><tr><td>NF10Â </td><td>97.4Â </td><td>1.8Â </td><td>0.8Â </td><td>98.3Â </td><td>1.1Â </td><td>0.6Â </td></tr><tr><td>XF12_18_05_11Â </td><td>98.0Â </td><td>1.4Â </td><td>0.6Â </td><td>99.0Â </td><td>0.6Â </td><td>0.4Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.7Â </td><td>1.5Â </td><td>0.8Â </td><td>99.3Â </td><td>0.3Â </td><td>0.3Â </td></tr><tr><td>XD8Â </td><td>95.5Â </td><td>3.9Â </td><td>0.7Â </td><td>95.7Â </td><td>3.6Â </td><td>0.7Â </td></tr><tr><td>JONÂ </td><td>95.2Â </td><td>4.2Â </td><td>0.6Â </td><td>96.1Â </td><td>3.7Â </td><td>0.3Â </td></tr><tr><td>JC9Â </td><td>96.4Â </td><td>2.9Â </td><td>0.7Â </td><td>98.4Â </td><td>1.5Â </td><td>0.1Â </td></tr><tr><td>MedianÂ </td><td>97.6Â </td><td>1.5Â </td><td>0.8Â </td><td>98.7Â </td><td>0.7Â </td><td>0.4Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.8 Â± 1.7Â </td><td>2.4 Â± 1.8Â </td><td>0.8 Â± 0.3Â </td><td>98.2 Â± 1.4Â </td><td>1.4 Â± 1.3Â </td><td>0.4 Â± 0.3Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>WGA and non-WGA data were analysed for <em>de novo</em> InDel calling concordance. Calls were performed simultaneously on both WGA and related non-WGA. Columns on the left show comparison of all calls covering the entire genome, whereas columns on the right show a subset of all calls covering only the high-quality coding regions of the genome. Values in the perfect concordance columns represent the proportion of calls that were identical in both WGA and non-WGA samples. WGA new allele columns show the proportion of InDels that were present only in the WGA and not in their matching non-WGA alleles. Undetermined column shows the proportion of InDels that were called in wrong or missing alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU028TB3" data-id="DSU028TB3" class="table-wrap-title"><div class="label -label">TableÂ 3.</div><div class="caption"><p>WGA and non-WGA InDel concordance analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> InDel calls in whole genome<hr />Â </th><th colspan="3"><em>De novo</em> InDel calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>UndeterminedÂ </th><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>95.2Â </td><td>4.8Â </td><td>0.0Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>JC3Â </td><td>97.6Â </td><td>1.5Â </td><td>0.9Â </td><td>98.8Â </td><td>0.8Â </td><td>0.4Â </td></tr><tr><td>QF5Â </td><td>97.2Â </td><td>2.1Â </td><td>0.6Â </td><td>96.1Â </td><td>3.6Â </td><td>0.3Â </td></tr><tr><td>JF6Â </td><td>91.5Â </td><td>7.8Â </td><td>0.7Â </td><td>96.5Â </td><td>3.4Â </td><td>0.2Â </td></tr><tr><td>TF1Â </td><td>98.2Â </td><td>1.2Â </td><td>0.5Â </td><td>99.4Â </td><td>0.5Â </td><td>0.1Â </td></tr><tr><td>DEV_18_05_11Â </td><td>98.1Â </td><td>1.0Â </td><td>0.8Â </td><td>99.2Â </td><td>0.3Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.8Â </td><td>98.7Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.9Â </td><td>98.6Â </td><td>0.7Â </td><td>0.7Â </td></tr><tr><td>WE2Â </td><td>98.0Â </td><td>1.1Â </td><td>0.9Â </td><td>99.0Â </td><td>0.7Â </td><td>0.3Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.8Â </td><td>2.5Â </td><td>1.7Â </td><td>97.1Â </td><td>1.6Â </td><td>1.3Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.8Â </td><td>1.0Â </td><td>1.2Â </td><td>99.1Â </td><td>0.3Â </td><td>0.6Â </td></tr><tr><td>NF10Â </td><td>97.4Â </td><td>1.8Â </td><td>0.8Â </td><td>98.3Â </td><td>1.1Â </td><td>0.6Â </td></tr><tr><td>XF12_18_05_11Â </td><td>98.0Â </td><td>1.4Â </td><td>0.6Â </td><td>99.0Â </td><td>0.6Â </td><td>0.4Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.7Â </td><td>1.5Â </td><td>0.8Â </td><td>99.3Â </td><td>0.3Â </td><td>0.3Â </td></tr><tr><td>XD8Â </td><td>95.5Â </td><td>3.9Â </td><td>0.7Â </td><td>95.7Â </td><td>3.6Â </td><td>0.7Â </td></tr><tr><td>JONÂ </td><td>95.2Â </td><td>4.2Â </td><td>0.6Â </td><td>96.1Â </td><td>3.7Â </td><td>0.3Â </td></tr><tr><td>JC9Â </td><td>96.4Â </td><td>2.9Â </td><td>0.7Â </td><td>98.4Â </td><td>1.5Â </td><td>0.1Â </td></tr><tr><td>MedianÂ </td><td>97.6Â </td><td>1.5Â </td><td>0.8Â </td><td>98.7Â </td><td>0.7Â </td><td>0.4Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.8 Â± 1.7Â </td><td>2.4 Â± 1.8Â </td><td>0.8 Â± 0.3Â </td><td>98.2 Â± 1.4Â </td><td>1.4 Â± 1.3Â </td><td>0.4 Â± 0.3Â </td></tr></tbody></table></div><div content-id="DSU028TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SampleÂ </th><th colspan="3"><em>De novo</em> InDel calls in whole genome<hr />Â </th><th colspan="3"><em>De novo</em> InDel calls in good quality regions<hr />Â </th></tr><tr><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>UndeterminedÂ </th><th>Perfect concordance (%)Â </th><th>WGA New InDels (%)Â </th><th>Undetermined (%)Â </th></tr></thead><tbody><tr><td>3D7_GlasgowÂ </td><td>95.2Â </td><td>4.8Â </td><td>0.0Â </td><td>100.0Â </td><td>0.0Â </td><td>0.0Â </td></tr><tr><td>JC3Â </td><td>97.6Â </td><td>1.5Â </td><td>0.9Â </td><td>98.8Â </td><td>0.8Â </td><td>0.4Â </td></tr><tr><td>QF5Â </td><td>97.2Â </td><td>2.1Â </td><td>0.6Â </td><td>96.1Â </td><td>3.6Â </td><td>0.3Â </td></tr><tr><td>JF6Â </td><td>91.5Â </td><td>7.8Â </td><td>0.7Â </td><td>96.5Â </td><td>3.4Â </td><td>0.2Â </td></tr><tr><td>TF1Â </td><td>98.2Â </td><td>1.2Â </td><td>0.5Â </td><td>99.4Â </td><td>0.5Â </td><td>0.1Â </td></tr><tr><td>DEV_18_05_11Â </td><td>98.1Â </td><td>1.0Â </td><td>0.8Â </td><td>99.2Â </td><td>0.3Â </td><td>0.5Â </td></tr><tr><td>7G8_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.8Â </td><td>98.7Â </td><td>0.6Â </td><td>0.8Â </td></tr><tr><td>GB4_NIHÂ </td><td>97.9Â </td><td>1.2Â </td><td>0.9Â </td><td>98.6Â </td><td>0.7Â </td><td>0.7Â </td></tr><tr><td>WE2Â </td><td>98.0Â </td><td>1.1Â </td><td>0.9Â </td><td>99.0Â </td><td>0.7Â </td><td>0.3Â </td></tr><tr><td>D2_18_05_11Â </td><td>95.8Â </td><td>2.5Â </td><td>1.7Â </td><td>97.1Â </td><td>1.6Â </td><td>1.3Â </td></tr><tr><td>NIC_18_05_11Â </td><td>97.8Â </td><td>1.0Â </td><td>1.2Â </td><td>99.1Â </td><td>0.3Â </td><td>0.6Â </td></tr><tr><td>NF10Â </td><td>97.4Â </td><td>1.8Â </td><td>0.8Â </td><td>98.3Â </td><td>1.1Â </td><td>0.6Â </td></tr><tr><td>XF12_18_05_11Â </td><td>98.0Â </td><td>1.4Â </td><td>0.6Â </td><td>99.0Â </td><td>0.6Â </td><td>0.4Â </td></tr><tr><td>AL2_13_05_11Â </td><td>97.7Â </td><td>1.5Â </td><td>0.8Â </td><td>99.3Â </td><td>0.3Â </td><td>0.3Â </td></tr><tr><td>XD8Â </td><td>95.5Â </td><td>3.9Â </td><td>0.7Â </td><td>95.7Â </td><td>3.6Â </td><td>0.7Â </td></tr><tr><td>JONÂ </td><td>95.2Â </td><td>4.2Â </td><td>0.6Â </td><td>96.1Â </td><td>3.7Â </td><td>0.3Â </td></tr><tr><td>JC9Â </td><td>96.4Â </td><td>2.9Â </td><td>0.7Â </td><td>98.4Â </td><td>1.5Â </td><td>0.1Â </td></tr><tr><td>MedianÂ </td><td>97.6Â </td><td>1.5Â </td><td>0.8Â </td><td>98.7Â </td><td>0.7Â </td><td>0.4Â </td></tr><tr><td>Mean Â± SDÂ </td><td>96.8 Â± 1.7Â </td><td>2.4 Â± 1.8Â </td><td>0.8 Â± 0.3Â </td><td>98.2 Â± 1.4Â </td><td>1.4 Â± 1.3Â </td><td>0.4 Â± 0.3Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN3"><div class="fn"><p>WGA and non-WGA data were analysed for <em>de novo</em> InDel calling concordance. Calls were performed simultaneously on both WGA and related non-WGA. Columns on the left show comparison of all calls covering the entire genome, whereas columns on the right show a subset of all calls covering only the high-quality coding regions of the genome. Values in the perfect concordance columns represent the proportion of calls that were identical in both WGA and non-WGA samples. WGA new allele columns show the proportion of InDels that were present only in the WGA and not in their matching non-WGA alleles. Undetermined column shows the proportion of InDels that were called in wrong or missing alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div></div></p>                            <h4 scrollto-destination=87197873 id="87197873" class="section-title" data-legacy-id=s3f2>3.6.2. Concordance of in silico genotyping</h4>
<div class="block-child-p">Using a list of high-quality SNP positions and alleles from the genetic crosses, we performed <em>in silico</em> genotyping of both the WGA and non-WGA samples on 20,737 positions, counting alleles present in at least five reads. Call comparisons between the WGA and non-WGA from same samples were grouped into â€˜Perfect Concordance (identical)â€™, â€˜WGA Missing Allelesâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™ (missing allele). Identical alleles with perfect concordance between WGA and non-WGA samples had a median of 97.95% (Fig.Â <a reveal-id="DSU028F6" data-open="DSU028F6" class="link link-reveal link-table xref-fig">6</a>). A median of 0.48% calls represent alleles that were called in the WGA, but not in the non-WGA, samples. These reflect cases where the WGA sample had a mixed call and the non-WGA sample had a single-allele call. A median of 0.94% represent alleles that were missing in WGA, but were called in non-WGA samples. Discordant single-allele calls were extremely rare, with a median of zero and a mean of 2.9 SNPs per sample. Four of the 17 samples showed such discordant calls, and only three samples had more than one such call. The 3D7 (3D7_Glasgow) reference sample showed 99.58% identical calls, and only one new allele in WGA sample was returned as a mixed call (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). <div data-id="DSU028F6" class="fig fig-section"><div class="label fig-label">FigureÂ 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02806.jpeg?Expires=2147483647&amp;Signature=H6L7-~tlERQGE-LUuorIk0WEb3iFt1k4Ai9YOpYj~HnV~l7XxDc4Bw8TPTt9cNxEvjrAefFNybMf6Pe--EDpMnm~JGnC4Li~NGdckH43uodziAlY~Cqh0NcGABlXM2B4072DF5GxR6ambQ441C6RKGErM4yxB6P24Zo87FhCbTTIV~veKvtuBzOs5~egSqEE6TiYKC388hQGL6rthZznxm-RYu4oR-k94WnL~gqckPQLvRbmV81M99ZfYNRK0lrYWW3ss-C1BOfCSBFM0KfvI4bYI8K1jZ1x-~XNPmIYvnZHOmo4EKbh2uNIKqM1vS4V9H9xq0cYWWXZ3E7ddPeauw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Boxplot of in silico genotyping concordance analysing. A total of 20,737 high-quality SNP positions were genotyped in both WGA and non-WGA samples. Call comparison was performed between matched pairs of corresponding WGA and non-WGA datasets and results were grouped into â€˜Perfect Concordanceâ€™, â€˜WGA Missing Allelesâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™. Perfect_Concordance (identical alleles) had a median of 97.95% (range 83.11â€“99.58%; n = 17). A median of 0.48% calls represent alleles that were called in the WGA samples, but not in the non-WGA samples (WGA_New_Alleles). A median of 0.94% calls were present in non-WGA, but absent in WGA, samples (WGA_Missing_Alleles). The proportion of calls that were missing (Undetermined) had a median of 0.96% (Supplementary Data." path-from-xml="dsu02806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197874/dsu02806.jpeg" path-from-xml="dsu02806.jpeg" target="_blank">View large</a><a section="87197874" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02806.jpeg?Expires=2147483647&amp;Signature=JfU4qrFF3FdMr9--8ugaU4dhRwJHOjlmL7QhA~H6NwKwmNSbVHXRp6ZEAR0oY17KXAEEim9FwoFbvhN1XpVvbobnq44AieYfmX60AidxUfm577WK9F0~vTWsw2pGjhOF9pWa7Mel~8dIk49X~W8YPiDX49kvOSaqww9cVtxgKRUiMLaepXPyNcD~chW~LSdaP0XJq19WzJOjjpBhHTcCoq2VUKWxhwCmJAqmtTRA~lnf7afBiPJiP61T2T23Czp5PeJvv8~K1ppJ-K~C~rzWZIjiodh01cBkpXqF028hjLz2ODPYoLacoaU53oR4ME~MoyMhYWZ8UgesSK7uprUfyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197874&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Boxplot of <em>in silico</em> genotyping concordance analysing. A total of 20,737 high-quality SNP positions were genotyped in both WGA and non-WGA samples. Call comparison was performed between matched pairs of corresponding WGA and non-WGA datasets and results were grouped into â€˜Perfect Concordanceâ€™, â€˜WGA Missing Allelesâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™. Perfect_Concordance (identical alleles) had a median of 97.95% (range 83.11â€“99.58%; <em>n</em> = 17). A median of 0.48% calls represent alleles that were called in the WGA samples, but not in the non-WGA samples (WGA_New_Alleles). A median of 0.94% calls were present in non-WGA, but absent in WGA, samples (WGA_Missing_Alleles). The proportion of calls that were missing (Undetermined) had a median of 0.96% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div><div content-id="DSU028F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/m_dsu02806.jpeg?Expires=2147483647&amp;Signature=H6L7-~tlERQGE-LUuorIk0WEb3iFt1k4Ai9YOpYj~HnV~l7XxDc4Bw8TPTt9cNxEvjrAefFNybMf6Pe--EDpMnm~JGnC4Li~NGdckH43uodziAlY~Cqh0NcGABlXM2B4072DF5GxR6ambQ441C6RKGErM4yxB6P24Zo87FhCbTTIV~veKvtuBzOs5~egSqEE6TiYKC388hQGL6rthZznxm-RYu4oR-k94WnL~gqckPQLvRbmV81M99ZfYNRK0lrYWW3ss-C1BOfCSBFM0KfvI4bYI8K1jZ1x-~XNPmIYvnZHOmo4EKbh2uNIKqM1vS4V9H9xq0cYWWXZ3E7ddPeauw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Boxplot of in silico genotyping concordance analysing. A total of 20,737 high-quality SNP positions were genotyped in both WGA and non-WGA samples. Call comparison was performed between matched pairs of corresponding WGA and non-WGA datasets and results were grouped into â€˜Perfect Concordanceâ€™, â€˜WGA Missing Allelesâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™. Perfect_Concordance (identical alleles) had a median of 97.95% (range 83.11â€“99.58%; n = 17). A median of 0.48% calls represent alleles that were called in the WGA samples, but not in the non-WGA samples (WGA_New_Alleles). A median of 0.94% calls were present in non-WGA, but absent in WGA, samples (WGA_Missing_Alleles). The proportion of calls that were missing (Undetermined) had a median of 0.96% (Supplementary Data." path-from-xml="dsu02806.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197874/dsu02806.jpeg" path-from-xml="dsu02806.jpeg" target="_blank">View large</a><a section="87197874" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu02806.jpeg?Expires=2147483647&amp;Signature=JfU4qrFF3FdMr9--8ugaU4dhRwJHOjlmL7QhA~H6NwKwmNSbVHXRp6ZEAR0oY17KXAEEim9FwoFbvhN1XpVvbobnq44AieYfmX60AidxUfm577WK9F0~vTWsw2pGjhOF9pWa7Mel~8dIk49X~W8YPiDX49kvOSaqww9cVtxgKRUiMLaepXPyNcD~chW~LSdaP0XJq19WzJOjjpBhHTcCoq2VUKWxhwCmJAqmtTRA~lnf7afBiPJiP61T2T23Czp5PeJvv8~K1ppJ-K~C~rzWZIjiodh01cBkpXqF028hjLz2ODPYoLacoaU53oR4ME~MoyMhYWZ8UgesSK7uprUfyw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197874&amp;ar=2754550&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02806.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Boxplot of <em>in silico</em> genotyping concordance analysing. A total of 20,737 high-quality SNP positions were genotyped in both WGA and non-WGA samples. Call comparison was performed between matched pairs of corresponding WGA and non-WGA datasets and results were grouped into â€˜Perfect Concordanceâ€™, â€˜WGA Missing Allelesâ€™, â€˜WGA New Allelesâ€™ and â€˜Undeterminedâ€™. Perfect_Concordance (identical alleles) had a median of 97.95% (range 83.11â€“99.58%; <em>n</em> = 17). A median of 0.48% calls represent alleles that were called in the WGA samples, but not in the non-WGA samples (WGA_New_Alleles). A median of 0.94% calls were present in non-WGA, but absent in WGA, samples (WGA_Missing_Alleles). The proportion of calls that were missing (Undetermined) had a median of 0.96% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div>                            <h2 scrollto-destination=87197875 id="87197875" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>High-throughput DNA sequencing technologies have gained a wide range of applications in infectious disease research, including global surveillance of the emergence and spread of drug resistance, detection of regions of the genome under selection and identification of genetic determinants of clinical phenotypes through genome-wide association studies. The <em>P. falciparum</em> genome presents inherent technical challenges to next-generation DNA sequencing, such as its extreme AT-richness. Whole-genome sequencing of <em>P. falciparum</em> field isolates poses additional difficulties due to the small quantity of DNA often retrieved from patients in the clinical setting and the high degree of host DNA contamination from leucocytes. This study addressed the first of these key problems by developing a method for WGA capable of producing high-quality sequence data from just 10 pg of input <em>P. falciparum</em> DNA.</p><p>Although PCR-based WGA techniques, such as primer extension PCR, ligation-mediated PCR and degenerate oligonucleotide-primed PCR, have been successfully used in some studies such as single-cell amplification,<sup><a reveal-id="DSU028C29" data-open="DSU028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> their wider application has been limited. PCR-based WGA methods produce relatively shorter products, non-specific amplification artefacts and incomplete genome coverage.<sup><a reveal-id="DSU028C19" data-open="DSU028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU028C20" data-open="DSU028C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSU028C30" data-open="DSU028C30" class="link link-ref link-reveal xref-bibr">30</a></sup> MDA has been associated with base-bias and generation of chimeric products.<sup><a reveal-id="DSU028C29" data-open="DSU028C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Nonetheless, MDA has been the method of choice for a wider range of genome amplification studies, because it produces longer DNA products with extensive genome coverage. MDA also produces higher DNA yields with relatively less amplification bias.<sup><a reveal-id="DSU028C3" data-open="DSU028C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSU028C11" data-open="DSU028C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSU028C19" data-open="DSU028C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>Here, we have assessed MDA on an AT-rich genome using a range of input DNA quantities. We describe an optimized WGA method incorporating TMAC reagent that improves amplification coverage of the difficult AT-rich loci of the genome. We establish 10 pg of input DNA as the lowest threshold from which our optimized WGA protocol generates an amplification product with optimal <em>P. falciparum</em> genome coverage for most genome sequencing analysis. This amount equates to âˆ¼380 parasite genomes, equivalent to âˆ¼1 Âµl of blood in a patient with 0.01% parasitemia. Furthermore, we show that the optimized conditions significantly reduce the formation of chimeric reads, thereby improving the overall quality of the amplified product.</p><p>Standard WGA from such small quantities of input material is often associated with bias and incomplete genome coverage. A single-cell genomic approach that uses infected red blood cell sorting technology has recently been reported that achieved a genome coverage of âˆ¼50% with a standard WGA method.<sup><a reveal-id="DSU028C31" data-open="DSU028C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Although genome coverage in single-cell approach is still low, this technology has opened avenues for single-cell genomics studies in malaria and offers great opportunities for dissecting multiple genotype infection. Our optimized WGA method described here will be useful for optimizing genome coverage in such single-cell genome amplifications, as well as direct field applications for small sample sequencing. For improved malaria clinical sequencing, we routinely employ the combination of host depletion methods<sup><a reveal-id="DSU028C32" data-open="DSU028C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSU028C33" data-open="DSU028C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and the current optimized WGA procedure to generate high-quality whole genome sequencing data.</p><p>We have assessed the quality of the amplified products generated from 10 pg input DNA using high-throughput sequencing of DNA extracted from 17 progeny clones derived from genetic crosses between two laboratory strains. We performed a comparative analysis between the WGA data and their corresponding non-WGA counterparts. We show that coverage was sufficient for allele calling and other whole-genome analyses. Although the number of uncovered bases was slightly higher in WGA (median, 1.5%) than non-WGA (median, 1.2%) samples (TableÂ <a reveal-id="DSU028TB1" data-open="DSU028TB1" class="link link-reveal link-table xref-fig">1</a>), the difference was insignificant (<em>t</em>-test, <em>P</em> = 0.053, 95% CI). The same applies to preference for mitochondria DNA amplification, which showed a median of 2.6% in WGA against 0.6% in the non-WGA samples.</p><p>Another key aspect of <em>P. falciparum</em> whole-genome sequencing from clinical specimens is removing host DNA contamination. This can be achieved either at the blood sample processing stage through leucocyte depletion or through selective enrichment of parasite DNA after extraction.<sup><a reveal-id="DSU028C32" data-open="DSU028C32" class="link link-ref link-reveal xref-bibr">32</a>â€“<a reveal-id="DSU028C34" data-open="DSU028C34" class="link link-ref link-reveal xref-bibr">34</a></sup> The combination of an effective method for removing human DNA that is applicable to the field setting, and the ability to perform whole-genome sequencing from very low quantities of input DNA as described in this study, has the potential to greatly increase the scope and scale of <em>P. falciparum</em> genomic research. This, in turn, would contribute significantly to malaria genetic surveillance and control strategies.</p>                            <h2 scrollto-destination=87197882 id="87197882" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>The optimized amplification conditions described here have generated high-quality whole-genome sequence data (99.8% genome coverage) from a minute amount of input DNA, equivalent to &lt;400 <em>P. falciparum</em> genomes. This work shows for the first time that accurate <em>in silico</em> genotyping and <em>de novo</em> calling of genetic variants is achievable on a WGA sample using &lt;1 ng of input DNA from an extremely AT-rich genome. We anticipate that sequencing from small quantities of input DNA (&lt;1 ng) will become a significant aid to genetic and genomic studies of <em>P. falciparum</em> in the field, particularly when combined with effective methods for removal of host DNA contamination.</p>                            <h2 scrollto-destination=87197884 id="87197884" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This research was supported by the Wellcome Trust through the Wellcome Trust Sanger Institute (098051), the Resource Centre for Genomic Epidemiology of Malaria (090770/Z/09/Z) and the Wellcome Trust Centre for Human Genetics (090532/Z/09/Z). The Centre for Genomics and Global Health is supported by the Medical Research Council (G0600718). Funding to pay the Open Access publication charges for this article was provided by the Wellcome Trust Sanger Institute.</p>                            <h2 scrollto-destination=87197886 id="87197886" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Jean-Bosco Ouedraogo (Institut de Recherche en Sciences de la SantÃ©, Burkina Faso) and Francois Nosten (Nuffield Department of Medicine, University of Oxford) for providing field samples for analysis. We thank Richard Pearson and Thomas Otto for useful discussions on sequence analysis.</p><p><em>Conflict of Interest statement</em>. None declared.</p>                            <h2 scrollto-destination=87197889 id="87197889" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miotto</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Campino</div>
               <div class="given-names">S.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Analysis of <em>Plasmodium falciparum</em> diversity in natural infections by deep sequencing</div>
          <div class="source">Nature</div>Â 
          <div class="year">2012</div>
          <div class="volume">487</div>
          <div class="fpage">375</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20Plasmodium%20falciparum%20diversity%20in%20natural%20infections%20by%20deep%20sequencing&amp;author=M.%20Manske&amp;author=O.%20Miotto&amp;author=S.%20Campino&amp;publication_year=2012&amp;journal=Nature&amp;volume=487&amp;pages=375-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22722859" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Miotto</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Almagro-Garcia</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Multiple populations of artemisinin-resistant <em>Plasmodium falciparum</em> in Cambodia</div>
          <div class="source">Nat. Genet.</div>Â 
          <div class="year">2013</div>
          <div class="volume">45</div>
          <div class="fpage">648</div>
          <div class="lpage">55</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20populations%20of%20artemisinin-resistant%20Plasmodium%20falciparum%20in%20Cambodia&amp;author=O.%20Miotto&amp;author=J.%20Almagro-Garcia&amp;author=M.%20Manske&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=648-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2624" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2624" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2624"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23624527" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lasken</div>
               <div class="given-names">R.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Egholm</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Whole genome amplification: abundant supplies of DNA from precious samples or clinical specimens</div>
          <div class="source">Trends Biotechnol.</div>Â 
          <div class="year">2003</div>
          <div class="volume">21</div>
          <div class="fpage">531</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%3A%20abundant%20supplies%20of%20DNA%20from%20precious%20samples%20or%20clinical%20specimens&amp;author=R.S.%20Lasken&amp;author=M.%20Egholm&amp;publication_year=2003&amp;journal=Trends%20Biotechnol.&amp;volume=21&amp;pages=531-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2003.09.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2003.09.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2003.09.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14624861" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coombs</div>
               <div class="given-names">N.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gough</div>
               <div class="given-names">A.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Primrose</div>
               <div class="given-names">J.N.</div>
            </div>
          </div>
          <div class="article-title">Optimisation of DNA and RNA extraction from archival formalin-fixed tissue</div>
          <div class="source">Nucleic Acids Res.</div>Â 
          <div class="year">1999</div>
          <div class="volume">27</div>
          <div class="fpage">e12</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimisation%20of%20DNA%20and%20RNA%20extraction%20from%20archival%20formalin-fixed%20tissue&amp;author=N.J.%20Coombs&amp;author=A.C.%20Gough&amp;author=J.N.%20Primrose&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=e12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.16.e12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.16.e12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.16.e12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10454649" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Farnert</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Arez</div>
               <div class="given-names">A.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Correia</div>
               <div class="given-names">A.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bjorkman</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Snounou</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">do Rosario</div>
               <div class="given-names">V.</div>
            </div>
          </div>
          <div class="article-title">Sampling and storage of blood and the detection of malaria parasites by polymerase chain reaction</div>
          <div class="source">Trans. R. Soc. Trop. Med. Hyg.</div>Â 
          <div class="year">1999</div>
          <div class="volume">93</div>
          <div class="fpage">50</div>
          <div class="lpage">3</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sampling%20and%20storage%20of%20blood%20and%20the%20detection%20of%20malaria%20parasites%20by%20polymerase%20chain%20reaction&amp;author=A.%20Farnert&amp;author=A.P.%20Arez&amp;author=A.T.%20Correia&amp;author=A.%20Bjorkman&amp;author=G.%20Snounou&amp;author=V.%20do%20Rosario&amp;publication_year=1999&amp;journal=Trans.%20R.%20Soc.%20Trop.%20Med.%20Hyg.&amp;volume=93&amp;pages=50-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0035-9203(99)90177-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0035-9203(99)90177-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0035-9203(99)90177-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10492790" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jackson</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laikre</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Baker</div>
               <div class="given-names">C.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kendall</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">Guidelines for collecting and maintaining archives for genetic monitoring</div>
          <div class="source">Conservation Genet. Resour.</div>Â 
          <div class="year">2012</div>
          <div class="volume">4</div>
          <div class="fpage">527</div>
          <div class="lpage">36</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Guidelines%20for%20collecting%20and%20maintaining%20archives%20for%20genetic%20monitoring&amp;author=J.%20Jackson&amp;author=L.%20Laikre&amp;author=C.S.%20Baker&amp;author=K.%20Kendall&amp;publication_year=2012&amp;journal=Conservation%20Genet.%20Resour.&amp;volume=4&amp;pages=527-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12686-011-9545-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12686-011-9545-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12686-011-9545-x"> </span></div>Â </div></div></div></div></div><div content-id="DSU028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shi</div>
               <div class="given-names">S.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cote</div>
               <div class="given-names">R.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">L.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">DNA extraction from archival formalin-fixed, paraffin-embedded tissue sections based on the antigen retrieval principle: heating under the influence of pH</div>
          <div class="source">J. Histochem. Cytochem.</div>Â 
          <div class="year">2002</div>
          <div class="volume">50</div>
          <div class="fpage">1005</div>
          <div class="lpage">11</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20extraction%20from%20archival%20formalin-fixed%2C%20paraffin-embedded%20tissue%20sections%20based%20on%20the%20antigen%20retrieval%20principle%3A%20heating%20under%20the%20influence%20of%20pH&amp;author=S.R.%20Shi&amp;author=R.J.%20Cote&amp;author=L.%20Wu&amp;publication_year=2002&amp;journal=J.%20Histochem.%20Cytochem.&amp;volume=50&amp;pages=1005-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/002215540205000802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F002215540205000802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F002215540205000802"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12133903" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bereczky</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Martensson</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gil</div>
               <div class="given-names">J.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Farnert</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Short report: rapid DNA extraction from archive blood spots on filter paper for genotyping of <em>Plasmodium falciparum</em></div>
          <div class="source">Am. J. Trop. Med. Hyg.</div>Â 
          <div class="year">2005</div>
          <div class="volume">72</div>
          <div class="fpage">249</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Short%20report%3A%20rapid%20DNA%20extraction%20from%20archive%20blood%20spots%20on%20filter%20paper%20for%20genotyping%20of%20Plasmodium%20falciparum&amp;author=S.%20Bereczky&amp;author=A.%20Martensson&amp;author=J.P.%20Gil&amp;author=A.%20Farnert&amp;publication_year=2005&amp;journal=Am.%20J.%20Trop.%20Med.%20Hyg.&amp;volume=72&amp;pages=249-51" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15772315" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hwang</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jaroensuk</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Leimanis</div>
               <div class="given-names">M.L.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Long-term storage limits PCR-based analyses of malaria parasites in archival dried blood spots</div>
          <div class="source">Malar. J.</div>Â 
          <div class="year">2012</div>
          <div class="volume">11</div>
          <div class="fpage">339</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-term%20storage%20limits%20PCR-based%20analyses%20of%20malaria%20parasites%20in%20archival%20dried%20blood%20spots&amp;author=J.%20Hwang&amp;author=J.%20Jaroensuk&amp;author=M.L.%20Leimanis&amp;publication_year=2012&amp;journal=Malar.%20J.&amp;volume=11&amp;pages=339" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2875-11-339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2875-11-339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2875-11-339"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23043522" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kain</div>
               <div class="given-names">K.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lanar</div>
               <div class="given-names">D.E.</div>
            </div>
          </div>
          <div class="article-title">Determination of genetic variation within <em>Plasmodium falciparum</em> by using enzymatically amplified DNA from filter paper disks impregnated with whole blood</div>
          <div class="source">J. Clin. Microbiol.</div>Â 
          <div class="year">1991</div>
          <div class="volume">29</div>
          <div class="fpage">1171</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20genetic%20variation%20within%20Plasmodium%20falciparum%20by%20using%20enzymatically%20amplified%20DNA%20from%20filter%20paper%20disks%20impregnated%20with%20whole%20blood&amp;author=K.C.%20Kain&amp;author=D.E.%20Lanar&amp;publication_year=1991&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=29&amp;pages=1171-4" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1864936" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spits</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Le Caignec</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">De Rycke</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Whole-genome multiple displacement amplification from single cells</div>
          <div class="source">Nat. Protoc.</div>Â 
          <div class="year">2006</div>
          <div class="volume">1</div>
          <div class="fpage">1965</div>
          <div class="lpage">70</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20multiple%20displacement%20amplification%20from%20single%20cells&amp;author=C.%20Spits&amp;author=C.%20Le%20Caignec&amp;author=M.%20De%20Rycke&amp;publication_year=2006&amp;journal=Nat.%20Protoc.&amp;volume=1&amp;pages=1965-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2006.326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2006.326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2006.326"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17487184" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dean</div>
               <div class="given-names">F.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hosono</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fang</div>
               <div class="given-names">L.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Comprehensive human genome amplification using multiple displacement amplification</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2002</div>
          <div class="volume">99</div>
          <div class="fpage">5261</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20human%20genome%20amplification%20using%20multiple%20displacement%20amplification&amp;author=F.B.%20Dean&amp;author=S.%20Hosono&amp;author=L.%20Fang&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=5261-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.082089499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.082089499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.082089499"> </span></div>Â </div></div></div></div></div><div content-id="DSU028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dean</div>
               <div class="given-names">F.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nelson</div>
               <div class="given-names">J.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Giesler</div>
               <div class="given-names">T.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lasken</div>
               <div class="given-names">R.S.</div>
            </div>
          </div>
          <div class="article-title">Rapid amplification of plasmid and phage DNA using Phi 29 DNA polymerase and multiply-primed rolling circle amplification</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2001</div>
          <div class="volume">11</div>
          <div class="fpage">1095</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20amplification%20of%20plasmid%20and%20phage%20DNA%20using%20Phi%2029%20DNA%20polymerase%20and%20multiply-primed%20rolling%20circle%20amplification&amp;author=F.B.%20Dean&amp;author=J.R.%20Nelson&amp;author=T.L.%20Giesler&amp;author=R.S.%20Lasken&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1095-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11381035" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Telenius</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Carter</div>
               <div class="given-names">N.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bebb</div>
               <div class="given-names">C.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nordenskjold</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ponder</div>
               <div class="given-names">B.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tunnacliffe</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Degenerate oligonucleotide-primed PCR: general amplification of target DNA by a single degenerate primer</div>
          <div class="source">Genomics</div>Â 
          <div class="year">1992</div>
          <div class="volume">13</div>
          <div class="fpage">718</div>
          <div class="lpage">25</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degenerate%20oligonucleotide-primed%20PCR%3A%20general%20amplification%20of%20target%20DNA%20by%20a%20single%20degenerate%20primer&amp;author=H.%20Telenius&amp;author=N.P.%20Carter&amp;author=C.E.%20Bebb&amp;author=M.%20Nordenskjold&amp;author=B.A.%20Ponder&amp;author=A.%20Tunnacliffe&amp;publication_year=1992&amp;journal=Genomics&amp;volume=13&amp;pages=718-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(92)90147-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(92)90147-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(92)90147-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1639399" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cui</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schmitt</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hubert</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Navidi</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Arnheim</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="article-title">Whole genome amplification from a single cell: implications for genetic analysis</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">1992</div>
          <div class="volume">89</div>
          <div class="fpage">5847</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%20from%20a%20single%20cell%3A%20implications%20for%20genetic%20analysis&amp;author=L.%20Zhang&amp;author=X.%20Cui&amp;author=K.%20Schmitt&amp;author=R.%20Hubert&amp;author=W.%20Navidi&amp;author=N.%20Arnheim&amp;publication_year=1992&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=5847-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.13.5847" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.13.5847" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.13.5847"> </span></div>Â </div></div></div></div></div><div content-id="DSU028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lasken</div>
               <div class="given-names">R.S.</div>
            </div>
          </div>
          <div class="article-title">Genomic DNA amplification by the multiple displacement amplification (MDA) method</div>
          <div class="source">Biochem. Soc. Trans.</div>Â 
          <div class="year">2009</div>
          <div class="volume">37</div>
          <div class="fpage">450</div>
          <div class="lpage">3</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20DNA%20amplification%20by%20the%20multiple%20displacement%20amplification%20(MDA)%20method&amp;author=R.S.%20Lasken&amp;publication_year=2009&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=37&amp;pages=450-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/BST0370450" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2FBST0370450" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2FBST0370450"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19290880" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morin</div>
               <div class="given-names">J.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cao</div>
               <div class="given-names">F.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lazaro</div>
               <div class="given-names">J.M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Active DNA unwinding dynamics during processive DNA replication</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2012</div>
          <div class="volume">109</div>
          <div class="fpage">8115</div>
          <div class="lpage">20</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Active%20DNA%20unwinding%20dynamics%20during%20processive%20DNA%20replication&amp;author=J.A.%20Morin&amp;author=F.J.%20Cao&amp;author=J.M.%20Lazaro&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=8115-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1204759109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1204759109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1204759109"> </span></div>Â </div></div></div></div></div><div content-id="DSU028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">D.Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brandwein</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hsuih</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.B.</div>
            </div>
          </div>
          <div class="article-title">Ramification amplification: a novel isothermal DNA amplification method</div>
          <div class="source">Mol. Diagn.</div>Â 
          <div class="year">2001</div>
          <div class="volume">6</div>
          <div class="fpage">141</div>
          <div class="lpage">50</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ramification%20amplification%3A%20a%20novel%20isothermal%20DNA%20amplification%20method&amp;author=D.Y.%20Zhang&amp;author=M.%20Brandwein&amp;author=T.%20Hsuih&amp;author=H.B.%20Li&amp;publication_year=2001&amp;journal=Mol.%20Diagn.&amp;volume=6&amp;pages=141-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2165/00066982-200106020-00010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2165%2F00066982-200106020-00010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2165%2F00066982-200106020-00010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11468700" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pinard</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">de Winter</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sarkis</div>
               <div class="given-names">G.J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Assessment of whole genome amplification-induced bias through high-throughput, massively parallel whole genome sequencing</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2006</div>
          <div class="volume">7</div>
          <div class="fpage">216</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20whole%20genome%20amplification-induced%20bias%20through%20high-throughput%2C%20massively%20parallel%20whole%20genome%20sequencing&amp;author=R.%20Pinard&amp;author=A.%20de%20Winter&amp;author=G.J.%20Sarkis&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=216" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16928277" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pugh</div>
               <div class="given-names">T.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Delaney</div>
               <div class="given-names">A.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Farnoud</div>
               <div class="given-names">N.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Impact of whole genome amplification on analysis of copy number variants</div>
          <div class="source">Nucleic Acids Res.</div>Â 
          <div class="year">2008</div>
          <div class="volume">36</div>
          <div class="fpage">e80</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Impact%20of%20whole%20genome%20amplification%20on%20analysis%20of%20copy%20number%20variants&amp;author=T.J.%20Pugh&amp;author=A.D.%20Delaney&amp;author=N.%20Farnoud&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=e80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn378" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn378" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18559357" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kozarewa</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ning</div>
               <div class="given-names">Z.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Quail</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sanders</div>
               <div class="given-names">M.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berriman</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
          <div class="article-title">Amplification-free Illumina sequencing-library preparation facilitates improved mapping and assembly of (G+C)-biased genomes</div>
          <div class="source">Nat. Methods</div>Â 
          <div class="year">2009</div>
          <div class="volume">6</div>
          <div class="fpage">291</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amplification-free%20Illumina%20sequencing-library%20preparation%20facilitates%20improved%20mapping%20and%20assembly%20of%20(G%2BC)-biased%20genomes&amp;author=I.%20Kozarewa&amp;author=Z.%20Ning&amp;author=M.A.%20Quail&amp;author=M.J.%20Sanders&amp;author=M.%20Berriman&amp;author=D.J.%20Turner&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=291-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19287394" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Otto</div>
               <div class="given-names">T.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2012</div>
          <div class="volume">13</div>
          <div class="fpage">1</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimizing%20Illumina%20next-generation%20sequencing%20library%20preparation%20for%20extremely%20AT-biased%20genomes&amp;author=S.O.%20Oyola&amp;author=T.D.%20Otto&amp;author=Y.%20Gu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22214261" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hayton</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gaur</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">A.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Erythrocyte binding protein PfRH5 polymorphisms determine species-specific pathways of <em>Plasmodium falciparum</em> invasion</div>
          <div class="source">Cell Host Microbe</div>Â 
          <div class="year">2008</div>
          <div class="volume">4</div>
          <div class="fpage">40</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Erythrocyte%20binding%20protein%20PfRH5%20polymorphisms%20determine%20species-specific%20pathways%20of%20Plasmodium%20falciparum%20invasion&amp;author=K.%20Hayton&amp;author=D.%20Gaur&amp;author=A.%20Liu&amp;publication_year=2008&amp;journal=Cell%20Host%20Microbe&amp;volume=4&amp;pages=40-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chom.2008.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chom.2008.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chom.2008.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18621009" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Han</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chang</div>
               <div class="given-names">C.-W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kwekel</div>
               <div class="given-names">J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Characterization of whole genome amplified (WGA) DNA for use in genotyping assay development</div>
          <div class="source">BMC Genomics</div>Â 
          <div class="year">2012</div>
          <div class="volume">13</div>
          <div class="fpage">217</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20whole%20genome%20amplified%20(WGA)%20DNA%20for%20use%20in%20genotyping%20assay%20development&amp;author=T.%20Han&amp;author=C.-W.%20Chang&amp;author=J.%20Kwekel&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=217" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-217" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-217" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-217"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22655855" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spits</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Le Caignec</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">De Rycke</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Optimization and evaluation of single-cell whole-genome multiple displacement amplification</div>
          <div class="source">Hum. Mutat.</div>Â 
          <div class="year">2006</div>
          <div class="volume">27</div>
          <div class="fpage">496</div>
          <div class="lpage">503</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20and%20evaluation%20of%20single-cell%20whole-genome%20multiple%20displacement%20amplification&amp;author=C.%20Spits&amp;author=C.%20Le%20Caignec&amp;author=M.%20De%20Rycke&amp;publication_year=2006&amp;journal=Hum.%20Mutat.&amp;volume=27&amp;pages=496-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16619243" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Benson</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>
          <div class="source">Nucleic Acids Res.</div>Â 
          <div class="year">1999</div>
          <div class="volume">27</div>
          <div class="fpage">573</div>
          <div class="lpage">80</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lasken</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stockwell</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="article-title">Mechanism of chimera formation during the multiple displacement amplification reaction</div>
          <div class="source">BMC Biotechnol.</div>Â 
          <div class="year">2007</div>
          <div class="volume">7</div>
          <div class="fpage">19</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20chimera%20formation%20during%20the%20multiple%20displacement%20amplification%20reaction&amp;author=R.%20Lasken&amp;author=T.%20Stockwell&amp;publication_year=2007&amp;journal=BMC%20Biotechnol.&amp;volume=7&amp;pages=19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1472-6750-7-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1472-6750-7-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1472-6750-7-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17430586" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McKenna</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hanna</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Banks</div>
               <div class="given-names">E.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2010</div>
          <div class="volume">20</div>
          <div class="fpage">1297</div>
          <div class="lpage">303</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zong</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lu</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chapman</div>
               <div class="given-names">A.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">X.S.</div>
            </div>
          </div>
          <div class="article-title">Genome-wide detection of single-nucleotide and copy-number variations of a single human cell</div>
          <div class="source">Science</div>Â 
          <div class="year">2012</div>
          <div class="volume">338</div>
          <div class="fpage">1622</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20detection%20of%20single-nucleotide%20and%20copy-number%20variations%20of%20a%20single%20human%20cell&amp;author=C.%20Zong&amp;author=S.%20Lu&amp;author=A.R.%20Chapman&amp;author=X.S.%20Xie&amp;publication_year=2012&amp;journal=Science&amp;volume=338&amp;pages=1622-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1229164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1229164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1229164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23258894" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schneider</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Balogh</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Naveran</div>
               <div class="given-names">N.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Whole genome amplificationâ€”the solution for a common problem in forensic casework?</div>
          <div class="source">Int. Congr. Ser.</div>Â 
          <div class="year">2004</div>
          <div class="volume">12,61</div>
          <div class="fpage">24</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20amplification%E2%80%94the%20solution%20for%20a%20common%20problem%20in%20forensic%20casework%3F&amp;author=P.%20Schneider&amp;author=K.%20Balogh&amp;author=N.%20Naveran&amp;publication_year=2004&amp;journal=Int.%20Congr.%20Ser.&amp;volume=12,61&amp;pages=24-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0531-5131(03)01846-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0531-5131(03)01846-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0531-5131(03)01846-6"> </span></div>Â </div></div></div></div></div><div content-id="DSU028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nair</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nkhoma</div>
               <div class="given-names">S.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Serre</div>
               <div class="given-names">D.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Single-cell genomics for dissection of complex malaria infections</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2014</div>
          <div class="volume">24</div>
          <div class="fpage">1028</div>
          <div class="lpage">38</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20genomics%20for%20dissection%20of%20complex%20malaria%20infections&amp;author=S.%20Nair&amp;author=S.C.%20Nkhoma&amp;author=D.%20Serre&amp;publication_year=2014&amp;journal=Genome%20Res.&amp;volume=24&amp;pages=1028-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.168286.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.168286.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.168286.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24812326" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Manske</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Efficient depletion of host DNA contamination in malaria clinical sequencing</div>
          <div class="source">J. Clin. Microbiol.</div>Â 
          <div class="year">2013</div>
          <div class="volume">51</div>
          <div class="fpage">745</div>
          <div class="lpage">51</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20depletion%20of%20host%20DNA%20contamination%20in%20malaria%20clinical%20sequencing&amp;author=S.O.%20Oyola&amp;author=Y.%20Gu&amp;author=M.%20Manske&amp;publication_year=2013&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=51&amp;pages=745-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.02507-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.02507-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.02507-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23224084" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Feehery</div>
               <div class="given-names">G.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yigit</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Oyola</div>
               <div class="given-names">S.O.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">A method for selectively enriching microbial DNA from contaminating vertebrate host DNA</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2013</div>
          <div class="volume">8</div>
          <div class="fpage">e76096</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20method%20for%20selectively%20enriching%20microbial%20DNA%20from%20contaminating%20vertebrate%20host%20DNA&amp;author=G.R.%20Feehery&amp;author=E.%20Yigit&amp;author=S.O.%20Oyola&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e76096" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0076096" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0076096" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0076096"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24204593" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU028C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Venkatesan</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Amaratunga</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Campino</div>
               <div class="given-names">S.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Using CF11 cellulose columns to inexpensively and effectively remove human DNA from <em>Plasmodium falciparum</em>-infected whole blood samples</div>
          <div class="source">Malar. J.</div>Â 
          <div class="year">2012</div>
          <div class="volume">11</div>
          <div class="fpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20CF11%20cellulose%20columns%20to%20inexpensively%20and%20effectively%20remove%20human%20DNA%20from%20Plasmodium%20falciparum-infected%20whole%20blood%20samples&amp;author=M.%20Venkatesan&amp;author=C.%20Amaratunga&amp;author=S.%20Campino&amp;publication_year=2012&amp;journal=Malar.%20J.&amp;volume=11&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1475-2875-11-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1475-2875-11-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1475-2875-11-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22321373" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754550&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu028/2/dsu028_Supplementary_Data.zip?Expires=2147483647&amp;Signature=QBvtfTRjnIf0pNO2fb8EY7vViOMezCuAGAMS2KgOkEcR3w-wTF~OwbfnTshFuKlqg31Vf1bM92O4jad-ZoZka8Bf054Zbvc5Sx9J40lU3NijiWuSENgds6SFpFI8S-FHlWT6jH4IaxwYMOWL68zTGRAaKcglMUEQpQtt1c5VqnXCrELsoxRZzDf56Tcwb6m7qPm7a0RcBatrMfAXjAxDKvllMt1QPYGAa49Zj04Qhr~CbkJbFjTWbMN~iZzpCn5NE5FZN07zMppTmZ2n~IC-4rQy0RZ3Vxg7p5AqLpW87TYNEJCLaP~fc3EAzurjNs8hhStlfEu6HgMVpYKUd6hiJw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">568</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">12</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754550"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f6%2f661%2f2754550">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000347101100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (12)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F6%2F661%2F2754550" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754550'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Optimized Whole-Genome Amplification Strategy for Extremely AT-Biased Template" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f661%2f2754550">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

