

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/19/439863","@type":"ScholarlyArticle","name":"Integrative structural annotation of  de novo  RNA-Seq provides an accurate reference gene set of the enormous genome of the onion ( Allium cepa  L.)","datePublished":"2014-10-31","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu035","keywords":["<italic>de novo</italic> transcriptome","gene prediction","reference gene set","onion","non-coding sequence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/19/439863","author":[{"name":"Kim, Seungill","affiliation":"1  Department of Plant Science ,  Seoul National University ,  Seoul ,  Korea    2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Kim, Myung-Shin","affiliation":"1  Department of Plant Science ,  Seoul National University ,  Seoul ,  Korea    2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Kim, Yong-Min","affiliation":"3  Korea Bioinformation Center ,  Korea Research Institute of Bioscience and Biotechnology ,  Daejon ,  Korea","@type":"Person"},{"name":"Yeom, Seon-In","affiliation":"4  Plant Genomics and Breeding Institute ,  Seoul National University ,  Seoul ,  Korea    5  Department of Horticulture ,  Institute of Agriculture and Life Science, Gyeongsang National University ,  Jinju ,  Korea","@type":"Person"},{"name":"Cheong, Kyeongchae","affiliation":"2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Kim, Ki-Tae","affiliation":"6  Department of Agricultural Biotechnology ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Jeon, Jongbum","affiliation":"2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Kim, Sunggil","affiliation":"7  Department of Plant Biotechnology ,  Biotechnology Research Institute, Chonnam National University ,  Gwangju ,  Korea","@type":"Person"},{"name":"Kim, Do-Sun","affiliation":"8  National Institute of Horticultural and Herbal Science, RDA ,  Suwon ,  Korea","@type":"Person"},{"name":"Sohn, Seong-Han","affiliation":"9  National Institute of Agricultural Science, RDA ,  Suwon ,  Korea","@type":"Person"},{"name":"Lee, Yong-Hwan","affiliation":"2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea    4  Plant Genomics and Breeding Institute ,  Seoul National University ,  Seoul ,  Korea    6  Department of Agricultural Biotechnology ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"},{"name":"Choi, Doil","affiliation":"1  Department of Plant Science ,  Seoul National University ,  Seoul ,  Korea    2  Interdisciplinary Program in Agricultural Genomics ,  Seoul National University ,  Seoul ,  Korea    4  Plant Genomics and Breeding Institute ,  Seoul National University ,  Seoul ,  Korea","@type":"Person"}],"description":"Abstract.  The onion (Allium cepa L.) is one of the most widely cultivated and consumed vegetable crops in the world. Although a considerable amount of onion tr","pageStart":"19","pageEnd":"27","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Integrative structural annotation of  de novo  RNA-Seq provides an accurate reference gene set of the enormous genome of the onion ( Allium cepa  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrative structural annotation of  de novo  RNA-Seq provides an accurate reference gene set of the enormous genome of the onion ( Allium cepa  L.)" />
    <meta property="og:description" content="Abstract.  The onion (Allium cepa L.) is one of the most widely cultivated and consumed vegetable crops in the world. Although a considerable amount of onion tr" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/19/439863" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-10-31" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kim, Seungill" /><meta name="citation_author_institution" content="Department of Plant Science, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Kim, Myung-Shin" /><meta name="citation_author_institution" content="Department of Plant Science, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Kim, Yong-Min" /><meta name="citation_author_institution" content="Korea Bioinformation Center, Korea Research Institute of Bioscience and Biotechnology, Daejon, Korea" /><meta name="citation_author" content="Yeom, Seon-In" /><meta name="citation_author_institution" content="Plant Genomics and Breeding Institute, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Department of Horticulture, Institute of Agriculture and Life Science, Gyeongsang National University, Jinju, Korea" /><meta name="citation_author" content="Cheong, Kyeongchae" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Kim, Ki-Tae" /><meta name="citation_author_institution" content="Department of Agricultural Biotechnology, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Jeon, Jongbum" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Kim, Sunggil" /><meta name="citation_author_institution" content="Department of Plant Biotechnology, Biotechnology Research Institute, Chonnam National University, Gwangju, Korea" /><meta name="citation_author" content="Kim, Do-Sun" /><meta name="citation_author_institution" content="National Institute of Horticultural and Herbal Science, RDA, Suwon, Korea" /><meta name="citation_author" content="Sohn, Seong-Han" /><meta name="citation_author_institution" content="National Institute of Agricultural Science, RDA, Suwon, Korea" /><meta name="citation_author" content="Lee, Yong-Hwan" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Plant Genomics and Breeding Institute, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Department of Agricultural Biotechnology, Seoul National University, Seoul, Korea" /><meta name="citation_author" content="Choi, Doil" /><meta name="citation_author_institution" content="Department of Plant Science, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Interdisciplinary Program in Agricultural Genomics, Seoul National University, Seoul, Korea" /><meta name="citation_author_institution" content="Plant Genomics and Breeding Institute, Seoul National University, Seoul, Korea" /><meta name="citation_title" content="Integrative structural annotation of de novo RNA-Seq provides an accurate reference gene set of the enormous genome of the onion (Allium cepa L.)" /><meta name="citation_firstpage" content="19" /><meta name="citation_lastpage" content="27" /><meta name="citation_doi" content="10.1093/dnares/dsu035" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=CABI, Wallingford, UK; Cambridge, MA; Onions and other vegetable alliums;  citation_year=2008; " /><meta name="citation_reference" content="citation_title=Onions: a source of unique dietary flavonoids; J. Agric. Food Chem.;  citation_year=2007;  citation_volume=55;  citation_pages=10067-80; " /><meta name="citation_reference" content="citation_title=Varietal differences in phenolic content and antioxidant and antiproliferative activities of onions; J. Agric. Food Chem.;  citation_year=2004;  citation_volume=52;  citation_pages=6787-93; " /><meta name="citation_reference" content="citation_title=Total phenol, flavonoid, proanthocyanidin and vitamin C levels and antioxidant activities of Mauritian vegetables; J. Sci. Food AGR;  citation_year=2004;  citation_volume=84;  citation_pages=1553-61; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing to produce SNP-based genetic maps of onion; Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=2093-101; " /><meta name="citation_reference" content="citation_title=Sequence assembly demystified; Nat. Rev. Genet.;  citation_year=2013;  citation_volume=14;  citation_pages=157-67; " /><meta name="citation_reference" content="citation_title=Current challenges in de novo plant genome sequencing and assembly; Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=243" /><meta name="citation_reference" content="citation_title=A beginner's guide to eukaryotic genome annotation; Nat. Rev. Genet.;  citation_year=2012;  citation_volume=13;  citation_pages=329-42; " /><meta name="citation_reference" content="citation_title=Analyses of pig genomes provide insight into porcine demography and evolution; Nature;  citation_year=2012;  citation_volume=491;  citation_pages=393-8; " /><meta name="citation_reference" content="citation_title=RNA-Seq improves annotation of protein-coding genes in the cucumber genome; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=540" /><meta name="citation_reference" content="citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq; Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=469-77; " /><meta name="citation_reference" content="citation_title=The genome of the cucumber, Cucumis sativus L.; Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1275-81; " /><meta name="citation_reference" content="citation_title=Annotating genomes with massive-scale RNA sequencing; Genome Biol.;  citation_year=2008;  citation_volume=9;  citation_pages=R175" /><meta name="citation_reference" content="citation_title=De novo transcriptome assembly in chili pepper (Capsicum frutescens) to identify genes involved in the biosynthesis of capsaicinoids; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e48156" /><meta name="citation_reference" content="citation_title=RNA sequencing: advances, challenges and opportunities; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=87-98; " /><meta name="citation_reference" content="citation_title=Next-generation transcriptome assembly; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=671-82; " /><meta name="citation_reference" content="citation_title=Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (Cajanus cajan L.); DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=153-64; " /><meta name="citation_reference" content="citation_title=Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq; Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1238-49; " /><meta name="citation_reference" content="citation_title=A de novo assembly of the newt transcriptome combined with proteomic validation identifies new protein families expressed during tissue regeneration; Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R16" /><meta name="citation_reference" content="citation_title=De novo assembly and characterization of the garlic (Allium sativum) bud transcriptome by Illumina sequencing; Plant Cell Rep.;  citation_year=2012;  citation_volume=31;  citation_pages=1823-8; " /><meta name="citation_reference" content="citation_title=De novo assembly of the pepper transcriptome (Capsicum annuum): a benchmark for in silico discovery of SNPs, SSRs and candidate genes; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=571" /><meta name="citation_reference" content="citation_title=Next generation sequencing and de novo transcriptome analysis of Costus pictus D. Don, a non-model plant with potent anti-diabetic properties; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=663" /><meta name="citation_reference" content="citation_title=Comprehensive transcriptome analysis of the highly complex Pisum sativum genome using next generation sequencing; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=227" /><meta name="citation_reference" content="citation_title=Long noncoding RNAs regulate adipogenesis; Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=3387-92; " /><meta name="citation_reference" content="citation_title=Assessment of transcript reconstruction methods for RNA-seq; Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1177-84; " /><meta name="citation_reference" content="citation_title=Control of somatic tissue differentiation by the long non-coding RNA TINCR; Nature;  citation_year=2013;  citation_volume=493;  citation_pages=231-5; " /><meta name="citation_reference" content="citation_title=Transcriptome analyses and differential gene expression in a non-model fish species with alternative mating tactics; BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=167" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing and expression analysis of terpenoid biosynthesis genes in Litsea cubeba; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e76890" /><meta name="citation_reference" content="citation_title=De novo transcriptome assembly of drought tolerant CAM plants, Agave deserti and Agave tequilana; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=563" /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals differentially expressed genes between soft-endocarp and hard-endocarp hawthorns; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e72910" /><meta name="citation_reference" content="citation_title=Transcriptome analysis of Litopenaeus vannamei in response to white spot syndrome virus infection; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e73218" /><meta name="citation_reference" content="citation_title=Sequencing and de novo assembly of the Asian clam (Corbicula fluminea) transcriptome using the Illumina GAIIx method; PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e79516" /><meta name="citation_reference" content="citation_title=De novo transcriptome sequencing in Anopheles funestus using Illumina RNA-seq technology; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e14202" /><meta name="citation_reference" content="citation_title=High-throughput Illumina strand-specific RNA sequencing library preparation; Cold Spring Harb. Prot.;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species; Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=270" /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3211-7; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1086-92; " /><meta name="citation_reference" content="citation_title=The banana (Musa acuminata) genome and the evolution of monocotyledonous plants; Nature;  citation_year=2012;  citation_volume=488;  citation_pages=213-7; " /><meta name="citation_reference" content="citation_title=The Rice Annotation Project Database (RAP-DB): hub for Oryza sativa ssp. japonica genome information; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D741-4; " /><meta name="citation_reference" content="citation_title=The draft genome of the fast-growing non-timber forest species moso bamboo (Phyllostachys heterocycla); citation_text=461e451–2; Nat. Genet.;  citation_year=2013;  citation_volume=45;  citation_pages=456-61; " /><meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses; Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of the model grass Brachypodium distachyon; citation_author=International Brachypodium, I.; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=763-8; " /><meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=31" /><meta name="citation_reference" content="citation_title=AUGUSTUS: ab initio prediction of alternative transcripts; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W435-9; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification; Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D130-5; " /><meta name="citation_reference" content="citation_title=Update on activities at the Universal Protein Resource (UniProt) in 2013; citation_author=UniProt, C; Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D43-7; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3; Fly;  citation_year=2012;  citation_volume=6;  citation_pages=80-92; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/19/1206719/dsu035.pdf" /><meta name="description" content="Abstract.  The onion (Allium cepa L.) is one of the most widely cultivated and consumed vegetable crops in the world. Although a considerable amount of onion tr" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/19/439863" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/19/439863" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/19/439863");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu035';
    </script>
<title>
	Integrative structural annotation of de novo RNA-Seq provides an accurate reference gene set of the enormous genome of the onion (Allium cepa L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./439863" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbV8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIFNldW5naWxsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcmRpc2NpcGxpbmFyeSBQcm9ncmFtIGluIEFncmljdWx0dXJhbCBHZW5vbWljcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBNeXVuZy1TaGluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcmRpc2NpcGxpbmFyeSBQcm9ncmFtIGluIEFncmljdWx0dXJhbCBHZW5vbWljcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBZb25nLU1pbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktvcmVhIEJpb2luZm9ybWF0aW9uIENlbnRlciwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIERhZWpvbiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZZW9tLCBTZW9uLUluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgR2Vub21pY3MgYW5kIEJyZWVkaW5nIEluc3RpdHV0ZSwgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlLCBHeWVvbmdzYW5nIE5hdGlvbmFsIFVuaXZlcnNpdHksIEppbmp1LCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW9uZywgS3llb25nY2hhZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVyZGlzY2lwbGluYXJ5IFByb2dyYW0gaW4gQWdyaWN1bHR1cmFsIEdlbm9taWNzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW0sIEtpLVRhZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQWdyaWN1bHR1cmFsIEJpb3RlY2hub2xvZ3ksIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFNlb3VsLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikplb24sIEpvbmdidW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcmRpc2NpcGxpbmFyeSBQcm9ncmFtIGluIEFncmljdWx0dXJhbCBHZW5vbWljcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2ltLCBTdW5nZ2lsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBCaW90ZWNobm9sb2d5LCBCaW90ZWNobm9sb2d5IFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hvbm5hbSBOYXRpb25hbCBVbml2ZXJzaXR5LCBHd2FuZ2p1LCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktpbSwgRG8tU3VuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEhvcnRpY3VsdHVyYWwgYW5kIEhlcmJhbCBTY2llbmNlLCBSREEsIFN1d29uLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNvaG4sIFNlb25nLUhhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgUkRBLCBTdXdvbiwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIFlvbmctSHdhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkludGVyZGlzY2lwbGluYXJ5IFByb2dyYW0gaW4gQWdyaWN1bHR1cmFsIEdlbm9taWNzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQbGFudCBHZW5vbWljcyBhbmQgQnJlZWRpbmcgSW5zdGl0dXRlLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFncmljdWx0dXJhbCBCaW90ZWNobm9sb2d5LCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG9pLCBEb2lsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCwgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcmRpc2NpcGxpbmFyeSBQcm9ncmFtIGluIEFncmljdWx0dXJhbCBHZW5vbWljcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgR2Vub21pY3MgYW5kIEJyZWVkaW5nIEluc3RpdHV0ZSwgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJJbnRlZ3JhdGl2ZSBzdHJ1Y3R1cmFsIGFubm90YXRpb24gb2YgZGUgbm92byBSTkEtU2VxIHByb3ZpZGVzIGFuIGFjY3VyYXRlIHJlZmVyZW5jZSBnZW5lIHNldCBvZiB0aGUgZW5vcm1vdXMgZ2Vub21lIG9mIHRoZSBvbmlvbiAoQWxsaXVtIGNlcGEgTC4pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q0FCSSwgV2FsbGluZ2ZvcmQsIFVLOyBDYW1icmlkZ2UsIE1BOyBPbmlvbnMgYW5kIG90aGVyIHZlZ2V0YWJsZSBhbGxpdW1zOyAgY2l0YXRpb25feWVhcj0yMDA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmlvbnM6IGEgc291cmNlIG9mIHVuaXF1ZSBkaWV0YXJ5IGZsYXZvbm9pZHM7IEouIEFncmljLiBGb29kIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTAwNjctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhcmlldGFsIGRpZmZlcmVuY2VzIGluIHBoZW5vbGljIGNvbnRlbnQgYW5kIGFudGlveGlkYW50IGFuZCBhbnRpcHJvbGlmZXJhdGl2ZSBhY3Rpdml0aWVzIG9mIG9uaW9uczsgSi4gQWdyaWMuIEZvb2QgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz02Nzg3LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3RhbCBwaGVub2wsIGZsYXZvbm9pZCwgcHJvYW50aG9jeWFuaWRpbiBhbmQgdml0YW1pbiBDIGxldmVscyBhbmQgYW50aW94aWRhbnQgYWN0aXZpdGllcyBvZiBNYXVyaXRpYW4gdmVnZXRhYmxlczsgSi4gU2NpLiBGb29kIEFHUjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTE1NTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyB0byBwcm9kdWNlIFNOUC1iYXNlZCBnZW5ldGljIG1hcHMgb2Ygb25pb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MjA5My0xMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGFzc2VtYmx5IGRlbXlzdGlmaWVkOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTU3LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXJyZW50IGNoYWxsZW5nZXMgaW4gZGUgbm92byBwbGFudCBnZW5vbWUgc2VxdWVuY2luZyBhbmQgYXNzZW1ibHk7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgYmVnaW5uZXIncyBndWlkZSB0byBldWthcnlvdGljIGdlbm9tZSBhbm5vdGF0aW9uOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MzI5LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNlcyBvZiBwaWcgZ2Vub21lcyBwcm92aWRlIGluc2lnaHQgaW50byBwb3JjaW5lIGRlbW9ncmFwaHkgYW5kIGV2b2x1dGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ5MTsgIGNpdGF0aW9uX3BhZ2VzPTM5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtU2VxIGltcHJvdmVzIGFubm90YXRpb24gb2YgcHJvdGVpbi1jb2RpbmcgZ2VuZXMgaW4gdGhlIGN1Y3VtYmVyIGdlbm9tZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBtZXRob2RzIGZvciB0cmFuc2NyaXB0b21lIGFubm90YXRpb24gYW5kIHF1YW50aWZpY2F0aW9uIHVzaW5nIFJOQS1zZXE7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDY5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBjdWN1bWJlciwgQ3VjdW1pcyBzYXRpdnVzIEwuOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFubm90YXRpbmcgZ2Vub21lcyB3aXRoIG1hc3NpdmUtc2NhbGUgUk5BIHNlcXVlbmNpbmc7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9UjE3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBpbiBjaGlsaSBwZXBwZXIgKENhcHNpY3VtIGZydXRlc2NlbnMpIHRvIGlkZW50aWZ5IGdlbmVzIGludm9sdmVkIGluIHRoZSBiaW9zeW50aGVzaXMgb2YgY2Fwc2FpY2lub2lkczsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU0ODE1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBzZXF1ZW5jaW5nOiBhZHZhbmNlcywgY2hhbGxlbmdlcyBhbmQgb3Bwb3J0dW5pdGllczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gdHJhbnNjcmlwdG9tZSBhc3NlbWJseTsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVmaW5pbmcgdGhlIHRyYW5zY3JpcHRvbWUgYXNzZW1ibHkgYW5kIGl0cyB1c2UgZm9yIGdlbm9tZSBkeW5hbWljcyBhbmQgdHJhbnNjcmlwdG9tZSBwcm9maWxpbmcgc3R1ZGllcyBpbiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4gTC4pOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb24gYW5ub3RhdGlvbiBvZiB0aGUgcmljZSB0cmFuc2NyaXB0b21lIGF0IHNpbmdsZS1udWNsZW90aWRlIHJlc29sdXRpb24gYnkgUk5BLXNlcTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjM4LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIG5ld3QgdHJhbnNjcmlwdG9tZSBjb21iaW5lZCB3aXRoIHByb3Rlb21pYyB2YWxpZGF0aW9uIGlkZW50aWZpZXMgbmV3IHByb3RlaW4gZmFtaWxpZXMgZXhwcmVzc2VkIGR1cmluZyB0aXNzdWUgcmVnZW5lcmF0aW9uOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBnYXJsaWMgKEFsbGl1bSBzYXRpdnVtKSBidWQgdHJhbnNjcmlwdG9tZSBieSBJbGx1bWluYSBzZXF1ZW5jaW5nOyBQbGFudCBDZWxsIFJlcC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xODIzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgdGhlIHBlcHBlciB0cmFuc2NyaXB0b21lIChDYXBzaWN1bSBhbm51dW0pOiBhIGJlbmNobWFyayBmb3IgaW4gc2lsaWNvIGRpc2NvdmVyeSBvZiBTTlBzLCBTU1JzIGFuZCBjYW5kaWRhdGUgZ2VuZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTU3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBkZSBub3ZvIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgQ29zdHVzIHBpY3R1cyBELiBEb24sIGEgbm9uLW1vZGVsIHBsYW50IHdpdGggcG90ZW50IGFudGktZGlhYmV0aWMgcHJvcGVydGllczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHJlaGVuc2l2ZSB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIHRoZSBoaWdobHkgY29tcGxleCBQaXN1bSBzYXRpdnVtIGdlbm9tZSB1c2luZyBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZyBub25jb2RpbmcgUk5BcyByZWd1bGF0ZSBhZGlwb2dlbmVzaXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTMzODctOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc21lbnQgb2YgdHJhbnNjcmlwdCByZWNvbnN0cnVjdGlvbiBtZXRob2RzIGZvciBSTkEtc2VxOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMTc3LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIHNvbWF0aWMgdGlzc3VlIGRpZmZlcmVudGlhdGlvbiBieSB0aGUgbG9uZyBub24tY29kaW5nIFJOQSBUSU5DUjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5MzsgIGNpdGF0aW9uX3BhZ2VzPTIzMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2VzIGFuZCBkaWZmZXJlbnRpYWwgZ2VuZSBleHByZXNzaW9uIGluIGEgbm9uLW1vZGVsIGZpc2ggc3BlY2llcyB3aXRoIGFsdGVybmF0aXZlIG1hdGluZyB0YWN0aWNzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgYW5kIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGVycGVub2lkIGJpb3N5bnRoZXNpcyBnZW5lcyBpbiBMaXRzZWEgY3ViZWJhOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTc2ODkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IG9mIGRyb3VnaHQgdG9sZXJhbnQgQ0FNIHBsYW50cywgQWdhdmUgZGVzZXJ0aSBhbmQgQWdhdmUgdGVxdWlsYW5hOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01NjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGFzc2VtYmx5IGFuZCBxdWFudGlmaWNhdGlvbiBieSBSTkEtU2VxIHJldmVhbHMgZGlmZmVyZW50aWFsbHkgZXhwcmVzc2VkIGdlbmVzIGJldHdlZW4gc29mdC1lbmRvY2FycCBhbmQgaGFyZC1lbmRvY2FycCBoYXd0aG9ybnM7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzI5MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIExpdG9wZW5hZXVzIHZhbm5hbWVpIGluIHJlc3BvbnNlIHRvIHdoaXRlIHNwb3Qgc3luZHJvbWUgdmlydXMgaW5mZWN0aW9uOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTczMjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgZGUgbm92byBhc3NlbWJseSBvZiB0aGUgQXNpYW4gY2xhbSAoQ29yYmljdWxhIGZsdW1pbmVhKSB0cmFuc2NyaXB0b21lIHVzaW5nIHRoZSBJbGx1bWluYSBHQUlJeCBtZXRob2Q7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzk1MTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBpbiBBbm9waGVsZXMgZnVuZXN0dXMgdXNpbmcgSWxsdW1pbmEgUk5BLXNlcSB0ZWNobm9sb2d5OyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTE0MjAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IElsbHVtaW5hIHN0cmFuZC1zcGVjaWZpYyBSTkEgc2VxdWVuY2luZyBsaWJyYXJ5IHByZXBhcmF0aW9uOyBDb2xkIFNwcmluZyBIYXJiLiBQcm90LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBob3QgcGVwcGVyIHByb3ZpZGVzIGluc2lnaHRzIGludG8gdGhlIGV2b2x1dGlvbiBvZiBwdW5nZW5jeSBpbiBDYXBzaWN1bSBzcGVjaWVzOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTI3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDI7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvcnRNZVJOQTogZmFzdCBhbmQgYWNjdXJhdGUgZmlsdGVyaW5nIG9mIHJpYm9zb21hbCBSTkFzIGluIG1ldGF0cmFuc2NyaXB0b21pYyBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTMyMTEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVsdmV0OiBhbGdvcml0aG1zIGZvciBkZSBub3ZvIHNob3J0IHJlYWQgYXNzZW1ibHkgdXNpbmcgZGUgQnJ1aWpuIGdyYXBoczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T2FzZXM6IHJvYnVzdCBkZSBub3ZvIFJOQS1zZXEgYXNzZW1ibHkgYWNyb3NzIHRoZSBkeW5hbWljIHJhbmdlIG9mIGV4cHJlc3Npb24gbGV2ZWxzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTEwODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBiYW5hbmEgKE11c2EgYWN1bWluYXRhKSBnZW5vbWUgYW5kIHRoZSBldm9sdXRpb24gb2YgbW9ub2NvdHlsZWRvbm91cyBwbGFudHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODg7ICBjaXRhdGlvbl9wYWdlcz0yMTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFJpY2UgQW5ub3RhdGlvbiBQcm9qZWN0IERhdGFiYXNlIChSQVAtREIpOiBodWIgZm9yIE9yeXphIHNhdGl2YSBzc3AuIGphcG9uaWNhIGdlbm9tZSBpbmZvcm1hdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9RDc0MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZHJhZnQgZ2Vub21lIG9mIHRoZSBmYXN0LWdyb3dpbmcgbm9uLXRpbWJlciBmb3Jlc3Qgc3BlY2llcyBtb3NvIGJhbWJvbyAoUGh5bGxvc3RhY2h5cyBoZXRlcm9jeWNsYSk7IGNpdGF0aW9uX3RleHQ9NDYxZTQ1MeKAkzI7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9NDU2LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU29yZ2h1bSBiaWNvbG9yIGdlbm9tZSBhbmQgdGhlIGRpdmVyc2lmaWNhdGlvbiBvZiBncmFzc2VzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDU3OyAgY2l0YXRpb25fcGFnZXM9NTUxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiB0aGUgbW9kZWwgZ3Jhc3MgQnJhY2h5cG9kaXVtIGRpc3RhY2h5b247IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIEJyYWNoeXBvZGl1bSwgSS47IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjM7ICBjaXRhdGlvbl9wYWdlcz03NjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGdlbmVyYXRpb24gb2YgaGV1cmlzdGljcyBmb3IgYmlvbG9naWNhbCBzZXF1ZW5jZSBjb21wYXJpc29uOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QVVHVVNUVVM6IGFiIGluaXRpbyBwcmVkaWN0aW9uIG9mIGFsdGVybmF0aXZlIHRyYW5zY3JpcHRzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XNDM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbiA1OiBnZW5vbWUtc2NhbGUgcHJvdGVpbiBmdW5jdGlvbiBjbGFzc2lmaWNhdGlvbjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIFJlZmVyZW5jZSBTZXF1ZW5jZXMgKFJlZlNlcSk6IGN1cnJlbnQgc3RhdHVzLCBuZXcgZmVhdHVyZXMgYW5kIGdlbm9tZSBhbm5vdGF0aW9uIHBvbGljeTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9RDEzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VcGRhdGUgb24gYWN0aXZpdGllcyBhdCB0aGUgVW5pdmVyc2FsIFByb3RlaW4gUmVzb3VyY2UgKFVuaVByb3QpIGluIDIwMTM7IGNpdGF0aW9uX2F1dGhvcj1VbmlQcm90LCBDOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENDMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHJvZ3JhbSBmb3IgYW5ub3RhdGluZyBhbmQgcHJlZGljdGluZyB0aGUgZWZmZWN0cyBvZiBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zLCBTbnBFZmY6IFNOUHMgaW4gdGhlIGdlbm9tZSBvZiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBzdHJhaW4gdzExMTg7IGlzby0yOyBpc28tMzsgRmx5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIyLzEvMTkvMTIwNjcxOS9kc3UwMzUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBvbmlvbiAoQWxsaXVtIGNlcGEgTC4pIGlzIG9uZSBvZiB0aGUgbW9zdCB3aWRlbHkgY3VsdGl2YXRlZCBhbmQgY29uc3VtZWQgdmVnZXRhYmxlIGNyb3BzIGluIHRoZSB3b3JsZC4gQWx0aG91Z2ggYSBjb25zaWRlcmFibGUgYW1vdW50IG9mIG9uaW9uIHRyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00Mzk4NjMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkfLoqy67FKLK5oFDLVU9lclwFpMw=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f19%2f439863" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f19%2f439863" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6568835">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568835">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568837">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568837">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568842">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568842">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568855">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568855">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568881">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568881">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568886">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568886">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568888">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568890">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568890">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6568892">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6568892">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/13/2754540">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/29/440058">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Integrative structural annotation of <em>de novo</em> RNA-Seq provides an accurate reference gene set of the enormous genome of the onion (<em>Allium cepa</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seungill Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seungill+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Seungill Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Myung-Shin Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Myung-Shin+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Myung-Shin Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong-Min Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Korea Bioinformation Center</div>, <div class="insititution">Korea Research Institute of Bioscience and Biotechnology</div>, <div class="addr-line">Daejon</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong-Min+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yong-Min Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seon-In Yeom</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Plant Genomics and Breeding Institute</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Horticulture</div>, <div class="insititution">Institute of Agriculture and Life Science, Gyeongsang National University</div>, <div class="addr-line">Jinju</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seon-In+Yeom">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yeom S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yeom S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Seon-In Yeom</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyeongchae Cheong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyeongchae+Cheong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cheong K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cheong K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyeongchae Cheong</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ki-Tae Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Agricultural Biotechnology</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ki-Tae+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ki-Tae Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jongbum Jeon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jongbum+Jeon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeon J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeon J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jongbum Jeon</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sunggil Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Plant Biotechnology</div>, <div class="insititution">Biotechnology Research Institute, Chonnam National University</div>, <div class="addr-line">Gwangju</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sunggil+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sunggil Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Do-Sun Kim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="insititution">National Institute of Horticultural and Herbal Science, RDA</div>, <div class="addr-line">Suwon</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Do-Sun+Kim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kim D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kim D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Do-Sun Kim</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Seong-Han Sohn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="insititution">National Institute of Agricultural Science, RDA</div>, <div class="addr-line">Suwon</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Seong-Han+Sohn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sohn S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sohn S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Seong-Han Sohn</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong-Hwan Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Plant Genomics and Breeding Institute</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Agricultural Biotechnology</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong-Hwan+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yong-Hwan Lee</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Doil Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Interdisciplinary Program in Agricultural Genomics</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Plant Genomics and Breeding Institute</div>, <div class="insititution">Seoul National University</div>, <div class="addr-line">Seoul</div>, <div class="country">Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-880-4568. Fax. +82 2-873-2056. E-mail: <a href="mailto:doil@snu.ac.kr" target="_blank">doil@snu.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Doil+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Doil Choi</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 19–27, <a href='https://doi.org/10.1093/dnares/dsu035'>https://doi.org/10.1093/dnares/dsu035</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">31 October 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 August 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 October 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="439863" href="/dnaresearch/article-pdf/22/1/19/1206719/dsu035.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Seungill Kim, Myung-Shin Kim, Yong-Min Kim, Seon-In Yeom, Kyeongchae Cheong, Ki-Tae Kim, Jongbum Jeon, Sunggil Kim, Do-Sun Kim, Seong-Han Sohn, Yong-Hwan Lee, Doil Choi;  Integrative structural annotation of <em>de novo</em> RNA-Seq provides an accurate reference gene set of the enormous genome of the onion (<em>Allium cepa</em> L.), <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 19–27, <a href='https://doi.org/10.1093/dnares/dsu035'>https://doi.org/10.1093/dnares/dsu035</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/439863?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/439863?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Integrative%20structural%20annotation%20of%20%26lt%3Bem%26gt%3Bde%20novo%26lt%3B%2Fem%26gt%3B%20RNA-Seq%20provides%20an%20accurate%20reference%20gene%20set%20of%20the%20enormous%20genome%20of%20the%20onion%20(%26lt%3Bem%26gt%3BAllium%20cepa%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2014-10-31&amp;volumeNum=22&amp;issueNum=1&amp;author=Kim%2C%20Seungill%3B%20Kim%2C%20Myung-Shin&amp;startPage=19&amp;endPage=27&amp;contentId=10.1093%2Fdnares%2Fdsu035&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f19%2f439863" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6568835 id="6568835" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The onion (<em>Allium cepa</em> L.) is one of the most widely cultivated and consumed vegetable crops in the world. Although a considerable amount of onion transcriptome data has been deposited into public databases, the sequences of the protein-coding genes are not accurate enough to be used, owing to non-coding sequences intermixed with the coding sequences. We generated a high-quality, annotated onion transcriptome from <em>de novo</em> sequence assembly and intensive structural annotation using the integrated structural gene annotation pipeline (ISGAP), which identified 54,165 protein-coding genes among 165,179 assembled transcripts totalling 203.0 Mb by eliminating the intron sequences. ISGAP performed reliable annotation, recognizing accurate gene structures based on reference proteins, and <em>ab initio</em> gene models of the assembled transcripts. Integrative functional annotation and gene-based SNP analysis revealed a whole biological repertoire of genes and transcriptomic variation in the onion. The method developed in this study provides a powerful tool for the construction of reference gene sets for organisms based solely on <em>de novo</em> transcriptome data. Furthermore, the reference genes and their variation described here for the onion represent essential tools for molecular breeding and gene cloning in <em>Allium</em> spp.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo transcriptome&quot;"><em>de novo</em> transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene prediction&quot;">gene prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reference gene set&quot;">reference gene set</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="onion">onion</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-coding sequence&quot;">non-coding sequence</a></div>                            <h2 scrollto-destination=6568837 id="6568837" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The onion (<em>Allium cepa</em>) belongs to the family Amaryllidaceae, containing over 300 species; of which, 70 have been cultivated for 4,700 years or more.<sup><a reveal-id="DSU035C1" data-open="DSU035C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The onion is one of the major vegetable crops in the world. In 2012, onions were grown in 170 countries with global production of 87 million tons (<a class="link link-uri" href="http://faostat.fao.org/">http://faostat.fao.org/</a>). Onions contain outstanding levels of polyphenols, vitamins, and sulphur-containing compounds, which are responsible for their pungency.<sup><a reveal-id="DSU035C2  DSU035C4" data-open="DSU035C2  DSU035C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Those compounds also affect various aspects of human health, including support for bone and connective tissues, anti-inflammatory effects, diabetes prevention, digestive tract health, and cancer protection.<sup><a reveal-id="DSU035C2" data-open="DSU035C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Hence, many traits were developed or are under development for onion breeding, including bulb shape, bulb colour, bulb size, flowering time, pungency, nutritional value, and disease resistance.</p><p>Despite the importance and significance of <em>Allium</em> species as major vegetable crops with nutritional and medicinal values, poor genomic information is available because of the enormous size of the genome (16.3 Gb).<sup><a reveal-id="DSU035C5" data-open="DSU035C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Although sequencing technologies have advanced rapidly in terms of higher throughput and longer read lengths, analysis of the complex and huge genome of the onion, a non-model plant, has remained a Herculean task.<sup><a reveal-id="DSU035C6" data-open="DSU035C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU035C7" data-open="DSU035C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Currently, only a few information resources on the onion genome are available, including 511 proteins and 20,159 expressed sequence tags (ESTs) in GenBank, and those are insufficient for use in molecular breeding. Therefore, alternative sources of genomic information about the onion are required and have begun to be developed through transcriptome sequencing.<sup><a reveal-id="DSU035C5" data-open="DSU035C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>High-throughput RNA sequencing (RNA-Seq) is a powerful and cost-effective tool to determine the structures of whole genes within genomes<sup><a reveal-id="DSU035C8  DSU035C13" data-open="DSU035C8  DSU035C13" class="link link-ref link-reveal xref-bibr">8–13</a></sup> as well as to reveal a variety of biological information.<sup><a reveal-id="DSU035C14  DSU035C18" data-open="DSU035C14  DSU035C18" class="link link-ref link-reveal xref-bibr">14–18</a></sup> Furthermore, for non-model organisms, it has been feasible to obtain whole transcripts without prior knowledge of a reference genome using <em>de novo</em> transcriptome analysis.<sup><a reveal-id="DSU035C19  DSU035C23" data-open="DSU035C19  DSU035C23" class="link link-ref link-reveal xref-bibr">19–23</a></sup> Recent studies reported, however, that most RNA-Seq sequences and assembled transcripts contain various types of sequences such as introns, transposable elements (TEs), and non-coding RNAs, which are unnecessary information for construction of protein-coding genes.<sup><a reveal-id="DSU035C16" data-open="DSU035C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU035C24  DSU035C26" data-open="DSU035C24  DSU035C26" class="link link-ref link-reveal xref-bibr">24–26</a></sup> In most cases, gene annotation of <em>de novo</em> transcriptome assemblies is performed using ‘a six-frame translation’ approach without considering non-coding sequences.<sup><a reveal-id="DSU035C17" data-open="DSU035C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU035C27  DSU035C33" data-open="DSU035C27  DSU035C33" class="link link-ref link-reveal xref-bibr">27–33</a></sup> Large groups of non-coding sequences are therefore fully or partially included in the assembled transcripts, creating a potential barrier to accurate gene annotation. Constructing a precise and refined reference transcriptome is a prerequisite for reliability in further studies.</p><p>To this end, we generated a high-quality <em>de novo</em> assembly of onion transcriptome containing a large, annotated gene set using a standard gene-prediction pipeline by combining reference mapping and <em>ab initio</em> gene models. By validating the data using pre-existing gene sets, we have provided a reliable reference gene set for future genomic and genetic research on <em>Allium</em> spp. Furthermore, our study provides a useful model of a comprehensive approach to high-quality <em>de novo</em> transcriptome assembly and annotation in non-model species.</p>                            <h2 scrollto-destination=6568842 id="6568842" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6568843 id="6568843" class="section-title" data-legacy-id=s2a>2.1. Plant materials and cDNA library construction</h3>
<p>A short-day type, doubled haploid onion with red bulbs (H6) and a lab-derived inbred line (SP3B) were used for cDNA synthesis and sequencing. Six weeks after planting vernalized bulbs, whole seedlings were harvested and then frozen in liquid nitrogen. Total RNA of each samples were extracted using TRIzol reagent (Invitrogen), and quality of extracted RNAs was inspected with Pico RNA chip and Pico6000 RNA reagent (Agilent Technologies) using a Bioanalyzer (Agilent Technologies). Strand-specific RNA-Seq paired-end libraries (insert size of 300–400 bp) were prepared according to the previous protocol,<sup><a reveal-id="DSU035C34" data-open="DSU035C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and the constructed libraries were used for transcriptome sequencing by Illumina HiSeq 2000.</p>                            <h3 scrollto-destination=6568845 id="6568845" class="section-title" data-legacy-id=s2b>2.2. <em>De novo</em> transcriptome assembly</h3>
<p>The raw sequences of the onion transcriptome were processed using an in-house preprocessing pipeline to remove unnecessary sequences for the assembly.<sup><a reveal-id="DSU035C35" data-open="DSU035C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The preprocessing pipeline consisted of four steps. First, contaminating bacterial sequences were filtered out by mapping the reads to reference bacterial genomes in GenBank using Bowtie2 v2.0.0-beta7 (--local -D 15 -R 2 -N 0 -L 20 -i S,1,0.65).<sup><a reveal-id="DSU035C36" data-open="DSU035C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Secondly, duplicated short reads were removed. Thirdly, low-quality sequences with quality scores below Q20 were eliminated and sequences over 70 bp were remained using an in-house perl script. Finally, rRNA sequences were filtered out using SortMeRNA v1.9 (default parameter).<sup><a reveal-id="DSU035C37" data-open="DSU035C37" class="link link-ref link-reveal xref-bibr">37</a></sup> The preprocessed sequences from H6 and SP3B and also a combined library of the sequences from both accessions were assembled using Velvet v1.2.08<sup><a reveal-id="DSU035C38" data-open="DSU035C38" class="link link-ref link-reveal xref-bibr">38</a></sup> (-ins_length 400 -ins_length_sd 200) and Oases v0.2.06<sup><a reveal-id="DSU035C39" data-open="DSU035C39" class="link link-ref link-reveal xref-bibr">39</a></sup> with a default parameter. To increase the assembly quality, iterative assemblies were performed to identify the <em>k</em>-mer values that gave the optimal total and average transcript lengths for each library. Thus, the optimal read lengths of 49, 47, and 53 bp were selected for the final assembly of H6, SP3B, and the combined library, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=6568847 id="6568847" class="section-title" data-legacy-id=s2c>2.3. Structural and functional annotation</h3>
<p>Structural gene annotation of the transcriptome was performed using the integrated structural gene annotation pipeline (ISGAP; Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>). First, we used the reference gene annotations of monocot plants including <em>Musa acuminata</em> version 1.0,<sup><a reveal-id="DSU035C40" data-open="DSU035C40" class="link link-ref link-reveal xref-bibr">40</a></sup><em>Oryza sativa</em> RAP version 7.0,<sup><a reveal-id="DSU035C41" data-open="DSU035C41" class="link link-ref link-reveal xref-bibr">41</a></sup><em>Phyllostachys heterocycla</em> version 1.0,<sup><a reveal-id="DSU035C42" data-open="DSU035C42" class="link link-ref link-reveal xref-bibr">42</a></sup><em>Sorghum bicolor</em> version 1.0,<sup><a reveal-id="DSU035C43" data-open="DSU035C43" class="link link-ref link-reveal xref-bibr">43</a></sup> and <em>Brachypodium distachyon</em> version 1.0<sup><a reveal-id="DSU035C44" data-open="DSU035C44" class="link link-ref link-reveal xref-bibr">44</a></sup> to perform protein alignments using Exonerate v2.2.0<sup><a reveal-id="DSU035C45" data-open="DSU035C45" class="link link-ref link-reveal xref-bibr">45</a></sup> with parameters --percent 30 and --maxintron 50,000 to find gene structures within the assembled transcripts. We then merged the detected gene structures which have same exon–exon junctions and constructed consensus sequences to use as initial gene models in the first step. During that process, we removed gene models that included frame shifts and early stop codons as well as remained one of the gene structures, which derived from a larger amount of evidence proteins and have a higher mapping score than other gene models in same regions. In the second step, partial genes in the initial gene models were extended through additional translation based on fixed frame of each gene model starting from five or three prime end of the partial genes to start or end region of the assembled transcript until translation of start or stop codon. In the third step, we constructed a training set of onion gene models using 2,000 of the complete genes generated by the second process, and we then ran Augustus<sup><a reveal-id="DSU035C46" data-open="DSU035C46" class="link link-ref link-reveal xref-bibr">46</a></sup> using the training set. After filtering the abnormal gene models from Augustus, the initial gene models were integrated into a new gene model by extending the reference proteins without overlap. In the final step, after filtering the resulting gene models against the NR database in GenBank, we determined the final gene models. </p>                        <a id="6568849" scrollto-destination="6568849"></a>
<div id="DSU035F1" data-id="DSU035F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03501.png?Expires=2147483647&Signature=FJsCfofnDYFX2Ma4nn5nT5Q8JZ6drTm2kfCyii-6vuWkO8uWHo7hrjY39jK96E2jCSaEijXIDOmwgrCBm~HuX0DsnHcsrIWe4Mc~7qW-mj0Zt884X2zkQEdtNrAYTJ4DWSMIehN2NLjSYh6V6v-bHiqI7bqG8R8D41FcncpPboo~89lPB~VKvzvkuHjunHqPVbOMNBmY-nfRIKPxkAUQ5FDXd451e8giG9VWQwsrcQ1S7tTG73sNTrUKEg~fHLikPJlrF--TjVG~XLJu57inANxfjn2hZ~Ll9CCZZW1wVyJ5Nas0axWXheqW0A76tN8Pa6QlKgtgYPjo8u2VGxHZ0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integrated structural gene annotation pipeline (ISGAP). (A) ISGAP based on reference proteins and ab initio prediction. (B) The six-frame translation method as an independent process." path-from-xml="dsu03501.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568849/dsu03501.tif" path-from-xml="dsu03501.tif" target="_blank">View large</a><a section="6568849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03501.png?Expires=2147483647&Signature=1cF6Vw8i6uvCkd3l1uRjgE8Eng33Ad3zoAbLzrqpYTpkfkVy-ok-DsACYxPXSBjhJWL8xngMSdN7y2qVifcvIpcT1hIaUt-C6FC~BRbWjjtAdM7-KfaNFb4mqgbJ5jps~0IfvND2zPGTWhvSlSzYfEkZF8Imqxs93o50r2UsJZRN9saaxBYwmZxAyI-mhsjS4KcsV9LDhMujdZddz~ZzATLb91R1mdKKNYhhxRYbh9n5KZgVRDYCQDRrF7yHS4k2K7-Xj-PnukPjc~Mk8jLULkTmhKmwzrn5mC-pYAIU7TnWlTcytgXp74yKmJLcT8rT7-eMQw~SnmT7Ev92f4EZNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568849&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03501.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integrated structural gene annotation pipeline (ISGAP). (A) ISGAP based on reference proteins and <em>ab initio</em> prediction. (B) The six-frame translation method as an independent process.</p></div></div><div content-id="DSU035F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03501.png?Expires=2147483647&Signature=FJsCfofnDYFX2Ma4nn5nT5Q8JZ6drTm2kfCyii-6vuWkO8uWHo7hrjY39jK96E2jCSaEijXIDOmwgrCBm~HuX0DsnHcsrIWe4Mc~7qW-mj0Zt884X2zkQEdtNrAYTJ4DWSMIehN2NLjSYh6V6v-bHiqI7bqG8R8D41FcncpPboo~89lPB~VKvzvkuHjunHqPVbOMNBmY-nfRIKPxkAUQ5FDXd451e8giG9VWQwsrcQ1S7tTG73sNTrUKEg~fHLikPJlrF--TjVG~XLJu57inANxfjn2hZ~Ll9CCZZW1wVyJ5Nas0axWXheqW0A76tN8Pa6QlKgtgYPjo8u2VGxHZ0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integrated structural gene annotation pipeline (ISGAP). (A) ISGAP based on reference proteins and ab initio prediction. (B) The six-frame translation method as an independent process." path-from-xml="dsu03501.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568849/dsu03501.tif" path-from-xml="dsu03501.tif" target="_blank">View large</a><a section="6568849" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03501.png?Expires=2147483647&Signature=1cF6Vw8i6uvCkd3l1uRjgE8Eng33Ad3zoAbLzrqpYTpkfkVy-ok-DsACYxPXSBjhJWL8xngMSdN7y2qVifcvIpcT1hIaUt-C6FC~BRbWjjtAdM7-KfaNFb4mqgbJ5jps~0IfvND2zPGTWhvSlSzYfEkZF8Imqxs93o50r2UsJZRN9saaxBYwmZxAyI-mhsjS4KcsV9LDhMujdZddz~ZzATLb91R1mdKKNYhhxRYbh9n5KZgVRDYCQDRrF7yHS4k2K7-Xj-PnukPjc~Mk8jLULkTmhKmwzrn5mC-pYAIU7TnWlTcytgXp74yKmJLcT8rT7-eMQw~SnmT7Ev92f4EZNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568849&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03501.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integrated structural gene annotation pipeline (ISGAP). (A) ISGAP based on reference proteins and <em>ab initio</em> prediction. (B) The six-frame translation method as an independent process.</p></div></div><p>For the final gene models, we extracted representative gene models remaining one of the genes, which have a longest complete form existing both start and stop codon in same locus of assembled transcripts. In case of no complete genes in the locus, we selected a longest partial gene as a representative gene model in the locus. The biological functions of the final gene models were assigned using InterProScan version 5-46,<sup><a reveal-id="DSU035C47" data-open="DSU035C47" class="link link-ref link-reveal xref-bibr">47</a></sup> the plant proteins in the RefSeq database,<sup><a reveal-id="DSU035C48" data-open="DSU035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and the Uniprot database,<sup><a reveal-id="DSU035C49" data-open="DSU035C49" class="link link-ref link-reveal xref-bibr">49</a></sup> which contains the SWISS-PROT and TrEMBL databases with cut-off values <em>e</em>-value over 1<em>e</em>-5.</p>                            <h3 scrollto-destination=6568851 id="6568851" class="section-title" data-legacy-id=s2d>2.4. Evaluation of the predicted gene sets</h3>
<p>The gene models predicted by ISGAP and the six-frame translation were validated by aligning the assembled transcripts to both the onion and the plant proteins of the RefSeq database<sup><a reveal-id="DSU035C48" data-open="DSU035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> in GenBank using Exonerate v2.2.0.<sup><a reveal-id="DSU035C45" data-open="DSU035C45" class="link link-ref link-reveal xref-bibr">45</a></sup> To obtain accurate gene regions, we applied strict parameters for both validations (--percent 70 and over 90% mapping coverage). All of the correctly aligned regions that allowed redundancy, except for cases where a frame shift and early stop codon were detected, were compared as independent query sequences to the gene models predicted by ISGAP and six-frame translation. The gene models were evaluated based on their ability to represent query sequences considering the matched region, strand, and frame.</p>                            <h3 scrollto-destination=6568853 id="6568853" class="section-title" data-legacy-id=s2e>2.5. Sequence variation between the two accessions</h3>
<p>A comparative transcriptomic analysis was performed to detect variation between H6 and SP3B. To detect variation, a reference-guided assembly was conducted by mapping the preprocessed raw SP3B sequence data to the H6 assembly via Bowtie2 v2.0.0-beta7<sup><a reveal-id="DSU035C36" data-open="DSU035C36" class="link link-ref link-reveal xref-bibr">36</a></sup> (default parameter) Samtools v0.1.18<sup><a reveal-id="DSU035C50" data-open="DSU035C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and SNPeff v3.3h<sup><a reveal-id="DSU035C51" data-open="DSU035C51" class="link link-ref link-reveal xref-bibr">51</a></sup> (-minC 5 -minQ 20) identified sequence variation and classified type of the variation by region of the assembled transcripts. To find more reliable variants, we extracted the sequence variants with their bi-directional 50-mer flanking sequences and confirmed that the sequence variants were detected in the flanking regions of the SP3B assembly using the in-house perl script.</p>                            <h2 scrollto-destination=6568855 id="6568855" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6568856 id="6568856" class="section-title" data-legacy-id=s3a>3.1. Sequencing, assembly, and repeat annotation</h3>
<p>We obtained 15.0 and 11.6 Gb of whole-transcriptome sequences from H6 and SP3B by Illumina HiSeq 2000 using 101-bp paired-end reads (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). After preprocessing, a total of 10.1 and 7.9 Gb of sequences were remained for H6 and SP3B, respectively, and were used for <em>de novo</em> assembly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Using the combined library, a total of 203.0 Mb were assembled into 165,179 transcripts (mean length = 1,228.9 bp; N50 = 1,756 bp; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The length distribution of the assembly showed that 76,699 transcripts (76.8% of the total) were longer than 1,000 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). In addition, the assembly of the H6 and SP3B libraries separately produced 108,450 and 94,051 transcripts with an average length of 1,271.2 and 1,211.8 bp, respectively.</p><p>To validate the three sets of assembled transcripts, we conducted BLASTN using 20,159 onion ESTs downloaded from GenBank, 18,393 (91.2%) of which were detected among the assembled transcripts from the combined library with 98% sequence identity (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>A). Moreover, most (83.4%) of the ESTs were represented among the assembled transcripts with at least 98% sequence identity and 80% coverage. Additionally, we confirmed that most of the ESTs were matched and covered by the assemblies of the H6 and SP3B libraries using the same parameters (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>B and C).</p><p>Prior to structural gene prediction, we performed repeat masking using a constructed <em>de novo</em> repeat library made from the assembly of the combined library. A total of 38.9 Mb (19.2%) of the transcript sequences were determined to be TE-related repeats, including classified and unclassified repeats (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). A recent review reported that highly conserved and multicopy genes such as histone and tubulin can be recognized as repeat sequences during repeat masking.<sup><a reveal-id="DSU035C8" data-open="DSU035C8" class="link link-ref link-reveal xref-bibr">8</a></sup> To avoid missing conserved protein-coding genes, we masked only the classified TEs including long interspersed nuclear elements, short interspersed nuclear elements, and long terminal repeats (LTRs) in the assembly of the combined library. In the assembly of the H6 and SP3B libraries, classified TEs were also masked to achieve optimal gene prediction for each assembly (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                            <h3 scrollto-destination=6568860 id="6568860" class="section-title" data-legacy-id=s3b>3.2. Structural annotation</h3>
<p>Protein-coding genes in the transcriptome have mostly been predicted by classical methods such as six-frame translation of transcripts.<sup><a reveal-id="DSU035C17" data-open="DSU035C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU035C27  DSU035C33" data-open="DSU035C27  DSU035C33" class="link link-ref link-reveal xref-bibr">27–33</a></sup> That approach has been controversial, however, because of intermixed intron regions in the transcriptome. In order to predict more reliable gene sets, we established ISGAP to compare and integrate various gene-prediction approaches (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>). ISGAP comprises various gene-prediction approaches such as six-frame translation, evidence-based gene models based on protein mapping, <em>ab initio</em> gene prediction, and combined gene models.</p><p>To structurally annotate the genes in the onion transcriptome, protein alignment of the assembled transcripts was carried out to detect gene structures (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A). To extend the gene models extracted from the protein alignments using reference genes, we conducted additional translation for partial genes, which have no start or stop positions (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A). Thus, 42,435 gene models containing 26,598 complete genes were generated using reference proteins for further analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). After the construction of the training set of Augustus,<sup><a reveal-id="DSU035C46" data-open="DSU035C46" class="link link-ref link-reveal xref-bibr">46</a></sup> 51,092 gene models were predicted by Augustus and used to generate combined gene models with the gene models previously produced by Step 2 in ISGAP. Finally, 54,165 total genes and 20,447 representative protein-coding genes were annotated as a reference gene set of the onion after filtering against the NR database (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A and Table <a reveal-id="DSU035TB1" data-open="DSU035TB1" class="link link-reveal link-table xref-fig">1</a>). For the H6 and SP3B assemblies, we performed gene prediction using the same pipeline (Table <a reveal-id="DSU035TB1" data-open="DSU035TB1" class="link link-reveal link-table xref-fig">1</a>). Additionally, to compare the gene models from ISGAP, six-frame translation was conducted as an independent step, and 65,645 genes were obtained (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>B and Table <a reveal-id="DSU035TB2" data-open="DSU035TB2" class="link link-reveal link-table xref-fig">2</a>). The gene sets predicted by ISGAP showed a longer average length than those predicted by six-frame translation (Table <a reveal-id="DSU035TB2" data-open="DSU035TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). </p>                        <a id="6568863" scrollto-destination="6568863"></a>
<div content-id="DSU035TB1" class="table-modal"><div class="table-wrap"><div id="DSU035TB1" class="table-wrap-title" data-id="DSU035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the annotated onion gene sets from ISGAP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Combined </th><th>H6 </th><th>SP3B </th></tr></thead><tbody><tr><td colspan="4">Whole </td></tr><tr><td> Number of genes </td><td>54,165 </td><td>38,004 </td><td>35,750 </td></tr><tr><td> Total length (Mb) </td><td>59.6 </td><td>42.3 </td><td>40.6 </td></tr><tr><td> Average length (bp) </td><td>1,100.4 </td><td>1,112.1 </td><td>1,136.5 </td></tr><tr><td colspan="4">Representative </td></tr><tr><td> Number of genes </td><td>20,447 </td><td>18,034 </td><td>17,101 </td></tr><tr><td> Total length (Mb) </td><td>22.0 </td><td>20.5 </td><td>19.4 </td></tr><tr><td> Average length (bp) </td><td>1,075.0 </td><td>1,135.2 </td><td>1,134.9 </td></tr></tbody></table></div><div content-id="DSU035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Combined </th><th>H6 </th><th>SP3B </th></tr></thead><tbody><tr><td colspan="4">Whole </td></tr><tr><td> Number of genes </td><td>54,165 </td><td>38,004 </td><td>35,750 </td></tr><tr><td> Total length (Mb) </td><td>59.6 </td><td>42.3 </td><td>40.6 </td></tr><tr><td> Average length (bp) </td><td>1,100.4 </td><td>1,112.1 </td><td>1,136.5 </td></tr><tr><td colspan="4">Representative </td></tr><tr><td> Number of genes </td><td>20,447 </td><td>18,034 </td><td>17,101 </td></tr><tr><td> Total length (Mb) </td><td>22.0 </td><td>20.5 </td><td>19.4 </td></tr><tr><td> Average length (bp) </td><td>1,075.0 </td><td>1,135.2 </td><td>1,134.9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568863" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU035TB1" class="table-wrap-title" data-id="DSU035TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of the annotated onion gene sets from ISGAP</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Combined </th><th>H6 </th><th>SP3B </th></tr></thead><tbody><tr><td colspan="4">Whole </td></tr><tr><td> Number of genes </td><td>54,165 </td><td>38,004 </td><td>35,750 </td></tr><tr><td> Total length (Mb) </td><td>59.6 </td><td>42.3 </td><td>40.6 </td></tr><tr><td> Average length (bp) </td><td>1,100.4 </td><td>1,112.1 </td><td>1,136.5 </td></tr><tr><td colspan="4">Representative </td></tr><tr><td> Number of genes </td><td>20,447 </td><td>18,034 </td><td>17,101 </td></tr><tr><td> Total length (Mb) </td><td>22.0 </td><td>20.5 </td><td>19.4 </td></tr><tr><td> Average length (bp) </td><td>1,075.0 </td><td>1,135.2 </td><td>1,134.9 </td></tr></tbody></table></div><div content-id="DSU035TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Combined </th><th>H6 </th><th>SP3B </th></tr></thead><tbody><tr><td colspan="4">Whole </td></tr><tr><td> Number of genes </td><td>54,165 </td><td>38,004 </td><td>35,750 </td></tr><tr><td> Total length (Mb) </td><td>59.6 </td><td>42.3 </td><td>40.6 </td></tr><tr><td> Average length (bp) </td><td>1,100.4 </td><td>1,112.1 </td><td>1,136.5 </td></tr><tr><td colspan="4">Representative </td></tr><tr><td> Number of genes </td><td>20,447 </td><td>18,034 </td><td>17,101 </td></tr><tr><td> Total length (Mb) </td><td>22.0 </td><td>20.5 </td><td>19.4 </td></tr><tr><td> Average length (bp) </td><td>1,075.0 </td><td>1,135.2 </td><td>1,134.9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568863" target="_blank">View Large</a></div></div>                        <a id="6568864" scrollto-destination="6568864"></a>
<div content-id="DSU035TB2" class="table-modal"><div class="table-wrap"><div id="DSU035TB2" class="table-wrap-title" data-id="DSU035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Detailed statistics for all the annotated gene sets from six-frame translation and ISGAP using the combined library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Six-frame translation </th><th>Step 1<sup>a</sup> </th><th>Step 2<sup>b</sup> </th><th>Step 3<sup>c</sup> </th><th>Steps 2 + 3<sup>d</sup> </th><th>Final </th></tr></thead><tbody><tr><td>Number of genes </td><td>65,645 </td><td>42,435 </td><td>42,435 </td><td>51,092 </td><td>61,852 </td><td>54,165 </td></tr><tr><td>Number of genes containing multiple exons </td><td>N/A </td><td>9,481 </td><td>9,481 </td><td>10,207 </td><td>13,516 </td><td>11,496 </td></tr><tr><td>Number of introns </td><td>N/A </td><td>11,015 </td><td>11,015 </td><td>12,436 </td><td>16,600 </td><td>13,543 </td></tr><tr><td>Average length of exons (bp) </td><td>945.0 </td><td>752.9 </td><td>859.8 </td><td>886.1 </td><td>813.6 </td><td>880.3 </td></tr><tr><td>Average length of introns (bp) </td><td>N/A </td><td>298.9 </td><td>298.9 </td><td>344.7 </td><td>310.6 </td><td>307.7 </td></tr></tbody></table></div><div content-id="DSU035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Six-frame translation </th><th>Step 1<sup>a</sup> </th><th>Step 2<sup>b</sup> </th><th>Step 3<sup>c</sup> </th><th>Steps 2 + 3<sup>d</sup> </th><th>Final </th></tr></thead><tbody><tr><td>Number of genes </td><td>65,645 </td><td>42,435 </td><td>42,435 </td><td>51,092 </td><td>61,852 </td><td>54,165 </td></tr><tr><td>Number of genes containing multiple exons </td><td>N/A </td><td>9,481 </td><td>9,481 </td><td>10,207 </td><td>13,516 </td><td>11,496 </td></tr><tr><td>Number of introns </td><td>N/A </td><td>11,015 </td><td>11,015 </td><td>12,436 </td><td>16,600 </td><td>13,543 </td></tr><tr><td>Average length of exons (bp) </td><td>945.0 </td><td>752.9 </td><td>859.8 </td><td>886.1 </td><td>813.6 </td><td>880.3 </td></tr><tr><td>Average length of introns (bp) </td><td>N/A </td><td>298.9 </td><td>298.9 </td><td>344.7 </td><td>310.6 </td><td>307.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><div class="label -label">a</div><p>Gene model derived from reference proteins.</p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">b</div><p>Extended gene model through the translation of partial genes in Step 2.</p></div></div><div content-id="TN3"><div class="fn"><div class="label -label">c</div><p><em>Ab initio</em> predicted a gene model.</p></div></div><div content-id="TN4"><div class="fn"><div class="label -label">d</div><p>Integrated gene model of Steps 2 and 3.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568864" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU035TB2" class="table-wrap-title" data-id="DSU035TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Detailed statistics for all the annotated gene sets from six-frame translation and ISGAP using the combined library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Six-frame translation </th><th>Step 1<sup>a</sup> </th><th>Step 2<sup>b</sup> </th><th>Step 3<sup>c</sup> </th><th>Steps 2 + 3<sup>d</sup> </th><th>Final </th></tr></thead><tbody><tr><td>Number of genes </td><td>65,645 </td><td>42,435 </td><td>42,435 </td><td>51,092 </td><td>61,852 </td><td>54,165 </td></tr><tr><td>Number of genes containing multiple exons </td><td>N/A </td><td>9,481 </td><td>9,481 </td><td>10,207 </td><td>13,516 </td><td>11,496 </td></tr><tr><td>Number of introns </td><td>N/A </td><td>11,015 </td><td>11,015 </td><td>12,436 </td><td>16,600 </td><td>13,543 </td></tr><tr><td>Average length of exons (bp) </td><td>945.0 </td><td>752.9 </td><td>859.8 </td><td>886.1 </td><td>813.6 </td><td>880.3 </td></tr><tr><td>Average length of introns (bp) </td><td>N/A </td><td>298.9 </td><td>298.9 </td><td>344.7 </td><td>310.6 </td><td>307.7 </td></tr></tbody></table></div><div content-id="DSU035TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Six-frame translation </th><th>Step 1<sup>a</sup> </th><th>Step 2<sup>b</sup> </th><th>Step 3<sup>c</sup> </th><th>Steps 2 + 3<sup>d</sup> </th><th>Final </th></tr></thead><tbody><tr><td>Number of genes </td><td>65,645 </td><td>42,435 </td><td>42,435 </td><td>51,092 </td><td>61,852 </td><td>54,165 </td></tr><tr><td>Number of genes containing multiple exons </td><td>N/A </td><td>9,481 </td><td>9,481 </td><td>10,207 </td><td>13,516 </td><td>11,496 </td></tr><tr><td>Number of introns </td><td>N/A </td><td>11,015 </td><td>11,015 </td><td>12,436 </td><td>16,600 </td><td>13,543 </td></tr><tr><td>Average length of exons (bp) </td><td>945.0 </td><td>752.9 </td><td>859.8 </td><td>886.1 </td><td>813.6 </td><td>880.3 </td></tr><tr><td>Average length of introns (bp) </td><td>N/A </td><td>298.9 </td><td>298.9 </td><td>344.7 </td><td>310.6 </td><td>307.7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><div class="label -label">a</div><p>Gene model derived from reference proteins.</p></div></div><div content-id="TN2"><div class="fn"><div class="label -label">b</div><p>Extended gene model through the translation of partial genes in Step 2.</p></div></div><div content-id="TN3"><div class="fn"><div class="label -label">c</div><p><em>Ab initio</em> predicted a gene model.</p></div></div><div content-id="TN4"><div class="fn"><div class="label -label">d</div><p>Integrated gene model of Steps 2 and 3.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568864" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6568865 id="6568865" class="section-title" data-legacy-id=s3c>3.3. Validation and comparison of the gene models from ISGAP and six-frame translation</h3>
<p>To evaluate the gene models predicted by ISGAP and six-frame translation, we aligned the onion proteins in GenBank to the assembled transcripts. With over 90% mapping coverage, 398 of 511 (77.9%) of the onion proteins were detected among 1,008 regions in the assembled transcripts (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). As a result, 348 and 281 genes predicted by ISGAP and six-frame translation, respectively, covered 351 and 344 (88.2 and 86.4%) of the mapped onion proteins with over 99% query coverage. Moreover, 866 and 746 (85.9 and 74.0%) of the mapped regions were represented by the genes predicted by ISGAP and six-frame translation, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). </p>                        <a id="6568867" scrollto-destination="6568867"></a>
<div id="DSU035F2" data-id="DSU035F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03502.png?Expires=2147483647&Signature=ASmHTR0ScKgBGHg8gu8auxTkyWsGdKsZJ83J3YiHaIgwFAEvLHWcX7Xyf5dw77H4sRX~DgPGdCtEKEB0EiEk9i08tNUPx-TNNWOME2aew0Ix3Q4v~cnl3Ar~pKhcvdHhjp9lDPuowT8ZAopFilR~rmutCKuX1pgmeObmeFbGzR~rUjxWB-YNLsty0MM065cn9-NVFDZlr-MfhsodCkIMHSpLZHOy4r1pdEvsLFypm3CWxVOyfgDDGP8ktxtY4IN8cF3XX9~Av3697SSfeAPmg~wrpHZQAbjMBqo9KEXvSGhXcVXDwOcCv6YvHCg45ioYFxFluZC7qWyd~AAWqxA9cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the annotated onion gene sets predicted the combined library by six-frame translation and ISGAP. For the black dotted line, the left and right of the histogram represent the numbers of covered query sequences and predicted genes, respectively. (A) Validation of the predicted gene sets using 511 onion proteins. (B) Assessment of the predicted proteins against the plant proteins in the RefSeq database." path-from-xml="dsu03502.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568867/dsu03502.tif" path-from-xml="dsu03502.tif" target="_blank">View large</a><a section="6568867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03502.png?Expires=2147483647&Signature=mbDMxq2I15qgC0XGp5ru~quxGlj8vzDPUrO-BK0z0sDyjahJj~Tt6MbWVLdwVjoxN-Ak58AbcbUzPpObw3F4zEFDaeYXCyz87Q0XJ~ozmu6SnnDGXdH3zzFd6AzxjadsSNkf1YbP0JViVdW~k7iQ9~BySC2qMRXeDMq4OkLGXC-iC5vzeKgKMXjItJlCvPZgBUqhmXbV2leAqKHJrxW~I~7nJt1x0A68ZWO~vu0f9tm2kVWwsMLmOO~CNfkHh-QrN7Reyb8R2E8p0xPE3ZQDU22eEU3Yjt89c8ej5p0CwpQqCB77aWH3an2o-lOrJ7OeKOaJyukxp-RI4gXNHBtixw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568867&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03502.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the annotated onion gene sets predicted the combined library by six-frame translation and ISGAP. For the black dotted line, the left and right of the histogram represent the numbers of covered query sequences and predicted genes, respectively. (A) Validation of the predicted gene sets using 511 onion proteins. (B) Assessment of the predicted proteins against the plant proteins in the RefSeq database.</p></div></div><div content-id="DSU035F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03502.png?Expires=2147483647&Signature=ASmHTR0ScKgBGHg8gu8auxTkyWsGdKsZJ83J3YiHaIgwFAEvLHWcX7Xyf5dw77H4sRX~DgPGdCtEKEB0EiEk9i08tNUPx-TNNWOME2aew0Ix3Q4v~cnl3Ar~pKhcvdHhjp9lDPuowT8ZAopFilR~rmutCKuX1pgmeObmeFbGzR~rUjxWB-YNLsty0MM065cn9-NVFDZlr-MfhsodCkIMHSpLZHOy4r1pdEvsLFypm3CWxVOyfgDDGP8ktxtY4IN8cF3XX9~Av3697SSfeAPmg~wrpHZQAbjMBqo9KEXvSGhXcVXDwOcCv6YvHCg45ioYFxFluZC7qWyd~AAWqxA9cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the annotated onion gene sets predicted the combined library by six-frame translation and ISGAP. For the black dotted line, the left and right of the histogram represent the numbers of covered query sequences and predicted genes, respectively. (A) Validation of the predicted gene sets using 511 onion proteins. (B) Assessment of the predicted proteins against the plant proteins in the RefSeq database." path-from-xml="dsu03502.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568867/dsu03502.tif" path-from-xml="dsu03502.tif" target="_blank">View large</a><a section="6568867" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03502.png?Expires=2147483647&Signature=mbDMxq2I15qgC0XGp5ru~quxGlj8vzDPUrO-BK0z0sDyjahJj~Tt6MbWVLdwVjoxN-Ak58AbcbUzPpObw3F4zEFDaeYXCyz87Q0XJ~ozmu6SnnDGXdH3zzFd6AzxjadsSNkf1YbP0JViVdW~k7iQ9~BySC2qMRXeDMq4OkLGXC-iC5vzeKgKMXjItJlCvPZgBUqhmXbV2leAqKHJrxW~I~7nJt1x0A68ZWO~vu0f9tm2kVWwsMLmOO~CNfkHh-QrN7Reyb8R2E8p0xPE3ZQDU22eEU3Yjt89c8ej5p0CwpQqCB77aWH3an2o-lOrJ7OeKOaJyukxp-RI4gXNHBtixw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568867&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03502.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the annotated onion gene sets predicted the combined library by six-frame translation and ISGAP. For the black dotted line, the left and right of the histogram represent the numbers of covered query sequences and predicted genes, respectively. (A) Validation of the predicted gene sets using 511 onion proteins. (B) Assessment of the predicted proteins against the plant proteins in the RefSeq database.</p></div></div><p>Although the reference onion proteins matched more of the genes predicted by ISGAP than those predicted by six-frame translation, it was difficult to evaluate whole annotated genes because of the small number of known onion proteins in GenBank. To validate the whole genes in the annotated gene sets intensively, we aligned the assembled transcripts to the plant proteins in the RefSeq database.<sup><a reveal-id="DSU035C48" data-open="DSU035C48" class="link link-ref link-reveal xref-bibr">48</a></sup> A total of 49,257 RefSeq proteins were discovered among 126,607 regions in the assembled transcripts (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Similar numbers of proteins were detected with over 99% query coverage in both the ISGAP (45,724) and the six-frame translation (44,448) libraries. The numbers of mapped regions and the corresponding genes were different, however, between the ISGAP and six-frame translation libraries (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The 8,324 genes predicted by ISGAP covered 114,593 (90.5%) of the mapped regions, whereas the 6,526 genes predicted by six-frame translation covered 93,256 (73.7%) of the mapped regions, suggesting that the number of precisely annotated genes in the ISGAP library was substantially higher than that in the six-frame translation library (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>).</p><p>To investigate differences between the gene sets, we selected and assessed all cases except both gene sets covered the same mapped regions in the validation results with 99% query coverage (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). For the onion reference proteins, 120 and 35 were represented by 94 and 27 gene annotations determined by ISGAP and six-frame translation, respectively (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>A). For the RefSeq proteins, 16,373 and 3,393 were represented by 2,341 and 543 gene annotations determined by ISGAP and six-frame translation, respectively (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>B). Although similar numbers of onion and RefSeq proteins were covered by genes from ISGAP and six-frame translation in the whole validation, the genes predicted by ISGAP in the extracted cases covered higher numbers of onion and RefSeq proteins, due to the fact that the same onion and RefSeq proteins were mapped in a different form on multiple transcripts (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>A).</p>                            <h3 scrollto-destination=6568870 id="6568870" class="section-title" data-legacy-id=s3d>3.4. Assessment of multiple exon genes and annotated gene structures</h3>
<p>Some of the onion and RefSeq proteins were mapped as genes containing multiple exons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>A). To validate those genes, including the multiple exons, we identified the cases where the onion and RefSeq proteins were mapped as genes containing multiple exons in both of the validation results (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). In the validation using the onion proteins, 92 proteins were mapped onto 124 of the mapped regions (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>A and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). For the proteins with over 99% query coverage, 23 genes predicted by ISGAP were represented by 78 (84.8%) of the onion proteins; however, the genes predicted by six-frame translation were not represented by any of the onion proteins (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>A). Moreover, 14,450 of the RefSeq proteins were detected among 24,719 of the mapped regions. With the same query coverage, 12,904 (89.3%) of the RefSeq proteins were correctly matched by 1,573 of the genes predicted by ISGAP (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). In contrast, only 6 (0.03%) of the RefSeq proteins were correctly matched by three of the genes predicted by six-frame translation (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>B and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Rarely, the genes were fully matched to other RefSeq proteins mapped as single-exon genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>B).</p><p>Through the evaluation, we discovered miss-annotated genes derived from the six-frame translation. To verify the reasons for the miss-annotation, we examined the detailed structure of each gene model and identified representative cases among the validation results (Fig. <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>). We found that the miss-annotations from the six-frame translation were caused by the retention of introns and the translation of the inappropriate region or strand (Fig. <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>). For the corresponding regions, ISGAP successfully performed the gene annotations (Fig. <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>). Hence, ISGAP could detect and extract accurate exon regions based on the structure of the reference protein and <em>ab initio</em> gene models, whereas six-frame translation could not. </p>                        <a id="6568873" scrollto-destination="6568873"></a>
<div id="DSU035F3" data-id="DSU035F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03503.png?Expires=2147483647&Signature=tzfGh~yOaNLC~oxPNjw5r36CrpEKxg6S~RAjLlRLCaLfXZ8ruoi3X6q~lUhBEmNOBFQHtMwvcVvo53biZouMxj90FEJpoYBgMzSeQkp4EGvT5Ir~merWfPH1DNVHomVTy6VflONuCV6rA-vToDV8DTSJX33UB598n6wsfz3lvpGpsxn64fhtrMhgeHwhLfqSO02D8G83hr6rLLZQsf8NE8kPzkDWUJuHb9k7eoYf6AAqr5cIWrprdHNb3TwFwp3EewCm87McqidJrLJnhhzbGMITuJdJvZSFDgq9CPgo5BtZRozs0zTwquHJTB4byy6wEbVODgznmG-1~bOaV8y7cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative cases of well-annotated genes predicted by ISGAP compared with the genes predicted by six-frame translation. The genes predicted by ISGAP and six-frame translation are shown, as well as the onion, RefSeq, reference, and ab initio gene models. The plus and minus signs in the brackets indicate the strand of mapped or predicted genes. (A and B) Cases of genes containing multiple exons; (C) gene annotation with the correct region. (D) Gene annotation with the correct strand." path-from-xml="dsu03503.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568873/dsu03503.tif" path-from-xml="dsu03503.tif" target="_blank">View large</a><a section="6568873" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03503.png?Expires=2147483647&Signature=oH4Q~y0tSpr-lM~eKpVYeJ7YBV2ANVcEobiLlKAWJH6WkovxLXTFa~aZxncrzLd2juQwp0rn3x4zaTjEr7dEGkgtxX1FTjnqnKHIfUMnAqs6Ly3fgvzCEVf6HSgX2V3WHBahz55tIvIw7DNFc3s8Tisp1-Vir0cJTNVkgghyPMtK7-sNb~z9fXposhgiS7PZTyZmckTPzd-fU~bhi3yg7dDtE2ApcNpLFlj2Oqou9YA18~Z2~nnVHwxUJX9R775Fu3geiNUnVfFj4-L6Ntv6WGfUvvFcVD1eimbPvP~hE4w1K924WgwV4Bw~L-Zblf6gBG5xaMGcr5Z~7Oa0w-PvxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568873&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03503.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative cases of well-annotated genes predicted by ISGAP compared with the genes predicted by six-frame translation. The genes predicted by ISGAP and six-frame translation are shown, as well as the onion, RefSeq, reference, and <em>ab initio</em> gene models. The plus and minus signs in the brackets indicate the strand of mapped or predicted genes. (A and B) Cases of genes containing multiple exons; (C) gene annotation with the correct region. (D) Gene annotation with the correct strand.</p></div></div><div content-id="DSU035F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03503.png?Expires=2147483647&Signature=tzfGh~yOaNLC~oxPNjw5r36CrpEKxg6S~RAjLlRLCaLfXZ8ruoi3X6q~lUhBEmNOBFQHtMwvcVvo53biZouMxj90FEJpoYBgMzSeQkp4EGvT5Ir~merWfPH1DNVHomVTy6VflONuCV6rA-vToDV8DTSJX33UB598n6wsfz3lvpGpsxn64fhtrMhgeHwhLfqSO02D8G83hr6rLLZQsf8NE8kPzkDWUJuHb9k7eoYf6AAqr5cIWrprdHNb3TwFwp3EewCm87McqidJrLJnhhzbGMITuJdJvZSFDgq9CPgo5BtZRozs0zTwquHJTB4byy6wEbVODgznmG-1~bOaV8y7cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representative cases of well-annotated genes predicted by ISGAP compared with the genes predicted by six-frame translation. The genes predicted by ISGAP and six-frame translation are shown, as well as the onion, RefSeq, reference, and ab initio gene models. The plus and minus signs in the brackets indicate the strand of mapped or predicted genes. (A and B) Cases of genes containing multiple exons; (C) gene annotation with the correct region. (D) Gene annotation with the correct strand." path-from-xml="dsu03503.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568873/dsu03503.tif" path-from-xml="dsu03503.tif" target="_blank">View large</a><a section="6568873" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03503.png?Expires=2147483647&Signature=oH4Q~y0tSpr-lM~eKpVYeJ7YBV2ANVcEobiLlKAWJH6WkovxLXTFa~aZxncrzLd2juQwp0rn3x4zaTjEr7dEGkgtxX1FTjnqnKHIfUMnAqs6Ly3fgvzCEVf6HSgX2V3WHBahz55tIvIw7DNFc3s8Tisp1-Vir0cJTNVkgghyPMtK7-sNb~z9fXposhgiS7PZTyZmckTPzd-fU~bhi3yg7dDtE2ApcNpLFlj2Oqou9YA18~Z2~nnVHwxUJX9R775Fu3geiNUnVfFj4-L6Ntv6WGfUvvFcVD1eimbPvP~hE4w1K924WgwV4Bw~L-Zblf6gBG5xaMGcr5Z~7Oa0w-PvxQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568873&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03503.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representative cases of well-annotated genes predicted by ISGAP compared with the genes predicted by six-frame translation. The genes predicted by ISGAP and six-frame translation are shown, as well as the onion, RefSeq, reference, and <em>ab initio</em> gene models. The plus and minus signs in the brackets indicate the strand of mapped or predicted genes. (A and B) Cases of genes containing multiple exons; (C) gene annotation with the correct region. (D) Gene annotation with the correct strand.</p></div></div>                            <h3 scrollto-destination=6568874 id="6568874" class="section-title" data-legacy-id=s3e>3.5. Functional annotation and transcriptomic variation for the construction of genomic resources</h3>
<p>To date, the repertory of whole genes in the onion is unknown because of the absence of a reference genome. Therefore, we assigned functional information to the onion transcriptome based on the gene set predicted by ISGAP. We performed functional annotation using InterProScan version 5-46,<sup><a reveal-id="DSU035C47" data-open="DSU035C47" class="link link-ref link-reveal xref-bibr">47</a></sup> containing Pfam, SMART, ProSite, and GO Term analysis. A total of 27,421 genes (50.6% of the total) were assigned functions based on the definitions of their domains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Among the annotated genes, the protein kinase domain was the most abundant and was detected in 1,283 genes (Fig. <a reveal-id="DSU035F4" data-open="DSU035F4" class="link link-reveal link-table xref-fig">4</a>A). Moreover, the top 20 domains revealed the most actively expressed gene families in the onion (Fig. <a reveal-id="DSU035F4" data-open="DSU035F4" class="link link-reveal link-table xref-fig">4</a>A). </p>                        <a id="6568876" scrollto-destination="6568876"></a>
<div id="DSU035F4" data-id="DSU035F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03504.png?Expires=2147483647&Signature=ViojzogihFNBb3sFBUMD0E~duVrwKKixCrhJSOaay6yRFt882MqydvWiFx3yu~zdDlPa46CO8sfkvXvSNTUEnJwOW5ZNP3Cw3bbvDSTDeYEBMu52HZwHwdiD8dWHi~P7mu0CO6J7DjEQLwf1RoEvE~kXLxjit-5Mpy-mZS3NceQPXHvMa2fsoBzHvXuwzUKXTVw85SO4w-yAvKmqh4f60ykScGGVdMacnTLarUAZj8PI9G0gCOpfCdQcXq-JYV4UDZ~FG16vJl7hzBdUjQyxKM6QOjL4DshudHcssuUCK-6dz4KN-GB3Cw2DDwK3iKb7c21IqRgIQUffdkbxwaseEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of biological functions and coverage graph for monocot and dicot plants. (A) Top 20 InterPro domains among the onion genes from the combined library. (B) Coverage graph of the onion genes in the assembly of the combined library on monocot and dicot plants. The line graph and histogram illustrate the proportions of onion genes and plant proteins in each species, respectively." path-from-xml="dsu03504.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568876/dsu03504.tif" path-from-xml="dsu03504.tif" target="_blank">View large</a><a section="6568876" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03504.png?Expires=2147483647&Signature=BM5jLsK8mF3yLTt4i0JUssm9PA686IpZnepMJOhF57k49zGJi1AF4juMINV9llYg0XS3tThxev3k5DUgkZUeN0KscSYPvA~Scdq82gx-iX4G177dnG4ExleR2E1G6JqQsyv2eWKYLdM7~qIxUnqiMiKO6dFnuCxgtk3yr7eCc8rQs-JEbNjqiIkGDxev6KNLsxHMWXuZezuImhACDD0s8GZFeKBRA-2ZZtt8Y8Dny-3NUiZ8~zVvPPfh4hTx4BSZpUfkWdXC0jpJ2v6blQ6sOr5aNklCZPklU2AqN2MAED8qII2hK98lpo6FPmLoGk6GgewH7Wdv~GXxkv3jW1dAEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568876&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03504.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of biological functions and coverage graph for monocot and dicot plants. (A) Top 20 InterPro domains among the onion genes from the combined library. (B) Coverage graph of the onion genes in the assembly of the combined library on monocot and dicot plants. The line graph and histogram illustrate the proportions of onion genes and plant proteins in each species, respectively.</p></div></div><div content-id="DSU035F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/m_dsu03504.png?Expires=2147483647&Signature=ViojzogihFNBb3sFBUMD0E~duVrwKKixCrhJSOaay6yRFt882MqydvWiFx3yu~zdDlPa46CO8sfkvXvSNTUEnJwOW5ZNP3Cw3bbvDSTDeYEBMu52HZwHwdiD8dWHi~P7mu0CO6J7DjEQLwf1RoEvE~kXLxjit-5Mpy-mZS3NceQPXHvMa2fsoBzHvXuwzUKXTVw85SO4w-yAvKmqh4f60ykScGGVdMacnTLarUAZj8PI9G0gCOpfCdQcXq-JYV4UDZ~FG16vJl7hzBdUjQyxKM6QOjL4DshudHcssuUCK-6dz4KN-GB3Cw2DDwK3iKb7c21IqRgIQUffdkbxwaseEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of biological functions and coverage graph for monocot and dicot plants. (A) Top 20 InterPro domains among the onion genes from the combined library. (B) Coverage graph of the onion genes in the assembly of the combined library on monocot and dicot plants. The line graph and histogram illustrate the proportions of onion genes and plant proteins in each species, respectively." path-from-xml="dsu03504.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6568876/dsu03504.tif" path-from-xml="dsu03504.tif" target="_blank">View large</a><a section="6568876" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu03504.png?Expires=2147483647&Signature=BM5jLsK8mF3yLTt4i0JUssm9PA686IpZnepMJOhF57k49zGJi1AF4juMINV9llYg0XS3tThxev3k5DUgkZUeN0KscSYPvA~Scdq82gx-iX4G177dnG4ExleR2E1G6JqQsyv2eWKYLdM7~qIxUnqiMiKO6dFnuCxgtk3yr7eCc8rQs-JEbNjqiIkGDxev6KNLsxHMWXuZezuImhACDD0s8GZFeKBRA-2ZZtt8Y8Dny-3NUiZ8~zVvPPfh4hTx4BSZpUfkWdXC0jpJ2v6blQ6sOr5aNklCZPklU2AqN2MAED8qII2hK98lpo6FPmLoGk6GgewH7Wdv~GXxkv3jW1dAEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6568876&ar=439863&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03504.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of biological functions and coverage graph for monocot and dicot plants. (A) Top 20 InterPro domains among the onion genes from the combined library. (B) Coverage graph of the onion genes in the assembly of the combined library on monocot and dicot plants. The line graph and histogram illustrate the proportions of onion genes and plant proteins in each species, respectively.</p></div></div><p>To provide a more specific functional description of the onion genes, we conducted a BLAST analysis using the Uniprot database. A total of 44,710 onion genes (82.5% of the total) were discovered based on their best alignment with an <em>e</em>-value cut-off of 1<em>e</em>-5 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Moreover, using the RefSeq database, we performed additional functional annotations for the genes that were unassigned by the Uniprot database, and a total of 50,352 genes (93.0% of the total) were discovered in Uniprot or RefSeq databases and assigned biological functions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The same functional annotation was performed for the H6 and SP3B gene sets (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7 and S8</a>).</p><p>To estimate the coverage of the gene sets, we compared them to the proteins of each plant species in the RefSeq database via BLAST as an alternative approach to matching them with the whole genes of the onion. The predicted onion gene model covered from 60.5 to 68.0% of monocot proteins (<em>Oryza branchyantha</em>, <em>Brachypodium distachyon</em>, and <em>Zea mays</em>) and from 68.1 to 71.3% of dicot proteins (<em>Vitis vinifera</em>, <em>Cucumis sativus</em>, and <em>Solanum lycopersicum</em>) with cut-offs of 80% coverage and <em>e</em>-value of 1<em>e</em>-10 (Fig. <a reveal-id="DSU035F4" data-open="DSU035F4" class="link link-reveal link-table xref-fig">4</a>B).</p><p>To build up the genomic resources for molecular marker development in the onion, we performed a comparative transcriptomic analysis to detect sequence variation between H6 and SP3B. We identified a total 50,064 SNPs and 14,016 InDels between H6 and SP3B (Table <a reveal-id="DSU035TB3" data-open="DSU035TB3" class="link link-reveal link-table xref-fig">3</a>). For efficient marker development, we removed the SNPs and INDELs that did not have conserved flanking sequences in the assembly of SP3B. Consequently, a total 5,502 SNPs within protein-coding sequences and 5,942 SNPs within non-coding sequences were identified (Table <a reveal-id="DSU035TB3" data-open="DSU035TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). </p>                        <a id="6568880" scrollto-destination="6568880"></a>
<div content-id="DSU035TB3" class="table-modal"><div class="table-wrap"><div id="DSU035TB3" class="table-wrap-title" data-id="DSU035TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Sequence variation between two cultivated onions, H6, and SP3B</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Whole variation<hr /> </th><th colspan="4">Confirmed variation<sup>a</sup><hr /> </th></tr><tr><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th></tr></thead><tbody><tr><td>SNPs </td><td>9,875 </td><td>1,357 </td><td>38,832 </td><td>50,064 </td><td>5,502 </td><td>300 </td><td>5,642 </td><td>11,444 </td></tr><tr><td>INDELs </td><td>766 </td><td>834 </td><td>12,416 </td><td>14,016 </td><td>47 </td><td>19 </td><td>431 </td><td>497 </td></tr><tr><td>Total </td><td>10,641 </td><td>2,191 </td><td>51,248 </td><td>64,080 </td><td>5,549 </td><td>319 </td><td>6,073 </td><td>11,941 </td></tr></tbody></table></div><div content-id="DSU035TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Whole variation<hr /> </th><th colspan="4">Confirmed variation<sup>a</sup><hr /> </th></tr><tr><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th></tr></thead><tbody><tr><td>SNPs </td><td>9,875 </td><td>1,357 </td><td>38,832 </td><td>50,064 </td><td>5,502 </td><td>300 </td><td>5,642 </td><td>11,444 </td></tr><tr><td>INDELs </td><td>766 </td><td>834 </td><td>12,416 </td><td>14,016 </td><td>47 </td><td>19 </td><td>431 </td><td>497 </td></tr><tr><td>Total </td><td>10,641 </td><td>2,191 </td><td>51,248 </td><td>64,080 </td><td>5,549 </td><td>319 </td><td>6,073 </td><td>11,941 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN5"><div class="fn"><div class="label -label">a</div><p>Variations that have conserved flanking sequences in both assemblies.</p></div></div><div content-id="TN6"><div class="fn"><div class="label -label">b</div><p>Regions except exon and intron.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568880" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSU035TB3" class="table-wrap-title" data-id="DSU035TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Sequence variation between two cultivated onions, H6, and SP3B</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Whole variation<hr /> </th><th colspan="4">Confirmed variation<sup>a</sup><hr /> </th></tr><tr><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th></tr></thead><tbody><tr><td>SNPs </td><td>9,875 </td><td>1,357 </td><td>38,832 </td><td>50,064 </td><td>5,502 </td><td>300 </td><td>5,642 </td><td>11,444 </td></tr><tr><td>INDELs </td><td>766 </td><td>834 </td><td>12,416 </td><td>14,016 </td><td>47 </td><td>19 </td><td>431 </td><td>497 </td></tr><tr><td>Total </td><td>10,641 </td><td>2,191 </td><td>51,248 </td><td>64,080 </td><td>5,549 </td><td>319 </td><td>6,073 </td><td>11,941 </td></tr></tbody></table></div><div content-id="DSU035TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4">Whole variation<hr /> </th><th colspan="4">Confirmed variation<sup>a</sup><hr /> </th></tr><tr><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th><th>Exon </th><th>Intron </th><th>Others<sup>b</sup> </th><th>Sum </th></tr></thead><tbody><tr><td>SNPs </td><td>9,875 </td><td>1,357 </td><td>38,832 </td><td>50,064 </td><td>5,502 </td><td>300 </td><td>5,642 </td><td>11,444 </td></tr><tr><td>INDELs </td><td>766 </td><td>834 </td><td>12,416 </td><td>14,016 </td><td>47 </td><td>19 </td><td>431 </td><td>497 </td></tr><tr><td>Total </td><td>10,641 </td><td>2,191 </td><td>51,248 </td><td>64,080 </td><td>5,549 </td><td>319 </td><td>6,073 </td><td>11,941 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN5"><div class="fn"><div class="label -label">a</div><p>Variations that have conserved flanking sequences in both assemblies.</p></div></div><div content-id="TN6"><div class="fn"><div class="label -label">b</div><p>Regions except exon and intron.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6568880" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=6568881 id="6568881" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Since next-generation sequencing technology was developed, a large number of genomes have been sequenced from various organisms. Plants with very large genome sizes have not been fully sequenced because of features of the next-generation sequencing technology based on short reads. Because of the huge genome size of <em>Allium</em> spp., little genomic information has been deposited, creating an obstacle for the construction of an efficient molecular breeding platform. The emergence of high-throughput transcriptome sequencing offers an attractive solution for genomic studies of non-model organisms including the onion. Despite the advantages of transcriptome sequencing in terms of cost savings and reduced computing requirements, non-coding sequences in <em>de novo</em> transcriptomes have disrupted the construction of accurate genomic resources such as protein-coding gene libraries.<sup><a reveal-id="DSU035C25" data-open="DSU035C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Therefore, in-depth gene prediction for <em>de novo</em> transcriptome assembly is required to construct a fundamental reference gene set as an alternative genomic resource.</p><p>Typically, the six-frame translation approach has been used to extract protein-coding genes from <em>de novo</em> transcriptome assemblies. Six-frame translation is based, however, on the concept that assembled transcripts are composed of purified coding sequences, despite the presence of non-coding sequences. We generated <em>de novo</em> transcriptome assemblies with annotated gene sets of two onion accessions using ISGAP, an automated pipeline, with intensive validation. We focused our attention on constructing ISGAP to optimize accurate gene prediction for <em>de novo</em> transcriptome assembly with multiple gene-prediction steps (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A). As the key steps of ISGAP, protein alignment was integrated to detect accurate exon regions, and an additional translation step was applied to extend the partial genes based on the detected exon regions (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A). In addition, gene prediction using Augustus with a training step was integrated to obtain genes for transcript regions without matching reference proteins (Fig. <a reveal-id="DSU035F1" data-open="DSU035F1" class="link link-reveal link-table xref-fig">1</a>A).</p><p>ISGAP predicted more genes accurately than six-frame translation did (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>), and it produced more specifically accurate reference genes (Fig. <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a>). The differences between the gene sets predicted by ISGAP and six-frame translation were mainly caused by the existence or non-existence of introns within the assembled transcripts (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 4</a>A). Due to the retained intron sequences, the correct annotation by six-frame translation was interrupted, leading to miss-annotation or the construction of partial genes. For genes containing multiple exons, ISGAP could recognize the intron sequences and merge the exons by removing the introns, despite the protein-coding genes being broken into at least two exons separated by intron sequences in the assembled transcripts (Fig. <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>A and B). Six-frame translation could not correctly annotate the genes containing multiple exons (Figs <a reveal-id="DSU035F2" data-open="DSU035F2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>A and B). In addition, for the single-exon genes, six-frame translation often extracted miss-annotated genes by translation with an inappropriate region or strand (Fig. <a reveal-id="DSU035F3" data-open="DSU035F3" class="link link-reveal link-table xref-fig">3</a>C and D). The protein-coding genes in the transcriptome need to be extracted by the consideration of the gene structure, and ISGAP successfully performed annotation based on reference proteins and <em>ab initio</em> gene models.</p><p>As shown in Fig. <a reveal-id="DSU035F4" data-open="DSU035F4" class="link link-reveal link-table xref-fig">4</a>B, even if accurate comparison between the genes predicted by ISGAP and the plant proteins was difficult due to phylogenetic distance between the onion and the other plants, over 60% of the genes from each plant species were covered by half of the genes predicted by ISGAP with over 80% query coverage, suggesting that the gene set represented a large portion of the genes in the onion genome, although it was not enough to cover all of the genes in the onion. Nevertheless, transcriptome sequencing is still valuable for non-model organisms, owing to the absence of better alternative resources. The gene set predicted by ISGAP will contribute to the construction of a molecular breeding platform for the onion to boost the development of important traits such as shape, colour, size, pungency, and resistance to diseases. In conclusion, we expect that: (i) our comprehensive approach for accurate gene prediction using <em>de novo</em> transcriptome sequencing will provide a standard method for <em>de novo</em> transcriptome assembly in non-model organisms; (ii) the newly annotated gene set of the onion including various genomic resources such as SNPs and InDels will support molecular breeding and gene cloning in <em>Allium</em> spp.</p>                            <h2 scrollto-destination=6568886 id="6568886" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>The whole-transcriptome sequences of two cultivated onion are deposited in the GenBank database under SRP041918. The assembled transcripts of combined, H6, and SP3B libraries are deposited under the accession GBRQ01000000, GBRO01000000, and GBRN01000000, respectively. The version described in this paper is the first version. Further information, containing transcriptome assembly and annotated genes are uploaded on our website at <a class="link link-uri" href="http://onion.snu.ac.kr">http://onion.snu.ac.kr</a>.</p>                            <h2 scrollto-destination=6568888 id="6568888" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6568890 id="6568890" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the Agricultural Genome Center of the Next-Generation Biogreen 21 Program (PJ008199012012), Rural Development Administration of the Korean government, and a grant (PJ710001-03) from Vegetable Breeding Research Center, an ARC program of Ministry for Food, Agriculture, Forestry and Fisheries, Republic of Korea. Funding to pay the Open Access publication charges for this article was provided by the Agricultural Genome Center of the Next-Generation Biogreen 21 Program (PJ008199012012), Rural Development Administration of the Korean government, and a grant (PJ710001-03) from Vegetable Breeding Research Center, an ARC program of Ministry for Food, Agriculture, Forestry and Fisheries, Republic of Korea.</p>                            <h2 scrollto-destination=6568892 id="6568892" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU035C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brewster</div> <div class="given-names">J.L</div></div>. </div>, <div class="source">Onions and other vegetable alliums</div> , <div class="year">2008</div><div class="publisher-loc">Wallingford, UK; Cambridge, MA</div><div class="publisher-name">CABI</div></div></div></div></div><div content-id="DSU035C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slimestad</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fossen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vagen</div> <div class="given-names">I.M.</div></div>. </div><div class="article-title">Onions: a source of unique dietary flavonoids</div>, <div class="source">J. Agric. Food Chem.</div> , <div class="year">2007</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">10067</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Onions%3A%20a%20source%20of%20unique%20dietary%20flavonoids&amp;author=R.%20Slimestad&amp;author=T.%20Fossen&amp;author=I.M.%20Vagen&amp;publication_year=2007&amp;journal=J.%20Agric.%20Food%20Chem.&amp;volume=55&amp;pages=10067-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf0712503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf0712503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf0712503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17997520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Meyers</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">van der Heide</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">R.H.</div></div>. </div><div class="article-title">Varietal differences in phenolic content and antioxidant and antiproliferative activities of onions</div>, <div class="source">J. Agric. Food Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">6787</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Varietal%20differences%20in%20phenolic%20content%20and%20antioxidant%20and%20antiproliferative%20activities%20of%20onions&amp;author=J.%20Yang&amp;author=K.J.%20Meyers&amp;author=J.%20van%20der%20Heide&amp;author=R.H.%20Liu&amp;publication_year=2004&amp;journal=J.%20Agric.%20Food%20Chem.&amp;volume=52&amp;pages=6787-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf0307144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf0307144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf0307144"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15506817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bahorun</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Luximon-Ramma</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Crozier</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Aruoma</div> <div class="given-names">O.I.</div></div>. </div><div class="article-title">Total phenol, flavonoid, proanthocyanidin and vitamin C levels and antioxidant activities of Mauritian vegetables</div>, <div class="source">J. Sci. Food AGR</div> , <div class="year">2004</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">1553</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Total%20phenol%2C%20flavonoid%2C%20proanthocyanidin%20and%20vitamin%20C%20levels%20and%20antioxidant%20activities%20of%20Mauritian%20vegetables&amp;author=T.%20Bahorun&amp;author=A.%20Luximon-Ramma&amp;author=A.%20Crozier&amp;author=O.I.%20Aruoma&amp;publication_year=2004&amp;journal=J.%20Sci.%20Food%20AGR&amp;volume=84&amp;pages=1553-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0010"> </span></div> </div></div></div></div></div><div content-id="DSU035C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duangjit</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bohanec</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Chan</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Town</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Havey</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Transcriptome sequencing to produce SNP-based genetic maps of onion</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">2093</div>-<div class="lpage">101</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20to%20produce%20SNP-based%20genetic%20maps%20of%20onion&amp;author=J.%20Duangjit&amp;author=B.%20Bohanec&amp;author=A.P.%20Chan&amp;author=C.D.%20Town&amp;author=M.J.%20Havey&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=2093-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2121-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2121-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2121-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23689743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagarajan</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Sequence assembly demystified</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">157</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20assembly%20demystified&amp;author=N.%20Nagarajan&amp;author=M.%20Pop&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=14&amp;pages=157-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23358380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schatz</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Witkowski</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">McCombie</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Current challenges in de novo plant genome sequencing and assembly</div>, <div class="source">Genome Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">243</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20challenges%20in%20de%20novo%20plant%20genome%20sequencing%20and%20assembly&amp;author=M.C.%20Schatz&amp;author=J.%20Witkowski&amp;author=W.R.%20McCombie&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-4-243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-4-243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-4-243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22546054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yandell</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ence</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A beginner's guide to eukaryotic genome annotation</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">329</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20beginner's%20guide%20to%20eukaryotic%20genome%20annotation&amp;author=M.%20Yandell&amp;author=D.%20Ence&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=13&amp;pages=329-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22510764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groenen</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Archibald</div> <div class="given-names">A.L.</div></div>,  <div class="name"><div class="surname">Uenishi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Analyses of pig genomes provide insight into porcine demography and evolution</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">491</div> (pg. <div class="fpage">393</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyses%20of%20pig%20genomes%20provide%20insight%20into%20porcine%20demography%20and%20evolution&amp;author=M.A.%20Groenen&amp;author=A.L.%20Archibald&amp;author=H.%20Uenishi&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=393-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23151582" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fei</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">K.</div></div>. </div><div class="article-title">RNA-Seq improves annotation of protein-coding genes in the cucumber genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">540</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-Seq%20improves%20annotation%20of%20protein-coding%20genes%20in%20the%20cucumber%20genome&amp;author=Z.%20Li&amp;author=Z.%20Zhang&amp;author=P.%20Yan&amp;author=S.%20Huang&amp;author=Z.%20Fei&amp;author=K.%20Lin&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=540" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22047402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garber</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Grabherr</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Computational methods for transcriptome annotation and quantification using RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="year">2011</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">469</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20methods%20for%20transcriptome%20annotation%20and%20quantification%20using%20RNA-seq&amp;author=M.%20Garber&amp;author=M.G.%20Grabherr&amp;author=M.%20Guttman&amp;author=C.%20Trapnell&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=469-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21623353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, et al. </div><div class="article-title">The genome of the cucumber, <em>Cucumis sativus</em> L.</div>, <div class="source">Nat. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1275</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20cucumber%2C%20Cucumis%20sativus%20L.&amp;author=S.%20Huang&amp;author=R.%20Li&amp;author=Z.%20Zhang&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1275-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Denoeud</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Da Silva</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Annotating genomes with massive-scale RNA sequencing</div>, <div class="source">Genome Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">R175</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Annotating%20genomes%20with%20massive-scale%20RNA%20sequencing&amp;author=F.%20Denoeud&amp;author=J.M.%20Aury&amp;author=C.%20Da%20Silva&amp;publication_year=2008&amp;journal=Genome%20Biol.&amp;volume=9&amp;pages=R175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-12-r175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-12-r175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-12-r175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19087247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Lei</div> <div class="given-names">J.</div></div>. </div><div class="article-title">De novo transcriptome assembly in chili pepper (<em>Capsicum frutescens</em>) to identify genes involved in the biosynthesis of capsaicinoids</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e48156</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20assembly%20in%20chili%20pepper%20(Capsicum%20frutescens)%20to%20identify%20genes%20involved%20in%20the%20biosynthesis%20of%20capsaicinoids&amp;author=S.%20Liu&amp;author=W.%20Li&amp;author=Y.%20Wu&amp;author=C.%20Chen&amp;author=J.%20Lei&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e48156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0048156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0048156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0048156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23349661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozsolak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Milos</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">RNA sequencing: advances, challenges and opportunities</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">87</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequencing%3A%20advances%2C%20challenges%20and%20opportunities&amp;author=F.%20Ozsolak&amp;author=P.M.%20Milos&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=87-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21191423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martin</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Next-generation transcriptome assembly</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">671</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20transcriptome%20assembly&amp;author=J.A.%20Martin&amp;author=Z.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=671-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21897427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dubey</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Farmer</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Defining the transcriptome assembly and its use for genome dynamics and transcriptome profiling studies in pigeonpea (<em>Cajanus cajan</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">153</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Defining%20the%20transcriptome%20assembly%20and%20its%20use%20for%20genome%20dynamics%20and%20transcriptome%20profiling%20studies%20in%20pigeonpea%20(Cajanus%20cajan%20L.)&amp;author=A.%20Dubey&amp;author=A.%20Farmer&amp;author=J.%20Schlueter&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=153-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21565938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Function annotation of the rice transcriptome at single-nucleotide resolution by RNA-seq</div>, <div class="source">Genome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1238</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Function%20annotation%20of%20the%20rice%20transcriptome%20at%20single-nucleotide%20resolution%20by%20RNA-seq&amp;author=T.%20Lu&amp;author=G.%20Lu&amp;author=D.%20Fan&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1238-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.106120.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.106120.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.106120.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20627892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Looso</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Preussner</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sousounis</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A de novo assembly of the newt transcriptome combined with proteomic validation identifies new protein families expressed during tissue regeneration</div>, <div class="source">Genome Biol.</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">R16</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20de%20novo%20assembly%20of%20the%20newt%20transcriptome%20combined%20with%20proteomic%20validation%20identifies%20new%20protein%20families%20expressed%20during%20tissue%20regeneration&amp;author=M.%20Looso&amp;author=J.%20Preussner&amp;author=K.%20Sousounis&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-2-r16" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-2-r16" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-2-r16"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23425577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Meng</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>. </div><div class="article-title">De novo assembly and characterization of the garlic (<em>Allium sativum</em>) bud transcriptome by Illumina sequencing</div>, <div class="source">Plant Cell Rep.</div> , <div class="year">2012</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">1823</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20and%20characterization%20of%20the%20garlic%20(Allium%20sativum)%20bud%20transcriptome%20by%20Illumina%20sequencing&amp;author=X.%20Sun&amp;author=S.%20Zhou&amp;author=F.%20Meng&amp;author=S.%20Liu&amp;publication_year=2012&amp;journal=Plant%20Cell%20Rep.&amp;volume=31&amp;pages=1823-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-012-1295-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-012-1295-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-012-1295-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22684307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashrafi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hill</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">De novo assembly of the pepper transcriptome (<em>Capsicum annuum</em>): a benchmark for in silico discovery of SNPs, SSRs and candidate genes</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">571</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20pepper%20transcriptome%20(Capsicum%20annuum)%3A%20a%20benchmark%20for%20in%20silico%20discovery%20of%20SNPs%2C%20SSRs%20and%20candidate%20genes&amp;author=H.%20Ashrafi&amp;author=T.%20Hill&amp;author=K.%20Stoffel&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=571" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-571" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-571" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-571"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23110314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Annadurai</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Jayakumar</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Mugasimangalam</div> <div class="given-names">R.C.</div></div>, et al. </div><div class="article-title">Next generation sequencing and de novo transcriptome analysis of <em>Costus pictus</em> D. Don, a non-model plant with potent anti-diabetic properties</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">663</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20and%20de%20novo%20transcriptome%20analysis%20of%20Costus%20pictus%20D.%20Don%2C%20a%20non-model%20plant%20with%20potent%20anti-diabetic%20properties&amp;author=R.S.%20Annadurai&amp;author=V.%20Jayakumar&amp;author=R.C.%20Mugasimangalam&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=663" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23176672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Franssen</div> <div class="given-names">S.U.</div></div>,  <div class="name"><div class="surname">Shrestha</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Brautigam</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bornberg-Bauer</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Weber</div> <div class="given-names">A.P.</div></div>. </div><div class="article-title">Comprehensive transcriptome analysis of the highly complex <em>Pisum sativum</em> genome using next generation sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">227</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20transcriptome%20analysis%20of%20the%20highly%20complex%20Pisum%20sativum%20genome%20using%20next%20generation%20sequencing&amp;author=S.U.%20Franssen&amp;author=R.P.%20Shrestha&amp;author=A.%20Brautigam&amp;author=E.%20Bornberg-Bauer&amp;author=A.P.%20Weber&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=227" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Goff</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Long noncoding RNAs regulate adipogenesis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2013</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">3387</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20noncoding%20RNAs%20regulate%20adipogenesis&amp;author=L.%20Sun&amp;author=L.A.%20Goff&amp;author=C.%20Trapnell&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=3387-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1222643110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1222643110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1222643110"> </span></div> </div></div></div></div></div><div content-id="DSU035C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steijger</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abril</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Engstrom</div> <div class="given-names">P.G.</div></div>, et al. </div><div class="article-title">Assessment of transcript reconstruction methods for RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="year">2013</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1177</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T.%20Steijger&amp;author=J.F.%20Abril&amp;author=P.G.%20Engstrom&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1177-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kretz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Siprashvili</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Control of somatic tissue differentiation by the long non-coding RNA TINCR</div>, <div class="source">Nature</div> , <div class="year">2013</div>, vol. <div class="volume">493</div> (pg. <div class="fpage">231</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20somatic%20tissue%20differentiation%20by%20the%20long%20non-coding%20RNA%20TINCR&amp;author=M.%20Kretz&amp;author=Z.%20Siprashvili&amp;author=C.%20Chu&amp;publication_year=2013&amp;journal=Nature&amp;volume=493&amp;pages=231-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23201690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schunter</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Vollmer</div> <div class="given-names">S.V.</div></div>,  <div class="name"><div class="surname">Macpherson</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pascual</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transcriptome analyses and differential gene expression in a non-model fish species with alternative mating tactics</div>, <div class="source">BMC Genomics</div> , <div class="year">2014</div>, vol. <div class="volume">15</div> pg. <div class="fpage">167</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analyses%20and%20differential%20gene%20expression%20in%20a%20non-model%20fish%20species%20with%20alternative%20mating%20tactics&amp;author=C.%20Schunter&amp;author=S.V.%20Vollmer&amp;author=E.%20Macpherson&amp;author=M.%20Pascual&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24581002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Han</div> <div class="given-names">X.J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.D.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">L.Y.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">Q.K.</div></div>. </div><div class="article-title">Transcriptome sequencing and expression analysis of terpenoid biosynthesis genes in <em>Litsea cubeba</em></div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e76890</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20and%20expression%20analysis%20of%20terpenoid%20biosynthesis%20genes%20in%20Litsea%20cubeba&amp;author=X.J.%20Han&amp;author=Y.D.%20Wang&amp;author=Y.C.%20Chen&amp;author=L.Y.%20Lin&amp;author=Q.K.%20Wu&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e76890" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0076890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0076890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0076890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24130803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gross</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Abraham-Juarez</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Visel</div> <div class="given-names">A.</div></div>. </div><div class="article-title">De novo transcriptome assembly of drought tolerant CAM plants, <em>Agave deserti</em> and <em>Agave tequilana</em></div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">563</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20assembly%20of%20drought%20tolerant%20CAM%20plants%2C%20Agave%20deserti%20and%20Agave%20tequilana&amp;author=S.M.%20Gross&amp;author=J.A.%20Martin&amp;author=J.%20Simpson&amp;author=M.J.%20Abraham-Juarez&amp;author=Z.%20Wang&amp;author=A.%20Visel&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23957668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Han</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Transcript assembly and quantification by RNA-Seq reveals differentially expressed genes between soft-endocarp and hard-endocarp hawthorns</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e72910</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20differentially%20expressed%20genes%20between%20soft-endocarp%20and%20hard-endocarp%20hawthorns&amp;author=H.%20Dai&amp;author=G.%20Han&amp;author=Y.%20Yan&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e72910" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0072910" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0072910" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0072910"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24039819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of <em>Litopenaeus vannamei</em> in response to white spot syndrome virus infection</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e73218</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20Litopenaeus%20vannamei%20in%20response%20to%20white%20spot%20syndrome%20virus%20infection&amp;author=X.%20Chen&amp;author=D.%20Zeng&amp;author=X.%20Chen&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e73218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0073218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0073218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0073218"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23991181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zha</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Bu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Sequencing and de novo assembly of the Asian clam (<em>Corbicula fluminea</em>) transcriptome using the Illumina GAIIx method</div>, <div class="source">PLoS ONE</div> , <div class="year">2013</div>, vol. <div class="volume">8</div> pg. <div class="fpage">e79516</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20de%20novo%20assembly%20of%20the%20Asian%20clam%20(Corbicula%20fluminea)%20transcriptome%20using%20the%20Illumina%20GAIIx%20method&amp;author=H.%20Chen&amp;author=J.%20Zha&amp;author=X.%20Liang&amp;author=J.%20Bu&amp;author=M.%20Wang&amp;author=Z.%20Wang&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e79516" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0079516" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0079516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0079516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24244519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crawford</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Guelbeogo</div> <div class="given-names">W.M.</div></div>,  <div class="name"><div class="surname">Sanou</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">De novo transcriptome sequencing in <em>Anopheles funestus</em> using Illumina RNA-seq technology</div>, <div class="source">PLoS ONE</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e14202</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20sequencing%20in%20Anopheles%20funestus%20using%20Illumina%20RNA-seq%20technology&amp;author=J.E.%20Crawford&amp;author=W.M.%20Guelbeogo&amp;author=A.%20Sanou&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e14202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0014202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0014202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0014202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhong</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Joung</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">High-throughput Illumina strand-specific RNA sequencing library preparation</div>, <div class="source">Cold Spring Harb. Prot.</div> , <div class="year">2011</div><div class="pub-id-doi"><a href="https://doi.org/10.1101/pdb.prot5652" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/pdb.prot5652</a></div></div></div></div></div><div content-id="DSU035C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yeom</div> <div class="given-names">S.I.</div></div>, et al. </div><div class="article-title">Genome sequence of the hot pepper provides insights into the evolution of pungency in Capsicum species</div>, <div class="source">Nat. Genet.</div> , <div class="year">2014</div>, vol. <div class="volume">46</div> pg. <div class="fpage">270</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20hot%20pepper%20provides%20insights%20into%20the%20evolution%20of%20pungency%20in%20Capsicum%20species&amp;author=S.%20Kim&amp;author=M.%20Park&amp;author=S.I.%20Yeom&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=270" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24441736" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="year">2012</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">357</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kopylova</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Noe</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Touzet</div> <div class="given-names">H.</div></div>. </div><div class="article-title">SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">3211</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SortMeRNA%3A%20fast%20and%20accurate%20filtering%20of%20ribosomal%20RNAs%20in%20metatranscriptomic%20data&amp;author=E.%20Kopylova&amp;author=L.%20Noe&amp;author=H.%20Touzet&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3211-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23071270" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schulz</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Vingron</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">1086</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oases%3A%20robust%20de%20novo%20RNA-seq%20assembly%20across%20the%20dynamic%20range%20of%20expression%20levels&amp;author=M.H.%20Schulz&amp;author=D.R.%20Zerbino&amp;author=M.%20Vingron&amp;author=E.%20Birney&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1086-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22368243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">D'Hont</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Denoeud</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Aury</div> <div class="given-names">J.M.</div></div>, et al. </div><div class="article-title">The banana (<em>Musa acuminata</em>) genome and the evolution of monocotyledonous plants</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">488</div> (pg. <div class="fpage">213</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20banana%20(Musa%20acuminata)%20genome%20and%20the%20evolution%20of%20monocotyledonous%20plants&amp;author=A.%20D'Hont&amp;author=F.%20Denoeud&amp;author=J.M.%20Aury&amp;publication_year=2012&amp;journal=Nature&amp;volume=488&amp;pages=213-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22801500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohyanagi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The Rice Annotation Project Database (RAP-DB): hub for <em>Oryza sativa</em> ssp. japonica genome information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">D741</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Rice%20Annotation%20Project%20Database%20(RAP-DB)%3A%20hub%20for%20Oryza%20sativa%20ssp.%20japonica%20genome%20information&amp;author=H.%20Ohyanagi&amp;author=T.%20Tanaka&amp;author=H.%20Sakai&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D741-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj094" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj094" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj094"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">The draft genome of the fast-growing non-timber forest species moso bamboo (<em>Phyllostachys heterocycla</em>)</div>, <div class="source">Nat. Genet.</div> , <div class="year">2013</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">456</div>-<div class="lpage">61</div>) <div class="comment">461e451–2</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20the%20fast-growing%20non-timber%20forest%20species%20moso%20bamboo%20(Phyllostachys%20heterocycla)&amp;author=Z.%20Peng&amp;author=Y.%20Lu&amp;author=L.%20Li&amp;publication_year=2013&amp;journal=Nat.%20Genet.&amp;volume=45&amp;pages=456-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2569" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2569" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2569"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23435089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Bowers</div> <div class="given-names">J.E.</div></div>,  <div class="name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">457</div> (pg. <div class="fpage">551</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.H.%20Paterson&amp;author=J.E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="collab">International Brachypodium, I.</div><div class="article-title">Genome sequencing and analysis of the model grass <em>Brachypodium distachyon</em></div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">763</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08747" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08747" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08747"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20148030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slater</div> <div class="given-names">G.S.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">31</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.S.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Keller</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Gunduz</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Hayes</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Waack</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>. </div><div class="article-title">AUGUSTUS: ab initio prediction of alternative transcripts</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W435</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%3A%20ab%20initio%20prediction%20of%20alternative%20transcripts&amp;author=M.%20Stanke&amp;author=O.%20Keller&amp;author=I.%20Gunduz&amp;author=A.%20Hayes&amp;author=S.%20Waack&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W435-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Chang</div> <div class="given-names">H.Y.</div></div>, et al. </div><div class="article-title">InterProScan 5: genome-scale protein function classification</div>, <div class="source">Bioinformatics</div> , <div class="year">2014</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1236</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">G.R.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI Reference Sequences (RefSeq): current status, new features and genome annotation policy</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">D130</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequences%20(RefSeq)%3A%20current%20status%2C%20new%20features%20and%20genome%20annotation%20policy&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=G.R.%20Brown&amp;author=D.R.%20Maglott&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D130-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22121212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="collab">UniProt, C</div><div class="article-title">Update on activities at the Universal Protein Resource (UniProt) in 2013</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2013</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">D43</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1068"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161681" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2078</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU035C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU035C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cingolani</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Platts</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wang le</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <em>Drosophila melanogaster</em> strain w1118; iso-2; iso-3</div>, <div class="source">Fly</div> , <div class="year">2012</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">80</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;author=P.%20Cingolani&amp;author=A.%20Platts&amp;author=L.%20Wang%20le&amp;publication_year=2012&amp;journal=Fly&amp;volume=6&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/fly.19695" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Ffly.19695" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Ffly.19695"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22728672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=439863&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table9.xls?Expires=2147483647&amp;Signature=IGRMIyV8wDBORAi~fjl2tktqCZUcoXJpRPNEHj4FUqkbhbJZU7kxgHWHF1x3UvQhA6XqzzifUrlmpET5SBJlmWwOoWFyX7MXkcxSebHLg69-14qmnQUARJPhZ042eo2~~d09HMm5BwOT~~XMKLbv7A0OTXbOAZLhPtZrqX~BCU4iiqbYgl7Hk2cChGv4PwKhEacnmhVpx-Kodv0yRBbm6Igq~n8txuYX-taFT93ICA-QFSjwExubvxnp~DIff5TlwsBkYPsixe8kMLADdc5Mba6iQA9apgrF0kH112h4YNEvTbUo-Bzf8d5BOpcyk-3tGHbo9S5WZm4GUjQ7uVKnNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table8.xls?Expires=2147483647&amp;Signature=oeV7CvjdSjRusyNRf8TMlGEZbJ-ZJFEh4wofjdafqYPmjiHu8oeIHdWw6Cmp1GxBg63GQW5bLbojlxhac6QWrYzQ99cFxopPnP6WgHKHHKIqMK09TOWGly~1sz1HIl5m3z7rEL8kG5jBbylv9cKsZ2Q1ynrVZWIFMijtM4YN8wc3mySH476Dhp99EuK-PNsFAIaaSSELj8RQd2Bpz3~TNCLxB2Ueth9XMsRL-5rvTp~BRkfrHBMralHzSX86mWQHSVtEF5xxOtAJbH4Dow7VmX7QYA95dox54CXnP-Gt0adtj7Wm5RB~UNeN8rAMy8MR0oJLN3RvbPiiyiko3lamVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table7.xls?Expires=2147483647&amp;Signature=fHg53SyF-JqLPe11G-Gu2CxScH5~kdqxme894iqfKFnPC6TPH2bpTpTLQilIBoCdbDd4xhPMrqO7mmuyaCr-MJv6R~STOmuzf0BdX19IjLDZN0HsfNxLZCQbhQ-ZXvH94hwDfC0HQ2EIk~30xkEXqqcl7rhBLLaoAV-zMM30nViuCwT9LQw12qbqbpsX6p5sz-8~ZwJ7JcY-I6SxsKfIkmU67Alq2FJ6h0uS6MHvqB8y30TAO0N~iVpiZBtOJle0k8s3fkFvTlTNa3S9YmDoYIyGkiw9cn84-fclMYgwyzibhN3AKBGLoeE60r7km~7gi8Pb~L1LP~W~xOjKo7C-Ow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table6.xls?Expires=2147483647&amp;Signature=sjBwi5N-Gv8ZS4ddPBkA8KJ9pYGoXVY~7Of3BRXSA3dPfCfoP~lhrsLBE1xgZH-wnXCkWnFqMftWJdWFLrL7GXX3Cj5iqyNdhMmzf3b3xdvBFRpwPPlwNXU0VoRXeNKG0J0iV-b3H5rzY8wVyiIU7kG4Vq0-00-K4dt-~2fC35HFRRJrt9F-TY7RfPfwerkrzXtg0BDZfOIkjAJ8rkjQgro1RO3BT5MOZGpt-Gm9wgBX9iaF-JDp2vp9DuQ6KMvD58hsT0bjOIEcxwW-VRNAi6Q96tUpljvgg5QL2JfsTdDmLeMcxA6ReaQ-hhxbyVwORDPBoET0DCsuCDhm0~9oCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table5.pdf?Expires=2147483647&amp;Signature=M4FUoNhiu7ufdAfUe3lD41HjlVl-G0IOddDVENABLPeLdE9aogkF7SN0Vk2agj-wtXuRPdgaWn8SID~~k5fIc3-hCD-1BfbD1DcvvDThmA0quZMAHocfyJhAxnOWysULpTcRYal4G-r5BMiSWHClmX0ZG7FOeORkDRGwDgvRIxQSBsrl8zyfYBuodK~R5gVRhZpHLA8QYwwBoFqRNhIouZrZaUQMBQBqK0LzgM4OF1fxEC8EkBLBADUTM3XcZhgm~O14RnkAYDaoLhjLUpvPCq0M3f1zVPX9JdY0W9dhesiOplTWeVsusu4uCTe4BxAe3kqdPfft9dTtAASKsMfVkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table4.pdf?Expires=2147483647&amp;Signature=v3o3t5q5-8Bo-Kd7IcDLJi2I7ZIg0gI5RDs3prs31~Q8aBJ9~b-vjQixCmLv4ct0IIHRUY~OjiPN3p2FhI8MiHzgPCYi70QhWDH6sbgO1Q8LAL-Zf1A7rKtRaEIAjimuG9lSIEtYIREYIMgstj6X8ZZhf7QRP~ZosGyE2JhINMFQbogA9JV2S7MXqVDtUjPrIxSWtAnT-xbJV4gmHd132CubWKT8i60vrcvfaEpis0bcAMRnQvsTFsetMux38Nv5cadPvT~PcBFCALg5VYsKvsjFUuSAmQKtcDyWTqzuZs7v3z7TXZuu8P2UgbG2MkdBqpZJ9BU1RNpUgsPJ3n9OMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table3.pdf?Expires=2147483647&amp;Signature=iZPpHzBbd4kuOpYhUTSFhRQoDTi~CJeje7H3cu4BkQB9r--rCOy3Clr-ZLmiYc1lKeGdVZ1pxFSoHhu7r0dn6CAdC1MZ6FjKEjU~E-wrfzW1Tc6nJ3P06OKcI6Cg2Qwsjbg4KkhlWD9tvW46VbB1N6R3z-Taie3lNUL3p5psrqccWbHgRWva5PxeIXb0n8c-bpK7baDumT6D-BsNVT8OGIaus-FBrYvBN5cTSyFjiXZ24l4JZMMl1O9dFsO7lxPbMZqmSGdaln7cZyfbuXvoSSKDNgzZYpWYdqWUuC6sIYcDoUXuvjfAXO0SHf~dCuebl2MMqAVPGE-GGwIpR4Plxw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table2.pdf?Expires=2147483647&amp;Signature=YcCSfXMVxfmiVgVEv0A8pZLHhstf6O9KrkQkBidQgsOZ~-0eTRbcry5cUeBjy0sKdO3kAGnqv6O852D7d-drZbaU--b9GbUK6BIF7p67T0EBOImWq85DAF6WNSdtkE0F1lIpnOro2ihsIuDo6vu3eTCMLwn~IPNv7HoFZX~spxpu-taculu-SQIBrW9YdzU-qnTQpQyFmKGHoojBzSeS9k9tY2CyEtzKzeEAVGdL7P8Zca7j8df9dIV2Hy15a-gc4Zpl~hJqoj8UahCETIzvTWfvethIJF76-HS7BtJzsq5f2eV8oxPIOBsMf27CLxmD42OL3JxjQPzP2joNhbIgiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_table1.pdf?Expires=2147483647&amp;Signature=jN6D43rku~XNngTklBBb5bDWzRM4eCm3TiM9mivM7yxk~BugrtdITJ4QGvIr50asRYwZtLkh~lIPlVSaORs1hDlmxsClP52-sFMEEYj9jl7Umeos6Ble16Ce55mLlo4p9ngVc1tpva6gzsnNe9aOU0B4gUXt8QIP7lc2aIUxEA6hlXzz9VKrxNcY1XaulA4j5DBeKrrlCL7~5CrHi0zUHc8qGbjY9IJIXCSYfMgU5UecCQ~7Dowlm-iNPGqXxyMXu69O5M4hi9FJYIeksu7MVIgMLSRbXCGbDS4DNJ8a7FBGyNVeCYDldE4eZVPe0HOBxFfyHLXK2nVEn1xRSylgoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_figure4.pdf?Expires=2147483647&amp;Signature=ek5QT2YVp68A1mdPm5~~dm9eOYp6DI5PiSbWnZ3acD46GiVwFa1Mc6FoacNQmoEAuHxGYmzYnYv1wlGfRvsBDlCjN4ATYH5QK8D2uWc5xAVhNJpLUNKPwUdIH7tda8m3hb-jC0kSXTcpQibWDUG7oYDQ~fW5h0R-bXRvnIkoh0FjHlYT3WfzKya0uirqXlZn8Dab1UhEuDMMLCLzs351Q7PB9UPlE-oMelNxkYzfiRAdw4v2BxEXriVIt~sekDkkH5z2JTwZEG2ulYDF0wmitxZ6lg-to8-P2AoJeUJHhriLERSnJvujaEFH2SJdBKwtxNiXh6e~gT4QnTQpVNGY8Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_figure3.pdf?Expires=2147483647&amp;Signature=ps9XimASpd9m109gsm4EBv0uDb--ETh~1QogEIzsj8XDow2C03gCIIDfGHwlyH6AZw4T-dnZvBivB6JRMfuFfsgHfh2oYKEqJsl-47SxrqrpILH5uuh5R7a9yQGiRmHD8YQJMdiYEWHwpIWyM96o~L0JuKqMyEn9xloVbOBfVt~5omRwuzQkOGw0RSq2WRIKiya9~DVGqKcm9yjlkRvq5gnJs44UaDal8LhMQiR4akuzu-PaJZDZA2upSs~OZarvAgKfHM1QWbc-9r5GQb-mXz9uNV4AuJHAHZqnFOk8L5jUZil5ErhnZQopK2Sny75y-vqaPYW15H4wRCkRbUEGrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_figure2.pdf?Expires=2147483647&amp;Signature=1nok3RMfF661247CT6TyJG-Y9mfFyAs9et8WIghNm0eFRHj6V25~MejZ053ziSxmqg5N7yfFyZnRcgatBklpmiijNYoDjVRa52qWzj9LsHaoWfK20mu17I~TwzIwB05OVLkGrGwrcGJxNoAwvU0LEsOOpEzW5vRVzT3~2cGlbhQfqIX8H-oe9xpvTTtUNfFe9vnCkd7BWoUk2805mlB43skEeZd6Y67rmmTsL4kF9-wLWfbXycUAV1JgdR8YA53N7tx0v0-bYSudEAXOiXQcVnn09sVy3mLjykQYpx0gu1TYXrkwbYTZI-lJApCe5LrY6n0VZB4yTDoEY7XvV3M8hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu035/2/dsu035-dsu035supp_figure1.pdf?Expires=2147483647&amp;Signature=kce2~5nKqAnddiKjKxpoIWczcIQovGFXI74mmK6dqQYo9sfm68Budi51bNEHAokQ4aWaOQ6a9LViIkd55f0cudO~BNILTvnJwLzTDcLXuViAsN6uSQO7XfLRngHDsu8lskU2Ivft1EfCBMIsgMUvoHHsqnj~1zQO2PvPWor~LXUQ~mV5SLDIGzKBVoeD3ja6P0Qh4maVb2R3pn0tJiTYRWulyxoxbTQ7ZFRpkO8SHuZISuRbfdYwyAn6cJjdxfs94ixzfaxINj87c6stoXedWNwD7yN59sTa4bDACj5Pk5GkCaR99QgRCfv6RXdrz6JvBxo~mtCqbTpI--yQLtXVbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">562</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">16</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu035"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="439863"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f19%2f439863">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (16)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F19%2F439863" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/439863'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrative structural annotation of de novo RNA-Seq provides an accurate reference gene set of the enormous genome of the onion (Allium cepa L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu035");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f19%2f439863">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

