

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/3/233/591590","@type":"ScholarlyArticle","name":"Genome-wide view of natural antisense transcripts in  Arabidopsis thaliana ","datePublished":"2015-04-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2015-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv008","keywords":["genome-wide","natural antisense transcript","NAT-siRNA","epigenetic modification","regulation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/3/233/591590","author":[{"name":"Yuan, Chunhui","affiliation":"1  Department of Bioinformatics, College of Life Sciences ,  Zhejiang University ,  Zijingang Campus, Yu Hang Tang Road 866# ,  Hangzhou 310058 ,  China","@type":"Person"},{"name":"Wang, Jingjing","affiliation":"1  Department of Bioinformatics, College of Life Sciences ,  Zhejiang University ,  Zijingang Campus, Yu Hang Tang Road 866# ,  Hangzhou 310058 ,  China    2  James D. Watson Institute of Genome Sciences ,  Zhejiang University ,  Hangzhou 310058 ,  China","@type":"Person"},{"name":"Harrison, Andrew P.","affiliation":"3  Department of Mathematical Sciences ,  University of Essex ,  Colchester, Essex CO4 3SQ ,  UK","@type":"Person"},{"name":"Meng, Xianwen","affiliation":"1  Department of Bioinformatics, College of Life Sciences ,  Zhejiang University ,  Zijingang Campus, Yu Hang Tang Road 866# ,  Hangzhou 310058 ,  China","@type":"Person"},{"name":"Chen, Dijun","affiliation":"1  Department of Bioinformatics, College of Life Sciences ,  Zhejiang University ,  Zijingang Campus, Yu Hang Tang Road 866# ,  Hangzhou 310058 ,  China    4  Department of Molecular Genetics ,  Leibniz Institute of Plant Genetics and Crop Plant Research Gatersleben (IPK) ,  Gatersleben ,  Germany","@type":"Person"},{"name":"Chen, Ming","affiliation":"1  Department of Bioinformatics, College of Life Sciences ,  Zhejiang University ,  Zijingang Campus, Yu Hang Tang Road 866# ,  Hangzhou 310058 ,  China    2  James D. Watson Institute of Genome Sciences ,  Zhejiang University ,  Hangzhou 310058 ,  China","@type":"Person"}],"description":"Abstract.  Natural antisense transcripts (NATs) are endogenous transcripts that can form double-stranded RNA structures. Many protein-coding genes (PCs) and non","pageStart":"233","pageEnd":"243","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide view of natural antisense transcripts in  Arabidopsis thaliana ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide view of natural antisense transcripts in  Arabidopsis thaliana " />
    <meta property="og:description" content="Abstract.  Natural antisense transcripts (NATs) are endogenous transcripts that can form double-stranded RNA structures. Many protein-coding genes (PCs) and non" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/3/233/591590" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-04-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yuan, Chunhui" /><meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Zijingang Campus, Yu Hang Tang Road 866#, Hangzhou 310058, China" /><meta name="citation_author" content="Wang, Jingjing" /><meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Zijingang Campus, Yu Hang Tang Road 866#, Hangzhou 310058, China" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_author" content="Harrison, Andrew P." /><meta name="citation_author_institution" content="Department of Mathematical Sciences, University of Essex, Colchester, Essex CO4 3SQ, UK" /><meta name="citation_author" content="Meng, Xianwen" /><meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Zijingang Campus, Yu Hang Tang Road 866#, Hangzhou 310058, China" /><meta name="citation_author" content="Chen, Dijun" /><meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Zijingang Campus, Yu Hang Tang Road 866#, Hangzhou 310058, China" /><meta name="citation_author_institution" content="Department of Molecular Genetics, Leibniz Institute of Plant Genetics and Crop Plant Research Gatersleben (IPK), Gatersleben, Germany" /><meta name="citation_author" content="Chen, Ming" /><meta name="citation_author_institution" content="Department of Bioinformatics, College of Life Sciences, Zhejiang University, Zijingang Campus, Yu Hang Tang Road 866#, Hangzhou 310058, China" /><meta name="citation_author_institution" content="James D. Watson Institute of Genome Sciences, Zhejiang University, Hangzhou 310058, China" /><meta name="citation_title" content="Genome-wide view of natural antisense transcripts in Arabidopsis thaliana" /><meta name="citation_firstpage" content="233" /><meta name="citation_lastpage" content="243" /><meta name="citation_doi" content="10.1093/dnares/dsv008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2015/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genome-wide natural antisense transcription: coupling its regulation to its different regulatory mechanisms; citation_author=Lapidot  M.; citation_author=Pilpel  Y.;  citation_journal_title=EMBO Rep;  citation_year=2006;  citation_volume=7;  citation_pages=1216-22; " /><meta name="citation_reference" content="citation_title=PlantNATsDB: a comprehensive database of plant natural antisense transcripts; citation_author=Chen  D.; citation_author=Yuan  C.; citation_author=Zhang  J.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1187-93; " /><meta name="citation_reference" content="citation_title=Natural antisense transcripts: sound or silence?; citation_author=Werner  A.; citation_author=Berdal  A.;  citation_journal_title=Physiol Genomics;  citation_year=2005;  citation_volume=23;  citation_pages=125-31; " /><meta name="citation_reference" content="citation_title=NATsDB: Natural Antisense Transcripts DataBase; citation_author=Zhang  Y.; citation_author=Li  J.; citation_author=Kong  L.; citation_author=Gao  G.; citation_author=Liu  Q.R.; citation_author=Wei  L.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D156-61; " /><meta name="citation_reference" content="citation_title=Genome-wide prediction and identification of cis-natural antisense transcripts in Arabidopsis thaliana; citation_author=Wang  X.J.; citation_author=Gaasterland  T.; citation_author=Chua  N.H.;  citation_journal_title=Genome Biol;  citation_year=2005;  citation_volume=6;  citation_pages=R30" /><meta name="citation_reference" content="citation_title=Transcriptional collision between convergent genes in budding yeast; citation_author=Prescott  E.M.; citation_author=Proudfoot  N.J.;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2002;  citation_volume=99;  citation_pages=8796-801; " /><meta name="citation_reference" content="citation_title=Expression of the thyroid hormone receptor gene, erbAalpha, in B lymphocytes: alternative mRNA processing is independent of differentiation but correlates with antisense RNA levels; citation_author=Hastings  M.L.; citation_author=Milcarek  C.; citation_author=Martincic  K.; citation_author=Peterson  M.L.; citation_author=Munroe  S.H.;  citation_journal_title=Nucleic Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=4296-300; " /><meta name="citation_reference" content="citation_title=An internal antisense RNA regulates expression of the photosynthesis gene isiA; citation_author=Duhring  U.; citation_author=Axmann  I.M.; citation_author=Hess  W.R.; citation_author=Wilde  A.;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2006;  citation_volume=103;  citation_pages=7054-8; " /><meta name="citation_reference" content="citation_title=Regulatory roles of natural antisense transcripts; citation_author=Faghihi  M.A.; citation_author=Wahlestedt  C.;  citation_journal_title=Nat Rev Mol Cell Biol;  citation_year=2009;  citation_volume=10;  citation_pages=637-43; " /><meta name="citation_reference" content="citation_title=Endogenous siRNAs derived from a pair of natural cis-antisense transcripts regulate salt tolerance in Arabidopsis; citation_author=Borsani  O.; citation_author=Zhu  J.; citation_author=Verslues  P.E.; citation_author=Sunkar  R.; citation_author=Zhu  J.K.;  citation_journal_title=Cell;  citation_year=2005;  citation_volume=123;  citation_pages=1279-91; " /><meta name="citation_reference" content="citation_title=Proper regulation of a sperm-specific cis-nat-siRNA is essential for double fertilization in Arabidopsis; citation_author=Ron  M.; citation_author=Alandete Saez  M.; citation_author=Eshed Williams  L.; citation_author=Fletcher  J.C.; citation_author=McCormick  S.;  citation_journal_title=Genes Dev;  citation_year=2010;  citation_volume=24;  citation_pages=1010-21; " /><meta name="citation_reference" content="citation_title=Two distinct mechanisms generate endogenous siRNAs from bidirectional transcription in Drosophila melanogaster; citation_author=Okamura  K.; citation_author=Balla  S.; citation_author=Martin  R.; citation_author=Liu  N.; citation_author=Lai  E.C.;  citation_journal_title=Nat Struct Mol Biol;  citation_year=2008;  citation_volume=15;  citation_pages=581-90; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of plant nat-siRNAs reveals insights into their distribution, biogenesis and function; citation_author=Zhang  X.; citation_author=Xia  J.; citation_author=Lii  Y.E.;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R20" /><meta name="citation_reference" content="citation_title=The role of chromatin during transcription; citation_author=Li  B.; citation_author=Carey  M.; citation_author=Workman  J.L.;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=128;  citation_pages=707-19; " /><meta name="citation_reference" content="citation_title=Genic and global functions for Paf1C in chromatin modification and gene expression in Arabidopsis; citation_author=Oh  S.; citation_author=Park  S.; citation_author=van Nocker  S.;  citation_journal_title=PLoS Genet;  citation_year=2008;  citation_volume=4;  citation_pages=e1000077" /><meta name="citation_reference" content="citation_title=Epigenomic consequences of immortalized plant cell suspension culture; citation_author=Tanurdzic  M.; citation_author=Vaughn  M.W.; citation_author=Jiang  H.;  citation_journal_title=PLoS Biol;  citation_year=2008;  citation_volume=6;  citation_pages=2880-95; " /><meta name="citation_reference" content="citation_title=Dynamic landscapes of four histone modifications during deetiolation in Arabidopsis; citation_author=Charron  J.B.; citation_author=He  H.; citation_author=Elling  A.A.; citation_author=Deng  X.W.;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=3732-48; " /><meta name="citation_reference" content="citation_title=Genome-wide profiling of histone H3 lysine 9 acetylation and dimethylation in Arabidopsis reveals correlation between multiple histone marks and gene expression; citation_author=Zhou  J.; citation_author=Wang  X.; citation_author=He  K.; citation_author=Charron  J.B.; citation_author=Elling  A.A.; citation_author=Deng  X.W.;  citation_journal_title=Plant Mol Biol;  citation_year=2010;  citation_volume=72;  citation_pages=585-95; " /><meta name="citation_reference" content="citation_title=DNA methylation dynamics in plant genomes; citation_author=Gehring  M.; citation_author=Henikoff  S.;  citation_journal_title=Biochim Biophys Acta;  citation_year=2007;  citation_volume=1769;  citation_pages=276-86; " /><meta name="citation_reference" content="citation_title=DNA cytosine methylation in plant development; citation_author=Zhang  M.; citation_author=Kimatu  J.N.; citation_author=Xu  K.; citation_author=Liu  B.;  citation_journal_title=J Genet Genomics;  citation_year=2010;  citation_volume=37;  citation_pages=1-12; " /><meta name="citation_reference" content="citation_title=Transcription of antisense RNA leading to gene silencing and methylation as a novel cause of human genetic disease; citation_author=Tufarelli  C.; citation_author=Stanley  J.A.; citation_author=Garrick  D.;  citation_journal_title=Nat Genet;  citation_year=2003;  citation_volume=34;  citation_pages=157-65; " /><meta name="citation_reference" content="citation_title=Epigenetic silencing of tumour suppressor gene p15 by its antisense RNA; citation_author=Yu  W.; citation_author=Gius  D.; citation_author=Onyango  P.;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=451;  citation_pages=202-6; " /><meta name="citation_reference" content="citation_title=Bidirectional transcription directs both transcriptional gene activation and suppression in human cells; citation_author=Morris  K.V.; citation_author=Santoso  S.; citation_author=Turner  A.M.; citation_author=Pastori  C.; citation_author=Hawkins  P.G.;  citation_journal_title=PLoS Genet;  citation_year=2008;  citation_volume=4;  citation_pages=e1000258" /><meta name="citation_reference" content="citation_title=Epigenetic regulation of human cis-natural antisense transcripts; citation_author=Conley  A.B.; citation_author=Jordan  I.K.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=1438-45; " /><meta name="citation_reference" content="citation_title=Encoding memory of winter by noncoding RNAs; citation_author=Heo  J.B.; citation_author=Sung  S.;  citation_journal_title=Epigenetics;  citation_year=2011;  citation_volume=6;  citation_pages=544-7; " /><meta name="citation_reference" content="citation_title=Cold-induced silencing by long antisense transcripts of an Arabidopsis Polycomb target; citation_author=Swiezewski  S.; citation_author=Liu  F.; citation_author=Magusin  A.; citation_author=Dean  C.;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=462;  citation_pages=799-802; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks; citation_author=Trapnell  C.; citation_author=Roberts  A.; citation_author=Goff  L.;  citation_journal_title=Nat Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation; citation_author=Trapnell  C.; citation_author=Williams  B.A.; citation_author=Pertea  G.;  citation_journal_title=Nat Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=CPAT: Coding-Potential Assessment Tool using an alignment-free logistic regression model; citation_author=Wang  L.; citation_author=Park  H.J.; citation_author=Dasari  S.; citation_author=Wang  S.; citation_author=Kocher  J.P.; citation_author=Li  W.;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=e74" /><meta name="citation_reference" content="citation_title=BLAST+: architecture and applications; citation_author=Camacho  C.; citation_author=Coulouris  G.; citation_author=Avagyan  V.;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=421" /><meta name="citation_reference" content="citation_title=DINAMelt web server for nucleic acid melting prediction; citation_author=Markham  N.R.; citation_author=Zuker  M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W577-81; " /><meta name="citation_reference" content="citation_title=fRNAdb: a platform for mining/annotating functional RNA candidates from non-coding RNA sequences; citation_author=Kin  T.; citation_author=Yamada  K.; citation_author=Terai  G.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D145-48; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=Langmead  B.; citation_author=Trapnell  C.; citation_author=Pop  M.; citation_author=Salzberg  S.L.;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=Integrative epigenomic mapping defines four main chromatin states in Arabidopsis; citation_author=Roudier  F.; citation_author=Ahmed  I.; citation_author=Berard  C.;  citation_journal_title=EMBO J;  citation_year=2011;  citation_volume=30;  citation_pages=1928-38; " /><meta name="citation_reference" content="citation_title=MultiChIPmixHMM: an R package for ChIP-chip data analysis modeling spatial dependencies and multiple replicates; citation_author=Berard  C.; citation_author=Seifert  M.; citation_author=Mary-Huard  T.; citation_author=Martin-Magniette  M.L.;  citation_journal_title=BMC Bioinformatics;  citation_year=2013;  citation_volume=14;  citation_pages=271" /><meta name="citation_reference" content="citation_title=Cytosine methylation changes in enhancer regions of core pro-fibrotic genes characterize kidney fibrosis development; citation_author=Ko  Y.A.; citation_author=Mohtat  D.; citation_author=Suzuki  M.;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R108" /><meta name="citation_reference" content="citation_title=Functional demarcation of active and silent chromatin domains in human HOX loci by noncoding RNAs; citation_author=Rinn  J.L.; citation_author=Kertesz  M.; citation_author=Wang  J.K.;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=129;  citation_pages=1311-23; " /><meta name="citation_reference" content="citation_title=Genome-wide identification and analysis of small RNAs originated from natural antisense transcripts in Oryza sativa; citation_author=Zhou  X.; citation_author=Sunkar  R.; citation_author=Jin  H.; citation_author=Zhu  J.K.; citation_author=Zhang  W.;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=70-8; " /><meta name="citation_reference" content="citation_title=The diversity of RNA silencing pathways in plants; citation_author=Brodersen  P.; citation_author=Voinnet  O.;  citation_journal_title=Trends Genet;  citation_year=2006;  citation_volume=22;  citation_pages=268-80; " /><meta name="citation_reference" content="citation_title=Dicer-like (DCL) proteins in plants; citation_author=Liu  Q.; citation_author=Feng  Y.; citation_author=Zhu  Z.;  citation_journal_title=Funct Integr Genomics;  citation_year=2009;  citation_volume=9;  citation_pages=277-86; " /><meta name="citation_reference" content="citation_title=DICER-LIKE 4 functions in trans-acting small interfering RNA biogenesis and vegetative phase change in Arabidopsis thaliana; citation_author=Xie  Z.; citation_author=Allen  E.; citation_author=Wilken  A.; citation_author=Carrington  J.C.;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2005;  citation_volume=102;  citation_pages=12984-9; " /><meta name="citation_reference" content="citation_title=Histone methylation in higher plants; citation_author=Liu  C.; citation_author=Lu  F.; citation_author=Cui  X.; citation_author=Cao  X.;  citation_journal_title=Annu Rev Plant Biol;  citation_year=2010;  citation_volume=61;  citation_pages=395-420; " /><meta name="citation_reference" content="citation_title=Arabidopsis TFL2/LHP1 specifically associates with genes marked by trimethylation of histone H3 lysine 27; citation_author=Turck  F.; citation_author=Roudier  F.; citation_author=Farrona  S.;  citation_journal_title=PLoS Genet;  citation_year=2007;  citation_volume=3;  citation_pages=e86" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of mono-, di- and trimethylation of histone H3 lysine 4 in Arabidopsis thaliana; citation_author=Zhang  X.; citation_author=Bernatavichute  Y.V.; citation_author=Cokus  S.; citation_author=Pellegrini  M.; citation_author=Jacobsen  S.E.;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R62" /><meta name="citation_reference" content="citation_title=Control of DNA methylation and heterochromatic silencing by histone H2B deubiquitination; citation_author=Sridhar  V.V.; citation_author=Kapoor  A.; citation_author=Zhang  K.;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=447;  citation_pages=735-8; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of transposable elements in plants; citation_author=Lisch  D.;  citation_journal_title=Annu Rev Plant Biol;  citation_year=2009;  citation_volume=60;  citation_pages=43-66; " /><meta name="citation_reference" content="citation_title=DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape; citation_author=Xie  M.; citation_author=Hong  C.; citation_author=Zhang  B.;  citation_journal_title=Nat Genet;  citation_year=2013;  citation_volume=45;  citation_pages=836-41; " /><meta name="citation_reference" content="citation_title=Protein-coding cis-natural antisense transcripts have high and broad expression in Arabidopsis; citation_author=Zhan  S.; citation_author=Lukens  L.;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=161;  citation_pages=2171-80; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of coordinate expression and evolution of human cis-encoded sense-antisense transcripts; citation_author=Chen  J.; citation_author=Sun  M.; citation_author=Hurst  L.D.; citation_author=Carmichael  G.G.; citation_author=Rowley  J.D.;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=326-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/3/233/1594540/dsv008.pdf" /><meta name="description" content="Abstract.  Natural antisense transcripts (NATs) are endogenous transcripts that can form double-stranded RNA structures. Many protein-coding genes (PCs) and non" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/3/233/591590" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/3/233/591590" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/3/233/591590");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv008';
    </script>
<title>
	Genome-wide view of natural antisense transcripts in Arabidopsis thaliana | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./591590" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfqcATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXVhbiwgQ2h1bmh1aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIENvbGxlZ2Ugb2YgTGlmZSBTY2llbmNlcywgWmhlamlhbmcgVW5pdmVyc2l0eSwgWmlqaW5nYW5nIENhbXB1cywgWXUgSGFuZyBUYW5nIFJvYWQgODY2IywgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhbmcsIEppbmdqaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcywgQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2VzLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBaaWppbmdhbmcgQ2FtcHVzLCBZdSBIYW5nIFRhbmcgUm9hZCA4NjYjLCBIYW5nemhvdSAzMTAwNTgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSmFtZXMgRC4gV2F0c29uIEluc3RpdHV0ZSBvZiBHZW5vbWUgU2NpZW5jZXMsIFpoZWppYW5nIFVuaXZlcnNpdHksIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXJyaXNvbiwgQW5kcmV3IFAuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNYXRoZW1hdGljYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgRXNzZXgsIENvbGNoZXN0ZXIsIEVzc2V4IENPNCAzU1EsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWVuZywgWGlhbndlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvaW5mb3JtYXRpY3MsIENvbGxlZ2Ugb2YgTGlmZSBTY2llbmNlcywgWmhlamlhbmcgVW5pdmVyc2l0eSwgWmlqaW5nYW5nIENhbXB1cywgWXUgSGFuZyBUYW5nIFJvYWQgODY2IywgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIERpanVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9pbmZvcm1hdGljcywgQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2VzLCBaaGVqaWFuZyBVbml2ZXJzaXR5LCBaaWppbmdhbmcgQ2FtcHVzLCBZdSBIYW5nIFRhbmcgUm9hZCA4NjYjLCBIYW5nemhvdSAzMTAwNTgsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgR2VuZXRpY3MsIExlaWJuaXogSW5zdGl0dXRlIG9mIFBsYW50IEdlbmV0aWNzIGFuZCBDcm9wIFBsYW50IFJlc2VhcmNoIEdhdGVyc2xlYmVuIChJUEspLCBHYXRlcnNsZWJlbiwgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZW4sIE1pbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2luZm9ybWF0aWNzLCBDb2xsZWdlIG9mIExpZmUgU2NpZW5jZXMsIFpoZWppYW5nIFVuaXZlcnNpdHksIFppamluZ2FuZyBDYW1wdXMsIFl1IEhhbmcgVGFuZyBSb2FkIDg2NiMsIEhhbmd6aG91IDMxMDA1OCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYW1lcyBELiBXYXRzb24gSW5zdGl0dXRlIG9mIEdlbm9tZSBTY2llbmNlcywgWmhlamlhbmcgVW5pdmVyc2l0eSwgSGFuZ3pob3UgMzEwMDU4LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lLXdpZGUgdmlldyBvZiBuYXR1cmFsIGFudGlzZW5zZSB0cmFuc2NyaXB0cyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBuYXR1cmFsIGFudGlzZW5zZSB0cmFuc2NyaXB0aW9uOiBjb3VwbGluZyBpdHMgcmVndWxhdGlvbiB0byBpdHMgZGlmZmVyZW50IHJlZ3VsYXRvcnkgbWVjaGFuaXNtczsgY2l0YXRpb25fYXV0aG9yPUxhcGlkb3QgIE0uOyBjaXRhdGlvbl9hdXRob3I9UGlscGVsICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEyMTYtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50TkFUc0RCOiBhIGNvbXByZWhlbnNpdmUgZGF0YWJhc2Ugb2YgcGxhbnQgbmF0dXJhbCBhbnRpc2Vuc2UgdHJhbnNjcmlwdHM7IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBELjsgY2l0YXRpb25fYXV0aG9yPVl1YW4gIEMuOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQxMTg3LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYXR1cmFsIGFudGlzZW5zZSB0cmFuc2NyaXB0czogc291bmQgb3Igc2lsZW5jZT87IGNpdGF0aW9uX2F1dGhvcj1XZXJuZXIgIEEuOyBjaXRhdGlvbl9hdXRob3I9QmVyZGFsICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5c2lvbCBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TkFUc0RCOiBOYXR1cmFsIEFudGlzZW5zZSBUcmFuc2NyaXB0cyBEYXRhQmFzZTsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPUxpICBKLjsgY2l0YXRpb25fYXV0aG9yPUtvbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9R2FvICBHLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgUS5SLjsgY2l0YXRpb25fYXV0aG9yPVdlaSAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDE1Ni02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgcHJlZGljdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2YgY2lzLW5hdHVyYWwgYW50aXNlbnNlIHRyYW5zY3JpcHRzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgWC5KLjsgY2l0YXRpb25fYXV0aG9yPUdhYXN0ZXJsYW5kICBULjsgY2l0YXRpb25fYXV0aG9yPUNodWEgIE4uSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1SMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgY29sbGlzaW9uIGJldHdlZW4gY29udmVyZ2VudCBnZW5lcyBpbiBidWRkaW5nIHllYXN0OyBjaXRhdGlvbl9hdXRob3I9UHJlc2NvdHQgIEUuTS47IGNpdGF0aW9uX2F1dGhvcj1Qcm91ZGZvb3QgIE4uSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz04Nzk2LTgwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiB0aGUgdGh5cm9pZCBob3Jtb25lIHJlY2VwdG9yIGdlbmUsIGVyYkFhbHBoYSwgaW4gQiBseW1waG9jeXRlczogYWx0ZXJuYXRpdmUgbVJOQSBwcm9jZXNzaW5nIGlzIGluZGVwZW5kZW50IG9mIGRpZmZlcmVudGlhdGlvbiBidXQgY29ycmVsYXRlcyB3aXRoIGFudGlzZW5zZSBSTkEgbGV2ZWxzOyBjaXRhdGlvbl9hdXRob3I9SGFzdGluZ3MgIE0uTC47IGNpdGF0aW9uX2F1dGhvcj1NaWxjYXJlayAgQy47IGNpdGF0aW9uX2F1dGhvcj1NYXJ0aW5jaWMgIEsuOyBjaXRhdGlvbl9hdXRob3I9UGV0ZXJzb24gIE0uTC47IGNpdGF0aW9uX2F1dGhvcj1NdW5yb2UgIFMuSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDI5Ni0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVybmFsIGFudGlzZW5zZSBSTkEgcmVndWxhdGVzIGV4cHJlc3Npb24gb2YgdGhlIHBob3Rvc3ludGhlc2lzIGdlbmUgaXNpQTsgY2l0YXRpb25fYXV0aG9yPUR1aHJpbmcgIFUuOyBjaXRhdGlvbl9hdXRob3I9QXhtYW5uICBJLk0uOyBjaXRhdGlvbl9hdXRob3I9SGVzcyAgVy5SLjsgY2l0YXRpb25fYXV0aG9yPVdpbGRlICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYyBOYXRsIEFjYWQgU2NpIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz03MDU0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgcm9sZXMgb2YgbmF0dXJhbCBhbnRpc2Vuc2UgdHJhbnNjcmlwdHM7IGNpdGF0aW9uX2F1dGhvcj1GYWdoaWhpICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9V2FobGVzdGVkdCAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBSZXYgTW9sIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTYzNy00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kb2dlbm91cyBzaVJOQXMgZGVyaXZlZCBmcm9tIGEgcGFpciBvZiBuYXR1cmFsIGNpcy1hbnRpc2Vuc2UgdHJhbnNjcmlwdHMgcmVndWxhdGUgc2FsdCB0b2xlcmFuY2UgaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1Cb3JzYW5pICBPLjsgY2l0YXRpb25fYXV0aG9yPVpodSAgSi47IGNpdGF0aW9uX2F1dGhvcj1WZXJzbHVlcyAgUC5FLjsgY2l0YXRpb25fYXV0aG9yPVN1bmthciAgUi47IGNpdGF0aW9uX2F1dGhvcj1aaHUgIEouSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9MTI3OS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvcGVyIHJlZ3VsYXRpb24gb2YgYSBzcGVybS1zcGVjaWZpYyBjaXMtbmF0LXNpUk5BIGlzIGVzc2VudGlhbCBmb3IgZG91YmxlIGZlcnRpbGl6YXRpb24gaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1Sb24gIE0uOyBjaXRhdGlvbl9hdXRob3I9QWxhbmRldGUgU2FleiAgTS47IGNpdGF0aW9uX2F1dGhvcj1Fc2hlZCBXaWxsaWFtcyAgTC47IGNpdGF0aW9uX2F1dGhvcj1GbGV0Y2hlciAgSi5DLjsgY2l0YXRpb25fYXV0aG9yPU1jQ29ybWljayAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEwMTAtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBkaXN0aW5jdCBtZWNoYW5pc21zIGdlbmVyYXRlIGVuZG9nZW5vdXMgc2lSTkFzIGZyb20gYmlkaXJlY3Rpb25hbCB0cmFuc2NyaXB0aW9uIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyBjaXRhdGlvbl9hdXRob3I9T2thbXVyYSAgSy47IGNpdGF0aW9uX2F1dGhvcj1CYWxsYSAgUy47IGNpdGF0aW9uX2F1dGhvcj1NYXJ0aW4gIFIuOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBOLjsgY2l0YXRpb25fYXV0aG9yPUxhaSAgRS5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IFN0cnVjdCBNb2wgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTU4MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzaXMgb2YgcGxhbnQgbmF0LXNpUk5BcyByZXZlYWxzIGluc2lnaHRzIGludG8gdGhlaXIgZGlzdHJpYnV0aW9uLCBiaW9nZW5lc2lzIGFuZCBmdW5jdGlvbjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBYLjsgY2l0YXRpb25fYXV0aG9yPVhpYSAgSi47IGNpdGF0aW9uX2F1dGhvcj1MaWkgIFkuRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9UjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgY2hyb21hdGluIGR1cmluZyB0cmFuc2NyaXB0aW9uOyBjaXRhdGlvbl9hdXRob3I9TGkgIEIuOyBjaXRhdGlvbl9hdXRob3I9Q2FyZXkgIE0uOyBjaXRhdGlvbl9hdXRob3I9V29ya21hbiAgSi5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz03MDctMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmljIGFuZCBnbG9iYWwgZnVuY3Rpb25zIGZvciBQYWYxQyBpbiBjaHJvbWF0aW4gbW9kaWZpY2F0aW9uIGFuZCBnZW5lIGV4cHJlc3Npb24gaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1PaCAgUy47IGNpdGF0aW9uX2F1dGhvcj1QYXJrICBTLjsgY2l0YXRpb25fYXV0aG9yPXZhbiBOb2NrZXIgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDA3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbm9taWMgY29uc2VxdWVuY2VzIG9mIGltbW9ydGFsaXplZCBwbGFudCBjZWxsIHN1c3BlbnNpb24gY3VsdHVyZTsgY2l0YXRpb25fYXV0aG9yPVRhbnVyZHppYyAgTS47IGNpdGF0aW9uX2F1dGhvcj1WYXVnaG4gIE0uVy47IGNpdGF0aW9uX2F1dGhvcj1KaWFuZyAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9Mjg4MC05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBsYW5kc2NhcGVzIG9mIGZvdXIgaGlzdG9uZSBtb2RpZmljYXRpb25zIGR1cmluZyBkZWV0aW9sYXRpb24gaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1DaGFycm9uICBKLkIuOyBjaXRhdGlvbl9hdXRob3I9SGUgIEguOyBjaXRhdGlvbl9hdXRob3I9RWxsaW5nICBBLkEuOyBjaXRhdGlvbl9hdXRob3I9RGVuZyAgWC5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM3MzItNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHByb2ZpbGluZyBvZiBoaXN0b25lIEgzIGx5c2luZSA5IGFjZXR5bGF0aW9uIGFuZCBkaW1ldGh5bGF0aW9uIGluIEFyYWJpZG9wc2lzIHJldmVhbHMgY29ycmVsYXRpb24gYmV0d2VlbiBtdWx0aXBsZSBoaXN0b25lIG1hcmtzIGFuZCBnZW5lIGV4cHJlc3Npb247IGNpdGF0aW9uX2F1dGhvcj1aaG91ICBKLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIFguOyBjaXRhdGlvbl9hdXRob3I9SGUgIEsuOyBjaXRhdGlvbl9hdXRob3I9Q2hhcnJvbiAgSi5CLjsgY2l0YXRpb25fYXV0aG9yPUVsbGluZyAgQS5BLjsgY2l0YXRpb25fYXV0aG9yPURlbmcgIFguVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9NTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gZHluYW1pY3MgaW4gcGxhbnQgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUdlaHJpbmcgIE0uOyBjaXRhdGlvbl9hdXRob3I9SGVuaWtvZmYgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltIEJpb3BoeXMgQWN0YTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY5OyAgY2l0YXRpb25fcGFnZXM9Mjc2LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgY3l0b3NpbmUgbWV0aHlsYXRpb24gaW4gcGxhbnQgZGV2ZWxvcG1lbnQ7IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgTS47IGNpdGF0aW9uX2F1dGhvcj1LaW1hdHUgIEouTi47IGNpdGF0aW9uX2F1dGhvcj1YdSAgSy47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KIEdlbmV0IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBvZiBhbnRpc2Vuc2UgUk5BIGxlYWRpbmcgdG8gZ2VuZSBzaWxlbmNpbmcgYW5kIG1ldGh5bGF0aW9uIGFzIGEgbm92ZWwgY2F1c2Ugb2YgaHVtYW4gZ2VuZXRpYyBkaXNlYXNlOyBjaXRhdGlvbl9hdXRob3I9VHVmYXJlbGxpICBDLjsgY2l0YXRpb25fYXV0aG9yPVN0YW5sZXkgIEouQS47IGNpdGF0aW9uX2F1dGhvcj1HYXJyaWNrICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTU3LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIHNpbGVuY2luZyBvZiB0dW1vdXIgc3VwcHJlc3NvciBnZW5lIHAxNSBieSBpdHMgYW50aXNlbnNlIFJOQTsgY2l0YXRpb25fYXV0aG9yPVl1ICBXLjsgY2l0YXRpb25fYXV0aG9yPUdpdXMgIEQuOyBjaXRhdGlvbl9hdXRob3I9T255YW5nbyAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTE7ICBjaXRhdGlvbl9wYWdlcz0yMDItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlkaXJlY3Rpb25hbCB0cmFuc2NyaXB0aW9uIGRpcmVjdHMgYm90aCB0cmFuc2NyaXB0aW9uYWwgZ2VuZSBhY3RpdmF0aW9uIGFuZCBzdXBwcmVzc2lvbiBpbiBodW1hbiBjZWxsczsgY2l0YXRpb25fYXV0aG9yPU1vcnJpcyAgSy5WLjsgY2l0YXRpb25fYXV0aG9yPVNhbnRvc28gIFMuOyBjaXRhdGlvbl9hdXRob3I9VHVybmVyICBBLk0uOyBjaXRhdGlvbl9hdXRob3I9UGFzdG9yaSAgQy47IGNpdGF0aW9uX2F1dGhvcj1IYXdraW5zICBQLkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDI1OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBodW1hbiBjaXMtbmF0dXJhbCBhbnRpc2Vuc2UgdHJhbnNjcmlwdHM7IGNpdGF0aW9uX2F1dGhvcj1Db25sZXkgIEEuQi47IGNpdGF0aW9uX2F1dGhvcj1Kb3JkYW4gIEkuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTQzOC00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5jb2RpbmcgbWVtb3J5IG9mIHdpbnRlciBieSBub25jb2RpbmcgUk5BczsgY2l0YXRpb25fYXV0aG9yPUhlbyAgSi5CLjsgY2l0YXRpb25fYXV0aG9yPVN1bmcgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FcGlnZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NTQ0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGQtaW5kdWNlZCBzaWxlbmNpbmcgYnkgbG9uZyBhbnRpc2Vuc2UgdHJhbnNjcmlwdHMgb2YgYW4gQXJhYmlkb3BzaXMgUG9seWNvbWIgdGFyZ2V0OyBjaXRhdGlvbl9hdXRob3I9U3dpZXpld3NraSAgUy47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEYuOyBjaXRhdGlvbl9hdXRob3I9TWFndXNpbiAgQS47IGNpdGF0aW9uX2F1dGhvcj1EZWFuICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ2MjsgIGNpdGF0aW9uX3BhZ2VzPTc5OS04MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBnZW5lIGFuZCB0cmFuc2NyaXB0IGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgUk5BLXNlcSBleHBlcmltZW50cyB3aXRoIFRvcEhhdCBhbmQgQ3VmZmxpbmtzOyBjaXRhdGlvbl9hdXRob3I9VHJhcG5lbGwgIEMuOyBjaXRhdGlvbl9hdXRob3I9Um9iZXJ0cyAgQS47IGNpdGF0aW9uX2F1dGhvcj1Hb2ZmICBMLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTYyLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGFzc2VtYmx5IGFuZCBxdWFudGlmaWNhdGlvbiBieSBSTkEtU2VxIHJldmVhbHMgdW5hbm5vdGF0ZWQgdHJhbnNjcmlwdHMgYW5kIGlzb2Zvcm0gc3dpdGNoaW5nIGR1cmluZyBjZWxsIGRpZmZlcmVudGlhdGlvbjsgY2l0YXRpb25fYXV0aG9yPVRyYXBuZWxsICBDLjsgY2l0YXRpb25fYXV0aG9yPVdpbGxpYW1zICBCLkEuOyBjaXRhdGlvbl9hdXRob3I9UGVydGVhICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01MTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1BBVDogQ29kaW5nLVBvdGVudGlhbCBBc3Nlc3NtZW50IFRvb2wgdXNpbmcgYW4gYWxpZ25tZW50LWZyZWUgbG9naXN0aWMgcmVncmVzc2lvbiBtb2RlbDsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEwuOyBjaXRhdGlvbl9hdXRob3I9UGFyayAgSC5KLjsgY2l0YXRpb25fYXV0aG9yPURhc2FyaSAgUy47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBTLjsgY2l0YXRpb25fYXV0aG9yPUtvY2hlciAgSi5QLjsgY2l0YXRpb25fYXV0aG9yPUxpICBXLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1lNzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFTVCs6IGFyY2hpdGVjdHVyZSBhbmQgYXBwbGljYXRpb25zOyBjaXRhdGlvbl9hdXRob3I9Q2FtYWNobyAgQy47IGNpdGF0aW9uX2F1dGhvcj1Db3Vsb3VyaXMgIEcuOyBjaXRhdGlvbl9hdXRob3I9QXZhZ3lhbiAgVi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURJTkFNZWx0IHdlYiBzZXJ2ZXIgZm9yIG51Y2xlaWMgYWNpZCBtZWx0aW5nIHByZWRpY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1NYXJraGFtICBOLlIuOyBjaXRhdGlvbl9hdXRob3I9WnVrZXIgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVc1NzctODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWZSTkFkYjogYSBwbGF0Zm9ybSBmb3IgbWluaW5nL2Fubm90YXRpbmcgZnVuY3Rpb25hbCBSTkEgY2FuZGlkYXRlcyBmcm9tIG5vbi1jb2RpbmcgUk5BIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPUtpbiAgVC47IGNpdGF0aW9uX2F1dGhvcj1ZYW1hZGEgIEsuOyBjaXRhdGlvbl9hdXRob3I9VGVyYWkgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQxNDUtNDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyBjaXRhdGlvbl9hdXRob3I9TGFuZ21lYWQgIEIuOyBjaXRhdGlvbl9hdXRob3I9VHJhcG5lbGwgIEMuOyBjaXRhdGlvbl9hdXRob3I9UG9wICBNLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBTLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aXZlIGVwaWdlbm9taWMgbWFwcGluZyBkZWZpbmVzIGZvdXIgbWFpbiBjaHJvbWF0aW4gc3RhdGVzIGluIEFyYWJpZG9wc2lzOyBjaXRhdGlvbl9hdXRob3I9Um91ZGllciAgRi47IGNpdGF0aW9uX2F1dGhvcj1BaG1lZCAgSS47IGNpdGF0aW9uX2F1dGhvcj1CZXJhcmQgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xOTI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aUNoSVBtaXhITU06IGFuIFIgcGFja2FnZSBmb3IgQ2hJUC1jaGlwIGRhdGEgYW5hbHlzaXMgbW9kZWxpbmcgc3BhdGlhbCBkZXBlbmRlbmNpZXMgYW5kIG11bHRpcGxlIHJlcGxpY2F0ZXM7IGNpdGF0aW9uX2F1dGhvcj1CZXJhcmQgIEMuOyBjaXRhdGlvbl9hdXRob3I9U2VpZmVydCAgTS47IGNpdGF0aW9uX2F1dGhvcj1NYXJ5LUh1YXJkICBULjsgY2l0YXRpb25fYXV0aG9yPU1hcnRpbi1NYWduaWV0dGUgIE0uTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9zaW5lIG1ldGh5bGF0aW9uIGNoYW5nZXMgaW4gZW5oYW5jZXIgcmVnaW9ucyBvZiBjb3JlIHByby1maWJyb3RpYyBnZW5lcyBjaGFyYWN0ZXJpemUga2lkbmV5IGZpYnJvc2lzIGRldmVsb3BtZW50OyBjaXRhdGlvbl9hdXRob3I9S28gIFkuQS47IGNpdGF0aW9uX2F1dGhvcj1Nb2h0YXQgIEQuOyBjaXRhdGlvbl9hdXRob3I9U3V6dWtpICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMTA4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBkZW1hcmNhdGlvbiBvZiBhY3RpdmUgYW5kIHNpbGVudCBjaHJvbWF0aW4gZG9tYWlucyBpbiBodW1hbiBIT1ggbG9jaSBieSBub25jb2RpbmcgUk5BczsgY2l0YXRpb25fYXV0aG9yPVJpbm4gIEouTC47IGNpdGF0aW9uX2F1dGhvcj1LZXJ0ZXN6ICBNLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEouSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI5OyAgY2l0YXRpb25fcGFnZXM9MTMxMS0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaWRlbnRpZmljYXRpb24gYW5kIGFuYWx5c2lzIG9mIHNtYWxsIFJOQXMgb3JpZ2luYXRlZCBmcm9tIG5hdHVyYWwgYW50aXNlbnNlIHRyYW5zY3JpcHRzIGluIE9yeXphIHNhdGl2YTsgY2l0YXRpb25fYXV0aG9yPVpob3UgIFguOyBjaXRhdGlvbl9hdXRob3I9U3Vua2FyICBSLjsgY2l0YXRpb25fYXV0aG9yPUppbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1aaHUgIEouSy47IGNpdGF0aW9uX2F1dGhvcj1aaGFuZyAgVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZGl2ZXJzaXR5IG9mIFJOQSBzaWxlbmNpbmcgcGF0aHdheXMgaW4gcGxhbnRzOyBjaXRhdGlvbl9hdXRob3I9QnJvZGVyc2VuICBQLjsgY2l0YXRpb25fYXV0aG9yPVZvaW5uZXQgIE8uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0yNjgtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpY2VyLWxpa2UgKERDTCkgcHJvdGVpbnMgaW4gcGxhbnRzOyBjaXRhdGlvbl9hdXRob3I9TGl1ICBRLjsgY2l0YXRpb25fYXV0aG9yPUZlbmcgIFkuOyBjaXRhdGlvbl9hdXRob3I9Wmh1ICBaLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuY3QgSW50ZWdyIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yNzctODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURJQ0VSLUxJS0UgNCBmdW5jdGlvbnMgaW4gdHJhbnMtYWN0aW5nIHNtYWxsIGludGVyZmVyaW5nIFJOQSBiaW9nZW5lc2lzIGFuZCB2ZWdldGF0aXZlIHBoYXNlIGNoYW5nZSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPVhpZSAgWi47IGNpdGF0aW9uX2F1dGhvcj1BbGxlbiAgRS47IGNpdGF0aW9uX2F1dGhvcj1XaWxrZW4gIEEuOyBjaXRhdGlvbl9hdXRob3I9Q2FycmluZ3RvbiAgSi5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYyBOYXRsIEFjYWQgU2NpIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDI7ICBjaXRhdGlvbl9wYWdlcz0xMjk4NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaXN0b25lIG1ldGh5bGF0aW9uIGluIGhpZ2hlciBwbGFudHM7IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEMuOyBjaXRhdGlvbl9hdXRob3I9THUgIEYuOyBjaXRhdGlvbl9hdXRob3I9Q3VpICBYLjsgY2l0YXRpb25fYXV0aG9yPUNhbyAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUgUmV2IFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz0zOTUtNDIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBURkwyL0xIUDEgc3BlY2lmaWNhbGx5IGFzc29jaWF0ZXMgd2l0aCBnZW5lcyBtYXJrZWQgYnkgdHJpbWV0aHlsYXRpb24gb2YgaGlzdG9uZSBIMyBseXNpbmUgMjc7IGNpdGF0aW9uX2F1dGhvcj1UdXJjayAgRi47IGNpdGF0aW9uX2F1dGhvcj1Sb3VkaWVyICBGLjsgY2l0YXRpb25fYXV0aG9yPUZhcnJvbmEgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBtb25vLSwgZGktIGFuZCB0cmltZXRoeWxhdGlvbiBvZiBoaXN0b25lIEgzIGx5c2luZSA0IGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFguOyBjaXRhdGlvbl9hdXRob3I9QmVybmF0YXZpY2h1dGUgIFkuVi47IGNpdGF0aW9uX2F1dGhvcj1Db2t1cyAgUy47IGNpdGF0aW9uX2F1dGhvcj1QZWxsZWdyaW5pICBNLjsgY2l0YXRpb25fYXV0aG9yPUphY29ic2VuICBTLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVI2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyb2wgb2YgRE5BIG1ldGh5bGF0aW9uIGFuZCBoZXRlcm9jaHJvbWF0aWMgc2lsZW5jaW5nIGJ5IGhpc3RvbmUgSDJCIGRldWJpcXVpdGluYXRpb247IGNpdGF0aW9uX2F1dGhvcj1TcmlkaGFyICBWLlYuOyBjaXRhdGlvbl9hdXRob3I9S2Fwb29yICBBLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NzsgIGNpdGF0aW9uX3BhZ2VzPTczNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIHJlZ3VsYXRpb24gb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHBsYW50czsgY2l0YXRpb25fYXV0aG9yPUxpc2NoICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udSBSZXYgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTQzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgaHlwb21ldGh5bGF0aW9uIHdpdGhpbiBzcGVjaWZpYyB0cmFuc3Bvc2FibGUgZWxlbWVudCBmYW1pbGllcyBhc3NvY2lhdGVzIHdpdGggdGlzc3VlLXNwZWNpZmljIGVuaGFuY2VyIGxhbmRzY2FwZTsgY2l0YXRpb25fYXV0aG9yPVhpZSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Ib25nICBDLjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9ODM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluLWNvZGluZyBjaXMtbmF0dXJhbCBhbnRpc2Vuc2UgdHJhbnNjcmlwdHMgaGF2ZSBoaWdoIGFuZCBicm9hZCBleHByZXNzaW9uIGluIEFyYWJpZG9wc2lzOyBjaXRhdGlvbl9hdXRob3I9WmhhbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1MdWtlbnMgIEwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTIxNzEtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIGNvb3JkaW5hdGUgZXhwcmVzc2lvbiBhbmQgZXZvbHV0aW9uIG9mIGh1bWFuIGNpcy1lbmNvZGVkIHNlbnNlLWFudGlzZW5zZSB0cmFuc2NyaXB0czsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIEouOyBjaXRhdGlvbl9hdXRob3I9U3VuICBNLjsgY2l0YXRpb25fYXV0aG9yPUh1cnN0ICBMLkQuOyBjaXRhdGlvbl9hdXRob3I9Q2FybWljaGFlbCAgRy5HLjsgY2l0YXRpb25fYXV0aG9yPVJvd2xleSAgSi5ELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzI2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8zLzIzMy8xNTk0NTQwL2RzdjAwOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTmF0dXJhbCBhbnRpc2Vuc2UgdHJhbnNjcmlwdHMgKE5BVHMpIGFyZSBlbmRvZ2Vub3VzIHRyYW5zY3JpcHRzIHRoYXQgY2FuIGZvcm0gZG91YmxlLXN0cmFuZGVkIFJOQSBzdHJ1Y3R1cmVzLiBNYW55IHByb3RlaW4tY29kaW5nIGdlbmVzIChQQ3MpIGFuZCBub24iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTU5MTU5MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRR6UoZ/YQyAfLuroUDhDC+Flrx4g==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f233%2f591590" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f233%2f591590" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23905" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&amp;Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/3">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="9636742">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636742">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636744">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636744">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636750">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636750">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636762">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636762">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636838">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636838">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636840">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636840">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636845">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636845">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9636847">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9636847">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/3/219/591394">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide view of natural antisense transcripts in <em>Arabidopsis thaliana</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chunhui Yuan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics, College of Life Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Zijingang Campus, Yu Hang Tang Road 866#</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chunhui+Yuan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yuan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yuan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chunhui Yuan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jingjing Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics, College of Life Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Zijingang Campus, Yu Hang Tang Road 866#</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">James D. Watson Institute of Genome Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jingjing+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jingjing Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew P. Harrison</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Mathematical Sciences</div>, <div class="insititution">University of Essex</div>, <div class="addr-line">Colchester, Essex CO4 3SQ</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+P.+Harrison">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harrison A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harrison A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew P. Harrison</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xianwen Meng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics, College of Life Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Zijingang Campus, Yu Hang Tang Road 866#</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xianwen+Meng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Meng X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Meng X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xianwen Meng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dijun Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics, College of Life Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Zijingang Campus, Yu Hang Tang Road 866#</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Molecular Genetics</div>, <div class="insititution">Leibniz Institute of Plant Genetics and Crop Plant Research Gatersleben (IPK)</div>, <div class="addr-line">Gatersleben</div>, <div class="country">Germany</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dijun+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dijun Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ming Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Bioinformatics, College of Life Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Zijingang Campus, Yu Hang Tang Road 866#</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">James D. Watson Institute of Genome Sciences</div>, <div class="insititution">Zhejiang University</div>, <div class="addr-line">Hangzhou 310058</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel/Fax. +86 571-88206612. E-mail: <a href="mailto:mchen@zju.edu.cn" target="_blank">mchen@zju.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ming+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ming Chen</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 233–243, <a href='https://doi.org/10.1093/dnares/dsv008'>https://doi.org/10.1093/dnares/dsv008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 April 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 December 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 April 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="591590" href="/dnaresearch/article-pdf/22/3/233/1594540/dsv008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Chunhui Yuan, Jingjing Wang, Andrew P. Harrison, Xianwen Meng, Dijun Chen, Ming Chen;  Genome-wide view of natural antisense transcripts in <em>Arabidopsis thaliana</em>, <em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 233–243, <a href='https://doi.org/10.1093/dnares/dsv008'>https://doi.org/10.1093/dnares/dsv008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/591590?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/591590?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20view%20of%20natural%20antisense%20transcripts%20in%20%26lt%3Bem%26gt%3BArabidopsis%20thaliana%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2015-04-28&amp;volumeNum=22&amp;issueNum=3&amp;author=Yuan%2C%20Chunhui%3B%20Wang%2C%20Jingjing&amp;startPage=233&amp;endPage=243&amp;contentId=10.1093%2Fdnares%2Fdsv008&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f233%2f591590" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=9636742 id="9636742" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Natural antisense transcripts (NATs) are endogenous transcripts that can form double-stranded RNA structures. Many protein-coding genes (PCs) and non-protein-coding genes (NPCs) tend to form <em>cis</em>-NATs and <em>trans</em>-NATs, respectively. In this work, we identified 4,080 <em>cis</em>-NATs and 2,491 <em>trans</em>-NATs genome-widely in <em>Arabidopsis.</em> Of these, 5,385 NAT-siRNAs were detected from the small RNA sequencing data. NAT-siRNAs are typically 21nt, and are processed by Dicer-like 1 (DCL1)/DCL2 and RDR6 and function in epigenetically activated situations, or 24nt, suggesting these are processed by DCL3 and RDR2 and function in environment stress. NAT-siRNAs are significantly derived from PC/PC pairs of <em>trans</em>-NATs and NPC/NPC pairs of <em>cis</em>-NATs. Furthermore, NAT pair genes typically have similar pattern of epigenetic status. <em>Cis</em>-NATs tend to be marked by euchromatic modifications, whereas <em>trans</em>-NATs tend to be marked by heterochromatic modifications.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome-wide">genome-wide</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;natural antisense transcript&quot;">natural antisense transcript</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NAT-siRNA">NAT-siRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;epigenetic modification&quot;">epigenetic modification</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="regulation">regulation</a></div>                            <h2 scrollto-destination=9636744 id="9636744" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Natural antisense transcripts (NATs) are endogenous transcripts that can form double-stranded RNAs with other transcripts according to the base complementary rule. They can be grouped into two categories: <em>cis</em>-NATs and <em>trans</em>-NATs, based on whether they work on <em>cis</em> or <em>trans. Cis</em>-NATs are transcribed from the same genomic locus as their sense transcripts but from the opposite DNA strand, and display perfect sequence complementarity. Whereas, <em>trans</em>-NATs are transcribed from separate genomic loci and typically form partial complementarity.<sup><a reveal-id="DSV008C1" data-open="DSV008C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Moreover, <em>cis</em>-NATs can be categorized into four subtypes: head-to-head (divergent), tail-to-tail (convergent), fully containing overlap (enclosed) and nearby (nearby head-to-head and nearby tail-to-tail) depending upon their relative orientation and degree of overlap.<sup><a reveal-id="DSV008C2" data-open="DSV008C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Various techniques and methods have been applied to identify NATs from transcriptome data. However, different methods and criteria to scrutinize the data get different results. A review by Werner and Berdal<sup><a reveal-id="DSV008C3" data-open="DSV008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> indicated that ∼22, 15 and 15% genes could form NATs in human, mouse and fly, respectively. But these values are increased to 40.7, 40.9 and 25.6% in a report of Zhang <em>et al</em>.<sup><a reveal-id="DSV008C4" data-open="DSV008C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The rice and <em>Arabidopsis</em> genome have around 6 and 9% NAT genes.<sup><a reveal-id="DSV008C3" data-open="DSV008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Wang <em>et al</em>.<sup><a reveal-id="DSV008C5" data-open="DSV008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> detected 1,340 potential <em>cis</em>-NAT pairs in <em>Arabidopsis thaliana</em>, including 957 <em>cis</em>-NAT pairs that were certified by massive parallel signature sequencing (MPSS) expression data. However, in this paper, a genome-wide method was used to identify NATs comprehensively, and it was found that 7,962 (23.9%) genes of <em>A. thaliana</em> could form NAT pairs, including 4,080 <em>cis</em>-NAT pairs and 2,491 <em>trans</em>-NAT pairs.</p><p>NATs have been proposed to play important roles, especially in regulating the expression of their target genes at several levels, including transcription,<sup><a reveal-id="DSV008C6" data-open="DSV008C6" class="link link-ref link-reveal xref-bibr">6</a></sup> mRNA processing, splicing,<sup><a reveal-id="DSV008C7" data-open="DSV008C7" class="link link-ref link-reveal xref-bibr">7</a></sup> RNA stability,<sup><a reveal-id="DSV008C8" data-open="DSV008C8" class="link link-ref link-reveal xref-bibr">8</a></sup> cellular transport and translational regulation.<sup><a reveal-id="DSV008C9" data-open="DSV008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Nowadays, more and more studies have focused on the role of plants NATs in post-transcription regulation. For example, Borsani <em>et al.</em><sup><a reveal-id="DSV008C10" data-open="DSV008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> demonstrated that the pair of <em>cis</em>-NAT SRO5 and P5CDH has an antagonistic function in the regulation of salt tolerance in <em>Arabidopsis</em>. The overlapping region of this NAT pair can produce a 24 nucleotides (nt) siRNA, which could cause post-transcriptional silencing of P5CDH through cleavage. The fragment of cleaved P5CDH could then be used by RNA-dependent RNA polymerase to produce secondary dsRNA, which is subsequently processed by DCL1 to generate the secondary 21nt siRNA, which can also target P5CDH. Endogenous siRNAs from NATs are referred to as ‘natural antisense transcript siRNAs’ (NAT-siRNAs). In plants, the specific siRNAs that derived from the cleavage of the double-stranded RNA (dsRNA) formed by the overlapping region of the <em>cis</em>-NATs (termed as ‘<em>cis-</em>NAT-siRNAs’) have functional roles in environmental stress responses<sup><a reveal-id="DSV008C10" data-open="DSV008C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV008C11" data-open="DSV008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and in double fertilization.<sup><a reveal-id="DSV008C12" data-open="DSV008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> SiRNAs are more likely to be generated from <em>cis</em>-NAT genes than from non-NAT genes.<sup><a reveal-id="DSV008C13" data-open="DSV008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The density of siRNAs in overlapping regions of <em>cis</em>-NAT pairs is greater than in non-overlapping regions.<sup><a reveal-id="DSV008C13" data-open="DSV008C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Chromatin is divided into different functional regions by epigenetic marks, such as DNA methylation and histone modifications. All of these chromatin marks have a substantial effect on chromatin structure and gene function. There are a variety of histone modification types, such as methylation and acetylation. Different histone modifications have distinct distribution and distinct functions. H3K4me3 and H3K36me3 are known as euchromatic marks and are often abundant in highly expressed genes.<sup><a reveal-id="DSV008C14" data-open="DSV008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> H3K4me3 accumulate in the promoters and 5′ genic regions,<sup><a reveal-id="DSV008C14" data-open="DSV008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> while H3K36me3 accumulate across the transcribed region.<sup><a reveal-id="DSV008C15" data-open="DSV008C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSV008C16" data-open="DSV008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> H3K9me2 and H3K27me3 are major silencing mechanisms in plants.<sup><a reveal-id="DSV008C17" data-open="DSV008C17" class="link link-ref link-reveal xref-bibr">17</a></sup> H3K9me2 is enriched in both the promoter and gene body, which has been found in a limited number of repressed genes in <em>Arabidopsis</em>,<sup><a reveal-id="DSV008C18" data-open="DSV008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> while H3K27me3 prefers to mark across transcribed regions which is associated with tissue-specific and developmentally regulated genes.<sup><a reveal-id="DSV008C15" data-open="DSV008C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSV008C17" data-open="DSV008C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV008C18" data-open="DSV008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Methylation can occur at any cytosine in plants: CG, CHG and CHH (where H = A, C or T). Genes are usually methylated within the promoters (so-called ‘promoter-methylated’), or within the transcribed regions away from the 3′ and 5′ end (so-called ‘body-methylated’).<sup><a reveal-id="DSV008C19" data-open="DSV008C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Promoter-methylation is usually linked to transcriptional silencing, whereas body-methylation prefers exons and likely plays an effect on exon definition during splicing.<sup><a reveal-id="DSV008C20" data-open="DSV008C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Growing evidence shows that NATs are involved in epigenetic regulation, including DNA methylation<sup><a reveal-id="DSV008C21" data-open="DSV008C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and histone modification.<sup><a reveal-id="DSV008C22" data-open="DSV008C22" class="link link-ref link-reveal xref-bibr">22</a></sup> For example, CDKN1A, an antisense transcript of a tumour suppressor gene, can recruit histone-modifying complexes, which induces H3K27me3 to target the promoter region of the sense transcript.<sup><a reveal-id="DSV008C23" data-open="DSV008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Furthermore, Conley and Jordan<sup><a reveal-id="DSV008C24" data-open="DSV008C24" class="link link-ref link-reveal xref-bibr">24</a></sup> found that <em>cis</em>-NATs’ expression are regulated epigenetically in human genome by analysing the chromatin environment of <em>cis</em>-NATs’ promoters. In <em>Arabidopsis</em>, two transcripts<em>, COOLAIR</em> and <em>COLDAIR</em>, are related with <em>FLOWERING LOCUS C</em> (<em>FLC</em>), which encodes a floral repressor in vernalization.<sup><a reveal-id="DSV008C25" data-open="DSV008C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSV008C26" data-open="DSV008C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p><p>NATs are involved in complex regulatory networks. Nowadays, most NATs research focuses on protein-coding genes (PCs), which typically lead to <em>cis</em>-NATs. However, long non-coding RNAs (lncRNAs), transposon elements genes (TEs) or pseudogenes should also be taken into consideration. These genome elements have been considered as genomic rubbish, but it is increasingly apparent that they have many roles in organism development. Here, we used a genome-wide method to discover <em>cis</em>- and <em>trans</em>-NATs of <em>A. thaliana</em>, and analysed their multiple aspects, such as expression relationships, production of siRNA and epigenetic patterns.</p>                            <h2 scrollto-destination=9636750 id="9636750" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=9636751 id="9636751" class="section-title" data-legacy-id=s2a>2.1. LncRNA prediction and transcriptome analysis</h3>
<p>We reanalysed seven RNA-sequencing data sets (total of 18 samples, including four tissues: seedling, leaf, flower and root) for transcriptome reconstruction and lncRNA prediction (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Reads were aligned to the <em>Arabidopsis</em> genome (TAIR10) first using Tophat,<sup><a reveal-id="DSV008C27" data-open="DSV008C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and then the resulting alignment files were used to assemble transcripts. Their expression levels (FPKM, fragments per kilobase of transcript per million fragments) were calculated using Cufflinks.<sup><a reveal-id="DSV008C28" data-open="DSV008C28" class="link link-ref link-reveal xref-bibr">28</a></sup> The assembled transcripts were merged together using the Cuffmerge software. The merged assembly transcripts provide a uniform basis for calculating transcript expression in each condition<sup><a reveal-id="DSV008C27" data-open="DSV008C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and the transcript class code. If the class code of transcript is ‘u’, ‘x’ or ‘i’, which means the transcript is intergenic sequence, antisense of known gene or intronic sequence, we took it to be a lncRNA candidate. The CPAT software<sup><a reveal-id="DSV008C29" data-open="DSV008C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was used to assess coding potential of candidate lncRNAs. A candidate was retained if it satisfied: (i) the length should be more than 200 nucleotides; (ii) the maximum ORF length should be no longer than 120 amino acids and (iii) the coding probability cut-off value by CPAT should be &lt;0.365. Furthermore, the transcript which is homologous with any annotated transcript in TAIR10 or with FPKM &lt; 2 in any sample would be removed. Genomic features of these lncRNAs, such as locus and sequences, were obtained using custom Perl scripts.</p>                            <h3 scrollto-destination=9636753 id="9636753" class="section-title" data-legacy-id=s2b>2.2. NATs identification</h3>
<p>We assumed that <em>cis</em>-NAT pairs are complementary on the same locus and <em>trans</em>-NAT pairs are complementary but from different loci, in the same way as in our previous study.<sup><a reveal-id="DSV008C2" data-open="DSV008C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The predicted lncRNAs were complementary to annotated genes in TAIR10 was used as reference of genes. If two transcripts overlap in the genome, they will be selected as putative <em>cis</em>-NAT pairs. For <em>cis</em>-NATs, they can be grouped into five categories: divergent (head-to-head overlap), convergent (tail-to-tail overlap), containing (full overlap), nearby head-to-head and nearby tail-to-tail according to their relative orientation and degree of overlap. <em>Trans</em>-NATs were identified by the following operations. Blast+<sup><a reveal-id="DSV008C30" data-open="DSV008C30" class="link link-ref link-reveal xref-bibr">30</a></sup> was applied to search for potential complementary of NAT transcripts pairs. DINAMelt<sup><a reveal-id="DSV008C31" data-open="DSV008C31" class="link link-ref link-reveal xref-bibr">31</a></sup> was used to verify whether the pairs could melt into RNA-RNA duplexes in the complementary regions. If the pairing region of two transcripts identified by DINAMelt matches with the Blast search, and if any bubble in the predicted pairing region is no longer than 10% of the region, they would be considered as <em>trans</em>-NAT pairs.</p>                            <h3 scrollto-destination=9636755 id="9636755" class="section-title" data-legacy-id=s2c>2.3. NAT-siRNAs detection and enrichment in overlapping regions</h3>
<p>The enrichment of small RNAs in the overlapping regions of NATs was examined as following: first, for each NAT pair, we have counted the number (<em>N</em><sub>o</sub>) of NAT-siRNAs from our smRNA library mapping to the overlapping region, and the number (<em>N</em><sub>g</sub>) of siRNAs matching with the non-overlapping regions of two genes. If the length of the overlapping region is <em>L</em><sub>o</sub>, and the sum of the lengths of the non-overlapping regions of two genes is <em>L</em><sub>g</sub>, then the density of siRNA loci in the overlapping region is <em>N</em><sub>o</sub>/<em>L</em><sub>o</sub>, and the density of the non-overlapping regions of the two genes is <em>N</em><sub>g</sub>/<em>L</em><sub>g</sub>. For each data set described in this study, we have computed the average densities of siRNA that spawned from NAT genes in the overlapping regions (<em>A</em><sub>o</sub>) and non-overlapping regions (<em>A</em><sub>g</sub>). The ratio <em>A</em><sub>o</sub>/<em>A</em><sub>g</sub> was considered as the enrichment score (<em>R</em>), and a one-tail paired two-sample <em>t</em>-test was performed to test the significance of this enrichment.</p><p>Forty smRNA-sequencing data sets collected from GEO were used to identify NAT-siRNAs. After removing the reads that match to fRNADB,<sup><a reveal-id="DSV008C32" data-open="DSV008C32" class="link link-ref link-reveal xref-bibr">32</a></sup> such as rRNA, snRNA, snoRNA or miRNA (but preserving tRNA) from the raw data, the filtered sequences were mapped to <em>Arabidopsis</em> genome sequences using bowtie.<sup><a reveal-id="DSV008C33" data-open="DSV008C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Then, the smRNA uniquely mapped onto the genome and the length of which is between 18nt and 28nt was selected to constitute our smRNA library. From this library, the smRNA that could be mapped to the overlapping region of a single NAT pair and be discovered in at least three data sets was considered as a NAT-siRNA. We discovered 5,385 NAT-siRNAs, of which 1,548 appeared in three tissues, and 945 and 142 specifically appearing only in seedling and flower, respectively.</p>                            <h3 scrollto-destination=9636758 id="9636758" class="section-title" data-legacy-id=s2d>2.4. Epigenetic analysis of NAT genes and non-NAT genes</h3>
<p>In order to explore the difference of histone modification patterns between NAT genes and non-NAT genes, we downloaded the normalized data sets of ChIP-chip data,<sup><a reveal-id="DSV008C34" data-open="DSV008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> including H2Bub, H3K27me3, H3K27me1, H3K4me3, H4K4me2 and H3K36me3.The normalized data were analysed using the MultichipmixHMM method.<sup><a reveal-id="DSV008C35" data-open="DSV008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Following MultichipmixHMM analysis, neighbouring enriched tiles were joined into domains, with allowing a maximal gap of 200 nucleotides. Only domains of at least 400 nucleotides were considered for further analysis. We acquired genomic locations of these domains using custom Perl scripts.</p><p>To investigate the patterns of DNA methylation on NAT genes and non-NAT genes, we obtained ChIP-seq data sets from GEO. Based on the genome-wide DNA methylation sites, we calculated the absolute methylation level<sup><a reveal-id="DSV008C36" data-open="DSV008C36" class="link link-ref link-reveal xref-bibr">36</a></sup> of <em>cis</em>-NAT genes, <em>trans</em>-NAT genes and non-NAT genes at CG, CHG and CHH sites, respectively.</p>                            <h2 scrollto-destination=9636762 id="9636762" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=9636765 id="9636765" class="section-title" data-legacy-id=s3a>3.1. Genome-wide identification of NATs in <em>A. thaliana</em></h3>
<p>Recent studies have shown that lncRNAs may act as cognate antisense transcripts of other genes<sup><a reveal-id="DSV008C26" data-open="DSV008C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV008C37" data-open="DSV008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> which may have roles in regulating gene expression via chromatin-dependent functions. In order to comprehensively identify <em>Arabidopsis</em> NATs, we first determined 1,138 lncRNAs, which were complementary to annotated genes in TAIR10. We predicted 6,571 NAT pairs (Table <a reveal-id="DSV008TB1" data-open="DSV008TB1" class="link link-reveal link-table xref-fig">1</a>), including 4,080 <em>cis</em>-NAT pairs and 2,491 <em>trans</em>-NAT pairs. Of these 4,080 <em>cis</em>-NAT pairs, 1,269 are of types ‘nearby head-to-head’ or ‘nearby tail-to-tail’. Although these are two accepted types of <em>cis</em>-NAT, their action mechanism is unknown. We focused our analysis on the other types of <em>cis</em>-NATs: convergent; containing and divergent; as well as all <em>trans</em>-NATs. Among all transcripts, 7,962 (23.9%) NAT genes from annotated genes (TAIR10) formed NAT pairs and 828 (72.8%) lncRNAs took part in NAT pairs. We discovered that most genes of <em>trans</em>-NATs are non-protein-coding genes (NPC) (66.2%), while the genes that are taking part in <em>cis</em>-NATs are mostly PCs (79.4%). This indicates that PCs prefer to form <em>cis-</em> pair rather than <em>trans-</em>, while NPCs have the opposite preference. To demonstrate this feature, the distribution of NAT genes along chromosomes was plotted. From Fig. <a reveal-id="DSV008F1" data-open="DSV008F1" class="link link-reveal link-table xref-fig">1</a>A, we can see that most <em>trans</em>-NAT pairs are located in the centriole, where many NPCs are located, thus, showing that NPCs are more likely to be part of <em>trans</em>-NATs than of <em>cis-</em>NATs. The smRNAs are found across the whole chromosomes with a lower level in the centriole and terminals. Whereas NAT-siRNAs derived from specific regions have a high level at some <em>trans</em>-NATs regions. We supposed that NAT-siRNAs were derived from not only <em>cis</em>-NAT pairs but also from <em>trans</em>-NAT pairs. </p>                        <a id="9636769" scrollto-destination="9636769"></a>
<div content-id="DSV008TB1" class="table-modal"><div class="table-wrap"><div id="DSV008TB1" class="table-wrap-title" data-id="DSV008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of genes and candidate NATs in <em>A. thaliana</em><sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Number </th></tr></thead><tbody><tr><td>Total genes </td><td>33,323 </td></tr><tr><td>NAT genes </td><td>7,322 (22.0%) </td></tr><tr><td><em>cis</em>-NAT pairs </td><td>3,180 </td></tr><tr><td> Convergent </td><td>1,485 </td></tr><tr><td> Containing </td><td>261 </td></tr><tr><td> Divergent </td><td>192 </td></tr><tr><td> Nearby </td><td>1,242 </td></tr><tr><td><em>trans</em>-NAT pairs </td><td>2,180 </td></tr><tr><td>Predicted lncRNA </td><td>1,138 </td></tr><tr><td>NAT lncRNA </td><td>828 (72.8%) </td></tr><tr><td>Total NAT transcripts </td><td>8,790 </td></tr><tr><td> Gene </td><td>7,962 </td></tr><tr><td> lncRNA </td><td>828 </td></tr><tr><td>Total <em>cis</em>-NAT pairs </td><td>4,080 </td></tr><tr><td> Convergent </td><td>1,616 </td></tr><tr><td> Containing </td><td>811 </td></tr><tr><td> Divergent </td><td>384 </td></tr><tr><td> Nearby </td><td>1,269 </td></tr><tr><td>Total <em>trans</em>-NAT pairs </td><td>2,491 </td></tr></tbody></table></div><div content-id="DSV008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Number </th></tr></thead><tbody><tr><td>Total genes </td><td>33,323 </td></tr><tr><td>NAT genes </td><td>7,322 (22.0%) </td></tr><tr><td><em>cis</em>-NAT pairs </td><td>3,180 </td></tr><tr><td> Convergent </td><td>1,485 </td></tr><tr><td> Containing </td><td>261 </td></tr><tr><td> Divergent </td><td>192 </td></tr><tr><td> Nearby </td><td>1,242 </td></tr><tr><td><em>trans</em>-NAT pairs </td><td>2,180 </td></tr><tr><td>Predicted lncRNA </td><td>1,138 </td></tr><tr><td>NAT lncRNA </td><td>828 (72.8%) </td></tr><tr><td>Total NAT transcripts </td><td>8,790 </td></tr><tr><td> Gene </td><td>7,962 </td></tr><tr><td> lncRNA </td><td>828 </td></tr><tr><td>Total <em>cis</em>-NAT pairs </td><td>4,080 </td></tr><tr><td> Convergent </td><td>1,616 </td></tr><tr><td> Containing </td><td>811 </td></tr><tr><td> Divergent </td><td>384 </td></tr><tr><td> Nearby </td><td>1,269 </td></tr><tr><td>Total <em>trans</em>-NAT pairs </td><td>2,491 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>In first part only used genes in TAIR10 to predicted NATs. In second part used genes in TAIR10 and lncRNA to predicted NATs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9636769" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV008TB1" class="table-wrap-title" data-id="DSV008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of genes and candidate NATs in <em>A. thaliana</em><sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Type </th><th>Number </th></tr></thead><tbody><tr><td>Total genes </td><td>33,323 </td></tr><tr><td>NAT genes </td><td>7,322 (22.0%) </td></tr><tr><td><em>cis</em>-NAT pairs </td><td>3,180 </td></tr><tr><td> Convergent </td><td>1,485 </td></tr><tr><td> Containing </td><td>261 </td></tr><tr><td> Divergent </td><td>192 </td></tr><tr><td> Nearby </td><td>1,242 </td></tr><tr><td><em>trans</em>-NAT pairs </td><td>2,180 </td></tr><tr><td>Predicted lncRNA </td><td>1,138 </td></tr><tr><td>NAT lncRNA </td><td>828 (72.8%) </td></tr><tr><td>Total NAT transcripts </td><td>8,790 </td></tr><tr><td> Gene </td><td>7,962 </td></tr><tr><td> lncRNA </td><td>828 </td></tr><tr><td>Total <em>cis</em>-NAT pairs </td><td>4,080 </td></tr><tr><td> Convergent </td><td>1,616 </td></tr><tr><td> Containing </td><td>811 </td></tr><tr><td> Divergent </td><td>384 </td></tr><tr><td> Nearby </td><td>1,269 </td></tr><tr><td>Total <em>trans</em>-NAT pairs </td><td>2,491 </td></tr></tbody></table></div><div content-id="DSV008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Type </th><th>Number </th></tr></thead><tbody><tr><td>Total genes </td><td>33,323 </td></tr><tr><td>NAT genes </td><td>7,322 (22.0%) </td></tr><tr><td><em>cis</em>-NAT pairs </td><td>3,180 </td></tr><tr><td> Convergent </td><td>1,485 </td></tr><tr><td> Containing </td><td>261 </td></tr><tr><td> Divergent </td><td>192 </td></tr><tr><td> Nearby </td><td>1,242 </td></tr><tr><td><em>trans</em>-NAT pairs </td><td>2,180 </td></tr><tr><td>Predicted lncRNA </td><td>1,138 </td></tr><tr><td>NAT lncRNA </td><td>828 (72.8%) </td></tr><tr><td>Total NAT transcripts </td><td>8,790 </td></tr><tr><td> Gene </td><td>7,962 </td></tr><tr><td> lncRNA </td><td>828 </td></tr><tr><td>Total <em>cis</em>-NAT pairs </td><td>4,080 </td></tr><tr><td> Convergent </td><td>1,616 </td></tr><tr><td> Containing </td><td>811 </td></tr><tr><td> Divergent </td><td>384 </td></tr><tr><td> Nearby </td><td>1,269 </td></tr><tr><td>Total <em>trans</em>-NAT pairs </td><td>2,491 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>In first part only used genes in TAIR10 to predicted NATs. In second part used genes in TAIR10 and lncRNA to predicted NATs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9636769" target="_blank">View Large</a></div></div>                        <a id="9636772" scrollto-destination="9636772"></a>
<div id="DSV008F1" data-id="DSV008F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00801.jpeg?Expires=2147483647&Signature=2HlYLU0wKHz2wBHRa~4PdTmHytVp5ZjJifkO6CWVcJ08yIx8Z~YiD5ib7bIRTZP8Hlks9Nttnn04yMIBn31pIhK51SZiHs0c6C3QVY6GYugPPxOuOd43I44X36HhPO6XUy0rzYG3PV9mAtYt~yfXeoWP-Mft8lNPe2DpgdKl4iBMxVeOm92mUPHWkB0z-YAq~dXGqr8R8-LBRvOAVFCx-ZDg1nRO2oFq-d56zp9Mgd1y78CD1l8PtJ9h4ii2RfZBPevBzIcra62uXpsP10bIzd6xcsGUcryryt8JNOgUeuN8OYb-sEfzyTslITIAZEhWwrYxW7plaEVwz8l1mRG2TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="NATs in Arabidopsis. (A) NAT genes and NAT-siRNAs in chromosomes. From outermost to innermost, the first track demonstrates Arabidopsis chromosomes, the second and third tracks show the NAT genes on Watson and Crick strands, respectively, the fourth and fifth tracks show the distributions of all smRNAs and NAT-siRNAs, innermost links demonstrate the NAT pairs’ loci in chromosomes. (B) The network of NAT pairwises. Node means gene and edge means relationship of NAT pairs. (C) The number of protein-coding NAT genes with different overlapping positions. (D) The number of NAT-siRNAs with different lengths. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636772/dsv00801.jpeg" path-from-xml="dsv00801.jpeg" target="_blank">View large</a><a section="9636772" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00801.jpeg?Expires=2147483647&Signature=AR6ovTQKOXcWkF-2hINeidm742J6Oqj2t4mDKZVlKcHIdV3BZjVgoyzXCN2adt5hqlY~YwaK7~z3cBkZ2VUJmvj5H1vbk6E1LLTXjhbKAd5GOkvQqydkZgsY1mGVYur-~IB5GttPb4FApJakVZFhLo9UgutjjFUGkAUGskb26gNCTIC3fFxhM8abEW~xel4EZ-Eunv4teDcnHg62GnQIxY6qu0Fio0Cx5GdcZxTNbG3vJPeHlsglSsGQoL2f7c599mCHsGEsEv8EkXV5px-JDLw5ND94ExavBbauskW5P4-vtaOXdV4QGRx1hH5NsOjlOKmS5WAHqs8heKUK0jug~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636772&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>NATs in <em>Arabidopsis</em>. (A) NAT genes and NAT-siRNAs in chromosomes. From outermost to innermost, the first track demonstrates <em>Arabidopsis</em> chromosomes, the second and third tracks show the NAT genes on Watson and Crick strands, respectively, the fourth and fifth tracks show the distributions of all smRNAs and NAT-siRNAs, innermost links demonstrate the NAT pairs’ loci in chromosomes. (B) The network of NAT pairwises. Node means gene and edge means relationship of NAT pairs. (C) The number of protein-coding NAT genes with different overlapping positions. (D) The number of NAT-siRNAs with different lengths. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00801.jpeg?Expires=2147483647&Signature=2HlYLU0wKHz2wBHRa~4PdTmHytVp5ZjJifkO6CWVcJ08yIx8Z~YiD5ib7bIRTZP8Hlks9Nttnn04yMIBn31pIhK51SZiHs0c6C3QVY6GYugPPxOuOd43I44X36HhPO6XUy0rzYG3PV9mAtYt~yfXeoWP-Mft8lNPe2DpgdKl4iBMxVeOm92mUPHWkB0z-YAq~dXGqr8R8-LBRvOAVFCx-ZDg1nRO2oFq-d56zp9Mgd1y78CD1l8PtJ9h4ii2RfZBPevBzIcra62uXpsP10bIzd6xcsGUcryryt8JNOgUeuN8OYb-sEfzyTslITIAZEhWwrYxW7plaEVwz8l1mRG2TQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="NATs in Arabidopsis. (A) NAT genes and NAT-siRNAs in chromosomes. From outermost to innermost, the first track demonstrates Arabidopsis chromosomes, the second and third tracks show the NAT genes on Watson and Crick strands, respectively, the fourth and fifth tracks show the distributions of all smRNAs and NAT-siRNAs, innermost links demonstrate the NAT pairs’ loci in chromosomes. (B) The network of NAT pairwises. Node means gene and edge means relationship of NAT pairs. (C) The number of protein-coding NAT genes with different overlapping positions. (D) The number of NAT-siRNAs with different lengths. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636772/dsv00801.jpeg" path-from-xml="dsv00801.jpeg" target="_blank">View large</a><a section="9636772" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00801.jpeg?Expires=2147483647&Signature=AR6ovTQKOXcWkF-2hINeidm742J6Oqj2t4mDKZVlKcHIdV3BZjVgoyzXCN2adt5hqlY~YwaK7~z3cBkZ2VUJmvj5H1vbk6E1LLTXjhbKAd5GOkvQqydkZgsY1mGVYur-~IB5GttPb4FApJakVZFhLo9UgutjjFUGkAUGskb26gNCTIC3fFxhM8abEW~xel4EZ-Eunv4teDcnHg62GnQIxY6qu0Fio0Cx5GdcZxTNbG3vJPeHlsglSsGQoL2f7c599mCHsGEsEv8EkXV5px-JDLw5ND94ExavBbauskW5P4-vtaOXdV4QGRx1hH5NsOjlOKmS5WAHqs8heKUK0jug~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636772&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>NATs in <em>Arabidopsis</em>. (A) NAT genes and NAT-siRNAs in chromosomes. From outermost to innermost, the first track demonstrates <em>Arabidopsis</em> chromosomes, the second and third tracks show the NAT genes on Watson and Crick strands, respectively, the fourth and fifth tracks show the distributions of all smRNAs and NAT-siRNAs, innermost links demonstrate the NAT pairs’ loci in chromosomes. (B) The network of NAT pairwises. Node means gene and edge means relationship of NAT pairs. (C) The number of protein-coding NAT genes with different overlapping positions. (D) The number of NAT-siRNAs with different lengths. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>A NAT network was constructed based on the observed relationships between NAT-genes so as to determine whether any genes are involved in several NAT pairs. Figure <a reveal-id="DSV008F1" data-open="DSV008F1" class="link link-reveal link-table xref-fig">1</a>B shows the subnetworks of the NAT-network, in which the node number is more than 10. From the NAT-network, we found that many transcripts have more than one partners and are involved in several NAT pairs. Gene ontology (GO) enrichment analysis was conducted for each subnetwork, but only the tRNA-hub subnetwork is found to have significant enrichment in GO terms. That may be because most large subnetworks were constructed by NPCs, which are not annotated by GO. It was observed that the biggest subnetwork contains 445 (91.2%) NPCs and most of them could form <em>trans</em>-NATs, indicating that NPCs are not only more likely to form <em>trans</em>-NAT pairs with other NPC genes but also they have more than one partner. In contrast, PCs always have only one partner to form a <em>cis</em>-NAT pair and PCs were hardly ever within a wider subnetwork. The NAT network suggests that the expression of genes may be influenced by other genes in a common network. There are many star-nets of the subnetworks, which suggest that the hub of the net can be matched with many genes to produce NAT pairs. Strikingly, the hubs of stat-nets mostly are tRNA-genes, as previously reported by Zhou in <em>Oryza sativa</em>.<sup><a reveal-id="DSV008C38" data-open="DSV008C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Maybe some tRNAs participate in regulation though double-stranded structures; however, this needs more deep studies.</p><p>We counted the numbers of overlapping position types in PC NAT-genes to explore which parts of genes are found to overlap. Figure <a reveal-id="DSV008F1" data-open="DSV008F1" class="link link-reveal link-table xref-fig">1</a>C illustrates the positions of PC NAT-genes prefer to form overlaps in the following trend: 3′-UTR &gt; 5′-UTR &gt; across the coding sequence, which is similar to the former observation<sup><a reveal-id="DSV008C13" data-open="DSV008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> that NAT-siRNAs predominantly appear in non-coding regions.</p>                            <h3 scrollto-destination=9636779 id="9636779" class="section-title" data-legacy-id=s3b>3.2. siRNA derived from NAT</h3>
<p>To discover if NATs prefer to derive small RNA, we collected 40 small RNA sequencing data sets and obtained a total of more than 100 million raw sequencing reads. After removing the sequencing reads matched to miRNAs, snoRNAs and other small RNAs from fRNADB,<sup><a reveal-id="DSV008C32" data-open="DSV008C32" class="link link-ref link-reveal xref-bibr">32</a></sup> the remaining reads were mapped to the chromosome sequences using bowtie.<sup><a reveal-id="DSV008C33" data-open="DSV008C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Unique sequences with length between 18nt and 28nt were selected for further analysis.</p><p>To evaluate whether small RNAs are enriched in the overlapping region rather than non-overlapping region of NATs, we computed the significance of enrichment by <em>t</em>-test. The results are shown in Table <a reveal-id="DSV008TB2" data-open="DSV008TB2" class="link link-reveal link-table xref-fig">2</a> (all are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> sheet of small RNA enrichment). We calculated the enrichment score and found there was no significant siRNA enrichment in the overlapping region. However, it is significant that siRNA enrichment in the overlapping region was larger than in non-overlapping region which is consistent with Zhang's discovery.<sup><a reveal-id="DSV008C13" data-open="DSV008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Furthermore, the NATs were classified into three subtypes: PC/PC, PC/NPC and NPC/NPC, according to the transcript coding potential, both in <em>cis</em>- and <em>trans</em>- NATs. To explore if different NATs exhibit inequable raising small RNA, the siRNA density in the overlapping and non-overlapping region of each NAT subtype was calculated. Surprisingly, the enrichment scores of NPC/NPC in <em>cis</em>-NATs and PC/PC in <em>trans</em>-NATs were large and with significant <em>P</em>-values, which means small RNAs are enriched in NPC/NPC pairwise in <em>cis</em>-NATs and PC/PC pairwise in <em>trans</em>-NATs. So we could conclude that the NPC/NPC pairs in <em>cis</em>-NATs and PC/PC pairs in <em>trans</em>-NATs prefer to produce small RNAs. </p>                        <a id="9636782" scrollto-destination="9636782"></a>
<div content-id="DSV008TB2" class="table-modal"><div class="table-wrap"><div id="DSV008TB2" class="table-wrap-title" data-id="DSV008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Enrichment score of small RNAs in the overlapping region of different type NATs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Data set </th><th rowspan="2"><em> </em> </th><th>NATs </th><th colspan="4"><em>cis</em>-NATs<hr /> </th><th colspan="4"><em>trans</em>-NATs<hr /> </th></tr><tr><th>All NATs </th><th>All CIS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th><th>All TRANS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th></tr></thead><tbody><tr><td colspan="11">Flower </td></tr><tr><td> GSM118372 </td><td>Score<sup>a</sup> </td><td>1.235 </td><td>1.186 </td><td>0.589 </td><td>0.451 </td><td>17.153 </td><td>1.276 </td><td>5.861 </td><td>2.057 </td><td>0.666 </td></tr><tr><td> </td><td><em>P</em>-value<sup>b</sup> </td><td>0.181 </td><td>0.347 </td><td>0.980 </td><td>0.861 </td><td>0.026 </td><td>0.143 </td><td>0.000 </td><td>0.096 </td><td>0.943 </td></tr><tr><td> GSM154367 </td><td> </td><td>1.632 </td><td>2.744 </td><td>0.341 </td><td>0.683 </td><td>1,742.823 </td><td>0.845 </td><td>9.946 </td><td>16.990 </td><td>0.014 </td></tr><tr><td> </td><td> </td><td>0.176 </td><td>0.081 </td><td>0.999 </td><td>0.731 </td><td>0.028 </td><td>0.580 </td><td>0.282 </td><td>0.034 </td><td>0.932 </td></tr><tr><td> GSM280228 </td><td> </td><td>2.181 </td><td>5.477 </td><td>0.586 </td><td>5.826 </td><td>51.199 </td><td>0.999 </td><td>3.563 </td><td>1.314 </td><td>0.799 </td></tr><tr><td> </td><td> </td><td>0.056 </td><td>0.054 </td><td>0.977 </td><td>0.138 </td><td>0.054 </td><td>0.502 </td><td>0.002 </td><td>0.180 </td><td>0.926 </td></tr><tr><td colspan="11">Leaf </td></tr><tr><td> GSM118373 </td><td> </td><td>3.788 </td><td>13.755 </td><td>0.710 </td><td>1.184 </td><td>327.448 </td><td>0.726 </td><td>6.640 </td><td>1.866 </td><td>0.343 </td></tr><tr><td> </td><td> </td><td>0.158 </td><td>0.139 </td><td>0.812 </td><td>0.423 </td><td>0.143 </td><td>0.713 </td><td>0.003 </td><td>0.068 </td><td>0.874 </td></tr><tr><td> GSM154370 </td><td> </td><td>0.674 </td><td>3.535 </td><td>0.816 </td><td>0.288 </td><td>1,230.447 </td><td>0.130 </td><td>7.651 </td><td>0.991 </td><td>0.071 </td></tr><tr><td> </td><td> </td><td>0.722 </td><td>0.074 </td><td>0.583 </td><td>0.876 </td><td>0.024 </td><td>0.937 </td><td>0.059 </td><td>0.508 </td><td>0.944 </td></tr><tr><td> GSM738727 </td><td> </td><td>0.839 </td><td>1.943 </td><td>0.569 </td><td>0.064 </td><td>172.003 </td><td>0.276 </td><td>6.625 </td><td>1.601 </td><td>0.104 </td></tr><tr><td> </td><td> </td><td>0.621 </td><td>0.213 </td><td>0.986 </td><td>0.895 </td><td>0.030 </td><td>0.922 </td><td>0.000 </td><td>0.058 </td><td>0.949 </td></tr><tr><td colspan="11">Seedling </td></tr><tr><td> GSM118374 </td><td> </td><td>1.356 </td><td>2.600 </td><td>1.000 </td><td>0.579 </td><td>51.480 </td><td>0.844 </td><td>11.863 </td><td>3.809 </td><td>0.351 </td></tr><tr><td> </td><td> </td><td>0.191 </td><td>0.071 </td><td>0.501 </td><td>0.792 </td><td>0.027 </td><td>0.668 </td><td>0.002 </td><td>0.058 </td><td>0.969 </td></tr><tr><td> GSM647184 </td><td> </td><td>1.034 </td><td>2.834 </td><td>0.749 </td><td>0.388 </td><td>95.924 </td><td>0.445 </td><td>5.021 </td><td>1.725 </td><td>0.269 </td></tr><tr><td> </td><td> </td><td>0.471 </td><td>0.084 </td><td>0.880 </td><td>0.847 </td><td>0.027 </td><td>0.897 </td><td>0.000 </td><td>0.063 </td><td>0.937 </td></tr><tr><td> GSM647187 </td><td> </td><td>1.138 </td><td>2.698 </td><td>0.729 </td><td>0.487 </td><td>90.910 </td><td>0.581 </td><td>4.697 </td><td>2.215 </td><td>0.319 </td></tr><tr><td> </td><td> </td><td>0.372 </td><td>0.078 </td><td>0.865 </td><td>0.806 </td><td>0.025 </td><td>0.862 </td><td>0.000 </td><td>0.064 </td><td>0.945 </td></tr></tbody></table></div><div content-id="DSV008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Data set </th><th rowspan="2"><em> </em> </th><th>NATs </th><th colspan="4"><em>cis</em>-NATs<hr /> </th><th colspan="4"><em>trans</em>-NATs<hr /> </th></tr><tr><th>All NATs </th><th>All CIS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th><th>All TRANS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th></tr></thead><tbody><tr><td colspan="11">Flower </td></tr><tr><td> GSM118372 </td><td>Score<sup>a</sup> </td><td>1.235 </td><td>1.186 </td><td>0.589 </td><td>0.451 </td><td>17.153 </td><td>1.276 </td><td>5.861 </td><td>2.057 </td><td>0.666 </td></tr><tr><td> </td><td><em>P</em>-value<sup>b</sup> </td><td>0.181 </td><td>0.347 </td><td>0.980 </td><td>0.861 </td><td>0.026 </td><td>0.143 </td><td>0.000 </td><td>0.096 </td><td>0.943 </td></tr><tr><td> GSM154367 </td><td> </td><td>1.632 </td><td>2.744 </td><td>0.341 </td><td>0.683 </td><td>1,742.823 </td><td>0.845 </td><td>9.946 </td><td>16.990 </td><td>0.014 </td></tr><tr><td> </td><td> </td><td>0.176 </td><td>0.081 </td><td>0.999 </td><td>0.731 </td><td>0.028 </td><td>0.580 </td><td>0.282 </td><td>0.034 </td><td>0.932 </td></tr><tr><td> GSM280228 </td><td> </td><td>2.181 </td><td>5.477 </td><td>0.586 </td><td>5.826 </td><td>51.199 </td><td>0.999 </td><td>3.563 </td><td>1.314 </td><td>0.799 </td></tr><tr><td> </td><td> </td><td>0.056 </td><td>0.054 </td><td>0.977 </td><td>0.138 </td><td>0.054 </td><td>0.502 </td><td>0.002 </td><td>0.180 </td><td>0.926 </td></tr><tr><td colspan="11">Leaf </td></tr><tr><td> GSM118373 </td><td> </td><td>3.788 </td><td>13.755 </td><td>0.710 </td><td>1.184 </td><td>327.448 </td><td>0.726 </td><td>6.640 </td><td>1.866 </td><td>0.343 </td></tr><tr><td> </td><td> </td><td>0.158 </td><td>0.139 </td><td>0.812 </td><td>0.423 </td><td>0.143 </td><td>0.713 </td><td>0.003 </td><td>0.068 </td><td>0.874 </td></tr><tr><td> GSM154370 </td><td> </td><td>0.674 </td><td>3.535 </td><td>0.816 </td><td>0.288 </td><td>1,230.447 </td><td>0.130 </td><td>7.651 </td><td>0.991 </td><td>0.071 </td></tr><tr><td> </td><td> </td><td>0.722 </td><td>0.074 </td><td>0.583 </td><td>0.876 </td><td>0.024 </td><td>0.937 </td><td>0.059 </td><td>0.508 </td><td>0.944 </td></tr><tr><td> GSM738727 </td><td> </td><td>0.839 </td><td>1.943 </td><td>0.569 </td><td>0.064 </td><td>172.003 </td><td>0.276 </td><td>6.625 </td><td>1.601 </td><td>0.104 </td></tr><tr><td> </td><td> </td><td>0.621 </td><td>0.213 </td><td>0.986 </td><td>0.895 </td><td>0.030 </td><td>0.922 </td><td>0.000 </td><td>0.058 </td><td>0.949 </td></tr><tr><td colspan="11">Seedling </td></tr><tr><td> GSM118374 </td><td> </td><td>1.356 </td><td>2.600 </td><td>1.000 </td><td>0.579 </td><td>51.480 </td><td>0.844 </td><td>11.863 </td><td>3.809 </td><td>0.351 </td></tr><tr><td> </td><td> </td><td>0.191 </td><td>0.071 </td><td>0.501 </td><td>0.792 </td><td>0.027 </td><td>0.668 </td><td>0.002 </td><td>0.058 </td><td>0.969 </td></tr><tr><td> GSM647184 </td><td> </td><td>1.034 </td><td>2.834 </td><td>0.749 </td><td>0.388 </td><td>95.924 </td><td>0.445 </td><td>5.021 </td><td>1.725 </td><td>0.269 </td></tr><tr><td> </td><td> </td><td>0.471 </td><td>0.084 </td><td>0.880 </td><td>0.847 </td><td>0.027 </td><td>0.897 </td><td>0.000 </td><td>0.063 </td><td>0.937 </td></tr><tr><td> GSM647187 </td><td> </td><td>1.138 </td><td>2.698 </td><td>0.729 </td><td>0.487 </td><td>90.910 </td><td>0.581 </td><td>4.697 </td><td>2.215 </td><td>0.319 </td></tr><tr><td> </td><td> </td><td>0.372 </td><td>0.078 </td><td>0.865 </td><td>0.806 </td><td>0.025 </td><td>0.862 </td><td>0.000 </td><td>0.064 </td><td>0.945 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><sup>a</sup>The overlapping enrichment score, which captured by the ratio of siRNA densities in overlapping over siRNA densities in non-overlapping (<em>A</em><sub>o</sub>/<em>A</em><sub>g</sub>).</p></div></div><div content-id="TN3"><div class="fn"><p><sup>b</sup><em>P</em>-value for the enrichment score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9636782" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV008TB2" class="table-wrap-title" data-id="DSV008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Enrichment score of small RNAs in the overlapping region of different type NATs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Data set </th><th rowspan="2"><em> </em> </th><th>NATs </th><th colspan="4"><em>cis</em>-NATs<hr /> </th><th colspan="4"><em>trans</em>-NATs<hr /> </th></tr><tr><th>All NATs </th><th>All CIS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th><th>All TRANS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th></tr></thead><tbody><tr><td colspan="11">Flower </td></tr><tr><td> GSM118372 </td><td>Score<sup>a</sup> </td><td>1.235 </td><td>1.186 </td><td>0.589 </td><td>0.451 </td><td>17.153 </td><td>1.276 </td><td>5.861 </td><td>2.057 </td><td>0.666 </td></tr><tr><td> </td><td><em>P</em>-value<sup>b</sup> </td><td>0.181 </td><td>0.347 </td><td>0.980 </td><td>0.861 </td><td>0.026 </td><td>0.143 </td><td>0.000 </td><td>0.096 </td><td>0.943 </td></tr><tr><td> GSM154367 </td><td> </td><td>1.632 </td><td>2.744 </td><td>0.341 </td><td>0.683 </td><td>1,742.823 </td><td>0.845 </td><td>9.946 </td><td>16.990 </td><td>0.014 </td></tr><tr><td> </td><td> </td><td>0.176 </td><td>0.081 </td><td>0.999 </td><td>0.731 </td><td>0.028 </td><td>0.580 </td><td>0.282 </td><td>0.034 </td><td>0.932 </td></tr><tr><td> GSM280228 </td><td> </td><td>2.181 </td><td>5.477 </td><td>0.586 </td><td>5.826 </td><td>51.199 </td><td>0.999 </td><td>3.563 </td><td>1.314 </td><td>0.799 </td></tr><tr><td> </td><td> </td><td>0.056 </td><td>0.054 </td><td>0.977 </td><td>0.138 </td><td>0.054 </td><td>0.502 </td><td>0.002 </td><td>0.180 </td><td>0.926 </td></tr><tr><td colspan="11">Leaf </td></tr><tr><td> GSM118373 </td><td> </td><td>3.788 </td><td>13.755 </td><td>0.710 </td><td>1.184 </td><td>327.448 </td><td>0.726 </td><td>6.640 </td><td>1.866 </td><td>0.343 </td></tr><tr><td> </td><td> </td><td>0.158 </td><td>0.139 </td><td>0.812 </td><td>0.423 </td><td>0.143 </td><td>0.713 </td><td>0.003 </td><td>0.068 </td><td>0.874 </td></tr><tr><td> GSM154370 </td><td> </td><td>0.674 </td><td>3.535 </td><td>0.816 </td><td>0.288 </td><td>1,230.447 </td><td>0.130 </td><td>7.651 </td><td>0.991 </td><td>0.071 </td></tr><tr><td> </td><td> </td><td>0.722 </td><td>0.074 </td><td>0.583 </td><td>0.876 </td><td>0.024 </td><td>0.937 </td><td>0.059 </td><td>0.508 </td><td>0.944 </td></tr><tr><td> GSM738727 </td><td> </td><td>0.839 </td><td>1.943 </td><td>0.569 </td><td>0.064 </td><td>172.003 </td><td>0.276 </td><td>6.625 </td><td>1.601 </td><td>0.104 </td></tr><tr><td> </td><td> </td><td>0.621 </td><td>0.213 </td><td>0.986 </td><td>0.895 </td><td>0.030 </td><td>0.922 </td><td>0.000 </td><td>0.058 </td><td>0.949 </td></tr><tr><td colspan="11">Seedling </td></tr><tr><td> GSM118374 </td><td> </td><td>1.356 </td><td>2.600 </td><td>1.000 </td><td>0.579 </td><td>51.480 </td><td>0.844 </td><td>11.863 </td><td>3.809 </td><td>0.351 </td></tr><tr><td> </td><td> </td><td>0.191 </td><td>0.071 </td><td>0.501 </td><td>0.792 </td><td>0.027 </td><td>0.668 </td><td>0.002 </td><td>0.058 </td><td>0.969 </td></tr><tr><td> GSM647184 </td><td> </td><td>1.034 </td><td>2.834 </td><td>0.749 </td><td>0.388 </td><td>95.924 </td><td>0.445 </td><td>5.021 </td><td>1.725 </td><td>0.269 </td></tr><tr><td> </td><td> </td><td>0.471 </td><td>0.084 </td><td>0.880 </td><td>0.847 </td><td>0.027 </td><td>0.897 </td><td>0.000 </td><td>0.063 </td><td>0.937 </td></tr><tr><td> GSM647187 </td><td> </td><td>1.138 </td><td>2.698 </td><td>0.729 </td><td>0.487 </td><td>90.910 </td><td>0.581 </td><td>4.697 </td><td>2.215 </td><td>0.319 </td></tr><tr><td> </td><td> </td><td>0.372 </td><td>0.078 </td><td>0.865 </td><td>0.806 </td><td>0.025 </td><td>0.862 </td><td>0.000 </td><td>0.064 </td><td>0.945 </td></tr></tbody></table></div><div content-id="DSV008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Data set </th><th rowspan="2"><em> </em> </th><th>NATs </th><th colspan="4"><em>cis</em>-NATs<hr /> </th><th colspan="4"><em>trans</em>-NATs<hr /> </th></tr><tr><th>All NATs </th><th>All CIS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th><th>All TRANS </th><th>PC/PC </th><th>PC/NPC </th><th>NPC/NPC </th></tr></thead><tbody><tr><td colspan="11">Flower </td></tr><tr><td> GSM118372 </td><td>Score<sup>a</sup> </td><td>1.235 </td><td>1.186 </td><td>0.589 </td><td>0.451 </td><td>17.153 </td><td>1.276 </td><td>5.861 </td><td>2.057 </td><td>0.666 </td></tr><tr><td> </td><td><em>P</em>-value<sup>b</sup> </td><td>0.181 </td><td>0.347 </td><td>0.980 </td><td>0.861 </td><td>0.026 </td><td>0.143 </td><td>0.000 </td><td>0.096 </td><td>0.943 </td></tr><tr><td> GSM154367 </td><td> </td><td>1.632 </td><td>2.744 </td><td>0.341 </td><td>0.683 </td><td>1,742.823 </td><td>0.845 </td><td>9.946 </td><td>16.990 </td><td>0.014 </td></tr><tr><td> </td><td> </td><td>0.176 </td><td>0.081 </td><td>0.999 </td><td>0.731 </td><td>0.028 </td><td>0.580 </td><td>0.282 </td><td>0.034 </td><td>0.932 </td></tr><tr><td> GSM280228 </td><td> </td><td>2.181 </td><td>5.477 </td><td>0.586 </td><td>5.826 </td><td>51.199 </td><td>0.999 </td><td>3.563 </td><td>1.314 </td><td>0.799 </td></tr><tr><td> </td><td> </td><td>0.056 </td><td>0.054 </td><td>0.977 </td><td>0.138 </td><td>0.054 </td><td>0.502 </td><td>0.002 </td><td>0.180 </td><td>0.926 </td></tr><tr><td colspan="11">Leaf </td></tr><tr><td> GSM118373 </td><td> </td><td>3.788 </td><td>13.755 </td><td>0.710 </td><td>1.184 </td><td>327.448 </td><td>0.726 </td><td>6.640 </td><td>1.866 </td><td>0.343 </td></tr><tr><td> </td><td> </td><td>0.158 </td><td>0.139 </td><td>0.812 </td><td>0.423 </td><td>0.143 </td><td>0.713 </td><td>0.003 </td><td>0.068 </td><td>0.874 </td></tr><tr><td> GSM154370 </td><td> </td><td>0.674 </td><td>3.535 </td><td>0.816 </td><td>0.288 </td><td>1,230.447 </td><td>0.130 </td><td>7.651 </td><td>0.991 </td><td>0.071 </td></tr><tr><td> </td><td> </td><td>0.722 </td><td>0.074 </td><td>0.583 </td><td>0.876 </td><td>0.024 </td><td>0.937 </td><td>0.059 </td><td>0.508 </td><td>0.944 </td></tr><tr><td> GSM738727 </td><td> </td><td>0.839 </td><td>1.943 </td><td>0.569 </td><td>0.064 </td><td>172.003 </td><td>0.276 </td><td>6.625 </td><td>1.601 </td><td>0.104 </td></tr><tr><td> </td><td> </td><td>0.621 </td><td>0.213 </td><td>0.986 </td><td>0.895 </td><td>0.030 </td><td>0.922 </td><td>0.000 </td><td>0.058 </td><td>0.949 </td></tr><tr><td colspan="11">Seedling </td></tr><tr><td> GSM118374 </td><td> </td><td>1.356 </td><td>2.600 </td><td>1.000 </td><td>0.579 </td><td>51.480 </td><td>0.844 </td><td>11.863 </td><td>3.809 </td><td>0.351 </td></tr><tr><td> </td><td> </td><td>0.191 </td><td>0.071 </td><td>0.501 </td><td>0.792 </td><td>0.027 </td><td>0.668 </td><td>0.002 </td><td>0.058 </td><td>0.969 </td></tr><tr><td> GSM647184 </td><td> </td><td>1.034 </td><td>2.834 </td><td>0.749 </td><td>0.388 </td><td>95.924 </td><td>0.445 </td><td>5.021 </td><td>1.725 </td><td>0.269 </td></tr><tr><td> </td><td> </td><td>0.471 </td><td>0.084 </td><td>0.880 </td><td>0.847 </td><td>0.027 </td><td>0.897 </td><td>0.000 </td><td>0.063 </td><td>0.937 </td></tr><tr><td> GSM647187 </td><td> </td><td>1.138 </td><td>2.698 </td><td>0.729 </td><td>0.487 </td><td>90.910 </td><td>0.581 </td><td>4.697 </td><td>2.215 </td><td>0.319 </td></tr><tr><td> </td><td> </td><td>0.372 </td><td>0.078 </td><td>0.865 </td><td>0.806 </td><td>0.025 </td><td>0.862 </td><td>0.000 </td><td>0.064 </td><td>0.945 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><sup>a</sup>The overlapping enrichment score, which captured by the ratio of siRNA densities in overlapping over siRNA densities in non-overlapping (<em>A</em><sub>o</sub>/<em>A</em><sub>g</sub>).</p></div></div><div content-id="TN3"><div class="fn"><p><sup>b</sup><em>P</em>-value for the enrichment score.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9636782" target="_blank">View Large</a></div></div><p>To identify the characteristics of NAT-siRNA, 5,385 NAT-siRNAs were predicted from the above smRNA library, of which 1,548 appeared in three tissues simultaneously; with 945 and 142 NAT-siRNAs appearing in seedling and flower, respectively. The observation suggested that NAT-siRNAs are associated with specific tissues. Furthermore, we tested the length distribution of NAT-siRNAs (Fig. <a reveal-id="DSV008F1" data-open="DSV008F1" class="link link-reveal link-table xref-fig">1</a>D), and found that most NAT-siRNAs are mainly 21 and 24 nucleotides.</p><p>To investigate which type of NAT prefers to produce NAT-siRNAs, we calculated the number of siRNAs in each of the NAT pair subtype. From PC genes, 1,720 NAT-siRNAs were generated and 3,665 were from NPC genes. Some transcripts might have more than one partner, which means some NAT-siRNAs may be derived from different NAT pair subtypes. We call this type of NAT-siRNA ‘nonunique NAT-siRNA’. Whereas ‘unique NAT-siRNA’ means the NAT-siRNA-derived gene only formed one NAT pairwise. We classified the subtype more clearly into: PC–NPC, PC–PC, NPC–PC and NPC–NPC. The first transcript in the pairwise type is the one that could produce NAT-siRNA, and the second transcript is partner of the first accordingly. The numbers of unique NAT-siRNAs raised from different subtypes in <em>cis</em>-NATs are: PC–PC, 366; PC–NPC, 137; NPC–PC, 329 and NPC–NPC, 2,428, meanwhile in <em>trans</em>-NATs are: PC–PC, 418; PC–NPC, 421; NPC–PC, 216 and NPC–NPC, 525. This shows that <em>cis-</em>NAT-siRNAs mostly originated from NPCs, whereas <em>trans</em>NAT-siRNAs mostly originated from PCs. Moreover, most <em>cis-</em>NAT-siRNAs from NPCs are 21nt and a large number of <em>trans</em>NAT-siRNAs from PCs are 24nt (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                            <h3 scrollto-destination=9636785 id="9636785" class="section-title" data-legacy-id=s3c>3.3. The expression of NAT-siRNAs</h3>
<p>Diverse Dicer-like enzymes (DCLs) may take part in RNA silencing pathways, such as DCL1, DCL2, DCL3 and DCL4 within plants. DCL1 mainly associates with the production of miRNA and endogenous tasiRNA. DCL2 generates siRNAs from NATs. DCL3 generates siRNA that act as a guide for subsequent chromatin modifications, while DCL4 takes part in inverted-repeat (IR) transgenes silencing.<sup><a reveal-id="DSV008C39" data-open="DSV008C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSV008C40" data-open="DSV008C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The functions of DCLs are not only specialized and compartmentalized but are also complementary to each other. DCL2 and DCL4 are important in antiviral defence. DCL2 affects alternative processing of siRNA precursors in the absence of DCL4.<sup><a reveal-id="DSV008C41" data-open="DSV008C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The biogenesis of NAT-siRNA is dependent on DCL1, DCL2 and RDR6.<sup><a reveal-id="DSV008C10" data-open="DSV008C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV008C11" data-open="DSV008C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Furthermore, Zhang <em>et al</em>.<sup><a reveal-id="DSV008C13" data-open="DSV008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> found that the accumulation of NAT-siRNAs required DCL1, DCL3 and RDR2, although they only verified a few NAT-siRNAs, we supposed there should be other feature of NAT-siRNAs’ biogenesis and function.</p><p>The profile of NAT-siRNAs suggests that their biogenesis might be through different pathways (Fig. <a reveal-id="DSV008F2" data-open="DSV008F2" class="link link-reveal link-table xref-fig">2</a>A). The sample of the greatest accumulation of NAT-siRNA is <em>rdr2</em>, which implies that generation of most NAT-siRNA is RDR2-independent. The length and NAT type in each mutant sample of NAT-siRNAs were further analysed (Fig. <a reveal-id="DSV008F2" data-open="DSV008F2" class="link link-reveal link-table xref-fig">2</a>C). In the <em>rdr2</em> sample, most expressed NAT-siRNAs are 21nt and the containing type (<em>cis</em> type). However, in the <em>rdr6</em> sample, most are 24nt and <em>trans</em> type. These results indicated that 21nt NAT-siRNA could be RDR2-independent and 24nt could be RDR6-independent. We concluded that the biogenesis of 21nt NAT-siRNA is by RDR6 and DCL1 or DCL2 pathways and of 24nt NAT-siRNA is by RDR2 and DCL3 pathways. </p>                        <a id="9636788" scrollto-destination="9636788"></a>
<div id="DSV008F2" data-id="DSV008F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00802.jpeg?Expires=2147483647&Signature=HWQEXGu-jrbpp-eZBNvfcHA4YpxsFukY3Z5ag6SAMjNIzqjsIf4e-dmpU02ssAjf0RM2LZj9Wdn8eb0ge8cxKnoRmlOk6b5-QbUeMOsXVuyth8lPO45B5B8kt4tO5J2hqe~XgS29mtc2mXRYyceggyLAEQAGLeNAOnOqIH9~OAwESmqTsRIQxa139B3wWH4EPiNYZKSjnNBGzGD1rmhNuPc7cQLp8Av7j4qIRDVG7dOy2whEpYr1erqoXlQieIIxUjUkYcqzHMCv7NBNmU0GYYD385a4nGasKfJ7iEAwZ8tNNji7c5oXIgnG6AHccQlinY04siDnSYhtegRuebI4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of NAT-siRNAs. (A) The heatmap of NAT-siRNAs’ expression under different mutant samples. (B) The heatmap of NAT-siRNAs’ expression under stress conditions. (C) Diverse types of NAT-siRNAs in different conditions. (D) Reads per million of NAT-siRNAs in mutant samples. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636788/dsv00802.jpeg" path-from-xml="dsv00802.jpeg" target="_blank">View large</a><a section="9636788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00802.jpeg?Expires=2147483647&Signature=uLgK-aXPFy9uKGp6mQVY6VnBj50T-UKgu8NtDGr5hY2zxXciOY9LaqMFLSTvdMQ8Z3iSL9YTA1xwDKz72oVRKTzmVxwtgy8tUQOrhrQFHezr1HL9aFox6GQ3xnLhGHE84UP4Jkeqszrkad0wdJLN7SmEFxFso4l6gxSs4hBUE54~nKpQtn4rIXC2f~lR041-SuJWMpm96c3VCl7aH05SaPCm~sGX-VdlX6iBGWIzdSa6HtagiP2IEyDlfDTmdgolWEb8B1qOTw8fPtElUFaZLi4RNqGiP0N6GESDFFzWCAkBxtQpqB7sq5xXDGR1navbhEs-iC3E8aAs-Wa5KPBRMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636788&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of NAT-siRNAs. (A) The heatmap of NAT-siRNAs’ expression under different mutant samples. (B) The heatmap of NAT-siRNAs’ expression under stress conditions. (C) Diverse types of NAT-siRNAs in different conditions. (D) Reads per million of NAT-siRNAs in mutant samples. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00802.jpeg?Expires=2147483647&Signature=HWQEXGu-jrbpp-eZBNvfcHA4YpxsFukY3Z5ag6SAMjNIzqjsIf4e-dmpU02ssAjf0RM2LZj9Wdn8eb0ge8cxKnoRmlOk6b5-QbUeMOsXVuyth8lPO45B5B8kt4tO5J2hqe~XgS29mtc2mXRYyceggyLAEQAGLeNAOnOqIH9~OAwESmqTsRIQxa139B3wWH4EPiNYZKSjnNBGzGD1rmhNuPc7cQLp8Av7j4qIRDVG7dOy2whEpYr1erqoXlQieIIxUjUkYcqzHMCv7NBNmU0GYYD385a4nGasKfJ7iEAwZ8tNNji7c5oXIgnG6AHccQlinY04siDnSYhtegRuebI4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression of NAT-siRNAs. (A) The heatmap of NAT-siRNAs’ expression under different mutant samples. (B) The heatmap of NAT-siRNAs’ expression under stress conditions. (C) Diverse types of NAT-siRNAs in different conditions. (D) Reads per million of NAT-siRNAs in mutant samples. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636788/dsv00802.jpeg" path-from-xml="dsv00802.jpeg" target="_blank">View large</a><a section="9636788" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00802.jpeg?Expires=2147483647&Signature=uLgK-aXPFy9uKGp6mQVY6VnBj50T-UKgu8NtDGr5hY2zxXciOY9LaqMFLSTvdMQ8Z3iSL9YTA1xwDKz72oVRKTzmVxwtgy8tUQOrhrQFHezr1HL9aFox6GQ3xnLhGHE84UP4Jkeqszrkad0wdJLN7SmEFxFso4l6gxSs4hBUE54~nKpQtn4rIXC2f~lR041-SuJWMpm96c3VCl7aH05SaPCm~sGX-VdlX6iBGWIzdSa6HtagiP2IEyDlfDTmdgolWEb8B1qOTw8fPtElUFaZLi4RNqGiP0N6GESDFFzWCAkBxtQpqB7sq5xXDGR1navbhEs-iC3E8aAs-Wa5KPBRMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636788&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression of NAT-siRNAs. (A) The heatmap of NAT-siRNAs’ expression under different mutant samples. (B) The heatmap of NAT-siRNAs’ expression under stress conditions. (C) Diverse types of NAT-siRNAs in different conditions. (D) Reads per million of NAT-siRNAs in mutant samples. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The expression patterns of 21nt and 24nt <em>trans</em>NAT-siRNAs were investigated in the small RNA-sequencing data sets of <em>ddm1</em>, <em>rdr6</em> and <em>ddm1 rdr6</em> (Fig. <a reveal-id="DSV008F2" data-open="DSV008F2" class="link link-reveal link-table xref-fig">2</a>D). We calculated the RPM (reads per million) in each sample. 21nt <em>trans</em>NAT-siRNAs accumulated significantly compared with the 24nt <em>trans</em>NAT-siRNAs, in <em>ddm1</em>. Furthermore, fewer 21nt <em>trans</em>NAT-siRNAs were found in <em>rdr6</em> and <em>ddm1 rdr6</em>. Thus, the generation of 21nt <em>trans</em>NAT-siRNAs depends on RDR6 and rises in epigenetically activated <em>ddm1</em> mutant. This indicates that 21nt NAT-siRNAs generated from <em>trans</em>-NATs are important in repressing the expression of TE, especially in epigenetically activated situations.</p><p>The expression analysis of the NAT-siRNAs was studied in four different condition data sets (Fig. <a reveal-id="DSV008F2" data-open="DSV008F2" class="link link-reveal link-table xref-fig">2</a>B). Compared with the expression of NAT-siRNAs in control data set, more specific NAT-siRNAs were expressed under cold environment. Here, also some common NAT-siRNAs appeared in four data sets (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). By comparing the common NAT-siRNAs and specific NAT-siRNAs, we found that the percentages of <em>trans</em>-NATs and 24nt NAT-siRNAs increased in specific conditions, which is presumably because <em>trans</em>-NATs and 24nt NAT-siRNAs play a major role in stress responses. These results indicate that NATs, especially <em>trans</em>-NATs, have important function in response to environmental pressures.</p>                            <h3 scrollto-destination=9636791 id="9636791" class="section-title" data-legacy-id=s3d>3.4. Histone modifications among NAT genes</h3>
<p>In order to understand the relationship between histone modifications and NAT genes, we have analysed the distributions of six chromatin modifications (H2Bub, H3K27me3, H3K27me1, H3K4me3, H4K4me2 and H3K36me3) along <em>cis</em>-NAT genes, <em>trans</em>-NAT genes and non-NAT genes. According to the enriched domains definition (Materials and methods section), domains were discovered for each chromatin mark, and then average enrichment levels were calculated within and around genes for all marks. Values are higher within the transcribed region for all the six histone modifications and typically lower in either upstream or downstream regions (Fig. <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>A). H3K4me3, H3K4me2 and H3K36me3 accumulated in the transcribed region, and tilted slightly to the 5′-terminal promoter side, especially H3K4me3. In contrast, H2Bub, H3K27me1 and H3K27me3 prefer to mark across transcribed regions, and are higher towards the centre. However, distribution patterns vary for different marks, which are in agreement with previous reports.<sup><a reveal-id="DSV008C15" data-open="DSV008C15" class="link link-ref link-reveal xref-bibr">15</a>–<a reveal-id="DSV008C17" data-open="DSV008C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV008C42" data-open="DSV008C42" class="link link-ref link-reveal xref-bibr">42</a>–<a reveal-id="DSV008C44" data-open="DSV008C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Furthermore, these chromatin modifications exhibit preferential association with NATs. Figure <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>A shows that active marks are more enriched in <em>cis</em>-NAT genes, and illustrates the following enrichment trend: <em>cis</em>-NAT &gt; non-NAT &gt; <em>trans</em>-NAT genes. However, H3K27me1 is mostly enriched in <em>trans</em>-NAT genes, and a bit higher enriched in non-NAT genes than in <em>cis</em>-NAT genes. H3K27me3 is mainly enriched in non-NAT genes, with a similar degree of enrichment in <em>cis</em>-NAT and <em>trans</em>-NAT genes which are smaller than in non-NAT genes. </p>                        <a id="9636795" scrollto-destination="9636795"></a>
<div id="DSV008F3" data-id="DSV008F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00803.jpeg?Expires=2147483647&Signature=2OIdr6xfeok5gtilWcM2lMTiMjvsSUT7hDespO9rLW6Pcmh6n3AMd4sj8dGxCZrEM0e0O-uNBtQNr1yTAezhtVl6RGJaExvgFSzdrru--5MOGmyXA1dhecV47MXa1at2aWTTSrRRMePtGZ8WAbbF1bkddGgplT44ZMcmHObhOIt5Ede1~4d46VwidEXwFelJGfPf9riavuEbzGk2LuKiu8NBzXL6SZpSciFym7Bp2r~9~sXE7SLPNZ~ui2s3K2DeIxfdhWb5X06Nhl8pWwW7e7uXwBgUl-YALQTHrGz1RMStXEHNLknvpdg0HFvwKtS9WKBmiOp3X9iyp5EZfWt5gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The enrichment degree of histone marks among cis-NAT, trans-NAT and non-NAT genes. The average enrichment levels are plotted along transcribed region (scaled to accommodate for different gene lengths, bin size of 1%) as well as the upstream (−1kb) and downstream (+1kb) sequences (bin size of 10 bp). (B and C) Histone modification patterns of cis-NAT, trans-NAT and non-NAT genes. Group1 (G1) means the modifications, H3K4me2, H3K4me3, H3K36me3 and H2Bub, which are euchromatic marks; Group 2 (G2) contains H3K27me1 and H3K27me3, which are heterochromatic marks. G1 and G2 represents genes targeted by marks both G1 and G2. (B) All genes in NAT types and (C) transposon elements only. (D) The divergence histone modifications of two genes in each NAT pair. Different bars shows the percentages of genes of cis-NATs, trans-NATs, randomized cis pairs and randomized trans pairs in ten classes of modification patterns (G1_G1, G2_G2, G1_G2, G1+G2_G1, G1+G2_G2, G1+G2_G1+G2, G1+G2_NON, G1_NON, G2_NON and NON_NON). G1, G2 are euchromatic and heterochromatic marks, same as above. NON refers to genes are not marked by above six histone modifications.This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636795/dsv00803.jpeg" path-from-xml="dsv00803.jpeg" target="_blank">View large</a><a section="9636795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00803.jpeg?Expires=2147483647&Signature=yi88Ur7M6QhDD33XKJLw4BsybrKTJ~U-EV9qsHJSXjnSZreTcKVB~gNQ7jYoND3U6dQjZ33y81XOapEfo-jkrxtM9K8FLMs2p-fmyY1EMIfykfqjr-V22BemRl~xNUuMooLSMwi6a~a1w~OXQg8aAhS7dVplIzJnU2kwNBi0o2nPF7kaTFuHw1MYqF6T-XzrEEYyTuSTtASloUc8t8fSPwOTUPBAB8kOBdCcFPHvzF1qU84lCnZRgX8gSPZl820K9Xmwt2yqpFUueDTSOKT-fCEW6rQMp7u333p10vx~SjZBhiHXTAhEXQ9jFaZ06J8s6T4AHMbpZ0Blt9u6aStwPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636795&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The enrichment degree of histone marks among <em>cis</em>-NAT, <em>trans-</em>NAT and non-NAT genes. The average enrichment levels are plotted along transcribed region (scaled to accommodate for different gene lengths, bin size of 1%) as well as the upstream (−1kb) and downstream (+1kb) sequences (bin size of 10 bp). (B and C) Histone modification patterns of <em>cis</em>-NAT, <em>trans</em>-NAT and non-NAT genes. Group1 (G1) means the modifications, H3K4me2, H3K4me3, H3K36me3 and H2Bub, which are euchromatic marks; Group 2 (G2) contains H3K27me1 and H3K27me3, which are heterochromatic marks. G1 and G2 represents genes targeted by marks both G1 and G2. (B) All genes in NAT types and (C) transposon elements only. (D) The divergence histone modifications of two genes in each NAT pair. Different bars shows the percentages of genes of <em>cis</em>-NATs, <em>trans</em>-NATs, randomized <em>cis</em> pairs and randomized <em>trans</em> pairs in ten classes of modification patterns (G1_G1, G2_G2, G1_G2, G1+G2_G1, G1+G2_G2, G1+G2_G1+G2, G1+G2_NON, G1_NON, G2_NON and NON_NON). G1, G2 are euchromatic and heterochromatic marks, same as above. NON refers to genes are not marked by above six histone modifications.This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00803.jpeg?Expires=2147483647&Signature=2OIdr6xfeok5gtilWcM2lMTiMjvsSUT7hDespO9rLW6Pcmh6n3AMd4sj8dGxCZrEM0e0O-uNBtQNr1yTAezhtVl6RGJaExvgFSzdrru--5MOGmyXA1dhecV47MXa1at2aWTTSrRRMePtGZ8WAbbF1bkddGgplT44ZMcmHObhOIt5Ede1~4d46VwidEXwFelJGfPf9riavuEbzGk2LuKiu8NBzXL6SZpSciFym7Bp2r~9~sXE7SLPNZ~ui2s3K2DeIxfdhWb5X06Nhl8pWwW7e7uXwBgUl-YALQTHrGz1RMStXEHNLknvpdg0HFvwKtS9WKBmiOp3X9iyp5EZfWt5gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) The enrichment degree of histone marks among cis-NAT, trans-NAT and non-NAT genes. The average enrichment levels are plotted along transcribed region (scaled to accommodate for different gene lengths, bin size of 1%) as well as the upstream (−1kb) and downstream (+1kb) sequences (bin size of 10 bp). (B and C) Histone modification patterns of cis-NAT, trans-NAT and non-NAT genes. Group1 (G1) means the modifications, H3K4me2, H3K4me3, H3K36me3 and H2Bub, which are euchromatic marks; Group 2 (G2) contains H3K27me1 and H3K27me3, which are heterochromatic marks. G1 and G2 represents genes targeted by marks both G1 and G2. (B) All genes in NAT types and (C) transposon elements only. (D) The divergence histone modifications of two genes in each NAT pair. Different bars shows the percentages of genes of cis-NATs, trans-NATs, randomized cis pairs and randomized trans pairs in ten classes of modification patterns (G1_G1, G2_G2, G1_G2, G1+G2_G1, G1+G2_G2, G1+G2_G1+G2, G1+G2_NON, G1_NON, G2_NON and NON_NON). G1, G2 are euchromatic and heterochromatic marks, same as above. NON refers to genes are not marked by above six histone modifications.This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636795/dsv00803.jpeg" path-from-xml="dsv00803.jpeg" target="_blank">View large</a><a section="9636795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00803.jpeg?Expires=2147483647&Signature=yi88Ur7M6QhDD33XKJLw4BsybrKTJ~U-EV9qsHJSXjnSZreTcKVB~gNQ7jYoND3U6dQjZ33y81XOapEfo-jkrxtM9K8FLMs2p-fmyY1EMIfykfqjr-V22BemRl~xNUuMooLSMwi6a~a1w~OXQg8aAhS7dVplIzJnU2kwNBi0o2nPF7kaTFuHw1MYqF6T-XzrEEYyTuSTtASloUc8t8fSPwOTUPBAB8kOBdCcFPHvzF1qU84lCnZRgX8gSPZl820K9Xmwt2yqpFUueDTSOKT-fCEW6rQMp7u333p10vx~SjZBhiHXTAhEXQ9jFaZ06J8s6T4AHMbpZ0Blt9u6aStwPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636795&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) The enrichment degree of histone marks among <em>cis</em>-NAT, <em>trans-</em>NAT and non-NAT genes. The average enrichment levels are plotted along transcribed region (scaled to accommodate for different gene lengths, bin size of 1%) as well as the upstream (−1kb) and downstream (+1kb) sequences (bin size of 10 bp). (B and C) Histone modification patterns of <em>cis</em>-NAT, <em>trans</em>-NAT and non-NAT genes. Group1 (G1) means the modifications, H3K4me2, H3K4me3, H3K36me3 and H2Bub, which are euchromatic marks; Group 2 (G2) contains H3K27me1 and H3K27me3, which are heterochromatic marks. G1 and G2 represents genes targeted by marks both G1 and G2. (B) All genes in NAT types and (C) transposon elements only. (D) The divergence histone modifications of two genes in each NAT pair. Different bars shows the percentages of genes of <em>cis</em>-NATs, <em>trans</em>-NATs, randomized <em>cis</em> pairs and randomized <em>trans</em> pairs in ten classes of modification patterns (G1_G1, G2_G2, G1_G2, G1+G2_G1, G1+G2_G2, G1+G2_G1+G2, G1+G2_NON, G1_NON, G2_NON and NON_NON). G1, G2 are euchromatic and heterochromatic marks, same as above. NON refers to genes are not marked by above six histone modifications.This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>To investigate whether the chromatin environment of NAT genes differs from non-NAT genes, the epigenetic modifications between them were compared. According to the association with transcriptional activities, six histone modifications could be divided into two groups. Group 1 (G1) includes the following four histone modifications: H3K4me2, H3K4me3, H3K36me3 and H2Bub, which are known as euchromatic marks and often are related with active genes.<sup><a reveal-id="DSV008C15" data-open="DSV008C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSV008C44" data-open="DSV008C44" class="link link-ref link-reveal xref-bibr">44</a>,<a reveal-id="DSV008C45" data-open="DSV008C45" class="link link-ref link-reveal xref-bibr">45</a></sup> H3K27me1 and H3K27me3 make up the group 2 (G2), which are often known as heterochromatic marks or associated with silenced regions.<sup><a reveal-id="DSV008C34" data-open="DSV008C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSV008C43" data-open="DSV008C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Our analysis indicates that there are clear differences between the chromatin states for <em>cis</em>-NAT, <em>trans</em>-NAT and non-NAT genes (Fig. <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>B): <em>cis</em>-NAT genes tend to be targeted by histone marks from G1, whereas <em>trans</em>-NAT genes are mostly marked by histone modifications from G2. One possible explanation for the differential histone modification patterns among these three gene groups is that they have a different percentage of transposon element genes. The majority of TEs are inactive, and there is a steady epigenetic modification state that maintains the TEs silence.<sup><a reveal-id="DSV008C46" data-open="DSV008C46" class="link link-ref link-reveal xref-bibr">46</a></sup> TEs are mostly targeted by regressive marks, rather than active marks. We tested this assumption by calculating the percentage of TEs in three gene groups. As expected, the <em>trans</em>-NAT gene group has the largest proportion of TEs (36.5%, 653 of 1,788), and the largest proportion of other NPC genes (29.7%, 531 of 1,788). Although a similar proportion of PCs was observed for <em>cis</em>-NAT and non-NAT gene groups (79.4 and 81.2%, respectively), there are more TEs and less lncRNAs in the non-NAT gene group (12.0%, 3,083 of 25,671 and 1.2%, 310 of 25,671, respectively) than in the <em>cis</em>-NAT gene group (2.7%, 205 of 7,473 and 11.0%, 821 of 7,473, respectively). Compared with the chromatin status of TEs from <em>cis</em>-NAT, <em>trans</em>-NAT and non-NAT gene group (Fig. <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>C), TEs from the <em>trans</em>-NAT and non-NAT gene groups are hardly targeted by marks from G1. But in the <em>cis</em>-NAT gene group, more than 35% TEs are marked by histone modifications from G1. All these results indicate that although TEs in the <em>cis</em>-NAT gene group are few, they are more likely to be targeted by G1 than those within the <em>trans</em>-NAT genes group.</p><p>To check the assumption that NAT gene pairs share common histone modification patterns, all NAT pairs were divided into 10 groups according to the modification of each gene in the pairwise (Fig. <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>D). To further compare the epigenetic patterns between NAT pairs and randomized pairs in each group, two sets genes were randomly selected as the control sets from non-NAT genes, which have the same gene number with the <em>cis</em>-NATs and <em>trans</em>-NATs groups. The random selections were performed 10 000 times and the average percentage of each set in each group was calculated. As shown in Fig. <a reveal-id="DSV008F3" data-open="DSV008F3" class="link link-reveal link-table xref-fig">3</a>D, there are different percentages between <em>cis</em>-NAT pairs and <em>trans</em>-NAT pairs in each group. <em>Cis</em>-NAT pairs are mostly marked by group G1_G1, and <em>trans</em>-NAT pairs are marked by G2_G2, which indicate the two genes share a common histone modification pattern. Regarding to <em>cis</em>-NAT pairs, 67.7% (1,904 of 2,811) are marked by G1_G1, which is greater than the percentage of randomized pairs; however, only 3.2% (89 of 2,811) are marked by G2_G2, which is less than the percentage of randomized pairs. Among the 1,904 <em>cis</em>-NAT pairs, marked by activated modification G1_G1, 1,141 (70.6%, 1,141 of 1,616) are convergent pairs, 267 (69.5%, 267 of 384) are divergent pairs and 496 (61.2%, 496 of 811) are containing pairs. This indicates that different types of <em>cis</em>-NAT pair show no difference in their epigenetic patterns, and most of them are targeted by active modification marks. In contrast, almost half of (1,239 of 2,491) <em>trans</em>-NAT pairs are targeted by G2_G2 and just 8.0% (199 of 2,491) pairs are targeted by G1_G1; 2.6% (66 of 2,491) of <em>trans</em>-NAT pairs are marked by oppositely functional epigenetic marks, G1_G2.</p>                            <h3 scrollto-destination=9636804 id="9636804" class="section-title" data-legacy-id=s3e>3.5. DNA methylation among NAT genes</h3>
<p>Methylation of cytosine in nuclear DNA is a common form of epigenetic modifications that has been reported to exist in mammals and plants. To analyse the relationship of DNA methylation between NAT genes, absolute methylation levels were calculated within and around genes at CG, CHG and CHH sites from six methylation data sets. As shown in Fig. <a reveal-id="DSV008F4" data-open="DSV008F4" class="link link-reveal link-table xref-fig">4</a>A, the DNA methylation level in <em>trans</em>-NAT genes is higher than that in <em>cis</em>-NAT genes, indicating DNA methylation, as a repression mark, is more enriched in <em>trans</em>-NAT genes and less enriched in <em>cis</em>-NAT genes. As expected, TEs from the <em>cis</em>-NAT gene group had the lowest level of methylation compared with those TEs from the <em>trans</em>-NAT and non-NAT gene group (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). </p>                        <a id="9636809" scrollto-destination="9636809"></a>
<div id="DSV008F4" data-id="DSV008F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00804.jpeg?Expires=2147483647&Signature=eygQQ62YOBX~Fy31YCXsdJHkS9fyEniEFzOnV-v1-K33Uo-2FNbZk0lq4M4BhB39acOMedD~dsYSHQM5BR7FsCKt8rAuAIu2QHHYVhomzjAmYnyeLZAstiJ8-O14e7j25iojEZtsf5ggBxdjXLhGznIYAWB9pcJpjNXz6s9EB2FWdMR6mNi6aDO6qRoABb-VpUuafWocNvaOXOO-b7pxYa3SgxV92sO7FGyHkrrE7LtNgzoHrzgVodlpjJL7844GmRDOy0aaBFSDGtHljlnUk0jJDfI-WDUQR~p7lYDiQh6L50GEi0YE-C-nfH1y7ogNRg2uoKS3nVLGc4af3T0UEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Variation of methylation among cis-NATs, trans-NATs and non-NATs. The absolutely methylation levels are plotted in 5′-UTR and transcribed region of cis-NAT genes, trans-NAT genes and non-NAT genes, in six samples respectively. (B) Comparison of DNA methylation divergence between cis-NAT pairs and trans-NAT pairs at CG, CHG and CHH sites in 5′-UTR and transcribed region. The Y-axis is the average DNA methylation difference of NAT. -P refers to DNA methylated in 5′-UTR, whereas -B refers to DNA methylated in transcribed region. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636809/dsv00804.jpeg" path-from-xml="dsv00804.jpeg" target="_blank">View large</a><a section="9636809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00804.jpeg?Expires=2147483647&Signature=dEFMZOqdBs5ife5wVMdLDL5veUpbBMvR5S6uLvFl7mG8u9eT~WZzql2HIY1Ueynhgtjaozs76fLqYuqDXB5LL~4bHeyvadM81FLQ0pbmwfgNONRkN736xDe-ycV0gLV~Rxj2HfuR1-y05dSR0SNkvgOgWK3Yzjh36HfYcczM~NP6IjRC0x9haJQwjoe1aHlZPLGoisY0GggY6ORYbebXQltaNxfkEW73RuGXo45418qwJOuniMHCT0rOoqFS5ZSSYbdXfKi~MjCgWzI6Vl~LGHmT4edHNCpYmc7cgEdwWNtrGqKbJS57OTm6sGB1~Gj5ixgfyIHLArhiC9axqc8Gzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636809&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Variation of methylation among <em>cis</em>-NATs, <em>trans</em>-NATs and non-NATs. The absolutely methylation levels are plotted in 5′-UTR and transcribed region of <em>cis</em>-NAT genes, <em>trans</em>-NAT genes and non-NAT genes, in six samples respectively. (B) Comparison of DNA methylation divergence between <em>cis</em>-NAT pairs and <em>trans</em>-NAT pairs at CG, CHG and CHH sites in 5′-UTR and transcribed region. The Y-axis is the average DNA methylation difference of NAT. -P refers to DNA methylated in 5′-UTR, whereas -B refers to DNA methylated in transcribed region. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00804.jpeg?Expires=2147483647&Signature=eygQQ62YOBX~Fy31YCXsdJHkS9fyEniEFzOnV-v1-K33Uo-2FNbZk0lq4M4BhB39acOMedD~dsYSHQM5BR7FsCKt8rAuAIu2QHHYVhomzjAmYnyeLZAstiJ8-O14e7j25iojEZtsf5ggBxdjXLhGznIYAWB9pcJpjNXz6s9EB2FWdMR6mNi6aDO6qRoABb-VpUuafWocNvaOXOO-b7pxYa3SgxV92sO7FGyHkrrE7LtNgzoHrzgVodlpjJL7844GmRDOy0aaBFSDGtHljlnUk0jJDfI-WDUQR~p7lYDiQh6L50GEi0YE-C-nfH1y7ogNRg2uoKS3nVLGc4af3T0UEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Variation of methylation among cis-NATs, trans-NATs and non-NATs. The absolutely methylation levels are plotted in 5′-UTR and transcribed region of cis-NAT genes, trans-NAT genes and non-NAT genes, in six samples respectively. (B) Comparison of DNA methylation divergence between cis-NAT pairs and trans-NAT pairs at CG, CHG and CHH sites in 5′-UTR and transcribed region. The Y-axis is the average DNA methylation difference of NAT. -P refers to DNA methylated in 5′-UTR, whereas -B refers to DNA methylated in transcribed region. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636809/dsv00804.jpeg" path-from-xml="dsv00804.jpeg" target="_blank">View large</a><a section="9636809" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00804.jpeg?Expires=2147483647&Signature=dEFMZOqdBs5ife5wVMdLDL5veUpbBMvR5S6uLvFl7mG8u9eT~WZzql2HIY1Ueynhgtjaozs76fLqYuqDXB5LL~4bHeyvadM81FLQ0pbmwfgNONRkN736xDe-ycV0gLV~Rxj2HfuR1-y05dSR0SNkvgOgWK3Yzjh36HfYcczM~NP6IjRC0x9haJQwjoe1aHlZPLGoisY0GggY6ORYbebXQltaNxfkEW73RuGXo45418qwJOuniMHCT0rOoqFS5ZSSYbdXfKi~MjCgWzI6Vl~LGHmT4edHNCpYmc7cgEdwWNtrGqKbJS57OTm6sGB1~Gj5ixgfyIHLArhiC9axqc8Gzg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636809&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Variation of methylation among <em>cis</em>-NATs, <em>trans</em>-NATs and non-NATs. The absolutely methylation levels are plotted in 5′-UTR and transcribed region of <em>cis</em>-NAT genes, <em>trans</em>-NAT genes and non-NAT genes, in six samples respectively. (B) Comparison of DNA methylation divergence between <em>cis</em>-NAT pairs and <em>trans</em>-NAT pairs at CG, CHG and CHH sites in 5′-UTR and transcribed region. The Y-axis is the average DNA methylation difference of NAT. -P refers to DNA methylated in 5′-UTR, whereas -B refers to DNA methylated in transcribed region. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Previous studies have found some tissue-specific and subfamily-specific hypomethylation of TEs within 928 TE subfamilies in human embryonic and adult tissues by examining genome-wide DNA methylation status. The TEs are located closely to the genes whose functions are important for the relevant tissue type. The hypomethylation of TEs promoted the expression of the surrounding genes, suggesting TEs may be responsible for wiring tissue-specific regulatory networks.<sup><a reveal-id="DSV008C47" data-open="DSV008C47" class="link link-ref link-reveal xref-bibr">47</a></sup> We searched the genes around (2 kb) TEs from the <em>cis</em>-NAT gene group and obtained 114 functional annotations from the GO (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). According to the function annotation, 12 of the genes adjacent to TEs are involved in response to stresses, such as virus, bacteria and salt. This indicates that those TEs, which are marked by histone modifications from G1 and are less methylated, may play an important role in systemic resistance.</p><p>To explore if the two genes from NAT pair have more similar DNA methylation pattern than two randomly selected non-NAT genes, we compared the distance of DNA methylation levels between a NAT pair and a randomized non-NAT pair. We used one minus Pearson's product-moment correlation coefficient, i.e. <em>D</em><sub>mC</sub> = 1 – <em>r</em>, to define the distance of DNA methylation, specifically the distance of DNA methylation at CG, CHG and CHH sites associated in promoter and transcribed regions, respectively. A smaller value of <em>D</em><sub>mC</sub> indicates a greater similarity of DNA methylation pattern between sense and antisense gene of a NAT gene pair. Randomized pairs were selected from 27 910 non-NAT genes (including lncRNAs) with 10 000 repeats. As expected, we discovered that these six measures show a higher similarity degree of DNA methylation between <em>cis</em>-NAT gene pairs than that of randomized non-NAT gene pairs (Wilcoxon's rank-sum test: <em>P</em> &lt; 0.05 for all cases). For <em>trans</em>-NAT gene pairs, similar results hold (Wilcoxon's rank-sum test: <em>P</em> &lt; 0.05 for five cases), except at CHH sites, where <em>trans</em>-NAT gene pairs and randomized pairs show no difference in their DNA methylation distance.</p><p>We also compared the DNA methylation distance between <em>cis</em>-NAT pair genes and <em>trans</em>-NAT pair genes. As shown in Fig. <a reveal-id="DSV008F4" data-open="DSV008F4" class="link link-reveal link-table xref-fig">4</a>B, <em>trans</em>-NAT pairs have a much greater DNA methylation distance in transcribed regions than <em>cis</em>-NAT pairs. Whereas in 5′-promoter regions, <em>cis</em>-NAT pairs have a larger methylation distance between sense and antisense gene in CG and CHG except in CHH.</p>                            <h3 scrollto-destination=9636820 id="9636820" class="section-title" data-legacy-id=s3f>3.6. Expression network of NAT genes</h3>
<p>Many sense–antisense genes show correlated expression. For example, some <em>cis</em>-NATs can regulate the transcript abundance of their complement by triggering the biogenesis of natural antisense siRNA which subsequently guides transcript cleavage.<sup><a reveal-id="DSV008C48" data-open="DSV008C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Chen <em>et al</em>.<sup><a reveal-id="DSV008C49" data-open="DSV008C49" class="link link-ref link-reveal xref-bibr">49</a></sup> found that <em>cis</em>-NAT gene pairs tend to be coexpressed or inversely expressed more frequently than expected, and they are conserved through evolution. In order to explore comprehensively the correlation between two genes of a <em>cis</em>-NAT pair or a <em>trans</em>-NAT pair, a NAT expression network was constructed. We used dsRNA-sequencing data sets to determine whether NAT pairs formed double-stranded structures <em>in vivo</em>. We counted the average reads that mapped in the overlapping region and also upstream and downstream of this region, then found the dsRNA reads were enriched at overlapping region with significant <em>P</em>-values (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). This indicates that NATs forming double-stranded structure <em>in vivo</em> are widespread. One thousand eight hundred and twelve NAT pairs were detected to appear in dsRNA-sequencing data sets. The reads per kilobase per million reads (RPKM) of a gene's expression was counted from the 15 RNA-sequencing data sets. Genes occurring in less than three data sets were removed. Then, the Pearson correlation coefficient (PCC) of the expressions of two genes in a NAT pair was calculated. To show the network clearly, we focused on pairs with correlation |PCC| &gt; 0.8 to construct a network of NAT pairs’ expression (Fig. <a reveal-id="DSV008F5" data-open="DSV008F5" class="link link-reveal link-table xref-fig">5</a>). In this work, the coexpression of NATs is more common than expected. Many TEs are organized into subnetworks and most PC has coexpression relationship with just one partner. </p>                        <a id="9636823" scrollto-destination="9636823"></a>
<div id="DSV008F5" data-id="DSV008F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00805.jpeg?Expires=2147483647&Signature=ySYkmFw3Jv9t5iI~TkCmGoCpR9SZozxQqgdbj6WLJAbBen9KtNsk-8Rfwdf9sA87qE5QS7NjKTE~4Cnt0~ivz0WNltwAeUf0wmFIDhkCaxnXaNKiXSu9q54E90L3S3QSCWwDgiN-1gGwEZudEuJ0O3UwYBHJuFKALbQT7avo98qKam~9mN80gJEPCH0fovC8iyN-vmxm8vBX0uvV87ebKa~S~kzPCYrzhp6ZIsUhYn9~69Ur8sz3c93W9t4RjkO-V~JQElaDaypXWBvNlsKXVesFQ9rvNljVIjGy-gX55q3N5Fg5tANnD505Vshug0kAcDv9KnmovFDm-acGPfw2Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The expression network of NAT-genes. The network was constructed by the NAT pairs with expression correlation |PCC| ≥ 0.8. The solid and dash edge shows cis- and trans- type of the pair. Both means both of highly similar or distinct epigenetic modification and NAT-siRNA production features happen between this NAT pair. (PCC: Pearson correlation coefficient.) This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636823/dsv00805.jpeg" path-from-xml="dsv00805.jpeg" target="_blank">View large</a><a section="9636823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00805.jpeg?Expires=2147483647&Signature=mnRuVG8AVN~7gVclryG0BkqN3s2EscQaQpIEYuKQQe1DL2KnyR471tPB3~4-Y8dJK3IHhqIXnTZQxdFUiR9LP68hMbmph3HiD9czcvdYGwd6WrR7LO0NjhxtPRUq2RBpL4jBq-62Eix98GD8x32yXlyP9BcWnbM2EmQPzLs-1YmvpqUMF6JZojOcllLpC~mPCd5r9eJIE-dzii1D3kREkcI8Xl~TX3z5vQohUoZkreTbE0JSVaAkCwtGMX2pkoDOgv81CeyGBMrnXg7WqRmQdCioUr0fJVuHbhDPXGBXwyDTdOXu6HsKY1Wjp~GkuMXf7j1QHg4pzsgg0x8LJ3RpFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636823&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The expression network of NAT-genes. The network was constructed by the NAT pairs with expression correlation |PCC| ≥ 0.8. The solid and dash edge shows <em>cis-</em> and <em>trans-</em> type of the pair. Both means both of highly similar or distinct epigenetic modification and NAT-siRNA production features happen between this NAT pair. (PCC: Pearson correlation coefficient.) This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV008F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/m_dsv00805.jpeg?Expires=2147483647&Signature=ySYkmFw3Jv9t5iI~TkCmGoCpR9SZozxQqgdbj6WLJAbBen9KtNsk-8Rfwdf9sA87qE5QS7NjKTE~4Cnt0~ivz0WNltwAeUf0wmFIDhkCaxnXaNKiXSu9q54E90L3S3QSCWwDgiN-1gGwEZudEuJ0O3UwYBHJuFKALbQT7avo98qKam~9mN80gJEPCH0fovC8iyN-vmxm8vBX0uvV87ebKa~S~kzPCYrzhp6ZIsUhYn9~69Ur8sz3c93W9t4RjkO-V~JQElaDaypXWBvNlsKXVesFQ9rvNljVIjGy-gX55q3N5Fg5tANnD505Vshug0kAcDv9KnmovFDm-acGPfw2Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The expression network of NAT-genes. The network was constructed by the NAT pairs with expression correlation |PCC| ≥ 0.8. The solid and dash edge shows cis- and trans- type of the pair. Both means both of highly similar or distinct epigenetic modification and NAT-siRNA production features happen between this NAT pair. (PCC: Pearson correlation coefficient.) This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00805.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9636823/dsv00805.jpeg" path-from-xml="dsv00805.jpeg" target="_blank">View large</a><a section="9636823" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv00805.jpeg?Expires=2147483647&Signature=mnRuVG8AVN~7gVclryG0BkqN3s2EscQaQpIEYuKQQe1DL2KnyR471tPB3~4-Y8dJK3IHhqIXnTZQxdFUiR9LP68hMbmph3HiD9czcvdYGwd6WrR7LO0NjhxtPRUq2RBpL4jBq-62Eix98GD8x32yXlyP9BcWnbM2EmQPzLs-1YmvpqUMF6JZojOcllLpC~mPCd5r9eJIE-dzii1D3kREkcI8Xl~TX3z5vQohUoZkreTbE0JSVaAkCwtGMX2pkoDOgv81CeyGBMrnXg7WqRmQdCioUr0fJVuHbhDPXGBXwyDTdOXu6HsKY1Wjp~GkuMXf7j1QHg4pzsgg0x8LJ3RpFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9636823&ar=591590&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00805.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The expression network of NAT-genes. The network was constructed by the NAT pairs with expression correlation |PCC| ≥ 0.8. The solid and dash edge shows <em>cis-</em> and <em>trans-</em> type of the pair. Both means both of highly similar or distinct epigenetic modification and NAT-siRNA production features happen between this NAT pair. (PCC: Pearson correlation coefficient.) This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Since many of the NAT genes have an expression correlation and they also have a natural structure relationship, we supposed there might be some regulatory mechanisms shared. To discovery if this is true, we mapped the features of NATs to the expression network. And we focused on the regulation features of NAT-siRNA silencing and epigenetic modifications. We identified coexpressed NAT gene pairs that have the potential to produce NAT-siRNA as well as the coexpressed NAT gene pairs that have a strong epigenetic modification relationship (the absolute value of their PCC of modification pattern similarity is larger than 0.8). Incorporating the feature into the NAT expression network (Fig. <a reveal-id="DSV008F5" data-open="DSV008F5" class="link link-reveal link-table xref-fig">5</a>), we found that most coexpressed NATs have closely related epigenetic modifications. This suggests that some NATs influence each other through their epigenetic modifications and not only through endogenous NAT-siRNAs.</p>                            <h3 scrollto-destination=9636829 id="9636829" class="section-title" data-legacy-id=s3g>3.7. Conclusion</h3>
<p>We have made a genome-wide analysis of NATs in <em>A. thaliana</em>. To form NAT pairs, 23.9% of genes of <em>Arabidopsis</em> in TAIR10 and 72.8% of unannotated lncRNA were discovered, which indicated that most lncRNA prefer to form double-stranded structures. That may be a mechanism by which lncRNAs regulate other transcripts. We predicted 6,571 NAT pairs including 4,080 <em>cis</em>-NAT pairs and 2,491 <em>trans</em>-NAT pairs. From the predicted NAT pairs, we found not only lncRNAs take part in forming NATs but also TE transcripts. The non-coding transcripts prefer to form <em>trans</em>-NAT and PC transcripts prefer to form <em>cis</em>-NAT. PC transcripts often only have a single partner to form a pair of NAT; however, NPC have more than one partner and can form several NAT pairs, which leads to a regulatory network. That is a new view about NATs and it indicates different types of genes may have a special function.</p><p>One accepted mechanism of NAT regulation is that the antisense transcript could produce siRNA to repress sense transcript expression. By comparing the small RNA density in different regions of the genes, we have shown that small RNA enrichment is significant in the overlapping region of <em>cis</em>-NAT pair formed by two non-coding protein transcripts and in the overlapping region of <em>trans</em>-NAT pair formed by both coding protein transcripts. Furthermore, these two types of NATs also typically generate NAT-siRNAs. We detected the genome-wide prevalence of NAT-siRNAs and discovered that the lengths of NAT-siRNAs are mainly 21nt and 24nt. Most 21nt NAT-siRNAs are derived from <em>cis</em>-NATs, whereas most 24nt NAT-siNRAs are from <em>trans</em>-NATs. Based on the NAT-siRNA profiles investigated from smRNA sequencing data sets, 21nt NAT-siRNAs were detected to been processed by DCL1 and/or DCL2, RDR6 and 24nt NAT-siRNAs were found to been processed by DCL3, RDR2. Furthermore, 21nt <em>trans</em>NAT-siRNAs have a role in repressing transcripts expression in an epigenetically activated situation. Moreover, 24nt NAT-siRNAs were more likely expressed under stress conditions.</p><p>The analysis results of epigenetic modifications show the two genes in a NAT pair have similar histone modifications. <em>cis</em>-NAT genes are the most likely to be enriched with active marks, followed by non-NAT and then <em>trans</em>-NAT genes. However, heterochromatic marks such as H3K27me1 are enriched in <em>trans</em>-NAT genes, even though H3K27me1 is mainly found in non-NAT genes. To explain this observation, we analysed TEs in NATs. Although <em>cis</em>-NATs contain less TEs than <em>trans</em>-NATs, the percentage of TEs which are targeted by euchromatic histone marks in <em>cis</em>-NATs is much more than in <em>trans-</em>. We do not know the specific mechanism by which NAT formation influences the modification of TEs or by which specific TEs tends to favour the <em>cis</em>-NAT formation, but there are some relations. We also discovered the two genes of NAT pair have similar DNA methylation profiles in the promoter and translated region, with the <em>cis</em>-NAT gene pairs more similar than <em>trans</em>-NAT gene pairs. Eventually, the methylation levels of CG, CHG and CHH in <em>trans</em>-NAT all are higher than in <em>cis</em>-NAT.</p><p>All the analyses illustrate NAT as a natural formed molecular has important regulation functions both at transcriptional level and at post-transcriptional level. A regulatory network was constructed to illustrate the relationship between NAT genes expression and the potentially regulatory relationship. This indicates the regulation of expression between NATs may be through NAT-siRNA interference and/or epigenetic effects. We hope this study could expand the current views of NATs in plants, and will inspire more research efforts on the detailed regulatory mechanisms of NATs.</p>                            <h2 scrollto-destination=9636838 id="9636838" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=9636840 id="9636840" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the National Natural Sciences Foundation of China (No. 31371328, 31450110068), National Science and Technology Project of China (No. 2009DFA32030); the Fundamental Research Funds for the Central Universities, China; Sino-Germany Cooperation on Agricultural Science and Technology, Germany; and Henry Lester Trust, UK. Funding to pay the Open Access publication charges for this article was provided by the National Natural Sciences Foundation of China.</p>                            <h2 scrollto-destination=9636845 id="9636845" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Zhao Zhang, Guy J. Baudoux, Bjoern Sommer and Asif Ahsn for helpful discussions.</p>                            <h2 scrollto-destination=9636847 id="9636847" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lapidot</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Pilpel</div> <div class="given-names">Y.</div></div></div> <div class="year">2006</div>, <div class="article-title">Genome-wide natural antisense transcription: coupling its regulation to its different regulatory mechanisms</div>, <div class="source">EMBO Rep</div> , <div class="volume">7</div>, <div class="fpage">1216</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20natural%20antisense%20transcription%3A%20coupling%20its%20regulation%20to%20its%20different%20regulatory%20mechanisms&amp;author=M.%20Lapidot&amp;author=Y.%20Pilpel&amp;publication_year=2006&amp;journal=EMBO%20Rep&amp;volume=7&amp;pages=1216-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.embor.7400857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.embor.7400857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.embor.7400857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17139297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Yuan</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">PlantNATsDB: a comprehensive database of plant natural antisense transcripts</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">40</div>, <div class="fpage">D1187</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantNATsDB%3A%20a%20comprehensive%20database%20of%20plant%20natural%20antisense%20transcripts&amp;author=D.%20Chen&amp;author=C.%20Yuan&amp;author=J.%20Zhang&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1187-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr823" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr823" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr823"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22058132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Werner</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Berdal</div> <div class="given-names">A.</div></div></div> <div class="year">2005</div>, <div class="article-title">Natural antisense transcripts: sound or silence?</div>, <div class="source">Physiol Genomics</div> , <div class="volume">23</div>, <div class="fpage">125</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Natural%20antisense%20transcripts%3A%20sound%20or%20silence%3F&amp;author=A.%20Werner&amp;author=A.%20Berdal&amp;publication_year=2005&amp;journal=Physiol%20Genomics&amp;volume=23&amp;pages=125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/physiolgenomics.00124.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fphysiolgenomics.00124.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fphysiolgenomics.00124.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16230481" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Kong</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Gao</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">Q.R.</div></div> <div class="name"><div class="surname">Wei</div> <div class="given-names">L.</div></div></div> <div class="year">2007</div>, <div class="article-title">NATsDB: Natural Antisense Transcripts DataBase</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">35</div>, <div class="fpage">D156</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NATsDB%3A%20Natural%20Antisense%20Transcripts%20DataBase&amp;author=Y.%20Zhang&amp;author=J.%20Li&amp;author=L.%20Kong&amp;author=G.%20Gao&amp;author=Q.R.%20Liu&amp;author=L.%20Wei&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D156-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl782" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl782" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl782"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17082204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.J.</div></div> <div class="name"><div class="surname">Gaasterland</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Chua</div> <div class="given-names">N.H.</div></div></div> <div class="year">2005</div>, <div class="article-title">Genome-wide prediction and identification of cis-natural antisense transcripts in <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Biol</div> , <div class="volume">6</div>, <div class="fpage">R30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20prediction%20and%20identification%20of%20cis-natural%20antisense%20transcripts%20in%20Arabidopsis%20thaliana&amp;author=X.J.%20Wang&amp;author=T.%20Gaasterland&amp;author=N.H.%20Chua&amp;publication_year=2005&amp;journal=Genome%20Biol&amp;volume=6&amp;pages=R30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-4-r30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-4-r30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-4-r30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15833117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Prescott</div> <div class="given-names">E.M.</div></div> <div class="name"><div class="surname">Proudfoot</div> <div class="given-names">N.J.</div></div></div> <div class="year">2002</div>, <div class="article-title">Transcriptional collision between convergent genes in budding yeast</div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="volume">99</div>, <div class="fpage">8796</div>–<div class="lpage">801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20collision%20between%20convergent%20genes%20in%20budding%20yeast&amp;author=E.M.%20Prescott&amp;author=N.J.%20Proudfoot&amp;publication_year=2002&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=99&amp;pages=8796-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.132270899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.132270899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.132270899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12077310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Hastings</div> <div class="given-names">M.L.</div></div> <div class="name"><div class="surname">Milcarek</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Martincic</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Peterson</div> <div class="given-names">M.L.</div></div> <div class="name"><div class="surname">Munroe</div> <div class="given-names">S.H.</div></div></div> <div class="year">1997</div>, <div class="article-title">Expression of the thyroid hormone receptor gene, erbAalpha, in B lymphocytes: alternative mRNA processing is independent of differentiation but correlates with antisense RNA levels</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">25</div>, <div class="fpage">4296</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20thyroid%20hormone%20receptor%20gene%2C%20erbAalpha%2C%20in%20B%20lymphocytes%3A%20alternative%20mRNA%20processing%20is%20independent%20of%20differentiation%20but%20correlates%20with%20antisense%20RNA%20levels&amp;author=M.L.%20Hastings&amp;author=C.%20Milcarek&amp;author=K.%20Martincic&amp;author=M.L.%20Peterson&amp;author=S.H.%20Munroe&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res&amp;volume=25&amp;pages=4296-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.21.4296" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.21.4296" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.21.4296"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9336460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Duhring</div> <div class="given-names">U.</div></div> <div class="name"><div class="surname">Axmann</div> <div class="given-names">I.M.</div></div> <div class="name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div> <div class="name"><div class="surname">Wilde</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, <div class="article-title">An internal antisense RNA regulates expression of the photosynthesis gene isiA</div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="volume">103</div>, <div class="fpage">7054</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20internal%20antisense%20RNA%20regulates%20expression%20of%20the%20photosynthesis%20gene%20isiA&amp;author=U.%20Duhring&amp;author=I.M.%20Axmann&amp;author=W.R.%20Hess&amp;author=A.%20Wilde&amp;publication_year=2006&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=103&amp;pages=7054-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0600927103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0600927103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0600927103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16636284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Faghihi</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Wahlestedt</div> <div class="given-names">C.</div></div></div> <div class="year">2009</div>, <div class="article-title">Regulatory roles of natural antisense transcripts</div>, <div class="source">Nat Rev Mol Cell Biol</div> , <div class="volume">10</div>, <div class="fpage">637</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20roles%20of%20natural%20antisense%20transcripts&amp;author=M.A.%20Faghihi&amp;author=C.%20Wahlestedt&amp;publication_year=2009&amp;journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;volume=10&amp;pages=637-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm2738" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm2738" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm2738"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19638999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Borsani</div> <div class="given-names">O.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Verslues</div> <div class="given-names">P.E.</div></div> <div class="name"><div class="surname">Sunkar</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div></div> <div class="year">2005</div>, <div class="article-title">Endogenous siRNAs derived from a pair of natural cis-antisense transcripts regulate salt tolerance in Arabidopsis</div>, <div class="source">Cell</div> , <div class="volume">123</div>, <div class="fpage">1279</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endogenous%20siRNAs%20derived%20from%20a%20pair%20of%20natural%20cis-antisense%20transcripts%20regulate%20salt%20tolerance%20in%20Arabidopsis&amp;author=O.%20Borsani&amp;author=J.%20Zhu&amp;author=P.E.%20Verslues&amp;author=R.%20Sunkar&amp;author=J.K.%20Zhu&amp;publication_year=2005&amp;journal=Cell&amp;volume=123&amp;pages=1279-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2005.11.035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2005.11.035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2005.11.035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16377568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ron</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Alandete Saez</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Eshed Williams</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Fletcher</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">McCormick</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, <div class="article-title">Proper regulation of a sperm-specific cis-nat-siRNA is essential for double fertilization in Arabidopsis</div>, <div class="source">Genes Dev</div> , <div class="volume">24</div>, <div class="fpage">1010</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proper%20regulation%20of%20a%20sperm-specific%20cis-nat-siRNA%20is%20essential%20for%20double%20fertilization%20in%20Arabidopsis&amp;author=M.%20Ron&amp;author=M.%20Alandete%20Saez&amp;author=L.%20Eshed%20Williams&amp;author=J.C.%20Fletcher&amp;author=S.%20McCormick&amp;publication_year=2010&amp;journal=Genes%20Dev&amp;volume=24&amp;pages=1010-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1882810" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1882810" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1882810"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20478994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Okamura</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Balla</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Martin</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Lai</div> <div class="given-names">E.C.</div></div></div> <div class="year">2008</div>, <div class="article-title">Two distinct mechanisms generate endogenous siRNAs from bidirectional transcription in <em>Drosophila melanogaster</em></div>, <div class="source">Nat Struct Mol Biol</div> , <div class="volume">15</div>, <div class="fpage">581</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20distinct%20mechanisms%20generate%20endogenous%20siRNAs%20from%20bidirectional%20transcription%20in%20Drosophila%20melanogaster&amp;author=K.%20Okamura&amp;author=S.%20Balla&amp;author=R.%20Martin&amp;author=N.%20Liu&amp;author=E.C.%20Lai&amp;publication_year=2008&amp;journal=Nat%20Struct%20Mol%20Biol&amp;volume=15&amp;pages=581-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1438" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1438" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1438"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18500351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Xia</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Lii</div> <div class="given-names">Y.E.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Genome-wide analysis of plant nat-siRNAs reveals insights into their distribution, biogenesis and function</div>, <div class="source">Genome Biol</div> , <div class="volume">13</div>, <div class="fpage">R20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20plant%20nat-siRNAs%20reveals%20insights%20into%20their%20distribution%2C%20biogenesis%20and%20function&amp;author=X.%20Zhang&amp;author=J.%20Xia&amp;author=Y.E.%20Lii&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-3-r20" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-3-r20" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-3-r20"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22439910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Carey</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Workman</div> <div class="given-names">J.L.</div></div></div> <div class="year">2007</div>, <div class="article-title">The role of chromatin during transcription</div>, <div class="source">Cell</div> , <div class="volume">128</div>, <div class="fpage">707</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20chromatin%20during%20transcription&amp;author=B.%20Li&amp;author=M.%20Carey&amp;author=J.L.%20Workman&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=707-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.01.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.01.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.01.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17320508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Oh</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Park</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">van Nocker</div> <div class="given-names">S.</div></div></div> <div class="year">2008</div>, <div class="article-title">Genic and global functions for Paf1C in chromatin modification and gene expression in Arabidopsis</div>, <div class="source">PLoS Genet</div> , <div class="volume">4</div>, <div class="fpage">e1000077</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20and%20global%20functions%20for%20Paf1C%20in%20chromatin%20modification%20and%20gene%20expression%20in%20Arabidopsis&amp;author=S.%20Oh&amp;author=S.%20Park&amp;author=S.%20van%20Nocker&amp;publication_year=2008&amp;journal=PLoS%20Genet&amp;volume=4&amp;pages=e1000077" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18725930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tanurdzic</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Vaughn</div> <div class="given-names">M.W.</div></div> <div class="name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Epigenomic consequences of immortalized plant cell suspension culture</div>, <div class="source">PLoS Biol</div> , <div class="volume">6</div>, <div class="fpage">2880</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenomic%20consequences%20of%20immortalized%20plant%20cell%20suspension%20culture&amp;author=M.%20Tanurdzic&amp;author=M.W.%20Vaughn&amp;author=H.%20Jiang&amp;publication_year=2008&amp;journal=PLoS%20Biol&amp;volume=6&amp;pages=2880-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0060302" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0060302" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0060302"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19071958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Charron</div> <div class="given-names">J.B.</div></div> <div class="name"><div class="surname">He</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Elling</div> <div class="given-names">A.A.</div></div> <div class="name"><div class="surname">Deng</div> <div class="given-names">X.W.</div></div></div> <div class="year">2009</div>, <div class="article-title">Dynamic landscapes of four histone modifications during deetiolation in Arabidopsis</div>, <div class="source">Plant Cell</div> , <div class="volume">21</div>, <div class="fpage">3732</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20landscapes%20of%20four%20histone%20modifications%20during%20deetiolation%20in%20Arabidopsis&amp;author=J.B.%20Charron&amp;author=H.%20He&amp;author=A.A.%20Elling&amp;author=X.W.%20Deng&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=3732-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.066845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.066845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.066845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20008096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">He</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Charron</div> <div class="given-names">J.B.</div></div> <div class="name"><div class="surname">Elling</div> <div class="given-names">A.A.</div></div> <div class="name"><div class="surname">Deng</div> <div class="given-names">X.W.</div></div></div> <div class="year">2010</div>, <div class="article-title">Genome-wide profiling of histone H3 lysine 9 acetylation and dimethylation in Arabidopsis reveals correlation between multiple histone marks and gene expression</div>, <div class="source">Plant Mol Biol</div> , <div class="volume">72</div>, <div class="fpage">585</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20profiling%20of%20histone%20H3%20lysine%209%20acetylation%20and%20dimethylation%20in%20Arabidopsis%20reveals%20correlation%20between%20multiple%20histone%20marks%20and%20gene%20expression&amp;author=J.%20Zhou&amp;author=X.%20Wang&amp;author=K.%20He&amp;author=J.B.%20Charron&amp;author=A.A.%20Elling&amp;author=X.W.%20Deng&amp;publication_year=2010&amp;journal=Plant%20Mol%20Biol&amp;volume=72&amp;pages=585-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-009-9594-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-009-9594-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-009-9594-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20054610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gehring</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, <div class="article-title">DNA methylation dynamics in plant genomes</div>, <div class="source">Biochim Biophys Acta</div> , <div class="volume">1769</div>, <div class="fpage">276</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20dynamics%20in%20plant%20genomes&amp;author=M.%20Gehring&amp;author=S.%20Henikoff&amp;publication_year=2007&amp;journal=Biochim%20Biophys%20Acta&amp;volume=1769&amp;pages=276-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbaexp.2007.01.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbaexp.2007.01.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbaexp.2007.01.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17341434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Kimatu</div> <div class="given-names">J.N.</div></div> <div class="name"><div class="surname">Xu</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div></div> <div class="year">2010</div>, <div class="article-title">DNA cytosine methylation in plant development</div>, <div class="source">J Genet Genomics</div> , <div class="volume">37</div>, <div class="fpage">1</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20cytosine%20methylation%20in%20plant%20development&amp;author=M.%20Zhang&amp;author=J.N.%20Kimatu&amp;author=K.%20Xu&amp;author=B.%20Liu&amp;publication_year=2010&amp;journal=J%20Genet%20Genomics&amp;volume=37&amp;pages=1-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1673-8527(09)60020-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1673-8527(09)60020-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1673-8527(09)60020-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20171573" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tufarelli</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Stanley</div> <div class="given-names">J.A.</div></div> <div class="name"><div class="surname">Garrick</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">Transcription of antisense RNA leading to gene silencing and methylation as a novel cause of human genetic disease</div>, <div class="source">Nat Genet</div> , <div class="volume">34</div>, <div class="fpage">157</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20of%20antisense%20RNA%20leading%20to%20gene%20silencing%20and%20methylation%20as%20a%20novel%20cause%20of%20human%20genetic%20disease&amp;author=C.%20Tufarelli&amp;author=J.A.%20Stanley&amp;author=D.%20Garrick&amp;publication_year=2003&amp;journal=Nat%20Genet&amp;volume=34&amp;pages=157-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12730694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Gius</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Onyango</div> <div class="given-names">P.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">Epigenetic silencing of tumour suppressor gene p15 by its antisense RNA</div>, <div class="source">Nature</div> , <div class="volume">451</div>, <div class="fpage">202</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20silencing%20of%20tumour%20suppressor%20gene%20p15%20by%20its%20antisense%20RNA&amp;author=W.%20Yu&amp;author=D.%20Gius&amp;author=P.%20Onyango&amp;publication_year=2008&amp;journal=Nature&amp;volume=451&amp;pages=202-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18185590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Morris</div> <div class="given-names">K.V.</div></div> <div class="name"><div class="surname">Santoso</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Turner</div> <div class="given-names">A.M.</div></div> <div class="name"><div class="surname">Pastori</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Hawkins</div> <div class="given-names">P.G.</div></div></div> <div class="year">2008</div>, <div class="article-title">Bidirectional transcription directs both transcriptional gene activation and suppression in human cells</div>, <div class="source">PLoS Genet</div> , <div class="volume">4</div>, <div class="fpage">e1000258</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bidirectional%20transcription%20directs%20both%20transcriptional%20gene%20activation%20and%20suppression%20in%20human%20cells&amp;author=K.V.%20Morris&amp;author=S.%20Santoso&amp;author=A.M.%20Turner&amp;author=C.%20Pastori&amp;author=P.G.%20Hawkins&amp;publication_year=2008&amp;journal=PLoS%20Genet&amp;volume=4&amp;pages=e1000258" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000258" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000258" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000258"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19008947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Conley</div> <div class="given-names">A.B.</div></div> <div class="name"><div class="surname">Jordan</div> <div class="given-names">I.K.</div></div></div> <div class="year">2012</div>, <div class="article-title">Epigenetic regulation of human cis-natural antisense transcripts</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">40</div>, <div class="fpage">1438</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20human%20cis-natural%20antisense%20transcripts&amp;author=A.B.%20Conley&amp;author=I.K.%20Jordan&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=1438-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22371288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Heo</div> <div class="given-names">J.B.</div></div> <div class="name"><div class="surname">Sung</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, <div class="article-title">Encoding memory of winter by noncoding RNAs</div>, <div class="source">Epigenetics</div> , <div class="volume">6</div>, <div class="fpage">544</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Encoding%20memory%20of%20winter%20by%20noncoding%20RNAs&amp;author=J.B.%20Heo&amp;author=S.%20Sung&amp;publication_year=2011&amp;journal=Epigenetics&amp;volume=6&amp;pages=544-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/epi.6.5.15235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fepi.6.5.15235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fepi.6.5.15235"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21406964" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Swiezewski</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Magusin</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Dean</div> <div class="given-names">C.</div></div></div> <div class="year">2009</div>, <div class="article-title">Cold-induced silencing by long antisense transcripts of an Arabidopsis Polycomb target</div>, <div class="source">Nature</div> , <div class="volume">462</div>, <div class="fpage">799</div>–<div class="lpage">802</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cold-induced%20silencing%20by%20long%20antisense%20transcripts%20of%20an%20Arabidopsis%20Polycomb%20target&amp;author=S.%20Swiezewski&amp;author=F.%20Liu&amp;author=A.%20Magusin&amp;author=C.%20Dean&amp;publication_year=2009&amp;journal=Nature&amp;volume=462&amp;pages=799-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08618" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08618"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20010688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Goff</div> <div class="given-names">L.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>, <div class="source">Nat Protoc</div> , <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat%20Protoc&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div> <div class="name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat Biotechnol</div> , <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat%20Biotechnol&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Park</div> <div class="given-names">H.J.</div></div> <div class="name"><div class="surname">Dasari</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Kocher</div> <div class="given-names">J.P.</div></div> <div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2013</div>, <div class="article-title">CPAT: Coding-Potential Assessment Tool using an alignment-free logistic regression model</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">41</div>, <div class="fpage">e74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CPAT%3A%20Coding-Potential%20Assessment%20Tool%20using%20an%20alignment-free%20logistic%20regression%20model&amp;author=L.%20Wang&amp;author=H.J.%20Park&amp;author=S.%20Dasari&amp;author=S.%20Wang&amp;author=J.P.%20Kocher&amp;author=W.%20Li&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23335781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Camacho</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Coulouris</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Avagyan</div> <div class="given-names">V.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">BLAST+: architecture and applications</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">421</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=421" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-421"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Markham</div> <div class="given-names">N.R.</div></div> <div class="name"><div class="surname">Zuker</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, <div class="article-title">DINAMelt web server for nucleic acid melting prediction</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">33</div>, <div class="fpage">W577</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DINAMelt%20web%20server%20for%20nucleic%20acid%20melting%20prediction&amp;author=N.R.%20Markham&amp;author=M.%20Zuker&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W577-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki591" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki591" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki591"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kin</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Terai</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">fRNAdb: a platform for mining/annotating functional RNA candidates from non-coding RNA sequences</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">35</div>, <div class="fpage">D145</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=fRNAdb%3A%20a%20platform%20for%20mining%2Fannotating%20functional%20RNA%20candidates%20from%20non-coding%20RNA%20sequences&amp;author=T.%20Kin&amp;author=K.%20Yamada&amp;author=G.%20Terai&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D145-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl837" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl837" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl837"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17099231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Roudier</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Ahmed</div> <div class="given-names">I.</div></div> <div class="name"><div class="surname">Berard</div> <div class="given-names">C.</div></div>et al.  </div> <div class="year">2011</div>, <div class="article-title">Integrative epigenomic mapping defines four main chromatin states in Arabidopsis</div>, <div class="source">EMBO J</div> , <div class="volume">30</div>, <div class="fpage">1928</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20epigenomic%20mapping%20defines%20four%20main%20chromatin%20states%20in%20Arabidopsis&amp;author=F.%20Roudier&amp;author=I.%20Ahmed&amp;author=C.%20Berard&amp;publication_year=2011&amp;journal=EMBO%20J&amp;volume=30&amp;pages=1928-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/emboj.2011.103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Femboj.2011.103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Femboj.2011.103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21487388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Berard</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Seifert</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Mary-Huard</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Martin-Magniette</div> <div class="given-names">M.L.</div></div></div> <div class="year">2013</div>, <div class="article-title">MultiChIPmixHMM: an R package for ChIP-chip data analysis modeling spatial dependencies and multiple replicates</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">271</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MultiChIPmixHMM%3A%20an%20R%20package%20for%20ChIP-chip%20data%20analysis%20modeling%20spatial%20dependencies%20and%20multiple%20replicates&amp;author=C.%20Berard&amp;author=M.%20Seifert&amp;author=T.%20Mary-Huard&amp;author=M.L.%20Martin-Magniette&amp;publication_year=2013&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=271" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-14-271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-14-271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24015679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ko</div> <div class="given-names">Y.A.</div></div> <div class="name"><div class="surname">Mohtat</div> <div class="given-names">D.</div></div> <div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Cytosine methylation changes in enhancer regions of core pro-fibrotic genes characterize kidney fibrosis development</div>, <div class="source">Genome Biol</div> , <div class="volume">14</div>, <div class="fpage">R108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytosine%20methylation%20changes%20in%20enhancer%20regions%20of%20core%20pro-fibrotic%20genes%20characterize%20kidney%20fibrosis%20development&amp;author=Y.A.%20Ko&amp;author=D.%20Mohtat&amp;author=M.%20Suzuki&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-10-r108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-10-r108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-10-r108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24098934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rinn</div> <div class="given-names">J.L.</div></div> <div class="name"><div class="surname">Kertesz</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">J.K.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Functional demarcation of active and silent chromatin domains in human HOX loci by noncoding RNAs</div>, <div class="source">Cell</div> , <div class="volume">129</div>, <div class="fpage">1311</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20demarcation%20of%20active%20and%20silent%20chromatin%20domains%20in%20human%20HOX%20loci%20by%20noncoding%20RNAs&amp;author=J.L.%20Rinn&amp;author=M.%20Kertesz&amp;author=J.K.%20Wang&amp;publication_year=2007&amp;journal=Cell&amp;volume=129&amp;pages=1311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.05.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.05.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.05.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17604720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Sunkar</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Jin</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div></div> <div class="year">2009</div>, <div class="article-title">Genome-wide identification and analysis of small RNAs originated from natural antisense transcripts in <em>Oryza sativa</em></div>, <div class="source">Genome Res</div> , <div class="volume">19</div>, <div class="fpage">70</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20and%20analysis%20of%20small%20RNAs%20originated%20from%20natural%20antisense%20transcripts%20in%20Oryza%20sativa&amp;author=X.%20Zhou&amp;author=R.%20Sunkar&amp;author=H.%20Jin&amp;author=J.K.%20Zhu&amp;author=W.%20Zhang&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=70-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.084806.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.084806.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.084806.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18971307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brodersen</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Voinnet</div> <div class="given-names">O.</div></div></div> <div class="year">2006</div>, <div class="article-title">The diversity of RNA silencing pathways in plants</div>, <div class="source">Trends Genet</div> , <div class="volume">22</div>, <div class="fpage">268</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20diversity%20of%20RNA%20silencing%20pathways%20in%20plants&amp;author=P.%20Brodersen&amp;author=O.%20Voinnet&amp;publication_year=2006&amp;journal=Trends%20Genet&amp;volume=22&amp;pages=268-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2006.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2006.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2006.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16567016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">Q.</div></div> <div class="name"><div class="surname">Feng</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div></div> <div class="year">2009</div>, <div class="article-title">Dicer-like (DCL) proteins in plants</div>, <div class="source">Funct Integr Genomics</div> , <div class="volume">9</div>, <div class="fpage">277</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dicer-like%20(DCL)%20proteins%20in%20plants&amp;author=Q.%20Liu&amp;author=Y.%20Feng&amp;author=Z.%20Zhu&amp;publication_year=2009&amp;journal=Funct%20Integr%20Genomics&amp;volume=9&amp;pages=277-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-009-0111-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-009-0111-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-009-0111-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19221817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Allen</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Wilken</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Carrington</div> <div class="given-names">J.C.</div></div></div> <div class="year">2005</div>, <div class="article-title">DICER-LIKE 4 functions in trans-acting small interfering RNA biogenesis and vegetative phase change in <em>Arabidopsis thaliana</em></div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="volume">102</div>, <div class="fpage">12984</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DICER-LIKE%204%20functions%20in%20trans-acting%20small%20interfering%20RNA%20biogenesis%20and%20vegetative%20phase%20change%20in%20Arabidopsis%20thaliana&amp;author=Z.%20Xie&amp;author=E.%20Allen&amp;author=A.%20Wilken&amp;author=J.C.%20Carrington&amp;publication_year=2005&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=102&amp;pages=12984-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506426102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506426102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506426102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16129836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Lu</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Cui</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Cao</div> <div class="given-names">X.</div></div></div> <div class="year">2010</div>, <div class="article-title">Histone methylation in higher plants</div>, <div class="source">Annu Rev Plant Biol</div> , <div class="volume">61</div>, <div class="fpage">395</div>–<div class="lpage">420</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Histone%20methylation%20in%20higher%20plants&amp;author=C.%20Liu&amp;author=F.%20Lu&amp;author=X.%20Cui&amp;author=X.%20Cao&amp;publication_year=2010&amp;journal=Annu%20Rev%20Plant%20Biol&amp;volume=61&amp;pages=395-420" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.043008.091939" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.043008.091939" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.043008.091939"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20192747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Turck</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Roudier</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Farrona</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Arabidopsis TFL2/LHP1 specifically associates with genes marked by trimethylation of histone H3 lysine 27</div>, <div class="source">PLoS Genet</div> , <div class="volume">3</div>, <div class="fpage">e86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20TFL2%2FLHP1%20specifically%20associates%20with%20genes%20marked%20by%20trimethylation%20of%20histone%20H3%20lysine%2027&amp;author=F.%20Turck&amp;author=F.%20Roudier&amp;author=S.%20Farrona&amp;publication_year=2007&amp;journal=PLoS%20Genet&amp;volume=3&amp;pages=e86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030086" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030086" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030086"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17542647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Bernatavichute</div> <div class="given-names">Y.V.</div></div> <div class="name"><div class="surname">Cokus</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Pellegrini</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Jacobsen</div> <div class="given-names">S.E.</div></div></div> <div class="year">2009</div>, <div class="article-title">Genome-wide analysis of mono-, di- and trimethylation of histone H3 lysine 4 in <em>Arabidopsis thaliana</em></div>, <div class="source">Genome Biol</div> , <div class="volume">10</div>, <div class="fpage">R62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20mono-%2C%20di-%20and%20trimethylation%20of%20histone%20H3%20lysine%204%20in%20Arabidopsis%20thaliana&amp;author=X.%20Zhang&amp;author=Y.V.%20Bernatavichute&amp;author=S.%20Cokus&amp;author=M.%20Pellegrini&amp;author=S.E.%20Jacobsen&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-6-r62" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-6-r62" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-6-r62"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19508735" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sridhar</div> <div class="given-names">V.V.</div></div> <div class="name"><div class="surname">Kapoor</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Control of DNA methylation and heterochromatic silencing by histone H2B deubiquitination</div>, <div class="source">Nature</div> , <div class="volume">447</div>, <div class="fpage">735</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20DNA%20methylation%20and%20heterochromatic%20silencing%20by%20histone%20H2B%20deubiquitination&amp;author=V.V.%20Sridhar&amp;author=A.%20Kapoor&amp;author=K.%20Zhang&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=735-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17554311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lisch</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, <div class="article-title">Epigenetic regulation of transposable elements in plants</div>, <div class="source">Annu Rev Plant Biol</div> , <div class="volume">60</div>, <div class="fpage">43</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20transposable%20elements%20in%20plants&amp;author=D.%20Lisch&amp;publication_year=2009&amp;journal=Annu%20Rev%20Plant%20Biol&amp;volume=60&amp;pages=43-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092744" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092744" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092744"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19007329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Xie</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Hong</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">DNA hypomethylation within specific transposable element families associates with tissue-specific enhancer landscape</div>, <div class="source">Nat Genet</div> , <div class="volume">45</div>, <div class="fpage">836</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20hypomethylation%20within%20specific%20transposable%20element%20families%20associates%20with%20tissue-specific%20enhancer%20landscape&amp;author=M.%20Xie&amp;author=C.%20Hong&amp;author=B.%20Zhang&amp;publication_year=2013&amp;journal=Nat%20Genet&amp;volume=45&amp;pages=836-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2649" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2649" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2649"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23708189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhan</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Lukens</div> <div class="given-names">L.</div></div></div> <div class="year">2013</div>, <div class="article-title">Protein-coding cis-natural antisense transcripts have high and broad expression in Arabidopsis</div>, <div class="source">Plant Physiol</div> , <div class="volume">161</div>, <div class="fpage">2171</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein-coding%20cis-natural%20antisense%20transcripts%20have%20high%20and%20broad%20expression%20in%20Arabidopsis&amp;author=S.%20Zhan&amp;author=L.%20Lukens&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=161&amp;pages=2171-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.212100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.212100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.212100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23457227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV008C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV008C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Sun</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div> <div class="name"><div class="surname">Carmichael</div> <div class="given-names">G.G.</div></div> <div class="name"><div class="surname">Rowley</div> <div class="given-names">J.D.</div></div></div> <div class="year">2005</div>, <div class="article-title">Genome-wide analysis of coordinate expression and evolution of human cis-encoded sense-antisense transcripts</div>, <div class="source">Trends Genet</div> , <div class="volume">21</div>, <div class="fpage">326</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20coordinate%20expression%20and%20evolution%20of%20human%20cis-encoded%20sense-antisense%20transcripts&amp;author=J.%20Chen&amp;author=M.%20Sun&amp;author=L.D.%20Hurst&amp;author=G.G.%20Carmichael&amp;author=J.D.%20Rowley&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=326-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15922830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=591590&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_table3.xls?Expires=2147483647&amp;Signature=LyIANco7yWfBRTIoCF46VwYoE2HW~ZQP4TucbcT7MWJkp8WiZ-r68vDOTtPMY6L0sxAR-wQAUG-otOcLlyHn3nvDg7EC2fR7OaM182V1cmtX5F27mW8MU5Ieci~-md1EOAjX37QvfSsB-i9lHyiGr5pFOqijgBOHwLH83n4Ep3Cz2S8qoQP-gHfp8xKw366ZXxzCYmOWYv2GFQxwMidxOyqjsvWL9JAnHXxLLybjhPKVHGqChBsW6XvjkNuLTAB7jhdE0U2JtHem-nuDCqSeQ5Y4aDaK9JCL9b6nFwbkea1T0NgowCtmsku5dxAU8FwscaDDgHmxdzoPlZdOBatPgw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_table2.xls?Expires=2147483647&amp;Signature=BLaQQJBbiFgXawWtTCPO1m9-PNwWH-B9CDXCOb3HEvXz35PBxm6sP3XgQMOf~dGLUQXlsSJpBUl2E4ft-L-cLYtideDZDcliyP7zpMXfxU0Hb60bnqF9KG~PNqY8vApqAa45WNv5sS72PD7H3tzW-wFwRp7kWBfDUO2J5dW-2N3xgWkoScEmCbCks6ANo82uibvihus-IcL3C6eGtGajTPysoRnaydfY5~KvIm~BluWaaY9kbPycC2h79Drh5QLIu56SAYr3Q9apJiCD49FljEyhTw5icMT8En57pzPc3Bs8XECAmaiod3tr8a6dD2d~t81VhHlfpORjHqRtmVjw6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_table1.xls?Expires=2147483647&amp;Signature=MrYE~TvM9cMesm8sT9pWaH7tqua5R9qC5TFh881AEo~158zKGVHK~~dAj0YJ6frbZz4Aj5ZPdxelBCY7rPEqHgp5yMR6uGTSo2J8bWTaSp9O0Iju-o3edW5FrAGnOlW60CJNO6JnECD5ZfJaThsBdCxYNtV3Ywlw6mYkCqTk6Ls7jI287LmGcpr-99gFEWcqxHX-AgywzfY1f4d-FY4X7AssRAoXJ7XwIFz5wpWnVQh5D6L6DhIMmJBnj1ay0CGvlaVDUHBjjcUgbK8YRj5IegoNKshrnTgGVLVl06DBDWTTRjwijcMQuABnjgxNHMja~ur5Cz~6sHJWjBs4WwXakA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_fig4.png?Expires=2147483647&amp;Signature=z7XLZOErgMbq13cNmWZ-x1FXuTLB6g-fa68dcW1rYBmvl-h3H8WuPhdPfEYuszSIR6PGhxZoMLD25v4DW0RuGa9RK8bciXmQlNnd8OQk-rA3kCfyQecgHkxG6NzE~bvyKGcs8IVSyi7Bxycdktise5fCQyL7VjLd-EwPRtNE~TEAWF6tWsKtEQLVAHe6Vrmq9C2nJCuQ1nUp5eyUIcmvozO9M7hGeSqiWssA7-OwSv7mL21f6ovGHTAx9ZkBE7D1jQNV0s0cfAN~JBeCHaFUzEYRmlWPspEFw63Fgi-DkUt6QGD4ZEP--TdtP3Tze-AZZAnAQpTKPLbLwvmDY7nUBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_fig3.png?Expires=2147483647&amp;Signature=lNJ7UJcIgrQLsbV~MYxyTtqnUn5fcPK5Dur~yWAoljmQdn5KL7fKH90r7EQitbkPr3QurY4P3ge1FzPjkMbv9uvkJhY55GftZdqdPbckYQgr42laNetXo7r8CNrvFiSCP3uUSEx0wLcndBVNqgupMlDTIAscQ16ORnSr1kf1DNj5a1bc5oy3XWj5dk1gFwER~EGIUjYFor5p2ZlXT3FMtCRpuaS8Kj8soRSDwIZzc3qIkD7WJHrGaT0MyR8CWT3NPk95KNM-By2oS~bTuVefngk5Kje3F~RfjRRVz4dNRQ7uUsH~Xx9GOkTZVnNUtbOC~LdmsipcMC2GZ47dGv-MCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_fig2.png?Expires=2147483647&amp;Signature=zkf9yYxb7hCrK8KHDg4FoRInEM5bYFH7muP-wG4a8ZoVGS1pDkc-XwoZRKzzLe8Uun2o8qy91bkqWQdpCYUpOdAo44ePvqVyTdcK-bbGRfPN-AizCtKZH3bbOwukaE0xEtl6D7bL2xtj2QCDX1hcyMmscu5~8usefEltBSrG1FZROScFBgIhdZPmvsnCw-MBTbYS0o5c5gCwfYdcuDao5m5WLRjLQivjw2dwOyCCJJZF1mCRyrao8ItC~T~eiypT3~9y6lFqKyYRzVPVBELruysDEoydXTtebMvPybFSBq76Ad8kZh8GKHQRHXQEuGLCtCUAji3oRX2BIPj6Vzw0WA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv008/2/dsv008-dsv008supp_fig1.png?Expires=2147483647&amp;Signature=nUTsADgI~66XgAI8IZnCmbiRqzw6SnjJFkQ8ia8ueEWtrMzsgiDMnweazTC3NzKecVMjXo9x-Sf75M8MmIbailEh5v8NhljiSfun3s5y-ftq1R5akp7f6~DK4jszMWY8jzBefaPy7qruBul6y6j6C9kPmNa3ZyPzAuWQ4Vvaw8LRVtSVKlITcX~2H~GDOVwXP99hD064h4suM71PNA7GnAyWIVnnTvtP2eosjapMhvPcUsqFZUlhR5~2yrhenvAdyXa0hTK6bZxRfwvdAex61OjNlYeupvoyrh52oHVpg54cuIX9IIDANe1cOv7DlZ9QN7LAi08FxRDRnuyo2n5o6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">574</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="591590"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f3%2f233%2f591590">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000356658900005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F3%2F233%2F591590" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/591590'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide view of natural antisense transcripts in Arabidopsis thaliana" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f233%2f591590">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

