

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/1/29/2462430","@type":"ScholarlyArticle","name":"Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage ( Brassica oleracea  L.)","datePublished":"2015-11-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2016-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv034","keywords":["cabbage","genotyping-by-sequencing","genetic linkage map","clubroot","QTL"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/1/29/2462430","author":[{"name":"Lee, Jonghoon","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Izzah, Nur Kholilatul","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea    2  Indonesian Research Institute for Industrial and Beverage Crops (IRIIBC) ,  Pakuwon, Sukabumi ,  Indonesia","@type":"Person"},{"name":"Choi, Beom-Soon","affiliation":"3  Phyzen Genomics Institute ,  Seoul 151-836 ,  Republic of Korea","@type":"Person"},{"name":"Joh, Ho Jun","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Lee, Sang-Choon","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Perumal, Sampath","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea","@type":"Person"},{"name":"Seo, Joodeok","affiliation":"4  Joeun Seed ,  Goesan-Gun, Chungcheongbuk-Do 367-833 ,  Republic of Korea","@type":"Person"},{"name":"Ahn, Kyounggu","affiliation":"4  Joeun Seed ,  Goesan-Gun, Chungcheongbuk-Do 367-833 ,  Republic of Korea","@type":"Person"},{"name":"Jo, Eun Ju","affiliation":"5  Research Center for Biobased Chemistry ,  Korea Research Institute of Chemical Technology ,  Daejeon 305-600 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Gyung Ja","affiliation":"5  Research Center for Biobased Chemistry ,  Korea Research Institute of Chemical Technology ,  Daejeon 305-600 ,  Republic of Korea","@type":"Person"},{"name":"Nou, Ill-Sup","affiliation":"6  Department of Horticulture ,  Sunchon National University ,  Suncheon 540-950 ,  Republic of Korea","@type":"Person"},{"name":"Yu, Yeisoo","affiliation":"3  Phyzen Genomics Institute ,  Seoul 151-836 ,  Republic of Korea","@type":"Person"},{"name":"Yang, Tae-Jin","affiliation":"1  Department of Plant Science ,  Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University ,  Seoul 151-921 ,  Republic of Korea    7  Crop Biotechnology Institute/GreenBio Science and Technology, Seoul National University ,  Pyeongchang 232-916 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Clubroot is a devastating disease caused by Plasmodiophora brassicae and results in severe losses of yield and quality in Brassica crops. Many clubro","pageStart":"29","pageEnd":"41","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage ( Brassica oleracea  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage ( Brassica oleracea  L.)" />
    <meta property="og:description" content="Abstract.  Clubroot is a devastating disease caused by Plasmodiophora brassicae and results in severe losses of yield and quality in Brassica crops. Many clubro" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/1/29/2462430" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-11-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Lee, Jonghoon" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Izzah, Nur Kholilatul" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author_institution" content="Indonesian Research Institute for Industrial and Beverage Crops (IRIIBC), Pakuwon, Sukabumi, Indonesia" /><meta name="citation_author" content="Choi, Beom-Soon" /><meta name="citation_author_institution" content="Phyzen Genomics Institute, Seoul 151-836, Republic of Korea" /><meta name="citation_author" content="Joh, Ho Jun" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Lee, Sang-Choon" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Perumal, Sampath" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author" content="Seo, Joodeok" /><meta name="citation_author_institution" content="Joeun Seed, Goesan-Gun, Chungcheongbuk-Do 367-833, Republic of Korea" /><meta name="citation_author" content="Ahn, Kyounggu" /><meta name="citation_author_institution" content="Joeun Seed, Goesan-Gun, Chungcheongbuk-Do 367-833, Republic of Korea" /><meta name="citation_author" content="Jo, Eun Ju" /><meta name="citation_author_institution" content="Research Center for Biobased Chemistry, Korea Research Institute of Chemical Technology, Daejeon 305-600, Republic of Korea" /><meta name="citation_author" content="Choi, Gyung Ja" /><meta name="citation_author_institution" content="Research Center for Biobased Chemistry, Korea Research Institute of Chemical Technology, Daejeon 305-600, Republic of Korea" /><meta name="citation_author" content="Nou, Ill-Sup" /><meta name="citation_author_institution" content="Department of Horticulture, Sunchon National University, Suncheon 540-950, Republic of Korea" /><meta name="citation_author" content="Yu, Yeisoo" /><meta name="citation_author_institution" content="Phyzen Genomics Institute, Seoul 151-836, Republic of Korea" /><meta name="citation_author" content="Yang, Tae-Jin" /><meta name="citation_author_institution" content="Department of Plant Science, Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University, Seoul 151-921, Republic of Korea" /><meta name="citation_author_institution" content="Crop Biotechnology Institute/GreenBio Science and Technology, Seoul National University, Pyeongchang 232-916, Republic of Korea" /><meta name="citation_title" content="Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (Brassica oleracea L.)" /><meta name="citation_firstpage" content="29" /><meta name="citation_lastpage" content="41" /><meta name="citation_doi" content="10.1093/dnares/dsv034" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2016/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796" /><meta name="citation_reference" content="citation_title=Crop genome sequencing: lessons and rationales;  citation_journal_title=Trends Plant Sci.;  citation_year=2011;  citation_volume=16;  citation_pages=77-88; " /><meta name="citation_reference" content="citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e19379" /><meta name="citation_reference" content="citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing;  citation_journal_title=Genes Genomes Genetics;  citation_year=2013;  citation_volume=3;  citation_pages=1903-26; " /><meta name="citation_reference" content="citation_title=Comprehensive genotyping of the USA national maize inbred seed bank;  citation_journal_title=Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R55" /><meta name="citation_reference" content="citation_title=An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e54603" /><meta name="citation_reference" content="citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e32253" /><meta name="citation_reference" content="citation_title=Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2013;  citation_volume=126;  citation_pages=2699-716; " /><meta name="citation_reference" content="citation_title=Application of genotyping-by-sequencing on semiconductor sequencing platforms: a comparison of genetic and reference-based marker ordering in barley;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e76925" /><meta name="citation_reference" content="citation_title=Genome analysis in Brassica with special reference to the experimental formation of B. napus and peculiar mode of fertilization; citation_author=U, N.;  citation_journal_title=J. Jpn Bot.;  citation_year=1935;  citation_volume=7;  citation_pages=389-452; " /><meta name="citation_reference" content="citation_title=Construction and analysis of a high-density genetic linkage map in cabbage (Brassica oleracea L. var. capitata);  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=523" /><meta name="citation_reference" content="citation_title=The Brassica oleracea genome reveals the asymmetrical evolution of polyploid genomes;  citation_journal_title=Nat. Commun.;  citation_year=2014;  citation_volume=5;  citation_pages=3930" /><meta name="citation_reference" content="citation_title=Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea;  citation_journal_title=Genome Biol.;  citation_year=2014;  citation_volume=15;  citation_pages=R77" /><meta name="citation_reference" content="citation_title=Variation in Plasmodiophora brassicae and resistance to clubroot disease in brassicas and allied crops-a critical review;  citation_journal_title=Plant Breed. Abst.;  citation_year=1980;  citation_volume=50;  citation_pages=91-104; " /><meta name="citation_reference" content="citation_title=The susceptibility of Brassica oleracea cultivars to Plasmodiophora brassicae (clubroot);  citation_journal_title=Plant Pathol.;  citation_year=1986;  citation_volume=35;  citation_pages=101-7; " /><meta name="citation_reference" content="citation_title=Plasmodiophora brassicae: aspects of pathogenesis and resistance in Brassica oleracea;  citation_journal_title=Euphytica;  citation_year=1995;  citation_volume=83;  citation_pages=139-46; " /><meta name="citation_reference" content="citation_title=Genetics of clubroot resistance in Brassica species;  citation_journal_title=J. Plant Growth Regul.;  citation_year=2009;  citation_volume=28;  citation_pages=252-64; " /><meta name="citation_reference" content="citation_title=A novel locus for clubroot resistance in Brassica rapa and its linkage markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=639-43; " /><meta name="citation_reference" content="citation_title=SCAR and CAPS mapping of CRb, a gene conferring resistance to Plasmodiophora brassicae in Chinese cabbage (Brassica rapa ssp. pekinensis);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1458-1465; " /><meta name="citation_reference" content="citation_title=Simple sequence repeat-based comparative genomics between Brassica rapa and Arabidopsis thaliana: the genetic origin of clubroot resistance;  citation_journal_title=Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=309-19; " /><meta name="citation_reference" content="citation_title=Mapping of isolate-specific QTLs for clubroot resistance in Chinese cabbage (Brassica rapa L. ssp. pekinensis);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2008;  citation_volume=117;  citation_pages=759-67; " /><meta name="citation_reference" content="citation_title=Fine mapping of the clubroot resistance gene, Crr3, in Brassica rapa;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2006;  citation_volume=114;  citation_pages=81-91; " /><meta name="citation_reference" content="citation_title=Fine mapping of the clubroot resistance gene CRb and development of a useful selectable marker in Brassica rapa;  citation_journal_title=Breed. Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=116" /><meta name="citation_reference" content="citation_title=Fine mapping a clubroot resistance locus in Chinese cabbage;  citation_journal_title=J. Am. Soc. Hort. Sci.;  citation_year=2014;  citation_volume=139;  citation_pages=247-52; " /><meta name="citation_reference" content="citation_title=Fine genetic and physical mapping of the CRb gene conferring resistance to clubroot disease in Brassica rapa;  citation_journal_title=Mol. Breed.;  citation_year=2014;  citation_volume=34;  citation_pages=1173-83; " /><meta name="citation_reference" content="citation_title=The exploitation of genetic resources of Brassica oleracea in breeding for resistance to clubroot (Plasmodiophora brassicae);  citation_journal_title=Euphytica;  citation_year=1989;  citation_volume=42;  citation_pages=215-26; " /><meta name="citation_reference" content="citation_title=RAPD markers linked to a clubroot-resistance locus in Brassica rapa L;  citation_journal_title=Euphytica;  citation_year=1997;  citation_volume=98;  citation_pages=149-54; " /><meta name="citation_reference" content="citation_title=Conversion of an AFLP marker linked to clubroot resistance gene in Chinese cabbage into a SCAR marker;  citation_journal_title=Hort. Environ. Biotechnol.;  citation_year=2002;  citation_volume=43;  citation_pages=653-9; " /><meta name="citation_reference" content="citation_title=Identification of Novel QTLs for Isolate-specific partial resistance to Plasmodiophora brassicae in Brassica rapa;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e85307" /><meta name="citation_reference" content="citation_title=Identification of QTLs that control clubroot resistance in Brassica oleracea and comparative analysis of clubroot resistance genes between B. rapa and B. oleracea;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2010;  citation_volume=120;  citation_pages=1335-46; " /><meta name="citation_reference" content="citation_title=A modified protocol for rapid DNA isolation from plant tissues using cetyltrimethylammonium bromide;  citation_journal_title=Nat. Protocols;  citation_year=2006;  citation_volume=1;  citation_pages=2320-5; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP identification and QTL mapping for black rot resistance in cabbage;  citation_journal_title=BMC Plant Biol.;  citation_year=2015;  citation_volume=15;  citation_pages=32" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of two parental lines of cabbage (Brassica oleracea L. var. capitata L.) and construction of an EST-based genetic map;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=149" /><meta name="citation_reference" content="citation_title=Efficient large-scale development of microsatellites for marker and mapping applications in Brassica crop species;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=108;  citation_pages=1103-12; " /><meta name="citation_reference" content="citation_title=Construction of an oilseed rape (Brassica napus L.) genetic map with SSR markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1514-23; " /><meta name="citation_reference" content="citation_title=The reference genetic linkage map for the multinational Brassica rapa genome sequencing project;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=777-92; " /><meta name="citation_reference" content="citation_title=Database derived microsatellite markers (SSRs) for cultivar differentiation in Brassica oleracea;  citation_journal_title=Genet. Resourc. Crop Evol.;  citation_year=2007;  citation_volume=54;  citation_pages=1717-25; " /><meta name="citation_reference" content="citation_title=Development and genetic mapping of microsatellite markers from genome survey sequences in Brassica napus;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1121-31; " /><meta name="citation_reference" content="citation_title=Characterization of a new high copy Stowaway family MITE, BRAMI-1 in Brassica genome;  citation_journal_title=BMC Plant Biol.;  citation_year=2013;  citation_volume=13;  citation_pages=56" /><meta name="citation_reference" content="citation_title=Genome-wide comparative analysis of 20 miniature inverted-repeat transposable element families in Brassica rapa and B. oleracea;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e94499" /><meta name="citation_reference" content="citation_title=Resistance of cultivars of Chinese cabbage to Plasmodiophora brassicae isolates of several races collected in Korea;  citation_journal_title=Kor. J. Hort. Sci. Technol.;  citation_year=2011;  citation_volume=29;  citation_pages=610-6; " /><meta name="citation_reference" content="citation_title=A system for the determination of races of Plasmodiophora brassicae that infect Cabbage and Rutabaga;  citation_journal_title=Phytopathology;  citation_year=1966;  citation_volume=56;  citation_pages=624-6; " /><meta name="citation_reference" content="citation_title=The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes;  citation_journal_title=Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=535-42; " /><meta name="citation_reference" content="citation_title=Linkage analysis of RFLP markers for clubroot resistance and pigmentation in Chinese cabbage (Brassica rapa ssp. pekinensis);  citation_journal_title=Euphytica;  citation_year=1998;  citation_volume=104;  citation_pages=79-86; " /><meta name="citation_reference" content="citation_title=Identification of two loci for resistance to clubroot (Plasmodiophora brassicae Woronin) in Brassica rapa L;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=997-1002; " /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa;  citation_journal_title=Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=Identification and characterization of Crr1a, a gene for resistance to clubroot disease (Plasmodiophora brassicae Woronin) in Brassica rapa L;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e54745" /><meta name="citation_reference" content="citation_title=The map-based sequence of the rice genome; citation_author=International Rice Genome Sequencing Project.;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=436;  citation_pages=793-800; " /><meta name="citation_reference" content="citation_title=Construction and utility of 10-kb libraries for efficient clone-gap closure for rice genome sequencing;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2003;  citation_volume=107;  citation_pages=652-60; " /><meta name="citation_reference" content="citation_title=Comparative genomics of Brassica oleracea and Arabidopsis thaliana reveal gene loss, fragmentation, and dispersal after polyploidy;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1348-59; " /><meta name="citation_reference" content="citation_title=Sequence-level analysis of the diploidization process in the triplicated FLOWERING LOCUS C region of Brassica rapa;  citation_journal_title=Plant Cell;  citation_year=2006;  citation_volume=18;  citation_pages=1339-47; " /><meta name="citation_reference" content="citation_title=Genotyping-by-sequencing for plant breeding and genetics;  citation_journal_title=Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=92-102; " /><meta name="citation_reference" content="citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=610-7; " /><meta name="citation_reference" content="citation_title=A genetic map for Brassica oleracea based on RFLP markers detected with expressed DNA sequences and mapping of resistance genes to race 2 of Plasmodiophora brassicae (Woronin);  citation_journal_title=Genome;  citation_year=1992;  citation_volume=35;  citation_pages=409-20; " /><meta name="citation_reference" content="citation_title=Deciphering the diploid ancestral genome of the mesohexaploid Brassica rapa;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=1541-54; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/1/29/17462984/dsv034.pdf" /><meta name="description" content="Abstract.  Clubroot is a devastating disease caused by Plasmodiophora brassicae and results in severe losses of yield and quality in Brassica crops. Many clubro" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/1/29/2462430" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/1/29/2462430" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/1/29/2462430");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv034';
    </script>
<title>
	Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (Brassica oleracea L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462430" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdOdATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVlLCBKb25naG9vbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgUGxhbnQgR2Vub21pY3MgYW5kIEJyZWVkaW5nIEluc3RpdHV0ZSwgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl6emFoLCBOdXIgS2hvbGlsYXR1bCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgUGxhbnQgU2NpZW5jZSwgUGxhbnQgR2Vub21pY3MgYW5kIEJyZWVkaW5nIEluc3RpdHV0ZSwgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBDb2xsZWdlIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBTZW91bCBOYXRpb25hbCBVbml2ZXJzaXR5LCBTZW91bCAxNTEtOTIxLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkluZG9uZXNpYW4gUmVzZWFyY2ggSW5zdGl0dXRlIGZvciBJbmR1c3RyaWFsIGFuZCBCZXZlcmFnZSBDcm9wcyAoSVJJSUJDKSwgUGFrdXdvbiwgU3VrYWJ1bWksIEluZG9uZXNpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNob2ksIEJlb20tU29vbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBoeXplbiBHZW5vbWljcyBJbnN0aXR1dGUsIFNlb3VsIDE1MS04MzYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSm9oLCBIbyBKdW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMTUxLTkyMSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMZWUsIFNhbmctQ2hvb24iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMTUxLTkyMSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZXJ1bWFsLCBTYW1wYXRoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBQbGFudCBTY2llbmNlLCBQbGFudCBHZW5vbWljcyBhbmQgQnJlZWRpbmcgSW5zdGl0dXRlLCBSZXNlYXJjaCBJbnN0aXR1dGUgb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIENvbGxlZ2Ugb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFNlb3VsIDE1MS05MjEsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2VvLCBKb29kZW9rIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSm9ldW4gU2VlZCwgR29lc2FuLUd1biwgQ2h1bmdjaGVvbmdidWstRG8gMzY3LTgzMywgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBaG4sIEt5b3VuZ2d1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSm9ldW4gU2VlZCwgR29lc2FuLUd1biwgQ2h1bmdjaGVvbmdidWstRG8gMzY3LTgzMywgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKbywgRXVuIEp1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUmVzZWFyY2ggQ2VudGVyIGZvciBCaW9iYXNlZCBDaGVtaXN0cnksIEtvcmVhIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBDaGVtaWNhbCBUZWNobm9sb2d5LCBEYWVqZW9uIDMwNS02MDAsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2hvaSwgR3l1bmcgSmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBDZW50ZXIgZm9yIEJpb2Jhc2VkIENoZW1pc3RyeSwgS29yZWEgUmVzZWFyY2ggSW5zdGl0dXRlIG9mIENoZW1pY2FsIFRlY2hub2xvZ3ksIERhZWplb24gMzA1LTYwMCwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb3UsIElsbC1TdXAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSwgU3VuY2hvbiBOYXRpb25hbCBVbml2ZXJzaXR5LCBTdW5jaGVvbiA1NDAtOTUwLCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1LCBZZWlzb28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQaHl6ZW4gR2Vub21pY3MgSW5zdGl0dXRlLCBTZW91bCAxNTEtODM2LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbmcsIFRhZS1KaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFBsYW50IFNjaWVuY2UsIFBsYW50IEdlbm9taWNzIGFuZCBCcmVlZGluZyBJbnN0aXR1dGUsIFJlc2VhcmNoIEluc3RpdHV0ZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgQ29sbGVnZSBvZiBBZ3JpY3VsdHVyZSBhbmQgTGlmZSBTY2llbmNlcywgU2VvdWwgTmF0aW9uYWwgVW5pdmVyc2l0eSwgU2VvdWwgMTUxLTkyMSwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDcm9wIEJpb3RlY2hub2xvZ3kgSW5zdGl0dXRlL0dyZWVuQmlvIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFNlb3VsIE5hdGlvbmFsIFVuaXZlcnNpdHksIFB5ZW9uZ2NoYW5nIDIzMi05MTYsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgbWFwIHBlcm1pdHMgaWRlbnRpZmljYXRpb24gb2YgY2x1YnJvb3QgcmVzaXN0YW5jZSBRVExzIGFuZCByZXZpc2lvbiBvZiB0aGUgcmVmZXJlbmNlIGdlbm9tZSBhc3NlbWJseSBpbiBjYWJiYWdlIChCcmFzc2ljYSBvbGVyYWNlYSBMLikiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3YwMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNi8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZsb3dlcmluZyBwbGFudCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwODsgIGNpdGF0aW9uX3BhZ2VzPTc5NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3AgZ2Vub21lIHNlcXVlbmNpbmc6IGxlc3NvbnMgYW5kIHJhdGlvbmFsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NzctODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcm9idXN0LCBzaW1wbGUgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIChHQlMpIGFwcHJvYWNoIGZvciBoaWdoIGRpdmVyc2l0eSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE5Mzc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBwcmVkaWN0aW9uIGluIG1haXplIGJyZWVkaW5nIHBvcHVsYXRpb25zIHdpdGggZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBHZW5vbWVzIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xOTAzLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGdlbm90eXBpbmcgb2YgdGhlIFVTQSBuYXRpb25hbCBtYWl6ZSBpbmJyZWQgc2VlZCBiYW5rOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbXByb3ZlZCBnZW5vdHlwaW5nIGJ5IHNlcXVlbmNpbmcgKEdCUykgYXBwcm9hY2ggb2ZmZXJpbmcgaW5jcmVhc2VkIHZlcnNhdGlsaXR5IGFuZCBlZmZpY2llbmN5IG9mIFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTQ2MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXBzIGZvciBiYXJsZXkgYW5kIHdoZWF0IHVzaW5nIGEgbm92ZWwgdHdvLWVuenltZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgYXBwcm9hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzIyNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmlkZ2luZyB0aGUgZ2Vub3R5cGluZyBnYXA6IHVzaW5nIGdlbm90eXBpbmcgYnkgc2VxdWVuY2luZyAoR0JTKSB0byBhZGQgaGlnaC1kZW5zaXR5IFNOUCBtYXJrZXJzIGFuZCBuZXcgdmFsdWUgdG8gdHJhZGl0aW9uYWwgYmktcGFyZW50YWwgbWFwcGluZyBhbmQgYnJlZWRpbmcgcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MjY5OS03MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIGdlbm90eXBpbmctYnktc2VxdWVuY2luZyBvbiBzZW1pY29uZHVjdG9yIHNlcXVlbmNpbmcgcGxhdGZvcm1zOiBhIGNvbXBhcmlzb24gb2YgZ2VuZXRpYyBhbmQgcmVmZXJlbmNlLWJhc2VkIG1hcmtlciBvcmRlcmluZyBpbiBiYXJsZXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzY5MjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgaW4gQnJhc3NpY2Egd2l0aCBzcGVjaWFsIHJlZmVyZW5jZSB0byB0aGUgZXhwZXJpbWVudGFsIGZvcm1hdGlvbiBvZiBCLiBuYXB1cyBhbmQgcGVjdWxpYXIgbW9kZSBvZiBmZXJ0aWxpemF0aW9uOyBjaXRhdGlvbl9hdXRob3I9VSwgTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEpwbiBCb3QuOyAgY2l0YXRpb25feWVhcj0xOTM1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zODktNDUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gYW5kIGFuYWx5c2lzIG9mIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbGlua2FnZSBtYXAgaW4gY2FiYmFnZSAoQnJhc3NpY2Egb2xlcmFjZWEgTC4gdmFyLiBjYXBpdGF0YSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTUyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCcmFzc2ljYSBvbGVyYWNlYSBnZW5vbWUgcmV2ZWFscyB0aGUgYXN5bW1ldHJpY2FsIGV2b2x1dGlvbiBvZiBwb2x5cGxvaWQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zOTMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmQgbWV0aHlsb21lIHByb2ZpbGluZyByZXZlYWxzIHJlbGljcyBvZiBnZW5vbWUgZG9taW5hbmNlIGluIHRoZSBtZXNvcG9seXBsb2lkIEJyYXNzaWNhIG9sZXJhY2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz1SNzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gUGxhc21vZGlvcGhvcmEgYnJhc3NpY2FlIGFuZCByZXNpc3RhbmNlIHRvIGNsdWJyb290IGRpc2Vhc2UgaW4gYnJhc3NpY2FzIGFuZCBhbGxpZWQgY3JvcHMtYSBjcml0aWNhbCByZXZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJyZWVkLiBBYnN0LjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHN1c2NlcHRpYmlsaXR5IG9mIEJyYXNzaWNhIG9sZXJhY2VhIGN1bHRpdmFycyB0byBQbGFzbW9kaW9waG9yYSBicmFzc2ljYWUgKGNsdWJyb290KTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFzbW9kaW9waG9yYSBicmFzc2ljYWU6IGFzcGVjdHMgb2YgcGF0aG9nZW5lc2lzIGFuZCByZXNpc3RhbmNlIGluIEJyYXNzaWNhIG9sZXJhY2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9ODM7ICBjaXRhdGlvbl9wYWdlcz0xMzktNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWNzIG9mIGNsdWJyb290IHJlc2lzdGFuY2UgaW4gQnJhc3NpY2Egc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgR3Jvd3RoIFJlZ3VsLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI1Mi02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBsb2N1cyBmb3IgY2x1YnJvb3QgcmVzaXN0YW5jZSBpbiBCcmFzc2ljYSByYXBhIGFuZCBpdHMgbGlua2FnZSBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTYzOS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0NBUiBhbmQgQ0FQUyBtYXBwaW5nIG9mIENSYiwgYSBnZW5lIGNvbmZlcnJpbmcgcmVzaXN0YW5jZSB0byBQbGFzbW9kaW9waG9yYSBicmFzc2ljYWUgaW4gQ2hpbmVzZSBjYWJiYWdlIChCcmFzc2ljYSByYXBhIHNzcC4gcGVraW5lbnNpcyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTQ1OC0xNDY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0LWJhc2VkIGNvbXBhcmF0aXZlIGdlbm9taWNzIGJldHdlZW4gQnJhc3NpY2EgcmFwYSBhbmQgQXJhYmlkb3BzaXMgdGhhbGlhbmE6IHRoZSBnZW5ldGljIG9yaWdpbiBvZiBjbHVicm9vdCByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzM7ICBjaXRhdGlvbl9wYWdlcz0zMDktMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgb2YgaXNvbGF0ZS1zcGVjaWZpYyBRVExzIGZvciBjbHVicm9vdCByZXNpc3RhbmNlIGluIENoaW5lc2UgY2FiYmFnZSAoQnJhc3NpY2EgcmFwYSBMLiBzc3AuIHBla2luZW5zaXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTExNzsgIGNpdGF0aW9uX3BhZ2VzPTc1OS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZSBtYXBwaW5nIG9mIHRoZSBjbHVicm9vdCByZXNpc3RhbmNlIGdlbmUsIENycjMsIGluIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9ODEtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgbWFwcGluZyBvZiB0aGUgY2x1YnJvb3QgcmVzaXN0YW5jZSBnZW5lIENSYiBhbmQgZGV2ZWxvcG1lbnQgb2YgYSB1c2VmdWwgc2VsZWN0YWJsZSBtYXJrZXIgaW4gQnJhc3NpY2EgcmFwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0xMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lIG1hcHBpbmcgYSBjbHVicm9vdCByZXNpc3RhbmNlIGxvY3VzIGluIENoaW5lc2UgY2FiYmFnZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIFNvYy4gSG9ydC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzk7ICBjaXRhdGlvbl9wYWdlcz0yNDctNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgZ2VuZXRpYyBhbmQgcGh5c2ljYWwgbWFwcGluZyBvZiB0aGUgQ1JiIGdlbmUgY29uZmVycmluZyByZXNpc3RhbmNlIHRvIGNsdWJyb290IGRpc2Vhc2UgaW4gQnJhc3NpY2EgcmFwYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xMTczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXhwbG9pdGF0aW9uIG9mIGdlbmV0aWMgcmVzb3VyY2VzIG9mIEJyYXNzaWNhIG9sZXJhY2VhIGluIGJyZWVkaW5nIGZvciByZXNpc3RhbmNlIHRvIGNsdWJyb290IChQbGFzbW9kaW9waG9yYSBicmFzc2ljYWUpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXBoeXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0yMTUtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJBUEQgbWFya2VycyBsaW5rZWQgdG8gYSBjbHVicm9vdC1yZXNpc3RhbmNlIGxvY3VzIGluIEJyYXNzaWNhIHJhcGEgTDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVwaHl0aWNhOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db252ZXJzaW9uIG9mIGFuIEFGTFAgbWFya2VyIGxpbmtlZCB0byBjbHVicm9vdCByZXNpc3RhbmNlIGdlbmUgaW4gQ2hpbmVzZSBjYWJiYWdlIGludG8gYSBTQ0FSIG1hcmtlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SG9ydC4gRW52aXJvbi4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz02NTMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgTm92ZWwgUVRMcyBmb3IgSXNvbGF0ZS1zcGVjaWZpYyBwYXJ0aWFsIHJlc2lzdGFuY2UgdG8gUGxhc21vZGlvcGhvcmEgYnJhc3NpY2FlIGluIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lODUzMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBRVExzIHRoYXQgY29udHJvbCBjbHVicm9vdCByZXNpc3RhbmNlIGluIEJyYXNzaWNhIG9sZXJhY2VhIGFuZCBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBjbHVicm9vdCByZXNpc3RhbmNlIGdlbmVzIGJldHdlZW4gQi4gcmFwYSBhbmQgQi4gb2xlcmFjZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MTMzNS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtb2RpZmllZCBwcm90b2NvbCBmb3IgcmFwaWQgRE5BIGlzb2xhdGlvbiBmcm9tIHBsYW50IHRpc3N1ZXMgdXNpbmcgY2V0eWx0cmltZXRoeWxhbW1vbml1bSBicm9taWRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvY29sczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MjMyMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3PigJNXaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2Vub21lIEFuYWx5c2lzIFRvb2xraXQ6IGEgTWFwUmVkdWNlIGZyYW1ld29yayBmb3IgYW5hbHl6aW5nIG5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBTTlAgaWRlbnRpZmljYXRpb24gYW5kIFFUTCBtYXBwaW5nIGZvciBibGFjayByb3QgcmVzaXN0YW5jZSBpbiBjYWJiYWdlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0zMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBvZiB0d28gcGFyZW50YWwgbGluZXMgb2YgY2FiYmFnZSAoQnJhc3NpY2Egb2xlcmFjZWEgTC4gdmFyLiBjYXBpdGF0YSBMLikgYW5kIGNvbnN0cnVjdGlvbiBvZiBhbiBFU1QtYmFzZWQgZ2VuZXRpYyBtYXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBsYXJnZS1zY2FsZSBkZXZlbG9wbWVudCBvZiBtaWNyb3NhdGVsbGl0ZXMgZm9yIG1hcmtlciBhbmQgbWFwcGluZyBhcHBsaWNhdGlvbnMgaW4gQnJhc3NpY2EgY3JvcCBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTExMDMtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhbiBvaWxzZWVkIHJhcGUgKEJyYXNzaWNhIG5hcHVzIEwuKSBnZW5ldGljIG1hcCB3aXRoIFNTUiBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWZlcmVuY2UgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIG11bHRpbmF0aW9uYWwgQnJhc3NpY2EgcmFwYSBnZW5vbWUgc2VxdWVuY2luZyBwcm9qZWN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTc3Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0YWJhc2UgZGVyaXZlZCBtaWNyb3NhdGVsbGl0ZSBtYXJrZXJzIChTU1JzKSBmb3IgY3VsdGl2YXIgZGlmZmVyZW50aWF0aW9uIGluIEJyYXNzaWNhIG9sZXJhY2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gUmVzb3VyYy4gQ3JvcCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTE3MTctMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBnZW5ldGljIG1hcHBpbmcgb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBmcm9tIGdlbm9tZSBzdXJ2ZXkgc2VxdWVuY2VzIGluIEJyYXNzaWNhIG5hcHVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTExMjEtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBuZXcgaGlnaCBjb3B5IFN0b3dhd2F5IGZhbWlseSBNSVRFLCBCUkFNSS0xIGluIEJyYXNzaWNhIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiAyMCBtaW5pYXR1cmUgaW52ZXJ0ZWQtcmVwZWF0IHRyYW5zcG9zYWJsZSBlbGVtZW50IGZhbWlsaWVzIGluIEJyYXNzaWNhIHJhcGEgYW5kIEIuIG9sZXJhY2VhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTk0NDk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzaXN0YW5jZSBvZiBjdWx0aXZhcnMgb2YgQ2hpbmVzZSBjYWJiYWdlIHRvIFBsYXNtb2Rpb3Bob3JhIGJyYXNzaWNhZSBpc29sYXRlcyBvZiBzZXZlcmFsIHJhY2VzIGNvbGxlY3RlZCBpbiBLb3JlYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9S29yLiBKLiBIb3J0LiBTY2kuIFRlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9NjEwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3lzdGVtIGZvciB0aGUgZGV0ZXJtaW5hdGlvbiBvZiByYWNlcyBvZiBQbGFzbW9kaW9waG9yYSBicmFzc2ljYWUgdGhhdCBpbmZlY3QgQ2FiYmFnZSBhbmQgUnV0YWJhZ2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXRvcGF0aG9sb2d5OyAgY2l0YXRpb25feWVhcj0xOTY2OyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9NjI0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBQkMncyBvZiBjb21wYXJhdGl2ZSBnZW5vbWljcyBpbiB0aGUgQnJhc3NpY2FjZWFlOiBidWlsZGluZyBibG9ja3Mgb2YgY3J1Y2lmZXIgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01MzUtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmthZ2UgYW5hbHlzaXMgb2YgUkZMUCBtYXJrZXJzIGZvciBjbHVicm9vdCByZXNpc3RhbmNlIGFuZCBwaWdtZW50YXRpb24gaW4gQ2hpbmVzZSBjYWJiYWdlIChCcmFzc2ljYSByYXBhIHNzcC4gcGVraW5lbnNpcyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1cGh5dGljYTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDQ7ICBjaXRhdGlvbl9wYWdlcz03OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgdHdvIGxvY2kgZm9yIHJlc2lzdGFuY2UgdG8gY2x1YnJvb3QgKFBsYXNtb2Rpb3Bob3JhIGJyYXNzaWNhZSBXb3JvbmluKSBpbiBCcmFzc2ljYSByYXBhIEw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9OTk3LTEwMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIG1lc29wb2x5cGxvaWQgY3JvcCBzcGVjaWVzIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTAzNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBDcnIxYSwgYSBnZW5lIGZvciByZXNpc3RhbmNlIHRvIGNsdWJyb290IGRpc2Vhc2UgKFBsYXNtb2Rpb3Bob3JhIGJyYXNzaWNhZSBXb3JvbmluKSBpbiBCcmFzc2ljYSByYXBhIEw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTQ3NDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWFwLWJhc2VkIHNlcXVlbmNlIG9mIHRoZSByaWNlIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUludGVybmF0aW9uYWwgUmljZSBHZW5vbWUgU2VxdWVuY2luZyBQcm9qZWN0LjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNjsgIGNpdGF0aW9uX3BhZ2VzPTc5My04MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgdXRpbGl0eSBvZiAxMC1rYiBsaWJyYXJpZXMgZm9yIGVmZmljaWVudCBjbG9uZS1nYXAgY2xvc3VyZSBmb3IgcmljZSBnZW5vbWUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz02NTItNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIG9mIEJyYXNzaWNhIG9sZXJhY2VhIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYSByZXZlYWwgZ2VuZSBsb3NzLCBmcmFnbWVudGF0aW9uLCBhbmQgZGlzcGVyc2FsIGFmdGVyIHBvbHlwbG9pZHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMzQ4LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1sZXZlbCBhbmFseXNpcyBvZiB0aGUgZGlwbG9pZGl6YXRpb24gcHJvY2VzcyBpbiB0aGUgdHJpcGxpY2F0ZWQgRkxPV0VSSU5HIExPQ1VTIEMgcmVnaW9uIG9mIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMzM5LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgZm9yIHBsYW50IGJyZWVkaW5nIGFuZCBnZW5ldGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05Mi0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxleGVkIHNob3RndW4gZ2Vub3R5cGluZyBmb3IgcmFwaWQgYW5kIGVmZmljaWVudCBnZW5ldGljIG1hcHBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjEwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXRpYyBtYXAgZm9yIEJyYXNzaWNhIG9sZXJhY2VhIGJhc2VkIG9uIFJGTFAgbWFya2VycyBkZXRlY3RlZCB3aXRoIGV4cHJlc3NlZCBETkEgc2VxdWVuY2VzIGFuZCBtYXBwaW5nIG9mIHJlc2lzdGFuY2UgZ2VuZXMgdG8gcmFjZSAyIG9mIFBsYXNtb2Rpb3Bob3JhIGJyYXNzaWNhZSAoV29yb25pbik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTQwOS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVjaXBoZXJpbmcgdGhlIGRpcGxvaWQgYW5jZXN0cmFsIGdlbm9tZSBvZiB0aGUgbWVzb2hleGFwbG9pZCBCcmFzc2ljYSByYXBhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTU0MS01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzEvMjkvMTc0NjI5ODQvZHN2MDM0LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBDbHVicm9vdCBpcyBhIGRldmFzdGF0aW5nIGRpc2Vhc2UgY2F1c2VkIGJ5IFBsYXNtb2Rpb3Bob3JhIGJyYXNzaWNhZSBhbmQgcmVzdWx0cyBpbiBzZXZlcmUgbG9zc2VzIG9mIHlpZWxkIGFuZCBxdWFsaXR5IGluIEJyYXNzaWNhIGNyb3BzLiBNYW55IGNsdWJybyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2MjQzMCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTxQOVz0eoDmt2F+MDjOS3ECwrScg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f29%2f2462430" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f29%2f2462430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83835" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&amp;Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/1">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198035">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198035">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198037">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198037">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198044">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198044">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198062">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198062">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198092">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198092">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198109">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198109">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198111">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198111">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198113">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198113">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/1/21/2462418">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/1/43/2462442">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (<em>Brassica oleracea</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jonghoon Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jonghoon+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jonghoon Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nur Kholilatul Izzah</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Indonesian Research Institute for Industrial and Beverage Crops (IRIIBC)</div>, <div class="addr-line">Pakuwon, Sukabumi</div>, <div class="country">Indonesia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nur+Kholilatul+Izzah">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Izzah N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Izzah N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nur Kholilatul Izzah</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Beom-Soon Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Phyzen Genomics Institute</div>, <div class="addr-line">Seoul 151-836</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Beom-Soon+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Beom-Soon Choi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ho Jun Joh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ho+Jun+Joh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Joh H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Joh H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ho Jun Joh</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sang-Choon Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sang-Choon+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sang-Choon Lee</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sampath Perumal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sampath+Perumal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Perumal S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Perumal S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sampath Perumal</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Joodeok Seo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Joeun Seed</div>, <div class="addr-line">Goesan-Gun, Chungcheongbuk-Do 367-833</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Joodeok+Seo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Seo J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Seo J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Joodeok Seo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyounggu Ahn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Joeun Seed</div>, <div class="addr-line">Goesan-Gun, Chungcheongbuk-Do 367-833</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyounggu+Ahn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ahn K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ahn K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyounggu Ahn</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eun Ju Jo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Research Center for Biobased Chemistry</div>, <div class="insititution">Korea Research Institute of Chemical Technology</div>, <div class="addr-line">Daejeon 305-600</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eun+Ju+Jo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jo E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jo E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eun Ju Jo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gyung Ja Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Research Center for Biobased Chemistry</div>, <div class="insititution">Korea Research Institute of Chemical Technology</div>, <div class="addr-line">Daejeon 305-600</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gyung+Ja+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Gyung Ja Choi</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ill-Sup Nou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Horticulture</div>, <div class="insititution">Sunchon National University</div>, <div class="addr-line">Suncheon 540-950</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ill-Sup+Nou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nou I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nou I&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ill-Sup Nou</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yeisoo Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Phyzen Genomics Institute</div>, <div class="addr-line">Seoul 151-836</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yeisoo+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yeisoo Yu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tae-Jin Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Plant Science</div>, <div class="insititution">Plant Genomics and Breeding Institute, Research Institute of Agriculture and Life Sciences, College of Agriculture and Life Sciences, Seoul National University</div>, <div class="addr-line">Seoul 151-921</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">7</div><div class="insititution">Crop Biotechnology Institute/GreenBio Science and Technology, Seoul National University</div>, <div class="addr-line">Pyeongchang 232-916</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 2-880-4547. Fax. +82 2-873-2056. E-mail: <a href="mailto:tjyang@snu.ac.kr" target="_blank">tjyang@snu.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tae-Jin+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tae-Jin Yang</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Doil Choi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 29–41, <a href='https://doi.org/10.1093/dnares/dsv034'>https://doi.org/10.1093/dnares/dsv034</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 November 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 May 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462430" href="/dnaresearch/article-pdf/23/1/29/17462984/dsv034.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jonghoon Lee, Nur Kholilatul Izzah, Beom-Soon Choi, Ho Jun Joh, Sang-Choon Lee, Sampath Perumal, Joodeok Seo, Kyounggu Ahn, Eun Ju Jo, Gyung Ja Choi, Ill-Sup Nou, Yeisoo Yu, Tae-Jin Yang;  Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (<em>Brassica oleracea</em> L.), <em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 29–41, <a href='https://doi.org/10.1093/dnares/dsv034'>https://doi.org/10.1093/dnares/dsv034</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462430?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genotyping-by-sequencing%20map%20permits%20identification%20of%20clubroot%20resistance%20QTLs%20and%20revision%20of%20the%20reference%20genome%20assembly%20in%20cabbage%20(%26lt%3Bem%26gt%3BBrassica%20oleracea%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2015-11-29&amp;volumeNum=23&amp;issueNum=1&amp;author=Lee%2C%20Jonghoon%3B%20Izzah%2C%20Nur%20Kholilatul&amp;startPage=29&amp;endPage=41&amp;contentId=10.1093%2Fdnares%2Fdsv034&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f29%2f2462430" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198035 id="87198035" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Clubroot is a devastating disease caused by <em>Plasmodiophora brassicae</em> and results in severe losses of yield and quality in <em>Brassica</em> crops. Many clubroot resistance genes and markers are available in <em>Brassica rapa</em> but less is known in <em>Brassica oleracea.</em> Here, we applied the genotyping-by-sequencing (GBS) technique to construct a high-resolution genetic map and identify clubroot resistance (CR) genes. A total of 43,821 SNPs were identified using GBS data for two parental lines, one resistant and one susceptible lines to clubroot, and 18,187 of them showed &gt;5× coverage in the GBS data. Among those, 4,103 were credibly genotyped for all 78 F<sub>2</sub> individual plants. These markers were clustered into nine linkage groups spanning 879.9 cM with an average interval of 1.15 cM. Quantitative trait loci (QTLs) survey based on three rounds of clubroot resistance tests using F<sub>2</sub><sub>:</sub><sub>3</sub> progenies revealed two and single major QTLs for Race 2 and Race 9 of <em>P. brassicae</em>, respectively. The QTLs show similar locations to the previously reported CR loci for Race 4 in <em>B. oleracea</em> but are in different positions from any of the CR loci found in <em>B. rapa</em>. We utilized two reference genome sequences in this study. The high-resolution genetic map developed herein allowed us to reposition 37 and 2 misanchored scaffolds in the 02–12 and TO1000DH genome sequences, respectively. Our data also support additional positioning of two unanchored 3.3 Mb scaffolds into the 02–12 genome sequence.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="cabbage">cabbage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genotyping-by-sequencing">genotyping-by-sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="clubroot">clubroot</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="QTL">QTL</a></div>                            <h2 scrollto-destination=87198037 id="87198037" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Since the completion of the first genome sequence of a plant, <em>Arabidopsis thaliana</em>,<sup><a reveal-id="DSV034C1" data-open="DSV034C1" class="link link-ref link-reveal xref-bibr">1</a></sup> next-generation sequencing (NGS) technologies have facilitated genome sequencing and construction of reference genome sequences of many important crops.<sup><a reveal-id="DSV034C2" data-open="DSV034C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The reference genome sequences provide valuable information for understanding genomic variation and genome evolution. Based on the utility of reference genome sequences, detection of single-nucleotide polymorphisms (SNPs) has become a valuable tool for genomic selection in plant molecular breeding due to their genome-wide abundance. Accordingly, rapid and inexpensive SNP detection methods are required for genotyping large and diverse germplasm collections or breeding populations. Genotyping by sequencing (GBS) is one such highly efficient method for genome-wide SNP detection with relatively low cost and time demands, because genome-wide reduced representation sequencing is conducted using NGS technology.<sup><a reveal-id="DSV034C3" data-open="DSV034C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>An important strength of GBS is that discovery of huge numbers of markers and genotyping of those markers occur simultaneously. GBS has been applied for construction of high-resolution genetic maps, association study for agricultural traits, and genomic prediction for development of breeding materials using populations of interest in several crop species, such as maize,<sup><a reveal-id="DSV034C3  DSV034C5" data-open="DSV034C3  DSV034C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> soybean,<sup><a reveal-id="DSV034C6" data-open="DSV034C6" class="link link-ref link-reveal xref-bibr">6</a></sup> barley,<sup><a reveal-id="DSV034C3" data-open="DSV034C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV034C7" data-open="DSV034C7" class="link link-ref link-reveal xref-bibr">7</a></sup> rice,<sup><a reveal-id="DSV034C8" data-open="DSV034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and wheat.<sup><a reveal-id="DSV034C7" data-open="DSV034C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Finally, the SNP data deduced from GBS analysis can be used for genomics-assisted breeding. GBS approaches have largely been applied to crops with well-sequenced reference genomes. In further, GBS approach successfully adapted for genetic mapping without reference genome sequences.<sup><a reveal-id="DSV034C9" data-open="DSV034C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p><em>Brassica oleracea</em> is one of the three basic diploid <em>Brassica</em> species belonging to the evolutionary U's triangle<sup><a reveal-id="DSV034C10" data-open="DSV034C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and is an economically important vegetable crop consumed worldwide. <em>Brassica oleracea</em> shows extremely diverse morphology and includes vegetables such as cabbages, broccoli, cauliflowers, kohlrabi, kalian, and kales. Recently, two reference genome sequences of <em>B. oleracea</em> were reported; two reference pseudochromosome sequences, of 385 and 447 Mb, were independently built by positioning and ordering of <em>de novo</em>-assembled scaffold sequences using 1,227 PCR-based markers<sup><a reveal-id="DSV034C11" data-open="DSV034C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and 15,909 GBS-based markers,<sup><a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> respectively.</p><p>Clubroot is a devastating disease caused by <em>Plasmodiophora brassicae</em> that results in severe losses of yield and quality not only in cabbage but also in other <em>Brassica</em> crops.<sup><a reveal-id="DSV034C14" data-open="DSV034C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The pathogen induces gall formation on the root, leading to suppressed transport of nutrients and water, and finally abnormal growth and yield loss.<sup><a reveal-id="DSV034C15" data-open="DSV034C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Because the pathogen can survive in soil as resting spores without host plants for long periods, it is difficult to control by cultural practices or chemical treatments.<sup><a reveal-id="DSV034C16" data-open="DSV034C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Therefore, development of cultivars with clubroot resistance (CR) has been considered the most effective method to minimize yield loss caused by clubroot disease.</p><p>In <em>B. rapa</em>, at least eight race-specific CR loci have been introduced from various European fodder turnips,<sup><a reveal-id="DSV034C17" data-open="DSV034C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the markers for <em>Crr1</em>, <em>Crr2</em>, <em>Crr3</em>, <em>Crr4</em>, <em>CRa</em>, <em>CRb</em>, <em>CRc</em>, and <em>CRk</em><sup><a reveal-id="DSV034C18  DSV034C21" data-open="DSV034C18  DSV034C21" class="link link-ref link-reveal xref-bibr">18–21</a></sup> have been identified in five different chromosomes. To increase efficiency in marker-assisted selection (MAS), fine mapping was carried out for each locus in several <em>B. rapa</em> studies.<sup><a reveal-id="DSV034C22  DSV034C25" data-open="DSV034C22  DSV034C25" class="link link-ref link-reveal xref-bibr">22–25</a></sup> In contrast to <em>B. rapa</em>, completely resistant accessions have only rarely been identified in <em>B. oleracea</em>.<sup><a reveal-id="DSV034C17" data-open="DSV034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV034C26" data-open="DSV034C26" class="link link-ref link-reveal xref-bibr">26</a></sup> In <em>B. rapa</em>, the CR traits are controlled in a qualitative<sup><a reveal-id="DSV034C27" data-open="DSV034C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSV034C28" data-open="DSV034C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and quantitative manner<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV034C29" data-open="DSV034C29" class="link link-ref link-reveal xref-bibr">29</a></sup> depending on genotypes studied, whereas genetic studies suggest that CR are controlled by many minor QTLs in <em>B. oleracea</em>.<sup><a reveal-id="DSV034C17" data-open="DSV034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> DNA markers linked to CR loci in <em>B. oleracea</em> have been developed by several research groups, but most of these studies did not disclose specific sequence information for the various marker types. To establish a CR breeding system in <em>B. oleracea</em>, it is important to identify the major CR genes active to various CR races.</p><p>In this study, we generated large-scale marker data sets using GBS analysis of 80 plant samples. The robust SNP marker sets developed in this study were valid for construction of a high-density genetic map and for QTL mapping of CR traits. In addition, the resulting high-resolution linkage map provided valuable information for identification of incorrectly anchored scaffold sequences and further improvement of reference genome assembly for <em>B. oleracea</em>. Finally, the QTLs identified herein, and the genetic map will be valuable for <em>B. oleracea</em> crop improvement.</p>                            <h2 scrollto-destination=87198044 id="87198044" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87198045 id="87198045" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Two cabbage (<em>B. oleracea</em> L. var. <em>capitata</em>) inbred lines, C1176 and C1220, were selected as parents to develop a mapping population. The two lines show different susceptibility to clubroot disease caused by <em>P. brassicae</em>; C1176 is susceptible, whereas C1220 is resistant. The mapping population consisted of 78 F<sub>2</sub> plants generated by crossing between C1220 and C1176. The 78 F<sub>2</sub> plants were vernalized and self-pollinated to produce F<sub>3</sub> progenies for CR tests. All plant materials examined in this study were obtained from Joeun Seeds Co. (Chungcheongbuk-Do, Korea).</p><p>Genomic DNAs from parental lines and F<sub>2</sub> plants were extracted from 2 g samples of young leaves, following the modified cetyltrimethylammonium bromide (CTAB) protocol.<sup><a reveal-id="DSV034C31" data-open="DSV034C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The quality and quantity of the DNA were examined using a NanoDrop ND-1000 (Thermo Fisher Scientific Inc., USA).</p>                            <h3 scrollto-destination=87198048 id="87198048" class="section-title" data-legacy-id=s2b>2.2. Library construction and sequencing</h3>
<p>Genomic DNAs of all 80 plants (two parents and 78 F2) were double digested with two different restriction endonucleases, <em>Ape</em>KI and <em>Msp</em>I. The former recognizes a 5-bp sequence (GCWGC) and creates a 3-bp overhang, while the latter recognizes 4-bp sequence (CCGG) and creates a 2-bp overhang. A set of barcode adapters that recognize <em>Ape</em>KI-compatible sequences with variable barcode sequences (4–9 bp) for each different plant and a set of common adapters that recognize <em>Msp</em>I-compatible sequences were ligated to the digested DNA of each plant. A total of 40 plant samples were pooled as a batch for further library construction steps, such as PCR amplification with adaptor-specific primers, and sequencing on two single lanes of Illumina Hi-seq 2000. DNA digestion, adapter ligation, library construction, and sequencing were carried out by Macrogen Co. (Korea).</p>                            <h3 scrollto-destination=87198050 id="87198050" class="section-title" data-legacy-id=s2c>2.3. Raw sequence data processing and SNP genotyping</h3>
<p>Raw paired-end (PE) sequences were filtered based on criteria for sequence quality and length using the program clc_quality_trim V4.3.0.114910 (<a class="link link-uri" href="http://www.clcbio.com">http://www.clcbio.com</a>). Sequence reads that had &lt;20 in sequence quality score and 50 bp of sequence length were removed, and then barcode sequences were eliminated. Filtered reads, which were ∼93 bp on average, were aligned to the reference sequences of <em>B. oleracea</em> var. <em>capitata</em> homozygous line 02–12 (02–12 reference genome) reported by Liu et al.<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> using the Burrows–Wheeler alignment tool (BWA)<sup><a reveal-id="DSV034C32" data-open="DSV034C32" class="link link-ref link-reveal xref-bibr">32</a></sup> with default parameters, and properly paired reads were sorted by the parameter of maximum insert size (800 bp). Read grouping and removal of PCR duplicates were done using Picard (<a class="link link-uri" href="http://picard.sourceforge.net">http://picard.sourceforge.net</a>). Genome analysis toolkit (GATK) was used to perform local realignment of reads to correct misalignments due to the presence of INDELs. Finally, UnifiedGenotyper,<sup><a reveal-id="DSV034C33" data-open="DSV034C33" class="link link-ref link-reveal xref-bibr">33</a></sup> which is one of the utilities in the GATK package, was used for detection of SNPs and calling of SNP genotypes. Additional filtering and genotyping processes were carried out with user-friendly option changes in the web-interfaced GBS Genotyping Pipeline, and the final genotype data were graphically displayed using GBS Browser (<a class="link link-uri" href="http://www.phyzen-lab.com/oleracea">http://www.phyzen-lab.com/oleracea</a>) developed by Phyzen Genomics Institute (Korea).</p>                            <h3 scrollto-destination=87198052 id="87198052" class="section-title" data-legacy-id=s2d>2.4. Molecular marker analysis</h3>
<p>We analysed 32 markers for genotyping as position markers for comparison with the GBS genotyping results: 2 EST-based dCAPS, 9 whole-genome resequencing-based dCAPS,<sup><a reveal-id="DSV034C34" data-open="DSV034C34" class="link link-ref link-reveal xref-bibr">34</a></sup> 3 EST-SSRs,<sup><a reveal-id="DSV034C35" data-open="DSV034C35" class="link link-ref link-reveal xref-bibr">35</a></sup> 16 genomic SSRs,<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSV034C36  DSV034C40" data-open="DSV034C36  DSV034C40" class="link link-ref link-reveal xref-bibr">36–40</a></sup> and 2 polymorphic markers based on miniature inverted transposable element (MITE) insertion polymorphism (MIP)<sup><a reveal-id="DSV034C41" data-open="DSV034C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSV034C42" data-open="DSV034C42" class="link link-ref link-reveal xref-bibr">42</a></sup> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The amplicons of the markers were analysed by visualization on a UV trans-illuminator after electrophoresis and stained with ethidium bromide using 9% non-denaturing polyacrylamide gels or 1% agarose gels depending on amplified fragment size.</p>                            <h3 scrollto-destination=87198054 id="87198054" class="section-title" data-legacy-id=s2e>2.5. Genetic map construction</h3>
<p>Detected GBS-SNP markers were grouped and named based on the position on the reference chromosome sequences. Genotype results of all markers were listed for each F<sub>2</sub> individual in order of position on pseudochromosomes. The genotyping scores of GBS markers and PCR-based markers were integrated, and linkage analysis and map construction were performed using JoinMap version 4.1 with the same parameters as in previous studies.<sup><a reveal-id="DSV034C34" data-open="DSV034C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSV034C35" data-open="DSV034C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Genetic and physical map positions showed the same order for most GBS-SNPs, but some SNPs were assigned different positions between the genetic map and the physical map. When the genetic mapping position of the GBS-SNP did not coincide with the physical mapping at the pseudomolecule reference genome sequence, we considered these differential positions as misallocated. These areas were named as sequence variance blocks (v-blocks) if the positions of genetic map and physical map were differential for more than three contiguous SNPs, and as small variance blocks (s-blocks) if only one or two contiguous SNPs showed different positions between the genetic and physical maps (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A and B</a></span>). Additionally, we divided the pseudomolecule into different blocks if it was interrupted by v-blocks or showed 10–15% recombination between adjacent markers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1C and B</a></span>). We labelled each block in alphabetical order, and the information for each block on the reference genome is described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h3 scrollto-destination=87198056 id="87198056" class="section-title" data-legacy-id=s2f>2.6. Clubroot resistance test and QTL analysis</h3>
<p>Inoculum preparation and resistance tests were performed by the Korea Research Institute of Chemical Technology (Daejeon, Korea). Two isolates of <em>P. brassicae</em>, YC and GN, which were collected in Yeoncheon and Gangneung, Korea, respectively, were determined as Race 2 and 9, respectively,<sup><a reveal-id="DSV034C43" data-open="DSV034C43" class="link link-ref link-reveal xref-bibr">43</a></sup> based on Williams classification.<sup><a reveal-id="DSV034C44" data-open="DSV034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The pathogen was inoculated onto roots of Chinese cabbage, and the induced clubs were collected from infected roots and stored at −20°C until needed. For preparation of inocula, the clubs were ground in distilled water using a blender, and homogenized tissue was sifted through four-layered gauze. The resting spore suspensions of YC and GN isolates were diluted with sterile distilled water for inoculation. The final resting spore concentration of YC and GN was adjusted to 1.4 × 10<sup>8</sup> and 0.4 × 10<sup>8</sup> spores/ml, respectively.</p><p>A total of three inoculation tests were performed, two with GN isolate in 2012 and 2014, and one with YC isolate in 2014. All analyses were carried out under the same conditions with 10–15 F<sub>2 : 3</sub> plants of each individual F<sub>2</sub> line randomly selected for genotyping analysis. The F<sub>3</sub> seeds were sown and grown in 5 × 8 plastic pots for 2 weeks in a greenhouse, then the F<sub>3</sub> plants were inoculated by applying 5 ml spore suspension at the bottom of the stem base using pipettes. Each plastic pot (40 plants) received 80 ml spore suspension, and the inoculated plants were moved into a dew chamber with the temperature set at 20°C. After incubation for 1 week, the plants were supplied with water from the bottom of the pots for 3 days and then water was withhold for another 4 days. The inoculated plants were transferred to a greenhouse at 20–25°C. The plants were evaluated for clubroot infection 5 weeks after the transfer. After the roots were completely washed, the severity of the clubroot symptoms was scored as follows; 0 = no visible clubs, 1 = a few slight clubs usually confined to lateral roots, 2 = moderate clubbing on lateral roots, 3 = larger clubs in main roots and slight clubs on lateral roots, and 4 = severe clubbing on main roots and lateral roots.</p><p>A disease index for each F<sub>2</sub> individual was calculated as the mean score of 10–15 F<sub>3</sub> seedlings. QTLs for resistance against <em>P. brassicae</em> were evaluated using composite interval mapping (CIM) analysis with 0.5 cM scan intervals using the QGene program. CIM was performed with LOD (logarithm of odds) threshold values that were estimated using 1,000 permutation tests at 5% significance.</p>                            <h3 scrollto-destination=87198060 id="87198060" class="section-title" data-legacy-id=s2g>2.7. Comparative analysis of CR loci</h3>
<p>The genomes of <em>B. oleracea</em> and <em>B. rapa</em> share a set of 24 conserved chromosomal blocks.<sup><a reveal-id="DSV034C45" data-open="DSV034C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The complete <em>B. oleracea</em> draft genome also demonstrates generally strong conservation with <em>B. rapa</em> in large segments at the pseudomolecule level.<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Based on comparison with the chromosomal block reported in Liu et al.,<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> syntenic blocks were compared between <em>B. rapa</em> and <em>B. oleracea</em>. We compared the loci of reported CR genes and QTLs in <em>B. rapa</em><sup><a reveal-id="DSV034C20  DSV034C23" data-open="DSV034C20  DSV034C23" class="link link-ref link-reveal xref-bibr">20–23</a>,<a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSV034C47" data-open="DSV034C47" class="link link-ref link-reveal xref-bibr">47</a></sup> and <em>B. oleracea</em><sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> to our CR-QTLs, using basic local alignment search tool (BLAST).</p>                            <h2 scrollto-destination=87198062 id="87198062" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87198063 id="87198063" class="section-title" data-legacy-id=s3a>3.1. Sequence production and alignment</h3>
<p>A total of ∼824 million PE reads (83.2 Gb) were generated using the Illumina sequencing platform for 80 plants, with an average of 10.3 million reads per plant sample (Table <a reveal-id="DSV034TB1" data-open="DSV034TB1" class="link link-reveal link-table xref-fig">1</a>). After removing low-quality reads and barcode sequences, 45.9 Gb of sequence data remained, with an average read length of 93.5 bp. Among 80 samples, the read amounts ranged between 256 and 897 Mb (Fig. <a reveal-id="DSV034F1" data-open="DSV034F1" class="link link-reveal link-table xref-fig">1</a>A). Based on mapping on the 02–12 genome sequence,<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> we classified the raw data as discarded reads (because of lack of paired read alignment on the reference genome sequence), unmapped reads, or properly aligned reads. Although the amount of data produced from each plant sample was different, the proportion of discarded reads, unmapped reads, and properly aligned reads was similar for each individual plant (Fig. <a reveal-id="DSV034F1" data-open="DSV034F1" class="link link-reveal link-table xref-fig">1</a>B). In total, 66.4% (30.5 Gb) of the filtered sequence data was mapped on the reference genome, which accounted for an average of 380.9 Mb per each plant. From the mapped sequence, we did not use the reads on multi-copy regions and instead used only unique mapped reads for further analysis, representing 40.5% (a total of 18.6 Gb, which accounted for an average of 232.4 Mb per plant) of raw reads. </p>                        <a id="87198065" scrollto-destination="87198065"></a>
<div content-id="DSV034TB1" class="table-modal"><div class="table-wrap"><div id="DSV034TB1" class="table-wrap-title" data-id="DSV034TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of GBS sequence data and alignment to the reference sequence</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Total </th><th>Average/plant </th></tr></thead><tbody><tr><td colspan="3">Raw data </td></tr><tr><td> Reads </td><td>823,976,286 </td><td>10,299,704 </td></tr><tr><td> Bases (bp) </td><td>83,221,604,886 </td><td>1,040,270,061 </td></tr><tr><td colspan="3">After filtering and removing barcodes </td></tr><tr><td> Reads </td><td>491,664,470 </td><td>6,145,806 </td></tr><tr><td> Bases (bp) </td><td>45,888,404,252 </td><td>573,605,053 </td></tr><tr><td colspan="3">Mapped reads on reference genome </td></tr><tr><td colspan="3"> All mapped </td></tr><tr><td>  Reads </td><td>326,477,878 </td><td>4,080,973 </td></tr><tr><td>  Bases (bp) </td><td>30,471,082,942 </td><td>380,888,537 </td></tr><tr><td> </td><td>(66.4%) </td><td> </td></tr><tr><td colspan="3"> Unique mapped </td></tr><tr><td>  Reads </td><td>199,164,830 </td><td>2,489,560 </td></tr><tr><td>  Bases (bp) </td><td>18,588,604,200 </td><td>232,357,552 </td></tr><tr><td> </td><td>(40.5%) </td><td> </td></tr></tbody></table></div><div content-id="DSV034TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Total </th><th>Average/plant </th></tr></thead><tbody><tr><td colspan="3">Raw data </td></tr><tr><td> Reads </td><td>823,976,286 </td><td>10,299,704 </td></tr><tr><td> Bases (bp) </td><td>83,221,604,886 </td><td>1,040,270,061 </td></tr><tr><td colspan="3">After filtering and removing barcodes </td></tr><tr><td> Reads </td><td>491,664,470 </td><td>6,145,806 </td></tr><tr><td> Bases (bp) </td><td>45,888,404,252 </td><td>573,605,053 </td></tr><tr><td colspan="3">Mapped reads on reference genome </td></tr><tr><td colspan="3"> All mapped </td></tr><tr><td>  Reads </td><td>326,477,878 </td><td>4,080,973 </td></tr><tr><td>  Bases (bp) </td><td>30,471,082,942 </td><td>380,888,537 </td></tr><tr><td> </td><td>(66.4%) </td><td> </td></tr><tr><td colspan="3"> Unique mapped </td></tr><tr><td>  Reads </td><td>199,164,830 </td><td>2,489,560 </td></tr><tr><td>  Bases (bp) </td><td>18,588,604,200 </td><td>232,357,552 </td></tr><tr><td> </td><td>(40.5%) </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198065" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV034TB1" class="table-wrap-title" data-id="DSV034TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Overview of GBS sequence data and alignment to the reference sequence</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Total </th><th>Average/plant </th></tr></thead><tbody><tr><td colspan="3">Raw data </td></tr><tr><td> Reads </td><td>823,976,286 </td><td>10,299,704 </td></tr><tr><td> Bases (bp) </td><td>83,221,604,886 </td><td>1,040,270,061 </td></tr><tr><td colspan="3">After filtering and removing barcodes </td></tr><tr><td> Reads </td><td>491,664,470 </td><td>6,145,806 </td></tr><tr><td> Bases (bp) </td><td>45,888,404,252 </td><td>573,605,053 </td></tr><tr><td colspan="3">Mapped reads on reference genome </td></tr><tr><td colspan="3"> All mapped </td></tr><tr><td>  Reads </td><td>326,477,878 </td><td>4,080,973 </td></tr><tr><td>  Bases (bp) </td><td>30,471,082,942 </td><td>380,888,537 </td></tr><tr><td> </td><td>(66.4%) </td><td> </td></tr><tr><td colspan="3"> Unique mapped </td></tr><tr><td>  Reads </td><td>199,164,830 </td><td>2,489,560 </td></tr><tr><td>  Bases (bp) </td><td>18,588,604,200 </td><td>232,357,552 </td></tr><tr><td> </td><td>(40.5%) </td><td> </td></tr></tbody></table></div><div content-id="DSV034TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Total </th><th>Average/plant </th></tr></thead><tbody><tr><td colspan="3">Raw data </td></tr><tr><td> Reads </td><td>823,976,286 </td><td>10,299,704 </td></tr><tr><td> Bases (bp) </td><td>83,221,604,886 </td><td>1,040,270,061 </td></tr><tr><td colspan="3">After filtering and removing barcodes </td></tr><tr><td> Reads </td><td>491,664,470 </td><td>6,145,806 </td></tr><tr><td> Bases (bp) </td><td>45,888,404,252 </td><td>573,605,053 </td></tr><tr><td colspan="3">Mapped reads on reference genome </td></tr><tr><td colspan="3"> All mapped </td></tr><tr><td>  Reads </td><td>326,477,878 </td><td>4,080,973 </td></tr><tr><td>  Bases (bp) </td><td>30,471,082,942 </td><td>380,888,537 </td></tr><tr><td> </td><td>(66.4%) </td><td> </td></tr><tr><td colspan="3"> Unique mapped </td></tr><tr><td>  Reads </td><td>199,164,830 </td><td>2,489,560 </td></tr><tr><td>  Bases (bp) </td><td>18,588,604,200 </td><td>232,357,552 </td></tr><tr><td> </td><td>(40.5%) </td><td> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198065" target="_blank">View Large</a></div></div>                        <a id="87198066" scrollto-destination="87198066"></a>
<div id="DSV034F1" data-id="DSV034F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03401.png?Expires=2147483647&Signature=Lf-egKd5e6zI1gGCA-Rf0KwSGf3U5hqlpMpmBT~e6etje3y8G~PTn7v4Grpvp8QcHXC0mQh5N5fchgeMFPSlLnLNr6kceNvWg9GgzdEGK5jUeRIx0Klx4yp3REo6n5osRHMlY5c6VBtXh-4nVEcKjiuE5JoS95Jz0GQn6O10FOiVngsAEc82VLHOtDXJzWf8AApMMpVdWMsdgkB9te5tATMJX-Ya81d5Zf9IiP0AF43nEUKrnIbWDcxxDlWm8etqusDgf50zUhzNl89spgA9z52z-yrliFRYLh6LrGuYe0IoC5OYHRDsLdok1PapWaD8OLZRXbFvHs66gLA6Jexlyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of sequencing data for each sample after filtering and mapping ratio of generated data. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198066/dsv03401.tiff" path-from-xml="dsv03401.tiff" target="_blank">View large</a><a section="87198066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03401.png?Expires=2147483647&Signature=FIR8ZH~5m4EeniqF~WpXzYGLBmSk5RpOSGgPxJ7z5NihJIft-LuKRTxBt681HeZDgOqZuy~sTJxvci-i3CGZCQAG8e-SknbOa~PDMgxuBxrUvCOv5YlXK3oD3~z8Ar3nyJsEY2ASeLTOJJD9MtvpF48VBEw7K4ovC9TICb9b1zDdWqYEUVutwwQ2WJ~Z4v-95wI91XqMwduvKMR8uMW-qdeo~Hhb~HENe4mg7VSFIQvtkv2AsOW31E9gLdAh3njO2n1n6FEmdkWN~QyNgCvleXQ6Y0zvL4qNUyoGLktddQ~fknGtmKxN~Uc7ATmonhq3EEo2NIeh9E-f~mQlDjUKLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198066&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing data for each sample after filtering and mapping ratio of generated data. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03401.png?Expires=2147483647&Signature=Lf-egKd5e6zI1gGCA-Rf0KwSGf3U5hqlpMpmBT~e6etje3y8G~PTn7v4Grpvp8QcHXC0mQh5N5fchgeMFPSlLnLNr6kceNvWg9GgzdEGK5jUeRIx0Klx4yp3REo6n5osRHMlY5c6VBtXh-4nVEcKjiuE5JoS95Jz0GQn6O10FOiVngsAEc82VLHOtDXJzWf8AApMMpVdWMsdgkB9te5tATMJX-Ya81d5Zf9IiP0AF43nEUKrnIbWDcxxDlWm8etqusDgf50zUhzNl89spgA9z52z-yrliFRYLh6LrGuYe0IoC5OYHRDsLdok1PapWaD8OLZRXbFvHs66gLA6Jexlyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of sequencing data for each sample after filtering and mapping ratio of generated data. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198066/dsv03401.tiff" path-from-xml="dsv03401.tiff" target="_blank">View large</a><a section="87198066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03401.png?Expires=2147483647&Signature=FIR8ZH~5m4EeniqF~WpXzYGLBmSk5RpOSGgPxJ7z5NihJIft-LuKRTxBt681HeZDgOqZuy~sTJxvci-i3CGZCQAG8e-SknbOa~PDMgxuBxrUvCOv5YlXK3oD3~z8Ar3nyJsEY2ASeLTOJJD9MtvpF48VBEw7K4ovC9TICb9b1zDdWqYEUVutwwQ2WJ~Z4v-95wI91XqMwduvKMR8uMW-qdeo~Hhb~HENe4mg7VSFIQvtkv2AsOW31E9gLdAh3njO2n1n6FEmdkWN~QyNgCvleXQ6Y0zvL4qNUyoGLktddQ~fknGtmKxN~Uc7ATmonhq3EEo2NIeh9E-f~mQlDjUKLQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198066&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of sequencing data for each sample after filtering and mapping ratio of generated data. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87198067 id="87198067" class="section-title" data-legacy-id=s3b>3.2. SNP discovery and genotyping</h3>
<p><em>Brassica oleracea</em> genome underwent whole-genome triplication ∼15.9 million yrs ago.<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV034C48" data-open="DSV034C48" class="link link-ref link-reveal xref-bibr">48</a></sup> To avoid incorrect genotyping by mismapping of reads in paralogous regions instead of the original position, we used the correctly mapped PE reads for which both reads were uniquely mapped with correct direction and interval on the reference genome. A total of 43,821 SNPs were detected between the two parental lines, and 18,187 of those remained after removing SNPs whose the minimum mapping depth was lower than five (Table <a reveal-id="DSV034TB2" data-open="DSV034TB2" class="link link-reveal link-table xref-fig">2</a>). The SNPs were distributed throughout the nine chromosomes, with the most SNPs (3,261) on chromosome C03 and the fewest (1,458) on chromosome C02. </p>                        <a id="87198069" scrollto-destination="87198069"></a>
<div content-id="DSV034TB2" class="table-modal"><div class="table-wrap"><div id="DSV034TB2" class="table-wrap-title" data-id="DSV034TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SNPs detected between C1220 and C1176</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th colspan="2">SNPs between C1220 and C1176<hr /> </th><th rowspan="2">Genotyped SNPs in 78 F2 population (Genetic bins)<sup>a</sup> </th></tr><tr><th>Detected </th><th>Filtered </th></tr></thead><tbody><tr><td>C01 </td><td>4,381 </td><td>1,805 </td><td>410 (79) </td></tr><tr><td>C02 </td><td>4,053 </td><td>1,458 </td><td>306 (64) </td></tr><tr><td>C03 </td><td>7,151 </td><td>3,261 </td><td>865 (132) </td></tr><tr><td>C04 </td><td>4,149 </td><td>1,834 </td><td>432 (87) </td></tr><tr><td>C05 </td><td>4,478 </td><td>1,924 </td><td>441 (68) </td></tr><tr><td>C06 </td><td>5,360 </td><td>2,103 </td><td>434 (84) </td></tr><tr><td>C07 </td><td>4,787 </td><td>1,953 </td><td>455 (78) </td></tr><tr><td>C08 </td><td>5,532 </td><td>2,094 </td><td>388 (71) </td></tr><tr><td>C09 </td><td>3,930 </td><td>1,755 </td><td>372 (72) </td></tr><tr><td>Total </td><td>43,821 </td><td>18,187 </td><td>4,103 (735) </td></tr></tbody></table></div><div content-id="DSV034TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th colspan="2">SNPs between C1220 and C1176<hr /> </th><th rowspan="2">Genotyped SNPs in 78 F2 population (Genetic bins)<sup>a</sup> </th></tr><tr><th>Detected </th><th>Filtered </th></tr></thead><tbody><tr><td>C01 </td><td>4,381 </td><td>1,805 </td><td>410 (79) </td></tr><tr><td>C02 </td><td>4,053 </td><td>1,458 </td><td>306 (64) </td></tr><tr><td>C03 </td><td>7,151 </td><td>3,261 </td><td>865 (132) </td></tr><tr><td>C04 </td><td>4,149 </td><td>1,834 </td><td>432 (87) </td></tr><tr><td>C05 </td><td>4,478 </td><td>1,924 </td><td>441 (68) </td></tr><tr><td>C06 </td><td>5,360 </td><td>2,103 </td><td>434 (84) </td></tr><tr><td>C07 </td><td>4,787 </td><td>1,953 </td><td>455 (78) </td></tr><tr><td>C08 </td><td>5,532 </td><td>2,094 </td><td>388 (71) </td></tr><tr><td>C09 </td><td>3,930 </td><td>1,755 </td><td>372 (72) </td></tr><tr><td>Total </td><td>43,821 </td><td>18,187 </td><td>4,103 (735) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Grouping same genotyping results with adjacent markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198069" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV034TB2" class="table-wrap-title" data-id="DSV034TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary of SNPs detected between C1220 and C1176</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th colspan="2">SNPs between C1220 and C1176<hr /> </th><th rowspan="2">Genotyped SNPs in 78 F2 population (Genetic bins)<sup>a</sup> </th></tr><tr><th>Detected </th><th>Filtered </th></tr></thead><tbody><tr><td>C01 </td><td>4,381 </td><td>1,805 </td><td>410 (79) </td></tr><tr><td>C02 </td><td>4,053 </td><td>1,458 </td><td>306 (64) </td></tr><tr><td>C03 </td><td>7,151 </td><td>3,261 </td><td>865 (132) </td></tr><tr><td>C04 </td><td>4,149 </td><td>1,834 </td><td>432 (87) </td></tr><tr><td>C05 </td><td>4,478 </td><td>1,924 </td><td>441 (68) </td></tr><tr><td>C06 </td><td>5,360 </td><td>2,103 </td><td>434 (84) </td></tr><tr><td>C07 </td><td>4,787 </td><td>1,953 </td><td>455 (78) </td></tr><tr><td>C08 </td><td>5,532 </td><td>2,094 </td><td>388 (71) </td></tr><tr><td>C09 </td><td>3,930 </td><td>1,755 </td><td>372 (72) </td></tr><tr><td>Total </td><td>43,821 </td><td>18,187 </td><td>4,103 (735) </td></tr></tbody></table></div><div content-id="DSV034TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Chr </th><th colspan="2">SNPs between C1220 and C1176<hr /> </th><th rowspan="2">Genotyped SNPs in 78 F2 population (Genetic bins)<sup>a</sup> </th></tr><tr><th>Detected </th><th>Filtered </th></tr></thead><tbody><tr><td>C01 </td><td>4,381 </td><td>1,805 </td><td>410 (79) </td></tr><tr><td>C02 </td><td>4,053 </td><td>1,458 </td><td>306 (64) </td></tr><tr><td>C03 </td><td>7,151 </td><td>3,261 </td><td>865 (132) </td></tr><tr><td>C04 </td><td>4,149 </td><td>1,834 </td><td>432 (87) </td></tr><tr><td>C05 </td><td>4,478 </td><td>1,924 </td><td>441 (68) </td></tr><tr><td>C06 </td><td>5,360 </td><td>2,103 </td><td>434 (84) </td></tr><tr><td>C07 </td><td>4,787 </td><td>1,953 </td><td>455 (78) </td></tr><tr><td>C08 </td><td>5,532 </td><td>2,094 </td><td>388 (71) </td></tr><tr><td>C09 </td><td>3,930 </td><td>1,755 </td><td>372 (72) </td></tr><tr><td>Total </td><td>43,821 </td><td>18,187 </td><td>4,103 (735) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Grouping same genotyping results with adjacent markers.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198069" target="_blank">View Large</a></div></div><p>Among 18,187 SNP sites, we selected 4,103 SNPs that were genotyped for all 78 F<sub>2</sub> plants as the final data set for genetic mapping, to reduce uncertainty of placement potentially conferred by loci derived from missing data. We grouped GBS markers showing the same genotype as genetic bins, or loci, to simplify the genetic map (e.g. the first locus of linkage group C1, C1a-1(7), represents seven GBS-SNPs C1a-1 to C1a-7 in tandem) (Table <a reveal-id="DSV034TB2" data-open="DSV034TB2" class="link link-reveal link-table xref-fig">2</a>). We noted the number of SNPs and bins per each 3 Mb along the nine chromosomes, revealing biased distribution for SNP density and genetic recombination bins. Generally, SNP density and genetic recombination bins were richer in euchromatic regions near the telomere but rare in heterochromatic regions near the centromere of each chromosome (Fig. <a reveal-id="DSV034F2" data-open="DSV034F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87198071" scrollto-destination="87198071"></a>
<div id="DSV034F2" data-id="DSV034F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03402.png?Expires=2147483647&Signature=47Wo48~vcGmoU4a-qz-E2MR~1P6jEB9pdjQswNUkcl2M43XPnXPEKMgAom1g7GDfevF0581h-LdfDzBa5XoiXkO79xniJHTbI9bndmpC20k-8~fo9TCiG8-CWYKhNS-~bysTG3rKDtdrd8dDj9orlzWPyLn7UtT3hfYY2oqsKESJY22rG--c6j0CceAm1usu4ZxdK3frwZ5lA1BpZOcNKYqgljggVQsMDREvaY7BOX7FEvy0kytwLcsxGSyX~PjToTGUapkkyjgQaznkqOiX2iS5jPSS3F6eT8VDtn0eY7pLIl00olX3uBZMbEi~SALXqLO30RhmpXc3ea1Z5Rmr-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GBS-SNPs and bins in 3 Mb intervals along nine pseudochromosomes of 02–12 genome sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198071/dsv03402.tiff" path-from-xml="dsv03402.tiff" target="_blank">View large</a><a section="87198071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03402.png?Expires=2147483647&Signature=0HTOYr0acPtNZiEjrYdVXEL~EcWJIMEbItETSqEdGR5JQ47I5lTcSrQbDpB-vy3H4ceol716Thfl23D9wQyoPiUWIy6fdqM8xiyP8Kuo~IILUISrFAjTJiAPSlNnX4Ef-G1exGY6GCfDjNHwuOghWR3BrBdJQUoV-0t221Ukh3tTrX6FDWyKKe7A9Qs9ZF~bPdKVco94fa80Xm8ROxqZLxIecjAbTXs608G6biGzU5Huz7fcrTA-MsLSX9N5gIkCeTgo1JpwH~ckNRtG0CevOVCkDvu5xGRo8Y6fCkwkBfGp1iEQKiX8sRfTqsY0r09ICfel~dvPIAE6oqZQ0yU4hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198071&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GBS-SNPs and bins in 3 Mb intervals along nine pseudochromosomes of 02–12 genome sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03402.png?Expires=2147483647&Signature=47Wo48~vcGmoU4a-qz-E2MR~1P6jEB9pdjQswNUkcl2M43XPnXPEKMgAom1g7GDfevF0581h-LdfDzBa5XoiXkO79xniJHTbI9bndmpC20k-8~fo9TCiG8-CWYKhNS-~bysTG3rKDtdrd8dDj9orlzWPyLn7UtT3hfYY2oqsKESJY22rG--c6j0CceAm1usu4ZxdK3frwZ5lA1BpZOcNKYqgljggVQsMDREvaY7BOX7FEvy0kytwLcsxGSyX~PjToTGUapkkyjgQaznkqOiX2iS5jPSS3F6eT8VDtn0eY7pLIl00olX3uBZMbEi~SALXqLO30RhmpXc3ea1Z5Rmr-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of GBS-SNPs and bins in 3 Mb intervals along nine pseudochromosomes of 02–12 genome sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198071/dsv03402.tiff" path-from-xml="dsv03402.tiff" target="_blank">View large</a><a section="87198071" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03402.png?Expires=2147483647&Signature=0HTOYr0acPtNZiEjrYdVXEL~EcWJIMEbItETSqEdGR5JQ47I5lTcSrQbDpB-vy3H4ceol716Thfl23D9wQyoPiUWIy6fdqM8xiyP8Kuo~IILUISrFAjTJiAPSlNnX4Ef-G1exGY6GCfDjNHwuOghWR3BrBdJQUoV-0t221Ukh3tTrX6FDWyKKe7A9Qs9ZF~bPdKVco94fa80Xm8ROxqZLxIecjAbTXs608G6biGzU5Huz7fcrTA-MsLSX9N5gIkCeTgo1JpwH~ckNRtG0CevOVCkDvu5xGRo8Y6fCkwkBfGp1iEQKiX8sRfTqsY0r09ICfel~dvPIAE6oqZQ0yU4hQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198071&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of GBS-SNPs and bins in 3 Mb intervals along nine pseudochromosomes of 02–12 genome sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87198072 id="87198072" class="section-title" data-legacy-id=s3c>3.3. High-density genetic mapping and refinement of two reference genomes sequence</h3>
<p>A genetic map was built based on F<sub>2</sub> genotyping results of 735 GBS loci represented by 4,103 markers and 32 PCR-based reported markers used for identifying linkage groups (Table <a reveal-id="DSV034TB3" data-open="DSV034TB3" class="link link-reveal link-table xref-fig">3</a>, Fig. <a reveal-id="DSV034F3" data-open="DSV034F3" class="link link-reveal link-table xref-fig">3</a>). All genotyped markers were allocated into nine linkage groups. Nine linkage groups spanned 879.9 cM with an average distance of 1.15 cM between neighbouring loci. The biggest gap in the genetic map was 15.8 cM in linkage group C02. </p>                        <a id="87198074" scrollto-destination="87198074"></a>
<div content-id="DSV034TB3" class="table-modal"><div class="table-wrap"><div id="DSV034TB3" class="table-wrap-title" data-id="DSV034TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of GBS loci and PCR-based markers on the cabbage genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>C1 </th><th>C2 </th><th>C3 </th><th>C4 </th><th>C5 </th><th>C6 </th><th>C7 </th><th>C8 </th><th>C9 </th><th>Total </th></tr></thead><tbody><tr><td colspan="11">GBS bin (GBS-SNPs) </td></tr><tr><td> Derived from target chr. </td><td>74 (382) </td><td>55 (262) </td><td>124 (833) </td><td>79 (392) </td><td>66 (434) </td><td>77 (405) </td><td>69 (427) </td><td>65 (371) </td><td>70 (359) </td><td>679 (3,876) </td></tr><tr><td> Derived from other chr. </td><td>2 (7) </td><td>8 (24) </td><td>11 (49) </td><td>3 (8) </td><td>8 (37) </td><td>4 (16) </td><td>7 (43) </td><td>2 (11) </td><td>11 (43) </td><td>56 (238) </td></tr><tr><td> Total </td><td>76 (389) </td><td>63 (286) </td><td>135 (882) </td><td>82 (400) </td><td>74 (471) </td><td>81 (421) </td><td>76 (470) </td><td>67 (382) </td><td>81 (402) </td><td>735 (4,103) </td></tr><tr><td>PCR-based markers </td><td>4 </td><td>9 </td><td>4 </td><td>14 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>32 </td></tr><tr><td>Total </td><td>80 </td><td>72 </td><td>139 </td><td>96 </td><td>74 </td><td>81 </td><td>77 </td><td>67 </td><td>81 </td><td>767 </td></tr><tr><td>Length (cM) </td><td>97.5 </td><td>106.1 </td><td>139.5 </td><td>120.3 </td><td>85.1 </td><td>77.9 </td><td>68 </td><td>67.4 </td><td>118.1 </td><td>879.9 </td></tr><tr><td>Average interval (cM) </td><td>1.22 </td><td>1.47 </td><td>1.00 </td><td>1.25 </td><td>1.15 </td><td>0.96 </td><td>0.88 </td><td>1.01 </td><td>1.46 </td><td>1.15 </td></tr></tbody></table></div><div content-id="DSV034TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>C1 </th><th>C2 </th><th>C3 </th><th>C4 </th><th>C5 </th><th>C6 </th><th>C7 </th><th>C8 </th><th>C9 </th><th>Total </th></tr></thead><tbody><tr><td colspan="11">GBS bin (GBS-SNPs) </td></tr><tr><td> Derived from target chr. </td><td>74 (382) </td><td>55 (262) </td><td>124 (833) </td><td>79 (392) </td><td>66 (434) </td><td>77 (405) </td><td>69 (427) </td><td>65 (371) </td><td>70 (359) </td><td>679 (3,876) </td></tr><tr><td> Derived from other chr. </td><td>2 (7) </td><td>8 (24) </td><td>11 (49) </td><td>3 (8) </td><td>8 (37) </td><td>4 (16) </td><td>7 (43) </td><td>2 (11) </td><td>11 (43) </td><td>56 (238) </td></tr><tr><td> Total </td><td>76 (389) </td><td>63 (286) </td><td>135 (882) </td><td>82 (400) </td><td>74 (471) </td><td>81 (421) </td><td>76 (470) </td><td>67 (382) </td><td>81 (402) </td><td>735 (4,103) </td></tr><tr><td>PCR-based markers </td><td>4 </td><td>9 </td><td>4 </td><td>14 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>32 </td></tr><tr><td>Total </td><td>80 </td><td>72 </td><td>139 </td><td>96 </td><td>74 </td><td>81 </td><td>77 </td><td>67 </td><td>81 </td><td>767 </td></tr><tr><td>Length (cM) </td><td>97.5 </td><td>106.1 </td><td>139.5 </td><td>120.3 </td><td>85.1 </td><td>77.9 </td><td>68 </td><td>67.4 </td><td>118.1 </td><td>879.9 </td></tr><tr><td>Average interval (cM) </td><td>1.22 </td><td>1.47 </td><td>1.00 </td><td>1.25 </td><td>1.15 </td><td>0.96 </td><td>0.88 </td><td>1.01 </td><td>1.46 </td><td>1.15 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198074" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV034TB3" class="table-wrap-title" data-id="DSV034TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Distribution of GBS loci and PCR-based markers on the cabbage genetic map</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>C1 </th><th>C2 </th><th>C3 </th><th>C4 </th><th>C5 </th><th>C6 </th><th>C7 </th><th>C8 </th><th>C9 </th><th>Total </th></tr></thead><tbody><tr><td colspan="11">GBS bin (GBS-SNPs) </td></tr><tr><td> Derived from target chr. </td><td>74 (382) </td><td>55 (262) </td><td>124 (833) </td><td>79 (392) </td><td>66 (434) </td><td>77 (405) </td><td>69 (427) </td><td>65 (371) </td><td>70 (359) </td><td>679 (3,876) </td></tr><tr><td> Derived from other chr. </td><td>2 (7) </td><td>8 (24) </td><td>11 (49) </td><td>3 (8) </td><td>8 (37) </td><td>4 (16) </td><td>7 (43) </td><td>2 (11) </td><td>11 (43) </td><td>56 (238) </td></tr><tr><td> Total </td><td>76 (389) </td><td>63 (286) </td><td>135 (882) </td><td>82 (400) </td><td>74 (471) </td><td>81 (421) </td><td>76 (470) </td><td>67 (382) </td><td>81 (402) </td><td>735 (4,103) </td></tr><tr><td>PCR-based markers </td><td>4 </td><td>9 </td><td>4 </td><td>14 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>32 </td></tr><tr><td>Total </td><td>80 </td><td>72 </td><td>139 </td><td>96 </td><td>74 </td><td>81 </td><td>77 </td><td>67 </td><td>81 </td><td>767 </td></tr><tr><td>Length (cM) </td><td>97.5 </td><td>106.1 </td><td>139.5 </td><td>120.3 </td><td>85.1 </td><td>77.9 </td><td>68 </td><td>67.4 </td><td>118.1 </td><td>879.9 </td></tr><tr><td>Average interval (cM) </td><td>1.22 </td><td>1.47 </td><td>1.00 </td><td>1.25 </td><td>1.15 </td><td>0.96 </td><td>0.88 </td><td>1.01 </td><td>1.46 </td><td>1.15 </td></tr></tbody></table></div><div content-id="DSV034TB3" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>C1 </th><th>C2 </th><th>C3 </th><th>C4 </th><th>C5 </th><th>C6 </th><th>C7 </th><th>C8 </th><th>C9 </th><th>Total </th></tr></thead><tbody><tr><td colspan="11">GBS bin (GBS-SNPs) </td></tr><tr><td> Derived from target chr. </td><td>74 (382) </td><td>55 (262) </td><td>124 (833) </td><td>79 (392) </td><td>66 (434) </td><td>77 (405) </td><td>69 (427) </td><td>65 (371) </td><td>70 (359) </td><td>679 (3,876) </td></tr><tr><td> Derived from other chr. </td><td>2 (7) </td><td>8 (24) </td><td>11 (49) </td><td>3 (8) </td><td>8 (37) </td><td>4 (16) </td><td>7 (43) </td><td>2 (11) </td><td>11 (43) </td><td>56 (238) </td></tr><tr><td> Total </td><td>76 (389) </td><td>63 (286) </td><td>135 (882) </td><td>82 (400) </td><td>74 (471) </td><td>81 (421) </td><td>76 (470) </td><td>67 (382) </td><td>81 (402) </td><td>735 (4,103) </td></tr><tr><td>PCR-based markers </td><td>4 </td><td>9 </td><td>4 </td><td>14 </td><td>0 </td><td>0 </td><td>1 </td><td>0 </td><td>0 </td><td>32 </td></tr><tr><td>Total </td><td>80 </td><td>72 </td><td>139 </td><td>96 </td><td>74 </td><td>81 </td><td>77 </td><td>67 </td><td>81 </td><td>767 </td></tr><tr><td>Length (cM) </td><td>97.5 </td><td>106.1 </td><td>139.5 </td><td>120.3 </td><td>85.1 </td><td>77.9 </td><td>68 </td><td>67.4 </td><td>118.1 </td><td>879.9 </td></tr><tr><td>Average interval (cM) </td><td>1.22 </td><td>1.47 </td><td>1.00 </td><td>1.25 </td><td>1.15 </td><td>0.96 </td><td>0.88 </td><td>1.01 </td><td>1.46 </td><td>1.15 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198074" target="_blank">View Large</a></div></div>                        <a id="87198075" scrollto-destination="87198075"></a>
<div id="DSV034F3" data-id="DSV034F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03403.png?Expires=2147483647&Signature=25KMmnff~FgcC~NwhOva-n~0atuQ0n1UwGOKpOI1iqx53w6E0G0O~mJlGZUfLasodc86QjJ-av1e3SiXhwved48t8THZ5q537DNWHXSrNVw5gO3SMWWjZaBcRSitGikvS~PRGMzidkz0lUxJAFu6lEfCGGaQ9XvwmzA8WBFy6DpctMzueMDG~4ZtE-jH5FB-UwZ7R2sbKEmz6AFjJxsv8tQVSwszHYrDEYUlncKzubVfmtKOj2Zrmgj3PT5Mx4I68VsqxIwT3iN1igp8uYX9pkGR~aZBTpWQReEp0i1W0kV~O3ESyc3Pnq2ZoZ2eon40aCNEJEBRbzl22bHJQA03WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of cabbage constructed using 735 GBS loci, representing 4,103 SNP markers, and 32 PCR-based markers. Markers in blue (gray) are PCR-based markers and in red (light gray) are loci derived from other pseudochromosomes. QTLs identified in inoculation tests with GN in 2012, 2014, and with YC in 2014 are shown as red (dark gray on C02 and left dark gray on C03), green (light gray), and pink (right dark gray on C03) bars, respectively. The numbers in parentheses after locus names or genetic bins refer to the number of co-segregating SNP markers. The position of the peak LOD score in each QTL is indicated by an arrowhead. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198075/dsv03403.tiff" path-from-xml="dsv03403.tiff" target="_blank">View large</a><a section="87198075" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03403.png?Expires=2147483647&Signature=vOkVia4MWHl7fc6lXSun8SxnSs7bDCp11lgp~ScIbbqkBhdynmEzef3J79AEOm1xARHnDG9ReXeHrwBGo2zlouBODiVJWDIKwTy0FWmcq5T4BHcvMjmICvoS3RdksZ07ubGWGkT8koGPWPTE46MjIBe5Be~tAY4kEHSN5ZAS--U78C6-pW8HyG0xIHYPo90-npodpYN2nJZeGNh~xXEdn~fGPc7HRXKdTAQd57J1S~K4lPZNJgQx4L96dm0-a4Krww-2BPAfLlT5ysbSx2MSPyeXgmThiXbWA-C660rSUdsx2wiBggns3gqF5a5AFgJYhBLYZOkZvKRKmRAfmD-gBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198075&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of cabbage constructed using 735 GBS loci, representing 4,103 SNP markers, and 32 PCR-based markers. Markers in blue (gray) are PCR-based markers and in red (light gray) are loci derived from other pseudochromosomes. QTLs identified in inoculation tests with GN in 2012, 2014, and with YC in 2014 are shown as red (dark gray on C02 and left dark gray on C03), green (light gray), and pink (right dark gray on C03) bars, respectively. The numbers in parentheses after locus names or genetic bins refer to the number of co-segregating SNP markers. The position of the peak LOD score in each QTL is indicated by an arrowhead. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03403.png?Expires=2147483647&Signature=25KMmnff~FgcC~NwhOva-n~0atuQ0n1UwGOKpOI1iqx53w6E0G0O~mJlGZUfLasodc86QjJ-av1e3SiXhwved48t8THZ5q537DNWHXSrNVw5gO3SMWWjZaBcRSitGikvS~PRGMzidkz0lUxJAFu6lEfCGGaQ9XvwmzA8WBFy6DpctMzueMDG~4ZtE-jH5FB-UwZ7R2sbKEmz6AFjJxsv8tQVSwszHYrDEYUlncKzubVfmtKOj2Zrmgj3PT5Mx4I68VsqxIwT3iN1igp8uYX9pkGR~aZBTpWQReEp0i1W0kV~O3ESyc3Pnq2ZoZ2eon40aCNEJEBRbzl22bHJQA03WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage map of cabbage constructed using 735 GBS loci, representing 4,103 SNP markers, and 32 PCR-based markers. Markers in blue (gray) are PCR-based markers and in red (light gray) are loci derived from other pseudochromosomes. QTLs identified in inoculation tests with GN in 2012, 2014, and with YC in 2014 are shown as red (dark gray on C02 and left dark gray on C03), green (light gray), and pink (right dark gray on C03) bars, respectively. The numbers in parentheses after locus names or genetic bins refer to the number of co-segregating SNP markers. The position of the peak LOD score in each QTL is indicated by an arrowhead. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198075/dsv03403.tiff" path-from-xml="dsv03403.tiff" target="_blank">View large</a><a section="87198075" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03403.png?Expires=2147483647&Signature=vOkVia4MWHl7fc6lXSun8SxnSs7bDCp11lgp~ScIbbqkBhdynmEzef3J79AEOm1xARHnDG9ReXeHrwBGo2zlouBODiVJWDIKwTy0FWmcq5T4BHcvMjmICvoS3RdksZ07ubGWGkT8koGPWPTE46MjIBe5Be~tAY4kEHSN5ZAS--U78C6-pW8HyG0xIHYPo90-npodpYN2nJZeGNh~xXEdn~fGPc7HRXKdTAQd57J1S~K4lPZNJgQx4L96dm0-a4Krww-2BPAfLlT5ysbSx2MSPyeXgmThiXbWA-C660rSUdsx2wiBggns3gqF5a5AFgJYhBLYZOkZvKRKmRAfmD-gBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198075&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage map of cabbage constructed using 735 GBS loci, representing 4,103 SNP markers, and 32 PCR-based markers. Markers in blue (gray) are PCR-based markers and in red (light gray) are loci derived from other pseudochromosomes. QTLs identified in inoculation tests with GN in 2012, 2014, and with YC in 2014 are shown as red (dark gray on C02 and left dark gray on C03), green (light gray), and pink (right dark gray on C03) bars, respectively. The numbers in parentheses after locus names or genetic bins refer to the number of co-segregating SNP markers. The position of the peak LOD score in each QTL is indicated by an arrowhead. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The genetic map displayed overall, but with some exceptions, contiguously collinear marker order, because markers were named based on the backbone of reference genome sequence. All linkage groups included several variance blocks (v- or s-blocks), which showed disordered markers with different order or derived from different chromosomes. Chromosome C05 and C09 had only one v-block, whereas C07 included five v-blocks (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In addition, chromosomes C02, C03, C06, C08, and C09 included at least one s-block. According to our linkage map data, we suggest that all s- and v-blocks likely should be re-allocated into the proper positions. Linkage groups C3 and C9 contained the most complex structure. Both included 11 genetic loci derived from other pseudochromosomes (Table <a reveal-id="DSV034TB3" data-open="DSV034TB3" class="link link-reveal link-table xref-fig">3</a> and see the marker names in red in Fig. <a reveal-id="DSV034F3" data-open="DSV034F3" class="link link-reveal link-table xref-fig">3</a>). C3 included 49 GBS markers derived from seven v-blocks, C1-v4, C7-v5, C6-v1, C6-v3, C7-v1, C1-v3, and C2-v2 derived from other chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Most s- and v-blocks were assigned to other chromosomes, except C3-v1 and C6-s1 block, which were misallocated into different positions in the original chromosome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span> and Fig. <a reveal-id="DSV034F3" data-open="DSV034F3" class="link link-reveal link-table xref-fig">3</a>).</p><p>Comparative analysis between the genetic map and the reference genome revealed the new positions for all 27 v-blocks and 10 s-blocks and several inversions of some alphabetically named blocks (Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>A). The positions of GBS markers between chromosomes C04, C05, and C06 and linkage group C4, C5, and C6 suggested that there was less collinearity in those chromosomes than in the other chromosomes. </p>                        <a id="87198078" scrollto-destination="87198078"></a>
<div id="DSV034F4" data-id="DSV034F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03404.png?Expires=2147483647&Signature=QngCgDIYtry8B4dBkQQ54FNhB9vXdN0BM5qtkR7GwclAdHjlrxsvV8i23RPW8PaOpbla-S1n~azybUoRibbj61MAIF2FSabz2DTpgk0RxJJjKaJ5y3lkUQdaWrbWOZzTrrbYAoSlanE5yapz9UsPrJVU4gp6ZiiQndzLSrK7Y2j9MeyIS-Y0tq4QCtQewGgskY-hBOpAqiaqtbPq17r~ikn3Yy86CYURDOTpBbPNXW-FWhUsQR7NxMLm-L8pCtrz1T-7JEBDNOBFK8bI1pRbHF98vz8ycvdT1-2HL57nXoX5slA1KVrUiXoz4mkfHQkULUAiuw1ZOQqTtBSn67iDkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the loci in the genetic map and physical position of SNPs in nine pseudochromosomes of (A) 02–12 and (B) TO1000DH genome sequences. Misallocated scaffolds denoted with stars were compared with their corresponding sequences in the other reference genomes (Fig. 7). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198078/dsv03404.tiff" path-from-xml="dsv03404.tiff" target="_blank">View large</a><a section="87198078" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03404.png?Expires=2147483647&Signature=SnhFtKavv0predlr9cwnTbfxKSEVYybGP7DrcAynQ6TlNk2kHdykNM92b-tBNIjxM3AZYfHZVLzgme15wNDCxexLqjmFBP6vz572P1ca9MBufFYl7F19DPKsB3fhczU5SLziMaVHclVEwKk8m-BUNC9EW4hHLRpMQ7kkcwBqH1pl5kp4z~4znwdU~QCMA~bmA0zJK8MiNdoyzcU4qUcRypiq4Pijja9oYRiQkXuYsVlkYfft0OTbRQSQNQzrMH8PI0Fu6AFmUup8hpPj4vZ8DUVxtkwLHzJJULnB0tcbbckzCffLmLmOC8wqoxd9EEvQq1xI98wGv8hfojId7ZLGxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198078&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between the loci in the genetic map and physical position of SNPs in nine pseudochromosomes of (A) 02–12 and (B) TO1000DH genome sequences. Misallocated scaffolds denoted with stars were compared with their corresponding sequences in the other reference genomes (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03404.png?Expires=2147483647&Signature=QngCgDIYtry8B4dBkQQ54FNhB9vXdN0BM5qtkR7GwclAdHjlrxsvV8i23RPW8PaOpbla-S1n~azybUoRibbj61MAIF2FSabz2DTpgk0RxJJjKaJ5y3lkUQdaWrbWOZzTrrbYAoSlanE5yapz9UsPrJVU4gp6ZiiQndzLSrK7Y2j9MeyIS-Y0tq4QCtQewGgskY-hBOpAqiaqtbPq17r~ikn3Yy86CYURDOTpBbPNXW-FWhUsQR7NxMLm-L8pCtrz1T-7JEBDNOBFK8bI1pRbHF98vz8ycvdT1-2HL57nXoX5slA1KVrUiXoz4mkfHQkULUAiuw1ZOQqTtBSn67iDkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the loci in the genetic map and physical position of SNPs in nine pseudochromosomes of (A) 02–12 and (B) TO1000DH genome sequences. Misallocated scaffolds denoted with stars were compared with their corresponding sequences in the other reference genomes (Fig. 7). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198078/dsv03404.tiff" path-from-xml="dsv03404.tiff" target="_blank">View large</a><a section="87198078" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03404.png?Expires=2147483647&Signature=SnhFtKavv0predlr9cwnTbfxKSEVYybGP7DrcAynQ6TlNk2kHdykNM92b-tBNIjxM3AZYfHZVLzgme15wNDCxexLqjmFBP6vz572P1ca9MBufFYl7F19DPKsB3fhczU5SLziMaVHclVEwKk8m-BUNC9EW4hHLRpMQ7kkcwBqH1pl5kp4z~4znwdU~QCMA~bmA0zJK8MiNdoyzcU4qUcRypiq4Pijja9oYRiQkXuYsVlkYfft0OTbRQSQNQzrMH8PI0Fu6AFmUup8hpPj4vZ8DUVxtkwLHzJJULnB0tcbbckzCffLmLmOC8wqoxd9EEvQq1xI98wGv8hfojId7ZLGxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198078&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between the loci in the genetic map and physical position of SNPs in nine pseudochromosomes of (A) 02–12 and (B) TO1000DH genome sequences. Misallocated scaffolds denoted with stars were compared with their corresponding sequences in the other reference genomes (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Based on the relative ratio between physical and genetic distance, possible heterochromatin and euchromatin regions could be inferred (represented by black and red dotted lines between chromosomes and linkage groups, respectively in Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>). Presumed heterochromatin regions from chromosomes C01, C02, C03, C07, C08, and C09 occupied large portions of pseudochromosomes, but showed low frequency of recombination in genetic maps and also low density of SNPs and bins on each pseudochromosome (Fig. <a reveal-id="DSV034F2" data-open="DSV034F2" class="link link-reveal link-table xref-fig">2</a>). Many v-blocks and s-genotyped markers in pseudochromosomes tended to be localized near heterochromatin regions in linkage groups (Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>). Conversely, the regions showing high frequency of recombination (represented by red dotted lines in Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>) were considered as euchromatin.</p><p>Another cabbage reference sequence was reported by Parkin et al.<sup><a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> using the kale-like TO1000DH genotype (TO1000DH genome sequence). When we performed GBS read mapping on the TO1000DH genome sequence backbone (Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>B and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>), the genetic map consisted of nine linkage groups with 4,338 GBS markers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The genetic map identified two misanchored sequence scaffolds (v-blocks), which were mapped on the other chromosomes instead of the original position of sequences in the pseudomolecule of TO1000DH genome (Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>B). We compared the v-block sequences in one unique genome with the corresponding sequence in the other reference genome sequence (denoted as star in Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>). Approximately 1.5 Mb sequence of a region (39.5–41.0 Mbp) including the C7-v5 block in 02–12 genome which mapped into linkage group C3 in our map did not show corresponding sequence in chromosome C07 of TO1000DH. However, ∼673 Kb of the sequence in that region corresponded to the C3a block in chromosome C03 of TO1000DH (Fig. <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>A), which is coincident with our map position (stars in Fig. <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>). The remaining ∼860 Kb sequence could not be placed due to lack of GBS markers. The two mispositioned scaffolds of TO1000DH were identified in two v-blocks are derived from the heterochromatin regions with rare SNPs and a low recombination rate. Approximately 300 Kb of the C2-v block in chromosome C02 of TO1000DH was not present in the corresponding sequence of 02–12, and the mapping data suggested that the sequence should be repositioned in linkage group C8 (Figs <a reveal-id="DSV034F4" data-open="DSV034F4" class="link link-reveal link-table xref-fig">4</a>B and <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>B). </p>                        <a id="87198081" scrollto-destination="87198081"></a>
<div id="DSV034F5" data-id="DSV034F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03405.png?Expires=2147483647&Signature=yTEfgyu~SbRtYokK6iSOmNNBjFG~5aG-7R8nsU4fU7NyubwNhpu9a-k3pcBsDGw4H~vHMyGBCgcBcM2HZPt5QJcSTw69~z379jNNQ0fr8YyDCzjEa0x-WS8lqs4ebuUV9Usdso2gTnvwubHjF0O9Kp3UtTyudqh5xOR4ZvezhAY8KG7xREpvU8CVVMlH7ZdzaN0YpggRbWdHAHUht~6Up5UrcCAC2VM~cmLIjdGvdMZDRxzQ~oPqp-wCNVqqIGmUqWRYDXszvlXXeS5Xd2~86Q9RqTrsaWDOp3zCxktBXm5uvVvvGcyo4IHOaPYPlf927qa9dBv4SXc-pfV1AZWoZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of reference sequences containing V-blocks between 02–12 and TO1000DH genomes. (A) Only flanking sequences of the C7-v5 block in the 02–12 genome, estimated to be in euchromatin regions with abundant GBS markers, were found in the corresponding sequences of the TO1000DH genome. (B) Corresponding sequence of the C2-v block region in TO1000DH, which seemed to be in heterochromatin regions with sparse GBS markers, was not observed in chromosome C02 of the 02–12 genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198081/dsv03405.tiff" path-from-xml="dsv03405.tiff" target="_blank">View large</a><a section="87198081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03405.png?Expires=2147483647&Signature=ROeCfr4esuPMpWuudcRTVUbQDutScZBd8fDvc~I1IJ40RWXlUJzVk3gM60Om9xg111pRQ-5JIWNIUNN1h56F6XviYf-hOmbL2cZosqm~wBdxwuzCNPU9pt~tA25Xs2H-HYEQAxO-VsWlb-CDJ~0mHiEAqM~MvJliAAyuwvQgvM0wnejLWg--9v6PcJLT2toV1IQuE07D7ET2WQo0yD6UjjJVXnCmlnC7qLkHVAaN4PqwCmNebQ~37NJy6u0S1h76roxzcaTEMYfyLXGdPgwtVX79reHMy4-ZbeFpKz23V0fIHI5EEbOmMwbqLvbnuBcXtEcB~Z5FBQK06uLPBmA7Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198081&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of reference sequences containing V-blocks between 02–12 and TO1000DH genomes. (A) Only flanking sequences of the C7-v5 block in the 02–12 genome, estimated to be in euchromatin regions with abundant GBS markers, were found in the corresponding sequences of the TO1000DH genome. (B) Corresponding sequence of the C2-v block region in TO1000DH, which seemed to be in heterochromatin regions with sparse GBS markers, was not observed in chromosome C02 of the 02–12 genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03405.png?Expires=2147483647&Signature=yTEfgyu~SbRtYokK6iSOmNNBjFG~5aG-7R8nsU4fU7NyubwNhpu9a-k3pcBsDGw4H~vHMyGBCgcBcM2HZPt5QJcSTw69~z379jNNQ0fr8YyDCzjEa0x-WS8lqs4ebuUV9Usdso2gTnvwubHjF0O9Kp3UtTyudqh5xOR4ZvezhAY8KG7xREpvU8CVVMlH7ZdzaN0YpggRbWdHAHUht~6Up5UrcCAC2VM~cmLIjdGvdMZDRxzQ~oPqp-wCNVqqIGmUqWRYDXszvlXXeS5Xd2~86Q9RqTrsaWDOp3zCxktBXm5uvVvvGcyo4IHOaPYPlf927qa9dBv4SXc-pfV1AZWoZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of reference sequences containing V-blocks between 02–12 and TO1000DH genomes. (A) Only flanking sequences of the C7-v5 block in the 02–12 genome, estimated to be in euchromatin regions with abundant GBS markers, were found in the corresponding sequences of the TO1000DH genome. (B) Corresponding sequence of the C2-v block region in TO1000DH, which seemed to be in heterochromatin regions with sparse GBS markers, was not observed in chromosome C02 of the 02–12 genome. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198081/dsv03405.tiff" path-from-xml="dsv03405.tiff" target="_blank">View large</a><a section="87198081" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03405.png?Expires=2147483647&Signature=ROeCfr4esuPMpWuudcRTVUbQDutScZBd8fDvc~I1IJ40RWXlUJzVk3gM60Om9xg111pRQ-5JIWNIUNN1h56F6XviYf-hOmbL2cZosqm~wBdxwuzCNPU9pt~tA25Xs2H-HYEQAxO-VsWlb-CDJ~0mHiEAqM~MvJliAAyuwvQgvM0wnejLWg--9v6PcJLT2toV1IQuE07D7ET2WQo0yD6UjjJVXnCmlnC7qLkHVAaN4PqwCmNebQ~37NJy6u0S1h76roxzcaTEMYfyLXGdPgwtVX79reHMy4-ZbeFpKz23V0fIHI5EEbOmMwbqLvbnuBcXtEcB~Z5FBQK06uLPBmA7Xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198081&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of reference sequences containing V-blocks between 02–12 and TO1000DH genomes. (A) Only flanking sequences of the C7-v5 block in the 02–12 genome, estimated to be in euchromatin regions with abundant GBS markers, were found in the corresponding sequences of the TO1000DH genome. (B) Corresponding sequence of the C2-v block region in TO1000DH, which seemed to be in heterochromatin regions with sparse GBS markers, was not observed in chromosome C02 of the 02–12 genome. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87198082 id="87198082" class="section-title" data-legacy-id=s3d>3.4. Clubroot resistance assays and QTL analysis</h3>
<p>Three independent inoculation trials were performed for clubroot resistance, with GN isolate in 2012 and with GN and YC isolates in 2014. The disease index for F<sub>2</sub> plants was determined by calculating the average value of the clubroot disease indices obtained from 10–15 F<sub>2 : 3</sub> progeny plants in each inoculation test. Disease symptoms caused by YC isolate (Race 2) were much severer than those caused by GN isolate (Race 9) (Fig. <a reveal-id="DSV034F6" data-open="DSV034F6" class="link link-reveal link-table xref-fig">6</a>). The disease symptoms induced by GN isolate were slightly different between the two independent tests in different years, possibly due to different environmental conditions. </p>                        <a id="87198084" scrollto-destination="87198084"></a>
<div id="DSV034F6" data-id="DSV034F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03406.png?Expires=2147483647&Signature=2ojjQY00NgfFN~co5FZUQRG8uvwV5HgPh2C6BIgldUXAfT9mcksd9iLI3L4MzrUyShwyZdGRN3sLwoa9nPPV-Yknxm7uyzNJq8jYVzaz44MGhzBaiCEpsFW-cuXOw4qkO9xkQUhOqFQ7bERuC7PCSzHIUty6GXelm8gceiL~BlAMvyTOpsXUOPlCVXJUkvCzueU6d4XFH0JLYCLGfTR-wAMZT7HPlztA-T4JMWVOW-hTwUijNocsiQKKAwmwt1ilL37whm8NCYqDXktJxzEv-7ON08NBGMLJ4FKSt2Wo8VLhg02ilQfiKCE9O02hJTx3chpyqomhOrnDVfIALAKPyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Disease index distribution of F2 population, evaluated by average scores resulted from inoculated F2 : 3 plants. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198084/dsv03406.tiff" path-from-xml="dsv03406.tiff" target="_blank">View large</a><a section="87198084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03406.png?Expires=2147483647&Signature=fIBlqWkL888DQ1PtJBVJsnVJ2ysWNWCPSurAhkM3Uw4xWfvvYU9w9qWqz~uZzp4L~EpjaWi1iHNzGK5cwn4alh3Of5B5FPif40xDQk7fmk4rEF3cXQ7GDE2r7YlL5E2xkUfn1Q0JfRU5zEKBXWSqOWG2rXUXPI9b34kpdwpPbeg759XE3pWVLIRPmXpsb4oa3pIoENs~aKyw8e5o5C8D84rU3AgrmD~rkq6tMvSq3k8RowTqS810nCM1UaNOPS4y9p82n1hWoLozwx0lmUH9Jw-KImVI3u9TPEfgSl5Izgrl73mV9UNxqQ~fGOfB6i3LK41T2niKifABb3zICWuJHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198084&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Disease index distribution of F<sub>2</sub> population, evaluated by average scores resulted from inoculated F<sub>2 : 3</sub> plants. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03406.png?Expires=2147483647&Signature=2ojjQY00NgfFN~co5FZUQRG8uvwV5HgPh2C6BIgldUXAfT9mcksd9iLI3L4MzrUyShwyZdGRN3sLwoa9nPPV-Yknxm7uyzNJq8jYVzaz44MGhzBaiCEpsFW-cuXOw4qkO9xkQUhOqFQ7bERuC7PCSzHIUty6GXelm8gceiL~BlAMvyTOpsXUOPlCVXJUkvCzueU6d4XFH0JLYCLGfTR-wAMZT7HPlztA-T4JMWVOW-hTwUijNocsiQKKAwmwt1ilL37whm8NCYqDXktJxzEv-7ON08NBGMLJ4FKSt2Wo8VLhg02ilQfiKCE9O02hJTx3chpyqomhOrnDVfIALAKPyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Disease index distribution of F2 population, evaluated by average scores resulted from inoculated F2 : 3 plants. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198084/dsv03406.tiff" path-from-xml="dsv03406.tiff" target="_blank">View large</a><a section="87198084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03406.png?Expires=2147483647&Signature=fIBlqWkL888DQ1PtJBVJsnVJ2ysWNWCPSurAhkM3Uw4xWfvvYU9w9qWqz~uZzp4L~EpjaWi1iHNzGK5cwn4alh3Of5B5FPif40xDQk7fmk4rEF3cXQ7GDE2r7YlL5E2xkUfn1Q0JfRU5zEKBXWSqOWG2rXUXPI9b34kpdwpPbeg759XE3pWVLIRPmXpsb4oa3pIoENs~aKyw8e5o5C8D84rU3AgrmD~rkq6tMvSq3k8RowTqS810nCM1UaNOPS4y9p82n1hWoLozwx0lmUH9Jw-KImVI3u9TPEfgSl5Izgrl73mV9UNxqQ~fGOfB6i3LK41T2niKifABb3zICWuJHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198084&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Disease index distribution of F<sub>2</sub> population, evaluated by average scores resulted from inoculated F<sub>2 : 3</sub> plants. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>QTL analyses were performed independently for each of the three tests. We detected significant QTLs, based on LOD scores higher than the thresholds calculated in the permutation tests; LOD threshold values for the tests with GN isolate in 2012 and 2014 were 2.982 and 2.862, respectively, and the value with YC in 2014 was 2.914. From the two tests with GN isolate, two significant QTL regions were repeatedly detected; CRQTL-GN_1 on chromosome C02 and CRQTL-GN_2 on chromosome C03 (Fig. <a reveal-id="DSV034F3" data-open="DSV034F3" class="link link-reveal link-table xref-fig">3</a>). In the first test, CRQTL-GN_1 had higher values for LOD score, additive effect and variance explained compared with CRQTL-GN_2 (Table <a reveal-id="DSV034TB4" data-open="DSV034TB4" class="link link-reveal link-table xref-fig">4</a>). The contrary results were detected in the second test. The test with YC isolate identified only a signle QTL region, named CRQTL-YC, which was in chromosome C03 with a location similar to CRQTL-GN_2. CRQTL-YC showed the largest QTL effect, 8.723 LOD score, for clubroot resistance and explained 47.1% of the phenotypic variation (Table <a reveal-id="DSV034TB4" data-open="DSV034TB4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="87198086" scrollto-destination="87198086"></a>
<div content-id="DSV034TB4" class="table-modal"><div class="table-wrap"><div id="DSV034TB4" class="table-wrap-title" data-id="DSV034TB4"><div class="label -label">Table 4.</div><div class="caption"><p>QTLs for resistance to <em>Plasmodiophora brassicae</em>, position of the QTL on the map, LOD scores, additive and dominant effects, and percentage of variance explained</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Inoculation tests </th><th>QTL name </th><th>Linkage group </th><th>Marker interval </th><th>Nearest marker of peak LOD score </th><th>LOD<sup>a</sup> </th><th>Additive effect<sup>b</sup> </th><th>Variance explained (%)<sup>c</sup> </th></tr></thead><tbody><tr><td rowspan="2">GN (Race 9) in 2012 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2d-1(2)–C2g-1(1) </td><td>C2h-1(4) </td><td>4.907 </td><td>0.813 </td><td>29.7 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-37(6)–C3b-14(6) </td><td>C3b-3(8) </td><td>3.717 </td><td>−0.736 </td><td>23.5 </td></tr><tr><td rowspan="2">GN (Race 9) in 2014 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2f-36(1)–C3-s1 </td><td>C2h-5(4) </td><td>3.402 </td><td>0.58 </td><td>22 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-1(11)–C3b-11(1) </td><td>C3a-34(2) </td><td>4.698 </td><td>−0.686 </td><td>29.1 </td></tr><tr><td>YC (Race 2) in 2014 </td><td>CRQTL-YC </td><td>C3 </td><td>C3a-1(11)–C3b-153(3) </td><td>C3a-65(8) </td><td>8.723 </td><td>−0.827 </td><td>47.1 </td></tr></tbody></table></div><div content-id="DSV034TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Inoculation tests </th><th>QTL name </th><th>Linkage group </th><th>Marker interval </th><th>Nearest marker of peak LOD score </th><th>LOD<sup>a</sup> </th><th>Additive effect<sup>b</sup> </th><th>Variance explained (%)<sup>c</sup> </th></tr></thead><tbody><tr><td rowspan="2">GN (Race 9) in 2012 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2d-1(2)–C2g-1(1) </td><td>C2h-1(4) </td><td>4.907 </td><td>0.813 </td><td>29.7 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-37(6)–C3b-14(6) </td><td>C3b-3(8) </td><td>3.717 </td><td>−0.736 </td><td>23.5 </td></tr><tr><td rowspan="2">GN (Race 9) in 2014 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2f-36(1)–C3-s1 </td><td>C2h-5(4) </td><td>3.402 </td><td>0.58 </td><td>22 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-1(11)–C3b-11(1) </td><td>C3a-34(2) </td><td>4.698 </td><td>−0.686 </td><td>29.1 </td></tr><tr><td>YC (Race 2) in 2014 </td><td>CRQTL-YC </td><td>C3 </td><td>C3a-1(11)–C3b-153(3) </td><td>C3a-65(8) </td><td>8.723 </td><td>−0.827 </td><td>47.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Peak LOD score of the QTL.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Additive or dominant effect of C1176 allele.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>c</sup>Percentage of variance explained at the peak of QTL.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198086" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV034TB4" class="table-wrap-title" data-id="DSV034TB4"><div class="label -label">Table 4.</div><div class="caption"><p>QTLs for resistance to <em>Plasmodiophora brassicae</em>, position of the QTL on the map, LOD scores, additive and dominant effects, and percentage of variance explained</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Inoculation tests </th><th>QTL name </th><th>Linkage group </th><th>Marker interval </th><th>Nearest marker of peak LOD score </th><th>LOD<sup>a</sup> </th><th>Additive effect<sup>b</sup> </th><th>Variance explained (%)<sup>c</sup> </th></tr></thead><tbody><tr><td rowspan="2">GN (Race 9) in 2012 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2d-1(2)–C2g-1(1) </td><td>C2h-1(4) </td><td>4.907 </td><td>0.813 </td><td>29.7 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-37(6)–C3b-14(6) </td><td>C3b-3(8) </td><td>3.717 </td><td>−0.736 </td><td>23.5 </td></tr><tr><td rowspan="2">GN (Race 9) in 2014 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2f-36(1)–C3-s1 </td><td>C2h-5(4) </td><td>3.402 </td><td>0.58 </td><td>22 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-1(11)–C3b-11(1) </td><td>C3a-34(2) </td><td>4.698 </td><td>−0.686 </td><td>29.1 </td></tr><tr><td>YC (Race 2) in 2014 </td><td>CRQTL-YC </td><td>C3 </td><td>C3a-1(11)–C3b-153(3) </td><td>C3a-65(8) </td><td>8.723 </td><td>−0.827 </td><td>47.1 </td></tr></tbody></table></div><div content-id="DSV034TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Inoculation tests </th><th>QTL name </th><th>Linkage group </th><th>Marker interval </th><th>Nearest marker of peak LOD score </th><th>LOD<sup>a</sup> </th><th>Additive effect<sup>b</sup> </th><th>Variance explained (%)<sup>c</sup> </th></tr></thead><tbody><tr><td rowspan="2">GN (Race 9) in 2012 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2d-1(2)–C2g-1(1) </td><td>C2h-1(4) </td><td>4.907 </td><td>0.813 </td><td>29.7 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-37(6)–C3b-14(6) </td><td>C3b-3(8) </td><td>3.717 </td><td>−0.736 </td><td>23.5 </td></tr><tr><td rowspan="2">GN (Race 9) in 2014 </td><td>CRQTL-GN_1 </td><td>C2 </td><td>C2f-36(1)–C3-s1 </td><td>C2h-5(4) </td><td>3.402 </td><td>0.58 </td><td>22 </td></tr><tr><td>CRQTL-GN_2 </td><td>C3 </td><td>C3a-1(11)–C3b-11(1) </td><td>C3a-34(2) </td><td>4.698 </td><td>−0.686 </td><td>29.1 </td></tr><tr><td>YC (Race 2) in 2014 </td><td>CRQTL-YC </td><td>C3 </td><td>C3a-1(11)–C3b-153(3) </td><td>C3a-65(8) </td><td>8.723 </td><td>−0.827 </td><td>47.1 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Peak LOD score of the QTL.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Additive or dominant effect of C1176 allele.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>c</sup>Percentage of variance explained at the peak of QTL.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198086" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87198087 id="87198087" class="section-title" data-legacy-id=s3e>3.5. Comparative genetic analysis of CR loci</h3>
<p>We next compared the QTL regions identified in this study with reported CR loci in <em>B. rapa</em> and <em>B. oleracea</em> (Table <a reveal-id="DSV034TB5" data-open="DSV034TB5" class="link link-reveal link-table xref-fig">5</a>). Chromosome C02 harbouring CRQTL-GN_1 exhibited strong collinearity with A02 in <em>B. rapa</em>,<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> where the <em>CRc</em> gene, the linked marker for which is m6R,<sup><a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a></sup> was mapped (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>A). In addition, two QTLs in <em>B. oleracea</em>, <em>PbBo(Anju</em>)1 and <em>PbBo(Anju)2</em> as a major and a minor QTL, respectively, were identified in linkage group O02, which also represents chromosome C02, in Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>PbBo(Anju)2</em> is also linked to the m6R marker, and KBrH059L13 and BRMS-228 are the closely linked markers to <em>PbBo(Anju)1</em>. BLAST searches with the m6R marker revealed that that <em>CRc</em> and the <em>PbBo(Anju)2</em> locus were between the C2b-9(6) and C2b-15(9) GBS markers, which were not included in CRQTL-GN_1. While the position of KBrH059L13 in the reference genome was found near the C2h-5(4) locus, BRMS-228 could not be found in the pseudochromosomes, but rather in unanchored scaffold000122_p2. Therefore, unused reads for alignment on pseudochromosomes were mapped to the scaffold with the same criteria, and two different blocks that showed different genotyping results were detected and named scaffold000122_p2a and scaffold 000122_p2b. A single SNP marker found in scaffold000122_p2b had same genotyping results as the C3-s1 locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). The region between C2h-5(4) and C3-s1 overlapped with CRQTL-GN_1 (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>A). </p>                        <a id="87198089" scrollto-destination="87198089"></a>
<div content-id="DSV034TB5" class="table-modal"><div class="table-wrap"><div id="DSV034TB5" class="table-wrap-title" data-id="DSV034TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Clubroot-resistant genes and QTLs reported in <em>Brassica rapa</em> and <em>Brassica oleracea</em>, and positional comparison with the genetic map in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">CR genes or QTLs </th><th rowspan="2">Linked markers or sequence </th><th rowspan="2">Reference </th><th colspan="3">Position in <em>B. rapa</em><hr /> </th><th rowspan="2">Chr. in <em>B. oleracea</em> </th><th rowspan="2">Closest markers in the genetic map </th></tr><tr><th>Chr. </th><th>Start </th><th>End </th></tr></thead><tbody><tr><td><em>Pb(Anju)2</em><sup>a</sup>, <em>CRc</em><sup>b</sup> </td><td>m6R </td><td>Sakamoto et al.<sup><a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02 </td><td>2,112,653 </td><td>2,113,153 </td><td>C02 </td><td>C2b-9(6), C2b-15(9) </td></tr><tr><td><em>Pb(Anju)1</em><sup>a</sup> </td><td>KBrH059L13<br />BRMS-228 </td><td>Suwabe et al.<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02<br />A02 </td><td>24,842,317<br />25,208,934 </td><td>24,842,620<br />25,209,181 </td><td>C02<br />Scaffold000122_p2 </td><td>C2h-5(4), C3-s1<sup>c</sup> </td></tr><tr><td><em>Pb(Anju)3</em><sup>a</sup> </td><td>BRMS-330 </td><td>Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A03 </td><td>1,699,749 </td><td>1,699,978 </td><td>C03 </td><td>C3a-73(6), C3a-79(4) </td></tr><tr><td><em>CRa, CRb</em><sup>b</sup> </td><td>HC352b<br />B4732<br />B1005 </td><td>Matsumoto et al.<sup><a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup>,<br />Kato et al.<sup><a reveal-id="DSV034C23" data-open="DSV034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> </td><td>A03<br />A03<br />A03 </td><td>23,614,238<br />24,348,058<br />24,376,820 </td><td>23,614,816<br />24,348,275<br />24,377,054 </td><td>C03 </td><td>C7-v5-1(6)<sup>c</sup>, C3b-183(20) </td></tr><tr><td><em>Crr3, CRk</em><sup>b</sup> </td><td>BrSTS61 </td><td>Saito et al.<sup><a reveal-id="DSV034C22" data-open="DSV034C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td><td>A03 </td><td>15,161,430 </td><td>15,161,655 </td><td>Scaffold000040 </td><td> </td></tr><tr><td><em>Crr1</em><sup>b</sup> </td><td>BSA7 </td><td>Hatakeyama et al.<sup><a reveal-id="DSV034C49" data-open="DSV034C49" class="link link-ref link-reveal xref-bibr">49</a></sup> </td><td>A08 </td><td>10,766,540 </td><td>10,766,858 </td><td>C03 </td><td>C3e-19(14) </td></tr></tbody></table></div><div content-id="DSV034TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">CR genes or QTLs </th><th rowspan="2">Linked markers or sequence </th><th rowspan="2">Reference </th><th colspan="3">Position in <em>B. rapa</em><hr /> </th><th rowspan="2">Chr. in <em>B. oleracea</em> </th><th rowspan="2">Closest markers in the genetic map </th></tr><tr><th>Chr. </th><th>Start </th><th>End </th></tr></thead><tbody><tr><td><em>Pb(Anju)2</em><sup>a</sup>, <em>CRc</em><sup>b</sup> </td><td>m6R </td><td>Sakamoto et al.<sup><a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02 </td><td>2,112,653 </td><td>2,113,153 </td><td>C02 </td><td>C2b-9(6), C2b-15(9) </td></tr><tr><td><em>Pb(Anju)1</em><sup>a</sup> </td><td>KBrH059L13<br />BRMS-228 </td><td>Suwabe et al.<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02<br />A02 </td><td>24,842,317<br />25,208,934 </td><td>24,842,620<br />25,209,181 </td><td>C02<br />Scaffold000122_p2 </td><td>C2h-5(4), C3-s1<sup>c</sup> </td></tr><tr><td><em>Pb(Anju)3</em><sup>a</sup> </td><td>BRMS-330 </td><td>Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A03 </td><td>1,699,749 </td><td>1,699,978 </td><td>C03 </td><td>C3a-73(6), C3a-79(4) </td></tr><tr><td><em>CRa, CRb</em><sup>b</sup> </td><td>HC352b<br />B4732<br />B1005 </td><td>Matsumoto et al.<sup><a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup>,<br />Kato et al.<sup><a reveal-id="DSV034C23" data-open="DSV034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> </td><td>A03<br />A03<br />A03 </td><td>23,614,238<br />24,348,058<br />24,376,820 </td><td>23,614,816<br />24,348,275<br />24,377,054 </td><td>C03 </td><td>C7-v5-1(6)<sup>c</sup>, C3b-183(20) </td></tr><tr><td><em>Crr3, CRk</em><sup>b</sup> </td><td>BrSTS61 </td><td>Saito et al.<sup><a reveal-id="DSV034C22" data-open="DSV034C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td><td>A03 </td><td>15,161,430 </td><td>15,161,655 </td><td>Scaffold000040 </td><td> </td></tr><tr><td><em>Crr1</em><sup>b</sup> </td><td>BSA7 </td><td>Hatakeyama et al.<sup><a reveal-id="DSV034C49" data-open="DSV034C49" class="link link-ref link-reveal xref-bibr">49</a></sup> </td><td>A08 </td><td>10,766,540 </td><td>10,766,858 </td><td>C03 </td><td>C3e-19(14) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>QTLs identified in <em>B. oleracea</em>.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>b</sup>CR loci identified in <em>B. rapa</em>.</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>c</sup>Markers represented v-block or s-genotyped marker.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198089" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV034TB5" class="table-wrap-title" data-id="DSV034TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Clubroot-resistant genes and QTLs reported in <em>Brassica rapa</em> and <em>Brassica oleracea</em>, and positional comparison with the genetic map in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">CR genes or QTLs </th><th rowspan="2">Linked markers or sequence </th><th rowspan="2">Reference </th><th colspan="3">Position in <em>B. rapa</em><hr /> </th><th rowspan="2">Chr. in <em>B. oleracea</em> </th><th rowspan="2">Closest markers in the genetic map </th></tr><tr><th>Chr. </th><th>Start </th><th>End </th></tr></thead><tbody><tr><td><em>Pb(Anju)2</em><sup>a</sup>, <em>CRc</em><sup>b</sup> </td><td>m6R </td><td>Sakamoto et al.<sup><a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02 </td><td>2,112,653 </td><td>2,113,153 </td><td>C02 </td><td>C2b-9(6), C2b-15(9) </td></tr><tr><td><em>Pb(Anju)1</em><sup>a</sup> </td><td>KBrH059L13<br />BRMS-228 </td><td>Suwabe et al.<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02<br />A02 </td><td>24,842,317<br />25,208,934 </td><td>24,842,620<br />25,209,181 </td><td>C02<br />Scaffold000122_p2 </td><td>C2h-5(4), C3-s1<sup>c</sup> </td></tr><tr><td><em>Pb(Anju)3</em><sup>a</sup> </td><td>BRMS-330 </td><td>Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A03 </td><td>1,699,749 </td><td>1,699,978 </td><td>C03 </td><td>C3a-73(6), C3a-79(4) </td></tr><tr><td><em>CRa, CRb</em><sup>b</sup> </td><td>HC352b<br />B4732<br />B1005 </td><td>Matsumoto et al.<sup><a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup>,<br />Kato et al.<sup><a reveal-id="DSV034C23" data-open="DSV034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> </td><td>A03<br />A03<br />A03 </td><td>23,614,238<br />24,348,058<br />24,376,820 </td><td>23,614,816<br />24,348,275<br />24,377,054 </td><td>C03 </td><td>C7-v5-1(6)<sup>c</sup>, C3b-183(20) </td></tr><tr><td><em>Crr3, CRk</em><sup>b</sup> </td><td>BrSTS61 </td><td>Saito et al.<sup><a reveal-id="DSV034C22" data-open="DSV034C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td><td>A03 </td><td>15,161,430 </td><td>15,161,655 </td><td>Scaffold000040 </td><td> </td></tr><tr><td><em>Crr1</em><sup>b</sup> </td><td>BSA7 </td><td>Hatakeyama et al.<sup><a reveal-id="DSV034C49" data-open="DSV034C49" class="link link-ref link-reveal xref-bibr">49</a></sup> </td><td>A08 </td><td>10,766,540 </td><td>10,766,858 </td><td>C03 </td><td>C3e-19(14) </td></tr></tbody></table></div><div content-id="DSV034TB5" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">CR genes or QTLs </th><th rowspan="2">Linked markers or sequence </th><th rowspan="2">Reference </th><th colspan="3">Position in <em>B. rapa</em><hr /> </th><th rowspan="2">Chr. in <em>B. oleracea</em> </th><th rowspan="2">Closest markers in the genetic map </th></tr><tr><th>Chr. </th><th>Start </th><th>End </th></tr></thead><tbody><tr><td><em>Pb(Anju)2</em><sup>a</sup>, <em>CRc</em><sup>b</sup> </td><td>m6R </td><td>Sakamoto et al.<sup><a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02 </td><td>2,112,653 </td><td>2,113,153 </td><td>C02 </td><td>C2b-9(6), C2b-15(9) </td></tr><tr><td><em>Pb(Anju)1</em><sup>a</sup> </td><td>KBrH059L13<br />BRMS-228 </td><td>Suwabe et al.<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a></sup>,<br />Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A02<br />A02 </td><td>24,842,317<br />25,208,934 </td><td>24,842,620<br />25,209,181 </td><td>C02<br />Scaffold000122_p2 </td><td>C2h-5(4), C3-s1<sup>c</sup> </td></tr><tr><td><em>Pb(Anju)3</em><sup>a</sup> </td><td>BRMS-330 </td><td>Nagaoka et al.<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> </td><td>A03 </td><td>1,699,749 </td><td>1,699,978 </td><td>C03 </td><td>C3a-73(6), C3a-79(4) </td></tr><tr><td><em>CRa, CRb</em><sup>b</sup> </td><td>HC352b<br />B4732<br />B1005 </td><td>Matsumoto et al.<sup><a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup>,<br />Kato et al.<sup><a reveal-id="DSV034C23" data-open="DSV034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> </td><td>A03<br />A03<br />A03 </td><td>23,614,238<br />24,348,058<br />24,376,820 </td><td>23,614,816<br />24,348,275<br />24,377,054 </td><td>C03 </td><td>C7-v5-1(6)<sup>c</sup>, C3b-183(20) </td></tr><tr><td><em>Crr3, CRk</em><sup>b</sup> </td><td>BrSTS61 </td><td>Saito et al.<sup><a reveal-id="DSV034C22" data-open="DSV034C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td><td>A03 </td><td>15,161,430 </td><td>15,161,655 </td><td>Scaffold000040 </td><td> </td></tr><tr><td><em>Crr1</em><sup>b</sup> </td><td>BSA7 </td><td>Hatakeyama et al.<sup><a reveal-id="DSV034C49" data-open="DSV034C49" class="link link-ref link-reveal xref-bibr">49</a></sup> </td><td>A08 </td><td>10,766,540 </td><td>10,766,858 </td><td>C03 </td><td>C3e-19(14) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p><sup>a</sup>QTLs identified in <em>B. oleracea</em>.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>b</sup>CR loci identified in <em>B. rapa</em>.</p></div></div><div content-id="tblfn7"><div class="fn"><p><sup>c</sup>Markers represented v-block or s-genotyped marker.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198089" target="_blank">View Large</a></div></div>                        <a id="87198090" scrollto-destination="87198090"></a>
<div id="DSV034F7" data-id="DSV034F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03407.png?Expires=2147483647&Signature=oDg967M4ecCFC-dasjliCDkdwx1W4NubIL5-W9ofU6DZ6bGkhsrU2QMgsAckM3wnPb~~5yGwNCjB82sRXzQihQbzCAD7hBpSMSFX03236YOrfur6aM-qShYF5pFhA8lJRaK4ke0JTHor8uco4O6-9OJuTzstBmcsAKiNcZ5Onlpnmj9ekF6lP6ke3GQcXZpQ~ugLGzNJgc2DKvRVab0t5h1G96Q3E~qkPbi7hMjXmEOUo6mwoUo-xsM-jSiWY4rLGY-r~oGWISUDpqp0ANbHdFBnMiBARLTU-p9DoJRySFTi8xd58OsatRdtq-SioQIW2CIgVa8P8pTs-Tox0Y1EEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of syntenic regions containing identified CRQTLs in this study; (A) linkage group C02 harbouring CRQTL-GN_1, and (B) linkage group C03 including CRQTL-GN_2 and CRQTL-YC with pseudochromosomes of B. oleracea (02–12 genotype) and B. rapa. QTL names in red (Pb-Anju1, 2, and 3) were identified in B. oleracea, and CR genes in black (CRa, CRb, CRc, CRk, Crr1, and Crr3) were identified in B. rapa. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03407.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198090/dsv03407.tiff" path-from-xml="dsv03407.tiff" target="_blank">View large</a><a section="87198090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03407.png?Expires=2147483647&Signature=ZfOLm3fXueX~LUhYxK605D7Bxwg0hghIgqa4jIrhIEH6sSfL8mKR07mOAR2zO~AAToEujU3bnuxxRIGM2uGtLqIup09bGHk4otvbaqG7JulGjfe1~ov770c~EUCApWvAii4otts9Nj7HHa0vTGT3HdXbLrucXVv6~duSfNrHNm1vzMlDsxmwUpyYLmYu0lK2dLWZnwsRooDC-pkr3dgfNLX4MDSuE2n84rc0-5DiNhWVlwqoBWeFfgIj9yedLU2bd5ylxSmlebVjCGKN~TgYAqTnkJfMm~oRq2La9FRwf8QDs4MsVIRgVjVls9YSHbm7C6niLO1GEkw4XkHXVUhSNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198090&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03407.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of syntenic regions containing identified CRQTLs in this study; (A) linkage group C02 harbouring CRQTL-GN_1, and (B) linkage group C03 including CRQTL-GN_2 and CRQTL-YC with pseudochromosomes of <em>B. oleracea</em> (02–12 genotype) and <em>B. rapa</em>. QTL names in red (<em>Pb-Anju</em>1, 2, and 3) were identified in <em>B. oleracea</em>, and CR genes in black (<em>CRa, CRb, CRc, CRk, Crr1</em>, and <em>Crr3</em>) were identified in <em>B. rapa</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV034F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/m_dsv03407.png?Expires=2147483647&Signature=oDg967M4ecCFC-dasjliCDkdwx1W4NubIL5-W9ofU6DZ6bGkhsrU2QMgsAckM3wnPb~~5yGwNCjB82sRXzQihQbzCAD7hBpSMSFX03236YOrfur6aM-qShYF5pFhA8lJRaK4ke0JTHor8uco4O6-9OJuTzstBmcsAKiNcZ5Onlpnmj9ekF6lP6ke3GQcXZpQ~ugLGzNJgc2DKvRVab0t5h1G96Q3E~qkPbi7hMjXmEOUo6mwoUo-xsM-jSiWY4rLGY-r~oGWISUDpqp0ANbHdFBnMiBARLTU-p9DoJRySFTi8xd58OsatRdtq-SioQIW2CIgVa8P8pTs-Tox0Y1EEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of syntenic regions containing identified CRQTLs in this study; (A) linkage group C02 harbouring CRQTL-GN_1, and (B) linkage group C03 including CRQTL-GN_2 and CRQTL-YC with pseudochromosomes of B. oleracea (02–12 genotype) and B. rapa. QTL names in red (Pb-Anju1, 2, and 3) were identified in B. oleracea, and CR genes in black (CRa, CRb, CRc, CRk, Crr1, and Crr3) were identified in B. rapa. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03407.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198090/dsv03407.tiff" path-from-xml="dsv03407.tiff" target="_blank">View large</a><a section="87198090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv03407.png?Expires=2147483647&Signature=ZfOLm3fXueX~LUhYxK605D7Bxwg0hghIgqa4jIrhIEH6sSfL8mKR07mOAR2zO~AAToEujU3bnuxxRIGM2uGtLqIup09bGHk4otvbaqG7JulGjfe1~ov770c~EUCApWvAii4otts9Nj7HHa0vTGT3HdXbLrucXVv6~duSfNrHNm1vzMlDsxmwUpyYLmYu0lK2dLWZnwsRooDC-pkr3dgfNLX4MDSuE2n84rc0-5DiNhWVlwqoBWeFfgIj9yedLU2bd5ylxSmlebVjCGKN~TgYAqTnkJfMm~oRq2La9FRwf8QDs4MsVIRgVjVls9YSHbm7C6niLO1GEkw4XkHXVUhSNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198090&ar=2462430&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03407.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of syntenic regions containing identified CRQTLs in this study; (A) linkage group C02 harbouring CRQTL-GN_1, and (B) linkage group C03 including CRQTL-GN_2 and CRQTL-YC with pseudochromosomes of <em>B. oleracea</em> (02–12 genotype) and <em>B. rapa</em>. QTL names in red (<em>Pb-Anju</em>1, 2, and 3) were identified in <em>B. oleracea</em>, and CR genes in black (<em>CRa, CRb, CRc, CRk, Crr1</em>, and <em>Crr3</em>) were identified in <em>B. rapa</em>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Chromosome C03 containing two similar QTLs, CRQTL-GN_2 and CRQTL-YC, show syntenic counterpart regions present in chromosome A03, A06, and A08 in <em>B. rapa</em><sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>B). The syntenic region of A03 harbours the loci for <em>Crr3</em>, <em>CRk</em>, <em>CRa</em>, and <em>CRb</em>. The locus of <em>CRa</em> is linked to HC352b sequence,<sup><a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup> and <em>CRb</em> is linked to B4732 and B1005 markers.<sup><a reveal-id="DSV034C23" data-open="DSV034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> All of those marker sequences could be found near C7-v5 block represented by C7-v5–1(6) in this study. C7-v5-1(6) was also linked to the same locus as C3b-183(20) (Fig. <a reveal-id="DSV034F3" data-open="DSV034F3" class="link link-reveal link-table xref-fig">3</a>). To find the <em>Crr1</em> locus present in A08, linked BSA7 marker<sup><a reveal-id="DSV034C49" data-open="DSV034C49" class="link link-ref link-reveal xref-bibr">49</a></sup> sequences were searched in our genetic map, and C3e-19(14) in chromosome C03 was found to be the closest locus. In addition, BrSTS61 marker linked to <em>Crr3</em> and <em>CRk</em><sup><a reveal-id="DSV034C22" data-open="DSV034C22" class="link link-ref link-reveal xref-bibr">22</a></sup> was found in unanchored scaffold000040. Genotyping of SNPs in scaffold00040 revealed that its position could be between C3c and C3d blocks (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>). The position of BRMS-330 marker linked to <em>PbBo(Anju)3</em><sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> was found in our QTLs CRQTL-GN_2 and CRQTL-YC, in chromosome C03. Although we could not find the sequence information for BRMS-125, which is linked to <em>Crr4</em> and contained in A06,<sup><a reveal-id="DSV034C20" data-open="DSV034C20" class="link link-ref link-reveal xref-bibr">20</a></sup> the syntenic counterpart of A06 to C03 was not related to QTL regions (Fig. <a reveal-id="DSV034F7" data-open="DSV034F7" class="link link-reveal link-table xref-fig">7</a>B).</p>                            <h2 scrollto-destination=87198092 id="87198092" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=87198093 id="87198093" class="section-title" data-legacy-id=s4a>4.1. GBS browser and GBS-based genetic mapping</h3>
<p>In a previous study, we identified ∼670,000 SNPs between two elite cabbage breeding lines using 20× genome coverage whole-genome resequencing.<sup><a reveal-id="DSV034C34" data-open="DSV034C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Here, a total of 43,821 GBS-based SNPs were detected between parental lines, 15-fold fewer than detected by resequencing. Although the detection power was reduced, GBS-based genotyping provided a high-density genetic map at low cost in a highly efficient manner.</p><p>GBS analysis encounters bioinformatic, statistical, and computational challenges, because the large data set is frequently accompanied by a large number of erroneous SNP calling if the data are not properly filtered.<sup><a reveal-id="DSV034C8" data-open="DSV034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> To find suitable conditions for precise genotyping, the data set generated in this study was analysed under various conditions with various filtering options using a built-in GBS browser. In the browser, we could change the options to optimize the output (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The average depth of read mapping was 11×, and we utilized SNPs derived from the regions with 5–100× read mapping depth. We filtered out SNPs derived from reads with &lt;5× coverage to reduce uncertain genotyping due to low-coverage sequences. We also removed SNPs derived from high-depth reads, over than 100× coverage, to alleviate malalignment of reads derived from duplicated genome or repeat sequences. We further changed the criteria for genotyping of heterozygous loci and selected threshold value &lt;0.9. We applied a stringent condition to select high-confidence GBS markers only when a genotype was present in all 80 samples with proper read mapping depth. Afterwards, we validated the genotyping results of the GBS markers by comparison to the genotyping results of 32 PCR-based markers. Consequently, we were able to obtain a high-density genetic map composed of 4,103 SNP markers even though both parental lines are cabbage breeding lines with good agriculture traits and narrow genetic diversity.</p><p>GBS analysis performed by Parkin et al.<sup><a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> identified 826 bins in <em>B. oleracea</em>, which was more than identified in this study (679 bins). Our fewer bin numbers are probably due to the difference in genetic diversity of parental lines as well as the number of segregating progeny we used for GBS analysis. We used two elite breeding lines with narrow genetic diversity as parental lines, whereas Parkin et al.<sup><a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> analysed the population between doubled haploid (DH) kale-like and DH broccoli lines. We also analysed 78 F<sub>2</sub> plants for mapping. If we increase the number of F<sub>2</sub> individuals for fine mapping, we may identify more recombination bins.</p>                            <h3 scrollto-destination=87198097 id="87198097" class="section-title" data-legacy-id=s4b>4.2. Our GBS-based high-resolution map improves the reference genome sequence</h3>
<p>High-quality reference genome sequences provide information for identification of genes and regulatory elements, and thus, it is helpful for researchers to understand genomic variation.<sup><a reveal-id="DSV034C2" data-open="DSV034C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The genomes of <em>A. thaliana</em><sup><a reveal-id="DSV034C8" data-open="DSV034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and <em>Oryza sativa</em><sup><a reveal-id="DSV034C50" data-open="DSV034C50" class="link link-ref link-reveal xref-bibr">50</a></sup> were assembled by physical map-based approaches, which involve sequencing of minimally overlapping bacterial artificial chromosome (BAC) clones and filling gaps by various attempts.<sup><a reveal-id="DSV034C51" data-open="DSV034C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Thus, the reference sequence covers almost the complete whole genome. However, most of the recent plant genome sequences were primarily assembled by whole-genome shotgun sequencing strategies using NGS technologies.<sup><a reveal-id="DSV034C2" data-open="DSV034C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>We performed GBS analysis based on the backbone of two reference genome sequences, 02–12 genome sequence reported by Liu et al.<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and TO1000DH genome sequence reported by Parkin et al.<sup><a reveal-id="DSV034C13" data-open="DSV034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> to construct a high-density genetic map. The pseudomolecules of 02–12 were constructed based on the genetic mapping of 1,227 PCR-based markers such as SSRs and SNPs.<sup><a reveal-id="DSV034C11" data-open="DSV034C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The TO1000DH genome sequence was developed by genotyping based on 15,509 GBS-based SNPs for anchoring of scaffold sequences. The TO1000DH genome sequence comprises much larger pseudomolecules (447 Mb) than the pseudochromosomes of 02–12 genome sequence (385 Mb). Here, we utilized 40.5% of the total filtered GBS reads for genotyping by excluding multi-mapped (25.9%) and un-mapped (33.6%) reads. Unmapped reads might be derived from outside of the reference pseudomolecules, probably from unanchored scaffolds or unassembled repeat sequences. In addition, 11.9 Gb of reads (25.9% of data produced in this study) showed multiple hits to the reference sequence, which is probably due to mapping of reads derived from paralogous regions associated with whole-genome triplication events in the <em>Brassiceae</em> tribe<sup><a reveal-id="DSV034C52" data-open="DSV034C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSV034C53" data-open="DSV034C53" class="link link-ref link-reveal xref-bibr">53</a></sup> or repetitive sequence regions.</p><p>We also expected improvement of the 02–12 genome sequence<sup><a reveal-id="DSV034C12" data-open="DSV034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> by discovery and ordering of unanchored scaffold sequences.<sup><a reveal-id="DSV034C54" data-open="DSV034C54" class="link link-ref link-reveal xref-bibr">54</a></sup> We were able to position two unanchored scaffolds, scaffold000040 and scaffold000122_p2, which are 2,345,746 bp and 983,585 bp, respectively, into the pseudomolecule (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The present 02–12 genome sequence includes ∼150 Mb of unanchored scaffolds. In our analysis, &gt;33% of GBS reads were remained as unmapped (Table <a reveal-id="DSV034TB1" data-open="DSV034TB1" class="link link-reveal link-table xref-fig">1</a>) that could be utilized as guidance for mapping of the remaining unanchored scaffolds, similar to efforts made in barley and wheat genomes.<sup><a reveal-id="DSV034C7" data-open="DSV034C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV034C54" data-open="DSV034C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Likewise, the genome of <em>Drosophila simulans</em> was developed using GBS approach, in which 30% of the unassembled scaffolds were newly assigned to linkage groups.<sup><a reveal-id="DSV034C55" data-open="DSV034C55" class="link link-ref link-reveal xref-bibr">55</a></sup></p><p>Furthermore, the 37 and 2 scaffold sequences in v-blocks of the 02–12 and TO1000DH genome sequences, respectively, could be updated via proper repositioning (Fig. <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>). The v-blocks are usually derived from repeat-rich and recombination-deficient heterochromatin regions. Comparison of two v-blocks in both reference genome sequences identified the proper position of 673 Kb (C7-v5 block, Fig. <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>A) among the 1,533 Kb mispositioned 02–12 genome sequence. The remaining 860 Kb sequence could not be placed due to lack of GBS markers (Fig. <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>A). The C2-v blocks in TO1000DH identified ∼300 Kb additional sequence that mapped in other C8 chromosome. However, we could not find the corresponding 300 Kb sequence in the 02–12 genome sequence (Fig. <a reveal-id="DSV034F5" data-open="DSV034F5" class="link link-reveal link-table xref-fig">5</a>B). We assume that the lack of corresponding sequence in the 02–12 genome sequence is because this sequence is derived from a repeat-rich heterochromatin region. Moreover, we identified some misordered scaffolds in same chromosome. For example, the C2e block seemed to be within the C2c block, and the C6f, C6e, and C6 g blocks were likely to be inverted (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The benefits of having both genome sequences are innumerable for genomics studies such as this one. Refining and completing the genome sequences is an ongoing task, and our analyses suggest potential errors to address. However, it remains possible that such differences are derived from utilization of different mapping populations in different <em>B. oleracea</em> genotypes with chromosomal-level reorganization.</p>                            <h3 scrollto-destination=87198102 id="87198102" class="section-title" data-legacy-id=s4c>4.3. Clubroot resistance loci differ in the <em>B. rapa</em> and <em>B. oleracea</em> genomes</h3>
<p>We identified two QTLs (CRQTL-GN_1 and CRQTL-GN_2) for resistance to GN isolate (Race 9) and one QTL (CRQTL-YC) for resistance to YC isolate (Race 2). Although the LOD values of the two QTLs for GN isolate were different between two tests, both QTLs were detected repeatedly in independent inoculation tests, suggesting that both QTLs are strong candidates for major QTLs against Race 9. CRQTL-YC showed the highest LOD score among all trials and also overlapped with CRQTL-GN_2. This finding implies that the CRQTL-YC locus is probably responsible for resistance against both Race 2 and Race 9. Although resistance to the YC isolate was detected only as a single resistance QTL, we think that the resistance against race 2 is likely controlled by this major QTL, CRQTL-YC, and also affected by many minor quantitative loci, because the disease index distribution of the F<sub>2</sub> population did not follow the Mendelian single-gene segregation ratio (Fig. <a reveal-id="DSV034F6" data-open="DSV034F6" class="link link-reveal link-table xref-fig">6</a>).</p><p>The comparison of genetic loci between our QTLs in <em>B. oleracea</em> and eight CR loci in <em>B. rapa</em> revealed no coincidence in chromosomal positions, even though <em>CRa</em>, <em>CRb</em>, <em>CRk</em>, <em>Crr1</em>, <em>Crr3</em>, and <em>Crr4</em> in <em>B. rapa</em> are known to be involved in resistance to Race 2,<sup><a reveal-id="DSV034C17" data-open="DSV034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSV034C21" data-open="DSV034C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV034C29" data-open="DSV034C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSV034C46" data-open="DSV034C46" class="link link-ref link-reveal xref-bibr">46</a></sup> the same isolate as YC. Although <em>CRa</em> and <em>CRb</em> in <em>B. oleracea</em> were also reported as resistance genes against Race 2 using swede cv Wilhelmsburger as a resistance source,<sup><a reveal-id="DSV034C56" data-open="DSV034C56" class="link link-ref link-reveal xref-bibr">56</a></sup> these results raise the possibility that none of the reported CR genes in <em>B. rapa</em> and <em>B. oleracea</em> are related to resistance in our cabbage materials, and that they have a different genetic origin of resistance genes. Because genetic loci for Race 9 resistance have not been identified in either species, our results regarding CRQTL-GN_1 and 2 can provide valuable resources for further research.</p><p>The <em>Brassicaceae</em> family evolved from rearrangement of shared 24 chromosomal blocks (A–X blocks), which are denoted as the ancestral translocation Proto-Calepineae Karyotype (tPCK). Hundreds of species in the Brassiceae tribe are derived from whole genome triplication of the tPCK, followed by genome rearrangement.<sup><a reveal-id="DSV034C52" data-open="DSV034C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSV034C53" data-open="DSV034C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSV034C57" data-open="DSV034C57" class="link link-ref link-reveal xref-bibr">57</a></sup> The fate of paralogous gene function can be differentiated in other <em>Brassica</em> species. We evaluated paralogous genomic blocks harbouring our QTL regions and compared the blocks to the clubroot resistance loci in <em>B. rapa.</em><em>Crr3</em> and <em>CRk</em> were found in the F block, <em>CRa</em> and <em>CRb</em> were found in the J block of chromosome A03, and <em>Crr1</em> and <em>Crr2</em> found in the U blocks of chromosome A08 and A01, respectively, of <em>B. rapa</em>. These F, J, and U blocks were not related to our CRQTL regions. The <em>CRc</em> gene was identified in the R block of chromosome A02 of <em>B. rapa,</em> and our CRQTL-YC and CRQTL-GN_2 as well as <em>Pb-Anju3</em> are identified from the R block of chromosome C03 of <em>B. oleracea</em> (Fig. <a reveal-id="DSV034F6" data-open="DSV034F6" class="link link-reveal link-table xref-fig">6</a>B). It is likely that the paralogous gene of <em>CRc</em> in the R block of chromosome C03 can be the candidate clubroot resistance gene in <em>B. oleracea</em>.</p><p>Among five QTLs identified in <em>B. oleracea</em> by Nagaoka et al.,<sup><a reveal-id="DSV034C30" data-open="DSV034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> who used Race 4 for inoculation tests, <em>PbBo</em>(<em>Anju</em>)1, which had a maximum LOD score, and <em>PbBo</em>(<em>Anju</em>)3, regarded as a minor effect QTL, seemed to overlap with CRQTL-GN_1 and CRQTL-GN_2 (or CRQTL-YC), respectively. This, despite the difference in pathogen tested, two overlapping QTLs were identified, suggesting that CRQTL-GN_1 and CRQTL-GN_2 could confer resistance to Races 2, 4, and Races 2, 4, 9, respectively. In the future, cloning of CR genes and QTL analyses using more diverse races are required to characterize the relationship between clubroot resistance and the breeding materials in both <em>B. oleracea</em> and <em>B. rapa</em>.</p>                            <h3 scrollto-destination=87198107 id="87198107" class="section-title" data-legacy-id=s4d>4.4. Conclusions</h3>
<p>We performed GBS for high-resolution mapping using an F2 population between two elite cabbage breeding lines with narrow genetic diversity. We constructed a high-resolution genetic map using 4,103 stringently filtered GBS-SNPs and 32 reference markers. Based on comparison of the high-resolution map and the position of each marker on the reference genome, we identified 37 putatively mislocated scaffolds and their suggested revised positions and also could anchor two unassigned scaffolds, thereby improving the reference genome sequence of cabbage. Moreover, two QTL regions were identified for clubroot resistance against two different <em>P. brassicae</em> isolates. We thus demonstrated that GBS can successfully be applied for rapid genotyping of many individuals for molecular breeding and for genetics and genomics research in cabbage.</p>                            <h2 scrollto-destination=87198109 id="87198109" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was supported by the Golden Seed Project (Center for Horticultural Seed Development, No. 213003-04-3-SB430) of the Ministry of Agriculture, Food and Rural Affairs (MAFRA) in the Republic of Korea and the Next Biogreen 21 program (PJ01101101) of the Rural Development Administration (RDA) and the Korea Forest Service (KFS). Funding to pay the Open Access publication charges for this article was provided by Golden Seed Project (Center for Horticultural Seed Development, No. 213003-04-3-SB430) of the Ministry of Agriculture, Food and Rural Affairs (MAFRA) in the Republic of Korea.</p>                            <h2 scrollto-destination=87198111 id="87198111" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank all lab-members in Laboratory of Functional Crop Genomics and Biotechnology, Seoul National University for their technical assistance.</p>                            <h2 scrollto-destination=87198113 id="87198113" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV034C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Initiative</div>
               <div class="given-names">A.G.</div>
            </div>
          </div>
          <div class="year">2000</div>, <div class="article-title">Analysis of the genome sequence of the flowering plant <em>Arabidopsis thaliana</em></div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20genome%20sequence%20of%20the%20flowering%20plant%20Arabidopsis%20thaliana&amp;author=A.G.%20Initiative&amp;publication_year=2000&amp;journal=Nature&amp;volume=408&amp;pages=796" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Feuillet</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Leach</div> <div class="given-names">J.E.</div></div>, <div class="name string-name"><div class="surname">Rogers</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schnable</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Eversole</div> <div class="given-names">K.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Crop genome sequencing: lessons and rationales</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">16</div>, <div class="fpage">77</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crop%20genome%20sequencing%3A%20lessons%20and%20rationales&amp;author=C.%20Feuillet&amp;author=J.E.%20Leach&amp;author=J.%20Rogers&amp;author=P.S.%20Schnable&amp;author=K.%20Eversole&amp;publication_year=2011&amp;journal=Trends%20Plant%20Sci.&amp;volume=16&amp;pages=77-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.10.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.10.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.10.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21081278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Elshire</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e19379</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20robust%2C%20simple%20genotyping-by-sequencing%20(GBS)%20approach%20for%20high%20diversity%20species&amp;author=R.J.%20Elshire&amp;author=J.C.%20Glaubitz&amp;author=Q.%20Sun&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e19379" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0019379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0019379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0019379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21573248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crossa</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Beyene</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kassa</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing</div>, <div class="source">Genes Genomes Genetics</div> , <div class="volume">3</div>, <div class="fpage">1903</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20prediction%20in%20maize%20breeding%20populations%20with%20genotyping-by-sequencing&amp;author=J.%20Crossa&amp;author=Y.%20Beyene&amp;author=S.%20Kassa&amp;publication_year=2013&amp;journal=Genes%20Genomes%20Genetics&amp;volume=3&amp;pages=1903-26" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24022750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Romay</div>
               <div class="given-names">M.C.</div>
            </div>, <div class="name string-name"><div class="surname">Millard</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Glaubitz</div> <div class="given-names">J.C.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Comprehensive genotyping of the USA national maize inbred seed bank</div>, <div class="source">Genome Biol.</div> , <div class="volume">14</div>, <div class="fpage">R55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20genotyping%20of%20the%20USA%20national%20maize%20inbred%20seed%20bank&amp;author=M.C.%20Romay&amp;author=M.J.%20Millard&amp;author=J.C.%20Glaubitz&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-6-r55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-6-r55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-6-r55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23759205" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sonah</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Bastien</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Iquira</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">An improved genotyping by sequencing (GBS) approach offering increased versatility and efficiency of SNP discovery and genotyping</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e54603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20genotyping%20by%20sequencing%20(GBS)%20approach%20offering%20increased%20versatility%20and%20efficiency%20of%20SNP%20discovery%20and%20genotyping&amp;author=H.%20Sonah&amp;author=M.%20Bastien&amp;author=E.%20Iquira&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e54603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0054603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0054603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0054603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23372741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Poland</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Sorrells</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Jannink</div> <div class="given-names">J.-L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e32253</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20high-density%20genetic%20maps%20for%20barley%20and%20wheat%20using%20a%20novel%20two-enzyme%20genotyping-by-sequencing%20approach&amp;author=J.A.%20Poland&amp;author=P.J.%20Brown&amp;author=M.E.%20Sorrells&amp;author=J.-L.%20Jannink&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e32253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0032253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0032253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0032253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Spindel</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Bridging the genotyping gap: using genotyping by sequencing (GBS) to add high-density SNP markers and new value to traditional bi-parental mapping and breeding populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">126</div>, <div class="fpage">2699</div>–<div class="lpage">716</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bridging%20the%20genotyping%20gap%3A%20using%20genotyping%20by%20sequencing%20(GBS)%20to%20add%20high-density%20SNP%20markers%20and%20new%20value%20to%20traditional%20bi-parental%20mapping%20and%20breeding%20populations&amp;author=J.%20Spindel&amp;author=M.%20Wright&amp;author=C.%20Chen&amp;publication_year=2013&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=126&amp;pages=2699-716" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-013-2166-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-013-2166-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-013-2166-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23918062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mascher</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Amand</div> <div class="given-names">P.S.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Poland</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Application of genotyping-by-sequencing on semiconductor sequencing platforms: a comparison of genetic and reference-based marker ordering in barley</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e76925</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20genotyping-by-sequencing%20on%20semiconductor%20sequencing%20platforms%3A%20a%20comparison%20of%20genetic%20and%20reference-based%20marker%20ordering%20in%20barley&amp;author=M.%20Mascher&amp;author=S.%20Wu&amp;author=P.S.%20Amand&amp;author=N.%20Stein&amp;author=J.%20Poland&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e76925" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0076925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0076925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0076925"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24098570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="collab">U, N.</div>
          <div class="year">1935</div>, <div class="article-title">Genome analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilization</div>, <div class="source">J. Jpn Bot.</div> , <div class="volume">7</div>, <div class="fpage">389</div>–<div class="lpage">452</div>.</div></div></div></div><div content-id="DSV034C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">W.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Construction and analysis of a high-density genetic linkage map in cabbage (<em>Brassica oleracea</em> L. var. <em>capitata</em>)</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">523</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20analysis%20of%20a%20high-density%20genetic%20linkage%20map%20in%20cabbage%20(Brassica%20oleracea%20L.%20var.%20capitata)&amp;author=W.%20Wang&amp;author=S.%20Huang&amp;author=Y.%20Liu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23033896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">The <em>Brassica oleracea</em> genome reveals the asymmetrical evolution of polyploid genomes</div>, <div class="source">Nat. Commun.</div> , <div class="volume">5</div>, <div class="fpage">3930</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Brassica%20oleracea%20genome%20reveals%20the%20asymmetrical%20evolution%20of%20polyploid%20genomes&amp;author=S.%20Liu&amp;author=Y.%20Liu&amp;author=X.%20Yang&amp;publication_year=2014&amp;journal=Nat.%20Commun.&amp;volume=5&amp;pages=3930" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24852848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parkin</div>
               <div class="given-names">I.A.</div>
            </div>, <div class="name string-name"><div class="surname">Koh</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid <em>Brassica oleracea</em></div>, <div class="source">Genome Biol.</div> , <div class="volume">15</div>, <div class="fpage">R77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20and%20methylome%20profiling%20reveals%20relics%20of%20genome%20dominance%20in%20the%20mesopolyploid%20Brassica%20oleracea&amp;author=I.A.%20Parkin&amp;author=C.%20Koh&amp;author=H.%20Tang&amp;publication_year=2014&amp;journal=Genome%20Biol.&amp;volume=15&amp;pages=R77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-6-r77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-6-r77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-6-r77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24916971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crute</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Gray</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Crisp</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Buczacki</div> <div class="given-names">S.</div></div></div>
          <div class="year">1980</div>, <div class="article-title">Variation in <em>Plasmodiophora brassicae</em> and resistance to clubroot disease in brassicas and allied crops-a critical review</div>, <div class="source">Plant Breed. Abst.</div> , <div class="volume">50</div>, <div class="fpage">91</div>–<div class="lpage">104</div>.</div></div></div></div><div content-id="DSV034C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dixon</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">D.</div></div></div>
          <div class="year">1986</div>, <div class="article-title">The susceptibility of <em>Brassica oleracea</em> cultivars to <em>Plasmodiophora brassicae</em> (clubroot)</div>, <div class="source">Plant Pathol.</div> , <div class="volume">35</div>, <div class="fpage">101</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20susceptibility%20of%20Brassica%20oleracea%20cultivars%20to%20Plasmodiophora%20brassicae%20(clubroot)&amp;author=G.%20Dixon&amp;author=D.%20Robinson&amp;publication_year=1986&amp;journal=Plant%20Pathol.&amp;volume=35&amp;pages=101-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppa.1986.35.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppa.1986.35.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppa.1986.35.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSV034C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voorrips</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">1995</div>, <div class="article-title"><em>Plasmodiophora brassicae</em>: aspects of pathogenesis and resistance in <em>Brassica oleracea</em></div>, <div class="source">Euphytica</div> , <div class="volume">83</div>, <div class="fpage">139</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plasmodiophora%20brassicae%3A%20aspects%20of%20pathogenesis%20and%20resistance%20in%20Brassica%20oleracea&amp;author=R.E.%20Voorrips&amp;publication_year=1995&amp;journal=Euphytica&amp;volume=83&amp;pages=139-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01678041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01678041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01678041"> </span></div> </div></div></div></div></div><div content-id="DSV034C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piao</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Lim</div> <div class="given-names">Y.P.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Genetics of clubroot resistance in <em>Brassica</em> species</div>, <div class="source">J. Plant Growth Regul.</div> , <div class="volume">28</div>, <div class="fpage">252</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20clubroot%20resistance%20in%20Brassica%20species&amp;author=Z.%20Piao&amp;author=N.%20Ramchiary&amp;author=Y.P.%20Lim&amp;publication_year=2009&amp;journal=J.%20Plant%20Growth%20Regul.&amp;volume=28&amp;pages=252-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00344-009-9093-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00344-009-9093-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00344-009-9093-8"> </span></div> </div></div></div></div></div><div content-id="DSV034C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hirai</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Harada</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Tsukada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">A novel locus for clubroot resistance in <em>Brassica rapa</em> and its linkage markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">108</div>, <div class="fpage">639</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20locus%20for%20clubroot%20resistance%20in%20Brassica%20rapa%20and%20its%20linkage%20markers&amp;author=M.%20Hirai&amp;author=T.%20Harada&amp;author=N.%20Kubo&amp;author=M.%20Tsukada&amp;author=K.%20Suwabe&amp;author=S.%20Matsumoto&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=639-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1475-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1475-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1475-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14551685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piao</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lim</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">SCAR and CAPS mapping of <em>CRb</em>, a gene conferring resistance to <em>Plasmodiophora brassicae</em> in Chinese cabbage (<em>Brassica rapa</em> ssp. <em>pekinensis</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">108</div>, <div class="fpage">1458</div>–<div class="lpage">1465</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SCAR%20and%20CAPS%20mapping%20of%20CRb%2C%20a%20gene%20conferring%20resistance%20to%20Plasmodiophora%20brassicae%20in%20Chinese%20cabbage%20(Brassica%20rapa%20ssp.%20pekinensis)&amp;author=Z.%20Piao&amp;author=Y.%20Deng&amp;author=S.%20Choi&amp;author=Y.%20Park&amp;author=Y.%20Lim&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1458-1465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1577-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1577-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1577-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14997298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Suwabe</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Tsukazaki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Simple sequence repeat-based comparative genomics between <em>Brassica rapa</em> and <em>Arabidopsis thaliana</em>: the genetic origin of clubroot resistance</div>, <div class="source">Genetics</div> , <div class="volume">173</div>, <div class="fpage">309</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat-based%20comparative%20genomics%20between%20Brassica%20rapa%20and%20Arabidopsis%20thaliana%3A%20the%20genetic%20origin%20of%20clubroot%20resistance&amp;author=K.%20Suwabe&amp;author=H.%20Tsukazaki&amp;author=H.%20Iketani&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=309-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.038968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.038968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.038968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16723420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakamoto</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hayashida</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Taguchi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">E.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Mapping of isolate-specific QTLs for clubroot resistance in Chinese cabbage (<em>Brassica rapa</em> L. ssp. <em>pekinensis</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">117</div>, <div class="fpage">759</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20isolate-specific%20QTLs%20for%20clubroot%20resistance%20in%20Chinese%20cabbage%20(Brassica%20rapa%20L.%20ssp.%20pekinensis)&amp;author=K.%20Sakamoto&amp;author=A.%20Saito&amp;author=N.%20Hayashida&amp;author=G.%20Taguchi&amp;author=E.%20Matsumoto&amp;publication_year=2008&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=117&amp;pages=759-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0817-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0817-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0817-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18612625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saito</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tsukada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Fine mapping of the clubroot resistance gene, <em>Crr3</em>, in <em>Brassica rapa</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">114</div>, <div class="fpage">81</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20the%20clubroot%20resistance%20gene%2C%20Crr3%2C%20in%20Brassica%20rapa&amp;author=M.%20Saito&amp;author=N.%20Kubo&amp;author=S.%20Matsumoto&amp;author=K.%20Suwabe&amp;author=M.%20Tsukada&amp;author=M.%20Hirai&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=81-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0412-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0412-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0412-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17039346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kato</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Hatakeyama</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fukino</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Fine mapping of the clubroot resistance gene <em>CRb</em> and development of a useful selectable marker in <em>Brassica rapa</em></div>, <div class="source">Breed. Sci.</div> , <div class="volume">63</div>, <div class="fpage">116</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20the%20clubroot%20resistance%20gene%20CRb%20and%20development%20of%20a%20useful%20selectable%20marker%20in%20Brassica%20rapa&amp;author=T.%20Kato&amp;author=K.%20Hatakeyama&amp;author=N.%20Fukino&amp;author=S.%20Matsumoto&amp;publication_year=2013&amp;journal=Breed.%20Sci.&amp;volume=63&amp;pages=116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23641188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Hirani</div> <div class="given-names">A.H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Fine mapping a clubroot resistance locus in Chinese cabbage</div>, <div class="source">J. Am. Soc. Hort. Sci.</div> , <div class="volume">139</div>, <div class="fpage">247</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="DSV034C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Fine genetic and physical mapping of the <em>CRb</em> gene conferring resistance to clubroot disease in <em>Brassica rapa</em></div>, <div class="source">Mol. Breed.</div> , <div class="volume">34</div>, <div class="fpage">1173</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20genetic%20and%20physical%20mapping%20of%20the%20CRb%20gene%20conferring%20resistance%20to%20clubroot%20disease%20in%20Brassica%20rapa&amp;author=T.%20Zhang&amp;author=Z.%20Zhao&amp;author=C.%20Zhang&amp;publication_year=2014&amp;journal=Mol.%20Breed.&amp;volume=34&amp;pages=1173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-014-0108-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-014-0108-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-014-0108-1"> </span></div> </div></div></div></div></div><div content-id="DSV034C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crisp</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Crute</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Sutherland</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">1989</div>, <div class="article-title">The exploitation of genetic resources of <em>Brassica oleracea</em> in breeding for resistance to clubroot (<em>Plasmodiophora brassicae</em>)</div>, <div class="source">Euphytica</div> , <div class="volume">42</div>, <div class="fpage">215</div>–<div class="lpage">26</div>.</div></div></div></div><div content-id="DSV034C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kuginuki</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Ajisaka</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yui</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yoshikawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hida</div> <div class="given-names">K.-i.</div></div>, <div class="name string-name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">RAPD markers linked to a clubroot-resistance locus in <em>Brassica rapa</em> L</div>, <div class="source">Euphytica</div> , <div class="volume">98</div>, <div class="fpage">149</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAPD%20markers%20linked%20to%20a%20clubroot-resistance%20locus%20in%20Brassica%20rapa%20L&amp;author=Y.%20Kuginuki&amp;author=H.%20Ajisaka&amp;author=M.%20Yui&amp;author=H.%20Yoshikawa&amp;author=K.-i.%20Hida&amp;author=M.%20Hirai&amp;publication_year=1997&amp;journal=Euphytica&amp;volume=98&amp;pages=149-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1003147815692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1003147815692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1003147815692"> </span></div> </div></div></div></div></div><div content-id="DSV034C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhong Yun</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Yong Joo</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Su Ryun</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">Conversion of an AFLP marker linked to clubroot resistance gene in Chinese cabbage into a SCAR marker</div>, <div class="source">Hort. Environ. Biotechnol.</div> , <div class="volume">43</div>, <div class="fpage">653</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="DSV034C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Jing</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhan</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Piao</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Identification of Novel QTLs for Isolate-specific partial resistance to <em>Plasmodiophora brassicae</em> in <em>Brassica rapa</em></div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e85307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20Novel%20QTLs%20for%20Isolate-specific%20partial%20resistance%20to%20Plasmodiophora%20brassicae%20in%20Brassica%20rapa&amp;author=J.%20Chen&amp;author=J.%20Jing&amp;author=Z.%20Zhan&amp;author=T.%20Zhang&amp;author=C.%20Zhang&amp;author=Z.%20Piao&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e85307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0085307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0085307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0085307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24376876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nagaoka</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Doullah</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Identification of QTLs that control clubroot resistance in <em>Brassica oleracea</em> and comparative analysis of clubroot resistance genes between <em>B. rapa</em> and <em>B. oleracea</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">120</div>, <div class="fpage">1335</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20QTLs%20that%20control%20clubroot%20resistance%20in%20Brassica%20oleracea%20and%20comparative%20analysis%20of%20clubroot%20resistance%20genes%20between%20B.%20rapa%20and%20B.%20oleracea&amp;author=T.%20Nagaoka&amp;author=M.%20Doullah&amp;author=S.%20Matsumoto&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=120&amp;pages=1335-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1259-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1259-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1259-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20069415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Flores-Vergara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Krasynanski</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Thompson</div> <div class="given-names">W.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">A modified protocol for rapid DNA isolation from plant tissues using cetyltrimethylammonium bromide</div>, <div class="source">Nat. Protocols</div> , <div class="volume">1</div>, <div class="fpage">2320</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20modified%20protocol%20for%20rapid%20DNA%20isolation%20from%20plant%20tissues%20using%20cetyltrimethylammonium%20bromide&amp;author=G.%20Allen&amp;author=M.%20Flores-Vergara&amp;author=S.%20Krasynanski&amp;author=S.%20Kumar&amp;author=W.%20Thompson&amp;publication_year=2006&amp;journal=Nat.%20Protocols&amp;volume=1&amp;pages=2320-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2006.384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2006.384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2006.384"> </span></div> </div></div></div></div></div><div content-id="DSV034C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows–Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McKenna</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lee</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Izzah</div> <div class="given-names">N.K.</div></div>, <div class="name string-name"><div class="surname">Jayakodi</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Genome-wide SNP identification and QTL mapping for black rot resistance in cabbage</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">15</div>, <div class="fpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20identification%20and%20QTL%20mapping%20for%20black%20rot%20resistance%20in%20cabbage&amp;author=J.%20Lee&amp;author=N.K.%20Izzah&amp;author=M.%20Jayakodi&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol.&amp;volume=15&amp;pages=32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0424-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0424-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0424-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25644124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Izzah</div>
               <div class="given-names">N.K.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jayakodi</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Transcriptome sequencing of two parental lines of cabbage (<em>Brassica oleracea</em> L. var. <em>capitata</em> L.) and construction of an EST-based genetic map</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">149</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20two%20parental%20lines%20of%20cabbage%20(Brassica%20oleracea%20L.%20var.%20capitata%20L.)%20and%20construction%20of%20an%20EST-based%20genetic%20map&amp;author=N.K.%20Izzah&amp;author=J.%20Lee&amp;author=M.%20Jayakodi&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-149" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-149" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-149"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24559437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lowe</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Moule</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Trick</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">K.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Efficient large-scale development of microsatellites for marker and mapping applications in <em>Brassica</em> crop species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">108</div>, <div class="fpage">1103</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20large-scale%20development%20of%20microsatellites%20for%20marker%20and%20mapping%20applications%20in%20Brassica%20crop%20species&amp;author=A.%20Lowe&amp;author=C.%20Moule&amp;author=M.%20Trick&amp;author=K.%20Edwards&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=108&amp;pages=1103-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1522-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1522-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1522-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15067397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Piquemal</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Cinquin</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Couton</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Construction of an oilseed rape (<em>Brassica napus</em> L.) genetic map with SSR markers</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">111</div>, <div class="fpage">1514</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20an%20oilseed%20rape%20(Brassica%20napus%20L.)%20genetic%20map%20with%20SSR%20markers&amp;author=J.%20Piquemal&amp;author=E.%20Cinquin&amp;author=F.%20Couton&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1514-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0080-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0080-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0080-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16187118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Choi</div>
               <div class="given-names">S.R.</div>
            </div>, <div class="name string-name"><div class="surname">Teakle</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Plaha</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">The reference genetic linkage map for the multinational <em>Brassica rapa</em> genome sequencing project</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">115</div>, <div class="fpage">777</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genetic%20linkage%20map%20for%20the%20multinational%20Brassica%20rapa%20genome%20sequencing%20project&amp;author=S.R.%20Choi&amp;author=G.R.%20Teakle&amp;author=P.%20Plaha&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=777-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0608-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0608-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0608-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Louarn</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Torp</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Holme</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Andersen</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Jensen</div> <div class="given-names">B.D.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Database derived microsatellite markers (SSRs) for cultivar differentiation in <em>Brassica oleracea</em></div>, <div class="source">Genet. Resourc. Crop Evol.</div> , <div class="volume">54</div>, <div class="fpage">1717</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Database%20derived%20microsatellite%20markers%20(SSRs)%20for%20cultivar%20differentiation%20in%20Brassica%20oleracea&amp;author=S.%20Louarn&amp;author=A.M.%20Torp&amp;author=I.%20Holme&amp;author=S.B.%20Andersen&amp;author=B.D.%20Jensen&amp;publication_year=2007&amp;journal=Genet.%20Resourc.%20Crop%20Evol.&amp;volume=54&amp;pages=1717-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10722-006-9181-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10722-006-9181-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10722-006-9181-6"> </span></div> </div></div></div></div></div><div content-id="DSV034C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Development and genetic mapping of microsatellite markers from genome survey sequences in <em>Brassica napus</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">118</div>, <div class="fpage">1121</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20genetic%20mapping%20of%20microsatellite%20markers%20from%20genome%20survey%20sequences%20in%20Brassica%20napus&amp;author=X.%20Cheng&amp;author=J.%20Xu&amp;author=S.%20Xia&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1121-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-0967-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-0967-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-0967-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19190889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sampath</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.-C.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Characterization of a new high copy <em>Stowaway</em> family MITE, <em>BRAMI-1</em> in <em>Brassica</em> genome</div>, <div class="source">BMC Plant Biol.</div> , <div class="volume">13</div>, <div class="fpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20new%20high%20copy%20Stowaway%20family%20MITE%2C%20BRAMI-1%20in%20Brassica%20genome&amp;author=P.%20Sampath&amp;author=S.-C.%20Lee&amp;author=J.%20Lee&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol.&amp;volume=13&amp;pages=56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-13-56" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-56" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-13-56"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23547712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sampath</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Murukarthick</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Izzah</div> <div class="given-names">N.K.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome-wide comparative analysis of 20 miniature inverted-repeat transposable element families in <em>Brassica rapa</em> and <em>B. oleracea</em></div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e94499</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20analysis%20of%2020%20miniature%20inverted-repeat%20transposable%20element%20families%20in%20Brassica%20rapa%20and%20B.%20oleracea&amp;author=P.%20Sampath&amp;author=J.%20Murukarthick&amp;author=N.K.%20Izzah&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e94499" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0094499" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0094499" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0094499"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24747717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">SuJung</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">SunAh</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">KyoungSoo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">YongHo</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">JinCheol</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">GyungJa</div> <div class="given-names">C.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Resistance of cultivars of Chinese cabbage to <em>Plasmodiophora brassicae</em> isolates of several races collected in Korea</div>, <div class="source">Kor. J. Hort. Sci. Technol.</div> , <div class="volume">29</div>, <div class="fpage">610</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSV034C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Williams</div>
               <div class="given-names">P.H.</div>
            </div>
          </div>
          <div class="year">1966</div>, <div class="article-title">A system for the determination of races of <em>Plasmodiophora brassicae</em> that infect Cabbage and Rutabaga</div>, <div class="source">Phytopathology</div> , <div class="volume">56</div>, <div class="fpage">624</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="DSV034C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schranz</div>
               <div class="given-names">M.E.</div>
            </div>, <div class="name string-name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">11</div>, <div class="fpage">535</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ABC's%20of%20comparative%20genomics%20in%20the%20Brassicaceae%3A%20building%20blocks%20of%20crucifer%20genomes&amp;author=M.E.%20Schranz&amp;author=M.A.%20Lysak&amp;author=T.%20Mitchell-Olds&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=535-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Matsumoto</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Yasui</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ohi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tsukada</div> <div class="given-names">M.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Linkage analysis of RFLP markers for clubroot resistance and pigmentation in Chinese cabbage (<em>Brassica rapa</em> ssp. <em>pekinensis</em>)</div>, <div class="source">Euphytica</div> , <div class="volume">104</div>, <div class="fpage">79</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20analysis%20of%20RFLP%20markers%20for%20clubroot%20resistance%20and%20pigmentation%20in%20Chinese%20cabbage%20(Brassica%20rapa%20ssp.%20pekinensis)&amp;author=E.%20Matsumoto&amp;author=C.%20Yasui&amp;author=M.%20Ohi&amp;author=M.%20Tsukada&amp;publication_year=1998&amp;journal=Euphytica&amp;volume=104&amp;pages=79-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1018370418201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1018370418201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1018370418201"> </span></div> </div></div></div></div></div><div content-id="DSV034C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Suwabe</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Tsukazaki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Identification of two loci for resistance to clubroot (<em>Plasmodiophora brassicae</em> Woronin) in <em>Brassica rapa</em> L</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">107</div>, <div class="fpage">997</div>–<div class="lpage">1002</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20two%20loci%20for%20resistance%20to%20clubroot%20(Plasmodiophora%20brassicae%20Woronin)%20in%20Brassica%20rapa%20L&amp;author=K.%20Suwabe&amp;author=H.%20Tsukazaki&amp;author=H.%20Iketani&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=997-1002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1309-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1309-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1309-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12955203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em></div>, <div class="source">Nat. Genet.</div> , <div class="volume">43</div>, <div class="fpage">1035</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hatakeyama</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tomita</div> <div class="given-names">R.N.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Identification and characterization of <em>Crr1a</em>, a gene for resistance to clubroot disease (<em>Plasmodiophora brassicae</em> Woronin) in <em>Brassica rapa</em> L</div>, <div class="source">PLoS ONE</div> , <div class="volume">8</div>, <div class="fpage">e54745</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20Crr1a%2C%20a%20gene%20for%20resistance%20to%20clubroot%20disease%20(Plasmodiophora%20brassicae%20Woronin)%20in%20Brassica%20rapa%20L&amp;author=K.%20Hatakeyama&amp;author=K.%20Suwabe&amp;author=R.N.%20Tomita&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e54745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0054745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0054745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0054745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23382954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="collab">International Rice Genome Sequencing Project.</div>
          <div class="year">2005</div>, <div class="article-title">The map-based sequence of the rice genome</div>, <div class="source">Nature</div> , <div class="volume">436</div>, <div class="fpage">793</div>–<div class="lpage">800</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03895" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03895" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03895"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16100779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">T.-J.</div>
            </div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nah</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">Construction and utility of 10-kb libraries for efficient clone-gap closure for rice genome sequencing</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="volume">107</div>, <div class="fpage">652</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20utility%20of%2010-kb%20libraries%20for%20efficient%20clone-gap%20closure%20for%20rice%20genome%20sequencing&amp;author=T.-J.%20Yang&amp;author=Y.%20Yu&amp;author=G.%20Nah&amp;publication_year=2003&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=107&amp;pages=652-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-003-1302-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-003-1302-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-003-1302-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12783166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Town</div>
               <div class="given-names">C.D.</div>
            </div>, <div class="name string-name"><div class="surname">Cheung</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Maiti</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Comparative genomics of <em>Brassica oleracea</em> and <em>Arabidopsis thaliana</em> reveal gene loss, fragmentation, and dispersal after polyploidy</div>, <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">1348</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20of%20Brassica%20oleracea%20and%20Arabidopsis%20thaliana%20reveal%20gene%20loss%2C%20fragmentation%2C%20and%20dispersal%20after%20polyploidy&amp;author=C.D.%20Town&amp;author=F.%20Cheung&amp;author=R.%20Maiti&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1348-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.106.041665" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.106.041665" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.106.041665"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">T.-J.</div>
            </div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Kwon</div> <div class="given-names">S.-J.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Sequence-level analysis of the diploidization process in the triplicated <em>FLOWERING LOCUS C</em> region of <em>Brassica rapa</em></div>, <div class="source">Plant Cell</div> , <div class="volume">18</div>, <div class="fpage">1339</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-level%20analysis%20of%20the%20diploidization%20process%20in%20the%20triplicated%20FLOWERING%20LOCUS%20C%20region%20of%20Brassica%20rapa&amp;author=T.-J.%20Yang&amp;author=J.S.%20Kim&amp;author=S.-J.%20Kwon&amp;publication_year=2006&amp;journal=Plant%20Cell&amp;volume=18&amp;pages=1339-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.105.040535" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.105.040535" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.105.040535"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16632644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Poland</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Rife</div> <div class="given-names">T.W.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Genotyping-by-sequencing for plant breeding and genetics</div>, <div class="source">Plant Genome</div> , <div class="volume">5</div>, <div class="fpage">92</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping-by-sequencing%20for%20plant%20breeding%20and%20genetics&amp;author=J.A.%20Poland&amp;author=T.W.%20Rife&amp;publication_year=2012&amp;journal=Plant%20Genome&amp;volume=5&amp;pages=92-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2012.05.0005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2012.05.0005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2012.05.0005"> </span></div> </div></div></div></div></div><div content-id="DSV034C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andolfatto</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Davison</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Erezyilmaz</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">610</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplexed%20shotgun%20genotyping%20for%20rapid%20and%20efficient%20genetic%20mapping&amp;author=P.%20Andolfatto&amp;author=D.%20Davison&amp;author=D.%20Erezyilmaz&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=610-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.115402.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.115402.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.115402.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21233398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV034C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Landry</div>
               <div class="given-names">B.S.</div>
            </div>, <div class="name string-name"><div class="surname">Hubert</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Crete</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Lincoln</div> <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Etoh</div> <div class="given-names">T.</div></div></div>
          <div class="year">1992</div>, <div class="article-title">A genetic map for <em>Brassica oleracea</em> based on RFLP markers detected with expressed DNA sequences and mapping of resistance genes to race 2 of <em>Plasmodiophora brassicae</em> (Woronin)</div>, <div class="source">Genome</div> , <div class="volume">35</div>, <div class="fpage">409</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20map%20for%20Brassica%20oleracea%20based%20on%20RFLP%20markers%20detected%20with%20expressed%20DNA%20sequences%20and%20mapping%20of%20resistance%20genes%20to%20race%202%20of%20Plasmodiophora%20brassicae%20(Woronin)&amp;author=B.S.%20Landry&amp;author=N.%20Hubert&amp;author=R.%20Crete&amp;author=M.S.%20Chang&amp;author=S.E.%20Lincoln&amp;author=T.%20Etoh&amp;publication_year=1992&amp;journal=Genome&amp;volume=35&amp;pages=409-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g92-061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg92-061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg92-061"> </span></div> </div></div></div></div></div><div content-id="DSV034C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV034C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Mandáková</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Deciphering the diploid ancestral genome of the mesohexaploid <em>Brassica rapa</em></div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">1541</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20the%20diploid%20ancestral%20genome%20of%20the%20mesohexaploid%20Brassica%20rapa&amp;author=F.%20Cheng&amp;author=T.%20Mand%C3%A1kov%C3%A1&amp;author=J.%20Wu&amp;author=Q.%20Xie&amp;author=M.A.%20Lysak&amp;author=X.%20Wang&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=1541-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.110486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.110486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.110486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23653472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462430&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv034/1/dsv034_Supplementary_Data.zip?Expires=2147483647&amp;Signature=33qKEuRsYZqMrrVlD7ldl3-svUI0Wpr50vtIQ~qTOEcEBf4OiTZ~qSXrlQmFMRH9zf23eum2HMxYZpl6yp4IUgiIaLuIFMUhXe~9HmrG1FDcA7Yd4x0Fyt1DYVr~k~JDLRh3zpXMYetdsAav6gdLmoyTo23u-utTXWA4rqqWECf6fizRK4jkd0audAk3h2y1ysyzaxvbFiqkzScV8RL3yC6zeUAVQ4BJDU0h75wNr4RudOQSK7vwY7K~HPe2froqJT0rHtrDRdZSeqmEw9if02cBrXsm3-pput~28hAkjo15u-cTTT9kH-HKdnFiolZVm4CgFhgQSdPPPXNJHcZ6Pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">986</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">18</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv034"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462430"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f1%2f29%2f2462430">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000370973100004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (18)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F1%2F29%2F2462430" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462430'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genotyping-by-sequencing map permits identification of clubroot resistance QTLs and revision of the reference genome assembly in cabbage (Brassica oleracea L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv034");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f29%2f2462430">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

