

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/3/245/475122","@type":"ScholarlyArticle","name":"Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions","datePublished":"2012-02-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2012-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss008","keywords":["tRNA","identity element prediction","extended consensus partition"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/3/245/475122","author":[{"name":"Szenes, Áron","affiliation":"1  Department of Biochemistry ,  Eötvös University ,  Pázmány P. stny. 1/C. ,  Budapest H-1117 ,  Hungary    2  Laboratory of Proteomics ,  Eötvös University ,  Budapest H-1117 ,  Hungary","@type":"Person"},{"name":"Pál, Gábor","affiliation":"1  Department of Biochemistry ,  Eötvös University ,  Pázmány P. stny. 1/C. ,  Budapest H-1117 ,  Hungary","@type":"Person"}],"description":"Abstract.  The recently published discrete mathematical method, extended consensus partition (ECP), identifies nucleotide types at each position that are strict","pageStart":"245","pageEnd":"258","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions" />
    <meta property="og:description" content="Abstract.  The recently published discrete mathematical method, extended consensus partition (ECP), identifies nucleotide types at each position that are strict" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/3/245/475122" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Szenes, Áron" /><meta name="citation_author_institution" content="Department of Biochemistry, Eötvös University, Pázmány P. stny. 1/C., Budapest H-1117, Hungary" /><meta name="citation_author_institution" content="Laboratory of Proteomics, Eötvös University, Budapest H-1117, Hungary" /><meta name="citation_author" content="Pál, Gábor" /><meta name="citation_author_institution" content="Department of Biochemistry, Eötvös University, Pázmány P. stny. 1/C., Budapest H-1117, Hungary" /><meta name="citation_title" content="Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions" /><meta name="citation_firstpage" content="245" /><meta name="citation_lastpage" content="258" /><meta name="citation_doi" content="10.1093/dnares/dss008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2012/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Recognition of tRNAs by aminoacyl-tRNA synthetases; FASEB J.;  citation_year=1993;  citation_volume=7;  citation_pages=79-86; " /><meta name="citation_reference" content="citation_title=Rules that govern tRNA identity in protein synthesis; J. Mol. Biol.;  citation_year=1993;  citation_volume=234;  citation_pages=257-80; " /><meta name="citation_reference" content="citation_title=Universal rules and idiosyncratic features in tRNA identity; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=5017-35; " /><meta name="citation_reference" content="citation_title=Aminoacyl-tRNA synthesis; Annu. Rev. Biochem.;  citation_year=2000;  citation_volume=69;  citation_pages=617-50; " /><meta name="citation_reference" content="citation_title=Toward a more complete view of tRNA biology; Nat. Struct. Mol. Biol.;  citation_year=2008;  citation_volume=15;  citation_pages=1007-14; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=GtRNAdb: a database of transfer RNA genes detected in genomic sequence; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D93-7; " /><meta name="citation_reference" content="citation_title=tRNAdb 2009: compilation of tRNA sequences and tRNA genes; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D159-62; " /><meta name="citation_reference" content="citation_title=tRNADB-CE 2011: tRNA gene database curated manually by experts; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=D210-3; " /><meta name="citation_reference" content="citation_title=A statistical method for correlating tRNA sequence with amino acid specificity; Nucleic Acids Res.;  citation_year=1986;  citation_volume=14;  citation_pages=375-80; " /><meta name="citation_reference" content="citation_title=Searching tRNA sequences for relatedness to aminoacyl-tRNA synthetase families; J. Mol. Evol.;  citation_year=1995;  citation_volume=40;  citation_pages=482-6; " /><meta name="citation_reference" content="citation_title=RNA sequence analysis using covariance models; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=2079-88; " /><meta name="citation_reference" content="citation_title=TFAM detects co-evolution of tRNA identity rules with lateral transfer of histidyl-tRNA synthetase; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=893-904; " /><meta name="citation_reference" content="citation_title=Visualizing bacterial tRNA identity determinants and antideterminants using function logos and inverse function logos; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=905-16; " /><meta name="citation_reference" content="citation_title=New computational methods reveal tRNA identity element divergence between Proteobacteria and Cyanobacteria; Biochimie;  citation_year=2007;  citation_volume=89;  citation_pages=1276-88; " /><meta name="citation_reference" content="citation_title=Computational analysis of tRNA identity; FEBS Lett.;  citation_year=2010;  citation_volume=584;  citation_pages=325-33; " /><meta name="citation_reference" content="citation_title=In silico detection of tRNA sequence features characteristic to aminoacyl-tRNA synthetase class membership; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=5593-609; " /><meta name="citation_reference" content="citation_title=Tri-split tRNA is a transfer RNA made from 3 transcripts that provides insight into the evolution of fragmented tRNAs in archaea; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=2683-7; " /><meta name="citation_reference" content="citation_title=Comprehensive analysis of archaeal tRNA genes reveals rapid increase of tRNA introns in the order thermoproteales; Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=2709-16; " /><meta name="citation_reference" content="citation_title=Permuted tRNA genes in the nuclear and nucleomorph genomes of photosynthetic eukaryotes; Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=1070-6; " /><meta name="citation_reference" content="citation_title=Discovery of permuted and recently split transfer RNAs in Archaea; Genome Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=R38" /><meta name="citation_reference" content="citation_title=Sequence evidence in the archaeal genomes that tRNAs emerged through the combination of ancestral genes as 5′ and 3′ tRNA halves; PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e1622" /><meta name="citation_reference" content="citation_title=An enhanced RNA alignment benchmark for sequence alignment programs; Algorithms Mol. Biol.;  citation_year=2006;  citation_volume=1;  citation_pages=19" /><meta name="citation_reference" content="citation_title=tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features; RNA;  citation_year=2002;  citation_volume=8;  citation_pages=1189-232; " /><meta name="citation_reference" content="citation_title=Conservation of a tRNA core for aminoacylation; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4743-50; " /><meta name="citation_reference" content="citation_title=Molecular recognition of the identity-determinant set of isoleucine transfer RNA from Escherichia coli; J. Mol. Biol.;  citation_year=1994;  citation_volume=236;  citation_pages=710-24; " /><meta name="citation_reference" content="citation_title=A 2-thiouridine derivative in tRNAGlu is a positive determinant for aminoacylation by Escherichia coli glutamyl-tRNA synthetase; Biochemistry;  citation_year=1993;  citation_volume=32;  citation_pages=3836-41; " /><meta name="citation_reference" content="citation_title=Major identity determinants in the ‘augmented D helix’ of tRNA(Glu) from Escherichia coli; J. Mol. Biol.;  citation_year=1996;  citation_volume=256;  citation_pages=685-700; " /><meta name="citation_reference" content="citation_title=The identity determinants required for the discrimination between tRNAGlu and tRNAAsp by glutamyl-tRNA synthetase from Escherichia coli; Eur J Biochem.;  citation_year=1999;  citation_volume=261;  citation_pages=354-60; " /><meta name="citation_reference" content="citation_title=An unusual RNA tertiary interaction has a role for the specific aminoacylation of a transfer RNA; Proc. Natl Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=6776-80; " /><meta name="citation_reference" content="citation_title=Aminoacylation of tRNA in the evolution of an aminoacyl-tRNA synthetase; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=13495-500; " /><meta name="citation_reference" content="citation_title=Escherichia coli tRNA(Asp) recognition mechanism differing from that of the yeast system; Biochem. Biophys. Res. Commun.;  citation_year=1992;  citation_volume=189;  citation_pages=856-62; " /><meta name="citation_reference" content="citation_title=Aspartate identity of transfer RNAs; Biochimie;  citation_year=1996;  citation_volume=78;  citation_pages=605-23; " /><meta name="citation_reference" content="citation_title=Functional idiosyncrasies of tRNA isoacceptors in cognate and noncognate aminoacylation systems; Biochimie;  citation_year=2004;  citation_volume=86;  citation_pages=21-9; " /><meta name="citation_reference" content="citation_title=Discrimination among E. coli tRNAs with a long variable arm; Nucleic Acids Symp. Ser.;  citation_year=1993;  citation_pages=207-8; " /><meta name="citation_reference" content="citation_title=Critical role of the acceptor stem of tRNAs(Met) in their aminoacylation by Escherichia coli methionyl-tRNA synthetase; J. Mol. Biol.;  citation_year=1993;  citation_volume=229;  citation_pages=26-36; " /><meta name="citation_reference" content="citation_title=Discrimination among tRNAs intermediate in glutamate and glutamine acceptor identity; Biochemistry;  citation_year=1993;  citation_volume=32;  citation_pages=14210-9; " /><meta name="citation_reference" content="citation_title=Interactions between tRNA identity nucleotides and their recognition sites in glutaminyl-tRNA synthetase determine the cognate amino acid affinity of the enzyme; Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=6953-8; " /><meta name="citation_reference" content="citation_title=Glutaminyl-tRNA synthetase; Biol. Chem.;  citation_year=1997;  citation_volume=378;  citation_pages=1103-17; " /><meta name="citation_reference" content="citation_title=Eight base changes are sufficient to convert a leucine-inserting tRNA into a serine-inserting tRNA; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=5680-4; " /><meta name="citation_reference" content="citation_title=Nucleotides that contribute to the identity of Escherichia coli tRNA(Phe); J Mol Biol.;  citation_year=1988;  citation_volume=202;  citation_pages=697-709; " /><meta name="citation_reference" content="citation_title=Nucleotides in yeast tRNAPhe required for the specific recognition by its cognate synthetase; Science;  citation_year=1989;  citation_volume=243;  citation_pages=1363-6; " /><meta name="citation_reference" content="citation_title=Recognition nucleotides for human phenylalanyl-tRNA synthetase; Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=475-8; " /><meta name="citation_reference" content="citation_title=Recognition of tRNA(Gly) by three widely diverged glycyl-tRNA synthetases; J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=640-7; " /><meta name="citation_reference" content="citation_title=Reconstitution of alanine acceptor activity from fragments of yeast tRNA-Ala II; Nature;  citation_year=1969;  citation_volume=222;  citation_pages=1147-8; " /><meta name="citation_reference" content="citation_title=The anticodon loop is a major identity determinant of Saccharomyces cerevisiae tRNA(Leu); J. Mol. Biol.;  citation_year=1996;  citation_volume=263;  citation_pages=707-14; " /><meta name="citation_reference" content="citation_title=Identity elements for specific aminoacylation of yeast tRNA(Asp) by cognate aspartyl-tRNA synthetase; Science;  citation_year=1991;  citation_volume=252;  citation_pages=1696-9; " /><meta name="citation_reference" content="citation_title=Identity switches between tRNAs aminoacylated by class I glutaminyl- and class II aspartyl-tRNA synthetases; Biochemistry;  citation_year=1994;  citation_volume=33;  citation_pages=9912-21; " /><meta name="citation_reference" content="citation_title=Identity elements of archaeal tRNA; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=235-46; " /><meta name="citation_reference" content="citation_title=Identity determinants of E. coli threonine tRNA; Biochem. Biophys. Res. Commun.;  citation_year=1992;  citation_volume=184;  citation_pages=478-84; " /><meta name="citation_reference" content="citation_title=Identity elements of tRNA(Thr) towards Saccharomyces cerevisiae threonyl-tRNA synthetase; Nucleic Acids Res.;  citation_year=1995;  citation_volume=23;  citation_pages=2831-6; " /><meta name="citation_reference" content="citation_title=Identity elements of Thermus thermophilus tRNA(Thr); FEBS Lett.;  citation_year=1996;  citation_volume=396;  citation_pages=201-7; " /><meta name="citation_reference" content="citation_title=Molecular recognition of threonine tRNA by threonyl-tRNA synthetase from an extreme thermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Res.;  citation_year=2002;  citation_issue=Suppl;  citation_pages=81-2; " /><meta name="citation_reference" content="citation_title=Determination of phenylalanine tRNA recognition sites by phenylalanyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Symp. Ser. (Oxf);  citation_year=2007;  citation_pages=367-8; " /><meta name="citation_reference" content="citation_title=Differential modes of transfer RNASer recognition in Methanosarcina barkeri; J. Biol. Chem.;  citation_year=2004;  citation_volume=279;  citation_pages=48780-6; " /><meta name="citation_reference" content="citation_title=Differences in tyrosine tRNA identity between Escherichia coli and archaeon, Aeropyrum pernix K1; Nucleic Acids Res.;  citation_year=2002;  citation_issue=Suppl;  citation_pages=225-6; " /><meta name="citation_reference" content="citation_title=Overexpression, purification and crystallization of tyrosyl-tRNA synthetase from the hyperthermophilic archaeon Aeropyrum pernix K1; Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.;  citation_year=2005;  citation_volume=61;  citation_pages=1003-5; " /><meta name="citation_reference" content="citation_title=Conversion of a methionine initiator tRNA into a tryptophan-inserting elongator tRNA in vivo; Biochemistry;  citation_year=1992;  citation_volume=31;  citation_pages=3303-9; " /><meta name="citation_reference" content="citation_title=Analysis of acceptor stem base pairing on tRNA(Trp) aminoacylation and function in vivo; J. Biol. Chem.;  citation_year=1994;  citation_volume=269;  citation_pages=2277-82; " /><meta name="citation_reference" content="citation_title=Switching tRNA(Gln) identity from glutamine to tryptophan; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=3463-7; " /><meta name="citation_reference" content="citation_title=Mutational analysis of conserved positions potentially important for initiator tRNA function in Saccharomyces cerevisiae; Mol. Cell. Biol.;  citation_year=1992;  citation_volume=12;  citation_pages=1432-42; " /><meta name="citation_reference" content="citation_title=The role of nucleotides conserved in eukaryotic initiator methionine tRNAs in initiation of protein synthesis; J. Biol. Chem.;  citation_year=1993;  citation_volume=268;  citation_pages=25221-8; " /><meta name="citation_reference" content="citation_title=Involvement of the size and sequence of the anticodon loop in tRNA recognition by mammalian and E. coli methionyl-tRNA synthetases; Nucleic Acids Res.;  citation_year=1992;  citation_volume=20;  citation_pages=4741-6; " /><meta name="citation_reference" content="citation_title=The modified wobble base inosine in yeast tRNAIle is a positive determinant for aminoacylation by isoleucyl-tRNA synthetase; Biochemistry;  citation_year=1997;  citation_volume=36;  citation_pages=8269-75; " /><meta name="citation_reference" content="citation_title=The structure of an AspRS-tRNA(Asp) complex reveals a tRNA-dependent control mechanism; EMBO J.;  citation_year=2001;  citation_volume=20;  citation_pages=5290-301; " /><meta name="citation_reference" content="citation_title=tRNA aminoacylation by arginyl-tRNA synthetase: induced conformations during substrates binding; EMBO J.;  citation_year=2000;  citation_volume=19;  citation_pages=5599-610; " /><meta name="citation_reference" content="citation_title=Class II aminoacyl transfer RNA synthetases: crystal structure of yeast aspartyl-tRNA synthetase complexed with tRNA(Asp); Science;  citation_year=1991;  citation_volume=252;  citation_pages=1682-9; " /><meta name="citation_reference" content="citation_title=Escherichia coli proline tRNA synthetase is sensitive to changes in the core region of tRNA(Pro); Biochemistry;  citation_year=1994;  citation_volume=33;  citation_pages=12708-14; " /><meta name="citation_reference" content="citation_title=Yeast tRNA(Asp) charging accuracy is threatened by the N-terminal extension of aspartyl-tRNA synthetase; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=9683-90; " /><meta name="citation_reference" content="citation_title=Unique protein architecture of alanyl-tRNA synthetase for aminoacylation, editing, and dimerization; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=8489-94; " /><meta name="citation_reference" content="citation_title=Molecular basis of alanine discrimination in editing site; Proc. Natl Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=11669-74; " /><meta name="citation_reference" content="citation_title=Evolutionary divergence of the archaeal aspartyl-tRNA synthetases into discriminating and nondiscriminating forms; J. Biol. Chem.;  citation_year=2002;  citation_volume=277;  citation_pages=37184-90; " /><meta name="citation_reference" content="citation_title=Crystal structure of aspartyl-tRNA synthetase from Pyrococcus kodakaraensis KOD: archaeon specificity and catalytic mechanism of adenylate formation; EMBO J.;  citation_year=1998;  citation_volume=17;  citation_pages=5227-37; " /><meta name="citation_reference" content="citation_title=Expanding tRNA recognition of a tRNA synthetase by a single amino acid change; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=5676-81; " /><meta name="citation_reference" content="citation_title=Recognition sites of glycine tRNA for glycyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Symp. Ser. (Oxf);  citation_year=2005;  citation_pages=299-300; " /><meta name="citation_reference" content="citation_title=Molecular recognition of histidine tRNA by histidyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Symp. Ser. (Oxf);  citation_year=2005;  citation_pages=307-8; " /><meta name="citation_reference" content="citation_title=Molecular recognition of proline tRNA by prolyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Res.;  citation_year=2003;  citation_issue=Suppl;  citation_pages=247-8; " /><meta name="citation_reference" content="citation_title=Shuffling of discrete tRNASer regions reveals differently utilized identity elements in yeast and methanogenic archaea; J. Mol. Biol.;  citation_year=2006;  citation_volume=361;  citation_pages=128-39; " /><meta name="citation_reference" content="citation_title=Recognition of tRNA by aminoacyl-tRNA synthetase from hyperthermophilic archaea, Aeropyrum pernix K1; Nucleic Acids Res.;  citation_year=2001;  citation_issue=Suppl;  citation_pages=117-8; " /><meta name="citation_reference" content="citation_title=Threonyl-tRNA synthetase of archaea: importance of the discriminator base in the aminoacylation of threonine tRNA; Nucleic Acids Symp. Ser.;  citation_year=2000;  citation_pages=83-4; " /><meta name="citation_reference" content="citation_title=Determination of tryptophan tRNA recognition sites for tryptophanyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1; Nucleic Acids Symp. Ser. (Oxf);  citation_year=2004;  citation_pages=185-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/3/245/1284414/dss008.pdf" /><meta name="description" content="Abstract.  The recently published discrete mathematical method, extended consensus partition (ECP), identifies nucleotide types at each position that are strict" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/3/245/475122" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/3/245/475122" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/3/245/475122");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss008';
    </script>
<title>
	Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./475122" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BemjATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3plbmVzLCDDgXJvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvY2hlbWlzdHJ5LCBFw7Z0dsO2cyBVbml2ZXJzaXR5LCBQw6F6bcOhbnkgUC4gc3RueS4gMS9DLiwgQnVkYXBlc3QgSC0xMTE3LCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBQcm90ZW9taWNzLCBFw7Z0dsO2cyBVbml2ZXJzaXR5LCBCdWRhcGVzdCBILTExMTcsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQw6FsLCBHw6Fib3IiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSwgRcO2dHbDtnMgVW5pdmVyc2l0eSwgUMOhem3DoW55IFAuIHN0bnkuIDEvQy4sIEJ1ZGFwZXN0IEgtMTExNywgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTWFwcGluZyBIaWRkZW4gUG90ZW50aWFsIElkZW50aXR5IEVsZW1lbnRzIGJ5IENvbXB1dGluZyB0aGUgQXZlcmFnZSBEaXNjcmltaW5hdGluZyBQb3dlciBvZiBJbmRpdmlkdWFsIHRSTkEgUG9zaXRpb25zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29nbml0aW9uIG9mIHRSTkFzIGJ5IGFtaW5vYWN5bC10Uk5BIHN5bnRoZXRhc2VzOyBGQVNFQiBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NzktODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJ1bGVzIHRoYXQgZ292ZXJuIHRSTkEgaWRlbnRpdHkgaW4gcHJvdGVpbiBzeW50aGVzaXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjM0OyAgY2l0YXRpb25fcGFnZXM9MjU3LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vbml2ZXJzYWwgcnVsZXMgYW5kIGlkaW9zeW5jcmF0aWMgZmVhdHVyZXMgaW4gdFJOQSBpZGVudGl0eTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTAxNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm9hY3lsLXRSTkEgc3ludGhlc2lzOyBBbm51LiBSZXYuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9NjE3LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3dhcmQgYSBtb3JlIGNvbXBsZXRlIHZpZXcgb2YgdFJOQSBiaW9sb2d5OyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BUkFHT1JOLCBhIHByb2dyYW0gdG8gZGV0ZWN0IHRSTkEgZ2VuZXMgYW5kIHRtUk5BIGdlbmVzIGluIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HdFJOQWRiOiBhIGRhdGFiYXNlIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBkZXRlY3RlZCBpbiBnZW5vbWljIHNlcXVlbmNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EOTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQWRiIDIwMDk6IGNvbXBpbGF0aW9uIG9mIHRSTkEgc2VxdWVuY2VzIGFuZCB0Uk5BIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EMTU5LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5BREItQ0UgMjAxMTogdFJOQSBnZW5lIGRhdGFiYXNlIGN1cmF0ZWQgbWFudWFsbHkgYnkgZXhwZXJ0czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9RDIxMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0YXRpc3RpY2FsIG1ldGhvZCBmb3IgY29ycmVsYXRpbmcgdFJOQSBzZXF1ZW5jZSB3aXRoIGFtaW5vIGFjaWQgc3BlY2lmaWNpdHk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTM3NS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhcmNoaW5nIHRSTkEgc2VxdWVuY2VzIGZvciByZWxhdGVkbmVzcyB0byBhbWlub2FjeWwtdFJOQSBzeW50aGV0YXNlIGZhbWlsaWVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NDgyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBzZXF1ZW5jZSBhbmFseXNpcyB1c2luZyBjb3ZhcmlhbmNlIG1vZGVsczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjA3OS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VEZBTSBkZXRlY3RzIGNvLWV2b2x1dGlvbiBvZiB0Uk5BIGlkZW50aXR5IHJ1bGVzIHdpdGggbGF0ZXJhbCB0cmFuc2ZlciBvZiBoaXN0aWR5bC10Uk5BIHN5bnRoZXRhc2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTg5My05MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpc3VhbGl6aW5nIGJhY3RlcmlhbCB0Uk5BIGlkZW50aXR5IGRldGVybWluYW50cyBhbmQgYW50aWRldGVybWluYW50cyB1c2luZyBmdW5jdGlvbiBsb2dvcyBhbmQgaW52ZXJzZSBmdW5jdGlvbiBsb2dvczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9OTA1LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgY29tcHV0YXRpb25hbCBtZXRob2RzIHJldmVhbCB0Uk5BIGlkZW50aXR5IGVsZW1lbnQgZGl2ZXJnZW5jZSBiZXR3ZWVuIFByb3Rlb2JhY3RlcmlhIGFuZCBDeWFub2JhY3RlcmlhOyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0xMjc2LTg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuYWx5c2lzIG9mIHRSTkEgaWRlbnRpdHk7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTg0OyAgY2l0YXRpb25fcGFnZXM9MzI1LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiBzaWxpY28gZGV0ZWN0aW9uIG9mIHRSTkEgc2VxdWVuY2UgZmVhdHVyZXMgY2hhcmFjdGVyaXN0aWMgdG8gYW1pbm9hY3lsLXRSTkEgc3ludGhldGFzZSBjbGFzcyBtZW1iZXJzaGlwOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz01NTkzLTYwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpLXNwbGl0IHRSTkEgaXMgYSB0cmFuc2ZlciBSTkEgbWFkZSBmcm9tIDMgdHJhbnNjcmlwdHMgdGhhdCBwcm92aWRlcyBpbnNpZ2h0IGludG8gdGhlIGV2b2x1dGlvbiBvZiBmcmFnbWVudGVkIHRSTkFzIGluIGFyY2hhZWE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MjY4My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIGFyY2hhZWFsIHRSTkEgZ2VuZXMgcmV2ZWFscyByYXBpZCBpbmNyZWFzZSBvZiB0Uk5BIGludHJvbnMgaW4gdGhlIG9yZGVyIHRoZXJtb3Byb3RlYWxlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI3MDktMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcm11dGVkIHRSTkEgZ2VuZXMgaW4gdGhlIG51Y2xlYXIgYW5kIG51Y2xlb21vcnBoIGdlbm9tZXMgb2YgcGhvdG9zeW50aGV0aWMgZXVrYXJ5b3RlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTEwNzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIHBlcm11dGVkIGFuZCByZWNlbnRseSBzcGxpdCB0cmFuc2ZlciBSTkFzIGluIEFyY2hhZWE7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPVIzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGV2aWRlbmNlIGluIHRoZSBhcmNoYWVhbCBnZW5vbWVzIHRoYXQgdFJOQXMgZW1lcmdlZCB0aHJvdWdoIHRoZSBjb21iaW5hdGlvbiBvZiBhbmNlc3RyYWwgZ2VuZXMgYXMgNeKAsiBhbmQgM+KAsiB0Uk5BIGhhbHZlczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUxNjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZW5oYW5jZWQgUk5BIGFsaWdubWVudCBiZW5jaG1hcmsgZm9yIHNlcXVlbmNlIGFsaWdubWVudCBwcm9ncmFtczsgQWxnb3JpdGhtcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTm9taWNzOiBhbmFseXNpcyBvZiB0Uk5BIGdlbmVzIGZyb20gNTAgZ2Vub21lcyBvZiBFdWthcnlhLCBBcmNoYWVhLCBhbmQgQmFjdGVyaWEgcmV2ZWFscyBhbnRpY29kb24tc3BhcmluZyBzdHJhdGVnaWVzIGFuZCBkb21haW4tc3BlY2lmaWMgZmVhdHVyZXM7IFJOQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTE4OS0yMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZhdGlvbiBvZiBhIHRSTkEgY29yZSBmb3IgYW1pbm9hY3lsYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTQ3NDMtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciByZWNvZ25pdGlvbiBvZiB0aGUgaWRlbnRpdHktZGV0ZXJtaW5hbnQgc2V0IG9mIGlzb2xldWNpbmUgdHJhbnNmZXIgUk5BIGZyb20gRXNjaGVyaWNoaWEgY29saTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMzY7ICBjaXRhdGlvbl9wYWdlcz03MTAtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgMi10aGlvdXJpZGluZSBkZXJpdmF0aXZlIGluIHRSTkFHbHUgaXMgYSBwb3NpdGl2ZSBkZXRlcm1pbmFudCBmb3IgYW1pbm9hY3lsYXRpb24gYnkgRXNjaGVyaWNoaWEgY29saSBnbHV0YW15bC10Uk5BIHN5bnRoZXRhc2U7IEJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTM4MzYtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ham9yIGlkZW50aXR5IGRldGVybWluYW50cyBpbiB0aGUg4oCYYXVnbWVudGVkIEQgaGVsaXjigJkgb2YgdFJOQShHbHUpIGZyb20gRXNjaGVyaWNoaWEgY29saTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTY7ICBjaXRhdGlvbl9wYWdlcz02ODUtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaWRlbnRpdHkgZGV0ZXJtaW5hbnRzIHJlcXVpcmVkIGZvciB0aGUgZGlzY3JpbWluYXRpb24gYmV0d2VlbiB0Uk5BR2x1IGFuZCB0Uk5BQXNwIGJ5IGdsdXRhbXlsLXRSTkEgc3ludGhldGFzZSBmcm9tIEVzY2hlcmljaGlhIGNvbGk7IEV1ciBKIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI2MTsgIGNpdGF0aW9uX3BhZ2VzPTM1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gdW51c3VhbCBSTkEgdGVydGlhcnkgaW50ZXJhY3Rpb24gaGFzIGEgcm9sZSBmb3IgdGhlIHNwZWNpZmljIGFtaW5vYWN5bGF0aW9uIG9mIGEgdHJhbnNmZXIgUk5BOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9Njc3Ni04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1pbm9hY3lsYXRpb24gb2YgdFJOQSBpbiB0aGUgZXZvbHV0aW9uIG9mIGFuIGFtaW5vYWN5bC10Uk5BIHN5bnRoZXRhc2U7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9OTU7ICBjaXRhdGlvbl9wYWdlcz0xMzQ5NS01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzY2hlcmljaGlhIGNvbGkgdFJOQShBc3ApIHJlY29nbml0aW9uIG1lY2hhbmlzbSBkaWZmZXJpbmcgZnJvbSB0aGF0IG9mIHRoZSB5ZWFzdCBzeXN0ZW07IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xODk7ICBjaXRhdGlvbl9wYWdlcz04NTYtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzcGFydGF0ZSBpZGVudGl0eSBvZiB0cmFuc2ZlciBSTkFzOyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz02MDUtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgaWRpb3N5bmNyYXNpZXMgb2YgdFJOQSBpc29hY2NlcHRvcnMgaW4gY29nbmF0ZSBhbmQgbm9uY29nbmF0ZSBhbWlub2FjeWxhdGlvbiBzeXN0ZW1zOyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz0yMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjcmltaW5hdGlvbiBhbW9uZyBFLiBjb2xpIHRSTkFzIHdpdGggYSBsb25nIHZhcmlhYmxlIGFybTsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fcGFnZXM9MjA3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyaXRpY2FsIHJvbGUgb2YgdGhlIGFjY2VwdG9yIHN0ZW0gb2YgdFJOQXMoTWV0KSBpbiB0aGVpciBhbWlub2FjeWxhdGlvbiBieSBFc2NoZXJpY2hpYSBjb2xpIG1ldGhpb255bC10Uk5BIHN5bnRoZXRhc2U7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjI5OyAgY2l0YXRpb25fcGFnZXM9MjYtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NyaW1pbmF0aW9uIGFtb25nIHRSTkFzIGludGVybWVkaWF0ZSBpbiBnbHV0YW1hdGUgYW5kIGdsdXRhbWluZSBhY2NlcHRvciBpZGVudGl0eTsgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTQyMTAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJhY3Rpb25zIGJldHdlZW4gdFJOQSBpZGVudGl0eSBudWNsZW90aWRlcyBhbmQgdGhlaXIgcmVjb2duaXRpb24gc2l0ZXMgaW4gZ2x1dGFtaW55bC10Uk5BIHN5bnRoZXRhc2UgZGV0ZXJtaW5lIHRoZSBjb2duYXRlIGFtaW5vIGFjaWQgYWZmaW5pdHkgb2YgdGhlIGVuenltZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTY5NTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2x1dGFtaW55bC10Uk5BIHN5bnRoZXRhc2U7IEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM3ODsgIGNpdGF0aW9uX3BhZ2VzPTExMDMtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVpZ2h0IGJhc2UgY2hhbmdlcyBhcmUgc3VmZmljaWVudCB0byBjb252ZXJ0IGEgbGV1Y2luZS1pbnNlcnRpbmcgdFJOQSBpbnRvIGEgc2VyaW5lLWluc2VydGluZyB0Uk5BOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9NTY4MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlcyB0aGF0IGNvbnRyaWJ1dGUgdG8gdGhlIGlkZW50aXR5IG9mIEVzY2hlcmljaGlhIGNvbGkgdFJOQShQaGUpOyBKIE1vbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMDI7ICBjaXRhdGlvbl9wYWdlcz02OTctNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlcyBpbiB5ZWFzdCB0Uk5BUGhlIHJlcXVpcmVkIGZvciB0aGUgc3BlY2lmaWMgcmVjb2duaXRpb24gYnkgaXRzIGNvZ25hdGUgc3ludGhldGFzZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDM7ICBjaXRhdGlvbl9wYWdlcz0xMzYzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29nbml0aW9uIG51Y2xlb3RpZGVzIGZvciBodW1hbiBwaGVueWxhbGFueWwtdFJOQSBzeW50aGV0YXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz00NzUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb2duaXRpb24gb2YgdFJOQShHbHkpIGJ5IHRocmVlIHdpZGVseSBkaXZlcmdlZCBnbHljeWwtdFJOQSBzeW50aGV0YXNlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjg7ICBjaXRhdGlvbl9wYWdlcz02NDAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdGl0dXRpb24gb2YgYWxhbmluZSBhY2NlcHRvciBhY3Rpdml0eSBmcm9tIGZyYWdtZW50cyBvZiB5ZWFzdCB0Uk5BLUFsYSBJSTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTY5OyAgY2l0YXRpb25fdm9sdW1lPTIyMjsgIGNpdGF0aW9uX3BhZ2VzPTExNDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFudGljb2RvbiBsb29wIGlzIGEgbWFqb3IgaWRlbnRpdHkgZGV0ZXJtaW5hbnQgb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIHRSTkEoTGV1KTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjM7ICBjaXRhdGlvbl9wYWdlcz03MDctMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aXR5IGVsZW1lbnRzIGZvciBzcGVjaWZpYyBhbWlub2FjeWxhdGlvbiBvZiB5ZWFzdCB0Uk5BKEFzcCkgYnkgY29nbmF0ZSBhc3BhcnR5bC10Uk5BIHN5bnRoZXRhc2U7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjUyOyAgY2l0YXRpb25fcGFnZXM9MTY5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGl0eSBzd2l0Y2hlcyBiZXR3ZWVuIHRSTkFzIGFtaW5vYWN5bGF0ZWQgYnkgY2xhc3MgSSBnbHV0YW1pbnlsLSBhbmQgY2xhc3MgSUkgYXNwYXJ0eWwtdFJOQSBzeW50aGV0YXNlczsgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9OTkxMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpdHkgZWxlbWVudHMgb2YgYXJjaGFlYWwgdFJOQTsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMzUtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aXR5IGRldGVybWluYW50cyBvZiBFLiBjb2xpIHRocmVvbmluZSB0Uk5BOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9NDc4LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGl0eSBlbGVtZW50cyBvZiB0Uk5BKFRocikgdG93YXJkcyBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgdGhyZW9ueWwtdFJOQSBzeW50aGV0YXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yODMxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aXR5IGVsZW1lbnRzIG9mIFRoZXJtdXMgdGhlcm1vcGhpbHVzIHRSTkEoVGhyKTsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zOTY7ICBjaXRhdGlvbl9wYWdlcz0yMDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHJlY29nbml0aW9uIG9mIHRocmVvbmluZSB0Uk5BIGJ5IHRocmVvbnlsLXRSTkEgc3ludGhldGFzZSBmcm9tIGFuIGV4dHJlbWUgdGhlcm1vcGhpbGljIGFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbDsgIGNpdGF0aW9uX3BhZ2VzPTgxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYXRpb24gb2YgcGhlbnlsYWxhbmluZSB0Uk5BIHJlY29nbml0aW9uIHNpdGVzIGJ5IHBoZW55bGFsYW55bC10Uk5BIHN5bnRoZXRhc2UgZnJvbSBoeXBlcnRoZXJtb3BoaWxpYyBhcmNoYWVvbiwgQWVyb3B5cnVtIHBlcm5peCBLMTsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuIChPeGYpOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fcGFnZXM9MzY3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBtb2RlcyBvZiB0cmFuc2ZlciBSTkFTZXIgcmVjb2duaXRpb24gaW4gTWV0aGFub3NhcmNpbmEgYmFya2VyaTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9NDg3ODAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW5jZXMgaW4gdHlyb3NpbmUgdFJOQSBpZGVudGl0eSBiZXR3ZWVuIEVzY2hlcmljaGlhIGNvbGkgYW5kIGFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbDsgIGNpdGF0aW9uX3BhZ2VzPTIyNS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdmVyZXhwcmVzc2lvbiwgcHVyaWZpY2F0aW9uIGFuZCBjcnlzdGFsbGl6YXRpb24gb2YgdHlyb3N5bC10Uk5BIHN5bnRoZXRhc2UgZnJvbSB0aGUgaHlwZXJ0aGVybW9waGlsaWMgYXJjaGFlb24gQWVyb3B5cnVtIHBlcm5peCBLMTsgQWN0YSBDcnlzdGFsbG9nci4gU2VjdC4gRiBTdHJ1Y3QuIEJpb2wuIENyeXN0LiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MTAwMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db252ZXJzaW9uIG9mIGEgbWV0aGlvbmluZSBpbml0aWF0b3IgdFJOQSBpbnRvIGEgdHJ5cHRvcGhhbi1pbnNlcnRpbmcgZWxvbmdhdG9yIHRSTkEgaW4gdml2bzsgQmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzMwMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBhY2NlcHRvciBzdGVtIGJhc2UgcGFpcmluZyBvbiB0Uk5BKFRycCkgYW1pbm9hY3lsYXRpb24gYW5kIGZ1bmN0aW9uIGluIHZpdm87IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI2OTsgIGNpdGF0aW9uX3BhZ2VzPTIyNzctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN3aXRjaGluZyB0Uk5BKEdsbikgaWRlbnRpdHkgZnJvbSBnbHV0YW1pbmUgdG8gdHJ5cHRvcGhhbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTM0NjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBhbmFseXNpcyBvZiBjb25zZXJ2ZWQgcG9zaXRpb25zIHBvdGVudGlhbGx5IGltcG9ydGFudCBmb3IgaW5pdGlhdG9yIHRSTkEgZnVuY3Rpb24gaW4gU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlOyBNb2wuIENlbGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTQzMi00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgbnVjbGVvdGlkZXMgY29uc2VydmVkIGluIGV1a2FyeW90aWMgaW5pdGlhdG9yIG1ldGhpb25pbmUgdFJOQXMgaW4gaW5pdGlhdGlvbiBvZiBwcm90ZWluIHN5bnRoZXNpczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjY4OyAgY2l0YXRpb25fcGFnZXM9MjUyMjEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52b2x2ZW1lbnQgb2YgdGhlIHNpemUgYW5kIHNlcXVlbmNlIG9mIHRoZSBhbnRpY29kb24gbG9vcCBpbiB0Uk5BIHJlY29nbml0aW9uIGJ5IG1hbW1hbGlhbiBhbmQgRS4gY29saSBtZXRoaW9ueWwtdFJOQSBzeW50aGV0YXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDc0MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbW9kaWZpZWQgd29iYmxlIGJhc2UgaW5vc2luZSBpbiB5ZWFzdCB0Uk5BSWxlIGlzIGEgcG9zaXRpdmUgZGV0ZXJtaW5hbnQgZm9yIGFtaW5vYWN5bGF0aW9uIGJ5IGlzb2xldWN5bC10Uk5BIHN5bnRoZXRhc2U7IEJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTgyNjktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHJ1Y3R1cmUgb2YgYW4gQXNwUlMtdFJOQShBc3ApIGNvbXBsZXggcmV2ZWFscyBhIHRSTkEtZGVwZW5kZW50IGNvbnRyb2wgbWVjaGFuaXNtOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NTI5MC0zMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkEgYW1pbm9hY3lsYXRpb24gYnkgYXJnaW55bC10Uk5BIHN5bnRoZXRhc2U6IGluZHVjZWQgY29uZm9ybWF0aW9ucyBkdXJpbmcgc3Vic3RyYXRlcyBiaW5kaW5nOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NTU5OS02MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsYXNzIElJIGFtaW5vYWN5bCB0cmFuc2ZlciBSTkEgc3ludGhldGFzZXM6IGNyeXN0YWwgc3RydWN0dXJlIG9mIHllYXN0IGFzcGFydHlsLXRSTkEgc3ludGhldGFzZSBjb21wbGV4ZWQgd2l0aCB0Uk5BKEFzcCk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjUyOyAgY2l0YXRpb25fcGFnZXM9MTY4Mi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIHByb2xpbmUgdFJOQSBzeW50aGV0YXNlIGlzIHNlbnNpdGl2ZSB0byBjaGFuZ2VzIGluIHRoZSBjb3JlIHJlZ2lvbiBvZiB0Uk5BKFBybyk7IEJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTEyNzA4LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ZZWFzdCB0Uk5BKEFzcCkgY2hhcmdpbmcgYWNjdXJhY3kgaXMgdGhyZWF0ZW5lZCBieSB0aGUgTi10ZXJtaW5hbCBleHRlbnNpb24gb2YgYXNwYXJ0eWwtdFJOQSBzeW50aGV0YXNlOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz05NjgzLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlxdWUgcHJvdGVpbiBhcmNoaXRlY3R1cmUgb2YgYWxhbnlsLXRSTkEgc3ludGhldGFzZSBmb3IgYW1pbm9hY3lsYXRpb24sIGVkaXRpbmcsIGFuZCBkaW1lcml6YXRpb247IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9ODQ4OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGJhc2lzIG9mIGFsYW5pbmUgZGlzY3JpbWluYXRpb24gaW4gZWRpdGluZyBzaXRlOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMjsgIGNpdGF0aW9uX3BhZ2VzPTExNjY5LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZGl2ZXJnZW5jZSBvZiB0aGUgYXJjaGFlYWwgYXNwYXJ0eWwtdFJOQSBzeW50aGV0YXNlcyBpbnRvIGRpc2NyaW1pbmF0aW5nIGFuZCBub25kaXNjcmltaW5hdGluZyBmb3JtczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MzcxODQtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyeXN0YWwgc3RydWN0dXJlIG9mIGFzcGFydHlsLXRSTkEgc3ludGhldGFzZSBmcm9tIFB5cm9jb2NjdXMga29kYWthcmFlbnNpcyBLT0Q6IGFyY2hhZW9uIHNwZWNpZmljaXR5IGFuZCBjYXRhbHl0aWMgbWVjaGFuaXNtIG9mIGFkZW55bGF0ZSBmb3JtYXRpb247IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz01MjI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBhbmRpbmcgdFJOQSByZWNvZ25pdGlvbiBvZiBhIHRSTkEgc3ludGhldGFzZSBieSBhIHNpbmdsZSBhbWlubyBhY2lkIGNoYW5nZTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz01Njc2LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvZ25pdGlvbiBzaXRlcyBvZiBnbHljaW5lIHRSTkEgZm9yIGdseWN5bC10Uk5BIHN5bnRoZXRhc2UgZnJvbSBoeXBlcnRoZXJtb3BoaWxpYyBhcmNoYWVvbiwgQWVyb3B5cnVtIHBlcm5peCBLMTsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuIChPeGYpOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHJlY29nbml0aW9uIG9mIGhpc3RpZGluZSB0Uk5BIGJ5IGhpc3RpZHlsLXRSTkEgc3ludGhldGFzZSBmcm9tIGh5cGVydGhlcm1vcGhpbGljIGFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxOyBOdWNsZWljIEFjaWRzIFN5bXAuIFNlci4gKE94Zik7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl9wYWdlcz0zMDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHJlY29nbml0aW9uIG9mIHByb2xpbmUgdFJOQSBieSBwcm9seWwtdFJOQSBzeW50aGV0YXNlIGZyb20gaHlwZXJ0aGVybW9waGlsaWMgYXJjaGFlb24sIEFlcm9weXJ1bSBwZXJuaXggSzE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsOyAgY2l0YXRpb25fcGFnZXM9MjQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNodWZmbGluZyBvZiBkaXNjcmV0ZSB0Uk5BU2VyIHJlZ2lvbnMgcmV2ZWFscyBkaWZmZXJlbnRseSB1dGlsaXplZCBpZGVudGl0eSBlbGVtZW50cyBpbiB5ZWFzdCBhbmQgbWV0aGFub2dlbmljIGFyY2hhZWE7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzYxOyAgY2l0YXRpb25fcGFnZXM9MTI4LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvZ25pdGlvbiBvZiB0Uk5BIGJ5IGFtaW5vYWN5bC10Uk5BIHN5bnRoZXRhc2UgZnJvbSBoeXBlcnRoZXJtb3BoaWxpYyBhcmNoYWVhLCBBZXJvcHlydW0gcGVybml4IEsxOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbDsgIGNpdGF0aW9uX3BhZ2VzPTExNy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaHJlb255bC10Uk5BIHN5bnRoZXRhc2Ugb2YgYXJjaGFlYTogaW1wb3J0YW5jZSBvZiB0aGUgZGlzY3JpbWluYXRvciBiYXNlIGluIHRoZSBhbWlub2FjeWxhdGlvbiBvZiB0aHJlb25pbmUgdFJOQTsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fcGFnZXM9ODMtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZXJtaW5hdGlvbiBvZiB0cnlwdG9waGFuIHRSTkEgcmVjb2duaXRpb24gc2l0ZXMgZm9yIHRyeXB0b3BoYW55bC10Uk5BIHN5bnRoZXRhc2UgZnJvbSBoeXBlcnRoZXJtb3BoaWxpYyBhcmNoYWVvbiwgQWVyb3B5cnVtIHBlcm5peCBLMTsgTnVjbGVpYyBBY2lkcyBTeW1wLiBTZXIuIChPeGYpOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fcGFnZXM9MTg1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS8zLzI0NS8xMjg0NDE0L2RzczAwOC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHJlY2VudGx5IHB1Ymxpc2hlZCBkaXNjcmV0ZSBtYXRoZW1hdGljYWwgbWV0aG9kLCBleHRlbmRlZCBjb25zZW5zdXMgcGFydGl0aW9uIChFQ1ApLCBpZGVudGlmaWVzIG51Y2xlb3RpZGUgdHlwZXMgYXQgZWFjaCBwb3NpdGlvbiB0aGF0IGFyZSBzdHJpY3QiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ3NTEyMiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGST1E5W1ZgrLuaRGzsrdsCccqVuZw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f245%2f475122" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f245%2f475122" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18620" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/3">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7163467">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163467">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163480">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163480">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163510">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163510">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163538">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163538">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163606">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163606">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163608">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163608">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163610">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163610">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7163612">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7163612">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/3/231/474863">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/3/259/475336">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">&#193;ron Szenes</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">Eötvös University</div>, <div class="addr-line">Pázmány P. stny. 1/C.</div>, <div class="addr-line">Budapest H-1117</div>, <div class="country">Hungary</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Proteomics</div>, <div class="insititution">Eötvös University</div>, <div class="addr-line">Budapest H-1117</div>, <div class="country">Hungary</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=%c3%81ron+Szenes">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Szenes &#193;">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Szenes &#193;&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Áron Szenes</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">G&#225;bor P&#225;l</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biochemistry</div>, <div class="insititution">Eötvös University</div>, <div class="addr-line">Pázmány P. stny. 1/C.</div>, <div class="addr-line">Budapest H-1117</div>, <div class="country">Hungary</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +36 1-2722500, ext. 8577. Fax. +36 1-3812172. Email: <a href="mailto:palgabor@elte.hu" target="_blank">palgabor@elte.hu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=G%c3%a1bor+P%c3%a1l">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=P&#225;l G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;P&#225;l G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gábor Pál</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 245–258, <a href='https://doi.org/10.1093/dnares/dss008'>https://doi.org/10.1093/dnares/dss008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 October 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 February 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="475122" href="/dnaresearch/article-pdf/19/3/245/1284414/dss008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Áron Szenes, Gábor Pál;  Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions, <em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 245–258, <a href='https://doi.org/10.1093/dnares/dss008'>https://doi.org/10.1093/dnares/dss008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/475122?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/475122?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mapping%20Hidden%20Potential%20Identity%20Elements%20by%20Computing%20the%20Average%20Discriminating%20Power%20of%20Individual%20tRNA%20Positions&amp;publicationDate=2012-02-28&amp;volumeNum=19&amp;issueNum=3&amp;author=Szenes%2C%20%C3%81ron%3B%20P%C3%A1l%2C%20G%C3%A1bor&amp;startPage=245&amp;endPage=258&amp;contentId=10.1093%2Fdnares%2Fdss008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f245%2f475122" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7163467 id="7163467" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The recently published discrete mathematical method, extended consensus partition (ECP), identifies nucleotide types at each position that are strictly absent from a given sequence set, while occur in other sets. These are defined as discriminating elements (DEs). In this study using the ECP approach, we mapped potential hidden identity elements that discriminate the 20 different tRNA identities. We filtered the tDNA data set for the obligatory presence of well-established tRNA features, and then separately for each identity set, the presence of already experimentally identified strictly present identity elements. The analysis was performed on the three kingdoms of life. We determined the number of DE, e.g. the number of sets discriminated by the given position, for each tRNA position of each tRNA identity set. Then, from the positional DE numbers obtained from the 380 pairwise comparisons of the 20 identity sets, we calculated the average excluding value (AEV) for each tRNA position. The AEV provides a measure on the overall discriminating power of each position. Using a statistical analysis, we show that positional AEVs correlate with the number of already identified identity elements. Positions having high AEV but lacking published identity elements predict hitherto undiscovered tRNA identity elements.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="tRNA">tRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;identity element prediction&quot;">identity element prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;extended consensus partition&quot;">extended consensus partition</a></div>                            <h2 scrollto-destination=7163480 id="7163480" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>In all organisms, the 20 aminoacyl-tRNA synthetase (AARS) enzymes have to recognize their amino acid substrates and the corresponding tRNA molecules with high precision to produce only legitimate aminoacyl-tRNA products. This exquisite specificity is of central importance as this enables the genetic information to be faithfully translated into protein sequences by following the rules defined in the genetic code. Although principles and many fine details of this selective recognition event have already been discovered,<sup><a reveal-id="DSS008C1  DSS008C4" data-open="DSS008C1  DSS008C4" class="link link-ref link-reveal xref-bibr">1–4</a></sup> several questions remained still unanswered.<sup><a reveal-id="DSS008C5" data-open="DSS008C5" class="link link-ref link-reveal xref-bibr">5</a></sup> tRNA positions that have utmost roles in the selective interaction with the cognate AARS and thus define the identity of the tRNA are denoted as identity elements. While only laboratory experiments can decisively define the identity elements, the large number of potential positions and the laborious nature of the experiments prompted a great variety of bioinformatics studies to predict such elements. These studies require large numbers of individual input tRNA sequences to locate statistically significant identity-related sequence properties. This magnitude of input data became available in the form of genomic DNA sequences, from which tRNA-detecting algorithms<sup><a reveal-id="DSS008C6" data-open="DSS008C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS008C7" data-open="DSS008C7" class="link link-ref link-reveal xref-bibr">7</a></sup> can identify functionally relevant tDNA sequences. Such analyses yielded numerous different tDNA databases.<sup><a reveal-id="DSS008C8  DSS008C10" data-open="DSS008C8  DSS008C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> Several computational studies reported successful functional annotation and <em>in silico</em> identity element determination.<sup><a reveal-id="DSS008C11" data-open="DSS008C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS008C12" data-open="DSS008C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Improved secondary structure-predicting algorithm-driven tRNA alignments<sup><a reveal-id="DSS008C13" data-open="DSS008C13" class="link link-ref link-reveal xref-bibr">13</a></sup> yielded high-quality input data sets. These high-quality sets allowed for innovative sequence logo and inverse sequence logo-based analyses of tRNA features and identity element predictions.<sup><a reveal-id="DSS008C14" data-open="DSS008C14" class="link link-ref link-reveal xref-bibr">14</a></sup> An information theory-based approach opened new frontiers in visualizing tRNA sequence features and predicting determinants and anti-determinants.<sup><a reveal-id="DSS008C15" data-open="DSS008C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS008C16" data-open="DSS008C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Computational tRNA identity analysing methods were compared in a recent review of Ardell.<sup><a reveal-id="DSS008C17" data-open="DSS008C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>In this paper, we introduce a new approach based on the recently published ‘extended consensus partition’ (ECP) algorithm.<sup><a reveal-id="DSS008C18" data-open="DSS008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The ECP algorithm provides a discrete mathematical measure of pairwise distances of functionally related aligned sequence sets. It was first introduced to reveal characteristic sequence features that discriminate the two tRNA sets corresponding to Class I and Class II AARS enzymes.</p><p>In this study, we applied the ECP algorithm to assess the potential of each tRNA position to discriminate the 20 different tRNA identity sets from each other. The ECP method heavily relies on characteristic positional absence of nucleotide base types. Because of that, the method is sensitive even to the rare occurrence of atypical sequences. For removing such sequences, we filtered the tDNA data sets for the obligatory presence of well-established tRNA features. Moreover, as all bioinformatic studies, the ECP analysis also requires a large number of input sequences. In this case, it is needed to reliably identify nucleotide types that are strictly absent from a given position of the aligned sequence set, i.e. their absence is not due to stochastic sampling error.</p><p>In order to provide the necessary large input sets, we performed the ECP analyses on the three kingdoms of life instead of individual species. Nevertheless, we aimed to compare tRNA identity sets that contain isofunctional sequences in spite of being originated from different species. Therefore, separately for each identity set, we further filtered the sets for the presence of experimentally verified strictly present identity elements. As a control experiment, we also performed the analysis by omitting this second filtration step.</p><p>We argued that tRNA molecules sharing a large set of experimentally verified identity elements should interact with their corresponding AARS enzyme similarly and therefore should also share yet unidentified common identity elements.</p><p>By combining the ECP method with simple statistics, we generated average excluding values (AEVs) providing a measure on the overall discriminating power of each tRNA position. We show that both with and without the second filtering step, positional AEVs correlate with the number of already identified identity elements. We argue that positions having high AEV but lacking already published identity elements predict hitherto undiscovered tRNA identity elements.</p><p>The analysis located such potential identity elements on the anticodon arm (30:40 and 31:39) and suggests that the core region also contributes to defining tRNA identity.</p>                            <h2 scrollto-destination=7163510 id="7163510" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7163511 id="7163511" class="section-title" data-legacy-id=s2a>2.1. Data set building</h3>
<p>The tDNA sequences of Bacteria and Eukaryotes were downloaded from the tRNAdb database.<sup><a reveal-id="DSS008C9" data-open="DSS008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The Archaea set of this database has not yet contained the recently discovered and characterized<sup><a reveal-id="DSS008C19  DSS008C22" data-open="DSS008C19  DSS008C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> split tRNA, which have been organized in the SPLITSdb database.<sup><a reveal-id="DSS008C20" data-open="DSS008C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Split tRNA data are already included in the tRNADB-CE database,<sup><a reveal-id="DSS008C10" data-open="DSS008C10" class="link link-ref link-reveal xref-bibr">10</a></sup> which however (unlike Sprinzl and tRNAdb) does not contain aligned sequences. Therefore, we downloaded both normal and split Archaea tDNA from the tRNADB-CE database and aligned the sequences by the ClustalW software and manually as described by Fujishima <em>et al</em>.<sup><a reveal-id="DSS008C23" data-open="DSS008C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSS008C24" data-open="DSS008C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In the case of Archaea data set, we omitted the variable loop sequences from the analysis because of alignment complications.</p><p>The downloaded set was filtered for sequences that fulfil several criteria.</p>                            <h3 scrollto-destination=7163514 id="7163514" class="section-title" data-legacy-id=s2b>2.2. First filtering step for all data sets</h3>
<p>In the first ECP-based study,<sup><a reveal-id="DSS008C18" data-open="DSS008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we used the database from the tRNomics study of Marck and Grosjean.<sup><a reveal-id="DSS008C25" data-open="DSS008C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Although for the present study, we used a larger, updated database, in the case of Bacteria and Eukarya, we could still use the well-established kingdom-specific strictly present elements—as filtering rules—defined by that tRNomics study. For Archaea, we used the filtering rules of Fujishima <em>et al</em>.<sup><a reveal-id="DSS008C23" data-open="DSS008C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The obligatory presence of these elements established our first filtering criteria. The element sets for the three kingdoms were as follows. Bacteria: H14, G18, R19, Y33, G53:C61, T54, T55, Y56, D57, A58. Archaea: Y8, A14, G15, G18, G19, R21, T33, Y48, G53, T54, T55, C56, R57, A58. Eukarya: Y8, Y11, A14, -17a, G18, G19, R21, R24, H32, Y33, R37, H38, G53, H54, T55, C56, R57, A58, C61. (Note that nucleotides and their sets are denoted by IUPAC nucleotide codes.) Discarding sequences that lacked any of the strictly present kingdom-specific elements removed incorrectly sequenced or most likely non-functional tDNA data.</p>                            <h3 scrollto-destination=7163516 id="7163516" class="section-title" data-legacy-id=s2c>2.3. Second filtering step for the bacterial and eukaryotic data sets</h3>
<p>We grouped each sequence based on identity and filtered for the presence of already identified and published major, strictly present identity elements characteristic to the given amino acid identity set.<sup><a reveal-id="DSS008C3" data-open="DSS008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> These sets for <em>Escherichia coli</em> and <em>Saccharomyces cerevisiae</em> are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. By excluding sequences that lacked these elements, we aimed to generate tRNA identity sets that contain isofunctional sequences expected to function in <em>E. coli</em> or yeast, respectively. We argued that if hidden identity elements still exist, those could also be shared by these filtered sequences.</p><p>Out of the published identity elements, only the determinants were included, while the anti-determinants were not considered. One determinant, the G15:G48 Levitt pair of tRNA<sup>Cys</sup>, was omitted from the filtering as this is idiosyncratic to <em>E. coli</em> (more accurately, it could have emerged in the common ancestor of <em>E. coli</em> and <em>Haemophilus influenzae</em>).<sup><a reveal-id="DSS008C26" data-open="DSS008C26" class="link link-ref link-reveal xref-bibr">26</a></sup></p>                            <h3 scrollto-destination=7163519 id="7163519" class="section-title" data-legacy-id=s2d>2.4. Third filtering step for all data sets</h3>
<p>Finally, in order not to bias the statistical analysis, we removed any redundancy from the data set by keeping only unique sequences.</p><p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> organized by the three kingdoms contains the species list corresponding to our raw data set and indicates the number of sequences contributed by each species in the raw set and after each filtering step.</p><p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S1</a> shows the resulted sets after the final filtering step. It contains six databases in a multi-fasta format, two for each kingdom. For each kingdom, one database contains a set of non-redundant sequences, while the other set contains all sequences minus the non-redundant set, thus it contains all ‘siblings’.</p>                            <h3 scrollto-destination=7163523 id="7163523" class="section-title" data-legacy-id=s2e>2.5. Determination of discriminating elements by the extended consensus partition algorithm</h3>
<p>Filtered data sets for each kingdom were analysed by the already published extended consensus partition (ECP) algorithm.<sup><a reveal-id="DSS008C18" data-open="DSS008C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Principles of the analysis and the algorithm remained the same. However, in this case, not the two AARS-based tRNA classes were compared, but all pairs of the 20 tRNA identity sets. The logic of the algorithm is illustrated in Fig. <a reveal-id="DSS008F1" data-open="DSS008F1" class="link link-reveal link-table xref-fig">1</a>. Because the pairwise ECP analysis is non-symmetrical, from the filtered data set and separately for each kingdom, we produced all 380 (20 × 19) identity pairs. For each pair, we identified the discriminating elements (DEs) through the ECP algorithm as follows. </p>                        <a id="7163525" scrollto-destination="7163525"></a>
<div id="DSS008F1" data-id="DSS008F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00801.jpeg?Expires=2147483647&Signature=BTZUN0ASb0S4c-WePJHmMvg8juSk2Os-8xwjOcEiaNs3gf49igP-t7WD6ctkIWgfGLGSxo3JIttHp-FvJ4-1BfdmTPK-UINLF9XSVveIuxu2f4KgFfpWCoA6G8UplUqDWxWJpM3cMzm4yK7eZzM0QXukWzm9FljKAbrEOQdgybtUhX8dXNmGpDVOMn3LM~q75NhC2OBPE76OnUFRQtddtYlioNZKFgCWtkqOMMvdsIOZHmp-R4edFgniVkHnmG-XmdhQOzmAXWYxY89EuuxaSnwDuAr8HJNn4cPbsLV~YyM4~Rj4ylMGye5kap-E2ynoweZ-I7hmmK-DqvDRI5Acmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of calculating the AEV using short artificial sequences. (A) Identifying DEs using an artificial set of short sequences belonging to three amino acid identity sets. Identities are labelled as 1aa, 2aa and 3aa and are highlighted as cyan, yellow and magenta, respectively. Each identity set contains four short, tetramer sequences. The calculated DE is either one base or a combination of bases and it is labelled by IUPAC codes of bases or base sets. Each step of the DE-generating algorithm is explained in Section 2. (B) The calculated DE for each identity pairs. Note that the DE-generating relationship is non-symmetrical, i.e. the 1aa vs. 2aa pair has a DE different from that of the 2aa vs. 1aa pair. (C) Positional summation of DE. Positional sum of the DE values (shown in the lowest row) provides the number of pairwise discriminations provided by the given position. The sums of the DE values are input data for calculating the AEVs). AEV is generated by dividing the positional sum of DE with the number of the identities (which in a real case is 20 while in this didactic case 3). Formalism and more detailed description are provided in Section 2. This figure can be viewed in colour online." path-from-xml="dss00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163525/dss00801.jpeg" path-from-xml="dss00801.jpeg" target="_blank">View large</a><a section="7163525" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00801.jpeg?Expires=2147483647&Signature=sMhI-MA6I0GHwnwbiUMhddfQ16eHv13PcShU7zwTmBWhFhsDSyTmGpkqI2XHp3h6M1xloNClhgj4yPXwfb9DLhj72ATzHD4PvxW3wQYWIFepxiR~8S0yDAqkqknbog9rAY-EWJSQ0jUMmBqd9jPWHm9aFZBCDUPtCfEPyNLQZ3VcuQgiJq2yW~irouN3mc9Nh-QtotGUDQT~kjdSnK9DIr6DaGbPB4Rmg908OTdg7N7-ZfAvTmcAJZfQnsr67mSafWAjj9j-Daek0acHMLnjGmOpD7joty7Xzk1nQwM6YEuAvbV~a1bt8ssmfYRBv~F2AAwmE2R-VXdSFiuQ7AeR8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163525&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of calculating the AEV using short artificial sequences. (A) Identifying DEs using an artificial set of short sequences belonging to three amino acid identity sets. Identities are labelled as 1aa, 2aa and 3aa and are highlighted as cyan, yellow and magenta, respectively. Each identity set contains four short, tetramer sequences. The calculated DE is either one base or a combination of bases and it is labelled by IUPAC codes of bases or base sets. Each step of the DE-generating algorithm is explained in Section 2. (B) The calculated DE for each identity pairs. Note that the DE-generating relationship is non-symmetrical, i.e. the 1aa vs. 2aa pair has a DE different from that of the 2aa vs. 1aa pair. (C) Positional summation of DE. Positional sum of the DE values (shown in the lowest row) provides the number of pairwise discriminations provided by the given position. The sums of the DE values are input data for calculating the AEVs). AEV is generated by dividing the positional sum of DE with the number of the identities (which in a real case is 20 while in this didactic case 3). Formalism and more detailed description are provided in Section 2. This figure can be viewed in colour online.</p></div></div><div content-id="DSS008F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00801.jpeg?Expires=2147483647&Signature=BTZUN0ASb0S4c-WePJHmMvg8juSk2Os-8xwjOcEiaNs3gf49igP-t7WD6ctkIWgfGLGSxo3JIttHp-FvJ4-1BfdmTPK-UINLF9XSVveIuxu2f4KgFfpWCoA6G8UplUqDWxWJpM3cMzm4yK7eZzM0QXukWzm9FljKAbrEOQdgybtUhX8dXNmGpDVOMn3LM~q75NhC2OBPE76OnUFRQtddtYlioNZKFgCWtkqOMMvdsIOZHmp-R4edFgniVkHnmG-XmdhQOzmAXWYxY89EuuxaSnwDuAr8HJNn4cPbsLV~YyM4~Rj4ylMGye5kap-E2ynoweZ-I7hmmK-DqvDRI5Acmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Illustration of calculating the AEV using short artificial sequences. (A) Identifying DEs using an artificial set of short sequences belonging to three amino acid identity sets. Identities are labelled as 1aa, 2aa and 3aa and are highlighted as cyan, yellow and magenta, respectively. Each identity set contains four short, tetramer sequences. The calculated DE is either one base or a combination of bases and it is labelled by IUPAC codes of bases or base sets. Each step of the DE-generating algorithm is explained in Section 2. (B) The calculated DE for each identity pairs. Note that the DE-generating relationship is non-symmetrical, i.e. the 1aa vs. 2aa pair has a DE different from that of the 2aa vs. 1aa pair. (C) Positional summation of DE. Positional sum of the DE values (shown in the lowest row) provides the number of pairwise discriminations provided by the given position. The sums of the DE values are input data for calculating the AEVs). AEV is generated by dividing the positional sum of DE with the number of the identities (which in a real case is 20 while in this didactic case 3). Formalism and more detailed description are provided in Section 2. This figure can be viewed in colour online." path-from-xml="dss00801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163525/dss00801.jpeg" path-from-xml="dss00801.jpeg" target="_blank">View large</a><a section="7163525" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00801.jpeg?Expires=2147483647&Signature=sMhI-MA6I0GHwnwbiUMhddfQ16eHv13PcShU7zwTmBWhFhsDSyTmGpkqI2XHp3h6M1xloNClhgj4yPXwfb9DLhj72ATzHD4PvxW3wQYWIFepxiR~8S0yDAqkqknbog9rAY-EWJSQ0jUMmBqd9jPWHm9aFZBCDUPtCfEPyNLQZ3VcuQgiJq2yW~irouN3mc9Nh-QtotGUDQT~kjdSnK9DIr6DaGbPB4Rmg908OTdg7N7-ZfAvTmcAJZfQnsr67mSafWAjj9j-Daek0acHMLnjGmOpD7joty7Xzk1nQwM6YEuAvbV~a1bt8ssmfYRBv~F2AAwmE2R-VXdSFiuQ7AeR8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163525&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Illustration of calculating the AEV using short artificial sequences. (A) Identifying DEs using an artificial set of short sequences belonging to three amino acid identity sets. Identities are labelled as 1aa, 2aa and 3aa and are highlighted as cyan, yellow and magenta, respectively. Each identity set contains four short, tetramer sequences. The calculated DE is either one base or a combination of bases and it is labelled by IUPAC codes of bases or base sets. Each step of the DE-generating algorithm is explained in Section 2. (B) The calculated DE for each identity pairs. Note that the DE-generating relationship is non-symmetrical, i.e. the 1aa vs. 2aa pair has a DE different from that of the 2aa vs. 1aa pair. (C) Positional summation of DE. Positional sum of the DE values (shown in the lowest row) provides the number of pairwise discriminations provided by the given position. The sums of the DE values are input data for calculating the AEVs). AEV is generated by dividing the positional sum of DE with the number of the identities (which in a real case is 20 while in this didactic case 3). Formalism and more detailed description are provided in Section 2. This figure can be viewed in colour online.</p></div></div><p>In each identity set and for each kingdom, we scanned the positions of the filtered and aligned<sup><a reveal-id="DSS008C9" data-open="DSS008C9" class="link link-ref link-reveal xref-bibr">9</a></sup> data set. At each position, we documented the strictly absent elements, i.e. bases that at the given position are missing from each sequence of the given identity set. For each detected strictly absent element, we checked each other identity set in a pairwise manner whether any of the sequences of the other identity set contains that element. If yes, the detected strictly absent element is a DE that discriminates the two identity sets. For each kingdom and each position, these pairwise-interpreted DE elements were documented.</p><p>Mathematical description of the above procedure is described below.</p><p>We introduce the variable, <em>Y</em>. Elements of <em>Y</em> are nucleotide bases; therefore, elements are <em>Y</em>∈<em>χ</em>, where <em>χ</em> = {A,C,G,T}. The value of <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss008ileq1.gif?Expires=2147483647&Signature=N1c3e~zqZ7tjB46lotqbiDSueivmNjjjMvxZpZhvRBmOATvW83d0fBfI5MD9ZtZwgcfBwmFyiZyvDXN313ko~sjth7f5O7DI8KsAdQrRy~w3L5gMI5fVuSbja5~QAi3H2~OAjWOfZDb7XLQ54SBqR8WIJexTtaYQxVyfxV3LOojpb0e-EJsAPTUtvAgB76DXve7dpzRe9K-G7QOP3IF6-aeYR4Are8b1egGr9oI1aPkl8IGhd49f625rGYc~pH6wKYiXdHKYtzQ7cLIe79Gkp6TN9duO~AuuIaWRvWPzDj-f-tvv7T8m0rS6SxeDr94iw~gd6HGzDRd8hgo0xRr2YQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dss008ileq1.gif" /></span> is the nucleotide base corresponding to position <em>j</em> (<em>j</em> = 1, …, <em>L</em>, <em>L</em> = 96; from normal position 0 to position 73, including the extra positions from e1 to e22) of the sequence <em>k</em> (<em>k</em>= 1, …, <em>M<sub>i</sub></em>, where the value of <em>M<sub>i</sub></em> varies for individual species) of amino acid identity <em>i</em> (<em>i</em>= 1, …, <em>N</em>, <em>N</em> = 20).</p><div class="block-child-p">Then, we introduce the set of bases existing at the position <em>j</em> of identity <em>i</em>:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008ueq1.gif?Expires=2147483647&Signature=iacJOSKfa3qoI5UV-nbWudDvi9g2QBaUQk2CKOA53T6xXbv6shy-vjXJt5-J5~MZtkNjXxZ~07F3BtbfTPwv5ZosdI7yK39OBRp1ftGtQmVTFP2Didxma2hDqjWz8ZmysJ1ZxUDgN5gqqb9FPH40U0kF6BNTe3TunVjZN4N8rvRCUATa8--afQTh-I~FxON5sHC97s9EJy8razdhW9yxvZ21Jhl-Mcsz48RBqEE6c6rt4zFXMtVl3rXfpT9vCMKhrQgUaRg2lj5uPP~tEGoxnD7TLc2C2MitraYmeHN1S9htNWZ2W0eBmPE5dcueSRHu2aWsC6M1AjALLJSFtNJTdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss008ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss008ueq1.gif?Expires=2147483647&Signature=2amx09CBLxjCRmrSz6ipZfWYbd7pXerqa7-pzJcb1c~NvFT3YqC1ERIdQzHnhHHCUvIoKFj7bE9KegF8Mo3s3J5tlKnQ1W5mQfYWkJYIurKQg9WdmYow922RIYyZ~5Dy1Etie6qqcffmooHaWEuAFgRsyDYqngny9tdlSoQPlhNMZI6N~kE8Ys~GHhJUeHD-NE~7T0AjMZ8sqPtlVDts9x-p0taCYPcm31oTQ7ychSF3Afmb56Ljf5CaIASb7BUSHNh1jC2L3MfoyRTUmhNQXosGootKl~vsnUKNjxw~XVXpSlwEYwMv1BrEv2aySjlxlWaPc~1ul1Yfo3Y5JmCyIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss008ueq1.gif" /></a></div></div></div></div><div class="block-child-p">DE of identity set <em>i</em> against identity set <em>l</em> (where <em>l</em>= 1, …, <em>N</em>, <em>N</em> = 20) at position <em>j</em> is defined as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008ueq2.gif?Expires=2147483647&Signature=WMtnSxk3fABHiFAaduwWhUU8JmTgO95K0vODrmMtBAMzak1L5-RXj6yUXOEeCgZ5T-VHsH1Z8u8oTrRZclxrdRj7QLLx37A-MYOFj6raaFEaaVtyOzZ~hvKuMPwvl2BZbskp8bDBwSXoMwaeBDdCV4T9~HMw237f4O8R3osBe2o1813VtBGu6RP6B3IeI5iIZJ0uMIEQ~dgm3lr-zdXMXhzbYA~XR69EgzkYyORosLYQhzxpiwe40mXni~WvU3nScBUw91~8~edf43gbKSWl9PuENoPGC5bF5ffiNVOyhRVl0gTaAsWNiNjc8bfF-6g6KyGAOidxSURc0XmIrumZ2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss008ueq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss008ueq2.gif?Expires=2147483647&Signature=2JhgmbJ9qT-eQm3n-NvQt9VekrQYUegv7y6~3Q1IvekwppFyN7TqD5fBPyHB0x4FW~ot~H34dh2~FbM-t69b~hVTGO0zdinzo3HRF-CfGankCcEwDNEYAZ0dE5gyOZag7sytpp-TugxtQ70mx2HvqqXlSLzu7hUOBiU2oef7Ru1ottz~2ombRV87rMKC5pgGS4XiYeDqSNgbevNcP7wa6es0hcDWXg70qQH57RYDLgoI9TuN3WBvwmviAmP50W1GNKIAUexSfd87a5DPzHaehHDsYypnhTLyTq94v5iUxT2JoixFTjI4wXDTIFAJAiSpm7kdO0cQ9ZJpprgTceREbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss008ueq2.gif" /></a></div></div></div></div>                            <h3 scrollto-destination=7163531 id="7163531" class="section-title" data-legacy-id=s2f>2.6. The AEV</h3>
<p>We introduced the AEV to determine the weighted average frequency of DE at each position as follows. At each identity set and at each position, we determined how many identity set pairs are discriminated by the given position. These numbers from each set were summed up and were divided by 20 (the number of identities), resulting in the AEVs that demonstrate the discriminating potential of each tRNA position.</p><div class="block-child-p">In mathematical terms, the AEV is defined by the following functions:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dss008ueq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss008ueq3.gif?Expires=2147483647&amp;Signature=4q7QOhIyfVQbpOaveLJeb055rpixp9XWz9ISVV8x7WXa-qLgmYMr5l-Mv~p9kjgF-StR2grq8VkX1riVO0pnEeovWtoLlqthXd2m96D0LKabPcFD2OLdtcGIxqbTDP3XsG~KOHRNuTttILBC8APUKu73kRUxkgXo68vWfjU1jK3MJY50nEWi2THLLlolrt3sE2psjIWedRzUg3tpLlM9pUKNv~1mu5iImRDdINNUYt5FbiseWoiPIJBfHK6TK~os4rtR1NkPFCiH5LnEEA3Np19uBJJZSiIOhHHlwbr4p6p5gsdYXGi-~HPtfTpxXnKY6OeR1uA3ZAm51mADD7~k0A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss008ueq3.gif" /></a></div></div></div><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder"  id="dss008ueq4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss008ueq4.gif?Expires=2147483647&amp;Signature=jMUjAB19woedwqLUydhx-6jCpvQASz3YJ1ZfOfk0ngaBmurWYiu2gP~SvjYSSm7yhk7IGqr7Zop35nYtVnXk87tXJwTHdfBckP5sQzICRCn3OzWt2EFihVQzIfvc18Za3sVckfxzLmtyUmZ3wyCgMx72jWNs7dmZtJExdsSbKsumHbBhMVS4MT4cyGiaxy9eEGTJEA2-kdhGxFdyygNFZeSI3USA531ZhPDlbSEKsv5p75Z7RB8iNu2xs5DiOSYlay5uYsThlRTujxhHW-xB5Ky4FrVzRu3LY1XWN1xkq3Y9oVBLg4br5ibT5C8w-hY5-MfOYozseEF6uGcSmsCmWw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss008ueq4.gif" /></a></div></div></div></div><p>The <em>n <sup>j</sup></em> value is denoted as the AEV.</p>                            <h3 scrollto-destination=7163535 id="7163535" class="section-title" data-legacy-id=s2g>2.7. Statistical analysis</h3>
<p>We performed a simple statistical analysis to assess how the positional AEVs relate to the number of hitherto identified identity determinants. We assume that unidentified determinants still exist; therefore, the determinant set is only a subset of all existing elements. The test was done only on the bacterial set because only that sequence set contained enough input sequences. We correlated the AEVs with the number of published determinants (NPDs) with both the Pearson and Spearman analyses.</p><p>As both types of correlation were relatively weak, we also performed a bootstrap analysis as explained in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S2</a>. In addition, the positional NPD and AEVs were compared with their respective weighted average values and ranked from very low to very high values based on standard deviation.</p>                            <h2 scrollto-destination=7163538 id="7163538" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>The ECP analysis was performed both on the final filtered data sets and omitting the second filtration step (see later). At each position, the AEV was calculated and compared with the positional NPD value. Positional AEV/NPD values for bacterial and eukaryotic data and AEVs for Archaea are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. In Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a> at each position, AEV and NPD values were plotted and colour coded in the same diagram as explained in the figure legend. </p>                        <a id="7163540" scrollto-destination="7163540"></a>
<div id="DSS008F2" data-id="DSS008F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00802.jpeg?Expires=2147483647&Signature=QYNXGN7esA2lYAkO-mlqz3zT8O5-WQB1RqkIvXLjAK6wrVJhztvxozC-7oL9IO97qZ5jxNf1Bq9n50RHZ42wYrxynb0oKUHZxcAvG3I5XJcUZPGcl39BlFxkn~t5nRtG3XWl7k2m2v9RWUlEEZQc8GDD1ywKt9YpdZfbM1tS9ZL6VX9-p2RWTCpeSifbhB7S3~ZfoCwSpwAMnCKdx5Ew5g68MkZ~jH0m~ZGDNzLSpCEnZwomGwbXUzYuy5USiQPsQguGOq5Jgz~~0Ygb~V4sMIe07opOMQlyLx8d1NKYz5bhEjxq0hr8wF1x0KQUAysbpeV5SJ25qEYfiGA4DQjBYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Results of the Bacterial (A and B); Eukaryote (C and D) and Archaea (E and F) data set. ECP-diagrams (A, C, E). In each ECP diagram, each column belongs to a position. The upper column set refers to positional AEVs. The colour codes for statistical analysis. For each kingdom, the weighted average of AEV calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 6.45 ± 3.54; Eukaryote 6.31 ± 3.49 and Archaea 6.72 ± 3.69. Segments of columns are colour coded based on their deviation from the weighted average as follows: below −1.5 SD blue; between −1.5 and −0.5 SD cyan; between −0.5 SD and +0.5 SD green; between +0.5 and +1.5 yellow; above +1.5 SD red. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the unpopulated e1 position in the Achaea set. The lower column set illustrates the positional NPDs. The logic of the colour-coding scheme is the same as for the AEVs. For each kingdom, the weighted average of NPD calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 1.63 ± 3.63; Eukaryote 0.49 ± 1.7. Archaea have too few experimentally verified determinants (listed in Table 2) to perform this analysis. ECP-cloverleaf (B, D, F). The cloverleaf structure illustrates spatial relationships of many base-pairing residues. Each position is illustrated as a circle. The upper half of the circle corresponds to the AEV and its colour coding is the same as for the corresponding ECP diagram. The lower half of the circle corresponds to the number of published identity elements. The colour coding is similar to that in the corresponding ECP diagrams except for positions where the NPD is zero. These positions are indicated as gray. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the single unpopulated position e1 in the Achaea set. This figure can be viewed in colour online." path-from-xml="dss00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163540/dss00802.jpeg" path-from-xml="dss00802.jpeg" target="_blank">View large</a><a section="7163540" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00802.jpeg?Expires=2147483647&Signature=VlfG0RNdsbqH8tt8T8mSgItdY4upU3sQzOiDufB3w3yXjDlVzMMN5AEnVYaH7rChQ7DmSWiu9G55JcO9Qw7d19TFHLCtKaDw8IQmArbRn3Q1PhLBl~GAEKw9-TX5~BHsKyBAGjvOmSv9HPHq5CGWiMT2PQ2fC3uCzWdl5V0bySEIS8rgVcUgVSWO6udTxqkZwK8fe0o~dpDQZCyI7vAPp6eW0Oni7Q5vuv-wWGN1lwLAj3VpkY4pnk4fPp1GcxgKIAs4L0G0Gkl1gBvWS0~OoHNfUZgvQup6yoeJaDjGVXnX~MBCts4HGILgff3~L9oyxGZgCbgxj7XRflz~m7zI8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163540&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Results of the Bacterial (A and B); Eukaryote (C and D) and Archaea (E and F) data set. ECP-diagrams (A, C, E). In each ECP diagram, each column belongs to a position. The upper column set refers to positional AEVs. The colour codes for statistical analysis. For each kingdom, the weighted average of AEV calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 6.45 ± 3.54; Eukaryote 6.31 ± 3.49 and Archaea 6.72 ± 3.69. Segments of columns are colour coded based on their deviation from the weighted average as follows: below −1.5 SD blue; between −1.5 and −0.5 SD cyan; between −0.5 SD and +0.5 SD green; between +0.5 and +1.5 yellow; above +1.5 SD red. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the unpopulated e1 position in the Achaea set. The lower column set illustrates the positional NPDs. The logic of the colour-coding scheme is the same as for the AEVs. For each kingdom, the weighted average of NPD calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 1.63 ± 3.63; Eukaryote 0.49 ± 1.7. Archaea have too few experimentally verified determinants (listed in Table <a reveal-id="DSS008TB2" data-open="DSS008TB2" class="link link-reveal link-table xref-fig">2</a>) to perform this analysis. ECP-cloverleaf (B, D, F). The cloverleaf structure illustrates spatial relationships of many base-pairing residues. Each position is illustrated as a circle. The upper half of the circle corresponds to the AEV and its colour coding is the same as for the corresponding ECP diagram. The lower half of the circle corresponds to the number of published identity elements. The colour coding is similar to that in the corresponding ECP diagrams except for positions where the NPD is zero. These positions are indicated as gray. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the single unpopulated position e1 in the Achaea set. This figure can be viewed in colour online.</p></div></div><div content-id="DSS008F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00802.jpeg?Expires=2147483647&Signature=QYNXGN7esA2lYAkO-mlqz3zT8O5-WQB1RqkIvXLjAK6wrVJhztvxozC-7oL9IO97qZ5jxNf1Bq9n50RHZ42wYrxynb0oKUHZxcAvG3I5XJcUZPGcl39BlFxkn~t5nRtG3XWl7k2m2v9RWUlEEZQc8GDD1ywKt9YpdZfbM1tS9ZL6VX9-p2RWTCpeSifbhB7S3~ZfoCwSpwAMnCKdx5Ew5g68MkZ~jH0m~ZGDNzLSpCEnZwomGwbXUzYuy5USiQPsQguGOq5Jgz~~0Ygb~V4sMIe07opOMQlyLx8d1NKYz5bhEjxq0hr8wF1x0KQUAysbpeV5SJ25qEYfiGA4DQjBYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Results of the Bacterial (A and B); Eukaryote (C and D) and Archaea (E and F) data set. ECP-diagrams (A, C, E). In each ECP diagram, each column belongs to a position. The upper column set refers to positional AEVs. The colour codes for statistical analysis. For each kingdom, the weighted average of AEV calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 6.45 ± 3.54; Eukaryote 6.31 ± 3.49 and Archaea 6.72 ± 3.69. Segments of columns are colour coded based on their deviation from the weighted average as follows: below −1.5 SD blue; between −1.5 and −0.5 SD cyan; between −0.5 SD and +0.5 SD green; between +0.5 and +1.5 yellow; above +1.5 SD red. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the unpopulated e1 position in the Achaea set. The lower column set illustrates the positional NPDs. The logic of the colour-coding scheme is the same as for the AEVs. For each kingdom, the weighted average of NPD calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 1.63 ± 3.63; Eukaryote 0.49 ± 1.7. Archaea have too few experimentally verified determinants (listed in Table 2) to perform this analysis. ECP-cloverleaf (B, D, F). The cloverleaf structure illustrates spatial relationships of many base-pairing residues. Each position is illustrated as a circle. The upper half of the circle corresponds to the AEV and its colour coding is the same as for the corresponding ECP diagram. The lower half of the circle corresponds to the number of published identity elements. The colour coding is similar to that in the corresponding ECP diagrams except for positions where the NPD is zero. These positions are indicated as gray. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the single unpopulated position e1 in the Achaea set. This figure can be viewed in colour online." path-from-xml="dss00802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163540/dss00802.jpeg" path-from-xml="dss00802.jpeg" target="_blank">View large</a><a section="7163540" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00802.jpeg?Expires=2147483647&Signature=VlfG0RNdsbqH8tt8T8mSgItdY4upU3sQzOiDufB3w3yXjDlVzMMN5AEnVYaH7rChQ7DmSWiu9G55JcO9Qw7d19TFHLCtKaDw8IQmArbRn3Q1PhLBl~GAEKw9-TX5~BHsKyBAGjvOmSv9HPHq5CGWiMT2PQ2fC3uCzWdl5V0bySEIS8rgVcUgVSWO6udTxqkZwK8fe0o~dpDQZCyI7vAPp6eW0Oni7Q5vuv-wWGN1lwLAj3VpkY4pnk4fPp1GcxgKIAs4L0G0Gkl1gBvWS0~OoHNfUZgvQup6yoeJaDjGVXnX~MBCts4HGILgff3~L9oyxGZgCbgxj7XRflz~m7zI8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163540&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Results of the Bacterial (A and B); Eukaryote (C and D) and Archaea (E and F) data set. ECP-diagrams (A, C, E). In each ECP diagram, each column belongs to a position. The upper column set refers to positional AEVs. The colour codes for statistical analysis. For each kingdom, the weighted average of AEV calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 6.45 ± 3.54; Eukaryote 6.31 ± 3.49 and Archaea 6.72 ± 3.69. Segments of columns are colour coded based on their deviation from the weighted average as follows: below −1.5 SD blue; between −1.5 and −0.5 SD cyan; between −0.5 SD and +0.5 SD green; between +0.5 and +1.5 yellow; above +1.5 SD red. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the unpopulated e1 position in the Achaea set. The lower column set illustrates the positional NPDs. The logic of the colour-coding scheme is the same as for the AEVs. For each kingdom, the weighted average of NPD calculated for all positions from 0 to 73 as a single set and the corresponding standard deviation is as follows: Bacteria 1.63 ± 3.63; Eukaryote 0.49 ± 1.7. Archaea have too few experimentally verified determinants (listed in Table <a reveal-id="DSS008TB2" data-open="DSS008TB2" class="link link-reveal link-table xref-fig">2</a>) to perform this analysis. ECP-cloverleaf (B, D, F). The cloverleaf structure illustrates spatial relationships of many base-pairing residues. Each position is illustrated as a circle. The upper half of the circle corresponds to the AEV and its colour coding is the same as for the corresponding ECP diagram. The lower half of the circle corresponds to the number of published identity elements. The colour coding is similar to that in the corresponding ECP diagrams except for positions where the NPD is zero. These positions are indicated as gray. White illustrates positions where the analysis is not applicable. These are the 3′ CCA end for each kingdom, and the single unpopulated position e1 in the Achaea set. This figure can be viewed in colour online.</p></div></div><p>The positional AEV/NPD patterns show characteristic similarities. In the following paragraphs, we organized the results from the highest to the lowest AEVs.</p>                            <h3 scrollto-destination=7163542 id="7163542" class="section-title" data-legacy-id=s3a>3.1. Bacterial (coli-like) data set</h3>
<p>The density function of the AEVs in the bacterial set shows a normal distribution as the number of positions having values over the weighted average plus 0.5 SD (31 positions) practically equals those that are below the weighted average minus 0.5 SD values (32 positions). In order to facilitate the visual comparison of NPD and the AEVs at each position, these are illustrated in a composite plot as shown in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>A and B.</p><p>Importantly, two anticodon positions, 35 and 36, have the highest AEVs (located in the red zone) and these have the highest NPD values as well. On the other hand, position 34 (located in the yellow zone), which pairs with the third, wobbling codon position, has significantly (over one sigma) lower discriminating potential. Notably, out of the three anticodon positions, this contains the lowest NPD as well. Most positions with high AEVs (located in the yellow zone) give place to known identity elements and it is also clear that the AEVs of positions that base pair with each other correlate. In the acceptor arm, three position pairs contain determinant for many identities. These are 1:72 (Trp, Gly, Thr, Gln), 2:71 (Met, Trp, Asp, Gly, Ser, Cys, Ala, Gln) and 3:70 (Val, Met, Trp, Gly, Ser, Cys, Ala, Gln), which as a set carry known determinants for half of the identities.<sup><a reveal-id="DSS008C3" data-open="DSS008C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The discriminator base in position 73 has the third highest AEV right behind positions 35 and 36.</p><p>In the tRNA core region,<sup><a reveal-id="DSS008C27" data-open="DSS008C27" class="link link-ref link-reveal xref-bibr">27</a></sup> the 12:23 pair having high AEVs overlaps with the know tRNA<sup>Ile</sup> identity element T12:A23. Furthermore, high AEV positions 13, 22 and 46 have a published role as tRNA<sup>Glu</sup> identity element T13:G22:A46<sup><a reveal-id="DSS008C28  DSS008C30" data-open="DSS008C28  DSS008C30" class="link link-ref link-reveal xref-bibr">28–30</a></sup> and deletion of position 47 with high AEV was also identified as tRNA<sup>Glu</sup> determinant. The 13:22 pair has been identified as determinant for tRNA<sup>Cys</sup> as well.<sup><a reveal-id="DSS008C31" data-open="DSS008C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSS008C32" data-open="DSS008C32" class="link link-ref link-reveal xref-bibr">32</a></sup></p><p>Other positions having higher than average AEVs host several identity elements as follows. Position 38 in the anticodon loop contains determinant for tRNA<sup>Ile</sup>, tRNA<sup>Asp</sup> and tRNA<sup>Gln</sup>,<sup><a reveal-id="DSS008C33" data-open="DSS008C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSS008C34" data-open="DSS008C34" class="link link-ref link-reveal xref-bibr">34</a></sup> while positions 10 (tRNA<sup>Asp</sup>, tRNA<sup>Gln</sup>); 11:24 (tRNA<sup>Ser</sup>, tRNA<sup>Glu</sup>); 15:84 (tRNA<sup>Cys</sup>, tRNA<sup>Pro</sup>); 20 (tRNA<sup>Phe</sup>, tRNA<sup>Arg</sup>, tRNA<sup>Ala</sup>); 29:41 (tRNA<sup>Ile</sup>) and 45 (tRNA<sup>Phe</sup>) also have a few indicated determinants.<sup><a reveal-id="DSS008C3" data-open="DSS008C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS008C35" data-open="DSS008C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>At the more conserved T-loop position 60 having average AEV, there is another tRNA<sup>Phe</sup> determinant.<sup><a reveal-id="DSS008C3" data-open="DSS008C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS008C35" data-open="DSS008C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The variable loop position having the highest, but otherwise average, AEV is e2, which is a known identity element of tRNA<sup>Ser</sup>.<sup><a reveal-id="DSS008C36" data-open="DSS008C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p><p>The position pair 31:39 at the anticodon loop has no known identity element associated. In 2338 out of the 2406 analysed non-redundant bacterial sequences, these form normal Watson–Crick base pair, but based on the high AEVs, the exact identity of the given base pair might serve an identity element function.</p><p>Based on the high AEV but low NPD values, position pairs 12:23 and 13:22 as well as Position 46 might contain hitherto unidentified determinants.</p><p>The facultative elements (17, 17a, 20a and 20b) in the D-loop and in position 47 might also have identity functions (that will be detailed in Section 3.5).</p><p>Several elements with the lowest AEVs indicated in blue in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>A and C do not show any discrimination potentials. These are functionally highly conserved elements shared by all tRNA sequences and most of these were used in the first filtering step. Positions with still lower than average AEVs highlighted with cyan or those with average AEV highlighted as green typically coincide with regions that do not contain known identity elements. For example, at the 6:67 pair, the NPD is zero. Nevertheless, there are some exceptions. For the 5:68 position pair, a single identity element has been published, the A5:T68 for tRNA<sup>Met</sup>.<sup><a reveal-id="DSS008C37" data-open="DSS008C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Moreover, the conserved A37 in the anticodon loop has been identified as Ile, Met, Glu and Gln determinant,<sup><a reveal-id="DSS008C27" data-open="DSS008C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSS008C37  DSS008C40" data-open="DSS008C37  DSS008C40" class="link link-ref link-reveal xref-bibr">37–40</a></sup> although over half of the tRNA sequences harbour adenine at this position. Finally, in spite of being located at low AEV regions, the U8:A14 for tRNA<sup>Leu<a reveal-id="DSS008C41" data-open="DSS008C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and the G27:C43, G28:C42 and T59 for tRNA<sup>Phe</sup> have also been shown to be determinants.<sup><a reveal-id="DSS008C42" data-open="DSS008C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=7163552 id="7163552" class="section-title" data-legacy-id=s3b>3.2. Eukaryotic (yeast-like) data set</h3>
<p>While the bacterial data set represented coli-like identity rules, the eukaryotic set contained sequences that conformed to the already established yeast identity rules. The results are illustrated in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a> in the same way as for the bacterial set.</p><p>Just like in the case of the bacterial set, the majority of known determinants are located at positions having the highest AEVs, namely the three anticodon positions and the discriminator position 73. The three base pairs of the acceptor arm have the next highest NPD values and these also represent higher than average AEVs. Another high AEV position where a single known identity element for tRNA<sup>Phe</sup> exists is position 20.<sup><a reveal-id="DSS008C43" data-open="DSS008C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p><p>There are several positions where higher than average AEVs exist, but no determinants have been identified. At some of these positions such as 12:23; 13:22 and 45; 46; 47 that participate in establishing the core region, there are already published determinants for the bacterial set.</p><p>Interestingly, the high AEV positions 31:39 and 30:40 in the anticodon stem contain no identified yeast determinants, but harbour published determinants for human tRNA<sup>Phe</sup>.<sup><a reveal-id="DSS008C44" data-open="DSS008C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p><p>The lower than average AEVs indicated with blue and cyan colours in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>C and D are again all associated with conserved positions lacking any known identity elements. Very few published identity elements coincide with positions having average AEVs indicated with green in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>C and D. Position 3 harbours determinants for tRNA<sup>Gly</sup> and tRNA<sup>Ala</sup>, the anticodon loop position 37 for tRNA<sup>Leu</sup> and positions 38 and 10:25 for tRNA<sup>Asp</sup>.<sup><a reveal-id="DSS008C45" data-open="DSS008C45" class="link link-ref link-reveal xref-bibr">45</a>–<a reveal-id="DSS008C49" data-open="DSS008C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Position 70, which pairs with position 3, has a higher than average AEV.</p>                            <h3 scrollto-destination=7163558 id="7163558" class="section-title" data-legacy-id=s3c>3.3. Testing potential biases introduced by the second filtering step</h3>
<p>We have checked how the second filtering step alters the input sequence set from various species. For bacteria, the effect of filtering nicely mirrored evolutionary relations. Data in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> show that after the second filtering step, species containing the highest number of retained sequences are the closest relatives of <em>E. coli</em>. Namely, from the Gammaproteobacteria class (genera <em>Escherichia</em>, <em>Haemophilus</em>, <em>Salmonella</em>, <em>Yersinia</em>, <em>Buchnera</em>, <em>Shigella</em>), 75–90% of the input sequences are retained by the second filtering step. Moving away from <em>E. coli</em> on the phylogenetic tree, the proportion of retained sequences gradually decreases. Still high (around 70%) proportion of sequences are retained in the case of Proteobacteria (genera <em>Desulfovibrio</em>, <em>Brucella</em>, <em>Campylobacter</em>), but, for example, in the case of Firmicutes (genera <em>Streptococcus</em>, <em>Bacillus</em>, <em>Lactobacillus</em>, <em>Lactococcus</em>, <em>Staphylococcus</em>) only 50–70%, while in the case of Tenericutes (Mycoplasma, Ureaplasma) or Actinobacteria (Mycobacterium, Streptomyces), only 30–50% of the input sequences are retained.</p><p>In the case of Eukarya, no such trend was observed. This might be due to the—compared with bacteria—much lower number and possibly more general nature of determinants published for yeast.</p><p>The positional AEVs measure the average distance of functionally defined sets. One might think that the second filtering for the presence of identity elements could increase the separation of the identity sets and thus increase the AEVs at identity element positions. In order to assess this potential effect, we repeated the analysis for the bacterial and the eukaryotic sets by omitting the second filtration. The data are organized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>, Table <a reveal-id="DSS008TB1" data-open="DSS008TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figure S1</a>. </p>                        <a id="7163562" scrollto-destination="7163562"></a>
<div content-id="DSS008TB1" class="table-modal"><div class="table-wrap"><div id="DSS008TB1" class="table-wrap-title" data-id="DSS008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Representative data for tDNA sequence set processing and analysis with and without filtering for the presence of known identity elements</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Bacteria<hr /> </th><th colspan="2">Eukaryota<hr /> </th><th>Archaea </th></tr><tr><th>Without<sup>a</sup> second filtration </th><th>With second filtration </th><th>Without second filtration </th><th>With second filtration </th><th>No second filtration </th></tr></thead><tbody><tr><td colspan="6">No. of sequences </td></tr><tr><td> Raw data </td><td colspan="2">6243 </td><td colspan="2">2222 </td><td>1552 </td></tr><tr><td> First filtration </td><td colspan="2">6144 </td><td colspan="2">1930 </td><td>1384 </td></tr><tr><td> Second filtration </td><td>— </td><td>3901 </td><td>— </td><td>1672 </td><td>— </td></tr><tr><td> Non-redundant data set </td><td>3946 </td><td>2406 </td><td>1495 </td><td>1264 </td><td>1041 </td></tr><tr><td colspan="6">AEV </td></tr><tr><td> Average </td><td>6.45 </td><td>5.59 </td><td>5.79 </td><td>6.31 </td><td>7.34 </td></tr><tr><td> SD </td><td>3.54 </td><td>3.51 </td><td>3.43 </td><td>3.49 </td><td>3.97 </td></tr><tr><td>Pearson<sup>b</sup> (<em>R</em>) </td><td>0.53 </td><td>0.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Spearman<sup>b</sup> (<em>ρ</em>) </td><td>0.39 </td><td>0.54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td colspan="6">Bootstrap<sup>c</sup> </td></tr><tr><td> Mean </td><td>224 </td><td>258 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> SD </td><td>16.9 </td><td>17.1 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Cumulative AEV threshold </td><td>358.95 </td><td>344.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Significance (<em>P</em>) </td><td>1.33e<sup>−15</sup> </td><td>3.54e<sup>−7</sup> </td><td>— </td><td>— </td><td>— </td></tr></tbody></table></div><div content-id="DSS008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Bacteria<hr /> </th><th colspan="2">Eukaryota<hr /> </th><th>Archaea </th></tr><tr><th>Without<sup>a</sup> second filtration </th><th>With second filtration </th><th>Without second filtration </th><th>With second filtration </th><th>No second filtration </th></tr></thead><tbody><tr><td colspan="6">No. of sequences </td></tr><tr><td> Raw data </td><td colspan="2">6243 </td><td colspan="2">2222 </td><td>1552 </td></tr><tr><td> First filtration </td><td colspan="2">6144 </td><td colspan="2">1930 </td><td>1384 </td></tr><tr><td> Second filtration </td><td>— </td><td>3901 </td><td>— </td><td>1672 </td><td>— </td></tr><tr><td> Non-redundant data set </td><td>3946 </td><td>2406 </td><td>1495 </td><td>1264 </td><td>1041 </td></tr><tr><td colspan="6">AEV </td></tr><tr><td> Average </td><td>6.45 </td><td>5.59 </td><td>5.79 </td><td>6.31 </td><td>7.34 </td></tr><tr><td> SD </td><td>3.54 </td><td>3.51 </td><td>3.43 </td><td>3.49 </td><td>3.97 </td></tr><tr><td>Pearson<sup>b</sup> (<em>R</em>) </td><td>0.53 </td><td>0.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Spearman<sup>b</sup> (<em>ρ</em>) </td><td>0.39 </td><td>0.54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td colspan="6">Bootstrap<sup>c</sup> </td></tr><tr><td> Mean </td><td>224 </td><td>258 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> SD </td><td>16.9 </td><td>17.1 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Cumulative AEV threshold </td><td>358.95 </td><td>344.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Significance (<em>P</em>) </td><td>1.33e<sup>−15</sup> </td><td>3.54e<sup>−7</sup> </td><td>— </td><td>— </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The optional second filtration was performed based on the presence of experimentally verified <em>E. coli</em> (for Bacteria) or yeast (for Eukaryota) identity elements. Archaea lack enough verified identity elements; therefore, the second filtration was not performed.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Correlation of the NPDs and the AEVs was done as described in Section 2.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The bootstrap analysis is described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S2</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7163562" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS008TB1" class="table-wrap-title" data-id="DSS008TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Representative data for tDNA sequence set processing and analysis with and without filtering for the presence of known identity elements</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Bacteria<hr /> </th><th colspan="2">Eukaryota<hr /> </th><th>Archaea </th></tr><tr><th>Without<sup>a</sup> second filtration </th><th>With second filtration </th><th>Without second filtration </th><th>With second filtration </th><th>No second filtration </th></tr></thead><tbody><tr><td colspan="6">No. of sequences </td></tr><tr><td> Raw data </td><td colspan="2">6243 </td><td colspan="2">2222 </td><td>1552 </td></tr><tr><td> First filtration </td><td colspan="2">6144 </td><td colspan="2">1930 </td><td>1384 </td></tr><tr><td> Second filtration </td><td>— </td><td>3901 </td><td>— </td><td>1672 </td><td>— </td></tr><tr><td> Non-redundant data set </td><td>3946 </td><td>2406 </td><td>1495 </td><td>1264 </td><td>1041 </td></tr><tr><td colspan="6">AEV </td></tr><tr><td> Average </td><td>6.45 </td><td>5.59 </td><td>5.79 </td><td>6.31 </td><td>7.34 </td></tr><tr><td> SD </td><td>3.54 </td><td>3.51 </td><td>3.43 </td><td>3.49 </td><td>3.97 </td></tr><tr><td>Pearson<sup>b</sup> (<em>R</em>) </td><td>0.53 </td><td>0.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Spearman<sup>b</sup> (<em>ρ</em>) </td><td>0.39 </td><td>0.54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td colspan="6">Bootstrap<sup>c</sup> </td></tr><tr><td> Mean </td><td>224 </td><td>258 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> SD </td><td>16.9 </td><td>17.1 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Cumulative AEV threshold </td><td>358.95 </td><td>344.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Significance (<em>P</em>) </td><td>1.33e<sup>−15</sup> </td><td>3.54e<sup>−7</sup> </td><td>— </td><td>— </td><td>— </td></tr></tbody></table></div><div content-id="DSS008TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Bacteria<hr /> </th><th colspan="2">Eukaryota<hr /> </th><th>Archaea </th></tr><tr><th>Without<sup>a</sup> second filtration </th><th>With second filtration </th><th>Without second filtration </th><th>With second filtration </th><th>No second filtration </th></tr></thead><tbody><tr><td colspan="6">No. of sequences </td></tr><tr><td> Raw data </td><td colspan="2">6243 </td><td colspan="2">2222 </td><td>1552 </td></tr><tr><td> First filtration </td><td colspan="2">6144 </td><td colspan="2">1930 </td><td>1384 </td></tr><tr><td> Second filtration </td><td>— </td><td>3901 </td><td>— </td><td>1672 </td><td>— </td></tr><tr><td> Non-redundant data set </td><td>3946 </td><td>2406 </td><td>1495 </td><td>1264 </td><td>1041 </td></tr><tr><td colspan="6">AEV </td></tr><tr><td> Average </td><td>6.45 </td><td>5.59 </td><td>5.79 </td><td>6.31 </td><td>7.34 </td></tr><tr><td> SD </td><td>3.54 </td><td>3.51 </td><td>3.43 </td><td>3.49 </td><td>3.97 </td></tr><tr><td>Pearson<sup>b</sup> (<em>R</em>) </td><td>0.53 </td><td>0.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Spearman<sup>b</sup> (<em>ρ</em>) </td><td>0.39 </td><td>0.54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td colspan="6">Bootstrap<sup>c</sup> </td></tr><tr><td> Mean </td><td>224 </td><td>258 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> SD </td><td>16.9 </td><td>17.1 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Cumulative AEV threshold </td><td>358.95 </td><td>344.55 </td><td>— </td><td>— </td><td>— </td></tr><tr><td> Significance (<em>P</em>) </td><td>1.33e<sup>−15</sup> </td><td>3.54e<sup>−7</sup> </td><td>— </td><td>— </td><td>— </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The optional second filtration was performed based on the presence of experimentally verified <em>E. coli</em> (for Bacteria) or yeast (for Eukaryota) identity elements. Archaea lack enough verified identity elements; therefore, the second filtration was not performed.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Correlation of the NPDs and the AEVs was done as described in Section 2.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The bootstrap analysis is described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S2</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7163562" target="_blank">View Large</a></div></div><p>Briefly, in the case of the bacterial data set, where 40 positions carry known identity elements, second filtration removed 39% of the input sequences, e.g. those that lacked at least one required coli identity element. As a result, at positions, where known identity elements exist (positive NPD), the sum of the AEV increased with 22%, while at positions with no reported identity elements (zero NPD), the increase was only 8%. Out of the 40 positions, this AEV increase exceeded the standard deviation at 10 positions.</p><p>In the case of the eukaryotic data set, where only 15 positions carry known identity elements, second filtration caused a much smaller effect. It removed only 15% of the input sequences, e.g. those that lacked at least one required yeast identity element. As a result, at positions where known identity elements exist (positive NPD), the sum of the AEV decreased with 5%, while at positions with no reported identity elements (zero NPD), the decrease was 9%. However, this decrease was statistically significant only at position 23, where no identity element has been reported.</p><p>For the bacterial set, we performed several statistical analyses both in the case of the second filtered and non-filtered data sets (Table <a reveal-id="DSS008TB1" data-open="DSS008TB1" class="link link-reveal link-table xref-fig">1</a>). The Pearson correlation gave an <em>R</em>-value of 0.55 for the filtered and 0.53 for the non-filtered case, while the Spearmen correlation yielded <em>ρ</em>-values 0.54 and 0.39, respectively.</p><p>We also applied a bootstrap analysis to test the statistical significance of high AEV positions overlapping with positions harbouring known identity elements (Table <a reveal-id="DSS008TB1" data-open="DSS008TB1" class="link link-reveal link-table xref-fig">1</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data S2</a>). Importantly, the overlap was statistically significant both in the non-filtered and in the filtered case, demonstrating that omitting the second filtration step did not change the overall distribution of the AEVs. The highest AEVs are at the anticodon positions, the discriminator base and the acceptor arm (1:72; 2:71). These are well-known positions of identity elements.</p><p>This suggests that this filtration does not introduce artefacts. On the other hand, as we emphasized, only the second filtration yields a data set, in which sequences belonging to the same identity set share known identity elements, suggesting that they might also share yet unidentified common identity elements.</p>                            <h3 scrollto-destination=7163568 id="7163568" class="section-title" data-legacy-id=s3d>3.4. Archaea data set</h3>
<p>The second filtration step was not performed for the Archaea data set as only sparse experimental data are available for such elements (listed in Table <a reveal-id="DSS008TB2" data-open="DSS008TB2" class="link link-reveal link-table xref-fig">2</a>). The only comprehensive analysis available for each identity set is an <em>in silico</em> study based on sequence alignments.<sup><a reveal-id="DSS008C50" data-open="DSS008C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p>                        <a id="7163570" scrollto-destination="7163570"></a>
<div content-id="DSS008TB2" class="table-modal"><div class="table-wrap"><div id="DSS008TB2" class="table-wrap-title" data-id="DSS008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Experimentally determined and published Archaea identity elements</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Identity </th><th>Element </th><th>Species </th><th>Refs. </th></tr></thead><tbody><tr><td>Ala </td><td>G3:U70 </td><td><em>Archaeoglobus fulgidus</em>; <em>Pyrococcus horikoshii</em> </td><td><a reveal-id="DSS008C71" data-open="DSS008C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSS008C72" data-open="DSS008C72" class="link link-ref link-reveal xref-bibr">72</a> </td></tr><tr><td>Asp </td><td>C36 </td><td><em>Pyrococcus kodakaraensis</em> </td><td><a reveal-id="DSS008C73  DSS008C75" data-open="DSS008C73  DSS008C75" class="link link-ref link-reveal xref-bibr">73–75</a> </td></tr><tr><td>Gly </td><td>C35, C36, C2:G71, G3:C70 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C76" data-open="DSS008C76" class="link link-ref link-reveal xref-bibr">76</a> </td></tr><tr><td>His </td><td>C73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C77" data-open="DSS008C77" class="link link-ref link-reveal xref-bibr">77</a> </td></tr><tr><td>Phe </td><td>G34, A35 A36, A73, G20 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C55" data-open="DSS008C55" class="link link-ref link-reveal xref-bibr">55</a> </td></tr><tr><td>Pro </td><td>G35, G36, A73, G1:C72 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C78" data-open="DSS008C78" class="link link-ref link-reveal xref-bibr">78</a> </td></tr><tr><td>Ser </td><td>G30:C40, G73, variable loop G1:C72, C3:G70 variable loop </td><td><em>Methanosarcina barkeri</em> (Archaeal RS); <em>Methanococcus maripaludis</em> </td><td><a reveal-id="DSS008C56" data-open="DSS008C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSS008C79" data-open="DSS008C79" class="link link-ref link-reveal xref-bibr">79</a> </td></tr><tr><td>Thr </td><td>U73, C2:G71 </td><td><em>Haloferax volcanii</em>; <em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C54" data-open="DSS008C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSS008C80" data-open="DSS008C80" class="link link-ref link-reveal xref-bibr">80</a>,<a reveal-id="DSS008C81" data-open="DSS008C81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td>Trp </td><td>C34, C35, A36, A73, G1:C72, G2:C71 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C82" data-open="DSS008C82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td>Tyr </td><td>C1:G72, A73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C57" data-open="DSS008C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS008C58" data-open="DSS008C58" class="link link-ref link-reveal xref-bibr">58</a> </td></tr></tbody></table></div><div content-id="DSS008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Identity </th><th>Element </th><th>Species </th><th>Refs. </th></tr></thead><tbody><tr><td>Ala </td><td>G3:U70 </td><td><em>Archaeoglobus fulgidus</em>; <em>Pyrococcus horikoshii</em> </td><td><a reveal-id="DSS008C71" data-open="DSS008C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSS008C72" data-open="DSS008C72" class="link link-ref link-reveal xref-bibr">72</a> </td></tr><tr><td>Asp </td><td>C36 </td><td><em>Pyrococcus kodakaraensis</em> </td><td><a reveal-id="DSS008C73  DSS008C75" data-open="DSS008C73  DSS008C75" class="link link-ref link-reveal xref-bibr">73–75</a> </td></tr><tr><td>Gly </td><td>C35, C36, C2:G71, G3:C70 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C76" data-open="DSS008C76" class="link link-ref link-reveal xref-bibr">76</a> </td></tr><tr><td>His </td><td>C73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C77" data-open="DSS008C77" class="link link-ref link-reveal xref-bibr">77</a> </td></tr><tr><td>Phe </td><td>G34, A35 A36, A73, G20 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C55" data-open="DSS008C55" class="link link-ref link-reveal xref-bibr">55</a> </td></tr><tr><td>Pro </td><td>G35, G36, A73, G1:C72 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C78" data-open="DSS008C78" class="link link-ref link-reveal xref-bibr">78</a> </td></tr><tr><td>Ser </td><td>G30:C40, G73, variable loop G1:C72, C3:G70 variable loop </td><td><em>Methanosarcina barkeri</em> (Archaeal RS); <em>Methanococcus maripaludis</em> </td><td><a reveal-id="DSS008C56" data-open="DSS008C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSS008C79" data-open="DSS008C79" class="link link-ref link-reveal xref-bibr">79</a> </td></tr><tr><td>Thr </td><td>U73, C2:G71 </td><td><em>Haloferax volcanii</em>; <em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C54" data-open="DSS008C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSS008C80" data-open="DSS008C80" class="link link-ref link-reveal xref-bibr">80</a>,<a reveal-id="DSS008C81" data-open="DSS008C81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td>Trp </td><td>C34, C35, A36, A73, G1:C72, G2:C71 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C82" data-open="DSS008C82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td>Tyr </td><td>C1:G72, A73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C57" data-open="DSS008C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS008C58" data-open="DSS008C58" class="link link-ref link-reveal xref-bibr">58</a> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7163570" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS008TB2" class="table-wrap-title" data-id="DSS008TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Experimentally determined and published Archaea identity elements</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Identity </th><th>Element </th><th>Species </th><th>Refs. </th></tr></thead><tbody><tr><td>Ala </td><td>G3:U70 </td><td><em>Archaeoglobus fulgidus</em>; <em>Pyrococcus horikoshii</em> </td><td><a reveal-id="DSS008C71" data-open="DSS008C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSS008C72" data-open="DSS008C72" class="link link-ref link-reveal xref-bibr">72</a> </td></tr><tr><td>Asp </td><td>C36 </td><td><em>Pyrococcus kodakaraensis</em> </td><td><a reveal-id="DSS008C73  DSS008C75" data-open="DSS008C73  DSS008C75" class="link link-ref link-reveal xref-bibr">73–75</a> </td></tr><tr><td>Gly </td><td>C35, C36, C2:G71, G3:C70 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C76" data-open="DSS008C76" class="link link-ref link-reveal xref-bibr">76</a> </td></tr><tr><td>His </td><td>C73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C77" data-open="DSS008C77" class="link link-ref link-reveal xref-bibr">77</a> </td></tr><tr><td>Phe </td><td>G34, A35 A36, A73, G20 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C55" data-open="DSS008C55" class="link link-ref link-reveal xref-bibr">55</a> </td></tr><tr><td>Pro </td><td>G35, G36, A73, G1:C72 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C78" data-open="DSS008C78" class="link link-ref link-reveal xref-bibr">78</a> </td></tr><tr><td>Ser </td><td>G30:C40, G73, variable loop G1:C72, C3:G70 variable loop </td><td><em>Methanosarcina barkeri</em> (Archaeal RS); <em>Methanococcus maripaludis</em> </td><td><a reveal-id="DSS008C56" data-open="DSS008C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSS008C79" data-open="DSS008C79" class="link link-ref link-reveal xref-bibr">79</a> </td></tr><tr><td>Thr </td><td>U73, C2:G71 </td><td><em>Haloferax volcanii</em>; <em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C54" data-open="DSS008C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSS008C80" data-open="DSS008C80" class="link link-ref link-reveal xref-bibr">80</a>,<a reveal-id="DSS008C81" data-open="DSS008C81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td>Trp </td><td>C34, C35, A36, A73, G1:C72, G2:C71 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C82" data-open="DSS008C82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td>Tyr </td><td>C1:G72, A73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C57" data-open="DSS008C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS008C58" data-open="DSS008C58" class="link link-ref link-reveal xref-bibr">58</a> </td></tr></tbody></table></div><div content-id="DSS008TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Identity </th><th>Element </th><th>Species </th><th>Refs. </th></tr></thead><tbody><tr><td>Ala </td><td>G3:U70 </td><td><em>Archaeoglobus fulgidus</em>; <em>Pyrococcus horikoshii</em> </td><td><a reveal-id="DSS008C71" data-open="DSS008C71" class="link link-ref link-reveal xref-bibr">71</a>,<a reveal-id="DSS008C72" data-open="DSS008C72" class="link link-ref link-reveal xref-bibr">72</a> </td></tr><tr><td>Asp </td><td>C36 </td><td><em>Pyrococcus kodakaraensis</em> </td><td><a reveal-id="DSS008C73  DSS008C75" data-open="DSS008C73  DSS008C75" class="link link-ref link-reveal xref-bibr">73–75</a> </td></tr><tr><td>Gly </td><td>C35, C36, C2:G71, G3:C70 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C76" data-open="DSS008C76" class="link link-ref link-reveal xref-bibr">76</a> </td></tr><tr><td>His </td><td>C73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C77" data-open="DSS008C77" class="link link-ref link-reveal xref-bibr">77</a> </td></tr><tr><td>Phe </td><td>G34, A35 A36, A73, G20 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C55" data-open="DSS008C55" class="link link-ref link-reveal xref-bibr">55</a> </td></tr><tr><td>Pro </td><td>G35, G36, A73, G1:C72 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C78" data-open="DSS008C78" class="link link-ref link-reveal xref-bibr">78</a> </td></tr><tr><td>Ser </td><td>G30:C40, G73, variable loop G1:C72, C3:G70 variable loop </td><td><em>Methanosarcina barkeri</em> (Archaeal RS); <em>Methanococcus maripaludis</em> </td><td><a reveal-id="DSS008C56" data-open="DSS008C56" class="link link-ref link-reveal xref-bibr">56</a>,<a reveal-id="DSS008C79" data-open="DSS008C79" class="link link-ref link-reveal xref-bibr">79</a> </td></tr><tr><td>Thr </td><td>U73, C2:G71 </td><td><em>Haloferax volcanii</em>; <em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C54" data-open="DSS008C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSS008C80" data-open="DSS008C80" class="link link-ref link-reveal xref-bibr">80</a>,<a reveal-id="DSS008C81" data-open="DSS008C81" class="link link-ref link-reveal xref-bibr">81</a> </td></tr><tr><td>Trp </td><td>C34, C35, A36, A73, G1:C72, G2:C71 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C82" data-open="DSS008C82" class="link link-ref link-reveal xref-bibr">82</a> </td></tr><tr><td>Tyr </td><td>C1:G72, A73 </td><td><em>Aeropyrum pernix</em> K1 </td><td><a reveal-id="DSS008C57" data-open="DSS008C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS008C58" data-open="DSS008C58" class="link link-ref link-reveal xref-bibr">58</a> </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7163570" target="_blank">View Large</a></div></div><p>Positions of typical determinants, such as the discriminator base or members of the anticodon, have significantly higher than average AEVs (yellow and red zones in Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>A and C). For some Archaea species and some identity sets, these have been experimentally verified as determinants (see in Table <a reveal-id="DSS008TB2" data-open="DSS008TB2" class="link link-reveal link-table xref-fig">2</a>). However, it is noteworthy how differently the individual kingdoms (or sometimes groups within a kingdom) use identity elements on the acceptor stem. This can be illustrated through the example of tRNA<sup>Thr</sup>.</p><p>In the case of <em>E. coli</em>,<sup><a reveal-id="DSS008C51" data-open="DSS008C51" class="link link-ref link-reveal xref-bibr">51</a></sup> the discriminator base is not a tRNA<sup>Thr</sup> determinant, while the first three base pairs (1:72; 2:71; 3:70) are identity elements. The most important one is the 2:71 pair. For yeast,<sup><a reveal-id="DSS008C52" data-open="DSS008C52" class="link link-ref link-reveal xref-bibr">52</a></sup> the discriminator base and the first and third acceptor stem base pairs are determinants, and similar results were published for the <em>Thermus thermophilus</em> bacterium.<sup><a reveal-id="DSS008C53" data-open="DSS008C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Studies on the tRNA<sup>Thr</sup> identity elements for two Archaea species revealed that their discriminator base and acceptor stem base pairs are used differently. In the case of <em>Haloferax volcanii</em>, these were used similarly as in yeast and <em>T. thermophilus</em>, while in <em>Aeropyrum pernix</em>, these elements were used as in <em>E. coli</em>.<sup><a reveal-id="DSS008C54" data-open="DSS008C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Based on the AEVs at this area in the archahea set, the majority of the Archaea might have an identity element distribution like that in <em>A. pernix</em>. Moreover, the role of the discriminator base relative to the anticodon set appears to be dampened in this kingdom, suggesting that it might have roles in fewer identity sets than in the other two kingdoms.</p><p>The 3:70 base pair positions have excelling AEVs and this pair was shown to be determinant for tRNA<sup>Ala</sup>, tRNA<sup>Gly</sup> and tRNA<sup>Ser</sup> in a few Archaea species (Table <a reveal-id="DSS008TB2" data-open="DSS008TB2" class="link link-reveal link-table xref-fig">2</a>). Position 20 also has a higher than average AEV (yellow zone) and this position carries an identity element for tRNA<sup>Phe</sup> in Archaea.<sup><a reveal-id="DSS008C55" data-open="DSS008C55" class="link link-ref link-reveal xref-bibr">55</a></sup></p><p>The position pairs 29:41 and 31:39 at the anticodon stem also have higher than average AEVs, yet no determinants have been identified at these positions. On the other hand, in the case of <em>Methanosarcina barkeri</em>, at the 30:40 pair having average or low AEV, a Ser identity element has been found.<sup><a reveal-id="DSS008C56" data-open="DSS008C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Just like in the case of the bacterial set, there are several positions in the core region and at the facultative base positions where the AEVs are higher than average, yet no identity element has been published. </p>                        <a id="7163575" scrollto-destination="7163575"></a>
<div id="DSS008F3" data-id="DSS008F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00803.jpeg?Expires=2147483647&Signature=COr2zC1Gd7aE-3l~1jgZ64KsMpcXd2H9Tp2M4ge6R4AfJVm~lmfzXzJdtENP~uD7xJIq91AVnUt0uiRCPG9WFv0ej8cyQGYly17Km~UsVdoJjC6I~~OUtCneRQoxex0XgK3-7f1zk2JX5Kxf9nCegKcSW-TxdoFeq8IAJ3cS0X~-R--mbIscGqD2C0zIh6lZPCcvJm2YPVeDdGWhSxxcavGgG-EK8sP1p03ahvT4DFPS-IGbYWrN4L53COnJIfG~ON87qPhfL8mVuxHy0dZ6ezbG1DYLcPv7r28ZCIJSLj-e47Jq88KDqUgY8MG-SK9mMLzngrvoeZ5FUvhFy~pWkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hitherto unidentified potential identity elements. Positions having high AEV but low NPDs could harbour hitherto unidentified identity elements. Grey or black highlights the most likely positions of such elements. For positions highlighted as black, we also propose the corresponding identity: bold corresponding to yeast, while italic to E. coli. Middle grey highlights ‘core region’ positions having high AEV and low NPD in all three kingdoms. Light grey highlights the highest AEV base pair in the Archaea kingdom." path-from-xml="dss00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163575/dss00803.jpeg" path-from-xml="dss00803.jpeg" target="_blank">View large</a><a section="7163575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00803.jpeg?Expires=2147483647&Signature=N82JwqMYt2xD4DiMGkXknaB9k0QjfjJnLail0-HQfpn7P7NWr65JMm4lqD4rCuo2890tOg56f0O2u0L09ZElNdgPQWrkM0T5zslEBLA01qJncunmhVN5qVe-YXyaMfoGFLtzLBZDUOLYCJAPAAjasMyLJKiLrSvFf~VODFECas5xAJ5uMtQdDiyOR3d7augbE4hmEWoIhvXJSiLZ1kq9jk1y0vnZrV3RUErbLfMVZgumWYwcpBn1dKVCDoJYaeWpQwLXu4Go5~Aol0DpNR7Dr1Z827hQcvgNOyan~mh1Jl21Jz~4Mry0N7qrGQe4rOalMbAJ7KxzbhNPGJz1PMhouQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163575&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hitherto unidentified potential identity elements. Positions having high AEV but low NPDs could harbour hitherto unidentified identity elements. Grey or black highlights the most likely positions of such elements. For positions highlighted as black, we also propose the corresponding identity: bold corresponding to yeast, while italic to <em>E. coli</em>. Middle grey highlights ‘core region’ positions having high AEV and low NPD in all three kingdoms. Light grey highlights the highest AEV base pair in the Archaea kingdom.</p></div></div><div content-id="DSS008F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00803.jpeg?Expires=2147483647&Signature=COr2zC1Gd7aE-3l~1jgZ64KsMpcXd2H9Tp2M4ge6R4AfJVm~lmfzXzJdtENP~uD7xJIq91AVnUt0uiRCPG9WFv0ej8cyQGYly17Km~UsVdoJjC6I~~OUtCneRQoxex0XgK3-7f1zk2JX5Kxf9nCegKcSW-TxdoFeq8IAJ3cS0X~-R--mbIscGqD2C0zIh6lZPCcvJm2YPVeDdGWhSxxcavGgG-EK8sP1p03ahvT4DFPS-IGbYWrN4L53COnJIfG~ON87qPhfL8mVuxHy0dZ6ezbG1DYLcPv7r28ZCIJSLj-e47Jq88KDqUgY8MG-SK9mMLzngrvoeZ5FUvhFy~pWkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hitherto unidentified potential identity elements. Positions having high AEV but low NPDs could harbour hitherto unidentified identity elements. Grey or black highlights the most likely positions of such elements. For positions highlighted as black, we also propose the corresponding identity: bold corresponding to yeast, while italic to E. coli. Middle grey highlights ‘core region’ positions having high AEV and low NPD in all three kingdoms. Light grey highlights the highest AEV base pair in the Archaea kingdom." path-from-xml="dss00803.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163575/dss00803.jpeg" path-from-xml="dss00803.jpeg" target="_blank">View large</a><a section="7163575" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00803.jpeg?Expires=2147483647&Signature=N82JwqMYt2xD4DiMGkXknaB9k0QjfjJnLail0-HQfpn7P7NWr65JMm4lqD4rCuo2890tOg56f0O2u0L09ZElNdgPQWrkM0T5zslEBLA01qJncunmhVN5qVe-YXyaMfoGFLtzLBZDUOLYCJAPAAjasMyLJKiLrSvFf~VODFECas5xAJ5uMtQdDiyOR3d7augbE4hmEWoIhvXJSiLZ1kq9jk1y0vnZrV3RUErbLfMVZgumWYwcpBn1dKVCDoJYaeWpQwLXu4Go5~Aol0DpNR7Dr1Z827hQcvgNOyan~mh1Jl21Jz~4Mry0N7qrGQe4rOalMbAJ7KxzbhNPGJz1PMhouQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163575&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00803.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hitherto unidentified potential identity elements. Positions having high AEV but low NPDs could harbour hitherto unidentified identity elements. Grey or black highlights the most likely positions of such elements. For positions highlighted as black, we also propose the corresponding identity: bold corresponding to yeast, while italic to <em>E. coli</em>. Middle grey highlights ‘core region’ positions having high AEV and low NPD in all three kingdoms. Light grey highlights the highest AEV base pair in the Archaea kingdom.</p></div></div><p>Similar to the bacterial set, the AEV of position 34 is lower than those of positions 35 and 36. However, in this case, position 34 has a much lower, only average AEV (green zone), while in bacteria, it belonged to the highest category (red zone).</p><p>Positions having the lowest AEVs, just like in the case of the bacterial data set, are located at conserved tRNA architecture defining positions. A noteworthy difference compared with the bacterial data is that in Archaea, the 1:72 position pair has only average AEVs (Fig. <a reveal-id="DSS008F2" data-open="DSS008F2" class="link link-reveal link-table xref-fig">2</a>A and C). This appears to be due to the fact that in ∼90% of the Archaea tRNA sequences, there is a G1:C72 pair at this position. Based on the already published sequence analysis,<sup><a reveal-id="DSS008C50" data-open="DSS008C50" class="link link-ref link-reveal xref-bibr">50</a></sup> the only few exceptions are detected for the initiator tRNA<sup>Met</sup> and for tRNA<sup>Gln</sup> and tRNA<sup>Tyr</sup>. The C1:G72 pair of tRNA<sup>Tyr</sup> has been experimentally verified to be a genuine identity element.<sup><a reveal-id="DSS008C57" data-open="DSS008C57" class="link link-ref link-reveal xref-bibr">57</a>,<a reveal-id="DSS008C58" data-open="DSS008C58" class="link link-ref link-reveal xref-bibr">58</a></sup></p>                            <h3 scrollto-destination=7163578 id="7163578" class="section-title" data-legacy-id=s3e>3.5. Potential hidden identity elements</h3>
<p>Positions having higher than average AEVs but containing no published determinants that would have been used in our second filtering step might harbour hitherto unidentified determinants. The most likely hidden identity elements are illustrated in Fig. <a reveal-id="DSS008F3" data-open="DSS008F3" class="link link-reveal link-table xref-fig">3</a>. We searched the literature for any identified determinants at such positions reported for species other than coli-like bacteria or yeast. Such cases were already mentioned above for the 30:40 and 31:39 base pairs that are identified human determinants.<sup><a reveal-id="DSS008C44" data-open="DSS008C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Therefore, we analysed these two base pairs by checking our data set for each identity set. As a simplest scenario, we searched for base pairs that, in any one of the 20 identity sets, differ from all other 19 sets. In the bacterial set, there was a single such case, a normal Watson–Crick base pair, while for the eukaryotic set, all such pairs were non-Watson–Crick base pairs.</p>                            <h4 scrollto-destination=7163580 id="7163580" class="section-title" data-legacy-id=s3e1>3.5.1. Escherichia coli tRNA<sup>Trp</sup> T31:A39</h4>
<p>This base pair is present in the tRNA<sup>Trp</sup> sequences in all but a very few species that provided tRNA<sup>Trp</sup> sequences to our filtered coli-like data set. In four coli-like species, only a single tRNA<sup>Ser</sup>, while in two species, only a single tRNA<sup>Gln</sup>, has the same base pair. We note that only the anticodon loop, the discriminator base and the acceptor stem were experimentally tested for coli tRNA<sup>Trp</sup> determinants,<sup><a reveal-id="DSS008C59  DSS008C61" data-open="DSS008C59  DSS008C61" class="link link-ref link-reveal xref-bibr">59–61</a></sup> thus the T31:A39 pair might be a hitherto undetected identity element.</p>                            <h4 scrollto-destination=7163582 id="7163582" class="section-title" data-legacy-id=s3e2>3.5.2. Yeast Met T31:T39</h4>
<p>This unconventional base pair is characteristic to eukaryotic elongator tRNA<sup>Met</sup>. Both eukaryotic as well as coli <em>initiator</em> tRNA<sup>Met</sup> and the coli elongator tRNA<sup>Met</sup> contain a normal G31:C39 pair at these positions. In the coli initiator tRNA, the normal G31:C39 base pair is required for binding to the P site on the ribosome and for protein synthesis initiation, while in the coli elongator tRNA<sup>Met</sup>, the same pair is required for proper acylation by the corresponding AARS enzyme. Eukaryotic initiator tRNA<sup>Met</sup> from a wide source were shown to be good substrates of the coli Met-RS enzyme, while the eukaryotic elongator tRNA<sup>Met</sup> from the same wide source were all poor substrates. When the original T31:T39 pair in the eukaryotic tRNA<sup>Met</sup> was replaced with a G31:C39 pair, it became a good substrate for the coli enzyme, and symmetrically, when the G31:C39 pair was replaced with a T31:T39 pair in the coli elongator tRNA<sup>Met</sup>, it became a good substrate for the eukaryotic enzyme. Thus, the kingdom-specific base pair is a determinant in both kingdoms. Moreover, it was also shown that in the elongator tRNA, it is not the identity of the bases that matter, but the fact whether these form a Watson–Crick base pair or not. If they do, the corresponding tRNA<sup>Met</sup> is a good substrate of the bacterial enzyme and a poor substrate of the eukaryotic enzyme. If they do not, the corresponding tRNA<sup>Met</sup> is a good substrate for the eukaryotic enzyme and a poor substrate for the bacterial enzyme. Thus, a Watson–Crick base pair at these positions or the lack of it affects the structure and/or malleability of the anticodon loop, which has an important role in the proper interaction with the AARS enzyme.</p><p>It has also been shown that replacing the G31:C39 base pair with a T31:T39 pair in the yeast initiatior tRNA renders it being able to participate in the elongation phase.</p><p>Note that this base pair has not been defined as a determinant <em>per se</em>, as the studies compared either isospecific elongator tRNAs from two different kingdoms or elongator vs. initiator tRNAs rather than two different elongator tRNA identities from the same species.<sup><a reveal-id="DSS008C62  DSS008C64" data-open="DSS008C62  DSS008C64" class="link link-ref link-reveal xref-bibr">62–64</a></sup></p>                            <h4 scrollto-destination=7163586 id="7163586" class="section-title" data-legacy-id=s3e3>3.5.3. Yeast Ile T30:G40</h4>
<p>In the case of yeast tRNA<sup>Ile</sup>, only the three anticodon positions were studied as identity elements,<sup><a reveal-id="DSS008C65" data-open="DSS008C65" class="link link-ref link-reveal xref-bibr">65</a></sup> and the potential role of this nearby non-Watson–Crick base pair has not been tested. In our data set, this base pair is present in the majority of eukaryotic species from Arabidopsis through Drosophila to Human for several Ile isoacceptors, suggesting that this base pair might have an identity defining role.</p>                            <h4 scrollto-destination=7163588 id="7163588" class="section-title" data-legacy-id=s3e4>3.5.4. Yeast Asp G30:T40</h4>
<p>At the same positions as for tRNA<sup>Ile</sup>, a different non-Watson-Crick base pair is present in yeast tRNA<sup>Asp</sup> sequences. This base pair is not conserved in eukaryotic species. In our data set, it is present only in yeast and <em>Caenorhabditis elegans</em>. As high AEVs suggest that the G30:T40 pair might be an identity element in yeast (and also in <em>C. elegans</em>), we checked the PDB for other yeast tRNA/AARS complex structures. Unfortunately, besides the Asp,<sup><a reveal-id="DSS008C66" data-open="DSS008C66" class="link link-ref link-reveal xref-bibr">66</a></sup> only two other structures are available. One is for tRNATyr (PDB: 2DLC), which is incomplete, and another one for tRNAArg.<sup><a reveal-id="DSS008C67" data-open="DSS008C67" class="link link-ref link-reveal xref-bibr">67</a></sup> Both have a normal G30:C40 base pair. Note that ArgRS and TyrRS belong to Class I, while AspRS belongs to Class II. As enzymes from the two classes approach the tRNA from opposite sides, we did not expect any similarities in the enzyme–tRNA interactions themselves.</p><p>As could be expected, the interactions are not homologous. In the case of tRNA<sup>Asp</sup>, the ribose-phosphate portion of G30 is in 2.9–3.3 Å (there are two complexes in the unit cell) distance from the Lys88 side chain of the enzyme, which suggests a salt bridge between the tRNA and the enzyme at this position. The equivalent residue of the AspRS Lys88 in the ArgRS is Lys78, which does not interact with the tRNA. On the other hand in the tRNA<sup>Arg</sup> structure, the phosphate of C40 appears to form a H-bond with Ser440 of the enzyme, which is in 2.6 Å distance. The incomplete tRNA<sup>Tyr</sup> structure either does not have an interaction with the enzyme at this position or it is not resolved in the model. It is plausible that a G:T pair is disrupted more readily than a G:C pair. It appears that such local disruption of a G:T pair is required for forming a stable salt bridge with the enzyme in the case of tRNA<sup>Asp</sup>. On the other hand, tRNA<sup>Arg</sup> has a different stabilizing interaction, which does not necessitate such a local perturbation.</p><p>We also checked whether a different base pair exists at these positions in bacterial tRNA<sup>Asp</sup> and if so, whether it affects the recognition mode by their respective AARS enzymes. In the coli tRNA homologue, there is a traditional G30:C40 pair and fortunately there is a tRNA:AARS complex structure for bacterial tRNA<sup>Asp</sup> in the PDB,<sup><a reveal-id="DSS008C68" data-open="DSS008C68" class="link link-ref link-reveal xref-bibr">68</a></sup> so we could compare the two homologous interactions from the two different kingdoms. Differences are shown in Fig. <a reveal-id="DSS008F4" data-open="DSS008F4" class="link link-reveal link-table xref-fig">4</a>. As already mentioned, in the yeast system, the ribose-phosphate portion of G30 of the tRNA forms a salt bridge with the Lys88 side chain of the enzyme. A Needleman–Wunsch alignment of the coli and yeast synthetase sequences identified Lys58 as the coli equivalent of Lys88. This side chain is in 10.3 Å distance from the G30 phosphate moiety therefore no salt bridge is formed. This comparison clearly shows that identical tRNA positions are differently recognized by the two iso-functional AARS enzymes. </p>                        <a id="7163592" scrollto-destination="7163592"></a>
<div id="DSS008F4" data-id="DSS008F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00804.jpeg?Expires=2147483647&Signature=qIMH5dHny1yb6f-xmsHGvG9S5-zo5vZKkDBFlM4qJhNyp2YHXIOG-uBNPMHtHdzEanC3wOGZATeJCdky8vqOqs5jcae~u-Lhq9fvnFp5eWBplyitBsQE-hM-Dc8tgP0VqrwWwVmrEXqEi01H59InkOBdVm--7Vw7KqnjXFkmSVtHt5yDm3lKkJXecgIRqwSIJY-0cfm789zf~ucwaJuProZ92sNOI~NgkAP0Ronb~wlOLiLE54iG1eE20hp476oikLmfDFwlwawh0wqAfM5iVAe0cTV9QwTr~gqhAKMKr~MnLyaZYXuSBBBoyT-sRTF6FEOMcWaLLMWBHiJjetsvdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of tRNAAsp/AspRS structures from E. coli and yeast. (A) Structure of the E. coli tRNAAsp/AspRS complex68; PDB: 1IL2. (B) Structure of the yeast tRNAAsp/AspRS complex66; PDB: 1ASY. Colour coding: AARS, ‘wheat’; tRNA, pale cyan; Lys58 (E. coli, panel A) and Lys88 (yeast, panel B) and the G30:C40 (E. coli, panel A) and G30:U40 (yeast, panel B) base pairs are green. Other bases illustrated as sticks are already published identity elements. See Section 3 for details. This figure can be viewed in colour online." path-from-xml="dss00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163592/dss00804.jpeg" path-from-xml="dss00804.jpeg" target="_blank">View large</a><a section="7163592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00804.jpeg?Expires=2147483647&Signature=aou8V39apb7WJfSJF7ZwGlRNLxU1j4IDWrx9JQETlVlbetkqTz6NXzxwJXt0SQAoRBlJ4-Uc50LwFr9~6xq1SBbdCun-MiuPyN6aewbd~vC7G4k3Gol0GqbpJnqhPaqMUh72lq3mlHY66Y0PT3bnvqrM8Xe8fleQ3Mb-kCYMRuBfG7TkkhTSSS4wH3S0hYM~1TFpBZPrH5BBObltrBeOZ0JAqMV-YROgVA6BUVF1YQqyVKw9B59fGNDNyyRSssQ-64BCWERNoztl5Mf7jv0x9WU0~WE3fqOgMUq9qAiPYU-kVHrGWXwz46NxBQmKUkRBaVrxiaEiaWSk82wzkP9log__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163592&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of tRNA<sup>Asp</sup>/AspRS structures from <em>E. coli</em> and yeast. (A) Structure of the <em>E. coli</em> tRNA<sup>Asp</sup>/AspRS complex<sup><a reveal-id="DSS008C68" data-open="DSS008C68" class="link link-ref link-reveal xref-bibr">68</a></sup>; PDB: 1IL2. (B) Structure of the yeast tRNA<sup>Asp</sup>/AspRS complex<sup><a reveal-id="DSS008C66" data-open="DSS008C66" class="link link-ref link-reveal xref-bibr">66</a></sup>; PDB: 1ASY. Colour coding: AARS, ‘wheat’; tRNA, pale cyan; Lys58 (<em>E. coli</em>, panel A) and Lys88 (yeast, panel B) and the G30:C40 (<em>E. coli</em>, panel A) and G30:U40 (yeast, panel B) base pairs are green. Other bases illustrated as sticks are already published identity elements. See Section 3 for details. This figure can be viewed in colour online.</p></div></div><div content-id="DSS008F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/m_dss00804.jpeg?Expires=2147483647&Signature=qIMH5dHny1yb6f-xmsHGvG9S5-zo5vZKkDBFlM4qJhNyp2YHXIOG-uBNPMHtHdzEanC3wOGZATeJCdky8vqOqs5jcae~u-Lhq9fvnFp5eWBplyitBsQE-hM-Dc8tgP0VqrwWwVmrEXqEi01H59InkOBdVm--7Vw7KqnjXFkmSVtHt5yDm3lKkJXecgIRqwSIJY-0cfm789zf~ucwaJuProZ92sNOI~NgkAP0Ronb~wlOLiLE54iG1eE20hp476oikLmfDFwlwawh0wqAfM5iVAe0cTV9QwTr~gqhAKMKr~MnLyaZYXuSBBBoyT-sRTF6FEOMcWaLLMWBHiJjetsvdw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of tRNAAsp/AspRS structures from E. coli and yeast. (A) Structure of the E. coli tRNAAsp/AspRS complex68; PDB: 1IL2. (B) Structure of the yeast tRNAAsp/AspRS complex66; PDB: 1ASY. Colour coding: AARS, ‘wheat’; tRNA, pale cyan; Lys58 (E. coli, panel A) and Lys88 (yeast, panel B) and the G30:C40 (E. coli, panel A) and G30:U40 (yeast, panel B) base pairs are green. Other bases illustrated as sticks are already published identity elements. See Section 3 for details. This figure can be viewed in colour online." path-from-xml="dss00804.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7163592/dss00804.jpeg" path-from-xml="dss00804.jpeg" target="_blank">View large</a><a section="7163592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss00804.jpeg?Expires=2147483647&Signature=aou8V39apb7WJfSJF7ZwGlRNLxU1j4IDWrx9JQETlVlbetkqTz6NXzxwJXt0SQAoRBlJ4-Uc50LwFr9~6xq1SBbdCun-MiuPyN6aewbd~vC7G4k3Gol0GqbpJnqhPaqMUh72lq3mlHY66Y0PT3bnvqrM8Xe8fleQ3Mb-kCYMRuBfG7TkkhTSSS4wH3S0hYM~1TFpBZPrH5BBObltrBeOZ0JAqMV-YROgVA6BUVF1YQqyVKw9B59fGNDNyyRSssQ-64BCWERNoztl5Mf7jv0x9WU0~WE3fqOgMUq9qAiPYU-kVHrGWXwz46NxBQmKUkRBaVrxiaEiaWSk82wzkP9log__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7163592&ar=475122&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00804.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of tRNA<sup>Asp</sup>/AspRS structures from <em>E. coli</em> and yeast. (A) Structure of the <em>E. coli</em> tRNA<sup>Asp</sup>/AspRS complex<sup><a reveal-id="DSS008C68" data-open="DSS008C68" class="link link-ref link-reveal xref-bibr">68</a></sup>; PDB: 1IL2. (B) Structure of the yeast tRNA<sup>Asp</sup>/AspRS complex<sup><a reveal-id="DSS008C66" data-open="DSS008C66" class="link link-ref link-reveal xref-bibr">66</a></sup>; PDB: 1ASY. Colour coding: AARS, ‘wheat’; tRNA, pale cyan; Lys58 (<em>E. coli</em>, panel A) and Lys88 (yeast, panel B) and the G30:C40 (<em>E. coli</em>, panel A) and G30:U40 (yeast, panel B) base pairs are green. Other bases illustrated as sticks are already published identity elements. See Section 3 for details. This figure can be viewed in colour online.</p></div></div><p>Furthermore, the observed direct interaction of the G30:T40 pair with the AARS enzyme suggests that it might be an identity element in yeast.</p>                            <h4 scrollto-destination=7163594 id="7163594" class="section-title" data-legacy-id=s3e5>3.5.5. Core region</h4>
<p>In <em>E. coli</em>, it has been shown that the core region, formed by the 15:48 pair and by [13:20]:46, has identity functions. The G15:G48 was shown to be a Cys identity element.<sup><a reveal-id="DSS008C26" data-open="DSS008C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The core region was shown to be important for tRNA<sup>Pro</sup> identity<sup><a reveal-id="DSS008C69" data-open="DSS008C69" class="link link-ref link-reveal xref-bibr">69</a></sup> and for discriminating tRNA<sup>Glu</sup> from tRNA<sup>Asp</sup>.<sup><a reveal-id="DSS008C30" data-open="DSS008C30" class="link link-ref link-reveal xref-bibr">30</a></sup> While in the bacterial set, the 15:48 pair has medium level (green zone) AEV, positions 22 and 46 have high (yellow zone) and position 13 very high (red zone) AEV. The eukaryote set also shows high AEVs at these positions (all in the yellow zone), suggesting that the core region might have similar identity roles in the eukaryotic kingdom as well.<sup><a reveal-id="DSS008C70" data-open="DSS008C70" class="link link-ref link-reveal xref-bibr">70</a></sup></p>                            <h3 scrollto-destination=7163596 id="7163596" class="section-title" data-legacy-id=s3f>3.6. Conclusions</h3>
<p>Deciphering identity elements of tRNAs has been one of the most interesting problems of molecular biology that has a long and successful history.<sup><a reveal-id="DSS008C2  DSS008C5" data-open="DSS008C2  DSS008C5" class="link link-ref link-reveal xref-bibr">2–5</a></sup> It is unquestionable that only properly designed mutations combined with <em>in vitro</em> and/or <em>in vivo</em> experiments can identify such elements beyond any doubts. However, the large number of even the ‘reasonable’ mutations and the astronomical number of their combinations renders these laborious studies rather challenging. As a consequence, the existing collection of already determined identity element set cannot be considered complete, and several bioinformatics studies aimed to predict hitherto hidden such elements.<sup><a reveal-id="DSS008C6" data-open="DSS008C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS008C13  DSS008C17" data-open="DSS008C13  DSS008C17" class="link link-ref link-reveal xref-bibr">13–17</a></sup> Most of these studies applied classical sequence analysing tools and searched for conserved sequence features.</p><p>The ECP analysis, on the other hand, is based on strictly absent elements and provides a simple but meaningful measure of pairwise mathematical distances of functionally related sequence sets.<sup><a reveal-id="DSS008C18" data-open="DSS008C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>The reliability of bioinformatic studies is strongly related to the number of input sequences that are compared. Individual species have only a few tRNA sequences for each identity set. Comparing tRNA sequences from a pool of related species could improve the signal/noise ratio of the analysis, but it can be justified only if the comparison is functionally relevant, i.e. tRNA from one species would be properly charged by the corresponding AARS from the other species.</p><p>We produced starting data sets, in which sequences grouped for the 20 identity from bacteria were filtered for the presence of major coli identity elements while eukaryotic sequences for the presence of yeast identity elements. We argue that common presence of such identity elements in the starting data set favours interspecies compatibility. If so, isofunctional tRNAs from different species sharing already published identity elements should also share their yet undiscovered determinants as well. It means that such an analysis should identify elements that have not been recognized yet and in the same time identify those too that had already been published, but not included in the filtering set. If hitherto unknown elements are detected, they should be considered potential identity elements only in bacterial or eukaryotic species that also use the coli or the yeast identity element sets, respectively, that were applied for filtering. Thus, this analysis could clearly miss elements that are not conserved across species.</p><p>We demonstrated that positional AEVs measure sequence feature distances of functional groups and correlate with the number of already identified determinants (Table <a reveal-id="DSS008TB1" data-open="DSS008TB1" class="link link-reveal link-table xref-fig">1</a>). Based on the above arguments, we suggested that positions having high AEV but few or no identified determinants predict locations of hidden identity elements. We listed the most characteristic such positions and for one of these suggested a structural rationale for being identity element.</p><p>We also showed that omitting the second filtration in the case of both Bacteria and Eukarya still preserves the characteristic pattern of high AEV positions, which overlaps with the positions of the most important identity elements such as the anticodon bases and the discriminator base.</p><p>In this study, we have demonstrated that the ECP algorithm is capable of assessing the level of discriminating power of positions in separating functionally different sequence sets. As genomic sequencing continues at an increasing rate, our ECP analysis can be performed over and over on ever-increasing filtered databases.</p><p>We believe that when enough input data are available, we can analyse the discriminating power not only of individual positions but combinations of positions as well.</p><p>Nevertheless, only carefully designed mutations and laboratory experiments can assess the predicting potentials of the ECP algorithm to identify hidden determinants.</p>                            <h2 scrollto-destination=7163606 id="7163606" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to Dr Catherine Florentz for critical reading of the manuscript and for her highly valuable suggestions. We also thank Márk Szenes for his help in defining the mathematical formalism of the AEV calculating algorithm and for the useful discussions about the statistical analyses.</p>                            <h2 scrollto-destination=7163608 id="7163608" class="backsection-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7163610 id="7163610" class="backsection-title" data-legacy-id=s5>Funding</h2>
<p>The study was supported by the Hungarian Scientific Research Fund (OTKA) K68408, NK81950 and NK100769 as well as by the European Union and European Social Fund (TÁMOP) 4.2.1./B-09/KMR-2010-0003. G.P. is supported by the János Bolyai Research Fellowship.</p>                            <h2 scrollto-destination=7163612 id="7163612" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS008C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cavarelli</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Moras</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Recognition of tRNAs by aminoacyl-tRNA synthetases</div>, <div class="source">FASEB J.</div> , <div class="year">1993</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">79</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20tRNAs%20by%20aminoacyl-tRNA%20synthetases&amp;author=J.%20Cavarelli&amp;author=D.%20Moras&amp;publication_year=1993&amp;journal=FASEB%20J.&amp;volume=7&amp;pages=79-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8422978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McClain</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Rules that govern tRNA identity in protein synthesis</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">234</div> (pg. <div class="fpage">257</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rules%20that%20govern%20tRNA%20identity%20in%20protein%20synthesis&amp;author=W.H.%20McClain&amp;publication_year=1993&amp;journal=J.%20Mol.%20Biol.&amp;volume=234&amp;pages=257-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1993.1582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1993.1582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1993.1582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8230212" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sissler</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Florentz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Universal rules and idiosyncratic features in tRNA identity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">5017</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20rules%20and%20idiosyncratic%20features%20in%20tRNA%20identity&amp;author=R.%20Giege&amp;author=M.%20Sissler&amp;author=C.%20Florentz&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=5017-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.22.5017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.22.5017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.22.5017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9801296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Aminoacyl-tRNA synthesis</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">617</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoacyl-tRNA%20synthesis&amp;author=M.%20Ibba&amp;author=D.%20Soll&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=69&amp;pages=617-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.69.1.617" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.69.1.617" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.69.1.617"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10966471" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Toward a more complete view of tRNA biology</div>, <div class="source">Nat. Struct. Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1007</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20more%20complete%20view%20of%20tRNA%20biology&amp;author=R.%20Giege&amp;publication_year=2008&amp;journal=Nat.%20Struct.%20Mol.%20Biol.&amp;volume=15&amp;pages=1007-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1498" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1498" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1498"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18836497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laslett</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Canback</div> <div class="given-names">B.</div></div>. </div><div class="article-title">ARAGORN, a program to detect tRNA genes and tmRNA genes in nucleotide sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">11</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ARAGORN%2C%20a%20program%20to%20detect%20tRNA%20genes%20and%20tmRNA%20genes%20in%20nucleotide%20sequences&amp;author=D.%20Laslett&amp;author=B.%20Canback&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chan</div> <div class="given-names">P.P.</div></div>,  <div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">GtRNAdb: a database of transfer RNA genes detected in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D93</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GtRNAdb%3A%20a%20database%20of%20transfer%20RNA%20genes%20detected%20in%20genomic%20sequence&amp;author=P.P.%20Chan&amp;author=T.M.%20Lowe&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D93-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Juhling</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Morl</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hartmann</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Sprinzl</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stadler</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Putz</div> <div class="given-names">J.</div></div>. </div><div class="article-title">tRNAdb 2009: compilation of tRNA sequences and tRNA genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D159</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAdb%202009%3A%20compilation%20of%20tRNA%20sequences%20and%20tRNA%20genes&amp;author=F.%20Juhling&amp;author=M.%20Morl&amp;author=R.K.%20Hartmann&amp;author=M.%20Sprinzl&amp;author=P.F.%20Stadler&amp;author=J.%20Putz&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D159-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18957446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abe</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Sugahara</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">tRNADB-CE 2011: tRNA gene database curated manually by experts</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">D210</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNADB-CE%202011%3A%20tRNA%20gene%20database%20curated%20manually%20by%20experts&amp;author=T.%20Abe&amp;author=T.%20Ikemura&amp;author=J.%20Sugahara&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=D210-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21071414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Atilgan</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nicholas</div> <div class="given-names">H.B.</div>Jr.</div>,  <div class="name"><div class="surname">McClain</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">A statistical method for correlating tRNA sequence with amino acid specificity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1986</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">375</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20method%20for%20correlating%20tRNA%20sequence%20with%20amino%20acid%20specificity&amp;author=T.%20Atilgan&amp;author=H.B.%20Nicholas&amp;author=W.H.%20McClain&amp;publication_year=1986&amp;journal=Nucleic%20Acids%20Res.&amp;volume=14&amp;pages=375-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/14.1.375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F14.1.375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F14.1.375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3633085" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nicholas</div> <div class="given-names">H.B.</div>Jr.</div>,  <div class="name"><div class="surname">McClain</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">Searching tRNA sequences for relatedness to aminoacyl-tRNA synthetase families</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">482</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Searching%20tRNA%20sequences%20for%20relatedness%20to%20aminoacyl-tRNA%20synthetase%20families&amp;author=H.B.%20Nicholas&amp;author=W.H.%20McClain&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=40&amp;pages=482-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00166616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00166616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00166616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7783223" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">RNA sequence analysis using covariance models</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">2079</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequence%20analysis%20using%20covariance%20models&amp;author=S.R.%20Eddy&amp;author=R.%20Durbin&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=2079-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.11.2079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.11.2079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.11.2079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8029015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ardell</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">S.G.</div></div>. </div><div class="article-title">TFAM detects co-evolution of tRNA identity rules with lateral transfer of histidyl-tRNA synthetase</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">893</div>-<div class="lpage">904</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TFAM%20detects%20co-evolution%20of%20tRNA%20identity%20rules%20with%20lateral%20transfer%20of%20histidyl-tRNA%20synthetase&amp;author=D.H.%20Ardell&amp;author=S.G.%20Andersson&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=893-904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16473847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Freyhult</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moulton</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Ardell</div> <div class="given-names">D.H.</div></div>. </div><div class="article-title">Visualizing bacterial tRNA identity determinants and antideterminants using function logos and inverse function logos</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">905</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Visualizing%20bacterial%20tRNA%20identity%20determinants%20and%20antideterminants%20using%20function%20logos%20and%20inverse%20function%20logos&amp;author=E.%20Freyhult&amp;author=V.%20Moulton&amp;author=D.H.%20Ardell&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=905-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16473848" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Freyhult</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Cui</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Ardell</div> <div class="given-names">D.H.</div></div>. </div><div class="article-title">New computational methods reveal tRNA identity element divergence between Proteobacteria and Cyanobacteria</div>, <div class="source">Biochimie</div> , <div class="year">2007</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1276</div>-<div class="lpage">88</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20computational%20methods%20reveal%20tRNA%20identity%20element%20divergence%20between%20Proteobacteria%20and%20Cyanobacteria&amp;author=E.%20Freyhult&amp;author=Y.%20Cui&amp;author=O.%20Nilsson&amp;author=D.H.%20Ardell&amp;publication_year=2007&amp;journal=Biochimie&amp;volume=89&amp;pages=1276-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2007.07.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2007.07.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biochi.2007.07.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17889982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ardell</div> <div class="given-names">D.H.</div></div>. </div><div class="article-title">Computational analysis of tRNA identity</div>, <div class="source">FEBS Lett.</div> , <div class="year">2010</div>, vol. <div class="volume">584</div> (pg. <div class="fpage">325</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20analysis%20of%20tRNA%20identity&amp;author=D.H.%20Ardell&amp;publication_year=2010&amp;journal=FEBS%20Lett.&amp;volume=584&amp;pages=325-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2009.11.084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2009.11.084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2009.11.084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19944694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jako</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ittzes</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Szenes</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kun</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Szathmary</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pal</div> <div class="given-names">G.</div></div>. </div><div class="article-title">In silico detection of tRNA sequence features characteristic to aminoacyl-tRNA synthetase class membership</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">5593</div>-<div class="lpage">609</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20detection%20of%20tRNA%20sequence%20features%20characteristic%20to%20aminoacyl-tRNA%20synthetase%20class%20membership&amp;author=E.%20Jako&amp;author=P.%20Ittzes&amp;author=A.%20Szenes&amp;author=A.%20Kun&amp;author=E.%20Szathmary&amp;author=G.%20Pal&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=5593-609" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17704131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujishima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugahara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kikuta</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Tri-split tRNA is a transfer RNA made from 3 transcripts that provides insight into the evolution of fragmented tRNAs in archaea</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">2683</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tri-split%20tRNA%20is%20a%20transfer%20RNA%20made%20from%203%20transcripts%20that%20provides%20insight%20into%20the%20evolution%20of%20fragmented%20tRNAs%20in%20archaea&amp;author=K.%20Fujishima&amp;author=J.%20Sugahara&amp;author=K.%20Kikuta&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=2683-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0808246106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0808246106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0808246106"> </span></div> </div></div></div></div></div><div content-id="DSS008C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugahara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kikuta</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujishima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yachie</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Comprehensive analysis of archaeal tRNA genes reveals rapid increase of tRNA introns in the order thermoproteales</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">2709</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20analysis%20of%20archaeal%20tRNA%20genes%20reveals%20rapid%20increase%20of%20tRNA%20introns%20in%20the%20order%20thermoproteales&amp;author=J.%20Sugahara&amp;author=K.%20Kikuta&amp;author=K.%20Fujishima&amp;author=N.%20Yachie&amp;author=M.%20Tomita&amp;author=A.%20Kanai&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=2709-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn216" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn216" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn216"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18832079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maruyama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sugahara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nozaki</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Permuted tRNA genes in the nuclear and nucleomorph genomes of photosynthetic eukaryotes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1070</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Permuted%20tRNA%20genes%20in%20the%20nuclear%20and%20nucleomorph%20genomes%20of%20photosynthetic%20eukaryotes&amp;author=S.%20Maruyama&amp;author=J.%20Sugahara&amp;author=A.%20Kanai&amp;author=H.%20Nozaki&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=1070-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20022888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chan</div> <div class="given-names">P.P.</div></div>,  <div class="name"><div class="surname">Cozen</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Discovery of permuted and recently split transfer RNAs in Archaea</div>, <div class="source">Genome Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">R38</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20permuted%20and%20recently%20split%20transfer%20RNAs%20in%20Archaea&amp;author=P.P.%20Chan&amp;author=A.E.%20Cozen&amp;author=T.M.%20Lowe&amp;publication_year=2011&amp;journal=Genome%20Biol.&amp;volume=12&amp;pages=R38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-4-r38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-4-r38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-4-r38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21489296" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujishima</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sugahara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Sequence evidence in the archaeal genomes that tRNAs emerged through the combination of ancestral genes as 5′ and 3′ tRNA halves</div>, <div class="source">PLoS One</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> pg. <div class="fpage">e1622</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20evidence%20in%20the%20archaeal%20genomes%20that%20tRNAs%20emerged%20through%20the%20combination%20of%20ancestral%20genes%20as%205%E2%80%B2%20and%203%E2%80%B2%20tRNA%20halves&amp;author=K.%20Fujishima&amp;author=J.%20Sugahara&amp;author=M.%20Tomita&amp;author=A.%20Kanai&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e1622" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18286179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wilm</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mainz</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Steger</div> <div class="given-names">G.</div></div>. </div><div class="article-title">An enhanced RNA alignment benchmark for sequence alignment programs</div>, <div class="source">Algorithms Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">1</div> pg. <div class="fpage">19</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20enhanced%20RNA%20alignment%20benchmark%20for%20sequence%20alignment%20programs&amp;author=A.%20Wilm&amp;author=I.%20Mainz&amp;author=G.%20Steger&amp;publication_year=2006&amp;journal=Algorithms%20Mol.%20Biol.&amp;volume=1&amp;pages=19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1748-7188-1-19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1748-7188-1-19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1748-7188-1-19"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17062125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marck</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Grosjean</div> <div class="given-names">H.</div></div>. </div><div class="article-title">tRNomics: analysis of tRNA genes from 50 genomes of Eukarya, Archaea, and Bacteria reveals anticodon-sparing strategies and domain-specific features</div>, <div class="source">RNA</div> , <div class="year">2002</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1189</div>-<div class="lpage">232</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNomics%3A%20analysis%20of%20tRNA%20genes%20from%2050%20genomes%20of%20Eukarya%2C%20Archaea%2C%20and%20Bacteria%20reveals%20anticodon-sparing%20strategies%20and%20domain-specific%20features&amp;author=C.%20Marck&amp;author=H.%20Grosjean&amp;publication_year=2002&amp;journal=RNA&amp;volume=8&amp;pages=1189-232" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1355838202022021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1355838202022021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1355838202022021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12403461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">Y.M.</div></div>,  <div class="name"><div class="surname">Motegi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Hamann</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Shiba</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Conservation of a tRNA core for aminoacylation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4743</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20a%20tRNA%20core%20for%20aminoacylation&amp;author=Y.M.%20Hou&amp;author=H.%20Motegi&amp;author=R.S.%20Lipman&amp;author=C.S.%20Hamann&amp;author=K.%20Shiba&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4743-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.24.4743" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.24.4743" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.24.4743"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10572174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nureki</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Niimi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Muramatsu</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Molecular recognition of the identity-determinant set of isoleucine transfer RNA from <em>Escherichia coli</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">236</div> (pg. <div class="fpage">710</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20recognition%20of%20the%20identity-determinant%20set%20of%20isoleucine%20transfer%20RNA%20from%20Escherichia%20coli&amp;author=O.%20Nureki&amp;author=T.%20Niimi&amp;author=T.%20Muramatsu&amp;publication_year=1994&amp;journal=J.%20Mol.%20Biol.&amp;volume=236&amp;pages=710-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1994.1184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1994.1184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1994.1184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8114089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sylvers</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Rogers</div> <div class="given-names">K.C.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohtsuka</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A 2-thiouridine derivative in tRNAGlu is a positive determinant for aminoacylation by Escherichia coli glutamyl-tRNA synthetase</div>, <div class="source">Biochemistry</div> , <div class="year">1993</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">3836</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%202-thiouridine%20derivative%20in%20tRNAGlu%20is%20a%20positive%20determinant%20for%20aminoacylation%20by%20Escherichia%20coli%20glutamyl-tRNA%20synthetase&amp;author=L.A.%20Sylvers&amp;author=K.C.%20Rogers&amp;author=M.%20Shimizu&amp;author=E.%20Ohtsuka&amp;author=D.%20Soll&amp;publication_year=1993&amp;journal=Biochemistry&amp;volume=32&amp;pages=3836-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00066a002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00066a002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00066a002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8385989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sekine</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nureki</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Sakamoto</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Major identity determinants in the ‘augmented D helix’ of tRNA(Glu) from Escherichia coli</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">256</div> (pg. <div class="fpage">685</div>-<div class="lpage">700</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Major%20identity%20determinants%20in%20the%20%E2%80%98augmented%20D%20helix%E2%80%99%20of%20tRNA(Glu)%20from%20Escherichia%20coli&amp;author=S.%20Sekine&amp;author=O.%20Nureki&amp;author=K.%20Sakamoto&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=256&amp;pages=685-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8642591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sekine</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nureki</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Tateno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yokoyama</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The identity determinants required for the discrimination between tRNAGlu and tRNAAsp by glutamyl-tRNA synthetase from <em>Escherichia coli</em></div>, <div class="source">Eur J Biochem.</div> , <div class="year">1999</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">354</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20identity%20determinants%20required%20for%20the%20discrimination%20between%20tRNAGlu%20and%20tRNAAsp%20by%20glutamyl-tRNA%20synthetase%20from%20Escherichia%20coli&amp;author=S.%20Sekine&amp;author=O.%20Nureki&amp;author=M.%20Tateno&amp;author=S.%20Yokoyama&amp;publication_year=1999&amp;journal=Eur%20J%20Biochem.&amp;volume=261&amp;pages=354-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1327.1999.00293.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1327.1999.00293.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1327.1999.00293.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10215844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">Y.M.</div></div>,  <div class="name"><div class="surname">Westhof</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>. </div><div class="article-title">An unusual RNA tertiary interaction has a role for the specific aminoacylation of a transfer RNA</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">6776</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20unusual%20RNA%20tertiary%20interaction%20has%20a%20role%20for%20the%20specific%20aminoacylation%20of%20a%20transfer%20RNA&amp;author=Y.M.%20Hou&amp;author=E.%20Westhof&amp;author=R.%20Giege&amp;publication_year=1993&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=6776-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.14.6776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.14.6776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.14.6776"> </span></div> </div></div></div></div></div><div content-id="DSS008C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lipman</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">Y.M.</div></div>. </div><div class="article-title">Aminoacylation of tRNA in the evolution of an aminoacyl-tRNA synthetase</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">13495</div>-<div class="lpage">500</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aminoacylation%20of%20tRNA%20in%20the%20evolution%20of%20an%20aminoacyl-tRNA%20synthetase&amp;author=R.S.%20Lipman&amp;author=Y.M.%20Hou&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=13495-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.23.13495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.23.13495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.23.13495"> </span></div> </div></div></div></div></div><div content-id="DSS008C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nameki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Himeno</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Asahara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>Escherichia coli</em> tRNA(Asp) recognition mechanism differing from that of the yeast system</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">1992</div>, vol. <div class="volume">189</div> (pg. <div class="fpage">856</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20tRNA(Asp)%20recognition%20mechanism%20differing%20from%20that%20of%20the%20yeast%20system&amp;author=N.%20Nameki&amp;author=K.%20Tamura&amp;author=H.%20Himeno&amp;author=H.%20Asahara&amp;author=T.%20Hasegawa&amp;author=M.%20Shimizu&amp;publication_year=1992&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=189&amp;pages=856-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(92)92282-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(92)92282-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(92)92282-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1472058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Florentz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Kern</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Gangloff</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Eriani</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Moras</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Aspartate identity of transfer RNAs</div>, <div class="source">Biochimie</div> , <div class="year">1996</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">605</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aspartate%20identity%20of%20transfer%20RNAs&amp;author=R.%20Giege&amp;author=C.%20Florentz&amp;author=D.%20Kern&amp;author=J.%20Gangloff&amp;author=G.%20Eriani&amp;author=D.%20Moras&amp;publication_year=1996&amp;journal=Biochimie&amp;volume=78&amp;pages=605-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0300-9084(96)80007-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0300-9084(96)80007-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0300-9084(96)80007-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8955904" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fender</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sissler</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Florentz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Functional idiosyncrasies of tRNA isoacceptors in cognate and noncognate aminoacylation systems</div>, <div class="source">Biochimie</div> , <div class="year">2004</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">21</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20idiosyncrasies%20of%20tRNA%20isoacceptors%20in%20cognate%20and%20noncognate%20aminoacylation%20systems&amp;author=A.%20Fender&amp;author=M.%20Sissler&amp;author=C.%20Florentz&amp;author=R.%20Giege&amp;publication_year=2004&amp;journal=Biochimie&amp;volume=86&amp;pages=21-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2003.11.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2003.11.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biochi.2003.11.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14987797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asahara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Himeno</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nameki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Discrimination among <em>E. coli</em> tRNAs with a long variable arm</div>, <div class="source">Nucleic Acids Symp. Ser.</div> , <div class="year">1993</div>(pg. <div class="fpage">207</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS008C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meinnel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mechulam</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Lazennec</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Blanquet</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fayat</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Critical role of the acceptor stem of tRNAs(Met) in their aminoacylation by <em>Escherichia coli</em> methionyl-tRNA synthetase</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">229</div> (pg. <div class="fpage">26</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Critical%20role%20of%20the%20acceptor%20stem%20of%20tRNAs(Met)%20in%20their%20aminoacylation%20by%20Escherichia%20coli%20methionyl-tRNA%20synthetase&amp;author=T.%20Meinnel&amp;author=Y.%20Mechulam&amp;author=C.%20Lazennec&amp;author=S.%20Blanquet&amp;author=G.%20Fayat&amp;publication_year=1993&amp;journal=J.%20Mol.%20Biol.&amp;volume=229&amp;pages=26-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1993.1005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1993.1005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1993.1005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8421312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogers</div> <div class="given-names">K.C.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Discrimination among tRNAs intermediate in glutamate and glutamine acceptor identity</div>, <div class="source">Biochemistry</div> , <div class="year">1993</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">14210</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discrimination%20among%20tRNAs%20intermediate%20in%20glutamate%20and%20glutamine%20acceptor%20identity&amp;author=K.C.%20Rogers&amp;author=D.%20Soll&amp;publication_year=1993&amp;journal=Biochemistry&amp;volume=32&amp;pages=14210-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00214a021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00214a021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00214a021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7505112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hong</div> <div class="given-names">K.W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Sever</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Interactions between tRNA identity nucleotides and their recognition sites in glutaminyl-tRNA synthetase determine the cognate amino acid affinity of the enzyme</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">6953</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interactions%20between%20tRNA%20identity%20nucleotides%20and%20their%20recognition%20sites%20in%20glutaminyl-tRNA%20synthetase%20determine%20the%20cognate%20amino%20acid%20affinity%20of%20the%20enzyme&amp;author=M.%20Ibba&amp;author=K.W.%20Hong&amp;author=J.M.%20Sherman&amp;author=S.%20Sever&amp;author=D.%20Soll&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=6953-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.14.6953" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.14.6953" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.14.6953"> </span></div> </div></div></div></div></div><div content-id="DSS008C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Freist</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Gauss</div> <div class="given-names">D.H.</div></div>,  <div class="name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Glutaminyl-tRNA synthetase</div>, <div class="source">Biol. Chem.</div> , <div class="year">1997</div>, vol. <div class="volume">378</div> (pg. <div class="fpage">1103</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glutaminyl-tRNA%20synthetase&amp;author=W.%20Freist&amp;author=D.H.%20Gauss&amp;author=M.%20Ibba&amp;author=D.%20Soll&amp;publication_year=1997&amp;journal=Biol.%20Chem.&amp;volume=378&amp;pages=1103-17" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9372179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Normanly</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ollick</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abelson</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Eight base changes are sufficient to convert a leucine-inserting tRNA into a serine-inserting tRNA</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">5680</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Eight%20base%20changes%20are%20sufficient%20to%20convert%20a%20leucine-inserting%20tRNA%20into%20a%20serine-inserting%20tRNA&amp;author=J.%20Normanly&amp;author=T.%20Ollick&amp;author=J.%20Abelson&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=5680-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.12.5680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.12.5680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.12.5680"> </span></div> </div></div></div></div></div><div content-id="DSS008C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McClain</div> <div class="given-names">W.H.</div></div>,  <div class="name"><div class="surname">Foss</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Nucleotides that contribute to the identity of <em>Escherichia coli</em> tRNA(Phe)</div>, <div class="source">J Mol Biol.</div> , <div class="year">1988</div>, vol. <div class="volume">202</div> (pg. <div class="fpage">697</div>-<div class="lpage">709</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotides%20that%20contribute%20to%20the%20identity%20of%20Escherichia%20coli%20tRNA(Phe)&amp;author=W.H.%20McClain&amp;author=K.%20Foss&amp;publication_year=1988&amp;journal=J%20Mol%20Biol.&amp;volume=202&amp;pages=697-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(88)90551-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(88)90551-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(88)90551-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2459397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sampson</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">DiRenzo</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Behlen</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Uhlenbeck</div> <div class="given-names">O.C.</div></div>. </div><div class="article-title">Nucleotides in yeast tRNAPhe required for the specific recognition by its cognate synthetase</div>, <div class="source">Science</div> , <div class="year">1989</div>, vol. <div class="volume">243</div> (pg. <div class="fpage">1363</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotides%20in%20yeast%20tRNAPhe%20required%20for%20the%20specific%20recognition%20by%20its%20cognate%20synthetase&amp;author=J.R.%20Sampson&amp;author=A.B.%20DiRenzo&amp;author=L.S.%20Behlen&amp;author=O.C.%20Uhlenbeck&amp;publication_year=1989&amp;journal=Science&amp;volume=243&amp;pages=1363-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2646717" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2646717" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2646717"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2646717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nazarenko</div> <div class="given-names">I.A.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">E.T.</div></div>,  <div class="name"><div class="surname">Zakharova</div> <div class="given-names">O.D.</div></div>,  <div class="name"><div class="surname">Lavrik</div> <div class="given-names">O.I.</div></div>,  <div class="name"><div class="surname">Uhlenbeck</div> <div class="given-names">O.C.</div></div>. </div><div class="article-title">Recognition nucleotides for human phenylalanyl-tRNA synthetase</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">475</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20nucleotides%20for%20human%20phenylalanyl-tRNA%20synthetase&amp;author=I.A.%20Nazarenko&amp;author=E.T.%20Peterson&amp;author=O.D.%20Zakharova&amp;author=O.I.%20Lavrik&amp;author=O.C.%20Uhlenbeck&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=475-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.3.475" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.3.475" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.3.475"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1741281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nameki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asahara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Recognition of tRNA(Gly) by three widely diverged glycyl-tRNA synthetases</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">640</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20tRNA(Gly)%20by%20three%20widely%20diverged%20glycyl-tRNA%20synthetases&amp;author=N.%20Nameki&amp;author=K.%20Tamura&amp;author=H.%20Asahara&amp;author=T.%20Hasegawa&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=640-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9171287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Imura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Weiss</div> <div class="given-names">G.B.</div></div>,  <div class="name"><div class="surname">Chambers</div> <div class="given-names">R.W.</div></div>. </div><div class="article-title">Reconstitution of alanine acceptor activity from fragments of yeast tRNA-Ala II</div>, <div class="source">Nature</div> , <div class="year">1969</div>, vol. <div class="volume">222</div> (pg. <div class="fpage">1147</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstitution%20of%20alanine%20acceptor%20activity%20from%20fragments%20of%20yeast%20tRNA-Ala%20II&amp;author=N.%20Imura&amp;author=G.B.%20Weiss&amp;author=R.W.%20Chambers&amp;publication_year=1969&amp;journal=Nature&amp;volume=222&amp;pages=1147-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/2221147a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F2221147a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F2221147a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5788979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soma</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kumagai</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Himeno</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The anticodon loop is a major identity determinant of Saccharomyces cerevisiae tRNA(Leu)</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">707</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20anticodon%20loop%20is%20a%20major%20identity%20determinant%20of%20Saccharomyces%20cerevisiae%20tRNA(Leu)&amp;author=A.%20Soma&amp;author=R.%20Kumagai&amp;author=K.%20Nishikawa&amp;author=H.%20Himeno&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=263&amp;pages=707-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8947570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Putz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Puglisi</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Florentz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Identity elements for specific aminoacylation of yeast tRNA(Asp) by cognate aspartyl-tRNA synthetase</div>, <div class="source">Science</div> , <div class="year">1991</div>, vol. <div class="volume">252</div> (pg. <div class="fpage">1696</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20elements%20for%20specific%20aminoacylation%20of%20yeast%20tRNA(Asp)%20by%20cognate%20aspartyl-tRNA%20synthetase&amp;author=J.%20Putz&amp;author=J.D.%20Puglisi&amp;author=C.%20Florentz&amp;author=R.%20Giege&amp;publication_year=1991&amp;journal=Science&amp;volume=252&amp;pages=1696-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2047878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2047878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2047878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2047878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frugier</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Florentz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Identity switches between tRNAs aminoacylated by class I glutaminyl- and class II aspartyl-tRNA synthetases</div>, <div class="source">Biochemistry</div> , <div class="year">1994</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">9912</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20switches%20between%20tRNAs%20aminoacylated%20by%20class%20I%20glutaminyl-%20and%20class%20II%20aspartyl-tRNA%20synthetases&amp;author=M.%20Frugier&amp;author=D.%20Soll&amp;author=R.%20Giege&amp;author=C.%20Florentz&amp;publication_year=1994&amp;journal=Biochemistry&amp;volume=33&amp;pages=9912-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00199a013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00199a013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00199a013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8060999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mallick</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Chakrabarti</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Identity elements of archaeal tRNA</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">235</div>-<div class="lpage">46</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20elements%20of%20archaeal%20tRNA&amp;author=B.%20Mallick&amp;author=J.%20Chakrabarti&amp;author=S.%20Sahoo&amp;author=Z.%20Ghosh&amp;author=S.%20Das&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=235-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsi008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsi008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsi008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16769686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Miyano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Himeno</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sano</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Identity determinants of <em>E. coli</em> threonine tRNA</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">1992</div>, vol. <div class="volume">184</div> (pg. <div class="fpage">478</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20determinants%20of%20E.%20coli%20threonine%20tRNA&amp;author=T.%20Hasegawa&amp;author=M.%20Miyano&amp;author=H.%20Himeno&amp;author=Y.%20Sano&amp;author=K.%20Kimura&amp;author=M.%20Shimizu&amp;publication_year=1992&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=184&amp;pages=478-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0006-291X(92)91219-G" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0006-291X(92)91219-G" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0006-291X(92)91219-G"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1567450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nameki</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Identity elements of tRNA(Thr) towards <em>Saccharomyces cerevisiae</em> threonyl-tRNA synthetase</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1995</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2831</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20elements%20of%20tRNA(Thr)%20towards%20Saccharomyces%20cerevisiae%20threonyl-tRNA%20synthetase&amp;author=N.%20Nameki&amp;publication_year=1995&amp;journal=Nucleic%20Acids%20Res.&amp;volume=23&amp;pages=2831-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/23.15.2831" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F23.15.2831" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F23.15.2831"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7659504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nameki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Asahara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Identity elements of <em>Thermus thermophilus</em> tRNA(Thr)</div>, <div class="source">FEBS Lett.</div> , <div class="year">1996</div>, vol. <div class="volume">396</div> (pg. <div class="fpage">201</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identity%20elements%20of%20Thermus%20thermophilus%20tRNA(Thr)&amp;author=N.%20Nameki&amp;author=H.%20Asahara&amp;author=T.%20Hasegawa&amp;publication_year=1996&amp;journal=FEBS%20Lett.&amp;volume=396&amp;pages=201-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(96)01094-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(96)01094-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(96)01094-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8914987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagaoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yokozawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Umehara</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Molecular recognition of threonine tRNA by threonyl-tRNA synthetase from an extreme thermophilic archaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div><div class="issue">Suppl</div>(pg. <div class="fpage">81</div>-<div class="lpage">2</div>)</div></div></div></div><div content-id="DSS008C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Determination of phenylalanine tRNA recognition sites by phenylalanyl-tRNA synthetase from hyperthermophilic archaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Symp. Ser. (Oxf)</div> , <div class="year">2007</div>(pg. <div class="fpage">367</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS008C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korencic</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Polycarpo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Weygand-Durasevic</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Differential modes of transfer RNASer recognition in Methanosarcina barkeri</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2004</div>, vol. <div class="volume">279</div> (pg. <div class="fpage">48780</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20modes%20of%20transfer%20RNASer%20recognition%20in%20Methanosarcina%20barkeri&amp;author=D.%20Korencic&amp;author=C.%20Polycarpo&amp;author=I.%20Weygand-Durasevic&amp;author=D.%20Soll&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem.&amp;volume=279&amp;pages=48780-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M408753200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M408753200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M408753200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15364939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwaki</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Asahara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nagaoka</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Differences in tyrosine tRNA identity between Escherichia coli and archaeon, Aeropyrum pernix K1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div><div class="issue">Suppl</div>(pg. <div class="fpage">225</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DSS008C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwaki</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fujimoto</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Momma</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Overexpression, purification and crystallization of tyrosyl-tRNA synthetase from the hyperthermophilic archaeon Aeropyrum pernix K1</div>, <div class="source">Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.</div> , <div class="year">2005</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">1003</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overexpression%2C%20purification%20and%20crystallization%20of%20tyrosyl-tRNA%20synthetase%20from%20the%20hyperthermophilic%20archaeon%20Aeropyrum%20pernix%20K1&amp;author=J.%20Iwaki&amp;author=R.%20Suzuki&amp;author=Z.%20Fujimoto&amp;author=M.%20Momma&amp;author=A.%20Kuno&amp;author=T.%20Hasegawa&amp;publication_year=2005&amp;journal=Acta%20Crystallogr.%20Sect.%20F%20Struct.%20Biol.%20Cryst.%20Commun.&amp;volume=61&amp;pages=1003-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1107/S1744309105033245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1107%2FS1744309105033245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1107%2FS1744309105033245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16511219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pak</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pallanck</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Schulman</div> <div class="given-names">L.H.</div></div>. </div><div class="article-title">Conversion of a methionine initiator tRNA into a tryptophan-inserting elongator tRNA <em>in vivo</em></div>, <div class="source">Biochemistry</div> , <div class="year">1992</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3303</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conversion%20of%20a%20methionine%20initiator%20tRNA%20into%20a%20tryptophan-inserting%20elongator%20tRNA%20in%20vivo&amp;author=M.%20Pak&amp;author=L.%20Pallanck&amp;author=L.H.%20Schulman&amp;publication_year=1992&amp;journal=Biochemistry&amp;volume=31&amp;pages=3303-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00128a001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00128a001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00128a001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1554714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pak</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Willis</div> <div class="given-names">I.M.</div></div>,  <div class="name"><div class="surname">Schulman</div> <div class="given-names">L.H.</div></div>. </div><div class="article-title">Analysis of acceptor stem base pairing on tRNA(Trp) aminoacylation and function <em>in vivo</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1994</div>, vol. <div class="volume">269</div> (pg. <div class="fpage">2277</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20acceptor%20stem%20base%20pairing%20on%20tRNA(Trp)%20aminoacylation%20and%20function%20in%20vivo&amp;author=M.%20Pak&amp;author=I.M.%20Willis&amp;author=L.H.%20Schulman&amp;publication_year=1994&amp;journal=J.%20Biol.%20Chem.&amp;volume=269&amp;pages=2277-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8294486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C61" class="ref-content "><div class="label -label">61</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rogers</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Inokuchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Switching tRNA(Gln) identity from glutamine to tryptophan</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">3463</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Switching%20tRNA(Gln)%20identity%20from%20glutamine%20to%20tryptophan&amp;author=M.J.%20Rogers&amp;author=T.%20Adachi&amp;author=H.%20Inokuchi&amp;author=D.%20Soll&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=3463-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.8.3463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.8.3463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.8.3463"> </span></div> </div></div></div></div></div><div content-id="DSS008C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C62" class="ref-content "><div class="label -label">62</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">von Pawel-Rammingen</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Astrom</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bystrom</div> <div class="given-names">A. S.</div></div>. </div><div class="article-title">Mutational analysis of conserved positions potentially important for initiator tRNA function in Saccharomyces cerevisiae</div>, <div class="source">Mol. Cell. Biol.</div> , <div class="year">1992</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1432</div>-<div class="lpage">42</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20analysis%20of%20conserved%20positions%20potentially%20important%20for%20initiator%20tRNA%20function%20in%20Saccharomyces%20cerevisiae&amp;author=U.%20von%20Pawel-Rammingen&amp;author=S.%20Astrom&amp;author=A.%20S.%20Bystrom&amp;publication_year=1992&amp;journal=Mol.%20Cell.%20Biol.&amp;volume=12&amp;pages=1432-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.12.4.1432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.12.4.1432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.12.4.1432"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1549105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C63" class="ref-content "><div class="label -label">63</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Drabkin</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Helk</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">RajBhandary</div> <div class="given-names">U.L.</div></div>. </div><div class="article-title">The role of nucleotides conserved in eukaryotic initiator methionine tRNAs in initiation of protein synthesis</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1993</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">25221</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20nucleotides%20conserved%20in%20eukaryotic%20initiator%20methionine%20tRNAs%20in%20initiation%20of%20protein%20synthesis&amp;author=H.J.%20Drabkin&amp;author=B.%20Helk&amp;author=U.L.%20RajBhandary&amp;publication_year=1993&amp;journal=J.%20Biol.%20Chem.&amp;volume=268&amp;pages=25221-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8227087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C64" class="ref-content "><div class="label -label">64</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meinnel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mechulam</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fayat</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Blanquet</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Involvement of the size and sequence of the anticodon loop in tRNA recognition by mammalian and <em>E. coli</em> methionyl-tRNA synthetases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">4741</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20the%20size%20and%20sequence%20of%20the%20anticodon%20loop%20in%20tRNA%20recognition%20by%20mammalian%20and%20E.%20coli%20methionyl-tRNA%20synthetases&amp;author=T.%20Meinnel&amp;author=Y.%20Mechulam&amp;author=G.%20Fayat&amp;author=S.%20Blanquet&amp;publication_year=1992&amp;journal=Nucleic%20Acids%20Res.&amp;volume=20&amp;pages=4741-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.18.4741" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.18.4741" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.18.4741"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1408786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C65" class="ref-content "><div class="label -label">65</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Senger</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Auxilien</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Englisch</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Cramer</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Fasiolo</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The modified wobble base inosine in yeast tRNAIle is a positive determinant for aminoacylation by isoleucyl-tRNA synthetase</div>, <div class="source">Biochemistry</div> , <div class="year">1997</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">8269</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20modified%20wobble%20base%20inosine%20in%20yeast%20tRNAIle%20is%20a%20positive%20determinant%20for%20aminoacylation%20by%20isoleucyl-tRNA%20synthetase&amp;author=B.%20Senger&amp;author=S.%20Auxilien&amp;author=U.%20Englisch&amp;author=F.%20Cramer&amp;author=F.%20Fasiolo&amp;publication_year=1997&amp;journal=Biochemistry&amp;volume=36&amp;pages=8269-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi970206l" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi970206l" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi970206l"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9204872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C66" class="ref-content "><div class="label -label">66</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moulinier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Eiler</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Eriani</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">The structure of an AspRS-tRNA(Asp) complex reveals a tRNA-dependent control mechanism</div>, <div class="source">EMBO J.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">5290</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20an%20AspRS-tRNA(Asp)%20complex%20reveals%20a%20tRNA-dependent%20control%20mechanism&amp;author=L.%20Moulinier&amp;author=S.%20Eiler&amp;author=G.%20Eriani&amp;publication_year=2001&amp;journal=EMBO%20J.&amp;volume=20&amp;pages=5290-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/20.18.5290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F20.18.5290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F20.18.5290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11566892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C67" class="ref-content "><div class="label -label">67</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delagoutte</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Moras</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cavarelli</div> <div class="given-names">J.</div></div>. </div><div class="article-title">tRNA aminoacylation by arginyl-tRNA synthetase: induced conformations during substrates binding</div>, <div class="source">EMBO J.</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">5599</div>-<div class="lpage">610</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20aminoacylation%20by%20arginyl-tRNA%20synthetase%3A%20induced%20conformations%20during%20substrates%20binding&amp;author=B.%20Delagoutte&amp;author=D.%20Moras&amp;author=J.%20Cavarelli&amp;publication_year=2000&amp;journal=EMBO%20J.&amp;volume=19&amp;pages=5599-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/19.21.5599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F19.21.5599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F19.21.5599"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11060012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C68" class="ref-content "><div class="label -label">68</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ruff</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Krishnaswamy</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Boeglin</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Class II aminoacyl transfer RNA synthetases: crystal structure of yeast aspartyl-tRNA synthetase complexed with tRNA(Asp)</div>, <div class="source">Science</div> , <div class="year">1991</div>, vol. <div class="volume">252</div> (pg. <div class="fpage">1682</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Class%20II%20aminoacyl%20transfer%20RNA%20synthetases%3A%20crystal%20structure%20of%20yeast%20aspartyl-tRNA%20synthetase%20complexed%20with%20tRNA(Asp)&amp;author=M.%20Ruff&amp;author=S.%20Krishnaswamy&amp;author=M.%20Boeglin&amp;publication_year=1991&amp;journal=Science&amp;volume=252&amp;pages=1682-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2047877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2047877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2047877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2047877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C69" class="ref-content "><div class="label -label">69</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Musier-Forsyth</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Escherichia coli proline tRNA synthetase is sensitive to changes in the core region of tRNA(Pro)</div>, <div class="source">Biochemistry</div> , <div class="year">1994</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">12708</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20proline%20tRNA%20synthetase%20is%20sensitive%20to%20changes%20in%20the%20core%20region%20of%20tRNA(Pro)&amp;author=H.%20Liu&amp;author=K.%20Musier-Forsyth&amp;publication_year=1994&amp;journal=Biochemistry&amp;volume=33&amp;pages=12708-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi00208a023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi00208a023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi00208a023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7522561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C70" class="ref-content "><div class="label -label">70</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ryckelynck</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Giege</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Frugier</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Yeast tRNA(Asp) charging accuracy is threatened by the N-terminal extension of aspartyl-tRNA synthetase</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">9683</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Yeast%20tRNA(Asp)%20charging%20accuracy%20is%20threatened%20by%20the%20N-terminal%20extension%20of%20aspartyl-tRNA%20synthetase&amp;author=M.%20Ryckelynck&amp;author=R.%20Giege&amp;author=M.%20Frugier&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=9683-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M211035200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M211035200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M211035200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12486031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C71" class="ref-content "><div class="label -label">71</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naganuma</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sekine</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fukunaga</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yokoyama</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Unique protein architecture of alanyl-tRNA synthetase for aminoacylation, editing, and dimerization</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2009</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">8489</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unique%20protein%20architecture%20of%20alanyl-tRNA%20synthetase%20for%20aminoacylation%2C%20editing%2C%20and%20dimerization&amp;author=M.%20Naganuma&amp;author=S.%20Sekine&amp;author=R.%20Fukunaga&amp;author=S.%20Yokoyama&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=8489-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0901572106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0901572106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0901572106"> </span></div> </div></div></div></div></div><div content-id="DSS008C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C72" class="ref-content "><div class="label -label">72</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sokabe</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Okada</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakashima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Molecular basis of alanine discrimination in editing site</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">11669</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20of%20alanine%20discrimination%20in%20editing%20site&amp;author=M.%20Sokabe&amp;author=A.%20Okada&amp;author=M.%20Yao&amp;author=T.%20Nakashima&amp;author=I.%20Tanaka&amp;publication_year=2005&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=11669-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0502119102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0502119102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0502119102"> </span></div> </div></div></div></div></div><div content-id="DSS008C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C73" class="ref-content "><div class="label -label">73</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tumbula-Hansen</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Toogood</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stetter</div> <div class="given-names">K.O.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Evolutionary divergence of the archaeal aspartyl-tRNA synthetases into discriminating and nondiscriminating forms</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">277</div> (pg. <div class="fpage">37184</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20divergence%20of%20the%20archaeal%20aspartyl-tRNA%20synthetases%20into%20discriminating%20and%20nondiscriminating%20forms&amp;author=D.%20Tumbula-Hansen&amp;author=L.%20Feng&amp;author=H.%20Toogood&amp;author=K.O.%20Stetter&amp;author=D.%20Soll&amp;publication_year=2002&amp;journal=J.%20Biol.%20Chem.&amp;volume=277&amp;pages=37184-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M204767200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M204767200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M204767200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12149259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C74" class="ref-content "><div class="label -label">74</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmitt</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moulinier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Fujiwara</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Imanaka</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Thierry</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Moras</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Crystal structure of aspartyl-tRNA synthetase from Pyrococcus kodakaraensis KOD: archaeon specificity and catalytic mechanism of adenylate formation</div>, <div class="source">EMBO J.</div> , <div class="year">1998</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">5227</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crystal%20structure%20of%20aspartyl-tRNA%20synthetase%20from%20Pyrococcus%20kodakaraensis%20KOD%3A%20archaeon%20specificity%20and%20catalytic%20mechanism%20of%20adenylate%20formation&amp;author=E.%20Schmitt&amp;author=L.%20Moulinier&amp;author=S.%20Fujiwara&amp;author=T.%20Imanaka&amp;author=J.C.%20Thierry&amp;author=D.%20Moras&amp;publication_year=1998&amp;journal=EMBO%20J.&amp;volume=17&amp;pages=5227-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/17.17.5227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F17.17.5227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F17.17.5227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9724658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C75" class="ref-content "><div class="label -label">75</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feng</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Tumbula-Hansen</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Toogood</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Soll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Expanding tRNA recognition of a tRNA synthetase by a single amino acid change</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">5676</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expanding%20tRNA%20recognition%20of%20a%20tRNA%20synthetase%20by%20a%20single%20amino%20acid%20change&amp;author=L.%20Feng&amp;author=D.%20Tumbula-Hansen&amp;author=H.%20Toogood&amp;author=D.%20Soll&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=5676-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0631525100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0631525100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0631525100"> </span></div> </div></div></div></div></div><div content-id="DSS008C76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C76" class="ref-content "><div class="label -label">76</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Recognition sites of glycine tRNA for glycyl-tRNA synthetase from hyperthermophilic archaeon, Aeropyrum pernix K1</div>, <div class="source">Nucleic Acids Symp. Ser. (Oxf)</div> , <div class="year">2005</div>(pg. <div class="fpage">299</div>-<div class="lpage">300</div>)</div></div></div></div><div content-id="DSS008C77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C77" class="ref-content "><div class="label -label">77</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagatoyo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwaki</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Molecular recognition of histidine tRNA by histidyl-tRNA synthetase from hyperthermophilic archaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Symp. Ser. (Oxf)</div> , <div class="year">2005</div>(pg. <div class="fpage">307</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS008C78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C78" class="ref-content "><div class="label -label">78</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yokozawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kawarabayasi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Molecular recognition of proline tRNA by prolyl-tRNA synthetase from hyperthermophilic archaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div><div class="issue">Suppl</div>(pg. <div class="fpage">247</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS008C79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C79" class="ref-content "><div class="label -label">79</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gruic-Sovulj</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Jaric</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dulic</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cindric</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Weygand-Durasevic</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Shuffling of discrete tRNASer regions reveals differently utilized identity elements in yeast and methanogenic archaea</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">361</div> (pg. <div class="fpage">128</div>-<div class="lpage">39</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shuffling%20of%20discrete%20tRNASer%20regions%20reveals%20differently%20utilized%20identity%20elements%20in%20yeast%20and%20methanogenic%20archaea&amp;author=I.%20Gruic-Sovulj&amp;author=J.%20Jaric&amp;author=M.%20Dulic&amp;author=M.%20Cindric&amp;author=I.%20Weygand-Durasevic&amp;publication_year=2006&amp;journal=J.%20Mol.%20Biol.&amp;volume=361&amp;pages=128-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2006.06.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2006.06.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2006.06.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16822522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS008C80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C80" class="ref-content "><div class="label -label">80</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yokozawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nagaoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Umehara</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Recognition of tRNA by aminoacyl-tRNA synthetase from hyperthermophilic archaea, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div><div class="issue">Suppl</div>(pg. <div class="fpage">117</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS008C81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C81" class="ref-content "><div class="label -label">81</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikura</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nagaoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yokozawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Umehara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Threonyl-tRNA synthetase of archaea: importance of the discriminator base in the aminoacylation of threonine tRNA</div>, <div class="source">Nucleic Acids Symp. Ser.</div> , <div class="year">2000</div>(pg. <div class="fpage">83</div>-<div class="lpage">4</div>)</div></div></div></div><div content-id="DSS008C82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS008C82" class="ref-content "><div class="label -label">82</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsuchiya</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Umehara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kuno</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hasegawa</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Determination of tryptophan tRNA recognition sites for tryptophanyl-tRNA synthetase from hyperthermophilic archaeon, <em>Aeropyrum pernix</em> K1</div>, <div class="source">Nucleic Acids Symp. Ser. (Oxf)</div> , <div class="year">2004</div>(pg. <div class="fpage">185</div>-<div class="lpage">6</div>)</div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=475122&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp_table3.xls?Expires=2147483647&amp;Signature=0xUPcl8ksSQQwZe5pbxAhJnLNzYwFbIooHcXJOrfbIdaoQQIMpoA3ZPhwSejip5pezupLwFAvs9-6Lu~uDLxVh4zBYJMXxt5K6DPZqMSVrjVawyY-kzAmTtSODRu3Vj4GQ0VaJr2k7gYh2BarkdoLLiEPzajDBNvJPy~z3RGFswJYJrlJ2sSG7Sg4An4Gmzg7POPfsmSRId6RGxCRdktKuneoBhBUgWlga5eDRTEy9vD4iwuk-pIRcAR-U-A31S-YEslcBstxZ1MdloNVG7nSZpgGtQFNQitJ9G2JgDueYA~AmJLE-6xNS8Mk7yQTekh65DNOTkgkxfP~NxP0e46Yg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp_table2.xls?Expires=2147483647&amp;Signature=mfvYS5VpF74a9XMcuUZ4~3z6IcV~~29Nxo06cF-xU5QdZOOZYg9SK6Txphzasbeh7XOc-Yb7xjBD-rLnbylK2~ruUpfpVO9km1glcVC3VG7A8r1Rxj1R3WNnDc13imDvVEEXRTuXNGMrXc4T10eIuLtsiBZCIIGb6sStun~Nfnv~vyPr1SjwEJ0K~v79HUuh1p~IySktFyICoJXsrpnuEDZZsVyCzKleud7aB8RR7Jeb-IMH5erKj2QzSHfuCrrOJ9lhCoWFcYAvi0WJa8rb2Ghk3OwS~LAogIeYAcbsbC8GYMdrHb5l-FeAWY-9IVBeA2kH0iJmblYyeuz3YuWK4Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp_table1.pdf?Expires=2147483647&amp;Signature=adbdZLuQmkFT~ZGDbcrQDj9e0-AQhshRrIK4WNHL7NvXjbQgjza~VKZfH1~hGNK1vzf067P6qP7VvV3toTgk9qKmZb1ajy1ywbcxYF2~t8gfc2ijeLgOpBqho96MuJ44QLWOHk~VrWqyDkqR~iblQLrEuzME-Es0tahf-aj~-1pR8UPkVNBhOXOlaKpzekltp5ScSwxu3OFjYCXHEz0j~i7B1FOfmawaEcCSDPJo0jWyuQ952YD93EgGcpm79X3HPA~hSQX6oCX7hSvYK1f45Uh-Mj9YeqIZeT2UBGUoQGdX4WtBIiVKJCka1P4AQZVbuQv5rQrqu8jR5qV6auZt-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp_fig1.png?Expires=2147483647&amp;Signature=zN4FgsryjJe3kLJjyoqo17HrRwuzwTAG~yIpKbVVOrD0Lj5dQKHZg7wTrQKv6NiuFGAg~Y2R~DFDwa9txchjsCAR61Bl7YM1DshEKaITRp73yTpc85yx5tDHRBeBv4ypQbwB2OcOVqCmtrlvTKHMkHIfHjT8UP~CK80ckfhKUIH8hl90TPlJKx9-L9XCvcCatThiRZ6fNDLswOen2rGLC3-7HqKXS1ElnO3p446rONw6pBzK3FLYB09wKUp4XY-5TcV1yllniWsKXUWFq768GigebG5Vu5X1NInadffnJxOt8AYOWpZ2hAokyhwVlwYNCi53~cjNPY~LLslP3X-Uaw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp2.pdf?Expires=2147483647&amp;Signature=PTa7yx~izrciMyYmLEks1RHlmoHmdOb20fOttN3QjkpP1jYh8Df9r2sanN-jWYFNAKCMZyaAiL3Louch32LyvffAaqLAgjv0cPhe4T4I3nRim8rdpRSt2CuNWhnYnFjb3tDvjEnAJUyFr3aQgWyqQ-MoUhGh11w3x-ZqV~xaIquzPp3zXAGBcs9Sh5Pxrws6CvoO-mUe365AMgcJRm43zJYXYyEUhwN9BCSc-pSjCTYsMBsHBSQCfQjd3mFBnpGlgMDHjSKlqnhezFGBH-1NR4ZYF9UlNnfR1DSoyopkOh5obTyGeBs133AjzE2sfsYk2cyp4XkdHcQrE6nGPT~TxQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss008/2/dss008-dss008supp1.pdf?Expires=2147483647&amp;Signature=qswVNSGCnThQPEnTwAAcia~eXapgV38VXe8N0MigGTXlBIXMXUKNhKhqDH8vEB79EaMLv4~sk3LrBoNyhoroVUHDXE~vhEcRyLLkxIGoIEk3JbxY8Xy5CecqqkkYWs6FsHW0hP2IkD2hFzIYJpeLYdIMMiLopfEB3kwxvh9aG9unH0U-ETND62RF4mbWDVfORnYm6UhrNIEFdY963UA5D1SMYkBYfVJ6XUZuAcdNlArzGnqI4vM2ooeOCUtjlRqsem3qXQvgL1C4FB3Xm2WfnZshrSx6hO99m5E5XyChUGezGhthGnTvrw-z547FtMBGSdaHA1stDQ2qhPjdaTI-3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">117</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="475122"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f3%2f245%2f475122">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000305723200004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F3%2F245%2F475122" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/475122'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mapping Hidden Potential Identity Elements by Computing the Average Discriminating Power of Individual tRNA Positions" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f245%2f475122">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

