

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/353/2469968","@type":"ScholarlyArticle","name":"A novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms","datePublished":"2016-06-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw021","keywords":["Chargaff’s second parity rule (PR2)","dinucleotide relative abundances (odds ratios)","GC mutational pressure","substitution strand-biases","PolIII a-subunit isoforms"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/353/2469968","author":[{"name":"Apostolou-Karampelis, Konstantinos","affiliation":"Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece","@type":"Person"},{"name":"Nikolaou, Christoforos","affiliation":"Computational Genomics Group, Department of Biology, University of Crete, 71409 Heraklion, Greece","@type":"Person"},{"name":"Almirantis, Yannis","affiliation":"Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece","@type":"Person"}],"description":"Abstract.   Strand biases reflect deviations from a null expectation of DNA evolution that assumes strand-symmetric substitution rates. Here, we present strong ","pageStart":"353","pageEnd":"363","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms" />
    <meta property="og:description" content="Abstract.   Strand biases reflect deviations from a null expectation of DNA evolution that assumes strand-symmetric substitution rates. Here, we present strong " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/353/2469968" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-06-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Apostolou-Karampelis, Konstantinos" /><meta name="citation_author_institution" content=" Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece " /><meta name="citation_author" content="Nikolaou, Christoforos" /><meta name="citation_author_institution" content=" Computational Genomics Group, Department of Biology, University of Crete, 71409 Heraklion, Greece " /><meta name="citation_author" content="Almirantis, Yannis" /><meta name="citation_author_institution" content=" Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece " /><meta name="citation_title" content="A novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms" /><meta name="citation_firstpage" content="353" /><meta name="citation_lastpage" content="363" /><meta name="citation_doi" content="10.1093/dnares/dsw021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Evolution of DNA base composition under no-strand-bias conditions when the substitution rates are not constant; citation_author=Lobry  J.R.; citation_author=Lobry  C.;  citation_journal_title=Mol. Biol. Evol;  citation_year=1999;  citation_volume=16;  citation_pages=719-23; " /><meta name="citation_reference" content="citation_title=Intrastrand parity rules of DNA base composition and usage biases of synonymous codons; citation_author=Sueoka  N.;  citation_journal_title=J. Mol. Evol;  citation_year=1995;  citation_volume=40;  citation_pages=318-25; " /><meta name="citation_reference" content="citation_title=Properties of a general model of DNA evolution under no-strand-bias conditions; citation_author=Lobry  J.R.;  citation_journal_title=J. Mol. Evol;  citation_year=1995;  citation_volume=40;  citation_pages=326-30; " /><meta name="citation_reference" content="citation_title=Symmetry observations in long nucleotide sequences; citation_author=Prabhu  V.V.;  citation_journal_title=Nucleic Acids Res;  citation_year=1993;  citation_volume=21;  citation_pages=2797-800; " /><meta name="citation_reference" content="citation_title=Accounting units in DNA; citation_author=Bell  S.J.; citation_author=Forsdyke  D.R.;  citation_journal_title=J. Theor. Biol;  citation_year=1999;  citation_volume=197;  citation_pages=51-61; " /><meta name="citation_reference" content="citation_title=A test of Chargaff’s second rule; citation_author=Mitchell  D.; citation_author=Bridge  R.;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2006;  citation_volume=340;  citation_pages=90-4; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria; citation_author=Lobry  J.R.;  citation_journal_title=Mol. Biol. Evol;  citation_year=1996;  citation_volume=13;  citation_pages=660-5; " /><meta name="citation_reference" content="citation_title=Asymmetric directional mutation pressures in bacteria; citation_author=Lobry  J.R.; citation_author=Sueoka  N.;  citation_journal_title=Genome Biol;  citation_year=2002;  citation_volume=3;  citation_pages=RESEARCH0058" /><meta name="citation_reference" content="citation_title=Universal replication biases in bacteria; citation_author=Rocha  E.P.; citation_author=Danchin  A.; citation_author=Viari  A.;  citation_journal_title=Mol. Microbiol;  citation_year=1999;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=Strand compositional asymmetry in bacterial and large viral genomes; citation_author=Mrázek  J.; citation_author=Karlin  S.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=3720-5; " /><meta name="citation_reference" content="citation_title=A study on the correlation of nucleotide skews and the positioning of the origin of replication: different modes of replication in bacterial species; citation_author=Nikolaou  C.; citation_author=Almirantis  Y.;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=6816-22; " /><meta name="citation_reference" content="citation_title=The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes; citation_author=Tillier  E.R.; citation_author=Collins  R. A.;  citation_journal_title=J. Mol. Evol;  citation_year=2000;  citation_volume=50;  citation_pages=249-57; " /><meta name="citation_reference" content="citation_title=Similar compositional biases are caused by very different mutational effects; citation_author=Rocha  E.P.C.; citation_author=Touchon  M.; citation_author=Feil  E.J.;  citation_journal_title=Genome Res;  citation_year=2006;  citation_volume=16;  citation_pages=1537-47; " /><meta name="citation_reference" content="citation_title=Separating the effects of mutation and selection in producing DNA skew in bacterial chromosomes; citation_author=Morton  R.A.; citation_author=Morton  B.R.;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=369" /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms; citation_author=Frank  A.C.; citation_author=Lobry  J.R.;  citation_journal_title=Gene;  citation_year=1999;  citation_volume=238;  citation_pages=65-77; " /><meta name="citation_reference" content="citation_title=Atypical AT skew in Firmicute genomes results from selection and not from mutation; citation_author=Charneski  C.A.; citation_author=Honti  F.; citation_author=Bryant  J.M.; citation_author=Hurst  L.D.; citation_author=Feil  E.J.;  citation_journal_title=PLoS Genet;  citation_year=2011;  citation_volume=7;  citation_pages=e1002283" /><meta name="citation_reference" content="citation_title=Deviations from Chargaff’s second parity rule in organellar DNA Insights into the evolution of organellar genomes; citation_author=Nikolaou  C.; citation_author=Almirantis  Y.;  citation_journal_title=Gene;  citation_year=2006;  citation_volume=381;  citation_pages=34-41; " /><meta name="citation_reference" content="citation_title=The organization of the bacterial genome; citation_author=Rocha  E.P.C.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2008;  citation_volume=42;  citation_pages=211-33; " /><meta name="citation_reference" content="citation_title=Strong asymmetric mutation bias in endosymbiont genomes coincide with loss of genes for replication restart pathways; citation_author=Klasson  L.; citation_author=Andersson  S.G.E.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2006;  citation_volume=23;  citation_pages=1031-9; " /><meta name="citation_reference" content="citation_title=Genomes and evolution; citation_author=Danchin  A.;  citation_journal_title=Curr. Issues Mol. Biol;  citation_year=2003;  citation_volume=5;  citation_pages=37-42; " /><meta name="citation_reference" content="citation_title=Ongoing evolution of strand composition in bacterial genomes; citation_author=Rocha  E.P.; citation_author=Danchin  A.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2001;  citation_volume=18;  citation_pages=1789-99; " /><meta name="citation_reference" content="citation_title=A new method for assessing the effect of replication on DNA base composition asymmetry; citation_author=Necşulea  A.; citation_author=Lobry  J.R.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=2169-79; " /><meta name="citation_reference" content="citation_title=Origin of replication in circular prokaryotic chromosomes; citation_author=Worning  P.; citation_author=Jensen  L.J.; citation_author=Hallin  P.F.; citation_author=Staerfeldt  H.-H.; citation_author=Ussery  D.W.;  citation_journal_title=Environ. Microbiol;  citation_year=2006;  citation_volume=8;  citation_pages=353-61; " /><meta name="citation_reference" content="citation_title=Deviations from Chargaff’s second parity rule correlate with direction of transcription; citation_author=Bell  S.J.; citation_author=Forsdyke  D.R.;  citation_journal_title=J. Theor. Biol;  citation_year=1999;  citation_volume=197;  citation_pages=63-76; " /><meta name="citation_reference" content="citation_title=Composition strand asymmetries in prokaryotic genomes: mutational bias and biased gene orientation. C.; citation_author=Lopez  P.; citation_author=Philippe  H.;  citation_journal_title=R. Acad. Sci. III;  citation_year=2001;  citation_volume=324;  citation_pages=201-8; " /><meta name="citation_reference" content="citation_title=Asymmetries generated by transcription-coupled repair in enterobacterial genes; citation_author=Francino  M.P.; citation_author=Chao  L.; citation_author=Riley  M.A.; citation_author=Ochman  H.;  citation_journal_title=Science;  citation_year=1996;  citation_volume=272;  citation_pages=107-9; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system; citation_author=Ikemura  T.;  citation_journal_title=J. Mol. Biol;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity; citation_author=Gouy  M.; citation_author=Gautier  C.;  citation_journal_title=Nucleic Acids Res;  citation_year=1982;  citation_volume=10;  citation_pages=7055-74; " /><meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage; citation_author=Bulmer  M.;  citation_journal_title=Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=897-907; " /><meta name="citation_reference" content="citation_title=How optimized is the translational machinery in Escherichia coli, Salmonella typhimurium and Saccharomyces cerevisiae?; citation_author=Xia  X.;  citation_journal_title=Genetics;  citation_year=1998;  citation_volume=149;  citation_pages=37-44; " /><meta name="citation_reference" content="citation_title=Compositional correlation between deoxyribonucleic acid and protein; citation_author=Sueoka  N.;  citation_journal_title=Cold Spring Harb. Symp. Quant. Biol;  citation_year=1961;  citation_volume=26;  citation_pages=35-43; " /><meta name="citation_reference" content="citation_title=The correlation of protein hydropathy with the base composition of coding sequences; citation_author=D’Onofrio  G.; citation_author=Jabbari  K.; citation_author=Musto  H.; citation_author=Bernardi  G.;  citation_journal_title=Gene;  citation_year=1999;  citation_volume=238;  citation_pages=3-14; " /><meta name="citation_reference" content="citation_title=Skewed oligomers and origins of replication; citation_author=Salzberg  S.L.; citation_author=Salzberg  A.J.; citation_author=Kerlavage  A.R.; citation_author=Tomb  J.F.;  citation_journal_title=Gene;  citation_year=1998;  citation_volume=217;  citation_pages=57-67; " /><meta name="citation_reference" content="citation_title=Patterns of nucleotide asymmetries in plant and animal genomes; citation_author=Mascher  M.; citation_author=Schubert  I.; citation_author=Scholz  U.; citation_author=Friedel  S.;  citation_journal_title=Biosystems;  citation_year=2013;  citation_volume=111;  citation_pages=181-9; " /><meta name="citation_reference" content="citation_title=Global dinucleotide signatures and analysis of genomic heterogeneity; citation_author=Karlin  S.;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=1998;  citation_volume=1;  citation_pages=598-610; " /><meta name="citation_reference" content="citation_title=Enzymatic synthesis of deoxyribonucleic acid. VIII. Frequencies of nearest neighbor base sequences in deoxyribonucleic acid; citation_author=Josse  J.; citation_author=Kaiser  A.D.; citation_author=Kornberg  A.;  citation_journal_title=J. Biol. Chem;  citation_year=1961;  citation_volume=236;  citation_pages=864-75; " /><meta name="citation_reference" content="citation_title=Nearest neighbor nucleotide patterns. Structural and biological implications; citation_author=Nussinov  R.;  citation_journal_title=J. Biol. Chem;  citation_year=1981;  citation_volume=256;  citation_pages=8458-62; " /><meta name="citation_reference" content="citation_title=Strong doublet preferences in nucleotide sequences and DNA geometry; citation_author=Nussinov  R.;  citation_journal_title=J. Mol. Evol;  citation_year=1984;  citation_volume=20;  citation_pages=111-9; " /><meta name="citation_reference" content="citation_title=Skew of mononucleotide frequencies, relative abundance of dinucleotides, and DNA strand asymmetry; citation_author=Shioiri  C.; citation_author=Takahata  N.;  citation_journal_title=J. Mol. Evol;  citation_year=2001;  citation_volume=53;  citation_pages=364-76; " /><meta name="citation_reference" content="citation_title=Why are complementary DNA strands symmetric?; citation_author=Baisnée  P.-F.; citation_author=Hampson  S.; citation_author=Baldi  P.;  citation_journal_title=Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=1021-33; " /><meta name="citation_reference" content="citation_title=DoriC 5.0: an updated database of oriC regions in both bacterial and archaeal genomes; citation_author=Gao  F.; citation_author=Luo  H.; citation_author=Zhang  C.-T.;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D90-3; " /><meta name="citation_reference" content="citation_title=The percentage of bacterial genes on leading versus lagging strands is influenced by multiple balancing forces; citation_author=Mao  X.; citation_author=Zhang  H.; citation_author=Yin  Y.; citation_author=Xu  Y.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=8210-8; " /><meta name="citation_reference" content="citation_title=Association of purine asymmetry, strand-biased gene distribution and PolC within Firmicutes and beyond: a new appraisal; citation_author=Saha  S.K.; citation_author=Goswami  A.; citation_author=Dutta  C.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=430" /><meta name="citation_reference" content="citation_title=Testing and dating of structural changes in practice; citation_author=Zeileis  A.; citation_author=Kleiber  C.; citation_author=Krämer  W.; citation_author=Hornik  K.;  citation_journal_title=Comput. Stat. Data Anal;  citation_year=2003;  citation_volume=44;  citation_pages=109-23; " /><meta name="citation_reference" content="citation_title=Testing, monitoring, and dating structural changes in exchange rate regimes; citation_author=Zeileis  A.; citation_author=Shah  A.; citation_author=Patnaik  I.;  citation_journal_title=Comput. Stat. Data Anal;  citation_year=2010;  citation_volume=54;  citation_pages=1696-706; " /><meta name="citation_reference" content="citation_title=strucchange t an R package for testing for structural change in linear regression models; citation_author=Zeileis  A.; citation_author=Leisch  F.; citation_author=Hornik  K.; citation_author=Kleiber  C.;  citation_journal_title=J. Stat. Softw;  citation_year=2002;  citation_volume=7;  citation_pages=1-38; " /><meta name="citation_reference" content="citation_title=On information and sufficiency; citation_author=Kullback  S.; citation_author=Leibler  R.A.;  citation_journal_title=Ann. Math. Stat;  citation_year=1951;  citation_volume=22;  citation_pages=79-86; " /><meta name="citation_reference" content="citation_title=A novel algorithm and web-based tool for comparing two alternative phylogenetic trees; citation_author=Nye  T.M.W.; citation_author=Liò  P.; citation_author=Gilks  W.R.;  citation_journal_title=Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=117-9; " /><meta name="citation_reference" content="citation_title=The NCBI Taxonomy database; citation_author=Federhen  S.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D136-43; " /><meta name="citation_reference" content="citation_title=Dinucleotide relative abundance extremes: a genomic signature; citation_author=Karlin  S.; citation_author=Burge  C.;  citation_journal_title=Trends Genet;  citation_year=1995;  citation_volume=11;  citation_pages=283-90; " /><meta name="citation_reference" content="citation_title=Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA; citation_author=Campbell  A.; citation_author=Mrázek  J.; citation_author=Karlin  S.;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1999;  citation_volume=96;  citation_pages=9184-9; " /><meta name="citation_reference" content="citation_title=eggNOG v4.0: nested orthology inference across 3686 organisms; citation_author=Powell  S.; citation_author=Forslund  K.; citation_author=Szklarczyk  D.;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D231-9; " /><meta name="citation_reference" content="citation_title=Identification and measurement of neighbor-dependent nucleotide substitution processes; citation_author=Arndt  P.F.k; citation_author=Hwa  T.;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=2322-8; " /><meta name="citation_reference" content="citation_title=Similar rates but different modes of sequence evolution in introns and at exonic silent sites in rodents: evidence for selectively driven codon usage; citation_author=Chamary  J.-V.; citation_author=Hurst  L. D.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2004;  citation_volume=21;  citation_pages=1014-23; " /><meta name="citation_reference" content="citation_title=KEGG-PATH: Kyoto encyclopedia of genes and genomes-based pathway analysis using a path analysis model; citation_author=Du  J.; citation_author=Yuan  Z.; citation_author=Ma  Z.; citation_author=Song  J.; citation_author=Xie  X.; citation_author=Chen  Y.;  citation_journal_title=Mol. Biosyst;  citation_year=2014;  citation_volume=10;  citation_pages=2441-7; " /><meta name="citation_reference" content="citation_title=Analyzing genomes with cumulative skew diagrams; citation_author=Grigoriev  A.;  citation_journal_title=Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=2286-90; " /><meta name="citation_reference" content="citation_title=Strand asymmetries in DNA evolution; citation_author=Francino  M.P.; citation_author=Ochman  H.;  citation_journal_title=Trends Genet;  citation_year=1997;  citation_volume=13;  citation_pages=240-5; " /><meta name="citation_reference" content="citation_title=Compositional differences within and between eukaryotic genomes; citation_author=Karlin  S.; citation_author=Mrázek  J.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=10227-32; " /><meta name="citation_reference" content="  Becker R.A.  Chambers J.M.  Wilks A. R.   1988 , The new S language: a programming environment for data analysis and graphics. Pacific Grove, CA, USA: Wadsworth &amp; Brooks/Cole Advanced Books &amp; Software. ISBN 0-534-09192-X. " /><meta name="citation_reference" content="citation_title=A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes; citation_author=Knight  R.D.; citation_author=Freeland  S.J.; citation_author=Landweber  L.F.;  citation_journal_title=Genome Biol;  citation_year=2001;  citation_volume=2;  citation_pages=RESEARCH0010" /><meta name="citation_reference" content="citation_title=The guanine and cytosine content of genomic DNA and bacterial evolution; citation_author=Muto  A.; citation_author=Osawa  S.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=Compositional dynamics of guanine and cytosine content in prokaryotic genomes; citation_author=Hu  J.; citation_author=Zhao  X.; citation_author=Zhang  Z.; citation_author=Yu  J.;  citation_journal_title=Res. Microbiol;  citation_year=2007;  citation_volume=158;  citation_pages=363-70; " /><meta name="citation_reference" content="citation_title=Codon evolution is governed by linear formulas; citation_author=Sorimachi  K.; citation_author=Okayasu  T.;  citation_journal_title=Amino Acids;  citation_year=2008;  citation_volume=34;  citation_pages=661-8; " /><meta name="citation_reference" content="citation_title=A general model of codon bias due to GC mutational bias; citation_author=Palidwor  G.A.; citation_author=Perkins  T.J.; citation_author=Xia  X.;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e13431" /><meta name="citation_reference" content="citation_title=A content-centric organization of the genetic code; citation_author=Yu  J.;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2007;  citation_volume=5;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Modeling compositional dynamics based on GC and purine contents of protein-coding sequences; citation_author=Zhang  Z.; citation_author=Yu  J.;  citation_journal_title=Biol. Direct;  citation_year=2010;  citation_volume=5;  citation_pages=63" /><meta name="citation_reference" content="citation_title=Escherichia coli mfd mutant deficient in “mutation frequency decline” lacks strand-specific repair: in vitro complementation with purified coupling factor; citation_author=Selby  C.P.; citation_author=Witkin  E.M.; citation_author=Sancar  A.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=11574-8; " /><meta name="citation_reference" content="citation_title=Differential repair of premutational UV-lesions at tRNA genes in E. coli; citation_author=Bockrath  R.C.; citation_author=Palmer  J. E.;  citation_journal_title=Mol. Gen. Genet;  citation_year=1977;  citation_volume=156;  citation_pages=133-40; " /><meta name="citation_reference" content="citation_title=Transcription-repair coupling determines the strandedness of ultraviolet mutagenesis in Escherichia coli; citation_author=Oller  A.R.; citation_author=Fijalkowska  I.J.; citation_author=Dunn  R.L.; citation_author=Schaaper  R.M.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=11036-40; " /><meta name="citation_reference" content="citation_title=RNA polymerase between lesion bypass and DNA repair; citation_author=Deaconescu  A.M.;  citation_journal_title=Cell. Mol. Life Sci;  citation_year=2013;  citation_volume=70;  citation_pages=4495-509; " /><meta name="citation_reference" content="citation_title=Accelerated deamination of cytosine residues in UV-induced cyclobutane pyrimidine dimers leads to CC–&gt;TT transitions; citation_author=Peng  W.; citation_author=Shaw  B.R.;  citation_journal_title=Biochemistry;  citation_year=1996;  citation_volume=35;  citation_pages=10172-81; " /><meta name="citation_reference" content="citation_title=Comparative analysis of eubacterial DNA polymerase III alpha subunits; citation_author=Zhao  X.-Q.; citation_author=Hu  J.-F.; citation_author=Yu  J.;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2006;  citation_volume=4;  citation_pages=203-11; " /><meta name="citation_reference" content="citation_title=Is there a role for replication fork asymmetry in the distribution of genes in bacterial genomes?; citation_author=Rocha  E.P.C.;  citation_journal_title=Trends Microbiol;  citation_year=2002;  citation_volume=10;  citation_pages=393-5; " /><meta name="citation_reference" content="citation_title=Two essential DNA polymerases at the bacterial replication fork; citation_author=Dervyn  E.; citation_author=Suski  C.; citation_author=Daniel  R.;  citation_journal_title=Science;  citation_year=2001;  citation_volume=294;  citation_pages=1716-9; " /><meta name="citation_reference" content="citation_title=DnaE2 polymerase contributes to in vivo survival and the emergence of drug resistance in Mycobacterium tuberculosis; citation_author=Boshoff  H.I.M.; citation_author=Reed  M.B.; citation_author=Barry  C.E.; citation_author=Mizrahi  V.;  citation_journal_title=Cell;  citation_year=2003;  citation_volume=113;  citation_pages=183-93; " /><meta name="citation_reference" content="citation_title=An SOS-regulated operon involved in damage-inducible mutagenesis in Caulobacter crescentus; citation_author=Galhardo  R.S.; citation_author=Rocha  R.P.; citation_author=Marques  M.V; citation_author=Menck  C.F.M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=2603-14; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/353/17463076/dsw021.pdf" /><meta name="description" content="Abstract.   Strand biases reflect deviations from a null expectation of DNA evolution that assumes strand-symmetric substitution rates. Here, we present strong " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/353/2469968" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/353/2469968" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/353/2469968");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw021';
    </script>
<title>
	novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469968" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd7KATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQXBvc3RvbG91LUthcmFtcGVsaXMsIEtvbnN0YW50aW5vcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBJbnN0aXR1dGUgb2YgQmlvc2NpZW5jZXMgYW5kIEFwcGxpY2F0aW9ucywgTmF0aW9uYWwgQ2VudGVyIGZvciBTY2llbnRpZmljIFJlc2VhcmNoIOKAnERlbW9rcml0b3PigJ0sIDE1MzEwIEF0aGVucywgR3JlZWNlICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5pa29sYW91LCBDaHJpc3RvZm9yb3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgQ29tcHV0YXRpb25hbCBHZW5vbWljcyBHcm91cCwgRGVwYXJ0bWVudCBvZiBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIENyZXRlLCA3MTQwOSBIZXJha2xpb24sIEdyZWVjZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBbG1pcmFudGlzLCBZYW5uaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIgSW5zdGl0dXRlIG9mIEJpb3NjaWVuY2VzIGFuZCBBcHBsaWNhdGlvbnMsIE5hdGlvbmFsIENlbnRlciBmb3IgU2NpZW50aWZpYyBSZXNlYXJjaCDigJxEZW1va3JpdG9z4oCdLCAxNTMxMCBBdGhlbnMsIEdyZWVjZSAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgbm92ZWwgc2tldyBhbmFseXNpcyByZXZlYWxzIHN1YnN0aXR1dGlvbiBhc3ltbWV0cmllcyBsaW5rZWQgdG8gZ2VuZXRpYyBjb2RlIEdDLWJpYXNlcyBhbmQgUG9sSUlJIGEtc3VidW5pdCBpc29mb3JtcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgRE5BIGJhc2UgY29tcG9zaXRpb24gdW5kZXIgbm8tc3RyYW5kLWJpYXMgY29uZGl0aW9ucyB3aGVuIHRoZSBzdWJzdGl0dXRpb24gcmF0ZXMgYXJlIG5vdCBjb25zdGFudDsgY2l0YXRpb25fYXV0aG9yPUxvYnJ5ICBKLlIuOyBjaXRhdGlvbl9hdXRob3I9TG9icnkgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz03MTktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJhc3RyYW5kIHBhcml0eSBydWxlcyBvZiBETkEgYmFzZSBjb21wb3NpdGlvbiBhbmQgdXNhZ2UgYmlhc2VzIG9mIHN5bm9ueW1vdXMgY29kb25zOyBjaXRhdGlvbl9hdXRob3I9U3Vlb2thICBOLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MzE4LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9wZXJ0aWVzIG9mIGEgZ2VuZXJhbCBtb2RlbCBvZiBETkEgZXZvbHV0aW9uIHVuZGVyIG5vLXN0cmFuZC1iaWFzIGNvbmRpdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1Mb2JyeSAgSi5SLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MzI2LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW1tZXRyeSBvYnNlcnZhdGlvbnMgaW4gbG9uZyBudWNsZW90aWRlIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPVByYWJodSAgVi5WLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yNzk3LTgwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjb3VudGluZyB1bml0cyBpbiBETkE7IGNpdGF0aW9uX2F1dGhvcj1CZWxsICBTLkouOyBjaXRhdGlvbl9hdXRob3I9Rm9yc2R5a2UgIEQuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFRoZW9yLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRlc3Qgb2YgQ2hhcmdhZmbigJlzIHNlY29uZCBydWxlOyBjaXRhdGlvbl9hdXRob3I9TWl0Y2hlbGwgIEQuOyBjaXRhdGlvbl9hdXRob3I9QnJpZGdlICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQwOyAgY2l0YXRpb25fcGFnZXM9OTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpYyBzdWJzdGl0dXRpb24gcGF0dGVybnMgaW4gdGhlIHR3byBETkEgc3RyYW5kcyBvZiBiYWN0ZXJpYTsgY2l0YXRpb25fYXV0aG9yPUxvYnJ5ICBKLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpYyBkaXJlY3Rpb25hbCBtdXRhdGlvbiBwcmVzc3VyZXMgaW4gYmFjdGVyaWE7IGNpdGF0aW9uX2F1dGhvcj1Mb2JyeSAgSi5SLjsgY2l0YXRpb25fYXV0aG9yPVN1ZW9rYSAgTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1SRVNFQVJDSDAwNTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vbml2ZXJzYWwgcmVwbGljYXRpb24gYmlhc2VzIGluIGJhY3RlcmlhOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuUC47IGNpdGF0aW9uX2F1dGhvcj1EYW5jaGluICBBLjsgY2l0YXRpb25fYXV0aG9yPVZpYXJpICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhbmQgY29tcG9zaXRpb25hbCBhc3ltbWV0cnkgaW4gYmFjdGVyaWFsIGFuZCBsYXJnZSB2aXJhbCBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9TXLDoXplayAgSi47IGNpdGF0aW9uX2F1dGhvcj1LYXJsaW4gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTM3MjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdHVkeSBvbiB0aGUgY29ycmVsYXRpb24gb2YgbnVjbGVvdGlkZSBza2V3cyBhbmQgdGhlIHBvc2l0aW9uaW5nIG9mIHRoZSBvcmlnaW4gb2YgcmVwbGljYXRpb246IGRpZmZlcmVudCBtb2RlcyBvZiByZXBsaWNhdGlvbiBpbiBiYWN0ZXJpYWwgc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPU5pa29sYW91ICBDLjsgY2l0YXRpb25fYXV0aG9yPUFsbWlyYW50aXMgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY4MTYtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb250cmlidXRpb25zIG9mIHJlcGxpY2F0aW9uIG9yaWVudGF0aW9uLCBnZW5lIGRpcmVjdGlvbiwgYW5kIHNpZ25hbCBzZXF1ZW5jZXMgdG8gYmFzZS1jb21wb3NpdGlvbiBhc3ltbWV0cmllcyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPVRpbGxpZXIgIEUuUi47IGNpdGF0aW9uX2F1dGhvcj1Db2xsaW5zICBSLiBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MjQ5LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1pbGFyIGNvbXBvc2l0aW9uYWwgYmlhc2VzIGFyZSBjYXVzZWQgYnkgdmVyeSBkaWZmZXJlbnQgbXV0YXRpb25hbCBlZmZlY3RzOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuUC5DLjsgY2l0YXRpb25fYXV0aG9yPVRvdWNob24gIE0uOyBjaXRhdGlvbl9hdXRob3I9RmVpbCAgRS5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE1MzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcGFyYXRpbmcgdGhlIGVmZmVjdHMgb2YgbXV0YXRpb24gYW5kIHNlbGVjdGlvbiBpbiBwcm9kdWNpbmcgRE5BIHNrZXcgaW4gYmFjdGVyaWFsIGNocm9tb3NvbWVzOyBjaXRhdGlvbl9hdXRob3I9TW9ydG9uICBSLkEuOyBjaXRhdGlvbl9hdXRob3I9TW9ydG9uICBCLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWMgc3Vic3RpdHV0aW9uIHBhdHRlcm5zOiBhIHJldmlldyBvZiBwb3NzaWJsZSB1bmRlcmx5aW5nIG11dGF0aW9uYWwgb3Igc2VsZWN0aXZlIG1lY2hhbmlzbXM7IGNpdGF0aW9uX2F1dGhvcj1GcmFuayAgQS5DLjsgY2l0YXRpb25fYXV0aG9yPUxvYnJ5ICBKLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzODsgIGNpdGF0aW9uX3BhZ2VzPTY1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdHlwaWNhbCBBVCBza2V3IGluIEZpcm1pY3V0ZSBnZW5vbWVzIHJlc3VsdHMgZnJvbSBzZWxlY3Rpb24gYW5kIG5vdCBmcm9tIG11dGF0aW9uOyBjaXRhdGlvbl9hdXRob3I9Q2hhcm5lc2tpICBDLkEuOyBjaXRhdGlvbl9hdXRob3I9SG9udGkgIEYuOyBjaXRhdGlvbl9hdXRob3I9QnJ5YW50ICBKLk0uOyBjaXRhdGlvbl9hdXRob3I9SHVyc3QgIEwuRC47IGNpdGF0aW9uX2F1dGhvcj1GZWlsICBFLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjI4MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmlhdGlvbnMgZnJvbSBDaGFyZ2FmZuKAmXMgc2Vjb25kIHBhcml0eSBydWxlIGluIG9yZ2FuZWxsYXIgRE5BIEluc2lnaHRzIGludG8gdGhlIGV2b2x1dGlvbiBvZiBvcmdhbmVsbGFyIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1OaWtvbGFvdSAgQy47IGNpdGF0aW9uX2F1dGhvcj1BbG1pcmFudGlzICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODE7ICBjaXRhdGlvbl9wYWdlcz0zNC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG9yZ2FuaXphdGlvbiBvZiB0aGUgYmFjdGVyaWFsIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPVJvY2hhICBFLlAuQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0yMTEtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cm9uZyBhc3ltbWV0cmljIG11dGF0aW9uIGJpYXMgaW4gZW5kb3N5bWJpb250IGdlbm9tZXMgY29pbmNpZGUgd2l0aCBsb3NzIG9mIGdlbmVzIGZvciByZXBsaWNhdGlvbiByZXN0YXJ0IHBhdGh3YXlzOyBjaXRhdGlvbl9hdXRob3I9S2xhc3NvbiAgTC47IGNpdGF0aW9uX2F1dGhvcj1BbmRlcnNzb24gIFMuRy5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTAzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWVzIGFuZCBldm9sdXRpb247IGNpdGF0aW9uX2F1dGhvcj1EYW5jaGluICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gSXNzdWVzIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzctNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uZ29pbmcgZXZvbHV0aW9uIG9mIHN0cmFuZCBjb21wb3NpdGlvbiBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPVJvY2hhICBFLlAuOyBjaXRhdGlvbl9hdXRob3I9RGFuY2hpbiAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE3ODktOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IG1ldGhvZCBmb3IgYXNzZXNzaW5nIHRoZSBlZmZlY3Qgb2YgcmVwbGljYXRpb24gb24gRE5BIGJhc2UgY29tcG9zaXRpb24gYXN5bW1ldHJ5OyBjaXRhdGlvbl9hdXRob3I9TmVjxZ91bGVhICBBLjsgY2l0YXRpb25fYXV0aG9yPUxvYnJ5ICBKLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yMTY5LTc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gb2YgcmVwbGljYXRpb24gaW4gY2lyY3VsYXIgcHJva2FyeW90aWMgY2hyb21vc29tZXM7IGNpdGF0aW9uX2F1dGhvcj1Xb3JuaW5nICBQLjsgY2l0YXRpb25fYXV0aG9yPUplbnNlbiAgTC5KLjsgY2l0YXRpb25fYXV0aG9yPUhhbGxpbiAgUC5GLjsgY2l0YXRpb25fYXV0aG9yPVN0YWVyZmVsZHQgIEguLUguOyBjaXRhdGlvbl9hdXRob3I9VXNzZXJ5ICBELlcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM1My02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2aWF0aW9ucyBmcm9tIENoYXJnYWZm4oCZcyBzZWNvbmQgcGFyaXR5IHJ1bGUgY29ycmVsYXRlIHdpdGggZGlyZWN0aW9uIG9mIHRyYW5zY3JpcHRpb247IGNpdGF0aW9uX2F1dGhvcj1CZWxsICBTLkouOyBjaXRhdGlvbl9hdXRob3I9Rm9yc2R5a2UgIEQuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFRoZW9yLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTYzLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wb3NpdGlvbiBzdHJhbmQgYXN5bW1ldHJpZXMgaW4gcHJva2FyeW90aWMgZ2Vub21lczogbXV0YXRpb25hbCBiaWFzIGFuZCBiaWFzZWQgZ2VuZSBvcmllbnRhdGlvbi4gQy47IGNpdGF0aW9uX2F1dGhvcj1Mb3BleiAgUC47IGNpdGF0aW9uX2F1dGhvcj1QaGlsaXBwZSAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVIuIEFjYWQuIFNjaS4gSUlJOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTIwMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3ltbWV0cmllcyBnZW5lcmF0ZWQgYnkgdHJhbnNjcmlwdGlvbi1jb3VwbGVkIHJlcGFpciBpbiBlbnRlcm9iYWN0ZXJpYWwgZ2VuZXM7IGNpdGF0aW9uX2F1dGhvcj1GcmFuY2lubyAgTS5QLjsgY2l0YXRpb25fYXV0aG9yPUNoYW8gIEwuOyBjaXRhdGlvbl9hdXRob3I9UmlsZXkgIE0uQS47IGNpdGF0aW9uX2F1dGhvcj1PY2htYW4gIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM6IGEgcHJvcG9zYWwgZm9yIGEgc3lub255bW91cyBjb2RvbiBjaG9pY2UgdGhhdCBpcyBvcHRpbWFsIGZvciB0aGUgRS4gY29saSB0cmFuc2xhdGlvbmFsIHN5c3RlbTsgY2l0YXRpb25fYXV0aG9yPUlrZW11cmEgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gYmFjdGVyaWE6IGNvcnJlbGF0aW9uIHdpdGggZ2VuZSBleHByZXNzaXZpdHk7IGNpdGF0aW9uX2F1dGhvcj1Hb3V5ICBNLjsgY2l0YXRpb25fYXV0aG9yPUdhdXRpZXIgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTcwNTUtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZWxlY3Rpb24tbXV0YXRpb24tZHJpZnQgdGhlb3J5IG9mIHN5bm9ueW1vdXMgY29kb24gdXNhZ2U7IGNpdGF0aW9uX2F1dGhvcj1CdWxtZXIgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjk7ICBjaXRhdGlvbl9wYWdlcz04OTctOTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgb3B0aW1pemVkIGlzIHRoZSB0cmFuc2xhdGlvbmFsIG1hY2hpbmVyeSBpbiBFc2NoZXJpY2hpYSBjb2xpLCBTYWxtb25lbGxhIHR5cGhpbXVyaXVtIGFuZCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU/OyBjaXRhdGlvbl9hdXRob3I9WGlhICBYLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTQ5OyAgY2l0YXRpb25fcGFnZXM9MzctNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgY29ycmVsYXRpb24gYmV0d2VlbiBkZW94eXJpYm9udWNsZWljIGFjaWQgYW5kIHByb3RlaW47IGNpdGF0aW9uX2F1dGhvcj1TdWVva2EgIE4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db2xkIFNwcmluZyBIYXJiLiBTeW1wLiBRdWFudC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk2MTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTM1LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29ycmVsYXRpb24gb2YgcHJvdGVpbiBoeWRyb3BhdGh5IHdpdGggdGhlIGJhc2UgY29tcG9zaXRpb24gb2YgY29kaW5nIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPUTigJlPbm9mcmlvICBHLjsgY2l0YXRpb25fYXV0aG9yPUphYmJhcmkgIEsuOyBjaXRhdGlvbl9hdXRob3I9TXVzdG8gIEguOyBjaXRhdGlvbl9hdXRob3I9QmVybmFyZGkgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzODsgIGNpdGF0aW9uX3BhZ2VzPTMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNrZXdlZCBvbGlnb21lcnMgYW5kIG9yaWdpbnMgb2YgcmVwbGljYXRpb247IGNpdGF0aW9uX2F1dGhvcj1TYWx6YmVyZyAgUy5MLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBBLkouOyBjaXRhdGlvbl9hdXRob3I9S2VybGF2YWdlICBBLlIuOyBjaXRhdGlvbl9hdXRob3I9VG9tYiAgSi5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMTc7ICBjaXRhdGlvbl9wYWdlcz01Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgbnVjbGVvdGlkZSBhc3ltbWV0cmllcyBpbiBwbGFudCBhbmQgYW5pbWFsIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1NYXNjaGVyICBNLjsgY2l0YXRpb25fYXV0aG9yPVNjaHViZXJ0ICBJLjsgY2l0YXRpb25fYXV0aG9yPVNjaG9seiAgVS47IGNpdGF0aW9uX2F1dGhvcj1GcmllZGVsICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qmlvc3lzdGVtczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0xODEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGRpbnVjbGVvdGlkZSBzaWduYXR1cmVzIGFuZCBhbmFseXNpcyBvZiBnZW5vbWljIGhldGVyb2dlbmVpdHk7IGNpdGF0aW9uX2F1dGhvcj1LYXJsaW4gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTU5OC02MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuenltYXRpYyBzeW50aGVzaXMgb2YgZGVveHlyaWJvbnVjbGVpYyBhY2lkLiBWSUlJLiBGcmVxdWVuY2llcyBvZiBuZWFyZXN0IG5laWdoYm9yIGJhc2Ugc2VxdWVuY2VzIGluIGRlb3h5cmlib251Y2xlaWMgYWNpZDsgY2l0YXRpb25fYXV0aG9yPUpvc3NlICBKLjsgY2l0YXRpb25fYXV0aG9yPUthaXNlciAgQS5ELjsgY2l0YXRpb25fYXV0aG9yPUtvcm5iZXJnICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk2MTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzY7ICBjaXRhdGlvbl9wYWdlcz04NjQtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5lYXJlc3QgbmVpZ2hib3IgbnVjbGVvdGlkZSBwYXR0ZXJucy4gU3RydWN0dXJhbCBhbmQgYmlvbG9naWNhbCBpbXBsaWNhdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1OdXNzaW5vdiAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2wuIENoZW07ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9MjU2OyAgY2l0YXRpb25fcGFnZXM9ODQ1OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Ryb25nIGRvdWJsZXQgcHJlZmVyZW5jZXMgaW4gbnVjbGVvdGlkZSBzZXF1ZW5jZXMgYW5kIEROQSBnZW9tZXRyeTsgY2l0YXRpb25fYXV0aG9yPU51c3Npbm92ICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTExLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNrZXcgb2YgbW9ub251Y2xlb3RpZGUgZnJlcXVlbmNpZXMsIHJlbGF0aXZlIGFidW5kYW5jZSBvZiBkaW51Y2xlb3RpZGVzLCBhbmQgRE5BIHN0cmFuZCBhc3ltbWV0cnk7IGNpdGF0aW9uX2F1dGhvcj1TaGlvaXJpICBDLjsgY2l0YXRpb25fYXV0aG9yPVRha2FoYXRhICBOLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MzY0LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaHkgYXJlIGNvbXBsZW1lbnRhcnkgRE5BIHN0cmFuZHMgc3ltbWV0cmljPzsgY2l0YXRpb25fYXV0aG9yPUJhaXNuw6llICBQLi1GLjsgY2l0YXRpb25fYXV0aG9yPUhhbXBzb24gIFMuOyBjaXRhdGlvbl9hdXRob3I9QmFsZGkgIFAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEwMjEtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvcmlDIDUuMDogYW4gdXBkYXRlZCBkYXRhYmFzZSBvZiBvcmlDIHJlZ2lvbnMgaW4gYm90aCBiYWN0ZXJpYWwgYW5kIGFyY2hhZWFsIGdlbm9tZXM7IGNpdGF0aW9uX2F1dGhvcj1HYW8gIEYuOyBjaXRhdGlvbl9hdXRob3I9THVvICBILjsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBDLi1ULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1EOTAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBlcmNlbnRhZ2Ugb2YgYmFjdGVyaWFsIGdlbmVzIG9uIGxlYWRpbmcgdmVyc3VzIGxhZ2dpbmcgc3RyYW5kcyBpcyBpbmZsdWVuY2VkIGJ5IG11bHRpcGxlIGJhbGFuY2luZyBmb3JjZXM7IGNpdGF0aW9uX2F1dGhvcj1NYW8gIFguOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIEguOyBjaXRhdGlvbl9hdXRob3I9WWluICBZLjsgY2l0YXRpb25fYXV0aG9yPVh1ICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz04MjEwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc29jaWF0aW9uIG9mIHB1cmluZSBhc3ltbWV0cnksIHN0cmFuZC1iaWFzZWQgZ2VuZSBkaXN0cmlidXRpb24gYW5kIFBvbEMgd2l0aGluIEZpcm1pY3V0ZXMgYW5kIGJleW9uZDogYSBuZXcgYXBwcmFpc2FsOyBjaXRhdGlvbl9hdXRob3I9U2FoYSAgUy5LLjsgY2l0YXRpb25fYXV0aG9yPUdvc3dhbWkgIEEuOyBjaXRhdGlvbl9hdXRob3I9RHV0dGEgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz00MzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UZXN0aW5nIGFuZCBkYXRpbmcgb2Ygc3RydWN0dXJhbCBjaGFuZ2VzIGluIHByYWN0aWNlOyBjaXRhdGlvbl9hdXRob3I9WmVpbGVpcyAgQS47IGNpdGF0aW9uX2F1dGhvcj1LbGVpYmVyICBDLjsgY2l0YXRpb25fYXV0aG9yPUtyw6RtZXIgIFcuOyBjaXRhdGlvbl9hdXRob3I9SG9ybmlrICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0LiBTdGF0LiBEYXRhIEFuYWw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz0xMDktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlc3RpbmcsIG1vbml0b3JpbmcsIGFuZCBkYXRpbmcgc3RydWN0dXJhbCBjaGFuZ2VzIGluIGV4Y2hhbmdlIHJhdGUgcmVnaW1lczsgY2l0YXRpb25fYXV0aG9yPVplaWxlaXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9U2hhaCAgQS47IGNpdGF0aW9uX2F1dGhvcj1QYXRuYWlrICBJLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29tcHV0LiBTdGF0LiBEYXRhIEFuYWw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xNjk2LTcwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9c3RydWNjaGFuZ2UgdCBhbiBSIHBhY2thZ2UgZm9yIHRlc3RpbmcgZm9yIHN0cnVjdHVyYWwgY2hhbmdlIGluIGxpbmVhciByZWdyZXNzaW9uIG1vZGVsczsgY2l0YXRpb25fYXV0aG9yPVplaWxlaXMgIEEuOyBjaXRhdGlvbl9hdXRob3I9TGVpc2NoICBGLjsgY2l0YXRpb25fYXV0aG9yPUhvcm5payAgSy47IGNpdGF0aW9uX2F1dGhvcj1LbGVpYmVyICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gU3RhdC4gU29mdHc7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIGluZm9ybWF0aW9uIGFuZCBzdWZmaWNpZW5jeTsgY2l0YXRpb25fYXV0aG9yPUt1bGxiYWNrICBTLjsgY2l0YXRpb25fYXV0aG9yPUxlaWJsZXIgIFIuQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gTWF0aC4gU3RhdDsgIGNpdGF0aW9uX3llYXI9MTk1MTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTc5LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGFsZ29yaXRobSBhbmQgd2ViLWJhc2VkIHRvb2wgZm9yIGNvbXBhcmluZyB0d28gYWx0ZXJuYXRpdmUgcGh5bG9nZW5ldGljIHRyZWVzOyBjaXRhdGlvbl9hdXRob3I9TnllICBULk0uVy47IGNpdGF0aW9uX2F1dGhvcj1MacOyICBQLjsgY2l0YXRpb25fYXV0aG9yPUdpbGtzICBXLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTExNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTkNCSSBUYXhvbm9teSBkYXRhYmFzZTsgY2l0YXRpb25fYXV0aG9yPUZlZGVyaGVuICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMTM2LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaW51Y2xlb3RpZGUgcmVsYXRpdmUgYWJ1bmRhbmNlIGV4dHJlbWVzOiBhIGdlbm9taWMgc2lnbmF0dXJlOyBjaXRhdGlvbl9hdXRob3I9S2FybGluICBTLjsgY2l0YXRpb25fYXV0aG9yPUJ1cmdlICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjgzLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2lnbmF0dXJlIGNvbXBhcmlzb25zIGFtb25nIHByb2thcnlvdGUsIHBsYXNtaWQsIGFuZCBtaXRvY2hvbmRyaWFsIEROQTsgY2l0YXRpb25fYXV0aG9yPUNhbXBiZWxsICBBLjsgY2l0YXRpb25fYXV0aG9yPU1yw6F6ZWsgIEouOyBjaXRhdGlvbl9hdXRob3I9S2FybGluICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9OTE4NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1lZ2dOT0cgdjQuMDogbmVzdGVkIG9ydGhvbG9neSBpbmZlcmVuY2UgYWNyb3NzIDM2ODYgb3JnYW5pc21zOyBjaXRhdGlvbl9hdXRob3I9UG93ZWxsICBTLjsgY2l0YXRpb25fYXV0aG9yPUZvcnNsdW5kICBLLjsgY2l0YXRpb25fYXV0aG9yPVN6a2xhcmN6eWsgIEQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQyMzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIG1lYXN1cmVtZW50IG9mIG5laWdoYm9yLWRlcGVuZGVudCBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiBwcm9jZXNzZXM7IGNpdGF0aW9uX2F1dGhvcj1Bcm5kdCAgUC5GLms7IGNpdGF0aW9uX2F1dGhvcj1Id2EgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIzMjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltaWxhciByYXRlcyBidXQgZGlmZmVyZW50IG1vZGVzIG9mIHNlcXVlbmNlIGV2b2x1dGlvbiBpbiBpbnRyb25zIGFuZCBhdCBleG9uaWMgc2lsZW50IHNpdGVzIGluIHJvZGVudHM6IGV2aWRlbmNlIGZvciBzZWxlY3RpdmVseSBkcml2ZW4gY29kb24gdXNhZ2U7IGNpdGF0aW9uX2F1dGhvcj1DaGFtYXJ5ICBKLi1WLjsgY2l0YXRpb25fYXV0aG9yPUh1cnN0ICBMLiBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTAxNC0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0VHRy1QQVRIOiBLeW90byBlbmN5Y2xvcGVkaWEgb2YgZ2VuZXMgYW5kIGdlbm9tZXMtYmFzZWQgcGF0aHdheSBhbmFseXNpcyB1c2luZyBhIHBhdGggYW5hbHlzaXMgbW9kZWw7IGNpdGF0aW9uX2F1dGhvcj1EdSAgSi47IGNpdGF0aW9uX2F1dGhvcj1ZdWFuICBaLjsgY2l0YXRpb25fYXV0aG9yPU1hICBaLjsgY2l0YXRpb25fYXV0aG9yPVNvbmcgIEouOyBjaXRhdGlvbl9hdXRob3I9WGllICBYLjsgY2l0YXRpb25fYXV0aG9yPUNoZW4gIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb3N5c3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yNDQxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5emluZyBnZW5vbWVzIHdpdGggY3VtdWxhdGl2ZSBza2V3IGRpYWdyYW1zOyBjaXRhdGlvbl9hdXRob3I9R3JpZ29yaWV2ICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0yMjg2LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhbmQgYXN5bW1ldHJpZXMgaW4gRE5BIGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPUZyYW5jaW5vICBNLlAuOyBjaXRhdGlvbl9hdXRob3I9T2NobWFuICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjQwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBvc2l0aW9uYWwgZGlmZmVyZW5jZXMgd2l0aGluIGFuZCBiZXR3ZWVuIGV1a2FyeW90aWMgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUthcmxpbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1NcsOhemVrICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDIyNy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBCZWNrZXIgUi5BLiAgQ2hhbWJlcnMgSi5NLiAgV2lsa3MgQS4gUi4gICAxOTg4ICwgVGhlIG5ldyBTIGxhbmd1YWdlOiBhIHByb2dyYW1taW5nIGVudmlyb25tZW50IGZvciBkYXRhIGFuYWx5c2lzIGFuZCBncmFwaGljcy4gUGFjaWZpYyBHcm92ZSwgQ0EsIFVTQTogV2Fkc3dvcnRoICZhbXA7IEJyb29rcy9Db2xlIEFkdmFuY2VkIEJvb2tzICZhbXA7IFNvZnR3YXJlLiBJU0JOIDAtNTM0LTA5MTkyLVguICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIG1vZGVsIGJhc2VkIG9uIG11dGF0aW9uIGFuZCBzZWxlY3Rpb24gZXhwbGFpbnMgdHJlbmRzIGluIGNvZG9uIGFuZCBhbWluby1hY2lkIHVzYWdlIGFuZCBHQyBjb21wb3NpdGlvbiB3aXRoaW4gYW5kIGFjcm9zcyBnZW5vbWVzOyBjaXRhdGlvbl9hdXRob3I9S25pZ2h0ICBSLkQuOyBjaXRhdGlvbl9hdXRob3I9RnJlZWxhbmQgIFMuSi47IGNpdGF0aW9uX2F1dGhvcj1MYW5kd2ViZXIgIEwuRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1SRVNFQVJDSDAwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ3VhbmluZSBhbmQgY3l0b3NpbmUgY29udGVudCBvZiBnZW5vbWljIEROQSBhbmQgYmFjdGVyaWFsIGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPU11dG8gIEEuOyBjaXRhdGlvbl9hdXRob3I9T3Nhd2EgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wb3NpdGlvbmFsIGR5bmFtaWNzIG9mIGd1YW5pbmUgYW5kIGN5dG9zaW5lIGNvbnRlbnQgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUh1ICBKLjsgY2l0YXRpb25fYXV0aG9yPVpoYW8gIFguOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIFouOyBjaXRhdGlvbl9hdXRob3I9WXUgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SZXMuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz0zNjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGV2b2x1dGlvbiBpcyBnb3Zlcm5lZCBieSBsaW5lYXIgZm9ybXVsYXM7IGNpdGF0aW9uX2F1dGhvcj1Tb3JpbWFjaGkgIEsuOyBjaXRhdGlvbl9hdXRob3I9T2theWFzdSAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtaW5vIEFjaWRzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NjYxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2VuZXJhbCBtb2RlbCBvZiBjb2RvbiBiaWFzIGR1ZSB0byBHQyBtdXRhdGlvbmFsIGJpYXM7IGNpdGF0aW9uX2F1dGhvcj1QYWxpZHdvciAgRy5BLjsgY2l0YXRpb25fYXV0aG9yPVBlcmtpbnMgIFQuSi47IGNpdGF0aW9uX2F1dGhvcj1YaWEgIFguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEzNDMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb250ZW50LWNlbnRyaWMgb3JnYW5pemF0aW9uIG9mIHRoZSBnZW5ldGljIGNvZGU7IGNpdGF0aW9uX2F1dGhvcj1ZdSAgSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzIFByb3Rlb21pY3MgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWxpbmcgY29tcG9zaXRpb25hbCBkeW5hbWljcyBiYXNlZCBvbiBHQyBhbmQgcHVyaW5lIGNvbnRlbnRzIG9mIHByb3RlaW4tY29kaW5nIHNlcXVlbmNlczsgY2l0YXRpb25fYXV0aG9yPVpoYW5nICBaLjsgY2l0YXRpb25fYXV0aG9yPVl1ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gRGlyZWN0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz02MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzY2hlcmljaGlhIGNvbGkgbWZkIG11dGFudCBkZWZpY2llbnQgaW4g4oCcbXV0YXRpb24gZnJlcXVlbmN5IGRlY2xpbmXigJ0gbGFja3Mgc3RyYW5kLXNwZWNpZmljIHJlcGFpcjogaW4gdml0cm8gY29tcGxlbWVudGF0aW9uIHdpdGggcHVyaWZpZWQgY291cGxpbmcgZmFjdG9yOyBjaXRhdGlvbl9hdXRob3I9U2VsYnkgIEMuUC47IGNpdGF0aW9uX2F1dGhvcj1XaXRraW4gIEUuTS47IGNpdGF0aW9uX2F1dGhvcj1TYW5jYXIgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT04ODsgIGNpdGF0aW9uX3BhZ2VzPTExNTc0LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCByZXBhaXIgb2YgcHJlbXV0YXRpb25hbCBVVi1sZXNpb25zIGF0IHRSTkEgZ2VuZXMgaW4gRS4gY29saTsgY2l0YXRpb25fYXV0aG9yPUJvY2tyYXRoICBSLkMuOyBjaXRhdGlvbl9hdXRob3I9UGFsbWVyICBKLiBFLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW4uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTEzMy00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbi1yZXBhaXIgY291cGxpbmcgZGV0ZXJtaW5lcyB0aGUgc3RyYW5kZWRuZXNzIG9mIHVsdHJhdmlvbGV0IG11dGFnZW5lc2lzIGluIEVzY2hlcmljaGlhIGNvbGk7IGNpdGF0aW9uX2F1dGhvcj1PbGxlciAgQS5SLjsgY2l0YXRpb25fYXV0aG9yPUZpamFsa293c2thICBJLkouOyBjaXRhdGlvbl9hdXRob3I9RHVubiAgUi5MLjsgY2l0YXRpb25fYXV0aG9yPVNjaGFhcGVyICBSLk0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTExMDM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgcG9seW1lcmFzZSBiZXR3ZWVuIGxlc2lvbiBieXBhc3MgYW5kIEROQSByZXBhaXI7IGNpdGF0aW9uX2F1dGhvcj1EZWFjb25lc2N1ICBBLk0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsLiBNb2wuIExpZmUgU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTcwOyAgY2l0YXRpb25fcGFnZXM9NDQ5NS01MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjY2VsZXJhdGVkIGRlYW1pbmF0aW9uIG9mIGN5dG9zaW5lIHJlc2lkdWVzIGluIFVWLWluZHVjZWQgY3ljbG9idXRhbmUgcHlyaW1pZGluZSBkaW1lcnMgbGVhZHMgdG8gQ0PigJMmZ3Q7VFQgdHJhbnNpdGlvbnM7IGNpdGF0aW9uX2F1dGhvcj1QZW5nICBXLjsgY2l0YXRpb25fYXV0aG9yPVNoYXcgIEIuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEwMTcyLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBldWJhY3RlcmlhbCBETkEgcG9seW1lcmFzZSBJSUkgYWxwaGEgc3VidW5pdHM7IGNpdGF0aW9uX2F1dGhvcj1aaGFvICBYLi1RLjsgY2l0YXRpb25fYXV0aG9yPUh1ICBKLi1GLjsgY2l0YXRpb25fYXV0aG9yPVl1ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3MgUHJvdGVvbWljcyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JcyB0aGVyZSBhIHJvbGUgZm9yIHJlcGxpY2F0aW9uIGZvcmsgYXN5bW1ldHJ5IGluIHRoZSBkaXN0cmlidXRpb24gb2YgZ2VuZXMgaW4gYmFjdGVyaWFsIGdlbm9tZXM/OyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIEUuUC5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTM5My01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gZXNzZW50aWFsIEROQSBwb2x5bWVyYXNlcyBhdCB0aGUgYmFjdGVyaWFsIHJlcGxpY2F0aW9uIGZvcms7IGNpdGF0aW9uX2F1dGhvcj1EZXJ2eW4gIEUuOyBjaXRhdGlvbl9hdXRob3I9U3Vza2kgIEMuOyBjaXRhdGlvbl9hdXRob3I9RGFuaWVsICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzE2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURuYUUyIHBvbHltZXJhc2UgY29udHJpYnV0ZXMgdG8gaW4gdml2byBzdXJ2aXZhbCBhbmQgdGhlIGVtZXJnZW5jZSBvZiBkcnVnIHJlc2lzdGFuY2UgaW4gTXljb2JhY3Rlcml1bSB0dWJlcmN1bG9zaXM7IGNpdGF0aW9uX2F1dGhvcj1Cb3Nob2ZmICBILkkuTS47IGNpdGF0aW9uX2F1dGhvcj1SZWVkICBNLkIuOyBjaXRhdGlvbl9hdXRob3I9QmFycnkgIEMuRS47IGNpdGF0aW9uX2F1dGhvcj1NaXpyYWhpICBWLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0xODMtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIFNPUy1yZWd1bGF0ZWQgb3Blcm9uIGludm9sdmVkIGluIGRhbWFnZS1pbmR1Y2libGUgbXV0YWdlbmVzaXMgaW4gQ2F1bG9iYWN0ZXIgY3Jlc2NlbnR1czsgY2l0YXRpb25fYXV0aG9yPUdhbGhhcmRvICBSLlMuOyBjaXRhdGlvbl9hdXRob3I9Um9jaGEgIFIuUC47IGNpdGF0aW9uX2F1dGhvcj1NYXJxdWVzICBNLlY7IGNpdGF0aW9uX2F1dGhvcj1NZW5jayAgQy5GLk0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MDMtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy80LzM1My8xNzQ2MzA3Ni9kc3cwMjEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBTdHJhbmQgYmlhc2VzIHJlZmxlY3QgZGV2aWF0aW9ucyBmcm9tIGEgbnVsbCBleHBlY3RhdGlvbiBvZiBETkEgZXZvbHV0aW9uIHRoYXQgYXNzdW1lcyBzdHJhbmQtc3ltbWV0cmljIHN1YnN0aXR1dGlvbiByYXRlcy4gSGVyZSwgd2UgcHJlc2VudCBzdHJvbmcgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDY5OTY4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZMI0KCzhJkwAOWaOS5szTTR2g5PU" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f353%2f2469968" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f353%2f2469968" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198878">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198878">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198881">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198881">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198895">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198895">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198932">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198932">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198995">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198995">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199007">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199007">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199011">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199011">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199018">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199018">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/339/2469970">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/4/365/2469965">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Konstantinos Apostolou-Karampelis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw021-cor1" data-open="dsw021-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel: +302106503601. Email: <a href="mailto:apostolou@bio.demokritos.gr" target="_blank">apostolou@bio.demokritos.gr</a> (K.A.K.); Email: <a href="mailto:yalmir@bio.demokritos.gr" target="_blank">yalmir@bio.demokritos.gr</a> (Y.A.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Konstantinos+Apostolou-Karampelis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Apostolou-Karampelis K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Apostolou-Karampelis K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Konstantinos Apostolou-Karampelis</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christoforos Nikolaou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div> Computational Genomics Group, Department of Biology, University of Crete, 71409 Heraklion, Greece </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christoforos+Nikolaou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nikolaou C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nikolaou C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christoforos Nikolaou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yannis Almirantis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Institute of Biosciences and Applications, National Center for Scientific Research “Demokritos”, 15310 Athens, Greece </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw021-cor1" data-open="dsw021-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel: +302106503601. Email: <a href="mailto:apostolou@bio.demokritos.gr" target="_blank">apostolou@bio.demokritos.gr</a> (K.A.K.); Email: <a href="mailto:yalmir@bio.demokritos.gr" target="_blank">yalmir@bio.demokritos.gr</a> (Y.A.) 
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yannis+Almirantis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Almirantis Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Almirantis Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yannis Almirantis</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw021-FM1"><div class="fn"><div class="footnote-content">Edited by Dr Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 353–363, <a href='https://doi.org/10.1093/dnares/dsw021'>https://doi.org/10.1093/dnares/dsw021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 June 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 February 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">09 May 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469968" href="/dnaresearch/article-pdf/23/4/353/17463076/dsw021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Konstantinos Apostolou-Karampelis, Christoforos Nikolaou, Yannis Almirantis;  A novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms, <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 353–363, <a href='https://doi.org/10.1093/dnares/dsw021'>https://doi.org/10.1093/dnares/dsw021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469968?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20novel%20skew%20analysis%20reveals%20substitution%20asymmetries%20linked%20to%20genetic%20code%20GC-biases%20and%20PolIII%20a-subunit%20isoforms&amp;publicationDate=2016-06-26&amp;volumeNum=23&amp;issueNum=4&amp;author=Apostolou-Karampelis%2C%20Konstantinos%3B%20Nikolaou%2C%20Christoforos&amp;startPage=353&amp;endPage=363&amp;contentId=10.1093%2Fdnares%2Fdsw021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f353%2f2469968" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87198878 id="87198878" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> Strand biases reflect deviations from a null expectation of DNA evolution that assumes strand-symmetric substitution rates. Here, we present strong evidence that nearest-neighbour preferences are a strand-biased feature of bacterial genomes, indicating neighbour-dependent substitution asymmetries. To detect such asymmetries we introduce an alignment free index (relative abundance skews). The profiles of relative abundance skews along coding sequences can trace the phylogenetic relations of bacteria, suggesting that the patterns of neighbour-dependent substitution strand-biases are not common among different lineages, but are rather species-specific. Analysis of neighbour-dependent and codon-site skews sheds light on the origins of substitution asymmetries. Via a simple model we argue that the structure of the genetic code imposes position-dependent substitution strand-biases along coding sequences, as a response to GC mutation pressure. Thus, the organization of the genetic code <em>per se</em> can lead to an uneven distribution of nucleotides among different codon sites, even when requirements for specific codons and amino-acids are not accounted for. Moreover, our results suggest that strand-biases in replication fidelity of PolIII α-subunit induce substitution asymmetries, both neighbour-dependent and independent, on a genome scale. The role of DNA repair systems, such as transcription-coupled repair, is also considered. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Chargaff’s second parity rule (PR2)&quot;">Chargaff’s second parity rule (PR2)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;dinucleotide relative abundances (odds ratios)&quot;">dinucleotide relative abundances (odds ratios)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;GC mutational pressure&quot;">GC mutational pressure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;substitution strand-biases&quot;">substitution strand-biases</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;PolIII a-subunit isoforms&quot;">PolIII a-subunit isoforms</a></div>                            <h2 scrollto-destination=87198881 id="87198881" class="section-title" >1. Introduction</h2>
<p> The evolution of genomic base composition depends, in a complex fashion, on mutation and selection. <a reveal-id="dsw021-B1" data-open="dsw021-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> When there is no strand-specific bias in mutation rates and selection pressure, the interstrand base-pairing rule reduces the maximum number of substitution rates to six: r <sub>A→T </sub> = <sub> </sub> r <sub>T→A</sub> , r <sub>G→C </sub> = <sub> </sub> r <sub>C→G</sub> , r <sub>A→G </sub> = <sub> </sub> r <sub>T→C</sub> , r <sub>G→A </sub> = <sub> </sub> r <sub>C→T</sub> , r <sub>A→C </sub> = <sub> </sub> r <sub>T→G</sub> , and r <sub>C→A </sub> = <sub> </sub> r <sub>G→T</sub> , as stated by parity rule 1 (PR1). <a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Under PR1, the average intrastrand base composition of a chromosome is expected to reach an equilibrium state, at which [A] = [T] and [G] = [C], a relationship known as PR2, <a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw021-B3" data-open="dsw021-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> also referred to as Chargaff’s second parity rule. </p><p> Most of bacteria abide by PR2 on a chromosome-wide scale. <a reveal-id="dsw021-B4 dsw021-B5 dsw021-B6" data-open="dsw021-B4 dsw021-B5 dsw021-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> Nonetheless, they display distinctive local deviations from this parity, as shown by previous studies. <a reveal-id="dsw021-B7 dsw021-B8 dsw021-B9 dsw021-B10 dsw021-B11 dsw021-B12 dsw021-B13 dsw021-B14 dsw021-B15 dsw021-B16" data-open="dsw021-B7 dsw021-B8 dsw021-B9 dsw021-B10 dsw021-B11 dsw021-B12 dsw021-B13 dsw021-B14 dsw021-B15 dsw021-B16" class="link link-ref link-reveal xref-bibr"><sup>7–16</sup></a> The departure from PR2 provides a window into fundamental processes, such as replication, transcription, and repair mechanisms, which act differentially along each DNA strand and induce strand biased substitution rates, shaping DNA sequence contrary to the null expectation of [A] = [T] and [G] = [C] equilibrium. <a reveal-id="dsw021-B8" data-open="dsw021-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsw021-B16 dsw021-B17 dsw021-B18 dsw021-B19 dsw021-B20 dsw021-B21 dsw021-B22" data-open="dsw021-B16 dsw021-B17 dsw021-B18 dsw021-B19 dsw021-B20 dsw021-B21 dsw021-B22" class="link link-ref link-reveal xref-bibr"><sup>16–22</sup></a></p><p> Lobry suggested that the observed compositional strand-asymmetries are due to differences in replication-associated mutagenesis and repair between leading and lagging strands, <a reveal-id="dsw021-B7" data-open="dsw021-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and following studies argued along the same lines. <a reveal-id="dsw021-B9" data-open="dsw021-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw021-B23" data-open="dsw021-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> These asymmetries are also attributed to the uneven gene partition between replication strands <a reveal-id="dsw021-B11" data-open="dsw021-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw021-B16" data-open="dsw021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw021-B22" data-open="dsw021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw021-B24" data-open="dsw021-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsw021-B25" data-open="dsw021-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> coupled with transcription-induced mutagenesis and repair, <a reveal-id="dsw021-B26" data-open="dsw021-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> as well as codon and amino acid usage biases. <a reveal-id="dsw021-B27 dsw021-B28 dsw021-B29 dsw021-B30" data-open="dsw021-B27 dsw021-B28 dsw021-B29 dsw021-B30" class="link link-ref link-reveal xref-bibr"><sup>27–30</sup></a></p><p> From another perspective, nucleotide composition of genes, mainly reflected on GC content, may affect amino acid biases of proteins, as suggested by compositional correlation between DNA and protein sequences, <a reveal-id="dsw021-B31" data-open="dsw021-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and hydropathy profile studies. <a reveal-id="dsw021-B32" data-open="dsw021-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Accordingly, mutational pressure, identified on the basis of GC content and DNA compositional biases, can have a major effect on the long-term molecular evolution of proteins. </p><p> Though there are numerous studies on mononucleotide strand asymmetries, the distribution of oligonucleotides among different strands seems to have attracted less attention. <a reveal-id="dsw021-B33" data-open="dsw021-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw021-B34" data-open="dsw021-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Interestingly, no study has extensively assessed strand asymmetries at the level of dinucleotides, though these oligomers are the primary ordering units of DNA bases. <a reveal-id="dsw021-B35" data-open="dsw021-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p> Dinucleotide occurrence ‘preferences’ have been reported since very early <a reveal-id="dsw021-B36" data-open="dsw021-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsw021-B37" data-open="dsw021-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> as a result of nearest-neighbour constraints. The relative abundances (odds-ratios) of dinucleotides quantify such preferences. It has been noted that within the same DNA strand the relative abundance of a given dinucleotide is approximately equal to the one of each reverse complement, <a reveal-id="dsw021-B38 dsw021-B39 dsw021-B40" data-open="dsw021-B38 dsw021-B39 dsw021-B40" class="link link-ref link-reveal xref-bibr"><sup>38–40</sup></a> suggesting a base order intra-strand parity between relative abundances of reverse complementary dinucleotides. Mrázek and Karlin <a reveal-id="dsw021-B10" data-open="dsw021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> considered whether deviations from this base order intra-strand parity exist. However, based on the limited number of complete genomes then available, they concluded that ‘dinucleotide relative abundances tend to be symmetric and effectively constant relative to the leading and lagging strand for all dinucleotides, despite the strand compositional asymmetry’. <a reveal-id="dsw021-B10" data-open="dsw021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>In this work, we provide statistically solid evidence for the existence of local deviations from intra-strand parity at the level of dinucleotide relative abundances. We show that such skews along coding sequences (CDS) can trace the standard phylogenetic relationships more efficiently than other compositional measures. The skew analysis that we present supports that the GC-biased structure of genetic code shapes compositional asymmetries along coding regions. Moreover, we assess whole-genome substitution strand-biases induced by the molecular machinery of replication, transcription and DNA-repair.</p>                            <h2 scrollto-destination=87198895 id="87198895" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=87198896 id="87198896" class="section-title" >2.1 Dataset</h3>
<p> We perform our analysis on the dataset used by Necşulea and Lobry to study DNA base composition asymmetries in prokaryotes. <a reveal-id="dsw021-B22" data-open="dsw021-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> For each bacterial sequence, we retrieved the latest version deposited in NCBI database until 20 April 2015, restricting our collection to species which belong to phyla with more than three members present in the initial dataset. For each chromosome, we retrieve the coordinates of the origin of replication ( <em>ori</em> ) from DoriC 5.0 database. <a reveal-id="dsw021-B41" data-open="dsw021-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> We exclude linear chromosomes whose origin of replication is located less than one-fifth of their total length away from their ends. In total, 340 DNA sequences of the initial dataset meet the above prerequisites and constitute our genomic dataset. </p>                            <h3 scrollto-destination=87198900 id="87198900" class="section-title" >2.2 Definition of skews</h3>
<p> Mononucleotide skews are computed as the following ratios: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msup><mi>S</mi><mrow><mtext>A-T</mtext></mrow></msup><mo>=</mo><mfrac><mrow><msup><mi>f</mi><mtext>A</mtext></msup><mo>−</mo><msup><mi>f</mi><mtext>T</mtext></msup></mrow><mrow><msup><mi>f</mi><mtext>A</mtext></msup><mo>+</mo><msup><mi>f</mi><mtext>T</mtext></msup></mrow></mfrac><mo> </mo></mrow></mrow></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msup><mi>S</mi><mrow><mtext>G-C</mtext></mrow></msup><mo>=</mo><mfrac><mrow><msup><mi>f</mi><mtext>G</mtext></msup><mo>−</mo><msup><mi>f</mi><mtext>C</mtext></msup></mrow><mrow><msup><mi>f</mi><mtext>G</mtext></msup><mo>+</mo><msup><mi>f</mi><mtext>C</mtext></msup></mrow></mfrac></mrow></mrow></math></span> , where <em>f</em><sup>X</sup> is the observed frequency of X ∈ {A, T, G, C} in a given DNA sequence. </p><p> For each pair of reverse complementary dinucleotides, the observed frequency skew is computed as the following ratio: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msup><mi>S</mi><mrow><mtext>XY</mtext><mo>−</mo><mtext>Y'X'</mtext></mrow></msup><mo>=</mo><mfrac><mrow><msup><mi>f</mi><mrow><mtext>XY</mtext></mrow></msup><mo>−</mo><msup><mi>f</mi><mrow><mtext>Y'X'</mtext></mrow></msup></mrow><mrow><msup><mi>f</mi><mrow><mtext>XY</mtext></mrow></msup><mo>+</mo><msup><mi>f</mi><mrow><mtext>Y'X'</mtext></mrow></msup></mrow></mfrac></mrow></mrow></math></span> , where Y'X' is the reverse complement of XY and <em>f</em><sup>XY</sup> , <em>f</em><sup>Y'X'</sup> are the observed frequencies of the corresponding dinucleotides, as in reference. <a reveal-id="dsw021-B39" data-open="dsw021-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p><p> We introduce the relative abundance skews of reverse complementary dinucleotides, which are given by the formula: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msup><mi>P</mi><mrow><mtext>XY</mtext><mo>−</mo><mtext>Y'X'</mtext></mrow></msup><mo>=</mo><msup><mi>ρ</mi><mrow><mtext>XY</mtext></mrow></msup><mo>−</mo><msup><mi>ρ</mi><mrow><mtext>Y'X'</mtext></mrow></msup></mrow></mrow></math></span> where Y'X' is the reverse complement of XY and <em>ρ</em><sup>XY</sup> , <em>ρ</em><sup>Y'X'</sup> are the relative abundances of the corresponding dinucleotides. The relative abundance of a given dinucleotide is the ratio of its observed frequency to expected frequency, the later being the product of the observed frequencies of its mononucleotide components. Hence <em>ρ</em><sup>XY </sup> = <sup> </sup><em>f</em><sup>XY</sup><strong>/</strong><em>f</em><sup>X</sup><em>f</em><sup>Y</sup> . </p><p>Note that, from the 16 distinct dinucleotides, 4 of them are identical to their reverse complementary ones. The remaining 12 form 6 pairs of reverse complementary dinucleotides, namely AG-CT, GA-TC, GG-CC, AA-TT, AC-GT and CA-TG, for which we study the dinucleotide frequency and relative abundance skews.</p>                            <h3 scrollto-destination=87198908 id="87198908" class="section-title" >2.3 Graphical representation of skews</h3>
<p> For every chromosome in our collection, we calculate the skews along the plus strand in reference sequence in adjacent, non-overlapping windows of 10 <sup>4</sup> bp length and plot their non-cumulative and cumulative diagrams. The coordinates of all circular chromosomes are shifted so that the <em>ori</em> is placed at the middle of their published sequence. Then, the <em>ter</em> is set as the location of <em>ori</em> plus half of the chromosomes length, as in previous genomic studies. <a reveal-id="dsw021-B42" data-open="dsw021-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsw021-B43" data-open="dsw021-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> In this way, the <em>ter</em> region corresponds to the end (or, which is the same, to the start), of the shifted sequences, with the first half of each diagram corresponding to the lagging strand, and the second half to the leading strand. </p>                            <h3 scrollto-destination=87198910 id="87198910" class="section-title" >2.4 CDS concatenates</h3>
<p> From each DNA sequence, we retrieve the coding strand of all its protein-coding genes, in their sense direction, and concatenate them to form an artificial sequence, the ‘CDS concatenate’. The ordering of these coding strands maintains their relative position along the original DNA sequence. As the <em>ori</em> usually does not overlap with protein-coding genes, we detect the region of CDS concatenates which is closer to the <em>ori</em> . For all circular sequences we shift the coordinates of CDS concatenates so that this region will be placed at the middle of these artificial sequences. </p>                            <h3 scrollto-destination=87198912 id="87198912" class="section-title" >2.5 Detecting breakpoints in skew patterns</h3>
<p> In many bacterial chromosomes, mononucleotide skews are close to constant along the leading or lagging strand, with their sign changing near the <em>ori</em> and <em>ter</em> . Thus, it is reasonable to assume that mononucleotide skews fit a linear regression model along each replication strand. In this context, <em>ori</em> (or <em>ter</em> ) represents a breakpoint, where the model coefficients shift from one stable regression relationship to a different one. To test the statistical significance of these structural changes in mononucleotide skew patterns and assess whether such changes also appear in dinucleotide frequency and relative abundance skews, we use a dynamic programming algorithm developed in references, <a reveal-id="dsw021-B44" data-open="dsw021-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw021-B45" data-open="dsw021-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> as implemented in the R package strucchange. <a reveal-id="dsw021-B46" data-open="dsw021-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The algorithm allows simultaneous estimation of multiple breakpoints by assessing deviations from stability in the fitted linear regression model. It computes the number and position of the optimal breakpoints from data alone, with no a priori knowledge. We apply this algorithm in skews along both plus strand and CDS concatenates, and plot the resulting fitted models. </p>                            <h3 scrollto-destination=87198914 id="87198914" class="section-title" >2.6 Correlation of strand compositional asymmetries with species phylogeny</h3>
<p> We group the skews along the CDS concatenates into three classes; ‘mononucleotide skews’ ( <em>V</em><sup>MONO</sup> ): <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> , ‘dinucleotide skews’ ( <em>V</em><sup>DI</sup> ): <em>S</em><sub>CDS</sub><sup>XY-Y′X′</sup> , and ‘relative abundance skews’ ( <em>V</em><sup>RA</sup> ): <em>P</em><sub>CDS</sub><sup>XY</sup><sup>−</sup><sup>Y′X′</sup> , where XY−Y′X′ ∈ {AG-CT, GA-TC, GG-CC, AA-TT, AC-GT, CA-TG}. Each of these classes can be seen as a multidimensional random variable, corresponding to a list of vectors of skew values. We compute the V <sup>MONO</sup> , V <sup>DI</sup> , and V <sup>RA</sup> along the CDS concatenates in adjacent, non-overlapping windows of 10 <sup>4</sup> bp length. To assess the similarity between the skew patterns of any two CDS concatenates, we pairwise compare the multivariate distribution of each of their variables (V <sup>MONO</sup> , V <sup>DI</sup> , V <sup>RA</sup> ) using the symmetric Kullback-Leibler (KL) divergence. <a reveal-id="dsw021-B47" data-open="dsw021-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Thus, the symmetric KL-divergence quantifies the dissimilarity between each pair of bacteria present in our collection, in terms of skew profiles along the corresponding CDS concatenates. </p><p> We group the bacteria of our collection according to their phylum, and, in the case of Proteobacteria (which is by far the largest phylum in our dataset), according to their class. For each of these groups, we perform complete-linkage hierarchical clustering of the CDS concatenates, using the symmetric KL-divergence of their V <sup>MONO</sup> or V <sup>DI</sup> or V <sup>RA</sup> , to obtain the corresponding cladograms (skew-based trees). For purposes of this analysis, if a species has more than one chromosomes, we keep the largest chromosome excluding any other (a total of 29 DNA sequences). We employ the web-based tool Compare2Trees <a reveal-id="dsw021-B48" data-open="dsw021-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> to compare the topology of the skew-based trees and the species-trees, corresponding to the phylogenetic relations of bacteria. We obtained the species-tree from NCBI Taxonomy, <a reveal-id="dsw021-B49" data-open="dsw021-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> which provides a sequence-based phylogenetic classification, manually curated to reflect the current consensus in the systematic literature. The scores resulting from Compare2Trees express the percent topological similarity of the trees and indicate whether V <sup>MONO</sup> , V <sup>DI</sup> , V <sup>RA</sup> can trace bacterial species phylogeny. We use as benchmark the topological scores corresponding to cladograms we construct based on the δ <strong>-</strong> distance of the genomic signatures. <a reveal-id="dsw021-B50" data-open="dsw021-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsw021-B51" data-open="dsw021-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a></p><p> In order to detect correlations of skews with bacterial phylogeny, we use the multidimensional variables V <sup>MONO</sup> , V <sup>DI</sup> , and V <sup>RA</sup> instead of each skew separately, because if i.e. the rates of A→G transitions differer among species, this will simultaneously affect both <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> . Moreover, in the same example, if the rates of A→G are context <strong>-</strong> dependent, this may also affect all relative abundance skews, since the six reverse complementary dinucleotide pairs we examine have all at least one dinucleotide which contains an A or a G. </p>                            <h3 scrollto-destination=87198920 id="87198920" class="section-title" >2.7 Evolutionary relations of CDSs</h3>
<p> We retrieve the non-supervised orthologous groups from EggNOG v4.0 database. <a reveal-id="dsw021-B52" data-open="dsw021-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> We consider orthologous genes at the taxonomic levels for which our cladistic diagrams are constructed. For all species within a given taxonomic level, we pick out of each chromosome the coding strands of the genes which have orthologs in no &gt;10% of these species (or in just one of them, for taxonomic groups with few members present in our collection). These CDSs form the <strong>‘</strong> low <strong>-</strong> coverage orthologs’ group of the chromosome. All other orthologs detected in the respective chromosome form the <strong>‘</strong> remaining orthologs, (complementary) group. </p><p>For species with more than one chromosomes we keet the largest one and exclude all the others. We also exclude from our analysis the chromosomes of the species which are not represented in the eggNOG v4.0 database. Thereafter, a total of 266 DNA sequences are taken into account.</p>                            <h3 scrollto-destination=87198925 id="87198925" class="section-title" >2.8 Transcription- and replication-associated skews</h3>
<p> To detect mutation-derived skews we use third 4-fold-degenerate sites (third-ff), which are not subject to selection on the encoded amino-acid. We consider nucleotide composition in the sense direction and distinguish between codon-sites of genes lying on leading and lagging strand, where replication and transcription progress in the same or opposite direction, respectively. For each skew, let SK <sub>sense,leading</sub> and SK <sub>sense,lagging</sub> be its values in the first and second set of codon-sites, respectively. For explanatory purposes, let us denote by <em>α</em> the transcription-induced component of this skew and by <em>β</em> the replication-induced component. Transcription-induced skews have opposite signs in sense versus anti-sense strands. We consider <em>α</em> with respect to the sense strand. Likewise, replication-induced skews have opposite signs in leading versus lagging strand. We consider <em>β</em> with respect to the leading strand. Then, SK <sub><strong>sense,leading </strong></sub> ≈  <em>α</em>  +  <em>β</em> and SK <sub><strong>sense,lagging </strong></sub> ≈  <em>α</em> – <em>β</em> . We define the transcription-associated (Trs) skew as <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mfrac><mrow><msub><mrow><mtext>SK</mtext></mrow><mrow><mtext>sense,leading</mtext></mrow></msub><mo>+</mo><msub><mrow><mtext>SK</mtext></mrow><mrow><mtext>sense,lagging</mtext></mrow></msub></mrow><mtext>2</mtext></mfrac></mrow></mrow></math></span> , which is an estimate of <em>α</em> . Likewise, we define the replication-associated (Rep) skew as <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><mfrac><mrow><msub><mrow><mtext>SK</mtext></mrow><mrow><mtext>sense,leading</mtext></mrow></msub><mo>−</mo><msub><mrow><mtext>SK</mtext></mrow><mrow><mtext>sense,lagging</mtext></mrow></msub></mrow><mtext>2</mtext></mfrac></mrow></mrow></math></span> , which is an estimate of <em>β</em> . The above formulation provides a heuristic decoupling of the effect of replication and transcription on stand asymmetries, represented by Rep and Trs, respectively. </p><p> Because certain base substitutions depend on the identity of the adjacent nucleotides, <a reveal-id="dsw021-B53" data-open="dsw021-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> we also have to take into account flanking neighbours of third-ff sites. Since there are no 4-fold degenerate codons with A at their second position, we only consider dinucleotides, X <sub>3</sub> Y <sub>1</sub> , occupying a third-ff site and the first site of the adjacent codon, as described in. <a reveal-id="dsw021-B54" data-open="dsw021-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In this context, the relative abundance of X <sub>3</sub> Y <sub>1</sub> is: ρ <sup>X</sup><sub>3</sub><sup>Y</sup><sub>1</sub>  =  <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>f</mtext></mrow><mrow><mtext>3</mtext></mrow><mrow><mtext>X</mtext></mrow></msubsup></math></span><sup>Y</sup><sub>1</sub> / <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>f</mtext></mrow><mrow><mtext>3</mtext></mrow><mrow><mtext>X</mtext></mrow></msubsup></math></span><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>f</mtext></mrow><mrow><mtext>1</mtext></mrow><mrow><mtext>Y</mtext></mrow></msubsup></math></span> , with <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>f</mtext></mrow><mrow><mtext>3</mtext></mrow><mrow><mtext>X</mtext></mrow></msubsup></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>f</mtext></mrow><mrow><mtext>1</mtext></mrow><mrow><mtext>Y</mtext></mrow></msubsup></math></span> being the frequencies of X and Y at third-ff and their neighbouring first sites, respectively, with X, Y ∈ {A, T, G, C}. </p>                            <h3 scrollto-destination=87198929 id="87198929" class="section-title" >2.9 Transcription-coupled repair and PolIII α-subunit isoforms</h3>
<p> To identify which bacteria possess the <em>mfd</em> gene [transcription-coupled repair (TCR)-proficient] and what type of polymerase III (PolIII) <em>α</em><strong>-</strong> subunit isoform they bear, we retrieve the corresponding data from the KEGG Orthology database <strong>.</strong><a reveal-id="dsw021-B55" data-open="dsw021-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p>                            <h2 scrollto-destination=87198932 id="87198932" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=87198933 id="87198933" class="section-title" >3.1 Relative abundance skews constitute a general property of bacterial genomes</h3>
<p> A convenient way to analyse strand biases of single base distribution is by means of S <sup>A</sup><sup>−</sup><sup>T</sup> and S <sup>G</sup><sup>−</sup><sup>C</sup> cumulative diagrams. <a reveal-id="dsw021-B56" data-open="dsw021-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Likewise, cumulative diagrams of dinucleotide and relative abundance skews can be used to assess compositional asymmetries at the level of dinucleotide observed frequencies ( <em>f</em> ) and relative abundances ( <em>ρ</em> ), respectively. In <a reveal-id="dsw021-F1" data-open="dsw021-F1" class="link link-reveal link-table xref-fig">Figure 1</a> we indicatively present the cumulative skew diagrams of CA–TG and AC–GT pairs along the plus strand of the chromosome of <em>Carboxydothermus hydrogenoformans</em> Z − 2901. The diagrams, in terms of both <em>f</em> and <em>ρ</em> , present a characteristic V/inverted-V shape, with their extreme located at the <em>ori</em> site. Relative abundance skew patterns are reported here for the first time. These highly structured patterns point to intra-strand asymmetries that should be rather significant, both statistically and biologically. </p>                        <a id="87198936" scrollto-destination="87198936"></a>
<div id="dsw021-F1" data-id="dsw021-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f1p.png?Expires=2147483647&Signature=oKg6CJkBx3-EiLrFS01guTynuT4FlrtYTl~NaoOuZX4XbBIbyYapRMTY-Eb3Mt4hkNwrdhWwvwiUS9uO9s7PTum2SE8jrcNltjEqfa7F4-nT648bUTeHk1fXsk8VPVCvwdA6Wi5Cr0NYK9Fl5osz5KTXG-bxk1sizJ4NduwebQDXXBxmKM2Nz9d2-0i9Rjv8wzXvZcfmWDXnyrBigRxg7oRUZrONl3czqt-9FC3XH3HV8prIwZO7tiuSmR4t159aLVQazUZUx177w37jN2vapTPjUMDwztzoTmckZDpb-uWnJBOxtAOfLj0GTk4Oz1FVe2anxsiefKiaYjZcvkwuIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Cumulative skew diagrams along the plus strand of the chromosome of Carboxydothermus hydrogenoformans Z − 2,901. In each diagram, both dinucleotide skews (continuous line) and relative abundance skews (dashed line) are plotted. For the full set of cumulative skew diagrams, corresponding to all genomes in our collection, see link to Additional Figures and Tables. " path-from-xml="dsw021f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198936/dsw021f1p.png" path-from-xml="dsw021f1p.png" target="_blank">View large</a><a section="87198936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f1p.png?Expires=2147483647&Signature=bebEEPsRe5fWSeYGQFe11Y9vHqOWV9X7FcNXxXE5NWhT9yLDHYKLFEuKF6pCvXub1eDi~1cS~eddysuv0VT-ttKeyXYUg3VVXOO9KRm8FezgvG4H7S7d3BZgqw2tPaefpbP8-AY9SUf99WG2CXH798PmDqSsz4QQRPq59gzF~6zv~LKAnppJTZWbQd4ZFdUzXIhANEMs4rja4mSY30ahXLD-eivRSxp5jHATF56N7y7~4V1TiN5cmo7Zf6o7VJ5imOf~vao4HHLfEtztfMA9HZbQXvYJ8jgHF0vXo8EIQbXKqQZIwLMkBwT-uSv5zTLy~0SyqAxxM0xKFdq1hgBEYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198936&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Cumulative skew diagrams along the plus strand of the chromosome of <em>Carboxydothermus hydrogenoformans</em> Z − 2,901. In each diagram, both dinucleotide skews (continuous line) and relative abundance skews (dashed line) are plotted. For the full set of cumulative skew diagrams, corresponding to all genomes in our collection, see link to Additional Figures and Tables. </p></div></div><div content-id="dsw021-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f1p.png?Expires=2147483647&Signature=oKg6CJkBx3-EiLrFS01guTynuT4FlrtYTl~NaoOuZX4XbBIbyYapRMTY-Eb3Mt4hkNwrdhWwvwiUS9uO9s7PTum2SE8jrcNltjEqfa7F4-nT648bUTeHk1fXsk8VPVCvwdA6Wi5Cr0NYK9Fl5osz5KTXG-bxk1sizJ4NduwebQDXXBxmKM2Nz9d2-0i9Rjv8wzXvZcfmWDXnyrBigRxg7oRUZrONl3czqt-9FC3XH3HV8prIwZO7tiuSmR4t159aLVQazUZUx177w37jN2vapTPjUMDwztzoTmckZDpb-uWnJBOxtAOfLj0GTk4Oz1FVe2anxsiefKiaYjZcvkwuIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Cumulative skew diagrams along the plus strand of the chromosome of Carboxydothermus hydrogenoformans Z − 2,901. In each diagram, both dinucleotide skews (continuous line) and relative abundance skews (dashed line) are plotted. For the full set of cumulative skew diagrams, corresponding to all genomes in our collection, see link to Additional Figures and Tables. " path-from-xml="dsw021f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198936/dsw021f1p.png" path-from-xml="dsw021f1p.png" target="_blank">View large</a><a section="87198936" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f1p.png?Expires=2147483647&Signature=bebEEPsRe5fWSeYGQFe11Y9vHqOWV9X7FcNXxXE5NWhT9yLDHYKLFEuKF6pCvXub1eDi~1cS~eddysuv0VT-ttKeyXYUg3VVXOO9KRm8FezgvG4H7S7d3BZgqw2tPaefpbP8-AY9SUf99WG2CXH798PmDqSsz4QQRPq59gzF~6zv~LKAnppJTZWbQd4ZFdUzXIhANEMs4rja4mSY30ahXLD-eivRSxp5jHATF56N7y7~4V1TiN5cmo7Zf6o7VJ5imOf~vao4HHLfEtztfMA9HZbQXvYJ8jgHF0vXo8EIQbXKqQZIwLMkBwT-uSv5zTLy~0SyqAxxM0xKFdq1hgBEYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198936&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Cumulative skew diagrams along the plus strand of the chromosome of <em>Carboxydothermus hydrogenoformans</em> Z − 2,901. In each diagram, both dinucleotide skews (continuous line) and relative abundance skews (dashed line) are plotted. For the full set of cumulative skew diagrams, corresponding to all genomes in our collection, see link to Additional Figures and Tables. </p></div></div><p> Strand asymmetries in substitution rates of a nucleotide can be drastically affected by its flanking neighbours, in a genome-wide scale. Dinucleotide frequency skews can emerge from strand-biased substitution processes which are either neighbour-dependent or-independent. On the contrary <strong>,</strong> persistent patterns of relative abundance skews can only result from strand biases of neighbour-dependent substitutions. </p><p> The <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>CA</mtext></mrow></msubsup></math></span><sup>–</sup><sup>TG</sup> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>CA</mtext></mrow></msubsup></math></span><sup>–</sup><sup>TG</sup> curves point towards the same direction, both displaying a clear inverted-V shape. On the other hand, the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> curves point in the opposite direction, the former bearing an inverted-V and the latter a V shape. This is an indication of the different modalities acting on the formation of compositional skews (i.e. <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> ) and nearest-neighbour preference skews (i.e. <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> ). </p><p> From <a reveal-id="dsw021-F1" data-open="dsw021-F1" class="link link-reveal link-table xref-fig">Figure 1b</a> it follows that GT is more frequently occurring in leading strand than AC, in accordance with an excess of G and T versus C and A. However, the relative abundance of AC in leading strand is higher than the one of GT. Unlike observed frequencies, the relative abundances of dinucleotides factor out the effect of single-base composition and their skews cannot be inferred from the corresponding mono- and di-nucleotide ones. Instead, relative abundance skews constitute an alignment-free index which effectively detects chromosomal regions where context-dependent substitutions are strand-biased. </p><p> The patterns in <a reveal-id="dsw021-F1" data-open="dsw021-F1" class="link link-reveal link-table xref-fig">Figure 1</a> indicate that the depicted skews are quasi-constant along the leading or lagging strand and have opposite sign upstream and downstream the <em>ori</em> . To assess the statistical significance of the observed changes in skew polarity we use a linear regression setup. <a reveal-id="dsw021-B44" data-open="dsw021-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw021-B45" data-open="dsw021-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> The fitted models of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>plus</mtext></mrow><mrow><mtext>AC</mtext></mrow></msubsup></math></span><sup>–</sup><sup>GT</sup> display sharp, step-like breakpoints at the <em>ori</em> site ( <a reveal-id="dsw021-F2" data-open="dsw021-F2" class="link link-reveal link-table xref-fig">Fig. 2a and c</a> ), denoting that dinucleotide and relative abundance skews significantly correlate with the mode of replication. These breakpoints illustrate asymmetric substitution patterns in the two replication strands. <a reveal-id="dsw021-B8" data-open="dsw021-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p>                        <a id="87198945" scrollto-destination="87198945"></a>
<div id="dsw021-F2" data-id="dsw021-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f2p.png?Expires=2147483647&Signature=qSwthU2MhGKraKBkWdDLIOsdKWpaKBEdgCXrbFxN9-meP7EYg~wGmgcWMlMuX0VqP19EvfeUQrt0XBS-4TL1jJLZ9nOuV~cGKNbGSPTMmIUAAelgMVh-Opnuju4e-IbkwM39gmd64YIJfL9G-JEPNawAHeKAhNN4gU2NoTr6wZdbW3oRHpRn-NpB2qqtxw0oscS72U7hGRD5OZyMEMCxsUKgKCraI41X5brz-CdvKUvP~BxS-uAEwi9Y5wvfhvtOoPThuxo73yHk6ehKruLd5qwhW8pkFKkNtVFdyYWfh97xfQZ~W1ik9MfTNn2RJmOlZ5Ze13WlY6G9krQFRCFQyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Diagrams (thin curve) and fitted linear models (thick line) for SAC-GT and PAC-GT in Carboxydothermus hydrogenoformans Z − 2,901, along the plus (published) strand ( a and c ) and the CDS concatenate ( b and d ). The vertical dashed straight line indicates the ori . The horizontal dashed straight line is drawn at the mean value of the skew. Along the plus strand skews display a step-like breakpoint at the ori and their mean values approximately equal zero, since skews in leading and lagging strand are almost cancelled out on a genome scale (a and c). On the contrary, skews along CDS concatenates display no structural change near the ori and their mean considerably departs from zero (b and d). " path-from-xml="dsw021f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198945/dsw021f2p.png" path-from-xml="dsw021f2p.png" target="_blank">View large</a><a section="87198945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f2p.png?Expires=2147483647&Signature=QqmJXEtVJAufoAT5I0NiH9NIE3Y~EkCLMNZl~f5xN3-H-V1lTnT8GZpcIr4FQqPulU4Sh9u9KSRZBGvTBI57eSXGi3sUl8UbcR50snSUnKODyhJJ0G83fpnHWeN~3D2OXBQL70xKip3srlKUtQ3DX2ifCHOLPl77WCZ8DdbOl8Ytv3nKuJwzh02~SoB8n7nu8iwvde4tQlbY2OWdaG-RWVBb6lrlD3552Us5mGfv9JFd3M70ok5qfqsJafvmkE2mNfTTm-v75IrjM7JyTxbf5gA1NrwhkJuhq9W3VIecpMOEB~H4m0EKVReml6rEVUEGbfDWZaRv8vyI3OCRLve8jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198945&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Diagrams (thin curve) and fitted linear models (thick line) for <em>S</em><sup>AC-GT</sup> and <em>P</em><sup>AC-GT</sup> in <em>Carboxydothermus hydrogenoformans</em> Z − 2,901, along the plus (published) strand ( <strong>a</strong> and <strong>c</strong> ) and the CDS concatenate ( <strong>b</strong> and <strong>d</strong> ). The vertical dashed straight line indicates the <em>ori</em> . The horizontal dashed straight line is drawn at the mean value of the skew. Along the plus strand skews display a step-like breakpoint at the <em>ori</em> and their mean values approximately equal zero, since skews in leading and lagging strand are almost cancelled out on a genome scale (a and c). On the contrary, skews along CDS concatenates display no structural change near the <em>ori</em> and their mean considerably departs from zero (b and d). </p></div></div><div content-id="dsw021-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f2p.png?Expires=2147483647&Signature=qSwthU2MhGKraKBkWdDLIOsdKWpaKBEdgCXrbFxN9-meP7EYg~wGmgcWMlMuX0VqP19EvfeUQrt0XBS-4TL1jJLZ9nOuV~cGKNbGSPTMmIUAAelgMVh-Opnuju4e-IbkwM39gmd64YIJfL9G-JEPNawAHeKAhNN4gU2NoTr6wZdbW3oRHpRn-NpB2qqtxw0oscS72U7hGRD5OZyMEMCxsUKgKCraI41X5brz-CdvKUvP~BxS-uAEwi9Y5wvfhvtOoPThuxo73yHk6ehKruLd5qwhW8pkFKkNtVFdyYWfh97xfQZ~W1ik9MfTNn2RJmOlZ5Ze13WlY6G9krQFRCFQyw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Diagrams (thin curve) and fitted linear models (thick line) for SAC-GT and PAC-GT in Carboxydothermus hydrogenoformans Z − 2,901, along the plus (published) strand ( a and c ) and the CDS concatenate ( b and d ). The vertical dashed straight line indicates the ori . The horizontal dashed straight line is drawn at the mean value of the skew. Along the plus strand skews display a step-like breakpoint at the ori and their mean values approximately equal zero, since skews in leading and lagging strand are almost cancelled out on a genome scale (a and c). On the contrary, skews along CDS concatenates display no structural change near the ori and their mean considerably departs from zero (b and d). " path-from-xml="dsw021f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198945/dsw021f2p.png" path-from-xml="dsw021f2p.png" target="_blank">View large</a><a section="87198945" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f2p.png?Expires=2147483647&Signature=QqmJXEtVJAufoAT5I0NiH9NIE3Y~EkCLMNZl~f5xN3-H-V1lTnT8GZpcIr4FQqPulU4Sh9u9KSRZBGvTBI57eSXGi3sUl8UbcR50snSUnKODyhJJ0G83fpnHWeN~3D2OXBQL70xKip3srlKUtQ3DX2ifCHOLPl77WCZ8DdbOl8Ytv3nKuJwzh02~SoB8n7nu8iwvde4tQlbY2OWdaG-RWVBb6lrlD3552Us5mGfv9JFd3M70ok5qfqsJafvmkE2mNfTTm-v75IrjM7JyTxbf5gA1NrwhkJuhq9W3VIecpMOEB~H4m0EKVReml6rEVUEGbfDWZaRv8vyI3OCRLve8jA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198945&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Diagrams (thin curve) and fitted linear models (thick line) for <em>S</em><sup>AC-GT</sup> and <em>P</em><sup>AC-GT</sup> in <em>Carboxydothermus hydrogenoformans</em> Z − 2,901, along the plus (published) strand ( <strong>a</strong> and <strong>c</strong> ) and the CDS concatenate ( <strong>b</strong> and <strong>d</strong> ). The vertical dashed straight line indicates the <em>ori</em> . The horizontal dashed straight line is drawn at the mean value of the skew. Along the plus strand skews display a step-like breakpoint at the <em>ori</em> and their mean values approximately equal zero, since skews in leading and lagging strand are almost cancelled out on a genome scale (a and c). On the contrary, skews along CDS concatenates display no structural change near the <em>ori</em> and their mean considerably departs from zero (b and d). </p></div></div><p> Compositional skews may also emerge due to substitution strand-biases associated with CDSs. <a reveal-id="dsw021-B26" data-open="dsw021-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> To address this question, we use the same linear regression setup to model <em>S</em><sub>CDS</sub><sup>AC</sup><sup>–</sup><sup>GT</sup> and <em>P</em><sub>CDS</sub><sup>AC</sup><sup>–</sup><sup>GT</sup> along the corresponding CDS concatenates, where all genes are considered in the direction of transcription. In this way, consecutive parts of the leading and lagging strand are alternating within both halves of the genome as delimited by <em>ori</em> . Thus, along CDS concatenates the effect of replication on the observed skews is cancelled out within a window of a few thousands bps. As seen in <a reveal-id="dsw021-F2" data-open="dsw021-F2" class="link link-reveal link-table xref-fig">Figure 2b and d</a> though skew diagrams display fluctuations, there is no structural change near the position of the <em>ori</em> . The fitted models are constant along CDS concatenates, highlighting that the transcription orientation of genes is a major determinant of strand biases, at the level of both base composition ( <em>S</em><sub>CDS</sub><sup>AC</sup><sup>–</sup><sup>GT</sup> ) and base order ( <em>P</em><sub>CDS</sub><sup>AC</sup><sup>–</sup><sup>GT</sup> ). <a reveal-id="dsw021-B26" data-open="dsw021-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsw021-B57" data-open="dsw021-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a></p><p> Analysis of 340 bacterial chromosomes with cumulative diagrams and linear regression models shows that both dinucleotide and relative abundance skews are prevalent features of bacterial genomes in general. The corresponding plots are provided in a web-page (see link to Additional Figures and Tables), along with tabulated descriptive statistics of the skew distributions within our collection. The form of our cumulative skew diagrams ranges from the characteristic V or inverted-V shapes, to heavily distorted curves with irregular peaks and troughs, in line with previously known skew patterns of bacterial genomes. <a reveal-id="dsw021-B15" data-open="dsw021-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw021-B16" data-open="dsw021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Overall inspection of dinucleotide and relative abundance skew diagrams shows that their patterns cannot be directly inferred from one another. When focusing on the linear regression models of relative abundance skews, strand-asymmetries of context-dependent substitutions seem to significantly correlate with replication- and CDS-associated biases in up to &gt;50 and 80% of our collection, respectively ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> ). </p>                            <h3 scrollto-destination=87198950 id="87198950" class="section-title" >3.2 Strand biases of CDSs are species-specific</h3>
<p> The set of strand-symmetrised relative abundances ( <em>ρ</em> * <sub>ΧΥ</sub> ), namely ‘genomic signatures’, have been used for phylogenetic reconstruction. <a reveal-id="dsw021-B58" data-open="dsw021-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> For a dinucleotide, XY, <em>ρ</em> * <sub><strong>ΧΥ</strong></sub> = <em>f</em> * <sub><strong>XY</strong></sub> / <em>f</em> * <sub><strong>X</strong></sub> f* <sub><strong>Y</strong></sub> , where <em>f</em> * <sub><strong>XY</strong></sub> , <em>f</em> * <sub><strong>X</strong></sub> and <em>f</em> * <sub><strong>Y</strong></sub> are the frequencies of dinucleotide XY and mononucleotides X, Y, respectively, computed along the concatenate of the sequence with its reverse complement. <a reveal-id="dsw021-B35" data-open="dsw021-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsw021-B58" data-open="dsw021-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> We assess whether strand biases of relative abundances, quantified by relative abundance skews, can also track the evolutionary path of divergent species. Based on pairwise comparisons of relative abundance skew profiles along CDSs, we construct the corresponding cladograms (see Methods section and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span> ). Then, we compare the skew-based trees to the species trees and obtain their percent topological similarity ( <a reveal-id="dsw021-F3" data-open="dsw021-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a> ). </p>                        <a id="87198953" scrollto-destination="87198953"></a>
<div id="dsw021-F3" data-id="dsw021-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f3p.png?Expires=2147483647&Signature=dTv-b9wd6t-ls1B~GCX5hvPXNw139vwcvbCa39dRmrLkTflQ5Qf2G5qEIpdliTT16bSIuV7Wphm9d7zTqYy2lQzQ2v~NWUTGgT2TKLiNNUuZNQiuOJFGTOdTtRvuN5dyT3bzvU-6IfYiYP5gHwMYO2-jBGR29-u48vsbbcel~55vtgmm~bNsOKkkJQf6M0NKexuiIfiVjrSxqJSH8~aKnzBROVwqjgUu~PtFVSf6CDJmIuDOwQtScKrEjfSA8tBaVC9mEm6SprkmwtN7GsW1V5b5Tshns1uKHk~Q4MQtSNiggkw9fXOxwCNceatjWvQSB-qfR0N-q0UkrHCbFcUBKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="a . Boxplots of topological scores, depicting the percent topological similarity of the skew-based trees to the species-trees. Comparisons are made between species that belong to the same taxonomic rank (phylum or class). b. Boxplots of percent rank of similarity between chromosomes of the same organism, in terms of their skew distribution along their CDSs. a and b. Skews are grouped into three classes: mononucleotide skews ( VMONO ), dinucleotide skews ( VDI ), and relative abundance skews ( VRA ). The topological scores and the percent rank of similarity based on δ -distances between genomic signatures are presented as a benchmark. c and d. Regression of the skews ( SCDSG-C and PCDSAG-CT ) in ‘low-coverage orthologs’ to the skews of the ‘remaining orthologs’. Each point represents a chromosome of our collection. Points that fall on the diagonal (slope = 1, y-intercept = 0; dashed line) indicate that skews in ‘low-coverage orthologs’ are identical to the skews in ‘remaining orthologs’. SCDSG-C regression line roughly coincides with the diagonal (slope = 0.939, y-intercept = 2.75 × 10  −   3 ), indicating that SCDSG-C of the ‘low-coverage orthologs’ comply with SCDSG-C of the remaining orthologs , in almost all the chromosomes we examined. PCDSAG-CT regression line intercepts the diagonal, with its slope close to unity (0.823) and y-intercept close to zero (−1.74 ×  10−2 ), indicating that PAG-CT is also approximately equal in ‘low-coverage’ and ‘remaining orthologs’. " path-from-xml="dsw021f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198953/dsw021f3p.png" path-from-xml="dsw021f3p.png" target="_blank">View large</a><a section="87198953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f3p.png?Expires=2147483647&Signature=ue2NCUMoysYzMW8UYFNWAasYmcUcABPZBkcQHISypDbr7~2Z7I5qGFelwgXgK3~L3q42UH1AkIdknG9Agq80p673RM4AjweIQh~rnnMnP6nBSN6PakPAOUEv1DQ-PtVNCAUZf0uQYCv~n9aH5D57ddLDhV71iDYc-W3ZMBFUdsAT-8tsMm4ZYFFVSbd0fceIQGHJIYxoFeR9flvDcvi2IrBHMcHFdMkGEj1F3ljD9WP1HH5hHGQDhaG-RhqU7R73yCMww4u5rYLFoNVqRqtj50No6RApdoXBSMRkPybXUoLzQtZeEFWZWU1g2cOIuDXscBwMC-wAVAeDISVFusKuoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198953&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong>a</strong> . Boxplots of topological scores, depicting the percent topological similarity of the skew-based trees to the species-trees. Comparisons are made between species that belong to the same taxonomic rank (phylum or class). <strong>b.</strong> Boxplots of percent rank of similarity between chromosomes of the same organism, in terms of their skew distribution along their CDSs. <strong>a and b.</strong> Skews are grouped into three classes: mononucleotide skews ( <em>V</em><sup>MONO</sup> ), dinucleotide skews ( <em>V</em><sup>DI</sup> ), and relative abundance skews ( <em>V</em><sup>RA</sup> ). The topological scores and the percent rank of similarity based on <em>δ</em> -distances between genomic signatures are presented as a benchmark. <strong>c and d.</strong> Regression of the skews ( <em>S</em><sub>CDS</sub><sup>G-C</sup> and <em>P</em><sub>CDS</sub><sup>AG-CT</sup> ) in ‘low-coverage orthologs’ to the skews of the ‘remaining orthologs’. Each point represents a chromosome of our collection. Points that fall on the diagonal (slope = 1, y-intercept = 0; dashed line) indicate that skews in ‘low-coverage orthologs’ are identical to the skews in ‘remaining orthologs’. <em>S</em><sub>CDS</sub><sup>G-C</sup> regression line roughly coincides with the diagonal (slope = 0.939, y-intercept = 2.75 × 10  <sup>−</sup>   <a reveal-id="dsw021-B3" data-open="dsw021-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> ), indicating that <em>S</em><sub>CDS</sub><sup>G-C</sup> of the ‘low-coverage orthologs’ comply with <em>S</em><sub>CDS</sub><sup>G-C</sup> of the <em>remaining orthologs</em> , in almost all the chromosomes we examined. <em>P</em><sub>CDS</sub><sup>AG-CT</sup> regression line intercepts the diagonal, with its slope close to unity (0.823) and y-intercept close to zero (−1.74 ×  <a reveal-id="dsw021-B10" data-open="dsw021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><sup>−</sup><a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> ), indicating that <em>P</em><sup>AG-CT</sup> is also approximately equal in ‘low-coverage’ and ‘remaining orthologs’. </p></div></div><div content-id="dsw021-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f3p.png?Expires=2147483647&Signature=dTv-b9wd6t-ls1B~GCX5hvPXNw139vwcvbCa39dRmrLkTflQ5Qf2G5qEIpdliTT16bSIuV7Wphm9d7zTqYy2lQzQ2v~NWUTGgT2TKLiNNUuZNQiuOJFGTOdTtRvuN5dyT3bzvU-6IfYiYP5gHwMYO2-jBGR29-u48vsbbcel~55vtgmm~bNsOKkkJQf6M0NKexuiIfiVjrSxqJSH8~aKnzBROVwqjgUu~PtFVSf6CDJmIuDOwQtScKrEjfSA8tBaVC9mEm6SprkmwtN7GsW1V5b5Tshns1uKHk~Q4MQtSNiggkw9fXOxwCNceatjWvQSB-qfR0N-q0UkrHCbFcUBKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="a . Boxplots of topological scores, depicting the percent topological similarity of the skew-based trees to the species-trees. Comparisons are made between species that belong to the same taxonomic rank (phylum or class). b. Boxplots of percent rank of similarity between chromosomes of the same organism, in terms of their skew distribution along their CDSs. a and b. Skews are grouped into three classes: mononucleotide skews ( VMONO ), dinucleotide skews ( VDI ), and relative abundance skews ( VRA ). The topological scores and the percent rank of similarity based on δ -distances between genomic signatures are presented as a benchmark. c and d. Regression of the skews ( SCDSG-C and PCDSAG-CT ) in ‘low-coverage orthologs’ to the skews of the ‘remaining orthologs’. Each point represents a chromosome of our collection. Points that fall on the diagonal (slope = 1, y-intercept = 0; dashed line) indicate that skews in ‘low-coverage orthologs’ are identical to the skews in ‘remaining orthologs’. SCDSG-C regression line roughly coincides with the diagonal (slope = 0.939, y-intercept = 2.75 × 10  −   3 ), indicating that SCDSG-C of the ‘low-coverage orthologs’ comply with SCDSG-C of the remaining orthologs , in almost all the chromosomes we examined. PCDSAG-CT regression line intercepts the diagonal, with its slope close to unity (0.823) and y-intercept close to zero (−1.74 ×  10−2 ), indicating that PAG-CT is also approximately equal in ‘low-coverage’ and ‘remaining orthologs’. " path-from-xml="dsw021f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198953/dsw021f3p.png" path-from-xml="dsw021f3p.png" target="_blank">View large</a><a section="87198953" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f3p.png?Expires=2147483647&Signature=ue2NCUMoysYzMW8UYFNWAasYmcUcABPZBkcQHISypDbr7~2Z7I5qGFelwgXgK3~L3q42UH1AkIdknG9Agq80p673RM4AjweIQh~rnnMnP6nBSN6PakPAOUEv1DQ-PtVNCAUZf0uQYCv~n9aH5D57ddLDhV71iDYc-W3ZMBFUdsAT-8tsMm4ZYFFVSbd0fceIQGHJIYxoFeR9flvDcvi2IrBHMcHFdMkGEj1F3ljD9WP1HH5hHGQDhaG-RhqU7R73yCMww4u5rYLFoNVqRqtj50No6RApdoXBSMRkPybXUoLzQtZeEFWZWU1g2cOIuDXscBwMC-wAVAeDISVFusKuoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198953&ar=2469968&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw021f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><strong>a</strong> . Boxplots of topological scores, depicting the percent topological similarity of the skew-based trees to the species-trees. Comparisons are made between species that belong to the same taxonomic rank (phylum or class). <strong>b.</strong> Boxplots of percent rank of similarity between chromosomes of the same organism, in terms of their skew distribution along their CDSs. <strong>a and b.</strong> Skews are grouped into three classes: mononucleotide skews ( <em>V</em><sup>MONO</sup> ), dinucleotide skews ( <em>V</em><sup>DI</sup> ), and relative abundance skews ( <em>V</em><sup>RA</sup> ). The topological scores and the percent rank of similarity based on <em>δ</em> -distances between genomic signatures are presented as a benchmark. <strong>c and d.</strong> Regression of the skews ( <em>S</em><sub>CDS</sub><sup>G-C</sup> and <em>P</em><sub>CDS</sub><sup>AG-CT</sup> ) in ‘low-coverage orthologs’ to the skews of the ‘remaining orthologs’. Each point represents a chromosome of our collection. Points that fall on the diagonal (slope = 1, y-intercept = 0; dashed line) indicate that skews in ‘low-coverage orthologs’ are identical to the skews in ‘remaining orthologs’. <em>S</em><sub>CDS</sub><sup>G-C</sup> regression line roughly coincides with the diagonal (slope = 0.939, y-intercept = 2.75 × 10  <sup>−</sup>   <a reveal-id="dsw021-B3" data-open="dsw021-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> ), indicating that <em>S</em><sub>CDS</sub><sup>G-C</sup> of the ‘low-coverage orthologs’ comply with <em>S</em><sub>CDS</sub><sup>G-C</sup> of the <em>remaining orthologs</em> , in almost all the chromosomes we examined. <em>P</em><sub>CDS</sub><sup>AG-CT</sup> regression line intercepts the diagonal, with its slope close to unity (0.823) and y-intercept close to zero (−1.74 ×  <a reveal-id="dsw021-B10" data-open="dsw021-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><sup>−</sup><a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> ), indicating that <em>P</em><sup>AG-CT</sup> is also approximately equal in ‘low-coverage’ and ‘remaining orthologs’. </p></div></div><p> As seen in the corresponding boxplots, both strand-specific ( <em>ρ</em><sub>ΧΥ</sub> − <em>ρ</em><sub>Y'X'</sub> ) and symmetrised ( <em>ρ</em> * <sub>ΧΥ</sub> ) versions of nearest-neighbour correlations are species-specific. For all taxonomic subdivisions the inferred topology largely agrees with the species phylogeny, with the resulting scores being higher than 72% in all cases except Actinobacteria and δ-Proteobacteria ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> ). Moreover, skew-based trees tend to be more accurate compared with genomic signature trees, as evidenced by their median scores. Thus, it can be argued that the strand bias of first-neighbour correlations constitute an idiosyncratic genomic feature that is linked to the evolutionary dynamics of the DNA strands. </p><p> We also present the topological scores of mono- and di-nucleotide skew-based trees ( <a reveal-id="dsw021-F3" data-open="dsw021-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a> ). Base frequencies and base order are fused at the level of base doublets, and hence dinucleotide skews reflect the combined effect of mononucleotide and relative abundance strand-biases. As shown by the median scores, skews contain information that enables increasingly accurate phylogenetic reconstruction as we shift from base singlets to doublets and eventually to doublet relative abundances. </p><p>To rule out the possibility that the observed skew similarities between different genomes are the result of CDS homology, we compare the skews of non-homologous parts of the same genome.</p><p> We sort in ascending order all pairs of chromosomes according to the similarity of their skews, in terms of symmetric KL-divergence, as obtained by pairwise comparisons of the distribution of each of their variables, V <sup>MONO</sup> , V <sup>DI</sup> , V <sup>RA</sup> along the CDS concatenates. We focused on the bacteria which carry more than one chromosomes (a total of 26 in our collection; plasmids are excluded). In <a reveal-id="dsw021-F3" data-open="dsw021-F3" class="link link-reveal link-table xref-fig">Figure 3b</a> , we present the ranks of similarity between chromosomes that belong to the same organism (a total of 55 pairs). When comparing dinucleotide or relative abundance skews, the distributions of ranks are quite compact, while ranks corresponding to mononucleotide skews are more scattered. Yet in all cases the median values are higher than 98%. Since different chromosomes of the same organism share limited, if any, homologous regions, their high ranks of skew similarity cannot be attributed to sequence similarity. </p><p> Moreover, in each chromosome we distinguish between two separate groups of genes, on the basis of the abundance of their orthologs within a given phylum or class (‘low-coverage’ vs. ‘remaining’ orthologs). For each skew, a single value is computed along the coding strand of genes which belong to the same group. In <a reveal-id="dsw021-F3" data-open="dsw021-F3" class="link link-reveal link-table xref-fig">Figure 3c and d</a> we regress <em>S</em><sub>CDS</sub><sup>G-C</sup> and <em>P</em><sub>CDS</sub><sup>AG-CT</sup> of the ‘low-coverage’ orthologs on <em>S</em><sub>CDS</sub><sup>G-C</sup> and <em>P</em><sub>CDS</sub><sup>AG-CT</sup> of the ‘remaining’ ones. The slope and y-intercept are close to unity and zero, respectively, indicating that ‘low-coverage’ and ‘remaining’ orthologs exhibit highly similar patterns of <em>S</em><sub>CDS</sub><sup>G-C</sup> and <em>P</em><sub>CDS</sub><sup>AG-CT</sup> . Similar correlation patterns are detected for all skews ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table 3</a></span> ). We conclude that within a given chromosome CDSs have quite similar contribution on the overall CDS skews, irrespective of their evolutionary descent. </p><p>Taken together, our results strongly suggest that the correlation of skews with the evolutionary course of bacteria cannot be attributed to sequence divergence of CDSs originating from a shared ancestry. Regarding in particular relative abundance skews, we obtained clear indications that they reflect species-specific substitution strand-biases, which are context-dependent.</p>                            <h3 scrollto-destination=87198963 id="87198963" class="section-title" >3.3 The GC biased structure of genetic code imposes site-specific strand-asymmetries</h3>
<p> Skews can be effectively implemented in the analysis of evolutionary processes that distinguish between coding and transcribed strands, resulting in strand-biased substitutions. Here, we inquire site-specific biases in nucleotide distribution along CDSs. We compute the <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> at each codon site. Regarding third codon position, 4-fold (third-ff) and 2-fold (third-tf) sites are considered separately. For all chromosomes in our collection, we compute the Pearson's product-moment correlation coefficient (Pearson’s <em>r</em> ) <a reveal-id="dsw021-B59" data-open="dsw021-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> between <em>S</em><sub>CDS</sub><sup>G-C</sup> at a given codon site and the frequency of each codon, and plot the corresponding values in heatmaps ( <a reveal-id="dsw021-F4" data-open="dsw021-F4" class="link link-reveal link-table xref-fig">Fig. 4a–d</a> ). </p>                        <a id="87198966" scrollto-destination="87198966"></a>
<div id="dsw021-F4" data-id="dsw021-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f4p.png?Expires=2147483647&amp;Signature=mYPX1mfTzM-xrGta0ljsonRKPGKIjf3fabKKlUt9vtiDtILo0BthK2xCimzFgm1zdr9tA3m1Up3TFIYIvBQoVy4yZggZc7B2EKhJ6G3rASdo26-KyGZ1wa6TrhhLKFPew4u0Xw1GujYrPIRSniwLieI2Bt8Sz2j49gPc4kcHrLsN9SVCAWmOoXtksyQGdMu0skXiPlRh3R5e7VQresrUK5TlH8V-k968iAdHQnQlcHZwEQ0piWDHSEC3weCZT7Ot~tDnDF0~WBbAjZMAFK~poihukP8pnGLJYYqjMJ5Gl9mQpPZAvMzUlFfod4AwN5m5l9gMBnO6M4ftDH~6fk9Z2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Heatmaps of Pearson’s product-moment correlation coefficient ( r ) for codon usage versus SCDSG-C . ( a–d)SCDSG-C is calculated at the third-ff, the first, second, and the third-tf codon sites, in genomic data. ( e and f)SCDSG-C is calculated at third-tf of the artificial sequences produced by our synonymous-codon GC biased model and the null-hypothesis model. Columns indicate the base composition of the codon site at which skews are computed. Stippled cells depict negative r , non-stippled cells depict positive r . The intensity of the color indicates the degree of correlation, as recited in the color key. +1, perfect positive correlation; −1, perfect negative correlation; 0, no correlation. Start and stop codons are excluded from our calculations. ATG (start codon) and, in some cases, TGA (opal stop codon) are also found inside CDS, the first encoding for Methionine and the second for Tryptophane or Selenocysteine, and thus correlations of SCDSA-T and SCDSG-C with [ATG] and [TGA] appear in the heatmaps. The six-fold degenerate codons of Leucine and Arginine are labeled in the heatmaps. For further remarks see Supplementary Figure S4 . Codons used in calculations of skews at third 4- and 2-fold-degenerate sites are marked with ‘ * ’ and ‘•’, respectively. ‘start’, initiation codon; ‘stop’, termination codons. " path-from-xml="dsw021f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198966/dsw021f4p.png" path-from-xml="dsw021f4p.png" target="_blank">View large</a><a section="87198966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f4p.png?Expires=2147483647&amp;Signature=Gvpq6rGsm6syElSvvVhDDVD0SKFoG2A4HXvpVlyee7IoXG6lnrnffsHUhmtAO8bNKbncXQ-gPPxgFWv~FzLY77T8Du33Vv8W6lSN0xLxjvmi7xcAwuHpvoSDCgrMxc5S-4gQXhhUD35RXFr8jUcZiGbvZuXMmhFpRTqmZZgv4ftRgKqRkVGPvm4nWn9ZRDY3DO3UqSz8Mfch-BJd8-RO0Gdvqfh14hWzjrEN7vU1XcnJgfWJG6S39h-yLNAyuhz7wOKrVNYwGAc3O4XTbwX05C0-d1I1dYAcgF7pXyGdArjxstEyLl~8j8kfq8h0PydaBUy8Tk32VKhyvWZ6GWMjWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198966&amp;ar=2469968&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw021f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Heatmaps of Pearson’s product-moment correlation coefficient ( <em>r</em> ) for codon usage versus <em>S</em><sub>CDS</sub><sup>G-C</sup> . ( <strong>a–d)</strong><em>S</em><sub>CDS</sub><sup>G-C</sup> is calculated at the third-ff, the first, second, and the third-tf codon sites, in genomic data. ( <strong>e and f)</strong><em>S</em><sub>CDS</sub><sup>G-C</sup> is calculated at third-tf of the artificial sequences produced by our synonymous-codon GC biased model and the null-hypothesis model. Columns indicate the base composition of the codon site at which skews are computed. Stippled cells depict negative <em>r</em> , non-stippled cells depict positive <em>r</em> . The intensity of the color indicates the degree of correlation, as recited in the color key. +1, perfect positive correlation; −1, perfect negative correlation; 0, no correlation. Start and stop codons are excluded from our calculations. ATG (start codon) and, in some cases, TGA (opal stop codon) are also found inside CDS, the first encoding for Methionine and the second for Tryptophane or Selenocysteine, and thus correlations of <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> with [ATG] and [TGA] appear in the heatmaps. The six-fold degenerate codons of Leucine and Arginine are labeled in the heatmaps. For further remarks see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> . Codons used in calculations of skews at third 4- and 2-fold-degenerate sites are marked with ‘ <strong>*</strong> ’ and ‘•’, respectively. ‘start’, initiation codon; ‘stop’, termination codons. </p></div></div><div content-id="dsw021-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/m_dsw021f4p.png?Expires=2147483647&amp;Signature=mYPX1mfTzM-xrGta0ljsonRKPGKIjf3fabKKlUt9vtiDtILo0BthK2xCimzFgm1zdr9tA3m1Up3TFIYIvBQoVy4yZggZc7B2EKhJ6G3rASdo26-KyGZ1wa6TrhhLKFPew4u0Xw1GujYrPIRSniwLieI2Bt8Sz2j49gPc4kcHrLsN9SVCAWmOoXtksyQGdMu0skXiPlRh3R5e7VQresrUK5TlH8V-k968iAdHQnQlcHZwEQ0piWDHSEC3weCZT7Ot~tDnDF0~WBbAjZMAFK~poihukP8pnGLJYYqjMJ5Gl9mQpPZAvMzUlFfod4AwN5m5l9gMBnO6M4ftDH~6fk9Z2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Heatmaps of Pearson’s product-moment correlation coefficient ( r ) for codon usage versus SCDSG-C . ( a–d)SCDSG-C is calculated at the third-ff, the first, second, and the third-tf codon sites, in genomic data. ( e and f)SCDSG-C is calculated at third-tf of the artificial sequences produced by our synonymous-codon GC biased model and the null-hypothesis model. Columns indicate the base composition of the codon site at which skews are computed. Stippled cells depict negative r , non-stippled cells depict positive r . The intensity of the color indicates the degree of correlation, as recited in the color key. +1, perfect positive correlation; −1, perfect negative correlation; 0, no correlation. Start and stop codons are excluded from our calculations. ATG (start codon) and, in some cases, TGA (opal stop codon) are also found inside CDS, the first encoding for Methionine and the second for Tryptophane or Selenocysteine, and thus correlations of SCDSA-T and SCDSG-C with [ATG] and [TGA] appear in the heatmaps. The six-fold degenerate codons of Leucine and Arginine are labeled in the heatmaps. For further remarks see Supplementary Figure S4 . Codons used in calculations of skews at third 4- and 2-fold-degenerate sites are marked with ‘ * ’ and ‘•’, respectively. ‘start’, initiation codon; ‘stop’, termination codons. " path-from-xml="dsw021f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198966/dsw021f4p.png" path-from-xml="dsw021f4p.png" target="_blank">View large</a><a section="87198966" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021f4p.png?Expires=2147483647&amp;Signature=Gvpq6rGsm6syElSvvVhDDVD0SKFoG2A4HXvpVlyee7IoXG6lnrnffsHUhmtAO8bNKbncXQ-gPPxgFWv~FzLY77T8Du33Vv8W6lSN0xLxjvmi7xcAwuHpvoSDCgrMxc5S-4gQXhhUD35RXFr8jUcZiGbvZuXMmhFpRTqmZZgv4ftRgKqRkVGPvm4nWn9ZRDY3DO3UqSz8Mfch-BJd8-RO0Gdvqfh14hWzjrEN7vU1XcnJgfWJG6S39h-yLNAyuhz7wOKrVNYwGAc3O4XTbwX05C0-d1I1dYAcgF7pXyGdArjxstEyLl~8j8kfq8h0PydaBUy8Tk32VKhyvWZ6GWMjWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198966&amp;ar=2469968&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw021f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Heatmaps of Pearson’s product-moment correlation coefficient ( <em>r</em> ) for codon usage versus <em>S</em><sub>CDS</sub><sup>G-C</sup> . ( <strong>a–d)</strong><em>S</em><sub>CDS</sub><sup>G-C</sup> is calculated at the third-ff, the first, second, and the third-tf codon sites, in genomic data. ( <strong>e and f)</strong><em>S</em><sub>CDS</sub><sup>G-C</sup> is calculated at third-tf of the artificial sequences produced by our synonymous-codon GC biased model and the null-hypothesis model. Columns indicate the base composition of the codon site at which skews are computed. Stippled cells depict negative <em>r</em> , non-stippled cells depict positive <em>r</em> . The intensity of the color indicates the degree of correlation, as recited in the color key. +1, perfect positive correlation; −1, perfect negative correlation; 0, no correlation. Start and stop codons are excluded from our calculations. ATG (start codon) and, in some cases, TGA (opal stop codon) are also found inside CDS, the first encoding for Methionine and the second for Tryptophane or Selenocysteine, and thus correlations of <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> with [ATG] and [TGA] appear in the heatmaps. The six-fold degenerate codons of Leucine and Arginine are labeled in the heatmaps. For further remarks see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> . Codons used in calculations of skews at third 4- and 2-fold-degenerate sites are marked with ‘ <strong>*</strong> ’ and ‘•’, respectively. ‘start’, initiation codon; ‘stop’, termination codons. </p></div></div><p> If the observed compositional asymmetries resulted primarily from synonymous codon usage, clear correlation patterns should appear when considering third-ff sites, where all substitutions are synonymous. <a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> However, <em>S</em><sub>CDS</sub><sup>G-C</sup> at third-ff sites displays only weak correlation with codon frequencies. At the first, second, and the third-tf sites <em>S</em><sub>CDS</sub><sup>G-C</sup> is associated with codon frequencies in a way that distinguishes between A/T- and G/C-ending codons. Namely, at third-tf and first sites <em>r</em> is positive for A/T-ending codons and negative for G/C-ending codons, while the inverse holds true at second sites. These rather structured patterns, which are also evident in the case of <em>S</em><sub>CDS</sub><sup>A-T</sup> ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> ), are, <em>prima facie</em> , counterintuitive. Third-tf sites are susceptible only to GC-changing substitutions (A↔G or C↔T) without altering the encoding amino-acid. Only codons for Arg and Leu allow synonymous, GC-changing substitutions in both first and third position, while substitutions in the remaining first and all second sites are non-synonymous. If skews at the first, second, and the third-tf sites resulted primarily from selective constraints, they should be correlated with the frequencies of codons specifying particular amino-acids. <a reveal-id="dsw021-B2" data-open="dsw021-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Instead, their values respond to the GC% of the third codon sites. In the evolutionary timescale, GC% changes much faster at third than first and second codon sites, and thus more accurately reflects genome-wide mutational biases towards a particular GC content. <a reveal-id="dsw021-B60" data-open="dsw021-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a></p><div class="block-child-p"> Previous studies documented that species with similar GC content share similar patterns of codon and amino-acid usage. <a reveal-id="dsw021-B31" data-open="dsw021-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsw021-B61 dsw021-B62 dsw021-B63" data-open="dsw021-B61 dsw021-B62 dsw021-B63" class="link link-ref link-reveal xref-bibr"><sup>61–63</sup></a> These empirical relations imply a direction of causality from GC-biased substitutions to codon and amino-acid usage, and not vice versa, since codon frequencies can be arranged in many different ways resulting in the same GC content. <a reveal-id="dsw021-B60" data-open="dsw021-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsw021-B64" data-open="dsw021-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Motivated by this, we put forward a simple model in which the codon usage of an artificial sequence is determined solely by its GC content, taking into account the GC variability within groups of synonymous codons. We formulate the response of each codon to the GC content of the sequence as a function with three parameters: <em> n <sub>i</sub></em> , <em> R <sub>i</sub></em><sup>GC</sup> , and <em> R <sub>i</sub></em><sup>AT</sup> . <em> n <sub>i</sub></em> is the number of G and C of the <em>i</em> th codon. <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msubsup><mi>R</mi><mi>i</mi><mrow><mtext>GC</mtext></mrow></msubsup><mo>=</mo><mfrac><mrow><msub><mrow><mtext>GC</mtext></mrow><mi>i</mi></msub></mrow><mrow><mfrac><mtext>1</mtext><mrow><msub><mi>a</mi><mi>i</mi></msub></mrow></mfrac><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mtext>1</mtext></mrow><mrow><msub><mi>a</mi><mi>i</mi></msub></mrow></msubsup><mrow><msub><mrow><mtext>GC</mtext></mrow><mi>j</mi></msub></mrow></mstyle></mrow></mfrac></mrow></mrow></math></div></div></div> where GC <sub><em>i</em></sub> is the GC content of the <em>i</em> th codon and <em> a <sub>i</sub></em> the number of codons in its synonymous group. Similarly <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msubsup><mi>R</mi><mi>i</mi><mrow><mtext>AT</mtext></mrow></msubsup><mo>=</mo><mfrac><mrow><msub><mrow><mtext>AT</mtext></mrow><mi>i</mi></msub></mrow><mrow><mfrac><mtext>1</mtext><mrow><msub><mi>a</mi><mi>i</mi></msub></mrow></mfrac><mstyle displaystyle="true"><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><msub><mi>a</mi><mi>i</mi></msub></mrow></msubsup><mrow><msub><mrow><mtext>AT</mtext></mrow><mi>j</mi></msub></mrow></mstyle></mrow></mfrac></mrow></mrow></math></div></div></div> where AT <sub><em>i </em></sub> =1 – GC <sub><em>i</em></sub> . We introduce <em> R <sub>i</sub></em><sup>GC</sup> and <em> R <sub>i</sub></em><sup>AT</sup> to assess the response of the <em>i</em> th codon to the overall GC content of the sequence, given the mean GC content of its synonymous group. Thus, <em> R <sub>i</sub></em><sup>GC</sup> and <em> R <sub>i</sub></em><sup>AT</sup> introduce synonymous-codon GC biases in our model. For a given GC content (GC) we generate an artificial sequence comprised of 10 <sup>6</sup> codons. Codon probabilities ( <em>P</em><sub>i</sub> ) are estimated as: <a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msub><mi>P</mi><mi>i</mi></msub><mo>=</mo><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mtext>GC</mtext></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><msub><mi>n</mi><mi>i</mi></msub><msubsup><mi>R</mi><mi>i</mi><mrow><mtext>GC</mtext></mrow></msubsup></mrow></msup><msup><mrow><mrow><mo stretchy="true">(</mo><mrow><mtext>1</mtext><mo>−</mo><mtext>GC</mtext></mrow><mo stretchy="true">)</mo></mrow></mrow><mrow><mrow><mo stretchy="true">(</mo><mrow><mtext>3</mtext><mo>−</mo><msub><mi>n</mi><mi>i</mi></msub></mrow><mo stretchy="true">)</mo></mrow><msubsup><mi>R</mi><mi>i</mi><mrow><mtext>AT</mtext></mrow></msubsup></mrow></msup></mrow></mrow></math></div></div></div> with their sum normalized to unity. </div><p> We produce a set of artificial sequences with their GC content ranging from 0.2 to 0.8, with a step of 0.005 ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span> ). Then, we compute Pearson’s <em>r</em> between site-specific skews and codon frequencies, as derived from the artificial sequences. Note that our model is not built to predict the codon usage actually observed in bacterial genomes. Instead, it is aimed to assess whether there is a causal link between strand asymmetries in CDSs and the structure of genetic code, driven by GC content variability, as implied by the heatmaps in <a reveal-id="dsw021-F4" data-open="dsw021-F4" class="link link-reveal link-table xref-fig">Figure 4b–d</a> . The parameters <em> n <sub>i</sub></em> , <em> R <sub>i</sub></em><sup>GC</sup> and <em> R <sub>i</sub></em><sup>AT</sup> are symmetric with respect to the complementary nucleotide bases and do not account for base order within codons ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span> ). Thus, any compositional asymmetries in the produced artificial sequences and their correlation with codon usage, can only be attributed to the partition of codons into synonymous groups and the GC varibility within these groups. </p><p> The corresponding heatmaps show no significant correlation between skews at third-ff sites and codon usage, but largely reproduce the correlation patterns detected in our genomic dataset when considering skews at third-tf sites (cf. <a reveal-id="dsw021-F4" data-open="dsw021-F4" class="link link-reveal link-table xref-fig">Figure 4d,e</a> ), first and second sites (cf. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> e–p). Namely, at third-tf and first sites of the artificial sequences <em>S</em><sub>CDS</sub><sup>A-T</sup> and <em>S</em><sub>CDS</sub><sup>G-C</sup> increase with A/T-ending codons and counter to G/C-ending ones. The correlation is reversed at second codon sites of these sequences ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> n and p). The only marked discrepancy between genomic data and our model regards <em>S</em><sub>CDS</sub><sup>A-T</sup> at second codon sites, where the inverse pattern of correlation is detected (cf. <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span> m and n). Among other factors, this discrepancy can be attributed to selective pressures, which have been previously associated with the emergence of atypical <em>S</em><sub>CDS</sub><sup>A-T</sup> patterns. <a reveal-id="dsw021-B16" data-open="dsw021-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p><p> Our model suggests that given the GC bias of synonymous codons ( <em> R <sub>i</sub></em><sup><strong>GC</strong></sup> and <em> R <sub>i</sub></em><sup><strong>AT</strong></sup> ) the interspecies variation of GC content suffices to yield the observed correlations between codon usage and site-specific skews along CDSs. To further test our argument, we consider a null-hypothesis model, in which <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msub><mi>P</mi><mi>i</mi></msub><mo>=</mo><msup><mrow><mo stretchy="false">(</mo><mtext>GC</mtext><mo stretchy="false">)</mo></mrow><mrow><msub><mi>n</mi><mi>i</mi></msub></mrow></msup><msup><mrow><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mtext>GC</mtext><mo stretchy="false">)</mo></mrow><mrow><mo stretchy="false">(</mo><mn>3</mn><mo>−</mo><msub><mi>n</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></msup></mrow></mrow></math></span> , and thus synonymous-codon GC biases ( <em> R <sub>i</sub></em><sup><strong>GC</strong></sup> and <em> R <sub>i</sub></em><sup><strong>AT</strong></sup> ) are not accounted for. Under this null-model the previously observed patterns collapse ( <a reveal-id="dsw021-F4" data-open="dsw021-F4" class="link link-reveal link-table xref-fig">Fig. 4f</a> ), indicating that synonymous-codon GC biases are indeed a decisive factor linking skews to codon usage. </p><p> GC mutational biases tailored with the structure of the genetic code can drastically modulate the nucleotide and codon composition of CDSs; <a reveal-id="dsw021-B65" data-open="dsw021-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsw021-B66" data-open="dsw021-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> and references given therein. Here we presented evidence that the synonymous-codon GC biases along with GC content variability among genomes, attributed to directional mutation pressure, though not <em>per se</em> strand-biased, can lead to asymmetries between coding and transcribed strand. This holds true even when selection of specific codons and amino-acids is not accounted for. This is more prominent at the first, second, and the third-tf sites, where such asymmetries are inherently imposed by the genetic code. Our findings imply that, given the overall GC content of the genome, the partition of codons into synonymous groups provides the ground for an <em>a priori</em> estimation of codon site-specific skews. Such skews correspond to an uneven baseline distribution of nucleotides among different codon sites, which should be considered when estimating the expected number of substitutions per site in comparative genomic analysis. </p><p> As discussed earlier and shown in <a reveal-id="dsw021-F4" data-open="dsw021-F4" class="link link-reveal link-table xref-fig">Figure 4a</a> , in third-ff sites strand asymmetries exhibit weak correlations with codon usage. Subsequently, we address the role of specific molecular systems, which may induce replication- and transcription-associated biases, in shaping third-ff asymmetries. </p>                            <h3 scrollto-destination=87198979 id="87198979" class="section-title" >3.4 Transcription-coupled repair and PolIII α-subunit isoforms induce whole-genome mutational biases</h3>
<p>Intra-strand asymmetries provide valuable insight into molecular processes with distinct modes of action on each DNA strand. We analyse the distribution of skews in our genomic dataset to infer substitution asymmetries related to two such processes, namely transcription-coupled repair (TCR) and DNA replication. In the following analysis, all comparisons of skew distributions are performed using the two-tailed Wilcoxon's rank sum test.</p><p> Bacteria with active TCR possess homologs of the <em>mfd</em> gene, which codes for the transcription-repair coupling factor (TRCF). <a reveal-id="dsw021-B67" data-open="dsw021-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> To assess the effect of TCR on mutational biases, we compare the distribution of transcription-associated (Trs) skews in relation to the existence of the <em>mfd</em> gene (TCR <sup><strong>+</strong></sup> : <em>mfd</em> present, TCR <sup><strong>−</strong></sup> : <em>mfd</em> absent). As seen in <a reveal-id="dsw021-T1" data-open="dsw021-T1" class="link link-reveal link-table xref-fig">Table 1A</a> , the TCR <sup><strong>+ </strong></sup> group shows a significantly different <em>S</em><sub>Trs</sub><sup>A-T</sup> distribution compared with the TCR <sup><strong>− </strong></sup> one ( <em>P</em><em>-</em> value &lt; 10 <sup>−5</sup> ). Moreover, three out of six relative abundance skews ( <em>P</em><sub>Trs</sub><sup>GG-CC</sup> , <em>P</em><sub>Trs</sub><sup>AA-TT</sup> , and <em>P</em><sub>Trs</sub><sup>AC-GT</sup> ) strongly correlate with the presence of <em>mfd</em> ( <em>P</em><em>-</em> value &lt; 10 <sup>−7</sup> ), while there is also a moderate differentiation of <em>P</em><sub>Trs</sub><sup>GA-TC</sup> distribution in TCR <sup><strong>+ </strong></sup> and TCR <sup><strong>− </strong></sup> bacteria ( <em>P</em><em>-</em> value &lt; 0.05). </p>                        <a id="87198985" scrollto-destination="87198985"></a>
<div content-id="dsw021-T1" class="table-modal"><div class="table-wrap"><div id="dsw021-T1" class="table-wrap-title" data-id="dsw021-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Analysis of skew distributions in relation to TCR and PolIII α-subunit</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2"><strong>A.</strong> Transcription-associated (Trs) skews  </th><th><strong><em>S</em><sub>Trs</sub><sup>A-T</sup></strong> </th><th><strong><em>S</em><sub>Trs</sub><sup>G-C</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AG-CT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GA-TC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GG-CC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AA-TT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AC-GT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>CA-TG</sup></strong> </th></tr></thead><tbody><tr><td colspan="2">  TCR <sup>−</sup> </td><td char=".">−0.0642 </td><td char=".">−0.00305 </td><td char=".">−0.178 </td><td char=".">−0.0193 </td><td char=".">−0.205 </td><td char=".">0.0333 </td><td char=".">−0.0806 </td><td char=".">0.0604 </td></tr><tr><td colspan="2">  TCR <sup>+</sup> </td><td char=".">−0.156 </td><td char=".">−0.0412 </td><td char=".">−0.191 </td><td char=".">0.0358 </td><td char=".">0.173 </td><td char=".">−0.186 </td><td char=".">0.313 </td><td char=".">0.0912 </td></tr><tr><td colspan="2"><em><strong>P</strong></em><strong>-value</strong> </td><td colspan="8"> </td></tr><tr><td> TCR <sup>−</sup> TCR <sup>+</sup> </td><td> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>—</strong> </td><td><strong>*</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"><strong>B.</strong> Replication-associated (Rep) skews  </td><td><strong><em>S</em><sub>Rep</sub><sup>A-T</sup></strong> </td><td><strong><em>S</em><sub>Rep</sub><sup>G-C</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AG-CT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GA-TC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GG-CC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AA-TT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AC-GT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>CA-TG</sup></strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"> dnaE </td><td char=".">−0.0357 </td><td char=".">0.0806 </td><td char=".">0.00521 </td><td char=".">0.0137 </td><td char=".">0.000299 </td><td char=".">0.0161 </td><td char=".">−0.0309 </td><td char=".">0.00614 </td></tr><tr><td colspan="2"> dnaE2 </td><td char=".">−0.07 </td><td char=".">0.0386 </td><td char=".">0.017 </td><td char=".">0.0152 </td><td char=".">0.0168 </td><td char=".">0.0371 </td><td char=".">−0.0532 </td><td char=".">0.0116 </td></tr><tr><td colspan="2"> polC </td><td char=".">0.0241 </td><td char=".">0.132 </td><td char=".">0.000189 </td><td char=".">0.012 </td><td char=".">−0.0222 </td><td char=".">−0.029 </td><td char=".">−0.0376 </td><td char=".">0.0203 </td></tr><tr><td><em><strong>P</strong></em><strong>-value</strong> </td></tr><tr><td>dnaEdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>*</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td>dnaEpolC </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>–</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td></tr><tr><td>polCdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>**</strong> </td><td><strong>**</strong> </td></tr></tbody></table></div><div content-id="dsw021-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2"><strong>A.</strong> Transcription-associated (Trs) skews  </th><th><strong><em>S</em><sub>Trs</sub><sup>A-T</sup></strong> </th><th><strong><em>S</em><sub>Trs</sub><sup>G-C</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AG-CT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GA-TC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GG-CC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AA-TT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AC-GT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>CA-TG</sup></strong> </th></tr></thead><tbody><tr><td colspan="2">  TCR <sup>−</sup> </td><td char=".">−0.0642 </td><td char=".">−0.00305 </td><td char=".">−0.178 </td><td char=".">−0.0193 </td><td char=".">−0.205 </td><td char=".">0.0333 </td><td char=".">−0.0806 </td><td char=".">0.0604 </td></tr><tr><td colspan="2">  TCR <sup>+</sup> </td><td char=".">−0.156 </td><td char=".">−0.0412 </td><td char=".">−0.191 </td><td char=".">0.0358 </td><td char=".">0.173 </td><td char=".">−0.186 </td><td char=".">0.313 </td><td char=".">0.0912 </td></tr><tr><td colspan="2"><em><strong>P</strong></em><strong>-value</strong> </td><td colspan="8"> </td></tr><tr><td> TCR <sup>−</sup> TCR <sup>+</sup> </td><td> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>—</strong> </td><td><strong>*</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"><strong>B.</strong> Replication-associated (Rep) skews  </td><td><strong><em>S</em><sub>Rep</sub><sup>A-T</sup></strong> </td><td><strong><em>S</em><sub>Rep</sub><sup>G-C</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AG-CT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GA-TC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GG-CC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AA-TT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AC-GT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>CA-TG</sup></strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"> dnaE </td><td char=".">−0.0357 </td><td char=".">0.0806 </td><td char=".">0.00521 </td><td char=".">0.0137 </td><td char=".">0.000299 </td><td char=".">0.0161 </td><td char=".">−0.0309 </td><td char=".">0.00614 </td></tr><tr><td colspan="2"> dnaE2 </td><td char=".">−0.07 </td><td char=".">0.0386 </td><td char=".">0.017 </td><td char=".">0.0152 </td><td char=".">0.0168 </td><td char=".">0.0371 </td><td char=".">−0.0532 </td><td char=".">0.0116 </td></tr><tr><td colspan="2"> polC </td><td char=".">0.0241 </td><td char=".">0.132 </td><td char=".">0.000189 </td><td char=".">0.012 </td><td char=".">−0.0222 </td><td char=".">−0.029 </td><td char=".">−0.0376 </td><td char=".">0.0203 </td></tr><tr><td><em><strong>P</strong></em><strong>-value</strong> </td></tr><tr><td>dnaEdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>*</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td>dnaEpolC </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>–</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td></tr><tr><td>polCdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>**</strong> </td><td><strong>**</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw021-TF1"><div class="fn"><p> For each group of genomes the median skew values are provided. We pairwise compare the skew distributions among the considered genome groups. The <em>p</em> -values for the two-tailed Wilcoxon’s rank sum tests conducted denote the statistical significance of their difference. —, <em>P</em> -value ≥ 0.05; *,0.05 &gt; <em>P</em> -value ≥ 0.01; **, 0.01 &gt; <em>P</em> -value ≥ 0.001; ***, <em>P</em> -value &lt; 0.001. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198985" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw021-T1" class="table-wrap-title" data-id="dsw021-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Analysis of skew distributions in relation to TCR and PolIII α-subunit</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2"><strong>A.</strong> Transcription-associated (Trs) skews  </th><th><strong><em>S</em><sub>Trs</sub><sup>A-T</sup></strong> </th><th><strong><em>S</em><sub>Trs</sub><sup>G-C</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AG-CT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GA-TC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GG-CC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AA-TT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AC-GT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>CA-TG</sup></strong> </th></tr></thead><tbody><tr><td colspan="2">  TCR <sup>−</sup> </td><td char=".">−0.0642 </td><td char=".">−0.00305 </td><td char=".">−0.178 </td><td char=".">−0.0193 </td><td char=".">−0.205 </td><td char=".">0.0333 </td><td char=".">−0.0806 </td><td char=".">0.0604 </td></tr><tr><td colspan="2">  TCR <sup>+</sup> </td><td char=".">−0.156 </td><td char=".">−0.0412 </td><td char=".">−0.191 </td><td char=".">0.0358 </td><td char=".">0.173 </td><td char=".">−0.186 </td><td char=".">0.313 </td><td char=".">0.0912 </td></tr><tr><td colspan="2"><em><strong>P</strong></em><strong>-value</strong> </td><td colspan="8"> </td></tr><tr><td> TCR <sup>−</sup> TCR <sup>+</sup> </td><td> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>—</strong> </td><td><strong>*</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"><strong>B.</strong> Replication-associated (Rep) skews  </td><td><strong><em>S</em><sub>Rep</sub><sup>A-T</sup></strong> </td><td><strong><em>S</em><sub>Rep</sub><sup>G-C</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AG-CT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GA-TC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GG-CC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AA-TT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AC-GT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>CA-TG</sup></strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"> dnaE </td><td char=".">−0.0357 </td><td char=".">0.0806 </td><td char=".">0.00521 </td><td char=".">0.0137 </td><td char=".">0.000299 </td><td char=".">0.0161 </td><td char=".">−0.0309 </td><td char=".">0.00614 </td></tr><tr><td colspan="2"> dnaE2 </td><td char=".">−0.07 </td><td char=".">0.0386 </td><td char=".">0.017 </td><td char=".">0.0152 </td><td char=".">0.0168 </td><td char=".">0.0371 </td><td char=".">−0.0532 </td><td char=".">0.0116 </td></tr><tr><td colspan="2"> polC </td><td char=".">0.0241 </td><td char=".">0.132 </td><td char=".">0.000189 </td><td char=".">0.012 </td><td char=".">−0.0222 </td><td char=".">−0.029 </td><td char=".">−0.0376 </td><td char=".">0.0203 </td></tr><tr><td><em><strong>P</strong></em><strong>-value</strong> </td></tr><tr><td>dnaEdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>*</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td>dnaEpolC </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>–</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td></tr><tr><td>polCdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>**</strong> </td><td><strong>**</strong> </td></tr></tbody></table></div><div content-id="dsw021-T1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2"><strong>A.</strong> Transcription-associated (Trs) skews  </th><th><strong><em>S</em><sub>Trs</sub><sup>A-T</sup></strong> </th><th><strong><em>S</em><sub>Trs</sub><sup>G-C</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AG-CT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GA-TC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>GG-CC</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AA-TT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>AC-GT</sup></strong> </th><th><strong><em>P</em><sub>Trs</sub><sup>CA-TG</sup></strong> </th></tr></thead><tbody><tr><td colspan="2">  TCR <sup>−</sup> </td><td char=".">−0.0642 </td><td char=".">−0.00305 </td><td char=".">−0.178 </td><td char=".">−0.0193 </td><td char=".">−0.205 </td><td char=".">0.0333 </td><td char=".">−0.0806 </td><td char=".">0.0604 </td></tr><tr><td colspan="2">  TCR <sup>+</sup> </td><td char=".">−0.156 </td><td char=".">−0.0412 </td><td char=".">−0.191 </td><td char=".">0.0358 </td><td char=".">0.173 </td><td char=".">−0.186 </td><td char=".">0.313 </td><td char=".">0.0912 </td></tr><tr><td colspan="2"><em><strong>P</strong></em><strong>-value</strong> </td><td colspan="8"> </td></tr><tr><td> TCR <sup>−</sup> TCR <sup>+</sup> </td><td> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>—</strong> </td><td><strong>*</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"><strong>B.</strong> Replication-associated (Rep) skews  </td><td><strong><em>S</em><sub>Rep</sub><sup>A-T</sup></strong> </td><td><strong><em>S</em><sub>Rep</sub><sup>G-C</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AG-CT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GA-TC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>GG-CC</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AA-TT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>AC-GT</sup></strong> </td><td><strong><em>P</em><sub>Rep</sub><sup>CA-TG</sup></strong> </td></tr><tr><td colspan="10"><hr /> </td></tr><tr><td colspan="2"> dnaE </td><td char=".">−0.0357 </td><td char=".">0.0806 </td><td char=".">0.00521 </td><td char=".">0.0137 </td><td char=".">0.000299 </td><td char=".">0.0161 </td><td char=".">−0.0309 </td><td char=".">0.00614 </td></tr><tr><td colspan="2"> dnaE2 </td><td char=".">−0.07 </td><td char=".">0.0386 </td><td char=".">0.017 </td><td char=".">0.0152 </td><td char=".">0.0168 </td><td char=".">0.0371 </td><td char=".">−0.0532 </td><td char=".">0.0116 </td></tr><tr><td colspan="2"> polC </td><td char=".">0.0241 </td><td char=".">0.132 </td><td char=".">0.000189 </td><td char=".">0.012 </td><td char=".">−0.0222 </td><td char=".">−0.029 </td><td char=".">−0.0376 </td><td char=".">0.0203 </td></tr><tr><td><em><strong>P</strong></em><strong>-value</strong> </td></tr><tr><td>dnaEdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>*</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td></tr><tr><td>dnaEpolC </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>–</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>**</strong> </td></tr><tr><td>polCdnaE2 </td><td> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>—</strong> </td><td><strong>***</strong> </td><td><strong>***</strong> </td><td><strong>**</strong> </td><td><strong>**</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw021-TF1"><div class="fn"><p> For each group of genomes the median skew values are provided. We pairwise compare the skew distributions among the considered genome groups. The <em>p</em> -values for the two-tailed Wilcoxon’s rank sum tests conducted denote the statistical significance of their difference. —, <em>P</em> -value ≥ 0.05; *,0.05 &gt; <em>P</em> -value ≥ 0.01; **, 0.01 &gt; <em>P</em> -value ≥ 0.001; ***, <em>P</em> -value &lt; 0.001. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87198985" target="_blank">View Large</a></div></div><p> TCR specifically targets lesions on the transcribed strand, <a reveal-id="dsw021-B68" data-open="dsw021-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> enhancing the bias of TAM, such as C→T transitions, towards the coding strand. <a reveal-id="dsw021-B26" data-open="dsw021-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsw021-B69" data-open="dsw021-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a><sup>,</sup><a reveal-id="dsw021-B70" data-open="dsw021-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Skew analysis captures this strand-bias, with the distribution of <em>S</em><sub>Trs</sub><sup>A-T</sup> (TCR <sup><strong>+</strong></sup> ) being shifted to the left of <em>S</em><sub>Trs</sub><sup>A-T</sup> (TCR <sup><strong>−</strong></sup> ) distribution, in accordance with an excess of C→T over G→A substitutions on the coding strand. However, there is no correlation of <em>S</em><sub>Trs</sub><sup>G-C</sup> with TCR ( <em>P</em><em>-</em> value = 0.239), indicating that this strand-bias does not originate solely from C→T asymmetries, in line with <a reveal-id="dsw021-B21" data-open="dsw021-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> . The rates of TAM are known to be context-dependent, with C→T being greatly accelerated when C occurs in pyrimidine dimers. <a reveal-id="dsw021-B71" data-open="dsw021-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> This may lead to the observed correlation of <em>mfd</em> with <em>P</em><sub>Trs</sub><sup>GG-CC</sup> and possibly with <em>P</em><sub>Trs</sub><sup>AA-TT</sup> . However, as seen in the case of <em>P</em><sub>Trs</sub><sup>AC-GT</sup> , <em>mfd</em> also strongly correlates with relative abundance skews of dinucleotides that cannot form pyrimidine dimers. Thus, the efficiency of TCR may itself depend on sequence context. </p><p> We extend our analysis to replication-induced substitution asymmetries. <a reveal-id="dsw021-B9" data-open="dsw021-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In bacteria, both DNA strands are synthesized by the <em>α</em> -subunit dimer of DNA PolIII. <a reveal-id="dsw021-B72" data-open="dsw021-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> We partition our genomic dataset into three groups, according to the dimeric combination of their <em>α</em> -subunit isoforms: dnaE (homodimer of DnaE1), dnaE2 (heterodimer of DnaE1 with DnaE2) and polC (heterodimer of PolC with either DnaE1 or DnaE3). <a reveal-id="dsw021-B62" data-open="dsw021-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> Note that the polC group is comprised of all Firmicutes and Tenericutes of our collection. </p><p> In a previous study <a reveal-id="dsw021-B73" data-open="dsw021-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> no correlation was found between skews and α-subunit isoforms. There, however, bacteria were grouped in relation to the existence of <em>pol</em> C only, while <em>dna</em> E2 was not accounted for. <a reveal-id="dsw021-T1" data-open="dsw021-T1" class="link link-reveal link-table xref-fig">Table 1B</a> shows that replication-associated (Rep) skews are strongly correlated to the grouping scheme we employ. Both <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>A-T</mtext></mrow></msubsup></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>G-C</mtext></mrow></msubsup></math></span> are distinctively distributed among the three groups ( <em>P</em><em>-</em> value ≪ 10 <sup>−</sup><sup>4</sup> ). Relative abundance skews also significantly correlate with the type of <em>α</em> -subunit dimers (i.e. <em>P</em><sub>Trs</sub><sup>AA-TT</sup> , <em>P</em><sub>Trs</sub><sup>GG-CC</sup> , <em>P</em><sub>Trs</sub><sup>AC-GT</sup> ). Moreover, PolC largely determines the sign of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>S</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>A-T</mtext></mrow></msubsup></math></span> (positive; see also <a reveal-id="dsw021-B23" data-open="dsw021-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> ), <em>P</em><sub>Rep</sub><sup>GG-CC</sup> and <em>P</em><sub>Rep</sub><sup>AA-TT</sup> (both negative). The other skews are of the same sign in all three groups. </p><p> When present, PolC replicates the leading strand, <a reveal-id="dsw021-B74" data-open="dsw021-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> while DnaE2 performs SOS-induced translesion synthesis. <a reveal-id="dsw021-B75" data-open="dsw021-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a><sup>,</sup><a reveal-id="dsw021-B76" data-open="dsw021-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> Both PolC and DnaE2 form heterodimers whose components possess distinct nucleotide incorporation and proofreading activities. Our results reveal that such asymmetries of α-subunit induce specific strand-biased substitutions on a genome-wide scale. In many cases these biases are neighbour-dependent, as indicated by relative abundance skews ( <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>AA-TT</mtext></mrow></msubsup></math></span> , <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>GG-CC</mtext></mrow></msubsup></math></span> , <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mtext>P</mtext></mrow><mrow><mtext>Rep</mtext></mrow><mrow><mtext>AC-GT</mtext></mrow></msubsup></math></span> ). In the dnaE group the replication of both strands is catalyzed by the same α-subunit isoform. Thus, the observed skews should be attributed to the intrinsic asymmetries of the replication fork. <a reveal-id="dsw021-B7" data-open="dsw021-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p> Analysis of skew distribution can also provide useful information about molecular processes, besides replication and transcription, which are not yet determined to act in a strand-specific manner. Since we detect no significant correlation between <em>S</em><sub>Trs</sub><sup>G-C</sup> and TCR, we compare the distribution of <em>S</em><sub>Trs</sub><sup>G-C</sup> in relation to other DNA repair systems ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5 and S6</a></span> ). We consider direct reversal of DNA damage, base excision repair, nucleotide excision repair, mismatch repair and recombination repair pathways. In 7 out of 20 cases studied, we detect a statistically significant correlation (adjusted <em>P</em> -value &lt; 0.01, see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span> ) between <em>S</em><sub>Trs</sub><sup>G</sup><sup><strong>-</strong></sup><sup>C</sup> and specific DNA repair systems. The presence of these systems is linked to a shift of the <em>S</em><sub>Trs</sub><sup>G-C</sup> distribution from lower to higher (signed) values, in a remarkably uniform way, indicating an inversion of mutational bias when the corresponding repair systems are active. </p>                            <h2 scrollto-destination=87198995 id="87198995" class="section-title" >4. Conclusions</h2>
<p>Strand-asymmetries are deeply rooted into DNA evolution. To our knowledge, this is the first work to provide solid evidence that nearest-neighbour preferences are a strand-biased feature of bacterial genomes. The relative abundance skews, which we herein introduce, quantify these biases to assess strand-asymmetries in context-dependent substitutions.</p><p>We show that skews along CDSs are distinct among different species but similar between genes of the same organism. Moreover, relative abundance skews are more strongly correlated with species phylogeny, even when compared with genomic signatures. Hence, CDS-associated substitution asymmetries do not emanate from evolutionary trends which are universal across bacteria, but are species-specific.</p><p>Site-specific and context-dependent skew analysis facilitates the inquiry about the origin of certain substitution asymmetries. When CDSs are subjected to mutational bias towards a particular GC content, the structure of the genetic code imposes site-specific strand-biases on nucleotide substitutions. This leads to baseline position-dependent strand-asymmetries that should be taken into account when considering the effect of directional mutation and purifying selection on biases in CDSs’ composition.</p><p> Furthermore, the molecular machinery of transcription and replication are implicated in strand-asymmetries. TCR results in substitution biases towards coding strands, and our skew analysis efficiently captures these asymmetries. The isoforms of the PolIII <em>α</em> -subunit introduce replication errors at different rates on each DNA strand. Our results indicate that these biases are prevalent enough to induce whole-genome substitution asymmetries, both context-dependent and -independent. </p><p><strong>Additional Figures and Tables</strong> are available in the following link: </p><p><a class="link link-uri" href="http://bio.demokritos.gr/index.php?option=com_content&amp;view=article&amp;id=165&amp;Itemid=167&amp;lang=el" target="_blank">http://bio.demokritos.gr/index.php?option=com_content&amp;view=article&amp;id=165&amp;Itemid=167&amp;lang=el</a></p>                            <h2 scrollto-destination=87199007 id="87199007" class="section-title" >Funding</h2>
<p>K.A.K. acknowledges support through a scholarship awarded by NCSR “Demokritos”. Funding to pay the Open Access publication charges for this article was provided by NCSR “Demokritos”. </p>                            <h2 scrollto-destination=87199011 id="87199011" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by a scholarship from National Center for Scientific Research ‘Demokritos’. We thank Prof Nikolaos Glykos, Dr George Giannakopoulos, Dr Panagiota Tsitoura, and Dr Dimitris Polychronopoulos for helpful discussion.</p>                            <h3 scrollto-destination=87199015 id="87199015" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=87199018 id="87199018" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw021-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
             <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Evolution of DNA base composition under no-strand-bias conditions when the substitution rates are not constant</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">719</div>
            –
            <div class="lpage">23</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20DNA%20base%20composition%20under%20no-strand-bias%20conditions%20when%20the%20substitution%20rates%20are%20not%20constant&amp;author=J.R.%20Lobry&amp;author=C.%20Lobry&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol&amp;volume=16&amp;pages=719-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10368950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sueoka</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="year">1995</div>
            , 
            <div class="article-title">Intrastrand parity rules of DNA base composition and usage biases of synonymous codons</div>
            . 
            <div class="source">J. Mol. Evol</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">318</div>
            –
            <div class="lpage">25</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intrastrand%20parity%20rules%20of%20DNA%20base%20composition%20and%20usage%20biases%20of%20synonymous%20codons&amp;author=N.%20Sueoka&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol&amp;volume=40&amp;pages=318-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00163236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
          </div>
          <div class="year">1995</div>
            , 
            <div class="article-title">Properties of a general model of DNA evolution under no-strand-bias conditions</div>
            . 
            <div class="source">J. Mol. Evol</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">326</div>
            –
            <div class="lpage">30</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20of%20a%20general%20model%20of%20DNA%20evolution%20under%20no-strand-bias%20conditions&amp;author=J.R.%20Lobry&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol&amp;volume=40&amp;pages=326-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00163237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Prabhu</div>
               <div class="given-names">V.V.</div>
            </div>
          </div>
          <div class="year">1993</div>
            , 
            <div class="article-title">Symmetry observations in long nucleotide sequences</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">2797</div>
            –
            <div class="lpage">800</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Symmetry%20observations%20in%20long%20nucleotide%20sequences&amp;author=V.V.%20Prabhu&amp;publication_year=1993&amp;journal=Nucleic%20Acids%20Res&amp;volume=21&amp;pages=2797-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.12.2797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.12.2797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.12.2797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8332488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bell</div>
               <div class="given-names">S.J.</div>
            </div>
             <div class="name">
              <div class="surname">Forsdyke</div>
               <div class="given-names">D.R.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Accounting units in DNA</div>
            . 
            <div class="source">J. Theor. Biol</div> 
            ., 
            <div class="volume">197</div>
            , 
            <div class="fpage">51</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accounting%20units%20in%20DNA&amp;author=S.J.%20Bell&amp;author=D.R.%20Forsdyke&amp;publication_year=1999&amp;journal=J.%20Theor.%20Biol&amp;volume=197&amp;pages=51-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1998.0857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1998.0857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1998.0857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10036207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mitchell</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Bridge</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">A test of Chargaff’s second rule</div>
            . 
            <div class="source">Biochem. Biophys. Res. Commun</div> 
            ., 
            <div class="volume">340</div>
            , 
            <div class="fpage">90</div>
            –
            <div class="lpage">4</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20test%20of%20Chargaff%E2%80%99s%20second%20rule&amp;author=D.%20Mitchell&amp;author=R.%20Bridge&amp;publication_year=2006&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=340&amp;pages=90-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2005.11.160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2005.11.160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2005.11.160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16364245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
          </div>
          <div class="year">1996</div>
            , 
            <div class="article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">13</div>
            , 
            <div class="fpage">660</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%20in%20the%20two%20DNA%20strands%20of%20bacteria&amp;author=J.R.%20Lobry&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol&amp;volume=13&amp;pages=660-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
             <div class="name">
              <div class="surname">Sueoka</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Asymmetric directional mutation pressures in bacteria</div>
            . 
            <div class="source">Genome Biol</div> 
            ., 
            <div class="volume">3</div>
            , 
            <div class="fpage">RESEARCH0058</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20directional%20mutation%20pressures%20in%20bacteria&amp;author=J.R.%20Lobry&amp;author=N.%20Sueoka&amp;publication_year=2002&amp;journal=Genome%20Biol&amp;volume=3&amp;pages=RESEARCH0058" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2002-3-10-research0058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2002-3-10-research0058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2002-3-10-research0058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12372146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.</div>
            </div>
             <div class="name">
              <div class="surname">Danchin</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Viari</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Universal replication biases in bacteria</div>
            . 
            <div class="source">Mol. Microbiol</div> 
            ., 
            <div class="volume">32</div>
            , 
            <div class="fpage">11</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20replication%20biases%20in%20bacteria&amp;author=E.P.%20Rocha&amp;author=A.%20Danchin&amp;author=A.%20Viari&amp;publication_year=1999&amp;journal=Mol.%20Microbiol&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01334.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1999.01334.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1999.01334.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10216855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mrázek</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Strand compositional asymmetry in bacterial and large viral genomes</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            , 
            <div class="volume">95</div>
            , 
            <div class="fpage">3720</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20compositional%20asymmetry%20in%20bacterial%20and%20large%20viral%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=3720-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.7.3720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.7.3720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.7.3720"> </span></div> </div></div></div></div></div><div content-id="dsw021-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nikolaou</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Almirantis</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">A study on the correlation of nucleotide skews and the positioning of the origin of replication: different modes of replication in bacterial species</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">33</div>
            , 
            <div class="fpage">6816</div>
            –
            <div class="lpage">22</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20study%20on%20the%20correlation%20of%20nucleotide%20skews%20and%20the%20positioning%20of%20the%20origin%20of%20replication%3A%20different%20modes%20of%20replication%20in%20bacterial%20species&amp;author=C.%20Nikolaou&amp;author=Y.%20Almirantis&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=6816-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki988" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki988" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki988"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16321966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tillier</div>
               <div class="given-names">E.R.</div>
            </div>
             <div class="name">
              <div class="surname">Collins</div>
               <div class="given-names">R. A.</div>
            </div>
          </div>
          <div class="year">2000</div>
            , 
            <div class="article-title">The contributions of replication orientation, gene direction, and signal sequences to base-composition asymmetries in bacterial genomes</div>
            . 
            <div class="source">J. Mol. Evol</div> 
            ., 
            <div class="volume">50</div>
            , 
            <div class="fpage">249</div>
            –
            <div class="lpage">57</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20contributions%20of%20replication%20orientation%2C%20gene%20direction%2C%20and%20signal%20sequences%20to%20base-composition%20asymmetries%20in%20bacterial%20genomes&amp;author=E.R.%20Tillier&amp;author=R.%20A.%20Collins&amp;publication_year=2000&amp;journal=J.%20Mol.%20Evol&amp;volume=50&amp;pages=249-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002399910029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002399910029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002399910029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10754068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.C.</div>
            </div>
             <div class="name">
              <div class="surname">Touchon</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Feil</div>
               <div class="given-names">E.J.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Similar compositional biases are caused by very different mutational effects</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">1537</div>
            –
            <div class="lpage">47</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similar%20compositional%20biases%20are%20caused%20by%20very%20different%20mutational%20effects&amp;author=E.P.C.%20Rocha&amp;author=M.%20Touchon&amp;author=E.J.%20Feil&amp;publication_year=2006&amp;journal=Genome%20Res&amp;volume=16&amp;pages=1537-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5525106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5525106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5525106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17068325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Morton</div>
               <div class="given-names">R.A.</div>
            </div>
             <div class="name">
              <div class="surname">Morton</div>
               <div class="given-names">B.R.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Separating the effects of mutation and selection in producing DNA skew in bacterial chromosomes</div>
            . 
            <div class="source">BMC Genomics</div> 
            , 
            <div class="volume">8</div>
            , 
            <div class="fpage">369</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Separating%20the%20effects%20of%20mutation%20and%20selection%20in%20producing%20DNA%20skew%20in%20bacterial%20chromosomes&amp;author=R.A.%20Morton&amp;author=B.R.%20Morton&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=369" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-369" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-369" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-369"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17935620" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Frank</div>
               <div class="given-names">A.C.</div>
            </div>
             <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms</div>
            . 
            <div class="source">Gene</div> 
            , 
            <div class="volume">238</div>
            , 
            <div class="fpage">65</div>
            –
            <div class="lpage">77</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%3A%20a%20review%20of%20possible%20underlying%20mutational%20or%20selective%20mechanisms&amp;author=A.C.%20Frank&amp;author=J.R.%20Lobry&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=65-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00297-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00297-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00297-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Charneski</div>
               <div class="given-names">C.A.</div>
            </div>
             <div class="name">
              <div class="surname">Honti</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Bryant</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Hurst</div>
               <div class="given-names">L.D.</div>
            </div>
             <div class="name">
              <div class="surname">Feil</div>
               <div class="given-names">E.J.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Atypical AT skew in Firmicute genomes results from selection and not from mutation</div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">e1002283</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Atypical%20AT%20skew%20in%20Firmicute%20genomes%20results%20from%20selection%20and%20not%20from%20mutation&amp;author=C.A.%20Charneski&amp;author=F.%20Honti&amp;author=J.M.%20Bryant&amp;author=L.D.%20Hurst&amp;author=E.J.%20Feil&amp;publication_year=2011&amp;journal=PLoS%20Genet&amp;volume=7&amp;pages=e1002283" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21935355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nikolaou</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Almirantis</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Deviations from Chargaff’s second parity rule in organellar DNA Insights into the evolution of organellar genomes</div>
            . 
            <div class="source">Gene</div> 
            , 
            <div class="volume">381</div>
            , 
            <div class="fpage">34</div>
            –
            <div class="lpage">41</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deviations%20from%20Chargaff%E2%80%99s%20second%20parity%20rule%20in%20organellar%20DNA%20Insights%20into%20the%20evolution%20of%20organellar%20genomes&amp;author=C.%20Nikolaou&amp;author=Y.%20Almirantis&amp;publication_year=2006&amp;journal=Gene&amp;volume=381&amp;pages=34-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.06.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.06.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.06.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16893615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.C.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">The organization of the bacterial genome</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">42</div>
            , 
            <div class="fpage">211</div>
            –
            <div class="lpage">33</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20organization%20of%20the%20bacterial%20genome&amp;author=E.P.C.%20Rocha&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet&amp;volume=42&amp;pages=211-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18605898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Klasson</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Andersson</div>
               <div class="given-names">S.G.E.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Strong asymmetric mutation bias in endosymbiont genomes coincide with loss of genes for replication restart pathways</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">23</div>
            , 
            <div class="fpage">1031</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20asymmetric%20mutation%20bias%20in%20endosymbiont%20genomes%20coincide%20with%20loss%20of%20genes%20for%20replication%20restart%20pathways&amp;author=L.%20Klasson&amp;author=S.G.E.%20Andersson&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol&amp;volume=23&amp;pages=1031-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16476690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Danchin</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Genomes and evolution</div>
            . 
            <div class="source">Curr. Issues Mol. Biol</div> 
            ., 
            <div class="volume">5</div>
            , 
            <div class="fpage">37</div>
            –
            <div class="lpage">42</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomes%20and%20evolution&amp;author=A.%20Danchin&amp;publication_year=2003&amp;journal=Curr.%20Issues%20Mol.%20Biol&amp;volume=5&amp;pages=37-42" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12793527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.</div>
            </div>
             <div class="name">
              <div class="surname">Danchin</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">Ongoing evolution of strand composition in bacterial genomes</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">18</div>
            , 
            <div class="fpage">1789</div>
            –
            <div class="lpage">99</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ongoing%20evolution%20of%20strand%20composition%20in%20bacterial%20genomes&amp;author=E.P.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol&amp;volume=18&amp;pages=1789-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003966" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003966" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003966"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11504858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Necşulea</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Lobry</div>
               <div class="given-names">J.R.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">A new method for assessing the effect of replication on DNA base composition asymmetry</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">24</div>
            , 
            <div class="fpage">2169</div>
            –
            <div class="lpage">79</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20assessing%20the%20effect%20of%20replication%20on%20DNA%20base%20composition%20asymmetry&amp;author=A.%20Nec%C5%9Fulea&amp;author=J.R.%20Lobry&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=2169-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Worning</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Jensen</div>
               <div class="given-names">L.J.</div>
            </div>
             <div class="name">
              <div class="surname">Hallin</div>
               <div class="given-names">P.F.</div>
            </div>
             <div class="name">
              <div class="surname">Staerfeldt</div>
               <div class="given-names">H.-H.</div>
            </div>
             <div class="name">
              <div class="surname">Ussery</div>
               <div class="given-names">D.W.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Origin of replication in circular prokaryotic chromosomes</div>
            . 
            <div class="source">Environ. Microbiol</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">353</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20replication%20in%20circular%20prokaryotic%20chromosomes&amp;author=P.%20Worning&amp;author=L.J.%20Jensen&amp;author=P.F.%20Hallin&amp;author=H.-H.%20Staerfeldt&amp;author=D.W.%20Ussery&amp;publication_year=2006&amp;journal=Environ.%20Microbiol&amp;volume=8&amp;pages=353-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2006.8.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2006.8.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2006.8.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bell</div>
               <div class="given-names">S.J.</div>
            </div>
             <div class="name">
              <div class="surname">Forsdyke</div>
               <div class="given-names">D.R.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Deviations from Chargaff’s second parity rule correlate with direction of transcription</div>
            . 
            <div class="source">J. Theor. Biol</div> 
            ., 
            <div class="volume">197</div>
            , 
            <div class="fpage">63</div>
            –
            <div class="lpage">76</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deviations%20from%20Chargaff%E2%80%99s%20second%20parity%20rule%20correlate%20with%20direction%20of%20transcription&amp;author=S.J.%20Bell&amp;author=D.R.%20Forsdyke&amp;publication_year=1999&amp;journal=J.%20Theor.%20Biol&amp;volume=197&amp;pages=63-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1998.0858" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1998.0858" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1998.0858"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10036208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lopez</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Philippe</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">
              Composition strand asymmetries in prokaryotic genomes: mutational bias and biased gene orientation. 
              <em>C.</em></div><div class="source">R. Acad. Sci. III</div> 
            ., 
            <div class="volume">324</div>
            , 
            <div class="fpage">201</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Composition%20strand%20asymmetries%20in%20prokaryotic%20genomes%3A%20mutational%20bias%20and%20biased%20gene%20orientation.%20C.&amp;author=P.%20Lopez&amp;author=H.%20Philippe&amp;publication_year=2001&amp;journal=R.%20Acad.%20Sci.%20III&amp;volume=324&amp;pages=201-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0764-4469(00)01298-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0764-4469(00)01298-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0764-4469(00)01298-1"> </span></div> </div></div></div></div></div><div content-id="dsw021-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Francino</div>
               <div class="given-names">M.P.</div>
            </div>
             <div class="name">
              <div class="surname">Chao</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Riley</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Ochman</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">1996</div>
            , 
            <div class="article-title">Asymmetries generated by transcription-coupled repair in enterobacterial genes</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">272</div>
            , 
            <div class="fpage">107</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetries%20generated%20by%20transcription-coupled%20repair%20in%20enterobacterial%20genes&amp;author=M.P.%20Francino&amp;author=L.%20Chao&amp;author=M.A.%20Riley&amp;author=H.%20Ochman&amp;publication_year=1996&amp;journal=Science&amp;volume=272&amp;pages=107-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.272.5258.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.272.5258.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.272.5258.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8600517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1981</div>
            , 
            <div class="article-title">Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system</div>
            . 
            <div class="source">J. Mol. Biol</div> 
            ., 
            <div class="volume">151</div>
            , 
            <div class="fpage">389</div>
            –
            <div class="lpage">409</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gouy</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Gautier</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">1982</div>
            , 
            <div class="article-title">Codon usage in bacteria: correlation with gene expressivity</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">10</div>
            , 
            <div class="fpage">7055</div>
            –
            <div class="lpage">74</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20bacteria%3A%20correlation%20with%20gene%20expressivity&amp;author=M.%20Gouy&amp;author=C.%20Gautier&amp;publication_year=1982&amp;journal=Nucleic%20Acids%20Res&amp;volume=10&amp;pages=7055-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.22.7055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.22.7055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6760125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bulmer</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1991</div>
            , 
            <div class="article-title">The selection-mutation-drift theory of synonymous codon usage</div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">129</div>
            , 
            <div class="fpage">897</div>
            –
            <div class="lpage">907</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20selection-mutation-drift%20theory%20of%20synonymous%20codon%20usage&amp;author=M.%20Bulmer&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=897-907" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1752426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Xia</div>
               <div class="given-names">X.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">How optimized is the translational machinery in Escherichia coli, Salmonella typhimurium and Saccharomyces cerevisiae?</div><div class="source">Genetics</div> 
            , 
            <div class="volume">149</div>
            , 
            <div class="fpage">37</div>
            –
            <div class="lpage">44</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20optimized%20is%20the%20translational%20machinery%20in%20Escherichia%20coli%2C%20Salmonella%20typhimurium%20and%20Saccharomyces%20cerevisiae%3F&amp;author=X.%20Xia&amp;publication_year=1998&amp;journal=Genetics&amp;volume=149&amp;pages=37-44" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9584084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sueoka</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="year">1961</div>
            , 
            <div class="article-title">Compositional correlation between deoxyribonucleic acid and protein</div>
            . 
            <div class="source">Cold Spring Harb. Symp. Quant. Biol</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">35</div>
            –
            <div class="lpage">43</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20correlation%20between%20deoxyribonucleic%20acid%20and%20protein&amp;author=N.%20Sueoka&amp;publication_year=1961&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol&amp;volume=26&amp;pages=35-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/SQB.1961.026.01.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2FSQB.1961.026.01.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2FSQB.1961.026.01.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13918160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">D’Onofrio</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Jabbari</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Musto</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Bernardi</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">The correlation of protein hydropathy with the base composition of coding sequences</div>
            . 
            <div class="source">Gene</div> 
            , 
            <div class="volume">238</div>
            , 
            <div class="fpage">3</div>
            –
            <div class="lpage">14</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20correlation%20of%20protein%20hydropathy%20with%20the%20base%20composition%20of%20coding%20sequences&amp;author=G.%20D%E2%80%99Onofrio&amp;author=K.%20Jabbari&amp;author=H.%20Musto&amp;author=G.%20Bernardi&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=3-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00257-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00257-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00257-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S.L.</div>
            </div>
             <div class="name">
              <div class="surname">Salzberg</div>
               <div class="given-names">A.J.</div>
            </div>
             <div class="name">
              <div class="surname">Kerlavage</div>
               <div class="given-names">A.R.</div>
            </div>
             <div class="name">
              <div class="surname">Tomb</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Skewed oligomers and origins of replication</div>
            . 
            <div class="source">Gene</div> 
            , 
            <div class="volume">217</div>
            , 
            <div class="fpage">57</div>
            –
            <div class="lpage">67</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Skewed%20oligomers%20and%20origins%20of%20replication&amp;author=S.L.%20Salzberg&amp;author=A.J.%20Salzberg&amp;author=A.R.%20Kerlavage&amp;author=J.F.%20Tomb&amp;publication_year=1998&amp;journal=Gene&amp;volume=217&amp;pages=57-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(98)00374-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(98)00374-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(98)00374-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9795135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mascher</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Schubert</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Scholz</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Friedel</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Patterns of nucleotide asymmetries in plant and animal genomes</div>
            . 
            <div class="source">Biosystems</div> 
            , 
            <div class="volume">111</div>
            , 
            <div class="fpage">181</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20nucleotide%20asymmetries%20in%20plant%20and%20animal%20genomes&amp;author=M.%20Mascher&amp;author=I.%20Schubert&amp;author=U.%20Scholz&amp;author=S.%20Friedel&amp;publication_year=2013&amp;journal=Biosystems&amp;volume=111&amp;pages=181-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biosystems.2013.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biosystems.2013.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biosystems.2013.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23438636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Global dinucleotide signatures and analysis of genomic heterogeneity</div>
            . 
            <div class="source">Curr. Opin. Microbiol</div> 
            ., 
            <div class="volume">1</div>
            , 
            <div class="fpage">598</div>
            –
            <div class="lpage">610</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20dinucleotide%20signatures%20and%20analysis%20of%20genomic%20heterogeneity&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=1&amp;pages=598-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(98)80095-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(98)80095-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(98)80095-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10066522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Josse</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Kaiser</div>
               <div class="given-names">A.D.</div>
            </div>
             <div class="name">
              <div class="surname">Kornberg</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1961</div>
            , 
            <div class="article-title">Enzymatic synthesis of deoxyribonucleic acid. VIII. Frequencies of nearest neighbor base sequences in deoxyribonucleic acid</div>
            . 
            <div class="source">J. Biol. Chem</div> 
            ., 
            <div class="volume">236</div>
            , 
            <div class="fpage">864</div>
            –
            <div class="lpage">75</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enzymatic%20synthesis%20of%20deoxyribonucleic%20acid.%20VIII.%20Frequencies%20of%20nearest%20neighbor%20base%20sequences%20in%20deoxyribonucleic%20acid&amp;author=J.%20Josse&amp;author=A.D.%20Kaiser&amp;author=A.%20Kornberg&amp;publication_year=1961&amp;journal=J.%20Biol.%20Chem&amp;volume=236&amp;pages=864-75" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13790780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nussinov</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">1981</div>
            , 
            <div class="article-title">Nearest neighbor nucleotide patterns. Structural and biological implications</div>
            . 
            <div class="source">J. Biol. Chem</div> 
            ., 
            <div class="volume">256</div>
            , 
            <div class="fpage">8458</div>
            –
            <div class="lpage">62</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nearest%20neighbor%20nucleotide%20patterns.%20Structural%20and%20biological%20implications&amp;author=R.%20Nussinov&amp;publication_year=1981&amp;journal=J.%20Biol.%20Chem&amp;volume=256&amp;pages=8458-62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6943145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nussinov</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">1984</div>
            , 
            <div class="article-title">Strong doublet preferences in nucleotide sequences and DNA geometry</div>
            . 
            <div class="source">J. Mol. Evol</div> 
            ., 
            <div class="volume">20</div>
            , 
            <div class="fpage">111</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20doublet%20preferences%20in%20nucleotide%20sequences%20and%20DNA%20geometry&amp;author=R.%20Nussinov&amp;publication_year=1984&amp;journal=J.%20Mol.%20Evol&amp;volume=20&amp;pages=111-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02257371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02257371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02257371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6433029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shioiri</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Takahata</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">Skew of mononucleotide frequencies, relative abundance of dinucleotides, and DNA strand asymmetry</div>
            . 
            <div class="source">J. Mol. Evol</div> 
            ., 
            <div class="volume">53</div>
            , 
            <div class="fpage">364</div>
            –
            <div class="lpage">76</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Skew%20of%20mononucleotide%20frequencies%2C%20relative%20abundance%20of%20dinucleotides%2C%20and%20DNA%20strand%20asymmetry&amp;author=C.%20Shioiri&amp;author=N.%20Takahata&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol&amp;volume=53&amp;pages=364-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11675596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Baisnée</div>
               <div class="given-names">P.-F.</div>
            </div>
             <div class="name">
              <div class="surname">Hampson</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Baldi</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Why are complementary DNA strands symmetric?</div><div class="source">Bioinformatics</div> 
            , 
            <div class="volume">18</div>
            , 
            <div class="fpage">1021</div>
            –
            <div class="lpage">33</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20are%20complementary%20DNA%20strands%20symmetric%3F&amp;author=P.-F.%20Baisn%C3%A9e&amp;author=S.%20Hampson&amp;author=P.%20Baldi&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=1021-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.8.1021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.8.1021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.8.1021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gao</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Luo</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">C.-T.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">DoriC 5.0: an updated database of oriC regions in both bacterial and archaeal genomes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">41</div>
            , 
            <div class="fpage">D90</div>
            –
            <div class="lpage">3</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DoriC%205.0%3A%20an%20updated%20database%20of%20oriC%20regions%20in%20both%20bacterial%20and%20archaeal%20genomes&amp;author=F.%20Gao&amp;author=H.%20Luo&amp;author=C.-T.%20Zhang&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D90-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23093601" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mao</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Yin</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Xu</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The percentage of bacterial genes on leading versus lagging strands is influenced by multiple balancing forces</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">8210</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20percentage%20of%20bacterial%20genes%20on%20leading%20versus%20lagging%20strands%20is%20influenced%20by%20multiple%20balancing%20forces&amp;author=X.%20Mao&amp;author=H.%20Zhang&amp;author=Y.%20Yin&amp;author=Y.%20Xu&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=8210-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22735706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Saha</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Goswami</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Dutta</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Association of purine asymmetry, strand-biased gene distribution and PolC within Firmicutes and beyond: a new appraisal</div>
            . 
            <div class="source">BMC Genomics</div> 
            , 
            <div class="volume">15</div>
            , 
            <div class="fpage">430</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20purine%20asymmetry%2C%20strand-biased%20gene%20distribution%20and%20PolC%20within%20Firmicutes%20and%20beyond%3A%20a%20new%20appraisal&amp;author=S.K.%20Saha&amp;author=A.%20Goswami&amp;author=C.%20Dutta&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=430" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-430" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-430" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-430"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24899249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zeileis</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Kleiber</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Krämer</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Hornik</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Testing and dating of structural changes in practice</div>
            . 
            <div class="source">Comput. Stat. Data Anal</div> 
            ., 
            <div class="volume">44</div>
            , 
            <div class="fpage">109</div>
            –
            <div class="lpage">23</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Testing%20and%20dating%20of%20structural%20changes%20in%20practice&amp;author=A.%20Zeileis&amp;author=C.%20Kleiber&amp;author=W.%20Kr%C3%A4mer&amp;author=K.%20Hornik&amp;publication_year=2003&amp;journal=Comput.%20Stat.%20Data%20Anal&amp;volume=44&amp;pages=109-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-9473(03)00030-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-9473(03)00030-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-9473(03)00030-6"> </span></div> </div></div></div></div></div><div content-id="dsw021-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zeileis</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Patnaik</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Testing, monitoring, and dating structural changes in exchange rate regimes</div>
            . 
            <div class="source">Comput. Stat. Data Anal</div> 
            ., 
            <div class="volume">54</div>
            , 
            <div class="fpage">1696</div>
            –
            <div class="lpage">706</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Testing%2C%20monitoring%2C%20and%20dating%20structural%20changes%20in%20exchange%20rate%20regimes&amp;author=A.%20Zeileis&amp;author=A.%20Shah&amp;author=I.%20Patnaik&amp;publication_year=2010&amp;journal=Comput.%20Stat.%20Data%20Anal&amp;volume=54&amp;pages=1696-706" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.csda.2009.12.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.csda.2009.12.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.csda.2009.12.005"> </span></div> </div></div></div></div></div><div content-id="dsw021-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zeileis</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Leisch</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Hornik</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Kleiber</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">strucchange t an R package for testing for structural change in linear regression models</div>
            . 
            <div class="source">J. Stat. Softw</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">1</div>
            –
            <div class="lpage">38</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=strucchange%20t%20an%20R%20package%20for%20testing%20for%20structural%20change%20in%20linear%20regression%20models&amp;author=A.%20Zeileis&amp;author=F.%20Leisch&amp;author=K.%20Hornik&amp;author=C.%20Kleiber&amp;publication_year=2002&amp;journal=J.%20Stat.%20Softw&amp;volume=7&amp;pages=1-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.18637/jss.v007.i02" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.18637%2Fjss.v007.i02" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.18637%2Fjss.v007.i02"> </span></div> </div></div></div></div></div><div content-id="dsw021-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kullback</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Leibler</div>
               <div class="given-names">R.A.</div>
            </div>
          </div>
          <div class="year">1951</div>
            , 
            <div class="article-title">On information and sufficiency</div>
            . 
            <div class="source">Ann. Math. Stat</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">79</div>
            –
            <div class="lpage">86</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20information%20and%20sufficiency&amp;author=S.%20Kullback&amp;author=R.A.%20Leibler&amp;publication_year=1951&amp;journal=Ann.%20Math.%20Stat&amp;volume=22&amp;pages=79-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/aoms/1177729694" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faoms%2F1177729694" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1214%2Faoms%2F1177729694"> </span></div> </div></div></div></div></div><div content-id="dsw021-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nye</div>
               <div class="given-names">T.M.W.</div>
            </div>
             <div class="name">
              <div class="surname">Liò</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Gilks</div>
               <div class="given-names">W.R.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">A novel algorithm and web-based tool for comparing two alternative phylogenetic trees</div>
            . 
            <div class="source">Bioinformatics</div> 
            , 
            <div class="volume">22</div>
            , 
            <div class="fpage">117</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20algorithm%20and%20web-based%20tool%20for%20comparing%20two%20alternative%20phylogenetic%20trees&amp;author=T.M.W.%20Nye&amp;author=P.%20Li%C3%B2&amp;author=W.R.%20Gilks&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=117-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16234319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Federhen</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The NCBI Taxonomy database</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">D136</div>
            –
            <div class="lpage">43</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20NCBI%20Taxonomy%20database&amp;author=S.%20Federhen&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D136-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1178" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22139910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Burge</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">1995</div>
            , 
            <div class="article-title">Dinucleotide relative abundance extremes: a genomic signature</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">11</div>
            , 
            <div class="fpage">283</div>
            –
            <div class="lpage">90</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dinucleotide%20relative%20abundance%20extremes%3A%20a%20genomic%20signature&amp;author=S.%20Karlin&amp;author=C.%20Burge&amp;publication_year=1995&amp;journal=Trends%20Genet&amp;volume=11&amp;pages=283-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(00)89076-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(00)89076-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(00)89076-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7482779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Campbell</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Mrázek</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Genome signature comparisons among prokaryote, plasmid, and mitochondrial DNA</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> 
            ., 
            <div class="volume">96</div>
            , 
            <div class="fpage">9184</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20signature%20comparisons%20among%20prokaryote%2C%20plasmid%2C%20and%20mitochondrial%20DNA&amp;author=A.%20Campbell&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Karlin&amp;publication_year=1999&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=96&amp;pages=9184-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.16.9184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.16.9184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.16.9184"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10430917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Powell</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Forslund</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Szklarczyk</div>
               <div class="given-names">D.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">eggNOG v4.0: nested orthology inference across 3686 organisms</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">42</div>
            , 
            <div class="fpage">D231</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=eggNOG%20v4.0%3A%20nested%20orthology%20inference%20across%203686%20organisms&amp;author=S.%20Powell&amp;author=K.%20Forslund&amp;author=D.%20Szklarczyk&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D231-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297252" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Arndt</div>
               <div class="given-names">P.F.k</div>
            </div>
             <div class="name">
              <div class="surname">Hwa</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Identification and measurement of neighbor-dependent nucleotide substitution processes</div>
            . 
            <div class="source">Bioinformatics</div> 
            , 
            <div class="volume">21</div>
            , 
            <div class="fpage">2322</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20measurement%20of%20neighbor-dependent%20nucleotide%20substitution%20processes&amp;author=P.F.k%20Arndt&amp;author=T.%20Hwa&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=2322-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15769841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chamary</div>
               <div class="given-names">J.-V.</div>
            </div>
             <div class="name">
              <div class="surname">Hurst</div>
               <div class="given-names">L. D.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Similar rates but different modes of sequence evolution in introns and at exonic silent sites in rodents: evidence for selectively driven codon usage</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">1014</div>
            –
            <div class="lpage">23</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similar%20rates%20but%20different%20modes%20of%20sequence%20evolution%20in%20introns%20and%20at%20exonic%20silent%20sites%20in%20rodents%3A%20evidence%20for%20selectively%20driven%20codon%20usage&amp;author=J.-V.%20Chamary&amp;author=L.%20D.%20Hurst&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol&amp;volume=21&amp;pages=1014-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15014158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Du</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Yuan</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Ma</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Song</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Xie</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">KEGG-PATH: Kyoto encyclopedia of genes and genomes-based pathway analysis using a path analysis model</div>
            . 
            <div class="source">Mol. Biosyst</div> 
            ., 
            <div class="volume">10</div>
            , 
            <div class="fpage">2441</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG-PATH%3A%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes-based%20pathway%20analysis%20using%20a%20path%20analysis%20model&amp;author=J.%20Du&amp;author=Z.%20Yuan&amp;author=Z.%20Ma&amp;author=J.%20Song&amp;author=X.%20Xie&amp;author=Y.%20Chen&amp;publication_year=2014&amp;journal=Mol.%20Biosyst&amp;volume=10&amp;pages=2441-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/C4MB00287C" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2FC4MB00287C" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2FC4MB00287C"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24994036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grigoriev</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">Analyzing genomes with cumulative skew diagrams</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">2286</div>
            –
            <div class="lpage">90</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20genomes%20with%20cumulative%20skew%20diagrams&amp;author=A.%20Grigoriev&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=2286-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.10.2286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.10.2286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.10.2286"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9580676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Francino</div>
               <div class="given-names">M.P.</div>
            </div>
             <div class="name">
              <div class="surname">Ochman</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">1997</div>
            , 
            <div class="article-title">Strand asymmetries in DNA evolution</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">13</div>
            , 
            <div class="fpage">240</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20asymmetries%20in%20DNA%20evolution&amp;author=M.P.%20Francino&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=Trends%20Genet&amp;volume=13&amp;pages=240-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01118-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01118-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01118-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9196330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Karlin</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Mrázek</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">1997</div>
            , 
            <div class="article-title">Compositional differences within and between eukaryotic genomes</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            , 
            <div class="volume">94</div>
            , 
            <div class="fpage">10227</div>
            –
            <div class="lpage">32</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20differences%20within%20and%20between%20eukaryotic%20genomes&amp;author=S.%20Karlin&amp;author=J.%20Mr%C3%A1zek&amp;publication_year=1997&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=10227-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.19.10227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.19.10227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.19.10227"> </span></div> </div></div></div></div></div><div content-id="dsw021-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Becker</div>
               <div class="given-names">R.A.</div>
            </div>
             <div class="name">
              <div class="surname">Chambers</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Wilks</div>
               <div class="given-names">A. R.</div>
            </div>
          </div>
          <div class="year">1988</div>
            , The new S language: a programming environment for data analysis and graphics. Pacific Grove, CA, USA: Wadsworth &amp; Brooks/Cole Advanced Books &amp; Software. ISBN 0-534-09192-X.
          </div></div></div></div><div content-id="dsw021-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Knight</div>
               <div class="given-names">R.D.</div>
            </div>
             <div class="name">
              <div class="surname">Freeland</div>
               <div class="given-names">S.J.</div>
            </div>
             <div class="name">
              <div class="surname">Landweber</div>
               <div class="given-names">L.F.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">A simple model based on mutation and selection explains trends in codon and amino-acid usage and GC composition within and across genomes</div>
            . 
            <div class="source">Genome Biol</div> 
            ., 
            <div class="volume">2</div>
            , 
            <div class="fpage">RESEARCH0010</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20model%20based%20on%20mutation%20and%20selection%20explains%20trends%20in%20codon%20and%20amino-acid%20usage%20and%20GC%20composition%20within%20and%20across%20genomes&amp;author=R.D.%20Knight&amp;author=S.J.%20Freeland&amp;author=L.F.%20Landweber&amp;publication_year=2001&amp;journal=Genome%20Biol&amp;volume=2&amp;pages=RESEARCH0010" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11305938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Muto</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Osawa</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">1987</div>
            , 
            <div class="article-title">The guanine and cytosine content of genomic DNA and bacterial evolution</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            , 
            <div class="volume">84</div>
            , 
            <div class="fpage">166</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20guanine%20and%20cytosine%20content%20of%20genomic%20DNA%20and%20bacterial%20evolution&amp;author=A.%20Muto&amp;author=S.%20Osawa&amp;publication_year=1987&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.1.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.1.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.1.166"> </span></div> </div></div></div></div></div><div content-id="dsw021-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hu</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Compositional dynamics of guanine and cytosine content in prokaryotic genomes</div>
            . 
            <div class="source">Res. Microbiol</div> 
            ., 
            <div class="volume">158</div>
            , 
            <div class="fpage">363</div>
            –
            <div class="lpage">70</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20dynamics%20of%20guanine%20and%20cytosine%20content%20in%20prokaryotic%20genomes&amp;author=J.%20Hu&amp;author=X.%20Zhao&amp;author=Z.%20Zhang&amp;author=J.%20Yu&amp;publication_year=2007&amp;journal=Res.%20Microbiol&amp;volume=158&amp;pages=363-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.resmic.2007.02.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.resmic.2007.02.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.resmic.2007.02.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17449227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sorimachi</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Okayasu</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Codon evolution is governed by linear formulas</div>
            . 
            <div class="source">Amino Acids</div> 
            , 
            <div class="volume">34</div>
            , 
            <div class="fpage">661</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20evolution%20is%20governed%20by%20linear%20formulas&amp;author=K.%20Sorimachi&amp;author=T.%20Okayasu&amp;publication_year=2008&amp;journal=Amino%20Acids&amp;volume=34&amp;pages=661-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00726-007-0024-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00726-007-0024-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00726-007-0024-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18180868" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Palidwor</div>
               <div class="given-names">G.A.</div>
            </div>
             <div class="name">
              <div class="surname">Perkins</div>
               <div class="given-names">T.J.</div>
            </div>
             <div class="name">
              <div class="surname">Xia</div>
               <div class="given-names">X.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">A general model of codon bias due to GC mutational bias</div>
            . 
            <div class="source">PLoS One</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">e13431</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20model%20of%20codon%20bias%20due%20to%20GC%20mutational%20bias&amp;author=G.A.%20Palidwor&amp;author=T.J.%20Perkins&amp;author=X.%20Xia&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e13431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0013431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0013431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0013431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21048949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">A content-centric organization of the genetic code</div>
            . 
            <div class="source">Genomics Proteomics Bioinformatics</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">1</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20content-centric%20organization%20of%20the%20genetic%20code&amp;author=J.%20Yu&amp;publication_year=2007&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=5&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1672-0229(07)60008-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-0229(07)60008-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1672-0229(07)60008-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17572358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Modeling compositional dynamics based on GC and purine contents of protein-coding sequences</div>
            . 
            <div class="source">Biol. Direct</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">63</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20compositional%20dynamics%20based%20on%20GC%20and%20purine%20contents%20of%20protein-coding%20sequences&amp;author=Z.%20Zhang&amp;author=J.%20Yu&amp;publication_year=2010&amp;journal=Biol.%20Direct&amp;volume=5&amp;pages=63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-5-63" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-5-63" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-5-63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21059261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Selby</div>
               <div class="given-names">C.P.</div>
            </div>
             <div class="name">
              <div class="surname">Witkin</div>
               <div class="given-names">E.M.</div>
            </div>
             <div class="name">
              <div class="surname">Sancar</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1991</div>
            , 
            <div class="article-title">Escherichia coli mfd mutant deficient in “mutation frequency decline” lacks strand-specific repair: in vitro complementation with purified coupling factor</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            , 
            <div class="volume">88</div>
            , 
            <div class="fpage">11574</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20mfd%20mutant%20deficient%20in%20%E2%80%9Cmutation%20frequency%20decline%E2%80%9D%20lacks%20strand-specific%20repair%3A%20in%20vitro%20complementation%20with%20purified%20coupling%20factor&amp;author=C.P.%20Selby&amp;author=E.M.%20Witkin&amp;author=A.%20Sancar&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=11574-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.24.11574" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.24.11574" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.24.11574"> </span></div> </div></div></div></div></div><div content-id="dsw021-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Bockrath</div>
               <div class="given-names">R.C.</div>
            </div>
             <div class="name">
              <div class="surname">Palmer</div>
               <div class="given-names">J. E.</div>
            </div>
          </div>
          <div class="year">1977</div>
            , 
            <div class="article-title">Differential repair of premutational UV-lesions at tRNA genes in E. coli</div>
            . 
            <div class="source">Mol. Gen. Genet</div> 
            ., 
            <div class="volume">156</div>
            , 
            <div class="fpage">133</div>
            –
            <div class="lpage">40</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20repair%20of%20premutational%20UV-lesions%20at%20tRNA%20genes%20in%20E.%20coli&amp;author=R.C.%20Bockrath&amp;author=J.%20E.%20Palmer&amp;publication_year=1977&amp;journal=Mol.%20Gen.%20Genet&amp;volume=156&amp;pages=133-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00283485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00283485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00283485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/340899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Oller</div>
               <div class="given-names">A.R.</div>
            </div>
             <div class="name">
              <div class="surname">Fijalkowska</div>
               <div class="given-names">I.J.</div>
            </div>
             <div class="name">
              <div class="surname">Dunn</div>
               <div class="given-names">R.L.</div>
            </div>
             <div class="name">
              <div class="surname">Schaaper</div>
               <div class="given-names">R.M.</div>
            </div>
          </div>
          <div class="year">1992</div>
            , 
            <div class="article-title">Transcription-repair coupling determines the strandedness of ultraviolet mutagenesis in Escherichia coli</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            , 
            <div class="volume">89</div>
            , 
            <div class="fpage">11036</div>
            –
            <div class="lpage">40</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-repair%20coupling%20determines%20the%20strandedness%20of%20ultraviolet%20mutagenesis%20in%20Escherichia%20coli&amp;author=A.R.%20Oller&amp;author=I.J.%20Fijalkowska&amp;author=R.L.%20Dunn&amp;author=R.M.%20Schaaper&amp;publication_year=1992&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=11036-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.22.11036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.22.11036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.22.11036"> </span></div> </div></div></div></div></div><div content-id="dsw021-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Deaconescu</div>
               <div class="given-names">A.M.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">RNA polymerase between lesion bypass and DNA repair</div>
            . 
            <div class="source">Cell. Mol. Life Sci</div> 
            ., 
            <div class="volume">70</div>
            , 
            <div class="fpage">4495</div>
            –
            <div class="lpage">509</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20polymerase%20between%20lesion%20bypass%20and%20DNA%20repair&amp;author=A.M.%20Deaconescu&amp;publication_year=2013&amp;journal=Cell.%20Mol.%20Life%20Sci&amp;volume=70&amp;pages=4495-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-013-1384-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-013-1384-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-013-1384-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23807206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Peng</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Shaw</div>
               <div class="given-names">B.R.</div>
            </div>
          </div>
          <div class="year">1996</div>
            , 
            <div class="article-title">Accelerated deamination of cytosine residues in UV-induced cyclobutane pyrimidine dimers leads to CC–&gt;TT transitions</div>
            . 
            <div class="source">Biochemistry</div> 
            , 
            <div class="volume">35</div>
            , 
            <div class="fpage">10172</div>
            –
            <div class="lpage">81</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20deamination%20of%20cytosine%20residues%20in%20UV-induced%20cyclobutane%20pyrimidine%20dimers%20leads%20to%20CC%E2%80%93%3ETT%20transitions&amp;author=W.%20Peng&amp;author=B.R.%20Shaw&amp;publication_year=1996&amp;journal=Biochemistry&amp;volume=35&amp;pages=10172-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi960001x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi960001x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi960001x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8756482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zhao</div>
               <div class="given-names">X.-Q.</div>
            </div>
             <div class="name">
              <div class="surname">Hu</div>
               <div class="given-names">J.-F.</div>
            </div>
             <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Comparative analysis of eubacterial DNA polymerase III alpha subunits</div>
            . 
            <div class="source">Genomics Proteomics Bioinformatics</div> 
            , 
            <div class="volume">4</div>
            , 
            <div class="fpage">203</div>
            –
            <div class="lpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20eubacterial%20DNA%20polymerase%20III%20alpha%20subunits&amp;author=X.-Q.%20Zhao&amp;author=J.-F.%20Hu&amp;author=J.%20Yu&amp;publication_year=2006&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=4&amp;pages=203-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1672-0229(07)60001-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-0229(07)60001-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1672-0229(07)60001-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17531796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">E.P.C.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Is there a role for replication fork asymmetry in the distribution of genes in bacterial genomes?</div><div class="source">Trends Microbiol</div> 
            ., 
            <div class="volume">10</div>
            , 
            <div class="fpage">393</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Is%20there%20a%20role%20for%20replication%20fork%20asymmetry%20in%20the%20distribution%20of%20genes%20in%20bacterial%20genomes%3F&amp;author=E.P.C.%20Rocha&amp;publication_year=2002&amp;journal=Trends%20Microbiol&amp;volume=10&amp;pages=393-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(02)02420-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(02)02420-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(02)02420-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12217498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dervyn</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Suski</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Daniel</div>
               <div class="given-names">R.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2001</div>
            , 
            <div class="article-title">Two essential DNA polymerases at the bacterial replication fork</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">294</div>
            , 
            <div class="fpage">1716</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20essential%20DNA%20polymerases%20at%20the%20bacterial%20replication%20fork&amp;author=E.%20Dervyn&amp;author=C.%20Suski&amp;author=R.%20Daniel&amp;publication_year=2001&amp;journal=Science&amp;volume=294&amp;pages=1716-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1066351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1066351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1066351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11721055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Boshoff</div>
               <div class="given-names">H.I.M.</div>
            </div>
             <div class="name">
              <div class="surname">Reed</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Barry</div>
               <div class="given-names">C.E.</div>
            </div>
             <div class="name">
              <div class="surname">Mizrahi</div>
               <div class="given-names">V.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">DnaE2 polymerase contributes to in vivo survival and the emergence of drug resistance in Mycobacterium tuberculosis</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">113</div>
            , 
            <div class="fpage">183</div>
            –
            <div class="lpage">93</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaE2%20polymerase%20contributes%20to%20in%20vivo%20survival%20and%20the%20emergence%20of%20drug%20resistance%20in%20Mycobacterium%20tuberculosis&amp;author=H.I.M.%20Boshoff&amp;author=M.B.%20Reed&amp;author=C.E.%20Barry&amp;author=V.%20Mizrahi&amp;publication_year=2003&amp;journal=Cell&amp;volume=113&amp;pages=183-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(03)00270-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(03)00270-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(03)00270-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12705867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw021-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw021-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Galhardo</div>
               <div class="given-names">R.S.</div>
            </div>
             <div class="name">
              <div class="surname">Rocha</div>
               <div class="given-names">R.P.</div>
            </div>
             <div class="name">
              <div class="surname">Marques</div>
               <div class="given-names">M.V</div>
            </div>
             <div class="name">
              <div class="surname">Menck</div>
               <div class="given-names">C.F.M.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">An SOS-regulated operon involved in damage-inducible mutagenesis in Caulobacter crescentus</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">33</div>
            , 
            <div class="fpage">2603</div>
            –
            <div class="lpage">14</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20SOS-regulated%20operon%20involved%20in%20damage-inducible%20mutagenesis%20in%20Caulobacter%20crescentus&amp;author=R.S.%20Galhardo&amp;author=R.P.%20Rocha&amp;author=M.V%20Marques&amp;author=C.F.M.%20Menck&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=2603-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki551" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15886391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a> ), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469968&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw021/2/dsw021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=tgfJfAWHm0Xk-Vd1HqeKXC1~WORgFU49JEKO-b0MMabZCs0xfCakIlCjwdGwtnpjxI8tiy77G69cDmzE6Bz3XdsGz5levNXcJ~u04vqCubVjgW-c5-q1ccn0OQq-oxl4GLoDrIiPD42WFUdMoC7~hLQ1nQsk0CFlbE7gKPDnKWd3FvCTa6TvvoCa6F1yOMug30QZPW05FxfbExGLYIHMcHHxPLxQmZCL0jyrKQlvi8YeoEcS86gYusZLIgJJ27AJlMIvO92U4QJVrVWjfwH4FYl6IWbkAsV-7a-xht7vb1KXOGV~m0-d~8Uhmn57bRi4ckfv6Ou9wKCxAlFvWuBqyg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,930</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469968"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f353%2f2469968">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F353%2F2469968" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469968'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="novel skew analysis reveals substitution asymmetries linked to genetic code GC-biases and PolIII a-subunit isoforms" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f353%2f2469968">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

