

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/3/259/475336","@type":"ScholarlyArticle","name":"Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in  Streptomyces ,\r\nRevealed the Extent and Complexity of the AdpA Regulatory Network","datePublished":"2012-03-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2012-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss010","keywords":["ChIP-seq","regulatory cascade","<italic>Streptomyces</italic>","transcription factor"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/3/259/475336","author":[{"name":"Higo, Akiyoshi","affiliation":"Department of Biotechnology ,  Graduate School of Agriculture and Life Sciences, The University of Tokyo ,  Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Hara, Hirofumi","affiliation":"Department of Biotechnology ,  Graduate School of Agriculture and Life Sciences, The University of Tokyo ,  Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Horinouchi, Sueharu","affiliation":"Department of Biotechnology ,  Graduate School of Agriculture and Life Sciences, The University of Tokyo ,  Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"},{"name":"Ohnishi, Yasuo","affiliation":"Department of Biotechnology ,  Graduate School of Agriculture and Life Sciences, The University of Tokyo ,  Bunkyo-ku, Tokyo 113-8657 ,  Japan","@type":"Person"}],"description":"Abstract.  AdpA is a global transcriptional activator triggering morphological differentiation and secondary metabolism in Streptomyces griseus. AdpA influences","pageStart":"259","pageEnd":"274","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in  Streptomyces ,\r\nRevealed the Extent and Complexity of the AdpA Regulatory Network","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in  Streptomyces ,
Revealed the Extent and Complexity of the AdpA Regulatory Network" />
    <meta property="og:description" content="Abstract.  AdpA is a global transcriptional activator triggering morphological differentiation and secondary metabolism in Streptomyces griseus. AdpA influences" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/3/259/475336" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-03-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Higo, Akiyoshi" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Hara, Hirofumi" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Horinouchi, Sueharu" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Ohnishi, Yasuo" /><meta name="citation_author_institution" content="Department of Biotechnology, Graduate School of Agriculture and Life Sciences, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan" /><meta name="citation_title" content="Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in Streptomyces,
Revealed the Extent and Complexity of the AdpA Regulatory Network" /><meta name="citation_firstpage" content="259" /><meta name="citation_lastpage" content="274" /><meta name="citation_doi" content="10.1093/dnares/dss010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2012/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The A-factor regulatory cascade leading to streptomycin biosynthesis in Streptomyces griseus: identification of a target gene of the A-factor receptor; Mol. Microbiol.;  citation_year=1999;  citation_volume=34;  citation_pages=102-11; " /><meta name="citation_reference" content="citation_title=AdpA, a central transcriptional regulator in the A-factor regulatory cascade that leads to morphological development and secondary metabolism in Streptomyces griseus; Biosci. Biotechnol. Biochem.;  citation_year=2005;  citation_volume=69;  citation_pages=431-9; " /><meta name="citation_reference" content="citation_title=Genome sequence of the streptomycin-producing microorganism Streptomyces griseus IFO 13350; J. Bacteriol.;  citation_year=2008;  citation_volume=190;  citation_pages=4050-60; " /><meta name="citation_reference" content="citation_title=Control of the Streptomyces subtilisin inhibitor gene by AdpA in the A-factor regulatory cascade in Streptomyces griseus; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=6207-16; " /><meta name="citation_reference" content="citation_title=DNA microarray analysis of global gene regulation by A-factor in Streptomyces griseus; Microbiology;  citation_year=2009;  citation_volume=155;  citation_pages=2197-210; " /><meta name="citation_reference" content="citation_title=Dynamic changes in the extracellular proteome caused by absence of a pleiotropic regulator AdpA in Streptomyces griseus; Mol. Microbiol.;  citation_year=2009;  citation_volume=73;  citation_pages=898-912; " /><meta name="citation_reference" content="citation_title=DNA-binding specificity of AdpA, a transcriptional activator in the A-factor regulatory cascade in Streptomyces griseus; Mol. Microbiol.;  citation_year=2004;  citation_volume=53;  citation_pages=555-72; " /><meta name="citation_reference" content="citation_title=Transcriptional regulation by cAMP and its receptor protein; Annu. Rev. Biochem.;  citation_year=1993;  citation_volume=62;  citation_pages=749-95; " /><meta name="citation_reference" content="citation_title=Mutagenesis and inducible responses to deoxyribonucleic acid damage in Escherichia coli; Microbiol. Rev.;  citation_year=1984;  citation_volume=48;  citation_pages=60-93; " /><meta name="citation_reference" content="citation_title=The Spo0A regulon of Bacillus subtilis; Mol. Microbiol.;  citation_year=2003;  citation_volume=50;  citation_pages=1683-701; " /><meta name="citation_reference" content="citation_title=Distribution of stable DnaA-binding sites on the Bacillus subtilis genome detected using a modified ChIP-chip method; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=155-68; " /><meta name="citation_reference" content="citation_title=ChIP-seq: advantages and challenges of a maturing technology; Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=669-80; " /><meta name="citation_reference" content="citation_title=Strict regulation of morphological differentiation and secondary metabolism by a positive feedback loop between two global regulators AdpA and BldA in Streptomyces griseus; Mol. Microbiol.;  citation_year=2011;  citation_volume=81;  citation_pages=1607-22; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=An integrated software system for analyzing ChIP-chip and ChIP-seq data; Nat. Biotechnol.;  citation_year=2008;  citation_volume=26;  citation_pages=1293-300; " /><meta name="citation_reference" content="citation_title=Autorepression of AdpA of the AraC/XylS family, a key transcriptional activator in the A-factor regulatory cascade in Streptomyces griseus; J. Mol. Biol.;  citation_year=2005;  citation_volume=350;  citation_pages=12-26; " /><meta name="citation_reference" content="citation_title=Methods and statistics for combining motif match scores; J. Comput. Biol.;  citation_year=1998;  citation_volume=5;  citation_pages=211-21; " /><meta name="citation_reference" content="citation_title=Control of aerial mycelium formation by the BldK oligopeptide ABC transporter in Streptomyces griseus; FEMS Microbiol. Lett.;  citation_year=2011;  citation_volume=315;  citation_pages=54-62; " /><meta name="citation_reference" content="citation_title=Genes essential for morphological development and antibiotic production in Streptomyces coelicolor are targets of BldD during vegetative growth; Mol. Microbiol.;  citation_year=2010;  citation_volume=78;  citation_pages=361-79; " /><meta name="citation_reference" content="citation_title=The Escherichia coli RutR transcription factor binds at targets within genes as well as intergenic regions; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=3950-5; " /><meta name="citation_reference" content="citation_title=Genome-wide binding profiles of the Bacillus subtilis transition state regulator AbrB and its homolog Abh reveals their interactive role in transcriptional regulation; Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=414-28; " /><meta name="citation_reference" content="citation_title=Studies of the distribution of Escherichia coli cAMP-receptor protein and RNA polymerase along the E. coli chromosome; Proc. Natl. Acad. Sci. USA;  citation_year=2005;  citation_volume=102;  citation_pages=17693-8; " /><meta name="citation_reference" content="citation_title=Genes directly controlled by CtrA, a master regulator of the Caulobacter cell cycle; , Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=4632-7; " /><meta name="citation_reference" content="citation_title=Genomic analysis of LexA binding reveals the permissive nature of the Escherichia coli genome and identifies unconventional target sites; Genes Dev.;  citation_year=2005;  citation_volume=19;  citation_pages=2619-30; " /><meta name="citation_reference" content="citation_title=Transcription factor distribution in Escherichia coli: studies with FNR protein; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=269-78; " /><meta name="citation_reference" content="citation_title=The complex extracellular biology of Streptomyces; FEMS Microbiol. Rev.;  citation_year=2010;  citation_volume=34;  citation_pages=171-98; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/3/259/1284930/dss010.pdf" /><meta name="description" content="Abstract.  AdpA is a global transcriptional activator triggering morphological differentiation and secondary metabolism in Streptomyces griseus. AdpA influences" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/3/259/475336" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/3/259/475336" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/3/259/475336");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss010';
    </script>
<title>
	Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in Streptomyces,
Revealed the Extent and Complexity of the AdpA Regulatory Network | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./475336" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdVDPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaWdvLCBBa2l5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgQnVua3lvLWt1LCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXJhLCBIaXJvZnVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJlIGFuZCBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgQnVua3lvLWt1LCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb3Jpbm91Y2hpLCBTdWVoYXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBCdW5reW8ta3UsIFRva3lvIDExMy04NjU3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obmlzaGksIFlhc3VvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUgYW5kIExpZmUgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBCdW5reW8ta3UsIFRva3lvIDExMy04NjU3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lLXdpZGUgRGlzdHJpYnV0aW9uIG9mIEFkcEEsIGEgR2xvYmFsIFJlZ3VsYXRvciBmb3IgU2Vjb25kYXJ5IE1ldGFib2xpc20gYW5kIE1vcnBob2xvZ2ljYWwgRGlmZmVyZW50aWF0aW9uIGluIFN0cmVwdG9teWNlcywKUmV2ZWFsZWQgdGhlIEV4dGVudCBhbmQgQ29tcGxleGl0eSBvZiB0aGUgQWRwQSBSZWd1bGF0b3J5IE5ldHdvcmsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI3NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEEtZmFjdG9yIHJlZ3VsYXRvcnkgY2FzY2FkZSBsZWFkaW5nIHRvIHN0cmVwdG9teWNpbiBiaW9zeW50aGVzaXMgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM6IGlkZW50aWZpY2F0aW9uIG9mIGEgdGFyZ2V0IGdlbmUgb2YgdGhlIEEtZmFjdG9yIHJlY2VwdG9yOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xMDItMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkcEEsIGEgY2VudHJhbCB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yIGluIHRoZSBBLWZhY3RvciByZWd1bGF0b3J5IGNhc2NhZGUgdGhhdCBsZWFkcyB0byBtb3JwaG9sb2dpY2FsIGRldmVsb3BtZW50IGFuZCBzZWNvbmRhcnkgbWV0YWJvbGlzbSBpbiBTdHJlcHRvbXljZXMgZ3Jpc2V1czsgQmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTQzMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIHN0cmVwdG9teWNpbi1wcm9kdWNpbmcgbWljcm9vcmdhbmlzbSBTdHJlcHRvbXljZXMgZ3Jpc2V1cyBJRk8gMTMzNTA7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTkwOyAgY2l0YXRpb25fcGFnZXM9NDA1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbCBvZiB0aGUgU3RyZXB0b215Y2VzIHN1YnRpbGlzaW4gaW5oaWJpdG9yIGdlbmUgYnkgQWRwQSBpbiB0aGUgQS1mYWN0b3IgcmVndWxhdG9yeSBjYXNjYWRlIGluIFN0cmVwdG9teWNlcyBncmlzZXVzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTYyMDctMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtaWNyb2FycmF5IGFuYWx5c2lzIG9mIGdsb2JhbCBnZW5lIHJlZ3VsYXRpb24gYnkgQS1mYWN0b3IgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz0yMTk3LTIxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBjaGFuZ2VzIGluIHRoZSBleHRyYWNlbGx1bGFyIHByb3Rlb21lIGNhdXNlZCBieSBhYnNlbmNlIG9mIGEgcGxlaW90cm9waWMgcmVndWxhdG9yIEFkcEEgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03MzsgIGNpdGF0aW9uX3BhZ2VzPTg5OC05MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQS1iaW5kaW5nIHNwZWNpZmljaXR5IG9mIEFkcEEsIGEgdHJhbnNjcmlwdGlvbmFsIGFjdGl2YXRvciBpbiB0aGUgQS1mYWN0b3IgcmVndWxhdG9yeSBjYXNjYWRlIGluIFN0cmVwdG9teWNlcyBncmlzZXVzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz01NTUtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIGJ5IGNBTVAgYW5kIGl0cyByZWNlcHRvciBwcm90ZWluOyBBbm51LiBSZXYuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NzQ5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhZ2VuZXNpcyBhbmQgaW5kdWNpYmxlIHJlc3BvbnNlcyB0byBkZW94eXJpYm9udWNsZWljIGFjaWQgZGFtYWdlIGluIEVzY2hlcmljaGlhIGNvbGk7IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTYwLTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU3BvMEEgcmVndWxvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpczsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MTY4My03MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3RyaWJ1dGlvbiBvZiBzdGFibGUgRG5hQS1iaW5kaW5nIHNpdGVzIG9uIHRoZSBCYWNpbGx1cyBzdWJ0aWxpcyBnZW5vbWUgZGV0ZWN0ZWQgdXNpbmcgYSBtb2RpZmllZCBDaElQLWNoaXAgbWV0aG9kOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE1NS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hJUC1zZXE6IGFkdmFudGFnZXMgYW5kIGNoYWxsZW5nZXMgb2YgYSBtYXR1cmluZyB0ZWNobm9sb2d5OyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NjY5LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJpY3QgcmVndWxhdGlvbiBvZiBtb3JwaG9sb2dpY2FsIGRpZmZlcmVudGlhdGlvbiBhbmQgc2Vjb25kYXJ5IG1ldGFib2xpc20gYnkgYSBwb3NpdGl2ZSBmZWVkYmFjayBsb29wIGJldHdlZW4gdHdvIGdsb2JhbCByZWd1bGF0b3JzIEFkcEEgYW5kIEJsZEEgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTE2MDctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIHNvZnR3YXJlIHN5c3RlbSBmb3IgYW5hbHl6aW5nIENoSVAtY2hpcCBhbmQgQ2hJUC1zZXEgZGF0YTsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEyOTMtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvcmVwcmVzc2lvbiBvZiBBZHBBIG9mIHRoZSBBcmFDL1h5bFMgZmFtaWx5LCBhIGtleSB0cmFuc2NyaXB0aW9uYWwgYWN0aXZhdG9yIGluIHRoZSBBLWZhY3RvciByZWd1bGF0b3J5IGNhc2NhZGUgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzUwOyAgY2l0YXRpb25fcGFnZXM9MTItMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGhvZHMgYW5kIHN0YXRpc3RpY3MgZm9yIGNvbWJpbmluZyBtb3RpZiBtYXRjaCBzY29yZXM7IEouIENvbXB1dC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbCBvZiBhZXJpYWwgbXljZWxpdW0gZm9ybWF0aW9uIGJ5IHRoZSBCbGRLIG9saWdvcGVwdGlkZSBBQkMgdHJhbnNwb3J0ZXIgaW4gU3RyZXB0b215Y2VzIGdyaXNldXM7IEZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTU7ICBjaXRhdGlvbl9wYWdlcz01NC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgZXNzZW50aWFsIGZvciBtb3JwaG9sb2dpY2FsIGRldmVsb3BtZW50IGFuZCBhbnRpYmlvdGljIHByb2R1Y3Rpb24gaW4gU3RyZXB0b215Y2VzIGNvZWxpY29sb3IgYXJlIHRhcmdldHMgb2YgQmxkRCBkdXJpbmcgdmVnZXRhdGl2ZSBncm93dGg7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTM2MS03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVzY2hlcmljaGlhIGNvbGkgUnV0UiB0cmFuc2NyaXB0aW9uIGZhY3RvciBiaW5kcyBhdCB0YXJnZXRzIHdpdGhpbiBnZW5lcyBhcyB3ZWxsIGFzIGludGVyZ2VuaWMgcmVnaW9uczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9Mzk1MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBiaW5kaW5nIHByb2ZpbGVzIG9mIHRoZSBCYWNpbGx1cyBzdWJ0aWxpcyB0cmFuc2l0aW9uIHN0YXRlIHJlZ3VsYXRvciBBYnJCIGFuZCBpdHMgaG9tb2xvZyBBYmggcmV2ZWFscyB0aGVpciBpbnRlcmFjdGl2ZSByb2xlIGluIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz00MTQtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb2YgdGhlIGRpc3RyaWJ1dGlvbiBvZiBFc2NoZXJpY2hpYSBjb2xpIGNBTVAtcmVjZXB0b3IgcHJvdGVpbiBhbmQgUk5BIHBvbHltZXJhc2UgYWxvbmcgdGhlIEUuIGNvbGkgY2hyb21vc29tZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9MTc2OTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgZGlyZWN0bHkgY29udHJvbGxlZCBieSBDdHJBLCBhIG1hc3RlciByZWd1bGF0b3Igb2YgdGhlIENhdWxvYmFjdGVyIGNlbGwgY3ljbGU7ICwgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTQ2MzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBvZiBMZXhBIGJpbmRpbmcgcmV2ZWFscyB0aGUgcGVybWlzc2l2ZSBuYXR1cmUgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgZ2Vub21lIGFuZCBpZGVudGlmaWVzIHVuY29udmVudGlvbmFsIHRhcmdldCBzaXRlczsgR2VuZXMgRGV2LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTI2MTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24gZmFjdG9yIGRpc3RyaWJ1dGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOiBzdHVkaWVzIHdpdGggRk5SIHByb3RlaW47IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTI2OS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXggZXh0cmFjZWxsdWxhciBiaW9sb2d5IG9mIFN0cmVwdG9teWNlczsgRkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0xNzEtOTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS8zLzI1OS8xMjg0OTMwL2RzczAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQWRwQSBpcyBhIGdsb2JhbCB0cmFuc2NyaXB0aW9uYWwgYWN0aXZhdG9yIHRyaWdnZXJpbmcgbW9ycGhvbG9naWNhbCBkaWZmZXJlbnRpYXRpb24gYW5kIHNlY29uZGFyeSBtZXRhYm9saXNtIGluIFN0cmVwdG9teWNlcyBncmlzZXVzLiBBZHBBIGluZmx1ZW5jZXMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ3NTMzNiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRHf4njMdsezIMQArx8ArH00rUVgA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f259%2f475336" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f259%2f475336" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18620" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/3">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7167199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167199">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167201">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167201">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167206">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167206">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167222">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167222">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167297">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167297">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167340">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167340">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167345">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167345">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167352">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167352">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7167358">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7167358">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/3/245/475122">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/3/275/475438">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in <em>Streptomyces</em>,
Revealed the Extent and Complexity of the AdpA Regulatory Network
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiyoshi Higo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Agriculture and Life Sciences, The University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiyoshi+Higo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Higo A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Higo A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akiyoshi Higo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hirofumi Hara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Agriculture and Life Sciences, The University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hirofumi+Hara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hara H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hara H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hirofumi Hara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sueharu Horinouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Agriculture and Life Sciences, The University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sueharu+Horinouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Horinouchi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Horinouchi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sueharu Horinouchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuo Ohnishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biotechnology</div>, <div class="insititution">Graduate School of Agriculture and Life Sciences, The University of Tokyo</div>, <div class="addr-line">Bunkyo-ku, Tokyo 113-8657</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-3-58415123. Fax. +81-3-58418021.
E-mail: <a href="mailto:ayasuo@mail.ecc.u-tokyo.ac.jp" target="_blank">ayasuo@mail.ecc.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuo+Ohnishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohnishi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohnishi Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuo Ohnishi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Department of Biomedical Engineering, Okayama University of Science, Okayama 700-0005, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Deceased on 12 July 2009.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 259–274, <a href='https://doi.org/10.1093/dnares/dss010'>https://doi.org/10.1093/dnares/dss010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 March 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 January 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 February 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="475336" href="/dnaresearch/article-pdf/19/3/259/1284930/dss010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Akiyoshi Higo, Hirofumi Hara, Sueharu Horinouchi, Yasuo Ohnishi;  Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in <em>Streptomyces</em>,
Revealed the Extent and Complexity of the AdpA Regulatory Network, <em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 259–274, <a href='https://doi.org/10.1093/dnares/dss010'>https://doi.org/10.1093/dnares/dss010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/475336?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/475336?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20Distribution%20of%20AdpA%2C%20a%20Global%20Regulator%20for%20Secondary%20Metabolism%20and%20Morphological%20Differentiation%20in%20%26lt%3Bem%26gt%3BStreptomyces%26lt%3B%2Fem%26gt%3B%2C%0D%0ARevealed%20the%20Extent%20and%20Complexity%20of%20the%20AdpA%20Regulatory%20Network&amp;publicationDate=2012-03-26&amp;volumeNum=19&amp;issueNum=3&amp;author=Higo%2C%20Akiyoshi%3B%20Hara%2C%20Hirofumi&amp;startPage=259&amp;endPage=274&amp;contentId=10.1093%2Fdnares%2Fdss010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f259%2f475336" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7167199 id="7167199" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>AdpA is a global transcriptional activator triggering morphological differentiation and secondary metabolism in <em>Streptomyces griseus</em>. AdpA influences the expression of &gt;1000 genes; however, the overall picture of the AdpA regulon remains obscure. Here, we took snapshots of the distribution of AdpA across the chromosome in living <em>S. griseus</em> cells using chromatin immunoprecipitation/chromatin affinity precipitation-seq analysis. In both liquid and solid cultures, AdpA bound to &gt;1200 similar sites, which were located on not only in putative regulatory regions (65%), but also in regions (35%) that appeared not to affect transcription. Transcriptome analysis indicated that ∼40% of the AdpA-binding sites in putative regulatory regions were involved in gene regulation. AdpA was indicated to act as a transcriptional repressor as well as an activator. Expression profiles of AdpA-target genes were very different between liquid and solid cultures, despite their similar AdpA-binding profiles. We concluded that AdpA directly controls &gt;500 genes in cooperation with other regulatory proteins. A comprehensive competitive gel mobility shift assay of AdpA with 304 selected AdpA-binding sites revealed several unique characteristics of the DNA-binding property of AdpA. This study provides the first experimental insight into the extent of the AdpA regulon, indicating that many genes are under the direct control of AdpA.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="ChIP-seq">ChIP-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;regulatory cascade&quot;">regulatory cascade</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Streptomyces"><em>Streptomyces</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor&quot;">transcription factor</a></div>                            <h2 scrollto-destination=7167201 id="7167201" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The Gram-positive, soil-inhabiting, filamentous bacterial genus <em>Streptomyces</em> is characterized by its ability to produce a wide variety of secondary metabolites. Another characteristic feature of the genus is its complex multicellular development. Spores germinate to form a branched, multinucleoid substrate mycelium, which then produces an aerial mycelium. After septa have been formed at regular intervals along the aerial hyphae, long chains of uninucleoid spores are formed. Generally, secondary metabolites begin to be produced after the onset of morphological differentiation. Many genes required for these two events are coordinately regulated. However, the regulatory networks of such genes are largely unknown.</p><p>In the streptomycin producer <em>Streptomyces griseus</em>, a microbial hormone, A-factor (2-isocapryloyl-3-<em>R</em>-hydroxymethyl-γ-butyrolactone), triggers both secondary metabolism and morphological differentiation. We have long studied the A-factor regulatory cascade. A-factor switches on the transcription of <em>adpA</em> by binding to ArpA, the A-factor receptor protein that binds to the promoter of <em>adpA</em>, and dissociating DNA-bound ArpA from the DNA.<sup><a reveal-id="DSS010C1" data-open="DSS010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> AdpA, an AraC/XylS family transcriptional regulator, activates a number of genes required for morphological differentiation and secondary metabolite formation.<sup><a reveal-id="DSS010C2" data-open="DSS010C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Thus, an <em>adpA-</em>deleted (Δ<em>adpA</em>) mutant cannot exhibit morphological development or production of many (but not all) secondary metabolites. Our transcriptome analysis showed that 639 and 373 genes were transcriptionally downregulated and upregulated, respectively, in the Δ<em>adpA</em> mutant, when compared with the wild-type strain.<sup><a reveal-id="DSS010C3" data-open="DSS010C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>Up until 2006, we had demonstrated that AdpA directly activates 14 transcripts (17 genes), which include several genes encoding key regulators of secondary metabolism and morphological differentiation, extracellular proteases and an extracellular protease inhibitor protein, and proteins of unknown function.<sup><a reveal-id="DSS010C2" data-open="DSS010C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS010C4" data-open="DSS010C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The 14 AdpA-target promoters are termed ‘category-I’ promoters hereafter. Twenty AdpA-binding sites were identified in the upstream (or just downstream, in the case of <em>adsA</em>) region of the promoters; these are termed ‘category-I’ AdpA-binding sites hereafter. Recent DNA microarray analysis, in which the effect of exogenously supplemented A-factor on the transcriptome in an A-factor-deficient mutant was determined, and comprehensive electrophoretic mobility shift assays (EMSAs) suggested 34 candidates for novel AdpA-target promoters.<sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Furthermore, our proteome analysis of extracellular proteins resulted in the identification of 11 novel candidates for AdpA-target promoters, 4 of which were included in the 34 candidates.<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup> These results indicate that AdpA directly activates a considerable number of genes. We have already determined a consensus AdpA-binding sequence, 5′-TGGCSNGWWY-3′ (S: G or C, W: A or T, Y: C or T, and N: A, C, G, or T),<sup><a reveal-id="DSS010C7" data-open="DSS010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> which is not strict compared with the consensus binding sequences of well-known bacterial global transcriptional regulators, such as cAMP-receptor protein (CRP),<sup><a reveal-id="DSS010C8" data-open="DSS010C8" class="link link-ref link-reveal xref-bibr">8</a></sup> LexA,<sup><a reveal-id="DSS010C9" data-open="DSS010C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and Spo0A.<sup><a reveal-id="DSS010C10" data-open="DSS010C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Accordingly, a bioinformatics search predicted more than 40 000 AdpA-binding sites in the genome of <em>S. griseus</em>. It is rather unlikely that AdpA binds to all the predicted sites; thus, actual AdpA-binding sites cannot be found by <em>in silico</em> analysis alone.</p><p>In the present study, we performed chromatin affinity precipitation (ChAP),<sup><a reveal-id="DSS010C11" data-open="DSS010C11" class="link link-ref link-reveal xref-bibr">11</a></sup> as well as chromatin immunoprecipitation (ChIP), in combination with high-throughput sequencing technology, to determine the distribution of AdpA across the entire <em>S. griseus</em> chromosome. DNA fragments obtained by ChIP or ChAP were identified with a high-throughput sequencer, which promised higher spatial resolution and sensitivity than microarray-based methods (e.g. ChIP-chip analysis).<sup><a reveal-id="DSS010C12" data-open="DSS010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> By the ChIP/ChAP-seq analysis, we were able to take snapshots of the distribution of AdpA across the chromosome in living <em>S. griseus</em> cells, not only in liquid culture, but also in solid culture. This information, in combination with transcriptional profiling using DNA microarray, comprehensive EMSA, and <em>in silico</em> prediction of AdpA-binding sites, revealed the extent and complexity of the AdpA regulatory network.</p>                            <h2 scrollto-destination=7167206 id="7167206" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7167207 id="7167207" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains and growth conditions</h3>
<p>The wild-type strain <em>S. griseus</em> IFO13350 and the <em>adpA</em> mutant were described previously.<sup><a reveal-id="DSS010C1" data-open="DSS010C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Strains were cultured as described previously.<sup><a reveal-id="DSS010C13" data-open="DSS010C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p>                            <h3 scrollto-destination=7167209 id="7167209" class="section-title" data-legacy-id=s2b>2.2. <em>In vivo</em> cross-linking of AdpA and DNA</h3>
<p>For <em>in vivo</em> cross-linking of AdpA and DNA in cells of liquid culture, the Δ<em>adpA</em> strain expressing <em>Nhis-adpA</em> was grown in 100 ml YMPD for 18 h at 30°C and treated with formaldehyde (1% final concentration) for 15 min at room temperature. For <em>in vivo</em> cross-linking of AdpA and DNA in cells of solid culture, the same strain was grown on an YMPD-cellophane plate for 1, 2, and 3 days at 28°C. The mycelia were collected with a spatula, resuspended with phosphate-buffered saline (PBS), and treated with formaldehyde (1% final concentration) for 20 min at room temperature. In both cases, formaldehyde was quenched by adding glycine to a final concentration of 250 mM. Finally, the cells were washed with tris-buffered saline buffer (pH 7.5) more than twice and stored at −80°C until use.</p>                            <h3 scrollto-destination=7167211 id="7167211" class="section-title" data-legacy-id=s2c>2.3. Chromatin immunoprecipitation</h3>
<p>Cross-linked cells in 1-ml ChIP buffer (50 mM HEPES–KOH, 140 mM NaCl, 1 mM EDTA, 1% Triton X-100, and 0.1% sodium deoxycolate, pH 7.5) were sonicated on ice to give an average DNA fragment size of 300–600 bp. After centrifugation at 12 000<em>g</em> for 10 min, the supernatant was collected and diluted with ChIP buffer (a final protein concentration of 3–5 mg/ml). At the same time, 50 μl of anti-mouse IgG dynabeads (Invitrogen), which had been washed twice with PBS containing 5 mg/ml bovine serum albumin (BSA), was mixed with anti-poly-histidine antibody (Qiagen) at 4°C for 4 h, washed twice with 1 ml of PBS–BSA, and resuspended with PBS–BSA. The anti-poly-histidine antibody-conjugated beads were added to the 1 ml of the diluted supernatant, followed by incubation for 6 h at 4°C with gentle rotation. The beads were then washed once with ChIP buffer, once with ChIP high-salt buffer (ChIP buffer containing 300 mM NaCl), twice with ChIP wash buffer (100 mM Tris–HCl, 250 mM LiCl, 1 mM EDTA, 0.5% Nonidet P-40, and 0.5% sodium deoxycolate, pH 8.0), and twice with TE (10 mM Tris and 1 mM EDTA, pH 8.0). Proteins bound to the beads were eluted with 100 μl of elution buffer [50 mM Tris–HCl, 10 mM EDTA, and 1% sodium dodecyl sulphate (SDS), pH 8.0]. After the cross-linking between AdpA and the DNA was destroyed by heating at 65°C overnight, 300 μl of TE was added to the reverse-cross-linked sample. The sample was then treated with RNase A (a final concentration of 0.4 mg/ml) for 2 h at 37°C and with proteinase K (a final concentration of 0.2 mg/ml) for 2 h at 55°C. Finally, DNA fragments were purified by Qiaquick PCR purification kit (Qiagen).</p>                            <h3 scrollto-destination=7167213 id="7167213" class="section-title" data-legacy-id=s2d>2.4. Chromatin affinity precipitation</h3>
<p>Cross-linked cells in 1-ml ChAP buffer [25 mM HEPES–KOH, 10 mM imidazole, 6 M guanidine-HCl, 0.5 M NaCl, 0.5% Triton X-100, and 10% (w/v) glycerol, pH 7.5] were sonicated on ice to give an average DNA fragment size of 300–600 bp. After centrifugation at 12 000<em>g</em> for 10 min, the supernatant was collected and diluted with ChAP buffer (a final protein concentration of 3–5 mg/ml). Fifty microlitres of dynabeads TALON (Invitrogen), which had been washed twice with ChAP buffer, were then added to the 1 ml of the diluted supernatant, followed by incubation for 2 h at room temperature with gentle rotation. The beads were washed four times with ChAP buffer and twice with TE. DNAs bound to the beads were eluted with 100 μl of elution buffer (50 mM Tris–HCl, 0.5 M imidazole, and 0.5% SDS, pH 7.5). After cross-linking between AdpA and DNA was destroyed by heating at 65°C overnight, reverse cross-linked DNA was purified by phenol–chloroform treatment and ethanol precipitation. The DNA pellet was resuspended in 100-μl TE containing 25 μg/ml RNase A, followed by incubation for 2 h at 37°C. DNA fragments were finally purified by Qiaquick PCR purification kit (Qiagen).</p>                            <h3 scrollto-destination=7167215 id="7167215" class="section-title" data-legacy-id=s2e>2.5. Sequencing of the DNA fragments enriched by ChIP or ChAP</h3>
<p>DNA samples were submitted to Takara Bio for sequencing using an Illumina Genome Analyzer II. Libraries were prepared according to the manufacture's instructions. Sequence reads were mapped to the genome of <em>S. griseus</em> using Bowtie.<sup><a reveal-id="DSS010C14" data-open="DSS010C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Parameters were set to allow mapping of reads to the genome with up to two mismatches. Reads mapped to multiple sites, except for those to terminal inverted repeats and rRNA gene clusters, were excluded from the analysis. Reads were clustered and analysed by CisGenome.<sup><a reveal-id="DSS010C15" data-open="DSS010C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p>                            <h3 scrollto-destination=7167217 id="7167217" class="section-title" data-legacy-id=s2f>2.6. Other materials and methods</h3>
<p>Other Materials and methods are described in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data</a>.</p>                            <h3 scrollto-destination=7167219 id="7167219" class="section-title" data-legacy-id=s2g>2.7. Accession number</h3>
<p>All microarray and ChIP-seq data have been deposited in the NCBI Gene Expression Omnibus (GEO; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) under accession number GSE33994.</p>                            <h2 scrollto-destination=7167222 id="7167222" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7167225 id="7167225" class="section-title" data-legacy-id=s3a>3.1. Expression of polyHis-tagged AdpA in <em>S. griseus</em></h3>
<p>For ChIP and ChAP analyses, Δ<em>adpA</em> strains producing AdpA with a polyHis-tag at either its N- or C-terminus (N-His-AdpA and AdpA-C-His, respectively) were generated. The defects in morphological differentiation (Fig. <a reveal-id="DSS010F1" data-open="DSS010F1" class="link link-reveal link-table xref-fig">1</a>A) and streptomycin production (Fig. <a reveal-id="DSS010F1" data-open="DSS010F1" class="link link-reveal link-table xref-fig">1</a>B) of the Δ<em>adpA</em> strain were completely complemented by the production of N-His-AdpA and AdpA-C-His proteins, as well as by the native AdpA protein, demonstrating that the attached polyHis-tag did not interfere the <em>in vivo</em> function of AdpA. Western blotting analysis with an anti-polyHis-tag antibody showed that the amount of AdpA increased gradually during the exponential growth phase and then decreased gradually during the stationary phase in the mycelium cultured in the nutrient-rich YMPD liquid medium (Fig. <a reveal-id="DSS010F1" data-open="DSS010F1" class="link link-reveal link-table xref-fig">1</a>C). Maximal accumulation of AdpA was observed at 18 h, when the growth became slower than that in the exponential growth phase (Fig. <a reveal-id="DSS010F1" data-open="DSS010F1" class="link link-reveal link-table xref-fig">1</a>D). Western blotting analysis with N-His-AdpA-expressing strain showed that the C-terminal portion of AdpA was partially degraded on solid culture (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). Degradation of the polyHis-containing C-terminal portion of AdpA-C-His should be also possible; therefore, we used the Δ<em>adpA</em> strain producing N-His-AdpA, not AdpA-C-His, for further analyses. </p>                        <a id="7167230" scrollto-destination="7167230"></a>
<div id="DSS010F1" data-id="DSS010F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01001.jpeg?Expires=2147483647&Signature=AuKzEecHjfp6qXr7HeKGXZVcuBtZZ0g9T1uponSfw05zZzwNqpEZXVCxADdkY0PMe4RreL38DH9Z84dAaCabilbvkXTV8X4jhUVYoo1~zZQgFVv4-YbmcxgwyEgM5LgHdHBrDxmUt453FKZhxnQIThkWWPvrO~ZnTC8rdSW~ys4Ok66QyNDAGjn8igIjUfoLfyn8jAOwfs5q~Iosn-xFLiSAR3C4vvDcPFh3uJEv4-uBj7YS5iYkIZvRMN4Uo0MqAEBcQt4zSarKAv13A7daTm~1I1OqjjE65oMLMKaEYoLSLuDVWIxoaXdZ2PN~qPDVwVpN2QQxvfuB1b29w9BNqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The polyHis-tag attached does not interfere with the in vivo function of AdpA. (A) Restoration of morphological differentiation in the ΔadpA mutant by the production of polyHis-tagged AdpA. Strains were grown on YMPD agar for 3 days at 28°C. (B) Restoration of streptomycin production in the ΔadpA mutant by the production of polyHis-tagged AdpA. After the indicated strains were grown on YMP (YMPD without glucose) agar for 4 days at 28°C, an indicator strain was overlaid and incubated overnight at 28°C. (C) Western blotting analysis of polyHis-tagged AdpA. Total protein was extracted at 6 (Lane 1), 12 (Lane 2), 18 (Lane 3), 24 (Lane 4), and 36 (Lane 5) h of liquid culture and 6 μg of each was subjected to electrophoresis. Arrow indicates polyHis-tagged AdpA. Note that western blotting analysis gave stronger signals to purified AdpA-C-His than to purified N-His-AdpA when equal amounts of them were subjected to electrophoresis (data not shown). Therefore, intracellular levels of N-His-AdpA and AdpA-C-His seem to be similar, although AdpA-C-His gave stronger signals than N-His-AdpA. (D) Growth curve in YMPD liquid culture, as determined by the amount of intracellular protein. Closed circles represent the ΔadpA strain producing N-His-AdpA and open circles represent the control ΔadpA strain." path-from-xml="dss01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167230/dss01001.jpeg" path-from-xml="dss01001.jpeg" target="_blank">View large</a><a section="7167230" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01001.jpeg?Expires=2147483647&Signature=Ivj8zd4CUxWsDh7z~5Jas1kmMx-FhW30ea-8AnTTHbZCc4BIpPtF8vfGutS0JvSvJK9H55Mibxqp0vkdQ9bcPwREwkgAzWALM8c61M8c-wf8i0RJNM3WEc7OPVV6Qkf8yXkHPATf1ctJvfj56QwjPqyh-taHcBAUd5zkI~EkfTi8vDnZ2oS-aeIUQN-wzKB2UvHsKIrKow1S7ZWF54G3sSJYFIFheY3iI~Sf2yaphV5GFbJl4AEzjlev0Qh4LI20Mh2PGaF~2qot107HLLCZMITvqKIihTKJsKcInGLDMTzS-4WLuXSIETfUadFcJlebBuKHGsGmgjK8UOUfQXfr2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167230&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The polyHis-tag attached does not interfere with the <em>in vivo</em> function of AdpA. (A) Restoration of morphological differentiation in the Δ<em>adpA</em> mutant by the production of polyHis-tagged AdpA. Strains were grown on YMPD agar for 3 days at 28°C. (B) Restoration of streptomycin production in the Δ<em>adpA</em> mutant by the production of polyHis-tagged AdpA. After the indicated strains were grown on YMP (YMPD without glucose) agar for 4 days at 28°C, an indicator strain was overlaid and incubated overnight at 28°C. (C) Western blotting analysis of polyHis-tagged AdpA. Total protein was extracted at 6 (Lane 1), 12 (Lane 2), 18 (Lane 3), 24 (Lane 4), and 36 (Lane 5) h of liquid culture and 6 μg of each was subjected to electrophoresis. Arrow indicates polyHis-tagged AdpA. Note that western blotting analysis gave stronger signals to purified AdpA-C-His than to purified N-His-AdpA when equal amounts of them were subjected to electrophoresis (data not shown). Therefore, intracellular levels of N-His-AdpA and AdpA-C-His seem to be similar, although AdpA-C-His gave stronger signals than N-His-AdpA. (D) Growth curve in YMPD liquid culture, as determined by the amount of intracellular protein. Closed circles represent the Δ<em>adpA</em> strain producing N-His-AdpA and open circles represent the control Δ<em>adpA</em> strain.</p></div></div><div content-id="DSS010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01001.jpeg?Expires=2147483647&Signature=AuKzEecHjfp6qXr7HeKGXZVcuBtZZ0g9T1uponSfw05zZzwNqpEZXVCxADdkY0PMe4RreL38DH9Z84dAaCabilbvkXTV8X4jhUVYoo1~zZQgFVv4-YbmcxgwyEgM5LgHdHBrDxmUt453FKZhxnQIThkWWPvrO~ZnTC8rdSW~ys4Ok66QyNDAGjn8igIjUfoLfyn8jAOwfs5q~Iosn-xFLiSAR3C4vvDcPFh3uJEv4-uBj7YS5iYkIZvRMN4Uo0MqAEBcQt4zSarKAv13A7daTm~1I1OqjjE65oMLMKaEYoLSLuDVWIxoaXdZ2PN~qPDVwVpN2QQxvfuB1b29w9BNqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The polyHis-tag attached does not interfere with the in vivo function of AdpA. (A) Restoration of morphological differentiation in the ΔadpA mutant by the production of polyHis-tagged AdpA. Strains were grown on YMPD agar for 3 days at 28°C. (B) Restoration of streptomycin production in the ΔadpA mutant by the production of polyHis-tagged AdpA. After the indicated strains were grown on YMP (YMPD without glucose) agar for 4 days at 28°C, an indicator strain was overlaid and incubated overnight at 28°C. (C) Western blotting analysis of polyHis-tagged AdpA. Total protein was extracted at 6 (Lane 1), 12 (Lane 2), 18 (Lane 3), 24 (Lane 4), and 36 (Lane 5) h of liquid culture and 6 μg of each was subjected to electrophoresis. Arrow indicates polyHis-tagged AdpA. Note that western blotting analysis gave stronger signals to purified AdpA-C-His than to purified N-His-AdpA when equal amounts of them were subjected to electrophoresis (data not shown). Therefore, intracellular levels of N-His-AdpA and AdpA-C-His seem to be similar, although AdpA-C-His gave stronger signals than N-His-AdpA. (D) Growth curve in YMPD liquid culture, as determined by the amount of intracellular protein. Closed circles represent the ΔadpA strain producing N-His-AdpA and open circles represent the control ΔadpA strain." path-from-xml="dss01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167230/dss01001.jpeg" path-from-xml="dss01001.jpeg" target="_blank">View large</a><a section="7167230" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01001.jpeg?Expires=2147483647&Signature=Ivj8zd4CUxWsDh7z~5Jas1kmMx-FhW30ea-8AnTTHbZCc4BIpPtF8vfGutS0JvSvJK9H55Mibxqp0vkdQ9bcPwREwkgAzWALM8c61M8c-wf8i0RJNM3WEc7OPVV6Qkf8yXkHPATf1ctJvfj56QwjPqyh-taHcBAUd5zkI~EkfTi8vDnZ2oS-aeIUQN-wzKB2UvHsKIrKow1S7ZWF54G3sSJYFIFheY3iI~Sf2yaphV5GFbJl4AEzjlev0Qh4LI20Mh2PGaF~2qot107HLLCZMITvqKIihTKJsKcInGLDMTzS-4WLuXSIETfUadFcJlebBuKHGsGmgjK8UOUfQXfr2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167230&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The polyHis-tag attached does not interfere with the <em>in vivo</em> function of AdpA. (A) Restoration of morphological differentiation in the Δ<em>adpA</em> mutant by the production of polyHis-tagged AdpA. Strains were grown on YMPD agar for 3 days at 28°C. (B) Restoration of streptomycin production in the Δ<em>adpA</em> mutant by the production of polyHis-tagged AdpA. After the indicated strains were grown on YMP (YMPD without glucose) agar for 4 days at 28°C, an indicator strain was overlaid and incubated overnight at 28°C. (C) Western blotting analysis of polyHis-tagged AdpA. Total protein was extracted at 6 (Lane 1), 12 (Lane 2), 18 (Lane 3), 24 (Lane 4), and 36 (Lane 5) h of liquid culture and 6 μg of each was subjected to electrophoresis. Arrow indicates polyHis-tagged AdpA. Note that western blotting analysis gave stronger signals to purified AdpA-C-His than to purified N-His-AdpA when equal amounts of them were subjected to electrophoresis (data not shown). Therefore, intracellular levels of N-His-AdpA and AdpA-C-His seem to be similar, although AdpA-C-His gave stronger signals than N-His-AdpA. (D) Growth curve in YMPD liquid culture, as determined by the amount of intracellular protein. Closed circles represent the Δ<em>adpA</em> strain producing N-His-AdpA and open circles represent the control Δ<em>adpA</em> strain.</p></div></div>                            <h3 scrollto-destination=7167233 id="7167233" class="section-title" data-legacy-id=s3b>3.2. Genome-wide distribution of AdpA across the chromosome of the mycelium in liquid culture</h3>
<p>We performed ChAP-seq and ChIP-seq analyses using the N-His-AdpA-producing Δ<em>adpA</em> strain cultured in the YMPD liquid medium for 18 h. Although these two analyses are based on different principles to enrich the AdpA-bound DNA fragments, very similar results were obtained; the correlation coefficient of signal intensity of peaks obtained from ChIP-seq and ChAP-seq was 0.84 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). As an example, Fig. <a reveal-id="DSS010F2" data-open="DSS010F2" class="link link-reveal link-table xref-fig">2</a>A shows a small fraction of the result, in which the binding of AdpA on a 50-kb region containing <em>sgmA</em> is depicted. Peaks obtained from both analyses were in good agreement with the known AdpA-binding sites (Sites A and B) for <em>sgmA</em>.<sup><a reveal-id="DSS010C7" data-open="DSS010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Regarding Site A, peaks from ChIP-seq and ChAP-seq analyses are shifted from the actual binding site by only 36 and 11 bp, respectively. On the other hand, a peak is located right on Site B (Fig. <a reveal-id="DSS010F2" data-open="DSS010F2" class="link link-reveal link-table xref-fig">2</a>B) in both analyses. Furthermore, we confirmed that a peak was invariably detected within 50 bp from all ‘category-I’ AdpA-binding sites, except the AdpA-binding site of <em>sgiA</em> (AdBS-<em>sgiA</em>), which was not detected (see below). Thus, we successfully obtained a high-resolution map of the <em>in vivo</em> AdpA-binding regions. Figure <a reveal-id="DSS010F3" data-open="DSS010F3" class="link link-reveal link-table xref-fig">3</a> shows the entire AdpA-binding chromosome map obtained by ChAP; genome-wide distribution of AdpA across the chromosome is obvious. </p>                        <a id="7167238" scrollto-destination="7167238"></a>
<div id="DSS010F2" data-id="DSS010F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01002.jpeg?Expires=2147483647&Signature=PjbN8EwxcdoCdfRFPLMyX1nxBJPTrB1QdhItr1u-nI3aQU9V75spUw592zv0fLe5IN059mKCpP1c3vm8fR3nMI~PQX3DSyTto8HTWPBge2L2MDn6tq8sOC6eY-m-cbaazFJ7uxF7MQHM-DXIoEjcUFwVcjGKj78EyuRd2t0IV0HGrVPsZ0GS6QC3nU9h0Hn0x90DdnaECR9va1pzX1lUF9Nd4GNY~L3avG3nDyTIFbPGZd5gAyeZAxo3CYjzqXTZHoizdIeBetX9F79S4McgIAeWtF3mHmnDS2txE6x2uuHLtJoeVp8zeFVbOryl-AygOfhvlOqzcWPcZKNUYc8scg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ChIP/ChAP-seq analysis of AdpA. The results of ChAP-seq (upper) and ChIP-seq (lower) are shown. (A) A 50-kb region containing sgmA. (B) A 2-kb region containing sgmA. Two AdpA-binding sites (Sites A and B) determined by in vitro experiments7 are indicated." path-from-xml="dss01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167238/dss01002.jpeg" path-from-xml="dss01002.jpeg" target="_blank">View large</a><a section="7167238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01002.jpeg?Expires=2147483647&Signature=NRoOVW6cg7UJMKhibXtJUHZwFcN~UyGuQgL1O9KJ6kRgqRnq64ng9f~jsUvFTF3qHVuLDXJtkiI2zDHUHhdzh69tbHihJbkgrPG3Qf~C1BoNXSvpe-eExFmJYNFalMgiWH34KKySmKVhaAtR-c8qv7TbnqZ4EoAVBojTO~u~TvezyCKf2mjZspsyTHXXNwlJ4-IcxppZHDkh97mz835grwjDUqmRT8KOcR7TxdICfj1pMvs~mT-FSspGnWbf9HxyqvmEZ-mshW~1j7~i~k8p~EmDaJMKM7v6g50f~0a0d2mh04aGQkv9zkJ4et6LGbVRl5CWRFg-8-~xhmH~tenqvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167238&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ChIP/ChAP-seq analysis of AdpA. The results of ChAP-seq (upper) and ChIP-seq (lower) are shown. (A) A 50-kb region containing <em>sgmA</em>. (B) A 2-kb region containing <em>sgmA</em>. Two AdpA-binding sites (Sites A and B) determined by <em>in vitro</em> experiments<sup><a reveal-id="DSS010C7" data-open="DSS010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> are indicated.</p></div></div><div content-id="DSS010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01002.jpeg?Expires=2147483647&Signature=PjbN8EwxcdoCdfRFPLMyX1nxBJPTrB1QdhItr1u-nI3aQU9V75spUw592zv0fLe5IN059mKCpP1c3vm8fR3nMI~PQX3DSyTto8HTWPBge2L2MDn6tq8sOC6eY-m-cbaazFJ7uxF7MQHM-DXIoEjcUFwVcjGKj78EyuRd2t0IV0HGrVPsZ0GS6QC3nU9h0Hn0x90DdnaECR9va1pzX1lUF9Nd4GNY~L3avG3nDyTIFbPGZd5gAyeZAxo3CYjzqXTZHoizdIeBetX9F79S4McgIAeWtF3mHmnDS2txE6x2uuHLtJoeVp8zeFVbOryl-AygOfhvlOqzcWPcZKNUYc8scg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="ChIP/ChAP-seq analysis of AdpA. The results of ChAP-seq (upper) and ChIP-seq (lower) are shown. (A) A 50-kb region containing sgmA. (B) A 2-kb region containing sgmA. Two AdpA-binding sites (Sites A and B) determined by in vitro experiments7 are indicated." path-from-xml="dss01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167238/dss01002.jpeg" path-from-xml="dss01002.jpeg" target="_blank">View large</a><a section="7167238" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01002.jpeg?Expires=2147483647&Signature=NRoOVW6cg7UJMKhibXtJUHZwFcN~UyGuQgL1O9KJ6kRgqRnq64ng9f~jsUvFTF3qHVuLDXJtkiI2zDHUHhdzh69tbHihJbkgrPG3Qf~C1BoNXSvpe-eExFmJYNFalMgiWH34KKySmKVhaAtR-c8qv7TbnqZ4EoAVBojTO~u~TvezyCKf2mjZspsyTHXXNwlJ4-IcxppZHDkh97mz835grwjDUqmRT8KOcR7TxdICfj1pMvs~mT-FSspGnWbf9HxyqvmEZ-mshW~1j7~i~k8p~EmDaJMKM7v6g50f~0a0d2mh04aGQkv9zkJ4et6LGbVRl5CWRFg-8-~xhmH~tenqvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167238&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>ChIP/ChAP-seq analysis of AdpA. The results of ChAP-seq (upper) and ChIP-seq (lower) are shown. (A) A 50-kb region containing <em>sgmA</em>. (B) A 2-kb region containing <em>sgmA</em>. Two AdpA-binding sites (Sites A and B) determined by <em>in vitro</em> experiments<sup><a reveal-id="DSS010C7" data-open="DSS010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> are indicated.</p></div></div>                        <a id="7167240" scrollto-destination="7167240"></a>
<div id="DSS010F3" data-id="DSS010F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01003.jpeg?Expires=2147483647&Signature=kd-VhmfCi9M0eSeG10jQMXYEPhwS5xXKPuXQYhJYd7WGw4JvtFUTnR5ntZC3mRMXNPeBwDS4I3Mspd1-iXyeZcy-6s5Le62na02dp~Uu7emAmmu7w80HBujuCmrAODtdKB0AK09Kh8AZw12rTXQqXH-NwuF~l2zNIZ6Bu9s7~9xlR1a97FRaYaIjXwlPGQZ00IuAGFdFzKijom3pOWHLe26LTh-nG~efKmcM-UnEUPpUPMdvAq-5~vykGBh82CbCtevJ2YnWkWK9y79jhQJ27FQO~roeOLccyLc7M8nPxaRyfINrHVRyGEnSxWgsy-3XgFg5Sm8Qv5b21GLTofE0Zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide distribution of AdpA. The result of ChAP-seq analysis is shown. Horizontal lines indicated by arrows represent the threshold for AdpA binding. The chromosome of S. griseus has terminal inverted repeats (TIR), which are identical 13920-bp sequences, at both ends. AdpA binding cannot be distinguished between the right and left TIR sequences in ChIP/ChAP-seq analysis. Therefore, the intensity of signals detected on TIR is equally divided between right and left TIR sequences." path-from-xml="dss01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167240/dss01003.jpeg" path-from-xml="dss01003.jpeg" target="_blank">View large</a><a section="7167240" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01003.jpeg?Expires=2147483647&Signature=QqK6uQcPOpkwXwET05yaq3LlMCNX8ZMsCQPbt7WumP8-2fZ6l8iX65u3x3Dd1g1Xhg~AE-z8KBjfVNwIeIxBnIc5VvTOgFfQYTNqLezCQ2TMDyAoKdux8i2WZ2Mxx~TKZPnQfRqWTiXiDl75bvBIxpjeAjVlsHBYIJ4O9zLSMJ73cEOhxUViuU76jNoQLAqli45BrXNwAXZvsnwEwLx2PKgdMuPZRTT0pfEgnWMdgyBepHwl~Z0CGa65QR2ht6hjqDuW~ieilTr4AIkg2b2NkOzZG3Fn23XBvpFZB3hyF-x08eG2PRO-Qy2f5QeZtrkiw2BZ4ue5zmMQe-3R6oS1sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167240&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distribution of AdpA. The result of ChAP-seq analysis is shown. Horizontal lines indicated by arrows represent the threshold for AdpA binding. The chromosome of <em>S. griseus</em> has terminal inverted repeats (TIR), which are identical 13920-bp sequences, at both ends. AdpA binding cannot be distinguished between the right and left TIR sequences in ChIP/ChAP-seq analysis. Therefore, the intensity of signals detected on TIR is equally divided between right and left TIR sequences.</p></div></div><div content-id="DSS010F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01003.jpeg?Expires=2147483647&Signature=kd-VhmfCi9M0eSeG10jQMXYEPhwS5xXKPuXQYhJYd7WGw4JvtFUTnR5ntZC3mRMXNPeBwDS4I3Mspd1-iXyeZcy-6s5Le62na02dp~Uu7emAmmu7w80HBujuCmrAODtdKB0AK09Kh8AZw12rTXQqXH-NwuF~l2zNIZ6Bu9s7~9xlR1a97FRaYaIjXwlPGQZ00IuAGFdFzKijom3pOWHLe26LTh-nG~efKmcM-UnEUPpUPMdvAq-5~vykGBh82CbCtevJ2YnWkWK9y79jhQJ27FQO~roeOLccyLc7M8nPxaRyfINrHVRyGEnSxWgsy-3XgFg5Sm8Qv5b21GLTofE0Zw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide distribution of AdpA. The result of ChAP-seq analysis is shown. Horizontal lines indicated by arrows represent the threshold for AdpA binding. The chromosome of S. griseus has terminal inverted repeats (TIR), which are identical 13920-bp sequences, at both ends. AdpA binding cannot be distinguished between the right and left TIR sequences in ChIP/ChAP-seq analysis. Therefore, the intensity of signals detected on TIR is equally divided between right and left TIR sequences." path-from-xml="dss01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167240/dss01003.jpeg" path-from-xml="dss01003.jpeg" target="_blank">View large</a><a section="7167240" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01003.jpeg?Expires=2147483647&Signature=QqK6uQcPOpkwXwET05yaq3LlMCNX8ZMsCQPbt7WumP8-2fZ6l8iX65u3x3Dd1g1Xhg~AE-z8KBjfVNwIeIxBnIc5VvTOgFfQYTNqLezCQ2TMDyAoKdux8i2WZ2Mxx~TKZPnQfRqWTiXiDl75bvBIxpjeAjVlsHBYIJ4O9zLSMJ73cEOhxUViuU76jNoQLAqli45BrXNwAXZvsnwEwLx2PKgdMuPZRTT0pfEgnWMdgyBepHwl~Z0CGa65QR2ht6hjqDuW~ieilTr4AIkg2b2NkOzZG3Fn23XBvpFZB3hyF-x08eG2PRO-Qy2f5QeZtrkiw2BZ4ue5zmMQe-3R6oS1sA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167240&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide distribution of AdpA. The result of ChAP-seq analysis is shown. Horizontal lines indicated by arrows represent the threshold for AdpA binding. The chromosome of <em>S. griseus</em> has terminal inverted repeats (TIR), which are identical 13920-bp sequences, at both ends. AdpA binding cannot be distinguished between the right and left TIR sequences in ChIP/ChAP-seq analysis. Therefore, the intensity of signals detected on TIR is equally divided between right and left TIR sequences.</p></div></div><p>In both analyses, the signal intensity of the AdpA-binding site for <em>adsA</em> (AdBS-<em>adsA</em>) was lowest among all of the ‘category-I’ AdpA-binding sites, except AdBS-<em>sgiA</em>. Therefore, we used this value as a threshold to exclude background noise in each analysis; peaks with signal intensities equal to or more than that of AdBS-<em>adsA</em> were considered as candidates for AdpA-binding sites (Fig. <a reveal-id="DSS010F3" data-open="DSS010F3" class="link link-reveal link-table xref-fig">3</a>). Finally, 1355 sites, which were detected in both analyses, were regarded as the <em>in vivo</em> AdpA-binding sites.</p><p>We classified these sites into three groups by their location in relation to the surrounding genes. First, ‘putative regulatory region’: from −600 to +100 of protein-coding genes (taking a putative translational start site as +1) and −300 to +50 for RNA genes (taking a transcriptional start point of RNA gene as +1). Secondly, ‘coding region’: from +101 to the 3′-terminus (excluding AdpA-binding sites that can be considered as ‘putative regulatory region’ for downstream genes). The last group contained the remainder, which was classified as ‘intergenic region’ (Table <a reveal-id="DSS010TB1" data-open="DSS010TB1" class="link link-reveal link-table xref-fig">1</a>). This classification suggested that 69% of the AdpA-binding sites were located in ‘putative regulatory regions’ and 31% of them were on regions that appeared not to affect transcription (i.e. ‘coding’ and ‘intergenic’ regions). </p>                        <a id="7167247" scrollto-destination="7167247"></a>
<div content-id="DSS010TB1" class="table-modal"><div class="table-wrap"><div id="DSS010TB1" class="table-wrap-title" data-id="DSS010TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The number of AdpA-binding sites in liquid and solid cultures</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th>Liquid </th><th colspan="3">Solid (days)<hr /> </th><th colspan="3">Overlapped<sup>d</sup> (days)<hr /> </th></tr><tr><th> </th><th>1 </th><th>2 </th><th>3 </th><th>1 </th><th>2 </th><th>3 </th></tr></thead><tbody><tr><td colspan="8">‘Regulatory region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>892 </td><td>772 </td><td>732 </td><td>768 </td><td>706 </td><td>662 </td><td>662 </td></tr><tr><td> tRNA<sup>b</sup> </td><td>23 </td><td>11 </td><td>5 </td><td>5 </td><td>11 </td><td>5 </td><td>5 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> sRNA </td><td>7 </td><td>4 </td><td>3 </td><td>2 </td><td>4 </td><td>3 </td><td>2 </td></tr><tr><td> Total </td><td>929 </td><td>794 </td><td>747 </td><td>782 </td><td>728 </td><td>677 </td><td>676 </td></tr><tr><td colspan="8">‘Coding region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>393 </td><td>415 </td><td>425 </td><td>479 </td><td>341 </td><td>334 </td><td>338 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> Total </td><td>400 </td><td>422 </td><td>432 </td><td>486 </td><td>348 </td><td>341 </td><td>345 </td></tr><tr><td colspan="8">‘Intergenic region’ </td></tr><tr><td> Total </td><td>26 </td><td>23 </td><td>23 </td><td>24 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>Total </td><td>1355 </td><td>1239 </td><td>1202 </td><td>1292 </td><td>1093 </td><td>1035 </td><td>1038 </td></tr></tbody></table></div><div content-id="DSS010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th>Liquid </th><th colspan="3">Solid (days)<hr /> </th><th colspan="3">Overlapped<sup>d</sup> (days)<hr /> </th></tr><tr><th> </th><th>1 </th><th>2 </th><th>3 </th><th>1 </th><th>2 </th><th>3 </th></tr></thead><tbody><tr><td colspan="8">‘Regulatory region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>892 </td><td>772 </td><td>732 </td><td>768 </td><td>706 </td><td>662 </td><td>662 </td></tr><tr><td> tRNA<sup>b</sup> </td><td>23 </td><td>11 </td><td>5 </td><td>5 </td><td>11 </td><td>5 </td><td>5 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> sRNA </td><td>7 </td><td>4 </td><td>3 </td><td>2 </td><td>4 </td><td>3 </td><td>2 </td></tr><tr><td> Total </td><td>929 </td><td>794 </td><td>747 </td><td>782 </td><td>728 </td><td>677 </td><td>676 </td></tr><tr><td colspan="8">‘Coding region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>393 </td><td>415 </td><td>425 </td><td>479 </td><td>341 </td><td>334 </td><td>338 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> Total </td><td>400 </td><td>422 </td><td>432 </td><td>486 </td><td>348 </td><td>341 </td><td>345 </td></tr><tr><td colspan="8">‘Intergenic region’ </td></tr><tr><td> Total </td><td>26 </td><td>23 </td><td>23 </td><td>24 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>Total </td><td>1355 </td><td>1239 </td><td>1202 </td><td>1292 </td><td>1093 </td><td>1035 </td><td>1038 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>AdpA-binding sites in the TIR region are counted only once.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>Streptomyces griseus</em> has 66 tRNA genes, including ‘duplicated’ gene sets. Duplicated genes cannot be distinguished from each other in ChIP/ChAP analysis; therefore, they were excluded from the analysis. Only 26 transcriptional units containing 27 tRNA genes can be discriminated from others; AdpA was shown to bind to the upstream regions of 23 transcriptional units containing 24 tRNA genes.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>AdpA-binding sites in the common region of six rRNA gene clusters are counted only once.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The number of AdpA-binding sites detected in both liquid and solid cultures.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167247" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS010TB1" class="table-wrap-title" data-id="DSS010TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The number of AdpA-binding sites in liquid and solid cultures</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th>Liquid </th><th colspan="3">Solid (days)<hr /> </th><th colspan="3">Overlapped<sup>d</sup> (days)<hr /> </th></tr><tr><th> </th><th>1 </th><th>2 </th><th>3 </th><th>1 </th><th>2 </th><th>3 </th></tr></thead><tbody><tr><td colspan="8">‘Regulatory region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>892 </td><td>772 </td><td>732 </td><td>768 </td><td>706 </td><td>662 </td><td>662 </td></tr><tr><td> tRNA<sup>b</sup> </td><td>23 </td><td>11 </td><td>5 </td><td>5 </td><td>11 </td><td>5 </td><td>5 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> sRNA </td><td>7 </td><td>4 </td><td>3 </td><td>2 </td><td>4 </td><td>3 </td><td>2 </td></tr><tr><td> Total </td><td>929 </td><td>794 </td><td>747 </td><td>782 </td><td>728 </td><td>677 </td><td>676 </td></tr><tr><td colspan="8">‘Coding region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>393 </td><td>415 </td><td>425 </td><td>479 </td><td>341 </td><td>334 </td><td>338 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> Total </td><td>400 </td><td>422 </td><td>432 </td><td>486 </td><td>348 </td><td>341 </td><td>345 </td></tr><tr><td colspan="8">‘Intergenic region’ </td></tr><tr><td> Total </td><td>26 </td><td>23 </td><td>23 </td><td>24 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>Total </td><td>1355 </td><td>1239 </td><td>1202 </td><td>1292 </td><td>1093 </td><td>1035 </td><td>1038 </td></tr></tbody></table></div><div content-id="DSS010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">  </th><th>Liquid </th><th colspan="3">Solid (days)<hr /> </th><th colspan="3">Overlapped<sup>d</sup> (days)<hr /> </th></tr><tr><th> </th><th>1 </th><th>2 </th><th>3 </th><th>1 </th><th>2 </th><th>3 </th></tr></thead><tbody><tr><td colspan="8">‘Regulatory region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>892 </td><td>772 </td><td>732 </td><td>768 </td><td>706 </td><td>662 </td><td>662 </td></tr><tr><td> tRNA<sup>b</sup> </td><td>23 </td><td>11 </td><td>5 </td><td>5 </td><td>11 </td><td>5 </td><td>5 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> sRNA </td><td>7 </td><td>4 </td><td>3 </td><td>2 </td><td>4 </td><td>3 </td><td>2 </td></tr><tr><td> Total </td><td>929 </td><td>794 </td><td>747 </td><td>782 </td><td>728 </td><td>677 </td><td>676 </td></tr><tr><td colspan="8">‘Coding region’ </td></tr><tr><td> Protein<sup>a</sup> </td><td>393 </td><td>415 </td><td>425 </td><td>479 </td><td>341 </td><td>334 </td><td>338 </td></tr><tr><td> rRNA<sup>c</sup> </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td><td>7 </td></tr><tr><td> Total </td><td>400 </td><td>422 </td><td>432 </td><td>486 </td><td>348 </td><td>341 </td><td>345 </td></tr><tr><td colspan="8">‘Intergenic region’ </td></tr><tr><td> Total </td><td>26 </td><td>23 </td><td>23 </td><td>24 </td><td>17 </td><td>17 </td><td>17 </td></tr><tr><td>Total </td><td>1355 </td><td>1239 </td><td>1202 </td><td>1292 </td><td>1093 </td><td>1035 </td><td>1038 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>AdpA-binding sites in the TIR region are counted only once.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup><em>Streptomyces griseus</em> has 66 tRNA genes, including ‘duplicated’ gene sets. Duplicated genes cannot be distinguished from each other in ChIP/ChAP analysis; therefore, they were excluded from the analysis. Only 26 transcriptional units containing 27 tRNA genes can be discriminated from others; AdpA was shown to bind to the upstream regions of 23 transcriptional units containing 24 tRNA genes.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>AdpA-binding sites in the common region of six rRNA gene clusters are counted only once.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>The number of AdpA-binding sites detected in both liquid and solid cultures.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167247" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7167249 id="7167249" class="section-title" data-legacy-id=s3c>3.3. Relationship between AdpA binding and transcriptional regulation</h3>
<p>We examined whether AdpA binding to each ‘putative regulatory region’ could affect transcription of the target gene. For protein-coding genes, gene expression profiles were compared by DNA microarray analysis between the N-His-AdpA-producing Δ<em>adpA</em> strain and the Δ<em>adpA</em> strain containing the empty integration vector pTYM19 on the chromosome. The transcriptome analysis showed that 736 and 457 genes were expressed at higher and lower levels, respectively, in the N-His-AdpA-producing strain when compared with the control Δ<em>adpA</em> strain (fold change ≥2 and ≤0.5, respectively, and <em>P</em>-value &lt;0.05). We obtained expression data of 662 genes from among the 796 genes that have one or more <em>in vivo</em> AdpA-binding sites in their ‘putative regulatory region’. Thus, we could evaluate the apparent significance of 743 of the 892 <em>in vivo</em> AdpA-binding sites in ‘putative regulatory region’; 214 sites (for 175 transcriptional units) and 89 sites (for 81 transcriptional units) were apparently involved in transcriptional activation and repression, respectively. We also examined apparent significance of several AdpA-binding sites in the ‘putative regulatory region’ for tRNA genes and rRNA genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Expression of only <em>bldA</em> (<em>trn42</em>) was higher in the N-His-AdpA-producing strain when compared with the control Δ<em>adpA</em> strain. We have confirmed that transcription of <em>bldA</em> is activated by AdpA.<sup><a reveal-id="DSS010C13" data-open="DSS010C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Altogether, we evaluated the apparent significance of 773 sites of the 922 <em>in vivo</em> AdpA-binding sites in ‘putative regulatory regions’. Of 773 sites, 215 sites (28%, for 176 transcriptional units) and 89 sites (12%, for 81 transcriptional units) were possibly involved in transcriptional activation and repression, respectively. In this study, we termed these 304 sites as ‘apparently relevant’ sites. Although we demonstrated that AdpA represses transcription of its own gene,<sup><a reveal-id="DSS010C16" data-open="DSS010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> it was unexpected that AdpA appears to repress more than 80 other transcriptional units (see Discussion). The <em>in vivo</em> binding of AdpA to the remaining 469 ‘putative regulatory region’ sites seemed not to affect gene expression, at least under the growth conditions examined. We termed these 469 sites as ‘apparently irrelevant’ sites.</p><p>We then compared the distribution of ChIP/ChAP-seq signal intensities (average values), which reflects the strength of AdpA–DNA interaction <em>in vivo</em>, between apparently relevant and irrelevant sites (Fig. <a reveal-id="DSS010F4" data-open="DSS010F4" class="link link-reveal link-table xref-fig">4</a>). No significant difference was observed between them. Furthermore, there was no significant difference in distribution of ChIP/ChAP-seq signal intensities among the three positional classifications of AdpA-binding sites (‘putative regulatory’, ‘coding’, and ‘intergenic’ regions) (Fig. <a reveal-id="DSS010F4" data-open="DSS010F4" class="link link-reveal link-table xref-fig">4</a>). These results indicated that <em>in vivo</em> AdpA-binding strength is not related to the functional significance of particular AdpA-binding sites. </p>                        <a id="7167258" scrollto-destination="7167258"></a>
<div id="DSS010F4" data-id="DSS010F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01004.jpeg?Expires=2147483647&Signature=CEHjzVxZIMAG-wA5TXxjDdEIemcShRWfLKWzUv6KCXRNj0IfZxxKf1mjWgMvDbPOycj0rLDlN1QK0GfDWqGU4hFlO9NH2tbck3L3C7w8qYci9y-GhUILHAtvnmB-D92Khblu~HxKeO084l~509c~SVPky3TONNusdUkM~A3DmFcl8y8w-BCWkxvuHMA7N7WhaOUvfMsv6qbfuHlkupPdBtlvWsLljCmPKfpzQUSZoTmyVMMWQdaijOSBUK0xZw852vmahhkFOPlf71gsF4kgpD6m2XxE-njN2S4dCIS3GmMTKP-yrMd7~dIUJ4fhSv4KB3A88Xr9XlHKXNILJxDj4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vivo AdpA-binding strength is not related to the functional significance of particular AdpA-binding sites. One-dimensional scatter plots of AdpA-binding strength in vivo are shown for four respective classifications of AdpA-binding sites. The means of peak signals derived from ChIP-seq and ChAP-seq are plotted. Box plots are also shown. In the box plot, the bottom and top of the box are the 25th and 75th percentile (the lower and upper quartiles), respectively, and the band near the middle of the box is the 50th percentile (the median). The ends of the whiskers represent the lowest datum still within 1.5 IQR of the lower quartile, and the highest datum still within 1.5 IQR of the upper quartile (IQR, inter-quartile range, is equal to the difference between the upper and lower quartiles)." path-from-xml="dss01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167258/dss01004.jpeg" path-from-xml="dss01004.jpeg" target="_blank">View large</a><a section="7167258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01004.jpeg?Expires=2147483647&Signature=0gHDNKFM6gU5FURIBvwpfShkMYXmrIU4d0Ir7fguqDsjppSOQV2JAMm~F2drrBOhjzsyA7eQF43oyz-4ZaHzxb~i6mRIa5WNYDAlrXCCtfWH7j2BmXRWSw7gcwVLwDLNR445rku29u5id2qAkDB5d7SxtwA513qdhOYD9tHqjP7yoQlgy8X-BtVJkYjVh-J~Ed34IkIg2eI6q72y1hRge3T2tPvSZa8WZATcKV8zE0ap-Os8Hfq1~uV3qnnolCNLPHo9qJE~k4nyZxSlEWjutSKayGUhi4C6QMw3NDFhdRy-S5uk4hP6sQdHP6N8vQWa0ewcXIpDQYWjpgwdN460Ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167258&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>In vivo AdpA-binding strength is not related to the functional significance of particular AdpA-binding sites. One-dimensional scatter plots of AdpA-binding strength <em>in vivo</em> are shown for four respective classifications of AdpA-binding sites. The means of peak signals derived from ChIP-seq and ChAP-seq are plotted. Box plots are also shown. In the box plot, the bottom and top of the box are the 25th and 75th percentile (the lower and upper quartiles), respectively, and the band near the middle of the box is the 50th percentile (the median). The ends of the whiskers represent the lowest datum still within 1.5 IQR of the lower quartile, and the highest datum still within 1.5 IQR of the upper quartile (IQR, inter-quartile range, is equal to the difference between the upper and lower quartiles).</p></div></div><div content-id="DSS010F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01004.jpeg?Expires=2147483647&Signature=CEHjzVxZIMAG-wA5TXxjDdEIemcShRWfLKWzUv6KCXRNj0IfZxxKf1mjWgMvDbPOycj0rLDlN1QK0GfDWqGU4hFlO9NH2tbck3L3C7w8qYci9y-GhUILHAtvnmB-D92Khblu~HxKeO084l~509c~SVPky3TONNusdUkM~A3DmFcl8y8w-BCWkxvuHMA7N7WhaOUvfMsv6qbfuHlkupPdBtlvWsLljCmPKfpzQUSZoTmyVMMWQdaijOSBUK0xZw852vmahhkFOPlf71gsF4kgpD6m2XxE-njN2S4dCIS3GmMTKP-yrMd7~dIUJ4fhSv4KB3A88Xr9XlHKXNILJxDj4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="In vivo AdpA-binding strength is not related to the functional significance of particular AdpA-binding sites. One-dimensional scatter plots of AdpA-binding strength in vivo are shown for four respective classifications of AdpA-binding sites. The means of peak signals derived from ChIP-seq and ChAP-seq are plotted. Box plots are also shown. In the box plot, the bottom and top of the box are the 25th and 75th percentile (the lower and upper quartiles), respectively, and the band near the middle of the box is the 50th percentile (the median). The ends of the whiskers represent the lowest datum still within 1.5 IQR of the lower quartile, and the highest datum still within 1.5 IQR of the upper quartile (IQR, inter-quartile range, is equal to the difference between the upper and lower quartiles)." path-from-xml="dss01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167258/dss01004.jpeg" path-from-xml="dss01004.jpeg" target="_blank">View large</a><a section="7167258" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01004.jpeg?Expires=2147483647&Signature=0gHDNKFM6gU5FURIBvwpfShkMYXmrIU4d0Ir7fguqDsjppSOQV2JAMm~F2drrBOhjzsyA7eQF43oyz-4ZaHzxb~i6mRIa5WNYDAlrXCCtfWH7j2BmXRWSw7gcwVLwDLNR445rku29u5id2qAkDB5d7SxtwA513qdhOYD9tHqjP7yoQlgy8X-BtVJkYjVh-J~Ed34IkIg2eI6q72y1hRge3T2tPvSZa8WZATcKV8zE0ap-Os8Hfq1~uV3qnnolCNLPHo9qJE~k4nyZxSlEWjutSKayGUhi4C6QMw3NDFhdRy-S5uk4hP6sQdHP6N8vQWa0ewcXIpDQYWjpgwdN460Ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167258&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>In vivo AdpA-binding strength is not related to the functional significance of particular AdpA-binding sites. One-dimensional scatter plots of AdpA-binding strength <em>in vivo</em> are shown for four respective classifications of AdpA-binding sites. The means of peak signals derived from ChIP-seq and ChAP-seq are plotted. Box plots are also shown. In the box plot, the bottom and top of the box are the 25th and 75th percentile (the lower and upper quartiles), respectively, and the band near the middle of the box is the 50th percentile (the median). The ends of the whiskers represent the lowest datum still within 1.5 IQR of the lower quartile, and the highest datum still within 1.5 IQR of the upper quartile (IQR, inter-quartile range, is equal to the difference between the upper and lower quartiles).</p></div></div>                            <h3 scrollto-destination=7167261 id="7167261" class="section-title" data-legacy-id=s3d>3.4. Confirmation of <em>in vitro</em> AdpAbinding to the <em>in vivo</em> AdpA-binding sites detected by ChIP/ChAP-seq</h3>
<p>We examined whether the AdpA-C-His protein purified from <em>Escherichia coli</em> binds to the <em>in vivo</em> AdpA-binding sites by comprehensive competitive EMSA. We tested all 304 ‘apparently relevant’ AdpA-binding sites. ChIP/ChAP-seq signal peaks were detected within 50 bp from all of the ‘category-I’ AdpA-binding sites, except AdBS-<em>sgiA</em>; therefore, 130–200 bp DNA fragments containing a centrally positioned predicted AdpA-binding site (ChIP/ChAP-seq peak) were amplified by PCR. These DNA fragments were used as the competitors in EMSA using AdpA-C-His. In the competitive EMSA, we used two DIG-labelled probes with different AdpA-binding affinities: the high-affinity <em>sgmA</em> probe and the low-affinity <em>sgiA</em> probe.<sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> We detected 247 DNA fragments that at the least competed with the <em>sgiA</em> probe; however, 57 DNA fragments did not. AdpA frequently shows relatively low DNA-binding strength to short DNA fragments<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and true AdpA-binding sites may be separated from the ChIP/ChAP signal peaks by more than 50 bp in some cases; we increased the lengths of these 57 AdpA-unbound DNA fragments to 200–250 bp and re-examined their AdpA binding by competitive EMSA. As a result, we detected 37 DNA fragments that at least competed with the <em>sgiA</em> probe, but the remaining 20 DNA fragments did not. We identified at least one sequence that is similar to the AdpA-binding consensus sequence on each of the 284 DNA fragments to which AdpA bound <em>in vitro</em>. We regarded the consensus-like sequences as probable AdpA-binding sequences. When more than two possible AdpA-binding sequences existed in a DNA fragment, we selected the sequence closest to the ChIP/ChAP-seq signal peak in this study. In summary, of 304 ‘apparently relevant’ <em>in vivo</em> AdpA-binding sites, 284 sites (93%) were confirmed to be bound by AdpA <em>in vitro</em>. The remaining 20 AdpA-binding sites may not have resulted from false-positive signals, because the peaks were detected similarly by both ChIP-seq and ChAP-seq analyses, and because there was an AdpA-binding consensus-like sequence around each peak. It is possible that AdpA binds to these sites cooperatively with some other DNA-binding protein <em>in vivo</em> (see Discussion).</p>                            <h3 scrollto-destination=7167266 id="7167266" class="section-title" data-legacy-id=s3e>3.5. Comparison of the strength of the AdpA–DNA interaction between <em>in vivo</em> and <em>in vitro</em></h3>
<p>We next determined whether there was a correlation in the strength of the AdpA–DNA interaction between <em>in vivo</em> and <em>in vitro</em> for the 304 ‘apparently relevant’ AdpA-binding sites. To roughly evaluate <em>in vitro</em> AdpA-binding strength, we rated the binding strength of AdpA to each DNA fragment in six ranks (rank ‘0’ to ‘5’) as described previously.<sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Briefly, the strength of the AdpA–DNA interaction <em>in vitro</em> was assessed by competitive EMSA using two DIG-labelled probes with different AdpA-binding affinities. <em>In vitro</em> AdpA-binding strength is often underestimated when an AdpA-binding sequence is located at the end of the DNA fragment.<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Therefore, we selected 115 AdpA-binding sequences, which were located within 30 bp from the centre of each DNA fragment (130–200 bp), and compared <em>in vitro</em> AdpA-binding strength to <em>in vivo</em> AdpA-binding strength deduced from ChIP/ChAP-seq signal intensities (average values) (Fig. <a reveal-id="DSS010F5" data-open="DSS010F5" class="link link-reveal link-table xref-fig">5</a>). Contrary to our expectation, there was very poor correlation of AdpA-binding strength between <em>in vitro</em> and <em>in vivo</em> conditions (correlation coefficient was 0.43). However, the sequences that were strongly bound by AdpA <em>in vitro</em> (those classified into ranks ‘5’ and ‘4’) generally showed relatively high <em>in vivo</em> AdpA-binding strength. </p>                        <a id="7167272" scrollto-destination="7167272"></a>
<div id="DSS010F5" data-id="DSS010F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01005.jpeg?Expires=2147483647&Signature=WxD6qkD4hWuj6LdwtyN33ze2te~6xVDybfwVvsjbeuJjLQ6HdVca55e7WAIZkbQj1UzhRISzRjdzZ5VDrgUaE1WOI1DN0XYqLSunoBK34YUWUmNCLcnfHD2MlZvuVioIIVLzcSs2KqoeFMtm~IO5sSGIgh0h9ebc8Ro-P6oq3FNIQj1z0g-eZoyKu~vKFa4g805baXepdLcni~HvYgd0iPpsqACBDh5x--n8HBPDR3F7r-JO9eRdXNDUzL4D8XFQObATo0YdkLk2YocR8UB2G4Xyi8k4bvKhv9N80dA4QATV7VuV20GLX9fpOTNOFwL2HvEHVcamCjigM~vnd9Zg0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the strength of the AdpA–DNA interaction in vivo and in vitro. AdpA binding to 115 regions are examined (see text). One-dimensional scatter plots of in vivo AdpA-binding strength are shown for six respective ranks of in vitro AdpA-binding strength. Box plots (see the legend for Fig. 4) are also shown." path-from-xml="dss01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167272/dss01005.jpeg" path-from-xml="dss01005.jpeg" target="_blank">View large</a><a section="7167272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01005.jpeg?Expires=2147483647&Signature=rrS8LcDOSHjgvPj3ndcLVgiW~I2-UGhXuI1AQnv97ULEwHtlVaY7S2R9Lbkl4mLXqc0hgvYtvz3ftVRXtX6dRMJxABSr~OW0p0rIXENfI8lrksdK3QEZpGMsLBCPqC2UCTXqS7Ogy-G6ElUkf4h6yZg2pwgwwryeYuj35geEZBkJXICQognZMiEdabTtnKtXvztbq3zsBSWP7qVr6dr9k2-CvkIFTyLfI7OEsziw14nxFk7yt0lQ65q3hV3NJisr87zBZCKyF9smnS-WzMoH3J4W02tSlbpaROH41yeehoVUi4KANASxbzV6xFL9yGHIpXkoAAcmMcclTltIV0SQyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167272&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the strength of the AdpA–DNA interaction <em>in vivo</em> and <em>in vitro</em>. AdpA binding to 115 regions are examined (see text). One-dimensional scatter plots of <em>in vivo</em> AdpA-binding strength are shown for six respective ranks of <em>in vitro</em> AdpA-binding strength. Box plots (see the legend for Fig. <a reveal-id="DSS010F4" data-open="DSS010F4" class="link link-reveal link-table xref-fig">4</a>) are also shown.</p></div></div><div content-id="DSS010F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01005.jpeg?Expires=2147483647&Signature=WxD6qkD4hWuj6LdwtyN33ze2te~6xVDybfwVvsjbeuJjLQ6HdVca55e7WAIZkbQj1UzhRISzRjdzZ5VDrgUaE1WOI1DN0XYqLSunoBK34YUWUmNCLcnfHD2MlZvuVioIIVLzcSs2KqoeFMtm~IO5sSGIgh0h9ebc8Ro-P6oq3FNIQj1z0g-eZoyKu~vKFa4g805baXepdLcni~HvYgd0iPpsqACBDh5x--n8HBPDR3F7r-JO9eRdXNDUzL4D8XFQObATo0YdkLk2YocR8UB2G4Xyi8k4bvKhv9N80dA4QATV7VuV20GLX9fpOTNOFwL2HvEHVcamCjigM~vnd9Zg0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the strength of the AdpA–DNA interaction in vivo and in vitro. AdpA binding to 115 regions are examined (see text). One-dimensional scatter plots of in vivo AdpA-binding strength are shown for six respective ranks of in vitro AdpA-binding strength. Box plots (see the legend for Fig. 4) are also shown." path-from-xml="dss01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167272/dss01005.jpeg" path-from-xml="dss01005.jpeg" target="_blank">View large</a><a section="7167272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01005.jpeg?Expires=2147483647&Signature=rrS8LcDOSHjgvPj3ndcLVgiW~I2-UGhXuI1AQnv97ULEwHtlVaY7S2R9Lbkl4mLXqc0hgvYtvz3ftVRXtX6dRMJxABSr~OW0p0rIXENfI8lrksdK3QEZpGMsLBCPqC2UCTXqS7Ogy-G6ElUkf4h6yZg2pwgwwryeYuj35geEZBkJXICQognZMiEdabTtnKtXvztbq3zsBSWP7qVr6dr9k2-CvkIFTyLfI7OEsziw14nxFk7yt0lQ65q3hV3NJisr87zBZCKyF9smnS-WzMoH3J4W02tSlbpaROH41yeehoVUi4KANASxbzV6xFL9yGHIpXkoAAcmMcclTltIV0SQyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167272&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the strength of the AdpA–DNA interaction <em>in vivo</em> and <em>in vitro</em>. AdpA binding to 115 regions are examined (see text). One-dimensional scatter plots of <em>in vivo</em> AdpA-binding strength are shown for six respective ranks of <em>in vitro</em> AdpA-binding strength. Box plots (see the legend for Fig. <a reveal-id="DSS010F4" data-open="DSS010F4" class="link link-reveal link-table xref-fig">4</a>) are also shown.</p></div></div><p>Next, we calculated the ‘match score’ of these 115 AdpA-binding sequences to the consensus AdpA-binding sequence by Mast<sup><a reveal-id="DSS010C17" data-open="DSS010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and examined a relationship between the match score (which represented the degree of similarity to the AdpA-binding consensus sequence) and AdpA-binding strength. There was almost no correlation between the match score and <em>in vivo</em> AdpA-binding strength (correlation coefficient was 0.23) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4A</a>). However, the sequences that were strongly bound by AdpA <em>in vivo</em> tended to show relatively high match scores. There was also a very poor correlation between the match score and <em>in vitro</em> AdpA-binding strength (correlation coefficient was 0.36), but the sequences that were strongly bound by AdpA <em>in vitro</em> (especially those classified into ranks ‘5’ and ‘4’) showed relatively high match scores (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4B</a>). Note that AdpA binds DNA in two different ways. Type I binding occurs when both subunits of the AdpA dimer bind each of the two divergent AdpA-binding sequences and type II, where only one subunit binds a single AdpA-binding sequence to anchor the AdpA dimer. However, we did not consider any possible type-I AdpA binding in the analyses described above; only single AdpA-binding consensus-like sequences found in each AdpA-binding region were used for the analyses (see Discussion). Among the 115 AdpA-binding sequences, there were six kinds of duplicate sequences and four kinds of triplicate sequences. For example, the AdpA-binding sequences 5′-TGGCCGAAAA-3′ and 5′-TGTCCGAAAA-3′ were found in two and three different AdpA-binding sites, respectively. Interestingly, in 7 of the 10 cases, <em>in vitro</em> AdpA-binding strength significantly differed between two (or three) DNA fragments containing an identical AdpA-binding sequence (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). This result indicated that not only the 10 nucleotides of the AdpA-binding consensus sequence, but also the neighbouring nucleotide sequences affect the AdpA-binding strength <em>in vitro</em> (see Discussion).<sup><a reveal-id="DSS010C2" data-open="DSS010C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS010C7" data-open="DSS010C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                            <h3 scrollto-destination=7167277 id="7167277" class="section-title" data-legacy-id=s3f>3.6. In vivo AdpA-binding profiles on solid culture</h3>
<p>Our previous DNA microarray analysis showed that <em>S. griseus</em> had significantly different gene expression profiles between liquid and solid cultures (unpublished data). There had also been a suggestion that there should be a large difference in the genes regulated by AdpA between liquid and solid cultures (unpublished data). Therefore, we were interested in the <em>in vivo</em> difference in the AdpA-binding profiles between liquid and solid cultures.</p><p>First, we compared the transcriptomes using a DNA microarray between the N-His-AdpA-producing Δ<em>adpA</em> strain and the control Δ<em>adpA</em> strain containing the empty vector pTYM19. RNA was extracted from both strains grown on YMPD agar at 28°C for 1–3 days. In this culture, the N-His-AdpA-producing Δ<em>adpA</em> strain grew as substrate mycelium at day 1, as substrate mycelium plus aerial mycelium at day 2, and as substrate mycelium plus aerial mycelium with spores at day 3. The control Δ<em>adpA</em> strain could not make aerial mycelium and spores throughout culture. At day 1, 531 and 457 genes were expressed at higher and lower levels, respectively, in the N-His-AdpA-producing strain when compared with the control Δ<em>adpA</em> strain (fold change ≥2 and ≤0.5, respectively, and <em>P</em>-value &lt;0.05). This indicated that 531 and 457 genes were activated and repressed, respectively, by AdpA (directly or indirectly) at day 1. Similarly, the numbers of apparently AdpA-activated and repressed genes were 509 and 394 (at day 2), and 611 and 606 (at day 3), respectively. The mycelium at day 1 on solid culture should roughly correspond to the mycelium of 18-h liquid culture, in that both mycelia were in the late exponential growth phase; therefore, we compared AdpA-regulated genes between the 18-h liquid culture and 1-day solid culture. When genes whose transcriptional data were statistically available in both the transcriptome analyses were compared, only 281 genes were apparently activated by AdpA in both liquid and solid cultures, while 437 and 202 genes were activated specifically in liquid and solid cultures, respectively (Fig. <a reveal-id="DSS010F6" data-open="DSS010F6" class="link link-reveal link-table xref-fig">6</a>A). On the other hand, only 100 genes were apparently repressed by AdpA in both liquid and solid cultures, while 348 and 264 genes were specifically repressed in liquid and solid cultures, respectively (Fig. <a reveal-id="DSS010F6" data-open="DSS010F6" class="link link-reveal link-table xref-fig">6</a>A). Thus, we confirmed that there was indeed a large difference in the genes regulated by AdpA between liquid and solid cultures. </p>                        <a id="7167283" scrollto-destination="7167283"></a>
<div id="DSS010F6" data-id="DSS010F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01006.jpeg?Expires=2147483647&Signature=4kwui2L5o~tjuxIDizjagMQzRxzPzhclmGOqNR7s75mZjOlNS9EavebkkJeL88OGyjHVdqZwpn7aROQZ3kz~GqUmhkzcXUcz6ru7QMWAKZdW-4-NdF0XJggSzeWqwR43K6Hz86FN22bGFssYjHVZm443yBVFmlgG~3Pm~QoOKNfjgCeWk7aznCtXkxAWTb74tpp7w2ty6cxeMnazxx0MC6gaQpMiC5O12F1q-py7UTyp2RKF-xY9a3c9flYN5OJbd78gT5z27vGx1gsi3wbOzG9uiVhXRVPy92UCVcQ736He6jXaf3XY1vIKTbUTcLAl7bVTw44XB6ZkCnEuhbadfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagrams showing different expressions of AdpA-regulated genes (A) and similar AdpA-binding profiles (B) between 18-h liquid and 1-day solid cultures. (A) Numbers of genes activated (left) and repressed (right) by AdpA directly or indirectly. (B) Numbers of AdpA-binding sites in ‘putative regulatory regions’." path-from-xml="dss01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167283/dss01006.jpeg" path-from-xml="dss01006.jpeg" target="_blank">View large</a><a section="7167283" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01006.jpeg?Expires=2147483647&Signature=MegptKK4BKsZ7z4xx80shRLBsWZtutlVliP0lQLstxcjV4SNryMLa-VJn4r~~Ze5QcCTGE02qeVP1UqJP-Ebeoouwsr~1FxrDdInlEov6eQK0WQ-seb33riWdy8l15DXm7hUfjV6Dhut~ZvDKgnFEswyDxwWhTo9od8kTUL3uwbZLu3CBNt~MI-YxZyBVUStzPNSXiFT7zRIUEue55mc-ZI9p42tcydBLDmE5z1V5L37bhk7Hre28Tkj3ZP3FfmBsmgkaJDgo2bB2KOb6clEnL59rBRejDvUwomzWuLWrMzptrPYd1NvGPukt7Ast0jmu-6V88uCvAlLjrHGS~Ry-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167283&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagrams showing different expressions of AdpA-regulated genes (A) and similar AdpA-binding profiles (B) between 18-h liquid and 1-day solid cultures. (A) Numbers of genes activated (left) and repressed (right) by AdpA directly or indirectly. (B) Numbers of AdpA-binding sites in ‘putative regulatory regions’.</p></div></div><div content-id="DSS010F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/m_dss01006.jpeg?Expires=2147483647&Signature=4kwui2L5o~tjuxIDizjagMQzRxzPzhclmGOqNR7s75mZjOlNS9EavebkkJeL88OGyjHVdqZwpn7aROQZ3kz~GqUmhkzcXUcz6ru7QMWAKZdW-4-NdF0XJggSzeWqwR43K6Hz86FN22bGFssYjHVZm443yBVFmlgG~3Pm~QoOKNfjgCeWk7aznCtXkxAWTb74tpp7w2ty6cxeMnazxx0MC6gaQpMiC5O12F1q-py7UTyp2RKF-xY9a3c9flYN5OJbd78gT5z27vGx1gsi3wbOzG9uiVhXRVPy92UCVcQ736He6jXaf3XY1vIKTbUTcLAl7bVTw44XB6ZkCnEuhbadfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Venn diagrams showing different expressions of AdpA-regulated genes (A) and similar AdpA-binding profiles (B) between 18-h liquid and 1-day solid cultures. (A) Numbers of genes activated (left) and repressed (right) by AdpA directly or indirectly. (B) Numbers of AdpA-binding sites in ‘putative regulatory regions’." path-from-xml="dss01006.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7167283/dss01006.jpeg" path-from-xml="dss01006.jpeg" target="_blank">View large</a><a section="7167283" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss01006.jpeg?Expires=2147483647&Signature=MegptKK4BKsZ7z4xx80shRLBsWZtutlVliP0lQLstxcjV4SNryMLa-VJn4r~~Ze5QcCTGE02qeVP1UqJP-Ebeoouwsr~1FxrDdInlEov6eQK0WQ-seb33riWdy8l15DXm7hUfjV6Dhut~ZvDKgnFEswyDxwWhTo9od8kTUL3uwbZLu3CBNt~MI-YxZyBVUStzPNSXiFT7zRIUEue55mc-ZI9p42tcydBLDmE5z1V5L37bhk7Hre28Tkj3ZP3FfmBsmgkaJDgo2bB2KOb6clEnL59rBRejDvUwomzWuLWrMzptrPYd1NvGPukt7Ast0jmu-6V88uCvAlLjrHGS~Ry-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7167283&ar=475336&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01006.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Venn diagrams showing different expressions of AdpA-regulated genes (A) and similar AdpA-binding profiles (B) between 18-h liquid and 1-day solid cultures. (A) Numbers of genes activated (left) and repressed (right) by AdpA directly or indirectly. (B) Numbers of AdpA-binding sites in ‘putative regulatory regions’.</p></div></div><p>The AdpA-binding profiles at days 1, 2, and 3 were then examined by ChAP-seq. Similar to the ChIP/ChAP-seq analysis in liquid culture, the signal intensity of AdBS-<em>adsA</em> was used as a threshold to exclude background noise. Contrary to our expectation, the AdpA-binding profiles did not change significantly among 18-h liquid culture and 1-, 2-, and 3-day solid cultures (Table <a reveal-id="DSS010TB1" data-open="DSS010TB1" class="link link-reveal link-table xref-fig">1</a>). When <em>in vivo</em> AdpA-binding sites on ‘putative regulatory regions’ were compared between 18-h liquid culture and 1-day solid culture, 728 sites were shared, and 201 and 66 sites were specifically detected in the liquid and solid cultures, respectively (Fig. <a reveal-id="DSS010F6" data-open="DSS010F6" class="link link-reveal link-table xref-fig">6</a>B). Interestingly, many of these specifically detected sites have relatively low ChIP-seq and/or ChAP-seq signal intensities. Transcriptome analysis indicated that only 3 of the 201 sites specifically detected in the liquid culture were apparently involved in gene regulation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). In other words, only three liquid culture-specific AdpA-binding sites were ‘apparently relevant’. Similarly, only 3 of the 66 solid culture-specific sites were ‘apparently relevant’ (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Biological significance of the liquid or solid culture-specific regulation of these genes by AdpA is unknown. However, from these results, we concluded that the large difference in the genes regulated by AdpA between liquid and solid cultures resulted from regulatory elements other than AdpA binding; AdpA-binding profiles did not change significantly between cultures.</p>                            <h3 scrollto-destination=7167288 id="7167288" class="section-title" data-legacy-id=s3g>3.7. Extraction of putative AdpA-target genes</h3>
<p>ChAP-seq and transcriptome analyses in solid culture suggested 150 new ‘apparently relevant’ AdpA-binding sites in addition to the 304 sites described above. In total, of 1001 <em>in vivo</em> AdpA-binding sites on ‘putative regulatory region’ of protein-coding genes in liquid and/or solid cultures, we identified 262 sites (26%, for 217 transcriptional units, 342 genes) and 172 sites (17%, for 156 transcriptional units, 224 genes) that were apparently involved in transcriptional activation and repression, respectively, in either liquid or solid culture. In addition, 20 AdpA-binding sites that apparently had contradictory effects on gene regulation among 18-h liquid culture and 1-, 2-, and 3-day solid cultures were detected (e.g. <em>SGR4671</em> was apparently activated in 18-h liquid culture by AdpA and repressed in 2- and 3-day solid culture). These sites were excluded from the following lists (Tables <a reveal-id="DSS010TB2" data-open="DSS010TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSS010TB3" data-open="DSS010TB3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>). We then catalogued candidates for direct target genes of AdpA. AdpA presumably activates the promoters of 88 transcriptional units containing 145 genes in both liquid and solid cultures, which include 7 ‘category-I’ AdpA-target promoters (<em>adsA</em>, <em>amfR</em>, <em>strR</em>, <em>sprB</em>, <em>sgmA</em>, AdBS2-<em>orf1</em>, and AdBS4-<em>orf1</em>) and 20 probable AdpA-target promoters suggested by Hara <em>et al.</em><sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and/or Akanuma <em>et al</em>.<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup> (Table <a reveal-id="DSS010TB2" data-open="DSS010TB2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). In accordance with the proposed function of AdpA, these AdpA-target genes included many genes probably involved in secondary metabolism and morphological differentiation. On the other hand, AdpA presumably represses 40 transcriptional units containing 54 genes in both liquid and solid cultures (<a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss010/-/DC1" target="_blank">Table 3</a>). We also listed 54 AdpA-target transcriptional units (79 genes) that are activated specifically on solid culture (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). They included four ‘category-I’ AdpA-target genes (<em>ssgA</em>, <em>sprU</em>, <em>sprT</em>, and AdBS3-<em>orfA</em>) and four known candidates for AdpA-target transcriptional units<sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup>; however, many of them (56%) were categorized as function-unknown genes. Note that we estimated transcriptional units from transcriptome data, gene synteny, and functional relationship with its neighbouring genes. </p>                        <a id="7167292" scrollto-destination="7167292"></a>
<div content-id="DSS010TB2" class="table-modal"><div class="table-wrap"><div id="DSS010TB2" class="table-wrap-title" data-id="DSS010TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of genes that are probably activated directly by AdpA in both liquid and solid cultures (selected<sup>a</sup>)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo<sup>b</sup> </th><th>Identified
previously<sup>c</sup> </th></tr></thead><tbody><tr><td>SGR745 </td><td>SGR745 </td><td>Putative M23-family secreted peptidase </td><td>p </td><td> </td></tr><tr><td>SGR919–SGR916 </td><td>SGR919 </td><td>Hypothetical protein </td><td> </td><td>2 </td></tr><tr><td> </td><td>SGR918 </td><td>Putative subtilisin-like serine protease </td><td>p </td><td> </td></tr><tr><td> </td><td>SGR917 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td> </td><td>SGR916 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>SGR1063–SGR1059 </td><td>SGR1063 </td><td><em>rarA</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1062 </td><td><em>rarB</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1061 </td><td><em>rarC</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1060 </td><td><em>rarD</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1059 </td><td><em>rarE</em> </td><td>m, s </td><td> </td></tr><tr><td>SGR2079 </td><td>SGR2079 </td><td>terpene cyclase (<em>gcoA</em>) </td><td>s </td><td>2 </td></tr><tr><td>SGR2095 </td><td>SGR2095 </td><td><em>sgmA</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR2393 </td><td>SGR2393 </td><td><em>amfR</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR2446–SGR2447 </td><td>SGR2446 </td><td>Tyrosinase co-factor protein (<em>melC1–2</em>) </td><td>s </td><td>2, 3 </td></tr><tr><td> </td><td>SGR2447 </td><td>Tyrosinase (<em>melC2–2</em>) </td><td>s </td><td> </td></tr><tr><td>SGR3307–SGR3306 </td><td>SGR3307 </td><td>Anti-sigma factor antagonist (<em>bldG</em>) </td><td>m </td><td> </td></tr><tr><td> </td><td>SGR3306 </td><td>Putative anti-sigma factor </td><td> </td><td> </td></tr><tr><td>SGR3340 </td><td>SGR3340 </td><td>Putative WhiB-family transcriptional regulator (<em>wblA</em>) </td><td>m, s </td><td>2 </td></tr><tr><td>SGR3902 </td><td>SGR3902 </td><td>Sporulation associated protein ORF1590 </td><td>m </td><td>2 </td></tr><tr><td>SGR4151 </td><td>SGR4151 </td><td><em>adsA</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR4809 </td><td>SGR4809 </td><td>Putative lantibiotic modifying enzyme </td><td>s </td><td> </td></tr><tr><td>SGR5762 </td><td>SGR5762 </td><td><em>sprB</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR5914 </td><td>SGR5914 </td><td><em>strU</em> </td><td>s </td><td>2 </td></tr><tr><td>SGR5922–SGR5923 </td><td>SGR5922 </td><td><em>stsB</em> </td><td>s </td><td> </td></tr><tr><td> </td><td>SGR5923 </td><td><em>stsA</em> </td><td>s </td><td> </td></tr><tr><td>SGR5931–SGR5932 </td><td>SGR5931 </td><td><em>strR</em> </td><td>s </td><td>1, 2 </td></tr><tr><td> </td><td>SGR5932 </td><td><em>aphD</em> </td><td>s </td><td> </td></tr><tr><td>SGR6071 </td><td>SGR6071 </td><td>Putative LAL-subfamily transcriptional regulator
(<em>AdBS4-orf1</em>) </td><td>s </td><td>1 </td></tr><tr><td>trn42 </td><td>trn42 </td><td><em>bldA</em> </td><td>m, s </td><td> </td></tr></tbody></table></div><div content-id="DSS010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo<sup>b</sup> </th><th>Identified
previously<sup>c</sup> </th></tr></thead><tbody><tr><td>SGR745 </td><td>SGR745 </td><td>Putative M23-family secreted peptidase </td><td>p </td><td> </td></tr><tr><td>SGR919–SGR916 </td><td>SGR919 </td><td>Hypothetical protein </td><td> </td><td>2 </td></tr><tr><td> </td><td>SGR918 </td><td>Putative subtilisin-like serine protease </td><td>p </td><td> </td></tr><tr><td> </td><td>SGR917 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td> </td><td>SGR916 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>SGR1063–SGR1059 </td><td>SGR1063 </td><td><em>rarA</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1062 </td><td><em>rarB</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1061 </td><td><em>rarC</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1060 </td><td><em>rarD</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1059 </td><td><em>rarE</em> </td><td>m, s </td><td> </td></tr><tr><td>SGR2079 </td><td>SGR2079 </td><td>terpene cyclase (<em>gcoA</em>) </td><td>s </td><td>2 </td></tr><tr><td>SGR2095 </td><td>SGR2095 </td><td><em>sgmA</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR2393 </td><td>SGR2393 </td><td><em>amfR</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR2446–SGR2447 </td><td>SGR2446 </td><td>Tyrosinase co-factor protein (<em>melC1–2</em>) </td><td>s </td><td>2, 3 </td></tr><tr><td> </td><td>SGR2447 </td><td>Tyrosinase (<em>melC2–2</em>) </td><td>s </td><td> </td></tr><tr><td>SGR3307–SGR3306 </td><td>SGR3307 </td><td>Anti-sigma factor antagonist (<em>bldG</em>) </td><td>m </td><td> </td></tr><tr><td> </td><td>SGR3306 </td><td>Putative anti-sigma factor </td><td> </td><td> </td></tr><tr><td>SGR3340 </td><td>SGR3340 </td><td>Putative WhiB-family transcriptional regulator (<em>wblA</em>) </td><td>m, s </td><td>2 </td></tr><tr><td>SGR3902 </td><td>SGR3902 </td><td>Sporulation associated protein ORF1590 </td><td>m </td><td>2 </td></tr><tr><td>SGR4151 </td><td>SGR4151 </td><td><em>adsA</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR4809 </td><td>SGR4809 </td><td>Putative lantibiotic modifying enzyme </td><td>s </td><td> </td></tr><tr><td>SGR5762 </td><td>SGR5762 </td><td><em>sprB</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR5914 </td><td>SGR5914 </td><td><em>strU</em> </td><td>s </td><td>2 </td></tr><tr><td>SGR5922–SGR5923 </td><td>SGR5922 </td><td><em>stsB</em> </td><td>s </td><td> </td></tr><tr><td> </td><td>SGR5923 </td><td><em>stsA</em> </td><td>s </td><td> </td></tr><tr><td>SGR5931–SGR5932 </td><td>SGR5931 </td><td><em>strR</em> </td><td>s </td><td>1, 2 </td></tr><tr><td> </td><td>SGR5932 </td><td><em>aphD</em> </td><td>s </td><td> </td></tr><tr><td>SGR6071 </td><td>SGR6071 </td><td>Putative LAL-subfamily transcriptional regulator
(<em>AdBS4-orf1</em>) </td><td>s </td><td>1 </td></tr><tr><td>trn42 </td><td>trn42 </td><td><em>bldA</em> </td><td>m, s </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>An extract of genes involved in morphological differentiation, secondary metabolism, and extracellular protease. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> for full list. <sup>b</sup>m, involved in morphological differentiation; s, involved in secondary metabolism; p, extracellular protease. <sup>c</sup>1, ‘category-I’ target; 2, suggested by Hara <em>et al.</em><sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup>; 3, suggested by Akanuma <em>et al</em>.<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167292" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS010TB2" class="table-wrap-title" data-id="DSS010TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of genes that are probably activated directly by AdpA in both liquid and solid cultures (selected<sup>a</sup>)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo<sup>b</sup> </th><th>Identified
previously<sup>c</sup> </th></tr></thead><tbody><tr><td>SGR745 </td><td>SGR745 </td><td>Putative M23-family secreted peptidase </td><td>p </td><td> </td></tr><tr><td>SGR919–SGR916 </td><td>SGR919 </td><td>Hypothetical protein </td><td> </td><td>2 </td></tr><tr><td> </td><td>SGR918 </td><td>Putative subtilisin-like serine protease </td><td>p </td><td> </td></tr><tr><td> </td><td>SGR917 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td> </td><td>SGR916 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>SGR1063–SGR1059 </td><td>SGR1063 </td><td><em>rarA</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1062 </td><td><em>rarB</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1061 </td><td><em>rarC</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1060 </td><td><em>rarD</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1059 </td><td><em>rarE</em> </td><td>m, s </td><td> </td></tr><tr><td>SGR2079 </td><td>SGR2079 </td><td>terpene cyclase (<em>gcoA</em>) </td><td>s </td><td>2 </td></tr><tr><td>SGR2095 </td><td>SGR2095 </td><td><em>sgmA</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR2393 </td><td>SGR2393 </td><td><em>amfR</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR2446–SGR2447 </td><td>SGR2446 </td><td>Tyrosinase co-factor protein (<em>melC1–2</em>) </td><td>s </td><td>2, 3 </td></tr><tr><td> </td><td>SGR2447 </td><td>Tyrosinase (<em>melC2–2</em>) </td><td>s </td><td> </td></tr><tr><td>SGR3307–SGR3306 </td><td>SGR3307 </td><td>Anti-sigma factor antagonist (<em>bldG</em>) </td><td>m </td><td> </td></tr><tr><td> </td><td>SGR3306 </td><td>Putative anti-sigma factor </td><td> </td><td> </td></tr><tr><td>SGR3340 </td><td>SGR3340 </td><td>Putative WhiB-family transcriptional regulator (<em>wblA</em>) </td><td>m, s </td><td>2 </td></tr><tr><td>SGR3902 </td><td>SGR3902 </td><td>Sporulation associated protein ORF1590 </td><td>m </td><td>2 </td></tr><tr><td>SGR4151 </td><td>SGR4151 </td><td><em>adsA</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR4809 </td><td>SGR4809 </td><td>Putative lantibiotic modifying enzyme </td><td>s </td><td> </td></tr><tr><td>SGR5762 </td><td>SGR5762 </td><td><em>sprB</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR5914 </td><td>SGR5914 </td><td><em>strU</em> </td><td>s </td><td>2 </td></tr><tr><td>SGR5922–SGR5923 </td><td>SGR5922 </td><td><em>stsB</em> </td><td>s </td><td> </td></tr><tr><td> </td><td>SGR5923 </td><td><em>stsA</em> </td><td>s </td><td> </td></tr><tr><td>SGR5931–SGR5932 </td><td>SGR5931 </td><td><em>strR</em> </td><td>s </td><td>1, 2 </td></tr><tr><td> </td><td>SGR5932 </td><td><em>aphD</em> </td><td>s </td><td> </td></tr><tr><td>SGR6071 </td><td>SGR6071 </td><td>Putative LAL-subfamily transcriptional regulator
(<em>AdBS4-orf1</em>) </td><td>s </td><td>1 </td></tr><tr><td>trn42 </td><td>trn42 </td><td><em>bldA</em> </td><td>m, s </td><td> </td></tr></tbody></table></div><div content-id="DSS010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo<sup>b</sup> </th><th>Identified
previously<sup>c</sup> </th></tr></thead><tbody><tr><td>SGR745 </td><td>SGR745 </td><td>Putative M23-family secreted peptidase </td><td>p </td><td> </td></tr><tr><td>SGR919–SGR916 </td><td>SGR919 </td><td>Hypothetical protein </td><td> </td><td>2 </td></tr><tr><td> </td><td>SGR918 </td><td>Putative subtilisin-like serine protease </td><td>p </td><td> </td></tr><tr><td> </td><td>SGR917 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td> </td><td>SGR916 </td><td>Hypothetical protein </td><td> </td><td> </td></tr><tr><td>SGR1063–SGR1059 </td><td>SGR1063 </td><td><em>rarA</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1062 </td><td><em>rarB</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1061 </td><td><em>rarC</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1060 </td><td><em>rarD</em> </td><td>m, s </td><td> </td></tr><tr><td> </td><td>SGR1059 </td><td><em>rarE</em> </td><td>m, s </td><td> </td></tr><tr><td>SGR2079 </td><td>SGR2079 </td><td>terpene cyclase (<em>gcoA</em>) </td><td>s </td><td>2 </td></tr><tr><td>SGR2095 </td><td>SGR2095 </td><td><em>sgmA</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR2393 </td><td>SGR2393 </td><td><em>amfR</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR2446–SGR2447 </td><td>SGR2446 </td><td>Tyrosinase co-factor protein (<em>melC1–2</em>) </td><td>s </td><td>2, 3 </td></tr><tr><td> </td><td>SGR2447 </td><td>Tyrosinase (<em>melC2–2</em>) </td><td>s </td><td> </td></tr><tr><td>SGR3307–SGR3306 </td><td>SGR3307 </td><td>Anti-sigma factor antagonist (<em>bldG</em>) </td><td>m </td><td> </td></tr><tr><td> </td><td>SGR3306 </td><td>Putative anti-sigma factor </td><td> </td><td> </td></tr><tr><td>SGR3340 </td><td>SGR3340 </td><td>Putative WhiB-family transcriptional regulator (<em>wblA</em>) </td><td>m, s </td><td>2 </td></tr><tr><td>SGR3902 </td><td>SGR3902 </td><td>Sporulation associated protein ORF1590 </td><td>m </td><td>2 </td></tr><tr><td>SGR4151 </td><td>SGR4151 </td><td><em>adsA</em> </td><td>m </td><td>1 </td></tr><tr><td>SGR4809 </td><td>SGR4809 </td><td>Putative lantibiotic modifying enzyme </td><td>s </td><td> </td></tr><tr><td>SGR5762 </td><td>SGR5762 </td><td><em>sprB</em> </td><td>p </td><td>1 </td></tr><tr><td>SGR5914 </td><td>SGR5914 </td><td><em>strU</em> </td><td>s </td><td>2 </td></tr><tr><td>SGR5922–SGR5923 </td><td>SGR5922 </td><td><em>stsB</em> </td><td>s </td><td> </td></tr><tr><td> </td><td>SGR5923 </td><td><em>stsA</em> </td><td>s </td><td> </td></tr><tr><td>SGR5931–SGR5932 </td><td>SGR5931 </td><td><em>strR</em> </td><td>s </td><td>1, 2 </td></tr><tr><td> </td><td>SGR5932 </td><td><em>aphD</em> </td><td>s </td><td> </td></tr><tr><td>SGR6071 </td><td>SGR6071 </td><td>Putative LAL-subfamily transcriptional regulator
(<em>AdBS4-orf1</em>) </td><td>s </td><td>1 </td></tr><tr><td>trn42 </td><td>trn42 </td><td><em>bldA</em> </td><td>m, s </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>An extract of genes involved in morphological differentiation, secondary metabolism, and extracellular protease. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> for full list. <sup>b</sup>m, involved in morphological differentiation; s, involved in secondary metabolism; p, extracellular protease. <sup>c</sup>1, ‘category-I’ target; 2, suggested by Hara <em>et al.</em><sup><a reveal-id="DSS010C5" data-open="DSS010C5" class="link link-ref link-reveal xref-bibr">5</a></sup>; 3, suggested by Akanuma <em>et al</em>.<sup><a reveal-id="DSS010C6" data-open="DSS010C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167292" target="_blank">View Large</a></div></div>                        <a id="7167293" scrollto-destination="7167293"></a>
<div content-id="DSS010TB3" class="table-modal"><div class="table-wrap"><div id="DSS010TB3" class="table-wrap-title" data-id="DSS010TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of genes that are probably repressed directly by AdpA in both liquid and solid cultures</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo </th></tr></thead><tbody><tr><td>SGR291 </td><td>SGR291 </td><td>Hyaluronidase </td><td> </td></tr><tr><td>SGR955 </td><td>SGR955 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR966–SGR960 </td><td>SGR966 </td><td>Putative squalene/phytoene synthase (<em>hopE</em>) </td><td> </td></tr><tr><td> </td><td>SGR965 </td><td>Putative squalene/phytoene synthase (<em>hopD</em>) </td><td> </td></tr><tr><td> </td><td>SGR964 </td><td>Putative squalene/phytoene dehydrogenase (<em>hopC</em>) </td><td> </td></tr><tr><td> </td><td>SGR963 </td><td>Putative polyprenyl diphosphate synthase (<em>hopB</em>) </td><td> </td></tr><tr><td> </td><td>SGR962 </td><td>Putative squalene-hopene cyclase (<em>hopA</em>) </td><td> </td></tr><tr><td> </td><td>SGR961 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR960 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR1311–SGR1309 </td><td>SGR1311 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1310 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1309 </td><td>Putative uricase </td><td> </td></tr><tr><td>SGR1869 </td><td>SGR1869 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR1903 </td><td>SGR1903 </td><td>Putative acylphosphatase </td><td>pm </td></tr><tr><td>SGR2045–SGR2046 </td><td>SGR2045 </td><td>Putative glycine cleavage system protein T </td><td>pm </td></tr><tr><td> </td><td>SGR2046 </td><td>Putative glycine cleavage system protein H </td><td>pm </td></tr><tr><td>SGR2195 </td><td>SGR2195 </td><td>Putative Ku70/Ku80 protein </td><td> </td></tr><tr><td>SGR2702 </td><td>SGR2702 </td><td>Putative tryptophanyl-tRNA synthetase </td><td>pm </td></tr><tr><td>SGR2910 </td><td>SGR2910 </td><td>Putative methylated-DNA–protein-cysteine <em>S</em>-methyltransferase </td><td> </td></tr><tr><td>SGR3022 </td><td>SGR3022 </td><td>Putative TetR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR3072 </td><td>SGR3072 </td><td>Putative iron dependent regulatory protein </td><td> </td></tr><tr><td>SGR3524–SGR3525 </td><td>SGR3524 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR3525 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3620 </td><td>SGR3620 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3646–SGR3645 </td><td>SGR3646 </td><td>Putative cytochrome D ubiquinol oxidase subunit I </td><td>pm </td></tr><tr><td> </td><td>SGR3645 </td><td>Putative cytochrome D ubiquinol oxidase subunit II </td><td>pm </td></tr><tr><td>SGR3675 </td><td>SGR3675 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3782 </td><td>SGR3782 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR3897 </td><td>SGR3897 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR3928–SGR3929 </td><td>SGR3928 </td><td>Putative Pit accessory protein </td><td>pm </td></tr><tr><td> </td><td>SGR3929 </td><td>Putative low-affinity inorganic phosphate transporter (<em>pitH</em>) </td><td>pm </td></tr><tr><td>SGR4276 </td><td>SGR4276 </td><td>RNA polymerase principal sigma factor (<em>hrdD</em>) </td><td> </td></tr><tr><td>SGR4380 </td><td>SGR4380 </td><td>Putative membrane protein </td><td> </td></tr><tr><td>SGR4383 </td><td>SGR4383 </td><td>Putative ribose-phosphate pyrophosphokinase </td><td>pm </td></tr><tr><td>SGR4455 </td><td>SGR4455 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4456 </td><td>SGR4456 </td><td>Putative RpiR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR4482 </td><td>SGR4482 </td><td>Putative acyl-CoA dehydrogenase </td><td>pm </td></tr><tr><td>SGR4489 </td><td>SGR4489 </td><td>Putative acyl-CoA hydrolase </td><td>pm </td></tr><tr><td>SGR4623 </td><td>SGR4623 </td><td>Putative ATP-dependent Clp protease adaptor protein ClpS </td><td> </td></tr><tr><td>SGR4652 </td><td>SGR4652 </td><td>Putative secreted protein </td><td> </td></tr><tr><td>SGR4803 </td><td>SGR4803 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4919 </td><td>SGR4919 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR4930 </td><td>SGR4930 </td><td>Putative nucleoside diphosphate kinase </td><td>pm </td></tr><tr><td>SGR4995 </td><td>SGR4995 </td><td>Putative 2-nitropropane dioxygenase </td><td> </td></tr><tr><td>SGR5104 </td><td>SGR5104 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR5469 </td><td>SGR5469 </td><td>Putative indol-3-glycerol phosphate synthase </td><td>pm </td></tr><tr><td>SGR5674 </td><td>SGR5674 </td><td>Putative molybdenum cofactor biosynthesis protein A </td><td> </td></tr><tr><td>SGR5721 </td><td>SGR5721 </td><td>Putative CTP synthetase </td><td>pm </td></tr><tr><td>SGR6046 </td><td>SGR6046 </td><td>Pyrimidine operon regulatory protein </td><td>pm </td></tr><tr><td>SGR6069–SGR6070 </td><td>SGR6069 </td><td>Putative hydrolase </td><td> </td></tr><tr><td> </td><td>SGR6070 </td><td>Putative amino oxidase </td><td> </td></tr><tr><td>SGR6240 </td><td>SGR6240 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR6353–SGR6354 </td><td>SGR6353 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR6354 </td><td>Hypothetical protein </td><td> </td></tr></tbody></table></div><div content-id="DSS010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo </th></tr></thead><tbody><tr><td>SGR291 </td><td>SGR291 </td><td>Hyaluronidase </td><td> </td></tr><tr><td>SGR955 </td><td>SGR955 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR966–SGR960 </td><td>SGR966 </td><td>Putative squalene/phytoene synthase (<em>hopE</em>) </td><td> </td></tr><tr><td> </td><td>SGR965 </td><td>Putative squalene/phytoene synthase (<em>hopD</em>) </td><td> </td></tr><tr><td> </td><td>SGR964 </td><td>Putative squalene/phytoene dehydrogenase (<em>hopC</em>) </td><td> </td></tr><tr><td> </td><td>SGR963 </td><td>Putative polyprenyl diphosphate synthase (<em>hopB</em>) </td><td> </td></tr><tr><td> </td><td>SGR962 </td><td>Putative squalene-hopene cyclase (<em>hopA</em>) </td><td> </td></tr><tr><td> </td><td>SGR961 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR960 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR1311–SGR1309 </td><td>SGR1311 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1310 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1309 </td><td>Putative uricase </td><td> </td></tr><tr><td>SGR1869 </td><td>SGR1869 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR1903 </td><td>SGR1903 </td><td>Putative acylphosphatase </td><td>pm </td></tr><tr><td>SGR2045–SGR2046 </td><td>SGR2045 </td><td>Putative glycine cleavage system protein T </td><td>pm </td></tr><tr><td> </td><td>SGR2046 </td><td>Putative glycine cleavage system protein H </td><td>pm </td></tr><tr><td>SGR2195 </td><td>SGR2195 </td><td>Putative Ku70/Ku80 protein </td><td> </td></tr><tr><td>SGR2702 </td><td>SGR2702 </td><td>Putative tryptophanyl-tRNA synthetase </td><td>pm </td></tr><tr><td>SGR2910 </td><td>SGR2910 </td><td>Putative methylated-DNA–protein-cysteine <em>S</em>-methyltransferase </td><td> </td></tr><tr><td>SGR3022 </td><td>SGR3022 </td><td>Putative TetR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR3072 </td><td>SGR3072 </td><td>Putative iron dependent regulatory protein </td><td> </td></tr><tr><td>SGR3524–SGR3525 </td><td>SGR3524 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR3525 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3620 </td><td>SGR3620 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3646–SGR3645 </td><td>SGR3646 </td><td>Putative cytochrome D ubiquinol oxidase subunit I </td><td>pm </td></tr><tr><td> </td><td>SGR3645 </td><td>Putative cytochrome D ubiquinol oxidase subunit II </td><td>pm </td></tr><tr><td>SGR3675 </td><td>SGR3675 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3782 </td><td>SGR3782 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR3897 </td><td>SGR3897 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR3928–SGR3929 </td><td>SGR3928 </td><td>Putative Pit accessory protein </td><td>pm </td></tr><tr><td> </td><td>SGR3929 </td><td>Putative low-affinity inorganic phosphate transporter (<em>pitH</em>) </td><td>pm </td></tr><tr><td>SGR4276 </td><td>SGR4276 </td><td>RNA polymerase principal sigma factor (<em>hrdD</em>) </td><td> </td></tr><tr><td>SGR4380 </td><td>SGR4380 </td><td>Putative membrane protein </td><td> </td></tr><tr><td>SGR4383 </td><td>SGR4383 </td><td>Putative ribose-phosphate pyrophosphokinase </td><td>pm </td></tr><tr><td>SGR4455 </td><td>SGR4455 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4456 </td><td>SGR4456 </td><td>Putative RpiR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR4482 </td><td>SGR4482 </td><td>Putative acyl-CoA dehydrogenase </td><td>pm </td></tr><tr><td>SGR4489 </td><td>SGR4489 </td><td>Putative acyl-CoA hydrolase </td><td>pm </td></tr><tr><td>SGR4623 </td><td>SGR4623 </td><td>Putative ATP-dependent Clp protease adaptor protein ClpS </td><td> </td></tr><tr><td>SGR4652 </td><td>SGR4652 </td><td>Putative secreted protein </td><td> </td></tr><tr><td>SGR4803 </td><td>SGR4803 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4919 </td><td>SGR4919 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR4930 </td><td>SGR4930 </td><td>Putative nucleoside diphosphate kinase </td><td>pm </td></tr><tr><td>SGR4995 </td><td>SGR4995 </td><td>Putative 2-nitropropane dioxygenase </td><td> </td></tr><tr><td>SGR5104 </td><td>SGR5104 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR5469 </td><td>SGR5469 </td><td>Putative indol-3-glycerol phosphate synthase </td><td>pm </td></tr><tr><td>SGR5674 </td><td>SGR5674 </td><td>Putative molybdenum cofactor biosynthesis protein A </td><td> </td></tr><tr><td>SGR5721 </td><td>SGR5721 </td><td>Putative CTP synthetase </td><td>pm </td></tr><tr><td>SGR6046 </td><td>SGR6046 </td><td>Pyrimidine operon regulatory protein </td><td>pm </td></tr><tr><td>SGR6069–SGR6070 </td><td>SGR6069 </td><td>Putative hydrolase </td><td> </td></tr><tr><td> </td><td>SGR6070 </td><td>Putative amino oxidase </td><td> </td></tr><tr><td>SGR6240 </td><td>SGR6240 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR6353–SGR6354 </td><td>SGR6353 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR6354 </td><td>Hypothetical protein </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>pm, presumably involved in primary metabolism.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167293" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS010TB3" class="table-wrap-title" data-id="DSS010TB3"><div class="label -label">Table 3.</div><div class="caption"><p>List of genes that are probably repressed directly by AdpA in both liquid and solid cultures</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo </th></tr></thead><tbody><tr><td>SGR291 </td><td>SGR291 </td><td>Hyaluronidase </td><td> </td></tr><tr><td>SGR955 </td><td>SGR955 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR966–SGR960 </td><td>SGR966 </td><td>Putative squalene/phytoene synthase (<em>hopE</em>) </td><td> </td></tr><tr><td> </td><td>SGR965 </td><td>Putative squalene/phytoene synthase (<em>hopD</em>) </td><td> </td></tr><tr><td> </td><td>SGR964 </td><td>Putative squalene/phytoene dehydrogenase (<em>hopC</em>) </td><td> </td></tr><tr><td> </td><td>SGR963 </td><td>Putative polyprenyl diphosphate synthase (<em>hopB</em>) </td><td> </td></tr><tr><td> </td><td>SGR962 </td><td>Putative squalene-hopene cyclase (<em>hopA</em>) </td><td> </td></tr><tr><td> </td><td>SGR961 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR960 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR1311–SGR1309 </td><td>SGR1311 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1310 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1309 </td><td>Putative uricase </td><td> </td></tr><tr><td>SGR1869 </td><td>SGR1869 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR1903 </td><td>SGR1903 </td><td>Putative acylphosphatase </td><td>pm </td></tr><tr><td>SGR2045–SGR2046 </td><td>SGR2045 </td><td>Putative glycine cleavage system protein T </td><td>pm </td></tr><tr><td> </td><td>SGR2046 </td><td>Putative glycine cleavage system protein H </td><td>pm </td></tr><tr><td>SGR2195 </td><td>SGR2195 </td><td>Putative Ku70/Ku80 protein </td><td> </td></tr><tr><td>SGR2702 </td><td>SGR2702 </td><td>Putative tryptophanyl-tRNA synthetase </td><td>pm </td></tr><tr><td>SGR2910 </td><td>SGR2910 </td><td>Putative methylated-DNA–protein-cysteine <em>S</em>-methyltransferase </td><td> </td></tr><tr><td>SGR3022 </td><td>SGR3022 </td><td>Putative TetR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR3072 </td><td>SGR3072 </td><td>Putative iron dependent regulatory protein </td><td> </td></tr><tr><td>SGR3524–SGR3525 </td><td>SGR3524 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR3525 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3620 </td><td>SGR3620 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3646–SGR3645 </td><td>SGR3646 </td><td>Putative cytochrome D ubiquinol oxidase subunit I </td><td>pm </td></tr><tr><td> </td><td>SGR3645 </td><td>Putative cytochrome D ubiquinol oxidase subunit II </td><td>pm </td></tr><tr><td>SGR3675 </td><td>SGR3675 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3782 </td><td>SGR3782 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR3897 </td><td>SGR3897 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR3928–SGR3929 </td><td>SGR3928 </td><td>Putative Pit accessory protein </td><td>pm </td></tr><tr><td> </td><td>SGR3929 </td><td>Putative low-affinity inorganic phosphate transporter (<em>pitH</em>) </td><td>pm </td></tr><tr><td>SGR4276 </td><td>SGR4276 </td><td>RNA polymerase principal sigma factor (<em>hrdD</em>) </td><td> </td></tr><tr><td>SGR4380 </td><td>SGR4380 </td><td>Putative membrane protein </td><td> </td></tr><tr><td>SGR4383 </td><td>SGR4383 </td><td>Putative ribose-phosphate pyrophosphokinase </td><td>pm </td></tr><tr><td>SGR4455 </td><td>SGR4455 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4456 </td><td>SGR4456 </td><td>Putative RpiR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR4482 </td><td>SGR4482 </td><td>Putative acyl-CoA dehydrogenase </td><td>pm </td></tr><tr><td>SGR4489 </td><td>SGR4489 </td><td>Putative acyl-CoA hydrolase </td><td>pm </td></tr><tr><td>SGR4623 </td><td>SGR4623 </td><td>Putative ATP-dependent Clp protease adaptor protein ClpS </td><td> </td></tr><tr><td>SGR4652 </td><td>SGR4652 </td><td>Putative secreted protein </td><td> </td></tr><tr><td>SGR4803 </td><td>SGR4803 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4919 </td><td>SGR4919 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR4930 </td><td>SGR4930 </td><td>Putative nucleoside diphosphate kinase </td><td>pm </td></tr><tr><td>SGR4995 </td><td>SGR4995 </td><td>Putative 2-nitropropane dioxygenase </td><td> </td></tr><tr><td>SGR5104 </td><td>SGR5104 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR5469 </td><td>SGR5469 </td><td>Putative indol-3-glycerol phosphate synthase </td><td>pm </td></tr><tr><td>SGR5674 </td><td>SGR5674 </td><td>Putative molybdenum cofactor biosynthesis protein A </td><td> </td></tr><tr><td>SGR5721 </td><td>SGR5721 </td><td>Putative CTP synthetase </td><td>pm </td></tr><tr><td>SGR6046 </td><td>SGR6046 </td><td>Pyrimidine operon regulatory protein </td><td>pm </td></tr><tr><td>SGR6069–SGR6070 </td><td>SGR6069 </td><td>Putative hydrolase </td><td> </td></tr><tr><td> </td><td>SGR6070 </td><td>Putative amino oxidase </td><td> </td></tr><tr><td>SGR6240 </td><td>SGR6240 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR6353–SGR6354 </td><td>SGR6353 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR6354 </td><td>Hypothetical protein </td><td> </td></tr></tbody></table></div><div content-id="DSS010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Transcriptional unit </th><th>ID </th><th>Description </th><th>Memo </th></tr></thead><tbody><tr><td>SGR291 </td><td>SGR291 </td><td>Hyaluronidase </td><td> </td></tr><tr><td>SGR955 </td><td>SGR955 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR966–SGR960 </td><td>SGR966 </td><td>Putative squalene/phytoene synthase (<em>hopE</em>) </td><td> </td></tr><tr><td> </td><td>SGR965 </td><td>Putative squalene/phytoene synthase (<em>hopD</em>) </td><td> </td></tr><tr><td> </td><td>SGR964 </td><td>Putative squalene/phytoene dehydrogenase (<em>hopC</em>) </td><td> </td></tr><tr><td> </td><td>SGR963 </td><td>Putative polyprenyl diphosphate synthase (<em>hopB</em>) </td><td> </td></tr><tr><td> </td><td>SGR962 </td><td>Putative squalene-hopene cyclase (<em>hopA</em>) </td><td> </td></tr><tr><td> </td><td>SGR961 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR960 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR1311–SGR1309 </td><td>SGR1311 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1310 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR1309 </td><td>Putative uricase </td><td> </td></tr><tr><td>SGR1869 </td><td>SGR1869 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR1903 </td><td>SGR1903 </td><td>Putative acylphosphatase </td><td>pm </td></tr><tr><td>SGR2045–SGR2046 </td><td>SGR2045 </td><td>Putative glycine cleavage system protein T </td><td>pm </td></tr><tr><td> </td><td>SGR2046 </td><td>Putative glycine cleavage system protein H </td><td>pm </td></tr><tr><td>SGR2195 </td><td>SGR2195 </td><td>Putative Ku70/Ku80 protein </td><td> </td></tr><tr><td>SGR2702 </td><td>SGR2702 </td><td>Putative tryptophanyl-tRNA synthetase </td><td>pm </td></tr><tr><td>SGR2910 </td><td>SGR2910 </td><td>Putative methylated-DNA–protein-cysteine <em>S</em>-methyltransferase </td><td> </td></tr><tr><td>SGR3022 </td><td>SGR3022 </td><td>Putative TetR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR3072 </td><td>SGR3072 </td><td>Putative iron dependent regulatory protein </td><td> </td></tr><tr><td>SGR3524–SGR3525 </td><td>SGR3524 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR3525 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3620 </td><td>SGR3620 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3646–SGR3645 </td><td>SGR3646 </td><td>Putative cytochrome D ubiquinol oxidase subunit I </td><td>pm </td></tr><tr><td> </td><td>SGR3645 </td><td>Putative cytochrome D ubiquinol oxidase subunit II </td><td>pm </td></tr><tr><td>SGR3675 </td><td>SGR3675 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR3782 </td><td>SGR3782 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR3897 </td><td>SGR3897 </td><td>Putative M23-family secreted peptidase </td><td> </td></tr><tr><td>SGR3928–SGR3929 </td><td>SGR3928 </td><td>Putative Pit accessory protein </td><td>pm </td></tr><tr><td> </td><td>SGR3929 </td><td>Putative low-affinity inorganic phosphate transporter (<em>pitH</em>) </td><td>pm </td></tr><tr><td>SGR4276 </td><td>SGR4276 </td><td>RNA polymerase principal sigma factor (<em>hrdD</em>) </td><td> </td></tr><tr><td>SGR4380 </td><td>SGR4380 </td><td>Putative membrane protein </td><td> </td></tr><tr><td>SGR4383 </td><td>SGR4383 </td><td>Putative ribose-phosphate pyrophosphokinase </td><td>pm </td></tr><tr><td>SGR4455 </td><td>SGR4455 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4456 </td><td>SGR4456 </td><td>Putative RpiR-family transcriptional regulator </td><td> </td></tr><tr><td>SGR4482 </td><td>SGR4482 </td><td>Putative acyl-CoA dehydrogenase </td><td>pm </td></tr><tr><td>SGR4489 </td><td>SGR4489 </td><td>Putative acyl-CoA hydrolase </td><td>pm </td></tr><tr><td>SGR4623 </td><td>SGR4623 </td><td>Putative ATP-dependent Clp protease adaptor protein ClpS </td><td> </td></tr><tr><td>SGR4652 </td><td>SGR4652 </td><td>Putative secreted protein </td><td> </td></tr><tr><td>SGR4803 </td><td>SGR4803 </td><td>Hypothetical protein </td><td> </td></tr><tr><td>SGR4919 </td><td>SGR4919 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR4930 </td><td>SGR4930 </td><td>Putative nucleoside diphosphate kinase </td><td>pm </td></tr><tr><td>SGR4995 </td><td>SGR4995 </td><td>Putative 2-nitropropane dioxygenase </td><td> </td></tr><tr><td>SGR5104 </td><td>SGR5104 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR5469 </td><td>SGR5469 </td><td>Putative indol-3-glycerol phosphate synthase </td><td>pm </td></tr><tr><td>SGR5674 </td><td>SGR5674 </td><td>Putative molybdenum cofactor biosynthesis protein A </td><td> </td></tr><tr><td>SGR5721 </td><td>SGR5721 </td><td>Putative CTP synthetase </td><td>pm </td></tr><tr><td>SGR6046 </td><td>SGR6046 </td><td>Pyrimidine operon regulatory protein </td><td>pm </td></tr><tr><td>SGR6069–SGR6070 </td><td>SGR6069 </td><td>Putative hydrolase </td><td> </td></tr><tr><td> </td><td>SGR6070 </td><td>Putative amino oxidase </td><td> </td></tr><tr><td>SGR6240 </td><td>SGR6240 </td><td>Conserved hypothetical protein </td><td> </td></tr><tr><td>SGR6353–SGR6354 </td><td>SGR6353 </td><td>Hypothetical protein </td><td> </td></tr><tr><td> </td><td>SGR6354 </td><td>Hypothetical protein </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>pm, presumably involved in primary metabolism.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7167293" target="_blank">View Large</a></div></div><p>Of the 14 ‘category-I’ AdpA-target transcriptional units, two genes (<em>sprA</em> and <em>sprD</em>) were not detected as genes activated by AdpA in the present DNA microarray analysis, probably because of technical limitations of the analysis. However, <em>in vivo</em> AdpA binding to their ‘category-I’ AdpA-binding sites was detected. In contrast, the DNA microarray analysis showed that the ‘category-I’ AdpA-target gene <em>sgiA</em><sup><a reveal-id="DSS010C4" data-open="DSS010C4" class="link link-ref link-reveal xref-bibr">4</a></sup> was activated by AdpA, but AdpA binding to its AdpA-binding site <em>in vivo</em> was not detected by ChIP/ChAP-seq analysis for an unknown reason. It should be noted that the <em>sgiA</em> locus has been suggested to have some high-order DNA structure. The putative high-order DNA structure of the <em>sgiA</em> locus may hamper the efficiency of ChIP/ChAP with N-His-AdpA. In summary, of the 14 ‘category-I’ AdpA-target transcriptional units, only 3 units were dropped from the candidate lists. Therefore, a majority of AdpA-target genes were listed in this study.</p>                            <h2 scrollto-destination=7167297 id="7167297" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, ChIP/ChAP-seq technology was used to take snapshots of distribution of AdpA across the chromosome in living <em>S. griseus</em> cells. In combination with transcriptome analysis, the results indicated that AdpA directly controls &gt;500 genes; AdpA regulon seems to be more extensive than previously suggested. Genes that are likely to be under the direct control of AdpA were also extensively catalogued (Tables <a reveal-id="DSS010TB2" data-open="DSS010TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSS010TB3" data-open="DSS010TB3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>), which should contribute to future studies of the regulation of secondary metabolism and morphological differentiation in <em>S. griseus</em>. In fact, <em>bldA</em> was identified as an AdpA-target gene by this study, revealing a unique positive feedback loop between two global regulators, AdpA and BldA, in <em>S. griseus</em>.<sup><a reveal-id="DSS010C13" data-open="DSS010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In addition to the identification of AdpA regulon, certain unique characteristics of AdpA regulation were revealed, as discussed separately below.</p>                            <h3 scrollto-destination=7167299 id="7167299" class="section-title" data-legacy-id=s4a>4.1. Numbers of <em>in vivo</em> AdpA-binding sites and transcriptional units belonging to the AdpA regulon</h3>
<p>Although the AdpA-binding consensus sequence (5′-TGGCSNGWWY-3′) has four alternative nucleotides and one free nucleotide, the ‘category-I’ AdpA-binding sites usually contain a few inconsistent nucleotides within the consensus sequence. Thus, the DNA-binding specificity of AdpA is much lower than that of other well-known regulators in bacteria. ChIP/ChAP analysis of 18-h liquid culture and ChAP analysis of 1-, 2-, and 3-day solid cultures indicated ∼1500 AdpA-binding sites. Approximately 35% of them were located on regions that appeared not to affect transcription (i.e. ‘coding’ and ‘intergenic’ regions). Furthermore, transcriptome analysis indicated that ∼60% of the AdpA-binding sites in ‘putative regulatory regions’ should not affect transcription. Taken together, ∼75% of the <em>in vivo</em> AdpA-binding sites seemed not to be involved in gene regulation, and only ∼25% of them were ‘apparently relevant’. We found 262 and 172 AdpA-binding sites that are apparently involved in transcriptional activation (217 transcriptional units, 342 genes) and repression (156 transcriptional units, 224 genes), respectively, in either liquid or solid culture. To confirm the AdpA dependency of these transcriptional units, the effect of mutation of AdpA-binding sites on transcription of each of the putative target genes should be examined. We previously revealed that the <em>bldK</em> operon is indirectly activated by AdpA, in spite of AdpA binding to the upstream region of <em>bldK</em>.<sup><a reveal-id="DSS010C18" data-open="DSS010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Therefore, it is possible that some of the putative AdpA-dependent transcriptional units described above are actually regulated indirectly by AdpA (see the legend for <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). It is also possible that many of them are directly regulated by AdpA, like the ‘category-I’ AdpA-target genes.</p><p>Recently, den Hengst <em>et al</em>.<sup><a reveal-id="DSS010C19" data-open="DSS010C19" class="link link-ref link-reveal xref-bibr">19</a></sup> used ChIP-chip technology to identify <em>in vivo</em> binding regions of BldD, which is a global transcriptional repressor for both morphological differentiation and secondary metabolism in <em>Streptomyces coelicolor</em> A3(2). They identified 167 putative BldD-binding sites across the chromosome. Transcriptome analysis with a <em>bldD</em> mutant and its parent strain confirmed that at least 29 of the 167 BldD-binding sites were relevant. Similarly, Molle <em>et al</em>.<sup><a reveal-id="DSS010C10" data-open="DSS010C10" class="link link-ref link-reveal xref-bibr">10</a></sup> identified 54 transcriptional units (121 genes) that are under the direct control of Spo0A, the master regulator for entry into sporulation in <em>Bacillus subtilis</em>. Our current study revealed that AdpA directly controls many more genes than these global transcriptional regulators for bacterial differentiation. To the best of our knowledge, the AdpA regulon seems to be the largest one in bacteria.</p>                            <h3 scrollto-destination=7167304 id="7167304" class="section-title" data-legacy-id=s4b>4.2. The strength of the AdpA–DNA interaction is not related to the biological significance of AdpA-binding sites</h3>
<p>Approximately 75% of the <em>in vivo</em> AdpA-binding sites apparently have no function in the regulation of gene expression. Similar results have been observed for <em>E. coli</em> RutR,<sup><a reveal-id="DSS010C20" data-open="DSS010C20" class="link link-ref link-reveal xref-bibr">20</a></sup><em>Bacillus subtilis</em> AbrB,<sup><a reveal-id="DSS010C21" data-open="DSS010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and <em>Streptomyces coelicolor</em> BldD;<sup><a reveal-id="DSS010C19" data-open="DSS010C19" class="link link-ref link-reveal xref-bibr">19</a></sup> binding of them to most of their <em>in vivo</em> binding sites did not apparently affect gene expression. We speculate that AdpA binding to such regions has no or little biological significance. ChIP-chip analysis by Grainger <em>et al</em>.<sup><a reveal-id="DSS010C22" data-open="DSS010C22" class="link link-ref link-reveal xref-bibr">22</a></sup> revealed that CRP in <em>E. coli</em> bound to 68 high-affinity sites, including 29 known CRP-dependent promoter regions, and interacted with thousands of weaker sites across the chromosome. They suggested that CRP should be considered as a chromosome-shaping protein contributing to the compaction of the chromosome (because DNA-bound CRP bends its target sharply), in addition to its function as a promoter-specific regulator. We cannot exclude the possibility that AdpA also functions as a chromosome-shaping protein in <em>S. griseus</em>. However, a striking difference between the DNA-binding profiles of CRP and AdpA concerns their DNA-binding strengths. As described above, CRP binds strongly to its target promoter regions and interacts weakly with ‘background’ regions. In contrast, AdpA binds with a similar strength to ‘apparently irrelevant’ regions similarly to ‘apparently relevant’ regions. Thus, the strength of the AdpA–DNA interaction is not related to the functional significance of particular AdpA-binding sites. The broad range of DNA-binding strengths to a variety of genuine target promoter regions, as well as ‘apparently irrelevant’ regions, is one of the distinctive properties of AdpA. We hypothesize that the position of the AdpA-binding site relative to the transcriptional start point, rather than the binding strength, is important for transcriptional regulation by AdpA. In fact, our previous studies have shown that AdpA activates 10 of 14 ‘category-I’ AdpA-target genes by binding to –50 to –70 regions in respect to their transcriptional start points.</p>                            <h3 scrollto-destination=7167309 id="7167309" class="section-title" data-legacy-id=s4c>4.3. Complexity of the DNA-binding properties of AdpA</h3>
<p>Regarding the selected 115 probable AdpA-binding sequences, we showed poor correlation of AdpA-binding strength between <em>in vitro</em> and <em>in vivo</em> conditions, while the sequences that were strongly bound by AdpA <em>in vitro</em> generally showed relatively high AdpA-binding strength <em>in vivo</em>. Furthermore, of the 304 <em>in vivo</em> AdpA-binding regions examined, 20 regions (6.6%) were not bound by AdpA <em>in vitro</em>, although they contain AdpA-binding consensus-like sequences. The ChIP-chip analyses of several bacterial global regulators, such as CtrA in <em>Caulobacter crescentus</em><sup><a reveal-id="DSS010C23" data-open="DSS010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and LexA and FNR in <em>E. coli</em>,<sup><a reveal-id="DSS010C24" data-open="DSS010C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS010C25" data-open="DSS010C25" class="link link-ref link-reveal xref-bibr">25</a></sup> have indicated that these regulators bind dozens of ‘non-canonical’ sites <em>in vivo</em>, which are not similar to their consensus binding sequences. Interestingly, these unconventional targets were not bound by the respective regulators <em>in vitro</em>. Wade <em>et al</em>.<sup><a reveal-id="DSS010C24" data-open="DSS010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> suggested that binding of LexA to these unconventional targets requires additional factors, such as cooperative interaction with other proteins and/or unusual DNA conformation (e.g. bent or supercoiled DNA). Thus, AdpA may also require similar additional factors to bind some of the AdpA-binding sites detected by ChIP/ChAP-seq. Such additional factors may explain the poor correlation of AdpA-binding strength between <em>in vitro</em> and <em>in vivo</em> conditions. We postulated that AdpA might frequently interact with other DNA-binding proteins to regulate target genes. This assumption is also supported by the observation that the expression profiles of apparently AdpA-dependent genes were very different between liquid and solid cultures, despite the similar AdpA-binding profiles in both cultures. Thus, <em>in vivo</em>, AdpA seems to exert its regulatory functions cooperatively with other proteins in many cases. Detailed analysis of cooperative regulation between AdpA and other regulatory proteins will form a very important part of our future studies.</p><p>Importantly, the degree of similarity to the AdpA-binding consensus sequence was not necessarily consistent with the strength of the AdpA–DNA interaction of each AdpA-binding site (both <em>in vivo</em> and <em>in vitro</em>), although the sequences that were strongly bound by AdpA <em>in vitro</em> showed a relatively high similarity to the consensus sequence. This is a striking contrast to the binding mode of LexA to its conventional targets, in which the strength of the LexA–DNA interaction can be well approximated by the degree of similarity to the LexA-binding consensus sequence.<sup><a reveal-id="DSS010C24" data-open="DSS010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> However, there are analytical limitations that should be taken into consideration, as well as our incomplete understanding of the molecular mechanism of DNA recognition by AdpA. First, in this analysis, type-I AdpA binding, in which both subunits of the AdpA dimer bind each of the two divergent AdpA-binding sequences, was not considered, because we could not conclusively identify type-I-binding sites without DNase I footprinting. When AdpA binds to a type-I site, another AdpA-binding sequence in the vicinity should affect the strength of the AdpA–DNA interaction. Therefore, regarding genuine type-I sites, we could not correctly estimate the relationship between the strength of the AdpA–DNA interaction and the degree of similarity to the consensus sequence. Secondly, it is possible that some DNA fragments used for the competitive EMSA had two (or more) AdpA-binding sites. Our ChIP/ChAP-seq analysis clearly detected two peaks that are separated by 200 bp in the upstream region of <em>sgmA</em> (Fig. <a reveal-id="DSS010F2" data-open="DSS010F2" class="link link-reveal link-table xref-fig">2</a>B). However, we failed to discriminate two AdpA-binding sites<sup><a reveal-id="DSS010C13" data-open="DSS010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> that are separated by 50 bp in the upstream region of <em>bldA</em> (data not shown). When AdpA binds to two (or more) sites in a DNA fragment, it may lead to overestimation of the binding strength of the identified AdpA-binding sequence. Finally, in some cases, the additional eight nucleotides 3′ to the consensus sequence may contribute to the DNA-binding strength more significantly than expected. AdpA, which belongs to the AraC/XylS family, has two helix-turn-helix DNA-binding motifs. Based on the DNA-binding characteristics of some AraC/XylS family regulators, such as AraC, XylS, and SoxS, we proposed that the helix-turn-helix at the N-terminal side (HTH-1) recognizes the relatively highly conserved TGGCS sequence at the 5′-end of the consensus sequence.<sup><a reveal-id="DSS010C2" data-open="DSS010C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The other helix-turn-helix at the C-terminal side (HTH-2) recognizes the other nucleotides in the consensus sequence with the next eight nucleotides.<sup><a reveal-id="DSS010C2" data-open="DSS010C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Although nucleotide recognition by HTH-2 seems to be less important than that by HTH-1, it may significantly affect the binding strength in some cases.</p>                            <h3 scrollto-destination=7167316 id="7167316" class="section-title" data-legacy-id=s4d>4.4. AdpA as a transcriptional repressor</h3>
<p>It was unexpected that AdpA could directly repress more than 100 transcriptional units in ether liquid or solid culture. Our previous study indicated that AdpA represses transcription of its own gene by binding to three sites in the 5′ upstream region of <em>adpA</em>, probably forming a loop structure of the DNA.<sup><a reveal-id="DSS010C16" data-open="DSS010C16" class="link link-ref link-reveal xref-bibr">16</a></sup> However, ChIP/ChAP-seq analysis detected only one AdpA-binding site in the ‘putative regulatory region’ of most (&gt;90%) of the transcriptional units that were apparently repressed by AdpA. In such cases, AdpA may bind to the −35 and/or −10 sequences to hamper RNA polymerase from binding to them to initiate transcription. Interestingly, several genes that are presumably involved in primary metabolism are included in the list of possible genes directly repressed by AdpA in both liquid and solid cultures (Table <a reveal-id="DSS010TB3" data-open="DSS010TB3" class="link link-reveal link-table xref-fig">3</a>). To reveal a new aspect of global gene regulation by AdpA, further detailed analyses of the repressor function of AdpA is necessary.</p>                            <h3 scrollto-destination=7167324 id="7167324" class="section-title" data-legacy-id=s4e>4.5. Biological significance of the low DNA-binding specificity of AdpA</h3>
<p>Although bacterial genomes are packaged by histone-like proteins into a nucleoid structure, this structure is not analogous to eukaryotic chromatin; bacterial genomes are permissive to transcription factor binding.<sup><a reveal-id="DSS010C21" data-open="DSS010C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS010C22" data-open="DSS010C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSS010C24" data-open="DSS010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Wade <em>et al</em>.<sup><a reveal-id="DSS010C24" data-open="DSS010C24" class="link link-ref link-reveal xref-bibr">24</a></sup> proposed that bacteria must evolve transcription factors with high DNA-binding specificity because of equal accessibility of the genomes. However, this is obviously not applicable to AdpA; the DNA-binding specificity of AdpA is very low, and AdpA can bind to &gt;1500 sites (including many ‘apparently irrelevant’ sites) across the chromosome. AdpA orthologues are highly conserved (more than 90% amino acid similarity) among many <em>Streptomyces</em> species, and their essential roles in morphological differentiation and/or secondary metabolism have been revealed in various <em>Streptomyces</em> strains.<sup><a reveal-id="DSS010C13" data-open="DSS010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Surprisingly, the two helix-turn-helix DNA-binding motifs are completely identical among these proteins, indicating that they have the same DNA-binding specificity. Thus, the low DNA-binding specificity of AdpA has been maintained during evolution, suggesting that this atypical characteristic of AdpA as a bacterial transcription factor should be very important for the biology of <em>Streptomyces</em>. There could be some advantages conferred by the low DNA-binding specificity of AdpA for <em>Streptomyces</em>. First, the low DNA-binding specificity of AdpA enables binding to many sites on the chromosome, which facilitates regulation of many genes. For transition from vegetative growth to reproductive (differentiating) growth, including secondary metabolite production, a drastic change of gene expression is undoubtedly necessary. Therefore, the low DNA-binding specificity seems to be advantageous to AdpA as a pivotal regulator for this process. Secondly, the low DNA-binding specificity of AdpA may generate evolutional flexibility of the regulatory network. It is thought that <em>Streptomyces</em> achieves the productivity of a wide variety of secondary metabolites by acquiring foreign biosynthetic enzyme genes through horizontal gene transfer.<sup><a reveal-id="DSS010C26" data-open="DSS010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The low DNA-binding specificity of AdpA may increase the occurrence of an AdpA-binding site(s) in the regulatory region of newly acquired genes, which may increase the chance that AdpA regulates them. Thus, the low DNA-binding specificity of AdpA may enable frequent reconstruction of local gene regulation. This may be advantageous to the evolution of <em>Streptomyces</em>.</p>                            <h3 scrollto-destination=7167332 id="7167332" class="section-title" data-legacy-id=s4f>4.6. The A-factor regulatory system in <em>Streptomyces griseus</em></h3>
<p><em>Streptomyces griseus</em> seems to have evolved the highly integrated A-factor regulatory cascade, and the function of AdpA as a global regulator for morphological differentiation and secondary metabolism may be much more significant in <em>S. griseus</em> compared with other <em>Streptomyces</em> species. In the A-factor regulatory system, the hormonal signal triggers the production of only one transcriptional regulator, AdpA, which directly regulates the transcription of more than 500 genes. This ‘highly converged’ regulatory system seems to represent an example of sophisticated global gene regulation in bacteria. This system has the advantage of being able to switch the expression profiles of many genes in a relatively short period. We think that this system may reflect a fundamental aspect of differentiation of <em>Streptomyces</em>.</p>                            <h2 scrollto-destination=7167340 id="7167340" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank N. Nakata for technical assistance with the competitive EMSA.</p>                            <h2 scrollto-destination=7167345 id="7167345" class="backsection-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a> www.dnaresearch.oxfordjournals.org.</p>                            <h2 scrollto-destination=7167352 id="7167352" class="backsection-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Targeted Proteins Research Program (TPRP) of the Ministry of Education, Culture, Sports, Science, and Technology of Japan (MEXT) (grant number A2); and a Funding Program for Next Generation World-Leading Researchers from the Bureau of Science, Technology, and Innovation Policy, Cabinet Office, Government of Japan (grant number GS006).</p>                            <h2 scrollto-destination=7167358 id="7167358" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kameyama</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Onaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">The A-factor regulatory cascade leading to streptomycin biosynthesis in <em>Streptomyces griseus</em>: identification of a target gene of the A-factor receptor</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">102</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20A-factor%20regulatory%20cascade%20leading%20to%20streptomycin%20biosynthesis%20in%20Streptomyces%20griseus%3A%20identification%20of%20a%20target%20gene%20of%20the%20A-factor%20receptor&amp;author=Y.%20Ohnishi&amp;author=S.%20Kameyama&amp;author=H.%20Onaka&amp;author=S.%20Horinouchi&amp;publication_year=1999&amp;journal=Mol.%20Microbiol.&amp;volume=34&amp;pages=102-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01579.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1999.01579.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1999.01579.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10540289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Tomono</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">AdpA, a central transcriptional regulator in the A-factor regulatory cascade that leads to morphological development and secondary metabolism in <em>Streptomyces griseus</em></div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">431</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AdpA%2C%20a%20central%20transcriptional%20regulator%20in%20the%20A-factor%20regulatory%20cascade%20that%20leads%20to%20morphological%20development%20and%20secondary%20metabolism%20in%20Streptomyces%20griseus&amp;author=Y.%20Ohnishi&amp;author=H.%20Yamazaki&amp;author=J.Y.%20Kato&amp;author=A.%20Tomono&amp;author=S.%20Horinouchi&amp;publication_year=2005&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=69&amp;pages=431-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.69.431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.69.431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.69.431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15784968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ishikawa</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genome sequence of the streptomycin-producing microorganism <em>Streptomyces griseus</em> IFO 13350</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2008</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">4050</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20streptomycin-producing%20microorganism%20Streptomyces%20griseus%20IFO%2013350&amp;author=Y.%20Ohnishi&amp;author=J.%20Ishikawa&amp;author=H.%20Hara&amp;publication_year=2008&amp;journal=J.%20Bacteriol.&amp;volume=190&amp;pages=4050-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00204-08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00204-08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00204-08"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18375553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hirano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Control of the <em>Streptomyces</em> subtilisin inhibitor gene by AdpA in the A-factor regulatory cascade in <em>Streptomyces griseus</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">6207</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20the%20Streptomyces%20subtilisin%20inhibitor%20gene%20by%20AdpA%20in%20the%20A-factor%20regulatory%20cascade%20in%20Streptomyces%20griseus&amp;author=S.%20Hirano&amp;author=J.Y.%20Kato&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=6207-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00662-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00662-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00662-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16923887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DNA microarray analysis of global gene regulation by A-factor in <em>Streptomyces griseus</em></div>, <div class="source">Microbiology</div> , <div class="year">2009</div>, vol. <div class="volume">155</div> (pg. <div class="fpage">2197</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20microarray%20analysis%20of%20global%20gene%20regulation%20by%20A-factor%20in%20Streptomyces%20griseus&amp;author=H.%20Hara&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2009&amp;journal=Microbiology&amp;volume=155&amp;pages=2197-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.027862-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.027862-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.027862-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19389771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akanuma</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Hara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Dynamic changes in the extracellular proteome caused by absence of a pleiotropic regulator AdpA in <em>Streptomyces griseus</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2009</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">898</div>-<div class="lpage">912</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20changes%20in%20the%20extracellular%20proteome%20caused%20by%20absence%20of%20a%20pleiotropic%20regulator%20AdpA%20in%20Streptomyces%20griseus&amp;author=G.%20Akanuma&amp;author=H.%20Hara&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2009&amp;journal=Mol.%20Microbiol.&amp;volume=73&amp;pages=898-912" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2009.73.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2009.73.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2009.73.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19678896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamazaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tomono</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">DNA-binding specificity of AdpA, a transcriptional activator in the A-factor regulatory cascade in Streptomyces griseus</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">555</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA-binding%20specificity%20of%20AdpA%2C%20a%20transcriptional%20activator%20in%20the%20A-factor%20regulatory%20cascade%20in%20Streptomyces%20griseus&amp;author=H.%20Yamazaki&amp;author=A.%20Tomono&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=53&amp;pages=555-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2004.53.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2004.53.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2004.53.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15228534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kolb</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Buc</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Garges</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Adhya</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Transcriptional regulation by cAMP and its receptor protein</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="year">1993</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">749</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20regulation%20by%20cAMP%20and%20its%20receptor%20protein&amp;author=A.%20Kolb&amp;author=S.%20Busby&amp;author=H.%20Buc&amp;author=S.%20Garges&amp;author=S.%20Adhya&amp;publication_year=1993&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=62&amp;pages=749-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.62.070193.003533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.62.070193.003533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.62.070193.003533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8394684" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walker</div> <div class="given-names">G.C.</div></div>. </div><div class="article-title">Mutagenesis and inducible responses to deoxyribonucleic acid damage in <em>Escherichia coli</em></div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1984</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">60</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutagenesis%20and%20inducible%20responses%20to%20deoxyribonucleic%20acid%20damage%20in%20Escherichia%20coli&amp;author=G.C.%20Walker&amp;publication_year=1984&amp;journal=Microbiol.%20Rev.&amp;volume=48&amp;pages=60-93" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6371470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Molle</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fujita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">S.T.</div></div>, et al. </div><div class="article-title">The Spo0A regulon of <em>Bacillus subtilis</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2003</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">1683</div>-<div class="lpage">701</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Spo0A%20regulon%20of%20Bacillus%20subtilis&amp;author=V.%20Molle&amp;author=M.%20Fujita&amp;author=S.T.%20Jensen&amp;publication_year=2003&amp;journal=Mol.%20Microbiol.&amp;volume=50&amp;pages=1683-701" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2003.03818.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2003.03818.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2003.03818.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14651647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ogura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yoshimura</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Distribution of stable DnaA-binding sites on the <em>Bacillus subtilis</em> genome detected using a modified ChIP-chip method</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">155</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Distribution%20of%20stable%20DnaA-binding%20sites%20on%20the%20Bacillus%20subtilis%20genome%20detected%20using%20a%20modified%20ChIP-chip%20method&amp;author=S.%20Ishikawa&amp;author=Y.%20Ogura&amp;author=M.%20Yoshimura&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=155-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17932079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Park</div> <div class="given-names">P. J.</div></div>. </div><div class="article-title">ChIP-seq: advantages and challenges of a maturing technology</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">669</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ChIP-seq%3A%20advantages%20and%20challenges%20of%20a%20maturing%20technology&amp;author=P.%20J.%20Park&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=669-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2641" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2641" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2641"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19736561" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Strict regulation of morphological differentiation and secondary metabolism by a positive feedback loop between two global regulators AdpA and BldA in <em>Streptomyces griseus</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2011</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">1607</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strict%20regulation%20of%20morphological%20differentiation%20and%20secondary%20metabolism%20by%20a%20positive%20feedback%20loop%20between%20two%20global%20regulators%20AdpA%20and%20BldA%20in%20Streptomyces%20griseus&amp;author=A.%20Higo&amp;author=S.%20Horinouchi&amp;author=Y.%20Ohnishi&amp;publication_year=2011&amp;journal=Mol.%20Microbiol.&amp;volume=81&amp;pages=1607-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2011.07795.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2011.07795.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2011.07795.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21883521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R25</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ji</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Myers</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">An integrated software system for analyzing ChIP-chip and ChIP-seq data</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1293</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20software%20system%20for%20analyzing%20ChIP-chip%20and%20ChIP-seq%20data&amp;author=H.%20Ji&amp;author=H.%20Jiang&amp;author=W.%20Ma&amp;author=D.S.%20Johnson&amp;author=R.M.%20Myers&amp;author=W.H.%20Wong&amp;publication_year=2008&amp;journal=Nat.%20Biotechnol.&amp;volume=26&amp;pages=1293-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1505" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1505" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1505"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kato</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Autorepression of AdpA of the AraC/XylS family, a key transcriptional activator in the A-factor regulatory cascade in <em>Streptomyces griseus</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">350</div> (pg. <div class="fpage">12</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autorepression%20of%20AdpA%20of%20the%20AraC%2FXylS%20family%2C%20a%20key%20transcriptional%20activator%20in%20the%20A-factor%20regulatory%20cascade%20in%20Streptomyces%20griseus&amp;author=J.Y.%20Kato&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2005&amp;journal=J.%20Mol.%20Biol.&amp;volume=350&amp;pages=12-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.04.058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.04.058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.04.058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15907934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Gribskov</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Methods and statistics for combining motif match scores</div>, <div class="source">J. Comput. Biol.</div> , <div class="year">1998</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">211</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20and%20statistics%20for%20combining%20motif%20match%20scores&amp;author=T.L.%20Bailey&amp;author=M.%20Gribskov&amp;publication_year=1998&amp;journal=J.%20Comput.%20Biol.&amp;volume=5&amp;pages=211-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.1998.5.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.1998.5.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.1998.5.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9672829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akanuma</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ueki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ishizuka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Control of aerial mycelium formation by the BldK oligopeptide ABC transporter in <em>Streptomyces griseus</em></div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="year">2011</div>, vol. <div class="volume">315</div> (pg. <div class="fpage">54</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20aerial%20mycelium%20formation%20by%20the%20BldK%20oligopeptide%20ABC%20transporter%20in%20Streptomyces%20griseus&amp;author=G.%20Akanuma&amp;author=M.%20Ueki&amp;author=M.%20Ishizuka&amp;author=Y.%20Ohnishi&amp;author=S.%20Horinouchi&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=315&amp;pages=54-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2011.315.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2011.315.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2011.315.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21175743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">den Hengst</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Tran</div> <div class="given-names">N.T.</div></div>,  <div class="name"><div class="surname">Bibb</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Chandra</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Leskiw</div> <div class="given-names">B.K.</div></div>,  <div class="name"><div class="surname">Buttner</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Genes essential for morphological development and antibiotic production in <em>Streptomyces coelicolor</em> are targets of BldD during vegetative growth</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">361</div>-<div class="lpage">79</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20essential%20for%20morphological%20development%20and%20antibiotic%20production%20in%20Streptomyces%20coelicolor%20are%20targets%20of%20BldD%20during%20vegetative%20growth&amp;author=C.D.%20den%20Hengst&amp;author=N.T.%20Tran&amp;author=M.J.%20Bibb&amp;author=G.%20Chandra&amp;author=B.K.%20Leskiw&amp;author=M.J.%20Buttner&amp;publication_year=2010&amp;journal=Mol.%20Microbiol.&amp;volume=78&amp;pages=361-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2010.78.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2010.78.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2010.78.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979333" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ishihama</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Grainger</div> <div class="given-names">D.C.</div></div>. </div><div class="article-title">The <em>Escherichia coli</em> RutR transcription factor binds at targets within genes as well as intergenic regions</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3950</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Escherichia%20coli%20RutR%20transcription%20factor%20binds%20at%20targets%20within%20genes%20as%20well%20as%20intergenic%20regions&amp;author=T.%20Shimada&amp;author=A.%20Ishihama&amp;author=S.J.%20Busby&amp;author=D.C.%20Grainger&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=3950-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn339"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18515344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chumsakul</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Oshima</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome-wide binding profiles of the <em>Bacillus subtilis</em> transition state regulator AbrB and its homolog Abh reveals their interactive role in transcriptional regulation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2011</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">414</div>-<div class="lpage">28</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20binding%20profiles%20of%20the%20Bacillus%20subtilis%20transition%20state%20regulator%20AbrB%20and%20its%20homolog%20Abh%20reveals%20their%20interactive%20role%20in%20transcriptional%20regulation&amp;author=O.%20Chumsakul&amp;author=H.%20Takahashi&amp;author=T.%20Oshima&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=414-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq780" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq780" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq780"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20817675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grainger</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Hurd</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Holdstock</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Studies of the distribution of <em>Escherichia coli</em> cAMP-receptor protein and RNA polymerase along the <em>E. coli</em> chromosome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2005</div>, vol. <div class="volume">102</div> (pg. <div class="fpage">17693</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20the%20distribution%20of%20Escherichia%20coli%20cAMP-receptor%20protein%20and%20RNA%20polymerase%20along%20the%20E.%20coli%20chromosome&amp;author=D.C.%20Grainger&amp;author=D.%20Hurd&amp;author=M.%20Harrison&amp;author=J.%20Holdstock&amp;author=S.J.%20Busby&amp;publication_year=2005&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=17693-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0506687102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0506687102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0506687102"> </span></div> </div></div></div></div></div><div content-id="DSS010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laub</div> <div class="given-names">M.T.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">McAdams</div> <div class="given-names">H.H.</div></div>. </div><div class="article-title">Genes directly controlled by CtrA, a master regulator of the <em>Caulobacter</em> cell cycle</div>, <div class="source">, <em>Proc. Natl Acad. Sci. USA</em></div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">4632</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20directly%20controlled%20by%20CtrA%2C%20a%20master%20regulator%20of%20the%20Caulobacter%20cell%20cycle&amp;author=M.T.%20Laub&amp;author=S.L.%20Chen&amp;author=L.%20Shapiro&amp;author=H.H.%20McAdams&amp;publication_year=2002&amp;journal=%2C%20Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=4632-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.062065699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.062065699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.062065699"> </span></div> </div></div></div></div></div><div content-id="DSS010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wade</div> <div class="given-names">J.T.</div></div>,  <div class="name"><div class="surname">Reppas</div> <div class="given-names">N.B.</div></div>,  <div class="name"><div class="surname">Church</div> <div class="given-names">G.M.</div></div>,  <div class="name"><div class="surname">Struhl</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genomic analysis of LexA binding reveals the permissive nature of the <em>Escherichia coli</em> genome and identifies unconventional target sites</div>, <div class="source">Genes Dev.</div> , <div class="year">2005</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">2619</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20LexA%20binding%20reveals%20the%20permissive%20nature%20of%20the%20Escherichia%20coli%20genome%20and%20identifies%20unconventional%20target%20sites&amp;author=J.T.%20Wade&amp;author=N.B.%20Reppas&amp;author=G.M.%20Church&amp;author=K.%20Struhl&amp;publication_year=2005&amp;journal=Genes%20Dev.&amp;volume=19&amp;pages=2619-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1355605" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1355605" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1355605"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16264194" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grainger</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Aiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hurd</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Browning</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Busby</div> <div class="given-names">S.J.</div></div>. </div><div class="article-title">Transcription factor distribution in <em>Escherichia coli</em>: studies with FNR protein</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">269</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20factor%20distribution%20in%20Escherichia%20coli%3A%20studies%20with%20FNR%20protein&amp;author=D.C.%20Grainger&amp;author=H.%20Aiba&amp;author=D.%20Hurd&amp;author=D.F.%20Browning&amp;author=S.J.%20Busby&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=269-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl1023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl1023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl1023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17164287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS010C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chater</div> <div class="given-names">K.F.</div></div>,  <div class="name"><div class="surname">Biró</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Palmer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Schrempf</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The complex extracellular biology of <em>Streptomyces</em></div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="year">2010</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">171</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20extracellular%20biology%20of%20Streptomyces&amp;author=K.F.%20Chater&amp;author=S.%20Bir%C3%B3&amp;author=K.J.%20Lee&amp;author=T.%20Palmer&amp;author=H.%20Schrempf&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=34&amp;pages=171-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2009.00206.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2009.00206.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2009.00206.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20088961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=475336&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss010/2/dss010-dss010supp.pdf?Expires=2147483647&amp;Signature=d-P0Yc5J~2UjSqC7tLvlp3stiXrecqh~cCkx7x7eI7~YVYDoAWKAT-GZdwCLeHeoH2qmJWltPE-vOy-dWCkQmYse0OEV4zEcEZFOHDaM~6nML9Wpfw175fyjJ27kuNozbSqjaiaS8xBrt0WqTmPOgJ8cI5sIrcnF6n7X3Pe4dfEEG72IHw7srpSSr7uo4ivO-zqig~mQAnyQFlrI8jSIZuEqcTob7yIucMV0moF-Fqp6~8cUaN1Y-2EWNw95Wn6A2vG208k0ni7RzeOKlQtw3grZZCI3Gjufvz6Tv-MPNCA7eX1Yt33kOyYevx~7OjuiSCn~xQGVrN03q3GMxLKkqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">385</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">45</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="475336"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f3%2f259%2f475336">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000305723200005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (45)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F3%2F259%2F475336" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/475336'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide Distribution of AdpA, a Global Regulator for Secondary Metabolism and Morphological Differentiation in Streptomyces,
Revealed the Extent and Complexity of the AdpA Regulatory Network" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f259%2f475336">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

