

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/6/413/2462440","@type":"ScholarlyArticle","name":"Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance","datePublished":"2015-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2015-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv023","keywords":["soil microbiome","metagenome","recalcitrant aromatic compounds","phage","robustness"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/6/413/2462440","author":[{"name":"Kato, Hiromi","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Mori, Hiroshi","affiliation":"2  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  2-12-1 Ookayama, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Maruyama, Fumito","affiliation":"3  Graduate School of Medicine ,  Kyoto University ,  Kyoto 606-8501 ,  Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"4  Center for Information Biology ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"},{"name":"Oshima, Kenshiro","affiliation":"5  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa 277-8561 ,  Japan","@type":"Person"},{"name":"Endo, Ryo","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Fuchu, Genki","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Miyakoshi, Masatoshi","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Dozono, Ayumi","affiliation":"2  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  2-12-1 Ookayama, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Ohtsubo, Yoshiyuki","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Nagata, Yuji","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"5  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa 277-8561 ,  Japan","@type":"Person"},{"name":"Fujiyama, Asao","affiliation":"4  Center for Information Biology ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan    6  Principles of Informatics Research Division ,  National Institute of Informatics ,  Hitotsubashi, Tokyo 101-8430 ,  Japan","@type":"Person"},{"name":"Kurokawa, Ken","affiliation":"2  Department of Biological Information, Graduate School of Bioscience and Biotechnology ,  Tokyo Institute of Technology ,  2-12-1 Ookayama, Tokyo 152-8550 ,  Japan    7  Earth-Life Science Institute ,  Tokyo Institute of Technology ,  2-12-1 Ookayama, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Tsuda, Masataka","affiliation":"1  Graduate School of Life Sciences ,  Tohoku University ,  2-1-1 Katahira, Sendai 980-8577 ,  Japan","@type":"Person"}],"description":"Abstract.  Soil microbial communities have great potential for bioremediation of recalcitrant aromatic compounds. However, it is unclear which taxa and genes in","pageStart":"413","pageEnd":"424","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance" />
    <meta property="og:description" content="Abstract.  Soil microbial communities have great potential for bioremediation of recalcitrant aromatic compounds. However, it is unclear which taxa and genes in" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/6/413/2462440" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-10-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kato, Hiromi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Mori, Hiroshi" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Maruyama, Fumito" /><meta name="citation_author_institution" content="Graduate School of Medicine, Kyoto University, Kyoto 606-8501, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="Center for Information Biology, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author" content="Oshima, Kenshiro" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8561, Japan" /><meta name="citation_author" content="Endo, Ryo" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Fuchu, Genki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Miyakoshi, Masatoshi" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Dozono, Ayumi" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Ohtsubo, Yoshiyuki" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Nagata, Yuji" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa 277-8561, Japan" /><meta name="citation_author" content="Fujiyama, Asao" /><meta name="citation_author_institution" content="Center for Information Biology, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author_institution" content="Principles of Informatics Research Division, National Institute of Informatics, Hitotsubashi, Tokyo 101-8430, Japan" /><meta name="citation_author" content="Kurokawa, Ken" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Tokyo 152-8550, Japan" /><meta name="citation_author_institution" content="Earth-Life Science Institute, Tokyo Institute of Technology, 2-12-1 Ookayama, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Tsuda, Masataka" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Sendai 980-8577, Japan" /><meta name="citation_title" content="Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance" /><meta name="citation_firstpage" content="413" /><meta name="citation_lastpage" content="424" /><meta name="citation_doi" content="10.1093/dnares/dsv023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2015/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Fundamentals of microbial community resistance and resilience;  citation_journal_title=Front. Microbiol.;  citation_year=2012;  citation_volume=3;  citation_pages=417" /><meta name="citation_reference" content="citation_title=Resistance, resilience, and redundancy in microbial communities;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_issue=Suppl. 1;  citation_pages=11512-19; " /><meta name="citation_reference" content="citation_title=Structure, function and diversity of the healthy human microbiome; citation_author=The Human Microbiome Project Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=486;  citation_pages=207-14; " /><meta name="citation_reference" content="citation_title=The Earth Microbiome project: successes and aspirations;  citation_journal_title=BMC Biol.;  citation_year=2014;  citation_volume=12;  citation_pages=69" /><meta name="citation_reference" content="citation_title=Time series community genomics analysis reveals rapid shifts in bacterial species, strains, and phage during infant gut colonization;  citation_journal_title=Genome Res.;  citation_year=2013;  citation_volume=23;  citation_pages=111-20; " /><meta name="citation_reference" content="citation_title=Succession of microbial consortia in the developing infant gut microbiome;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_issue=Suppl. 1;  citation_pages=4578-85; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis of a permafrost microbial community reveals a rapid response to thaw;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=480;  citation_pages=368-71; " /><meta name="citation_reference" content="citation_title=Structure, fluctuation and magnitude of a natural grassland soil metagenome;  citation_journal_title=ISME J.;  citation_year=2012;  citation_volume=6;  citation_pages=1677-87; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics of microbial communities;  citation_journal_title=Science;  citation_year=2005;  citation_volume=308;  citation_pages=554-7; " /><meta name="citation_reference" content="citation_title=Identifying the dominant soil bacterial taxa in libraries of 16S rRNA and 16S rRNA genes;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=1719-28; " /><meta name="citation_reference" content="citation_title=Soil microbial community responses to a decade of warming as revealed by comparative metagenomics;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2014;  citation_volume=80;  citation_pages=1777-86; " /><meta name="citation_reference" content="citation_title=Isolation of oxygenase genes for indigo-forming activity from an artificially polluted soil metagenome by functional screening using Pseudomonas putida strains as hosts;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2015;  citation_volume=99;  citation_pages=4453-70; " /><meta name="citation_reference" content="citation_title=Design and experimental application of a novel non-degenerate universal primer set that amplifies prokaryotic 16S rRNA genes with a low possibility to amplify eukaryotic rRNA genes;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=217-27; " /><meta name="citation_reference" content="citation_title=Search and clustering orders of magnitude faster than BLAST;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=2460-1; " /><meta name="citation_reference" content="citation_title=UCHIME improves sensitivity and speed of chimera detection;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2194-200; " /><meta name="citation_reference" content="citation_title=Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2007;  citation_volume=73;  citation_pages=5261-7; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=IDBA-UD: a de novo assembler for single-cell and metagenomic sequencing data with highly uneven depth;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1420-8; " /><meta name="citation_reference" content="citation_title=Ab initio gene identification in metagenomic sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e132" /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=A human gut microbial gene catalogue established by metagenomic sequencing;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=59-65; " /><meta name="citation_reference" content="citation_title=Prediction of effective genome size in metagenomic samples;  citation_journal_title=Genome Biol.;  citation_year=2007;  citation_volume=8;  citation_pages=R10" /><meta name="citation_reference" content="citation_title=A dendrite method for cluster analysis;  citation_journal_title=Commun. Stat.;  citation_year=1974;  citation_volume=3;  citation_pages=1-27; " /><meta name="citation_reference" content="citation_title=Enterotypes of the human gut microbiome;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=473;  citation_pages=174-80; " /><meta name="citation_reference" content="citation_title=SHOT: a web server for the construction of genome phylogenies;  citation_journal_title=Trends Genet.;  citation_year=2002;  citation_volume=18;  citation_pages=158-62; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes;  citation_journal_title=DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=The role of active-site residues in naphthalene dioxygenase;  citation_journal_title=J. Ind. Microbiol. Biotechnol.;  citation_year=2003;  citation_volume=30;  citation_pages=271-8; " /><meta name="citation_reference" content="citation_title=Analysis of the microbial gene landscape and transcriptome for aromatic pollutants and alkane degradation using a novel internally calibrated microarray system;  citation_journal_title=Environ. Microbiol.;  citation_year=2013;  citation_volume=15;  citation_pages=1016-39; " /><meta name="citation_reference" content="citation_title=Metaproteogenomic insights beyond bacterial response to naphthalene exposure and bio-stimulation;  citation_journal_title=ISME J.;  citation_year=2012;  citation_volume=7;  citation_pages=122-36; " /><meta name="citation_reference" content="citation_title=ACLAME: a classification of mobile genetic elements, update 2010;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=D57-61; " /><meta name="citation_reference" content="citation_title=Orthologous gene clusters and taxon signature genes for viruses of prokaryotes;  citation_journal_title=J. Bacteriol.;  citation_year=2013;  citation_volume=195;  citation_pages=941-50; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=PHAST: a fast phage search tool;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W347-52; " /><meta name="citation_reference" content="citation_title=GenoFrag: software to design primers optimized for whole genome scanning by long-range PCR amplification;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=17-24; " /><meta name="citation_reference" content="citation_title=Microbial biodegradation of polyaromatic hydrocarbons;  citation_journal_title=FEMS Microbiol. Rev.;  citation_year=2008;  citation_volume=32;  citation_pages=927-55; " /><meta name="citation_reference" content="citation_title=Resistance and resilience of the forest soil microbiome to logging-associated compaction;  citation_journal_title=ISME J.;  citation_year=2014;  citation_volume=8;  citation_pages=226-44; " /><meta name="citation_reference" content="citation_title=Metagenomic insights into the evolution, function, and complexity of the planktonic microbial community of Lake Lanier, a temperate freshwater ecosystem;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2011;  citation_volume=77;  citation_pages=6000-11; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis of the bioremediation of diesel-contaminated Canadian high arctic soils;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e30058" /><meta name="citation_reference" content="citation_title=Genomic analysis of the potential for aromatic compounds biodegradation in Burkholderiales;  citation_journal_title=Environ. Microbiol.;  citation_year=2012;  citation_volume=14;  citation_pages=1091-117; " /><meta name="citation_reference" content="citation_title=Comparative genomics in the genus Burkholderia; citation_publisher=Caister Academic Press, Norfolk, UK;  citation_title=Burkholderia: From Genomes to Function;  citation_year=2014;  citation_pages=31-50; " /><meta name="citation_reference" content="citation_title=Pseudomonas genomes: diverse and adaptable;  citation_journal_title=FEMS Microbiol. Rev.;  citation_year=2011;  citation_volume=35;  citation_pages=652-80; " /><meta name="citation_reference" content="citation_title=Reconstructing the microbial diversity and function of pre-agricultural tallgrass prairie soils in the United States;  citation_journal_title=Science;  citation_year=2013;  citation_volume=342;  citation_pages=621-4; " /><meta name="citation_reference" content="citation_title=Biological robustness;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=826-37; " /><meta name="citation_reference" content="citation_title=Metagenomic analysis reveals microbial functional redundancies and specificities in a soil under different tillage and crop-management regimes;  citation_journal_title=Appl. Soil Ecol.;  citation_year=2015;  citation_volume=86;  citation_pages=106-12; " /><meta name="citation_reference" content="citation_title=Microbial genes and enzymes in the degradation of chlorinated compounds;  citation_journal_title=Chem. Rec.;  citation_year=2003;  citation_volume=3;  citation_pages=158-71; " /><meta name="citation_reference" content="citation_title=Analysis of a bacterial community in 3-chlorobenzoate-contaminated soil by PCR-DGGE targeting the 16S rRNA gene and benzoate 1,2-dioxygenase gene (benA);  citation_journal_title=Microbes Environ.;  citation_year=2005;  citation_volume=20;  citation_pages=151-9; " /><meta name="citation_reference" content="citation_title=Novel organization of the genes for phthalate degradation from Burkholderia cepacia DBO1;  citation_journal_title=J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=6529-37; " /><meta name="citation_reference" content="citation_title=Purification and characterization of a salicylate hydroxylase involved in 1-hydroxy-2-naphthoic acid hydroxylation from the naphthalene and phenanthrene-degrading bacterial strain Pseudomonas putida BS202-P1;  citation_journal_title=Biodegradation;  citation_year=2001;  citation_volume=12;  citation_pages=179-88; " /><meta name="citation_reference" content="citation_title=Molecular characterization of a phenanthrene degradation pathway in Mycobacterium vanbaalenii PYR-1;  citation_journal_title=Biochem. Biophys. Res. Commun.;  citation_year=2004;  citation_volume=322;  citation_pages=133-46; " /><meta name="citation_reference" content="citation_title=Degradation of phenanthrene and anthracene by cell suspensions of Mycobacterium sp. strain PYR-1;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2001;  citation_volume=67;  citation_pages=1476-83; " /><meta name="citation_reference" content="citation_title=Ethyl tert-butyl ether (ETBE) biodegradation by a syntrophic association of Rhodococcus sp. IFP 2042 and Bradyrhizobium sp. IFP 2049 isolated from a polluted aquifer;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2013;  citation_volume=97;  citation_pages=10531-39; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of a phenanthrene degrader, Mycobacterium sp. strain EPa45 (NBRC 110737), isolated from a phenanthrene-degrading consortium;  citation_journal_title=Genome Announc.;  citation_year=2015;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Role of phages in the pathogenesis of Burkholderia, or 'where are the toxin genes in Burkholderia phages?;  citation_journal_title=Curr. Opin. Microbiol.;  citation_year=2007;  citation_volume=10;  citation_pages=410-7; " /><meta name="citation_reference" content="citation_title=Genomics of Burkholderia phages; citation_publisher=Caister Academic Press, Norfolk, UK;  citation_title=Burkholderia: From Genomes to Function;  citation_year=2014;  citation_pages=230-49; " /><meta name="citation_reference" content="citation_title=Viral and microbial community dynamics in four aquatic environments;  citation_journal_title=ISME J.;  citation_year=2010;  citation_volume=4;  citation_pages=739-51; " /><meta name="citation_reference" content="citation_title=Deciphering microbial community robustness through synthetic ecology and molecular systems synecology;  citation_journal_title=Curr. Opin. Biotechnol.;  citation_year=2015;  citation_volume=33;  citation_pages=305-17; " /><meta name="citation_reference" content="citation_title=Community-integrated omics links dominance of a microbial generalist to fine-tuned resource usage;  citation_journal_title=Nat. Commun.;  citation_year=2014;  citation_volume=5;  citation_pages=5603" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/6/413/17462969/dsv023.pdf" /><meta name="description" content="Abstract.  Soil microbial communities have great potential for bioremediation of recalcitrant aromatic compounds. However, it is unclear which taxa and genes in" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/6/413/2462440" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/6/413/2462440" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/6/413/2462440");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv023';
    </script>
<title>
	Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462440" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdiXATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F0bywgSGlyb21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCAyLTEtMSBLYXRhaGlyYSwgU2VuZGFpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmksIEhpcm9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgSW5mb3JtYXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgMi0xMi0xIE9va2F5YW1hLCBUb2t5byAxNTItODU1MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXJ1eWFtYSwgRnVtaXRvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljaW5lLCBLeW90byBVbml2ZXJzaXR5LCBLeW90byA2MDYtODUwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb3lvZGEsIEF0c3VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPc2hpbWEsIEtlbnNoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIDUtMS01IEthc2hpd2Fub2hhLCBLYXNoaXdhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkVuZG8sIFJ5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGdWNodSwgR2Vua2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWl5YWtvc2hpLCBNYXNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRG96b25vLCBBeXVtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEgT29rYXlhbWEsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9odHN1Ym8sIFlvc2hpeXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWdhdGEsIFl1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIDItMS0xIEthdGFoaXJhLCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgTWFzYWhpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2EgMjc3LTg1NjEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVqaXlhbWEsIEFzYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQcmluY2lwbGVzIG9mIEluZm9ybWF0aWNzIFJlc2VhcmNoIERpdmlzaW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgSW5mb3JtYXRpY3MsIEhpdG90c3ViYXNoaSwgVG9reW8gMTAxLTg0MzAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3Vyb2thd2EsIEtlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEgT29rYXlhbWEsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVhcnRoLUxpZmUgU2NpZW5jZSBJbnN0aXR1dGUsIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEgT29rYXlhbWEsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWRhLCBNYXNhdGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgMi0xLTEgS2F0YWhpcmEsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRpbWUtc2VyaWVzIG1ldGFnZW5vbWljIGFuYWx5c2lzIHJldmVhbHMgcm9idXN0bmVzcyBvZiBzb2lsIG1pY3JvYmlvbWUgYWdhaW5zdCBjaGVtaWNhbCBkaXN0dXJiYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5kYW1lbnRhbHMgb2YgbWljcm9iaWFsIGNvbW11bml0eSByZXNpc3RhbmNlIGFuZCByZXNpbGllbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzaXN0YW5jZSwgcmVzaWxpZW5jZSwgYW5kIHJlZHVuZGFuY3kgaW4gbWljcm9iaWFsIGNvbW11bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMTsgIGNpdGF0aW9uX3BhZ2VzPTExNTEyLTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUsIGZ1bmN0aW9uIGFuZCBkaXZlcnNpdHkgb2YgdGhlIGhlYWx0aHkgaHVtYW4gbWljcm9iaW9tZTsgY2l0YXRpb25fYXV0aG9yPVRoZSBIdW1hbiBNaWNyb2Jpb21lIFByb2plY3QgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NjsgIGNpdGF0aW9uX3BhZ2VzPTIwNy0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVhcnRoIE1pY3JvYmlvbWUgcHJvamVjdDogc3VjY2Vzc2VzIGFuZCBhc3BpcmF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW1lIHNlcmllcyBjb21tdW5pdHkgZ2Vub21pY3MgYW5hbHlzaXMgcmV2ZWFscyByYXBpZCBzaGlmdHMgaW4gYmFjdGVyaWFsIHNwZWNpZXMsIHN0cmFpbnMsIGFuZCBwaGFnZSBkdXJpbmcgaW5mYW50IGd1dCBjb2xvbml6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTExLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWNjZXNzaW9uIG9mIG1pY3JvYmlhbCBjb25zb3J0aWEgaW4gdGhlIGRldmVsb3BpbmcgaW5mYW50IGd1dCBtaWNyb2Jpb21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMTsgIGNpdGF0aW9uX3BhZ2VzPTQ1NzgtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljIGFuYWx5c2lzIG9mIGEgcGVybWFmcm9zdCBtaWNyb2JpYWwgY29tbXVuaXR5IHJldmVhbHMgYSByYXBpZCByZXNwb25zZSB0byB0aGF3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDgwOyAgY2l0YXRpb25fcGFnZXM9MzY4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUsIGZsdWN0dWF0aW9uIGFuZCBtYWduaXR1ZGUgb2YgYSBuYXR1cmFsIGdyYXNzbGFuZCBzb2lsIG1ldGFnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE2NzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1ldGFnZW5vbWljcyBvZiBtaWNyb2JpYWwgY29tbXVuaXRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA4OyAgY2l0YXRpb25fcGFnZXM9NTU0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZ5aW5nIHRoZSBkb21pbmFudCBzb2lsIGJhY3RlcmlhbCB0YXhhIGluIGxpYnJhcmllcyBvZiAxNlMgclJOQSBhbmQgMTZTIHJSTkEgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz0xNzE5LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2lsIG1pY3JvYmlhbCBjb21tdW5pdHkgcmVzcG9uc2VzIHRvIGEgZGVjYWRlIG9mIHdhcm1pbmcgYXMgcmV2ZWFsZWQgYnkgY29tcGFyYXRpdmUgbWV0YWdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9MTc3Ny04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIG94eWdlbmFzZSBnZW5lcyBmb3IgaW5kaWdvLWZvcm1pbmcgYWN0aXZpdHkgZnJvbSBhbiBhcnRpZmljaWFsbHkgcG9sbHV0ZWQgc29pbCBtZXRhZ2Vub21lIGJ5IGZ1bmN0aW9uYWwgc2NyZWVuaW5nIHVzaW5nIFBzZXVkb21vbmFzIHB1dGlkYSBzdHJhaW5zIGFzIGhvc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9NDQ1My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVzaWduIGFuZCBleHBlcmltZW50YWwgYXBwbGljYXRpb24gb2YgYSBub3ZlbCBub24tZGVnZW5lcmF0ZSB1bml2ZXJzYWwgcHJpbWVyIHNldCB0aGF0IGFtcGxpZmllcyBwcm9rYXJ5b3RpYyAxNlMgclJOQSBnZW5lcyB3aXRoIGEgbG93IHBvc3NpYmlsaXR5IHRvIGFtcGxpZnkgZXVrYXJ5b3RpYyByUk5BIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTIxNy0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhcmNoIGFuZCBjbHVzdGVyaW5nIG9yZGVycyBvZiBtYWduaXR1ZGUgZmFzdGVyIHRoYW4gQkxBU1Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MjQ2MC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VQ0hJTUUgaW1wcm92ZXMgc2Vuc2l0aXZpdHkgYW5kIHNwZWVkIG9mIGNoaW1lcmEgZGV0ZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIxOTQtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYWl2ZSBCYXllc2lhbiBjbGFzc2lmaWVyIGZvciByYXBpZCBhc3NpZ25tZW50IG9mIHJSTkEgc2VxdWVuY2VzIGludG8gdGhlIG5ldyBiYWN0ZXJpYWwgdGF4b25vbXk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz01MjYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlEQkEtVUQ6IGEgZGUgbm92byBhc3NlbWJsZXIgZm9yIHNpbmdsZS1jZWxsIGFuZCBtZXRhZ2Vub21pYyBzZXF1ZW5jaW5nIGRhdGEgd2l0aCBoaWdobHkgdW5ldmVuIGRlcHRoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE0MjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWIgaW5pdGlvIGdlbmUgaWRlbnRpZmljYXRpb24gaW4gbWV0YWdlbm9taWMgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1lMTMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgbG9uZy1yZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGh1bWFuIGd1dCBtaWNyb2JpYWwgZ2VuZSBjYXRhbG9ndWUgZXN0YWJsaXNoZWQgYnkgbWV0YWdlbm9taWMgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIGVmZmVjdGl2ZSBnZW5vbWUgc2l6ZSBpbiBtZXRhZ2Vub21pYyBzYW1wbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGVuZHJpdGUgbWV0aG9kIGZvciBjbHVzdGVyIGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21tdW4uIFN0YXQuOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xLTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnRlcm90eXBlcyBvZiB0aGUgaHVtYW4gZ3V0IG1pY3JvYmlvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NzM7ICBjaXRhdGlvbl9wYWdlcz0xNzQtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNIT1Q6IGEgd2ViIHNlcnZlciBmb3IgdGhlIGNvbnN0cnVjdGlvbiBvZiBnZW5vbWUgcGh5bG9nZW5pZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1ldGFnZW5vbWljcyByZXZlYWxlZCBjb21tb25seSBlbnJpY2hlZCBnZW5lIHNldHMgaW4gaHVtYW4gZ3V0IG1pY3JvYmlvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgYWN0aXZlLXNpdGUgcmVzaWR1ZXMgaW4gbmFwaHRoYWxlbmUgZGlveHlnZW5hc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEluZC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgbWljcm9iaWFsIGdlbmUgbGFuZHNjYXBlIGFuZCB0cmFuc2NyaXB0b21lIGZvciBhcm9tYXRpYyBwb2xsdXRhbnRzIGFuZCBhbGthbmUgZGVncmFkYXRpb24gdXNpbmcgYSBub3ZlbCBpbnRlcm5hbGx5IGNhbGlicmF0ZWQgbWljcm9hcnJheSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMDE2LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhcHJvdGVvZ2Vub21pYyBpbnNpZ2h0cyBiZXlvbmQgYmFjdGVyaWFsIHJlc3BvbnNlIHRvIG5hcGh0aGFsZW5lIGV4cG9zdXJlIGFuZCBiaW8tc3RpbXVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEyMi0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QUNMQU1FOiBhIGNsYXNzaWZpY2F0aW9uIG9mIG1vYmlsZSBnZW5ldGljIGVsZW1lbnRzLCB1cGRhdGUgMjAxMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9RDU3LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcnRob2xvZ291cyBnZW5lIGNsdXN0ZXJzIGFuZCB0YXhvbiBzaWduYXR1cmUgZ2VuZXMgZm9yIHZpcnVzZXMgb2YgcHJva2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTk1OyAgY2l0YXRpb25fcGFnZXM9OTQxLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGdhcHBlZC1yZWFkIGFsaWdubWVudCB3aXRoIEJvd3RpZSAyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QSEFTVDogYSBmYXN0IHBoYWdlIHNlYXJjaCB0b29sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMzQ3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vRnJhZzogc29mdHdhcmUgdG8gZGVzaWduIHByaW1lcnMgb3B0aW1pemVkIGZvciB3aG9sZSBnZW5vbWUgc2Nhbm5pbmcgYnkgbG9uZy1yYW5nZSBQQ1IgYW1wbGlmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYmlhbCBiaW9kZWdyYWRhdGlvbiBvZiBwb2x5YXJvbWF0aWMgaHlkcm9jYXJib25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTkyNy01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzaXN0YW5jZSBhbmQgcmVzaWxpZW5jZSBvZiB0aGUgZm9yZXN0IHNvaWwgbWljcm9iaW9tZSB0byBsb2dnaW5nLWFzc29jaWF0ZWQgY29tcGFjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjI2LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhZ2Vub21pYyBpbnNpZ2h0cyBpbnRvIHRoZSBldm9sdXRpb24sIGZ1bmN0aW9uLCBhbmQgY29tcGxleGl0eSBvZiB0aGUgcGxhbmt0b25pYyBtaWNyb2JpYWwgY29tbXVuaXR5IG9mIExha2UgTGFuaWVyLCBhIHRlbXBlcmF0ZSBmcmVzaHdhdGVyIGVjb3N5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTYwMDAtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGFnZW5vbWljIGFuYWx5c2lzIG9mIHRoZSBiaW9yZW1lZGlhdGlvbiBvZiBkaWVzZWwtY29udGFtaW5hdGVkIENhbmFkaWFuIGhpZ2ggYXJjdGljIHNvaWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTMwMDU4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBvZiB0aGUgcG90ZW50aWFsIGZvciBhcm9tYXRpYyBjb21wb3VuZHMgYmlvZGVncmFkYXRpb24gaW4gQnVya2hvbGRlcmlhbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTA5MS0xMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIGluIHRoZSBnZW51cyBCdXJraG9sZGVyaWE7IGNpdGF0aW9uX3B1Ymxpc2hlcj1DYWlzdGVyIEFjYWRlbWljIFByZXNzLCBOb3Jmb2xrLCBVSzsgIGNpdGF0aW9uX3RpdGxlPUJ1cmtob2xkZXJpYTogRnJvbSBHZW5vbWVzIHRvIEZ1bmN0aW9uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fcGFnZXM9MzEtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBzZXVkb21vbmFzIGdlbm9tZXM6IGRpdmVyc2UgYW5kIGFkYXB0YWJsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz02NTItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29uc3RydWN0aW5nIHRoZSBtaWNyb2JpYWwgZGl2ZXJzaXR5IGFuZCBmdW5jdGlvbiBvZiBwcmUtYWdyaWN1bHR1cmFsIHRhbGxncmFzcyBwcmFpcmllIHNvaWxzIGluIHRoZSBVbml0ZWQgU3RhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM0MjsgIGNpdGF0aW9uX3BhZ2VzPTYyMS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9sb2dpY2FsIHJvYnVzdG5lc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTgyNi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWMgYW5hbHlzaXMgcmV2ZWFscyBtaWNyb2JpYWwgZnVuY3Rpb25hbCByZWR1bmRhbmNpZXMgYW5kIHNwZWNpZmljaXRpZXMgaW4gYSBzb2lsIHVuZGVyIGRpZmZlcmVudCB0aWxsYWdlIGFuZCBjcm9wLW1hbmFnZW1lbnQgcmVnaW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gU29pbCBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTEwNi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9iaWFsIGdlbmVzIGFuZCBlbnp5bWVzIGluIHRoZSBkZWdyYWRhdGlvbiBvZiBjaGxvcmluYXRlZCBjb21wb3VuZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNoZW0uIFJlYy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTE1OC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgYSBiYWN0ZXJpYWwgY29tbXVuaXR5IGluIDMtY2hsb3JvYmVuem9hdGUtY29udGFtaW5hdGVkIHNvaWwgYnkgUENSLURHR0UgdGFyZ2V0aW5nIHRoZSAxNlMgclJOQSBnZW5lIGFuZCBiZW56b2F0ZSAxLDItZGlveHlnZW5hc2UgZ2VuZSAoYmVuQSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmVzIEVudmlyb24uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTUxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vdmVsIG9yZ2FuaXphdGlvbiBvZiB0aGUgZ2VuZXMgZm9yIHBodGhhbGF0ZSBkZWdyYWRhdGlvbiBmcm9tIEJ1cmtob2xkZXJpYSBjZXBhY2lhIERCTzE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9NjUyOS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHVyaWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgc2FsaWN5bGF0ZSBoeWRyb3h5bGFzZSBpbnZvbHZlZCBpbiAxLWh5ZHJveHktMi1uYXBodGhvaWMgYWNpZCBoeWRyb3h5bGF0aW9uIGZyb20gdGhlIG5hcGh0aGFsZW5lIGFuZCBwaGVuYW50aHJlbmUtZGVncmFkaW5nIGJhY3RlcmlhbCBzdHJhaW4gUHNldWRvbW9uYXMgcHV0aWRhIEJTMjAyLVAxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9kZWdyYWRhdGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3OS04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gb2YgYSBwaGVuYW50aHJlbmUgZGVncmFkYXRpb24gcGF0aHdheSBpbiBNeWNvYmFjdGVyaXVtIHZhbmJhYWxlbmlpIFBZUi0xOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzIyOyAgY2l0YXRpb25fcGFnZXM9MTMzLTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWdyYWRhdGlvbiBvZiBwaGVuYW50aHJlbmUgYW5kIGFudGhyYWNlbmUgYnkgY2VsbCBzdXNwZW5zaW9ucyBvZiBNeWNvYmFjdGVyaXVtIHNwLiBzdHJhaW4gUFlSLTE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0xNDc2LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdGh5bCB0ZXJ0LWJ1dHlsIGV0aGVyIChFVEJFKSBiaW9kZWdyYWRhdGlvbiBieSBhIHN5bnRyb3BoaWMgYXNzb2NpYXRpb24gb2YgUmhvZG9jb2NjdXMgc3AuIElGUCAyMDQyIGFuZCBCcmFkeXJoaXpvYml1bSBzcC4gSUZQIDIwNDkgaXNvbGF0ZWQgZnJvbSBhIHBvbGx1dGVkIGFxdWlmZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xMDUzMS0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIGEgcGhlbmFudGhyZW5lIGRlZ3JhZGVyLCBNeWNvYmFjdGVyaXVtIHNwLiBzdHJhaW4gRVBhNDUgKE5CUkMgMTEwNzM3KSwgaXNvbGF0ZWQgZnJvbSBhIHBoZW5hbnRocmVuZS1kZWdyYWRpbmcgY29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEFubm91bmMuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgcGhhZ2VzIGluIHRoZSBwYXRob2dlbmVzaXMgb2YgQnVya2hvbGRlcmlhLCBvciAnd2hlcmUgYXJlIHRoZSB0b3hpbiBnZW5lcyBpbiBCdXJraG9sZGVyaWEgcGhhZ2VzPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQxMC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBCdXJraG9sZGVyaWEgcGhhZ2VzOyBjaXRhdGlvbl9wdWJsaXNoZXI9Q2Fpc3RlciBBY2FkZW1pYyBQcmVzcywgTm9yZm9saywgVUs7ICBjaXRhdGlvbl90aXRsZT1CdXJraG9sZGVyaWE6IEZyb20gR2Vub21lcyB0byBGdW5jdGlvbjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTIzMC00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlyYWwgYW5kIG1pY3JvYmlhbCBjb21tdW5pdHkgZHluYW1pY3MgaW4gZm91ciBhcXVhdGljIGVudmlyb25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NzM5LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWNpcGhlcmluZyBtaWNyb2JpYWwgY29tbXVuaXR5IHJvYnVzdG5lc3MgdGhyb3VnaCBzeW50aGV0aWMgZWNvbG9neSBhbmQgbW9sZWN1bGFyIHN5c3RlbXMgc3luZWNvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbW11bml0eS1pbnRlZ3JhdGVkIG9taWNzIGxpbmtzIGRvbWluYW5jZSBvZiBhIG1pY3JvYmlhbCBnZW5lcmFsaXN0IHRvIGZpbmUtdHVuZWQgcmVzb3VyY2UgdXNhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NTYwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi82LzQxMy8xNzQ2Mjk2OS9kc3YwMjMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFNvaWwgbWljcm9iaWFsIGNvbW11bml0aWVzIGhhdmUgZ3JlYXQgcG90ZW50aWFsIGZvciBiaW9yZW1lZGlhdGlvbiBvZiByZWNhbGNpdHJhbnQgYXJvbWF0aWMgY29tcG91bmRzLiBIb3dldmVyLCBpdCBpcyB1bmNsZWFyIHdoaWNoIHRheGEgYW5kIGdlbmVzIGluIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDYyNDQwJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZOTBbr61DgeEEdzDGqd3EbToozdS" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f413%2f2462440" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f413%2f2462440" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83834" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/6/1/m_cover.gif?Expires=2147483647&amp;Signature=083aE4wPido26a6~EOoRT3Nb4fPpp5HV1MZzrjjay94a3evfhfRTvESRMhc7ROqyA2UIz2umFef89XL3Lu7hUZFod2Ix~QtkT4CP~8MDkYY61O0I1KLQQcYNpXVti4-0uwuVQWc0Ik7X46YCFzJAu2cTqwHZ2B0OBVFR-OZWKIwha86xHbqGB8R5L83t0zl8dqx1CkCPppI95mus6SUAdvZ0-wLyrVhsfEErb~4S7v8v0rjWvz1OZ1A2b6wWGyo1cjX3sRRxlgjkSlblmiowk2fs-R4agAPGw2bGiz2-0n1EnfsgKN8CSyFRqrRfdmccMFKJr5L5jh5C1s7L6OJB0g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/6">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197904">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197904">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197906">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197906">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197909">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197969">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197969">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198023">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198023">4. Data availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198026">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198026">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198028">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198028">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/6/403/2462415">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/6/425/2462458">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromi Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromi+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiromi Kato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Mori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These authors contributed equally to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Mori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mori H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mori H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiroshi Mori</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumito Maruyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Medicine</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kyoto 606-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumito+Maruyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maruyama F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maruyama F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fumito Maruyama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Center for Information Biology</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenshiro Oshima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenshiro+Oshima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oshima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oshima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenshiro Oshima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryo Endo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryo+Endo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Endo R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Endo R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ryo Endo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Genki Fuchu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Genki+Fuchu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fuchu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fuchu G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Genki Fuchu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masatoshi Miyakoshi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">‡</div><p>Present address: Department of Biotechnology, Akita Prefectural University, 241-438 Shimoshinjo Nakano, Akita, 010-0195, Japan.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masatoshi+Miyakoshi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyakoshi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyakoshi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masatoshi Miyakoshi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayumi Dozono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayumi+Dozono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dozono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dozono A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ayumi Dozono</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiyuki Ohtsubo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiyuki+Ohtsubo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohtsubo Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohtsubo Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yoshiyuki Ohtsubo</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Nagata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Nagata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagata Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagata Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yuji Nagata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asao Fujiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Center for Information Biology</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Principles of Informatics Research Division</div>, <div class="insititution">National Institute of Informatics</div>, <div class="addr-line">Hitotsubashi, Tokyo 101-8430</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asao+Fujiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujiyama A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujiyama A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Asao Fujiyama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ken Kurokawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Information, Graduate School of Bioscience and Biotechnology</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Tokyo 152-8550</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Earth-Life Science Institute</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ken+Kurokawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kurokawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kurokawa K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ken Kurokawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masataka Tsuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">2-1-1 Katahira, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 22-217-5699. Fax: +81 22-217-5699. E-mail: <a href="mailto:mtsuda@ige.tohoku.ac.jp" target="_blank">mtsuda@ige.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masataka+Tsuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuda M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masataka Tsuda</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Department of Biotechnology, Akita Prefectural University, 241-438 Shimoshinjo Nakano, Akita, 010-0195, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="footnote-content">Edited by Dr Naotake Ogasawara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 413–424, <a href='https://doi.org/10.1093/dnares/dsv023'>https://doi.org/10.1093/dnares/dsv023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 October 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 September 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462440" href="/dnaresearch/article-pdf/22/6/413/17462969/dsv023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hiromi Kato, Hiroshi Mori, Fumito Maruyama, Atsushi Toyoda, Kenshiro Oshima, Ryo Endo, Genki Fuchu, Masatoshi Miyakoshi, Ayumi Dozono, Yoshiyuki Ohtsubo, Yuji Nagata, Masahira Hattori, Asao Fujiyama, Ken Kurokawa, Masataka Tsuda;  Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance, <em>DNA Research</em>, Volume 22, Issue 6, 1 December 2015, Pages 413–424, <a href='https://doi.org/10.1093/dnares/dsv023'>https://doi.org/10.1093/dnares/dsv023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462440?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Time-series%20metagenomic%20analysis%20reveals%20robustness%20of%20soil%20microbiome%20against%20chemical%20disturbance&amp;publicationDate=2015-10-01&amp;volumeNum=22&amp;issueNum=6&amp;author=Kato%2C%20Hiromi%3B%20Mori%2C%20Hiroshi&amp;startPage=413&amp;endPage=424&amp;contentId=10.1093%2Fdnares%2Fdsv023&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f413%2f2462440" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87197904 id="87197904" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Soil microbial communities have great potential for bioremediation of recalcitrant aromatic compounds. However, it is unclear which taxa and genes in the communities, and how they contribute to the bioremediation in the polluted soils. To get clues about this fundamental question here, time-course (up to 24 weeks) metagenomic analysis of microbial community in a closed soil microcosm artificially polluted with four aromatic compounds, including phenanthrene, was conducted to investigate the changes in the community structures and gene pools. The pollution led to drastic changes in the community structures and the gene sets for pollutant degradation. Complete degradation of phenanthrene was strongly suggested to occur by the syntrophic metabolism by <em>Mycobacterium</em> and the most proliferating genus, <em>Burkholderia</em>. The community structure at Week 24 (∼12 weeks after disappearance of the pollutants) returned to the structure similar to that before pollution. Our time-course metagenomic analysis of phage genes strongly suggested the involvement of the ‘kill-the-winner’ phenomenon (i.e. phage predation of <em>Burkholderia</em> cells) for the returning of the microbial community structure. The pollution resulted in a decrease in taxonomic diversity and a drastic increase in diversity of gene pools in the communities, showing the functional redundancy and robustness of the communities against chemical disturbance.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;soil microbiome&quot;">soil microbiome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="metagenome">metagenome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;recalcitrant aromatic compounds&quot;">recalcitrant aromatic compounds</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phage">phage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="robustness">robustness</a></div>                            <h2 scrollto-destination=87197906 id="87197906" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Microbial communities in natural environments almost always encounter a wide variety of environmental changes (or ‘disturbances,’ in the field of ecology) in physical, chemical, and biological factors.<sup><a reveal-id="DSV023C1" data-open="DSV023C1" class="link link-ref link-reveal xref-bibr">1</a></sup> It has been well recognized that indigenous microbial communities adapt to such changes through succession of their community structures.<sup><a reveal-id="DSV023C2" data-open="DSV023C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, it still remains unclear (i) how the community <em>per se</em> responds to the change of environmental factors and (ii) which member(s) or group(s) and what kinds of genes in the community indeed play important roles in responding to the changes in each environment. Taking into consideration that &gt;99% of environmental microbes are still unculturable under laboratory conditions, application of metagenomic approach has great advantages for investigating the structure and functional potentials of various microbial communities. Numerous ongoing projects involving deep sequencing of a variety of microbiota from different environments are providing a worldwide catalogue of metagenomic characteristics of microbial communities, and subsequent comparison of the characteristics is expected to clarify the environmental factors that govern the taxonomic and functional differences in the microbial communities.<sup><a reveal-id="DSV023C3" data-open="DSV023C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSV023C4" data-open="DSV023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> However, real-world environments are often subject to unpredictable and simultaneous changes of multiple physicochemical parameters in addition to frequent migration of organisms. For this reason, it is difficult to use metagenomic analysis of microbiota from natural environments to obtain information on the response of microbial community to the change of a specific environmental factor. Such information can be more definitively obtained by investigating the time-course change of microbiomes in a physically closed experimental ecosystem (i.e. microcosm) than by comparing a collection of ‘snapshots’ of metagenomes from various real-world microbial communities. Although several studies of time-course changes in human gut microbiomes have been reported,<sup><a reveal-id="DSV023C5" data-open="DSV023C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV023C6" data-open="DSV023C6" class="link link-ref link-reveal xref-bibr">6</a></sup> the application of such a strategy to soil environments is still challenging.<sup><a reveal-id="DSV023C7" data-open="DSV023C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSV023C8" data-open="DSV023C8" class="link link-ref link-reveal xref-bibr">8</a></sup> This is because soil environments exhibit the most tremendous biodiversity on the Earth.<sup><a reveal-id="DSV023C9  DSV023C11" data-open="DSV023C9  DSV023C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> Our aim of this study under the closed soil microcosm conditions was to investigate which members in a microbial community respond, and in what way, to the addition of chemical pollutants, which was considered as a change of one environmental factor. For this purpose, a closed soil microcosm was artificially amended simultaneously with four recalcitrant aromatic compounds, and the time-course changes in the taxonomic and gene compositions of the soil microbiota were investigated by metagenomic analysis. Our results revealed (i) drastic changes in the community structures and the gene sets for pollutant degradation, (ii) strong suggestion of the complete degradation of one aromatic compound by syntrophic metabolism with the involvement of two phylogenetically distant bacterial phyla, (iii) the return of community structure long after disappearance of the pollutants to the structure similar to that before pollution by the decrease of the most successfully propagated genus most probably through phage predation, and (iv) the redundancy and robustness of functional potentials of the community against the chemical disturbance.</p>                            <h2 scrollto-destination=87197909 id="87197909" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87197911 id="87197911" class="section-title" data-legacy-id=s2a>2.1 Preparation of artificially polluted soil samples</h3>
<p>A farm soil (soil type: brown forest soil) at the Ehime Research Institute of Agriculture, Forestry, and Fisheries (33°58′42″ N and 132°48′03″ E), in Matsuyama, Japan<sup><a reveal-id="DSV023C12" data-open="DSV023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> was used in this study, and this soil had not been polluted with harmful aromatic compounds before our sampling. The soil was sampled at depths of 5–10 cm from the surface in April 2008, and large particles were removed with a 2-mm mesh sieve. The soil consisted of 77% sand, 11% of silt, and 12% clay, and was classified as sandy loam. Other physicochemical properties of the soil sample are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. The detailed procedure for artificially polluting the soil with four aromatic compounds, 3-chlorobenzoate (3CB), phenanthrene, biphenyl, and carbazole (i.e. 9-azafluerene; an aromatic heterocyclic organic compound), each at a final concentration of 125 mg/kg of wet soil, was described in our previous report.<sup><a reveal-id="DSV023C12" data-open="DSV023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Each 200-g portion of the polluted or non-polluted control soil sample was transferred to a sterilized glass pot with a loose lid, and sterilized distilled water was then added to the soil to adjust its water content to 60% of the maximum water-holding capacity. The pots were incubated at 28°C in the dark for up to 24 weeks. Our monitoring of soil samples revealed the &lt;8% changes in the water contents during the 24-week incubation (data not shown). At an appropriate time point after the incubation, all of the soil in each pot was fully harvested to (i) measure the amounts of pollutants, (ii) extract metagenomic DNA, (iii) store the microbial community at −80°C after its suspension in 15% glycerol or dimethyl sulfoxide solution, and (iv) store the remaining soil sample at −80°C.</p>                            <h3 scrollto-destination=87197916 id="87197916" class="section-title" data-legacy-id=s2b>2.2 Preparation of metagenomic DNA</h3>
<p>Metagenomic DNA from 10-g soil sample was prepared by using a PowerMax<sup>TM</sup> Soil DNA Isolation Kit (MO BIO Laboratories, Carlsbad, CA, USA). When the extracted DNA solution contained a large amount of humic substances, they were removed by repeating column purification step according to the protocol by the manufacture. The DNA solution from a total of 20-g soil was concentrated by ethanol precipitation, and used for the metagenomic sequencing and the templates for PCR reactions.</p>                            <h3 scrollto-destination=87197920 id="87197920" class="section-title" data-legacy-id=s2c>2.3 Pyrosequencing of 16S rRNA gene amplicons and their taxonomic analysis</h3>
<p>The detailed analysis of the soil microbial communities on the basis of 16S rRNA gene sequences was performed by the Roche 454 pyrosequencing of PCR amplicons of 16S rRNA genes using 11 metagenomic DNA samples that were prepared from the control and polluted soils as well as the soil just before the pollution. Parts of the 16S rRNA genes [the V3–V4 region (positions from 342 to 806 in the <em>Escherichia coli</em> numbering)] were PCR-amplified using our non-degenerate universal primer set of 342F and 806R (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>)<sup><a reveal-id="DSV023C13" data-open="DSV023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and <em>EX Taq HS</em> polymerase (TAKARA BIO, Ohtsu, Japan). More detailed information on the primer set and PCR conditions has been described in our previous publication.<sup><a reveal-id="DSV023C13" data-open="DSV023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> After addition of the sequencing adapters A and B, the amplicons were sequenced using a Roche 454 GS FLX Titanium platform (Roche, Basel, Switzerland) according to the manufacturer's protocol.</p><p>Our previous protocol was also used to select high-quality 16S rRNA gene amplicon sequences (with &gt;350 and &lt;550 bases in lengths) and for their subsequent processing (i.e. alignment to a standard sense strand and removal of primer sequences for pyrosequencing). Sequence clustering of these high-quality reads was conducted using the UCLUST program<sup><a reveal-id="DSV023C14" data-open="DSV023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> version 6.0.307 with identity of ≥97% and query and reference coverage of ≥80%. Chimera clusters detected by the UCHIME program<sup><a reveal-id="DSV023C15" data-open="DSV023C15" class="link link-ref link-reveal xref-bibr">15</a></sup> in <em>de novo</em> mode and its reference mode [with searching the reference Gold Database (DB) (<a class="link link-uri" href="http://drive5.com/uchime/gold.fa">http://drive5.com/uchime/gold.fa</a>)] were removed. Taxonomic assignment of the resulting Roche 454 reads was performed by using the RDP Classifier program<sup><a reveal-id="DSV023C16" data-open="DSV023C16" class="link link-ref link-reveal xref-bibr">16</a></sup> version 2.6 with bootstrap value ≥0.5 against the representative sequences of each 97% sequence cluster that was chosen by the UCLUST program. Unless otherwise stated, the programs for read assemblage and analysis in this study were run under default parameters.</p>                            <h3 scrollto-destination=87197926 id="87197926" class="section-title" data-legacy-id=s2d>2.4 Sequencing of metagenomic DNA using Illumina sequencer and subsequent filtering of low quality and eukaryotic and its viral DNA reads</h3>
<p>The 11 soil metagenomic DNA samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) that were employed for the amplification of 16S rRNA genes and subsequent 454 pyrosequencing were also used for paired-end sequencing by an Illumina GA IIx platform (Illumina, San Diego, CA, USA). The metagenomic DNA was sheared to 200 bp (on average) by a Covaris-S instrument (Covaris, Woburn, MA, USA) to construct the DNA library, and its paired-end sequencing with a maximum read length of 75 bases (at the time of sequencing) was carried out using the base-calling pipeline versions 1.3 and 1.5.</p><p>The Illumina metagenomic reads that were evaluated as being of quality ‘<em>N</em>’ by the chastity filtering in the GA pipeline and those containing more than one nucleotide with the quality flag ‘B’ in the first 60 nucleotides were eliminated. The reads originated from eukaryotes and their viruses were identified as follows. The reads were searched by the BLASTX program<sup><a reveal-id="DSV023C17" data-open="DSV023C17" class="link link-ref link-reveal xref-bibr">17</a></sup> version 2.2.26 with parameters ‘-e 50 -M PAM30’ against the GenBank-nr DB. The taxonomic origin of each read that fulfilled the parameters was next assigned by the highest BLAST scoring hit with identity ≥70% and BLAST bit-score ≥40 against the taxonomy of the database sequence in the NCBI Taxonomy DB (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/pub/taxonomy/">ftp://ftp.ncbi.nih.gov/pub/taxonomy/</a>). The reads assigned to eukaryotic and eukaryotic viral origins were eliminated. The typical taxonomy nomenclature system is not applied to taxonomy of viruses, and their DNA sequences in the GenBank-nr DB are not associated with their taxonomic description. Therefore, use of only GenBank-nr DB was difficult to identify eukaryotic viral DNA reads in our metagenomic samples. To identify as many such reads as possible, our in-house eukaryotic viral protein sequence DB was constructed by manual removal of archaeal and bacterial virus sequences from the NCBI Virus genome DB (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Viruses/all.faa.tar.gz">ftp://ftp.ncbi.nih.gov/genomes/Viruses/all.faa.tar.gz</a>). The resulting in-house database was used for the BLASTX search with the same parameters and thresholds as used for the GenBank-nr DB analysis. When one of the paired-end reads was predicted to be eukaryotic or eukaryotic viral origin, the two reads constituting the paired-reads were eliminated.</p>                            <h3 scrollto-destination=87197932 id="87197932" class="section-title" data-legacy-id=s2e>2.5 Illumina read assembly and protein coding sequences prediction</h3>
<p>Metagenomic reads from each sample were separately assembled using the IDBA-UD program<sup><a reveal-id="DSV023C18" data-open="DSV023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> version 1.1.0 with parameters ‘--mink 20 --maxk 70 --step 5’. The protein coding sequences (CDS) in the contigs were predicted by using the MetaGeneMark program<sup><a reveal-id="DSV023C19" data-open="DSV023C19" class="link link-ref link-reveal xref-bibr">19</a></sup> version 2.10.</p>                            <h3 scrollto-destination=87197936 id="87197936" class="section-title" data-legacy-id=s2f>2.6 Taxonomic and functional assignment of Illumina metagenomic reads</h3>
<p>For taxonomic assignment of the Illumina metagenomic reads, a nucleotide sequence similarity search was conducted using the BWA-MEM program<sup><a reveal-id="DSV023C20" data-open="DSV023C20" class="link link-ref link-reveal xref-bibr">20</a></sup> version 0.7.10 with the strict parameters ‘-k 25 -w 10 -c 100000’ against the draft and complete prokaryote genome sequences in the NCBI DB in December 2013, and the draft sequences of the three <em>Burkholderia</em> strains and one <em>Mycobacterium</em> strain isolated from the polluted soil samples in this study.</p><p>Functional annotation of each read was performed by BLASTX search with parameters ‘-e 50 -M PAM30’ against the KEGG protein sequence DB<sup><a reveal-id="DSV023C21" data-open="DSV023C21" class="link link-ref link-reveal xref-bibr">21</a></sup> that was obtained in February 2014. Taxonomic (phylum, class, order, family, and genus levels) and functional [KEGG Orthology (KO)-level] assignment was achieved for the reads that exhibited the highest BLAST scoring hit with identity ≥70% and BLAST bit-score ≥40. The KO abundance was calculated by dividing the number of reads belonging to the same KO by the median length of proteins that were annotated to the same KO in the KEGG protein DB. The KEGG pathway abundance was calculated by summing the abundances of KOs belonging to one pathway. To compare the abundances of individual KOs and their pathways among different samples, such abundances in each metagenomic sample were normalized by the number of genomes that were estimated as the average abundance of <em>gyrB</em> and 35 universal single-copy genes (USCGs),<sup><a reveal-id="DSV023C22" data-open="DSV023C22" class="link link-ref link-reveal xref-bibr">22</a></sup> each of which is carried as a single-copy form by almost all prokaryotic genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). (<em>gyrB</em> and USCGs are hereafter described as ‘<em>gyrB-</em>added USCGs’ in this paper.) This method for normalizing functional abundances among the samples was applied to all of the BLAST-based functional assignments in this study.</p>                            <h3 scrollto-destination=87197945 id="87197945" class="section-title" data-legacy-id=s2g>2.7 KEGG-based comparison of Illumina reads among different metagenomic samples</h3>
<div class="block-child-p">The time-course changes in the relative abundances of genes at their functional level in metagenomic samples from the control and polluted soils were compared by conducting <em>k</em>-means clustering of KEGG pathway abundances. The pathways that (i) are absent from prokaryotes, (ii) had a normalized number of maximum hits &lt;0.0001 throughout the time course, and (iii) had zero abundance in the samples were not analysed. The variation in abundance of each KEGG pathway at an appropriate time point compared with the abundance at the previous time point, <em>V<sub>p</sub></em>(<em>t</em>), was calculated as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>V</mi><mi>p</mi></msub></mrow><mo stretchy="false" xmlns="">(</mo><mi xmlns="">t</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mrow xmlns=""><msub><mi>log</mi><mn>2</mn></msub></mrow><mstyle xmlns=""><mrow><mfrac><mrow><mrow><msub><mi>K</mi><mi>p</mi></msub></mrow><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mrow><msub><mi>K</mi><mi>p</mi></msub></mrow><mo stretchy="false">(</mo><mi>t</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mfrac></mrow><mo>,</mo></mstyle></math></div></div></div> where <em>K<sub>p</sub></em>(<em>t</em>) is the abundance of pathway <em>p</em> at time point <em>t</em>, and <em>t</em> − 1 is the previous time point. Five <em>V<sub>p</sub></em>(<em>t</em>) scores for each pathway were calculated for <em>t</em> = (1, 3, 6, 12, 24) and <em>t</em> − 1 = (0, 1, 3, 6, 12), and the five <em>V<sub>p</sub></em>(<em>t</em>) scores for every pathway were used for conducting the <em>k</em>-means clustering. To infer the statistically optimal number of clusters for the separate pathways, we used the Calinski–Harabasz (CH) Index.<sup><a reveal-id="DSV023C23" data-open="DSV023C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSV023C24" data-open="DSV023C24" class="link link-ref link-reveal xref-bibr">24</a></sup> This index considers the ratio between inter- and intra-cluster distances. A high value of the CH index suggests the robustness of the clustering by the cluster number. We chose a cluster number with the highest CH index. The <em>k</em>-means clustering and the calculation of the CH indexes were conducted using the R software version 3.1.1 (<a class="link link-uri" href="http://www.r-project.org/">http://www.r-project.org/</a>).</div><p>Overall differences in the taxonomic and functional gene compositions among the different samples were analysed by calculating the 1 – Pearson correlation coefficient of abundances of (i) 97% operational taxonomic unit (OTU), (ii) Genus, (iii) KO, and (iv) KEGG pathway among the samples, and were thereafter conducted through hierarchical clustering with the complete linkage algorithm by using the R software.</p>                            <h3 scrollto-destination=87197951 id="87197951" class="section-title" data-legacy-id=s2h>2.8 Comparison of gene pools among samples using all Illumina reads</h3>
<p>To analyse overall differences in gene pools among the 11 metagenomic samples, all-against-all BLASTN (version 2.2.26) comparison for 121 pairs of samples (11 samples × 11 samples including self-comparisons) was conducted with parameters ‘-e 0.01’. The functional difference between each pair of samples was analysed by calculating the distance D2, which can normalize such differences by using the BLAST bit-scores of the best hit.<sup><a reveal-id="DSV023C25" data-open="DSV023C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSV023C26" data-open="DSV023C26" class="link link-ref link-reveal xref-bibr">26</a></sup> After calculating the D2 values for all pairs of samples, the hierarchical clustering according to the D2 values with the complete linkage algorithm was conducted using the R software.</p>                            <h3 scrollto-destination=87197954 id="87197954" class="section-title" data-legacy-id=s2i>2.9 Construction of in-house protein sequence DB for aromatic compound-degrading enzymes and its use for functional assignment of Illumina metagenomic reads</h3>
<p>The KEGG DB lacks a number of genes that have recently been deposited in public databases, and this is also the case with the genes involved in the upstream parts of microbial aerobic and anaerobic degradation routes for the pollutants. Another major inconvenience is that many enzymes for the aerobic degradation, especially those with dioxygenation activities, are encoded by three to four genes, and the substrate specificities of enzymes are determined mainly by the gene products for the terminal dioxygenase, but not by the remaining ones.<sup><a reveal-id="DSV023C27" data-open="DSV023C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Therefore, we constructed our own in-house DB consisting of amino acid sequences of the enzymes for the following aerobic and anaerobic degradation routes of 3CB, phenanthrene, biphenyl, and carbazole: phenanthrene-phthalate-protocatechuate route, biphenyl-benzoate route, carbazole-anthranilate route, naphthalene-salicylate route, catechol 1,2-and 2,3-dioxygenation route, gentisate route, anaerobic polycyclic aromatic hydrocarbon (naphthalene) route, and anaerobic benzoate and toluene route (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> for the reaction steps in each route). Among the protein sequences available as the relevant enzyme names in the NCBI DB (in January 2014), the sequences accompanied by one or more publications about the bacterial degradation of aromatic compounds were first collected as the initial reference sequences of our in-house DB. Protein sequences in two other in-house DBs for the enzymes for degradation of aromatic compounds that were constructed by Vilchez-Vargas et al.<sup><a reveal-id="DSV023C28" data-open="DSV023C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and Guazzaroni et al.,<sup><a reveal-id="DSV023C29" data-open="DSV023C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and the protein sequences deposited with relevant KOs in the KEGG DB were also integrated into our DB. The reference sequence collection thus constructed was BLASTP-searched (version 2.2.26) with parameters ‘-e 1e-8 -b 10 -v 10’ and threshold identity ≥90%, query and reference sequence coverage ≥70% against the GenBank-nr DB in April 2014. The protein sequences that fulfilled these stringent parameters were incorporated into our final in-house DB (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). For the multisubunit enzymes, only the subunits with clearly determined substrate specificities were usually deposited in our database.</p><p>All the high-quality Illumina metagenomic reads were BLASTX-searched with parameters ‘-e 50 -M PAM30’ against our in-house DB, and the reads with identity ≥70% and BLAST bit-score ≥40 in the BLASTX results were collected as candidate-hit reads. These reads were next subjected to BLASTX search against the GenBank-nr DB with parameters ‘-e 50 -M PAM30’, and the reads that were the best hits to the sequences in the final in-house DB were further used for their functional assignment. The abundance of each gene for an aromatic compound-degrading enzyme in a metagenomic sample was normalized by (i) the amino acid length of the enzyme and (ii) the average abundance of the <em>gyrB</em>-added USCGs.</p>                            <h3 scrollto-destination=87197960 id="87197960" class="section-title" data-legacy-id=s2j>2.10 Analysis of phage genes and phage genome-derived contigs</h3>
<p>Illumina metagenomic reads putatively from phage origins were surveyed using the BLASTX search with parameters ‘-e 50 -M PAM30’ against the ACLAME (A CLAssification of Mobile genetic Elements) phage protein sequence DB<sup><a reveal-id="DSV023C30" data-open="DSV023C30" class="link link-ref link-reveal xref-bibr">30</a></sup> version 0.4. The abundance of each phage gene in each metagenomic sample was normalized by (i) the length of the phage gene, and (ii) the average abundances of the <em>gyrB</em>-added USCGs, which were calculated from the KEGG analysis.</p><p>All CDS found in all assembled contigs were BLASTP-searched with the parameter ‘-e 0.001’ against ‘Phage Orthologous Groups (POGs)’ DB<sup><a reveal-id="DSV023C31" data-open="DSV023C31" class="link link-ref link-reveal xref-bibr">31</a></sup> to find out the contigs from phage genomes. All the Illumina reads that were allocated to such contigs were obtained using the Bowtie 2 program<sup><a reveal-id="DSV023C32" data-open="DSV023C32" class="link link-ref link-reveal xref-bibr">32</a></sup> version 2.1.0. All such reads from the 11 metagenomic samples were assembled using the IDBA-UD program with parameters ‘--mink 20 --maxk 70 --step 5’ to construct more accurate phage gene-containing contigs. In this study, the ‘putative contigs from the phage genome’ were defined as those that (i) were &gt;2 kb in length, (ii) belonged to the phage genome-related contigs, (iii) contained at least one CDS with an amino acid sequence identity ≥40% against proteins in the POGs DB, and (iv) contained at least one phage genome-derived region that could be identified using the PHAST (PHAge Search Tool) program.<sup><a reveal-id="DSV023C33" data-open="DSV023C33" class="link link-ref link-reveal xref-bibr">33</a></sup> This program was also used to analyse their host taxa in which the phages can proliferate, and to evaluate the completeness of the phage genome. The abundance of the phage genome-derived contig DNA in each metagenomic sample was calculated by the numbers of hit reads counted by BLASTN search (100% identity and 100% query coverage) of Illumina metagenomic reads against the phage contigs. The abundance was normalized by the contig length and the numbers of <em>gyrB</em>-added USCGs.</p><p>Time-course change in abundance of the longest 44.5-kb phage genome-derived contig (Contig70-3) in metagenomic samples was investigated by long-range PCR amplification. Primer sets of PCR for scanning the phage genome were designed using the GenoFrag software<sup><a reveal-id="DSV023C34" data-open="DSV023C34" class="link link-ref link-reveal xref-bibr">34</a></sup> version 2.1: the primer sets of 1–4 (1F-1R, 2R-2R, 3F-3R, and 4F-4R, respectively) for checking the size of the genome, and the primer set 5 (5F-5R) for checking the presence of circular or concatemer form of phage genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The PCR reaction using PrimeSTAR GXL polymerase (TAKARA BIO) and the soil metagenomic DNA samples as the templates was performed under the condition of 30 cycles of 98°C for 10 s, 55°C for 15 s, and 68°C for 10 min. The sizes of the PCR products were determined by 0.8% agarose gel electrophoresis. A quantitative PCR of the phage genome was performed using the primer set 6, 6F-6R (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), SYBR Premix <em>Ex Taq</em> (TAKARA BIO), and a DNA engine OpticonTM2 system (MJ Research, Waltham, MA, USA).</p>                            <h2 scrollto-destination=87197969 id="87197969" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87197972 id="87197972" class="section-title" data-legacy-id=s3a>3.1 Overview and preliminary experiments</h3>
<p>Accidental pollution of natural environments by harmful aromatic compounds has often occurred by a mixture of various compounds [e.g. various polycyclic aromatic hydrocarbons (PAHs) in petroleum and intermediate products, final products, and by-products from chemical plants], which range from biodegradative to highly persistent compounds. In order to investigate the microbial community-based degradation of such pollutants in natural environments as a model system, a farm soil sample from Ehime, Japan (see Section 2) was artificially polluted with a mixture of four aromatic compounds, 3CB (as a relatively biodegradable monocyclic and chlorinated compound) and carbazole, phenanthrene, and biphenyl (as much more recalcitrant compounds; a nitrogen-containing heterocyclic compound and PAHs, respectively), each at a concentration of ∼1 mM. Each 200-g portion of the polluted and control soil samples was transferred to a sterilized glass pot with a loose lid, generating many experimental pots. These pots were incubated at 28°C in the dark for up to 24 weeks. At an appropriate time point, all the soil harvested from each pot were used to investigate the decrease of pollutants and the number of culturable heterotrophs and to analyse the metagenome of soil microbiota. 3CB began to decrease immediately after the incubation and disappeared within the first 3 weeks. In contrast, carbazole and the two PAHs remained or decreased very slowly during these 3 weeks, then decreased rapidly and constantly, and disappeared at Week 12 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The numbers of colony-formable heterotrophs in the polluted soil samples during the 24-week incubation were similar with those in the control soil samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). On the basis of our preliminary survey of time-course changes in the taxonomic compositions of soil microbial communities in the pots (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary SN1</a></span>), 11 soil metagenomic DNA samples were prepared at the time point just before the pollution (at Week 0) (designated 0 hereafter) and at five subsequent time points (at Weeks 1, 3, 6, 12, and 24) from the control (1C, 3C, 6C, 12C, and 24C) and polluted (1, 3, 6, 12, and 24 M) soils. These samples were used for (i) the PCR amplification of 16S rRNA gene fragments and their subsequent sequencing by the Roche 454 system to investigate the time-course changes in the taxonomic compositions, and (ii) paired-end metagenomic sequencing by the Illumina system to investigate the time-course changes in the functional and taxonomic compositions of gene pools.</p>                            <h3 scrollto-destination=87197976 id="87197976" class="section-title" data-legacy-id=s3b>3.2 Time-course taxonomic changes of microbial community members in soils</h3>
<p>Our pyrosequencing of PCR amplicons covering the V3–V4 regions of 16S rRNA genes from the 11 metagenomic samples yielded a total of 318,982 high-quality reads (average = 28,998 ± 3,311 per sample) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Use of the RDP Classifier led to the successful taxonomical assignment of 95.6 and 78.7% (on average) of the reads to phyla and genera, respectively, and the proportions of unassigned taxa at the phylum level were considerably low in the communities from the polluted soil at Weeks 1, 3, and 6 (Fig. <a reveal-id="DSV023F1" data-open="DSV023F1" class="link link-reveal link-table xref-fig">1</a>A and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The number of assigned genera in each sample was ∼250, and the proportion of <em>Archaea</em> in each of the 11 communities was &lt;0.1%. The soil microbial community at Week 0 was, similarly to other reported soil microbial communities,<sup><a reveal-id="DSV023C10" data-open="DSV023C10" class="link link-ref link-reveal xref-bibr">10</a></sup> dominated by the phyla <em>Acidobacteria</em> (especially subgroups <em>Gp4</em> and <em>Gp6</em>), <em>Verrucomicrobia</em> (especially genus <em>incertae sedis Spartobacteria</em>), and <em>Proteobacteria</em> (Fig. <a reveal-id="DSV023F1" data-open="DSV023F1" class="link link-reveal link-table xref-fig">1</a>B). The community compositions in the control soil changed slightly during the 24-week incubation; the relative abundance of the genus <em>Methylophilus</em> (in <em>Proteobacteria</em>) increased at Weeks 1 and 3, and thereafter decreased. The increase in this methanol-utilizing genus was possibly due to a trace amount of residual methanol in the control soil; to pollute the soil uniformly with 3CB and the remaining very water-insoluble aromatic compounds, the four compounds were dissolved in methanol, which was thereafter removed as much as possible, and the same process without the pollutants was employed for preparation of the control soil (see Section 2). On the other hand, very drastic and complicated changes in the microbial community members, especially those in <em>Proteobacteria</em>, were observed in the polluted soil. Time-course changes in the relative abundance of the top 30 abundant genera (including genera <em>incertae sedis</em>) were further investigated by hierarchical clustering analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), giving rise to four clusters (Clusters 1–4) on the basis of co-occurrence patterns. The genera that were dominant members at Week 0 but became minor ones immediately after the pollution belonged to Cluster 1, and only one-third of them (represented by <em>Acidobacteria Gp4</em> and <em>Spartobacteria</em>) regained their dominance after Week 12. The genera belonging to Cluster 2 (e.g. <em>Sphingomonas, Bradyrhizobium</em>, and subgroup <em>Gp1</em> in <em>Acidobacteria</em>) increased in their relative abundances after the disappearance of carbazole and the two PAHs. The remaining two clusters consisted of the genera that transiently increased in their relative abundances at Week 1 (e.g. <em>Burkholderia</em> and <em>Pseudomonas</em> in Cluster 4) and at Weeks 3 and 6 (e.g. <em>Dyella</em>, <em>Dongia</em>, <em>Mycobacterium</em>, <em>TM7</em>, and several anaerobes in Cluster 3). The genera in the last two clusters include various well-characterized aromatic compound-degrading strains,<sup><a reveal-id="DSV023C35" data-open="DSV023C35" class="link link-ref link-reveal xref-bibr">35</a></sup> suggesting their direct or indirect involvement in the degradation of pollutants. The increase in relative abundances of 16S rRNA genes in the soil DNA samples did not definitively confirm the proliferation of their host cells in the soil environment. Since the three genera, <em>Burkholderia</em>, <em>Pseudomonas</em>, and <em>Mycobacterium</em>, were noteworthy taxa in the Illumina analysis of the metagenome (see below), the absolute abundances of their 16S rRNA genes in the control and polluted soil metagenomic samples were measured by quantitative PCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). The copy number of the 16S rRNA gene from each genus thus determined was in good agreement (Pearson correlations &gt;0.90, <em>P</em>-value &lt;0.01) with the relative abundance of the corresponding genus that was estimated by the assignment of 16S rRNA genes from the Roche 454 reads. This indicates that at least the three genera indeed proliferated in the polluted soil. </p>                        <a id="87197981" scrollto-destination="87197981"></a>
<div id="DSV023F1" data-id="DSV023F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02301.png?Expires=2147483647&Signature=JyNe-v10zOZxtgOxopqTyU9m2c~MT5Lt2Rs78foH3GJpHsg~VhQgQguRQkrnLAjTCChk-CqljqUFctQBtcXQgfPShMWF4rAc~LEoAh~4ckgSWV8JepPbL71iTFlc~cDR2a1L3N~v5PhsnzeAsEMO7WwM66bxcosbWgVx8xio3Tz5W5SH4nqIMjsIRtzHM9~2dj2lxoXNtve2AuLzAIkmHyrQfPBdxaHpxyQjmOyB87mnLeOSBX2kzpoUTqDvbmmN0TvbymhH3UbibV~1TUsigzejVBzw2Sm1n5~hfNx9QFuLDRcGM8bB8puF7~v1MpsnC924pyMQl1il5vvK6vGEVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylum- and genus-level taxonomical succession of microbial communities in control and polluted soils. PCR amplicons of V3–V4 regions in the 16S rRNA genes from metagenomic samples were pyrosequenced, and taxonomically assigned by the RDP classifier (see the text for details). (A) Phylum-level succession. (B) Genus-level succession. C and M: metagenomic samples from the control and polluted soils, respectively. The numerals before C and M are the weeks after the pollution. Only the top 15 prokaryotic genera are shown for simplicity. Taxa with asterisks are genera incertae sedis." path-from-xml="dsv02301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197981/dsv02301.tiff" path-from-xml="dsv02301.tiff" target="_blank">View large</a><a section="87197981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02301.png?Expires=2147483647&Signature=X5l4SK4HxtrycPT~Ze9~r1gMqBBtO78nT6uNIvnbUKk21nEFUZDONYbNPgVwiHfEtItWBromF0PYwABmetNOyYBlmqCOWtz9yZW4FNPh9oCYzpJJ9IYa76eGQWwLQmYKwqUd9LEM1bJYGg1ZpzOCKuia~1b4GHAzYK8GPSoUWqRT6ENoYYjl~BkzRyXNUDwJQYI2liTD-jY3V6tfjeQjgJsf7iUIuSXZBAUQZ97j-EROlRMbvOwlOkX2-Y0gkuohpnwT9o2eRLh8E5RvZkMRxpgrVn4de1oPjPu5RS72xxSD7PQS3sGvcuom0lxrFPlM2bLcBAt4SpXPLrkSp7rKZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197981&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylum- and genus-level taxonomical succession of microbial communities in control and polluted soils. PCR amplicons of V3–V4 regions in the 16S rRNA genes from metagenomic samples were pyrosequenced, and taxonomically assigned by the RDP classifier (see the text for details). (A) Phylum-level succession. (B) Genus-level succession. C and M: metagenomic samples from the control and polluted soils, respectively. The numerals before C and M are the weeks after the pollution. Only the top 15 prokaryotic genera are shown for simplicity. Taxa with asterisks are genera <em>incertae sedis</em>.</p></div></div><div content-id="DSV023F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02301.png?Expires=2147483647&Signature=JyNe-v10zOZxtgOxopqTyU9m2c~MT5Lt2Rs78foH3GJpHsg~VhQgQguRQkrnLAjTCChk-CqljqUFctQBtcXQgfPShMWF4rAc~LEoAh~4ckgSWV8JepPbL71iTFlc~cDR2a1L3N~v5PhsnzeAsEMO7WwM66bxcosbWgVx8xio3Tz5W5SH4nqIMjsIRtzHM9~2dj2lxoXNtve2AuLzAIkmHyrQfPBdxaHpxyQjmOyB87mnLeOSBX2kzpoUTqDvbmmN0TvbymhH3UbibV~1TUsigzejVBzw2Sm1n5~hfNx9QFuLDRcGM8bB8puF7~v1MpsnC924pyMQl1il5vvK6vGEVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylum- and genus-level taxonomical succession of microbial communities in control and polluted soils. PCR amplicons of V3–V4 regions in the 16S rRNA genes from metagenomic samples were pyrosequenced, and taxonomically assigned by the RDP classifier (see the text for details). (A) Phylum-level succession. (B) Genus-level succession. C and M: metagenomic samples from the control and polluted soils, respectively. The numerals before C and M are the weeks after the pollution. Only the top 15 prokaryotic genera are shown for simplicity. Taxa with asterisks are genera incertae sedis." path-from-xml="dsv02301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197981/dsv02301.tiff" path-from-xml="dsv02301.tiff" target="_blank">View large</a><a section="87197981" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02301.png?Expires=2147483647&Signature=X5l4SK4HxtrycPT~Ze9~r1gMqBBtO78nT6uNIvnbUKk21nEFUZDONYbNPgVwiHfEtItWBromF0PYwABmetNOyYBlmqCOWtz9yZW4FNPh9oCYzpJJ9IYa76eGQWwLQmYKwqUd9LEM1bJYGg1ZpzOCKuia~1b4GHAzYK8GPSoUWqRT6ENoYYjl~BkzRyXNUDwJQYI2liTD-jY3V6tfjeQjgJsf7iUIuSXZBAUQZ97j-EROlRMbvOwlOkX2-Y0gkuohpnwT9o2eRLh8E5RvZkMRxpgrVn4de1oPjPu5RS72xxSD7PQS3sGvcuom0lxrFPlM2bLcBAt4SpXPLrkSp7rKZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197981&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylum- and genus-level taxonomical succession of microbial communities in control and polluted soils. PCR amplicons of V3–V4 regions in the 16S rRNA genes from metagenomic samples were pyrosequenced, and taxonomically assigned by the RDP classifier (see the text for details). (A) Phylum-level succession. (B) Genus-level succession. C and M: metagenomic samples from the control and polluted soils, respectively. The numerals before C and M are the weeks after the pollution. Only the top 15 prokaryotic genera are shown for simplicity. Taxa with asterisks are genera <em>incertae sedis</em>.</p></div></div><p>Analysis of the changes in abundance of the top 30 genera throughout the 24-week time course did not conclusively reveal whether the microbial community in the polluted soil was still fluctuating at Week 24. However, our analysis of overall differences in the 16S rRNA gene compositions (at the 97% OTU level) among the 11 samples revealed that the community compositions of the polluted soil at Weeks 12 and 24 were (i) more similar to that at Week 0 than those at Weeks 1, 3, and 6, but (ii) still different from that at Week 0 (Fig. <a reveal-id="DSV023F2" data-open="DSV023F2" class="link link-reveal link-table xref-fig">2</a>A). This implies that the microbial taxonomic compositions in the polluted soil tended to slowly, but not completely, return to that at Week 0. Such a gradual mode of return was more clearly observed in the case of the control soil. The resilience of microbial community compositions after environmental changes has been observed ubiquitously in various other environments, and the return to original compositions sometimes requires much longer periods (e.g. more than several years).<sup><a reveal-id="DSV023C1" data-open="DSV023C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV023C2" data-open="DSV023C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV023C36" data-open="DSV023C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                        <a id="87197986" scrollto-destination="87197986"></a>
<div id="DSV023F2" data-id="DSV023F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02302.png?Expires=2147483647&Signature=tiaXIycNW11LFzQugJ3B3DQp90Jn213AoOcgxNOJqkoBU78lCGq4lFVF2cs6lq29O9loAIdi6cpRpQXhC9NgvrM3S74pazXzk8S6F6Q2Igi85sOSzGY1WQU4IN7K3AxT5ReT6pccWy1VbKoknoeq7MsvQlfE4HjIc95vjo1LTY15T3ltLuY-4cjSzkFrt7NYHwt1fHXnQSWJMB0j5pZzjyeC50R9FReiZGAljzHUU1-9qgpputhbNnYpA82J8ntMXbLlNFH8CEPJLkvwKdl4ofq5SE0fv5fV4mfWLTth36Q~zoCz04SQrfhCgwQiyYUHziwiO3xdA~-lNR~thg4oWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering analysis of taxonomic and functional compositions of 11 soil metagenomic samples. Clustering of (A) taxonomic compositions of 97% OTUs of 16S rRNA gene amplicons, and (B) functional compositions of genes for KEGG KO-assigned proteins was performed by comparison of the Euclidean distances. See the text for details." path-from-xml="dsv02302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197986/dsv02302.tiff" path-from-xml="dsv02302.tiff" target="_blank">View large</a><a section="87197986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02302.png?Expires=2147483647&Signature=HV7DEoCsP1b6eE0cgUKQEG-6dWXoj7Lm3SU4LTMMV2VU8dQVDJRerEJfsWQ-7dOHQu5psHwA0hDBdfunCQp-WgUZIUA1wDVmzex0WMayVO4U121LOO67bY84P29Tnc1OeesZCq-Xxx6YFivp~pvpHZPlyZbaWG6HP5lnRaNC7~9KNtIBOUzQ65Czp6532n6Wm6b5wQt7HtTmcxQxYO5MK3No0ru-PxbKZsWHCwy4CFhcTRFjVhZf8DY0QeJ-pIAQlzElBQwMemWTME9dHuqz9Pcn9GJ8hDypJDAp7VXY7lq7dseBR~2M0IUZ4qwxL8Mgx-VRXIcrqfTQyAkznaMG~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197986&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering analysis of taxonomic and functional compositions of 11 soil metagenomic samples. Clustering of (A) taxonomic compositions of 97% OTUs of 16S rRNA gene amplicons, and (B) functional compositions of genes for KEGG KO-assigned proteins was performed by comparison of the Euclidean distances. See the text for details.</p></div></div><div content-id="DSV023F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02302.png?Expires=2147483647&Signature=tiaXIycNW11LFzQugJ3B3DQp90Jn213AoOcgxNOJqkoBU78lCGq4lFVF2cs6lq29O9loAIdi6cpRpQXhC9NgvrM3S74pazXzk8S6F6Q2Igi85sOSzGY1WQU4IN7K3AxT5ReT6pccWy1VbKoknoeq7MsvQlfE4HjIc95vjo1LTY15T3ltLuY-4cjSzkFrt7NYHwt1fHXnQSWJMB0j5pZzjyeC50R9FReiZGAljzHUU1-9qgpputhbNnYpA82J8ntMXbLlNFH8CEPJLkvwKdl4ofq5SE0fv5fV4mfWLTth36Q~zoCz04SQrfhCgwQiyYUHziwiO3xdA~-lNR~thg4oWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Hierarchical clustering analysis of taxonomic and functional compositions of 11 soil metagenomic samples. Clustering of (A) taxonomic compositions of 97% OTUs of 16S rRNA gene amplicons, and (B) functional compositions of genes for KEGG KO-assigned proteins was performed by comparison of the Euclidean distances. See the text for details." path-from-xml="dsv02302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197986/dsv02302.tiff" path-from-xml="dsv02302.tiff" target="_blank">View large</a><a section="87197986" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02302.png?Expires=2147483647&Signature=HV7DEoCsP1b6eE0cgUKQEG-6dWXoj7Lm3SU4LTMMV2VU8dQVDJRerEJfsWQ-7dOHQu5psHwA0hDBdfunCQp-WgUZIUA1wDVmzex0WMayVO4U121LOO67bY84P29Tnc1OeesZCq-Xxx6YFivp~pvpHZPlyZbaWG6HP5lnRaNC7~9KNtIBOUzQ65Czp6532n6Wm6b5wQt7HtTmcxQxYO5MK3No0ru-PxbKZsWHCwy4CFhcTRFjVhZf8DY0QeJ-pIAQlzElBQwMemWTME9dHuqz9Pcn9GJ8hDypJDAp7VXY7lq7dseBR~2M0IUZ4qwxL8Mgx-VRXIcrqfTQyAkznaMG~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197986&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Hierarchical clustering analysis of taxonomic and functional compositions of 11 soil metagenomic samples. Clustering of (A) taxonomic compositions of 97% OTUs of 16S rRNA gene amplicons, and (B) functional compositions of genes for KEGG KO-assigned proteins was performed by comparison of the Euclidean distances. See the text for details.</p></div></div>                            <h3 scrollto-destination=87197990 id="87197990" class="section-title" data-legacy-id=s3c>3.3 Time-course changes of microbial gene pools in soils</h3>
<p>The 11 soil DNA samples used for the pyrosequencing of 16S rRNA genes were also used for the metagenomic sequencing by the Illumina platform with a maximum read length of 75 bases (at the time of sequencing). The total number of reads from the 11 samples was 413 million (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The total number of high-quality reads with &gt;60 bases in lengths was ∼125 million (ranging from 7.3- to 14.4-million reads in each of the 11 samples), generating ∼9.3 Gb of sequences. Taxonomic and functional assignments of the Illumina-based metagenomic sequences in most other studies have been carried out after the assembly of reads.<sup><a reveal-id="DSV023C21" data-open="DSV023C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSV023C37" data-open="DSV023C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Therefore, our Illumina reads were assembled using the IDBA-UD software<sup><a reveal-id="DSV023C18" data-open="DSV023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> in order to obtain the contigs. The contig N50 median lengths from the 11 samples were very short (331–683 bases) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>), and only 205 and 9 contigs of &gt;5 kb in length were obtained from the polluted and control soil samples, respectively. Our CDS prediction by the MetaGeneMark program generated a total of 3,333 CDS from the 214 contigs, and the subsequent BLASTP search against the GenBank-nr DB revealed that 3,095 CDS showed &lt;50% identities (at the amino acid sequence level) with the functionally known genes or were assigned as hypothetical CDS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). These results were consistent with the diverse taxonomic compositions of microbiota in our soil samples. Among the 63 CDS (32 and 31 CDS from polluted and control soils, respectively) that showed &gt;70% identities (at the amino acid sequence level) with functionally annotated genes in the GenBank-nr DB, 15 CDS from the polluted soil samples were assigned to phage origins (see below for more details about the analysis of phage-inferred contigs), and the remaining 17 CDS were considered to have non-phage origins (e.g. genes for nucleotide metabolisms and ABC transporters). Most contigs from the control soil samples were probably the genome fragments derived from methylotrophs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), and appeared not to encode the functions either directly or indirectly involved in the degradation of pollutants.</p><p>Because we obtained a huge number of very short contigs and only a small number of long contigs, the Illumina high-quality short reads were used for subsequent analysis. To evaluate the accuracy of the assignment of such short Illumina reads, a computer simulation experiment was conducted. The simulation of artificial 60-base ‘Illumina’ reads generated from the Sanger read data of a Minnesota soil microbiome<sup><a reveal-id="DSV023C9" data-open="DSV023C9" class="link link-ref link-reveal xref-bibr">9</a></sup> revealed that identical assignments between the artificial Illumina data reads and their parental Sanger data reads at the phylum and KO levels were obtained with probabilities of 72 and 92%, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). These results reinforced the validity of our assignment of metagenomic sequences by using the Illumina short reads.</p><p>Taxonomic and functional assignment of the Illumina high-quality reads was performed by BLASTX search using the KEGG, GenBank-nr, and ACLAME DBs. The BLASTX search using the KEGG DB under the conditions of identity ≥70% and bit-score ≥40 led to the successful assignment of putative gene functions for 29% of the reads on average (ranging from 24 to 35%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span> for gene abundance of each KO-assigned protein). Our analysis of overall differences in the relative abundances of genes for KO-assigned proteins among the 11 metagenomic DNA samples showed that the gene compositions at Weeks 1, 3, and 6 in the polluted soil samples formed a cluster very different from another cluster that was organized by the compositions from the remaining samples (Fig. <a reveal-id="DSV023F2" data-open="DSV023F2" class="link link-reveal link-table xref-fig">2</a>B), indicating that the gene pools changed drastically in response to the chemical spike. Several metagenomic analyses of microbes in other real-world polluted soils also reported such changes in the gene pools.<sup><a reveal-id="DSV023C29" data-open="DSV023C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSV023C38" data-open="DSV023C38" class="link link-ref link-reveal xref-bibr">38</a></sup> The gene compositions at Weeks 12 and 24 in our polluted samples were still different from that at Week 0 and those in the control samples. The overall relationship among the compositions of the genes for KO-assigned proteins (by the Illumina sequencing) in the 11 metagenomic samples was shown to have some similarity to the overall relationship among the compositions of 16S rRNA genes (by pyrosequencing) (Fig. <a reveal-id="DSV023F2" data-open="DSV023F2" class="link link-reveal link-table xref-fig">2</a>). This implies that not only the taxonomic but also the gene compositions of the polluted soil microbiota tended to slowly return to those at Week 0. All the high-quality Illumina reads, including the functionally unassigned ones, were further compared among the 11 metagenomic DNA samples by the all-against-all BLASTN analysis, and subsequent clustering analysis exhibited that the gene compositions at Weeks 12 and 24 in the polluted soil were still distinct from that at Week 0 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The difference in the clustering patterns between all the reads and only the KO-assigned reads was likely ascribable to the biased property of KEGG DB, since this DB was constructed by using mainly the organisms for which complete genome sequences were available.</p><p>The time-course changes in the taxonomic and functional diversities of metagenomic samples were investigated by the Shannon-Wiener indices. Although the pollution of soil led to a decrease in the diversity index of the taxonomic compositions of the microbial community (Fig. <a reveal-id="DSV023F3" data-open="DSV023F3" class="link link-reveal link-table xref-fig">3</a>A), it also gave rise to a drastic increase in the diversity of the genes for KO-assigned proteins (Fig. <a reveal-id="DSV023F3" data-open="DSV023F3" class="link link-reveal link-table xref-fig">3</a>B). This increase was most plausibly ascribable to the two most proliferating genera, <em>Burkholderia</em> and <em>Pseudomonas</em>, in the early weeks after the pollution. These bacteria have large genome sizes, and are well recognized to carry a number of genes apparently unique to these genera, especially for the catabolic genes for various aromatic compounds.<sup><a reveal-id="DSV023C39" data-open="DSV023C39" class="link link-ref link-reveal xref-bibr">39</a>–<a reveal-id="DSV023C41" data-open="DSV023C41" class="link link-ref link-reveal xref-bibr">41</a></sup> A recent comparative metagenomic analysis of different microbiota from various tallgrass prairie soils indicated a positive correlation between the taxonomic and functional diversities of microbiomes.<sup><a reveal-id="DSV023C42" data-open="DSV023C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The difference between our closed soil microcosm samples and the real-world prairie soils may have been related to differences in environmental factors (chemical disturbance and physicochemical factors, respectively) or differences in the biological factors of the resident microbes. </p>                        <a id="87197999" scrollto-destination="87197999"></a>
<div id="DSV023F3" data-id="DSV023F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02303.png?Expires=2147483647&Signature=njo2RPcz-fkGbjG7JX4krkIsXl5qy9ePaElpXIPo~V7OKXCIFESBMV2ez1XCE4~ggleV8RL--W52lLoInANtWdMOggJYlZF4BpyTH2pRFktboWIh-NCyB2UZRnw7sK0MbHe5FnVCiUBzbNBBJoven93NEUZu4Na3b4~joN4QryQsYNj-srSf4a~weHnY~PjNSN54vVQmmPEpw5u961gUerFALdotLh3Ntl6FStJTqHHt5uh9nWZzZYxxfv2u7iRYnVqTaGVcluxN9zRUFpk8uS96KUUU7buVdwSAqTLOtSBBnsCHMjFsMVtSXG9nuqjXkX8mpE5M9-nUJF20hnZx5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in taxonomic and functional diversity of metagenomes in control and polluted soils. (A) Taxonomic diversity. (B) Functional diversity. Open and closed circles: control and polluted soils, respectively. The Shannon-Wiener indices (H′) for taxonomic and functional diversities of the metagenomic samples were calculated from the compositions of 97% OTUs of the 16S rRNA gene amplicons and the compositions of genes for KO-assigned proteins, respectively." path-from-xml="dsv02303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197999/dsv02303.tiff" path-from-xml="dsv02303.tiff" target="_blank">View large</a><a section="87197999" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02303.png?Expires=2147483647&Signature=3il1YB9x7iRWAvUbk0C7t80cgzmMCrFjTLchX-2OF5vx-tFOEGO5toog2s-qMtq~hzvNtbAOxyy6cdQKigTEd6PNsPuS1DsPfpFyqzZqHzzf66aPLiKYhfh2sDb8yL-6RMWFgSpjOjrSx2u8B-gLltOkpHC1I4WpTVYMMwKx4ysYjc82KYxshSQjdYskRNpoQ-diJ1xVrRfpaI3wwzMGevizLyWPlzlwT39NgXUMp5OH3kKeRfai23tKpKrZlXUN5fElZ7x7qtYBMfe4uCemUZfJkxg2nMl-Y~OI5W~PKcu6KV0b4fZ8ZCiO79UsLRJU4Jv0HdT7nr8KpBcVJFQ48w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197999&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in taxonomic and functional diversity of metagenomes in control and polluted soils. (A) Taxonomic diversity. (B) Functional diversity. Open and closed circles: control and polluted soils, respectively. The Shannon-Wiener indices (H′) for taxonomic and functional diversities of the metagenomic samples were calculated from the compositions of 97% OTUs of the 16S rRNA gene amplicons and the compositions of genes for KO-assigned proteins, respectively.</p></div></div><div content-id="DSV023F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02303.png?Expires=2147483647&Signature=njo2RPcz-fkGbjG7JX4krkIsXl5qy9ePaElpXIPo~V7OKXCIFESBMV2ez1XCE4~ggleV8RL--W52lLoInANtWdMOggJYlZF4BpyTH2pRFktboWIh-NCyB2UZRnw7sK0MbHe5FnVCiUBzbNBBJoven93NEUZu4Na3b4~joN4QryQsYNj-srSf4a~weHnY~PjNSN54vVQmmPEpw5u961gUerFALdotLh3Ntl6FStJTqHHt5uh9nWZzZYxxfv2u7iRYnVqTaGVcluxN9zRUFpk8uS96KUUU7buVdwSAqTLOtSBBnsCHMjFsMVtSXG9nuqjXkX8mpE5M9-nUJF20hnZx5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in taxonomic and functional diversity of metagenomes in control and polluted soils. (A) Taxonomic diversity. (B) Functional diversity. Open and closed circles: control and polluted soils, respectively. The Shannon-Wiener indices (H′) for taxonomic and functional diversities of the metagenomic samples were calculated from the compositions of 97% OTUs of the 16S rRNA gene amplicons and the compositions of genes for KO-assigned proteins, respectively." path-from-xml="dsv02303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197999/dsv02303.tiff" path-from-xml="dsv02303.tiff" target="_blank">View large</a><a section="87197999" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02303.png?Expires=2147483647&Signature=3il1YB9x7iRWAvUbk0C7t80cgzmMCrFjTLchX-2OF5vx-tFOEGO5toog2s-qMtq~hzvNtbAOxyy6cdQKigTEd6PNsPuS1DsPfpFyqzZqHzzf66aPLiKYhfh2sDb8yL-6RMWFgSpjOjrSx2u8B-gLltOkpHC1I4WpTVYMMwKx4ysYjc82KYxshSQjdYskRNpoQ-diJ1xVrRfpaI3wwzMGevizLyWPlzlwT39NgXUMp5OH3kKeRfai23tKpKrZlXUN5fElZ7x7qtYBMfe4uCemUZfJkxg2nMl-Y~OI5W~PKcu6KV0b4fZ8ZCiO79UsLRJU4Jv0HdT7nr8KpBcVJFQ48w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197999&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in taxonomic and functional diversity of metagenomes in control and polluted soils. (A) Taxonomic diversity. (B) Functional diversity. Open and closed circles: control and polluted soils, respectively. The Shannon-Wiener indices (H′) for taxonomic and functional diversities of the metagenomic samples were calculated from the compositions of 97% OTUs of the 16S rRNA gene amplicons and the compositions of genes for KO-assigned proteins, respectively.</p></div></div><p>In order to investigate what types of genes were enriched and correlated with one another during the drastic change in the gene compositions mentioned above, time-course variation patterns of KEGG pathway-level gene abundances (each of which was calculated by summing up the abundances of all genes for KO-assigned proteins in one pathway) were analysed by conducting <em>k</em>-means clustering. This analysis led to categorization of the pathway-level variation patterns into two clusters, designated M1 and M2 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12 and Fig. S7</a></span>); while the pathway-level gene abundances for cluster M1 apparently remained constant during the 24-week incubation, those for cluster M2 oscillated drastically (a drastic increase at Week 1 followed by a rapid decrease at Week 3, and a slight re-increase at Week 6) and were well correlated with the degradation events of the pollutants. Of the 146 prokaryote-derived pathways (see Section 2), 100 pathways that include many core metabolic pathways were classified into cluster M1. The finding that there were no apparent variations in the abundance of genes for the core metabolic pathways in spite of the decrease in taxonomic diversity (Fig. <a reveal-id="DSV023F3" data-open="DSV023F3" class="link link-reveal link-table xref-fig">3</a>A) might have been due to the functional redundancy of the genes for these common core pathways<sup><a reveal-id="DSV023C2" data-open="DSV023C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV023C43" data-open="DSV023C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSV023C44" data-open="DSV023C44" class="link link-ref link-reveal xref-bibr">44</a></sup> that were shared by the soil microbial members at Week 0 and the members at the weeks after the pollution. These results are in good agreement with other metagenomic studies that have addressed the degree of functional redundancy among the different human-associated microbial community compositions.<sup><a reveal-id="DSV023C3" data-open="DSV023C3" class="link link-ref link-reveal xref-bibr">3</a></sup> The remaining 46 pathways belonged to cluster M2, which contains most of the degradation pathways of aromatic compounds (e.g. benzoate) as well as the pathways for the catabolism of aromatic amino acids and cell surface-related functions (i.e. cell motility, ABC-type transporters, and various secretion systems). The apparently synchronous variation patterns between the genes for the pollutant-degradation pathways and those for other functional groups (e.g. cell surface-related functions) in cluster M2 in the polluted soil might have been due to the genome structures of pollutant-degrading prokaryotes (and non-degrading and co-resident prokaryotes) that also possess the genes for the latter groups of pathways. In the control soil, the abundance of the genes that were assigned to the M2 cluster in the polluted soil oscillated slightly with the different patterns (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13 and Fig. S7</a></span>).</p>                            <h3 scrollto-destination=87198004 id="87198004" class="section-title" data-legacy-id=s3d>3.4 Time-course changes of microbial pollutant-degrading genes in soils</h3>
<p>The change in the gene pools directly involved in the degradation of the pollutants was investigated using our in-house DB, which includes many amino acid sequences for the enzymes deposited recently in public DBs and those catalyzing aerobic and anaerobic degradation of the pollutants (see Section 2 and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Our BLAST search of the high-quality Illumina reads against the in-house and GenBank-nr DBs (see Section 2) revealed that 66,433 reads most plausibly carried parts of genes for the degradation of pollutants. These reads were used to calculate the abundances and phylum-level taxonomic origins of the genes for the degradation reaction steps (Fig. <a reveal-id="DSV023F4" data-open="DSV023F4" class="link link-reveal link-table xref-fig">4</a>; see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S8 and S9</a></span> for more comprehensive data). Our computer simulation of artificial ‘Illumina’ reads from the soil microbiome mentioned above indicated their ambiguous annotation of the short metagenomic reads at the genus level; the assignment accuracy was 34% for these reads. To clarify more accurately the genus-level taxonomic origins of the 66,433 metagenomic reads, nucleotide sequence similarity search was conducted for these reads using complete and draft prokaryote genome sequences (from 2,719 and 6,974 strains, respectively) in the NCBI DB and the draft sequences of three <em>Burkholderia</em> strains and one <em>Mycobacterium</em> strain that were isolated in this study from the polluted soil (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary SN2</a></span>). This analysis revealed that the metagenomic reads whose origins were assigned to <em>Burkholderia</em> and <em>Mycobacterium</em> in the in-house DB exhibited the same taxonomic assignment with probabilities of 72 and 69%, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>), indicating appropriate taxonomic assignments of the metagenomic reads at least for these two genera. The drastic increase in the abundance of <em>Burkholderia</em>-type genes for benzoate 1,2-dioxygenases (BpG; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span> for abbreviations of enzymes) (Fig. <a reveal-id="DSV023F4" data-open="DSV023F4" class="link link-reveal link-table xref-fig">4</a>), which have also been indicated to convert 3CB to chlorinated catechol,<sup><a reveal-id="DSV023C45" data-open="DSV023C45" class="link link-ref link-reveal xref-bibr">45</a></sup> coincided well with the relative abundance of <em>Burkholderia</em> cells (on the basis of community analysis by 16S rRNA gene profiling) as well as the rapid decrease of 3CB at Week 1. These data strongly suggested an important role of <em>Burkholderia</em>-type benzoate 1,2-dioxygenases in the degradation of 3CB. This suggestion was further supported by our preliminary transplantation experiment, in which the polluted soil-derived microbial population was inoculated into the sterilized soil; there was a rapid and drastic increase of <em>Burkholderia</em> after the pollution of the sterilized soil amended with 3CB (our unpublished observation). Another study on 3CB-amended soil also reported its degradation mainly by <em>Burkholderia</em>.<sup><a reveal-id="DSV023C46" data-open="DSV023C46" class="link link-ref link-reveal xref-bibr">46</a></sup> It is noteworthy that the drastic increase of <em>Burkholderia</em>-type genes was observed in other reaction steps, such as phthalate 4,5-dioxygenation (P45J) (Fig. <a reveal-id="DSV023F4" data-open="DSV023F4" class="link link-reveal link-table xref-fig">4</a>) and salicylate 1-hydroxylation (NG) at Week 1, at which time point no rapid degradation of the three other pollutants was observed. This simultaneous enrichment of degradation genes was most likely due to a common genomic feature of the genus <em>Burkholderia</em>; many of <em>Burkholderia</em> strains so far investigated encode the enzymes for the phthalate degradation route by its 4,5-dioxygenation (P45J) and for NG (note that salicylate and 1-hydroxy-2-naphthoate can be catabolized by the same enzyme<sup><a reveal-id="DSV023C39" data-open="DSV023C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSV023C47" data-open="DSV023C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSV023C48" data-open="DSV023C48" class="link link-ref link-reveal xref-bibr">48</a></sup>). The taxonomic compositions of aerobic degradation genes in the polluted soil at Week 6 indicated that an actinobacterial genus, <em>Mycobacterium</em>, was the most plausible contributor to the initial dioxygenation step of phenanthrene (PA) and the subsequent degradation to protocatechaute through the 3,4-dioxygenation sub-route of phthalate (P34J to P34L)<sup><a reveal-id="DSV023C49" data-open="DSV023C49" class="link link-ref link-reveal xref-bibr">49</a></sup> (Fig. <a reveal-id="DSV023F4" data-open="DSV023F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>), although the highest abundance of this genus in the communities of the polluted soil was 1% (on the basis of the16S rRNA gene profiling) at Week 6. The 4,5-dioxygenation sub-route from phthalate and a branching sub-route from 1-hydroxy-2-naphthoate (NG) were suggested to be mediated by the involvement of <em>Proteobacteria</em>, especially <em>Betaproteobacteria</em>. The genes for these sub-routes apparently showed biphasic response patterns in their abundances (increases at Weeks 1 and 6). Taking all of these observations into account, it is likely that the intermediate metabolites (e.g. phthalate and 1-hydroxy-2-naphthoate) of the mycobacterial degradation of phenanthrene in the soil were released to the extracellular environment<sup><a reveal-id="DSV023C50" data-open="DSV023C50" class="link link-ref link-reveal xref-bibr">50</a></sup> and utilized by a variety of co-resident microbial groups. There are indeed several reports of syntrophic associations between two phyla for the complete degradation of recalcitrant compounds.<sup><a reveal-id="DSV023C51" data-open="DSV023C51" class="link link-ref link-reveal xref-bibr">51</a></sup> A <em>Mycobacterium</em> strain, EPa45, with the ability to degrade phenanthrene to TCA compounds, was isolated from the polluted soil at Week 15. Our genome sequencing of EPa45 suggested its phenanthrene degradation to protocatechaute<sup><a reveal-id="DSV023C52" data-open="DSV023C52" class="link link-ref link-reveal xref-bibr">52</a></sup> via the same route as used for mycobacterial phenanthrene degradation that was proposed by the Illumina metagenomic read analysis. The numbers of metagenomic reads that hit the well-known genes for the initial dioxygenation enzymes for biphenyl (BpA) and carbazole (CA) were quite small (&lt;16 hit reads at Week 6), and this was consistent with the undetectable level of such genes in the metagenomic samples by PCR analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary SN1</a></span>). These results suggested the involvement of as-yet-unknown gene(s) and metabolic pathway(s) for the degradation of the two aromatic compounds. Involvement of anaerobic degradation pathways was also ambiguous in the functional analysis of the metagenomic reads. </p>                        <a id="87198010" scrollto-destination="87198010"></a>
<div id="DSV023F4" data-id="DSV023F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02304.png?Expires=2147483647&amp;Signature=ltveCQ8Y9WQFYg92D0sHZ5k9aPTBRGn9mq33iJTrVVwlROseCR9KJja8YyMaAS9o394N0aj2DkbzM0pCnYw5JfTya70qtcPYsebhvUsxUIrstSIabYr5cq20iVC2OiotOWGaDK-i7NCq2lrktXmjAzdSIRgGLYQt87niEKQIYv42QonbJ9mOEW-mQ00I1nCVaGc6YJEoP4w393xCmrwjKan0KLu~fbPtjltsylDjnkd368nf-rVhxyMY-AGPioWpddeXIeANIUMAOPjyn09ads1Ct81R4OqSCKpr5N~qRay2MZhDgbg2O27rUueO5UGbbLy~wpHYYKJHmrZPn-E9Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in abundances of genes for aerobic degradation of aromatic compounds in polluted soil. A simplified pathway map for well-known aerobic degradation routes of the four polluted compounds. Boxes located in each route indicate representative reaction steps, and several steps (dashed lines) are omitted for simplicity. Abbreviations of enzymes are shown in Supplementary Table S5. The two graphs indicate the taxonomic compositions and the abundance of the genes in each reaction step. The upper graph shows the abundances at the phylum level (with the exception that the domain and class levels are shown for Archaea and Proteobacteria, respectively), and the lower graph the abundances at the level limited to the genera of Mycobacterium and Burkholderia. See Section 2 for the details to calculate gene abundances. Depending on the abundances, the scales of Y-axes of graphs are conventionally categorized into four groups with the following colours: grey, 10−6; blue, 10−5; purple, 10−4; and red, 10−3. More detailed results at the overall pathway map level by using the metagenomic samples from the control and polluted soils are depicted in Supplementary Figs S8 and S9." path-from-xml="dsv02304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198010/dsv02304.tiff" path-from-xml="dsv02304.tiff" target="_blank">View large</a><a section="87198010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02304.png?Expires=2147483647&amp;Signature=UPPAXjOBSkAyD22J4mEB6JnlCCV7btJLLHFbkyZw9ZQ-i08ipJ3Yucp2WmDnx3cOBa6Znsw3NHFQaEzFX6Io76yCUBdLg2-MCE3qILy4C75OcMxsXuh2MrYoV4zqJ225Lk22Pvp1N7HfQlGC~vsCj93EBZy2fBMTzbQHbevQsmmLu8af4md-YdHnRL4CKG6kiCXmh6RkzjupT2LaeBjBLhXQpDbapRDxfvllxr3oiusUDZ~l94bDhEfWozop41p76Lbtb~nhP4A8V5~y5Ko28MB-25BfMUCIJP9-RCCK173KkPhJIo9hZAu8wpwMC9lou3iS5eTOSLP8kxReENDGpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198010&amp;ar=2462440&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv02304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in abundances of genes for aerobic degradation of aromatic compounds in polluted soil. A simplified pathway map for well-known aerobic degradation routes of the four polluted compounds. Boxes located in each route indicate representative reaction steps, and several steps (dashed lines) are omitted for simplicity. Abbreviations of enzymes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. The two graphs indicate the taxonomic compositions and the abundance of the genes in each reaction step. The upper graph shows the abundances at the phylum level (with the exception that the domain and class levels are shown for <em>Archaea</em> and <em>Proteobacteria</em>, respectively), and the lower graph the abundances at the level limited to the genera of <em>Mycobacterium</em> and <em>Burkholderia</em>. See Section 2 for the details to calculate gene abundances. Depending on the abundances, the scales of <em>Y</em>-axes of graphs are conventionally categorized into four groups with the following colours: grey, 10<sup>−6</sup>; blue, 10<sup>−5</sup>; purple, 10<sup>−4</sup>; and red, 10<sup>−3</sup>. More detailed results at the overall pathway map level by using the metagenomic samples from the control and polluted soils are depicted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S8 and S9</a></span>.</p></div></div><div content-id="DSV023F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02304.png?Expires=2147483647&amp;Signature=ltveCQ8Y9WQFYg92D0sHZ5k9aPTBRGn9mq33iJTrVVwlROseCR9KJja8YyMaAS9o394N0aj2DkbzM0pCnYw5JfTya70qtcPYsebhvUsxUIrstSIabYr5cq20iVC2OiotOWGaDK-i7NCq2lrktXmjAzdSIRgGLYQt87niEKQIYv42QonbJ9mOEW-mQ00I1nCVaGc6YJEoP4w393xCmrwjKan0KLu~fbPtjltsylDjnkd368nf-rVhxyMY-AGPioWpddeXIeANIUMAOPjyn09ads1Ct81R4OqSCKpr5N~qRay2MZhDgbg2O27rUueO5UGbbLy~wpHYYKJHmrZPn-E9Jw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in abundances of genes for aerobic degradation of aromatic compounds in polluted soil. A simplified pathway map for well-known aerobic degradation routes of the four polluted compounds. Boxes located in each route indicate representative reaction steps, and several steps (dashed lines) are omitted for simplicity. Abbreviations of enzymes are shown in Supplementary Table S5. The two graphs indicate the taxonomic compositions and the abundance of the genes in each reaction step. The upper graph shows the abundances at the phylum level (with the exception that the domain and class levels are shown for Archaea and Proteobacteria, respectively), and the lower graph the abundances at the level limited to the genera of Mycobacterium and Burkholderia. See Section 2 for the details to calculate gene abundances. Depending on the abundances, the scales of Y-axes of graphs are conventionally categorized into four groups with the following colours: grey, 10−6; blue, 10−5; purple, 10−4; and red, 10−3. More detailed results at the overall pathway map level by using the metagenomic samples from the control and polluted soils are depicted in Supplementary Figs S8 and S9." path-from-xml="dsv02304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198010/dsv02304.tiff" path-from-xml="dsv02304.tiff" target="_blank">View large</a><a section="87198010" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02304.png?Expires=2147483647&amp;Signature=UPPAXjOBSkAyD22J4mEB6JnlCCV7btJLLHFbkyZw9ZQ-i08ipJ3Yucp2WmDnx3cOBa6Znsw3NHFQaEzFX6Io76yCUBdLg2-MCE3qILy4C75OcMxsXuh2MrYoV4zqJ225Lk22Pvp1N7HfQlGC~vsCj93EBZy2fBMTzbQHbevQsmmLu8af4md-YdHnRL4CKG6kiCXmh6RkzjupT2LaeBjBLhXQpDbapRDxfvllxr3oiusUDZ~l94bDhEfWozop41p76Lbtb~nhP4A8V5~y5Ko28MB-25BfMUCIJP9-RCCK173KkPhJIo9hZAu8wpwMC9lou3iS5eTOSLP8kxReENDGpw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198010&amp;ar=2462440&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv02304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in abundances of genes for aerobic degradation of aromatic compounds in polluted soil. A simplified pathway map for well-known aerobic degradation routes of the four polluted compounds. Boxes located in each route indicate representative reaction steps, and several steps (dashed lines) are omitted for simplicity. Abbreviations of enzymes are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. The two graphs indicate the taxonomic compositions and the abundance of the genes in each reaction step. The upper graph shows the abundances at the phylum level (with the exception that the domain and class levels are shown for <em>Archaea</em> and <em>Proteobacteria</em>, respectively), and the lower graph the abundances at the level limited to the genera of <em>Mycobacterium</em> and <em>Burkholderia</em>. See Section 2 for the details to calculate gene abundances. Depending on the abundances, the scales of <em>Y</em>-axes of graphs are conventionally categorized into four groups with the following colours: grey, 10<sup>−6</sup>; blue, 10<sup>−5</sup>; purple, 10<sup>−4</sup>; and red, 10<sup>−3</sup>. More detailed results at the overall pathway map level by using the metagenomic samples from the control and polluted soils are depicted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S8 and S9</a></span>.</p></div></div>                            <h3 scrollto-destination=87198013 id="87198013" class="section-title" data-legacy-id=s3e>3.5 Time-course change of bacteriophage genes</h3>
<p>Our BLASTX analysis of Illumina read-derived long contigs against the GenBank-nr DB suggested that some such contigs in the metagenomic samples from the polluted soil were originated from phage genomes (see above). More detailed BLASTX search of Illumina reads against the ACLAME DB indicated that the relative abundance of total phage DNA reads in the polluted soil increased at Weeks 3 and 6 (Fig. <a reveal-id="DSV023F5" data-open="DSV023F5" class="link link-reveal link-table xref-fig">5</a>A), supporting the notion of an increase in prophage-carrying microbes and/or phage particles in the soil. Therefore, we further searched and analysed the phage gene-derived Illumina reads and their contigs (see Section 2.10). Among the re-assembled contigs from the Illumina metagenomic reads, 58 contigs had at least one CDS that showed &gt;40% identity (at the amino acid sequence level) with phage CDS in the POGs DB (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15</a></span>), and these contigs were subjected to the PHAST analysis.<sup><a reveal-id="DSV023C33" data-open="DSV023C33" class="link link-ref link-reveal xref-bibr">33</a></sup> This analysis revealed that (i) phage genome-derived regions were identified in eight large (15.6–93.6 kb) contigs, (ii) such contigs possessed many genes derived from <em>Pseudomonas</em> and <em>Burkholderia</em> phages, and (iii) a set of genes found in the 44.5-kb contig (Contig70-3) were evaluated as an ‘intact phage’ while the remaining contigs were evaluated as ‘incomplete phages’ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span>). The BLASTN search of the Illumina reads from each of 11 metagenomic samples against the above-described 8 contigs was conducted to examine their quantitative time-course changes. This search showed an extensive increase of all 8 contigs at Week 3 or 6 only in the polluted soil (by 10<sup>3</sup> to 10<sup>4</sup> times greater than those at Week 0) and a subsequent decrease at Week 12, implying that replication of the phage genomes occurred around Weeks 3 and 6 (Fig. <a reveal-id="DSV023F5" data-open="DSV023F5" class="link link-reveal link-table xref-fig">5</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Our BLASTN search of all high-quality Illumina reads against the 8 contigs revealed that the Illumina hit reads were evenly distributed across the whole range of each contig, suggesting that the 8 contigs were not the chimeras that were artificially generated at our assemblage stage. Further analysis of Contig70-3 exhibited moderate (at most 45% at the BLASTX search level) similarity with the genome of a <em>Burkholderia</em> phage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span>), BcepGomr (GI: 145321088), of soil origin.<sup><a reveal-id="DSV023C53" data-open="DSV023C53" class="link link-ref link-reveal xref-bibr">53</a>,<a reveal-id="DSV023C54" data-open="DSV023C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Our investigation of the relative abundances of the Illumina reads that corresponded to parts of Contig70-3 in each of 11 metagenomic samples indicated that such reads extensively increased only in the polluted soil (1.7 × 10<sup>4</sup>-fold increase at Week 6 compared with Week 1) (Fig. <a reveal-id="DSV023F5" data-open="DSV023F5" class="link link-reveal link-table xref-fig">5</a>B). It is noteworthy that the relative abundance of the genomic DNA of <em>Burkholderia</em> in the polluted soil was the greatest at Week 1 (23-fold higher at this week than at Week 0). This result shows that there was temporal and numerical asynchrony between the relative abundances of the phage genome contig and its putative host genome. The metagenomic DNA sample at Week 6 from the polluted soil was used to PCR-amplify various Contig70-3-derived phage genome fragments, and we were able to detect (i) the amplicons of the phage-derived fragments of expected sizes and (ii) the phage genome structure with circular or concatemer form (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11c</a></span>). Quantitative PCR also confirmed the extensive increase of the phage genome at Week 6 in the polluted soil (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11d</a></span>). One plausible reason for the asynchrony is phage predation of <em>Burkholderia</em> cells around Week 6. Analysis of other long contigs also suggested that a similar situation was the case between <em>Pseudomonas</em> and its putative phages, although <em>in silico</em> analysis suggested that the phage-inferred contigs were not long enough to ‘intact’ phage genomes. Phage predation of successfully proliferating microbes, known as the ‘kill-the-winner’ hypothesis, has been proposed as one of the mechanisms underlying the compositional stability of microbial communities, and the involvement of this mechanism has been supported in aquatic environments.<sup><a reveal-id="DSV023C55" data-open="DSV023C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Our present findings strongly suggested that phage predation targeting the most successfully proliferating microbe (<em>Burkholderia</em> in this case) also contributed to the stabilization of our soil microbial community structure. It is unclear at present (i) what molecular mechanism(s) are involved in this phenomenon and (ii) whether such phages have lysogenic or lytic lifestyles. </p>                        <a id="87198016" scrollto-destination="87198016"></a>
<div id="DSV023F5" data-id="DSV023F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02305.png?Expires=2147483647&Signature=QazxB9D9186y3axHhQH~4G9aCsroNaKcC5O4lM4qJ4dHDwAAJtmFhkShSLKNXVa9S3PCNU55cj706F5UXWgTxR71nBsKJ89T0wERKyzfEPAnOppbyxZVg0btZIwe9AV5Wsj9AZjS0nkd00ekw5qQD6xP9G1dB-pBdrSq7xoHA1ZosCUD7qthdH5o788wuO-Kp36opqxgv866UbcrCQUe3M5R~fL7RgUVsRGX7n8I1cRIM~zZ18mW0p0~OAf66MrBQAI4zHNGz0t07ayzDk5IpTlQUhslwHcvDhEB2iGXPo6a3tNc4M18vMPhkdpEmLy7NWTpYIw8b-2EZjlH5ud1zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in abundances of phage genome-derived sequences in soil metagenomic samples. (A) Abundance of phage genome-derived reads in control (open circle) and polluted (closed) soil metagenomic samples based on BLASTX search of the Illumina reads against the ACLAME DB. (B) Relative abundances of a 44.5-kb contig (Contig70-3) DNA putatively derived from a phage genome from Burkholderia (blue circle) and the Burkholderia genomic DNA (red circle) in the control (open circle) and the polluted (closed circle) samples. The abundance of Contig70-3 DNA in each sample was based on the numbers of hit reads counted in BLASTN search of the metagenomic reads against Contig70-3, whereas the abundance of Burkholderia genomic DNA was based on the numbers of hit reads taxonomically assigned to Burkholderia by BLASTX search against the KEGG DB. The hit numbers in both cases were normalized by the number of gyrB-added USCGs in each metagenomic sample. The relative abundances of the Contig70-3 and Burkholderia genomes are expressed by taking the respective smallest values as 1." path-from-xml="dsv02305.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198016/dsv02305.tiff" path-from-xml="dsv02305.tiff" target="_blank">View large</a><a section="87198016" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02305.png?Expires=2147483647&Signature=co8qKOyyUDzsnzCDPHAdEZzoUu8avgnwoGFCZQ-IjhJ-i95aX9RxX2sH90qkJXgaXJcvkC~2khQPjCrOGaHo227jk81DLx~aNFjNSikL9rAgiEqIm1MzCEKlYxl49w--gyqllpNfkJ3O5ropXsQ5GkeN86eFy8DJtFL6xsD6aK11~oMMIy2ZheFD-a~rb9-5CbBnVzGkigTigD5Pvnxt1votzh3DNMOx1JGSwAk03Z2ns4qwFWJ5jMlinf1tC4VE1-NQmqQU8w~pGV0X6knc62V6w4dHvPx5bBm3oi0Xd~b39NNjN-9p9M6QsmY~6RxMyH60JcZHU3G~D1eXLraY5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198016&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02305.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in abundances of phage genome-derived sequences in soil metagenomic samples. (A) Abundance of phage genome-derived reads in control (open circle) and polluted (closed) soil metagenomic samples based on BLASTX search of the Illumina reads against the ACLAME DB. (B) Relative abundances of a 44.5-kb contig (Contig70-3) DNA putatively derived from a phage genome from <em>Burkholderia</em> (blue circle) and the <em>Burkholderia</em> genomic DNA (red circle) in the control (open circle) and the polluted (closed circle) samples. The abundance of Contig70-3 DNA in each sample was based on the numbers of hit reads counted in BLASTN search of the metagenomic reads against Contig70-3, whereas the abundance of <em>Burkholderia</em> genomic DNA was based on the numbers of hit reads taxonomically assigned to <em>Burkholderia</em> by BLASTX search against the KEGG DB. The hit numbers in both cases were normalized by the number of <em>gyrB</em>-added USCGs in each metagenomic sample. The relative abundances of the Contig70-3 and <em>Burkholderia</em> genomes are expressed by taking the respective smallest values as 1.</p></div></div><div content-id="DSV023F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/m_dsv02305.png?Expires=2147483647&Signature=QazxB9D9186y3axHhQH~4G9aCsroNaKcC5O4lM4qJ4dHDwAAJtmFhkShSLKNXVa9S3PCNU55cj706F5UXWgTxR71nBsKJ89T0wERKyzfEPAnOppbyxZVg0btZIwe9AV5Wsj9AZjS0nkd00ekw5qQD6xP9G1dB-pBdrSq7xoHA1ZosCUD7qthdH5o788wuO-Kp36opqxgv866UbcrCQUe3M5R~fL7RgUVsRGX7n8I1cRIM~zZ18mW0p0~OAf66MrBQAI4zHNGz0t07ayzDk5IpTlQUhslwHcvDhEB2iGXPo6a3tNc4M18vMPhkdpEmLy7NWTpYIw8b-2EZjlH5ud1zA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Time-course changes in abundances of phage genome-derived sequences in soil metagenomic samples. (A) Abundance of phage genome-derived reads in control (open circle) and polluted (closed) soil metagenomic samples based on BLASTX search of the Illumina reads against the ACLAME DB. (B) Relative abundances of a 44.5-kb contig (Contig70-3) DNA putatively derived from a phage genome from Burkholderia (blue circle) and the Burkholderia genomic DNA (red circle) in the control (open circle) and the polluted (closed circle) samples. The abundance of Contig70-3 DNA in each sample was based on the numbers of hit reads counted in BLASTN search of the metagenomic reads against Contig70-3, whereas the abundance of Burkholderia genomic DNA was based on the numbers of hit reads taxonomically assigned to Burkholderia by BLASTX search against the KEGG DB. The hit numbers in both cases were normalized by the number of gyrB-added USCGs in each metagenomic sample. The relative abundances of the Contig70-3 and Burkholderia genomes are expressed by taking the respective smallest values as 1." path-from-xml="dsv02305.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198016/dsv02305.tiff" path-from-xml="dsv02305.tiff" target="_blank">View large</a><a section="87198016" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv02305.png?Expires=2147483647&Signature=co8qKOyyUDzsnzCDPHAdEZzoUu8avgnwoGFCZQ-IjhJ-i95aX9RxX2sH90qkJXgaXJcvkC~2khQPjCrOGaHo227jk81DLx~aNFjNSikL9rAgiEqIm1MzCEKlYxl49w--gyqllpNfkJ3O5ropXsQ5GkeN86eFy8DJtFL6xsD6aK11~oMMIy2ZheFD-a~rb9-5CbBnVzGkigTigD5Pvnxt1votzh3DNMOx1JGSwAk03Z2ns4qwFWJ5jMlinf1tC4VE1-NQmqQU8w~pGV0X6knc62V6w4dHvPx5bBm3oi0Xd~b39NNjN-9p9M6QsmY~6RxMyH60JcZHU3G~D1eXLraY5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198016&ar=2462440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv02305.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Time-course changes in abundances of phage genome-derived sequences in soil metagenomic samples. (A) Abundance of phage genome-derived reads in control (open circle) and polluted (closed) soil metagenomic samples based on BLASTX search of the Illumina reads against the ACLAME DB. (B) Relative abundances of a 44.5-kb contig (Contig70-3) DNA putatively derived from a phage genome from <em>Burkholderia</em> (blue circle) and the <em>Burkholderia</em> genomic DNA (red circle) in the control (open circle) and the polluted (closed circle) samples. The abundance of Contig70-3 DNA in each sample was based on the numbers of hit reads counted in BLASTN search of the metagenomic reads against Contig70-3, whereas the abundance of <em>Burkholderia</em> genomic DNA was based on the numbers of hit reads taxonomically assigned to <em>Burkholderia</em> by BLASTX search against the KEGG DB. The hit numbers in both cases were normalized by the number of <em>gyrB</em>-added USCGs in each metagenomic sample. The relative abundances of the Contig70-3 and <em>Burkholderia</em> genomes are expressed by taking the respective smallest values as 1.</p></div></div>                            <h3 scrollto-destination=87198018 id="87198018" class="section-title" data-legacy-id=s3f>3.6 Conclusions and perspectives</h3>
<p>This study was performed to investigate by metagenomic approach the time-course changes in microbial taxonomic compositions and gene pools in the closed soil microcosm to which four aromatic compounds were simultaneously added. This approach strongly suggested the biphasic degradation of 3CB and phenanthrene: immediate degradation of the former compound mainly by <em>Burkholderia</em> and the delayed degradation of the latter one syntrophically by two phyla. Such suggestion was further supported by our traditional and molecular microbiological analysis as well as genomic analysis of relevant strains isolated from the polluted soil.</p><p>Systems biology has proposed that the robustness of biological systems at various levels is achieved by feedback control, fail-safe by means of diversity, and modularity.<sup><a reveal-id="DSV023C43" data-open="DSV023C43" class="link link-ref link-reveal xref-bibr">43</a></sup> This proposal could be apparently applied to the microbiota in our closed soil microcosm: i.e. phage-mediated decrease of the most successfully propagated <em>Burkholderia</em> cells, functional redundancy of microbiota, and syntrophic degradation of pollutants, respectively. Although the proposal described above has been in part applicable to a few microbial communities such as anaerobic aquatic microbiota,<sup><a reveal-id="DSV023C56" data-open="DSV023C56" class="link link-ref link-reveal xref-bibr">56</a></sup> it has not yet been investigated whether this is the case with soil microbiota. Our time-series metagenomic analysis of the closed soil microcosm polluted with aromatic compounds contributes to clarifying robustness of the soil microbial community against the chemical spike.</p><p>There existed over 250 genera in the soil at Week 0. Although some genera in the polluted soil showed drastic time-course changes in their relative abundances, most other genera only exhibited a ‘slight’ response to the pollution with no apparent increase or decrease. The ecological function of such ‘silent audiences’ in our microbial community remains to be explored. It is still unclear whether or not the increases in the degradation genes in our soil were followed by their expression for the degradation of pollutants. Time-course analysis of microbial communities using a combination of metagenomics with metatranscriptomics and metaproteomics, as well as other analyses (e.g. use of the stable isotope probing technique<sup><a reveal-id="DSV023C29" data-open="DSV023C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSV023C57" data-open="DSV023C57" class="link link-ref link-reveal xref-bibr">57</a></sup>), will greatly contribute to evaluation of the ecological functions of microbial genes at the community level.</p>                            <h2 scrollto-destination=87198023 id="87198023" class="section-title" data-legacy-id=s4>4. Data availability</h2>
<p>The nucleotide sequences determined in this study have been deposited in DDBJ/EBI/NCBI databases. The accession numbers for Sanger sequencing of clone library of 16S rRNA genes are LC016889–LC017690. The genome sequence of <em>Mycobacterium</em> strain is under the accession number of CP011773, and those of three <em>Burkholderia</em> strains are under the accession numbers of BBSL01000001–BBSL01000774, BBSM01000001–BBSM01000421, and BBSK01000001–BBSK01000186. All raw metagenome sequence data have been deposited under the Bioproject number of PRJDB2729 in the DDBJ Sequence Read Archive (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>).</p>                            <h2 scrollto-destination=87198026 id="87198026" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by Grants-in-Aid for Scientific Research (no. 25292206 and Innovative Area ‘Genome Science’) from the Ministry of Education, Culture, Sports, Science and Technology, Japan, and by a grant from the Institute for Fermentation, Osaka (IFO) and a Global COE program ‘Ecosystem Adaptability Science for the Future’ at Tohoku University. Funding to pay the Open Access publication charges for this article was provided by the Ministry of Education, Culture, Sports, Science and Technology, Japan.</p>                            <h2 scrollto-destination=87198028 id="87198028" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV023C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shade</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Peter</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Allison</div> <div class="given-names">S.D.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Fundamentals of microbial community resistance and resilience</div>, <div class="source">Front. Microbiol.</div> , <div class="volume">3</div>, <div class="fpage">417</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fundamentals%20of%20microbial%20community%20resistance%20and%20resilience&amp;author=A.%20Shade&amp;author=H.%20Peter&amp;author=S.D.%20Allison&amp;publication_year=2012&amp;journal=Front.%20Microbiol.&amp;volume=3&amp;pages=417" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2012.00417" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2012.00417" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2012.00417"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23267351" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allison</div>
               <div class="given-names">S.D.</div>
            </div>, <div class="name string-name"><div class="surname">Martiny</div> <div class="given-names">J.B.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Resistance, resilience, and redundancy in microbial communities</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">105</div>(<div class="issue">Suppl. 1</div>), <div class="fpage">11512</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resistance%2C%20resilience%2C%20and%20redundancy%20in%20microbial%20communities&amp;author=S.D.%20Allison&amp;author=J.B.%20Martiny&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=11512-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0801925105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0801925105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0801925105"> </span></div> </div></div></div></div></div><div content-id="DSV023C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="collab">The Human Microbiome Project Consortium</div>. <div class="year">2012</div>, <div class="article-title">Structure, function and diversity of the healthy human microbiome</div>, <div class="source">Nature</div> , <div class="volume">486</div>, <div class="fpage">207</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22699609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gilbert</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Jansson</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">R.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">The Earth Microbiome project: successes and aspirations</div>, <div class="source">BMC Biol.</div> , <div class="volume">12</div>, <div class="fpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Earth%20Microbiome%20project%3A%20successes%20and%20aspirations&amp;author=J.A.%20Gilbert&amp;author=J.K.%20Jansson&amp;author=R.%20Knight&amp;publication_year=2014&amp;journal=BMC%20Biol.&amp;volume=12&amp;pages=69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12915-014-0069-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12915-014-0069-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12915-014-0069-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharon</div>
               <div class="given-names">I.</div>
            </div>, <div class="name string-name"><div class="surname">Morowitz</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">B.C.</div></div>, <div class="name string-name"><div class="surname">Costello</div> <div class="given-names">E.K.</div></div>, <div class="name string-name"><div class="surname">Relman</div> <div class="given-names">D.A.</div></div>, <div class="name string-name"><div class="surname">Banfield</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Time series community genomics analysis reveals rapid shifts in bacterial species, strains, and phage during infant gut colonization</div>, <div class="source">Genome Res.</div> , <div class="volume">23</div>, <div class="fpage">111</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Time%20series%20community%20genomics%20analysis%20reveals%20rapid%20shifts%20in%20bacterial%20species%2C%20strains%2C%20and%20phage%20during%20infant%20gut%20colonization&amp;author=I.%20Sharon&amp;author=M.J.%20Morowitz&amp;author=B.C.%20Thomas&amp;author=E.K.%20Costello&amp;author=D.A.%20Relman&amp;author=J.F.%20Banfield&amp;publication_year=2013&amp;journal=Genome%20Res.&amp;volume=23&amp;pages=111-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.142315.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.142315.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.142315.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22936250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Koenig</div>
               <div class="given-names">J.E.</div>
            </div>, <div class="name string-name"><div class="surname">Spor</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Scalfone</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Succession of microbial consortia in the developing infant gut microbiome</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">108</div>(<div class="issue">Suppl. 1</div>), <div class="fpage">4578</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Succession%20of%20microbial%20consortia%20in%20the%20developing%20infant%20gut%20microbiome&amp;author=J.E.%20Koenig&amp;author=A.%20Spor&amp;author=N.%20Scalfone&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=4578-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1000081107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1000081107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1000081107"> </span></div> </div></div></div></div></div><div content-id="DSV023C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mackelprang</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Waldrop</div> <div class="given-names">M.P.</div></div>, <div class="name string-name"><div class="surname">Deangelis</div> <div class="given-names">K.M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Metagenomic analysis of a permafrost microbial community reveals a rapid response to thaw</div>, <div class="source">Nature</div> , <div class="volume">480</div>, <div class="fpage">368</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20of%20a%20permafrost%20microbial%20community%20reveals%20a%20rapid%20response%20to%20thaw&amp;author=R.%20Mackelprang&amp;author=M.P.%20Waldrop&amp;author=K.M.%20Deangelis&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=368-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10576" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10576" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10576"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22056985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Delmont</div>
               <div class="given-names">T.O.</div>
            </div>, <div class="name string-name"><div class="surname">Prestat</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Keegan</div> <div class="given-names">K.P.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Structure, fluctuation and magnitude of a natural grassland soil metagenome</div>, <div class="source">ISME J.</div> , <div class="volume">6</div>, <div class="fpage">1677</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%2C%20fluctuation%20and%20magnitude%20of%20a%20natural%20grassland%20soil%20metagenome&amp;author=T.O.%20Delmont&amp;author=E.%20Prestat&amp;author=K.P.%20Keegan&amp;publication_year=2012&amp;journal=ISME%20J.&amp;volume=6&amp;pages=1677-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2011.197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2011.197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22297556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tringe</div>
               <div class="given-names">S.G.</div>
            </div>, <div class="name string-name"><div class="surname">von Mering</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Comparative metagenomics of microbial communities</div>, <div class="source">Science</div> , <div class="volume">308</div>, <div class="fpage">554</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20of%20microbial%20communities&amp;author=S.G.%20Tringe&amp;author=C.%20von%20Mering&amp;author=A.%20Kobayashi&amp;publication_year=2005&amp;journal=Science&amp;volume=308&amp;pages=554-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15845853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Janssen</div>
               <div class="given-names">P.H.</div>
            </div>
          </div>
          <div class="year">2006</div>, <div class="article-title">Identifying the dominant soil bacterial taxa in libraries of 16S rRNA and 16S rRNA genes</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">72</div>, <div class="fpage">1719</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20the%20dominant%20soil%20bacterial%20taxa%20in%20libraries%20of%2016S%20rRNA%20and%2016S%20rRNA%20genes&amp;author=P.H.%20Janssen&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=1719-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.72.3.1719-1728.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.72.3.1719-1728.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.72.3.1719-1728.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16517615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Luo</div>
               <div class="given-names">C.W.</div>
            </div>, <div class="name string-name"><div class="surname">Rodriguez-R</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Johnston</div> <div class="given-names">E.R.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Soil microbial community responses to a decade of warming as revealed by comparative metagenomics</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">80</div>, <div class="fpage">1777</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soil%20microbial%20community%20responses%20to%20a%20decade%20of%20warming%20as%20revealed%20by%20comparative%20metagenomics&amp;author=C.W.%20Luo&amp;author=L.M.%20Rodriguez-R&amp;author=E.R.%20Johnston&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=80&amp;pages=1777-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03712-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03712-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03712-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24375144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nagayama</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Sugawara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Endo</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Isolation of oxygenase genes for indigo-forming activity from an artificially polluted soil metagenome by functional screening using <em>Pseudomonas putida</em> strains as hosts</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="volume">99</div>, <div class="fpage">4453</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20oxygenase%20genes%20for%20indigo-forming%20activity%20from%20an%20artificially%20polluted%20soil%20metagenome%20by%20functional%20screening%20using%20Pseudomonas%20putida%20strains%20as%20hosts&amp;author=H.%20Nagayama&amp;author=T.%20Sugawara&amp;author=R.%20Endo&amp;publication_year=2015&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=99&amp;pages=4453-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-014-6322-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-014-6322-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-014-6322-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25573469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mori</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Maruyama</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kato</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Design and experimental application of a novel non-degenerate universal primer set that amplifies prokaryotic 16S rRNA genes with a low possibility to amplify eukaryotic rRNA genes</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">217</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20and%20experimental%20application%20of%20a%20novel%20non-degenerate%20universal%20primer%20set%20that%20amplifies%20prokaryotic%2016S%20rRNA%20genes%20with%20a%20low%20possibility%20to%20amplify%20eukaryotic%20rRNA%20genes&amp;author=H.%20Mori&amp;author=F.%20Maruyama&amp;author=H.%20Kato&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=217-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24277737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">Search and clustering orders of magnitude faster than BLAST</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">2460</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Search%20and%20clustering%20orders%20of%20magnitude%20faster%20than%20BLAST&amp;author=R.C.%20Edgar&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=2460-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq461"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20709691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Clemente</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Quince</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">R.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">UCHIME improves sensitivity and speed of chimera detection</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2194</div>–<div class="lpage">200</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;author=R.C.%20Edgar&amp;author=B.J.%20Haas&amp;author=J.C.%20Clemente&amp;author=C.%20Quince&amp;author=R.%20Knight&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Garrity</div> <div class="given-names">G.M.</div></div>, <div class="name string-name"><div class="surname">Tiedje</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J.R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Naive Bayesian classifier for rapid assignment of rRNA sequences into the new bacterial taxonomy</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">73</div>, <div class="fpage">5261</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Naive%20Bayesian%20classifier%20for%20rapid%20assignment%20of%20rRNA%20sequences%20into%20the%20new%20bacterial%20taxonomy&amp;author=Q.%20Wang&amp;author=G.M.%20Garrity&amp;author=J.M.%20Tiedje&amp;author=J.R.%20Cole&amp;publication_year=2007&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=73&amp;pages=5261-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.00062-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00062-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.00062-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Peng</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Leung</div> <div class="given-names">H.C.M.</div></div>, <div class="name string-name"><div class="surname">Yiu</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Chin</div> <div class="given-names">F.Y.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">IDBA-UD: a <em>de novo</em> assembler for single-cell and metagenomic sequencing data with highly uneven depth</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1420</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IDBA-UD%3A%20a%20de%20novo%20assembler%20for%20single-cell%20and%20metagenomic%20sequencing%20data%20with%20highly%20uneven%20depth&amp;author=Y.%20Peng&amp;author=H.C.M.%20Leung&amp;author=S.M.%20Yiu&amp;author=F.Y.L.%20Chin&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1420-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22495754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhu</div>
               <div class="given-names">W.H.</div>
            </div>, <div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title"><em>Ab initio</em> gene identification in metagenomic sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">e132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ab%20initio%20gene%20identification%20in%20metagenomic%20sequences&amp;author=W.H.%20Zhu&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Qin</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">A human gut microbial gene catalogue established by metagenomic sequencing</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">59</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20human%20gut%20microbial%20gene%20catalogue%20established%20by%20metagenomic%20sequencing&amp;author=J.%20Qin&amp;author=R.%20Li&amp;author=J.%20Raes&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=59-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08821" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08821" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08821"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20203603" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Raes</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Korbel</div> <div class="given-names">J.O.</div></div>, <div class="name string-name"><div class="surname">Lercher</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">von Mering</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Prediction of effective genome size in metagenomic samples</div>, <div class="source">Genome Biol.</div> , <div class="volume">8</div>, <div class="fpage">R10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20effective%20genome%20size%20in%20metagenomic%20samples&amp;author=J.%20Raes&amp;author=J.O.%20Korbel&amp;author=M.J.%20Lercher&amp;author=C.%20von%20Mering&amp;author=P.%20Bork&amp;publication_year=2007&amp;journal=Genome%20Biol.&amp;volume=8&amp;pages=R10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-1-r10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-1-r10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-1-r10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17224063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Caliński</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Harabasz</div> <div class="given-names">J.</div></div></div>
          <div class="year">1974</div>, <div class="article-title">A dendrite method for cluster analysis</div>, <div class="source">Commun. Stat.</div> , <div class="volume">3</div>, <div class="fpage">1</div>–<div class="lpage">27</div>.</div></div></div></div><div content-id="DSV023C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arumugam</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Pelletier</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Enterotypes of the human gut microbiome</div>, <div class="source">Nature</div> , <div class="volume">473</div>, <div class="fpage">174</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enterotypes%20of%20the%20human%20gut%20microbiome&amp;author=M.%20Arumugam&amp;author=J.%20Raes&amp;author=E.%20Pelletier&amp;publication_year=2011&amp;journal=Nature&amp;volume=473&amp;pages=174-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21508958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Korbel</div>
               <div class="given-names">J.O.</div>
            </div>, <div class="name string-name"><div class="surname">Snel</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Huynen</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">SHOT: a web server for the construction of genome phylogenies</div>, <div class="source">Trends Genet.</div> , <div class="volume">18</div>, <div class="fpage">158</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SHOT%3A%20a%20web%20server%20for%20the%20construction%20of%20genome%20phylogenies&amp;author=J.O.%20Korbel&amp;author=B.%20Snel&amp;author=M.A.%20Huynen&amp;author=P.%20Bork&amp;publication_year=2002&amp;journal=Trends%20Genet.&amp;volume=18&amp;pages=158-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(01)02597-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(01)02597-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(01)02597-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11858840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kurokawa</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes</div>, <div class="source">DNA Res.</div> , <div class="volume">14</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20revealed%20commonly%20enriched%20gene%20sets%20in%20human%20gut%20microbiomes&amp;author=K.%20Kurokawa&amp;author=T.%20Itoh&amp;author=T.%20Kuwahara&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17916580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parales</div>
               <div class="given-names">R.E.</div>
            </div>
          </div>
          <div class="year">2003</div>, <div class="article-title">The role of active-site residues in naphthalene dioxygenase</div>, <div class="source">J. Ind. Microbiol. Biotechnol.</div> , <div class="volume">30</div>, <div class="fpage">271</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20active-site%20residues%20in%20naphthalene%20dioxygenase&amp;author=R.E.%20Parales&amp;publication_year=2003&amp;journal=J.%20Ind.%20Microbiol.%20Biotechnol.&amp;volume=30&amp;pages=271-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10295-003-0043-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10295-003-0043-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10295-003-0043-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12695887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Vilchez-Vargas</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Geffers</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Suarez-Diez</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Analysis of the microbial gene landscape and transcriptome for aromatic pollutants and alkane degradation using a novel internally calibrated microarray system</div>, <div class="source">Environ. Microbiol.</div> , <div class="volume">15</div>, <div class="fpage">1016</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20microbial%20gene%20landscape%20and%20transcriptome%20for%20aromatic%20pollutants%20and%20alkane%20degradation%20using%20a%20novel%20internally%20calibrated%20microarray%20system&amp;author=R.%20Vilchez-Vargas&amp;author=R.%20Geffers&amp;author=M.%20Suarez-Diez&amp;publication_year=2013&amp;journal=Environ.%20Microbiol.&amp;volume=15&amp;pages=1016-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2013.15.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2013.15.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2013.15.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22515215" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guazzaroni</div>
               <div class="given-names">M.E.</div>
            </div>, <div class="name string-name"><div class="surname">Herbst</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Lores</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Metaproteogenomic insights beyond bacterial response to naphthalene exposure and bio-stimulation</div>, <div class="source">ISME J.</div> , <div class="volume">7</div>, <div class="fpage">122</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metaproteogenomic%20insights%20beyond%20bacterial%20response%20to%20naphthalene%20exposure%20and%20bio-stimulation&amp;author=M.E.%20Guazzaroni&amp;author=F.A.%20Herbst&amp;author=I.%20Lores&amp;publication_year=2012&amp;journal=ISME%20J.&amp;volume=7&amp;pages=122-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2012.82" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2012.82" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2012.82"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22832345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Leplae</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Lima-Mendez</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Toussaint</div> <div class="given-names">A.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">ACLAME: a classification of mobile genetic elements, update 2010</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">D57</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ACLAME%3A%20a%20classification%20of%20mobile%20genetic%20elements%2C%20update%202010&amp;author=R.%20Leplae&amp;author=G.%20Lima-Mendez&amp;author=A.%20Toussaint&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=D57-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkp938" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkp938" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkp938"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19933762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kristensen</div>
               <div class="given-names">D.M.</div>
            </div>, <div class="name string-name"><div class="surname">Waller</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Mushegian</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E.V.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Orthologous gene clusters and taxon signature genes for viruses of prokaryotes</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">195</div>, <div class="fpage">941</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Orthologous%20gene%20clusters%20and%20taxon%20signature%20genes%20for%20viruses%20of%20prokaryotes&amp;author=D.M.%20Kristensen&amp;author=A.S.%20Waller&amp;author=T.%20Yamada&amp;author=P.%20Bork&amp;author=A.R.%20Mushegian&amp;author=E.V.%20Koonin&amp;publication_year=2013&amp;journal=J.%20Bacteriol.&amp;volume=195&amp;pages=941-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01801-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01801-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01801-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23222723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhou</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Wishart</div> <div class="given-names">D.S.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">PHAST: a fast phage search tool</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">39</div>, <div class="fpage">W347</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHAST%3A%20a%20fast%20phage%20search%20tool&amp;author=Y.%20Zhou&amp;author=Y.%20Liang&amp;author=K.H.%20Lynch&amp;author=J.J.%20Dennis&amp;author=D.S.%20Wishart&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W347-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ben Zakour</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Andonov</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">GenoFrag: software to design primers optimized for whole genome scanning by long-range PCR amplification</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">32</div>, <div class="fpage">17</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenoFrag%3A%20software%20to%20design%20primers%20optimized%20for%20whole%20genome%20scanning%20by%20long-range%20PCR%20amplification&amp;author=N.%20Ben%20Zakour&amp;author=M.%20Gautier&amp;author=R.%20Andonov&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=17-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg928" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg928" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg928"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Peng</div>
               <div class="given-names">R.H.</div>
            </div>, <div class="name string-name"><div class="surname">Xiong</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Xue</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Microbial biodegradation of polyaromatic hydrocarbons</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="volume">32</div>, <div class="fpage">927</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20biodegradation%20of%20polyaromatic%20hydrocarbons&amp;author=R.H.%20Peng&amp;author=A.S.%20Xiong&amp;author=Y.%20Xue&amp;publication_year=2008&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=32&amp;pages=927-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00127.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00127.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00127.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18662317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hartmann</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Niklaus</div> <div class="given-names">P.A.</div></div>, <div class="name string-name"><div class="surname">Zimmermann</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Resistance and resilience of the forest soil microbiome to logging-associated compaction</div>, <div class="source">ISME J.</div> , <div class="volume">8</div>, <div class="fpage">226</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resistance%20and%20resilience%20of%20the%20forest%20soil%20microbiome%20to%20logging-associated%20compaction&amp;author=M.%20Hartmann&amp;author=P.A.%20Niklaus&amp;author=S.%20Zimmermann&amp;publication_year=2014&amp;journal=ISME%20J.&amp;volume=8&amp;pages=226-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2013.141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2013.141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2013.141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24030594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Oh</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Caro-Quintero</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tsementzi</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Metagenomic insights into the evolution, function, and complexity of the planktonic microbial community of Lake Lanier, a temperate freshwater ecosystem</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">77</div>, <div class="fpage">6000</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20insights%20into%20the%20evolution%2C%20function%2C%20and%20complexity%20of%20the%20planktonic%20microbial%20community%20of%20Lake%20Lanier%2C%20a%20temperate%20freshwater%20ecosystem&amp;author=S.%20Oh&amp;author=A.%20Caro-Quintero&amp;author=D.%20Tsementzi&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=77&amp;pages=6000-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.00107-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.00107-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.00107-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21764968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yergeau</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Sanschagrin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Beaumier</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Greer</div> <div class="given-names">C.W.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Metagenomic analysis of the bioremediation of diesel-contaminated Canadian high arctic soils</div>, <div class="source">PLoS ONE</div> , <div class="volume">7</div>, <div class="fpage">e30058</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20of%20the%20bioremediation%20of%20diesel-contaminated%20Canadian%20high%20arctic%20soils&amp;author=E.%20Yergeau&amp;author=S.%20Sanschagrin&amp;author=D.%20Beaumier&amp;author=C.W.%20Greer&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e30058" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Perez-Pantoja</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Donoso</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Agullo</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Genomic analysis of the potential for aromatic compounds biodegradation in <em>Burkholderiales</em></div>, <div class="source">Environ. Microbiol.</div> , <div class="volume">14</div>, <div class="fpage">1091</div>–<div class="lpage">117</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20the%20potential%20for%20aromatic%20compounds%20biodegradation%20in%20Burkholderiales&amp;author=D.%20Perez-Pantoja&amp;author=R.%20Donoso&amp;author=L.%20Agullo&amp;publication_year=2012&amp;journal=Environ.%20Microbiol.&amp;volume=14&amp;pages=1091-117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2012.14.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2012.14.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2012.14.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22026719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Leekitcharoenphon</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Buzard</div> <div class="given-names">G.S.</div></div>, <div class="name string-name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Comparative genomics in the genus <em>Burkholderia</em></div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Coenye</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mahenthiralingam</div> <div class="given-names">E.</div></div></div> (eds), <div class="source">Burkholderia: From Genomes to Function</div> . <div class="publisher-name">Caister Academic Press</div>: <div class="publisher-loc">Norfolk, UK</div>, pp. <div class="fpage">31</div>–<div class="lpage">50</div>.</div></div></div></div><div content-id="DSV023C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Silby</div>
               <div class="given-names">M.W.</div>
            </div>, <div class="name string-name"><div class="surname">Winstanley</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Godfrey</div> <div class="given-names">S.A.C.</div></div>, <div class="name string-name"><div class="surname">Levy</div> <div class="given-names">S.B.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">R.W.</div></div></div>
          <div class="year">2011</div>, <div class="article-title"><em>Pseudomonas</em> genomes: diverse and adaptable</div>, <div class="source">FEMS Microbiol. Rev.</div> , <div class="volume">35</div>, <div class="fpage">652</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudomonas%20genomes%3A%20diverse%20and%20adaptable&amp;author=M.W.%20Silby&amp;author=C.%20Winstanley&amp;author=S.A.C.%20Godfrey&amp;author=S.B.%20Levy&amp;author=R.W.%20Jackson&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=35&amp;pages=652-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2011.00269.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2011.00269.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2011.00269.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21361996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fierer</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Ladau</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Clemente</div> <div class="given-names">J.C.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Reconstructing the microbial diversity and function of pre-agricultural tallgrass prairie soils in the United States</div>, <div class="source">Science</div> , <div class="volume">342</div>, <div class="fpage">621</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20the%20microbial%20diversity%20and%20function%20of%20pre-agricultural%20tallgrass%20prairie%20soils%20in%20the%20United%20States&amp;author=N.%20Fierer&amp;author=J.%20Ladau&amp;author=J.C.%20Clemente&amp;publication_year=2013&amp;journal=Science&amp;volume=342&amp;pages=621-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1243768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1243768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1243768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kitano</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2004</div>, <div class="article-title">Biological robustness</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">5</div>, <div class="fpage">826</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20robustness&amp;author=H.%20Kitano&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=826-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15520792" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Souza</div>
               <div class="given-names">R.C.</div>
            </div>, <div class="name string-name"><div class="surname">Hungria</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cantao</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Vasconcelos</div> <div class="given-names">A.T.R.</div></div>, <div class="name string-name"><div class="surname">Nogueira</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Vicente</div> <div class="given-names">V.A.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Metagenomic analysis reveals microbial functional redundancies and specificities in a soil under different tillage and crop-management regimes</div>, <div class="source">Appl. Soil Ecol.</div> , <div class="volume">86</div>, <div class="fpage">106</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analysis%20reveals%20microbial%20functional%20redundancies%20and%20specificities%20in%20a%20soil%20under%20different%20tillage%20and%20crop-management%20regimes&amp;author=R.C.%20Souza&amp;author=M.%20Hungria&amp;author=M.E.%20Cantao&amp;author=A.T.R.%20Vasconcelos&amp;author=M.A.%20Nogueira&amp;author=V.A.%20Vicente&amp;publication_year=2015&amp;journal=Appl.%20Soil%20Ecol.&amp;volume=86&amp;pages=106-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.apsoil.2014.10.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.apsoil.2014.10.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.apsoil.2014.10.010"> </span></div> </div></div></div></div></div><div content-id="DSV023C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ogawa</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Miyashita</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Chakrabarty</div> <div class="given-names">A.M.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Microbial genes and enzymes in the degradation of chlorinated compounds</div>, <div class="source">Chem. Rec.</div> , <div class="volume">3</div>, <div class="fpage">158</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20genes%20and%20enzymes%20in%20the%20degradation%20of%20chlorinated%20compounds&amp;author=N.%20Ogawa&amp;author=K.%20Miyashita&amp;author=A.M.%20Chakrabarty&amp;publication_year=2003&amp;journal=Chem.%20Rec.&amp;volume=3&amp;pages=158-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1528-0691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1528-0691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1528-0691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12900936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morimoto</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Togami</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ogawa</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hasebe</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">T.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Analysis of a bacterial community in 3-chlorobenzoate-contaminated soil by PCR-DGGE targeting the 16S rRNA gene and benzoate 1,2-dioxygenase gene (<em>benA</em>)</div>, <div class="source">Microbes Environ.</div> , <div class="volume">20</div>, <div class="fpage">151</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20a%20bacterial%20community%20in%203-chlorobenzoate-contaminated%20soil%20by%20PCR-DGGE%20targeting%20the%2016S%20rRNA%20gene%20and%20benzoate%201%2C2-dioxygenase%20gene%20(benA)&amp;author=S.%20Morimoto&amp;author=K.%20Togami&amp;author=N.%20Ogawa&amp;author=A.%20Hasebe&amp;author=T.%20Fujii&amp;publication_year=2005&amp;journal=Microbes%20Environ.&amp;volume=20&amp;pages=151-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.20.151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.20.151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.20.151"> </span></div> </div></div></div></div></div><div content-id="DSV023C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chang</div>
               <div class="given-names">H.K.</div>
            </div>, <div class="name string-name"><div class="surname">Zylstra</div> <div class="given-names">G.J.</div></div></div>
          <div class="year">1998</div>, <div class="article-title">Novel organization of the genes for phthalate degradation from <em>Burkholderia cepacia</em> DBO1</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">180</div>, <div class="fpage">6529</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20organization%20of%20the%20genes%20for%20phthalate%20degradation%20from%20Burkholderia%20cepacia%20DBO1&amp;author=H.K.%20Chang&amp;author=G.J.%20Zylstra&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=6529-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9851995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Balashova</div>
               <div class="given-names">N.V.</div>
            </div>, <div class="name string-name"><div class="surname">Stolz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Knackmuss</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Kosheleva</div> <div class="given-names">I.A.</div></div>, <div class="name string-name"><div class="surname">Naumov</div> <div class="given-names">A.V.</div></div>, <div class="name string-name"><div class="surname">Boronin</div> <div class="given-names">A.M.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Purification and characterization of a salicylate hydroxylase involved in 1-hydroxy-2-naphthoic acid hydroxylation from the naphthalene and phenanthrene-degrading bacterial strain <em>Pseudomonas putida</em> BS202-P1</div>, <div class="source">Biodegradation</div> , <div class="volume">12</div>, <div class="fpage">179</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20a%20salicylate%20hydroxylase%20involved%20in%201-hydroxy-2-naphthoic%20acid%20hydroxylation%20from%20the%20naphthalene%20and%20phenanthrene-degrading%20bacterial%20strain%20Pseudomonas%20putida%20BS202-P1&amp;author=N.V.%20Balashova&amp;author=A.%20Stolz&amp;author=H.J.%20Knackmuss&amp;author=I.A.%20Kosheleva&amp;author=A.V.%20Naumov&amp;author=A.M.%20Boronin&amp;publication_year=2001&amp;journal=Biodegradation&amp;volume=12&amp;pages=179-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1013126723719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1013126723719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1013126723719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11826899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stingley</div>
               <div class="given-names">R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Khan</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Cerniglia</div> <div class="given-names">C.E.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Molecular characterization of a phenanthrene degradation pathway in <em>Mycobacterium vanbaalenii</em> PYR-1</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="volume">322</div>, <div class="fpage">133</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20a%20phenanthrene%20degradation%20pathway%20in%20Mycobacterium%20vanbaalenii%20PYR-1&amp;author=R.L.%20Stingley&amp;author=A.A.%20Khan&amp;author=C.E.%20Cerniglia&amp;publication_year=2004&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=322&amp;pages=133-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2004.07.089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2004.07.089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2004.07.089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15313184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Moody</div>
               <div class="given-names">J.D.</div>
            </div>, <div class="name string-name"><div class="surname">Freeman</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Doerge</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Cerniglia</div> <div class="given-names">C.E.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">Degradation of phenanthrene and anthracene by cell suspensions of <em>Mycobacterium</em> sp. strain PYR-1</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="volume">67</div>, <div class="fpage">1476</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degradation%20of%20phenanthrene%20and%20anthracene%20by%20cell%20suspensions%20of%20Mycobacterium%20sp.%20strain%20PYR-1&amp;author=J.D.%20Moody&amp;author=J.P.%20Freeman&amp;author=D.R.%20Doerge&amp;author=C.E.%20Cerniglia&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=67&amp;pages=1476-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.4.1476-1483.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.4.1476-1483.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.67.4.1476-1483.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Le Digabel</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Demaneche</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Benoit</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Fayolle-Guichard</div> <div class="given-names">F.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Ethyl <em>tert</em>-butyl ether (ETBE) biodegradation by a syntrophic association of <em>Rhodococcus</em> sp. IFP 2042 and <em>Bradyrhizobium</em> sp. IFP 2049 isolated from a polluted aquifer</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="volume">97</div>, <div class="fpage">10531</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ethyl%20tert-butyl%20ether%20(ETBE)%20biodegradation%20by%20a%20syntrophic%20association%20of%20Rhodococcus%20sp.%20IFP%202042%20and%20Bradyrhizobium%20sp.%20IFP%202049%20isolated%20from%20a%20polluted%20aquifer&amp;author=Y.%20Le%20Digabel&amp;author=S.%20Demaneche&amp;author=Y.%20Benoit&amp;author=T.M.%20Vogel&amp;author=F.%20Fayolle-Guichard&amp;publication_year=2013&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=97&amp;pages=10531-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-013-4803-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-013-4803-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-013-4803-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23474617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kato</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Ogawa</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ohtsubo</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Complete genome sequence of a phenanthrene degrader, <em>Mycobacterium</em> sp. strain EPa45 (NBRC 110737), isolated from a phenanthrene-degrading consortium</div>, <div class="source">Genome Announc.</div> , <div class="volume">3</div>, <div class="comment">e00782-15</div>.</div></div></div></div><div content-id="DSV023C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Summer</div>
               <div class="given-names">E.J.</div>
            </div>, <div class="name string-name"><div class="surname">Gill</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Upton</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gonzalez</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Young</div> <div class="given-names">R.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Role of phages in the pathogenesis of <em>Burkholderia</em>, or 'where are the toxin genes in <em>Burkholderia</em> phages?</div>’, <div class="source">Curr. Opin. Microbiol.</div> , <div class="volume">10</div>, <div class="fpage">410</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20phages%20in%20the%20pathogenesis%20of%20Burkholderia%2C%20or%20'where%20are%20the%20toxin%20genes%20in%20Burkholderia%20phages%3F&amp;author=E.J.%20Summer&amp;author=J.J.%20Gill&amp;author=C.%20Upton&amp;author=C.F.%20Gonzalez&amp;author=R.%20Young&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=10&amp;pages=410-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2007.05.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2007.05.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2007.05.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17719265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lynch</div>
               <div class="given-names">K.H.</div>
            </div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">J.J.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Genomics of <em>Burkholderia</em> phages</div>. In: <div class="person-group"><div class="name string-name"><div class="surname">Coenye</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mahenthiralingam</div> <div class="given-names">E.</div></div></div> (eds), <div class="source">Burkholderia: From Genomes to Function</div> . <div class="publisher-name">Caister Academic Press</div>: <div class="publisher-loc">Norfolk, UK</div>, pp. <div class="fpage">230</div>–<div class="lpage">49</div>.</div></div></div></div><div content-id="DSV023C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rodriguez-Brito</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wegley</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Viral and microbial community dynamics in four aquatic environments</div>, <div class="source">ISME J.</div> , <div class="volume">4</div>, <div class="fpage">739</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Viral%20and%20microbial%20community%20dynamics%20in%20four%20aquatic%20environments&amp;author=B.%20Rodriguez-Brito&amp;author=L.%20Li&amp;author=L.%20Wegley&amp;publication_year=2010&amp;journal=ISME%20J.&amp;volume=4&amp;pages=739-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2010.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2010.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2010.1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stenuit</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Agathos</div> <div class="given-names">S.N.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Deciphering microbial community robustness through synthetic ecology and molecular systems synecology</div>, <div class="source">Curr. Opin. Biotechnol.</div> , <div class="volume">33</div>, <div class="fpage">305</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20microbial%20community%20robustness%20through%20synthetic%20ecology%20and%20molecular%20systems%20synecology&amp;author=B.%20Stenuit&amp;author=S.N.%20Agathos&amp;publication_year=2015&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;volume=33&amp;pages=305-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.copbio.2015.03.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.copbio.2015.03.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.copbio.2015.03.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25880923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV023C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV023C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Muller</div>
               <div class="given-names">E.E.L.</div>
            </div>, <div class="name string-name"><div class="surname">Pinel</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Laczny</div> <div class="given-names">C.C.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Community-integrated omics links dominance of a microbial generalist to fine-tuned resource usage</div>, <div class="source">Nat. Commun.</div> , <div class="volume">5</div>, <div class="fpage">5603</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Community-integrated%20omics%20links%20dominance%20of%20a%20microbial%20generalist%20to%20fine-tuned%20resource%20usage&amp;author=E.E.L.%20Muller&amp;author=N.%20Pinel&amp;author=C.C.%20Laczny&amp;publication_year=2014&amp;journal=Nat.%20Commun.&amp;volume=5&amp;pages=5603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms6603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms6603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms6603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25424998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462440&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/6/10.1093_dnares_dsv023/1/dsv023_Supplementary_Data.zip?Expires=2147483647&amp;Signature=lgKdFf8eajD5BG97whzm55zqizjT51A18jbUuqD3uIXcX0Cu3nBz2erFBE3yzuKzmYU7kpPVuo8TKts-YcJ3S~VwqRpOo1ZeIBra2pbWacrr3D16JRs7xKPsQ4~pyZkDwtcFGzpOZRBMdEp6vtCU-iuTmkQMB~Qy5i~8qvd7FTQjBZFsN0eD3kPjl9cO-C4sQqF4jrtBR3jLA65xzLsfcsg9~8C-WuEUR~eCPDhD3I6o78hKsdAANdhR9Qem3d35kvIQadiUw316aJHPuStilVkSDEtJh4ecGvl2ypeD9pOMm-cGpWTB1-xb1alXnN4R~9-DHoXisSHPZcAdXyOtHw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">846</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462440"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000366631700003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f6%2f413%2f2462440">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000366631700003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F6%2F413%2F2462440" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462440'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Time-series metagenomic analysis reveals robustness of soil microbiome against chemical disturbance" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f6%2f413%2f2462440">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

