

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1583,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/6/325/363736","@type":"ScholarlyArticle","name":"A Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule)","datePublished":"2009-10-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2009-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp021","keywords":["chromosome","nucleotide composition","Chargaff's second parity rule","intra-strand frequency distribution parity","DNA replication"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/6/325/363736","author":[{"name":"Powdel, B.R.","affiliation":"1  Department of Mathematical Sciences ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Satapathy, Siddhartha Sankar","affiliation":"2  Department of Computer Science and Engineering ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Kumar, Aditya","affiliation":"3  Department of Molecular Biology and Biotechnology ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Jha, Pankaj Kumar","affiliation":"3  Department of Molecular Biology and Biotechnology ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Buragohain, Alak Kumar","affiliation":"3  Department of Molecular Biology and Biotechnology ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Borah, Munindra","affiliation":"1  Department of Mathematical Sciences ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"},{"name":"Ray, Suvendra Kumar","affiliation":"3  Department of Molecular Biology and Biotechnology ,  Tezpur University ,  Tezpur, Assam 784 028 ,  India","@type":"Person"}],"description":"Abstract.  Chargaff's rule of intra-strand parity (ISP) between complementary mono/oligonucleotides in chromosomes is well established in the scientific literat","pageStart":"325","pageEnd":"343","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff&#39;s Second Parity Rule)" />
    <meta property="og:description" content="Abstract.  Chargaff&#39;s rule of intra-strand parity (ISP) between complementary mono/oligonucleotides in chromosomes is well established in the scientific literat" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/6/325/363736" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-10-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Powdel, B.R." /><meta name="citation_author_institution" content="Department of Mathematical Sciences, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Satapathy, Siddhartha Sankar" /><meta name="citation_author_institution" content="Department of Computer Science and Engineering, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Kumar, Aditya" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Jha, Pankaj Kumar" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Buragohain, Alak Kumar" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Borah, Munindra" /><meta name="citation_author_institution" content="Department of Mathematical Sciences, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_author" content="Ray, Suvendra Kumar" /><meta name="citation_author_institution" content="Department of Molecular Biology and Biotechnology, Tezpur University, Tezpur, Assam 784 028, India" /><meta name="citation_title" content="A Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule)" /><meta name="citation_firstpage" content="325" /><meta name="citation_lastpage" content="343" /><meta name="citation_doi" content="10.1093/dnares/dsp021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2009/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Structure and function of nucleic acids as cell constituents; Fed. Proc.;  citation_year=1951;  citation_volume=10;  citation_pages=654-9; " /><meta name="citation_reference" content="citation_title=Chargaff's legacy; Gene;  citation_year=2000;  citation_volume=261;  citation_pages=127-37; " /><meta name="citation_reference" content="citation_title=Molecular structure of nucleic acids: a structure for deoxyribonucleic acid; Nature;  citation_year=1953;  citation_volume=171;  citation_pages=737-8; " /><meta name="citation_reference" content="citation_title=Separation of B. subtilis DNA into complementary strands, II. Template functions and composition as determined by transcription with RNA polymerases; Proc. Natl Acad. Sci. USA;  citation_year=1968;  citation_volume=60;  citation_pages=915-20; " /><meta name="citation_reference" content="citation_title=Separation of microbial deoxyribonucleic acids into complementary strands; Proc. Natl Acad. Sci. USA;  citation_year=1969;  citation_volume=63;  citation_pages=152-9; " /><meta name="citation_reference" content="citation_title=Symmetry observed in long nucleotide sequences; Nucleic Acid Res.;  citation_year=1993;  citation_volume=21;  citation_pages=2797-800; " /><meta name="citation_reference" content="citation_title=Compositional symmetries in complete genomes; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=557-9; " /><meta name="citation_reference" content="citation_title=Why are complementary DNA strands symmetric?; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=1021-33; " /><meta name="citation_reference" content="citation_title=Compositional Symmetry of DNA duplex in bacterial genomes; Curr. Sci.;  citation_year=2005;  citation_volume=89;  citation_pages=374-84; " /><meta name="citation_reference" content="citation_title=Asymptotically increasing compliance of genomes with Chargaff's second parity rules through inversions and inverted transpositions; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=17828-33; " /><meta name="citation_reference" content="citation_title=A test of Chargaff's second rule; Biochem. Biophys. Res. Commun.;  citation_year=2006;  citation_volume=340;  citation_pages=1-5; " /><meta name="citation_reference" content="citation_title=Deviation from Chargaff's second parity rule in organellar DNA insights into the evolution of organellar genomes; Gene;  citation_year=2006;  citation_volume=381;  citation_pages=34-41; " /><meta name="citation_reference" content="citation_title=Mismatch repair error implies Chargaff's second parity rule, arXiv:0704.2191v2 [q-bio.GN], 1–9;  citation_url=http://arxiv.org/PS_cache/arxiv/pdf/0704/0704.2191v2.pdf;  citation_URL_Access_Date=];  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Intra-strand parity rules of DNA base composition and usage biases of synonymous codons; J. Mol. Evol.;  citation_year=1995;  citation_volume=40;  citation_pages=318-25; " /><meta name="citation_reference" content="citation_title=Properties of a general model of DNA evolution under no-strand-bias conditions; J. Mol. Evol.;  citation_year=1995;  citation_volume=40;  citation_pages=326-30; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms; Gene;  citation_year=1999;  citation_volume=238;  citation_pages=65-77; " /><meta name="citation_reference" content="citation_title=A study on the correlation of nucleotide skews and the positioning of the origin of replication: different modes of replication in bacterial species; Nucleic Acid Res.;  citation_year=2005;  citation_volume=33;  citation_pages=6816-22; " /><meta name="citation_reference" content="citation_title=Translation-coupled violation of parity rule 2 in human genes is not the cause of heterogeneity of the DNA G+C content of third codon position; Gene;  citation_year=1999;  citation_volume=238;  citation_pages=53-8; " /><meta name="citation_reference" content="citation_title=Universal replication biases in bacteria; Mol. Microbiol.;  citation_year=1999;  citation_volume=32;  citation_pages=11-6; " /><meta name="citation_reference" content="citation_title=Asymmetric directional mutation pressures in bacteria; citation_text=research0058.1–0058.14; Genome Biol.;  citation_year=2002;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Analyzing genomes with cumulative skew diagrams; Nucleic Acid Res.;  citation_year=1998;  citation_volume=26;  citation_pages=2286-90; " /><meta name="citation_reference" content="citation_title=Strand asymmetries in DNA evolution; Trends Genet.;  citation_year=1997;  citation_volume=13;  citation_pages=240-5; " /><meta name="citation_reference" content="citation_title=Cellular DNA replicases: components and dynamics at the replication fork; Annu. Rev. Biochem.;  citation_year=2005;  citation_volume=74;  citation_pages=283-314; " /><meta name="citation_reference" content="citation_title=Accounting units in DNA; J. Theor. Biol.;  citation_year=1999;  citation_volume=197;  citation_pages=51-61; " /><meta name="citation_reference" content="citation_title=Asymmetries generated by transcription-coupled repair in enterobacterial genes; Science;  citation_year=1996;  citation_volume=272;  citation_pages=107-9; " /><meta name="citation_reference" content="citation_title=Base composition skews, replication, and orientation in 12 prokaryote genomes; J. Mol. Evol.;  citation_year=1998;  citation_volume=47;  citation_pages=691-6; " /><meta name="citation_reference" content="citation_title=Skew of mononucleotide frequencies, relative abundance of dinucleotides and DNA strand asymmetry; J. Mol. Evol.;  citation_year=2001;  citation_volume=53;  citation_pages=364-76; " /><meta name="citation_reference" content="citation_title=Algorithm AS 288: Exact two-sample Smirnov test for arbitrary distributions; Appl. Stat.;  citation_year=1994;  citation_volume=43;  citation_pages=265-84; " /><meta name="citation_reference" content="citation_title=Confidence limits for an unknown distribution function; Ann. Math. Stat.;  citation_year=1941;  citation_volume=12;  citation_pages=461-3; " /><meta name="citation_reference" content="citation_title=On the estimation of the discrepancy between empirical curves of distribution for two independent samples; Bull. Moscow Univ.;  citation_year=1939;  citation_volume=2;  citation_pages=3-14; " /><meta name="citation_reference" content="citation_title=Oriloc: prediction of replication boundaries in annotated bacterial chromosomes; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=560-1; " /><meta name="citation_reference" content="citation_title=Origin of replication in circular prokaryotic chromosomes; Environ. Microbiol.;  citation_year=2006;  citation_volume=8;  citation_pages=353-61; " /><meta name="citation_reference" content="citation_title=Identification of replication origins in prokaryotic genomes; Brief. Bioinform.;  citation_year=2008;  citation_volume=9;  citation_pages=376-91; " /><meta name="citation_reference" content="citation_title=Variation in the strength of selected codon usage bias among bacteria; Nucleic Acid Res.;  citation_year=2005;  citation_volume=33;  citation_pages=1141-53; " /><meta name="citation_reference" content="citation_title=A new perspective on isochore evolution; Gene;  citation_year=2006;  citation_volume=385;  citation_pages=71-4; " /><meta name="citation_reference" content="citation_title=Strong doublet preferences in nucleotide sequences and DNA geometry; J. Mol. Evol.;  citation_year=1984;  citation_volume=20;  citation_pages=111-9; " /><meta name="citation_reference" content="citation_title=A stem–loop ‘kissing’ model for the initiation of recombination and the origin of introns; Mol. Biol. Evol.;  citation_year=1995;  citation_volume=12;  citation_pages=949-58; " /><meta name="citation_reference" content="citation_title=Negative correlation between compositional symmetries and local recombination rates; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3951-8; " /><meta name="citation_reference" content="citation_title=Evolutionary implications of inversions that have caused intra-strand parity in DNA; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=160" /><meta name="citation_reference" content="citation_title=The replication-related organization of bacterial genomes; Microbiology;  citation_year=2004;  citation_volume=150;  citation_pages=1609-27; " /><meta name="citation_reference" content="citation_title=Gene essentiality determines chromosome organization in bacteria; Nucleic Acid Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6570-7; " /><meta name="citation_reference" content="citation_title=The organization of the bacterial genome; Annu. Rev. Genet.;  citation_year=2008;  citation_volume=42;  citation_pages=211-33; " /><meta name="citation_reference" content="citation_title=DPVweb: An open access internet resource on plant viruses and virus diseases; Outlooks on Pest Management;  citation_year=2005;  citation_volume=16;  citation_pages=268-70; " /><meta name="citation_reference" content="citation_title=DPVweb: a comprehensive database of plant and fungal virus genes and genomes;  citation_url=http://www.dpvweb.net/; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34; " /><meta name="citation_reference" content="citation_title=The selection-mutation-drift theory of synonymous codon usage; Genetics;  citation_year=1991;  citation_volume=192;  citation_pages=897-907; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/6/325/1039502/dsp021.pdf" /><meta name="description" content="Abstract.  Chargaff's rule of intra-strand parity (ISP) between complementary mono/oligonucleotides in chromosomes is well established in the scientific literat" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/6/325/363736" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/6/325/363736" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/6/325/363736");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp021';
    </script>
<title>
	Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./363736" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbVgPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQb3dkZWwsIEIuUi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1hdGhlbWF0aWNhbCBTY2llbmNlcywgVGV6cHVyIFVuaXZlcnNpdHksIFRlenB1ciwgQXNzYW0gNzg0IDAyOCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRhcGF0aHksIFNpZGRoYXJ0aGEgU2Fua2FyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRlciBTY2llbmNlIGFuZCBFbmdpbmVlcmluZywgVGV6cHVyIFVuaXZlcnNpdHksIFRlenB1ciwgQXNzYW0gNzg0IDAyOCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdW1hciwgQWRpdHlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgQmlvbG9neSBhbmQgQmlvdGVjaG5vbG9neSwgVGV6cHVyIFVuaXZlcnNpdHksIFRlenB1ciwgQXNzYW0gNzg0IDAyOCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaGEsIFBhbmthaiBLdW1hciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEJpb3RlY2hub2xvZ3ksIFRlenB1ciBVbml2ZXJzaXR5LCBUZXpwdXIsIEFzc2FtIDc4NCAwMjgsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQnVyYWdvaGFpbiwgQWxhayBLdW1hciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEJpb3RlY2hub2xvZ3ksIFRlenB1ciBVbml2ZXJzaXR5LCBUZXpwdXIsIEFzc2FtIDc4NCAwMjgsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9yYWgsIE11bmluZHJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNYXRoZW1hdGljYWwgU2NpZW5jZXMsIFRlenB1ciBVbml2ZXJzaXR5LCBUZXpwdXIsIEFzc2FtIDc4NCAwMjgsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmF5LCBTdXZlbmRyYSBLdW1hciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEJpb3RlY2hub2xvZ3ksIFRlenB1ciBVbml2ZXJzaXR5LCBUZXpwdXIsIEFzc2FtIDc4NCAwMjgsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBIFN0dWR5IGluIEVudGlyZSBDaHJvbW9zb21lcyBvZiBWaW9sYXRpb25zIG9mIHRoZSBJbnRyYS1zdHJhbmQgUGFyaXR5IG9mIENvbXBsZW1lbnRhcnkgTnVjbGVvdGlkZXMgKENoYXJnYWZmJ3MgU2Vjb25kIFBhcml0eSBSdWxlKSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjMyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNwMDIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDkvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uIG9mIG51Y2xlaWMgYWNpZHMgYXMgY2VsbCBjb25zdGl0dWVudHM7IEZlZC4gUHJvYy47ICBjaXRhdGlvbl95ZWFyPTE5NTE7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02NTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmdhZmYncyBsZWdhY3k7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjYxOyAgY2l0YXRpb25fcGFnZXM9MTI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgc3RydWN0dXJlIG9mIG51Y2xlaWMgYWNpZHM6IGEgc3RydWN0dXJlIGZvciBkZW94eXJpYm9udWNsZWljIGFjaWQ7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk1MzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz03MzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VwYXJhdGlvbiBvZiBCLiBzdWJ0aWxpcyBETkEgaW50byBjb21wbGVtZW50YXJ5IHN0cmFuZHMsIElJLiBUZW1wbGF0ZSBmdW5jdGlvbnMgYW5kIGNvbXBvc2l0aW9uIGFzIGRldGVybWluZWQgYnkgdHJhbnNjcmlwdGlvbiB3aXRoIFJOQSBwb2x5bWVyYXNlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk2ODsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTkxNS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VwYXJhdGlvbiBvZiBtaWNyb2JpYWwgZGVveHlyaWJvbnVjbGVpYyBhY2lkcyBpbnRvIGNvbXBsZW1lbnRhcnkgc3RyYW5kczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk2OTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTE1Mi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW1tZXRyeSBvYnNlcnZlZCBpbiBsb25nIG51Y2xlb3RpZGUgc2VxdWVuY2VzOyBOdWNsZWljIEFjaWQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTI3OTctODAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wb3NpdGlvbmFsIHN5bW1ldHJpZXMgaW4gY29tcGxldGUgZ2Vub21lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz01NTctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2h5IGFyZSBjb21wbGVtZW50YXJ5IEROQSBzdHJhbmRzIHN5bW1ldHJpYz87IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTAyMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb25hbCBTeW1tZXRyeSBvZiBETkEgZHVwbGV4IGluIGJhY3RlcmlhbCBnZW5vbWVzOyBDdXJyLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9Mzc0LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3ltcHRvdGljYWxseSBpbmNyZWFzaW5nIGNvbXBsaWFuY2Ugb2YgZ2Vub21lcyB3aXRoIENoYXJnYWZmJ3Mgc2Vjb25kIHBhcml0eSBydWxlcyB0aHJvdWdoIGludmVyc2lvbnMgYW5kIGludmVydGVkIHRyYW5zcG9zaXRpb25zOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTE3ODI4LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHRlc3Qgb2YgQ2hhcmdhZmYncyBzZWNvbmQgcnVsZTsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0MDsgIGNpdGF0aW9uX3BhZ2VzPTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2aWF0aW9uIGZyb20gQ2hhcmdhZmYncyBzZWNvbmQgcGFyaXR5IHJ1bGUgaW4gb3JnYW5lbGxhciBETkEgaW5zaWdodHMgaW50byB0aGUgZXZvbHV0aW9uIG9mIG9yZ2FuZWxsYXIgZ2Vub21lczsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODE7ICBjaXRhdGlvbl9wYWdlcz0zNC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWlzbWF0Y2ggcmVwYWlyIGVycm9yIGltcGxpZXMgQ2hhcmdhZmYncyBzZWNvbmQgcGFyaXR5IHJ1bGUsIGFyWGl2OjA3MDQuMjE5MXYyIFtxLWJpby5HTl0sIDHigJM5OyAgY2l0YXRpb25fdXJsPWh0dHA6Ly9hcnhpdi5vcmcvUFNfY2FjaGUvYXJ4aXYvcGRmLzA3MDQvMDcwNC4yMTkxdjIucGRmOyAgY2l0YXRpb25fVVJMX0FjY2Vzc19EYXRlPV07ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJhLXN0cmFuZCBwYXJpdHkgcnVsZXMgb2YgRE5BIGJhc2UgY29tcG9zaXRpb24gYW5kIHVzYWdlIGJpYXNlcyBvZiBzeW5vbnltb3VzIGNvZG9uczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTMxOC0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvcGVydGllcyBvZiBhIGdlbmVyYWwgbW9kZWwgb2YgRE5BIGV2b2x1dGlvbiB1bmRlciBuby1zdHJhbmQtYmlhcyBjb25kaXRpb25zOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MzI2LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3ltbWV0cmljIHN1YnN0aXR1dGlvbiBwYXR0ZXJuczogYSByZXZpZXcgb2YgcG9zc2libGUgdW5kZXJseWluZyBtdXRhdGlvbmFsIG9yIHNlbGVjdGl2ZSBtZWNoYW5pc21zOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzODsgIGNpdGF0aW9uX3BhZ2VzPTY1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0dWR5IG9uIHRoZSBjb3JyZWxhdGlvbiBvZiBudWNsZW90aWRlIHNrZXdzIGFuZCB0aGUgcG9zaXRpb25pbmcgb2YgdGhlIG9yaWdpbiBvZiByZXBsaWNhdGlvbjogZGlmZmVyZW50IG1vZGVzIG9mIHJlcGxpY2F0aW9uIGluIGJhY3RlcmlhbCBzcGVjaWVzOyBOdWNsZWljIEFjaWQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY4MTYtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uLWNvdXBsZWQgdmlvbGF0aW9uIG9mIHBhcml0eSBydWxlIDIgaW4gaHVtYW4gZ2VuZXMgaXMgbm90IHRoZSBjYXVzZSBvZiBoZXRlcm9nZW5laXR5IG9mIHRoZSBETkEgRytDIGNvbnRlbnQgb2YgdGhpcmQgY29kb24gcG9zaXRpb247IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9NTMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pdmVyc2FsIHJlcGxpY2F0aW9uIGJpYXNlcyBpbiBiYWN0ZXJpYTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpYyBkaXJlY3Rpb25hbCBtdXRhdGlvbiBwcmVzc3VyZXMgaW4gYmFjdGVyaWE7IGNpdGF0aW9uX3RleHQ9cmVzZWFyY2gwMDU4LjHigJMwMDU4LjE0OyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHl6aW5nIGdlbm9tZXMgd2l0aCBjdW11bGF0aXZlIHNrZXcgZGlhZ3JhbXM7IE51Y2xlaWMgQWNpZCBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MjI4Ni05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYW5kIGFzeW1tZXRyaWVzIGluIEROQSBldm9sdXRpb247IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yNDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbHVsYXIgRE5BIHJlcGxpY2FzZXM6IGNvbXBvbmVudHMgYW5kIGR5bmFtaWNzIGF0IHRoZSByZXBsaWNhdGlvbiBmb3JrOyBBbm51LiBSZXYuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9MjgzLTMxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjb3VudGluZyB1bml0cyBpbiBETkE7IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTc7ICBjaXRhdGlvbl9wYWdlcz01MS02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpZXMgZ2VuZXJhdGVkIGJ5IHRyYW5zY3JpcHRpb24tY291cGxlZCByZXBhaXIgaW4gZW50ZXJvYmFjdGVyaWFsIGdlbmVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlIGNvbXBvc2l0aW9uIHNrZXdzLCByZXBsaWNhdGlvbiwgYW5kIG9yaWVudGF0aW9uIGluIDEyIHByb2thcnlvdGUgZ2Vub21lczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTY5MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ta2V3IG9mIG1vbm9udWNsZW90aWRlIGZyZXF1ZW5jaWVzLCByZWxhdGl2ZSBhYnVuZGFuY2Ugb2YgZGludWNsZW90aWRlcyBhbmQgRE5BIHN0cmFuZCBhc3ltbWV0cnk7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0zNjQtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsZ29yaXRobSBBUyAyODg6IEV4YWN0IHR3by1zYW1wbGUgU21pcm5vdiB0ZXN0IGZvciBhcmJpdHJhcnkgZGlzdHJpYnV0aW9uczsgQXBwbC4gU3RhdC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0yNjUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbmZpZGVuY2UgbGltaXRzIGZvciBhbiB1bmtub3duIGRpc3RyaWJ1dGlvbiBmdW5jdGlvbjsgQW5uLiBNYXRoLiBTdGF0LjsgIGNpdGF0aW9uX3llYXI9MTk0MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQ2MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgZXN0aW1hdGlvbiBvZiB0aGUgZGlzY3JlcGFuY3kgYmV0d2VlbiBlbXBpcmljYWwgY3VydmVzIG9mIGRpc3RyaWJ1dGlvbiBmb3IgdHdvIGluZGVwZW5kZW50IHNhbXBsZXM7IEJ1bGwuIE1vc2NvdyBVbml2LjsgIGNpdGF0aW9uX3llYXI9MTkzOTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9My0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpbG9jOiBwcmVkaWN0aW9uIG9mIHJlcGxpY2F0aW9uIGJvdW5kYXJpZXMgaW4gYW5ub3RhdGVkIGJhY3RlcmlhbCBjaHJvbW9zb21lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01NjAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIHJlcGxpY2F0aW9uIGluIGNpcmN1bGFyIHByb2thcnlvdGljIGNocm9tb3NvbWVzOyBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0zNTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHJlcGxpY2F0aW9uIG9yaWdpbnMgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgQnJpZWYuIEJpb2luZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM3Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFyaWF0aW9uIGluIHRoZSBzdHJlbmd0aCBvZiBzZWxlY3RlZCBjb2RvbiB1c2FnZSBiaWFzIGFtb25nIGJhY3RlcmlhOyBOdWNsZWljIEFjaWQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTExNDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbmV3IHBlcnNwZWN0aXZlIG9uIGlzb2Nob3JlIGV2b2x1dGlvbjsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zODU7ICBjaXRhdGlvbl9wYWdlcz03MS00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJvbmcgZG91YmxldCBwcmVmZXJlbmNlcyBpbiBudWNsZW90aWRlIHNlcXVlbmNlcyBhbmQgRE5BIGdlb21ldHJ5OyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTExLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3RlbeKAk2xvb3Ag4oCYa2lzc2luZ+KAmSBtb2RlbCBmb3IgdGhlIGluaXRpYXRpb24gb2YgcmVjb21iaW5hdGlvbiBhbmQgdGhlIG9yaWdpbiBvZiBpbnRyb25zOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTQ5LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZWdhdGl2ZSBjb3JyZWxhdGlvbiBiZXR3ZWVuIGNvbXBvc2l0aW9uYWwgc3ltbWV0cmllcyBhbmQgbG9jYWwgcmVjb21iaW5hdGlvbiByYXRlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zOTUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBpbXBsaWNhdGlvbnMgb2YgaW52ZXJzaW9ucyB0aGF0IGhhdmUgY2F1c2VkIGludHJhLXN0cmFuZCBwYXJpdHkgaW4gRE5BOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZXBsaWNhdGlvbi1yZWxhdGVkIG9yZ2FuaXphdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTE2MDktMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXNzZW50aWFsaXR5IGRldGVybWluZXMgY2hyb21vc29tZSBvcmdhbml6YXRpb24gaW4gYmFjdGVyaWE7IE51Y2xlaWMgQWNpZCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NjU3MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JnYW5pemF0aW9uIG9mIHRoZSBiYWN0ZXJpYWwgZ2Vub21lOyBBbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RFBWd2ViOiBBbiBvcGVuIGFjY2VzcyBpbnRlcm5ldCByZXNvdXJjZSBvbiBwbGFudCB2aXJ1c2VzIGFuZCB2aXJ1cyBkaXNlYXNlczsgT3V0bG9va3Mgb24gUGVzdCBNYW5hZ2VtZW50OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjY4LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EUFZ3ZWI6IGEgY29tcHJlaGVuc2l2ZSBkYXRhYmFzZSBvZiBwbGFudCBhbmQgZnVuZ2FsIHZpcnVzIGdlbmVzIGFuZCBnZW5vbWVzOyAgY2l0YXRpb25fdXJsPWh0dHA6Ly93d3cuZHB2d2ViLm5ldC87IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlbGVjdGlvbi1tdXRhdGlvbi1kcmlmdCB0aGVvcnkgb2Ygc3lub255bW91cyBjb2RvbiB1c2FnZTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTkyOyAgY2l0YXRpb25fcGFnZXM9ODk3LTkwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzYvMzI1LzEwMzk1MDIvZHNwMDIxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBDaGFyZ2FmZidzIHJ1bGUgb2YgaW50cmEtc3RyYW5kIHBhcml0eSAoSVNQKSBiZXR3ZWVuIGNvbXBsZW1lbnRhcnkgbW9uby9vbGlnb251Y2xlb3RpZGVzIGluIGNocm9tb3NvbWVzIGlzIHdlbGwgZXN0YWJsaXNoZWQgaW4gdGhlIHNjaWVudGlmaWMgbGl0ZXJhdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzYzNzM2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZPMSHzrVSdEEFhvdE6+1cUoX+yuB" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f325%2f363736" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f325%2f363736" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14135" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/6/1/m_cover.gif?Expires=2147483647&amp;Signature=sOIaosdT6uXpAtt-bpJN4lGWjn89o2XivgxEg5n1d0b6uc4rutjtPlXH8KDnUug4~icfBszbFG-0HY5pufJ6XhNpk9T-WjxmPJk7r95Av27GD54YDSvnj0wsx2TG9xuLp7aypGeHjTYyfAqWQxmgT4WMKLKZ~bziJjeuuiOZ6JhEQXVX~MAA3Zt1fUOb9CdMTVZj~3ig4~HJVGzIabQIJJAixJWzKelG~tTDiZn3JXJTacFA1vuy9TeJf2GPMdGcE7jlnPZJysftGzu~rqySy8oeattNRvSoC4MzCGEev9vmEBUi4d~QtEtbGuzOY3ZLYzMg7sgtncIpR8sAs8c6ig__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/6">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5273940">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5273940">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5273942">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5273942">Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5273946">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5273946">Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5273956">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5273956">Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5273996">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5273996">Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5274008">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5274008">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5274011">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5274011">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5274015">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5274015">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/6/311/363519">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/6/345/362870">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">B.R. Powdel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Mathematical Sciences</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=B.R.+Powdel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Powdel B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Powdel B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">B.R. Powdel</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Siddhartha Sankar Satapathy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Computer Science and Engineering</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Siddhartha+Sankar+Satapathy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Satapathy S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Satapathy S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Siddhartha Sankar Satapathy</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aditya Kumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Molecular Biology and Biotechnology</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aditya+Kumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumar A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aditya Kumar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pankaj Kumar Jha</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Molecular Biology and Biotechnology</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pankaj+Kumar+Jha">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jha P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jha P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pankaj Kumar Jha</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alak Kumar Buragohain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Molecular Biology and Biotechnology</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alak+Kumar+Buragohain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Buragohain A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Buragohain A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alak Kumar Buragohain</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Munindra Borah</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Mathematical Sciences</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Munindra+Borah">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Borah M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Borah M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Munindra Borah</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suvendra Kumar Ray</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Molecular Biology and Biotechnology</div>, <div class="insititution">Tezpur University</div>, <div class="addr-line">Tezpur, Assam 784 028</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 3712-267007/008/009. Fax. <div class="fax">+91 3712-267005/6</div>. E-mail: <a href="mailto:suven@tezu.ernet.in" target="_blank">suven@tezu.ernet.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suvendra+Kumar+Ray">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ray S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ray S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Suvendra Kumar Ray</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Molecular Biophysics Unit, Indian Institute of Science, Bangalore 560012, India.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: MS-04/603, Kendriya Vihar, Sector 56, Gurgaon, Haryana 122011, India.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 6, 1 December 2009, Pages 325–343, <a href='https://doi.org/10.1093/dnares/dsp021'>https://doi.org/10.1093/dnares/dsp021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 October 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 June 2009</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 September 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="363736" href="/dnaresearch/article-pdf/16/6/325/1039502/dsp021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>B.R. Powdel, Siddhartha Sankar Satapathy, Aditya Kumar, Pankaj Kumar Jha, Alak Kumar Buragohain, Munindra Borah, Suvendra Kumar Ray;  A Study in Entire Chromosomes of Violations of the Intra-strand Parity of Complementary Nucleotides (Chargaff's Second Parity Rule), <em>DNA Research</em>, Volume 16, Issue 6, 1 December 2009, Pages 325–343, <a href='https://doi.org/10.1093/dnares/dsp021'>https://doi.org/10.1093/dnares/dsp021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/363736?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/363736?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20Study%20in%20Entire%20Chromosomes%20of%20Violations%20of%20the%20Intra-strand%20Parity%20of%20Complementary%20Nucleotides%20(Chargaff%26apos%3Bs%20Second%20Parity%20Rule)&amp;publicationDate=2009-10-27&amp;volumeNum=16&amp;issueNum=6&amp;author=Powdel%2C%20B.R.%3B%20Satapathy%2C%20Siddhartha%20Sankar&amp;startPage=325&amp;endPage=343&amp;contentId=10.1093%2Fdnares%2Fdsp021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f325%2f363736" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5273940 id="5273940" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Chargaff's rule of intra-strand parity (ISP) between complementary mono/oligonucleotides in chromosomes is well established in the scientific literature. Although a large numbers of papers have been published citing works and discussions on ISP in the genomic era, scientists are yet to find all the factors responsible for such a universal phenomenon in the chromosomes. In the present work, we have tried to address the issue from a new perspective, which is a parallel feature to ISP. The compositional abundance values of mono/oligonucleotides were determined in all non-overlapping sub-chromosomal regions of specific size. Also the frequency distributions of the mono/oligonucleotides among the regions were compared using the Kolmogorov–Smirnov test. Interestingly, the frequency distributions between the complementary mono/oligonucleotides revealed statistical similarity, which we named as intra-strand frequency distribution parity (ISFDP). ISFDP was observed as a general feature in chromosomes of bacteria, archaea and eukaryotes. Violation of ISFDP was also observed in several chromosomes. Chromosomes of different strains belonging a species in bacteria/archaea (<em>Haemophilus influenza</em>, <em>Xylella fastidiosa</em> etc.) and chromosomes of a eukaryote are found to be different among each other with respect to ISFDP violation. ISFDP correlates weakly with ISP in chromosomes suggesting that the latter one is not entirely responsible for the former. Asymmetry of replication topography and composition of forward-encoded sequences between the strands in chromosomes are found to be insufficient to explain the ISFDP feature in all chromosomes. This suggests that multiple factors in chromosomes are responsible for establishing ISFDP.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="chromosome">chromosome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nucleotide composition&quot;">nucleotide composition</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Chargaff's second parity rule&quot;">Chargaff's second parity rule</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intra-strand frequency distribution parity&quot;">intra-strand frequency distribution parity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA replication&quot;">DNA replication</a></div>                            <h2 scrollto-destination=5273942 id="5273942" class="section-title" data-legacy-id=s1>Introduction</h2>
<p>Chargaff's first parity rule based on the nucleotide composition of double-stranded DNA states that the complementary nucleotides have the same abundance values.<sup><a reveal-id="DSP021C1" data-open="DSP021C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP021C2" data-open="DSP021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> This is explained by the DNA double-helix model in which A pairs only with T and G pairs only with C.<sup><a reveal-id="DSP021C3" data-open="DSP021C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Chargaff and his colleagues<sup><a reveal-id="DSP021C4" data-open="DSP021C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSP021C5" data-open="DSP021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> came with a similar observation of compositional relationship between the complementary nucleotides even within individual DNA strands of bacterial chromosomes. In the post-genomic era, this intra-strand relationship between the complementary nucleotides is observed in double-stranded genomes of viruses, bacteria, archaea and eukaryotes, which is known as Chargaff's second parity rule or intra-strand parity (ISP).<sup><a reveal-id="DSP021C2" data-open="DSP021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> There is no such defined rule to describe ISP in chromosomes like the base-pairing rule in Chargaff's first parity. ISP is also observed between the complementary oligonucleotides in chromosomes,<sup><a reveal-id="DSP021C6  DSP021C9" data-open="DSP021C6  DSP021C9" class="link link-ref link-reveal xref-bibr">6–9</a></sup> which has been attributed to genome-wide large-scale inversion, inversion transposition<sup><a reveal-id="DSP021C10" data-open="DSP021C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and coding sequence compositional symmetry between the strands.<sup><a reveal-id="DSP021C9" data-open="DSP021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Violation of ISP is observed with respect to organellar (mitochondria and plastids) genomes of some organisms, single-stranded viral genomes or any RNA genome.<sup><a reveal-id="DSP021C11  DSP021C13" data-open="DSP021C11  DSP021C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup></p><p>Theoretically, under no strand bias in terms of mutation and selection, the base complementary relationship easily explains the presence of ISP in chromosomes.<sup><a reveal-id="DSP021C14" data-open="DSP021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSP021C15" data-open="DSP021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> However, several evidences now prove that both the strands are not identical in terms of mutation/selection.<sup><a reveal-id="DSP021C16" data-open="DSP021C16" class="link link-ref link-reveal xref-bibr">16</a></sup> This results into violation of ISP in sub-chromosomal regions. Longer the sub-chromosomal region, smaller is the violation of ISP observed.<sup><a reveal-id="DSP021C17" data-open="DSP021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The mechanisms that are responsible to cause violation are defined under three categories.<sup><a reveal-id="DSP021C18" data-open="DSP021C18" class="link link-ref link-reveal xref-bibr">18</a></sup> First, DNA replication: leading strand (LeS) is found to be composed of more K nucleotides (G and T) than the complementary M (A and C) nucleotides and the reverse holds true for the lagging strand (LaS).<sup><a reveal-id="DSP021C19" data-open="DSP021C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This is due to the fact that the LeS which functions as the template for Okazaki fragment synthesis (functions as template for LaS) remains exposed more as single-stranded than the LaS (functions as template for LeS) during replication that results into higher deamination of the cytosine residues<sup><a reveal-id="DSP021C20" data-open="DSP021C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSP021C21" data-open="DSP021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> in LeS (cytosine gets deaminated 140 times faster in ssDNA than in dsDNA<sup><a reveal-id="DSP021C22" data-open="DSP021C22" class="link link-ref link-reveal xref-bibr">22</a></sup>). In addition, the influence of Okazaki fragments and the sliding DNA clamp proteins associated with the synthesis of LaS create functional asymmetry of the mismatch repairing system on DNA.<sup><a reveal-id="DSP021C23" data-open="DSP021C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Second, transcription: genes are preferentially located in the LeS than in the LaS to avoid head on collision between the machineries of replication and transcription.<sup><a reveal-id="DSP021C24" data-open="DSP021C24" class="link link-ref link-reveal xref-bibr">24</a></sup> During transcription, the non-template strand remains more exposed as single-stranded than the template strand, which causes asymmetry in cytosine deamination between the strands.<sup><a reveal-id="DSP021C22" data-open="DSP021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The transcription-coupled repair system also acts only upon the template strand and thereby contributes to the strand asymmetry.<sup><a reveal-id="DSP021C25" data-open="DSP021C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Third, translation: uses of synonymous codons are influenced by differential abundance of tRNA molecules which results into the differential abundance of complementary nucleotides at the third position of family box codons. This causes parity violation.<sup><a reveal-id="DSP021C14" data-open="DSP021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In spite of these factors favoring violations of the parity in chromosomes, ISP is observed in an entire chromosome due to the cancellation effect of the local violations in opposite directions.<sup><a reveal-id="DSP021C14" data-open="DSP021C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Evolutionary biologists are more interested to understand the role of mutation and/or selection in the violation of ISP by analyzing the weakly selected or selectively neutral regions (third position of family box codons and non-coding regions) in chromosomes.<sup><a reveal-id="DSP021C14" data-open="DSP021C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSP021C26" data-open="DSP021C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Whether any specific feature(s) is/are associated with chromosomes exhibiting ISP is yet to be understood. Shioiri and Takahata<sup><a reveal-id="DSP021C27" data-open="DSP021C27" class="link link-ref link-reveal xref-bibr">27</a></sup> studied ISP by finding out the total AT skew (ATS) and GC skew (GCS) in the chromosomes of several bacteria. In their study, out of 36 bacterial chromosomes, <em>Xylella fastidiosa</em> exhibited maximum ATS and GCS. They observed variable ATS/GCS among chromosomes of different strains of a species as well as chromosomes within a bacterial cell. They also observed ATS and GCS may be different from each other within a chromosome. Since, they did not do any statistical analysis of the skew, the significance of the variability observed among chromosomes was not discussed by them. The usual statistical tool used to find out ISP in chromosomes is a correlation analysis of oligonucleotides abundance described by Prabhu.<sup><a reveal-id="DSP021C6" data-open="DSP021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The ISP study between the complimentary mononucleotides is important because it has been proven that oligonucleotide parity and mononucleotide parity are independent.<sup><a reveal-id="DSP021C8" data-open="DSP021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Baisnée <em>et al</em>.<sup><a reveal-id="DSP021C8" data-open="DSP021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> studied parity in chromosomes by measuring the S<sup>1</sup> index which is defined as the sum of the absolute values of the differences between complementary oligonucleotides (<em>n</em> mer) frequencies (<em>n</em> varies from 1 to 9 mer). Both these methods do not measure the statistical significance of differences between the abundance values of a mono/oligonucleotide and its reverse complement. For example, if a chromosome carries significant similarity between the abundance values of A and T but carries significant difference between the abundance values of G and C, this will not be identified separately. Similarly, the above methods are unable to find out parity violations in chromosomes with respect to the abundance values of an oligonucleotide and its reverse complement. We have developed a methodology here that can independently study ISP between S nucleotides (any oligonucleotide and its reverse complement) as well as between W nucleotides using the abundance values of mononucleotides. We use the well-known Kolmogorov–Smirnov (KS) test to study the frequency distribution of the compositional abundance values of the mononucleotides in a chromosome sequence, which gives the statistical significance of the similarity between the distributions of complementary nucleotides. This we called as intra-strand frequency distribution parity (ISFDP), which has been used here to study the chromosomes of bacteria, archaea and eukaryotes.</p>                            <h2 scrollto-destination=5273946 id="5273946" class="section-title" data-legacy-id=s2>Materials and methods</h2>
                            <h3 scrollto-destination=5273947 id="5273947" class="section-title" data-legacy-id=s2a>Frequency distribution calculation</h3>
<p>Chromosome sequences of different bacteria, archaea and eukaryotes (Tables <a reveal-id="DSP021TB1 DSP021TB2  DSP021TB3" data-open="DSP021TB1 DSP021TB2  DSP021TB3" class="link link-reveal link-table xref-fig">1–3</a>) were obtained from the genome information broker, DDBJ site (<a class="link link-uri" href="http://www.gib.genes.nig.ac.jp">www.gib.genes.nig.ac.jp</a>). Bacterial chromosomes were chosen randomly from the database starting the genus name from A to Z. Chromosome sequences of different strains belonging to the same species in the case of bacteria were taken in several cases to do the intra-species comparison. Each chromosome sequence was divided into smaller-size sequences of 1000 nucleotides each starting from the beginning, and the abundance value of the four nucleotides was determined using the computer program (developed for this study). The distribution of the abundance values of complementary nucleotides in different fragments were analyzed by the KS non-parametric test using XLSTAT program<sup><a reveal-id="DSP021C28" data-open="DSP021C28" class="link link-ref link-reveal xref-bibr">28</a>–<a reveal-id="DSP021C30" data-open="DSP021C30" class="link link-ref link-reveal xref-bibr">30</a></sup> (Kovach Computing Services, Anglesey, Wales). <em>H</em><sub>0</sub>: distribution patterns of any two nucleotides/oligonucleotides in a chromosome are similar; <em>H</em><sub>A</sub>: there is a difference between the two distributions. Owing to the large sample size, similarity was considered at the <em>P</em>-value of &gt;0.01, weak similarity was considered at the <em>P</em>-value between 0.01 and 10<sup>−4</sup>, and the value of &lt;10<sup>−4</sup> was considered as strong violation similarity. Group-frequency distributions of the abundance values were plotted to observe the frequency-distribution parity. In the case of the di- and trinucleotides, the abundance values were determined using a different computer program (developed here for this study) in the segments for the 16 dinucleotides and 64 trinucleotides. The analysis was done as described for the mononucleotides earlier.</p><p>Angular replication asymmetry of the chromosomes was calculated with the help of the information on <em>ori</em> (origin) and <em>ter</em> (termination) cited in the websites (<a class="link link-uri" href="http://www.cbs.dtu.dk/services/GenomeAtlas/suppl/origin/">http://www.cbs.dtu.dk/services/GenomeAtlas/suppl/origin/</a> and <a class="link link-uri" href="http://pbil.univ-lyon1.fr/software/Oriloc/oriloc.html">http://pbil.univ-lyon1.fr/software/Oriloc/oriloc.html</a>). The chromosomal region starting from <em>ori</em> to <em>ter</em> was considered as the leading region in the Watson strand (Ws) and the remaining portion of the chromosome as the lagging region. For a circular chromosome, the angular replication asymmetry was calculated as the amount of angular distance of leading region deviating from 180°.</p>                            <h3 scrollto-destination=5273950 id="5273950" class="section-title" data-legacy-id=s2b>Proportionate distribution of forward- and reverse-encoded sequences in a DNA strand</h3>
<p>From the DDBJ site, only coding sequences were downloaded. A continuous stretch of the nucleotide sequence was made from all the sequences by removing the gene names. This resembled a DNA strand only composed of forward-encoded sequences. Frequency distribution analysis was done on this. In another approach, 50% of the above strand was made reverse complement by <em>in silico</em> followed by joining with the rest. This resembled a DNA strand composed of 50% forward-encoded and 50% reverse-encoded sequences. Frequency-distribution study was carried out as described above.</p>                            <h3 scrollto-destination=5273952 id="5273952" class="section-title" data-legacy-id=s2c>Identification of leading and LaS region</h3>
<p>ATS and GCS analyses of the chromosome sequences were done as described earlier.<sup><a reveal-id="DSP021C21" data-open="DSP021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> This was used to find out the tentative leading and lagging portions in a DNA strand.</p>                            <h3 scrollto-destination=5273954 id="5273954" class="section-title" data-legacy-id=s2d>Relative proportion of coding sequence distribution</h3>
<p>This was found out by deducting ORF numbers between Ws (top strand) and Crick strand (Cs: bottom strand) followed by dividing that with the total number of ORFs. Gene orientation information was obtained from the website (<a class="link link-uri" href="http://cmr.jcvi.org/tigr-scripts/CMR/CmrHomePage.cgi">http://cmr.jcvi.org/tigr-scripts/CMR/CmrHomePage.cgi</a>).</p>                            <h2 scrollto-destination=5273956 id="5273956" class="section-title" data-legacy-id=s3>Results</h2>
                            <h3 scrollto-destination=5273957 id="5273957" class="section-title" data-legacy-id=s3a>ISFDP in chromosomes of bacteria</h3>
<p>In this study, a total of 112 bacterial chromosomes were considered, which includes different lineages of bacteria such as protobacteria, cyanobacteria, firmicutes, actinobacteria etc. Samples from each group were taken randomly. The bacteria included in the sample comprised a GC% variation from a minimum of 28% to a maximum of 75% and chromosome size variation from 580 kb to a maximum of 9105 kb. We have studied the frequency distributions of the abundance values of mononucleotides in the uniform sub-chromosomal length of 1000 nucleotides. A collective analysis of the nucleotide abundance values from all the segments of a chromosome was done by frequency distribution smooth curves using Microsoft Excel, and the similarity of the distributions of two complementary nucleotides was tested using the KS test (XL-Stat; <a class="link link-uri" href="http://www.xlstat.com/en/download">http://www.xlstat.com/en/download</a>). Figure <a reveal-id="DSP021F1" data-open="DSP021F1" class="link link-reveal link-table xref-fig">1</a>A(i), B(i), C(i), D(i) and E(i) represents the smooth curves of frequency distributions of nucleotides in chromosomes <em>Campylobacter jejuni</em> RM1221 (30.31%), <em>Escherichia coli</em> K12 MG1655 (50.79%), <em>Xanthomonas campestris</em> pv. <em>campestris</em> (<em>Xcc</em>; 65.07%), <em>X. fastidiosa</em> 9a5c (52.68%) and <em>X. fastidiosa</em> Temecula (51.78%). Smooth curves of complementary nucleotides overlap with each other in the first three chromosomes, whereas those of non-complementary ones do not. In the fourth chromosome, none of the curves overlap with each other. In <em>E. coli</em> chromosome [Fig. <a reveal-id="DSP021F1" data-open="DSP021F1" class="link link-reveal link-table xref-fig">1</a>B(i)], all the four smooth frequency curves are close to each other due to the closeness of the abundance values of the nucleotides, whereas in the graphs of <em>C. jejuni</em> and <em>Xcc</em>, the smooth frequency curves of W (A and T) and S (G and C) nucleotides are distinctly separated as GC% the chromosome are toward both extremes. The distribution was studied by the KS test and the results of the four chromosomes are shown in Fig. <a reveal-id="DSP021F1" data-open="DSP021F1" class="link link-reveal link-table xref-fig">1</a>A(ii, iii), B(ii, iii), C(ii, iii), D(ii, iii) and E(ii, iii). The graphs generated by the KS test suggest the complete overlapping between the complementary nucleotides in the chromosomes except the one of <em>X. fastidiosa</em> strain, which is in concordant with the smooth frequency curve. The distributional similarity between the complementary nucleotides is called as ISFDP. A total of 112 bacterial, 49 archaea and 18 eukaryotic chromosomes (Tables <a reveal-id="DSP021TB1 DSP021TB2  DSP021TB3" data-open="DSP021TB1 DSP021TB2  DSP021TB3" class="link link-reveal link-table xref-fig">1–3</a>) were analyzed by the KS test to study ISFDP. The <em>P</em>-values between the A and T distributions as well as between the G and C distributions are given in Tables <a reveal-id="DSP021TB1 DSP021TB2  DSP021TB3" data-open="DSP021TB1 DSP021TB2  DSP021TB3" class="link link-reveal link-table xref-fig">1–3</a>.</p>                        <a id="5273959" scrollto-destination="5273959"></a>
<div id="DSP021F1" data-id="DSP021F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02101.jpeg?Expires=2147483647&Signature=iTXUtlZa6cxMhOkL4fHTrxLCOsjBB0GjTYgtEVTpZozdtjqyaUoMvZ-t5GQODC3qOhFWynM0vsZLK6jiqRtVUEe~RguYDguwzVjKrVmwjVLTdp5PevxDKZ8Z5h4fu94oT4~yRC0-RXnsuzs80wOChA9f6W8x5r5g4KSqakzxskCXZNTa6PyOuEO5PujDSvX4drUbv8f16yQqEf5qg-HV1UFlUdmS-a2C2NQ6Nf36LyOicJhob-CMVh0XSbTY7mh1qtTc4HYvgRKnWw3-dVJsumyfCxC3qe-aHqP-bv2ieg4WOVIkT0oU95TPZl4UchEckFX2PY27tX7bF0Jd61gQTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A–E) Frequency distribution of nucleotides in chromosomes. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The X-axis represents the abundance values of the nucleotide spanning a range, whereas the Y-axis represents the frequency of the abundance values. In (A), the chromosome is AT rich; in (B), the chromosome is composed of similar AT and GC and in (C), the chromosome is GC rich. This is also evident from the group-frequency distribution curve. The smooth frequency curves of complementary nucleotides in these chromosomes are overlapping with each other. The KS test is shown for S and W nucleotides separately adjacent to the figures, respectively [a(ii, iii)–e(ii, iii)]. The KS test is in concordance with the curve obtained by smoothing group-frequency distribution. In (D) and (E), the group-frequency distribution for the chromosomes of two strains of X. fastidiosa is shown. In 9a5c strain chromosome, the smooth frequency curve between the complementary nucleotides does not overlap which is also suggested by the KS test. However, in Temecula 1 strain chromosome, the parity is maintained." path-from-xml="dsp02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273959/dsp02101.jpeg" path-from-xml="dsp02101.jpeg" target="_blank">View large</a><a section="5273959" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02101.jpeg?Expires=2147483647&Signature=l6IUQL6q8YcRxLqIfK~fDo9AEWbio0gakeaoxyEHxflBlVtuiAQW878Y2BKNzWAN8LueuDEJoG2XklyH~yHPmBupjPMEXxXC-hNjDpHRIYUTEG~tPCfro5LxrPEv2wK~s-I532fWEY1hI-LOWHkGRShgw2pTuOa2drOfjKi1SKf21gWPLpHvna4FOlo5d66Xrof6r9HWZinLHO3wYqFslSnUYLnEN8hDGDMMQPpB65TfoyrDYEtmtVE693MZSPaUVGZRDcvfQprtUTNVWFZgYy-QFY~D1P3Em6JPsfGA3Os7yAj338qmHxqkIdniqoz08nFYJRaTdSrtfqOwN2q~fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273959&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>–<strong>E</strong>) Frequency distribution of nucleotides in chromosomes. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The <em>X</em>-axis represents the abundance values of the nucleotide spanning a range, whereas the <em>Y</em>-axis represents the frequency of the abundance values. In (A), the chromosome is AT rich; in (B), the chromosome is composed of similar AT and GC and in (C), the chromosome is GC rich. This is also evident from the group-frequency distribution curve. The smooth frequency curves of complementary nucleotides in these chromosomes are overlapping with each other. The KS test is shown for S and W nucleotides separately adjacent to the figures, respectively [a(ii, iii)–e(ii, iii)]. The KS test is in concordance with the curve obtained by smoothing group-frequency distribution. In (D) and (E), the group-frequency distribution for the chromosomes of two strains of <em>X. fastidiosa</em> is shown. In 9a5c strain chromosome, the smooth frequency curve between the complementary nucleotides does not overlap which is also suggested by the KS test. However, in Temecula 1 strain chromosome, the parity is maintained.</p></div></div><div content-id="DSP021F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02101.jpeg?Expires=2147483647&Signature=iTXUtlZa6cxMhOkL4fHTrxLCOsjBB0GjTYgtEVTpZozdtjqyaUoMvZ-t5GQODC3qOhFWynM0vsZLK6jiqRtVUEe~RguYDguwzVjKrVmwjVLTdp5PevxDKZ8Z5h4fu94oT4~yRC0-RXnsuzs80wOChA9f6W8x5r5g4KSqakzxskCXZNTa6PyOuEO5PujDSvX4drUbv8f16yQqEf5qg-HV1UFlUdmS-a2C2NQ6Nf36LyOicJhob-CMVh0XSbTY7mh1qtTc4HYvgRKnWw3-dVJsumyfCxC3qe-aHqP-bv2ieg4WOVIkT0oU95TPZl4UchEckFX2PY27tX7bF0Jd61gQTg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A–E) Frequency distribution of nucleotides in chromosomes. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The X-axis represents the abundance values of the nucleotide spanning a range, whereas the Y-axis represents the frequency of the abundance values. In (A), the chromosome is AT rich; in (B), the chromosome is composed of similar AT and GC and in (C), the chromosome is GC rich. This is also evident from the group-frequency distribution curve. The smooth frequency curves of complementary nucleotides in these chromosomes are overlapping with each other. The KS test is shown for S and W nucleotides separately adjacent to the figures, respectively [a(ii, iii)–e(ii, iii)]. The KS test is in concordance with the curve obtained by smoothing group-frequency distribution. In (D) and (E), the group-frequency distribution for the chromosomes of two strains of X. fastidiosa is shown. In 9a5c strain chromosome, the smooth frequency curve between the complementary nucleotides does not overlap which is also suggested by the KS test. However, in Temecula 1 strain chromosome, the parity is maintained." path-from-xml="dsp02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273959/dsp02101.jpeg" path-from-xml="dsp02101.jpeg" target="_blank">View large</a><a section="5273959" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02101.jpeg?Expires=2147483647&Signature=l6IUQL6q8YcRxLqIfK~fDo9AEWbio0gakeaoxyEHxflBlVtuiAQW878Y2BKNzWAN8LueuDEJoG2XklyH~yHPmBupjPMEXxXC-hNjDpHRIYUTEG~tPCfro5LxrPEv2wK~s-I532fWEY1hI-LOWHkGRShgw2pTuOa2drOfjKi1SKf21gWPLpHvna4FOlo5d66Xrof6r9HWZinLHO3wYqFslSnUYLnEN8hDGDMMQPpB65TfoyrDYEtmtVE693MZSPaUVGZRDcvfQprtUTNVWFZgYy-QFY~D1P3Em6JPsfGA3Os7yAj338qmHxqkIdniqoz08nFYJRaTdSrtfqOwN2q~fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273959&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(<strong>A</strong>–<strong>E</strong>) Frequency distribution of nucleotides in chromosomes. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The <em>X</em>-axis represents the abundance values of the nucleotide spanning a range, whereas the <em>Y</em>-axis represents the frequency of the abundance values. In (A), the chromosome is AT rich; in (B), the chromosome is composed of similar AT and GC and in (C), the chromosome is GC rich. This is also evident from the group-frequency distribution curve. The smooth frequency curves of complementary nucleotides in these chromosomes are overlapping with each other. The KS test is shown for S and W nucleotides separately adjacent to the figures, respectively [a(ii, iii)–e(ii, iii)]. The KS test is in concordance with the curve obtained by smoothing group-frequency distribution. In (D) and (E), the group-frequency distribution for the chromosomes of two strains of <em>X. fastidiosa</em> is shown. In 9a5c strain chromosome, the smooth frequency curve between the complementary nucleotides does not overlap which is also suggested by the KS test. However, in Temecula 1 strain chromosome, the parity is maintained.</p></div></div>                        <a id="5273960" scrollto-destination="5273960"></a>
<div content-id="DSP021TB1" class="table-modal"><div class="table-wrap"><div id="DSP021TB1" class="table-wrap-title" data-id="DSP021TB1"><div class="label -label">Table 1</div><div class="caption"><p>ISFDP analysis in bacterial chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Bacterial group </th><th>TB (°) </th></tr></thead><tbody><tr><td>1 </td><td><em>Acinetobacter</em> sp. ADP1 </td><td>3598 </td><td>40.43 </td><td>0.745 </td><td><em>0.006</em> </td><td>0.00068 </td><td>0.00484 </td><td rowspan="5">G-Proteobacteria </td><td>7.07 </td></tr><tr><td>2 </td><td><em>Actinobacillus pleuropneumoniae</em> L20 serotype 5b </td><td>2274 </td><td>41.3 </td><td>0.436 </td><td>0.819 </td><td>0.00187 </td><td>0.00109 </td><td rowspan="4">NA </td></tr><tr><td>3 </td><td><em>Actinobacillus succinogenes</em> 130Z </td><td>2319 </td><td>44.91 </td><td>0.312 </td><td>0.291 </td><td>0.00232 </td><td>0.00291 </td></tr><tr><td>4 </td><td><em>Aeromonas hydrophila</em> subsp. <em>hydrophila</em> ATCC 7966 </td><td>4744 </td><td>61.55 </td><td>0.88 </td><td>0.19 </td><td>0.00141 </td><td>0.00139 </td></tr><tr><td>5 </td><td><em>Aeromonas salmonicida</em> subsp. <em>salmonicida</em> A449 </td><td>4702 </td><td>58.51 </td><td>0.04 </td><td>0.959 </td><td>0.00215 </td><td>0.00073 </td></tr><tr><td>6 </td><td><em>Agrobacterium tumefaciens</em> C58 (circular chromosome) </td><td>2841 </td><td>59.38 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00694 </td><td>0.00967 </td><td>A-Proteobacteria </td><td>7.37 </td></tr><tr><td>7 </td><td><em>Alkaliphilus oremlandii</em> OhILAs </td><td>3123 </td><td>36.26 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00615 </td><td>0.01324 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>8 </td><td><em>Anaeromyxobacter dehalogenans</em> 2CP-C </td><td>5013 </td><td>74.9 </td><td>0.077 </td><td><em>0.001</em> </td><td>0.00476 </td><td>0.00249 </td><td rowspan="2">D-Proteobacteria </td><td>70.57 </td></tr><tr><td>9 </td><td><em>Anaeromyxobacter</em> sp. Fw109-5 </td><td>5277 </td><td>73.53 </td><td>0.712 </td><td><em>0.008</em> </td><td>0.00073 </td><td>0.00216 </td><td>7.48 </td></tr><tr><td>10 </td><td><em>Bacillus anthracis</em> Ames </td><td>5227 </td><td>35.38 </td><td><em>0.004</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00581 </td><td rowspan="6">Firmicutes </td><td>NA </td></tr><tr><td>11 </td><td><em>Bacillus anthracis</em> 'Ames Ancestor' </td><td>5227 </td><td>35.38 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00582 </td><td>7.48 </td></tr><tr><td>12 </td><td><em>Bacillus anthracis</em> Sterne </td><td>5228 </td><td>35.38 </td><td><em>0.008</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00221 </td><td>0.00588 </td><td>7.46 </td></tr><tr><td>13 </td><td><em>Bacillus subtilis</em> </td><td>4214 </td><td>43.52 </td><td>0.219 </td><td>0.234 </td><td>0.00212 </td><td>0.00224 </td><td>13.69 </td></tr><tr><td>14 </td><td><em>Bacillus thuringiensis</em> Al Hakam </td><td>5257 </td><td>35.43 </td><td>0.123 </td><td><em>0.002</em> </td><td>0.00042 </td><td>0.00081 </td><td>NA </td></tr><tr><td>15 </td><td><em>Bacillus thuringiensis</em> serovar konkukian 97-27 </td><td>5237 </td><td>35.41 </td><td>0.015 </td><td><strong>&lt;0.0001</strong> </td><td>0.00194 </td><td>0.00438 </td><td>3.98 </td></tr><tr><td>16 </td><td><em>Bordetella parapertussis</em> 12822 </td><td>4773 </td><td>68.1 </td><td>0.433 </td><td><strong>&lt;0.0001</strong> </td><td>0.00247 </td><td>0.00776 </td><td rowspan="2">B-Proteobacteria </td><td>37.01 </td></tr><tr><td>17 </td><td><em>Bordetella pertussis</em> Tohama 1 </td><td>4086 </td><td>67.72 </td><td>0.861 </td><td><strong>&lt;0.0001</strong> </td><td>0.00022 </td><td>0.00390 </td><td>71.28 </td></tr><tr><td>18 </td><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>9105 </td><td>64.06 </td><td>0.512 </td><td>0.31 </td><td>0.00070 </td><td>0.00038 </td><td rowspan="3">A-Proteobacteria </td><td>7.07 </td></tr><tr><td>19 </td><td><em>Bradyrhizobium</em> sp. BTAi1 </td><td>8264 </td><td>64.92 </td><td>0.381 </td><td>0.01 </td><td>0.00100 </td><td>0.00163 </td><td rowspan="4">NA </td></tr><tr><td>20 </td><td><em>Brucella melitensis</em> 16M </td><td>1177 </td><td>57.35 </td><td>0.472 </td><td><em>0.008</em> </td><td>0.00227 </td><td>0.00312 </td></tr><tr><td>21 </td><td><em>Campylobacter concisus</em> 13826 </td><td>2052 </td><td>39.43 </td><td>0.033 </td><td>0.048 </td><td>0.00038 </td><td>0.00599 </td><td rowspan="5">E-Proteobacteria </td></tr><tr><td>22 </td><td><em>Campylobacter curvus</em> 525.92 </td><td>1971 </td><td>44.54 </td><td>0.028 </td><td>0.752 </td><td>0.00745 </td><td>0.00282 </td></tr><tr><td>23 </td><td><em>Campylobacter jejuni</em> RM1221 </td><td>1777 </td><td>30.31 </td><td>0.574 </td><td>0.23 </td><td>0.00330 </td><td>0.00436 </td><td>8.69 </td></tr><tr><td>24 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> 81116 </td><td>1628 </td><td>30.54 </td><td>0.491 </td><td>0.029 </td><td>0.00250 </td><td>0.00613 </td><td>NA </td></tr><tr><td>25 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> NCTC 11168 </td><td>1641 </td><td>30.55 </td><td>0.067 </td><td>0.132 </td><td>0.00296 </td><td>0.00457 </td><td>10.25 </td></tr><tr><td>26 </td><td>Candidatus <em>Desulfococcus oleovorans</em> Hxd3 </td><td>3944 </td><td>56.17 </td><td>0.258 </td><td>0.133 </td><td>0.00199 </td><td>0.00157 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>27 </td><td><em>Caulobacter crescentus</em> CB15 </td><td>4016 </td><td>67.22 </td><td>0.042 </td><td>0.171 </td><td>0.00396 </td><td>0.00188 </td><td>A-Proteobacteria </td><td>8.56 </td></tr><tr><td>28 </td><td><em>Chlamydia muridarum</em> Nigg </td><td>1072 </td><td>40.34 </td><td>0.221 </td><td>0.853 </td><td>0.00107 </td><td>0.00337 </td><td rowspan="3">Chlamydiae </td><td>1.17 </td></tr><tr><td>29 </td><td><em>Chlamydia trachomatis</em> AHAR-13 </td><td>1044 </td><td>41.31 </td><td>0.228 </td><td>0.284 </td><td>0.00230 </td><td>0.00059 </td><td>1.30 </td></tr><tr><td>30 </td><td><em>Chlamydophila abortus</em> S263 </td><td>1144 </td><td>39.87 </td><td>0.534 </td><td><em>0.002</em> </td><td>0.00065 </td><td>0.00361 </td><td>0.57 </td></tr><tr><td>31 </td><td><em>Coxiella burnetii</em> Dugway 7E9-12 </td><td>2158 </td><td>42.44 </td><td><em>0.004</em> </td><td><em>0.001</em> </td><td>0.00592 </td><td>0.00573 </td><td rowspan="2">G-Proteobacteria </td><td>NA </td></tr><tr><td>32 </td><td><em>Coxiella burnetii</em> RSA 493 </td><td>1995 </td><td>42.66 </td><td>0.014 </td><td>0.467 </td><td>0.00198 </td><td>0.00029 </td><td>31.15 </td></tr><tr><td>33 </td><td><em>Desulfovibrio desulfuricans</em> G20 </td><td>3730 </td><td>57.84 </td><td>0.59 </td><td><em>0.001</em> </td><td>0.00189 </td><td>0.00322 </td><td>Firmicutes </td><td>10.70 </td></tr><tr><td>34 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> DP4 </td><td>3462 </td><td>63.01 </td><td>0.3 </td><td>0.159 </td><td>0.00152 </td><td>0.00106 </td><td rowspan="2">D-Proteobacteria </td><td>NA </td></tr><tr><td>35 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> Hildenborough </td><td>3570 </td><td>63.14 </td><td>0.557 </td><td>0.082 </td><td>0.00143 </td><td>0.00024 </td><td>4.78 </td></tr><tr><td>36 </td><td><em>Enterobacter sakazakii</em> ATCC BAA-894 </td><td>4368 </td><td>56.77 </td><td>0.167 </td><td>0.388 </td><td>0.00359 </td><td>0.00044 </td><td rowspan="10">G-Proteobacteria </td><td>NA </td></tr><tr><td>37 </td><td><em>Enterobacter</em> sp. 638 </td><td>4518 </td><td>52.98 </td><td>0.645 </td><td>0.39 </td><td>0.00169 </td><td>0.00163 </td><td>NA </td></tr><tr><td>38 </td><td><em>Escherichia coli</em> 536 </td><td>4938 </td><td>50.52 </td><td>0.714 </td><td>0.084 </td><td>0.00062 </td><td>0.00328 </td><td>7.40 </td></tr><tr><td>39 </td><td><em>Escherichia coli</em> APEC O1 </td><td>5082 </td><td>50.55 </td><td>0.779 </td><td>0.576 </td><td>0.00032 </td><td>0.00070 </td><td>NA </td></tr><tr><td>40 </td><td><em>Escherichia coli</em> CFT073 </td><td>5231 </td><td>50.48 </td><td>0.112 </td><td>0.92 </td><td>0.00173 </td><td>0.00080 </td><td>5.66 </td></tr><tr><td>41 </td><td><em>Escherichia coli</em> E24377A </td><td>4979 </td><td>50.62 </td><td>0.736 </td><td>0.128 </td><td>0.00205 </td><td>0.00212 </td><td rowspan="2">NA </td></tr><tr><td>42 </td><td><em>Escherichia coli</em> HS </td><td>4643 </td><td>50.82 </td><td>0.328 </td><td>0.469 </td><td>0.00151 </td><td>0.00207 </td></tr><tr><td>43 </td><td><em>Escherichia coli</em> K12 MG1655 </td><td>4639 </td><td>50.79 </td><td>0.732 </td><td>0.587 </td><td>0.00054 </td><td>0.00113 </td><td>4.28 </td></tr><tr><td>44 </td><td><em>Escherichia coli</em> UTI89 </td><td>5065 </td><td>50.6 </td><td>0.51 </td><td>0.237 </td><td>0.00076 </td><td>0.00203 </td><td>3.70 </td></tr><tr><td>45 </td><td><em>Escherichia coli</em> W3110 </td><td>4646 </td><td>50.8 </td><td>0.873 </td><td>0.729 </td><td>0.00073 </td><td>0.00091 </td><td>12.64 </td></tr><tr><td>46 </td><td><em>Frankia alni</em> ACN14A chromosome </td><td>7497 </td><td>72.82 </td><td>0.463 </td><td>0.036 </td><td>0.00141 </td><td>0.00139 </td><td rowspan="2">Actinobacteria </td><td rowspan="5">NA </td></tr><tr><td>47 </td><td><em>Frankia</em> sp. CcI3 </td><td>5433 </td><td>70.08 </td><td>0.808 </td><td>0.662 </td><td>0.00129 </td><td>0.00017 </td></tr><tr><td>48 </td><td><em>Haemophilus influenzae</em> 86-028NP </td><td>1914 </td><td>38.16 </td><td>0.886 </td><td>0.654 </td><td>0.00089 </td><td>0.00044 </td><td rowspan="4">G-Proteobacteria </td></tr><tr><td>49 </td><td><em>Haemophilus influenzae</em> PittEE </td><td>1813 </td><td>38.04 </td><td>0.544 </td><td>0.038 </td><td>0.00054 </td><td>0.00317 </td></tr><tr><td>50 </td><td><em>Haemophilus influenzae</em> PittGG </td><td>1887 </td><td>38.01 </td><td>0.125 </td><td><strong>&lt;0.0001</strong> </td><td>0.00005 </td><td>0.01016 </td></tr><tr><td>51 </td><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>1830 </td><td>38.15 </td><td>0.154 </td><td><em>0.004</em> </td><td>0.00298 </td><td>0.00472 </td><td>46.61 </td></tr><tr><td>52 </td><td><em>Helicobacter acinonychis</em> Sheeba </td><td>1553 </td><td>38.18 </td><td><em>0</em> </td><td>0.596 </td><td>0.00869 </td><td>0.00164 </td><td rowspan="3">E-Proteobacteria </td><td>NA </td></tr><tr><td>53 </td><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>1799 </td><td>35.93 </td><td>0.161 </td><td><strong>&lt;0.0001</strong> </td><td>0.00499 </td><td>0.01518 </td><td>46.54 </td></tr><tr><td>54 </td><td><em>Helicobacter pylori</em> J99 </td><td>1643 </td><td>39.19 </td><td>0.246 </td><td>0.256 </td><td>0.00259 </td><td>0.00510 </td><td>10.97 </td></tr><tr><td>55 </td><td><em>Lactobacillus acidophilus</em> NCFM </td><td>1993 </td><td>34.72 </td><td>0.382 </td><td><strong>&lt;0.0001</strong> </td><td>0.00066 </td><td>0.01644 </td><td rowspan="6">Firmicutes </td><td>19.54 </td></tr><tr><td>56 </td><td><em>Lactobacillus brevis</em> ATCC 367 </td><td>2291 </td><td>46.22 </td><td>0.023 </td><td><strong>&lt;0.0001</strong> </td><td>0.00271 </td><td>0.02882 </td><td rowspan="6">NA </td></tr><tr><td>57 </td><td><em>Lactobacillus delbrueckii</em> subsp. <em>bulgaricus</em> ATCC BAA-365 </td><td>1856 </td><td>49.69 </td><td>0.491 </td><td>0.264 </td><td>0.00201 </td><td>0.00087 </td></tr><tr><td>58 </td><td><em>Lactobacillus reuteri</em> F275 </td><td>1999 </td><td>38.87 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00122 </td><td>0.01040 </td></tr><tr><td>59 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363 </td><td>2529 </td><td>35.75 </td><td>0.233 </td><td>0.056 </td><td>0.00352 </td><td>0.00524 </td></tr><tr><td>60 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> SK11 </td><td>2438 </td><td>35.86 </td><td>0.399 </td><td>0.521 </td><td>0.00147 </td><td>0.00136 </td></tr><tr><td>61 </td><td><em>Magnetococcus</em> sp. MC-1 </td><td>4719 </td><td>54.17 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00490 </td><td>0.01198 </td><td>Magnetococcus </td></tr><tr><td>62 </td><td><em>Magnetospirillum magneticum</em> AMB-1 </td><td>4967 </td><td>65.09 </td><td>0.031 </td><td><strong>&lt;0.0001</strong> </td><td>0.00339 </td><td>0.00288 </td><td>A-Proteobacteria </td><td>2.14 </td></tr><tr><td>63 </td><td><em>Methylobacillus flagellatus</em> KT </td><td>2971 </td><td>55.72 </td><td>0.03 </td><td>0.916 </td><td>0.00226 </td><td>0.00135 </td><td>B-Proteobacteria </td><td>10.57 </td></tr><tr><td>64 </td><td><em>Methylococcus capsulatus</em> Bath </td><td>3304 </td><td>63.59 </td><td>0.145 </td><td><em>0.004</em> </td><td>0.00150 </td><td>0.00287 </td><td>G-Proteobacteria </td><td>NA </td></tr><tr><td>65 </td><td><em>Mycobacterium leprae</em> TN </td><td>3268 </td><td>57.8 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00378 </td><td>0.00609 </td><td rowspan="4">Actinobacteria </td><td>7.04 </td></tr><tr><td>66 </td><td><em>Mycobacterium</em> sp. KMS </td><td>5737 </td><td>68.44 </td><td>0.389 </td><td>0.478 </td><td>0.00030 </td><td>0.00060 </td><td rowspan="3">NA </td></tr><tr><td>67 </td><td><em>Mycobacterium tuberculosis</em> F11 </td><td>4424 </td><td>65.62 </td><td>0.366 </td><td><em>0.007</em> </td><td>0.00006 </td><td>0.00198 </td></tr><tr><td>68 </td><td><em>Mycobacterium ulcerans</em> Agy99 </td><td>5631 </td><td>65.47 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00433 </td><td>0.00374 </td></tr><tr><td>69 </td><td><em>Mycoplasma gallisepticum</em> R </td><td>996 </td><td>31.45 </td><td>0.18 </td><td>0.615 </td><td>0.00626 </td><td>0.00021 </td><td rowspan="4">Tenericutes </td><td>9.32 </td></tr><tr><td>70 </td><td><em>Mycoplasma genitalium</em> G37 </td><td>580 </td><td>31.69 </td><td><em>0</em> </td><td>0.148 </td><td>0.01219 </td><td>0.00433 </td><td>3.75 </td></tr><tr><td>71 </td><td><em>Mycoplasma hyopneumoniae</em> J </td><td>897 </td><td>28.52 </td><td>0.033 </td><td>0.599 </td><td>0.01020 </td><td>0.00067 </td><td>NA </td></tr><tr><td>72 </td><td><em>Mycoplasma pneumoniae</em> M129 </td><td>816 </td><td>40.01 </td><td><em>0.001</em> </td><td>0.115 </td><td>0.01767 </td><td>0.00243 </td><td>16.23 </td></tr><tr><td>73 </td><td><em>Neisseria gonorrhoeae</em> FA 1090 </td><td>2153 </td><td>52.69 </td><td>0.07 </td><td>0.033 </td><td>0.00601 </td><td>0.00144 </td><td rowspan="2">B-Proteobacteria </td><td>9.20 </td></tr><tr><td>74 </td><td><em>Neisseria meningitidis</em> MC58 </td><td>2273 </td><td>51.52 </td><td>0.695 </td><td><em>0.004</em> </td><td>0.00135 </td><td>0.00806 </td><td rowspan="2">NA </td></tr><tr><td>75 </td><td><em>Nitrobacter hamburgensis</em> X14 </td><td>4406 </td><td>61.72 </td><td>0.332 </td><td>0.53 </td><td>0.00112 </td><td>0.00041 </td><td rowspan="2">A-Proteobacteria </td></tr><tr><td>76 </td><td><em>Nitrobacter winogradskyi</em> Nb-255 </td><td>3402 </td><td>62.05 </td><td>0.011 </td><td><strong>&lt;0.0001</strong> </td><td>0.00323 </td><td>0.00294 </td><td>37.15 </td></tr><tr><td>77 </td><td><em>Nitrosococcus oceani</em> ATCC 19707 </td><td>3481 </td><td>50.32 </td><td>0.02 </td><td>0.056 </td><td>0.00530 </td><td>0.00243 </td><td>G-Proteobacteria </td><td>8.39 </td></tr><tr><td>78 </td><td><em>Nitrosomonas eutropha</em> C91 </td><td>2661 </td><td>48.49 </td><td>0.992 </td><td>0.318 </td><td>0.00043 </td><td>0.00162 </td><td>B-Proteobacteria </td><td rowspan="2">NA </td></tr><tr><td>79 </td><td><em>Nostoc</em> sp. PCC 7120 </td><td>6413 </td><td>41.35 </td><td>0.134 </td><td>0.857 </td><td>0.00129 </td><td>0.00162 </td><td>Cyanobacteria </td></tr><tr><td>80 </td><td><em>Pseudomonas entomophila</em> L48 chromosome </td><td>5888 </td><td>64.16 </td><td>0.657 </td><td>0.251 </td><td>0.00078 </td><td>0.00173 </td><td rowspan="3">G-Proteobacteria </td><td>1.99 </td></tr><tr><td>81 </td><td><em>Pseudomonas fluorescens</em> PfO-1 </td><td>6438 </td><td>60.52 </td><td><em>0.003</em> </td><td>0.028 </td><td>0.00443 </td><td>0.00222 </td><td>3.18 </td></tr><tr><td>82 </td><td><em>Pseudomonas putida</em> F1 </td><td>5959 </td><td>61.86 </td><td>0.602 </td><td>0.013 </td><td>0.00113 </td><td>0.00187 </td><td>36.81 </td></tr><tr><td>83 </td><td><em>Ralstonia eutropha</em> H16 </td><td>2912 </td><td>66.78 </td><td>0.238 </td><td>0.47 </td><td>0.00483 </td><td>0.00023 </td><td rowspan="2">B-Proteobacteria </td><td>NA </td></tr><tr><td>84 </td><td><em>Ralstonia solanacearum</em> GMI1000 chromosome </td><td>3716 </td><td>67.04 </td><td>0.056 </td><td><strong>&lt;0.0001</strong> </td><td>0.00636 </td><td>0.00581 </td><td>22.40 </td></tr><tr><td>85 </td><td><em>Rhizobium etli</em> CFN 42 </td><td>4381 </td><td>61.27 </td><td>0.107 </td><td><strong>&lt;0.0001</strong> </td><td>0.00175 </td><td>0.01177 </td><td rowspan="6">A-Proteobacteria </td><td>17.65 </td></tr><tr><td>86 </td><td><em>Rhizobium leguminosarum</em> bv. <em>viciae</em> 3841 </td><td>5057 </td><td>61.09 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00363 </td><td>0.01196 </td><td>NA </td></tr><tr><td>87 </td><td><em>Rickettsia bellii</em> RML369-C </td><td>1522 </td><td>31.65 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00859 </td><td>0.01514 </td><td>26.08 </td></tr><tr><td>88 </td><td><em>Rickettsia conorii</em> Malish 7 </td><td>1268 </td><td>32.44 </td><td>0.584 </td><td>0.052 </td><td>0.00294 </td><td>0.00634 </td><td>16.28 </td></tr><tr><td>89 </td><td><em>Rickettsia rickettsii</em> 'Sheila Smith' </td><td>1257 </td><td>32.47 </td><td>0.575 </td><td><em>0.002</em> </td><td>0.00182 </td><td>0.00767 </td><td>NA </td></tr><tr><td>90 </td><td><em>Rickettsia typhi</em> Wilmington </td><td>1111 </td><td>28.92 </td><td>0.919 </td><td><em>0.007</em> </td><td>0.00020 </td><td>0.01395 </td><td>26.15 </td></tr><tr><td>91 </td><td><em>Salmonella enterica</em> subsp. <em>enterica</em> serovar Typhi CT18 </td><td>4809 </td><td>52.09 </td><td>0.267 </td><td>0.043 </td><td>0.00151 </td><td>0.00152 </td><td rowspan="4">G-Proteobacteria </td><td>9.85 </td></tr><tr><td>92 </td><td><em>Salmonella typhimurium</em> LT2 </td><td>4857 </td><td>52.22 </td><td>0.89 </td><td>0.585 </td><td>0.00043 </td><td>0.00008 </td><td>3.58 </td></tr><tr><td>93 </td><td><em>Shigella boydii</em> Sb227 </td><td>4519 </td><td>51.21 </td><td>0.571 </td><td><em>0.001</em> </td><td>0.00022 </td><td>0.00249 </td><td>11.05 </td></tr><tr><td>94 </td><td><em>Shigella flexneri</em> 58401 </td><td>4574 </td><td>50.92 </td><td>0.48 </td><td>0.268 </td><td>0.00147 </td><td>0.00214 </td><td>NA </td></tr><tr><td>95 </td><td><em>Staphylococcus aureus</em> RF122 </td><td>2742 </td><td>32.78 </td><td>0.788 </td><td>0.427 </td><td>0.00130 </td><td>0.00247 </td><td rowspan="6">Firmicutes </td><td>0.10 </td></tr><tr><td>96 </td><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>2499 </td><td>32.1 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.01246 </td><td>0.01087 </td><td>21.12 </td></tr><tr><td>97 </td><td><em>Staphylococcus haemolyticus</em> JCSC1435 </td><td>2685 </td><td>32.79 </td><td><em>0.001</em> </td><td><em>0</em> </td><td>0.00584 </td><td>0.00643 </td><td rowspan="2">NA </td></tr><tr><td>98 </td><td><em>Streptococcus mutans</em> UA159 </td><td>2030 </td><td>36.83 </td><td>0.111 </td><td>0.046 </td><td>0.00403 </td><td>0.00679 </td></tr><tr><td>99 </td><td><em>Streptococcus pyogenes</em> MGAS2096 </td><td>1860 </td><td>38.73 </td><td>0.619 </td><td>0.15 </td><td>0.00133 </td><td>0.00154 </td><td>3.71 </td></tr><tr><td>100 </td><td><em>Streptococcus thermophilus</em> CNRZ1066 </td><td>1796 </td><td>39.08 </td><td>0.05 </td><td>0.863 </td><td>0.00537 </td><td>0.00459 </td><td>2.63 </td></tr><tr><td>101 </td><td><em>Streptomyces coelicolor</em> A3(2) </td><td>8667 </td><td>72.12 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00394 </td><td>0.00134 </td><td>Actinobacteria </td><td>NA </td></tr><tr><td>102 </td><td><em>Thermotoga maritima</em> MSB8 </td><td>1860 </td><td>46.25 </td><td>0.171 </td><td><strong>&lt;0.0001</strong> </td><td>0.00344 </td><td>0.01548 </td><td rowspan="2">Thermotogae </td><td>39.15 </td></tr><tr><td>103 </td><td><em>Thermotoga petrophila</em> RKU-1 </td><td>1824 </td><td>46.09 </td><td>0.733 </td><td><strong>&lt;0.0001</strong> </td><td>0.00013 </td><td>0.01687 </td><td>NA </td></tr><tr><td>104 </td><td><em>Thiobacillus denitrificans</em> ATCC 25259 </td><td>2909 </td><td>66.07 </td><td>0.962 </td><td>0.086 </td><td>0.00027 </td><td>0.00059 </td><td>B-Proteobacteria </td><td>5.70 </td></tr><tr><td>105 </td><td><em>Vibrio cholerae</em> O395 </td><td>3024 </td><td>47.78 </td><td><strong>&lt;0.0001</strong> </td><td>0.069 </td><td>0.00514 </td><td>0.00105 </td><td rowspan="8">G-Proteobacteria </td><td rowspan="4">NA </td></tr><tr><td>106 </td><td><em>Vibrio fischeri</em> ES114 </td><td>1332 </td><td>37.03 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00994 </td><td>0.00491 </td></tr><tr><td>107 </td><td><em>Xanthomonas campestris</em> pv. <em>campestris</em> ATCC 33913 </td><td>5076 </td><td>65.07 </td><td>0.196 </td><td>0.719 </td><td>0.00302 </td><td>0.00038 </td></tr><tr><td>108 </td><td><em>Xanthomonas oryzae</em> pv. <em>oryzae</em> KACC 10331 </td><td>4941 </td><td>63.69 </td><td>0.87 </td><td>0.499 </td><td>0.00104 </td><td>0.00065 </td></tr><tr><td>109 </td><td><em>Xylella fastidiosa</em> 9a5c </td><td>2679 </td><td>52.68 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.04727 </td><td>0.05291 </td><td>62.97 </td></tr><tr><td>110 </td><td><em>Xylella fastidiosa</em> Temecula 1 </td><td>2519 </td><td>51.78 </td><td>0.044 </td><td><em>0</em> </td><td>0.00379 </td><td>0.01093 </td><td>6.44 </td></tr><tr><td>111 </td><td><em>Yersinia pestis</em> CO92 </td><td>4653 </td><td>47.64 </td><td>0.649 </td><td><em>0.001</em> </td><td>0.00090 </td><td>0.00520 </td><td rowspan="2">NA </td></tr><tr><td>112 </td><td><em>Yersinia pseudotuberculosis</em> IP32953 </td><td>4744 </td><td>47.61 </td><td>0.969 </td><td><em>0.001</em> </td><td>0.00124 </td><td>0.00496 </td></tr></tbody></table></div><div content-id="DSP021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Bacterial group </th><th>TB (°) </th></tr></thead><tbody><tr><td>1 </td><td><em>Acinetobacter</em> sp. ADP1 </td><td>3598 </td><td>40.43 </td><td>0.745 </td><td><em>0.006</em> </td><td>0.00068 </td><td>0.00484 </td><td rowspan="5">G-Proteobacteria </td><td>7.07 </td></tr><tr><td>2 </td><td><em>Actinobacillus pleuropneumoniae</em> L20 serotype 5b </td><td>2274 </td><td>41.3 </td><td>0.436 </td><td>0.819 </td><td>0.00187 </td><td>0.00109 </td><td rowspan="4">NA </td></tr><tr><td>3 </td><td><em>Actinobacillus succinogenes</em> 130Z </td><td>2319 </td><td>44.91 </td><td>0.312 </td><td>0.291 </td><td>0.00232 </td><td>0.00291 </td></tr><tr><td>4 </td><td><em>Aeromonas hydrophila</em> subsp. <em>hydrophila</em> ATCC 7966 </td><td>4744 </td><td>61.55 </td><td>0.88 </td><td>0.19 </td><td>0.00141 </td><td>0.00139 </td></tr><tr><td>5 </td><td><em>Aeromonas salmonicida</em> subsp. <em>salmonicida</em> A449 </td><td>4702 </td><td>58.51 </td><td>0.04 </td><td>0.959 </td><td>0.00215 </td><td>0.00073 </td></tr><tr><td>6 </td><td><em>Agrobacterium tumefaciens</em> C58 (circular chromosome) </td><td>2841 </td><td>59.38 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00694 </td><td>0.00967 </td><td>A-Proteobacteria </td><td>7.37 </td></tr><tr><td>7 </td><td><em>Alkaliphilus oremlandii</em> OhILAs </td><td>3123 </td><td>36.26 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00615 </td><td>0.01324 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>8 </td><td><em>Anaeromyxobacter dehalogenans</em> 2CP-C </td><td>5013 </td><td>74.9 </td><td>0.077 </td><td><em>0.001</em> </td><td>0.00476 </td><td>0.00249 </td><td rowspan="2">D-Proteobacteria </td><td>70.57 </td></tr><tr><td>9 </td><td><em>Anaeromyxobacter</em> sp. Fw109-5 </td><td>5277 </td><td>73.53 </td><td>0.712 </td><td><em>0.008</em> </td><td>0.00073 </td><td>0.00216 </td><td>7.48 </td></tr><tr><td>10 </td><td><em>Bacillus anthracis</em> Ames </td><td>5227 </td><td>35.38 </td><td><em>0.004</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00581 </td><td rowspan="6">Firmicutes </td><td>NA </td></tr><tr><td>11 </td><td><em>Bacillus anthracis</em> 'Ames Ancestor' </td><td>5227 </td><td>35.38 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00582 </td><td>7.48 </td></tr><tr><td>12 </td><td><em>Bacillus anthracis</em> Sterne </td><td>5228 </td><td>35.38 </td><td><em>0.008</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00221 </td><td>0.00588 </td><td>7.46 </td></tr><tr><td>13 </td><td><em>Bacillus subtilis</em> </td><td>4214 </td><td>43.52 </td><td>0.219 </td><td>0.234 </td><td>0.00212 </td><td>0.00224 </td><td>13.69 </td></tr><tr><td>14 </td><td><em>Bacillus thuringiensis</em> Al Hakam </td><td>5257 </td><td>35.43 </td><td>0.123 </td><td><em>0.002</em> </td><td>0.00042 </td><td>0.00081 </td><td>NA </td></tr><tr><td>15 </td><td><em>Bacillus thuringiensis</em> serovar konkukian 97-27 </td><td>5237 </td><td>35.41 </td><td>0.015 </td><td><strong>&lt;0.0001</strong> </td><td>0.00194 </td><td>0.00438 </td><td>3.98 </td></tr><tr><td>16 </td><td><em>Bordetella parapertussis</em> 12822 </td><td>4773 </td><td>68.1 </td><td>0.433 </td><td><strong>&lt;0.0001</strong> </td><td>0.00247 </td><td>0.00776 </td><td rowspan="2">B-Proteobacteria </td><td>37.01 </td></tr><tr><td>17 </td><td><em>Bordetella pertussis</em> Tohama 1 </td><td>4086 </td><td>67.72 </td><td>0.861 </td><td><strong>&lt;0.0001</strong> </td><td>0.00022 </td><td>0.00390 </td><td>71.28 </td></tr><tr><td>18 </td><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>9105 </td><td>64.06 </td><td>0.512 </td><td>0.31 </td><td>0.00070 </td><td>0.00038 </td><td rowspan="3">A-Proteobacteria </td><td>7.07 </td></tr><tr><td>19 </td><td><em>Bradyrhizobium</em> sp. BTAi1 </td><td>8264 </td><td>64.92 </td><td>0.381 </td><td>0.01 </td><td>0.00100 </td><td>0.00163 </td><td rowspan="4">NA </td></tr><tr><td>20 </td><td><em>Brucella melitensis</em> 16M </td><td>1177 </td><td>57.35 </td><td>0.472 </td><td><em>0.008</em> </td><td>0.00227 </td><td>0.00312 </td></tr><tr><td>21 </td><td><em>Campylobacter concisus</em> 13826 </td><td>2052 </td><td>39.43 </td><td>0.033 </td><td>0.048 </td><td>0.00038 </td><td>0.00599 </td><td rowspan="5">E-Proteobacteria </td></tr><tr><td>22 </td><td><em>Campylobacter curvus</em> 525.92 </td><td>1971 </td><td>44.54 </td><td>0.028 </td><td>0.752 </td><td>0.00745 </td><td>0.00282 </td></tr><tr><td>23 </td><td><em>Campylobacter jejuni</em> RM1221 </td><td>1777 </td><td>30.31 </td><td>0.574 </td><td>0.23 </td><td>0.00330 </td><td>0.00436 </td><td>8.69 </td></tr><tr><td>24 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> 81116 </td><td>1628 </td><td>30.54 </td><td>0.491 </td><td>0.029 </td><td>0.00250 </td><td>0.00613 </td><td>NA </td></tr><tr><td>25 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> NCTC 11168 </td><td>1641 </td><td>30.55 </td><td>0.067 </td><td>0.132 </td><td>0.00296 </td><td>0.00457 </td><td>10.25 </td></tr><tr><td>26 </td><td>Candidatus <em>Desulfococcus oleovorans</em> Hxd3 </td><td>3944 </td><td>56.17 </td><td>0.258 </td><td>0.133 </td><td>0.00199 </td><td>0.00157 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>27 </td><td><em>Caulobacter crescentus</em> CB15 </td><td>4016 </td><td>67.22 </td><td>0.042 </td><td>0.171 </td><td>0.00396 </td><td>0.00188 </td><td>A-Proteobacteria </td><td>8.56 </td></tr><tr><td>28 </td><td><em>Chlamydia muridarum</em> Nigg </td><td>1072 </td><td>40.34 </td><td>0.221 </td><td>0.853 </td><td>0.00107 </td><td>0.00337 </td><td rowspan="3">Chlamydiae </td><td>1.17 </td></tr><tr><td>29 </td><td><em>Chlamydia trachomatis</em> AHAR-13 </td><td>1044 </td><td>41.31 </td><td>0.228 </td><td>0.284 </td><td>0.00230 </td><td>0.00059 </td><td>1.30 </td></tr><tr><td>30 </td><td><em>Chlamydophila abortus</em> S263 </td><td>1144 </td><td>39.87 </td><td>0.534 </td><td><em>0.002</em> </td><td>0.00065 </td><td>0.00361 </td><td>0.57 </td></tr><tr><td>31 </td><td><em>Coxiella burnetii</em> Dugway 7E9-12 </td><td>2158 </td><td>42.44 </td><td><em>0.004</em> </td><td><em>0.001</em> </td><td>0.00592 </td><td>0.00573 </td><td rowspan="2">G-Proteobacteria </td><td>NA </td></tr><tr><td>32 </td><td><em>Coxiella burnetii</em> RSA 493 </td><td>1995 </td><td>42.66 </td><td>0.014 </td><td>0.467 </td><td>0.00198 </td><td>0.00029 </td><td>31.15 </td></tr><tr><td>33 </td><td><em>Desulfovibrio desulfuricans</em> G20 </td><td>3730 </td><td>57.84 </td><td>0.59 </td><td><em>0.001</em> </td><td>0.00189 </td><td>0.00322 </td><td>Firmicutes </td><td>10.70 </td></tr><tr><td>34 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> DP4 </td><td>3462 </td><td>63.01 </td><td>0.3 </td><td>0.159 </td><td>0.00152 </td><td>0.00106 </td><td rowspan="2">D-Proteobacteria </td><td>NA </td></tr><tr><td>35 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> Hildenborough </td><td>3570 </td><td>63.14 </td><td>0.557 </td><td>0.082 </td><td>0.00143 </td><td>0.00024 </td><td>4.78 </td></tr><tr><td>36 </td><td><em>Enterobacter sakazakii</em> ATCC BAA-894 </td><td>4368 </td><td>56.77 </td><td>0.167 </td><td>0.388 </td><td>0.00359 </td><td>0.00044 </td><td rowspan="10">G-Proteobacteria </td><td>NA </td></tr><tr><td>37 </td><td><em>Enterobacter</em> sp. 638 </td><td>4518 </td><td>52.98 </td><td>0.645 </td><td>0.39 </td><td>0.00169 </td><td>0.00163 </td><td>NA </td></tr><tr><td>38 </td><td><em>Escherichia coli</em> 536 </td><td>4938 </td><td>50.52 </td><td>0.714 </td><td>0.084 </td><td>0.00062 </td><td>0.00328 </td><td>7.40 </td></tr><tr><td>39 </td><td><em>Escherichia coli</em> APEC O1 </td><td>5082 </td><td>50.55 </td><td>0.779 </td><td>0.576 </td><td>0.00032 </td><td>0.00070 </td><td>NA </td></tr><tr><td>40 </td><td><em>Escherichia coli</em> CFT073 </td><td>5231 </td><td>50.48 </td><td>0.112 </td><td>0.92 </td><td>0.00173 </td><td>0.00080 </td><td>5.66 </td></tr><tr><td>41 </td><td><em>Escherichia coli</em> E24377A </td><td>4979 </td><td>50.62 </td><td>0.736 </td><td>0.128 </td><td>0.00205 </td><td>0.00212 </td><td rowspan="2">NA </td></tr><tr><td>42 </td><td><em>Escherichia coli</em> HS </td><td>4643 </td><td>50.82 </td><td>0.328 </td><td>0.469 </td><td>0.00151 </td><td>0.00207 </td></tr><tr><td>43 </td><td><em>Escherichia coli</em> K12 MG1655 </td><td>4639 </td><td>50.79 </td><td>0.732 </td><td>0.587 </td><td>0.00054 </td><td>0.00113 </td><td>4.28 </td></tr><tr><td>44 </td><td><em>Escherichia coli</em> UTI89 </td><td>5065 </td><td>50.6 </td><td>0.51 </td><td>0.237 </td><td>0.00076 </td><td>0.00203 </td><td>3.70 </td></tr><tr><td>45 </td><td><em>Escherichia coli</em> W3110 </td><td>4646 </td><td>50.8 </td><td>0.873 </td><td>0.729 </td><td>0.00073 </td><td>0.00091 </td><td>12.64 </td></tr><tr><td>46 </td><td><em>Frankia alni</em> ACN14A chromosome </td><td>7497 </td><td>72.82 </td><td>0.463 </td><td>0.036 </td><td>0.00141 </td><td>0.00139 </td><td rowspan="2">Actinobacteria </td><td rowspan="5">NA </td></tr><tr><td>47 </td><td><em>Frankia</em> sp. CcI3 </td><td>5433 </td><td>70.08 </td><td>0.808 </td><td>0.662 </td><td>0.00129 </td><td>0.00017 </td></tr><tr><td>48 </td><td><em>Haemophilus influenzae</em> 86-028NP </td><td>1914 </td><td>38.16 </td><td>0.886 </td><td>0.654 </td><td>0.00089 </td><td>0.00044 </td><td rowspan="4">G-Proteobacteria </td></tr><tr><td>49 </td><td><em>Haemophilus influenzae</em> PittEE </td><td>1813 </td><td>38.04 </td><td>0.544 </td><td>0.038 </td><td>0.00054 </td><td>0.00317 </td></tr><tr><td>50 </td><td><em>Haemophilus influenzae</em> PittGG </td><td>1887 </td><td>38.01 </td><td>0.125 </td><td><strong>&lt;0.0001</strong> </td><td>0.00005 </td><td>0.01016 </td></tr><tr><td>51 </td><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>1830 </td><td>38.15 </td><td>0.154 </td><td><em>0.004</em> </td><td>0.00298 </td><td>0.00472 </td><td>46.61 </td></tr><tr><td>52 </td><td><em>Helicobacter acinonychis</em> Sheeba </td><td>1553 </td><td>38.18 </td><td><em>0</em> </td><td>0.596 </td><td>0.00869 </td><td>0.00164 </td><td rowspan="3">E-Proteobacteria </td><td>NA </td></tr><tr><td>53 </td><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>1799 </td><td>35.93 </td><td>0.161 </td><td><strong>&lt;0.0001</strong> </td><td>0.00499 </td><td>0.01518 </td><td>46.54 </td></tr><tr><td>54 </td><td><em>Helicobacter pylori</em> J99 </td><td>1643 </td><td>39.19 </td><td>0.246 </td><td>0.256 </td><td>0.00259 </td><td>0.00510 </td><td>10.97 </td></tr><tr><td>55 </td><td><em>Lactobacillus acidophilus</em> NCFM </td><td>1993 </td><td>34.72 </td><td>0.382 </td><td><strong>&lt;0.0001</strong> </td><td>0.00066 </td><td>0.01644 </td><td rowspan="6">Firmicutes </td><td>19.54 </td></tr><tr><td>56 </td><td><em>Lactobacillus brevis</em> ATCC 367 </td><td>2291 </td><td>46.22 </td><td>0.023 </td><td><strong>&lt;0.0001</strong> </td><td>0.00271 </td><td>0.02882 </td><td rowspan="6">NA </td></tr><tr><td>57 </td><td><em>Lactobacillus delbrueckii</em> subsp. <em>bulgaricus</em> ATCC BAA-365 </td><td>1856 </td><td>49.69 </td><td>0.491 </td><td>0.264 </td><td>0.00201 </td><td>0.00087 </td></tr><tr><td>58 </td><td><em>Lactobacillus reuteri</em> F275 </td><td>1999 </td><td>38.87 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00122 </td><td>0.01040 </td></tr><tr><td>59 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363 </td><td>2529 </td><td>35.75 </td><td>0.233 </td><td>0.056 </td><td>0.00352 </td><td>0.00524 </td></tr><tr><td>60 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> SK11 </td><td>2438 </td><td>35.86 </td><td>0.399 </td><td>0.521 </td><td>0.00147 </td><td>0.00136 </td></tr><tr><td>61 </td><td><em>Magnetococcus</em> sp. MC-1 </td><td>4719 </td><td>54.17 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00490 </td><td>0.01198 </td><td>Magnetococcus </td></tr><tr><td>62 </td><td><em>Magnetospirillum magneticum</em> AMB-1 </td><td>4967 </td><td>65.09 </td><td>0.031 </td><td><strong>&lt;0.0001</strong> </td><td>0.00339 </td><td>0.00288 </td><td>A-Proteobacteria </td><td>2.14 </td></tr><tr><td>63 </td><td><em>Methylobacillus flagellatus</em> KT </td><td>2971 </td><td>55.72 </td><td>0.03 </td><td>0.916 </td><td>0.00226 </td><td>0.00135 </td><td>B-Proteobacteria </td><td>10.57 </td></tr><tr><td>64 </td><td><em>Methylococcus capsulatus</em> Bath </td><td>3304 </td><td>63.59 </td><td>0.145 </td><td><em>0.004</em> </td><td>0.00150 </td><td>0.00287 </td><td>G-Proteobacteria </td><td>NA </td></tr><tr><td>65 </td><td><em>Mycobacterium leprae</em> TN </td><td>3268 </td><td>57.8 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00378 </td><td>0.00609 </td><td rowspan="4">Actinobacteria </td><td>7.04 </td></tr><tr><td>66 </td><td><em>Mycobacterium</em> sp. KMS </td><td>5737 </td><td>68.44 </td><td>0.389 </td><td>0.478 </td><td>0.00030 </td><td>0.00060 </td><td rowspan="3">NA </td></tr><tr><td>67 </td><td><em>Mycobacterium tuberculosis</em> F11 </td><td>4424 </td><td>65.62 </td><td>0.366 </td><td><em>0.007</em> </td><td>0.00006 </td><td>0.00198 </td></tr><tr><td>68 </td><td><em>Mycobacterium ulcerans</em> Agy99 </td><td>5631 </td><td>65.47 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00433 </td><td>0.00374 </td></tr><tr><td>69 </td><td><em>Mycoplasma gallisepticum</em> R </td><td>996 </td><td>31.45 </td><td>0.18 </td><td>0.615 </td><td>0.00626 </td><td>0.00021 </td><td rowspan="4">Tenericutes </td><td>9.32 </td></tr><tr><td>70 </td><td><em>Mycoplasma genitalium</em> G37 </td><td>580 </td><td>31.69 </td><td><em>0</em> </td><td>0.148 </td><td>0.01219 </td><td>0.00433 </td><td>3.75 </td></tr><tr><td>71 </td><td><em>Mycoplasma hyopneumoniae</em> J </td><td>897 </td><td>28.52 </td><td>0.033 </td><td>0.599 </td><td>0.01020 </td><td>0.00067 </td><td>NA </td></tr><tr><td>72 </td><td><em>Mycoplasma pneumoniae</em> M129 </td><td>816 </td><td>40.01 </td><td><em>0.001</em> </td><td>0.115 </td><td>0.01767 </td><td>0.00243 </td><td>16.23 </td></tr><tr><td>73 </td><td><em>Neisseria gonorrhoeae</em> FA 1090 </td><td>2153 </td><td>52.69 </td><td>0.07 </td><td>0.033 </td><td>0.00601 </td><td>0.00144 </td><td rowspan="2">B-Proteobacteria </td><td>9.20 </td></tr><tr><td>74 </td><td><em>Neisseria meningitidis</em> MC58 </td><td>2273 </td><td>51.52 </td><td>0.695 </td><td><em>0.004</em> </td><td>0.00135 </td><td>0.00806 </td><td rowspan="2">NA </td></tr><tr><td>75 </td><td><em>Nitrobacter hamburgensis</em> X14 </td><td>4406 </td><td>61.72 </td><td>0.332 </td><td>0.53 </td><td>0.00112 </td><td>0.00041 </td><td rowspan="2">A-Proteobacteria </td></tr><tr><td>76 </td><td><em>Nitrobacter winogradskyi</em> Nb-255 </td><td>3402 </td><td>62.05 </td><td>0.011 </td><td><strong>&lt;0.0001</strong> </td><td>0.00323 </td><td>0.00294 </td><td>37.15 </td></tr><tr><td>77 </td><td><em>Nitrosococcus oceani</em> ATCC 19707 </td><td>3481 </td><td>50.32 </td><td>0.02 </td><td>0.056 </td><td>0.00530 </td><td>0.00243 </td><td>G-Proteobacteria </td><td>8.39 </td></tr><tr><td>78 </td><td><em>Nitrosomonas eutropha</em> C91 </td><td>2661 </td><td>48.49 </td><td>0.992 </td><td>0.318 </td><td>0.00043 </td><td>0.00162 </td><td>B-Proteobacteria </td><td rowspan="2">NA </td></tr><tr><td>79 </td><td><em>Nostoc</em> sp. PCC 7120 </td><td>6413 </td><td>41.35 </td><td>0.134 </td><td>0.857 </td><td>0.00129 </td><td>0.00162 </td><td>Cyanobacteria </td></tr><tr><td>80 </td><td><em>Pseudomonas entomophila</em> L48 chromosome </td><td>5888 </td><td>64.16 </td><td>0.657 </td><td>0.251 </td><td>0.00078 </td><td>0.00173 </td><td rowspan="3">G-Proteobacteria </td><td>1.99 </td></tr><tr><td>81 </td><td><em>Pseudomonas fluorescens</em> PfO-1 </td><td>6438 </td><td>60.52 </td><td><em>0.003</em> </td><td>0.028 </td><td>0.00443 </td><td>0.00222 </td><td>3.18 </td></tr><tr><td>82 </td><td><em>Pseudomonas putida</em> F1 </td><td>5959 </td><td>61.86 </td><td>0.602 </td><td>0.013 </td><td>0.00113 </td><td>0.00187 </td><td>36.81 </td></tr><tr><td>83 </td><td><em>Ralstonia eutropha</em> H16 </td><td>2912 </td><td>66.78 </td><td>0.238 </td><td>0.47 </td><td>0.00483 </td><td>0.00023 </td><td rowspan="2">B-Proteobacteria </td><td>NA </td></tr><tr><td>84 </td><td><em>Ralstonia solanacearum</em> GMI1000 chromosome </td><td>3716 </td><td>67.04 </td><td>0.056 </td><td><strong>&lt;0.0001</strong> </td><td>0.00636 </td><td>0.00581 </td><td>22.40 </td></tr><tr><td>85 </td><td><em>Rhizobium etli</em> CFN 42 </td><td>4381 </td><td>61.27 </td><td>0.107 </td><td><strong>&lt;0.0001</strong> </td><td>0.00175 </td><td>0.01177 </td><td rowspan="6">A-Proteobacteria </td><td>17.65 </td></tr><tr><td>86 </td><td><em>Rhizobium leguminosarum</em> bv. <em>viciae</em> 3841 </td><td>5057 </td><td>61.09 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00363 </td><td>0.01196 </td><td>NA </td></tr><tr><td>87 </td><td><em>Rickettsia bellii</em> RML369-C </td><td>1522 </td><td>31.65 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00859 </td><td>0.01514 </td><td>26.08 </td></tr><tr><td>88 </td><td><em>Rickettsia conorii</em> Malish 7 </td><td>1268 </td><td>32.44 </td><td>0.584 </td><td>0.052 </td><td>0.00294 </td><td>0.00634 </td><td>16.28 </td></tr><tr><td>89 </td><td><em>Rickettsia rickettsii</em> 'Sheila Smith' </td><td>1257 </td><td>32.47 </td><td>0.575 </td><td><em>0.002</em> </td><td>0.00182 </td><td>0.00767 </td><td>NA </td></tr><tr><td>90 </td><td><em>Rickettsia typhi</em> Wilmington </td><td>1111 </td><td>28.92 </td><td>0.919 </td><td><em>0.007</em> </td><td>0.00020 </td><td>0.01395 </td><td>26.15 </td></tr><tr><td>91 </td><td><em>Salmonella enterica</em> subsp. <em>enterica</em> serovar Typhi CT18 </td><td>4809 </td><td>52.09 </td><td>0.267 </td><td>0.043 </td><td>0.00151 </td><td>0.00152 </td><td rowspan="4">G-Proteobacteria </td><td>9.85 </td></tr><tr><td>92 </td><td><em>Salmonella typhimurium</em> LT2 </td><td>4857 </td><td>52.22 </td><td>0.89 </td><td>0.585 </td><td>0.00043 </td><td>0.00008 </td><td>3.58 </td></tr><tr><td>93 </td><td><em>Shigella boydii</em> Sb227 </td><td>4519 </td><td>51.21 </td><td>0.571 </td><td><em>0.001</em> </td><td>0.00022 </td><td>0.00249 </td><td>11.05 </td></tr><tr><td>94 </td><td><em>Shigella flexneri</em> 58401 </td><td>4574 </td><td>50.92 </td><td>0.48 </td><td>0.268 </td><td>0.00147 </td><td>0.00214 </td><td>NA </td></tr><tr><td>95 </td><td><em>Staphylococcus aureus</em> RF122 </td><td>2742 </td><td>32.78 </td><td>0.788 </td><td>0.427 </td><td>0.00130 </td><td>0.00247 </td><td rowspan="6">Firmicutes </td><td>0.10 </td></tr><tr><td>96 </td><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>2499 </td><td>32.1 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.01246 </td><td>0.01087 </td><td>21.12 </td></tr><tr><td>97 </td><td><em>Staphylococcus haemolyticus</em> JCSC1435 </td><td>2685 </td><td>32.79 </td><td><em>0.001</em> </td><td><em>0</em> </td><td>0.00584 </td><td>0.00643 </td><td rowspan="2">NA </td></tr><tr><td>98 </td><td><em>Streptococcus mutans</em> UA159 </td><td>2030 </td><td>36.83 </td><td>0.111 </td><td>0.046 </td><td>0.00403 </td><td>0.00679 </td></tr><tr><td>99 </td><td><em>Streptococcus pyogenes</em> MGAS2096 </td><td>1860 </td><td>38.73 </td><td>0.619 </td><td>0.15 </td><td>0.00133 </td><td>0.00154 </td><td>3.71 </td></tr><tr><td>100 </td><td><em>Streptococcus thermophilus</em> CNRZ1066 </td><td>1796 </td><td>39.08 </td><td>0.05 </td><td>0.863 </td><td>0.00537 </td><td>0.00459 </td><td>2.63 </td></tr><tr><td>101 </td><td><em>Streptomyces coelicolor</em> A3(2) </td><td>8667 </td><td>72.12 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00394 </td><td>0.00134 </td><td>Actinobacteria </td><td>NA </td></tr><tr><td>102 </td><td><em>Thermotoga maritima</em> MSB8 </td><td>1860 </td><td>46.25 </td><td>0.171 </td><td><strong>&lt;0.0001</strong> </td><td>0.00344 </td><td>0.01548 </td><td rowspan="2">Thermotogae </td><td>39.15 </td></tr><tr><td>103 </td><td><em>Thermotoga petrophila</em> RKU-1 </td><td>1824 </td><td>46.09 </td><td>0.733 </td><td><strong>&lt;0.0001</strong> </td><td>0.00013 </td><td>0.01687 </td><td>NA </td></tr><tr><td>104 </td><td><em>Thiobacillus denitrificans</em> ATCC 25259 </td><td>2909 </td><td>66.07 </td><td>0.962 </td><td>0.086 </td><td>0.00027 </td><td>0.00059 </td><td>B-Proteobacteria </td><td>5.70 </td></tr><tr><td>105 </td><td><em>Vibrio cholerae</em> O395 </td><td>3024 </td><td>47.78 </td><td><strong>&lt;0.0001</strong> </td><td>0.069 </td><td>0.00514 </td><td>0.00105 </td><td rowspan="8">G-Proteobacteria </td><td rowspan="4">NA </td></tr><tr><td>106 </td><td><em>Vibrio fischeri</em> ES114 </td><td>1332 </td><td>37.03 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00994 </td><td>0.00491 </td></tr><tr><td>107 </td><td><em>Xanthomonas campestris</em> pv. <em>campestris</em> ATCC 33913 </td><td>5076 </td><td>65.07 </td><td>0.196 </td><td>0.719 </td><td>0.00302 </td><td>0.00038 </td></tr><tr><td>108 </td><td><em>Xanthomonas oryzae</em> pv. <em>oryzae</em> KACC 10331 </td><td>4941 </td><td>63.69 </td><td>0.87 </td><td>0.499 </td><td>0.00104 </td><td>0.00065 </td></tr><tr><td>109 </td><td><em>Xylella fastidiosa</em> 9a5c </td><td>2679 </td><td>52.68 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.04727 </td><td>0.05291 </td><td>62.97 </td></tr><tr><td>110 </td><td><em>Xylella fastidiosa</em> Temecula 1 </td><td>2519 </td><td>51.78 </td><td>0.044 </td><td><em>0</em> </td><td>0.00379 </td><td>0.01093 </td><td>6.44 </td></tr><tr><td>111 </td><td><em>Yersinia pestis</em> CO92 </td><td>4653 </td><td>47.64 </td><td>0.649 </td><td><em>0.001</em> </td><td>0.00090 </td><td>0.00520 </td><td rowspan="2">NA </td></tr><tr><td>112 </td><td><em>Yersinia pseudotuberculosis</em> IP32953 </td><td>4744 </td><td>47.61 </td><td>0.969 </td><td><em>0.001</em> </td><td>0.00124 </td><td>0.00496 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>TB, termination bias. Chromosomes of bacteria analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as: KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T) and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting that the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273960" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP021TB1" class="table-wrap-title" data-id="DSP021TB1"><div class="label -label">Table 1</div><div class="caption"><p>ISFDP analysis in bacterial chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Bacterial group </th><th>TB (°) </th></tr></thead><tbody><tr><td>1 </td><td><em>Acinetobacter</em> sp. ADP1 </td><td>3598 </td><td>40.43 </td><td>0.745 </td><td><em>0.006</em> </td><td>0.00068 </td><td>0.00484 </td><td rowspan="5">G-Proteobacteria </td><td>7.07 </td></tr><tr><td>2 </td><td><em>Actinobacillus pleuropneumoniae</em> L20 serotype 5b </td><td>2274 </td><td>41.3 </td><td>0.436 </td><td>0.819 </td><td>0.00187 </td><td>0.00109 </td><td rowspan="4">NA </td></tr><tr><td>3 </td><td><em>Actinobacillus succinogenes</em> 130Z </td><td>2319 </td><td>44.91 </td><td>0.312 </td><td>0.291 </td><td>0.00232 </td><td>0.00291 </td></tr><tr><td>4 </td><td><em>Aeromonas hydrophila</em> subsp. <em>hydrophila</em> ATCC 7966 </td><td>4744 </td><td>61.55 </td><td>0.88 </td><td>0.19 </td><td>0.00141 </td><td>0.00139 </td></tr><tr><td>5 </td><td><em>Aeromonas salmonicida</em> subsp. <em>salmonicida</em> A449 </td><td>4702 </td><td>58.51 </td><td>0.04 </td><td>0.959 </td><td>0.00215 </td><td>0.00073 </td></tr><tr><td>6 </td><td><em>Agrobacterium tumefaciens</em> C58 (circular chromosome) </td><td>2841 </td><td>59.38 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00694 </td><td>0.00967 </td><td>A-Proteobacteria </td><td>7.37 </td></tr><tr><td>7 </td><td><em>Alkaliphilus oremlandii</em> OhILAs </td><td>3123 </td><td>36.26 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00615 </td><td>0.01324 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>8 </td><td><em>Anaeromyxobacter dehalogenans</em> 2CP-C </td><td>5013 </td><td>74.9 </td><td>0.077 </td><td><em>0.001</em> </td><td>0.00476 </td><td>0.00249 </td><td rowspan="2">D-Proteobacteria </td><td>70.57 </td></tr><tr><td>9 </td><td><em>Anaeromyxobacter</em> sp. Fw109-5 </td><td>5277 </td><td>73.53 </td><td>0.712 </td><td><em>0.008</em> </td><td>0.00073 </td><td>0.00216 </td><td>7.48 </td></tr><tr><td>10 </td><td><em>Bacillus anthracis</em> Ames </td><td>5227 </td><td>35.38 </td><td><em>0.004</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00581 </td><td rowspan="6">Firmicutes </td><td>NA </td></tr><tr><td>11 </td><td><em>Bacillus anthracis</em> 'Ames Ancestor' </td><td>5227 </td><td>35.38 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00582 </td><td>7.48 </td></tr><tr><td>12 </td><td><em>Bacillus anthracis</em> Sterne </td><td>5228 </td><td>35.38 </td><td><em>0.008</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00221 </td><td>0.00588 </td><td>7.46 </td></tr><tr><td>13 </td><td><em>Bacillus subtilis</em> </td><td>4214 </td><td>43.52 </td><td>0.219 </td><td>0.234 </td><td>0.00212 </td><td>0.00224 </td><td>13.69 </td></tr><tr><td>14 </td><td><em>Bacillus thuringiensis</em> Al Hakam </td><td>5257 </td><td>35.43 </td><td>0.123 </td><td><em>0.002</em> </td><td>0.00042 </td><td>0.00081 </td><td>NA </td></tr><tr><td>15 </td><td><em>Bacillus thuringiensis</em> serovar konkukian 97-27 </td><td>5237 </td><td>35.41 </td><td>0.015 </td><td><strong>&lt;0.0001</strong> </td><td>0.00194 </td><td>0.00438 </td><td>3.98 </td></tr><tr><td>16 </td><td><em>Bordetella parapertussis</em> 12822 </td><td>4773 </td><td>68.1 </td><td>0.433 </td><td><strong>&lt;0.0001</strong> </td><td>0.00247 </td><td>0.00776 </td><td rowspan="2">B-Proteobacteria </td><td>37.01 </td></tr><tr><td>17 </td><td><em>Bordetella pertussis</em> Tohama 1 </td><td>4086 </td><td>67.72 </td><td>0.861 </td><td><strong>&lt;0.0001</strong> </td><td>0.00022 </td><td>0.00390 </td><td>71.28 </td></tr><tr><td>18 </td><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>9105 </td><td>64.06 </td><td>0.512 </td><td>0.31 </td><td>0.00070 </td><td>0.00038 </td><td rowspan="3">A-Proteobacteria </td><td>7.07 </td></tr><tr><td>19 </td><td><em>Bradyrhizobium</em> sp. BTAi1 </td><td>8264 </td><td>64.92 </td><td>0.381 </td><td>0.01 </td><td>0.00100 </td><td>0.00163 </td><td rowspan="4">NA </td></tr><tr><td>20 </td><td><em>Brucella melitensis</em> 16M </td><td>1177 </td><td>57.35 </td><td>0.472 </td><td><em>0.008</em> </td><td>0.00227 </td><td>0.00312 </td></tr><tr><td>21 </td><td><em>Campylobacter concisus</em> 13826 </td><td>2052 </td><td>39.43 </td><td>0.033 </td><td>0.048 </td><td>0.00038 </td><td>0.00599 </td><td rowspan="5">E-Proteobacteria </td></tr><tr><td>22 </td><td><em>Campylobacter curvus</em> 525.92 </td><td>1971 </td><td>44.54 </td><td>0.028 </td><td>0.752 </td><td>0.00745 </td><td>0.00282 </td></tr><tr><td>23 </td><td><em>Campylobacter jejuni</em> RM1221 </td><td>1777 </td><td>30.31 </td><td>0.574 </td><td>0.23 </td><td>0.00330 </td><td>0.00436 </td><td>8.69 </td></tr><tr><td>24 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> 81116 </td><td>1628 </td><td>30.54 </td><td>0.491 </td><td>0.029 </td><td>0.00250 </td><td>0.00613 </td><td>NA </td></tr><tr><td>25 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> NCTC 11168 </td><td>1641 </td><td>30.55 </td><td>0.067 </td><td>0.132 </td><td>0.00296 </td><td>0.00457 </td><td>10.25 </td></tr><tr><td>26 </td><td>Candidatus <em>Desulfococcus oleovorans</em> Hxd3 </td><td>3944 </td><td>56.17 </td><td>0.258 </td><td>0.133 </td><td>0.00199 </td><td>0.00157 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>27 </td><td><em>Caulobacter crescentus</em> CB15 </td><td>4016 </td><td>67.22 </td><td>0.042 </td><td>0.171 </td><td>0.00396 </td><td>0.00188 </td><td>A-Proteobacteria </td><td>8.56 </td></tr><tr><td>28 </td><td><em>Chlamydia muridarum</em> Nigg </td><td>1072 </td><td>40.34 </td><td>0.221 </td><td>0.853 </td><td>0.00107 </td><td>0.00337 </td><td rowspan="3">Chlamydiae </td><td>1.17 </td></tr><tr><td>29 </td><td><em>Chlamydia trachomatis</em> AHAR-13 </td><td>1044 </td><td>41.31 </td><td>0.228 </td><td>0.284 </td><td>0.00230 </td><td>0.00059 </td><td>1.30 </td></tr><tr><td>30 </td><td><em>Chlamydophila abortus</em> S263 </td><td>1144 </td><td>39.87 </td><td>0.534 </td><td><em>0.002</em> </td><td>0.00065 </td><td>0.00361 </td><td>0.57 </td></tr><tr><td>31 </td><td><em>Coxiella burnetii</em> Dugway 7E9-12 </td><td>2158 </td><td>42.44 </td><td><em>0.004</em> </td><td><em>0.001</em> </td><td>0.00592 </td><td>0.00573 </td><td rowspan="2">G-Proteobacteria </td><td>NA </td></tr><tr><td>32 </td><td><em>Coxiella burnetii</em> RSA 493 </td><td>1995 </td><td>42.66 </td><td>0.014 </td><td>0.467 </td><td>0.00198 </td><td>0.00029 </td><td>31.15 </td></tr><tr><td>33 </td><td><em>Desulfovibrio desulfuricans</em> G20 </td><td>3730 </td><td>57.84 </td><td>0.59 </td><td><em>0.001</em> </td><td>0.00189 </td><td>0.00322 </td><td>Firmicutes </td><td>10.70 </td></tr><tr><td>34 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> DP4 </td><td>3462 </td><td>63.01 </td><td>0.3 </td><td>0.159 </td><td>0.00152 </td><td>0.00106 </td><td rowspan="2">D-Proteobacteria </td><td>NA </td></tr><tr><td>35 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> Hildenborough </td><td>3570 </td><td>63.14 </td><td>0.557 </td><td>0.082 </td><td>0.00143 </td><td>0.00024 </td><td>4.78 </td></tr><tr><td>36 </td><td><em>Enterobacter sakazakii</em> ATCC BAA-894 </td><td>4368 </td><td>56.77 </td><td>0.167 </td><td>0.388 </td><td>0.00359 </td><td>0.00044 </td><td rowspan="10">G-Proteobacteria </td><td>NA </td></tr><tr><td>37 </td><td><em>Enterobacter</em> sp. 638 </td><td>4518 </td><td>52.98 </td><td>0.645 </td><td>0.39 </td><td>0.00169 </td><td>0.00163 </td><td>NA </td></tr><tr><td>38 </td><td><em>Escherichia coli</em> 536 </td><td>4938 </td><td>50.52 </td><td>0.714 </td><td>0.084 </td><td>0.00062 </td><td>0.00328 </td><td>7.40 </td></tr><tr><td>39 </td><td><em>Escherichia coli</em> APEC O1 </td><td>5082 </td><td>50.55 </td><td>0.779 </td><td>0.576 </td><td>0.00032 </td><td>0.00070 </td><td>NA </td></tr><tr><td>40 </td><td><em>Escherichia coli</em> CFT073 </td><td>5231 </td><td>50.48 </td><td>0.112 </td><td>0.92 </td><td>0.00173 </td><td>0.00080 </td><td>5.66 </td></tr><tr><td>41 </td><td><em>Escherichia coli</em> E24377A </td><td>4979 </td><td>50.62 </td><td>0.736 </td><td>0.128 </td><td>0.00205 </td><td>0.00212 </td><td rowspan="2">NA </td></tr><tr><td>42 </td><td><em>Escherichia coli</em> HS </td><td>4643 </td><td>50.82 </td><td>0.328 </td><td>0.469 </td><td>0.00151 </td><td>0.00207 </td></tr><tr><td>43 </td><td><em>Escherichia coli</em> K12 MG1655 </td><td>4639 </td><td>50.79 </td><td>0.732 </td><td>0.587 </td><td>0.00054 </td><td>0.00113 </td><td>4.28 </td></tr><tr><td>44 </td><td><em>Escherichia coli</em> UTI89 </td><td>5065 </td><td>50.6 </td><td>0.51 </td><td>0.237 </td><td>0.00076 </td><td>0.00203 </td><td>3.70 </td></tr><tr><td>45 </td><td><em>Escherichia coli</em> W3110 </td><td>4646 </td><td>50.8 </td><td>0.873 </td><td>0.729 </td><td>0.00073 </td><td>0.00091 </td><td>12.64 </td></tr><tr><td>46 </td><td><em>Frankia alni</em> ACN14A chromosome </td><td>7497 </td><td>72.82 </td><td>0.463 </td><td>0.036 </td><td>0.00141 </td><td>0.00139 </td><td rowspan="2">Actinobacteria </td><td rowspan="5">NA </td></tr><tr><td>47 </td><td><em>Frankia</em> sp. CcI3 </td><td>5433 </td><td>70.08 </td><td>0.808 </td><td>0.662 </td><td>0.00129 </td><td>0.00017 </td></tr><tr><td>48 </td><td><em>Haemophilus influenzae</em> 86-028NP </td><td>1914 </td><td>38.16 </td><td>0.886 </td><td>0.654 </td><td>0.00089 </td><td>0.00044 </td><td rowspan="4">G-Proteobacteria </td></tr><tr><td>49 </td><td><em>Haemophilus influenzae</em> PittEE </td><td>1813 </td><td>38.04 </td><td>0.544 </td><td>0.038 </td><td>0.00054 </td><td>0.00317 </td></tr><tr><td>50 </td><td><em>Haemophilus influenzae</em> PittGG </td><td>1887 </td><td>38.01 </td><td>0.125 </td><td><strong>&lt;0.0001</strong> </td><td>0.00005 </td><td>0.01016 </td></tr><tr><td>51 </td><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>1830 </td><td>38.15 </td><td>0.154 </td><td><em>0.004</em> </td><td>0.00298 </td><td>0.00472 </td><td>46.61 </td></tr><tr><td>52 </td><td><em>Helicobacter acinonychis</em> Sheeba </td><td>1553 </td><td>38.18 </td><td><em>0</em> </td><td>0.596 </td><td>0.00869 </td><td>0.00164 </td><td rowspan="3">E-Proteobacteria </td><td>NA </td></tr><tr><td>53 </td><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>1799 </td><td>35.93 </td><td>0.161 </td><td><strong>&lt;0.0001</strong> </td><td>0.00499 </td><td>0.01518 </td><td>46.54 </td></tr><tr><td>54 </td><td><em>Helicobacter pylori</em> J99 </td><td>1643 </td><td>39.19 </td><td>0.246 </td><td>0.256 </td><td>0.00259 </td><td>0.00510 </td><td>10.97 </td></tr><tr><td>55 </td><td><em>Lactobacillus acidophilus</em> NCFM </td><td>1993 </td><td>34.72 </td><td>0.382 </td><td><strong>&lt;0.0001</strong> </td><td>0.00066 </td><td>0.01644 </td><td rowspan="6">Firmicutes </td><td>19.54 </td></tr><tr><td>56 </td><td><em>Lactobacillus brevis</em> ATCC 367 </td><td>2291 </td><td>46.22 </td><td>0.023 </td><td><strong>&lt;0.0001</strong> </td><td>0.00271 </td><td>0.02882 </td><td rowspan="6">NA </td></tr><tr><td>57 </td><td><em>Lactobacillus delbrueckii</em> subsp. <em>bulgaricus</em> ATCC BAA-365 </td><td>1856 </td><td>49.69 </td><td>0.491 </td><td>0.264 </td><td>0.00201 </td><td>0.00087 </td></tr><tr><td>58 </td><td><em>Lactobacillus reuteri</em> F275 </td><td>1999 </td><td>38.87 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00122 </td><td>0.01040 </td></tr><tr><td>59 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363 </td><td>2529 </td><td>35.75 </td><td>0.233 </td><td>0.056 </td><td>0.00352 </td><td>0.00524 </td></tr><tr><td>60 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> SK11 </td><td>2438 </td><td>35.86 </td><td>0.399 </td><td>0.521 </td><td>0.00147 </td><td>0.00136 </td></tr><tr><td>61 </td><td><em>Magnetococcus</em> sp. MC-1 </td><td>4719 </td><td>54.17 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00490 </td><td>0.01198 </td><td>Magnetococcus </td></tr><tr><td>62 </td><td><em>Magnetospirillum magneticum</em> AMB-1 </td><td>4967 </td><td>65.09 </td><td>0.031 </td><td><strong>&lt;0.0001</strong> </td><td>0.00339 </td><td>0.00288 </td><td>A-Proteobacteria </td><td>2.14 </td></tr><tr><td>63 </td><td><em>Methylobacillus flagellatus</em> KT </td><td>2971 </td><td>55.72 </td><td>0.03 </td><td>0.916 </td><td>0.00226 </td><td>0.00135 </td><td>B-Proteobacteria </td><td>10.57 </td></tr><tr><td>64 </td><td><em>Methylococcus capsulatus</em> Bath </td><td>3304 </td><td>63.59 </td><td>0.145 </td><td><em>0.004</em> </td><td>0.00150 </td><td>0.00287 </td><td>G-Proteobacteria </td><td>NA </td></tr><tr><td>65 </td><td><em>Mycobacterium leprae</em> TN </td><td>3268 </td><td>57.8 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00378 </td><td>0.00609 </td><td rowspan="4">Actinobacteria </td><td>7.04 </td></tr><tr><td>66 </td><td><em>Mycobacterium</em> sp. KMS </td><td>5737 </td><td>68.44 </td><td>0.389 </td><td>0.478 </td><td>0.00030 </td><td>0.00060 </td><td rowspan="3">NA </td></tr><tr><td>67 </td><td><em>Mycobacterium tuberculosis</em> F11 </td><td>4424 </td><td>65.62 </td><td>0.366 </td><td><em>0.007</em> </td><td>0.00006 </td><td>0.00198 </td></tr><tr><td>68 </td><td><em>Mycobacterium ulcerans</em> Agy99 </td><td>5631 </td><td>65.47 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00433 </td><td>0.00374 </td></tr><tr><td>69 </td><td><em>Mycoplasma gallisepticum</em> R </td><td>996 </td><td>31.45 </td><td>0.18 </td><td>0.615 </td><td>0.00626 </td><td>0.00021 </td><td rowspan="4">Tenericutes </td><td>9.32 </td></tr><tr><td>70 </td><td><em>Mycoplasma genitalium</em> G37 </td><td>580 </td><td>31.69 </td><td><em>0</em> </td><td>0.148 </td><td>0.01219 </td><td>0.00433 </td><td>3.75 </td></tr><tr><td>71 </td><td><em>Mycoplasma hyopneumoniae</em> J </td><td>897 </td><td>28.52 </td><td>0.033 </td><td>0.599 </td><td>0.01020 </td><td>0.00067 </td><td>NA </td></tr><tr><td>72 </td><td><em>Mycoplasma pneumoniae</em> M129 </td><td>816 </td><td>40.01 </td><td><em>0.001</em> </td><td>0.115 </td><td>0.01767 </td><td>0.00243 </td><td>16.23 </td></tr><tr><td>73 </td><td><em>Neisseria gonorrhoeae</em> FA 1090 </td><td>2153 </td><td>52.69 </td><td>0.07 </td><td>0.033 </td><td>0.00601 </td><td>0.00144 </td><td rowspan="2">B-Proteobacteria </td><td>9.20 </td></tr><tr><td>74 </td><td><em>Neisseria meningitidis</em> MC58 </td><td>2273 </td><td>51.52 </td><td>0.695 </td><td><em>0.004</em> </td><td>0.00135 </td><td>0.00806 </td><td rowspan="2">NA </td></tr><tr><td>75 </td><td><em>Nitrobacter hamburgensis</em> X14 </td><td>4406 </td><td>61.72 </td><td>0.332 </td><td>0.53 </td><td>0.00112 </td><td>0.00041 </td><td rowspan="2">A-Proteobacteria </td></tr><tr><td>76 </td><td><em>Nitrobacter winogradskyi</em> Nb-255 </td><td>3402 </td><td>62.05 </td><td>0.011 </td><td><strong>&lt;0.0001</strong> </td><td>0.00323 </td><td>0.00294 </td><td>37.15 </td></tr><tr><td>77 </td><td><em>Nitrosococcus oceani</em> ATCC 19707 </td><td>3481 </td><td>50.32 </td><td>0.02 </td><td>0.056 </td><td>0.00530 </td><td>0.00243 </td><td>G-Proteobacteria </td><td>8.39 </td></tr><tr><td>78 </td><td><em>Nitrosomonas eutropha</em> C91 </td><td>2661 </td><td>48.49 </td><td>0.992 </td><td>0.318 </td><td>0.00043 </td><td>0.00162 </td><td>B-Proteobacteria </td><td rowspan="2">NA </td></tr><tr><td>79 </td><td><em>Nostoc</em> sp. PCC 7120 </td><td>6413 </td><td>41.35 </td><td>0.134 </td><td>0.857 </td><td>0.00129 </td><td>0.00162 </td><td>Cyanobacteria </td></tr><tr><td>80 </td><td><em>Pseudomonas entomophila</em> L48 chromosome </td><td>5888 </td><td>64.16 </td><td>0.657 </td><td>0.251 </td><td>0.00078 </td><td>0.00173 </td><td rowspan="3">G-Proteobacteria </td><td>1.99 </td></tr><tr><td>81 </td><td><em>Pseudomonas fluorescens</em> PfO-1 </td><td>6438 </td><td>60.52 </td><td><em>0.003</em> </td><td>0.028 </td><td>0.00443 </td><td>0.00222 </td><td>3.18 </td></tr><tr><td>82 </td><td><em>Pseudomonas putida</em> F1 </td><td>5959 </td><td>61.86 </td><td>0.602 </td><td>0.013 </td><td>0.00113 </td><td>0.00187 </td><td>36.81 </td></tr><tr><td>83 </td><td><em>Ralstonia eutropha</em> H16 </td><td>2912 </td><td>66.78 </td><td>0.238 </td><td>0.47 </td><td>0.00483 </td><td>0.00023 </td><td rowspan="2">B-Proteobacteria </td><td>NA </td></tr><tr><td>84 </td><td><em>Ralstonia solanacearum</em> GMI1000 chromosome </td><td>3716 </td><td>67.04 </td><td>0.056 </td><td><strong>&lt;0.0001</strong> </td><td>0.00636 </td><td>0.00581 </td><td>22.40 </td></tr><tr><td>85 </td><td><em>Rhizobium etli</em> CFN 42 </td><td>4381 </td><td>61.27 </td><td>0.107 </td><td><strong>&lt;0.0001</strong> </td><td>0.00175 </td><td>0.01177 </td><td rowspan="6">A-Proteobacteria </td><td>17.65 </td></tr><tr><td>86 </td><td><em>Rhizobium leguminosarum</em> bv. <em>viciae</em> 3841 </td><td>5057 </td><td>61.09 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00363 </td><td>0.01196 </td><td>NA </td></tr><tr><td>87 </td><td><em>Rickettsia bellii</em> RML369-C </td><td>1522 </td><td>31.65 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00859 </td><td>0.01514 </td><td>26.08 </td></tr><tr><td>88 </td><td><em>Rickettsia conorii</em> Malish 7 </td><td>1268 </td><td>32.44 </td><td>0.584 </td><td>0.052 </td><td>0.00294 </td><td>0.00634 </td><td>16.28 </td></tr><tr><td>89 </td><td><em>Rickettsia rickettsii</em> 'Sheila Smith' </td><td>1257 </td><td>32.47 </td><td>0.575 </td><td><em>0.002</em> </td><td>0.00182 </td><td>0.00767 </td><td>NA </td></tr><tr><td>90 </td><td><em>Rickettsia typhi</em> Wilmington </td><td>1111 </td><td>28.92 </td><td>0.919 </td><td><em>0.007</em> </td><td>0.00020 </td><td>0.01395 </td><td>26.15 </td></tr><tr><td>91 </td><td><em>Salmonella enterica</em> subsp. <em>enterica</em> serovar Typhi CT18 </td><td>4809 </td><td>52.09 </td><td>0.267 </td><td>0.043 </td><td>0.00151 </td><td>0.00152 </td><td rowspan="4">G-Proteobacteria </td><td>9.85 </td></tr><tr><td>92 </td><td><em>Salmonella typhimurium</em> LT2 </td><td>4857 </td><td>52.22 </td><td>0.89 </td><td>0.585 </td><td>0.00043 </td><td>0.00008 </td><td>3.58 </td></tr><tr><td>93 </td><td><em>Shigella boydii</em> Sb227 </td><td>4519 </td><td>51.21 </td><td>0.571 </td><td><em>0.001</em> </td><td>0.00022 </td><td>0.00249 </td><td>11.05 </td></tr><tr><td>94 </td><td><em>Shigella flexneri</em> 58401 </td><td>4574 </td><td>50.92 </td><td>0.48 </td><td>0.268 </td><td>0.00147 </td><td>0.00214 </td><td>NA </td></tr><tr><td>95 </td><td><em>Staphylococcus aureus</em> RF122 </td><td>2742 </td><td>32.78 </td><td>0.788 </td><td>0.427 </td><td>0.00130 </td><td>0.00247 </td><td rowspan="6">Firmicutes </td><td>0.10 </td></tr><tr><td>96 </td><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>2499 </td><td>32.1 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.01246 </td><td>0.01087 </td><td>21.12 </td></tr><tr><td>97 </td><td><em>Staphylococcus haemolyticus</em> JCSC1435 </td><td>2685 </td><td>32.79 </td><td><em>0.001</em> </td><td><em>0</em> </td><td>0.00584 </td><td>0.00643 </td><td rowspan="2">NA </td></tr><tr><td>98 </td><td><em>Streptococcus mutans</em> UA159 </td><td>2030 </td><td>36.83 </td><td>0.111 </td><td>0.046 </td><td>0.00403 </td><td>0.00679 </td></tr><tr><td>99 </td><td><em>Streptococcus pyogenes</em> MGAS2096 </td><td>1860 </td><td>38.73 </td><td>0.619 </td><td>0.15 </td><td>0.00133 </td><td>0.00154 </td><td>3.71 </td></tr><tr><td>100 </td><td><em>Streptococcus thermophilus</em> CNRZ1066 </td><td>1796 </td><td>39.08 </td><td>0.05 </td><td>0.863 </td><td>0.00537 </td><td>0.00459 </td><td>2.63 </td></tr><tr><td>101 </td><td><em>Streptomyces coelicolor</em> A3(2) </td><td>8667 </td><td>72.12 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00394 </td><td>0.00134 </td><td>Actinobacteria </td><td>NA </td></tr><tr><td>102 </td><td><em>Thermotoga maritima</em> MSB8 </td><td>1860 </td><td>46.25 </td><td>0.171 </td><td><strong>&lt;0.0001</strong> </td><td>0.00344 </td><td>0.01548 </td><td rowspan="2">Thermotogae </td><td>39.15 </td></tr><tr><td>103 </td><td><em>Thermotoga petrophila</em> RKU-1 </td><td>1824 </td><td>46.09 </td><td>0.733 </td><td><strong>&lt;0.0001</strong> </td><td>0.00013 </td><td>0.01687 </td><td>NA </td></tr><tr><td>104 </td><td><em>Thiobacillus denitrificans</em> ATCC 25259 </td><td>2909 </td><td>66.07 </td><td>0.962 </td><td>0.086 </td><td>0.00027 </td><td>0.00059 </td><td>B-Proteobacteria </td><td>5.70 </td></tr><tr><td>105 </td><td><em>Vibrio cholerae</em> O395 </td><td>3024 </td><td>47.78 </td><td><strong>&lt;0.0001</strong> </td><td>0.069 </td><td>0.00514 </td><td>0.00105 </td><td rowspan="8">G-Proteobacteria </td><td rowspan="4">NA </td></tr><tr><td>106 </td><td><em>Vibrio fischeri</em> ES114 </td><td>1332 </td><td>37.03 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00994 </td><td>0.00491 </td></tr><tr><td>107 </td><td><em>Xanthomonas campestris</em> pv. <em>campestris</em> ATCC 33913 </td><td>5076 </td><td>65.07 </td><td>0.196 </td><td>0.719 </td><td>0.00302 </td><td>0.00038 </td></tr><tr><td>108 </td><td><em>Xanthomonas oryzae</em> pv. <em>oryzae</em> KACC 10331 </td><td>4941 </td><td>63.69 </td><td>0.87 </td><td>0.499 </td><td>0.00104 </td><td>0.00065 </td></tr><tr><td>109 </td><td><em>Xylella fastidiosa</em> 9a5c </td><td>2679 </td><td>52.68 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.04727 </td><td>0.05291 </td><td>62.97 </td></tr><tr><td>110 </td><td><em>Xylella fastidiosa</em> Temecula 1 </td><td>2519 </td><td>51.78 </td><td>0.044 </td><td><em>0</em> </td><td>0.00379 </td><td>0.01093 </td><td>6.44 </td></tr><tr><td>111 </td><td><em>Yersinia pestis</em> CO92 </td><td>4653 </td><td>47.64 </td><td>0.649 </td><td><em>0.001</em> </td><td>0.00090 </td><td>0.00520 </td><td rowspan="2">NA </td></tr><tr><td>112 </td><td><em>Yersinia pseudotuberculosis</em> IP32953 </td><td>4744 </td><td>47.61 </td><td>0.969 </td><td><em>0.001</em> </td><td>0.00124 </td><td>0.00496 </td></tr></tbody></table></div><div content-id="DSP021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Bacterial group </th><th>TB (°) </th></tr></thead><tbody><tr><td>1 </td><td><em>Acinetobacter</em> sp. ADP1 </td><td>3598 </td><td>40.43 </td><td>0.745 </td><td><em>0.006</em> </td><td>0.00068 </td><td>0.00484 </td><td rowspan="5">G-Proteobacteria </td><td>7.07 </td></tr><tr><td>2 </td><td><em>Actinobacillus pleuropneumoniae</em> L20 serotype 5b </td><td>2274 </td><td>41.3 </td><td>0.436 </td><td>0.819 </td><td>0.00187 </td><td>0.00109 </td><td rowspan="4">NA </td></tr><tr><td>3 </td><td><em>Actinobacillus succinogenes</em> 130Z </td><td>2319 </td><td>44.91 </td><td>0.312 </td><td>0.291 </td><td>0.00232 </td><td>0.00291 </td></tr><tr><td>4 </td><td><em>Aeromonas hydrophila</em> subsp. <em>hydrophila</em> ATCC 7966 </td><td>4744 </td><td>61.55 </td><td>0.88 </td><td>0.19 </td><td>0.00141 </td><td>0.00139 </td></tr><tr><td>5 </td><td><em>Aeromonas salmonicida</em> subsp. <em>salmonicida</em> A449 </td><td>4702 </td><td>58.51 </td><td>0.04 </td><td>0.959 </td><td>0.00215 </td><td>0.00073 </td></tr><tr><td>6 </td><td><em>Agrobacterium tumefaciens</em> C58 (circular chromosome) </td><td>2841 </td><td>59.38 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00694 </td><td>0.00967 </td><td>A-Proteobacteria </td><td>7.37 </td></tr><tr><td>7 </td><td><em>Alkaliphilus oremlandii</em> OhILAs </td><td>3123 </td><td>36.26 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00615 </td><td>0.01324 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>8 </td><td><em>Anaeromyxobacter dehalogenans</em> 2CP-C </td><td>5013 </td><td>74.9 </td><td>0.077 </td><td><em>0.001</em> </td><td>0.00476 </td><td>0.00249 </td><td rowspan="2">D-Proteobacteria </td><td>70.57 </td></tr><tr><td>9 </td><td><em>Anaeromyxobacter</em> sp. Fw109-5 </td><td>5277 </td><td>73.53 </td><td>0.712 </td><td><em>0.008</em> </td><td>0.00073 </td><td>0.00216 </td><td>7.48 </td></tr><tr><td>10 </td><td><em>Bacillus anthracis</em> Ames </td><td>5227 </td><td>35.38 </td><td><em>0.004</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00581 </td><td rowspan="6">Firmicutes </td><td>NA </td></tr><tr><td>11 </td><td><em>Bacillus anthracis</em> 'Ames Ancestor' </td><td>5227 </td><td>35.38 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00215 </td><td>0.00582 </td><td>7.48 </td></tr><tr><td>12 </td><td><em>Bacillus anthracis</em> Sterne </td><td>5228 </td><td>35.38 </td><td><em>0.008</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00221 </td><td>0.00588 </td><td>7.46 </td></tr><tr><td>13 </td><td><em>Bacillus subtilis</em> </td><td>4214 </td><td>43.52 </td><td>0.219 </td><td>0.234 </td><td>0.00212 </td><td>0.00224 </td><td>13.69 </td></tr><tr><td>14 </td><td><em>Bacillus thuringiensis</em> Al Hakam </td><td>5257 </td><td>35.43 </td><td>0.123 </td><td><em>0.002</em> </td><td>0.00042 </td><td>0.00081 </td><td>NA </td></tr><tr><td>15 </td><td><em>Bacillus thuringiensis</em> serovar konkukian 97-27 </td><td>5237 </td><td>35.41 </td><td>0.015 </td><td><strong>&lt;0.0001</strong> </td><td>0.00194 </td><td>0.00438 </td><td>3.98 </td></tr><tr><td>16 </td><td><em>Bordetella parapertussis</em> 12822 </td><td>4773 </td><td>68.1 </td><td>0.433 </td><td><strong>&lt;0.0001</strong> </td><td>0.00247 </td><td>0.00776 </td><td rowspan="2">B-Proteobacteria </td><td>37.01 </td></tr><tr><td>17 </td><td><em>Bordetella pertussis</em> Tohama 1 </td><td>4086 </td><td>67.72 </td><td>0.861 </td><td><strong>&lt;0.0001</strong> </td><td>0.00022 </td><td>0.00390 </td><td>71.28 </td></tr><tr><td>18 </td><td><em>Bradyrhizobium japonicum</em> USDA 110 </td><td>9105 </td><td>64.06 </td><td>0.512 </td><td>0.31 </td><td>0.00070 </td><td>0.00038 </td><td rowspan="3">A-Proteobacteria </td><td>7.07 </td></tr><tr><td>19 </td><td><em>Bradyrhizobium</em> sp. BTAi1 </td><td>8264 </td><td>64.92 </td><td>0.381 </td><td>0.01 </td><td>0.00100 </td><td>0.00163 </td><td rowspan="4">NA </td></tr><tr><td>20 </td><td><em>Brucella melitensis</em> 16M </td><td>1177 </td><td>57.35 </td><td>0.472 </td><td><em>0.008</em> </td><td>0.00227 </td><td>0.00312 </td></tr><tr><td>21 </td><td><em>Campylobacter concisus</em> 13826 </td><td>2052 </td><td>39.43 </td><td>0.033 </td><td>0.048 </td><td>0.00038 </td><td>0.00599 </td><td rowspan="5">E-Proteobacteria </td></tr><tr><td>22 </td><td><em>Campylobacter curvus</em> 525.92 </td><td>1971 </td><td>44.54 </td><td>0.028 </td><td>0.752 </td><td>0.00745 </td><td>0.00282 </td></tr><tr><td>23 </td><td><em>Campylobacter jejuni</em> RM1221 </td><td>1777 </td><td>30.31 </td><td>0.574 </td><td>0.23 </td><td>0.00330 </td><td>0.00436 </td><td>8.69 </td></tr><tr><td>24 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> 81116 </td><td>1628 </td><td>30.54 </td><td>0.491 </td><td>0.029 </td><td>0.00250 </td><td>0.00613 </td><td>NA </td></tr><tr><td>25 </td><td><em>Campylobacter jejuni</em> subsp. <em>jejuni</em> NCTC 11168 </td><td>1641 </td><td>30.55 </td><td>0.067 </td><td>0.132 </td><td>0.00296 </td><td>0.00457 </td><td>10.25 </td></tr><tr><td>26 </td><td>Candidatus <em>Desulfococcus oleovorans</em> Hxd3 </td><td>3944 </td><td>56.17 </td><td>0.258 </td><td>0.133 </td><td>0.00199 </td><td>0.00157 </td><td>Firmicutes </td><td>NA </td></tr><tr><td>27 </td><td><em>Caulobacter crescentus</em> CB15 </td><td>4016 </td><td>67.22 </td><td>0.042 </td><td>0.171 </td><td>0.00396 </td><td>0.00188 </td><td>A-Proteobacteria </td><td>8.56 </td></tr><tr><td>28 </td><td><em>Chlamydia muridarum</em> Nigg </td><td>1072 </td><td>40.34 </td><td>0.221 </td><td>0.853 </td><td>0.00107 </td><td>0.00337 </td><td rowspan="3">Chlamydiae </td><td>1.17 </td></tr><tr><td>29 </td><td><em>Chlamydia trachomatis</em> AHAR-13 </td><td>1044 </td><td>41.31 </td><td>0.228 </td><td>0.284 </td><td>0.00230 </td><td>0.00059 </td><td>1.30 </td></tr><tr><td>30 </td><td><em>Chlamydophila abortus</em> S263 </td><td>1144 </td><td>39.87 </td><td>0.534 </td><td><em>0.002</em> </td><td>0.00065 </td><td>0.00361 </td><td>0.57 </td></tr><tr><td>31 </td><td><em>Coxiella burnetii</em> Dugway 7E9-12 </td><td>2158 </td><td>42.44 </td><td><em>0.004</em> </td><td><em>0.001</em> </td><td>0.00592 </td><td>0.00573 </td><td rowspan="2">G-Proteobacteria </td><td>NA </td></tr><tr><td>32 </td><td><em>Coxiella burnetii</em> RSA 493 </td><td>1995 </td><td>42.66 </td><td>0.014 </td><td>0.467 </td><td>0.00198 </td><td>0.00029 </td><td>31.15 </td></tr><tr><td>33 </td><td><em>Desulfovibrio desulfuricans</em> G20 </td><td>3730 </td><td>57.84 </td><td>0.59 </td><td><em>0.001</em> </td><td>0.00189 </td><td>0.00322 </td><td>Firmicutes </td><td>10.70 </td></tr><tr><td>34 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> DP4 </td><td>3462 </td><td>63.01 </td><td>0.3 </td><td>0.159 </td><td>0.00152 </td><td>0.00106 </td><td rowspan="2">D-Proteobacteria </td><td>NA </td></tr><tr><td>35 </td><td><em>Desulfovibrio vulgaris</em> subsp. <em>vulgaris</em> Hildenborough </td><td>3570 </td><td>63.14 </td><td>0.557 </td><td>0.082 </td><td>0.00143 </td><td>0.00024 </td><td>4.78 </td></tr><tr><td>36 </td><td><em>Enterobacter sakazakii</em> ATCC BAA-894 </td><td>4368 </td><td>56.77 </td><td>0.167 </td><td>0.388 </td><td>0.00359 </td><td>0.00044 </td><td rowspan="10">G-Proteobacteria </td><td>NA </td></tr><tr><td>37 </td><td><em>Enterobacter</em> sp. 638 </td><td>4518 </td><td>52.98 </td><td>0.645 </td><td>0.39 </td><td>0.00169 </td><td>0.00163 </td><td>NA </td></tr><tr><td>38 </td><td><em>Escherichia coli</em> 536 </td><td>4938 </td><td>50.52 </td><td>0.714 </td><td>0.084 </td><td>0.00062 </td><td>0.00328 </td><td>7.40 </td></tr><tr><td>39 </td><td><em>Escherichia coli</em> APEC O1 </td><td>5082 </td><td>50.55 </td><td>0.779 </td><td>0.576 </td><td>0.00032 </td><td>0.00070 </td><td>NA </td></tr><tr><td>40 </td><td><em>Escherichia coli</em> CFT073 </td><td>5231 </td><td>50.48 </td><td>0.112 </td><td>0.92 </td><td>0.00173 </td><td>0.00080 </td><td>5.66 </td></tr><tr><td>41 </td><td><em>Escherichia coli</em> E24377A </td><td>4979 </td><td>50.62 </td><td>0.736 </td><td>0.128 </td><td>0.00205 </td><td>0.00212 </td><td rowspan="2">NA </td></tr><tr><td>42 </td><td><em>Escherichia coli</em> HS </td><td>4643 </td><td>50.82 </td><td>0.328 </td><td>0.469 </td><td>0.00151 </td><td>0.00207 </td></tr><tr><td>43 </td><td><em>Escherichia coli</em> K12 MG1655 </td><td>4639 </td><td>50.79 </td><td>0.732 </td><td>0.587 </td><td>0.00054 </td><td>0.00113 </td><td>4.28 </td></tr><tr><td>44 </td><td><em>Escherichia coli</em> UTI89 </td><td>5065 </td><td>50.6 </td><td>0.51 </td><td>0.237 </td><td>0.00076 </td><td>0.00203 </td><td>3.70 </td></tr><tr><td>45 </td><td><em>Escherichia coli</em> W3110 </td><td>4646 </td><td>50.8 </td><td>0.873 </td><td>0.729 </td><td>0.00073 </td><td>0.00091 </td><td>12.64 </td></tr><tr><td>46 </td><td><em>Frankia alni</em> ACN14A chromosome </td><td>7497 </td><td>72.82 </td><td>0.463 </td><td>0.036 </td><td>0.00141 </td><td>0.00139 </td><td rowspan="2">Actinobacteria </td><td rowspan="5">NA </td></tr><tr><td>47 </td><td><em>Frankia</em> sp. CcI3 </td><td>5433 </td><td>70.08 </td><td>0.808 </td><td>0.662 </td><td>0.00129 </td><td>0.00017 </td></tr><tr><td>48 </td><td><em>Haemophilus influenzae</em> 86-028NP </td><td>1914 </td><td>38.16 </td><td>0.886 </td><td>0.654 </td><td>0.00089 </td><td>0.00044 </td><td rowspan="4">G-Proteobacteria </td></tr><tr><td>49 </td><td><em>Haemophilus influenzae</em> PittEE </td><td>1813 </td><td>38.04 </td><td>0.544 </td><td>0.038 </td><td>0.00054 </td><td>0.00317 </td></tr><tr><td>50 </td><td><em>Haemophilus influenzae</em> PittGG </td><td>1887 </td><td>38.01 </td><td>0.125 </td><td><strong>&lt;0.0001</strong> </td><td>0.00005 </td><td>0.01016 </td></tr><tr><td>51 </td><td><em>Haemophilus influenzae</em> Rd KW20 </td><td>1830 </td><td>38.15 </td><td>0.154 </td><td><em>0.004</em> </td><td>0.00298 </td><td>0.00472 </td><td>46.61 </td></tr><tr><td>52 </td><td><em>Helicobacter acinonychis</em> Sheeba </td><td>1553 </td><td>38.18 </td><td><em>0</em> </td><td>0.596 </td><td>0.00869 </td><td>0.00164 </td><td rowspan="3">E-Proteobacteria </td><td>NA </td></tr><tr><td>53 </td><td><em>Helicobacter hepaticus</em> ATCC 51449 </td><td>1799 </td><td>35.93 </td><td>0.161 </td><td><strong>&lt;0.0001</strong> </td><td>0.00499 </td><td>0.01518 </td><td>46.54 </td></tr><tr><td>54 </td><td><em>Helicobacter pylori</em> J99 </td><td>1643 </td><td>39.19 </td><td>0.246 </td><td>0.256 </td><td>0.00259 </td><td>0.00510 </td><td>10.97 </td></tr><tr><td>55 </td><td><em>Lactobacillus acidophilus</em> NCFM </td><td>1993 </td><td>34.72 </td><td>0.382 </td><td><strong>&lt;0.0001</strong> </td><td>0.00066 </td><td>0.01644 </td><td rowspan="6">Firmicutes </td><td>19.54 </td></tr><tr><td>56 </td><td><em>Lactobacillus brevis</em> ATCC 367 </td><td>2291 </td><td>46.22 </td><td>0.023 </td><td><strong>&lt;0.0001</strong> </td><td>0.00271 </td><td>0.02882 </td><td rowspan="6">NA </td></tr><tr><td>57 </td><td><em>Lactobacillus delbrueckii</em> subsp. <em>bulgaricus</em> ATCC BAA-365 </td><td>1856 </td><td>49.69 </td><td>0.491 </td><td>0.264 </td><td>0.00201 </td><td>0.00087 </td></tr><tr><td>58 </td><td><em>Lactobacillus reuteri</em> F275 </td><td>1999 </td><td>38.87 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00122 </td><td>0.01040 </td></tr><tr><td>59 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363 </td><td>2529 </td><td>35.75 </td><td>0.233 </td><td>0.056 </td><td>0.00352 </td><td>0.00524 </td></tr><tr><td>60 </td><td><em>Lactococcus lactis</em> subsp. <em>cremoris</em> SK11 </td><td>2438 </td><td>35.86 </td><td>0.399 </td><td>0.521 </td><td>0.00147 </td><td>0.00136 </td></tr><tr><td>61 </td><td><em>Magnetococcus</em> sp. MC-1 </td><td>4719 </td><td>54.17 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00490 </td><td>0.01198 </td><td>Magnetococcus </td></tr><tr><td>62 </td><td><em>Magnetospirillum magneticum</em> AMB-1 </td><td>4967 </td><td>65.09 </td><td>0.031 </td><td><strong>&lt;0.0001</strong> </td><td>0.00339 </td><td>0.00288 </td><td>A-Proteobacteria </td><td>2.14 </td></tr><tr><td>63 </td><td><em>Methylobacillus flagellatus</em> KT </td><td>2971 </td><td>55.72 </td><td>0.03 </td><td>0.916 </td><td>0.00226 </td><td>0.00135 </td><td>B-Proteobacteria </td><td>10.57 </td></tr><tr><td>64 </td><td><em>Methylococcus capsulatus</em> Bath </td><td>3304 </td><td>63.59 </td><td>0.145 </td><td><em>0.004</em> </td><td>0.00150 </td><td>0.00287 </td><td>G-Proteobacteria </td><td>NA </td></tr><tr><td>65 </td><td><em>Mycobacterium leprae</em> TN </td><td>3268 </td><td>57.8 </td><td><em>0.003</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00378 </td><td>0.00609 </td><td rowspan="4">Actinobacteria </td><td>7.04 </td></tr><tr><td>66 </td><td><em>Mycobacterium</em> sp. KMS </td><td>5737 </td><td>68.44 </td><td>0.389 </td><td>0.478 </td><td>0.00030 </td><td>0.00060 </td><td rowspan="3">NA </td></tr><tr><td>67 </td><td><em>Mycobacterium tuberculosis</em> F11 </td><td>4424 </td><td>65.62 </td><td>0.366 </td><td><em>0.007</em> </td><td>0.00006 </td><td>0.00198 </td></tr><tr><td>68 </td><td><em>Mycobacterium ulcerans</em> Agy99 </td><td>5631 </td><td>65.47 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.00433 </td><td>0.00374 </td></tr><tr><td>69 </td><td><em>Mycoplasma gallisepticum</em> R </td><td>996 </td><td>31.45 </td><td>0.18 </td><td>0.615 </td><td>0.00626 </td><td>0.00021 </td><td rowspan="4">Tenericutes </td><td>9.32 </td></tr><tr><td>70 </td><td><em>Mycoplasma genitalium</em> G37 </td><td>580 </td><td>31.69 </td><td><em>0</em> </td><td>0.148 </td><td>0.01219 </td><td>0.00433 </td><td>3.75 </td></tr><tr><td>71 </td><td><em>Mycoplasma hyopneumoniae</em> J </td><td>897 </td><td>28.52 </td><td>0.033 </td><td>0.599 </td><td>0.01020 </td><td>0.00067 </td><td>NA </td></tr><tr><td>72 </td><td><em>Mycoplasma pneumoniae</em> M129 </td><td>816 </td><td>40.01 </td><td><em>0.001</em> </td><td>0.115 </td><td>0.01767 </td><td>0.00243 </td><td>16.23 </td></tr><tr><td>73 </td><td><em>Neisseria gonorrhoeae</em> FA 1090 </td><td>2153 </td><td>52.69 </td><td>0.07 </td><td>0.033 </td><td>0.00601 </td><td>0.00144 </td><td rowspan="2">B-Proteobacteria </td><td>9.20 </td></tr><tr><td>74 </td><td><em>Neisseria meningitidis</em> MC58 </td><td>2273 </td><td>51.52 </td><td>0.695 </td><td><em>0.004</em> </td><td>0.00135 </td><td>0.00806 </td><td rowspan="2">NA </td></tr><tr><td>75 </td><td><em>Nitrobacter hamburgensis</em> X14 </td><td>4406 </td><td>61.72 </td><td>0.332 </td><td>0.53 </td><td>0.00112 </td><td>0.00041 </td><td rowspan="2">A-Proteobacteria </td></tr><tr><td>76 </td><td><em>Nitrobacter winogradskyi</em> Nb-255 </td><td>3402 </td><td>62.05 </td><td>0.011 </td><td><strong>&lt;0.0001</strong> </td><td>0.00323 </td><td>0.00294 </td><td>37.15 </td></tr><tr><td>77 </td><td><em>Nitrosococcus oceani</em> ATCC 19707 </td><td>3481 </td><td>50.32 </td><td>0.02 </td><td>0.056 </td><td>0.00530 </td><td>0.00243 </td><td>G-Proteobacteria </td><td>8.39 </td></tr><tr><td>78 </td><td><em>Nitrosomonas eutropha</em> C91 </td><td>2661 </td><td>48.49 </td><td>0.992 </td><td>0.318 </td><td>0.00043 </td><td>0.00162 </td><td>B-Proteobacteria </td><td rowspan="2">NA </td></tr><tr><td>79 </td><td><em>Nostoc</em> sp. PCC 7120 </td><td>6413 </td><td>41.35 </td><td>0.134 </td><td>0.857 </td><td>0.00129 </td><td>0.00162 </td><td>Cyanobacteria </td></tr><tr><td>80 </td><td><em>Pseudomonas entomophila</em> L48 chromosome </td><td>5888 </td><td>64.16 </td><td>0.657 </td><td>0.251 </td><td>0.00078 </td><td>0.00173 </td><td rowspan="3">G-Proteobacteria </td><td>1.99 </td></tr><tr><td>81 </td><td><em>Pseudomonas fluorescens</em> PfO-1 </td><td>6438 </td><td>60.52 </td><td><em>0.003</em> </td><td>0.028 </td><td>0.00443 </td><td>0.00222 </td><td>3.18 </td></tr><tr><td>82 </td><td><em>Pseudomonas putida</em> F1 </td><td>5959 </td><td>61.86 </td><td>0.602 </td><td>0.013 </td><td>0.00113 </td><td>0.00187 </td><td>36.81 </td></tr><tr><td>83 </td><td><em>Ralstonia eutropha</em> H16 </td><td>2912 </td><td>66.78 </td><td>0.238 </td><td>0.47 </td><td>0.00483 </td><td>0.00023 </td><td rowspan="2">B-Proteobacteria </td><td>NA </td></tr><tr><td>84 </td><td><em>Ralstonia solanacearum</em> GMI1000 chromosome </td><td>3716 </td><td>67.04 </td><td>0.056 </td><td><strong>&lt;0.0001</strong> </td><td>0.00636 </td><td>0.00581 </td><td>22.40 </td></tr><tr><td>85 </td><td><em>Rhizobium etli</em> CFN 42 </td><td>4381 </td><td>61.27 </td><td>0.107 </td><td><strong>&lt;0.0001</strong> </td><td>0.00175 </td><td>0.01177 </td><td rowspan="6">A-Proteobacteria </td><td>17.65 </td></tr><tr><td>86 </td><td><em>Rhizobium leguminosarum</em> bv. <em>viciae</em> 3841 </td><td>5057 </td><td>61.09 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00363 </td><td>0.01196 </td><td>NA </td></tr><tr><td>87 </td><td><em>Rickettsia bellii</em> RML369-C </td><td>1522 </td><td>31.65 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00859 </td><td>0.01514 </td><td>26.08 </td></tr><tr><td>88 </td><td><em>Rickettsia conorii</em> Malish 7 </td><td>1268 </td><td>32.44 </td><td>0.584 </td><td>0.052 </td><td>0.00294 </td><td>0.00634 </td><td>16.28 </td></tr><tr><td>89 </td><td><em>Rickettsia rickettsii</em> 'Sheila Smith' </td><td>1257 </td><td>32.47 </td><td>0.575 </td><td><em>0.002</em> </td><td>0.00182 </td><td>0.00767 </td><td>NA </td></tr><tr><td>90 </td><td><em>Rickettsia typhi</em> Wilmington </td><td>1111 </td><td>28.92 </td><td>0.919 </td><td><em>0.007</em> </td><td>0.00020 </td><td>0.01395 </td><td>26.15 </td></tr><tr><td>91 </td><td><em>Salmonella enterica</em> subsp. <em>enterica</em> serovar Typhi CT18 </td><td>4809 </td><td>52.09 </td><td>0.267 </td><td>0.043 </td><td>0.00151 </td><td>0.00152 </td><td rowspan="4">G-Proteobacteria </td><td>9.85 </td></tr><tr><td>92 </td><td><em>Salmonella typhimurium</em> LT2 </td><td>4857 </td><td>52.22 </td><td>0.89 </td><td>0.585 </td><td>0.00043 </td><td>0.00008 </td><td>3.58 </td></tr><tr><td>93 </td><td><em>Shigella boydii</em> Sb227 </td><td>4519 </td><td>51.21 </td><td>0.571 </td><td><em>0.001</em> </td><td>0.00022 </td><td>0.00249 </td><td>11.05 </td></tr><tr><td>94 </td><td><em>Shigella flexneri</em> 58401 </td><td>4574 </td><td>50.92 </td><td>0.48 </td><td>0.268 </td><td>0.00147 </td><td>0.00214 </td><td>NA </td></tr><tr><td>95 </td><td><em>Staphylococcus aureus</em> RF122 </td><td>2742 </td><td>32.78 </td><td>0.788 </td><td>0.427 </td><td>0.00130 </td><td>0.00247 </td><td rowspan="6">Firmicutes </td><td>0.10 </td></tr><tr><td>96 </td><td><em>Staphylococcus epidermidis</em> ATCC 12228 </td><td>2499 </td><td>32.1 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.01246 </td><td>0.01087 </td><td>21.12 </td></tr><tr><td>97 </td><td><em>Staphylococcus haemolyticus</em> JCSC1435 </td><td>2685 </td><td>32.79 </td><td><em>0.001</em> </td><td><em>0</em> </td><td>0.00584 </td><td>0.00643 </td><td rowspan="2">NA </td></tr><tr><td>98 </td><td><em>Streptococcus mutans</em> UA159 </td><td>2030 </td><td>36.83 </td><td>0.111 </td><td>0.046 </td><td>0.00403 </td><td>0.00679 </td></tr><tr><td>99 </td><td><em>Streptococcus pyogenes</em> MGAS2096 </td><td>1860 </td><td>38.73 </td><td>0.619 </td><td>0.15 </td><td>0.00133 </td><td>0.00154 </td><td>3.71 </td></tr><tr><td>100 </td><td><em>Streptococcus thermophilus</em> CNRZ1066 </td><td>1796 </td><td>39.08 </td><td>0.05 </td><td>0.863 </td><td>0.00537 </td><td>0.00459 </td><td>2.63 </td></tr><tr><td>101 </td><td><em>Streptomyces coelicolor</em> A3(2) </td><td>8667 </td><td>72.12 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00394 </td><td>0.00134 </td><td>Actinobacteria </td><td>NA </td></tr><tr><td>102 </td><td><em>Thermotoga maritima</em> MSB8 </td><td>1860 </td><td>46.25 </td><td>0.171 </td><td><strong>&lt;0.0001</strong> </td><td>0.00344 </td><td>0.01548 </td><td rowspan="2">Thermotogae </td><td>39.15 </td></tr><tr><td>103 </td><td><em>Thermotoga petrophila</em> RKU-1 </td><td>1824 </td><td>46.09 </td><td>0.733 </td><td><strong>&lt;0.0001</strong> </td><td>0.00013 </td><td>0.01687 </td><td>NA </td></tr><tr><td>104 </td><td><em>Thiobacillus denitrificans</em> ATCC 25259 </td><td>2909 </td><td>66.07 </td><td>0.962 </td><td>0.086 </td><td>0.00027 </td><td>0.00059 </td><td>B-Proteobacteria </td><td>5.70 </td></tr><tr><td>105 </td><td><em>Vibrio cholerae</em> O395 </td><td>3024 </td><td>47.78 </td><td><strong>&lt;0.0001</strong> </td><td>0.069 </td><td>0.00514 </td><td>0.00105 </td><td rowspan="8">G-Proteobacteria </td><td rowspan="4">NA </td></tr><tr><td>106 </td><td><em>Vibrio fischeri</em> ES114 </td><td>1332 </td><td>37.03 </td><td><em>0.001</em> </td><td>0.037 </td><td>0.00994 </td><td>0.00491 </td></tr><tr><td>107 </td><td><em>Xanthomonas campestris</em> pv. <em>campestris</em> ATCC 33913 </td><td>5076 </td><td>65.07 </td><td>0.196 </td><td>0.719 </td><td>0.00302 </td><td>0.00038 </td></tr><tr><td>108 </td><td><em>Xanthomonas oryzae</em> pv. <em>oryzae</em> KACC 10331 </td><td>4941 </td><td>63.69 </td><td>0.87 </td><td>0.499 </td><td>0.00104 </td><td>0.00065 </td></tr><tr><td>109 </td><td><em>Xylella fastidiosa</em> 9a5c </td><td>2679 </td><td>52.68 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.04727 </td><td>0.05291 </td><td>62.97 </td></tr><tr><td>110 </td><td><em>Xylella fastidiosa</em> Temecula 1 </td><td>2519 </td><td>51.78 </td><td>0.044 </td><td><em>0</em> </td><td>0.00379 </td><td>0.01093 </td><td>6.44 </td></tr><tr><td>111 </td><td><em>Yersinia pestis</em> CO92 </td><td>4653 </td><td>47.64 </td><td>0.649 </td><td><em>0.001</em> </td><td>0.00090 </td><td>0.00520 </td><td rowspan="2">NA </td></tr><tr><td>112 </td><td><em>Yersinia pseudotuberculosis</em> IP32953 </td><td>4744 </td><td>47.61 </td><td>0.969 </td><td><em>0.001</em> </td><td>0.00124 </td><td>0.00496 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>TB, termination bias. Chromosomes of bacteria analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as: KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T) and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting that the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273960" target="_blank">View Large</a></div></div>                        <a id="5273961" scrollto-destination="5273961"></a>
<div content-id="DSP021TB2" class="table-modal"><div class="table-wrap"><div id="DSP021TB2" class="table-wrap-title" data-id="DSP021TB2"><div class="label -label">Table 2</div><div class="caption"><p>ISFDP analysis in archaea chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Archaea group </th></tr></thead><tbody><tr><td>1 </td><td><em>Aeropyrum pernix</em> K1 </td><td>1670 </td><td>56.3 </td><td><em>0.001</em> </td><td>0.025 </td><td>0.01292 </td><td>0.00695 </td><td>Crenarchaeota </td></tr><tr><td>2 </td><td><em>Archaeoglobus fulgidus</em> DSM 4304 </td><td>2179 </td><td>48.5 </td><td>0.037 </td><td>0.093 </td><td>0.00365 </td><td>0.00350 </td><td>Euryarchaeota </td></tr><tr><td>3 </td><td><em>Caldivirga maquilingensis</em> IC-167 </td><td>2078 </td><td>43.08 </td><td>0.586 </td><td>0.643 </td><td>0.00146 </td><td>0.00104 </td><td>Crenarchaeota </td></tr><tr><td>4 </td><td>Candidatus <em>Methanoregula boonei</em> 6A8 </td><td>2543 </td><td>54.51 </td><td>0.058 </td><td>0.191 </td><td>0.00311 </td><td>0.00108 </td><td>Euryarchaeota </td></tr><tr><td>5 </td><td><em>Cenarchaeum symbiosum</em> </td><td>2046 </td><td>57.34 </td><td>0.101 </td><td><em>0.006</em> </td><td>0.00574 </td><td>0.00161 </td><td>Crenarchaeota </td></tr><tr><td>6 </td><td><em>Haloarcula marismortui</em> ATCC 43049 chromosome 1 </td><td>3132 </td><td>62.35 </td><td>0.252 </td><td>0.905 </td><td>0.01075 </td><td>0.00024 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>7 </td><td><em>Halobacterium</em> sp. NRC-1 </td><td>2015 </td><td>67.88 </td><td>0.862 </td><td>0.313 </td><td>0.00056 </td><td>0.00151 </td></tr><tr><td>8 </td><td><em>Haloquadratum walsbyi</em> DSM 16790 </td><td>3133 </td><td>47.85 </td><td>0.578 </td><td>0.027 </td><td>0.00160 </td><td>0.00523 </td></tr><tr><td>9 </td><td><em>Hyperthermus butylicus</em> DSM 5456 </td><td>1668 </td><td>53.7 </td><td>0.019 </td><td>0.908 </td><td>0.00531 </td><td>0.00100 </td><td rowspan="2">Crenarchaeota </td></tr><tr><td>10 </td><td><em>Ignicoccus hospitalis</em> KIN4/I </td><td>1298 </td><td>56.5 </td><td>0.118 </td><td>0.901 </td><td>0.00199 </td><td>0.00014 </td></tr><tr><td>11 </td><td><em>Metallosphaera sedula</em> DSM 5348 </td><td>2192 </td><td>46.21 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00668 </td><td>0.01423 </td><td>Crenarchaeota </td></tr><tr><td>12 </td><td><em>Methanobrevibacter smithii</em> ATCC 35061 </td><td>1854 </td><td>31.02 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02048 </td><td>0.03768 </td><td rowspan="19">Euryarchaeota </td></tr><tr><td>13 </td><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>1666 </td><td>31.4 </td><td>0.132 </td><td>0.031 </td><td>0.00450 </td><td>0.01128 </td></tr><tr><td>14 </td><td><em>Methanococcoides burtonii</em> DSM 6242 </td><td>2576 </td><td>40.74 </td><td>0.078 </td><td><em>0.002</em> </td><td>0.00266 </td><td>0.00845 </td></tr><tr><td>15 </td><td><em>Methanococcus aeolicus</em> Nankai-3 </td><td>1570 </td><td>30.02 </td><td>0.218 </td><td>0.52 </td><td>0.00399 </td><td>0.00063 </td></tr><tr><td>16 </td><td><em>Methanococcus maripaludis</em> C5 </td><td>1781 </td><td>32.99 </td><td><em>0.001</em> </td><td>0.065 </td><td>0.00846 </td><td>0.00454 </td></tr><tr><td>17 </td><td><em>Methanococcus maripaludis</em> C6 </td><td>1745 </td><td>33.4 </td><td>0.045 </td><td>0.045 </td><td>0.00553 </td><td>0.00224 </td></tr><tr><td>18 </td><td><em>Methanococcus maripaludis</em> C7 </td><td>1773 </td><td>33.27 </td><td>0.256 </td><td>0.784 </td><td>0.00430 </td><td>0.00088 </td></tr><tr><td>19 </td><td><em>Methanococcus maripaludis</em> S2 </td><td>1662 </td><td>33.08 </td><td>0.021 </td><td>0.08 </td><td>0.00619 </td><td>0.00259 </td></tr><tr><td>20 </td><td><em>Methanococcus vannielii</em> SB </td><td>1721 </td><td>31.31 </td><td>0.505 </td><td>0.519 </td><td>0.00364 </td><td>0.00400 </td></tr><tr><td>21 </td><td><em>Methanocorpusculum labreanum</em> Z </td><td>1806 </td><td>49.97 </td><td>0.606 </td><td>0.05 </td><td>0.00097 </td><td>0.00404 </td></tr><tr><td>22 </td><td><em>Methanoculleus marisnigri</em> JR1 </td><td>2479 </td><td>62.04 </td><td>0.816 </td><td>0.745 </td><td>0.00234 </td><td>0.00000 </td></tr><tr><td>23 </td><td><em>Methanopyrus kandleri</em> AV19 </td><td>1696 </td><td>61.22 </td><td>0.556 </td><td>0.032 </td><td>0.00230 </td><td>0.00471 </td></tr><tr><td>24 </td><td><em>Methanosaeta thermophila</em> PT </td><td>1880 </td><td>53.53 </td><td>0.673 </td><td><em>0.004</em> </td><td>0.00018 </td><td>0.00595 </td></tr><tr><td>25 </td><td><em>Methanosarcina acetivorans</em> C2A </td><td>5752 </td><td>42.67 </td><td><strong>&lt;0.0001</strong> </td><td>0.839 </td><td>0.00628 </td><td>0.00083 </td></tr><tr><td>26 </td><td><em>Methanosarcina barkeri</em> Fusaro </td><td>4838 </td><td>39.27 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.003</em> </td><td>0.00475 </td><td>0.00391 </td></tr><tr><td>27 </td><td><em>Methanosarcina mazei</em> Goe1 </td><td>4097 </td><td>41.47 </td><td>0.252 </td><td>0.812 </td><td>0.00212 </td><td>0.00079 </td></tr><tr><td>28 </td><td><em>Methanosphaera stadtmanae</em> DSM 3091 </td><td>1768 </td><td>27.62 </td><td><em>0.002</em> </td><td>0.275 </td><td>0.00897 </td><td>0.00652 </td></tr><tr><td>29 </td><td><em>Methanospirillum hungatei</em> JF-1 </td><td>3545 </td><td>45.14 </td><td><strong>&lt;0.0001</strong> </td><td>0.015 </td><td>0.00951 </td><td>0.00411 </td></tr><tr><td>30 </td><td><em>Methanothermobacter thermautotrophicus</em> Delta H </td><td>1752 </td><td>49.52 </td><td>0.022 </td><td>0.114 </td><td>0.00566 </td><td>0.00166 </td></tr><tr><td>31 </td><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>491 </td><td>31.55 </td><td>0.549 </td><td>0.177 </td><td>0.00000 </td><td>0.00127 </td><td>Nanoarchaeota </td></tr><tr><td>32 </td><td><em>Natronomonas pharaonis</em> DSM 2160 </td><td>2596 </td><td>63.42 </td><td>0.473 </td><td>0.228 </td><td>0.00174 </td><td>0.00091 </td><td>Euryarchaeota </td></tr><tr><td>33 </td><td><em>Nitrosopumilus maritimus</em> SCM1 </td><td>1646 </td><td>31.15 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.002</em> </td><td>0.00921 </td><td>0.00855 </td><td>Crenarchaeota </td></tr><tr><td>34 </td><td><em>Picrophilus torridus</em> DSM 9790 </td><td>1546 </td><td>35.96 </td><td>0.296 </td><td><em>0.008</em> </td><td>0.00096 </td><td>0.00793 </td><td>Euryarchaeota </td></tr><tr><td>35 </td><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>2223 </td><td>51.34 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00727 </td><td>0.01022 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>36 </td><td><em>Pyrobaculum arsenaticum</em> DSM 13514 </td><td>2122 </td><td>54.98 </td><td>0.795 </td><td>0.431 </td><td>0.00138 </td><td>0.00316 </td></tr><tr><td>37 </td><td><em>Pyrobaculum calidifontis</em> JCM 11548 </td><td>2010 </td><td>57.13 </td><td>0.148 </td><td>0.337 </td><td>0.00294 </td><td>0.00008 </td></tr><tr><td>38 </td><td><em>Pyrobaculum islandicum</em> DSM 4184 </td><td>1827 </td><td>49.58 </td><td>0.305 </td><td>0.436 </td><td>0.00085 </td><td>0.00183 </td></tr><tr><td>39 </td><td><em>Pyrococcus abyssi</em> </td><td>1766 </td><td>44.69 </td><td>0.652 </td><td>0.574 </td><td>0.00219 </td><td>0.00342 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>40 </td><td><em>Pyrococcus furiosus</em> DSM 3638 </td><td>1909 </td><td>40.75 </td><td>0.754 </td><td>0.757 </td><td>0.00004 </td><td>0.00094 </td></tr><tr><td>41 </td><td><em>Pyrococcus horikoshii</em> OT3 </td><td>1739 </td><td>41.86 </td><td>0.133 </td><td><em>0.002</em> </td><td>0.00229 </td><td>0.01262 </td></tr><tr><td>42 </td><td><em>Staphylothermus marinus</em> F1 </td><td>1571 </td><td>35.71 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02078 </td><td>0.02726 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>43 </td><td><em>Sulfolobus acidocaldarius</em> DSM 639 </td><td>2227 </td><td>36.69 </td><td>0.413 </td><td>0.526 </td><td>0.00309 </td><td>0.00124 </td></tr><tr><td>44 </td><td><em>Sulfolobus solfataricus</em> P2 </td><td>2993 </td><td>35.77 </td><td><em>0.007</em> </td><td>0.747 </td><td>0.00533 </td><td>0.00241 </td></tr><tr><td>45 </td><td><em>Sulfolobus tokodaii</em> 7 </td><td>2695 </td><td>32.78 </td><td><em>0.005</em> </td><td>0.029 </td><td>0.00521 </td><td>0.00659 </td></tr><tr><td>46 </td><td><em>Thermococcus kodakarensis</em> KOD1 </td><td>2089 </td><td>51.98 </td><td>0.062 </td><td>0.328 </td><td>0.00418 </td><td>0.00160 </td><td>Euryarchaeota </td></tr><tr><td>47 </td><td><em>Thermofilum pendens</em> Hrk 5 </td><td>1782 </td><td>57.66 </td><td>0.014 </td><td><em>0.005</em> </td><td>0.00346 </td><td>0.00665 </td><td>Crenarchaeota </td></tr><tr><td>48 </td><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>1565 </td><td>45.99 </td><td>0.016 </td><td>0.016 </td><td>0.00680 </td><td>0.00383 </td><td rowspan="2">Euryarchaeota </td></tr><tr><td>49 </td><td><em>Thermoplasma volcanium</em> GSS1 </td><td>1585 </td><td>39.91 </td><td>0.055 </td><td>0.361 </td><td>0.00404 </td><td>0.00263 </td></tr></tbody></table></div><div content-id="DSP021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Archaea group </th></tr></thead><tbody><tr><td>1 </td><td><em>Aeropyrum pernix</em> K1 </td><td>1670 </td><td>56.3 </td><td><em>0.001</em> </td><td>0.025 </td><td>0.01292 </td><td>0.00695 </td><td>Crenarchaeota </td></tr><tr><td>2 </td><td><em>Archaeoglobus fulgidus</em> DSM 4304 </td><td>2179 </td><td>48.5 </td><td>0.037 </td><td>0.093 </td><td>0.00365 </td><td>0.00350 </td><td>Euryarchaeota </td></tr><tr><td>3 </td><td><em>Caldivirga maquilingensis</em> IC-167 </td><td>2078 </td><td>43.08 </td><td>0.586 </td><td>0.643 </td><td>0.00146 </td><td>0.00104 </td><td>Crenarchaeota </td></tr><tr><td>4 </td><td>Candidatus <em>Methanoregula boonei</em> 6A8 </td><td>2543 </td><td>54.51 </td><td>0.058 </td><td>0.191 </td><td>0.00311 </td><td>0.00108 </td><td>Euryarchaeota </td></tr><tr><td>5 </td><td><em>Cenarchaeum symbiosum</em> </td><td>2046 </td><td>57.34 </td><td>0.101 </td><td><em>0.006</em> </td><td>0.00574 </td><td>0.00161 </td><td>Crenarchaeota </td></tr><tr><td>6 </td><td><em>Haloarcula marismortui</em> ATCC 43049 chromosome 1 </td><td>3132 </td><td>62.35 </td><td>0.252 </td><td>0.905 </td><td>0.01075 </td><td>0.00024 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>7 </td><td><em>Halobacterium</em> sp. NRC-1 </td><td>2015 </td><td>67.88 </td><td>0.862 </td><td>0.313 </td><td>0.00056 </td><td>0.00151 </td></tr><tr><td>8 </td><td><em>Haloquadratum walsbyi</em> DSM 16790 </td><td>3133 </td><td>47.85 </td><td>0.578 </td><td>0.027 </td><td>0.00160 </td><td>0.00523 </td></tr><tr><td>9 </td><td><em>Hyperthermus butylicus</em> DSM 5456 </td><td>1668 </td><td>53.7 </td><td>0.019 </td><td>0.908 </td><td>0.00531 </td><td>0.00100 </td><td rowspan="2">Crenarchaeota </td></tr><tr><td>10 </td><td><em>Ignicoccus hospitalis</em> KIN4/I </td><td>1298 </td><td>56.5 </td><td>0.118 </td><td>0.901 </td><td>0.00199 </td><td>0.00014 </td></tr><tr><td>11 </td><td><em>Metallosphaera sedula</em> DSM 5348 </td><td>2192 </td><td>46.21 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00668 </td><td>0.01423 </td><td>Crenarchaeota </td></tr><tr><td>12 </td><td><em>Methanobrevibacter smithii</em> ATCC 35061 </td><td>1854 </td><td>31.02 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02048 </td><td>0.03768 </td><td rowspan="19">Euryarchaeota </td></tr><tr><td>13 </td><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>1666 </td><td>31.4 </td><td>0.132 </td><td>0.031 </td><td>0.00450 </td><td>0.01128 </td></tr><tr><td>14 </td><td><em>Methanococcoides burtonii</em> DSM 6242 </td><td>2576 </td><td>40.74 </td><td>0.078 </td><td><em>0.002</em> </td><td>0.00266 </td><td>0.00845 </td></tr><tr><td>15 </td><td><em>Methanococcus aeolicus</em> Nankai-3 </td><td>1570 </td><td>30.02 </td><td>0.218 </td><td>0.52 </td><td>0.00399 </td><td>0.00063 </td></tr><tr><td>16 </td><td><em>Methanococcus maripaludis</em> C5 </td><td>1781 </td><td>32.99 </td><td><em>0.001</em> </td><td>0.065 </td><td>0.00846 </td><td>0.00454 </td></tr><tr><td>17 </td><td><em>Methanococcus maripaludis</em> C6 </td><td>1745 </td><td>33.4 </td><td>0.045 </td><td>0.045 </td><td>0.00553 </td><td>0.00224 </td></tr><tr><td>18 </td><td><em>Methanococcus maripaludis</em> C7 </td><td>1773 </td><td>33.27 </td><td>0.256 </td><td>0.784 </td><td>0.00430 </td><td>0.00088 </td></tr><tr><td>19 </td><td><em>Methanococcus maripaludis</em> S2 </td><td>1662 </td><td>33.08 </td><td>0.021 </td><td>0.08 </td><td>0.00619 </td><td>0.00259 </td></tr><tr><td>20 </td><td><em>Methanococcus vannielii</em> SB </td><td>1721 </td><td>31.31 </td><td>0.505 </td><td>0.519 </td><td>0.00364 </td><td>0.00400 </td></tr><tr><td>21 </td><td><em>Methanocorpusculum labreanum</em> Z </td><td>1806 </td><td>49.97 </td><td>0.606 </td><td>0.05 </td><td>0.00097 </td><td>0.00404 </td></tr><tr><td>22 </td><td><em>Methanoculleus marisnigri</em> JR1 </td><td>2479 </td><td>62.04 </td><td>0.816 </td><td>0.745 </td><td>0.00234 </td><td>0.00000 </td></tr><tr><td>23 </td><td><em>Methanopyrus kandleri</em> AV19 </td><td>1696 </td><td>61.22 </td><td>0.556 </td><td>0.032 </td><td>0.00230 </td><td>0.00471 </td></tr><tr><td>24 </td><td><em>Methanosaeta thermophila</em> PT </td><td>1880 </td><td>53.53 </td><td>0.673 </td><td><em>0.004</em> </td><td>0.00018 </td><td>0.00595 </td></tr><tr><td>25 </td><td><em>Methanosarcina acetivorans</em> C2A </td><td>5752 </td><td>42.67 </td><td><strong>&lt;0.0001</strong> </td><td>0.839 </td><td>0.00628 </td><td>0.00083 </td></tr><tr><td>26 </td><td><em>Methanosarcina barkeri</em> Fusaro </td><td>4838 </td><td>39.27 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.003</em> </td><td>0.00475 </td><td>0.00391 </td></tr><tr><td>27 </td><td><em>Methanosarcina mazei</em> Goe1 </td><td>4097 </td><td>41.47 </td><td>0.252 </td><td>0.812 </td><td>0.00212 </td><td>0.00079 </td></tr><tr><td>28 </td><td><em>Methanosphaera stadtmanae</em> DSM 3091 </td><td>1768 </td><td>27.62 </td><td><em>0.002</em> </td><td>0.275 </td><td>0.00897 </td><td>0.00652 </td></tr><tr><td>29 </td><td><em>Methanospirillum hungatei</em> JF-1 </td><td>3545 </td><td>45.14 </td><td><strong>&lt;0.0001</strong> </td><td>0.015 </td><td>0.00951 </td><td>0.00411 </td></tr><tr><td>30 </td><td><em>Methanothermobacter thermautotrophicus</em> Delta H </td><td>1752 </td><td>49.52 </td><td>0.022 </td><td>0.114 </td><td>0.00566 </td><td>0.00166 </td></tr><tr><td>31 </td><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>491 </td><td>31.55 </td><td>0.549 </td><td>0.177 </td><td>0.00000 </td><td>0.00127 </td><td>Nanoarchaeota </td></tr><tr><td>32 </td><td><em>Natronomonas pharaonis</em> DSM 2160 </td><td>2596 </td><td>63.42 </td><td>0.473 </td><td>0.228 </td><td>0.00174 </td><td>0.00091 </td><td>Euryarchaeota </td></tr><tr><td>33 </td><td><em>Nitrosopumilus maritimus</em> SCM1 </td><td>1646 </td><td>31.15 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.002</em> </td><td>0.00921 </td><td>0.00855 </td><td>Crenarchaeota </td></tr><tr><td>34 </td><td><em>Picrophilus torridus</em> DSM 9790 </td><td>1546 </td><td>35.96 </td><td>0.296 </td><td><em>0.008</em> </td><td>0.00096 </td><td>0.00793 </td><td>Euryarchaeota </td></tr><tr><td>35 </td><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>2223 </td><td>51.34 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00727 </td><td>0.01022 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>36 </td><td><em>Pyrobaculum arsenaticum</em> DSM 13514 </td><td>2122 </td><td>54.98 </td><td>0.795 </td><td>0.431 </td><td>0.00138 </td><td>0.00316 </td></tr><tr><td>37 </td><td><em>Pyrobaculum calidifontis</em> JCM 11548 </td><td>2010 </td><td>57.13 </td><td>0.148 </td><td>0.337 </td><td>0.00294 </td><td>0.00008 </td></tr><tr><td>38 </td><td><em>Pyrobaculum islandicum</em> DSM 4184 </td><td>1827 </td><td>49.58 </td><td>0.305 </td><td>0.436 </td><td>0.00085 </td><td>0.00183 </td></tr><tr><td>39 </td><td><em>Pyrococcus abyssi</em> </td><td>1766 </td><td>44.69 </td><td>0.652 </td><td>0.574 </td><td>0.00219 </td><td>0.00342 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>40 </td><td><em>Pyrococcus furiosus</em> DSM 3638 </td><td>1909 </td><td>40.75 </td><td>0.754 </td><td>0.757 </td><td>0.00004 </td><td>0.00094 </td></tr><tr><td>41 </td><td><em>Pyrococcus horikoshii</em> OT3 </td><td>1739 </td><td>41.86 </td><td>0.133 </td><td><em>0.002</em> </td><td>0.00229 </td><td>0.01262 </td></tr><tr><td>42 </td><td><em>Staphylothermus marinus</em> F1 </td><td>1571 </td><td>35.71 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02078 </td><td>0.02726 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>43 </td><td><em>Sulfolobus acidocaldarius</em> DSM 639 </td><td>2227 </td><td>36.69 </td><td>0.413 </td><td>0.526 </td><td>0.00309 </td><td>0.00124 </td></tr><tr><td>44 </td><td><em>Sulfolobus solfataricus</em> P2 </td><td>2993 </td><td>35.77 </td><td><em>0.007</em> </td><td>0.747 </td><td>0.00533 </td><td>0.00241 </td></tr><tr><td>45 </td><td><em>Sulfolobus tokodaii</em> 7 </td><td>2695 </td><td>32.78 </td><td><em>0.005</em> </td><td>0.029 </td><td>0.00521 </td><td>0.00659 </td></tr><tr><td>46 </td><td><em>Thermococcus kodakarensis</em> KOD1 </td><td>2089 </td><td>51.98 </td><td>0.062 </td><td>0.328 </td><td>0.00418 </td><td>0.00160 </td><td>Euryarchaeota </td></tr><tr><td>47 </td><td><em>Thermofilum pendens</em> Hrk 5 </td><td>1782 </td><td>57.66 </td><td>0.014 </td><td><em>0.005</em> </td><td>0.00346 </td><td>0.00665 </td><td>Crenarchaeota </td></tr><tr><td>48 </td><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>1565 </td><td>45.99 </td><td>0.016 </td><td>0.016 </td><td>0.00680 </td><td>0.00383 </td><td rowspan="2">Euryarchaeota </td></tr><tr><td>49 </td><td><em>Thermoplasma volcanium</em> GSS1 </td><td>1585 </td><td>39.91 </td><td>0.055 </td><td>0.361 </td><td>0.00404 </td><td>0.00263 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Chromosomes of archaea analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but ≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T)and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273961" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP021TB2" class="table-wrap-title" data-id="DSP021TB2"><div class="label -label">Table 2</div><div class="caption"><p>ISFDP analysis in archaea chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Archaea group </th></tr></thead><tbody><tr><td>1 </td><td><em>Aeropyrum pernix</em> K1 </td><td>1670 </td><td>56.3 </td><td><em>0.001</em> </td><td>0.025 </td><td>0.01292 </td><td>0.00695 </td><td>Crenarchaeota </td></tr><tr><td>2 </td><td><em>Archaeoglobus fulgidus</em> DSM 4304 </td><td>2179 </td><td>48.5 </td><td>0.037 </td><td>0.093 </td><td>0.00365 </td><td>0.00350 </td><td>Euryarchaeota </td></tr><tr><td>3 </td><td><em>Caldivirga maquilingensis</em> IC-167 </td><td>2078 </td><td>43.08 </td><td>0.586 </td><td>0.643 </td><td>0.00146 </td><td>0.00104 </td><td>Crenarchaeota </td></tr><tr><td>4 </td><td>Candidatus <em>Methanoregula boonei</em> 6A8 </td><td>2543 </td><td>54.51 </td><td>0.058 </td><td>0.191 </td><td>0.00311 </td><td>0.00108 </td><td>Euryarchaeota </td></tr><tr><td>5 </td><td><em>Cenarchaeum symbiosum</em> </td><td>2046 </td><td>57.34 </td><td>0.101 </td><td><em>0.006</em> </td><td>0.00574 </td><td>0.00161 </td><td>Crenarchaeota </td></tr><tr><td>6 </td><td><em>Haloarcula marismortui</em> ATCC 43049 chromosome 1 </td><td>3132 </td><td>62.35 </td><td>0.252 </td><td>0.905 </td><td>0.01075 </td><td>0.00024 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>7 </td><td><em>Halobacterium</em> sp. NRC-1 </td><td>2015 </td><td>67.88 </td><td>0.862 </td><td>0.313 </td><td>0.00056 </td><td>0.00151 </td></tr><tr><td>8 </td><td><em>Haloquadratum walsbyi</em> DSM 16790 </td><td>3133 </td><td>47.85 </td><td>0.578 </td><td>0.027 </td><td>0.00160 </td><td>0.00523 </td></tr><tr><td>9 </td><td><em>Hyperthermus butylicus</em> DSM 5456 </td><td>1668 </td><td>53.7 </td><td>0.019 </td><td>0.908 </td><td>0.00531 </td><td>0.00100 </td><td rowspan="2">Crenarchaeota </td></tr><tr><td>10 </td><td><em>Ignicoccus hospitalis</em> KIN4/I </td><td>1298 </td><td>56.5 </td><td>0.118 </td><td>0.901 </td><td>0.00199 </td><td>0.00014 </td></tr><tr><td>11 </td><td><em>Metallosphaera sedula</em> DSM 5348 </td><td>2192 </td><td>46.21 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00668 </td><td>0.01423 </td><td>Crenarchaeota </td></tr><tr><td>12 </td><td><em>Methanobrevibacter smithii</em> ATCC 35061 </td><td>1854 </td><td>31.02 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02048 </td><td>0.03768 </td><td rowspan="19">Euryarchaeota </td></tr><tr><td>13 </td><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>1666 </td><td>31.4 </td><td>0.132 </td><td>0.031 </td><td>0.00450 </td><td>0.01128 </td></tr><tr><td>14 </td><td><em>Methanococcoides burtonii</em> DSM 6242 </td><td>2576 </td><td>40.74 </td><td>0.078 </td><td><em>0.002</em> </td><td>0.00266 </td><td>0.00845 </td></tr><tr><td>15 </td><td><em>Methanococcus aeolicus</em> Nankai-3 </td><td>1570 </td><td>30.02 </td><td>0.218 </td><td>0.52 </td><td>0.00399 </td><td>0.00063 </td></tr><tr><td>16 </td><td><em>Methanococcus maripaludis</em> C5 </td><td>1781 </td><td>32.99 </td><td><em>0.001</em> </td><td>0.065 </td><td>0.00846 </td><td>0.00454 </td></tr><tr><td>17 </td><td><em>Methanococcus maripaludis</em> C6 </td><td>1745 </td><td>33.4 </td><td>0.045 </td><td>0.045 </td><td>0.00553 </td><td>0.00224 </td></tr><tr><td>18 </td><td><em>Methanococcus maripaludis</em> C7 </td><td>1773 </td><td>33.27 </td><td>0.256 </td><td>0.784 </td><td>0.00430 </td><td>0.00088 </td></tr><tr><td>19 </td><td><em>Methanococcus maripaludis</em> S2 </td><td>1662 </td><td>33.08 </td><td>0.021 </td><td>0.08 </td><td>0.00619 </td><td>0.00259 </td></tr><tr><td>20 </td><td><em>Methanococcus vannielii</em> SB </td><td>1721 </td><td>31.31 </td><td>0.505 </td><td>0.519 </td><td>0.00364 </td><td>0.00400 </td></tr><tr><td>21 </td><td><em>Methanocorpusculum labreanum</em> Z </td><td>1806 </td><td>49.97 </td><td>0.606 </td><td>0.05 </td><td>0.00097 </td><td>0.00404 </td></tr><tr><td>22 </td><td><em>Methanoculleus marisnigri</em> JR1 </td><td>2479 </td><td>62.04 </td><td>0.816 </td><td>0.745 </td><td>0.00234 </td><td>0.00000 </td></tr><tr><td>23 </td><td><em>Methanopyrus kandleri</em> AV19 </td><td>1696 </td><td>61.22 </td><td>0.556 </td><td>0.032 </td><td>0.00230 </td><td>0.00471 </td></tr><tr><td>24 </td><td><em>Methanosaeta thermophila</em> PT </td><td>1880 </td><td>53.53 </td><td>0.673 </td><td><em>0.004</em> </td><td>0.00018 </td><td>0.00595 </td></tr><tr><td>25 </td><td><em>Methanosarcina acetivorans</em> C2A </td><td>5752 </td><td>42.67 </td><td><strong>&lt;0.0001</strong> </td><td>0.839 </td><td>0.00628 </td><td>0.00083 </td></tr><tr><td>26 </td><td><em>Methanosarcina barkeri</em> Fusaro </td><td>4838 </td><td>39.27 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.003</em> </td><td>0.00475 </td><td>0.00391 </td></tr><tr><td>27 </td><td><em>Methanosarcina mazei</em> Goe1 </td><td>4097 </td><td>41.47 </td><td>0.252 </td><td>0.812 </td><td>0.00212 </td><td>0.00079 </td></tr><tr><td>28 </td><td><em>Methanosphaera stadtmanae</em> DSM 3091 </td><td>1768 </td><td>27.62 </td><td><em>0.002</em> </td><td>0.275 </td><td>0.00897 </td><td>0.00652 </td></tr><tr><td>29 </td><td><em>Methanospirillum hungatei</em> JF-1 </td><td>3545 </td><td>45.14 </td><td><strong>&lt;0.0001</strong> </td><td>0.015 </td><td>0.00951 </td><td>0.00411 </td></tr><tr><td>30 </td><td><em>Methanothermobacter thermautotrophicus</em> Delta H </td><td>1752 </td><td>49.52 </td><td>0.022 </td><td>0.114 </td><td>0.00566 </td><td>0.00166 </td></tr><tr><td>31 </td><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>491 </td><td>31.55 </td><td>0.549 </td><td>0.177 </td><td>0.00000 </td><td>0.00127 </td><td>Nanoarchaeota </td></tr><tr><td>32 </td><td><em>Natronomonas pharaonis</em> DSM 2160 </td><td>2596 </td><td>63.42 </td><td>0.473 </td><td>0.228 </td><td>0.00174 </td><td>0.00091 </td><td>Euryarchaeota </td></tr><tr><td>33 </td><td><em>Nitrosopumilus maritimus</em> SCM1 </td><td>1646 </td><td>31.15 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.002</em> </td><td>0.00921 </td><td>0.00855 </td><td>Crenarchaeota </td></tr><tr><td>34 </td><td><em>Picrophilus torridus</em> DSM 9790 </td><td>1546 </td><td>35.96 </td><td>0.296 </td><td><em>0.008</em> </td><td>0.00096 </td><td>0.00793 </td><td>Euryarchaeota </td></tr><tr><td>35 </td><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>2223 </td><td>51.34 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00727 </td><td>0.01022 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>36 </td><td><em>Pyrobaculum arsenaticum</em> DSM 13514 </td><td>2122 </td><td>54.98 </td><td>0.795 </td><td>0.431 </td><td>0.00138 </td><td>0.00316 </td></tr><tr><td>37 </td><td><em>Pyrobaculum calidifontis</em> JCM 11548 </td><td>2010 </td><td>57.13 </td><td>0.148 </td><td>0.337 </td><td>0.00294 </td><td>0.00008 </td></tr><tr><td>38 </td><td><em>Pyrobaculum islandicum</em> DSM 4184 </td><td>1827 </td><td>49.58 </td><td>0.305 </td><td>0.436 </td><td>0.00085 </td><td>0.00183 </td></tr><tr><td>39 </td><td><em>Pyrococcus abyssi</em> </td><td>1766 </td><td>44.69 </td><td>0.652 </td><td>0.574 </td><td>0.00219 </td><td>0.00342 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>40 </td><td><em>Pyrococcus furiosus</em> DSM 3638 </td><td>1909 </td><td>40.75 </td><td>0.754 </td><td>0.757 </td><td>0.00004 </td><td>0.00094 </td></tr><tr><td>41 </td><td><em>Pyrococcus horikoshii</em> OT3 </td><td>1739 </td><td>41.86 </td><td>0.133 </td><td><em>0.002</em> </td><td>0.00229 </td><td>0.01262 </td></tr><tr><td>42 </td><td><em>Staphylothermus marinus</em> F1 </td><td>1571 </td><td>35.71 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02078 </td><td>0.02726 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>43 </td><td><em>Sulfolobus acidocaldarius</em> DSM 639 </td><td>2227 </td><td>36.69 </td><td>0.413 </td><td>0.526 </td><td>0.00309 </td><td>0.00124 </td></tr><tr><td>44 </td><td><em>Sulfolobus solfataricus</em> P2 </td><td>2993 </td><td>35.77 </td><td><em>0.007</em> </td><td>0.747 </td><td>0.00533 </td><td>0.00241 </td></tr><tr><td>45 </td><td><em>Sulfolobus tokodaii</em> 7 </td><td>2695 </td><td>32.78 </td><td><em>0.005</em> </td><td>0.029 </td><td>0.00521 </td><td>0.00659 </td></tr><tr><td>46 </td><td><em>Thermococcus kodakarensis</em> KOD1 </td><td>2089 </td><td>51.98 </td><td>0.062 </td><td>0.328 </td><td>0.00418 </td><td>0.00160 </td><td>Euryarchaeota </td></tr><tr><td>47 </td><td><em>Thermofilum pendens</em> Hrk 5 </td><td>1782 </td><td>57.66 </td><td>0.014 </td><td><em>0.005</em> </td><td>0.00346 </td><td>0.00665 </td><td>Crenarchaeota </td></tr><tr><td>48 </td><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>1565 </td><td>45.99 </td><td>0.016 </td><td>0.016 </td><td>0.00680 </td><td>0.00383 </td><td rowspan="2">Euryarchaeota </td></tr><tr><td>49 </td><td><em>Thermoplasma volcanium</em> GSS1 </td><td>1585 </td><td>39.91 </td><td>0.055 </td><td>0.361 </td><td>0.00404 </td><td>0.00263 </td></tr></tbody></table></div><div content-id="DSP021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Archaea group </th></tr></thead><tbody><tr><td>1 </td><td><em>Aeropyrum pernix</em> K1 </td><td>1670 </td><td>56.3 </td><td><em>0.001</em> </td><td>0.025 </td><td>0.01292 </td><td>0.00695 </td><td>Crenarchaeota </td></tr><tr><td>2 </td><td><em>Archaeoglobus fulgidus</em> DSM 4304 </td><td>2179 </td><td>48.5 </td><td>0.037 </td><td>0.093 </td><td>0.00365 </td><td>0.00350 </td><td>Euryarchaeota </td></tr><tr><td>3 </td><td><em>Caldivirga maquilingensis</em> IC-167 </td><td>2078 </td><td>43.08 </td><td>0.586 </td><td>0.643 </td><td>0.00146 </td><td>0.00104 </td><td>Crenarchaeota </td></tr><tr><td>4 </td><td>Candidatus <em>Methanoregula boonei</em> 6A8 </td><td>2543 </td><td>54.51 </td><td>0.058 </td><td>0.191 </td><td>0.00311 </td><td>0.00108 </td><td>Euryarchaeota </td></tr><tr><td>5 </td><td><em>Cenarchaeum symbiosum</em> </td><td>2046 </td><td>57.34 </td><td>0.101 </td><td><em>0.006</em> </td><td>0.00574 </td><td>0.00161 </td><td>Crenarchaeota </td></tr><tr><td>6 </td><td><em>Haloarcula marismortui</em> ATCC 43049 chromosome 1 </td><td>3132 </td><td>62.35 </td><td>0.252 </td><td>0.905 </td><td>0.01075 </td><td>0.00024 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>7 </td><td><em>Halobacterium</em> sp. NRC-1 </td><td>2015 </td><td>67.88 </td><td>0.862 </td><td>0.313 </td><td>0.00056 </td><td>0.00151 </td></tr><tr><td>8 </td><td><em>Haloquadratum walsbyi</em> DSM 16790 </td><td>3133 </td><td>47.85 </td><td>0.578 </td><td>0.027 </td><td>0.00160 </td><td>0.00523 </td></tr><tr><td>9 </td><td><em>Hyperthermus butylicus</em> DSM 5456 </td><td>1668 </td><td>53.7 </td><td>0.019 </td><td>0.908 </td><td>0.00531 </td><td>0.00100 </td><td rowspan="2">Crenarchaeota </td></tr><tr><td>10 </td><td><em>Ignicoccus hospitalis</em> KIN4/I </td><td>1298 </td><td>56.5 </td><td>0.118 </td><td>0.901 </td><td>0.00199 </td><td>0.00014 </td></tr><tr><td>11 </td><td><em>Metallosphaera sedula</em> DSM 5348 </td><td>2192 </td><td>46.21 </td><td><em>0</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00668 </td><td>0.01423 </td><td>Crenarchaeota </td></tr><tr><td>12 </td><td><em>Methanobrevibacter smithii</em> ATCC 35061 </td><td>1854 </td><td>31.02 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02048 </td><td>0.03768 </td><td rowspan="19">Euryarchaeota </td></tr><tr><td>13 </td><td><em>Methanocaldococcus jannaschii</em> DSM 2661 </td><td>1666 </td><td>31.4 </td><td>0.132 </td><td>0.031 </td><td>0.00450 </td><td>0.01128 </td></tr><tr><td>14 </td><td><em>Methanococcoides burtonii</em> DSM 6242 </td><td>2576 </td><td>40.74 </td><td>0.078 </td><td><em>0.002</em> </td><td>0.00266 </td><td>0.00845 </td></tr><tr><td>15 </td><td><em>Methanococcus aeolicus</em> Nankai-3 </td><td>1570 </td><td>30.02 </td><td>0.218 </td><td>0.52 </td><td>0.00399 </td><td>0.00063 </td></tr><tr><td>16 </td><td><em>Methanococcus maripaludis</em> C5 </td><td>1781 </td><td>32.99 </td><td><em>0.001</em> </td><td>0.065 </td><td>0.00846 </td><td>0.00454 </td></tr><tr><td>17 </td><td><em>Methanococcus maripaludis</em> C6 </td><td>1745 </td><td>33.4 </td><td>0.045 </td><td>0.045 </td><td>0.00553 </td><td>0.00224 </td></tr><tr><td>18 </td><td><em>Methanococcus maripaludis</em> C7 </td><td>1773 </td><td>33.27 </td><td>0.256 </td><td>0.784 </td><td>0.00430 </td><td>0.00088 </td></tr><tr><td>19 </td><td><em>Methanococcus maripaludis</em> S2 </td><td>1662 </td><td>33.08 </td><td>0.021 </td><td>0.08 </td><td>0.00619 </td><td>0.00259 </td></tr><tr><td>20 </td><td><em>Methanococcus vannielii</em> SB </td><td>1721 </td><td>31.31 </td><td>0.505 </td><td>0.519 </td><td>0.00364 </td><td>0.00400 </td></tr><tr><td>21 </td><td><em>Methanocorpusculum labreanum</em> Z </td><td>1806 </td><td>49.97 </td><td>0.606 </td><td>0.05 </td><td>0.00097 </td><td>0.00404 </td></tr><tr><td>22 </td><td><em>Methanoculleus marisnigri</em> JR1 </td><td>2479 </td><td>62.04 </td><td>0.816 </td><td>0.745 </td><td>0.00234 </td><td>0.00000 </td></tr><tr><td>23 </td><td><em>Methanopyrus kandleri</em> AV19 </td><td>1696 </td><td>61.22 </td><td>0.556 </td><td>0.032 </td><td>0.00230 </td><td>0.00471 </td></tr><tr><td>24 </td><td><em>Methanosaeta thermophila</em> PT </td><td>1880 </td><td>53.53 </td><td>0.673 </td><td><em>0.004</em> </td><td>0.00018 </td><td>0.00595 </td></tr><tr><td>25 </td><td><em>Methanosarcina acetivorans</em> C2A </td><td>5752 </td><td>42.67 </td><td><strong>&lt;0.0001</strong> </td><td>0.839 </td><td>0.00628 </td><td>0.00083 </td></tr><tr><td>26 </td><td><em>Methanosarcina barkeri</em> Fusaro </td><td>4838 </td><td>39.27 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.003</em> </td><td>0.00475 </td><td>0.00391 </td></tr><tr><td>27 </td><td><em>Methanosarcina mazei</em> Goe1 </td><td>4097 </td><td>41.47 </td><td>0.252 </td><td>0.812 </td><td>0.00212 </td><td>0.00079 </td></tr><tr><td>28 </td><td><em>Methanosphaera stadtmanae</em> DSM 3091 </td><td>1768 </td><td>27.62 </td><td><em>0.002</em> </td><td>0.275 </td><td>0.00897 </td><td>0.00652 </td></tr><tr><td>29 </td><td><em>Methanospirillum hungatei</em> JF-1 </td><td>3545 </td><td>45.14 </td><td><strong>&lt;0.0001</strong> </td><td>0.015 </td><td>0.00951 </td><td>0.00411 </td></tr><tr><td>30 </td><td><em>Methanothermobacter thermautotrophicus</em> Delta H </td><td>1752 </td><td>49.52 </td><td>0.022 </td><td>0.114 </td><td>0.00566 </td><td>0.00166 </td></tr><tr><td>31 </td><td><em>Nanoarchaeum equitans</em> Kin4-M </td><td>491 </td><td>31.55 </td><td>0.549 </td><td>0.177 </td><td>0.00000 </td><td>0.00127 </td><td>Nanoarchaeota </td></tr><tr><td>32 </td><td><em>Natronomonas pharaonis</em> DSM 2160 </td><td>2596 </td><td>63.42 </td><td>0.473 </td><td>0.228 </td><td>0.00174 </td><td>0.00091 </td><td>Euryarchaeota </td></tr><tr><td>33 </td><td><em>Nitrosopumilus maritimus</em> SCM1 </td><td>1646 </td><td>31.15 </td><td><strong>&lt;0.0001</strong> </td><td><em>0.002</em> </td><td>0.00921 </td><td>0.00855 </td><td>Crenarchaeota </td></tr><tr><td>34 </td><td><em>Picrophilus torridus</em> DSM 9790 </td><td>1546 </td><td>35.96 </td><td>0.296 </td><td><em>0.008</em> </td><td>0.00096 </td><td>0.00793 </td><td>Euryarchaeota </td></tr><tr><td>35 </td><td><em>Pyrobaculum aerophilum</em> IM2 </td><td>2223 </td><td>51.34 </td><td><em>0.001</em> </td><td><strong>&lt;0.0001</strong> </td><td>0.00727 </td><td>0.01022 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>36 </td><td><em>Pyrobaculum arsenaticum</em> DSM 13514 </td><td>2122 </td><td>54.98 </td><td>0.795 </td><td>0.431 </td><td>0.00138 </td><td>0.00316 </td></tr><tr><td>37 </td><td><em>Pyrobaculum calidifontis</em> JCM 11548 </td><td>2010 </td><td>57.13 </td><td>0.148 </td><td>0.337 </td><td>0.00294 </td><td>0.00008 </td></tr><tr><td>38 </td><td><em>Pyrobaculum islandicum</em> DSM 4184 </td><td>1827 </td><td>49.58 </td><td>0.305 </td><td>0.436 </td><td>0.00085 </td><td>0.00183 </td></tr><tr><td>39 </td><td><em>Pyrococcus abyssi</em> </td><td>1766 </td><td>44.69 </td><td>0.652 </td><td>0.574 </td><td>0.00219 </td><td>0.00342 </td><td rowspan="3">Euryarchaeota </td></tr><tr><td>40 </td><td><em>Pyrococcus furiosus</em> DSM 3638 </td><td>1909 </td><td>40.75 </td><td>0.754 </td><td>0.757 </td><td>0.00004 </td><td>0.00094 </td></tr><tr><td>41 </td><td><em>Pyrococcus horikoshii</em> OT3 </td><td>1739 </td><td>41.86 </td><td>0.133 </td><td><em>0.002</em> </td><td>0.00229 </td><td>0.01262 </td></tr><tr><td>42 </td><td><em>Staphylothermus marinus</em> F1 </td><td>1571 </td><td>35.71 </td><td><strong>&lt;0.0001</strong> </td><td><strong>&lt;0.0001</strong> </td><td>0.02078 </td><td>0.02726 </td><td rowspan="4">Crenarchaeota </td></tr><tr><td>43 </td><td><em>Sulfolobus acidocaldarius</em> DSM 639 </td><td>2227 </td><td>36.69 </td><td>0.413 </td><td>0.526 </td><td>0.00309 </td><td>0.00124 </td></tr><tr><td>44 </td><td><em>Sulfolobus solfataricus</em> P2 </td><td>2993 </td><td>35.77 </td><td><em>0.007</em> </td><td>0.747 </td><td>0.00533 </td><td>0.00241 </td></tr><tr><td>45 </td><td><em>Sulfolobus tokodaii</em> 7 </td><td>2695 </td><td>32.78 </td><td><em>0.005</em> </td><td>0.029 </td><td>0.00521 </td><td>0.00659 </td></tr><tr><td>46 </td><td><em>Thermococcus kodakarensis</em> KOD1 </td><td>2089 </td><td>51.98 </td><td>0.062 </td><td>0.328 </td><td>0.00418 </td><td>0.00160 </td><td>Euryarchaeota </td></tr><tr><td>47 </td><td><em>Thermofilum pendens</em> Hrk 5 </td><td>1782 </td><td>57.66 </td><td>0.014 </td><td><em>0.005</em> </td><td>0.00346 </td><td>0.00665 </td><td>Crenarchaeota </td></tr><tr><td>48 </td><td><em>Thermoplasma acidophilum</em> DSM 1728 </td><td>1565 </td><td>45.99 </td><td>0.016 </td><td>0.016 </td><td>0.00680 </td><td>0.00383 </td><td rowspan="2">Euryarchaeota </td></tr><tr><td>49 </td><td><em>Thermoplasma volcanium</em> GSS1 </td><td>1585 </td><td>39.91 </td><td>0.055 </td><td>0.361 </td><td>0.00404 </td><td>0.00263 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Chromosomes of archaea analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but ≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T)and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273961" target="_blank">View Large</a></div></div>                        <a id="5273962" scrollto-destination="5273962"></a>
<div content-id="DSP021TB3" class="table-modal"><div class="table-wrap"><div id="DSP021TB3" class="table-wrap-title" data-id="DSP021TB3"><div class="label -label">Table 3</div><div class="caption"><p>ISFDP analysis in eukaryotes chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Eukaryotes group </th></tr></thead><tbody><tr><td>1 </td><td><em>Guillardia theta</em> nucleomorph chromosome 01 </td><td>197 </td><td>25.64 </td><td>0.411 </td><td>0.468 </td><td>0.00080 </td><td>0.00517 </td><td rowspan="3">Cryptophyta </td></tr><tr><td>2 </td><td><em>Guillardia theta</em> nucleomorph chromosome 02 </td><td>181 </td><td>26.7 </td><td>0.435 </td><td>0.35 </td><td>0.00451 </td><td>0.00356 </td></tr><tr><td>3 </td><td><em>Guillardia theta</em> nucleomorph chromosome 03 </td><td>175 </td><td>26.81 </td><td>0.671 </td><td>0.403 </td><td>0.00051 </td><td>0.00622 </td></tr><tr><td>4 </td><td><em>Leishmania major</em> Friedlin chromosome 01 </td><td>270 </td><td>62.84 </td><td>0.055 </td><td><strong>&lt;0.0001</strong> </td><td>0.01290 </td><td>0.02500 </td><td>Euglenozoa </td></tr><tr><td>5 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 01 </td><td>644 </td><td>20.52 </td><td><em>0.001</em> </td><td>0.69 </td><td>0.02184 </td><td>0.01210 </td><td rowspan="6">Alveolata </td></tr><tr><td>6 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 05 </td><td>1344 </td><td>19.32 </td><td><em>0.006</em> </td><td><em>0.005</em> </td><td>0.01288 </td><td>0.01482 </td></tr><tr><td>7 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 11 </td><td>2036 </td><td>18.95 </td><td>0.043 </td><td>0.027 </td><td>0.00339 </td><td>0.00994 </td></tr><tr><td>8 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 12 </td><td>2272 </td><td>19.31 </td><td>0.05 </td><td>0.677 </td><td>0.00597 </td><td>0.00376 </td></tr><tr><td>9 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 13 </td><td>2733 </td><td>19.11 </td><td>0.105 </td><td>0.266 </td><td>0.00422 </td><td>0.00914 </td></tr><tr><td>10 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 14 </td><td>3292 </td><td>18.43 </td><td>0.258 </td><td>0.062 </td><td>0.00275 </td><td>0.00730 </td></tr><tr><td>11 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 01 </td><td>231 </td><td>39.14 </td><td>0.731 </td><td>0.088 </td><td>0.00100 </td><td>0.01231 </td><td rowspan="8">Fungi </td></tr><tr><td>12 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 04 </td><td>1532 </td><td>37.9 </td><td>0.807 </td><td>0.379 </td><td>0.00240 </td><td>0.00345 </td></tr><tr><td>13 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 07 </td><td>1091 </td><td>38.05 </td><td>0.285 </td><td>0.85 </td><td>0.00136 </td><td>0.00080 </td></tr><tr><td>14 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 12 </td><td>1079 </td><td>38.44 </td><td>0.055 </td><td>0.461 </td><td>0.00325 </td><td>0.00173 </td></tr><tr><td>15 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 15 </td><td>1092 </td><td>38.13 </td><td>0.181 </td><td>0.64 </td><td>0.00584 </td><td>0.00387 </td></tr><tr><td>16 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 01 </td><td>5574 </td><td>36.09 </td><td>0.4 </td><td>0.076 </td><td>0.00073 </td><td>0.00086 </td></tr><tr><td>17 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 02 </td><td>4510 </td><td>35.92 </td><td>0.461 </td><td>0.825 </td><td>0.00207 </td><td>0.00039 </td></tr><tr><td>18 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 03 </td><td>2453 </td><td>36.23 </td><td>0.152 </td><td>0.012 </td><td>0.00217 </td><td>0.00369 </td></tr></tbody></table></div><div content-id="DSP021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Eukaryotes group </th></tr></thead><tbody><tr><td>1 </td><td><em>Guillardia theta</em> nucleomorph chromosome 01 </td><td>197 </td><td>25.64 </td><td>0.411 </td><td>0.468 </td><td>0.00080 </td><td>0.00517 </td><td rowspan="3">Cryptophyta </td></tr><tr><td>2 </td><td><em>Guillardia theta</em> nucleomorph chromosome 02 </td><td>181 </td><td>26.7 </td><td>0.435 </td><td>0.35 </td><td>0.00451 </td><td>0.00356 </td></tr><tr><td>3 </td><td><em>Guillardia theta</em> nucleomorph chromosome 03 </td><td>175 </td><td>26.81 </td><td>0.671 </td><td>0.403 </td><td>0.00051 </td><td>0.00622 </td></tr><tr><td>4 </td><td><em>Leishmania major</em> Friedlin chromosome 01 </td><td>270 </td><td>62.84 </td><td>0.055 </td><td><strong>&lt;0.0001</strong> </td><td>0.01290 </td><td>0.02500 </td><td>Euglenozoa </td></tr><tr><td>5 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 01 </td><td>644 </td><td>20.52 </td><td><em>0.001</em> </td><td>0.69 </td><td>0.02184 </td><td>0.01210 </td><td rowspan="6">Alveolata </td></tr><tr><td>6 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 05 </td><td>1344 </td><td>19.32 </td><td><em>0.006</em> </td><td><em>0.005</em> </td><td>0.01288 </td><td>0.01482 </td></tr><tr><td>7 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 11 </td><td>2036 </td><td>18.95 </td><td>0.043 </td><td>0.027 </td><td>0.00339 </td><td>0.00994 </td></tr><tr><td>8 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 12 </td><td>2272 </td><td>19.31 </td><td>0.05 </td><td>0.677 </td><td>0.00597 </td><td>0.00376 </td></tr><tr><td>9 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 13 </td><td>2733 </td><td>19.11 </td><td>0.105 </td><td>0.266 </td><td>0.00422 </td><td>0.00914 </td></tr><tr><td>10 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 14 </td><td>3292 </td><td>18.43 </td><td>0.258 </td><td>0.062 </td><td>0.00275 </td><td>0.00730 </td></tr><tr><td>11 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 01 </td><td>231 </td><td>39.14 </td><td>0.731 </td><td>0.088 </td><td>0.00100 </td><td>0.01231 </td><td rowspan="8">Fungi </td></tr><tr><td>12 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 04 </td><td>1532 </td><td>37.9 </td><td>0.807 </td><td>0.379 </td><td>0.00240 </td><td>0.00345 </td></tr><tr><td>13 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 07 </td><td>1091 </td><td>38.05 </td><td>0.285 </td><td>0.85 </td><td>0.00136 </td><td>0.00080 </td></tr><tr><td>14 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 12 </td><td>1079 </td><td>38.44 </td><td>0.055 </td><td>0.461 </td><td>0.00325 </td><td>0.00173 </td></tr><tr><td>15 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 15 </td><td>1092 </td><td>38.13 </td><td>0.181 </td><td>0.64 </td><td>0.00584 </td><td>0.00387 </td></tr><tr><td>16 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 01 </td><td>5574 </td><td>36.09 </td><td>0.4 </td><td>0.076 </td><td>0.00073 </td><td>0.00086 </td></tr><tr><td>17 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 02 </td><td>4510 </td><td>35.92 </td><td>0.461 </td><td>0.825 </td><td>0.00207 </td><td>0.00039 </td></tr><tr><td>18 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 03 </td><td>2453 </td><td>36.23 </td><td>0.152 </td><td>0.012 </td><td>0.00217 </td><td>0.00369 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Chromosomes of eukaryotes analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but ≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T) and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting that the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273962" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP021TB3" class="table-wrap-title" data-id="DSP021TB3"><div class="label -label">Table 3</div><div class="caption"><p>ISFDP analysis in eukaryotes chromosomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Eukaryotes group </th></tr></thead><tbody><tr><td>1 </td><td><em>Guillardia theta</em> nucleomorph chromosome 01 </td><td>197 </td><td>25.64 </td><td>0.411 </td><td>0.468 </td><td>0.00080 </td><td>0.00517 </td><td rowspan="3">Cryptophyta </td></tr><tr><td>2 </td><td><em>Guillardia theta</em> nucleomorph chromosome 02 </td><td>181 </td><td>26.7 </td><td>0.435 </td><td>0.35 </td><td>0.00451 </td><td>0.00356 </td></tr><tr><td>3 </td><td><em>Guillardia theta</em> nucleomorph chromosome 03 </td><td>175 </td><td>26.81 </td><td>0.671 </td><td>0.403 </td><td>0.00051 </td><td>0.00622 </td></tr><tr><td>4 </td><td><em>Leishmania major</em> Friedlin chromosome 01 </td><td>270 </td><td>62.84 </td><td>0.055 </td><td><strong>&lt;0.0001</strong> </td><td>0.01290 </td><td>0.02500 </td><td>Euglenozoa </td></tr><tr><td>5 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 01 </td><td>644 </td><td>20.52 </td><td><em>0.001</em> </td><td>0.69 </td><td>0.02184 </td><td>0.01210 </td><td rowspan="6">Alveolata </td></tr><tr><td>6 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 05 </td><td>1344 </td><td>19.32 </td><td><em>0.006</em> </td><td><em>0.005</em> </td><td>0.01288 </td><td>0.01482 </td></tr><tr><td>7 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 11 </td><td>2036 </td><td>18.95 </td><td>0.043 </td><td>0.027 </td><td>0.00339 </td><td>0.00994 </td></tr><tr><td>8 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 12 </td><td>2272 </td><td>19.31 </td><td>0.05 </td><td>0.677 </td><td>0.00597 </td><td>0.00376 </td></tr><tr><td>9 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 13 </td><td>2733 </td><td>19.11 </td><td>0.105 </td><td>0.266 </td><td>0.00422 </td><td>0.00914 </td></tr><tr><td>10 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 14 </td><td>3292 </td><td>18.43 </td><td>0.258 </td><td>0.062 </td><td>0.00275 </td><td>0.00730 </td></tr><tr><td>11 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 01 </td><td>231 </td><td>39.14 </td><td>0.731 </td><td>0.088 </td><td>0.00100 </td><td>0.01231 </td><td rowspan="8">Fungi </td></tr><tr><td>12 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 04 </td><td>1532 </td><td>37.9 </td><td>0.807 </td><td>0.379 </td><td>0.00240 </td><td>0.00345 </td></tr><tr><td>13 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 07 </td><td>1091 </td><td>38.05 </td><td>0.285 </td><td>0.85 </td><td>0.00136 </td><td>0.00080 </td></tr><tr><td>14 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 12 </td><td>1079 </td><td>38.44 </td><td>0.055 </td><td>0.461 </td><td>0.00325 </td><td>0.00173 </td></tr><tr><td>15 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 15 </td><td>1092 </td><td>38.13 </td><td>0.181 </td><td>0.64 </td><td>0.00584 </td><td>0.00387 </td></tr><tr><td>16 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 01 </td><td>5574 </td><td>36.09 </td><td>0.4 </td><td>0.076 </td><td>0.00073 </td><td>0.00086 </td></tr><tr><td>17 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 02 </td><td>4510 </td><td>35.92 </td><td>0.461 </td><td>0.825 </td><td>0.00207 </td><td>0.00039 </td></tr><tr><td>18 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 03 </td><td>2453 </td><td>36.23 </td><td>0.152 </td><td>0.012 </td><td>0.00217 </td><td>0.00369 </td></tr></tbody></table></div><div content-id="DSP021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Serial number </th><th>Strain name </th><th>Size (kb) </th><th>GC% </th><th>KS (W) </th><th>KS (S) </th><th>|(∑A − ∑T)|/(∑A + ∑T) </th><th>|(∑G − ∑C)|/(∑G + ∑C) </th><th>Eukaryotes group </th></tr></thead><tbody><tr><td>1 </td><td><em>Guillardia theta</em> nucleomorph chromosome 01 </td><td>197 </td><td>25.64 </td><td>0.411 </td><td>0.468 </td><td>0.00080 </td><td>0.00517 </td><td rowspan="3">Cryptophyta </td></tr><tr><td>2 </td><td><em>Guillardia theta</em> nucleomorph chromosome 02 </td><td>181 </td><td>26.7 </td><td>0.435 </td><td>0.35 </td><td>0.00451 </td><td>0.00356 </td></tr><tr><td>3 </td><td><em>Guillardia theta</em> nucleomorph chromosome 03 </td><td>175 </td><td>26.81 </td><td>0.671 </td><td>0.403 </td><td>0.00051 </td><td>0.00622 </td></tr><tr><td>4 </td><td><em>Leishmania major</em> Friedlin chromosome 01 </td><td>270 </td><td>62.84 </td><td>0.055 </td><td><strong>&lt;0.0001</strong> </td><td>0.01290 </td><td>0.02500 </td><td>Euglenozoa </td></tr><tr><td>5 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 01 </td><td>644 </td><td>20.52 </td><td><em>0.001</em> </td><td>0.69 </td><td>0.02184 </td><td>0.01210 </td><td rowspan="6">Alveolata </td></tr><tr><td>6 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 05 </td><td>1344 </td><td>19.32 </td><td><em>0.006</em> </td><td><em>0.005</em> </td><td>0.01288 </td><td>0.01482 </td></tr><tr><td>7 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 11 </td><td>2036 </td><td>18.95 </td><td>0.043 </td><td>0.027 </td><td>0.00339 </td><td>0.00994 </td></tr><tr><td>8 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 12 </td><td>2272 </td><td>19.31 </td><td>0.05 </td><td>0.677 </td><td>0.00597 </td><td>0.00376 </td></tr><tr><td>9 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 13 </td><td>2733 </td><td>19.11 </td><td>0.105 </td><td>0.266 </td><td>0.00422 </td><td>0.00914 </td></tr><tr><td>10 </td><td><em>Plasmodium falciparum</em> 3D7 chromosome 14 </td><td>3292 </td><td>18.43 </td><td>0.258 </td><td>0.062 </td><td>0.00275 </td><td>0.00730 </td></tr><tr><td>11 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 01 </td><td>231 </td><td>39.14 </td><td>0.731 </td><td>0.088 </td><td>0.00100 </td><td>0.01231 </td><td rowspan="8">Fungi </td></tr><tr><td>12 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 04 </td><td>1532 </td><td>37.9 </td><td>0.807 </td><td>0.379 </td><td>0.00240 </td><td>0.00345 </td></tr><tr><td>13 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 07 </td><td>1091 </td><td>38.05 </td><td>0.285 </td><td>0.85 </td><td>0.00136 </td><td>0.00080 </td></tr><tr><td>14 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 12 </td><td>1079 </td><td>38.44 </td><td>0.055 </td><td>0.461 </td><td>0.00325 </td><td>0.00173 </td></tr><tr><td>15 </td><td><em>Saccharomyces cerevisiae</em> S288C chromosome 15 </td><td>1092 </td><td>38.13 </td><td>0.181 </td><td>0.64 </td><td>0.00584 </td><td>0.00387 </td></tr><tr><td>16 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 01 </td><td>5574 </td><td>36.09 </td><td>0.4 </td><td>0.076 </td><td>0.00073 </td><td>0.00086 </td></tr><tr><td>17 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 02 </td><td>4510 </td><td>35.92 </td><td>0.461 </td><td>0.825 </td><td>0.00207 </td><td>0.00039 </td></tr><tr><td>18 </td><td><em>Schizosaccharomyces pombe</em> 972h chromosome 03 </td><td>2453 </td><td>36.23 </td><td>0.152 </td><td>0.012 </td><td>0.00217 </td><td>0.00369 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Chromosomes of eukaryotes analyzed in this study. The KS test for significance between the frequency distribution of complementary nucleotide values are given as KS (W) between A and T and KS (S) between G and C. In bacteria, archaea and eukaryotes, <em>P</em>-values of &lt;10<sup>−4</sup> (strong violation of ISFDP) are shown in bold and <em>P</em>-values of &lt;0.01 but ≥10<sup>−4</sup> (weak violation of ISFDP) are shown in italics. The <em>P</em>-value between 10<sup>−4</sup> and 10<sup>−3</sup> is shown as 0.000. Relative absolute abundance value difference between the complementary nucleotides is given by |(∑A − ∑T)|/(∑A + ∑T) and |(∑G − ∑C)|/(∑G + ∑C) for ATS and GCS, respectively. In chromosome of <em>X. fastidiosa</em> 9a5c, the GCS/ATS value is highest suggesting that the difference between the abundance values of complementary nucleotides is high. The <em>P</em>-value by the KS test is in concordant with the ATS/GCS suggesting that the abundance difference can be represented by the frequency distribution study of the nucleotides. Similar relation is also observed in other chromosomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273962" target="_blank">View Large</a></div></div><p>Out of 112 bacterial chromosomes, 60 chromosomes exhibited ISFDP, 16 chromosomes exhibited violation between S nucleotides as well as between W nucleotides, 30 chromosomes exhibited violation only between S nucleotides and 7 chromosomes exhibited violation only between W nucleotides (Table <a reveal-id="DSP021TB4" data-open="DSP021TB4" class="link link-reveal link-table xref-fig">4</a>). Chromosomes of <em>Alkaliphilus oremlandii</em> OhILAs (36.26%), <em>Agrobacterium tumefaciens</em> C58 (circular; 59.38%), <em>Mycobacterium ulcerans</em> Agy99 (65.47%), <em>Staphylococcus epidermidis</em> ATCC 12228 (32.1%) and <em>X. fastidiosa</em> 9a5c (52.68%) exhibited strong violations between S nucleotides as well as between W nucleotides. Chromosomes of the three <em>Bacillus anthracis</em> (35.35%) strains, <em>Lactobacillus reuteri</em> F275 (38.87%), <em>Magnetococcus</em> sp. MC-1 (54.17%), <em>Mycobacterium leprae</em> TN (57.8%), <em>Rhizobium leguminosarum</em> bv. <em>viciae</em> 3841 (61.09%) and <em>Rickettsia bellii</em> RML369-C (31.65%) exhibited strong violation between S nucleotides as well as weak violation between W nucleotides. Chromosomes of <em>Coxiella burnetii</em> Dugway 7E9-12 (42.44%) and <em>Staphylococcus haemolyticus</em> JCSC1435 (32.79%) exhibited weak violation between S nucleotides as well as between W nucleotides. Chromosome of <em>Vibrio cholerae</em> O395 (47.78%) exhibited strong violation of ISFDP only between W nucleotides. Similarly, there are six chromosomes where weak violations only between W nucleotides were observed. Chromosomes of <em>Bacillus thuringiensis</em> serovar konkukian 97-27 (34.41%), <em>Bordetella parapertussis</em> 12822 (68.1%), <em>Bordetella pertussis</em> Tohama 1 (67.72%), <em>Haemophilus influenzae</em> PittGG (38.01%), <em>Helicobacter hepaticus</em> ATCC 51449 (35.93%), <em>Lactobacillus acidophilus</em> NCFM (34.72%), <em>Lactobacillus brevis</em> ATCC 367 (46.22%), <em>Nitrobacter winogradskyi</em> Nb-255 (62.05%), <em>Ralstonia solanacearum</em> GMI1000 chromosome (67.04%), <em>Rhizobium etli</em> CFN 42 (61.27%), <em>Thermotoga maritima</em> MSB8 (46.25%) and <em>Thermotoga petrophila</em> RKU-1 (46.09%) exhibited strong violation only between S nucleotides. Similarly there are 17 chromosomes exhibited weak violation only between S nucleotides. An interesting finding that came from this study is that violations of ISFDP within a chromosome with respect to S and W nucleotides may not be of similar magnitudes. This study suggests that although ISFDP is commonly observed among chromosomes, its violation is not as rare as described earlier.<sup><a reveal-id="DSP021C13" data-open="DSP021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> ISFDP violation found in bacteria belongs to different groups, possessing different GC% and with different genome sizes.</p>                        <a id="5273964" scrollto-destination="5273964"></a>
<div content-id="DSP021TB4" class="table-modal"><div class="table-wrap"><div id="DSP021TB4" class="table-wrap-title" data-id="DSP021TB4"><div class="label -label">Table 4</div><div class="caption"><p>Summary of ISFDP violations in chromosomes of Bacteria, Archaea and Eukaryotes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism </th><th>Number of chromosomes </th><th>Number of chromosomes exhibiting ISFDP for both W and S </th><th>Number of chromosomes violating* ISFDP for both W and S </th><th>Number of chromosomes violating ISFDP only between S nucleotides </th><th>Number of chromosomes violating ISFDP only between W nucleotides </th></tr></thead><tbody><tr><td>Bacteria </td><td>112 </td><td>60 </td><td>15 (5<sup>a</sup>+8<sup>b</sup>+0<sup>c</sup>+2<sup>d</sup>) </td><td>30 (13<sup>e</sup>+17<sup>f</sup>) </td><td>07 (1<sup>g</sup>+6<sup>h</sup>) </td></tr><tr><td>Archaea </td><td>49 </td><td>30 </td><td>06 (2<sup>a</sup>+2<sup>b</sup>+2<sup>c</sup>+0<sup>d</sup>) </td><td>06 (0<sup>e</sup>+6<sup>f</sup>) </td><td>07 (2<sup>g</sup>+5<sup>h</sup>) </td></tr><tr><td>Eukaryotes </td><td>18 </td><td>15 </td><td>01 (0<sup>a</sup>+0<sup>b</sup>+0<sup>c</sup>+1<sup>d</sup>) </td><td>01 (1<sup>e</sup>+0<sup>f</sup>) </td><td>01 (0<sup>g</sup>+1<sup>h</sup>) </td></tr></tbody></table></div><div content-id="DSP021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism </th><th>Number of chromosomes </th><th>Number of chromosomes exhibiting ISFDP for both W and S </th><th>Number of chromosomes violating* ISFDP for both W and S </th><th>Number of chromosomes violating ISFDP only between S nucleotides </th><th>Number of chromosomes violating ISFDP only between W nucleotides </th></tr></thead><tbody><tr><td>Bacteria </td><td>112 </td><td>60 </td><td>15 (5<sup>a</sup>+8<sup>b</sup>+0<sup>c</sup>+2<sup>d</sup>) </td><td>30 (13<sup>e</sup>+17<sup>f</sup>) </td><td>07 (1<sup>g</sup>+6<sup>h</sup>) </td></tr><tr><td>Archaea </td><td>49 </td><td>30 </td><td>06 (2<sup>a</sup>+2<sup>b</sup>+2<sup>c</sup>+0<sup>d</sup>) </td><td>06 (0<sup>e</sup>+6<sup>f</sup>) </td><td>07 (2<sup>g</sup>+5<sup>h</sup>) </td></tr><tr><td>Eukaryotes </td><td>18 </td><td>15 </td><td>01 (0<sup>a</sup>+0<sup>b</sup>+0<sup>c</sup>+1<sup>d</sup>) </td><td>01 (1<sup>e</sup>+0<sup>f</sup>) </td><td>01 (0<sup>g</sup>+1<sup>h</sup>) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Violation of ISFDP includes both weak (10<sup>−2</sup> &gt; <em>P</em> ≥ 10<sup>−4</sup>) and strong (<em>P</em> &lt; 10<sup>−4</sup>).</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Strong violation between S nucleotides as well as between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Strong violation between S nucleotides but weak violation between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Weak violation between S nucleotides but strong violation between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Weak violation between S nucleotides as well as between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Strong violation only between S nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Weak violation only between S nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Strong violation only between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>h</sup>Weak violation only between W nucleotides.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273964" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP021TB4" class="table-wrap-title" data-id="DSP021TB4"><div class="label -label">Table 4</div><div class="caption"><p>Summary of ISFDP violations in chromosomes of Bacteria, Archaea and Eukaryotes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism </th><th>Number of chromosomes </th><th>Number of chromosomes exhibiting ISFDP for both W and S </th><th>Number of chromosomes violating* ISFDP for both W and S </th><th>Number of chromosomes violating ISFDP only between S nucleotides </th><th>Number of chromosomes violating ISFDP only between W nucleotides </th></tr></thead><tbody><tr><td>Bacteria </td><td>112 </td><td>60 </td><td>15 (5<sup>a</sup>+8<sup>b</sup>+0<sup>c</sup>+2<sup>d</sup>) </td><td>30 (13<sup>e</sup>+17<sup>f</sup>) </td><td>07 (1<sup>g</sup>+6<sup>h</sup>) </td></tr><tr><td>Archaea </td><td>49 </td><td>30 </td><td>06 (2<sup>a</sup>+2<sup>b</sup>+2<sup>c</sup>+0<sup>d</sup>) </td><td>06 (0<sup>e</sup>+6<sup>f</sup>) </td><td>07 (2<sup>g</sup>+5<sup>h</sup>) </td></tr><tr><td>Eukaryotes </td><td>18 </td><td>15 </td><td>01 (0<sup>a</sup>+0<sup>b</sup>+0<sup>c</sup>+1<sup>d</sup>) </td><td>01 (1<sup>e</sup>+0<sup>f</sup>) </td><td>01 (0<sup>g</sup>+1<sup>h</sup>) </td></tr></tbody></table></div><div content-id="DSP021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism </th><th>Number of chromosomes </th><th>Number of chromosomes exhibiting ISFDP for both W and S </th><th>Number of chromosomes violating* ISFDP for both W and S </th><th>Number of chromosomes violating ISFDP only between S nucleotides </th><th>Number of chromosomes violating ISFDP only between W nucleotides </th></tr></thead><tbody><tr><td>Bacteria </td><td>112 </td><td>60 </td><td>15 (5<sup>a</sup>+8<sup>b</sup>+0<sup>c</sup>+2<sup>d</sup>) </td><td>30 (13<sup>e</sup>+17<sup>f</sup>) </td><td>07 (1<sup>g</sup>+6<sup>h</sup>) </td></tr><tr><td>Archaea </td><td>49 </td><td>30 </td><td>06 (2<sup>a</sup>+2<sup>b</sup>+2<sup>c</sup>+0<sup>d</sup>) </td><td>06 (0<sup>e</sup>+6<sup>f</sup>) </td><td>07 (2<sup>g</sup>+5<sup>h</sup>) </td></tr><tr><td>Eukaryotes </td><td>18 </td><td>15 </td><td>01 (0<sup>a</sup>+0<sup>b</sup>+0<sup>c</sup>+1<sup>d</sup>) </td><td>01 (1<sup>e</sup>+0<sup>f</sup>) </td><td>01 (0<sup>g</sup>+1<sup>h</sup>) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>*Violation of ISFDP includes both weak (10<sup>−2</sup> &gt; <em>P</em> ≥ 10<sup>−4</sup>) and strong (<em>P</em> &lt; 10<sup>−4</sup>).</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Strong violation between S nucleotides as well as between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Strong violation between S nucleotides but weak violation between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Weak violation between S nucleotides but strong violation between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Weak violation between S nucleotides as well as between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>Strong violation only between S nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>Weak violation only between S nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>g</sup>Strong violation only between W nucleotides.</p></div></div><div content-id=""><div class="fn"><p><sup>h</sup>Weak violation only between W nucleotides.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5273964" target="_blank">View Large</a></div></div><p>Usually, different strains within a species are found to be similar with respect to ISFDP such as the eight <em>E. coli</em> strains were observed to exhibit ISFDP between S nucleotides as well as between W nucleotides, the three <em>B. anthracis</em> strains are found to be similar in terms of their ISFDP violation (strong violation of ISFDP between S nucleotides as well as weak violations of ISFDP between W nucleotides). However, variation among the strains of a bacterial species with respect to ISFDP was observed as follows: out of the two strains of <em>C. burnetii</em>, Dugway 7E9-12 strain violated ISFDP, whereas RSA 493 strain exhibited ISFDP. Out of the four <em>H. influenza</em> strains, 86-028NP and PittEE exhibited violation of ISFDP, whereas PittGG and Rd KW20 exhibited strong and weak violations only between S nucleotides, respectively. <em>Xylella fastidiosa</em> 9a5c exhibited strong violation of ISFDP, whereas <em>X. fastidiosa</em> Temecula 1 exhibited weak violation of ISFDP only between S nucleotides. These are called as intra-species ISFDP violations. Chromosomes of four species of <em>Mycobacterium</em> genus exhibited a large difference among each other with respect to ISFDP. Chromosome of <em>Mycobacterium</em> sp. KMS (68.44%) exhibited parity between S nucleotides as well as between W nucleotides, whereas chromosome of <em>M. ulcerans</em> Agy99 (65.47%) exhibited strong violation of the parity between S nucleotides as well as between W nucleotides.</p>                            <h3 scrollto-destination=5273966 id="5273966" class="section-title" data-legacy-id=s3b>ISFDP in chromosomes of archaea and eukaryotes</h3>
<p>Out of the 49 archaea chromosomes, 30 exhibited ISFDP, 6 exhibited violations of it between S nucleotides as well as between W nucleotides, 6 exhibited violations only between S nucleotides and 7 exhibited violations only between W nucleotides (Table <a reveal-id="DSP021TB4" data-open="DSP021TB4" class="link link-reveal link-table xref-fig">4</a>). Chromosomes of <em>Methanobrevibacter smithii</em> ATCC 35061 (31.02%) and <em>Staphylothermus marinus</em> F1 (35.71%) exhibited strong violation of ISFDP between S nucleotides as well as between W nucleotides. Chromosomes of <em>Metallosphaera sedula</em> DSM 5348 (46.21%) and <em>Pyrobaculum aerophilum</em> IM2 (51.34%) exhibited strong violations between S nucleotides but weak violations between W nucleotides. Strong violation between W nucleotides and weak violation between S nucleotides were observed in chromosomes of <em>Methanosarcina barkeri</em> Fusaro (39.27%) and <em>Nitrosopumilus maritimus</em> SCM1 (31.15%). This suggests that within a chromosome, the magnitude of parity violation between S nucleotides may be different from that between W nucleotides in archaea also like that of bacteria. Intra-species parity violation was also observed in archaea in the case of <em>Methanococcus maripaludis</em>. The C5 strain exhibited ISFDP violation between W nucleotides but exhibited parity between S nucleotides. The C6, C7 and S2 strains exhibited ISFDP between S nucleotides as well as between W nucleotides.</p><p>Out of the 18 eukaryotic chromosomes belonging to five species, 15 chromosomes exhibited ISFDP (Table <a reveal-id="DSP021TB4" data-open="DSP021TB4" class="link link-reveal link-table xref-fig">4</a>). Strong violation of ISFDP only between S nucleotides is observed in <em>Leishmania major</em> Friedlin chromosome 01 (62.84%). <em>Plasmodium falciparum</em> 3D7chromosome 05 exhibited weak violation of parity between S nucleotides as well as between W nucleotides, whereas chromosome 01 exhibited violation of parity only between W nucleotides. The other four chromosomes of <em>P. falciparum</em> exhibited parity between S nucleotides as well as between W nucleotides. Similarly, the eight chromosomes of <em>Saccharomyces cerevisiae</em> even though exhibited parity between S nucleotides as well as between W nucleotides, the <em>P</em>-values either for S nucleotides or for W nucleotides is of more than 10-fold difference among the chromosomes. This differential ISFDP violation observed among chromosomes of an organism suggests that there may not be any strict rule inside a cell to maintain ISFDP.</p>                            <h3 scrollto-destination=5273969 id="5273969" class="section-title" data-legacy-id=s3c>ISFDP between complementary oligonucleotides in chromosomes</h3>
<p>ISP between compositional abundance values of complimentary oligonucleotides is well reported. We studied here the frequency distribution of complementary di- and trinucleotides in chromosomes as described for mononucleotides. The smooth curves of oligonucleotide frequencies have been shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>a and b, the frequency distributions of dinucleotides have been shown for <em>E. coli</em> K12 MG1655 and <em>Pseudomonas entomophila</em> L48 chromosome (64.16%). Out of the 12 smooth frequency curves (four palindromic dinucleotides were excluded), overlapping of the curves between complementary dinucleotides is observed. In Fig. <a reveal-id="DSP021F2" data-open="DSP021F2" class="link link-reveal link-table xref-fig">2</a>, though the abundance values of aa, tt, tg and ca dinucleotides in <em>E. coli</em> chromosome are close, the distributions between the complementary dinucleotides are found only overlapping and that of the non-complementary ones are different. The distributions for aa and tt follow a higher standard deviation (values not shown) than that of tg and ca. Similarly, gg and cc dinucleotides distributions exhibit a higher standard deviation (values not shown) than that of the dinucleotides tc and ga, although the abundance values of the four dinucleotides are close to each other. The significance of the similarity was studied by the KS test which suggested that the frequency distributions between complementary dinucleotides are statistically similar. Apart from this, dinucleotides distribution parity has been studied in three more bacterial chromosomes, two archaea chromosomes and one eukaryotic chromosome (data not shown) and similar result has been observed. In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>i and ii, the distribution of 22 trinucleotides of <em>E. coli</em> K12 MG1655 chromosome is shown. Like dinucleotides, overlapping between the distributions of complementary trinucleotides is also observed. Distribution similarity between complementary trinucleotides was studied by the KS test for the 64 trinucleotides which suggested that the distributions of complementary trinucleotides within a strand are similar. The same study was done in one more bacterial chromosome (data not shown) and similar results were obtained. Although we did not analyzed the chromosomes of archaea and eukaryotes for trinucleotide distribution parity, it is expected to be there because these chromosomes had exhibited ISFDP for mononucleotides as well as dinucleotides.</p>                        <a id="5273972" scrollto-destination="5273972"></a>
<div id="DSP021F2" data-id="DSP021F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02102.jpeg?Expires=2147483647&Signature=MpoOLSE8FMllfUweUZL4Z3464cYDSPny0RQFsN3cJeuW78bQHkUp~gbedVdgI4Gd~sAC2F9I-SKLJnUCXN1pRiE58UdmXUS8P3OP54KNAUxF3NiigrZB~JkxwHqjfZRJ0CqqWoUIAkFdPuNlWXKiF2JQE15dbC4F6aceOKd10cxxplR-pJhGixxCrtJpX~hmw9kB7ASx9oTV2M-XOrMgXtiFprZ5btSuNgXSBAbZNvlvwY0KPtdtFfDNzLDDHOa9Eo6DP~4Thwl0hqHS~FJ~VyGg7V5PzPBm08H~7hS-HlHZc~SLeXWtJd0K8Dl-nKDFXq4-Ttt17BxSvVSEZ87prw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of coding sequence arrangement studied. In the upper row, the entire DNA strand is composed of forward encoded sequences (black color). Parity is not observed in this case. In the lower row, the DNA strand is made up of 50% forward encoded sequences and the other 50% is the reverse encoded sequences (white color). Parity is observed in this case." path-from-xml="dsp02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273972/dsp02102.jpeg" path-from-xml="dsp02102.jpeg" target="_blank">View large</a><a section="5273972" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02102.jpeg?Expires=2147483647&Signature=BLWeGhnppuKNc1M6Ymxst6CEREmDdVpkmDDSWj7nGocO46vUp7ob6VkqfwR3cyionZGsAS8WdpGDDGLzpKbCaRcgZ2PPBfoaObwBTEF8WvjfPGXF8taRtRcphG4J-O-d4b5FRNBmSliKnAYEUMgasOZTZ9UUhZ10yJLmj0LG0y1aZ4wxxYnREnySu5S~yV4or1lw3RM7afE~qsEFSoXAs4c5GHBlDr9BeWBOOEUBEzhwo9Z8Fu29ygkwRwplT50M480BK5v5smnw6UMmaoACFzm2zp~k0hTQTI6t5XWqandkfwyxv9cm3sxEXxBQ-IpghA3pqrelGgwYlTgBb9F8Gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273972&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic representation of coding sequence arrangement studied. In the upper row, the entire DNA strand is composed of forward encoded sequences (black color). Parity is not observed in this case. In the lower row, the DNA strand is made up of 50% forward encoded sequences and the other 50% is the reverse encoded sequences (white color). Parity is observed in this case.</p></div></div><div content-id="DSP021F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02102.jpeg?Expires=2147483647&Signature=MpoOLSE8FMllfUweUZL4Z3464cYDSPny0RQFsN3cJeuW78bQHkUp~gbedVdgI4Gd~sAC2F9I-SKLJnUCXN1pRiE58UdmXUS8P3OP54KNAUxF3NiigrZB~JkxwHqjfZRJ0CqqWoUIAkFdPuNlWXKiF2JQE15dbC4F6aceOKd10cxxplR-pJhGixxCrtJpX~hmw9kB7ASx9oTV2M-XOrMgXtiFprZ5btSuNgXSBAbZNvlvwY0KPtdtFfDNzLDDHOa9Eo6DP~4Thwl0hqHS~FJ~VyGg7V5PzPBm08H~7hS-HlHZc~SLeXWtJd0K8Dl-nKDFXq4-Ttt17BxSvVSEZ87prw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic representation of coding sequence arrangement studied. In the upper row, the entire DNA strand is composed of forward encoded sequences (black color). Parity is not observed in this case. In the lower row, the DNA strand is made up of 50% forward encoded sequences and the other 50% is the reverse encoded sequences (white color). Parity is observed in this case." path-from-xml="dsp02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273972/dsp02102.jpeg" path-from-xml="dsp02102.jpeg" target="_blank">View large</a><a section="5273972" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02102.jpeg?Expires=2147483647&Signature=BLWeGhnppuKNc1M6Ymxst6CEREmDdVpkmDDSWj7nGocO46vUp7ob6VkqfwR3cyionZGsAS8WdpGDDGLzpKbCaRcgZ2PPBfoaObwBTEF8WvjfPGXF8taRtRcphG4J-O-d4b5FRNBmSliKnAYEUMgasOZTZ9UUhZ10yJLmj0LG0y1aZ4wxxYnREnySu5S~yV4or1lw3RM7afE~qsEFSoXAs4c5GHBlDr9BeWBOOEUBEzhwo9Z8Fu29ygkwRwplT50M480BK5v5smnw6UMmaoACFzm2zp~k0hTQTI6t5XWqandkfwyxv9cm3sxEXxBQ-IpghA3pqrelGgwYlTgBb9F8Gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273972&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic representation of coding sequence arrangement studied. In the upper row, the entire DNA strand is composed of forward encoded sequences (black color). Parity is not observed in this case. In the lower row, the DNA strand is made up of 50% forward encoded sequences and the other 50% is the reverse encoded sequences (white color). Parity is observed in this case.</p></div></div>                            <h3 scrollto-destination=5273975 id="5273975" class="section-title" data-legacy-id=s3d>ISFDP weakly correlates with Chargaff's second parity</h3>
<p>Comparison of ISFDP was done with the ATS/GCS in chromosomes to find out whether one can define the other. GCS was compared with ISFDP violation between S nucleotides and ATS was compared with ISFDP violation between W nucleotides. Among the bacterial chromosomes, maximum GCS was found in <em>X. fastidiosa</em> 9a5c with the value 0.0529. All of the 16 chromosomes with GCS ≥0.01 were found to violate ISFDP (14 strongly violated and 2 weakly violated). Out of the 18 chromosomes with GCS ≥0.005 but &lt;0.01, 6 exhibited insignificant violation, 7 exhibited strong violation and 5 exhibited weak violation of ISFDP. Similarly, out of 56 chromosomes with GCS ≥0.001 but &lt;0.005, 5 exhibited strong violation, 11 exhibited weak violation and 40 exhibited insignificant violation. Out of the 22 chromosomes with GCS &lt;0.001, except <em>B. thuringiensis</em> Al Hakam chromosome (with GCS value 0.00081 exhibited weak violation of ISFDP) all other exhibited insignificant violation. Maximum ATS was found in <em>X. fastidiosa</em> 9a5c with the value 0.04727. Out of the five chromosomes with ATS ≥0.01, four were found to violate ISFDP (two strongly violated and two weakly violated), whereas <em>Mycoplasma hyopneumoniae</em> J exhibited insignificant violation (with ATS 0.0102). Out of the 14 chromosomes with ATS ≥0.005 but &lt;0.01, 6 exhibited insignificant violation, 3 exhibited strong violation and 5 exhibited weak violation of ISFDP. Out of the 67 chromosomes with ATS ≥0.001 but &lt;0.005, 57 exhibited parity, 1 strongly violated and 9 violated weakly between the W nucleotides. All the 26 chromosomes with ATS ≤0.001 exhibited insignificant violation of ISFDP. These results suggest that chromosomes with high ATS/GCS (≥0.01) have a stronger propensity to violate ISFDP and chromosomes with low ATS/GCS (≤0.001) have a stronger propensity to exhibit ISFDP. However, chromosomes with intermediate ATS/GCS (≥0.001 and ≤0.01) have the possibility of either exhibiting parity or violating the parity.</p><p>Correlation analysis was done between the <em>P</em>-values (from the KS test between) of W nucleotides and ATS as well as between the <em>P</em>-values (from the KS test between) of S nucleotides and GCS. The <em>r</em>-values are −0.5572 and −0.4526 for W and S nucleotides, respectively. This suggests that the correlation between the two ISP features is weak. The correlation between ATS and GCS is 0.629, which suggests that parity violation between S nucleotides weakly correlates with parity violation between W nucleotides within a chromosome. Unlike ATS and GCS correlation, no correlation was found between the <em>P</em>-values (the KS test) of W nucleotides and that of S nucleotides, which supports that ISFDP and Chargaff's second parity are not the same.</p><p>In the case of the archaea chromosomes, the ISFDP analysis revealed similar results to that of bacterial chromosomes. Maximum GCS with the value 0.03768 was found in the chromosome of <em>M. smithii</em> ATCC 35061 (31.02%) followed by the value 0.02726 in <em>S. marinus</em> F1 (35.71%), in which significant ISFDP violation was also observed. In the GCS interval 0.005 &lt; GCS ≤ 0.01, there were eight chromosomes out of which five exhibited weak violation and three exhibited insignificant violation of ISFDP. Out of the 24 chromosomes in the interval 0.001 &lt; GCS ≤ 0.005, 2 exhibited weak violation and 22 exhibited insignificant violation of ISFDP. These results suggest that chromosomes with high ATS/GCS (≥0.01) are most likely going to violate ISFDP and chromosomes with low ATS/GCS (≤0.001) are most likely going to exhibit ISFDP. However, chromosomes with intermediate ATS/GCS ((≥0.001 and≤0.01) have the possibility of either exhibiting parity or violating the parity. Pearson's correlation coefficient between ATS and GCS was found to be 0.707847, which is similar to that of the bacterial analysis. The <em>r</em>-values between ATS and the <em>P</em>-values of KS (W) as well as GCS and the <em>P</em>-values of KS (S) were found to be −0.57495 and −0.47557, respectively, suggesting a weak correlation.</p>                            <h3 scrollto-destination=5273981 id="5273981" class="section-title" data-legacy-id=s3e>The chromosomes with asymmetric replication topography are more prone to ISFDP violation in bacteria</h3>
<p>Bacterial chromosome is a single replicon. Owing to the bidirectional mode of replication, one part of a strand is synthesized as LeS whereas the other part is synthesized as LaS. In most of the chromosomes, the mutational strand asymmetry causes K nucleotides &gt; M nucleotides in LeS and the reverse in (K nucleotides &lt; M nucleotides) in LaS. In an ideal case where the termination site is located symmetrically with respect to the origin of replication in a chromosome, the excess of K nucleotides in LeS will be similar to the excess of M nucleotides in LaS and therefore will cancel each other to exhibit Chargaff's second parity in chromosomes. Potential replication origin and termination sites for different chromosomes based on ATS, GCS, coding sequence skew, nucleotide skew at the third position of codons and oligonucleotides skew in chromosomes have been reported,<sup><a reveal-id="DSP021C31" data-open="DSP021C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSP021C32" data-open="DSP021C32" class="link link-ref link-reveal xref-bibr">32</a></sup> which has been reviewed in detail.<sup><a reveal-id="DSP021C33" data-open="DSP021C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Out of the 112 bacterial chromosomes analyzed in this study, information regarding the potential site for the origin and termination of 56 chromosomes is available. ISFDP violation between S nucleotides was compared with the angular deviation of termination site because G &gt; C in LeS is a more universal feature of chromosomes than T &gt; A in LeS. Of the 112 chromosomes, maximum angular deviation of 71.28° is reported in <em>B. pertussis</em> Tohama 1. Out of the 14 chromosomes where ≥20° angular deviation was observed, 12 exhibited violation of ISFDP between S nucleotides. <em>Pseudomonas putida</em> F1 (61.86%) with 36.8° and <em>C. burnetii</em> RSA 493 (42.66%) with 31.14° angular deviations exhibited insignificant parity violation. Out of the 11 chromosomes with deviation ≥10° but &lt;20°, 4 chromosomes exhibited ISFDP violation between S nucleotides. Out of the 30 strains with deviation ≥1.0° and ≤10°, 9 chromosomes exhibited parity violation between S nucleotides. <em>Chlamydophila abortus</em> S263 with angular deviation only 0.569°, parity violation was observed only between S nucleotides. This study indicates that chromosomes with higher asymmetric topography are more prone to violate the parity. However, chromosomes with symmetric replication topography were also observed to violate the parity.</p><p>The correlation coefficient between angular deviations and GCS as well as ATS values are 0.474 and 0.357, respectively, suggesting a weak correlation. The correlation between angular deviations and <em>P</em>-value of S (the KS test between S nucleotides) as well as that of W (the KS test between W nucleotides) are −0.259 and −0.048, respectively. The angular deviation in <em>X. fastidiosa</em> 9a5c is 62.96°, whereas the same in Temecula 1 is 6.44°. The difference in the magnitude of ISFDP violation between the strains might be attributed to the chromosome topography. Comparison for the four <em>H. influenzae</em> strains could not be done due to the unavailability of information for all the strains. The Rd KW20 chromosome (that violated ISFDP) has the angular deviation 46°, which might be an important factor to violate ISFDP. Archaea chromosomes have been reported to have more replication origin like eukaryotic chromosomes. Therefore, replication topography will not be applicable to study ISFDP violations in these cases.</p>                            <h3 scrollto-destination=5273984 id="5273984" class="section-title" data-legacy-id=s3f>Composition of forward- and reverse-encoded sequences within DNA strands might influence the parity</h3>
<p>Most of the regions in prokaryotic chromosomes are composed of coding sequences. Presence of both forward- and reverse-encoded sequences in bacterial chromosomes has been proposed for the observation of Chargaff's second parity in chromosomes.<sup><a reveal-id="DSP021C8" data-open="DSP021C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSP021C9" data-open="DSP021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> So we analyzed only coding sequences in chromosomes of bacteria and archaea to study ISFDP as follows (Fig. <a reveal-id="DSP021F2" data-open="DSP021F2" class="link link-reveal link-table xref-fig">2</a>): in one way (Case I), a DNA strand is only composed of only forward-encoded sequences, and in the other way (Case II), a DNA strand is composed of 50% forward-encoded and 50% reverse-encoded sequences. The result is shown for <em>E. coli</em> chromosome (Fig. <a reveal-id="DSP021F3" data-open="DSP021F3" class="link link-reveal link-table xref-fig">3</a>A and B). The smooth frequency curves of complementary nucleotides overlap in Fig. <a reveal-id="DSP021F3" data-open="DSP021F3" class="link link-reveal link-table xref-fig">3</a>B, whereas in Fig. <a reveal-id="DSP021F3" data-open="DSP021F3" class="link link-reveal link-table xref-fig">3</a>A, they do not overlap. The significance of these overlaps were studied by the KS test which suggests that the similarity between the distribution of complementary nucleotides in Case II. Similar results were obtained by the analysis of several bacterial (10) and archaea (15) chromosomes.</p>                        <a id="5273987" scrollto-destination="5273987"></a>
<div id="DSP021F3" data-id="DSP021F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02103.jpeg?Expires=2147483647&Signature=pabe0YmoM08SwaMYF97r5oAvnMttJWNYzvDccM3lLIjG4dMLTCUntQssYV40r~QvmQmhsOHtiKZ4T1tud5m0JNK8vKXvyeHsn0QsS9ybaSiCoVrkGC1e~~VRxcpL62QQfRN0pnxM6JhubOg0wecm3lpnapyAsyhVKuUGdEMuwRW9z1GQwECoXsJrTNHcdGDeuVARXjRKX97JeAPLDiSIfWN95mm5iYZ0cOewxmmIa9E6ER5IB4daL62JcV450zd2ylRdhwaihB6hd49wpAc-PPrqvh6tynEAEI~b-OgpKkYSR~dfy~A5YS-RiNC4oabaFZErwfOaG-cTGlbsoQ87NQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A and B) Frequency distribution study of nucleotides in coding sequences. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The X-axis represents the abundance values of the nucleotide spanning a range, whereas the Y-axis represents the frequency of the abundance values. In (A), the frequency of the nucleotides in a DNA strand only composed of forward encoded sequences of E. coli is shown (coding sequences analyzed for other chromosomes exhibited the similar feature). It is evident from (B) that the frequency distributions of the complementary nucleotides do not overlap. In (B), the frequency of the nucleotides of the same DNA strand done where 50% of the sequence was joined with the rest after reverse complementation (see the Materials and methods section). This resembled a strand composed of 50% forward encoded sequences and 50% reverse encoded sequences. It is evident from the figures that parity between the complementary nucleotides is observed in this case. These observations have been confirmed by the KS test." path-from-xml="dsp02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273987/dsp02103.jpeg" path-from-xml="dsp02103.jpeg" target="_blank">View large</a><a section="5273987" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02103.jpeg?Expires=2147483647&Signature=FhD0fG-c82M6r11dNchbxqmjHl7dloo5vQF6fmMUF-TF-SKYNIeAkqwq9HYgVMuPSI1ejBZVMfdNHLG~WHxC6WkyYsAqWxgjA4mmfl0jxmU-k6WIcK4scrltHUEaai-cy5EX0-7dBlGszlyHMNJuygHuRGfPZL7j8b3~X7dZ8knA1pYIZXUQuUavezJZvR1vt2DOcLXHRBzvxD~1-h1yzcjHhdQyRpc~cJsNqzRhN0r2sQYetFRXp8GmJomq0M5pTPdYHBmjC7nQqLs166XmW9HjykUdtxQ2UU9wUwC7jVDl3CFaX-~8oo4ZhMkWU-URlOO-xnj4CDJXl2NybJU1Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273987&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A and B) Frequency distribution study of nucleotides in coding sequences. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The <em>X</em>-axis represents the abundance values of the nucleotide spanning a range, whereas the <em>Y</em>-axis represents the frequency of the abundance values. In (A), the frequency of the nucleotides in a DNA strand only composed of forward encoded sequences of <em>E. coli</em> is shown (coding sequences analyzed for other chromosomes exhibited the similar feature). It is evident from (B) that the frequency distributions of the complementary nucleotides do not overlap. In (B), the frequency of the nucleotides of the same DNA strand done where 50% of the sequence was joined with the rest after reverse complementation (see the Materials and methods section). This resembled a strand composed of 50% forward encoded sequences and 50% reverse encoded sequences. It is evident from the figures that parity between the complementary nucleotides is observed in this case. These observations have been confirmed by the KS test.</p></div></div><div content-id="DSP021F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02103.jpeg?Expires=2147483647&Signature=pabe0YmoM08SwaMYF97r5oAvnMttJWNYzvDccM3lLIjG4dMLTCUntQssYV40r~QvmQmhsOHtiKZ4T1tud5m0JNK8vKXvyeHsn0QsS9ybaSiCoVrkGC1e~~VRxcpL62QQfRN0pnxM6JhubOg0wecm3lpnapyAsyhVKuUGdEMuwRW9z1GQwECoXsJrTNHcdGDeuVARXjRKX97JeAPLDiSIfWN95mm5iYZ0cOewxmmIa9E6ER5IB4daL62JcV450zd2ylRdhwaihB6hd49wpAc-PPrqvh6tynEAEI~b-OgpKkYSR~dfy~A5YS-RiNC4oabaFZErwfOaG-cTGlbsoQ87NQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A and B) Frequency distribution study of nucleotides in coding sequences. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The X-axis represents the abundance values of the nucleotide spanning a range, whereas the Y-axis represents the frequency of the abundance values. In (A), the frequency of the nucleotides in a DNA strand only composed of forward encoded sequences of E. coli is shown (coding sequences analyzed for other chromosomes exhibited the similar feature). It is evident from (B) that the frequency distributions of the complementary nucleotides do not overlap. In (B), the frequency of the nucleotides of the same DNA strand done where 50% of the sequence was joined with the rest after reverse complementation (see the Materials and methods section). This resembled a strand composed of 50% forward encoded sequences and 50% reverse encoded sequences. It is evident from the figures that parity between the complementary nucleotides is observed in this case. These observations have been confirmed by the KS test." path-from-xml="dsp02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273987/dsp02103.jpeg" path-from-xml="dsp02103.jpeg" target="_blank">View large</a><a section="5273987" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02103.jpeg?Expires=2147483647&Signature=FhD0fG-c82M6r11dNchbxqmjHl7dloo5vQF6fmMUF-TF-SKYNIeAkqwq9HYgVMuPSI1ejBZVMfdNHLG~WHxC6WkyYsAqWxgjA4mmfl0jxmU-k6WIcK4scrltHUEaai-cy5EX0-7dBlGszlyHMNJuygHuRGfPZL7j8b3~X7dZ8knA1pYIZXUQuUavezJZvR1vt2DOcLXHRBzvxD~1-h1yzcjHhdQyRpc~cJsNqzRhN0r2sQYetFRXp8GmJomq0M5pTPdYHBmjC7nQqLs166XmW9HjykUdtxQ2UU9wUwC7jVDl3CFaX-~8oo4ZhMkWU-URlOO-xnj4CDJXl2NybJU1Jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273987&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A and B) Frequency distribution study of nucleotides in coding sequences. Smooth curves present the group-frequency distribution of the four nucleotides a (square), t (asterisk), g (triangle) and c (rhombus). The <em>X</em>-axis represents the abundance values of the nucleotide spanning a range, whereas the <em>Y</em>-axis represents the frequency of the abundance values. In (A), the frequency of the nucleotides in a DNA strand only composed of forward encoded sequences of <em>E. coli</em> is shown (coding sequences analyzed for other chromosomes exhibited the similar feature). It is evident from (B) that the frequency distributions of the complementary nucleotides do not overlap. In (B), the frequency of the nucleotides of the same DNA strand done where 50% of the sequence was joined with the rest after reverse complementation (see the Materials and methods section). This resembled a strand composed of 50% forward encoded sequences and 50% reverse encoded sequences. It is evident from the figures that parity between the complementary nucleotides is observed in this case. These observations have been confirmed by the KS test.</p></div></div><p>A comparative analysis between the Ws and Cs in a chromosome with respect to their composition of forward-encoded sequences was done in <em>X. fastidiosa</em> species as well as in <em>H. influenza</em> species. The relative differences in the compositional abundance values of forward sequences in Ws and Cs of <em>X. fastidiosa</em> 9a5c and <em>X. fastidiosa</em> Temecula 1 chromosomes are 0.078 and 0.015, respectively, which indicate that the proportion of forward- and reverse-encoded sequence in 9a5c strain is more disproportionate than that of Temecula 1 strain, which might be the reason for a stronger parity violation in the former. The relative differences of the compositional abundance values of forward-encoded sequences in Ws and Cs of <em>H. influenzae</em> 86-028NP (exhibits parity) and <em>H. influenzae</em> Rd KW20 (violates parity) chromosomes are 0.030 and 0.005, respectively, which suggest that the proportion of forward- and reverse-encoded sequences in 86-028NP strain is more disproportionate than that of Rd KW20 strain. This is in contrast to the result of <em>X. fastidiosa</em>, i.e. parity violation is observed in the strain (Rd KW20) with more proportionate gene distribution between Ws and Cs, whereas insignificant parity violation is observed in chromosome with disproportionate gene distribution between the strands. A quantitative estimation of the coding sequences in both the strands of the chromosomes was done in few other bacteria and archaea such as <em>A. tumefaciens</em>, <em>B. subtilis</em>, <em>E. coli</em>, <em>M. smithii</em> and <em>S. marinus</em> (Fig. <a reveal-id="DSP021F4" data-open="DSP021F4" class="link link-reveal link-table xref-fig">4</a>). Maximum difference of ORF numbers between Ws and Cs was found in <em>S. marinus</em>, in which the parity violation was also observed. However, the relative difference of ORFs between the strands is found more in <em>B. subtilis</em> than in <em>M. smithii</em>. The former exhibited the parity whereas the latter violated it. <em>Agrobacterium tumefaciens</em> was shown to possess minimum relative difference of ORF numbers between the strands but violates parity. The results from this indicate that a higher disproportionate composition of forward- and reverse-encoded sequences within a strand has greater propensity to parity violation. However, proportionate composition of the sequences not necessarily implies the exhibition of parity.</p>                        <a id="5273993" scrollto-destination="5273993"></a>
<div id="DSP021F4" data-id="DSP021F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02104.jpeg?Expires=2147483647&Signature=Q5UIIJ5Om4EFThoSKhtg~JcLaFYE-sPnHZhSgeKfuZvwzjPd~xoM839o6lRXRgG2p~47C9OoqXHdxgt6sS5q-Pf-u65KE25j3FY-ja10GeWiMDEBH0g1Oj-tKiu0STERIef6J69YNG-5yYbkQj9u1IKzUoJCfRn19C~JXveRVMsLTgZyRSGfezrLRhCZ0Uyrwyfexv~sM3VDBVRjLljtQIoDEirBeAgWPh-qFgU5XvGt1sdbQ24EVOLw2Z1lq~hfEll088Q7SvzfcAdS7mZPP8vLN5dmIv-YN1mkLzSGOAx3BcI~Rfcj1O0YhiDi79FbiE-DPCdOeWLEAHKLw6RS-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative disproportionate composition of ORFs between Ws and Cs in chromosomes. The composition of ORFs in Ws and Cs of seven bacteria and two archaea was studied. Relative disproportionate composition was found out by deducting the ORF numbers between the two strands and then dividing the value obtained by the total number of ORFs present in both the strands. In A. tumefaciens, relative disproportionate value found to be minimum suggesting that the difference in the number of ORFs between the strands is relatively minimum when compared with others. In the archaea S. marinus, the value is found to be maximum among these nine strains. Both A. tumefaciens and S. marinus exhibited ISFDP violations, whereas insignificant ISFDP violation observed between E. coli and B. subtilis. Comparison between the strains of X. fastidiosa and H. influenzae is shown." path-from-xml="dsp02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273993/dsp02104.jpeg" path-from-xml="dsp02104.jpeg" target="_blank">View large</a><a section="5273993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02104.jpeg?Expires=2147483647&Signature=DY3mk3aSnXAnhf~pcVMMEEDDww6dEnPGEWa6ZG2tXYacDEJNuC1vmCkAOjYmRJwnuVgiSttbCai2nRiQda3h4mmM2uHEjgexcOm7eo67dPjoi4L2eaiJ5lkCc5K0PCJDcosL6lgDsHuWfR7I8sFw99UsVLa6W8F7bsCcbInLwdWqoi79fwTdjp191jLFf99Vm9lQKCDNSi4cGXp~JWrvyDRVXAVGSfVmKyCP~1aZb0ZZNnHtlQuBRtEG~--hR7uU6YHN0BmbfNA~5TUazkMPaqmmC1VbsekAil2T-YneV-TnURM2FeRi3AfaESBK3Ug-EyuVDRSPOpL~IztMditAOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273993&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative disproportionate composition of ORFs between Ws and Cs in chromosomes. The composition of ORFs in Ws and Cs of seven bacteria and two archaea was studied. Relative disproportionate composition was found out by deducting the ORF numbers between the two strands and then dividing the value obtained by the total number of ORFs present in both the strands. In <em>A. tumefaciens</em>, relative disproportionate value found to be minimum suggesting that the difference in the number of ORFs between the strands is relatively minimum when compared with others. In the archaea <em>S. marinus</em>, the value is found to be maximum among these nine strains. Both <em>A. tumefaciens</em> and <em>S. marinus</em> exhibited ISFDP violations, whereas insignificant ISFDP violation observed between <em>E. coli</em> and <em>B. subtilis</em>. Comparison between the strains of <em>X. fastidiosa</em> and <em>H. influenzae</em> is shown.</p></div></div><div content-id="DSP021F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/m_dsp02104.jpeg?Expires=2147483647&Signature=Q5UIIJ5Om4EFThoSKhtg~JcLaFYE-sPnHZhSgeKfuZvwzjPd~xoM839o6lRXRgG2p~47C9OoqXHdxgt6sS5q-Pf-u65KE25j3FY-ja10GeWiMDEBH0g1Oj-tKiu0STERIef6J69YNG-5yYbkQj9u1IKzUoJCfRn19C~JXveRVMsLTgZyRSGfezrLRhCZ0Uyrwyfexv~sM3VDBVRjLljtQIoDEirBeAgWPh-qFgU5XvGt1sdbQ24EVOLw2Z1lq~hfEll088Q7SvzfcAdS7mZPP8vLN5dmIv-YN1mkLzSGOAx3BcI~Rfcj1O0YhiDi79FbiE-DPCdOeWLEAHKLw6RS-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative disproportionate composition of ORFs between Ws and Cs in chromosomes. The composition of ORFs in Ws and Cs of seven bacteria and two archaea was studied. Relative disproportionate composition was found out by deducting the ORF numbers between the two strands and then dividing the value obtained by the total number of ORFs present in both the strands. In A. tumefaciens, relative disproportionate value found to be minimum suggesting that the difference in the number of ORFs between the strands is relatively minimum when compared with others. In the archaea S. marinus, the value is found to be maximum among these nine strains. Both A. tumefaciens and S. marinus exhibited ISFDP violations, whereas insignificant ISFDP violation observed between E. coli and B. subtilis. Comparison between the strains of X. fastidiosa and H. influenzae is shown." path-from-xml="dsp02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5273993/dsp02104.jpeg" path-from-xml="dsp02104.jpeg" target="_blank">View large</a><a section="5273993" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp02104.jpeg?Expires=2147483647&Signature=DY3mk3aSnXAnhf~pcVMMEEDDww6dEnPGEWa6ZG2tXYacDEJNuC1vmCkAOjYmRJwnuVgiSttbCai2nRiQda3h4mmM2uHEjgexcOm7eo67dPjoi4L2eaiJ5lkCc5K0PCJDcosL6lgDsHuWfR7I8sFw99UsVLa6W8F7bsCcbInLwdWqoi79fwTdjp191jLFf99Vm9lQKCDNSi4cGXp~JWrvyDRVXAVGSfVmKyCP~1aZb0ZZNnHtlQuBRtEG~--hR7uU6YHN0BmbfNA~5TUazkMPaqmmC1VbsekAil2T-YneV-TnURM2FeRi3AfaESBK3Ug-EyuVDRSPOpL~IztMditAOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5273993&ar=363736&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative disproportionate composition of ORFs between Ws and Cs in chromosomes. The composition of ORFs in Ws and Cs of seven bacteria and two archaea was studied. Relative disproportionate composition was found out by deducting the ORF numbers between the two strands and then dividing the value obtained by the total number of ORFs present in both the strands. In <em>A. tumefaciens</em>, relative disproportionate value found to be minimum suggesting that the difference in the number of ORFs between the strands is relatively minimum when compared with others. In the archaea <em>S. marinus</em>, the value is found to be maximum among these nine strains. Both <em>A. tumefaciens</em> and <em>S. marinus</em> exhibited ISFDP violations, whereas insignificant ISFDP violation observed between <em>E. coli</em> and <em>B. subtilis</em>. Comparison between the strains of <em>X. fastidiosa</em> and <em>H. influenzae</em> is shown.</p></div></div>                            <h2 scrollto-destination=5273996 id="5273996" class="section-title" data-legacy-id=s4>Discussion</h2>
<p>We have described in this study a new ISP feature in chromosomes, which is found in bacteria, archaea and eukaryotes. The methodology used to study this parity gives the statistical significance of similarity between the two distributions of complementary nucleotides/oligonucleotides. The basic qualitative feature of ISFDP is not changing for a chromosome even the segmentation is done at randomly taking any point out of the first 1000 nucleotides as the starting point. In other words, the sampling fluctuation is not affecting the feature. The correlation between the ISFDP and ISP is not strong, which is in accordance with the view that similarity in the total abundance values of two complementary nucleotides will not always yield similarity in their frequency distribution pattern. However, violation of ISP will definitely exhibit violation of ISFDP. Around 50% of the chromosomes in bacteria are found to exhibit ISFDP violations. Chromosomes of <em>H. influenzae</em> Rd KW20, <em>M. tuberculosis</em> F11, etc., which have been reported to exhibit ISP, are found to violate ISFDP.<sup><a reveal-id="DSP021C27" data-open="DSP021C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p><p>ISFDP violation observed in all possible combinations in chromosomes: (i) violation of parity between S nucleotides as well as between W nucleotides; (ii) only between S nucleotides and only between W nucleotides. The correlation between ATS and GCS is found to be not strong suggesting that parity violation between S nucleotides not necessarily always associate with parity violations between W nucleotides and <em>vice versa</em>. This can be called as intra-chromosomal parity violations. ISFDP violations of different magnitudes were found among chromosomes of different strains belonging to a species which can be referred as intra-species parity violations. Examples are <em>C. burnetii</em>, <em>H. influenzae</em> and <em>X. fastidiosa</em>. These intra-chromosomal and intra-species violations suggest that there may not be any strict rule existing in cells to maintain ISFDP in chromosomes. Differential ISP among chromosomes within a species and between chromosomes within a bacterium has already been reported in <em>Chlamydophila pneumoniae</em> strains and <em>Deinococcus radiodurans</em> R1 chromosomes,<sup><a reveal-id="DSP021C27" data-open="DSP021C27" class="link link-ref link-reveal xref-bibr">27</a></sup> respectively. However, these were not considered significant in their study due to the lack of statistical proof. Oligonucleotide skew patterns also have been found to be variable among strains of <em>Yersinia pestis</em>. These intra-species variations in the chromosomal features are interesting and need in-depth analysis of the genome sequences to find out the reason that might reveal the reason for ISP/ISFDP violation in chromosomes and between the two ISP features.</p><p>Enrichment of LeS with K nucleotides over M nucleotides and the <em>vice versa</em> in LaS due to the mutational strand asymmetry is a general observation in chromosomes. Owing to the bidirectional replication, GCS/ATS in LeS is cancelled with GCS/ATS in LaS which results in the establishment of parity in chromosomes. The cancellation effect indirectly suggests that the compositional abundance values between the two complementary nucleotides even though they differ within a sub-chromosomal region. This is in support of the observation here that chromosomes with higher GCS/ATS values are violating ISFDP and chromosomes with lower GCS/ATS are exhibiting the parity. However, the chromosomes with intermediate range GCS/ATS are found to exhibit parity as well as violate parity and this violation is independent of genome GC%. For example, <em>Streptococcus mutans</em> UA159, <em>Rickettsia conorii</em> Malish 7, <em>C. jejuni</em> subsp. <em>jejuni</em> 81116, <em>Campylobacter concisus</em> 13826 and <em>Lactococcus lactis</em> subsp. <em>cremoris</em> MG1363, <em>Helicobacter pylori</em> J99 are (all AT-rich organisms) chromosomes with GCS≥0.005 that exhibit ISFDP between S nucleotides, whereas chromosomes of <em>B. anthracis</em> strains (AT rich) with similar GCS (&gt;0.005) violate ISFDP between S nucleotides. So ISFDP in these chromosomes is an interesting aspect of future research.</p><p>In concordant with the view of the bidirectional replication and establishment of parity in chromosomes, several chromosomes with higher asymmetric replication topography were found to violate ISFDP. The exceptions are <em>P. putida</em> F1 and <em>C. burnetii</em> RSA 493 chromosomes with 36° and 31° angular deviations, respectively. Chromosomes of <em>C. abortus</em> S263 and <em>Magnetospirillum magneticum</em> AMB-1, with very less angular deviations 0.57° and 2.14°, respectively, are found violating ISFDP. This indicates that features apart from the replication topography might contribute to the parity establishment in chromosomes. Proportionate composition of forward-encoded sequences between the two strands though thought to be responsible to establish the parity after the analysis of artificially constructed chromosomes, several observations went against it. The extreme case is <em>A. tumefaciens</em> where the composition is very much proportionate but violations of ISFDP are strong. So the two factors such as asymmetric replication topography and disproportionate composition of forward-encoded sequences between the strands in chromosomes that were assumed to play important roles in determining ISFDP violations were found to be insufficient.</p><p>In spite of different selection/mutation pressures on chromosomes as exemplified by codon usage,<sup><a reveal-id="DSP021C34" data-open="DSP021C34" class="link link-ref link-reveal xref-bibr">34</a></sup> replication topography,<sup><a reveal-id="DSP021C31" data-open="DSP021C31" class="link link-ref link-reveal xref-bibr">31</a></sup> isochores<sup><a reveal-id="DSP021C35" data-open="DSP021C35" class="link link-ref link-reveal xref-bibr">35</a></sup> and GCS/ATS,<sup><a reveal-id="DSP021C21" data-open="DSP021C21" class="link link-ref link-reveal xref-bibr">21</a></sup> the tendency of the chromosomes of all types toward maintaining the ISFDP is interesting. Since ISFDP and ISP are the outcomes of compositional abundance of nucleotides (mono/oligo), theories proposed for ISP might hold true for ISFDP. The Nussinov–Forsdyke hypothesis is that stem–loop potential has an adaptive advantage, and therefore an important factor driving the compositional symmetry (ISP) between the complementary oligonucleotides<sup><a reveal-id="DSP021C36" data-open="DSP021C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSP021C37" data-open="DSP021C37" class="link link-ref link-reveal xref-bibr">37</a></sup> has been challenged recently by Chen and Zhao<sup><a reveal-id="DSP021C38" data-open="DSP021C38" class="link link-ref link-reveal xref-bibr">38</a></sup> for human chromosomes. This indicates that the stem–loop (recombination) hypothesis might not be the only explanation for ISP in chromosomes. Baisnée <em>et al</em>.<sup><a reveal-id="DSP021C8" data-open="DSP021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> have argued that the reverse complement symmetry does not result only from point mutation or from recombination, but from a combination effect of different mechanisms at different orders.<sup><a reveal-id="DSP021C8" data-open="DSP021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Two independent reports have theoretically shown that multiple inversion events in chromosomes can establish ISP.<sup><a reveal-id="DSP021C10" data-open="DSP021C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP021C39" data-open="DSP021C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Though this hypothesis looks fine theoretically, frequent inversion unable to explain the universal observation of opposite GCS/ATS in LeS and LaS,<sup><a reveal-id="DSP021C40" data-open="DSP021C40" class="link link-ref link-reveal xref-bibr">40</a></sup> gene distribution asymmetry between the strands<sup><a reveal-id="DSP021C41" data-open="DSP021C41" class="link link-ref link-reveal xref-bibr">41</a></sup> and the maintenance of gene orders among different bacterial chromosomes.<sup><a reveal-id="DSP021C42" data-open="DSP021C42" class="link link-ref link-reveal xref-bibr">42</a></sup> This hypothesis also does not describe any functional significance/advantage of the ISP/ISFDP feature, which is so wide spread in chromosomes. Theoretically, it has also been argued that the mismatch error repairing system is responsible to establish Chargaff's second parity rule in chromosomes.<sup><a reveal-id="DSP021C13" data-open="DSP021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> However, the intra-chromosomal parity violation observed in eukaryotes (this study) goes against this hypothesis.</p><p>We think the important factor that determines ISP/ISFDP in chromosomes is the bidirectional replication. This causes one part of a strand Ws/Cs as LeS and the other part as LaS. The strand mutational asymmetry and gene distribution asymmetry between LeS and LaS therefore cancel out each other within the strand to exhibit the parity. In the case of ssDNA/ssRNA viruses, gene distribution is restricted to one strand only depending on which these are called as either plus or – strand viruses. The genome size is also not large (&lt;10 kb) in these phages<sup><a reveal-id="DSP021C43" data-open="DSP021C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSP021C44" data-open="DSP021C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and during replication, one strand only acts as the template on which the other strand is made. Most likely these features are responsible for violating the parity in these genomes. The advantages of bidirectional replication in bacteria and archaea where the nucleus is absent are as follows: (i) quicker completion of replication than the unidirectional mode of replication and (ii) the meeting of the two replication forks might be sending some signal to the cell for the completion of chromosome replication where the nucleus is absent. Symmetric replication topography will help to terminate the replication from the origin in a lesser time in comparison with an asymmetric topography. So the selection pressure to maintain the symmetric replication topography in fast-growing bacteria is likely to be more than that in slow-growing bacteria. This proposition has similarity with the Selection Mutation Drift theory proposed for codon usage<sup><a reveal-id="DSP021C45" data-open="DSP021C45" class="link link-ref link-reveal xref-bibr">45</a></sup> in bacteria. Our study of ISFDP of <em>Vibrio</em> species (the generation time is 0.2–0.3 h; fast-growing) in this context seems to be also not holding true here because its chromosomes violate ISFDP between W nucleotides. Moreover, comparison of generation time<sup><a reveal-id="DSP021C40" data-open="DSP021C40" class="link link-ref link-reveal xref-bibr">40</a></sup> with asymmetry in replication topography of chromosomes<sup><a reveal-id="DSP021C32" data-open="DSP021C32" class="link link-ref link-reveal xref-bibr">32</a></sup> exhibits no correlation (data not shown). More research on this aspect will give a conclusive result if the growth rate has any relation with parity establishment in chromosomes. In conclusion, our study has revealed an interesting aspect of ISP. Future research will reveal the reason for the presence of this parity in chromosomes.</p>                            <h2 scrollto-destination=5274008 id="5274008" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5274011 id="5274011" class="backacknowledgements-title" >Acknowledgements</h2>
<p>A part of this work has been presented as a poster by S.K.R. in the International Conference ISMB2008 at Toronoto, Canada. Support to S.K.R. from DST (India), INSA (India) and Tezpur University for attending this conference is thankfully acknowledged. We thank the department of Biotechnology, Govt. of India for awarding MSc student fellowships to A.K. and P.K.J. The authors thank to J. R. Lobry for his critical comments on the work and are very much grateful to D. R. Forsdyke (the reviewer of the manuscript) for his comments on the manuscript.</p>                            <h2 scrollto-destination=5274015 id="5274015" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chargaff</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Structure and function of nucleic acids as cell constituents</div>, <div class="source">Fed. Proc.</div> , <div class="year">1951</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">654</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20function%20of%20nucleic%20acids%20as%20cell%20constituents&amp;author=E.%20Chargaff&amp;publication_year=1951&amp;journal=Fed.%20Proc.&amp;volume=10&amp;pages=654-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14887699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Forsdyke</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Mortimer</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Chargaff's legacy</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">127</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chargaff's%20legacy&amp;author=D.R.%20Forsdyke&amp;author=J.R.%20Mortimer&amp;publication_year=2000&amp;journal=Gene&amp;volume=261&amp;pages=127-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00472-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00472-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00472-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11164044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Crick</div> <div class="given-names">F.H.C.</div></div>. </div><div class="article-title">Molecular structure of nucleic acids: a structure for deoxyribonucleic acid</div>, <div class="source">Nature</div> , <div class="year">1953</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">737</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20structure%20of%20nucleic%20acids%3A%20a%20structure%20for%20deoxyribonucleic%20acid&amp;author=J.D.%20Watson&amp;author=F.H.C.%20Crick&amp;publication_year=1953&amp;journal=Nature&amp;volume=171&amp;pages=737-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/171737a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F171737a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F171737a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13054692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karkas</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Rudner</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chargaff</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Separation of <em>B. subtilis</em> DNA into complementary strands, II. Template functions and composition as determined by transcription with RNA polymerases</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1968</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">915</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Separation%20of%20B.%20subtilis%20DNA%20into%20complementary%20strands%2C%20II.%20Template%20functions%20and%20composition%20as%20determined%20by%20transcription%20with%20RNA%20polymerases&amp;author=J.D.%20Karkas&amp;author=R.%20Rudner&amp;author=E.%20Chargaff&amp;publication_year=1968&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=60&amp;pages=915-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.60.3.915" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.60.3.915" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.60.3.915"> </span></div> </div></div></div></div></div><div content-id="DSP021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rudner</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Karkas</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Chargaff</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Separation of microbial deoxyribonucleic acids into complementary strands</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1969</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">152</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Separation%20of%20microbial%20deoxyribonucleic%20acids%20into%20complementary%20strands&amp;author=R.%20Rudner&amp;author=J.D.%20Karkas&amp;author=E.%20Chargaff&amp;publication_year=1969&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=63&amp;pages=152-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.63.1.152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.63.1.152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.63.1.152"> </span></div> </div></div></div></div></div><div content-id="DSP021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Prabhu</div> <div class="given-names">V.V.</div></div>. </div><div class="article-title">Symmetry observed in long nucleotide sequences</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">1993</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">2797</div>-<div class="lpage">800</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Symmetry%20observed%20in%20long%20nucleotide%20sequences&amp;author=V.V.%20Prabhu&amp;publication_year=1993&amp;journal=Nucleic%20Acid%20Res.&amp;volume=21&amp;pages=2797-800" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/21.12.2797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F21.12.2797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F21.12.2797"> </span></div> </div></div></div></div></div><div content-id="DSP021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Cuticchia</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Compositional symmetries in complete genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">557</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20symmetries%20in%20complete%20genomes&amp;author=D.%20Qi&amp;author=A.J.%20Cuticchia&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=557-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.6.557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.6.557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.6.557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11395434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baisnée</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Hampson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Baldi</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Why are complementary DNA strands symmetric?</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1021</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20are%20complementary%20DNA%20strands%20symmetric%3F&amp;author=P.F.%20Baisn%C3%A9e&amp;author=S.%20Hampson&amp;author=P.%20Baldi&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=1021-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.8.1021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.8.1021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.8.1021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176825" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Verma</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Das</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Satapathy</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Buragohain</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Ray</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">Compositional Symmetry of DNA duplex in bacterial genomes</div>, <div class="source">Curr. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">374</div>-<div class="lpage">84</div>)</div></div></div></div><div content-id="DSP021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Albrecht-Buehler</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Asymptotically increasing compliance of genomes with Chargaff's second parity rules through inversions and inverted transpositions</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">17828</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymptotically%20increasing%20compliance%20of%20genomes%20with%20Chargaff's%20second%20parity%20rules%20through%20inversions%20and%20inverted%20transpositions&amp;author=G.%20Albrecht-Buehler&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=17828-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0605553103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0605553103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0605553103"> </span></div> </div></div></div></div></div><div content-id="DSP021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mitchell</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bridge</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A test of Chargaff's second rule</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2006</div>, vol. <div class="volume">340</div> (pg. <div class="fpage">1</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20test%20of%20Chargaff's%20second%20rule&amp;author=D.%20Mitchell&amp;author=R.%20Bridge&amp;publication_year=2006&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=340&amp;pages=1-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2005.11.160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2005.11.160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2005.11.160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16343431" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nikolaou</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Almirantis</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Deviation from Chargaff's second parity rule in organellar DNA insights into the evolution of organellar genomes</div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">381</div> (pg. <div class="fpage">34</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deviation%20from%20Chargaff's%20second%20parity%20rule%20in%20organellar%20DNA%20insights%20into%20the%20evolution%20of%20organellar%20genomes&amp;author=C.%20Nikolaou&amp;author=Y.%20Almirantis&amp;publication_year=2006&amp;journal=Gene&amp;volume=381&amp;pages=34-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.06.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.06.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.06.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16893615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Deng</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Mismatch repair error implies Chargaff's second parity rule, arXiv:0704.2191v2 [q-bio.GN], 1–9</div>, <div class="year">2007</div> <div class="comment">[<a class="link link-uri" href="http://arxiv.org/PS_cache/arxiv/pdf/0704/0704.2191v2.pdf">http://arxiv.org/PS_cache/arxiv/pdf/0704/0704.2191v2.pdf</a>]</div></div></div></div></div><div content-id="DSP021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Intra-strand parity rules of DNA base composition and usage biases of synonymous codons</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">318</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intra-strand%20parity%20rules%20of%20DNA%20base%20composition%20and%20usage%20biases%20of%20synonymous%20codons&amp;author=N.%20Sueoka&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=40&amp;pages=318-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00163236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Properties of a general model of DNA evolution under no-strand-bias conditions</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">326</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20of%20a%20general%20model%20of%20DNA%20evolution%20under%20no-strand-bias%20conditions&amp;author=J.R.%20Lobry&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=40&amp;pages=326-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00163237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00163237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00163237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7723059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">65</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%3A%20a%20review%20of%20possible%20underlying%20mutational%20or%20selective%20mechanisms&amp;author=A.C.%20Frank&amp;author=J.R.%20Lobry&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=65-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00297-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00297-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00297-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nikolaou</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Almirantis</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">A study on the correlation of nucleotide skews and the positioning of the origin of replication: different modes of replication in bacterial species</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">6816</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20study%20on%20the%20correlation%20of%20nucleotide%20skews%20and%20the%20positioning%20of%20the%20origin%20of%20replication%3A%20different%20modes%20of%20replication%20in%20bacterial%20species&amp;author=C.%20Nikolaou&amp;author=Y.%20Almirantis&amp;publication_year=2005&amp;journal=Nucleic%20Acid%20Res.&amp;volume=33&amp;pages=6816-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki988" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki988" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki988"> </span></div> </div></div></div></div></div><div content-id="DSP021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Translation-coupled violation of parity rule 2 in human genes is not the cause of heterogeneity of the DNA G+C content of third codon position</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">53</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation-coupled%20violation%20of%20parity%20rule%202%20in%20human%20genes%20is%20not%20the%20cause%20of%20heterogeneity%20of%20the%20DNA%20G%2BC%20content%20of%20third%20codon%20position&amp;author=N.%20Sueoka&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=53-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00320-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00320-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00320-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.C.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Viari</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Universal replication biases in bacteria</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">11</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Universal%20replication%20biases%20in%20bacteria&amp;author=E.P.C.%20Rocha&amp;author=A.%20Danchin&amp;author=A.%20Viari&amp;publication_year=1999&amp;journal=Mol.%20Microbiol.&amp;volume=32&amp;pages=11-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1999.01334.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1999.01334.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1999.01334.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10216855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Asymmetric directional mutation pressures in bacteria</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div>  <div class="comment">research0058.1–0058.14</div></div></div></div></div><div content-id="DSP021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grigoriev</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Analyzing genomes with cumulative skew diagrams</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2286</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20genomes%20with%20cumulative%20skew%20diagrams&amp;author=A.%20Grigoriev&amp;publication_year=1998&amp;journal=Nucleic%20Acid%20Res.&amp;volume=26&amp;pages=2286-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.10.2286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.10.2286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.10.2286"> </span></div> </div></div></div></div></div><div content-id="DSP021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Francino</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Strand asymmetries in DNA evolution</div>, <div class="source">Trends Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">240</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20asymmetries%20in%20DNA%20evolution&amp;author=M.P.%20Francino&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=Trends%20Genet.&amp;volume=13&amp;pages=240-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9525(97)01118-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9525(97)01118-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9525(97)01118-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9196330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">O'Donnell</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Cellular DNA replicases: components and dynamics at the replication fork</div>, <div class="source">Annu. Rev. Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">283</div>-<div class="lpage">314</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cellular%20DNA%20replicases%3A%20components%20and%20dynamics%20at%20the%20replication%20fork&amp;author=A.%20Johnson&amp;author=M.%20O'Donnell&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Biochem.&amp;volume=74&amp;pages=283-314" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.73.011303.073859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.73.011303.073859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.73.011303.073859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15952889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bell</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Forsdyke</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">Accounting units in DNA</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1999</div>, vol. <div class="volume">197</div> (pg. <div class="fpage">51</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accounting%20units%20in%20DNA&amp;author=S.J.%20Bell&amp;author=D.R.%20Forsdyke&amp;publication_year=1999&amp;journal=J.%20Theor.%20Biol.&amp;volume=197&amp;pages=51-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1998.0857" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1998.0857" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1998.0857"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10036207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Francino</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Riley</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Asymmetries generated by transcription-coupled repair in enterobacterial genes</div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">107</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetries%20generated%20by%20transcription-coupled%20repair%20in%20enterobacterial%20genes&amp;author=M.%20Francino&amp;author=L.%20Chao&amp;author=M.A.%20Riley&amp;author=H.%20Ochman&amp;publication_year=1996&amp;journal=Science&amp;volume=272&amp;pages=107-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.272.5258.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.272.5258.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.272.5258.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8600517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLean</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Base composition skews, replication, and orientation in 12 prokaryote genomes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">691</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20composition%20skews%2C%20replication%2C%20and%20orientation%20in%2012%20prokaryote%20genomes&amp;author=M.J.%20McLean&amp;author=K.H.%20Wolfe&amp;author=K.M.%20Devine&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol.&amp;volume=47&amp;pages=691-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shioiri</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Takahata</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Skew of mononucleotide frequencies, relative abundance of dinucleotides and DNA strand asymmetry</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">364</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Skew%20of%20mononucleotide%20frequencies%2C%20relative%20abundance%20of%20dinucleotides%20and%20DNA%20strand%20asymmetry&amp;author=C.%20Shioiri&amp;author=N.%20Takahata&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol.&amp;volume=53&amp;pages=364-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11675596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nikiforov</div> <div class="given-names">A.M.</div></div>. </div><div class="article-title">Algorithm AS 288: Exact two-sample Smirnov test for arbitrary distributions</div>, <div class="source">Appl. Stat.</div> , <div class="year">1994</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">265</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Algorithm%20AS%20288%3A%20Exact%20two-sample%20Smirnov%20test%20for%20arbitrary%20distributions&amp;author=A.M.%20Nikiforov&amp;publication_year=1994&amp;journal=Appl.%20Stat.&amp;volume=43&amp;pages=265-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2986126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2986126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2986126"> </span></div> </div></div></div></div></div><div content-id="DSP021C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kolmogorov</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Confidence limits for an unknown distribution function</div>, <div class="source">Ann. Math. Stat.</div> , <div class="year">1941</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">461</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Confidence%20limits%20for%20an%20unknown%20distribution%20function&amp;author=A.%20Kolmogorov&amp;publication_year=1941&amp;journal=Ann.%20Math.%20Stat.&amp;volume=12&amp;pages=461-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1214/aoms/1177731684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1214%2Faoms%2F1177731684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1214%2Faoms%2F1177731684"> </span></div> </div></div></div></div></div><div content-id="DSP021C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smirnov</div> <div class="given-names">N.V.</div></div>. </div><div class="article-title">On the estimation of the discrepancy between empirical curves of distribution for two independent samples</div>, <div class="source">Bull. Moscow Univ.</div> , <div class="year">1939</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">3</div>-<div class="lpage">14</div>)</div></div></div></div><div content-id="DSP021C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J.R.</div></div>. </div><div class="article-title">Oriloc: prediction of replication boundaries in annotated bacterial chromosomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">560</div>-<div class="lpage">1</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oriloc%3A%20prediction%20of%20replication%20boundaries%20in%20annotated%20bacterial%20chromosomes&amp;author=A.C.%20Frank&amp;author=J.R.%20Lobry&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=560-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.6.560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.6.560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.6.560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10980154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Worning</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.F.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H-H.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Origin of replication in circular prokaryotic chromosomes</div>, <div class="source">Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">353</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20replication%20in%20circular%20prokaryotic%20chromosomes&amp;author=P.%20Worning&amp;author=L.J.%20Jensen&amp;author=P.F.%20Hallin&amp;author=H-H.%20Staerfeldt&amp;author=D.W.%20Ussery&amp;publication_year=2006&amp;journal=Environ.%20Microbiol.&amp;volume=8&amp;pages=353-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/emi.2006.8.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Femi.2006.8.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Femi.2006.8.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16423021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sernova</div> <div class="given-names">N.V.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Identification of replication origins in prokaryotic genomes</div>, <div class="source">Brief. Bioinform.</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">376</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20replication%20origins%20in%20prokaryotic%20genomes&amp;author=N.V.%20Sernova&amp;author=M.S.%20Gelfand&amp;publication_year=2008&amp;journal=Brief.%20Bioinform.&amp;volume=9&amp;pages=376-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbn031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbn031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbn031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18660512" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Bailes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Grocock</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Sockett</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">Variation in the strength of selected codon usage bias among bacteria</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1141</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20strength%20of%20selected%20codon%20usage%20bias%20among%20bacteria&amp;author=P.M.%20Sharp&amp;author=E.%20Bailes&amp;author=R.J.%20Grocock&amp;author=J.F.%20Peden&amp;author=R.E.%20Sockett&amp;publication_year=2005&amp;journal=Nucleic%20Acid%20Res.&amp;volume=33&amp;pages=1141-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki242"> </span></div> </div></div></div></div></div><div content-id="DSP021C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Duret</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Galtier</div> <div class="given-names">N.</div></div>. </div><div class="article-title">A new perspective on isochore evolution</div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">385</div> (pg. <div class="fpage">71</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20perspective%20on%20isochore%20evolution&amp;author=L.%20Duret&amp;author=A.%20Eyre-Walker&amp;author=N.%20Galtier&amp;publication_year=2006&amp;journal=Gene&amp;volume=385&amp;pages=71-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.04.030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.04.030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.04.030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16971063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nussinov</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Strong doublet preferences in nucleotide sequences and DNA geometry</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1984</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">111</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20doublet%20preferences%20in%20nucleotide%20sequences%20and%20DNA%20geometry&amp;author=R.%20Nussinov&amp;publication_year=1984&amp;journal=J.%20Mol.%20Evol.&amp;volume=20&amp;pages=111-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02257371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02257371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02257371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6433029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Forsdyke</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">A stem–loop ‘kissing’ model for the initiation of recombination and the origin of introns</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">949</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20stem%E2%80%93loop%20%E2%80%98kissing%E2%80%99%20model%20for%20the%20initiation%20of%20recombination%20and%20the%20origin%20of%20introns&amp;author=D.R.%20Forsdyke&amp;publication_year=1995&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=12&amp;pages=949-58" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7476142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Negative correlation between compositional symmetries and local recombination rates</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3951</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Negative%20correlation%20between%20compositional%20symmetries%20and%20local%20recombination%20rates&amp;author=L.%20Chen&amp;author=H.%20Zhao&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3951-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti651" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti651" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti651"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16131519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Scherer</div> <div class="given-names">S.W.</div></div>. </div><div class="article-title">Evolutionary implications of inversions that have caused intra-strand parity in DNA</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">160</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20implications%20of%20inversions%20that%20have%20caused%20intra-strand%20parity%20in%20DNA&amp;author=K.%20Okamura&amp;author=J.%20Wei&amp;author=S.W.%20Scherer&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=160" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17562011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.C.</div></div>. </div><div class="article-title">The replication-related organization of bacterial genomes</div>, <div class="source">Microbiology</div> , <div class="year">2004</div>, vol. <div class="volume">150</div> (pg. <div class="fpage">1609</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20replication-related%20organization%20of%20bacterial%20genomes&amp;author=E.P.C.%20Rocha&amp;publication_year=2004&amp;journal=Microbiology&amp;volume=150&amp;pages=1609-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.26974-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.26974-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.26974-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15184548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.C.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Gene essentiality determines chromosome organization in bacteria</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6570</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20essentiality%20determines%20chromosome%20organization%20in%20bacteria&amp;author=E.P.C.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=2003&amp;journal=Nucleic%20Acid%20Res.&amp;volume=31&amp;pages=6570-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg859"> </span></div> </div></div></div></div></div><div content-id="DSP021C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.C.</div></div>. </div><div class="article-title">The organization of the bacterial genome</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">211</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20organization%20of%20the%20bacterial%20genome&amp;author=E.P.C.%20Rocha&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=42&amp;pages=211-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091653" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091653" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091653"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18605898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP021C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Antoniw</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">DPVweb: An open access internet resource on plant viruses and virus diseases</div>, <div class="source">Outlooks on Pest Management</div> , <div class="year">2005</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">268</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DPVweb%3A%20An%20open%20access%20internet%20resource%20on%20plant%20viruses%20and%20virus%20diseases&amp;author=M.J.%20Adams&amp;author=J.F.%20Antoniw&amp;publication_year=2005&amp;journal=Outlooks%20on%20Pest%20Management&amp;volume=16&amp;pages=268-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1564/16dec08" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1564%2F16dec08" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1564%2F16dec08"> </span></div> </div></div></div></div></div><div content-id="DSP021C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Adams</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Antoniw</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">DPVweb: a comprehensive database of plant and fungal virus genes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div>  <div class="comment">D382–5. <a class="link link-uri" href="http://www.dpvweb.net/">http://www.dpvweb.net/</a></div></div></div></div></div><div content-id="DSP021C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP021C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bulmer</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The selection-mutation-drift theory of synonymous codon usage</div>, <div class="source">Genetics</div> , <div class="year">1991</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">897</div>-<div class="lpage">907</div>)</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5/uk/) which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=363736&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/6/10.1093/dnares/dsp021/2/dsp021-dsp021supp.pdf?Expires=2147483647&amp;Signature=NNJRW22Qlt6sgXb0PJx41NiFEl~9251SWq5WVkgi3jbCS791bidXEvwDkEns211t7LJhnc~bUXHsbbvhiwVgJ-wgjFXaFbg~JgyAwEKx3z3ez9fwwfJXzVOEPalJ~vw7kXrDwka~a64H9P4bt8JMBlsJX6BUxTaXZ-ECuwhiLN7jnD1xOuGjYF6j7UjJ1M1JDj1V6Qd4N3SEuSoA-r8xh3BP0rlbAizzG7SJtuZPfOK6SP4mDG~29tJFkO5fUZvKfl57vXe5apl4ccaW378~H3PJk95FTEvoV~tGWJ9UEucwBnWpyiBAS4yu7ZCumJ8QtVfhSz-F~lD1ntl~nNHGiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">231</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000272832700002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">16</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="363736"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000272832700002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f6%2f325%2f363736">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000272832700002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (16)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F6%2F325%2F363736" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/363736'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f6%2f325%2f363736">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

