

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/6/441/344455","@type":"ScholarlyArticle","name":"Stable Triple-Stranded DNA Formation and its Application to the SNP Detection","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi022","keywords":["RecA protein","stable triple-stranded DNA","deoxyoligonucleotide probe","SNPs detection"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/6/441/344455","author":[{"name":"Shigemori, Yasushi","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan    2  Aisin Cosmos R&amp;D Co. Ltd  2-1-5 Kazusa-kamatari, Kisarazu, Chiba 292-0818 Japan","@type":"Person"},{"name":"Oishi, Michio","affiliation":"1  Department of Human Gene Research, Kazusa DNA Research Institute  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"}],"description":"Abstract.  We have found that a short stretch (30mer or larger) of triple-stranded DNA structure formed at the terminus (or very near) of linear DNA molecules i","pageStart":"441","pageEnd":"449","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Stable Triple-Stranded DNA Formation and its Application to the SNP Detection","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Stable Triple-Stranded DNA Formation and its Application to the SNP Detection" />
    <meta property="og:description" content="Abstract.  We have found that a short stretch (30mer or larger) of triple-stranded DNA structure formed at the terminus (or very near) of linear DNA molecules i" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/6/441/344455" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shigemori, Yasushi" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="2Aisin Cosmos R&amp;D Co. Ltd2-1-5 Kazusa-kamatari, Kisarazu, Chiba 292-0818 Japan" /><meta name="citation_author" content="Oishi, Michio" /><meta name="citation_author_institution" content="1Department of Human Gene Research, Kazusa DNA Research Institute2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Stable Triple-Stranded DNA Formation and its Application to the SNP Detection" /><meta name="citation_firstpage" content="441" /><meta name="citation_lastpage" content="449" /><meta name="citation_doi" content="10.1093/dnares/dsi022" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=RecA protein promotes homologous-pairing, strand-exchange reactions between duplex DNA molecules; Proc. Natl Acad. Sci. USA;  citation_year=1981;  citation_volume=78;  citation_pages=2100-2104; " /><meta name="citation_reference" content="citation_title=Triple-helical DNA pairing intermediates formed by recA protein; J. Biol. Chem.;  citation_year=1990;  citation_volume=265;  citation_pages=16898-16912; " /><meta name="citation_reference" content="citation_title=Pairing of homologous DNA sequences by proteins: evidence for three-stranded DNA; Genes Dev.;  citation_year=1990;  citation_volume=4;  citation_pages=1951-1963; " /><meta name="citation_reference" content="citation_title=RecA protein-mediated irreversible fixation of an oligodeoxyribonucleotide to specific site in DNA; Mutat. Res.;  citation_year=1996;  citation_volume=351;  citation_pages=117-124; " /><meta name="citation_reference" content="citation_title=Site-specific cleavage of a yeast chromosome by deoxyoligonucleotide-directed triple-helix formation; Science;  citation_year=1990;  citation_volume=249;  citation_pages=73-75; " /><meta name="citation_reference" content="citation_title=Cleaving yeast and Escherichia coli genomes at a single site; Science;  citation_year=1990;  citation_volume=250;  citation_pages=271-273; " /><meta name="citation_reference" content="citation_title=Physical mapping of the Saccharomyces cerevisiae Ap4A phosphorylaseI-encoding gene by the Achilles' cleavage method; Gene;  citation_year=1992;  citation_volume=111;  citation_pages=131-134; " /><meta name="citation_reference" content="citation_title=Selective cleavage of human DNA: RecA-assisted restriction endonuclease (RARE) cleavage; Science;  citation_year=1992;  citation_volume=254;  citation_pages=1494-1497; " /><meta name="citation_reference" content="citation_title=Site-specific cleavage of human chromosome 4 mediated by triple-helix formation; Science;  citation_year=1991;  citation_volume=254;  citation_pages=1639-1642; " /><meta name="citation_reference" content="citation_title=Intermolecular triplex formation distorts the DNA duplex in the regulatory region of human papillomavirus type-11; J. Biol. Chem.;  citation_year=1992;  citation_volume=267;  citation_pages=5488-5494; " /><meta name="citation_reference" content="citation_title=Fluorescent detection and isolation of DNA variants using stabilized RecA-coated oligonucleotides; Genome Res.;  citation_year=2003;  citation_volume=14;  citation_pages=116-125; " /><meta name="citation_reference" content="citation_title=Targeted mutagenesis of DNA with alkylating RecA assisted oligonucleotides; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=e38" /><meta name="citation_reference" content="citation_title=Direct probing: covalent attachment of probe DNA to double-stranded target DNA; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=5728-5733; " /><meta name="citation_reference" content="citation_title=Direct cloning by covalent attachment of probe DNA to target DNA; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=5734-5737; " /><meta name="citation_reference" content="citation_title=Sequence-specific ligation of DNA using RecA protein; Proc Natl Acad Sci USA;  citation_year=1998;  citation_volume=95;  citation_pages=2152-2157; " /><meta name="citation_reference" content="citation_title=Use of psoralen-modified deoxyoligonucleotide to trap three-stranded RecA-DNA complexes and repair of these cross-linked complexes by ABC exonuclease; J. Biol. Chem.;  citation_year=1988;  citation_volume=263;  citation_pages=15110-15117; " /><meta name="citation_reference" content="citation_title=Stable three-stranded DNA made by RecA protein; Proc. Natl Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=2984-2988; " /><meta name="citation_reference" content="citation_title=Unusual stability of recombination intermediates made by Escherichia coli RecA protein; EMBO J.;  citation_year=1992;  citation_volume=7;  citation_pages=2685-2693; " /><meta name="citation_reference" content="citation_title=Homologous recognition and triplex formation promoted by RecA protein between duplex oligonucleotides and single-stranded DNA; J. Mol. Biol.;  citation_year=1993;  citation_volume=229;  citation_pages=328-343; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/6/441/6718708/dsi022.pdf" /><meta name="description" content="Abstract.  We have found that a short stretch (30mer or larger) of triple-stranded DNA structure formed at the terminus (or very near) of linear DNA molecules i" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/6/441/344455" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/6/441/344455" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/6/441/344455");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi022';
    </script>
<title>
	Stable Triple-Stranded DNA Formation and its Application to the SNP Detection | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./344455" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BawvPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGlnZW1vcmksIFlhc3VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBIdW1hbiBHZW5lIFJlc2VhcmNoLCBLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZTItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJBaXNpbiBDb3Ntb3MgUiZhbXA7RCBDby4gTHRkMi0xLTUgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTggSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaXNoaSwgTWljaGlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgSHVtYW4gR2VuZSBSZXNlYXJjaCwgS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlN0YWJsZSBUcmlwbGUtU3RyYW5kZWQgRE5BIEZvcm1hdGlvbiBhbmQgaXRzIEFwcGxpY2F0aW9uIHRvIHRoZSBTTlAgRGV0ZWN0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2kwMjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNS8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY0EgcHJvdGVpbiBwcm9tb3RlcyBob21vbG9nb3VzLXBhaXJpbmcsIHN0cmFuZC1leGNoYW5nZSByZWFjdGlvbnMgYmV0d2VlbiBkdXBsZXggRE5BIG1vbGVjdWxlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTIxMDAtMjEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJpcGxlLWhlbGljYWwgRE5BIHBhaXJpbmcgaW50ZXJtZWRpYXRlcyBmb3JtZWQgYnkgcmVjQSBwcm90ZWluOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjU7ICBjaXRhdGlvbl9wYWdlcz0xNjg5OC0xNjkxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGFpcmluZyBvZiBob21vbG9nb3VzIEROQSBzZXF1ZW5jZXMgYnkgcHJvdGVpbnM6IGV2aWRlbmNlIGZvciB0aHJlZS1zdHJhbmRlZCBETkE7IEdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE5NTEtMTk2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjQSBwcm90ZWluLW1lZGlhdGVkIGlycmV2ZXJzaWJsZSBmaXhhdGlvbiBvZiBhbiBvbGlnb2Rlb3h5cmlib251Y2xlb3RpZGUgdG8gc3BlY2lmaWMgc2l0ZSBpbiBETkE7IE11dGF0LiBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM1MTsgIGNpdGF0aW9uX3BhZ2VzPTExNy0xMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpdGUtc3BlY2lmaWMgY2xlYXZhZ2Ugb2YgYSB5ZWFzdCBjaHJvbW9zb21lIGJ5IGRlb3h5b2xpZ29udWNsZW90aWRlLWRpcmVjdGVkIHRyaXBsZS1oZWxpeCBmb3JtYXRpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MjQ5OyAgY2l0YXRpb25fcGFnZXM9NzMtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsZWF2aW5nIHllYXN0IGFuZCBFc2NoZXJpY2hpYSBjb2xpIGdlbm9tZXMgYXQgYSBzaW5nbGUgc2l0ZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTA7ICBjaXRhdGlvbl9wYWdlcz0yNzEtMjczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlzaWNhbCBtYXBwaW5nIG9mIHRoZSBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgQXA0QSBwaG9zcGhvcnlsYXNlSS1lbmNvZGluZyBnZW5lIGJ5IHRoZSBBY2hpbGxlcycgY2xlYXZhZ2UgbWV0aG9kOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTEzMS0xMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGl2ZSBjbGVhdmFnZSBvZiBodW1hbiBETkE6IFJlY0EtYXNzaXN0ZWQgcmVzdHJpY3Rpb24gZW5kb251Y2xlYXNlIChSQVJFKSBjbGVhdmFnZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTQ7ICBjaXRhdGlvbl9wYWdlcz0xNDk0LTE0OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpdGUtc3BlY2lmaWMgY2xlYXZhZ2Ugb2YgaHVtYW4gY2hyb21vc29tZSA0IG1lZGlhdGVkIGJ5IHRyaXBsZS1oZWxpeCBmb3JtYXRpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjU0OyAgY2l0YXRpb25fcGFnZXM9MTYzOS0xNjQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcm1vbGVjdWxhciB0cmlwbGV4IGZvcm1hdGlvbiBkaXN0b3J0cyB0aGUgRE5BIGR1cGxleCBpbiB0aGUgcmVndWxhdG9yeSByZWdpb24gb2YgaHVtYW4gcGFwaWxsb21hdmlydXMgdHlwZS0xMTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjY3OyAgY2l0YXRpb25fcGFnZXM9NTQ4OC01NDk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbHVvcmVzY2VudCBkZXRlY3Rpb24gYW5kIGlzb2xhdGlvbiBvZiBETkEgdmFyaWFudHMgdXNpbmcgc3RhYmlsaXplZCBSZWNBLWNvYXRlZCBvbGlnb251Y2xlb3RpZGVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTExNi0xMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhcmdldGVkIG11dGFnZW5lc2lzIG9mIEROQSB3aXRoIGFsa3lsYXRpbmcgUmVjQSBhc3Npc3RlZCBvbGlnb251Y2xlb3RpZGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz1lMzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXJlY3QgcHJvYmluZzogY292YWxlbnQgYXR0YWNobWVudCBvZiBwcm9iZSBETkEgdG8gZG91YmxlLXN0cmFuZGVkIHRhcmdldCBETkE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU3MjgtNTczMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IGNsb25pbmcgYnkgY292YWxlbnQgYXR0YWNobWVudCBvZiBwcm9iZSBETkEgdG8gdGFyZ2V0IEROQTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTczNC01NzM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1zcGVjaWZpYyBsaWdhdGlvbiBvZiBETkEgdXNpbmcgUmVjQSBwcm90ZWluOyBQcm9jIE5hdGwgQWNhZCBTY2kgVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MjE1Mi0yMTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2Ugb2YgcHNvcmFsZW4tbW9kaWZpZWQgZGVveHlvbGlnb251Y2xlb3RpZGUgdG8gdHJhcCB0aHJlZS1zdHJhbmRlZCBSZWNBLUROQSBjb21wbGV4ZXMgYW5kIHJlcGFpciBvZiB0aGVzZSBjcm9zcy1saW5rZWQgY29tcGxleGVzIGJ5IEFCQyBleG9udWNsZWFzZTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MjYzOyAgY2l0YXRpb25fcGFnZXM9MTUxMTAtMTUxMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YWJsZSB0aHJlZS1zdHJhbmRlZCBETkEgbWFkZSBieSBSZWNBIHByb3RlaW47IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz0yOTg0LTI5ODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVudXN1YWwgc3RhYmlsaXR5IG9mIHJlY29tYmluYXRpb24gaW50ZXJtZWRpYXRlcyBtYWRlIGJ5IEVzY2hlcmljaGlhIGNvbGkgUmVjQSBwcm90ZWluOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yNjg1LTI2OTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvbW9sb2dvdXMgcmVjb2duaXRpb24gYW5kIHRyaXBsZXggZm9ybWF0aW9uIHByb21vdGVkIGJ5IFJlY0EgcHJvdGVpbiBiZXR3ZWVuIGR1cGxleCBvbGlnb251Y2xlb3RpZGVzIGFuZCBzaW5nbGUtc3RyYW5kZWQgRE5BOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTIyOTsgIGNpdGF0aW9uX3BhZ2VzPTMyOC0zNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xMi82LzQ0MS82NzE4NzA4L2RzaTAyMi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgaGF2ZSBmb3VuZCB0aGF0IGEgc2hvcnQgc3RyZXRjaCAoMzBtZXIgb3IgbGFyZ2VyKSBvZiB0cmlwbGUtc3RyYW5kZWQgRE5BIHN0cnVjdHVyZSBmb3JtZWQgYXQgdGhlIHRlcm1pbnVzIChvciB2ZXJ5IG5lYXIpIG9mIGxpbmVhciBETkEgbW9sZWN1bGVzIGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0NDQ1NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTqmHF1DneNPy6WmfhgS3OdjAyoyQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f441%2f344455" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f441%2f344455" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13225" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&amp;Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/6">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="35870162">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870162">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870164">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870164">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870167">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870167">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870174">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870174">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870194">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870194">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870199">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870199">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/6/429/344258">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/12/6/451/344552">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Stable Triple-Stranded DNA Formation and its Application to the SNP Detection
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasushi Shigemori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Aisin Cosmos R&amp;D Co. Ltd</div><div class="addr-line">2-1-5 Kazusa-kamatari, Kisarazu, Chiba 292-0818 Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasushi+Shigemori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shigemori Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shigemori Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasushi Shigemori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michio Oishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Human Gene Research, Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup> To whom correspondence should be addressed. Tel. +81-438-52-3944, Fax. +81-438-52-3911, E-mail: <a href="mailto:oishi@kazusa.or.jp" target="_blank">oishi@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michio+Oishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oishi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oishi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michio Oishi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Takao Sekiya</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 441–449, <a href='https://doi.org/10.1093/dnares/dsi022'>https://doi.org/10.1093/dnares/dsi022</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 December 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">11 December 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="344455" href="/dnaresearch/article-pdf/12/6/441/6718708/dsi022.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yasushi Shigemori, Michio Oishi;  Stable Triple-Stranded DNA Formation and its Application to the SNP Detection, <em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 441–449, <a href='https://doi.org/10.1093/dnares/dsi022'>https://doi.org/10.1093/dnares/dsi022</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/344455?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/344455?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Stable%20Triple-Stranded%20DNA%20Formation%20and%20its%20Application%20to%20the%20SNP%20Detection&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=6&amp;author=Shigemori%2C%20Yasushi%3B%20Oishi%2C%20Michio&amp;startPage=441&amp;endPage=449&amp;contentId=10.1093%2Fdnares%2Fdsi022&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f441%2f344455" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=35870162 id="35870162" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We have found that a short stretch (30mer or larger) of triple-stranded DNA structure formed at the terminus (or very near) of linear DNA molecules is unusually stable, withstanding heat treatment at as high as 95°C. The stable triple-stranded structure is formed only when deoxyoligonucleotides are complementary to the strand terminating with 5′-phosphate and not to the strand terminating with 3′-OH. Presence of a single mismatched base in the complementary deoxyoligonucleotides drastically reduces the stability. We show that these unique properties of the terminal triple-stranded structure can be applied to the detection of single nucleotide polymorphisms in genomes without DNA dissociation and/or hybridization.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RecA protein&quot;">RecA protein</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;stable triple-stranded DNA&quot;">stable triple-stranded DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;deoxyoligonucleotide probe&quot;">deoxyoligonucleotide probe</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;SNPs detection&quot;">SNPs detection</a></div>                            <h2 scrollto-destination=35870164 id="35870164" class="section-title" >1. Introduction</h2>
<p> Triple-stranded DNA has been generating increased interest not only because of its unique structural characteristics and possible biological significance but also because of its potential utility as a tool for DNA analysis. <a reveal-id="b1  b3" data-open="b1  b3" class="link link-ref link-reveal xref-bibr">1–3</a> There have been several studies reporting the use of triple-stranded DNA as a tool to cleave specific sites in DNA molecules, to knock out specific genes and to correct mutations. <a reveal-id="b4  b15" data-open="b4  b15" class="link link-ref link-reveal xref-bibr">4–15</a> Triple-stranded DNA structure particularly with short deoxyoligonucleotides has generally been considered unstable after removal of the protein. <a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> During the course of studies on RecA-mediated triple-stranded DNA formation with complementary deoxyoligonucleotides (40 to 120mer), we found that triple-stranded structures at the terminus (or near terminus) of double-stranded DNA is unusually stable against heat, provided that the sequence of the single strand deoxyoligonucleotide is complementary to the 5′ terminal sequence of the double-stranded DNA. This heat stability is drastically increased, to levels even higher than that of double-stranded DNA, when excess complementary deoxyoligonucleotides are present in the reaction mixture. Furthermore, we have found that the heat-stable terminal triple-stranded structures are formed with completely matched complementary deoxyoligonucleotides, but not with those carrying a single mismatched base regardless of its position. </p><p>In this manuscript, we describe the basic characteristics of terminal triple-stranded DNA, and show that our findings can be applied to single nucleotide polymorphism (SNP) detection without DNA dissociation and/or hybridization.</p>                            <h2 scrollto-destination=35870167 id="35870167" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=35870168 id="35870168" class="section-title" >2.1. Materials</h3>
<p> M13mp18RF DNA and pBR322 DNA were purchased from Takara-Bio (Shiga, Japan). pBR322 DNA with a substituted base was prepared by PCR using primers (40mer), one of which contained a mismatched base, and <em>Sca</em> I pBR322 DNA fragment as a template. The PCR products were purified by agarose gel (1%) electrophoresis and DNA was eluted from the corresponding band using a DNA extraction kit (QIAGEN, Hilden, Germany). Deoxyoligonucleotides (HPLC purified grade) were custom-synthesized by Sawady Technology (Tokyo, Japan). Deoxyoligonucleotides were labeled at 5′ terminus with T4 polynucleotide kinase and <sup>32</sup> P-ATP using a kit (MEGALABEL™ Labeling kit, Takara-Bio). RecA protein was purchased from Epicenter Technologies (Madison, USA). ATP-γS and proteinase K were obtained by Roche Diagnostics (Mannheim, Germany). </p>                            <h3 scrollto-destination=35870170 id="35870170" class="section-title" >2.2. Standard reaction for the triple-stranded DNA formation</h3>
<p>A reaction mixture (50 µl) containing a deoxyoligonucleotide complementary to target DNA (5 pmol), RecA protein (30 µg), ATP-γS (4.8 mM), Mg-acetate (2 mM) and Tris–acetate (30 mM, pH 7.2) was first incubated for 5 min at 37°C. The mixture was then combined with a mixture (50 µl) containing target DNA (10 µg), Mg-acetate (20 mM) and Tris–acetate (30 mM, pH 7.2) and incubated for another 60 min. The reaction was stopped by adding 5 µl SDS (10%, w/v), 5 µl EDTA (500 mM, pH 8.0) and 0.5 µl proteinase K (22 mg/ml), further incubated for 30 min at 37°C, and subjected to phenol/chloroform treatment. Excess deoxyoligonucleotides were removed through a Sephacryl S-400 HR column (Amersham-Bioscience, Uppsala, Sweden) which had been equilibrated with TE buffer (10 mM Tris–HCl pH 7.5, 1 mM EDTA).</p>                            <h3 scrollto-destination=35870172 id="35870172" class="section-title" >2.3. Analysis of SNPs in human genomic DNA</h3>
<p> The region of 950 bp sequence of human genomic DNA (GenBank accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/AC004465" target="_blank">AC004465</a> ) in which an SNP site of interest is positioned at 40 nt from the terminus in three human genomic DNA preparations was amplified by PCR. The genomic DNA preparations were kindly supplied by Drs Y. Ohnishi and Y. Nakamura, the University of Tokyo. The PCR products, prepared after 25 cycles of PCR using the genomic DNA (200 ng) and PCR primers (GenBank accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/AC004465" target="_blank">AC004465</a> , nt 2701–2720 and nt 3631–3650), were subjected to triple-stranded DNA formation with a set of labeled deoxyoligonucleotides. The reaction was performed by incubating a reaction mixture (20 µl) which contained amplified DNA (400 ng), a labeled probe (1 pmol), RecA protein (3 µg), ATP-γS (4.8 mM), Mg-acetate (20 mM) and Tris–acetate (30 mM, pH 7.2) for 60 min at 37°C. After removal of RecA protein and excess deoxyoligonucleotides through a TE buffer equilibrated Sephacryl S-400 HR column (two cycles), samples (20 µl) were incubated at 85°C for 10 min in the presence of a mixture of four cold deoxyoligonucleotides (10 pmol, each). The samples were then subjected to heat treatment at 80°C (85°C for the experiments shown in Figs 5 and 6) for 10 min (25°C for 10 min for the controls). The deoxyoligonucleotides were then removed through Sephacryl S-400 HR column (one cycle), and the samples were subjected to electrophoresis on agarose gel (1%) and autoradiographed. </p>                            <h2 scrollto-destination=35870174 id="35870174" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=35870175 id="35870175" class="section-title" >3.1. The polarity of the third strand in terminal triple-strand formation</h3>
<p> We performed an experiment in which M13mp18RF DNA (7249 bp) linearized by digestion with <em>Sna</em> BI was incubated with RecA protein and one of four <sup>32</sup> P-labeled 60mer deoxyoligonucleotides (oligo-1, 2, 3 and 4, <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a> ). The labeled deoxyoligonucleotide sequences were complementary to the terminal sequences of the double-stranded DNA. After removal of RecA protein, the products were electrophoresed and autoradiographed. Triple-stranded DNA formed with a deoxyoligonucleotide (oligo-1) complementary to the 5′-phosphate terminal sequence (nt 1270–1329) of the double-stranded DNA or that formed with oligo-3 identical to the 3′-OH terminal sequence (nt 1210–1269) displayed signal at the position where linear M13mp18RF DNA was expected to migrate ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a> , lane 1 or lane 3, respectively). In contrast, no signal was detected when the double-stranded DNA was incubated with oligo-2 identical to the 5′-phosphate terminal sequence (nt 1270–1329) or oligo-4 complementary to the 3′-OH terminal sequence (nt 1210–1269) ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1B</a> , lane 2 or lane 4, respectively). Similar results were obtained when the terminal sequence of linearized pBR322 DNA or pUC118 DNA was used as a target DNA for the formation of the triple-stranded molecules (data not shown). These results strongly suggest that a triple-stranded structure at the ends of a double-stranded DNA is formed with deoxyoligonucleotides complementary to the 5′-phosphate terminal sequences of both strands, but not with those complementary to the 3′-OH termini, and that the triple-stranded structures thus formed are stable even after the removal of RecA protein. We estimate that 80–90% of the target DNA is converted to triple-stranded structure under the conditions we employed (data not shown). </p>                        <a id="35870177" scrollto-destination="35870177"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f1.jpeg?Expires=2147483647&Signature=ajHSEJ~~CqFEIi02YEHx1s3eEcM-jB~71vVQ0P3ln4bA4tjqADKHhnSwvqtpo0afNckz3audUgbGtp6uVs7d0OOXEHrFp72L3iP77U~rsZQggc791oqhogyXIq58wmVB4ZK7vk5v-eZCiwXAeF6ASCVjWVnJskkclrp-Xn7LQmR-R1Kev1yzCPvOW97bWG~lZ6z6KdLVsoSiX0alXJCQV6JGkgGuWg1PmdZHu7vWNcydKhEYxodPIq8jqx~~xO072Cj1sq~S03LM0t5nFzF~ADd3dWfZxuE9sxhq3ZJbStdNlyt66pemZCRK3-bo7DPBbYDBWDT2hUyOZ7oG~ynWxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Triple-stranded DNA formation at the terminus of double-stranded DNA. M13mp18RF DNA linearized by Sna BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 2, 3 and 4). (A) Diagrammatic representation of Sna BI DNA fragment and the positions and directions of deoxyoligonucleotides (oligo-1 to 4) used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1) complementary to the 60 nt 5′ terminal sequence (nt 1270–1329); lane 2, DNA with a deoxyoligonucleotide (oligo-2) complementary to the 3′-OH strand at the same terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-3) complementary to the 60 nt 5′ terminal sequence (nt 1210–1269); lane 4, DNA with a deoxyoligonucleotide (oligo-4) complementary to the 3′-OH strand at the same terminus. (C) Ethidium bromide patterns of DNA products. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870177/dsi022f1.jpeg" path-from-xml="dsi022f1.jpeg" target="_blank">View large</a><a section="35870177" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f1.jpeg?Expires=2147483647&Signature=EzO72P14XFmIhoyCitbGH~NBiL5LofgkABDWn~lUsjUIlQimSQReFNd1QK~fcb8yOSNCxKmxobNLI5QYgejQ6HEj29Oruhsp4bIfQ9eo8PULoTZ5K5Uz8hXT6OGumbheZsLOFUFlS7NpKrwq0z5e9nzVUCI5QT4HNySzjsznzjn6vKQhnP9wQCY90h9xa7fA61OdZ7U-q0S5kWtlRtcmJxWRrYoxdx8pdYTDEnxrwDehpkDNB58S84caYFed5UtwPLXwc2TOgLsLmT5O2e049vyQJZYvNXh1qFrwPMDDrhnO0MqBPzzeKw7yRI0X7KVHPTRv5uVFoMkqgRPbiWz1Dg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870177&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Triple-stranded DNA formation at the terminus of double-stranded DNA. M13mp18RF DNA linearized by <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 2, 3 and 4). (A) Diagrammatic representation of <em>Sna</em> BI DNA fragment and the positions and directions of deoxyoligonucleotides (oligo-1 to 4) used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1) complementary to the 60 nt 5′ terminal sequence (nt 1270–1329); lane 2, DNA with a deoxyoligonucleotide (oligo-2) complementary to the 3′-OH strand at the same terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-3) complementary to the 60 nt 5′ terminal sequence (nt 1210–1269); lane 4, DNA with a deoxyoligonucleotide (oligo-4) complementary to the 3′-OH strand at the same terminus. (C) Ethidium bromide patterns of DNA products. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f1.jpeg?Expires=2147483647&Signature=ajHSEJ~~CqFEIi02YEHx1s3eEcM-jB~71vVQ0P3ln4bA4tjqADKHhnSwvqtpo0afNckz3audUgbGtp6uVs7d0OOXEHrFp72L3iP77U~rsZQggc791oqhogyXIq58wmVB4ZK7vk5v-eZCiwXAeF6ASCVjWVnJskkclrp-Xn7LQmR-R1Kev1yzCPvOW97bWG~lZ6z6KdLVsoSiX0alXJCQV6JGkgGuWg1PmdZHu7vWNcydKhEYxodPIq8jqx~~xO072Cj1sq~S03LM0t5nFzF~ADd3dWfZxuE9sxhq3ZJbStdNlyt66pemZCRK3-bo7DPBbYDBWDT2hUyOZ7oG~ynWxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Triple-stranded DNA formation at the terminus of double-stranded DNA. M13mp18RF DNA linearized by Sna BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 2, 3 and 4). (A) Diagrammatic representation of Sna BI DNA fragment and the positions and directions of deoxyoligonucleotides (oligo-1 to 4) used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1) complementary to the 60 nt 5′ terminal sequence (nt 1270–1329); lane 2, DNA with a deoxyoligonucleotide (oligo-2) complementary to the 3′-OH strand at the same terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-3) complementary to the 60 nt 5′ terminal sequence (nt 1210–1269); lane 4, DNA with a deoxyoligonucleotide (oligo-4) complementary to the 3′-OH strand at the same terminus. (C) Ethidium bromide patterns of DNA products. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870177/dsi022f1.jpeg" path-from-xml="dsi022f1.jpeg" target="_blank">View large</a><a section="35870177" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f1.jpeg?Expires=2147483647&Signature=EzO72P14XFmIhoyCitbGH~NBiL5LofgkABDWn~lUsjUIlQimSQReFNd1QK~fcb8yOSNCxKmxobNLI5QYgejQ6HEj29Oruhsp4bIfQ9eo8PULoTZ5K5Uz8hXT6OGumbheZsLOFUFlS7NpKrwq0z5e9nzVUCI5QT4HNySzjsznzjn6vKQhnP9wQCY90h9xa7fA61OdZ7U-q0S5kWtlRtcmJxWRrYoxdx8pdYTDEnxrwDehpkDNB58S84caYFed5UtwPLXwc2TOgLsLmT5O2e049vyQJZYvNXh1qFrwPMDDrhnO0MqBPzzeKw7yRI0X7KVHPTRv5uVFoMkqgRPbiWz1Dg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870177&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Triple-stranded DNA formation at the terminus of double-stranded DNA. M13mp18RF DNA linearized by <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 2, 3 and 4). (A) Diagrammatic representation of <em>Sna</em> BI DNA fragment and the positions and directions of deoxyoligonucleotides (oligo-1 to 4) used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1) complementary to the 60 nt 5′ terminal sequence (nt 1270–1329); lane 2, DNA with a deoxyoligonucleotide (oligo-2) complementary to the 3′-OH strand at the same terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-3) complementary to the 60 nt 5′ terminal sequence (nt 1210–1269); lane 4, DNA with a deoxyoligonucleotide (oligo-4) complementary to the 3′-OH strand at the same terminus. (C) Ethidium bromide patterns of DNA products. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div>                            <h3 scrollto-destination=35870178 id="35870178" class="section-title" >3.2. Effect of the position of triple-stranded structure relative to the DNA terminus</h3>
<p> The stable triple-stranded DNA structure seems to be limited to those formed at the very terminus of double-stranded DNAs. <em>Sna</em> BI digest of M13mp18RF DNA was subjected to triple-stranded DNA formation with labeled deoxyoligonucleotides complementary to different sequences at or near the DNA terminus ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ). As seen in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a> , when the 3′ position of the complementary deoxyoligonucleotides (60mer) was moved from the terminus (oligo-1, lane 1) to 10 (oligo-5, lane 2), 20 (oligo-6, lane 3) and 30 nt (oligo-7, lane 4) to the inside of the DNA molecule, the stability of triple-stranded structures formed was dramatically decreased in proportion to the distance from the DNA terminus. Essentially no stable triple-stranded DNA was formed with a deoxyoligonucleotide complementary to DNA sequences inwardly located 30 nt from the terminus ( <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a> , lane 4). The results indicate that stability of triple-stranded DNA is greatly affected by the position of the triple-strand, with those structures formed closer to the 5′ terminus of DNA demonstrating greater stability. </p>                        <a id="35870180" scrollto-destination="35870180"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f2.jpeg?Expires=2147483647&Signature=1WJD0KfhE1SAiZ3kSDQxo6c5zJBWNQSQUkrUYaStHWknYkq8P1VAbw8Z4NQ-H8vxiFx3td7xRy-o81QICsyZlnzhYFatiDaedJeNN-54uIbuvqsFBqGwl7K9N4mrui8CzQjPck-ItHg6zTgLOTmBJUv7wmLIenpyfstI3TLW~dZspthq9S38KicybDD-lZ2JKzDkTpXp3paWXz3Ht5Q7qyrW~DJQBS5PWd5joR07wIwpl84AC2vlyvhqQw3YBHMyOlaY2Pp20p3cM2R0pRkMkbjdL7XD5e07xhTolK5oCkP16tzd9ZJDwUkcStP0eXX7MFYpnn~WAb3A4fTLqIJLIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Positional effect of deoxyoligonucleotide on stable triple-stranded DNA formation. M13mp18RF DNA linearized with Sna BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 5, 6 and 7). (A) Diagrammatic representation of the Sna BI DNA fragment and the positions and directions of deoxyoligonucleotides used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1, nt 1270–1329) complementary to the 60 nt terminal sequence; lane 2, DNA with a deoxyoligonucleotide (oligo-5, nt 1279–1338) complementary to the 60 nt sequence starting 10 nt from the terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-6, nt 1289–1348) complementary to the 60 nt sequence starting 20 nt from the terminus; and lane 4, DNA with a deoxyoligonucleotide (oligo-7, nt 1299–1358) complementary to the 60 nt sequence starting 30 nt from the terminus. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870180/dsi022f2.jpeg" path-from-xml="dsi022f2.jpeg" target="_blank">View large</a><a section="35870180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f2.jpeg?Expires=2147483647&Signature=r-o4SX2Yn8p3Zmh-18qtVshKJWo1aYFzp~gFw9nAPzv3xNmWUzYIcMqU3g725rXSYpbtTxUAOuLmkTp-OyAMWAtkiPjWnDU1ptj7rR7W2IxEKAbRa71wVQ6txGVA~fE9Ok1dF4ZDJpZYB03R5mdefKQtPVm-zzBo24yzIfqtJ6NXDwQwlS~bbf7TBbcsgQ6nyetRw2NIG8mZaHrhP9FZxPt2uztvKrirPfCudvS0AWuJ65hxMQnwDXL2o5y62jNDQ0eHOFiFiPOZSsAQrZVtetT6uzzrpWdU63jiGgtPdp7ZAZ4KqkSNQGT3vYfHldReQ0Ikz1p2qanjdWXT-qVKOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870180&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Positional effect of deoxyoligonucleotide on stable triple-stranded DNA formation. M13mp18RF DNA linearized with <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 5, 6 and 7). (A) Diagrammatic representation of the <em>Sna</em> BI DNA fragment and the positions and directions of deoxyoligonucleotides used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1, nt 1270–1329) complementary to the 60 nt terminal sequence; lane 2, DNA with a deoxyoligonucleotide (oligo-5, nt 1279–1338) complementary to the 60 nt sequence starting 10 nt from the terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-6, nt 1289–1348) complementary to the 60 nt sequence starting 20 nt from the terminus; and lane 4, DNA with a deoxyoligonucleotide (oligo-7, nt 1299–1358) complementary to the 60 nt sequence starting 30 nt from the terminus. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f2.jpeg?Expires=2147483647&Signature=1WJD0KfhE1SAiZ3kSDQxo6c5zJBWNQSQUkrUYaStHWknYkq8P1VAbw8Z4NQ-H8vxiFx3td7xRy-o81QICsyZlnzhYFatiDaedJeNN-54uIbuvqsFBqGwl7K9N4mrui8CzQjPck-ItHg6zTgLOTmBJUv7wmLIenpyfstI3TLW~dZspthq9S38KicybDD-lZ2JKzDkTpXp3paWXz3Ht5Q7qyrW~DJQBS5PWd5joR07wIwpl84AC2vlyvhqQw3YBHMyOlaY2Pp20p3cM2R0pRkMkbjdL7XD5e07xhTolK5oCkP16tzd9ZJDwUkcStP0eXX7MFYpnn~WAb3A4fTLqIJLIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Positional effect of deoxyoligonucleotide on stable triple-stranded DNA formation. M13mp18RF DNA linearized with Sna BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 5, 6 and 7). (A) Diagrammatic representation of the Sna BI DNA fragment and the positions and directions of deoxyoligonucleotides used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1, nt 1270–1329) complementary to the 60 nt terminal sequence; lane 2, DNA with a deoxyoligonucleotide (oligo-5, nt 1279–1338) complementary to the 60 nt sequence starting 10 nt from the terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-6, nt 1289–1348) complementary to the 60 nt sequence starting 20 nt from the terminus; and lane 4, DNA with a deoxyoligonucleotide (oligo-7, nt 1299–1358) complementary to the 60 nt sequence starting 30 nt from the terminus. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870180/dsi022f2.jpeg" path-from-xml="dsi022f2.jpeg" target="_blank">View large</a><a section="35870180" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f2.jpeg?Expires=2147483647&Signature=r-o4SX2Yn8p3Zmh-18qtVshKJWo1aYFzp~gFw9nAPzv3xNmWUzYIcMqU3g725rXSYpbtTxUAOuLmkTp-OyAMWAtkiPjWnDU1ptj7rR7W2IxEKAbRa71wVQ6txGVA~fE9Ok1dF4ZDJpZYB03R5mdefKQtPVm-zzBo24yzIfqtJ6NXDwQwlS~bbf7TBbcsgQ6nyetRw2NIG8mZaHrhP9FZxPt2uztvKrirPfCudvS0AWuJ65hxMQnwDXL2o5y62jNDQ0eHOFiFiPOZSsAQrZVtetT6uzzrpWdU63jiGgtPdp7ZAZ4KqkSNQGT3vYfHldReQ0Ikz1p2qanjdWXT-qVKOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870180&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Positional effect of deoxyoligonucleotide on stable triple-stranded DNA formation. M13mp18RF DNA linearized with <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with one of four labeled 60mer deoxyoligonucleotide (oligo-1, 5, 6 and 7). (A) Diagrammatic representation of the <em>Sna</em> BI DNA fragment and the positions and directions of deoxyoligonucleotides used for triple-stranded DNA formation. Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA products. Lane 1, DNA with a deoxyoligonucleotide (oligo-1, nt 1270–1329) complementary to the 60 nt terminal sequence; lane 2, DNA with a deoxyoligonucleotide (oligo-5, nt 1279–1338) complementary to the 60 nt sequence starting 10 nt from the terminus; lane 3, DNA with a deoxyoligonucleotide (oligo-6, nt 1289–1348) complementary to the 60 nt sequence starting 20 nt from the terminus; and lane 4, DNA with a deoxyoligonucleotide (oligo-7, nt 1299–1358) complementary to the 60 nt sequence starting 30 nt from the terminus. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div><p>As one might expect, we have also found that the stability of triple-stranded DNA is dependent upon the size of the deoxyoligonucleotides employed. Linearized M13mp 18RF DNA was subjected to triple-stranded DNA formation with labeled deoxyoligonucleotides complementary to the terminal sequences, but with different chain (nucleotide) lengths. Stable triple-stranded DNA formation was observed with complementary deoxyoligonucleotides of &gt;40 nt (40mer), but the efficiency of the formation was drastically reduced (∼50% of that of 60 nt) with a deoxyoligonucleotide of 30mer. Essentially no stable triple-stranded DNA formation was observed with deoxyoligonucleotides of 20mer or less. (data not shown).</p>                            <h3 scrollto-destination=35870182 id="35870182" class="section-title" >3.3. Stability of terminal triple-stranded DNA structures</h3>
<p> To characterize the stability of terminal triple-stranded structures, triple-stranded DNA formed with M13mp 18RF DNA and labeled 60mer deoxyoligonucleotide (oligo-3) was exposed to different temperature (for 10 min) after removal of RecA protein, and radioactivity associated with M13mp18RF DNA was measured. As seen in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a> , the deoxyoligonucleotide started to dissociate from the complex at 60°C (lane 2), and was completely dissociated at 75°C (lane 5). The estimated melting temperature was ∼65°C. The melting temperature of a hybrid structure formed from the same deoxyoligonucleotide (oligo-3) and single-stranded M13mp18 DNA was ∼75°C under the same conditions (data not shown). To our surprise, when unlabeled deoxyoligonucleotide (oligo-3) (10-fold excess over the labeled deoxyoligonucleotide) was added during heat treatment, the stability of the formed triple-stranded DNA seemed dramatically increased. At temperatures as high as 95°C, a substantial portion (∼50%) of the deoxyoligonucleotide remained undissociated ( <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a> , lanes 1–9). No such stabilizing activity was observed when the same amount of unlabeled deoxyoligonucleotides (oligo-8) not complementary to the terminal DNA sequence was added (data not shown). The stabilizing effect of complementary deoxyoligonucleotides on terminal triple-stranded DNA is not fully understood at this time. The results are diagrammatically shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a> . </p>                        <a id="35870184" scrollto-destination="35870184"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f3.jpeg?Expires=2147483647&Signature=HWodJ1M6i42iIFuA8dFeUvL5yzslxONEYuPAOqkYPSV06XsKsy6~kUNC-LRqyvRNTiEaI2l4pXwFih3Dbvogq9szb-f72zYuUnbWUsretMIRU~P8Rr90C9E5ccZqskfBJajcPO2bnSVISq85WXM9v0kIBx9qQZWUq0CH~zN5DypRB6p28UjzFv288c8bhvrz9uUIdZ79BzQy2GKX4GR~NyJh~MxV4~WllJGlNawqmfOP2eHmLfWmrQvDULBrfsMTTaSBCOnO5WYivbmCWhNeksiQTWAmgQHSqf~ZXjnqbLqxt~FJZVfXebNFC6HtNEjhVpmvjKsmHpjLsQ4T3JHiIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Heat stability of triple-stranded DNA and effects of excess deoxyoligonucleotide. M13mp18RF DNA linearized by Sna BI (200 ng) was subjected to triple-stranded DNA formation with a labeled 60mer deoxyoligonucleotide (oligo-3). The DNA samples (20 µl) were then exposed to different temperatures for 10 min in the absence (A) or presence (B) of unlabeled oligo-3 (10 pmol). (A) Autoradiographic patterns in the absence of the unlabeled oligo-3. (B) Autoradiographic patterns in the presence of the unlabeled oligo-3. Throughout (A) and (B): lane 1, treated at 25°C; lane 2, treated at 60°C; lane 3, treated at 65°C; lane 4, treated at 70°C; lane 5, treated at 75°C; lane 6, treated at 80°C; lane 7, treated at 85°C; lane 8, treated at 90°C and lane 9, treated at 95°C. (C) Diagrammatic representation of the results shown in (A) and (B). The intensities of the bands after autoradiography were quantitated by a BAS200 Image Analyzer (Fuji photo film, Tokyo, Japan) and the results are shown as percentages of the intensities of the control sample (treated at 25°C). For details, see Section 2. " path-from-xml="dsi022f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870184/dsi022f3.jpeg" path-from-xml="dsi022f3.jpeg" target="_blank">View large</a><a section="35870184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f3.jpeg?Expires=2147483647&Signature=3P-EdKi39Miwa84wr-wGOtO8ex3Vt4o7vxPC~Z4mR16CqyHoctVtbNCcSRcPuM90KrCu7JkkT1e1zcNt4KJVclBpLmVxIvVxZkBTNvBltwFJxEmppY-UcTD1Pbag0rz-qnQmJoexy~pJ9aDPOxcz~z9EFSzVdTy1i8jflo6qkI7VqTnyKUDNPDdf7l0V0jk6RnbiWD3Y5-QdICYNMcdGsHIT4mQpasze3tEzg0PGapMU0OIOCBTS70oG954jjpK2~G42Cx24G2V5qFV~rsQ2uVsHu9uppNxJ9jQoXmvt4bZf1C7rnmQpEh6DjOV~mlIqg3Zg~GPnR5cU8VchlxHeig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870184&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Heat stability of triple-stranded DNA and effects of excess deoxyoligonucleotide. M13mp18RF DNA linearized by <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with a labeled 60mer deoxyoligonucleotide (oligo-3). The DNA samples (20 µl) were then exposed to different temperatures for 10 min in the absence (A) or presence (B) of unlabeled oligo-3 (10 pmol). (A) Autoradiographic patterns in the absence of the unlabeled oligo-3. (B) Autoradiographic patterns in the presence of the unlabeled oligo-3. Throughout (A) and (B): lane 1, treated at 25°C; lane 2, treated at 60°C; lane 3, treated at 65°C; lane 4, treated at 70°C; lane 5, treated at 75°C; lane 6, treated at 80°C; lane 7, treated at 85°C; lane 8, treated at 90°C and lane 9, treated at 95°C. (C) Diagrammatic representation of the results shown in (A) and (B). The intensities of the bands after autoradiography were quantitated by a BAS200 Image Analyzer (Fuji photo film, Tokyo, Japan) and the results are shown as percentages of the intensities of the control sample (treated at 25°C). For details, see Section 2. </p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f3.jpeg?Expires=2147483647&Signature=HWodJ1M6i42iIFuA8dFeUvL5yzslxONEYuPAOqkYPSV06XsKsy6~kUNC-LRqyvRNTiEaI2l4pXwFih3Dbvogq9szb-f72zYuUnbWUsretMIRU~P8Rr90C9E5ccZqskfBJajcPO2bnSVISq85WXM9v0kIBx9qQZWUq0CH~zN5DypRB6p28UjzFv288c8bhvrz9uUIdZ79BzQy2GKX4GR~NyJh~MxV4~WllJGlNawqmfOP2eHmLfWmrQvDULBrfsMTTaSBCOnO5WYivbmCWhNeksiQTWAmgQHSqf~ZXjnqbLqxt~FJZVfXebNFC6HtNEjhVpmvjKsmHpjLsQ4T3JHiIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Heat stability of triple-stranded DNA and effects of excess deoxyoligonucleotide. M13mp18RF DNA linearized by Sna BI (200 ng) was subjected to triple-stranded DNA formation with a labeled 60mer deoxyoligonucleotide (oligo-3). The DNA samples (20 µl) were then exposed to different temperatures for 10 min in the absence (A) or presence (B) of unlabeled oligo-3 (10 pmol). (A) Autoradiographic patterns in the absence of the unlabeled oligo-3. (B) Autoradiographic patterns in the presence of the unlabeled oligo-3. Throughout (A) and (B): lane 1, treated at 25°C; lane 2, treated at 60°C; lane 3, treated at 65°C; lane 4, treated at 70°C; lane 5, treated at 75°C; lane 6, treated at 80°C; lane 7, treated at 85°C; lane 8, treated at 90°C and lane 9, treated at 95°C. (C) Diagrammatic representation of the results shown in (A) and (B). The intensities of the bands after autoradiography were quantitated by a BAS200 Image Analyzer (Fuji photo film, Tokyo, Japan) and the results are shown as percentages of the intensities of the control sample (treated at 25°C). For details, see Section 2. " path-from-xml="dsi022f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870184/dsi022f3.jpeg" path-from-xml="dsi022f3.jpeg" target="_blank">View large</a><a section="35870184" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f3.jpeg?Expires=2147483647&Signature=3P-EdKi39Miwa84wr-wGOtO8ex3Vt4o7vxPC~Z4mR16CqyHoctVtbNCcSRcPuM90KrCu7JkkT1e1zcNt4KJVclBpLmVxIvVxZkBTNvBltwFJxEmppY-UcTD1Pbag0rz-qnQmJoexy~pJ9aDPOxcz~z9EFSzVdTy1i8jflo6qkI7VqTnyKUDNPDdf7l0V0jk6RnbiWD3Y5-QdICYNMcdGsHIT4mQpasze3tEzg0PGapMU0OIOCBTS70oG954jjpK2~G42Cx24G2V5qFV~rsQ2uVsHu9uppNxJ9jQoXmvt4bZf1C7rnmQpEh6DjOV~mlIqg3Zg~GPnR5cU8VchlxHeig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870184&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Heat stability of triple-stranded DNA and effects of excess deoxyoligonucleotide. M13mp18RF DNA linearized by <em>Sna</em> BI (200 ng) was subjected to triple-stranded DNA formation with a labeled 60mer deoxyoligonucleotide (oligo-3). The DNA samples (20 µl) were then exposed to different temperatures for 10 min in the absence (A) or presence (B) of unlabeled oligo-3 (10 pmol). (A) Autoradiographic patterns in the absence of the unlabeled oligo-3. (B) Autoradiographic patterns in the presence of the unlabeled oligo-3. Throughout (A) and (B): lane 1, treated at 25°C; lane 2, treated at 60°C; lane 3, treated at 65°C; lane 4, treated at 70°C; lane 5, treated at 75°C; lane 6, treated at 80°C; lane 7, treated at 85°C; lane 8, treated at 90°C and lane 9, treated at 95°C. (C) Diagrammatic representation of the results shown in (A) and (B). The intensities of the bands after autoradiography were quantitated by a BAS200 Image Analyzer (Fuji photo film, Tokyo, Japan) and the results are shown as percentages of the intensities of the control sample (treated at 25°C). For details, see Section 2. </p></div></div>                            <h3 scrollto-destination=35870185 id="35870185" class="section-title" >3.4. Detection of a single mismatched base</h3>
<p> We found that the formation of terminal triple-stranded DNA can be used for the detection and identification of a single mismatched base in DNA molecules. The linearized pBR322 DNA (GenBank accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/J01749" target="_blank">J01749</a> , 4361 bp) by digestion with <em>Sca</em> I (nt 3847–3846), in which the C–G pair at nucleotide position 3879 within the terminal 60 bp (nt 3847–3906, <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a> , target DNA I) had been replaced to other 3 bp (target DNA II, III and IV), was subjected to triple-stranded DNA formation. Each target DNA was incubated with one of four labeled deoxyoligonucleotides complementary to the first 60 bases at the 5′-terminal nucleotide sequence but having the base substitution at the position corresponding to nt 3879 of the template DNA ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a> , oligo-8, 9, 10 and 11). In the standard condition, all target DNAs formed stable triple-stranded structures with any combination of deoxyoligonucleotides with or without the base substitution ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a> , left panels), indicating that the formation of stable triple-stranded molecules was possible even in the presence of a single mismatched base in the oligonucleotides at 25°C. However, when the triple-stranded molecules formed at 37°C were exposed to a higher temperature (80°C, 10 min) in the presence of excess amounts of corresponding unlabeled deoxyoligonucleotides, the triple-stranded structures formed with completely matched complementary oligonucleotides were observed ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a> , right panels), while incubation of the triple-stranded DNAs with deoxyoligonucleotides having a single mismatched base resulted in complete dissociation of the structures. Essentially, the same results were obtained with triple-stranded DNA with complementary deoxyoligonucleotides in which a single nucleotide (nt 3879) is deleted ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a> , oligo-12) or an extra nucleotide is inserted ( <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a> , oligo-13, an extra C is inserted at nt 3878–3879). The results are shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4D</a> . <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Figure 4C</a> and Eare ethidium bromide patterns of DNA products. Lanes correspond to those shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B and D</a> , respectively. </p>                        <a id="35870187" scrollto-destination="35870187"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f4.jpeg?Expires=2147483647&Signature=hjKDw5gsUAREKY26pG~OZ4M6UO9zCibRRTPE7QzG3IuuYyQ1a0FV~tbDZqwiXNTKDgQCYCKO86uYR3rI~TstW4AvMsq4ACL2DZ66RUcmzOpcRRg9QIHlH35vnheSLV5h7lDDQXWwj8XcqXZKbaNuC0AmRpNgjotdxPgG6OUWxokKkbhrRe70PMhHy6mZaHOmjKP0m4iH1NyCxOIMGmUCyY1ir6nISGT0V0SY5tVaS9gSRPc5QJVcFNqQs~Bhx68HSndUiGZ4vuK-R52gCO72va9UjJMIG~485DpVUHQ2afm7mXQo-XXEOz8aKyCCgYiA34nY798O1Xmmxr-y9txN5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Detection of a single mismatched base. PBR322 DNA fragments (nt 3847–3846, 4361 bp) (200 ng), in which a single base within the terminal 60 bp (nt 3879) had been substituted by different bases (target DNA I, II, III and IV), were subjected to triple-stranded DNA formation with four labeled complementary deoxyoligonucleotides (nt 3847–3906, 60mer), each with a corresponding mismatched base (oligo-8, 9, 10 and 11). The DNA samples (20 µl) were incubated at 80°C in the presence of unlabeled completely matched deoxyoligonucleotide (10 pmol) (target DNA I with oligo-8, target DNA II with oligo-9, target DNA III with oligo-10 and target DNA IV with oligo-11). (A) Diagrammatic representation of target DNA (I, II, III and IV) in which a single mismatched base is introduced (top panel) and corresponding deoxyoligonucleotides used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number J01749 ). (B) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a mismatched base. Left panel: after incubation at 25°C for 10 min. Right panel: after incubation at 80°C for 10 min. Deoxyoligonucleotides used for triple-stranded DNA formation are shown at the top of each lane. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). (D) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a deleted or extra nucleotide. Target DNA I with triple-stranded structure formed with labeled oligo-8 (control), oligo-11 (mismatched), oligo-12 (deleted) or oligo-13 (inserted) was incubated in the presence of unlabeled oligo-8 (10 pmol) for 10 min at 80°C, subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. Left panel: after incubation at 25°C. Right panel: after incubation at 80°C. (E) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (D). For details, see Section 2. " path-from-xml="dsi022f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870187/dsi022f4.jpeg" path-from-xml="dsi022f4.jpeg" target="_blank">View large</a><a section="35870187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f4.jpeg?Expires=2147483647&Signature=DPpiurbJYQvvh0-F~xhMQD1-S-oNBbegqe3grb6aqL9oRWrpwVadznM4f~1FP3rYxdiL55PiFUpRJ8fg6Mkb0OXdRbTSpgIVOV80LdAB8EMfEtCEydxnL3AfxWibNsyptNQ1-GpO5sDL4Vogn~~wW4RkmIiZiR6t2qSUgWdyWeAZyVxGIXRCjcZTmCvJe-cXuS6mfJuFcW6iivIPePeWlz~bzBQZmYkMyDrNzzMcIfUbZzH22xkkC4TMbbFaKOC8sp9A2CMTDdlX87InsVgOVi1NhGikVuZ6pfsGZr-YHwG4JCgJyphc8CZlpBpsxvNbKxpflJ17mK9zfcrL1HBakQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870187&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Detection of a single mismatched base. PBR322 DNA fragments (nt 3847–3846, 4361 bp) (200 ng), in which a single base within the terminal 60 bp (nt 3879) had been substituted by different bases (target DNA I, II, III and IV), were subjected to triple-stranded DNA formation with four labeled complementary deoxyoligonucleotides (nt 3847–3906, 60mer), each with a corresponding mismatched base (oligo-8, 9, 10 and 11). The DNA samples (20 µl) were incubated at 80°C in the presence of unlabeled completely matched deoxyoligonucleotide (10 pmol) (target DNA I with oligo-8, target DNA II with oligo-9, target DNA III with oligo-10 and target DNA IV with oligo-11). (A) Diagrammatic representation of target DNA (I, II, III and IV) in which a single mismatched base is introduced (top panel) and corresponding deoxyoligonucleotides used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/J01749" target="_blank">J01749</a> ). (B) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a mismatched base. Left panel: after incubation at 25°C for 10 min. Right panel: after incubation at 80°C for 10 min. Deoxyoligonucleotides used for triple-stranded DNA formation are shown at the top of each lane. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). (D) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a deleted or extra nucleotide. Target DNA I with triple-stranded structure formed with labeled oligo-8 (control), oligo-11 (mismatched), oligo-12 (deleted) or oligo-13 (inserted) was incubated in the presence of unlabeled oligo-8 (10 pmol) for 10 min at 80°C, subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. Left panel: after incubation at 25°C. Right panel: after incubation at 80°C. (E) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (D). For details, see Section 2. </p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f4.jpeg?Expires=2147483647&Signature=hjKDw5gsUAREKY26pG~OZ4M6UO9zCibRRTPE7QzG3IuuYyQ1a0FV~tbDZqwiXNTKDgQCYCKO86uYR3rI~TstW4AvMsq4ACL2DZ66RUcmzOpcRRg9QIHlH35vnheSLV5h7lDDQXWwj8XcqXZKbaNuC0AmRpNgjotdxPgG6OUWxokKkbhrRe70PMhHy6mZaHOmjKP0m4iH1NyCxOIMGmUCyY1ir6nISGT0V0SY5tVaS9gSRPc5QJVcFNqQs~Bhx68HSndUiGZ4vuK-R52gCO72va9UjJMIG~485DpVUHQ2afm7mXQo-XXEOz8aKyCCgYiA34nY798O1Xmmxr-y9txN5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Detection of a single mismatched base. PBR322 DNA fragments (nt 3847–3846, 4361 bp) (200 ng), in which a single base within the terminal 60 bp (nt 3879) had been substituted by different bases (target DNA I, II, III and IV), were subjected to triple-stranded DNA formation with four labeled complementary deoxyoligonucleotides (nt 3847–3906, 60mer), each with a corresponding mismatched base (oligo-8, 9, 10 and 11). The DNA samples (20 µl) were incubated at 80°C in the presence of unlabeled completely matched deoxyoligonucleotide (10 pmol) (target DNA I with oligo-8, target DNA II with oligo-9, target DNA III with oligo-10 and target DNA IV with oligo-11). (A) Diagrammatic representation of target DNA (I, II, III and IV) in which a single mismatched base is introduced (top panel) and corresponding deoxyoligonucleotides used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number J01749 ). (B) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a mismatched base. Left panel: after incubation at 25°C for 10 min. Right panel: after incubation at 80°C for 10 min. Deoxyoligonucleotides used for triple-stranded DNA formation are shown at the top of each lane. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). (D) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a deleted or extra nucleotide. Target DNA I with triple-stranded structure formed with labeled oligo-8 (control), oligo-11 (mismatched), oligo-12 (deleted) or oligo-13 (inserted) was incubated in the presence of unlabeled oligo-8 (10 pmol) for 10 min at 80°C, subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. Left panel: after incubation at 25°C. Right panel: after incubation at 80°C. (E) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (D). For details, see Section 2. " path-from-xml="dsi022f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870187/dsi022f4.jpeg" path-from-xml="dsi022f4.jpeg" target="_blank">View large</a><a section="35870187" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f4.jpeg?Expires=2147483647&Signature=DPpiurbJYQvvh0-F~xhMQD1-S-oNBbegqe3grb6aqL9oRWrpwVadznM4f~1FP3rYxdiL55PiFUpRJ8fg6Mkb0OXdRbTSpgIVOV80LdAB8EMfEtCEydxnL3AfxWibNsyptNQ1-GpO5sDL4Vogn~~wW4RkmIiZiR6t2qSUgWdyWeAZyVxGIXRCjcZTmCvJe-cXuS6mfJuFcW6iivIPePeWlz~bzBQZmYkMyDrNzzMcIfUbZzH22xkkC4TMbbFaKOC8sp9A2CMTDdlX87InsVgOVi1NhGikVuZ6pfsGZr-YHwG4JCgJyphc8CZlpBpsxvNbKxpflJ17mK9zfcrL1HBakQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870187&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Detection of a single mismatched base. PBR322 DNA fragments (nt 3847–3846, 4361 bp) (200 ng), in which a single base within the terminal 60 bp (nt 3879) had been substituted by different bases (target DNA I, II, III and IV), were subjected to triple-stranded DNA formation with four labeled complementary deoxyoligonucleotides (nt 3847–3906, 60mer), each with a corresponding mismatched base (oligo-8, 9, 10 and 11). The DNA samples (20 µl) were incubated at 80°C in the presence of unlabeled completely matched deoxyoligonucleotide (10 pmol) (target DNA I with oligo-8, target DNA II with oligo-9, target DNA III with oligo-10 and target DNA IV with oligo-11). (A) Diagrammatic representation of target DNA (I, II, III and IV) in which a single mismatched base is introduced (top panel) and corresponding deoxyoligonucleotides used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/J01749" target="_blank">J01749</a> ). (B) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a mismatched base. Left panel: after incubation at 25°C for 10 min. Right panel: after incubation at 80°C for 10 min. Deoxyoligonucleotides used for triple-stranded DNA formation are shown at the top of each lane. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. (C) Ethidium bromide patterns of DNA products. Lanes correspond to those shown in (B). (D) Autoradiographic patterns of DNA samples with triple-stranded structure carrying a deleted or extra nucleotide. Target DNA I with triple-stranded structure formed with labeled oligo-8 (control), oligo-11 (mismatched), oligo-12 (deleted) or oligo-13 (inserted) was incubated in the presence of unlabeled oligo-8 (10 pmol) for 10 min at 80°C, subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. Left panel: after incubation at 25°C. Right panel: after incubation at 80°C. (E) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (D). For details, see Section 2. </p></div></div>                            <h3 scrollto-destination=35870188 id="35870188" class="section-title" >3.5. Effect of the location of mismatched bases</h3>
<p> A mismatched base can be detected at almost any location in the terminal DNA sequences within at least 120 bp. Triple-stranded DNA was formed at the terminus of <em>Sna</em> BI-treated M13mp18 DNA with labeled 120 bp complementary deoxyoligonucleotides carrying matched ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5A</a> , oligo-14) and mismatched bases ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5A</a> , oligo-15 to 23). A mismatched base (G, guanine) was placed at different sites over the 120 bp sequence, including one located at the very terminus (oligo-23). Triple-stranded samples obtained at 37°C were then exposed to high temperature (85°C, 10 min) in the presence of unlabeled completely matched deoxyoligonucleotide. As seen in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a> , although all triple-stranded structures were stably maintained after incubation at a lower temperature (25°C, 10 min) ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a> , left panel), deoxyoligonucleotides with a mismatched base were eliminated almost completely from the triple-stranded DNA molecules by exposure to high temperature (85°C, 10 min) ( <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a> , right panel). <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5C</a> shows the DNA products detected by ethidium bromide staining. Lanes correspond to those shown in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a> . </p>                        <a id="35870190" scrollto-destination="35870190"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f5.jpeg?Expires=2147483647&Signature=0bvAy9mdAKZ3lLlDY4koAAmH0s418izIhbmcaWpvIDrls6iEvzcHJRVqOWzOQZ79LuTez-w34j8zOACfH9zugYbr0O8yvGlJ6MPlKVqLhLF27FFlXD8BNOAGFKWOnidA~rTECoG6tVQVCL4ScUgCzaFXBl8F1JLlKmVNuoIH3T9hHTG-uCRdWdz0nX-gddIg9F3ln1H857ZFTAwWbnWP0sPKU44XFatCp1bOL6ujsDD6mGmzjzCLwGPwYmjCth82C4gBrk9ap82038rK4Nx7Z1r~UAFWvBSC0uaglrEkdnhxu9olWjvS4JoW6XoVnztmE38ZU-aEzif0D8~Hzf3bmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Effect of the position of mismatched bases. M13mp18 Sna BI fragments (nt 1270–1329, 60 bp) (200 ng) were subjected to terminal triple-stranded DNA formation using a series of labeled complementary deoxyoligonucleotides (nt 1270–1389, 120 mer) with a single mismatched base (guanine) at different positions (oligo-14, 15, 16, 17, 18, 19, 20, 21, 22 and 23). After removal of RecA protein as well as excess deoxyoligonucleotides through a Sephacryl S-400 spin column, samples (20 µl) were incubated at 85°C in the presence of unlabeled completely matched deoxyoligonucleotide (oligo-14) (10 pmol), subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. (A) Diagrammatic representation of M13mp18 Sna BI DNA fragment (top panel) and corresponding deoxyoligonucleotides (oligo-14 to 23) containing a mismatched base, as used for triple-stranded DNA formation (bottom panel). The exact positions of the mismatched bases are as follows. Oligo-14; no mismatched base, oligo-15; a mismatched base at nt 1280 (thymine is replaced by guanine), oligo-16; a mismatched base at nt 1290 (cytidine is replaced by guanine), oligo-17; a mismatched base at nt 1300 (cytidine is replaced by guanine), oligo-18; a mismatched base at nt 1310 (adenosine is replaced by guanine), oligo-19; a mismatched base at nt 1330 (adenosine is replaced by guanine), oligo-20; a mismatched base at nt 1350 (adenosine is replaced by guanine), oligo-21; a mismatched base at nt 1370 (cytidine is replaced by guanine), oligo-22; a mismatched base at nt 1378 (cytidine is replaced by guanine) and oligo-23; a mismatched base at nt 1389 (cytidine is replaced by guanine). Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA samples after incubation at 25°C (left panel) and 85°C (right panel). (C) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (B). For details, see Section 2. " path-from-xml="dsi022f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870190/dsi022f5.jpeg" path-from-xml="dsi022f5.jpeg" target="_blank">View large</a><a section="35870190" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f5.jpeg?Expires=2147483647&Signature=CQ9K5SGCaei5JEmap6SqW1fxomu-BpSnuooSZ8Jr86AQHP9mtC8Y5WRDakmmeSAUAnU-bKjCAOB1KuoFIF9LkW4mT14K1-YOtE~I4TdJyVCg07U-KZDLlhk7n3caBLCwefiS24gHaef~~RoMNM7groagbG2cy~Da6MJ5MQp7TUAXbEMez3p0Y5YOf4n7sgHc25Vv6MQCfiEU0kJr~2jsEok8LJ55BWffH5HxDrCvssvb~nZSgUf3ZbxUT9kaHsBG4P0J6wV2aue8qaJha2yKZUYnGhPXkedxE1dlLHfd1ZYnC1bFErZaUJEdpqy4VT3hx4jjwgSjTvKBv3J28GYyBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870190&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Effect of the position of mismatched bases. M13mp18 <em>Sna</em> BI fragments (nt 1270–1329, 60 bp) (200 ng) were subjected to terminal triple-stranded DNA formation using a series of labeled complementary deoxyoligonucleotides (nt 1270–1389, 120 mer) with a single mismatched base (guanine) at different positions (oligo-14, 15, 16, 17, 18, 19, 20, 21, 22 and 23). After removal of RecA protein as well as excess deoxyoligonucleotides through a Sephacryl S-400 spin column, samples (20 µl) were incubated at 85°C in the presence of unlabeled completely matched deoxyoligonucleotide (oligo-14) (10 pmol), subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. (A) Diagrammatic representation of M13mp18 <em>Sna</em> BI DNA fragment (top panel) and corresponding deoxyoligonucleotides (oligo-14 to 23) containing a mismatched base, as used for triple-stranded DNA formation (bottom panel). The exact positions of the mismatched bases are as follows. Oligo-14; no mismatched base, oligo-15; a mismatched base at nt 1280 (thymine is replaced by guanine), oligo-16; a mismatched base at nt 1290 (cytidine is replaced by guanine), oligo-17; a mismatched base at nt 1300 (cytidine is replaced by guanine), oligo-18; a mismatched base at nt 1310 (adenosine is replaced by guanine), oligo-19; a mismatched base at nt 1330 (adenosine is replaced by guanine), oligo-20; a mismatched base at nt 1350 (adenosine is replaced by guanine), oligo-21; a mismatched base at nt 1370 (cytidine is replaced by guanine), oligo-22; a mismatched base at nt 1378 (cytidine is replaced by guanine) and oligo-23; a mismatched base at nt 1389 (cytidine is replaced by guanine). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA samples after incubation at 25°C (left panel) and 85°C (right panel). (C) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (B). For details, see Section 2. </p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f5.jpeg?Expires=2147483647&Signature=0bvAy9mdAKZ3lLlDY4koAAmH0s418izIhbmcaWpvIDrls6iEvzcHJRVqOWzOQZ79LuTez-w34j8zOACfH9zugYbr0O8yvGlJ6MPlKVqLhLF27FFlXD8BNOAGFKWOnidA~rTECoG6tVQVCL4ScUgCzaFXBl8F1JLlKmVNuoIH3T9hHTG-uCRdWdz0nX-gddIg9F3ln1H857ZFTAwWbnWP0sPKU44XFatCp1bOL6ujsDD6mGmzjzCLwGPwYmjCth82C4gBrk9ap82038rK4Nx7Z1r~UAFWvBSC0uaglrEkdnhxu9olWjvS4JoW6XoVnztmE38ZU-aEzif0D8~Hzf3bmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Effect of the position of mismatched bases. M13mp18 Sna BI fragments (nt 1270–1329, 60 bp) (200 ng) were subjected to terminal triple-stranded DNA formation using a series of labeled complementary deoxyoligonucleotides (nt 1270–1389, 120 mer) with a single mismatched base (guanine) at different positions (oligo-14, 15, 16, 17, 18, 19, 20, 21, 22 and 23). After removal of RecA protein as well as excess deoxyoligonucleotides through a Sephacryl S-400 spin column, samples (20 µl) were incubated at 85°C in the presence of unlabeled completely matched deoxyoligonucleotide (oligo-14) (10 pmol), subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. (A) Diagrammatic representation of M13mp18 Sna BI DNA fragment (top panel) and corresponding deoxyoligonucleotides (oligo-14 to 23) containing a mismatched base, as used for triple-stranded DNA formation (bottom panel). The exact positions of the mismatched bases are as follows. Oligo-14; no mismatched base, oligo-15; a mismatched base at nt 1280 (thymine is replaced by guanine), oligo-16; a mismatched base at nt 1290 (cytidine is replaced by guanine), oligo-17; a mismatched base at nt 1300 (cytidine is replaced by guanine), oligo-18; a mismatched base at nt 1310 (adenosine is replaced by guanine), oligo-19; a mismatched base at nt 1330 (adenosine is replaced by guanine), oligo-20; a mismatched base at nt 1350 (adenosine is replaced by guanine), oligo-21; a mismatched base at nt 1370 (cytidine is replaced by guanine), oligo-22; a mismatched base at nt 1378 (cytidine is replaced by guanine) and oligo-23; a mismatched base at nt 1389 (cytidine is replaced by guanine). Nucleotide numbers are those registered in GenBank (Accession number X02513 ). (B) Autoradiographic patterns of DNA samples after incubation at 25°C (left panel) and 85°C (right panel). (C) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (B). For details, see Section 2. " path-from-xml="dsi022f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870190/dsi022f5.jpeg" path-from-xml="dsi022f5.jpeg" target="_blank">View large</a><a section="35870190" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f5.jpeg?Expires=2147483647&Signature=CQ9K5SGCaei5JEmap6SqW1fxomu-BpSnuooSZ8Jr86AQHP9mtC8Y5WRDakmmeSAUAnU-bKjCAOB1KuoFIF9LkW4mT14K1-YOtE~I4TdJyVCg07U-KZDLlhk7n3caBLCwefiS24gHaef~~RoMNM7groagbG2cy~Da6MJ5MQp7TUAXbEMez3p0Y5YOf4n7sgHc25Vv6MQCfiEU0kJr~2jsEok8LJ55BWffH5HxDrCvssvb~nZSgUf3ZbxUT9kaHsBG4P0J6wV2aue8qaJha2yKZUYnGhPXkedxE1dlLHfd1ZYnC1bFErZaUJEdpqy4VT3hx4jjwgSjTvKBv3J28GYyBg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870190&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Effect of the position of mismatched bases. M13mp18 <em>Sna</em> BI fragments (nt 1270–1329, 60 bp) (200 ng) were subjected to terminal triple-stranded DNA formation using a series of labeled complementary deoxyoligonucleotides (nt 1270–1389, 120 mer) with a single mismatched base (guanine) at different positions (oligo-14, 15, 16, 17, 18, 19, 20, 21, 22 and 23). After removal of RecA protein as well as excess deoxyoligonucleotides through a Sephacryl S-400 spin column, samples (20 µl) were incubated at 85°C in the presence of unlabeled completely matched deoxyoligonucleotide (oligo-14) (10 pmol), subjected to electrophoresis on agarose gel (1%) and autoradiographed using X-ray film. (A) Diagrammatic representation of M13mp18 <em>Sna</em> BI DNA fragment (top panel) and corresponding deoxyoligonucleotides (oligo-14 to 23) containing a mismatched base, as used for triple-stranded DNA formation (bottom panel). The exact positions of the mismatched bases are as follows. Oligo-14; no mismatched base, oligo-15; a mismatched base at nt 1280 (thymine is replaced by guanine), oligo-16; a mismatched base at nt 1290 (cytidine is replaced by guanine), oligo-17; a mismatched base at nt 1300 (cytidine is replaced by guanine), oligo-18; a mismatched base at nt 1310 (adenosine is replaced by guanine), oligo-19; a mismatched base at nt 1330 (adenosine is replaced by guanine), oligo-20; a mismatched base at nt 1350 (adenosine is replaced by guanine), oligo-21; a mismatched base at nt 1370 (cytidine is replaced by guanine), oligo-22; a mismatched base at nt 1378 (cytidine is replaced by guanine) and oligo-23; a mismatched base at nt 1389 (cytidine is replaced by guanine). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/X02513" target="_blank">X02513</a> ). (B) Autoradiographic patterns of DNA samples after incubation at 25°C (left panel) and 85°C (right panel). (C) Ethidium bromide patterns of DNA products. Lanes correspond those shown in (B). For details, see Section 2. </p></div></div>                            <h3 scrollto-destination=35870191 id="35870191" class="section-title" >3.6. Analysis of SNPs in human genomic DNA</h3>
<p> We applied our findings for the analysis of SNPs in human genomic DNA (GenBank accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/AC004465" target="_blank">AC004465</a> ). The region of 950 bp sequences shown in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Figure 6A</a> in which an SNP site of interest is positioned at 40 nucleotides from the terminus was amplified by PCR ( <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6A</a> ). The amplified products from three human genomic DNAs with known allelic variations (C/T, C/C and T/T for human DNA I, II and III, respectively, see <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6A</a> ) were then subjected to terminal triple-stranded DNA formation with a set of labeled deoxyoligonucleotides (oligo-24, 25, 26 and 27, see <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6A</a> ) which cover the primer sequences and SNP sites, and are complementary to one of the terminal sequence of the DNA product amplified from the genomic DNA template. As seen in <a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6B and C</a> , after brief heat treatment of the triple-stranded molecules formed at 85°C, very distinct patterns of the labeled products emerged, clearly distinguishing the three different polymorphisms. </p>                        <a id="35870193" scrollto-destination="35870193"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f6.jpeg?Expires=2147483647&Signature=TYmTzx2Svcfsb8hd5lHVhMfxGDqIgk8JUYLTRCmrnvgmjfQ9rHShRkqxMl2p4Inh1SbkEK-ELQcC93-OEqvn8LOBFsNVkQhH1UE6XQYuZS14Ce-PttAZrBDzSPaz~kjQHlqYiZhEdWKmva9U-FmrKDP7v0y~7TgaCKdMhUJ6Yn6iMXEsMxc1so~yGO3dNA53vx84e~lk-jMC8F0mvHqweTUfqQ73ycgjyJtszOwdlEMact4bjkyT~4cDefX-lgs6TrkThX1uFALk6f21~3~RJD670TarSlJ-HiUGIcVv1BtEmjdFjsY0lgSRf21sjh3Gvltm42WB~AmBZ4JvXxf8cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Analysis of SNPs in human genomic DNA preparations. Sequences (950 bp) covering an SNP site in three human genomic DNA preparations were amplified by PCR. The PCR products were subjected to triple-stranded DNA formation with a set of labeled deoxyoligonucleotides (oligo-24, 25, 26 and 27) which cover the SNP site and are complementary to one of the terminal sequence of the genomic DNA. (A) Diagrammatic representation of DNA sequences with an SNP site (human DNA I, II and III) in which polymorphic bases with allelic variations are indicated (top panel) and deoxyoligonucleotides (60mer covering the SNP site, oligo-24, 25, 26 and 27) used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number AC004465 ). (B) Autoradiographic patterns of DNA samples after heat treatment at 25°C for 10 min. (C) Autoradiographic patterns of DNA samples after heat treatment at 85°C for 10 min. Throughout (B) and (C): lanes 1–4, human DNA I; lanes 5–8, human DNA II; and lanes 9–12, human DNA III. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870193/dsi022f6.jpeg" path-from-xml="dsi022f6.jpeg" target="_blank">View large</a><a section="35870193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f6.jpeg?Expires=2147483647&Signature=uZNdTSZQHP-D4qmvuR-W4vapSlBFWhUb~soPsd~fLUhG9GSXpq29YG-cf7u3jTda~ben4OHwuY1CWUILkymEy4I6yusGtIC27zeoTIRD4KBF-sOtsgMxMHoLXsfegK5DFL~BDTOryChp-C80Y3iG7RkYdx9A81bEXgPKyxZC-T5LQx2IAJQTrlfiIlP6MvU15v8a7jL24rKr6~rTS~qyeKiRradZ520fTu9I765Rs3KUdG4oEQKGQ4T4hQRtmHs1mVBZWS3CFUQNqGCJuF12TcF2F5RLia5z7U-~U3NL-lz37PpR6t-SbnEQSDrQ8mc66foymCNIgTGSsQB65-pShg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870193&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Analysis of SNPs in human genomic DNA preparations. Sequences (950 bp) covering an SNP site in three human genomic DNA preparations were amplified by PCR. The PCR products were subjected to triple-stranded DNA formation with a set of labeled deoxyoligonucleotides (oligo-24, 25, 26 and 27) which cover the SNP site and are complementary to one of the terminal sequence of the genomic DNA. (A) Diagrammatic representation of DNA sequences with an SNP site (human DNA I, II and III) in which polymorphic bases with allelic variations are indicated (top panel) and deoxyoligonucleotides (60mer covering the SNP site, oligo-24, 25, 26 and 27) used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/AC004465" target="_blank">AC004465</a> ). (B) Autoradiographic patterns of DNA samples after heat treatment at 25°C for 10 min. (C) Autoradiographic patterns of DNA samples after heat treatment at 85°C for 10 min. Throughout (B) and (C): lanes 1–4, human DNA I; lanes 5–8, human DNA II; and lanes 9–12, human DNA III. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/m_dsi022f6.jpeg?Expires=2147483647&Signature=TYmTzx2Svcfsb8hd5lHVhMfxGDqIgk8JUYLTRCmrnvgmjfQ9rHShRkqxMl2p4Inh1SbkEK-ELQcC93-OEqvn8LOBFsNVkQhH1UE6XQYuZS14Ce-PttAZrBDzSPaz~kjQHlqYiZhEdWKmva9U-FmrKDP7v0y~7TgaCKdMhUJ6Yn6iMXEsMxc1so~yGO3dNA53vx84e~lk-jMC8F0mvHqweTUfqQ73ycgjyJtszOwdlEMact4bjkyT~4cDefX-lgs6TrkThX1uFALk6f21~3~RJD670TarSlJ-HiUGIcVv1BtEmjdFjsY0lgSRf21sjh3Gvltm42WB~AmBZ4JvXxf8cg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Analysis of SNPs in human genomic DNA preparations. Sequences (950 bp) covering an SNP site in three human genomic DNA preparations were amplified by PCR. The PCR products were subjected to triple-stranded DNA formation with a set of labeled deoxyoligonucleotides (oligo-24, 25, 26 and 27) which cover the SNP site and are complementary to one of the terminal sequence of the genomic DNA. (A) Diagrammatic representation of DNA sequences with an SNP site (human DNA I, II and III) in which polymorphic bases with allelic variations are indicated (top panel) and deoxyoligonucleotides (60mer covering the SNP site, oligo-24, 25, 26 and 27) used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number AC004465 ). (B) Autoradiographic patterns of DNA samples after heat treatment at 25°C for 10 min. (C) Autoradiographic patterns of DNA samples after heat treatment at 85°C for 10 min. Throughout (B) and (C): lanes 1–4, human DNA I; lanes 5–8, human DNA II; and lanes 9–12, human DNA III. Positions of the size markers ( Hin dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. " path-from-xml="dsi022f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870193/dsi022f6.jpeg" path-from-xml="dsi022f6.jpeg" target="_blank">View large</a><a section="35870193" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi022/3/dsi022f6.jpeg?Expires=2147483647&Signature=uZNdTSZQHP-D4qmvuR-W4vapSlBFWhUb~soPsd~fLUhG9GSXpq29YG-cf7u3jTda~ben4OHwuY1CWUILkymEy4I6yusGtIC27zeoTIRD4KBF-sOtsgMxMHoLXsfegK5DFL~BDTOryChp-C80Y3iG7RkYdx9A81bEXgPKyxZC-T5LQx2IAJQTrlfiIlP6MvU15v8a7jL24rKr6~rTS~qyeKiRradZ520fTu9I765Rs3KUdG4oEQKGQ4T4hQRtmHs1mVBZWS3CFUQNqGCJuF12TcF2F5RLia5z7U-~U3NL-lz37PpR6t-SbnEQSDrQ8mc66foymCNIgTGSsQB65-pShg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870193&ar=344455&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi022f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Analysis of SNPs in human genomic DNA preparations. Sequences (950 bp) covering an SNP site in three human genomic DNA preparations were amplified by PCR. The PCR products were subjected to triple-stranded DNA formation with a set of labeled deoxyoligonucleotides (oligo-24, 25, 26 and 27) which cover the SNP site and are complementary to one of the terminal sequence of the genomic DNA. (A) Diagrammatic representation of DNA sequences with an SNP site (human DNA I, II and III) in which polymorphic bases with allelic variations are indicated (top panel) and deoxyoligonucleotides (60mer covering the SNP site, oligo-24, 25, 26 and 27) used for triple-stranded DNA formation (bottom panel). Nucleotide numbers are those registered in GenBank (Accession number <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/AC004465" target="_blank">AC004465</a> ). (B) Autoradiographic patterns of DNA samples after heat treatment at 25°C for 10 min. (C) Autoradiographic patterns of DNA samples after heat treatment at 85°C for 10 min. Throughout (B) and (C): lanes 1–4, human DNA I; lanes 5–8, human DNA II; and lanes 9–12, human DNA III. Positions of the size markers ( <em>Hin</em> dIII digested λ DNA fragments) in kb are shown at the left side of each figure. For details, see Section 2. </p></div></div>                            <h2 scrollto-destination=35870194 id="35870194" class="section-title" >4. Conclusions</h2>
<p> In this article, we have shown that stable triple-stranded DNA can be formed at the termini of linear DNA molecules with oligonucleotides complementary to the DNA strand terminating with 5′ phosphate. The formed structure is unusually stable, withstanding heat treatment at as high as 95°C when excess completely matched oligonucleotides are present. In this respect, it should be noted that Radding and his co-workers reported that the stability of the triple-stranded DNA formed over a long stretch of complementary bases (more than a thousand bases) is affected by the polarity of the third strand. <a reveal-id="b17  b19" data-open="b17  b19" class="link link-ref link-reveal xref-bibr">17–19</a> At present, neither the specific molecular structure of terminal triple-stranded DNA nor the mechanism of its stability is clearly understood. Although we have faced difficulties in obtaining sufficient amounts of terminal triple-stranded DNA for structural analysis by means of CD spectrum or NMR or X-ray crystallography, this line of experiments is currently under way. If successful, the information obtained should provide vital clues to the interesting phenomena associated with terminal triple-stranded DNA structure. </p><p>The observation that the terminal triple-stranded structure which withstands at high temperatures is formed only with completely matched complementary oligonucleotides has made it possible to clearly distinguish triple-stranded DNA with a mismatched base from that with completely matched structure. This in turn offers a simple and efficient way to detect and identify single base substitutions in DNA molecules. We have shown that single base substitutions can be detected at almost any location in DNA molecules within at least 120 bp scanning range. This is in quite contrast to widely used conventional procedures for detection of mismatched bases by means of hybridization between single-stranded DNA target and oligonucleotide probes in respect to range of base pairs covered at once and capacity to identify mismatched bases. Needless to say that as the signals obtained with completely matched oligonucleotides are very clear this procedure can be simplified and be higher throughput by combining with other techniques. Efficient and reliable methods of detecting single nucleotide substitutions in DNA molecules are now in great demand, especially for the analysis of SNPs in higher organisms such as humans. We believe that all the procedures described above based upon the unique nature of the terminal triple-stranded DNA, direct probing and detection of single base substitutions should make current DNA technologies to be more efficient and be utilized in a more versatile manner.</p><p>We thank Osamu Ohara for valuable suggestions and, Koji Okumura and Kazuhiro Kondo for help and encouragement. Thanks are also extended to Masaki Oishi for critical reading the manusript.</p>                            <h2 scrollto-destination=35870199 id="35870199" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">West</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Cassuto</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Howard-Flanders</div> <div class="given-names">P.</div></div>. </div><div class="article-title">RecA protein promotes homologous-pairing, strand-exchange reactions between duplex DNA molecules</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1981</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">2100</div>-<div class="lpage">2104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RecA%20protein%20promotes%20homologous-pairing%2C%20strand-exchange%20reactions%20between%20duplex%20DNA%20molecules&amp;author=S.C.%20West&amp;author=E.%20Cassuto&amp;author=P.%20Howard-Flanders&amp;publication_year=1981&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=78&amp;pages=2100-2104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.78.4.2100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.78.4.2100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.78.4.2100"> </span></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Umlauf</div> <div class="given-names">S.W.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Inman</div> <div class="given-names">R.B.</div></div>. </div><div class="article-title">Triple-helical DNA pairing intermediates formed by recA protein</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1990</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">16898</div>-<div class="lpage">16912</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Triple-helical%20DNA%20pairing%20intermediates%20formed%20by%20recA%20protein&amp;author=S.W.%20Umlauf&amp;author=M.M.%20Cox&amp;author=R.B.%20Inman&amp;publication_year=1990&amp;journal=J.%20Biol.%20Chem.&amp;volume=265&amp;pages=16898-16912" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2170360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hsieh</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Camerini-Otero</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Pairing of homologous DNA sequences by proteins: evidence for three-stranded DNA</div>, <div class="source">Genes Dev.</div> , <div class="year">1990</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">1951</div>-<div class="lpage">1963</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pairing%20of%20homologous%20DNA%20sequences%20by%20proteins%3A%20evidence%20for%20three-stranded%20DNA&amp;author=P.%20Hsieh&amp;author=C.S.%20Camerini-Otero&amp;author=R.D.%20Camerini-Otero&amp;publication_year=1990&amp;journal=Genes%20Dev.&amp;volume=4&amp;pages=1951-1963" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.4.11.1951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.4.11.1951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.4.11.1951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2276627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Golub</div> <div class="given-names">E.I.</div></div>,  <div class="name"><div class="surname">Glazer</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Ward</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Radding</div> <div class="given-names">C.M.</div></div>. </div><div class="article-title">RecA protein-mediated irreversible fixation of an oligodeoxyribonucleotide to specific site in DNA</div>, <div class="source">Mutat. Res.</div> , <div class="year">1996</div>, vol. <div class="volume">351</div> (pg. <div class="fpage">117</div>-<div class="lpage">124</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RecA%20protein-mediated%20irreversible%20fixation%20of%20an%20oligodeoxyribonucleotide%20to%20specific%20site%20in%20DNA&amp;author=E.I.%20Golub&amp;author=P.M.%20Glazer&amp;author=D.C.%20Ward&amp;author=C.M.%20Radding&amp;publication_year=1996&amp;journal=Mutat.%20Res.&amp;volume=351&amp;pages=117-124" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0027-5107(95)00212-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0027-5107(95)00212-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0027-5107(95)00212-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8622705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strobel</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Dervan</div> <div class="given-names">P.B.</div></div>. </div><div class="article-title">Site-specific cleavage of a yeast chromosome by deoxyoligonucleotide-directed triple-helix formation</div>, <div class="source">Science</div> , <div class="year">1990</div>, vol. <div class="volume">249</div> (pg. <div class="fpage">73</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site-specific%20cleavage%20of%20a%20yeast%20chromosome%20by%20deoxyoligonucleotide-directed%20triple-helix%20formation&amp;author=S.A.%20Strobel&amp;author=P.B.%20Dervan&amp;publication_year=1990&amp;journal=Science&amp;volume=249&amp;pages=73-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2195655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2195655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2195655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2195655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koob</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Szybalski</div> <div class="given-names">W.</div></div>. </div><div class="article-title"> Cleaving yeast and <em>Escherichia coli</em> genomes at a single site </div>, <div class="source">Science</div> , <div class="year">1990</div>, vol. <div class="volume">250</div> (pg. <div class="fpage">271</div>-<div class="lpage">273</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cleaving%20yeast%20and%20Escherichia%20coli%20genomes%20at%20a%20single%20site&amp;author=M.%20Koob&amp;author=W.%20Szybalski&amp;publication_year=1990&amp;journal=Science&amp;volume=250&amp;pages=271-273" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.2218529" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.2218529" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.2218529"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2218529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garrison</div> <div class="given-names">P.N.</div></div>,  <div class="name"><div class="surname">Koob</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Barnes</div> <div class="given-names">L.D.</div></div>. </div><div class="article-title"> Physical mapping of the <em>Saccharomyces cerevisiae</em> Ap4A phosphorylaseI-encoding gene by the Achilles' cleavage method </div>, <div class="source">Gene</div> , <div class="year">1992</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">131</div>-<div class="lpage">134</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20mapping%20of%20the%20Saccharomyces%20cerevisiae%20Ap4A%20phosphorylaseI-encoding%20gene%20by%20the%20Achilles'%20cleavage%20method&amp;author=P.N.%20Garrison&amp;author=M.%20Koob&amp;author=L.D.%20Barnes&amp;publication_year=1992&amp;journal=Gene&amp;volume=111&amp;pages=131-134" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(92)90614-U" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(92)90614-U" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(92)90614-U"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1312500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferrin</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Selective cleavage of human DNA: RecA-assisted restriction endonuclease (RARE) cleavage</div>, <div class="source">Science</div> , <div class="year">1992</div>, vol. <div class="volume">254</div> (pg. <div class="fpage">1494</div>-<div class="lpage">1497</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selective%20cleavage%20of%20human%20DNA%3A%20RecA-assisted%20restriction%20endonuclease%20(RARE)%20cleavage&amp;author=L.J.%20Ferrin&amp;author=R.D.%20Camerini-Otero&amp;publication_year=1992&amp;journal=Science&amp;volume=254&amp;pages=1494-1497" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1962209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1962209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1962209"> </span></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Strobel</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Doucette-Stamm</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Riba</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Housman</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Dervan</div> <div class="given-names">P.B.</div></div>. </div><div class="article-title">Site-specific cleavage of human chromosome 4 mediated by triple-helix formation</div>, <div class="source">Science</div> , <div class="year">1991</div>, vol. <div class="volume">254</div> (pg. <div class="fpage">1639</div>-<div class="lpage">1642</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site-specific%20cleavage%20of%20human%20chromosome%204%20mediated%20by%20triple-helix%20formation&amp;author=S.A.%20Strobel&amp;author=L.A.%20Doucette-Stamm&amp;author=L.%20Riba&amp;author=D.E.%20Housman&amp;author=P.B.%20Dervan&amp;publication_year=1991&amp;journal=Science&amp;volume=254&amp;pages=1639-1642" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1836279" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1836279" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1836279"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1836279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hartman</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Kuo</div> <div class="given-names">S.R.</div></div>,  <div class="name"><div class="surname">Broker</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Chow</div> <div class="given-names">L.T.</div></div>,  <div class="name"><div class="surname">Wells</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Intermolecular triplex formation distorts the DNA duplex in the regulatory region of human papillomavirus type-11</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1992</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">5488</div>-<div class="lpage">5494</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intermolecular%20triplex%20formation%20distorts%20the%20DNA%20duplex%20in%20the%20regulatory%20region%20of%20human%20papillomavirus%20type-11&amp;author=D.A.%20Hartman&amp;author=S.R.%20Kuo&amp;author=T.R.%20Broker&amp;author=L.T.%20Chow&amp;author=R.D.%20Wells&amp;publication_year=1992&amp;journal=J.%20Biol.%20Chem.&amp;volume=267&amp;pages=5488-5494" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1312091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rice</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Heckman</div> <div class="given-names">B.M.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kmiec</div> <div class="given-names">E.B.</div></div>. </div><div class="article-title">Fluorescent detection and isolation of DNA variants using stabilized RecA-coated oligonucleotides</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">116</div>-<div class="lpage">125</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fluorescent%20detection%20and%20isolation%20of%20DNA%20variants%20using%20stabilized%20RecA-coated%20oligonucleotides&amp;author=M.C.%20Rice&amp;author=B.M.%20Heckman&amp;author=Y.%20Liu&amp;author=E.B.%20Kmiec&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=116-125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1386204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1386204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1386204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singer</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Podyminogin</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Metcalf</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">Targeted mutagenesis of DNA with alkylating RecA assisted oligonucleotides</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> pg. <div class="fpage">e38</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20mutagenesis%20of%20DNA%20with%20alkylating%20RecA%20assisted%20oligonucleotides&amp;author=M.J.%20Singer&amp;author=M.A.%20Podyminogin&amp;author=M.A.%20Metcalf&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=e38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.24.e38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.24.e38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.24.e38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10572190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujiwara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Oishi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Direct probing: covalent attachment of probe DNA to double-stranded target DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">5728</div>-<div class="lpage">5733</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20probing%3A%20covalent%20attachment%20of%20probe%20DNA%20to%20double-stranded%20target%20DNA&amp;author=J.%20Fujiwara&amp;author=M.%20Oishi&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=5728-5733" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.24.5728" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.24.5728" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.24.5728"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9838005" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujiwara</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Oishi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Direct cloning by covalent attachment of probe DNA to target DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">5734</div>-<div class="lpage">5737</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20cloning%20by%20covalent%20attachment%20of%20probe%20DNA%20to%20target%20DNA&amp;author=J.%20Fujiwara&amp;author=M.%20Oishi&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=5734-5737" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.24.5734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.24.5734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.24.5734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9838006" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ferrin</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Camerini-Otero</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Sequence-specific ligation of DNA using RecA protein</div>, <div class="source">Proc Natl Acad Sci USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">2152</div>-<div class="lpage">2157</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-specific%20ligation%20of%20DNA%20using%20RecA%20protein&amp;author=L.J.%20Ferrin&amp;author=R.D.%20Camerini-Otero&amp;publication_year=1998&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=95&amp;pages=2152-2157" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.5.2152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.5.2152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.5.2152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9482854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Van Houten</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Gamper</div> <div class="given-names">H.B.</div></div>,  <div class="name"><div class="surname">Sancar</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hearst</div> <div class="given-names">J.E.</div></div>. </div><div class="article-title">Use of psoralen-modified deoxyoligonucleotide to trap three-stranded RecA-DNA complexes and repair of these cross-linked complexes by ABC exonuclease</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1988</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">15110</div>-<div class="lpage">15117</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20psoralen-modified%20deoxyoligonucleotide%20to%20trap%20three-stranded%20RecA-DNA%20complexes%20and%20repair%20of%20these%20cross-linked%20complexes%20by%20ABC%20exonuclease&amp;author=S.%20Cheng&amp;author=B.%20Van%20Houten&amp;author=H.B.%20Gamper&amp;author=A.%20Sancar&amp;author=J.E.%20Hearst&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem.&amp;volume=263&amp;pages=15110-15117" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2844801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rao</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Dutreix</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Radding</div> <div class="given-names">C.M.</div></div>. </div><div class="article-title">Stable three-stranded DNA made by RecA protein</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1991</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">2984</div>-<div class="lpage">2988</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stable%20three-stranded%20DNA%20made%20by%20RecA%20protein&amp;author=B.J.%20Rao&amp;author=M.%20Dutreix&amp;author=C.M.%20Radding&amp;publication_year=1991&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=2984-2988" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.8.2984" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.8.2984" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.8.2984"> </span></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Muller</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Burdett</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">West</div> <div class="given-names">S.C.</div></div>. </div><div class="article-title"> Unusual stability of recombination intermediates made by <em>Escherichia coli</em> RecA protein </div>, <div class="source">EMBO J.</div> , <div class="year">1992</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">2685</div>-<div class="lpage">2693</div>)</div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rao</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Chiu</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Radding</div> <div class="given-names">C.M.</div></div>. </div><div class="article-title">Homologous recognition and triplex formation promoted by RecA protein between duplex oligonucleotides and single-stranded DNA</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">229</div> (pg. <div class="fpage">328</div>-<div class="lpage">343</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homologous%20recognition%20and%20triplex%20formation%20promoted%20by%20RecA%20protein%20between%20duplex%20oligonucleotides%20and%20single-stranded%20DNA&amp;author=B.J.%20Rao&amp;author=S.K.%20Chiu&amp;author=C.M.%20Radding&amp;publication_year=1993&amp;journal=J.%20Mol.%20Biol.&amp;volume=229&amp;pages=328-343" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1993.1038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1993.1038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1993.1038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8381491" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=344455&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,646</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi022"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="344455"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f6%2f441%2f344455">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119900005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F6%2F441%2F344455" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/344455'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Stable Triple-Stranded DNA Formation and its Application to the SNP Detection" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi022");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f441%2f344455">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

