

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":1441,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/6/477/481278","@type":"ScholarlyArticle","name":"Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes","datePublished":"2012-11-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2012-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss027","keywords":["successive synonymous codon bias","comparative analysis","prokaryotic genomes","Z scores","translation efficiency"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/6/477/481278","author":[{"name":"Guo, Feng-Biao","affiliation":"Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  China","@type":"Person"},{"name":"Ye, Yuan-Nong","affiliation":"Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  China","@type":"Person"},{"name":"Zhao, Hai-Long","affiliation":"Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  China","@type":"Person"},{"name":"Lin, Dan","affiliation":"Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  China","@type":"Person"},{"name":"Wei, Wen","affiliation":"Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  China","@type":"Person"}],"description":"Abstract.  There has been significant progress in understanding the process of protein translation in recent years. One of the best examples is the discovery of","pageStart":"477","pageEnd":"485","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes" />
    <meta property="og:description" content="Abstract.  There has been significant progress in understanding the process of protein translation in recent years. One of the best examples is the discovery of" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/6/477/481278" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-11-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Guo, Feng-Biao" /><meta name="citation_author_institution" content="Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, China" /><meta name="citation_author" content="Ye, Yuan-Nong" /><meta name="citation_author_institution" content="Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, China" /><meta name="citation_author" content="Zhao, Hai-Long" /><meta name="citation_author_institution" content="Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, China" /><meta name="citation_author" content="Lin, Dan" /><meta name="citation_author_institution" content="Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, China" /><meta name="citation_author" content="Wei, Wen" /><meta name="citation_author_institution" content="Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, China" /><meta name="citation_title" content="Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes" /><meta name="citation_firstpage" content="477" /><meta name="citation_lastpage" content="485" /><meta name="citation_doi" content="10.1093/dnares/dss027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2012/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A role for codon order in translation dynamics; Cell;  citation_year=2010;  citation_volume=141;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=How the sequence of a gene can tune its translation; Cell;  citation_year=2010;  citation_volume=141;  citation_pages=227-9; " /><meta name="citation_reference" content="citation_title=Coding-sequence determinants of gene expression in Escherichia coli; Science;  citation_year=2009;  citation_volume=324;  citation_pages=255-8; " /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation; Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy; Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=3645-50; " /><meta name="citation_reference" content="citation_title=General rules for optimal codon choice; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000556" /><meta name="citation_reference" content="citation_title=Codon catalog usage is a genome strategy modulated for gene expressivity; Nucleic Acids Res.;  citation_year=1981;  citation_volume=9;  citation_pages=r43-74; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system; J. Mol. Biol.;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Selection on codon bias; Ann. Rev. Genet.;  citation_year=2008;  citation_volume=42;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=An evolutionary perspective on synonymous codon usage in unicellular organisms; J. Mol. Evol.;  citation_year=1986;  citation_volume=24;  citation_pages=28-38; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Variation in the strength of selected codon usage bias among bacteria; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=1141-53; " /><meta name="citation_reference" content="citation_title=tRNA properties help shape codon pair preferences in open reading frames; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=1015-27; " /><meta name="citation_reference" content="citation_title=Regularities of context-dependent codon bias in eukaryotic genes; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=1192-7; " /><meta name="citation_reference" content="citation_title=Nonrandom utilization of codon pairs in Escherichia coli; Proc. Natl. Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=3699-703; " /><meta name="citation_reference" content="citation_title=Codon pair utilization biases influence translational elongation step times; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=22801-6; " /><meta name="citation_reference" content="citation_title=Comparative context analysis of codon pairs on an ORFeome scale; Genome Biol.;  citation_year=2005;  citation_volume=6;  citation_pages=R28" /><meta name="citation_reference" content="citation_title=Avoidance of long mononucleotide repeats in codon pair usage; Genetics;  citation_year=2010;  citation_volume=186;  citation_pages=1077-84; " /><meta name="citation_reference" content="citation_title=Codon pairs in the genome of Escherichia coli; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=987-98; " /><meta name="citation_reference" content="citation_title=Structure of the elongating ribosome: arrangement of the two tRNAs before and after translocation; Proc. Natl. Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=945-50; " /><meta name="citation_reference" content="citation_title=tRNA-tRNA interactions within cellular ribosomes; Proc. Natl. Acad. Sci. USA;  citation_year=1989;  citation_volume=86;  citation_pages=4397-401; " /><meta name="citation_reference" content="citation_title=NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D61-5; " /><meta name="citation_reference" content="citation_title=Codon–anticodon pairing: the wobble hypothesis; J. Mol. Biol.;  citation_year=1966;  citation_volume=19;  citation_pages=548-55; " /><meta name="citation_reference" content="citation_publisher=Brooks/Cole Publishing Company, Cengage Learning, Inc., Independence, Kentucky, USA, ; Fundamentals of Biostatistics(7th Edition);  citation_year=2010; " /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications; Nucleic Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes; Nucleic Acids Res.;  citation_year=1986;  citation_volume=14;  citation_pages=5125-43; " /><meta name="citation_reference" content="citation_title=Compositional variation in bacterial genes and proteins with potential expression level; FEBS Lett.;  citation_year=2005;  citation_volume=579;  citation_pages=5205-10; " /><meta name="citation_reference" content="citation_title=Codon and amino acid usage in two major human pathogens of genus Bartonella—optimization between replicational-transcriptional selection, translational control and cost minimization; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in adenoviruses: influence of mutation, selection and protein hydropathy; Virus Res.;  citation_year=2006;  citation_volume=117;  citation_pages=227-36; " /><meta name="citation_reference" content="citation_title=Gene expression profile of the cyanobacterium Synechocystis genome; Gene;  citation_year=2012;  citation_volume=497;  citation_pages=344-52; " /><meta name="citation_reference" content="citation_title=Analyzing gene expression from relative codon usage bias in yeast genome: a statistical significance and biological relevance; Gene;  citation_year=2009;  citation_volume=443;  citation_pages=121-31; " /><meta name="citation_reference" content="citation_title=Predicting gene expression level from relative codon usage bias: an application to Escherichia coli genome; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=13-30; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Bacillus subtilis reflects both translational selection and mutational biases; Nucleic Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=8023-40; " /><meta name="citation_reference" content="citation_title=Codon usage patterns in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens: a review of the considerable within-species diversity; Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=8207-11; " /><meta name="citation_reference" content="citation_title=Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=2279-86; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=The systemic imprint of growth and its uses in ecological (meta)genomics; PLoS Genet.;  citation_year=2010;  citation_volume=6;  citation_pages=e1000808" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/6/477/1301385/dss027.pdf" /><meta name="description" content="Abstract.  There has been significant progress in understanding the process of protein translation in recent years. One of the best examples is the discovery of" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/6/477/481278" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/6/477/481278" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/6/477/481278");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss027';
    </script>
<title>
	Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./481278" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdBcPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdW8sIEZlbmctQmlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBvZiBCaW9pbmZvcm1hdGljcyBhbmQgS2V5IExhYm9yYXRvcnkgZm9yIE5ldXJvSW5mb3JtYXRpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBVbml2ZXJzaXR5IG9mIEVsZWN0cm9uaWMgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBvZiBDaGluYSwgQ2hlbmdkdSA2MTAwNTQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWUsIFl1YW4tTm9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBvZiBCaW9pbmZvcm1hdGljcyBhbmQgS2V5IExhYm9yYXRvcnkgZm9yIE5ldXJvSW5mb3JtYXRpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBVbml2ZXJzaXR5IG9mIEVsZWN0cm9uaWMgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBvZiBDaGluYSwgQ2hlbmdkdSA2MTAwNTQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbywgSGFpLUxvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgb2YgQmlvaW5mb3JtYXRpY3MgYW5kIEtleSBMYWJvcmF0b3J5IGZvciBOZXVyb0luZm9ybWF0aW9uIG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVW5pdmVyc2l0eSBvZiBFbGVjdHJvbmljIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgb2YgQ2hpbmEsIENoZW5nZHUgNjEwMDU0LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpbiwgRGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIG9mIEJpb2luZm9ybWF0aWNzIGFuZCBLZXkgTGFib3JhdG9yeSBmb3IgTmV1cm9JbmZvcm1hdGlvbiBvZiBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFVuaXZlcnNpdHkgb2YgRWxlY3Ryb25pYyBTY2llbmNlIGFuZCBUZWNobm9sb2d5IG9mIENoaW5hLCBDaGVuZ2R1IDYxMDA1NCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWksIFdlbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBvZiBCaW9pbmZvcm1hdGljcyBhbmQgS2V5IExhYm9yYXRvcnkgZm9yIE5ldXJvSW5mb3JtYXRpb24gb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBTY2hvb2wgb2YgTGlmZSBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBVbml2ZXJzaXR5IG9mIEVsZWN0cm9uaWMgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBvZiBDaGluYSwgQ2hlbmdkdSA2MTAwNTQsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJVbml2ZXJzYWwgUGF0dGVybiBhbmQgRGl2ZXJzZSBTdHJlbmd0aHMgb2YgU3VjY2Vzc2l2ZSBTeW5vbnltb3VzIENvZG9uIEJpYXMgaW4gVGhyZWUgRG9tYWlucyBvZiBMaWZlLCBQYXJ0aWN1bGFybHkgQW1vbmcgUHJva2FyeW90aWMgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQ3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJvbGUgZm9yIGNvZG9uIG9yZGVyIGluIHRyYW5zbGF0aW9uIGR5bmFtaWNzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG93IHRoZSBzZXF1ZW5jZSBvZiBhIGdlbmUgY2FuIHR1bmUgaXRzIHRyYW5zbGF0aW9uOyBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2Rpbmctc2VxdWVuY2UgZGV0ZXJtaW5hbnRzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTI1NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGJ1dCBub3QgdGhlIHNhbWU6IHRoZSBjYXVzZXMgYW5kIGNvbnNlcXVlbmNlcyBvZiBjb2RvbiBiaWFzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV2b2x1dGlvbmFyaWx5IGNvbnNlcnZlZCBtZWNoYW5pc20gZm9yIGNvbnRyb2xsaW5nIHRoZSBlZmZpY2llbmN5IG9mIHByb3RlaW4gdHJhbnNsYXRpb247IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MzQ0LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBlZmZpY2llbmN5IGlzIGRldGVybWluZWQgYnkgYm90aCBjb2RvbiBiaWFzIGFuZCBmb2xkaW5nIGVuZXJneTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MzY0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhbCBydWxlcyBmb3Igb3B0aW1hbCBjb2RvbiBjaG9pY2U7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDU1NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGNhdGFsb2cgdXNhZ2UgaXMgYSBnZW5vbWUgc3RyYXRlZ3kgbW9kdWxhdGVkIGZvciBnZW5lIGV4cHJlc3Npdml0eTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1yNDMtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvcnJlbGF0aW9uIGJldHdlZW4gdGhlIGFidW5kYW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIHRyYW5zZmVyIFJOQXMgYW5kIHRoZSBvY2N1cnJlbmNlIG9mIHRoZSByZXNwZWN0aXZlIGNvZG9ucyBpbiBpdHMgcHJvdGVpbiBnZW5lczogYSBwcm9wb3NhbCBmb3IgYSBzeW5vbnltb3VzIGNvZG9uIGNob2ljZSB0aGF0IGlzIG9wdGltYWwgZm9yIHRoZSBFLiBjb2xpIHRyYW5zbGF0aW9uYWwgc3lzdGVtOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBvbiBjb2RvbiBiaWFzOyBBbm4uIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcnkgcGVyc3BlY3RpdmUgb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiB1bmljZWxsdWxhciBvcmdhbmlzbXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yOC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGljZWxsdWxhciBvcmdhbmlzbXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gdGhlIHN0cmVuZ3RoIG9mIHNlbGVjdGVkIGNvZG9uIHVzYWdlIGJpYXMgYW1vbmcgYmFjdGVyaWE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTExNDEtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkEgcHJvcGVydGllcyBoZWxwIHNoYXBlIGNvZG9uIHBhaXIgcHJlZmVyZW5jZXMgaW4gb3BlbiByZWFkaW5nIGZyYW1lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTAxNS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhcml0aWVzIG9mIGNvbnRleHQtZGVwZW5kZW50IGNvZG9uIGJpYXMgaW4gZXVrYXJ5b3RpYyBnZW5lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTE5Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25yYW5kb20gdXRpbGl6YXRpb24gb2YgY29kb24gcGFpcnMgaW4gRXNjaGVyaWNoaWEgY29saTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz0zNjk5LTcwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gcGFpciB1dGlsaXphdGlvbiBiaWFzZXMgaW5mbHVlbmNlIHRyYW5zbGF0aW9uYWwgZWxvbmdhdGlvbiBzdGVwIHRpbWVzOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0yMjgwMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBjb250ZXh0IGFuYWx5c2lzIG9mIGNvZG9uIHBhaXJzIG9uIGFuIE9SRmVvbWUgc2NhbGU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9UjI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXZvaWRhbmNlIG9mIGxvbmcgbW9ub251Y2xlb3RpZGUgcmVwZWF0cyBpbiBjb2RvbiBwYWlyIHVzYWdlOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODY7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBwYWlycyBpbiB0aGUgZ2Vub21lIG9mIEVzY2hlcmljaGlhIGNvbGk7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9OTg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgdGhlIGVsb25nYXRpbmcgcmlib3NvbWU6IGFycmFuZ2VtZW50IG9mIHRoZSB0d28gdFJOQXMgYmVmb3JlIGFuZCBhZnRlciB0cmFuc2xvY2F0aW9uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQS10Uk5BIGludGVyYWN0aW9ucyB3aXRoaW4gY2VsbHVsYXIgcmlib3NvbWVzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT04NjsgIGNpdGF0aW9uX3BhZ2VzPTQzOTctNDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIHJlZmVyZW5jZSBzZXF1ZW5jZXMgKFJlZlNlcSk6IGEgY3VyYXRlZCBub24tcmVkdW5kYW50IHNlcXVlbmNlIGRhdGFiYXNlIG9mIGdlbm9tZXMsIHRyYW5zY3JpcHRzIGFuZCBwcm90ZWluczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9RDYxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9u4oCTYW50aWNvZG9uIHBhaXJpbmc6IHRoZSB3b2JibGUgaHlwb3RoZXNpczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk2NjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTU0OC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUJyb29rcy9Db2xlIFB1Ymxpc2hpbmcgQ29tcGFueSwgQ2VuZ2FnZSBMZWFybmluZywgSW5jLiwgSW5kZXBlbmRlbmNlLCBLZW50dWNreSwgVVNBLCA7IEZ1bmRhbWVudGFscyBvZiBCaW9zdGF0aXN0aWNzKDd0aCBFZGl0aW9uKTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUVHQTU6IG1vbGVjdWxhciBldm9sdXRpb25hcnkgZ2VuZXRpY3MgYW5hbHlzaXMgdXNpbmcgbWF4aW11bSBsaWtlbGlob29kLCBldm9sdXRpb25hcnkgZGlzdGFuY2UsIGFuZCBtYXhpbXVtIHBhcnNpbW9ueSBtZXRob2RzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGFsIFcgYW5kIENsdXN0YWwgWCB2ZXJzaW9uIDIuMDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yOTQ3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBBZGFwdGF0aW9uIEluZGV44oCUYSBtZWFzdXJlIG9mIGRpcmVjdGlvbmFsIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgYmlhcywgYW5kIGl0cyBwb3RlbnRpYWwgYXBwbGljYXRpb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMjgxLTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiB5ZWFzdDogY2x1c3RlciBhbmFseXNpcyBjbGVhcmx5IGRpZmZlcmVudGlhdGVzIGhpZ2hseSBhbmQgbG93bHkgZXhwcmVzc2VkIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz01MTI1LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wb3NpdGlvbmFsIHZhcmlhdGlvbiBpbiBiYWN0ZXJpYWwgZ2VuZXMgYW5kIHByb3RlaW5zIHdpdGggcG90ZW50aWFsIGV4cHJlc3Npb24gbGV2ZWw7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTc5OyAgY2l0YXRpb25fcGFnZXM9NTIwNS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYW5kIGFtaW5vIGFjaWQgdXNhZ2UgaW4gdHdvIG1ham9yIGh1bWFuIHBhdGhvZ2VucyBvZiBnZW51cyBCYXJ0b25lbGxh4oCUb3B0aW1pemF0aW9uIGJldHdlZW4gcmVwbGljYXRpb25hbC10cmFuc2NyaXB0aW9uYWwgc2VsZWN0aW9uLCB0cmFuc2xhdGlvbmFsIGNvbnRyb2wgYW5kIGNvc3QgbWluaW1pemF0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBhZGVub3ZpcnVzZXM6IGluZmx1ZW5jZSBvZiBtdXRhdGlvbiwgc2VsZWN0aW9uIGFuZCBwcm90ZWluIGh5ZHJvcGF0aHk7IFZpcnVzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9MjI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGV4cHJlc3Npb24gcHJvZmlsZSBvZiB0aGUgY3lhbm9iYWN0ZXJpdW0gU3luZWNob2N5c3RpcyBnZW5vbWU7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDk3OyAgY2l0YXRpb25fcGFnZXM9MzQ0LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXppbmcgZ2VuZSBleHByZXNzaW9uIGZyb20gcmVsYXRpdmUgY29kb24gdXNhZ2UgYmlhcyBpbiB5ZWFzdCBnZW5vbWU6IGEgc3RhdGlzdGljYWwgc2lnbmlmaWNhbmNlIGFuZCBiaW9sb2dpY2FsIHJlbGV2YW5jZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NDM7ICBjaXRhdGlvbl9wYWdlcz0xMjEtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpbmcgZ2VuZSBleHByZXNzaW9uIGxldmVsIGZyb20gcmVsYXRpdmUgY29kb24gdXNhZ2UgYmlhczogYW4gYXBwbGljYXRpb24gdG8gRXNjaGVyaWNoaWEgY29saSBnZW5vbWU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gQmFjaWxsdXMgc3VidGlsaXMgcmVmbGVjdHMgYm90aCB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBhbmQgbXV0YXRpb25hbCBiaWFzZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTgwMjMtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIHBhdHRlcm5zIGluIEVzY2hlcmljaGlhIGNvbGksIEJhY2lsbHVzIHN1YnRpbGlzLCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUsIFNjaGl6b3NhY2NoYXJvbXljZXMgcG9tYmUsIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGFuZCBIb21vIHNhcGllbnM6IGEgcmV2aWV3IG9mIHRoZSBjb25zaWRlcmFibGUgd2l0aGluLXNwZWNpZXMgZGl2ZXJzaXR5OyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04MjA3LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBiaWFzIGZyb20gdFJOQSdzIHBvaW50IG9mIHZpZXc6IHJlZHVuZGFuY3ksIHNwZWNpYWxpemF0aW9uLCBhbmQgZWZmaWNpZW50IGRlY29kaW5nIGZvciB0cmFuc2xhdGlvbiBvcHRpbWl6YXRpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjI3OS04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29sdmluZyB0aGUgcmlkZGxlIG9mIGNvZG9uIHVzYWdlIHByZWZlcmVuY2VzOiBhIHRlc3QgZm9yIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz01MDM2LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3lzdGVtaWMgaW1wcmludCBvZiBncm93dGggYW5kIGl0cyB1c2VzIGluIGVjb2xvZ2ljYWwgKG1ldGEpZ2Vub21pY3M7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDgwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xOS82LzQ3Ny8xMzAxMzg1L2RzczAyNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlcmUgaGFzIGJlZW4gc2lnbmlmaWNhbnQgcHJvZ3Jlc3MgaW4gdW5kZXJzdGFuZGluZyB0aGUgcHJvY2VzcyBvZiBwcm90ZWluIHRyYW5zbGF0aW9uIGluIHJlY2VudCB5ZWFycy4gT25lIG9mIHRoZSBiZXN0IGV4YW1wbGVzIGlzIHRoZSBkaXNjb3Zlcnkgb2YiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ4MTI3OCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQdCGNzPTa4lsswbAwk4oRgek9OjA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f477%2f481278" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f477%2f481278" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18878" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/6/1/m_cover.gif?Expires=2147483647&amp;Signature=Ymkrtwq9Mdq2ZQmOzrJQ9eKJPKFscP9Dz2fTIBloB-EFD65VEbXzJ4q~-snakwczzkKy9c50sLOIuFB24I0pJdxFnD8zEvpoQeJN~fsnXK1B-LAdZWf~TapU9jar-9Ld8SBPcVyEg-rfuS2fuiV~p46oyz8WKEmF6jt77FBZlqwLfrKM-gNHPhShm2YFMhabay-3DyoXYu6Q7X3TnYZMYg~174agIvVPSgQxLl9pv~mIvWSGZ03BEh643g2rDaJxMKBM3buLpU~BXQmbxGfuwt5uacMTBRmycBDC6hAgFbX~JMal3DalHVMVxXeDECp8wnY2n4dGA5ngut3kqgWBXQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/6">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7298569">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298569">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298571">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298571">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298576">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298576">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298589">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298589">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298608">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298608">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298613">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298613">Conflict of Interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298615">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298615">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298617">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298619">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298619">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7298621">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7298621">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/6/463/481047">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/6/487/481438">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng-Biao Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Fax. +86 28-8320-8238. Email: <a href="mailto:fbguo@uestc.edu.cn" target="_blank">fbguo@uestc.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng-Biao+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Feng-Biao Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuan-Nong Ye</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuan-Nong+Ye">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ye Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ye Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuan-Nong Ye</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hai-Long Zhao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hai-Long+Zhao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhao H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhao H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hai-Long Zhao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dan Lin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dan+Lin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lin D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lin D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dan Lin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wen Wei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Center of Bioinformatics and Key Laboratory for NeuroInformation of Ministry of Education, School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wen+Wei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wei W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wei W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wen Wei</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 477–485, <a href='https://doi.org/10.1093/dnares/dss027'>https://doi.org/10.1093/dnares/dss027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 November 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 July 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 October 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="481278" href="/dnaresearch/article-pdf/19/6/477/1301385/dss027.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Feng-Biao Guo, Yuan-Nong Ye, Hai-Long Zhao, Dan Lin, Wen Wei;  Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes, <em>DNA Research</em>, Volume 19, Issue 6, 1 December 2012, Pages 477–485, <a href='https://doi.org/10.1093/dnares/dss027'>https://doi.org/10.1093/dnares/dss027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/481278?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/481278?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Universal%20Pattern%20and%20Diverse%20Strengths%20of%20Successive%20Synonymous%20Codon%20Bias%20in%20Three%20Domains%20of%20Life%2C%20Particularly%20Among%20Prokaryotic%20Genomes&amp;publicationDate=2012-11-06&amp;volumeNum=19&amp;issueNum=6&amp;author=Guo%2C%20Feng-Biao%3B%20Ye%2C%20Yuan-Nong&amp;startPage=477&amp;endPage=485&amp;contentId=10.1093%2Fdnares%2Fdss027&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f477%2f481278" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7298569 id="7298569" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>There has been significant progress in understanding the process of protein translation in recent years. One of the best examples is the discovery of usage bias in successive synonymous codons and its role in eukaryotic translation efficiency. We observed here a similar type of bias in the other two life domains, bacteria and archaea, although the bias strength was much smaller than in eukaryotes. Among 136 prokaryotic genomes, 98 were found to have significant bias from random use of successive synonymous codons with Z scores larger than three. Furthermore, significantly different bias strengths were found between prokaryotes grouped by various genomic or biochemical characteristics. Interestingly, the bias strength measured by a general Z score could be fitted well (<em>R</em> = 0.83, <em>P</em> &lt; 10<sup>−15</sup>) by three genomic variables: genome size, G + C content, and tRNA gene number based on multiple linear regression. A different distribution of synonymous codon pairs between protein-coding genes and intergenic sequences suggests that bias is caused by translation selection. The present results indicate that protein translation is tuned by codon (pair) usage, and the intensity of the regulation is associated with genome size, tRNA gene number, and G + C content.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;successive synonymous codon bias&quot;">successive synonymous codon bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative analysis&quot;">comparative analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;prokaryotic genomes&quot;">prokaryotic genomes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Z scores&quot;">Z scores</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;translation efficiency&quot;">translation efficiency</a></div>                            <h2 scrollto-destination=7298571 id="7298571" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Recent studies show that protein-coding genes use codon patterns to fine-tune translation and increase protein synthesis efficiency.<sup><a reveal-id="DSS027C1  DSS027C7" data-open="DSS027C1  DSS027C7" class="link link-ref link-reveal xref-bibr">1–7</a></sup> Three types of codon usages have been proposed to influence translation. First, use of single codons may influence the speed and accuracy of translation.<sup><a reveal-id="DSS027C3" data-open="DSS027C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS027C5  DSS027C11" data-open="DSS027C5  DSS027C11" class="link link-ref link-reveal xref-bibr">5–11</a></sup> Frequent use of ‘preferred codons’ is believed to maximize translation efficiency.<sup><a reveal-id="DSS027C12" data-open="DSS027C12" class="link link-ref link-reveal xref-bibr">12</a></sup> This hypothesis takes previous observations in <em>Escherichia coli</em> and several other unicellular microbes as supporting evidence.<sup><a reveal-id="DSS027C13" data-open="DSS027C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In these small organisms, highly expressed genes are found to have more extreme codon bias, where codon bias denotes non-equilibrium usage of up to six synonymous codons encoding for the same amino acids, with ‘preferred codons’ in highly expressed genes corresponding to the most abundant isoaccepting tRNAs.<sup><a reveal-id="DSS027C12" data-open="DSS027C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>Second, codon pair usage is associated with translation efficiency, wherein a codon pair indicates two successive codons.<sup><a reveal-id="DSS027C14  DSS027C17" data-open="DSS027C14  DSS027C17" class="link link-ref link-reveal xref-bibr">14–17</a></sup> The biased use of codon pairs is a common phenomenon in a wide range of species.<sup><a reveal-id="DSS027C18" data-open="DSS027C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The observed codon pair frequency often deviates from expected values predicted from two single codons. Some codon pairs are overrepresented and others underrepresented. A variety of selective or non-selective factors are responsible for such bias.<sup><a reveal-id="DSS027C19" data-open="DSS027C19" class="link link-ref link-reveal xref-bibr">19</a></sup> One such factor is that codon pair usage affects translation.<sup><a reveal-id="DSS027C15" data-open="DSS027C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS027C20" data-open="DSS027C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In fact, peptide bond formation requires simultaneous accommodation of two codons and of two tRNAs in the ribosomal A and P sites.<sup><a reveal-id="DSS027C21" data-open="DSS027C21" class="link link-ref link-reveal xref-bibr">21</a></sup> For spatial reasons, it is thought that not all codon and tRNA combinations are equally compatible on the ribosome surface.<sup><a reveal-id="DSS027C22" data-open="DSS027C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Some combinations of codon pairs and tRNAs would be advantageous for translation efficiency.<sup><a reveal-id="DSS027C20" data-open="DSS027C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Structural features that regulate tRNA geometry within the ribosome govern codon pair patterns, driving enhanced translational fidelity and/or rate.<sup><a reveal-id="DSS027C14" data-open="DSS027C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Experimental results support such a mechanism.<sup><a reveal-id="DSS027C17" data-open="DSS027C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS027C22" data-open="DSS027C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>Third, an interesting bias of successive synonymous codon pairs was found in eight eukaryotic genomes.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS027C4" data-open="DSS027C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Synonymous codon pair denotes a codon that recurs after its synonymy within a gene, regardless of how many codons encode other amino acids, and requiring only that there are not other synonymies between the two. In this study, bias of synonymous codon pairs denotes a difference between actual and expected frequencies when they are independent. Cannarozzi <em>et al.</em><sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> found a strong tendency to use the same codon a second time as for the first synonymy. There is a bias towards the most closely related synonymous wobble codons, if the same codon is not reused. This predisposition towards selecting particular codons, rather than arbitrarily choosing one from the successive synonymous set, is termed ‘autocorrelation’ or ‘codon reuse’, and has important implications for protein translation. Based on comprehensive analyses of highly expressed genes, it was suggested that codon reuse may provide an effective mechanism to speed up translation.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Through wet-bench experiments, it was successfully demonstrated that translation on autocorrelated mRNA was substantially (30%) faster than on anti-correlated mRNA.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Therefore, this result reinforces the speculation that codon reuse could benefit translation efficiency.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup></p><p>Due to its intimate relationship with translation efficiency, biased use of single codons and codon pairs has been studied extensively. However, biased use of successive codon pairs is relatively new.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Cannarozzi and colleagues observed this phenomenon in eukaryotes.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Here, we performed cross-species analysis of ‘codon reuse’ in 136 prokaryotes. We not only show the existence of ‘codon reuse’ in various prokaryotic genomes and hence illustrate it as a universal mechanism among the three domains of life, but also compare the level of biases among various prokaryotes. Most importantly, we observed that the overall bias intensity for successive synonymous codon pairs is positively correlated with several genomic factors. Using genomic G + C content, genome size, and tRNA gene numbers as limiting factors, the bias value could be predicted with high accuracy. Thus, these data reinforce the notion that the genome contains all the information necessary for regulating protein translation.</p>                            <h2 scrollto-destination=7298576 id="7298576" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
<p>We randomly picked one strain from each of prokaryotic species sequenced. Genome sequences and annotations were downloaded from the NCBI RefSeq project (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria">ftp://ftp.ncbi.nih.gov/genomes/Bacteria</a>) for 136 prokaryotic strains before June 2010. These 136 strains consisted of 109 bacteria and 27 archaea. Information on genome size, G + C content, and tRNA gene number was extracted from the RefSeq annotation.<sup><a reveal-id="DSS027C23" data-open="DSS027C23" class="link link-ref link-reveal xref-bibr">23</a></sup> For all genomes, tRNA was assigned to 64 codons according to the extended wobble rule.<sup><a reveal-id="DSS027C24" data-open="DSS027C24" class="link link-ref link-reveal xref-bibr">24</a></sup> According to the wobble rule, we adopt the consistent pattern for assigning codons to isoaccepting tRNAs for all 136 prokaryotic genomes, and it is similar with Cannarozzi <em>et al</em>.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, the correspondence of codons to tRNAs is illustrated with <em>E. coli</em> as an example. In fact, there may exist some modifications of the wobble rule. For example, large bacterial genomes with high G + C% usually have tRNAs with a C-starting anticodon solely responsible for a G-ending codon, and the number of this tRNA gene is often multiple and larger than that for the respective isoaccepting tRNA responsible for both G- and A-ending codons. Another example is the presence of I in Arg tRNAs found in a wide range of bacteria. While A-ending codons are thought to be recognized by I-containing tRNA on the basis of the extended wobble rule, the efficiency of I-A recognition is low and there are often (but not always) other tRNAs responsible for A-ending (and G-ending) codons. However, we do not know the modification will appear in which specific genome. Therefore, we do not consider any of the modifications when calculating the bias of successive synonymous codon pairs.</p><p>For comparison, correlation of synonymous codon pairs in eight eukaryotes was also investigated (<em>Arabidopsis thaliana</em>, <em>Ashbya gossypii</em>, <em>Caenorhabditis elegans</em>, <em>Candida glabrata</em>, <em>Drosophila melanogaster</em>, <em>Homo sapiens</em>, <em>Saccharomyces cerevisiae</em>, and <em>Schizosaccharomyces pombe</em>).</p><div class="block-child-p">We focused only on pairs of consecutive synonymous codons, which may be separated by any number of codons from other amino acids, in each prokaryotic genome. We used the Z score<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> defined in Equation (1) to evaluate the difference in the actual number from the expected number of consecutive synonymous codon pairs and isoaccepting tRNA pairs. Similar to Cannarozzi <em>et al</em>.,<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> the number of synonymous codon pairs of the nine amino acids with at least two tRNAs was counted. And, the expected number of synonymous codon pairs was calculated as the products of the frequencies of the individual codons of each pair in each prokaryotes. A negative Z score means that the actual frequency is below the expected frequency, whereas a positive Z score means that the former is above the latter.<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The more positive a score is, the stronger is the translation selection in the considered synonymous codon pair. <a id="DSS027M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS027M1"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss027eq1.gif?Expires=2147483647&Signature=bL~e28bsJeMqcmZNAP8EPWIqViKPCpp~gVjeUqjT~kHjWaPphLemX38KFO-Rno4o-hbBZd~dSBGrBjiH2Av3AjafjbkFWzRmJ3Ij-pH0EC4YoeoY6BRDLOE2gKNWEYD4lcXO49wvFIV9ogKqF4nPsTpsKg~g-OBm1iy2uj6woz3Ztwanc4b7AtjZYXF9Ix2p0jTHmMYn6hFMEwBL~bexTprsA0LMwWNDxSMpTxZLXNU0tzY230LnXf0RABpmdGq7VgFl02zdgYD2qdL~V2rIfRIKffaflG~Gs22YkLCYMt-jLvlFy7iEVfnU5i2nIpSxTf48Rfe6tjnHkHGLpKdJEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss027eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss027eq1.gif?Expires=2147483647&Signature=RmpLhFhFRa4wpYJyBLp351ArM0wSJiccKXSLNj-0v931JYRRmjMqDvelEg~TjjDRNSwfZ5fuDpVVRB9C17mST6wt53Oo505rCgNjTsDZpC0Bq4E3CsnrhWpkx93x5xGdI580pB8rTJJPPxj7iPPNoPWdWO0wkCDIEKEOm49vSK5Jutgzecf8BJ~PPwxAWrup6yTfI8pgzz6Wt6Ahj4zuGvRdB7ytPMaypZ~4VYyKiGqZm~HJG6bPLgon6qvusFOS86MEL27HxIVLm98k0XDzgkS~BtUEfCUrmGrlIdGtBbAtfXg1d0BnceWT5jYGbSiPZzt22H2nrEXnu1SNPSc3RQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss027eq1.gif" /></a></div></div><div class="label">(1)</div></div></div><p>The standard deviation in Equation (1) is calculated based on actual numbers and expected numbers of the collection of all synonymous codon pairs or isoaccepting tRNA pairs. The distribution of codon pairs in Fig. <a reveal-id="DSS027F1" data-open="DSS027F1" class="link link-reveal link-table xref-fig">1</a> and genomes in Fig. <a reveal-id="DSS027F2" data-open="DSS027F2" class="link link-reveal link-table xref-fig">2</a> is fitted by the Gauss function.<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> A linear relationship between various genomic factors and the general Z score is fitted with single variable or multiple linear regression.<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Differences in Z scores between two groups of prokaryotic genomes were statistically validated by <em>t</em>-tests.<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> All statistical analyses were implemented with the freely available R package (<a class="link link-uri" href="http://www.r-project.org/">http://www.r-project.org/</a>). </p>                        <a id="7298581" scrollto-destination="7298581"></a>
<div id="DSS027F1" data-id="DSS027F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02701.jpeg?Expires=2147483647&Signature=uRkkyCkQ1BMs3YsJOgAT2hTyqlDZxpW0dBkMAdZvjW6OJQdNWTFUjZmD66d7Pe8HPBoBQT46Rj7qvLgxV-~8mZUobyQ2xJ8qBBCpmBwggLrdg1BKNS4v6Hwfucfae9gaB9h-dsg9W1wzcFwCXC8rPAByNwZNPuD36EmMeOqk7xTHuGz4V3GpydgXeSGD4vWdTGKJ82k1qJE8HOgstgvAaKps8QSWpPzc8I402yeQMjUAHlQF8cKUWR~DHy9iLdb5P7TfPPNQ6h2-WMfM3OTga5~H~Q0ONs4DOsHnT0oj9yoxQSeAyd9kkOGrZBH16SYdhMdJ0t~lJLb4nLaduj0iZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in three types of sequences in E. coli K12. (A) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in gene sequences. The means of the two distributions are different with a P-value = 2.2e-14. (B) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in sequences generated by randomly shuffling. The means of the two distributions are different with a P-value = 3.3e-11. (C) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in intergenic sequences. The means of the two distributions are different with a P-value = 1.0e-3. The difference between the two types of codon pairs for intergenic sequences is not only much smaller than that for the gene sequences but also quite smaller than that for the shuffled gene sequences. Therefore, the pattern of codon reuse is present in protein-coding sequences, and the conserved pattern appears to be rooted in translation selection." path-from-xml="dss02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298581/dss02701.jpeg" path-from-xml="dss02701.jpeg" target="_blank">View large</a><a section="7298581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02701.jpeg?Expires=2147483647&Signature=NRYGmpn1n7eDNWXb2K6dDlEKp~dMVtMD1tFfdrs-z~gm6iKCuREFaITGXEM~JVPaC2QyBYvMYPVtpCzZzdawzuA1a3V-dHvkl7eTvDihIYE5-YgiG8THVR5Iy-5J9XHu96DvfjrVvd3t7ku9TsqDRAwtgxKUIOA5xh2tWE9-eCSMWsDRTezHqgGvq-Q80w8K5sPMZdSUQMmw8CqWMN1~RTE9BRom-XPD8~54k1bpvsKQPJLUbSJQV8L6HK6nJlG-JrS2GYPbjnRY4yaYzNDM5PbrfehTWyBe8qwjVj5zUNJs4XH90iclNs4vnSsCj8Xq-W5buedx~jxYtLWnbJ5EiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298581&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in three types of sequences in <em>E. coli</em> K12. (A) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in gene sequences. The means of the two distributions are different with a <em>P</em>-value = 2.2e-14. (B) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in sequences generated by randomly shuffling. The means of the two distributions are different with a <em>P</em>-value = 3.3e-11. (C) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in intergenic sequences. The means of the two distributions are different with a <em>P</em>-value = 1.0e-3. The difference between the two types of codon pairs for intergenic sequences is not only much smaller than that for the gene sequences but also quite smaller than that for the shuffled gene sequences. Therefore, the pattern of codon reuse is present in protein-coding sequences, and the conserved pattern appears to be rooted in translation selection.</p></div></div><div content-id="DSS027F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02701.jpeg?Expires=2147483647&Signature=uRkkyCkQ1BMs3YsJOgAT2hTyqlDZxpW0dBkMAdZvjW6OJQdNWTFUjZmD66d7Pe8HPBoBQT46Rj7qvLgxV-~8mZUobyQ2xJ8qBBCpmBwggLrdg1BKNS4v6Hwfucfae9gaB9h-dsg9W1wzcFwCXC8rPAByNwZNPuD36EmMeOqk7xTHuGz4V3GpydgXeSGD4vWdTGKJ82k1qJE8HOgstgvAaKps8QSWpPzc8I402yeQMjUAHlQF8cKUWR~DHy9iLdb5P7TfPPNQ6h2-WMfM3OTga5~H~Q0ONs4DOsHnT0oj9yoxQSeAyd9kkOGrZBH16SYdhMdJ0t~lJLb4nLaduj0iZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in three types of sequences in E. coli K12. (A) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in gene sequences. The means of the two distributions are different with a P-value = 2.2e-14. (B) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in sequences generated by randomly shuffling. The means of the two distributions are different with a P-value = 3.3e-11. (C) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in intergenic sequences. The means of the two distributions are different with a P-value = 1.0e-3. The difference between the two types of codon pairs for intergenic sequences is not only much smaller than that for the gene sequences but also quite smaller than that for the shuffled gene sequences. Therefore, the pattern of codon reuse is present in protein-coding sequences, and the conserved pattern appears to be rooted in translation selection." path-from-xml="dss02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298581/dss02701.jpeg" path-from-xml="dss02701.jpeg" target="_blank">View large</a><a section="7298581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02701.jpeg?Expires=2147483647&Signature=NRYGmpn1n7eDNWXb2K6dDlEKp~dMVtMD1tFfdrs-z~gm6iKCuREFaITGXEM~JVPaC2QyBYvMYPVtpCzZzdawzuA1a3V-dHvkl7eTvDihIYE5-YgiG8THVR5Iy-5J9XHu96DvfjrVvd3t7ku9TsqDRAwtgxKUIOA5xh2tWE9-eCSMWsDRTezHqgGvq-Q80w8K5sPMZdSUQMmw8CqWMN1~RTE9BRom-XPD8~54k1bpvsKQPJLUbSJQV8L6HK6nJlG-JrS2GYPbjnRY4yaYzNDM5PbrfehTWyBe8qwjVj5zUNJs4XH90iclNs4vnSsCj8Xq-W5buedx~jxYtLWnbJ5EiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298581&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in three types of sequences in <em>E. coli</em> K12. (A) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in gene sequences. The means of the two distributions are different with a <em>P</em>-value = 2.2e-14. (B) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in sequences generated by randomly shuffling. The means of the two distributions are different with a <em>P</em>-value = 3.3e-11. (C) Z score histograms for two groups (isoaccepting and non-isoaccepting) of codon pairs in intergenic sequences. The means of the two distributions are different with a <em>P</em>-value = 1.0e-3. The difference between the two types of codon pairs for intergenic sequences is not only much smaller than that for the gene sequences but also quite smaller than that for the shuffled gene sequences. Therefore, the pattern of codon reuse is present in protein-coding sequences, and the conserved pattern appears to be rooted in translation selection.</p></div></div>                        <a id="7298585" scrollto-destination="7298585"></a>
<div id="DSS027F2" data-id="DSS027F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02702.jpeg?Expires=2147483647&Signature=QaZSbYwss3M4XjHq6KUDxLZZr8Qwd9v5LqTLSXlIxOlNeIIKSYBxKQy2NMT36Bo5u-4yFRqQ-L19NgLC2vSzgY~nrgT103Q92eCamt7ap~A9SDdxE9Rwht-DOUgbiNlsKZuIFuAnoPEsElqH5yZqArDtOT9ECGZ5uIDVUE8A1wReXbq-so-ZmFQvVK12OfB7Eqi6RgJrpHPPxnYx6R2rR-fvJQGi7J-SpT5GkdWBMOTe~IG~JLBMBsWlSkdR2~AgItrU1HqxT3tqljHbC0Flq4fja7f-ZVN7M-3kbgDNJnNtopaGEBEKlaKVYaBHsGm3Wfh6d5DB-OO3ASIFzr1JOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of the general Z scores among 136 prokaryotic genomes." path-from-xml="dss02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298585/dss02702.jpeg" path-from-xml="dss02702.jpeg" target="_blank">View large</a><a section="7298585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02702.jpeg?Expires=2147483647&Signature=Hibt~vCoCMosIGU7YD2J3mG4D0ceVNJJfkCgspUm0T6YlIUtzfHUSLGW9TBHjX8Wq7h98uiNeT57bMRmEFDUQXyOk2V3kNQAccC9i1qgkM-iV-X04M9lRRBVyOFGiK31fTn-btQKiKM4ZFvc8f9kg4WsZ1AQ74Jzuquu1Xwp8MrvvYeUJAL9K3WH7IlBpkSJaprv05EDuAq2AS~5IuM1SQoH4FkWyDGi20N9LRR0DxXkGScqakehT1YkCCDRXaN5txMQfMIG1~BukKE7w43HPGQ49-fj5FGf1ONowLTdxAbtbUjNfNsu-EBFabKl1DyJqE~PCwEJTShJLBzM~aE3hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298585&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of the general Z scores among 136 prokaryotic genomes.</p></div></div><div content-id="DSS027F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02702.jpeg?Expires=2147483647&Signature=QaZSbYwss3M4XjHq6KUDxLZZr8Qwd9v5LqTLSXlIxOlNeIIKSYBxKQy2NMT36Bo5u-4yFRqQ-L19NgLC2vSzgY~nrgT103Q92eCamt7ap~A9SDdxE9Rwht-DOUgbiNlsKZuIFuAnoPEsElqH5yZqArDtOT9ECGZ5uIDVUE8A1wReXbq-so-ZmFQvVK12OfB7Eqi6RgJrpHPPxnYx6R2rR-fvJQGi7J-SpT5GkdWBMOTe~IG~JLBMBsWlSkdR2~AgItrU1HqxT3tqljHbC0Flq4fja7f-ZVN7M-3kbgDNJnNtopaGEBEKlaKVYaBHsGm3Wfh6d5DB-OO3ASIFzr1JOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of the general Z scores among 136 prokaryotic genomes." path-from-xml="dss02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298585/dss02702.jpeg" path-from-xml="dss02702.jpeg" target="_blank">View large</a><a section="7298585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02702.jpeg?Expires=2147483647&Signature=Hibt~vCoCMosIGU7YD2J3mG4D0ceVNJJfkCgspUm0T6YlIUtzfHUSLGW9TBHjX8Wq7h98uiNeT57bMRmEFDUQXyOk2V3kNQAccC9i1qgkM-iV-X04M9lRRBVyOFGiK31fTn-btQKiKM4ZFvc8f9kg4WsZ1AQ74Jzuquu1Xwp8MrvvYeUJAL9K3WH7IlBpkSJaprv05EDuAq2AS~5IuM1SQoH4FkWyDGi20N9LRR0DxXkGScqakehT1YkCCDRXaN5txMQfMIG1~BukKE7w43HPGQ49-fj5FGf1ONowLTdxAbtbUjNfNsu-EBFabKl1DyJqE~PCwEJTShJLBzM~aE3hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298585&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of the general Z scores among 136 prokaryotic genomes.</p></div></div>                            <h2 scrollto-destination=7298589 id="7298589" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7298590 id="7298590" class="section-title" data-legacy-id=s3a>3.1. Synonymous codon correlation in E. coli K12 genome</h3>
<p>We evaluated 107 bacterial and 29 archaeal genomes. In the following two sections, <em>E. coli</em> K12 is taken as an example. We evaluated all pairs of consecutive synonymous codons in the <em>E. coli</em> K12. Pairs coding for nine amino acids (alanine, arginine, glycine, isoleucine, proline, leucine, serine, threonine, and valine), which have at least two tRNAs, were considered for further analysis. The frequency of all combinations (e.g. TCCTCT as one combination) was then calculated. Assuming a random distribution, the expected number of all combinations could be estimated from the actual single codon frequencies. According to Equation (1), the Z score quantifies the difference between the actual frequency of a combination from the expected frequency, in terms of the number of standard deviations.<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We classified each synonymous codon pair as favoured, if the difference is larger than 3 <span class="small-caps">s.d.</span>, as neutral if between −3 and +3 <span class="small-caps">s.d.</span>, or disfavoured if less than −3 <span class="small-caps">s.d</span>. The numbers for three groups of synonymous codon pairs for each amino acid are summarized in Table <a reveal-id="DSS027TB1" data-open="DSS027TB1" class="link link-reveal link-table xref-fig">1</a>. Among codon pairs with isoacceptors (sharing a tRNA), favoured numbers are all larger than neutral or disfavoured ones. However, the opposite result is obtained for synonymous codon pairs without isoacceptors. These results indicate that the reuse of codons sharing the same tRNA is a universal phenomenon for the nine <em>E. coli</em> amino acids examined. This observation is similar to that seen in eukaryotes.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> It is worth noting that the strength of synonymous codon correlation in <em>E. coli</em> is much smaller than that in yeast. For example, the mean Z score for 10 groups of synonymous codon pairs encoding the amino acid serine is 8.6469 in yeast<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and only 5.8762 in <em>E. coli</em>. As mentioned below, Z scores of synonymous codon pairs in prokaryotes are generally lower than those seen in eukaryotes. </p>                        <a id="7298592" scrollto-destination="7298592"></a>
<div content-id="DSS027TB1" class="table-modal"><div class="table-wrap"><div id="DSS027TB1" class="table-wrap-title" data-id="DSS027TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The numbers of three kinds of synonymous codon pairs for nine amino acids in <em>E. coli</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Isoaccepting<hr /> </th><th colspan="3">Non-isoaccepting<hr /> </th></tr><tr><th>Grouped by </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th></tr></thead><tbody><tr><td>Ala </td><td>4 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td><td>6 </td></tr><tr><td>Arg </td><td>6 </td><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>10 </td></tr><tr><td>Gly </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>1 </td><td>7 </td></tr><tr><td>Ile </td><td>3 </td><td>0 </td><td>2 </td><td>0 </td><td>2 </td><td>2 </td></tr><tr><td>Leu </td><td>5 </td><td>3 </td><td>2 </td><td>6 </td><td>13 </td><td>7 </td></tr><tr><td>Pro </td><td>6 </td><td>0 </td><td>2 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Ser </td><td>8 </td><td>2 </td><td>0 </td><td>1 </td><td>11 </td><td>14 </td></tr><tr><td>Thr </td><td>4 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td>Val </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>4 </td></tr><tr><td>Total </td><td>44 </td><td>15 </td><td>12 </td><td>26 </td><td>40 </td><td>60 </td></tr></tbody></table></div><div content-id="DSS027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Isoaccepting<hr /> </th><th colspan="3">Non-isoaccepting<hr /> </th></tr><tr><th>Grouped by </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th></tr></thead><tbody><tr><td>Ala </td><td>4 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td><td>6 </td></tr><tr><td>Arg </td><td>6 </td><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>10 </td></tr><tr><td>Gly </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>1 </td><td>7 </td></tr><tr><td>Ile </td><td>3 </td><td>0 </td><td>2 </td><td>0 </td><td>2 </td><td>2 </td></tr><tr><td>Leu </td><td>5 </td><td>3 </td><td>2 </td><td>6 </td><td>13 </td><td>7 </td></tr><tr><td>Pro </td><td>6 </td><td>0 </td><td>2 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Ser </td><td>8 </td><td>2 </td><td>0 </td><td>1 </td><td>11 </td><td>14 </td></tr><tr><td>Thr </td><td>4 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td>Val </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>4 </td></tr><tr><td>Total </td><td>44 </td><td>15 </td><td>12 </td><td>26 </td><td>40 </td><td>60 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Codon pairs are grouped into those with and without isoacceptors (sharing a tRNA), by parsimony. Within each group, pairs were classified as favoured (≥3 <span class="small-caps">s.d.</span>), neutral (between −3 and +3 <span class="small-caps">s.d.</span>), or disfavoured (≤−3 <span class="small-caps">s.d.</span>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298592" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB1" class="table-wrap-title" data-id="DSS027TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The numbers of three kinds of synonymous codon pairs for nine amino acids in <em>E. coli</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Isoaccepting<hr /> </th><th colspan="3">Non-isoaccepting<hr /> </th></tr><tr><th>Grouped by </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th></tr></thead><tbody><tr><td>Ala </td><td>4 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td><td>6 </td></tr><tr><td>Arg </td><td>6 </td><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>10 </td></tr><tr><td>Gly </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>1 </td><td>7 </td></tr><tr><td>Ile </td><td>3 </td><td>0 </td><td>2 </td><td>0 </td><td>2 </td><td>2 </td></tr><tr><td>Leu </td><td>5 </td><td>3 </td><td>2 </td><td>6 </td><td>13 </td><td>7 </td></tr><tr><td>Pro </td><td>6 </td><td>0 </td><td>2 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Ser </td><td>8 </td><td>2 </td><td>0 </td><td>1 </td><td>11 </td><td>14 </td></tr><tr><td>Thr </td><td>4 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td>Val </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>4 </td></tr><tr><td>Total </td><td>44 </td><td>15 </td><td>12 </td><td>26 </td><td>40 </td><td>60 </td></tr></tbody></table></div><div content-id="DSS027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Isoaccepting<hr /> </th><th colspan="3">Non-isoaccepting<hr /> </th></tr><tr><th>Grouped by </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th><th>Favoured </th><th>Neutral </th><th>Disfavoured </th></tr></thead><tbody><tr><td>Ala </td><td>4 </td><td>2 </td><td>2 </td><td>1 </td><td>1 </td><td>6 </td></tr><tr><td>Arg </td><td>6 </td><td>2 </td><td>4 </td><td>11 </td><td>3 </td><td>10 </td></tr><tr><td>Gly </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>1 </td><td>7 </td></tr><tr><td>Ile </td><td>3 </td><td>0 </td><td>2 </td><td>0 </td><td>2 </td><td>2 </td></tr><tr><td>Leu </td><td>5 </td><td>3 </td><td>2 </td><td>6 </td><td>13 </td><td>7 </td></tr><tr><td>Pro </td><td>6 </td><td>0 </td><td>2 </td><td>2 </td><td>2 </td><td>4 </td></tr><tr><td>Ser </td><td>8 </td><td>2 </td><td>0 </td><td>1 </td><td>11 </td><td>14 </td></tr><tr><td>Thr </td><td>4 </td><td>2 </td><td>0 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td>Val </td><td>4 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>4 </td></tr><tr><td>Total </td><td>44 </td><td>15 </td><td>12 </td><td>26 </td><td>40 </td><td>60 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Codon pairs are grouped into those with and without isoacceptors (sharing a tRNA), by parsimony. Within each group, pairs were classified as favoured (≥3 <span class="small-caps">s.d.</span>), neutral (between −3 and +3 <span class="small-caps">s.d.</span>), or disfavoured (≤−3 <span class="small-caps">s.d.</span>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298592" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7298593 id="7298593" class="section-title" data-legacy-id=s3b>3.2. Confirming the hypothesis of translation selection</h3>
<p>One reason may account for the codon correlation illustrated above. There is selection pressure for codon ordering in protein-coding genes, and, hence, synonymous codons sharing the same tRNA are successively used.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> However, there also exists a second explanation.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Different genes may be enriched in different codons, and the correlation observed at the genomic level may be due to the accumulation of given codons in specific genes. If the second case is real, the synonymous codon correlation should remain, if codon distribution is shuffled within each gene individually.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> In the first case, such codon shuffling would reduce the difference of codon correlation between isoacceptor pairs and non-isoacceptor pairs. We performed shuffle experiments in <em>E. coli</em> K12 to test which hypothesis is correct. Results are shown in Fig. <a reveal-id="DSS027F1" data-open="DSS027F1" class="link link-reveal link-table xref-fig">1</a>, where the vertical axis denotes synonymous codon pair frequency in the corresponding range of Z scores. Similar to Cannarozzi <em>et al</em>.,<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> codon correlations were found to decrease for isoaccepting pairs for the shuffled genes and increase for non-isoaccepting pairs. The difference between the isoaccepting pairs and non-isoaccepting pairs without shuffle (Fig. <a reveal-id="DSS027F1" data-open="DSS027F1" class="link link-reveal link-table xref-fig">1</a>A, <em>t</em>-test, <em>P</em> = 2.2 × 10<sup>−14</sup>) is significantly larger than the difference after shuffle (Fig. <a reveal-id="DSS027F1" data-open="DSS027F1" class="link link-reveal link-table xref-fig">1</a>B, <em>t</em>-test, <em>P</em> = 3.3 × 10<sup>−11</sup>). Thus, autocorrelation was not simply due to codon bias at the gene level, but due to codon ordering within genes. To reinforce the hypothesis of translation selection, we also analysed synonymous codon correlation in non-coding regions (i.e. intergenic sequences). The Z scores histograms for isoaccepting and non-isoaccepting pairs are shown in Fig. <a reveal-id="DSS027F1" data-open="DSS027F1" class="link link-reveal link-table xref-fig">1</a>C (<em>t</em>-test, <em>P</em> = 1.0 × 10<sup>−3</sup>). The difference between the two types of codon pairs for intergenic sequences is not only much smaller than that for the gene sequences but also smaller than that for the shuffled gene sequences. Based on this analysis and that by Cannarozzi <em>et al</em>.,<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> the pattern of codon reuse exists in protein-coding sequences and reinforces the hypothesis that the conserved pattern is rooted in translation selection.</p>                            <h3 scrollto-destination=7298595 id="7298595" class="section-title" data-legacy-id=s3c>3.3. Varied strengths of codon reuse in prokaryotic genomes</h3>
<p>We investigated synonymous codon correlation in 107 bacteria and 29 archaea. Taxonomic distribution of these genomes is summarized in Table <a reveal-id="DSS027TB2" data-open="DSS027TB2" class="link link-reveal link-table xref-fig">2</a>. As can be seen, 107 bacteria are widely distributed across 14 phyla, and 29 archaea are distributed across 5 phyla. Therefore, most known prokaryotic phyla have representatives in our dataset. To compare the strength of synonymous codon correlation among different genomes, we calculated the Z general score for each genome. This value is equal to the Z score averaged for nine amino acids, and for each amino acid, the Z score is averaged for all isoaccepting tRNA pairs. First, we compared the general Z sores in the three domains of life. Among the eight eukaryotes, the mean value of the general Z scores is 15.0. However, the value is as small as 5.6 in bacteria and 3.2 in archaea. This indicates that the strength of codon reuse is much smaller in prokaryotes than in eukaryotes, although the strength in the former is also significant. The 136 prokaryotic genomes, collectively, have a mean Z score of 5.1 and standard deviation of 3.1. <em>Streptomyces coelicolor</em> A3 has the largest Z score of 15.1. Note that this genome has the highest G + C content and is one of largest of the 136 prokaryote genomes. In contrast, <em>Nanoarchaeum equitans</em> Kin4-M has the smallest Z score of 0.72 and is also the smallest genome. Ninety-eight prokaryotic genomes have Z scores larger than 3.0, indicating that 72% of the genomes have significant usage bias of successive synonymous codons. Histogram of the general Z scores is shown in Fig. <a reveal-id="DSS027F2" data-open="DSS027F2" class="link link-reveal link-table xref-fig">2</a>, and, as can be seen, the genera distribution could not be well fitted by a simple Gaussian function because it has two peaks. </p>                        <a id="7298597" scrollto-destination="7298597"></a>
<div content-id="DSS027TB2" class="table-modal"><div class="table-wrap"><div id="DSS027TB2" class="table-wrap-title" data-id="DSS027TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Taxonomic distribution of 136 prokaryotic genomes analysed in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Phylum no. </th><th>Class no. </th><th>Order no. </th><th>Family no. </th><th>Genus no. </th><th>Species no. </th></tr></thead><tbody><tr><td>Bacteria </td><td>14 </td><td>19 </td><td>35 </td><td>55 </td><td>76 </td><td>107 </td></tr><tr><td>Archaea </td><td>5 </td><td>14 </td><td>18 </td><td>27 </td><td>29 </td><td>29 </td></tr></tbody></table></div><div content-id="DSS027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Phylum no. </th><th>Class no. </th><th>Order no. </th><th>Family no. </th><th>Genus no. </th><th>Species no. </th></tr></thead><tbody><tr><td>Bacteria </td><td>14 </td><td>19 </td><td>35 </td><td>55 </td><td>76 </td><td>107 </td></tr><tr><td>Archaea </td><td>5 </td><td>14 </td><td>18 </td><td>27 </td><td>29 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298597" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB2" class="table-wrap-title" data-id="DSS027TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Taxonomic distribution of 136 prokaryotic genomes analysed in this study</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Phylum no. </th><th>Class no. </th><th>Order no. </th><th>Family no. </th><th>Genus no. </th><th>Species no. </th></tr></thead><tbody><tr><td>Bacteria </td><td>14 </td><td>19 </td><td>35 </td><td>55 </td><td>76 </td><td>107 </td></tr><tr><td>Archaea </td><td>5 </td><td>14 </td><td>18 </td><td>27 </td><td>29 </td><td>29 </td></tr></tbody></table></div><div content-id="DSS027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Phylum no. </th><th>Class no. </th><th>Order no. </th><th>Family no. </th><th>Genus no. </th><th>Species no. </th></tr></thead><tbody><tr><td>Bacteria </td><td>14 </td><td>19 </td><td>35 </td><td>55 </td><td>76 </td><td>107 </td></tr><tr><td>Archaea </td><td>5 </td><td>14 </td><td>18 </td><td>27 </td><td>29 </td><td>29 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298597" target="_blank">View Large</a></div></div><p>The phylogenetic relationship among the 136 prokaryotic genomes is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. This tree is constructed using the neighbor-joining method<sup><a reveal-id="DSS027C26" data-open="DSS027C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS027C27" data-open="DSS027C27" class="link link-ref link-reveal xref-bibr">27</a></sup> based on 16S rRNA sequences. To comprehend the phylogenetic tree, some factors (e.g. taxonomy ID, organism name, general Z score, genome size, and G + C content) of 136 prokaryotic genomes are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. For each genome, the general Z score is marked on the right side of the name. When visualized in this manner, it is clear that the strength of synonymous codon correlation changes with phylogeny. In fact, there is no consistent pattern for Z score variation among different phylogenetic groups. Some groups have very similar scores, whereas others do not. To illustrate the trend from the phylogenetic order, Tables <a reveal-id="DSS027TB3" data-open="DSS027TB3" class="link link-reveal link-table xref-fig">3</a>, <a reveal-id="DSS027TB4" data-open="DSS027TB4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSS027TB5" data-open="DSS027TB5" class="link link-reveal link-table xref-fig">5</a> list the mean Z scores and standard deviation at three levels: phylum, family, and genus. As can be seen, the mean standard deviation changes from 1.53 to 1.42 to 0.90, when the classifying level changes from phylum to family to genus. Correspondingly, the mean variance coefficient changes from 0.34 to 0.24 to 0.21. Therefore, it appears to naturally follow that the Z scores will be more similar at a lower phylogenetic level. </p>                        <a id="7298599" scrollto-destination="7298599"></a>
<div content-id="DSS027TB3" class="table-modal"><div class="table-wrap"><div id="DSS027TB3" class="table-wrap-title" data-id="DSS027TB3"><div class="label -label">Table 3.</div><div class="caption"><p>The mean Z scores and SD at the level of phylum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Phylum </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/mean) </th></tr></thead><tbody><tr><td>Actinobacteria </td><td>7.49 </td><td>3.74 </td><td>8 </td><td>0.50 </td></tr><tr><td>Chlamydiae </td><td>1.61 </td><td>0.41 </td><td>5 </td><td>0.26 </td></tr><tr><td>Firmicutes </td><td>4.60 </td><td>1.37 </td><td>21 </td><td>0.30 </td></tr><tr><td>Proteobacteria </td><td>6.70 </td><td>3.31 </td><td>54 </td><td>0.49 </td></tr><tr><td>Spirochaetes </td><td>3.03 </td><td>0.31 </td><td>4 </td><td>0.10 </td></tr><tr><td>Tenericutes </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td>Crenarchaeota </td><td>3.16 </td><td>0.71 </td><td>6 </td><td>0.23 </td></tr><tr><td>Euryarchaeota </td><td>3.39 </td><td>1.57 </td><td>20 </td><td>0.46 </td></tr><tr><td> </td><td>4.00 </td><td>1.53 </td><td>15.4 </td><td>0.34 </td></tr></tbody></table></div><div content-id="DSS027TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Phylum </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/mean) </th></tr></thead><tbody><tr><td>Actinobacteria </td><td>7.49 </td><td>3.74 </td><td>8 </td><td>0.50 </td></tr><tr><td>Chlamydiae </td><td>1.61 </td><td>0.41 </td><td>5 </td><td>0.26 </td></tr><tr><td>Firmicutes </td><td>4.60 </td><td>1.37 </td><td>21 </td><td>0.30 </td></tr><tr><td>Proteobacteria </td><td>6.70 </td><td>3.31 </td><td>54 </td><td>0.49 </td></tr><tr><td>Spirochaetes </td><td>3.03 </td><td>0.31 </td><td>4 </td><td>0.10 </td></tr><tr><td>Tenericutes </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td>Crenarchaeota </td><td>3.16 </td><td>0.71 </td><td>6 </td><td>0.23 </td></tr><tr><td>Euryarchaeota </td><td>3.39 </td><td>1.57 </td><td>20 </td><td>0.46 </td></tr><tr><td> </td><td>4.00 </td><td>1.53 </td><td>15.4 </td><td>0.34 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>VC denotes variance coefficient in Table 3, 4 and 5, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298599" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB3" class="table-wrap-title" data-id="DSS027TB3"><div class="label -label">Table 3.</div><div class="caption"><p>The mean Z scores and SD at the level of phylum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Phylum </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/mean) </th></tr></thead><tbody><tr><td>Actinobacteria </td><td>7.49 </td><td>3.74 </td><td>8 </td><td>0.50 </td></tr><tr><td>Chlamydiae </td><td>1.61 </td><td>0.41 </td><td>5 </td><td>0.26 </td></tr><tr><td>Firmicutes </td><td>4.60 </td><td>1.37 </td><td>21 </td><td>0.30 </td></tr><tr><td>Proteobacteria </td><td>6.70 </td><td>3.31 </td><td>54 </td><td>0.49 </td></tr><tr><td>Spirochaetes </td><td>3.03 </td><td>0.31 </td><td>4 </td><td>0.10 </td></tr><tr><td>Tenericutes </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td>Crenarchaeota </td><td>3.16 </td><td>0.71 </td><td>6 </td><td>0.23 </td></tr><tr><td>Euryarchaeota </td><td>3.39 </td><td>1.57 </td><td>20 </td><td>0.46 </td></tr><tr><td> </td><td>4.00 </td><td>1.53 </td><td>15.4 </td><td>0.34 </td></tr></tbody></table></div><div content-id="DSS027TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Phylum </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/mean) </th></tr></thead><tbody><tr><td>Actinobacteria </td><td>7.49 </td><td>3.74 </td><td>8 </td><td>0.50 </td></tr><tr><td>Chlamydiae </td><td>1.61 </td><td>0.41 </td><td>5 </td><td>0.26 </td></tr><tr><td>Firmicutes </td><td>4.60 </td><td>1.37 </td><td>21 </td><td>0.30 </td></tr><tr><td>Proteobacteria </td><td>6.70 </td><td>3.31 </td><td>54 </td><td>0.49 </td></tr><tr><td>Spirochaetes </td><td>3.03 </td><td>0.31 </td><td>4 </td><td>0.10 </td></tr><tr><td>Tenericutes </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td>Crenarchaeota </td><td>3.16 </td><td>0.71 </td><td>6 </td><td>0.23 </td></tr><tr><td>Euryarchaeota </td><td>3.39 </td><td>1.57 </td><td>20 </td><td>0.46 </td></tr><tr><td> </td><td>4.00 </td><td>1.53 </td><td>15.4 </td><td>0.34 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>VC denotes variance coefficient in Table 3, 4 and 5, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298599" target="_blank">View Large</a></div></div>                        <a id="7298600" scrollto-destination="7298600"></a>
<div content-id="DSS027TB4" class="table-modal"><div class="table-wrap"><div id="DSS027TB4" class="table-wrap-title" data-id="DSS027TB4"><div class="label -label">Table 4.</div><div class="caption"><p>The mean Z scores and SD at the level of family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Family </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacteriaceae </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydiaceae </td><td>1.44 </td><td>0.19 </td><td>4 </td><td>0.13 </td></tr><tr><td>Bacillaceae </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillaceae </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcaceae </td><td>4.92 </td><td>1.08 </td><td>5 </td><td>0.22 </td></tr><tr><td>Brucellaceae </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Burkholderiaceae </td><td>8.78 </td><td>1.76 </td><td>3 </td><td>0.20 </td></tr><tr><td>Neisseriaceae </td><td>10.72 </td><td>3.85 </td><td>3 </td><td>0.36 </td></tr><tr><td>Enterobacteriaceae </td><td>7.43 </td><td>4.29 </td><td>7 </td><td>0.58 </td></tr><tr><td>Pasteurellaceae </td><td>4.32 </td><td>0.68 </td><td>3 </td><td>0.16 </td></tr><tr><td>Vibrionaceae </td><td>8.02 </td><td>0.66 </td><td>5 </td><td>0.08 </td></tr><tr><td>Xanthomonadaceae </td><td>9.75 </td><td>1.56 </td><td>4 </td><td>0.16 </td></tr><tr><td>Spirochaetaceae </td><td>2.98 </td><td>0.37 </td><td>3 </td><td>0.12 </td></tr><tr><td>Mycoplasmataceae </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td> </td><td>5.74 </td><td>1.42 </td><td>3.9 </td><td>0.24 </td></tr></tbody></table></div><div content-id="DSS027TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Family </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacteriaceae </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydiaceae </td><td>1.44 </td><td>0.19 </td><td>4 </td><td>0.13 </td></tr><tr><td>Bacillaceae </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillaceae </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcaceae </td><td>4.92 </td><td>1.08 </td><td>5 </td><td>0.22 </td></tr><tr><td>Brucellaceae </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Burkholderiaceae </td><td>8.78 </td><td>1.76 </td><td>3 </td><td>0.20 </td></tr><tr><td>Neisseriaceae </td><td>10.72 </td><td>3.85 </td><td>3 </td><td>0.36 </td></tr><tr><td>Enterobacteriaceae </td><td>7.43 </td><td>4.29 </td><td>7 </td><td>0.58 </td></tr><tr><td>Pasteurellaceae </td><td>4.32 </td><td>0.68 </td><td>3 </td><td>0.16 </td></tr><tr><td>Vibrionaceae </td><td>8.02 </td><td>0.66 </td><td>5 </td><td>0.08 </td></tr><tr><td>Xanthomonadaceae </td><td>9.75 </td><td>1.56 </td><td>4 </td><td>0.16 </td></tr><tr><td>Spirochaetaceae </td><td>2.98 </td><td>0.37 </td><td>3 </td><td>0.12 </td></tr><tr><td>Mycoplasmataceae </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td> </td><td>5.74 </td><td>1.42 </td><td>3.9 </td><td>0.24 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298600" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB4" class="table-wrap-title" data-id="DSS027TB4"><div class="label -label">Table 4.</div><div class="caption"><p>The mean Z scores and SD at the level of family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Family </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacteriaceae </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydiaceae </td><td>1.44 </td><td>0.19 </td><td>4 </td><td>0.13 </td></tr><tr><td>Bacillaceae </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillaceae </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcaceae </td><td>4.92 </td><td>1.08 </td><td>5 </td><td>0.22 </td></tr><tr><td>Brucellaceae </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Burkholderiaceae </td><td>8.78 </td><td>1.76 </td><td>3 </td><td>0.20 </td></tr><tr><td>Neisseriaceae </td><td>10.72 </td><td>3.85 </td><td>3 </td><td>0.36 </td></tr><tr><td>Enterobacteriaceae </td><td>7.43 </td><td>4.29 </td><td>7 </td><td>0.58 </td></tr><tr><td>Pasteurellaceae </td><td>4.32 </td><td>0.68 </td><td>3 </td><td>0.16 </td></tr><tr><td>Vibrionaceae </td><td>8.02 </td><td>0.66 </td><td>5 </td><td>0.08 </td></tr><tr><td>Xanthomonadaceae </td><td>9.75 </td><td>1.56 </td><td>4 </td><td>0.16 </td></tr><tr><td>Spirochaetaceae </td><td>2.98 </td><td>0.37 </td><td>3 </td><td>0.12 </td></tr><tr><td>Mycoplasmataceae </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td> </td><td>5.74 </td><td>1.42 </td><td>3.9 </td><td>0.24 </td></tr></tbody></table></div><div content-id="DSS027TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Family </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacteriaceae </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydiaceae </td><td>1.44 </td><td>0.19 </td><td>4 </td><td>0.13 </td></tr><tr><td>Bacillaceae </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillaceae </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcaceae </td><td>4.92 </td><td>1.08 </td><td>5 </td><td>0.22 </td></tr><tr><td>Brucellaceae </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Burkholderiaceae </td><td>8.78 </td><td>1.76 </td><td>3 </td><td>0.20 </td></tr><tr><td>Neisseriaceae </td><td>10.72 </td><td>3.85 </td><td>3 </td><td>0.36 </td></tr><tr><td>Enterobacteriaceae </td><td>7.43 </td><td>4.29 </td><td>7 </td><td>0.58 </td></tr><tr><td>Pasteurellaceae </td><td>4.32 </td><td>0.68 </td><td>3 </td><td>0.16 </td></tr><tr><td>Vibrionaceae </td><td>8.02 </td><td>0.66 </td><td>5 </td><td>0.08 </td></tr><tr><td>Xanthomonadaceae </td><td>9.75 </td><td>1.56 </td><td>4 </td><td>0.16 </td></tr><tr><td>Spirochaetaceae </td><td>2.98 </td><td>0.37 </td><td>3 </td><td>0.12 </td></tr><tr><td>Mycoplasmataceae </td><td>1.99 </td><td>0.81 </td><td>5 </td><td>0.41 </td></tr><tr><td> </td><td>5.74 </td><td>1.42 </td><td>3.9 </td><td>0.24 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298600" target="_blank">View Large</a></div></div>                        <a id="7298601" scrollto-destination="7298601"></a>
<div content-id="DSS027TB5" class="table-modal"><div class="table-wrap"><div id="DSS027TB5" class="table-wrap-title" data-id="DSS027TB5"><div class="label -label">Table 5.</div><div class="caption"><p>The mean Z scores and SD at the level of genus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacterium </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydophila </td><td>1.36 </td><td>0.11 </td><td>3 </td><td>0.08 </td></tr><tr><td>Bacillus </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillus </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcus </td><td>5.03 </td><td>1.22 </td><td>4 </td><td>0.24 </td></tr><tr><td>Brucella </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Vibrio </td><td>8.08 </td><td>0.74 </td><td>4 </td><td>0.09 </td></tr><tr><td>Xanthomonas </td><td>10.51 </td><td>0.43 </td><td>3 </td><td>0.04 </td></tr><tr><td>Mycoplasma </td><td>2.07 </td><td>0.91 </td><td>4 </td><td>0.44 </td></tr><tr><td> </td><td>5.23 </td><td>0.90 </td><td>3.4 </td><td>0.21 </td></tr></tbody></table></div><div content-id="DSS027TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacterium </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydophila </td><td>1.36 </td><td>0.11 </td><td>3 </td><td>0.08 </td></tr><tr><td>Bacillus </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillus </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcus </td><td>5.03 </td><td>1.22 </td><td>4 </td><td>0.24 </td></tr><tr><td>Brucella </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Vibrio </td><td>8.08 </td><td>0.74 </td><td>4 </td><td>0.09 </td></tr><tr><td>Xanthomonas </td><td>10.51 </td><td>0.43 </td><td>3 </td><td>0.04 </td></tr><tr><td>Mycoplasma </td><td>2.07 </td><td>0.91 </td><td>4 </td><td>0.44 </td></tr><tr><td> </td><td>5.23 </td><td>0.90 </td><td>3.4 </td><td>0.21 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298601" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB5" class="table-wrap-title" data-id="DSS027TB5"><div class="label -label">Table 5.</div><div class="caption"><p>The mean Z scores and SD at the level of genus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Genus </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacterium </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydophila </td><td>1.36 </td><td>0.11 </td><td>3 </td><td>0.08 </td></tr><tr><td>Bacillus </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillus </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcus </td><td>5.03 </td><td>1.22 </td><td>4 </td><td>0.24 </td></tr><tr><td>Brucella </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Vibrio </td><td>8.08 </td><td>0.74 </td><td>4 </td><td>0.09 </td></tr><tr><td>Xanthomonas </td><td>10.51 </td><td>0.43 </td><td>3 </td><td>0.04 </td></tr><tr><td>Mycoplasma </td><td>2.07 </td><td>0.91 </td><td>4 </td><td>0.44 </td></tr><tr><td> </td><td>5.23 </td><td>0.90 </td><td>3.4 </td><td>0.21 </td></tr></tbody></table></div><div content-id="DSS027TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Genus </th><th>Mean </th><th>SD </th><th>Genome no. </th><th>VC (SD/Mean) </th></tr></thead><tbody><tr><td>Mycobacterium </td><td>5.01 </td><td>1.08 </td><td>3 </td><td>0.22 </td></tr><tr><td>Chlamydophila </td><td>1.36 </td><td>0.11 </td><td>3 </td><td>0.08 </td></tr><tr><td>Bacillus </td><td>4.84 </td><td>2.29 </td><td>4 </td><td>0.47 </td></tr><tr><td>Lactobacillus </td><td>4.84 </td><td>1.14 </td><td>3 </td><td>0.24 </td></tr><tr><td>Streptococcus </td><td>5.03 </td><td>1.22 </td><td>4 </td><td>0.24 </td></tr><tr><td>Brucella </td><td>5.33 </td><td>0.14 </td><td>3 </td><td>0.03 </td></tr><tr><td>Vibrio </td><td>8.08 </td><td>0.74 </td><td>4 </td><td>0.09 </td></tr><tr><td>Xanthomonas </td><td>10.51 </td><td>0.43 </td><td>3 </td><td>0.04 </td></tr><tr><td>Mycoplasma </td><td>2.07 </td><td>0.91 </td><td>4 </td><td>0.44 </td></tr><tr><td> </td><td>5.23 </td><td>0.90 </td><td>3.4 </td><td>0.21 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298601" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7298602 id="7298602" class="section-title" data-legacy-id=s3d>3.4. Comparative analyses of codon reuse strengths among different groups</h3>
<p>The 136 prokaryotic genomes could be classified into two groups based on the Gram type, oxygen metabolism, growth rate, G + C content, genome size, and tRNA gene number, respectively. We performed comparative analyses of the general Z scores between any two groups based on the six classifying criteria and results are listed in Table <a reveal-id="DSS027TB6" data-open="DSS027TB6" class="link link-reveal link-table xref-fig">6</a>. For all criteria except Gram type and oxygen metabolism, the two groups are divided equally based on median criterion values. Z scores of the two groups are generally significantly different based on all classifying criteria, except Gram type. Among the five indices with significant differences, genome size and tRNA gene number are the most sensitive as the <em>P</em>-value is the smallest. Based on the descending order of statistical difference, the other three indices will be G + C content, oxygen metabolism, and growth rate. It is interesting that the two most frequently used classifying criteria, oxygen metabolism and Gram type, are not associated with variance of synonymous codon pair strength as are genomic features such as G + C content, genome size, and tRNA gene number. </p>                        <a id="7298604" scrollto-destination="7298604"></a>
<div content-id="DSS027TB6" class="table-modal"><div class="table-wrap"><div id="DSS027TB6" class="table-wrap-title" data-id="DSS027TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Comparison of the general Z scores between any two groups based on six classification criteria<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Classifying criteria </th><th>Mean </th><th>SD </th><th>Genome number </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Gram type </td></tr><tr><td> Gram negative </td><td>6.01 </td><td>3.43 </td><td>70 </td><td rowspan="2">0.064 </td></tr><tr><td> Gram positive </td><td>4.87 </td><td>2.64 </td><td>35 </td></tr><tr><td colspan="5">Growth rate<sup>b</sup> </td></tr><tr><td> Fast </td><td>6.28 </td><td>3.05 </td><td>54 </td><td rowspan="2">0.028 </td></tr><tr><td> Slow </td><td>4.92 </td><td>3.27 </td><td>53 </td></tr><tr><td colspan="5">Oxygen metabolism<sup>c</sup> </td></tr><tr><td> Aerobic </td><td>6.39 </td><td>3.05 </td><td>35 </td><td rowspan="2">0.017 </td></tr><tr><td> Anaerobic </td><td>4.58 </td><td>2.04 </td><td>16 </td></tr><tr><td colspan="5">G + C content </td></tr><tr><td> Low GC (&lt;46.2%) </td><td>3.57 </td><td>2.19 </td><td>68 </td><td rowspan="2">1.21e-09 </td></tr><tr><td> High GC (&gt;46.2%) </td><td>6.62 </td><td>3.11 </td><td>68 </td></tr><tr><td colspan="5">tRNA gene number </td></tr><tr><td> Less tRNA (&lt;32) </td><td>3.30 </td><td>2.15 </td><td>68 </td><td rowspan="2">1.37e-13 </td></tr><tr><td> More tRNA (&gt;32) </td><td>6.89 </td><td>2.84 </td><td>68 </td></tr><tr><td colspan="5">Genome size </td></tr><tr><td> Small size (&lt;2.55 Mb) </td><td>3.25 </td><td>1.97 </td><td>68 </td><td rowspan="2">3.08e-14 </td></tr><tr><td> Large size (&gt;2.55 Mb) </td><td>6.93 </td><td>2.90 </td><td>68 </td></tr></tbody></table></div><div content-id="DSS027TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Classifying criteria </th><th>Mean </th><th>SD </th><th>Genome number </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Gram type </td></tr><tr><td> Gram negative </td><td>6.01 </td><td>3.43 </td><td>70 </td><td rowspan="2">0.064 </td></tr><tr><td> Gram positive </td><td>4.87 </td><td>2.64 </td><td>35 </td></tr><tr><td colspan="5">Growth rate<sup>b</sup> </td></tr><tr><td> Fast </td><td>6.28 </td><td>3.05 </td><td>54 </td><td rowspan="2">0.028 </td></tr><tr><td> Slow </td><td>4.92 </td><td>3.27 </td><td>53 </td></tr><tr><td colspan="5">Oxygen metabolism<sup>c</sup> </td></tr><tr><td> Aerobic </td><td>6.39 </td><td>3.05 </td><td>35 </td><td rowspan="2">0.017 </td></tr><tr><td> Anaerobic </td><td>4.58 </td><td>2.04 </td><td>16 </td></tr><tr><td colspan="5">G + C content </td></tr><tr><td> Low GC (&lt;46.2%) </td><td>3.57 </td><td>2.19 </td><td>68 </td><td rowspan="2">1.21e-09 </td></tr><tr><td> High GC (&gt;46.2%) </td><td>6.62 </td><td>3.11 </td><td>68 </td></tr><tr><td colspan="5">tRNA gene number </td></tr><tr><td> Less tRNA (&lt;32) </td><td>3.30 </td><td>2.15 </td><td>68 </td><td rowspan="2">1.37e-13 </td></tr><tr><td> More tRNA (&gt;32) </td><td>6.89 </td><td>2.84 </td><td>68 </td></tr><tr><td colspan="5">Genome size </td></tr><tr><td> Small size (&lt;2.55 Mb) </td><td>3.25 </td><td>1.97 </td><td>68 </td><td rowspan="2">3.08e-14 </td></tr><tr><td> Large size (&gt;2.55 Mb) </td><td>6.93 </td><td>2.90 </td><td>68 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Because information of the upper three factors is not available for some of the genomes, the total genome number is less than 136 for these factors. Detailed information of each prokaryotic genomes is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Original growth rate data were obtained from Vieira-Silva and Rocha<sup><a reveal-id="DSS027C40" data-open="DSS027C40" class="link link-ref link-reveal xref-bibr">40</a></sup>. Genomes with generation time longer than 2 h are taken as slow growing, otherwise as fast growing.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Original data on oxygen metabolism were obtained from NCBI at <a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/Bacteria/lproks_0.txt">ftp://ftp.ncbi.nlm.nih.gov/genomes/Bacteria/lproks_0.txt</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298604" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS027TB6" class="table-wrap-title" data-id="DSS027TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Comparison of the general Z scores between any two groups based on six classification criteria<sup>a</sup></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Classifying criteria </th><th>Mean </th><th>SD </th><th>Genome number </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Gram type </td></tr><tr><td> Gram negative </td><td>6.01 </td><td>3.43 </td><td>70 </td><td rowspan="2">0.064 </td></tr><tr><td> Gram positive </td><td>4.87 </td><td>2.64 </td><td>35 </td></tr><tr><td colspan="5">Growth rate<sup>b</sup> </td></tr><tr><td> Fast </td><td>6.28 </td><td>3.05 </td><td>54 </td><td rowspan="2">0.028 </td></tr><tr><td> Slow </td><td>4.92 </td><td>3.27 </td><td>53 </td></tr><tr><td colspan="5">Oxygen metabolism<sup>c</sup> </td></tr><tr><td> Aerobic </td><td>6.39 </td><td>3.05 </td><td>35 </td><td rowspan="2">0.017 </td></tr><tr><td> Anaerobic </td><td>4.58 </td><td>2.04 </td><td>16 </td></tr><tr><td colspan="5">G + C content </td></tr><tr><td> Low GC (&lt;46.2%) </td><td>3.57 </td><td>2.19 </td><td>68 </td><td rowspan="2">1.21e-09 </td></tr><tr><td> High GC (&gt;46.2%) </td><td>6.62 </td><td>3.11 </td><td>68 </td></tr><tr><td colspan="5">tRNA gene number </td></tr><tr><td> Less tRNA (&lt;32) </td><td>3.30 </td><td>2.15 </td><td>68 </td><td rowspan="2">1.37e-13 </td></tr><tr><td> More tRNA (&gt;32) </td><td>6.89 </td><td>2.84 </td><td>68 </td></tr><tr><td colspan="5">Genome size </td></tr><tr><td> Small size (&lt;2.55 Mb) </td><td>3.25 </td><td>1.97 </td><td>68 </td><td rowspan="2">3.08e-14 </td></tr><tr><td> Large size (&gt;2.55 Mb) </td><td>6.93 </td><td>2.90 </td><td>68 </td></tr></tbody></table></div><div content-id="DSS027TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Classifying criteria </th><th>Mean </th><th>SD </th><th>Genome number </th><th><em>P</em>-value </th></tr></thead><tbody><tr><td colspan="5">Gram type </td></tr><tr><td> Gram negative </td><td>6.01 </td><td>3.43 </td><td>70 </td><td rowspan="2">0.064 </td></tr><tr><td> Gram positive </td><td>4.87 </td><td>2.64 </td><td>35 </td></tr><tr><td colspan="5">Growth rate<sup>b</sup> </td></tr><tr><td> Fast </td><td>6.28 </td><td>3.05 </td><td>54 </td><td rowspan="2">0.028 </td></tr><tr><td> Slow </td><td>4.92 </td><td>3.27 </td><td>53 </td></tr><tr><td colspan="5">Oxygen metabolism<sup>c</sup> </td></tr><tr><td> Aerobic </td><td>6.39 </td><td>3.05 </td><td>35 </td><td rowspan="2">0.017 </td></tr><tr><td> Anaerobic </td><td>4.58 </td><td>2.04 </td><td>16 </td></tr><tr><td colspan="5">G + C content </td></tr><tr><td> Low GC (&lt;46.2%) </td><td>3.57 </td><td>2.19 </td><td>68 </td><td rowspan="2">1.21e-09 </td></tr><tr><td> High GC (&gt;46.2%) </td><td>6.62 </td><td>3.11 </td><td>68 </td></tr><tr><td colspan="5">tRNA gene number </td></tr><tr><td> Less tRNA (&lt;32) </td><td>3.30 </td><td>2.15 </td><td>68 </td><td rowspan="2">1.37e-13 </td></tr><tr><td> More tRNA (&gt;32) </td><td>6.89 </td><td>2.84 </td><td>68 </td></tr><tr><td colspan="5">Genome size </td></tr><tr><td> Small size (&lt;2.55 Mb) </td><td>3.25 </td><td>1.97 </td><td>68 </td><td rowspan="2">3.08e-14 </td></tr><tr><td> Large size (&gt;2.55 Mb) </td><td>6.93 </td><td>2.90 </td><td>68 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Because information of the upper three factors is not available for some of the genomes, the total genome number is less than 136 for these factors. Detailed information of each prokaryotic genomes is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Original growth rate data were obtained from Vieira-Silva and Rocha<sup><a reveal-id="DSS027C40" data-open="DSS027C40" class="link link-ref link-reveal xref-bibr">40</a></sup>. Genomes with generation time longer than 2 h are taken as slow growing, otherwise as fast growing.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Original data on oxygen metabolism were obtained from NCBI at <a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/Bacteria/lproks_0.txt">ftp://ftp.ncbi.nlm.nih.gov/genomes/Bacteria/lproks_0.txt</a>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7298604" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7298605 id="7298605" class="section-title" data-legacy-id=s3e>3.5. Linear correlation between the general Z scores and genomic characteristics</h3>
<div class="block-child-p">The strength of synonymous codon correlation varied largely across prokaryotic genomes, as shown above. Furthermore, we have identified potential determinant factors of this observation. It would be interesting to identify a quantitative relationship between Z scores and genomic factors. Three factors that could be directly extracted from chromosomal DNA sequences are chromosome size, G + C content, and tRNA gene number. After obtaining values of each factor for all 136 prokaryotic genomes, linear fitting was performed between them and the general Z scores. The least squares method<sup><a reveal-id="DSS027C25" data-open="DSS027C25" class="link link-ref link-reveal xref-bibr">25</a></sup> was used for linear fitting between them and the general Z scores. Their correlation coefficient (<em>R</em>) and significance (<em>P</em>-value) were computed, respectively, with R package. The scatter plot of general Z scores against the three factors and linear fitting are shown in Fig. <a reveal-id="DSS027F3" data-open="DSS027F3" class="link link-reveal link-table xref-fig">3</a>. All three factors are significantly correlated (0.55 &lt; = <em>R</em> &lt; = 0.78, <em>P</em> &lt; 10<sup>−11</sup>) with the general Z scores. Genome size has the strongest correlation with general Z score, according to both <em>R</em> coefficient and <em>P</em>-value. This fact is consistent with the above comparative analysis, where genome size is shown to be the most effective distinguishing feature. In general, a phenomenon is often associated with multiple factors. The multiple linear regression method is more actual and effective than linear fitting method. To obtain a stronger correlation, multiple linear regression was also performed. Chromosome size, G + C content, and tRNA gene number constitute the three explanatory variables, with general Z score as the dependent variable. Using R package, the regression equation is defined as Equation (2). <a id="DSS027M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSS027M2"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss027eq2.gif?Expires=2147483647&Signature=ASCF-nqq9~LQSivQZ1KDuYPSTG5rPbukCqHj8DqDin1KBxmwZlqwQCr0vCt3jUAXJK-6yJ690QzVqpVGZ~Q4HfZNJSkw5W8ok-3XMBTZerYLDVZs52E9JWVCqHB1-wC8JwdA7UtBLfyoehLQBdNCiP2oEPC0sIVM9UE8k3jJWo5FKO-smNtNwo7g9767C-etKZ3csZ-9Jsggp91BSWQCnDijK8BFCV9R8PFisDMslIdLrpdBIfr697FcP79MHi6CRkmBfaK~2ZnLq2C~SOMJ-pnzgGBUSZ7kNsUQX6F-tG15G2aJMMtolekaUTlvcYSgfj5321cGnKadpf7NB3cYvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dss027eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss027eq2.gif?Expires=2147483647&Signature=coxEL7eDuHllXR6Edb9yNYKP-6IYKKKOFp7o1EwiGyZYCVLotkaTvjDl2N62lK5G5iM1Mkb3Tt~~Ou7SQtq7yNSYKbFeiGikJcUDeGGtJuOEuTQGFex2IGsyLGL5ccNEZeAdTgz5T5hnkV0RFo7ICAjlBzrVk7qQN47i0uY9x~EJls55iCZL0k8A0CIC6BQdP7ENNsld6WJP9apVM46iYAUtE4sKfAjAPuYJPj6yHcUsUbOHY7dWwJ05fpDFF424pbO85ei147qkcCM7Uh-z7QzQD5UTp6uipDrJrcJ0m6QhYj1-9grx~4TvTJgJj5eVDW8Xr5tLaIimPx~st29EEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dss027eq2.gif" /></a></div></div><div class="label">(2)</div></div> where Z denotes the general Z score, S denotes chromosome size, G denotes G + C content, T denotes tRNA gene number, and chromosome size is measured in millions of base pairs (Mb). The Pearson's coefficient (<em>r</em> value) of the multiple regression is 0.8334 and the <em>P</em> = 2.2 × 10<sup>−16</sup>. Using Equation (2), we seek to predict the strength of successive synonymous codon pairs in any sequenced prokaryotic genome with some reliability. The prediction error is only 1.72, and roughly speaking, the prediction accuracy is higher than 80%, given that the Pearson's coefficient is 0.8334. </div>                        <a id="7298607" scrollto-destination="7298607"></a>
<div id="DSS027F3" data-id="DSS027F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02703.jpeg?Expires=2147483647&Signature=4Vn0U6gziEKu3ZbTF2Aq4BSbWTml0umdyAONW1zKF~l6b7c7wz5GLuuFXr8-2-GdjsHqDGyA~XAmURTlmg~tE4bD2mvZdI1EhY4w1qWkQwoPd7meBFkLlHgivSr6BmiXfFy2bz0nlFjVp4xRs5NO4FqPdbR81moyl~XRm7ab0r~2xki3Cabtz3rIXY~suE5x90xAg3rPIp-YjMmNXaixFBhRlT7sdTefZ6mMnMSqjSB7RUr1OdB-7lI3jT33KLRF8KLEEtIwRZZivFhOUntuPP2osoWMqRcp882wVhGwh2r4vK0eT2FnrwI0L1lh~acNIC02TWxXID1URD2TzIFuXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of general Z scores against three factors (genome size, G + C content, and tRNA gene number) for 136 prokaryotic genomes. In the figure, each point corresponds to a prokaryotic genomes. (A) Scatter plot of general Z scores against genome size: linear fitting by least squares method. (B) Scatter plot of general Z scores against G + C content: linear fitting by least squares method. (C) Scatter plot of general Z scores against tRNA gene number: linear fitting by least squares method." path-from-xml="dss02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298607/dss02703.jpeg" path-from-xml="dss02703.jpeg" target="_blank">View large</a><a section="7298607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02703.jpeg?Expires=2147483647&Signature=2-5rjI87KiHeqXMlocCGqs7zOXaAFDVK8w0Ns1oj2tq7euPXxXexOve3jaEghGets5Pc-Z0Lh6fgmcs9NNQ9sl0Zf2kkEQLqjszQege8GXfmSUCAOSX4byHi~4ushiZ8iIGbSQ~bxSPhC7J3FkQ2VkQuRiKs6~e9eTSAQ7vxvG6r5yfIHPMDbXlhbUwsNAS7IueJBiFHwPw5RQAI0Ocy71F203gxYx3nx3H-FDY86JqLxs4UzykEGxMrXbqMTFSPPq~3Otu9-N8rbWt1CmercbAn0N4vCwI85DfOhiIRHAnO1mR~XlcjAqatMXJv~hsYWxo~PZieSgdNuqQc3-Qeqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298607&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of general Z scores against three factors (genome size, G + C content, and tRNA gene number) for 136 prokaryotic genomes. In the figure, each point corresponds to a prokaryotic genomes. (A) Scatter plot of general Z scores against genome size: linear fitting by least squares method. (B) Scatter plot of general Z scores against G + C content: linear fitting by least squares method. (C) Scatter plot of general Z scores against tRNA gene number: linear fitting by least squares method.</p></div></div><div content-id="DSS027F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/m_dss02703.jpeg?Expires=2147483647&Signature=4Vn0U6gziEKu3ZbTF2Aq4BSbWTml0umdyAONW1zKF~l6b7c7wz5GLuuFXr8-2-GdjsHqDGyA~XAmURTlmg~tE4bD2mvZdI1EhY4w1qWkQwoPd7meBFkLlHgivSr6BmiXfFy2bz0nlFjVp4xRs5NO4FqPdbR81moyl~XRm7ab0r~2xki3Cabtz3rIXY~suE5x90xAg3rPIp-YjMmNXaixFBhRlT7sdTefZ6mMnMSqjSB7RUr1OdB-7lI3jT33KLRF8KLEEtIwRZZivFhOUntuPP2osoWMqRcp882wVhGwh2r4vK0eT2FnrwI0L1lh~acNIC02TWxXID1URD2TzIFuXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot of general Z scores against three factors (genome size, G + C content, and tRNA gene number) for 136 prokaryotic genomes. In the figure, each point corresponds to a prokaryotic genomes. (A) Scatter plot of general Z scores against genome size: linear fitting by least squares method. (B) Scatter plot of general Z scores against G + C content: linear fitting by least squares method. (C) Scatter plot of general Z scores against tRNA gene number: linear fitting by least squares method." path-from-xml="dss02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7298607/dss02703.jpeg" path-from-xml="dss02703.jpeg" target="_blank">View large</a><a section="7298607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss02703.jpeg?Expires=2147483647&Signature=2-5rjI87KiHeqXMlocCGqs7zOXaAFDVK8w0Ns1oj2tq7euPXxXexOve3jaEghGets5Pc-Z0Lh6fgmcs9NNQ9sl0Zf2kkEQLqjszQege8GXfmSUCAOSX4byHi~4ushiZ8iIGbSQ~bxSPhC7J3FkQ2VkQuRiKs6~e9eTSAQ7vxvG6r5yfIHPMDbXlhbUwsNAS7IueJBiFHwPw5RQAI0Ocy71F203gxYx3nx3H-FDY86JqLxs4UzykEGxMrXbqMTFSPPq~3Otu9-N8rbWt1CmercbAn0N4vCwI85DfOhiIRHAnO1mR~XlcjAqatMXJv~hsYWxo~PZieSgdNuqQc3-Qeqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7298607&ar=481278&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot of general Z scores against three factors (genome size, G + C content, and tRNA gene number) for 136 prokaryotic genomes. In the figure, each point corresponds to a prokaryotic genomes. (A) Scatter plot of general Z scores against genome size: linear fitting by least squares method. (B) Scatter plot of general Z scores against G + C content: linear fitting by least squares method. (C) Scatter plot of general Z scores against tRNA gene number: linear fitting by least squares method.</p></div></div>                            <h2 scrollto-destination=7298608 id="7298608" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Cannarozzi <em>et al</em>. first observed the phenomenon of successive synonymous codon pairs in eukaryotic genomes in 2010.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Fredrick and Ibba comment that this work is one of the best examples illustrating how codon usage patterns control ribosome speed and fine-tune translation to increase protein synthesis efficiency.<sup><a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a></sup> They call on immediate work on bacterial genomes in which translation rates are substantially higher than eukaryotes.<sup><a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Interestingly, we found that most prokaryotic genomes have significant biases in successive synonymous codon pairs, suggesting that this pattern is universal to the three domains of life. Significantly, different distributions of synonymous codon pairs between protein-coding genes and intergenic sequences suggest that this bias is a result from translation selection. Combined with the previous work,<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> we conclude that the bias of successive synonymous codon pairs, as universal pattern in all living organisms, would be a translation-associated effect and could be used to fine-tune protein synthesis.</p><p>Furthermore, the strength of the bias varies strongly across different genomes. Eukaryotic cells have the strongest bias, whereas archaeal cells have the least. Among bacteria, there is also a range of differences. Usually, large bacterial genomes and those with high G + C content tend to have a stronger bias. This type of variation reflects the diversity of living species. A better understanding of the precise reason for varied strength may be clarified by comparing these results with single codon bias. Generally, highly expressed genes tend to have more of a bias with single codons<sup><a reveal-id="DSS027C8" data-open="DSS027C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS027C11" data-open="DSS027C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS027C28  DSS027C35" data-open="DSS027C28  DSS027C35" class="link link-ref link-reveal xref-bibr">28–35</a></sup> in unicellular organisms. Codon bias is thought to maximize translation efficiency, including speed and/or fidelity.<sup><a reveal-id="DSS027C9" data-open="DSS027C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS027C11" data-open="DSS027C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS027C12" data-open="DSS027C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSS027C28" data-open="DSS027C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS027C30  DSS027C35" data-open="DSS027C30  DSS027C35" class="link link-ref link-reveal xref-bibr">30–35</a></sup> The strength of the codon bias is determined by the strength of translation selection exerted on the genome.<sup><a reveal-id="DSS027C13" data-open="DSS027C13" class="link link-ref link-reveal xref-bibr">13</a></sup> For example, species exposed to selection for rapid growth tend to have more strongly selected codon usage bias.<sup><a reveal-id="DSS027C36" data-open="DSS027C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS027C37" data-open="DSS027C37" class="link link-ref link-reveal xref-bibr">37</a></sup> At the same time, fast-growing bacteria with low generation time generally have more tRNA genes to increase translation speed.<sup><a reveal-id="DSS027C13" data-open="DSS027C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS027C38" data-open="DSS027C38" class="link link-ref link-reveal xref-bibr">38</a></sup> And, tRNA gene number is positively correlated with genome size and G + C content.<sup><a reveal-id="DSS027C39" data-open="DSS027C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Because the general Z scores of bacteria are also associated with these factors, the correlation between the bias strength of successive codon pairs and various genomic or biochemical characters in prokaryotes may be caused by the translation selection as the single codon bias. One lingering question is why single codon bias is almost absent, or at least smaller, in higher eukaryotic genomes.<sup><a reveal-id="DSS027C37" data-open="DSS027C37" class="link link-ref link-reveal xref-bibr">37</a></sup> However, the strength of successive codon pair bias is much stronger in higher eukaryotes than in prokaryotes. Our current thought is that this may be due to different translation mechanisms among the three domains of life.</p><p>Another noticeable point is how translation influences synonymous codon pair usage or why codon reuse favours translation efficiency. Cannarozzi <em>et al</em>. have proposed that tRNA diffusion is slower than both reloading and translation.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> When the next amino acid that is incorporated is the same, a recently used tRNA would be more likely than any of its isoacceptors to still be in the vicinity of the ribosome.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Therefore, reuse of isoaccepting codons would accelerate the translation process.<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Direct observation of slower tRNA diffusion than reloading and translation would be the most potent proof for this hypothesis.</p><p>Successive synonymous codon pair represents the order of protein-coding sequences rather than their composition, which is different from single codon use.<sup><a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS027C4" data-open="DSS027C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Therefore, the observed bias of successive synonymous codon pairs, as a newly observed phenomenon, illustrates that regulatory information of protein translation is retained not only in nucleotide species but also in nucleotide order. Widely observed association of single codon,<sup><a reveal-id="DSS027C3" data-open="DSS027C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS027C5" data-open="DSS027C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS027C6" data-open="DSS027C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS027C8" data-open="DSS027C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS027C9" data-open="DSS027C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS027C11  DSS027C13" data-open="DSS027C11  DSS027C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> codon pair<sup><a reveal-id="DSS027C14  DSS027C22" data-open="DSS027C14  DSS027C22" class="link link-ref link-reveal xref-bibr">14–22</a></sup> or successive synonymous codon pair<sup><a reveal-id="DSS027C1" data-open="DSS027C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS027C2" data-open="DSS027C2" class="link link-ref link-reveal xref-bibr">2</a></sup> use and translation efficiency suggests that the latter exerts influence on the various types of codons. On the other hand, genome size, G + C content and tRNA gene number are all significantly associated with the bias strength of synonymous codon pairs, illustrating that translation selection exerts influence on different genomic features. Taking all factors into consideration, we conclude that translation selection is exerted on genome sequences at multiple levels and by various mechanisms. In other words, protein translation is a complex process and is associated with various factors such as usage of single codons, codon pairs (in particular successive synonymous codon pair), genome size, genomic G + C content, and tRNA gene number.</p>                            <h2 scrollto-destination=7298613 id="7298613" class="section-title" data-legacy-id=s5>Conflict of Interest statement</h2>
<p>The authors have no conflicts of interest to declare.</p>                            <h2 scrollto-destination=7298615 id="7298615" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss027/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=7298617 id="7298617" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported by the National Natural Science Foundation of China (grant 31071109), the special fund of China Postdoctoral Science Foundation (grant 201104687), and the program for New Century Excellent Talents in University (grant number NCET-11-0059). Funding for open access charge: National Natural Science Foundation of China.</p>                            <h2 scrollto-destination=7298619 id="7298619" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to the reviewers for their valuable comments that have led to improvements to this paper. We thank Miss Meng-Ze Du for useful discussions.</p>                            <h2 scrollto-destination=7298621 id="7298621" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS027C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cannarozzi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Schraudolph</div> <div class="given-names">N.N.</div></div>,  <div class="name"><div class="surname">Faty</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A role for codon order in translation dynamics</div>, <div class="source">Cell</div> , <div class="year">2010</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">355</div>-<div class="lpage">67</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2010.02.036" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2010.02.036</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20codon%20order%20in%20translation%20dynamics&amp;author=G.%20Cannarozzi&amp;author=N.N.%20Schraudolph&amp;author=M.%20Faty&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.02.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.02.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.02.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fredrick</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div>. </div><div class="article-title">How the sequence of a gene can tune its translation</div>, <div class="source">Cell</div> , <div class="year">2010</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">227</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2010.03.033" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2010.03.033</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20the%20sequence%20of%20a%20gene%20can%20tune%20its%20translation&amp;author=K.%20Fredrick&amp;author=M.%20Ibba&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=227-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Tollervey</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Coding-sequence determinants of gene expression in Escherichia coli</div>, <div class="source">Science</div> , <div class="year">2009</div>, vol. <div class="volume">324</div> (pg. <div class="fpage">255</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1170160" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1170160</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding-sequence%20determinants%20of%20gene%20expression%20in%20Escherichia%20coli&amp;author=G.%20Kudla&amp;author=A.W.%20Murray&amp;author=D.%20Tollervey&amp;author=J.B.%20Plotkin&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=255-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19359587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Kudla</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Synonymous but not the same: the causes and consequences of codon bias</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">32</div>-<div class="lpage">42</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2899" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg2899</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias&amp;author=J.B.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>, <div class="source">Cell</div> , <div class="year">2010</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">344</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2010.03.031" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2010.03.031</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Waldman</div> <div class="given-names">Y.Y.</div></div>,  <div class="name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ruppin</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Translation efficiency is determined by both codon bias and folding energy</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">3645</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0909910107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0909910107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=3645-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div> </div></div></div></div></div><div content-id="DSS027C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">General rules for optimal codon choice</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000556</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1000556" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pgen.1000556</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=General%20rules%20for%20optimal%20codon%20choice&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000556" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19593368" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jacobzone</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Codon catalog usage is a genome strategy modulated for gene expressivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1981</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">r43</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20is%20a%20genome%20strategy%20modulated%20for%20gene%20expressivity&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=M.%20Jacobzone&amp;author=R.%20Mercier&amp;publication_year=1981&amp;journal=Nucleic%20Acids%20Res.&amp;volume=9&amp;pages=r43-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/9.1.213-b" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F9.1.213-b" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F9.1.213-b"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7208352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">389</div>-<div class="lpage">409</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0022-2836(81)90003-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0022-2836(81)90003-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Selection on codon bias</div>, <div class="source">Ann. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">287</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.genet.42.110807.091442" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.genet.42.110807.091442</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20codon%20bias&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2008&amp;journal=Ann.%20Rev.%20Genet.&amp;volume=42&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091442"> </span></div> </div></div></div></div></div><div content-id="DSS027C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">An evolutionary perspective on synonymous codon usage in unicellular organisms</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1986</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">28</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02099948" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF02099948</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionary%20perspective%20on%20synonymous%20codon%20usage%20in%20unicellular%20organisms&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1986&amp;journal=J.%20Mol.%20Evol.&amp;volume=24&amp;pages=28-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02099948" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02099948" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02099948"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3104616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Bailes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Grocock</div> <div class="given-names">R.J.</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J.F.</div></div>,  <div class="name"><div class="surname">Sockett</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">Variation in the strength of selected codon usage bias among bacteria</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1141</div>-<div class="lpage">53</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gki242" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gki242</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20strength%20of%20selected%20codon%20usage%20bias%20among%20bacteria&amp;author=P.M.%20Sharp&amp;author=E.%20Bailes&amp;author=R.J.%20Grocock&amp;author=J.F.%20Peden&amp;author=R.E.%20Sockett&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=1141-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buchan</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Aucott</div> <div class="given-names">L.S.</div></div>,  <div class="name"><div class="surname">Stansfield</div> <div class="given-names">I.</div></div>. </div><div class="article-title">tRNA properties help shape codon pair preferences in open reading frames</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1015</div>-<div class="lpage">27</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkj488" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkj488</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA%20properties%20help%20shape%20codon%20pair%20preferences%20in%20open%20reading%20frames&amp;author=J.R.%20Buchan&amp;author=L.S.%20Aucott&amp;author=I.%20Stansfield&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=1015-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj488" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj488" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj488"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16473853" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fedorov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Saxonov</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gilbert</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Regularities of context-dependent codon bias in eukaryotic genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">1192</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/30.5.1192" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/30.5.1192</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regularities%20of%20context-dependent%20codon%20bias%20in%20eukaryotic%20genes&amp;author=A.%20Fedorov&amp;author=S.%20Saxonov&amp;author=W.%20Gilbert&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=1192-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.5.1192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.5.1192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.5.1192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11861911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutman</div> <div class="given-names">G.A.</div></div>,  <div class="name"><div class="surname">Hatfield</div> <div class="given-names">G.W.</div></div>. </div><div class="article-title">Nonrandom utilization of codon pairs in <em>Escherichia coli</em></div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1989</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">3699</div>-<div class="lpage">703</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.86.10.3699" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.86.10.3699</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonrandom%20utilization%20of%20codon%20pairs%20in%20Escherichia%20coli&amp;author=G.A.%20Gutman&amp;author=G.W.%20Hatfield&amp;publication_year=1989&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=3699-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.10.3699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.10.3699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.10.3699"> </span></div> </div></div></div></div></div><div content-id="DSS027C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irwin</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Heck</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Hatfield</div> <div class="given-names">G.W.</div></div>. </div><div class="article-title">Codon pair utilization biases influence translational elongation step times</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">22801</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.270.39.22801" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.270.39.22801</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20pair%20utilization%20biases%20influence%20translational%20elongation%20step%20times&amp;author=B.%20Irwin&amp;author=J.D.%20Heck&amp;author=G.W.%20Hatfield&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=22801-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.39.22801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.39.22801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.39.22801"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7559409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moura</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pinheiro</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Silva</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Comparative context analysis of codon pairs on an ORFeome scale</div>, <div class="source">Genome Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">R28</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2005-6-3-r28" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2005-6-3-r28</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20context%20analysis%20of%20codon%20pairs%20on%20an%20ORFeome%20scale&amp;author=G.%20Moura&amp;author=M.%20Pinheiro&amp;author=R.%20Silva&amp;publication_year=2005&amp;journal=Genome%20Biol.&amp;volume=6&amp;pages=R28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-3-r28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-3-r28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-3-r28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15774029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gu</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Araki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tian</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Avoidance of long mononucleotide repeats in codon pair usage</div>, <div class="source">Genetics</div> , <div class="year">2010</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">1077</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.110.121137" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.110.121137</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avoidance%20of%20long%20mononucleotide%20repeats%20in%20codon%20pair%20usage&amp;author=T.%20Gu&amp;author=S.%20Tan&amp;author=X.%20Gou&amp;author=H.%20Araki&amp;author=D.%20Tian&amp;publication_year=2010&amp;journal=Genetics&amp;volume=186&amp;pages=1077-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.121137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.121137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.121137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boycheva</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chkodrov</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Ivanov</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Codon pairs in the genome of <em>Escherichia coli</em></div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">987</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btg082" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btg082</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20pairs%20in%20the%20genome%20of%20Escherichia%20coli&amp;author=S.%20Boycheva&amp;author=G.%20Chkodrov&amp;author=I.%20Ivanov&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=987-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg082"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12761062" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nierhaus</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Wadzack</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Burkhardt</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Structure of the elongating ribosome: arrangement of the two tRNAs before and after translocation</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">945</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.95.3.945" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.95.3.945</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20the%20elongating%20ribosome%3A%20arrangement%20of%20the%20two%20tRNAs%20before%20and%20after%20translocation&amp;author=K.H.%20Nierhaus&amp;author=J.%20Wadzack&amp;author=N.%20Burkhardt&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=945-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.3.945" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.3.945" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.3.945"> </span></div> </div></div></div></div></div><div content-id="DSS027C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Yarus</div> <div class="given-names">M.</div></div>. </div><div class="article-title">tRNA-tRNA interactions within cellular ribosomes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1989</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">4397</div>-<div class="lpage">401</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.86.12.4397" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.86.12.4397</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNA-tRNA%20interactions%20within%20cellular%20ribosomes&amp;author=D.%20Smith&amp;author=M.%20Yarus&amp;publication_year=1989&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=86&amp;pages=4397-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.12.4397" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.12.4397" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.12.4397"> </span></div> </div></div></div></div></div><div content-id="DSS027C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K.D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D.R.</div></div>. </div><div class="article-title">NCBI reference sequences (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D61</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl842" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl842</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20reference%20sequences%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D61-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crick</div> <div class="given-names">F.H.</div></div>. </div><div class="article-title">Codon–anticodon pairing: the wobble hypothesis</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1966</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">548</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0022-2836(66)80022-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0022-2836(66)80022-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%E2%80%93anticodon%20pairing%3A%20the%20wobble%20hypothesis&amp;author=F.H.%20Crick&amp;publication_year=1966&amp;journal=J.%20Mol.%20Biol.&amp;volume=19&amp;pages=548-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(66)80022-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(66)80022-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(66)80022-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5969078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rosner</div> <div class="given-names">B.</div></div>. </div>, <div class="source">Fundamentals of Biostatistics(7th Edition)</div> , <div class="year">2010</div><div class="publisher-name">Brooks/Cole Publishing Company, Cengage Learning, Inc., Independence, Kentucky, USA</div></div></div></div></div><div content-id="DSS027C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msr121" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msr121</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div>, et al. </div><div class="article-title">Clustal W and Clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2947</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btm404" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btm404</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1987</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1281</div>-<div class="lpage">95</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/15.3.1281" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/15.3.1281</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20Adaptation%20Index%E2%80%94a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Tuohy</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Mosurski</div> <div class="given-names">K.R.</div></div>. </div><div class="article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1986</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">5125</div>-<div class="lpage">43</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/14.13.5125" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/14.13.5125</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20yeast%3A%20cluster%20analysis%20clearly%20differentiates%20highly%20and%20lowly%20expressed%20genes&amp;author=P.M.%20Sharp&amp;author=T.M.%20Tuohy&amp;author=K.R.%20Mosurski&amp;publication_year=1986&amp;journal=Nucleic%20Acids%20Res.&amp;volume=14&amp;pages=5125-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/14.13.5125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F14.13.5125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F14.13.5125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3526280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Compositional variation in bacterial genes and proteins with potential expression level</div>, <div class="source">FEBS Lett.</div> , <div class="year">2005</div>, vol. <div class="volume">579</div> (pg. <div class="fpage">5205</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.febslet.2005.08.042" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.febslet.2005.08.042</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20variation%20in%20bacterial%20genes%20and%20proteins%20with%20potential%20expression%20level&amp;author=S.%20Das&amp;author=S.%20Ghosh&amp;author=A.%20Pan&amp;author=C.%20Dutta&amp;publication_year=2005&amp;journal=FEBS%20Lett.&amp;volume=579&amp;pages=5205-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2005.08.042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2005.08.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2005.08.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16165133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Codon and amino acid usage in two major human pathogens of genus <em>Bartonella</em>—optimization between replicational-transcriptional selection, translational control and cost minimization</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">91</div>-<div class="lpage">102</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/12.2.91" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/12.2.91</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20and%20amino%20acid%20usage%20in%20two%20major%20human%20pathogens%20of%20genus%20Bartonella%E2%80%94optimization%20between%20replicational-transcriptional%20selection%2C%20translational%20control%20and%20cost%20minimization&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=S.%20Chatterjee&amp;author=C.%20Dutta&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Synonymous codon usage in adenoviruses: influence of mutation, selection and protein hydropathy</div>, <div class="source">Virus Res.</div> , <div class="year">2006</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">227</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.virusres.2005.10.007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.virusres.2005.10.007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20adenoviruses%3A%20influence%20of%20mutation%2C%20selection%20and%20protein%20hydropathy&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=C.%20Dutta&amp;publication_year=2006&amp;journal=Virus%20Res.&amp;volume=117&amp;pages=227-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virusres.2005.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virusres.2005.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virusres.2005.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16307819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Roymondal</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Chottopadhyay</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Gene expression profile of the cyanobacterium <em>Synechocystis</em> genome</div>, <div class="source">Gene</div> , <div class="year">2012</div>, vol. <div class="volume">497</div> (pg. <div class="fpage">344</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2012.01.023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2012.01.023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20profile%20of%20the%20cyanobacterium%20Synechocystis%20genome&amp;author=S.%20Das&amp;author=U.%20Roymondal&amp;author=B.%20Chottopadhyay&amp;author=S.%20Sahoo&amp;publication_year=2012&amp;journal=Gene&amp;volume=497&amp;pages=344-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2012.01.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2012.01.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2012.01.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22310391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Roymondal</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Analyzing gene expression from relative codon usage bias in yeast genome: a statistical significance and biological relevance</div>, <div class="source">Gene</div> , <div class="year">2009</div>, vol. <div class="volume">443</div> (pg. <div class="fpage">121</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.gene.2009.04.022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.gene.2009.04.022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20gene%20expression%20from%20relative%20codon%20usage%20bias%20in%20yeast%20genome%3A%20a%20statistical%20significance%20and%20biological%20relevance&amp;author=S.%20Das&amp;author=U.%20Roymondal&amp;author=S.%20Sahoo&amp;publication_year=2009&amp;journal=Gene&amp;volume=443&amp;pages=121-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2009.04.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2009.04.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2009.04.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19410638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roymondal</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sahoo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Predicting gene expression level from relative codon usage bias: an application to <em>Escherichia coli</em> genome</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">13</div>-<div class="lpage">30</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn029" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn029</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20gene%20expression%20level%20from%20relative%20codon%20usage%20bias%3A%20an%20application%20to%20Escherichia%20coli%20genome&amp;author=U.%20Roymondal&amp;author=S.%20Das&amp;author=S.%20Sahoo&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=13-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Bacillus subtilis</em> reflects both translational selection and mutational biases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1987</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">8023</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/15.19.8023" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/15.19.8023</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Bacillus%20subtilis%20reflects%20both%20translational%20selection%20and%20mutational%20biases&amp;author=D.C.%20Shields&amp;author=P.M.%20Sharp&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=8023-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.19.8023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.19.8023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.19.8023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3118331" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Cowe</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Shields</div> <div class="given-names">D.C.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Codon usage patterns in <em>Escherichia coli</em>, <em>Bacillus subtilis</em>, <em>Saccharomyces cerevisiae</em>, <em>Schizosaccharomyces pombe</em>, <em>Drosophila melanogaster</em> and <em>Homo sapiens</em>: a review of the considerable within-species diversity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1988</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">8207</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/16.17.8207" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/16.17.8207</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20patterns%20in%20Escherichia%20coli%2C%20Bacillus%20subtilis%2C%20Saccharomyces%20cerevisiae%2C%20Schizosaccharomyces%20pombe%2C%20Drosophila%20melanogaster%20and%20Homo%20sapiens%3A%20a%20review%20of%20the%20considerable%20within-species%20diversity&amp;author=P.M.%20Sharp&amp;author=E.%20Cowe&amp;author=D.G.%20Higgins&amp;author=D.C.%20Shields&amp;author=K.H.%20Wolfe&amp;author=F.%20Wright&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=8207-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.17.8207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.17.8207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.17.8207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3138659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.</div></div>. </div><div class="article-title">Codon usage bias from tRNA's point of view: redundancy, specialization, and efficient decoding for translation optimization</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2279</div>-<div class="lpage">86</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.2896904" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.2896904</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20bias%20from%20tRNA's%20point%20of%20view%3A%20redundancy%2C%20specialization%2C%20and%20efficient%20decoding%20for%20translation%20optimization&amp;author=E.P.%20Rocha&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=2279-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.2896904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.2896904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.2896904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15479947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">5036</div>-<div class="lpage">44</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkh834" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkh834</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection&amp;author=M.%20dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS027C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS027C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vieira-Silva</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.</div></div>. </div><div class="article-title">The systemic imprint of growth and its uses in ecological (meta)genomics</div>, <div class="source">PLoS Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e1000808</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pgen.1000808" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pgen.1000808</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20systemic%20imprint%20of%20growth%20and%20its%20uses%20in%20ecological%20(meta)genomics&amp;author=S.%20Vieira-Silva&amp;author=E.P.%20Rocha&amp;publication_year=2010&amp;journal=PLoS%20Genet.&amp;volume=6&amp;pages=e1000808" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000808" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000808" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000808"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20090831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=481278&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss027-dss027supp_table2.xls?Expires=2147483647&amp;Signature=NwqpKPKXKBy0pvGQWOiBA1U8WWZWlgm9b6d17Yit0wsqRQELBVPDTBZXYV8DxAqxqMvHj5X~Wal~wsIwXgHdDgu~vOYVLnwrwBnHwsgvOh-sKx6ubSRFqtBa0hHZGCE0QAQ1i5PoP2mxHCzl-nCuoYFNrItph1IxP61nzfnNIQFlus2qYk2HRYYxFrMGrodB~bOIfBHjE~EZ5bSD~JyAv1A8v~4AJ57atgSXWNKt4xin-zSK-A0IstObxagfpwTrimFUjc1AZMohoGzkhaWzlALPnlW37~oDPoMomJaqdP8x9WSVfDmRUwxE1CFICv2UIiHPhE40epx3uPxXMJoqcA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss027-dss027supp_table1.xls?Expires=2147483647&amp;Signature=EjY~ZPg7J5mTcm-FAgHfvnn53oAOV9AYpPwatfD-07sMXLSm4cEtC2MbC7Jbgc2KZwCAUhUfOpJKaEY6L4IG1E7GBWw-u1An~iypCb51Oru3Un2EMNBz7~dcor8LViYssdo7hyXL-5wD1WjvH5ntEMWL5~rIOpRhcF8-zE9JEbnaVSt~qVv0PIYIERuWJ0JISFyW9d731diCsHu-zfCzP0OS6lzyKARe1JjNfpjw7PmlRuTXhqqMY~9z8zW~BOZlpxDiMSat5rm8fb1CZdEiCET1fX4J3pwXxZ0iP5iN9yEuAH9rBD5S57My4XJ0C-GxmyBTawwZ64p8M8H80-DnSA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/6/10.1093/dnares/dss027/2/dss027-dss027supp_fig1.pdf?Expires=2147483647&amp;Signature=frMUkQhYxlbxRGiNNRO1uBxf~n6qf9tQs8cVCxjdGp-xEG3aOd3GaCH5WrJyF65ntM1gRI-db5NA7zUuz57G0UWhlcRmtZuoWUOlw4BA1qgElXZkYAxW8RodMPv4Z3p0-hdLH1kD6AXZE1-LXzlG262~aCt8Mi5r61tKnCjwfvvumIx0kYtsF7B-uhzf4F~0C9jtVEcxjHkkzpdrGm5l9I33iBVUYAH9xTz7bTGRd~6TxuXDOZ8tvze8-9LXhINmmVilLKxx08DHXHHl~RJwa9XzjFCZPvFgOVoi7pXQ0xkCwZ8XuxvdVI6x3jUrLYFcKNSShP9nWHxGdLqVs8pqhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">190</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="481278"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000313135200004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f6%2f477%2f481278">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000313135200004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F6%2F477%2F481278" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/481278'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Universal Pattern and Diverse Strengths of Successive Synonymous Codon Bias in Three Domains of Life, Particularly Among Prokaryotic Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f6%2f477%2f481278">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

