

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/1/1/381937","@type":"ScholarlyArticle","name":"Quantitative Trait Loci Mapping in  Brassica rapa  Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of  Brassica  Species","datePublished":"2012-12-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2013-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss029","keywords":["<italic>Brassica rapa</italic>","quantitative trait loci (QTL)","morphological traits","single nucleotide polymorphism (SNP)","conserved genome blocks"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/1/1/381937","author":[{"name":"Li, Xiaonan","affiliation":"1  Molecular Genetics and Genomics Laboratory, Department of Horticulture,  Chungnam National University ,  Gung-Dong, Daejeon, Yuseong-Gu 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Ramchiary, Nirala","affiliation":"1  Molecular Genetics and Genomics Laboratory, Department of Horticulture,  Chungnam National University ,  Gung-Dong, Daejeon, Yuseong-Gu 305-764 ,  Republic of Korea    2  Department of Biological Science ,  Institute of Science and Technology, Gauhati University ,  Guwahati, Assam 781 014 ,  India    3  School of Life Sciences ,  Jawaharlal Nehru University ,  Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Dhandapani, Vignesh","affiliation":"1  Molecular Genetics and Genomics Laboratory, Department of Horticulture,  Chungnam National University ,  Gung-Dong, Daejeon, Yuseong-Gu 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Choi, Su Ryun","affiliation":"1  Molecular Genetics and Genomics Laboratory, Department of Horticulture,  Chungnam National University ,  Gung-Dong, Daejeon, Yuseong-Gu 305-764 ,  Republic of Korea","@type":"Person"},{"name":"Hur, Yoonkang","affiliation":"4  Department of Biology ,  Chungnam National University ,  Daejeon 305-764 ,  Korea","@type":"Person"},{"name":"Nou, Ill-Sup","affiliation":"5  Department of Horticulture ,  Sunchon National University ,  Jeonnam 540-742 ,  Korea","@type":"Person"},{"name":"Yoon, Moo Kyoung","affiliation":"6  National Institute of Horticultural and Herbal Science, 203 Cheoncheon-ro, jangan-gu, Suwon-si, Gyeonggi-do 440-706 ,  Republic  of Korea","@type":"Person"},{"name":"Lim, Yong Pyo","affiliation":"1  Molecular Genetics and Genomics Laboratory, Department of Horticulture,  Chungnam National University ,  Gung-Dong, Daejeon, Yuseong-Gu 305-764 ,  Republic of Korea","@type":"Person"}],"description":"Abstract.  Brassica rapa is an important crop species that produces vegetables, oilseed, and fodder. Although many studies reported quantitative trait loci (QTL","pageStart":"1","pageEnd":"16","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Quantitative Trait Loci Mapping in  Brassica rapa  Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of  Brassica  Species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Quantitative Trait Loci Mapping in  Brassica rapa  Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of  Brassica  Species" />
    <meta property="og:description" content="Abstract.  Brassica rapa is an important crop species that produces vegetables, oilseed, and fodder. Although many studies reported quantitative trait loci (QTL" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/1/1/381937" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-12-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Xiaonan" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Laboratory, Department of Horticulture,Chungnam National University, Gung-Dong, Daejeon, Yuseong-Gu 305-764, Republic of Korea" /><meta name="citation_author" content="Ramchiary, Nirala" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Laboratory, Department of Horticulture,Chungnam National University, Gung-Dong, Daejeon, Yuseong-Gu 305-764, Republic of Korea" /><meta name="citation_author_institution" content="Department of Biological Science, Institute of Science and Technology, Gauhati University, Guwahati, Assam 781 014, India" /><meta name="citation_author_institution" content="School of Life Sciences, Jawaharlal Nehru University, Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Dhandapani, Vignesh" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Laboratory, Department of Horticulture,Chungnam National University, Gung-Dong, Daejeon, Yuseong-Gu 305-764, Republic of Korea" /><meta name="citation_author" content="Choi, Su Ryun" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Laboratory, Department of Horticulture,Chungnam National University, Gung-Dong, Daejeon, Yuseong-Gu 305-764, Republic of Korea" /><meta name="citation_author" content="Hur, Yoonkang" /><meta name="citation_author_institution" content="Department of Biology, Chungnam National University, Daejeon 305-764, Korea" /><meta name="citation_author" content="Nou, Ill-Sup" /><meta name="citation_author_institution" content="Department of Horticulture, Sunchon National University, Jeonnam 540-742, Korea" /><meta name="citation_author" content="Yoon, Moo Kyoung" /><meta name="citation_author_institution" content="National Institute of Horticultural and Herbal Science, 203 Cheoncheon-ro, jangan-gu, Suwon-si, Gyeonggi-do 440-706, Republicof Korea" /><meta name="citation_author" content="Lim, Yong Pyo" /><meta name="citation_author_institution" content="Molecular Genetics and Genomics Laboratory, Department of Horticulture,Chungnam National University, Gung-Dong, Daejeon, Yuseong-Gu 305-764, Republic of Korea" /><meta name="citation_title" content="Quantitative Trait Loci Mapping in Brassica rapa Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of Brassica Species" /><meta name="citation_firstpage" content="1" /><meta name="citation_lastpage" content="16" /><meta name="citation_doi" content="10.1093/dnares/dss029" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2013/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Simple sequence repeat-based comparative genomics between Brassica rapa and Arabidopsis thaliana: the genetic origin of clubroot resistance; Genetics;  citation_year=2006;  citation_volume=173;  citation_pages=309-19; " /><meta name="citation_reference" content="citation_title=Development of a high density integrated reference genetic linkage map for the multinational Brassica rapa Genome Sequencing Project; Genome;  citation_year=2010;  citation_volume=53;  citation_pages=939-47; " /><meta name="citation_reference" content="citation_title=Mapping a QTL for oleic acid concentration in spring turnip rape (Brassica rapa ssp. oleifera); Theor. Appl. Genet.;  citation_year=1996;  citation_volume=92;  citation_pages=952-6; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for glucosinolate accumulation in Brassica rapa leaves; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=1017-32; " /><meta name="citation_reference" content="citation_title=Structural and functional comparative mapping between the Brassica A genomes in allotetraploid Brassica napus and diploid Brassica rapa; Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=927-41; " /><meta name="citation_reference" content="citation_title=Fine mapping of the clubroot resistance gene, Crr3, in Brassica rapa; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=114;  citation_pages=81-91; " /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci for resistance to Xanthomonas campestris pv.campestris in Brassica rapa; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=114;  citation_pages=637-45; " /><meta name="citation_reference" content="citation_title=Quantitative trait loci for flowering time and morphological traits in multiple populations of Brassica rapa; J. Exp. Bot.;  citation_year=2007;  citation_volume=58;  citation_pages=4005-16; " /><meta name="citation_reference" content="citation_title=A Brassica rapa linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits; DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=311-23; " /><meta name="citation_reference" content="citation_title=Segmental structure of the Brassica napus genome based on comparative analysis with Arabidopsis thaliana; Genetics;  citation_year=2005;  citation_volume=171;  citation_pages=765-81; " /><meta name="citation_reference" content="citation_title=Comparative mapping of Brassica juncea and Arabidopsis thaliana using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=113" /><meta name="citation_reference" content="citation_title=Genome-wide comparative analysis of the Brassica rapa gene space reveals genome shrinkage and differential loss of duplicated genes after whole genome triplication; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R111" /><meta name="citation_reference" content="citation_title=Comparative genome mapping in Brassica; Genetics;  citation_year=1996;  citation_volume=144;  citation_pages=1903-10; " /><meta name="citation_reference" content="citation_title=The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes; Trends Plant Sci.;  citation_year=2006;  citation_volume=11;  citation_pages=535-42; " /><meta name="citation_reference" content="citation_title=The Korea Brassica Genome Project: a glimpse of the Brassica genome based on comparative genome analysis with Arabidopsis; Comp. Funct. Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=138-46; " /><meta name="citation_reference" content="citation_title=The genome of the mesopolyploid crop species Brassica rapa; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=1035-9; " /><meta name="citation_reference" content="citation_title=Map based cloning and characterization of a gene controlling hairiness and seed coat color traits in Brassica rapa; Plant Mol. Biol.;  citation_year=2009;  citation_volume=69;  citation_pages=553-63; " /><meta name="citation_reference" content="citation_title=Genic microsatellite markers in Brassica rapa: development, characterization, mapping, and their utility in other cultivated and wild Brassica relatives; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=305-20; " /><meta name="citation_reference" content="citation_publisher=North Carolina State University, Raleigh, NC; Windows QTL Cartographer 2.5;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=Mapping of yield influencing QTL in Brassica juncea: implications for breeding of a major oilseed crop of dryland areas; Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=807-17; " /><meta name="citation_reference" content="citation_title=Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (Brassica napus L.): 1. Identification of genomic regions from winter germplasm; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=549-61; " /><meta name="citation_reference" content="citation_title=Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (Brassica napus L.): 2. Identification of alleles from unadapted germplasm; Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=597-609; " /><meta name="citation_reference" content="citation_title=AUGUSTUS at EGASP: using EST, protein and genomic alignments for improved gene prediction in the human genome; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=S11" /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1851-8; " /><meta name="citation_reference" content="citation_title=Application of high-resolution DNA melting for genotyping and variant scanning of diploid and autotetraploid potato; Mol. Breed.;  citation_year=2010;  citation_volume=25;  citation_pages=67-90; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis in Brassica rapa under the abiotic stresses using Brassica 24K oligo microarray; Mol. Cells;  citation_year=2008;  citation_volume=26;  citation_pages=595-605; " /><meta name="citation_reference" content="citation_title=Analysis of the developmental roles of the Arabidopsis gibberellin 20-oxidases demonstrates that GA20ox1, -2, and -3 are the dominant paralogs; Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=941-60; " /><meta name="citation_reference" content="citation_title=Cytokinin oxidase regulates rice grain production; Science;  citation_year=2005;  citation_volume=309;  citation_pages=741-5; " /><meta name="citation_reference" content="citation_title=Cytokinin regulates the activity of reproductive meristems, flower organ size, ovule formation, and thus seed yield in Arabidopsis thaliana; Plant Cell;  citation_year=2011;  citation_volume=23;  citation_pages=69-80; " /><meta name="citation_reference" content="citation_title=GASA4, one of the 14-member Arabidopsis GASA family of small polypeptides, regulates flowering and seed development; Plant Cell Physiol.;  citation_year=2007;  citation_volume=48;  citation_pages=471-83; " /><meta name="citation_reference" content="citation_title=The AtGRF family of putative transcription factors is involved in leaf and cotyledon growth in Arabidopsis; Plant J.;  citation_year=2003;  citation_volume=36;  citation_pages=94-104; " /><meta name="citation_reference" content="citation_title=Asymmetric leaves1 and auxin activities converge to repress BREVIPEDICELLUS expression and promote leaf development in Arabidopsis; Development;  citation_year=2006;  citation_volume=133;  citation_pages=3955-61; " /><meta name="citation_reference" content="citation_title=Genetic analysis reveals that C19-GA 2-oxidation is a major gibberellin inactivation pathway in Arabidopsis; Plant Cell;  citation_year=2008;  citation_volume=20;  citation_pages=2420-36; " /><meta name="citation_reference" content="citation_title=Altered patterns of fractionation and exon deletions in Brassica rapa support a two-step model of paleohexaploidy; Genetics;  citation_year=2012;  citation_volume=190;  citation_pages=1563-74; " /><meta name="citation_reference" content="citation_title=Characterization and effects of the replicated flowering time gene FLC in Brassica rapa; Genetics;  citation_year=2002;  citation_volume=162;  citation_pages=1457-68; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/1/1/1083471/dss029.pdf" /><meta name="description" content="Abstract.  Brassica rapa is an important crop species that produces vegetables, oilseed, and fodder. Although many studies reported quantitative trait loci (QTL" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/1/1/381937" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/1/1/381937" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/1/1/381937");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss029';
    </script>
<title>
	Quantitative Trait Loci Mapping in Brassica rapa Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of Brassica Species | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./381937" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaxiPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMaSwgWGlhb25hbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBHZW5ldGljcyBhbmQgR2Vub21pY3MgTGFib3JhdG9yeSwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsQ2h1bmduYW0gTmF0aW9uYWwgVW5pdmVyc2l0eSwgR3VuZy1Eb25nLCBEYWVqZW9uLCBZdXNlb25nLUd1IDMwNS03NjQsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmFtY2hpYXJ5LCBOaXJhbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgR2VuZXRpY3MgYW5kIEdlbm9taWNzIExhYm9yYXRvcnksIERlcGFydG1lbnQgb2YgSG9ydGljdWx0dXJlLENodW5nbmFtIE5hdGlvbmFsIFVuaXZlcnNpdHksIEd1bmctRG9uZywgRGFlamVvbiwgWXVzZW9uZy1HdSAzMDUtNzY0LCBSZXB1YmxpYyBvZiBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlLCBJbnN0aXR1dGUgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgR2F1aGF0aSBVbml2ZXJzaXR5LCBHdXdhaGF0aSwgQXNzYW0gNzgxIDAxNCwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgSmF3YWhhcmxhbCBOZWhydSBVbml2ZXJzaXR5LCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEaGFuZGFwYW5pLCBWaWduZXNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTW9sZWN1bGFyIEdlbmV0aWNzIGFuZCBHZW5vbWljcyBMYWJvcmF0b3J5LCBEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSxDaHVuZ25hbSBOYXRpb25hbCBVbml2ZXJzaXR5LCBHdW5nLURvbmcsIERhZWplb24sIFl1c2VvbmctR3UgMzA1LTc2NCwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaG9pLCBTdSBSeXVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTW9sZWN1bGFyIEdlbmV0aWNzIGFuZCBHZW5vbWljcyBMYWJvcmF0b3J5LCBEZXBhcnRtZW50IG9mIEhvcnRpY3VsdHVyZSxDaHVuZ25hbSBOYXRpb25hbCBVbml2ZXJzaXR5LCBHdW5nLURvbmcsIERhZWplb24sIFl1c2VvbmctR3UgMzA1LTc2NCwgUmVwdWJsaWMgb2YgS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdXIsIFlvb25rYW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2d5LCBDaHVuZ25hbSBOYXRpb25hbCBVbml2ZXJzaXR5LCBEYWVqZW9uIDMwNS03NjQsIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTm91LCBJbGwtU3VwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsIFN1bmNob24gTmF0aW9uYWwgVW5pdmVyc2l0eSwgSmVvbm5hbSA1NDAtNzQyLCBLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illvb24sIE1vbyBLeW91bmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgSG9ydGljdWx0dXJhbCBhbmQgSGVyYmFsIFNjaWVuY2UsIDIwMyBDaGVvbmNoZW9uLXJvLCBqYW5nYW4tZ3UsIFN1d29uLXNpLCBHeWVvbmdnaS1kbyA0NDAtNzA2LCBSZXB1YmxpY29mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGltLCBZb25nIFB5byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBHZW5ldGljcyBhbmQgR2Vub21pY3MgTGFib3JhdG9yeSwgRGVwYXJ0bWVudCBvZiBIb3J0aWN1bHR1cmUsQ2h1bmduYW0gTmF0aW9uYWwgVW5pdmVyc2l0eSwgR3VuZy1Eb25nLCBEYWVqZW9uLCBZdXNlb25nLUd1IDMwNS03NjQsIFJlcHVibGljIG9mIEtvcmVhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJRdWFudGl0YXRpdmUgVHJhaXQgTG9jaSBNYXBwaW5nIGluIEJyYXNzaWNhIHJhcGEgUmV2ZWFsZWQgdGhlIFN0cnVjdHVyYWwgYW5kIEZ1bmN0aW9uYWwgQ29uc2VydmF0aW9uIG9mIEdlbmV0aWMgTG9jaSBHb3Zlcm5pbmcgTW9ycGhvbG9naWNhbCBhbmQgWWllbGQgQ29tcG9uZW50IFRyYWl0cyBpbiB0aGUgQSwgQiwgYW5kIEMgU3ViZ2Vub21lcyBvZiBCcmFzc2ljYSBTcGVjaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMy8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbXBsZSBzZXF1ZW5jZSByZXBlYXQtYmFzZWQgY29tcGFyYXRpdmUgZ2Vub21pY3MgYmV0d2VlbiBCcmFzc2ljYSByYXBhIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYTogdGhlIGdlbmV0aWMgb3JpZ2luIG9mIGNsdWJyb290IHJlc2lzdGFuY2U7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3MzsgIGNpdGF0aW9uX3BhZ2VzPTMwOS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgb2YgYSBoaWdoIGRlbnNpdHkgaW50ZWdyYXRlZCByZWZlcmVuY2UgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgdGhlIG11bHRpbmF0aW9uYWwgQnJhc3NpY2EgcmFwYSBHZW5vbWUgU2VxdWVuY2luZyBQcm9qZWN0OyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz05MzktNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYSBRVEwgZm9yIG9sZWljIGFjaWQgY29uY2VudHJhdGlvbiBpbiBzcHJpbmcgdHVybmlwIHJhcGUgKEJyYXNzaWNhIHJhcGEgc3NwLiBvbGVpZmVyYSk7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz05NTItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIGdsdWNvc2lub2xhdGUgYWNjdW11bGF0aW9uIGluIEJyYXNzaWNhIHJhcGEgbGVhdmVzOyBOZXcgUGh5dG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzk7ICBjaXRhdGlvbl9wYWdlcz0xMDE3LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuZCBmdW5jdGlvbmFsIGNvbXBhcmF0aXZlIG1hcHBpbmcgYmV0d2VlbiB0aGUgQnJhc3NpY2EgQSBnZW5vbWVzIGluIGFsbG90ZXRyYXBsb2lkIEJyYXNzaWNhIG5hcHVzIGFuZCBkaXBsb2lkIEJyYXNzaWNhIHJhcGE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9OTI3LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lIG1hcHBpbmcgb2YgdGhlIGNsdWJyb290IHJlc2lzdGFuY2UgZ2VuZSwgQ3JyMywgaW4gQnJhc3NpY2EgcmFwYTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTQ7ICBjaXRhdGlvbl9wYWdlcz04MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIHJlc2lzdGFuY2UgdG8gWGFudGhvbW9uYXMgY2FtcGVzdHJpcyBwdi5jYW1wZXN0cmlzIGluIEJyYXNzaWNhIHJhcGE7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTE0OyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBmb3IgZmxvd2VyaW5nIHRpbWUgYW5kIG1vcnBob2xvZ2ljYWwgdHJhaXRzIGluIG11bHRpcGxlIHBvcHVsYXRpb25zIG9mIEJyYXNzaWNhIHJhcGE7IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTQwMDUtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgQnJhc3NpY2EgcmFwYSBsaW5rYWdlIG1hcCBvZiBFU1QtYmFzZWQgU05QIG1hcmtlcnMgZm9yIGlkZW50aWZpY2F0aW9uIG9mIGNhbmRpZGF0ZSBnZW5lcyBjb250cm9sbGluZyBmbG93ZXJpbmcgdGltZSBhbmQgbGVhZiBtb3JwaG9sb2dpY2FsIHRyYWl0czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zMTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlZ21lbnRhbCBzdHJ1Y3R1cmUgb2YgdGhlIEJyYXNzaWNhIG5hcHVzIGdlbm9tZSBiYXNlZCBvbiBjb21wYXJhdGl2ZSBhbmFseXNpcyB3aXRoIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzE7ICBjaXRhdGlvbl9wYWdlcz03NjUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1hcHBpbmcgb2YgQnJhc3NpY2EganVuY2VhIGFuZCBBcmFiaWRvcHNpcyB0aGFsaWFuYSB1c2luZyBJbnRyb24gUG9seW1vcnBoaXNtIChJUCkgbWFya2VyczogaG9tb2VvbG9nb3VzIHJlbGF0aW9uc2hpcHMsIGRpdmVyc2lmaWNhdGlvbiBhbmQgZXZvbHV0aW9uIG9mIHRoZSBBLCBCIGFuZCBDIEJyYXNzaWNhIGdlbm9tZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIEJyYXNzaWNhIHJhcGEgZ2VuZSBzcGFjZSByZXZlYWxzIGdlbm9tZSBzaHJpbmthZ2UgYW5kIGRpZmZlcmVudGlhbCBsb3NzIG9mIGR1cGxpY2F0ZWQgZ2VuZXMgYWZ0ZXIgd2hvbGUgZ2Vub21lIHRyaXBsaWNhdGlvbjsgR2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjExMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9tZSBtYXBwaW5nIGluIEJyYXNzaWNhOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDQ7ICBjaXRhdGlvbl9wYWdlcz0xOTAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQUJDJ3Mgb2YgY29tcGFyYXRpdmUgZ2Vub21pY3MgaW4gdGhlIEJyYXNzaWNhY2VhZTogYnVpbGRpbmcgYmxvY2tzIG9mIGNydWNpZmVyIGdlbm9tZXM7IFRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTM1LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgS29yZWEgQnJhc3NpY2EgR2Vub21lIFByb2plY3Q6IGEgZ2xpbXBzZSBvZiB0aGUgQnJhc3NpY2EgZ2Vub21lIGJhc2VkIG9uIGNvbXBhcmF0aXZlIGdlbm9tZSBhbmFseXNpcyB3aXRoIEFyYWJpZG9wc2lzOyBDb21wLiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEzOC00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgbWVzb3BvbHlwbG9pZCBjcm9wIHNwZWNpZXMgQnJhc3NpY2EgcmFwYTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMDM1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcCBiYXNlZCBjbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2VuZSBjb250cm9sbGluZyBoYWlyaW5lc3MgYW5kIHNlZWQgY29hdCBjb2xvciB0cmFpdHMgaW4gQnJhc3NpY2EgcmFwYTsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTU1My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuaWMgbWljcm9zYXRlbGxpdGUgbWFya2VycyBpbiBCcmFzc2ljYSByYXBhOiBkZXZlbG9wbWVudCwgY2hhcmFjdGVyaXphdGlvbiwgbWFwcGluZywgYW5kIHRoZWlyIHV0aWxpdHkgaW4gb3RoZXIgY3VsdGl2YXRlZCBhbmQgd2lsZCBCcmFzc2ljYSByZWxhdGl2ZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MzA1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Tm9ydGggQ2Fyb2xpbmEgU3RhdGUgVW5pdmVyc2l0eSwgUmFsZWlnaCwgTkM7IFdpbmRvd3MgUVRMIENhcnRvZ3JhcGhlciAyLjU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgb2YgeWllbGQgaW5mbHVlbmNpbmcgUVRMIGluIEJyYXNzaWNhIGp1bmNlYTogaW1wbGljYXRpb25zIGZvciBicmVlZGluZyBvZiBhIG1ham9yIG9pbHNlZWQgY3JvcCBvZiBkcnlsYW5kIGFyZWFzOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExNTsgIGNpdGF0aW9uX3BhZ2VzPTgwNy0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGFuYWx5c2lzIG9mIHNlZWQgeWllbGQgYW5kIG90aGVyIGNvbXBsZXggdHJhaXRzIGluIGh5YnJpZCBzcHJpbmcgcmFwZXNlZWQgKEJyYXNzaWNhIG5hcHVzIEwuKTogMS4gSWRlbnRpZmljYXRpb24gb2YgZ2Vub21pYyByZWdpb25zIGZyb20gd2ludGVyIGdlcm1wbGFzbTsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz01NDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSB0cmFpdCBhbmFseXNpcyBvZiBzZWVkIHlpZWxkIGFuZCBvdGhlciBjb21wbGV4IHRyYWl0cyBpbiBoeWJyaWQgc3ByaW5nIHJhcGVzZWVkIChCcmFzc2ljYSBuYXB1cyBMLik6IDIuIElkZW50aWZpY2F0aW9uIG9mIGFsbGVsZXMgZnJvbSB1bmFkYXB0ZWQgZ2VybXBsYXNtOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTExMzsgIGNpdGF0aW9uX3BhZ2VzPTU5Ny02MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFVR1VTVFVTIGF0IEVHQVNQOiB1c2luZyBFU1QsIHByb3RlaW4gYW5kIGdlbm9taWMgYWxpZ25tZW50cyBmb3IgaW1wcm92ZWQgZ2VuZSBwcmVkaWN0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9UzExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFmYXN0IGFuZCBtZW1vcnktZWZmaWNpZW50IGFsaWdubWVudCBvZiBzaG9ydCBETkEgc2VxdWVuY2VzIHRvIHRoZSBodW1hbiBnZW5vbWU7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgc2hvcnQgRE5BIHNlcXVlbmNpbmcgcmVhZHMgYW5kIGNhbGxpbmcgdmFyaWFudHMgdXNpbmcgbWFwcGluZyBxdWFsaXR5IHNjb3JlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xODUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIGhpZ2gtcmVzb2x1dGlvbiBETkEgbWVsdGluZyBmb3IgZ2Vub3R5cGluZyBhbmQgdmFyaWFudCBzY2FubmluZyBvZiBkaXBsb2lkIGFuZCBhdXRvdGV0cmFwbG9pZCBwb3RhdG87IE1vbC4gQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NjctOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgaW4gQnJhc3NpY2EgcmFwYSB1bmRlciB0aGUgYWJpb3RpYyBzdHJlc3NlcyB1c2luZyBCcmFzc2ljYSAyNEsgb2xpZ28gbWljcm9hcnJheTsgTW9sLiBDZWxsczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU5NS02MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBkZXZlbG9wbWVudGFsIHJvbGVzIG9mIHRoZSBBcmFiaWRvcHNpcyBnaWJiZXJlbGxpbiAyMC1veGlkYXNlcyBkZW1vbnN0cmF0ZXMgdGhhdCBHQTIwb3gxLCAtMiwgYW5kIC0zIGFyZSB0aGUgZG9taW5hbnQgcGFyYWxvZ3M7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz05NDEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9raW5pbiBveGlkYXNlIHJlZ3VsYXRlcyByaWNlIGdyYWluIHByb2R1Y3Rpb247IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzA5OyAgY2l0YXRpb25fcGFnZXM9NzQxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9raW5pbiByZWd1bGF0ZXMgdGhlIGFjdGl2aXR5IG9mIHJlcHJvZHVjdGl2ZSBtZXJpc3RlbXMsIGZsb3dlciBvcmdhbiBzaXplLCBvdnVsZSBmb3JtYXRpb24sIGFuZCB0aHVzIHNlZWQgeWllbGQgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02OS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R0FTQTQsIG9uZSBvZiB0aGUgMTQtbWVtYmVyIEFyYWJpZG9wc2lzIEdBU0EgZmFtaWx5IG9mIHNtYWxsIHBvbHlwZXB0aWRlcywgcmVndWxhdGVzIGZsb3dlcmluZyBhbmQgc2VlZCBkZXZlbG9wbWVudDsgUGxhbnQgQ2VsbCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTQ3MS04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEF0R1JGIGZhbWlseSBvZiBwdXRhdGl2ZSB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgaXMgaW52b2x2ZWQgaW4gbGVhZiBhbmQgY290eWxlZG9uIGdyb3d0aCBpbiBBcmFiaWRvcHNpczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz05NC0xMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWMgbGVhdmVzMSBhbmQgYXV4aW4gYWN0aXZpdGllcyBjb252ZXJnZSB0byByZXByZXNzIEJSRVZJUEVESUNFTExVUyBleHByZXNzaW9uIGFuZCBwcm9tb3RlIGxlYWYgZGV2ZWxvcG1lbnQgaW4gQXJhYmlkb3BzaXM7IERldmVsb3BtZW50OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTM5NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzaXMgcmV2ZWFscyB0aGF0IEMxOS1HQSAyLW94aWRhdGlvbiBpcyBhIG1ham9yIGdpYmJlcmVsbGluIGluYWN0aXZhdGlvbiBwYXRod2F5IGluIEFyYWJpZG9wc2lzOyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjQyMC0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJlZCBwYXR0ZXJucyBvZiBmcmFjdGlvbmF0aW9uIGFuZCBleG9uIGRlbGV0aW9ucyBpbiBCcmFzc2ljYSByYXBhIHN1cHBvcnQgYSB0d28tc3RlcCBtb2RlbCBvZiBwYWxlb2hleGFwbG9pZHk7IEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5MDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjMtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gYW5kIGVmZmVjdHMgb2YgdGhlIHJlcGxpY2F0ZWQgZmxvd2VyaW5nIHRpbWUgZ2VuZSBGTEMgaW4gQnJhc3NpY2EgcmFwYTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYyOyAgY2l0YXRpb25fcGFnZXM9MTQ1Ny02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIwLzEvMS8xMDgzNDcxL2RzczAyOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQnJhc3NpY2EgcmFwYSBpcyBhbiBpbXBvcnRhbnQgY3JvcCBzcGVjaWVzIHRoYXQgcHJvZHVjZXMgdmVnZXRhYmxlcywgb2lsc2VlZCwgYW5kIGZvZGRlci4gQWx0aG91Z2ggbWFueSBzdHVkaWVzIHJlcG9ydGVkIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIChRVEwiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM4MTkzNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRgOOV4lEfc/pxmfbfqm3FTHBqTRw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f1%2f381937" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f1%2f381937" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14916" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/1/1/m_cover.gif?Expires=2147483647&amp;Signature=K1v8cGQnstZYdECB9GA3oc5Ko3ohhQ9lF83XAKOVlOLlyAqR0JX0Vn18cJ3Fh6Ni4coRVii1nVVwFuzd9baa8EpiNPdDbUxg77BFyv3S5jO4sVroBj5PfHgZd5D14mPzL5KK6FHcweE5FD2RLiGEj~ct-3TIgAM5vP7r7dob4djMEW-dmZ2mnE0M4g6gvpwd4meZtXx6YM6PJ-qadpYPQyOJp7E7APzkRfsaUptwFx9azq6qh9BweylqnGS-k05la9PhL7T57M1047efztU-bcJDc0lmA7wiGh4dEYA5ldVPoM0OpeZ~9-tJ2gANZpruMQldUUZX51aSqOwvf70tSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/1">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5592125">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592125">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592127">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592127">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592132">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592143">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592143">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592179">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592179">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592189">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592189">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592191">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592191">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592193">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592193">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5592195">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5592195">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/20/1/17/382150">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Quantitative Trait Loci Mapping in <em>Brassica rapa</em> Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of <em>Brassica</em> Species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaonan Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Laboratory, Department of Horticulture,</div><div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Daejeon, Yuseong-Gu 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaonan+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaonan Li</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nirala Ramchiary</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Laboratory, Department of Horticulture,</div><div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Daejeon, Yuseong-Gu 305-764</div>, <div class="country">Republic of Korea</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biological Science</div>, <div class="insititution">Institute of Science and Technology, Gauhati University</div>, <div class="addr-line">Guwahati, Assam 781 014</div>, <div class="country">India</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">School of Life Sciences</div>, <div class="insititution">Jawaharlal Nehru University</div>, <div class="addr-line">Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nirala+Ramchiary">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ramchiary N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ramchiary N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nirala Ramchiary</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vignesh Dhandapani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Laboratory, Department of Horticulture,</div><div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Daejeon, Yuseong-Gu 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vignesh+Dhandapani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dhandapani V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dhandapani V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vignesh Dhandapani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Su Ryun Choi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Laboratory, Department of Horticulture,</div><div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Daejeon, Yuseong-Gu 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Su+Ryun+Choi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Choi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Choi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Su Ryun Choi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoonkang Hur</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Biology</div>, <div class="insititution">Chungnam National University</div>, <div class="addr-line">Daejeon 305-764</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoonkang+Hur">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hur Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hur Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoonkang Hur</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ill-Sup Nou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Horticulture</div>, <div class="insititution">Sunchon National University</div>, <div class="addr-line">Jeonnam 540-742</div>, <div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ill-Sup+Nou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nou I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nou I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ill-Sup Nou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Moo Kyoung Yoon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">National Institute of Horticultural and Herbal Science, 203 Cheoncheon-ro, jangan-gu, Suwon-si, Gyeonggi-do 440-706</div>, <div class="country">Republic</div><div class="country">of Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Moo+Kyoung+Yoon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoon M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoon M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Moo Kyoung Yoon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yong Pyo Lim</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Molecular Genetics and Genomics Laboratory, Department of Horticulture,</div><div class="insititution">Chungnam National University</div>, <div class="addr-line">Gung-Dong, Daejeon, Yuseong-Gu 305-764</div>, <div class="country">Republic of Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82-42-821-5739. Fax. +82-42-821-8847. Email: <a href="mailto:yplim@cnu.ac.kr" target="_blank">yplim@cnu.ac.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yong+Pyo+Lim">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lim Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lim Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yong Pyo Lim</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 1–16, <a href='https://doi.org/10.1093/dnares/dss029'>https://doi.org/10.1093/dnares/dss029</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 December 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">18 June 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 October 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="381937" href="/dnaresearch/article-pdf/20/1/1/1083471/dss029.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xiaonan Li, Nirala Ramchiary, Vignesh Dhandapani, Su Ryun Choi, Yoonkang Hur, Ill-Sup Nou, Moo Kyoung Yoon, Yong Pyo Lim;  Quantitative Trait Loci Mapping in <em>Brassica rapa</em> Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of <em>Brassica</em> Species, <em>DNA Research</em>, Volume 20, Issue 1, 1 February 2013, Pages 1–16, <a href='https://doi.org/10.1093/dnares/dss029'>https://doi.org/10.1093/dnares/dss029</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/381937?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/381937?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Quantitative%20Trait%20Loci%20Mapping%20in%20%26lt%3Bem%26gt%3BBrassica%20rapa%26lt%3B%2Fem%26gt%3B%20Revealed%20the%20Structural%20and%20Functional%20Conservation%20of%20Genetic%20Loci%20Governing%20Morphological%20and%20Yield%20Component%20Traits%20in%20the%20A%2C%20B%2C%20and%20C%20Subgenomes%20of%20%26lt%3Bem%26gt%3BBrassica%26lt%3B%2Fem%26gt%3B%20Species&amp;publicationDate=2012-12-07&amp;volumeNum=20&amp;issueNum=1&amp;author=Li%2C%20Xiaonan%3B%20Ramchiary%2C%20Nirala&amp;startPage=1&amp;endPage=16&amp;contentId=10.1093%2Fdnares%2Fdss029&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f1%2f381937" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5592125 id="5592125" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Brassica rapa</em> is an important crop species that produces vegetables, oilseed, and fodder. Although many studies reported quantitative trait loci (QTL) mapping, the genes governing most of its economically important traits are still unknown. In this study, we report QTL mapping for morphological and yield component traits in <em>B. rapa</em> and comparative map alignment between <em>B. rapa</em>, <em>B. napus</em>, <em>B. juncea</em>, and <em>Arabidopsis thaliana</em> to identify candidate genes and conserved QTL blocks between them. A total of 95 QTL were identified in different crucifer blocks of the <em>B. rapa</em> genome. Through synteny analysis with <em>A. thaliana</em>, <em>B. rapa</em> candidate genes and intronic and exonic single nucleotide polymorphisms in the parental lines were detected from whole genome resequenced data, a few of which were validated by mapping them to the QTL regions. Semi-quantitative reverse transcriptase PCR analysis showed differences in the expression levels of a few genes in parental lines. Comparative mapping identified five key major evolutionarily conserved crucifer blocks (R, J, F, E, and W) harbouring QTL for morphological and yield components traits between the A, B, and C subgenomes of <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. napus</em>. The information of the identified candidate genes could be used for breeding <em>B. rapa</em> and other related <em>Brassica</em> species.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Brassica rapa&quot;"><em>Brassica rapa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;quantitative trait loci (QTL)&quot;">quantitative trait loci (QTL)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;morphological traits&quot;">morphological traits</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism (SNP)&quot;">single nucleotide polymorphism (SNP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;conserved genome blocks&quot;">conserved genome blocks</a></div>                            <h2 scrollto-destination=5592127 id="5592127" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Brassica rapa</em> (2<em>n</em> = 20, AA) is an important <em>Brassica</em> species that is grown widely to produce leafy vegetables in Korea, China, and Japan, for vegetable oil in India, China, and Canada, and as a fodder crop in Europe. Chinese cabbage, pak choi, sarson, and turnips are distinct morphotypes of <em>B. rapa</em> belonging to different subspecies that are adapted to different geographical regions and climatic conditions. During the last two decades, studies by different laboratories using diverse germplasm led the development of several molecular markers and genetic linkage maps<sup><a reveal-id="DSS029C1" data-open="DSS029C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS029C2" data-open="DSS029C2" class="link link-ref link-reveal xref-bibr">2</a></sup> besides mapping quantitative trait loci (QTL) for erucic acid,<sup><a reveal-id="DSS029C3" data-open="DSS029C3" class="link link-ref link-reveal xref-bibr">3</a></sup> glucosinolates,<sup><a reveal-id="DSS029C4" data-open="DSS029C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS029C5" data-open="DSS029C5" class="link link-ref link-reveal xref-bibr">5</a></sup> disease resistance,<sup><a reveal-id="DSS029C6" data-open="DSS029C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS029C7" data-open="DSS029C7" class="link link-ref link-reveal xref-bibr">7</a></sup> plant morphology, flowering time, and yield component traits.<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Furthermore, comparative mapping between different cultivated <em>Brassica</em> species revealed the structural conservation between the homoeologous chromosomes of the A, B, and C subgenomes that were originally derived from three diploid <em>Brassica</em> species, namely, the <em>Brassica rapa</em>, <em>Brassica nigra</em> (BB, <em>n</em> = 8), and <em>Brassica oleracea</em> (CC, <em>n</em> = 9) genomes, and their polyploid derivatives, i.e. the important oilseed crops <em>Brassica juncea</em> (AABB, <em>n</em> = 18) and <em>Brassica napus</em> (AACC, <em>n</em> = 19), respectively.<sup><a reveal-id="DSS029C5" data-open="DSS029C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS029C10" data-open="DSS029C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Furthermore, several comparative mapping studies between <em>Brassica</em> and <em>Arabidopsis thaliana</em> revealed the triplicate nature of the <em>Brassica</em> genome, with an average of three copies of each chromosomal segment of <em>A. thaliana</em>, which resulted from the triplication of the whole <em>Brassica</em> genome at ∼11–12 MYA,<sup><a reveal-id="DSS029C12" data-open="DSS029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> although segmental conservation between <em>Brassica</em> and <em>A. thaliana</em> chromosomes has been observed at the gross level.<sup><a reveal-id="DSS029C10" data-open="DSS029C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS029C13" data-open="DSS029C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS029C14" data-open="DSS029C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Therefore, to decode the whole genome sequences of the complicated <em>Brassica</em> genomes to be used in breeding programmes and to study the divergence of gene function and genome evolution associated with polyploidy and extensive duplications, the ‘Multinational <em>Brassica</em> Genome Sequencing Project Consortium’ was initiated in 2003. The first <em>Brassica</em> genome to be sequenced among the six cultivated <em>Brassica</em> species was the A genome of the <em>B. rapa</em> Chinese cabbage Chiifu-401 cultivar that has a comparatively small genome (529 Mb) among the cultivated <em>Brassica</em> species,<sup><a reveal-id="DSS029C15" data-open="DSS029C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and the draft genome sequence was published in 2011.<sup><a reveal-id="DSS029C16" data-open="DSS029C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>With the availability of recently developed advanced next-generation sequencing (NGS) technology to sequence the whole genome of crop plants in a short time span, the emphasis is being shifted to genomics-assisted breeding from traditional crop breeding using conventional molecular markers. The combined use of QTL mapping, which detects functional loci for traits of interest, and whole genome sequence information to identify candidate genes and their variation between the parental lines of <em>B. rapa</em> will greatly supplement the development of gene-specific molecular markers for breeding this crop with desired plant architecture and quality. Many oleifera and sarson types were exploited for breeding high seed yield component traits in <em>B. rapa</em>, but the vegetable types were not exploited for this purpose. Although several QTL have been mapped for leaf morphology, yield components, and other quality traits, most of the genes underlying trait variation have not been identified.<sup><a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSS029C17" data-open="DSS029C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>Therefore, in the present study, we used chromosome-specific bacterial artificial chromosome-derived simple sequence repeats (BAC-SSRs) and gene specific markers, e.g. intron polymorphisms (IPs) and expressed sequenced tag-derived SSRs (EST-SSRs) to map QTL governing morphological and yield component traits. Using whole genome next-generation sequence information data of the parental lines in combination with comparative alignment with the <em>A. thaliana</em> genome, potential candidate genes and single nucleotide variations within some of the potential candidate genes were identified. Furthermore, the chromosomal regions of <em>B. rapa</em> containing clusters of QTL were aligned with the QTL regions of <em>B. juncea</em> and <em>B. napus</em> to identify structural and functional conservation between the A, B, and C subgenomes, so that the candidate gene information of <em>B. rapa</em> could be used for breeding these crops.</p>                            <h2 scrollto-destination=5592132 id="5592132" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5592133 id="5592133" class="section-title" data-legacy-id=s2a>2.1. Plant materials, growth conditions, and trait measurement</h3>
<p>The genetic map developed earlier by us<sup><a reveal-id="DSS029C2" data-open="DSS029C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS029C18" data-open="DSS029C18" class="link link-ref link-reveal xref-bibr">18</a></sup> using a CRF<sub>2</sub> mapping population that was derived by crossing the diverse parental lines ‘Chiifu 401–42,’ a vegetable-type Chinese cabbage, and ‘rapid cycling <em>B. rapa</em>’ (hereafter referred to as ‘RCBr’), was used for QTL mapping in this study. For phenotypic investigation, 12 F<sub>3</sub> plants derived by selfing each of the 190 F<sub>2</sub> plants were planted per replication in 3 replications in 2008, 2009, and 2010 from March to July. The seeds were germinated in cell trays in a greenhouse for 1 month. In 2009 and 2010, two sets were grown, one set was vernalized after 20 days of germination for 1 month and the other set was grown without vernalization. All the plants were transplanted to the open field of Chungnam National University, Daejeon, Korea. Fourteen morphological and yield component traits (Table <a reveal-id="DSS029TB1" data-open="DSS029TB1" class="link link-reveal link-table xref-fig">1</a>) were recorded in the CRF<sub>3</sub> mapping population and parental lines. Four plants from the middle of each row from each replication were used for phenotypic data measurement for each family, and the average value of three replications was taken as trait data. Flowering time was the only trait that was recorded in the vernalized plants. </p>                        <a id="5592135" scrollto-destination="5592135"></a>
<div content-id="DSS029TB1" class="table-modal"><div class="table-wrap"><div id="DSS029TB1" class="table-wrap-title" data-id="DSS029TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Details of traits measurement in CRF<sub>3</sub> population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait name </th><th>ABS </th><th>Trait description </th><th>Scale </th><th>Year </th></tr></thead><tbody><tr><td>Flowering time </td><td>FT </td><td>Days from sowing to opening of the first flower </td><td>Days </td><td>2008/2009/2010 </td></tr><tr><td>Bolting time </td><td>DB </td><td>Days from sowing to the emergence of bud </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Flowering time after vernalization </td><td>FT* </td><td>Days from sowing to opening of the first flower after vernalization for 1 month </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Plant height </td><td>PH </td><td>Height from ground to the stem top when first flower opens </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Leaf length </td><td>LL </td><td>From base of petiole to the tip of leaf </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Leaf width </td><td>LW </td><td>Leaf width at the widest point </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib length </td><td>MRL </td><td>Length from the bottom to the apex of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib width </td><td>MRW </td><td>Width of the bottom of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Petiole length </td><td>PL </td><td>From base of petiole to the bottom of lamina </td><td>cm </td><td>2009 </td></tr><tr><td>Siliqua length </td><td>SQL </td><td>Length from pedicel of siliqua to the top of seed beak </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua width </td><td>SQW </td><td>Width at the lengthwise midpoint of each siliqua </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua beak length </td><td>SBL </td><td>Length from the top of siliqua to the top of beak </td><td>cm </td><td>2009/2010 </td></tr><tr><td>Seeds per siliqua </td><td>SSQ </td><td>Number of seed per siliqua </td><td> </td><td>2009/2010 </td></tr><tr><td>Seed weight </td><td>SW </td><td>Seed weight of 100 seeds each line </td><td>mg </td><td>2009/2010 </td></tr></tbody></table></div><div content-id="DSS029TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait name </th><th>ABS </th><th>Trait description </th><th>Scale </th><th>Year </th></tr></thead><tbody><tr><td>Flowering time </td><td>FT </td><td>Days from sowing to opening of the first flower </td><td>Days </td><td>2008/2009/2010 </td></tr><tr><td>Bolting time </td><td>DB </td><td>Days from sowing to the emergence of bud </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Flowering time after vernalization </td><td>FT* </td><td>Days from sowing to opening of the first flower after vernalization for 1 month </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Plant height </td><td>PH </td><td>Height from ground to the stem top when first flower opens </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Leaf length </td><td>LL </td><td>From base of petiole to the tip of leaf </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Leaf width </td><td>LW </td><td>Leaf width at the widest point </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib length </td><td>MRL </td><td>Length from the bottom to the apex of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib width </td><td>MRW </td><td>Width of the bottom of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Petiole length </td><td>PL </td><td>From base of petiole to the bottom of lamina </td><td>cm </td><td>2009 </td></tr><tr><td>Siliqua length </td><td>SQL </td><td>Length from pedicel of siliqua to the top of seed beak </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua width </td><td>SQW </td><td>Width at the lengthwise midpoint of each siliqua </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua beak length </td><td>SBL </td><td>Length from the top of siliqua to the top of beak </td><td>cm </td><td>2009/2010 </td></tr><tr><td>Seeds per siliqua </td><td>SSQ </td><td>Number of seed per siliqua </td><td> </td><td>2009/2010 </td></tr><tr><td>Seed weight </td><td>SW </td><td>Seed weight of 100 seeds each line </td><td>mg </td><td>2009/2010 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>* Represent the vernalization treatment.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592135" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS029TB1" class="table-wrap-title" data-id="DSS029TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Details of traits measurement in CRF<sub>3</sub> population</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Trait name </th><th>ABS </th><th>Trait description </th><th>Scale </th><th>Year </th></tr></thead><tbody><tr><td>Flowering time </td><td>FT </td><td>Days from sowing to opening of the first flower </td><td>Days </td><td>2008/2009/2010 </td></tr><tr><td>Bolting time </td><td>DB </td><td>Days from sowing to the emergence of bud </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Flowering time after vernalization </td><td>FT* </td><td>Days from sowing to opening of the first flower after vernalization for 1 month </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Plant height </td><td>PH </td><td>Height from ground to the stem top when first flower opens </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Leaf length </td><td>LL </td><td>From base of petiole to the tip of leaf </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Leaf width </td><td>LW </td><td>Leaf width at the widest point </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib length </td><td>MRL </td><td>Length from the bottom to the apex of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib width </td><td>MRW </td><td>Width of the bottom of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Petiole length </td><td>PL </td><td>From base of petiole to the bottom of lamina </td><td>cm </td><td>2009 </td></tr><tr><td>Siliqua length </td><td>SQL </td><td>Length from pedicel of siliqua to the top of seed beak </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua width </td><td>SQW </td><td>Width at the lengthwise midpoint of each siliqua </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua beak length </td><td>SBL </td><td>Length from the top of siliqua to the top of beak </td><td>cm </td><td>2009/2010 </td></tr><tr><td>Seeds per siliqua </td><td>SSQ </td><td>Number of seed per siliqua </td><td> </td><td>2009/2010 </td></tr><tr><td>Seed weight </td><td>SW </td><td>Seed weight of 100 seeds each line </td><td>mg </td><td>2009/2010 </td></tr></tbody></table></div><div content-id="DSS029TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Trait name </th><th>ABS </th><th>Trait description </th><th>Scale </th><th>Year </th></tr></thead><tbody><tr><td>Flowering time </td><td>FT </td><td>Days from sowing to opening of the first flower </td><td>Days </td><td>2008/2009/2010 </td></tr><tr><td>Bolting time </td><td>DB </td><td>Days from sowing to the emergence of bud </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Flowering time after vernalization </td><td>FT* </td><td>Days from sowing to opening of the first flower after vernalization for 1 month </td><td>Days </td><td>2009/2010 </td></tr><tr><td>Plant height </td><td>PH </td><td>Height from ground to the stem top when first flower opens </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Leaf length </td><td>LL </td><td>From base of petiole to the tip of leaf </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Leaf width </td><td>LW </td><td>Leaf width at the widest point </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib length </td><td>MRL </td><td>Length from the bottom to the apex of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Midrib width </td><td>MRW </td><td>Width of the bottom of midrib </td><td>cm </td><td>2008/2009 </td></tr><tr><td>Petiole length </td><td>PL </td><td>From base of petiole to the bottom of lamina </td><td>cm </td><td>2009 </td></tr><tr><td>Siliqua length </td><td>SQL </td><td>Length from pedicel of siliqua to the top of seed beak </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua width </td><td>SQW </td><td>Width at the lengthwise midpoint of each siliqua </td><td>cm </td><td>2008/2009/2010 </td></tr><tr><td>Siliqua beak length </td><td>SBL </td><td>Length from the top of siliqua to the top of beak </td><td>cm </td><td>2009/2010 </td></tr><tr><td>Seeds per siliqua </td><td>SSQ </td><td>Number of seed per siliqua </td><td> </td><td>2009/2010 </td></tr><tr><td>Seed weight </td><td>SW </td><td>Seed weight of 100 seeds each line </td><td>mg </td><td>2009/2010 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>* Represent the vernalization treatment.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592135" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5592136 id="5592136" class="section-title" data-legacy-id=s2b>2.2. Statistical analysis and QTL mapping</h3>
<p>The SAS 9.0 program (SAS institute, Inc., Cary, NC, USA) was used for correlation coefficient analysis. The previously described genetic map<sup><a reveal-id="DSS029C18" data-open="DSS029C18" class="link link-ref link-reveal xref-bibr">18</a></sup> was used for QTL mapping. WinQTL cartographer version 2.5<sup><a reveal-id="DSS029C19" data-open="DSS029C19" class="link link-ref link-reveal xref-bibr">19</a></sup> was used to perform QTL analysis using the composite interval mapping function as described previously.<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Tests for the presence of QTL were performed at 2-cM intervals using a 10-cM window. Significant QTL-defining logarithm of odds (LOD) values were calculated by 1000 permutations for phenotypic traits derived from each year.</p>                            <h3 scrollto-destination=5592138 id="5592138" class="section-title" data-legacy-id=s2c>2.3. Comparative map alignment between the A, B, and C subgenomes</h3>
<p>To identify the functionally conserved loci between <em>B. rapa</em> with <em>B. juncea</em> and <em>B. napus</em> for morphological and yield component traits, comparative alignment of QTL maps was performed between these three species. <em>B. juncea</em> contains the A and B subgenomes while <em>B. napus</em> contains the A and C subgenomes. For comparison, the <em>B. juncea</em> QTL map of Ramchiary et al.<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and the <em>B. napus</em> QTL maps of Quijada et al.<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and Udall et al.<sup><a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup> were used. The marker sequences from the respective maps<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup> were downloaded from the National Center for Biotechnology Information and aligned with the <em>A. thaliana</em> genome using BLAST analysis, and crucifer building blocks<sup><a reveal-id="DSS029C14" data-open="DSS029C14" class="link link-ref link-reveal xref-bibr">14</a></sup> containing important trait QTL were defined. The updated map of <em>B. juncea</em> based on IP markers<sup><a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> was used to redraw the QTL map of Ramchiary et al.<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> For QTL map alignment, the homoeologous A, B, and C subgenome groups of <em>Brassica</em> species defined by Panjabi et al.<sup><a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> were used.</p>                            <h3 scrollto-destination=5592140 id="5592140" class="section-title" data-legacy-id=s2d>2.4. Whole genome resequencing, identification of SNPs in candidate genes, and semi-quantitative RT-PCR analysis</h3>
<p>Whole genome resequencing of parental line RCBr was performed using an Illumina GAII next-generation sequencer. Sequence assembly and single nucleotide polymorphism (SNP) identification were performed in a stepwise manner: (i) the scaffold sequences from each linkage group (LG) of the <em>B. rapa</em> Chiifu 401–42 cultivar genome (<a class="link link-uri" href="http://brassicadb.org/brad/">http://brassicadb.org/brad/</a>) were used as reference sequence, (ii) Augustus program<sup><a reveal-id="DSS029C23" data-open="DSS029C23" class="link link-ref link-reveal xref-bibr">23</a></sup> was used to predict candidate genes in QTL regions, (iii) short-read sequences generated from RCBr were aligned to the reference <em>B. rapa</em> genome using Bowtie (<a class="link link-uri" href="http://bowtie.cbcb.umd.edu)">http://bowtie.cbcb.umd.edu)</a><sup><a reveal-id="DSS029C24" data-open="DSS029C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and mapped onto each reference LG, (iv) alignments from Bowtie were transformed to MAQ<sup><a reveal-id="DSS029C25" data-open="DSS029C25" class="link link-ref link-reveal xref-bibr">25</a></sup> to produce consensus short-read sequences and identify SNPs.</p><p>Because we previously demarcated 24 crucifer building blocks in the <em>B. rapa</em> genome by comparing it with <em>A. thaliana</em>,<sup><a reveal-id="DSS029C18" data-open="DSS029C18" class="link link-ref link-reveal xref-bibr">18</a></sup> we selected already characterized candidate genes from <em>A. thaliana</em> and looked for orthologous genes in the <em>B. rapa</em> genome in those blocks harbouring important trait QTL using a homology search approach and synteny analysis. SNPs were identified between the start and stop codons through comparative alignment between the sequences of the parental lines. Gene-specific primer pairs were designed for SNP validation, with an amplicon size of not more than 300 bp. DNA extraction, PCR using 1 × LC Green Plus (Idaho Technologies), and mapping functions were performed as described previously.<sup><a reveal-id="DSS029C18" data-open="DSS029C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The PCR conditions were 4 min at 94°C, 45 cycles of 20 s at 94°C, 20 s at 55–60°C, and 20 s at 72°C, with a final extension step of 7 min at 72°C. For the detection of SNPs between the parental lines and the F<sub>2</sub> mapping population, we used a Light Scanner System (Idaho Technologies), as described previously.<sup><a reveal-id="DSS029C26" data-open="DSS029C26" class="link link-ref link-reveal xref-bibr">26</a></sup> RNA was extracted from 20-day-old leaf samples, and semi-quantitative RT-PCR analysis was performed following Lee et al.<sup><a reveal-id="DSS029C27" data-open="DSS029C27" class="link link-ref link-reveal xref-bibr">27</a></sup> using gene-specific primer pairs listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>.</p>                            <h2 scrollto-destination=5592143 id="5592143" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5592144 id="5592144" class="section-title" data-legacy-id=s3a>3.1. Variation of phenotypic traits in the parental lines and segregating populations</h3>
<p>The parental line Chiifu is a heading-type Chinese cabbage, whereas RCBr is a short life cycle <em>Brassica</em> plant, also known as the Wisconsin fast plant. The two parental lines showed significant differences for flowering, leaf, siliqua, and seed traits (Table <a reveal-id="DSS029TB2" data-open="DSS029TB2" class="link link-reveal link-table xref-fig">2</a>). Chiifu showed higher phenotypic values for all the traits studied. The CRF<sub>3</sub> population showed transgressive segregation for some of the traits, e.g. plant height in 2008, midrib length and width, siliqua length and width, siliqua beak length, seeds per siliqua, and seed weight. For flowering and bolting time, the transgression phenomenon trend was towards RCBr because Chiifu did not flower until seed harvesting time. As the bolting and flowering time were influenced by temperature and photoperiod, the mean value of flowering time measured in spring (2009 and 2010) was ∼30 days shorter than in winter (2008), even when they were grown in a heated greenhouse. The mean values of the F<sub>3</sub> lines for the leaf traits and plant height evaluated in spring 2009 and 2010 were also lower than those of the F<sub>3</sub> lines grown in 2008. Distribution analysis of the phenotypic values of 14 traits in the mapping population showed a normal distribution for all the traits, suggesting that each trait was governed by many genes. Pearson's correlation coefficient analysis showed moderate to strong positive correlation among leaf traits (LL, LW, MRL, and MRW, correlation coefficient <em>r</em> = 0.53–0.92). Siliqua length, width, and beak length were significantly positively correlated with each other, but seed weight was not correlated with other siliqua traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). In addition, we found a positive correlation between bolting time, flowering time, and four leaf traits (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). </p>                        <a id="5592146" scrollto-destination="5592146"></a>
<div content-id="DSS029TB2" class="table-modal"><div class="table-wrap"><div id="DSS029TB2" class="table-wrap-title" data-id="DSS029TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Phenotypic trait values of parental lines and the CRF<sub>3</sub> mapping population of <em>B. rapa</em> observed in different years</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Traits </th><th colspan="3">Mean trait value of parent Chiifu<hr /> </th><th colspan="3">Mean trait value of parent RCBr<hr /> </th><th colspan="3">Range in F<sub>3</sub> population<hr /> </th><th colspan="3">Mean value in F<sub>3</sub> population<hr /> </th></tr><tr><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th></tr></thead><tbody><tr><td>Flowering time </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>39.0 </td><td>45.0 </td><td>45.0 </td><td>33.0–139.0 </td><td>30.0–59.0 </td><td>45.2–64.0 </td><td>70.6 </td><td>41.9 </td><td>53.4 </td></tr><tr><td>Bolting time </td><td>– </td><td>90.0 </td><td>95.0 </td><td>– </td><td>29.0 </td><td>29.0 </td><td>– </td><td>11.0–44.7 </td><td>34.0–65.0 </td><td>– </td><td>26.5 </td><td>42.3 </td></tr><tr><td>Flowering time after vernalization </td><td>– </td><td>135.0 </td><td>145.0 </td><td>– </td><td>30.0 </td><td>30.0 </td><td>– </td><td>54.2–68.8 </td><td>66.0–83.5 </td><td>– </td><td>57.9 </td><td>73.0 </td></tr><tr><td>Plant height </td><td>65.0 </td><td>100.0 </td><td>98.0 </td><td>17.0 </td><td>20.0 </td><td>18.5 </td><td>5.0–98.0 </td><td>3.6–34.5 </td><td>4.0–24.8 </td><td>43.0 </td><td>13.6 </td><td>13.1 </td></tr><tr><td>Leaf length </td><td>28.0 </td><td>20.0 </td><td>– </td><td>9.0 </td><td>6.5 </td><td>– </td><td>14.0–49.0 </td><td>2.8–8.0 </td><td>– </td><td>30.2 </td><td>5.0 </td><td>– </td></tr><tr><td>Leaf width </td><td>14.0 </td><td>16.0 </td><td>– </td><td>3.8 </td><td>3.0 </td><td>– </td><td>4.5–19.0 </td><td>2.2–5.8 </td><td>– </td><td>11.2 </td><td>3.5 </td><td>– </td></tr><tr><td>Midrib length </td><td>14.0 </td><td>7.7 </td><td>– </td><td>7.4 </td><td>4.5 </td><td>– </td><td>9.0–36.0 </td><td>3.3–10.0 </td><td>– </td><td>20.8 </td><td>5.8 </td><td>– </td></tr><tr><td>Midrib width </td><td>3.0 </td><td>2.5 </td><td>– </td><td>0.5 </td><td>0.3 </td><td>– </td><td>0.5–3.5 </td><td>0.3–1.5 </td><td>– </td><td>1.7 </td><td>0.6 </td><td>– </td></tr><tr><td>Petiole length </td><td>– </td><td>0.0 </td><td>– </td><td>– </td><td>6.5 </td><td>– </td><td>– </td><td>0.0–6.3 </td><td>– </td><td>– </td><td>3.3 </td><td>– </td></tr><tr><td>Siliqua length </td><td>3.0 </td><td>3.8 </td><td>3.5 </td><td>2.8 </td><td>3.2 </td><td>3.0 </td><td>1.5–6.2 </td><td>2.0–6.8 </td><td>2.2–6.7 </td><td>3.7 </td><td>4.2 </td><td>4.1 </td></tr><tr><td>Siliqua width </td><td>0.5 </td><td>0.5 </td><td>0.6 </td><td>0.2 </td><td>0.3 </td><td>0.2 </td><td>0.2–0.7 </td><td>0.3–0.8 </td><td>0.2–0.8 </td><td>0.5 </td><td>0.5 </td><td>0.4 </td></tr><tr><td>Siliqua beak length </td><td>– </td><td>1.0 </td><td>1.0 </td><td>– </td><td>0.7 </td><td>0.5 </td><td>– </td><td>0.3–1.6 </td><td>0.2–1.5 </td><td>– </td><td>0.8 </td><td>0.7 </td></tr><tr><td>Seeds per siliqua </td><td>– </td><td>20.0 </td><td>25.0 </td><td>– </td><td>11.0 </td><td>15.0 </td><td>– </td><td>4.1–27.1 </td><td>2.0–24.9 </td><td>– </td><td>14.3 </td><td>12.6 </td></tr><tr><td>Seed weight </td><td>– </td><td>320.0 </td><td>345.0 </td><td>– </td><td>105.0 </td><td>20.0 </td><td>– </td><td>115.0–337.0 </td><td>30.0–257.0 </td><td>– </td><td>211.0 </td><td>141.4 </td></tr></tbody></table></div><div content-id="DSS029TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Traits </th><th colspan="3">Mean trait value of parent Chiifu<hr /> </th><th colspan="3">Mean trait value of parent RCBr<hr /> </th><th colspan="3">Range in F<sub>3</sub> population<hr /> </th><th colspan="3">Mean value in F<sub>3</sub> population<hr /> </th></tr><tr><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th></tr></thead><tbody><tr><td>Flowering time </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>39.0 </td><td>45.0 </td><td>45.0 </td><td>33.0–139.0 </td><td>30.0–59.0 </td><td>45.2–64.0 </td><td>70.6 </td><td>41.9 </td><td>53.4 </td></tr><tr><td>Bolting time </td><td>– </td><td>90.0 </td><td>95.0 </td><td>– </td><td>29.0 </td><td>29.0 </td><td>– </td><td>11.0–44.7 </td><td>34.0–65.0 </td><td>– </td><td>26.5 </td><td>42.3 </td></tr><tr><td>Flowering time after vernalization </td><td>– </td><td>135.0 </td><td>145.0 </td><td>– </td><td>30.0 </td><td>30.0 </td><td>– </td><td>54.2–68.8 </td><td>66.0–83.5 </td><td>– </td><td>57.9 </td><td>73.0 </td></tr><tr><td>Plant height </td><td>65.0 </td><td>100.0 </td><td>98.0 </td><td>17.0 </td><td>20.0 </td><td>18.5 </td><td>5.0–98.0 </td><td>3.6–34.5 </td><td>4.0–24.8 </td><td>43.0 </td><td>13.6 </td><td>13.1 </td></tr><tr><td>Leaf length </td><td>28.0 </td><td>20.0 </td><td>– </td><td>9.0 </td><td>6.5 </td><td>– </td><td>14.0–49.0 </td><td>2.8–8.0 </td><td>– </td><td>30.2 </td><td>5.0 </td><td>– </td></tr><tr><td>Leaf width </td><td>14.0 </td><td>16.0 </td><td>– </td><td>3.8 </td><td>3.0 </td><td>– </td><td>4.5–19.0 </td><td>2.2–5.8 </td><td>– </td><td>11.2 </td><td>3.5 </td><td>– </td></tr><tr><td>Midrib length </td><td>14.0 </td><td>7.7 </td><td>– </td><td>7.4 </td><td>4.5 </td><td>– </td><td>9.0–36.0 </td><td>3.3–10.0 </td><td>– </td><td>20.8 </td><td>5.8 </td><td>– </td></tr><tr><td>Midrib width </td><td>3.0 </td><td>2.5 </td><td>– </td><td>0.5 </td><td>0.3 </td><td>– </td><td>0.5–3.5 </td><td>0.3–1.5 </td><td>– </td><td>1.7 </td><td>0.6 </td><td>– </td></tr><tr><td>Petiole length </td><td>– </td><td>0.0 </td><td>– </td><td>– </td><td>6.5 </td><td>– </td><td>– </td><td>0.0–6.3 </td><td>– </td><td>– </td><td>3.3 </td><td>– </td></tr><tr><td>Siliqua length </td><td>3.0 </td><td>3.8 </td><td>3.5 </td><td>2.8 </td><td>3.2 </td><td>3.0 </td><td>1.5–6.2 </td><td>2.0–6.8 </td><td>2.2–6.7 </td><td>3.7 </td><td>4.2 </td><td>4.1 </td></tr><tr><td>Siliqua width </td><td>0.5 </td><td>0.5 </td><td>0.6 </td><td>0.2 </td><td>0.3 </td><td>0.2 </td><td>0.2–0.7 </td><td>0.3–0.8 </td><td>0.2–0.8 </td><td>0.5 </td><td>0.5 </td><td>0.4 </td></tr><tr><td>Siliqua beak length </td><td>– </td><td>1.0 </td><td>1.0 </td><td>– </td><td>0.7 </td><td>0.5 </td><td>– </td><td>0.3–1.6 </td><td>0.2–1.5 </td><td>– </td><td>0.8 </td><td>0.7 </td></tr><tr><td>Seeds per siliqua </td><td>– </td><td>20.0 </td><td>25.0 </td><td>– </td><td>11.0 </td><td>15.0 </td><td>– </td><td>4.1–27.1 </td><td>2.0–24.9 </td><td>– </td><td>14.3 </td><td>12.6 </td></tr><tr><td>Seed weight </td><td>– </td><td>320.0 </td><td>345.0 </td><td>– </td><td>105.0 </td><td>20.0 </td><td>– </td><td>115.0–337.0 </td><td>30.0–257.0 </td><td>– </td><td>211.0 </td><td>141.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>‘Chiifu’ did not flower until populations seed harvest.</p></div></div><div content-id=""><div class="fn"><p>‘–’, Not measured in the corresponding year.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592146" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS029TB2" class="table-wrap-title" data-id="DSS029TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Phenotypic trait values of parental lines and the CRF<sub>3</sub> mapping population of <em>B. rapa</em> observed in different years</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Traits </th><th colspan="3">Mean trait value of parent Chiifu<hr /> </th><th colspan="3">Mean trait value of parent RCBr<hr /> </th><th colspan="3">Range in F<sub>3</sub> population<hr /> </th><th colspan="3">Mean value in F<sub>3</sub> population<hr /> </th></tr><tr><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th></tr></thead><tbody><tr><td>Flowering time </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>39.0 </td><td>45.0 </td><td>45.0 </td><td>33.0–139.0 </td><td>30.0–59.0 </td><td>45.2–64.0 </td><td>70.6 </td><td>41.9 </td><td>53.4 </td></tr><tr><td>Bolting time </td><td>– </td><td>90.0 </td><td>95.0 </td><td>– </td><td>29.0 </td><td>29.0 </td><td>– </td><td>11.0–44.7 </td><td>34.0–65.0 </td><td>– </td><td>26.5 </td><td>42.3 </td></tr><tr><td>Flowering time after vernalization </td><td>– </td><td>135.0 </td><td>145.0 </td><td>– </td><td>30.0 </td><td>30.0 </td><td>– </td><td>54.2–68.8 </td><td>66.0–83.5 </td><td>– </td><td>57.9 </td><td>73.0 </td></tr><tr><td>Plant height </td><td>65.0 </td><td>100.0 </td><td>98.0 </td><td>17.0 </td><td>20.0 </td><td>18.5 </td><td>5.0–98.0 </td><td>3.6–34.5 </td><td>4.0–24.8 </td><td>43.0 </td><td>13.6 </td><td>13.1 </td></tr><tr><td>Leaf length </td><td>28.0 </td><td>20.0 </td><td>– </td><td>9.0 </td><td>6.5 </td><td>– </td><td>14.0–49.0 </td><td>2.8–8.0 </td><td>– </td><td>30.2 </td><td>5.0 </td><td>– </td></tr><tr><td>Leaf width </td><td>14.0 </td><td>16.0 </td><td>– </td><td>3.8 </td><td>3.0 </td><td>– </td><td>4.5–19.0 </td><td>2.2–5.8 </td><td>– </td><td>11.2 </td><td>3.5 </td><td>– </td></tr><tr><td>Midrib length </td><td>14.0 </td><td>7.7 </td><td>– </td><td>7.4 </td><td>4.5 </td><td>– </td><td>9.0–36.0 </td><td>3.3–10.0 </td><td>– </td><td>20.8 </td><td>5.8 </td><td>– </td></tr><tr><td>Midrib width </td><td>3.0 </td><td>2.5 </td><td>– </td><td>0.5 </td><td>0.3 </td><td>– </td><td>0.5–3.5 </td><td>0.3–1.5 </td><td>– </td><td>1.7 </td><td>0.6 </td><td>– </td></tr><tr><td>Petiole length </td><td>– </td><td>0.0 </td><td>– </td><td>– </td><td>6.5 </td><td>– </td><td>– </td><td>0.0–6.3 </td><td>– </td><td>– </td><td>3.3 </td><td>– </td></tr><tr><td>Siliqua length </td><td>3.0 </td><td>3.8 </td><td>3.5 </td><td>2.8 </td><td>3.2 </td><td>3.0 </td><td>1.5–6.2 </td><td>2.0–6.8 </td><td>2.2–6.7 </td><td>3.7 </td><td>4.2 </td><td>4.1 </td></tr><tr><td>Siliqua width </td><td>0.5 </td><td>0.5 </td><td>0.6 </td><td>0.2 </td><td>0.3 </td><td>0.2 </td><td>0.2–0.7 </td><td>0.3–0.8 </td><td>0.2–0.8 </td><td>0.5 </td><td>0.5 </td><td>0.4 </td></tr><tr><td>Siliqua beak length </td><td>– </td><td>1.0 </td><td>1.0 </td><td>– </td><td>0.7 </td><td>0.5 </td><td>– </td><td>0.3–1.6 </td><td>0.2–1.5 </td><td>– </td><td>0.8 </td><td>0.7 </td></tr><tr><td>Seeds per siliqua </td><td>– </td><td>20.0 </td><td>25.0 </td><td>– </td><td>11.0 </td><td>15.0 </td><td>– </td><td>4.1–27.1 </td><td>2.0–24.9 </td><td>– </td><td>14.3 </td><td>12.6 </td></tr><tr><td>Seed weight </td><td>– </td><td>320.0 </td><td>345.0 </td><td>– </td><td>105.0 </td><td>20.0 </td><td>– </td><td>115.0–337.0 </td><td>30.0–257.0 </td><td>– </td><td>211.0 </td><td>141.4 </td></tr></tbody></table></div><div content-id="DSS029TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Traits </th><th colspan="3">Mean trait value of parent Chiifu<hr /> </th><th colspan="3">Mean trait value of parent RCBr<hr /> </th><th colspan="3">Range in F<sub>3</sub> population<hr /> </th><th colspan="3">Mean value in F<sub>3</sub> population<hr /> </th></tr><tr><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th><th>2008 </th><th>2009 </th><th>2010 </th></tr></thead><tbody><tr><td>Flowering time </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>–<sup>a</sup> </td><td>39.0 </td><td>45.0 </td><td>45.0 </td><td>33.0–139.0 </td><td>30.0–59.0 </td><td>45.2–64.0 </td><td>70.6 </td><td>41.9 </td><td>53.4 </td></tr><tr><td>Bolting time </td><td>– </td><td>90.0 </td><td>95.0 </td><td>– </td><td>29.0 </td><td>29.0 </td><td>– </td><td>11.0–44.7 </td><td>34.0–65.0 </td><td>– </td><td>26.5 </td><td>42.3 </td></tr><tr><td>Flowering time after vernalization </td><td>– </td><td>135.0 </td><td>145.0 </td><td>– </td><td>30.0 </td><td>30.0 </td><td>– </td><td>54.2–68.8 </td><td>66.0–83.5 </td><td>– </td><td>57.9 </td><td>73.0 </td></tr><tr><td>Plant height </td><td>65.0 </td><td>100.0 </td><td>98.0 </td><td>17.0 </td><td>20.0 </td><td>18.5 </td><td>5.0–98.0 </td><td>3.6–34.5 </td><td>4.0–24.8 </td><td>43.0 </td><td>13.6 </td><td>13.1 </td></tr><tr><td>Leaf length </td><td>28.0 </td><td>20.0 </td><td>– </td><td>9.0 </td><td>6.5 </td><td>– </td><td>14.0–49.0 </td><td>2.8–8.0 </td><td>– </td><td>30.2 </td><td>5.0 </td><td>– </td></tr><tr><td>Leaf width </td><td>14.0 </td><td>16.0 </td><td>– </td><td>3.8 </td><td>3.0 </td><td>– </td><td>4.5–19.0 </td><td>2.2–5.8 </td><td>– </td><td>11.2 </td><td>3.5 </td><td>– </td></tr><tr><td>Midrib length </td><td>14.0 </td><td>7.7 </td><td>– </td><td>7.4 </td><td>4.5 </td><td>– </td><td>9.0–36.0 </td><td>3.3–10.0 </td><td>– </td><td>20.8 </td><td>5.8 </td><td>– </td></tr><tr><td>Midrib width </td><td>3.0 </td><td>2.5 </td><td>– </td><td>0.5 </td><td>0.3 </td><td>– </td><td>0.5–3.5 </td><td>0.3–1.5 </td><td>– </td><td>1.7 </td><td>0.6 </td><td>– </td></tr><tr><td>Petiole length </td><td>– </td><td>0.0 </td><td>– </td><td>– </td><td>6.5 </td><td>– </td><td>– </td><td>0.0–6.3 </td><td>– </td><td>– </td><td>3.3 </td><td>– </td></tr><tr><td>Siliqua length </td><td>3.0 </td><td>3.8 </td><td>3.5 </td><td>2.8 </td><td>3.2 </td><td>3.0 </td><td>1.5–6.2 </td><td>2.0–6.8 </td><td>2.2–6.7 </td><td>3.7 </td><td>4.2 </td><td>4.1 </td></tr><tr><td>Siliqua width </td><td>0.5 </td><td>0.5 </td><td>0.6 </td><td>0.2 </td><td>0.3 </td><td>0.2 </td><td>0.2–0.7 </td><td>0.3–0.8 </td><td>0.2–0.8 </td><td>0.5 </td><td>0.5 </td><td>0.4 </td></tr><tr><td>Siliqua beak length </td><td>– </td><td>1.0 </td><td>1.0 </td><td>– </td><td>0.7 </td><td>0.5 </td><td>– </td><td>0.3–1.6 </td><td>0.2–1.5 </td><td>– </td><td>0.8 </td><td>0.7 </td></tr><tr><td>Seeds per siliqua </td><td>– </td><td>20.0 </td><td>25.0 </td><td>– </td><td>11.0 </td><td>15.0 </td><td>– </td><td>4.1–27.1 </td><td>2.0–24.9 </td><td>– </td><td>14.3 </td><td>12.6 </td></tr><tr><td>Seed weight </td><td>– </td><td>320.0 </td><td>345.0 </td><td>– </td><td>105.0 </td><td>20.0 </td><td>– </td><td>115.0–337.0 </td><td>30.0–257.0 </td><td>– </td><td>211.0 </td><td>141.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>‘Chiifu’ did not flower until populations seed harvest.</p></div></div><div content-id=""><div class="fn"><p>‘–’, Not measured in the corresponding year.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592146" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5592147 id="5592147" class="section-title" data-legacy-id=s3b>3.2. QTL mapping and identification of crucifer building blocks in the B. rapa genome</h3>
<p>QTL mapping identified a total of 95 QTL for 14 traits in the CRF<sub>3</sub> mapping population. It was observed that almost all the trait-enhancing alleles, except for flowering time, were contributed by the Chiifu parental line. The number of QTL detected ranged from 4 QTL for petiole length, seed number per siliqua and seed weight to 14 QTL for flowering traits, and the confidence interval ranged from 1 to 25 cM (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). </p>                        <a id="5592149" scrollto-destination="5592149"></a>
<div id="DSS029F1" data-id="DSS029F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02901a.jpeg?Expires=2147483647&amp;Signature=4hOiUkHIG~2sU0Jywv2I6zxV3hbhhRmClNJW5A0hFKSO4RlR-DvcwcTLJWJnz9zfAkZGAvDpVBl5tG90dTOBRYxtYafEU6tPXXnwHRvKp5xxDj5DDXYWXH4erif~OcRTg8TPJM1tzwrrzhebhEO1vl~pO2qfPT7rO35N5uyBC132-gymf6tJ0~z8SkfLL3vVkvv6yQGpCasGMPBZvjDmqIziQrKhLiRP~JQSQfIKlido-s6S1beNg~ivuiiMwwd7ZXHaTF2nlUkMvhRGDMAiQxxgc1Q8Au-jnyFIMgRGy~SEEj-o4fqyUMlDgD-7GAOnpz0snakUV2Hnz3aIdn8FEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of QTL for morphological and yield component traits in B. rapa genome. QTL names are indicated by abbreviations of trait names as shown in Table 1. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of B. rapa, which are homologous to five chromosomes of A. thaliana At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters." path-from-xml="dss02901a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592149/dss02901a.jpeg" path-from-xml="dss02901a.jpeg" target="_blank">View large</a><a section="5592149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02901a.jpeg?Expires=2147483647&amp;Signature=OtEQFyRQaOfOE~lAPNcu3Y1Ly38D9JNWsPtV88b7kM~hvDuggj5R5zeNEOqmU-rdPPk6McJ0ArGW33RN1VwqHFFSq5mTO5reCKFRP2dYSsjemIMAoIgQEGWA9DlCW5WFLwP3NTSuP9XCgemySbGr8fM~wEMs8WsOs352ZCyR543MZ7QjPuuKSkE5tEIJi9dyTCXBPza9QOYzpSsD-fiH3MNAKCBRIH9QyQsD1lWdGY4BLrQCY-4Bfypaq5Wg6O3Dydq-6Tc0ltm3MVwYCBJxHJGyFf-n8MDqOLm5O7u0d-OdYsCq7GBJlu1LwebgjZ4Vql3h7gSVcy6T-O5vozzl9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5592149&amp;ar=381937&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss02901a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02901b.jpeg?Expires=2147483647&amp;Signature=0SXBCYECTXe5P~FFV8i7J022BRIak1qjkrop3KCY4YfTpzUCCjJ2YlY~-iGer6BQg8T2IooJAosdvNmR7oLE9kPUKMo5hMLDmzt3adUz8n4VkyYdwtvJ8vqJkbxcfTnZwvqt5MfZhjjRdqp4jh4hGA8k6n8HKVE8TB0J-~gt-~ptwuAvQMBMmQA~UaOOlBEEBKppsjvEHV5JaC3pKKECtHS22JwqiEO~wv1dHkZYiTnP481XokmsR8~5-uwQESYvsT4fyEEp~K09hoZe-V8PExZ4DYSdSLS2bvea~88KEMX7ou4jDrqYceEiHmjQC4aQ2iq3ysutA-UhS8HA45~XzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of QTL for morphological and yield component traits in B. rapa genome. QTL names are indicated by abbreviations of trait names as shown in Table 1. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of B. rapa, which are homologous to five chromosomes of A. thaliana At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters." path-from-xml="dss02901b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592149/dss02901b.jpeg" path-from-xml="dss02901b.jpeg" target="_blank">View large</a><a section="5592149" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02901b.jpeg?Expires=2147483647&amp;Signature=BzPp4oJ7sKEuzxZLV73euQcz35eEOOHR7qvMNBR4MMofVzS~2j1zcHk2exp11ib-AuzctYeyGyMIyg7ZEkBxAao8-004HHWGBbQExEBSaqel6N0FRUGP59OXSjsHcJPtH8CyMq-KNa8SojinADdVe2Jhc684sjwIeXj7zEiWAwOjpGBgvRn6anBfrTUIEkQDTReodYTf0gQvL2pgDpaf2TbW5JJrGOIxD3Qu53nyUiTC6J6HpUNVXBFADU6K8UGn135oOoZVeCwLWUnjy1o-a~ueMYqQjAidwcpjv61nNyXGGWFQrTZY0R4QvLXpz5k-H9UT-6gpcAylF4Elt2wBcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5592149&amp;ar=381937&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss02901b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of QTL for morphological and yield component traits in <em>B. rapa</em> genome. QTL names are indicated by abbreviations of trait names as shown in <a reveal-id="DSS029TB1" data-open="DSS029TB1" class="link link-reveal link-table xref-fig">Table 1</a>. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of <em>B. rapa</em>, which are homologous to five chromosomes of <em>A. thaliana</em> At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters.</p></div></div><div content-id="DSS029F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02901a.jpeg?Expires=2147483647&amp;Signature=4hOiUkHIG~2sU0Jywv2I6zxV3hbhhRmClNJW5A0hFKSO4RlR-DvcwcTLJWJnz9zfAkZGAvDpVBl5tG90dTOBRYxtYafEU6tPXXnwHRvKp5xxDj5DDXYWXH4erif~OcRTg8TPJM1tzwrrzhebhEO1vl~pO2qfPT7rO35N5uyBC132-gymf6tJ0~z8SkfLL3vVkvv6yQGpCasGMPBZvjDmqIziQrKhLiRP~JQSQfIKlido-s6S1beNg~ivuiiMwwd7ZXHaTF2nlUkMvhRGDMAiQxxgc1Q8Au-jnyFIMgRGy~SEEj-o4fqyUMlDgD-7GAOnpz0snakUV2Hnz3aIdn8FEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of QTL for morphological and yield component traits in B. rapa genome. QTL names are indicated by abbreviations of trait names as shown in Table 1. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of B. rapa, which are homologous to five chromosomes of A. thaliana At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters." path-from-xml="dss02901a.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592149/dss02901a.jpeg" path-from-xml="dss02901a.jpeg" target="_blank">View large</a><a section="5592149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02901a.jpeg?Expires=2147483647&amp;Signature=OtEQFyRQaOfOE~lAPNcu3Y1Ly38D9JNWsPtV88b7kM~hvDuggj5R5zeNEOqmU-rdPPk6McJ0ArGW33RN1VwqHFFSq5mTO5reCKFRP2dYSsjemIMAoIgQEGWA9DlCW5WFLwP3NTSuP9XCgemySbGr8fM~wEMs8WsOs352ZCyR543MZ7QjPuuKSkE5tEIJi9dyTCXBPza9QOYzpSsD-fiH3MNAKCBRIH9QyQsD1lWdGY4BLrQCY-4Bfypaq5Wg6O3Dydq-6Tc0ltm3MVwYCBJxHJGyFf-n8MDqOLm5O7u0d-OdYsCq7GBJlu1LwebgjZ4Vql3h7gSVcy6T-O5vozzl9A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5592149&amp;ar=381937&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss02901a.jpeg" target="_blank">Download slide</a></div></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02901b.jpeg?Expires=2147483647&amp;Signature=0SXBCYECTXe5P~FFV8i7J022BRIak1qjkrop3KCY4YfTpzUCCjJ2YlY~-iGer6BQg8T2IooJAosdvNmR7oLE9kPUKMo5hMLDmzt3adUz8n4VkyYdwtvJ8vqJkbxcfTnZwvqt5MfZhjjRdqp4jh4hGA8k6n8HKVE8TB0J-~gt-~ptwuAvQMBMmQA~UaOOlBEEBKppsjvEHV5JaC3pKKECtHS22JwqiEO~wv1dHkZYiTnP481XokmsR8~5-uwQESYvsT4fyEEp~K09hoZe-V8PExZ4DYSdSLS2bvea~88KEMX7ou4jDrqYceEiHmjQC4aQ2iq3ysutA-UhS8HA45~XzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of QTL for morphological and yield component traits in B. rapa genome. QTL names are indicated by abbreviations of trait names as shown in Table 1. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of B. rapa, which are homologous to five chromosomes of A. thaliana At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters." path-from-xml="dss02901b.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592149/dss02901b.jpeg" path-from-xml="dss02901b.jpeg" target="_blank">View large</a><a section="5592149" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02901b.jpeg?Expires=2147483647&amp;Signature=BzPp4oJ7sKEuzxZLV73euQcz35eEOOHR7qvMNBR4MMofVzS~2j1zcHk2exp11ib-AuzctYeyGyMIyg7ZEkBxAao8-004HHWGBbQExEBSaqel6N0FRUGP59OXSjsHcJPtH8CyMq-KNa8SojinADdVe2Jhc684sjwIeXj7zEiWAwOjpGBgvRn6anBfrTUIEkQDTReodYTf0gQvL2pgDpaf2TbW5JJrGOIxD3Qu53nyUiTC6J6HpUNVXBFADU6K8UGn135oOoZVeCwLWUnjy1o-a~ueMYqQjAidwcpjv61nNyXGGWFQrTZY0R4QvLXpz5k-H9UT-6gpcAylF4Elt2wBcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=5592149&amp;ar=381937&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dss02901b.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of QTL for morphological and yield component traits in <em>B. rapa</em> genome. QTL names are indicated by abbreviations of trait names as shown in <a reveal-id="DSS029TB1" data-open="DSS029TB1" class="link link-reveal link-table xref-fig">Table 1</a>. The numbers in parenthesis indicate the year of QTL detection. The crucifer building blocks in each LGs of <em>B. rapa</em>, which are homologous to five chromosomes of <em>A. thaliana</em> At C1-At C5, are indicated by different colours. Putative candidate genes identified within the QTL blocks are shown in bold black letters on the left of each LG, and those outside QTL intervals are shown in bold grey letters.</p></div></div>                            <h4 scrollto-destination=5592150 id="5592150" class="section-title" data-legacy-id=s3b1>Plant height</h4>
<p>A total of seven QTL were detected for plant height, one in each LG, i.e. A2 and A3 (R block), A3 and A5 (J block), A7 (E block), A9 (H block), and A10 (A block). The QTL on A2 (<em>qPH</em>1) and A7 (<em>qPH4</em>) showed higher LOD values and phenotypic explanation and were detected both in 2009 and 2010 (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The percentage of phenotypic variation explained by individual QTL ranged from 5 to 19%.</p>                            <h4 scrollto-destination=5592152 id="5592152" class="section-title" data-legacy-id=s3b2>Leaf traits</h4>
<p>Five leaf traits, i.e. leaf length and width, midrib length and width, and petiole length, were used for QTL analysis (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). A total of six QTL for leaf length were detected in four LGs. <em>qLL1</em> and <em>qLL5</em> in the R blocks of A3 and A10, respectively, were consistently detected in two consecutive years (2008 and 2009). The percentage of phenotypic variation explained by individual QTL ranged from 7 to 49%. For leaf width, 10 QTL were detected, of which <em>qLW1</em> in the U block of A1, <em>qLW2</em> in the R block of A2, and <em>qLW3</em> in the J block of A3 were consistently found in 2008 and 2009 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). A total of 10 QTL were detected for midrib length, of which QTL in the J (<em>qMRL3</em>) and T (<em>qMRL4</em>) blocks of A3 and <em>qMRL</em>6 in the A block of A6 showed comparatively higher LOD values, but none of them were detected consecutively in different years. For midrib width, five QTL were detected, of which QTL in A2 (<em>qMRW2</em>) and A10 (<em>qMRW5</em>) were detected in the R blocks of both LGs<em>. qMRW3</em> in T/U block of A3 and <em>qMRW4</em> in E block of A7 were detected both in 2008 and 2009. Four QTL, two on A2 (<em>qPL1</em> in R block and <em>qPL2</em> in W block) and one each on LG A6 (<em>qPL3</em>) and A10 (<em>qPL4</em> in R block) were detected for petiole length. The QTL on <em>qPL3</em> was major QTL showing the highest LOD value of 11.8. The phenotypic variation explained by individual QTL ranged from 6 to 13%.</p>                            <h4 scrollto-destination=5592154 id="5592154" class="section-title" data-legacy-id=s3b3>Bolting and flowering traits</h4>
<p>A total of 10 QTL were detected for bolting time, of which <em>qDB2</em> in the R block of A2, <em>qDB4</em> in the J block of A3, <em>qDB7</em> in the E block of A7, <em>qDB8</em> in the A block of A8, and <em>qDB10</em> in the R block of A10 were detected in 2008 and 2009 (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The QTL on A2, A7, and A10 were major loci, and the percentage of phenotypic variation explained by individual QTL ranged from 4 to 29%.</p><p>For flowering time in the non-vernalized plants, a total of 12 QTL were identified in 8 LGs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> and Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). Of these, QTL mapped to the R (<em>qFT2</em> and <em>qFT3</em>) and W (<em>qFT4</em>) blocks of A2, R block of A3 (<em>qFT5</em>), E block of A7 (<em>qFT10</em>), N block of A9 (<em>qFT13</em>), and R block of A10 (<em>qFT15</em>) were major loci and were detected consecutively for 3 yrs. A total of nine QTL, including two new QTL, i.e. <em>qFT1</em> in the U block of A1 and <em>qFT13</em> in the A block of A10, were detected in vernalized population in addition to the already detected QTL in the non-vernalized population (marked by asterisks, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a> and Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). <em>qFT2</em> and <em>qFT3</em>, both on A2, <em>qFT10</em> on A7, <em>qFT13</em> on A9, and <em>qFT14</em> on A10, were major flowering time QTL detected in the vernalized and non-vernalized populations.</p>                            <h4 scrollto-destination=5592157 id="5592157" class="section-title" data-legacy-id=s3b4>Siliqua traits</h4>
<p>A total of nine QTL were detected for siliqua length, two QTL on A2 (<em>qSQL1</em> in the R/W block and <em>qSQL2</em> in E block), three QTL on A3 (<em>qSQL3</em> in R block, <em>qSQL4</em> in J block, and <em>qSQL5</em> in T/U block), and one each on A7 (<em>qSQL6</em> in the E block), A8 (<em>qSQL7</em> in the B block), and A9 (<em>qSQL8</em> in B block), and A10 (<em>qSQL9</em> in the W block) (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). <em>qSQL3</em>, <em>qSQL5</em>, and <em>qSQL9</em> were detected in two consecutive years while <em>qSQL6</em> on A7 was detected in three consecutive years. The QTL on A3 (<em>qSQL5</em>), A7 (<em>qSQL6</em>), and A10 (<em>qSQL9</em>) were major loci, contributing to phenotypic variation ranging from 7 to 19% in different years.</p><p>Seven QTL, one each on LGs A1 (<em>qSQW1</em> in the F block), A2 (<em>qSQW2</em> in the K Block), A8 (<em>qSQW6</em> in the C block), and A9 (<em>qSQW7</em> in the N block) and three on A3 (<em>qSQW3</em> in the R, <em>qSQW4</em> in the J, and <em>qSQW5</em> in the T/U blocks, respectively), were detected for siliqua width. <em>qSQW1</em> on A1 and <em>qSQW3</em> and <em>qSQW5</em> on A3 were major loci that were detected in two consecutive years and explained 3–16% of the phenotypic variation.</p><p>For siliqua beak length, five QTL were detected, one each on A3 (<em>qSBL1</em> in the J block), A4 (<em>qSBL2</em> in the N Block), A5 (<em>qSBL3</em> in the C block), A7 (<em>qSBL4</em> in the E block), and A9 (<em>qSBL5</em> in H block). The QTL on A3 (<em>qSBL1</em>) and A5 (<em>qSBL3</em>) were comparatively major loci with high LOD values and were detected in two different years (2009 and 2010).</p>                            <h4 scrollto-destination=5592161 id="5592161" class="section-title" data-legacy-id=s3b5>Seed traits</h4>
<p>The seed numbers per siliqua and seed weight are important traits for increasing yield (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). For the number of seeds per siliqua, one QTL each on A2 (<em>qSSQ1</em> in the R/W block) and A10 (<em>qSSQ4</em> in the W block) and two QTL on A3 (<em>qSSQ2</em> in the J block and <em>qSSQ3</em> in the T block) were detected. Only <em>qSSQ4</em> on A10 was detected in two years, although the other three QTL also showed significantly high LOD values. For seed weight, one QTL each was detected on A2 (<em>qSW1</em> in the R block), A3 (<em>qSW2</em> in the J block), A6 (<em>qSW3</em> in the X block), and A10 (<em>qSW4</em> in the R block). Of these, only <em>qSW1</em> and <em>qSW4</em> were detected in more than 1 yr (2009 and 2010). The phenotypic variation explained by individual QTL ranged from 6 to 17%.</p>                            <h3 scrollto-destination=5592163 id="5592163" class="section-title" data-legacy-id=s3c>3.3. Identification of potential candidate genes in the QTL blocks</h3>
<p>As synteny between our <em>B. rapa</em> map and <em>A. thaliana</em> was established,<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup> we searched for orthologs of previously characterized <em>A. thaliana</em> genes in the <em>B. rapa</em> genome (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSS029TB3" data-open="DSS029TB3" class="link link-reveal link-table xref-fig">3</a>). The most important candidate genes, identified in the R blocks of A2, A3, and A10 harbouring the major QTL clusters for flowering and bolting, leaf, and siliqua traits, were orthologous to genes in the gibberellic acid (GA) oxidase 3 (<em>AtGA20OX3</em>),<sup><a reveal-id="DSS029C28" data-open="DSS029C28" class="link link-ref link-reveal xref-bibr">28</a></sup> cytokinin oxidase/dehydrogenase 5 (<em>AtCKX5</em>),<sup><a reveal-id="DSS029C29" data-open="DSS029C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS029C30" data-open="DSS029C30" class="link link-ref link-reveal xref-bibr">30</a></sup> flowering locus C (<em>FLC</em>),<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a></sup> GA-stimulated <em>Arabidopsis</em> (<em>GASA4</em>),<sup><a reveal-id="DSS029C31" data-open="DSS029C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and constans-like (<em>COL</em>) gene families. The <em>Brassica</em> homologs <em>BraFLC</em> and <em>BraGA20OX3</em> were identified in the R blocks of A2, A3, and A10. However, some paralogs were differentially present on <em>Brassica</em> genomes such as <em>BraCKX5</em> on A2 and A10, but not on A3, <em>BraGASA4</em> only on A2 and A3, <em>BraCOL1</em> on A2 and A10, etc. (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). These suggest the preferential retention and fractionation of paralogous genes in duplicated segments of the <em>B. rapa</em> genome. </p>                        <a id="5592165" scrollto-destination="5592165"></a>
<div content-id="DSS029TB3" class="table-modal"><div class="table-wrap"><div id="DSS029TB3" class="table-wrap-title" data-id="DSS029TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Putative candidate genes identified in the QTL blocks of <em>B. rapa</em> genome and SNPs observed between gene sequences of the parental lines, Chiifu and RCBr</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene family </th><th>Gene name </th><th>At gene ID </th><th><em>B. rapa</em> gene ID </th><th>Block (LGs) </th><th>QTL co-mapping </th><th>SNP location (number) </th></tr></thead><tbody><tr><td rowspan="4">Gibberellin biosythesis </td><td>AtGA2OX4 </td><td>AT1G47990 </td><td>Bra032238 </td><td>C (A5) </td><td><em>qSBL3</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">AtGA20OX3 </td><td rowspan="3">AT5G07200 </td><td>Bra028706 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>– </td></tr><tr><td>Bra005927 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Exon (1) </td></tr><tr><td>Bra009285 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>, <em>qPL4</em> </td><td>– </td></tr><tr><td rowspan="9">Cytokinin oxidase/dehydrogenases </td><td>AtCKX5 </td><td>AT1G75450 </td><td>Bra015842 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="2">AtCKX2 </td><td rowspan="2">AT2G19500 </td><td>Bra036719 </td><td>H (A9) </td><td><em>qSBL5</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td>Bra040677 </td><td>H (A7) </td><td><em>qLW6</em>,<em>qDB5</em>,<em>qFT9</em> </td><td>Intron (1) </td></tr><tr><td rowspan="2">AtCKX1 </td><td rowspan="2">AT2G41510 </td><td>Bra000229 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>– </td></tr><tr><td>Bra016928 </td><td>J (A4) </td><td><em>qLL4</em>,<em>qMRL5</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td rowspan="2"><em>AtCKX5</em> (<em>CKX7</em>) </td><td rowspan="2">AT5G21482 </td><td>Bra020157 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (7) </td></tr><tr><td>Bra002371 </td><td>R (A10) </td><td>– </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="2">AtCKX3 </td><td rowspan="2">AT5G56970 </td><td>Bra035640 </td><td>W (A2) </td><td><em>qFT4</em>,<em>qDB3</em>,<em>qPL2</em>,<em>qSQL1</em>,<em>qSSQ1</em> </td><td>Intron (1) </td></tr><tr><td>Bra002777 </td><td>W (A10) </td><td><em>qSQL8</em>,<em>qSSQ4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="5">Gast1 protein homolog </td><td rowspan="2">GASA4 </td><td>AT5G15230 </td><td>Bra023513 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td> </td><td>Bra006291 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Intron (1) </td></tr><tr><td rowspan="3">GASA1 </td><td rowspan="3">AT1G75750 </td><td>Bra008222 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra015820 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Exon (1) </td></tr><tr><td>Bra003743 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron (2) </td></tr><tr><td rowspan="5">Growth-regulating factor </td><td rowspan="2">AtGRF2 </td><td rowspan="2">AT4G37740 </td><td>Bra011781 </td><td>U (A1) </td><td><em>qMRL1</em>,<em>qMRW1</em> </td><td>Exon (3) </td></tr><tr><td>Bra017851 </td><td>U (A3) </td><td>– </td><td>Exon (4) </td></tr><tr><td rowspan="3">AtGRF3 </td><td rowspan="3">AT2G36400 </td><td>Bra023066 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Intron (1) </td></tr><tr><td>Bra017240 </td><td>J (A4) </td><td>– </td><td>Intron (4) </td></tr><tr><td>Bra005268 </td><td>J (A5) </td><td><em>qFT7</em>,<em>qPH3</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td rowspan="2">Apetala1 </td><td rowspan="2">AP1 </td><td rowspan="2">AT1G69120 </td><td>Bra004007 </td><td>E (A7) </td><td><em>qSQL5</em>,<em>qSBL4</em> </td><td>Intron (6) </td></tr><tr><td>Bra038326 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td rowspan="2">Apetala2 </td><td rowspan="2">AP2 </td><td rowspan="2">AT4G36920 </td><td>Bra011741 </td><td>U (A1) </td><td><em>qFT1</em> </td><td>Intron &amp; exon (5) </td></tr><tr><td>Bra017809 </td><td>U (A3) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Asymmetric leaves 1 </td><td rowspan="2">AS1 </td><td rowspan="2">AT2G37630 </td><td>Bra000011 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Exon (1) </td></tr><tr><td>Bra005177 </td><td>J (A5) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Constans like </td><td rowspan="2">COL1 </td><td rowspan="2">AT5G15850 </td><td>Bra023541 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Exon (2) </td></tr><tr><td>Bra008668 </td><td>R (A10) </td><td>– </td><td>Exon (7) </td></tr><tr><td rowspan="2">Flowering locus T </td><td rowspan="2">FT </td><td rowspan="2">AT1G65480 </td><td>Bra022475 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra004117 </td><td>E (A7) </td><td>– </td><td>– </td></tr><tr><td>Leafy </td><td>LFY </td><td>AT5G61850 </td><td>Bra019619 </td><td>X (A6) </td><td><em>qDB5</em> </td><td>– </td></tr><tr><td>GA-insensitive dwarf 1C </td><td>GA </td><td>AT5G27320 </td><td>Bra009970 </td><td>Q (A6) </td><td><em>qFT8</em>,<em>qMRL7</em>,<em>qLW5</em>,<em>qSW3</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td>Mads affecting flowering 3 </td><td>MAF3 </td><td>AT5G65060 </td><td>Bra024350 </td><td>X (A6) </td><td><em>qDB4</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">Flowering locus C </td><td rowspan="3">FLC </td><td rowspan="3">AT5G10140 </td><td>Bra028599 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>Intron (1) </td></tr><tr><td>Bra006051 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>– </td></tr><tr><td>Bra009055 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>,<em>qPL4</em> </td><td>Intron (1) </td></tr></tbody></table></div><div content-id="DSS029TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene family </th><th>Gene name </th><th>At gene ID </th><th><em>B. rapa</em> gene ID </th><th>Block (LGs) </th><th>QTL co-mapping </th><th>SNP location (number) </th></tr></thead><tbody><tr><td rowspan="4">Gibberellin biosythesis </td><td>AtGA2OX4 </td><td>AT1G47990 </td><td>Bra032238 </td><td>C (A5) </td><td><em>qSBL3</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">AtGA20OX3 </td><td rowspan="3">AT5G07200 </td><td>Bra028706 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>– </td></tr><tr><td>Bra005927 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Exon (1) </td></tr><tr><td>Bra009285 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>, <em>qPL4</em> </td><td>– </td></tr><tr><td rowspan="9">Cytokinin oxidase/dehydrogenases </td><td>AtCKX5 </td><td>AT1G75450 </td><td>Bra015842 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="2">AtCKX2 </td><td rowspan="2">AT2G19500 </td><td>Bra036719 </td><td>H (A9) </td><td><em>qSBL5</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td>Bra040677 </td><td>H (A7) </td><td><em>qLW6</em>,<em>qDB5</em>,<em>qFT9</em> </td><td>Intron (1) </td></tr><tr><td rowspan="2">AtCKX1 </td><td rowspan="2">AT2G41510 </td><td>Bra000229 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>– </td></tr><tr><td>Bra016928 </td><td>J (A4) </td><td><em>qLL4</em>,<em>qMRL5</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td rowspan="2"><em>AtCKX5</em> (<em>CKX7</em>) </td><td rowspan="2">AT5G21482 </td><td>Bra020157 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (7) </td></tr><tr><td>Bra002371 </td><td>R (A10) </td><td>– </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="2">AtCKX3 </td><td rowspan="2">AT5G56970 </td><td>Bra035640 </td><td>W (A2) </td><td><em>qFT4</em>,<em>qDB3</em>,<em>qPL2</em>,<em>qSQL1</em>,<em>qSSQ1</em> </td><td>Intron (1) </td></tr><tr><td>Bra002777 </td><td>W (A10) </td><td><em>qSQL8</em>,<em>qSSQ4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="5">Gast1 protein homolog </td><td rowspan="2">GASA4 </td><td>AT5G15230 </td><td>Bra023513 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td> </td><td>Bra006291 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Intron (1) </td></tr><tr><td rowspan="3">GASA1 </td><td rowspan="3">AT1G75750 </td><td>Bra008222 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra015820 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Exon (1) </td></tr><tr><td>Bra003743 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron (2) </td></tr><tr><td rowspan="5">Growth-regulating factor </td><td rowspan="2">AtGRF2 </td><td rowspan="2">AT4G37740 </td><td>Bra011781 </td><td>U (A1) </td><td><em>qMRL1</em>,<em>qMRW1</em> </td><td>Exon (3) </td></tr><tr><td>Bra017851 </td><td>U (A3) </td><td>– </td><td>Exon (4) </td></tr><tr><td rowspan="3">AtGRF3 </td><td rowspan="3">AT2G36400 </td><td>Bra023066 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Intron (1) </td></tr><tr><td>Bra017240 </td><td>J (A4) </td><td>– </td><td>Intron (4) </td></tr><tr><td>Bra005268 </td><td>J (A5) </td><td><em>qFT7</em>,<em>qPH3</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td rowspan="2">Apetala1 </td><td rowspan="2">AP1 </td><td rowspan="2">AT1G69120 </td><td>Bra004007 </td><td>E (A7) </td><td><em>qSQL5</em>,<em>qSBL4</em> </td><td>Intron (6) </td></tr><tr><td>Bra038326 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td rowspan="2">Apetala2 </td><td rowspan="2">AP2 </td><td rowspan="2">AT4G36920 </td><td>Bra011741 </td><td>U (A1) </td><td><em>qFT1</em> </td><td>Intron &amp; exon (5) </td></tr><tr><td>Bra017809 </td><td>U (A3) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Asymmetric leaves 1 </td><td rowspan="2">AS1 </td><td rowspan="2">AT2G37630 </td><td>Bra000011 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Exon (1) </td></tr><tr><td>Bra005177 </td><td>J (A5) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Constans like </td><td rowspan="2">COL1 </td><td rowspan="2">AT5G15850 </td><td>Bra023541 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Exon (2) </td></tr><tr><td>Bra008668 </td><td>R (A10) </td><td>– </td><td>Exon (7) </td></tr><tr><td rowspan="2">Flowering locus T </td><td rowspan="2">FT </td><td rowspan="2">AT1G65480 </td><td>Bra022475 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra004117 </td><td>E (A7) </td><td>– </td><td>– </td></tr><tr><td>Leafy </td><td>LFY </td><td>AT5G61850 </td><td>Bra019619 </td><td>X (A6) </td><td><em>qDB5</em> </td><td>– </td></tr><tr><td>GA-insensitive dwarf 1C </td><td>GA </td><td>AT5G27320 </td><td>Bra009970 </td><td>Q (A6) </td><td><em>qFT8</em>,<em>qMRL7</em>,<em>qLW5</em>,<em>qSW3</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td>Mads affecting flowering 3 </td><td>MAF3 </td><td>AT5G65060 </td><td>Bra024350 </td><td>X (A6) </td><td><em>qDB4</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">Flowering locus C </td><td rowspan="3">FLC </td><td rowspan="3">AT5G10140 </td><td>Bra028599 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>Intron (1) </td></tr><tr><td>Bra006051 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>– </td></tr><tr><td>Bra009055 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>,<em>qPL4</em> </td><td>Intron (1) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592165" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS029TB3" class="table-wrap-title" data-id="DSS029TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Putative candidate genes identified in the QTL blocks of <em>B. rapa</em> genome and SNPs observed between gene sequences of the parental lines, Chiifu and RCBr</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene family </th><th>Gene name </th><th>At gene ID </th><th><em>B. rapa</em> gene ID </th><th>Block (LGs) </th><th>QTL co-mapping </th><th>SNP location (number) </th></tr></thead><tbody><tr><td rowspan="4">Gibberellin biosythesis </td><td>AtGA2OX4 </td><td>AT1G47990 </td><td>Bra032238 </td><td>C (A5) </td><td><em>qSBL3</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">AtGA20OX3 </td><td rowspan="3">AT5G07200 </td><td>Bra028706 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>– </td></tr><tr><td>Bra005927 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Exon (1) </td></tr><tr><td>Bra009285 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>, <em>qPL4</em> </td><td>– </td></tr><tr><td rowspan="9">Cytokinin oxidase/dehydrogenases </td><td>AtCKX5 </td><td>AT1G75450 </td><td>Bra015842 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="2">AtCKX2 </td><td rowspan="2">AT2G19500 </td><td>Bra036719 </td><td>H (A9) </td><td><em>qSBL5</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td>Bra040677 </td><td>H (A7) </td><td><em>qLW6</em>,<em>qDB5</em>,<em>qFT9</em> </td><td>Intron (1) </td></tr><tr><td rowspan="2">AtCKX1 </td><td rowspan="2">AT2G41510 </td><td>Bra000229 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>– </td></tr><tr><td>Bra016928 </td><td>J (A4) </td><td><em>qLL4</em>,<em>qMRL5</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td rowspan="2"><em>AtCKX5</em> (<em>CKX7</em>) </td><td rowspan="2">AT5G21482 </td><td>Bra020157 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (7) </td></tr><tr><td>Bra002371 </td><td>R (A10) </td><td>– </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="2">AtCKX3 </td><td rowspan="2">AT5G56970 </td><td>Bra035640 </td><td>W (A2) </td><td><em>qFT4</em>,<em>qDB3</em>,<em>qPL2</em>,<em>qSQL1</em>,<em>qSSQ1</em> </td><td>Intron (1) </td></tr><tr><td>Bra002777 </td><td>W (A10) </td><td><em>qSQL8</em>,<em>qSSQ4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="5">Gast1 protein homolog </td><td rowspan="2">GASA4 </td><td>AT5G15230 </td><td>Bra023513 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td> </td><td>Bra006291 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Intron (1) </td></tr><tr><td rowspan="3">GASA1 </td><td rowspan="3">AT1G75750 </td><td>Bra008222 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra015820 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Exon (1) </td></tr><tr><td>Bra003743 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron (2) </td></tr><tr><td rowspan="5">Growth-regulating factor </td><td rowspan="2">AtGRF2 </td><td rowspan="2">AT4G37740 </td><td>Bra011781 </td><td>U (A1) </td><td><em>qMRL1</em>,<em>qMRW1</em> </td><td>Exon (3) </td></tr><tr><td>Bra017851 </td><td>U (A3) </td><td>– </td><td>Exon (4) </td></tr><tr><td rowspan="3">AtGRF3 </td><td rowspan="3">AT2G36400 </td><td>Bra023066 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Intron (1) </td></tr><tr><td>Bra017240 </td><td>J (A4) </td><td>– </td><td>Intron (4) </td></tr><tr><td>Bra005268 </td><td>J (A5) </td><td><em>qFT7</em>,<em>qPH3</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td rowspan="2">Apetala1 </td><td rowspan="2">AP1 </td><td rowspan="2">AT1G69120 </td><td>Bra004007 </td><td>E (A7) </td><td><em>qSQL5</em>,<em>qSBL4</em> </td><td>Intron (6) </td></tr><tr><td>Bra038326 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td rowspan="2">Apetala2 </td><td rowspan="2">AP2 </td><td rowspan="2">AT4G36920 </td><td>Bra011741 </td><td>U (A1) </td><td><em>qFT1</em> </td><td>Intron &amp; exon (5) </td></tr><tr><td>Bra017809 </td><td>U (A3) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Asymmetric leaves 1 </td><td rowspan="2">AS1 </td><td rowspan="2">AT2G37630 </td><td>Bra000011 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Exon (1) </td></tr><tr><td>Bra005177 </td><td>J (A5) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Constans like </td><td rowspan="2">COL1 </td><td rowspan="2">AT5G15850 </td><td>Bra023541 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Exon (2) </td></tr><tr><td>Bra008668 </td><td>R (A10) </td><td>– </td><td>Exon (7) </td></tr><tr><td rowspan="2">Flowering locus T </td><td rowspan="2">FT </td><td rowspan="2">AT1G65480 </td><td>Bra022475 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra004117 </td><td>E (A7) </td><td>– </td><td>– </td></tr><tr><td>Leafy </td><td>LFY </td><td>AT5G61850 </td><td>Bra019619 </td><td>X (A6) </td><td><em>qDB5</em> </td><td>– </td></tr><tr><td>GA-insensitive dwarf 1C </td><td>GA </td><td>AT5G27320 </td><td>Bra009970 </td><td>Q (A6) </td><td><em>qFT8</em>,<em>qMRL7</em>,<em>qLW5</em>,<em>qSW3</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td>Mads affecting flowering 3 </td><td>MAF3 </td><td>AT5G65060 </td><td>Bra024350 </td><td>X (A6) </td><td><em>qDB4</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">Flowering locus C </td><td rowspan="3">FLC </td><td rowspan="3">AT5G10140 </td><td>Bra028599 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>Intron (1) </td></tr><tr><td>Bra006051 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>– </td></tr><tr><td>Bra009055 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>,<em>qPL4</em> </td><td>Intron (1) </td></tr></tbody></table></div><div content-id="DSS029TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene family </th><th>Gene name </th><th>At gene ID </th><th><em>B. rapa</em> gene ID </th><th>Block (LGs) </th><th>QTL co-mapping </th><th>SNP location (number) </th></tr></thead><tbody><tr><td rowspan="4">Gibberellin biosythesis </td><td>AtGA2OX4 </td><td>AT1G47990 </td><td>Bra032238 </td><td>C (A5) </td><td><em>qSBL3</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">AtGA20OX3 </td><td rowspan="3">AT5G07200 </td><td>Bra028706 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>– </td></tr><tr><td>Bra005927 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Exon (1) </td></tr><tr><td>Bra009285 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>, <em>qPL4</em> </td><td>– </td></tr><tr><td rowspan="9">Cytokinin oxidase/dehydrogenases </td><td>AtCKX5 </td><td>AT1G75450 </td><td>Bra015842 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="2">AtCKX2 </td><td rowspan="2">AT2G19500 </td><td>Bra036719 </td><td>H (A9) </td><td><em>qSBL5</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td>Bra040677 </td><td>H (A7) </td><td><em>qLW6</em>,<em>qDB5</em>,<em>qFT9</em> </td><td>Intron (1) </td></tr><tr><td rowspan="2">AtCKX1 </td><td rowspan="2">AT2G41510 </td><td>Bra000229 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>– </td></tr><tr><td>Bra016928 </td><td>J (A4) </td><td><em>qLL4</em>,<em>qMRL5</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td rowspan="2"><em>AtCKX5</em> (<em>CKX7</em>) </td><td rowspan="2">AT5G21482 </td><td>Bra020157 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (7) </td></tr><tr><td>Bra002371 </td><td>R (A10) </td><td>– </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="2">AtCKX3 </td><td rowspan="2">AT5G56970 </td><td>Bra035640 </td><td>W (A2) </td><td><em>qFT4</em>,<em>qDB3</em>,<em>qPL2</em>,<em>qSQL1</em>,<em>qSSQ1</em> </td><td>Intron (1) </td></tr><tr><td>Bra002777 </td><td>W (A10) </td><td><em>qSQL8</em>,<em>qSSQ4</em> </td><td>Intron &amp; exon (2) </td></tr><tr><td rowspan="5">Gast1 protein homolog </td><td rowspan="2">GASA4 </td><td>AT5G15230 </td><td>Bra023513 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td> </td><td>Bra006291 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>Intron (1) </td></tr><tr><td rowspan="3">GASA1 </td><td rowspan="3">AT1G75750 </td><td>Bra008222 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra015820 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Exon (1) </td></tr><tr><td>Bra003743 </td><td>E (A7) </td><td><em>qFT10</em>,<em>qDB7</em>,<em>qMRL8</em>,<em>qMRW3</em>,<em>qPH4</em> </td><td>Intron (2) </td></tr><tr><td rowspan="5">Growth-regulating factor </td><td rowspan="2">AtGRF2 </td><td rowspan="2">AT4G37740 </td><td>Bra011781 </td><td>U (A1) </td><td><em>qMRL1</em>,<em>qMRW1</em> </td><td>Exon (3) </td></tr><tr><td>Bra017851 </td><td>U (A3) </td><td>– </td><td>Exon (4) </td></tr><tr><td rowspan="3">AtGRF3 </td><td rowspan="3">AT2G36400 </td><td>Bra023066 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Intron (1) </td></tr><tr><td>Bra017240 </td><td>J (A4) </td><td>– </td><td>Intron (4) </td></tr><tr><td>Bra005268 </td><td>J (A5) </td><td><em>qFT7</em>,<em>qPH3</em> </td><td>Intron &amp; exon (3) </td></tr><tr><td rowspan="2">Apetala1 </td><td rowspan="2">AP1 </td><td rowspan="2">AT1G69120 </td><td>Bra004007 </td><td>E (A7) </td><td><em>qSQL5</em>,<em>qSBL4</em> </td><td>Intron (6) </td></tr><tr><td>Bra038326 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td rowspan="2">Apetala2 </td><td rowspan="2">AP2 </td><td rowspan="2">AT4G36920 </td><td>Bra011741 </td><td>U (A1) </td><td><em>qFT1</em> </td><td>Intron &amp; exon (5) </td></tr><tr><td>Bra017809 </td><td>U (A3) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Asymmetric leaves 1 </td><td rowspan="2">AS1 </td><td rowspan="2">AT2G37630 </td><td>Bra000011 </td><td>J (A3) </td><td><em>qPH2</em>,<em>qLL2</em>,<em>qLW3</em>,<em>qMRL3</em>,<em>qDB4</em>,<em>qFT6</em>,<em>qSQL3</em>,<em>qSQW4</em>,<em>qSBL1</em>,<em>qSSQ2</em>, <em>qSW2</em> </td><td>Exon (1) </td></tr><tr><td>Bra005177 </td><td>J (A5) </td><td>– </td><td>– </td></tr><tr><td rowspan="2">Constans like </td><td rowspan="2">COL1 </td><td rowspan="2">AT5G15850 </td><td>Bra023541 </td><td>R (A2) </td><td><em>qFT3</em>,<em>qDB2</em>,<em>qPL1</em> </td><td>Exon (2) </td></tr><tr><td>Bra008668 </td><td>R (A10) </td><td>– </td><td>Exon (7) </td></tr><tr><td rowspan="2">Flowering locus T </td><td rowspan="2">FT </td><td rowspan="2">AT1G65480 </td><td>Bra022475 </td><td>E (A2) </td><td><em>qSQL2</em> </td><td>– </td></tr><tr><td>Bra004117 </td><td>E (A7) </td><td>– </td><td>– </td></tr><tr><td>Leafy </td><td>LFY </td><td>AT5G61850 </td><td>Bra019619 </td><td>X (A6) </td><td><em>qDB5</em> </td><td>– </td></tr><tr><td>GA-insensitive dwarf 1C </td><td>GA </td><td>AT5G27320 </td><td>Bra009970 </td><td>Q (A6) </td><td><em>qFT8</em>,<em>qMRL7</em>,<em>qLW5</em>,<em>qSW3</em> </td><td>Intron &amp; exon (6) </td></tr><tr><td>Mads affecting flowering 3 </td><td>MAF3 </td><td>AT5G65060 </td><td>Bra024350 </td><td>X (A6) </td><td><em>qDB4</em> </td><td>Intron &amp; exon (4) </td></tr><tr><td rowspan="3">Flowering locus C </td><td rowspan="3">FLC </td><td rowspan="3">AT5G10140 </td><td>Bra028599 </td><td>R (A2) </td><td><em>qFT2</em>,<em>qDB1</em>,<em>qPH1</em>,<em>qLL1</em>,<em>qLW2</em>,<em>qMRL2</em>,<em>qMRW2</em>,<em>qSW1</em> </td><td>Intron (1) </td></tr><tr><td>Bra006051 </td><td>R (A3) </td><td><em>qFT5</em>,<em>qSQW3</em> </td><td>– </td></tr><tr><td>Bra009055 </td><td>R (A10) </td><td><em>qFT14</em>,<em>qDB10</em>,<em>qMRL10</em>,<em>qMRW4</em>,<em>qSW4</em>,<em>qLL5</em>,<em>qPL4</em> </td><td>Intron (1) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5592165" target="_blank">View Large</a></div></div><p>We detected two W blocks, one each on A2 and A10, that harboured QTL for petiole length, seeds per siliqua, leaf traits, days to bolting and flowering, and siliqua length (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). These two blocks harbour paralogous <em>BraCKX3</em> genes (Table <a reveal-id="DSS029TB3" data-open="DSS029TB3" class="link link-reveal link-table xref-fig">3</a>). The E block on A2 only contained QTL for siliqua traits, but the paralogous block on A7 contained major QTL clusters for leaf traits, flowering and bolting, plant height, and siliqua traits. We detected <em>GASA1</em>, <em>flowering locus T</em> (<em>BraFT</em>), and <em>apetala 1</em> (<em>BraAP1</em>) on both LGs. <em>FT</em> and <em>AP</em> are components of the flowering gene network.<sup>.</sup><em>BraCKX5</em> was found only on A7 (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>).</p><p>The J block of A3 contained many QTL for leaf, siliqua, and seed traits, plant height, and days to bolting and flowering, whereas the same blocks in A4 harboured QTL for leaf length and midrib length, and A5 contained QTL for flowering time and plant height (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). <em>A. thaliana</em> growth-regulating factor<sup><a reveal-id="DSS029C32" data-open="DSS029C32" class="link link-ref link-reveal xref-bibr">32</a></sup> (<em>BraGRF3</em>) was found in all three LGs. However, <em>BraCKX1</em> paralogs were found only on A3 and A4, whereas asymmetric leaves 1<sup><a reveal-id="DSS029C33" data-open="DSS029C33" class="link link-ref link-reveal xref-bibr">33</a></sup> (<em>BraAS1</em>) paralogs were found only on A3 and A5 (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>).</p><p>The candidate gene <em>BraGA20X4</em><sup><a reveal-id="DSS029C34" data-open="DSS029C34" class="link link-ref link-reveal xref-bibr">34</a></sup> was identified in the C block of A5, which contains QTL for siliqua beak length. The H block of A7 contained QTL for days to flowering, bolting, and leaf width, whereas the paralogous block of A9 contained QTL for siliqua beak length. <em>BraCKX2</em> paralogs were found in the QTL regions of both LGs (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). The U block of A1 contained QTL for flowering and leaf width, whereas the paralogous block on A3 (U/T block) contained QTL for leaf width, length, and seeds per siliqua. The candidate genes <em>BraGRF2</em> and <em>BraAP2</em> were detected in both LGs. The X block in A6 contained QTL for days to bolting and harboured candidate gene <em>LEAFY</em> and <em>mads affecting flowering time 3</em>. The Q block harboured QTL for seed weight, leaf, and flowering traits and contained the <em>GA-insensitive 1</em> gene (<em>BraGA</em>) that interacts with GA and is involved in floral transition and GA signalling.</p>                            <h3 scrollto-destination=5592169 id="5592169" class="section-title" data-legacy-id=s3d>3.4. Identification, validation, and functional significance analysis of SNPs in candidate genes</h3>
<p>To develop gene-specific SNP markers for breeding morphological and yield-related traits, we aligned all the identified candidate gene sequences from the parental lines and identified varying number of SNPs, ranging from one to many, between them (Fig. <a reveal-id="DSS029F2" data-open="DSS029F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSS029TB3" data-open="DSS029TB3" class="link link-reveal link-table xref-fig">3</a>). As expected, we found more SNPs in the intronic regions than in the exonic regions. However, we did not detect any SNPs in a few genes, e.g. <em>BraGA20OX3</em> in the R blocks of A2 and A10 and <em>BraCKX1</em> in the J block of A03 (Fig. <a reveal-id="DSS029F2" data-open="DSS029F2" class="link link-reveal link-table xref-fig">2</a> and Table <a reveal-id="DSS029TB3" data-open="DSS029TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="5592171" scrollto-destination="5592171"></a>
<div id="DSS029F2" data-id="DSS029F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02902.jpeg?Expires=2147483647&Signature=NwWrGuSlUo6dE7KGbCYY5QmMt6dSsazaClRzu76DpPIY7dTNrLZH9CfySiYCWMjP2wA9dsQISIEED9q8tM210VTuWBth2jvIsCqdBHxNKReCI-G6NkWTzK8QCnWSCkKKgJZ0Nn5Zi0W-xgJKI8OvsUpqcKrsp1R9Jq-cww7YYge0RH9FTf6gmUjBoefW5gR8LYJJvRZWgO6IwVzwFwAIjrs7UIE0v5~0NyDBPTqoQtrhZpOQsO9UcheFrKzhD0z9EsVGel-V5uUhz2~989fphIe3oIYLymCz~wFZDEQItgiZ8-CBqD4c1JWGeDmHyuLvEqtNCtK4mMgpkLRlxfXMMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs identified for different cytokinin oxidases/dehydrogenases (CKX) paralogs between Chiifu and RCBr derived from whole genome resequenced data. Exons and introns are represented by blue rectangular bars and lines, respectively. Numbers indicated by arrows show the SNP positions from the start codon for Chiifu (left nucleotide) and RCBr (right nucleotide)." path-from-xml="dss02902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592171/dss02902.jpeg" path-from-xml="dss02902.jpeg" target="_blank">View large</a><a section="5592171" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02902.jpeg?Expires=2147483647&Signature=yiGZd7ELccejbiK2xELR6SWEHbZwHh2b1B6fRQmyPMhDg2W8ikkUEABASJ5xnx5sOrT7gwxMfhC3nKc4PcI4f7pvTOjXeOucL2KfAntjh1zVKe8fhk2B3FJqfX4P2WvI-6gBt8GxVGz0dOYu6FA9Go0d2OD66Iz1iuwS~a9RorxSJVhOcmt062Q-G5yKKLl2C9F~XI5FRPkZ~u5oMXGEwqXpU-E3V7i9nTQe-EW9Zuxjgzlcc3ZgD2YR-DFYhTzpYCNz1Udw29j1ARw55HsfCih-MTescYUWYBaaCStyId7j2wPvh4Fmkj1shYWG~TxY~7w7UDE8vxuSXFgqfpq2Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592171&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs identified for different cytokinin oxidases/dehydrogenases (<em>CKX</em>) paralogs between Chiifu and RCBr derived from whole genome resequenced data. Exons and introns are represented by blue rectangular bars and lines, respectively. Numbers indicated by arrows show the SNP positions from the start codon for Chiifu (left nucleotide) and RCBr (right nucleotide).</p></div></div><div content-id="DSS029F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02902.jpeg?Expires=2147483647&Signature=NwWrGuSlUo6dE7KGbCYY5QmMt6dSsazaClRzu76DpPIY7dTNrLZH9CfySiYCWMjP2wA9dsQISIEED9q8tM210VTuWBth2jvIsCqdBHxNKReCI-G6NkWTzK8QCnWSCkKKgJZ0Nn5Zi0W-xgJKI8OvsUpqcKrsp1R9Jq-cww7YYge0RH9FTf6gmUjBoefW5gR8LYJJvRZWgO6IwVzwFwAIjrs7UIE0v5~0NyDBPTqoQtrhZpOQsO9UcheFrKzhD0z9EsVGel-V5uUhz2~989fphIe3oIYLymCz~wFZDEQItgiZ8-CBqD4c1JWGeDmHyuLvEqtNCtK4mMgpkLRlxfXMMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs identified for different cytokinin oxidases/dehydrogenases (CKX) paralogs between Chiifu and RCBr derived from whole genome resequenced data. Exons and introns are represented by blue rectangular bars and lines, respectively. Numbers indicated by arrows show the SNP positions from the start codon for Chiifu (left nucleotide) and RCBr (right nucleotide)." path-from-xml="dss02902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592171/dss02902.jpeg" path-from-xml="dss02902.jpeg" target="_blank">View large</a><a section="5592171" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02902.jpeg?Expires=2147483647&Signature=yiGZd7ELccejbiK2xELR6SWEHbZwHh2b1B6fRQmyPMhDg2W8ikkUEABASJ5xnx5sOrT7gwxMfhC3nKc4PcI4f7pvTOjXeOucL2KfAntjh1zVKe8fhk2B3FJqfX4P2WvI-6gBt8GxVGz0dOYu6FA9Go0d2OD66Iz1iuwS~a9RorxSJVhOcmt062Q-G5yKKLl2C9F~XI5FRPkZ~u5oMXGEwqXpU-E3V7i9nTQe-EW9Zuxjgzlcc3ZgD2YR-DFYhTzpYCNz1Udw29j1ARw55HsfCih-MTescYUWYBaaCStyId7j2wPvh4Fmkj1shYWG~TxY~7w7UDE8vxuSXFgqfpq2Mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592171&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs identified for different cytokinin oxidases/dehydrogenases (<em>CKX</em>) paralogs between Chiifu and RCBr derived from whole genome resequenced data. Exons and introns are represented by blue rectangular bars and lines, respectively. Numbers indicated by arrows show the SNP positions from the start codon for Chiifu (left nucleotide) and RCBr (right nucleotide).</p></div></div><p>To validate these SNPs experimentally, we designed a total of 12 primer pairs, 1 each from different genes flanking the SNPs and genotyped the parental lines and mapping population. Of the 12 SNPs, 10 primer pairs revealed clear polymorphisms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). We genotyped using these 10 primer pairs in the mapping population and mapped them precisely in the <em>B. rapa</em> genome, which indicated their co-localization with the QTL positions on the respective chromosomes (Figs <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a> and <a reveal-id="DSS029F3" data-open="DSS029F3" class="link link-reveal link-table xref-fig">3</a>). Association analysis with the different phenotypes of the F<sub>3</sub> population showed a correlation with the phenotypes and their co-localization in the QTL regions. We sequenced some of the genes containing SNPs using the resequenced data to validate whether the NGS data of the parental lines were correct. Resequencing of 10 genes showed the exact sequences as shown by NGS. We observed that the SNPs in coding regions of some potential candidate genes affected amino acid sequence, suggesting possible functional roles in phenotypic differences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The SNP in exon 2 of <em>BraGA20OX3b</em> codes for isoleucine in Chiifu and leucine in RCBr, methionine/threonine in exon 2 of <em>BraCKX1b</em> while serine was found in Chiifu, but no counterpart amino acid was observed in RCBr in exon 5 of <em>BraCKX3b</em>. Two amino acid changes were observed in exon 1 of <em>BraGA2OX4</em> (arginine/serine and serine/threonine), one each in exons 2 (aspartate/valine), and four (alanine/threonine) of <em>BraCKX7a</em>, two each in exon 1 (valine/isoleucine and glycine/serine) of <em>BraCKX7b</em> and exon 4 (isoleucine/serine and threonine/alanine) of <em>BraGRF2a</em>, one each in exon 2 (leucine/arginine) and exon 4 (glycine/serine) of <em>BraGRF3c</em>, and two in exon 1 (methionine /valine and threonine/serine) of <em>BraCOL1</em> gene. </p>                        <a id="5592173" scrollto-destination="5592173"></a>
<div id="DSS029F3" data-id="DSS029F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02903.jpeg?Expires=2147483647&Signature=F8MBpZZvENf7gJJS9J2aDJr8V-rNJdveD~NltroTehoGcIHuvlCtPeuf8kw8kDyOVsvSMRZag09xxj9qC77toWy-yACDZcEVz7xLzLCOb8KQkWkTJ3iaUcKahPYGFv-gD9cijXRH3-PzhZEuI-mgWGr6US2d4~cKPuvFG4w-AzOJ9wVZx5zlC14DT9bIgNDckY0fxPLszNaUn4X5dKPzzIHnN7u9ma3VB2u1MqAziEMBwScMwXQrd77Mb8KbNhI6gNKwpA1Dpxe4W~p3NYNWIEd1-RaW2zy0Tp1QZjp7JrP5E9JLpG3yWOhC~ug3thqrSAE7oFyFg8d2ib583Y35IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amplicon scanning analysis of the CRF2 mapping population for SNP markers designed from candidate gene BraGASA4b located in A3 using Light Scanner. Plot A represents the curve-shifted plot, and B represents a difference plot showing ‘Chiifu’ type in red colour, ‘RCBr’ type in grey colour, and hetero type in green colour." path-from-xml="dss02903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592173/dss02903.jpeg" path-from-xml="dss02903.jpeg" target="_blank">View large</a><a section="5592173" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02903.jpeg?Expires=2147483647&Signature=1TkcPGt8Ge5OPWdkMDoTso83OnbE0aPEqb3AAs1O1tlvup~-HltfahyxrefiwXXTUIDfSGUjQ1XTBR4~Gv15Vyn8I9s2GclrrW5o~GE-z3s8r6zhpA2VsTM4eQwZZoa0094llk1QbI5ALed30trdiTS1YaiK9lzFSunijA0VbZ0cmm7uC6gyA1AJCmSgeUi1xnUeKbu3WKyJhz6MoQVbq4qI5qjHb~bbFTP4tCSqghhzCB3lisHfiKpIKG7h0QE3Ft~~Ch-STCsKAEUnLJds0N5fSR74sszPgfLVaBucmD303mtdsyHttSacYlumjKZLPffUXhHrsxEgHt-BR-77Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592173&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amplicon scanning analysis of the CRF<sub>2</sub> mapping population for SNP markers designed from candidate gene <em>BraGASA4b</em> located in A3 using Light Scanner. Plot A represents the curve-shifted plot, and B represents a difference plot showing ‘Chiifu’ type in red colour, ‘RCBr’ type in grey colour, and hetero type in green colour.</p></div></div><div content-id="DSS029F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02903.jpeg?Expires=2147483647&Signature=F8MBpZZvENf7gJJS9J2aDJr8V-rNJdveD~NltroTehoGcIHuvlCtPeuf8kw8kDyOVsvSMRZag09xxj9qC77toWy-yACDZcEVz7xLzLCOb8KQkWkTJ3iaUcKahPYGFv-gD9cijXRH3-PzhZEuI-mgWGr6US2d4~cKPuvFG4w-AzOJ9wVZx5zlC14DT9bIgNDckY0fxPLszNaUn4X5dKPzzIHnN7u9ma3VB2u1MqAziEMBwScMwXQrd77Mb8KbNhI6gNKwpA1Dpxe4W~p3NYNWIEd1-RaW2zy0Tp1QZjp7JrP5E9JLpG3yWOhC~ug3thqrSAE7oFyFg8d2ib583Y35IA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Amplicon scanning analysis of the CRF2 mapping population for SNP markers designed from candidate gene BraGASA4b located in A3 using Light Scanner. Plot A represents the curve-shifted plot, and B represents a difference plot showing ‘Chiifu’ type in red colour, ‘RCBr’ type in grey colour, and hetero type in green colour." path-from-xml="dss02903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592173/dss02903.jpeg" path-from-xml="dss02903.jpeg" target="_blank">View large</a><a section="5592173" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02903.jpeg?Expires=2147483647&Signature=1TkcPGt8Ge5OPWdkMDoTso83OnbE0aPEqb3AAs1O1tlvup~-HltfahyxrefiwXXTUIDfSGUjQ1XTBR4~Gv15Vyn8I9s2GclrrW5o~GE-z3s8r6zhpA2VsTM4eQwZZoa0094llk1QbI5ALed30trdiTS1YaiK9lzFSunijA0VbZ0cmm7uC6gyA1AJCmSgeUi1xnUeKbu3WKyJhz6MoQVbq4qI5qjHb~bbFTP4tCSqghhzCB3lisHfiKpIKG7h0QE3Ft~~Ch-STCsKAEUnLJds0N5fSR74sszPgfLVaBucmD303mtdsyHttSacYlumjKZLPffUXhHrsxEgHt-BR-77Pw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592173&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Amplicon scanning analysis of the CRF<sub>2</sub> mapping population for SNP markers designed from candidate gene <em>BraGASA4b</em> located in A3 using Light Scanner. Plot A represents the curve-shifted plot, and B represents a difference plot showing ‘Chiifu’ type in red colour, ‘RCBr’ type in grey colour, and hetero type in green colour.</p></div></div><p>Semi-quantitative RT-PCR analysis was used to evaluate expression differences of candidate genes between the parental lines. The gibberellin biosynthesis and oxidase, cytokinin oxidases/dehydrogenases and growth-regulating factor gene families that affect leaf, plant, and flower development, showed differential gene expression between Chiifu and RCBr lines (Fig. <a reveal-id="DSS029F4" data-open="DSS029F4" class="link link-reveal link-table xref-fig">4</a>). <em>BraCKX5</em>, <em>BraCKX3a</em>, <em>BraCKX1a</em>, and <em>BraCKX7a</em> showed higher expression in Chiifu, whereas very little or no expression was seen in RCBr. Genes that are involved in flowering pathways, such as <em>FLC</em>, <em>FT</em>, <em>COL</em>, and <em>GA</em>, were also analysed for differential expression between the parental lines. Of the three <em>BraFLC</em> paralogs, only <em>BraFLCa</em> on A2 did not show any expression in RCBr while the other two <em>BraFLCs</em> showed no difference in expression. <em>BraMAF3</em> on A6 showed higher expression level in Chiifu when compared with RCBr. <em>BraAP2a</em> on A1, <em>BraCOL1a</em> on A2, <em>BraGA</em> on A6 and <em>BraFTb</em> on A7 showed strong expression in RCBr, but very little to no expression in Chiifu<em>. BraGRF2a</em>, <em>BraGRF3a</em>, and <em>BraGA2OX4</em> showed comparatively higher expression in Chiifu while <em>BraGA20OX3c</em> showed higher expression in RCBr. This differential gene expression and the amino acid changes suggest possible roles in phenotypic differences between the parental lines and in the segregating population. </p>                        <a id="5592175" scrollto-destination="5592175"></a>
<div id="DSS029F4" data-id="DSS029F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02904.jpeg?Expires=2147483647&Signature=i00iDufqxUeoLk~1ekxaTewWkq23b3h3J39Ksj9C6re4NKe50p6XY3DCdJePIp3ldTgpAeUwPr34Twa~Trp-C6Rj1NZDTpi7FCpTUWYMcdKPhqUHRFJ080msz66Uf4sUlqE2S1D6X88rcVK4h5DDyoB6X9W8yNLXJanyCR8RlUlwFAW9VNUDzLPw5P9OpKe2lmoMYyx1fCAMogt2uRcHUDN6uPoRCBEjJDt~roPEDIK9SsN4G1vFIuOWWp2eHzeTaHNaqyHq18vjfLYDVV3V59eRZvWHfuzsyJgPthrcKyuVZj06xtj39liU71uHLmUF6wCvg02txTgEagYYH6lEUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Semi-quantitative RT-PCR analysis of candidate genes using RNA samples extracted from 20-day-old seedlings of Chiifu and RCBr. Actin gene amplicons were used as control for RNA quantity." path-from-xml="dss02904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592175/dss02904.jpeg" path-from-xml="dss02904.jpeg" target="_blank">View large</a><a section="5592175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02904.jpeg?Expires=2147483647&Signature=0pFyp1KthJGIF7aXHibxzx5Yel4NEjiK09HRE10Kyb0UkJv5uspa3510BHlqm6EsDvt7nlbhPqMwTAsmjOA0pip2uF~NDvw7Jm27zX6EVQN5~OZbb6vqrMSOMMUcvWEHy~FgfKzoP7jLeCpv6MlCqEh1ghL~geYFqiwZBP0RTO19yZ3HAQSJP3RjxYBNKqNw60gbVE-GQxRT6rckGyYvAnfLNiOObevRfAkdcsT2vQerEFiBbil5GnENhbKc5--xYnIsqg4qjnCeljqVjjzYTJ6o93LRJgpILac55ai~ZSiIbf~EQbAKgzkLdTa73ha8lIvxPTs6sJ25zPSpUbBoZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592175&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Semi-quantitative RT-PCR analysis of candidate genes using RNA samples extracted from 20-day-old seedlings of Chiifu and RCBr. Actin gene amplicons were used as control for RNA quantity.</p></div></div><div content-id="DSS029F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02904.jpeg?Expires=2147483647&Signature=i00iDufqxUeoLk~1ekxaTewWkq23b3h3J39Ksj9C6re4NKe50p6XY3DCdJePIp3ldTgpAeUwPr34Twa~Trp-C6Rj1NZDTpi7FCpTUWYMcdKPhqUHRFJ080msz66Uf4sUlqE2S1D6X88rcVK4h5DDyoB6X9W8yNLXJanyCR8RlUlwFAW9VNUDzLPw5P9OpKe2lmoMYyx1fCAMogt2uRcHUDN6uPoRCBEjJDt~roPEDIK9SsN4G1vFIuOWWp2eHzeTaHNaqyHq18vjfLYDVV3V59eRZvWHfuzsyJgPthrcKyuVZj06xtj39liU71uHLmUF6wCvg02txTgEagYYH6lEUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Semi-quantitative RT-PCR analysis of candidate genes using RNA samples extracted from 20-day-old seedlings of Chiifu and RCBr. Actin gene amplicons were used as control for RNA quantity." path-from-xml="dss02904.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592175/dss02904.jpeg" path-from-xml="dss02904.jpeg" target="_blank">View large</a><a section="5592175" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02904.jpeg?Expires=2147483647&Signature=0pFyp1KthJGIF7aXHibxzx5Yel4NEjiK09HRE10Kyb0UkJv5uspa3510BHlqm6EsDvt7nlbhPqMwTAsmjOA0pip2uF~NDvw7Jm27zX6EVQN5~OZbb6vqrMSOMMUcvWEHy~FgfKzoP7jLeCpv6MlCqEh1ghL~geYFqiwZBP0RTO19yZ3HAQSJP3RjxYBNKqNw60gbVE-GQxRT6rckGyYvAnfLNiOObevRfAkdcsT2vQerEFiBbil5GnENhbKc5--xYnIsqg4qjnCeljqVjjzYTJ6o93LRJgpILac55ai~ZSiIbf~EQbAKgzkLdTa73ha8lIvxPTs6sJ25zPSpUbBoZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592175&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02904.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Semi-quantitative RT-PCR analysis of candidate genes using RNA samples extracted from 20-day-old seedlings of Chiifu and RCBr. Actin gene amplicons were used as control for RNA quantity.</p></div></div>                            <h3 scrollto-destination=5592176 id="5592176" class="section-title" data-legacy-id=s3e>3.5. Functional conservation of crucifer building blocks between the A, B, and C subgenomes of <em>Brassica</em> species as revealed by comparative QTL mapping</h3>
<p>We aligned the QTL map of <em>B. rapa</em> from this study with the <em>B. juncea</em> map of Ramchiary et al.<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and <em>B. napus</em> QTL maps of Quijada et al.<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and Udall et al.<sup><a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup> to identify the structural and functional conservation of QTL blocks for important morphological and yield component traits. Comparative alignment of the QTL maps between these three species identified a total of four homologous/homoeologous groups (HGs) containing QTL for agronomic and yield component traits (Fig. <a reveal-id="DSS029F5" data-open="DSS029F5" class="link link-reveal link-table xref-fig">5</a>). The first group was the R block of A<sup>r</sup>2, A<sup>j</sup>2, A<sup>n</sup>2, C<sup>n</sup>2, and B<sup>j</sup>2 (r, j, and n stand for <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. napus</em>, respectively). Although many major QTL for different traits were detected in the <em>B. rapa</em> A subgenome, in comparison with <em>B. napus</em>, three traits, namely, plant height, flowering time, and seed yield, were commonly detected in the A subgenome of the two species and the homoeologous region of the same block in the C subgenome of <em>B. napus</em>. We did not detect any QTL in the A<sup>j</sup>2 or B<sup>j</sup>2 subgenomes of <em>B. juncea</em> (Fig. <a reveal-id="DSS029F3" data-open="DSS029F3" class="link link-reveal link-table xref-fig">3</a>). The second HG was the A3, B3, and C3 chromosomes of the A, B, and C subgenomes of <em>Brassica</em> species. Three blocks, namely, R, J, and F, harboured common QTL between the three subgenomes. The R block of this HG contained common QTL for seed weight, yield, and test weight (hereafter referred to collectively as SW) and flowering time QTL between the A and C subgenomes of <em>B. napus</em> and the A subgenome of <em>B. rapa</em>. In addition, the homoeologous region of this block contained QTL for a number of branches of the B subgenome of <em>B. juncea</em>. The J block of this group contained many QTL in A<sup>r</sup>3, but few in A<sup>n</sup>3, A<sup>j</sup>3, and B<sup>j</sup>3. The common loci were for SW and flowering time in addition to siliqua traits, e.g. siliqua length, density, and number, and plant height. The last common block, i.e. the F block of this group, harboured common trait QTL mainly for SW and plant height between A<sup>n</sup>3 and C<sup>n</sup>3. The most important common block of HG7 was the E block. This block contained common QTL for plant height, SW, flowering time, siliqua number and density, and seed number per siliqua in A<sup>r</sup>7, A<sup>j</sup>7, C<sup>n</sup>7, and B<sup>j</sup>7. In addition, some leaf traits were detected in A<sup>r</sup>7. The last common group of the A, B, and C <em>Brassica</em> subgenomes was HG10 that contained the W and R blocks. The W block contained common QTL for siliqua length, seed number per siliqua, and SW between A<sup>r</sup>10, A<sup>n</sup>10, and A<sup>j</sup>10. In addition, QTL for flowering, plant height, and leaf traits were detected in A<sup>r</sup>10 and A<sup>n</sup>10, respectively. However, in this block, no single or corresponding common QTL were detected in B<sup>j</sup>10 and C<sup>n</sup>10. The most important block of HG10 was the R block. This block harboured common major QTL for plant height, branch number, flowering time, siliqua traits, and SW in A<sup>r</sup>10, A<sup>j</sup>10, A<sup>n</sup>10, B<sup>j</sup>10, and C<sup>n</sup>10, respectively. Similarly, the middle region of A<sup>r</sup>6 (X and Q block) in the present study detected QTL for SW, leaf, and flowering traits while the same chromosomal region in <em>B. juncea</em> detected major QTL for siliqua length and density.<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> We also detected common QTL for flowering and SW, both in A<sup>r</sup>6 and C<sup>n</sup>6 subgenomes.<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                        <a id="5592178" scrollto-destination="5592178"></a>
<div id="DSS029F5" data-id="DSS029F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02905.jpeg?Expires=2147483647&Signature=1fc3OGzo9GtMcXofE~BoF48oWst3b0DiuhN58GWvQNWmtASipTOM3MS085NtPU7jSWMJ1r7IRf72jBNGH6OWWtMPwrLla6USbMW3WAIwvvZ2Ic-gsYh6L2MnZ~glFoYES9bV56QVUwkaS2fD5iUuaE3NN7K6WoIO2MDDiKqsii9zPuphqTUCUZZpmW0hQAhrneP7jnGkW-dE5Gx~un3SviudhCLa8uJW5A-di231Q4UPfqQv80DIjJ6ibjAXcpx-GmCImUGV0Sbn9igKMNt3i-cbxTUmX1YxDgWfBb19DsslxlLkodMx~7vrA3RVHJ7FE-f8FlmLrJ~t0a04MZj1-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative QTL mapping between B. rapa, B. juncea, and B. napus revealed conservation of crucifer building blocks governing morphological and yield component traits. Superscripts ‘r’, ‘n’, and ‘j’ after each A, B, and C LGs denote chromosomes that were derived from B. rapa, B. napus, and B. juncea, respectively. QTL in B genome of B. juncea were taken from Ramchiary et al.,20 and the C genome of B. napus from Quijada et al.,21 and Udall et al.22" path-from-xml="dss02905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592178/dss02905.jpeg" path-from-xml="dss02905.jpeg" target="_blank">View large</a><a section="5592178" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02905.jpeg?Expires=2147483647&Signature=yzQ7ZYLxyBuIZ07gJ1n5m-MO2zitGP6awwvWjQR2M0d3hikknQ95GshKrDRdB8mxrOuuSitxs09dQ8ZEg0dCXMsDPZknTDGQ8veg2gRODIKiKcnt6QhKgdAcOJaCKP6kMIUNmownzYvXaeZNIs4Ft5K-WXI3sd4-8-7ThtJQHLtSuB8b-lIwtxhBJfHLGMDTSSP-P1wbD77jaCz5SGN6G30b4c3HQ5MHpKAjdTr4Xg9dvqiVd0HLEOIRWjg3WbLRId7b7QcAUcWIX8HvcxdOVJI1y~tSZ4jXds~7ldAbwQbUt9Ju4vf18lC9b3UjyKoylYMxj52wK2r7lOuI7LvbHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592178&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative QTL mapping between <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. napus</em> revealed conservation of crucifer building blocks governing morphological and yield component traits. Superscripts ‘r’, ‘n’, and ‘j’ after each A, B, and C LGs denote chromosomes that were derived from <em>B. rapa</em>, <em>B. napus</em>, and <em>B. juncea</em>, respectively. QTL in B genome of <em>B. juncea</em> were taken from Ramchiary et al.,<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and the C genome of <em>B. napus</em> from Quijada et al.,<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and Udall et al.<sup><a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div><div content-id="DSS029F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/m_dss02905.jpeg?Expires=2147483647&Signature=1fc3OGzo9GtMcXofE~BoF48oWst3b0DiuhN58GWvQNWmtASipTOM3MS085NtPU7jSWMJ1r7IRf72jBNGH6OWWtMPwrLla6USbMW3WAIwvvZ2Ic-gsYh6L2MnZ~glFoYES9bV56QVUwkaS2fD5iUuaE3NN7K6WoIO2MDDiKqsii9zPuphqTUCUZZpmW0hQAhrneP7jnGkW-dE5Gx~un3SviudhCLa8uJW5A-di231Q4UPfqQv80DIjJ6ibjAXcpx-GmCImUGV0Sbn9igKMNt3i-cbxTUmX1YxDgWfBb19DsslxlLkodMx~7vrA3RVHJ7FE-f8FlmLrJ~t0a04MZj1-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative QTL mapping between B. rapa, B. juncea, and B. napus revealed conservation of crucifer building blocks governing morphological and yield component traits. Superscripts ‘r’, ‘n’, and ‘j’ after each A, B, and C LGs denote chromosomes that were derived from B. rapa, B. napus, and B. juncea, respectively. QTL in B genome of B. juncea were taken from Ramchiary et al.,20 and the C genome of B. napus from Quijada et al.,21 and Udall et al.22" path-from-xml="dss02905.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5592178/dss02905.jpeg" path-from-xml="dss02905.jpeg" target="_blank">View large</a><a section="5592178" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss02905.jpeg?Expires=2147483647&Signature=yzQ7ZYLxyBuIZ07gJ1n5m-MO2zitGP6awwvWjQR2M0d3hikknQ95GshKrDRdB8mxrOuuSitxs09dQ8ZEg0dCXMsDPZknTDGQ8veg2gRODIKiKcnt6QhKgdAcOJaCKP6kMIUNmownzYvXaeZNIs4Ft5K-WXI3sd4-8-7ThtJQHLtSuB8b-lIwtxhBJfHLGMDTSSP-P1wbD77jaCz5SGN6G30b4c3HQ5MHpKAjdTr4Xg9dvqiVd0HLEOIRWjg3WbLRId7b7QcAUcWIX8HvcxdOVJI1y~tSZ4jXds~7ldAbwQbUt9Ju4vf18lC9b3UjyKoylYMxj52wK2r7lOuI7LvbHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5592178&ar=381937&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02905.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative QTL mapping between <em>B. rapa</em>, <em>B. juncea</em>, and <em>B. napus</em> revealed conservation of crucifer building blocks governing morphological and yield component traits. Superscripts ‘r’, ‘n’, and ‘j’ after each A, B, and C LGs denote chromosomes that were derived from <em>B. rapa</em>, <em>B. napus</em>, and <em>B. juncea</em>, respectively. QTL in B genome of <em>B. juncea</em> were taken from Ramchiary et al.,<sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and the C genome of <em>B. napus</em> from Quijada et al.,<sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and Udall et al.<sup><a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div>                            <h2 scrollto-destination=5592179 id="5592179" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5592180 id="5592180" class="section-title" data-legacy-id=s4a>4.1. QTL mapping revealed some crucifer blocks that were functionally more active than duplicated paralogous blocks in the B. rapa genome</h3>
<p>Although in many studies, immortal mapping populations such as doubled haploid and recombinant inbred lines are used, in our present study, we used F<sub>3</sub> families for three consecutive years to detect QTL, as F<sub>3</sub> families represent the comprehensive genetic information of heterozygous and homozygous plants and their interacting effects on phenotype. As the parental lines Chiifu and RCBr were very different with respect to all the traits under study, it was easy to detect many QTL in <em>B. rapa</em> using the segregating F<sub>3</sub> population. Correlation coefficient analysis revealed significant correlations between leaf and yield-related traits that were further supported by the co-mapping of several QTL governing different traits in a few chromosomal blocks of the <em>B. rapa</em> genome. The co-localization of QTL governing different traits in the same genetic interval has been suggested to be due to either pleiotropy exhibited by a single gene or tight linkage of genetic loci governing different traits in the same genetic interval.<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The R block of A2 contained QTL for eight traits while the same paralogous blocks on A10 and A3 contained QTL for five and four traits, respectively (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). Prominent QTL clusters were observed for seven traits related to leaf, flowering, bolting, and siliqua traits on the E block of A7 while the same block on A2 contained only one QTL for siliqua length. A similar case was observed for the J blocks, where the same block on A3 harboured QTL for 10 traits, but only harboured QTL for 2 traits on A4 and A5. These findings suggest that the R block of A2, E block of A7, and J block of A3 contain more active genes that govern leaf traits, plant morphology, and seed traits. Lou et al.<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and Li et al.<sup><a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a></sup> also observed QTL clusters in these LGs for days to flowering, bolting, and leaf traits. We presumed that the chromosomal blocks of Lou et al.<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a></sup> were the R block on A2 and E block on A7. The W block on A2 contained QTL clusters for days to flowering and bolting and seeds per siliqua, whereas A10 contained QTL for siliqua and leaf traits. This kind of observation suggests that only those crucifer blocks harbouring QTL might possess functionally active genes for the traits under study, as reported by Wang et al.<sup><a reveal-id="DSS029C16" data-open="DSS029C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Tang et al.<sup><a reveal-id="DSS029C35" data-open="DSS029C35" class="link link-ref link-reveal xref-bibr">35</a></sup> for the <em>B. rapa</em> genome. They reported that, of the triplicate duplicated subgenomes, fractionation and deletion of genes were more frequent in some subgenomes when compared with other paralogs. We also observed that some candidate genes were missing in major QTL blocks, such as <em>BraGASA4</em> from A10 and <em>BraCKX5</em> from A3, which both belong to the R block that is homologous to <em>A. thaliana</em> chromosome 5. Other reasons for not finding equal QTL/genes in all paralogous blocks may be that the genes are not functionally active or the absence of polymorphisms in the causal genes between the parental lines.</p>                            <h3 scrollto-destination=5592182 id="5592182" class="section-title" data-legacy-id=s4b>4.2. Gene family members were detected in different QTL blocks of the B. rapa genome</h3>
<p>Many candidate genes in QTL blocks for flowering, leaf, seed, and siliqua traits were identified through synteny analysis of the <em>B. rapa</em> and <em>A. thaliana</em> genomes. The R block, homologous to <em>A. thaliana</em> chromosome 5, harbours many flowering genes, including <em>FLC</em> and <em>COL1</em>. We identified <em>BraFLC</em> and <em>BraCOL1</em> paralogs in those blocks containing QTL for flowering time, as reported by earlier studies.<sup><a reveal-id="DSS029C8" data-open="DSS029C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS029C36" data-open="DSS029C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Recent studies have shown that members of cytokinin oxidase/dehydrogenases (<em>CKX</em>), GA-stimulated <em>Arabidopsis</em> (<em>GASA</em>), and GA oxidase (<em>GA2OX</em>, <em>GA20OXs</em>, etc.) gene families are involved in the regulation of flower and inflorescence development, thereby increasing the seed size and yield.<sup><a reveal-id="DSS029C28  DSS029C31" data-open="DSS029C28  DSS029C31" class="link link-ref link-reveal xref-bibr">28–31</a></sup><em>CKX</em> genes are important for agricultural productivity as mutations in these genes cause accumulation of high levels of cytokinin in the inflorescence meristem and increased reproductive organs and larger flowers and seeds.<sup><a reveal-id="DSS029C29" data-open="DSS029C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS029C30" data-open="DSS029C30" class="link link-ref link-reveal xref-bibr">30</a></sup> These genes co-localized in the QTL blocks of <em>B. rapa</em> LGs A2, A3, A4, A7, A9, and A10. <em>GASA</em> genes are active in the shoot apical meristem, developing flowers, and embryos. <em>GASA</em> mutants and <em>GASA</em> over-expressing plants demonstrated that <em>GASA</em> genes are involved in floral meristem identity and affect seed size and yield in <em>A. thaliana</em>.<sup><a reveal-id="DSS029C29" data-open="DSS029C29" class="link link-ref link-reveal xref-bibr">29</a></sup> We observed two paralogs of <em>BraGASA4</em>, one on each R block of A2 and A3, but not on the R block of A10, suggesting that the paralogs on A10 may have been lost over time, although all blocks harboured QTL for flowering, siliqua, and seed traits. GA oxidases were observed in the QTL regions of the R, J, and other blocks containing prominent QTL for leaf and flowering traits, and a possible involvement of this gene family in leaf traits was reported by Li et al.<sup><a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The <em>BraGRF</em> members found on A1 and A5 are also candidates for leaf and flowering traits, as supported by their co-localization with these QTL regions. Several members of the <em>BraCOL</em> and <em>BraFT</em> families were also found in different LGs in the QTL regions, suggesting an involvement in trait variation.</p><p>SNPs analysis revealed some amino acid changes between the parental lines in many genes belonging to constans like 1(<em>COL1</em>), cytokinin oxidases/dehydrogenases, growth-regulating factor, and GA oxidases gene families. Further expression analysis showed higher expression of genes that facilitate early flowering such as <em>BraCOL1a</em>, <em>BraFT</em>, <em>BraAP2</em> in RCBr, but minimal expression in Chiifu, whereas the flowering repressor gene <em>BraFLCa</em> was highly expressed in Chiifu suggesting an association between flowering trait and expression of these genes. The co-mapping of these genes with flowering QTL in different LGs further supported this notion. Members of growth-regulating factor, GA oxidase, and cytokinin oxidase/dehydrogenases gene families that co-localized within the QTL region showed higher expression in Chiifu, suggesting their role in better growth and development of leaf, plants, and reproductive organs when compared with RCBr. Either non-synonymous SNPs or differential genes expression or a combination of these factors may underlie phenotypic differences between the parental lines and in the segregating populations. However, this should be further assessed in different plant tissues.</p>                            <h3 scrollto-destination=5592185 id="5592185" class="section-title" data-legacy-id=s4c>4.3. QTL blocks are evolutionarily conserved between the A, B, and C subgenomes of <em>Brassica</em> species</h3>
<p>Our comparative QTL map alignment between <em>B. rapa</em>, <em>B. napus</em>, and <em>B. juncea</em> revealed conservation of QTL blocks that contain several genetic loci influencing plant height, flowering, siliqua, and seed traits between the A, B, and C subgenomes of <em>Brassica</em> species (Fig. <a reveal-id="DSS029F5" data-open="DSS029F5" class="link link-reveal link-table xref-fig">5</a>). Although many earlier studies reported the conservation of chromosomal blocks at the gross level,<sup><a reveal-id="DSS029C10" data-open="DSS029C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSS029C14" data-open="DSS029C14" class="link link-ref link-reveal xref-bibr">14</a></sup> a comparison of QTL location has not been performed for yield component traits between different <em>Brassica</em> subgenomes. In our present study, we identified conserved QTL blocks in 4 of the 10 HGs of the A, B, and C subgenomes reported by Panjabi et al.<sup><a reveal-id="DSS029C11" data-open="DSS029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> As plant height, flowering time, siliqua, and seed traits were the only common traits studied in the three species, we could study the conservation of these traits in the four HG QTL blocks. The R blocks of HG2, HG3, and HG10, E block of HG7, W block of HG10, and J and F blocks of HG2 were major QTL blocks harbouring common QTL for SW, plant height, flowering, and siliqua traits between the A, B, and C subgenomes of these three species. This suggest that the genes governing these traits are structurally and functionally conserved not only between the A, B, and C subgenomes of <em>Brassica</em> species but also between <em>A. thaliana</em> and <em>Brassica</em> genomes, even though they diverged from their common ancestor a long time ago.<sup><a reveal-id="DSS029C12" data-open="DSS029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> We previously found the conservation of QTL for seed oil, total seed glucosinolates, and seed protein content between the A subgenomes of <em>B. rapa</em> and <em>B. napus</em>.<sup><a reveal-id="DSS029C5" data-open="DSS029C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Studies on <em>B. napus</em><sup><a reveal-id="DSS029C21" data-open="DSS029C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS029C22" data-open="DSS029C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and <em>B. juncea</em><sup><a reveal-id="DSS029C20" data-open="DSS029C20" class="link link-ref link-reveal xref-bibr">20</a></sup> detected QTL in homoeologous LGs of the A and C and A and B subgenomes, respectively, but detailed information with respect to specific crucifer blocks was not provided. We found more common QTL between the A and C subgenomes when compared with the B subgenome with either the A or C subgenomes. Although we believe that more comparisons using QTL maps from different mapping populations are needed, the present study suggests that A and C (diverged 8 MYA) are more close than the A/B and C subgenomes, as reported earlier.<sup><a reveal-id="DSS029C12" data-open="DSS029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> We also observed many QTL in one species that had no counterparts controlling the same traits in the corresponding locations of other species, which might have been the result of a lack of genetic polymorphisms between the parental lines or diversified molecular polymorphisms between the species due to the presence of genomic rearrangements/structural changes causing differences in functional expression between different <em>Brassica</em> species. We believe that adding more markers to generate a high-density map and a detailed comparison of QTL for common traits using different mapping populations of more <em>Brassica</em> species would give more detailed information about the conservation and diversification of genetic loci governing morphological and yield component traits.</p>                            <h3 scrollto-destination=5592187 id="5592187" class="section-title" data-legacy-id=s4d>4.4. Breeding opportunities in Brassica species using candidate gene information from B. rapa</h3>
<p>The present QTL mapping study identified some interesting findings regarding the presence of candidate genes for important traits in a few specific crucifer building blocks of the <em>B. rapa</em> genome (Fig. <a reveal-id="DSS029F1" data-open="DSS029F1" class="link link-reveal link-table xref-fig">1</a>). Although, the crucifer blocks demonstrated the co-localization of important QTL and candidate genes, association analysis using large segregating population with different trait phenotypes and functional validation of candidate genes by transformation should be done because each QTL block contains several genes governing overlapping traits. We observed few gene paralogs within the QTL interval in one duplicated block but not in others, suggesting the need for further validation. However, we believe that some of the SNPs identified in candidate genes such as <em>BraCKXs</em>, <em>BraGASA4</em>, <em>BraGA20OXs</em>, <em>BraGRFs</em>, <em>BraFLC</em>, <em>BraCOLs</em>, <em>BraMAF</em>, <em>and BraAP2</em> would be helpful in breeding leaf, flowering, bolting, siliqua, and seed traits of <em>B. rapa</em>, as their differential expressions in parental lines and previous studies suggest their role in creating phenotypic differences<sup><a reveal-id="DSS029C9" data-open="DSS029C9" class="link link-ref link-reveal xref-bibr">9</a></sup>. However, additional expression studies using different tissues are needed. Although genome sequencing of other cultivated <em>Brassica</em> species is ongoing, gene sequence information of <em>B. rapa</em> could be used to isolate and develop molecular markers for breeding other <em>Brassica</em> species, especially <em>B. juncea</em> and <em>B. napus</em>, as <em>B. rapa</em> is one of the progenitor species of both species and there still exists QTL block conservation for important agronomic and yield component traits between these three species.</p>                            <h2 scrollto-destination=5592189 id="5592189" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss029/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=5592191 id="5592191" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was partially supported by Chinese Cabbage Molecular Marker Research Center (Grant No. 607003-05) and Agriculture Research Center (Grant No. 06362010-0010) from the Technology Development Program for Agriculture and Forestry, Ministry of Agriculture, Forestry and Fisheries, as well as a grant from the Next-Generation BioGreen 21 Program (Plant Molecular Breeding Center No. PJ007992), Rural Development Administration, Republic of Korea.</p>                            <h2 scrollto-destination=5592193 id="5592193" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are thankful to the Korea Brassica Genome Resource Bank for providing Chiifu and CRF<sub>3</sub> mapping population, Prof. Deepak Pental and Prof. Akshay K. Pradhan, Department of Genetics, Delhi University South Campus, India for the updated <em>B. juncea</em> map.</p>                            <h2 scrollto-destination=5592195 id="5592195" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS029C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsukazaki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Iketani</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Simple sequence repeat-based comparative genomics between <em>Brassica rapa</em> and <em>Arabidopsis thaliana</em>: the genetic origin of clubroot resistance</div>, <div class="source">Genetics</div> , <div class="year">2006</div>, vol. <div class="volume">173</div> (pg. <div class="fpage">309</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.104.038968" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.104.038968</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeat-based%20comparative%20genomics%20between%20Brassica%20rapa%20and%20Arabidopsis%20thaliana%3A%20the%20genetic%20origin%20of%20clubroot%20resistance&amp;author=K.%20Suwabe&amp;author=H.%20Tsukazaki&amp;author=H.%20Iketani&amp;publication_year=2006&amp;journal=Genetics&amp;volume=173&amp;pages=309-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.104.038968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.104.038968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.104.038968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16723420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">S.R.</div></div>, et al. </div><div class="article-title">Development of a high density integrated reference genetic linkage map for the multinational <em>Brassica rapa</em> Genome Sequencing Project</div>, <div class="source">Genome</div> , <div class="year">2010</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">939</div>-<div class="lpage">47</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1139/G10-054" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1139/G10-054</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20high%20density%20integrated%20reference%20genetic%20linkage%20map%20for%20the%20multinational%20Brassica%20rapa%20Genome%20Sequencing%20Project&amp;author=X.%20Li&amp;author=N.%20Ramchiary&amp;author=S.R.%20Choi&amp;publication_year=2010&amp;journal=Genome&amp;volume=53&amp;pages=939-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G10-054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG10-054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG10-054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21076509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanhuanpaa</div> <div class="given-names">P.K.</div></div>,  <div class="name"><div class="surname">Vilkke</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Vikki</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Mapping a QTL for oleic acid concentration in spring turnip rape (<em>Brassica rapa</em> ssp. <em>oleifera</em>)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1996</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">952</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF00224034" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF00224034</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20a%20QTL%20for%20oleic%20acid%20concentration%20in%20spring%20turnip%20rape%20(Brassica%20rapa%20ssp.%20oleifera)&amp;author=P.K.%20Tanhuanpaa&amp;author=J.P.%20Vilkke&amp;author=H.J.%20Vikki&amp;publication_year=1996&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=92&amp;pages=952-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00224034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00224034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00224034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24166621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">He</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Quantitative trait loci for glucosinolate accumulation in <em>Brassica rapa</em> leaves</div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">1017</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2008.02530.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1469-8137.2008.02530.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20glucosinolate%20accumulation%20in%20Brassica%20rapa%20leaves&amp;author=P.%20Lou&amp;author=J.%20Zhao&amp;author=H.%20He&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=1017-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18564302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Structural and functional comparative mapping between the Brassica A genomes in allotetraploid <em>Brassica napus</em> and diploid <em>Brassica rapa</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">927</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-011-1637-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-011-1637-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20comparative%20mapping%20between%20the%20Brassica%20A%20genomes%20in%20allotetraploid%20Brassica%20napus%20and%20diploid%20Brassica%20rapa&amp;author=C.%20Jiang&amp;author=N.%20Ramchiary&amp;author=Y.%20Ma&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=927-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1637-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1637-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1637-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21761162" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kubo</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tsukada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hirai</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Fine mapping of the clubroot resistance gene, Crr3, in <em>Brassica rapa</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">81</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0412-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0412-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20the%20clubroot%20resistance%20gene%2C%20Crr3%2C%20in%20Brassica%20rapa&amp;author=M.%20Saito&amp;author=N.%20Kubo&amp;author=S.%20Matsumoto&amp;author=K.%20Suwabe&amp;author=M.%20Tsukada&amp;author=M.%20Hirai&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=81-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0412-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0412-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0412-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17039346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Soengas</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hand</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Vicente</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Pole</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Pink</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Identification of quantitative trait loci for resistance to <em>Xanthomonas campestris</em> pv.<em>campestris</em> in <em>Brassica rapa</em></div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">114</div> (pg. <div class="fpage">637</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0464-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0464-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20for%20resistance%20to%20Xanthomonas%20campestris%20pv.campestris%20in%20Brassica%20rapa&amp;author=P.%20Soengas&amp;author=P.%20Hand&amp;author=J.G.%20Vicente&amp;author=J.M.%20Pole&amp;author=D.A.%20Pink&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=114&amp;pages=637-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0464-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0464-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0464-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17160671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>, et al. </div><div class="article-title">Quantitative trait loci for flowering time and morphological traits in multiple populations of <em>Brassica rapa</em></div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">4005</div>-<div class="lpage">16</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erm255" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erm255</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20loci%20for%20flowering%20time%20and%20morphological%20traits%20in%20multiple%20populations%20of%20Brassica%20rapa&amp;author=P.%20Lou&amp;author=J.J.%20Zhao&amp;author=J.S.%20Kim&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot.&amp;volume=58&amp;pages=4005-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18048374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Kitashiba</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Inaba</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishio</div> <div class="given-names">T.</div></div>. </div><div class="article-title">A <em>Brassica rapa</em> linkage map of EST-based SNP markers for identification of candidate genes controlling flowering time and leaf morphological traits</div>, <div class="source">DNA Res.</div> , <div class="year">2009</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">311</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsp020" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsp020</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Brassica%20rapa%20linkage%20map%20of%20EST-based%20SNP%20markers%20for%20identification%20of%20candidate%20genes%20controlling%20flowering%20time%20and%20leaf%20morphological%20traits&amp;author=F.%20Li&amp;author=H.%20Kitashiba&amp;author=K.%20Inaba&amp;author=T.%20Nishio&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=311-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19884167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkin</div> <div class="given-names">I.A.P.</div></div>,  <div class="name"><div class="surname">Gulden</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Sharpe</div> <div class="given-names">A.G.</div></div>, et al. </div><div class="article-title">Segmental structure of the <em>Brassica napus</em> genome based on comparative analysis with <em>Arabidopsis thaliana</em></div>, <div class="source">Genetics</div> , <div class="year">2005</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">765</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.105.042093" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.105.042093</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segmental%20structure%20of%20the%20Brassica%20napus%20genome%20based%20on%20comparative%20analysis%20with%20Arabidopsis%20thaliana&amp;author=I.A.P.%20Parkin&amp;author=S.M.%20Gulden&amp;author=A.G.%20Sharpe&amp;publication_year=2005&amp;journal=Genetics&amp;volume=171&amp;pages=765-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.105.042093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.105.042093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.105.042093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16020789" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Panjabi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Jagannath</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bisht</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Comparative mapping of <em>Brassica juncea</em> and <em>Arabidopsis thaliana</em> using Intron Polymorphism (IP) markers: homoeologous relationships, diversification and evolution of the A, B and C Brassica genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">9</div> pg. <div class="fpage">113</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-9-113" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-9-113</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20of%20Brassica%20juncea%20and%20Arabidopsis%20thaliana%20using%20Intron%20Polymorphism%20(IP)%20markers%3A%20homoeologous%20relationships%2C%20diversification%20and%20evolution%20of%20the%20A%2C%20B%20and%20C%20Brassica%20genomes&amp;author=P.%20Panjabi&amp;author=A.%20Jagannath&amp;author=N.%20Bisht&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=113" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18315867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mun</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Kwon</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">T.J.</div></div>, et al. </div><div class="article-title">Genome-wide comparative analysis of the <em>Brassica rapa</em> gene space reveals genome shrinkage and differential loss of duplicated genes after whole genome triplication</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R111</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2009-10-10-r111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2009-10-10-r111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20analysis%20of%20the%20Brassica%20rapa%20gene%20space%20reveals%20genome%20shrinkage%20and%20differential%20loss%20of%20duplicated%20genes%20after%20whole%20genome%20triplication&amp;author=J.H.%20Mun&amp;author=S.J.%20Kwon&amp;author=T.J.%20Yang&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-10-r111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-10-r111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-10-r111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19821981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lagercrantz</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Lydiate</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Comparative genome mapping in <em>Brassica</em></div>, <div class="source">Genetics</div> , <div class="year">1996</div>, vol. <div class="volume">144</div> (pg. <div class="fpage">1903</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20mapping%20in%20Brassica&amp;author=U.%20Lagercrantz&amp;author=D.J.%20Lydiate&amp;publication_year=1996&amp;journal=Genetics&amp;volume=144&amp;pages=1903-10" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8978073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schranz</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Lysak</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Mitchell-Olds</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The ABC's of comparative genomics in the Brassicaceae: building blocks of crucifer genomes</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">535</div>-<div class="lpage">42</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.tplants.2006.09.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.tplants.2006.09.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ABC's%20of%20comparative%20genomics%20in%20the%20Brassicaceae%3A%20building%20blocks%20of%20crucifer%20genomes&amp;author=M.E.%20Schranz&amp;author=M.A.%20Lysak&amp;author=T.%20Mitchell-Olds&amp;publication_year=2006&amp;journal=Trends%20Plant%20Sci.&amp;volume=11&amp;pages=535-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2006.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2006.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2006.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17029932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">K.B.</div></div>, et al. </div><div class="article-title">The Korea Brassica Genome Project: a glimpse of the <em>Brassica</em> genome based on comparative genome analysis with <em>Arabidopsis</em></div>, <div class="source">Comp. Funct. Genomics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">138</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/cfg.465" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/cfg.465</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Korea%20Brassica%20Genome%20Project%3A%20a%20glimpse%20of%20the%20Brassica%20genome%20based%20on%20comparative%20genome%20analysis%20with%20Arabidopsis&amp;author=T.J.%20Yang&amp;author=J.S.%20Kim&amp;author=K.B.%20Lim&amp;publication_year=2005&amp;journal=Comp.%20Funct.%20Genomics&amp;volume=6&amp;pages=138-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/cfg.465" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcfg.465" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fcfg.465"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18629219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The genome of the mesopolyploid crop species <em>Brassica rapa</em></div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">1035</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.919" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.919</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20mesopolyploid%20crop%20species%20Brassica%20rapa&amp;author=X.%20Wang&amp;author=H.%20Wang&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=1035-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21873998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Map based cloning and characterization of a gene controlling hairiness and seed coat color traits in <em>Brassica rapa</em></div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">553</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-008-9437-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-008-9437-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Map%20based%20cloning%20and%20characterization%20of%20a%20gene%20controlling%20hairiness%20and%20seed%20coat%20color%20traits%20in%20Brassica%20rapa&amp;author=J.%20Zhang&amp;author=Y.%20Lu&amp;author=Y.%20Yuan&amp;publication_year=2009&amp;journal=Plant%20Mol.%20Biol.&amp;volume=69&amp;pages=553-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9437-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9437-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9437-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19039665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">D.V.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Genic microsatellite markers in <em>Brassica rapa</em>: development, characterization, mapping, and their utility in other cultivated and wild <em>Brassica</em> relatives</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">305</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsr017" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsr017</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genic%20microsatellite%20markers%20in%20Brassica%20rapa%3A%20development%2C%20characterization%2C%20mapping%2C%20and%20their%20utility%20in%20other%20cultivated%20and%20wild%20Brassica%20relatives&amp;author=N.%20Ramchiary&amp;author=D.V.%20Nguyen&amp;author=X.%20Li&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=305-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21768136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Basten</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">Z.B.</div></div>. </div>, <div class="source">Windows QTL Cartographer 2.5</div> , <div class="year">2005</div><div class="publisher-loc">Raleigh, NC</div><div class="publisher-name">North Carolina State University</div></div></div></div></div><div content-id="DSS029C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramchiary</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Padmaja</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Sharma</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Mapping of yield influencing QTL in <em>Brassica juncea</em>: implications for breeding of a major oilseed crop of dryland areas</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">807</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-007-0610-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-007-0610-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20yield%20influencing%20QTL%20in%20Brassica%20juncea%3A%20implications%20for%20breeding%20of%20a%20major%20oilseed%20crop%20of%20dryland%20areas&amp;author=N.%20Ramchiary&amp;author=K.L.%20Padmaja&amp;author=S.%20Sharma&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=807-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0610-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0610-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0610-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646960" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quijada</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (<em>Brassica napus</em> L.): 1. Identification of genomic regions from winter germplasm</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">549</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0323-1" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0323-1</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20analysis%20of%20seed%20yield%20and%20other%20complex%20traits%20in%20hybrid%20spring%20rapeseed%20(Brassica%20napus%20L.)%3A%201.%20Identification%20of%20genomic%20regions%20from%20winter%20germplasm&amp;author=P.A.%20Quijada&amp;author=J.A.%20Udall&amp;author=B.%20Lambert&amp;author=T.C.%20Osborn&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=549-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0323-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0323-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0323-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16767447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Udall</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Quijada</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Quantitative trait analysis of seed yield and other complex traits in hybrid spring rapeseed (<em>Brassica napus</em> L.): 2. Identification of alleles from unadapted germplasm</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2006</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">597</div>-<div class="lpage">609</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-006-0324-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-006-0324-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20analysis%20of%20seed%20yield%20and%20other%20complex%20traits%20in%20hybrid%20spring%20rapeseed%20(Brassica%20napus%20L.)%3A%202.%20Identification%20of%20alleles%20from%20unadapted%20germplasm&amp;author=J.A.%20Udall&amp;author=P.A.%20Quijada&amp;author=B.%20Lambert&amp;author=T.C.%20Osborn&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=597-609" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0324-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0324-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0324-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16767446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tzvetkova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Morgenstern</div> <div class="given-names">B.</div></div>. </div><div class="article-title">AUGUSTUS at EGASP: using EST, protein and genomic alignments for improved gene prediction in the human genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">S11</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2006-7-s1-s11" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2006-7-s1-s11</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AUGUSTUS%20at%20EGASP%3A%20using%20EST%2C%20protein%20and%20genomic%20alignments%20for%20improved%20gene%20prediction%20in%20the%20human%20genome&amp;author=M.%20Stanke&amp;author=A.%20Tzvetkova&amp;author=B.%20Morgenstern&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=S11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-s1-s11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-s1-s11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-s1-s11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16925833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R25</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2009-10-3-r25" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2009-10-3-r25</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kristiansen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1851</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.078212.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.078212.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20short%20DNA%20sequencing%20reads%20and%20calling%20variants%20using%20mapping%20quality%20scores&amp;author=R.%20Li&amp;author=Y.%20Li&amp;author=K.%20Kristiansen&amp;author=J.%20Wang&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1851-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078212.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078212.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078212.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18714091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Koeyer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Douglass</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Murphy</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Application of high-resolution DNA melting for genotyping and variant scanning of diploid and autotetraploid potato</div>, <div class="source">Mol. Breed.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">67</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-009-9309-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-009-9309-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20high-resolution%20DNA%20melting%20for%20genotyping%20and%20variant%20scanning%20of%20diploid%20and%20autotetraploid%20potato&amp;author=D.%20De%20Koeyer&amp;author=K.%20Douglass&amp;author=A.%20Murphy&amp;publication_year=2010&amp;journal=Mol.%20Breed.&amp;volume=25&amp;pages=67-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-009-9309-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-009-9309-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-009-9309-4"> </span></div> </div></div></div></div></div><div content-id="DSS029C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Lim</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">Transcriptome analysis in <em>Brassica rapa</em> under the abiotic stresses using Brassica 24K oligo microarray</div>, <div class="source">Mol. Cells</div> , <div class="year">2008</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">595</div>-<div class="lpage">605</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20in%20Brassica%20rapa%20under%20the%20abiotic%20stresses%20using%20Brassica%2024K%20oligo%20microarray&amp;author=S.C.%20Lee&amp;author=M.H.%20Lim&amp;author=J.A.%20Kim&amp;publication_year=2008&amp;journal=Mol.%20Cells&amp;volume=26&amp;pages=595-605" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18797175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plackett</div> <div class="given-names">A.R.G.</div></div>,  <div class="name"><div class="surname">Powers</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Fernandez-Garcia</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Analysis of the developmental roles of the <em>Arabidopsis</em> gibberellin 20-oxidases demonstrates that GA20ox1, -2, and -3 are the dominant paralogs</div>, <div class="source">Plant Cell</div> , <div class="year">2012</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">941</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.111.095109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.111.095109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20developmental%20roles%20of%20the%20Arabidopsis%20gibberellin%2020-oxidases%20demonstrates%20that%20GA20ox1%2C%20-2%2C%20and%20-3%20are%20the%20dominant%20paralogs&amp;author=A.R.G.%20Plackett&amp;author=S.J.%20Powers&amp;author=N.%20Fernandez-Garcia&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=941-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.095109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.095109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.095109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22427334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashikari</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakakibara</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Cytokinin oxidase regulates rice grain production</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">309</div> (pg. <div class="fpage">741</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1113373" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1113373</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytokinin%20oxidase%20regulates%20rice%20grain%20production&amp;author=M.%20Ashikari&amp;author=H.%20Sakakibara&amp;author=S.%20Lin&amp;publication_year=2005&amp;journal=Science&amp;volume=309&amp;pages=741-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1113373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1113373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1113373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15976269" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bartrina</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Otto</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Strnad</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Werner</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Schmulling</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Cytokinin regulates the activity of reproductive meristems, flower organ size, ovule formation, and thus seed yield in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell</div> , <div class="year">2011</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">69</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.110.079079" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.110.079079</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytokinin%20regulates%20the%20activity%20of%20reproductive%20meristems%2C%20flower%20organ%20size%2C%20ovule%20formation%2C%20and%20thus%20seed%20yield%20in%20Arabidopsis%20thaliana&amp;author=I.%20Bartrina&amp;author=E.%20Otto&amp;author=M.%20Strnad&amp;author=T.%20Werner&amp;author=T.%20Schmulling&amp;publication_year=2011&amp;journal=Plant%20Cell&amp;volume=23&amp;pages=69-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.110.079079" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.110.079079" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.110.079079"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21224426" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Roxrud</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lid</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Fletcher</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Opsahl-Sorteberg</div> <div class="given-names">H.G.</div></div>. </div><div class="article-title">GASA4, one of the 14-member Arabidopsis GASA family of small polypeptides, regulates flowering and seed development</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">48</div> (pg. <div class="fpage">471</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcm016" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/pcp/pcm016</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GASA4%2C%20one%20of%20the%2014-member%20Arabidopsis%20GASA%20family%20of%20small%20polypeptides%2C%20regulates%20flowering%20and%20seed%20development&amp;author=I.%20Roxrud&amp;author=S.E.%20Lid&amp;author=J.C.%20Fletcher&amp;author=E.D.%20Schmidt&amp;author=H.G.%20Opsahl-Sorteberg&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol.&amp;volume=48&amp;pages=471-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17284469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kim</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kende</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The AtGRF family of putative transcription factors is involved in leaf and cotyledon growth in Arabidopsis</div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">94</div>-<div class="lpage">104</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.2003.01862.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-313X.2003.01862.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20AtGRF%20family%20of%20putative%20transcription%20factors%20is%20involved%20in%20leaf%20and%20cotyledon%20growth%20in%20Arabidopsis&amp;author=J.H.%20Kim&amp;author=D.%20Choi&amp;author=H.%20Kende&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=36&amp;pages=94-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01862.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01862.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01862.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12974814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hay</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Barkoulas</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tsiantis</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Asymmetric leaves1 and auxin activities converge to repress <em>BREVIPEDICELLUS</em> expression and promote leaf development in <em>Arabidopsis</em></div>, <div class="source">Development</div> , <div class="year">2006</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">3955</div>-<div class="lpage">61</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1242/dev.02545" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1242/dev.02545</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20leaves1%20and%20auxin%20activities%20converge%20to%20repress%20BREVIPEDICELLUS%20expression%20and%20promote%20leaf%20development%20in%20Arabidopsis&amp;author=A.%20Hay&amp;author=M.%20Barkoulas&amp;author=M.%20Tsiantis&amp;publication_year=2006&amp;journal=Development&amp;volume=133&amp;pages=3955-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.02545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.02545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.02545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16971475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rieu</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Eriksson</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Powers</div> <div class="given-names">S.J.</div></div>, et al. </div><div class="article-title">Genetic analysis reveals that C19-GA 2-oxidation is a major gibberellin inactivation pathway in <em>Arabidopsis</em></div>, <div class="source">Plant Cell</div> , <div class="year">2008</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">2420</div>-<div class="lpage">36</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.108.058818" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.108.058818</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20reveals%20that%20C19-GA%202-oxidation%20is%20a%20major%20gibberellin%20inactivation%20pathway%20in%20Arabidopsis&amp;author=I.%20Rieu&amp;author=S.%20Eriksson&amp;author=S.J.%20Powers&amp;publication_year=2008&amp;journal=Plant%20Cell&amp;volume=20&amp;pages=2420-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.058818" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.058818" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.058818"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18805991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Woodhouse</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Cheng</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Altered patterns of fractionation and exon deletions in <em>Brassica rapa</em> support a two-step model of paleohexaploidy</div>, <div class="source">Genetics</div> , <div class="year">2012</div>, vol. <div class="volume">190</div> (pg. <div class="fpage">1563</div>-<div class="lpage">74</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/genetics.111.137349" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/genetics.111.137349</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20patterns%20of%20fractionation%20and%20exon%20deletions%20in%20Brassica%20rapa%20support%20a%20two-step%20model%20of%20paleohexaploidy&amp;author=H.%20Tang&amp;author=M.R.%20Woodhouse&amp;author=F.%20Cheng&amp;publication_year=2012&amp;journal=Genetics&amp;volume=190&amp;pages=1563-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.111.137349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.111.137349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.111.137349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22308264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS029C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS029C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schranz</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Quijada</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sung</div> <div class="given-names">S-B.</div></div>,  <div class="name"><div class="surname">Lukens</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Amasino</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Osborn</div> <div class="given-names">T.C.</div></div>. </div><div class="article-title">Characterization and effects of the replicated flowering time gene FLC in <em>Brassica rapa</em></div>, <div class="source">Genetics</div> , <div class="year">2002</div>, vol. <div class="volume">162</div> (pg. <div class="fpage">1457</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20effects%20of%20the%20replicated%20flowering%20time%20gene%20FLC%20in%20Brassica%20rapa&amp;author=M.E.%20Schranz&amp;author=P.%20Quijada&amp;author=S-B.%20Sung&amp;author=L.%20Lukens&amp;author=R.%20Amasino&amp;author=T.C.%20Osborn&amp;publication_year=2002&amp;journal=Genetics&amp;volume=162&amp;pages=1457-68" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12454088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Masahiro Yano</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=381937&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp_table4.xls?Expires=2147483647&amp;Signature=VA8otW5fPlii40k2BZdu0fUXdWYeg9DJW83S9HmX~V7oi9NOYAkdZHRXor00bUVoEje~aT-xF125lKHoPImltmYEy5oYIqdFaFQS5xWAsHcPyHE6oqoSRu~RcsMw7VbTzB3dGFoHY2gethPVtIa7dp1XO0X8nTsosLJQyB0kkijOZlQD5XFvukRUwICT401alLeu3MRt67B3h23BDcYbqMDOP8cMzzDOXtPVYfRNwVBMUW5Tsn9pJomsLZdkRLgR6eTeb4PYTp-29OldYy3xZVm3gjwxjWt2QhAf3JPQDpVlYzUV1S7uS7PfK6CrGPp4saDB5mBbA0zwKdJlabNi-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp_table3.xls?Expires=2147483647&amp;Signature=U5hG--cLM9zF~6Uuzv2CAaaEQgRCV1QCNv8kQ-KluSJVUrWQYTHzGkHpwcdzCY10WsxN04HdQ9-y6m78ZsYyp0zFRmcOsn7Q8CUzyK7EZKAlm5tmXtFo0XMQsqjHETjsOEfymK-RRcnT~vQVOQ6bflvH6Xg1LKh80jMhZa8OHb2XPPKobIK3Can1pjaoqi0QCAYc0Gur9EbL9bwaiw-Oy~x9QrFYbbbav9gJAv9IclgvEiMheyb8VevJwp8FPtBolo~UeFsfQDfFN~yHBKduPq13cR4K1GxYXck1Aim7ZnFuYiunk26LTN1PbB3KTpK~K4ZX5jSrVytjm-uOF8hfpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp_table2.doc?Expires=2147483647&amp;Signature=Evq9QLG7DkV4HgSd6qon~wt1fpuQygoKOgCm-t45YkHsTWymhjuIvEYwsGZslIdYDnI-RhlvNjGNQUcCc8munZi3sKIgbZ9~6GoAerWRJzPTntUBES23-eckVJ-IeNIW0pVBsJvnIgEKkb9DZubUQub3yDAYOIRroqyVt9jSqOweYoGS-eXCaEPb-fWysyihmZh8fWOVxs7gq6rEli8HaI9s0237g6iQ4IQV2OPbyQmKQt6rTdINO0MTnlldS-BDLNAqjopdU7j07DlWQCbu-~tvE3DewIs1g5bK5XLFq0A-sh1naBXeg1eDMYJg1O7mXsxUg5~GZVCBEc03S9rhMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp_table1.doc?Expires=2147483647&amp;Signature=5MeW8lpVfaOp0hWCQp0wEeBvYpZKmrqKDDFrWsRLvtuaDC6r5kdb~iAMKVnV2sKFaLPaRfcnh5WMCWJG16MAHz3~MXw2YXVRvbrle6ATBDTjPY8FJuoImGrUp0ucwnfDAsAFwmB2us7MtTs1fuGwBey6xKH~i8KLxuT9hHR5LzLRXpqZ9r4tSnwL1c6XmKma0VjECUGXb-R1P2GdkjCBtzbDPMtQeNZhRg8H4lrz0PIPSqyX3LpzweB6AlZBjZ0-Y8DKLAJV5toBvY8TJGYVeO0bS5UVmBbu7zZiRAW1-m34Gi0jfONui3to2Ozq~TURZL3zbLmGbwE0zA9~02PlhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp_fig1.doc?Expires=2147483647&amp;Signature=4ZwdEIy7ds9uNSBPWUx4fnaPUsJWcHS8etEGwrsCWJvU5Ic23HPYFhRdBk-799sBV-lvWAGZ8Ig5ouudXEj~2ngvPXtN3tHh80Rx~Ty5fNh2GrCDs3kgIPXNqccN~HnXJXKGQFxjMku3dpTptMkHXg8xxuDDKli2zOlX0RSkJMFDmD~LcNUUErPOAWGRc~H-bxoIfo-Tht0Ya3smnj3Bl62OCo1H6gOHb6JeZcVUK3USSIWW949bOAKah23VAPrBxChV~WQ2QbRfAZRgDqPb1iRWXPNZu4bxU21GX~LwUR03MWq5xozJjQsTNqmKaQK1qw6-j~986zY7oVczNvcRnA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/1/10.1093/dnares/dss029/2/dss029-dss029supp.txt?Expires=2147483647&amp;Signature=4ZlB7F96GVE77ACYAQ9CLnTx1JPi0VOETTOl0p~1rmxUauxVaEZTPxPfAstynAUSNP8BLKZJkHkmy1HJ3LR5NHBr~XphSzWOFuXO0BS8cLPtDAPV1MgQ6dqjprcQQgosJO7HVrEfOnVWVdfkxzDJtI6uGxJpDakLig5Msa4cMJXAJvaSWeByyjyPVGW8dR9kRBeqIcJ29E3rUG8gMfvJg-JosRKsIY29hIst5EJ~VF5LRxDq0W0iAbkhjTSwhSzT2As2UbfR6ODSy03bWFPNlgJzs2mRshpeVzsEZxN44Pz3ggiZaUTDQoOyCKV5zDXgIHs51BSUQS-ek~CaUMXrYw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - txt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">403</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">34</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss029"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="381937"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000315666200001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f1%2f1%2f381937">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000315666200001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (34)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F1%2F1%2F381937" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/381937'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Quantitative Trait Loci Mapping in Brassica rapa Revealed the Structural and Functional Conservation of Genetic Loci Governing Morphological and Yield Component Traits in the A, B, and C Subgenomes of Brassica Species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss029");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f1%2f1%2f381937">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

